
<html lang="en"     class="pb-page"  data-request-id="d07da6e4-006d-46ab-9926-accf573e0a8f"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.6b01645;issue:issue:10.1021/jmcmar.2017.60.issue-5;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Discovery of Potent and Selective Inhibitors for G9a-Like Protein (GLP) Lysine Methyltransferase" /></meta><meta name="dc.Creator" content="Yan  Xiong" /></meta><meta name="dc.Creator" content="Fengling  Li" /></meta><meta name="dc.Creator" content="Nicolas  Babault" /></meta><meta name="dc.Creator" content="Aiping  Dong" /></meta><meta name="dc.Creator" content="Hong  Zeng" /></meta><meta name="dc.Creator" content="Hong  Wu" /></meta><meta name="dc.Creator" content="Xin  Chen" /></meta><meta name="dc.Creator" content="Cheryl H.  Arrowsmith" /></meta><meta name="dc.Creator" content="Peter J.  Brown" /></meta><meta name="dc.Creator" content="Jing  Liu" /></meta><meta name="dc.Creator" content="Masoud  Vedadi" /></meta><meta name="dc.Creator" content="Jian  Jin" /></meta><meta name="dc.Description" content="G9a-like protein (GLP) and G9a are highly homologous protein lysine methyltransferases (PKMTs) sharing approximately 80% sequence identity in their catalytic domains. GLP and G9a form a heterodimer..." /></meta><meta name="Description" content="G9a-like protein (GLP) and G9a are highly homologous protein lysine methyltransferases (PKMTs) sharing approximately 80% sequence identity in their catalytic domains. GLP and G9a form a heterodimer..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="February 14, 2017" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.6b01645" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2017 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.6b01645" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.6b01645" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.6b01645" /></link>
        
    
    

<title>Discovery of Potent and Selective Inhibitors for G9a-Like Protein (GLP) Lysine Methyltransferase | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.6b01645" /></meta><meta property="og:title" content="Discovery of Potent and Selective Inhibitors for G9a-Like Protein (GLP) Lysine Methyltransferase" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/large/jm-2016-01645c_0016.jpeg" /></meta><meta property="og:description" content="G9a-like protein (GLP) and G9a are highly homologous protein lysine methyltransferases (PKMTs) sharing approximately 80% sequence identity in their catalytic domains. GLP and G9a form a heterodimer complex and catalyze mono- and dimethylation of histone H3 lysine 9 and nonhistone substrates. Although they are closely related, GLP and G9a possess distinct physiological and pathophysiological functions. Thus, GLP or G9a selective small-molecule inhibitors are useful tools to dissect their distinct biological functions. We previously reported potent and selective G9a/GLP dual inhibitors including UNC0638 and UNC0642. Here we report the discovery of potent and selective GLP inhibitors including 4 (MS0124) and 18 (MS012), which are &gt;30-fold and 140-fold selective for GLP over G9a and other methyltransferases, respectively. The cocrystal structures of GLP and G9a in the complex with either 4 or 18 displayed virtually identical binding modes and interactions, highlighting the challenges in structure-based design of selective inhibitors for either enzyme." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.6b01645"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.6b01645">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.6b01645&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.6b01645&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.6b01645&amp;href=/doi/10.1021/acs.jmedchem.6b01645" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2017</span><span class="cit-fg-volume">, 60</span><span class="cit-fg-issue">, 5</span><span class="cit-fg-pageRange">, 1876-1891</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/60/5" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.6b01668" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.6b01670" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of Potent and Selective Inhibitors for G9a-Like Protein (GLP) Lysine Methyltransferase</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Yan++Xiong">Yan Xiong</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Fengling++Li">Fengling Li</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Nicolas++Babault">Nicolas Babault</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Aiping++Dong">Aiping Dong</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Hong++Zeng">Hong Zeng</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Hong++Wu">Hong Wu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Xin++Chen">Xin Chen</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Cheryl+H.++Arrowsmith">Cheryl H. Arrowsmith</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Peter+J.++Brown">Peter J. Brown</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jing++Liu">Jing Liu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Masoud++Vedadi">Masoud Vedadi</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jian++Jin">Jian Jin</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0002-2387-3862" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Department of Pharmacological Sciences and Department of Oncological Sciences, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">§</span> <span class="aff-text">Department of Pharmacology and Toxicology, University of Toronto, Toronto, Ontario M5S 1A8, Canada</span></div><div class="aff-info" id="aff4"><span class="aff-symbol">⊥</span> <span class="aff-text">Princess Margaret Cancer Centre and Department of Medical Biophysics, University of Toronto, Toronto, Ontario M5G 2M9, Canada</span></div><div class="corresp-info"><strong>*</strong>For J.J.: phone, (212) 659-8699; fax, (212) 849-2456; E-mail: <a href="/cdn-cgi/l/email-protection#49232028276723202709243a3a24672c2d3c"><span class="__cf_email__" data-cfemail="8ae0e3ebe4a4e0e3e4cae7f9f9e7a4efeeff">[email protected]</span></a></div><div class="corresp-info"><strong>*</strong>M.V.: phone, (416) 976-0897; e-mail, <a href="/cdn-cgi/l/email-protection#c0adeeb6a5a4a1a4a980b5b4afb2afaeb4afeea3a1"><span class="__cf_email__" data-cfemail="c6abe8b0a3a2a7a2af86b3b2a9b4a9a8b2a9e8a5a7">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.6b01645&amp;href=/doi/10.1021%2Facs.jmedchem.6b01645" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2017</span></span><span class="cit-volume">, 60</span><span class="cit-issue">, 5</span><span class="cit-pageRange">, 1876–1891</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">January 30, 2017</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>15 November 2016</li><li><span class="item_label"><b>Published</b> online</span>14 February 2017</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 9 March 2017</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.6b01645" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b01645</a></div><div class="article_header-article-copyright"><strong>Copyright © 2017 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D1876%26pageCount%3D16%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DYan%2BXiong%252C%2BFengling%2BLi%252C%2BNicolas%2BBabault%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D60%26issueNum%3D5%26contentID%3Dacs.jmedchem.6b01645%26title%3DDiscovery%2Bof%2BPotent%2Band%2BSelective%2BInhibitors%2Bfor%2BG9a-Like%2BProtein%2B%2528GLP%2529%2BLysine%2BMethyltransferase%26numPages%3D16%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D1891%26publicationDate%3DMarch%2B2017">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.6b01645"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2574</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">31</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.6b01645" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of Potent and Selective Inhibitors for G9a-Like Protein (GLP) Lysine Methyltransferase&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Yan&quot;,&quot;last_name&quot;:&quot;Xiong&quot;},{&quot;first_name&quot;:&quot;Fengling&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Nicolas&quot;,&quot;last_name&quot;:&quot;Babault&quot;},{&quot;first_name&quot;:&quot;Aiping&quot;,&quot;last_name&quot;:&quot;Dong&quot;},{&quot;first_name&quot;:&quot;Hong&quot;,&quot;last_name&quot;:&quot;Zeng&quot;},{&quot;first_name&quot;:&quot;Hong&quot;,&quot;last_name&quot;:&quot;Wu&quot;},{&quot;first_name&quot;:&quot;Xin&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Cheryl&quot;,&quot;last_name&quot;:&quot;H. Arrowsmith&quot;},{&quot;first_name&quot;:&quot;Peter&quot;,&quot;last_name&quot;:&quot;J. Brown&quot;},{&quot;first_name&quot;:&quot;Jing&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Masoud&quot;,&quot;last_name&quot;:&quot;Vedadi&quot;},{&quot;first_name&quot;:&quot;Jian&quot;,&quot;last_name&quot;:&quot;Jin&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2017&quot;,&quot;month&quot;:&quot;02&quot;,&quot;day&quot;:&quot;14&quot;,&quot;issue&quot;:&quot;5&quot;,&quot;volume&quot;:&quot;60&quot;,&quot;pages&quot;:&quot;1876-1891&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.6b01645&quot;},&quot;abstract&quot;:&quot;G9a-like protein (GLP) and G9a are highly homologous protein lysine methyltransferases (PKMTs) sharing approximately 80% sequence identity in their catalytic domains. GLP and G9a form a heterodimer complex and catalyze mono- and dimethylation of histone H3 lysine 9 and nonhistone substrates. Although they are closely related, GLP and G9a possess distinct physiological and pathophysiological functions. Thus, GLP or G9a selective small-molecule inhibitors are useful tools to dissect their distinct biological functions. We previously reported potent and selective G9a/GLP dual inhibitors including UNC0638 and UNC0642. Here we report the discovery of potent and selective GLP inhibitors including 4 (MS0124) and 18 (MS012), which are &gt;30-fold and 140-fold selective for GLP over G9a and other methyltransferases, respectively. The cocrystal structures of GLP and G9a in the complex with either 4 or 18 displayed virtually identical binding modes and interactions, highlighting the challenges in structure-based design of &quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b01645&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01645" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b01645&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01645" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b01645&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01645" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.6b01645&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01645&amp;href=/doi/10.1021/acs.jmedchem.6b01645" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.6b01645" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.6b01645" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b01645&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.6b01645%26sid%3Dliteratum%253Aachs%26pmid%3D28135087%26genre%3Darticle%26aulast%3DXiong%26date%3D2017%26atitle%3DDiscovery%2Bof%2BPotent%2Band%2BSelective%2BInhibitors%2Bfor%2BG9a-Like%2BProtein%2B%2528GLP%2529%2BLysine%2BMethyltransferase%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D60%26issue%3D5%26spage%3D1876%26epage%3D1891%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=290831" title="Monomers">Monomers</a>,</li><li><a href="/action/doSearch?ConceptID=291465" title="Structure activity relationship">Structure activity relationship</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=292349" title="Alkyls">Alkyls</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/60/5" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/jmcmar.2017.60.issue-5/20170309/jmcmar.2017.60.issue-5.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/medium/jm-2016-01645c_0016.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/large/jm-2016-01645c_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01645&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">G9a-like protein (GLP) and G9a are highly homologous protein lysine methyltransferases (PKMTs) sharing approximately 80% sequence identity in their catalytic domains. GLP and G9a form a heterodimer complex and catalyze mono- and dimethylation of histone H3 lysine 9 and nonhistone substrates. Although they are closely related, GLP and G9a possess distinct physiological and pathophysiological functions. Thus, GLP or G9a selective small-molecule inhibitors are useful tools to dissect their distinct biological functions. We previously reported potent and selective G9a/GLP dual inhibitors including UNC0638 and UNC0642. Here we report the discovery of potent and selective GLP inhibitors including <b>4</b> (MS0124) and <b>18</b> (MS012), which are >30-fold and 140-fold selective for GLP over G9a and other methyltransferases, respectively. The cocrystal structures of GLP and G9a in the complex with either <b>4</b> or <b>18</b> displayed virtually identical binding modes and interactions, highlighting the challenges in structure-based design of selective inhibitors for either enzyme.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_61662" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_61662" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Protein lysine methyltransferases (PKMTs) catalyze the transfer of the methyl group from the cofactor <i>S</i>-5′-adenosyl-<span class="smallcaps smallerCapital">l</span>-methionine (SAM) to the ε-amino group of the targeted lysine residue in histone and nonhistone substrates, leading to mono-, di-, or trimethylation.<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1-3)</a> Lysine methylation does not change the charge state of the residue. Instead, this modification increases the bulkiness of the amino group and alters the hydrophobicity and hydrogen bond formation potential.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> As a result, lysine methylation has dramatic effects on the interactions of the modified protein with other proteins, thus its physiological function.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Depending on the methylation state and location, histone lysine methylation can lead to either activation or repression of gene expression and transcription.<a onclick="showRef(event, 'ref3 ref4 ref5 ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref3 ref4 ref5 ref6 ref7">(3-7)</a> In general, methylation of histone H3 lysine 4, 36, and 79 (H3K4, H3K36, and H3K79) is associated with transcription activation and methylation of histone H3 lysine 9 and 27 (H3K9 and H3K27) is typically a hallmark of transcription repression.<a onclick="showRef(event, 'ref8 ref9 ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref8 ref9 ref10 ref11">(8-11)</a></div><div class="NLM_p">GLP (also known as euchromatic histone-lysine <i>N</i>-methyltransferase 1 (EHMT1) or lysine methyltransferase 1D (KMT1D)) and G9a (also known as EHMT2 or KMT1C) are two closely related PKMTs with approximately 80% sequence identity in their respective suppressor of variegation 3–9 enhancer-of-zeste trithorax (SET) domains.<a onclick="showRef(event, 'ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref12 ref13">(12, 13)</a> It has been shown that the heterodimeric complex of G9a and GLP is the main functional methyltransferase in mouse embryonic stem cells, depositing mono- and dimethylated H3K9 (H3K9me1 and H3K9me2).<a onclick="showRef(event, 'ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref14 ref15">(14, 15)</a> Besides H3K9, other histone substrates, such as histone H1,<a onclick="showRef(event, 'ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref16 ref17">(16, 17)</a> H3K27,<a onclick="showRef(event, 'ref18 ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref18 ref19 ref20">(18-20)</a> and H3K56,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> have been reported. G9a and/or GLP can also methylate a variety of nonhistone proteins,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> including the tumor suppressor p53,<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> sirtuin 1 (SIRT1),<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> reptin,<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> myogenic differentiation 1 (MyoD),<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> chromodomain Y-like protein (CDYL1),<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> widely interspaced zinc finger motifs protein (WIZ),<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> and G9a.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Furthermore, G9a and/or GLP have been reported to play roles<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> in germ cell development and meiosis,<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> embryo development,<a onclick="showRef(event, 'ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref14 ref15">(14, 15)</a> tumor cell growth and metathesis,<a onclick="showRef(event, 'ref23 ref31 ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref23 ref31 ref32 ref33">(23, 31-33)</a> immune response,<a onclick="showRef(event, 'ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref34 ref35">(34, 35)</a> cocaine-induced plasticity,<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> cognition and adaptive behavior,<a onclick="showRef(event, 'ref4 ref37'); return false;" href="javascript:void(0);" class="ref ref4 ref37">(4, 37)</a> and provirus silencing.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> Dysregulation of G9a and/or GLP has been implicated in many human diseases such as cancer, inflammatory diseases, and neurogenerative disorder.<a onclick="showRef(event, 'ref4 ref6 ref32 ref37 ref39 ref40'); return false;" href="javascript:void(0);" class="ref ref4 ref6 ref32 ref37 ref39 ref40">(4, 6, 32, 37, 39, 40)</a></div><div class="NLM_p">Although G9a and GLP share high sequence homology and many biological activities, they do possess distinct physiological and pathophysiological functions.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> For example, the ankyrin repeat domain of G9a mainly associates with H3K9me2, whereas the ankyrin domain of GLP preferentially associates with H3K9me1.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> Mutations at the ankyrin domain of G9a and GLP result in drastic phenotype differences in mice.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> In addition, G9a and GLP often display distinct tissue-specific expression profiles. Different from G9a, GLP is highly expressed in brown adipose tissue and controls brown adipose cell fate and energy homeostasis.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> Moreover, G9a and GLP have opposite roles during early steps of skeletal muscle terminal differentiation. GLP, but not G9a, stabilizes MyoD, which is the major player in triggering muscle terminal differentiation, by down-regulating proteasome subunit-encoding gene expression.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a></div><div class="NLM_p">Potent and selective small-molecule inhibitors of G9a/GLP are useful tools for investigating biological functions of these proteins and testing the therapeutic hypotheses associated with them. Over the past decade, a number of G9a/GLP inhibitors have been reported.<a onclick="showRef(event, 'ref12 ref13 ref45 ref46 ref47 ref48 ref49 ref50 ref51 ref52 ref53 ref54 ref55 ref56'); return false;" href="javascript:void(0);" class="ref ref12 ref13 ref45 ref46 ref47 ref48 ref49 ref50 ref51 ref52 ref53 ref54 ref55 ref56">(12, 13, 45-56)</a> Compound <b>1</b> (BIX01294) was the first selective dual G9a/GLP inhibitor (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>).<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> It was discovered via high throughput screening and is competitive with the peptide substrate and noncompetitive with the cofactor SAM. On the basis of the crystal structure of the GLP–<b>1</b> complex,<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> our group and others have discovered more potent G9a/GLP inhibitors including UNC0224,<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> UNC0321,<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> and <b>2</b> (E72).<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> Further optimization of this chemical series led to the discovery of potent, selective, and cell-active G9a/GLP dual inhibitors <b>3</b> (UNC0638)<a onclick="showRef(event, 'ref48 ref49 ref57'); return false;" href="javascript:void(0);" class="ref ref48 ref49 ref57">(48, 49, 57)</a> and UNC0642,<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> which has sufficient in vivo pharmacokinetic properties and is suitable for animal studies (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>). Among all of these inhibitors, compound <b>1</b> is the only one that has some selectivity for GLP over G9a (about 3–5-fold).<a onclick="showRef(event, 'ref13 ref49'); return false;" href="javascript:void(0);" class="ref ref13 ref49">(13, 49)</a> Because G9a and GLP are not functionally redundant and have been shown to have distinct biological functions, G9a or GLP selective inhibitors will be valuable tools to study distinct functions of these two closely related PKMTs. Here, we describe the design, synthesis, and biochemical evaluation of new compounds based on the quinazoline scaffold shared by compounds <b>1</b>–<b>3</b>. Our structure–activity relationship (SAR) studies resulted in the discovery of potent and selective GLP inhibitors including <b>18</b> (MS012), which is remarkably selective for GLP over G9a (>140-fold) in biochemical assays.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/medium/jm-2016-01645c_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/large/jm-2016-01645c_0001.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of selected G9a/GLP inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/large/jm-2016-01645c_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01645&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_28017" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_28017" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Discovery of <b>4</b> (MS0124) as a GLP Selective Inhibitor</h3><div class="NLM_p">To identify selective inhibitors for GLP or G9a, we screened our quinazoline compound collection against these two enzymes using radioactivity based scintillation proximity biochemical assays, which measure the potency of test compounds at inhibiting the transfer of the methyl group from <sup>3</sup>H-SAM to the peptide substrates.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> We were pleased to find compound <b>4</b> was 34-fold more selective for GLP (IC<sub>50</sub> = 13 ± 4 nM) over G9a (IC<sub>50</sub> = 440 ± 63 nM) (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). We were also able to obtain cocrystal structures of both GLP and G9a in complex with <b>4</b> in the presence of SAM at resolutions of 2.6 and 1.95 Å, respectively (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a> and Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.6b01645/suppl_file/jm6b01645_si_001.pdf" class="ext-link">Table 1</a>). These two cocrystal structures are extremely similar, with a root-mean-square deviation (RMSD) of 0.31 Å. Compound <b>4</b> occupies the peptide substrate binding site with the same binding mode for GLP and G9a. The interactions between compound <b>4</b> and these two enzymes are conserved. For example, the secondary amino group at the C4 position forms a direct hydrogen bond with the side chain of GLP Asp1171 and the equivalent G9a Asp1083. In addition, the N1 nitrogen on the pyrimidine ring forms another direct hydrogen with the side chain of GLP Asp1176 and G9a Asp1088. Moreover, the oxygen atoms of both 6- and 7-methoxy groups interact with a water molecule in both structures.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. SAR of the <i>N</i>-Capping Group of the Piperidine Moiety<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/medium/jm-2016-01645c_0010.gif" alt="" id="fx1" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/medium/jm-2016-01645c_0011.gif" alt="" id="fx2" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>Table a</sup><p class="last">IC<sub>50</sub> determination experiments were performed at substrate and cofactor concentrations equal to the respective <i>K</i><sub>m</sub> values for each enzyme. IC<sub>50</sub> determination experiments were performed in triplicate and the values are presented as mean ± SD.</p></div></div><div></div></div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/medium/jm-2016-01645c_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/large/jm-2016-01645c_0002.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. X-ray cocrystal structures of (A) GLP (green) (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TUZ">5TUZ</a>) and (B) G9a (blue) (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TUY">5TUY</a>) in complex with <b>4</b> in the presence of SAM. Water molecule is illustrated as a red sphere. Main interactions are shown in yellow dashed lines. (C) Overlay of (A) and (B).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/large/jm-2016-01645c_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01645&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">To improve potency and selectivity of <b>4</b>, we conducted structure–activity relationship (SAR) studies at the following four regions: (1) <i>N</i>-capping group of the piperidine moiety, (2) 6-methoxy moiety, (3) 7-methoxy moiety, and (4) 2-morpholine moiety (<a class="ref internalNav" href="#tbl1" aria-label="Tables 1">Tables 1</a>–<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>).</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Synthesis</h3><div class="NLM_p">We previously developed a two-step synthetic sequence to explore the 2-amino and 4-amino regions of the 6,7-dimethoxyquinazoline scaffolds.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> Using this efficient synthesis, we converted the commercially available 2,4-dichloro-6,7-dimethoxyquinazoline <b>5</b> to compounds <b>4</b> and <b>7</b>–<b>22</b> by two consecutive chloro displacement reactions (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a> and <a class="ref internalNav" href="#tbl1" aria-label="Tables 1">Tables 1</a> and <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>).</div><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/medium/jm-2016-01645c_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/large/jm-2016-01645c_0007.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of <b>4</b> and <b>7</b>–<b>22</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/large/jm-2016-01645c_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01645&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) 1-alkylpiperidin-4-ylamine, K<sub>2</sub>CO<sub>3</sub>, DMF, rt, 80–90%; (b) R′ amines, 4 N HCl in dixoane, <i>i</i>-PrOH, microwave, 160 °C, 75–85%.</p></p></figure><div class="NLM_p">To investigate the SAR of 6- and 7-methoxy moieties, we synthesized analogues with ethoxy, isopropoxy, fluoro, and methyl groups at C6 and C7 positions (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a> and <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>). We developed four synthetic routes to prepare 4,5-disubtituted-2-cyanoaniline intermediates <b>29</b>, <b>30</b>, <b>35</b>, and <b>36</b>. Briefly, etherification of commercially available phenols, <b>23</b> and <b>24</b>, followed by nitration and subsequent nitro reduction produced 2-cyanoanilines (<b>29</b> and <b>30</b>) bearing different alkoxy groups (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>A). The fluoro and methyl substituted intermediates, <b>35</b> and <b>36</b>, were synthesized from anilines <b>31</b> and <b>32</b> through bromination and subsequent cyanation reactions (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>B). The 4,5-disubstituted-2-cyanoaniline intermediates <b>29</b>, <b>30</b>, <b>35</b>, and <b>36</b> were then reacted with methyl chloroformate to yield carbamates <b>37</b>. After saponification of the cyano group, cyclization reaction afforded quinazolinediones <b>38</b>. Chlorination and subsequent two consecutive chloro substitutions provided quinazolines <b>41</b>–<b>48</b> (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>C and <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>).</div><figure id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/medium/jm-2016-01645c_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/large/jm-2016-01645c_0008.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of <b>41</b>–<b>48</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/large/jm-2016-01645c_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01645&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) EtI or <i>i</i>-PrBr, K<sub>2</sub>CO<sub>3</sub>, acetone, reflux; (b) HNO<sub>3</sub>, Ac<sub>2</sub>O, 0 °C to rt, 81–94% over 2 steps; (c) Fe dust, NH<sub>4</sub>OAc, H<sub>2</sub>O, reflux, 60–70%; (d) Bu<sub>4</sub>NBr<sub>3</sub>, ethyl acetate, 0 °C to rt, 80–90%; (e) Zn(CN)<sub>2</sub>, Pd(pph<sub>3</sub>)<sub>4</sub>, DMF, microwave, 130 °C, 50–72%; (f) Br<sub>2</sub>, Ac<sub>2</sub>O, 50 °C, 64%; then HCl, EtOH, 95%; (g) Br<sub>2</sub>, K<sub>2</sub>CO<sub>3</sub>, DCM, −15 °C, 70%; (h) CuCN, DMF, sealed tube, 160 °C, 56%; (i) ClCO<sub>2</sub>Me, DIEA, DCM/DMF, 0 °C to rt; (j) H<sub>2</sub>O<sub>2</sub>, NaOH, H<sub>2</sub>O/MeOH, reflux; (k) PhNEt<sub>2</sub>, POCl<sub>3</sub>, reflux, 30–40% over 3 steps; (l) 1-methylpiperidin-4-amine, K<sub>2</sub>CO<sub>3</sub>, DMF, rt; (m) morpholine, 4 N HCl in dixoane, <i>i</i>-PrOH, microwave, 160 °C, 75–85% over 2 steps.</p></p></figure><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. SAR of the 6- and 7-Methoxy Moieties<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/medium/jm-2016-01645c_0012.gif" alt="" id="fx3" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/medium/jm-2016-01645c_0013.gif" alt="" id="fx4" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>Table a</sup><p class="last">IC<sub>50</sub> determination experiments were performed at substrate and cofactor concentrations equal to the respective <i>K</i><sub>m</sub> values for each enzyme. IC<sub>50</sub> determination experiments were performed in triplicate and the values are presented as mean ± SD.</p></div></div><div></div></div><div class="NLM_p">Quinazoline analogues with alkyl groups at the C2 position (<b>53</b> and <b>54</b>) were synthesized from aniline <b>49</b>. Amide coupling of <b>49</b> with acid chlorides provided intermediates <b>50</b>, which were then converted to <b>51</b> through the saponification and subsequent cyclization reaction. Chlorination followed by chloro displacement provided 2-alkyl analogues <b>53</b> and <b>54</b> (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a> and <a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>).</div><figure id="sch3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/medium/jm-2016-01645c_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/large/jm-2016-01645c_0009.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of <b>53</b> and <b>54</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/large/jm-2016-01645c_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01645&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) acid chloride, DIEA, DCM/DMF, 0 °C to rt, 90–95%; (b) H<sub>2</sub>O<sub>2</sub>, NaOH, H<sub>2</sub>O/MeOH, reflux, 46–55% over 2 steps; (c) PhNEt<sub>2</sub>, POCl<sub>3</sub>, reflux, 46–55% over 2 steps; (d) 1-methylpiperidin-4-amine, DIEA, <i>i</i>-PrOH, microwave, 160 °C, 80–90%.</p></p></figure><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. SAR of the 2-Morpholine Moiety<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/medium/jm-2016-01645c_0014.gif" alt="" id="fx5" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/medium/jm-2016-01645c_0015.gif" alt="" id="fx6" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup>Table a</sup><p class="last">IC<sub>50</sub> determination experiments were performed at substrate and cofactor concentrations equal to the respective <i>K</i><sub>m</sub> values for each enzyme. IC<sub>50</sub> determination experiments were performed in triplicate and the values are presented as Mean ± SD.</p></div></div><div></div></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> SAR in GLP and G9a Biochemical Assays</h3><div class="NLM_p">All of the synthesized compounds were evaluated in the GLP and G9a radioactivity based scintillation proximity biochemical assays. IC<sub>50</sub> values of the compounds are summarized in <a class="ref internalNav" href="#tbl1" aria-label="Tables 1">Tables 1</a>–<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>.</div><div class="NLM_p">For the <i>N</i>-capping group of the piperidine moiety, we found that a variety of alkyl groups were well tolerated for both GLP and G9 (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). Replacing the methyl group (<b>4</b>) with the ethyl (<b>7</b>), <i>n</i>-propyl (<b>8</b>), <i>i</i>-propyl (<b>9</b>), or cyclopropyl (<b>10</b>) group did not significantly change the potency for either GLP or G9a. These SAR results are consistent with the crystal structures of GLP and G9a in complex with <b>4</b>, which show the <i>N</i>-capping group is solvent exposed. Because the methyl group in <b>4</b> provides slightly better selectivity and is the smallest substituent, we chose the methyl group as the <i>N</i>-capping group for the following SAR studies.</div><div class="NLM_p">In contrast to the <i>N</i>-capping groups, the 6-methoxy moiety was much more sensitive to structural changes (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>). Displacing the methoxy group with the ethoxy (<b>41</b>) or isopropoxy (<b>42</b>) group totally diminished their activities against both GLP and G9a (IC<sub>50</sub> > 5000 nM). From the cocrystal structures of G9a and GLP in complex with <b>4</b>, the 6-methoxy group occupies a shallow binding groove. A larger group such as ethoxy or isopropoxy likely clashes with the binding groove, thus decreasing the potency. The removal of the oxygen atom from the methoxy group and installation of a smaller group also led to less potent analogues. In addition, GLP was more sensitive to these structural changes than G9a. For example, 6-fluoro group (<b>43</b>) decreased the potency for GLP (IC<sub>50</sub> = 1230 ± 340 nM) by 95-fold and for G9a (IC<sub>50</sub> = 3690 ± 1780 nM) by 8-fold. Similarly, 6-methyl group (<b>44</b>) led to a 62-fold potency drop for GLP (IC<sub>50</sub> = 808 ± 378 nM) and a 9-fold potency decrease for G9a (IC<sub>50</sub> = 4070 ± 950 nM). These SAR results confirmed the importance of the interactions between the water molecule and 6-methoxy group in the cocrystal structures.</div><div class="NLM_p">The SAR of the C7 substituents followed the similar trend as the 6-substituents (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>). The ethoxy (<b>45</b>) or isopropoxy (<b>46</b>) group abolished the activities against both GLP and G9a. From the cocrystal structures, the 7-methoxy group is located at the neck of the lysine binding channel and also interacts with the water molecule. In our previous report, the pyrrolidin-1-yl-propoxy group at this position greatly increased G9a potency by forming an addition hydrogen bond with Leu1086 and cation−π interaction with Tyr1154.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> The drastic potency loss of the ethoxy and isopropoxy analogues highlights the importance of the interactions between the basic amino group and Leu1086 and Tyr1154. We also found that, while the fluoro group (<b>47</b>) completely abolished the activities against both GLP and G9a, the methyl group (<b>48</b>) was tolerated with only about 2–3-fold potency loss for GLP (IC<sub>50</sub> = 29 ± 12 nM) and G9a (IC<sub>50</sub> = 1150 ± 47 nM).</div><div class="NLM_p last">Having established initial SAR for the <i>N</i>-capping group, and 6- and 7-methoxy regions, we next explored the 2-morpholine moiety using cyclic amines (<b>11</b>–<b>15</b>), cyclic alkyl groups (<b>53</b> and <b>54</b>), and acyclic amines (<b>16</b>–<b>22</b>). We first compared the pyrrolidinyl (<b>11</b>), piperidinyl (<b>12</b>), and azepanyl (<b>14</b>) analogues and found the effect of the ring size on the potency for GLP was minimal with IC<sub>50</sub> values of 12 ± 1, 12 ± 1, and 16 ± 3 nM, respectively. However, the ring size was more important for modulating G9a potency. Among the three groups, the five-membered pyrrolidinyl group (<b>11</b>) led to the weakest G9a inhibition (IC<sub>50</sub> = 313 ± 133 nM) and the six-membered piperidinyl group (<b>12</b>) resulted in the most potent G9a inhibitor (IC<sub>50</sub> = 31 ± 9 nM). Substitutions on the cyclic amino groups also exhibited a drastic impact on the potency for both GLP and G9a. Compared with the piperidinyl analogue <b>12</b>, the 4,4-difluoro substituted piperidinyl group (<b>13</b>) decreased potency for GLP by 8-fold and for G9a by 34-fold. Compared with the azepanyl analogue <b>14</b>, the methylhomopiperazinyl analogue <b>15</b> increased potency for GLP by 4-fold and G9a by 21-fold. Replacing the 2-cycloamino group with a 2-cycloalkyl group led to much less potent compounds for both GLP and G9a. For example, compound <b>53</b> was 3-fold less potent for GLP and 14-fold less potent for G9a compared with <b>12</b>. In addition, compound <b>54</b> was 26-fold less potent for GLP and 4-fold less potent for G9a compared with <b>4</b>. These SAR results indicate the important role of the 2-amino group for maintaining high potency for both GLP and G9a. We next explored a number of acyclic alkyl amines (<b>16</b>–<b>22</b>) at the 2-position. For the secondary amine analogues <b>16</b>–<b>19</b>, the length of the alkyl chain had the most profound effect on the potency for GLP over G9a. We were delighted to find that compound <b>18</b>, which contains a 2-hexylamino group, was >140-fold selective for GLP over G9a, representing the most selective GLP inhibitor to date. The tertiary amine analogues <b>20</b>–<b>22</b>, which possess an extra <i>N</i>-methyl substitute, did not improve potency or selectivity for GLP over G9a.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Characterization of <b>4</b> and <b>18</b> in Selectivity, Biophysical, and Mechanism of Action Studies (MOA)</h3><div class="NLM_p">To further assess the selectivity of <b>4</b> and <b>18</b>, we tested them against 29 protein lysine methyltransferases (PKMTs), protein arginine methyltransferases (PRMTs), DNA methyltransferases (DNMTs), and RNA methyltransferases (RNMTs). We found that <b>4</b> did not inhibit any of these methyltransferases at 10 μM (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>A). While <b>18</b> did not inhibit any of these methyltransferase at 1 μM, it displayed modest inhibition (57%) of the <i>N</i>6-adenosine-methyltransferase subunit METTL3 and 14 complex (METTL3–14) at 10 μM (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>B).</div><figure id="fig3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/medium/jm-2016-01645c_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/large/jm-2016-01645c_0003.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Selectivity of <b>4</b> and <b>18</b> against methyltransferases. Selectivity of <b>4</b> (A) and <b>18</b> (B) against a panel of 29 PKMTs, PRMTs, DNMTs, and RNMTs was determined at two compound concentrations of 1 μM (blue bars) and 10 μM (red bars).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/large/jm-2016-01645c_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01645&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We next confirmed bindings of <b>4</b> and <b>18</b> to GLP and G9a using isothermal titration calorimetry (ITC) in the presence of SAM (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>). The binding of <b>4</b> and <b>18</b> to GLP and G9a is enthalpy driven. Compound <b>4</b> displayed higher binding affinity to GLP (<i>K</i><sub>d</sub> = 40 ± 5 nM) than G9a (<i>K</i><sub>d</sub> = 445 ± 40 nM) (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>A,B). Similarly, compound <b>18</b> also exhibited stronger binding affinity to GLP (<i>K</i><sub>d</sub> = 46 ± 15 nM) than G9a (<i>K</i><sub>d</sub> = 610 ± 68 nM) (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>C,D). Thus, in ITC biophysical assays, <b>4</b> and <b>18</b> were 11-fold and 13-fold selective for GLP over G9a, respectively.</div><figure id="fig4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/medium/jm-2016-01645c_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/large/jm-2016-01645c_0004.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Binding confirmation of <b>4</b> and <b>18</b>. Isothermal titration calorimetry (ITC) was used to confirm binding of <b>4</b> to (A) GLP with a <i>K</i><sub>d</sub> of 40 ± 5 nM and to (B) G9a with a <i>K</i><sub>d</sub> of 445 ± 40 nM. Similarly, ITC was used to confirm the binding of <b>18</b> to (C) GLP with a <i>K</i><sub>d</sub> of 46 ± 15 nM and to (D) G9a with a <i>K</i><sub>d</sub> of 610 ± 68 nM. ITC experiments for GLP and G9a were performed in triplicate.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/large/jm-2016-01645c_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01645&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To assess MOA of <b>4</b> and <b>18</b>, we evaluated the effect of peptide and SAM concentrations on IC<sub>50</sub> values of <b>4</b> and <b>18</b> against GLP. As illustrated in parts A and C of <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>, increasing the peptide substrate concentration significantly increased the IC<sub>50</sub> values of <b>4</b> and <b>18</b>, indicating that both compounds are peptide competitive. As illustrated in parts B and D of <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>, increasing the SAM concentration decreased the IC<sub>50</sub> values of <b>4</b> and <b>18</b>, indicating that both compounds are SAM uncompetitive.</div><figure id="fig5" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/medium/jm-2016-01645c_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/large/jm-2016-01645c_0005.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Mechanism of GLP inhibition by <b>4</b> and <b>18</b>. Mechanism of inhibition of GLP activity by <b>4</b> and <b>18</b> was explored by IC<sub>50</sub> determination at various peptide (A and C, respectively) or SAM (B and D, respectively) concentrations while the concentration of the other substrate was kept constant. Increase in peptide concentration in assays resulted in significant increase in IC<sub>50</sub> values, indicating that both compounds are peptide competitive (A,C). Decrease in IC<sub>50</sub> values upon increasing SAM concentration indicates that both compounds are SAM uncompetitive inhibitors (B,D).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/large/jm-2016-01645c_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01645&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Cocrystal Structures of <b>18</b></h3><div class="NLM_p">We solved the ternary crystal structures of GLP and G9a in complex with <b>18</b> in the presence of the cofactor SAM (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>A–C, and Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.6b01645/suppl_file/jm6b01645_si_001.pdf" class="ext-link">Table 2</a>). Similar to <b>4</b>, compound <b>18</b> conserved three key hydrogen bonds with GLP and G9a: (1) the secondary amino group at the 4-position interacts with GLP Asp1171 and G9a Asp1083; (2) the N1 nitrogen on the quinazoline ring interacts with GLP Asp1176 and G9a Asp1088; (3) the oxygen atoms of both 6- and 7-methoxy groups interact weakly with a water molecule in both GLP and G9a structures. Interestingly, the 2-hexylamino group of <b>18</b> points to a different direction compared to the 2-(3′-dimethylamino)propylamino group of compound <b>2</b> in the cocrystal structure of GLP–SAH–<b>2</b> (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>D).<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> The salt bridge<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> formed between the 3′-dimethylamino group of <b>2</b> and Asp1131 likely leads to the 2-(3′-dimethylamino)propylamino side chain adapting to the different geometry. Notably, the crystal structures of GLP and G9a in the complex with <b>4</b> or <b>18</b> revealed the virtually identical inhibitor binding modes and ligand–protein interactions and did not provide insight for the structural basis of the high selectivity for GLP over G9a. We are currently conducting additional studies to understand the structural basis of the observed high selectivity and will report our progress in future publications.</div><figure id="fig6" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/medium/jm-2016-01645c_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/large/jm-2016-01645c_0006.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. X-ray cocrystal structures of GLP and G9a in complex with <b>18</b>. (A) Structure of GLP–SAM–<b>18</b> (green) (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TTG">5TTG</a>). (B) Structure of G9a–SAM–<b>18</b> (blue) (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TTF">5TTF</a>). Water molecule is illustrated as a red sphere. Main interactions are shown in yellow dashed lines. (C) Overlay of (A) and (B). (D) Overlay of (A) and GLP–SAH–<b>2</b> (purple) (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3MO5">3MO5</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/large/jm-2016-01645c_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01645&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Lastly, we assessed membrane permeability of compound <b>18</b> in a Caco-2 cell permeability assay and found that <b>18</b> had poor membrane permeability (5.0 nm/s) and high efflux ratio (80). Thus, <b>18</b> is not suitable for cellular studies. Future work will attempt to improve membrane permeability and reduce efflux while maintaining the high selectivity for GLP over G9a.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_61370" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_61370" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">By testing our quinazoline compound collection, we identified compound <b>4</b> as a GLP selective inhibitor. We next conducted SAR studies and explored four regions of <b>4</b>, which led to the discovery of compound <b>18</b>, which is >140-fold selective for GLP over G9a. We confirmed the bindings of <b>4</b> and <b>18</b> to GLP and G9a by ITC. Both <b>4</b> and <b>18</b> are also selective for GLP over a broad range of other PKMTs, PRMTs, DNMTs, and RNMTs. We also obtained high-resolution X-ray cocrystal structures of GLP and G9a in complex with either <b>4</b> or <b>18</b>, which further confirmed the bindings of these inhibitors to GLP and G9a and revealed that the inhibitors bind to GLP and G9a in virtually identical binding modes. This work will help develop high-quality chemical probes of GLP.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_07812" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_07812" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Chemistry General Procedures</h3><div class="NLM_p">HPLC spectra for all compounds were acquired using an Agilent 1200 series system with DAD detector. Chromatography was performed on a 2.1 mm × 150 mm Zorbax 300SB-C<sub>18</sub> 5 μm column with water containing 0.1% formic acid as solvent A and acetonitrile containing 0.1% formic acid as solvent B at a flow rate of 0.4 mL/min. The gradient program was as follows: 1% B (0–1 min), 1–99% B (1–4 min), and 99% B (4–8 min). High-resolution mass spectra (HRMS) data were acquired in positive ion mode using an Agilent G1969A API-TOF with an electrospray ionization (ESI) source. Nuclear magnetic resonance (NMR) spectra were acquired on a Bruker DRX-600 spectrometer (600 MHz <sup>1</sup>H, 150 MHz <sup>13</sup>C) or a Varian Mercury spectrometer (400 MHz <sup>1</sup>H, 100 MHz <sup>13</sup>C). Chemical shifts are reported in ppm (δ). Preparative HPLC was performed on Agilent Prep 1200 series with UV detector set to 254 nm. Samples were injected into a Phenomenex Luna 750 mm × 30 mm, 5 μm, C<sub>18</sub> column at room temperature. The flow rate was 40 mL/min. A linear gradient was used with 10% (or 50%) of MeOH (A) in H<sub>2</sub>O (with 0.1% TFA) (B) to 100% of MeOH (A). HPLC was used to establish the purity of target compounds. All final compounds had >95% purity using the HPLC methods described above.</div><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17"> 6,7-Dimethoxy-<i>N</i>-(1-methylpiperidin-4-yl)-2-morpholinoquinazolin-4-amine (<b>4</b>)</h4><div class="NLM_p last">To a solution of 2,4-dichloro-6,7-dimethoxyquinazoline (50 mg, 0.19 mmol) in DMF (2 mL) were added K<sub>2</sub>CO<sub>3</sub> (55 mg, 0.40 mmol) and 4-methylpiperidin-1-amine (0.05 mL, 0.40 mmol). After being stirred for 2 h, the reaction was quenched with water and extracted with DCM (3 × 5 mL). The combined organic layers were dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The residue was dissolved in <i>i</i>-PrOH (1 mL) and treated with HCl (0.1 mL, 4 M in dioxane, 0.4 mmol) and morpholine (0.03 mL, 0.34 mmol). The solution was heated in a microwave reactor at 160 °C for 15 min before being concentrated under reduced pressure. The resulting residue was purified by preparative HPLC to yield the title compound (60 mg, 82% over 2 steps). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>) δ 6.92 (s, 1H), 6.79 (s, 1H), 5.21 (d, <i>J</i> = 7.2 Hz, 1H), 4.17–4.12 (m, 1H), 3.96 (s, 3H), 3.95 (s, 3H), 3.83–3.79 (m, 8H), 2.90 (d, <i>J</i> = 10.8 Hz, 2H), 2.33 (s, 3H), 2.21–2.15 (m, 4H), 1.71–1.61 (m, 2H). <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>) δ 158.95, 158.32, 154.45, 149.05, 145.68, 106.11, 103.39, 100.69, 67.10, 56.32, 55.98, 54.80, 47.75, 46.20, 44.67, 32.13. HRMS(ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>30</sub>N<sub>5</sub>O<sub>3</sub> 388.2343, found 388.2382.</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> <i>N</i>-(1-Ethylpiperidin-4-yl)-6,7-dimethoxy-2-morpholinoquinazolin-4-amine (<b>7</b>)</h4><div class="NLM_p last">The title compound (75% over 2 steps) was prepared according to synthetic procedure for <b>4</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.87 (s, 1H), 6.73 (s, 1H), 5.13 (d, <i>J</i> = 7.6 Hz, 1H), 4.08–4.05 (m, 1H), 3.90 (s, 3H), 3.88 (s, 3H), 3.78–3.73 (m, 8H), 2.93 (d, <i>J</i> = 12.0 Hz, 2H), 2.41 (q, <i>J</i> = 6.8 Hz, 2H), 2.14–2.08 (m, 4H), 1.63–1.54 (m, 2H), 1.07 (q, <i>J</i> = 7.2 Hz, 3H). MS (ESI) <i>m</i>/<i>z</i> 402.3 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> 6,7-Dimethoxy-2-morpholino-<i>N</i>-(1-propylpiperidin-4-yl)quinazolin-4-amine (<b>8</b>)</h4><div class="NLM_p last">The title compound (76% over 2 steps) was prepared according to synthetic procedure for <b>4</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.89 (s, 1H), 6.74 (s, 1H), 5.12 (d, <i>J</i> = 7.2 Hz, 1H), 4.13–4.10 (m, 1H), 3.93 (s, 3H), 3.91 (s, 3H), 3.81–3.74 (m, 8H), 2.92 (d, <i>J</i> = 11.6 Hz, 2H), 2.30 (q, <i>J</i> = 6.8 Hz, 2H), 2.15–2.11 (m, 4H), 1.61–1.47 (m, 4H), 0.90 (q, <i>J</i> = 7.2 Hz, 3H). MS (ESI) <i>m</i>/<i>z</i> 416.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> <i>N</i>-(1-Isopropylpiperidin-4-yl)-6,7-dimethoxy-2-morpholinoquinazolin-4-amine (<b>9</b>)</h4><div class="NLM_p last">The title compound (85% over 2 steps) was prepared according to synthetic procedure for <b>4</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.88 (s, 1H), 6.78 (s, 1H), 5.24 (d, <i>J</i> = 7.2 Hz, 1H), 4.12–4.03 (m, 1H), 3.91 (s, 3H), 3.89 (s, 3H), 3.81–3.74 (m, 8H), 2.91 (d, <i>J</i> = 12.0 Hz, 2H), 2.81–2.75 (m, 1H), 2.33 (t, <i>J</i> = 10.8 Hz, 2H), 2.16–2.14 (m, 2H), 1.67–1.58 (m, 2H), 1.06 (d, <i>J</i> = 6.8 Hz, 6H). MS (ESI) <i>m</i>/<i>z</i> 416.3 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> <i>N</i>-(1-Cyclopropylpiperidin-4-yl)-6,7-dimethoxy-2-morpholinoquinazolin-4-amine (<b>10</b>)</h4><div class="NLM_p last">The title compound (77% over 2 steps) was prepared according to synthetic procedure for <b>4</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.91 (s, 1H), 6.70 (s, 1H), 5.01 (d, <i>J</i> = 6.8 Hz, 1H), 4.18–4.08 (m, 1H), 3.95 (s, 3H), 3.92 (s, 3H), 3.83–3.77 (m, 8H), 3.04 (d, <i>J</i> = 12.4 Hz, 2H), 2.39 (td, <i>J</i> = 11.6, 3.6 Hz, 2H), 2.15–2.11 (m, 2H), 1.66–1.60 (m, 1H), 1.54 (qd, <i>J</i> = 11.2, 4.0 Hz, 2H), 0.49–0.45 (m, 2H), 0.44–0.40 (m, 2H). MS (ESI) <i>m</i>/<i>z</i> 414.3 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> 6,7-Dimethoxy-<i>N</i>-(1-methylpiperidin-4-yl)-2-(pyrrolidin-1-yl)quinazolin-4-amine (<b>11</b>)</h4><div class="NLM_p last">The title compound (80% over 2 steps) was prepared according to synthetic procedure for <b>4</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.92 (s, 1H), 6.78 (s, 1H), 5.12 (d, <i>J</i> = 6.8 Hz, 1H), 4.19–4.09 (m, 1H), 3.91 (s, 3H), 3.89 (s, 3H), 3.64–3.60 (m, 4H), 2.85 (d, <i>J</i> = 12.0 Hz, 2H), 2.30 (s, 3H), 2.18–2.13 (m, 4H), 1.96–1.93 (m, 4H), 1.64–1.55 (m, 2H). MS (ESI) <i>m</i>/<i>z</i> 372.3 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> 6,7-Dimethoxy-<i>N</i>-(1-methylpiperidin-4-yl)-2-(piperidin-1-yl)quinazolin-4-amine (<b>12</b>)</h4><div class="NLM_p last">The title compound (80% over 2 steps) was prepared according to synthetic procedure for <b>4</b>. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.64 (s, 1H), 7.11 (s, 1H), 4.58–4.46 (m, 1H), 3.94 (s, 3H), 3.90 (s, 3H), 3.87–3.81 (m, 4H), 3.63 (d, <i>J</i> = 12.8 Hz, 2H), 3.26–3.19 (m, 1H), 2.89 (s, 3H), 2.34 (d, <i>J</i> = 12.9 Hz, 2H), 2.10–1.96 (m, 2H), 1.82–1.66 (m, 7H). MS (ESI) <i>m</i>/<i>z</i> 386.3 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> 2-(4,4-Difluoropiperidin-1-yl)-6,7-dimethoxy-<i>N</i>-(1-methylpiperidin-4-yl)quinazolin-4-amine (<b>13</b>)</h4><div class="NLM_p last">The title compound (84% over 2 steps) was prepared according to synthetic procedure for <b>4</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.89 (s, 1H), 6.71 (s, 1H), 5.01 (d, <i>J</i> = 7.2 Hz, 1H), 4.15–4.05 (m, 1H), 3.99 (t, <i>J</i> = 6.0 Hz, 4H), 3.96 (s, 3H), 3.95 (s, 3H), 2.89–2.86 (m, 2H), 2.33 (s, 3H), 2.22–2.13 (m, 4H), 2.06–1.96 (m, 4H), 1.69–1.59 (m, 2H). MS (ESI) <i>m</i>/<i>z</i> 422.3 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> 2-(Azepan-1-yl)-6,7-dimethoxy-<i>N</i>-(1-methylpiperidin-4-yl)quinazolin-4-amine (<b>14</b>)</h4><div class="NLM_p last">The title compound (79% over 2 steps) was prepared according to synthetic procedure for <b>4</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.88 (s, 1H), 6.72 (s, 1H), 4.97 (d, <i>J</i> = 6.4 Hz, 1H), 4.12–4.05 (m, 1H), 3.93 (s, 3H), 3.91 (s, 3H), 3.78 (t, <i>J</i> = 6.0 Hz, 4H), 2.88–2.85 (m, 2H), 2.33 (s, 3H), 2.17–2.13 (m, 4H), 1.82–1.74 (m, 4H), 1.64–1.53 (m, 6H). MS (ESI) <i>m</i>/<i>z</i> 400.3 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> 6,7-Dimethoxy-2-(4-methyl-1,4-diazepan-1-yl)-<i>N</i>-(1-methylpiperidin-4-yl)quinazolin-4-amine (<b>15</b>)</h4><div class="NLM_p last">The title compound (78% over 2 steps) was prepared according to synthetic procedure for <b>4</b>. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.38 (s, 1H), 6.89 (s, 1H), 4.15–4.07 (m, 1H), 3.93–3.89 (m, 2H), 3.88 (s, 3H), 3.87 (s, 3H), 3.82 (t, <i>J</i> = 6.4 Hz, 2H), 2.94 (d, <i>J</i> = 12.1 Hz, 2H), 2.76–2.70 (m, 2H), 2.60–2.57 (m, 2H), 2.34 (s, 3H), 2.30 (s, 3H), 2.17 (td, <i>J</i> = 12.1, 2.2 Hz, 2H), 2.12–2.04 (m, 2H), 2.04–1.96 (m, 2H). MS (ESI) <i>m</i>/<i>z</i> 415.3 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> <i>N</i><sup>2</sup>-Hexyl-6,7-dimethoxy-<i>N</i><sup>4</sup>-(1-methylpiperidin-4-yl)quinazoline-2,4-diamine (<b>16</b>)</h4><div class="NLM_p last">The title compound (85% over 2 steps) was prepared according to synthetic procedure for <b>4</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.84 (s, 1H), 6.82 (s, 1H), 5.32 (d, <i>J</i> = 6.8 Hz, 1H), 4.79 (br, 1H), 4.13–4.11 (m, 1H), 3.87 (s, 3H), 3.85 (s, 3H), 3.45- 3.40 (m, 2H), 2.83 (d, <i>J</i> = 11.6 Hz, 2H), 2.28 (s, 3H), 2.15- 2.10 (m, 4H), 1.62–1.57 (m, 4H), 1.45–1.37 (m, 2H). 0.93 (t, <i>J</i> = 7.2 Hz, 3H). MS (ESI) <i>m</i>/<i>z</i> 374.3 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> 6,7-Dimethoxy-<i>N</i><sup>4</sup>-(1-methylpiperidin-4-yl)-<i>N</i><sup>2</sup>-pentylquinazoline-2,4-diamine (<b>17</b>)</h4><div class="NLM_p last">The title compound (76% over 2 steps) was prepared according to synthetic procedure for <b>4</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.83 (s, 1H), 6.82 (s, 1H), 5.40 (d, <i>J</i> = 5.2 Hz, 1H), 4.96 (br, 1H), 4.16–4.07 (m, 1H), 3.87 (s, 3H), 3.84 (s, 3H), 3.39 (q, <i>J</i> = 6.8 Hz, 2H), 2.84–2.81 (m, 2H), 2.26 (s, 3H), 2.14–2.08 (m, 4H), 1.63–1.54 (m, 4H), 1.34–1.31 (m, 4H), 0.86 (t, <i>J</i> = 7.2 Hz, 3H). MS (ESI) <i>m</i>/<i>z</i> 388.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> <i>N</i><sup>2</sup>-Hexyl-6,7-dimethoxy-<i>N</i><sup>4</sup>-(1-methylpiperidin-4-yl)quinazoline-2,4-diamine (<b>18</b>)</h4><div class="NLM_p last">The title compound (81% over 2 steps) was prepared according to synthetic procedure for <b>4</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.86 (s, 1H), 6.76 (s, 1H), 5.14 (d, <i>J</i> = 7.2 Hz, 1H), 4.77 (br, 1H), 4.15–4.12 (m, 1H), 3.92 (s, 3H), 3.90 (s, 3H), 3.46–3.40 (q, <i>J</i> = 6.8 Hz, 2H), 2.50 (d, <i>J</i> = 12.0 Hz, 2H), 2.30 (s, 3H), 2.18–2.12 (m, 4H), 1.60–1.56 (m, 4H), 1.32–1.28 (m, 6H), 0.88 (t, <i>J</i> = 7.2 Hz, 3H). MS (ESI): 402 [M + H]<sup>+</sup>. <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>) δ 158.68, 158.39, 154.77, 145.62, 104.35, 103.15, 101.36 (2C), 56.48, 56.12, 54.81, 47.97, 46.27, 41.58, 32.19, 31.68, 30.01, 26.79, 22.66, 14.10. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>22</sub>H<sub>36</sub>N<sub>5</sub>O<sub>2</sub> 402.2864, found 402.2896.</div></div><div id="sec4_1_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> <i>N</i><sup>2</sup>-Heptyl-6,7-dimethoxy-<i>N</i><sup>4</sup>-(1-methylpiperidin-4-yl)quinazoline-2,4-diamine (<b>19</b>)</h4><div class="NLM_p last">The title compound (75% over 2 steps) was prepared according to synthetic procedure for <b>4</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.84 (s, 1H), 6.82 (s, 1H), 5.46 (d, <i>J</i> = 6.7 Hz, 1H), 4.95–4.80 (m, 1H), 4.21- 4.03 (m, 1H), 3.85 (s, 3H), 3.82 (s, 3H), 3.39 (dd, <i>J</i> = 12.9, 6.9 Hz, 2H), 2.81 (d, <i>J</i> = 11.9 Hz, 2H), 2.25 (s, 3H), 2.17–1.98 (m, 4H), 1.67–1.48 (m, 4H), 1.43–1.14 (m, 8H), 0.83 (t, <i>J</i> = 6.9 Hz, 3H). MS (ESI) <i>m</i>/<i>z</i> 416.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> <i>N</i><sup>2</sup>-Butyl-6,7-dimethoxy-<i>N</i><sup>2</sup>-methyl-<i>N</i><sup>4</sup>-(1-methylpiperidin-4-yl)quinazoline-2,4-diamine (<b>20</b>)</h4><div class="NLM_p last">The title compound (83% over 2 steps) was prepared according to synthetic procedure for <b>4</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.90 (s, 1H), 6.74 (s, 1H), 5.02 (d, <i>J</i> = 7.0 Hz, 1H), 4.10 (dt, <i>J</i> = 11.0, 5.3 Hz, 1H), 3.93 (s, 3H), 3.91 (s, 3H), 3.79–3.55 (t, <i>J</i> = 7.2 Hz, 2H), 3.18 (s, 3H), 2.86 (t, <i>J</i> = 10.2 Hz, 2H), 2.31 (s, 3H), 2.16–2.11 (m, 4H), 1.72–1.48 (m, 4H), 1.44–1.27 (m, 2H), 0.94 (t, <i>J</i> = 7.4 Hz, 3H). MS (ESI) <i>m</i>/<i>z</i> 388.3 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> 6,7-Dimethoxy-<i>N</i><sup>2</sup>-methyl-<i>N</i><sup>4</sup>-(1-methylpiperidin-4-yl)-<i>N</i><sup>2</sup>-pentylquinazoline-2,4-diamine (<b>21</b>)</h4><div class="NLM_p last">The title compound (79% over 2 steps) was prepared according to synthetic procedure for <b>4</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.88 (s, 1H), 6.77 (s, 1H), 5.10 (d, <i>J</i> = 6.8 Hz, 1H), 4.13–4.03 (m, 1H), 3.90 (s, 3H), 3.87 (s, 3H), 3.62 (t, <i>J</i> = 7.2 Hz, 2H), 3.17 (s, 3H), 2.85 (d, <i>J</i> = 12.0 Hz, 2H), 2.29 (s, 3H), 2.15–2.09 (m, 4H), 1.64–1.54 (m, 4H), 1.37–1.23 (m, 4H), 0.88 (t, <i>J</i> = 7.2 Hz, 3H). MS (ESI) <i>m</i>/<i>z</i> 402.3 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> <i>N</i><sup>2</sup>-Hexyl-6,7-dimethoxy-<i>N</i><sup>2</sup>-methyl-<i>N</i><sup>4</sup>-(1-methylpiperidin-4-yl)quinazoline-2,4-diamine (<b>22</b>)</h4><div class="NLM_p last">The title compound (82% over 2 steps) was prepared according to synthetic procedure for <b>4</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.88 (s, 1H), 6.78 (s, 1H), 5.13 (d, <i>J</i> = 6.8 Hz, 1H), 4.13–4.03 (m, 1H), 3.90 (s, 3H), 3.87 (s, 3H), 3.62 (t, <i>J</i> = 7.2 Hz, 2H), 3.17 (s, 3H), 2.85 (d, <i>J</i> = 12.0 Hz, 2H), 2.29 (s, 3H), 2.15–2.09 (m, 4H), 1.64–1.54 (m, 4H), 1.37–1.23 (m, 6H), 0.88 (t, <i>J</i> = 7.2 Hz, 3H). MS (ESI) <i>m</i>/<i>z</i> 416.3 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> 2-Amino-5-ethoxy-4-methoxybenzonitrile (<b>29a</b>)</h4><div class="NLM_p last">The title compound (50% over 3 steps) was prepared according to synthetic procedure for <b>30a</b>. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>) δ 6.78 (s, 1H), 6.23 (s, 1H), 4.30–4.25 (br, 2H), 3.96 (q, <i>J</i> = 7.1 Hz, 2H), 3.81 (s, 3H), 1.40 (t, <i>J</i> = 7.0 Hz, 3H).</div></div><div id="sec4_1_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> 2-Amino-5-isopropoxy-4-methoxybenzonitrile (<b>29b</b>)</h4><div class="NLM_p last">The title compound (56% over 3 steps) was prepared according to synthetic procedure for <b>30a</b>. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>) δ 6.88 (s, 1H), 6.88 (s, 1H), 6.24 (s, 1H), 4.33–4.29 (m, 1H), 4.26–4.22 (br, 2H), 3.83 (s, 3H), 1.32 (d, <i>J</i> = 6.1 Hz, 6H).</div></div><div id="sec4_1_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> 2-Amino-4-ethoxy-5-methoxybenzonitrile (<b>30a</b>)</h4><div class="NLM_p last">To a solution of 4-hydroxy-3-methoxybenzonitrile (1.49 g, 10 mmol) in acetone (50 mL) were added ethyl bromide (1.4 mL, 19 mmol), followed by K<sub>2</sub>CO<sub>3</sub> (2.77 g, 20 mmol). The mixture was refluxed overnight. After removal of the solvent, the residue was suspended in water (50 mL) and extracted with ethyl acetate (3 × 50 mL). The combined organic phase was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure to provide intermediate <b>26a</b>. To a solution of intermediate <b>26a</b> in Ac<sub>2</sub>O (50 mL) was added HNO<sub>3</sub> (3.6 mL, 39 mmol) slowly at 0 °C. The solution was stirred at rt overnight, during which time solid was precipitated out. The reaction mixture was poured into ice water and filtered. The solid was collected and dried to yield the intermediate <b>28a</b> (2.09 g, 94%) as yellow solid. To a suspension of <b>28a</b> (2.09 g, 9.4 mmol) in a mixture of water (50 mL) and ethyl acetate (50 mL) were added iron powder (2.10 g, 37.6 mmol) and NH<sub>4</sub>OAc (4.43 g, 57.4 mmol). The resulting mixture was heated under reflux for 2 h. After cooling to rt, the mixture was filtered and the filtrate was extracted with ethyl acetate. The organic layer was dried with anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure to yield the title compound (1.06 g, 60%) as brown solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.84 (s, 1H), 6.17 (s, 1H), 5.80–5.50 (br, 2H), 4.06 (q, <i>J</i> = 7.0 Hz, 2H), 3.80 (s, 3H), 1.46 (t, <i>J</i> = 7.0 Hz, 3H).</div></div><div id="sec4_1_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> 2-Amino-4-isopropoxy-5-methoxybenzonitrile (<b>30b</b>)</h4><div class="NLM_p last">The title compound (55% over 3 steps) was prepared according to synthetic procedure for <b>30a</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.86 (s, 1H), 6.19 (s, 1H), 5.72–5.55 (br, 2H), 4.58–4.52 (m, 1H), 3.79 (s, 3H), 1.39 (d, <i>J</i> = 6.1 Hz, 6H).</div></div><div id="sec4_1_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> 2-Bromo-5-methoxy-4-methylaniline (<b>33a</b>)</h4><div class="NLM_p last">To a solution of 3-methoxy-4-methylaniline (420 mg, 3.1 mmol) in ethyl acetate (15 mL) was added Bu<sub>3</sub>NBr<sub>3</sub> (1.56 g, 3.2 mmol) at 0 °C. After being stirred at rt overnight, the reaction was quenched with saturated aqueous NaHCO<sub>3</sub> solution (10 mL). The organic layer was separated and dried with anhydrous Na<sub>2</sub>SO<sub>4</sub>. After removal of the solvent, the resulting residue was purified by silica gel chromatograph to yield the desire product (585 mg, 90%). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>) δ 7.15 (s, 1H), 6.31 (s, 1H), 4.00–3.90 (br, 2H), 3.79 (s, 3H), 2.11 (s, 3H).</div></div><div id="sec4_1_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> 2-Bromo-4-methoxy-5-methylaniline (<b>34a</b>)</h4><div class="NLM_p last">To a solution of 3-methyl-4-methoxyaniline (600 mg, 4.3 mmol) in acetic acid (10 mL) was added acetic anhydride (0.42 mL, 4.4 mmol) dropwise at rt. The reaction was heated at 50 °C, at which temperature bromine (0.23 mL, 4.4 mmol) was added. After being stirred for 2 h at 50 °C, the reaction was cooled to rt and poured into ice–water. The resulting precipitate was filtered, washed with water, and dissolved in ethyl acetate. The solution was washed with saturated sodium chloride aqueous solution, dried with sodium sulfate, and evaporated to dryness to give 2-bromo-4-methoxy-5-methylacetanilide as a crude product. This intermediate was dissolved in ethanol (10 mL) and treated with concentrated hydrochloric acid (2.6 mL). The mixture was refluxed for 2 h before being concentrated. The residue was treated with aqueous sodium hydroxide (1.5 M, 10 mL) to precipitate the crude product. After filtration, the solid was washed with water and dried under reduced pressure to give the title compound (743 mg, 80% over 2 steps). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>) δ 6.90 (s, 1H), 6.64 (s, 1H), 3.77 (s, 3H), 3.75–3.70 (br, 2H), 2.14 (s, 3H).</div></div><div id="sec4_1_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> 2-Bromo-5-fluoro-4-methoxyaniline (<b>34b</b>)</h4><div class="NLM_p last">To a suspension of 3-fluoro-4-methoxyaniline (5.06 g, 35.9 mmol) in DCM (160 mL) was added a solution of Br<sub>2</sub> (1.85 mL, 36.0 mmol) in DCM (20 mL) dropwise at −15 °C. After being stirred for 30 min, the reaction was quenched with water (100 mL). The organic layer was dried with anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The residue was purified by silica gel chromatograph to yield the desired compound (5.50 g, 70%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.02 (d, <i>J</i> = 8.5 Hz, 1H), 6.55 (d, <i>J</i> = 12.3 Hz, 1H), 3.88–3.80 (br, 2H), 3.79 (s, 3H).</div></div><div id="sec4_1_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> 2-Amino-4-methoxy-5-methylbenzonitrile (<b>35a</b>)</h4><div class="NLM_p last">To a solution of <b>33a</b> (97 mg, 0.46 mmol) in DMF (2 mL) was added Zn(CN)<sub>2</sub> (111 mg, 0.94 mmol) and Pd(PPh<sub>3</sub>)<sub>4</sub> (54 mg, 0.04 mmol). The suspension was heated in a microwave reactor at 0 °C for 15 min. After cooling to rt, the reaction was diluted with DCM (4 mL) and ammonium hydroxide solution (2 mL). The resulting mixture was extracted with DCM (3 × 4 mL). The combined organic layer was dried with anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The resulting residue was purified by silica gel chromatograph to yield the desired product (53 mg, 72%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.07 (s, 1H), 6.14 (s, 1H), 4.32–4.20 (br, 2H), 3.78 (s, 3H), 2.04 (s, 3H).</div></div><div id="sec4_1_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> 2-Amino-5-fluoro-4-methoxybenzonitrile (<b>35b</b>)</h4><div class="NLM_p last">The title compound (50% over 2 steps) was prepared according to the synthetic procedure for compound <b>35a.</b><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.05 (d, <i>J</i> = 10.5 Hz, 1H), 6.24 (d, <i>J</i> = 7.1 Hz, 2H), 4.38–4.17 (br, 2H), 3.86 (s, 3H).</div></div><div id="sec4_1_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> 2-Amino-5-methoxy-4-methylbenzonitrile (<b>36a</b>)</h4><div class="NLM_p last">The title compound (75%) was prepared from <b>34a</b> according to the synthetic procedure for compound <b>35a</b>. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>) δ 6.74 (s, 1H), 6.58 (s, 1H), 4.16–4.06 (br, 2H), 3.76 (s, 3H), 2.19 (s, 3H).</div></div><div id="sec4_1_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> 2-Amino-4-fluoro-5-methoxybenzonitrile (<b>36b</b>)</h4><div class="NLM_p last">To a solution of <b>34b</b> (127 mg, 0.58 mmol) in DMF (160 mL) was added CuCN (126 mg, 1.42 mmol). After being stirred for 18 h in sealed tube at 120 °C, the reaction was cooled to rt, poured into ammonium hydroxide solution (3 mL), and extracted with DCM (3 × 3 mL). The combined organic layers were dried with anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The residue was purified by silica gel chromatograph to yield the desired compound (54 mg, 56%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.92 (d, <i>J</i> = 8.7 Hz, 1H), 6.49 (d, <i>J</i> = 12.2 Hz, 1H), 4.30–4.16 (br, 2H), 3.79 (s, 3H).</div></div><div id="sec4_1_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> 2,4-Dichloro-7-ethoxy-6-methoxyquinazoline (<b>39a</b>)</h4><div class="NLM_p last">To a solution of <b>30a</b> (1.06 g, 5.5 mmol) in DCM (12 mL) and DMF (12 mL) were added DIEA (1.8 mL, 10.9 mmol) and methyl chloroformate (0.62 mL, 8.2 mmol) at 0 °C. After being stirred for 2 h, the reaction solution was poured into water and extracted with DCM. The organic layer was dried with anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure to yield <b>37a</b> as brown solid. To a solution of <b>37a</b> in EtOH (40 mL) was added H<sub>2</sub>O<sub>2</sub> (35 mL) and NaOH (278 mg, 6.9 mmol). The mixture was refluxed for 2 h before being cooled to rt. The precipitated solid was collected and dried to yield the intermediate <b>38a</b>. A suspension of <b>38a</b> in PhNEt<sub>2</sub> (1.5 mL, 1.4 mmol) and POCl<sub>3</sub> (10 mL) was refluxed for 4 h before being cooled to rt. After removal of the solvent, cold water was added, followed by saturated aqueous NaHCO<sub>3</sub> solution. The mixture was extracted with ethyl acetate (3 × 30 mL). The combined organic layers were dried with anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The resulting residue was purified by silica gel chromatograph to yield the title compound (450 mg, 30%). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>) δ 7.38 (s, 1H), 7.29 (s, 1H), 4.31 (q, <i>J</i> = 7.0 Hz, 2H), 4.09 (s, 3H), 1.60 (t, <i>J</i> = 7.2 Hz, 3H).</div></div><div id="sec4_1_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> 2,4-Dichloro-7-isopropoxy-6-methoxyquinazoline (<b>39b</b>)</h4><div class="NLM_p last">The title compound (40%) was prepared according to the synthetic procedure for <b>39a</b>. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>) δ 7.37 (s, 1H), 7.29 (s, 1H), 4.83–4.79 (m, 1H), 4.07 (s, 3H), 1.52 (d, <i>J</i> = 6.1 Hz, 6H).</div></div><div id="sec4_1_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> 2,4-Dichloro-6-ethoxy-7-methoxyquinazoline (<b>39c</b>)</h4><div class="NLM_p last">The title compound (38%) was prepared according to the synthetic procedure for <b>39a</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.27 (s, 1H), 7.26 (s, 1H), 4.28 (q, <i>J</i> = 7.0 Hz, 2H), 4.05 (s, 3H), 1.58 (t, <i>J</i> = 7.0 Hz, 3H).</div></div><div id="sec4_1_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> 2,4-Dichloro-6-isopropoxy-7-methoxyquinazoline (<b>39d</b>)</h4><div class="NLM_p last">The title compound (40%) was prepared according to the synthetic procedure for <b>39a</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.27 (s, 1H), 7.26 (s, 1H), 4.83–4.77 (m, 1H), 4.03 (s, 3H), 1.49 (d, <i>J</i> = 6.1 Hz, 6H).</div></div><div id="sec4_1_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> 2,4-Dichloro-6-methoxy-7-methylquinazoline (<b>39e</b>)</h4><div class="NLM_p last">The title compound (39%) was prepared according to the synthetic procedure for <b>39a</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.30 (s, 1H), 7.24 (s, 1H), 4.01 (s, 3H), 2.43 (s, 3H).</div></div><div id="sec4_1_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> 2,4-Dichloro-7-fluoro-6-methoxyquinazoline (<b>39f</b>)</h4><div class="NLM_p last">The title compound (31%) was prepared according to the synthetic procedure for <b>39a</b>. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>) δ 7.67 (d, <i>J</i> = 10.8 Hz, 1H), 7.55 (d, <i>J</i> = 8.4 Hz, 1H), 4.10 (s, 3H).</div></div><div id="sec4_1_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> 2,4-Dichloro-6-fluoro-7-methoxyquinazoline (<b>39g</b>)</h4><div class="NLM_p last">The title compound (30%) was prepared according to the synthetic procedure for <b>39a</b>. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>) 7.78 (dd, <i>J</i> = 10.3, 0.9 Hz, 1H), 7.33 (d, <i>J</i> = 7.4 Hz, 1H), 4.07 (s, 3H).</div></div><div id="sec4_1_36" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> 2,4-Dichloro-7-methoxy-6-methylquinazoline (<b>39h</b>)</h4><div class="NLM_p last">The title compound (34%) was prepared according to the synthetic procedure for <b>39a</b>. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>) δ 7.95 (s, 1H), 7.24 (s, 1H), 4.03 (s, 3H), 2.43 (s, 3H).</div></div><div id="sec4_1_37" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> 6-Ethoxy-7-methoxy-<i>N</i>-(1-methylpiperidin-4-yl)-2-morpholinoquinazolin-4-amine (<b>41</b>)</h4><div class="NLM_p last">The title compound (79%) was prepared from <b>39a</b> according to the synthetic procedure for <b>4</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.87 (s, 1H), 6.77 (s, 1H), 5.09 (d, <i>J</i> = 7.6 Hz, 1H), 4.10–4.05 (m, 3H), 3.879 (s, 3H), 3.80–3.73 (m, 8H), 2.84–2.82 (m, 2H), 2.28 (s, 3H), 2.17–2.10 (m, 4H), 1.64–1.54 (m, 2H), 1.44 (t, <i>J</i> = 6.8 Hz, 3H). MS (ESI) <i>m</i>/<i>z</i> 402.3 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_38" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> 6-Isopropoxy-7-methoxy-<i>N</i>-(1-methylpiperidin-4-yl)-2-morpholinoquinazolin-4-amine (<b>42</b>)</h4><div class="NLM_p last">The title compound (77%) was prepared from <b>39b</b> according to the synthetic procedure for <b>4</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.88 (s, 1H), 6.86 (s, 1H), 5.09 (d, <i>J</i> = 7.6 Hz, 1H), 4.52–4.46 (m, 1H), 4.12–4.03 (m, 1H), 3.87 (s, 3H), 3.79–3.72 (m, 8H), 2.84–2.81 (m, 2H), 2.28 (s, 3H), 2.17–2.08 (m, 4H), 1.64–1.54 (m, 2H), 1.32 (d, <i>J</i> = 6.4 Hz, 6H). MS (ESI) <i>m</i>/<i>z</i> 416.3 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_39" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> 6-Fluoro-7-methoxy-<i>N</i>-(1-methylpiperidin-4-yl)-2-morpholinoquinazolin-4-amine (<b>43</b>)</h4><div class="NLM_p last">The title compound (85%) was prepared from <b>39c</b> according to the synthetic procedure for <b>4</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.13 (d, <i>J</i> = 11.6 Hz, 1H), 6.90 (d, <i>J</i> = 8.4 Hz, 1H), 5.04 (d, <i>J</i> = 7.2 Hz, 1H), 4.11–4.02 (m, 1H), 3.91 (s, 3H), 3.81–3.76 (m, 8H), 2.83–2.80 (m, 2H), 2.29 (s, 3H), 2.19–2.07 (m, 4H), 1.64–1.55 (m, 2H). MS (ESI) <i>m</i>/<i>z</i> 376.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_40" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> 7-Methoxy-6-methyl-<i>N</i>-(1-methylpiperidin-4-yl)-2-morpholinoquinazolin-4-amine (<b>44</b>)</h4><div class="NLM_p last">The title compound (76%) was prepared from <b>39d</b> according to the synthetic procedure for <b>4</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.18 (s, 1H), 6.79 (s, 1H), 5.15 (d, <i>J</i> = 7.2 Hz, 1H), 4.12–4.03 (m, 1H), 3.86 (s, 3H), 3.81–3.76 (m, 8H), 2.83–2.80 (m, 2H), 2.29 (s, 3H), 2.23 (s, 3H), 2.18–2.08 (m, 4H), 1.64–1.554 (m, 2H). MS (ESI): 372.3 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_41" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> 7-Ethoxy-6-methoxy-<i>N</i>-(1-methylpiperidin-4-yl)-2-morpholinoquinazolin-4-amine (<b>45</b>)</h4><div class="NLM_p last">The title compound (78%) was prepared from <b>39e</b> according to the synthetic procedure for <b>4</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.85 (s, 1H), 6.77 (s, 1H), 5.17 (d, <i>J</i> = 7.6 Hz, 1H), 4.12 (q, <i>J</i> = 7.2 Hz, 2H), 4.09–4.03 (m, 1H), 3.87 (s, 3H), 3.78–3.74 (m, 8H), 2.84–2.81 (m, 2H), 2.27 (s, 3H), 2.16–2.09 (m, 4H), 1.63–1.53 (m, 2H), 1.07 (t, <i>J</i> = 6.8 Hz, 3H). MS (ESI) <i>m</i>/<i>z</i> 402.3 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_42" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> 7-Isopropoxy-6-methoxy-<i>N</i>-(1-methylpiperidin-4-yl)-2-morpholinoquinazolin-4-amine (<b>46</b>)</h4><div class="NLM_p last">The title compound (81%) was prepared from <b>39f</b> according to the synthetic procedure for <b>4</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.85 (s, 1H), 6.79 (s, 1H), 5.20 (d, <i>J</i> = 6.8 Hz, 1H), 4.69–4.60 (m, 1H), 4.12–4.03 (m, 1H), 3.85 (s, 3H), 3.79–3.72 (m, 8H), 2.84–2.81 (m, 2H), 2.26 (s, 3H), 2.15–2.08 (m, 4H), 1.62–1.53 (m, 2H), 1.36 (d, <i>J</i> = 6.0 Hz, 6H). MS (ESI) <i>m</i>/<i>z</i> 416.3 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_43" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> 7-Fluoro-6-methoxy-<i>N</i>-(1-methylpiperidin-4-yl)-2-morpholinoquinazolin-4-amine (<b>47</b>)</h4><div class="NLM_p last">The title compound (85%) was prepared from <b>39g</b> according to the synthetic procedure for <b>4</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.15 (d, <i>J</i> = 12.8 Hz, 1H), 6.90 (d, <i>J</i> = 8.8 Hz, 1H), 5.28 (d, <i>J</i> = 7.6 Hz, 1H), 4.19–4.10 (m, 1H), 3.95 (s, 3H), 3.83–3.74 (m, 8H), 2.97–2.94 (m, 2H), 2.38 (s, 3H), 2.56 (t, <i>J</i> = 11.2 Hz, 2H), 2.18–2.15 (m, 2H), 1.79–1.69 (m, 2H). MS (ESI) <i>m</i>/<i>z</i> 376.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_44" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> 6-Methoxy-7-methyl-<i>N</i>-(1-methylpiperidin-4-yl)-2-morpholinoquinazolin-4-amine (<b>48</b>)</h4><div class="NLM_p last">The title compound (75%) was prepared from <b>39h</b> according to the synthetic procedure for <b>4</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.26 (s, 1H), 6.65 (s, 1H), 5.19 (d, <i>J</i> = 7.2 Hz, 1H), 4.13–4.05 (m, 1H), 3.83 (s, 3H), 3.79–3.71 (m, 8H), 2.84–2.82 (m, 2H), 2.28 (s, 3H), 2.24 (s, 3H), 2.16–2.11 (m, 4H), 1.64–1.55 (m, 2H). MS (ESI) <i>m</i>/<i>z</i> 372.3 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_45" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> <i>N</i>-(2-Cyano-4,5-dimethoxyphenyl)tetrahydro-2<i>H</i>-pyran-4-carboxamide (<b>50a</b>)</h4><div class="NLM_p last">To a solution of 2-amino-4,5-dimethoxybenzonitrile (372 mg, 2.1 mmol) in DCM (10 mL) were added DIEA (0.55 mL, 3.3 mmol) and tetrahydro-2<i>H</i>-pyran-4-carbonyl chloride (374 mg, 2.5 mmol) at 0 °C. After being stirred for 2 h, the reaction was concentrated, and the resulting residue was purified by silica gel chromatography to yield the title compound (577 mg, 95%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.49 (s, 1H), 6.92 (s, 1H), 5.92–5.51 (br, 1H), 4.07 (t, <i>J</i> = 3.5 Hz, 1H), 4.04 (t, <i>J</i> = 3.5 Hz, 1H), 4.00 (dd, <i>J</i> = 7.3, 1.2 Hz, 1H), 3.96 (s, 3H), 3.89 (s, 3H), 3.52–3.41 (m, 2H), 1.96–1.85 (m, 4H).</div></div><div id="sec4_1_46" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> 4-Chloro-6,7-dimethoxy-2-(tetrahydro-2<i>H</i>-pyran-4-yl)quinolone (<b>52a</b>)</h4><div class="NLM_p last">To a solution of <b>50a</b> (405 mg, 1.4 mmol) in EtOH (10 mL) were added H<sub>2</sub>O<sub>2</sub> (10 mL) and NaOH (160 mg, 4.0 mmol). The resulting mixture was heated under reflux for 2 h before being cooled to rt. The precipitated solid was collected and dried under vacuum to yield intermediate <b>51a</b> as a yellow solid. A mixture of <b>51a</b>, PhNEt<sub>2</sub> (1.5 mL, 1.4 mmol), and POCl<sub>3</sub> (10 mL) was refluxed for 4 h before being cooled to rt. After removal of the solvent, cold water was added, followed by saturated aqueous NaHCO<sub>3</sub> solution. The mixture was extracted with ethyl acetate (3 × 30 mL). The combined organic layers were dried with anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The resulting residue was purified by silica gel chromatograph to yield the title compound (240 mg, 55%). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>) δ 7.40 (s, 1H), 7.33 (s, 1H), 4.14 (dd, <i>J</i> = 11.4, 3.2 Hz, 2H), 4.09 (s, 3H), 4.09 (s, 3H), 3.61 (t, <i>J</i> = 11.0 Hz, 2H), 3.34–3.11 (m, 1H), 2.11 (m 2H), 2.03 (m, 2H).</div></div><div id="sec4_1_47" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> 4-Chloro-2-cyclohexyl-6,7-dimethoxyquinoline (<b>52b</b>)</h4><div class="NLM_p last">The title compound (41% over 3 steps) was prepared according to synthetic procedure for <b>52a</b>. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>) δ 7.39 (s, 1H), 7.32 (s, 1H), 4.08 (s, 6H), 2.97 (tt, <i>J</i> = 11.9, 3.4 Hz, 1H), 2.16–2.01 (m, 2H), 1.92–1.90 (m, 2H), 1.83–1.67 (m, 3H), 1.52–1.41 (m, 2H), 1.40–1.30 (m, 1H).</div></div><div id="sec4_1_48" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> 2-Cyclohexyl-6,7-dimethoxy-<i>N</i>-(1-methylpiperidin-4-yl)quinazolin-4-amine (<b>53</b>)</h4><div class="NLM_p last">A mixture of <b>52a</b> (60 mg, 0.2 mmol), 4-methylpiperidin-1-amine (0.05 mL, 0.40 mmol), and DIEA (0.06 mL, 0.36 mmol) in <i>i</i>-PrOH (1 mL) was heated by microwave irradiation at 160 °C for 15 min. After concentration, the crude product was purified by preparative HPLC to yield the title compound (69 mg, 90%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.13 (s, 1H), 6.86 (s, 1H), 5.31 (d, <i>J</i> = 6.8 Hz, 1H), 4.25–4.16 (m, 1H), 3.90 (s, 3H), 3.89 (s, 3H), 2.86–2.83 (m, 2H), 2.70 (tt, <i>J</i> = 11.6, 3.6 Hz, 1H), 2.29 (s, 3H) 2.21–2.12 (m, 4H), 1.97–1.94 (m, 2H), 1.82–1.79 (m, 2H), 1.72–1.54 (m, 5H), 1.42–1.22 (m, 3H). MS (ESI) <i>m</i>/<i>z</i> 385.3 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_49" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> 6,7-Dimethoxy-<i>N</i>-(1-methylpiperidin-4-yl)-2-(tetrahydro-2<i>H</i>-pyran-4-yl)quinazolin-4-amine (<b>54</b>)</h4><div class="NLM_p last">The title compound (80%) was prepared according to synthetic procedure for <b>53</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.12 (s, 1H), 6.83 (s, 1H), 5.26 (d, <i>J</i> = 7.6 Hz, 1H), 4.25–4.16 (m, 1H), 4.05 (dd, <i>J</i> = 11.2, 4.4 Hz, 2H), 3.92 (s, 6H), 3.52 (td, <i>J</i> = 12.0, 1.6 Hz, 2H), 2.93 (tt, <i>J</i> = 12.0, 4.0 Hz, 1H), 2.86–2.83 (m, 2H), 2.29 (s, 3H) 2.20–2.12 (m, 4H), 2.04 (qd, <i>J</i> = 12.0, 4.4 Hz, 2H), 1.87–1.84 (m, 2H), 1.59 (qd, <i>J</i> = 12.0, 4.0 Hz, 2H). MS (ESI) <i>m</i>/<i>z</i> 387.3 [M + H]<sup>+</sup>.</div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i66" class="anchor-spacer"></div><h3 class="article-section__title" id="_i66"> GLP and G9a IC<sub>50</sub> Determination</h3><div class="NLM_p last">Methyltransferase activity assays for G9a and GLP were performed by monitoring the incorporation of tritium-labeled methyl group to lysine 9 of H3 (1–25) peptide using scintillation proximity assay (SPA). The enzymatic reactions were performed at 23 °C with 20 min incubation of 10 μL of reaction mixture in 25 mM potassium phosphate pH 8.0, 1 mM EDTA, 2 mM MgCl<sub>2</sub>, 0.01% Triton X-100 containing 8 μM of cold SAM, and 2 μM of <sup>3</sup>H-SAM (catalogue no. NET155 V250UC; PerkinElmer; <a href="http://www.perkinelmer.com" class="extLink">www.perkinelmer.com</a>), 1 μM of biotinylated H3 (1–25), 5 nM G9a or GLP, and compound titrations from 1.5 nM to 25 μM. To stop the reactions, 10 μL of 7.5 M guanidine hydrochloride was added, followed by 60 μL of buffer (20 mM Tris, pH 8.0), mixed, and transferred to a 384-well streptavidin coated Flash-plate (PerkinElmer, <a href="http://www.perkinelmer.ca" class="extLink">http://www.perkinelmer.ca</a>). After mixing, the mixtures in Flash-plate were incubated for 2 h, and the CPM counts were measured using Topcount plate reader (PerkinElmer, <a href="http://www.perkinelmer.com" class="extLink">www.perkinelmer.com</a>). The CPM counts in the absence of compound for each data set were defined as 100% activity. In the absence of the enzyme, the CPM counts in each data set were defined as background (0%). All enzymatic reactions were performed in triplicate, and IC<sub>50</sub> values were determined by fitting the data to Four Parameter Logistic equation using GraphPad Prism 7 software.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i67" class="anchor-spacer"></div><h3 class="article-section__title" id="_i67"> Selectivity Assays</h3><div class="NLM_p">Effect of compound on methyltransferase activity of SUV39H1, SUV39H2, SETDB1, SETD8, SUV420H1, SUV420H2, SETD7, MLL1 trimeric complex, MLL3 pentameric complex, EZH2 trimeric complex, PRMT1, PRMT3, PRMT4, PRMT5/MEP50 complex, PRMT6, PRMT7, PRMT8, PRMT9, PRDM9, SETD2, BCDIN3D, METTL3-14, SMYD2, SMYD3, and DNMT1 was assessed by monitoring the incorporation of tritium-labeled methyl group to lysine or arginine residues of peptide substrates using scintillation proximity assay (SPA). Assays were performed in a 10 μL reaction mixture containing <sup>3</sup>H-SAM (catalogue no. NET155 V250UC; PerkinElmer; <a href="http://www.perkinelmer.com" class="extLink">www.perkinelmer.com</a>) and substrate at concentrations close to <i>K</i><sub>m</sub> values for each enzyme. Two concentrations (1 μM and 10 μM) of compound were used in all selectivity assays. To stop the enzymatic reactions, 10 μL of 7.5 M guanidine hydrochloride was added, followed by 180 μL of buffer (20 mM Tris, pH 8.0), mixed, and then transferred to a 96-well FlashPlate (catalogue no. SMP103; PerkinElmer; <a href="http://www.perkinelmer.com" class="extLink">www.perkinelmer.com</a>). After mixing, the reaction mixtures in Flash plates were incubated for 1 h and the CPM were measured using Topcount plate reader (PerkinElmer, <a href="http://www.perkinelmer.com" class="extLink">www.perkinelmer.com</a>). The CPM counts in the absence of compound for each data set were defined as 100% activity. In the absence of the enzyme, the CPM counts in each data set were defined as background (0%).</div><div class="NLM_p">For DNMT1, the dsDNA substrate was prepared by annealing two complementary strands (biotinylated forward strand B-GAGCCCGTAAGCCCGTTCAGGTCG and reverse strand CGACCTGAACGGGCTTACGGGCTC), synthesized by Eurofins MWG Operon.</div><div class="NLM_p last">For DOT1L, ASH1L, DNMT3A/3L, and DNMT3B/3L, a filter-based assay was used. In this assay, 10 μL of reaction mixtures were incubated at RT for 1 h, and 50 μL of 10% TCA was added, mixed, and transferred to filter-plates (Millipore; catalogue no. MSFBN6B10; <a href="http://www.millipore.com" class="extLink">www.millipore.com</a>). Plates were centrifuged at 2000 rpm (Allegra X-15R - Beckman Coulter, Inc.) for 2 min followed by two additional 10% TCA washes and one ethanol wash (180 μL) followed by centrifugation. Plates were dried, and 100 μL of MicroO (MicroScint-O; catalogue no. 6013611, PerkinElmer; <a href="http://www.perkinelmer.com" class="extLink">www.perkinelmer.com</a>) were added to each well, centrifuged, and removed. Then 70 μL of MicroO was added again and CPM was measured using a Topcount plate reader.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i68" class="anchor-spacer"></div><h3 class="article-section__title" id="_i68"> Isothermal Titration Calorimetry</h3><div class="NLM_p last">Isothermal titration calorimetry (ITC) measurements were made at 25 °C on a MicroCal ITC200 Instrument (Malvern Instruments). Co-concentrated G9a–SAM and GLP–SAM (protein/SAM molar ratio of 1:5) were diluted at 35 μM in ITC buffer [50 mM Tris (pH 8.0), 150 mM NaCl] supplemented with 1% DMSO. Compounds <b>4</b> and <b>18</b> were prepared in DMSO at 50 mM and diluted to 0.5 mM in ITC buffer with a final DMSO concentration of 1%. Binding constants were calculated by fitting the data using the ITC data analysis module in Origin 7.0 (OriginLab Corp.).</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i69" class="anchor-spacer"></div><h3 class="article-section__title" id="_i69"> Mechanism of Action (MOA) Studies</h3><div class="NLM_p last">IC<sub>50</sub> values were determined for compound <b>4</b> and <b>18</b> at various concentrations of SAM (3.1, 6.2,12.5, 25, 50, and 100 μM), 5 nM of GLP, 5 μM of biotinylated H3 (1–25) peptide, or at various concentrations of biotinylated H3 (1–25) peptide substrate (0.31, 0.62, 1.2, 2.5, 5, and 10 μM), 5 nM of GLP, and 50 μM SAM. The reaction mixtures were incubated for 15 min at 23 °C. To stop the enzymatic reactions, an equal volume of 7.5 M guanidine hydrochloride was added, followed by 180 μL of buffer (20 mM Tris, pH 8.0), mixed, and then transferred to a 96-well FlashPlate (catalogue no. SMP103; PerkinElmer; <a href="http://www.perkinelmer.com" class="extLink">www.perkinelmer.com</a>). After mixing, the reaction mixtures in Flash plates were incubated for 2 h and the CPM were measured using a Topcount plate reader (PerkinElmer, <a href="http://www.perkinelmer.com" class="extLink">www.perkinelmer.com</a>). The CPM counts in the absence of compound for each data set were defined as 100% activity. In the absence of the enzyme, the CPM counts in each data set were defined as background (0%). The IC<sub>50</sub> values were determined using GraphPad Prism 7 software.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i70" class="anchor-spacer"></div><h3 class="article-section__title" id="_i70"> Protein Expression, Purification, Crystallization, Data Collection, and Structure Determination for Compound <b>4</b></h3><div class="NLM_p">The recombinant proteins GLP catalytic SET domain (<b>982</b>–<b>1266</b>) and G9a catalytic SET domain (<b>913</b>–<b>1193</b>), from plasmids given by Alexander Plotnikov, were overexpressed in <i>Escherichia coli</i> BL21 (DE3) codon plus RIL strain (Stratagene) by addition of 0.3 mM isopropyl-1-thio-<span class="smallcaps smallerCapital">d</span>-galactopyranoside with a supplement of 1 mM zinc sulfate and incubated overnight at 15 °C. Harvested cells were resuspended in 50 mM sodium phosphate buffer, pH 7.5, supplemented with 0.5 mM sodium chloride and 5% glycerol, and lysed using a microfluidizer (Microfluidics) at 10000 psi. After clarification of the crude extract by high-speed centrifugation, the lysate was loaded onto a 5 mL HiTrap chelating column (GE Healthcare) charged with Ni2+. The column was washed, and the protein was eluted with 50 mM Tris buffer, pH 8.0, 250 mM sodium chloride, 250 mM imidazole, and supplemented with 0.5 mM of TCEP. The protein was next purified on a Superdex200 column (26/600; GE Healthcare) equilibrated with 50 mM Tris-HCl buffer, pH 8.0, and 150 mM sodium chloride, and the elution fractions were pooled and supplemented with 0.5 mM of TCEP. All purification steps were performed at 4 °C and in the presence of a protease inhibitor AEBSF (Goldbio). The size and the purity of the recombinant G9a and GLP proteins were checked by SDS-PAGE and mass spectroscopy.</div><div class="NLM_p">Purified GLP and G9a were mixed in the presence of <i>S</i>-adenosyl-<span class="smallcaps smallerCapital">l</span>-methionine (Sigma) and compound <b>4</b> (protein/SAM/<b>4</b> molar ratio of 1:5:10), coconcentrated to about 20 mg/mL and crystallized using the hanging drop vapor diffusion method at 17 °C by mixing equal volume of the protein solution with the reservoir solution. The GLP–SAM–<b>4</b> ternary complex was crystallized in 20% PEG 20,000, 2% (v/v) 1,4-dioxane, 0.1 M bicine (pH 9.0). The G9a–SAM–<b>4</b> ternary complex was crystallized in 20% (w/v) PEG 3350, 0.2 M NaF, 0.1 M Bis-Tris propane (pH 6.5). GLP–SAM–<b>4</b> crystals were soaked in the corresponding mother liquor supplemented with 20% ethylene glycol, and the G9a–SAM–<b>4</b> complex crystals in a mixture of parafin 70% and silicon 30% as cryoprotectant before flash freezing in liquid nitrogen.</div><div class="NLM_p last">X-ray diffraction data were collected at 100 K at NE-CAT beamline 24-ID-E of Advanced Photon Source (APS) at Argonne National Laboratory. The data were processed with MOSFLM, SCALA, and other programs from the CCP4 suite.<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> The structures of the G9a and GLP complexes were solved by molecular replacement using the PHASER<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> and the atomic model of the G9a and GLP SET domains (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3K5K">3K5K</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3HNA">3HNA</a>, respectively). The locations of the bound molecules were determined from a Fo–Fc difference electron density map. REFMAC<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> and phenix.refine<a onclick="showRef(event, 'ref61 ref62'); return false;" href="javascript:void(0);" class="ref ref61 ref62">(61, 62)</a> were used for structure refinement. Graphic program COOT<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a> was used for model building and visualization. The overall assessment of model quality was performed using MOLPROBITY.<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a></div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i71" class="anchor-spacer"></div><h3 class="article-section__title" id="_i71"> Protein Expression, Purification, Crystallization, Data Collection, and Structure Determination for Compound <b>18</b></h3><div class="NLM_p">Human G9a and GLP catalytic domains were cloned, expressed, and purified as previously described.<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> Purified G9a and GLP proteins were premixed with compound <b>18</b> at 1:2 molar ratio of protein:compound. The protein–compound complexes were crystallized using sitting drop vapor diffusion method at 20 °C by mixing an equal volume of the protein solution with the reservoir solution containing 25% PEG3350, 0.2 M NaCl, 0.1 M Bis-Tris pH 6.5 (for G9a), and 20% 2-propanol, 20% PEG 4 K, 0.1 M Na citrate, pH 5.6 (for GLP).</div><div class="NLM_p last">X-ray diffraction data for G9a–<b>18</b> and GLP–<b>18</b> were both collected at 100 K at beamline 19ID of Advanced Photon Source (APS), Argonne National Laboratory. Both data sets were processed using the HKL-3000 suite.<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> The structures of G9a and GLP with <b>18</b> were solved by molecular replacement using MOLREP,<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> with PDBs <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3K5K">3K5K</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4I51">4I51</a> as search template, respectively. REFMAC<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> was used for structure refinement. Geometry restraints for the compound refinement were prepared with by GRADE Developed at Global Phasing Ltd. (O. S. Smart, T. O. Womack, A. Sharff, C. Flensburg, P. Keller, W. Paciorek, C. Vonrhein, and G. Bricogne (2011) grade v1.102. Cambridge, United Kingdom, Global Phasing Ltd.) Graphics program COOT<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a> was used for model building and visualization. MOLPROBITY<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> was used for structure validation. Crystal diffraction data and refinement statistics for the structure are displayed in <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i72"><a href="/doi/suppl/10.1021/acs.jmedchem.6b01645">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_10335" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_10335" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.6b01645" class="ext-link">10.1021/acs.jmedchem.6b01645</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline"><sup>1</sup>H and <sup>13</sup>C NMR spectra of compound <b>4</b> and <b>18</b>; crystallography data and refinement statistics (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01645/suppl_file/jm6b01645_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01645/suppl_file/jm6b01645_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01645/suppl_file/jm6b01645_si_001.pdf">jm6b01645_si_001.pdf (282.66 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01645/suppl_file/jm6b01645_si_002.csv">jm6b01645_si_002.csv (1.83 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The structure of GLP in complex with <b>4</b> has been deposited under PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TUZ">5TUZ</a>. The structure of G9a in complex with <b>4</b> has been deposited under PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TUY">5TUY</a>. The structure of GLP in complex with <b>18</b> has been deposited under PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TTG">5TTG</a>. The structure of G9a in complex with <b>18</b> has been deposited under PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TTF">5TTF</a>. Authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.6b01645" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_40535" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_40535" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Masoud Vedadi</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Pharmacology and Toxicology, University
of Toronto, Toronto, Ontario M5S 1A8, Canada</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#95f8bbe3f0f1f4f1fcd5e0e1fae7fafbe1fabbf6f4"><span class="__cf_email__" data-cfemail="513c7f2734353035381124253e233e3f253e7f3230">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jian Jin</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacological Sciences and Department of Oncological Sciences, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2387-3862" title="Orcid link">http://orcid.org/0000-0002-2387-3862</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#c3a9aaa2adeda9aaad83aeb0b0aeeda6a7b6"><span class="__cf_email__" data-cfemail="63090a020d4d090a0d230e10100e4d060716">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yan Xiong</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacological Sciences and Department of Oncological Sciences, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fengling Li</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nicolas Babault</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacological Sciences and Department of Oncological Sciences, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Aiping Dong</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hong Zeng</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hong Wu</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xin Chen</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacological Sciences and Department of Oncological Sciences, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Cheryl H. Arrowsmith</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Princess
Margaret Cancer Centre and Department of Medical Biophysics, University of Toronto, Toronto, Ontario M5G 2M9, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peter J. Brown</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jing Liu</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacological Sciences and Department of Oncological Sciences, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d225e3733-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i74">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_74066" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_74066" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Albina Bolotokova for compound management and Taraneh Hajian for protein purification. The research described here was supported by the grant R01GM103893 (to J.J.) from the U.S. National Institutes of Health. The SGC is a registered charity (no. 1097737) that receives funds from AbbVie, Bayer Pharma AG, Boehringer Ingelheim, Canada Foundation for Innovation, Eshelman Institute for Innovation, Genome Canada, Innovative Medicines Initiative (EU/EFPIA) [ULTRA-DD grant no. 115766], Janssen, Merck & Co., Novartis Pharma AG, Ontario Ministry of Economic Development and Innovation, Pfizer, São Paulo Research Foundation-FAPESP, Takeda, and the Wellcome Trust. We thank the staff at the 24-ID-E beamline of Advanced Photon Source (APS) at Argonne National Laboratory for facilitating x-ray data collection.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i75" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i75"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i76" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i76"> Abbreviations Used</h2><tr><td class="NLM_term">GLP</td><td class="NLM_def"><p class="first last">G9a-like protein 1</p></td></tr><tr><td class="NLM_term">PKMT</td><td class="NLM_def"><p class="first last">protein lysine methyltransferase</p></td></tr><tr><td class="NLM_term">PRMT</td><td class="NLM_def"><p class="first last">protein arginine methyltransferase</p></td></tr><tr><td class="NLM_term">H3K9me1</td><td class="NLM_def"><p class="first last">monomethylation of lysine 9 on histone H3</p></td></tr><tr><td class="NLM_term">H3K9me2</td><td class="NLM_def"><p class="first last">dimethylation of lysine 9 on histone H3</p></td></tr><tr><td class="NLM_term">SAM</td><td class="NLM_def"><p class="first last"><i>S</i>-5′-adenosyl-<span class="smallcaps smallerCapital">l</span>-methionine</p></td></tr><tr><td class="NLM_term">SAH</td><td class="NLM_def"><p class="first last">SAH, <i>S</i>-5′-adenosyl-<span class="smallcaps smallerCapital">l</span>-homocysteine</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">DNMT</td><td class="NLM_def"><p class="first last">DNA methyltransferase</p></td></tr><tr><td class="NLM_term">RNMT</td><td class="NLM_def"><p class="first last">RNA methyltransferases</p></td></tr><tr><td class="NLM_term">SET</td><td class="NLM_def"><p class="first last">suppressor of variegation 3–9 enhancer-of-zeste trithorax</p></td></tr><tr><td class="NLM_term">SIRT1</td><td class="NLM_def"><p class="first last">sirtuin 1</p></td></tr><tr><td class="NLM_term">MyoD</td><td class="NLM_def"><p class="first last">myogenic differentiation 1</p></td></tr><tr><td class="NLM_term">CDYL1</td><td class="NLM_def"><p class="first last">chromodomain Y-like protein</p></td></tr><tr><td class="NLM_term">WIZ</td><td class="NLM_def"><p class="first last">widely interspaced zinc finger motifs protein</p></td></tr><tr><td class="NLM_term">RMSD</td><td class="NLM_def"><p class="first last">root-mean-square deviation</p></td></tr><tr><td class="NLM_term">METTL3–14</td><td class="NLM_def"><p class="first last"><i>N</i>6-adenosine-methyltransferase subunit METTL3 and 14 complex</p></td></tr><tr><td class="NLM_term">ITC</td><td class="NLM_def"><p class="first last">isothermal titration calorimetry</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i77">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_83480" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_83480" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 67 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Martin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span> </span><span class="NLM_article-title">The diverse functions of histone lysine methylation</span> <span class="citation_source-journal">Nat. Rev. Mol. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">838</span><span class="NLM_x">–</span> <span class="NLM_lpage">849</span><span class="refDoi"> DOI: 10.1038/nrm1761</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1038%2Fnrm1761" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=16261189" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtFKlsL3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2005&pages=838-849&author=C.+Martinauthor=Y.+Zhang&title=The+diverse+functions+of+histone+lysine+methylation&doi=10.1038%2Fnrm1761"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The diverse functions of histone lysine methylation</span></div><div class="casAuthors">Martin, Cyrus; Zhang, Yi</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">838-849</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Covalent modifications of histone tails play fundamental roles in chromatin structure and function.  One such modification, lysine methylation, has important functions in many biol. processes that include heterochromatin formation, X-chromosome inactivation, and transcriptional regulation.  Here, the authors summarize recent advances in the understanding of how lysine methylation functions in these diverse biol. processes, and raise questions that need to be addressed in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotrSycSeCRB7Vg90H21EOLACvtfcHk0lgMDpAdZQZN1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtFKlsL3N&md5=8ca89b050ff6f72b0b12e05a3b5500d7</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrm1761&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm1761%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DThe%2520diverse%2520functions%2520of%2520histone%2520lysine%2520methylation%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2005%26volume%3D6%26spage%3D838%26epage%3D849%26doi%3D10.1038%2Fnrm1761" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Kaniskan, H. U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Chemical probes of histone lysine methyltransferases</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">40</span><span class="NLM_x">–</span> <span class="NLM_lpage">50</span><span class="refDoi"> DOI: 10.1021/cb500785t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500785t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFOht7%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=40-50&author=H.+U.+Kaniskanauthor=J.+Jin&title=Chemical+probes+of+histone+lysine+methyltransferases&doi=10.1021%2Fcb500785t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical probes of histone lysine methyltransferases</span></div><div class="casAuthors">Kaniskan, H. Umit; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">40-50</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Growing evidence suggests that histone-lysine methyltransferases (HMTs, also known as protein methyltransferases (PMTs)) play an important role in diverse biol. processes and human diseases by regulating gene expression and the chromatin state.  Therefore, HMTs have been increasingly recognized by the biomedical community as a class of potential therapeutic targets.  High quality chem. probes of HMTs, as tools for deciphering their physiol. functions and roles in human diseases and testing therapeutic hypotheses, are crit. for advancing this promising field.  Here, the authors focus on the discovery, characterization, and biol. applications of chem. probes for HMTs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpp8kb_c7Xl57Vg90H21EOLACvtfcHk0lgMDpAdZQZN1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFOht7%252FE&md5=4eb8a696e2227c60686df8d3c60777ce</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Fcb500785t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500785t%26sid%3Dliteratum%253Aachs%26aulast%3DKaniskan%26aufirst%3DH.%2BU.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DChemical%2520probes%2520of%2520histone%2520lysine%2520methyltransferases%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D40%26epage%3D50%26doi%3D10.1021%2Fcb500785t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Kaniskan, H. U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Konze, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Selective inhibitors of protein methyltransferases</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">1596</span><span class="NLM_x">–</span> <span class="NLM_lpage">1629</span><span class="refDoi"> DOI: 10.1021/jm501234a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501234a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFKqtbbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=1596-1629&author=H.+U.+Kaniskanauthor=K.+D.+Konzeauthor=J.+Jin&title=Selective+inhibitors+of+protein+methyltransferases&doi=10.1021%2Fjm501234a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibitors of protein methyltransferases</span></div><div class="casAuthors">Kaniskan, H. Umit; Konze, Kyle D.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1596-1629</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Mounting evidence suggests that protein methyltransferases (PMTs), which catalyze the methylation of histone and non-histone proteins, play a crucial role in diverse biol. processes and human diseases.  In particular, PMTs have been recognized as major players in regulating gene expression and chromatin state.  PMTs are divided into 2 categories: protein lysine methyltransferases (PKMTs) and protein arginine methyltransferases (PRMTs).  There has been a steadily growing interest in these enzymes as potential therapeutic targets and therefore discovery of PMT inhibitors has also been pursued increasingly over the past decade.  Here, the authors present a perspective on selective, small-mol. inhibitors of PMTs with an emphasis on their discovery, characterization, and applicability as chem. tools for deciphering the target PMTs' physiol. functions and involvement in human diseases.  The authors highlight the current state of PMT inhibitors and discuss future directions and opportunities for PMT inhibitor discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEWF_XeLEY1rVg90H21EOLACvtfcHk0lgMDpAdZQZN1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFKqtbbJ&md5=8fb2b2d21d306c02a2da60a3fb88c430</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Fjm501234a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501234a%26sid%3Dliteratum%253Aachs%26aulast%3DKaniskan%26aufirst%3DH.%2BU.%26aulast%3DKonze%26aufirst%3DK.%2BD.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DSelective%2520inhibitors%2520of%2520protein%2520methyltransferases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D1596%26epage%3D1629%26doi%3D10.1021%2Fjm501234a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Benevento, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van de Molengraft, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Westen, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Bokhoven, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kasri, N. N.</span><span> </span><span class="NLM_article-title">The role of chromatin repressive marks in cognition and disease: A focus on the repressive complex GLP/G9a</span> <span class="citation_source-journal">Neurobiol. Learn. Mem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">124</span><span class="NLM_x">, </span> <span class="NLM_fpage">88</span><span class="NLM_x">–</span> <span class="NLM_lpage">96</span><span class="refDoi"> DOI: 10.1016/j.nlm.2015.06.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1016%2Fj.nlm.2015.06.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=26143996" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFemtL7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2015&pages=88-96&author=M.+Beneventoauthor=M.+van+de+Molengraftauthor=R.+van+Westenauthor=H.+van+Bokhovenauthor=N.+N.+Kasri&title=The+role+of+chromatin+repressive+marks+in+cognition+and+disease%3A+A+focus+on+the+repressive+complex+GLP%2FG9a&doi=10.1016%2Fj.nlm.2015.06.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The role of chromatin repressive marks in cognition and disease: A focus on the repressive complex GLP/G9a</span></div><div class="casAuthors">Benevento, Marco; van de Molengraft, Marise; van Westen, Rhode; van Bokhoven, Hans; Nadif Kasri, Nael</div><div class="citationInfo"><span class="NLM_cas:title">Neurobiology of Learning and Memory</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">124</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">88-96</span>CODEN:
                <span class="NLM_cas:coden">NLMEFR</span>;
        ISSN:<span class="NLM_cas:issn">1074-7427</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Histone post-translational modifications are key epigenetic processes controlling the regulation of gene transcription.  In recent years it has become apparent that chromatin modifications contribute to cognition through the modulation of gene expression required for the expression and consolidation of memories.  In this review, we focus on the role of histone methylation in the nervous system.  Histone methylation is involved in a no. of cognitive disturbances, such as intellectual disability, cocaine addiction and age-related cognitive decline.  We provide an overview of the dynamic changes in methylation of histone lysine residues during learning and memory.  With a special focus on H3K9 histone methyltransferases GLP and G9a, we summarize the effects of deficiencies in writer and eraser enzymes on neuronal plasticity and cognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGob0r1EZs1wF7Vg90H21EOLACvtfcHk0lgs5ZkdB5mccA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFemtL7L&md5=3ef82f509c3a18b57502d1ddc707e51e</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.nlm.2015.06.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.nlm.2015.06.013%26sid%3Dliteratum%253Aachs%26aulast%3DBenevento%26aufirst%3DM.%26aulast%3Dvan%2Bde%2BMolengraft%26aufirst%3DM.%26aulast%3Dvan%2BWesten%26aufirst%3DR.%26aulast%3Dvan%2BBokhoven%26aufirst%3DH.%26aulast%3DKasri%26aufirst%3DN.%2BN.%26atitle%3DThe%2520role%2520of%2520chromatin%2520repressive%2520marks%2520in%2520cognition%2520and%2520disease%253A%2520A%2520focus%2520on%2520the%2520repressive%2520complex%2520GLP%252FG9a%26jtitle%3DNeurobiol.%2520Learn.%2520Mem.%26date%3D2015%26volume%3D124%26spage%3D88%26epage%3D96%26doi%3D10.1016%2Fj.nlm.2015.06.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Black, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Rechem, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whetstine, J. R.</span><span> </span><span class="NLM_article-title">Histone lysine methylation dynamics: establishment, regulation, and biological impact</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">491</span><span class="NLM_x">–</span> <span class="NLM_lpage">507</span><span class="refDoi"> DOI: 10.1016/j.molcel.2012.11.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1016%2Fj.molcel.2012.11.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=23200123" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslKqtLvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2012&pages=491-507&author=J.+C.+Blackauthor=C.+Van+Rechemauthor=J.+R.+Whetstine&title=Histone+lysine+methylation+dynamics%3A+establishment%2C+regulation%2C+and+biological+impact&doi=10.1016%2Fj.molcel.2012.11.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Lysine Methylation Dynamics: Establishment, Regulation, and Biological Impact</span></div><div class="casAuthors">Black, Joshua C.; Van Rechem, Capucine; Whetstine, Johnathan R.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">491-507</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  Histone lysine methylation has emerged as a crit. player in the regulation of gene expression, cell cycle, genome stability, and nuclear architecture.  Over the past decade, a tremendous amt. of progress has led to the characterization of Me modifications and the lysine methyltransferases (KMTs) and lysine demethylases (KDMs) that regulate them.  Here, we review the discovery and characterization of the KMTs and KDMs and the Me modifications they regulate.  We discuss the localization of the KMTs and KDMs as well as the distribution of lysine methylation throughout the genome.  We highlight how these data have shaped our view of lysine methylation as a key determinant of complex chromatin states.  Finally, we discuss the regulation of KMTs and KDMs by proteasomal degrdn., posttranscriptional mechanisms, and metabolic status.  We propose key questions for the field and highlight areas that we predict will yield exciting discoveries in the years to come.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbnBk0Al-Ce7Vg90H21EOLACvtfcHk0lgs5ZkdB5mccA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslKqtLvM&md5=b9a8e9c29c630592f5413f7e36974735</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2012.11.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2012.11.006%26sid%3Dliteratum%253Aachs%26aulast%3DBlack%26aufirst%3DJ.%2BC.%26aulast%3DVan%2BRechem%26aufirst%3DC.%26aulast%3DWhetstine%26aufirst%3DJ.%2BR.%26atitle%3DHistone%2520lysine%2520methylation%2520dynamics%253A%2520establishment%252C%2520regulation%252C%2520and%2520biological%2520impact%26jtitle%3DMol.%2520Cell%26date%3D2012%26volume%3D48%26spage%3D491%26epage%3D507%26doi%3D10.1016%2Fj.molcel.2012.11.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Mozzetta, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pontis, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ait-Si-Ali, S.</span><span> </span><span class="NLM_article-title">Functional crosstalk between lysine methyltransferases on histone substrates: the case of G9A/GLP and polycomb repressive complex 2</span> <span class="citation_source-journal">Antioxid. Redox Signaling</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">1365</span><span class="NLM_x">–</span> <span class="NLM_lpage">1381</span><span class="refDoi"> DOI: 10.1089/ars.2014.6116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1089%2Fars.2014.6116" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=25365549" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BC2MXotFajt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2015&pages=1365-1381&author=C.+Mozzettaauthor=J.+Pontisauthor=S.+Ait-Si-Ali&title=Functional+crosstalk+between+lysine+methyltransferases+on+histone+substrates%3A+the+case+of+G9A%2FGLP+and+polycomb+repressive+complex+2&doi=10.1089%2Fars.2014.6116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Functional Crosstalk Between Lysine Methyltransferases on Histone Substrates: The Case of G9A/GLP and Polycomb Repressive Complex 2</span></div><div class="casAuthors">Mozzetta, Chiara; Pontis, Julien; Ait-Si-Ali, Slimane</div><div class="citationInfo"><span class="NLM_cas:title">Antioxidants & Redox Signaling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1365-1381</span>CODEN:
                <span class="NLM_cas:coden">ARSIF2</span>;
        ISSN:<span class="NLM_cas:issn">1523-0864</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">Significance: Methylation of histone H3 on lysine 9 and 27 (H3K9 and H3K27) are two epigenetic modifications that have been linked to several crucial biol. processes, among which are transcriptional silencing and cell differentiation.  Recent Advances: Deposition of these marks is catalyzed by H3K9 lysine methyltransferases (KMTs) and polycomb repressive complex 2, resp.  Increasing evidence is emerging in favor of a functional crosstalk between these two major KMT families.  Crit. Issues: Here, we review the current knowledge on the mechanisms of action and function of these enzymes, with particular emphasis on their interplay in the regulation of chromatin states and biol. processes.  We outline their crucial roles played in tissue homeostasis, by controlling the fate of embryonic and tissue-specific stem cells, highlighting how their deregulation is often linked to the emergence of a no. of malignancies and neurol. disorders.  Future Directions: Histone methyltransferases are starting to be tested as drug targets.  A new generation of highly selective chem. inhibitors is starting to emerge.  These hold great promise for a rapid translation of targeting epigenetic drugs into clin. practice for a no. of aggressive cancers and neurol. disorders.  Antioxid. Redox Signal. 22, 1365-1381.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqA2o7h77ex5LVg90H21EOLACvtfcHk0lgs5ZkdB5mccA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXotFajt7o%253D&md5=67d37b6c59e5f10bec09a071d59ce443</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1089%2Fars.2014.6116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fars.2014.6116%26sid%3Dliteratum%253Aachs%26aulast%3DMozzetta%26aufirst%3DC.%26aulast%3DPontis%26aufirst%3DJ.%26aulast%3DAit-Si-Ali%26aufirst%3DS.%26atitle%3DFunctional%2520crosstalk%2520between%2520lysine%2520methyltransferases%2520on%2520histone%2520substrates%253A%2520the%2520case%2520of%2520G9A%252FGLP%2520and%2520polycomb%2520repressive%2520complex%25202%26jtitle%3DAntioxid.%2520Redox%2520Signaling%26date%3D2015%26volume%3D22%26spage%3D1365%26epage%3D1381%26doi%3D10.1089%2Fars.2014.6116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Mozzetta, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boyarchuk, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pontis, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ait-Si-Ali, S.</span><span> </span><span class="NLM_article-title">Sound of silence: the properties and functions of repressive Lys methyltransferases</span> <span class="citation_source-journal">Nat. Rev. Mol. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">499</span><span class="NLM_x">–</span> <span class="NLM_lpage">513</span><span class="refDoi"> DOI: 10.1038/nrm4029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1038%2Fnrm4029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=26204160" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtl2ht77E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2015&pages=499-513&author=C.+Mozzettaauthor=E.+Boyarchukauthor=J.+Pontisauthor=S.+Ait-Si-Ali&title=Sound+of+silence%3A+the+properties+and+functions+of+repressive+Lys+methyltransferases&doi=10.1038%2Fnrm4029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Sound of silence: the properties and functions of repressive Lys methyltransferases</span></div><div class="casAuthors">Mozzetta, Chiara; Boyarchuk, Ekaterina; Pontis, Julien; Ait-Si-Ali, Slimane</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">499-513</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The methylation of histone Lys residues by Lys methyltransferases (KMTs) regulates chromatin organization and either activates or represses gene expression, depending on the residue that is targeted.  KMTs are emerging as key components in several cellular processes, and their deregulation is often assocd. with pathogenesis.  Here, we review the current knowledge on the main KMTs that are assocd. with gene silencing: namely, those responsible for methylating histone H3 Lys 9 (H3K9), H3K27 and H4K20.  We discuss their biochem. properties and the various mechanisms by which they are targeted to the chromatin and regulate gene expression, as well as new data on the interplay between them and other chromatin modifiers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzrGSxI8FBnbVg90H21EOLACvtfcHk0lhsOAwCx3xkRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtl2ht77E&md5=e32aa80d89533b1cfd04a90c168882d9</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnrm4029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm4029%26sid%3Dliteratum%253Aachs%26aulast%3DMozzetta%26aufirst%3DC.%26aulast%3DBoyarchuk%26aufirst%3DE.%26aulast%3DPontis%26aufirst%3DJ.%26aulast%3DAit-Si-Ali%26aufirst%3DS.%26atitle%3DSound%2520of%2520silence%253A%2520the%2520properties%2520and%2520functions%2520of%2520repressive%2520Lys%2520methyltransferases%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2015%26volume%3D16%26spage%3D499%26epage%3D513%26doi%3D10.1038%2Fnrm4029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Kouzarides, T.</span><span> </span><span class="NLM_article-title">Chromatin modifications and their function</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">128</span><span class="NLM_x">, </span> <span class="NLM_fpage">693</span><span class="NLM_x">–</span> <span class="NLM_lpage">705</span><span class="refDoi"> DOI: 10.1016/j.cell.2007.02.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1016%2Fj.cell.2007.02.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=17320507" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BD2sXis12ju7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2007&pages=693-705&author=T.+Kouzarides&title=Chromatin+modifications+and+their+function&doi=10.1016%2Fj.cell.2007.02.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatin modifications and their function</span></div><div class="casAuthors">Kouzarides, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">693-705</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The surface of nucleosomes is studded with a multiplicity of modifications.  At least eight different classes have been characterized to date and many different sites have been identified for each class.  Operationally, modifications function either by disrupting chromatin contacts or by affecting the recruitment of nonhistone proteins to chromatin.  Their presence on histones can dictate the higher-order chromatin structure in which DNA is packaged and can orchestrate the ordered recruitment of enzyme complexes to manipulate DNA.  In this way, histone modifications have the potential to influence many fundamental biol. processes, some of which may be epigenetically inherited.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAH9AX2dGicbVg90H21EOLACvtfcHk0lhsOAwCx3xkRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXis12ju7Y%253D&md5=c5995fe68fd9473f4268d59c01d3b9de</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.02.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.02.005%26sid%3Dliteratum%253Aachs%26aulast%3DKouzarides%26aufirst%3DT.%26atitle%3DChromatin%2520modifications%2520and%2520their%2520function%26jtitle%3DCell%26date%3D2007%26volume%3D128%26spage%3D693%26epage%3D705%26doi%3D10.1016%2Fj.cell.2007.02.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Bannister, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kouzarides, T.</span><span> </span><span class="NLM_article-title">Regulation of chromatin by histone modifications</span> <span class="citation_source-journal">Cell Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">381</span><span class="NLM_x">–</span> <span class="NLM_lpage">395</span><span class="refDoi"> DOI: 10.1038/cr.2011.22</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1038%2Fcr.2011.22" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=21321607" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BC3MXivVCgurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=381-395&author=A.+J.+Bannisterauthor=T.+Kouzarides&title=Regulation+of+chromatin+by+histone+modifications&doi=10.1038%2Fcr.2011.22"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of chromatin by histone modifications</span></div><div class="casAuthors">Bannister, Andrew J.; Kouzarides, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Cell Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">381-395</span>CODEN:
                <span class="NLM_cas:coden">CREEB6</span>;
        ISSN:<span class="NLM_cas:issn">1001-0602</span>.
    
            (<span class="NLM_cas:orgname">NPG Nature Asia-Pacific</span>)
        </div><div class="casAbstract">A review.  Chromatin is not an inert structure, but rather an instructive DNA scaffold that can respond to external cues to regulate the many uses of DNA.  A principal component of chromatin that plays a key role in this regulation is the modification of histones.  There is an ever-growing list of these modifications and the complexity of their action is only just beginning to be understood.  However, it is clear that histone modifications play fundamental roles in most biol. processes that are involved in the manipulation and expression of DNA.  Here, we describe the known histone modifications, define where they are found genomically and discuss some of their functional consequences, concg. mostly on transcription where the majority of characterization has taken place.  Cell Research (2011) 21:381-395. doi:10.1038/cr.2011.22; published online 15 Feb. 2011.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2ObWaVBHPorVg90H21EOLACvtfcHk0lhsOAwCx3xkRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXivVCgurc%253D&md5=bbdf55be25985810b90ba2efe6399f1b</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fcr.2011.22&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcr.2011.22%26sid%3Dliteratum%253Aachs%26aulast%3DBannister%26aufirst%3DA.%2BJ.%26aulast%3DKouzarides%26aufirst%3DT.%26atitle%3DRegulation%2520of%2520chromatin%2520by%2520histone%2520modifications%26jtitle%3DCell%2520Res.%26date%3D2011%26volume%3D21%26spage%3D381%26epage%3D395%26doi%3D10.1038%2Fcr.2011.22" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Barski, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cuddapah, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cui, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roh, T. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schones, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chepelev, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, K.</span><span> </span><span class="NLM_article-title">High-resolution profiling of histone methylations in the human genome</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">129</span><span class="NLM_x">, </span> <span class="NLM_fpage">823</span><span class="NLM_x">–</span> <span class="NLM_lpage">837</span><span class="refDoi"> DOI: 10.1016/j.cell.2007.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1016%2Fj.cell.2007.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=17512414" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmtFKjsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2007&pages=823-837&author=A.+Barskiauthor=S.+Cuddapahauthor=K.+Cuiauthor=T.+Y.+Rohauthor=D.+E.+Schonesauthor=Z.+Wangauthor=G.+Weiauthor=I.+Chepelevauthor=K.+Zhao&title=High-resolution+profiling+of+histone+methylations+in+the+human+genome&doi=10.1016%2Fj.cell.2007.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">High-resolution profiling of histone methylations in the human genome</span></div><div class="casAuthors">Barski, Artem; Cuddapah, Suresh; Cui, Kairong; Roh, Tae-Young; Schones, Dustin E.; Wang, Zhibin; Wei, Gang; Chepelev, Iouri; Zhao, Keji</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">129</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">823-837</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Histone modifications are implicated in influencing gene expression.  We have generated high-resoln. maps for the genome-wide distribution of 20 histone lysine and arginine methylations as well as histone variant H2A.Z, RNA polymerase II, and the insulator binding protein CTCF across the human genome using the Solexa 1G sequencing technol.  Typical patterns of histone methylations exhibited at promoters, insulators, enhancers, and transcribed regions are identified.  The monomethylations of H3K27, H3K9, H4K20, H3K79, and H2BK5 are all linked to gene activation, whereas trimethylations of H3K27, H3K9, and H3K79 are linked to repression.  H2A.Z assocs. with functional regulatory elements, and CTCF marks boundaries of histone methylation domains.  Chromosome banding patterns are correlated with unique patterns of histone modifications.  Chromosome breakpoints detected in T cell cancers frequently reside in chromatin regions assocd. with H3K4 methylations.  Our data provide new insights into the function of histone methylation and chromatin organization in genome function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxkecQmWJMZbVg90H21EOLACvtfcHk0lhsOAwCx3xkRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmtFKjsro%253D&md5=8e37e2d641f10e3cdd3ed51d1da82c96</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DBarski%26aufirst%3DA.%26aulast%3DCuddapah%26aufirst%3DS.%26aulast%3DCui%26aufirst%3DK.%26aulast%3DRoh%26aufirst%3DT.%2BY.%26aulast%3DSchones%26aufirst%3DD.%2BE.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DWei%26aufirst%3DG.%26aulast%3DChepelev%26aufirst%3DI.%26aulast%3DZhao%26aufirst%3DK.%26atitle%3DHigh-resolution%2520profiling%2520of%2520histone%2520methylations%2520in%2520the%2520human%2520genome%26jtitle%3DCell%26date%3D2007%26volume%3D129%26spage%3D823%26epage%3D837%26doi%3D10.1016%2Fj.cell.2007.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Zhang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pugh, B. F.</span><span> </span><span class="NLM_article-title">High-resolution genome-wide mapping of the primary structure of chromatin</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">144</span><span class="NLM_x">, </span> <span class="NLM_fpage">175</span><span class="NLM_x">–</span> <span class="NLM_lpage">186</span><span class="refDoi"> DOI: 10.1016/j.cell.2011.01.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1016%2Fj.cell.2011.01.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=21241889" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BC3MXntVagug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=144&publication_year=2011&pages=175-186&author=Z.+Zhangauthor=B.+F.+Pugh&title=High-resolution+genome-wide+mapping+of+the+primary+structure+of+chromatin&doi=10.1016%2Fj.cell.2011.01.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">High-Resolution Genome-wide Mapping of the Primary Structure of Chromatin</span></div><div class="casAuthors">Zhang, Zhenhai; Pugh, B. Franklin</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">144</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">175-186</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review primer.  The genomic organization of chromatin is increasingly recognized as a key regulator of cell behavior, but deciphering its regulation mechanisms requires detailed knowledge of chromatin's primary structure - the assembly of nucleosomes throughout the genome.  This Primer explains the principles for mapping and analyzing the primary organization of chromatin on a genomic scale.  After introducing chromatin organization and its impact on gene regulation and human health, the authors then describe methods that detect nucleosome positioning and occupancy levels using chromatin immunopptn. in combination with deep sequencing (ChIP-Seq), a strategy that is now straightforward and cost efficient.  The authors then explore current strategies for converting the sequence information into knowledge about chromatin, an exciting challenge for biologists and bioinformaticians.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaIXii3VQMsbVg90H21EOLACvtfcHk0lis4zOFQSr9xA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXntVagug%253D%253D&md5=8fc3faf35d565a736df953165d665588</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2011.01.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2011.01.003%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DPugh%26aufirst%3DB.%2BF.%26atitle%3DHigh-resolution%2520genome-wide%2520mapping%2520of%2520the%2520primary%2520structure%2520of%2520chromatin%26jtitle%3DCell%26date%3D2011%26volume%3D144%26spage%3D175%26epage%3D186%26doi%3D10.1016%2Fj.cell.2011.01.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Kubicek, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Sullivan, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">August, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hickey, E. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teodoro, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rea, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mechtler, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kowalski, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Homon, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kelly, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenuwein, T.</span><span> </span><span class="NLM_article-title">Reversal of H3K9me2 by a small-molecule inhibitor for the G9a histone methyltransferase</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">473</span><span class="NLM_x">–</span> <span class="NLM_lpage">481</span><span class="refDoi"> DOI: 10.1016/j.molcel.2007.01.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1016%2Fj.molcel.2007.01.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=17289593" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BD2sXitl2jurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2007&pages=473-481&author=S.+Kubicekauthor=R.+J.+O%E2%80%99Sullivanauthor=E.+M.+Augustauthor=E.+R.+Hickeyauthor=Q.+Zhangauthor=M.+L.+Teodoroauthor=S.+Reaauthor=K.+Mechtlerauthor=J.+A.+Kowalskiauthor=C.+A.+Homonauthor=T.+A.+Kellyauthor=T.+Jenuwein&title=Reversal+of+H3K9me2+by+a+small-molecule+inhibitor+for+the+G9a+histone+methyltransferase&doi=10.1016%2Fj.molcel.2007.01.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Reversal of H3K9me2 by a small-molecule inhibitor for the G9a histone methyltransferase</span></div><div class="casAuthors">Kubicek, Stefan; O'Sullivan, Roderick J.; August, E. Michael; Hickey, Eugene R.; Zhang, Qiang; Teodoro, Miguel L.; Rea, Stephen; Mechtler, Karl; Kowalski, Jennifer A.; Homon, Carol Ann; Kelly, Terence A.; Jenuwein, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">473-481</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Histone lysine methylation has important roles in the organization of chromatin domains and the regulation of gene expression.  To analyze its function and modulate its activity, we screened for specific inhibitors against histone lysine methyltransferases (HMTases) using recombinant G9a as the target enzyme.  From a chem. library comprising 125,000 preselected compds., seven hits were identified.  Of those, one inhibitor, BIX-01294 (diazepinquinazolin-amine deriv.), does not compete with the cofactor S-adenosyl-methionine, and selectively impairs the G9a HMTase and the generation of H3K9me2 in vitro.  In cellular assays, transient incubation of several cell lines with BIX-01294 lowers bulk H3K9me2 levels that are restored upon removal of the inhibitor.  Importantly, chromatin immunopptn. at several G9a target genes demonstrates reversible redn. of promoter-proximal H3K9me2 in inhibitor-treated mouse ES cells and fibroblasts.  Our data identify a biol. active HMTase inhibitor that allows for the transient modulation of H3K9me2 marks in mammalian chromatin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3SW8czS4WjbVg90H21EOLACvtfcHk0lis4zOFQSr9xA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXitl2jurs%253D&md5=d2f5290f1252e630a9b05489b0775c2c</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2007.01.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2007.01.017%26sid%3Dliteratum%253Aachs%26aulast%3DKubicek%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Sullivan%26aufirst%3DR.%2BJ.%26aulast%3DAugust%26aufirst%3DE.%2BM.%26aulast%3DHickey%26aufirst%3DE.%2BR.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DTeodoro%26aufirst%3DM.%2BL.%26aulast%3DRea%26aufirst%3DS.%26aulast%3DMechtler%26aufirst%3DK.%26aulast%3DKowalski%26aufirst%3DJ.%2BA.%26aulast%3DHomon%26aufirst%3DC.%2BA.%26aulast%3DKelly%26aufirst%3DT.%2BA.%26aulast%3DJenuwein%26aufirst%3DT.%26atitle%3DReversal%2520of%2520H3K9me2%2520by%2520a%2520small-molecule%2520inhibitor%2520for%2520the%2520G9a%2520histone%2520methyltransferase%26jtitle%3DMol.%2520Cell%26date%3D2007%26volume%3D25%26spage%3D473%26epage%3D481%26doi%3D10.1016%2Fj.molcel.2007.01.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Chang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horton, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Upadhyay, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spannhoff, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Snyder, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, X.</span><span> </span><span class="NLM_article-title">Structural basis for G9a-like protein lysine methyltransferase inhibition by BIX-01294</span> <span class="citation_source-journal">Nat. Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">312</span><span class="NLM_x">–</span> <span class="NLM_lpage">317</span><span class="refDoi"> DOI: 10.1038/nsmb.1560</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1038%2Fnsmb.1560" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=19219047" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhvFKjt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2009&pages=312-317&author=Y.+Changauthor=X.+Zhangauthor=J.+R.+Hortonauthor=A.+K.+Upadhyayauthor=A.+Spannhoffauthor=J.+Liuauthor=J.+P.+Snyderauthor=M.+T.+Bedfordauthor=X.+Cheng&title=Structural+basis+for+G9a-like+protein+lysine+methyltransferase+inhibition+by+BIX-01294&doi=10.1038%2Fnsmb.1560"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for G9a-like protein lysine methyltransferase inhibition by BIX-01294</span></div><div class="casAuthors">Chang, Yanqi; Zhang, Xing; Horton, John R.; Upadhyay, Anup K.; Spannhoff, Astrid; Liu, Jin; Snyder, James P.; Bedford, Mark T.; Cheng, Xiaodong</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">312-317</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Histone lysine methylation is an important epigenetic mark that regulates gene expression and chromatin organization.  G9a and G9a-like protein (GLP) are euchromatin-assocd. methyltransferases that repress transcription by methylating histone H3 Lys9.  BIX-01294 was originally identified as a G9a inhibitor during a chem. library screen of small mols. and has previously been used in the generation of induced pluripotent stem cells.  Here we present the crystal structure of the catalytic SET domain of GLP in complex with BIX-01294 and S-adenosyl-L-homocysteine.  The inhibitor is bound in the substrate peptide groove at the location where the histone H3 residues N-terminal to the target lysine lie in the previously solved structure of the complex with histone peptide.  The inhibitor resembles the bound conformation of histone H3 Lys4 to Arg8, and is positioned in place by residues specific for G9a and GLP through specific interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhQepqN3zr_bVg90H21EOLACvtfcHk0lis4zOFQSr9xA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhvFKjt7g%253D&md5=6a16baaf33420fc1caf1df70d103d270</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.1560&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.1560%26sid%3Dliteratum%253Aachs%26aulast%3DChang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DHorton%26aufirst%3DJ.%2BR.%26aulast%3DUpadhyay%26aufirst%3DA.%2BK.%26aulast%3DSpannhoff%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DSnyder%26aufirst%3DJ.%2BP.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DCheng%26aufirst%3DX.%26atitle%3DStructural%2520basis%2520for%2520G9a-like%2520protein%2520lysine%2520methyltransferase%2520inhibition%2520by%2520BIX-01294%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2009%26volume%3D16%26spage%3D312%26epage%3D317%26doi%3D10.1038%2Fnsmb.1560" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Tachibana, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sugimoto, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nozaki, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ueda, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohta, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohki, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fukuda, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takeda, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niida, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kato, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shinkai, Y.</span><span> </span><span class="NLM_article-title">G9a histone methyltransferase plays a dominant role in euchromatic histone H3 lysine 9 methylation and is essential for early embryogenesis</span> <span class="citation_source-journal">Genes Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">1779</span><span class="NLM_x">–</span> <span class="NLM_lpage">1791</span><span class="refDoi"> DOI: 10.1101/gad.989402</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1101%2Fgad.989402" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=12130538" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BD38Xls1Onurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2002&pages=1779-1791&author=M.+Tachibanaauthor=K.+Sugimotoauthor=M.+Nozakiauthor=J.+Uedaauthor=T.+Ohtaauthor=M.+Ohkiauthor=M.+Fukudaauthor=N.+Takedaauthor=H.+Niidaauthor=H.+Katoauthor=Y.+Shinkai&title=G9a+histone+methyltransferase+plays+a+dominant+role+in+euchromatic+histone+H3+lysine+9+methylation+and+is+essential+for+early+embryogenesis&doi=10.1101%2Fgad.989402"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">G9a histone methyltransferase plays a dominant role in euchromatic histone H3 lysine 9 methylation and is essential for early embryogenesis</span></div><div class="casAuthors">Tachibana, Makoto; Sugimoto, Kenji; Nozaki, Masami; Ueda, Jun; Ohta, Tsutomu; Ohki, Misao; Fukuda, Mikiko; Takeda, Naoki; Niida, Hiroyuki; Kato, Hiroyuki; Shinkai, Yoichi</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1779-1791</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Covalent modification of histone tails is crucial for transcriptional regulation, mitotic chromosomal condensation, and heterochromatin formation.  Histone H3 lysine 9 (H3-K9) methylation catalyzed by the Suv39h family proteins is essential for establishing the architecture of pericentric heterochromatin.  We recently identified a mammalian histone methyltransferase (HMTase), G9a, which has strong HMTase activity towards H3-K9 in vitro.  To investigate the in vivo functions of G9a, we generated G9a-deficient mice and embryonic stem (ES) cells.  We found that H3-K9 methylation was drastically decreased in G9a-deficient embryos, which displayed severe growth retardation and early lethality.  G9a-deficient ES cells also exhibited reduced H3-K9 methylation compared to wild-type cells, indicating that G9a is a dominant H3-K9 HMTase in vivo.  Importantly, the loss of G9a abolished methylated H3-K9 mostly in euchromatic regions.  Finally, G9a exerted a transcriptionally suppressive function that depended on its HMTase activity.  Our results indicate that euchromatic H3-K9 methylation regulated by G9a is essential for early embryogenesis and is involved in the transcriptional repression of developmental genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5EeLfBMH1_LVg90H21EOLACvtfcHk0li-ZHj7_k518g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xls1Onurg%253D&md5=bde55b2c2d49fd61e5f0337d780a94db</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1101%2Fgad.989402&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.989402%26sid%3Dliteratum%253Aachs%26aulast%3DTachibana%26aufirst%3DM.%26aulast%3DSugimoto%26aufirst%3DK.%26aulast%3DNozaki%26aufirst%3DM.%26aulast%3DUeda%26aufirst%3DJ.%26aulast%3DOhta%26aufirst%3DT.%26aulast%3DOhki%26aufirst%3DM.%26aulast%3DFukuda%26aufirst%3DM.%26aulast%3DTakeda%26aufirst%3DN.%26aulast%3DNiida%26aufirst%3DH.%26aulast%3DKato%26aufirst%3DH.%26aulast%3DShinkai%26aufirst%3DY.%26atitle%3DG9a%2520histone%2520methyltransferase%2520plays%2520a%2520dominant%2520role%2520in%2520euchromatic%2520histone%2520H3%2520lysine%25209%2520methylation%2520and%2520is%2520essential%2520for%2520early%2520embryogenesis%26jtitle%3DGenes%2520Dev.%26date%3D2002%26volume%3D16%26spage%3D1779%26epage%3D1791%26doi%3D10.1101%2Fgad.989402" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Tachibana, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ueda, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fukuda, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takeda, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohta, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iwanari, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sakihama, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kodama, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamakubo, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shinkai, Y.</span><span> </span><span class="NLM_article-title">Histone methyltransferases G9a and GLP form heteromeric complexes and are both crucial for methylation of euchromatin at H3-K9</span> <span class="citation_source-journal">Genes Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">815</span><span class="NLM_x">–</span> <span class="NLM_lpage">826</span><span class="refDoi"> DOI: 10.1101/gad.1284005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1101%2Fgad.1284005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=15774718" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjt1OjtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2005&pages=815-826&author=M.+Tachibanaauthor=J.+Uedaauthor=M.+Fukudaauthor=N.+Takedaauthor=T.+Ohtaauthor=H.+Iwanariauthor=T.+Sakihamaauthor=T.+Kodamaauthor=T.+Hamakuboauthor=Y.+Shinkai&title=Histone+methyltransferases+G9a+and+GLP+form+heteromeric+complexes+and+are+both+crucial+for+methylation+of+euchromatin+at+H3-K9&doi=10.1101%2Fgad.1284005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Histone methyltransferases G9a and GLP form heteromeric complexes and are both crucial for methylation of euchromatin at H3-K9</span></div><div class="casAuthors">Tachibana, Makoto; Ueda, Jun; Fukuda, Mikiko; Takeda, Naoki; Ohta, Tsutomu; Iwanari, Hiroko; Sakihama, Toshiko; Kodama, Tatsuhiko; Hamakubo, Takao; Shinkai, Yoichi</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">815-826</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Histone H3 Lys 9 (H3-K9) methylation is a crucial epigenetic mark for transcriptional silencing.  G9a is the major mammalian H3-K9 methyltransferase that targets euchromatic regions and is essential for murine embryogenesis.  There is a single G9a-related methyltransferase in mammals, called GLP/Eu-HMTase1.  Here the authors show that GLP is also important for H3-K9 methylation of mouse euchromatin.  GLP-deficiency led to embryonic lethality, a severe redn. of H3-K9 mono- and dimethylation, the induction of Mage-a gene expression, and HP1 relocalization in embryonic stem cells, all of which were phenotypes of G9a-deficiency.  Furthermore, the authors show that G9a and GLP formed a stoichiometric heteromeric complex in a wide variety of cell types.  Biochem. analyses revealed that formation of the G9a/GLP complex was dependent on their enzymic SET domains.  Taken together, these new findings revealed that G9a and GLP cooperatively exert H3-K9 methyltransferase function in vivo, likely through the formation of higher-order heteromeric complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUXBASXATfh7Vg90H21EOLACvtfcHk0li-ZHj7_k518g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjt1OjtLc%253D&md5=cd0b7f8b9f62b4abe81b5ea590ab218c</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1101%2Fgad.1284005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.1284005%26sid%3Dliteratum%253Aachs%26aulast%3DTachibana%26aufirst%3DM.%26aulast%3DUeda%26aufirst%3DJ.%26aulast%3DFukuda%26aufirst%3DM.%26aulast%3DTakeda%26aufirst%3DN.%26aulast%3DOhta%26aufirst%3DT.%26aulast%3DIwanari%26aufirst%3DH.%26aulast%3DSakihama%26aufirst%3DT.%26aulast%3DKodama%26aufirst%3DT.%26aulast%3DHamakubo%26aufirst%3DT.%26aulast%3DShinkai%26aufirst%3DY.%26atitle%3DHistone%2520methyltransferases%2520G9a%2520and%2520GLP%2520form%2520heteromeric%2520complexes%2520and%2520are%2520both%2520crucial%2520for%2520methylation%2520of%2520euchromatin%2520at%2520H3-K9%26jtitle%3DGenes%2520Dev.%26date%3D2005%26volume%3D19%26spage%3D815%26epage%3D826%26doi%3D10.1101%2Fgad.1284005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Trojer, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yonezawa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmidt, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenuwein, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinberg, D.</span><span> </span><span class="NLM_article-title">Dynamic histone H1 isotype 4 methylation and demethylation by histone lysine methyltransferase G9a/KMT1C and the jumonji domain-containing JMJD2/KDM4 proteins</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">284</span><span class="NLM_x">, </span> <span class="NLM_fpage">8395</span><span class="NLM_x">–</span> <span class="NLM_lpage">8405</span><span class="refDoi"> DOI: 10.1074/jbc.M807818200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1074%2Fjbc.M807818200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=8395-8405&author=P.+Trojerauthor=J.+Zhangauthor=M.+Yonezawaauthor=A.+Schmidtauthor=H.+Zhengauthor=T.+Jenuweinauthor=D.+Reinberg&title=Dynamic+histone+H1+isotype+4+methylation+and+demethylation+by+histone+lysine+methyltransferase+G9a%2FKMT1C+and+the+jumonji+domain-containing+JMJD2%2FKDM4+proteins&doi=10.1074%2Fjbc.M807818200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M807818200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M807818200%26sid%3Dliteratum%253Aachs%26aulast%3DTrojer%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DYonezawa%26aufirst%3DM.%26aulast%3DSchmidt%26aufirst%3DA.%26aulast%3DZheng%26aufirst%3DH.%26aulast%3DJenuwein%26aufirst%3DT.%26aulast%3DReinberg%26aufirst%3DD.%26atitle%3DDynamic%2520histone%2520H1%2520isotype%25204%2520methylation%2520and%2520demethylation%2520by%2520histone%2520lysine%2520methyltransferase%2520G9a%252FKMT1C%2520and%2520the%2520jumonji%2520domain-containing%2520JMJD2%252FKDM4%2520proteins%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2009%26volume%3D284%26spage%3D8395%26epage%3D8405%26doi%3D10.1074%2Fjbc.M807818200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Weiss, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hergeth, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeissler, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Izzo, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tropberger, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zee, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dundr, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daujat, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneider, R.</span><span> </span><span class="NLM_article-title">Histone H1 variant-specific lysine methylation by G9a/KMT1C and Glp1/KMT1D</span> <span class="citation_source-journal">Epigenet. Chromatin</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">7</span><span class="refDoi"> DOI: 10.1186/1756-8935-3-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1186%2F1756-8935-3-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=20334638" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A280%3ADC%252BC3c3pt1Clsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2010&pages=7&author=T.+Weissauthor=S.+Hergethauthor=U.+Zeisslerauthor=A.+Izzoauthor=P.+Tropbergerauthor=B.+M.+Zeeauthor=M.+Dundrauthor=B.+A.+Garciaauthor=S.+Daujatauthor=R.+Schneider&title=Histone+H1+variant-specific+lysine+methylation+by+G9a%2FKMT1C+and+Glp1%2FKMT1D&doi=10.1186%2F1756-8935-3-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Histone H1 variant-specific lysine methylation by G9a/KMT1C and Glp1/KMT1D</span></div><div class="casAuthors">Weiss Thomas; Hergeth Sonja; Zeissler Ulrike; Izzo Annalisa; Tropberger Philipp; Zee Barry M; Dundr Miroslav; Garcia Benjamin A; Daujat Sylvain; Schneider Robert</div><div class="citationInfo"><span class="NLM_cas:title">Epigenetics & chromatin</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">7</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  The linker histone H1 has a key role in establishing and maintaining higher order chromatin structure and in regulating gene expression.  Mammals express up to 11 different H1 variants, with H1.2 and H1.4 being the predominant ones in most somatic cells.  Like core histones, H1 has high levels of covalent modifications; however, the full set of modifications and their biological role are largely unknown.  RESULTS:  In this study, we used a candidate screen to identify enzymes that methylate H1 and to map their corresponding methylation sites.  We found that the histone lysine methyltransferases G9a/KMT1C and Glp1/KMT1D methylate H1.2 in vitro and in vivo, and we mapped this novel site to lysine 187 (H1.2K187) in the C-terminus of H1.  This H1.2K187 methylation is variant-specific.  The main target for methylation by G9a in H1.2, H1.3, H1.5 and H1.0 is in the C-terminus, whereas H1.4 is preferentially methylated at K26 (H1.4K26me) in the N-terminus.  We found that the readout of these marks is different; H1.4K26me can recruit HP1, but H1.2K187me cannot.  Likewise, JMJD2D/KDM4 only reverses H1.4K26 methylation, clearly distinguishing these two methylation sites.  Further, in contrast to C-terminal H1 phosphorylation, H1.2K187 methylation level is steady throughout the cell cycle.  CONCLUSIONS:  We have characterised a novel methylation site in the C-terminus of H1 that is the target of G9a/Glp1 both in vitro and in vivo.  To our knowledge, this is the first demonstration of variant-specific histone methylation by the same methyltransferases, but with differing downstream readers, thereby supporting the hypothesis of H1 variants having specific functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRiHGsW3wTCe1HFYivNCdPRfW6udTcc2ebEq9sF9mxtR7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3c3pt1Clsg%253D%253D&md5=a140e14a1dfff30b74e9fbef28091475</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1186%2F1756-8935-3-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1756-8935-3-7%26sid%3Dliteratum%253Aachs%26aulast%3DWeiss%26aufirst%3DT.%26aulast%3DHergeth%26aufirst%3DS.%26aulast%3DZeissler%26aufirst%3DU.%26aulast%3DIzzo%26aufirst%3DA.%26aulast%3DTropberger%26aufirst%3DP.%26aulast%3DZee%26aufirst%3DB.%2BM.%26aulast%3DDundr%26aufirst%3DM.%26aulast%3DGarcia%26aufirst%3DB.%2BA.%26aulast%3DDaujat%26aufirst%3DS.%26aulast%3DSchneider%26aufirst%3DR.%26atitle%3DHistone%2520H1%2520variant-specific%2520lysine%2520methylation%2520by%2520G9a%252FKMT1C%2520and%2520Glp1%252FKMT1D%26jtitle%3DEpigenet.%2520Chromatin%26date%3D2010%26volume%3D3%26spage%3D7%26doi%3D10.1186%2F1756-8935-3-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Patnaik, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chin, H. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Esteve, P. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benner, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobsen, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pradhan, S.</span><span> </span><span class="NLM_article-title">Substrate specificity and kinetic mechanism of mammalian G9a histone H3 methyltransferase</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">279</span><span class="NLM_x">, </span> <span class="NLM_fpage">53248</span><span class="NLM_x">–</span> <span class="NLM_lpage">53258</span><span class="refDoi"> DOI: 10.1074/jbc.M409604200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1074%2Fjbc.M409604200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=279&publication_year=2004&pages=53248-53258&author=D.+Patnaikauthor=H.+G.+Chinauthor=P.+O.+Esteveauthor=J.+Bennerauthor=S.+E.+Jacobsenauthor=S.+Pradhan&title=Substrate+specificity+and+kinetic+mechanism+of+mammalian+G9a+histone+H3+methyltransferase&doi=10.1074%2Fjbc.M409604200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M409604200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M409604200%26sid%3Dliteratum%253Aachs%26aulast%3DPatnaik%26aufirst%3DD.%26aulast%3DChin%26aufirst%3DH.%2BG.%26aulast%3DEsteve%26aufirst%3DP.%2BO.%26aulast%3DBenner%26aufirst%3DJ.%26aulast%3DJacobsen%26aufirst%3DS.%2BE.%26aulast%3DPradhan%26aufirst%3DS.%26atitle%3DSubstrate%2520specificity%2520and%2520kinetic%2520mechanism%2520of%2520mammalian%2520G9a%2520histone%2520H3%2520methyltransferase%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2004%26volume%3D279%26spage%3D53248%26epage%3D53258%26doi%3D10.1074%2Fjbc.M409604200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Tachibana, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sugimoto, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fukushima, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shinkai, Y.</span><span> </span><span class="NLM_article-title">SET domain-containing protein, G9a, is a novel lysine-preferring mammalian histone methyltransferase with hyperactivity and specific selectivity to lysines 9 and 27 of histone H3</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">276</span><span class="NLM_x">, </span> <span class="NLM_fpage">25309</span><span class="NLM_x">–</span> <span class="NLM_lpage">25317</span><span class="refDoi"> DOI: 10.1074/jbc.M101914200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1074%2Fjbc.M101914200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=2001&pages=25309-25317&author=M.+Tachibanaauthor=K.+Sugimotoauthor=T.+Fukushimaauthor=Y.+Shinkai&title=SET+domain-containing+protein%2C+G9a%2C+is+a+novel+lysine-preferring+mammalian+histone+methyltransferase+with+hyperactivity+and+specific+selectivity+to+lysines+9+and+27+of+histone+H3&doi=10.1074%2Fjbc.M101914200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M101914200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M101914200%26sid%3Dliteratum%253Aachs%26aulast%3DTachibana%26aufirst%3DM.%26aulast%3DSugimoto%26aufirst%3DK.%26aulast%3DFukushima%26aufirst%3DT.%26aulast%3DShinkai%26aufirst%3DY.%26atitle%3DSET%2520domain-containing%2520protein%252C%2520G9a%252C%2520is%2520a%2520novel%2520lysine-preferring%2520mammalian%2520histone%2520methyltransferase%2520with%2520hyperactivity%2520and%2520specific%2520selectivity%2520to%2520lysines%25209%2520and%252027%2520of%2520histone%2520H3%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2001%26volume%3D276%26spage%3D25309%26epage%3D25317%26doi%3D10.1074%2Fjbc.M101914200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiong, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, X. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, B.</span><span> </span><span class="NLM_article-title">Histone methyltransferase G9a contributes to H3K27 methylation in vivo</span> <span class="citation_source-journal">Cell Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">365</span><span class="NLM_x">–</span> <span class="NLM_lpage">367</span><span class="refDoi"> DOI: 10.1038/cr.2010.157</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1038%2Fcr.2010.157" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=21079650" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVOmurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=365-367&author=H.+Wuauthor=X.+Z.+Chenauthor=J.+Xiongauthor=Y.+F.+Liauthor=H.+Liauthor=X.+J.+Dingauthor=S.+Liuauthor=S.+Chenauthor=S.+R.+Gaoauthor=B.+Zhu&title=Histone+methyltransferase+G9a+contributes+to+H3K27+methylation+in+vivo&doi=10.1038%2Fcr.2010.157"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Histone methyltransferase G9a contributes to H3K27 methylation in vivo</span></div><div class="casAuthors">Wu, Hui; Chen, Xiuzhen; Xiong, Jun; Li, Yingfeng; Li, Hong; Ding, Xiaojun; Liu, Sheng; Chen, She; Gao, Shaorong; Zhu, Bing</div><div class="citationInfo"><span class="NLM_cas:title">Cell Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">365-367</span>CODEN:
                <span class="NLM_cas:coden">CREEB6</span>;
        ISSN:<span class="NLM_cas:issn">1001-0602</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The role of G9a histone methyltransferase in H3K27 methylation was studied.  His-tagged H3 histones contg. K9C mutation were converted into His-H3Kc9me2 with the MLA reaction to mimic homogeneously di-methylated H3K9.  Recombinant G9a displayed robust activity on nucleosomes assembled with H3Kc9me2, which suggests that either G9a may be a trimethylase or may react with other H3 lysine residues on a nucleosomal substrate.  Thus, the K27A mutation was introduced into the His-tagged pre-methylated H3, which nearly completely abolished G9a's activity, indicating that G9a is mainly a dimethylase and also reacts with H3K27.  Current results collectively suggest G9a's contribution to H3K27 methylation in addn. to its well-characterized role in H3K9 methylation in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8nL82DjJqz7Vg90H21EOLACvtfcHk0liYGVpTrE45Zg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVOmurc%253D&md5=c1796390312963ef66e77a749f508e9b</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fcr.2010.157&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcr.2010.157%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DX.%2BZ.%26aulast%3DXiong%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DY.%2BF.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DDing%26aufirst%3DX.%2BJ.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DGao%26aufirst%3DS.%2BR.%26aulast%3DZhu%26aufirst%3DB.%26atitle%3DHistone%2520methyltransferase%2520G9a%2520contributes%2520to%2520H3K27%2520methylation%2520in%2520vivo%26jtitle%3DCell%2520Res.%26date%3D2011%26volume%3D21%26spage%3D365%26epage%3D367%26doi%3D10.1038%2Fcr.2010.157" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Yu, Y. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, C. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Q. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DiMaggio, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">York, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carey, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grunstein, M.</span><span> </span><span class="NLM_article-title">Histone H3 lysine 56 methylation regulates DNA replication through its interaction with PCNA</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">7</span><span class="NLM_x">–</span> <span class="NLM_lpage">17</span><span class="refDoi"> DOI: 10.1016/j.molcel.2012.01.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1016%2Fj.molcel.2012.01.019" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2012&pages=7-17&author=Y.+X.+Yuauthor=C.+Y.+Songauthor=Q.+Y.+Zhangauthor=P.+A.+DiMaggioauthor=B.+A.+Garciaauthor=A.+Yorkauthor=M.+F.+Careyauthor=M.+Grunstein&title=Histone+H3+lysine+56+methylation+regulates+DNA+replication+through+its+interaction+with+PCNA&doi=10.1016%2Fj.molcel.2012.01.019"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2012.01.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2012.01.019%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DY.%2BX.%26aulast%3DSong%26aufirst%3DC.%2BY.%26aulast%3DZhang%26aufirst%3DQ.%2BY.%26aulast%3DDiMaggio%26aufirst%3DP.%2BA.%26aulast%3DGarcia%26aufirst%3DB.%2BA.%26aulast%3DYork%26aufirst%3DA.%26aulast%3DCarey%26aufirst%3DM.%2BF.%26aulast%3DGrunstein%26aufirst%3DM.%26atitle%3DHistone%2520H3%2520lysine%252056%2520methylation%2520regulates%2520DNA%2520replication%2520through%2520its%2520interaction%2520with%2520PCNA%26jtitle%3DMol.%2520Cell%26date%3D2012%26volume%3D46%26spage%3D7%26epage%3D17%26doi%3D10.1016%2Fj.molcel.2012.01.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Biggar, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, S. S. C.</span><span> </span><span class="NLM_article-title">Non-histone protein methylation as a regulator of cellular signalling and function</span> <span class="citation_source-journal">Nat. Rev. Mol. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">5</span><span class="NLM_x">–</span> <span class="NLM_lpage">17</span><span class="refDoi"> DOI: 10.1038/nrm3915</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1038%2Fnrm3915" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=25491103" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitV2nsrzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2015&pages=5-17&author=K.+K.+Biggarauthor=S.+S.+C.+Li&title=Non-histone+protein+methylation+as+a+regulator+of+cellular+signalling+and+function&doi=10.1038%2Fnrm3915"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Non-histone protein methylation as a regulator of cellular signalling and function</span></div><div class="casAuthors">Biggar, Kyle K.; Li, Shawn S.-C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5-17</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Methylation of Lys and Arg residues on non-histone proteins has emerged as a prevalent post-translational modification and as an important regulator of cellular signal transduction mediated by the MAPK, WNT, BMP, Hippo and JAK-STAT signalling pathways.  Crosstalk between methylation and other types of post-translational modifications, and between histone and non-histone protein methylation frequently occurs and affects cellular functions such as chromatin remodelling, gene transcription, protein synthesis, signal transduction and DNA repair.  With recent advances in proteomic techniques, in particular mass spectrometry, the stage is now set to decode the methylproteome and define its functions in health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5E-jYGUpGCbVg90H21EOLACvtfcHk0liYGVpTrE45Zg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitV2nsrzM&md5=3f60179eadac0ad72295b516003ec048</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fnrm3915&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm3915%26sid%3Dliteratum%253Aachs%26aulast%3DBiggar%26aufirst%3DK.%2BK.%26aulast%3DLi%26aufirst%3DS.%2BS.%2BC.%26atitle%3DNon-histone%2520protein%2520methylation%2520as%2520a%2520regulator%2520of%2520cellular%2520signalling%2520and%2520function%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2015%26volume%3D16%26spage%3D5%26epage%3D17%26doi%3D10.1038%2Fnrm3915" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Huang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dorsey, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chuikov, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perez-Burgos, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenuwein, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinberg, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berger, S. L.</span><span> </span><span class="NLM_article-title">G9a and Glp methylate lysine 373 in the tumor suppressor p53</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">285</span><span class="NLM_x">, </span> <span class="NLM_fpage">18122</span><span class="NLM_x">–</span> <span class="NLM_lpage">18122</span><span class="refDoi"> DOI: 10.1074/jbc.A109.062588</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1074%2Fjbc.A109.062588" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=18122-18122&author=J.+Huangauthor=J.+Dorseyauthor=S.+Chuikovauthor=L.+Perez-Burgosauthor=X.+Y.+Zhangauthor=T.+Jenuweinauthor=D.+Reinbergauthor=S.+L.+Berger&title=G9a+and+Glp+methylate+lysine+373+in+the+tumor+suppressor+p53&doi=10.1074%2Fjbc.A109.062588"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1074%2Fjbc.A109.062588&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.A109.062588%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DDorsey%26aufirst%3DJ.%26aulast%3DChuikov%26aufirst%3DS.%26aulast%3DPerez-Burgos%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DX.%2BY.%26aulast%3DJenuwein%26aufirst%3DT.%26aulast%3DReinberg%26aufirst%3DD.%26aulast%3DBerger%26aufirst%3DS.%2BL.%26atitle%3DG9a%2520and%2520Glp%2520methylate%2520lysine%2520373%2520in%2520the%2520tumor%2520suppressor%2520p53%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D18122%26epage%3D18122%26doi%3D10.1074%2Fjbc.A109.062588" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Moore, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlson, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Camp, N. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheung, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">James, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chua, K. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolf-Yadlin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gozani, O.</span><span> </span><span class="NLM_article-title">A general molecular affinity strategy for global detection and proteomic analysis of lysine methylation</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">444</span><span class="NLM_x">–</span> <span class="NLM_lpage">456</span><span class="refDoi"> DOI: 10.1016/j.molcel.2013.03.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1016%2Fj.molcel.2013.03.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=23583077" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlvVCgsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2013&pages=444-456&author=K.+E.+Mooreauthor=S.+M.+Carlsonauthor=N.+D.+Campauthor=P.+Cheungauthor=R.+G.+Jamesauthor=K.+F.+Chuaauthor=A.+Wolf-Yadlinauthor=O.+Gozani&title=A+general+molecular+affinity+strategy+for+global+detection+and+proteomic+analysis+of+lysine+methylation&doi=10.1016%2Fj.molcel.2013.03.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">A General Molecular Affinity Strategy for Global Detection and Proteomic Analysis of Lysine Methylation</span></div><div class="casAuthors">Moore, Kaitlyn E.; Carlson, Scott M.; Camp, Nathan D.; Cheung, Peggie; James, Richard G.; Chua, Katrin F.; Wolf-Yadlin, Alejandro; Gozani, Or</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">444-456</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Lysine methylation of histone proteins regulates chromatin dynamics and plays important roles in diverse physiol. and pathol. processes.  However, beyond histone proteins, the proteome-wide extent of lysine methylation remains largely unknown.  We have engineered the naturally occurring MBT domain repeats of L3MBTL1 to serve as a universal affinity reagent for detecting, enriching, and identifying proteins carrying a mono- or dimethylated lysine.  The domain is broadly specific for methylated lysine (pan-specific) and can be applied to any biol. system.  We have used our approach to demonstrate that SIRT1 is a substrate of the methyltransferase G9a both in vitro and in cells, to perform proteome-wide detection and enrichment of methylated proteins, and to identify candidate in-cell substrates of G9a and the related methyltransferase GLP.  Together, our results demonstrate a powerful new approach for global and quant. anal. of methylated lysine, and they represent the first systems biol. understanding of lysine methylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr245nYBEsNyLVg90H21EOLACvtfcHk0liEwiBMPY9BZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlvVCgsr4%253D&md5=a826252b90d161331eefcb18a7340574</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2013.03.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2013.03.005%26sid%3Dliteratum%253Aachs%26aulast%3DMoore%26aufirst%3DK.%2BE.%26aulast%3DCarlson%26aufirst%3DS.%2BM.%26aulast%3DCamp%26aufirst%3DN.%2BD.%26aulast%3DCheung%26aufirst%3DP.%26aulast%3DJames%26aufirst%3DR.%2BG.%26aulast%3DChua%26aufirst%3DK.%2BF.%26aulast%3DWolf-Yadlin%26aufirst%3DA.%26aulast%3DGozani%26aufirst%3DO.%26atitle%3DA%2520general%2520molecular%2520affinity%2520strategy%2520for%2520global%2520detection%2520and%2520proteomic%2520analysis%2520of%2520lysine%2520methylation%26jtitle%3DMol.%2520Cell%26date%3D2013%26volume%3D50%26spage%3D444%26epage%3D456%26doi%3D10.1016%2Fj.molcel.2013.03.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Lee, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, I. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choi, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shin, H. J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, J. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seo, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Binda, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gozani, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Semenza, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, K. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hwang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baek, S. H.</span><span> </span><span class="NLM_article-title">Negative regulation of hypoxic responses via induced reptin methylation</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x">, </span> <span class="NLM_fpage">71</span><span class="NLM_x">–</span> <span class="NLM_lpage">85</span><span class="refDoi"> DOI: 10.1016/j.molcel.2010.06.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1016%2Fj.molcel.2010.06.008" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2010&pages=71-85&author=J.+S.+Leeauthor=Y.+Kimauthor=I.+S.+Kimauthor=B.+Kimauthor=H.+J.+Choiauthor=J.+M.+Leeauthor=H.+J.+R.+Shinauthor=J.+H.+Kimauthor=J.+Y.+Kimauthor=S.+B.+Seoauthor=H.+Leeauthor=O.+Bindaauthor=O.+Gozaniauthor=G.+L.+Semenzaauthor=M.+Kimauthor=K.+I.+Kimauthor=D.+Hwangauthor=S.+H.+Baek&title=Negative+regulation+of+hypoxic+responses+via+induced+reptin+methylation&doi=10.1016%2Fj.molcel.2010.06.008"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2010.06.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2010.06.008%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DJ.%2BS.%26aulast%3DKim%26aufirst%3DY.%26aulast%3DKim%26aufirst%3DI.%2BS.%26aulast%3DKim%26aufirst%3DB.%26aulast%3DChoi%26aufirst%3DH.%2BJ.%26aulast%3DLee%26aufirst%3DJ.%2BM.%26aulast%3DShin%26aufirst%3DH.%2BJ.%2BR.%26aulast%3DKim%26aufirst%3DJ.%2BH.%26aulast%3DKim%26aufirst%3DJ.%2BY.%26aulast%3DSeo%26aufirst%3DS.%2BB.%26aulast%3DLee%26aufirst%3DH.%26aulast%3DBinda%26aufirst%3DO.%26aulast%3DGozani%26aufirst%3DO.%26aulast%3DSemenza%26aufirst%3DG.%2BL.%26aulast%3DKim%26aufirst%3DM.%26aulast%3DKim%26aufirst%3DK.%2BI.%26aulast%3DHwang%26aufirst%3DD.%26aulast%3DBaek%26aufirst%3DS.%2BH.%26atitle%3DNegative%2520regulation%2520of%2520hypoxic%2520responses%2520via%2520induced%2520reptin%2520methylation%26jtitle%3DMol.%2520Cell%26date%3D2010%26volume%3D39%26spage%3D71%26epage%3D85%26doi%3D10.1016%2Fj.molcel.2010.06.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Ling, B. M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bharathy, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, T. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kok, W. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tan, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rao, V. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gopinadhan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sartorelli, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walsh, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taneja, R.</span><span> </span><span class="NLM_article-title">Lysine methyltransferase G9a methylates the transcription factor MyoD and regulates skeletal muscle differentiation</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">109</span><span class="NLM_x">, </span> <span class="NLM_fpage">841</span><span class="NLM_x">–</span> <span class="NLM_lpage">846</span><span class="refDoi"> DOI: 10.1073/pnas.1111628109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1073%2Fpnas.1111628109" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=841-846&author=B.+M.+T.+Lingauthor=N.+Bharathyauthor=T.+K.+Chungauthor=W.+K.+Kokauthor=S.+Liauthor=Y.+H.+Tanauthor=V.+K.+Raoauthor=S.+Gopinadhanauthor=V.+Sartorelliauthor=M.+J.+Walshauthor=R.+Taneja&title=Lysine+methyltransferase+G9a+methylates+the+transcription+factor+MyoD+and+regulates+skeletal+muscle+differentiation&doi=10.1073%2Fpnas.1111628109"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1111628109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1111628109%26sid%3Dliteratum%253Aachs%26aulast%3DLing%26aufirst%3DB.%2BM.%2BT.%26aulast%3DBharathy%26aufirst%3DN.%26aulast%3DChung%26aufirst%3DT.%2BK.%26aulast%3DKok%26aufirst%3DW.%2BK.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DTan%26aufirst%3DY.%2BH.%26aulast%3DRao%26aufirst%3DV.%2BK.%26aulast%3DGopinadhan%26aufirst%3DS.%26aulast%3DSartorelli%26aufirst%3DV.%26aulast%3DWalsh%26aufirst%3DM.%2BJ.%26aulast%3DTaneja%26aufirst%3DR.%26atitle%3DLysine%2520methyltransferase%2520G9a%2520methylates%2520the%2520transcription%2520factor%2520MyoD%2520and%2520regulates%2520skeletal%2520muscle%2520differentiation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26spage%3D841%26epage%3D846%26doi%3D10.1073%2Fpnas.1111628109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Rathert, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhayalan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murakami, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tamas, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jurkowska, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Komatsu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shinkai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, X. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeltsch, A.</span><span> </span><span class="NLM_article-title">Protein lysine methyltransferase G9a acts on non-histone targets</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">344</span><span class="NLM_x">–</span> <span class="NLM_lpage">346</span><span class="refDoi"> DOI: 10.1038/nchembio.88</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1038%2Fnchembio.88" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=18438403" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlvFOisrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2008&pages=344-346&author=P.+Rathertauthor=A.+Dhayalanauthor=M.+Murakamiauthor=X.+Zhangauthor=R.+Tamasauthor=R.+Jurkowskaauthor=Y.+Komatsuauthor=Y.+Shinkaiauthor=X.+D.+Chengauthor=A.+Jeltsch&title=Protein+lysine+methyltransferase+G9a+acts+on+non-histone+targets&doi=10.1038%2Fnchembio.88"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Protein lysine methyltransferase G9a acts on non-histone targets</span></div><div class="casAuthors">Rathert, Philipp; Dhayalan, Arunkumar; Murakami, Marie; Zhang, Xing; Tamas, Raluca; Jurkowska, Renata; Komatsu, Yasuhiko; Shinkai, Yoichi; Cheng, Xiaodong; Jeltsch, Albert</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">344-346</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">By methylation of peptide arrays, we detd. the specificity profile of the protein methyltransferase G9a.  We show that it mostly recognizes an Arg-Lys sequence and that its activity is inhibited by methylation of the arginine residue.  Using the specificity profile, we identified new non-histone protein targets of G9a, including CDYL1, WIZ, ACINUS and G9a (automethylation), as well as peptides derived from CSB.  We demonstrate potential downstream signaling pathways for methylation of non-histone proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo89lj_ybb7_7Vg90H21EOLACvtfcHk0libhu9km2YuiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlvFOisrs%253D&md5=acebc9dbbe888944e1d9775b78e2483b</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.88&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.88%26sid%3Dliteratum%253Aachs%26aulast%3DRathert%26aufirst%3DP.%26aulast%3DDhayalan%26aufirst%3DA.%26aulast%3DMurakami%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DTamas%26aufirst%3DR.%26aulast%3DJurkowska%26aufirst%3DR.%26aulast%3DKomatsu%26aufirst%3DY.%26aulast%3DShinkai%26aufirst%3DY.%26aulast%3DCheng%26aufirst%3DX.%2BD.%26aulast%3DJeltsch%26aufirst%3DA.%26atitle%3DProtein%2520lysine%2520methyltransferase%2520G9a%2520acts%2520on%2520non-histone%2520targets%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2008%26volume%3D4%26spage%3D344%26epage%3D346%26doi%3D10.1038%2Fnchembio.88" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Sampath, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marazzi, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yap, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sampath, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krutchinsky, A. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mecklenbrauker, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Viale, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rudensky, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chait, B. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tarakhovsky, A.</span><span> </span><span class="NLM_article-title">Methylation of a histone mimic within the histone methyltransferase G9a regulates protein complex assembly</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">596</span><span class="NLM_x">–</span> <span class="NLM_lpage">608</span><span class="refDoi"> DOI: 10.1016/j.molcel.2007.06.026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1016%2Fj.molcel.2007.06.026" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=17707231" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVSmtL3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2007&pages=596-608&author=S.+C.+Sampathauthor=I.+Marazziauthor=K.+L.+Yapauthor=S.+C.+Sampathauthor=A.+N.+Krutchinskyauthor=I.+Mecklenbraukerauthor=A.+Vialeauthor=E.+Rudenskyauthor=M.+M.+Zhouauthor=B.+T.+Chaitauthor=A.+Tarakhovsky&title=Methylation+of+a+histone+mimic+within+the+histone+methyltransferase+G9a+regulates+protein+complex+assembly&doi=10.1016%2Fj.molcel.2007.06.026"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Methylation of a histone mimic within the histone methyltransferase G9a regulates protein complex assembly</span></div><div class="casAuthors">Sampath, Srihari C.; Marazzi, Ivan; Yap, Kyoko L.; Sampath, Srinath C.; Krutchinsky, Andrew N.; Mecklenbrauker, Ingrid; Viale, Agnes; Rudensky, Eugene; Zhou, Ming-Ming; Chait, Brian T.; Tarakhovsky, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">596-608</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Epigenetic gene silencing in eukaryotes is regulated in part by lysine methylation of the core histone proteins.  While histone lysine methylation is known to control gene expression through the recruitment of modification-specific effector proteins, it remains unknown whether nonhistone chromatin proteins are targets for similar modification-recognition systems.  Here we show that the histone H3 methyltransferase G9a contains a conserved methylation motif with marked sequence similarity to H3 itself.  As with methylation of H3 lysine 9, autocatalytic G9a methylation is necessary and sufficient to mediate in vivo interaction with the epigenetic regulator heterochromatin protein 1 (HP1), and this methyl-dependent interaction can be reversed by adjacent G9a phosphorylation.  NMR anal. indicates that the HP1 chromodomain recognizes methyl-G9a through a binding mode similar to that used in recognition of methyl-H3K9, demonstrating that the chromodomain functions as a generalized methyl-lysine binding module.  These data reveal histone-like modification cassettes-or "histone mimics"-as a distinct class of nonhistone methylation targets and directly extend the principles of the histone code to the regulation of nonhistone proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFIcB4m0CyVbVg90H21EOLACvtfcHk0libhu9km2YuiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVSmtL3J&md5=3002c1fd27ba6061da0cef626680b504</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2007.06.026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2007.06.026%26sid%3Dliteratum%253Aachs%26aulast%3DSampath%26aufirst%3DS.%2BC.%26aulast%3DMarazzi%26aufirst%3DI.%26aulast%3DYap%26aufirst%3DK.%2BL.%26aulast%3DSampath%26aufirst%3DS.%2BC.%26aulast%3DKrutchinsky%26aufirst%3DA.%2BN.%26aulast%3DMecklenbrauker%26aufirst%3DI.%26aulast%3DViale%26aufirst%3DA.%26aulast%3DRudensky%26aufirst%3DE.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26aulast%3DChait%26aufirst%3DB.%2BT.%26aulast%3DTarakhovsky%26aufirst%3DA.%26atitle%3DMethylation%2520of%2520a%2520histone%2520mimic%2520within%2520the%2520histone%2520methyltransferase%2520G9a%2520regulates%2520protein%2520complex%2520assembly%26jtitle%3DMol.%2520Cell%26date%3D2007%26volume%3D27%26spage%3D596%26epage%3D608%26doi%3D10.1016%2Fj.molcel.2007.06.026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Shinkai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tachibana, M.</span><span> </span><span class="NLM_article-title">H3K9 methyltransferase G9a and the related molecule GLP</span> <span class="citation_source-journal">Genes Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">781</span><span class="NLM_x">–</span> <span class="NLM_lpage">788</span><span class="refDoi"> DOI: 10.1101/gad.2027411</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1101%2Fgad.2027411" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=21498567" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BC3MXltlKntrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2011&pages=781-788&author=Y.+Shinkaiauthor=M.+Tachibana&title=H3K9+methyltransferase+G9a+and+the+related+molecule+GLP&doi=10.1101%2Fgad.2027411"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">H3K9 methyltransferase G9a and the related molecule GLP</span></div><div class="casAuthors">Shinkai, Yoichi; Tachibana, Makoto</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">781-788</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">A review.  The discovery of Suv39h1, the first SET domain-contg. histone lysine methyltransferase (HKMT), was reported in 2000.  Since then, research on histone methylation has progressed rapidly.  Among the identified HKMTs in mammals, G9a and GLP are the primary enzymes for mono- and dimethylation at Lys 9 of histone H3 (H3K9me1 and H3K9me2), and exist predominantly as a G9a-GLP heteromeric complex that appears to be a functional H3K9 methyltransferase in vivo.  Recently, many important studies have reported that G9a and GLP play crit. roles in various biol. processes.  The physiol. relevance of G9a/GLP-mediated epigenetic gene regulation is discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCYw7kIeMuFbVg90H21EOLACvtfcHk0libhu9km2YuiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXltlKntrg%253D&md5=f688e93e1b6b7966d04c473536956ef8</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1101%2Fgad.2027411&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.2027411%26sid%3Dliteratum%253Aachs%26aulast%3DShinkai%26aufirst%3DY.%26aulast%3DTachibana%26aufirst%3DM.%26atitle%3DH3K9%2520methyltransferase%2520G9a%2520and%2520the%2520related%2520molecule%2520GLP%26jtitle%3DGenes%2520Dev.%26date%3D2011%26volume%3D25%26spage%3D781%26epage%3D788%26doi%3D10.1101%2Fgad.2027411" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Tachibana, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nozaki, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takeda, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shinkai, Y.</span><span> </span><span class="NLM_article-title">Functional dynamics of H3K9 methylation during meiotic prophase progression</span> <span class="citation_source-journal">EMBO J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">3346</span><span class="NLM_x">–</span> <span class="NLM_lpage">3359</span><span class="refDoi"> DOI: 10.1038/sj.emboj.7601767</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1038%2Fsj.emboj.7601767" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2007&pages=3346-3359&author=M.+Tachibanaauthor=M.+Nozakiauthor=N.+Takedaauthor=Y.+Shinkai&title=Functional+dynamics+of+H3K9+methylation+during+meiotic+prophase+progression&doi=10.1038%2Fsj.emboj.7601767"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fsj.emboj.7601767&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.emboj.7601767%26sid%3Dliteratum%253Aachs%26aulast%3DTachibana%26aufirst%3DM.%26aulast%3DNozaki%26aufirst%3DM.%26aulast%3DTakeda%26aufirst%3DN.%26aulast%3DShinkai%26aufirst%3DY.%26atitle%3DFunctional%2520dynamics%2520of%2520H3K9%2520methylation%2520during%2520meiotic%2520prophase%2520progression%26jtitle%3DEMBO%2520J.%26date%3D2007%26volume%3D26%26spage%3D3346%26epage%3D3359%26doi%3D10.1038%2Fsj.emboj.7601767" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Kondo, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurokawa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Masuko, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanaka, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kato, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mizuno, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yokoe, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sugauchi, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirashima, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orito, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Osada, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ueda, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Issa, J. P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sekido, Y.</span><span> </span><span class="NLM_article-title">Alterations of DNA methylation and histone modifications contribute to gene silencing in hepatocellular carcinomas</span> <span class="citation_source-journal">Hepatol. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">37</span><span class="NLM_x">, </span> <span class="NLM_fpage">974</span><span class="NLM_x">–</span> <span class="NLM_lpage">983</span><span class="refDoi"> DOI: 10.1111/j.1872-034X.2007.00141.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1111%2Fj.1872-034X.2007.00141.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=17584191" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlCns7fK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2007&pages=974-983&author=Y.+Kondoauthor=L.+Shenauthor=S.+Suzukiauthor=T.+Kurokawaauthor=K.+Masukoauthor=Y.+Tanakaauthor=H.+Katoauthor=Y.+Mizunoauthor=M.+Yokoeauthor=F.+Sugauchiauthor=N.+Hirashimaauthor=E.+Oritoauthor=H.+Osadaauthor=R.+Uedaauthor=Y.+Guoauthor=X.+L.+Chenauthor=J.+P.+J.+Issaauthor=Y.+Sekido&title=Alterations+of+DNA+methylation+and+histone+modifications+contribute+to+gene+silencing+in+hepatocellular+carcinomas&doi=10.1111%2Fj.1872-034X.2007.00141.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Alterations of DNA methylation and histone modifications contribute to gene silencing in hepatocellular carcinomas</span></div><div class="casAuthors">Kondo, Yutaka; Shen, Lanlan; Suzuki, Seiji; Kurokawa, Tsuyoshi; Masuko, Kazuo; Tanaka, Yasuhito; Kato, Hideaki; Mizuno, Yoshiki; Yokoe, Masamichi; Sugauchi, Fuminaka; Hirashima, Noboru; Orito, Etsuro; Osada, Hirotaka; Ueda, Ryuzo; Guo, Yi; Chen, Xinli; Issa, Jean-Pierre J.; Sekido, Yoshitaka</div><div class="citationInfo"><span class="NLM_cas:title">Hepatology Research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">974-983</span>CODEN:
                <span class="NLM_cas:coden">HPRSFM</span>;
        ISSN:<span class="NLM_cas:issn">1386-6346</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Asia Pty Ltd.</span>)
        </div><div class="casAbstract">Aim: The aim of the present study was to examine DNA methylation and histone modification changes in hepatocellular carcinomas (HCC).  Methods: DNA methylation in the P16, RASSF1a, progesterone receptor (PGR) and estrogen receptor α (ERα) promoters was detd. by quant. bisulfite-pyrosequencing technique in HCC patients.  Histone H3-lysine (K) 4, H3-K9 and H3-K27 modifications in all these four genes were examd. by chromatin immunopptn. (ChIP) assay in HCC cell lines.  Expression of two DNA methyltransferases (DNMT1 and DNMT3b) and three histone methyltransferases (SUV39H1, G9a and EZH2) in HCC patients was measured by real-time polymerase chain reaction.  Results: Aberrant DNA methylation was detected in all the HCC.  Patients with DNA methylation in the RASSF1a, PGR and ERα promoters in cancers also had substantial DNA methylation in their non-cancerous liver tissues, whereas DNA methylation in the P16 promoter was cancer specific.  Epigenetic states in HCC cell lines showed that silencing of P16 and RASSF1a depended on DNA methylation and histone H3-K9 methylation.  However, silencing of the PGR and ERα genes was more closely related to H3-K27 methylation rather than DNA methylation.  Consistent with the alteration of histone status, higher expression of G9a and EZH2 was found in HCC than in non-cancerous liver tissues (P < 0.01).  Conclusion: These data suggest that multiple epigenetic silencing mechanisms are inappropriately active in HCC cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwkWfvYpsUX7Vg90H21EOLACvtfcHk0liM_nm-myIVcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlCns7fK&md5=afc7013e4adfd4dd30d0b0e07ce8d9ef</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1111%2Fj.1872-034X.2007.00141.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1872-034X.2007.00141.x%26sid%3Dliteratum%253Aachs%26aulast%3DKondo%26aufirst%3DY.%26aulast%3DShen%26aufirst%3DL.%26aulast%3DSuzuki%26aufirst%3DS.%26aulast%3DKurokawa%26aufirst%3DT.%26aulast%3DMasuko%26aufirst%3DK.%26aulast%3DTanaka%26aufirst%3DY.%26aulast%3DKato%26aufirst%3DH.%26aulast%3DMizuno%26aufirst%3DY.%26aulast%3DYokoe%26aufirst%3DM.%26aulast%3DSugauchi%26aufirst%3DF.%26aulast%3DHirashima%26aufirst%3DN.%26aulast%3DOrito%26aufirst%3DE.%26aulast%3DOsada%26aufirst%3DH.%26aulast%3DUeda%26aufirst%3DR.%26aulast%3DGuo%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DX.%2BL.%26aulast%3DIssa%26aufirst%3DJ.%2BP.%2BJ.%26aulast%3DSekido%26aufirst%3DY.%26atitle%3DAlterations%2520of%2520DNA%2520methylation%2520and%2520histone%2520modifications%2520contribute%2520to%2520gene%2520silencing%2520in%2520hepatocellular%2520carcinomas%26jtitle%3DHepatol.%2520Res.%26date%3D2007%26volume%3D37%26spage%3D974%26epage%3D983%26doi%3D10.1111%2Fj.1872-034X.2007.00141.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Watanabe, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soejima, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yasuda, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawada, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakachi, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoda, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Naoki, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ishizaka, A.</span><span> </span><span class="NLM_article-title">Deregulation of histone lysine methyltransferases contributes to oncogenic transformation of human bronchoepithelial cells</span> <span class="citation_source-journal">Cancer Cell Int.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">15</span><span class="refDoi"> DOI: 10.1186/1475-2867-8-15</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1186%2F1475-2867-8-15" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=18980680" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A280%3ADC%252BD1cjltFSntg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2008&pages=15&author=H.+Watanabeauthor=K.+Soejimaauthor=H.+Yasudaauthor=I.+Kawadaauthor=I.+Nakachiauthor=S.+Yodaauthor=K.+Naokiauthor=A.+Ishizaka&title=Deregulation+of+histone+lysine+methyltransferases+contributes+to+oncogenic+transformation+of+human+bronchoepithelial+cells&doi=10.1186%2F1475-2867-8-15"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Deregulation of histone lysine methyltransferases contributes to oncogenic transformation of human bronchoepithelial cells</span></div><div class="casAuthors">Watanabe Hideo; Soejima Kenzo; Yasuda Hiroyuki; Kawada Ichiro; Nakachi Ichiro; Yoda Satoshi; Naoki Katsuhiko; Ishizaka Akitoshi</div><div class="citationInfo"><span class="NLM_cas:title">Cancer cell international</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">15</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Alterations in the processing of the genetic information in carcinogenesis result from stable genetic mutations or epigenetic modifications.  It is becoming clear that nucleosomal histones are central to proper gene expression and that aberrant DNA methylation of genes and histone methylation plays important roles in tumor progression.  To date, several histone lysine methyltransferases (HKMTs) have been identified and histone lysine methylation is now considered to be a critical regulator of transcription.  However, still relatively little is known about the role of HKMTs in tumorigenesis.  RESULTS:  We observed differential HKMT expression in a lung cancer model in which normal human bronchial epithelial (NHBE) cells expressing telomerase, SV40 large T antigen, and Ras were immortal, formed colonies in soft agar, and expressed specific HKMTs for H3 lysine 9 and 27 residues but not for H3 lysine 4 residue.  Modifications in the H3 tails affect the binding of proteins to the histone tails and regulate protein function and the position of lysine methylation marks a gene to be either activated or repressed.  In the present study, suppression by siRNA of HKMTs (EZH2, G9A, SETDB1 and SUV39H1) that are over-expressed in immortalized and transformed cells lead to reduced cell proliferation and much less anchorage-independent colony growth.  We also found that the suppression of H3-K9, G9A and SUV39H1 induced apoptosis and the suppression of H3-K27, EZH2 caused G1 arrest.  CONCLUSION:  Our results indicate the potential of these HKMTs in addition to the other targets for epigenetics such as DNMTs and HDACs to be interesting therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ8X-3k_fB763y-c-E1v-fmfW6udTcc2eYQv9Ko2Otcq7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1cjltFSntg%253D%253D&md5=ac77cc1b0cca2a4d8bf111fb6463ee7e</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1186%2F1475-2867-8-15&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1475-2867-8-15%26sid%3Dliteratum%253Aachs%26aulast%3DWatanabe%26aufirst%3DH.%26aulast%3DSoejima%26aufirst%3DK.%26aulast%3DYasuda%26aufirst%3DH.%26aulast%3DKawada%26aufirst%3DI.%26aulast%3DNakachi%26aufirst%3DI.%26aulast%3DYoda%26aufirst%3DS.%26aulast%3DNaoki%26aufirst%3DK.%26aulast%3DIshizaka%26aufirst%3DA.%26atitle%3DDeregulation%2520of%2520histone%2520lysine%2520methyltransferases%2520contributes%2520to%2520oncogenic%2520transformation%2520of%2520human%2520bronchoepithelial%2520cells%26jtitle%3DCancer%2520Cell%2520Int.%26date%3D2008%26volume%3D8%26spage%3D15%26doi%3D10.1186%2F1475-2867-8-15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Kondo, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ahmed, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boumber, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sekido, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haddad, B. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Issa, J. P. J.</span><span> </span><span class="NLM_article-title">Downregulation of histone H3 lysine 9 methyltransferase G9a induces centrosome disruption and chromosome instability in cancer cells</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">e2037</span><span class="refDoi"> DOI: 10.1371/journal.pone.0002037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1371%2Fjournal.pone.0002037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=18446223" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A280%3ADC%252BD1c3otlyhtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2008&pages=e2037&author=Y.+Kondoauthor=L.+Shenauthor=S.+Ahmedauthor=Y.+Boumberauthor=Y.+Sekidoauthor=B.+R.+Haddadauthor=J.+P.+J.+Issa&title=Downregulation+of+histone+H3+lysine+9+methyltransferase+G9a+induces+centrosome+disruption+and+chromosome+instability+in+cancer+cells&doi=10.1371%2Fjournal.pone.0002037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Downregulation of histone H3 lysine 9 methyltransferase G9a induces centrosome disruption and chromosome instability in cancer cells</span></div><div class="casAuthors">Kondo Yutaka; Shen Lanlan; Ahmed Saira; Boumber Yanis; Sekido Yoshitaka; Haddad Bassem R; Issa Jean-Pierre J</div><div class="citationInfo"><span class="NLM_cas:title">PloS one</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">e2037</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Modifications of the histone amino-terminal tails affect access of regulatory factors and complexes to chromatin and thereby influence biological processes.  Cancer cells are characterized by prominent epigenetic dysregulation, including histone modifications.  However, the functional roles of the histone methyltransferases (HMT) in cancer remain unclear.  METHODOLOGY/PRINCIPAL FINDINGS:  We studied RNAi-based inhibition (knockdown, KD) of 2 different H3K9 HMTs, SUV39H1 and G9a.  Knockdown of the 2 HMTs in PC3 cancer cell line markedly inhibited cell growth and caused profound morphological changes with loss of telomerase activity and shortened telomeres.  SUV39H1 KD cells showed substantial increase in G2/M fraction.  G9a KD cells showed increased DNA content (1.7-fold in 2 independent clones) compared with FACS analyses to control.  Karyotype analyses showed that this was due to an increased number of chromosomes (from 61 to 102) in G9a KD cells compared to parental PC3.  Intriguingly, we found abnormal centrosome morphology and number in about 25% of the G9a KD cells, while centrosomes were morphologically normal in control cells.  Microarray analyses after KD of SUV39H1 or G9a showed very few genes up-regulated among the 39,000 genes.  The silenced tumor-suppressor genes p16 and RASSF1A were not activated in KD cells.  CONCLUSIONS/SIGNIFICANCE:  These data suggest that the 2 HMTs, SUV39H1 and G9a are required to perpetuate the malignant phenotype.  Furthermore, G9a plays a critical role in regulating centrosome duplication presumably through chromatin structure rather than through affecting gene expression in cancer cells.  Targeting these histone methyltransferases may be of therapeutic benefit in cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQhnx1F8nyNwDEwO064Z95rfW6udTcc2eYQv9Ko2Otcq7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1c3otlyhtw%253D%253D&md5=22936eeeffa4dc29e7254dfc546ebcfd</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0002037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0002037%26sid%3Dliteratum%253Aachs%26aulast%3DKondo%26aufirst%3DY.%26aulast%3DShen%26aufirst%3DL.%26aulast%3DAhmed%26aufirst%3DS.%26aulast%3DBoumber%26aufirst%3DY.%26aulast%3DSekido%26aufirst%3DY.%26aulast%3DHaddad%26aufirst%3DB.%2BR.%26aulast%3DIssa%26aufirst%3DJ.%2BP.%2BJ.%26atitle%3DDownregulation%2520of%2520histone%2520H3%2520lysine%25209%2520methyltransferase%2520G9a%2520induces%2520centrosome%2520disruption%2520and%2520chromosome%2520instability%2520in%2520cancer%2520cells%26jtitle%3DPLoS%2520One%26date%3D2008%26volume%3D3%26spage%3De2037%26doi%3D10.1371%2Fjournal.pone.0002037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Thomas, L. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyashita, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cobb, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pierce, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tachibana, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hobeika, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reth, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shinkai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oltz, E. M.</span><span> </span><span class="NLM_article-title">Functional analysis of histone methyltransferase G9a in B and T lymphocytes</span> <span class="citation_source-journal">J. Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">181</span><span class="NLM_x">, </span> <span class="NLM_fpage">485</span><span class="NLM_x">–</span> <span class="NLM_lpage">493</span><span class="refDoi"> DOI: 10.4049/jimmunol.181.1.485</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.4049%2Fjimmunol.181.1.485" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=181&publication_year=2008&pages=485-493&author=L.+R.+Thomasauthor=H.+Miyashitaauthor=R.+M.+Cobbauthor=S.+Pierceauthor=M.+Tachibanaauthor=E.+Hobeikaauthor=M.+Rethauthor=Y.+Shinkaiauthor=E.+M.+Oltz&title=Functional+analysis+of+histone+methyltransferase+G9a+in+B+and+T+lymphocytes&doi=10.4049%2Fjimmunol.181.1.485"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.4049%2Fjimmunol.181.1.485&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4049%252Fjimmunol.181.1.485%26sid%3Dliteratum%253Aachs%26aulast%3DThomas%26aufirst%3DL.%2BR.%26aulast%3DMiyashita%26aufirst%3DH.%26aulast%3DCobb%26aufirst%3DR.%2BM.%26aulast%3DPierce%26aufirst%3DS.%26aulast%3DTachibana%26aufirst%3DM.%26aulast%3DHobeika%26aufirst%3DE.%26aulast%3DReth%26aufirst%3DM.%26aulast%3DShinkai%26aufirst%3DY.%26aulast%3DOltz%26aufirst%3DE.%2BM.%26atitle%3DFunctional%2520analysis%2520of%2520histone%2520methyltransferase%2520G9a%2520in%2520B%2520and%2520T%2520lymphocytes%26jtitle%3DJ.%2520Immunol.%26date%3D2008%26volume%3D181%26spage%3D485%26epage%3D493%26doi%3D10.4049%2Fjimmunol.181.1.485" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Lehnertz, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Northrop, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Antignano, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burrows, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hadidi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mullaly, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rossi, F. M. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zaph, C.</span><span> </span><span class="NLM_article-title">Activating and inhibitory functions for the histone lysine methyltransferase G9a in T helper cell differentiation and function</span> <span class="citation_source-journal">J. Exp. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">207</span><span class="NLM_x">, </span> <span class="NLM_fpage">915</span><span class="NLM_x">–</span> <span class="NLM_lpage">922</span><span class="refDoi"> DOI: 10.1084/jem.20100363</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1084%2Fjem.20100363" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=207&publication_year=2010&pages=915-922&author=B.+Lehnertzauthor=J.+P.+Northropauthor=F.+Antignanoauthor=K.+Burrowsauthor=S.+Hadidiauthor=S.+C.+Mullalyauthor=F.+M.+V.+Rossiauthor=C.+Zaph&title=Activating+and+inhibitory+functions+for+the+histone+lysine+methyltransferase+G9a+in+T+helper+cell+differentiation+and+function&doi=10.1084%2Fjem.20100363"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1084%2Fjem.20100363&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1084%252Fjem.20100363%26sid%3Dliteratum%253Aachs%26aulast%3DLehnertz%26aufirst%3DB.%26aulast%3DNorthrop%26aufirst%3DJ.%2BP.%26aulast%3DAntignano%26aufirst%3DF.%26aulast%3DBurrows%26aufirst%3DK.%26aulast%3DHadidi%26aufirst%3DS.%26aulast%3DMullaly%26aufirst%3DS.%2BC.%26aulast%3DRossi%26aufirst%3DF.%2BM.%2BV.%26aulast%3DZaph%26aufirst%3DC.%26atitle%3DActivating%2520and%2520inhibitory%2520functions%2520for%2520the%2520histone%2520lysine%2520methyltransferase%2520G9a%2520in%2520T%2520helper%2520cell%2520differentiation%2520and%2520function%26jtitle%3DJ.%2520Exp.%2520Med.%26date%3D2010%26volume%3D207%26spage%3D915%26epage%3D922%26doi%3D10.1084%2Fjem.20100363" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Maze, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Covington, H. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dietz, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LaPlant, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Renthal, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Russo, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mechanic, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mouzon, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neve, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haggarty, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ren, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sampath, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurd, Y. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greengard, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tarakhovsky, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schaefer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nestler, E. J.</span><span> </span><span class="NLM_article-title">Essential role of the histone methyltransferase G9a in cocaine-induced plasticity</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">327</span><span class="NLM_x">, </span> <span class="NLM_fpage">213</span><span class="NLM_x">–</span> <span class="NLM_lpage">216</span><span class="refDoi"> DOI: 10.1126/science.1179438</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1126%2Fscience.1179438" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=20056891" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1egtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=327&publication_year=2010&pages=213-216&author=I.+Mazeauthor=H.+E.+Covingtonauthor=D.+M.+Dietzauthor=Q.+LaPlantauthor=W.+Renthalauthor=S.+J.+Russoauthor=M.+Mechanicauthor=E.+Mouzonauthor=R.+L.+Neveauthor=S.+J.+Haggartyauthor=Y.+H.+Renauthor=S.+C.+Sampathauthor=Y.+L.+Hurdauthor=P.+Greengardauthor=A.+Tarakhovskyauthor=A.+Schaeferauthor=E.+J.+Nestler&title=Essential+role+of+the+histone+methyltransferase+G9a+in+cocaine-induced+plasticity&doi=10.1126%2Fscience.1179438"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Essential Role of the Histone Methyltransferase G9a in Cocaine-Induced Plasticity</span></div><div class="casAuthors">Maze, Ian; Covington, Herbert E., III; Dietz, David M.; LaPlant, Quincey; Renthal, William; Russo, Scott J.; Mechanic, Max; Mouzon, Ezekiell; Neve, Rachael L.; Haggarty, Stephen J.; Ren, Yanhua; Sampath, Srihari C.; Hurd, Yasmin L.; Greengard, Paul; Tarakhovsky, Alexander; Schaefer, Anne; Nestler, Eric J.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">327</span>
        (<span class="NLM_cas:issue">5962</span>),
    <span class="NLM_cas:pages">213-216</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Cocaine-induced alterations in gene expression cause changes in neuronal morphol. and behavior that may underlie cocaine addiction.  In mice, we identified an essential role for histone 3 lysine 9 (H3K9) dimethylation and the lysine dimethyltransferase G9a in cocaine-induced structural and behavioral plasticity.  Repeated cocaine administration reduced global levels of H3K9 dimethylation in the nucleus accumbens.  This redn. in histone methylation was mediated through the repression of G9a in this brain region, which was regulated by the cocaine-induced transcription factor ΔFosB.  Using conditional mutagenesis and viral-mediated gene transfer, we found that G9a down-regulation increased the dendritic spine plasticity of nucleus accumbens neurons and enhanced the preference for cocaine, thereby establishing a crucial role for histone methylation in the long-term actions of cocaine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8SPD826wEGrVg90H21EOLACvtfcHk0ljxX93ZO_OFuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1egtA%253D%253D&md5=a8347500f81862078787e50fff53b6b9</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1126%2Fscience.1179438&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1179438%26sid%3Dliteratum%253Aachs%26aulast%3DMaze%26aufirst%3DI.%26aulast%3DCovington%26aufirst%3DH.%2BE.%26aulast%3DDietz%26aufirst%3DD.%2BM.%26aulast%3DLaPlant%26aufirst%3DQ.%26aulast%3DRenthal%26aufirst%3DW.%26aulast%3DRusso%26aufirst%3DS.%2BJ.%26aulast%3DMechanic%26aufirst%3DM.%26aulast%3DMouzon%26aufirst%3DE.%26aulast%3DNeve%26aufirst%3DR.%2BL.%26aulast%3DHaggarty%26aufirst%3DS.%2BJ.%26aulast%3DRen%26aufirst%3DY.%2BH.%26aulast%3DSampath%26aufirst%3DS.%2BC.%26aulast%3DHurd%26aufirst%3DY.%2BL.%26aulast%3DGreengard%26aufirst%3DP.%26aulast%3DTarakhovsky%26aufirst%3DA.%26aulast%3DSchaefer%26aufirst%3DA.%26aulast%3DNestler%26aufirst%3DE.%2BJ.%26atitle%3DEssential%2520role%2520of%2520the%2520histone%2520methyltransferase%2520G9a%2520in%2520cocaine-induced%2520plasticity%26jtitle%3DScience%26date%3D2010%26volume%3D327%26spage%3D213%26epage%3D216%26doi%3D10.1126%2Fscience.1179438" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Schaefer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sampath, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Intrator, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Min, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gertler, T. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Surmeier, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tarakhovsky, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greengard, P.</span><span> </span><span class="NLM_article-title">Control of cognition and adaptive behavior by the GLP/G9a epigenetic suppressor complex</span> <span class="citation_source-journal">Neuron</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">64</span><span class="NLM_x">, </span> <span class="NLM_fpage">678</span><span class="NLM_x">–</span> <span class="NLM_lpage">691</span><span class="refDoi"> DOI: 10.1016/j.neuron.2009.11.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1016%2Fj.neuron.2009.11.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=20005824" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BC3cXks1Cls7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2009&pages=678-691&author=A.+Schaeferauthor=S.+C.+Sampathauthor=A.+Intratorauthor=A.+Minauthor=T.+S.+Gertlerauthor=D.+J.+Surmeierauthor=A.+Tarakhovskyauthor=P.+Greengard&title=Control+of+cognition+and+adaptive+behavior+by+the+GLP%2FG9a+epigenetic+suppressor+complex&doi=10.1016%2Fj.neuron.2009.11.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Control of cognition and adaptive behavior by the GLP/G9a epigenetic suppressor complex</span></div><div class="casAuthors">Schaefer, Anne; Sampath, Srihari C.; Intrator, Adam; Min, Alice; Gertler, Tracy S.; Surmeier, D. James; Tarakhovsky, Alexander; Greengard, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Neuron</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">678-691</span>CODEN:
                <span class="NLM_cas:coden">NERNET</span>;
        ISSN:<span class="NLM_cas:issn">0896-6273</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The genetic basis of cognition and behavioral adaptation to the environment remains poorly understood.  Here we demonstrate that the histone methyltransferase complex GLP/G9a controls cognition and adaptive responses in a region-specific fashion in the adult brain.  Using conditional mutagenesis in mice, we show that postnatal, neuron-specific deficiency of GLP/G9a leads to derepression of numerous nonneuronal and neuron progenitor genes in adult neurons.  This transcriptional alteration is assocd. with complex behavioral abnormalities, including defects in learning, motivation, and environmental adaptation.  The behavioral changes triggered by GLP/G9a deficiency are similar to key symptoms of the human 9q34 mental retardation syndrome that is assocd. with structural alterations of the GLP/EHMT1 gene.  The likely causal role of GLP/G9a in mental retardation in mice and humans suggests a key role for the GLP/G9a-controlled histone H3K9 dimethylation in regulation of brain function through maintenance of the transcriptional homeostasis in adult neurons.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr22UcjjqkKdLVg90H21EOLACvtfcHk0ljxX93ZO_OFuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXks1Cls7c%253D&md5=ece4e94ad097a8d0e67a5e8d27970e64</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.neuron.2009.11.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuron.2009.11.019%26sid%3Dliteratum%253Aachs%26aulast%3DSchaefer%26aufirst%3DA.%26aulast%3DSampath%26aufirst%3DS.%2BC.%26aulast%3DIntrator%26aufirst%3DA.%26aulast%3DMin%26aufirst%3DA.%26aulast%3DGertler%26aufirst%3DT.%2BS.%26aulast%3DSurmeier%26aufirst%3DD.%2BJ.%26aulast%3DTarakhovsky%26aufirst%3DA.%26aulast%3DGreengard%26aufirst%3DP.%26atitle%3DControl%2520of%2520cognition%2520and%2520adaptive%2520behavior%2520by%2520the%2520GLP%252FG9a%2520epigenetic%2520suppressor%2520complex%26jtitle%3DNeuron%26date%3D2009%26volume%3D64%26spage%3D678%26epage%3D691%26doi%3D10.1016%2Fj.neuron.2009.11.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Leung, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, K. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maksakova, I. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goyal, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Appanah, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tachibana, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shinkai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lehnertz, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mager, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rossi, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lorincz, M. C.</span><span> </span><span class="NLM_article-title">Lysine methyltransferase G9a is required for de novo DNA methylation and the establishment, but not the maintenance, of proviral silencing</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">108</span><span class="NLM_x">, </span> <span class="NLM_fpage">5718</span><span class="NLM_x">–</span> <span class="NLM_lpage">5723</span><span class="refDoi"> DOI: 10.1073/pnas.1014660108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1073%2Fpnas.1014660108" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=5718-5723&author=D.+C.+Leungauthor=K.+B.+Dongauthor=I.+A.+Maksakovaauthor=P.+Goyalauthor=R.+Appanahauthor=S.+Leeauthor=M.+Tachibanaauthor=Y.+Shinkaiauthor=B.+Lehnertzauthor=D.+L.+Magerauthor=F.+Rossiauthor=M.+C.+Lorincz&title=Lysine+methyltransferase+G9a+is+required+for+de+novo+DNA+methylation+and+the+establishment%2C+but+not+the+maintenance%2C+of+proviral+silencing&doi=10.1073%2Fpnas.1014660108"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1014660108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1014660108%26sid%3Dliteratum%253Aachs%26aulast%3DLeung%26aufirst%3DD.%2BC.%26aulast%3DDong%26aufirst%3DK.%2BB.%26aulast%3DMaksakova%26aufirst%3DI.%2BA.%26aulast%3DGoyal%26aufirst%3DP.%26aulast%3DAppanah%26aufirst%3DR.%26aulast%3DLee%26aufirst%3DS.%26aulast%3DTachibana%26aufirst%3DM.%26aulast%3DShinkai%26aufirst%3DY.%26aulast%3DLehnertz%26aufirst%3DB.%26aulast%3DMager%26aufirst%3DD.%2BL.%26aulast%3DRossi%26aufirst%3DF.%26aulast%3DLorincz%26aufirst%3DM.%2BC.%26atitle%3DLysine%2520methyltransferase%2520G9a%2520is%2520required%2520for%2520de%2520novo%2520DNA%2520methylation%2520and%2520the%2520establishment%252C%2520but%2520not%2520the%2520maintenance%252C%2520of%2520proviral%2520silencing%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26spage%3D5718%26epage%3D5723%26doi%3D10.1073%2Fpnas.1014660108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Kleefstra, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smidt, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banning, M. J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oudakker, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Esch, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Brouwer, A. P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nillesen, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sistermans, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamel, B. C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Bruijn, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fryns, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yntema, H. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brunner, H. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Vries, B. B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Bokhoven, H.</span><span> </span><span class="NLM_article-title">Disruption of the gene euchromatin histone methyl transferase1 (Eu-HMTase1) is associated with the 9q34 subtelomeric deletion syndrome</span> <span class="citation_source-journal">J. Med. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">299</span><span class="NLM_x">–</span> <span class="NLM_lpage">306</span><span class="refDoi"> DOI: 10.1136/jmg.2004.028464</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1136%2Fjmg.2004.028464" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=15805155" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BD2MXktFGgtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2005&pages=299-306&author=T.+Kleefstraauthor=M.+Smidtauthor=M.+J.+G.+Banningauthor=A.+R.+Oudakkerauthor=H.+Van+Eschauthor=A.+P.+M.+de+Brouwerauthor=W.+Nillesenauthor=E.+A.+Sistermansauthor=B.+C.+J.+Hamelauthor=D.+de+Bruijnauthor=J.+P.+Frynsauthor=H.+G.+Yntemaauthor=H.+G.+Brunnerauthor=B.+B.+A.+de+Vriesauthor=H.+van+Bokhoven&title=Disruption+of+the+gene+euchromatin+histone+methyl+transferase1+%28Eu-HMTase1%29+is+associated+with+the+9q34+subtelomeric+deletion+syndrome&doi=10.1136%2Fjmg.2004.028464"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Disruption of the gene euchromatin histone methyl transferase1 (Eu-HMTase1) is associated with the 9q34 subtelomeric deletion syndrome</span></div><div class="casAuthors">Kleefstra, T.; Smidt, M.; Banning, M. J. G.; Oudakker, A. R.; van Esch, H.; de Brouwer, A. P. M.; Nillesen, W.; Sistermans, E. A.; Hamel, B. C. J.; de Bruijn, D.; Fryns, J.-P.; Yntema, H. G.; Brunner, H. G.; de Vries, B. B. A.; van Bokhoven, H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medical Genetics</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">299-306</span>CODEN:
                <span class="NLM_cas:coden">JMDGAE</span>;
        ISSN:<span class="NLM_cas:issn">0022-2593</span>.
    
            (<span class="NLM_cas:orgname">BMJ Publishing Group</span>)
        </div><div class="casAbstract">Background: A new syndrome has been recognized following thorough anal. of patients with a terminal submicroscopic subtelomeric deletion of chromosome 9q.  These have in common severe mental retardation, hypotonia, brachycephaly, flat face with hypertelorism, synophrys, anteverted nares, thickened lower lip, carp mouth with macroglossia, and canotruncal heart defects.  The min. crit. region responsible for this 9q subtelomeric deletion syndrome (9q-) is approx. 1.2 Mb and encompasses at least 14 genes.  Objective: To characterize the breakpoints of a de novo balanced translocation t(X;9)(p11.23;q34.3) in a mentally retarded female patient with clin. features similar to the 9q- syndrome.  Results: Sequence anal. of the break points showed that the translocation was fully balanced and only one gene on chromosome 9 was disrupted-Euchromatin Histone Me Transferase1 (Eu-HMTase1)-encoding a histone H3 lysine 9 methyltransferase (H3-K9 HMTase).  This indicates that haploinsufficiency of Eu-HMTase1 is responsible for the 9q submicroscopic subtelomeric deletion syndrome.  This observation was further supported by the spatio-temporal expression of the gene.  Using tissue in situ hybridization studies in mouse embryos and adult brain, Eu-HMTase1 was shown to be expressed in the developing nervous system and in specific peripheral tissues.  While expression is selectively downregulated in adult brain, substantial expression is retained in the olfactory bulb, anterior/ventral lateral ventricular wall, and hippocampus and weakly in the piriform cortex.  Conclusions: The expression pattern of this gene suggests a role in the CNS development and function, which is in line with the severe mental retardation and behavior problems in patients who lack one copy of the gene.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnHiNhjDYOqrVg90H21EOLACvtfcHk0liAbFfiO4wnTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXktFGgtrY%253D&md5=518802ab24e9b6a4ffed3e0f20b374c6</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1136%2Fjmg.2004.028464&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Fjmg.2004.028464%26sid%3Dliteratum%253Aachs%26aulast%3DKleefstra%26aufirst%3DT.%26aulast%3DSmidt%26aufirst%3DM.%26aulast%3DBanning%26aufirst%3DM.%2BJ.%2BG.%26aulast%3DOudakker%26aufirst%3DA.%2BR.%26aulast%3DVan%2BEsch%26aufirst%3DH.%26aulast%3Dde%2BBrouwer%26aufirst%3DA.%2BP.%2BM.%26aulast%3DNillesen%26aufirst%3DW.%26aulast%3DSistermans%26aufirst%3DE.%2BA.%26aulast%3DHamel%26aufirst%3DB.%2BC.%2BJ.%26aulast%3Dde%2BBruijn%26aufirst%3DD.%26aulast%3DFryns%26aufirst%3DJ.%2BP.%26aulast%3DYntema%26aufirst%3DH.%2BG.%26aulast%3DBrunner%26aufirst%3DH.%2BG.%26aulast%3Dde%2BVries%26aufirst%3DB.%2BB.%2BA.%26aulast%3Dvan%2BBokhoven%26aufirst%3DH.%26atitle%3DDisruption%2520of%2520the%2520gene%2520euchromatin%2520histone%2520methyl%2520transferase1%2520%2528Eu-HMTase1%2529%2520is%2520associated%2520with%2520the%25209q34%2520subtelomeric%2520deletion%2520syndrome%26jtitle%3DJ.%2520Med.%2520Genet.%26date%3D2005%26volume%3D42%26spage%3D299%26epage%3D306%26doi%3D10.1136%2Fjmg.2004.028464" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Antignano, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zaph, C.</span><span> </span><span class="NLM_article-title">Regulation of CD4 T-cell differentiation and inflammation by repressive histone methylation</span> <span class="citation_source-journal">Immunol. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">93</span><span class="NLM_x">, </span> <span class="NLM_fpage">245</span><span class="NLM_x">–</span> <span class="NLM_lpage">252</span><span class="refDoi"> DOI: 10.1038/icb.2014.115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1038%2Ficb.2014.115" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=25582341" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFGks70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=2015&pages=245-252&author=F.+Antignanoauthor=C.+Zaph&title=Regulation+of+CD4+T-cell+differentiation+and+inflammation+by+repressive+histone+methylation&doi=10.1038%2Ficb.2014.115"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of CD4 T-cell differentiation and inflammation by repressive histone methylation</span></div><div class="casAuthors">Antignano, Frann; Zaph, Colby</div><div class="citationInfo"><span class="NLM_cas:title">Immunology & Cell Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">245-252</span>CODEN:
                <span class="NLM_cas:coden">ICBIEZ</span>;
        ISSN:<span class="NLM_cas:issn">0818-9641</span>.
    
            (<span class="NLM_cas:orgname">NPG Nature Asia-Pacific</span>)
        </div><div class="casAbstract">Repressive epigenetic modifications such as dimethylation and trimethylation histone H3 at lysine 9 (H3K9me2 and H3K9me3) and H3K27me3 have been shown to be crit. for embryonic stem (ES) cell differentiation by silencing cell lineage-promiscuous genes.  CD4+ T helper (TH) cell differentiation is a powerful model to study the mol. mechanisms assocd. with cellular lineage choice in adult cells.  Naive TH cells have the capacity to differentiate into one of the several phenotypically and functionally distinct and stable lineages.  Although some repressive epigenetic mechanisms have a crit. role in TH cell differentiation in a similar manner to that in ES cells, it is clear that there are disparate functions for certain modifications between ES cells and TH cells.  Here we review the role of repressive histone modifications in the differentiation and function of TH cells in health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvWkSixxmeHLVg90H21EOLACvtfcHk0liAbFfiO4wnTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFGks70%253D&md5=db9f3617c548d1dabccfc7ceaeb687de</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1038%2Ficb.2014.115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Ficb.2014.115%26sid%3Dliteratum%253Aachs%26aulast%3DAntignano%26aufirst%3DF.%26aulast%3DZaph%26aufirst%3DC.%26atitle%3DRegulation%2520of%2520CD4%2520T-cell%2520differentiation%2520and%2520inflammation%2520by%2520repressive%2520histone%2520methylation%26jtitle%3DImmunol.%2520Cell%2520Biol.%26date%3D2015%26volume%3D93%26spage%3D245%26epage%3D252%26doi%3D10.1038%2Ficb.2014.115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Kramer, J. M.</span><span> </span><span class="NLM_article-title">Regulation of cell differentiation and function by the euchromatin histone methyltranserfases G9a and GLP</span> <span class="citation_source-journal">Biochem. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">94</span><span class="NLM_x">, </span> <span class="NLM_fpage">26</span><span class="NLM_x">–</span> <span class="NLM_lpage">32</span><span class="refDoi"> DOI: 10.1139/bcb-2015-0017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1139%2Fbcb-2015-0017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=26198080" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1SqtLfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=2016&pages=26-32&author=J.+M.+Kramer&title=Regulation+of+cell+differentiation+and+function+by+the+euchromatin+histone+methyltranserfases+G9a+and+GLP&doi=10.1139%2Fbcb-2015-0017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of cell differentiation and function by the euchromatin histone methyltranserfases G9a and GLP1</span></div><div class="casAuthors">Kramer, Jamie M.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry and Cell Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">26-32</span>CODEN:
                <span class="NLM_cas:coden">BCBIEQ</span>;
        ISSN:<span class="NLM_cas:issn">0829-8211</span>.
    
            (<span class="NLM_cas:orgname">Canadian Science Publishing</span>)
        </div><div class="casAbstract">The euchromatin histone methyltransferases (EHMTs) are an evolutionarily conserved protein family that are known for their ability to dimethylate histone 3 at lysine 9 in euchromatic regions of the genome.  In mammals there are two EHMT proteins, G9a, encoded by EHMT2, and GLP, encoded by EHMT1.  EHMTs have diverse roles in the differentiation of different tissues and cell types and are involved in adult-specific processes like memory, drug addiction, and immune response.  This review discusses recent findings from rodent and Drosophila models that are beginning to reveal the broad biol. role and complex mechanistic functioning of EHMT proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqABwbxmYijlLVg90H21EOLACvtfcHk0lh_i213EvOsRQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1SqtLfI&md5=c49f7a8898c0c8ea824ca387a01f681b</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1139%2Fbcb-2015-0017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1139%252Fbcb-2015-0017%26sid%3Dliteratum%253Aachs%26aulast%3DKramer%26aufirst%3DJ.%2BM.%26atitle%3DRegulation%2520of%2520cell%2520differentiation%2520and%2520function%2520by%2520the%2520euchromatin%2520histone%2520methyltranserfases%2520G9a%2520and%2520GLP%26jtitle%3DBiochem.%2520Cell%2520Biol.%26date%3D2016%26volume%3D94%26spage%3D26%26epage%3D32%26doi%3D10.1139%2Fbcb-2015-0017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Liu, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Z. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meng, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiong, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, Z. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, X. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cai, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, B.</span><span> </span><span class="NLM_article-title">Recognition of H3K9 methylation by GLP is required for efficient establishment of H3K9 methylation, rapid target gene repression, and mouse viability</span> <span class="citation_source-journal">Genes Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">379</span><span class="NLM_x">–</span> <span class="NLM_lpage">393</span><span class="refDoi"> DOI: 10.1101/gad.254425.114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1101%2Fgad.254425.114" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=379-393&author=N.+Liuauthor=Z.+Q.+Zhangauthor=H.+Wuauthor=Y.+H.+Jiangauthor=L.+J.+Mengauthor=J.+Xiongauthor=Z.+D.+Zhaoauthor=X.+H.+Zhouauthor=J.+Liauthor=H.+Liauthor=Y.+Zhengauthor=S.+Chenauthor=T.+Caiauthor=S.+R.+Gaoauthor=B.+Zhu&title=Recognition+of+H3K9+methylation+by+GLP+is+required+for+efficient+establishment+of+H3K9+methylation%2C+rapid+target+gene+repression%2C+and+mouse+viability&doi=10.1101%2Fgad.254425.114"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1101%2Fgad.254425.114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.254425.114%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DN.%26aulast%3DZhang%26aufirst%3DZ.%2BQ.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DJiang%26aufirst%3DY.%2BH.%26aulast%3DMeng%26aufirst%3DL.%2BJ.%26aulast%3DXiong%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DZ.%2BD.%26aulast%3DZhou%26aufirst%3DX.%2BH.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DZheng%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DCai%26aufirst%3DT.%26aulast%3DGao%26aufirst%3DS.%2BR.%26aulast%3DZhu%26aufirst%3DB.%26atitle%3DRecognition%2520of%2520H3K9%2520methylation%2520by%2520GLP%2520is%2520required%2520for%2520efficient%2520establishment%2520of%2520H3K9%2520methylation%252C%2520rapid%2520target%2520gene%2520repression%252C%2520and%2520mouse%2520viability%26jtitle%3DGenes%2520Dev.%26date%3D2015%26volume%3D29%26spage%3D379%26epage%3D393%26doi%3D10.1101%2Fgad.254425.114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Ohno, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shinoda, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohyama, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharp, L. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kajimura, S.</span><span> </span><span class="NLM_article-title">EHMT1 controls brown adipose cell fate and thermogenesis through the PRDM16 complex</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">504</span><span class="NLM_x">, </span> <span class="NLM_fpage">163</span><span class="NLM_x">–</span> <span class="NLM_lpage">167</span><span class="refDoi"> DOI: 10.1038/nature12652</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1038%2Fnature12652" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=24196706" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVyisLbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=504&publication_year=2013&pages=163-167&author=H.+Ohnoauthor=K.+Shinodaauthor=K.+Ohyamaauthor=L.+Z.+Sharpauthor=S.+Kajimura&title=EHMT1+controls+brown+adipose+cell+fate+and+thermogenesis+through+the+PRDM16+complex&doi=10.1038%2Fnature12652"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">EHMT1 controls brown adipose cell fate and thermogenesis through the PRDM16 complex</span></div><div class="casAuthors">Ohno, Haruya; Shinoda, Kosaku; Ohyama, Kana; Sharp, Louis Z.; Kajimura, Shingo</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">504</span>
        (<span class="NLM_cas:issue">7478</span>),
    <span class="NLM_cas:pages">163-167</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Brown adipose tissue (BAT) dissipates chem. energy in the form of heat as a defense against hypothermia and obesity.  Current evidence indicates that brown adipocytes arise from Myf5+ dermotomal precursors through the action of PR domain contg. protein 16 (PRDM16) transcriptional complex.  However, the enzymic component of the mol. switch that dets. lineage specification of brown adipocytes remains unknown.  Here, the authors show that euchromatic histone-lysine N-methyltransferase 1 (EHMT1) is an essential BAT-enriched lysine methyltransferase in the PRDM16 transcriptional complex and controls brown adipose cell fate.  Loss of EHMT1 in brown adipocytes causes a severe loss of brown fat characteristics and induces muscle differentiation in vivo through demethylation of histone 3 lysine 9 (H3K9me2 and 3) of the muscle-selective gene promoters.  Conversely, EHMT1 expression pos. regulates the BAT-selective thermogenic program by stabilizing the PRDM16 protein.  Notably, adipose-specific deletion of EHMT1 leads to a marked redn. of BAT-mediated adaptive thermogenesis, obesity and systemic insulin resistance.  These data indicate that EHMT1 is an essential enzymic switch that controls brown adipose cell fate and energy homeostasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZ8WN3L9wpErVg90H21EOLACvtfcHk0lh_i213EvOsRQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVyisLbO&md5=f0f0a8bf8b229a161ce2e5d8aa1530e2</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1038%2Fnature12652&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12652%26sid%3Dliteratum%253Aachs%26aulast%3DOhno%26aufirst%3DH.%26aulast%3DShinoda%26aufirst%3DK.%26aulast%3DOhyama%26aufirst%3DK.%26aulast%3DSharp%26aufirst%3DL.%2BZ.%26aulast%3DKajimura%26aufirst%3DS.%26atitle%3DEHMT1%2520controls%2520brown%2520adipose%2520cell%2520fate%2520and%2520thermogenesis%2520through%2520the%2520PRDM16%2520complex%26jtitle%3DNature%26date%3D2013%26volume%3D504%26spage%3D163%26epage%3D167%26doi%3D10.1038%2Fnature12652" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Battisti, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pontis, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boyarchuk, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fritsch, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robin, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ait-Si-Ali, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joliot, V.</span><span> </span><span class="NLM_article-title">Unexpected distinct roles of the related histone H3 lysine 9 methyltransferases G9a and G9a-like protein in myoblasts</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">428</span><span class="NLM_x">, </span> <span class="NLM_fpage">2329</span><span class="NLM_x">–</span> <span class="NLM_lpage">2343</span><span class="refDoi"> DOI: 10.1016/j.jmb.2016.03.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1016%2Fj.jmb.2016.03.029" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=428&publication_year=2016&pages=2329-2343&author=V.+Battistiauthor=J.+Pontisauthor=E.+Boyarchukauthor=L.+Fritschauthor=P.+Robinauthor=S.+Ait-Si-Aliauthor=V.+Joliot&title=Unexpected+distinct+roles+of+the+related+histone+H3+lysine+9+methyltransferases+G9a+and+G9a-like+protein+in+myoblasts&doi=10.1016%2Fj.jmb.2016.03.029"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2016.03.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2016.03.029%26sid%3Dliteratum%253Aachs%26aulast%3DBattisti%26aufirst%3DV.%26aulast%3DPontis%26aufirst%3DJ.%26aulast%3DBoyarchuk%26aufirst%3DE.%26aulast%3DFritsch%26aufirst%3DL.%26aulast%3DRobin%26aufirst%3DP.%26aulast%3DAit-Si-Ali%26aufirst%3DS.%26aulast%3DJoliot%26aufirst%3DV.%26atitle%3DUnexpected%2520distinct%2520roles%2520of%2520the%2520related%2520histone%2520H3%2520lysine%25209%2520methyltransferases%2520G9a%2520and%2520G9a-like%2520protein%2520in%2520myoblasts%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2016%26volume%3D428%26spage%3D2329%26epage%3D2343%26doi%3D10.1016%2Fj.jmb.2016.03.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quinn, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kozieradzki, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siarheyeva, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kireev, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jadhav, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herold, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Discovery of a 2,4-diamino-7-aminoalkoxyquinazoline as a potent and selective inhibitor of histone lysine methyltransferase G9a</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">7950</span><span class="NLM_x">–</span> <span class="NLM_lpage">7953</span><span class="refDoi"> DOI: 10.1021/jm901543m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901543m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlOiurzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=7950-7953&author=F.+Liuauthor=X.+Chenauthor=A.+Allali-Hassaniauthor=A.+M.+Quinnauthor=G.+A.+Wasneyauthor=A.+P.+Dongauthor=D.+Barsyteauthor=I.+Kozieradzkiauthor=G.+Senisterraauthor=I.+Chauauthor=A.+Siarheyevaauthor=D.+B.+Kireevauthor=A.+Jadhavauthor=J.+M.+Heroldauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=A.+Simeonovauthor=M.+Vedadiauthor=J.+Jin&title=Discovery+of+a+2%2C4-diamino-7-aminoalkoxyquinazoline+as+a+potent+and+selective+inhibitor+of+histone+lysine+methyltransferase+G9a&doi=10.1021%2Fjm901543m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a 2,4-Diamino-7-aminoalkoxyquinazoline as a Potent and Selective Inhibitor of Histone Lysine Methyltransferase G9a</span></div><div class="casAuthors">Liu, Feng; Chen, Xin; Allali-Hassani, Abdellah; Quinn, Amy M.; Wasney, Gregory A.; Dong, Aiping; Barsyte, Dalia; Kozieradzki, Ivona; Senisterra, Guillermo; Chau, Irene; Siarheyeva, Alena; Kireev, Dmitri B.; Jadhav, Ajit; Herold, J. Martin; Frye, Stephen V.; Arrowsmith, Cheryl H.; Brown, Peter J.; Simeonov, Anton; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">7950-7953</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">SAR exploration of the 2,4-diamino-6,7-dimethoxyquinazoline template led to the discovery of 8 (I) as a potent and selective G9a inhibitor.  A high resoln. X-ray crystal structure of the G9a-8 complex, the first cocrystal structure of G9a with a small mol. inhibitor, was obtained.  The cocrystal structure validated our binding hypothesis and will enable structure-based design of novel inhibitors.  8 Is a useful tool for investigating the biol. of G9a and its roles in chromatin remodeling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjWt5XsGXyj7Vg90H21EOLACvtfcHk0ljV0e0DKSpeyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlOiurzN&md5=2d7e34a8c4b5bf1cafd4e01ad2a86de1</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fjm901543m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901543m%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DQuinn%26aufirst%3DA.%2BM.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DDong%26aufirst%3DA.%2BP.%26aulast%3DBarsyte%26aufirst%3DD.%26aulast%3DKozieradzki%26aufirst%3DI.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DKireev%26aufirst%3DD.%2BB.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DHerold%26aufirst%3DJ.%2BM.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520a%25202%252C4-diamino-7-aminoalkoxyquinazoline%2520as%2520a%2520potent%2520and%2520selective%2520inhibitor%2520of%2520histone%2520lysine%2520methyltransferase%2520G9a%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D7950%26epage%3D7953%26doi%3D10.1021%2Fjm901543m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quinn, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siarheyeva, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norris, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kireev, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jadhav, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herold, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzen, W. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J. A.</span><span> </span><span class="NLM_article-title">Protein lysine methyltransferase G9a inhibitors: design, synthesis, and structure activity relationships of 2,4-diamino-7-aminoalkoxy-quinazolines</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">5844</span><span class="NLM_x">–</span> <span class="NLM_lpage">5857</span><span class="refDoi"> DOI: 10.1021/jm100478y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100478y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BC3cXos1ekurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=5844-5857&author=F.+Liuauthor=X.+Chenauthor=A.+Allali-Hassaniauthor=A.+M.+Quinnauthor=T.+J.+Wigleauthor=G.+A.+Wasneyauthor=A.+P.+Dongauthor=G.+Senisterraauthor=I.+Chauauthor=A.+Siarheyevaauthor=J.+L.+Norrisauthor=D.+B.+Kireevauthor=A.+Jadhavauthor=J.+M.+Heroldauthor=W.+P.+Janzenauthor=C.+H.+Arrowsmithauthor=S.+V.+Fryeauthor=P.+J.+Brownauthor=A.+Simeonovauthor=M.+Vedadiauthor=J.+A.+Jin&title=Protein+lysine+methyltransferase+G9a+inhibitors%3A+design%2C+synthesis%2C+and+structure+activity+relationships+of+2%2C4-diamino-7-aminoalkoxy-quinazolines&doi=10.1021%2Fjm100478y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Lysine Methyltransferase G9a Inhibitors: Design, Synthesis, and Structure Activity Relationships of 2,4-Diamino-7-aminoalkoxy-quinazolines</span></div><div class="casAuthors">Liu, Feng; Chen, Xin; Allali-Hassani, Abdellah; Quinn, Amy M.; Wigle, Tim J.; Wasney, Gregory A.; Dong, Aiping; Senisterra, Guillermo; Chau, Irene; Siarheyeva, Alena; Norris, Jacqueline L.; Kireev, Dmitri B.; Jadhav, Ajit; Herold, J. Martin; Janzen, William P.; Arrowsmith, Cheryl H.; Frye, Stephen V.; Brown, Peter J.; Simeonov, Anton; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5844-5857</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein lysine methyltransferase G9a, which catalyzes methylation of lysine 9 of histone H3 (H3K9) and lysine 373 (K373) of p53, is overexpressed in human cancers.  Genetic knockdown of G9a inhibits cancer cell growth; and the dimethylation of p53 K373 results in the inactivation of p53.  Initial SAR exploration of the 2,4-diamino-6,7-dimethoxyquinazoline template represented by I ( R1 = Bn; R2 = Me)(BIX01294), a selective small mol. inhibitor of G9a and GLP (G9a-like protein), led to the discovery of I [R1 = Me; R2 = (CH2)3NMe2](II)(UNC0224) as a potent G9a inhibitor with excellent selectivity.  A high resoln. X-ray crystal structure of the G9a-II complex, the first cocrystal structure of G9a with a small mol. inhibitor, was obtained.  On the basis of the structural insights revealed by this cocrystal structure, optimization of the 7-(dimethylamino)propoxy side chain of II resulted in the discovery of I [R1 = Me; R2 = (CH2)2O(CH2)2NMe2](UNC0321, Morrison Ki = 63 pM), which is the first G9a inhibitor with picomolar potency and the most potent G9a inhibitor to date.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgS0YxdNdNxLVg90H21EOLACvtfcHk0ljV0e0DKSpeyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXos1ekurk%253D&md5=8bd703c0a975a33bf9e5c019f1f95ee6</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fjm100478y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100478y%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DQuinn%26aufirst%3DA.%2BM.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DDong%26aufirst%3DA.%2BP.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DKireev%26aufirst%3DD.%2BB.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DHerold%26aufirst%3DJ.%2BM.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%2BA.%26atitle%3DProtein%2520lysine%2520methyltransferase%2520G9a%2520inhibitors%253A%2520design%252C%2520synthesis%252C%2520and%2520structure%2520activity%2520relationships%2520of%25202%252C4-diamino-7-aminoalkoxy-quinazolines%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D5844%26epage%3D5857%26doi%3D10.1021%2Fjm100478y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Chang, Y. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ganesh, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horton, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spannhoff, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shinkai, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Snyder, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, X. D.</span><span> </span><span class="NLM_article-title">Adding a lysine mimic in the design of potent inhibitors of histone lysine methyltransferases</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">400</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">7</span><span class="refDoi"> DOI: 10.1016/j.jmb.2010.04.048</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1016%2Fj.jmb.2010.04.048" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=20434463" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BC3cXntlaju7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=400&publication_year=2010&pages=1-7&author=Y.+Q.+Changauthor=T.+Ganeshauthor=J.+R.+Hortonauthor=A.+Spannhoffauthor=J.+Liuauthor=A.+M.+Sunauthor=X.+Zhangauthor=M.+T.+Bedfordauthor=Y.+C.+Shinkaiauthor=J.+P.+Snyderauthor=X.+D.+Cheng&title=Adding+a+lysine+mimic+in+the+design+of+potent+inhibitors+of+histone+lysine+methyltransferases&doi=10.1016%2Fj.jmb.2010.04.048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Adding a Lysine Mimic in the Design of Potent Inhibitors of Histone Lysine Methyltransferases</span></div><div class="casAuthors">Chang, Yanqi; Ganesh, Thota; Horton, John R.; Spannhoff, Astrid; Liu, Jin; Sun, Aiming; Zhang, Xing; Bedford, Mark T.; Shinkai, Yoichi; Snyder, James P.; Cheng, Xiaodong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">400</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-7</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Dynamic histone lysine methylation involves the activities of modifying enzymes (writers), enzymes removing modifications (erasers), and readers of the histone code.  One common feature of these activities is the recognition of lysines in methylated and unmethylated states, whether they are substrates, reaction products, or binding partners.  The authors applied the concept of adding a lysine mimic to an established inhibitor (BIX-01294) of histone H3 lysine 9 methyltransferases G9a and G9a-like protein by including a 5-aminopentyloxy moiety, which is inserted into the target lysine-binding channel and becomes methylated by G9a-like protein, albeit slowly.  The compd. enhances its potency in vitro and reduces cell toxicity in vivo.  The authors suggest that adding a lysine or methyl-lysine mimic should be considered in the design of small-mol. inhibitors for other methyl-lysine writers, erasers, and readers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsrNWhUZuRrLVg90H21EOLACvtfcHk0ljV0e0DKSpeyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXntlaju7c%253D&md5=7ec7ce81db069cd6c1c6637b3ad57007</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2010.04.048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2010.04.048%26sid%3Dliteratum%253Aachs%26aulast%3DChang%26aufirst%3DY.%2BQ.%26aulast%3DGanesh%26aufirst%3DT.%26aulast%3DHorton%26aufirst%3DJ.%2BR.%26aulast%3DSpannhoff%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DSun%26aufirst%3DA.%2BM.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DShinkai%26aufirst%3DY.%2BC.%26aulast%3DSnyder%26aufirst%3DJ.%2BP.%26aulast%3DCheng%26aufirst%3DX.%2BD.%26atitle%3DAdding%2520a%2520lysine%2520mimic%2520in%2520the%2520design%2520of%2520potent%2520inhibitors%2520of%2520histone%2520lysine%2520methyltransferases%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2010%26volume%3D400%26spage%3D1%26epage%3D7%26doi%3D10.1016%2Fj.jmb.2010.04.048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, Y. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herold, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yates, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Optimization of cellular activity of G9a inhibitors 7-aminoalkoxy-quinazolines</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">6139</span><span class="NLM_x">–</span> <span class="NLM_lpage">6150</span><span class="refDoi"> DOI: 10.1021/jm200903z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200903z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpvVWltLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=6139-6150&author=F.+Liuauthor=D.+Barsyte-Lovejoyauthor=A.+Allali-Hassaniauthor=Y.+L.+Heauthor=J.+M.+Heroldauthor=X.+Chenauthor=C.+M.+Yatesauthor=S.+V.+Fryeauthor=P.+J.+Brownauthor=J.+Huangauthor=M.+Vedadiauthor=C.+H.+Arrowsmithauthor=J.+Jin&title=Optimization+of+cellular+activity+of+G9a+inhibitors+7-aminoalkoxy-quinazolines&doi=10.1021%2Fjm200903z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of cellular activity of G9a inhibitors 7-aminoalkoxy-quinazolines</span></div><div class="casAuthors">Liu, Feng; Barsyte-Lovejoy, Dalia; Allali-Hassani, Abdellah; He, Yunlong; Herold, J. Martin; Chen, Xin; Yates, Christopher M.; Frye, Stephen V.; Brown, Peter J.; Huang, Jing; Vedadi, Masoud; Arrowsmith, Cheryl H.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">6139-6150</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein lysine methyltransferase G9a plays key roles in the transcriptional repression of a variety of genes via dimethylation of lysine 9 on histone H3 (H3K9me2) of chromatin as well as dimethylation of nonhistone proteins including tumor suppressor p53.  We previously reported the discovery of UNC0321, the most potent G9a inhibitor to date, via structure-based design and structure-activity relationship (SAR) exploration of the quinazoline scaffold represented by BIX01294.  Despite its very high in vitro potency, compd. UNC0321 lacks sufficient cellular potency.  The design and synthesis of several generations of new analogs aimed at improving cell membrane permeability while maintaining high in vitro potency resulted in the discovery of a no. of novel G9a inhibitors such as UNC0646 I and UNC0631 II with excellent potency in a variety of cell lines and excellent sepn. of functional potency vs. cell toxicity.  The design, synthesis, and cellular SAR of these potent G9a inhibitors are described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrC91yKB7S34rVg90H21EOLACvtfcHk0lh8PzGanXUPNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpvVWltLg%253D&md5=f3f54b91e6b24aa248498f9ff1fdb1ae</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Fjm200903z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200903z%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DHe%26aufirst%3DY.%2BL.%26aulast%3DHerold%26aufirst%3DJ.%2BM.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DYates%26aufirst%3DC.%2BM.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DOptimization%2520of%2520cellular%2520activity%2520of%2520G9a%2520inhibitors%25207-aminoalkoxy-quinazolines%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D6139%26epage%3D6150%26doi%3D10.1021%2Fjm200903z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rival-Gervier, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labrie, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DiMaggio, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siarheyeva, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tempel, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mangano, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simpson, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pattenden, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norris, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kireev, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tripathy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edwards, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzen, W. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petronis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellis, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">A chemical probe selectively inhibits G9a and GLP methyltransferase activity in cells</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">648</span><span class="NLM_x">–</span> <span class="NLM_lpage">648</span><span class="refDoi"> DOI: 10.1038/nchembio0911-648c</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1038%2Fnchembio0911-648c" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1Slu7rE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=648-648&author=M.+Vedadiauthor=D.+Barsyte-Lovejoyauthor=F.+Liuauthor=S.+Rival-Gervierauthor=A.+Allali-Hassaniauthor=V.+Labrieauthor=T.+J.+Wigleauthor=P.+A.+DiMaggioauthor=G.+A.+Wasneyauthor=A.+Siarheyevaauthor=A.+P.+Dongauthor=W.+Tempelauthor=S.+C.+Wangauthor=X.+Chenauthor=I.+Chauauthor=T.+J.+Manganoauthor=X.+P.+Huangauthor=C.+D.+Simpsonauthor=S.+G.+Pattendenauthor=J.+L.+Norrisauthor=D.+B.+Kireevauthor=A.+Tripathyauthor=A.+Edwardsauthor=B.+L.+Rothauthor=W.+P.+Janzenauthor=B.+A.+Garciaauthor=A.+Petronisauthor=J.+Ellisauthor=P.+J.+Brownauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=J.+Jin&title=A+chemical+probe+selectively+inhibits+G9a+and+GLP+methyltransferase+activity+in+cells&doi=10.1038%2Fnchembio0911-648c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">A chemical probe selectively inhibits G9a and GLP methyltransferase activity in cells [Erratum to document cited in CA155:320969]</span></div><div class="casAuthors">Vedadi, Masoud; Barsyte-Lovejoy, Dalia; Liu, Feng; Rival-Gervier, Sylvie; Allali-Hassani, Abdellah; Labrie, Viviane; Wigle, Tim J.; DiMaggio, Peter A.; Wasney, Gregory A.; Siarheyeva, Alena; Dong, Aiping; Tempel, Wolfram; Wang, Sun-Chong; Chen, Sin; Chau, Irene; Mangano, Thomas J.; Huang, Xi-ping; Simpson, Catherine D.; Pattenden, Samantha G.; Norris, Jacqueline L.; Kireev, Dmitri B.; Tripathy, Ashutosh; Edwards, Aled; Roth, Bryan L.; Janzen, William P.; Garcia, Benjamin A.; Petronis, Arturas; Ellis, James; Brown, Peter J.; Frye, Stephen V.; Arrowsmith, Cheryl H.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">648</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">On page 568, paragraph 3 of the results section entitled "UNCO638 is a potent and substrate-competitive inhibitor" contained a unit error; the cor. units are given and the HTML and PDF versions have been cor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkmoZ87Wwn3rVg90H21EOLACvtfcHk0lh8PzGanXUPNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1Slu7rE&md5=12d5b26948997d3eefa4f0aab7ea7aad</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1038%2Fnchembio0911-648c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio0911-648c%26sid%3Dliteratum%253Aachs%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DRival-Gervier%26aufirst%3DS.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DLabrie%26aufirst%3DV.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DDiMaggio%26aufirst%3DP.%2BA.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DDong%26aufirst%3DA.%2BP.%26aulast%3DTempel%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DS.%2BC.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DMangano%26aufirst%3DT.%2BJ.%26aulast%3DHuang%26aufirst%3DX.%2BP.%26aulast%3DSimpson%26aufirst%3DC.%2BD.%26aulast%3DPattenden%26aufirst%3DS.%2BG.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DKireev%26aufirst%3DD.%2BB.%26aulast%3DTripathy%26aufirst%3DA.%26aulast%3DEdwards%26aufirst%3DA.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26aulast%3DGarcia%26aufirst%3DB.%2BA.%26aulast%3DPetronis%26aufirst%3DA.%26aulast%3DEllis%26aufirst%3DJ.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DA%2520chemical%2520probe%2520selectively%2520inhibits%2520G9a%2520and%2520GLP%2520methyltransferase%2520activity%2520in%2520cells%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D648%26epage%3D648%26doi%3D10.1038%2Fnchembio0911-648c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiong, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korboukh, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzen, W. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Discovery of an in vivo chemical probe of the lysine methyltransferases G9a and GLP</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">8931</span><span class="NLM_x">–</span> <span class="NLM_lpage">8942</span><span class="refDoi"> DOI: 10.1021/jm401480r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401480r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsF2htrvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=8931-8942&author=F.+Liuauthor=D.+Barsyte-Lovejoyauthor=F.+L.+Liauthor=Y.+Xiongauthor=V.+Korboukhauthor=X.+P.+Huangauthor=A.+Allali-Hassaniauthor=W.+P.+Janzenauthor=B.+L.+Rothauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=M.+Vedadiauthor=J.+Jin&title=Discovery+of+an+in+vivo+chemical+probe+of+the+lysine+methyltransferases+G9a+and+GLP&doi=10.1021%2Fjm401480r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of an in vivo chemical probe of the lysine methyltransferases G9a and GLP</span></div><div class="casAuthors">Liu, Feng; Barsyte-Lovejoy, Dalia; Li, Fengling; Xiong, Yan; Korboukh, Victoria; Huang, Xi-Ping; Allali-Hassani, Abdellah; Janzen, William P.; Roth, Bryan L.; Frye, Stephen V.; Arrowsmith, Cheryl H.; Brown, Peter J.; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">8931-8942</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Among epigenetic writers, readers, and erasers, the lysine methyltransferases G9a and GLP, which catalyze mono- and dimethylation of histone H3 lysine 9 (H3K9me2) and nonhistone proteins, have been implicated in a variety of human diseases.  A toolkit of well-characterized chem. probes will allow biol. and disease hypotheses concerning these proteins to be tested in cell-based and animal models with high confidence.  We previously discovered potent and selective G9a/GLP inhibitors including the cellular chem. probe UNC0638, which displays an excellent sepn. of functional potency and cell toxicity.  However, this inhibitor is not suitable for animal studies due to its poor pharmacokinetic (PK) properties.  Here, we report the discovery of the first G9a and GLP in vivo chem. probe UNC0642, which not only maintains high in vitro and cellular potency, low cell toxicity, and excellent selectivity, but also displays improved in vivo PK properties, making it suitable for animal studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowOI3TSEz3ZbVg90H21EOLACvtfcHk0lgXn8zybQbJEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsF2htrvK&md5=933f0c49a938b4dd8146f6d09aa21e21</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fjm401480r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401480r%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DF.%2BL.%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DKorboukh%26aufirst%3DV.%26aulast%3DHuang%26aufirst%3DX.%2BP.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520an%2520in%2520vivo%2520chemical%2520probe%2520of%2520the%2520lysine%2520methyltransferases%2520G9a%2520and%2520GLP%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D8931%26epage%3D8942%26doi%3D10.1021%2Fjm401480r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Konze, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pattenden, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simon, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">A chemical tool for in vitro and in vivo precipitation of lysine methyltransferase G9a</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">549</span><span class="NLM_x">–</span> <span class="NLM_lpage">553</span><span class="refDoi"> DOI: 10.1002/cmdc.201300450</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1002%2Fcmdc.201300450" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=24443078" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BC2cXptFeiug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=549-553&author=K.+D.+Konzeauthor=S.+G.+Pattendenauthor=F.+Liuauthor=D.+Barsyte-Lovejoyauthor=F.+L.+Liauthor=J.+M.+Simonauthor=I.+J.+Davisauthor=M.+Vedadiauthor=J.+Jin&title=A+chemical+tool+for+in+vitro+and+in+vivo+precipitation+of+lysine+methyltransferase+G9a&doi=10.1002%2Fcmdc.201300450"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">A Chemical Tool for In Vitro and In Vivo Precipitation of Lysine Methyltransferase G9a</span></div><div class="casAuthors">Konze, Kyle D.; Pattenden, Samantha G.; Liu, Feng; Barsyte-Lovejoy, Dalia; Li, Fengling; Simon, Jeremy M.; Davis, Ian J.; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">549-553</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Here we report the design, synthesis, and biochem. characterization of a new chem. tool, UNC0965.  UNC0965 is a biotinylated version of our previously reported G9a chem. probe, UNC0638.  Importantly, UNC0965 maintains high in vitro potency and is cell penetrant.  The biotinylated tag of UNC0965 enables "chemipptn." of G9a from whole cell lysates.  Further, the cell penetrance of UNC0965 allowed us to explore the localization of G9a on chromatin both in vitro and in vivo through chem. inhibitor-based chromatin immunopptn. (chem-ChIP).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMnElEl-LrHbVg90H21EOLACvtfcHk0lgXn8zybQbJEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXptFeiug%253D%253D&md5=bbefd445671ed6ee5c22a80eefe61268</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201300450&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201300450%26sid%3Dliteratum%253Aachs%26aulast%3DKonze%26aufirst%3DK.%2BD.%26aulast%3DPattenden%26aufirst%3DS.%2BG.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DF.%2BL.%26aulast%3DSimon%26aufirst%3DJ.%2BM.%26aulast%3DDavis%26aufirst%3DI.%2BJ.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DA%2520chemical%2520tool%2520for%2520in%2520vitro%2520and%2520in%2520vivo%2520precipitation%2520of%2520lysine%2520methyltransferase%2520G9a%26jtitle%3DChemMedChem%26date%3D2014%26volume%3D9%26spage%3D549%26epage%3D553%26doi%3D10.1002%2Fcmdc.201300450" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">Sweis, R. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pliushchev, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maag, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petros, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soni, N. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tse, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michaelides, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiang, G. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pappano, W. N.</span><span> </span><span class="NLM_article-title">Discovery and development of potent and selective inhibitors of histone methyltransferase G9a</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">205</span><span class="NLM_x">–</span> <span class="NLM_lpage">209</span><span class="refDoi"> DOI: 10.1021/ml400496h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml400496h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFKmsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=205-209&author=R.+F.+Sweisauthor=M.+Pliushchevauthor=P.+J.+Brownauthor=J.+Guoauthor=F.+L.+Liauthor=D.+Maagauthor=A.+M.+Petrosauthor=N.+B.+Soniauthor=C.+Tseauthor=M.+Vedadiauthor=M.+R.+Michaelidesauthor=G.+G.+Chiangauthor=W.+N.+Pappano&title=Discovery+and+development+of+potent+and+selective+inhibitors+of+histone+methyltransferase+G9a&doi=10.1021%2Fml400496h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Development of Potent and Selective Inhibitors of Histone Methyltransferase G9a</span></div><div class="casAuthors">Sweis, Ramzi F.; Pliushchev, Marina; Brown, Peter J.; Guo, Jun; Li, Fengling; Maag, David; Petros, Andrew M.; Soni, Nirupama B.; Tse, Chris; Vedadi, Masoud; Michaelides, Michael R.; Chiang, Gary G.; Pappano, William N.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">205-209</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">G9a is a histone lysine methyltransferase responsible for the methylation of histone H3 lysine 9.  The discovery of A-366 arose from a unique diversity screening hit, which was optimized by incorporation of a propyl-pyrrolidine subunit to occupy the enzyme lysine channel.  A-366 (I) is a potent inhibitor of G9a (IC50: 3.3 nM) with greater than 1000-fold selectivity over 21 other methyltransferases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMvdTiUgx-oLVg90H21EOLACvtfcHk0lgXn8zybQbJEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFKmsA%253D%253D&md5=26affad72f2950510aa9f253e540d575</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Fml400496h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml400496h%26sid%3Dliteratum%253Aachs%26aulast%3DSweis%26aufirst%3DR.%2BF.%26aulast%3DPliushchev%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DGuo%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DF.%2BL.%26aulast%3DMaag%26aufirst%3DD.%26aulast%3DPetros%26aufirst%3DA.%2BM.%26aulast%3DSoni%26aufirst%3DN.%2BB.%26aulast%3DTse%26aufirst%3DC.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DMichaelides%26aufirst%3DM.%2BR.%26aulast%3DChiang%26aufirst%3DG.%2BG.%26aulast%3DPappano%26aufirst%3DW.%2BN.%26atitle%3DDiscovery%2520and%2520development%2520of%2520potent%2520and%2520selective%2520inhibitors%2520of%2520histone%2520methyltransferase%2520G9a%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D5%26spage%3D205%26epage%3D209%26doi%3D10.1021%2Fml400496h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group">Yuan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paulk, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kubicek, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kemp, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shamji, A. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wagner, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schreiber, S. L.</span><span> </span><span class="NLM_article-title">A small-molecule probe of the histone methyltransferase G9a induces cellular senescence in pancreatic adenocarcinoma</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">1152</span><span class="NLM_x">–</span> <span class="NLM_lpage">1157</span><span class="refDoi"> DOI: 10.1021/cb300139y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb300139y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=1152-1157&author=Y.+Yuanauthor=Q.+Wangauthor=J.+Paulkauthor=S.+Kubicekauthor=M.+M.+Kempauthor=D.+J.+Adamsauthor=A.+F.+Shamjiauthor=B.+K.+Wagnerauthor=S.+L.+Schreiber&title=A+small-molecule+probe+of+the+histone+methyltransferase+G9a+induces+cellular+senescence+in+pancreatic+adenocarcinoma&doi=10.1021%2Fcb300139y"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fcb300139y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb300139y%26sid%3Dliteratum%253Aachs%26aulast%3DYuan%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DKubicek%26aufirst%3DS.%26aulast%3DKemp%26aufirst%3DM.%2BM.%26aulast%3DAdams%26aufirst%3DD.%2BJ.%26aulast%3DShamji%26aufirst%3DA.%2BF.%26aulast%3DWagner%26aufirst%3DB.%2BK.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26atitle%3DA%2520small-molecule%2520probe%2520of%2520the%2520histone%2520methyltransferase%2520G9a%2520induces%2520cellular%2520senescence%2520in%2520pancreatic%2520adenocarcinoma%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2012%26volume%3D7%26spage%3D1152%26epage%3D1157%26doi%3D10.1021%2Fcb300139y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group">Devkota, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lohse, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Staerk, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berthelsen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clausen, R. P.</span><span> </span><span class="NLM_article-title">Analogues of the natural product sinefungin as inhibitors of EHMT1 and EHMT2</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">293</span><span class="NLM_x">–</span> <span class="NLM_lpage">297</span><span class="refDoi"> DOI: 10.1021/ml4002503</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml4002503" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1Oks7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=293-297&author=K.+Devkotaauthor=B.+Lohseauthor=Q.+Liuauthor=M.+W.+Wangauthor=D.+Staerkauthor=J.+Berthelsenauthor=R.+P.+Clausen&title=Analogues+of+the+natural+product+sinefungin+as+inhibitors+of+EHMT1+and+EHMT2&doi=10.1021%2Fml4002503"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Analogs of the Natural Product Sinefungin as Inhibitors of EHMT1 and EHMT2</span></div><div class="casAuthors">Devkota, Kanchan; Lohse, Brian; Liu, Qing; Wang, Ming-Wei; Staerk, Dan; Berthelsen, Jens; Clausen, Rasmus Praetorius</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">293-297</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of analogs of the natural product sinefungin lacking the amino acid moiety was synthesized and probed for their ability to inhibit EHMT1 and EHMT2.  This study led to inhibitors, e.g. I, of methyltransferase activity of EHMT1 and EHMT2 and it demonstrates that such analogs constitute an interesting scaffold to develop selective methyltransferase inhibitors.  Surprisingly, the inhibition was not competitive toward AdoMet.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLcJbLzR-Sy7Vg90H21EOLACvtfcHk0lghpsC5hzNw3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1Oks7g%253D&md5=d5992e33faf67b007932bb70c0bd8ed5</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Fml4002503&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml4002503%26sid%3Dliteratum%253Aachs%26aulast%3DDevkota%26aufirst%3DK.%26aulast%3DLohse%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DM.%2BW.%26aulast%3DStaerk%26aufirst%3DD.%26aulast%3DBerthelsen%26aufirst%3DJ.%26aulast%3DClausen%26aufirst%3DR.%2BP.%26atitle%3DAnalogues%2520of%2520the%2520natural%2520product%2520sinefungin%2520as%2520inhibitors%2520of%2520EHMT1%2520and%2520EHMT2%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D5%26spage%3D293%26epage%3D297%26doi%3D10.1021%2Fml4002503" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group">Kondengaden, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, L. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, M. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zang, L. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bataba, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, B. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, P. G.</span><span> </span><span class="NLM_article-title">Discovery of novel small molecule inhibitors of lysine methyltransferase G9a and their mechanism in leukemia cell lines</span> <span class="citation_source-journal">Eur. J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">122</span><span class="NLM_x">, </span> <span class="NLM_fpage">382</span><span class="NLM_x">–</span> <span class="NLM_lpage">393</span><span class="refDoi"> DOI: 10.1016/j.ejmech.2016.06.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1016%2Fj.ejmech.2016.06.028" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=122&publication_year=2016&pages=382-393&author=S.+M.+Kondengadenauthor=L.+F.+Luoauthor=K.+Huangauthor=M.+Y.+Zhuauthor=L.+L.+Zangauthor=E.+Batabaauthor=R.+L.+Wangauthor=C.+Luoauthor=B.+H.+Wangauthor=K.+K.+Liauthor=P.+G.+Wang&title=Discovery+of+novel+small+molecule+inhibitors+of+lysine+methyltransferase+G9a+and+their+mechanism+in+leukemia+cell+lines&doi=10.1016%2Fj.ejmech.2016.06.028"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2016.06.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2016.06.028%26sid%3Dliteratum%253Aachs%26aulast%3DKondengaden%26aufirst%3DS.%2BM.%26aulast%3DLuo%26aufirst%3DL.%2BF.%26aulast%3DHuang%26aufirst%3DK.%26aulast%3DZhu%26aufirst%3DM.%2BY.%26aulast%3DZang%26aufirst%3DL.%2BL.%26aulast%3DBataba%26aufirst%3DE.%26aulast%3DWang%26aufirst%3DR.%2BL.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DB.%2BH.%26aulast%3DLi%26aufirst%3DK.%2BK.%26aulast%3DWang%26aufirst%3DP.%2BG.%26atitle%3DDiscovery%2520of%2520novel%2520small%2520molecule%2520inhibitors%2520of%2520lysine%2520methyltransferase%2520G9a%2520and%2520their%2520mechanism%2520in%2520leukemia%2520cell%2520lines%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2016%26volume%3D122%26spage%3D382%26epage%3D393%26doi%3D10.1016%2Fj.ejmech.2016.06.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group">Chen, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Z. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, T. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, D. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, X. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">You, Q. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, X. K.</span><span> </span><span class="NLM_article-title">Discovery, design and synthesis of 6H-anthra[1,9-cd]isoxazol-6-one scaffold as G9a inhibitor through a combination of shape-based virtual screening and structure-based molecular modification</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">6102</span><span class="NLM_x">–</span> <span class="NLM_lpage">6108</span><span class="refDoi"> DOI: 10.1016/j.bmc.2016.09.071</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1016%2Fj.bmc.2016.09.071" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2016&pages=6102-6108&author=W.+L.+Chenauthor=Z.+H.+Wangauthor=T.+T.+Fengauthor=D.+D.+Liauthor=C.+H.+Wangauthor=X.+L.+Xuauthor=X.+J.+Zhangauthor=Q.+D.+Youauthor=X.+K.+Guo&title=Discovery%2C+design+and+synthesis+of+6H-anthra%5B1%2C9-cd%5Disoxazol-6-one+scaffold+as+G9a+inhibitor+through+a+combination+of+shape-based+virtual+screening+and+structure-based+molecular+modification&doi=10.1016%2Fj.bmc.2016.09.071"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2016.09.071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2016.09.071%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DW.%2BL.%26aulast%3DWang%26aufirst%3DZ.%2BH.%26aulast%3DFeng%26aufirst%3DT.%2BT.%26aulast%3DLi%26aufirst%3DD.%2BD.%26aulast%3DWang%26aufirst%3DC.%2BH.%26aulast%3DXu%26aufirst%3DX.%2BL.%26aulast%3DZhang%26aufirst%3DX.%2BJ.%26aulast%3DYou%26aufirst%3DQ.%2BD.%26aulast%3DGuo%26aufirst%3DX.%2BK.%26atitle%3DDiscovery%252C%2520design%2520and%2520synthesis%2520of%25206H-anthra%255B1%252C9-cd%255Disoxazol-6-one%2520scaffold%2520as%2520G9a%2520inhibitor%2520through%2520a%2520combination%2520of%2520shape-based%2520virtual%2520screening%2520and%2520structure-based%2520molecular%2520modification%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2016%26volume%3D24%26spage%3D6102%26epage%3D6108%26doi%3D10.1016%2Fj.bmc.2016.09.071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group">Lehnertz, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pabst, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Su, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krosl, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yung, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirschner, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosten, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Underhill, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hebert, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sauvageau, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Humphries, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rossi, F. M.</span><span> </span><span class="NLM_article-title">The methyltransferase G9a regulates HoxA9-dependent transcription in AML</span> <span class="citation_source-journal">Genes Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">317</span><span class="NLM_x">–</span> <span class="NLM_lpage">327</span><span class="refDoi"> DOI: 10.1101/gad.236794.113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1101%2Fgad.236794.113" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2014&pages=317-327&author=B.+Lehnertzauthor=C.+Pabstauthor=L.+Suauthor=M.+Millerauthor=F.+Liuauthor=L.+Yiauthor=R.+Zhangauthor=J.+Kroslauthor=E.+Yungauthor=J.+Kirschnerauthor=P.+Rostenauthor=T.+M.+Underhillauthor=J.+Jinauthor=J.+Hebertauthor=G.+Sauvageauauthor=R.+K.+Humphriesauthor=F.+M.+Rossi&title=The+methyltransferase+G9a+regulates+HoxA9-dependent+transcription+in+AML&doi=10.1101%2Fgad.236794.113"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1101%2Fgad.236794.113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.236794.113%26sid%3Dliteratum%253Aachs%26aulast%3DLehnertz%26aufirst%3DB.%26aulast%3DPabst%26aufirst%3DC.%26aulast%3DSu%26aufirst%3DL.%26aulast%3DMiller%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DYi%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DR.%26aulast%3DKrosl%26aufirst%3DJ.%26aulast%3DYung%26aufirst%3DE.%26aulast%3DKirschner%26aufirst%3DJ.%26aulast%3DRosten%26aufirst%3DP.%26aulast%3DUnderhill%26aufirst%3DT.%2BM.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DHebert%26aufirst%3DJ.%26aulast%3DSauvageau%26aufirst%3DG.%26aulast%3DHumphries%26aufirst%3DR.%2BK.%26aulast%3DRossi%26aufirst%3DF.%2BM.%26atitle%3DThe%2520methyltransferase%2520G9a%2520regulates%2520HoxA9-dependent%2520transcription%2520in%2520AML%26jtitle%3DGenes%2520Dev.%26date%3D2014%26volume%3D28%26spage%3D317%26epage%3D327%26doi%3D10.1101%2Fgad.236794.113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group">Bailey, S.</span><span> </span><span class="NLM_article-title">The CCP4 suite - programs for protein crystallography</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">760</span><span class="NLM_x">–</span> <span class="NLM_lpage">763</span><span class="refDoi"> DOI: 10.1107/S0907444994003112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1107%2FS0907444994003112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=15299374" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=1994&pages=760-763&author=S.+Bailey&title=The+CCP4+suite+-+programs+for+protein+crystallography&doi=10.1107%2FS0907444994003112"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1107%2FS0907444994003112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444994003112%26sid%3Dliteratum%253Aachs%26aulast%3DBailey%26aufirst%3DS.%26atitle%3DThe%2520CCP4%2520suite%2520-%2520programs%2520for%2520protein%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1994%26volume%3D50%26spage%3D760%26epage%3D763%26doi%3D10.1107%2FS0907444994003112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group">McCoy, A. J.</span><span> </span><span class="NLM_article-title">Solving structures of protein complexes by molecular replacement with Phaser</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">63</span><span class="NLM_x">, </span> <span class="NLM_fpage">32</span><span class="NLM_x">–</span> <span class="NLM_lpage">41</span><span class="refDoi"> DOI: 10.1107/S0907444906045975</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1107%2FS0907444906045975" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=17164524" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtlSqu77F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2007&pages=32-41&author=A.+J.+McCoy&title=Solving+structures+of+protein+complexes+by+molecular+replacement+with+Phaser&doi=10.1107%2FS0907444906045975"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Solving structures of protein complexes by molecular replacement with Phaser</span></div><div class="casAuthors">McCoy, Airlie J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">32-41</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Mol. replacement (MR) generally becomes more difficult as the no. of components in the asym. unit requiring sep. MR models (i.e. the dimensionality of the search) increases.  When the proportion of the total scattering contributed by each search component is small, the signal in the search for each component in isolation is weak or non-existent.  Maximum-likelihood MR functions enable complex asym. units to be built up from individual components with a 'tree search with pruning' approach.  This method, as implemented in the automated search procedure of the program Phaser, has been very successful in solving many previously intractable MR problems.  However, there are a no. of cases in which the automated search procedure of Phaser is suboptimal or encounters difficulties.  These include cases where there are a large no. of copies of the same component in the asym. unit or where the components of the asym. unit have greatly varying B factors.  Two case studies are presented to illustrate how Phaser can be used to best advantage in the std. 'automated MR' mode and two case studies are used to show how to modify the automated search strategy for problematic cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAvs9W8hQ3l7Vg90H21EOLACvtfcHk0lhVSQ_Dzkc6Rg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtlSqu77F&md5=323a5297bdf622ebaf398a87c39dc794</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1107%2FS0907444906045975&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444906045975%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26atitle%3DSolving%2520structures%2520of%2520protein%2520complexes%2520by%2520molecular%2520replacement%2520with%2520Phaser%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2007%26volume%3D63%26spage%3D32%26epage%3D41%26doi%3D10.1107%2FS0907444906045975" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group">Murshudov, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vagin, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodson, E. J.</span><span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">240</span><span class="NLM_x">–</span> <span class="NLM_lpage">255</span><span class="refDoi"> DOI: 10.1107/S0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method&doi=10.1107%2FS0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0lhVSQ_Dzkc6Rg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255%26doi%3D10.1107%2FS0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Afonine, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunkoczi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, I. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Echols, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hung, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapral, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moriarty, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oeffner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terwilliger, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zwart, P. H.</span><span> </span><span class="NLM_article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">213</span><span class="NLM_x">–</span> <span class="NLM_lpage">221</span><span class="refDoi"> DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunkocziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.+W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+a+comprehensive+Python-based+system+for+macromolecular+structure+solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0lhVSQ_Dzkc6Rg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunkoczi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.%2BW.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520a%2520comprehensive%2520Python-based%2520system%2520for%2520macromolecular%2520structure%2520solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group">Afonine, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Echols, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moriarty, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mustyakimov, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terwilliger, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Urzhumtsev, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zwart, P. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, P. D.</span><span> </span><span class="NLM_article-title">Towards automated crystallographic structure refinement with phenix.refine</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">68</span><span class="NLM_x">, </span> <span class="NLM_fpage">352</span><span class="NLM_x">–</span> <span class="NLM_lpage">367</span><span class="refDoi"> DOI: 10.1107/S0907444912001308</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1107%2FS0907444912001308" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=22505256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BC38Xlt1Ggu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2012&pages=352-367&author=P.+V.+Afonineauthor=R.+W.+Grosse-Kunstleveauthor=N.+Echolsauthor=J.+J.+Headdauthor=N.+W.+Moriartyauthor=M.+Mustyakimovauthor=T.+C.+Terwilligerauthor=A.+Urzhumtsevauthor=P.+H.+Zwartauthor=P.+D.+Adams&title=Towards+automated+crystallographic+structure+refinement+with+phenix.refine&doi=10.1107%2FS0907444912001308"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Towards automated crystallographic structure refinement with phenix.refine</span></div><div class="casAuthors">Afonine, Pavel V.; Grosse-Kunstleve, Ralf W.; Echols, Nathaniel; Headd, Jeffrey J.; Moriarty, Nigel W.; Mustyakimov, Marat; Terwilliger, Thomas C.; Urzhumtsev, Alexandre; Zwart, Peter H.; Adams, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">352-367</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phenix.refine is a program within the PHENIX package that supports crystallog. structure refinement against exptl. data with a wide range of upper resoln. limits using a large repertoire of model parameterizations.  It has several automation features and is also highly flexible.  Several hundred parameters enable extensive customizations for complex use cases.  Multiple user-defined refinement strategies can be applied to specific parts of the model in a single refinement run.  An intuitive graphical user interface is available to guide novice users and to assist advanced users in managing refinement projects.  X-ray or neutron diffraction data can be used sep. or jointly in refinement. phenix.refine is tightly integrated into the PHENIX suite, where it serves as a crit. component in automated model building, final structure refinement, structure validation and deposition to the wwPDB.  This paper presents an overview of the major phenix.refine features, with extensive literature refs. for readers interested in more detailed discussions of the methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnhn9QwIEkTbVg90H21EOLACvtfcHk0lhKtPwrChzjAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xlt1Ggu7c%253D&md5=1cc7da2cea396f1a75646ea709140cf1</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1107%2FS0907444912001308&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444912001308%26sid%3Dliteratum%253Aachs%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DMustyakimov%26aufirst%3DM.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DUrzhumtsev%26aufirst%3DA.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26atitle%3DTowards%2520automated%2520crystallographic%2520structure%2520refinement%2520with%2520phenix.refine%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2012%26volume%3D68%26spage%3D352%26epage%3D367%26doi%3D10.1107%2FS0907444912001308" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lohkamp, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Features and development of Coot</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">486</span><span class="NLM_x">–</span> <span class="NLM_lpage">501</span><span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lhKtPwrChzjAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group">Davis, I. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, D. C.</span><span> </span><span class="NLM_article-title">MolProbity: structure validation and all-atom contact analysis for nucleic acids and their complexes</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">W615</span><span class="NLM_x">–</span> <span class="NLM_lpage">W619</span><span class="refDoi"> DOI: 10.1093/nar/gkh398</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1093%2Fnar%2Fgkh398" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=15215462" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BD2cXlvFKmsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2004&pages=W615-W619&author=I.+W.+Davisauthor=L.+W.+Murrayauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=MolProbity%3A+structure+validation+and+all-atom+contact+analysis+for+nucleic+acids+and+their+complexes&doi=10.1093%2Fnar%2Fgkh398"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">MolPROBITY: Structure validation and all-atom contact analysis for nucleic acids and their complexes</span></div><div class="casAuthors">Davis, Ian W.; Murray, Laura Weston; Richardson, Jane S.; Richardson, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">Web Server</span>),
    <span class="NLM_cas:pages">W615-W619</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">MolProbity is a general-purpose web service offering quality validation for three-dimensional (3D) structures of proteins, nucleic acids and complexes.  It provides detailed all-atom contact anal. of any steric problems within the mols. and can calc. and display the H-bond and van der Waals contacts in the interfaces between components.  An integral step in the process is the addn. and full optimization of all hydrogen atoms, both polar and nonpolar.  The results are reported in multiple forms: as overall numeric scores, as lists, as downloadable PDB and graphics files, and most notably as informative, manipulable 3D kinemage graphics shown online in the KING viewer.  This service is available free to all users at http://kinemage.biochem.duke.edu.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9I64e-HQbnbVg90H21EOLACvtfcHk0lhKtPwrChzjAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXlvFKmsLY%253D&md5=2050f7acf49743addde69c5ce9c9e2e4</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkh398&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkh398%26sid%3Dliteratum%253Aachs%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DMurray%26aufirst%3DL.%2BW.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMolProbity%253A%2520structure%2520validation%2520and%2520all-atom%2520contact%2520analysis%2520for%2520nucleic%2520acids%2520and%2520their%2520complexes%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2004%26volume%3D32%26spage%3DW615%26epage%3DW619%26doi%3D10.1093%2Fnar%2Fgkh398" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Min, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lunin, V. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Antoshenko, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dombrovski, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campagna-Slater, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plotnikov, A. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span> </span><span class="NLM_article-title">Structural biology of human H3K9 methyltransferases</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">e8570</span><span class="refDoi"> DOI: 10.1371/journal.pone.0008570</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1371%2Fjournal.pone.0008570" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=20084102" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A280%3ADC%252BC3c%252FjtlGgsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=e8570&author=H.+Wuauthor=J.+R.+Minauthor=V.+V.+Luninauthor=T.+Antoshenkoauthor=L.+Dombrovskiauthor=H.+Zengauthor=A.+Allali-Hassaniauthor=V.+Campagna-Slaterauthor=M.+Vedadiauthor=C.+H.+Arrowsmithauthor=A.+N.+Plotnikovauthor=M.+Schapira&title=Structural+biology+of+human+H3K9+methyltransferases&doi=10.1371%2Fjournal.pone.0008570"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Structural biology of human H3K9 methyltransferases</span></div><div class="casAuthors">Wu Hong; Min Jinrong; Lunin Vladimir V; Antoshenko Tatiana; Dombrovski Ludmila; Zeng Hong; Allali-Hassani Abdellah; Campagna-Slater Valerie; Vedadi Masoud; Arrowsmith Cheryl H; Plotnikov Alexander N; Schapira Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">PloS one</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">e8570</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">SET domain methyltransferases deposit methyl marks on specific histone tail lysine residues and play a major role in epigenetic regulation of gene transcription.  We solved the structures of the catalytic domains of GLP, G9a, Suv39H2 and PRDM2, four of the eight known human H3K9 methyltransferases in their apo conformation or in complex with the methyl donating cofactor, and peptide substrates.  We analyzed the structural determinants for methylation state specificity, and designed a G9a mutant able to tri-methylate H3K9.  We show that the I-SET domain acts as a rigid docking platform, while induced-fit of the Post-SET domain is necessary to achieve a catalytically competent conformation.  We also propose a model where long-range electrostatics bring enzyme and histone substrate together, while the presence of an arginine upstream of the target lysine is critical for binding and specificity.  ENHANCED VERSION: This article can also be viewed as an enhanced version in which the text of the article is integrated with interactive 3D representations and animated transitions.  Please note that a web plugin is required to access this enhanced functionality.  Instructions for the installation and use of the web plugin are available in Text S1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTAEo3Y9d9dncRAV4D_Ins_fW6udTcc2eZ2F0VdyeoptLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3c%252FjtlGgsA%253D%253D&md5=2649472b1401c69035f99e0ca20822a3</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0008570&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0008570%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DH.%26aulast%3DMin%26aufirst%3DJ.%2BR.%26aulast%3DLunin%26aufirst%3DV.%2BV.%26aulast%3DAntoshenko%26aufirst%3DT.%26aulast%3DDombrovski%26aufirst%3DL.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DCampagna-Slater%26aufirst%3DV.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DPlotnikov%26aufirst%3DA.%2BN.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DStructural%2520biology%2520of%2520human%2520H3K9%2520methyltransferases%26jtitle%3DPLoS%2520One%26date%3D2010%26volume%3D5%26spage%3De8570%26doi%3D10.1371%2Fjournal.pone.0008570" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group">Otwinowski, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minor, W.</span><span> </span><span class="NLM_article-title">Processing of X-ray diffraction data collected in oscillation mode</span> <span class="citation_source-journal">Methods Enzymol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">276</span><span class="NLM_x">, </span> <span class="NLM_fpage">307</span><span class="NLM_x">–</span> <span class="NLM_lpage">326</span><span class="refDoi"> DOI: 10.1016/S0076-6879(97)76066-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=27754618" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=1997&pages=307-326&author=Z.+Otwinowskiauthor=W.+Minor&title=Processing+of+X-ray+diffraction+data+collected+in+oscillation+mode&doi=10.1016%2FS0076-6879%2897%2976066-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Processing of x-ray diffraction data collected in oscillation mode</span></div><div class="casAuthors">Otwinowski, Zbyszek; Minor, Wladek</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">Macromolecular Crystallography, Part A</span>),
    <span class="NLM_cas:pages">307-326</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Macromol. crystallog. is an iterative process.  Rarely do the first crystals provide all the necessary data to solve the biol. problem being studied.  Each step benefits from experience learned in previous steps.  To monitor the progress, the HKL package provides 2 tools: (1) statistics, both weighted (χ2) and unweighted (R-merge), are provided, and the Bayesian reasoning and multicomponent error model facilitates obtaining the proper error ests. and (2) visualization of the process plays a double role by helping the operator to confirm that the process of data redn., including the resulting statistics, is correct, and allowing one to evaluate problems for which there are no good statistical criteria.  Visualization also provides confidence that the point of diminishing returns in data collection and redn. has been reached.  At that point, the effort should be directed to solving the structure.  The methods presented here have been applied to solve a large variety of problems, from inorg. mols. with 5 Å unit cell to rotavirus of 700 Å diam. crystd. in 700 × 1000 × 1400 Å cell.  Overall quality of the method was tested by many researchers by successful application of the programs to MAD structure detns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA3fvBOHqhhrVg90H21EOLACvtfcHk0lj8wvcmuIAN3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D&md5=c9536971d4e32cc35352c40fb9368131</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DMinor%26aufirst%3DW.%26atitle%3DProcessing%2520of%2520X-ray%2520diffraction%2520data%2520collected%2520in%2520oscillation%2520mode%26jtitle%3DMethods%2520Enzymol.%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D326%26doi%3D10.1016%2FS0076-6879%2897%2976066-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group">Vagin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teplyakov, A.</span><span> </span><span class="NLM_article-title">MOLREP: an automated program for molecular replacement</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">1022</span><span class="NLM_x">–</span> <span class="NLM_lpage">1025</span><span class="refDoi"> DOI: 10.1107/S0021889897006766</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1107%2FS0021889897006766" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1997&pages=1022-1025&author=A.+Vaginauthor=A.+Teplyakov&title=MOLREP%3A+an+automated+program+for+molecular+replacement&doi=10.1107%2FS0021889897006766"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">MOLREP: an automated program for molecular replacement</span></div><div class="casAuthors">Vagin, Alexei; Teplyakov, Alexei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1022-1025</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard International Publishers Ltd.</span>)
        </div><div class="casAbstract">MOLREP is an automated program for mol. replacement which uses effective new approaches in data processing and rotational and translational searching.  These include an automatic choice of all parameters, scaling by Patterson origin peaks and soft resoln. cutoff.  One of the cornerstones of the program is an original full-symmetry translation function combined with a packing function.  Information from the model already placed in the cell is incorporated in both translation and packing functions.  A no. of tests using exptl. data proved the ability of the program to find the correct soln. in difficult cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkGXdZ-nTKxLVg90H21EOLACvtfcHk0lj8wvcmuIAN3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D&md5=96af64aedd8a8f018c3c18b57a6b4b21</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1107%2FS0021889897006766&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889897006766%26sid%3Dliteratum%253Aachs%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DTeplyakov%26aufirst%3DA.%26atitle%3DMOLREP%253A%2520an%2520automated%2520program%2520for%2520molecular%2520replacement%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D1997%26volume%3D30%26spage%3D1022%26epage%3D1025%26doi%3D10.1107%2FS0021889897006766" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 31 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">M. Ramya Chandar Charles, Mu-Chun Li, Hsing-Pang Hsieh, <span class="NLM_string-name hlFld-ContribAuthor">Mohane Selvaraj Coumar</span>. </span><span class="cited-content_cbyCitation_article-title">Mimicking H3 Substrate Arginine in the Design of G9a Lysine Methyltransferase Inhibitors for Cancer Therapy: A Computational Study for Structure-Based Drug Design. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2021,</strong> <em>6 </em>
                                    (9)
                                     , 6100-6111. <a href="https://doi.org/10.1021/acsomega.0c04710" title="DOI URL">https://doi.org/10.1021/acsomega.0c04710</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.0c04710&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.0c04710%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DMimicking%252BH3%252BSubstrate%252BArginine%252Bin%252Bthe%252BDesign%252Bof%252BG9a%252BLysine%252BMethyltransferase%252BInhibitors%252Bfor%252BCancer%252BTherapy%25253A%252BA%252BComputational%252BStudy%252Bfor%252BStructure-Based%252BDrug%252BDesign%26aulast%3DChandar%2BCharles%26aufirst%3DM.%2BRamya%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D24092020%26date%3D09022021%26date%3D22022021%26volume%3D6%26issue%3D9%26spage%3D6100%26epage%3D6111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="NLM_string-name hlFld-ContribAuthor">Matthew D. Lloyd</span>. </span><span class="cited-content_cbyCitation_article-title">High-Throughput Screening for the Discovery of Enzyme Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (19)
                                     , 10742-10772. <a href="https://doi.org/10.1021/acs.jmedchem.0c00523" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00523</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00523&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00523%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DHigh-Throughput%252BScreening%252Bfor%252Bthe%252BDiscovery%252Bof%252BEnzyme%252BInhibitors%26aulast%3DLloyd%26aufirst%3DMatthew%2BD.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D30032020%26date%3D10062020%26date%3D20052020%26volume%3D63%26issue%3D19%26spage%3D10742%26epage%3D10772" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Ciro Milite, Alessandra Feoli, John R. Horton, Donatella Rescigno, Alessandra Cipriano, Vincenzo Pisapia, Monica Viviano, Giacomo Pepe, Giorgio Amendola, Ettore Novellino, Sandro Cosconati, Xiaodong Cheng, Sabrina Castellano, <span class="NLM_string-name hlFld-ContribAuthor">Gianluca Sbardella</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of a Novel Chemotype of Histone Lysine Methyltransferase EHMT1/2 (GLP/G9a) Inhibitors: Rational Design, Synthesis, Biological Evaluation, and Co-crystal Structure. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (5)
                                     , 2666-2689. <a href="https://doi.org/10.1021/acs.jmedchem.8b02008" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b02008</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b02008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b02008%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Ba%252BNovel%252BChemotype%252Bof%252BHistone%252BLysine%252BMethyltransferase%252BEHMT1%25252F2%252B%252528GLP%25252FG9a%252529%252BInhibitors%25253A%252BRational%252BDesign%25252C%252BSynthesis%25252C%252BBiological%252BEvaluation%25252C%252Band%252BCo-crystal%252BStructure%26aulast%3DMilite%26aufirst%3DCiro%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D21122018%26date%3D26022019%26date%3D12022019%26volume%3D62%26issue%3D5%26spage%3D2666%26epage%3D2689" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">H. Ümit  Kaniskan</span>, <span class="hlFld-ContribAuthor ">Michael L.  Martini</span>, and <span class="hlFld-ContribAuthor ">Jian  Jin</span>  . </span><span class="cited-content_cbyCitation_article-title">Inhibitors of Protein Methyltransferases and Demethylases. </span><span class="cited-content_cbyCitation_journal-name">Chemical Reviews</span><span> <strong>2018,</strong> <em>118 </em>
                                    (3)
                                     , 989-1068. <a href="https://doi.org/10.1021/acs.chemrev.6b00801" title="DOI URL">https://doi.org/10.1021/acs.chemrev.6b00801</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.chemrev.6b00801&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.chemrev.6b00801%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Reviews%26atitle%3DInhibitors%252Bof%252BProtein%252BMethyltransferases%252Band%252BDemethylases%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259Cmit%26date%3D2018%26date%3D2017%26date%3D2016%26date%3D30112016%26date%3D24032017%26date%3D14022018%26volume%3D118%26issue%3D3%26spage%3D989%26epage%3D1068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kamakoti P.  Bhat</span>, <span class="hlFld-ContribAuthor ">H.  Ümit Kaniskan</span>, <span class="hlFld-ContribAuthor ">Jian  Jin</span>, <span class="hlFld-ContribAuthor ">Or  Gozani</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetics and beyond: targeting writers of protein lysine methylation to treat disease. </span><span class="cited-content_cbyCitation_journal-name">Nature Reviews Drug Discovery</span><span> <strong>2021,</strong> <em>20 </em>
                                    (4)
                                     , 265-286. <a href="https://doi.org/10.1038/s41573-020-00108-x" title="DOI URL">https://doi.org/10.1038/s41573-020-00108-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41573-020-00108-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41573-020-00108-x%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Reviews%2520Drug%2520Discovery%26atitle%3DEpigenetics%252Band%252Bbeyond%25253A%252Btargeting%252Bwriters%252Bof%252Bprotein%252Blysine%252Bmethylation%252Bto%252Btreat%252Bdisease%26aulast%3DBhat%26aufirst%3DKamakoti%2BP.%26date%3D2021%26date%3D2021%26volume%3D20%26issue%3D4%26spage%3D265%26epage%3D286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yuki  Mizukawa</span>, <span class="hlFld-ContribAuthor ">Mayumi  Ikegami-Kawai</span>, <span class="hlFld-ContribAuthor ">Masako  Horiuchi</span>, <span class="hlFld-ContribAuthor ">Marcel  Kaiser</span>, <span class="hlFld-ContribAuthor ">Masayoshi  Kojima</span>, <span class="hlFld-ContribAuthor ">Seiki  Sakanoue</span>, <span class="hlFld-ContribAuthor ">Seiya  Miyagi</span>, <span class="hlFld-ContribAuthor ">Christian  Nanga Chick</span>, <span class="hlFld-ContribAuthor ">Hiroyuki  Togashi</span>, <span class="hlFld-ContribAuthor ">Masayoshi  Tsubuki</span>, <span class="hlFld-ContribAuthor ">Masataka  Ihara</span>, <span class="hlFld-ContribAuthor ">Toyonobu  Usuki</span>, <span class="hlFld-ContribAuthor ">Isamu  Itoh</span>. </span><span class="cited-content_cbyCitation_article-title">Quest for a potent antimalarial drug lead: Synthesis and evaluation of 6,7-dimethoxyquinazoline-2,4-diamines. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2021,</strong> <em>33 </em>, 116018. <a href="https://doi.org/10.1016/j.bmc.2021.116018" title="DOI URL">https://doi.org/10.1016/j.bmc.2021.116018</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2021.116018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2021.116018%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DQuest%252Bfor%252Ba%252Bpotent%252Bantimalarial%252Bdrug%252Blead%25253A%252BSynthesis%252Band%252Bevaluation%252Bof%252B6%25252C7-dimethoxyquinazoline-2%25252C4-diamines%26aulast%3DMizukawa%26aufirst%3DYuki%26date%3D2021%26volume%3D33%26spage%3D116018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">John W.  Wiley</span>, <span class="hlFld-ContribAuthor ">Ye  Zong</span>, <span class="hlFld-ContribAuthor ">Gen  Zheng</span>, <span class="hlFld-ContribAuthor ">Shengtao  Zhu</span>, <span class="hlFld-ContribAuthor ">Shuangsong  Hong</span>. </span><span class="cited-content_cbyCitation_article-title">Histone H3K9 methylation regulates chronic stress and IL‐6–induced colon epithelial permeability and visceral pain. </span><span class="cited-content_cbyCitation_journal-name">Neurogastroenterology & Motility</span><span> <strong>2020,</strong> <em>32 </em>
                                    (12)
                                     <a href="https://doi.org/10.1111/nmo.13941" title="DOI URL">https://doi.org/10.1111/nmo.13941</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1111/nmo.13941&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1111%2Fnmo.13941%26sid%3Dliteratum%253Aachs%26jtitle%3DNeurogastroenterology%2520%2526%2520Motility%26atitle%3DHistone%252BH3K9%252Bmethylation%252Bregulates%252Bchronic%252Bstress%252Band%252BIL%2525E2%252580%2525906%2525E2%252580%252593induced%252Bcolon%252Bepithelial%252Bpermeability%252Band%252Bvisceral%252Bpain%26aulast%3DWiley%26aufirst%3DJohn%2BW.%26date%3D2020%26date%3D2020%26volume%3D32%26issue%3D12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rajendra P.  Pangeni</span>, <span class="hlFld-ContribAuthor ">Lu  Yang</span>, <span class="hlFld-ContribAuthor ">Keqiang  Zhang</span>, <span class="hlFld-ContribAuthor ">Jinhui  Wang</span>, <span class="hlFld-ContribAuthor ">Wendong  Li</span>, <span class="hlFld-ContribAuthor ">Chao  Guo</span>, <span class="hlFld-ContribAuthor ">Xinwei  Yun</span>, <span class="hlFld-ContribAuthor ">Ting  Sun</span>, <span class="hlFld-ContribAuthor ">Jami  Wang</span>, <span class="hlFld-ContribAuthor ">Dan J.  Raz</span>. </span><span class="cited-content_cbyCitation_article-title">G9a regulates tumorigenicity and stemness through genome-wide DNA methylation reprogramming in non-small cell lung cancer. </span><span class="cited-content_cbyCitation_journal-name">Clinical Epigenetics</span><span> <strong>2020,</strong> <em>12 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13148-020-00879-5" title="DOI URL">https://doi.org/10.1186/s13148-020-00879-5</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13148-020-00879-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13148-020-00879-5%26sid%3Dliteratum%253Aachs%26jtitle%3DClinical%2520Epigenetics%26atitle%3DG9a%252Bregulates%252Btumorigenicity%252Band%252Bstemness%252Bthrough%252Bgenome-wide%252BDNA%252Bmethylation%252Breprogramming%252Bin%252Bnon-small%252Bcell%252Blung%252Bcancer%26aulast%3DPangeni%26aufirst%3DRajendra%2BP.%26date%3D2020%26date%3D2020%26volume%3D12%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dhanusha A.  Nalawansha</span>, <span class="hlFld-ContribAuthor ">Craig M.  Crews</span>. </span><span class="cited-content_cbyCitation_article-title">PROTACs: An Emerging Therapeutic Modality in Precision Medicine. </span><span class="cited-content_cbyCitation_journal-name">Cell Chemical Biology</span><span> <strong>2020,</strong> <em>27 </em>
                                    (8)
                                     , 998-1014. <a href="https://doi.org/10.1016/j.chembiol.2020.07.020" title="DOI URL">https://doi.org/10.1016/j.chembiol.2020.07.020</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chembiol.2020.07.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chembiol.2020.07.020%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Chemical%2520Biology%26atitle%3DPROTACs%25253A%252BAn%252BEmerging%252BTherapeutic%252BModality%252Bin%252BPrecision%252BMedicine%26aulast%3DNalawansha%26aufirst%3DDhanusha%2BA.%26date%3D2020%26volume%3D27%26issue%3D8%26spage%3D998%26epage%3D1014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mingyue  Zhao</span>, <span class="hlFld-ContribAuthor ">Wei  Wang</span>, <span class="hlFld-ContribAuthor ">Zhijun  Jiang</span>, <span class="hlFld-ContribAuthor ">Zemeng  Zhu</span>, <span class="hlFld-ContribAuthor ">Dexiang  Liu</span>, <span class="hlFld-ContribAuthor ">Fang  Pan</span>. </span><span class="cited-content_cbyCitation_article-title">Long-Term Effect of Post-traumatic Stress in Adolescence on Dendrite Development and H3K9me2/BDNF Expression in Male Rat Hippocampus and Prefrontal Cortex. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Cell and Developmental Biology</span><span> <strong>2020,</strong> <em>8 </em><a href="https://doi.org/10.3389/fcell.2020.00682" title="DOI URL">https://doi.org/10.3389/fcell.2020.00682</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fcell.2020.00682&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffcell.2020.00682%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Cell%2520and%2520Developmental%2520Biology%26atitle%3DLong-Term%252BEffect%252Bof%252BPost-traumatic%252BStress%252Bin%252BAdolescence%252Bon%252BDendrite%252BDevelopment%252Band%252BH3K9me2%25252FBDNF%252BExpression%252Bin%252BMale%252BRat%252BHippocampus%252Band%252BPrefrontal%252BCortex%26aulast%3DZhao%26aufirst%3DMingyue%26date%3D2020%26date%3D2020%26volume%3D8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Laia  Richart</span>, <span class="hlFld-ContribAuthor ">Raphaël  Margueron</span>. </span><span class="cited-content_cbyCitation_article-title">Drugging histone methyltransferases in cancer. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2020,</strong> <em>56 </em>, 51-62. <a href="https://doi.org/10.1016/j.cbpa.2019.11.009" title="DOI URL">https://doi.org/10.1016/j.cbpa.2019.11.009</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2019.11.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2019.11.009%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DDrugging%252Bhistone%252Bmethyltransferases%252Bin%252Bcancer%26aulast%3DRichart%26aufirst%3DLaia%26date%3D2020%26volume%3D56%26spage%3D51%26epage%3D62" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ning-Ning  Dang</span>, <span class="hlFld-ContribAuthor ">Jing  Jiao</span>, <span class="hlFld-ContribAuthor ">Xianguang  Meng</span>, <span class="hlFld-ContribAuthor ">Yunhe  An</span>, <span class="hlFld-ContribAuthor ">Chen  Han</span>, <span class="hlFld-ContribAuthor ">Shuhong  Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Abnormal overexpression of G9a in melanoma cells promotes cancer progression via upregulation of the Notch1 signaling pathway. </span><span class="cited-content_cbyCitation_journal-name">Aging</span><span> <strong>2020,</strong> <em>12 </em>
                                    (3)
                                     , 2393-2407. <a href="https://doi.org/10.18632/aging.102750" title="DOI URL">https://doi.org/10.18632/aging.102750</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.18632/aging.102750&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.18632%2Faging.102750%26sid%3Dliteratum%253Aachs%26jtitle%3DAging%26atitle%3DAbnormal%252Boverexpression%252Bof%252BG9a%252Bin%252Bmelanoma%252Bcells%252Bpromotes%252Bcancer%252Bprogression%252Bvia%252Bupregulation%252Bof%252Bthe%252BNotch1%252Bsignaling%252Bpathway%26aulast%3DDang%26aufirst%3DNing-Ning%26date%3D2020%26date%3D2020%26volume%3D12%26issue%3D3%26spage%3D2393%26epage%3D2407" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">P. Ann  Boriack‐Sjodin</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetic Proteins as Emerging Drug Targets. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 423-448. <a href="https://doi.org/10.1002/9781118681121.ch18" title="DOI URL">https://doi.org/10.1002/9781118681121.ch18</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9781118681121.ch18&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9781118681121.ch18%26sid%3Dliteratum%253Aachs%26atitle%3DEpigenetic%252BProteins%252Bas%252BEmerging%252BDrug%252BTargets%26aulast%3DBoriack%25E2%2580%2590Sjodin%26aufirst%3DP.%2BAnn%26date%3D2020%26date%3D2020%26spage%3D423%26epage%3D448%26pub%3DWiley%26atitle%3DStructural%252BBiology%252Bin%252BDrug%252BDiscovery%26aulast%3DRenaud%26aufirst%3DJean%25E2%2580%2590Paul%26date%3D2020%26date%3D2020%26volume%3D62" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Iwona A.  Ciechomska</span>, <span class="hlFld-ContribAuthor ">Chinchu  Jayaprakash</span>, <span class="hlFld-ContribAuthor ">Marta  Maleszewska</span>, <span class="hlFld-ContribAuthor ">Bozena  Kaminska</span>. </span><span class="cited-content_cbyCitation_article-title">Histone Modifying Enzymes and Chromatin Modifiers in Glioma Pathobiology and Therapy Responses. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 259-279. <a href="https://doi.org/10.1007/978-3-030-30651-9_13" title="DOI URL">https://doi.org/10.1007/978-3-030-30651-9_13</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-3-030-30651-9_13&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-3-030-30651-9_13%26sid%3Dliteratum%253Aachs%26atitle%3DHistone%252BModifying%252BEnzymes%252Band%252BChromatin%252BModifiers%252Bin%252BGlioma%252BPathobiology%252Band%252BTherapy%252BResponses%26aulast%3DCiechomska%26aufirst%3DIwona%2BA.%26date%3D2020%26date%3D2020%26spage%3D259%26epage%3D279%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DGlioma%252BSignaling%26aulast%3DBara%25C5%2584ska%26aufirst%3DJolanta%26date%3D2020%26volume%3D1202" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ruben  Leenders</span>, <span class="hlFld-ContribAuthor ">Remco  Zijlmans</span>, <span class="hlFld-ContribAuthor ">Bart  van Bree</span>, <span class="hlFld-ContribAuthor ">Marc  van de Sande</span>, <span class="hlFld-ContribAuthor ">Federica  Trivarelli</span>, <span class="hlFld-ContribAuthor ">Eddy  Damen</span>, <span class="hlFld-ContribAuthor ">Anita  Wegert</span>, <span class="hlFld-ContribAuthor ">Daniel  Müller</span>, <span class="hlFld-ContribAuthor ">Jan Erik  Ehlert</span>, <span class="hlFld-ContribAuthor ">Daniel  Feger</span>, <span class="hlFld-ContribAuthor ">Carolin  Heidemann-Dinger</span>, <span class="hlFld-ContribAuthor ">Michael  Kubbutat</span>, <span class="hlFld-ContribAuthor ">Christoph  Schächtele</span>, <span class="hlFld-ContribAuthor ">Danny C.  Lenstra</span>, <span class="hlFld-ContribAuthor ">Jasmin  Mecinović</span>, <span class="hlFld-ContribAuthor ">Gerhard  Müller</span>. </span><span class="cited-content_cbyCitation_article-title">Novel SAR for quinazoline inhibitors of EHMT1 and EHMT2. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2019,</strong> <em>29 </em>
                                    (17)
                                     , 2516-2524. <a href="https://doi.org/10.1016/j.bmcl.2019.06.012" title="DOI URL">https://doi.org/10.1016/j.bmcl.2019.06.012</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2019.06.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2019.06.012%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DNovel%252BSAR%252Bfor%252Bquinazoline%252Binhibitors%252Bof%252BEHMT1%252Band%252BEHMT2%26aulast%3DLeenders%26aufirst%3DRuben%26date%3D2019%26volume%3D29%26issue%3D17%26spage%3D2516%26epage%3D2524" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">M  Ramya Chandar Charles</span>, <span class="hlFld-ContribAuthor ">Hsing-Pang  Hsieh</span>, <span class="hlFld-ContribAuthor ">Mohane  Selvaraj Coumar</span>. </span><span class="cited-content_cbyCitation_article-title">Delineating the active site architecture of G9a lysine methyltransferase through substrate and inhibitor binding mode analysis: a molecular dynamics study. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biomolecular Structure and Dynamics</span><span> <strong>2019,</strong> <em>37 </em>
                                    (10)
                                     , 2581-2592. <a href="https://doi.org/10.1080/07391102.2018.1491422" title="DOI URL">https://doi.org/10.1080/07391102.2018.1491422</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/07391102.2018.1491422&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F07391102.2018.1491422%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biomolecular%2520Structure%2520and%2520Dynamics%26atitle%3DDelineating%252Bthe%252Bactive%252Bsite%252Barchitecture%252Bof%252BG9a%252Blysine%252Bmethyltransferase%252Bthrough%252Bsubstrate%252Band%252Binhibitor%252Bbinding%252Bmode%252Banalysis%25253A%252Ba%252Bmolecular%252Bdynamics%252Bstudy%26aulast%3DRamya%2BChandar%2BCharles%26aufirst%3DM%26date%3D2019%26date%3D2018%26volume%3D37%26issue%3D10%26spage%3D2581%26epage%3D2592" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mariasoosai Ramya Chandar  Charles</span>, <span class="hlFld-ContribAuthor ">Arunkumar  Dhayalan</span>, <span class="hlFld-ContribAuthor ">Hsing-Pang  Hsieh</span>, <span class="hlFld-ContribAuthor ">Mohane Selvaraj  Coumar</span>. </span><span class="cited-content_cbyCitation_article-title">Insights for the design of protein lysine methyltransferase G9a inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2019,</strong> <em>11 </em>
                                    (9)
                                     , 993-1014. <a href="https://doi.org/10.4155/fmc-2018-0396" title="DOI URL">https://doi.org/10.4155/fmc-2018-0396</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2018-0396&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2018-0396%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DInsights%252Bfor%252Bthe%252Bdesign%252Bof%252Bprotein%252Blysine%252Bmethyltransferase%252BG9a%252Binhibitors%26aulast%3DCharles%26aufirst%3DMariasoosai%2BRamya%2BChandar%26date%3D2019%26volume%3D11%26issue%3D9%26spage%3D993%26epage%3D1014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Frida S.  Herrera-Vázquez</span>, <span class="hlFld-ContribAuthor ">Francisco  Hernández-Luis</span>, <span class="hlFld-ContribAuthor ">José L.  Medina Franco</span>. </span><span class="cited-content_cbyCitation_article-title">Quinazolines as inhibitors of chromatin-associated proteins in histones. </span><span class="cited-content_cbyCitation_journal-name">Medicinal Chemistry Research</span><span> <strong>2019,</strong> <em>28 </em>
                                    (4)
                                     , 395-416. <a href="https://doi.org/10.1007/s00044-019-02300-0" title="DOI URL">https://doi.org/10.1007/s00044-019-02300-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00044-019-02300-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00044-019-02300-0%26sid%3Dliteratum%253Aachs%26jtitle%3DMedicinal%2520Chemistry%2520Research%26atitle%3DQuinazolines%252Bas%252Binhibitors%252Bof%252Bchromatin-associated%252Bproteins%252Bin%252Bhistones%26aulast%3DHerrera-V%25C3%25A1zquez%26aufirst%3DFrida%2BS.%26date%3D2019%26date%3D2019%26volume%3D28%26issue%3D4%26spage%3D395%26epage%3D416" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">H. Ümit  Kaniskan</span>, <span class="hlFld-ContribAuthor ">Jian  Jin</span>. </span><span class="cited-content_cbyCitation_article-title">Selective Small‐Molecule Inhibitors of Protein Methyltransferases. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 201-220. <a href="https://doi.org/10.1002/9783527809257.ch9" title="DOI URL">https://doi.org/10.1002/9783527809257.ch9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527809257.ch9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527809257.ch9%26sid%3Dliteratum%253Aachs%26atitle%3DSelective%252BSmall%2525E2%252580%252590Molecule%252BInhibitors%252Bof%252BProtein%252BMethyltransferases%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259Cmit%26date%3D2019%26date%3D2019%26spage%3D201%26epage%3D220%26pub%3DWiley%26atitle%3DEpigenetic%252BDrug%252BDiscovery%26aulast%3DSippl%26aufirst%3DWolfgang%26date%3D2019%26date%3D2019%26volume%3D11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dylan M.  Marchione</span>, <span class="hlFld-ContribAuthor ">Benjamin A.  Garcia</span>, <span class="hlFld-ContribAuthor ">John  Wojcik</span>. </span><span class="cited-content_cbyCitation_article-title">Proteomic approaches for cancer epigenetics research. </span><span class="cited-content_cbyCitation_journal-name">Expert Review of Proteomics</span><span> <strong>2019,</strong> <em>16 </em>
                                    (1)
                                     , 33-47. <a href="https://doi.org/10.1080/14789450.2019.1550363" title="DOI URL">https://doi.org/10.1080/14789450.2019.1550363</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/14789450.2019.1550363&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F14789450.2019.1550363%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Review%2520of%2520Proteomics%26atitle%3DProteomic%252Bapproaches%252Bfor%252Bcancer%252Bepigenetics%252Bresearch%26aulast%3DMarchione%26aufirst%3DDylan%2BM.%26date%3D2019%26date%3D2018%26volume%3D16%26issue%3D1%26spage%3D33%26epage%3D47" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Giulia  Stazi</span>, <span class="hlFld-ContribAuthor ">Clemens  Zwergel</span>, <span class="hlFld-ContribAuthor ">Sergio  Valente</span>. </span><span class="cited-content_cbyCitation_article-title">Lysine Methyltransferases and Their Inhibitors. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 123-157. <a href="https://doi.org/10.1007/7355_2019_72" title="DOI URL">https://doi.org/10.1007/7355_2019_72</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/7355_2019_72&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F7355_2019_72%26sid%3Dliteratum%253Aachs%26atitle%3DLysine%252BMethyltransferases%252Band%252BTheir%252BInhibitors%26aulast%3DStazi%26aufirst%3DGiulia%26date%3D2019%26date%3D2019%26spage%3D123%26epage%3D157%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DChemical%252BEpigenetics%26aulast%3DMai%26aufirst%3DAntonello%26date%3D2020%26volume%3D33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Carol A.  Eisenberg</span>, <span class="hlFld-ContribAuthor ">Leonard M.  Eisenberg</span>. </span><span class="cited-content_cbyCitation_article-title">G9a and G9a-Like Histone Methyltransferases and Their Effect on Cell Phenotype, Embryonic Development, and Human Disease. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 399-433. <a href="https://doi.org/10.1007/978-3-030-14792-1_16" title="DOI URL">https://doi.org/10.1007/978-3-030-14792-1_16</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-3-030-14792-1_16&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-3-030-14792-1_16%26sid%3Dliteratum%253Aachs%26atitle%3DG9a%252Band%252BG9a-Like%252BHistone%252BMethyltransferases%252Band%252BTheir%252BEffect%252Bon%252BCell%252BPhenotype%25252C%252BEmbryonic%252BDevelopment%25252C%252Band%252BHuman%252BDisease%26aulast%3DEisenberg%26aufirst%3DCarol%2BA.%26date%3D2019%26date%3D2019%26spage%3D399%26epage%3D433%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DThe%252BDNA%25252C%252BRNA%25252C%252Band%252BHistone%252BMethylomes%26aulast%3DJurga%26aufirst%3DStefan%26date%3D2019%26volume%3D10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tomoya  Hirano</span>, <span class="hlFld-ContribAuthor ">Shuichi  Mori</span>, <span class="hlFld-ContribAuthor ">Hiroyuki  Kagechika</span>. </span><span class="cited-content_cbyCitation_article-title">Recent Advances in Chemical Tools for the Regulation and Study of Protein Lysine Methyltransferases. </span><span class="cited-content_cbyCitation_journal-name">The Chemical Record</span><span> <strong>2018,</strong> <em>18 </em>
                                    (12)
                                     , 1745-1759. <a href="https://doi.org/10.1002/tcr.201800034" title="DOI URL">https://doi.org/10.1002/tcr.201800034</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/tcr.201800034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Ftcr.201800034%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Chemical%2520Record%26atitle%3DRecent%252BAdvances%252Bin%252BChemical%252BTools%252Bfor%252Bthe%252BRegulation%252Band%252BStudy%252Bof%252BProtein%252BLysine%252BMethyltransferases%26aulast%3DHirano%26aufirst%3DTomoya%26date%3D2018%26date%3D2018%26volume%3D18%26issue%3D12%26spage%3D1745%26epage%3D1759" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mohammad  Alzrigat</span>, <span class="hlFld-ContribAuthor ">Alba Atienza  Párraga</span>, <span class="hlFld-ContribAuthor ">Helena  Jernberg-Wiklund</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetics in multiple myeloma: From mechanisms to therapy. </span><span class="cited-content_cbyCitation_journal-name">Seminars in Cancer Biology</span><span> <strong>2018,</strong> <em>51 </em>, 101-115. <a href="https://doi.org/10.1016/j.semcancer.2017.09.007" title="DOI URL">https://doi.org/10.1016/j.semcancer.2017.09.007</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.semcancer.2017.09.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.semcancer.2017.09.007%26sid%3Dliteratum%253Aachs%26jtitle%3DSeminars%2520in%2520Cancer%2520Biology%26atitle%3DEpigenetics%252Bin%252Bmultiple%252Bmyeloma%25253A%252BFrom%252Bmechanisms%252Bto%252Btherapy%26aulast%3DAlzrigat%26aufirst%3DMohammad%26date%3D2018%26volume%3D51%26spage%3D101%26epage%3D115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">A.  Ganesan</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetic drug discovery: a success story for cofactor interference. </span><span class="cited-content_cbyCitation_journal-name">Philosophical Transactions of the Royal Society B: Biological Sciences</span><span> <strong>2018,</strong> <em>373 </em>
                                    (1748)
                                     , 20170069. <a href="https://doi.org/10.1098/rstb.2017.0069" title="DOI URL">https://doi.org/10.1098/rstb.2017.0069</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1098/rstb.2017.0069&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1098%2Frstb.2017.0069%26sid%3Dliteratum%253Aachs%26jtitle%3DPhilosophical%2520Transactions%2520of%2520the%2520Royal%2520Society%2520B%253A%2520Biological%2520Sciences%26atitle%3DEpigenetic%252Bdrug%252Bdiscovery%25253A%252Ba%252Bsuccess%252Bstory%252Bfor%252Bcofactor%252Binterference%26aulast%3DGanesan%26aufirst%3DA.%26date%3D2018%26date%3D2018%26volume%3D373%26issue%3D1748%26spage%3D20170069" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Michael J.  Morrison</span>, <span class="hlFld-ContribAuthor ">P. Ann  Boriack-Sjodin</span>, <span class="hlFld-ContribAuthor ">Kerren K.  Swinger</span>, <span class="hlFld-ContribAuthor ">Tim J.  Wigle</span>, <span class="hlFld-ContribAuthor ">Dipti  Sadalge</span>, <span class="hlFld-ContribAuthor ">Kevin W.  Kuntz</span>, <span class="hlFld-ContribAuthor ">Margaret Porter  Scott</span>, <span class="hlFld-ContribAuthor ">William P.  Janzen</span>, <span class="hlFld-ContribAuthor ">Richard  Chesworth</span>, <span class="hlFld-ContribAuthor ">Kenneth W.  Duncan</span>, <span class="hlFld-ContribAuthor ">Darren M.  Harvey</span>, <span class="hlFld-ContribAuthor ">John W.  Lampe</span>, <span class="hlFld-ContribAuthor ">Lorna H.  Mitchell</span>, <span class="hlFld-ContribAuthor ">Robert A.  Copeland</span>, . </span><span class="cited-content_cbyCitation_article-title">Identification of a peptide inhibitor for the histone methyltransferase WHSC1. </span><span class="cited-content_cbyCitation_journal-name">PLOS ONE</span><span> <strong>2018,</strong> <em>13 </em>
                                    (5)
                                     , e0197082. <a href="https://doi.org/10.1371/journal.pone.0197082" title="DOI URL">https://doi.org/10.1371/journal.pone.0197082</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pone.0197082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pone.0197082%26sid%3Dliteratum%253Aachs%26jtitle%3DPLOS%2520ONE%26atitle%3DIdentification%252Bof%252Ba%252Bpeptide%252Binhibitor%252Bfor%252Bthe%252Bhistone%252Bmethyltransferase%252BWHSC1%26aulast%3DMorrison%26aufirst%3DMichael%2BJ.%26date%3D2018%26date%3D2018%26volume%3D13%26issue%3D5%26spage%3De0197082" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Danny C.  Lenstra</span>, <span class="hlFld-ContribAuthor ">Abbas H.K.  Al Temimi</span>, <span class="hlFld-ContribAuthor ">Jasmin  Mecinović</span>. </span><span class="cited-content_cbyCitation_article-title">Inhibition of histone lysine methyltransferases G9a and GLP by ejection of structural Zn(II). </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2018,</strong> <em>28 </em>
                                    (7)
                                     , 1234-1238. <a href="https://doi.org/10.1016/j.bmcl.2018.02.043" title="DOI URL">https://doi.org/10.1016/j.bmcl.2018.02.043</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2018.02.043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2018.02.043%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DInhibition%252Bof%252Bhistone%252Blysine%252Bmethyltransferases%252BG9a%252Band%252BGLP%252Bby%252Bejection%252Bof%252Bstructural%252BZn%252528II%252529%26aulast%3DLenstra%26aufirst%3DDanny%2BC.%26date%3D2018%26volume%3D28%26issue%3D7%26spage%3D1234%26epage%3D1238" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Moses  Moustakim</span>, <span class="hlFld-ContribAuthor ">Suet Ling  Felce</span>, <span class="hlFld-ContribAuthor ">Nancy  Zaarour</span>, <span class="hlFld-ContribAuthor ">Gillian  Farnie</span>, <span class="hlFld-ContribAuthor ">Fiona E.  McCann</span>, <span class="hlFld-ContribAuthor ">Paul E.  Brennan</span>. </span><span class="cited-content_cbyCitation_article-title">Target Identification Using Chemical Probes. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2018,</strong>,, 27-58. <a href="https://doi.org/10.1016/bs.mie.2018.09.013" title="DOI URL">https://doi.org/10.1016/bs.mie.2018.09.013</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/bs.mie.2018.09.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fbs.mie.2018.09.013%26sid%3Dliteratum%253Aachs%26atitle%3DTarget%252BIdentification%252BUsing%252BChemical%252BProbes%26aulast%3DMoustakim%26aufirst%3DMoses%26date%3D2018%26spage%3D27%26epage%3D58%26pub%3DElsevier%26atitle%3DModern%252BApproaches%252Bin%252BDrug%252BDiscovery%26date%3D2018%26volume%3D610" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Pintu  Maity</span>, <span class="hlFld-ContribAuthor ">Debasish  Kundu</span>, <span class="hlFld-ContribAuthor ">Tubai  Ghosh</span>, <span class="hlFld-ContribAuthor ">Brindaban C.  Ranu</span>. </span><span class="cited-content_cbyCitation_article-title">Copper catalyzed cyanation through CC bond cleavage of
              gem
              -aryl dibromide followed by second cyanation of iodoarene by a released CN unit. </span><span class="cited-content_cbyCitation_journal-name">Organic Chemistry Frontiers</span><span> <strong>2018,</strong> <em>5 </em>
                                    (10)
                                     , 1586-1599. <a href="https://doi.org/10.1039/C8QO00108A" title="DOI URL">https://doi.org/10.1039/C8QO00108A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C8QO00108A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC8QO00108A%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520Chemistry%2520Frontiers%26atitle%3DCopper%252Bcatalyzed%252Bcyanation%252Bthrough%252BC%2525EE%252580%252581C%252Bbond%252Bcleavage%252Bof%252Bgem%252B-aryl%252Bdibromide%252Bfollowed%252Bby%252Bsecond%252Bcyanation%252Bof%252Biodoarene%252Bby%252Ba%252Breleased%252BCN%252Bunit%26aulast%3DMaity%26aufirst%3DPintu%26date%3D2018%26date%3D2018%26volume%3D5%26issue%3D10%26spage%3D1586%26epage%3D1599" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">H. Ümit  Kaniskan</span>, <span class="hlFld-ContribAuthor ">Jian  Jin</span>. </span><span class="cited-content_cbyCitation_article-title">Recent progress in developing selective inhibitors of protein methyltransferases. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2017,</strong> <em>39 </em>, 100-108. <a href="https://doi.org/10.1016/j.cbpa.2017.06.013" title="DOI URL">https://doi.org/10.1016/j.cbpa.2017.06.013</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2017.06.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2017.06.013%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DRecent%252Bprogress%252Bin%252Bdeveloping%252Bselective%252Binhibitors%252Bof%252Bprotein%252Bmethyltransferases%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259Cmit%26date%3D2017%26volume%3D39%26spage%3D100%26epage%3D108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yan  Xiong</span>, <span class="hlFld-ContribAuthor ">Fengling  Li</span>, <span class="hlFld-ContribAuthor ">Nicolas  Babault</span>, <span class="hlFld-ContribAuthor ">Hong  Wu</span>, <span class="hlFld-ContribAuthor ">Aiping  Dong</span>, <span class="hlFld-ContribAuthor ">Hong  Zeng</span>, <span class="hlFld-ContribAuthor ">Xin  Chen</span>, <span class="hlFld-ContribAuthor ">Cheryl H.  Arrowsmith</span>, <span class="hlFld-ContribAuthor ">Peter J.  Brown</span>, <span class="hlFld-ContribAuthor ">Jing  Liu</span>, <span class="hlFld-ContribAuthor ">Masoud  Vedadi</span>, <span class="hlFld-ContribAuthor ">Jian  Jin</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-activity relationship studies of G9a-like protein (GLP) inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2017,</strong> <em>25 </em>
                                    (16)
                                     , 4414-4423. <a href="https://doi.org/10.1016/j.bmc.2017.06.021" title="DOI URL">https://doi.org/10.1016/j.bmc.2017.06.021</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2017.06.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2017.06.021%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DStructure-activity%252Brelationship%252Bstudies%252Bof%252BG9a-like%252Bprotein%252B%252528GLP%252529%252Binhibitors%26aulast%3DXiong%26aufirst%3DYan%26date%3D2017%26volume%3D25%26issue%3D16%26spage%3D4414%26epage%3D4423" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/medium/jm-2016-01645c_0016.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/large/jm-2016-01645c_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01645&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/medium/jm-2016-01645c_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/large/jm-2016-01645c_0001.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of selected G9a/GLP inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/large/jm-2016-01645c_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01645&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/medium/jm-2016-01645c_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/large/jm-2016-01645c_0002.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. X-ray cocrystal structures of (A) GLP (green) (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TUZ">5TUZ</a>) and (B) G9a (blue) (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TUY">5TUY</a>) in complex with <b>4</b> in the presence of SAM. Water molecule is illustrated as a red sphere. Main interactions are shown in yellow dashed lines. (C) Overlay of (A) and (B).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/large/jm-2016-01645c_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01645&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/medium/jm-2016-01645c_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/large/jm-2016-01645c_0007.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of <b>4</b> and <b>7</b>–<b>22</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/large/jm-2016-01645c_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01645&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) 1-alkylpiperidin-4-ylamine, K<sub>2</sub>CO<sub>3</sub>, DMF, rt, 80–90%; (b) R′ amines, 4 N HCl in dixoane, <i>i</i>-PrOH, microwave, 160 °C, 75–85%.</p></p></figure><figure data-id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/medium/jm-2016-01645c_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/large/jm-2016-01645c_0008.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of <b>41</b>–<b>48</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/large/jm-2016-01645c_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01645&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) EtI or <i>i</i>-PrBr, K<sub>2</sub>CO<sub>3</sub>, acetone, reflux; (b) HNO<sub>3</sub>, Ac<sub>2</sub>O, 0 °C to rt, 81–94% over 2 steps; (c) Fe dust, NH<sub>4</sub>OAc, H<sub>2</sub>O, reflux, 60–70%; (d) Bu<sub>4</sub>NBr<sub>3</sub>, ethyl acetate, 0 °C to rt, 80–90%; (e) Zn(CN)<sub>2</sub>, Pd(pph<sub>3</sub>)<sub>4</sub>, DMF, microwave, 130 °C, 50–72%; (f) Br<sub>2</sub>, Ac<sub>2</sub>O, 50 °C, 64%; then HCl, EtOH, 95%; (g) Br<sub>2</sub>, K<sub>2</sub>CO<sub>3</sub>, DCM, −15 °C, 70%; (h) CuCN, DMF, sealed tube, 160 °C, 56%; (i) ClCO<sub>2</sub>Me, DIEA, DCM/DMF, 0 °C to rt; (j) H<sub>2</sub>O<sub>2</sub>, NaOH, H<sub>2</sub>O/MeOH, reflux; (k) PhNEt<sub>2</sub>, POCl<sub>3</sub>, reflux, 30–40% over 3 steps; (l) 1-methylpiperidin-4-amine, K<sub>2</sub>CO<sub>3</sub>, DMF, rt; (m) morpholine, 4 N HCl in dixoane, <i>i</i>-PrOH, microwave, 160 °C, 75–85% over 2 steps.</p></p></figure><figure data-id="sch3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/medium/jm-2016-01645c_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/large/jm-2016-01645c_0009.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of <b>53</b> and <b>54</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/large/jm-2016-01645c_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01645&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) acid chloride, DIEA, DCM/DMF, 0 °C to rt, 90–95%; (b) H<sub>2</sub>O<sub>2</sub>, NaOH, H<sub>2</sub>O/MeOH, reflux, 46–55% over 2 steps; (c) PhNEt<sub>2</sub>, POCl<sub>3</sub>, reflux, 46–55% over 2 steps; (d) 1-methylpiperidin-4-amine, DIEA, <i>i</i>-PrOH, microwave, 160 °C, 80–90%.</p></p></figure><figure data-id="fig3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/medium/jm-2016-01645c_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/large/jm-2016-01645c_0003.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Selectivity of <b>4</b> and <b>18</b> against methyltransferases. Selectivity of <b>4</b> (A) and <b>18</b> (B) against a panel of 29 PKMTs, PRMTs, DNMTs, and RNMTs was determined at two compound concentrations of 1 μM (blue bars) and 10 μM (red bars).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/large/jm-2016-01645c_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01645&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/medium/jm-2016-01645c_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/large/jm-2016-01645c_0004.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Binding confirmation of <b>4</b> and <b>18</b>. Isothermal titration calorimetry (ITC) was used to confirm binding of <b>4</b> to (A) GLP with a <i>K</i><sub>d</sub> of 40 ± 5 nM and to (B) G9a with a <i>K</i><sub>d</sub> of 445 ± 40 nM. Similarly, ITC was used to confirm the binding of <b>18</b> to (C) GLP with a <i>K</i><sub>d</sub> of 46 ± 15 nM and to (D) G9a with a <i>K</i><sub>d</sub> of 610 ± 68 nM. ITC experiments for GLP and G9a were performed in triplicate.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/large/jm-2016-01645c_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01645&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/medium/jm-2016-01645c_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/large/jm-2016-01645c_0005.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Mechanism of GLP inhibition by <b>4</b> and <b>18</b>. Mechanism of inhibition of GLP activity by <b>4</b> and <b>18</b> was explored by IC<sub>50</sub> determination at various peptide (A and C, respectively) or SAM (B and D, respectively) concentrations while the concentration of the other substrate was kept constant. Increase in peptide concentration in assays resulted in significant increase in IC<sub>50</sub> values, indicating that both compounds are peptide competitive (A,C). Decrease in IC<sub>50</sub> values upon increasing SAM concentration indicates that both compounds are SAM uncompetitive inhibitors (B,D).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/large/jm-2016-01645c_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01645&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/medium/jm-2016-01645c_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/large/jm-2016-01645c_0006.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. X-ray cocrystal structures of GLP and G9a in complex with <b>18</b>. (A) Structure of GLP–SAM–<b>18</b> (green) (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TTG">5TTG</a>). (B) Structure of G9a–SAM–<b>18</b> (blue) (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TTF">5TTF</a>). Water molecule is illustrated as a red sphere. Main interactions are shown in yellow dashed lines. (C) Overlay of (A) and (B). (D) Overlay of (A) and GLP–SAH–<b>2</b> (purple) (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3MO5">3MO5</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01645/20170303/images/large/jm-2016-01645c_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01645&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i77">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_86115" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_86115" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 67 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Martin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span> </span><span class="NLM_article-title">The diverse functions of histone lysine methylation</span> <span class="citation_source-journal">Nat. Rev. Mol. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">838</span><span class="NLM_x">–</span> <span class="NLM_lpage">849</span><span class="refDoi"> DOI: 10.1038/nrm1761</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1038%2Fnrm1761" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=16261189" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtFKlsL3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2005&pages=838-849&author=C.+Martinauthor=Y.+Zhang&title=The+diverse+functions+of+histone+lysine+methylation&doi=10.1038%2Fnrm1761"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The diverse functions of histone lysine methylation</span></div><div class="casAuthors">Martin, Cyrus; Zhang, Yi</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">838-849</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Covalent modifications of histone tails play fundamental roles in chromatin structure and function.  One such modification, lysine methylation, has important functions in many biol. processes that include heterochromatin formation, X-chromosome inactivation, and transcriptional regulation.  Here, the authors summarize recent advances in the understanding of how lysine methylation functions in these diverse biol. processes, and raise questions that need to be addressed in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotrSycSeCRB7Vg90H21EOLACvtfcHk0ljLl9eOAdOI9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtFKlsL3N&md5=8ca89b050ff6f72b0b12e05a3b5500d7</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrm1761&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm1761%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DThe%2520diverse%2520functions%2520of%2520histone%2520lysine%2520methylation%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2005%26volume%3D6%26spage%3D838%26epage%3D849%26doi%3D10.1038%2Fnrm1761" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Kaniskan, H. U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Chemical probes of histone lysine methyltransferases</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">40</span><span class="NLM_x">–</span> <span class="NLM_lpage">50</span><span class="refDoi"> DOI: 10.1021/cb500785t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500785t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFOht7%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=40-50&author=H.+U.+Kaniskanauthor=J.+Jin&title=Chemical+probes+of+histone+lysine+methyltransferases&doi=10.1021%2Fcb500785t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical probes of histone lysine methyltransferases</span></div><div class="casAuthors">Kaniskan, H. Umit; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">40-50</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Growing evidence suggests that histone-lysine methyltransferases (HMTs, also known as protein methyltransferases (PMTs)) play an important role in diverse biol. processes and human diseases by regulating gene expression and the chromatin state.  Therefore, HMTs have been increasingly recognized by the biomedical community as a class of potential therapeutic targets.  High quality chem. probes of HMTs, as tools for deciphering their physiol. functions and roles in human diseases and testing therapeutic hypotheses, are crit. for advancing this promising field.  Here, the authors focus on the discovery, characterization, and biol. applications of chem. probes for HMTs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpp8kb_c7Xl57Vg90H21EOLACvtfcHk0ljLl9eOAdOI9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFOht7%252FE&md5=4eb8a696e2227c60686df8d3c60777ce</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Fcb500785t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500785t%26sid%3Dliteratum%253Aachs%26aulast%3DKaniskan%26aufirst%3DH.%2BU.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DChemical%2520probes%2520of%2520histone%2520lysine%2520methyltransferases%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D40%26epage%3D50%26doi%3D10.1021%2Fcb500785t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Kaniskan, H. U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Konze, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Selective inhibitors of protein methyltransferases</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">1596</span><span class="NLM_x">–</span> <span class="NLM_lpage">1629</span><span class="refDoi"> DOI: 10.1021/jm501234a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501234a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFKqtbbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=1596-1629&author=H.+U.+Kaniskanauthor=K.+D.+Konzeauthor=J.+Jin&title=Selective+inhibitors+of+protein+methyltransferases&doi=10.1021%2Fjm501234a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibitors of protein methyltransferases</span></div><div class="casAuthors">Kaniskan, H. Umit; Konze, Kyle D.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1596-1629</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Mounting evidence suggests that protein methyltransferases (PMTs), which catalyze the methylation of histone and non-histone proteins, play a crucial role in diverse biol. processes and human diseases.  In particular, PMTs have been recognized as major players in regulating gene expression and chromatin state.  PMTs are divided into 2 categories: protein lysine methyltransferases (PKMTs) and protein arginine methyltransferases (PRMTs).  There has been a steadily growing interest in these enzymes as potential therapeutic targets and therefore discovery of PMT inhibitors has also been pursued increasingly over the past decade.  Here, the authors present a perspective on selective, small-mol. inhibitors of PMTs with an emphasis on their discovery, characterization, and applicability as chem. tools for deciphering the target PMTs' physiol. functions and involvement in human diseases.  The authors highlight the current state of PMT inhibitors and discuss future directions and opportunities for PMT inhibitor discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEWF_XeLEY1rVg90H21EOLACvtfcHk0liJNSZgskPJhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFKqtbbJ&md5=8fb2b2d21d306c02a2da60a3fb88c430</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Fjm501234a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501234a%26sid%3Dliteratum%253Aachs%26aulast%3DKaniskan%26aufirst%3DH.%2BU.%26aulast%3DKonze%26aufirst%3DK.%2BD.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DSelective%2520inhibitors%2520of%2520protein%2520methyltransferases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D1596%26epage%3D1629%26doi%3D10.1021%2Fjm501234a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Benevento, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van de Molengraft, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Westen, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Bokhoven, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kasri, N. N.</span><span> </span><span class="NLM_article-title">The role of chromatin repressive marks in cognition and disease: A focus on the repressive complex GLP/G9a</span> <span class="citation_source-journal">Neurobiol. Learn. Mem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">124</span><span class="NLM_x">, </span> <span class="NLM_fpage">88</span><span class="NLM_x">–</span> <span class="NLM_lpage">96</span><span class="refDoi"> DOI: 10.1016/j.nlm.2015.06.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1016%2Fj.nlm.2015.06.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=26143996" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFemtL7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2015&pages=88-96&author=M.+Beneventoauthor=M.+van+de+Molengraftauthor=R.+van+Westenauthor=H.+van+Bokhovenauthor=N.+N.+Kasri&title=The+role+of+chromatin+repressive+marks+in+cognition+and+disease%3A+A+focus+on+the+repressive+complex+GLP%2FG9a&doi=10.1016%2Fj.nlm.2015.06.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The role of chromatin repressive marks in cognition and disease: A focus on the repressive complex GLP/G9a</span></div><div class="casAuthors">Benevento, Marco; van de Molengraft, Marise; van Westen, Rhode; van Bokhoven, Hans; Nadif Kasri, Nael</div><div class="citationInfo"><span class="NLM_cas:title">Neurobiology of Learning and Memory</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">124</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">88-96</span>CODEN:
                <span class="NLM_cas:coden">NLMEFR</span>;
        ISSN:<span class="NLM_cas:issn">1074-7427</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Histone post-translational modifications are key epigenetic processes controlling the regulation of gene transcription.  In recent years it has become apparent that chromatin modifications contribute to cognition through the modulation of gene expression required for the expression and consolidation of memories.  In this review, we focus on the role of histone methylation in the nervous system.  Histone methylation is involved in a no. of cognitive disturbances, such as intellectual disability, cocaine addiction and age-related cognitive decline.  We provide an overview of the dynamic changes in methylation of histone lysine residues during learning and memory.  With a special focus on H3K9 histone methyltransferases GLP and G9a, we summarize the effects of deficiencies in writer and eraser enzymes on neuronal plasticity and cognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGob0r1EZs1wF7Vg90H21EOLACvtfcHk0liJNSZgskPJhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFemtL7L&md5=3ef82f509c3a18b57502d1ddc707e51e</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.nlm.2015.06.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.nlm.2015.06.013%26sid%3Dliteratum%253Aachs%26aulast%3DBenevento%26aufirst%3DM.%26aulast%3Dvan%2Bde%2BMolengraft%26aufirst%3DM.%26aulast%3Dvan%2BWesten%26aufirst%3DR.%26aulast%3Dvan%2BBokhoven%26aufirst%3DH.%26aulast%3DKasri%26aufirst%3DN.%2BN.%26atitle%3DThe%2520role%2520of%2520chromatin%2520repressive%2520marks%2520in%2520cognition%2520and%2520disease%253A%2520A%2520focus%2520on%2520the%2520repressive%2520complex%2520GLP%252FG9a%26jtitle%3DNeurobiol.%2520Learn.%2520Mem.%26date%3D2015%26volume%3D124%26spage%3D88%26epage%3D96%26doi%3D10.1016%2Fj.nlm.2015.06.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Black, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Rechem, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whetstine, J. R.</span><span> </span><span class="NLM_article-title">Histone lysine methylation dynamics: establishment, regulation, and biological impact</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">491</span><span class="NLM_x">–</span> <span class="NLM_lpage">507</span><span class="refDoi"> DOI: 10.1016/j.molcel.2012.11.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1016%2Fj.molcel.2012.11.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=23200123" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslKqtLvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2012&pages=491-507&author=J.+C.+Blackauthor=C.+Van+Rechemauthor=J.+R.+Whetstine&title=Histone+lysine+methylation+dynamics%3A+establishment%2C+regulation%2C+and+biological+impact&doi=10.1016%2Fj.molcel.2012.11.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Lysine Methylation Dynamics: Establishment, Regulation, and Biological Impact</span></div><div class="casAuthors">Black, Joshua C.; Van Rechem, Capucine; Whetstine, Johnathan R.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">491-507</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  Histone lysine methylation has emerged as a crit. player in the regulation of gene expression, cell cycle, genome stability, and nuclear architecture.  Over the past decade, a tremendous amt. of progress has led to the characterization of Me modifications and the lysine methyltransferases (KMTs) and lysine demethylases (KDMs) that regulate them.  Here, we review the discovery and characterization of the KMTs and KDMs and the Me modifications they regulate.  We discuss the localization of the KMTs and KDMs as well as the distribution of lysine methylation throughout the genome.  We highlight how these data have shaped our view of lysine methylation as a key determinant of complex chromatin states.  Finally, we discuss the regulation of KMTs and KDMs by proteasomal degrdn., posttranscriptional mechanisms, and metabolic status.  We propose key questions for the field and highlight areas that we predict will yield exciting discoveries in the years to come.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbnBk0Al-Ce7Vg90H21EOLACvtfcHk0liJNSZgskPJhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslKqtLvM&md5=b9a8e9c29c630592f5413f7e36974735</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2012.11.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2012.11.006%26sid%3Dliteratum%253Aachs%26aulast%3DBlack%26aufirst%3DJ.%2BC.%26aulast%3DVan%2BRechem%26aufirst%3DC.%26aulast%3DWhetstine%26aufirst%3DJ.%2BR.%26atitle%3DHistone%2520lysine%2520methylation%2520dynamics%253A%2520establishment%252C%2520regulation%252C%2520and%2520biological%2520impact%26jtitle%3DMol.%2520Cell%26date%3D2012%26volume%3D48%26spage%3D491%26epage%3D507%26doi%3D10.1016%2Fj.molcel.2012.11.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Mozzetta, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pontis, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ait-Si-Ali, S.</span><span> </span><span class="NLM_article-title">Functional crosstalk between lysine methyltransferases on histone substrates: the case of G9A/GLP and polycomb repressive complex 2</span> <span class="citation_source-journal">Antioxid. Redox Signaling</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">1365</span><span class="NLM_x">–</span> <span class="NLM_lpage">1381</span><span class="refDoi"> DOI: 10.1089/ars.2014.6116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1089%2Fars.2014.6116" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=25365549" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BC2MXotFajt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2015&pages=1365-1381&author=C.+Mozzettaauthor=J.+Pontisauthor=S.+Ait-Si-Ali&title=Functional+crosstalk+between+lysine+methyltransferases+on+histone+substrates%3A+the+case+of+G9A%2FGLP+and+polycomb+repressive+complex+2&doi=10.1089%2Fars.2014.6116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Functional Crosstalk Between Lysine Methyltransferases on Histone Substrates: The Case of G9A/GLP and Polycomb Repressive Complex 2</span></div><div class="casAuthors">Mozzetta, Chiara; Pontis, Julien; Ait-Si-Ali, Slimane</div><div class="citationInfo"><span class="NLM_cas:title">Antioxidants & Redox Signaling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1365-1381</span>CODEN:
                <span class="NLM_cas:coden">ARSIF2</span>;
        ISSN:<span class="NLM_cas:issn">1523-0864</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">Significance: Methylation of histone H3 on lysine 9 and 27 (H3K9 and H3K27) are two epigenetic modifications that have been linked to several crucial biol. processes, among which are transcriptional silencing and cell differentiation.  Recent Advances: Deposition of these marks is catalyzed by H3K9 lysine methyltransferases (KMTs) and polycomb repressive complex 2, resp.  Increasing evidence is emerging in favor of a functional crosstalk between these two major KMT families.  Crit. Issues: Here, we review the current knowledge on the mechanisms of action and function of these enzymes, with particular emphasis on their interplay in the regulation of chromatin states and biol. processes.  We outline their crucial roles played in tissue homeostasis, by controlling the fate of embryonic and tissue-specific stem cells, highlighting how their deregulation is often linked to the emergence of a no. of malignancies and neurol. disorders.  Future Directions: Histone methyltransferases are starting to be tested as drug targets.  A new generation of highly selective chem. inhibitors is starting to emerge.  These hold great promise for a rapid translation of targeting epigenetic drugs into clin. practice for a no. of aggressive cancers and neurol. disorders.  Antioxid. Redox Signal. 22, 1365-1381.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqA2o7h77ex5LVg90H21EOLACvtfcHk0lhB5oNMeHN2Bw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXotFajt7o%253D&md5=67d37b6c59e5f10bec09a071d59ce443</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1089%2Fars.2014.6116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fars.2014.6116%26sid%3Dliteratum%253Aachs%26aulast%3DMozzetta%26aufirst%3DC.%26aulast%3DPontis%26aufirst%3DJ.%26aulast%3DAit-Si-Ali%26aufirst%3DS.%26atitle%3DFunctional%2520crosstalk%2520between%2520lysine%2520methyltransferases%2520on%2520histone%2520substrates%253A%2520the%2520case%2520of%2520G9A%252FGLP%2520and%2520polycomb%2520repressive%2520complex%25202%26jtitle%3DAntioxid.%2520Redox%2520Signaling%26date%3D2015%26volume%3D22%26spage%3D1365%26epage%3D1381%26doi%3D10.1089%2Fars.2014.6116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Mozzetta, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boyarchuk, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pontis, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ait-Si-Ali, S.</span><span> </span><span class="NLM_article-title">Sound of silence: the properties and functions of repressive Lys methyltransferases</span> <span class="citation_source-journal">Nat. Rev. Mol. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">499</span><span class="NLM_x">–</span> <span class="NLM_lpage">513</span><span class="refDoi"> DOI: 10.1038/nrm4029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1038%2Fnrm4029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=26204160" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtl2ht77E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2015&pages=499-513&author=C.+Mozzettaauthor=E.+Boyarchukauthor=J.+Pontisauthor=S.+Ait-Si-Ali&title=Sound+of+silence%3A+the+properties+and+functions+of+repressive+Lys+methyltransferases&doi=10.1038%2Fnrm4029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Sound of silence: the properties and functions of repressive Lys methyltransferases</span></div><div class="casAuthors">Mozzetta, Chiara; Boyarchuk, Ekaterina; Pontis, Julien; Ait-Si-Ali, Slimane</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">499-513</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The methylation of histone Lys residues by Lys methyltransferases (KMTs) regulates chromatin organization and either activates or represses gene expression, depending on the residue that is targeted.  KMTs are emerging as key components in several cellular processes, and their deregulation is often assocd. with pathogenesis.  Here, we review the current knowledge on the main KMTs that are assocd. with gene silencing: namely, those responsible for methylating histone H3 Lys 9 (H3K9), H3K27 and H4K20.  We discuss their biochem. properties and the various mechanisms by which they are targeted to the chromatin and regulate gene expression, as well as new data on the interplay between them and other chromatin modifiers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzrGSxI8FBnbVg90H21EOLACvtfcHk0lhB5oNMeHN2Bw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtl2ht77E&md5=e32aa80d89533b1cfd04a90c168882d9</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnrm4029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm4029%26sid%3Dliteratum%253Aachs%26aulast%3DMozzetta%26aufirst%3DC.%26aulast%3DBoyarchuk%26aufirst%3DE.%26aulast%3DPontis%26aufirst%3DJ.%26aulast%3DAit-Si-Ali%26aufirst%3DS.%26atitle%3DSound%2520of%2520silence%253A%2520the%2520properties%2520and%2520functions%2520of%2520repressive%2520Lys%2520methyltransferases%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2015%26volume%3D16%26spage%3D499%26epage%3D513%26doi%3D10.1038%2Fnrm4029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Kouzarides, T.</span><span> </span><span class="NLM_article-title">Chromatin modifications and their function</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">128</span><span class="NLM_x">, </span> <span class="NLM_fpage">693</span><span class="NLM_x">–</span> <span class="NLM_lpage">705</span><span class="refDoi"> DOI: 10.1016/j.cell.2007.02.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1016%2Fj.cell.2007.02.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=17320507" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BD2sXis12ju7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2007&pages=693-705&author=T.+Kouzarides&title=Chromatin+modifications+and+their+function&doi=10.1016%2Fj.cell.2007.02.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatin modifications and their function</span></div><div class="casAuthors">Kouzarides, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">693-705</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The surface of nucleosomes is studded with a multiplicity of modifications.  At least eight different classes have been characterized to date and many different sites have been identified for each class.  Operationally, modifications function either by disrupting chromatin contacts or by affecting the recruitment of nonhistone proteins to chromatin.  Their presence on histones can dictate the higher-order chromatin structure in which DNA is packaged and can orchestrate the ordered recruitment of enzyme complexes to manipulate DNA.  In this way, histone modifications have the potential to influence many fundamental biol. processes, some of which may be epigenetically inherited.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAH9AX2dGicbVg90H21EOLACvtfcHk0lhB5oNMeHN2Bw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXis12ju7Y%253D&md5=c5995fe68fd9473f4268d59c01d3b9de</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.02.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.02.005%26sid%3Dliteratum%253Aachs%26aulast%3DKouzarides%26aufirst%3DT.%26atitle%3DChromatin%2520modifications%2520and%2520their%2520function%26jtitle%3DCell%26date%3D2007%26volume%3D128%26spage%3D693%26epage%3D705%26doi%3D10.1016%2Fj.cell.2007.02.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Bannister, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kouzarides, T.</span><span> </span><span class="NLM_article-title">Regulation of chromatin by histone modifications</span> <span class="citation_source-journal">Cell Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">381</span><span class="NLM_x">–</span> <span class="NLM_lpage">395</span><span class="refDoi"> DOI: 10.1038/cr.2011.22</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1038%2Fcr.2011.22" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=21321607" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BC3MXivVCgurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=381-395&author=A.+J.+Bannisterauthor=T.+Kouzarides&title=Regulation+of+chromatin+by+histone+modifications&doi=10.1038%2Fcr.2011.22"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of chromatin by histone modifications</span></div><div class="casAuthors">Bannister, Andrew J.; Kouzarides, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Cell Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">381-395</span>CODEN:
                <span class="NLM_cas:coden">CREEB6</span>;
        ISSN:<span class="NLM_cas:issn">1001-0602</span>.
    
            (<span class="NLM_cas:orgname">NPG Nature Asia-Pacific</span>)
        </div><div class="casAbstract">A review.  Chromatin is not an inert structure, but rather an instructive DNA scaffold that can respond to external cues to regulate the many uses of DNA.  A principal component of chromatin that plays a key role in this regulation is the modification of histones.  There is an ever-growing list of these modifications and the complexity of their action is only just beginning to be understood.  However, it is clear that histone modifications play fundamental roles in most biol. processes that are involved in the manipulation and expression of DNA.  Here, we describe the known histone modifications, define where they are found genomically and discuss some of their functional consequences, concg. mostly on transcription where the majority of characterization has taken place.  Cell Research (2011) 21:381-395. doi:10.1038/cr.2011.22; published online 15 Feb. 2011.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2ObWaVBHPorVg90H21EOLACvtfcHk0lgtsbAfFpklmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXivVCgurc%253D&md5=bbdf55be25985810b90ba2efe6399f1b</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fcr.2011.22&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcr.2011.22%26sid%3Dliteratum%253Aachs%26aulast%3DBannister%26aufirst%3DA.%2BJ.%26aulast%3DKouzarides%26aufirst%3DT.%26atitle%3DRegulation%2520of%2520chromatin%2520by%2520histone%2520modifications%26jtitle%3DCell%2520Res.%26date%3D2011%26volume%3D21%26spage%3D381%26epage%3D395%26doi%3D10.1038%2Fcr.2011.22" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Barski, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cuddapah, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cui, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roh, T. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schones, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chepelev, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, K.</span><span> </span><span class="NLM_article-title">High-resolution profiling of histone methylations in the human genome</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">129</span><span class="NLM_x">, </span> <span class="NLM_fpage">823</span><span class="NLM_x">–</span> <span class="NLM_lpage">837</span><span class="refDoi"> DOI: 10.1016/j.cell.2007.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1016%2Fj.cell.2007.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=17512414" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmtFKjsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2007&pages=823-837&author=A.+Barskiauthor=S.+Cuddapahauthor=K.+Cuiauthor=T.+Y.+Rohauthor=D.+E.+Schonesauthor=Z.+Wangauthor=G.+Weiauthor=I.+Chepelevauthor=K.+Zhao&title=High-resolution+profiling+of+histone+methylations+in+the+human+genome&doi=10.1016%2Fj.cell.2007.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">High-resolution profiling of histone methylations in the human genome</span></div><div class="casAuthors">Barski, Artem; Cuddapah, Suresh; Cui, Kairong; Roh, Tae-Young; Schones, Dustin E.; Wang, Zhibin; Wei, Gang; Chepelev, Iouri; Zhao, Keji</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">129</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">823-837</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Histone modifications are implicated in influencing gene expression.  We have generated high-resoln. maps for the genome-wide distribution of 20 histone lysine and arginine methylations as well as histone variant H2A.Z, RNA polymerase II, and the insulator binding protein CTCF across the human genome using the Solexa 1G sequencing technol.  Typical patterns of histone methylations exhibited at promoters, insulators, enhancers, and transcribed regions are identified.  The monomethylations of H3K27, H3K9, H4K20, H3K79, and H2BK5 are all linked to gene activation, whereas trimethylations of H3K27, H3K9, and H3K79 are linked to repression.  H2A.Z assocs. with functional regulatory elements, and CTCF marks boundaries of histone methylation domains.  Chromosome banding patterns are correlated with unique patterns of histone modifications.  Chromosome breakpoints detected in T cell cancers frequently reside in chromatin regions assocd. with H3K4 methylations.  Our data provide new insights into the function of histone methylation and chromatin organization in genome function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxkecQmWJMZbVg90H21EOLACvtfcHk0lgtsbAfFpklmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmtFKjsro%253D&md5=8e37e2d641f10e3cdd3ed51d1da82c96</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DBarski%26aufirst%3DA.%26aulast%3DCuddapah%26aufirst%3DS.%26aulast%3DCui%26aufirst%3DK.%26aulast%3DRoh%26aufirst%3DT.%2BY.%26aulast%3DSchones%26aufirst%3DD.%2BE.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DWei%26aufirst%3DG.%26aulast%3DChepelev%26aufirst%3DI.%26aulast%3DZhao%26aufirst%3DK.%26atitle%3DHigh-resolution%2520profiling%2520of%2520histone%2520methylations%2520in%2520the%2520human%2520genome%26jtitle%3DCell%26date%3D2007%26volume%3D129%26spage%3D823%26epage%3D837%26doi%3D10.1016%2Fj.cell.2007.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Zhang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pugh, B. F.</span><span> </span><span class="NLM_article-title">High-resolution genome-wide mapping of the primary structure of chromatin</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">144</span><span class="NLM_x">, </span> <span class="NLM_fpage">175</span><span class="NLM_x">–</span> <span class="NLM_lpage">186</span><span class="refDoi"> DOI: 10.1016/j.cell.2011.01.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1016%2Fj.cell.2011.01.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=21241889" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BC3MXntVagug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=144&publication_year=2011&pages=175-186&author=Z.+Zhangauthor=B.+F.+Pugh&title=High-resolution+genome-wide+mapping+of+the+primary+structure+of+chromatin&doi=10.1016%2Fj.cell.2011.01.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">High-Resolution Genome-wide Mapping of the Primary Structure of Chromatin</span></div><div class="casAuthors">Zhang, Zhenhai; Pugh, B. Franklin</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">144</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">175-186</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review primer.  The genomic organization of chromatin is increasingly recognized as a key regulator of cell behavior, but deciphering its regulation mechanisms requires detailed knowledge of chromatin's primary structure - the assembly of nucleosomes throughout the genome.  This Primer explains the principles for mapping and analyzing the primary organization of chromatin on a genomic scale.  After introducing chromatin organization and its impact on gene regulation and human health, the authors then describe methods that detect nucleosome positioning and occupancy levels using chromatin immunopptn. in combination with deep sequencing (ChIP-Seq), a strategy that is now straightforward and cost efficient.  The authors then explore current strategies for converting the sequence information into knowledge about chromatin, an exciting challenge for biologists and bioinformaticians.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaIXii3VQMsbVg90H21EOLACvtfcHk0lgtsbAfFpklmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXntVagug%253D%253D&md5=8fc3faf35d565a736df953165d665588</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2011.01.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2011.01.003%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DPugh%26aufirst%3DB.%2BF.%26atitle%3DHigh-resolution%2520genome-wide%2520mapping%2520of%2520the%2520primary%2520structure%2520of%2520chromatin%26jtitle%3DCell%26date%3D2011%26volume%3D144%26spage%3D175%26epage%3D186%26doi%3D10.1016%2Fj.cell.2011.01.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Kubicek, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Sullivan, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">August, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hickey, E. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teodoro, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rea, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mechtler, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kowalski, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Homon, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kelly, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenuwein, T.</span><span> </span><span class="NLM_article-title">Reversal of H3K9me2 by a small-molecule inhibitor for the G9a histone methyltransferase</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">473</span><span class="NLM_x">–</span> <span class="NLM_lpage">481</span><span class="refDoi"> DOI: 10.1016/j.molcel.2007.01.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1016%2Fj.molcel.2007.01.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=17289593" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BD2sXitl2jurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2007&pages=473-481&author=S.+Kubicekauthor=R.+J.+O%E2%80%99Sullivanauthor=E.+M.+Augustauthor=E.+R.+Hickeyauthor=Q.+Zhangauthor=M.+L.+Teodoroauthor=S.+Reaauthor=K.+Mechtlerauthor=J.+A.+Kowalskiauthor=C.+A.+Homonauthor=T.+A.+Kellyauthor=T.+Jenuwein&title=Reversal+of+H3K9me2+by+a+small-molecule+inhibitor+for+the+G9a+histone+methyltransferase&doi=10.1016%2Fj.molcel.2007.01.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Reversal of H3K9me2 by a small-molecule inhibitor for the G9a histone methyltransferase</span></div><div class="casAuthors">Kubicek, Stefan; O'Sullivan, Roderick J.; August, E. Michael; Hickey, Eugene R.; Zhang, Qiang; Teodoro, Miguel L.; Rea, Stephen; Mechtler, Karl; Kowalski, Jennifer A.; Homon, Carol Ann; Kelly, Terence A.; Jenuwein, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">473-481</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Histone lysine methylation has important roles in the organization of chromatin domains and the regulation of gene expression.  To analyze its function and modulate its activity, we screened for specific inhibitors against histone lysine methyltransferases (HMTases) using recombinant G9a as the target enzyme.  From a chem. library comprising 125,000 preselected compds., seven hits were identified.  Of those, one inhibitor, BIX-01294 (diazepinquinazolin-amine deriv.), does not compete with the cofactor S-adenosyl-methionine, and selectively impairs the G9a HMTase and the generation of H3K9me2 in vitro.  In cellular assays, transient incubation of several cell lines with BIX-01294 lowers bulk H3K9me2 levels that are restored upon removal of the inhibitor.  Importantly, chromatin immunopptn. at several G9a target genes demonstrates reversible redn. of promoter-proximal H3K9me2 in inhibitor-treated mouse ES cells and fibroblasts.  Our data identify a biol. active HMTase inhibitor that allows for the transient modulation of H3K9me2 marks in mammalian chromatin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3SW8czS4WjbVg90H21EOLACvtfcHk0lgtsbAfFpklmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXitl2jurs%253D&md5=d2f5290f1252e630a9b05489b0775c2c</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2007.01.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2007.01.017%26sid%3Dliteratum%253Aachs%26aulast%3DKubicek%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Sullivan%26aufirst%3DR.%2BJ.%26aulast%3DAugust%26aufirst%3DE.%2BM.%26aulast%3DHickey%26aufirst%3DE.%2BR.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DTeodoro%26aufirst%3DM.%2BL.%26aulast%3DRea%26aufirst%3DS.%26aulast%3DMechtler%26aufirst%3DK.%26aulast%3DKowalski%26aufirst%3DJ.%2BA.%26aulast%3DHomon%26aufirst%3DC.%2BA.%26aulast%3DKelly%26aufirst%3DT.%2BA.%26aulast%3DJenuwein%26aufirst%3DT.%26atitle%3DReversal%2520of%2520H3K9me2%2520by%2520a%2520small-molecule%2520inhibitor%2520for%2520the%2520G9a%2520histone%2520methyltransferase%26jtitle%3DMol.%2520Cell%26date%3D2007%26volume%3D25%26spage%3D473%26epage%3D481%26doi%3D10.1016%2Fj.molcel.2007.01.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Chang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horton, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Upadhyay, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spannhoff, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Snyder, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, X.</span><span> </span><span class="NLM_article-title">Structural basis for G9a-like protein lysine methyltransferase inhibition by BIX-01294</span> <span class="citation_source-journal">Nat. Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">312</span><span class="NLM_x">–</span> <span class="NLM_lpage">317</span><span class="refDoi"> DOI: 10.1038/nsmb.1560</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1038%2Fnsmb.1560" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=19219047" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhvFKjt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2009&pages=312-317&author=Y.+Changauthor=X.+Zhangauthor=J.+R.+Hortonauthor=A.+K.+Upadhyayauthor=A.+Spannhoffauthor=J.+Liuauthor=J.+P.+Snyderauthor=M.+T.+Bedfordauthor=X.+Cheng&title=Structural+basis+for+G9a-like+protein+lysine+methyltransferase+inhibition+by+BIX-01294&doi=10.1038%2Fnsmb.1560"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for G9a-like protein lysine methyltransferase inhibition by BIX-01294</span></div><div class="casAuthors">Chang, Yanqi; Zhang, Xing; Horton, John R.; Upadhyay, Anup K.; Spannhoff, Astrid; Liu, Jin; Snyder, James P.; Bedford, Mark T.; Cheng, Xiaodong</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">312-317</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Histone lysine methylation is an important epigenetic mark that regulates gene expression and chromatin organization.  G9a and G9a-like protein (GLP) are euchromatin-assocd. methyltransferases that repress transcription by methylating histone H3 Lys9.  BIX-01294 was originally identified as a G9a inhibitor during a chem. library screen of small mols. and has previously been used in the generation of induced pluripotent stem cells.  Here we present the crystal structure of the catalytic SET domain of GLP in complex with BIX-01294 and S-adenosyl-L-homocysteine.  The inhibitor is bound in the substrate peptide groove at the location where the histone H3 residues N-terminal to the target lysine lie in the previously solved structure of the complex with histone peptide.  The inhibitor resembles the bound conformation of histone H3 Lys4 to Arg8, and is positioned in place by residues specific for G9a and GLP through specific interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhQepqN3zr_bVg90H21EOLACvtfcHk0ljy_xGWsK-tNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhvFKjt7g%253D&md5=6a16baaf33420fc1caf1df70d103d270</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.1560&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.1560%26sid%3Dliteratum%253Aachs%26aulast%3DChang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DHorton%26aufirst%3DJ.%2BR.%26aulast%3DUpadhyay%26aufirst%3DA.%2BK.%26aulast%3DSpannhoff%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DSnyder%26aufirst%3DJ.%2BP.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DCheng%26aufirst%3DX.%26atitle%3DStructural%2520basis%2520for%2520G9a-like%2520protein%2520lysine%2520methyltransferase%2520inhibition%2520by%2520BIX-01294%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2009%26volume%3D16%26spage%3D312%26epage%3D317%26doi%3D10.1038%2Fnsmb.1560" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Tachibana, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sugimoto, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nozaki, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ueda, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohta, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohki, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fukuda, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takeda, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niida, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kato, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shinkai, Y.</span><span> </span><span class="NLM_article-title">G9a histone methyltransferase plays a dominant role in euchromatic histone H3 lysine 9 methylation and is essential for early embryogenesis</span> <span class="citation_source-journal">Genes Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">1779</span><span class="NLM_x">–</span> <span class="NLM_lpage">1791</span><span class="refDoi"> DOI: 10.1101/gad.989402</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1101%2Fgad.989402" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=12130538" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BD38Xls1Onurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2002&pages=1779-1791&author=M.+Tachibanaauthor=K.+Sugimotoauthor=M.+Nozakiauthor=J.+Uedaauthor=T.+Ohtaauthor=M.+Ohkiauthor=M.+Fukudaauthor=N.+Takedaauthor=H.+Niidaauthor=H.+Katoauthor=Y.+Shinkai&title=G9a+histone+methyltransferase+plays+a+dominant+role+in+euchromatic+histone+H3+lysine+9+methylation+and+is+essential+for+early+embryogenesis&doi=10.1101%2Fgad.989402"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">G9a histone methyltransferase plays a dominant role in euchromatic histone H3 lysine 9 methylation and is essential for early embryogenesis</span></div><div class="casAuthors">Tachibana, Makoto; Sugimoto, Kenji; Nozaki, Masami; Ueda, Jun; Ohta, Tsutomu; Ohki, Misao; Fukuda, Mikiko; Takeda, Naoki; Niida, Hiroyuki; Kato, Hiroyuki; Shinkai, Yoichi</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1779-1791</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Covalent modification of histone tails is crucial for transcriptional regulation, mitotic chromosomal condensation, and heterochromatin formation.  Histone H3 lysine 9 (H3-K9) methylation catalyzed by the Suv39h family proteins is essential for establishing the architecture of pericentric heterochromatin.  We recently identified a mammalian histone methyltransferase (HMTase), G9a, which has strong HMTase activity towards H3-K9 in vitro.  To investigate the in vivo functions of G9a, we generated G9a-deficient mice and embryonic stem (ES) cells.  We found that H3-K9 methylation was drastically decreased in G9a-deficient embryos, which displayed severe growth retardation and early lethality.  G9a-deficient ES cells also exhibited reduced H3-K9 methylation compared to wild-type cells, indicating that G9a is a dominant H3-K9 HMTase in vivo.  Importantly, the loss of G9a abolished methylated H3-K9 mostly in euchromatic regions.  Finally, G9a exerted a transcriptionally suppressive function that depended on its HMTase activity.  Our results indicate that euchromatic H3-K9 methylation regulated by G9a is essential for early embryogenesis and is involved in the transcriptional repression of developmental genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5EeLfBMH1_LVg90H21EOLACvtfcHk0ljy_xGWsK-tNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xls1Onurg%253D&md5=bde55b2c2d49fd61e5f0337d780a94db</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1101%2Fgad.989402&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.989402%26sid%3Dliteratum%253Aachs%26aulast%3DTachibana%26aufirst%3DM.%26aulast%3DSugimoto%26aufirst%3DK.%26aulast%3DNozaki%26aufirst%3DM.%26aulast%3DUeda%26aufirst%3DJ.%26aulast%3DOhta%26aufirst%3DT.%26aulast%3DOhki%26aufirst%3DM.%26aulast%3DFukuda%26aufirst%3DM.%26aulast%3DTakeda%26aufirst%3DN.%26aulast%3DNiida%26aufirst%3DH.%26aulast%3DKato%26aufirst%3DH.%26aulast%3DShinkai%26aufirst%3DY.%26atitle%3DG9a%2520histone%2520methyltransferase%2520plays%2520a%2520dominant%2520role%2520in%2520euchromatic%2520histone%2520H3%2520lysine%25209%2520methylation%2520and%2520is%2520essential%2520for%2520early%2520embryogenesis%26jtitle%3DGenes%2520Dev.%26date%3D2002%26volume%3D16%26spage%3D1779%26epage%3D1791%26doi%3D10.1101%2Fgad.989402" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Tachibana, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ueda, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fukuda, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takeda, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohta, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iwanari, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sakihama, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kodama, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamakubo, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shinkai, Y.</span><span> </span><span class="NLM_article-title">Histone methyltransferases G9a and GLP form heteromeric complexes and are both crucial for methylation of euchromatin at H3-K9</span> <span class="citation_source-journal">Genes Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">815</span><span class="NLM_x">–</span> <span class="NLM_lpage">826</span><span class="refDoi"> DOI: 10.1101/gad.1284005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1101%2Fgad.1284005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=15774718" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjt1OjtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2005&pages=815-826&author=M.+Tachibanaauthor=J.+Uedaauthor=M.+Fukudaauthor=N.+Takedaauthor=T.+Ohtaauthor=H.+Iwanariauthor=T.+Sakihamaauthor=T.+Kodamaauthor=T.+Hamakuboauthor=Y.+Shinkai&title=Histone+methyltransferases+G9a+and+GLP+form+heteromeric+complexes+and+are+both+crucial+for+methylation+of+euchromatin+at+H3-K9&doi=10.1101%2Fgad.1284005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Histone methyltransferases G9a and GLP form heteromeric complexes and are both crucial for methylation of euchromatin at H3-K9</span></div><div class="casAuthors">Tachibana, Makoto; Ueda, Jun; Fukuda, Mikiko; Takeda, Naoki; Ohta, Tsutomu; Iwanari, Hiroko; Sakihama, Toshiko; Kodama, Tatsuhiko; Hamakubo, Takao; Shinkai, Yoichi</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">815-826</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Histone H3 Lys 9 (H3-K9) methylation is a crucial epigenetic mark for transcriptional silencing.  G9a is the major mammalian H3-K9 methyltransferase that targets euchromatic regions and is essential for murine embryogenesis.  There is a single G9a-related methyltransferase in mammals, called GLP/Eu-HMTase1.  Here the authors show that GLP is also important for H3-K9 methylation of mouse euchromatin.  GLP-deficiency led to embryonic lethality, a severe redn. of H3-K9 mono- and dimethylation, the induction of Mage-a gene expression, and HP1 relocalization in embryonic stem cells, all of which were phenotypes of G9a-deficiency.  Furthermore, the authors show that G9a and GLP formed a stoichiometric heteromeric complex in a wide variety of cell types.  Biochem. analyses revealed that formation of the G9a/GLP complex was dependent on their enzymic SET domains.  Taken together, these new findings revealed that G9a and GLP cooperatively exert H3-K9 methyltransferase function in vivo, likely through the formation of higher-order heteromeric complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUXBASXATfh7Vg90H21EOLACvtfcHk0ljy_xGWsK-tNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjt1OjtLc%253D&md5=cd0b7f8b9f62b4abe81b5ea590ab218c</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1101%2Fgad.1284005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.1284005%26sid%3Dliteratum%253Aachs%26aulast%3DTachibana%26aufirst%3DM.%26aulast%3DUeda%26aufirst%3DJ.%26aulast%3DFukuda%26aufirst%3DM.%26aulast%3DTakeda%26aufirst%3DN.%26aulast%3DOhta%26aufirst%3DT.%26aulast%3DIwanari%26aufirst%3DH.%26aulast%3DSakihama%26aufirst%3DT.%26aulast%3DKodama%26aufirst%3DT.%26aulast%3DHamakubo%26aufirst%3DT.%26aulast%3DShinkai%26aufirst%3DY.%26atitle%3DHistone%2520methyltransferases%2520G9a%2520and%2520GLP%2520form%2520heteromeric%2520complexes%2520and%2520are%2520both%2520crucial%2520for%2520methylation%2520of%2520euchromatin%2520at%2520H3-K9%26jtitle%3DGenes%2520Dev.%26date%3D2005%26volume%3D19%26spage%3D815%26epage%3D826%26doi%3D10.1101%2Fgad.1284005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Trojer, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yonezawa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmidt, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenuwein, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinberg, D.</span><span> </span><span class="NLM_article-title">Dynamic histone H1 isotype 4 methylation and demethylation by histone lysine methyltransferase G9a/KMT1C and the jumonji domain-containing JMJD2/KDM4 proteins</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">284</span><span class="NLM_x">, </span> <span class="NLM_fpage">8395</span><span class="NLM_x">–</span> <span class="NLM_lpage">8405</span><span class="refDoi"> DOI: 10.1074/jbc.M807818200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1074%2Fjbc.M807818200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=8395-8405&author=P.+Trojerauthor=J.+Zhangauthor=M.+Yonezawaauthor=A.+Schmidtauthor=H.+Zhengauthor=T.+Jenuweinauthor=D.+Reinberg&title=Dynamic+histone+H1+isotype+4+methylation+and+demethylation+by+histone+lysine+methyltransferase+G9a%2FKMT1C+and+the+jumonji+domain-containing+JMJD2%2FKDM4+proteins&doi=10.1074%2Fjbc.M807818200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M807818200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M807818200%26sid%3Dliteratum%253Aachs%26aulast%3DTrojer%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DYonezawa%26aufirst%3DM.%26aulast%3DSchmidt%26aufirst%3DA.%26aulast%3DZheng%26aufirst%3DH.%26aulast%3DJenuwein%26aufirst%3DT.%26aulast%3DReinberg%26aufirst%3DD.%26atitle%3DDynamic%2520histone%2520H1%2520isotype%25204%2520methylation%2520and%2520demethylation%2520by%2520histone%2520lysine%2520methyltransferase%2520G9a%252FKMT1C%2520and%2520the%2520jumonji%2520domain-containing%2520JMJD2%252FKDM4%2520proteins%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2009%26volume%3D284%26spage%3D8395%26epage%3D8405%26doi%3D10.1074%2Fjbc.M807818200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Weiss, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hergeth, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeissler, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Izzo, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tropberger, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zee, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dundr, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daujat, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneider, R.</span><span> </span><span class="NLM_article-title">Histone H1 variant-specific lysine methylation by G9a/KMT1C and Glp1/KMT1D</span> <span class="citation_source-journal">Epigenet. Chromatin</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">7</span><span class="refDoi"> DOI: 10.1186/1756-8935-3-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1186%2F1756-8935-3-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=20334638" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A280%3ADC%252BC3c3pt1Clsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2010&pages=7&author=T.+Weissauthor=S.+Hergethauthor=U.+Zeisslerauthor=A.+Izzoauthor=P.+Tropbergerauthor=B.+M.+Zeeauthor=M.+Dundrauthor=B.+A.+Garciaauthor=S.+Daujatauthor=R.+Schneider&title=Histone+H1+variant-specific+lysine+methylation+by+G9a%2FKMT1C+and+Glp1%2FKMT1D&doi=10.1186%2F1756-8935-3-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Histone H1 variant-specific lysine methylation by G9a/KMT1C and Glp1/KMT1D</span></div><div class="casAuthors">Weiss Thomas; Hergeth Sonja; Zeissler Ulrike; Izzo Annalisa; Tropberger Philipp; Zee Barry M; Dundr Miroslav; Garcia Benjamin A; Daujat Sylvain; Schneider Robert</div><div class="citationInfo"><span class="NLM_cas:title">Epigenetics & chromatin</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">7</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  The linker histone H1 has a key role in establishing and maintaining higher order chromatin structure and in regulating gene expression.  Mammals express up to 11 different H1 variants, with H1.2 and H1.4 being the predominant ones in most somatic cells.  Like core histones, H1 has high levels of covalent modifications; however, the full set of modifications and their biological role are largely unknown.  RESULTS:  In this study, we used a candidate screen to identify enzymes that methylate H1 and to map their corresponding methylation sites.  We found that the histone lysine methyltransferases G9a/KMT1C and Glp1/KMT1D methylate H1.2 in vitro and in vivo, and we mapped this novel site to lysine 187 (H1.2K187) in the C-terminus of H1.  This H1.2K187 methylation is variant-specific.  The main target for methylation by G9a in H1.2, H1.3, H1.5 and H1.0 is in the C-terminus, whereas H1.4 is preferentially methylated at K26 (H1.4K26me) in the N-terminus.  We found that the readout of these marks is different; H1.4K26me can recruit HP1, but H1.2K187me cannot.  Likewise, JMJD2D/KDM4 only reverses H1.4K26 methylation, clearly distinguishing these two methylation sites.  Further, in contrast to C-terminal H1 phosphorylation, H1.2K187 methylation level is steady throughout the cell cycle.  CONCLUSIONS:  We have characterised a novel methylation site in the C-terminus of H1 that is the target of G9a/Glp1 both in vitro and in vivo.  To our knowledge, this is the first demonstration of variant-specific histone methylation by the same methyltransferases, but with differing downstream readers, thereby supporting the hypothesis of H1 variants having specific functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRiHGsW3wTCe1HFYivNCdPRfW6udTcc2ebDZ94vF7RT67ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3c3pt1Clsg%253D%253D&md5=a140e14a1dfff30b74e9fbef28091475</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1186%2F1756-8935-3-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1756-8935-3-7%26sid%3Dliteratum%253Aachs%26aulast%3DWeiss%26aufirst%3DT.%26aulast%3DHergeth%26aufirst%3DS.%26aulast%3DZeissler%26aufirst%3DU.%26aulast%3DIzzo%26aufirst%3DA.%26aulast%3DTropberger%26aufirst%3DP.%26aulast%3DZee%26aufirst%3DB.%2BM.%26aulast%3DDundr%26aufirst%3DM.%26aulast%3DGarcia%26aufirst%3DB.%2BA.%26aulast%3DDaujat%26aufirst%3DS.%26aulast%3DSchneider%26aufirst%3DR.%26atitle%3DHistone%2520H1%2520variant-specific%2520lysine%2520methylation%2520by%2520G9a%252FKMT1C%2520and%2520Glp1%252FKMT1D%26jtitle%3DEpigenet.%2520Chromatin%26date%3D2010%26volume%3D3%26spage%3D7%26doi%3D10.1186%2F1756-8935-3-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Patnaik, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chin, H. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Esteve, P. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benner, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobsen, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pradhan, S.</span><span> </span><span class="NLM_article-title">Substrate specificity and kinetic mechanism of mammalian G9a histone H3 methyltransferase</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">279</span><span class="NLM_x">, </span> <span class="NLM_fpage">53248</span><span class="NLM_x">–</span> <span class="NLM_lpage">53258</span><span class="refDoi"> DOI: 10.1074/jbc.M409604200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1074%2Fjbc.M409604200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=279&publication_year=2004&pages=53248-53258&author=D.+Patnaikauthor=H.+G.+Chinauthor=P.+O.+Esteveauthor=J.+Bennerauthor=S.+E.+Jacobsenauthor=S.+Pradhan&title=Substrate+specificity+and+kinetic+mechanism+of+mammalian+G9a+histone+H3+methyltransferase&doi=10.1074%2Fjbc.M409604200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M409604200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M409604200%26sid%3Dliteratum%253Aachs%26aulast%3DPatnaik%26aufirst%3DD.%26aulast%3DChin%26aufirst%3DH.%2BG.%26aulast%3DEsteve%26aufirst%3DP.%2BO.%26aulast%3DBenner%26aufirst%3DJ.%26aulast%3DJacobsen%26aufirst%3DS.%2BE.%26aulast%3DPradhan%26aufirst%3DS.%26atitle%3DSubstrate%2520specificity%2520and%2520kinetic%2520mechanism%2520of%2520mammalian%2520G9a%2520histone%2520H3%2520methyltransferase%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2004%26volume%3D279%26spage%3D53248%26epage%3D53258%26doi%3D10.1074%2Fjbc.M409604200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Tachibana, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sugimoto, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fukushima, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shinkai, Y.</span><span> </span><span class="NLM_article-title">SET domain-containing protein, G9a, is a novel lysine-preferring mammalian histone methyltransferase with hyperactivity and specific selectivity to lysines 9 and 27 of histone H3</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">276</span><span class="NLM_x">, </span> <span class="NLM_fpage">25309</span><span class="NLM_x">–</span> <span class="NLM_lpage">25317</span><span class="refDoi"> DOI: 10.1074/jbc.M101914200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1074%2Fjbc.M101914200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=2001&pages=25309-25317&author=M.+Tachibanaauthor=K.+Sugimotoauthor=T.+Fukushimaauthor=Y.+Shinkai&title=SET+domain-containing+protein%2C+G9a%2C+is+a+novel+lysine-preferring+mammalian+histone+methyltransferase+with+hyperactivity+and+specific+selectivity+to+lysines+9+and+27+of+histone+H3&doi=10.1074%2Fjbc.M101914200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M101914200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M101914200%26sid%3Dliteratum%253Aachs%26aulast%3DTachibana%26aufirst%3DM.%26aulast%3DSugimoto%26aufirst%3DK.%26aulast%3DFukushima%26aufirst%3DT.%26aulast%3DShinkai%26aufirst%3DY.%26atitle%3DSET%2520domain-containing%2520protein%252C%2520G9a%252C%2520is%2520a%2520novel%2520lysine-preferring%2520mammalian%2520histone%2520methyltransferase%2520with%2520hyperactivity%2520and%2520specific%2520selectivity%2520to%2520lysines%25209%2520and%252027%2520of%2520histone%2520H3%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2001%26volume%3D276%26spage%3D25309%26epage%3D25317%26doi%3D10.1074%2Fjbc.M101914200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiong, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, X. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, B.</span><span> </span><span class="NLM_article-title">Histone methyltransferase G9a contributes to H3K27 methylation in vivo</span> <span class="citation_source-journal">Cell Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">365</span><span class="NLM_x">–</span> <span class="NLM_lpage">367</span><span class="refDoi"> DOI: 10.1038/cr.2010.157</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1038%2Fcr.2010.157" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=21079650" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVOmurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=365-367&author=H.+Wuauthor=X.+Z.+Chenauthor=J.+Xiongauthor=Y.+F.+Liauthor=H.+Liauthor=X.+J.+Dingauthor=S.+Liuauthor=S.+Chenauthor=S.+R.+Gaoauthor=B.+Zhu&title=Histone+methyltransferase+G9a+contributes+to+H3K27+methylation+in+vivo&doi=10.1038%2Fcr.2010.157"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Histone methyltransferase G9a contributes to H3K27 methylation in vivo</span></div><div class="casAuthors">Wu, Hui; Chen, Xiuzhen; Xiong, Jun; Li, Yingfeng; Li, Hong; Ding, Xiaojun; Liu, Sheng; Chen, She; Gao, Shaorong; Zhu, Bing</div><div class="citationInfo"><span class="NLM_cas:title">Cell Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">365-367</span>CODEN:
                <span class="NLM_cas:coden">CREEB6</span>;
        ISSN:<span class="NLM_cas:issn">1001-0602</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The role of G9a histone methyltransferase in H3K27 methylation was studied.  His-tagged H3 histones contg. K9C mutation were converted into His-H3Kc9me2 with the MLA reaction to mimic homogeneously di-methylated H3K9.  Recombinant G9a displayed robust activity on nucleosomes assembled with H3Kc9me2, which suggests that either G9a may be a trimethylase or may react with other H3 lysine residues on a nucleosomal substrate.  Thus, the K27A mutation was introduced into the His-tagged pre-methylated H3, which nearly completely abolished G9a's activity, indicating that G9a is mainly a dimethylase and also reacts with H3K27.  Current results collectively suggest G9a's contribution to H3K27 methylation in addn. to its well-characterized role in H3K9 methylation in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8nL82DjJqz7Vg90H21EOLACvtfcHk0liTCNLwQQv5AA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVOmurc%253D&md5=c1796390312963ef66e77a749f508e9b</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fcr.2010.157&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcr.2010.157%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DX.%2BZ.%26aulast%3DXiong%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DY.%2BF.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DDing%26aufirst%3DX.%2BJ.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DGao%26aufirst%3DS.%2BR.%26aulast%3DZhu%26aufirst%3DB.%26atitle%3DHistone%2520methyltransferase%2520G9a%2520contributes%2520to%2520H3K27%2520methylation%2520in%2520vivo%26jtitle%3DCell%2520Res.%26date%3D2011%26volume%3D21%26spage%3D365%26epage%3D367%26doi%3D10.1038%2Fcr.2010.157" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Yu, Y. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, C. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Q. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DiMaggio, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">York, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carey, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grunstein, M.</span><span> </span><span class="NLM_article-title">Histone H3 lysine 56 methylation regulates DNA replication through its interaction with PCNA</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">7</span><span class="NLM_x">–</span> <span class="NLM_lpage">17</span><span class="refDoi"> DOI: 10.1016/j.molcel.2012.01.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1016%2Fj.molcel.2012.01.019" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2012&pages=7-17&author=Y.+X.+Yuauthor=C.+Y.+Songauthor=Q.+Y.+Zhangauthor=P.+A.+DiMaggioauthor=B.+A.+Garciaauthor=A.+Yorkauthor=M.+F.+Careyauthor=M.+Grunstein&title=Histone+H3+lysine+56+methylation+regulates+DNA+replication+through+its+interaction+with+PCNA&doi=10.1016%2Fj.molcel.2012.01.019"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2012.01.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2012.01.019%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DY.%2BX.%26aulast%3DSong%26aufirst%3DC.%2BY.%26aulast%3DZhang%26aufirst%3DQ.%2BY.%26aulast%3DDiMaggio%26aufirst%3DP.%2BA.%26aulast%3DGarcia%26aufirst%3DB.%2BA.%26aulast%3DYork%26aufirst%3DA.%26aulast%3DCarey%26aufirst%3DM.%2BF.%26aulast%3DGrunstein%26aufirst%3DM.%26atitle%3DHistone%2520H3%2520lysine%252056%2520methylation%2520regulates%2520DNA%2520replication%2520through%2520its%2520interaction%2520with%2520PCNA%26jtitle%3DMol.%2520Cell%26date%3D2012%26volume%3D46%26spage%3D7%26epage%3D17%26doi%3D10.1016%2Fj.molcel.2012.01.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Biggar, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, S. S. C.</span><span> </span><span class="NLM_article-title">Non-histone protein methylation as a regulator of cellular signalling and function</span> <span class="citation_source-journal">Nat. Rev. Mol. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">5</span><span class="NLM_x">–</span> <span class="NLM_lpage">17</span><span class="refDoi"> DOI: 10.1038/nrm3915</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1038%2Fnrm3915" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=25491103" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitV2nsrzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2015&pages=5-17&author=K.+K.+Biggarauthor=S.+S.+C.+Li&title=Non-histone+protein+methylation+as+a+regulator+of+cellular+signalling+and+function&doi=10.1038%2Fnrm3915"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Non-histone protein methylation as a regulator of cellular signalling and function</span></div><div class="casAuthors">Biggar, Kyle K.; Li, Shawn S.-C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5-17</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Methylation of Lys and Arg residues on non-histone proteins has emerged as a prevalent post-translational modification and as an important regulator of cellular signal transduction mediated by the MAPK, WNT, BMP, Hippo and JAK-STAT signalling pathways.  Crosstalk between methylation and other types of post-translational modifications, and between histone and non-histone protein methylation frequently occurs and affects cellular functions such as chromatin remodelling, gene transcription, protein synthesis, signal transduction and DNA repair.  With recent advances in proteomic techniques, in particular mass spectrometry, the stage is now set to decode the methylproteome and define its functions in health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5E-jYGUpGCbVg90H21EOLACvtfcHk0ljZkMlGL--Rnw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitV2nsrzM&md5=3f60179eadac0ad72295b516003ec048</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fnrm3915&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm3915%26sid%3Dliteratum%253Aachs%26aulast%3DBiggar%26aufirst%3DK.%2BK.%26aulast%3DLi%26aufirst%3DS.%2BS.%2BC.%26atitle%3DNon-histone%2520protein%2520methylation%2520as%2520a%2520regulator%2520of%2520cellular%2520signalling%2520and%2520function%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2015%26volume%3D16%26spage%3D5%26epage%3D17%26doi%3D10.1038%2Fnrm3915" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Huang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dorsey, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chuikov, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perez-Burgos, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenuwein, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinberg, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berger, S. L.</span><span> </span><span class="NLM_article-title">G9a and Glp methylate lysine 373 in the tumor suppressor p53</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">285</span><span class="NLM_x">, </span> <span class="NLM_fpage">18122</span><span class="NLM_x">–</span> <span class="NLM_lpage">18122</span><span class="refDoi"> DOI: 10.1074/jbc.A109.062588</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1074%2Fjbc.A109.062588" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=18122-18122&author=J.+Huangauthor=J.+Dorseyauthor=S.+Chuikovauthor=L.+Perez-Burgosauthor=X.+Y.+Zhangauthor=T.+Jenuweinauthor=D.+Reinbergauthor=S.+L.+Berger&title=G9a+and+Glp+methylate+lysine+373+in+the+tumor+suppressor+p53&doi=10.1074%2Fjbc.A109.062588"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1074%2Fjbc.A109.062588&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.A109.062588%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DDorsey%26aufirst%3DJ.%26aulast%3DChuikov%26aufirst%3DS.%26aulast%3DPerez-Burgos%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DX.%2BY.%26aulast%3DJenuwein%26aufirst%3DT.%26aulast%3DReinberg%26aufirst%3DD.%26aulast%3DBerger%26aufirst%3DS.%2BL.%26atitle%3DG9a%2520and%2520Glp%2520methylate%2520lysine%2520373%2520in%2520the%2520tumor%2520suppressor%2520p53%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D18122%26epage%3D18122%26doi%3D10.1074%2Fjbc.A109.062588" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Moore, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlson, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Camp, N. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheung, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">James, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chua, K. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolf-Yadlin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gozani, O.</span><span> </span><span class="NLM_article-title">A general molecular affinity strategy for global detection and proteomic analysis of lysine methylation</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">444</span><span class="NLM_x">–</span> <span class="NLM_lpage">456</span><span class="refDoi"> DOI: 10.1016/j.molcel.2013.03.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1016%2Fj.molcel.2013.03.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=23583077" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlvVCgsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2013&pages=444-456&author=K.+E.+Mooreauthor=S.+M.+Carlsonauthor=N.+D.+Campauthor=P.+Cheungauthor=R.+G.+Jamesauthor=K.+F.+Chuaauthor=A.+Wolf-Yadlinauthor=O.+Gozani&title=A+general+molecular+affinity+strategy+for+global+detection+and+proteomic+analysis+of+lysine+methylation&doi=10.1016%2Fj.molcel.2013.03.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">A General Molecular Affinity Strategy for Global Detection and Proteomic Analysis of Lysine Methylation</span></div><div class="casAuthors">Moore, Kaitlyn E.; Carlson, Scott M.; Camp, Nathan D.; Cheung, Peggie; James, Richard G.; Chua, Katrin F.; Wolf-Yadlin, Alejandro; Gozani, Or</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">444-456</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Lysine methylation of histone proteins regulates chromatin dynamics and plays important roles in diverse physiol. and pathol. processes.  However, beyond histone proteins, the proteome-wide extent of lysine methylation remains largely unknown.  We have engineered the naturally occurring MBT domain repeats of L3MBTL1 to serve as a universal affinity reagent for detecting, enriching, and identifying proteins carrying a mono- or dimethylated lysine.  The domain is broadly specific for methylated lysine (pan-specific) and can be applied to any biol. system.  We have used our approach to demonstrate that SIRT1 is a substrate of the methyltransferase G9a both in vitro and in cells, to perform proteome-wide detection and enrichment of methylated proteins, and to identify candidate in-cell substrates of G9a and the related methyltransferase GLP.  Together, our results demonstrate a powerful new approach for global and quant. anal. of methylated lysine, and they represent the first systems biol. understanding of lysine methylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr245nYBEsNyLVg90H21EOLACvtfcHk0ljZkMlGL--Rnw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlvVCgsr4%253D&md5=a826252b90d161331eefcb18a7340574</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2013.03.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2013.03.005%26sid%3Dliteratum%253Aachs%26aulast%3DMoore%26aufirst%3DK.%2BE.%26aulast%3DCarlson%26aufirst%3DS.%2BM.%26aulast%3DCamp%26aufirst%3DN.%2BD.%26aulast%3DCheung%26aufirst%3DP.%26aulast%3DJames%26aufirst%3DR.%2BG.%26aulast%3DChua%26aufirst%3DK.%2BF.%26aulast%3DWolf-Yadlin%26aufirst%3DA.%26aulast%3DGozani%26aufirst%3DO.%26atitle%3DA%2520general%2520molecular%2520affinity%2520strategy%2520for%2520global%2520detection%2520and%2520proteomic%2520analysis%2520of%2520lysine%2520methylation%26jtitle%3DMol.%2520Cell%26date%3D2013%26volume%3D50%26spage%3D444%26epage%3D456%26doi%3D10.1016%2Fj.molcel.2013.03.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Lee, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, I. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choi, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shin, H. J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, J. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seo, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Binda, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gozani, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Semenza, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, K. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hwang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baek, S. H.</span><span> </span><span class="NLM_article-title">Negative regulation of hypoxic responses via induced reptin methylation</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x">, </span> <span class="NLM_fpage">71</span><span class="NLM_x">–</span> <span class="NLM_lpage">85</span><span class="refDoi"> DOI: 10.1016/j.molcel.2010.06.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1016%2Fj.molcel.2010.06.008" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2010&pages=71-85&author=J.+S.+Leeauthor=Y.+Kimauthor=I.+S.+Kimauthor=B.+Kimauthor=H.+J.+Choiauthor=J.+M.+Leeauthor=H.+J.+R.+Shinauthor=J.+H.+Kimauthor=J.+Y.+Kimauthor=S.+B.+Seoauthor=H.+Leeauthor=O.+Bindaauthor=O.+Gozaniauthor=G.+L.+Semenzaauthor=M.+Kimauthor=K.+I.+Kimauthor=D.+Hwangauthor=S.+H.+Baek&title=Negative+regulation+of+hypoxic+responses+via+induced+reptin+methylation&doi=10.1016%2Fj.molcel.2010.06.008"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2010.06.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2010.06.008%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DJ.%2BS.%26aulast%3DKim%26aufirst%3DY.%26aulast%3DKim%26aufirst%3DI.%2BS.%26aulast%3DKim%26aufirst%3DB.%26aulast%3DChoi%26aufirst%3DH.%2BJ.%26aulast%3DLee%26aufirst%3DJ.%2BM.%26aulast%3DShin%26aufirst%3DH.%2BJ.%2BR.%26aulast%3DKim%26aufirst%3DJ.%2BH.%26aulast%3DKim%26aufirst%3DJ.%2BY.%26aulast%3DSeo%26aufirst%3DS.%2BB.%26aulast%3DLee%26aufirst%3DH.%26aulast%3DBinda%26aufirst%3DO.%26aulast%3DGozani%26aufirst%3DO.%26aulast%3DSemenza%26aufirst%3DG.%2BL.%26aulast%3DKim%26aufirst%3DM.%26aulast%3DKim%26aufirst%3DK.%2BI.%26aulast%3DHwang%26aufirst%3DD.%26aulast%3DBaek%26aufirst%3DS.%2BH.%26atitle%3DNegative%2520regulation%2520of%2520hypoxic%2520responses%2520via%2520induced%2520reptin%2520methylation%26jtitle%3DMol.%2520Cell%26date%3D2010%26volume%3D39%26spage%3D71%26epage%3D85%26doi%3D10.1016%2Fj.molcel.2010.06.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Ling, B. M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bharathy, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, T. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kok, W. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tan, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rao, V. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gopinadhan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sartorelli, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walsh, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taneja, R.</span><span> </span><span class="NLM_article-title">Lysine methyltransferase G9a methylates the transcription factor MyoD and regulates skeletal muscle differentiation</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">109</span><span class="NLM_x">, </span> <span class="NLM_fpage">841</span><span class="NLM_x">–</span> <span class="NLM_lpage">846</span><span class="refDoi"> DOI: 10.1073/pnas.1111628109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1073%2Fpnas.1111628109" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=841-846&author=B.+M.+T.+Lingauthor=N.+Bharathyauthor=T.+K.+Chungauthor=W.+K.+Kokauthor=S.+Liauthor=Y.+H.+Tanauthor=V.+K.+Raoauthor=S.+Gopinadhanauthor=V.+Sartorelliauthor=M.+J.+Walshauthor=R.+Taneja&title=Lysine+methyltransferase+G9a+methylates+the+transcription+factor+MyoD+and+regulates+skeletal+muscle+differentiation&doi=10.1073%2Fpnas.1111628109"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1111628109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1111628109%26sid%3Dliteratum%253Aachs%26aulast%3DLing%26aufirst%3DB.%2BM.%2BT.%26aulast%3DBharathy%26aufirst%3DN.%26aulast%3DChung%26aufirst%3DT.%2BK.%26aulast%3DKok%26aufirst%3DW.%2BK.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DTan%26aufirst%3DY.%2BH.%26aulast%3DRao%26aufirst%3DV.%2BK.%26aulast%3DGopinadhan%26aufirst%3DS.%26aulast%3DSartorelli%26aufirst%3DV.%26aulast%3DWalsh%26aufirst%3DM.%2BJ.%26aulast%3DTaneja%26aufirst%3DR.%26atitle%3DLysine%2520methyltransferase%2520G9a%2520methylates%2520the%2520transcription%2520factor%2520MyoD%2520and%2520regulates%2520skeletal%2520muscle%2520differentiation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26spage%3D841%26epage%3D846%26doi%3D10.1073%2Fpnas.1111628109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Rathert, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhayalan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murakami, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tamas, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jurkowska, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Komatsu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shinkai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, X. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeltsch, A.</span><span> </span><span class="NLM_article-title">Protein lysine methyltransferase G9a acts on non-histone targets</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">344</span><span class="NLM_x">–</span> <span class="NLM_lpage">346</span><span class="refDoi"> DOI: 10.1038/nchembio.88</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1038%2Fnchembio.88" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=18438403" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlvFOisrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2008&pages=344-346&author=P.+Rathertauthor=A.+Dhayalanauthor=M.+Murakamiauthor=X.+Zhangauthor=R.+Tamasauthor=R.+Jurkowskaauthor=Y.+Komatsuauthor=Y.+Shinkaiauthor=X.+D.+Chengauthor=A.+Jeltsch&title=Protein+lysine+methyltransferase+G9a+acts+on+non-histone+targets&doi=10.1038%2Fnchembio.88"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Protein lysine methyltransferase G9a acts on non-histone targets</span></div><div class="casAuthors">Rathert, Philipp; Dhayalan, Arunkumar; Murakami, Marie; Zhang, Xing; Tamas, Raluca; Jurkowska, Renata; Komatsu, Yasuhiko; Shinkai, Yoichi; Cheng, Xiaodong; Jeltsch, Albert</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">344-346</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">By methylation of peptide arrays, we detd. the specificity profile of the protein methyltransferase G9a.  We show that it mostly recognizes an Arg-Lys sequence and that its activity is inhibited by methylation of the arginine residue.  Using the specificity profile, we identified new non-histone protein targets of G9a, including CDYL1, WIZ, ACINUS and G9a (automethylation), as well as peptides derived from CSB.  We demonstrate potential downstream signaling pathways for methylation of non-histone proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo89lj_ybb7_7Vg90H21EOLACvtfcHk0ljVc8wrGYl8RA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlvFOisrs%253D&md5=acebc9dbbe888944e1d9775b78e2483b</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.88&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.88%26sid%3Dliteratum%253Aachs%26aulast%3DRathert%26aufirst%3DP.%26aulast%3DDhayalan%26aufirst%3DA.%26aulast%3DMurakami%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DTamas%26aufirst%3DR.%26aulast%3DJurkowska%26aufirst%3DR.%26aulast%3DKomatsu%26aufirst%3DY.%26aulast%3DShinkai%26aufirst%3DY.%26aulast%3DCheng%26aufirst%3DX.%2BD.%26aulast%3DJeltsch%26aufirst%3DA.%26atitle%3DProtein%2520lysine%2520methyltransferase%2520G9a%2520acts%2520on%2520non-histone%2520targets%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2008%26volume%3D4%26spage%3D344%26epage%3D346%26doi%3D10.1038%2Fnchembio.88" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Sampath, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marazzi, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yap, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sampath, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krutchinsky, A. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mecklenbrauker, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Viale, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rudensky, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chait, B. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tarakhovsky, A.</span><span> </span><span class="NLM_article-title">Methylation of a histone mimic within the histone methyltransferase G9a regulates protein complex assembly</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">596</span><span class="NLM_x">–</span> <span class="NLM_lpage">608</span><span class="refDoi"> DOI: 10.1016/j.molcel.2007.06.026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1016%2Fj.molcel.2007.06.026" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=17707231" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVSmtL3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2007&pages=596-608&author=S.+C.+Sampathauthor=I.+Marazziauthor=K.+L.+Yapauthor=S.+C.+Sampathauthor=A.+N.+Krutchinskyauthor=I.+Mecklenbraukerauthor=A.+Vialeauthor=E.+Rudenskyauthor=M.+M.+Zhouauthor=B.+T.+Chaitauthor=A.+Tarakhovsky&title=Methylation+of+a+histone+mimic+within+the+histone+methyltransferase+G9a+regulates+protein+complex+assembly&doi=10.1016%2Fj.molcel.2007.06.026"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Methylation of a histone mimic within the histone methyltransferase G9a regulates protein complex assembly</span></div><div class="casAuthors">Sampath, Srihari C.; Marazzi, Ivan; Yap, Kyoko L.; Sampath, Srinath C.; Krutchinsky, Andrew N.; Mecklenbrauker, Ingrid; Viale, Agnes; Rudensky, Eugene; Zhou, Ming-Ming; Chait, Brian T.; Tarakhovsky, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">596-608</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Epigenetic gene silencing in eukaryotes is regulated in part by lysine methylation of the core histone proteins.  While histone lysine methylation is known to control gene expression through the recruitment of modification-specific effector proteins, it remains unknown whether nonhistone chromatin proteins are targets for similar modification-recognition systems.  Here we show that the histone H3 methyltransferase G9a contains a conserved methylation motif with marked sequence similarity to H3 itself.  As with methylation of H3 lysine 9, autocatalytic G9a methylation is necessary and sufficient to mediate in vivo interaction with the epigenetic regulator heterochromatin protein 1 (HP1), and this methyl-dependent interaction can be reversed by adjacent G9a phosphorylation.  NMR anal. indicates that the HP1 chromodomain recognizes methyl-G9a through a binding mode similar to that used in recognition of methyl-H3K9, demonstrating that the chromodomain functions as a generalized methyl-lysine binding module.  These data reveal histone-like modification cassettes-or "histone mimics"-as a distinct class of nonhistone methylation targets and directly extend the principles of the histone code to the regulation of nonhistone proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFIcB4m0CyVbVg90H21EOLACvtfcHk0ljVc8wrGYl8RA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVSmtL3J&md5=3002c1fd27ba6061da0cef626680b504</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2007.06.026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2007.06.026%26sid%3Dliteratum%253Aachs%26aulast%3DSampath%26aufirst%3DS.%2BC.%26aulast%3DMarazzi%26aufirst%3DI.%26aulast%3DYap%26aufirst%3DK.%2BL.%26aulast%3DSampath%26aufirst%3DS.%2BC.%26aulast%3DKrutchinsky%26aufirst%3DA.%2BN.%26aulast%3DMecklenbrauker%26aufirst%3DI.%26aulast%3DViale%26aufirst%3DA.%26aulast%3DRudensky%26aufirst%3DE.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26aulast%3DChait%26aufirst%3DB.%2BT.%26aulast%3DTarakhovsky%26aufirst%3DA.%26atitle%3DMethylation%2520of%2520a%2520histone%2520mimic%2520within%2520the%2520histone%2520methyltransferase%2520G9a%2520regulates%2520protein%2520complex%2520assembly%26jtitle%3DMol.%2520Cell%26date%3D2007%26volume%3D27%26spage%3D596%26epage%3D608%26doi%3D10.1016%2Fj.molcel.2007.06.026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Shinkai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tachibana, M.</span><span> </span><span class="NLM_article-title">H3K9 methyltransferase G9a and the related molecule GLP</span> <span class="citation_source-journal">Genes Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">781</span><span class="NLM_x">–</span> <span class="NLM_lpage">788</span><span class="refDoi"> DOI: 10.1101/gad.2027411</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1101%2Fgad.2027411" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=21498567" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BC3MXltlKntrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2011&pages=781-788&author=Y.+Shinkaiauthor=M.+Tachibana&title=H3K9+methyltransferase+G9a+and+the+related+molecule+GLP&doi=10.1101%2Fgad.2027411"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">H3K9 methyltransferase G9a and the related molecule GLP</span></div><div class="casAuthors">Shinkai, Yoichi; Tachibana, Makoto</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">781-788</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">A review.  The discovery of Suv39h1, the first SET domain-contg. histone lysine methyltransferase (HKMT), was reported in 2000.  Since then, research on histone methylation has progressed rapidly.  Among the identified HKMTs in mammals, G9a and GLP are the primary enzymes for mono- and dimethylation at Lys 9 of histone H3 (H3K9me1 and H3K9me2), and exist predominantly as a G9a-GLP heteromeric complex that appears to be a functional H3K9 methyltransferase in vivo.  Recently, many important studies have reported that G9a and GLP play crit. roles in various biol. processes.  The physiol. relevance of G9a/GLP-mediated epigenetic gene regulation is discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCYw7kIeMuFbVg90H21EOLACvtfcHk0lhcR8IhJ0uOpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXltlKntrg%253D&md5=f688e93e1b6b7966d04c473536956ef8</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1101%2Fgad.2027411&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.2027411%26sid%3Dliteratum%253Aachs%26aulast%3DShinkai%26aufirst%3DY.%26aulast%3DTachibana%26aufirst%3DM.%26atitle%3DH3K9%2520methyltransferase%2520G9a%2520and%2520the%2520related%2520molecule%2520GLP%26jtitle%3DGenes%2520Dev.%26date%3D2011%26volume%3D25%26spage%3D781%26epage%3D788%26doi%3D10.1101%2Fgad.2027411" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Tachibana, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nozaki, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takeda, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shinkai, Y.</span><span> </span><span class="NLM_article-title">Functional dynamics of H3K9 methylation during meiotic prophase progression</span> <span class="citation_source-journal">EMBO J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">3346</span><span class="NLM_x">–</span> <span class="NLM_lpage">3359</span><span class="refDoi"> DOI: 10.1038/sj.emboj.7601767</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1038%2Fsj.emboj.7601767" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2007&pages=3346-3359&author=M.+Tachibanaauthor=M.+Nozakiauthor=N.+Takedaauthor=Y.+Shinkai&title=Functional+dynamics+of+H3K9+methylation+during+meiotic+prophase+progression&doi=10.1038%2Fsj.emboj.7601767"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fsj.emboj.7601767&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.emboj.7601767%26sid%3Dliteratum%253Aachs%26aulast%3DTachibana%26aufirst%3DM.%26aulast%3DNozaki%26aufirst%3DM.%26aulast%3DTakeda%26aufirst%3DN.%26aulast%3DShinkai%26aufirst%3DY.%26atitle%3DFunctional%2520dynamics%2520of%2520H3K9%2520methylation%2520during%2520meiotic%2520prophase%2520progression%26jtitle%3DEMBO%2520J.%26date%3D2007%26volume%3D26%26spage%3D3346%26epage%3D3359%26doi%3D10.1038%2Fsj.emboj.7601767" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Kondo, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurokawa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Masuko, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanaka, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kato, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mizuno, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yokoe, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sugauchi, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirashima, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orito, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Osada, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ueda, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Issa, J. P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sekido, Y.</span><span> </span><span class="NLM_article-title">Alterations of DNA methylation and histone modifications contribute to gene silencing in hepatocellular carcinomas</span> <span class="citation_source-journal">Hepatol. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">37</span><span class="NLM_x">, </span> <span class="NLM_fpage">974</span><span class="NLM_x">–</span> <span class="NLM_lpage">983</span><span class="refDoi"> DOI: 10.1111/j.1872-034X.2007.00141.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1111%2Fj.1872-034X.2007.00141.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=17584191" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlCns7fK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2007&pages=974-983&author=Y.+Kondoauthor=L.+Shenauthor=S.+Suzukiauthor=T.+Kurokawaauthor=K.+Masukoauthor=Y.+Tanakaauthor=H.+Katoauthor=Y.+Mizunoauthor=M.+Yokoeauthor=F.+Sugauchiauthor=N.+Hirashimaauthor=E.+Oritoauthor=H.+Osadaauthor=R.+Uedaauthor=Y.+Guoauthor=X.+L.+Chenauthor=J.+P.+J.+Issaauthor=Y.+Sekido&title=Alterations+of+DNA+methylation+and+histone+modifications+contribute+to+gene+silencing+in+hepatocellular+carcinomas&doi=10.1111%2Fj.1872-034X.2007.00141.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Alterations of DNA methylation and histone modifications contribute to gene silencing in hepatocellular carcinomas</span></div><div class="casAuthors">Kondo, Yutaka; Shen, Lanlan; Suzuki, Seiji; Kurokawa, Tsuyoshi; Masuko, Kazuo; Tanaka, Yasuhito; Kato, Hideaki; Mizuno, Yoshiki; Yokoe, Masamichi; Sugauchi, Fuminaka; Hirashima, Noboru; Orito, Etsuro; Osada, Hirotaka; Ueda, Ryuzo; Guo, Yi; Chen, Xinli; Issa, Jean-Pierre J.; Sekido, Yoshitaka</div><div class="citationInfo"><span class="NLM_cas:title">Hepatology Research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">974-983</span>CODEN:
                <span class="NLM_cas:coden">HPRSFM</span>;
        ISSN:<span class="NLM_cas:issn">1386-6346</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Asia Pty Ltd.</span>)
        </div><div class="casAbstract">Aim: The aim of the present study was to examine DNA methylation and histone modification changes in hepatocellular carcinomas (HCC).  Methods: DNA methylation in the P16, RASSF1a, progesterone receptor (PGR) and estrogen receptor α (ERα) promoters was detd. by quant. bisulfite-pyrosequencing technique in HCC patients.  Histone H3-lysine (K) 4, H3-K9 and H3-K27 modifications in all these four genes were examd. by chromatin immunopptn. (ChIP) assay in HCC cell lines.  Expression of two DNA methyltransferases (DNMT1 and DNMT3b) and three histone methyltransferases (SUV39H1, G9a and EZH2) in HCC patients was measured by real-time polymerase chain reaction.  Results: Aberrant DNA methylation was detected in all the HCC.  Patients with DNA methylation in the RASSF1a, PGR and ERα promoters in cancers also had substantial DNA methylation in their non-cancerous liver tissues, whereas DNA methylation in the P16 promoter was cancer specific.  Epigenetic states in HCC cell lines showed that silencing of P16 and RASSF1a depended on DNA methylation and histone H3-K9 methylation.  However, silencing of the PGR and ERα genes was more closely related to H3-K27 methylation rather than DNA methylation.  Consistent with the alteration of histone status, higher expression of G9a and EZH2 was found in HCC than in non-cancerous liver tissues (P < 0.01).  Conclusion: These data suggest that multiple epigenetic silencing mechanisms are inappropriately active in HCC cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwkWfvYpsUX7Vg90H21EOLACvtfcHk0lhcR8IhJ0uOpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlCns7fK&md5=afc7013e4adfd4dd30d0b0e07ce8d9ef</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1111%2Fj.1872-034X.2007.00141.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1872-034X.2007.00141.x%26sid%3Dliteratum%253Aachs%26aulast%3DKondo%26aufirst%3DY.%26aulast%3DShen%26aufirst%3DL.%26aulast%3DSuzuki%26aufirst%3DS.%26aulast%3DKurokawa%26aufirst%3DT.%26aulast%3DMasuko%26aufirst%3DK.%26aulast%3DTanaka%26aufirst%3DY.%26aulast%3DKato%26aufirst%3DH.%26aulast%3DMizuno%26aufirst%3DY.%26aulast%3DYokoe%26aufirst%3DM.%26aulast%3DSugauchi%26aufirst%3DF.%26aulast%3DHirashima%26aufirst%3DN.%26aulast%3DOrito%26aufirst%3DE.%26aulast%3DOsada%26aufirst%3DH.%26aulast%3DUeda%26aufirst%3DR.%26aulast%3DGuo%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DX.%2BL.%26aulast%3DIssa%26aufirst%3DJ.%2BP.%2BJ.%26aulast%3DSekido%26aufirst%3DY.%26atitle%3DAlterations%2520of%2520DNA%2520methylation%2520and%2520histone%2520modifications%2520contribute%2520to%2520gene%2520silencing%2520in%2520hepatocellular%2520carcinomas%26jtitle%3DHepatol.%2520Res.%26date%3D2007%26volume%3D37%26spage%3D974%26epage%3D983%26doi%3D10.1111%2Fj.1872-034X.2007.00141.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Watanabe, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soejima, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yasuda, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawada, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakachi, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoda, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Naoki, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ishizaka, A.</span><span> </span><span class="NLM_article-title">Deregulation of histone lysine methyltransferases contributes to oncogenic transformation of human bronchoepithelial cells</span> <span class="citation_source-journal">Cancer Cell Int.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">15</span><span class="refDoi"> DOI: 10.1186/1475-2867-8-15</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1186%2F1475-2867-8-15" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=18980680" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A280%3ADC%252BD1cjltFSntg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2008&pages=15&author=H.+Watanabeauthor=K.+Soejimaauthor=H.+Yasudaauthor=I.+Kawadaauthor=I.+Nakachiauthor=S.+Yodaauthor=K.+Naokiauthor=A.+Ishizaka&title=Deregulation+of+histone+lysine+methyltransferases+contributes+to+oncogenic+transformation+of+human+bronchoepithelial+cells&doi=10.1186%2F1475-2867-8-15"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Deregulation of histone lysine methyltransferases contributes to oncogenic transformation of human bronchoepithelial cells</span></div><div class="casAuthors">Watanabe Hideo; Soejima Kenzo; Yasuda Hiroyuki; Kawada Ichiro; Nakachi Ichiro; Yoda Satoshi; Naoki Katsuhiko; Ishizaka Akitoshi</div><div class="citationInfo"><span class="NLM_cas:title">Cancer cell international</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">15</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Alterations in the processing of the genetic information in carcinogenesis result from stable genetic mutations or epigenetic modifications.  It is becoming clear that nucleosomal histones are central to proper gene expression and that aberrant DNA methylation of genes and histone methylation plays important roles in tumor progression.  To date, several histone lysine methyltransferases (HKMTs) have been identified and histone lysine methylation is now considered to be a critical regulator of transcription.  However, still relatively little is known about the role of HKMTs in tumorigenesis.  RESULTS:  We observed differential HKMT expression in a lung cancer model in which normal human bronchial epithelial (NHBE) cells expressing telomerase, SV40 large T antigen, and Ras were immortal, formed colonies in soft agar, and expressed specific HKMTs for H3 lysine 9 and 27 residues but not for H3 lysine 4 residue.  Modifications in the H3 tails affect the binding of proteins to the histone tails and regulate protein function and the position of lysine methylation marks a gene to be either activated or repressed.  In the present study, suppression by siRNA of HKMTs (EZH2, G9A, SETDB1 and SUV39H1) that are over-expressed in immortalized and transformed cells lead to reduced cell proliferation and much less anchorage-independent colony growth.  We also found that the suppression of H3-K9, G9A and SUV39H1 induced apoptosis and the suppression of H3-K27, EZH2 caused G1 arrest.  CONCLUSION:  Our results indicate the potential of these HKMTs in addition to the other targets for epigenetics such as DNMTs and HDACs to be interesting therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ8X-3k_fB763y-c-E1v-fmfW6udTcc2ebtkc0psPHeyrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1cjltFSntg%253D%253D&md5=ac77cc1b0cca2a4d8bf111fb6463ee7e</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1186%2F1475-2867-8-15&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1475-2867-8-15%26sid%3Dliteratum%253Aachs%26aulast%3DWatanabe%26aufirst%3DH.%26aulast%3DSoejima%26aufirst%3DK.%26aulast%3DYasuda%26aufirst%3DH.%26aulast%3DKawada%26aufirst%3DI.%26aulast%3DNakachi%26aufirst%3DI.%26aulast%3DYoda%26aufirst%3DS.%26aulast%3DNaoki%26aufirst%3DK.%26aulast%3DIshizaka%26aufirst%3DA.%26atitle%3DDeregulation%2520of%2520histone%2520lysine%2520methyltransferases%2520contributes%2520to%2520oncogenic%2520transformation%2520of%2520human%2520bronchoepithelial%2520cells%26jtitle%3DCancer%2520Cell%2520Int.%26date%3D2008%26volume%3D8%26spage%3D15%26doi%3D10.1186%2F1475-2867-8-15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Kondo, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ahmed, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boumber, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sekido, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haddad, B. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Issa, J. P. J.</span><span> </span><span class="NLM_article-title">Downregulation of histone H3 lysine 9 methyltransferase G9a induces centrosome disruption and chromosome instability in cancer cells</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">e2037</span><span class="refDoi"> DOI: 10.1371/journal.pone.0002037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1371%2Fjournal.pone.0002037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=18446223" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A280%3ADC%252BD1c3otlyhtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2008&pages=e2037&author=Y.+Kondoauthor=L.+Shenauthor=S.+Ahmedauthor=Y.+Boumberauthor=Y.+Sekidoauthor=B.+R.+Haddadauthor=J.+P.+J.+Issa&title=Downregulation+of+histone+H3+lysine+9+methyltransferase+G9a+induces+centrosome+disruption+and+chromosome+instability+in+cancer+cells&doi=10.1371%2Fjournal.pone.0002037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Downregulation of histone H3 lysine 9 methyltransferase G9a induces centrosome disruption and chromosome instability in cancer cells</span></div><div class="casAuthors">Kondo Yutaka; Shen Lanlan; Ahmed Saira; Boumber Yanis; Sekido Yoshitaka; Haddad Bassem R; Issa Jean-Pierre J</div><div class="citationInfo"><span class="NLM_cas:title">PloS one</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">e2037</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Modifications of the histone amino-terminal tails affect access of regulatory factors and complexes to chromatin and thereby influence biological processes.  Cancer cells are characterized by prominent epigenetic dysregulation, including histone modifications.  However, the functional roles of the histone methyltransferases (HMT) in cancer remain unclear.  METHODOLOGY/PRINCIPAL FINDINGS:  We studied RNAi-based inhibition (knockdown, KD) of 2 different H3K9 HMTs, SUV39H1 and G9a.  Knockdown of the 2 HMTs in PC3 cancer cell line markedly inhibited cell growth and caused profound morphological changes with loss of telomerase activity and shortened telomeres.  SUV39H1 KD cells showed substantial increase in G2/M fraction.  G9a KD cells showed increased DNA content (1.7-fold in 2 independent clones) compared with FACS analyses to control.  Karyotype analyses showed that this was due to an increased number of chromosomes (from 61 to 102) in G9a KD cells compared to parental PC3.  Intriguingly, we found abnormal centrosome morphology and number in about 25% of the G9a KD cells, while centrosomes were morphologically normal in control cells.  Microarray analyses after KD of SUV39H1 or G9a showed very few genes up-regulated among the 39,000 genes.  The silenced tumor-suppressor genes p16 and RASSF1A were not activated in KD cells.  CONCLUSIONS/SIGNIFICANCE:  These data suggest that the 2 HMTs, SUV39H1 and G9a are required to perpetuate the malignant phenotype.  Furthermore, G9a plays a critical role in regulating centrosome duplication presumably through chromatin structure rather than through affecting gene expression in cancer cells.  Targeting these histone methyltransferases may be of therapeutic benefit in cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQhnx1F8nyNwDEwO064Z95rfW6udTcc2eYvUNQwx_Tj1bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1c3otlyhtw%253D%253D&md5=22936eeeffa4dc29e7254dfc546ebcfd</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0002037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0002037%26sid%3Dliteratum%253Aachs%26aulast%3DKondo%26aufirst%3DY.%26aulast%3DShen%26aufirst%3DL.%26aulast%3DAhmed%26aufirst%3DS.%26aulast%3DBoumber%26aufirst%3DY.%26aulast%3DSekido%26aufirst%3DY.%26aulast%3DHaddad%26aufirst%3DB.%2BR.%26aulast%3DIssa%26aufirst%3DJ.%2BP.%2BJ.%26atitle%3DDownregulation%2520of%2520histone%2520H3%2520lysine%25209%2520methyltransferase%2520G9a%2520induces%2520centrosome%2520disruption%2520and%2520chromosome%2520instability%2520in%2520cancer%2520cells%26jtitle%3DPLoS%2520One%26date%3D2008%26volume%3D3%26spage%3De2037%26doi%3D10.1371%2Fjournal.pone.0002037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Thomas, L. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyashita, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cobb, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pierce, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tachibana, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hobeika, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reth, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shinkai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oltz, E. M.</span><span> </span><span class="NLM_article-title">Functional analysis of histone methyltransferase G9a in B and T lymphocytes</span> <span class="citation_source-journal">J. Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">181</span><span class="NLM_x">, </span> <span class="NLM_fpage">485</span><span class="NLM_x">–</span> <span class="NLM_lpage">493</span><span class="refDoi"> DOI: 10.4049/jimmunol.181.1.485</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.4049%2Fjimmunol.181.1.485" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=181&publication_year=2008&pages=485-493&author=L.+R.+Thomasauthor=H.+Miyashitaauthor=R.+M.+Cobbauthor=S.+Pierceauthor=M.+Tachibanaauthor=E.+Hobeikaauthor=M.+Rethauthor=Y.+Shinkaiauthor=E.+M.+Oltz&title=Functional+analysis+of+histone+methyltransferase+G9a+in+B+and+T+lymphocytes&doi=10.4049%2Fjimmunol.181.1.485"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.4049%2Fjimmunol.181.1.485&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4049%252Fjimmunol.181.1.485%26sid%3Dliteratum%253Aachs%26aulast%3DThomas%26aufirst%3DL.%2BR.%26aulast%3DMiyashita%26aufirst%3DH.%26aulast%3DCobb%26aufirst%3DR.%2BM.%26aulast%3DPierce%26aufirst%3DS.%26aulast%3DTachibana%26aufirst%3DM.%26aulast%3DHobeika%26aufirst%3DE.%26aulast%3DReth%26aufirst%3DM.%26aulast%3DShinkai%26aufirst%3DY.%26aulast%3DOltz%26aufirst%3DE.%2BM.%26atitle%3DFunctional%2520analysis%2520of%2520histone%2520methyltransferase%2520G9a%2520in%2520B%2520and%2520T%2520lymphocytes%26jtitle%3DJ.%2520Immunol.%26date%3D2008%26volume%3D181%26spage%3D485%26epage%3D493%26doi%3D10.4049%2Fjimmunol.181.1.485" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Lehnertz, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Northrop, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Antignano, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burrows, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hadidi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mullaly, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rossi, F. M. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zaph, C.</span><span> </span><span class="NLM_article-title">Activating and inhibitory functions for the histone lysine methyltransferase G9a in T helper cell differentiation and function</span> <span class="citation_source-journal">J. Exp. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">207</span><span class="NLM_x">, </span> <span class="NLM_fpage">915</span><span class="NLM_x">–</span> <span class="NLM_lpage">922</span><span class="refDoi"> DOI: 10.1084/jem.20100363</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1084%2Fjem.20100363" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=207&publication_year=2010&pages=915-922&author=B.+Lehnertzauthor=J.+P.+Northropauthor=F.+Antignanoauthor=K.+Burrowsauthor=S.+Hadidiauthor=S.+C.+Mullalyauthor=F.+M.+V.+Rossiauthor=C.+Zaph&title=Activating+and+inhibitory+functions+for+the+histone+lysine+methyltransferase+G9a+in+T+helper+cell+differentiation+and+function&doi=10.1084%2Fjem.20100363"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1084%2Fjem.20100363&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1084%252Fjem.20100363%26sid%3Dliteratum%253Aachs%26aulast%3DLehnertz%26aufirst%3DB.%26aulast%3DNorthrop%26aufirst%3DJ.%2BP.%26aulast%3DAntignano%26aufirst%3DF.%26aulast%3DBurrows%26aufirst%3DK.%26aulast%3DHadidi%26aufirst%3DS.%26aulast%3DMullaly%26aufirst%3DS.%2BC.%26aulast%3DRossi%26aufirst%3DF.%2BM.%2BV.%26aulast%3DZaph%26aufirst%3DC.%26atitle%3DActivating%2520and%2520inhibitory%2520functions%2520for%2520the%2520histone%2520lysine%2520methyltransferase%2520G9a%2520in%2520T%2520helper%2520cell%2520differentiation%2520and%2520function%26jtitle%3DJ.%2520Exp.%2520Med.%26date%3D2010%26volume%3D207%26spage%3D915%26epage%3D922%26doi%3D10.1084%2Fjem.20100363" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Maze, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Covington, H. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dietz, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LaPlant, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Renthal, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Russo, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mechanic, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mouzon, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neve, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haggarty, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ren, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sampath, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurd, Y. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greengard, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tarakhovsky, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schaefer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nestler, E. J.</span><span> </span><span class="NLM_article-title">Essential role of the histone methyltransferase G9a in cocaine-induced plasticity</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">327</span><span class="NLM_x">, </span> <span class="NLM_fpage">213</span><span class="NLM_x">–</span> <span class="NLM_lpage">216</span><span class="refDoi"> DOI: 10.1126/science.1179438</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1126%2Fscience.1179438" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=20056891" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1egtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=327&publication_year=2010&pages=213-216&author=I.+Mazeauthor=H.+E.+Covingtonauthor=D.+M.+Dietzauthor=Q.+LaPlantauthor=W.+Renthalauthor=S.+J.+Russoauthor=M.+Mechanicauthor=E.+Mouzonauthor=R.+L.+Neveauthor=S.+J.+Haggartyauthor=Y.+H.+Renauthor=S.+C.+Sampathauthor=Y.+L.+Hurdauthor=P.+Greengardauthor=A.+Tarakhovskyauthor=A.+Schaeferauthor=E.+J.+Nestler&title=Essential+role+of+the+histone+methyltransferase+G9a+in+cocaine-induced+plasticity&doi=10.1126%2Fscience.1179438"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Essential Role of the Histone Methyltransferase G9a in Cocaine-Induced Plasticity</span></div><div class="casAuthors">Maze, Ian; Covington, Herbert E., III; Dietz, David M.; LaPlant, Quincey; Renthal, William; Russo, Scott J.; Mechanic, Max; Mouzon, Ezekiell; Neve, Rachael L.; Haggarty, Stephen J.; Ren, Yanhua; Sampath, Srihari C.; Hurd, Yasmin L.; Greengard, Paul; Tarakhovsky, Alexander; Schaefer, Anne; Nestler, Eric J.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">327</span>
        (<span class="NLM_cas:issue">5962</span>),
    <span class="NLM_cas:pages">213-216</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Cocaine-induced alterations in gene expression cause changes in neuronal morphol. and behavior that may underlie cocaine addiction.  In mice, we identified an essential role for histone 3 lysine 9 (H3K9) dimethylation and the lysine dimethyltransferase G9a in cocaine-induced structural and behavioral plasticity.  Repeated cocaine administration reduced global levels of H3K9 dimethylation in the nucleus accumbens.  This redn. in histone methylation was mediated through the repression of G9a in this brain region, which was regulated by the cocaine-induced transcription factor ΔFosB.  Using conditional mutagenesis and viral-mediated gene transfer, we found that G9a down-regulation increased the dendritic spine plasticity of nucleus accumbens neurons and enhanced the preference for cocaine, thereby establishing a crucial role for histone methylation in the long-term actions of cocaine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8SPD826wEGrVg90H21EOLACvtfcHk0lgt6pi4Igtd4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1egtA%253D%253D&md5=a8347500f81862078787e50fff53b6b9</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1126%2Fscience.1179438&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1179438%26sid%3Dliteratum%253Aachs%26aulast%3DMaze%26aufirst%3DI.%26aulast%3DCovington%26aufirst%3DH.%2BE.%26aulast%3DDietz%26aufirst%3DD.%2BM.%26aulast%3DLaPlant%26aufirst%3DQ.%26aulast%3DRenthal%26aufirst%3DW.%26aulast%3DRusso%26aufirst%3DS.%2BJ.%26aulast%3DMechanic%26aufirst%3DM.%26aulast%3DMouzon%26aufirst%3DE.%26aulast%3DNeve%26aufirst%3DR.%2BL.%26aulast%3DHaggarty%26aufirst%3DS.%2BJ.%26aulast%3DRen%26aufirst%3DY.%2BH.%26aulast%3DSampath%26aufirst%3DS.%2BC.%26aulast%3DHurd%26aufirst%3DY.%2BL.%26aulast%3DGreengard%26aufirst%3DP.%26aulast%3DTarakhovsky%26aufirst%3DA.%26aulast%3DSchaefer%26aufirst%3DA.%26aulast%3DNestler%26aufirst%3DE.%2BJ.%26atitle%3DEssential%2520role%2520of%2520the%2520histone%2520methyltransferase%2520G9a%2520in%2520cocaine-induced%2520plasticity%26jtitle%3DScience%26date%3D2010%26volume%3D327%26spage%3D213%26epage%3D216%26doi%3D10.1126%2Fscience.1179438" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Schaefer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sampath, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Intrator, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Min, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gertler, T. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Surmeier, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tarakhovsky, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greengard, P.</span><span> </span><span class="NLM_article-title">Control of cognition and adaptive behavior by the GLP/G9a epigenetic suppressor complex</span> <span class="citation_source-journal">Neuron</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">64</span><span class="NLM_x">, </span> <span class="NLM_fpage">678</span><span class="NLM_x">–</span> <span class="NLM_lpage">691</span><span class="refDoi"> DOI: 10.1016/j.neuron.2009.11.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1016%2Fj.neuron.2009.11.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=20005824" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BC3cXks1Cls7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2009&pages=678-691&author=A.+Schaeferauthor=S.+C.+Sampathauthor=A.+Intratorauthor=A.+Minauthor=T.+S.+Gertlerauthor=D.+J.+Surmeierauthor=A.+Tarakhovskyauthor=P.+Greengard&title=Control+of+cognition+and+adaptive+behavior+by+the+GLP%2FG9a+epigenetic+suppressor+complex&doi=10.1016%2Fj.neuron.2009.11.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Control of cognition and adaptive behavior by the GLP/G9a epigenetic suppressor complex</span></div><div class="casAuthors">Schaefer, Anne; Sampath, Srihari C.; Intrator, Adam; Min, Alice; Gertler, Tracy S.; Surmeier, D. James; Tarakhovsky, Alexander; Greengard, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Neuron</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">678-691</span>CODEN:
                <span class="NLM_cas:coden">NERNET</span>;
        ISSN:<span class="NLM_cas:issn">0896-6273</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The genetic basis of cognition and behavioral adaptation to the environment remains poorly understood.  Here we demonstrate that the histone methyltransferase complex GLP/G9a controls cognition and adaptive responses in a region-specific fashion in the adult brain.  Using conditional mutagenesis in mice, we show that postnatal, neuron-specific deficiency of GLP/G9a leads to derepression of numerous nonneuronal and neuron progenitor genes in adult neurons.  This transcriptional alteration is assocd. with complex behavioral abnormalities, including defects in learning, motivation, and environmental adaptation.  The behavioral changes triggered by GLP/G9a deficiency are similar to key symptoms of the human 9q34 mental retardation syndrome that is assocd. with structural alterations of the GLP/EHMT1 gene.  The likely causal role of GLP/G9a in mental retardation in mice and humans suggests a key role for the GLP/G9a-controlled histone H3K9 dimethylation in regulation of brain function through maintenance of the transcriptional homeostasis in adult neurons.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr22UcjjqkKdLVg90H21EOLACvtfcHk0lg3wv5WVfW_YQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXks1Cls7c%253D&md5=ece4e94ad097a8d0e67a5e8d27970e64</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.neuron.2009.11.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuron.2009.11.019%26sid%3Dliteratum%253Aachs%26aulast%3DSchaefer%26aufirst%3DA.%26aulast%3DSampath%26aufirst%3DS.%2BC.%26aulast%3DIntrator%26aufirst%3DA.%26aulast%3DMin%26aufirst%3DA.%26aulast%3DGertler%26aufirst%3DT.%2BS.%26aulast%3DSurmeier%26aufirst%3DD.%2BJ.%26aulast%3DTarakhovsky%26aufirst%3DA.%26aulast%3DGreengard%26aufirst%3DP.%26atitle%3DControl%2520of%2520cognition%2520and%2520adaptive%2520behavior%2520by%2520the%2520GLP%252FG9a%2520epigenetic%2520suppressor%2520complex%26jtitle%3DNeuron%26date%3D2009%26volume%3D64%26spage%3D678%26epage%3D691%26doi%3D10.1016%2Fj.neuron.2009.11.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Leung, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, K. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maksakova, I. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goyal, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Appanah, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tachibana, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shinkai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lehnertz, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mager, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rossi, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lorincz, M. C.</span><span> </span><span class="NLM_article-title">Lysine methyltransferase G9a is required for de novo DNA methylation and the establishment, but not the maintenance, of proviral silencing</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">108</span><span class="NLM_x">, </span> <span class="NLM_fpage">5718</span><span class="NLM_x">–</span> <span class="NLM_lpage">5723</span><span class="refDoi"> DOI: 10.1073/pnas.1014660108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1073%2Fpnas.1014660108" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=5718-5723&author=D.+C.+Leungauthor=K.+B.+Dongauthor=I.+A.+Maksakovaauthor=P.+Goyalauthor=R.+Appanahauthor=S.+Leeauthor=M.+Tachibanaauthor=Y.+Shinkaiauthor=B.+Lehnertzauthor=D.+L.+Magerauthor=F.+Rossiauthor=M.+C.+Lorincz&title=Lysine+methyltransferase+G9a+is+required+for+de+novo+DNA+methylation+and+the+establishment%2C+but+not+the+maintenance%2C+of+proviral+silencing&doi=10.1073%2Fpnas.1014660108"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1014660108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1014660108%26sid%3Dliteratum%253Aachs%26aulast%3DLeung%26aufirst%3DD.%2BC.%26aulast%3DDong%26aufirst%3DK.%2BB.%26aulast%3DMaksakova%26aufirst%3DI.%2BA.%26aulast%3DGoyal%26aufirst%3DP.%26aulast%3DAppanah%26aufirst%3DR.%26aulast%3DLee%26aufirst%3DS.%26aulast%3DTachibana%26aufirst%3DM.%26aulast%3DShinkai%26aufirst%3DY.%26aulast%3DLehnertz%26aufirst%3DB.%26aulast%3DMager%26aufirst%3DD.%2BL.%26aulast%3DRossi%26aufirst%3DF.%26aulast%3DLorincz%26aufirst%3DM.%2BC.%26atitle%3DLysine%2520methyltransferase%2520G9a%2520is%2520required%2520for%2520de%2520novo%2520DNA%2520methylation%2520and%2520the%2520establishment%252C%2520but%2520not%2520the%2520maintenance%252C%2520of%2520proviral%2520silencing%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26spage%3D5718%26epage%3D5723%26doi%3D10.1073%2Fpnas.1014660108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Kleefstra, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smidt, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banning, M. J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oudakker, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Esch, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Brouwer, A. P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nillesen, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sistermans, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamel, B. C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Bruijn, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fryns, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yntema, H. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brunner, H. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Vries, B. B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Bokhoven, H.</span><span> </span><span class="NLM_article-title">Disruption of the gene euchromatin histone methyl transferase1 (Eu-HMTase1) is associated with the 9q34 subtelomeric deletion syndrome</span> <span class="citation_source-journal">J. Med. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">299</span><span class="NLM_x">–</span> <span class="NLM_lpage">306</span><span class="refDoi"> DOI: 10.1136/jmg.2004.028464</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1136%2Fjmg.2004.028464" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=15805155" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BD2MXktFGgtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2005&pages=299-306&author=T.+Kleefstraauthor=M.+Smidtauthor=M.+J.+G.+Banningauthor=A.+R.+Oudakkerauthor=H.+Van+Eschauthor=A.+P.+M.+de+Brouwerauthor=W.+Nillesenauthor=E.+A.+Sistermansauthor=B.+C.+J.+Hamelauthor=D.+de+Bruijnauthor=J.+P.+Frynsauthor=H.+G.+Yntemaauthor=H.+G.+Brunnerauthor=B.+B.+A.+de+Vriesauthor=H.+van+Bokhoven&title=Disruption+of+the+gene+euchromatin+histone+methyl+transferase1+%28Eu-HMTase1%29+is+associated+with+the+9q34+subtelomeric+deletion+syndrome&doi=10.1136%2Fjmg.2004.028464"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Disruption of the gene euchromatin histone methyl transferase1 (Eu-HMTase1) is associated with the 9q34 subtelomeric deletion syndrome</span></div><div class="casAuthors">Kleefstra, T.; Smidt, M.; Banning, M. J. G.; Oudakker, A. R.; van Esch, H.; de Brouwer, A. P. M.; Nillesen, W.; Sistermans, E. A.; Hamel, B. C. J.; de Bruijn, D.; Fryns, J.-P.; Yntema, H. G.; Brunner, H. G.; de Vries, B. B. A.; van Bokhoven, H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medical Genetics</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">299-306</span>CODEN:
                <span class="NLM_cas:coden">JMDGAE</span>;
        ISSN:<span class="NLM_cas:issn">0022-2593</span>.
    
            (<span class="NLM_cas:orgname">BMJ Publishing Group</span>)
        </div><div class="casAbstract">Background: A new syndrome has been recognized following thorough anal. of patients with a terminal submicroscopic subtelomeric deletion of chromosome 9q.  These have in common severe mental retardation, hypotonia, brachycephaly, flat face with hypertelorism, synophrys, anteverted nares, thickened lower lip, carp mouth with macroglossia, and canotruncal heart defects.  The min. crit. region responsible for this 9q subtelomeric deletion syndrome (9q-) is approx. 1.2 Mb and encompasses at least 14 genes.  Objective: To characterize the breakpoints of a de novo balanced translocation t(X;9)(p11.23;q34.3) in a mentally retarded female patient with clin. features similar to the 9q- syndrome.  Results: Sequence anal. of the break points showed that the translocation was fully balanced and only one gene on chromosome 9 was disrupted-Euchromatin Histone Me Transferase1 (Eu-HMTase1)-encoding a histone H3 lysine 9 methyltransferase (H3-K9 HMTase).  This indicates that haploinsufficiency of Eu-HMTase1 is responsible for the 9q submicroscopic subtelomeric deletion syndrome.  This observation was further supported by the spatio-temporal expression of the gene.  Using tissue in situ hybridization studies in mouse embryos and adult brain, Eu-HMTase1 was shown to be expressed in the developing nervous system and in specific peripheral tissues.  While expression is selectively downregulated in adult brain, substantial expression is retained in the olfactory bulb, anterior/ventral lateral ventricular wall, and hippocampus and weakly in the piriform cortex.  Conclusions: The expression pattern of this gene suggests a role in the CNS development and function, which is in line with the severe mental retardation and behavior problems in patients who lack one copy of the gene.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnHiNhjDYOqrVg90H21EOLACvtfcHk0lg3wv5WVfW_YQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXktFGgtrY%253D&md5=518802ab24e9b6a4ffed3e0f20b374c6</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1136%2Fjmg.2004.028464&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Fjmg.2004.028464%26sid%3Dliteratum%253Aachs%26aulast%3DKleefstra%26aufirst%3DT.%26aulast%3DSmidt%26aufirst%3DM.%26aulast%3DBanning%26aufirst%3DM.%2BJ.%2BG.%26aulast%3DOudakker%26aufirst%3DA.%2BR.%26aulast%3DVan%2BEsch%26aufirst%3DH.%26aulast%3Dde%2BBrouwer%26aufirst%3DA.%2BP.%2BM.%26aulast%3DNillesen%26aufirst%3DW.%26aulast%3DSistermans%26aufirst%3DE.%2BA.%26aulast%3DHamel%26aufirst%3DB.%2BC.%2BJ.%26aulast%3Dde%2BBruijn%26aufirst%3DD.%26aulast%3DFryns%26aufirst%3DJ.%2BP.%26aulast%3DYntema%26aufirst%3DH.%2BG.%26aulast%3DBrunner%26aufirst%3DH.%2BG.%26aulast%3Dde%2BVries%26aufirst%3DB.%2BB.%2BA.%26aulast%3Dvan%2BBokhoven%26aufirst%3DH.%26atitle%3DDisruption%2520of%2520the%2520gene%2520euchromatin%2520histone%2520methyl%2520transferase1%2520%2528Eu-HMTase1%2529%2520is%2520associated%2520with%2520the%25209q34%2520subtelomeric%2520deletion%2520syndrome%26jtitle%3DJ.%2520Med.%2520Genet.%26date%3D2005%26volume%3D42%26spage%3D299%26epage%3D306%26doi%3D10.1136%2Fjmg.2004.028464" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Antignano, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zaph, C.</span><span> </span><span class="NLM_article-title">Regulation of CD4 T-cell differentiation and inflammation by repressive histone methylation</span> <span class="citation_source-journal">Immunol. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">93</span><span class="NLM_x">, </span> <span class="NLM_fpage">245</span><span class="NLM_x">–</span> <span class="NLM_lpage">252</span><span class="refDoi"> DOI: 10.1038/icb.2014.115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1038%2Ficb.2014.115" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=25582341" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFGks70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=2015&pages=245-252&author=F.+Antignanoauthor=C.+Zaph&title=Regulation+of+CD4+T-cell+differentiation+and+inflammation+by+repressive+histone+methylation&doi=10.1038%2Ficb.2014.115"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of CD4 T-cell differentiation and inflammation by repressive histone methylation</span></div><div class="casAuthors">Antignano, Frann; Zaph, Colby</div><div class="citationInfo"><span class="NLM_cas:title">Immunology & Cell Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">245-252</span>CODEN:
                <span class="NLM_cas:coden">ICBIEZ</span>;
        ISSN:<span class="NLM_cas:issn">0818-9641</span>.
    
            (<span class="NLM_cas:orgname">NPG Nature Asia-Pacific</span>)
        </div><div class="casAbstract">Repressive epigenetic modifications such as dimethylation and trimethylation histone H3 at lysine 9 (H3K9me2 and H3K9me3) and H3K27me3 have been shown to be crit. for embryonic stem (ES) cell differentiation by silencing cell lineage-promiscuous genes.  CD4+ T helper (TH) cell differentiation is a powerful model to study the mol. mechanisms assocd. with cellular lineage choice in adult cells.  Naive TH cells have the capacity to differentiate into one of the several phenotypically and functionally distinct and stable lineages.  Although some repressive epigenetic mechanisms have a crit. role in TH cell differentiation in a similar manner to that in ES cells, it is clear that there are disparate functions for certain modifications between ES cells and TH cells.  Here we review the role of repressive histone modifications in the differentiation and function of TH cells in health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvWkSixxmeHLVg90H21EOLACvtfcHk0ljcCA57Uf8k6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFGks70%253D&md5=db9f3617c548d1dabccfc7ceaeb687de</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1038%2Ficb.2014.115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Ficb.2014.115%26sid%3Dliteratum%253Aachs%26aulast%3DAntignano%26aufirst%3DF.%26aulast%3DZaph%26aufirst%3DC.%26atitle%3DRegulation%2520of%2520CD4%2520T-cell%2520differentiation%2520and%2520inflammation%2520by%2520repressive%2520histone%2520methylation%26jtitle%3DImmunol.%2520Cell%2520Biol.%26date%3D2015%26volume%3D93%26spage%3D245%26epage%3D252%26doi%3D10.1038%2Ficb.2014.115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Kramer, J. M.</span><span> </span><span class="NLM_article-title">Regulation of cell differentiation and function by the euchromatin histone methyltranserfases G9a and GLP</span> <span class="citation_source-journal">Biochem. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">94</span><span class="NLM_x">, </span> <span class="NLM_fpage">26</span><span class="NLM_x">–</span> <span class="NLM_lpage">32</span><span class="refDoi"> DOI: 10.1139/bcb-2015-0017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1139%2Fbcb-2015-0017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=26198080" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1SqtLfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=2016&pages=26-32&author=J.+M.+Kramer&title=Regulation+of+cell+differentiation+and+function+by+the+euchromatin+histone+methyltranserfases+G9a+and+GLP&doi=10.1139%2Fbcb-2015-0017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of cell differentiation and function by the euchromatin histone methyltranserfases G9a and GLP1</span></div><div class="casAuthors">Kramer, Jamie M.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry and Cell Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">26-32</span>CODEN:
                <span class="NLM_cas:coden">BCBIEQ</span>;
        ISSN:<span class="NLM_cas:issn">0829-8211</span>.
    
            (<span class="NLM_cas:orgname">Canadian Science Publishing</span>)
        </div><div class="casAbstract">The euchromatin histone methyltransferases (EHMTs) are an evolutionarily conserved protein family that are known for their ability to dimethylate histone 3 at lysine 9 in euchromatic regions of the genome.  In mammals there are two EHMT proteins, G9a, encoded by EHMT2, and GLP, encoded by EHMT1.  EHMTs have diverse roles in the differentiation of different tissues and cell types and are involved in adult-specific processes like memory, drug addiction, and immune response.  This review discusses recent findings from rodent and Drosophila models that are beginning to reveal the broad biol. role and complex mechanistic functioning of EHMT proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqABwbxmYijlLVg90H21EOLACvtfcHk0ljcCA57Uf8k6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1SqtLfI&md5=c49f7a8898c0c8ea824ca387a01f681b</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1139%2Fbcb-2015-0017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1139%252Fbcb-2015-0017%26sid%3Dliteratum%253Aachs%26aulast%3DKramer%26aufirst%3DJ.%2BM.%26atitle%3DRegulation%2520of%2520cell%2520differentiation%2520and%2520function%2520by%2520the%2520euchromatin%2520histone%2520methyltranserfases%2520G9a%2520and%2520GLP%26jtitle%3DBiochem.%2520Cell%2520Biol.%26date%3D2016%26volume%3D94%26spage%3D26%26epage%3D32%26doi%3D10.1139%2Fbcb-2015-0017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Liu, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Z. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meng, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiong, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, Z. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, X. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cai, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, B.</span><span> </span><span class="NLM_article-title">Recognition of H3K9 methylation by GLP is required for efficient establishment of H3K9 methylation, rapid target gene repression, and mouse viability</span> <span class="citation_source-journal">Genes Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">379</span><span class="NLM_x">–</span> <span class="NLM_lpage">393</span><span class="refDoi"> DOI: 10.1101/gad.254425.114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1101%2Fgad.254425.114" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=379-393&author=N.+Liuauthor=Z.+Q.+Zhangauthor=H.+Wuauthor=Y.+H.+Jiangauthor=L.+J.+Mengauthor=J.+Xiongauthor=Z.+D.+Zhaoauthor=X.+H.+Zhouauthor=J.+Liauthor=H.+Liauthor=Y.+Zhengauthor=S.+Chenauthor=T.+Caiauthor=S.+R.+Gaoauthor=B.+Zhu&title=Recognition+of+H3K9+methylation+by+GLP+is+required+for+efficient+establishment+of+H3K9+methylation%2C+rapid+target+gene+repression%2C+and+mouse+viability&doi=10.1101%2Fgad.254425.114"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1101%2Fgad.254425.114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.254425.114%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DN.%26aulast%3DZhang%26aufirst%3DZ.%2BQ.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DJiang%26aufirst%3DY.%2BH.%26aulast%3DMeng%26aufirst%3DL.%2BJ.%26aulast%3DXiong%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DZ.%2BD.%26aulast%3DZhou%26aufirst%3DX.%2BH.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DZheng%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DCai%26aufirst%3DT.%26aulast%3DGao%26aufirst%3DS.%2BR.%26aulast%3DZhu%26aufirst%3DB.%26atitle%3DRecognition%2520of%2520H3K9%2520methylation%2520by%2520GLP%2520is%2520required%2520for%2520efficient%2520establishment%2520of%2520H3K9%2520methylation%252C%2520rapid%2520target%2520gene%2520repression%252C%2520and%2520mouse%2520viability%26jtitle%3DGenes%2520Dev.%26date%3D2015%26volume%3D29%26spage%3D379%26epage%3D393%26doi%3D10.1101%2Fgad.254425.114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Ohno, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shinoda, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohyama, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharp, L. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kajimura, S.</span><span> </span><span class="NLM_article-title">EHMT1 controls brown adipose cell fate and thermogenesis through the PRDM16 complex</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">504</span><span class="NLM_x">, </span> <span class="NLM_fpage">163</span><span class="NLM_x">–</span> <span class="NLM_lpage">167</span><span class="refDoi"> DOI: 10.1038/nature12652</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1038%2Fnature12652" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=24196706" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVyisLbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=504&publication_year=2013&pages=163-167&author=H.+Ohnoauthor=K.+Shinodaauthor=K.+Ohyamaauthor=L.+Z.+Sharpauthor=S.+Kajimura&title=EHMT1+controls+brown+adipose+cell+fate+and+thermogenesis+through+the+PRDM16+complex&doi=10.1038%2Fnature12652"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">EHMT1 controls brown adipose cell fate and thermogenesis through the PRDM16 complex</span></div><div class="casAuthors">Ohno, Haruya; Shinoda, Kosaku; Ohyama, Kana; Sharp, Louis Z.; Kajimura, Shingo</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">504</span>
        (<span class="NLM_cas:issue">7478</span>),
    <span class="NLM_cas:pages">163-167</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Brown adipose tissue (BAT) dissipates chem. energy in the form of heat as a defense against hypothermia and obesity.  Current evidence indicates that brown adipocytes arise from Myf5+ dermotomal precursors through the action of PR domain contg. protein 16 (PRDM16) transcriptional complex.  However, the enzymic component of the mol. switch that dets. lineage specification of brown adipocytes remains unknown.  Here, the authors show that euchromatic histone-lysine N-methyltransferase 1 (EHMT1) is an essential BAT-enriched lysine methyltransferase in the PRDM16 transcriptional complex and controls brown adipose cell fate.  Loss of EHMT1 in brown adipocytes causes a severe loss of brown fat characteristics and induces muscle differentiation in vivo through demethylation of histone 3 lysine 9 (H3K9me2 and 3) of the muscle-selective gene promoters.  Conversely, EHMT1 expression pos. regulates the BAT-selective thermogenic program by stabilizing the PRDM16 protein.  Notably, adipose-specific deletion of EHMT1 leads to a marked redn. of BAT-mediated adaptive thermogenesis, obesity and systemic insulin resistance.  These data indicate that EHMT1 is an essential enzymic switch that controls brown adipose cell fate and energy homeostasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZ8WN3L9wpErVg90H21EOLACvtfcHk0ljcCA57Uf8k6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVyisLbO&md5=f0f0a8bf8b229a161ce2e5d8aa1530e2</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1038%2Fnature12652&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12652%26sid%3Dliteratum%253Aachs%26aulast%3DOhno%26aufirst%3DH.%26aulast%3DShinoda%26aufirst%3DK.%26aulast%3DOhyama%26aufirst%3DK.%26aulast%3DSharp%26aufirst%3DL.%2BZ.%26aulast%3DKajimura%26aufirst%3DS.%26atitle%3DEHMT1%2520controls%2520brown%2520adipose%2520cell%2520fate%2520and%2520thermogenesis%2520through%2520the%2520PRDM16%2520complex%26jtitle%3DNature%26date%3D2013%26volume%3D504%26spage%3D163%26epage%3D167%26doi%3D10.1038%2Fnature12652" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Battisti, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pontis, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boyarchuk, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fritsch, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robin, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ait-Si-Ali, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joliot, V.</span><span> </span><span class="NLM_article-title">Unexpected distinct roles of the related histone H3 lysine 9 methyltransferases G9a and G9a-like protein in myoblasts</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">428</span><span class="NLM_x">, </span> <span class="NLM_fpage">2329</span><span class="NLM_x">–</span> <span class="NLM_lpage">2343</span><span class="refDoi"> DOI: 10.1016/j.jmb.2016.03.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1016%2Fj.jmb.2016.03.029" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=428&publication_year=2016&pages=2329-2343&author=V.+Battistiauthor=J.+Pontisauthor=E.+Boyarchukauthor=L.+Fritschauthor=P.+Robinauthor=S.+Ait-Si-Aliauthor=V.+Joliot&title=Unexpected+distinct+roles+of+the+related+histone+H3+lysine+9+methyltransferases+G9a+and+G9a-like+protein+in+myoblasts&doi=10.1016%2Fj.jmb.2016.03.029"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2016.03.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2016.03.029%26sid%3Dliteratum%253Aachs%26aulast%3DBattisti%26aufirst%3DV.%26aulast%3DPontis%26aufirst%3DJ.%26aulast%3DBoyarchuk%26aufirst%3DE.%26aulast%3DFritsch%26aufirst%3DL.%26aulast%3DRobin%26aufirst%3DP.%26aulast%3DAit-Si-Ali%26aufirst%3DS.%26aulast%3DJoliot%26aufirst%3DV.%26atitle%3DUnexpected%2520distinct%2520roles%2520of%2520the%2520related%2520histone%2520H3%2520lysine%25209%2520methyltransferases%2520G9a%2520and%2520G9a-like%2520protein%2520in%2520myoblasts%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2016%26volume%3D428%26spage%3D2329%26epage%3D2343%26doi%3D10.1016%2Fj.jmb.2016.03.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quinn, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kozieradzki, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siarheyeva, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kireev, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jadhav, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herold, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Discovery of a 2,4-diamino-7-aminoalkoxyquinazoline as a potent and selective inhibitor of histone lysine methyltransferase G9a</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">7950</span><span class="NLM_x">–</span> <span class="NLM_lpage">7953</span><span class="refDoi"> DOI: 10.1021/jm901543m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901543m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlOiurzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=7950-7953&author=F.+Liuauthor=X.+Chenauthor=A.+Allali-Hassaniauthor=A.+M.+Quinnauthor=G.+A.+Wasneyauthor=A.+P.+Dongauthor=D.+Barsyteauthor=I.+Kozieradzkiauthor=G.+Senisterraauthor=I.+Chauauthor=A.+Siarheyevaauthor=D.+B.+Kireevauthor=A.+Jadhavauthor=J.+M.+Heroldauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=A.+Simeonovauthor=M.+Vedadiauthor=J.+Jin&title=Discovery+of+a+2%2C4-diamino-7-aminoalkoxyquinazoline+as+a+potent+and+selective+inhibitor+of+histone+lysine+methyltransferase+G9a&doi=10.1021%2Fjm901543m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a 2,4-Diamino-7-aminoalkoxyquinazoline as a Potent and Selective Inhibitor of Histone Lysine Methyltransferase G9a</span></div><div class="casAuthors">Liu, Feng; Chen, Xin; Allali-Hassani, Abdellah; Quinn, Amy M.; Wasney, Gregory A.; Dong, Aiping; Barsyte, Dalia; Kozieradzki, Ivona; Senisterra, Guillermo; Chau, Irene; Siarheyeva, Alena; Kireev, Dmitri B.; Jadhav, Ajit; Herold, J. Martin; Frye, Stephen V.; Arrowsmith, Cheryl H.; Brown, Peter J.; Simeonov, Anton; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">7950-7953</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">SAR exploration of the 2,4-diamino-6,7-dimethoxyquinazoline template led to the discovery of 8 (I) as a potent and selective G9a inhibitor.  A high resoln. X-ray crystal structure of the G9a-8 complex, the first cocrystal structure of G9a with a small mol. inhibitor, was obtained.  The cocrystal structure validated our binding hypothesis and will enable structure-based design of novel inhibitors.  8 Is a useful tool for investigating the biol. of G9a and its roles in chromatin remodeling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjWt5XsGXyj7Vg90H21EOLACvtfcHk0liCn6CVhJzj-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlOiurzN&md5=2d7e34a8c4b5bf1cafd4e01ad2a86de1</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fjm901543m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901543m%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DQuinn%26aufirst%3DA.%2BM.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DDong%26aufirst%3DA.%2BP.%26aulast%3DBarsyte%26aufirst%3DD.%26aulast%3DKozieradzki%26aufirst%3DI.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DKireev%26aufirst%3DD.%2BB.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DHerold%26aufirst%3DJ.%2BM.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520a%25202%252C4-diamino-7-aminoalkoxyquinazoline%2520as%2520a%2520potent%2520and%2520selective%2520inhibitor%2520of%2520histone%2520lysine%2520methyltransferase%2520G9a%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D7950%26epage%3D7953%26doi%3D10.1021%2Fjm901543m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quinn, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siarheyeva, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norris, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kireev, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jadhav, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herold, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzen, W. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J. A.</span><span> </span><span class="NLM_article-title">Protein lysine methyltransferase G9a inhibitors: design, synthesis, and structure activity relationships of 2,4-diamino-7-aminoalkoxy-quinazolines</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">5844</span><span class="NLM_x">–</span> <span class="NLM_lpage">5857</span><span class="refDoi"> DOI: 10.1021/jm100478y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100478y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BC3cXos1ekurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=5844-5857&author=F.+Liuauthor=X.+Chenauthor=A.+Allali-Hassaniauthor=A.+M.+Quinnauthor=T.+J.+Wigleauthor=G.+A.+Wasneyauthor=A.+P.+Dongauthor=G.+Senisterraauthor=I.+Chauauthor=A.+Siarheyevaauthor=J.+L.+Norrisauthor=D.+B.+Kireevauthor=A.+Jadhavauthor=J.+M.+Heroldauthor=W.+P.+Janzenauthor=C.+H.+Arrowsmithauthor=S.+V.+Fryeauthor=P.+J.+Brownauthor=A.+Simeonovauthor=M.+Vedadiauthor=J.+A.+Jin&title=Protein+lysine+methyltransferase+G9a+inhibitors%3A+design%2C+synthesis%2C+and+structure+activity+relationships+of+2%2C4-diamino-7-aminoalkoxy-quinazolines&doi=10.1021%2Fjm100478y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Lysine Methyltransferase G9a Inhibitors: Design, Synthesis, and Structure Activity Relationships of 2,4-Diamino-7-aminoalkoxy-quinazolines</span></div><div class="casAuthors">Liu, Feng; Chen, Xin; Allali-Hassani, Abdellah; Quinn, Amy M.; Wigle, Tim J.; Wasney, Gregory A.; Dong, Aiping; Senisterra, Guillermo; Chau, Irene; Siarheyeva, Alena; Norris, Jacqueline L.; Kireev, Dmitri B.; Jadhav, Ajit; Herold, J. Martin; Janzen, William P.; Arrowsmith, Cheryl H.; Frye, Stephen V.; Brown, Peter J.; Simeonov, Anton; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5844-5857</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein lysine methyltransferase G9a, which catalyzes methylation of lysine 9 of histone H3 (H3K9) and lysine 373 (K373) of p53, is overexpressed in human cancers.  Genetic knockdown of G9a inhibits cancer cell growth; and the dimethylation of p53 K373 results in the inactivation of p53.  Initial SAR exploration of the 2,4-diamino-6,7-dimethoxyquinazoline template represented by I ( R1 = Bn; R2 = Me)(BIX01294), a selective small mol. inhibitor of G9a and GLP (G9a-like protein), led to the discovery of I [R1 = Me; R2 = (CH2)3NMe2](II)(UNC0224) as a potent G9a inhibitor with excellent selectivity.  A high resoln. X-ray crystal structure of the G9a-II complex, the first cocrystal structure of G9a with a small mol. inhibitor, was obtained.  On the basis of the structural insights revealed by this cocrystal structure, optimization of the 7-(dimethylamino)propoxy side chain of II resulted in the discovery of I [R1 = Me; R2 = (CH2)2O(CH2)2NMe2](UNC0321, Morrison Ki = 63 pM), which is the first G9a inhibitor with picomolar potency and the most potent G9a inhibitor to date.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgS0YxdNdNxLVg90H21EOLACvtfcHk0liCn6CVhJzj-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXos1ekurk%253D&md5=8bd703c0a975a33bf9e5c019f1f95ee6</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fjm100478y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100478y%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DQuinn%26aufirst%3DA.%2BM.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DDong%26aufirst%3DA.%2BP.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DKireev%26aufirst%3DD.%2BB.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DHerold%26aufirst%3DJ.%2BM.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%2BA.%26atitle%3DProtein%2520lysine%2520methyltransferase%2520G9a%2520inhibitors%253A%2520design%252C%2520synthesis%252C%2520and%2520structure%2520activity%2520relationships%2520of%25202%252C4-diamino-7-aminoalkoxy-quinazolines%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D5844%26epage%3D5857%26doi%3D10.1021%2Fjm100478y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Chang, Y. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ganesh, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horton, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spannhoff, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shinkai, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Snyder, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, X. D.</span><span> </span><span class="NLM_article-title">Adding a lysine mimic in the design of potent inhibitors of histone lysine methyltransferases</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">400</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">7</span><span class="refDoi"> DOI: 10.1016/j.jmb.2010.04.048</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1016%2Fj.jmb.2010.04.048" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=20434463" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BC3cXntlaju7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=400&publication_year=2010&pages=1-7&author=Y.+Q.+Changauthor=T.+Ganeshauthor=J.+R.+Hortonauthor=A.+Spannhoffauthor=J.+Liuauthor=A.+M.+Sunauthor=X.+Zhangauthor=M.+T.+Bedfordauthor=Y.+C.+Shinkaiauthor=J.+P.+Snyderauthor=X.+D.+Cheng&title=Adding+a+lysine+mimic+in+the+design+of+potent+inhibitors+of+histone+lysine+methyltransferases&doi=10.1016%2Fj.jmb.2010.04.048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Adding a Lysine Mimic in the Design of Potent Inhibitors of Histone Lysine Methyltransferases</span></div><div class="casAuthors">Chang, Yanqi; Ganesh, Thota; Horton, John R.; Spannhoff, Astrid; Liu, Jin; Sun, Aiming; Zhang, Xing; Bedford, Mark T.; Shinkai, Yoichi; Snyder, James P.; Cheng, Xiaodong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">400</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-7</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Dynamic histone lysine methylation involves the activities of modifying enzymes (writers), enzymes removing modifications (erasers), and readers of the histone code.  One common feature of these activities is the recognition of lysines in methylated and unmethylated states, whether they are substrates, reaction products, or binding partners.  The authors applied the concept of adding a lysine mimic to an established inhibitor (BIX-01294) of histone H3 lysine 9 methyltransferases G9a and G9a-like protein by including a 5-aminopentyloxy moiety, which is inserted into the target lysine-binding channel and becomes methylated by G9a-like protein, albeit slowly.  The compd. enhances its potency in vitro and reduces cell toxicity in vivo.  The authors suggest that adding a lysine or methyl-lysine mimic should be considered in the design of small-mol. inhibitors for other methyl-lysine writers, erasers, and readers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsrNWhUZuRrLVg90H21EOLACvtfcHk0lgNmHKou8EBQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXntlaju7c%253D&md5=7ec7ce81db069cd6c1c6637b3ad57007</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2010.04.048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2010.04.048%26sid%3Dliteratum%253Aachs%26aulast%3DChang%26aufirst%3DY.%2BQ.%26aulast%3DGanesh%26aufirst%3DT.%26aulast%3DHorton%26aufirst%3DJ.%2BR.%26aulast%3DSpannhoff%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DSun%26aufirst%3DA.%2BM.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DShinkai%26aufirst%3DY.%2BC.%26aulast%3DSnyder%26aufirst%3DJ.%2BP.%26aulast%3DCheng%26aufirst%3DX.%2BD.%26atitle%3DAdding%2520a%2520lysine%2520mimic%2520in%2520the%2520design%2520of%2520potent%2520inhibitors%2520of%2520histone%2520lysine%2520methyltransferases%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2010%26volume%3D400%26spage%3D1%26epage%3D7%26doi%3D10.1016%2Fj.jmb.2010.04.048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, Y. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herold, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yates, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Optimization of cellular activity of G9a inhibitors 7-aminoalkoxy-quinazolines</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">6139</span><span class="NLM_x">–</span> <span class="NLM_lpage">6150</span><span class="refDoi"> DOI: 10.1021/jm200903z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200903z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpvVWltLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=6139-6150&author=F.+Liuauthor=D.+Barsyte-Lovejoyauthor=A.+Allali-Hassaniauthor=Y.+L.+Heauthor=J.+M.+Heroldauthor=X.+Chenauthor=C.+M.+Yatesauthor=S.+V.+Fryeauthor=P.+J.+Brownauthor=J.+Huangauthor=M.+Vedadiauthor=C.+H.+Arrowsmithauthor=J.+Jin&title=Optimization+of+cellular+activity+of+G9a+inhibitors+7-aminoalkoxy-quinazolines&doi=10.1021%2Fjm200903z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of cellular activity of G9a inhibitors 7-aminoalkoxy-quinazolines</span></div><div class="casAuthors">Liu, Feng; Barsyte-Lovejoy, Dalia; Allali-Hassani, Abdellah; He, Yunlong; Herold, J. Martin; Chen, Xin; Yates, Christopher M.; Frye, Stephen V.; Brown, Peter J.; Huang, Jing; Vedadi, Masoud; Arrowsmith, Cheryl H.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">6139-6150</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein lysine methyltransferase G9a plays key roles in the transcriptional repression of a variety of genes via dimethylation of lysine 9 on histone H3 (H3K9me2) of chromatin as well as dimethylation of nonhistone proteins including tumor suppressor p53.  We previously reported the discovery of UNC0321, the most potent G9a inhibitor to date, via structure-based design and structure-activity relationship (SAR) exploration of the quinazoline scaffold represented by BIX01294.  Despite its very high in vitro potency, compd. UNC0321 lacks sufficient cellular potency.  The design and synthesis of several generations of new analogs aimed at improving cell membrane permeability while maintaining high in vitro potency resulted in the discovery of a no. of novel G9a inhibitors such as UNC0646 I and UNC0631 II with excellent potency in a variety of cell lines and excellent sepn. of functional potency vs. cell toxicity.  The design, synthesis, and cellular SAR of these potent G9a inhibitors are described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrC91yKB7S34rVg90H21EOLACvtfcHk0lgNmHKou8EBQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpvVWltLg%253D&md5=f3f54b91e6b24aa248498f9ff1fdb1ae</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Fjm200903z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200903z%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DHe%26aufirst%3DY.%2BL.%26aulast%3DHerold%26aufirst%3DJ.%2BM.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DYates%26aufirst%3DC.%2BM.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DOptimization%2520of%2520cellular%2520activity%2520of%2520G9a%2520inhibitors%25207-aminoalkoxy-quinazolines%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D6139%26epage%3D6150%26doi%3D10.1021%2Fjm200903z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rival-Gervier, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labrie, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DiMaggio, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siarheyeva, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tempel, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mangano, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simpson, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pattenden, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norris, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kireev, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tripathy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edwards, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzen, W. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petronis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellis, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">A chemical probe selectively inhibits G9a and GLP methyltransferase activity in cells</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">648</span><span class="NLM_x">–</span> <span class="NLM_lpage">648</span><span class="refDoi"> DOI: 10.1038/nchembio0911-648c</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1038%2Fnchembio0911-648c" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1Slu7rE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=648-648&author=M.+Vedadiauthor=D.+Barsyte-Lovejoyauthor=F.+Liuauthor=S.+Rival-Gervierauthor=A.+Allali-Hassaniauthor=V.+Labrieauthor=T.+J.+Wigleauthor=P.+A.+DiMaggioauthor=G.+A.+Wasneyauthor=A.+Siarheyevaauthor=A.+P.+Dongauthor=W.+Tempelauthor=S.+C.+Wangauthor=X.+Chenauthor=I.+Chauauthor=T.+J.+Manganoauthor=X.+P.+Huangauthor=C.+D.+Simpsonauthor=S.+G.+Pattendenauthor=J.+L.+Norrisauthor=D.+B.+Kireevauthor=A.+Tripathyauthor=A.+Edwardsauthor=B.+L.+Rothauthor=W.+P.+Janzenauthor=B.+A.+Garciaauthor=A.+Petronisauthor=J.+Ellisauthor=P.+J.+Brownauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=J.+Jin&title=A+chemical+probe+selectively+inhibits+G9a+and+GLP+methyltransferase+activity+in+cells&doi=10.1038%2Fnchembio0911-648c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">A chemical probe selectively inhibits G9a and GLP methyltransferase activity in cells [Erratum to document cited in CA155:320969]</span></div><div class="casAuthors">Vedadi, Masoud; Barsyte-Lovejoy, Dalia; Liu, Feng; Rival-Gervier, Sylvie; Allali-Hassani, Abdellah; Labrie, Viviane; Wigle, Tim J.; DiMaggio, Peter A.; Wasney, Gregory A.; Siarheyeva, Alena; Dong, Aiping; Tempel, Wolfram; Wang, Sun-Chong; Chen, Sin; Chau, Irene; Mangano, Thomas J.; Huang, Xi-ping; Simpson, Catherine D.; Pattenden, Samantha G.; Norris, Jacqueline L.; Kireev, Dmitri B.; Tripathy, Ashutosh; Edwards, Aled; Roth, Bryan L.; Janzen, William P.; Garcia, Benjamin A.; Petronis, Arturas; Ellis, James; Brown, Peter J.; Frye, Stephen V.; Arrowsmith, Cheryl H.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">648</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">On page 568, paragraph 3 of the results section entitled "UNCO638 is a potent and substrate-competitive inhibitor" contained a unit error; the cor. units are given and the HTML and PDF versions have been cor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkmoZ87Wwn3rVg90H21EOLACvtfcHk0lgoz6RHeU5z0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1Slu7rE&md5=12d5b26948997d3eefa4f0aab7ea7aad</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1038%2Fnchembio0911-648c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio0911-648c%26sid%3Dliteratum%253Aachs%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DRival-Gervier%26aufirst%3DS.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DLabrie%26aufirst%3DV.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DDiMaggio%26aufirst%3DP.%2BA.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DDong%26aufirst%3DA.%2BP.%26aulast%3DTempel%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DS.%2BC.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DMangano%26aufirst%3DT.%2BJ.%26aulast%3DHuang%26aufirst%3DX.%2BP.%26aulast%3DSimpson%26aufirst%3DC.%2BD.%26aulast%3DPattenden%26aufirst%3DS.%2BG.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DKireev%26aufirst%3DD.%2BB.%26aulast%3DTripathy%26aufirst%3DA.%26aulast%3DEdwards%26aufirst%3DA.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26aulast%3DGarcia%26aufirst%3DB.%2BA.%26aulast%3DPetronis%26aufirst%3DA.%26aulast%3DEllis%26aufirst%3DJ.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DA%2520chemical%2520probe%2520selectively%2520inhibits%2520G9a%2520and%2520GLP%2520methyltransferase%2520activity%2520in%2520cells%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D648%26epage%3D648%26doi%3D10.1038%2Fnchembio0911-648c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiong, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korboukh, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzen, W. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Discovery of an in vivo chemical probe of the lysine methyltransferases G9a and GLP</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">8931</span><span class="NLM_x">–</span> <span class="NLM_lpage">8942</span><span class="refDoi"> DOI: 10.1021/jm401480r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401480r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsF2htrvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=8931-8942&author=F.+Liuauthor=D.+Barsyte-Lovejoyauthor=F.+L.+Liauthor=Y.+Xiongauthor=V.+Korboukhauthor=X.+P.+Huangauthor=A.+Allali-Hassaniauthor=W.+P.+Janzenauthor=B.+L.+Rothauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=M.+Vedadiauthor=J.+Jin&title=Discovery+of+an+in+vivo+chemical+probe+of+the+lysine+methyltransferases+G9a+and+GLP&doi=10.1021%2Fjm401480r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of an in vivo chemical probe of the lysine methyltransferases G9a and GLP</span></div><div class="casAuthors">Liu, Feng; Barsyte-Lovejoy, Dalia; Li, Fengling; Xiong, Yan; Korboukh, Victoria; Huang, Xi-Ping; Allali-Hassani, Abdellah; Janzen, William P.; Roth, Bryan L.; Frye, Stephen V.; Arrowsmith, Cheryl H.; Brown, Peter J.; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">8931-8942</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Among epigenetic writers, readers, and erasers, the lysine methyltransferases G9a and GLP, which catalyze mono- and dimethylation of histone H3 lysine 9 (H3K9me2) and nonhistone proteins, have been implicated in a variety of human diseases.  A toolkit of well-characterized chem. probes will allow biol. and disease hypotheses concerning these proteins to be tested in cell-based and animal models with high confidence.  We previously discovered potent and selective G9a/GLP inhibitors including the cellular chem. probe UNC0638, which displays an excellent sepn. of functional potency and cell toxicity.  However, this inhibitor is not suitable for animal studies due to its poor pharmacokinetic (PK) properties.  Here, we report the discovery of the first G9a and GLP in vivo chem. probe UNC0642, which not only maintains high in vitro and cellular potency, low cell toxicity, and excellent selectivity, but also displays improved in vivo PK properties, making it suitable for animal studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowOI3TSEz3ZbVg90H21EOLACvtfcHk0lgoz6RHeU5z0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsF2htrvK&md5=933f0c49a938b4dd8146f6d09aa21e21</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fjm401480r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401480r%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DF.%2BL.%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DKorboukh%26aufirst%3DV.%26aulast%3DHuang%26aufirst%3DX.%2BP.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520an%2520in%2520vivo%2520chemical%2520probe%2520of%2520the%2520lysine%2520methyltransferases%2520G9a%2520and%2520GLP%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D8931%26epage%3D8942%26doi%3D10.1021%2Fjm401480r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Konze, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pattenden, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simon, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">A chemical tool for in vitro and in vivo precipitation of lysine methyltransferase G9a</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">549</span><span class="NLM_x">–</span> <span class="NLM_lpage">553</span><span class="refDoi"> DOI: 10.1002/cmdc.201300450</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1002%2Fcmdc.201300450" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=24443078" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BC2cXptFeiug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=549-553&author=K.+D.+Konzeauthor=S.+G.+Pattendenauthor=F.+Liuauthor=D.+Barsyte-Lovejoyauthor=F.+L.+Liauthor=J.+M.+Simonauthor=I.+J.+Davisauthor=M.+Vedadiauthor=J.+Jin&title=A+chemical+tool+for+in+vitro+and+in+vivo+precipitation+of+lysine+methyltransferase+G9a&doi=10.1002%2Fcmdc.201300450"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">A Chemical Tool for In Vitro and In Vivo Precipitation of Lysine Methyltransferase G9a</span></div><div class="casAuthors">Konze, Kyle D.; Pattenden, Samantha G.; Liu, Feng; Barsyte-Lovejoy, Dalia; Li, Fengling; Simon, Jeremy M.; Davis, Ian J.; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">549-553</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Here we report the design, synthesis, and biochem. characterization of a new chem. tool, UNC0965.  UNC0965 is a biotinylated version of our previously reported G9a chem. probe, UNC0638.  Importantly, UNC0965 maintains high in vitro potency and is cell penetrant.  The biotinylated tag of UNC0965 enables "chemipptn." of G9a from whole cell lysates.  Further, the cell penetrance of UNC0965 allowed us to explore the localization of G9a on chromatin both in vitro and in vivo through chem. inhibitor-based chromatin immunopptn. (chem-ChIP).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMnElEl-LrHbVg90H21EOLACvtfcHk0lgoz6RHeU5z0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXptFeiug%253D%253D&md5=bbefd445671ed6ee5c22a80eefe61268</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201300450&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201300450%26sid%3Dliteratum%253Aachs%26aulast%3DKonze%26aufirst%3DK.%2BD.%26aulast%3DPattenden%26aufirst%3DS.%2BG.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DF.%2BL.%26aulast%3DSimon%26aufirst%3DJ.%2BM.%26aulast%3DDavis%26aufirst%3DI.%2BJ.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DA%2520chemical%2520tool%2520for%2520in%2520vitro%2520and%2520in%2520vivo%2520precipitation%2520of%2520lysine%2520methyltransferase%2520G9a%26jtitle%3DChemMedChem%26date%3D2014%26volume%3D9%26spage%3D549%26epage%3D553%26doi%3D10.1002%2Fcmdc.201300450" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">Sweis, R. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pliushchev, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maag, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petros, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soni, N. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tse, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michaelides, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiang, G. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pappano, W. N.</span><span> </span><span class="NLM_article-title">Discovery and development of potent and selective inhibitors of histone methyltransferase G9a</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">205</span><span class="NLM_x">–</span> <span class="NLM_lpage">209</span><span class="refDoi"> DOI: 10.1021/ml400496h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml400496h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFKmsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=205-209&author=R.+F.+Sweisauthor=M.+Pliushchevauthor=P.+J.+Brownauthor=J.+Guoauthor=F.+L.+Liauthor=D.+Maagauthor=A.+M.+Petrosauthor=N.+B.+Soniauthor=C.+Tseauthor=M.+Vedadiauthor=M.+R.+Michaelidesauthor=G.+G.+Chiangauthor=W.+N.+Pappano&title=Discovery+and+development+of+potent+and+selective+inhibitors+of+histone+methyltransferase+G9a&doi=10.1021%2Fml400496h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Development of Potent and Selective Inhibitors of Histone Methyltransferase G9a</span></div><div class="casAuthors">Sweis, Ramzi F.; Pliushchev, Marina; Brown, Peter J.; Guo, Jun; Li, Fengling; Maag, David; Petros, Andrew M.; Soni, Nirupama B.; Tse, Chris; Vedadi, Masoud; Michaelides, Michael R.; Chiang, Gary G.; Pappano, William N.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">205-209</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">G9a is a histone lysine methyltransferase responsible for the methylation of histone H3 lysine 9.  The discovery of A-366 arose from a unique diversity screening hit, which was optimized by incorporation of a propyl-pyrrolidine subunit to occupy the enzyme lysine channel.  A-366 (I) is a potent inhibitor of G9a (IC50: 3.3 nM) with greater than 1000-fold selectivity over 21 other methyltransferases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMvdTiUgx-oLVg90H21EOLACvtfcHk0lhiOLeXwIpkeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFKmsA%253D%253D&md5=26affad72f2950510aa9f253e540d575</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Fml400496h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml400496h%26sid%3Dliteratum%253Aachs%26aulast%3DSweis%26aufirst%3DR.%2BF.%26aulast%3DPliushchev%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DGuo%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DF.%2BL.%26aulast%3DMaag%26aufirst%3DD.%26aulast%3DPetros%26aufirst%3DA.%2BM.%26aulast%3DSoni%26aufirst%3DN.%2BB.%26aulast%3DTse%26aufirst%3DC.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DMichaelides%26aufirst%3DM.%2BR.%26aulast%3DChiang%26aufirst%3DG.%2BG.%26aulast%3DPappano%26aufirst%3DW.%2BN.%26atitle%3DDiscovery%2520and%2520development%2520of%2520potent%2520and%2520selective%2520inhibitors%2520of%2520histone%2520methyltransferase%2520G9a%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D5%26spage%3D205%26epage%3D209%26doi%3D10.1021%2Fml400496h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group">Yuan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paulk, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kubicek, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kemp, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shamji, A. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wagner, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schreiber, S. L.</span><span> </span><span class="NLM_article-title">A small-molecule probe of the histone methyltransferase G9a induces cellular senescence in pancreatic adenocarcinoma</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">1152</span><span class="NLM_x">–</span> <span class="NLM_lpage">1157</span><span class="refDoi"> DOI: 10.1021/cb300139y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb300139y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=1152-1157&author=Y.+Yuanauthor=Q.+Wangauthor=J.+Paulkauthor=S.+Kubicekauthor=M.+M.+Kempauthor=D.+J.+Adamsauthor=A.+F.+Shamjiauthor=B.+K.+Wagnerauthor=S.+L.+Schreiber&title=A+small-molecule+probe+of+the+histone+methyltransferase+G9a+induces+cellular+senescence+in+pancreatic+adenocarcinoma&doi=10.1021%2Fcb300139y"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fcb300139y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb300139y%26sid%3Dliteratum%253Aachs%26aulast%3DYuan%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DKubicek%26aufirst%3DS.%26aulast%3DKemp%26aufirst%3DM.%2BM.%26aulast%3DAdams%26aufirst%3DD.%2BJ.%26aulast%3DShamji%26aufirst%3DA.%2BF.%26aulast%3DWagner%26aufirst%3DB.%2BK.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26atitle%3DA%2520small-molecule%2520probe%2520of%2520the%2520histone%2520methyltransferase%2520G9a%2520induces%2520cellular%2520senescence%2520in%2520pancreatic%2520adenocarcinoma%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2012%26volume%3D7%26spage%3D1152%26epage%3D1157%26doi%3D10.1021%2Fcb300139y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group">Devkota, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lohse, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Staerk, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berthelsen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clausen, R. P.</span><span> </span><span class="NLM_article-title">Analogues of the natural product sinefungin as inhibitors of EHMT1 and EHMT2</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">293</span><span class="NLM_x">–</span> <span class="NLM_lpage">297</span><span class="refDoi"> DOI: 10.1021/ml4002503</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml4002503" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1Oks7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=293-297&author=K.+Devkotaauthor=B.+Lohseauthor=Q.+Liuauthor=M.+W.+Wangauthor=D.+Staerkauthor=J.+Berthelsenauthor=R.+P.+Clausen&title=Analogues+of+the+natural+product+sinefungin+as+inhibitors+of+EHMT1+and+EHMT2&doi=10.1021%2Fml4002503"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Analogs of the Natural Product Sinefungin as Inhibitors of EHMT1 and EHMT2</span></div><div class="casAuthors">Devkota, Kanchan; Lohse, Brian; Liu, Qing; Wang, Ming-Wei; Staerk, Dan; Berthelsen, Jens; Clausen, Rasmus Praetorius</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">293-297</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of analogs of the natural product sinefungin lacking the amino acid moiety was synthesized and probed for their ability to inhibit EHMT1 and EHMT2.  This study led to inhibitors, e.g. I, of methyltransferase activity of EHMT1 and EHMT2 and it demonstrates that such analogs constitute an interesting scaffold to develop selective methyltransferase inhibitors.  Surprisingly, the inhibition was not competitive toward AdoMet.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLcJbLzR-Sy7Vg90H21EOLACvtfcHk0lhiOLeXwIpkeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1Oks7g%253D&md5=d5992e33faf67b007932bb70c0bd8ed5</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Fml4002503&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml4002503%26sid%3Dliteratum%253Aachs%26aulast%3DDevkota%26aufirst%3DK.%26aulast%3DLohse%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DM.%2BW.%26aulast%3DStaerk%26aufirst%3DD.%26aulast%3DBerthelsen%26aufirst%3DJ.%26aulast%3DClausen%26aufirst%3DR.%2BP.%26atitle%3DAnalogues%2520of%2520the%2520natural%2520product%2520sinefungin%2520as%2520inhibitors%2520of%2520EHMT1%2520and%2520EHMT2%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D5%26spage%3D293%26epage%3D297%26doi%3D10.1021%2Fml4002503" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group">Kondengaden, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, L. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, M. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zang, L. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bataba, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, B. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, P. G.</span><span> </span><span class="NLM_article-title">Discovery of novel small molecule inhibitors of lysine methyltransferase G9a and their mechanism in leukemia cell lines</span> <span class="citation_source-journal">Eur. J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">122</span><span class="NLM_x">, </span> <span class="NLM_fpage">382</span><span class="NLM_x">–</span> <span class="NLM_lpage">393</span><span class="refDoi"> DOI: 10.1016/j.ejmech.2016.06.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1016%2Fj.ejmech.2016.06.028" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=122&publication_year=2016&pages=382-393&author=S.+M.+Kondengadenauthor=L.+F.+Luoauthor=K.+Huangauthor=M.+Y.+Zhuauthor=L.+L.+Zangauthor=E.+Batabaauthor=R.+L.+Wangauthor=C.+Luoauthor=B.+H.+Wangauthor=K.+K.+Liauthor=P.+G.+Wang&title=Discovery+of+novel+small+molecule+inhibitors+of+lysine+methyltransferase+G9a+and+their+mechanism+in+leukemia+cell+lines&doi=10.1016%2Fj.ejmech.2016.06.028"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2016.06.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2016.06.028%26sid%3Dliteratum%253Aachs%26aulast%3DKondengaden%26aufirst%3DS.%2BM.%26aulast%3DLuo%26aufirst%3DL.%2BF.%26aulast%3DHuang%26aufirst%3DK.%26aulast%3DZhu%26aufirst%3DM.%2BY.%26aulast%3DZang%26aufirst%3DL.%2BL.%26aulast%3DBataba%26aufirst%3DE.%26aulast%3DWang%26aufirst%3DR.%2BL.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DB.%2BH.%26aulast%3DLi%26aufirst%3DK.%2BK.%26aulast%3DWang%26aufirst%3DP.%2BG.%26atitle%3DDiscovery%2520of%2520novel%2520small%2520molecule%2520inhibitors%2520of%2520lysine%2520methyltransferase%2520G9a%2520and%2520their%2520mechanism%2520in%2520leukemia%2520cell%2520lines%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2016%26volume%3D122%26spage%3D382%26epage%3D393%26doi%3D10.1016%2Fj.ejmech.2016.06.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group">Chen, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Z. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, T. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, D. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, X. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">You, Q. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, X. K.</span><span> </span><span class="NLM_article-title">Discovery, design and synthesis of 6H-anthra[1,9-cd]isoxazol-6-one scaffold as G9a inhibitor through a combination of shape-based virtual screening and structure-based molecular modification</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">6102</span><span class="NLM_x">–</span> <span class="NLM_lpage">6108</span><span class="refDoi"> DOI: 10.1016/j.bmc.2016.09.071</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1016%2Fj.bmc.2016.09.071" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2016&pages=6102-6108&author=W.+L.+Chenauthor=Z.+H.+Wangauthor=T.+T.+Fengauthor=D.+D.+Liauthor=C.+H.+Wangauthor=X.+L.+Xuauthor=X.+J.+Zhangauthor=Q.+D.+Youauthor=X.+K.+Guo&title=Discovery%2C+design+and+synthesis+of+6H-anthra%5B1%2C9-cd%5Disoxazol-6-one+scaffold+as+G9a+inhibitor+through+a+combination+of+shape-based+virtual+screening+and+structure-based+molecular+modification&doi=10.1016%2Fj.bmc.2016.09.071"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2016.09.071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2016.09.071%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DW.%2BL.%26aulast%3DWang%26aufirst%3DZ.%2BH.%26aulast%3DFeng%26aufirst%3DT.%2BT.%26aulast%3DLi%26aufirst%3DD.%2BD.%26aulast%3DWang%26aufirst%3DC.%2BH.%26aulast%3DXu%26aufirst%3DX.%2BL.%26aulast%3DZhang%26aufirst%3DX.%2BJ.%26aulast%3DYou%26aufirst%3DQ.%2BD.%26aulast%3DGuo%26aufirst%3DX.%2BK.%26atitle%3DDiscovery%252C%2520design%2520and%2520synthesis%2520of%25206H-anthra%255B1%252C9-cd%255Disoxazol-6-one%2520scaffold%2520as%2520G9a%2520inhibitor%2520through%2520a%2520combination%2520of%2520shape-based%2520virtual%2520screening%2520and%2520structure-based%2520molecular%2520modification%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2016%26volume%3D24%26spage%3D6102%26epage%3D6108%26doi%3D10.1016%2Fj.bmc.2016.09.071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group">Lehnertz, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pabst, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Su, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krosl, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yung, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirschner, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosten, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Underhill, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hebert, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sauvageau, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Humphries, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rossi, F. M.</span><span> </span><span class="NLM_article-title">The methyltransferase G9a regulates HoxA9-dependent transcription in AML</span> <span class="citation_source-journal">Genes Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">317</span><span class="NLM_x">–</span> <span class="NLM_lpage">327</span><span class="refDoi"> DOI: 10.1101/gad.236794.113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1101%2Fgad.236794.113" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2014&pages=317-327&author=B.+Lehnertzauthor=C.+Pabstauthor=L.+Suauthor=M.+Millerauthor=F.+Liuauthor=L.+Yiauthor=R.+Zhangauthor=J.+Kroslauthor=E.+Yungauthor=J.+Kirschnerauthor=P.+Rostenauthor=T.+M.+Underhillauthor=J.+Jinauthor=J.+Hebertauthor=G.+Sauvageauauthor=R.+K.+Humphriesauthor=F.+M.+Rossi&title=The+methyltransferase+G9a+regulates+HoxA9-dependent+transcription+in+AML&doi=10.1101%2Fgad.236794.113"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1101%2Fgad.236794.113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.236794.113%26sid%3Dliteratum%253Aachs%26aulast%3DLehnertz%26aufirst%3DB.%26aulast%3DPabst%26aufirst%3DC.%26aulast%3DSu%26aufirst%3DL.%26aulast%3DMiller%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DYi%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DR.%26aulast%3DKrosl%26aufirst%3DJ.%26aulast%3DYung%26aufirst%3DE.%26aulast%3DKirschner%26aufirst%3DJ.%26aulast%3DRosten%26aufirst%3DP.%26aulast%3DUnderhill%26aufirst%3DT.%2BM.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DHebert%26aufirst%3DJ.%26aulast%3DSauvageau%26aufirst%3DG.%26aulast%3DHumphries%26aufirst%3DR.%2BK.%26aulast%3DRossi%26aufirst%3DF.%2BM.%26atitle%3DThe%2520methyltransferase%2520G9a%2520regulates%2520HoxA9-dependent%2520transcription%2520in%2520AML%26jtitle%3DGenes%2520Dev.%26date%3D2014%26volume%3D28%26spage%3D317%26epage%3D327%26doi%3D10.1101%2Fgad.236794.113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group">Bailey, S.</span><span> </span><span class="NLM_article-title">The CCP4 suite - programs for protein crystallography</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">760</span><span class="NLM_x">–</span> <span class="NLM_lpage">763</span><span class="refDoi"> DOI: 10.1107/S0907444994003112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1107%2FS0907444994003112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=15299374" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=1994&pages=760-763&author=S.+Bailey&title=The+CCP4+suite+-+programs+for+protein+crystallography&doi=10.1107%2FS0907444994003112"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1107%2FS0907444994003112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444994003112%26sid%3Dliteratum%253Aachs%26aulast%3DBailey%26aufirst%3DS.%26atitle%3DThe%2520CCP4%2520suite%2520-%2520programs%2520for%2520protein%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1994%26volume%3D50%26spage%3D760%26epage%3D763%26doi%3D10.1107%2FS0907444994003112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group">McCoy, A. J.</span><span> </span><span class="NLM_article-title">Solving structures of protein complexes by molecular replacement with Phaser</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">63</span><span class="NLM_x">, </span> <span class="NLM_fpage">32</span><span class="NLM_x">–</span> <span class="NLM_lpage">41</span><span class="refDoi"> DOI: 10.1107/S0907444906045975</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1107%2FS0907444906045975" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=17164524" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtlSqu77F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2007&pages=32-41&author=A.+J.+McCoy&title=Solving+structures+of+protein+complexes+by+molecular+replacement+with+Phaser&doi=10.1107%2FS0907444906045975"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Solving structures of protein complexes by molecular replacement with Phaser</span></div><div class="casAuthors">McCoy, Airlie J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">32-41</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Mol. replacement (MR) generally becomes more difficult as the no. of components in the asym. unit requiring sep. MR models (i.e. the dimensionality of the search) increases.  When the proportion of the total scattering contributed by each search component is small, the signal in the search for each component in isolation is weak or non-existent.  Maximum-likelihood MR functions enable complex asym. units to be built up from individual components with a 'tree search with pruning' approach.  This method, as implemented in the automated search procedure of the program Phaser, has been very successful in solving many previously intractable MR problems.  However, there are a no. of cases in which the automated search procedure of Phaser is suboptimal or encounters difficulties.  These include cases where there are a large no. of copies of the same component in the asym. unit or where the components of the asym. unit have greatly varying B factors.  Two case studies are presented to illustrate how Phaser can be used to best advantage in the std. 'automated MR' mode and two case studies are used to show how to modify the automated search strategy for problematic cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAvs9W8hQ3l7Vg90H21EOLACvtfcHk0lhnv5piVVIn9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtlSqu77F&md5=323a5297bdf622ebaf398a87c39dc794</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1107%2FS0907444906045975&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444906045975%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26atitle%3DSolving%2520structures%2520of%2520protein%2520complexes%2520by%2520molecular%2520replacement%2520with%2520Phaser%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2007%26volume%3D63%26spage%3D32%26epage%3D41%26doi%3D10.1107%2FS0907444906045975" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group">Murshudov, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vagin, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodson, E. J.</span><span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">240</span><span class="NLM_x">–</span> <span class="NLM_lpage">255</span><span class="refDoi"> DOI: 10.1107/S0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method&doi=10.1107%2FS0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0lhnv5piVVIn9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255%26doi%3D10.1107%2FS0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Afonine, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunkoczi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, I. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Echols, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hung, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapral, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moriarty, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oeffner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terwilliger, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zwart, P. H.</span><span> </span><span class="NLM_article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">213</span><span class="NLM_x">–</span> <span class="NLM_lpage">221</span><span class="refDoi"> DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunkocziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.+W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+a+comprehensive+Python-based+system+for+macromolecular+structure+solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0ljSYAkhb7PLLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunkoczi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.%2BW.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520a%2520comprehensive%2520Python-based%2520system%2520for%2520macromolecular%2520structure%2520solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group">Afonine, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Echols, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moriarty, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mustyakimov, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terwilliger, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Urzhumtsev, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zwart, P. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, P. D.</span><span> </span><span class="NLM_article-title">Towards automated crystallographic structure refinement with phenix.refine</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">68</span><span class="NLM_x">, </span> <span class="NLM_fpage">352</span><span class="NLM_x">–</span> <span class="NLM_lpage">367</span><span class="refDoi"> DOI: 10.1107/S0907444912001308</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1107%2FS0907444912001308" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=22505256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BC38Xlt1Ggu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2012&pages=352-367&author=P.+V.+Afonineauthor=R.+W.+Grosse-Kunstleveauthor=N.+Echolsauthor=J.+J.+Headdauthor=N.+W.+Moriartyauthor=M.+Mustyakimovauthor=T.+C.+Terwilligerauthor=A.+Urzhumtsevauthor=P.+H.+Zwartauthor=P.+D.+Adams&title=Towards+automated+crystallographic+structure+refinement+with+phenix.refine&doi=10.1107%2FS0907444912001308"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Towards automated crystallographic structure refinement with phenix.refine</span></div><div class="casAuthors">Afonine, Pavel V.; Grosse-Kunstleve, Ralf W.; Echols, Nathaniel; Headd, Jeffrey J.; Moriarty, Nigel W.; Mustyakimov, Marat; Terwilliger, Thomas C.; Urzhumtsev, Alexandre; Zwart, Peter H.; Adams, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">352-367</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phenix.refine is a program within the PHENIX package that supports crystallog. structure refinement against exptl. data with a wide range of upper resoln. limits using a large repertoire of model parameterizations.  It has several automation features and is also highly flexible.  Several hundred parameters enable extensive customizations for complex use cases.  Multiple user-defined refinement strategies can be applied to specific parts of the model in a single refinement run.  An intuitive graphical user interface is available to guide novice users and to assist advanced users in managing refinement projects.  X-ray or neutron diffraction data can be used sep. or jointly in refinement. phenix.refine is tightly integrated into the PHENIX suite, where it serves as a crit. component in automated model building, final structure refinement, structure validation and deposition to the wwPDB.  This paper presents an overview of the major phenix.refine features, with extensive literature refs. for readers interested in more detailed discussions of the methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnhn9QwIEkTbVg90H21EOLACvtfcHk0ljSYAkhb7PLLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xlt1Ggu7c%253D&md5=1cc7da2cea396f1a75646ea709140cf1</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1107%2FS0907444912001308&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444912001308%26sid%3Dliteratum%253Aachs%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DMustyakimov%26aufirst%3DM.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DUrzhumtsev%26aufirst%3DA.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26atitle%3DTowards%2520automated%2520crystallographic%2520structure%2520refinement%2520with%2520phenix.refine%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2012%26volume%3D68%26spage%3D352%26epage%3D367%26doi%3D10.1107%2FS0907444912001308" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lohkamp, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Features and development of Coot</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">486</span><span class="NLM_x">–</span> <span class="NLM_lpage">501</span><span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0ljSYAkhb7PLLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group">Davis, I. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, D. C.</span><span> </span><span class="NLM_article-title">MolProbity: structure validation and all-atom contact analysis for nucleic acids and their complexes</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">W615</span><span class="NLM_x">–</span> <span class="NLM_lpage">W619</span><span class="refDoi"> DOI: 10.1093/nar/gkh398</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1093%2Fnar%2Fgkh398" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=15215462" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADC%252BD2cXlvFKmsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2004&pages=W615-W619&author=I.+W.+Davisauthor=L.+W.+Murrayauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=MolProbity%3A+structure+validation+and+all-atom+contact+analysis+for+nucleic+acids+and+their+complexes&doi=10.1093%2Fnar%2Fgkh398"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">MolPROBITY: Structure validation and all-atom contact analysis for nucleic acids and their complexes</span></div><div class="casAuthors">Davis, Ian W.; Murray, Laura Weston; Richardson, Jane S.; Richardson, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">Web Server</span>),
    <span class="NLM_cas:pages">W615-W619</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">MolProbity is a general-purpose web service offering quality validation for three-dimensional (3D) structures of proteins, nucleic acids and complexes.  It provides detailed all-atom contact anal. of any steric problems within the mols. and can calc. and display the H-bond and van der Waals contacts in the interfaces between components.  An integral step in the process is the addn. and full optimization of all hydrogen atoms, both polar and nonpolar.  The results are reported in multiple forms: as overall numeric scores, as lists, as downloadable PDB and graphics files, and most notably as informative, manipulable 3D kinemage graphics shown online in the KING viewer.  This service is available free to all users at http://kinemage.biochem.duke.edu.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9I64e-HQbnbVg90H21EOLACvtfcHk0lg3q1hY9nTaAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXlvFKmsLY%253D&md5=2050f7acf49743addde69c5ce9c9e2e4</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkh398&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkh398%26sid%3Dliteratum%253Aachs%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DMurray%26aufirst%3DL.%2BW.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMolProbity%253A%2520structure%2520validation%2520and%2520all-atom%2520contact%2520analysis%2520for%2520nucleic%2520acids%2520and%2520their%2520complexes%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2004%26volume%3D32%26spage%3DW615%26epage%3DW619%26doi%3D10.1093%2Fnar%2Fgkh398" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Min, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lunin, V. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Antoshenko, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dombrovski, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campagna-Slater, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plotnikov, A. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span> </span><span class="NLM_article-title">Structural biology of human H3K9 methyltransferases</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">e8570</span><span class="refDoi"> DOI: 10.1371/journal.pone.0008570</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1371%2Fjournal.pone.0008570" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=20084102" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A280%3ADC%252BC3c%252FjtlGgsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=e8570&author=H.+Wuauthor=J.+R.+Minauthor=V.+V.+Luninauthor=T.+Antoshenkoauthor=L.+Dombrovskiauthor=H.+Zengauthor=A.+Allali-Hassaniauthor=V.+Campagna-Slaterauthor=M.+Vedadiauthor=C.+H.+Arrowsmithauthor=A.+N.+Plotnikovauthor=M.+Schapira&title=Structural+biology+of+human+H3K9+methyltransferases&doi=10.1371%2Fjournal.pone.0008570"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Structural biology of human H3K9 methyltransferases</span></div><div class="casAuthors">Wu Hong; Min Jinrong; Lunin Vladimir V; Antoshenko Tatiana; Dombrovski Ludmila; Zeng Hong; Allali-Hassani Abdellah; Campagna-Slater Valerie; Vedadi Masoud; Arrowsmith Cheryl H; Plotnikov Alexander N; Schapira Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">PloS one</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">e8570</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">SET domain methyltransferases deposit methyl marks on specific histone tail lysine residues and play a major role in epigenetic regulation of gene transcription.  We solved the structures of the catalytic domains of GLP, G9a, Suv39H2 and PRDM2, four of the eight known human H3K9 methyltransferases in their apo conformation or in complex with the methyl donating cofactor, and peptide substrates.  We analyzed the structural determinants for methylation state specificity, and designed a G9a mutant able to tri-methylate H3K9.  We show that the I-SET domain acts as a rigid docking platform, while induced-fit of the Post-SET domain is necessary to achieve a catalytically competent conformation.  We also propose a model where long-range electrostatics bring enzyme and histone substrate together, while the presence of an arginine upstream of the target lysine is critical for binding and specificity.  ENHANCED VERSION: This article can also be viewed as an enhanced version in which the text of the article is integrated with interactive 3D representations and animated transitions.  Please note that a web plugin is required to access this enhanced functionality.  Instructions for the installation and use of the web plugin are available in Text S1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTAEo3Y9d9dncRAV4D_Ins_fW6udTcc2eYqy_V0h6qoVrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3c%252FjtlGgsA%253D%253D&md5=2649472b1401c69035f99e0ca20822a3</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0008570&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0008570%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DH.%26aulast%3DMin%26aufirst%3DJ.%2BR.%26aulast%3DLunin%26aufirst%3DV.%2BV.%26aulast%3DAntoshenko%26aufirst%3DT.%26aulast%3DDombrovski%26aufirst%3DL.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DCampagna-Slater%26aufirst%3DV.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DPlotnikov%26aufirst%3DA.%2BN.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DStructural%2520biology%2520of%2520human%2520H3K9%2520methyltransferases%26jtitle%3DPLoS%2520One%26date%3D2010%26volume%3D5%26spage%3De8570%26doi%3D10.1371%2Fjournal.pone.0008570" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group">Otwinowski, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minor, W.</span><span> </span><span class="NLM_article-title">Processing of X-ray diffraction data collected in oscillation mode</span> <span class="citation_source-journal">Methods Enzymol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">276</span><span class="NLM_x">, </span> <span class="NLM_fpage">307</span><span class="NLM_x">–</span> <span class="NLM_lpage">326</span><span class="refDoi"> DOI: 10.1016/S0076-6879(97)76066-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=27754618" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=1997&pages=307-326&author=Z.+Otwinowskiauthor=W.+Minor&title=Processing+of+X-ray+diffraction+data+collected+in+oscillation+mode&doi=10.1016%2FS0076-6879%2897%2976066-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Processing of x-ray diffraction data collected in oscillation mode</span></div><div class="casAuthors">Otwinowski, Zbyszek; Minor, Wladek</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">Macromolecular Crystallography, Part A</span>),
    <span class="NLM_cas:pages">307-326</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Macromol. crystallog. is an iterative process.  Rarely do the first crystals provide all the necessary data to solve the biol. problem being studied.  Each step benefits from experience learned in previous steps.  To monitor the progress, the HKL package provides 2 tools: (1) statistics, both weighted (χ2) and unweighted (R-merge), are provided, and the Bayesian reasoning and multicomponent error model facilitates obtaining the proper error ests. and (2) visualization of the process plays a double role by helping the operator to confirm that the process of data redn., including the resulting statistics, is correct, and allowing one to evaluate problems for which there are no good statistical criteria.  Visualization also provides confidence that the point of diminishing returns in data collection and redn. has been reached.  At that point, the effort should be directed to solving the structure.  The methods presented here have been applied to solve a large variety of problems, from inorg. mols. with 5 Å unit cell to rotavirus of 700 Å diam. crystd. in 700 × 1000 × 1400 Å cell.  Overall quality of the method was tested by many researchers by successful application of the programs to MAD structure detns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA3fvBOHqhhrVg90H21EOLACvtfcHk0lg3q1hY9nTaAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D&md5=c9536971d4e32cc35352c40fb9368131</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DMinor%26aufirst%3DW.%26atitle%3DProcessing%2520of%2520X-ray%2520diffraction%2520data%2520collected%2520in%2520oscillation%2520mode%26jtitle%3DMethods%2520Enzymol.%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D326%26doi%3D10.1016%2FS0076-6879%2897%2976066-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group">Vagin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teplyakov, A.</span><span> </span><span class="NLM_article-title">MOLREP: an automated program for molecular replacement</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">1022</span><span class="NLM_x">–</span> <span class="NLM_lpage">1025</span><span class="refDoi"> DOI: 10.1107/S0021889897006766</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=10.1107%2FS0021889897006766" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;key=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1997&pages=1022-1025&author=A.+Vaginauthor=A.+Teplyakov&title=MOLREP%3A+an+automated+program+for+molecular+replacement&doi=10.1107%2FS0021889897006766"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">MOLREP: an automated program for molecular replacement</span></div><div class="casAuthors">Vagin, Alexei; Teplyakov, Alexei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1022-1025</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard International Publishers Ltd.</span>)
        </div><div class="casAbstract">MOLREP is an automated program for mol. replacement which uses effective new approaches in data processing and rotational and translational searching.  These include an automatic choice of all parameters, scaling by Patterson origin peaks and soft resoln. cutoff.  One of the cornerstones of the program is an original full-symmetry translation function combined with a packing function.  Information from the model already placed in the cell is incorporated in both translation and packing functions.  A no. of tests using exptl. data proved the ability of the program to find the correct soln. in difficult cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkGXdZ-nTKxLVg90H21EOLACvtfcHk0lg3q1hY9nTaAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D&md5=96af64aedd8a8f018c3c18b57a6b4b21</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1107%2FS0021889897006766&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889897006766%26sid%3Dliteratum%253Aachs%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DTeplyakov%26aufirst%3DA.%26atitle%3DMOLREP%253A%2520an%2520automated%2520program%2520for%2520molecular%2520replacement%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D1997%26volume%3D30%26spage%3D1022%26epage%3D1025%26doi%3D10.1107%2FS0021889897006766" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TUZ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TUZ','PDB','5TUZ'); return false;">PDB: 5TUZ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TUY" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TUY','PDB','5TUY'); return false;">PDB: 5TUY</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TTG" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TTG','PDB','5TTG'); return false;">PDB: 5TTG</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TTF" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TTF','PDB','5TTF'); return false;">PDB: 5TTF</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3MO5" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3MO5','PDB','3MO5'); return false;">PDB: 3MO5</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3K5K" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3K5K','PDB','3K5K'); return false;">PDB: 3K5K</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3HNA" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3HNA','PDB','3HNA'); return false;">PDB: 3HNA</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4I51" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4I51','PDB','4I51'); return false;">PDB: 4I51</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i72"><a href="/doi/suppl/10.1021/acs.jmedchem.6b01645">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_20222"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.6b01645">10.1021/acs.jmedchem.6b01645</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline"><sup>1</sup>H and <sup>13</sup>C NMR spectra of compound <b>4</b> and <b>18</b>; crystallography data and refinement statistics (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b01645/suppl_file/jm6b01645_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b01645/suppl_file/jm6b01645_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01645/suppl_file/jm6b01645_si_001.pdf">jm6b01645_si_001.pdf (282.66 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01645/suppl_file/jm6b01645_si_002.csv">jm6b01645_si_002.csv (1.83 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The structure of GLP in complex with <b>4</b> has been deposited under PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TUZ">5TUZ</a>. The structure of G9a in complex with <b>4</b> has been deposited under PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TUY">5TUY</a>. The structure of GLP in complex with <b>18</b> has been deposited under PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TTG">5TTG</a>. The structure of G9a in complex with <b>18</b> has been deposited under PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TTF">5TTF</a>. Authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.6b01645&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.6b01645%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2017.60.issue-5%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.6b01645" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679994038c9c21e9","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
