
<html lang="en"     class="pb-page"  data-request-id="db034a97-a117-428e-92a7-dab64644de78"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;article:article:10.1021/acs.jmedchem.9b00522;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2019.62.issue-20;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Discovery of a Potent and Selective Fragment-like Inhibitor of Methyllysine Reader Protein Spindlin 1 (SPIN1)" /></meta><meta name="dc.Creator" content="Yan  Xiong" /></meta><meta name="dc.Creator" content="Holger  Greschik" /></meta><meta name="dc.Creator" content="Catrine  Johansson" /></meta><meta name="dc.Creator" content="Ludwig  Seifert" /></meta><meta name="dc.Creator" content="Johannes  Bacher" /></meta><meta name="dc.Creator" content="Kwang-su  Park" /></meta><meta name="dc.Creator" content="Nicolas  Babault" /></meta><meta name="dc.Creator" content="Michael  Martini" /></meta><meta name="dc.Creator" content="Vincent  Fagan" /></meta><meta name="dc.Creator" content="Fengling  Li" /></meta><meta name="dc.Creator" content="Irene  Chau" /></meta><meta name="dc.Creator" content="Thomas  Christott" /></meta><meta name="dc.Creator" content="David  Dilworth" /></meta><meta name="dc.Creator" content="Dalia  Barsyte-Lovejoy" /></meta><meta name="dc.Creator" content="Masoud  Vedadi" /></meta><meta name="dc.Creator" content="Cheryl H.  Arrowsmith" /></meta><meta name="dc.Creator" content="Paul  Brennan" /></meta><meta name="dc.Creator" content="Oleg  Fedorov" /></meta><meta name="dc.Creator" content="Manfred  Jung" /></meta><meta name="dc.Creator" content="Gillian  Farnie" /></meta><meta name="dc.Creator" content="Jing  Liu" /></meta><meta name="dc.Creator" content="Udo  Oppermann" /></meta><meta name="dc.Creator" content="Roland  Schüle" /></meta><meta name="dc.Creator" content="Jian  Jin" /></meta><meta name="dc.Description" content="By screening an epigenetic compound library, we identified that UNC0638, a highly potent inhibitor of the histone methyltransferases G9a and GLP, was a weak inhibitor of SPIN1 (spindlin 1), a methy..." /></meta><meta name="Description" content="By screening an epigenetic compound library, we identified that UNC0638, a highly potent inhibitor of the histone methyltransferases G9a and GLP, was a weak inhibitor of SPIN1 (spindlin 1), a methy..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="July 1, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b00522" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00522" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b00522" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00522" /></link>
        
    
    

<title>Discovery of a Potent and Selective Fragment-like Inhibitor of Methyllysine Reader Protein Spindlin 1 (SPIN1) | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b00522" /></meta><meta property="og:title" content="Discovery of a Potent and Selective Fragment-like Inhibitor of Methyllysine Reader Protein Spindlin 1 (SPIN1)" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/large/jm-2019-00522j_0011.jpeg" /></meta><meta property="og:description" content="By screening an epigenetic compound library, we identified that UNC0638, a highly potent inhibitor of the histone methyltransferases G9a and GLP, was a weak inhibitor of SPIN1 (spindlin 1), a methyllysine reader protein. Our optimization of this weak hit resulted in the discovery of a potent, selective, and cell-active SPIN1 inhibitor, compound 3 (MS31). Compound 3 potently inhibited binding of trimethyllysine-containing peptides to SPIN1, displayed high binding affinity, was highly selective for SPIN1 over other epigenetic readers and writers, directly engaged SPIN1 in cells, and was not toxic to nontumorigenic cells. The crystal structure of the SPIN1–compound 3 complex indicated that it selectively binds tudor domain II of SPIN1. We also designed a structurally similar but inactive compound 4 (MS31N) as a negative control. Our results have demonstrated for the first time that potent, selective, and cell-active fragment-like inhibitors can be generated by targeting a single tudor domain." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b00522"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b00522">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b00522&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b00522&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b00522&amp;href=/doi/10.1021/acs.jmedchem.9b00522" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 20</span><span class="cit-fg-pageRange">, 8996-9007</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/20" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b00444" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b00562" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of a Potent and Selective Fragment-like Inhibitor of Methyllysine Reader Protein Spindlin 1 (SPIN1)</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Yan Xiong</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yan Xiong</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Mount Sinai Center for Therapeutics Discovery, Departments of Pharmacological Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yan++Xiong">Yan Xiong</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Holger Greschik</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Holger Greschik</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Urology and Center for Clinical Research, University Freiburg Medical Center, Freiburg 79106, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Holger++Greschik">Holger Greschik</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Catrine Johansson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Catrine Johansson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, Botnar Research Center, NIHR Oxford BRU, University of Oxford, Oxford OX37LD, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Catrine++Johansson">Catrine Johansson</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ludwig Seifert</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ludwig Seifert</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Pharmaceutical Sciences, University of Freiburg, Freiburg 79104, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ludwig++Seifert">Ludwig Seifert</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Johannes Bacher</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Johannes Bacher</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Pharmaceutical Sciences, University of Freiburg, Freiburg 79104, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Johannes++Bacher">Johannes Bacher</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kwang-su Park</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kwang-su Park</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Mount Sinai Center for Therapeutics Discovery, Departments of Pharmacological Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kwang-su++Park">Kwang-su Park</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Nicolas Babault</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nicolas Babault</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Mount Sinai Center for Therapeutics Discovery, Departments of Pharmacological Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nicolas++Babault">Nicolas Babault</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Michael Martini</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Michael Martini</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Mount Sinai Center for Therapeutics Discovery, Departments of Pharmacological Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Michael++Martini">Michael Martini</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Vincent Fagan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Vincent Fagan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium & Target Discovery Institute, University of Oxford, Oxford OX37DQ and OX37FZ, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Vincent++Fagan">Vincent Fagan</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Fengling Li</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Fengling Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Fengling++Li">Fengling Li</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Irene Chau</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Irene Chau</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Irene++Chau">Irene Chau</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Thomas Christott</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Thomas Christott</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium & Target Discovery Institute, University of Oxford, Oxford OX37DQ and OX37FZ, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Thomas++Christott">Thomas Christott</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">David Dilworth</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">David Dilworth</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=David++Dilworth">David Dilworth</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Dalia Barsyte-Lovejoy</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dalia Barsyte-Lovejoy</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</div><div class="loa-info-affiliations-info">Nature Research Center, Akademijos 2, Vilnius 08412, Lithuania</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dalia++Barsyte-Lovejoy">Dalia Barsyte-Lovejoy</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Masoud Vedadi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Masoud Vedadi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</div><div class="loa-info-affiliations-info">Department of Pharmacology and Toxicology, University of Toronto, Toronto, Ontario M5S 1A8, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Masoud++Vedadi">Masoud Vedadi</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Cheryl H. Arrowsmith</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Cheryl H. Arrowsmith</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium,  Princess Margaret Cancer Centre and Department of Medical Biophysics, University of Toronto, Toronto, Ontario M5G 1L7, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Cheryl+H.++Arrowsmith">Cheryl H. Arrowsmith</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4971-3250" title="Orcid link">http://orcid.org/0000-0002-4971-3250</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Paul Brennan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Paul Brennan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium & Target Discovery Institute, University of Oxford, Oxford OX37DQ and OX37FZ, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Paul++Brennan">Paul Brennan</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8950-7646" title="Orcid link">http://orcid.org/0000-0002-8950-7646</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Oleg Fedorov</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Oleg Fedorov</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium & Target Discovery Institute, University of Oxford, Oxford OX37DQ and OX37FZ, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Oleg++Fedorov">Oleg Fedorov</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Manfred Jung</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Manfred Jung</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Pharmaceutical Sciences, University of Freiburg, Freiburg 79104, Germany</div><div class="loa-info-affiliations-info">German Cancer Research Centre (DKFZ), Heidelberg 69120, Germany</div><div class="loa-info-affiliations-info">German Cancer Consortium (DKTK), Freiburg 79106, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Manfred++Jung">Manfred Jung</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6361-7716" title="Orcid link">http://orcid.org/0000-0002-6361-7716</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Gillian Farnie</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Gillian Farnie</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium & Target Discovery Institute, University of Oxford, Oxford OX37DQ and OX37FZ, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Gillian++Farnie">Gillian Farnie</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jing Liu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jing Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Mount Sinai Center for Therapeutics Discovery, Departments of Pharmacological Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jing++Liu">Jing Liu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Udo Oppermann</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Udo Oppermann</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, Botnar Research Center, NIHR Oxford BRU, University of Oxford, Oxford OX37LD, U.K.</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#364352591859464653445b5758587645515518594e18575518435d"><span class="__cf_email__" data-cfemail="d2a7b6bdfcbda2a2b7a0bfb3bcbc92a1b5b1fcbdaafcb3b1fca7b9">[email protected]</span></a> (U.O.).</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Udo++Oppermann">Udo Oppermann</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9984-5342" title="Orcid link">http://orcid.org/0000-0001-9984-5342</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Roland Schüle</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Roland Schüle</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Urology and Center for Clinical Research, University Freiburg Medical Center, Freiburg 79106, Germany</div><div class="loa-info-affiliations-info">BIOSS Centre of Biological Signalling Studies, University of Freiburg, Freiburg 79106, Germany</div><div class="loa-info-affiliations-info">German Cancer Consortium (DKTK), Freiburg 79106, Germany</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#95e7faf9f4fbf1bbe6f6fde0f0f9f0d5e0fbfcfef9fcfbfcfeb8f3e7f0fcf7e0e7f2bbf1f0"><span class="__cf_email__" data-cfemail="32405d5e535c561c41515a47575e5772475c5b595e5b5c5b591f5440575b504740551c5657">[email protected]</span></a> (R.S.).</span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Roland++Sch%C3%BCle">Roland Schüle</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Jian Jin</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jian Jin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Mount Sinai Center for Therapeutics Discovery, Departments of Pharmacological Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#9af0f3fbf4b4f0f3f4daf7e9e9f7b4fffeef"><span class="__cf_email__" data-cfemail="c0aaa9a1aeeeaaa9ae80adb3b3adeea5a4b5">[email protected]</span></a> (J.J.).</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jian++Jin">Jian Jin</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2387-3862" title="Orcid link">http://orcid.org/0000-0002-2387-3862</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00522&amp;href=/doi/10.1021%2Facs.jmedchem.9b00522" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 20</span><span class="cit-pageRange">, 8996–9007</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">July 1, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>25 March 2019</li><li><span class="item_label"><b>Published</b> online</span>1 July 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 24 October 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b00522" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00522</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D8996%26pageCount%3D12%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DYan%2BXiong%252C%2BHolger%2BGreschik%252C%2BCatrine%2BJohansson%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D62%26issueNum%3D20%26contentID%3Dacs.jmedchem.9b00522%26title%3DDiscovery%2Bof%2Ba%2BPotent%2Band%2BSelective%2BFragment-like%2BInhibitor%2Bof%2BMethyllysine%2BReader%2BProtein%2BSpindlin%2B1%2B%2528SPIN1%2529%26numPages%3D12%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D9007%26publicationDate%3DOctober%2B2019">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b00522"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1942</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">10</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b00522" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of a Potent and Selective Fragment-like Inhibitor of Methyllysine Reader Protein Spindlin 1 (SPIN1)&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Yan&quot;,&quot;last_name&quot;:&quot;Xiong&quot;},{&quot;first_name&quot;:&quot;Holger&quot;,&quot;last_name&quot;:&quot;Greschik&quot;},{&quot;first_name&quot;:&quot;Catrine&quot;,&quot;last_name&quot;:&quot;Johansson&quot;},{&quot;first_name&quot;:&quot;Ludwig&quot;,&quot;last_name&quot;:&quot;Seifert&quot;},{&quot;first_name&quot;:&quot;Johannes&quot;,&quot;last_name&quot;:&quot;Bacher&quot;},{&quot;first_name&quot;:&quot;Kwang-su&quot;,&quot;last_name&quot;:&quot;Park&quot;},{&quot;first_name&quot;:&quot;Nicolas&quot;,&quot;last_name&quot;:&quot;Babault&quot;},{&quot;first_name&quot;:&quot;Michael&quot;,&quot;last_name&quot;:&quot;Martini&quot;},{&quot;first_name&quot;:&quot;Vincent&quot;,&quot;last_name&quot;:&quot;Fagan&quot;},{&quot;first_name&quot;:&quot;Fengling&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Irene&quot;,&quot;last_name&quot;:&quot;Chau&quot;},{&quot;first_name&quot;:&quot;Thomas&quot;,&quot;last_name&quot;:&quot;Christott&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;Dilworth&quot;},{&quot;first_name&quot;:&quot;Dalia&quot;,&quot;last_name&quot;:&quot;Barsyte-Lovejoy&quot;},{&quot;first_name&quot;:&quot;Masoud&quot;,&quot;last_name&quot;:&quot;Vedadi&quot;},{&quot;first_name&quot;:&quot;Cheryl&quot;,&quot;last_name&quot;:&quot;H. Arrowsmith&quot;},{&quot;first_name&quot;:&quot;Paul&quot;,&quot;last_name&quot;:&quot;Brennan&quot;},{&quot;first_name&quot;:&quot;Oleg&quot;,&quot;last_name&quot;:&quot;Fedorov&quot;},{&quot;first_name&quot;:&quot;Manfred&quot;,&quot;last_name&quot;:&quot;Jung&quot;},{&quot;first_name&quot;:&quot;Gillian&quot;,&quot;last_name&quot;:&quot;Farnie&quot;},{&quot;first_name&quot;:&quot;Jing&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Udo&quot;,&quot;last_name&quot;:&quot;Oppermann&quot;},{&quot;first_name&quot;:&quot;Roland&quot;,&quot;last_name&quot;:&quot;Schüle&quot;},{&quot;first_name&quot;:&quot;Jian&quot;,&quot;last_name&quot;:&quot;Jin&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;07&quot;,&quot;day&quot;:&quot;01&quot;,&quot;issue&quot;:&quot;20&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;8996-9007&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b00522&quot;},&quot;abstract&quot;:&quot;By screening an epigenetic compound library, we identified that UNC0638, a highly potent inhibitor of the histone methyltransferases G9a and GLP, was a weak inhibitor of SPIN1 (spindlin 1), a methyllysine reader protein. Our optimization of this weak hit resulted in the discovery of a potent, selective, and cell-active SPIN1 inhibitor, compound 3 (MS31). Compound 3 potently inhibited binding of trimethyllysine-containing peptides to SPIN1, displayed high binding affinity, was highly selective for SPIN1 over other epigenetic readers and writers, directly engaged SPIN1 in cells, and was not toxic to nontumorigenic cells. The crystal structure of the SPIN1–compound 3 complex indicated that it selectively binds tudor domain II of SPIN1. We also designed a structurally similar but inactive compound 4 (MS31N) as a negative control. Our results have demonstrated for the first time that potent, selective, and cell-active fragment-like inhibitors can be generated by targeting a single tudor domain.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00522&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00522" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00522&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00522" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00522&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00522" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00522&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00522&amp;href=/doi/10.1021/acs.jmedchem.9b00522" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b00522" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b00522" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (2 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00522&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b00522%26sid%3Dliteratum%253Aachs%26pmid%3D31260300%26genre%3Darticle%26aulast%3DXiong%26date%3D2019%26atitle%3DDiscovery%2Bof%2Ba%2BPotent%2Band%2BSelective%2BFragment-like%2BInhibitor%2Bof%2BMethyllysine%2BReader%2BProtein%2BSpindlin%2B1%2B%2528SPIN1%2529%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D20%26spage%3D8996%26epage%3D9007%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=290860" title="Solution chemistry">Solution chemistry</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=291255" title="Ultraperformance liquid chromatography">Ultraperformance liquid chromatography</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/20" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/jmcmar.2019.62.issue-20/20191024/jmcmar.2019.62.issue-20.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/medium/jm-2019-00522j_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/large/jm-2019-00522j_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00522&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">By screening an epigenetic compound library, we identified that UNC0638, a highly potent inhibitor of the histone methyltransferases G9a and GLP, was a weak inhibitor of SPIN1 (spindlin 1), a methyllysine reader protein. Our optimization of this weak hit resulted in the discovery of a potent, selective, and cell-active SPIN1 inhibitor, compound <b>3</b> (MS31). Compound <b>3</b> potently inhibited binding of trimethyllysine-containing peptides to SPIN1, displayed high binding affinity, was highly selective for SPIN1 over other epigenetic readers and writers, directly engaged SPIN1 in cells, and was not toxic to nontumorigenic cells. The crystal structure of the SPIN1–compound <b>3</b> complex indicated that it selectively binds tudor domain II of SPIN1. We also designed a structurally similar but inactive compound <b>4</b> (MS31N) as a negative control. Our results have demonstrated for the first time that potent, selective, and cell-active fragment-like inhibitors can be generated by targeting a single tudor domain.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_17016" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_17016" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Epigenetic regulation plays an important role in gene expression and transcription, which are critical for a variety of cellular processes. Epigenetic modifications can be divided into two main categories: DNA methylation and histone modifications.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Histone post-translational modifications (PTMs)<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> require three types of proteins: the enzymes that create the modifications (the “writers”),<a onclick="showRef(event, 'ref3 ref4 ref5 ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref3 ref4 ref5 ref6 ref7">(3−7)</a> the enzymes that remove the modifications (the “erasers”),<a onclick="showRef(event, 'ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref7 ref8">(7,8)</a> and the proteins that recognize the modifications (the “readers”).<a onclick="showRef(event, 'ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref9 ref10">(9,10)</a> Growing evidence suggests that reader proteins are implicated in a number of human diseases including cancer.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> Therefore, reader proteins are increasingly being pursued as potential therapeutic targets.<a onclick="showRef(event, 'ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref12 ref13">(12,13)</a> However, unlike the significant progress made in the discovery of small-molecule inhibitors of histone methyltransferases (MTs), bromodomain-containing proteins (which recognize acetylated lysine residues), and histone demethylases,<a onclick="showRef(event, 'ref5 ref6 ref7 ref14 ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref5 ref6 ref7 ref14 ref15 ref16">(5−7,14−16)</a> only a very limited number of small-molecule inhibitors targeting methyllysine reader proteins have been reported, including UNC1215 (a potent and selective L3MBTL3 inhibitor), UNC3866 (a potent and selective peptide-based CBX7/4 inhibitor), EML631 (a selective and cell-active SPIN1 inhibitor with 685 Da molecule weight), and A366 (a nonselective SPIN1 inhibitor) (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).<a onclick="showRef(event, 'ref17 ref18 ref19 ref20 ref21 ref22 ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref17 ref18 ref19 ref20 ref21 ref22 ref23 ref24">(17−24)</a> Furthermore, it is quite challenging to achieve sufficient potency and selectivity by targeting a single methyllysine reader domain such as a tudor domain or malignant brain tumor (MBT) domain as several reported potents, and selective inhibitors of methyllysine reader proteins such as UNC1215 and EML631 achieved their potency and selectivity by simultaneously targeting two methyllysine reader domains.<a onclick="showRef(event, 'ref21 ref24'); return false;" href="javascript:void(0);" class="ref ref21 ref24">(21,24)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/medium/jm-2019-00522j_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/large/jm-2019-00522j_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Representative inhibitors of methyllysine reader proteins.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/large/jm-2019-00522j_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00522&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">SPIN1 (spindlin 1) is a chromatin reader protein, which recognizes trimethylated histone H3 lysine 4 (H3K4me3) through the protein–protein interaction at the defined “aromatic cage” in tudor domain II, and the interaction between asymmetrically dimethylated histone H3 arginine 8 (H3R8me2a) and tudor domain I further increases the affinity.<a onclick="showRef(event, 'ref25 ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref25 ref26 ref27">(25−27)</a> SPIN1 was found to be overexpressed in several types of malignant tumors, including ovarian cancer, certain types of liver carcinomas, nonsmall-cell lung cancers, and liposarcoma.<a onclick="showRef(event, 'ref28 ref29 ref30 ref31 ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref28 ref29 ref30 ref31 ref32 ref33">(28−33)</a> Upregulation of SPIN1 has been shown to increase cellular proliferation, abnormal mitosis, and chromosomal instability.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> In addition, SPIN1 is involved in several signaling pathways, such as Wnt/TCF-4 and RET signaling pathways.<a onclick="showRef(event, 'ref29 ref35'); return false;" href="javascript:void(0);" class="ref ref29 ref35">(29,35)</a> Therefore, small molecules that selectively disrupt the protein–protein interactions between SPIN1 and its respective binding partners (such as H3K4me3) are valuable chemical tools for investigating biological functions of SPIN1 and assessing the potential of SPIN1 as a therapeutic target. Two small-molecule inhibitors of SPIN1 have been reported to date. We previously reported that A366 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) was a potent but not a selective inhibitor of SPIN1 (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00522/suppl_file/jm9b00522_si_001.pdf" class="ext-link">Figure S1</a>).<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> A class of bivalent compounds represented by EML631 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) that occupy tudor domains I and II of SPIN1 were shown to be selective and cell-active inhibitors of SPIN1 with a relatively weak binding affinity.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> The relatively high molecule weight of EML631 (685 Da) may also render difficulties for further optimization.</div><div class="NLM_p last">Fragment-like inhibitors may possess high intrinsic binding energy and often exhibit high ligand efficiency (LE).<a onclick="showRef(event, 'ref36 ref37'); return false;" href="javascript:void(0);" class="ref ref36 ref37">(36,37)</a> Their low molecular weights leave large room for installing additional functional groups in lead optimization, which could result in drug candidates with higher potency and improved physicochemical properties, such as aqueous solubility, membrane permeability, and oral bioavailability. Therefore, fragment-like inhibitors are valuable starting points for drug development efforts, in addition to being useful tools for chemical biology studies.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> Here, we report our discovery of a fragment-like inhibitor, compound <b>3</b> (MS31) with a molecular weight less than 350, which potently and selectively disrupted the protein–protein interactions between SPIN1 and H3K4me3.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_76234" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_76234" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Discovery of a Fragment-like Hit (Compound <b>2</b>)</h3><div class="NLM_p">Using AlphaLISA and fluorescence polarization (FP)-based biochemical assays, we screened the epigenetic compound library generated by our lab, which includes hundreds of small-molecule modulators of epigenetic writers, readers, and erasers, and identified UNC0638 (compound <b>1</b>, <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>), a highly potent inhibitor of the histone MTs G9a and GLP,<a onclick="showRef(event, 'ref39 ref40'); return false;" href="javascript:void(0);" class="ref ref39 ref40">(39,40)</a> as a weak SPIN1 inhibitor (IC<sub>50</sub> = 3.2 ± 0.3 μM (AlphaLISA) and 7.4 ± 0.4 μM (FP)) (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00522/suppl_file/jm9b00522_si_001.pdf" class="ext-link">Figure S1</a>). To improve potency and selectivity of UNC0638 for SPIN1 over G9a and GLP, we analyzed the cocrystal structure of compound <b>1</b> in the complex with G9a (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3RJW">3RJW</a>)<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> and the cocrystal structure of SPIN1 in the complex with a H3 peptide (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4H75">4H75</a>)<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> and generated a docking model of compound <b>1</b> in the complex with SPIN1 (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00522/suppl_file/jm9b00522_si_001.pdf" class="ext-link">Figure S2</a>). The docking model suggests that the 3-(pyrrolidin-1-yl)propoxyl group of compound <b>1</b> likely mimics H3K4me3, interacting with the aromatic cage of the SPIN1 tudor domain II, while the 4-amino piperidine group on the quinazoline ring extends out of the SPIN1 tudor domain II and the 2-cyclohexyl group on the quinazoline ring does not appear to make any interactions. Based on these observations, we simplified the structure of compound <b>1</b> by replacing the 2,4-disubstituted quinazoline with a disubstituted phenyl ring while keeping the 3-(pyrrolidin-1-yl)propoxyl and methoxy groups. As a result, we discovered a much simpler compound with a molecular weight less than 300, <b>2</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>), as a SPIN1 inhibitor. Importantly, compared with compound <b>1</b>, compound <b>2</b> showed approximately 10-fold improvement in potency for SPIN1 (IC<sub>50</sub> = 338 ± 30 nM (AlphaLISA) and 741 ± 46 nM (FP)) (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A,B). We determined that compound <b>2</b> bound SPIN1 with a <i>K</i><sub>D</sub> value of 390 ± 54 nM using isothermal titration calorimetry (ITC) (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C). To our delight, <b>2</b> did not bind either G9a or GLP using the same ITC experiments (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D and <a href="/doi/suppl/10.1021/acs.jmedchem.9b00522/suppl_file/jm9b00522_si_001.pdf" class="ext-link">S3</a>).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/medium/jm-2019-00522j_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/large/jm-2019-00522j_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Discovery of compound <b>3</b> and its inactive control compound <b>4</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/large/jm-2019-00522j_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00522&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/medium/jm-2019-00522j_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/large/jm-2019-00522j_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Compound <b>2</b> is a selective SPIN 1 inhibitor. Compound <b>2</b> inhibited the interactions between SPIN1 and H3K4me3 in a concentration dependent manner with IC<sub>50</sub> values of (A) 338 ± 30 nM in AlphaLISA assay (<i>n</i> = 3) and (B) 741 ± 46 nM in FP assay (<i>n</i> = 3). (C) Compound <b>2</b> bound SPIN1 with a <i>K</i><sub>D</sub> value of 390 ± 54 nM in ITC assay (<i>n</i> = 3). (D) Compound <b>2</b> did not bind G9a.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/large/jm-2019-00522j_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00522&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Discovery of Compound <b>3</b></h3><div class="NLM_p">To improve potency of compound <b>2</b>, we replaced the pyrrolidinyl group with an isoindolinyl group, trying to achieve some π–π stacking interactions in the aromatic cage of the SPIN1 tudor domain II. As what we expected, compared with <b>2</b>, the resulting compound <b>3</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>) showed >4-fold higher potency for SPIN1 (IC<sub>50</sub> = 77 ± 3 nM (AlphaLISA) and 243 ± 10 nM (FP)) (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A), while maintaining a low molecular weight of 341. The potency of compound <b>3</b> is comparable to that of A366 (IC<sub>50</sub> = 72 ± 2 nM (AlphaLISA) and 207 ± 10 nM (FP)) (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00522/suppl_file/jm9b00522_si_001.pdf" class="ext-link">Figure S1</a>), a G9a/GLP inhibitor<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> whose SPIN1 activity was previously reported by us.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> We next assessed the binding affinity of compound <b>3</b> to SPIN1 using ITC and found that it bound SPIN1 with a <i>K</i><sub>D</sub> value of 91 ± 4 nM (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B left panel), which is consistent with its potency in biochemical assays. The molar ratio of 1 was observed, suggesting that compound <b>3</b> likely bound only one out of three tudor domains of SPIN1. Importantly, similar to compound <b>2</b>, compound <b>3</b> did not bind either G9a or GLP in ITC experiments (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C). In addition, compound <b>3</b> showed moderate binding affinities (ranging from 170 nM to 1.7 μM) to other closely related SPIN subfamily members (SPIN2B, SPIN3, and SPIN4) in ITC experiments (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00522/suppl_file/jm9b00522_si_001.pdf" class="ext-link">Figure S4</a>), suggesting similar domain architectures and binding modes amongst the SPIN subfamily members.<a onclick="showRef(event, 'ref43 ref44 ref45'); return false;" href="javascript:void(0);" class="ref ref43 ref44 ref45">(43−45)</a> To further assess its selectivity, we tested compound <b>3</b> against 33 MTs (including G9a and GLP) and 5 acetyltransferases and found that it did not significantly inhibit these MTs and acetyltransferases at up to 50 μM (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A and <a href="/doi/suppl/10.1021/acs.jmedchem.9b00522/suppl_file/jm9b00522_si_001.pdf" class="ext-link">Table S1</a>). In addition, we assessed selectivity of compound <b>3</b> against 20 methyllysine, methylarginine, and acetyllysine reader proteins using a thermal shift assay and found that compound <b>3</b> did not induce a significant thermal shift for any of the 20 reader proteins at 20 or 200 μM (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B and <a href="/doi/suppl/10.1021/acs.jmedchem.9b00522/suppl_file/jm9b00522_si_001.pdf" class="ext-link">Table S2</a>). Thus, compound <b>3</b> is selective over a broad range of epigenetic proteins.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/medium/jm-2019-00522j_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/large/jm-2019-00522j_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Compound <b>3</b> is a potent SPIN1 inhibitor. (A) Compound <b>3</b>, but not compound <b>4</b>, potently inhibited the interactions between SPIN1 and H3K4me3 with IC<sub>50</sub> values of 77 ± 3 nM in AlphaLISA assay (<i>n</i> = 3) and 243 ± 10 nM in FP assay (<i>n</i> = 3). (B) Compound <b>3</b> bound SPIN1 with a <i>K</i><sub>D</sub> value of 91 ± 4 nM (<i>n</i> = 3) while compound <b>4</b> did not bind SPIN1 in ITC experiments. (C) Compound <b>3</b> did not bind either G9a or GLP in ITC experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/large/jm-2019-00522j_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00522&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/medium/jm-2019-00522j_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/large/jm-2019-00522j_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Compound <b>3</b> is a selective SPIN1 inhibitor. (A) Compound <b>3</b> did not significantly inhibit the enzymatic activity of 33 MTs and 5 acetyltransferases at 10 μM (blue) and 50 μM (red). (B) Compound <b>3</b> did not induce significant thermal shifts for 20 methyllysine, methylarginine and acetyllysine reader proteins at 20 μM or 200 μM in thermal shift assays (<i>n</i> = 2).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/large/jm-2019-00522j_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00522&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Determination of the Cocrystal Structure of Compound <b>3</b> in the Complex with SPIN1</h3><div class="NLM_p">We next solved the X-ray crystal structure of SPIN1 in the complex with compound <b>3</b> at 1.6 Å resolution (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QPL">6QPL</a>, <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>, and <a href="/doi/suppl/10.1021/acs.jmedchem.9b00522/suppl_file/jm9b00522_si_001.pdf" class="ext-link">Table S3</a>). As illustrated in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A, compound <b>3</b> only occupied the SPIN1 tudor domain II, which recognizes H3K4me3. As expected, the isoindolinyl group occupied the aromatic cage (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B,C). The protonated amino group in the isoindolinyl group not only formed a hydrogen bond (H-bond) with Y179 but also interacted with Y170, W151, and F141 in the aromatic cage through cation−π interactions (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C). The phenyl ring of the isoindoline group interacted with W151 through π–π stacking, which may explain the increased potency of compound <b>3</b> over compound <b>2</b> (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C). Besides these important interactions formed by the isoindolinyl group, two aminomethylene side chains formed a few critical H-bonds. The amino group of the 3-aminomethylene side chain formed a H-bond with D95 on the loop of the tudor domain I of SPIN1 (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C). The amino group of the 5-aminomethylene side chain formed a direct H-bond with D184 on the α-helix of tudor domain II and two water-mediated H-bonds with M140 and D189. Disrupting these H-bond interactions by switching the 5-aminomethylene group of compound <b>3</b> to an amido group resulted in a structurally similar but inactive compound, <b>4</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>), which was completely inactive in the SPIN1 AlphaLISA and FP assays (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A). In addition, compound <b>4</b> did not show detectable binding affinity to SPIN1 in ITC experiments (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B right panel). Thus, compound <b>4</b> could be used as a negative control in chemical biology studies.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/medium/jm-2019-00522j_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/large/jm-2019-00522j_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Crystal structure of the SPIN1–compound <b>3</b> complex. (A) Compound <b>3</b> (green) occupies the tudor domain II (pink) of SPIN1. (B) Electrostatic potential surface view of the structure, ranging from −6 <i>kT</i>/e (red) to +6 <i>kT</i>/e (blue). (C) Close-up view of the SPIN1–compound <b>3</b> complex structure with key ligand–protein interactions. Yellow dashes, hydrogen bonds. Red balls, water molecules.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/large/jm-2019-00522j_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00522&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Evaluation of Compound <b>3</b> in Cellular Assays</h3><div class="NLM_p">Using a NanoBRET target engagement assay,<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> we next demonstrated that compound <b>3</b> can engage SPIN1 in cells. As illustrated in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A,B, compound <b>3</b>, but not compound <b>4</b>, reduced the interaction between SPIN1 and histone H3 in U2OS cells in a concentration-dependent manner with an IC<sub>50</sub> value of 3.2 ± 0.7 μM. Thus, compound <b>3</b> is cell permeable and can effectively engage SPIN1 in cells. Finally, we assessed off-target toxicity of compound <b>3</b> in two transformed cell lines, C2C12 and 293T, and one normal human primary fibroblasts cell line, HFF-1. We found that compound <b>3</b> was not toxic to these cells at up to 30 μM (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>C). Thus, compound <b>3</b> is a useful tool compound for cellular studies.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/medium/jm-2019-00522j_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/large/jm-2019-00522j_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Compound <b>3</b> binds SPIN1 in cells and shows no cytotoxicity. (A) Compound <b>3</b> disrupted the interaction between SPIN1 and histone H3 (IC<sub>50</sub> = 3.2 ± 0.7 μM (<i>n</i> = 4)) in U2OS cells in a NanoBRET assay. (B) Compound <b>4</b> was inactive in the NanoBRET assay (<i>n</i> = 3). (C) Compound <b>3</b> was not toxic to C2C12 and 293T cell lines or primary fibroblasts HFF1, at up to 30 μM (<i>n</i> = 3). Indicated cell lines were cultured in the presence of indicated concentrations of compound <b>3</b> for 6 days. Cell viability was measured using Alamar blue.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/large/jm-2019-00522j_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00522&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Chemical Synthesis</h3><div class="NLM_p">Compounds <b>2</b>, <b>3</b>, and <b>4</b> were synthesized using the synthetic routes shown in <a class="ref internalNav" href="#sch1" aria-label="Schemes 1">Schemes 1</a>–<named-content content-type="anchor" rid="sch2" type="simple"></named-content><a class="ref internalNav" href="#sch3" aria-label="3">3</a>.</div><figure id="sch1" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/medium/jm-2019-00522j_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/large/jm-2019-00522j_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compound <b>2</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/large/jm-2019-00522j_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00522&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagent and conditions: (a) BnBr, K<sub>2</sub>CO<sub>3</sub>, DMF, rt, 12 h, 90%; (b) CuCN, NMP, 170 °C, 18 h, 40%; (c) H<sub>2</sub> (1 atm), 10% Pd/C, EtOAc, rt, 5 h; (d) 1-bromo-3-chloropropane, DMF, rt, 12 h, 60% in 2 steps; (e) pyrrolidine, K<sub>2</sub>CO<sub>3</sub>, KI, DMF, 60 °C, 12 h, 80%; (f) H<sub>2</sub> (1 atm), Raney Ni, NH<sub>4</sub>OH, MeOH, rt, 12 h, 70%.</p></p></figure><figure id="sch2" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/medium/jm-2019-00522j_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/large/jm-2019-00522j_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compound <b>3</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/large/jm-2019-00522j_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00522&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagent and conditions: (a) isoindoline, K<sub>2</sub>CO<sub>3</sub>, KI, DMF, 60 °C, 12 h, 60%; (b) H<sub>2</sub> (1 atm), Raney Ni, NH<sub>4</sub>OH, MeOH, rt, 12 h, 65%.</p></p></figure><figure id="sch3" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/medium/jm-2019-00522j_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/large/jm-2019-00522j_0010.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compound <b>4</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/large/jm-2019-00522j_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00522&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagent and conditions: (a) BnBr, K<sub>2</sub>CO<sub>3</sub>, DMF, rt, 12 h, 91%; (b) CuCN, NMP, 170 °C, 18 h, 54%; (c) NaOH, MeOH, H<sub>2</sub>O, rt, 2 h. (d) (COCl)<sub>2</sub>, DMF, CH<sub>2</sub>Cl<sub>2</sub>, rt, 1 h; NH<sub>4</sub>OH, 0 °C, 1 h, 41% in 3 steps; (e) H<sub>2</sub> (1 atm), 10% Pd/C, EtOAc, rt, 5 h; (f) 1-bromo-3-chloropropane, DMF, rt, 12 h, 55% in 2 steps; (g) isoindoline, K<sub>2</sub>CO<sub>3</sub>, KI, DMF, 60 °C, 12 h, 61%; (h) H<sub>2</sub> (1 atm), Raney Ni, NH<sub>4</sub>OH, MeOH, rt, 12 h, 72%.</p></p></figure><div class="NLM_p">The preparation of compound <b>2</b> was started from 3,5-dibromo-2-methoxyphenol (<b>5</b>).<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> Protection of the phenol group using benzyl bromide under basic conditions provided benzyl ether intermediate <b>6</b>. Next, the dibromo groups on the phenyl ring were converted to dicyano groups under Rosenmund–von Braun reaction conditions, resulting in intermediate <b>7</b>. Benzyl ether deprotection under palladium-catalyzed hydrogenation conditions provided phenol <b>8</b>, which was subsequently converted to 3-chloropropyl ether <b>9</b>. Substitution of the chloro group with pyrrolidine in the presence of potassium iodide yielded intermediate <b>10</b>. Lastly, reduction of the dicyano groups under Raney nickel-mediated hydrogenation provided compound <b>2</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>).</div><div class="NLM_p">Compound <b>3</b> was synthesized following similar procedures for preparation of compound <b>2</b>. Briefly, isoindoline substitution of the chloro group of intermediate <b>9</b> yielded compound <b>11</b>. The dicyano groups were subsequently reduced to diaminomethylene groups to afford compound <b>3</b> (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>).</div><div class="NLM_p last">Compound <b>4</b> was prepared from commercially available compound <b>12</b>. The carboxylic acid and phenolic hydroxyl groups of <b>12</b> were simultaneously protected as benzyl ester and benzyl ether groups, respectively, to provide intermediate <b>13</b>. The bromo group of <b>13</b> was substituted with the cyano group using the Rosenmund–von Braun reaction to yield intermediate <b>14</b>. After the hydrolysis of the benzyl ester under basic conditions, the resulting carboxylic acid was activated as acid chloride, which was subsequently converted to amide <b>16</b> in the presence of ammonium hydroxide aqueous solution. The benzyl ether was deprotected to provide intermediate <b>17</b> with a free phenol group, which was converted to the 3-isoindolinylpropyl ether product <b>4</b> in three steps (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>), using the similar reaction sequences for the preparation of compounds <b>2</b> and <b>3</b>.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_18130" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_18130" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In summary, we optimized compound <b>1</b>, a weak SPIN1 but highly potent G9a/GLP inhibitor, into compound <b>3</b>, a potent and selective fragment-like inhibitor of SPIN1. Compound <b>3</b> displayed high potency in SPIN1 biochemical assays (IC<sub>50</sub> = 77 nM (AlphaLISA) and 243 nM (FP)) and high binding affinity to SPIN1 (<i>K</i><sub>D</sub> = 91 nM) by ITC. Compound <b>3</b> was completely inactive against G9a and GLP and selective for SPIN1 over a broad range of epigenetic proteins. We also obtained an X-ray crystal structure of SPIN1 in the complex with compound <b>3</b>, which confirmed that compound <b>3</b> occupied tudor domain II of SPIN1. Based on the cocrystal structure, we designed compound <b>4</b>, a close analogue of compound <b>3</b> as a negative control, which was indeed inactive in SPIN1 biochemical and biophysical assays. We demonstrated that compound <b>3</b>, but not the negative control compound <b>4</b>, engaged SPIN1 in cells using a NanoBRET assay, and compound <b>3</b> was not toxic in the nontumorigenic cells evaluated. Thus, compound <b>3</b> and compound <b>4</b> are a pair of useful tool compounds for investigating biological functions and disease associations of SPIN1. Remarkably, compound <b>3</b> achieved high potency and selectivity for SPIN1 by targeting a single methyllysine reader domain. These results have demonstrated that it is feasible to generate potent, selective, and cell-active inhibitors by targeting a single tudor domain and paved the way for discovering improved inhibitors of methyllysine reader proteins.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_17652" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_17652" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Chemistry General Procedures</h3><div class="NLM_p last">High-performance liquid chromatography (HPLC) spectra for compounds were acquired using an Agilent 1200 Series system with a DAD detector. Chromatography was performed on a 2.1 × 150 mm Zorbax 300SB-C<sub>18</sub> 5 μm column with water containing 0.1% formic acid as solvent A and acetonitrile containing 0.1% formic acid as solvent B at a flow rate of 0.4 mL/min. The gradient program was as follows: 1% B (0–1 min), 1–99% B (1–4 min), and 99% B (4–8 min). Ultra-performance liquid chromatography (UPLC) spectra for compounds were acquired using a Waters Acquity I-Class UPLC system with a PDA detector. Chromatography was performed on a 2.1 × 30 mm ACQUITY UPLC BEH C<sub>18</sub> 1.7 μm column with water containing 3% acetonitrile, 0.1% formic acid as solvent A and acetonitrile containing 0.1% formic acid as solvent B at a flow rate of 0.8 mL/min. The gradient program was as follows: 1–99% B (1–1.5 min), and 99–1% B (1.5–2.5 min). High-resolution mass spectra (HRMS) data were acquired in the positive ion mode using Agilent G1969A API-TOF with an electrospray ionization (ESI) source. Nuclear magnetic resonance (NMR) spectra were acquired on either a Bruker DRX-600 spectrometer (600 MHz <sup>1</sup>H) or a Bruker DXI 800 MHz spectrometer (800 MHz <sup>1</sup>H, 200 MHz <sup>13</sup>C). Chemical shifts are reported in ppm (δ). Preparative HPLC was performed on Agilent Prep 1200 series with an UV detector set to 254 nm. Samples were injected into a Phenomenex Luna 75 × 30 mm, 5 μm, C<sub>18</sub> column at room temperature. The flow rate was 40 mL/min. A linear gradient was used with 10% (or 50%) of MeOH (A) in H<sub>2</sub>O [with 0.1% trifluoroacetic acid (TFA)] (B) to 100% of MeOH (A). HPLC and UPLC were used to establish the purity of target compounds. All final compounds had >95% purity using the HPLC and UPLC methods described above. Compounds <b>2</b> and <b>3</b> were tested in biological assays in their HCl salt forms and compound <b>4</b> in its CF<sub>3</sub>CO<sub>2</sub>H salt form.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> (4-Methoxy-5-(3-(pyrrolidin-1-yl)propoxy)-1,3-phenylene)dimethanamine (<b>2</b>)</h3><div class="NLM_p last">Compound <b>2</b> was synthesized from the intermediate <b>6</b> following the procedures described below. To a solution of <b>10</b> (127 mg, 0.32 mmol) in MeOH (2 mL) was added Raney Ni (50 mg), followed by aqueous ammonia solution (0.1 mL). The mixture was stirred under a hydrogen atmosphere (1 atm) for 12 h, before the insoluble solid was filtered. The filtrate was collected and concentrated. The resulting residue was purified by prep-HPLC to yield the title compound as brown oil (90 mg, 70% yield). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD): δ 7.38 (s, 1H), 7.16 (s, 1H), 4.28 (t, <i>J</i> = 5.5 Hz, 2H), 4.16 (s, 2H), 4.14 (s, 2H), 4.01 (s, 3H), 3.82–3.69 (m, 2H), 3.52–3.44 (m, 2H), 3.23–3.13 (m, 2H), 2.41–2.32 (m, 2H), 2.26–2.16 (m, 2H), 2.14–2.04 (m, 2H). HPLC >95%, <i>t</i><sub>R</sub> = 1.24 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>16</sub>H<sub>28</sub>N<sub>3</sub>O<sub>2</sub>, 294.2176; found, 294.2150.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> (5-(3-(Isoindolin-2-yl)propoxy)-4-methoxy-1,3-phenylene)dimethanamine (<b>3</b>)</h3><div class="NLM_p last">Compound <b>3</b> was synthesized from the intermediate of <b>11</b> according to the procedures for the preparation of <b>2</b> as brown oil (65% yield). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD): δ 7.49–7.42 (m, 4H), 7.40 (s, 1H), 7.15 (s, 1H), 5.02 (d, <i>J</i> = 13.8 Hz, 2H), 4.68 (d, <i>J</i> = 13.9 Hz, 2H), 4.33 (t, <i>J</i> = 5.8 Hz, 2H), 4.16 (s, 2H), 4.14 (s, 2H), 4.01 (s, 3H), 3.74 (t, <i>J</i> = 7.8 Hz, 2H), 2.50–2.38 (m, 2H). <sup>13</sup>C NMR (200 MHz, MeOD): δ 151.60, 148.24, 133.57, 129.30, 128.79, 126.98, 122.93, 122.60, 115.77, 65.82, 60.35, 58.51, 52.32, 42.47, 38.30, 25.46. HPLC >95%, <i>t</i><sub>R</sub> = 1.31 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>28</sub>N<sub>3</sub>O<sub>2</sub>, 342.2176; found, 342.2145.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> 3-(Aminomethyl)-5-(3-(isoindolin-2-yl)propoxy)-4-methoxybenzamide (<b>4</b>)</h3><div class="NLM_p last">Compound <b>4</b> was synthesized from intermediate <b>19</b> according to the procedures for the preparation of compound <b>2</b> as brown oil (72% yield). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD): δ 7.67 (s, 1H), 7.59 (s, 1H), 7.48–7.44 (m, 4H), 5.08–4.94 (m, 2H), 4.77–4.58 (m, 2H), 4.32 (t, <i>J</i> = 5.7 Hz, 2H), 4.22 (s, 2H), 3.99 (s, 3H), 3.78 (d, <i>J</i> = 7.8 Hz, 2H), 2.42–2.32 (m, 2H). <sup>13</sup>C NMR (200 MHz, MeOD): δ 169.50, 152.05, 148.83, 133.55, 129.93, 128.83, 126.40, 122.60, 121.41, 112.85, 70.05, 58.60, 55.26, 52.59, 38.20, 26.39. UPLC >95%, <i>t</i><sub>R</sub> = 0.66 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>26</sub>N<sub>3</sub>O<sub>3</sub>, 356.1969; found, 356.1994.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> 1-(Benzyloxy)-3,5-dibromo-2-methoxybenzene (<b>6</b>)</h3><div class="NLM_p last">To a solution of <b>5</b><a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> (1.71 g, 6.1 mmol) in dimethylformamide (DMF) (10 mL) was added K<sub>2</sub>CO<sub>3</sub> (1.60 g, 11.6 mmol), followed by benzyl bromide (0.89 mL, 7.5 mmol). The resulting suspension was stirred at rt for 12 h, before water (10 mL) was added to quench the reaction. The mixture was extracted with ether (3 × 20 mL). The combined organic phase was dried with anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The resulting residue was purified by flash chromatography to yield the title compound as the white solid (2.05 g, 90% yield). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.49–7.36 (m, 5H), 7.33 (s, 1H), 7.08 (s, 1H), 5.12 (s, 2H), 3.89 (s, 3H).</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> 5-(Benzyloxy)-4-methoxyisophthalonitrile (<b>7</b>)</h3><div class="NLM_p last">To a solution of <b>6</b> (2.05 g, 5.5 mmol) in N-Methyl-2-pyrrolidone (NMP) (10 mL) was added CuCN (1.08 g, 12.1 mmol). The mixture was heated in a sealed tube for 18 h. After cooling to room temperature, aqueous ammonia solution (2 mL) was added. The reaction was extracted with ether (3 × 20 mL). The combined organic phase was dried with anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. The residue was purified by flash chromatography to yield the title compound as colorless oil (580 mg, 40% yield). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.49 (s, 1H), 7.48–7.40 (m, 5H), 7.37 (s, 1H), 5.18 (s, 2H), 4.18 (s, 3H). UPLC >95%, <i>t</i><sub>R</sub> = 1.62 min.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> 5-(3-Chloropropoxy)-4-methoxyisophthalonitrile (<b>9</b>)</h3><div class="NLM_p last">To a solution of <b>7</b> (580 mg, 2.2 mmol) in ethyl acetate (10 mL) was added 10% palladium on carbon (50 mg). The mixture was stirred in a hydrogen atmosphere (1 atm) for 5 h, before being filtered. The filtrate was concentrated. The resulting residue was dissolved in DMF (5 mL). To the resulting solution was added K<sub>2</sub>CO<sub>3</sub> (455 mg, 3.3 mmol), followed by 1-bromo-3-chloropropane (0.43 mL, 4.4 mmol). The mixture was stirred at room temperature for 12 h, before water was added to quench the reaction. The mixture was extracted with ether (3 × 20 mL). The combined organic phase was combined, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and concentrated. The resulting residue was purified by flash chromatography to yield the title compound as colorless oil (335 mg, 60% yield). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.50 (s, 1H), 7.37 (s, 1H), 4.24 (t, <i>J</i> = 5.8 Hz, 2H), 4.16 (s, 3H), 3.78 (t, <i>J</i> = 6.1 Hz, 2H), 2.39–2.32 (m, 2H). UPLC >95%, <i>t</i><sub>R</sub> = 1.47 min.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> 4-Methoxy-5-(3-(pyrrolidin-1-yl)propoxy)isophthalonitrile (<b>10</b>)</h3><div class="NLM_p last">To a solution of <b>9</b> (100 mg, 0.4 mmol) in DMF (2 mL) was added pyrrolidine (84 μL, 1 mmol), K<sub>2</sub>CO<sub>3</sub> (140 mg, 1 mmol), and KI (15 mg, 0.1 mmol). The resulting suspension was heated at 60 °C for 12 h. After cooling to room temperature, the reaction was quenched with water (5 mL) and extracted with CH<sub>2</sub>Cl<sub>2</sub> (3 × 5 mL). The combined organic phase was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. The resulting residue was purified by pre-HPLC to yield the title compound as brown oil (127 mg, 80% yield). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD): δ 7.74 (s, 1H), 7.71 (s, 1H), 4.26 (t, <i>J</i> = 5.7 Hz, 2H), 4.16 (s, 3H), 3.82–3.69 (m, 2H), 3.49–3.41 (m, 2H), 3.22–3.11 (m, 2H), 2.40–2.29 (m, 2H), 2.28–2.17 (m, 2H), 2.14–2.00 (m, 2H). UPLC >95%, <i>t</i><sub>R</sub> = 0.95 min.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> 5-(3-(Isoindolin-2-yl)propoxy)-4-methoxyisophthalonitrile (<b>11</b>)</h3><div class="NLM_p last">Compound <b>11</b> was synthesized from intermediate <b>9</b> according to the procedures for the preparation of <b>10</b> as brown oil (60% yield). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD): δ 7.72 (dd, <i>J</i> = 3.8, 1.7 Hz, 2H), 7.47–7.44 (m, 4H), 5.00 (br, 2H), 4.66 (br, 2H), 4.31 (t, <i>J</i> = 5.8 Hz, 2H), 4.14 (s, 3H), 3.72–3.69 (m, 2H), 2.49–2.37 (m, 2H). HPLC >95%, <i>t</i><sub>R</sub> = 1.24 min.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Benzyl 3-(Benzyloxy)-5-bromo-4-methoxybenzoate (<b>13</b>)</h3><div class="NLM_p last">To a solution of 3-bromo-5-hydroxy-4-methoxybenzoic acid (1.1 g, 4.4 mmol) in DMF (12 mL) was added benzyl bromide (1.1 mL, 9.2 mmol) and K<sub>2</sub>CO<sub>3</sub> (1.6 g, 11.3 mmol). The mixture was stirred at room temperature for 12 h, before being quenched with water. The mixture was extracted with ether (3 × 30 mL). The combined organic phase was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. The resulting residue was purified by flash chromatography to yield the title compound as the white solid (1.7 g, 91% yield).</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Benzyl 3-(Benzyloxy)-5-cyano-4-methoxybenzoate (<b>14</b>)</h3><div class="NLM_p last">Compound <b>14</b> was synthesized from intermediate <b>13</b> according to the procedures for the preparation of <b>7</b> as the brown solid (54% yield). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.92 (d, <i>J</i> = 1.8 Hz, 1H), 7.60 (d, <i>J</i> = 1.8 Hz, 1H), 7.57–7.54 (m, 2H), 7.50–7.46 (m, 2H), 7.46–7.34 (m, 6H), 5.39 (s, 2H), 5.15 (s, 2H), 3.94 (s, 3H). UPLC >95%, <i>t</i><sub>R</sub> = 1.74 min.</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> 3-(Benzyloxy)-5-cyano-4-methoxybenzamide (<b>16</b>)</h3><div class="NLM_p last">To a solution of <b>14</b> (460 mg, 1.2 mmol) in MeOH (5 mL) was added aqueous NaOH (1.5 mL, 1 N, 1.5 mmol) solution. The solution was stirred at room temperature for 2 h, before being acidified with aqueous HCl (2 mL, 1 N, 2 mmol). The resulting suspension was filtered. The collected solid was dried and used for the next step without further purification. The crude acid (370 mg, 1.3 mmol) was dissolved in CH<sub>2</sub>Cl<sub>2</sub> (5 mL). To the resulting solution was added (COCl)<sub>2</sub> (0.22 mL, 2.6 mmol) and DMF (several drops). The solution was stirred at room temperature for 1 h, before being concentrated. The resulting residue was dissolved in CH<sub>2</sub>Cl<sub>2</sub> (5 mL). To the solution was added aqueous ammonia solution (1 mL, 28%, 14.8 mmol) at 0 °C. The mixture was stirred for 1 h at 0 °C before being poured into a separatory funnel. The organic phase was collected. The aqueous phase was extracted with CH<sub>2</sub>Cl<sub>2</sub> (2 × 5 mL). The organic phase was combined, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. The resulting residue was purified by flash chromatography to yield the title compound as the brown solid (140 mg, 41% in 2 steps). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.74 (d, <i>J</i> = 1.9 Hz, 1H), 7.67 (d, <i>J</i> = 1.9 Hz, 1H), 7.45–7.41 (m, 2H), 7.35–7.25 (m, 3H), 5.30 (s, 2H), 4.69 (s, 2H), 3.96 (s, 3H). UPLC >95%, <i>t</i><sub>R</sub> = 1.29 min.</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> 3-(3-Chloropropoxy)-5-cyano-4-methoxybenzamide (<b>18</b>)</h3><div class="NLM_p last">Compound <b>18</b> was synthesized from intermediate <b>16</b> according to the procedures for the preparation of <b>9</b> as the off-white solid (55% yield in 2 steps). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD): δ 7.74 (s, 1H), 7.69 (s, 1H), 4.41 (t, <i>J</i> = 5.6 Hz, 2H), 4.26 (s, 2H), 3.94 (s, 3H), 3.83 (t, <i>J</i> = 6.2 Hz, 2H), 2.26–2.17 (m, 2H). UPLC >95%, <i>t</i><sub>R</sub> = 1.20 min.</div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> 3-Cyano-5-(3-(isoindolin-2-yl)propoxy)-4-methoxybenzamide (<b>19</b>)</h3><div class="NLM_p last">Compound <b>19</b> was synthesized from intermediate <b>18</b> according to the procedures for the preparation of <b>11</b> as brown oil (61% yield). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD): δ 7.86 (d, <i>J</i> = 2.0 Hz, 1H), 7.79 (d, <i>J</i> = 1.9 Hz, 1H), 7.56–7.35 (m, 4H), 5.06–4.96 (m, 2H), 4.72–4.65 (m, 2H), 4.46 (t, <i>J</i> = 5.6 Hz, 2H), 4.00 (s, 3H), 3.89–3.79 (m, 2H), 2.43–2.27 (m, 2H). UPLC >95%, <i>t</i><sub>R</sub> = 0.90 min.</div></div><div id="sec4_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> Crystallization and X-ray Cocrystal Structure Determination</h3><div class="NLM_p">SPIN1<sub>Pro49–Ser262</sub> was crystallized with compound <b>3</b> at 4 °C using the sitting drop vapor diffusion method. SPIN1 was incubated with 1.5 mM of compound <b>3</b> using 39 mg/mL protein, and crystals appeared in drops consisting of 75 nL protein compound mixture and 75 nL precipitant consisting of 60% 2-Methyl-2,4-pentanediol (MPD), and 0.1 M SPG (succinate, phosphate, and glycine) buffer pH 6.0. A 1.6 Å resolution dataset was collected on beamline I04 at Diamond Light Source UK. The dataset was processed, scaled, and merged at the Diamond Light Source using Xia2.<a onclick="showRef(event, 'ref48 ref49 ref50 ref51 ref52'); return false;" href="javascript:void(0);" class="ref ref48 ref49 ref50 ref51 ref52">(48−52)</a></div><div class="NLM_p last">The structure was solved by molecular replacement using PHASER and SPIN1 PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6I8L">6I8L</a> as a search model. The structure was refined in PHENIX<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> with electron density map inspections and model improvement in COOT<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> and terminated when there were no significant changes in the <i>R</i><sub>work</sub> and <i>R</i><sub>free</sub> values, and inspection of the electron density map suggested that no further corrections or additions were justified. Statistics for data collection and refinement are shown in <a href="/doi/suppl/10.1021/acs.jmedchem.9b00522/suppl_file/jm9b00522_si_001.pdf" class="ext-link">Table S3</a>. Structural analyses were performed with PyMOL (<a href="http://www.pymol.org" class="extLink">http://www.pymol.org</a>) and the coordinates deposited in PDB with the ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QPL">6QPL</a>.</div></div><div id="sec4_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> SPIN1 FP Assay</h3><div class="NLM_p last">FP experiments were performed as previously described.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> The compounds were measured as a 12-point serial dilution with concentrations ranging from 24 nM to 50 μM, resulting in a final dimethyl sulfoxide (DMSO) concentration of 0.5%. Blank values were measured in six wells on the first and last row of the assay plate. Each concentration as well as positive and negative controls were tested in triplicates. The incubation time was modified to 30 min.</div></div><div id="sec4_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> SPIN1 AlphaLISA Assay</h3><div class="NLM_p">AlphaLISA experiments were performed as previously described.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> The compounds were measured as a 14-point serial dilution with concentrations ranging from 6.1 nM to 50 μM, resulting in a final DMSO concentration of 0.5%. Blank values were measured in seven wells on the first and last row of the assay plate. Each concentration as well as positive and negative controls were done in triplicates. The incubation time after addition of the beads was modified to 90 min.</div><div class="NLM_p last">IC<sub>50</sub> values were calculated using the combined inhibition values of two independent experiments and applying a sigmoidal dose–response fit (variable slope) using GraphPad Prism (version 7).</div></div><div id="sec4_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> G9a and GLP ITC Experiments</h3><div class="NLM_p last">ITC measurements were made at 25 °C on a MicroCal ITC200 Instrument (Malvern Instruments). Coconcentrated G9a–SAM and GLP–SAM (protein/SAM molar ratio of 1:5) were diluted at 35 μM in ITC buffer [50 mM Tris (pH 8.0), 150 mM NaCl] supplemented with 1% DMSO. Small molecule compounds were dissolved in DMSO at 50 mM and diluted to 0.5 mM in ITC buffer with a final DMSO concentration of 1%. Binding constants were calculated by fitting the data using the ITC data analysis module in Origin 7.0 (OriginLab Corp.).</div></div><div id="sec4_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> SPIN1 ITC Experiments</h3><div class="NLM_p">ITC experiments were performed at 20 °C with a MicroCal VP-ITC microcalorimeter (GE Healthcare) by injection of 8 μL of compound solution (100–300 μM) during 16 s into the sample cell containing 10 μM of His–SPIN1(49–262) in ITC assay buffer. DMSO was adapted in a syringe and sample cell to 0.1–0.3%. A total of 33 injections were performed with a spacing of 240 s and a reference power of 10 μcal s<sup>–1</sup>.</div><div class="NLM_p last">Automated baseline assignment and peak integration were performed with NITPIC version 1.0.1.<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> Isotherms were plotted by global analysis of multiple titrations, with the simplex algorithm, with SEDPHAT version 10.58d.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> Isotherms were subsequently validated by two-dimensional error surface projections at <i>P</i> levels of 0.68 and 0.95 in SEDFIT.<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a></div></div><div id="sec4_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> Other SPIN Subfamily ITC Experiments</h3><div class="NLM_p last">Purified protein was diluted to 10–30 μM in ITC buffer (100 mM NaCl, 20 mM Tris, pH 8.0) and then dialyzed against a 1000 times excess of ITC buffer at 4 °C overnight using D-Tube Dialyzer tubes (Merck/Sigma-Aldrich) with a 3.5 kDa molecular weight cut-off. The dialyzed sample was then centrifuged at 14 000 rpm for 10 min in a standard chilled benchtop centrifuge at 4 °C. Protein concentration was then verified via absorption at 280 nm using a NanoDrop ND1000 Spectrophotometer. Compounds were diluted to 200 μM in ITC buffer. Experiments were performed on a NanoITC Standard Volume instrument (960 μL cell volume, 250 μL syringe volume; TA Instruments) using direct titration (protein in cell, compound in syringe) at 20 °C, a stir rate of 350 rpm, and an injection rate of 50 ms/step (∼1.4 μL/s). An initial injection of 3.7 μL followed after 200 s by 30 injections of 7.96 μL spaced at 300 s. Data were analyzed using an independent fit model with the NanoAnalyse software (TA Instruments, version 3.8.0).</div></div><div id="sec4_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> Thermal Shift (<i>T</i><sub>m</sub>) Assay</h3><div class="NLM_p">Compounds were dispensed on white PCR plates using an Echo 550 acoustic liquid dispenser to a final concentration of 20 or 200 μM with two technical replicates per concentration per protein. Protein in <i>T</i><sub>m</sub> shift buffer (20 mM HEPES, 500 mM NaCl, pH 7.5, SYPRO orange dye at 1:1000 dilution from purchased stock) was added at 2 μM final concentration. The lower concentrations were back-filled with DMSO to the same amount dispensed as for the highest concentration, and DMSO-only controls were also included (<i>n</i> = 4 per protein). Experiments were performed on Agilent Mx3005P qPCR machines (reaction volume 20 μL) or Roche Lightcycler 480 (reaction volume 5 μL). The temperature gradient was run from 25 to 95 °C over 25 min. Melting temperatures <i>T</i><sub>m</sub> were estimated as the inflection point of a Boltzmann equation fitted to the fluorescence intensity <i>I</i>(<i>T</i>) from the onset (<i>I</i><sub>onset</sub>) to the peak (<i>I</i><sub>peak</sub>) of intensity where <i>S</i> is the slope of the curve<span class="NLM_disp-formula" id="ueq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/medium/jm-2019-00522j_m001.gif" alt="" id="_i35" /></img></span></div><div class="NLM_p last">Shifts in unfolding (D<i>T</i><sub>m</sub>) were then calculated as the difference between the individual <i>T</i><sub>m</sub> from the mean of the DMSO controls.</div></div><div id="sec4_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> MT and Acetyltransferase Selectivity Assays</h3><div class="NLM_p last">The effect of compound <b>3</b> on activities of 33 MTs and 5 acetyltransferases was assessed using activity assays as previously described.<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a></div></div><div id="sec4_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> NanoBRET Assay</h3><div class="NLM_p last">U20S cells (2.8 × 10<sup>5</sup>) were plated in each well of a 6-well plate. After 6 h cells were cotransfected with C-terminal HaloTag–histone 3.3 (NM_002107) and an N-terminal NanoLuciferase fusion of full length SPIN1 at a 1:500 (NanoLuc to HaloTag) ratio, respectively, with a FuGENE HD transfection reagent. 16 h post-transfection, cells were collected, washed with phosphate-buffered saline, and exchanged into media containing phenol red-free Dulbecco’s modified Eagle’s medium (DMEM) and 4% fetal bovine serum (FBS) in the absence (control sample) or the presence (experimental sample) of the 100 nM NanoBRET 618 fluorescent ligand (Promega). Cells were then replated in a 384-well assay white plate (Greiner #3570) at 2.7 × 10<sup>3</sup> cells per well. Compound <b>3</b> and compound <b>4</b> were then added directly to media at final concentrations 0–30 μM or an equivalent amount of DMSO as a vehicle control, and the plates were incubated for 24 h at 37 °C in the presence of 5% CO<sub>2</sub>. The NanoBRET Nano-Glo substrate (Promega) was added to both control and experimental samples at a final concentration of 10 μM. Readings were performed within 10 min using ClarioSTAR (BMG Labtech). A corrected BRET ratio was calculated and is defined as the ratio of the emission at 610 nm/460 nm for experimental samples minus the emission at 610 nm/460 nm for control samples (without NanoBRET fluorescent ligand). BRET ratios are expressed as milliBRET units (mBU), where 1 mBU corresponds to the corrected BRET ratio multiplied by 1000. The assay was further validated by the domain-specific site directed mutagenesis (Y170A) ablating peptide and ligand binding.</div></div><div id="sec4_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> Cell Viability Assay</h3><div class="NLM_p last">293T (gift from Dr. Benchimol), C2C12 (gift from Dr. McPherson), and HFF1 (ATTC) cells were cultured following standard protocols in DMEM (Gibco) 10% FBS (Wisent) and penicillin–streptomycin (Gibco). Cells were seeded in 96-well plates (seeding density: 2000 for HFF-1, 1000 for 293T and 500 for C2C12), recovered for 8 h, and treated with several different concentrations of compound <b>3</b> for 6 days. Following cell treatment, cell viability was measured by adding resazurin (Sigma) to the media at 0.01 mg/mL, incubating plates for 2–4 h in a 37 °C CO<sub>2</sub> incubator, and measuring fluorescence at 590 nm on a CLARIOstar microplate reader (BMG Labtech).</div></div><div id="sec4_25" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> Construct Design, Cloning, Expression, and Purification of Human Spindlin Proteins</h3><div class="NLM_p last">A plasmid-encoding full-length human SPIN1 was obtained from Source Bioscience (IOH9972-pDEST26) and used as a template to clone SPIN1<sub>M26–Ser262</sub> and SPIN1<sub>P49–S262</sub> into the pNIC-CTHF vector with a TEV (tobacco etch virus) cleavable C-terminal His<sub>6</sub> tag. SPIN1<sub>G21–S262</sub> was cloned into the pNIC-Bio2 vector with a TEV cleavable N-terminal His<sub>10</sub> tag and a C-terminal biotinylation sequence. SPI2B<sub>P45–S258</sub>, SPIN3<sub>M27–S258</sub>, and SPIN4<sub>T36–P249</sub> were cloned into the pNIC vector with a TEV cleavable N-terminal His6 tag using templates obtained from the Mammalian Gene Collection and Source Bioscience (SPIN2B: cDNA clone IMAGE id: 6729986, SPIN3 IMAGENE: IRCBp5005F0211Q and SPIN4 IMAGE id 40032302). The recombinant proteins were expressed in a phage-resistant derivative of <i>Escherichia coli</i> strain BL21(DE3) carrying the pRARE2 plasmid for rare codon expression. Cells were grown at 37 °C in Terrific broth supplemented with 50 μg/mL kanamycin and 34 μg/mL chloramphenicol, until the culture reached an OD<sub>600</sub> of 2.0. The temperature was decreased to 18 °C and protein expression induced with 0.1 mM IPTG (isopropyl β-<span class="smallcaps smallerCapital">d</span>-thiogalactopyranoside) overnight. Cells were collected by centrifugation and frozen at −80 °C. For purification, cells were resuspended in 50 mM HEPES pH 7.5, 500 mM NaCl, 10 mM imidazole, 5% glycerol, 0.5 mM TCEP, and a protease inhibitor cocktail (Sigma), and lysed by sonication. The cell lysate was clarified by centrifugation, and the proteins were purified by nickel-affinity chromatography (GE Healthcare) using a stepwise gradient of imidazole. This was followed by size exclusion chromatography (Superdex 75 or Superdex 200, GE Healthcare) as previously described.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> For the ITC assay, aliquots of purified His–SPIN1(49–262) protein were stored at −80 °C in the ITC assay buffer containing 20 mM Tris-HCl (pH 8.0), 200 mM NaCl. For crystallization studies, the histidine tag was removed by incubation with TEV protease at 4 °C overnight, and the TEV protease and the uncleaved proteins were then removed by nickel-affinity chromatography. Proteins were concentrated using an Amicon centrifugal filtration unit, and the mass was verified by ESI time of flight mass spectrometry (ESI-TOF-TOF: Agilent LC/MSD).</div></div><div id="sec4_26" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> Construct Design, Cloning, Expression, and Purification of G9a and GLP Proteins</h3><div class="NLM_p last">Human G9a and GLP catalytic domains were cloned, expressed, and purified as previously described.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i41"><a href="/doi/suppl/10.1021/acs.jmedchem.9b00522" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_55271" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_55271" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.9b00522" class="ext-link">10.1021/acs.jmedchem.9b00522</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Concentration response curves of A366 and compound <b>1</b> in the SPIN1 AlphaLISA and FP biochemical assays; docking model of compound <b>1</b> in the complex with SPIN1; binding of compound <b>2</b> to GLP by ITC; binding of compound <b>3</b> to SPIN2B, SPIN3, and SPIN4 by ITC; <sup>1</sup>H NMR and <sup>13</sup>C NMR spectra of compounds <b>3</b> and <b>4</b>; raw data of the enzymatic inhibition assay against 33 MTs and 5 acetyltransferases; raw data results of the <i>T</i><sub>m</sub> shift assay against 20 methyllysine, methylarginine, and acetyllysine reader proteins; crystallography data and refinement statistics; and description of cell lines for cytotoxicity assays (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00522/suppl_file/jm9b00522_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00522/suppl_file/jm9b00522_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00522/suppl_file/jm9b00522_si_001.pdf">jm9b00522_si_001.pdf (2.75 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00522/suppl_file/jm9b00522_si_002.csv">jm9b00522_si_002.csv (0.6 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The structure of SPIN1 in the complex with compound <b>3</b> has been deposited under PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QPL">6QPL</a>. The authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.9b00522" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_08554" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_08554" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Udo Oppermann</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, Botnar Research Center, NIHR Oxford BRU, University of Oxford, Oxford OX37LD, U.K.</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9984-5342" title="Orcid link">http://orcid.org/0000-0001-9984-5342</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#e297868dcc8d929287908f838c8ca2918581cc8d9acc8381cc9789"><span class="__cf_email__" data-cfemail="85f0e1eaabeaf5f5e0f7e8e4ebebc5f6e2e6abeafdabe4e6abf0ee">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Roland Schüle</span> - <span class="hlFld-Affiliation affiliation">Department
of Urology and Center for Clinical Research, University Freiburg Medical Center, Freiburg 79106, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">BIOSS
Centre of Biological Signalling Studies, University of Freiburg, Freiburg 79106, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">German Cancer Consortium (DKTK), Freiburg 79106, Germany</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#beccd1d2dfd0da90cdddd6cbdbd2dbfecbd0d7d5d2d7d0d7d593d8ccdbd7dccbccd990dadb"><span class="__cf_email__" data-cfemail="71031e1d101f155f02121904141d1431041f181a1d181f181a5c17031418130403165f1514">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jian Jin</span> - <span class="hlFld-Affiliation affiliation">Mount
Sinai Center for Therapeutics Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2387-3862" title="Orcid link">http://orcid.org/0000-0002-2387-3862</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#29434048470743404769445a5a44074c4d5c"><span class="__cf_email__" data-cfemail="a6cccfc7c888cccfc8e6cbd5d5cb88c3c2d3">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yan Xiong</span> - <span class="hlFld-Affiliation affiliation">Mount
Sinai Center for Therapeutics Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Holger Greschik</span> - <span class="hlFld-Affiliation affiliation">Department
of Urology and Center for Clinical Research, University Freiburg Medical Center, Freiburg 79106, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Catrine Johansson</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, Botnar Research Center, NIHR Oxford BRU, University of Oxford, Oxford OX37LD, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ludwig Seifert</span> - <span class="hlFld-Affiliation affiliation">Institute
of Pharmaceutical Sciences, University of
Freiburg, Freiburg 79104, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Johannes Bacher</span> - <span class="hlFld-Affiliation affiliation">Institute
of Pharmaceutical Sciences, University of
Freiburg, Freiburg 79104, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kwang-su Park</span> - <span class="hlFld-Affiliation affiliation">Mount
Sinai Center for Therapeutics Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nicolas Babault</span> - <span class="hlFld-Affiliation affiliation">Mount
Sinai Center for Therapeutics Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michael Martini</span> - <span class="hlFld-Affiliation affiliation">Mount
Sinai Center for Therapeutics Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Vincent Fagan</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium & Target Discovery Institute, University of Oxford, Oxford OX37DQ and OX37FZ, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fengling Li</span> - <span class="hlFld-Affiliation affiliation">Structural Genomics Consortium and , University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Irene Chau</span> - <span class="hlFld-Affiliation affiliation">Structural Genomics Consortium and , University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Thomas Christott</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium & Target Discovery Institute, University of Oxford, Oxford OX37DQ and OX37FZ, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David Dilworth</span> - <span class="hlFld-Affiliation affiliation">Structural Genomics Consortium and , University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dalia Barsyte-Lovejoy</span> - <span class="hlFld-Affiliation affiliation">Structural Genomics Consortium, 
     and , University of Toronto, Toronto, Ontario M5G 1L7, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Nature
Research Center, Akademijos
2, Vilnius 08412, Lithuania</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Masoud Vedadi</span> - <span class="hlFld-Affiliation affiliation">Structural Genomics Consortium, 
     and , University of Toronto, Toronto, Ontario M5G 1L7, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Pharmacology and Toxicology, University
of Toronto, Toronto, Ontario M5S 1A8, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Cheryl H. Arrowsmith</span> - <span class="hlFld-Affiliation affiliation">Structural Genomics Consortium, 
    Princess Margaret Cancer Centre
and Department
of Medical Biophysics, 
     and , University of Toronto, Toronto, Ontario M5G 1L7, Canada</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4971-3250" title="Orcid link">http://orcid.org/0000-0002-4971-3250</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paul Brennan</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium & Target Discovery Institute, University of Oxford, Oxford OX37DQ and OX37FZ, U.K.</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8950-7646" title="Orcid link">http://orcid.org/0000-0002-8950-7646</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Oleg Fedorov</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium & Target Discovery Institute, University of Oxford, Oxford OX37DQ and OX37FZ, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Manfred Jung</span> - <span class="hlFld-Affiliation affiliation">Institute
of Pharmaceutical Sciences, University of
Freiburg, Freiburg 79104, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">German Cancer Research Centre (DKFZ), Heidelberg 69120, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">German Cancer Consortium (DKTK), Freiburg 79106, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6361-7716" title="Orcid link">http://orcid.org/0000-0002-6361-7716</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gillian Farnie</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium & Target Discovery Institute, University of Oxford, Oxford OX37DQ and OX37FZ, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jing Liu</span> - <span class="hlFld-Affiliation affiliation">Mount
Sinai Center for Therapeutics Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>Y.X., H.G., and C.J. contributed equally to this work.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i44">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_71086" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_71086" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">R.S. acknowledges support by grants of the European Research Council (ERC AdGrant 322844) and the Deutsche Forschungsgemeinschaft CRC 992, 850, and Schu688/15-1. M.J. thanks the DFG (Deutsche Forschungsgemeinschaft, CRC992) for funding. The SGC is a registered charity (number 1097737) that receives funds from AbbVie, Bayer Pharma AG, Boehringer Ingelheim, Canada Foundation for Innovation, Eshelman Institute for Innovation, Genome Canada, Innovative Medicines Initiative (EU/EFPIA) [ULTRA-DD grant no. 115766], Janssen, Merck KGaA Darmstadt Germany, MSD, Novartis Pharma AG, Ontario Ministry of Economic Development and Innovation, Pfizer, São Paulo Research Foundation-FAPESP, Takeda, and Wellcome [106169/ZZ14/Z]. We acknowledge the Natural Sciences and Engineering Research Council of Canada (NSERC) for a postdoctoral fellowship awarded to D.D. We thank Drs. Benchimol and McPherson for providing 293T and C2C12 cells.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">SPIN1</td><td class="NLM_def"><p class="first last">spindlin 1</p></td></tr><tr><td class="NLM_term">GLP</td><td class="NLM_def"><p class="first last">G9a-like protein 1</p></td></tr><tr><td class="NLM_term">L3MBTL3</td><td class="NLM_def"><p class="first last">lethal(3) malignant brain tumor-like protein 3</p></td></tr><tr><td class="NLM_term">CBX</td><td class="NLM_def"><p class="first last">chromobox</p></td></tr><tr><td class="NLM_term">MBT</td><td class="NLM_def"><p class="first last">malignant brain tumor</p></td></tr><tr><td class="NLM_term">H3K4me3</td><td class="NLM_def"><p class="first last">trimethylation of lysine 4 on histone H3</p></td></tr><tr><td class="NLM_term">H3R8me2</td><td class="NLM_def"><p class="first last">dimethylation of arginine 9 on histone H3</p></td></tr><tr><td class="NLM_term">PTMs</td><td class="NLM_def"><p class="first last">post-translational modifications</p></td></tr><tr><td class="NLM_term">Wnt</td><td class="NLM_def"><p class="first last">wingless/integrated</p></td></tr><tr><td class="NLM_term">TCF</td><td class="NLM_def"><p class="first last">T-cell factor/lymphoid enhancer-binding factor</p></td></tr><tr><td class="NLM_term">LE</td><td class="NLM_def"><p class="first last">ligand efficiency</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">Alpha</td><td class="NLM_def"><p class="first last">amplified luminescent proximity homogeneous assay</p></td></tr><tr><td class="NLM_term">FP</td><td class="NLM_def"><p class="first last">fluorescence polarization</p></td></tr><tr><td class="NLM_term">ITC</td><td class="NLM_def"><p class="first last">isothermal titration calorimetry</p></td></tr><tr><td class="NLM_term">BRET</td><td class="NLM_def"><p class="first last">bioluminescence resonance energy transfer</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i46">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_34781" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_34781" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 59 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bernstein, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meissner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lander, E. S.</span></span> <span> </span><span class="NLM_article-title">The mammalian epigenome</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>128</i></span>,  <span class="NLM_fpage">669</span>– <span class="NLM_lpage">681</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2007.01.033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1016%2Fj.cell.2007.01.033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=17320505" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BD2sXis12ju7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2007&pages=669-681&author=B.+E.+Bernsteinauthor=A.+Meissnerauthor=E.+S.+Lander&title=The+mammalian+epigenome&doi=10.1016%2Fj.cell.2007.01.033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The mammalian epigenome</span></div><div class="casAuthors">Bernstein, Bradley E.; Meissner, Alexander; Lander, Eric S.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">669-681</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Chem. modifications to DNA and histone proteins form a complex regulatory network that modulates chromatin structure and genome function.  The epigenome refers to the complete description of these potentially heritable changes across the genome.  The compn. of the epigenome within a given cell is a function of genetic determinants, lineage, and environment.  With the sequencing of the human genome completed, investigators now seek a comprehensive view of the epigenetic changes that det. how genetic information is made manifest across an incredibly varied background of developmental stages, tissue types, and disease states.  Here we review current research efforts, with an emphasis on large-scale studies, emerging technologies, and challenges ahead.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKJfRRDsxAHbVg90H21EOLACvtfcHk0lioJnJZsdYN_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXis12ju7g%253D&md5=963fb487bbf473ac801e09a558ac0b0e</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.01.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.01.033%26sid%3Dliteratum%253Aachs%26aulast%3DBernstein%26aufirst%3DB.%2BE.%26aulast%3DMeissner%26aufirst%3DA.%26aulast%3DLander%26aufirst%3DE.%2BS.%26atitle%3DThe%2520mammalian%2520epigenome%26jtitle%3DCell%26date%3D2007%26volume%3D128%26spage%3D669%26epage%3D681%26doi%3D10.1016%2Fj.cell.2007.01.033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Strahl, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allis, C. D.</span></span> <span> </span><span class="NLM_article-title">The language of covalent histone modifications</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>403</i></span>,  <span class="NLM_fpage">41</span>– <span class="NLM_lpage">45</span>, <span class="refDoi"> DOI: 10.1038/47412</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1038%2F47412" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10638745" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A280%3ADC%252BD3c7gt1arsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=403&publication_year=2000&pages=41-45&author=B.+D.+Strahlauthor=C.+D.+Allis&title=The+language+of+covalent+histone+modifications&doi=10.1038%2F47412"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">The language of covalent histone modifications</span></div><div class="casAuthors">Strahl B D; Allis C D</div><div class="citationInfo"><span class="NLM_cas:title">Nature</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">403</span>
        (<span class="NLM_cas:issue">6765</span>),
    <span class="NLM_cas:pages">41-5</span>
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">Histone proteins and the nucleosomes they form with DNA are the fundamental building blocks of eukaryotic chromatin.  A diverse array of post-translational modifications that often occur on tail domains of these proteins has been well documented.  Although the function of these highly conserved modifications has remained elusive, converging biochemical and genetic evidence suggests functions in several chromatin-based processes.  We propose that distinct histone modifications, on one or more tails, act sequentially or in combination to form a 'histone code' that is, read by other proteins to bring about distinct downstream events.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSyzhdlYh9sX7t7lRqEFUJVfW6udTcc2eb2Z5yptdJk-7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3c7gt1arsQ%253D%253D&md5=2eb7d093151c6122ccd70fa2adcfd94b</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2F47412&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F47412%26sid%3Dliteratum%253Aachs%26aulast%3DStrahl%26aufirst%3DB.%2BD.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DThe%2520language%2520of%2520covalent%2520histone%2520modifications%26jtitle%3DNature%26date%3D2000%26volume%3D403%26spage%3D41%26epage%3D45%26doi%3D10.1038%2F47412" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rea, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eisenhaber, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Carroll, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strahl, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Z.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmid, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Opravil, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mechtler, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ponting, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allis, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenuwein, T.</span></span> <span> </span><span class="NLM_article-title">Regulation of chromatin structure by site-specific histone H3 methyltransferases</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>406</i></span>,  <span class="NLM_fpage">593</span>– <span class="NLM_lpage">599</span>, <span class="refDoi"> DOI: 10.1038/35020506</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1038%2F35020506" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10949293" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BD3cXmtFegs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=406&publication_year=2000&pages=593-599&author=S.+Reaauthor=F.+Eisenhaberauthor=D.+O%E2%80%99Carrollauthor=B.+D.+Strahlauthor=Z.-W.+Sunauthor=M.+Schmidauthor=S.+Opravilauthor=K.+Mechtlerauthor=C.+P.+Pontingauthor=C.+D.+Allisauthor=T.+Jenuwein&title=Regulation+of+chromatin+structure+by+site-specific+histone+H3+methyltransferases&doi=10.1038%2F35020506"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of chromatin structure by site-specific histone H3 methyltransferases</span></div><div class="casAuthors">Rea, Stephen; Elsenhaber, Frank; O'Carroll, Donal; Strahl, Brian D.; Sun, Zu-Wen; Schmid, Manfred; Opravil, Susanne; Mechtler, Karl; Ponting, Chris P.; Allis, C. David; Jenuwein, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">406</span>
        (<span class="NLM_cas:issue">6796</span>),
    <span class="NLM_cas:pages">593-599</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The organization of chromatin into higher-order structures influences chromosome function and epigenetic gene regulation.  Higher-order chromatin has been proposed to be nucleated by the covalent modification of histone tails and the subsequent establishment of chromosomal subdomains by non-histone modifier factors.  Here we show that human SUV39H1 and murine Suv39h1 - mammalian homologs of Drosophila Su(var)3-9 and of Schizosaccharomyces pombe clr4 - encode histone H3-specific methyltransferases that selectively methylate lysine 9 of the amino terminus of histone H3 in vitro.  We mapped the catalytic motif to the evolutionarily conserved SET domain, which requires adjacent cysteine-rich regions to confer histone methyltransferase activity.  Methylation of lysine 9 interferes with phosphorylation of serine 10, but is also influenced by pre-existing modifications in the amino terminus of H3.  In vivo, deregulated SUV39H1 or disrupted Suv39h activity modulate H3 serine 10 phosphorylation in native chromatin and induce aberrant mitotic divisions.  Our data reveal a functional interdependence of site-specific H3 tail modifications and suggest a dynamic mechanism for the regulation of higher-order chromatin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2yioyTahy77Vg90H21EOLACvtfcHk0lioJnJZsdYN_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXmtFegs7s%253D&md5=65e14733d25f97fadaaeed602600bd23</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2F35020506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F35020506%26sid%3Dliteratum%253Aachs%26aulast%3DRea%26aufirst%3DS.%26aulast%3DEisenhaber%26aufirst%3DF.%26aulast%3DO%25E2%2580%2599Carroll%26aufirst%3DD.%26aulast%3DStrahl%26aufirst%3DB.%2BD.%26aulast%3DSun%26aufirst%3DZ.-W.%26aulast%3DSchmid%26aufirst%3DM.%26aulast%3DOpravil%26aufirst%3DS.%26aulast%3DMechtler%26aufirst%3DK.%26aulast%3DPonting%26aufirst%3DC.%2BP.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26aulast%3DJenuwein%26aufirst%3DT.%26atitle%3DRegulation%2520of%2520chromatin%2520structure%2520by%2520site-specific%2520histone%2520H3%2520methyltransferases%26jtitle%3DNature%26date%3D2000%26volume%3D406%26spage%3D593%26epage%3D599%26doi%3D10.1038%2F35020506" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kouzarides, T.</span></span> <span> </span><span class="NLM_article-title">Chromatin modifications and their function</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>128</i></span>,  <span class="NLM_fpage">693</span>– <span class="NLM_lpage">705</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2007.02.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1016%2Fj.cell.2007.02.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=17320507" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BD2sXis12ju7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2007&pages=693-705&author=T.+Kouzarides&title=Chromatin+modifications+and+their+function&doi=10.1016%2Fj.cell.2007.02.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatin modifications and their function</span></div><div class="casAuthors">Kouzarides, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">693-705</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The surface of nucleosomes is studded with a multiplicity of modifications.  At least eight different classes have been characterized to date and many different sites have been identified for each class.  Operationally, modifications function either by disrupting chromatin contacts or by affecting the recruitment of nonhistone proteins to chromatin.  Their presence on histones can dictate the higher-order chromatin structure in which DNA is packaged and can orchestrate the ordered recruitment of enzyme complexes to manipulate DNA.  In this way, histone modifications have the potential to influence many fundamental biol. processes, some of which may be epigenetically inherited.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAH9AX2dGicbVg90H21EOLACvtfcHk0lhQP9rgjoRs5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXis12ju7Y%253D&md5=c5995fe68fd9473f4268d59c01d3b9de</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.02.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.02.005%26sid%3Dliteratum%253Aachs%26aulast%3DKouzarides%26aufirst%3DT.%26atitle%3DChromatin%2520modifications%2520and%2520their%2520function%26jtitle%3DCell%26date%3D2007%26volume%3D128%26spage%3D693%26epage%3D705%26doi%3D10.1016%2Fj.cell.2007.02.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H. Ü.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Chemical probes of histone lysine methyltransferases</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">40</span>– <span class="NLM_lpage">50</span>, <span class="refDoi"> DOI: 10.1021/cb500785t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500785t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFOht7%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=40-50&author=H.+%C3%9C.+Kaniskanauthor=J.+Jin&title=Chemical+probes+of+histone+lysine+methyltransferases&doi=10.1021%2Fcb500785t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical probes of histone lysine methyltransferases</span></div><div class="casAuthors">Kaniskan, H. Umit; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">40-50</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Growing evidence suggests that histone-lysine methyltransferases (HMTs, also known as protein methyltransferases (PMTs)) play an important role in diverse biol. processes and human diseases by regulating gene expression and the chromatin state.  Therefore, HMTs have been increasingly recognized by the biomedical community as a class of potential therapeutic targets.  High quality chem. probes of HMTs, as tools for deciphering their physiol. functions and roles in human diseases and testing therapeutic hypotheses, are crit. for advancing this promising field.  Here, the authors focus on the discovery, characterization, and biol. applications of chem. probes for HMTs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpp8kb_c7Xl57Vg90H21EOLACvtfcHk0lhQP9rgjoRs5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFOht7%252FE&md5=4eb8a696e2227c60686df8d3c60777ce</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Fcb500785t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500785t%26sid%3Dliteratum%253Aachs%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259C.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DChemical%2520probes%2520of%2520histone%2520lysine%2520methyltransferases%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D40%26epage%3D50%26doi%3D10.1021%2Fcb500785t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H. Ü.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konze, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Selective inhibitors of protein methyltransferases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">1596</span>– <span class="NLM_lpage">1629</span>, <span class="refDoi"> DOI: 10.1021/jm501234a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501234a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFKqtbbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=1596-1629&author=H.+%C3%9C.+Kaniskanauthor=K.+D.+Konzeauthor=J.+Jin&title=Selective+inhibitors+of+protein+methyltransferases&doi=10.1021%2Fjm501234a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibitors of protein methyltransferases</span></div><div class="casAuthors">Kaniskan, H. Umit; Konze, Kyle D.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1596-1629</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Mounting evidence suggests that protein methyltransferases (PMTs), which catalyze the methylation of histone and non-histone proteins, play a crucial role in diverse biol. processes and human diseases.  In particular, PMTs have been recognized as major players in regulating gene expression and chromatin state.  PMTs are divided into 2 categories: protein lysine methyltransferases (PKMTs) and protein arginine methyltransferases (PRMTs).  There has been a steadily growing interest in these enzymes as potential therapeutic targets and therefore discovery of PMT inhibitors has also been pursued increasingly over the past decade.  Here, the authors present a perspective on selective, small-mol. inhibitors of PMTs with an emphasis on their discovery, characterization, and applicability as chem. tools for deciphering the target PMTs' physiol. functions and involvement in human diseases.  The authors highlight the current state of PMT inhibitors and discuss future directions and opportunities for PMT inhibitor discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEWF_XeLEY1rVg90H21EOLACvtfcHk0lhQP9rgjoRs5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFKqtbbJ&md5=8fb2b2d21d306c02a2da60a3fb88c430</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Fjm501234a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501234a%26sid%3Dliteratum%253Aachs%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259C.%26aulast%3DKonze%26aufirst%3DK.%2BD.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DSelective%2520inhibitors%2520of%2520protein%2520methyltransferases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D1596%26epage%3D1629%26doi%3D10.1021%2Fjm501234a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H. Ü.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martini, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Inhibitors of protein methyltransferases and demethylases</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">989</span>– <span class="NLM_lpage">1068</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.6b00801</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.6b00801" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkslynsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2018&pages=989-1068&author=H.+%C3%9C.+Kaniskanauthor=M.+L.+Martiniauthor=J.+Jin&title=Inhibitors+of+protein+methyltransferases+and+demethylases&doi=10.1021%2Facs.chemrev.6b00801"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of Protein Methyltransferases and Demethylases</span></div><div class="casAuthors">Kaniskan, H. Umit; Martini, Michael L.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">989-1068</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Post-translational modifications of histones by protein methyltransferases (PMTs) and histone demethylases (KDMs) play an important role in the regulation of gene expression and transcription and are implicated in cancer and many other diseases.  Many of these enzymes also target various nonhistone proteins impacting numerous crucial biol. pathways.  Given their key biol. functions and implications in human diseases, there has been a growing interest in assessing these enzymes as potential therapeutic targets.  Consequently, discovering and developing inhibitors of these enzymes has become a very active and fast-growing research area over the past decade.  In this review, the authors cover the discovery, characterization, and biol. application of inhibitors of PMTs and KDMs with emphasis on key advancements in the field.  The authors also discuss challenges, opportunities, and future directions in this emerging, exciting research field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJYjiYtw_n4rVg90H21EOLACvtfcHk0li84TU8C-sVhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkslynsL8%253D&md5=1ed520c6b0c42d2aa8a7f66b0a2d4e40</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.6b00801&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.6b00801%26sid%3Dliteratum%253Aachs%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259C.%26aulast%3DMartini%26aufirst%3DM.%2BL.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DInhibitors%2520of%2520protein%2520methyltransferases%2520and%2520demethylases%26jtitle%3DChem.%2520Rev.%26date%3D2018%26volume%3D118%26spage%3D989%26epage%3D1068%26doi%3D10.1021%2Facs.chemrev.6b00801" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulligan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whetstine, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casero, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span> <span> </span><span class="NLM_article-title">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>119</i></span>,  <span class="NLM_fpage">941</span>– <span class="NLM_lpage">953</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2004.12.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1016%2Fj.cell.2004.12.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=15620353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2004&pages=941-953&author=Y.+Shiauthor=F.+Lanauthor=C.+Matsonauthor=P.+Mulliganauthor=J.+R.+Whetstineauthor=P.+A.+Coleauthor=R.+A.+Caseroauthor=Y.+Shi&title=Histone+demethylation+mediated+by+the+nuclear+amine+oxidase+homolog+LSD1&doi=10.1016%2Fj.cell.2004.12.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span></div><div class="casAuthors">Shi, Yujiang; Lan, Fei; Matson, Caitlin; Mulligan, Peter; Whetstine, Johnathan R.; Cole, Philip A.; Casero, Robert A.; Shi, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">941-953</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Posttranslational modifications of histone N-terminal tails impact chromatin structure and gene transcription.  While the extent of histone acetylation is detd. by both acetyltransferases and deacetylases, it has been unclear whether histone methylation is also regulated by enzymes with opposing activities.  Here, we provide evidence that LSD1 (KIAA0601), a nuclear homolog of amine oxidases, functions as a histone demethylase and transcriptional corepressor.  LSD1 specifically demethylates histone H3 lysine 4, which is linked to active transcription.  Lysine demethylation occurs via an oxidn. reaction that generates formaldehyde.  Importantly, RNAi inhibition of LSD1 causes an increase in H3 lysine 4 methylation and concomitant derepression of target genes, suggesting that LSD1 represses transcription via histone demethylation.  The results thus identify a histone demethylase conserved from S. pombe to human and reveal dynamic regulation of histone methylation by both histone methylases and demethylases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXHr-8kwGdXrVg90H21EOLACvtfcHk0li84TU8C-sVhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D&md5=3474569204255e90a832667ba654748b</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2004.12.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2004.12.012%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26aulast%3DLan%26aufirst%3DF.%26aulast%3DMatson%26aufirst%3DC.%26aulast%3DMulligan%26aufirst%3DP.%26aulast%3DWhetstine%26aufirst%3DJ.%2BR.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DHistone%2520demethylation%2520mediated%2520by%2520the%2520nuclear%2520amine%2520oxidase%2520homolog%2520LSD1%26jtitle%3DCell%26date%3D2004%26volume%3D119%26spage%3D941%26epage%3D953%26doi%3D10.1016%2Fj.cell.2004.12.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dhalluin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aggarwal, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.-M.</span></span> <span> </span><span class="NLM_article-title">Structure and ligand of a histone acetyltransferase bromodomain</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>399</i></span>,  <span class="NLM_fpage">491</span>– <span class="NLM_lpage">496</span>, <span class="refDoi"> DOI: 10.1038/20974</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1038%2F20974" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10365964" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADyaK1MXjvV2isLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=399&publication_year=1999&pages=491-496&author=C.+Dhalluinauthor=J.+E.+Carlsonauthor=L.+Zengauthor=C.+Heauthor=A.+K.+Aggarwalauthor=M.-M.+Zhouauthor=M.-M.+Zhou&title=Structure+and+ligand+of+a+histone+acetyltransferase+bromodomain&doi=10.1038%2F20974"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and ligand of a histone acetyltransferase bromodomain</span></div><div class="casAuthors">Dhalluin, Christophe; Carlson, Justin E.; Zeng, Lei; He, Cheng; Aggarwal, Aneel K.; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">399</span>
        (<span class="NLM_cas:issue">6735</span>),
    <span class="NLM_cas:pages">491-496</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">Histone acetylation is important in chromatin remodelling and gene activation.  Nearly all known histone-acetyltransferase (HAT)-assocd. transcriptional co-activators contain bromodomains, which are ∼110-amino-acid modules found in many chromatin-assocd. proteins.  Despite the wide occurrence of these bromodomains, their three-dimensional structure and binding partners remain unknown.  Here we report the soln. structure of the bromodomain of the HAT co-activator P/CAF (p300/CBP-assocd. factor).  The structure reveals an unusual left-handed up-and-down four-helix bundle.  In addn., we show by a combination of structural and site-directed mutagenesis studies that bromodomains can interact specifically with acetylated lysine, making them the first known protein modules to do so.  The nature of the recognition of acetyl-lysine by the P/CAF bromodomain is similar to that of acetyl-CoA by histone acetyltransferase.  Thus, the bromodomain is functionally linked to the HAT activity of co-activators in the regulation of gene transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsTbwcjeBG_rVg90H21EOLACvtfcHk0li84TU8C-sVhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjvV2isLo%253D&md5=ff1063ba81d81b431065dd6e3751ba79</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2F20974&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F20974%26sid%3Dliteratum%253Aachs%26aulast%3DDhalluin%26aufirst%3DC.%26aulast%3DCarlson%26aufirst%3DJ.%2BE.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DHe%26aufirst%3DC.%26aulast%3DAggarwal%26aufirst%3DA.%2BK.%26aulast%3DZhou%26aufirst%3DM.-M.%26aulast%3DZhou%26aufirst%3DM.-M.%26atitle%3DStructure%2520and%2520ligand%2520of%2520a%2520histone%2520acetyltransferase%2520bromodomain%26jtitle%3DNature%26date%3D1999%26volume%3D399%26spage%3D491%26epage%3D496%26doi%3D10.1038%2F20974" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Su, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denu, J. M.</span></span> <span> </span><span class="NLM_article-title">Reading the combinatorial histone language</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">564</span>– <span class="NLM_lpage">574</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00864</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00864" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVagsL%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=564-574&author=Z.+Suauthor=J.+M.+Denu&title=Reading+the+combinatorial+histone+language&doi=10.1021%2Facschembio.5b00864"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Reading the Combinatorial Histone Language</span></div><div class="casAuthors">Su, Zhangli; Denu, John M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">564-574</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Histones are subject to frequent combinatorial post-translational modifications (PTMs), forming a complex chem. "language" that is interpreted by PTM-specific histone-interacting protein modules (reader domains).  These specific interactions are thought to instruct gene expression and downstream biol. functions.  While the majority of studies have focused on individual modifications, the authors' current understanding of the combinatorial PTM patterns on histones is starting to emerge, benefiting from the convergence of multiple technologies.  Here, the authors review the key tech. advances and progress on discovery and characterization of combinatorial histone PTM patterns.  The authors focus on the interactions between reader domains and combinatorial PTMs, which is essential for understanding the mechanism and biol. meaning of establishing and interpreting information embedded in histone PTM patterns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHpmgsQnFL7rVg90H21EOLACvtfcHk0li84TU8C-sVhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVagsL%252FK&md5=819be9f59c1c1f0ae1ee09665b2d51da</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00864&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00864%26sid%3Dliteratum%253Aachs%26aulast%3DSu%26aufirst%3DZ.%26aulast%3DDenu%26aufirst%3DJ.%2BM.%26atitle%3DReading%2520the%2520combinatorial%2520histone%2520language%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D564%26epage%3D574%26doi%3D10.1021%2Facschembio.5b00864" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouzarides, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huntly, B. J. P.</span></span> <span> </span><span class="NLM_article-title">Targeting epigenetic readers in cancer</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>367</i></span>,  <span class="NLM_fpage">647</span>– <span class="NLM_lpage">657</span>, <span class="refDoi"> DOI: 10.1056/nejmra1112635</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1056%2Fnejmra1112635" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=22894577" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1Chu77F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=367&publication_year=2012&pages=647-657&author=M.+A.+Dawsonauthor=T.+Kouzaridesauthor=B.+J.+P.+Huntly&title=Targeting+epigenetic+readers+in+cancer&doi=10.1056%2Fnejmra1112635"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting epigenetic readers in cancer</span></div><div class="casAuthors">Dawson, Mark A.; Kouzarides, Tony; Huntly, Brian J. P.</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">367</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">647-657</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">A review on assocn. between epigenetics and chromatin biol., BET bromodomain inhibitors, and role of BET inhibitors in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnz_YQy6Z2T7Vg90H21EOLACvtfcHk0lganlfMEHFojg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1Chu77F&md5=f5e1e82c8a1105f798b1f7719138721c</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1056%2Fnejmra1112635&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252Fnejmra1112635%26sid%3Dliteratum%253Aachs%26aulast%3DDawson%26aufirst%3DM.%2BA.%26aulast%3DKouzarides%26aufirst%3DT.%26aulast%3DHuntly%26aufirst%3DB.%2BJ.%2BP.%26atitle%3DTargeting%2520epigenetic%2520readers%2520in%2520cancer%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2012%26volume%3D367%26spage%3D647%26epage%3D657%26doi%3D10.1056%2Fnejmra1112635" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bountra, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fish, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span> <span> </span><span class="NLM_article-title">Epigenetic protein families: a new frontier for drug discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">384</span>– <span class="NLM_lpage">400</span>, <span class="refDoi"> DOI: 10.1038/nrd3674</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1038%2Fnrd3674" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=22498752" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC38XlsFWjsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=384-400&author=C.+H.+Arrowsmithauthor=C.+Bountraauthor=P.+V.+Fishauthor=K.+Leeauthor=M.+Schapira&title=Epigenetic+protein+families%3A+a+new+frontier+for+drug+discovery&doi=10.1038%2Fnrd3674"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic protein families: a new frontier for drug discovery</span></div><div class="casAuthors">Arrowsmith, Cheryl H.; Bountra, Chas; Fish, Paul V.; Lee, Kevin; Schapira, Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">384-400</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Epigenetic regulation of gene expression is a dynamic and reversible process that establishes normal cellular phenotypes but also contributes to human diseases.  At the mol. level, epigenetic regulation involves hierarchical covalent modification of DNA and the proteins that package DNA, such as histones.  Here, we review the key protein families that mediate epigenetic signalling through the acetylation and methylation of histones, including histone deacetylases, protein methyltransferases, lysine demethylases, bromodomain-contg. proteins and proteins that bind to methylated histones.  These protein families are emerging as druggable classes of enzymes and druggable classes of protein-protein interaction domains.  In this article, we discuss the known links with disease, basic mol. mechanisms of action and recent progress in the pharmacol. modulation of each class of proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcD99eCigNtLVg90H21EOLACvtfcHk0lganlfMEHFojg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlsFWjsbs%253D&md5=eb82034466a43107aa74a18ccf6d29f5</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnrd3674&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3674%26sid%3Dliteratum%253Aachs%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DFish%26aufirst%3DP.%2BV.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DEpigenetic%2520protein%2520families%253A%2520a%2520new%2520frontier%2520for%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2012%26volume%3D11%26spage%3D384%26epage%3D400%26doi%3D10.1038%2Fnrd3674" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Helin, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhanak, D.</span></span> <span> </span><span class="NLM_article-title">Chromatin proteins and modifications as drug targets</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>502</i></span>,  <span class="NLM_fpage">480</span>– <span class="NLM_lpage">488</span>, <span class="refDoi"> DOI: 10.1038/nature12751</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1038%2Fnature12751" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=24153301" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1Krt77E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=502&publication_year=2013&pages=480-488&author=K.+Helinauthor=D.+Dhanak&title=Chromatin+proteins+and+modifications+as+drug+targets&doi=10.1038%2Fnature12751"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatin proteins and modifications as drug targets</span></div><div class="casAuthors">Helin, Kristian; Dhanak, Dashyant</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">502</span>
        (<span class="NLM_cas:issue">7472</span>),
    <span class="NLM_cas:pages">480-488</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  A plethora of groundbreaking studies have demonstrated the importance of chromatin-assocd. proteins and post-translational modifications of histones, proteins and DNA (so-called epigenetic modifications) for transcriptional control and normal development.  Disruption of epigenetic control is a frequent event in disease, and the first epigenetic-based therapies for cancer treatment have been approved.  A generation of new classes of potent and specific inhibitors for several chromatin-assocd. proteins have shown promise in preclin. trials.  Although the biol. of epigenetic regulation is complex, new inhibitors such as these will hopefully be of clin. use in the coming years.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpa6LKkNrgP9bVg90H21EOLACvtfcHk0lganlfMEHFojg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1Krt77E&md5=ca3f12c6aaba31bb571095bacc02550c</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnature12751&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12751%26sid%3Dliteratum%253Aachs%26aulast%3DHelin%26aufirst%3DK.%26aulast%3DDhanak%26aufirst%3DD.%26atitle%3DChromatin%2520proteins%2520and%2520modifications%2520as%2520drug%2520targets%26jtitle%3DNature%26date%3D2013%26volume%3D502%26spage%3D480%26epage%3D488%26doi%3D10.1038%2Fnature12751" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Højfeldt, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agger, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helin, K.</span></span> <span> </span><span class="NLM_article-title">Histone lysine demethylases as targets for anticancer therapy</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">917</span>– <span class="NLM_lpage">930</span>, <span class="refDoi"> DOI: 10.1038/nrd4154</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1038%2Fnrd4154" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=24232376" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslygtrrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=917-930&author=J.+W.+H%C3%B8jfeldtauthor=K.+Aggerauthor=K.+Helin&title=Histone+lysine+demethylases+as+targets+for+anticancer+therapy&doi=10.1038%2Fnrd4154"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Histone lysine demethylases as targets for anticancer therapy</span></div><div class="casAuthors">Hojfeldt, Jonas W.; Agger, Karl; Helin, Kristian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">917-930</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  It has recently been demonstrated that the genes controlling the epigenetic programs that are required for maintaining chromatin structure and cell identity include genes that drive human cancer.  This observation has led to an increased awareness of chromatin-assocd. proteins as potentially interesting drug targets.  The successful introduction of DNA methylation and histone deacetylase (HDAC) inhibitors for the treatment of specific subtypes of cancer has paved the way for the use of epigenetic therapy.  Here, we highlight key biol. findings demonstrating the roles of members of the histone lysine demethylase class of enzymes in the development of cancers, discuss the potential and challenges of therapeutically targeting them, and highlight emerging small-mol. inhibitors of these enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoThyggRtM7qrVg90H21EOLACvtfcHk0lhAAo-v-yNLtA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslygtrrP&md5=b120d43a0353078b016c8dc8f7cac644</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fnrd4154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4154%26sid%3Dliteratum%253Aachs%26aulast%3DH%25C3%25B8jfeldt%26aufirst%3DJ.%2BW.%26aulast%3DAgger%26aufirst%3DK.%26aulast%3DHelin%26aufirst%3DK.%26atitle%3DHistone%2520lysine%2520demethylases%2520as%2520targets%2520for%2520anticancer%2520therapy%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2013%26volume%3D12%26spage%3D917%26epage%3D930%26doi%3D10.1038%2Fnrd4154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Audia, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Austin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blagg, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bountra, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buser-Doepner, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahlin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhanak, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frederiksen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimshaw, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hepworth, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kotz, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruger, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mader, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsden, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller-Fahrnow, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Hagan, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overington, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenberg, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundström, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Superti-Furga, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taunton, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toledo-Sherman, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walpole, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walters, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willson, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Workman, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuercher, W. J.</span></span> <span> </span><span class="NLM_article-title">The promise and peril of chemical probes</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">536</span>– <span class="NLM_lpage">541</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1867</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1038%2Fnchembio.1867" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=26196764" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Wlu7zO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=536-541&author=C.+H.+Arrowsmithauthor=J.+E.+Audiaauthor=C.+Austinauthor=J.+Baellauthor=J.+Bennettauthor=J.+Blaggauthor=C.+Bountraauthor=P.+E.+Brennanauthor=P.+J.+Brownauthor=M.+E.+Bunnageauthor=C.+Buser-Doepnerauthor=R.+M.+Campbellauthor=A.+J.+Carterauthor=P.+Cohenauthor=R.+A.+Copelandauthor=B.+Cravattauthor=J.+L.+Dahlinauthor=D.+Dhanakauthor=A.+M.+Edwardsauthor=M.+Frederiksenauthor=S.+V.+Fryeauthor=N.+Grayauthor=C.+E.+Grimshawauthor=D.+Hepworthauthor=T.+Howeauthor=K.+V.+M.+Huberauthor=J.+Jinauthor=S.+Knappauthor=J.+D.+Kotzauthor=R.+G.+Krugerauthor=D.+Loweauthor=M.+M.+Maderauthor=B.+Marsdenauthor=A.+Mueller-Fahrnowauthor=S.+M%C3%BCllerauthor=R.+C.+O%E2%80%99Haganauthor=J.+P.+Overingtonauthor=D.+R.+Owenauthor=S.+H.+Rosenbergauthor=R.+Rossauthor=B.+Rothauthor=M.+Schapiraauthor=S.+L.+Schreiberauthor=B.+Shoichetauthor=M.+Sundstr%C3%B6mauthor=G.+Superti-Furgaauthor=J.+Tauntonauthor=L.+Toledo-Shermanauthor=C.+Walpoleauthor=M.+A.+Waltersauthor=T.+M.+Willsonauthor=P.+Workmanauthor=R.+N.+Youngauthor=W.+J.+Zuercher&title=The+promise+and+peril+of+chemical+probes&doi=10.1038%2Fnchembio.1867"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">The promise and peril of chemical probes</span></div><div class="casAuthors">Arrowsmith, Cheryl H.; Audia, James E.; Austin, Christopher; Baell, Jonathan; Bennett, Jonathan; Blagg, Julian; Bountra, Chas; Brennan, Paul E.; Brown, Peter J.; Bunnage, Mark E.; Buser-Doepner, Carolyn; Campbell, Robert M.; Carter, Adrian J.; Cohen, Philip; Copeland, Robert A.; Cravatt, Ben; Dahlin, Jayme L.; Dhanak, Dashyant; Edwards, Aled M.; Frye, Stephen V.; Gray, Nathanael; Grimshaw, Charles E.; Hepworth, David; Howe, Trevor; Huber, Kilian V. M.; Jin, Jian; Knapp, Stefan; Kotz, Joanne D.; Kruger, Ryan G.; Lowe, Derek; Mader, Mary M.; Marsden, Brian; Mueller-Fahrnow, Anke; Muller, Susanne; O'Hagan, Ronan C.; Overington, John P.; Owen, Dafydd R.; Rosenberg, Saul H.; Roth, Brian; Ross, Ruth; Schapira, Matthieu; Schreiber, Stuart L.; Shoichet, Brian; Sundstrom, Michael; Superti-Furga, Giulio; Taunton, Jack; Toledo-Sherman, Leticia; Walpole, Chris; Walters, Michael A.; Willson, Timothy M.; Workman, Paul; Young, Robert N.; Zuercher, William J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">536-541</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Chem. probes are powerful reagents with increasing impacts on biomedical research.  However, probes of poor quality or that are used incorrectly generate misleading results.  To help address these shortcomings, we will create a community-driven wiki resource to improve quality and convey current best practice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-li4dxq487rVg90H21EOLACvtfcHk0lhAAo-v-yNLtA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Wlu7zO&md5=d8c7a7af94921ab4cdcc871487cceadd</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1867&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1867%26sid%3Dliteratum%253Aachs%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26aulast%3DAustin%26aufirst%3DC.%26aulast%3DBaell%26aufirst%3DJ.%26aulast%3DBennett%26aufirst%3DJ.%26aulast%3DBlagg%26aufirst%3DJ.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DBuser-Doepner%26aufirst%3DC.%26aulast%3DCampbell%26aufirst%3DR.%2BM.%26aulast%3DCarter%26aufirst%3DA.%2BJ.%26aulast%3DCohen%26aufirst%3DP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DCravatt%26aufirst%3DB.%26aulast%3DDahlin%26aufirst%3DJ.%2BL.%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DEdwards%26aufirst%3DA.%2BM.%26aulast%3DFrederiksen%26aufirst%3DM.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DGray%26aufirst%3DN.%26aulast%3DGrimshaw%26aufirst%3DC.%2BE.%26aulast%3DHepworth%26aufirst%3DD.%26aulast%3DHowe%26aufirst%3DT.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DKotz%26aufirst%3DJ.%2BD.%26aulast%3DKruger%26aufirst%3DR.%2BG.%26aulast%3DLowe%26aufirst%3DD.%26aulast%3DMader%26aufirst%3DM.%2BM.%26aulast%3DMarsden%26aufirst%3DB.%26aulast%3DMueller-Fahrnow%26aufirst%3DA.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Hagan%26aufirst%3DR.%2BC.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DRosenberg%26aufirst%3DS.%2BH.%26aulast%3DRoss%26aufirst%3DR.%26aulast%3DRoth%26aufirst%3DB.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DShoichet%26aufirst%3DB.%26aulast%3DSundstr%25C3%25B6m%26aufirst%3DM.%26aulast%3DSuperti-Furga%26aufirst%3DG.%26aulast%3DTaunton%26aufirst%3DJ.%26aulast%3DToledo-Sherman%26aufirst%3DL.%26aulast%3DWalpole%26aufirst%3DC.%26aulast%3DWalters%26aufirst%3DM.%2BA.%26aulast%3DWillson%26aufirst%3DT.%2BM.%26aulast%3DWorkman%26aufirst%3DP.%26aulast%3DYoung%26aufirst%3DR.%2BN.%26aulast%3DZuercher%26aufirst%3DW.%2BJ.%26atitle%3DThe%2520promise%2520and%2520peril%2520of%2520chemical%2520probes%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D536%26epage%3D541%26doi%3D10.1038%2Fnchembio.1867" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.-M.</span></span> <span> </span><span class="NLM_article-title">The bromodomain: a new target in emerging epigenetic medicine</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">598</span>– <span class="NLM_lpage">608</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00831</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00831" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVyit7fJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=598-608&author=S.+G.+Smithauthor=M.-M.+Zhou&title=The+bromodomain%3A+a+new+target+in+emerging+epigenetic+medicine&doi=10.1021%2Facschembio.5b00831"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">The Bromodomain: A New Target in Emerging Epigenetic Medicine</span></div><div class="casAuthors">Smith, Steven G.; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">598-608</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromodomain (BrD) is a conserved protein modular domain found in many chromatin- and transcription-assocd. proteins that has the ability to recognize acetylated lysine residues.  This activity allows bromodomains to play a vital role in many acetylation-mediated protein-protein interactions in the cell, ranging from substrate recruitment for histone acetyltransferases (HATs) to aiding in multiple-protein complex assembly for gene transcriptional activation or suppression in chromatin.  In recent years, considerable efforts have been made to develop chem. inhibitors of these bromodomains in an effort to probe their cellular functions.  Potent and selective inhibitors have been extensively developed for one group of the bromodomain family termed BET proteins that consist of tandem bromodomains followed by an extra terminal domain.  Drug developers are actively designing inhibitors of other bromodomains and working to move the most successful inhibitors into the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTQF2Qjn_iuLVg90H21EOLACvtfcHk0lhJuUuZZxAo3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVyit7fJ&md5=dd34fc591550ff1528e6dccd1d04f1c1</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00831&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00831%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DS.%2BG.%26aulast%3DZhou%26aufirst%3DM.-M.%26atitle%3DThe%2520bromodomain%253A%2520a%2520new%2520target%2520in%2520emerging%2520epigenetic%2520medicine%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D598%26epage%3D608%26doi%3D10.1021%2Facschembio.5b00831" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">James, L. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S. V.</span></span> <span> </span><span class="NLM_article-title">Chemical probes for methyl lysine reader domains</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">135</span>– <span class="NLM_lpage">141</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2016.06.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1016%2Fj.cbpa.2016.06.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=27348158" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsF2qt7jE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2016&pages=135-141&author=L.+I.+Jamesauthor=S.+V.+Frye&title=Chemical+probes+for+methyl+lysine+reader+domains&doi=10.1016%2Fj.cbpa.2016.06.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical probes for methyl lysine reader domains</span></div><div class="casAuthors">James, Lindsey I.; Frye, Stephen V.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">135-141</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The primary intent of a chem. probe is to establish the relationship between a mol. target, usually a protein whose function is modulated by the probe, and the biol. consequences of that modulation.  In order to fulfill this purpose, a chem. probe must be profiled for selectivity, mechanism of action, and cellular activity, as the cell is the minimal system in which 'biol.' can be explored.  This review provides a brief overview of progress towards chem. probes for Me lysine reader domains with a focus on recent progress targeting chromodomains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrv3D9GBmCaebVg90H21EOLACvtfcHk0lhJuUuZZxAo3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsF2qt7jE&md5=e0f49e320758bcdbebf53ffd1fe05883</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2016.06.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2016.06.004%26sid%3Dliteratum%253Aachs%26aulast%3DJames%26aufirst%3DL.%2BI.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26atitle%3DChemical%2520probes%2520for%2520methyl%2520lysine%2520reader%2520domains%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D33%26spage%3D135%26epage%3D141%26doi%3D10.1016%2Fj.cbpa.2016.06.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Milosevich, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hof, F.</span></span> <span> </span><span class="NLM_article-title">Chemical inhibitors of epigenetic methyllysine reader proteins</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">1570</span>– <span class="NLM_lpage">1583</span>, <span class="refDoi"> DOI: 10.1021/acs.biochem.5b01073</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biochem.5b01073" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFOmtrnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=1570-1583&author=N.+Milosevichauthor=F.+Hof&title=Chemical+inhibitors+of+epigenetic+methyllysine+reader+proteins&doi=10.1021%2Facs.biochem.5b01073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical Inhibitors of Epigenetic Methyllysine Reader Proteins</span></div><div class="casAuthors">Milosevich, Natalia; Hof, Fraser</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1570-1583</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein methylation is a common post-translational modification with diverse biol. functions.  Methyllysine reader proteins are increasingly a focus of epigenetics research and play important roles in regulating many cellular processes.  These reader proteins are vital players in development, cell cycle regulation, stress responses, oncogenesis, and other disease pathways.  The recent emergence of a small no. of chem. inhibitors for methyllysine reader proteins supports the viability of these proteins as targets for drug development.  This article introduces the biochem. and biol. of methyllysine reader proteins, provides an overview of functions for those families of readers that have been targeted to date (MBT, PHD, tudor, and chromodomains), and reviews the development of synthetic agents that directly block their methyllysine reading functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGps23HnAOk3T7Vg90H21EOLACvtfcHk0lhJuUuZZxAo3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFOmtrnM&md5=0d9ec539ad7b345bec4ecb8a252a9ff8</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Facs.biochem.5b01073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biochem.5b01073%26sid%3Dliteratum%253Aachs%26aulast%3DMilosevich%26aufirst%3DN.%26aulast%3DHof%26aufirst%3DF.%26atitle%3DChemical%2520inhibitors%2520of%2520epigenetic%2520methyllysine%2520reader%2520proteins%26jtitle%3DBiochemistry%26date%3D2016%26volume%3D55%26spage%3D1570%26epage%3D1583%26doi%3D10.1021%2Facs.biochem.5b01073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Greschik, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schüle, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Günther, T.</span></span> <span> </span><span class="NLM_article-title">Selective targeting of epigenetic reader domains</span>. <i>Expert Opin. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">449</span>– <span class="NLM_lpage">463</span>, <span class="refDoi"> DOI: 10.1080/17460441.2017.1303474</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1080%2F17460441.2017.1303474" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=28277835" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC2sXktlKmt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=449-463&author=H.+Greschikauthor=R.+Sch%C3%BCleauthor=T.+G%C3%BCnther&title=Selective+targeting+of+epigenetic+reader+domains&doi=10.1080%2F17460441.2017.1303474"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Selective targeting of epigenetic reader domains</span></div><div class="casAuthors">Greschik, Holger; Schuele, Roland; Guenther, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">449-463</span>CODEN:
                <span class="NLM_cas:coden">EODDBX</span>;
        ISSN:<span class="NLM_cas:issn">1746-0441</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  Epigenetic regulators including writers, erasers, and readers of chromatin marks have been implicated in numerous diseases and are therefore subject of intense academic and pharmaceutical research.  While several small-mol. inhibitors targeting writers or erasers are either approved drugs or are currently being evaluated in clin. trials, the targeting of epigenetic readers has lagged behind.  Proof-of-principle that epigenetic readers are also relevant drug targets was provided by landmark discoveries of selective inhibitors targeting the BET family of acetyl-lysine readers.  More recently, high affinity chem. probes for non-BET acetyl- and methyl-lysine reader domains have also been developed.  This article covers recent advances with the identification and validation of inhibitors and chem. probes targeting epigenetic reader domains.  Issues related to epigenetic reader druggability, quality requirements for chem. probes, interpretation of cellular action, unexpected cross-talk, and future challenges are also discussed.  Chem. probes provide a powerful means to unravel biol. functions of epigenetic readers and evaluate their potential as drug targets.  To yield meaningful results, potency, selectivity, and cellular target engagement of chem. probes need to be stringently validated.  Future chem. probes will probably need to fulfil addnl. criteria such as strict target specificity or the targeting of readers within protein complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpN_FtZE2UW-rVg90H21EOLACvtfcHk0lhJuUuZZxAo3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXktlKmt70%253D&md5=a060b82da46d848a3713131dd3b438b9</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1080%2F17460441.2017.1303474&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F17460441.2017.1303474%26sid%3Dliteratum%253Aachs%26aulast%3DGreschik%26aufirst%3DH.%26aulast%3DSch%25C3%25BCle%26aufirst%3DR.%26aulast%3DG%25C3%25BCnther%26aufirst%3DT.%26atitle%3DSelective%2520targeting%2520of%2520epigenetic%2520reader%2520domains%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2017%26volume%3D12%26spage%3D449%26epage%3D463%26doi%3D10.1080%2F17460441.2017.1303474" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zaware, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.-M.</span></span> <span> </span><span class="NLM_article-title">Chemical modulators for epigenome reader domains as emerging epigenetic therapies for cancer and inflammation</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">116</span>– <span class="NLM_lpage">125</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2017.06.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1016%2Fj.cbpa.2017.06.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=28689146" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFChtbfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2017&pages=116-125&author=N.+Zawareauthor=M.-M.+Zhou&title=Chemical+modulators+for+epigenome+reader+domains+as+emerging+epigenetic+therapies+for+cancer+and+inflammation&doi=10.1016%2Fj.cbpa.2017.06.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical modulators for epigenome reader domains as emerging epigenetic therapies for cancer and inflammation</span></div><div class="casAuthors">Zaware, Nilesh; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">116-125</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Site-specific lysine acetylation and methylation on histones are crit. post-translational modifications (PTMs) that govern ordered gene transcription in chromatin.  Mis-regulation of these histone PTM-mediated processes has been shown to be assocd. with human diseases.  Since the 2010 landmark reports of small mols. (+)-JQ1 and I-BET762 that target the acetyl-lysine 'reader' Bromodomain and Extra Terminal domain (BET) proteins, there have been relentless efforts to develop epigenetic therapy with small mols. to modulate mol. interactions of epigenome reader domain proteins with PTMs.  In addn. to BET, the other emerging targets include non-BET acetyl-lysine and methyl-lysine reader domains.  This review covers the key chem. modulators of the aforementioned epigenome reader proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGps_AJz3UXkTrVg90H21EOLACvtfcHk0lj_HQxBsAU38g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFChtbfF&md5=b33b5288f202e4b2e4ccf1f7b92bf654</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2017.06.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2017.06.012%26sid%3Dliteratum%253Aachs%26aulast%3DZaware%26aufirst%3DN.%26aulast%3DZhou%26aufirst%3DM.-M.%26atitle%3DChemical%2520modulators%2520for%2520epigenome%2520reader%2520domains%2520as%2520emerging%2520epigenetic%2520therapies%2520for%2520cancer%2520and%2520inflammation%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D39%26spage%3D116%26epage%3D125%26doi%3D10.1016%2Fj.cbpa.2017.06.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">James, L. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krichevsky, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korboukh, V. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herold, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacNevin, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norris, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sagum, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tempel, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcon, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emili, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenblatt, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kireev, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janzen, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S. V.</span></span> <span> </span><span class="NLM_article-title">Discovery of a chemical probe for the L3MBTL3 methyllysine reader domain</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">184</span>– <span class="NLM_lpage">191</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1157</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1038%2Fnchembio.1157" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=23292653" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjslGjsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=184-191&author=L.+I.+Jamesauthor=D.+Barsyte-Lovejoyauthor=N.+Zhongauthor=L.+Krichevskyauthor=V.+K.+Korboukhauthor=J.+M.+Heroldauthor=C.+J.+MacNevinauthor=J.+L.+Norrisauthor=C.+A.+Sagumauthor=W.+Tempelauthor=E.+Marconauthor=H.+Guoauthor=C.+Gaoauthor=X.-P.+Huangauthor=S.+Duanauthor=A.+Emiliauthor=J.+F.+Greenblattauthor=D.+B.+Kireevauthor=J.+Jinauthor=W.+P.+Janzenauthor=P.+J.+Brownauthor=M.+T.+Bedfordauthor=C.+H.+Arrowsmithauthor=S.+V.+Frye&title=Discovery+of+a+chemical+probe+for+the+L3MBTL3+methyllysine+reader+domain&doi=10.1038%2Fnchembio.1157"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a chemical probe for the L3MBTL3 methyllysine reader domain</span></div><div class="casAuthors">James, Lindsey I.; Barsyte-Lovejoy, Dalia; Zhong, Nan; Krichevsky, Liubov; Korboukh, Victoria K.; Herold, J. Martin; MacNevin, Christopher J.; Norris, Jacqueline L.; Sagum, Cari A.; Tempel, Wolfram; Marcon, Edyta; Guo, Hongbo; Gao, Cen; Huang, Xi-Ping; Duan, Shili; Emili, Andrew; Greenblatt, Jack F.; Kireev, Dmitri B.; Jin, Jian; Janzen, William P.; Brown, Peter J.; Bedford, Mark T.; Arrowsmith, Cheryl H.; Frye, Stephen V.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">184-191</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We describe the discovery of UNC1215, a potent and selective chem. probe for the methyllysine (Kme) reading function of L3MBTL3, a member of the malignant brain tumor (MBT) family of chromatin-interacting transcriptional repressors.  UNC1215 binds L3MBTL3 with a Kd of 120 nM, competitively displacing mono- or dimethyllysine-contg. peptides, and is greater than 50-fold more potent toward L3MBTL3 than other members of the MBT family while also demonstrating selectivity against more than 200 other reader domains examd.  X-ray crystallog. identified a unique 2:2 polyvalent mode of interaction between UNC1215 and L3MBTL3.  In cells, UNC1215 is nontoxic and directly binds L3MBTL3 via the Kme-binding pocket of the MBT domains.  UNC1215 increases the cellular mobility of GFP-L3MBTL3 fusion proteins, and point mutants that disrupt the Kme-binding function of GFP-L3MBTL3 phenocopy the effects of UNC1215 on localization.  Finally, UNC1215 was used to reveal a new Kme-dependent interaction of L3MBTL3 with BCLAF1 (BCL2-assocd. transcription factor 1), a protein implicated in DNA damage repair and apoptosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqta8WQQkm04rVg90H21EOLACvtfcHk0lj_HQxBsAU38g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjslGjsw%253D%253D&md5=ea8779ddceed07b5e4c822f2eb888acb</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1157&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1157%26sid%3Dliteratum%253Aachs%26aulast%3DJames%26aufirst%3DL.%2BI.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DZhong%26aufirst%3DN.%26aulast%3DKrichevsky%26aufirst%3DL.%26aulast%3DKorboukh%26aufirst%3DV.%2BK.%26aulast%3DHerold%26aufirst%3DJ.%2BM.%26aulast%3DMacNevin%26aufirst%3DC.%2BJ.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DSagum%26aufirst%3DC.%2BA.%26aulast%3DTempel%26aufirst%3DW.%26aulast%3DMarcon%26aufirst%3DE.%26aulast%3DGuo%26aufirst%3DH.%26aulast%3DGao%26aufirst%3DC.%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DDuan%26aufirst%3DS.%26aulast%3DEmili%26aufirst%3DA.%26aulast%3DGreenblatt%26aufirst%3DJ.%2BF.%26aulast%3DKireev%26aufirst%3DD.%2BB.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26atitle%3DDiscovery%2520of%2520a%2520chemical%2520probe%2520for%2520the%2520L3MBTL3%2520methyllysine%2520reader%2520domain%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2013%26volume%3D9%26spage%3D184%26epage%3D191%26doi%3D10.1038%2Fnchembio.1157" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dickson, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norris, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cholensky, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tempel, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sagum, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Black, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baughman, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senisterra, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pattenden, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, L. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S. V.</span></span> <span> </span><span class="NLM_article-title">A cellular chemical probe targeting the chromodomains of polycomb repressive complex 1</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">180</span>– <span class="NLM_lpage">187</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1038%2Fnchembio.2007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=26807715" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFyis7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=180-187&author=J.+I.+Stuckeyauthor=B.+M.+Dicksonauthor=N.+Chengauthor=Y.+Liuauthor=J.+L.+Norrisauthor=S.+H.+Cholenskyauthor=W.+Tempelauthor=S.+Qinauthor=K.+G.+Huberauthor=C.+Sagumauthor=K.+Blackauthor=F.+Liauthor=X.-P.+Huangauthor=B.+L.+Rothauthor=B.+M.+Baughmanauthor=G.+Senisterraauthor=S.+G.+Pattendenauthor=M.+Vedadiauthor=P.+J.+Brownauthor=M.+T.+Bedfordauthor=J.+Minauthor=C.+H.+Arrowsmithauthor=L.+I.+Jamesauthor=S.+V.+Frye&title=A+cellular+chemical+probe+targeting+the+chromodomains+of+polycomb+repressive+complex+1&doi=10.1038%2Fnchembio.2007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">A cellular chemical probe targeting the chromodomains of Polycomb repressive complex 1</span></div><div class="casAuthors">Stuckey, Jacob I.; Dickson, Bradley M.; Cheng, Nancy; Liu, Yanli; Norris, Jacqueline L.; Cholensky, Stephanie H.; Tempel, Wolfram; Qin, Su; Huber, Katherine G.; Sagum, Cari; Black, Karynne; Li, Fengling; Huang, Xi-Ping; Roth, Bryan L.; Baughman, Brandi M.; Senisterra, Guillermo; Pattenden, Samantha G.; Vedadi, Masoud; Brown, Peter J.; Bedford, Mark T.; Min, Jinrong; Arrowsmith, Cheryl H.; James, Lindsey I.; Frye, Stephen V.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">180-187</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We report the design and characterization of UNC3866, a potent antagonist of the methyllysine (Kme) reading function of the Polycomb CBX and CDY families of chromodomains.  Polycomb CBX proteins regulate gene expression by targeting Polycomb repressive complex 1 (PRC1) to sites of H3K27me3 via their chromodomains.  UNC3866 binds the chromodomains of CBX4 and CBX7 most potently, with a Kd of ∼100 nM for each, and is 6- to 18-fold selective as compared to seven other CBX and CDY chromodomains while being highly selective over >250 other protein targets.  X-ray crystallog. revealed that UNC3866's interactions with the CBX chromodomains closely mimic those of the methylated H3 tail.  UNC4195, a biotinylated deriv. of UNC3866, was used to demonstrate that UNC3866 engages intact PRC1 and that EED incorporation into PRC1 is isoform dependent in PC3 prostate cancer cells.  Finally, UNC3866 inhibits PC3 cell proliferation, consistent with the known ability of CBX7 overexpression to confer a growth advantage, whereas UNC4219, a methylated neg. control compd., has negligible effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobRZ8YoHZWk7Vg90H21EOLACvtfcHk0lj_HQxBsAU38g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFyis7o%253D&md5=97fe8dd8e35094a8c91d5d4bdc265b4f</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2007%26sid%3Dliteratum%253Aachs%26aulast%3DStuckey%26aufirst%3DJ.%2BI.%26aulast%3DDickson%26aufirst%3DB.%2BM.%26aulast%3DCheng%26aufirst%3DN.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DCholensky%26aufirst%3DS.%2BH.%26aulast%3DTempel%26aufirst%3DW.%26aulast%3DQin%26aufirst%3DS.%26aulast%3DHuber%26aufirst%3DK.%2BG.%26aulast%3DSagum%26aufirst%3DC.%26aulast%3DBlack%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DBaughman%26aufirst%3DB.%2BM.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DPattenden%26aufirst%3DS.%2BG.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DMin%26aufirst%3DJ.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DJames%26aufirst%3DL.%2BI.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26atitle%3DA%2520cellular%2520chemical%2520probe%2520targeting%2520the%2520chromodomains%2520of%2520polycomb%2520repressive%2520complex%25201%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3D180%26epage%3D187%26doi%3D10.1038%2Fnchembio.2007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greschik, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgahn, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidtkunz, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schott, A.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMillan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baranauskienė, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oppermann, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matulis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schüle, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, M.</span></span> <span> </span><span class="NLM_article-title">Identification of a small-molecule ligand of the epigenetic reader protein Spindlin1 via a versatile screening platform</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>44</i></span>, <span class="NLM_elocation-id">e88</span> <span class="refDoi"> DOI: 10.1093/nar/gkw089</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1093%2Fnar%2Fgkw089" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=26893353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslCqurvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&author=T.+Wagnerauthor=H.+Greschikauthor=T.+Burgahnauthor=K.+Schmidtkunzauthor=A.-K.+Schottauthor=J.+McMillanauthor=L.+Baranauskien%C4%97author=Y.+Xiongauthor=O.+Fedorovauthor=J.+Jinauthor=U.+Oppermannauthor=D.+Matulisauthor=R.+Sch%C3%BCleauthor=M.+Jung&title=Identification+of+a+small-molecule+ligand+of+the+epigenetic+reader+protein+Spindlin1+via+a+versatile+screening+platform&doi=10.1093%2Fnar%2Fgkw089"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a small-molecule ligand of the epigenetic reader protein Spindlin1 via a versatile screening platform</span></div><div class="casAuthors">Wagner, Tobias; Greschik, Holger; Burgahn, Teresa; Schmidtkunz, Karin; Schott, Anne-Kathrin; McMillan, Joel; Baranauskiene, Lina; Xiong, Yan; Fedorov, Oleg; Jin, Jian; Oppermann, Udo; Matulis, Daumantas; Schuele, Roland; Jung, Manfred</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">e88/1-e88/22</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Epigenetic modifications of histone tails play an essential role in the regulation of eukaryotic transcription.  Writer and eraser enzymes establish and maintain the epigenetic code by creating or removing posttranslational marks.  Specific binding proteins, called readers, recognize the modifications and mediate epigenetic signalling.  Here, we present a versatile assay platform for the investigation of the interaction between Me lysine readers and their ligands.  This can be utilized for the screening of small mol. inhibitors of such protein-protein interactions and the detailed characterization of the inhibition.  Our platform is constructed in a modular way consisting of orthogonal in vitro binding assays for ligand screening and verification of initial hits and biophys., label-free techniques for further kinetic characterization of confirmed ligands.  A stability assay for the investigation of target engagement in a cellular context complements the platform.  We applied the complete evaluation chain to the Tudor domain contg. protein Spindlin1 and established the in vitro test systems for the double Tudor domain of the histone demethylase JMJD2C.  We finally conducted an exploratory screen for inhibitors of the interaction between Spindlin1 and H3K4me3 and identified A366 as the first nanomolar small-mol. ligand of a Tudor domain contg. Me lysine reader.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGql-bO41bi49bVg90H21EOLACvtfcHk0ljCab4Ub2PpXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslCqurvK&md5=3c6536be1ee6f83c5d2849baa02da318</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkw089&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkw089%26sid%3Dliteratum%253Aachs%26aulast%3DWagner%26aufirst%3DT.%26aulast%3DGreschik%26aufirst%3DH.%26aulast%3DBurgahn%26aufirst%3DT.%26aulast%3DSchmidtkunz%26aufirst%3DK.%26aulast%3DSchott%26aufirst%3DA.-K.%26aulast%3DMcMillan%26aufirst%3DJ.%26aulast%3DBaranauskien%25C4%2597%26aufirst%3DL.%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DOppermann%26aufirst%3DU.%26aulast%3DMatulis%26aufirst%3DD.%26aulast%3DSch%25C3%25BCle%26aufirst%3DR.%26aulast%3DJung%26aufirst%3DM.%26atitle%3DIdentification%2520of%2520a%2520small-molecule%2520ligand%2520of%2520the%2520epigenetic%2520reader%2520protein%2520Spindlin1%2520via%2520a%2520versatile%2520screening%2520platform%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2016%26volume%3D44%26doi%3D10.1093%2Fnar%2Fgkw089" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bae, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viviano, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lv, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sagum, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castellano, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khalil, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sbardella, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span> <span> </span><span class="NLM_article-title">Developing Spindlin1 small-molecule inhibitors by using protein microarrays</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">750</span>– <span class="NLM_lpage">756</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2377</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1038%2Fnchembio.2377" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=28504676" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnslKktrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=750-756&author=N.+Baeauthor=M.+Vivianoauthor=X.+Suauthor=J.+Lvauthor=D.+Chengauthor=C.+Sagumauthor=S.+Castellanoauthor=X.+Baiauthor=C.+Johnsonauthor=M.+I.+Khalilauthor=J.+Shenauthor=K.+Chenauthor=H.+Liauthor=G.+Sbardellaauthor=M.+T.+Bedford&title=Developing+Spindlin1+small-molecule+inhibitors+by+using+protein+microarrays&doi=10.1038%2Fnchembio.2377"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Developing Spindlin1 small-molecule inhibitors by using protein microarrays</span></div><div class="casAuthors">Bae, Narkhyun; Viviano, Monica; Su, Xiaonan; Lv, Jie; Cheng, Donghang; Sagum, Cari; Castellano, Sabrina; Bai, Xue; Johnson, Claire; Khalil, Mahmoud Ibrahim; Shen, Jianjun; Chen, Kaifu; Li, Haitao; Sbardella, Gianluca; Bedford, Mark T.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">750-756</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The discovery of inhibitors of methyl- and acetyl-binding domains has provided evidence for the 'druggability' of epigenetic effector mols.  The small-mol. probe, UNC1215, prevents methyl-dependent protein-protein interactions by engaging the arom. cage of MBT domains and, with lower affinity, Tudor domains.  Here, using a library of tagged UNC1215 analogs, we screened a protein-domain microarray of human methyllysine effector mols. to rapidly detect compds. with new binding profiles with either increased or decreased specificity.  Using this approach, we identified a compd. (EML405) that acquired a novel interaction with Tudor-domain-contg. protein Spindlin1 (SPIN1).  Structural studies facilitated the rational synthesis of SPIN1 inhibitors with increased selectivity (EML631-633), which engaged SPIN1 in cells, blocked its ability to 'read' H3K4me3 marks, and inhibited its transcriptional-coactivator activity.  Protein microarrays can thus be used as a platform to 'target-hop' and identify small mols. that bind and compete with domain-motif interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6diBsmZCcm7Vg90H21EOLACvtfcHk0ljCab4Ub2PpXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnslKktrc%253D&md5=5ac8aab47d4a67cbe0f7c1f04d4ae410</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2377&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2377%26sid%3Dliteratum%253Aachs%26aulast%3DBae%26aufirst%3DN.%26aulast%3DViviano%26aufirst%3DM.%26aulast%3DSu%26aufirst%3DX.%26aulast%3DLv%26aufirst%3DJ.%26aulast%3DCheng%26aufirst%3DD.%26aulast%3DSagum%26aufirst%3DC.%26aulast%3DCastellano%26aufirst%3DS.%26aulast%3DBai%26aufirst%3DX.%26aulast%3DJohnson%26aufirst%3DC.%26aulast%3DKhalil%26aufirst%3DM.%2BI.%26aulast%3DShen%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DSbardella%26aufirst%3DG.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DDeveloping%2520Spindlin1%2520small-molecule%2520inhibitors%2520by%2520using%2520protein%2520microarrays%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26spage%3D750%26epage%3D756%26doi%3D10.1038%2Fnchembio.2377" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rong, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartlam, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Z.</span></span> <span> </span><span class="NLM_article-title">Structure of human Spindlin1</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>282</i></span>,  <span class="NLM_fpage">647</span>– <span class="NLM_lpage">656</span>, <span class="refDoi"> DOI: 10.1074/jbc.m604029200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1074%2Fjbc.m604029200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=17082182" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BD2sXkslej" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=2007&pages=647-656&author=Q.+Zhaoauthor=L.+Qinauthor=F.+Jiangauthor=B.+Wuauthor=W.+Yueauthor=F.+Xuauthor=Z.+Rongauthor=H.+Yuanauthor=X.+Xieauthor=Y.+Gaoauthor=C.+Baiauthor=M.+Bartlamauthor=X.+Peiauthor=Z.+Rao&title=Structure+of+human+Spindlin1&doi=10.1074%2Fjbc.m604029200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of Human Spindlin1: Tandem Tudor-Like Domains for Cell Cycle Regulation</span></div><div class="casAuthors">Zhao, Qiang; Qin, Lipeng; Jiang, Fuguo; Wu, Beili; Yue, Wen; Xu, Feng; Rong, Zhili; Yuan, Hongfeng; Xie, Xiaoyan; Gao, Yanhong; Bai, Cixian; Bartlam, Mark; Pei, Xuetao; Rao, Zihe</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">282</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">647-656</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Spindlin1, a meiotic spindle-binding protein that is highly expressed in ovarian cancer cells, was first identified as a gene involved in gametogenesis.  It appeared to be a target for cell cycle-dependent phosphorylation and was demonstrated to disturb the cell cycle.  Here we report the crystal structure of human spindlin1 to 2.2Å of resoln., representing the first three-dimensional structure from the spin/ssty (Y-linked spermiogenesis-specific transcript) gene family.  The refined structure, contg. three repeats of five/four anti-parallel β-strands, exhibits a novel arrangement of tandem Tudor-like domains.  Two phosphate ions, chelated by Thr-95 and other residues, appear to stabilize the long loop between domains I and II, which might mediate the cell cycle regulation activity of spindlin1.  Flow cytometry expts. indicate that cells expressing spindlin1 display a different cell cycle distribution in mitosis, whereas those expressing a T95A mutant, which had a great decrease in phosphorous content, have little effect on the cell cycle.  We further identified assocns. of spindlin1 with nucleic acid to provide a biochem. basis for its cell cycle regulation and other functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGfMhFJXqtRLVg90H21EOLACvtfcHk0ljsL_P0RO558g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXkslej&md5=14747516da6b164560f2d1384002d6e5</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1074%2Fjbc.m604029200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.m604029200%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DQ.%26aulast%3DQin%26aufirst%3DL.%26aulast%3DJiang%26aufirst%3DF.%26aulast%3DWu%26aufirst%3DB.%26aulast%3DYue%26aufirst%3DW.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DRong%26aufirst%3DZ.%26aulast%3DYuan%26aufirst%3DH.%26aulast%3DXie%26aufirst%3DX.%26aulast%3DGao%26aufirst%3DY.%26aulast%3DBai%26aufirst%3DC.%26aulast%3DBartlam%26aufirst%3DM.%26aulast%3DPei%26aufirst%3DX.%26aulast%3DRao%26aufirst%3DZ.%26atitle%3DStructure%2520of%2520human%2520Spindlin1%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2007%26volume%3D282%26spage%3D647%26epage%3D656%26doi%3D10.1074%2Fjbc.m604029200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, R.-M.</span></span> <span> </span><span class="NLM_article-title">Distinct mode of methylated lysine-4 of histone H3 recognition by tandem tudor-like domains of Spindlin1</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">17954</span>– <span class="NLM_lpage">17959</span>, <span class="refDoi"> DOI: 10.1073/pnas.1208517109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1073%2Fpnas.1208517109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=23077255" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhsl2ktbvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=17954-17959&author=N.+Yangauthor=W.+Wangauthor=Y.+Wangauthor=M.+Wangauthor=Q.+Zhaoauthor=Z.+Raoauthor=B.+Zhuauthor=R.-M.+Xu&title=Distinct+mode+of+methylated+lysine-4+of+histone+H3+recognition+by+tandem+tudor-like+domains+of+Spindlin1&doi=10.1073%2Fpnas.1208517109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Distinct mode of methylated lysine-4 of histone H3 recognition by tandem tudor-like domains of Spindlin1</span></div><div class="casAuthors">Yang, Na; Wang, Weixiang; Wang, Yan; Wang, Mingzhu; Zhao, Qiang; Rao, Zihe; Zhu, Bing; Xu, Rui-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">17954-17959, S17954/1-S17954/5</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Recognition of methylated histone tail lysine residues by tudor domains plays important roles in epigenetic control of gene expression and DNA damage response.  Previous studies revealed the binding of methyllysine in a cage of arom. residues, but the mol. mechanism by which the sequence specificity for surrounding histone tail residues is achieved remains poorly understood.  In the crystal structure of a trimethylated histone H3 lysine 4 (H3K4) peptide bound to the tudor-like domains of Spindlin1 presented here, an atypical mode of methyllysine recognition by an arom. pocket of Spindlin1 is obsd.  Furthermore, the histone sequence is recognized in a distinct manner involving the amino terminus and a pair of arginine residues of histone H3, and disruption of the binding impaired stimulation of pre-RNA expression by Spindlin1.  Our anal. demonstrates considerable diversities of methyllysine recognition and sequence-specific binding of histone tails by tudor domains, and the revelation furthers the understanding of tudor domain proteins in deciphering epigenetic marks on histone tails.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrVCXmD_E2cLVg90H21EOLACvtfcHk0ljsL_P0RO558g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhsl2ktbvK&md5=e785fc0c54885f0188965aea7ef2cfbb</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1208517109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1208517109%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DN.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DZhao%26aufirst%3DQ.%26aulast%3DRao%26aufirst%3DZ.%26aulast%3DZhu%26aufirst%3DB.%26aulast%3DXu%26aufirst%3DR.-M.%26atitle%3DDistinct%2520mode%2520of%2520methylated%2520lysine-4%2520of%2520histone%2520H3%2520recognition%2520by%2520tandem%2520tudor-like%2520domains%2520of%2520Spindlin1%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2012%26volume%3D109%26spage%3D17954%26epage%3D17959%26doi%3D10.1073%2Fpnas.1208517109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Su, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span> <span> </span><span class="NLM_article-title">Molecular basis underlying histone H3 lysine-arginine methylation pattern readout by Spin/Ssty repeats of Spindlin1</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">622</span>– <span class="NLM_lpage">636</span>, <span class="refDoi"> DOI: 10.1101/gad.233239.113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1101%2Fgad.233239.113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=24589551" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC2cXlsF2ntr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2014&pages=622-636&author=X.+Suauthor=G.+Zhuauthor=X.+Dingauthor=S.+Y.+Leeauthor=Y.+Douauthor=B.+Zhuauthor=W.+Wuauthor=H.+Li&title=Molecular+basis+underlying+histone+H3+lysine-arginine+methylation+pattern+readout+by+Spin%2FSsty+repeats+of+Spindlin1&doi=10.1101%2Fgad.233239.113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular basis underlying histone H3 lysine-arginine methylation pattern readout by Spin/Ssty repeats of Spindlin1</span></div><div class="casAuthors">Su, Xiaonan; Zhu, Guixin; Ding, Xiaozhe; Lee, Shirley Y.; Dou, Yali; Zhu, Bing; Wu, Wei; Li, Haitao</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">622-636</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Histone modification patterns and their combinatorial readout have emerged as a fundamental mechanism for epigenetic regulation.  Here the authors characterized Spindlin1 as a histone effector that senses a cis-tail histone H3 methylation pattern involving trimethyllysine 4 (H3K4me3) and asym. dimethylarginine 8 (H3R8me2a) marks.  Spindlin1 consists of triple tudor-like Spin/Ssty repeats.  Cocrystal structure detn. established concurrent recognition of H3K4me3 and H3R8me2a by Spin/Ssty repeats 2 and 1, resp.  Both H3K4me3 and H3R8me2a are recognized using an "insertion cavity" recognition mode, contributing to a methylation state-specific layer of regulation.  In vivo functional studies suggest that Spindlin1 activates Wnt/β-catenin signaling downstream from protein arginine methyltransferase 2 (PRMT2) and the MLL complex, which together are capable of generating a specific H3 "K4me3-R8me2a" pattern.  Mutagenesis of Spindlin1 reader pockets impairs activation of Wnt target genes.  Taken together, this work connects a histone "lysine-arginine" methylation pattern readout by Spindlin1-to-Wnt signaling at the transcriptional level.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2eotrU6XRPrVg90H21EOLACvtfcHk0ljsL_P0RO558g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXlsF2ntr8%253D&md5=6872607b2651d15d16b80a6d33555bbd</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1101%2Fgad.233239.113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.233239.113%26sid%3Dliteratum%253Aachs%26aulast%3DSu%26aufirst%3DX.%26aulast%3DZhu%26aufirst%3DG.%26aulast%3DDing%26aufirst%3DX.%26aulast%3DLee%26aufirst%3DS.%2BY.%26aulast%3DDou%26aufirst%3DY.%26aulast%3DZhu%26aufirst%3DB.%26aulast%3DWu%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DH.%26atitle%3DMolecular%2520basis%2520underlying%2520histone%2520H3%2520lysine-arginine%2520methylation%2520pattern%2520readout%2520by%2520Spin%252FSsty%2520repeats%2520of%2520Spindlin1%26jtitle%3DGenes%2520Dev.%26date%3D2014%26volume%3D28%26spage%3D622%26epage%3D636%26doi%3D10.1101%2Fgad.233239.113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Z.</span></span> <span> </span><span class="NLM_article-title">Expression, purification, crystallization and preliminary X-ray analysis of human Spindlin1, an ovarian cancer-related protein</span>. <i>Protein Pept. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">203</span>– <span class="NLM_lpage">205</span>, <span class="refDoi"> DOI: 10.2174/092986606775101661</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.2174%2F092986606775101661" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=16472086" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BD28XitlGntr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2006&pages=203-205&author=F.+Jiangauthor=Q.+Zhaoauthor=L.+Qinauthor=H.+Pangauthor=X.+Peiauthor=Z.+Rao&title=Expression%2C+purification%2C+crystallization+and+preliminary+X-ray+analysis+of+human+Spindlin1%2C+an+ovarian+cancer-related+protein&doi=10.2174%2F092986606775101661"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Expression, purification, crystallization and preliminary X-ray analysis of human spindlin 1, an ovarian cancer-related protein</span></div><div class="casAuthors">Jiang, Fuguo; Zhao, Qiang; Qin, Lipeng; Pang, Hai; Pei, Xuetao; Rao, Zihe</div><div class="citationInfo"><span class="NLM_cas:title">Protein & Peptide Letters</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">203-205</span>CODEN:
                <span class="NLM_cas:coden">PPELEN</span>;
        ISSN:<span class="NLM_cas:issn">0929-8665</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">Human spindlin 1 is a newly screened and identified gene product related to ovarian carcinomas and is highly homologous to mouse spindlin.  It is an abundant maternal transcript expressed in the mouse during the transition from oocyte to embryo.  Here, the recombinant human spindlin1 has been overexpressed in Escherichia coli BL21, purified and crystd. using the hanging-drop vapor-diffusion method.  Crystals diffracting to 2.25 Å resoln. were obtained using ammonium sulfate as precipitant.  The crystals belong to the space group P212121, with unit-cell parameters a = 40.7 Å, b = 84.4 Å, c = 136.4 Å, α=β=γ=90°.  Assuming two mols. per asym. unit, the solvent content is calcd. to be 42.4%.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8o5Sz0E4XxrVg90H21EOLACvtfcHk0liQ2p2T3b0IlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XitlGntr8%253D&md5=e0968bde24c824de9633702979233a39</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.2174%2F092986606775101661&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F092986606775101661%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DF.%26aulast%3DZhao%26aufirst%3DQ.%26aulast%3DQin%26aufirst%3DL.%26aulast%3DPang%26aufirst%3DH.%26aulast%3DPei%26aufirst%3DX.%26aulast%3DRao%26aufirst%3DZ.%26atitle%3DExpression%252C%2520purification%252C%2520crystallization%2520and%2520preliminary%2520X-ray%2520analysis%2520of%2520human%2520Spindlin1%252C%2520an%2520ovarian%2520cancer-related%2520protein%26jtitle%3DProtein%2520Pept.%2520Lett.%26date%3D2006%26volume%3D13%26spage%3D203%26epage%3D205%26doi%3D10.2174%2F092986606775101661" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Franz, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greschik, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willmann, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozretic, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jilg, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wardelmann, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buettner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schule, R.</span></span> <span> </span><span class="NLM_article-title">The histone code reader SPIN1 controls RET signaling in liposarcoma</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">4773</span>– <span class="NLM_lpage">4789</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.3000</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.18632%2Foncotarget.3000" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=25749382" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A280%3ADC%252BC2MnitFCluw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=4773-4789&author=H.+Franzauthor=H.+Greschikauthor=D.+Willmannauthor=L.+Ozreticauthor=C.+A.+Jilgauthor=E.+Wardelmannauthor=M.+Jungauthor=R.+Buettnerauthor=R.+Schule&title=The+histone+code+reader+SPIN1+controls+RET+signaling+in+liposarcoma&doi=10.18632%2Foncotarget.3000"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">The histone code reader SPIN1 controls RET signaling in liposarcoma</span></div><div class="casAuthors">Franz Henriette; Greschik Holger; Willmann Dominica; Jilg Cordula Annette; Schule Roland; Ozretic Luka; Buettner Reinhard; Wardelmann Eva; Jung Manfred; Jung Manfred; Schule Roland; Schule Roland</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">4773-89</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The histone code reader Spindlin1 (SPIN1) has been implicated in tumorigenesis and tumor growth, but the underlying molecular mechanisms remain poorly understood.  Here, we show that reducing SPIN1 levels strongly impairs proliferation and increases apoptosis of liposarcoma cells in vitro and in xenograft mouse models.  Combining signaling pathway, genome-wide chromatin binding, and transcriptome analyses, we found that SPIN1 directly enhances expression of GDNF, an activator of the RET signaling pathway, in cooperation with the transcription factor MAZ.  Accordingly, knockdown of SPIN1 or MAZ results in reduced levels of GDNF and activated RET explaining diminished liposarcoma cell proliferation and survival.  In line with these observations, levels of SPIN1, GDNF, activated RET, and MAZ are increased in human liposarcoma compared to normal adipose tissue or lipoma.  Importantly, a mutation of SPIN1 within the reader domain interfering with chromatin binding reduces liposarcoma cell proliferation and survival.  Together, our data describe a molecular mechanism for SPIN1 function in liposarcoma and suggest that targeting SPIN1 chromatin association with small molecule inhibitors may represent a novel therapeutic strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS5Ma1donXoAsNIRw0GKUoEfW6udTcc2eYrsFQMdO9gF7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MnitFCluw%253D%253D&md5=4b404511399ebfdf8a553af0822e522f</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.3000&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.3000%26sid%3Dliteratum%253Aachs%26aulast%3DFranz%26aufirst%3DH.%26aulast%3DGreschik%26aufirst%3DH.%26aulast%3DWillmann%26aufirst%3DD.%26aulast%3DOzretic%26aufirst%3DL.%26aulast%3DJilg%26aufirst%3DC.%2BA.%26aulast%3DWardelmann%26aufirst%3DE.%26aulast%3DJung%26aufirst%3DM.%26aulast%3DBuettner%26aufirst%3DR.%26aulast%3DSchule%26aufirst%3DR.%26atitle%3DThe%2520histone%2520code%2520reader%2520SPIN1%2520controls%2520RET%2520signaling%2520in%2520liposarcoma%26jtitle%3DOncotarget%26date%3D2015%26volume%3D6%26spage%3D4773%26epage%3D4789%26doi%3D10.18632%2Foncotarget.3000" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xing, A.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, D.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, P.</span></span> <span> </span><span class="NLM_article-title">Suppression of SPIN1-mediated PI3K–Akt pathway by miR-489 increases chemosensitivity in breast cancer</span>. <i>J. Pathol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>239</i></span>,  <span class="NLM_fpage">459</span>– <span class="NLM_lpage">472</span>, <span class="refDoi"> DOI: 10.1002/path.4743</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1002%2Fpath.4743" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=27171498" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1SmtL7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=239&publication_year=2016&pages=459-472&author=X.+Chenauthor=Y.-W.+Wangauthor=A.-Y.+Xingauthor=S.+Xiangauthor=D.-B.+Shiauthor=L.+Liuauthor=Y.-X.+Liauthor=P.+Gao&title=Suppression+of+SPIN1-mediated+PI3K%E2%80%93Akt+pathway+by+miR-489+increases+chemosensitivity+in+breast+cancer&doi=10.1002%2Fpath.4743"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Suppression of SPIN1-mediated PI3K-Akt pathway by miR-489 increases chemosensitivity in breast cancer</span></div><div class="casAuthors">Chen, Xu; Wang, Ya-Wen; Xing, Ai-Yan; Xiang, Shuai; Shi, Duan-Bo; Liu, Lei; Li, Yan-Xiang; Gao, Peng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pathology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">239</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">459-472</span>CODEN:
                <span class="NLM_cas:coden">JPTLAS</span>;
        ISSN:<span class="NLM_cas:issn">1096-9896</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Ltd.</span>)
        </div><div class="casAbstract">Drug resistance is one of the major obstacles for improving the prognosis of breast cancer patients.  Increasing evidence has linked the assocn. of aberrantly expressed microRNAs (miRNAs) with tumor development and progression as well as chemoresistance.  Despite recent advances, there is still little known about the potential role and mechanism of miRNAs in breast cancer chemoresistance.  Here we describe that 16 miRNAs were found to be significantly down-regulated and 11 up-regulated in drug-resistant breast cancer tissues compared with drug-sensitive tissues, using a miRNA microarray.  The results also showed miR-489 to be one of the most down-regulated miRNAs in drug-resistant tissues and cell lines, as confirmed by miRNA microarray screening and real-time quant. PCR.  A decrease in miR-489 expression was assocd. with chemoresistance as well as lymph node metastasis, increased tumor size, advanced pTNM stage and poor prognosis in breast cancer.  Functional anal. revealed that miR-489 increased breast cancer chemosensitivity and inhibited cell proliferation, migration and invasion, both in vitro and in vivo.  Furthermore, SPIN1, VAV3, BCL2 and AKT3 were found to be direct targets of miR-489.  SPIN1 was significantly elevated in drug-resistant and metastatic breast cancer tissues and inversely correlated with miR-489 expression.  High expression of SPIN1 was assocd. with higher histol. grade, lymph node metastasis, advanced pTNM stage and pos. progesterone receptor (PR) status.  Increased SPIN1 expression enhanced cell migration and invasion, inhibited apoptosis and partially antagonized the effects of miR-489 in breast cancer.  PIK3CA, AKT, CREB1 and BCL2 in the PI3K-Akt signalling pathway, demonstrated to be elevated in drug-resistant breast cancer tissues, were identified as downstream effectors of SPIN1.  It was further found that either inhibition of SPIN1 or overexpression of miR-489 suppressed the PI3K-Akt signalling pathway.  These data indicate that miR-489 could reverse the chemoresistance of breast cancer via the PI3K-Akt pathway by targeting SPIN1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlYdZmKYimMbVg90H21EOLACvtfcHk0liQ2p2T3b0IlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1SmtL7O&md5=fe996113e57b3a6c2c0cce1b15c6ad0c</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1002%2Fpath.4743&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpath.4743%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DY.-W.%26aulast%3DXing%26aufirst%3DA.-Y.%26aulast%3DXiang%26aufirst%3DS.%26aulast%3DShi%26aufirst%3DD.-B.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DY.-X.%26aulast%3DGao%26aufirst%3DP.%26atitle%3DSuppression%2520of%2520SPIN1-mediated%2520PI3K%25E2%2580%2593Akt%2520pathway%2520by%2520miR-489%2520increases%2520chemosensitivity%2520in%2520breast%2520cancer%26jtitle%3DJ.%2520Pathol.%26date%3D2016%26volume%3D239%26spage%3D459%26epage%3D472%26doi%3D10.1002%2Fpath.4743" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plummer, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Margolin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, S. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, H.</span></span> <span> </span><span class="NLM_article-title">SPIN1 promotes tumorigenesis by blocking the uL18 (universal large ribosomal subunit protein 18)-MDM2-p53 pathway in human cancer</span>. <i>eLife</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">e31275</span> <span class="refDoi"> DOI: 10.7554/elife.31275</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.7554%2Felife.31275" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=29547122" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2018&author=Z.+Fangauthor=B.+Caoauthor=J.-M.+Liaoauthor=J.+Dengauthor=K.+D.+Plummerauthor=P.+Liaoauthor=T.+Liuauthor=W.+Zhangauthor=K.+Zhangauthor=L.+Liauthor=D.+Margolinauthor=S.+X.+Zengauthor=J.+Xiongauthor=H.+Lu&title=SPIN1+promotes+tumorigenesis+by+blocking+the+uL18+%28universal+large+ribosomal+subunit+protein+18%29-MDM2-p53+pathway+in+human+cancer&doi=10.7554%2Felife.31275"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.7554%2Felife.31275&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252Felife.31275%26sid%3Dliteratum%253Aachs%26aulast%3DFang%26aufirst%3DZ.%26aulast%3DCao%26aufirst%3DB.%26aulast%3DLiao%26aufirst%3DJ.-M.%26aulast%3DDeng%26aufirst%3DJ.%26aulast%3DPlummer%26aufirst%3DK.%2BD.%26aulast%3DLiao%26aufirst%3DP.%26aulast%3DLiu%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DMargolin%26aufirst%3DD.%26aulast%3DZeng%26aufirst%3DS.%2BX.%26aulast%3DXiong%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DH.%26atitle%3DSPIN1%2520promotes%2520tumorigenesis%2520by%2520blocking%2520the%2520uL18%2520%2528universal%2520large%2520ribosomal%2520subunit%2520protein%252018%2529-MDM2-p53%2520pathway%2520in%2520human%2520cancer%26jtitle%3DeLife%26date%3D2018%26volume%3D7%26doi%3D10.7554%2Felife.31275" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.</span></span> <span> </span><span class="NLM_article-title">miR-489 inhibits proliferation, cell cycle progression and induces apoptosis of glioma cells via targeting SPIN1-mediated PI3K/AKT pathway</span>. <i>Biomed. Pharmacother.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">435</span>– <span class="NLM_lpage">443</span>, <span class="refDoi"> DOI: 10.1016/j.biopha.2017.06.058</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1016%2Fj.biopha.2017.06.058" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=28666210" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVyht7%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=2017&pages=435-443&author=Y.+Liauthor=X.+Maauthor=Y.+Wangauthor=G.+Li&title=miR-489+inhibits+proliferation%2C+cell+cycle+progression+and+induces+apoptosis+of+glioma+cells+via+targeting+SPIN1-mediated+PI3K%2FAKT+pathway&doi=10.1016%2Fj.biopha.2017.06.058"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">miR-489 inhibits proliferation, cell cycle progression and induces apoptosis of glioma cells via targeting SPIN1-mediated PI3K/AKT pathway</span></div><div class="casAuthors">Li, Yan; Ma, Xiaolin; Wang, Yanpeng; Li, Guohua</div><div class="citationInfo"><span class="NLM_cas:title">Biomedicine & Pharmacotherapy</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">435-443</span>CODEN:
                <span class="NLM_cas:coden">BIPHEX</span>;
        ISSN:<span class="NLM_cas:issn">0753-3322</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">MicroRNA-489 (miR-489), a newly identified tumor-related miRNA, functions as an oncogene or tumor suppressor via regulating growth and metastasis of human cancers.  But, the clin. significance, biol. function and underlying mechanisms of miR-489 in glioma remain rarely known.  Here, we showed that the levels of miR-489 in glioma tissues were notably underexpressed compared to corresponding non-tumor tissues.  In accordance, the relative levels of miR-489 were decreased in glioma cell lines compared with NHA cells.  Kaplan-Meier plots indicated that miR-489 low expressing glioma patients showed a prominent shorter overall survival.  In addn., miR-489 overexpression prohibited proliferation and cell cycle progression, and promoted apoptosis in U251 cells.  While, miR-489 knockdown showed opposite effects on these cellular processes of U87 cells.  In vivo expts. demonstrated that miR-489 restoration reduced the tumor vol. and wt. of s.c. glioma xenografts in nude mice.  Notably, Spindlin 1 (SPIN1) was inversely and directly regulated by miR-489 in glioma cells.  A neg. correlation between the expression of miR-489 and SPIN1 mRNA was confirmed in glioma tissues.  Interestingly, miR-489 inversely modulated activation of PI3K/AKT pathway and expression of downstream targets including p-mTOR, Cyclin D1 and BCL-XL.  SPIN1 re-expression abolished the effects of miR-489 on U251 cells with enhanced activation of PI3K/AKT pathway and malignant phenotype.  Meanwhile, AKT inhibitor MK-2206 blocked activation of PI3K/AKT pathway and resulted in reduced proliferation, cell cycle arrest and increased apoptosis in miR-489 down-regulating U87 cells.  Altogether, our data support that miR-489 loss facilitates malignant phenotype of glioma cells probably via SPIN1-mediated PI3K/AKT pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq39T3IjNfjurVg90H21EOLACvtfcHk0lgGYYmtySaREw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVyht7%252FE&md5=4cd57f05e0bbfec6fd2206244d73c291</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.biopha.2017.06.058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biopha.2017.06.058%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DG.%26atitle%3DmiR-489%2520inhibits%2520proliferation%252C%2520cell%2520cycle%2520progression%2520and%2520induces%2520apoptosis%2520of%2520glioma%2520cells%2520via%2520targeting%2520SPIN1-mediated%2520PI3K%252FAKT%2520pathway%26jtitle%3DBiomed.%2520Pharmacother.%26date%3D2017%26volume%3D93%26spage%3D435%26epage%3D443%26doi%3D10.1016%2Fj.biopha.2017.06.058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, G.</span></span> <span> </span><span class="NLM_article-title">Long noncoding RNA MHENCR promotes melanoma progression via regulating miR-425/489-mediated PI3K-Akt pathway</span>. <i>Am. J. Transl. Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">90</span>– <span class="NLM_lpage">102</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=28123636" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1ChsLzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=90-102&author=X.+Chenauthor=H.+Dongauthor=S.+Liuauthor=L.+Yuauthor=D.+Yanauthor=X.+Yaoauthor=W.+Sunauthor=D.+Hanauthor=G.+Gao&title=Long+noncoding+RNA+MHENCR+promotes+melanoma+progression+via+regulating+miR-425%2F489-mediated+PI3K-Akt+pathway"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Long noncoding RNA MHENCR promotes melanoma progression via regulating miR-425/489-mediated PI3K-Akt pathway</span></div><div class="casAuthors">Chen, Xiangjun; Dong, Hao; Liu, Sha; Yu, Li; Yan, Dexiong; Yao, Xingwei; Sun, Weijing; Han, Dezhi; Gao, Guozhen</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Translational Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">90-105, S1-S2</span>CODEN:
                <span class="NLM_cas:coden">AJTRA7</span>;
        ISSN:<span class="NLM_cas:issn">1943-8141</span>.
    
            (<span class="NLM_cas:orgname">e-Century Publishing Corp.</span>)
        </div><div class="casAbstract">Increasing evidences demonstrated that long noncoding RNAs (lncRNAs) are frequently dysregulated and have crit. roles in many tumors.  However, the roles and functional mechanisms of lncRNAs in melanoma remain largely unknown.  In this study, we identified a novel lncRNA MHENCR which was upregulated in melanoma tissues and further upregulated in metastatic melanoma.  Increased expression of MHENCR indicted poor survival of melanoma patients.  Functional expts. revealed that MHENCR knockdown significantly inhibited melanoma cells proliferation, induced cell cycle arrest and apoptosis, and also attenuated melanoma cells migration in vitro.  Furthermore, we identified MHENCR as a competitively endogenous RNA, which specifically bound to miR-425 and miR-489, upregulated their target genes IGF1 and SPIN1 expression, and further activated PI3K-Akt pathway.  Statistically significant correlations were obsd. between MHENCR expression and IGF1 and SPIN1 in melanoma tissues.  In vivo functional expts. further confirmed the pro-growth and pro-metastasis roles of MHENCR.  Collectively, our findings revealed that MHENCR functions as an oncogene in melanoma via activating miR-425/489-mediated PI3K-Akt pathway, and may be a therapeutic target for melanoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPAKn59fHqdrVg90H21EOLACvtfcHk0lgGYYmtySaREw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1ChsLzK&md5=829b7d67ae7fe250ad4d6576dd3913c6</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DX.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DYu%26aufirst%3DL.%26aulast%3DYan%26aufirst%3DD.%26aulast%3DYao%26aufirst%3DX.%26aulast%3DSun%26aufirst%3DW.%26aulast%3DHan%26aufirst%3DD.%26aulast%3DGao%26aufirst%3DG.%26atitle%3DLong%2520noncoding%2520RNA%2520MHENCR%2520promotes%2520melanoma%2520progression%2520via%2520regulating%2520miR-425%252F489-mediated%2520PI3K-Akt%2520pathway%26jtitle%3DAm.%2520J.%2520Transl.%2520Res.%26date%3D2017%26volume%3D9%26spage%3D90%26epage%3D102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, X.</span></span> <span> </span><span class="NLM_article-title">Overexpression of Spindlin1 induces cellular senescence, multinucleation and apoptosis</span>. <i>Gene</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>410</i></span>,  <span class="NLM_fpage">67</span>– <span class="NLM_lpage">74</span>, <span class="refDoi"> DOI: 10.1016/j.gene.2007.11.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1016%2Fj.gene.2007.11.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=18201843" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BD1cXitlWksL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=410&publication_year=2008&pages=67-74&author=H.+Yuanauthor=P.+Zhangauthor=L.+Qinauthor=L.+Chenauthor=S.+Shiauthor=Y.+Luauthor=F.+Yanauthor=C.+Baiauthor=X.+Nanauthor=D.+Liuauthor=Y.+Liauthor=W.+Yueauthor=X.+Pei&title=Overexpression+of+Spindlin1+induces+cellular+senescence%2C+multinucleation+and+apoptosis&doi=10.1016%2Fj.gene.2007.11.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Overexpression of SPINDLIN1 induces cellular senescence, multinucleation and apoptosis</span></div><div class="casAuthors">Yuan, Hongfeng; Zhang, Peng; Qin, Lipeng; Chen, Lin; Shi, Shuangshuang; Lu, Yang; Yan, Fang; Bai, Cixian; Nan, Xue; Liu, Daqing; Li, Yanhua; Yue, Wen; Pei, Xuetao</div><div class="citationInfo"><span class="NLM_cas:title">Gene</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">410</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">67-74</span>CODEN:
                <span class="NLM_cas:coden">GENED6</span>;
        ISSN:<span class="NLM_cas:issn">0378-1119</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Human or mouse SPLINDIN1 is expressed in various tissues and cells, but its biol. functions are poorly understood.  In this study, we show that human SPINDLIN1 is localized to interphase nucleus and mitotic chromosomes, and its expression in HeLa cells is not regulated in a cell cycle-dependent manner.  When SPINDLIN1 is stably overexpressed in HeLa cells, it results in multinucleation of cells, and these multinucleated cells exhibits characteristic features of senescence and apoptosis shown by growth and morphol. alterations, β-galactosidase activity, and Annexin V/7-Aminoactinomycin D staining.  Mouse Spindlin1 is highly homologous with human SPLINDIN1, when overexpressed in NIH3T3 cells, it also induces multinucleation, senescence and apoptosis in murine cells.  These results demonstrate that SPINDLIN1 is an important gene for mammalian mitotic chromosome functions, and disrupted regulation results in abnormal cell division, a mechanism that may be involved in tumorigenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpuwUiRH1KJwbVg90H21EOLACvtfcHk0ljHaWEPoew6WQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXitlWksL0%253D&md5=e9cf822e1145404f82024fb326d57a92</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.gene.2007.11.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.gene.2007.11.019%26sid%3Dliteratum%253Aachs%26aulast%3DYuan%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DP.%26aulast%3DQin%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DShi%26aufirst%3DS.%26aulast%3DLu%26aufirst%3DY.%26aulast%3DYan%26aufirst%3DF.%26aulast%3DBai%26aufirst%3DC.%26aulast%3DNan%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DYue%26aufirst%3DW.%26aulast%3DPei%26aufirst%3DX.%26atitle%3DOverexpression%2520of%2520Spindlin1%2520induces%2520cellular%2520senescence%252C%2520multinucleation%2520and%2520apoptosis%26jtitle%3DGene%26date%3D2008%26volume%3D410%26spage%3D67%26epage%3D74%26doi%3D10.1016%2Fj.gene.2007.11.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, X.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, H.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhai, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.-N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jia, Y.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, X.-T.</span></span> <span> </span><span class="NLM_article-title">Spindline1 promotes cancer cell proliferation through activation of WNT/TCF-4 signaling</span>. <i>Mol. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">326</span>– <span class="NLM_lpage">335</span>, <span class="refDoi"> DOI: 10.1158/1541-7786.mcr-11-0440</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1158%2F1541-7786.mcr-11-0440" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=22258766" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC38XktVamsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2012&pages=326-335&author=J.-X.+Wangauthor=Q.+Zengauthor=L.+Chenauthor=J.-C.+Duauthor=X.-L.+Yanauthor=H.-F.+Yuanauthor=C.+Zhaiauthor=J.-N.+Zhouauthor=Y.-L.+Jiaauthor=W.+Yueauthor=X.-T.+Pei&title=Spindline1+promotes+cancer+cell+proliferation+through+activation+of+WNT%2FTCF-4+signaling&doi=10.1158%2F1541-7786.mcr-11-0440"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">SPINDLIN1 Promotes Cancer Cell Proliferation through Activation of WNT/TCF-4 Signaling</span></div><div class="casAuthors">Wang, Jing-Xue; Zeng, Quan; Chen, Lin; Du, Ji-Chao; Yan, Xin-Long; Yuan, Hong-Feng; Zhai, Chao; Zhou, Jun-Nian; Jia, Ya-Li; Yue, Wen; Pei, Xue-Tao</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">326-335</span>CODEN:
                <span class="NLM_cas:coden">MCROC5</span>;
        ISSN:<span class="NLM_cas:issn">1541-7786</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">SPINDLIN1, a new member of the SPIN/SSTY gene family, was first identified as a gene highly expressed in ovarian cancer cells.  We have previously shown that it is involved in the process of spindle organization and chromosomal stability and plays a role in the development of cancer.  Nevertheless, the mechanisms underlying its oncogenic role are still largely unknown.  Here, we first showed that expression of SPINDLIN1 is upregulated in clin. tumors.  Ectopic expression of SPINDLIN1 promoted cancer cell proliferation and activated WNT/T-cell factor (TCF)-4 signaling.  The Ser84 and Ser99 amino acids within SPINDLIN1 were further identified as the key functional sites in WNT/TCF-4 signaling activation.  Mutation of these two sites of SPINDLIN1 abolished its effects on promoting WNT/TCF-4 signaling and cancer cell proliferation.  We further found that Aurora-A could interact with and phosphorylate SPINDLIN1 at its key functional sites, Ser84 and Ser99, suggesting that phosphorylation of SPINDLIN1 is involved in its oncogenic function.  Collectively, these results suggest that SPINDLIN1, which may be a novel substrate of the Aurora-A kinase, promotes cancer cell growth through WNT/TCF-4 signaling activation.  Mol Cancer Res; 10(3); 326-35.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoie76oMHEUN7Vg90H21EOLACvtfcHk0ljHaWEPoew6WQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XktVamsrc%253D&md5=0446631fe80c7b518ce305c394075143</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1158%2F1541-7786.mcr-11-0440&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1541-7786.mcr-11-0440%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.-X.%26aulast%3DZeng%26aufirst%3DQ.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DDu%26aufirst%3DJ.-C.%26aulast%3DYan%26aufirst%3DX.-L.%26aulast%3DYuan%26aufirst%3DH.-F.%26aulast%3DZhai%26aufirst%3DC.%26aulast%3DZhou%26aufirst%3DJ.-N.%26aulast%3DJia%26aufirst%3DY.-L.%26aulast%3DYue%26aufirst%3DW.%26aulast%3DPei%26aufirst%3DX.-T.%26atitle%3DSpindline1%2520promotes%2520cancer%2520cell%2520proliferation%2520through%2520activation%2520of%2520WNT%252FTCF-4%2520signaling%26jtitle%3DMol.%2520Cancer%2520Res.%26date%3D2012%26volume%3D10%26spage%3D326%26epage%3D335%26doi%3D10.1158%2F1541-7786.mcr-11-0440" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abell, C.</span></span> <span> </span><span class="NLM_article-title">Fragment-based approaches to enzyme inhibition</span>. <i>Curr. Opin. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">489</span>– <span class="NLM_lpage">496</span>, <span class="refDoi"> DOI: 10.1016/j.copbio.2007.09.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1016%2Fj.copbio.2007.09.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=17959370" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BD1cXislOrsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2007&pages=489-496&author=A.+Ciulliauthor=C.+Abell&title=Fragment-based+approaches+to+enzyme+inhibition&doi=10.1016%2Fj.copbio.2007.09.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based approaches to enzyme inhibition</span></div><div class="casAuthors">Ciulli, Alessio; Abell, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Biotechnology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">489-496</span>CODEN:
                <span class="NLM_cas:coden">CUOBE3</span>;
        ISSN:<span class="NLM_cas:issn">0958-1669</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Fragment-based approaches have provided a new paradigm for small-mol. drug discovery.  The methodol. is complementary to high-throughput screening approaches, starting from fragments of low mol. complexity and high ligand efficiency, and building up to more potent inhibitors.  The approach, which depends heavily on a no. of biophys. techniques, is now being taken up by more groups in both industry and academia.  This article describes key aspects of the process and highlights recent developments and applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzMH6I2sXlL7Vg90H21EOLACvtfcHk0ljHaWEPoew6WQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXislOrsg%253D%253D&md5=ab79b47a77574d5174b9dd0c989c0e3c</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.copbio.2007.09.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.copbio.2007.09.003%26sid%3Dliteratum%253Aachs%26aulast%3DCiulli%26aufirst%3DA.%26aulast%3DAbell%26aufirst%3DC.%26atitle%3DFragment-based%2520approaches%2520to%2520enzyme%2520inhibition%26jtitle%3DCurr.%2520Opin.%2520Biotechnol.%26date%3D2007%26volume%3D18%26spage%3D489%26epage%3D496%26doi%3D10.1016%2Fj.copbio.2007.09.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rees, D. C.</span></span> <span> </span><span class="NLM_article-title">The rise of fragment-based drug discovery</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">187</span>– <span class="NLM_lpage">192</span>, <span class="refDoi"> DOI: 10.1038/nchem.217</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1038%2Fnchem.217" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=21378847" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmtF2rs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2009&pages=187-192&author=C.+W.+Murrayauthor=D.+C.+Rees&title=The+rise+of+fragment-based+drug+discovery&doi=10.1038%2Fnchem.217"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">The rise of fragment-based drug discovery</span></div><div class="casAuthors">Murray, Christopher W.; Rees, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">187-192</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The search for new drugs is plagued by high attrition rates at all stages in research and development.  Chemists have an opportunity to tackle this problem because attrition can be traced back, in part, to the quality of the chem. leads.  Fragment-based drug discovery (FBDD) is a new approach, increasingly used in the pharmaceutical industry, for reducing attrition and providing leads for previously intractable biol. targets.  FBDD identifies low-mol.-wt. ligands (∼150 Da) that bind to biol. important macromols.  The three-dimensional exptl. binding mode of these fragments is detd. using X-ray crystallog. or NMR spectroscopy, and is used to facilitate their optimization into potent mols. with drug-like properties.  Compared with high-throughput-screening, the fragment approach requires fewer compds. to be screened, and, despite the lower initial potency of the screening hits, offers more efficient and fruitful optimization campaigns.  Here, we review the rise of FBDD, including its application to discovering clin. candidates against targets for which other chem. approaches have struggled.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqg3ip2FKcYJbVg90H21EOLACvtfcHk0ljHaWEPoew6WQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmtF2rs78%253D&md5=0bd1a40fd927cce9200f0fa296a7462f</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1038%2Fnchem.217&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.217%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DRees%26aufirst%3DD.%2BC.%26atitle%3DThe%2520rise%2520of%2520fragment-based%2520drug%2520discovery%26jtitle%3DNat.%2520Chem.%26date%3D2009%26volume%3D1%26spage%3D187%26epage%3D192%26doi%3D10.1038%2Fnchem.217" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leeson, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Springthorpe, B.</span></span> <span> </span><span class="NLM_article-title">The influence of drug-like concepts on decision-making in medicinal chemistry</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">881</span>– <span class="NLM_lpage">890</span>, <span class="refDoi"> DOI: 10.1038/nrd2445</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1038%2Fnrd2445" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=17971784" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1els7rL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2007&pages=881-890&author=P.+D.+Leesonauthor=B.+Springthorpe&title=The+influence+of+drug-like+concepts+on+decision-making+in+medicinal+chemistry&doi=10.1038%2Fnrd2445"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">The influence of drug-like concepts on decision-making in medicinal chemistry</span></div><div class="casAuthors">Leeson, Paul D.; Springthorpe, Brian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">881-890</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Despite the wide acceptance of drug-like principles such as the 'rule of five', this anal. of mols. currently being synthesized in leading pharmaceutical companies reveals that their phys. properties differ significantly from those of recently discovered oral drugs.  The marked increase in lipophilicity in particular could increase the likelihood of attrition in drug development.  The application of guidelines linked to the concept of drug-likeness, such as the 'rule of five', has gained wide acceptance as an approach to reduce attrition in drug discovery and development.  However, despite this acceptance, anal. of recent trends reveals that the phys. properties of mols. that are currently being synthesized in leading drug discovery companies differ significantly from those of recently discovered oral drugs and compds. in clin. development.  The consequences of the marked increase in lipophilicity - the most important drug-like phys. property - include a greater likelihood of lack of selectivity and attrition in drug development.  Tackling the threat of compd.-related toxicol. attrition needs to move to the mainstream of medicinal chem. decision-making.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTUeonLD6iebVg90H21EOLACvtfcHk0lgjbjyR8KB-IQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1els7rL&md5=5665a9e2dc266a1ee096e20625757ef7</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1038%2Fnrd2445&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2445%26sid%3Dliteratum%253Aachs%26aulast%3DLeeson%26aufirst%3DP.%2BD.%26aulast%3DSpringthorpe%26aufirst%3DB.%26atitle%3DThe%2520influence%2520of%2520drug-like%2520concepts%2520on%2520decision-making%2520in%2520medicinal%2520chemistry%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2007%26volume%3D6%26spage%3D881%26epage%3D890%26doi%3D10.1038%2Fnrd2445" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allali-Hassani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herold, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yates, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Optimization of cellular activity of G9a inhibitors 7-aminoalkoxy-quinazolines</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">6139</span>– <span class="NLM_lpage">6150</span>, <span class="refDoi"> DOI: 10.1021/jm200903z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200903z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpvVWltLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=6139-6150&author=F.+Liuauthor=D.+Barsyte-Lovejoyauthor=A.+Allali-Hassaniauthor=Y.+Heauthor=J.+M.+Heroldauthor=X.+Chenauthor=C.+M.+Yatesauthor=S.+V.+Fryeauthor=P.+J.+Brownauthor=J.+Huangauthor=M.+Vedadiauthor=C.+H.+Arrowsmithauthor=J.+Jin&title=Optimization+of+cellular+activity+of+G9a+inhibitors+7-aminoalkoxy-quinazolines&doi=10.1021%2Fjm200903z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of cellular activity of G9a inhibitors 7-aminoalkoxy-quinazolines</span></div><div class="casAuthors">Liu, Feng; Barsyte-Lovejoy, Dalia; Allali-Hassani, Abdellah; He, Yunlong; Herold, J. Martin; Chen, Xin; Yates, Christopher M.; Frye, Stephen V.; Brown, Peter J.; Huang, Jing; Vedadi, Masoud; Arrowsmith, Cheryl H.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">6139-6150</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein lysine methyltransferase G9a plays key roles in the transcriptional repression of a variety of genes via dimethylation of lysine 9 on histone H3 (H3K9me2) of chromatin as well as dimethylation of nonhistone proteins including tumor suppressor p53.  We previously reported the discovery of UNC0321, the most potent G9a inhibitor to date, via structure-based design and structure-activity relationship (SAR) exploration of the quinazoline scaffold represented by BIX01294.  Despite its very high in vitro potency, compd. UNC0321 lacks sufficient cellular potency.  The design and synthesis of several generations of new analogs aimed at improving cell membrane permeability while maintaining high in vitro potency resulted in the discovery of a no. of novel G9a inhibitors such as UNC0646 I and UNC0631 II with excellent potency in a variety of cell lines and excellent sepn. of functional potency vs. cell toxicity.  The design, synthesis, and cellular SAR of these potent G9a inhibitors are described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrC91yKB7S34rVg90H21EOLACvtfcHk0lgjbjyR8KB-IQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpvVWltLg%253D&md5=f3f54b91e6b24aa248498f9ff1fdb1ae</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fjm200903z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200903z%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DHe%26aufirst%3DY.%26aulast%3DHerold%26aufirst%3DJ.%2BM.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DYates%26aufirst%3DC.%2BM.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DOptimization%2520of%2520cellular%2520activity%2520of%2520G9a%2520inhibitors%25207-aminoalkoxy-quinazolines%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D6139%26epage%3D6150%26doi%3D10.1021%2Fjm200903z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korboukh, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allali-Hassani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janzen, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of an in vivo chemical probe of the lysine methyltransferases G9a and GLP</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">8931</span>– <span class="NLM_lpage">8942</span>, <span class="refDoi"> DOI: 10.1021/jm401480r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401480r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsF2htrvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=8931-8942&author=F.+Liuauthor=D.+Barsyte-Lovejoyauthor=F.+Liauthor=Y.+Xiongauthor=V.+Korboukhauthor=X.-P.+Huangauthor=A.+Allali-Hassaniauthor=W.+P.+Janzenauthor=B.+L.+Rothauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=M.+Vedadiauthor=J.+Jin&title=Discovery+of+an+in+vivo+chemical+probe+of+the+lysine+methyltransferases+G9a+and+GLP&doi=10.1021%2Fjm401480r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of an in vivo chemical probe of the lysine methyltransferases G9a and GLP</span></div><div class="casAuthors">Liu, Feng; Barsyte-Lovejoy, Dalia; Li, Fengling; Xiong, Yan; Korboukh, Victoria; Huang, Xi-Ping; Allali-Hassani, Abdellah; Janzen, William P.; Roth, Bryan L.; Frye, Stephen V.; Arrowsmith, Cheryl H.; Brown, Peter J.; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">8931-8942</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Among epigenetic writers, readers, and erasers, the lysine methyltransferases G9a and GLP, which catalyze mono- and dimethylation of histone H3 lysine 9 (H3K9me2) and nonhistone proteins, have been implicated in a variety of human diseases.  A toolkit of well-characterized chem. probes will allow biol. and disease hypotheses concerning these proteins to be tested in cell-based and animal models with high confidence.  We previously discovered potent and selective G9a/GLP inhibitors including the cellular chem. probe UNC0638, which displays an excellent sepn. of functional potency and cell toxicity.  However, this inhibitor is not suitable for animal studies due to its poor pharmacokinetic (PK) properties.  Here, we report the discovery of the first G9a and GLP in vivo chem. probe UNC0642, which not only maintains high in vitro and cellular potency, low cell toxicity, and excellent selectivity, but also displays improved in vivo PK properties, making it suitable for animal studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowOI3TSEz3ZbVg90H21EOLACvtfcHk0lgjbjyR8KB-IQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsF2htrvK&md5=933f0c49a938b4dd8146f6d09aa21e21</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fjm401480r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401480r%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DKorboukh%26aufirst%3DV.%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520an%2520in%2520vivo%2520chemical%2520probe%2520of%2520the%2520lysine%2520methyltransferases%2520G9a%2520and%2520GLP%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D8931%26epage%3D8942%26doi%3D10.1021%2Fjm401480r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rival-Gervier, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allali-Hassani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labrie, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wigle, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dimaggio, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wasney, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siarheyeva, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tempel, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chau, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangano, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.-p.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simpson, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pattenden, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norris, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kireev, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tripathy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janzen, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petronis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">A chemical probe selectively inhibits G9a and GLP methyltransferase activity in cells</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">566</span>– <span class="NLM_lpage">574</span>, <span class="refDoi"> DOI: 10.1038/nchembio.599</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1038%2Fnchembio.599" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=21743462" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC3MXos1OmtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=566-574&author=M.+Vedadiauthor=D.+Barsyte-Lovejoyauthor=F.+Liuauthor=S.+Rival-Gervierauthor=A.+Allali-Hassaniauthor=V.+Labrieauthor=T.+J.+Wigleauthor=P.+A.+Dimaggioauthor=G.+A.+Wasneyauthor=A.+Siarheyevaauthor=A.+Dongauthor=W.+Tempelauthor=S.-C.+Wangauthor=X.+Chenauthor=I.+Chauauthor=T.+J.+Manganoauthor=X.-p.+Huangauthor=C.+D.+Simpsonauthor=S.+G.+Pattendenauthor=J.+L.+Norrisauthor=D.+B.+Kireevauthor=A.+Tripathyauthor=A.+Edwardsauthor=B.+L.+Rothauthor=W.+P.+Janzenauthor=B.+A.+Garciaauthor=A.+Petronisauthor=J.+Ellisauthor=P.+J.+Brownauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=J.+Jin&title=A+chemical+probe+selectively+inhibits+G9a+and+GLP+methyltransferase+activity+in+cells&doi=10.1038%2Fnchembio.599"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">A chemical probe selectively inhibits G9a and GLP methyltransferase activity in cells</span></div><div class="casAuthors">Vedadi, Masoud; Barsyte-Lovejoy, Dalia; Liu, Feng; Rival-Gervier, Sylvie; Allali-Hassani, Abdellah; Labrie, Viviane; Wigle, Tim J.; DiMaggio, Peter A.; Wasney, Gregory A.; Siarheyeva, Alena; Dong, Aiping; Tempel, Wolfram; Wang, Sun-Chong; Chen, Xin; Chau, Irene; Mangano, Thomas J.; Huang, Xi-ping; Simpson, Catherine D.; Pattenden, Samantha G.; Norris, Jacqueline L.; Kireev, Dmitri B.; Tripathy, Ashutosh; Edwards, Aled; Roth, Bryan L.; Janzen, William P.; Garcia, Benjamin A.; Petronis, Arturas; Ellis, James; Brown, Peter J.; Frye, Stephen V.; Arrowsmith, Cheryl H.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">566-574</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein lysine methyltransferases G9a and GLP modulate the transcriptional repression of a variety of genes via dimethylation of Lys9 on histone H3 (H3K9me2) as well as dimethylation of non-histone targets.  Here we report the discovery of UNC0638, an inhibitor of G9a and GLP with excellent potency and selectivity over a wide range of epigenetic and non-epigenetic targets.  UNC0638 treatment of a variety of cell lines resulted in lower global H3K9me2 levels, equiv. to levels obsd. for small hairpin RNA knockdown of G9a and GLP with the functional potency of UNC0638 being well sepd. from its toxicity.  UNC0638 markedly reduced the clonogenicity of MCF7 cells, reduced the abundance of H3K9me2 marks at promoters of known G9a-regulated endogenous genes and disproportionately affected several genomic loci encoding microRNAs.  In mouse embryonic stem cells, UNC0638 reactivated G9a-silenced genes and a retroviral reporter gene in a concn.-dependent manner without promoting differentiation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphv-2bZcBqwLVg90H21EOLACvtfcHk0lhY9FlI73Wvlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXos1OmtLs%253D&md5=a944e99a910194c1a15240fc6dc8f852</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.599&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.599%26sid%3Dliteratum%253Aachs%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DRival-Gervier%26aufirst%3DS.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DLabrie%26aufirst%3DV.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DDimaggio%26aufirst%3DP.%2BA.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DTempel%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DS.-C.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DMangano%26aufirst%3DT.%2BJ.%26aulast%3DHuang%26aufirst%3DX.-p.%26aulast%3DSimpson%26aufirst%3DC.%2BD.%26aulast%3DPattenden%26aufirst%3DS.%2BG.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DKireev%26aufirst%3DD.%2BB.%26aulast%3DTripathy%26aufirst%3DA.%26aulast%3DEdwards%26aufirst%3DA.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26aulast%3DGarcia%26aufirst%3DB.%2BA.%26aulast%3DPetronis%26aufirst%3DA.%26aulast%3DEllis%26aufirst%3DJ.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DA%2520chemical%2520probe%2520selectively%2520inhibits%2520G9a%2520and%2520GLP%2520methyltransferase%2520activity%2520in%2520cells%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D566%26epage%3D574%26doi%3D10.1038%2Fnchembio.599" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sweis, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pliushchev, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maag, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petros, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soni, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tse, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michaelides, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pappano, W. N.</span></span> <span> </span><span class="NLM_article-title">Discovery and Development of Potent and Selective Inhibitors of Histone Methyltransferase G9a</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">205</span>– <span class="NLM_lpage">209</span>, <span class="refDoi"> DOI: 10.1021/ml400496h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml400496h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFKmsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=205-209&author=R.+F.+Sweisauthor=M.+Pliushchevauthor=P.+J.+Brownauthor=J.+Guoauthor=F.+Liauthor=D.+Maagauthor=A.+M.+Petrosauthor=N.+B.+Soniauthor=C.+Tseauthor=M.+Vedadiauthor=M.+R.+Michaelidesauthor=G.+G.+Chiangauthor=W.+N.+Pappano&title=Discovery+and+Development+of+Potent+and+Selective+Inhibitors+of+Histone+Methyltransferase+G9a&doi=10.1021%2Fml400496h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Development of Potent and Selective Inhibitors of Histone Methyltransferase G9a</span></div><div class="casAuthors">Sweis, Ramzi F.; Pliushchev, Marina; Brown, Peter J.; Guo, Jun; Li, Fengling; Maag, David; Petros, Andrew M.; Soni, Nirupama B.; Tse, Chris; Vedadi, Masoud; Michaelides, Michael R.; Chiang, Gary G.; Pappano, William N.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">205-209</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">G9a is a histone lysine methyltransferase responsible for the methylation of histone H3 lysine 9.  The discovery of A-366 arose from a unique diversity screening hit, which was optimized by incorporation of a propyl-pyrrolidine subunit to occupy the enzyme lysine channel.  A-366 (I) is a potent inhibitor of G9a (IC50: 3.3 nM) with greater than 1000-fold selectivity over 21 other methyltransferases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMvdTiUgx-oLVg90H21EOLACvtfcHk0lhY9FlI73Wvlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFKmsA%253D%253D&md5=26affad72f2950510aa9f253e540d575</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fml400496h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml400496h%26sid%3Dliteratum%253Aachs%26aulast%3DSweis%26aufirst%3DR.%2BF.%26aulast%3DPliushchev%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DGuo%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DMaag%26aufirst%3DD.%26aulast%3DPetros%26aufirst%3DA.%2BM.%26aulast%3DSoni%26aufirst%3DN.%2BB.%26aulast%3DTse%26aufirst%3DC.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DMichaelides%26aufirst%3DM.%2BR.%26aulast%3DChiang%26aufirst%3DG.%2BG.%26aulast%3DPappano%26aufirst%3DW.%2BN.%26atitle%3DDiscovery%2520and%2520Development%2520of%2520Potent%2520and%2520Selective%2520Inhibitors%2520of%2520Histone%2520Methyltransferase%2520G9a%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D5%26spage%3D205%26epage%3D209%26doi%3D10.1021%2Fml400496h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oh, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solter, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knowles, B. B.</span></span> <span> </span><span class="NLM_article-title">Spindlin, a major maternal transcript expressed in the mouse during the transition from oocyte to embryo</span>. <i>Development</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>124</i></span>,  <span class="NLM_fpage">493</span>– <span class="NLM_lpage">503</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=9053325" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADyaK2sXhvVWmsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=1997&pages=493-503&author=B.+Ohauthor=S.+Y.+Hwangauthor=D.+Solterauthor=B.+B.+Knowles&title=Spindlin%2C+a+major+maternal+transcript+expressed+in+the+mouse+during+the+transition+from+oocyte+to+embryo"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Spindlin, a major maternal transcript expressed in the mouse during the transition from oocyte to embryo</span></div><div class="casAuthors">Oh, Bermseok; Hwang, Sue-Yun; Solter, Davor; Knowles, Barbara B.</div><div class="citationInfo"><span class="NLM_cas:title">Development (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">124</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">493-503</span>CODEN:
                <span class="NLM_cas:coden">DEVPED</span>;
        ISSN:<span class="NLM_cas:issn">0950-1991</span>.
    
            (<span class="NLM_cas:orgname">Company of Biologists</span>)
        </div><div class="casAbstract">Timely translation of maternal transcriptions and post-translational modification of their gene products control the initial development of preimplantation-stage embryos.  We have isolated and characterized a gene encoding a stage-specific embryonic protein.  This novel gene, spindlin (Spin), is an abundant maternal transcript present in the unfertilized egg and 2-cell, but not 8-cell, stage embryo.  Spin exhibits high homol. to a multicopy gene, Y-linked spermiogenesis-specific transcript (Ssty), and together they form a new gene family expressed during gametogenesis.  We find that spindlin assocs. with the meiotic spindle and is modified by phosphorylation in a cell-cycle-dependent fashion.  Furthermore, it comigrates with the previously described 30 × 103 Mr metaphase complex which is posttranslationally modified during the first mitotic cell cycle.  Our data suggest that spindlin plays a role in cell-cycle regulation during the transition from gamete to embryo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBqamj85CalbVg90H21EOLACvtfcHk0lhY9FlI73Wvlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXhvVWmsbk%253D&md5=12c37613f401dedd7d68597ef3e63b0b</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DOh%26aufirst%3DB.%26aulast%3DHwang%26aufirst%3DS.%2BY.%26aulast%3DSolter%26aufirst%3DD.%26aulast%3DKnowles%26aufirst%3DB.%2BB.%26atitle%3DSpindlin%252C%2520a%2520major%2520maternal%2520transcript%2520expressed%2520in%2520the%2520mouse%2520during%2520the%2520transition%2520from%2520oocyte%2520to%2520embryo%26jtitle%3DDevelopment%26date%3D1997%26volume%3D124%26spage%3D493%26epage%3D503" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Staub, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mennerich, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenthal, A.</span></span> <span> </span><span class="NLM_article-title">The Spin/Ssty repeat: a new motif identified in proteins involved in vertebrate development from gamete to embryo</span>. <i>Genome Biol.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">RESEARCH0003.1</span>, <span class="refDoi"> DOI: 10.1186/gb-2001-3-1-research0003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1186%2Fgb-2001-3-1-research0003" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2001&pages=RESEARCH0003.1&author=E.+Staubauthor=D.+Mennerichauthor=A.+Rosenthal&title=The+Spin%2FSsty+repeat%3A+a+new+motif+identified+in+proteins+involved+in+vertebrate+development+from+gamete+to+embryo&doi=10.1186%2Fgb-2001-3-1-research0003"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1186%2Fgb-2001-3-1-research0003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fgb-2001-3-1-research0003%26sid%3Dliteratum%253Aachs%26aulast%3DStaub%26aufirst%3DE.%26aulast%3DMennerich%26aufirst%3DD.%26aulast%3DRosenthal%26aufirst%3DA.%26atitle%3DThe%2520Spin%252FSsty%2520repeat%253A%2520a%2520new%2520motif%2520identified%2520in%2520proteins%2520involved%2520in%2520vertebrate%2520development%2520from%2520gamete%2520to%2520embryo%26jtitle%3DGenome%2520Biol.%26date%3D2001%26volume%3D3%26spage%3DRESEARCH0003.1%26doi%3D10.1186%2Fgb-2001-3-1-research0003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bae, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Premkumar, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sbardella, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span> <span> </span><span class="NLM_article-title">A transcriptional coregulator, SPIN.DOC, attenuates the coactivator activity of Spindlin1</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>292</i></span>,  <span class="NLM_fpage">20808</span>– <span class="NLM_lpage">20817</span>, <span class="refDoi"> DOI: 10.1074/jbc.m117.814913</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1074%2Fjbc.m117.814913" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=29061846" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitVejsbrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=292&publication_year=2017&pages=20808-20817&author=N.+Baeauthor=M.+Gaoauthor=X.+Liauthor=T.+Premkumarauthor=G.+Sbardellaauthor=J.+Chenauthor=M.+T.+Bedford&title=A+transcriptional+coregulator%2C+SPIN.DOC%2C+attenuates+the+coactivator+activity+of+Spindlin1&doi=10.1074%2Fjbc.m117.814913"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">A transcriptional coregulator, SPIN·DOC, attenuates the coactivator activity of Spindlin1</span></div><div class="casAuthors">Bae, Narkhyun; Gao, Min; Li, Xu; Premkumar, Tolkappiyan; Sbardella, Gianluca; Chen, Junjie; Bedford, Mark T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">292</span>
        (<span class="NLM_cas:issue">51</span>),
    <span class="NLM_cas:pages">20808-20817</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Spindlin1 (SPIN1) is a transcriptional coactivator with crit. functions in embryonic development and emerging roles in cancer.  SPIN1 harbors three Tudor domains, two of which engage the tail of histone H3 by reading the H3-Lys-4 trimethylation and H3-Arg-8 asym. dimethylation marks.  To gain mechanistic insight into how SPIN1 functions as a transcriptional coactivator, here we purified its interacting proteins.  We identified an uncharacterized protein (C11orf84), which we renamed SPIN1 docking protein (SPIN·DOC), that directly binds SPIN1 and strongly disrupts its histone methylation reading ability, causing it to disassoc. from chromatin.  The Spindlin family of coactivators has five related members (SPIN1, 2A, 2B, 3, and 4), and we found that all of them bind SPIN·DOC.  It has been reported previously that SPIN1 regulates gene expression in the Wnt signaling pathway by directly interacting with transcription factor 4 (TCF4).  We obsd. here that SPIN·DOC assocs. with TCF4 in a SPIN1-dependent manner and dampens SPIN1 coactivator activity in TOPflash reporter assays.  Furthermore, knockdown and overexpression expts. indicated that SPIN·DOC represses the expression of a no. of SPIN1-regulated genes, including those encoding rRNA and the cytokine IL1B.  In conclusion, we have identified SPIN·DOC as a transcriptional repressor that binds SPIN1 and masks its ability to engage the H3-Lys-4 trimethylation activation mark.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_Fa2GbBnnLrVg90H21EOLACvtfcHk0lj8IjLj-r5IDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitVejsbrE&md5=69c4c937798fe7e27f6234156291217f</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1074%2Fjbc.m117.814913&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.m117.814913%26sid%3Dliteratum%253Aachs%26aulast%3DBae%26aufirst%3DN.%26aulast%3DGao%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DPremkumar%26aufirst%3DT.%26aulast%3DSbardella%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DA%2520transcriptional%2520coregulator%252C%2520SPIN.DOC%252C%2520attenuates%2520the%2520coactivator%2520activity%2520of%2520Spindlin1%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2017%26volume%3D292%26spage%3D20808%26epage%3D20817%26doi%3D10.1074%2Fjbc.m117.814913" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Machleidt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodroofe, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwinn, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Méndez, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robers, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otto, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirkland, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, K. V.</span></span> <span> </span><span class="NLM_article-title">NanoBRET—a novel BRET platform for the analysis of protein–protein interactions</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1797</span>– <span class="NLM_lpage">1804</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00143</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00143" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC2MXovFKkt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1797-1804&author=T.+Machleidtauthor=C.+C.+Woodroofeauthor=M.+K.+Schwinnauthor=J.+M%C3%A9ndezauthor=M.+B.+Robersauthor=K.+Zimmermanauthor=P.+Ottoauthor=D.+L.+Danielsauthor=T.+A.+Kirklandauthor=K.+V.+Wood&title=NanoBRET%E2%80%94a+novel+BRET+platform+for+the+analysis+of+protein%E2%80%93protein+interactions&doi=10.1021%2Facschembio.5b00143"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">NanoBRET-A Novel BRET Platform for the Analysis of Protein-Protein Interactions</span></div><div class="casAuthors">Machleidt, Thomas; Woodroofe, Carolyn C.; Schwinn, Marie K.; Mendez, Jacqui; Robers, Matthew B.; Zimmerman, Kris; Otto, Paul; Daniels, Danette L.; Kirkland, Thomas A.; Wood, Keith V.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1797-1804</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Dynamic interactions between proteins comprise a key mechanism for temporal control of cellular function and thus hold promise for development of novel drug therapies.  It remains tech. challenging, however, to quant. characterize these interactions within the biol. relevant context of living cells.  Although, bioluminescence resonance energy transfer (BRET) has often been used for this purpose, its general applicability has been hindered by limited sensitivity and dynamic range.  We have addressed this by combining an extremely bright luciferase (Nanoluc) with a means for tagging intracellular proteins with a long-wavelength fluorophore (HaloTag).  The small size (19 kDa), high emission intensity, and relatively narrow spectrum (460 nm peak intensity) make Nanoluc luciferase well suited as an energy donor.  By selecting an efficient red-emitting fluorophore (635 nm peak intensity) for attachment onto the HaloTag, an overall spectral sepn. exceeding 175 nm was achieved.  This combination of greater light intensity with improved spectral resoln. results in substantially increased detection sensitivity and dynamic range over current BRET technologies.  Enhanced performance is demonstrated using several established model systems, as well as the ability to image BRET in individual cells.  The capabilities are further exhibited in a novel assay developed for analyzing the interactions of bromodomain proteins with chromatin in living cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6-h9HlReNabVg90H21EOLACvtfcHk0lj8IjLj-r5IDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXovFKkt7c%253D&md5=9f5f773b6cd14c40074a2ffbd6472008</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00143&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00143%26sid%3Dliteratum%253Aachs%26aulast%3DMachleidt%26aufirst%3DT.%26aulast%3DWoodroofe%26aufirst%3DC.%2BC.%26aulast%3DSchwinn%26aufirst%3DM.%2BK.%26aulast%3DM%25C3%25A9ndez%26aufirst%3DJ.%26aulast%3DRobers%26aufirst%3DM.%2BB.%26aulast%3DZimmerman%26aufirst%3DK.%26aulast%3DOtto%26aufirst%3DP.%26aulast%3DDaniels%26aufirst%3DD.%2BL.%26aulast%3DKirkland%26aufirst%3DT.%2BA.%26aulast%3DWood%26aufirst%3DK.%2BV.%26atitle%3DNanoBRET%25E2%2580%2594a%2520novel%2520BRET%2520platform%2520for%2520the%2520analysis%2520of%2520protein%25E2%2580%2593protein%2520interactions%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D1797%26epage%3D1804%26doi%3D10.1021%2Facschembio.5b00143" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Green, K.</span></span> <span> </span><span class="NLM_article-title">Improved metalation of 2,4,6-tribromoanisole - sythesis of 2-methoxyresorcinol</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">4325</span>– <span class="NLM_lpage">4326</span>, <span class="refDoi"> DOI: 10.1021/jo00013a052</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo00013a052" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADyaK3MXktVyiur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=1991&pages=4325-4326&author=K.+Green&title=Improved+metalation+of+2%2C4%2C6-tribromoanisole+-+sythesis+of+2-methoxyresorcinol&doi=10.1021%2Fjo00013a052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Improved metalation of 2,4,6-tribromoanisole:  synthesis of 2-methoxyresorcinol</span></div><div class="casAuthors">Green, Kenneth</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">4325-6</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    </div><div class="casAbstract">The low temp., heterogeneous metalation of 2,4,6-tribromoanisole in pentane using butyllithium is described.  This occurs without the aid of any chelating agent such as N,N,N',N'-tetramethylethylenediamine.  By performing the metalation in a heterogeneous manner, the products obtained from a protocol of (a) trimethylborate quench (b) peracetic acid oxidn. are significantly cleaner than when the same sequence of reactions occurs under homogeneous conditions.  Variation of the n-butyllithium stoichiometry results in products obtained effectively from either the mono- or dianions of the tribromide.  While examples resulting solely from reaction with trimethylborate are shown, it appears that the method may be general for other electrophiles.  A concise and efficient synthesis of 2-methoxyresorcinol is also demonstrated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP9S_3K5bf3bVg90H21EOLACvtfcHk0lj8IjLj-r5IDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXktVyiur4%253D&md5=ebb0e4b6da8807507036e7c1e1196e9b</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fjo00013a052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo00013a052%26sid%3Dliteratum%253Aachs%26aulast%3DGreen%26aufirst%3DK.%26atitle%3DImproved%2520metalation%2520of%25202%252C4%252C6-tribromoanisole%2520-%2520sythesis%2520of%25202-methoxyresorcinol%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1991%26volume%3D56%26spage%3D4325%26epage%3D4326%26doi%3D10.1021%2Fjo00013a052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P.</span></span> <span> </span><span class="NLM_article-title">Scaling and assessment of data quality</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">72</span>– <span class="NLM_lpage">82</span>, <span class="refDoi"> DOI: 10.1107/s0907444905036693</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1107%2Fs0907444905036693" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=16369096" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlant7jM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2006&pages=72-82&author=P.+Evans&title=Scaling+and+assessment+of+data+quality&doi=10.1107%2Fs0907444905036693"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Scaling and assessment of data quality</span></div><div class="casAuthors">Evans, Philip</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">D62</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">72-82</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">The various phys. factors affecting measured diffraction intensities are discussed, as are the scaling models which may be used to put the data on a consistent scale.  After scaling, the intensities can be analyzed to set the real resoln. of the data set, to detect bad regions (e.g. bad images), to analyze radiation damage and to assess the overall quality of the data set.  The significance of any anomalous signal may be assessed by probability and correlation anal.  The algorithms used by the CCP4 scaling program SCALA are described.  A requirement for the scaling and merging of intensities is knowledge of the Laue group and point-group symmetries: the possible symmetry of the diffraction pattern may be detd. from scores such as correlation coeffs. between observations which might be symmetry-related.  These scoring functions are implemented in a new program POINTLESS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGravLSLU62RhLVg90H21EOLACvtfcHk0lhdh1zsi0WUUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlant7jM&md5=293d3876e534c0c57813990515bb3c76</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1107%2Fs0907444905036693&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0907444905036693%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%26atitle%3DScaling%2520and%2520assessment%2520of%2520data%2520quality%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2006%26volume%3D62%26spage%3D72%26epage%3D82%26doi%3D10.1107%2Fs0907444905036693" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Axford, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foadi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, N.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choudhury, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwata, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beis, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alguel, Y.</span></span> <span> </span><span class="NLM_article-title">Structure determination of an integral membrane protein at room temperature from crystals in situ</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">1228</span>– <span class="NLM_lpage">1237</span>, <span class="refDoi"> DOI: 10.1107/s139900471500423x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1107%2Fs139900471500423x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=26057664" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVKisLbI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2015&pages=1228-1237&author=D.+Axfordauthor=J.+Foadiauthor=N.-J.+Huauthor=H.+G.+Choudhuryauthor=S.+Iwataauthor=K.+Beisauthor=G.+Evansauthor=Y.+Alguel&title=Structure+determination+of+an+integral+membrane+protein+at+room+temperature+from+crystals+in+situ&doi=10.1107%2Fs139900471500423x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Structure determination of an integral membrane protein at room temperature from crystals in situ</span></div><div class="casAuthors">Axford, Danny; Foadi, James; Hu, Nien-Jen; Choudhury, Hassanul Ghani; Iwata, So; Beis, Konstantinos; Evans, Gwyndaf; Alguel, Yilmaz</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1228-1237</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">1399-0047</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The structure detn. of an integral membrane protein using synchrotron X-ray diffraction data collected at room temp. directly in vapor-diffusion crystn. plates (in situ) is demonstrated.  Exposing the crystals in situ eliminates manual sample handling and, since it is performed at room temp., removes the complication of cryoprotection and potential structural anomalies induced by sample cryocooling.  Essential to the method is the ability to limit radiation damage by recording a small amt. of data per sample from many samples and subsequently assembling the resulting data sets using specialized software.  The validity of this procedure is established by the structure detn. of Haemophilus influenza TehA at 2.3 Å resoln.  The method presented offers an effective protocol for the fast and efficient detn. of membrane-protein structures at room temp. using third-generation synchrotron beamlines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm6BVlKwCapbVg90H21EOLACvtfcHk0lhdh1zsi0WUUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVKisLbI&md5=bb8215e22dc3d619ebabcbdde703bfd6</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1107%2Fs139900471500423x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs139900471500423x%26sid%3Dliteratum%253Aachs%26aulast%3DAxford%26aufirst%3DD.%26aulast%3DFoadi%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DN.-J.%26aulast%3DChoudhury%26aufirst%3DH.%2BG.%26aulast%3DIwata%26aufirst%3DS.%26aulast%3DBeis%26aufirst%3DK.%26aulast%3DEvans%26aufirst%3DG.%26aulast%3DAlguel%26aufirst%3DY.%26atitle%3DStructure%2520determination%2520of%2520an%2520integral%2520membrane%2520protein%2520at%2520room%2520temperature%2520from%2520crystals%2520in%2520situ%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2015%26volume%3D71%26spage%3D1228%26epage%3D1237%26doi%3D10.1107%2Fs139900471500423x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballard, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keegan, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krissinel, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNicholas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potterton, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, K. S.</span></span> <span> </span><span class="NLM_article-title">Overview of the CCP4 suite and current developments</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1107/s0907444910045749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=235-242&author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=R.+M.+Keeganauthor=E.+B.+Krissinelauthor=A.+G.+W.+Leslieauthor=A.+McCoyauthor=S.+J.+McNicholasauthor=G.+N.+Murshudovauthor=N.+S.+Pannuauthor=E.+A.+Pottertonauthor=H.+R.+Powellauthor=R.+J.+Readauthor=A.+Vaginauthor=K.+S.+Wilson&title=Overview+of+the+CCP4+suite+and+current+developments&doi=10.1107%2Fs0907444910045749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0lhdh1zsi0WUUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DKeegan%26aufirst%3DR.%2BM.%26aulast%3DKrissinel%26aufirst%3DE.%2BB.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%26aulast%3DMcNicholas%26aufirst%3DS.%2BJ.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DPotterton%26aufirst%3DE.%2BA.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520the%2520CCP4%2520suite%2520and%2520current%2520developments%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D235%26epage%3D242%26doi%3D10.1107%2Fs0907444910045749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G.</span></span> <span> </span><span class="NLM_article-title">xia2: an expert system for macromolecular crystallography data reduction</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">186</span>– <span class="NLM_lpage">190</span>, <span class="refDoi"> DOI: 10.1107/s0021889809045701</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1107%2FS0021889809045701" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFalsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2010&pages=186-190&author=G.+Winter&title=xia2%3A+an+expert+system+for+macromolecular+crystallography+data+reduction&doi=10.1107%2Fs0021889809045701"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">xia2: An expert system for macromolecular crystallography data reduction</span></div><div class="casAuthors">Winter, G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">186-190, S186/1-S186/5</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">An expert system for macromol. crystallog. data redn. is presented, which builds on existing software to automate the complete data redn. process from images to merged structure factor amplitudes.  This can automatically identify multi-wedge, multi-pass and multiwavelength data sets and includes explicit procedures to test for crystallog. special cases.  With the push towards high-throughput crystallog. at synchrotron beamlines and automation of structure soln., the ability to reduce data with no user input fills an important gap in the pipeline.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOi3u0ydXbSrVg90H21EOLACvtfcHk0ljrvhw1TD-6KA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFalsrk%253D&md5=27771caa813f3e828abdf2a331ae55e3</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1107%2FS0021889809045701&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889809045701%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DG.%26atitle%3Dxia2%253A%2520an%2520expert%2520system%2520for%2520macromolecular%2520crystallography%2520data%2520reduction%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2010%26volume%3D43%26spage%3D186%26epage%3D190%26doi%3D10.1107%2Fs0021889809045701" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waterman, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parkhurst, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brewster, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gildea, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuentes-Montero, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vollmar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michels-Clark, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauter, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, G.</span></span> <span> </span><span class="NLM_article-title">DIALS: implementation and evaluation of a new integration package</span>. <i>Acta Crystallogr., Sect. D: Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">85</span>– <span class="NLM_lpage">97</span>, <span class="refDoi"> DOI: 10.1107/s2059798317017235</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1107%2FS2059798317017235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC1cXksFarurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2018&pages=85-97&author=G.+Winterauthor=D.+G.+Watermanauthor=J.+M.+Parkhurstauthor=A.+S.+Brewsterauthor=R.+J.+Gildeaauthor=M.+Gerstelauthor=L.+Fuentes-Monteroauthor=M.+Vollmarauthor=T.+Michels-Clarkauthor=I.+D.+Youngauthor=N.+K.+Sauterauthor=G.+Evans&title=DIALS%3A+implementation+and+evaluation+of+a+new+integration+package&doi=10.1107%2Fs2059798317017235"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">DIALS: implementation and evaluation of a new integration package</span></div><div class="casAuthors">Winter, Graeme; Waterman, David G.; Parkhurst, James M.; Brewster, Aaron S.; Gildea, Richard J.; Gerstel, Markus; Fuentes-Montero, Luis; Vollmar, Melanie; Michels-Clark, Tara; Young, Iris D.; Sauter, Nicholas K.; Evans, Gwyndaf</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Structural Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">85-97</span>CODEN:
                <span class="NLM_cas:coden">ACSDAD</span>;
        ISSN:<span class="NLM_cas:issn">2059-7983</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The DIALS project is a collaboration between Diamond Light Source, Lawrence Berkeley National Lab. and CCP4 to develop a new software suite for the anal. of crystallog. X-ray diffraction data, initially encompassing spot finding, indexing, refinement and integration.  The design, core algorithms and structure of the software are introduced, alongside results from the anal. of data from biol. and chem. crystallog. expts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkZqQrzPvxlbVg90H21EOLACvtfcHk0ljrvhw1TD-6KA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXksFarurk%253D&md5=f4572135ac40643815064914216f6d4c</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1107%2FS2059798317017235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS2059798317017235%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DG.%26aulast%3DWaterman%26aufirst%3DD.%2BG.%26aulast%3DParkhurst%26aufirst%3DJ.%2BM.%26aulast%3DBrewster%26aufirst%3DA.%2BS.%26aulast%3DGildea%26aufirst%3DR.%2BJ.%26aulast%3DGerstel%26aufirst%3DM.%26aulast%3DFuentes-Montero%26aufirst%3DL.%26aulast%3DVollmar%26aufirst%3DM.%26aulast%3DMichels-Clark%26aufirst%3DT.%26aulast%3DYoung%26aufirst%3DI.%2BD.%26aulast%3DSauter%26aufirst%3DN.%2BK.%26aulast%3DEvans%26aufirst%3DG.%26atitle%3DDIALS%253A%2520implementation%2520and%2520evaluation%2520of%2520a%2520new%2520integration%2520package%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Struct.%2520Biol.%26date%3D2018%26volume%3D74%26spage%3D85%26epage%3D97%26doi%3D10.1107%2Fs2059798317017235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkóczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1107/s0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.-W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+a+comprehensive+Python-based+system+for+macromolecular+structure+solution&doi=10.1107%2Fs0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0ljrvhw1TD-6KA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.-W.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520a%2520comprehensive%2520Python-based%2520system%2520for%2520macromolecular%2520structure%2520solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2Fs0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/s0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2Fs0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0li5hOAsOCh_3A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2Fs0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vargas, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piszczek, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brautigam, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuck, P.</span></span> <span> </span><span class="NLM_article-title">High-precision isothermal titration calorimetry with automated peak-shape analysis</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>84</i></span>,  <span class="NLM_fpage">5066</span>– <span class="NLM_lpage">5073</span>, <span class="refDoi"> DOI: 10.1021/ac3007522</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ac3007522" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC38XmtVGlsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2012&pages=5066-5073&author=S.+Kellerauthor=C.+Vargasauthor=H.+Zhaoauthor=G.+Piszczekauthor=C.+A.+Brautigamauthor=P.+Schuck&title=High-precision+isothermal+titration+calorimetry+with+automated+peak-shape+analysis&doi=10.1021%2Fac3007522"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">High-Precision Isothermal Titration Calorimetry with Automated Peak-Shape Analysis</span></div><div class="casAuthors">Keller, Sandro; Vargas, Carolyn; Zhao, Huaying; Piszczek, Grzegorz; Brautigam, Chad A.; Schuck, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">84</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5066-5073</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Isothermal titrn. calorimetry (ITC) is a powerful classical method that enables researchers in many fields to study the thermodn. of mol. interactions.  Primary ITC data comprise the temporal evolution of differential power reporting the heat of reaction during a series of injections of aliquots of a reactant into a sample cell.  By integration of each injection peak, an isotherm can be constructed of total changes in enthalpy as a function of changes in soln. compn., which is rich in thermodn. information on the reaction.  However, the signals from the injection peaks are superimposed by the stochastically varying time-course of the instrumental baseline power, limiting the precision of ITC isotherms.  The authors describe a method for automated peak assignment based on peak-shape anal. via singular value decompn. in combination with detailed least-squares modeling of local pre- and post- injection baselines.  This approach can effectively filter out contributions of short-term noise and adventitious events in the power trace.  This method also provides, for the first time, statistical error ests. for the individual isotherm data points.  In turn, this results in improved detection limits for high-affinity or low-enthalpy binding reactions and significantly higher precision of the derived thermodn. parameters.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJcKwO_fHPFbVg90H21EOLACvtfcHk0li5hOAsOCh_3A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmtVGlsbs%253D&md5=bb157e956fa9cef94a88974d733cad1b</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Fac3007522&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac3007522%26sid%3Dliteratum%253Aachs%26aulast%3DKeller%26aufirst%3DS.%26aulast%3DVargas%26aufirst%3DC.%26aulast%3DZhao%26aufirst%3DH.%26aulast%3DPiszczek%26aufirst%3DG.%26aulast%3DBrautigam%26aufirst%3DC.%2BA.%26aulast%3DSchuck%26aufirst%3DP.%26atitle%3DHigh-precision%2520isothermal%2520titration%2520calorimetry%2520with%2520automated%2520peak-shape%2520analysis%26jtitle%3DAnal.%2520Chem.%26date%3D2012%26volume%3D84%26spage%3D5066%26epage%3D5073%26doi%3D10.1021%2Fac3007522" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Langmead, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trapnell, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pop, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salzberg, S. L.</span></span> <span> </span><span class="NLM_article-title">Ultrafast and memory-efficient alignment of short DNA sequences to the human genome</span>. <i>Genome Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">R25</span>, <span class="refDoi"> DOI: 10.1186/gb-2009-10-3-r25</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1186%2Fgb-2009-10-3-r25" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=19261174" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A280%3ADC%252BD1MzotVGnsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2009&pages=R25&author=B.+Langmeadauthor=C.+Trapnellauthor=M.+Popauthor=S.+L.+Salzberg&title=Ultrafast+and+memory-efficient+alignment+of+short+DNA+sequences+to+the+human+genome&doi=10.1186%2Fgb-2009-10-3-r25"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Ultrafast and memory-efficient alignment of short DNA sequences to the human genome</span></div><div class="casAuthors">Langmead Ben; Trapnell Cole; Pop Mihai; Salzberg Steven L</div><div class="citationInfo"><span class="NLM_cas:title">Genome biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">R25</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Bowtie is an ultrafast, memory-efficient alignment program for aligning short DNA sequence reads to large genomes.  For the human genome, Burrows-Wheeler indexing allows Bowtie to align more than 25 million reads per CPU hour with a memory footprint of approximately 1.3 gigabytes.  Bowtie extends previous Burrows-Wheeler techniques with a novel quality-aware backtracking algorithm that permits mismatches.  Multiple processor cores can be used simultaneously to achieve even greater alignment speeds.  Bowtie is open source (http://bowtie.cbcb.umd.edu).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSod2io_r8eDEloaUqgkFHkfW6udTcc2eafQlowveyMSLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1MzotVGnsQ%253D%253D&md5=710abfa0067372cd5f72a3cbc0fe4271</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1186%2Fgb-2009-10-3-r25&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fgb-2009-10-3-r25%26sid%3Dliteratum%253Aachs%26aulast%3DLangmead%26aufirst%3DB.%26aulast%3DTrapnell%26aufirst%3DC.%26aulast%3DPop%26aufirst%3DM.%26aulast%3DSalzberg%26aufirst%3DS.%2BL.%26atitle%3DUltrafast%2520and%2520memory-efficient%2520alignment%2520of%2520short%2520DNA%2520sequences%2520to%2520the%2520human%2520genome%26jtitle%3DGenome%2520Biol.%26date%3D2009%26volume%3D10%26spage%3DR25%26doi%3D10.1186%2Fgb-2009-10-3-r25" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eeckhoute, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernstein, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussbaum, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myers, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X. S.</span></span> <span> </span><span class="NLM_article-title">Model-based analysis of ChIP-Seq (MACS)</span>. <i>Genome Biol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">R137</span>, <span class="refDoi"> DOI: 10.1186/gb-2008-9-9-r137</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1186%2Fgb-2008-9-9-r137" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=18798982" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2008&pages=R137&author=Y.+Zhangauthor=T.+Liuauthor=C.+A.+Meyerauthor=J.+Eeckhouteauthor=D.+S.+Johnsonauthor=B.+E.+Bernsteinauthor=C.+Nussbaumauthor=R.+M.+Myersauthor=M.+Brownauthor=W.+Liauthor=X.+S.+Liu&title=Model-based+analysis+of+ChIP-Seq+%28MACS%29&doi=10.1186%2Fgb-2008-9-9-r137"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1186%2Fgb-2008-9-9-r137&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fgb-2008-9-9-r137%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DT.%26aulast%3DMeyer%26aufirst%3DC.%2BA.%26aulast%3DEeckhoute%26aufirst%3DJ.%26aulast%3DJohnson%26aufirst%3DD.%2BS.%26aulast%3DBernstein%26aufirst%3DB.%2BE.%26aulast%3DNussbaum%26aufirst%3DC.%26aulast%3DMyers%26aufirst%3DR.%2BM.%26aulast%3DBrown%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DX.%2BS.%26atitle%3DModel-based%2520analysis%2520of%2520ChIP-Seq%2520%2528MACS%2529%26jtitle%3DGenome%2520Biol.%26date%3D2008%26volume%3D9%26spage%3DR137%26doi%3D10.1186%2Fgb-2008-9-9-r137" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scheer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ackloo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medina, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Northrop, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLeod, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zepeda-Velazquez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Carvalho, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaph, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C.</span></span> <span> </span><span class="NLM_article-title">A chemical biology toolbox to study protein methyltransferases and epigenetic signaling</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">19</span>, <span class="refDoi"> DOI: 10.1038/s41467-018-07905-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1038%2Fs41467-018-07905-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=30604761" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC1MXmvVCmtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=19&author=S.+Scheerauthor=S.+Acklooauthor=T.+Medinaauthor=M.+Schapiraauthor=F.+Liauthor=J.+Wardauthor=A.+Lewisauthor=J.+Northropauthor=P.+Richardsonauthor=H.+Kaniskanauthor=Y.+Shenauthor=J.+Liuauthor=D.+Smilauthor=D.+McLeodauthor=C.+Zepeda-Velazquezauthor=M.+Luoauthor=J.+Jinauthor=D.+Barsyte-Lovejoyauthor=K.+Huberauthor=D.+De+Carvalhoauthor=M.+Vedadiauthor=C.+Zaphauthor=P.+Brownauthor=C.+Arrowsmith&title=A+chemical+biology+toolbox+to+study+protein+methyltransferases+and+epigenetic+signaling&doi=10.1038%2Fs41467-018-07905-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">A chemical biology toolbox to study protein methyltransferases and epigenetic signaling</span></div><div class="casAuthors">Scheer, Sebastian; Ackloo, Suzanne; Medina, Tiago S.; Schapira, Matthieu; Li, Fengling; Ward, Jennifer A.; Lewis, Andrew M.; Northrop, Jeffrey P.; Richardson, Paul L.; Kaniskan, H. Umit; Shen, Yudao; Liu, Jing; Smil, David; McLeod, David; Zepeda-Velazquez, Carlos A.; Luo, Minkui; Jin, Jian; Barsyte-Lovejoy, Dalia; Huber, Kilian V. M.; De Carvalho, Daniel D.; Vedadi, Masoud; Zaph, Colby; Brown, Peter J.; Arrowsmith, Cheryl H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">19</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Protein methyltransferases (PMTs) comprise a major class of epigenetic regulatory enzymes with therapeutic relevance.  Here we present a collection of chem. probes and assocd. reagents and data to elucidate the function of human and murine PMTs in cellular studies.  Our collection provides inhibitors and antagonists that together modulate most of the key regulatory methylation marks on histones H3 and H4, providing an important resource for modulating cellular epigenomes.  We describe a comprehensive and comparative characterization of the probe collection with respect to their potency, selectivity, and mode of inhibition.  We demonstrate the utility of this collection in CD4+ T cell differentiation assays revealing the potential of individual probes to alter multiple T cell subpopulations which may have implications for T cell-mediated processes such as inflammation and immuno-oncol.  In particular, we demonstrate a role for DOT1L in limiting Th1 cell differentiation and maintaining lineage integrity.  This chem. probe collection and assocd. data form a resource for the study of methylation-mediated signaling in epigenetics, inflammation and beyond.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopA5rgeiWDvrVg90H21EOLACvtfcHk0lglnj_xaPsKLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXmvVCmtro%253D&md5=008fec084426171f5c91657e54b73900</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1038%2Fs41467-018-07905-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-018-07905-4%26sid%3Dliteratum%253Aachs%26aulast%3DScheer%26aufirst%3DS.%26aulast%3DAckloo%26aufirst%3DS.%26aulast%3DMedina%26aufirst%3DT.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DWard%26aufirst%3DJ.%26aulast%3DLewis%26aufirst%3DA.%26aulast%3DNorthrop%26aufirst%3DJ.%26aulast%3DRichardson%26aufirst%3DP.%26aulast%3DKaniskan%26aufirst%3DH.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DMcLeod%26aufirst%3DD.%26aulast%3DZepeda-Velazquez%26aufirst%3DC.%26aulast%3DLuo%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DHuber%26aufirst%3DK.%26aulast%3DDe%2BCarvalho%26aufirst%3DD.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DZaph%26aufirst%3DC.%26aulast%3DBrown%26aufirst%3DP.%26aulast%3DArrowsmith%26aufirst%3DC.%26atitle%3DA%2520chemical%2520biology%2520toolbox%2520to%2520study%2520protein%2520methyltransferases%2520and%2520epigenetic%2520signaling%26jtitle%3DNat.%2520Commun.%26date%3D2019%26volume%3D10%26spage%3D19%26doi%3D10.1038%2Fs41467-018-07905-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sciaky, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hulbert, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Everitt, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.-h.</span></span> <span> </span><span class="NLM_article-title">Targeting the histone methyltransferase G9a activates imprinted genes and improves survival of a mouse model of Prader-Willi syndrome</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">222</span>, <span class="refDoi"> DOI: 10.1038/nm.4257</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1038%2Fnm.4257" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=28024084" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFWntbnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2017&pages=213-222&author=Y.+Kimauthor=H.-M.+Leeauthor=Y.+Xiongauthor=N.+Sciakyauthor=S.+W.+Hulbertauthor=X.+Caoauthor=J.+I.+Everittauthor=J.+Jinauthor=B.+L.+Rothauthor=Y.-h.+Jiang&title=Targeting+the+histone+methyltransferase+G9a+activates+imprinted+genes+and+improves+survival+of+a+mouse+model+of+Prader-Willi+syndrome&doi=10.1038%2Fnm.4257"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the histone methyltransferase G9a activates imprinted genes and improves survival of a mouse model of Prader-Willi syndrome</span></div><div class="casAuthors">Kim, Yuna; Lee, Hyeong-Min; Xiong, Yan; Sciaky, Noah; Hulbert, Samuel W.; Cao, Xinyu; Everitt, Jeffrey I.; Jin, Jian; Roth, Bryan L.; Jiang, Yong-hui</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-222</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Prader-Willi syndrome (PWS) is an imprinting disorder caused by a deficiency of paternally expressed gene(s) in the 15q11-q13 chromosomal region.  The regulation of imprinted gene expression in this region is coordinated by an imprinting center (PWS-IC).  In individuals with PWS, genes responsible for PWS on the maternal chromosome are present, but repressed epigenetically, which provides an opportunity for the use of epigenetic therapy to restore expression from the maternal copies of PWS-assocd. genes.  Through a high-content screen (HCS) of >9,000 small mols., we discovered that UNC0638 and UNC0642-two selective inhibitors of euchromatic histone lysine N-methyltransferase-2 (EHMT2, also known as G9a)-activated the maternal (m) copy of candidate genes underlying PWS, including the SnoRNA cluster SNORD116, in cells from humans with PWS and also from a mouse model of PWS carrying a paternal (p) deletion from small nuclear ribonucleoprotein N (Snrpn (S)) to ubiquitin protein ligase E3A (Ube3a (U)) (mouse model referred to hereafter as m+/pΔS-U).  Both UNC0642 and UNC0638 caused a selective redn. of the dimethylation of histone H3 lysine 9 (H3K9me2) at PWS-IC, without changing DNA methylation, when analyzed by bisulfite genomic sequencing.  This indicates that histone modification is essential for the imprinting of candidate genes underlying PWS.  UNC0642 displayed therapeutic effects in the PWS mouse model by improving the survival and the growth of m+/pΔS-U newborn pups.  This study provides the first proof of principle for an epigenetics-based therapy for PWS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQUdUqWlf7RbVg90H21EOLACvtfcHk0lglnj_xaPsKLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFWntbnJ&md5=78ccbdde43e3fe79ea72ffa385893d58</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1038%2Fnm.4257&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.4257%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DY.%26aulast%3DLee%26aufirst%3DH.-M.%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DSciaky%26aufirst%3DN.%26aulast%3DHulbert%26aufirst%3DS.%2BW.%26aulast%3DCao%26aufirst%3DX.%26aulast%3DEveritt%26aufirst%3DJ.%2BI.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DJiang%26aufirst%3DY.-h.%26atitle%3DTargeting%2520the%2520histone%2520methyltransferase%2520G9a%2520activates%2520imprinted%2520genes%2520and%2520improves%2520survival%2520of%2520a%2520mouse%2520model%2520of%2520Prader-Willi%2520syndrome%26jtitle%3DNat.%2520Med.%26date%3D2017%26volume%3D23%26spage%3D213%26epage%3D222%26doi%3D10.1038%2Fnm.4257" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 10 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Marton  Siklos</span>, <span class="hlFld-ContribAuthor ">Stefan  Kubicek</span>. </span><span class="cited-content_cbyCitation_article-title">T
              herapeutic targeting of chromatin: status and opportunities. </span><span class="cited-content_cbyCitation_journal-name">The FEBS Journal</span><span> <strong>2021,</strong> <em>17 </em><a href="https://doi.org/10.1111/febs.15966" title="DOI URL">https://doi.org/10.1111/febs.15966</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1111/febs.15966&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1111%2Ffebs.15966%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520FEBS%2520Journal%26atitle%3DT%252Bherapeutic%252Btargeting%252Bof%252Bchromatin%25253A%252Bstatus%252Band%252Bopportunities%26aulast%3DSiklos%26aufirst%3DMarton%26date%3D2021%26date%3D2021%26volume%3D17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chiara  Luise</span>, <span class="hlFld-ContribAuthor ">Dina  Robaa</span>, <span class="hlFld-ContribAuthor ">Pierre  Regenass</span>, <span class="hlFld-ContribAuthor ">David  Maurer</span>, <span class="hlFld-ContribAuthor ">Dmytro  Ostrovskyi</span>, <span class="hlFld-ContribAuthor ">Ludwig  Seifert</span>, <span class="hlFld-ContribAuthor ">Johannes  Bacher</span>, <span class="hlFld-ContribAuthor ">Teresa  Burgahn</span>, <span class="hlFld-ContribAuthor ">Tobias  Wagner</span>, <span class="hlFld-ContribAuthor ">Johannes  Seitz</span>, <span class="hlFld-ContribAuthor ">Holger  Greschik</span>, <span class="hlFld-ContribAuthor ">Kwang-Su  Park</span>, <span class="hlFld-ContribAuthor ">Yan  Xiong</span>, <span class="hlFld-ContribAuthor ">Jian  Jin</span>, <span class="hlFld-ContribAuthor ">Roland  Schüle</span>, <span class="hlFld-ContribAuthor ">Bernhard  Breit</span>, <span class="hlFld-ContribAuthor ">Manfred  Jung</span>, <span class="hlFld-ContribAuthor ">Wolfgang  Sippl</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-Based Design, Docking and Binding Free Energy Calculations of A366 Derivatives as Spindlin1 Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2021,</strong> <em>22 </em>
                                    (11)
                                     , 5910. <a href="https://doi.org/10.3390/ijms22115910" title="DOI URL">https://doi.org/10.3390/ijms22115910</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms22115910&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms22115910%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DStructure-Based%252BDesign%25252C%252BDocking%252Band%252BBinding%252BFree%252BEnergy%252BCalculations%252Bof%252BA366%252BDerivatives%252Bas%252BSpindlin1%252BInhibitors%26aulast%3DLuise%26aufirst%3DChiara%26date%3D2021%26date%3D2021%26volume%3D22%26issue%3D11%26spage%3D5910" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chiara  Luise</span>, <span class="hlFld-ContribAuthor ">Dina  Robaa</span>, <span class="hlFld-ContribAuthor ">Wolfgang  Sippl</span>. </span><span class="cited-content_cbyCitation_article-title">Exploring aromatic cage flexibility of the histone methyllysine reader protein Spindlin1 and its impact on binding mode prediction: an in silico study. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2021,</strong> <em>35 </em>
                                    (6)
                                     , 695-706. <a href="https://doi.org/10.1007/s10822-021-00391-9" title="DOI URL">https://doi.org/10.1007/s10822-021-00391-9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-021-00391-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-021-00391-9%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DExploring%252Baromatic%252Bcage%252Bflexibility%252Bof%252Bthe%252Bhistone%252Bmethyllysine%252Breader%252Bprotein%252BSpindlin1%252Band%252Bits%252Bimpact%252Bon%252Bbinding%252Bmode%252Bprediction%25253A%252Ban%252Bin%252Bsilico%252Bstudy%26aulast%3DLuise%26aufirst%3DChiara%26date%3D2021%26date%3D2021%26volume%3D35%26issue%3D6%26spage%3D695%26epage%3D706" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yinping  Guo</span>, <span class="hlFld-ContribAuthor ">Xin  Mao</span>, <span class="hlFld-ContribAuthor ">Liang  Xiong</span>, <span class="hlFld-ContribAuthor ">Anjie  Xia</span>, <span class="hlFld-ContribAuthor ">Jing  You</span>, <span class="hlFld-ContribAuthor ">Guifeng  Lin</span>, <span class="hlFld-ContribAuthor ">Chengyong  Wu</span>, <span class="hlFld-ContribAuthor ">Luyi  Huang</span>, <span class="hlFld-ContribAuthor ">Yiwei  Wang</span>, <span class="hlFld-ContribAuthor ">Shengyong  Yang</span>. </span><span class="cited-content_cbyCitation_article-title">Structure‐Guided Discovery of a Potent and Selective Cell‐Active Inhibitor of SETDB1 Tudor Domain. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie International Edition</span><span> <strong>2021,</strong> <em>60 </em>
                                    (16)
                                     , 8760-8765. <a href="https://doi.org/10.1002/anie.202017200" title="DOI URL">https://doi.org/10.1002/anie.202017200</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/anie.202017200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fanie.202017200%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%2520International%2520Edition%26atitle%3DStructure%2525E2%252580%252590Guided%252BDiscovery%252Bof%252Ba%252BPotent%252Band%252BSelective%252BCell%2525E2%252580%252590Active%252BInhibitor%252Bof%252BSETDB1%252BTudor%252BDomain%26aulast%3DGuo%26aufirst%3DYinping%26date%3D2021%26date%3D2021%26volume%3D60%26issue%3D16%26spage%3D8760%26epage%3D8765" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yinping  Guo</span>, <span class="hlFld-ContribAuthor ">Xin  Mao</span>, <span class="hlFld-ContribAuthor ">Liang  Xiong</span>, <span class="hlFld-ContribAuthor ">Anjie  Xia</span>, <span class="hlFld-ContribAuthor ">Jing  You</span>, <span class="hlFld-ContribAuthor ">Guifeng  Lin</span>, <span class="hlFld-ContribAuthor ">Chengyong  Wu</span>, <span class="hlFld-ContribAuthor ">Luyi  Huang</span>, <span class="hlFld-ContribAuthor ">Yiwei  Wang</span>, <span class="hlFld-ContribAuthor ">Shengyong  Yang</span>. </span><span class="cited-content_cbyCitation_article-title">Structure‐Guided Discovery of a Potent and Selective Cell‐Active Inhibitor of SETDB1 Tudor Domain. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie</span><span> <strong>2021,</strong> <em>133 </em>
                                    (16)
                                     , 8842-8847. <a href="https://doi.org/10.1002/ange.202017200" title="DOI URL">https://doi.org/10.1002/ange.202017200</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ange.202017200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fange.202017200%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%26atitle%3DStructure%2525E2%252580%252590Guided%252BDiscovery%252Bof%252Ba%252BPotent%252Band%252BSelective%252BCell%2525E2%252580%252590Active%252BInhibitor%252Bof%252BSETDB1%252BTudor%252BDomain%26aulast%3DGuo%26aufirst%3DYinping%26date%3D2021%26date%3D2021%26volume%3D133%26issue%3D16%26spage%3D8842%26epage%3D8847" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jenna N.  Beyer</span>, <span class="hlFld-ContribAuthor ">Nicole R.  Raniszewski</span>, <span class="hlFld-ContribAuthor ">George M.  Burslem</span>. </span><span class="cited-content_cbyCitation_article-title">Advances and Opportunities in Epigenetic Chemical Biology. </span><span class="cited-content_cbyCitation_journal-name">ChemBioChem</span><span> <strong>2021,</strong> <em>22 </em>
                                    (1)
                                     , 17-42. <a href="https://doi.org/10.1002/cbic.202000459" title="DOI URL">https://doi.org/10.1002/cbic.202000459</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cbic.202000459&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcbic.202000459%26sid%3Dliteratum%253Aachs%26jtitle%3DChemBioChem%26atitle%3DAdvances%252Band%252BOpportunities%252Bin%252BEpigenetic%252BChemical%252BBiology%26aulast%3DBeyer%26aufirst%3DJenna%2BN.%26date%3D2021%26date%3D2020%26volume%3D22%26issue%3D1%26spage%3D17%26epage%3D42" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xingyu  Liu</span>, <span class="hlFld-ContribAuthor ">Ying  Zhang</span>, <span class="hlFld-ContribAuthor ">Zhihui  Wen</span>, <span class="hlFld-ContribAuthor ">Yan  Hao</span>, <span class="hlFld-ContribAuthor ">Charles A. S.  Banks</span>, <span class="hlFld-ContribAuthor ">Jeffrey J.  Lange</span>, <span class="hlFld-ContribAuthor ">Brian D.  Slaughter</span>, <span class="hlFld-ContribAuthor ">Jay R.  Unruh</span>, <span class="hlFld-ContribAuthor ">Laurence  Florens</span>, <span class="hlFld-ContribAuthor ">Susan M.  Abmayr</span>, <span class="hlFld-ContribAuthor ">Jerry L.  Workman</span>, <span class="hlFld-ContribAuthor ">Michael P.  Washburn</span>. </span><span class="cited-content_cbyCitation_article-title">Driving integrative structural modeling with serial capture affinity purification. </span><span class="cited-content_cbyCitation_journal-name">Proceedings of the National Academy of Sciences</span><span> <strong>2020,</strong> <em>117 </em>
                                    (50)
                                     , 31861-31870. <a href="https://doi.org/10.1073/pnas.2007931117" title="DOI URL">https://doi.org/10.1073/pnas.2007931117</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1073/pnas.2007931117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1073%2Fpnas.2007931117%26sid%3Dliteratum%253Aachs%26jtitle%3DProceedings%2520of%2520the%2520National%2520Academy%2520of%2520Sciences%26atitle%3DDriving%252Bintegrative%252Bstructural%252Bmodeling%252Bwith%252Bserial%252Bcapture%252Baffinity%252Bpurification%26aulast%3DLiu%26aufirst%3DXingyu%26date%3D2020%26date%3D2020%26volume%3D117%26issue%3D50%26spage%3D31861%26epage%3D31870" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Velten  Horn</span>, <span class="hlFld-ContribAuthor ">Seino A. K.  Jongkees</span>, <span class="hlFld-ContribAuthor ">Hugo  van Ingen</span>. </span><span class="cited-content_cbyCitation_article-title">Mimicking the Nucleosomal Context in Peptide-Based Binders of a H3K36me Reader Increases Binding Affinity While Altering the Binding Mode. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2020,</strong> <em>25 </em>
                                    (21)
                                     , 4951. <a href="https://doi.org/10.3390/molecules25214951" title="DOI URL">https://doi.org/10.3390/molecules25214951</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules25214951&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules25214951%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DMimicking%252Bthe%252BNucleosomal%252BContext%252Bin%252BPeptide-Based%252BBinders%252Bof%252Ba%252BH3K36me%252BReader%252BIncreases%252BBinding%252BAffinity%252BWhile%252BAltering%252Bthe%252BBinding%252BMode%26aulast%3DHorn%26aufirst%3DVelten%26date%3D2020%26date%3D2020%26volume%3D25%26issue%3D21%26spage%3D4951" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alessandra  Cipriano</span>, <span class="hlFld-ContribAuthor ">Gianluca  Sbardella</span>, <span class="hlFld-ContribAuthor ">Alessio  Ciulli</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting epigenetic reader domains by chemical biology. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2020,</strong> <em>57 </em>, 82-94. <a href="https://doi.org/10.1016/j.cbpa.2020.05.006" title="DOI URL">https://doi.org/10.1016/j.cbpa.2020.05.006</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2020.05.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2020.05.006%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DTargeting%252Bepigenetic%252Breader%252Bdomains%252Bby%252Bchemical%252Bbiology%26aulast%3DCipriano%26aufirst%3DAlessandra%26date%3D2020%26volume%3D57%26spage%3D82%26epage%3D94" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Brian M  Linhares</span>, <span class="hlFld-ContribAuthor ">Jolanta  Grembecka</span>, <span class="hlFld-ContribAuthor ">Tomasz  Cierpicki</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting epigenetic protein–protein interactions with small-molecule inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2020,</strong> <em>12 </em>
                                    (14)
                                     , 1305-1326. <a href="https://doi.org/10.4155/fmc-2020-0082" title="DOI URL">https://doi.org/10.4155/fmc-2020-0082</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2020-0082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2020-0082%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DTargeting%252Bepigenetic%252Bprotein%2525E2%252580%252593protein%252Binteractions%252Bwith%252Bsmall-molecule%252Binhibitors%26aulast%3DLinhares%26aufirst%3DBrian%2BM%26date%3D2020%26volume%3D12%26issue%3D14%26spage%3D1305%26epage%3D1326" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/medium/jm-2019-00522j_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/large/jm-2019-00522j_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00522&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/medium/jm-2019-00522j_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/large/jm-2019-00522j_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Representative inhibitors of methyllysine reader proteins.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/large/jm-2019-00522j_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00522&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/medium/jm-2019-00522j_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/large/jm-2019-00522j_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Discovery of compound <b>3</b> and its inactive control compound <b>4</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/large/jm-2019-00522j_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00522&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/medium/jm-2019-00522j_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/large/jm-2019-00522j_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Compound <b>2</b> is a selective SPIN 1 inhibitor. Compound <b>2</b> inhibited the interactions between SPIN1 and H3K4me3 in a concentration dependent manner with IC<sub>50</sub> values of (A) 338 ± 30 nM in AlphaLISA assay (<i>n</i> = 3) and (B) 741 ± 46 nM in FP assay (<i>n</i> = 3). (C) Compound <b>2</b> bound SPIN1 with a <i>K</i><sub>D</sub> value of 390 ± 54 nM in ITC assay (<i>n</i> = 3). (D) Compound <b>2</b> did not bind G9a.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/large/jm-2019-00522j_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00522&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/medium/jm-2019-00522j_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/large/jm-2019-00522j_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Compound <b>3</b> is a potent SPIN1 inhibitor. (A) Compound <b>3</b>, but not compound <b>4</b>, potently inhibited the interactions between SPIN1 and H3K4me3 with IC<sub>50</sub> values of 77 ± 3 nM in AlphaLISA assay (<i>n</i> = 3) and 243 ± 10 nM in FP assay (<i>n</i> = 3). (B) Compound <b>3</b> bound SPIN1 with a <i>K</i><sub>D</sub> value of 91 ± 4 nM (<i>n</i> = 3) while compound <b>4</b> did not bind SPIN1 in ITC experiments. (C) Compound <b>3</b> did not bind either G9a or GLP in ITC experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/large/jm-2019-00522j_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00522&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/medium/jm-2019-00522j_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/large/jm-2019-00522j_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Compound <b>3</b> is a selective SPIN1 inhibitor. (A) Compound <b>3</b> did not significantly inhibit the enzymatic activity of 33 MTs and 5 acetyltransferases at 10 μM (blue) and 50 μM (red). (B) Compound <b>3</b> did not induce significant thermal shifts for 20 methyllysine, methylarginine and acetyllysine reader proteins at 20 μM or 200 μM in thermal shift assays (<i>n</i> = 2).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/large/jm-2019-00522j_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00522&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/medium/jm-2019-00522j_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/large/jm-2019-00522j_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Crystal structure of the SPIN1–compound <b>3</b> complex. (A) Compound <b>3</b> (green) occupies the tudor domain II (pink) of SPIN1. (B) Electrostatic potential surface view of the structure, ranging from −6 <i>kT</i>/e (red) to +6 <i>kT</i>/e (blue). (C) Close-up view of the SPIN1–compound <b>3</b> complex structure with key ligand–protein interactions. Yellow dashes, hydrogen bonds. Red balls, water molecules.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/large/jm-2019-00522j_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00522&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/medium/jm-2019-00522j_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/large/jm-2019-00522j_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Compound <b>3</b> binds SPIN1 in cells and shows no cytotoxicity. (A) Compound <b>3</b> disrupted the interaction between SPIN1 and histone H3 (IC<sub>50</sub> = 3.2 ± 0.7 μM (<i>n</i> = 4)) in U2OS cells in a NanoBRET assay. (B) Compound <b>4</b> was inactive in the NanoBRET assay (<i>n</i> = 3). (C) Compound <b>3</b> was not toxic to C2C12 and 293T cell lines or primary fibroblasts HFF1, at up to 30 μM (<i>n</i> = 3). Indicated cell lines were cultured in the presence of indicated concentrations of compound <b>3</b> for 6 days. Cell viability was measured using Alamar blue.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/large/jm-2019-00522j_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00522&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/medium/jm-2019-00522j_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/large/jm-2019-00522j_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compound <b>2</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/large/jm-2019-00522j_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00522&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagent and conditions: (a) BnBr, K<sub>2</sub>CO<sub>3</sub>, DMF, rt, 12 h, 90%; (b) CuCN, NMP, 170 °C, 18 h, 40%; (c) H<sub>2</sub> (1 atm), 10% Pd/C, EtOAc, rt, 5 h; (d) 1-bromo-3-chloropropane, DMF, rt, 12 h, 60% in 2 steps; (e) pyrrolidine, K<sub>2</sub>CO<sub>3</sub>, KI, DMF, 60 °C, 12 h, 80%; (f) H<sub>2</sub> (1 atm), Raney Ni, NH<sub>4</sub>OH, MeOH, rt, 12 h, 70%.</p></p></figure><figure data-id="sch2" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/medium/jm-2019-00522j_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/large/jm-2019-00522j_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compound <b>3</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/large/jm-2019-00522j_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00522&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagent and conditions: (a) isoindoline, K<sub>2</sub>CO<sub>3</sub>, KI, DMF, 60 °C, 12 h, 60%; (b) H<sub>2</sub> (1 atm), Raney Ni, NH<sub>4</sub>OH, MeOH, rt, 12 h, 65%.</p></p></figure><figure data-id="sch3" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/medium/jm-2019-00522j_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/large/jm-2019-00522j_0010.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compound <b>4</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00522/20191018/images/large/jm-2019-00522j_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00522&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagent and conditions: (a) BnBr, K<sub>2</sub>CO<sub>3</sub>, DMF, rt, 12 h, 91%; (b) CuCN, NMP, 170 °C, 18 h, 54%; (c) NaOH, MeOH, H<sub>2</sub>O, rt, 2 h. (d) (COCl)<sub>2</sub>, DMF, CH<sub>2</sub>Cl<sub>2</sub>, rt, 1 h; NH<sub>4</sub>OH, 0 °C, 1 h, 41% in 3 steps; (e) H<sub>2</sub> (1 atm), 10% Pd/C, EtOAc, rt, 5 h; (f) 1-bromo-3-chloropropane, DMF, rt, 12 h, 55% in 2 steps; (g) isoindoline, K<sub>2</sub>CO<sub>3</sub>, KI, DMF, 60 °C, 12 h, 61%; (h) H<sub>2</sub> (1 atm), Raney Ni, NH<sub>4</sub>OH, MeOH, rt, 12 h, 72%.</p></p></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i46">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_05141" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_05141" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 59 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bernstein, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meissner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lander, E. S.</span></span> <span> </span><span class="NLM_article-title">The mammalian epigenome</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>128</i></span>,  <span class="NLM_fpage">669</span>– <span class="NLM_lpage">681</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2007.01.033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1016%2Fj.cell.2007.01.033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=17320505" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BD2sXis12ju7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2007&pages=669-681&author=B.+E.+Bernsteinauthor=A.+Meissnerauthor=E.+S.+Lander&title=The+mammalian+epigenome&doi=10.1016%2Fj.cell.2007.01.033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The mammalian epigenome</span></div><div class="casAuthors">Bernstein, Bradley E.; Meissner, Alexander; Lander, Eric S.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">669-681</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Chem. modifications to DNA and histone proteins form a complex regulatory network that modulates chromatin structure and genome function.  The epigenome refers to the complete description of these potentially heritable changes across the genome.  The compn. of the epigenome within a given cell is a function of genetic determinants, lineage, and environment.  With the sequencing of the human genome completed, investigators now seek a comprehensive view of the epigenetic changes that det. how genetic information is made manifest across an incredibly varied background of developmental stages, tissue types, and disease states.  Here we review current research efforts, with an emphasis on large-scale studies, emerging technologies, and challenges ahead.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKJfRRDsxAHbVg90H21EOLACvtfcHk0lipd7HCmTxC0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXis12ju7g%253D&md5=963fb487bbf473ac801e09a558ac0b0e</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.01.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.01.033%26sid%3Dliteratum%253Aachs%26aulast%3DBernstein%26aufirst%3DB.%2BE.%26aulast%3DMeissner%26aufirst%3DA.%26aulast%3DLander%26aufirst%3DE.%2BS.%26atitle%3DThe%2520mammalian%2520epigenome%26jtitle%3DCell%26date%3D2007%26volume%3D128%26spage%3D669%26epage%3D681%26doi%3D10.1016%2Fj.cell.2007.01.033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Strahl, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allis, C. D.</span></span> <span> </span><span class="NLM_article-title">The language of covalent histone modifications</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>403</i></span>,  <span class="NLM_fpage">41</span>– <span class="NLM_lpage">45</span>, <span class="refDoi"> DOI: 10.1038/47412</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1038%2F47412" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10638745" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A280%3ADC%252BD3c7gt1arsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=403&publication_year=2000&pages=41-45&author=B.+D.+Strahlauthor=C.+D.+Allis&title=The+language+of+covalent+histone+modifications&doi=10.1038%2F47412"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">The language of covalent histone modifications</span></div><div class="casAuthors">Strahl B D; Allis C D</div><div class="citationInfo"><span class="NLM_cas:title">Nature</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">403</span>
        (<span class="NLM_cas:issue">6765</span>),
    <span class="NLM_cas:pages">41-5</span>
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">Histone proteins and the nucleosomes they form with DNA are the fundamental building blocks of eukaryotic chromatin.  A diverse array of post-translational modifications that often occur on tail domains of these proteins has been well documented.  Although the function of these highly conserved modifications has remained elusive, converging biochemical and genetic evidence suggests functions in several chromatin-based processes.  We propose that distinct histone modifications, on one or more tails, act sequentially or in combination to form a 'histone code' that is, read by other proteins to bring about distinct downstream events.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSyzhdlYh9sX7t7lRqEFUJVfW6udTcc2eZMVDoA6NXGSbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3c7gt1arsQ%253D%253D&md5=2eb7d093151c6122ccd70fa2adcfd94b</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2F47412&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F47412%26sid%3Dliteratum%253Aachs%26aulast%3DStrahl%26aufirst%3DB.%2BD.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DThe%2520language%2520of%2520covalent%2520histone%2520modifications%26jtitle%3DNature%26date%3D2000%26volume%3D403%26spage%3D41%26epage%3D45%26doi%3D10.1038%2F47412" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rea, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eisenhaber, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Carroll, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strahl, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Z.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmid, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Opravil, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mechtler, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ponting, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allis, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenuwein, T.</span></span> <span> </span><span class="NLM_article-title">Regulation of chromatin structure by site-specific histone H3 methyltransferases</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>406</i></span>,  <span class="NLM_fpage">593</span>– <span class="NLM_lpage">599</span>, <span class="refDoi"> DOI: 10.1038/35020506</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1038%2F35020506" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10949293" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BD3cXmtFegs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=406&publication_year=2000&pages=593-599&author=S.+Reaauthor=F.+Eisenhaberauthor=D.+O%E2%80%99Carrollauthor=B.+D.+Strahlauthor=Z.-W.+Sunauthor=M.+Schmidauthor=S.+Opravilauthor=K.+Mechtlerauthor=C.+P.+Pontingauthor=C.+D.+Allisauthor=T.+Jenuwein&title=Regulation+of+chromatin+structure+by+site-specific+histone+H3+methyltransferases&doi=10.1038%2F35020506"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of chromatin structure by site-specific histone H3 methyltransferases</span></div><div class="casAuthors">Rea, Stephen; Elsenhaber, Frank; O'Carroll, Donal; Strahl, Brian D.; Sun, Zu-Wen; Schmid, Manfred; Opravil, Susanne; Mechtler, Karl; Ponting, Chris P.; Allis, C. David; Jenuwein, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">406</span>
        (<span class="NLM_cas:issue">6796</span>),
    <span class="NLM_cas:pages">593-599</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The organization of chromatin into higher-order structures influences chromosome function and epigenetic gene regulation.  Higher-order chromatin has been proposed to be nucleated by the covalent modification of histone tails and the subsequent establishment of chromosomal subdomains by non-histone modifier factors.  Here we show that human SUV39H1 and murine Suv39h1 - mammalian homologs of Drosophila Su(var)3-9 and of Schizosaccharomyces pombe clr4 - encode histone H3-specific methyltransferases that selectively methylate lysine 9 of the amino terminus of histone H3 in vitro.  We mapped the catalytic motif to the evolutionarily conserved SET domain, which requires adjacent cysteine-rich regions to confer histone methyltransferase activity.  Methylation of lysine 9 interferes with phosphorylation of serine 10, but is also influenced by pre-existing modifications in the amino terminus of H3.  In vivo, deregulated SUV39H1 or disrupted Suv39h activity modulate H3 serine 10 phosphorylation in native chromatin and induce aberrant mitotic divisions.  Our data reveal a functional interdependence of site-specific H3 tail modifications and suggest a dynamic mechanism for the regulation of higher-order chromatin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2yioyTahy77Vg90H21EOLACvtfcHk0lipd7HCmTxC0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXmtFegs7s%253D&md5=65e14733d25f97fadaaeed602600bd23</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2F35020506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F35020506%26sid%3Dliteratum%253Aachs%26aulast%3DRea%26aufirst%3DS.%26aulast%3DEisenhaber%26aufirst%3DF.%26aulast%3DO%25E2%2580%2599Carroll%26aufirst%3DD.%26aulast%3DStrahl%26aufirst%3DB.%2BD.%26aulast%3DSun%26aufirst%3DZ.-W.%26aulast%3DSchmid%26aufirst%3DM.%26aulast%3DOpravil%26aufirst%3DS.%26aulast%3DMechtler%26aufirst%3DK.%26aulast%3DPonting%26aufirst%3DC.%2BP.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26aulast%3DJenuwein%26aufirst%3DT.%26atitle%3DRegulation%2520of%2520chromatin%2520structure%2520by%2520site-specific%2520histone%2520H3%2520methyltransferases%26jtitle%3DNature%26date%3D2000%26volume%3D406%26spage%3D593%26epage%3D599%26doi%3D10.1038%2F35020506" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kouzarides, T.</span></span> <span> </span><span class="NLM_article-title">Chromatin modifications and their function</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>128</i></span>,  <span class="NLM_fpage">693</span>– <span class="NLM_lpage">705</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2007.02.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1016%2Fj.cell.2007.02.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=17320507" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BD2sXis12ju7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2007&pages=693-705&author=T.+Kouzarides&title=Chromatin+modifications+and+their+function&doi=10.1016%2Fj.cell.2007.02.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatin modifications and their function</span></div><div class="casAuthors">Kouzarides, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">693-705</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The surface of nucleosomes is studded with a multiplicity of modifications.  At least eight different classes have been characterized to date and many different sites have been identified for each class.  Operationally, modifications function either by disrupting chromatin contacts or by affecting the recruitment of nonhistone proteins to chromatin.  Their presence on histones can dictate the higher-order chromatin structure in which DNA is packaged and can orchestrate the ordered recruitment of enzyme complexes to manipulate DNA.  In this way, histone modifications have the potential to influence many fundamental biol. processes, some of which may be epigenetically inherited.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAH9AX2dGicbVg90H21EOLACvtfcHk0lipd7HCmTxC0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXis12ju7Y%253D&md5=c5995fe68fd9473f4268d59c01d3b9de</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.02.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.02.005%26sid%3Dliteratum%253Aachs%26aulast%3DKouzarides%26aufirst%3DT.%26atitle%3DChromatin%2520modifications%2520and%2520their%2520function%26jtitle%3DCell%26date%3D2007%26volume%3D128%26spage%3D693%26epage%3D705%26doi%3D10.1016%2Fj.cell.2007.02.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H. Ü.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Chemical probes of histone lysine methyltransferases</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">40</span>– <span class="NLM_lpage">50</span>, <span class="refDoi"> DOI: 10.1021/cb500785t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500785t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFOht7%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=40-50&author=H.+%C3%9C.+Kaniskanauthor=J.+Jin&title=Chemical+probes+of+histone+lysine+methyltransferases&doi=10.1021%2Fcb500785t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical probes of histone lysine methyltransferases</span></div><div class="casAuthors">Kaniskan, H. Umit; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">40-50</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Growing evidence suggests that histone-lysine methyltransferases (HMTs, also known as protein methyltransferases (PMTs)) play an important role in diverse biol. processes and human diseases by regulating gene expression and the chromatin state.  Therefore, HMTs have been increasingly recognized by the biomedical community as a class of potential therapeutic targets.  High quality chem. probes of HMTs, as tools for deciphering their physiol. functions and roles in human diseases and testing therapeutic hypotheses, are crit. for advancing this promising field.  Here, the authors focus on the discovery, characterization, and biol. applications of chem. probes for HMTs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpp8kb_c7Xl57Vg90H21EOLACvtfcHk0li_C-iRkWYnuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFOht7%252FE&md5=4eb8a696e2227c60686df8d3c60777ce</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Fcb500785t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500785t%26sid%3Dliteratum%253Aachs%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259C.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DChemical%2520probes%2520of%2520histone%2520lysine%2520methyltransferases%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D40%26epage%3D50%26doi%3D10.1021%2Fcb500785t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H. Ü.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konze, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Selective inhibitors of protein methyltransferases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">1596</span>– <span class="NLM_lpage">1629</span>, <span class="refDoi"> DOI: 10.1021/jm501234a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501234a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFKqtbbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=1596-1629&author=H.+%C3%9C.+Kaniskanauthor=K.+D.+Konzeauthor=J.+Jin&title=Selective+inhibitors+of+protein+methyltransferases&doi=10.1021%2Fjm501234a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibitors of protein methyltransferases</span></div><div class="casAuthors">Kaniskan, H. Umit; Konze, Kyle D.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1596-1629</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Mounting evidence suggests that protein methyltransferases (PMTs), which catalyze the methylation of histone and non-histone proteins, play a crucial role in diverse biol. processes and human diseases.  In particular, PMTs have been recognized as major players in regulating gene expression and chromatin state.  PMTs are divided into 2 categories: protein lysine methyltransferases (PKMTs) and protein arginine methyltransferases (PRMTs).  There has been a steadily growing interest in these enzymes as potential therapeutic targets and therefore discovery of PMT inhibitors has also been pursued increasingly over the past decade.  Here, the authors present a perspective on selective, small-mol. inhibitors of PMTs with an emphasis on their discovery, characterization, and applicability as chem. tools for deciphering the target PMTs' physiol. functions and involvement in human diseases.  The authors highlight the current state of PMT inhibitors and discuss future directions and opportunities for PMT inhibitor discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEWF_XeLEY1rVg90H21EOLACvtfcHk0li_C-iRkWYnuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFKqtbbJ&md5=8fb2b2d21d306c02a2da60a3fb88c430</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Fjm501234a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501234a%26sid%3Dliteratum%253Aachs%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259C.%26aulast%3DKonze%26aufirst%3DK.%2BD.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DSelective%2520inhibitors%2520of%2520protein%2520methyltransferases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D1596%26epage%3D1629%26doi%3D10.1021%2Fjm501234a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H. Ü.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martini, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Inhibitors of protein methyltransferases and demethylases</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">989</span>– <span class="NLM_lpage">1068</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.6b00801</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.6b00801" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkslynsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2018&pages=989-1068&author=H.+%C3%9C.+Kaniskanauthor=M.+L.+Martiniauthor=J.+Jin&title=Inhibitors+of+protein+methyltransferases+and+demethylases&doi=10.1021%2Facs.chemrev.6b00801"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of Protein Methyltransferases and Demethylases</span></div><div class="casAuthors">Kaniskan, H. Umit; Martini, Michael L.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">989-1068</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Post-translational modifications of histones by protein methyltransferases (PMTs) and histone demethylases (KDMs) play an important role in the regulation of gene expression and transcription and are implicated in cancer and many other diseases.  Many of these enzymes also target various nonhistone proteins impacting numerous crucial biol. pathways.  Given their key biol. functions and implications in human diseases, there has been a growing interest in assessing these enzymes as potential therapeutic targets.  Consequently, discovering and developing inhibitors of these enzymes has become a very active and fast-growing research area over the past decade.  In this review, the authors cover the discovery, characterization, and biol. application of inhibitors of PMTs and KDMs with emphasis on key advancements in the field.  The authors also discuss challenges, opportunities, and future directions in this emerging, exciting research field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJYjiYtw_n4rVg90H21EOLACvtfcHk0li_C-iRkWYnuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkslynsL8%253D&md5=1ed520c6b0c42d2aa8a7f66b0a2d4e40</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.6b00801&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.6b00801%26sid%3Dliteratum%253Aachs%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259C.%26aulast%3DMartini%26aufirst%3DM.%2BL.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DInhibitors%2520of%2520protein%2520methyltransferases%2520and%2520demethylases%26jtitle%3DChem.%2520Rev.%26date%3D2018%26volume%3D118%26spage%3D989%26epage%3D1068%26doi%3D10.1021%2Facs.chemrev.6b00801" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulligan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whetstine, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casero, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span> <span> </span><span class="NLM_article-title">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>119</i></span>,  <span class="NLM_fpage">941</span>– <span class="NLM_lpage">953</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2004.12.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1016%2Fj.cell.2004.12.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=15620353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2004&pages=941-953&author=Y.+Shiauthor=F.+Lanauthor=C.+Matsonauthor=P.+Mulliganauthor=J.+R.+Whetstineauthor=P.+A.+Coleauthor=R.+A.+Caseroauthor=Y.+Shi&title=Histone+demethylation+mediated+by+the+nuclear+amine+oxidase+homolog+LSD1&doi=10.1016%2Fj.cell.2004.12.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span></div><div class="casAuthors">Shi, Yujiang; Lan, Fei; Matson, Caitlin; Mulligan, Peter; Whetstine, Johnathan R.; Cole, Philip A.; Casero, Robert A.; Shi, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">941-953</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Posttranslational modifications of histone N-terminal tails impact chromatin structure and gene transcription.  While the extent of histone acetylation is detd. by both acetyltransferases and deacetylases, it has been unclear whether histone methylation is also regulated by enzymes with opposing activities.  Here, we provide evidence that LSD1 (KIAA0601), a nuclear homolog of amine oxidases, functions as a histone demethylase and transcriptional corepressor.  LSD1 specifically demethylates histone H3 lysine 4, which is linked to active transcription.  Lysine demethylation occurs via an oxidn. reaction that generates formaldehyde.  Importantly, RNAi inhibition of LSD1 causes an increase in H3 lysine 4 methylation and concomitant derepression of target genes, suggesting that LSD1 represses transcription via histone demethylation.  The results thus identify a histone demethylase conserved from S. pombe to human and reveal dynamic regulation of histone methylation by both histone methylases and demethylases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXHr-8kwGdXrVg90H21EOLACvtfcHk0lis9vE_miRS-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D&md5=3474569204255e90a832667ba654748b</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2004.12.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2004.12.012%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26aulast%3DLan%26aufirst%3DF.%26aulast%3DMatson%26aufirst%3DC.%26aulast%3DMulligan%26aufirst%3DP.%26aulast%3DWhetstine%26aufirst%3DJ.%2BR.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DHistone%2520demethylation%2520mediated%2520by%2520the%2520nuclear%2520amine%2520oxidase%2520homolog%2520LSD1%26jtitle%3DCell%26date%3D2004%26volume%3D119%26spage%3D941%26epage%3D953%26doi%3D10.1016%2Fj.cell.2004.12.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dhalluin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aggarwal, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.-M.</span></span> <span> </span><span class="NLM_article-title">Structure and ligand of a histone acetyltransferase bromodomain</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>399</i></span>,  <span class="NLM_fpage">491</span>– <span class="NLM_lpage">496</span>, <span class="refDoi"> DOI: 10.1038/20974</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1038%2F20974" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10365964" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADyaK1MXjvV2isLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=399&publication_year=1999&pages=491-496&author=C.+Dhalluinauthor=J.+E.+Carlsonauthor=L.+Zengauthor=C.+Heauthor=A.+K.+Aggarwalauthor=M.-M.+Zhouauthor=M.-M.+Zhou&title=Structure+and+ligand+of+a+histone+acetyltransferase+bromodomain&doi=10.1038%2F20974"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and ligand of a histone acetyltransferase bromodomain</span></div><div class="casAuthors">Dhalluin, Christophe; Carlson, Justin E.; Zeng, Lei; He, Cheng; Aggarwal, Aneel K.; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">399</span>
        (<span class="NLM_cas:issue">6735</span>),
    <span class="NLM_cas:pages">491-496</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">Histone acetylation is important in chromatin remodelling and gene activation.  Nearly all known histone-acetyltransferase (HAT)-assocd. transcriptional co-activators contain bromodomains, which are ∼110-amino-acid modules found in many chromatin-assocd. proteins.  Despite the wide occurrence of these bromodomains, their three-dimensional structure and binding partners remain unknown.  Here we report the soln. structure of the bromodomain of the HAT co-activator P/CAF (p300/CBP-assocd. factor).  The structure reveals an unusual left-handed up-and-down four-helix bundle.  In addn., we show by a combination of structural and site-directed mutagenesis studies that bromodomains can interact specifically with acetylated lysine, making them the first known protein modules to do so.  The nature of the recognition of acetyl-lysine by the P/CAF bromodomain is similar to that of acetyl-CoA by histone acetyltransferase.  Thus, the bromodomain is functionally linked to the HAT activity of co-activators in the regulation of gene transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsTbwcjeBG_rVg90H21EOLACvtfcHk0lis9vE_miRS-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjvV2isLo%253D&md5=ff1063ba81d81b431065dd6e3751ba79</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2F20974&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F20974%26sid%3Dliteratum%253Aachs%26aulast%3DDhalluin%26aufirst%3DC.%26aulast%3DCarlson%26aufirst%3DJ.%2BE.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DHe%26aufirst%3DC.%26aulast%3DAggarwal%26aufirst%3DA.%2BK.%26aulast%3DZhou%26aufirst%3DM.-M.%26aulast%3DZhou%26aufirst%3DM.-M.%26atitle%3DStructure%2520and%2520ligand%2520of%2520a%2520histone%2520acetyltransferase%2520bromodomain%26jtitle%3DNature%26date%3D1999%26volume%3D399%26spage%3D491%26epage%3D496%26doi%3D10.1038%2F20974" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Su, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denu, J. M.</span></span> <span> </span><span class="NLM_article-title">Reading the combinatorial histone language</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">564</span>– <span class="NLM_lpage">574</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00864</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00864" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVagsL%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=564-574&author=Z.+Suauthor=J.+M.+Denu&title=Reading+the+combinatorial+histone+language&doi=10.1021%2Facschembio.5b00864"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Reading the Combinatorial Histone Language</span></div><div class="casAuthors">Su, Zhangli; Denu, John M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">564-574</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Histones are subject to frequent combinatorial post-translational modifications (PTMs), forming a complex chem. "language" that is interpreted by PTM-specific histone-interacting protein modules (reader domains).  These specific interactions are thought to instruct gene expression and downstream biol. functions.  While the majority of studies have focused on individual modifications, the authors' current understanding of the combinatorial PTM patterns on histones is starting to emerge, benefiting from the convergence of multiple technologies.  Here, the authors review the key tech. advances and progress on discovery and characterization of combinatorial histone PTM patterns.  The authors focus on the interactions between reader domains and combinatorial PTMs, which is essential for understanding the mechanism and biol. meaning of establishing and interpreting information embedded in histone PTM patterns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHpmgsQnFL7rVg90H21EOLACvtfcHk0lis9vE_miRS-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVagsL%252FK&md5=819be9f59c1c1f0ae1ee09665b2d51da</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00864&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00864%26sid%3Dliteratum%253Aachs%26aulast%3DSu%26aufirst%3DZ.%26aulast%3DDenu%26aufirst%3DJ.%2BM.%26atitle%3DReading%2520the%2520combinatorial%2520histone%2520language%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D564%26epage%3D574%26doi%3D10.1021%2Facschembio.5b00864" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouzarides, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huntly, B. J. P.</span></span> <span> </span><span class="NLM_article-title">Targeting epigenetic readers in cancer</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>367</i></span>,  <span class="NLM_fpage">647</span>– <span class="NLM_lpage">657</span>, <span class="refDoi"> DOI: 10.1056/nejmra1112635</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1056%2Fnejmra1112635" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=22894577" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1Chu77F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=367&publication_year=2012&pages=647-657&author=M.+A.+Dawsonauthor=T.+Kouzaridesauthor=B.+J.+P.+Huntly&title=Targeting+epigenetic+readers+in+cancer&doi=10.1056%2Fnejmra1112635"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting epigenetic readers in cancer</span></div><div class="casAuthors">Dawson, Mark A.; Kouzarides, Tony; Huntly, Brian J. P.</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">367</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">647-657</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">A review on assocn. between epigenetics and chromatin biol., BET bromodomain inhibitors, and role of BET inhibitors in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnz_YQy6Z2T7Vg90H21EOLACvtfcHk0lis9vE_miRS-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1Chu77F&md5=f5e1e82c8a1105f798b1f7719138721c</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1056%2Fnejmra1112635&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252Fnejmra1112635%26sid%3Dliteratum%253Aachs%26aulast%3DDawson%26aufirst%3DM.%2BA.%26aulast%3DKouzarides%26aufirst%3DT.%26aulast%3DHuntly%26aufirst%3DB.%2BJ.%2BP.%26atitle%3DTargeting%2520epigenetic%2520readers%2520in%2520cancer%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2012%26volume%3D367%26spage%3D647%26epage%3D657%26doi%3D10.1056%2Fnejmra1112635" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bountra, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fish, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span> <span> </span><span class="NLM_article-title">Epigenetic protein families: a new frontier for drug discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">384</span>– <span class="NLM_lpage">400</span>, <span class="refDoi"> DOI: 10.1038/nrd3674</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1038%2Fnrd3674" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=22498752" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC38XlsFWjsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=384-400&author=C.+H.+Arrowsmithauthor=C.+Bountraauthor=P.+V.+Fishauthor=K.+Leeauthor=M.+Schapira&title=Epigenetic+protein+families%3A+a+new+frontier+for+drug+discovery&doi=10.1038%2Fnrd3674"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic protein families: a new frontier for drug discovery</span></div><div class="casAuthors">Arrowsmith, Cheryl H.; Bountra, Chas; Fish, Paul V.; Lee, Kevin; Schapira, Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">384-400</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Epigenetic regulation of gene expression is a dynamic and reversible process that establishes normal cellular phenotypes but also contributes to human diseases.  At the mol. level, epigenetic regulation involves hierarchical covalent modification of DNA and the proteins that package DNA, such as histones.  Here, we review the key protein families that mediate epigenetic signalling through the acetylation and methylation of histones, including histone deacetylases, protein methyltransferases, lysine demethylases, bromodomain-contg. proteins and proteins that bind to methylated histones.  These protein families are emerging as druggable classes of enzymes and druggable classes of protein-protein interaction domains.  In this article, we discuss the known links with disease, basic mol. mechanisms of action and recent progress in the pharmacol. modulation of each class of proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcD99eCigNtLVg90H21EOLACvtfcHk0liv15PWNR1eDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlsFWjsbs%253D&md5=eb82034466a43107aa74a18ccf6d29f5</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnrd3674&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3674%26sid%3Dliteratum%253Aachs%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DFish%26aufirst%3DP.%2BV.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DEpigenetic%2520protein%2520families%253A%2520a%2520new%2520frontier%2520for%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2012%26volume%3D11%26spage%3D384%26epage%3D400%26doi%3D10.1038%2Fnrd3674" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Helin, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhanak, D.</span></span> <span> </span><span class="NLM_article-title">Chromatin proteins and modifications as drug targets</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>502</i></span>,  <span class="NLM_fpage">480</span>– <span class="NLM_lpage">488</span>, <span class="refDoi"> DOI: 10.1038/nature12751</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1038%2Fnature12751" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=24153301" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1Krt77E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=502&publication_year=2013&pages=480-488&author=K.+Helinauthor=D.+Dhanak&title=Chromatin+proteins+and+modifications+as+drug+targets&doi=10.1038%2Fnature12751"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatin proteins and modifications as drug targets</span></div><div class="casAuthors">Helin, Kristian; Dhanak, Dashyant</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">502</span>
        (<span class="NLM_cas:issue">7472</span>),
    <span class="NLM_cas:pages">480-488</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  A plethora of groundbreaking studies have demonstrated the importance of chromatin-assocd. proteins and post-translational modifications of histones, proteins and DNA (so-called epigenetic modifications) for transcriptional control and normal development.  Disruption of epigenetic control is a frequent event in disease, and the first epigenetic-based therapies for cancer treatment have been approved.  A generation of new classes of potent and specific inhibitors for several chromatin-assocd. proteins have shown promise in preclin. trials.  Although the biol. of epigenetic regulation is complex, new inhibitors such as these will hopefully be of clin. use in the coming years.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpa6LKkNrgP9bVg90H21EOLACvtfcHk0liv15PWNR1eDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1Krt77E&md5=ca3f12c6aaba31bb571095bacc02550c</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnature12751&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12751%26sid%3Dliteratum%253Aachs%26aulast%3DHelin%26aufirst%3DK.%26aulast%3DDhanak%26aufirst%3DD.%26atitle%3DChromatin%2520proteins%2520and%2520modifications%2520as%2520drug%2520targets%26jtitle%3DNature%26date%3D2013%26volume%3D502%26spage%3D480%26epage%3D488%26doi%3D10.1038%2Fnature12751" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Højfeldt, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agger, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helin, K.</span></span> <span> </span><span class="NLM_article-title">Histone lysine demethylases as targets for anticancer therapy</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">917</span>– <span class="NLM_lpage">930</span>, <span class="refDoi"> DOI: 10.1038/nrd4154</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1038%2Fnrd4154" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=24232376" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslygtrrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=917-930&author=J.+W.+H%C3%B8jfeldtauthor=K.+Aggerauthor=K.+Helin&title=Histone+lysine+demethylases+as+targets+for+anticancer+therapy&doi=10.1038%2Fnrd4154"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Histone lysine demethylases as targets for anticancer therapy</span></div><div class="casAuthors">Hojfeldt, Jonas W.; Agger, Karl; Helin, Kristian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">917-930</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  It has recently been demonstrated that the genes controlling the epigenetic programs that are required for maintaining chromatin structure and cell identity include genes that drive human cancer.  This observation has led to an increased awareness of chromatin-assocd. proteins as potentially interesting drug targets.  The successful introduction of DNA methylation and histone deacetylase (HDAC) inhibitors for the treatment of specific subtypes of cancer has paved the way for the use of epigenetic therapy.  Here, we highlight key biol. findings demonstrating the roles of members of the histone lysine demethylase class of enzymes in the development of cancers, discuss the potential and challenges of therapeutically targeting them, and highlight emerging small-mol. inhibitors of these enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoThyggRtM7qrVg90H21EOLACvtfcHk0liv15PWNR1eDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslygtrrP&md5=b120d43a0353078b016c8dc8f7cac644</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fnrd4154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4154%26sid%3Dliteratum%253Aachs%26aulast%3DH%25C3%25B8jfeldt%26aufirst%3DJ.%2BW.%26aulast%3DAgger%26aufirst%3DK.%26aulast%3DHelin%26aufirst%3DK.%26atitle%3DHistone%2520lysine%2520demethylases%2520as%2520targets%2520for%2520anticancer%2520therapy%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2013%26volume%3D12%26spage%3D917%26epage%3D930%26doi%3D10.1038%2Fnrd4154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Audia, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Austin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blagg, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bountra, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buser-Doepner, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahlin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhanak, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frederiksen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimshaw, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hepworth, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kotz, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruger, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mader, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsden, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller-Fahrnow, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Hagan, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overington, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenberg, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundström, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Superti-Furga, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taunton, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toledo-Sherman, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walpole, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walters, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willson, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Workman, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuercher, W. J.</span></span> <span> </span><span class="NLM_article-title">The promise and peril of chemical probes</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">536</span>– <span class="NLM_lpage">541</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1867</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1038%2Fnchembio.1867" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=26196764" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Wlu7zO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=536-541&author=C.+H.+Arrowsmithauthor=J.+E.+Audiaauthor=C.+Austinauthor=J.+Baellauthor=J.+Bennettauthor=J.+Blaggauthor=C.+Bountraauthor=P.+E.+Brennanauthor=P.+J.+Brownauthor=M.+E.+Bunnageauthor=C.+Buser-Doepnerauthor=R.+M.+Campbellauthor=A.+J.+Carterauthor=P.+Cohenauthor=R.+A.+Copelandauthor=B.+Cravattauthor=J.+L.+Dahlinauthor=D.+Dhanakauthor=A.+M.+Edwardsauthor=M.+Frederiksenauthor=S.+V.+Fryeauthor=N.+Grayauthor=C.+E.+Grimshawauthor=D.+Hepworthauthor=T.+Howeauthor=K.+V.+M.+Huberauthor=J.+Jinauthor=S.+Knappauthor=J.+D.+Kotzauthor=R.+G.+Krugerauthor=D.+Loweauthor=M.+M.+Maderauthor=B.+Marsdenauthor=A.+Mueller-Fahrnowauthor=S.+M%C3%BCllerauthor=R.+C.+O%E2%80%99Haganauthor=J.+P.+Overingtonauthor=D.+R.+Owenauthor=S.+H.+Rosenbergauthor=R.+Rossauthor=B.+Rothauthor=M.+Schapiraauthor=S.+L.+Schreiberauthor=B.+Shoichetauthor=M.+Sundstr%C3%B6mauthor=G.+Superti-Furgaauthor=J.+Tauntonauthor=L.+Toledo-Shermanauthor=C.+Walpoleauthor=M.+A.+Waltersauthor=T.+M.+Willsonauthor=P.+Workmanauthor=R.+N.+Youngauthor=W.+J.+Zuercher&title=The+promise+and+peril+of+chemical+probes&doi=10.1038%2Fnchembio.1867"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">The promise and peril of chemical probes</span></div><div class="casAuthors">Arrowsmith, Cheryl H.; Audia, James E.; Austin, Christopher; Baell, Jonathan; Bennett, Jonathan; Blagg, Julian; Bountra, Chas; Brennan, Paul E.; Brown, Peter J.; Bunnage, Mark E.; Buser-Doepner, Carolyn; Campbell, Robert M.; Carter, Adrian J.; Cohen, Philip; Copeland, Robert A.; Cravatt, Ben; Dahlin, Jayme L.; Dhanak, Dashyant; Edwards, Aled M.; Frye, Stephen V.; Gray, Nathanael; Grimshaw, Charles E.; Hepworth, David; Howe, Trevor; Huber, Kilian V. M.; Jin, Jian; Knapp, Stefan; Kotz, Joanne D.; Kruger, Ryan G.; Lowe, Derek; Mader, Mary M.; Marsden, Brian; Mueller-Fahrnow, Anke; Muller, Susanne; O'Hagan, Ronan C.; Overington, John P.; Owen, Dafydd R.; Rosenberg, Saul H.; Roth, Brian; Ross, Ruth; Schapira, Matthieu; Schreiber, Stuart L.; Shoichet, Brian; Sundstrom, Michael; Superti-Furga, Giulio; Taunton, Jack; Toledo-Sherman, Leticia; Walpole, Chris; Walters, Michael A.; Willson, Timothy M.; Workman, Paul; Young, Robert N.; Zuercher, William J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">536-541</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Chem. probes are powerful reagents with increasing impacts on biomedical research.  However, probes of poor quality or that are used incorrectly generate misleading results.  To help address these shortcomings, we will create a community-driven wiki resource to improve quality and convey current best practice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-li4dxq487rVg90H21EOLACvtfcHk0lg4Ia8pyoqeLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Wlu7zO&md5=d8c7a7af94921ab4cdcc871487cceadd</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1867&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1867%26sid%3Dliteratum%253Aachs%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26aulast%3DAustin%26aufirst%3DC.%26aulast%3DBaell%26aufirst%3DJ.%26aulast%3DBennett%26aufirst%3DJ.%26aulast%3DBlagg%26aufirst%3DJ.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DBuser-Doepner%26aufirst%3DC.%26aulast%3DCampbell%26aufirst%3DR.%2BM.%26aulast%3DCarter%26aufirst%3DA.%2BJ.%26aulast%3DCohen%26aufirst%3DP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DCravatt%26aufirst%3DB.%26aulast%3DDahlin%26aufirst%3DJ.%2BL.%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DEdwards%26aufirst%3DA.%2BM.%26aulast%3DFrederiksen%26aufirst%3DM.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DGray%26aufirst%3DN.%26aulast%3DGrimshaw%26aufirst%3DC.%2BE.%26aulast%3DHepworth%26aufirst%3DD.%26aulast%3DHowe%26aufirst%3DT.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DKotz%26aufirst%3DJ.%2BD.%26aulast%3DKruger%26aufirst%3DR.%2BG.%26aulast%3DLowe%26aufirst%3DD.%26aulast%3DMader%26aufirst%3DM.%2BM.%26aulast%3DMarsden%26aufirst%3DB.%26aulast%3DMueller-Fahrnow%26aufirst%3DA.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Hagan%26aufirst%3DR.%2BC.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DRosenberg%26aufirst%3DS.%2BH.%26aulast%3DRoss%26aufirst%3DR.%26aulast%3DRoth%26aufirst%3DB.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DShoichet%26aufirst%3DB.%26aulast%3DSundstr%25C3%25B6m%26aufirst%3DM.%26aulast%3DSuperti-Furga%26aufirst%3DG.%26aulast%3DTaunton%26aufirst%3DJ.%26aulast%3DToledo-Sherman%26aufirst%3DL.%26aulast%3DWalpole%26aufirst%3DC.%26aulast%3DWalters%26aufirst%3DM.%2BA.%26aulast%3DWillson%26aufirst%3DT.%2BM.%26aulast%3DWorkman%26aufirst%3DP.%26aulast%3DYoung%26aufirst%3DR.%2BN.%26aulast%3DZuercher%26aufirst%3DW.%2BJ.%26atitle%3DThe%2520promise%2520and%2520peril%2520of%2520chemical%2520probes%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D536%26epage%3D541%26doi%3D10.1038%2Fnchembio.1867" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.-M.</span></span> <span> </span><span class="NLM_article-title">The bromodomain: a new target in emerging epigenetic medicine</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">598</span>– <span class="NLM_lpage">608</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00831</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00831" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVyit7fJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=598-608&author=S.+G.+Smithauthor=M.-M.+Zhou&title=The+bromodomain%3A+a+new+target+in+emerging+epigenetic+medicine&doi=10.1021%2Facschembio.5b00831"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">The Bromodomain: A New Target in Emerging Epigenetic Medicine</span></div><div class="casAuthors">Smith, Steven G.; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">598-608</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromodomain (BrD) is a conserved protein modular domain found in many chromatin- and transcription-assocd. proteins that has the ability to recognize acetylated lysine residues.  This activity allows bromodomains to play a vital role in many acetylation-mediated protein-protein interactions in the cell, ranging from substrate recruitment for histone acetyltransferases (HATs) to aiding in multiple-protein complex assembly for gene transcriptional activation or suppression in chromatin.  In recent years, considerable efforts have been made to develop chem. inhibitors of these bromodomains in an effort to probe their cellular functions.  Potent and selective inhibitors have been extensively developed for one group of the bromodomain family termed BET proteins that consist of tandem bromodomains followed by an extra terminal domain.  Drug developers are actively designing inhibitors of other bromodomains and working to move the most successful inhibitors into the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTQF2Qjn_iuLVg90H21EOLACvtfcHk0lg4Ia8pyoqeLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVyit7fJ&md5=dd34fc591550ff1528e6dccd1d04f1c1</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00831&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00831%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DS.%2BG.%26aulast%3DZhou%26aufirst%3DM.-M.%26atitle%3DThe%2520bromodomain%253A%2520a%2520new%2520target%2520in%2520emerging%2520epigenetic%2520medicine%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D598%26epage%3D608%26doi%3D10.1021%2Facschembio.5b00831" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">James, L. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S. V.</span></span> <span> </span><span class="NLM_article-title">Chemical probes for methyl lysine reader domains</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">135</span>– <span class="NLM_lpage">141</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2016.06.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1016%2Fj.cbpa.2016.06.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=27348158" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsF2qt7jE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2016&pages=135-141&author=L.+I.+Jamesauthor=S.+V.+Frye&title=Chemical+probes+for+methyl+lysine+reader+domains&doi=10.1016%2Fj.cbpa.2016.06.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical probes for methyl lysine reader domains</span></div><div class="casAuthors">James, Lindsey I.; Frye, Stephen V.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">135-141</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The primary intent of a chem. probe is to establish the relationship between a mol. target, usually a protein whose function is modulated by the probe, and the biol. consequences of that modulation.  In order to fulfill this purpose, a chem. probe must be profiled for selectivity, mechanism of action, and cellular activity, as the cell is the minimal system in which 'biol.' can be explored.  This review provides a brief overview of progress towards chem. probes for Me lysine reader domains with a focus on recent progress targeting chromodomains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrv3D9GBmCaebVg90H21EOLACvtfcHk0lg4Ia8pyoqeLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsF2qt7jE&md5=e0f49e320758bcdbebf53ffd1fe05883</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2016.06.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2016.06.004%26sid%3Dliteratum%253Aachs%26aulast%3DJames%26aufirst%3DL.%2BI.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26atitle%3DChemical%2520probes%2520for%2520methyl%2520lysine%2520reader%2520domains%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D33%26spage%3D135%26epage%3D141%26doi%3D10.1016%2Fj.cbpa.2016.06.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Milosevich, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hof, F.</span></span> <span> </span><span class="NLM_article-title">Chemical inhibitors of epigenetic methyllysine reader proteins</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">1570</span>– <span class="NLM_lpage">1583</span>, <span class="refDoi"> DOI: 10.1021/acs.biochem.5b01073</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biochem.5b01073" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFOmtrnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=1570-1583&author=N.+Milosevichauthor=F.+Hof&title=Chemical+inhibitors+of+epigenetic+methyllysine+reader+proteins&doi=10.1021%2Facs.biochem.5b01073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical Inhibitors of Epigenetic Methyllysine Reader Proteins</span></div><div class="casAuthors">Milosevich, Natalia; Hof, Fraser</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1570-1583</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein methylation is a common post-translational modification with diverse biol. functions.  Methyllysine reader proteins are increasingly a focus of epigenetics research and play important roles in regulating many cellular processes.  These reader proteins are vital players in development, cell cycle regulation, stress responses, oncogenesis, and other disease pathways.  The recent emergence of a small no. of chem. inhibitors for methyllysine reader proteins supports the viability of these proteins as targets for drug development.  This article introduces the biochem. and biol. of methyllysine reader proteins, provides an overview of functions for those families of readers that have been targeted to date (MBT, PHD, tudor, and chromodomains), and reviews the development of synthetic agents that directly block their methyllysine reading functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGps23HnAOk3T7Vg90H21EOLACvtfcHk0lg4Ia8pyoqeLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFOmtrnM&md5=0d9ec539ad7b345bec4ecb8a252a9ff8</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Facs.biochem.5b01073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biochem.5b01073%26sid%3Dliteratum%253Aachs%26aulast%3DMilosevich%26aufirst%3DN.%26aulast%3DHof%26aufirst%3DF.%26atitle%3DChemical%2520inhibitors%2520of%2520epigenetic%2520methyllysine%2520reader%2520proteins%26jtitle%3DBiochemistry%26date%3D2016%26volume%3D55%26spage%3D1570%26epage%3D1583%26doi%3D10.1021%2Facs.biochem.5b01073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Greschik, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schüle, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Günther, T.</span></span> <span> </span><span class="NLM_article-title">Selective targeting of epigenetic reader domains</span>. <i>Expert Opin. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">449</span>– <span class="NLM_lpage">463</span>, <span class="refDoi"> DOI: 10.1080/17460441.2017.1303474</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1080%2F17460441.2017.1303474" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=28277835" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC2sXktlKmt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=449-463&author=H.+Greschikauthor=R.+Sch%C3%BCleauthor=T.+G%C3%BCnther&title=Selective+targeting+of+epigenetic+reader+domains&doi=10.1080%2F17460441.2017.1303474"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Selective targeting of epigenetic reader domains</span></div><div class="casAuthors">Greschik, Holger; Schuele, Roland; Guenther, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">449-463</span>CODEN:
                <span class="NLM_cas:coden">EODDBX</span>;
        ISSN:<span class="NLM_cas:issn">1746-0441</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  Epigenetic regulators including writers, erasers, and readers of chromatin marks have been implicated in numerous diseases and are therefore subject of intense academic and pharmaceutical research.  While several small-mol. inhibitors targeting writers or erasers are either approved drugs or are currently being evaluated in clin. trials, the targeting of epigenetic readers has lagged behind.  Proof-of-principle that epigenetic readers are also relevant drug targets was provided by landmark discoveries of selective inhibitors targeting the BET family of acetyl-lysine readers.  More recently, high affinity chem. probes for non-BET acetyl- and methyl-lysine reader domains have also been developed.  This article covers recent advances with the identification and validation of inhibitors and chem. probes targeting epigenetic reader domains.  Issues related to epigenetic reader druggability, quality requirements for chem. probes, interpretation of cellular action, unexpected cross-talk, and future challenges are also discussed.  Chem. probes provide a powerful means to unravel biol. functions of epigenetic readers and evaluate their potential as drug targets.  To yield meaningful results, potency, selectivity, and cellular target engagement of chem. probes need to be stringently validated.  Future chem. probes will probably need to fulfil addnl. criteria such as strict target specificity or the targeting of readers within protein complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpN_FtZE2UW-rVg90H21EOLACvtfcHk0lis0ILjW93OYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXktlKmt70%253D&md5=a060b82da46d848a3713131dd3b438b9</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1080%2F17460441.2017.1303474&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F17460441.2017.1303474%26sid%3Dliteratum%253Aachs%26aulast%3DGreschik%26aufirst%3DH.%26aulast%3DSch%25C3%25BCle%26aufirst%3DR.%26aulast%3DG%25C3%25BCnther%26aufirst%3DT.%26atitle%3DSelective%2520targeting%2520of%2520epigenetic%2520reader%2520domains%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2017%26volume%3D12%26spage%3D449%26epage%3D463%26doi%3D10.1080%2F17460441.2017.1303474" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zaware, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.-M.</span></span> <span> </span><span class="NLM_article-title">Chemical modulators for epigenome reader domains as emerging epigenetic therapies for cancer and inflammation</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">116</span>– <span class="NLM_lpage">125</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2017.06.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1016%2Fj.cbpa.2017.06.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=28689146" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFChtbfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2017&pages=116-125&author=N.+Zawareauthor=M.-M.+Zhou&title=Chemical+modulators+for+epigenome+reader+domains+as+emerging+epigenetic+therapies+for+cancer+and+inflammation&doi=10.1016%2Fj.cbpa.2017.06.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical modulators for epigenome reader domains as emerging epigenetic therapies for cancer and inflammation</span></div><div class="casAuthors">Zaware, Nilesh; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">116-125</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Site-specific lysine acetylation and methylation on histones are crit. post-translational modifications (PTMs) that govern ordered gene transcription in chromatin.  Mis-regulation of these histone PTM-mediated processes has been shown to be assocd. with human diseases.  Since the 2010 landmark reports of small mols. (+)-JQ1 and I-BET762 that target the acetyl-lysine 'reader' Bromodomain and Extra Terminal domain (BET) proteins, there have been relentless efforts to develop epigenetic therapy with small mols. to modulate mol. interactions of epigenome reader domain proteins with PTMs.  In addn. to BET, the other emerging targets include non-BET acetyl-lysine and methyl-lysine reader domains.  This review covers the key chem. modulators of the aforementioned epigenome reader proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGps_AJz3UXkTrVg90H21EOLACvtfcHk0lis0ILjW93OYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFChtbfF&md5=b33b5288f202e4b2e4ccf1f7b92bf654</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2017.06.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2017.06.012%26sid%3Dliteratum%253Aachs%26aulast%3DZaware%26aufirst%3DN.%26aulast%3DZhou%26aufirst%3DM.-M.%26atitle%3DChemical%2520modulators%2520for%2520epigenome%2520reader%2520domains%2520as%2520emerging%2520epigenetic%2520therapies%2520for%2520cancer%2520and%2520inflammation%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D39%26spage%3D116%26epage%3D125%26doi%3D10.1016%2Fj.cbpa.2017.06.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">James, L. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krichevsky, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korboukh, V. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herold, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacNevin, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norris, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sagum, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tempel, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcon, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emili, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenblatt, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kireev, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janzen, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S. V.</span></span> <span> </span><span class="NLM_article-title">Discovery of a chemical probe for the L3MBTL3 methyllysine reader domain</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">184</span>– <span class="NLM_lpage">191</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1157</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1038%2Fnchembio.1157" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=23292653" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjslGjsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=184-191&author=L.+I.+Jamesauthor=D.+Barsyte-Lovejoyauthor=N.+Zhongauthor=L.+Krichevskyauthor=V.+K.+Korboukhauthor=J.+M.+Heroldauthor=C.+J.+MacNevinauthor=J.+L.+Norrisauthor=C.+A.+Sagumauthor=W.+Tempelauthor=E.+Marconauthor=H.+Guoauthor=C.+Gaoauthor=X.-P.+Huangauthor=S.+Duanauthor=A.+Emiliauthor=J.+F.+Greenblattauthor=D.+B.+Kireevauthor=J.+Jinauthor=W.+P.+Janzenauthor=P.+J.+Brownauthor=M.+T.+Bedfordauthor=C.+H.+Arrowsmithauthor=S.+V.+Frye&title=Discovery+of+a+chemical+probe+for+the+L3MBTL3+methyllysine+reader+domain&doi=10.1038%2Fnchembio.1157"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a chemical probe for the L3MBTL3 methyllysine reader domain</span></div><div class="casAuthors">James, Lindsey I.; Barsyte-Lovejoy, Dalia; Zhong, Nan; Krichevsky, Liubov; Korboukh, Victoria K.; Herold, J. Martin; MacNevin, Christopher J.; Norris, Jacqueline L.; Sagum, Cari A.; Tempel, Wolfram; Marcon, Edyta; Guo, Hongbo; Gao, Cen; Huang, Xi-Ping; Duan, Shili; Emili, Andrew; Greenblatt, Jack F.; Kireev, Dmitri B.; Jin, Jian; Janzen, William P.; Brown, Peter J.; Bedford, Mark T.; Arrowsmith, Cheryl H.; Frye, Stephen V.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">184-191</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We describe the discovery of UNC1215, a potent and selective chem. probe for the methyllysine (Kme) reading function of L3MBTL3, a member of the malignant brain tumor (MBT) family of chromatin-interacting transcriptional repressors.  UNC1215 binds L3MBTL3 with a Kd of 120 nM, competitively displacing mono- or dimethyllysine-contg. peptides, and is greater than 50-fold more potent toward L3MBTL3 than other members of the MBT family while also demonstrating selectivity against more than 200 other reader domains examd.  X-ray crystallog. identified a unique 2:2 polyvalent mode of interaction between UNC1215 and L3MBTL3.  In cells, UNC1215 is nontoxic and directly binds L3MBTL3 via the Kme-binding pocket of the MBT domains.  UNC1215 increases the cellular mobility of GFP-L3MBTL3 fusion proteins, and point mutants that disrupt the Kme-binding function of GFP-L3MBTL3 phenocopy the effects of UNC1215 on localization.  Finally, UNC1215 was used to reveal a new Kme-dependent interaction of L3MBTL3 with BCLAF1 (BCL2-assocd. transcription factor 1), a protein implicated in DNA damage repair and apoptosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqta8WQQkm04rVg90H21EOLACvtfcHk0lis0ILjW93OYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjslGjsw%253D%253D&md5=ea8779ddceed07b5e4c822f2eb888acb</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1157&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1157%26sid%3Dliteratum%253Aachs%26aulast%3DJames%26aufirst%3DL.%2BI.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DZhong%26aufirst%3DN.%26aulast%3DKrichevsky%26aufirst%3DL.%26aulast%3DKorboukh%26aufirst%3DV.%2BK.%26aulast%3DHerold%26aufirst%3DJ.%2BM.%26aulast%3DMacNevin%26aufirst%3DC.%2BJ.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DSagum%26aufirst%3DC.%2BA.%26aulast%3DTempel%26aufirst%3DW.%26aulast%3DMarcon%26aufirst%3DE.%26aulast%3DGuo%26aufirst%3DH.%26aulast%3DGao%26aufirst%3DC.%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DDuan%26aufirst%3DS.%26aulast%3DEmili%26aufirst%3DA.%26aulast%3DGreenblatt%26aufirst%3DJ.%2BF.%26aulast%3DKireev%26aufirst%3DD.%2BB.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26atitle%3DDiscovery%2520of%2520a%2520chemical%2520probe%2520for%2520the%2520L3MBTL3%2520methyllysine%2520reader%2520domain%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2013%26volume%3D9%26spage%3D184%26epage%3D191%26doi%3D10.1038%2Fnchembio.1157" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dickson, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norris, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cholensky, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tempel, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sagum, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Black, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baughman, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senisterra, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pattenden, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, L. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S. V.</span></span> <span> </span><span class="NLM_article-title">A cellular chemical probe targeting the chromodomains of polycomb repressive complex 1</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">180</span>– <span class="NLM_lpage">187</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1038%2Fnchembio.2007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=26807715" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFyis7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=180-187&author=J.+I.+Stuckeyauthor=B.+M.+Dicksonauthor=N.+Chengauthor=Y.+Liuauthor=J.+L.+Norrisauthor=S.+H.+Cholenskyauthor=W.+Tempelauthor=S.+Qinauthor=K.+G.+Huberauthor=C.+Sagumauthor=K.+Blackauthor=F.+Liauthor=X.-P.+Huangauthor=B.+L.+Rothauthor=B.+M.+Baughmanauthor=G.+Senisterraauthor=S.+G.+Pattendenauthor=M.+Vedadiauthor=P.+J.+Brownauthor=M.+T.+Bedfordauthor=J.+Minauthor=C.+H.+Arrowsmithauthor=L.+I.+Jamesauthor=S.+V.+Frye&title=A+cellular+chemical+probe+targeting+the+chromodomains+of+polycomb+repressive+complex+1&doi=10.1038%2Fnchembio.2007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">A cellular chemical probe targeting the chromodomains of Polycomb repressive complex 1</span></div><div class="casAuthors">Stuckey, Jacob I.; Dickson, Bradley M.; Cheng, Nancy; Liu, Yanli; Norris, Jacqueline L.; Cholensky, Stephanie H.; Tempel, Wolfram; Qin, Su; Huber, Katherine G.; Sagum, Cari; Black, Karynne; Li, Fengling; Huang, Xi-Ping; Roth, Bryan L.; Baughman, Brandi M.; Senisterra, Guillermo; Pattenden, Samantha G.; Vedadi, Masoud; Brown, Peter J.; Bedford, Mark T.; Min, Jinrong; Arrowsmith, Cheryl H.; James, Lindsey I.; Frye, Stephen V.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">180-187</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We report the design and characterization of UNC3866, a potent antagonist of the methyllysine (Kme) reading function of the Polycomb CBX and CDY families of chromodomains.  Polycomb CBX proteins regulate gene expression by targeting Polycomb repressive complex 1 (PRC1) to sites of H3K27me3 via their chromodomains.  UNC3866 binds the chromodomains of CBX4 and CBX7 most potently, with a Kd of ∼100 nM for each, and is 6- to 18-fold selective as compared to seven other CBX and CDY chromodomains while being highly selective over >250 other protein targets.  X-ray crystallog. revealed that UNC3866's interactions with the CBX chromodomains closely mimic those of the methylated H3 tail.  UNC4195, a biotinylated deriv. of UNC3866, was used to demonstrate that UNC3866 engages intact PRC1 and that EED incorporation into PRC1 is isoform dependent in PC3 prostate cancer cells.  Finally, UNC3866 inhibits PC3 cell proliferation, consistent with the known ability of CBX7 overexpression to confer a growth advantage, whereas UNC4219, a methylated neg. control compd., has negligible effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobRZ8YoHZWk7Vg90H21EOLACvtfcHk0lifbAFVtA_0DQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFyis7o%253D&md5=97fe8dd8e35094a8c91d5d4bdc265b4f</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2007%26sid%3Dliteratum%253Aachs%26aulast%3DStuckey%26aufirst%3DJ.%2BI.%26aulast%3DDickson%26aufirst%3DB.%2BM.%26aulast%3DCheng%26aufirst%3DN.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DCholensky%26aufirst%3DS.%2BH.%26aulast%3DTempel%26aufirst%3DW.%26aulast%3DQin%26aufirst%3DS.%26aulast%3DHuber%26aufirst%3DK.%2BG.%26aulast%3DSagum%26aufirst%3DC.%26aulast%3DBlack%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DBaughman%26aufirst%3DB.%2BM.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DPattenden%26aufirst%3DS.%2BG.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DMin%26aufirst%3DJ.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DJames%26aufirst%3DL.%2BI.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26atitle%3DA%2520cellular%2520chemical%2520probe%2520targeting%2520the%2520chromodomains%2520of%2520polycomb%2520repressive%2520complex%25201%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3D180%26epage%3D187%26doi%3D10.1038%2Fnchembio.2007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greschik, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgahn, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidtkunz, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schott, A.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMillan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baranauskienė, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oppermann, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matulis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schüle, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, M.</span></span> <span> </span><span class="NLM_article-title">Identification of a small-molecule ligand of the epigenetic reader protein Spindlin1 via a versatile screening platform</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>44</i></span>, <span class="NLM_elocation-id">e88</span> <span class="refDoi"> DOI: 10.1093/nar/gkw089</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1093%2Fnar%2Fgkw089" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=26893353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslCqurvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&author=T.+Wagnerauthor=H.+Greschikauthor=T.+Burgahnauthor=K.+Schmidtkunzauthor=A.-K.+Schottauthor=J.+McMillanauthor=L.+Baranauskien%C4%97author=Y.+Xiongauthor=O.+Fedorovauthor=J.+Jinauthor=U.+Oppermannauthor=D.+Matulisauthor=R.+Sch%C3%BCleauthor=M.+Jung&title=Identification+of+a+small-molecule+ligand+of+the+epigenetic+reader+protein+Spindlin1+via+a+versatile+screening+platform&doi=10.1093%2Fnar%2Fgkw089"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a small-molecule ligand of the epigenetic reader protein Spindlin1 via a versatile screening platform</span></div><div class="casAuthors">Wagner, Tobias; Greschik, Holger; Burgahn, Teresa; Schmidtkunz, Karin; Schott, Anne-Kathrin; McMillan, Joel; Baranauskiene, Lina; Xiong, Yan; Fedorov, Oleg; Jin, Jian; Oppermann, Udo; Matulis, Daumantas; Schuele, Roland; Jung, Manfred</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">e88/1-e88/22</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Epigenetic modifications of histone tails play an essential role in the regulation of eukaryotic transcription.  Writer and eraser enzymes establish and maintain the epigenetic code by creating or removing posttranslational marks.  Specific binding proteins, called readers, recognize the modifications and mediate epigenetic signalling.  Here, we present a versatile assay platform for the investigation of the interaction between Me lysine readers and their ligands.  This can be utilized for the screening of small mol. inhibitors of such protein-protein interactions and the detailed characterization of the inhibition.  Our platform is constructed in a modular way consisting of orthogonal in vitro binding assays for ligand screening and verification of initial hits and biophys., label-free techniques for further kinetic characterization of confirmed ligands.  A stability assay for the investigation of target engagement in a cellular context complements the platform.  We applied the complete evaluation chain to the Tudor domain contg. protein Spindlin1 and established the in vitro test systems for the double Tudor domain of the histone demethylase JMJD2C.  We finally conducted an exploratory screen for inhibitors of the interaction between Spindlin1 and H3K4me3 and identified A366 as the first nanomolar small-mol. ligand of a Tudor domain contg. Me lysine reader.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGql-bO41bi49bVg90H21EOLACvtfcHk0lifbAFVtA_0DQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslCqurvK&md5=3c6536be1ee6f83c5d2849baa02da318</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkw089&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkw089%26sid%3Dliteratum%253Aachs%26aulast%3DWagner%26aufirst%3DT.%26aulast%3DGreschik%26aufirst%3DH.%26aulast%3DBurgahn%26aufirst%3DT.%26aulast%3DSchmidtkunz%26aufirst%3DK.%26aulast%3DSchott%26aufirst%3DA.-K.%26aulast%3DMcMillan%26aufirst%3DJ.%26aulast%3DBaranauskien%25C4%2597%26aufirst%3DL.%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DOppermann%26aufirst%3DU.%26aulast%3DMatulis%26aufirst%3DD.%26aulast%3DSch%25C3%25BCle%26aufirst%3DR.%26aulast%3DJung%26aufirst%3DM.%26atitle%3DIdentification%2520of%2520a%2520small-molecule%2520ligand%2520of%2520the%2520epigenetic%2520reader%2520protein%2520Spindlin1%2520via%2520a%2520versatile%2520screening%2520platform%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2016%26volume%3D44%26doi%3D10.1093%2Fnar%2Fgkw089" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bae, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viviano, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lv, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sagum, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castellano, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khalil, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sbardella, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span> <span> </span><span class="NLM_article-title">Developing Spindlin1 small-molecule inhibitors by using protein microarrays</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">750</span>– <span class="NLM_lpage">756</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2377</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1038%2Fnchembio.2377" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=28504676" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnslKktrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=750-756&author=N.+Baeauthor=M.+Vivianoauthor=X.+Suauthor=J.+Lvauthor=D.+Chengauthor=C.+Sagumauthor=S.+Castellanoauthor=X.+Baiauthor=C.+Johnsonauthor=M.+I.+Khalilauthor=J.+Shenauthor=K.+Chenauthor=H.+Liauthor=G.+Sbardellaauthor=M.+T.+Bedford&title=Developing+Spindlin1+small-molecule+inhibitors+by+using+protein+microarrays&doi=10.1038%2Fnchembio.2377"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Developing Spindlin1 small-molecule inhibitors by using protein microarrays</span></div><div class="casAuthors">Bae, Narkhyun; Viviano, Monica; Su, Xiaonan; Lv, Jie; Cheng, Donghang; Sagum, Cari; Castellano, Sabrina; Bai, Xue; Johnson, Claire; Khalil, Mahmoud Ibrahim; Shen, Jianjun; Chen, Kaifu; Li, Haitao; Sbardella, Gianluca; Bedford, Mark T.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">750-756</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The discovery of inhibitors of methyl- and acetyl-binding domains has provided evidence for the 'druggability' of epigenetic effector mols.  The small-mol. probe, UNC1215, prevents methyl-dependent protein-protein interactions by engaging the arom. cage of MBT domains and, with lower affinity, Tudor domains.  Here, using a library of tagged UNC1215 analogs, we screened a protein-domain microarray of human methyllysine effector mols. to rapidly detect compds. with new binding profiles with either increased or decreased specificity.  Using this approach, we identified a compd. (EML405) that acquired a novel interaction with Tudor-domain-contg. protein Spindlin1 (SPIN1).  Structural studies facilitated the rational synthesis of SPIN1 inhibitors with increased selectivity (EML631-633), which engaged SPIN1 in cells, blocked its ability to 'read' H3K4me3 marks, and inhibited its transcriptional-coactivator activity.  Protein microarrays can thus be used as a platform to 'target-hop' and identify small mols. that bind and compete with domain-motif interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6diBsmZCcm7Vg90H21EOLACvtfcHk0ljNF_naq5MiDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnslKktrc%253D&md5=5ac8aab47d4a67cbe0f7c1f04d4ae410</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2377&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2377%26sid%3Dliteratum%253Aachs%26aulast%3DBae%26aufirst%3DN.%26aulast%3DViviano%26aufirst%3DM.%26aulast%3DSu%26aufirst%3DX.%26aulast%3DLv%26aufirst%3DJ.%26aulast%3DCheng%26aufirst%3DD.%26aulast%3DSagum%26aufirst%3DC.%26aulast%3DCastellano%26aufirst%3DS.%26aulast%3DBai%26aufirst%3DX.%26aulast%3DJohnson%26aufirst%3DC.%26aulast%3DKhalil%26aufirst%3DM.%2BI.%26aulast%3DShen%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DSbardella%26aufirst%3DG.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DDeveloping%2520Spindlin1%2520small-molecule%2520inhibitors%2520by%2520using%2520protein%2520microarrays%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26spage%3D750%26epage%3D756%26doi%3D10.1038%2Fnchembio.2377" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rong, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartlam, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Z.</span></span> <span> </span><span class="NLM_article-title">Structure of human Spindlin1</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>282</i></span>,  <span class="NLM_fpage">647</span>– <span class="NLM_lpage">656</span>, <span class="refDoi"> DOI: 10.1074/jbc.m604029200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1074%2Fjbc.m604029200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=17082182" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BD2sXkslej" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=2007&pages=647-656&author=Q.+Zhaoauthor=L.+Qinauthor=F.+Jiangauthor=B.+Wuauthor=W.+Yueauthor=F.+Xuauthor=Z.+Rongauthor=H.+Yuanauthor=X.+Xieauthor=Y.+Gaoauthor=C.+Baiauthor=M.+Bartlamauthor=X.+Peiauthor=Z.+Rao&title=Structure+of+human+Spindlin1&doi=10.1074%2Fjbc.m604029200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of Human Spindlin1: Tandem Tudor-Like Domains for Cell Cycle Regulation</span></div><div class="casAuthors">Zhao, Qiang; Qin, Lipeng; Jiang, Fuguo; Wu, Beili; Yue, Wen; Xu, Feng; Rong, Zhili; Yuan, Hongfeng; Xie, Xiaoyan; Gao, Yanhong; Bai, Cixian; Bartlam, Mark; Pei, Xuetao; Rao, Zihe</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">282</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">647-656</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Spindlin1, a meiotic spindle-binding protein that is highly expressed in ovarian cancer cells, was first identified as a gene involved in gametogenesis.  It appeared to be a target for cell cycle-dependent phosphorylation and was demonstrated to disturb the cell cycle.  Here we report the crystal structure of human spindlin1 to 2.2Å of resoln., representing the first three-dimensional structure from the spin/ssty (Y-linked spermiogenesis-specific transcript) gene family.  The refined structure, contg. three repeats of five/four anti-parallel β-strands, exhibits a novel arrangement of tandem Tudor-like domains.  Two phosphate ions, chelated by Thr-95 and other residues, appear to stabilize the long loop between domains I and II, which might mediate the cell cycle regulation activity of spindlin1.  Flow cytometry expts. indicate that cells expressing spindlin1 display a different cell cycle distribution in mitosis, whereas those expressing a T95A mutant, which had a great decrease in phosphorous content, have little effect on the cell cycle.  We further identified assocns. of spindlin1 with nucleic acid to provide a biochem. basis for its cell cycle regulation and other functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGfMhFJXqtRLVg90H21EOLACvtfcHk0ljNF_naq5MiDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXkslej&md5=14747516da6b164560f2d1384002d6e5</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1074%2Fjbc.m604029200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.m604029200%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DQ.%26aulast%3DQin%26aufirst%3DL.%26aulast%3DJiang%26aufirst%3DF.%26aulast%3DWu%26aufirst%3DB.%26aulast%3DYue%26aufirst%3DW.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DRong%26aufirst%3DZ.%26aulast%3DYuan%26aufirst%3DH.%26aulast%3DXie%26aufirst%3DX.%26aulast%3DGao%26aufirst%3DY.%26aulast%3DBai%26aufirst%3DC.%26aulast%3DBartlam%26aufirst%3DM.%26aulast%3DPei%26aufirst%3DX.%26aulast%3DRao%26aufirst%3DZ.%26atitle%3DStructure%2520of%2520human%2520Spindlin1%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2007%26volume%3D282%26spage%3D647%26epage%3D656%26doi%3D10.1074%2Fjbc.m604029200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, R.-M.</span></span> <span> </span><span class="NLM_article-title">Distinct mode of methylated lysine-4 of histone H3 recognition by tandem tudor-like domains of Spindlin1</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">17954</span>– <span class="NLM_lpage">17959</span>, <span class="refDoi"> DOI: 10.1073/pnas.1208517109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1073%2Fpnas.1208517109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=23077255" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhsl2ktbvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=17954-17959&author=N.+Yangauthor=W.+Wangauthor=Y.+Wangauthor=M.+Wangauthor=Q.+Zhaoauthor=Z.+Raoauthor=B.+Zhuauthor=R.-M.+Xu&title=Distinct+mode+of+methylated+lysine-4+of+histone+H3+recognition+by+tandem+tudor-like+domains+of+Spindlin1&doi=10.1073%2Fpnas.1208517109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Distinct mode of methylated lysine-4 of histone H3 recognition by tandem tudor-like domains of Spindlin1</span></div><div class="casAuthors">Yang, Na; Wang, Weixiang; Wang, Yan; Wang, Mingzhu; Zhao, Qiang; Rao, Zihe; Zhu, Bing; Xu, Rui-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">17954-17959, S17954/1-S17954/5</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Recognition of methylated histone tail lysine residues by tudor domains plays important roles in epigenetic control of gene expression and DNA damage response.  Previous studies revealed the binding of methyllysine in a cage of arom. residues, but the mol. mechanism by which the sequence specificity for surrounding histone tail residues is achieved remains poorly understood.  In the crystal structure of a trimethylated histone H3 lysine 4 (H3K4) peptide bound to the tudor-like domains of Spindlin1 presented here, an atypical mode of methyllysine recognition by an arom. pocket of Spindlin1 is obsd.  Furthermore, the histone sequence is recognized in a distinct manner involving the amino terminus and a pair of arginine residues of histone H3, and disruption of the binding impaired stimulation of pre-RNA expression by Spindlin1.  Our anal. demonstrates considerable diversities of methyllysine recognition and sequence-specific binding of histone tails by tudor domains, and the revelation furthers the understanding of tudor domain proteins in deciphering epigenetic marks on histone tails.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrVCXmD_E2cLVg90H21EOLACvtfcHk0ljNF_naq5MiDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhsl2ktbvK&md5=e785fc0c54885f0188965aea7ef2cfbb</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1208517109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1208517109%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DN.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DZhao%26aufirst%3DQ.%26aulast%3DRao%26aufirst%3DZ.%26aulast%3DZhu%26aufirst%3DB.%26aulast%3DXu%26aufirst%3DR.-M.%26atitle%3DDistinct%2520mode%2520of%2520methylated%2520lysine-4%2520of%2520histone%2520H3%2520recognition%2520by%2520tandem%2520tudor-like%2520domains%2520of%2520Spindlin1%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2012%26volume%3D109%26spage%3D17954%26epage%3D17959%26doi%3D10.1073%2Fpnas.1208517109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Su, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span> <span> </span><span class="NLM_article-title">Molecular basis underlying histone H3 lysine-arginine methylation pattern readout by Spin/Ssty repeats of Spindlin1</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">622</span>– <span class="NLM_lpage">636</span>, <span class="refDoi"> DOI: 10.1101/gad.233239.113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1101%2Fgad.233239.113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=24589551" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC2cXlsF2ntr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2014&pages=622-636&author=X.+Suauthor=G.+Zhuauthor=X.+Dingauthor=S.+Y.+Leeauthor=Y.+Douauthor=B.+Zhuauthor=W.+Wuauthor=H.+Li&title=Molecular+basis+underlying+histone+H3+lysine-arginine+methylation+pattern+readout+by+Spin%2FSsty+repeats+of+Spindlin1&doi=10.1101%2Fgad.233239.113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular basis underlying histone H3 lysine-arginine methylation pattern readout by Spin/Ssty repeats of Spindlin1</span></div><div class="casAuthors">Su, Xiaonan; Zhu, Guixin; Ding, Xiaozhe; Lee, Shirley Y.; Dou, Yali; Zhu, Bing; Wu, Wei; Li, Haitao</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">622-636</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Histone modification patterns and their combinatorial readout have emerged as a fundamental mechanism for epigenetic regulation.  Here the authors characterized Spindlin1 as a histone effector that senses a cis-tail histone H3 methylation pattern involving trimethyllysine 4 (H3K4me3) and asym. dimethylarginine 8 (H3R8me2a) marks.  Spindlin1 consists of triple tudor-like Spin/Ssty repeats.  Cocrystal structure detn. established concurrent recognition of H3K4me3 and H3R8me2a by Spin/Ssty repeats 2 and 1, resp.  Both H3K4me3 and H3R8me2a are recognized using an "insertion cavity" recognition mode, contributing to a methylation state-specific layer of regulation.  In vivo functional studies suggest that Spindlin1 activates Wnt/β-catenin signaling downstream from protein arginine methyltransferase 2 (PRMT2) and the MLL complex, which together are capable of generating a specific H3 "K4me3-R8me2a" pattern.  Mutagenesis of Spindlin1 reader pockets impairs activation of Wnt target genes.  Taken together, this work connects a histone "lysine-arginine" methylation pattern readout by Spindlin1-to-Wnt signaling at the transcriptional level.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2eotrU6XRPrVg90H21EOLACvtfcHk0lhPmaE1Z2imCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXlsF2ntr8%253D&md5=6872607b2651d15d16b80a6d33555bbd</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1101%2Fgad.233239.113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.233239.113%26sid%3Dliteratum%253Aachs%26aulast%3DSu%26aufirst%3DX.%26aulast%3DZhu%26aufirst%3DG.%26aulast%3DDing%26aufirst%3DX.%26aulast%3DLee%26aufirst%3DS.%2BY.%26aulast%3DDou%26aufirst%3DY.%26aulast%3DZhu%26aufirst%3DB.%26aulast%3DWu%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DH.%26atitle%3DMolecular%2520basis%2520underlying%2520histone%2520H3%2520lysine-arginine%2520methylation%2520pattern%2520readout%2520by%2520Spin%252FSsty%2520repeats%2520of%2520Spindlin1%26jtitle%3DGenes%2520Dev.%26date%3D2014%26volume%3D28%26spage%3D622%26epage%3D636%26doi%3D10.1101%2Fgad.233239.113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Z.</span></span> <span> </span><span class="NLM_article-title">Expression, purification, crystallization and preliminary X-ray analysis of human Spindlin1, an ovarian cancer-related protein</span>. <i>Protein Pept. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">203</span>– <span class="NLM_lpage">205</span>, <span class="refDoi"> DOI: 10.2174/092986606775101661</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.2174%2F092986606775101661" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=16472086" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BD28XitlGntr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2006&pages=203-205&author=F.+Jiangauthor=Q.+Zhaoauthor=L.+Qinauthor=H.+Pangauthor=X.+Peiauthor=Z.+Rao&title=Expression%2C+purification%2C+crystallization+and+preliminary+X-ray+analysis+of+human+Spindlin1%2C+an+ovarian+cancer-related+protein&doi=10.2174%2F092986606775101661"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Expression, purification, crystallization and preliminary X-ray analysis of human spindlin 1, an ovarian cancer-related protein</span></div><div class="casAuthors">Jiang, Fuguo; Zhao, Qiang; Qin, Lipeng; Pang, Hai; Pei, Xuetao; Rao, Zihe</div><div class="citationInfo"><span class="NLM_cas:title">Protein & Peptide Letters</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">203-205</span>CODEN:
                <span class="NLM_cas:coden">PPELEN</span>;
        ISSN:<span class="NLM_cas:issn">0929-8665</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">Human spindlin 1 is a newly screened and identified gene product related to ovarian carcinomas and is highly homologous to mouse spindlin.  It is an abundant maternal transcript expressed in the mouse during the transition from oocyte to embryo.  Here, the recombinant human spindlin1 has been overexpressed in Escherichia coli BL21, purified and crystd. using the hanging-drop vapor-diffusion method.  Crystals diffracting to 2.25 Å resoln. were obtained using ammonium sulfate as precipitant.  The crystals belong to the space group P212121, with unit-cell parameters a = 40.7 Å, b = 84.4 Å, c = 136.4 Å, α=β=γ=90°.  Assuming two mols. per asym. unit, the solvent content is calcd. to be 42.4%.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8o5Sz0E4XxrVg90H21EOLACvtfcHk0lhPmaE1Z2imCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XitlGntr8%253D&md5=e0968bde24c824de9633702979233a39</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.2174%2F092986606775101661&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F092986606775101661%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DF.%26aulast%3DZhao%26aufirst%3DQ.%26aulast%3DQin%26aufirst%3DL.%26aulast%3DPang%26aufirst%3DH.%26aulast%3DPei%26aufirst%3DX.%26aulast%3DRao%26aufirst%3DZ.%26atitle%3DExpression%252C%2520purification%252C%2520crystallization%2520and%2520preliminary%2520X-ray%2520analysis%2520of%2520human%2520Spindlin1%252C%2520an%2520ovarian%2520cancer-related%2520protein%26jtitle%3DProtein%2520Pept.%2520Lett.%26date%3D2006%26volume%3D13%26spage%3D203%26epage%3D205%26doi%3D10.2174%2F092986606775101661" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Franz, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greschik, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willmann, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozretic, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jilg, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wardelmann, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buettner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schule, R.</span></span> <span> </span><span class="NLM_article-title">The histone code reader SPIN1 controls RET signaling in liposarcoma</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">4773</span>– <span class="NLM_lpage">4789</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.3000</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.18632%2Foncotarget.3000" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=25749382" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A280%3ADC%252BC2MnitFCluw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=4773-4789&author=H.+Franzauthor=H.+Greschikauthor=D.+Willmannauthor=L.+Ozreticauthor=C.+A.+Jilgauthor=E.+Wardelmannauthor=M.+Jungauthor=R.+Buettnerauthor=R.+Schule&title=The+histone+code+reader+SPIN1+controls+RET+signaling+in+liposarcoma&doi=10.18632%2Foncotarget.3000"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">The histone code reader SPIN1 controls RET signaling in liposarcoma</span></div><div class="casAuthors">Franz Henriette; Greschik Holger; Willmann Dominica; Jilg Cordula Annette; Schule Roland; Ozretic Luka; Buettner Reinhard; Wardelmann Eva; Jung Manfred; Jung Manfred; Schule Roland; Schule Roland</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">4773-89</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The histone code reader Spindlin1 (SPIN1) has been implicated in tumorigenesis and tumor growth, but the underlying molecular mechanisms remain poorly understood.  Here, we show that reducing SPIN1 levels strongly impairs proliferation and increases apoptosis of liposarcoma cells in vitro and in xenograft mouse models.  Combining signaling pathway, genome-wide chromatin binding, and transcriptome analyses, we found that SPIN1 directly enhances expression of GDNF, an activator of the RET signaling pathway, in cooperation with the transcription factor MAZ.  Accordingly, knockdown of SPIN1 or MAZ results in reduced levels of GDNF and activated RET explaining diminished liposarcoma cell proliferation and survival.  In line with these observations, levels of SPIN1, GDNF, activated RET, and MAZ are increased in human liposarcoma compared to normal adipose tissue or lipoma.  Importantly, a mutation of SPIN1 within the reader domain interfering with chromatin binding reduces liposarcoma cell proliferation and survival.  Together, our data describe a molecular mechanism for SPIN1 function in liposarcoma and suggest that targeting SPIN1 chromatin association with small molecule inhibitors may represent a novel therapeutic strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS5Ma1donXoAsNIRw0GKUoEfW6udTcc2eZVX0l2EH1YHrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MnitFCluw%253D%253D&md5=4b404511399ebfdf8a553af0822e522f</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.3000&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.3000%26sid%3Dliteratum%253Aachs%26aulast%3DFranz%26aufirst%3DH.%26aulast%3DGreschik%26aufirst%3DH.%26aulast%3DWillmann%26aufirst%3DD.%26aulast%3DOzretic%26aufirst%3DL.%26aulast%3DJilg%26aufirst%3DC.%2BA.%26aulast%3DWardelmann%26aufirst%3DE.%26aulast%3DJung%26aufirst%3DM.%26aulast%3DBuettner%26aufirst%3DR.%26aulast%3DSchule%26aufirst%3DR.%26atitle%3DThe%2520histone%2520code%2520reader%2520SPIN1%2520controls%2520RET%2520signaling%2520in%2520liposarcoma%26jtitle%3DOncotarget%26date%3D2015%26volume%3D6%26spage%3D4773%26epage%3D4789%26doi%3D10.18632%2Foncotarget.3000" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xing, A.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, D.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, P.</span></span> <span> </span><span class="NLM_article-title">Suppression of SPIN1-mediated PI3K–Akt pathway by miR-489 increases chemosensitivity in breast cancer</span>. <i>J. Pathol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>239</i></span>,  <span class="NLM_fpage">459</span>– <span class="NLM_lpage">472</span>, <span class="refDoi"> DOI: 10.1002/path.4743</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1002%2Fpath.4743" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=27171498" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1SmtL7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=239&publication_year=2016&pages=459-472&author=X.+Chenauthor=Y.-W.+Wangauthor=A.-Y.+Xingauthor=S.+Xiangauthor=D.-B.+Shiauthor=L.+Liuauthor=Y.-X.+Liauthor=P.+Gao&title=Suppression+of+SPIN1-mediated+PI3K%E2%80%93Akt+pathway+by+miR-489+increases+chemosensitivity+in+breast+cancer&doi=10.1002%2Fpath.4743"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Suppression of SPIN1-mediated PI3K-Akt pathway by miR-489 increases chemosensitivity in breast cancer</span></div><div class="casAuthors">Chen, Xu; Wang, Ya-Wen; Xing, Ai-Yan; Xiang, Shuai; Shi, Duan-Bo; Liu, Lei; Li, Yan-Xiang; Gao, Peng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pathology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">239</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">459-472</span>CODEN:
                <span class="NLM_cas:coden">JPTLAS</span>;
        ISSN:<span class="NLM_cas:issn">1096-9896</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Ltd.</span>)
        </div><div class="casAbstract">Drug resistance is one of the major obstacles for improving the prognosis of breast cancer patients.  Increasing evidence has linked the assocn. of aberrantly expressed microRNAs (miRNAs) with tumor development and progression as well as chemoresistance.  Despite recent advances, there is still little known about the potential role and mechanism of miRNAs in breast cancer chemoresistance.  Here we describe that 16 miRNAs were found to be significantly down-regulated and 11 up-regulated in drug-resistant breast cancer tissues compared with drug-sensitive tissues, using a miRNA microarray.  The results also showed miR-489 to be one of the most down-regulated miRNAs in drug-resistant tissues and cell lines, as confirmed by miRNA microarray screening and real-time quant. PCR.  A decrease in miR-489 expression was assocd. with chemoresistance as well as lymph node metastasis, increased tumor size, advanced pTNM stage and poor prognosis in breast cancer.  Functional anal. revealed that miR-489 increased breast cancer chemosensitivity and inhibited cell proliferation, migration and invasion, both in vitro and in vivo.  Furthermore, SPIN1, VAV3, BCL2 and AKT3 were found to be direct targets of miR-489.  SPIN1 was significantly elevated in drug-resistant and metastatic breast cancer tissues and inversely correlated with miR-489 expression.  High expression of SPIN1 was assocd. with higher histol. grade, lymph node metastasis, advanced pTNM stage and pos. progesterone receptor (PR) status.  Increased SPIN1 expression enhanced cell migration and invasion, inhibited apoptosis and partially antagonized the effects of miR-489 in breast cancer.  PIK3CA, AKT, CREB1 and BCL2 in the PI3K-Akt signalling pathway, demonstrated to be elevated in drug-resistant breast cancer tissues, were identified as downstream effectors of SPIN1.  It was further found that either inhibition of SPIN1 or overexpression of miR-489 suppressed the PI3K-Akt signalling pathway.  These data indicate that miR-489 could reverse the chemoresistance of breast cancer via the PI3K-Akt pathway by targeting SPIN1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlYdZmKYimMbVg90H21EOLACvtfcHk0lhPmaE1Z2imCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1SmtL7O&md5=fe996113e57b3a6c2c0cce1b15c6ad0c</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1002%2Fpath.4743&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpath.4743%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DY.-W.%26aulast%3DXing%26aufirst%3DA.-Y.%26aulast%3DXiang%26aufirst%3DS.%26aulast%3DShi%26aufirst%3DD.-B.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DY.-X.%26aulast%3DGao%26aufirst%3DP.%26atitle%3DSuppression%2520of%2520SPIN1-mediated%2520PI3K%25E2%2580%2593Akt%2520pathway%2520by%2520miR-489%2520increases%2520chemosensitivity%2520in%2520breast%2520cancer%26jtitle%3DJ.%2520Pathol.%26date%3D2016%26volume%3D239%26spage%3D459%26epage%3D472%26doi%3D10.1002%2Fpath.4743" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plummer, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Margolin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, S. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, H.</span></span> <span> </span><span class="NLM_article-title">SPIN1 promotes tumorigenesis by blocking the uL18 (universal large ribosomal subunit protein 18)-MDM2-p53 pathway in human cancer</span>. <i>eLife</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">e31275</span> <span class="refDoi"> DOI: 10.7554/elife.31275</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.7554%2Felife.31275" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=29547122" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2018&author=Z.+Fangauthor=B.+Caoauthor=J.-M.+Liaoauthor=J.+Dengauthor=K.+D.+Plummerauthor=P.+Liaoauthor=T.+Liuauthor=W.+Zhangauthor=K.+Zhangauthor=L.+Liauthor=D.+Margolinauthor=S.+X.+Zengauthor=J.+Xiongauthor=H.+Lu&title=SPIN1+promotes+tumorigenesis+by+blocking+the+uL18+%28universal+large+ribosomal+subunit+protein+18%29-MDM2-p53+pathway+in+human+cancer&doi=10.7554%2Felife.31275"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.7554%2Felife.31275&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252Felife.31275%26sid%3Dliteratum%253Aachs%26aulast%3DFang%26aufirst%3DZ.%26aulast%3DCao%26aufirst%3DB.%26aulast%3DLiao%26aufirst%3DJ.-M.%26aulast%3DDeng%26aufirst%3DJ.%26aulast%3DPlummer%26aufirst%3DK.%2BD.%26aulast%3DLiao%26aufirst%3DP.%26aulast%3DLiu%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DMargolin%26aufirst%3DD.%26aulast%3DZeng%26aufirst%3DS.%2BX.%26aulast%3DXiong%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DH.%26atitle%3DSPIN1%2520promotes%2520tumorigenesis%2520by%2520blocking%2520the%2520uL18%2520%2528universal%2520large%2520ribosomal%2520subunit%2520protein%252018%2529-MDM2-p53%2520pathway%2520in%2520human%2520cancer%26jtitle%3DeLife%26date%3D2018%26volume%3D7%26doi%3D10.7554%2Felife.31275" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.</span></span> <span> </span><span class="NLM_article-title">miR-489 inhibits proliferation, cell cycle progression and induces apoptosis of glioma cells via targeting SPIN1-mediated PI3K/AKT pathway</span>. <i>Biomed. Pharmacother.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">435</span>– <span class="NLM_lpage">443</span>, <span class="refDoi"> DOI: 10.1016/j.biopha.2017.06.058</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1016%2Fj.biopha.2017.06.058" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=28666210" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVyht7%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=2017&pages=435-443&author=Y.+Liauthor=X.+Maauthor=Y.+Wangauthor=G.+Li&title=miR-489+inhibits+proliferation%2C+cell+cycle+progression+and+induces+apoptosis+of+glioma+cells+via+targeting+SPIN1-mediated+PI3K%2FAKT+pathway&doi=10.1016%2Fj.biopha.2017.06.058"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">miR-489 inhibits proliferation, cell cycle progression and induces apoptosis of glioma cells via targeting SPIN1-mediated PI3K/AKT pathway</span></div><div class="casAuthors">Li, Yan; Ma, Xiaolin; Wang, Yanpeng; Li, Guohua</div><div class="citationInfo"><span class="NLM_cas:title">Biomedicine & Pharmacotherapy</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">435-443</span>CODEN:
                <span class="NLM_cas:coden">BIPHEX</span>;
        ISSN:<span class="NLM_cas:issn">0753-3322</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">MicroRNA-489 (miR-489), a newly identified tumor-related miRNA, functions as an oncogene or tumor suppressor via regulating growth and metastasis of human cancers.  But, the clin. significance, biol. function and underlying mechanisms of miR-489 in glioma remain rarely known.  Here, we showed that the levels of miR-489 in glioma tissues were notably underexpressed compared to corresponding non-tumor tissues.  In accordance, the relative levels of miR-489 were decreased in glioma cell lines compared with NHA cells.  Kaplan-Meier plots indicated that miR-489 low expressing glioma patients showed a prominent shorter overall survival.  In addn., miR-489 overexpression prohibited proliferation and cell cycle progression, and promoted apoptosis in U251 cells.  While, miR-489 knockdown showed opposite effects on these cellular processes of U87 cells.  In vivo expts. demonstrated that miR-489 restoration reduced the tumor vol. and wt. of s.c. glioma xenografts in nude mice.  Notably, Spindlin 1 (SPIN1) was inversely and directly regulated by miR-489 in glioma cells.  A neg. correlation between the expression of miR-489 and SPIN1 mRNA was confirmed in glioma tissues.  Interestingly, miR-489 inversely modulated activation of PI3K/AKT pathway and expression of downstream targets including p-mTOR, Cyclin D1 and BCL-XL.  SPIN1 re-expression abolished the effects of miR-489 on U251 cells with enhanced activation of PI3K/AKT pathway and malignant phenotype.  Meanwhile, AKT inhibitor MK-2206 blocked activation of PI3K/AKT pathway and resulted in reduced proliferation, cell cycle arrest and increased apoptosis in miR-489 down-regulating U87 cells.  Altogether, our data support that miR-489 loss facilitates malignant phenotype of glioma cells probably via SPIN1-mediated PI3K/AKT pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq39T3IjNfjurVg90H21EOLACvtfcHk0liCr-fcZ14kag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVyht7%252FE&md5=4cd57f05e0bbfec6fd2206244d73c291</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.biopha.2017.06.058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biopha.2017.06.058%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DG.%26atitle%3DmiR-489%2520inhibits%2520proliferation%252C%2520cell%2520cycle%2520progression%2520and%2520induces%2520apoptosis%2520of%2520glioma%2520cells%2520via%2520targeting%2520SPIN1-mediated%2520PI3K%252FAKT%2520pathway%26jtitle%3DBiomed.%2520Pharmacother.%26date%3D2017%26volume%3D93%26spage%3D435%26epage%3D443%26doi%3D10.1016%2Fj.biopha.2017.06.058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, G.</span></span> <span> </span><span class="NLM_article-title">Long noncoding RNA MHENCR promotes melanoma progression via regulating miR-425/489-mediated PI3K-Akt pathway</span>. <i>Am. J. Transl. Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">90</span>– <span class="NLM_lpage">102</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=28123636" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1ChsLzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=90-102&author=X.+Chenauthor=H.+Dongauthor=S.+Liuauthor=L.+Yuauthor=D.+Yanauthor=X.+Yaoauthor=W.+Sunauthor=D.+Hanauthor=G.+Gao&title=Long+noncoding+RNA+MHENCR+promotes+melanoma+progression+via+regulating+miR-425%2F489-mediated+PI3K-Akt+pathway"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Long noncoding RNA MHENCR promotes melanoma progression via regulating miR-425/489-mediated PI3K-Akt pathway</span></div><div class="casAuthors">Chen, Xiangjun; Dong, Hao; Liu, Sha; Yu, Li; Yan, Dexiong; Yao, Xingwei; Sun, Weijing; Han, Dezhi; Gao, Guozhen</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Translational Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">90-105, S1-S2</span>CODEN:
                <span class="NLM_cas:coden">AJTRA7</span>;
        ISSN:<span class="NLM_cas:issn">1943-8141</span>.
    
            (<span class="NLM_cas:orgname">e-Century Publishing Corp.</span>)
        </div><div class="casAbstract">Increasing evidences demonstrated that long noncoding RNAs (lncRNAs) are frequently dysregulated and have crit. roles in many tumors.  However, the roles and functional mechanisms of lncRNAs in melanoma remain largely unknown.  In this study, we identified a novel lncRNA MHENCR which was upregulated in melanoma tissues and further upregulated in metastatic melanoma.  Increased expression of MHENCR indicted poor survival of melanoma patients.  Functional expts. revealed that MHENCR knockdown significantly inhibited melanoma cells proliferation, induced cell cycle arrest and apoptosis, and also attenuated melanoma cells migration in vitro.  Furthermore, we identified MHENCR as a competitively endogenous RNA, which specifically bound to miR-425 and miR-489, upregulated their target genes IGF1 and SPIN1 expression, and further activated PI3K-Akt pathway.  Statistically significant correlations were obsd. between MHENCR expression and IGF1 and SPIN1 in melanoma tissues.  In vivo functional expts. further confirmed the pro-growth and pro-metastasis roles of MHENCR.  Collectively, our findings revealed that MHENCR functions as an oncogene in melanoma via activating miR-425/489-mediated PI3K-Akt pathway, and may be a therapeutic target for melanoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPAKn59fHqdrVg90H21EOLACvtfcHk0liCr-fcZ14kag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1ChsLzK&md5=829b7d67ae7fe250ad4d6576dd3913c6</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DX.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DYu%26aufirst%3DL.%26aulast%3DYan%26aufirst%3DD.%26aulast%3DYao%26aufirst%3DX.%26aulast%3DSun%26aufirst%3DW.%26aulast%3DHan%26aufirst%3DD.%26aulast%3DGao%26aufirst%3DG.%26atitle%3DLong%2520noncoding%2520RNA%2520MHENCR%2520promotes%2520melanoma%2520progression%2520via%2520regulating%2520miR-425%252F489-mediated%2520PI3K-Akt%2520pathway%26jtitle%3DAm.%2520J.%2520Transl.%2520Res.%26date%3D2017%26volume%3D9%26spage%3D90%26epage%3D102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, X.</span></span> <span> </span><span class="NLM_article-title">Overexpression of Spindlin1 induces cellular senescence, multinucleation and apoptosis</span>. <i>Gene</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>410</i></span>,  <span class="NLM_fpage">67</span>– <span class="NLM_lpage">74</span>, <span class="refDoi"> DOI: 10.1016/j.gene.2007.11.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1016%2Fj.gene.2007.11.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=18201843" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BD1cXitlWksL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=410&publication_year=2008&pages=67-74&author=H.+Yuanauthor=P.+Zhangauthor=L.+Qinauthor=L.+Chenauthor=S.+Shiauthor=Y.+Luauthor=F.+Yanauthor=C.+Baiauthor=X.+Nanauthor=D.+Liuauthor=Y.+Liauthor=W.+Yueauthor=X.+Pei&title=Overexpression+of+Spindlin1+induces+cellular+senescence%2C+multinucleation+and+apoptosis&doi=10.1016%2Fj.gene.2007.11.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Overexpression of SPINDLIN1 induces cellular senescence, multinucleation and apoptosis</span></div><div class="casAuthors">Yuan, Hongfeng; Zhang, Peng; Qin, Lipeng; Chen, Lin; Shi, Shuangshuang; Lu, Yang; Yan, Fang; Bai, Cixian; Nan, Xue; Liu, Daqing; Li, Yanhua; Yue, Wen; Pei, Xuetao</div><div class="citationInfo"><span class="NLM_cas:title">Gene</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">410</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">67-74</span>CODEN:
                <span class="NLM_cas:coden">GENED6</span>;
        ISSN:<span class="NLM_cas:issn">0378-1119</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Human or mouse SPLINDIN1 is expressed in various tissues and cells, but its biol. functions are poorly understood.  In this study, we show that human SPINDLIN1 is localized to interphase nucleus and mitotic chromosomes, and its expression in HeLa cells is not regulated in a cell cycle-dependent manner.  When SPINDLIN1 is stably overexpressed in HeLa cells, it results in multinucleation of cells, and these multinucleated cells exhibits characteristic features of senescence and apoptosis shown by growth and morphol. alterations, β-galactosidase activity, and Annexin V/7-Aminoactinomycin D staining.  Mouse Spindlin1 is highly homologous with human SPLINDIN1, when overexpressed in NIH3T3 cells, it also induces multinucleation, senescence and apoptosis in murine cells.  These results demonstrate that SPINDLIN1 is an important gene for mammalian mitotic chromosome functions, and disrupted regulation results in abnormal cell division, a mechanism that may be involved in tumorigenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpuwUiRH1KJwbVg90H21EOLACvtfcHk0lgwOTB2s41Vlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXitlWksL0%253D&md5=e9cf822e1145404f82024fb326d57a92</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.gene.2007.11.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.gene.2007.11.019%26sid%3Dliteratum%253Aachs%26aulast%3DYuan%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DP.%26aulast%3DQin%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DShi%26aufirst%3DS.%26aulast%3DLu%26aufirst%3DY.%26aulast%3DYan%26aufirst%3DF.%26aulast%3DBai%26aufirst%3DC.%26aulast%3DNan%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DYue%26aufirst%3DW.%26aulast%3DPei%26aufirst%3DX.%26atitle%3DOverexpression%2520of%2520Spindlin1%2520induces%2520cellular%2520senescence%252C%2520multinucleation%2520and%2520apoptosis%26jtitle%3DGene%26date%3D2008%26volume%3D410%26spage%3D67%26epage%3D74%26doi%3D10.1016%2Fj.gene.2007.11.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, X.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, H.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhai, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.-N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jia, Y.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, X.-T.</span></span> <span> </span><span class="NLM_article-title">Spindline1 promotes cancer cell proliferation through activation of WNT/TCF-4 signaling</span>. <i>Mol. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">326</span>– <span class="NLM_lpage">335</span>, <span class="refDoi"> DOI: 10.1158/1541-7786.mcr-11-0440</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1158%2F1541-7786.mcr-11-0440" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=22258766" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC38XktVamsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2012&pages=326-335&author=J.-X.+Wangauthor=Q.+Zengauthor=L.+Chenauthor=J.-C.+Duauthor=X.-L.+Yanauthor=H.-F.+Yuanauthor=C.+Zhaiauthor=J.-N.+Zhouauthor=Y.-L.+Jiaauthor=W.+Yueauthor=X.-T.+Pei&title=Spindline1+promotes+cancer+cell+proliferation+through+activation+of+WNT%2FTCF-4+signaling&doi=10.1158%2F1541-7786.mcr-11-0440"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">SPINDLIN1 Promotes Cancer Cell Proliferation through Activation of WNT/TCF-4 Signaling</span></div><div class="casAuthors">Wang, Jing-Xue; Zeng, Quan; Chen, Lin; Du, Ji-Chao; Yan, Xin-Long; Yuan, Hong-Feng; Zhai, Chao; Zhou, Jun-Nian; Jia, Ya-Li; Yue, Wen; Pei, Xue-Tao</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">326-335</span>CODEN:
                <span class="NLM_cas:coden">MCROC5</span>;
        ISSN:<span class="NLM_cas:issn">1541-7786</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">SPINDLIN1, a new member of the SPIN/SSTY gene family, was first identified as a gene highly expressed in ovarian cancer cells.  We have previously shown that it is involved in the process of spindle organization and chromosomal stability and plays a role in the development of cancer.  Nevertheless, the mechanisms underlying its oncogenic role are still largely unknown.  Here, we first showed that expression of SPINDLIN1 is upregulated in clin. tumors.  Ectopic expression of SPINDLIN1 promoted cancer cell proliferation and activated WNT/T-cell factor (TCF)-4 signaling.  The Ser84 and Ser99 amino acids within SPINDLIN1 were further identified as the key functional sites in WNT/TCF-4 signaling activation.  Mutation of these two sites of SPINDLIN1 abolished its effects on promoting WNT/TCF-4 signaling and cancer cell proliferation.  We further found that Aurora-A could interact with and phosphorylate SPINDLIN1 at its key functional sites, Ser84 and Ser99, suggesting that phosphorylation of SPINDLIN1 is involved in its oncogenic function.  Collectively, these results suggest that SPINDLIN1, which may be a novel substrate of the Aurora-A kinase, promotes cancer cell growth through WNT/TCF-4 signaling activation.  Mol Cancer Res; 10(3); 326-35.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoie76oMHEUN7Vg90H21EOLACvtfcHk0lgwOTB2s41Vlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XktVamsrc%253D&md5=0446631fe80c7b518ce305c394075143</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1158%2F1541-7786.mcr-11-0440&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1541-7786.mcr-11-0440%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.-X.%26aulast%3DZeng%26aufirst%3DQ.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DDu%26aufirst%3DJ.-C.%26aulast%3DYan%26aufirst%3DX.-L.%26aulast%3DYuan%26aufirst%3DH.-F.%26aulast%3DZhai%26aufirst%3DC.%26aulast%3DZhou%26aufirst%3DJ.-N.%26aulast%3DJia%26aufirst%3DY.-L.%26aulast%3DYue%26aufirst%3DW.%26aulast%3DPei%26aufirst%3DX.-T.%26atitle%3DSpindline1%2520promotes%2520cancer%2520cell%2520proliferation%2520through%2520activation%2520of%2520WNT%252FTCF-4%2520signaling%26jtitle%3DMol.%2520Cancer%2520Res.%26date%3D2012%26volume%3D10%26spage%3D326%26epage%3D335%26doi%3D10.1158%2F1541-7786.mcr-11-0440" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abell, C.</span></span> <span> </span><span class="NLM_article-title">Fragment-based approaches to enzyme inhibition</span>. <i>Curr. Opin. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">489</span>– <span class="NLM_lpage">496</span>, <span class="refDoi"> DOI: 10.1016/j.copbio.2007.09.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1016%2Fj.copbio.2007.09.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=17959370" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BD1cXislOrsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2007&pages=489-496&author=A.+Ciulliauthor=C.+Abell&title=Fragment-based+approaches+to+enzyme+inhibition&doi=10.1016%2Fj.copbio.2007.09.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based approaches to enzyme inhibition</span></div><div class="casAuthors">Ciulli, Alessio; Abell, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Biotechnology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">489-496</span>CODEN:
                <span class="NLM_cas:coden">CUOBE3</span>;
        ISSN:<span class="NLM_cas:issn">0958-1669</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Fragment-based approaches have provided a new paradigm for small-mol. drug discovery.  The methodol. is complementary to high-throughput screening approaches, starting from fragments of low mol. complexity and high ligand efficiency, and building up to more potent inhibitors.  The approach, which depends heavily on a no. of biophys. techniques, is now being taken up by more groups in both industry and academia.  This article describes key aspects of the process and highlights recent developments and applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzMH6I2sXlL7Vg90H21EOLACvtfcHk0lgwOTB2s41Vlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXislOrsg%253D%253D&md5=ab79b47a77574d5174b9dd0c989c0e3c</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.copbio.2007.09.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.copbio.2007.09.003%26sid%3Dliteratum%253Aachs%26aulast%3DCiulli%26aufirst%3DA.%26aulast%3DAbell%26aufirst%3DC.%26atitle%3DFragment-based%2520approaches%2520to%2520enzyme%2520inhibition%26jtitle%3DCurr.%2520Opin.%2520Biotechnol.%26date%3D2007%26volume%3D18%26spage%3D489%26epage%3D496%26doi%3D10.1016%2Fj.copbio.2007.09.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rees, D. C.</span></span> <span> </span><span class="NLM_article-title">The rise of fragment-based drug discovery</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">187</span>– <span class="NLM_lpage">192</span>, <span class="refDoi"> DOI: 10.1038/nchem.217</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1038%2Fnchem.217" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=21378847" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmtF2rs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2009&pages=187-192&author=C.+W.+Murrayauthor=D.+C.+Rees&title=The+rise+of+fragment-based+drug+discovery&doi=10.1038%2Fnchem.217"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">The rise of fragment-based drug discovery</span></div><div class="casAuthors">Murray, Christopher W.; Rees, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">187-192</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The search for new drugs is plagued by high attrition rates at all stages in research and development.  Chemists have an opportunity to tackle this problem because attrition can be traced back, in part, to the quality of the chem. leads.  Fragment-based drug discovery (FBDD) is a new approach, increasingly used in the pharmaceutical industry, for reducing attrition and providing leads for previously intractable biol. targets.  FBDD identifies low-mol.-wt. ligands (∼150 Da) that bind to biol. important macromols.  The three-dimensional exptl. binding mode of these fragments is detd. using X-ray crystallog. or NMR spectroscopy, and is used to facilitate their optimization into potent mols. with drug-like properties.  Compared with high-throughput-screening, the fragment approach requires fewer compds. to be screened, and, despite the lower initial potency of the screening hits, offers more efficient and fruitful optimization campaigns.  Here, we review the rise of FBDD, including its application to discovering clin. candidates against targets for which other chem. approaches have struggled.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqg3ip2FKcYJbVg90H21EOLACvtfcHk0lgwOTB2s41Vlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmtF2rs78%253D&md5=0bd1a40fd927cce9200f0fa296a7462f</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1038%2Fnchem.217&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.217%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DRees%26aufirst%3DD.%2BC.%26atitle%3DThe%2520rise%2520of%2520fragment-based%2520drug%2520discovery%26jtitle%3DNat.%2520Chem.%26date%3D2009%26volume%3D1%26spage%3D187%26epage%3D192%26doi%3D10.1038%2Fnchem.217" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leeson, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Springthorpe, B.</span></span> <span> </span><span class="NLM_article-title">The influence of drug-like concepts on decision-making in medicinal chemistry</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">881</span>– <span class="NLM_lpage">890</span>, <span class="refDoi"> DOI: 10.1038/nrd2445</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1038%2Fnrd2445" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=17971784" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1els7rL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2007&pages=881-890&author=P.+D.+Leesonauthor=B.+Springthorpe&title=The+influence+of+drug-like+concepts+on+decision-making+in+medicinal+chemistry&doi=10.1038%2Fnrd2445"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">The influence of drug-like concepts on decision-making in medicinal chemistry</span></div><div class="casAuthors">Leeson, Paul D.; Springthorpe, Brian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">881-890</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Despite the wide acceptance of drug-like principles such as the 'rule of five', this anal. of mols. currently being synthesized in leading pharmaceutical companies reveals that their phys. properties differ significantly from those of recently discovered oral drugs.  The marked increase in lipophilicity in particular could increase the likelihood of attrition in drug development.  The application of guidelines linked to the concept of drug-likeness, such as the 'rule of five', has gained wide acceptance as an approach to reduce attrition in drug discovery and development.  However, despite this acceptance, anal. of recent trends reveals that the phys. properties of mols. that are currently being synthesized in leading drug discovery companies differ significantly from those of recently discovered oral drugs and compds. in clin. development.  The consequences of the marked increase in lipophilicity - the most important drug-like phys. property - include a greater likelihood of lack of selectivity and attrition in drug development.  Tackling the threat of compd.-related toxicol. attrition needs to move to the mainstream of medicinal chem. decision-making.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTUeonLD6iebVg90H21EOLACvtfcHk0lh6lvgBPkYNRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1els7rL&md5=5665a9e2dc266a1ee096e20625757ef7</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1038%2Fnrd2445&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2445%26sid%3Dliteratum%253Aachs%26aulast%3DLeeson%26aufirst%3DP.%2BD.%26aulast%3DSpringthorpe%26aufirst%3DB.%26atitle%3DThe%2520influence%2520of%2520drug-like%2520concepts%2520on%2520decision-making%2520in%2520medicinal%2520chemistry%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2007%26volume%3D6%26spage%3D881%26epage%3D890%26doi%3D10.1038%2Fnrd2445" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allali-Hassani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herold, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yates, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Optimization of cellular activity of G9a inhibitors 7-aminoalkoxy-quinazolines</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">6139</span>– <span class="NLM_lpage">6150</span>, <span class="refDoi"> DOI: 10.1021/jm200903z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200903z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpvVWltLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=6139-6150&author=F.+Liuauthor=D.+Barsyte-Lovejoyauthor=A.+Allali-Hassaniauthor=Y.+Heauthor=J.+M.+Heroldauthor=X.+Chenauthor=C.+M.+Yatesauthor=S.+V.+Fryeauthor=P.+J.+Brownauthor=J.+Huangauthor=M.+Vedadiauthor=C.+H.+Arrowsmithauthor=J.+Jin&title=Optimization+of+cellular+activity+of+G9a+inhibitors+7-aminoalkoxy-quinazolines&doi=10.1021%2Fjm200903z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of cellular activity of G9a inhibitors 7-aminoalkoxy-quinazolines</span></div><div class="casAuthors">Liu, Feng; Barsyte-Lovejoy, Dalia; Allali-Hassani, Abdellah; He, Yunlong; Herold, J. Martin; Chen, Xin; Yates, Christopher M.; Frye, Stephen V.; Brown, Peter J.; Huang, Jing; Vedadi, Masoud; Arrowsmith, Cheryl H.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">6139-6150</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein lysine methyltransferase G9a plays key roles in the transcriptional repression of a variety of genes via dimethylation of lysine 9 on histone H3 (H3K9me2) of chromatin as well as dimethylation of nonhistone proteins including tumor suppressor p53.  We previously reported the discovery of UNC0321, the most potent G9a inhibitor to date, via structure-based design and structure-activity relationship (SAR) exploration of the quinazoline scaffold represented by BIX01294.  Despite its very high in vitro potency, compd. UNC0321 lacks sufficient cellular potency.  The design and synthesis of several generations of new analogs aimed at improving cell membrane permeability while maintaining high in vitro potency resulted in the discovery of a no. of novel G9a inhibitors such as UNC0646 I and UNC0631 II with excellent potency in a variety of cell lines and excellent sepn. of functional potency vs. cell toxicity.  The design, synthesis, and cellular SAR of these potent G9a inhibitors are described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrC91yKB7S34rVg90H21EOLACvtfcHk0lh6lvgBPkYNRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpvVWltLg%253D&md5=f3f54b91e6b24aa248498f9ff1fdb1ae</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fjm200903z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200903z%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DHe%26aufirst%3DY.%26aulast%3DHerold%26aufirst%3DJ.%2BM.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DYates%26aufirst%3DC.%2BM.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DOptimization%2520of%2520cellular%2520activity%2520of%2520G9a%2520inhibitors%25207-aminoalkoxy-quinazolines%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D6139%26epage%3D6150%26doi%3D10.1021%2Fjm200903z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korboukh, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allali-Hassani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janzen, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of an in vivo chemical probe of the lysine methyltransferases G9a and GLP</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">8931</span>– <span class="NLM_lpage">8942</span>, <span class="refDoi"> DOI: 10.1021/jm401480r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401480r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsF2htrvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=8931-8942&author=F.+Liuauthor=D.+Barsyte-Lovejoyauthor=F.+Liauthor=Y.+Xiongauthor=V.+Korboukhauthor=X.-P.+Huangauthor=A.+Allali-Hassaniauthor=W.+P.+Janzenauthor=B.+L.+Rothauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=M.+Vedadiauthor=J.+Jin&title=Discovery+of+an+in+vivo+chemical+probe+of+the+lysine+methyltransferases+G9a+and+GLP&doi=10.1021%2Fjm401480r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of an in vivo chemical probe of the lysine methyltransferases G9a and GLP</span></div><div class="casAuthors">Liu, Feng; Barsyte-Lovejoy, Dalia; Li, Fengling; Xiong, Yan; Korboukh, Victoria; Huang, Xi-Ping; Allali-Hassani, Abdellah; Janzen, William P.; Roth, Bryan L.; Frye, Stephen V.; Arrowsmith, Cheryl H.; Brown, Peter J.; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">8931-8942</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Among epigenetic writers, readers, and erasers, the lysine methyltransferases G9a and GLP, which catalyze mono- and dimethylation of histone H3 lysine 9 (H3K9me2) and nonhistone proteins, have been implicated in a variety of human diseases.  A toolkit of well-characterized chem. probes will allow biol. and disease hypotheses concerning these proteins to be tested in cell-based and animal models with high confidence.  We previously discovered potent and selective G9a/GLP inhibitors including the cellular chem. probe UNC0638, which displays an excellent sepn. of functional potency and cell toxicity.  However, this inhibitor is not suitable for animal studies due to its poor pharmacokinetic (PK) properties.  Here, we report the discovery of the first G9a and GLP in vivo chem. probe UNC0642, which not only maintains high in vitro and cellular potency, low cell toxicity, and excellent selectivity, but also displays improved in vivo PK properties, making it suitable for animal studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowOI3TSEz3ZbVg90H21EOLACvtfcHk0lh6lvgBPkYNRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsF2htrvK&md5=933f0c49a938b4dd8146f6d09aa21e21</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fjm401480r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401480r%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DKorboukh%26aufirst%3DV.%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520an%2520in%2520vivo%2520chemical%2520probe%2520of%2520the%2520lysine%2520methyltransferases%2520G9a%2520and%2520GLP%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D8931%26epage%3D8942%26doi%3D10.1021%2Fjm401480r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rival-Gervier, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allali-Hassani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labrie, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wigle, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dimaggio, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wasney, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siarheyeva, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tempel, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chau, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangano, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.-p.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simpson, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pattenden, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norris, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kireev, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tripathy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janzen, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petronis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">A chemical probe selectively inhibits G9a and GLP methyltransferase activity in cells</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">566</span>– <span class="NLM_lpage">574</span>, <span class="refDoi"> DOI: 10.1038/nchembio.599</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1038%2Fnchembio.599" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=21743462" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC3MXos1OmtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=566-574&author=M.+Vedadiauthor=D.+Barsyte-Lovejoyauthor=F.+Liuauthor=S.+Rival-Gervierauthor=A.+Allali-Hassaniauthor=V.+Labrieauthor=T.+J.+Wigleauthor=P.+A.+Dimaggioauthor=G.+A.+Wasneyauthor=A.+Siarheyevaauthor=A.+Dongauthor=W.+Tempelauthor=S.-C.+Wangauthor=X.+Chenauthor=I.+Chauauthor=T.+J.+Manganoauthor=X.-p.+Huangauthor=C.+D.+Simpsonauthor=S.+G.+Pattendenauthor=J.+L.+Norrisauthor=D.+B.+Kireevauthor=A.+Tripathyauthor=A.+Edwardsauthor=B.+L.+Rothauthor=W.+P.+Janzenauthor=B.+A.+Garciaauthor=A.+Petronisauthor=J.+Ellisauthor=P.+J.+Brownauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=J.+Jin&title=A+chemical+probe+selectively+inhibits+G9a+and+GLP+methyltransferase+activity+in+cells&doi=10.1038%2Fnchembio.599"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">A chemical probe selectively inhibits G9a and GLP methyltransferase activity in cells</span></div><div class="casAuthors">Vedadi, Masoud; Barsyte-Lovejoy, Dalia; Liu, Feng; Rival-Gervier, Sylvie; Allali-Hassani, Abdellah; Labrie, Viviane; Wigle, Tim J.; DiMaggio, Peter A.; Wasney, Gregory A.; Siarheyeva, Alena; Dong, Aiping; Tempel, Wolfram; Wang, Sun-Chong; Chen, Xin; Chau, Irene; Mangano, Thomas J.; Huang, Xi-ping; Simpson, Catherine D.; Pattenden, Samantha G.; Norris, Jacqueline L.; Kireev, Dmitri B.; Tripathy, Ashutosh; Edwards, Aled; Roth, Bryan L.; Janzen, William P.; Garcia, Benjamin A.; Petronis, Arturas; Ellis, James; Brown, Peter J.; Frye, Stephen V.; Arrowsmith, Cheryl H.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">566-574</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein lysine methyltransferases G9a and GLP modulate the transcriptional repression of a variety of genes via dimethylation of Lys9 on histone H3 (H3K9me2) as well as dimethylation of non-histone targets.  Here we report the discovery of UNC0638, an inhibitor of G9a and GLP with excellent potency and selectivity over a wide range of epigenetic and non-epigenetic targets.  UNC0638 treatment of a variety of cell lines resulted in lower global H3K9me2 levels, equiv. to levels obsd. for small hairpin RNA knockdown of G9a and GLP with the functional potency of UNC0638 being well sepd. from its toxicity.  UNC0638 markedly reduced the clonogenicity of MCF7 cells, reduced the abundance of H3K9me2 marks at promoters of known G9a-regulated endogenous genes and disproportionately affected several genomic loci encoding microRNAs.  In mouse embryonic stem cells, UNC0638 reactivated G9a-silenced genes and a retroviral reporter gene in a concn.-dependent manner without promoting differentiation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphv-2bZcBqwLVg90H21EOLACvtfcHk0liVS8B5-dCKZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXos1OmtLs%253D&md5=a944e99a910194c1a15240fc6dc8f852</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.599&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.599%26sid%3Dliteratum%253Aachs%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DRival-Gervier%26aufirst%3DS.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DLabrie%26aufirst%3DV.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DDimaggio%26aufirst%3DP.%2BA.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DTempel%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DS.-C.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DMangano%26aufirst%3DT.%2BJ.%26aulast%3DHuang%26aufirst%3DX.-p.%26aulast%3DSimpson%26aufirst%3DC.%2BD.%26aulast%3DPattenden%26aufirst%3DS.%2BG.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DKireev%26aufirst%3DD.%2BB.%26aulast%3DTripathy%26aufirst%3DA.%26aulast%3DEdwards%26aufirst%3DA.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26aulast%3DGarcia%26aufirst%3DB.%2BA.%26aulast%3DPetronis%26aufirst%3DA.%26aulast%3DEllis%26aufirst%3DJ.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DA%2520chemical%2520probe%2520selectively%2520inhibits%2520G9a%2520and%2520GLP%2520methyltransferase%2520activity%2520in%2520cells%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D566%26epage%3D574%26doi%3D10.1038%2Fnchembio.599" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sweis, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pliushchev, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maag, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petros, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soni, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tse, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michaelides, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pappano, W. N.</span></span> <span> </span><span class="NLM_article-title">Discovery and Development of Potent and Selective Inhibitors of Histone Methyltransferase G9a</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">205</span>– <span class="NLM_lpage">209</span>, <span class="refDoi"> DOI: 10.1021/ml400496h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml400496h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFKmsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=205-209&author=R.+F.+Sweisauthor=M.+Pliushchevauthor=P.+J.+Brownauthor=J.+Guoauthor=F.+Liauthor=D.+Maagauthor=A.+M.+Petrosauthor=N.+B.+Soniauthor=C.+Tseauthor=M.+Vedadiauthor=M.+R.+Michaelidesauthor=G.+G.+Chiangauthor=W.+N.+Pappano&title=Discovery+and+Development+of+Potent+and+Selective+Inhibitors+of+Histone+Methyltransferase+G9a&doi=10.1021%2Fml400496h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Development of Potent and Selective Inhibitors of Histone Methyltransferase G9a</span></div><div class="casAuthors">Sweis, Ramzi F.; Pliushchev, Marina; Brown, Peter J.; Guo, Jun; Li, Fengling; Maag, David; Petros, Andrew M.; Soni, Nirupama B.; Tse, Chris; Vedadi, Masoud; Michaelides, Michael R.; Chiang, Gary G.; Pappano, William N.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">205-209</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">G9a is a histone lysine methyltransferase responsible for the methylation of histone H3 lysine 9.  The discovery of A-366 arose from a unique diversity screening hit, which was optimized by incorporation of a propyl-pyrrolidine subunit to occupy the enzyme lysine channel.  A-366 (I) is a potent inhibitor of G9a (IC50: 3.3 nM) with greater than 1000-fold selectivity over 21 other methyltransferases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMvdTiUgx-oLVg90H21EOLACvtfcHk0liVS8B5-dCKZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFKmsA%253D%253D&md5=26affad72f2950510aa9f253e540d575</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fml400496h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml400496h%26sid%3Dliteratum%253Aachs%26aulast%3DSweis%26aufirst%3DR.%2BF.%26aulast%3DPliushchev%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DGuo%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DMaag%26aufirst%3DD.%26aulast%3DPetros%26aufirst%3DA.%2BM.%26aulast%3DSoni%26aufirst%3DN.%2BB.%26aulast%3DTse%26aufirst%3DC.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DMichaelides%26aufirst%3DM.%2BR.%26aulast%3DChiang%26aufirst%3DG.%2BG.%26aulast%3DPappano%26aufirst%3DW.%2BN.%26atitle%3DDiscovery%2520and%2520Development%2520of%2520Potent%2520and%2520Selective%2520Inhibitors%2520of%2520Histone%2520Methyltransferase%2520G9a%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D5%26spage%3D205%26epage%3D209%26doi%3D10.1021%2Fml400496h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oh, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solter, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knowles, B. B.</span></span> <span> </span><span class="NLM_article-title">Spindlin, a major maternal transcript expressed in the mouse during the transition from oocyte to embryo</span>. <i>Development</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>124</i></span>,  <span class="NLM_fpage">493</span>– <span class="NLM_lpage">503</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=9053325" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADyaK2sXhvVWmsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=1997&pages=493-503&author=B.+Ohauthor=S.+Y.+Hwangauthor=D.+Solterauthor=B.+B.+Knowles&title=Spindlin%2C+a+major+maternal+transcript+expressed+in+the+mouse+during+the+transition+from+oocyte+to+embryo"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Spindlin, a major maternal transcript expressed in the mouse during the transition from oocyte to embryo</span></div><div class="casAuthors">Oh, Bermseok; Hwang, Sue-Yun; Solter, Davor; Knowles, Barbara B.</div><div class="citationInfo"><span class="NLM_cas:title">Development (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">124</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">493-503</span>CODEN:
                <span class="NLM_cas:coden">DEVPED</span>;
        ISSN:<span class="NLM_cas:issn">0950-1991</span>.
    
            (<span class="NLM_cas:orgname">Company of Biologists</span>)
        </div><div class="casAbstract">Timely translation of maternal transcriptions and post-translational modification of their gene products control the initial development of preimplantation-stage embryos.  We have isolated and characterized a gene encoding a stage-specific embryonic protein.  This novel gene, spindlin (Spin), is an abundant maternal transcript present in the unfertilized egg and 2-cell, but not 8-cell, stage embryo.  Spin exhibits high homol. to a multicopy gene, Y-linked spermiogenesis-specific transcript (Ssty), and together they form a new gene family expressed during gametogenesis.  We find that spindlin assocs. with the meiotic spindle and is modified by phosphorylation in a cell-cycle-dependent fashion.  Furthermore, it comigrates with the previously described 30 × 103 Mr metaphase complex which is posttranslationally modified during the first mitotic cell cycle.  Our data suggest that spindlin plays a role in cell-cycle regulation during the transition from gamete to embryo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBqamj85CalbVg90H21EOLACvtfcHk0libH35WzHae0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXhvVWmsbk%253D&md5=12c37613f401dedd7d68597ef3e63b0b</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DOh%26aufirst%3DB.%26aulast%3DHwang%26aufirst%3DS.%2BY.%26aulast%3DSolter%26aufirst%3DD.%26aulast%3DKnowles%26aufirst%3DB.%2BB.%26atitle%3DSpindlin%252C%2520a%2520major%2520maternal%2520transcript%2520expressed%2520in%2520the%2520mouse%2520during%2520the%2520transition%2520from%2520oocyte%2520to%2520embryo%26jtitle%3DDevelopment%26date%3D1997%26volume%3D124%26spage%3D493%26epage%3D503" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Staub, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mennerich, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenthal, A.</span></span> <span> </span><span class="NLM_article-title">The Spin/Ssty repeat: a new motif identified in proteins involved in vertebrate development from gamete to embryo</span>. <i>Genome Biol.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">RESEARCH0003.1</span>, <span class="refDoi"> DOI: 10.1186/gb-2001-3-1-research0003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1186%2Fgb-2001-3-1-research0003" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2001&pages=RESEARCH0003.1&author=E.+Staubauthor=D.+Mennerichauthor=A.+Rosenthal&title=The+Spin%2FSsty+repeat%3A+a+new+motif+identified+in+proteins+involved+in+vertebrate+development+from+gamete+to+embryo&doi=10.1186%2Fgb-2001-3-1-research0003"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1186%2Fgb-2001-3-1-research0003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fgb-2001-3-1-research0003%26sid%3Dliteratum%253Aachs%26aulast%3DStaub%26aufirst%3DE.%26aulast%3DMennerich%26aufirst%3DD.%26aulast%3DRosenthal%26aufirst%3DA.%26atitle%3DThe%2520Spin%252FSsty%2520repeat%253A%2520a%2520new%2520motif%2520identified%2520in%2520proteins%2520involved%2520in%2520vertebrate%2520development%2520from%2520gamete%2520to%2520embryo%26jtitle%3DGenome%2520Biol.%26date%3D2001%26volume%3D3%26spage%3DRESEARCH0003.1%26doi%3D10.1186%2Fgb-2001-3-1-research0003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bae, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Premkumar, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sbardella, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span> <span> </span><span class="NLM_article-title">A transcriptional coregulator, SPIN.DOC, attenuates the coactivator activity of Spindlin1</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>292</i></span>,  <span class="NLM_fpage">20808</span>– <span class="NLM_lpage">20817</span>, <span class="refDoi"> DOI: 10.1074/jbc.m117.814913</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1074%2Fjbc.m117.814913" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=29061846" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitVejsbrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=292&publication_year=2017&pages=20808-20817&author=N.+Baeauthor=M.+Gaoauthor=X.+Liauthor=T.+Premkumarauthor=G.+Sbardellaauthor=J.+Chenauthor=M.+T.+Bedford&title=A+transcriptional+coregulator%2C+SPIN.DOC%2C+attenuates+the+coactivator+activity+of+Spindlin1&doi=10.1074%2Fjbc.m117.814913"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">A transcriptional coregulator, SPIN·DOC, attenuates the coactivator activity of Spindlin1</span></div><div class="casAuthors">Bae, Narkhyun; Gao, Min; Li, Xu; Premkumar, Tolkappiyan; Sbardella, Gianluca; Chen, Junjie; Bedford, Mark T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">292</span>
        (<span class="NLM_cas:issue">51</span>),
    <span class="NLM_cas:pages">20808-20817</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Spindlin1 (SPIN1) is a transcriptional coactivator with crit. functions in embryonic development and emerging roles in cancer.  SPIN1 harbors three Tudor domains, two of which engage the tail of histone H3 by reading the H3-Lys-4 trimethylation and H3-Arg-8 asym. dimethylation marks.  To gain mechanistic insight into how SPIN1 functions as a transcriptional coactivator, here we purified its interacting proteins.  We identified an uncharacterized protein (C11orf84), which we renamed SPIN1 docking protein (SPIN·DOC), that directly binds SPIN1 and strongly disrupts its histone methylation reading ability, causing it to disassoc. from chromatin.  The Spindlin family of coactivators has five related members (SPIN1, 2A, 2B, 3, and 4), and we found that all of them bind SPIN·DOC.  It has been reported previously that SPIN1 regulates gene expression in the Wnt signaling pathway by directly interacting with transcription factor 4 (TCF4).  We obsd. here that SPIN·DOC assocs. with TCF4 in a SPIN1-dependent manner and dampens SPIN1 coactivator activity in TOPflash reporter assays.  Furthermore, knockdown and overexpression expts. indicated that SPIN·DOC represses the expression of a no. of SPIN1-regulated genes, including those encoding rRNA and the cytokine IL1B.  In conclusion, we have identified SPIN·DOC as a transcriptional repressor that binds SPIN1 and masks its ability to engage the H3-Lys-4 trimethylation activation mark.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_Fa2GbBnnLrVg90H21EOLACvtfcHk0libH35WzHae0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitVejsbrE&md5=69c4c937798fe7e27f6234156291217f</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1074%2Fjbc.m117.814913&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.m117.814913%26sid%3Dliteratum%253Aachs%26aulast%3DBae%26aufirst%3DN.%26aulast%3DGao%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DPremkumar%26aufirst%3DT.%26aulast%3DSbardella%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DA%2520transcriptional%2520coregulator%252C%2520SPIN.DOC%252C%2520attenuates%2520the%2520coactivator%2520activity%2520of%2520Spindlin1%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2017%26volume%3D292%26spage%3D20808%26epage%3D20817%26doi%3D10.1074%2Fjbc.m117.814913" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Machleidt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodroofe, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwinn, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Méndez, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robers, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otto, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirkland, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, K. V.</span></span> <span> </span><span class="NLM_article-title">NanoBRET—a novel BRET platform for the analysis of protein–protein interactions</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1797</span>– <span class="NLM_lpage">1804</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00143</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00143" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC2MXovFKkt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1797-1804&author=T.+Machleidtauthor=C.+C.+Woodroofeauthor=M.+K.+Schwinnauthor=J.+M%C3%A9ndezauthor=M.+B.+Robersauthor=K.+Zimmermanauthor=P.+Ottoauthor=D.+L.+Danielsauthor=T.+A.+Kirklandauthor=K.+V.+Wood&title=NanoBRET%E2%80%94a+novel+BRET+platform+for+the+analysis+of+protein%E2%80%93protein+interactions&doi=10.1021%2Facschembio.5b00143"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">NanoBRET-A Novel BRET Platform for the Analysis of Protein-Protein Interactions</span></div><div class="casAuthors">Machleidt, Thomas; Woodroofe, Carolyn C.; Schwinn, Marie K.; Mendez, Jacqui; Robers, Matthew B.; Zimmerman, Kris; Otto, Paul; Daniels, Danette L.; Kirkland, Thomas A.; Wood, Keith V.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1797-1804</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Dynamic interactions between proteins comprise a key mechanism for temporal control of cellular function and thus hold promise for development of novel drug therapies.  It remains tech. challenging, however, to quant. characterize these interactions within the biol. relevant context of living cells.  Although, bioluminescence resonance energy transfer (BRET) has often been used for this purpose, its general applicability has been hindered by limited sensitivity and dynamic range.  We have addressed this by combining an extremely bright luciferase (Nanoluc) with a means for tagging intracellular proteins with a long-wavelength fluorophore (HaloTag).  The small size (19 kDa), high emission intensity, and relatively narrow spectrum (460 nm peak intensity) make Nanoluc luciferase well suited as an energy donor.  By selecting an efficient red-emitting fluorophore (635 nm peak intensity) for attachment onto the HaloTag, an overall spectral sepn. exceeding 175 nm was achieved.  This combination of greater light intensity with improved spectral resoln. results in substantially increased detection sensitivity and dynamic range over current BRET technologies.  Enhanced performance is demonstrated using several established model systems, as well as the ability to image BRET in individual cells.  The capabilities are further exhibited in a novel assay developed for analyzing the interactions of bromodomain proteins with chromatin in living cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6-h9HlReNabVg90H21EOLACvtfcHk0libH35WzHae0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXovFKkt7c%253D&md5=9f5f773b6cd14c40074a2ffbd6472008</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00143&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00143%26sid%3Dliteratum%253Aachs%26aulast%3DMachleidt%26aufirst%3DT.%26aulast%3DWoodroofe%26aufirst%3DC.%2BC.%26aulast%3DSchwinn%26aufirst%3DM.%2BK.%26aulast%3DM%25C3%25A9ndez%26aufirst%3DJ.%26aulast%3DRobers%26aufirst%3DM.%2BB.%26aulast%3DZimmerman%26aufirst%3DK.%26aulast%3DOtto%26aufirst%3DP.%26aulast%3DDaniels%26aufirst%3DD.%2BL.%26aulast%3DKirkland%26aufirst%3DT.%2BA.%26aulast%3DWood%26aufirst%3DK.%2BV.%26atitle%3DNanoBRET%25E2%2580%2594a%2520novel%2520BRET%2520platform%2520for%2520the%2520analysis%2520of%2520protein%25E2%2580%2593protein%2520interactions%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D1797%26epage%3D1804%26doi%3D10.1021%2Facschembio.5b00143" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Green, K.</span></span> <span> </span><span class="NLM_article-title">Improved metalation of 2,4,6-tribromoanisole - sythesis of 2-methoxyresorcinol</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">4325</span>– <span class="NLM_lpage">4326</span>, <span class="refDoi"> DOI: 10.1021/jo00013a052</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo00013a052" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADyaK3MXktVyiur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=1991&pages=4325-4326&author=K.+Green&title=Improved+metalation+of+2%2C4%2C6-tribromoanisole+-+sythesis+of+2-methoxyresorcinol&doi=10.1021%2Fjo00013a052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Improved metalation of 2,4,6-tribromoanisole:  synthesis of 2-methoxyresorcinol</span></div><div class="casAuthors">Green, Kenneth</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">4325-6</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    </div><div class="casAbstract">The low temp., heterogeneous metalation of 2,4,6-tribromoanisole in pentane using butyllithium is described.  This occurs without the aid of any chelating agent such as N,N,N',N'-tetramethylethylenediamine.  By performing the metalation in a heterogeneous manner, the products obtained from a protocol of (a) trimethylborate quench (b) peracetic acid oxidn. are significantly cleaner than when the same sequence of reactions occurs under homogeneous conditions.  Variation of the n-butyllithium stoichiometry results in products obtained effectively from either the mono- or dianions of the tribromide.  While examples resulting solely from reaction with trimethylborate are shown, it appears that the method may be general for other electrophiles.  A concise and efficient synthesis of 2-methoxyresorcinol is also demonstrated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP9S_3K5bf3bVg90H21EOLACvtfcHk0ljjI4U73VTmBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXktVyiur4%253D&md5=ebb0e4b6da8807507036e7c1e1196e9b</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fjo00013a052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo00013a052%26sid%3Dliteratum%253Aachs%26aulast%3DGreen%26aufirst%3DK.%26atitle%3DImproved%2520metalation%2520of%25202%252C4%252C6-tribromoanisole%2520-%2520sythesis%2520of%25202-methoxyresorcinol%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1991%26volume%3D56%26spage%3D4325%26epage%3D4326%26doi%3D10.1021%2Fjo00013a052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P.</span></span> <span> </span><span class="NLM_article-title">Scaling and assessment of data quality</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">72</span>– <span class="NLM_lpage">82</span>, <span class="refDoi"> DOI: 10.1107/s0907444905036693</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1107%2Fs0907444905036693" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=16369096" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlant7jM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2006&pages=72-82&author=P.+Evans&title=Scaling+and+assessment+of+data+quality&doi=10.1107%2Fs0907444905036693"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Scaling and assessment of data quality</span></div><div class="casAuthors">Evans, Philip</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">D62</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">72-82</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">The various phys. factors affecting measured diffraction intensities are discussed, as are the scaling models which may be used to put the data on a consistent scale.  After scaling, the intensities can be analyzed to set the real resoln. of the data set, to detect bad regions (e.g. bad images), to analyze radiation damage and to assess the overall quality of the data set.  The significance of any anomalous signal may be assessed by probability and correlation anal.  The algorithms used by the CCP4 scaling program SCALA are described.  A requirement for the scaling and merging of intensities is knowledge of the Laue group and point-group symmetries: the possible symmetry of the diffraction pattern may be detd. from scores such as correlation coeffs. between observations which might be symmetry-related.  These scoring functions are implemented in a new program POINTLESS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGravLSLU62RhLVg90H21EOLACvtfcHk0ljjI4U73VTmBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlant7jM&md5=293d3876e534c0c57813990515bb3c76</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1107%2Fs0907444905036693&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0907444905036693%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%26atitle%3DScaling%2520and%2520assessment%2520of%2520data%2520quality%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2006%26volume%3D62%26spage%3D72%26epage%3D82%26doi%3D10.1107%2Fs0907444905036693" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Axford, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foadi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, N.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choudhury, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwata, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beis, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alguel, Y.</span></span> <span> </span><span class="NLM_article-title">Structure determination of an integral membrane protein at room temperature from crystals in situ</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">1228</span>– <span class="NLM_lpage">1237</span>, <span class="refDoi"> DOI: 10.1107/s139900471500423x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1107%2Fs139900471500423x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=26057664" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVKisLbI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2015&pages=1228-1237&author=D.+Axfordauthor=J.+Foadiauthor=N.-J.+Huauthor=H.+G.+Choudhuryauthor=S.+Iwataauthor=K.+Beisauthor=G.+Evansauthor=Y.+Alguel&title=Structure+determination+of+an+integral+membrane+protein+at+room+temperature+from+crystals+in+situ&doi=10.1107%2Fs139900471500423x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Structure determination of an integral membrane protein at room temperature from crystals in situ</span></div><div class="casAuthors">Axford, Danny; Foadi, James; Hu, Nien-Jen; Choudhury, Hassanul Ghani; Iwata, So; Beis, Konstantinos; Evans, Gwyndaf; Alguel, Yilmaz</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1228-1237</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">1399-0047</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The structure detn. of an integral membrane protein using synchrotron X-ray diffraction data collected at room temp. directly in vapor-diffusion crystn. plates (in situ) is demonstrated.  Exposing the crystals in situ eliminates manual sample handling and, since it is performed at room temp., removes the complication of cryoprotection and potential structural anomalies induced by sample cryocooling.  Essential to the method is the ability to limit radiation damage by recording a small amt. of data per sample from many samples and subsequently assembling the resulting data sets using specialized software.  The validity of this procedure is established by the structure detn. of Haemophilus influenza TehA at 2.3 Å resoln.  The method presented offers an effective protocol for the fast and efficient detn. of membrane-protein structures at room temp. using third-generation synchrotron beamlines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm6BVlKwCapbVg90H21EOLACvtfcHk0ljjI4U73VTmBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVKisLbI&md5=bb8215e22dc3d619ebabcbdde703bfd6</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1107%2Fs139900471500423x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs139900471500423x%26sid%3Dliteratum%253Aachs%26aulast%3DAxford%26aufirst%3DD.%26aulast%3DFoadi%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DN.-J.%26aulast%3DChoudhury%26aufirst%3DH.%2BG.%26aulast%3DIwata%26aufirst%3DS.%26aulast%3DBeis%26aufirst%3DK.%26aulast%3DEvans%26aufirst%3DG.%26aulast%3DAlguel%26aufirst%3DY.%26atitle%3DStructure%2520determination%2520of%2520an%2520integral%2520membrane%2520protein%2520at%2520room%2520temperature%2520from%2520crystals%2520in%2520situ%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2015%26volume%3D71%26spage%3D1228%26epage%3D1237%26doi%3D10.1107%2Fs139900471500423x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballard, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keegan, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krissinel, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNicholas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potterton, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, K. S.</span></span> <span> </span><span class="NLM_article-title">Overview of the CCP4 suite and current developments</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1107/s0907444910045749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=235-242&author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=R.+M.+Keeganauthor=E.+B.+Krissinelauthor=A.+G.+W.+Leslieauthor=A.+McCoyauthor=S.+J.+McNicholasauthor=G.+N.+Murshudovauthor=N.+S.+Pannuauthor=E.+A.+Pottertonauthor=H.+R.+Powellauthor=R.+J.+Readauthor=A.+Vaginauthor=K.+S.+Wilson&title=Overview+of+the+CCP4+suite+and+current+developments&doi=10.1107%2Fs0907444910045749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0lhCAEzNzQ8tFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DKeegan%26aufirst%3DR.%2BM.%26aulast%3DKrissinel%26aufirst%3DE.%2BB.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%26aulast%3DMcNicholas%26aufirst%3DS.%2BJ.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DPotterton%26aufirst%3DE.%2BA.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520the%2520CCP4%2520suite%2520and%2520current%2520developments%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D235%26epage%3D242%26doi%3D10.1107%2Fs0907444910045749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G.</span></span> <span> </span><span class="NLM_article-title">xia2: an expert system for macromolecular crystallography data reduction</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">186</span>– <span class="NLM_lpage">190</span>, <span class="refDoi"> DOI: 10.1107/s0021889809045701</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1107%2FS0021889809045701" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFalsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2010&pages=186-190&author=G.+Winter&title=xia2%3A+an+expert+system+for+macromolecular+crystallography+data+reduction&doi=10.1107%2Fs0021889809045701"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">xia2: An expert system for macromolecular crystallography data reduction</span></div><div class="casAuthors">Winter, G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">186-190, S186/1-S186/5</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">An expert system for macromol. crystallog. data redn. is presented, which builds on existing software to automate the complete data redn. process from images to merged structure factor amplitudes.  This can automatically identify multi-wedge, multi-pass and multiwavelength data sets and includes explicit procedures to test for crystallog. special cases.  With the push towards high-throughput crystallog. at synchrotron beamlines and automation of structure soln., the ability to reduce data with no user input fills an important gap in the pipeline.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOi3u0ydXbSrVg90H21EOLACvtfcHk0lhCAEzNzQ8tFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFalsrk%253D&md5=27771caa813f3e828abdf2a331ae55e3</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1107%2FS0021889809045701&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889809045701%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DG.%26atitle%3Dxia2%253A%2520an%2520expert%2520system%2520for%2520macromolecular%2520crystallography%2520data%2520reduction%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2010%26volume%3D43%26spage%3D186%26epage%3D190%26doi%3D10.1107%2Fs0021889809045701" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waterman, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parkhurst, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brewster, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gildea, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuentes-Montero, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vollmar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michels-Clark, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauter, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, G.</span></span> <span> </span><span class="NLM_article-title">DIALS: implementation and evaluation of a new integration package</span>. <i>Acta Crystallogr., Sect. D: Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">85</span>– <span class="NLM_lpage">97</span>, <span class="refDoi"> DOI: 10.1107/s2059798317017235</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1107%2FS2059798317017235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC1cXksFarurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2018&pages=85-97&author=G.+Winterauthor=D.+G.+Watermanauthor=J.+M.+Parkhurstauthor=A.+S.+Brewsterauthor=R.+J.+Gildeaauthor=M.+Gerstelauthor=L.+Fuentes-Monteroauthor=M.+Vollmarauthor=T.+Michels-Clarkauthor=I.+D.+Youngauthor=N.+K.+Sauterauthor=G.+Evans&title=DIALS%3A+implementation+and+evaluation+of+a+new+integration+package&doi=10.1107%2Fs2059798317017235"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">DIALS: implementation and evaluation of a new integration package</span></div><div class="casAuthors">Winter, Graeme; Waterman, David G.; Parkhurst, James M.; Brewster, Aaron S.; Gildea, Richard J.; Gerstel, Markus; Fuentes-Montero, Luis; Vollmar, Melanie; Michels-Clark, Tara; Young, Iris D.; Sauter, Nicholas K.; Evans, Gwyndaf</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Structural Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">85-97</span>CODEN:
                <span class="NLM_cas:coden">ACSDAD</span>;
        ISSN:<span class="NLM_cas:issn">2059-7983</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The DIALS project is a collaboration between Diamond Light Source, Lawrence Berkeley National Lab. and CCP4 to develop a new software suite for the anal. of crystallog. X-ray diffraction data, initially encompassing spot finding, indexing, refinement and integration.  The design, core algorithms and structure of the software are introduced, alongside results from the anal. of data from biol. and chem. crystallog. expts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkZqQrzPvxlbVg90H21EOLACvtfcHk0lhCAEzNzQ8tFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXksFarurk%253D&md5=f4572135ac40643815064914216f6d4c</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1107%2FS2059798317017235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS2059798317017235%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DG.%26aulast%3DWaterman%26aufirst%3DD.%2BG.%26aulast%3DParkhurst%26aufirst%3DJ.%2BM.%26aulast%3DBrewster%26aufirst%3DA.%2BS.%26aulast%3DGildea%26aufirst%3DR.%2BJ.%26aulast%3DGerstel%26aufirst%3DM.%26aulast%3DFuentes-Montero%26aufirst%3DL.%26aulast%3DVollmar%26aufirst%3DM.%26aulast%3DMichels-Clark%26aufirst%3DT.%26aulast%3DYoung%26aufirst%3DI.%2BD.%26aulast%3DSauter%26aufirst%3DN.%2BK.%26aulast%3DEvans%26aufirst%3DG.%26atitle%3DDIALS%253A%2520implementation%2520and%2520evaluation%2520of%2520a%2520new%2520integration%2520package%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Struct.%2520Biol.%26date%3D2018%26volume%3D74%26spage%3D85%26epage%3D97%26doi%3D10.1107%2Fs2059798317017235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkóczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1107/s0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.-W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+a+comprehensive+Python-based+system+for+macromolecular+structure+solution&doi=10.1107%2Fs0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0ljvwfcvAzfaXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.-W.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520a%2520comprehensive%2520Python-based%2520system%2520for%2520macromolecular%2520structure%2520solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2Fs0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/s0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2Fs0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0ljvwfcvAzfaXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2Fs0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vargas, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piszczek, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brautigam, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuck, P.</span></span> <span> </span><span class="NLM_article-title">High-precision isothermal titration calorimetry with automated peak-shape analysis</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>84</i></span>,  <span class="NLM_fpage">5066</span>– <span class="NLM_lpage">5073</span>, <span class="refDoi"> DOI: 10.1021/ac3007522</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ac3007522" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC38XmtVGlsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2012&pages=5066-5073&author=S.+Kellerauthor=C.+Vargasauthor=H.+Zhaoauthor=G.+Piszczekauthor=C.+A.+Brautigamauthor=P.+Schuck&title=High-precision+isothermal+titration+calorimetry+with+automated+peak-shape+analysis&doi=10.1021%2Fac3007522"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">High-Precision Isothermal Titration Calorimetry with Automated Peak-Shape Analysis</span></div><div class="casAuthors">Keller, Sandro; Vargas, Carolyn; Zhao, Huaying; Piszczek, Grzegorz; Brautigam, Chad A.; Schuck, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">84</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5066-5073</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Isothermal titrn. calorimetry (ITC) is a powerful classical method that enables researchers in many fields to study the thermodn. of mol. interactions.  Primary ITC data comprise the temporal evolution of differential power reporting the heat of reaction during a series of injections of aliquots of a reactant into a sample cell.  By integration of each injection peak, an isotherm can be constructed of total changes in enthalpy as a function of changes in soln. compn., which is rich in thermodn. information on the reaction.  However, the signals from the injection peaks are superimposed by the stochastically varying time-course of the instrumental baseline power, limiting the precision of ITC isotherms.  The authors describe a method for automated peak assignment based on peak-shape anal. via singular value decompn. in combination with detailed least-squares modeling of local pre- and post- injection baselines.  This approach can effectively filter out contributions of short-term noise and adventitious events in the power trace.  This method also provides, for the first time, statistical error ests. for the individual isotherm data points.  In turn, this results in improved detection limits for high-affinity or low-enthalpy binding reactions and significantly higher precision of the derived thermodn. parameters.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJcKwO_fHPFbVg90H21EOLACvtfcHk0ljvwfcvAzfaXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmtVGlsbs%253D&md5=bb157e956fa9cef94a88974d733cad1b</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Fac3007522&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac3007522%26sid%3Dliteratum%253Aachs%26aulast%3DKeller%26aufirst%3DS.%26aulast%3DVargas%26aufirst%3DC.%26aulast%3DZhao%26aufirst%3DH.%26aulast%3DPiszczek%26aufirst%3DG.%26aulast%3DBrautigam%26aufirst%3DC.%2BA.%26aulast%3DSchuck%26aufirst%3DP.%26atitle%3DHigh-precision%2520isothermal%2520titration%2520calorimetry%2520with%2520automated%2520peak-shape%2520analysis%26jtitle%3DAnal.%2520Chem.%26date%3D2012%26volume%3D84%26spage%3D5066%26epage%3D5073%26doi%3D10.1021%2Fac3007522" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Langmead, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trapnell, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pop, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salzberg, S. L.</span></span> <span> </span><span class="NLM_article-title">Ultrafast and memory-efficient alignment of short DNA sequences to the human genome</span>. <i>Genome Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">R25</span>, <span class="refDoi"> DOI: 10.1186/gb-2009-10-3-r25</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1186%2Fgb-2009-10-3-r25" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=19261174" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A280%3ADC%252BD1MzotVGnsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2009&pages=R25&author=B.+Langmeadauthor=C.+Trapnellauthor=M.+Popauthor=S.+L.+Salzberg&title=Ultrafast+and+memory-efficient+alignment+of+short+DNA+sequences+to+the+human+genome&doi=10.1186%2Fgb-2009-10-3-r25"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Ultrafast and memory-efficient alignment of short DNA sequences to the human genome</span></div><div class="casAuthors">Langmead Ben; Trapnell Cole; Pop Mihai; Salzberg Steven L</div><div class="citationInfo"><span class="NLM_cas:title">Genome biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">R25</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Bowtie is an ultrafast, memory-efficient alignment program for aligning short DNA sequence reads to large genomes.  For the human genome, Burrows-Wheeler indexing allows Bowtie to align more than 25 million reads per CPU hour with a memory footprint of approximately 1.3 gigabytes.  Bowtie extends previous Burrows-Wheeler techniques with a novel quality-aware backtracking algorithm that permits mismatches.  Multiple processor cores can be used simultaneously to achieve even greater alignment speeds.  Bowtie is open source (http://bowtie.cbcb.umd.edu).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSod2io_r8eDEloaUqgkFHkfW6udTcc2eZ2vgt5TzxE67ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1MzotVGnsQ%253D%253D&md5=710abfa0067372cd5f72a3cbc0fe4271</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1186%2Fgb-2009-10-3-r25&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fgb-2009-10-3-r25%26sid%3Dliteratum%253Aachs%26aulast%3DLangmead%26aufirst%3DB.%26aulast%3DTrapnell%26aufirst%3DC.%26aulast%3DPop%26aufirst%3DM.%26aulast%3DSalzberg%26aufirst%3DS.%2BL.%26atitle%3DUltrafast%2520and%2520memory-efficient%2520alignment%2520of%2520short%2520DNA%2520sequences%2520to%2520the%2520human%2520genome%26jtitle%3DGenome%2520Biol.%26date%3D2009%26volume%3D10%26spage%3DR25%26doi%3D10.1186%2Fgb-2009-10-3-r25" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eeckhoute, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernstein, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussbaum, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myers, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X. S.</span></span> <span> </span><span class="NLM_article-title">Model-based analysis of ChIP-Seq (MACS)</span>. <i>Genome Biol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">R137</span>, <span class="refDoi"> DOI: 10.1186/gb-2008-9-9-r137</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1186%2Fgb-2008-9-9-r137" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=18798982" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2008&pages=R137&author=Y.+Zhangauthor=T.+Liuauthor=C.+A.+Meyerauthor=J.+Eeckhouteauthor=D.+S.+Johnsonauthor=B.+E.+Bernsteinauthor=C.+Nussbaumauthor=R.+M.+Myersauthor=M.+Brownauthor=W.+Liauthor=X.+S.+Liu&title=Model-based+analysis+of+ChIP-Seq+%28MACS%29&doi=10.1186%2Fgb-2008-9-9-r137"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1186%2Fgb-2008-9-9-r137&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fgb-2008-9-9-r137%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DT.%26aulast%3DMeyer%26aufirst%3DC.%2BA.%26aulast%3DEeckhoute%26aufirst%3DJ.%26aulast%3DJohnson%26aufirst%3DD.%2BS.%26aulast%3DBernstein%26aufirst%3DB.%2BE.%26aulast%3DNussbaum%26aufirst%3DC.%26aulast%3DMyers%26aufirst%3DR.%2BM.%26aulast%3DBrown%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DX.%2BS.%26atitle%3DModel-based%2520analysis%2520of%2520ChIP-Seq%2520%2528MACS%2529%26jtitle%3DGenome%2520Biol.%26date%3D2008%26volume%3D9%26spage%3DR137%26doi%3D10.1186%2Fgb-2008-9-9-r137" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scheer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ackloo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medina, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Northrop, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLeod, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zepeda-Velazquez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Carvalho, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaph, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C.</span></span> <span> </span><span class="NLM_article-title">A chemical biology toolbox to study protein methyltransferases and epigenetic signaling</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">19</span>, <span class="refDoi"> DOI: 10.1038/s41467-018-07905-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1038%2Fs41467-018-07905-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=30604761" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC1MXmvVCmtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=19&author=S.+Scheerauthor=S.+Acklooauthor=T.+Medinaauthor=M.+Schapiraauthor=F.+Liauthor=J.+Wardauthor=A.+Lewisauthor=J.+Northropauthor=P.+Richardsonauthor=H.+Kaniskanauthor=Y.+Shenauthor=J.+Liuauthor=D.+Smilauthor=D.+McLeodauthor=C.+Zepeda-Velazquezauthor=M.+Luoauthor=J.+Jinauthor=D.+Barsyte-Lovejoyauthor=K.+Huberauthor=D.+De+Carvalhoauthor=M.+Vedadiauthor=C.+Zaphauthor=P.+Brownauthor=C.+Arrowsmith&title=A+chemical+biology+toolbox+to+study+protein+methyltransferases+and+epigenetic+signaling&doi=10.1038%2Fs41467-018-07905-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">A chemical biology toolbox to study protein methyltransferases and epigenetic signaling</span></div><div class="casAuthors">Scheer, Sebastian; Ackloo, Suzanne; Medina, Tiago S.; Schapira, Matthieu; Li, Fengling; Ward, Jennifer A.; Lewis, Andrew M.; Northrop, Jeffrey P.; Richardson, Paul L.; Kaniskan, H. Umit; Shen, Yudao; Liu, Jing; Smil, David; McLeod, David; Zepeda-Velazquez, Carlos A.; Luo, Minkui; Jin, Jian; Barsyte-Lovejoy, Dalia; Huber, Kilian V. M.; De Carvalho, Daniel D.; Vedadi, Masoud; Zaph, Colby; Brown, Peter J.; Arrowsmith, Cheryl H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">19</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Protein methyltransferases (PMTs) comprise a major class of epigenetic regulatory enzymes with therapeutic relevance.  Here we present a collection of chem. probes and assocd. reagents and data to elucidate the function of human and murine PMTs in cellular studies.  Our collection provides inhibitors and antagonists that together modulate most of the key regulatory methylation marks on histones H3 and H4, providing an important resource for modulating cellular epigenomes.  We describe a comprehensive and comparative characterization of the probe collection with respect to their potency, selectivity, and mode of inhibition.  We demonstrate the utility of this collection in CD4+ T cell differentiation assays revealing the potential of individual probes to alter multiple T cell subpopulations which may have implications for T cell-mediated processes such as inflammation and immuno-oncol.  In particular, we demonstrate a role for DOT1L in limiting Th1 cell differentiation and maintaining lineage integrity.  This chem. probe collection and assocd. data form a resource for the study of methylation-mediated signaling in epigenetics, inflammation and beyond.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopA5rgeiWDvrVg90H21EOLACvtfcHk0liJOkLhQMWwvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXmvVCmtro%253D&md5=008fec084426171f5c91657e54b73900</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1038%2Fs41467-018-07905-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-018-07905-4%26sid%3Dliteratum%253Aachs%26aulast%3DScheer%26aufirst%3DS.%26aulast%3DAckloo%26aufirst%3DS.%26aulast%3DMedina%26aufirst%3DT.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DWard%26aufirst%3DJ.%26aulast%3DLewis%26aufirst%3DA.%26aulast%3DNorthrop%26aufirst%3DJ.%26aulast%3DRichardson%26aufirst%3DP.%26aulast%3DKaniskan%26aufirst%3DH.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DMcLeod%26aufirst%3DD.%26aulast%3DZepeda-Velazquez%26aufirst%3DC.%26aulast%3DLuo%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DHuber%26aufirst%3DK.%26aulast%3DDe%2BCarvalho%26aufirst%3DD.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DZaph%26aufirst%3DC.%26aulast%3DBrown%26aufirst%3DP.%26aulast%3DArrowsmith%26aufirst%3DC.%26atitle%3DA%2520chemical%2520biology%2520toolbox%2520to%2520study%2520protein%2520methyltransferases%2520and%2520epigenetic%2520signaling%26jtitle%3DNat.%2520Commun.%26date%3D2019%26volume%3D10%26spage%3D19%26doi%3D10.1038%2Fs41467-018-07905-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sciaky, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hulbert, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Everitt, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.-h.</span></span> <span> </span><span class="NLM_article-title">Targeting the histone methyltransferase G9a activates imprinted genes and improves survival of a mouse model of Prader-Willi syndrome</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">222</span>, <span class="refDoi"> DOI: 10.1038/nm.4257</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=10.1038%2Fnm.4257" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=28024084" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFWntbnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2017&pages=213-222&author=Y.+Kimauthor=H.-M.+Leeauthor=Y.+Xiongauthor=N.+Sciakyauthor=S.+W.+Hulbertauthor=X.+Caoauthor=J.+I.+Everittauthor=J.+Jinauthor=B.+L.+Rothauthor=Y.-h.+Jiang&title=Targeting+the+histone+methyltransferase+G9a+activates+imprinted+genes+and+improves+survival+of+a+mouse+model+of+Prader-Willi+syndrome&doi=10.1038%2Fnm.4257"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the histone methyltransferase G9a activates imprinted genes and improves survival of a mouse model of Prader-Willi syndrome</span></div><div class="casAuthors">Kim, Yuna; Lee, Hyeong-Min; Xiong, Yan; Sciaky, Noah; Hulbert, Samuel W.; Cao, Xinyu; Everitt, Jeffrey I.; Jin, Jian; Roth, Bryan L.; Jiang, Yong-hui</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-222</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Prader-Willi syndrome (PWS) is an imprinting disorder caused by a deficiency of paternally expressed gene(s) in the 15q11-q13 chromosomal region.  The regulation of imprinted gene expression in this region is coordinated by an imprinting center (PWS-IC).  In individuals with PWS, genes responsible for PWS on the maternal chromosome are present, but repressed epigenetically, which provides an opportunity for the use of epigenetic therapy to restore expression from the maternal copies of PWS-assocd. genes.  Through a high-content screen (HCS) of >9,000 small mols., we discovered that UNC0638 and UNC0642-two selective inhibitors of euchromatic histone lysine N-methyltransferase-2 (EHMT2, also known as G9a)-activated the maternal (m) copy of candidate genes underlying PWS, including the SnoRNA cluster SNORD116, in cells from humans with PWS and also from a mouse model of PWS carrying a paternal (p) deletion from small nuclear ribonucleoprotein N (Snrpn (S)) to ubiquitin protein ligase E3A (Ube3a (U)) (mouse model referred to hereafter as m+/pΔS-U).  Both UNC0642 and UNC0638 caused a selective redn. of the dimethylation of histone H3 lysine 9 (H3K9me2) at PWS-IC, without changing DNA methylation, when analyzed by bisulfite genomic sequencing.  This indicates that histone modification is essential for the imprinting of candidate genes underlying PWS.  UNC0642 displayed therapeutic effects in the PWS mouse model by improving the survival and the growth of m+/pΔS-U newborn pups.  This study provides the first proof of principle for an epigenetics-based therapy for PWS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQUdUqWlf7RbVg90H21EOLACvtfcHk0liJOkLhQMWwvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFWntbnJ&md5=78ccbdde43e3fe79ea72ffa385893d58</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1038%2Fnm.4257&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.4257%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DY.%26aulast%3DLee%26aufirst%3DH.-M.%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DSciaky%26aufirst%3DN.%26aulast%3DHulbert%26aufirst%3DS.%2BW.%26aulast%3DCao%26aufirst%3DX.%26aulast%3DEveritt%26aufirst%3DJ.%2BI.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DJiang%26aufirst%3DY.-h.%26atitle%3DTargeting%2520the%2520histone%2520methyltransferase%2520G9a%2520activates%2520imprinted%2520genes%2520and%2520improves%2520survival%2520of%2520a%2520mouse%2520model%2520of%2520Prader-Willi%2520syndrome%26jtitle%3DNat.%2520Med.%26date%3D2017%26volume%3D23%26spage%3D213%26epage%3D222%26doi%3D10.1038%2Fnm.4257" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3RJW" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3RJW','PDB','3RJW'); return false;">PDB: 3RJW</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4H75" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4H75','PDB','4H75'); return false;">PDB: 4H75</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QPL" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QPL','PDB','6QPL'); return false;">PDB: 6QPL</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6I8L" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6I8L','PDB','6I8L'); return false;">PDB: 6I8L</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i41"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00522">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_13035"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.9b00522">10.1021/acs.jmedchem.9b00522</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Concentration response curves of A366 and compound <b>1</b> in the SPIN1 AlphaLISA and FP biochemical assays; docking model of compound <b>1</b> in the complex with SPIN1; binding of compound <b>2</b> to GLP by ITC; binding of compound <b>3</b> to SPIN2B, SPIN3, and SPIN4 by ITC; <sup>1</sup>H NMR and <sup>13</sup>C NMR spectra of compounds <b>3</b> and <b>4</b>; raw data of the enzymatic inhibition assay against 33 MTs and 5 acetyltransferases; raw data results of the <i>T</i><sub>m</sub> shift assay against 20 methyllysine, methylarginine, and acetyllysine reader proteins; crystallography data and refinement statistics; and description of cell lines for cytotoxicity assays (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00522/suppl_file/jm9b00522_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00522/suppl_file/jm9b00522_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00522/suppl_file/jm9b00522_si_001.pdf">jm9b00522_si_001.pdf (2.75 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00522/suppl_file/jm9b00522_si_002.csv">jm9b00522_si_002.csv (0.6 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The structure of SPIN1 in the complex with compound <b>3</b> has been deposited under PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QPL">6QPL</a>. The authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b00522&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b00522%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-20%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b00522" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                2MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679935427ca63be6","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
