
<html lang="en"     class="pb-page"  data-request-id="939df1e6-9de2-4070-bd82-4375f2d0aa63"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2021.64.issue-12;pageGroup:string:Publication Pages;article:article:10.1021/acs.jmedchem.1c00430;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Discovery of an H3K36me3-Derived Peptidomimetic Ligand with Enhanced Affinity for Plant Homeodomain Finger Protein 1 (PHF1)" /></meta><meta name="dc.Creator" content="Isabelle A.  Engelberg" /></meta><meta name="dc.Creator" content="Jiuyang  Liu" /></meta><meta name="dc.Creator" content="Jacqueline L.  Norris-Drouin" /></meta><meta name="dc.Creator" content="Stephanie H.  Cholensky" /></meta><meta name="dc.Creator" content="Samantha A.  Ottavi" /></meta><meta name="dc.Creator" content="Stephen V.  Frye" /></meta><meta name="dc.Creator" content="Tatiana G.  Kutateladze" /></meta><meta name="dc.Creator" content="Lindsey I.  James" /></meta><meta name="dc.Description" content="Plant homeodomain finger protein 1 (PHF1) is an accessory component of the gene silencing complex polycomb repressive complex 2 and recognizes the active chromatin mark, trimethylated lysine 36 of ..." /></meta><meta name="Description" content="Plant homeodomain finger protein 1 (PHF1) is an accessory component of the gene silencing complex polycomb repressive complex 2 and recognizes the active chromatin mark, trimethylated lysine 36 of ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="May 17, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.1c00430" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00430" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.1c00430" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00430" /></link>
        
    
    

<title>Discovery of an H3K36me3-Derived Peptidomimetic Ligand with Enhanced Affinity for Plant Homeodomain Finger Protein 1 (PHF1) | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.1c00430" /></meta><meta property="og:title" content="Discovery of an H3K36me3-Derived Peptidomimetic Ligand with Enhanced Affinity for Plant Homeodomain Finger Protein 1 (PHF1)" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00430/20210616/images/large/jm1c00430_0006.jpeg" /></meta><meta property="og:description" content="Plant homeodomain finger protein 1 (PHF1) is an accessory component of the gene silencing complex polycomb repressive complex 2 and recognizes the active chromatin mark, trimethylated lysine 36 of histone H3 (H3K36me3). In addition to its role in transcriptional regulation, PHF1 has been implicated as a driver of endometrial stromal sarcoma and fibromyxoid tumors. We report the discovery and characterization of UNC6641, a peptidomimetic antagonist of the PHF1 Tudor domain which was optimized through in silico modeling and incorporation of non-natural amino acids. UNC6641 binds the PHF1 Tudor domain with a Kd value of 0.96 ± 0.03 μM while also binding the related protein PHF19 with similar potency. A crystal structure of PHF1 in complex with UNC6641, along with NMR and site-directed mutagenesis data, provided insight into the binding mechanism and requirements for binding. Additionally, UNC6641 enabled the development of a high-throughput assay to identify small molecule binders of PHF1." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.1c00430"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00430">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.1c00430&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.1c00430&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.1c00430&amp;href=/doi/10.1021/acs.jmedchem.1c00430" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 12</span><span class="cit-fg-pageRange">, 8510-8522</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/12" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.1c00420" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.1c00445" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of an H3K36me3-Derived Peptidomimetic Ligand with Enhanced Affinity for Plant Homeodomain Finger Protein 1 (PHF1)</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Isabelle A. Engelberg</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Isabelle A. Engelberg</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Center for Integrative Chemical Biology and Drug Discovery, Division of Chemical Biology and Medicinal Chemistry, UNC Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Isabelle+A.++Engelberg">Isabelle A. Engelberg</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jiuyang Liu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jiuyang Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmacology, University of Colorado School of Medicine, Aurora, Colorado 80045, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jiuyang++Liu">Jiuyang Liu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jacqueline L. Norris-Drouin</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jacqueline L. Norris-Drouin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Center for Integrative Chemical Biology and Drug Discovery, Division of Chemical Biology and Medicinal Chemistry, UNC Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jacqueline+L.++Norris-Drouin">Jacqueline L. Norris-Drouin</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Stephanie H. Cholensky</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Stephanie H. Cholensky</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Center for Integrative Chemical Biology and Drug Discovery, Division of Chemical Biology and Medicinal Chemistry, UNC Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Stephanie+H.++Cholensky">Stephanie H. Cholensky</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Samantha A. Ottavi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Samantha A. Ottavi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Center for Integrative Chemical Biology and Drug Discovery, Division of Chemical Biology and Medicinal Chemistry, UNC Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Samantha+A.++Ottavi">Samantha A. Ottavi</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Stephen V. Frye</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Stephen V. Frye</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Center for Integrative Chemical Biology and Drug Discovery, Division of Chemical Biology and Medicinal Chemistry, UNC Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Stephen+V.++Frye">Stephen V. Frye</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-2415-2215" title="Orcid link">https://orcid.org/0000-0002-2415-2215</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Tatiana G. Kutateladze</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Tatiana G. Kutateladze</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmacology, University of Colorado School of Medicine, Aurora, Colorado 80045, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#dbafbaafb2bab5baf5b0aeafbaafbeb7babfa1be9bb8aebab5a8b8b3aeafa1f5bebfae"><span class="__cf_email__" data-cfemail="d4a0b5a0bdb5bab5fabfa1a0b5a0b1b8b5b0aeb194b7a1b5baa7b7bca1a0aefab1b0a1">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Tatiana+G.++Kutateladze">Tatiana G. Kutateladze</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0001-7375-6990" title="Orcid link">https://orcid.org/0000-0001-7375-6990</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Lindsey I. James</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lindsey I. James</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Center for Integrative Chemical Biology and Drug Discovery, Division of Chemical Biology and Medicinal Chemistry, UNC Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#2a43444d4f58474b446a4f474b4346045f4449044f4e5f"><span class="__cf_email__" data-cfemail="7f1611181a0d121e113f1a121e1613510a111c511a1b0a">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lindsey+I.++James">Lindsey I. James</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-6034-7116" title="Orcid link">https://orcid.org/0000-0002-6034-7116</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00430&amp;href=/doi/10.1021%2Facs.jmedchem.1c00430" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 12</span><span class="cit-pageRange">, 8510–8522</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">May 17, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>9 March 2021</li><li><span class="item_label"><b>Published</b> online</span>17 May 2021</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 24 June 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.1c00430" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00430</a></div><div class="article_header-article-copyright"><strong>Copyright © 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D8510%26pageCount%3D13%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DIsabelle%2BA.%2BEngelberg%252C%2BJiuyang%2BLiu%252C%2BJacqueline%2BL.%2BNorris-Drouin%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D64%26issueNum%3D12%26contentID%3Dacs.jmedchem.1c00430%26title%3DDiscovery%2Bof%2Ban%2BH3K36me3-Derived%2BPeptidomimetic%2BLigand%2Bwith%2BEnhanced%2BAffinity%2Bfor%2BPlant%2BHomeodomain%2BFinger%2BProtein%2B1%2B%2528PHF1%2529%26numPages%3D13%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D8522%26publicationDate%3DJune%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.1c00430"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">434</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.1c00430" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of an H3K36me3-Derived Peptidomimetic Ligand with Enhanced Affinity for Plant Homeodomain Finger Protein 1 (PHF1)&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Isabelle&quot;,&quot;last_name&quot;:&quot;A. Engelberg&quot;},{&quot;first_name&quot;:&quot;Jiuyang&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Jacqueline&quot;,&quot;last_name&quot;:&quot;L. Norris-Drouin&quot;},{&quot;first_name&quot;:&quot;Stephanie&quot;,&quot;last_name&quot;:&quot;H. Cholensky&quot;},{&quot;first_name&quot;:&quot;Samantha&quot;,&quot;last_name&quot;:&quot;A. Ottavi&quot;},{&quot;first_name&quot;:&quot;Stephen&quot;,&quot;last_name&quot;:&quot;V. Frye&quot;},{&quot;first_name&quot;:&quot;Tatiana&quot;,&quot;last_name&quot;:&quot;G. Kutateladze&quot;},{&quot;first_name&quot;:&quot;Lindsey&quot;,&quot;last_name&quot;:&quot;I. James&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;05&quot;,&quot;day&quot;:&quot;17&quot;,&quot;issue&quot;:&quot;12&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;8510-8522&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.1c00430&quot;},&quot;abstract&quot;:&quot;Plant homeodomain finger protein 1 (PHF1) is an accessory component of the gene silencing complex polycomb repressive complex 2 and recognizes the active chromatin mark, trimethylated lysine 36 of histone H3 (H3K36me3). In addition to its role in transcriptional regulation, PHF1 has been implicated as a driver of endometrial stromal sarcoma and fibromyxoid tumors. We report the discovery and characterization of UNC6641, a peptidomimetic antagonist of the PHF1 Tudor domain which was optimized through in silico modeling and incorporation of non-natural amino acids. UNC6641 binds the PHF1 Tudor domain with a Kd value of 0.96 ± 0.03 μM while also binding the related protein PHF19 with similar potency. A crystal structure of PHF1 in complex with UNC6641, along with NMR and site-directed mutagenesis data, provided insight into the binding mechanism and requirements for binding. Additionally, UNC6641 enabled the development of a high-throughput assay to identify small molecule binders of PHF1.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00430&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00430" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00430&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00430" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00430&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00430" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00430&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00430&amp;href=/doi/10.1021/acs.jmedchem.1c00430" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.1c00430" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.1c00430" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.1c00430%26sid%3Dliteratum%253Aachs%26pmid%3D33999620%26genre%3Darticle%26aulast%3DEngelberg%26date%3D2021%26atitle%3DDiscovery%2Bof%2Ban%2BH3K36me3-Derived%2BPeptidomimetic%2BLigand%2Bwith%2BEnhanced%2BAffinity%2Bfor%2BPlant%2BHomeodomain%2BFinger%2BProtein%2B1%2B%2528PHF1%2529%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D12%26spage%3D8510%26epage%3D8522%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291553" title="Granular materials">Granular materials</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=290831" title="Monomers">Monomers</a>,</li><li><a href="/action/doSearch?ConceptID=292529" title="Aromatic compounds">Aromatic compounds</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/12" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/jmcmar.2021.64.issue-12/20210624/jmcmar.2021.64.issue-12.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00430/20210616/images/medium/jm1c00430_0006.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00430/20210616/images/large/jm1c00430_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00430&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Plant homeodomain finger protein 1 (PHF1) is an accessory component of the gene silencing complex polycomb repressive complex 2 and recognizes the active chromatin mark, trimethylated lysine 36 of histone H3 (H3K36me3). In addition to its role in transcriptional regulation, PHF1 has been implicated as a driver of endometrial stromal sarcoma and fibromyxoid tumors. We report the discovery and characterization of UNC6641, a peptidomimetic antagonist of the PHF1 Tudor domain which was optimized through in silico modeling and incorporation of non-natural amino acids. UNC6641 binds the PHF1 Tudor domain with a <i>K</i><sub>d</sub> value of 0.96 ± 0.03 μM while also binding the related protein PHF19 with similar potency. A crystal structure of PHF1 in complex with UNC6641, along with NMR and site-directed mutagenesis data, provided insight into the binding mechanism and requirements for binding. Additionally, UNC6641 enabled the development of a high-throughput assay to identify small molecule binders of PHF1.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div class="extra-info-sec articleNote"><h4><span></span> Special Issue</h4><p class="last">Published as part of the <i>Journal of Medicinal Chemistry</i> special issue “<a href="/page/vi/epigenetics" class="ext-link">Epigenetics 2022</a>”.</p></div><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_26076" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_26076" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Chromatin regulation is a dynamic process requiring oscillation between transcriptionally silent heterochromatin and transcriptionally active euchromatin which is tightly controlled by a network of epigenetic factors including DNA methylation, chromatin remodelers, histone variants, and histone post-translational modifications (PTMs). Regulation of this network is essential for proper gene transcription, while dysregulation of these factors has been linked to a variety of diseases from cancer to neurological disorders.<a onclick="showRef(event, 'ref1 ref2 ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3 ref4">(1−4)</a> Histone PTMs make up an extensive language known as the “histone code” which helps to dictate chromatin structure and function.<a onclick="showRef(event, 'ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref5 ref6">(5,6)</a> Examples of histone PTMs include acetylation, methylation, phosphorylation, and ubiquitylation, which are most commonly found on the flexible tails of histones H2A, H2B, H3, and H4. These marks can act both individually and in combination, enabling a diverse and complex network of downstream signals. Molecularly, these modifications typically function in one of two ways to influence chromatin accessibility: (1) by directly influencing histone–DNA interactions or (2) by recruiting specific effector proteins.</div><div class="NLM_p">Lysine mono-, di-, and trimethylation (Kme1/2/3) have been well characterized as essential and ubiquitous PTMs.<a onclick="showRef(event, 'ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref7 ref8">(7,8)</a> Kme1/2/3 are selectively recognized by a class of proteins known as methyl-lysine “readers”, which engage the methylammonium group largely through cation−π interactions with an aromatic cage.<a onclick="showRef(event, 'ref9 ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref9 ref10 ref11">(9−11)</a> The selectivity of this molecular recognition event for a specific methylation state is generally based on the size of the cage and number of aromatic residues in the cage, while selectivity for a certain lysine residue is often dictated by the surrounding histone sequence. Previous work has shown that the selectivity and potency of Kme readers for their cognate peptide substrates can be tuned by modifying the electronics of the aromatic cage residues.<a onclick="showRef(event, 'ref12 ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref12 ref13 ref14">(12−14)</a></div><div class="NLM_p">The systematic design of peptidomimetic ligands based on histone substrates can provide valuable insight into the molecular details of the binding event and an excellent starting point for ligand discovery.<a onclick="showRef(event, 'ref15 ref16 ref17 ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref15 ref16 ref17 ref18 ref19">(15−19)</a> Our lab has demonstrated the utility of such peptidomimetic ligands in a variety of applications, ranging from cellular target inhibition<a onclick="showRef(event, 'ref15 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref15 ref17 ref18">(15,17,18)</a> and activation<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> to assay development.<a onclick="showRef(event, 'ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref20 ref21">(20,21)</a></div><div class="NLM_p">In this paper, we focus on the Tudor domain of plant homeodomain finger protein 1 (PHF1), which functions as a reader for the active chromatin mark, trimethylated lysine 36 of histone H3 (H3K36me3).<a onclick="showRef(event, 'ref22 ref23 ref24 ref25 ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref22 ref23 ref24 ref25 ref26 ref27">(22−27)</a> PHF1 and its closely related homologue, PHF19, are the only annotated readers of H3K36me3 with affinities in the micromolar range. This recognition event has been shown to facilitate gene repression via the recruitment of the epigenetic silencing complex, polycomb repressive complex 2 (PRC2), to a subset of target genes.<a onclick="showRef(event, 'ref23 ref28 ref29 ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref23 ref28 ref29 ref30 ref31">(23,28−31)</a> Specifically, PHF1 and PHF19 levels increase dramatically during differentiation of mouse embryonic stem cells (mESCs),<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> and following differentiation, PHF1 and PHF19 facilitate silencing of stem cell genes by recruiting PRC2 which deposits the repressive mark, trimethylated lysine 27 on histone H3 (H3K27me3), to active genes previously marked by H3K36me3.<a onclick="showRef(event, 'ref26 ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref26 ref33 ref34">(26,33,34)</a> Additionally, the dysregulation of PHF1 has been implicated as a hallmark of disease. For example, aberrant fusions of PHF1 have been linked to endometrial stromal sarcoma and ossifying fibromyxoid tumors.<a onclick="showRef(event, 'ref35 ref36 ref37 ref38'); return false;" href="javascript:void(0);" class="ref ref35 ref36 ref37 ref38">(35−38)</a></div><div class="NLM_p">Despite the extensive validation of PHF1’s significance in normal and malignant biology, no high-quality ligands have been reported to date. Development of a potent and selective chemical probe for PHF1 will be essential to further validate the role of PHF1 in gene regulation and disease and assess its suitability for therapeutic intervention. As the downstream effects of altering protein expression via genetic methods can become convoluted,<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> especially for proteins that regulate transcription, small molecule antagonism of the Tudor domain of PHF1 could produce a functional phenotype that would directly elucidate the role of the Tudor domain in normal and disease states.</div><div class="NLM_p last">Using the endogenous binding partner of PHF1, H3K36me3, as a starting point, we sought to design potent and selective peptidomimetic ligands for PHF1. Through rational mutations and incorporation of synthetic Kme3 mimetics, we discovered UNC6641, which contains an aromatic lysine mimetic and binds the PHF1 Tudor domain with a <i>K</i><sub>d</sub> value of 0.96 ± 0.03 μM. X-ray crystallography revealed that the (isopropyl)phenethyl lysine mimetic of UNC6641 both strengthened existing interactions and created new contacts with the aromatic cage residues of PHF1. Most notably, the phenethyl group of UNC6641 demonstrated face-to-face and edge-to-face π–π stacking interactions with all four aromatic cage residues, likely contributing to its increased affinity for the Tudor domain. This appears to be the first example of a peptidomimetic ligand for a Kme reader domain that utilizes an aromatic substituent on the lysine amino group to better engage the aromatic cage. Through careful characterization of the molecular recognition of UNC6641 by the PHF1 Tudor domain, we have paved the way for future assay development, small molecule discovery, and investigation of the biological role of PHF1.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_48383" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_48383" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Structure-Based Ligand Design</h3><div class="NLM_p">We began the development of PHF1 ligands by determining the minimal H3K36me3 sequence required to maintain binding to the Tudor domain. H3K36me3 binds PHF1 in a surface-groove binding mode,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> making key interactions with a relatively exposed portion of the protein (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00430/suppl_file/jm1c00430_si_001.pdf" class="ext-link">Figure S1, Supporting Information</a>). Previous literature has reported binding affinities for PHF1 of 4, 14, and 16 μM using H3K36me3 peptides spanning 15, 14, and 10 residues, respectively.<a onclick="showRef(event, 'ref23 ref27 ref40'); return false;" href="javascript:void(0);" class="ref ref23 ref27 ref40">(23,27,40)</a> However, smaller peptides are more attractive starting points for ligand development, as they are more easily synthesized, enable concise and tractable structure–activity relationship (SAR) exploration, and demonstrate improved cell permeability. In order to decrease the peptide size while maintaining affinity, we synthesized a series of H3K36me3 variants based on the eight residues reported to make direct contacts with the PHF1 surface, H3(33–40), and known peptide SAR.<a onclick="showRef(event, 'ref22 ref23 ref31 ref41'); return false;" href="javascript:void(0);" class="ref ref22 ref23 ref31 ref41">(22,23,31,41)</a> We assessed binding affinities for these peptides using isothermal titration calorimetry (ITC), the results of which are outlined in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00430/suppl_file/jm1c00430_si_001.pdf" class="ext-link">Figure S2</a>). Our 8-mer peptide spanning H3(33–40) gave a <i>K</i><sub>d</sub> value of 22.3 ± 1.9 μM, a slighter weaker affinity than the previously reported 10-mer peptide H3(31–40) (<i>K</i><sub>d</sub> of 16 ± 4 μM).<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a><named-content content-type="anchor" rid="tbl1" type="simple"></named-content></div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Binding of H3K36me3 Variants to PHF1 Tudor<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00430/20210616/images/medium/jm1c00430_0009.gif" alt="" id="t100" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last"><i>K</i><sub>d</sub> values were determined by two independent ITC experiments (mean ± SD).</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">Peptide sequence differs from natural histone H3 sequence at residues denoted in blue.</p></div></div><div></div></div><div class="NLM_p">In order to further assess the minimal binding sequence required, we next truncated the C-terminal R40 residue. This residue is solvent-exposed in the reported X-ray crystal structure of H3K36me3 bound to the PHF1 Tudor domain (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HCZ">4HCZ</a>),<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> leading us to believe that it did not make stable contacts with the PHF1 surface. However, truncation of R40 resulted in a modest 2-fold decrease in affinity, likely due to loss of transient hydrogen bonds formed with the PHF1 surface. In an attempt to recoup this loss in affinity, we incorporated a G33V mutation into the H3(33–39) peptide. This mutation was previously reported to increase the affinity of H3K36me3 for the PHF1 Tudor domain, as shown by Western blot.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> However, we observed the same 2-fold loss in affinity resulting from the R40 truncation and no significant effect from the G33V mutation. Gratifyingly, we regained affinity by mutation of H39 to a leucine residue (H39L), which we hypothesized would better interact with the hydrophobic patch of PHF1 made up of L38, L45, L46, and L48.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Incorporation of the H39L mutant into the initial 8-mer peptide spanning H3(33–40) afforded a small boost in potency for a final <i>K</i><sub>d</sub> value of 18.7 ± 4.1 μM.</div><div class="NLM_p">Prior work has demonstrated the utility of non-natural Kme mimics in improving the binding affinity and selectivity of histone peptides and peptidomimetics for their target reader proteins.<a onclick="showRef(event, 'ref15 ref17 ref18 ref42 ref43 ref44'); return false;" href="javascript:void(0);" class="ref ref15 ref17 ref18 ref42 ref43 ref44">(15,17,18,42−44)</a> In addition, replacement of the quaternary amine in Kme3 with a secondary or tertiary amine is attractive in that it is likely to improve passive cell permeability by eliminating the permanent positive charge. While removal of the trimethylammonium can weaken the characteristic cation−π interactions with the aromatic cage residues, this can sometimes be overcome via incorporation of additional hydrophobic interactions. In the context of the GGVKme3KPLR peptide sequence, we substituted Kme3 with synthetic analogues that take advantage of the hydrophobic nature of the aromatic cage, as well as the potential for hydrogen bonding and π–π interactions (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, <a href="/doi/suppl/10.1021/acs.jmedchem.1c00430/suppl_file/jm1c00430_si_001.pdf" class="ext-link">Figure S2</a>). Encouragingly, we observed that replacing Kme3 with the corresponding (ethyl)isopropyl-lysine residue (UNC7643, <i>K</i><sub>d</sub> = 10.1 ± 1.2 μM) was well tolerated and moderately beneficial. The addition of a hydrogen bond acceptor in the (ethyl)ethylfuran-lysine (UNC6640, <i>K</i><sub>d</sub> = 17.9 ± 4.6 μM) did not further improve potency from the trimethylated peptide. The most significant improvement came with the incorporation of the aromatic (isopropyl)phenethyl-lysine residue in UNC6641, which resulted in a 20-fold increase in affinity versus the parent peptide to give a <i>K</i><sub>d</sub> value of 0.96 ± 0.03 μM.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Binding of Peptidomimetic Ligands with Non-Natural Kme Mimetics to PHF1 Tudor</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00430/20210616/images/medium/jm1c00430_0004.gif" alt="" id="fx1" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last"><i>K</i><sub>d</sub> values were determined by two independent ITC experiments (mean ± SD).</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">IC<sub>50</sub> values were determined by three technical replicates (mean ± SD).</p></div></div><div></div></div><div class="NLM_p last">As ITC is low throughput, we capitalized on our discovery of a more potent PHF1 ligand to develop a time-resolved fluorescence resonance energy transfer (TR-FRET) assay using a biotinylated analogue of UNC6641 as a bait ligand. Biotin-UNC6641 binds to the PHF1 Tudor domain about equipotently to UNC6641 (<i>K</i><sub>d</sub> ∼ 3 μM, <a href="/doi/suppl/10.1021/acs.jmedchem.1c00430/suppl_file/jm1c00430_si_001.pdf" class="ext-link">Figure S2</a>). TR-FRET assays have been used previously to screen Kme reader domains in a robust fashion, but they typically require a bait ligand that binds in the low micromolar range.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> This had not previously been feasible due to the low affinity of the endogenous histone substrate for the PHF1 Tudor domain. Using this assay, we were able to screen a larger panel of peptidomimetic ligands based on UNC6641, some of which are shown in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00430/suppl_file/jm1c00430_si_001.pdf" class="ext-link">Figure S3</a>). It should be noted that the IC<sub>50</sub> of UNC6641 as determined by TR-FRET closely agrees with the dissociation constant measured by ITC (<i>K</i><sub>d</sub> = 0.96 ± 0.03 μM, IC<sub>50</sub> = 1.6 ± 0.5 μM). We observed that increasing or decreasing the distance between the K36 amine and phenyl ring by a single methylene as in UNC7259 and UNC7253 resulted in a 5- and 12-fold loss in potency, respectively. This suggests that the phenyl substituent makes a fairly specific interaction(s) with the Tudor domain and the increase in potency is not solely the result of an increase in hydrophobicity and/or steric bulk. Additionally, removing the potential for π–π interactions by swapping the phenyl group for a cyclohexyl ring in UNC7258 also decreased the potency 13-fold relative to UNC6641. Taken together, these data suggest that the potent binding of UNC6641 to PHF1 Tudor is unique and specific, depending both on the aromaticity and the position of the phenethyl substituent.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Confirmation of the UNC6641 Binding Mode by NMR and Crystallography</h3><div class="NLM_p">Preliminary analysis of the binding mode of UNC6641 was performed by docking UNC6641 into the PHF1 Tudor domain (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HCZ">4HCZ</a>) using Schrödinger Glide docking (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00430/suppl_file/jm1c00430_si_001.pdf" class="ext-link">Figure S4</a>). The docking model suggested that the (isopropyl)phenethyl-lysine of UNC6641 filled the aromatic cage more efficiently than the Kme3 substituent of the natural histone peptide (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00430/suppl_file/jm1c00430_si_001.pdf" class="ext-link">Figure S4a</a>). In addition, the model offered a possible explanation for the increased affinity of the peptide for the PHF1 Tudor domain. Introduction of the phenethyl substituent appeared to provide additional π interactions with the aromatic cage through π–π stacking of the phenyl group of UNC6641 and PHF1 Y47 and W41 (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00430/suppl_file/jm1c00430_si_001.pdf" class="ext-link">Figure S4b</a>).</div><div class="NLM_p">To further validate this model, we investigated the binding of UNC6641 to the PHF1 Tudor domain by NMR. Following expression and purification of the uniformly <sup>15</sup>N-labeled PHF1 Tudor domain (14–87), <sup>1</sup>H,<sup>15</sup>N heteronuclear single quantum coherence (HSQC) spectra of the protein were collected while UNC6641 was added stepwise. Substantial chemical shift changes in the intermediate to slow exchange regime on the NMR time scale were observed upon addition of UNC6641, indicating direct and tight binding (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>a). Specific differences in chemical shifts for backbone amides of the PHF1 Tudor domain in the apo state and the UNC6641 bound state (at a molar protein to UNC6641 ratio of 1:4) are shown in the histogram in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>b. The Tudor domain residues most perturbed upon UNC6641 binding, including W41, L46, Y47, G49, F65, and E66, were then mapped onto the surface of the PHF1 Tudor domain (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>c). As expected, these residues included and clustered around the aromatic cage residues of W41, Y47, F65, and F71 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>d) and showed a similar CSP pattern to the titration of H3K36me3 reported previously<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00430/suppl_file/jm1c00430_si_001.pdf" class="ext-link">Figure S5</a>), further supporting that UNC6641 binds in the H3K36me3 binding site.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00430/20210616/images/medium/jm1c00430_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00430/20210616/images/large/jm1c00430_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Characterization of the interaction between the PHF1 Tudor domain and UNC6641 by NMR. (a) Overlay of <sup>1</sup>H,<sup>15</sup>N HSQC spectra of the <sup>15</sup>N-labeled PHF1 Tudor domain collected before and after the addition of UNC6641. Spectra are color coded according to the protein–peptide molar ratio. (b) Histogram showing chemical shift perturbations of backbone amides of the PHF1 Tudor domain at a 1:4 molar ratio of protein to UNC6641. The dashed line indicates an averaged chemical shift change plus one standard deviation. (c) The surface representation of the PHF1 Tudor domain (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HCZ">4HCZ</a>). Residues that exhibit substantial changes in part b are colored purple. (d) The PHF1 Tudor domain (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HCZ">4HCZ</a>) is depicted as a gray ribbon with the aromatic cage residues W41, Y47, F65, and F71 shown as pink sticks and labeled.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00430/20210616/images/large/jm1c00430_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00430&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To gain further insight into the molecular basis of the PHF1 Tudor–UNC6641 interaction, we co-crystallized the Tudor domain with UNC6641 and obtained a crystal structure of the complex at 1.85 Å resolution (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). The X-ray diffraction and structure refinement statistics are summarized in <a href="/doi/suppl/10.1021/acs.jmedchem.1c00430/suppl_file/jm1c00430_si_001.pdf" class="ext-link">Supplementary Table 1</a>. UNC6641 is bound in an elongated groove of the Tudor domain (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>a). The electron-density map of the UNC6641 peptide chain from G1 to R8 and the phenethyl and isopropyl lysine substituents can be unambiguously traced (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>b).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00430/20210616/images/medium/jm1c00430_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00430/20210616/images/large/jm1c00430_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Molecular mechanism for the association of the PHF1 Tudor domain with UNC6641 (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7LKY">7LKY</a>). (a) The electrostatic surface potential of the PHF1 Tudor domain. Bound UNC6641 is colored green, and the phenyl substituent of K4 is indicated as Lig. (b) The 2Fo-Fc electron-density map of UNC6641 contoured at 1 σ shown as a gray mesh. (c) Structure of the PHF1 Tudor domain in complex with UNC6641. The PHF1 Tudor domain is shown as a gray ribbon, UNC6641 is shown as green sticks, and the protein residues interacting with UNC6641 are shown as gray sticks (except for W41, Y47, F65, and F71, which are colored yellow). The hydrogen bonds are indicated by yellow dashed lines. (d) Zoomed-in view of the UNC6641 binding site. (e) Overlay of the structures of the of PHF1 Tudor domain in complex with the H3K36me3 peptide (purple sticks) (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HCZ">4HCZ</a>) and UNC6641 (green sticks).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00430/20210616/images/large/jm1c00430_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00430&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The structure belongs to the <i>P</i>1 space group with eight molecules of the PHF1 Tudor domain (each bound to UNC6641) present in one asymmetric unit. In the complex, the Tudor domain adopts a five-stranded β-barrel fold and contains the aromatic cage, which is comprised of W41, Y47, F65, and F71 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>c). The loops between β3 and β4 strands form an acidic groove, which is occupied by the N-terminal portion of UNC6641 (G1-K5), whereas L38, L45, L46, and L48 of the β3 strand form a hydrophobic patch where the C-terminal part of UNC6641 (P6-R8) is bound. The protein–ligand complex is further stabilized by several hydrogen bonds (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>d). Specifically, the N-terminal amino group of UNC6641 is restrained by a hydrogen bond with the side chain carboxyl group of D68. The side chain amino group of K5 of UNC6641 is hydrogen bonded to the carboxyl group of E66, and the backbone amino group of L7 of UNC6641 forms a hydrogen bond with the backbone carbonyl of L46. These important hydrogen bonding interactions are in agreement with the large resonance perturbations observed for these residues in NMR titration experiments (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>b).</div><div class="NLM_p">Additionally, the pyrrolidine ring of P6 of UNC6641 lies parallel to the hydroxyphenyl moiety of PHF1 Y47, interacting with it through C–H···π contacts. The hydrophobic contacts involving the backbone of G2 and the side chains of K4, L7, and R8 of UNC6641 and D67, Y47, L38, and L45 of the PHF1 Tudor domain further contribute to the interaction (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00430/suppl_file/jm1c00430_si_001.pdf" class="ext-link">Figure S6</a>, Ligplot).</div><div class="NLM_p">The K4 side chain of UNC6641 contains two substituents, a phenethyl group and an isopropyl group. In the complex, the phenethyl group of UNC6641 inserts deeply into the aromatic cage of the Tudor domain and participates in hydrophobic interactions with W41, Y47, F65, F71, and A39 of the protein. The aromatic ring of the phenethyl group is sandwiched between the aromatic side chains of F65 and W41 of the Tudor aromatic cage residues. Specifically, the benzene ring is engaged with the phenyl group of F65 through a π–π interaction (the distance between these aromatic moieties is ∼3.4 Å). Furthermore, the benzene ring is positioned almost perpendicular to the phenyl ring of F71 and to the hydroxyphenyl group of Y47, suggesting productive T-shaped edge-to-face interactions. The strength of these π-interactions seems to depend upon the precise placement of the benzene ring as dictated by the ethyl linker. As seen with UNC7259 and UNC7253, increasing or decreasing the linker length by a single methylene resulted in a modest decrease in potency. Finally, the two methyl groups of the isopropyl lysine substituent of UNC6641 interact via hydrophobic interactions with the side chains of W41, Y47, and F71 of the Tudor domain, likely stabilizing the binding of the phenethyl substituent (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>c,d).</div><div class="NLM_p last">Structural overlay of the PHF1 Tudor domain in complex with the H3K36me3 peptide and UNC6641 shows that the majority of residues within the peptide chains superimpose well, indicating similar binding modes of these ligands (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>e). However, several important differences are apparent. In the complex with the H3K36me3 peptide, G33 of the peptide does not interact with D68 of the Tudor domain, whereas G1 of UNC6641 forms a hydrogen bond with D68. Further, H39 in the H3K36me3 peptide is replaced with a leucine in UNC6641, which augments the association of UNC6641 with the hydrophobic patch. Importantly, the phenethyl substituent of UNC6641 is involved in π stacking and other productive interactions with the aromatic cage residues of the Tudor domain, which likely accounts for the tighter binding of UNC6641 than that of the H3K36me3 peptide (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>e).</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Biochemical Analysis of the UNC6641 Binding Mode</h3><div class="NLM_p">To tease apart the key π-stacking interactions between the Kme mimetic of UNC6641 and the PHF1 Tudor aromatic cage residues, we performed alanine scanning mutagenesis of residues W41, Y47, and F65. We used UNC7643 as a control compound, as it maintains the (ethyl)isopropyl lysine but lacks the characteristic phenyl group. UNC7643 binds the wildtype PHF1 Tudor domain about 10-fold less potently than UNC6641. Previous literature has shown that single-residue mutations in the aromatic cage can significantly decrease the binding affinity of H3K36me3 for PHF1 due to disruption of the essential cation−π interactions.<a onclick="showRef(event, 'ref23 ref27'); return false;" href="javascript:void(0);" class="ref ref23 ref27">(23,27)</a> However, with the added affinity from π–π stacking, we hypothesized that UNC6641 but not UNC7643 may maintain binding to these aromatic cage residue mutants, although likely to a lesser degree than wildtype. UNC7643 failed to bind any PHF1 aromatic cage mutants, while UNC6641 retained moderate binding to PHF1 Tudor W41A and F65A (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>, <a href="/doi/suppl/10.1021/acs.jmedchem.1c00430/suppl_file/jm1c00430_si_001.pdf" class="ext-link">Figure S7</a>). Interestingly, UNC6641 showed no measurable binding to the Y47A mutant, suggesting a preferential dependence on the interactions with this residue.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Mutation of Aromatic Cage Residues to Assess the Key Molecular Interactions between UNC6641 and the PHF1 Tudor Domain</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00430/20210616/images/medium/jm1c00430_0005.gif" alt="" id="fx2" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last"><i>K</i><sub>d</sub> values were determined by two independent ITC experiments (mean ± SD).</p></div></div><div></div></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Selectivity of UNC6641</h3><div class="NLM_p">Although we focused our initial studies on the antagonism of PHF1, we were curious to evaluate the binding of UNC6641 to other closely related Kme reader domains. PHF1 belongs to the polycomb-like (PCL) family of proteins which also consists of plant homeodomain finger protein 19 (PHF19) and metal response element binding transcription factor 2 (MTF2). While PHF1 and PHF19 each contain an aromatic cage composed of four key aromatic residues, MTF2 contains a serine in place of F71/Y75 which is thought to be responsible for “closing” the aromatic cage (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>a). In many cases, PHF1 and PHF19 have been shown to play redundant roles in normal biology distinct from that of MTF2,<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> suggesting that a dual ligand for PHF1 and PHF19 may be a valuable tool to investigate PCL biology. We were pleased to see that UNC6641 binds the PHF19 Tudor domain with a <i>K</i><sub>d</sub> value of 2.1 ± 0.2 μM by ITC, only 2-fold weaker than PHF1 (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00430/suppl_file/jm1c00430_si_001.pdf" class="ext-link">Figure S2</a>). In contrast, UNC6641 binds MTF2 with a <i>K</i><sub>d</sub> value of 18 ± 2 μM, ∼10- and 20-fold weaker than PHF19 and PHF1, respectively (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00430/suppl_file/jm1c00430_si_001.pdf" class="ext-link">Figure S8</a>). We next conducted a sequence alignment within the Tudor family, and those domains most closely related to the PCL proteins are shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>a, all of which share 3 of the 4 aromatic cage residues with the PCL proteins (for the full Tudor domain phylogenetic tree, see <a href="/doi/suppl/10.1021/acs.jmedchem.1c00430/suppl_file/jm1c00430_si_001.pdf" class="ext-link">Figure S9</a>). However, none of these Tudor domains including those of 53BP1, PHF20, and PHF20L1 demonstrated measurable binding to UNC6641 by ITC (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>b, <a href="/doi/suppl/10.1021/acs.jmedchem.1c00430/suppl_file/jm1c00430_si_001.pdf" class="ext-link">Figure S8</a>). Furthermore, preliminary screening of representative members of other Kme reader families with UNC6641 by ITC and/or TR-FRET showed >20-fold selectivity for PHF1/19 (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00430/suppl_file/jm1c00430_si_001.pdf" class="ext-link">Figure S10</a>). Overall, this suggests that both the sequence and unique Kme mimetic of UNC6641 confer selectivity over related proteins.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00430/20210616/images/medium/jm1c00430_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00430/20210616/images/large/jm1c00430_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Selectivity of UNC6641 across closely related Tudor domains. (a) Sequence alignment of Tudor domains most closely related to PHF1. Aromatic cage residues are in orange. * = Fully conserved. (b) Binding affinities of UNC6641 for closely related Tudor domains. <i>K</i><sub>d</sub> values were determined by two independent ITC experiments (mean ± SD).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00430/20210616/images/large/jm1c00430_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00430&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_26614" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_26614" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">PHF1 and its homologue, PHF19, are valuable accessory components of the ubiquitous repressive complex, PRC2, through their roles as readers of H3K36me3. A recent uptick in literature investigating these proteins revealed key roles for PHF1 and PHF19 in malignant biology and encouraged our pursuit of ligands that could be used in the development of a chemical probe.<a onclick="showRef(event, 'ref32 ref33 ref35 ref45 ref46 ref47'); return false;" href="javascript:void(0);" class="ref ref32 ref33 ref35 ref45 ref46 ref47">(32,33,35,45−47)</a> In this paper, we describe the discovery and characterization of UNC6641, which binds the Tudor domains of PHF1 and PHF19 with <i>K</i><sub>d</sub>’s of 0.96 ± 0.03 and 2.1 ± 0.2 μM, respectively.</div><div class="NLM_p">UNC6641 is an 8-mer peptidomimetic ligand that contains a unique (isopropyl)phenethyl-lysine residue in place of K36me3. To the best of our knowledge, this is the first example of an aromatic substituent on a lysine residue facilitating enhanced interactions with a Kme reader domain. Co-crystallization of UNC6641 with the PHF1 Tudor domain revealed key interactions that contribute to the potent binding of this ligand. Namely, the distinctive phenethyl group of UNC6641 is sandwiched between the PHF1 aromatic cage residues W41 and F65, engaging in face-to-face π–π stacking interactions with these residues. Additionally, UNC6641 engages in T-shaped edge-to-face π stacking interactions with the remaining cage residues Y47 and F71. Combined with the additional hydrogen bonds and hydrophobic interactions created from our rational sequence mutations, we were able to determine structure–activity relationships that resulted in a clear gain in affinity relative to the endogenous histone peptide. Additionally, we used site-directed mutagenesis of the aromatic cage residues to gauge which ligand interactions are most critical for binding, which revealed that the phenethyl group of UNC6641 makes critical contacts with PHF1 Y47. UNC6641 showed selectivity for PHF1/19 over the closely related protein MTF2, likely due to the dependence on π interactions with all four aromatic cage residues. Although MTF2 has high sequence similarity to PHF1/19, it lacks the final aromatic cage residue that is present in PHF1/19.</div><div class="NLM_p">While UNC6641 demonstrates the ligandability of PHF1 and shows promising potency and selectivity, peptidomimetic ligands are often limited in their utility by low cell permeability. However, the discovery and characterization of UNC6641 allowed us to develop and optimize a robust TR-FRET displacement assay for the PHF1 Tudor domain, which had previously not been feasible due to the low affinity of the endogenous histone substrate. Importantly, this assay can facilitate the screening of larger compound libraries toward the discovery of novel PHF1 Tudor domain antagonists. Additional screening efforts will shed light on the likelihood of uncovering potent small molecule chemical probes that bind PHF1. Recent reports have also demonstrated that improved potency and selectivity can be achieved for Tudor domain containing proteins such as Spindlin1 through the development of bidentate ligands which target multiple consecutive reader domains.<a onclick="showRef(event, 'ref48 ref49'); return false;" href="javascript:void(0);" class="ref ref48 ref49">(48,49)</a> While simultaneously targeting the Tudor and adjacent PHD domain (PHD1) of PHF1 may be another strategy to achieve potent antagonism, data supporting the reader activity of PHF1 PHD1 remains largely elusive.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a></div><div class="NLM_p last">A potent and cell permeable small molecule chemical probe for PHF1 would be a valuable tool for researchers invested in studying the role of the PCL proteins as well as PRC2 regulation. Additionally, a PHF1 chemical probe has the potential to catalyze new drug discovery efforts toward the treatment of endometrial stromal sarcomas and fibroxymoid tumors, as these diseases depend heavily on aberrant PHF1 fusion proteins.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_86687" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_86687" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Synthesis of Peptides and Peptidomimetics</h3><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i15" class="anchor-spacer"></div><h4 class="article-section__title" id="_i15"> General Procedures</h4><div class="NLM_p last">Reverse phase column chromatography was performed with a Teledyne ISCO CombiFlashRf 200 using C18 RediSepRf Gold columns with the UV detector set to 220 and 254 nm. The mobile phases used are indicated for each compound. Preparative HPLC was performed using an Agilent Prep 1200 series with the UV detector set to 220 and 254 nm. Samples were injected onto a Phenomenex Luna 250 × 30 mm<sup>2</sup>, 5 μm, C18 column at 25 °C. Mobile phases of A (H<sub>2</sub>O + 0.1% TFA) and B (CH<sub>3</sub>CN + 1% H<sub>2</sub>O) were used with a flow rate of 40 mL/min. A general gradient of 0–25 min increasing from 10 to 100% B, followed by a 100% B flush for another 5 min, was used. Small variations in this purification method were made as needed to achieve ideal separation for each compound. Analytical LC-MS data for all compounds were acquired using an Agilent 1260 Infinity II Series system with the UV detector set to 220 and 254 nm. Samples were injected (5 μL) onto an Agilent Eclipse Plus 4.6 × 3 × 50 mm<sup>3</sup>, 1.8 mm, C18 column at 25 °C. Mobile phases A (H<sub>2</sub>O + 0.1% acetic acid) and B (CH<sub>3</sub>CN + 0.1% acetic acid) were used with a linear gradient from 10 to 100% B in 5.0 min, followed by a flush at 100% B for another 2 min at a flow rate of 1.0 mL/min. Mass spectra (MS) data were acquired in positive ion mode using an Agilent 6110 single quadrupole mass spectrometer with an electrospray ionization (ESI) source or matrix-assisted laser desorption/ionization time-of-flight mass spectrometry (MALDI TOF/TOF) using an AB Sciex 5800. Nuclear magnetic resonance (NMR) spectra were recorded on a Varian Mercury spectrometer at 400 MHz for proton (<sup>1</sup>H) and carbon (<sup>13</sup>C) NMR; chemical shifts are reported in ppm (δ) relative to residual protons in deuterated solvent peaks, and coupling constants are reported in Hz. All compounds reported had >95% purity, as determined by LC-MS.</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> Solid-Phase Peptide Synthesis (SPPS)</h4><div class="NLM_p">Peptide compounds listed in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> were synthesized by SPPS on Fmoc Rink amide resin on polystyrene beads (0.81 mmol/g loading, 1 equiv, Anaspec). The resin was first swelled in DCM for 5 min followed by equilibration in DMF for another 5 min. The resin was then Fmoc deprotected in a solution of 2.5% 1,8-diazabicycloundec-7-ene (DBU) and 2.5% pyrrolidine in DMF for 10 min. Then, the resin was filtered and washed twice with DMF, methanol, DMF, and DCM (6 mL each) before adding the first amino acid for coupling. Fmoc-protected amino acids (3 equiv) except Fmoc-Arg(Pmc)-OH were preactivated with HBTU (3 equiv), HOAt (3 equiv), and DIPEA (10 equiv) for 5 min with periodic swirling in 5 mL of DMF and 3 mL of DCM. For Fmoc-Arg(Pmc)-OH coupling, the longer preincubation was omitted to prevent formation of the intramolecular cyclization byproduct. The solution was then added to the resin and left on a shaker at room temperature for 1 h. The resin was filtered and washed twice with DCM, DMF, methanol, and DMF again (6 mL each). The N-terminal Fmoc protecting group was then removed in a solution of 2.5% DBU and 2.5% pyrrolidine in DMF for 10 min. The resin was filtered and washed twice with DMF, methanol, DMF, and DCM (6 mL each) before adding the next amino acid for coupling. Following installation of the final residue, the resin was rinsed six times with DCM. Cleavage cocktail (95% trifluoroacetic acid, 2.5% triisopropylsilane, and 2.5% water) was added to the resin, the mixture was left on the shaker for 2 h, and the filtrate was collected. The resin was rinsed twice with DCM, and filtrates were pooled and concentrated under a vacuum. The crude product was then submitted to preparative HPLC for purification as described in the <a class="ref internalNav" href="#sec4_1_1" aria-label="General Procedures">General Procedures</a> section. The solvent was removed <i>in vacuo</i>, dissolved in water, and lyophilized to yield the desired products.</div><div id="sec4_1_2_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i17" class="anchor-spacer"></div><h5 class="article-section__title" id="_i17"> (<i>S</i>)-6-(((<i>S</i>)-6-Amino-1-((<i>S</i>)-2-(((S)-1-(((<i>S</i>)-1-amino-5-((amino(iminio)methyl)amino)-1-oxopentan-2-yl)amino)-3-(1<i>H</i>-imidazol-4-yl)-1-oxopropan-2-yl)carbamoyl)pyrrolidin-1-yl)-1-oxohexan-2-yl)amino)-5-((<i>S</i>)-2-(2-(2-aminoacetamido)acetamido)-3-methylbutanamido)-<i>N</i>,<i>N</i>,<i>N</i>-trimethyl-6-oxohexan-1-aminium (H3 Peptide: GGVKme3KPHR)</h5><div class="NLM_p last">White powder; yield 18%. <sup>1</sup>H NMR (MeOD-<i>d</i><sub>4</sub>, 400 MHz): δ 8.81 (d, <i>J</i> = 1.4 Hz, 1H), 7.43 (d, <i>J</i> = 1.4 Hz, 1H), 4.74–4.70 (m, 1H), 4.64 (dd, <i>J</i> = 8.6, 5.1 Hz, 1H), 4.44–4.38 (m, 2H), 4.36 (dd, <i>J</i> = 8.7, 5.2 Hz, 1H), 4.16 (d, <i>J</i> = 7.3 Hz, 1H), 4.05–3.93 (m, 2H), 3.84–3.77 (m, 1H), 3.76 (s, 2H), 3.68–3.62 (m, 1H), 3.21 (dt, <i>J</i> = 9.1, 6.6 Hz, 3H), 3.13 (s, 9H), 2.95 (t, <i>J</i> = 7.4 Hz, 2H), 2.28–2.16 (m, 1H), 2.12–1.73 (m, 12H), 1.68 (dt, <i>J</i> = 13.3, 7.1 Hz, 6H), 1.55–1.33 (m, 5H), 0.98 (dd, <i>J</i> = 6.8, 1.4 Hz, 6H); MS (ESI): <i>m</i>/<i>z</i> calcd for [C<sub>41</sub>H<sub>76</sub>N<sub>16</sub>O<sub>8</sub>]<sup>3+</sup>/2 460.80, found 460.30.</div></div><div id="sec4_1_2_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i18" class="anchor-spacer"></div><h5 class="article-section__title" id="_i18"> (<i>S</i>)-6-(((<i>S</i>)-6-Amino-1-((<i>S</i>)-2-(((<i>S</i>)-1-amino-3-(1<i>H</i>-imidazol-4-yl)-1-oxopropan-2-yl)carbamoyl)pyrrolidin-1-yl)-1-oxohexan-2-yl)amino)-5-((<i>S</i>)-2-(2-(2-aminoacetamido)acetamido)-3-methylbutanamido)-<i>N</i>,<i>N</i>,<i>N</i>-trimethyl-6-oxohexan-1-aminium (H3 Peptide: GGVKme3KPH)</h5><div class="NLM_p last">White powder; yield 10%. <sup>1</sup>H NMR (MeOD-<i>d</i><sub>4</sub>, 400 MHz): δ 8.82 (d, <i>J</i> = 1.4 Hz, 1H), 7.41 (d, <i>J</i> = 1.3 Hz, 1H), 4.65 (td, <i>J</i> = 9.1, 8.5, 5.3 Hz, 2H), 4.39 (ddd, <i>J</i> = 11.3, 8.8, 5.5 Hz, 2H), 4.15 (d, <i>J</i> = 7.2 Hz, 1H), 4.05–3.91 (m, 2H), 3.86–3.78 (m, 1H), 3.76 (s, 2H), 3.73–3.63 (m, 1H), 3.13 (s, 9H), 2.95 (t, <i>J</i> = 7.4 Hz, 2H), 2.22 (ddd, <i>J</i> = 15.1, 7.4, 4.2 Hz, 1H), 2.14–1.74 (m, 10H), 1.69 (ddt, <i>J</i> = 14.0, 10.9, 4.8 Hz, 3H), 1.54–1.40 (m, 4H), 1.38–1.31 (m, 3H), 0.97 (d, <i>J</i> = 6.5 Hz, 6H); MS (ESI): <i>m</i>/<i>z</i> calcd for [C<sub>35</sub>H<sub>63</sub>N<sub>12</sub>O<sub>7</sub>]<sup>2+</sup> 764.59, found 764.30.</div></div><div id="sec4_1_2_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i19" class="anchor-spacer"></div><h5 class="article-section__title" id="_i19"> (<i>S</i>)-6-(((<i>S</i>)-6-Amino-1-((<i>S</i>)-2-(((<i>S</i>)-1-amino-3-(1<i>H</i>-imidazol-4-yl)-1-oxopropan-2-yl)carbamoyl)pyrrolidin-1-yl)-1-oxohexan-2-yl)amino)-5-((<i>S</i>)-2-(2-((<i>S</i>)-2-amino-3-methylbutanamido)acetamido)-3-methylbutanamido)-<i>N</i>,<i>N</i>,<i>N</i>-trimethyl-6-oxohexan-1-aminium (H3 Peptide: VGVKme3KPH)</h5><div class="NLM_p last">White powder; yield 12%. <sup>1</sup>H NMR (MeOD-<i>d</i><sub>4</sub>, 400 MHz): δ 8.83 (d, <i>J</i> = 1.4 Hz, 1H), 7.41 (d, <i>J</i> = 1.3 Hz, 1H), 4.65 (ddd, <i>J</i> = 8.6, 5.4, 2.9 Hz, 2H), 4.39 (ddd, <i>J</i> = 11.7, 8.7, 5.6 Hz, 2H), 4.18 (d, <i>J</i> = 7.0 Hz, 1H), 4.06–3.94 (m, 2H), 3.84 (dt, <i>J</i> = 10.0, 6.4 Hz, 1H), 3.72–3.64 (m, 2H), 3.17 (d, <i>J</i> = 7.7 Hz, 1H), 3.14 (s, 1H), 3.13 (s, 9H), 2.95 (t, <i>J</i> = 7.4 Hz, 2H), 2.30–1.60 (m, 16H), 1.47 (dt, <i>J</i> = 21.3, 7.5 Hz, 4H), 1.07 (dd, <i>J</i> = 6.9, 1.4 Hz, 6H), 0.98 (dd, <i>J</i> = 6.8, 2.5 Hz, 6H); MS (ESI): <i>m</i>/<i>z</i> calcd for [C<sub>38</sub>H<sub>69</sub>N<sub>12</sub>O<sub>7</sub>]<sup>2+</sup> 806.54, found 806.40.</div></div><div id="sec4_1_2_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i20" class="anchor-spacer"></div><h5 class="article-section__title" id="_i20"> (<i>S</i>)-6-(((<i>S</i>)-6-Amino-1-((<i>S</i>)-2-(((<i>S</i>)-1-amino-4-methyl-1-oxopentan-2-yl)carbamoyl)pyrrolidin-1-yl)-1-oxohexan-2-yl)amino)-5-((<i>S</i>)-2-(2-((S)-2-amino-3-methylbutanamido)acetamido)-3-methylbutanamido)-<i>N</i>,<i>N</i>,<i>N</i>-trimethyl-6-oxohexan-1-aminium (H3 Peptide: VGVKme3KPL)</h5><div class="NLM_p last">White powder; yield 50%. <sup>1</sup>H NMR (MeOD-<i>d</i><sub>4</sub>, 400 MHz): δ 4.64–4.60 (m, 1H), 4.46–4.31 (m, 3H), 4.17 (dd, <i>J</i> = 7.2, 4.8 Hz, 1H), 4.07–3.92 (m, 2H), 3.80 (dd, <i>J</i> = 11.1, 5.2 Hz, 1H), 3.73–3.62 (m, 2H), 3.13 (s, 9H), 2.95 (t, <i>J</i> = 7.3 Hz, 2H), 2.21 (dt, <i>J</i> = 13.3, 6.6 Hz, 2H), 2.13–1.96 (m, 4H), 1.94–1.36 (m, 17H), 1.07 (d, <i>J</i> = 6.9 Hz, 6H), 0.99–0.92 (m, 12H); MS (ESI): <i>m</i>/<i>z</i> calcd for [C<sub>38</sub>H<sub>73</sub>N<sub>10</sub>O<sub>7</sub>]<sup>2+</sup>/2 391.29, found 391.30.</div></div><div id="sec4_1_2_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i21" class="anchor-spacer"></div><h5 class="article-section__title" id="_i21"> (<i>S</i>)-6-(((<i>S</i>)-6-Amino-1-((<i>S</i>)-2-(((<i>S</i>)-1-(((<i>S</i>)-1-amino-5-((amino(iminio)methyl)amino)-1-oxopentan-2-yl)amino)-4-methyl-1-oxopentan-2-yl)carbamoyl)pyrrolidin-1-yl)-1-oxohexan-2-yl)amino)-5-((<i>S</i>)-2-(2-(2-aminoacetamido)acetamido)-3-methylbutanamido)-<i>N</i>,<i>N</i>,<i>N</i>-trimethyl-6-oxohexan-1-aminium (H3 Peptide: GGVKme3KPLR)</h5><div class="NLM_p last">White powder; yield 44%. <sup>1</sup>H NMR (MeOD-<i>d</i><sub>4</sub>, 400 MHz): 4.62 (dd, <i>J</i> = 8.3, 5.4 Hz, 1H), 4.45 (dd, <i>J</i> = 8.4, 4.3 Hz, 1H), 4.42–4.30 (m, 3H), 4.16 (d, <i>J</i> = 7.3 Hz, 1H), 4.05–3.91 (m, 2H), 3.84–3.79 (m, 1H), 3.76 (s, 2H), 3.68–3.59 (m, 1H), 3.21 (q, <i>J</i> = 7.6 Hz, 2H), 3.13 (s, 9H), 2.94 (t, <i>J</i> = 7.4 Hz, 2H), 2.27–2.18 (m, 1H), 2.04 (ddt, <i>J</i> = 25.1, 14.3, 7.4 Hz, 4H), 1.91–1.57 (m, 16H), 1.46 (ddd, <i>J</i> = 24.7, 11.6, 7.4 Hz, 4H), 0.95 (dd, <i>J</i> = 17.1, 6.5 Hz, 12H); MS (ESI): <i>m</i>/<i>z</i> calcd for [C<sub>41</sub>H<sub>80</sub>N<sub>14</sub>O<sub>8</sub>]<sup>+</sup> 895.63, found 895.50.</div></div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> Mixed Synthesis of Peptidomimetic Compounds</h4><div class="NLM_p">Peptidomimetic compounds were synthesized using an integrated approach of solid- and solution-phase synthesis. Kme3 mimetics were synthesized in solution as Fmoc-protected amino acids using reductive aminations as described below and added to the corresponding peptide by SPPS.</div><div id="sec4_1_3_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i23" class="anchor-spacer"></div><h5 class="article-section__title" id="_i23"> Protocol A</h5><div class="NLM_p last">Fmoc-<i>L</i>-lysine (1 equiv) was dissolved in ethanol and stirred at rt. Aldehyde and/or ketone (8 equiv) was added to the cloudy solution, and the mixture was left to stir for 10 min followed by addition of acetic acid (3 equiv). Sodium cyanoborohydride (5 equiv) was then added to the reaction. The mixture was stirred at rt for 2 h. The crude mixture was concentrated <i>in vacuo</i> and purified by reverse phase flash chromatography (H<sub>2</sub>O + 0.1% trifluoroacetic acid and CH<sub>3</sub>CN) to yield the final product as a TFA salt. The product was redissolved in water and lyophilized to a hygroscopic, white powder.</div></div><div id="sec4_1_3_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i24" class="anchor-spacer"></div><h5 class="article-section__title" id="_i24"> Protocol B</h5><div class="NLM_p last">Fmoc-<i>L</i>-lysine (1 equiv) was dissolved in ethanol and stirred at rt. Ketone (8 equiv) was added to the cloudy solution, and the mixture was left to stir for 10 min followed by addition of acetic acid (3 equiv). Sodium cyanoborohydride (5 equiv) was then added to the reaction. The mixture was stirred at 60 °C for 16 h. Aldehyde (8 equiv) and an additional 5 equiv of sodium cyanoborohydride were added to the reaction. The reaction mixture was again stirred at 60 °C for 16 h. The crude mixture was concentrated <i>in vacuo</i> and purified by reverse phase flash chromatography (H<sub>2</sub>O + 0.1% trifluoroacetic acid and CH<sub>3</sub>CN) to yield the final product as a TFA salt. The product was redissolved in water and lyophilized to a hygroscopic, white powder.</div></div><div id="sec4_1_3_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i25" class="anchor-spacer"></div><h5 class="article-section__title" id="_i25"> <i>N</i><sup>2</sup>-(((9<i>H</i>-Fluoren-9-yl)methoxy)carbonyl)-<i>N</i><sup>6</sup>-ethyl-<i>N</i><sup>6</sup>-isopropyl-<i>L</i>-lysine (<b>1</b>)</h5></div><div id="sec4_1_3_3_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i26" class="anchor-spacer"></div><h5 class="article-section__title" id="_i26"> Protocol A</h5><div class="NLM_p last">White powder; yield 50%. <sup>1</sup>H NMR (MeOD-<i>d</i><sub>4</sub>, 400 MHz): δ 7.76 (d, <i>J</i> = 7.5 Hz, 2H), 7.64 (t, <i>J</i> = 8.2 Hz, 2H), 7.36 (t, <i>J</i> = 7.4 Hz, 2H), 7.28 (td, <i>J</i> = 7.5, 1.2 Hz, 2H), 4.39–4.26 (m, 2H), 4.21–4.12 (m, 2H), 3.62 (p, <i>J</i> = 6.7 Hz, 1H), 3.21–2.88 (m, 4H), 1.90 (dtd, <i>J</i> = 13.5, 8.1, 7.7, 4.9 Hz, 1H), 1.73 (qd, <i>J</i> = 9.4, 5.8 Hz, 3H), 1.51–1.40 (m, 2H), 1.28 (q, <i>J</i> = 6.6, 6.2 Hz, 9H); MS (ESI): <i>m</i>/<i>z</i> calcd for [C<sub>26</sub>H<sub>34</sub>N<sub>2</sub>O<sub>4</sub>]<sup>+</sup> 439.25, found 439.20.</div></div><div id="sec4_1_3_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i27" class="anchor-spacer"></div><h5 class="article-section__title" id="_i27"> Amino(((<i>S</i>)-5-amino-4-((<i>S</i>)-2-((<i>S</i>)-1-((2<i>S</i>,5<i>S</i>,8<i>S</i>)-14-amino-2-(4-aminobutyl)-5-(4-(ethyl(isopropyl)amino)butyl)-8-isopropyl-4,7,10,13-tetraoxo-3,6,9,12-tetraazatetradecanoyl)pyrrolidine-2-carboxamido)-4-methylpentanamido)-5-oxopentyl)amino)methaniminium (UNC7643)</h5><div class="NLM_p last">White powder; yield 40%. <sup>1</sup>H NMR (MeOD-<i>d</i><sub>4</sub>, 400 MHz): δ 4.61 (dd, <i>J</i> = 8.3, 5.3 Hz, 1H), 4.45 (dd, <i>J</i> = 8.2, 4.1 Hz, 1H), 4.39–4.29 (m, 3H), 4.17 (d, <i>J</i> = 7.2 Hz, 1H), 3.99 (s, 2H), 3.83–3.77 (m, 1H), 3.76 (s, 2H), 3.69 (td, <i>J</i> = 15.3, 14.3, 6.5 Hz, 2H), 3.28–3.11 (m, 5H), 3.06 (dd, <i>J</i> = 16.0, 7.6 Hz, 1H), 2.94 (t, <i>J</i> = 7.4 Hz, 2H), 2.27–2.17 (m, 1H), 2.16–1.94 (m, 4H), 1.90–1.79 (m, 3H), 1.77–1.60 (m, 12H), 1.50 (p, <i>J</i> = 8.2 Hz, 4H), 1.37–1.32 (m, 9H), 0.99–0.92 (m, 12H); MS (ESI): <i>m</i>/<i>z</i> calcd for [C<sub>43</sub>H<sub>83</sub>N<sub>14</sub>O<sub>8</sub>]<sup>2+</sup> 924.65, found 924.50.</div></div><div id="sec4_1_3_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i28" class="anchor-spacer"></div><h5 class="article-section__title" id="_i28"> <i>N</i><sup>2</sup>-(((9<i>H</i>-Fluoren-9-yl)methoxy)carbonyl)-<i>N</i><sup>6</sup>-ethyl-<i>N</i><sup>6</sup>-(1-(furan-2-yl)ethyl)-<i>L</i>-lysine (<b>2</b>)</h5></div><div id="sec4_1_3_5_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i29" class="anchor-spacer"></div><h5 class="article-section__title" id="_i29"> Protocol B</h5><div class="NLM_p last">White powder; yield 40%. <sup>1</sup>H NMR (MeOD-<i>d</i><sub>4</sub>, 400 MHz): δ 7.80 (d, <i>J</i> = 7.6 Hz, 2H), 7.66 (q, <i>J</i> = 12.8, 10.9 Hz, 3H), 7.40 (t, <i>J</i> = 7.5 Hz, 2H), 7.31 (t, <i>J</i> = 7.3 Hz, 2H), 6.71 (s, 1H), 6.50 (d, <i>J</i> = 15.2 Hz, 1H), 4.44–4.31 (m, 2H), 4.20 (dt, <i>J</i> = 24.1, 5.7 Hz, 2H), 3.02 (dd, <i>J</i> = 97.5, 53.8 Hz, 4H), 1.90 (dd, <i>J</i> = 13.9, 6.3 Hz, 1H), 1.70 (t, <i>J</i> = 7.5 Hz, 7H), 1.43 (s, 2H), 1.31 (t, <i>J</i> = 6.7 Hz, 3H); MS (ESI): <i>m</i>/<i>z</i> calcd for [C<sub>29</sub>H<sub>34</sub>N<sub>2</sub>O<sub>5</sub>]<sup>+</sup> 491.25, found 491.25.</div></div><div id="sec4_1_3_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i30" class="anchor-spacer"></div><h5 class="article-section__title" id="_i30"> Amino(((4<i>S</i>)-5-amino-4-((2<i>S</i>)-2-((2<i>S</i>)-1-((2<i>S</i>,5<i>S</i>,8<i>S</i>)-14-amino-2-(4-aminobutyl)-5-(4-(ethyl(1-(furan-2-yl)ethyl)amino)butyl)-8-isopropyl-4,7,10,13-tetraoxo-3,6,9,12-tetraazatetradecanoyl)pyrrolidine-2-carboxamido)-4-methylpentanamido)-5-oxopentyl)amino)methaniminium (UNC6640)</h5><div class="NLM_p last">White powder; yield 25%. <sup>1</sup>H NMR (MeOD-<i>d</i><sub>4</sub>, 400 MHz): δ 7.68 (dt, <i>J</i> = 1.9, 0.9 Hz, 1H), 6.74 (d, <i>J</i> = 3.4 Hz, 1H), 6.54 (dd, <i>J</i> = 3.4, 1.9 Hz, 1H), 4.61 (dd, <i>J</i> = 8.2, 5.4 Hz, 1H), 4.45 (dd, <i>J</i> = 8.3, 4.0 Hz, 1H), 4.39–4.30 (m, 3H), 4.18 (d, <i>J</i> = 7.1 Hz, 1H), 3.99 (s, 2H), 3.80 (dt, <i>J</i> = 10.0, 6.5 Hz, 1H), 3.76 (s, 2H), 3.68–3.60 (m, 1H), 3.22 (tt, <i>J</i> = 14.3, 6.9 Hz, 4H), 3.14–3.05 (m, 1H), 2.94 (t, <i>J</i> = 7.4 Hz, 2H), 2.22 (dt, <i>J</i> = 15.4, 9.1 Hz, 1H), 2.12–1.95 (m, 4H), 1.94–1.57 (m, 19H), 1.55–1.38 (m, 4H), 1.33 (d, <i>J</i> = 8.2 Hz, 4H), 0.99–0.91 (m, 12H); MS (ESI): <i>m</i>/<i>z</i> calcd for [C<sub>46</sub>H<sub>83</sub>N<sub>14</sub>O<sub>9</sub>]<sup>2+</sup> 976.64, found 976.50.</div></div><div id="sec4_1_3_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i31" class="anchor-spacer"></div><h5 class="article-section__title" id="_i31"> <i>N</i><sup>2</sup>-(((9<i>H</i>-Fluoren-9-yl)methoxy)carbonyl)-<i>N</i><sup>6</sup>-isopropyl-<i>N</i><sup>6</sup>-phenethyl-<i>L</i>-lysine (<b>3</b>)</h5></div><div id="sec4_1_3_7_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i32" class="anchor-spacer"></div><h5 class="article-section__title" id="_i32"> Protocol B</h5><div class="NLM_p last">White, sticky solid; yield 42%. <sup>1</sup>H NMR (MeOD-<i>d</i><sub>4</sub>, 400 MHz): δ 7.68 (dt, <i>J</i> = 1.9, 0.9 Hz, 1H), 6.74 (d, <i>J</i> = 3.4 Hz, 1H), 6.54 (dd, <i>J</i> = 3.4, 1.9 Hz, 1H), 4.61 (dd, <i>J</i> = 8.2, 5.4 Hz, 1H), 4.45 (dd, <i>J</i> = 8.3, 4.0 Hz, 1H), 4.39–4.30 (m, 3H), 4.18 (d, <i>J</i> = 7.1 Hz, 1H), 3.99 (s, 2H), 3.80 (dt, <i>J</i> = 10.0, 6.5 Hz, 1H), 3.76 (s, 2H), 3.68–3.60 (m, 1H), 3.22 (tt, <i>J</i> = 14.3, 6.9 Hz, 4H), 3.14–3.05 (m, 1H), 2.94 (t, <i>J</i> = 7.4 Hz, 2H), 2.22 (dt, <i>J</i> = 15.4, 9.1 Hz, 1H), 2.12–1.95 (m, 4H), 1.94–1.57 (m, 19H), 1.55–1.38 (m, 4H), 1.33 (d, <i>J</i> = 8.2 Hz, 4H), 0.99–0.91 (m, 12H); MS (ESI): <i>m</i>/<i>z</i> calcd for [C<sub>32</sub>H<sub>38</sub>N<sub>2</sub>O<sub>4</sub>]<sup>+</sup> 515.28, found 515.30.</div></div><div id="sec4_1_3_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i33" class="anchor-spacer"></div><h5 class="article-section__title" id="_i33"> Amino(((<i>S</i>)-5-amino-4-((<i>S</i>)-2-((<i>S</i>)-1-((2<i>S</i>,5<i>S</i>,8<i>S</i>)-14-amino-2-(4-aminobutyl)-8-isopropyl-5-(4-(isopropyl(phenethyl)amino)butyl)-4,7,10,13-tetraoxo-3,6,9,12-tetraazatetradecanoyl)pyrrolidine-2-carboxamido)-4-methylpentanamido)-5-oxopentyl)amino)methaniminium (UNC6641)</h5><div class="NLM_p last">White powder; yield 21%. <sup>1</sup>H NMR (MeOD-<i>d</i><sub>4</sub>, 400 MHz): δ 7.38–7.25 (m, 5H), 4.61 (dd, <i>J</i> = 8.4, 5.2 Hz, 1H), 4.45 (dd, <i>J</i> = 8.3, 4.2 Hz, 1H), 4.39–4.29 (m, 3H), 4.17 (d, <i>J</i> = 7.2 Hz, 1H), 3.98 (d, <i>J</i> = 5.1 Hz, 2H), 3.83–3.74 (m, 4H), 3.65 (dd, <i>J</i> = 10.8, 4.0 Hz, 1H), 3.38 (dt, <i>J</i> = 17.9, 8.3 Hz, 1H), 3.27–3.12 (m, 4H), 3.06 (dq, <i>J</i> = 12.2, 6.5, 5.7 Hz, 2H), 2.94 (t, <i>J</i> = 7.4 Hz, 2H), 2.23 (ddd, <i>J</i> = 12.7, 8.2, 3.8 Hz, 1H), 2.15–1.93 (m, 4H), 1.93–1.55 (m, 16H), 1.51 (q, <i>J</i> = 7.8 Hz, 4H), 1.37 (dd, <i>J</i> = 6.7, 2.0 Hz, 6H), 1.02–0.88 (m, 12H); MS (ESI): <i>m</i>/<i>z</i> calcd for [C<sub>49</sub>H<sub>87</sub>N<sub>14</sub>O<sub>8</sub>]<sup>2+</sup>/2 500.32, found 500.40.</div></div><div id="sec4_1_3_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i34" class="anchor-spacer"></div><h5 class="article-section__title" id="_i34"> <i>N</i><sup>2</sup>-(((9<i>H</i>-Fluoren-9-yl)methoxy)carbonyl)-<i>N</i><sup>6</sup>-isopropyl-<i>N</i><sup>6</sup>-(3-phenylpropyl)-<i>L</i>-lysine (<b>4</b>)</h5></div><div id="sec4_1_3_9_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i35" class="anchor-spacer"></div><h5 class="article-section__title" id="_i35"> Protocol B</h5><div class="NLM_p last">White, sticky solid; yield 58%. <sup>1</sup>H NMR (MeOD-<i>d</i><sub>4</sub>, 400 MHz): δ 7.77 (d, <i>J</i> = 7.5 Hz, 2H), 7.65 (dd, <i>J</i> = 10.4, 7.5 Hz, 2H), 7.37 (t, <i>J</i> = 7.4 Hz, 2H), 7.30 (dd, <i>J</i> = 7.5, 1.2 Hz, 2H), 7.28–7.25 (m, 2H), 7.24–7.19 (m, 3H), 4.38 (dd, <i>J</i> = 10.5, 6.9 Hz, 1H), 4.30 (dd, <i>J</i> = 10.4, 7.1 Hz, 1H), 4.22–4.15 (m, 2H), 3.61 (p, <i>J</i> = 7.5, 7.0 Hz, 1H), 3.12–2.94 (m, 4H), 2.70–2.65 (m, 2H), 1.99 (s, 2H), 1.79–1.57 (m, 4H), 1.45 (dt, <i>J</i> = 15.0, 6.3 Hz, 2H), 1.26–1.23 (m, 6H); MS (ESI): <i>m</i>/<i>z</i> calcd for [C<sub>33</sub>H<sub>40</sub>N<sub>2</sub>O<sub>4</sub>]<sup>+</sup> 529.30, found 529.30.</div></div><div id="sec4_1_3_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i36" class="anchor-spacer"></div><h5 class="article-section__title" id="_i36"> Amino(((<i>S</i>)-5-amino-4-((<i>S</i>)-2-((<i>S</i>)-1-((2<i>S</i>,5<i>S</i>,8<i>S</i>)-14-amino-2-(4-aminobutyl)-8-isopropyl-5-(4-(isopropyl(3-phenylpropyl)amino)butyl)-4,7,10,13-tetraoxo-3,6,9,12-tetraazatetradecanoyl)pyrrolidine-2-carboxamido)-4-methylpentanamido)-5-oxopentyl)amino)methaniminium (UNC7259)</h5><div class="NLM_p last">White powder; yield 21%. <sup>1</sup>H NMR (MeOD-<i>d</i><sub>4</sub>, 400 MHz): δ 7.35–7.17 (m, 5H), 4.62 (dd, <i>J</i> = 8.5, 5.2 Hz, 1H), 4.48–4.42 (m, 1H), 4.35 (tt, <i>J</i> = 9.4, 5.2 Hz, 3H), 4.17 (d, <i>J</i> = 7.2 Hz, 1H), 4.00 (d, <i>J</i> = 3.0 Hz, 1H), 3.81–3.76 (m, 3H), 3.21 (p, <i>J</i> = 6.9 Hz, 3H), 3.16–3.05 (m, 4H), 2.96 (t, <i>J</i> = 7.3 Hz, 2H), 2.74 (t, <i>J</i> = 7.6 Hz, 2H), 2.25 (d, <i>J</i> = 7.5 Hz, 1H), 2.16–1.94 (m, 5H), 1.92–1.59 (m, 17H), 1.53–1.40 (m, 4H), 1.32 (dd, <i>J</i> = 6.6, 1.6 Hz, 6H), 1.29 (q, <i>J</i> = 2.3 Hz, 1H), 0.99–0.92 (m, 12H); MS (ESI): <i>m</i>/<i>z</i> calcd for [C<sub>50</sub>H<sub>89</sub>N<sub>14</sub>O<sub>8</sub>]<sup>+</sup> 1013.70, found 1013.50.</div></div><div id="sec4_1_3_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i37" class="anchor-spacer"></div><h5 class="article-section__title" id="_i37"> <i>N</i><sup>2</sup>-(((9<i>H</i>-Fluoren-9-yl)methoxy)carbonyl)-<i>N</i><sup>6</sup>-benzyl-<i>N</i><sup>6</sup>-isopropyl-<i>L</i>-lysine (<b>5</b>)</h5></div><div id="sec4_1_3_11_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i38" class="anchor-spacer"></div><h5 class="article-section__title" id="_i38"> Protocol B</h5><div class="NLM_p last">White, sticky solid; yield 53%. <sup>1</sup>H NMR (MeOD-<i>d</i><sub>4</sub>, 400 MHz): δ 7.46 (s, 6H), 7.29 (d, <i>J</i> = 3.9 Hz, 7H), 5.31–5.26 (m, 1H), 5.17 (dd, <i>J</i> = 12.1, 1.9 Hz, 1H), 4.36 (dd, <i>J</i> = 13.1, 1.9 Hz, 1H), 4.11 (dd, <i>J</i> = 13.3, 10.3 Hz, 1H), 3.89 (dd, <i>J</i> = 13.7, 3.9 Hz, 2H), 3.59 (dd, <i>J</i> = 13.1, 6.6 Hz, 3H), 1.79–1.66 (m, 2H), 1.52 (d, <i>J</i> = 12.6 Hz, 1H), 1.40 (d, <i>J</i> = 6.6 Hz, 3H), 1.38–1.29 (m, 5H), 1.25–1.16 (m, 1H); MS (ESI): <i>m</i>/<i>z</i> calcd for [C<sub>31</sub>H<sub>36</sub>N<sub>2</sub>O<sub>4</sub>]<sup>+</sup> 501.27, found 501.30.</div></div><div id="sec4_1_3_12" class="NLM_sec NLM_sec_level_4"><div id="ac_i39" class="anchor-spacer"></div><h5 class="article-section__title" id="_i39"> Amino(((<i>S</i>)-5-amino-4-((<i>S</i>)-2-((<i>S</i>)-1-((2<i>S</i>,5<i>S</i>,8<i>S</i>)-14-amino-2-(4-aminobutyl)-5-(4-(benzyl(isopropyl)amino)butyl)-8-isopropyl-4,7,10,13-tetraoxo-3,6,9,12-tetraazatetradecanoyl)pyrrolidine-2-carboxamido)-4-methylpentanamido)-5-oxopentyl)amino)methaniminium (UNC7253)</h5><div class="NLM_p last">White powder; yield 12%. <sup>1</sup>H NMR (MeOD-<i>d</i><sub>4</sub>, 400 MHz): δ 7.57–7.45 (m, 5H), 4.60 (dd, <i>J</i> = 8.5, 5.2 Hz, 1H), 4.48–4.23 (m, 6H), 4.16 (d, <i>J</i> = 6.9 Hz, 1H), 3.98 (s, 6H), 3.69 (dt, <i>J</i> = 13.8, 6.8 Hz, 1H), 3.27–3.12 (m, 3H), 3.10–2.98 (m, 1H), 2.94 (t, <i>J</i> = 7.4 Hz, 2H), 2.22 (dt, <i>J</i> = 7.6, 3.3 Hz, 1H), 2.14–1.54 (m, 19H), 1.50 (q, <i>J</i> = 7.8 Hz, 2H), 1.39 (td, <i>J</i> = 19.3, 6.3 Hz, 8H), 0.94 (dd, <i>J</i> = 16.8, 6.3 Hz, 12H); MS (MALDI): <i>m</i>/<i>z</i> calcd for [C<sub>48</sub>H<sub>85</sub>N<sub>14</sub>O<sub>8</sub>]<sup>+</sup> 985.67, found 985.62.</div></div><div id="sec4_1_3_13" class="NLM_sec NLM_sec_level_4"><div id="ac_i40" class="anchor-spacer"></div><h5 class="article-section__title" id="_i40"> <i>N</i><sup>2</sup>-(((9<i>H</i>-Fluoren-9-yl)methoxy)carbonyl)-<i>N</i><sup>6</sup>-(2-cyclohexylethyl)-<i>N</i><sup>6</sup>-isopropyl-<i>L</i>-lysine (6)</h5></div><div id="sec4_1_3_13_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i41" class="anchor-spacer"></div><h5 class="article-section__title" id="_i41"> Protocol B</h5><div class="NLM_p last">White powder; yield 62%. <sup>1</sup>H NMR (MeOD-<i>d</i><sub>4</sub>, 400 MHz): δ 7.76 (t, <i>J</i> = 6.6 Hz, 2H), 7.68–7.60 (m, 2H), 7.36 (t, <i>J</i> = 7.6 Hz, 2H), 7.28 (t, <i>J</i> = 7.5 Hz, 2H), 4.31 (dtd, <i>J</i> = 17.6, 10.4, 9.0, 4.1 Hz, 2H), 4.19 (q, <i>J</i> = 7.5, 7.0 Hz, 2H), 3.59 (dt, <i>J</i> = 13.1, 6.8 Hz, 1H), 3.01 (s, 4H), 2.13 (d, <i>J</i> = 6.7 Hz, 1H), 1.98–1.86 (m, 1H), 1.86–1.32 (m, 19H), 1.23–1.08 (m, 4H); MS (ESI): <i>m</i>/<i>z</i> calcd for [C<sub>32</sub>H<sub>44</sub>N<sub>2</sub>O<sub>4</sub>]<sup>+</sup> 521.33, found 521.35.</div></div><div id="sec4_1_3_14" class="NLM_sec NLM_sec_level_4"><div id="ac_i42" class="anchor-spacer"></div><h5 class="article-section__title" id="_i42"> Amino(((<i>S</i>)-5-amino-4-((<i>S</i>)-2-((<i>S</i>)-1-((2<i>S</i>,5<i>S</i>,8<i>S</i>)-14-amino-2-(4-aminobutyl)-5-(4-((2-cyclohexylethyl)(isopropyl)amino)butyl)-8-isopropyl-4,7,10,13-tetraoxo-3,6,9,12-tetraazatetradecanoyl)pyrrolidine-2-carboxamido)-4-methylpentanamido)-5-oxopentyl)amino)methaniminium (UNC7258)</h5><div class="NLM_p last">White powder; yield 10%. <sup>1</sup>H NMR (MeOD-<i>d</i><sub>4</sub>, 400 MHz): δ 4.48–4.44 (m, 3H), 4.35 (ddd, <i>J</i> = 14.8, 9.0, 5.6 Hz, 4H), 4.18 (d, <i>J</i> = 7.2 Hz, 1H), 4.01 (d, <i>J</i> = 1.9 Hz, 1H), 3.80 (d, <i>J</i> = 5.9 Hz, 3H), 3.50–3.46 (m, 1H), 3.22 (q, <i>J</i> = 6.8 Hz, 2H), 3.18–3.03 (m, 4H), 2.97 (q, <i>J</i> = 7.3 Hz, 2H), 2.03–1.95 (m, 2H), 1.86 (s, 2H), 1.73 (d, <i>J</i> = 21.8 Hz, 14H), 1.69–1.61 (m, 7H), 1.51 (s, 4H), 1.35 (d, <i>J</i> = 6.5 Hz, 6H), 1.33–1.21 (m, 6H), 1.05 (d, <i>J</i> = 11.7 Hz, 2H), 1.00–0.92 (m, 12H); MS (MALDI): <i>m</i>/<i>z</i> calcd for [C<sub>49</sub>H<sub>93</sub>N<sub>14</sub>O<sub>8</sub>]<sup>+</sup> 1005.73, found 1005.65.</div></div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> Synthesis of Biotin-UNC6641</h4><div class="NLM_p">UNC6641 was first coupled on-bead to propargylamine to provide a clickable handle for attachment of azido-PEG5-biotin. The peptide was then capped with an additional glycine residue using SPPS as described. Following cleavage from the resin, the alkynyl peptide was coupled to PEG5-biotin using click chemistry.</div><div id="sec4_1_4_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i44" class="anchor-spacer"></div><h5 class="article-section__title" id="_i44"> Amino(((<i>S</i>)-5-amino-4-((<i>S</i>)-2-((<i>S</i>)-1-((2<i>S</i>,5<i>S</i>,8<i>S</i>)-2-(4-aminobutyl)-18-glycyl-8-isopropyl-5-(4-(isopropyl(phenethyl)amino)butyl)-4,7,10,13,16-pentaoxo-3,6,9,12,15,18-hexaazahenicos-20-ynoyl)pyrrolidine-2-carboxamido)-4-methylpentanamido)-5-oxopentyl)amino)methaniminium (alkynyl-UNC6641)</h5><div class="NLM_p last">To UNC6641 (0.215 g, 0.215 mmol, 1 equiv) on resin was added 1 M bromoacetic acid in DMF (2.5 mL, 12 equiv) and 1 M diisopropylmethanediimine in DMF (2.5 mL, 12 equiv). The mixture was stirred on the shaker at rt for 30 min after which the solution was expelled and the resin was washed with DMF 7× (5 mL each). One M propargylamine (5 mL, 27 equiv) was then added to the resin and stirred on the shaker at rt for 2 h. Again, the solution was expelled and the beads were washed with DMF 7× (5 mL each). The terminal glycine was added and the final peptide cleaved and purified as described in the <a class="ref internalNav" href="#sec4_1_2" aria-label="Solid-Phase Peptide Synthesis (SPPS)">Solid-Phase Peptide Synthesis (SPPS)</a> section to yield a white solid (20 mg, 15%). MS (ESI): <i>m</i>/<i>z</i> calcd for [C<sub>56</sub>H<sub>95</sub>N<sub>16</sub>O<sub>10</sub>]<sup>+</sup> 1152.74, found 1152.60.</div></div><div id="sec4_1_4_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i45" class="anchor-spacer"></div><h5 class="article-section__title" id="_i45"> Amino(((<i>S</i>)-5-amino-4-((<i>S</i>)-2-((<i>S</i>)-1-((2<i>S</i>,5<i>S</i>,8<i>S</i>)-20-amino-2-(4-aminobutyl)-8-isopropyl-5-(4-(isopropyl(phenethyl)amino)butyl)-4,7,10,13,16,19-hexaoxo-18-((1-(19-oxo-23-((3a<i>R</i>,4<i>R</i>,6a<i>S</i>)-2-oxohexahydro-1<i>H</i>-thieno[3,4-<i>d</i>]imidazol-4-yl)-3,6,9,12,15-pentaoxa-18-azatricosyl)-1<i>H</i>-1,2,3-triazol-5-yl)methyl)-3,6,9,12,15,18-hexaazaicosanoyl)pyrrolidine-2-carboxamido)-4-methylpentanamido)-5-oxopentyl)amino)methaniminium (Biotin-UNC6641)</h5><div class="NLM_p last">Alkynyl-UNC6641 (0.014 g, 0.012 mmol, 1 equiv) was dissolved in DMF (0.2 mL). Aqueous solutions of 0.6 M copper(II) sulfate (0.2 mL, 10 equiv) and 0.12 M sodium ascorbate (1 mL, 10 equiv) were added followed by <i>N</i>-(17-azido-3,6,9,12,15-pentaoxaheptadecyl)-5-((3a<i>S</i>,4<i>S</i>,6a<i>R</i>)-2-oxohexahydro-1<i>H</i>-thieno[3,4-<i>d</i>]imidazol-4-yl)pentanamide. The mixture was stirred at rt for 2 h. The solution was then filtered and concentrated <i>in vacuo</i>, followed by purification via preparative HPLC as described in the <a class="ref internalNav" href="#sec4_1_1" aria-label="General Procedures">General Procedures</a> section. The solvent was removed <i>in vacuo</i>, redissolved in water, and lyophilized to yield the desired products as a white solid (0.018 g, 89%). <sup>1</sup>H NMR (MeOD-<i>d</i><sub>4</sub>, 400 MHz): δ 7.38–7.25 (m, 5H), 4.71 (d, <i>J</i> = 5.0 Hz, 2H), 4.64–4.53 (m, 3H), 4.47 (ddd, <i>J</i> = 16.4, 8.1, 4.2 Hz, 2H), 4.39–4.29 (m, 4H), 4.21 (d, <i>J</i> = 21.9 Hz, 3H), 4.12 (dd, <i>J</i> = 7.4, 2.5 Hz, 1H), 3.98 (s, 3H), 3.92–3.85 (m, 4H), 3.79 (p, <i>J</i> = 6.6 Hz, 2H), 3.66–3.58 (m, 18H), 3.53 (t, <i>J</i> = 5.5 Hz, 2H), 3.35 (d, <i>J</i> = 1.8 Hz, 4H), 3.21 (q, <i>J</i> = 7.2 Hz, 4H), 3.07 (dt, <i>J</i> = 11.4, 7.0 Hz, 2H), 2.93 (dt, <i>J</i> = 10.2, 6.1 Hz, 3H), 2.70 (d, <i>J</i> = 12.8 Hz, 1H), 2.22 (t, <i>J</i> = 7.3 Hz, 3H), 2.16–1.91 (m, 4H), 1.91–1.61 (m, 18H), 1.60–1.42 (m, 8H), 1.38 (d, <i>J</i> = 6.7 Hz, 6H), 0.99–0.91 (m, 12H); MS (ESI): <i>m</i>/<i>z</i> calcd for [C<sub>78</sub>H<sub>135</sub>N<sub>22</sub>O<sub>17</sub>S]<sup>2+</sup> 1685.91, found 1685.60.</div></div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> Protein Expression and Purification</h3><div id="sec4_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> Unlabeled Protein Expression Constructs</h4><div class="NLM_p">The Tudor domains of PHF1 (residues 25–87 of NP_077084) and PHF19 (residues 49–115 of NP_001273769) were expressed with N-terminal His tags in pET28 expression vectors. PHF1 Tudor domain point mutants were generated using a QuickChange II Site-Directed Mutagenesis kit (Agilent, Santa Clara, CA) and the following primer sets:</div><div class="NLM_array scrollable-table-wrap"><table class="table"><tbody><row>W41A5′-agcagcccatcagtcgctctggccagcacatc-3′</row><row>5′-gatgtgctggccagagcgactgatgggctgct-3′</row><row>Y47A5′-ctttttgatggtacccaaggctagcagcccatcagtccat-3′</row><row>5′-atggactgatgggctgctagccttgggtaccatcaaaaag-3</row><row>F65A5′-aactgcgaatcatcctcagcctggaccagacacacctc-3′</row><row>5′-gaggtgtgtctggtccaggctgaggatgattcgcagtt-3′</row></tbody></table></div><div class="NLM_p last">The Tudor domain of 53BP1 (residues 1484–1603 of NP_005648) was expressed with an N-terminal His tag in a pET15 expression vector. The Tudor domains of MTF2 (residues 1–160 of NP_031384), PHF20 (residues 58–148 of NP_057520), and PHF20L1 (residues 1–150 of NP_057102) were expressed with N-terminal GST tags in pGEX expression vectors.</div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> Unlabeled Protein Expression and Purification</h4><div class="NLM_p">All expression constructs were transformed into Rosetta2 BL21(DE3)pLysS competent cells (Novagen, EMD Chemicals, San Diego, CA). Protein expression was induced by growing cells at 37 °C with shaking until the OD600 reached ∼0.6–0.8 at which time the temperature was lowered to 18 °C and expression was induced by adding 0.5 mM IPTG and continuing shaking overnight. Cells were harvested by centrifugation, and pellets were stored at −80 °C.</div><div class="NLM_p">His-tagged proteins were purified by resuspending thawed cell pellets in 30 mL of lysis buffer (50 mM sodium phosphate pH 7.2, 50 mM NaCl, 30 mM imidazole, 1× EDTA free protease inhibitor cocktail (Roche Diagnostics, Indianapolis, IN)) per liter of culture. Cells were lysed on ice by sonication with a Branson Digital 450 Sonifier (Branson Ultrasonics, Danbury, CT) at 40% amplitude for 12 cycles with each cycle consisting of a 20 s pulse followed by a 40 s rest. The cell lysate was clarified by centrifugation and loaded onto a HisTrap FF column (GE Healthcare, Piscataway, NJ) that had been pre-equilibrated with 10 column volumes of binding buffer (50 mM sodium phosphate pH 7.2, 500 mM NaCl, 30 mM imidazole) using an AKTA FPLC (GE Healthcare, Piscataway, NJ). The column was washed with 15 column volumes of binding buffer, and protein was eluted in a linear gradient to 100% elution buffer (50 mM sodium phosphate pH 7.2, 500 mM NaCl, 500 mM imidazole) over 20 column volumes. Peak fractions containing the desired protein were pooled and concentrated to 2 mL in Amicon Ultra-15 concentrators with a 3000 molecular weight cutoff (Merck Millipore, Carrigtwohill Co. Cork IRL). Concentrated protein was loaded onto a HiLoad 26/60 Superdex 75 prep grade column (GE Healthcare, Piscataway, NJ) that had been pre-equilibrated with 1.2 column volumes of sizing buffer (25 mM Tris pH 7.5, 250 mM NaCl, 2 mM DTT, 5% glycerol) using an ATKA Purifier (GE Healthcare, Piscataway, NJ). Protein was eluted isocratically in sizing buffer over 1.3 column volumes at a flow rate of 2 mL/min collecting 3 mL fractions. Peak fractions were analyzed for purity by SDS-PAGE, and those containing pure protein were pooled and concentrated using Amicon Ultra-15 concentrators with a 3000 molecular weight cutoff (Merck Millipore, Carrigtwohill Co. Cork IRL).</div><div class="NLM_p last">GST-tagged proteins were purified by resuspending thawed cell pellets in 30 mL of lysis buffer (1× PBS, 5 mM DTT, 1× EDTA free protease inhibitor cocktail (Roche Diagnostics, Indianapolis, IN)) per liter of culture. Cells were lysed on ice by sonication as described for His-tagged proteins. Clarified cell lysate was loaded onto a GSTrap FF column (GE Healthcare, Piscataway, NJ) that had been pre-equilibrated with 10 column volumes of binding buffer (1× PBS, 5 mM DTT) using a AKTA FPLC (GE Healthcare, Piscataway, NJ). The column was washed with 10 column volumes of binding buffer, and protein was eluted in 100% elution buffer (50 mM Tris pH 7.5, 150 mM NaCl, 10 mM reduced glutathione) over 10 column volumes. Peak fractions containing the desired protein were pooled and concentrated to 2 mL in Amicon Ultra-15 concentrators with a 10,000 molecular weight cutoff (Merck Millipore, Carrigtwohill Co. Cork IRL). Concentrated protein was loaded onto a HiLoad 26/60 Superdex 200 prep grade column (GE Healthcare, Piscataway, NJ) that had been pre-equilibrated with 1.2 column volumes of sizing buffer (25 mM Tris pH 7.5, 250 mM NaCl, 2 mM DTT, 5% glycerol) using an ATKA FPLC (GE Healthcare, Piscataway, NJ). Protein was eluted isocratically in sizing buffer over 1.3 column volumes at a flow rate of 2 mL/min collecting 3 mL fractions. Peak fractions were analyzed for purity by SDS-PAGE, and those containing pure protein were pooled and concentrated using Amicon Ultra-15 concentrators with a 10,000 molecular weight cutoff (Merck Millipore, Carrigtwohill Co. Cork IRL).</div></div><div id="sec4_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> GST Tag Removal</h4><div class="NLM_p last">The N-terminal GST tag was removed from MTF2, PHF20, and PHF20L1 by thrombin cleavage according to the manufacturer’s recommendations (Novagen, EMD Chemicals, San Diego, CA). Briefly, purified protein was incubated with biotinylated thrombin at a final concentration of 1 unit of thrombin/mg of tagged protein for 16 h at 4 °C. The cleavage reaction was then passed over a GSTrap FF column to remove protein that still retained the tag and free GST. Tag free proteins were further purified by collecting and concentrating the column flow through and running over a HiLoad 26/60 Superdex 75 prep grade column as described above. Peak fractions containing the desired protein were pooled and concentrated using Amico Ultra-15 concentrators with a 3000 molecular weight cutoff (Merck Millipore, Carrigtwohill Co. Cork IRL). All proteins were dialyzed into a buffer containing 25 mM Tris–HCl pH 7.5, 150 mM NaCl, and 2 mM β-mercaptoethanol prior to use for ITC.</div></div><div id="sec4_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> <sup>15</sup>N-Labeled Protein Expression and Purification</h4><div class="NLM_p last">The PHF1 Tudor domain constructs comprising residues 14–87 and 28–87 were expressed and purified as described previously.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Briefly, the proteins were expressed in <i>Escherichia coli</i> BL21 (DE3) RIL cells grown in LB media or M9 minimal media supplemented with <sup>15</sup>NH<sub>4</sub>Cl. Expression was induced by 0.2 mM IPTG with shaking at 16 °C for 20 h. Cells were harvested and lysed by sonication. The GST-tagged PHF1 Tudor proteins were purified on glutathione Sepharose 4B beads (GE Healthcare) in 20 mM Tris (pH 7.5) buffer supplemented with 200 mM NaCl, 5 mM DTT, and 1 mM phenylmethanesulfonyl fluoride. The GST tag was removed overnight at 4 °C with PreScission protease. Proteins were further purified by size exclusion chromatography.</div></div><div id="sec4_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> Isothermal Titration Calorimetry (ITC) Experiments</h4><div class="NLM_p last">All ITC measurements were recorded at 25 °C with an Auto-iTC200 isothermal titration calorimeter (MicroCal Inc., USA). All protein and compound stock samples were stored in ITC buffer (25 mM Tris–HCl pH 7.5, 150 mM NaCl, and 2 mM β-mercaptoethanol) and then diluted to achieve the desired concentrations. Typically, 100 μM protein and 1 mM compound were used; variations in these concentrations always maintained a 10:1 compound to protein ratio for all ITC experiments. The concentration of the protein stock solution was established using the Edelhoch method, whereas compound stock solutions were prepared based on mass. The experimental protocol included a single 0.2 mL compound injection into a 200 mL cell filled with protein, followed by 26 subsequent 1.5 mL injections of compound. Injections were performed with a spacing of 180 s and a reference power of 8 cal/s. The initial data point was routinely deleted. The titration data was analyzed using Origin 7 Software (MicroCal Inc., USA) by the nonlinear least-squares method, fitting the heats of binding as a function of the compound to protein ratio to a one-site binding model. At these concentrations, a reasonable curve could be fit up to <i>K</i><sub>d</sub> ∼ 100 μM. Subsequently, curves too weak to be fit were reported as >100 μM.</div></div><div id="sec4_2_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> General Time-Resolved Fluorescence Resonance Energy Transfer (TR-FRET) Assay Protocol</h4><div class="NLM_p">The general TR-FRET protocol for Kme readers was followed as described previously.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Assays were run using white, low-volume, flat-bottom, nonbinding, 384-well microplates (Greiner, 784904) containing a total assay volume of 10 μL per well. The assay buffer was composed of 20 mM Tris (pH 7.5), 150 mM NaCl, 0.05% Tween 20, and 2 mM DTT. LANCE Europium (Eu)-W1024 Streptavidin conjugate (2 nM) and LANCE <i>Ultra</i> U<i>Light</i>-anti6x-His antibody (10 nM) were used as donor and acceptor fluorophores associated with the tracer ligand and protein, respectively. Final assay concentrations of 60 nM 6× histidine tagged PHF1 Tudor protein (residues 28–87, N-terminal tag) and 90 nM biotin-UNC6641 tracer ligand were used for final compound testing. For PHF19, final assay concentrations of 5 nM 6× histidine tagged PHF19 Tudor protein (residues 49–115, N-terminal tag) and 50 nM of biotin-UNC6641 tracer ligand were used. Assay performance was evaluated using the <i>Z</i>′ factor calculation at varying DMSO concentrations up to 3%. Low signals were obtained using 100 μM UNC6641 to obtain complete inhibition, and high signals were obtained without compound (DMSO only). The <i>Z</i>′ factors for PHF1 and PHF19 were consistent at each DMSO concentration, revealing a DMSO tolerance of up to 3% for both proteins (PHF1 <i>Z</i>′ at 1% = 0.81, at 3% = 0.75; PHF19 <i>Z</i>′ at 1% = 0.90, at 3% = 0.90).</div><div class="NLM_p last">Peptides were tested in a high-throughput manner using 384-well assay-ready plates in standard plate format: columns 1 and 2 were used for low-signal controls (100% inhibition with UNC6641), columns 23 and 24 were used for high-signal controls (DMSO only), and columns 3–22 were used for 16-point, 3-fold serial dilutions of each compound. First, controls were added to a mother plate where columns 1 and 2 were filled with 10 mM stock of UNC6641 in DMSO and columns 23 and 24 were filled with DMSO. Test compounds were dispensed across the mother plate at 100× concentration in columns 3–22 using a TECAN Freedom EVO liquid handling workstation. Using a TTP Labtech Mosquito HTS liquid handling instrument, assay-ready plates were prepared by stamping 100 nL of control compound into columns 1 and 2, 10 nL of compounds from the mother plate into columns 3–22, and 25 nL of DMSO into columns 23 and 24. Protein, biotinylated tracer ligand, and the TR-FRET reagents were added together and gently mixed by pipetting and rocking. Ten μL was then added to each well of an assay-ready plate using a Multidrop Combi (Thermo Fisher, Waltham, MA). Percent inhibition was calculated on a scale of 0% (i.e., activity with DMSO vehicle only) to 100% (100 μM UNC6641) from the full column controls on each plate.</div></div><div id="sec4_2_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> Molecular Docking</h4><div class="NLM_p last">Molecular docking was performed using Maestro version 12.6.149 from Schrödinger Suite 2020-4 (Schrödinger Inc.). PHF1 Tudor (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HCZ">4HCZ</a>) was prepared for docking using Prime tools followed by OPLS3e force field minimization after deletion of water molecules. Ligands were uploaded as SDF structures and prepared for docking using LigPrep, which ionizes and standardizes molecules at pH 7.0 ± 2.0. For peptides, stereoisomer generation was limited to 5. The Glide docking grid was created using the H3K36me3 peptide from the original structure. Ligands were docked using standard precision peptide Glide docking with the backbone atoms constrained by the H3K36me3 peptide from the original structure. Ring-to-ring distances were measured from centroid to centroid, and ring-to-atom distances were measured from centroid to atom.</div></div><div id="sec4_2_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> HSQC NMR Spectroscopy</h4><div class="NLM_p last">NMR experiments were carried out at 298 K on a Varian INOVA 600 spectrometer as described.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> The NMR sample contained 0.2 mM uniformly <sup>15</sup>N-labeled PHF1 Tudor domain (residue 14–87) in 20 mM Tris (pH 6.8) buffer supplemented with 150 mM NaCl, 5 mM DTT, and 10% D<sub>2</sub>O. Binding was characterized by monitoring chemical shift changes in the proteins induced by UNC6641. Spectra were processed using NMRPipe.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a></div></div><div id="sec4_2_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> Crystallization, Data Collection, and Structure Determination</h4><div class="NLM_p last">The PHF1 Tudor domain (residue 28–87) was concentrated to 10 mg/mL in a buffer containing 20 mM HEPES (pH 7.0), 200 mM NaCl, and 1 mM TCEP. Protein was incubated with UNC6641 at a 1:2 molar ratio for 1 h before crystallization. The preliminary plate-cluster crystals were obtained using the sitting drop vapor diffusion method by mixing 1 μL of the protein–peptide solution with 1 μL of reservoir solution containing 0.01 M ZnCl<sub>2</sub>, 0.1 M MES pH 6.0, and 20% PEG 6000. The strike seeding method was applied in order to obtain single well diffracting crystals. The crystals were cryoprotected with addition of 20% glycerol before being flash-frozen in liquid nitrogen. X-ray diffraction data were collected at the Advanced Light Source beamline 4.2.2 administrated by the Molecular Biology Consortium. The data set was indexed, integrated, and scaled by the XDS package.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> The substituents (the phenethyl and isopropyl groups) on K4 of UNC6641 were generated using the Phenix eLBOW program.<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> The complex structure was determined by the Phaser-MR program in Phenix using the PHF1 Tudor domain-H3K36me3 peptide complex structure (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HCZ">4HCZ</a>)<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> as the search model. The initial models were built with Coot and refined using Phenix.<a onclick="showRef(event, 'ref55 ref56'); return false;" href="javascript:void(0);" class="ref ref55 ref56">(55,56)</a> The data collection and structure refinement statistics are summarized in <a href="/doi/suppl/10.1021/acs.jmedchem.1c00430/suppl_file/jm1c00430_si_001.pdf" class="ext-link">Supplementary Table 1</a>.</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i56"><a href="/doi/suppl/10.1021/acs.jmedchem.1c00430" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_80005" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_80005" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.1c00430?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00430</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Supporting figures and tables (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00430/suppl_file/jm1c00430_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00430/suppl_file/jm1c00430_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00430/suppl_file/jm1c00430_si_001.pdf">jm1c00430_si_001.pdf (4.68 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00430/suppl_file/jm1c00430_si_002.csv">jm1c00430_si_002.csv (1.9 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Protein Data Bank ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7LKY">7LKY</a> (UNC6641). Authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.1c00430" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_63486" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_63486" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tatiana G. Kutateladze</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacology, University of Colorado
School of Medicine, Aurora, Colorado 80045, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0001-7375-6990" title="Orcid link">https://orcid.org/0000-0001-7375-6990</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#a5d1c4d1ccc4cbc48bced0d1c4d1c0c9c4c1dfc0e5c6d0c4cbd6c6cdd0d1df8bc0c1d0"><span class="__cf_email__" data-cfemail="3e4a5f4a575f505f10554b4a5f4a5b525f5a445b7e5d4b5f504d5d564b4a44105b5a4b">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lindsey I. James</span> - <span class="hlFld-Affiliation affiliation">Center
for Integrative Chemical Biology and Drug Discovery, Division of Chemical
Biology and Medicinal Chemistry, UNC Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-6034-7116" title="Orcid link">https://orcid.org/0000-0002-6034-7116</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#ee8780898b9c838f80ae8b838f8782c09b808dc08b8a9b"><span class="__cf_email__" data-cfemail="70191e1715021d111e30151d11191c5e051e135e151405">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Isabelle A. Engelberg</span> - <span class="hlFld-Affiliation affiliation">Center
for Integrative Chemical Biology and Drug Discovery, Division of Chemical
Biology and Medicinal Chemistry, UNC Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jiuyang Liu</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacology, University of Colorado
School of Medicine, Aurora, Colorado 80045, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jacqueline L. Norris-Drouin</span> - <span class="hlFld-Affiliation affiliation">Center
for Integrative Chemical Biology and Drug Discovery, Division of Chemical
Biology and Medicinal Chemistry, UNC Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stephanie H. Cholensky</span> - <span class="hlFld-Affiliation affiliation">Center
for Integrative Chemical Biology and Drug Discovery, Division of Chemical
Biology and Medicinal Chemistry, UNC Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span>; 
    <span>Present Address:
                        S.H.C.: Center on Aging, University of Minnesota, Minneapolis, MN, 55455</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Samantha A. Ottavi</span> - <span class="hlFld-Affiliation affiliation">Center
for Integrative Chemical Biology and Drug Discovery, Division of Chemical
Biology and Medicinal Chemistry, UNC Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stephen V. Frye</span> - <span class="hlFld-Affiliation affiliation">Center
for Integrative Chemical Biology and Drug Discovery, Division of Chemical
Biology and Medicinal Chemistry, UNC Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-2415-2215" title="Orcid link">https://orcid.org/0000-0002-2415-2215</a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>I.A.E., J.L.: These authors contributed equally.</p><span class="author-information-subsection-header">Author Contributions</span><p>I.A.E. and L.I.J. developed the concept and designed the study. I.A.E. did peptide and peptidomimetic synthesis, ITC, TR-FRET, and Maestro docking. J.L. determined the crystal structure and performed NMR experiments. J.L.N.-D. and S.H.C. carried out protein preparation. S.A.O. contributed to peptide synthesis. I.A.E., J.L., T.G.K., and L.I.J. analyzed the data and wrote the manuscript. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e2753-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i61">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_87252" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_87252" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by the National Institute on Drug Abuse, US NIH (grant R61DA047023-01), to L.I.J., the National Institute of General Medical Sciences, US NIH (grant R01GM100919), to S.V.F., and US NIH (grants CA252707 and HL151334) to T.G.K. The authors thank Jay Nix and Hardin John for help with data collection and processing, Justin Rectenwald for assistance with development of the TR-FRET assay, and Bryce Hart, Devan Shell, and Jarod Waybright for reviewing the primary data supporting this manuscript.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">PTM</td><td class="NLM_def"><p class="first last">post-translational modification</p></td></tr><tr><td class="NLM_term">Kme</td><td class="NLM_def"><p class="first last">methyl-lysine</p></td></tr><tr><td class="NLM_term">PHF1</td><td class="NLM_def"><p class="first last">plant homeodomain finger protein 1</p></td></tr><tr><td class="NLM_term">PRC2</td><td class="NLM_def"><p class="first last">polycomb repressive complex 2</p></td></tr><tr><td class="NLM_term">H3K36me3</td><td class="NLM_def"><p class="first last">histone H3 lysine 36 trimethyl</p></td></tr><tr><td class="NLM_term">ITC</td><td class="NLM_def"><p class="first last">isothermal titration calorimetry</p></td></tr><tr><td class="NLM_term">TR-FRET</td><td class="NLM_def"><p class="first last">time-resolved fluorescence resonance transfer</p></td></tr><tr><td class="NLM_term">PCL</td><td class="NLM_def"><p class="first last">polycomb-like</p></td></tr><tr><td class="NLM_term">PHF19</td><td class="NLM_def"><p class="first last">plant homeodomain finger protein 19</p></td></tr><tr><td class="NLM_term">MTF2</td><td class="NLM_def"><p class="first last">metal response element binding transcription factor 2</p></td></tr><tr><td class="NLM_term">CSP</td><td class="NLM_def"><p class="first last">chemical shift perturbation</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i63">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_86431" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_86431" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 56 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Macdonald, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hathaway, N. A.</span></span> <span> </span><span class="NLM_article-title">Epigenetic Roots of Immunologic Disease and New Methods for Examining Chromatin Regulatory Pathways</span>. <i>Immunol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>93</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">261</span>– <span class="NLM_lpage">270</span>, <span class="refDoi"> DOI: 10.1038/icb.2014.105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1038%2Ficb.2014.105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=25533290" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFyhu7rF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=2015&pages=261-270&issue=3&author=I.+A.+Macdonaldauthor=N.+A.+Hathaway&title=Epigenetic+Roots+of+Immunologic+Disease+and+New+Methods+for+Examining+Chromatin+Regulatory+Pathways&doi=10.1038%2Ficb.2014.105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic roots of immunologic disease and new methods for examining chromatin regulatory pathways</span></div><div class="casAuthors">MacDonald, Ian A.; Hathaway, Nathaniel A.</div><div class="citationInfo"><span class="NLM_cas:title">Immunology & Cell Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">261-270</span>CODEN:
                <span class="NLM_cas:coden">ICBIEZ</span>;
        ISSN:<span class="NLM_cas:issn">0818-9641</span>.
    
            (<span class="NLM_cas:orgname">NPG Nature Asia-Pacific</span>)
        </div><div class="casAbstract">A review.  The ability to accurately quantitate and exptl. examine epigenetic modifications across the human genome has exploded in the past decade.  This has given rise to a wealth of new information concerning the contributions of epigenetic regulatory networks to the pathogenesis of human disease.  In particular, immunol. disorders have strong developmental roots in chromatin regulatory pathways.  In this review, we focus on the epigenetic signatures and new discoveries revealing the epigenetic compns. of specific immunol. cancers and autoimmune diseases.  We also comment on the conserved epigenetic roots among diverse immunol. disorders and suggest inhibition strategies that may be relevant for future treatment.  Finally, we highlight emerging exptl. tools with the capability to examine the mechanisms of chromatin regulatory enzymes with a high level of temporal control.  The knowledge of genetic and epigenetic defects in immunol. disease combined with new exptl. approaches will elucidate the contribution of individual enzymes in complex epigenetic regulatory networks.  This could lead to new diagnostic and therapeutic approaches for some very diverse and difficult to treat human diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9pqOiMk4yurVg90H21EOLACvtfcHk0ljWOjAOAeXSew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFyhu7rF&md5=8b76e7fd5b3ec62d39d69cc3cb0b37bd</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Ficb.2014.105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Ficb.2014.105%26sid%3Dliteratum%253Aachs%26aulast%3DMacdonald%26aufirst%3DI.%2BA.%26aulast%3DHathaway%26aufirst%3DN.%2BA.%26atitle%3DEpigenetic%2520Roots%2520of%2520Immunologic%2520Disease%2520and%2520New%2520Methods%2520for%2520Examining%2520Chromatin%2520Regulatory%2520Pathways%26jtitle%3DImmunol.%2520Cell%2520Biol.%26date%3D2015%26volume%3D93%26issue%3D3%26spage%3D261%26epage%3D270%26doi%3D10.1038%2Ficb.2014.105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mirabella, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foster, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartke, T.</span></span> <span> </span><span class="NLM_article-title">Chromatin Deregulation in Disease</span>. <i>Chromosoma</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>125</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">75</span>– <span class="NLM_lpage">93</span>, <span class="refDoi"> DOI: 10.1007/s00412-015-0530-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1007%2Fs00412-015-0530-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=26188466" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1WqtrfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2016&pages=75-93&issue=1&author=A.+C.+Mirabellaauthor=B.+M.+Fosterauthor=T.+Bartke&title=Chromatin+Deregulation+in+Disease&doi=10.1007%2Fs00412-015-0530-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatin deregulation in disease</span></div><div class="casAuthors">Mirabella, Anne C.; Foster, Benjamin M.; Bartke, Till</div><div class="citationInfo"><span class="NLM_cas:title">Chromosoma</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">75-93</span>CODEN:
                <span class="NLM_cas:coden">CHROAU</span>;
        ISSN:<span class="NLM_cas:issn">0009-5915</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  The regulation of chromatin by epigenetic mechanisms plays a central role in gene expression and is essential for development and maintenance of cell identity and function.  Aberrant chromatin regulation is obsd. in many diseases where it leads to defects in epigenetic gene regulation resulting in pathol. gene expression programs.  These defects are caused by inherited or acquired mutations in genes encoding enzymes that deposit or remove DNA and histone modifications and that shape chromatin architecture.  Chromatin deregulation often results in neurodevelopmental disorders and intellectual disabilities, frequently linked to phys. and developmental abnormalities, but can also cause neurodegenerative diseases, immunodeficiency, or muscle wasting syndromes.  Epigenetic diseases can either be of monogenic origin or manifest themselves as complex multifactorial diseases such as in congenital heart disease, autism spectrum disorders, or cancer in which mutations in chromatin regulators are contributing factors.  The environment directly influences the epigenome and can induce changes that cause or predispose to diseases through risk factors such as stress, malnutrition or exposure to harmful chems.  The plasticity of chromatin regulation makes targeting the enzymic machinery an attractive strategy for therapeutic intervention and an increasing no. of small mol. inhibitors against a variety of epigenetic regulators are in clin. use or under development.  In this review, we will give an overview of the mol. lesions that underlie epigenetic diseases, and we will discuss the impact of the environment and prospects for epigenetic therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrW99ySZUDhtLVg90H21EOLACvtfcHk0ljWOjAOAeXSew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1WqtrfK&md5=f19871296cb8383443499d2c7d78f527</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1007%2Fs00412-015-0530-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00412-015-0530-0%26sid%3Dliteratum%253Aachs%26aulast%3DMirabella%26aufirst%3DA.%2BC.%26aulast%3DFoster%26aufirst%3DB.%2BM.%26aulast%3DBartke%26aufirst%3DT.%26atitle%3DChromatin%2520Deregulation%2520in%2520Disease%26jtitle%3DChromosoma%26date%3D2016%26volume%3D125%26issue%3D1%26spage%3D75%26epage%3D93%26doi%3D10.1007%2Fs00412-015-0530-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Valencia, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadoch, C.</span></span> <span> </span><span class="NLM_article-title">Chromatin Regulatory Mechanisms and Therapeutic Opportunities in Cancer</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>21</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">152</span>– <span class="NLM_lpage">161</span>, <span class="refDoi"> DOI: 10.1038/s41556-018-0258-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1038%2Fs41556-018-0258-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=30602726" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC1MXlslKmsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2019&pages=152-161&issue=2&author=A.+M.+Valenciaauthor=C.+Kadoch&title=Chromatin+Regulatory+Mechanisms+and+Therapeutic+Opportunities+in+Cancer&doi=10.1038%2Fs41556-018-0258-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatin regulatory mechanisms and therapeutic opportunities in cancer</span></div><div class="casAuthors">Valencia, Alfredo M.; Kadoch, Cigall</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">152-161</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  Research over the past several decades has unmasked a major contribution of disrupted chromatin regulatory processes to human disease, particularly cancer.  Advances in genome-wide technologies have highlighted frequent mutations in genes encoding chromatin-assocd. proteins, identified unexpected synthetic lethal opportunities and enabled increasingly comprehensive structural and functional dissection.  Here, we review recent progress in our understanding of oncogenic mechanisms at each level of chromatin organization and regulation, and discuss new strategies towards therapeutic intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGog3wawAVdsr7Vg90H21EOLACvtfcHk0lhSwUF-w_Rtjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXlslKmsrY%253D&md5=fe9df9ef3fe0ffcf7e0e8ba377543251</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fs41556-018-0258-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41556-018-0258-1%26sid%3Dliteratum%253Aachs%26aulast%3DValencia%26aufirst%3DA.%2BM.%26aulast%3DKadoch%26aufirst%3DC.%26atitle%3DChromatin%2520Regulatory%2520Mechanisms%2520and%2520Therapeutic%2520Opportunities%2520in%2520Cancer%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2019%26volume%3D21%26issue%3D2%26spage%3D152%26epage%3D161%26doi%3D10.1038%2Fs41556-018-0258-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bird, A.</span></span> <span> </span><span class="NLM_article-title">The Methyl-CpG-Binding Protein MeCP2 and Neurological Disease</span>. <i>Biochem. Soc. Trans.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>36</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">575</span>– <span class="NLM_lpage">583</span>, <span class="refDoi"> DOI: 10.1042/BST0360575</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1042%2FBST0360575" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=18631120" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BD1cXoslCltrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2008&pages=575-583&issue=4&author=A.+Bird&title=The+Methyl-CpG-Binding+Protein+MeCP2+and+Neurological+Disease&doi=10.1042%2FBST0360575"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The methyl-CpG-binding protein MeCP2 and neurological disease</span></div><div class="casAuthors">Bird, Adrian</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Society Transactions</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">575-583</span>CODEN:
                <span class="NLM_cas:coden">BCSTB5</span>;
        ISSN:<span class="NLM_cas:issn">0300-5127</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">A review.  The methyl-CpG-binding protein MeCP2 was discovered over 15 years ago as part of a search for proteins that selectively bind methylated DNA.  It is a nuclear protein that is largely chromatin-bound and has a strong preference for binding to methylated DNA sequences in vivo.  Evidence from model systems shows that MeCP2 can recruit the Sin3a co-repressor complex to promoters leading to transcriptional repression, therefore suggesting that MeCP2 can interpret the DNA methylation signal to bring about gene silencing.  Mutations in the human MECP2 gene cause the autism spectrum disorder Rett Syndrome.  MeCP2 is most highly expressed in neurons, and mice lacking this protein show symptoms that strikingly parallel those of Rett patients.  Surprisingly, these symptoms are efficiently reversed by delayed activation of a stopped' Mecp2 gene, raising hopes that human Rett syndrome may also be reversible.  Future studies of MeCP2 promise to shed light upon brain function, neurol. disease and the biol. of DNA methylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPOd305g9ETrVg90H21EOLACvtfcHk0lhSwUF-w_Rtjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXoslCltrY%253D&md5=281fbc5aeb5d81318abad292ce8f9510</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1042%2FBST0360575&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBST0360575%26sid%3Dliteratum%253Aachs%26aulast%3DBird%26aufirst%3DA.%26atitle%3DThe%2520Methyl-CpG-Binding%2520Protein%2520MeCP2%2520and%2520Neurological%2520Disease%26jtitle%3DBiochem.%2520Soc.%2520Trans.%26date%3D2008%26volume%3D36%26issue%3D4%26spage%3D575%26epage%3D583%26doi%3D10.1042%2FBST0360575" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Strahl, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allis, C. D.</span></span> <span> </span><span class="NLM_article-title">The Language of Covalent Histone Modifications</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>403</i></span>,  <span class="NLM_fpage">41</span>– <span class="NLM_lpage">45</span>, <span class="refDoi"> DOI: 10.1038/47412</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1038%2F47412" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10638745" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A280%3ADC%252BD3c7gt1arsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=403&publication_year=2000&pages=41-45&author=B.+D.+Strahlauthor=C.+D.+Allis&title=The+Language+of+Covalent+Histone+Modifications&doi=10.1038%2F47412"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The language of covalent histone modifications</span></div><div class="casAuthors">Strahl B D; Allis C D</div><div class="citationInfo"><span class="NLM_cas:title">Nature</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">403</span>
        (<span class="NLM_cas:issue">6765</span>),
    <span class="NLM_cas:pages">41-5</span>
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">Histone proteins and the nucleosomes they form with DNA are the fundamental building blocks of eukaryotic chromatin.  A diverse array of post-translational modifications that often occur on tail domains of these proteins has been well documented.  Although the function of these highly conserved modifications has remained elusive, converging biochemical and genetic evidence suggests functions in several chromatin-based processes.  We propose that distinct histone modifications, on one or more tails, act sequentially or in combination to form a 'histone code' that is, read by other proteins to bring about distinct downstream events.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSyzhdlYh9sX7t7lRqEFUJVfW6udTcc2eZVnPCgaEzHY7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3c7gt1arsQ%253D%253D&md5=2eb7d093151c6122ccd70fa2adcfd94b</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2F47412&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F47412%26sid%3Dliteratum%253Aachs%26aulast%3DStrahl%26aufirst%3DB.%2BD.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DThe%2520Language%2520of%2520Covalent%2520Histone%2520Modifications%26jtitle%3DNature%26date%3D2000%26volume%3D403%26spage%3D41%26epage%3D45%26doi%3D10.1038%2F47412" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jenuwein, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allis, C. D.</span></span> <span> </span><span class="NLM_article-title">Translating the Histone Code</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>293</i></span>,  <span class="NLM_fpage">1074</span>– <span class="NLM_lpage">1080</span>, <span class="refDoi"> DOI: 10.1126/science.1063127</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1126%2Fscience.1063127" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=11498575" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BD3MXmtVWltro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=293&publication_year=2001&pages=1074-1080&author=T.+Jenuweinauthor=C.+D.+Allis&title=Translating+the+Histone+Code&doi=10.1126%2Fscience.1063127"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Translating the histone code</span></div><div class="casAuthors">Jenuwein, Thomas; Allis, C. David</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">293</span>
        (<span class="NLM_cas:issue">5532</span>),
    <span class="NLM_cas:pages">1074-1080</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The review.  Chromatin, the physiol. template of all eukaryotic genetic information, is subject to a diverse array of posttranslational modifications that largely impinge on histone amino termini, thereby regulating access to the underlying DNA.  Distinct histone amino-terminal modifications can generate synergistic or antagonistic interaction affinities for chromatin-assocd. proteins, which in turn dictate dynamic transitions between transcriptionally active or transcriptionally silent chromatin states.  The combinatorial nature of histone amino-terminal modifications thus reveals a "histone code" that considerably extends the information potential of the genetic code.  We propose that this epigenetic marking system represents a fundamental regulatory mechanism that has an impact on most, if not all; chromatin-templated processes, with far-reaching consequences for cell fate decisions and both normal and pathol. development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUFyHNepKcUbVg90H21EOLACvtfcHk0lgPz8wNajwYXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXmtVWltro%253D&md5=021dcd98a5e5bf2ed7eedef1897b2269</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1126%2Fscience.1063127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1063127%26sid%3Dliteratum%253Aachs%26aulast%3DJenuwein%26aufirst%3DT.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DTranslating%2520the%2520Histone%2520Code%26jtitle%3DScience%26date%3D2001%26volume%3D293%26spage%3D1074%26epage%3D1080%26doi%3D10.1126%2Fscience.1063127" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Musselman, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khorasanizadeh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutateladze, T. G.</span></span> <span> </span><span class="NLM_article-title">Towards Understanding Methyllysine Readout</span>. <i>Biochim. Biophys. Acta, Gene Regul. Mech.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>1839</i></span>,  <span class="NLM_fpage">686</span>– <span class="NLM_lpage">693</span>, <span class="refDoi"> DOI: 10.1016/j.bbagrm.2014.04.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1016%2Fj.bbagrm.2014.04.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=24727128" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC2cXntFGjtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1839&publication_year=2014&pages=686-693&author=C.+A.+Musselmanauthor=S.+Khorasanizadehauthor=T.+G.+Kutateladze&title=Towards+Understanding+Methyllysine+Readout&doi=10.1016%2Fj.bbagrm.2014.04.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Towards understanding methyllysine readout</span></div><div class="casAuthors">Musselman, Catherine A.; Khorasanizadeh, Sepideh; Kutateladze, Tatiana G.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Gene Regulatory Mechanisms</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">1839</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">686-693</span>CODEN:
                <span class="NLM_cas:coden">BBAGC6</span>;
        ISSN:<span class="NLM_cas:issn">1874-9399</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Lysine methylation is the most versatile covalent posttranslational modification (PTM) found in histones and non-histone proteins.  Over the past decade a no. of methyllysine-specific readers have been discovered and their interactions with histone tails have been structurally and biochem. characterized.  More recently innovative exptl. approaches have emerged that allow for studying reader interactions in the context of the full nucleosome and nucleosomal arrays.  In this review, we give a brief overview of the known mechanisms of histone lysine methylation readout, summarize progress recently made in exploring interactions with methylated nucleosomes, and discuss the latest advances in the development of small mol. inhibitors of the methyllysine-specific readers.  New studies reveal various reader-nucleosome contacts outside the methylated histone tail, thus offering a better model for assocn. of histone readers to chromatin and broadening our understanding of the functional implications of these interactions.  In addn., some progress has been made in the design of antagonists of these interactions.Specific lysine methylation patterns are commonly assocd. with certain chromatin states and genomic elements, and are linked to distinct biol. outcomes such as transcription activation or repression.  Disruption of patterns of histone modifications is assocd. with a no. of diseases, and there is tremendous therapeutic potential in targeting histone modification pathways.  Thus, investigating binding of readers of these modifications is not only important for elucidating fundamental mechanisms of chromatin regulation, but also necessary for the design of targeted therapeutics.  This article is part of a Special Issue entitled: Mol. mechanisms of histone modification function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoC_YadxhKu5LVg90H21EOLACvtfcHk0lgPz8wNajwYXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXntFGjtro%253D&md5=10c99cb631ecf00d8c7d941ed4d40fbb</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagrm.2014.04.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagrm.2014.04.001%26sid%3Dliteratum%253Aachs%26aulast%3DMusselman%26aufirst%3DC.%2BA.%26aulast%3DKhorasanizadeh%26aufirst%3DS.%26aulast%3DKutateladze%26aufirst%3DT.%2BG.%26atitle%3DTowards%2520Understanding%2520Methyllysine%2520Readout%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Gene%2520Regul.%2520Mech.%26date%3D2014%26volume%3D1839%26spage%3D686%26epage%3D693%26doi%3D10.1016%2Fj.bbagrm.2014.04.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Beaver, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waters, M. L.</span></span> <span> </span><span class="NLM_article-title">Molecular Recognition of Lys and Arg Methylation</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_fpage">643</span>– <span class="NLM_lpage">653</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00996</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00996" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVGitbjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&pages=643-653&author=J.+E.+Beaverauthor=M.+L.+Waters&title=Molecular+Recognition+of+Lys+and+Arg+Methylation&doi=10.1021%2Facschembio.5b00996"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Recognition of Lys and Arg Methylation</span></div><div class="casAuthors">Beaver, Joshua E.; Waters, Marcey L.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">643-653</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A network of reader proteins and enzymes precisely controls gene transcription through the dynamic addn., removal, and recognition of post-translational modifications (PTMs) of histone tails.  Histone PTMs work in concert with this network to regulate gene transcription through the histone code, and the dysregulation of PTM maintenance is linked to a large no. of diseases, including many types of cancer.  A wealth of research aims to elucidate the functions of this code, but our understanding of the effects of PTMs, specifically the methylation of lysine (Lys) and arginine (Arg), is lacking.  The development of new tools to study PTMs relies on a sophisticated understanding of the mechanisms that drive protein and small mol. recognition in water.  In this review, we outline the phys. org. concepts that drive the mol. recognition of Lys and Arg methylation by reader proteins and draw comparisons to the binding mechanisms of small mol. receptors for methylated Lys and Arg that have been developed recently.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPpKlWThh4vLVg90H21EOLACvtfcHk0lg3y_Yq5uhbzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVGitbjF&md5=f2bca0418453cad1a65626c2867617ba</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00996&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00996%26sid%3Dliteratum%253Aachs%26aulast%3DBeaver%26aufirst%3DJ.%2BE.%26aulast%3DWaters%26aufirst%3DM.%2BL.%26atitle%3DMolecular%2520Recognition%2520of%2520Lys%2520and%2520Arg%2520Methylation%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26spage%3D643%26epage%3D653%26doi%3D10.1021%2Facschembio.5b00996" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Taverna, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruthenburg, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allis, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, D. J.</span></span> <span> </span><span class="NLM_article-title">How Chromatin-Binding Modules Interpret Histone Modifications: Lessons from Professional Pocket Pickers</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">1025</span>– <span class="NLM_lpage">1040</span>, <span class="refDoi"> DOI: 10.1038/nsmb1338</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1038%2Fnsmb1338" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=17984965" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1Kjs7vE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2007&pages=1025-1040&author=S.+D.+Tavernaauthor=H.+Liauthor=A.+J.+Ruthenburgauthor=C.+D.+Allisauthor=D.+J.+Patel&title=How+Chromatin-Binding+Modules+Interpret+Histone+Modifications%3A+Lessons+from+Professional+Pocket+Pickers&doi=10.1038%2Fnsmb1338"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">How chromatin-binding modules interpret histone modifications: lessons from professional pocket pickers</span></div><div class="casAuthors">Taverna, Sean D.; Li, Haitao; Ruthenburg, Alexander J.; Allis, C. David; Patel, Dinshaw J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1025-1040</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Histones comprise the major protein component of chromatin, the scaffold in which the eukaryotic genome is packaged, and are subject to many types of post-translational modifications (PTMs), esp. on their flexible tails.  These modifications may constitute a 'histone code' and could be used to manage epigenetic information that helps extend the genetic message beyond DNA sequences.  This proposed code, read in part by histone PTM-binding 'effector' modules and their assocd. complexes, has been predicted to define unique functional states of chromatin and/or regulate various chromatin-templated processes.  A wealth of structural and functional data show how chromatin effector modules target their cognate covalent histone modifications.  Here, the authors summarize key features in the mol. recognition of histone PTMs by a diverse family of 'reader pockets' (including bromodomains, Royal superfamily modules, PHD fingers, WD40 protein repeats, 14-3-3 proteins, and BRCT domains), highlighting specific readout mechanisms for individual marks, common themes, and insights into the downstream functional consequences of the interactions.  Changes in these interactions may have far-reaching implications for human biol. and disease, notably cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGroaoP5OiDUOLVg90H21EOLACvtfcHk0lg3y_Yq5uhbzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1Kjs7vE&md5=4ff842f1ef5416b932a89c751707defd</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnsmb1338&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb1338%26sid%3Dliteratum%253Aachs%26aulast%3DTaverna%26aufirst%3DS.%2BD.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DRuthenburg%26aufirst%3DA.%2BJ.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26aulast%3DPatel%26aufirst%3DD.%2BJ.%26atitle%3DHow%2520Chromatin-Binding%2520Modules%2520Interpret%2520Histone%2520Modifications%253A%2520Lessons%2520from%2520Professional%2520Pocket%2520Pickers%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2007%26volume%3D14%26spage%3D1025%26epage%3D1040%26doi%3D10.1038%2Fnsmb1338" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Musselman, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lalonde, M.-E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Côté, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutateladze, T. G.</span></span> <span> </span><span class="NLM_article-title">Perceiving the Epigenetic Landscape through Histone Readers</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>19</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">1218</span>– <span class="NLM_lpage">1227</span>, <span class="refDoi"> DOI: 10.1038/nsmb.2436</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1038%2Fnsmb.2436" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=23211769" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslOktbbI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=1218-1227&issue=12&author=C.+A.+Musselmanauthor=M.-E.+Lalondeauthor=J.+C%C3%B4t%C3%A9author=T.+G.+Kutateladze&title=Perceiving+the+Epigenetic+Landscape+through+Histone+Readers&doi=10.1038%2Fnsmb.2436"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Perceiving the epigenetic landscape through histone readers</span></div><div class="casAuthors">Musselman, Catherine A.; Lalonde, Marie-Eve; Cote, Jacques; Kutateladze, Tatiana G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1218-1227</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Post-translational modifications (PTMs) of histones provide a fine-tuned mechanism for regulating chromatin structure and dynamics.  PTMs can alter direct interactions between histones and DNA and serve as docking sites for protein effectors, or readers, of these PTMs.  Binding of the readers recruits or stabilizes various components of the nuclear signaling machinery at specific genomic sites, mediating fundamental DNA-templated processes, including gene transcription and DNA recombination, replication and repair.  In this review, we highlight the latest advances in characterizing histone-binding mechanisms and identifying new epigenetic readers and summarize the functional significance of PTM recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIQ_OP4dJ93bVg90H21EOLACvtfcHk0lj_kxXJj9ukUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslOktbbI&md5=3eb922fae63c59ddea200c04f743a9ee</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.2436&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.2436%26sid%3Dliteratum%253Aachs%26aulast%3DMusselman%26aufirst%3DC.%2BA.%26aulast%3DLalonde%26aufirst%3DM.-E.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DJ.%26aulast%3DKutateladze%26aufirst%3DT.%2BG.%26atitle%3DPerceiving%2520the%2520Epigenetic%2520Landscape%2520through%2520Histone%2520Readers%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2012%26volume%3D19%26issue%3D12%26spage%3D1218%26epage%3D1227%26doi%3D10.1038%2Fnsmb.2436" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strahl, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutateladze, T. G.</span></span> <span> </span><span class="NLM_article-title">Insights into Newly Discovered Marks and Readers of Epigenetic Information</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">662</span>– <span class="NLM_lpage">668</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2149</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1038%2Fnchembio.2149" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=27538025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlOnsbbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=662-668&author=F.+H.+Andrewsauthor=B.+D.+Strahlauthor=T.+G.+Kutateladze&title=Insights+into+Newly+Discovered+Marks+and+Readers+of+Epigenetic+Information&doi=10.1038%2Fnchembio.2149"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Insights into newly discovered marks and readers of epigenetic information</span></div><div class="casAuthors">Andrews, Forest H.; Strahl, Brian D.; Kutateladze, Tatiana G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">662-668</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The field of chromatin biol. has been advancing at an accelerated pace.  Recent discoveries of previously uncharacterized sites and types of post-translational modifications (PTMs) and the identification of new sets of proteins responsible for the deposition, removal, and reading of these marks continue raising the complexity of an already exceedingly complicated biol. phenomenon.  Here, the authors examine the biol. importance of new types and sites of histone PTMs and summarize the mol. mechanisms of chromatin engagement by newly discovered epigenetic readers.  The authors also highlight the imperative role of structural insights in understanding PTM-reader interactions and discuss future directions to enhance the knowledge of PTM readout.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwyBPIk5aEQLVg90H21EOLACvtfcHk0lj_kxXJj9ukUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlOnsbbJ&md5=166d01c61bb022b393e24367afac17b8</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2149&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2149%26sid%3Dliteratum%253Aachs%26aulast%3DAndrews%26aufirst%3DF.%2BH.%26aulast%3DStrahl%26aufirst%3DB.%2BD.%26aulast%3DKutateladze%26aufirst%3DT.%2BG.%26atitle%3DInsights%2520into%2520Newly%2520Discovered%2520Marks%2520and%2520Readers%2520of%2520Epigenetic%2520Information%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3D662%26epage%3D668%26doi%3D10.1038%2Fnchembio.2149" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baril, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koenig, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krone, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albanese, K. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, C. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, G. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houk, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waters, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brustad, E. M.</span></span> <span> </span><span class="NLM_article-title">Investigation of Trimethyllysine Binding by the HP1 Chromodomain via Unnatural Amino Acid Mutagenesis</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>139</i></span> (<span class="NLM_issue">48</span>),  <span class="NLM_fpage">17253</span>– <span class="NLM_lpage">17256</span>, <span class="refDoi"> DOI: 10.1021/jacs.7b09223</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.7b09223" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslOns7fL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=139&publication_year=2017&pages=17253-17256&issue=48&author=S.+A.+Barilauthor=A.+L.+Koenigauthor=M.+W.+Kroneauthor=K.+I.+Albaneseauthor=C.+Q.+Heauthor=G.+Y.+Leeauthor=K.+N.+Houkauthor=M.+L.+Watersauthor=E.+M.+Brustad&title=Investigation+of+Trimethyllysine+Binding+by+the+HP1+Chromodomain+via+Unnatural+Amino+Acid+Mutagenesis&doi=10.1021%2Fjacs.7b09223"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Investigation of Trimethyllysine Binding by the HP1 Chromodomain via Unnatural Amino Acid Mutagenesis</span></div><div class="casAuthors">Baril, Stefanie A.; Koenig, Amber L.; Krone, Mackenzie W.; Albanese, Katherine I.; He, Cyndi Qixin; Lee, Ga Young; Houk, Kendall N.; Waters, Marcey L.; Brustad, Eric M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">139</span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">17253-17256</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Trimethyllysine (Kme3) reader proteins are targets for inhibition due to their role in mediating gene expression.  While all such reader proteins bind Kme3 in an arom. cage, the driving force for binding may differ; some readers exhibit evidence for cation-π interactions while others do not.  Here, we report a general unnatural amino acid mutagenesis (Tyr → Phe deriv. mutants) approach to quantify the contribution of individual Tyr residues to cation binding using the heterochromatin protein 1 (HP1) chromodomain as a model system.  We demonstrated that 2 Tyr residues (Tyr-24 and Tyr-48) bound to a Kme3-histone tail peptide via cation-π interactions, but linear free energy trends suggested they did not contribute equally to binding.  The x-ray crystal structures of the HP1 chromodomain Tyr → Phe deriv. mutants and computational anal. suggested that the distance and degree of contact between Tyr residues and Kme3 plays an important role in tuning cation-π-mediated Kme3 recognition.  While cation-π interactions have been studied in a no. of proteins, this work is the 1st to utilize direct binding assays, x-ray crystallog., and modeling, to pinpoint factors that influence the magnitude of the individual cation-π interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokN25CY4EicrVg90H21EOLACvtfcHk0lj_kxXJj9ukUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslOns7fL&md5=a19fe43332a270f2fc1ca4451477d4e0</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fjacs.7b09223&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.7b09223%26sid%3Dliteratum%253Aachs%26aulast%3DBaril%26aufirst%3DS.%2BA.%26aulast%3DKoenig%26aufirst%3DA.%2BL.%26aulast%3DKrone%26aufirst%3DM.%2BW.%26aulast%3DAlbanese%26aufirst%3DK.%2BI.%26aulast%3DHe%26aufirst%3DC.%2BQ.%26aulast%3DLee%26aufirst%3DG.%2BY.%26aulast%3DHouk%26aufirst%3DK.%2BN.%26aulast%3DWaters%26aufirst%3DM.%2BL.%26aulast%3DBrustad%26aufirst%3DE.%2BM.%26atitle%3DInvestigation%2520of%2520Trimethyllysine%2520Binding%2520by%2520the%2520HP1%2520Chromodomain%2520via%2520Unnatural%2520Amino%2520Acid%2520Mutagenesis%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2017%26volume%3D139%26issue%3D48%26spage%3D17253%26epage%3D17256%26doi%3D10.1021%2Fjacs.7b09223" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, Y.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tharp, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weber, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koenig, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waters, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Summerer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W. R.</span></span> <span> </span><span class="NLM_article-title">Genetically Encoded Fluorophenylalanines Enable Insights into the Recognition of Lysine Trimethylation by an Epigenetic Reader</span>. <i>Chem. Commun. (Cambridge, U. K.)</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>52</i></span> (<span class="NLM_issue">85</span>),  <span class="NLM_fpage">12606</span>– <span class="NLM_lpage">12609</span>, <span class="refDoi"> DOI: 10.1039/C6CC05959G</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1039%2FC6CC05959G" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=27711380" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFOrsLvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2016&pages=12606-12609&issue=85&author=Y.-J.+Leeauthor=M.+J.+Schmidtauthor=J.+M.+Tharpauthor=A.+Weberauthor=A.+L.+Koenigauthor=H.+Zhengauthor=J.+Gaoauthor=M.+L.+Watersauthor=D.+Summererauthor=W.+R.+Liu&title=Genetically+Encoded+Fluorophenylalanines+Enable+Insights+into+the+Recognition+of+Lysine+Trimethylation+by+an+Epigenetic+Reader&doi=10.1039%2FC6CC05959G"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Genetically encoded fluorophenylalanines enable insights into the recognition of lysine trimethylation by an epigenetic reader</span></div><div class="casAuthors">Lee, Yan-Jiun; Schmidt, M. J.; Tharp, Jeffery M.; Weber, Annemarie; Koenig, Amber L.; Zheng, Hong; Gao, Jianmin; Waters, Marcey L.; Summerer, Daniel; Liu, Wenshe R.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">85</span>),
    <span class="NLM_cas:pages">12606-12609</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Fluorophenylalanines bearing 2-5 fluorine atoms at the Ph ring have been genetically encoded by amber codon.  Replacement of F59, a phenylalanine residue that is directly involved in interactions with trimethylated K9 of histone H3, in the Mpp8 chromodomain recombinantly with fluorophenylalanines significantly impairs the binding to a K9-trimethylated H3 peptide.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqE6UCOJqwbPrVg90H21EOLACvtfcHk0lhyn1t6YcJiMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFOrsLvJ&md5=ae08d74f4665b47dc7928dfa591717e2</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1039%2FC6CC05959G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6CC05959G%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DY.-J.%26aulast%3DSchmidt%26aufirst%3DM.%2BJ.%26aulast%3DTharp%26aufirst%3DJ.%2BM.%26aulast%3DWeber%26aufirst%3DA.%26aulast%3DKoenig%26aufirst%3DA.%2BL.%26aulast%3DZheng%26aufirst%3DH.%26aulast%3DGao%26aufirst%3DJ.%26aulast%3DWaters%26aufirst%3DM.%2BL.%26aulast%3DSummerer%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DW.%2BR.%26atitle%3DGenetically%2520Encoded%2520Fluorophenylalanines%2520Enable%2520Insights%2520into%2520the%2520Recognition%2520of%2520Lysine%2520Trimethylation%2520by%2520an%2520Epigenetic%2520Reader%26jtitle%3DChem.%2520Commun.%2520%2528Cambridge%252C%2520U.%2520K.%2529%26date%3D2016%26volume%3D52%26issue%3D85%26spage%3D12606%26epage%3D12609%26doi%3D10.1039%2FC6CC05959G" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Milosevich, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hof, F.</span></span> <span> </span><span class="NLM_article-title">Chemical Inhibitors of Epigenetic Methyllysine Reader Proteins</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">1570</span>– <span class="NLM_lpage">1583</span>, <span class="refDoi"> DOI: 10.1021/acs.biochem.5b01073</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biochem.5b01073" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFOmtrnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=1570-1583&issue=11&author=N.+Milosevichauthor=F.+Hof&title=Chemical+Inhibitors+of+Epigenetic+Methyllysine+Reader+Proteins&doi=10.1021%2Facs.biochem.5b01073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical Inhibitors of Epigenetic Methyllysine Reader Proteins</span></div><div class="casAuthors">Milosevich, Natalia; Hof, Fraser</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1570-1583</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein methylation is a common post-translational modification with diverse biol. functions.  Methyllysine reader proteins are increasingly a focus of epigenetics research and play important roles in regulating many cellular processes.  These reader proteins are vital players in development, cell cycle regulation, stress responses, oncogenesis, and other disease pathways.  The recent emergence of a small no. of chem. inhibitors for methyllysine reader proteins supports the viability of these proteins as targets for drug development.  This article introduces the biochem. and biol. of methyllysine reader proteins, provides an overview of functions for those families of readers that have been targeted to date (MBT, PHD, tudor, and chromodomains), and reviews the development of synthetic agents that directly block their methyllysine reading functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGps23HnAOk3T7Vg90H21EOLACvtfcHk0lhyn1t6YcJiMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFOmtrnM&md5=0d9ec539ad7b345bec4ecb8a252a9ff8</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Facs.biochem.5b01073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biochem.5b01073%26sid%3Dliteratum%253Aachs%26aulast%3DMilosevich%26aufirst%3DN.%26aulast%3DHof%26aufirst%3DF.%26atitle%3DChemical%2520Inhibitors%2520of%2520Epigenetic%2520Methyllysine%2520Reader%2520Proteins%26jtitle%3DBiochemistry%26date%3D2016%26volume%3D55%26issue%3D11%26spage%3D1570%26epage%3D1583%26doi%3D10.1021%2Facs.biochem.5b01073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dickson, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norris, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cholensky, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tempel, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sagum, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Black, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baughman, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senisterra, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pattenden, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, L. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S. V.</span></span> <span> </span><span class="NLM_article-title">A Cellular Chemical Probe Targeting the Chromodomains of Polycomb Repressive Complex 1</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">180</span>– <span class="NLM_lpage">187</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1038%2Fnchembio.2007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=26807715" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFyis7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=180-187&issue=3&author=J.+I.+Stuckeyauthor=B.+M.+Dicksonauthor=N.+Chengauthor=Y.+Liuauthor=J.+L.+Norrisauthor=S.+H.+Cholenskyauthor=W.+Tempelauthor=S.+Qinauthor=K.+G.+Huberauthor=C.+Sagumauthor=K.+Blackauthor=F.+Liauthor=X.+P.+Huangauthor=B.+L.+Rothauthor=B.+M.+Baughmanauthor=G.+Senisterraauthor=S.+G.+Pattendenauthor=M.+Vedadiauthor=P.+J.+Brownauthor=M.+T.+Bedfordauthor=J.+Minauthor=C.+H.+Arrowsmithauthor=L.+I.+Jamesauthor=S.+V.+Frye&title=A+Cellular+Chemical+Probe+Targeting+the+Chromodomains+of+Polycomb+Repressive+Complex+1&doi=10.1038%2Fnchembio.2007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">A cellular chemical probe targeting the chromodomains of Polycomb repressive complex 1</span></div><div class="casAuthors">Stuckey, Jacob I.; Dickson, Bradley M.; Cheng, Nancy; Liu, Yanli; Norris, Jacqueline L.; Cholensky, Stephanie H.; Tempel, Wolfram; Qin, Su; Huber, Katherine G.; Sagum, Cari; Black, Karynne; Li, Fengling; Huang, Xi-Ping; Roth, Bryan L.; Baughman, Brandi M.; Senisterra, Guillermo; Pattenden, Samantha G.; Vedadi, Masoud; Brown, Peter J.; Bedford, Mark T.; Min, Jinrong; Arrowsmith, Cheryl H.; James, Lindsey I.; Frye, Stephen V.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">180-187</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We report the design and characterization of UNC3866, a potent antagonist of the methyllysine (Kme) reading function of the Polycomb CBX and CDY families of chromodomains.  Polycomb CBX proteins regulate gene expression by targeting Polycomb repressive complex 1 (PRC1) to sites of H3K27me3 via their chromodomains.  UNC3866 binds the chromodomains of CBX4 and CBX7 most potently, with a Kd of ∼100 nM for each, and is 6- to 18-fold selective as compared to seven other CBX and CDY chromodomains while being highly selective over >250 other protein targets.  X-ray crystallog. revealed that UNC3866's interactions with the CBX chromodomains closely mimic those of the methylated H3 tail.  UNC4195, a biotinylated deriv. of UNC3866, was used to demonstrate that UNC3866 engages intact PRC1 and that EED incorporation into PRC1 is isoform dependent in PC3 prostate cancer cells.  Finally, UNC3866 inhibits PC3 cell proliferation, consistent with the known ability of CBX7 overexpression to confer a growth advantage, whereas UNC4219, a methylated neg. control compd., has negligible effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobRZ8YoHZWk7Vg90H21EOLACvtfcHk0liMmQ8CUhl8pA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFyis7o%253D&md5=97fe8dd8e35094a8c91d5d4bdc265b4f</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2007%26sid%3Dliteratum%253Aachs%26aulast%3DStuckey%26aufirst%3DJ.%2BI.%26aulast%3DDickson%26aufirst%3DB.%2BM.%26aulast%3DCheng%26aufirst%3DN.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DCholensky%26aufirst%3DS.%2BH.%26aulast%3DTempel%26aufirst%3DW.%26aulast%3DQin%26aufirst%3DS.%26aulast%3DHuber%26aufirst%3DK.%2BG.%26aulast%3DSagum%26aufirst%3DC.%26aulast%3DBlack%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DHuang%26aufirst%3DX.%2BP.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DBaughman%26aufirst%3DB.%2BM.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DPattenden%26aufirst%3DS.%2BG.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DMin%26aufirst%3DJ.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DJames%26aufirst%3DL.%2BI.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26atitle%3DA%2520Cellular%2520Chemical%2520Probe%2520Targeting%2520the%2520Chromodomains%2520of%2520Polycomb%2520Repressive%2520Complex%25201%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26issue%3D3%26spage%3D180%26epage%3D187%26doi%3D10.1038%2Fnchembio.2007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simpson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norris-Drouin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cholensky, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasca, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dickson, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearce, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, L. I.</span></span> <span> </span><span class="NLM_article-title">Structure-Activity Relationships and Kinetic Studies of Peptidic Antagonists of CBX Chromodomains</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">19</span>),  <span class="NLM_fpage">8913</span>– <span class="NLM_lpage">8923</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00801</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00801" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVagu7zO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=8913-8923&issue=19&author=J.+I.+Stuckeyauthor=C.+Simpsonauthor=J.+L.+Norris-Drouinauthor=S.+H.+Cholenskyauthor=J.+Leeauthor=R.+Pascaauthor=N.+Chengauthor=B.+M.+Dicksonauthor=K.+H.+Pearceauthor=S.+V.+Fryeauthor=L.+I.+James&title=Structure-Activity+Relationships+and+Kinetic+Studies+of+Peptidic+Antagonists+of+CBX+Chromodomains&doi=10.1021%2Facs.jmedchem.6b00801"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Activity Relationships and Kinetic Studies of Peptidic Antagonists of CBX Chromodomains</span></div><div class="casAuthors">Stuckey, Jacob I.; Simpson, Catherine; Norris-Drouin, Jacqueline L.; Cholensky, Stephanie H.; Lee, Junghyun; Pasca, Ryan; Cheng, Nancy; Dickson, Bradley M.; Pearce, Kenneth H.; Frye, Stephen V.; James, Lindsey I.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8913-8923</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To better understand the contribution of methyl-lysine (Kme) binding proteins to various disease states, the authors recently developed and reported the discovery of I (UNC3866), a chem. probe that targets two families of Kme binding proteins, CBX and CDY chromodomains, with selectivity for CBX4 and -7.  The discovery of I was enabled in part by the use of mol. dynamics simulations performed with CBX7 and its endogenous substrate.  Herein, the authors describe the design, synthesis and structure-activity relationship studies that led to the development of I and provide support for the model of CBX7-ligand recognition by examg. the binding kinetics of the antagonists with CBX7 as detd. by surface-plasmon resonance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotQTqv5W7R6rVg90H21EOLACvtfcHk0liMmQ8CUhl8pA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVagu7zO&md5=c53cfc1aeaf1d0ea12b67a949d74635b</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00801&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00801%26sid%3Dliteratum%253Aachs%26aulast%3DStuckey%26aufirst%3DJ.%2BI.%26aulast%3DSimpson%26aufirst%3DC.%26aulast%3DNorris-Drouin%26aufirst%3DJ.%2BL.%26aulast%3DCholensky%26aufirst%3DS.%2BH.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DPasca%26aufirst%3DR.%26aulast%3DCheng%26aufirst%3DN.%26aulast%3DDickson%26aufirst%3DB.%2BM.%26aulast%3DPearce%26aufirst%3DK.%2BH.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DJames%26aufirst%3DL.%2BI.%26atitle%3DStructure-Activity%2520Relationships%2520and%2520Kinetic%2520Studies%2520of%2520Peptidic%2520Antagonists%2520of%2520CBX%2520Chromodomains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26issue%3D19%26spage%3D8913%26epage%3D8923%26doi%3D10.1021%2Facs.jmedchem.6b00801" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lamb, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bsteh, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dishman, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moussa, H. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norris, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cholensky, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sagum, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stanton, B. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearce, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kenakin, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kireev, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, L. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S. V.</span></span> <span> </span><span class="NLM_article-title">Discovery and Characterization of a Cellular Potent Positive Allosteric Modulator of the Polycomb Repressive Complex 1 Chromodomain, CBX7</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>26</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">1365</span>– <span class="NLM_lpage">1379.e22</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2019.07.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1016%2Fj.chembiol.2019.07.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=31422906" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1SitLnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2019&pages=1365-1379.e22&issue=10&author=K.+N.+Lambauthor=D.+Bstehauthor=S.+N.+Dishmanauthor=H.+F.+Moussaauthor=H.+Fanauthor=J.+I.+Stuckeyauthor=J.+L.+Norrisauthor=S.+H.+Cholenskyauthor=D.+Liauthor=J.+Wangauthor=C.+Sagumauthor=B.+Z.+Stantonauthor=M.+T.+Bedfordauthor=K.+H.+Pearceauthor=T.+P.+Kenakinauthor=D.+B.+Kireevauthor=G.+G.+Wangauthor=L.+I.+Jamesauthor=O.+Bellauthor=S.+V.+Frye&title=Discovery+and+Characterization+of+a+Cellular+Potent+Positive+Allosteric+Modulator+of+the+Polycomb+Repressive+Complex+1+Chromodomain%2C+CBX7&doi=10.1016%2Fj.chembiol.2019.07.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Characterization of a Cellular Potent Positive Allosteric Modulator of the Polycomb Repressive Complex 1 Chromodomain, CBX7</span></div><div class="casAuthors">Lamb, Kelsey N.; Bsteh, Daniel; Dishman, Sarah N.; Moussa, Hagar F.; Fan, Huitao; Stuckey, Jacob I.; Norris, Jacqueline L.; Cholensky, Stephanie H.; Li, Dongxu; Wang, Jingkui; Sagum, Cari; Stanton, Benjamin Z.; Bedford, Mark T.; Pearce, Kenneth H.; Kenakin, Terry P.; Kireev, Dmitri B.; Wang, Gang Greg; James, Lindsey I.; Bell, Oliver; Frye, Stephen V.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1365-1379.e22</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Polycomb-directed repression of gene expression is frequently misregulated in human diseases.  A quant. and target-specific cellular assay was utilized to discover the first potent pos. allosteric modulator (PAM) peptidomimetic, UNC4976, of nucleic acid binding by CBX7, a chromodomain methyl-lysine reader of Polycomb repressive complex 1.  The PAM activity of UNC4976 resulted in enhanced efficacy across three orthogonal cellular assays by simultaneously antagonizing H3K27me3-specific recruitment of CBX7 to target genes while increasing non-specific binding to DNA and RNA.  PAM activity thereby reequilibrates PRC1 away from H3K27me3 target regions.  Together, our discovery and characterization of UNC4976 not only revealed the most cellularly potent PRC1-specific chem. probe to date, but also uncovers a potential mechanism of Polycomb regulation with implications for non-histone lysine methylated interaction partners.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXrcSgaffNibVg90H21EOLACvtfcHk0lhZePuUU8ccIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhs1SitLnK&md5=81c0bc1a765f319637d8deb8e3d3e518</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2019.07.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2019.07.013%26sid%3Dliteratum%253Aachs%26aulast%3DLamb%26aufirst%3DK.%2BN.%26aulast%3DBsteh%26aufirst%3DD.%26aulast%3DDishman%26aufirst%3DS.%2BN.%26aulast%3DMoussa%26aufirst%3DH.%2BF.%26aulast%3DFan%26aufirst%3DH.%26aulast%3DStuckey%26aufirst%3DJ.%2BI.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DCholensky%26aufirst%3DS.%2BH.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DSagum%26aufirst%3DC.%26aulast%3DStanton%26aufirst%3DB.%2BZ.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DPearce%26aufirst%3DK.%2BH.%26aulast%3DKenakin%26aufirst%3DT.%2BP.%26aulast%3DKireev%26aufirst%3DD.%2BB.%26aulast%3DWang%26aufirst%3DG.%2BG.%26aulast%3DJames%26aufirst%3DL.%2BI.%26aulast%3DBell%26aufirst%3DO.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26atitle%3DDiscovery%2520and%2520Characterization%2520of%2520a%2520Cellular%2520Potent%2520Positive%2520Allosteric%2520Modulator%2520of%2520the%2520Polycomb%2520Repressive%2520Complex%25201%2520Chromodomain%252C%2520CBX7%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2019%26volume%3D26%26issue%3D10%26spage%3D1365%26epage%3D1379.e22%26doi%3D10.1016%2Fj.chembiol.2019.07.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barnash, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">The, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norris-Drouin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cholensky, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Worley, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, L. I.</span></span> <span> </span><span class="NLM_article-title">Discovery of Peptidomimetic Ligands of EED as Allosteric Inhibitors of PRC2</span>. <i>ACS Comb. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>19</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">161</span>– <span class="NLM_lpage">172</span>, <span class="refDoi"> DOI: 10.1021/acscombsci.6b00174</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscombsci.6b00174" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitFWgsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2017&pages=161-172&issue=3&author=K.+D.+Barnashauthor=J.+Theauthor=J.+L.+Norris-Drouinauthor=S.+H.+Cholenskyauthor=B.+M.+Worleyauthor=F.+Liauthor=J.+I.+Stuckeyauthor=P.+J.+Brownauthor=M.+Vedadiauthor=C.+H.+Arrowsmithauthor=S.+V.+Fryeauthor=L.+I.+James&title=Discovery+of+Peptidomimetic+Ligands+of+EED+as+Allosteric+Inhibitors+of+PRC2&doi=10.1021%2Facscombsci.6b00174"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Peptidomimetic Ligands of EED as Allosteric Inhibitors of PRC2</span></div><div class="casAuthors">Barnash, Kimberly D.; The, Juliana; Norris-Drouin, Jacqueline L.; Cholensky, Stephanie H.; Worley, Beau M.; Li, Fengling; Stuckey, Jacob I.; Brown, Peter J.; Vedadi, Masoud; Arrowsmith, Cheryl H.; Frye, Stephen V.; James, Lindsey I.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Combinatorial Science</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">161-172</span>CODEN:
                <span class="NLM_cas:coden">ACSCCC</span>;
        ISSN:<span class="NLM_cas:issn">2156-8944</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The function of EED within Polycomb repressive complex 2 (PRC2) is mediated by a complex network of protein-protein interactions.  Allosteric activation of PRC2 by binding of methylated proteins to EED's arom. cage is essential for full catalytic activity, but details of this regulation are not fully understood.  EED's recognition of the product of PRC2 activity, histone H3 lysine 27 trimethylation (H3K27me3), stimulates PRC2 methyltransferase activity at adjacent nucleosomes leading to H3K27me3 propagation and, ultimately, gene repression.  By coupling combinatorial chem. and structure-based design, we optimized a low affinity methylated Jarid2 peptide to a smaller, more potent peptidomimetic ligand (Kd = 1.14 ± 0.14 μM) of the arom. cage of EED.  Our strategy illustrates the effectiveness of applying combinatorial chem. to achieve both ligand potency and property optimization.  Furthermore, the resulting ligands, UNC5114 and UNC5115, demonstrate that targeted disruption of EED's reader function can lead to allosteric inhibition of PRC2 catalytic activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwOqOVJ8hSD7Vg90H21EOLACvtfcHk0lhZePuUU8ccIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitFWgsrw%253D&md5=0e7da46fc40d89f4095ac979374338fc</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Facscombsci.6b00174&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscombsci.6b00174%26sid%3Dliteratum%253Aachs%26aulast%3DBarnash%26aufirst%3DK.%2BD.%26aulast%3DThe%26aufirst%3DJ.%26aulast%3DNorris-Drouin%26aufirst%3DJ.%2BL.%26aulast%3DCholensky%26aufirst%3DS.%2BH.%26aulast%3DWorley%26aufirst%3DB.%2BM.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DStuckey%26aufirst%3DJ.%2BI.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DJames%26aufirst%3DL.%2BI.%26atitle%3DDiscovery%2520of%2520Peptidomimetic%2520Ligands%2520of%2520EED%2520as%2520Allosteric%2520Inhibitors%2520of%2520PRC2%26jtitle%3DACS%2520Comb.%2520Sci.%26date%3D2017%26volume%3D19%26issue%3D3%26spage%3D161%26epage%3D172%26doi%3D10.1021%2Facscombsci.6b00174" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Suh, J. L. J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnash, K. D. K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abramyan, T. M. T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">The, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engelberg, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J. P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kireev, D. B. D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H. C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, L. I. L. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S. V. S. V.</span></span> <span> </span><span class="NLM_article-title">Discovery of Selective Activators of PRC2Mutant EED-I363M</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">6524</span>, <span class="refDoi"> DOI: 10.1038/s41598-019-43005-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1038%2Fs41598-019-43005-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=31024026" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A280%3ADC%252BB3M%252Fptlajtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2019&pages=6524&issue=1&author=J.+L.+J.+L.+Suhauthor=K.+D.+K.+D.+Barnashauthor=T.+M.+T.+M.+Abramyanauthor=F.+Liauthor=J.+Theauthor=I.+A.+Engelbergauthor=M.+Vedadiauthor=P.+J.+P.+J.+Brownauthor=D.+B.+D.+B.+Kireevauthor=C.+H.+C.+H.+Arrowsmithauthor=L.+I.+L.+I.+Jamesauthor=S.+V.+S.+V.+Frye&title=Discovery+of+Selective+Activators+of+PRC2Mutant+EED-I363M&doi=10.1038%2Fs41598-019-43005-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of selective activators of PRC2 mutant EED-I363M</span></div><div class="casAuthors">Suh Junghyun L; Barnash Kimberly D; Abramyan Tigran M; Engelberg Isabelle A; Kireev Dmitri B; James Lindsey I; Frye Stephen V; Barnash Kimberly D; Li Fengling; The Juliana; Vedadi Masoud; Brown Peter J; Arrowsmith Cheryl H; Arrowsmith Cheryl H</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">6524</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Many common disease-causing mutations result in loss-of-function (LOF) of the proteins in which they occur.  LOF mutations have proven recalcitrant to pharmacologic intervention, presenting a challenge for the development of targeted therapeutics.  Polycomb repressive complex 2 (PRC2), which contains core subunits (EZH2, EED, and SUZ12), regulates gene activity by trimethylation of histone 3 lysine 27.  The dysregulation of PRC2 catalytic activity by mutations has been implicated in cancer and other diseases.  Among the mutations that cause PRC2 malfunction, an I363M LOF mutation of EED has been identified in myeloid disorders, where it prevents allosteric activation of EZH2 catalysis.  We describe structure-based design and computational simulations of ligands created to ameliorate this LOF.  Notably, these compounds selectively stimulate the catalytic activity of PRC2-EED-I363M over wildtype-PRC2.  Overall, this work demonstrates the feasibility of developing targeted therapeutics for PRC2-EED-I363M that act as allosteric agonists, potentially correcting this LOF mutant phenotype.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTy_y3UKz_BfTQLBVDQn4bCfW6udTcc2eaqlQbuN6_-zLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3M%252Fptlajtg%253D%253D&md5=92fb9e3260bdc49440a1b803c833ed9e</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fs41598-019-43005-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-019-43005-z%26sid%3Dliteratum%253Aachs%26aulast%3DSuh%26aufirst%3DJ.%2BL.%2BJ.%2BL.%26aulast%3DBarnash%26aufirst%3DK.%2BD.%2BK.%2BD.%26aulast%3DAbramyan%26aufirst%3DT.%2BM.%2BT.%2BM.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DThe%26aufirst%3DJ.%26aulast%3DEngelberg%26aufirst%3DI.%2BA.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%2BP.%2BJ.%26aulast%3DKireev%26aufirst%3DD.%2BB.%2BD.%2BB.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%2BC.%2BH.%26aulast%3DJames%26aufirst%3DL.%2BI.%2BL.%2BI.%26aulast%3DFrye%26aufirst%3DS.%2BV.%2BS.%2BV.%26atitle%3DDiscovery%2520of%2520Selective%2520Activators%2520of%2520PRC2Mutant%2520EED-I363M%26jtitle%3DSci.%2520Rep.%26date%3D2019%26volume%3D9%26issue%3D1%26spage%3D6524%26doi%3D10.1038%2Fs41598-019-43005-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barnash, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamb, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norris, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cholensky, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kireev, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, L. I.</span></span> <span> </span><span class="NLM_article-title">Chromodomain Ligand Optimization via Target-Class Directed Combinatorial Repurposing</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">2475</span>– <span class="NLM_lpage">2483</span>, <span class="refDoi"> DOI: 10.1021/acschembio.6b00415</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.6b00415" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVOgt7fK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=2475-2483&issue=9&author=K.+D.+Barnashauthor=K.+N.+Lambauthor=J.+I.+Stuckeyauthor=J.+L.+Norrisauthor=S.+H.+Cholenskyauthor=D.+B.+Kireevauthor=S.+V.+Fryeauthor=L.+I.+James&title=Chromodomain+Ligand+Optimization+via+Target-Class+Directed+Combinatorial+Repurposing&doi=10.1021%2Facschembio.6b00415"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Chromodomain Ligand Optimization via Target-Class Directed Combinatorial Repurposing</span></div><div class="casAuthors">Barnash, Kimberly D.; Lamb, Kelsey N.; Stuckey, Jacob I.; Norris, Jacqueline L.; Cholensky, Stephanie H.; Kireev, Dmitri B.; Frye, Stephen V.; James, Lindsey I.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2475-2483</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Efforts to develop strategies for small-mol. chem. probe discovery against the readers of the methyl-lysine (Kme) post-translational modification have been met with limited success.  Targeted disruption of these protein-protein interactions via peptidomimetic inhibitor optimization is a promising alternative to small-mol. hit discovery; however, recognition of identical peptide motifs by multiple Kme reader proteins presents a unique challenge in the development of selective Kme reader chem. probes.  These selectivity challenges are exemplified by the Polycomb repressive complex 1 (PRC1) chem. probe, UNC3866, which demonstrates submicromolar off-target affinity toward the non-PRC1 chromodomains CDYL2 and CDYL.  Moreover, since peptidomimetics are challenging subjects for structure-activity relationship (SAR) studies, traditional optimization of UNC3866 would prove costly and time-consuming.  Herein, we report a broadly applicable strategy for the affinity-based, target-class screening of chromodomains via the repurposing of UNC3866 in an efficient, combinatorial peptide library.  A first-generation library yielded UNC4991, a UNC3866 analog that exhibits a distinct selectivity profile while maintaining submicromolar affinity toward the CDYL chromodomains.  Addnl., in vitro pull-down expts. from HeLa nuclear lysates further demonstrate the selectivity and utility of this compd. for future elucidation of CDYL protein function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkyoB69iWV7bVg90H21EOLACvtfcHk0lj-ex_btmYYvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVOgt7fK&md5=89613f3d7e6d533caa5f27f7b7c0629c</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facschembio.6b00415&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.6b00415%26sid%3Dliteratum%253Aachs%26aulast%3DBarnash%26aufirst%3DK.%2BD.%26aulast%3DLamb%26aufirst%3DK.%2BN.%26aulast%3DStuckey%26aufirst%3DJ.%2BI.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DCholensky%26aufirst%3DS.%2BH.%26aulast%3DKireev%26aufirst%3DD.%2BB.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DJames%26aufirst%3DL.%2BI.%26atitle%3DChromodomain%2520Ligand%2520Optimization%2520via%2520Target-Class%2520Directed%2520Combinatorial%2520Repurposing%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26issue%3D9%26spage%3D2475%26epage%3D2483%26doi%3D10.1021%2Facschembio.6b00415" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rectenwald, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hardy, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norris-Drouin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cholensky, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, L. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearce, K. H.</span></span> <span> </span><span class="NLM_article-title">A General TR-FRET Assay Platform for High-Throughput Screening and Characterizing Inhibitors of Methyl-Lysine Reader Proteins</span>. <i>SLAS Discovery Adv. Life Sci. R&D</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>24</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">693</span>– <span class="NLM_lpage">700</span>, <span class="refDoi"> DOI: 10.1177/2472555219844569</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1177%2F2472555219844569" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2019&pages=693-700&issue=6&author=J.+M.+Rectenwaldauthor=P.+B.+Hardyauthor=J.+L.+Norris-Drouinauthor=S.+H.+Cholenskyauthor=L.+I.+Jamesauthor=S.+V.+Fryeauthor=K.+H.+Pearce&title=A+General+TR-FRET+Assay+Platform+for+High-Throughput+Screening+and+Characterizing+Inhibitors+of+Methyl-Lysine+Reader+Proteins&doi=10.1177%2F2472555219844569"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1177%2F2472555219844569&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F2472555219844569%26sid%3Dliteratum%253Aachs%26aulast%3DRectenwald%26aufirst%3DJ.%2BM.%26aulast%3DHardy%26aufirst%3DP.%2BB.%26aulast%3DNorris-Drouin%26aufirst%3DJ.%2BL.%26aulast%3DCholensky%26aufirst%3DS.%2BH.%26aulast%3DJames%26aufirst%3DL.%2BI.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DPearce%26aufirst%3DK.%2BH.%26atitle%3DA%2520General%2520TR-FRET%2520Assay%2520Platform%2520for%2520High-Throughput%2520Screening%2520and%2520Characterizing%2520Inhibitors%2520of%2520Methyl-Lysine%2520Reader%2520Proteins%26jtitle%3DSLAS%2520Discovery%2520Adv.%2520Life%2520Sci.%2520R%2526D%26date%3D2019%26volume%3D24%26issue%3D6%26spage%3D693%26epage%3D700%26doi%3D10.1177%2F2472555219844569" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Musselman, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Avvakumov, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watanabe, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abraham, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lalonde, M.-E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nuñez, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nickoloff, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kulesza, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yasui, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Côté, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutateladze, T. G.</span></span> <span> </span><span class="NLM_article-title">Molecular Basis for H3K36me3 Recognition by the Tudor Domain of PHF1</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>19</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">1266</span>– <span class="NLM_lpage">1272</span>, <span class="refDoi"> DOI: 10.1038/nsmb.2435</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1038%2Fnsmb.2435" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=23142980" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1Ckt7rE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=1266-1272&issue=12&author=C.+A.+Musselmanauthor=N.+Avvakumovauthor=R.+Watanabeauthor=C.+G.+Abrahamauthor=M.-E.+E.+Lalondeauthor=Z.+Hongauthor=C.+Allenauthor=S.+Royauthor=J.+K.+Nu%C3%B1ezauthor=J.+Nickoloffauthor=C.+A.+Kuleszaauthor=A.+Yasuiauthor=J.+C%C3%B4t%C3%A9author=T.+G.+Kutateladze&title=Molecular+Basis+for+H3K36me3+Recognition+by+the+Tudor+Domain+of+PHF1&doi=10.1038%2Fnsmb.2435"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular basis for H3K36me3 recognition by the Tudor domain of PHF1</span></div><div class="casAuthors">Musselman, Catherine A.; Avvakumov, Nikita; Watanabe, Reiko; Abraham, Christopher G.; Lalonde, Marie-Eve; Hong, Zehui; Allen, Christopher; Roy, Siddhartha; Nunez, James K.; Nickoloff, Jac; Kulesza, Caroline A.; Yasui, Akira; Cote, Jacques; Kutateladze, Tatiana G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1266-1272</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The PHD finger protein 1 (PHF1) is essential in epigenetic regulation and genome maintenance.  Here, the authors show that the Tudor domain of human PHF1 binds to histone H3 trimethylated at Lys-36 (H3K36me3).  The authors report a 1.9-Å resoln. crystal structure of the Tudor domain in complex with H3K36me3 peptide (residues 31-40) and describe the mol. mechanism of H3K36me3 recognition using NMR.  The binding of PHF1 to H3K36me3 inhibited the ability of the Polycomb PRC2 complex to methylate Lys-27 of histone H3 in vitro and in vivo.  Laser microirradn. data showed that PHF1 was transiently recruited to DNA double-strand breaks, and PHF1 mutants impaired in the H3K36me3 interaction exhibited reduced retention at double-strand break sites.  Together, these findings suggest that PHF1 can mediate the deposition of the repressive H3K27me3 mark and acts as a cofactor in early DNA-damage response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAczVxJthG_7Vg90H21EOLACvtfcHk0ljrsjZSM4QVBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1Ckt7rE&md5=f8e6e578ef81a645b5123e6b51836643</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.2435&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.2435%26sid%3Dliteratum%253Aachs%26aulast%3DMusselman%26aufirst%3DC.%2BA.%26aulast%3DAvvakumov%26aufirst%3DN.%26aulast%3DWatanabe%26aufirst%3DR.%26aulast%3DAbraham%26aufirst%3DC.%2BG.%26aulast%3DLalonde%26aufirst%3DM.-E.%2BE.%26aulast%3DHong%26aufirst%3DZ.%26aulast%3DAllen%26aufirst%3DC.%26aulast%3DRoy%26aufirst%3DS.%26aulast%3DNu%25C3%25B1ez%26aufirst%3DJ.%2BK.%26aulast%3DNickoloff%26aufirst%3DJ.%26aulast%3DKulesza%26aufirst%3DC.%2BA.%26aulast%3DYasui%26aufirst%3DA.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DJ.%26aulast%3DKutateladze%26aufirst%3DT.%2BG.%26atitle%3DMolecular%2520Basis%2520for%2520H3K36me3%2520Recognition%2520by%2520the%2520Tudor%2520Domain%2520of%2520PHF1%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2012%26volume%3D19%26issue%3D12%26spage%3D1266%26epage%3D1272%26doi%3D10.1038%2Fnsmb.2435" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rothbart, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.-Y. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tripathy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rockowitz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allis, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strahl, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, G. G.</span></span> <span> </span><span class="NLM_article-title">An H3K36 Methylation-Engaging Tudor Motif of Polycomb-like Proteins Mediates PRC2 Complex Targeting</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>49</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">571</span>– <span class="NLM_lpage">582</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2012.11.026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1016%2Fj.molcel.2012.11.026" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=23273982" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC3sXotFOl" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2013&pages=571-582&issue=3&author=L.+Caiauthor=S.+B.+Rothbartauthor=R.+Luauthor=B.+Xuauthor=W.-Y.+Y.+Chenauthor=A.+Tripathyauthor=S.+Rockowitzauthor=D.+Zhengauthor=D.+J.+Patelauthor=C.+D.+Allisauthor=B.+D.+Strahlauthor=J.+Songauthor=G.+G.+Wang&title=An+H3K36+Methylation-Engaging+Tudor+Motif+of+Polycomb-like+Proteins+Mediates+PRC2+Complex+Targeting&doi=10.1016%2Fj.molcel.2012.11.026"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">An H3K36 Methylation-Engaging Tudor Motif of Polycomb-like Proteins Mediates PRC2 Complex Targeting</span></div><div class="casAuthors">Cai, Ling; Rothbart, Scott B.; Lu, Rui; Xu, Bowen; Chen, Wei-Yi; Tripathy, Ashutosh; Rockowitz, Shira; Zheng, Deyou; Patel, Dinshaw J.; Allis, C. David; Strahl, Brian D.; Song, Jikui; Wang, Gang Greg</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">571-582</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Polycomb repressive complex 2 (PRC2) regulates pluripotency, differentiation, and tumorigenesis through catalysis of histone H3 lysine 27 trimethylation (H3K27me3) on chromatin.  However, the mechanisms that underlie PRC2 recruitment and spreading on chromatin remain unclear.  Here we report that histone H3 lysine 36 trimethylation (H3K36me3) binding activity is harbored in the Tudor motifs of PRC2-assocd. polycomb-like (PCL) proteins PHF1/PCL1 and PHF19/PCL3.  Ectopically expressed PHF1 induced Tudor-dependent stabilization of PRC2 complexes on bulk chromatin and mediated spreading of PRC2 and H3K27me3 into H3K36me3-contg. chromatin regions.  In murine pluripotent stem cells, we identified coexistence of H3K36me3, H3K27me3, and PHF19/PCL3 at a subset of poised developmental genes and demonstrated that PHF19/PCL3 Tudor function is required for optimal H3K27me3 and repression of these loci.  Collectively, our data suggest that PCL recognition of H3K36me3 promotes intrusion of PRC2 complexes into active chromatin regions to promote gene silencing and modulate the chromatin landscape during development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpMX9XeqddRbVg90H21EOLACvtfcHk0ljrsjZSM4QVBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXotFOl&md5=1bcf5ff44f9588863a4b286917055753</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2012.11.026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2012.11.026%26sid%3Dliteratum%253Aachs%26aulast%3DCai%26aufirst%3DL.%26aulast%3DRothbart%26aufirst%3DS.%2BB.%26aulast%3DLu%26aufirst%3DR.%26aulast%3DXu%26aufirst%3DB.%26aulast%3DChen%26aufirst%3DW.-Y.%2BY.%26aulast%3DTripathy%26aufirst%3DA.%26aulast%3DRockowitz%26aufirst%3DS.%26aulast%3DZheng%26aufirst%3DD.%26aulast%3DPatel%26aufirst%3DD.%2BJ.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26aulast%3DStrahl%26aufirst%3DB.%2BD.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DG.%2BG.%26atitle%3DAn%2520H3K36%2520Methylation-Engaging%2520Tudor%2520Motif%2520of%2520Polycomb-like%2520Proteins%2520Mediates%2520PRC2%2520Complex%2520Targeting%26jtitle%3DMol.%2520Cell%26date%3D2013%26volume%3D49%26issue%3D3%26spage%3D571%26epage%3D582%26doi%3D10.1016%2Fj.molcel.2012.11.026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bian, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, J.</span></span> <span> </span><span class="NLM_article-title">Tudor Domains of the PRC2 Components PHF1 and PHF19 Selectively Bind to Histone H3K36me3</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>430</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">547</span>– <span class="NLM_lpage">553</span>, <span class="refDoi"> DOI: 10.1016/j.bbrc.2012.11.116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1016%2Fj.bbrc.2012.11.116" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=23228662" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvV2ktbfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=430&publication_year=2013&pages=547-553&issue=2&author=S.+Qinauthor=Y.+Guoauthor=C.+Xuauthor=C.+Bianauthor=M.+Fuauthor=S.+Gongauthor=J.+Min&title=Tudor+Domains+of+the+PRC2+Components+PHF1+and+PHF19+Selectively+Bind+to+Histone+H3K36me3&doi=10.1016%2Fj.bbrc.2012.11.116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Tudor domains of the PRC2 components PHF1 and PHF19 selectively bind to histone H3K36me3</span></div><div class="casAuthors">Qin, Su; Guo, Yahong; Xu, Chao; Bian, Chuanbing; Fu, Minfei; Gong, Sarah; Min, Jinrong</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">430</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">547-553</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">PRC2 is the major H3K27 methyltransferase and is responsible for maintaining repressed gene expression patterns throughout development.  It contains four core components: EZH2, EED, SUZ12 and RbAp46/48 and some cell-type specific components.  In this study, we focused on characterizing the histone binding domains of PHF1 and PHF19, and found that the Tudor domains of PHF1 and PHF19 selectively bind to histone H3K36me3.  Structural anal. of these Tudor domains also shed light on how these Tudor domains selectively bind to histone H3K36me3.  The histone H3K36me3 binding by the Tudor domains of PHF1, PHF19 and likely MTF2 provide another recruitment and regulatory mechanism for the PRC2 complex.  In addn., we found that the first PHD domains of PHF1 and PHF19 do not exhibit histone H3K4 binding ability, nor do they affect the Tudor domain binding to histones.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLQ6VOXMl3FbVg90H21EOLACvtfcHk0lgvQgPWODfEWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvV2ktbfE&md5=90c224f4ec9068d18de5c01ca61962d3</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2012.11.116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2012.11.116%26sid%3Dliteratum%253Aachs%26aulast%3DQin%26aufirst%3DS.%26aulast%3DGuo%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DC.%26aulast%3DBian%26aufirst%3DC.%26aulast%3DFu%26aufirst%3DM.%26aulast%3DGong%26aufirst%3DS.%26aulast%3DMin%26aufirst%3DJ.%26atitle%3DTudor%2520Domains%2520of%2520the%2520PRC2%2520Components%2520PHF1%2520and%2520PHF19%2520Selectively%2520Bind%2520to%2520Histone%2520H3K36me3%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2013%26volume%3D430%26issue%3D2%26spage%3D547%26epage%3D553%26doi%3D10.1016%2Fj.bbrc.2012.11.116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ballaré, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lange, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lapinaite, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morey, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pascual, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liefke, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gozani, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlomagno, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benitah, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Croce, L.</span></span> <span> </span><span class="NLM_article-title">Phf19 Links Methylated Lys36 of Histone H3 to Regulation of Polycomb Activity</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>19</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">1257</span>– <span class="NLM_lpage">1265</span>, <span class="refDoi"> DOI: 10.1038/nsmb.2434</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1038%2Fnsmb.2434" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=23104054" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFOmtLrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=1257-1265&issue=12&author=C.+Ballar%C3%A9author=M.+Langeauthor=A.+Lapinaiteauthor=G.+M.+Martinauthor=L.+Moreyauthor=G.+Pascualauthor=R.+Liefkeauthor=B.+Simonauthor=Y.+Shiauthor=O.+Gozaniauthor=T.+Carlomagnoauthor=S.+A.+Benitahauthor=L.+Di+Croce&title=Phf19+Links+Methylated+Lys36+of+Histone+H3+to+Regulation+of+Polycomb+Activity&doi=10.1038%2Fnsmb.2434"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Phf19 links methylated Lys36 of histone H3 to regulation of Polycomb activity</span></div><div class="casAuthors">Ballare, Cecilia; Lange, Martin; Lapinaite, Audrone; Martin, Gloria Mas; Morey, Lluis; Pascual, Gloria; Liefke, Robert; Simon, Bernd; Shi, Yang; Gozani, Or; Carlomagno, Teresa; Benitah, Salvador Aznar; Di Croce, Luciano</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1257-1265</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Polycomb-group proteins are transcriptional repressors with essential roles in embryonic development.  Polycomb repressive complex 2 (PRC2) contains the methyltransferase activity for Lys27.  However, the role of other histone modifications in regulating PRC2 activity is just beginning to be understood.  Here we show that direct recognition of methylated histone H3 Lys36 (H3K36me), a mark assocd. with activation, by the PRC2 subunit Phf19 is required for the full enzymic activity of the PRC2 complex.  Using NMR spectroscopy, we provide structural evidence for this interaction.  Furthermore, we show that Phf19 binds to a subset of PRC2 targets in mouse embryonic stem cells and that this is required for their repression and for H3K27me3 deposition.  These findings show that the interaction of Phf19 with H3K36me2 and H3K36me3 is essential for PRC2 complex activity and for proper regulation of gene repression in embryonic stem cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTLpeahK4UbLVg90H21EOLACvtfcHk0lgvQgPWODfEWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFOmtLrM&md5=e465ee81ea9643a80487739332ae8683</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.2434&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.2434%26sid%3Dliteratum%253Aachs%26aulast%3DBallar%25C3%25A9%26aufirst%3DC.%26aulast%3DLange%26aufirst%3DM.%26aulast%3DLapinaite%26aufirst%3DA.%26aulast%3DMartin%26aufirst%3DG.%2BM.%26aulast%3DMorey%26aufirst%3DL.%26aulast%3DPascual%26aufirst%3DG.%26aulast%3DLiefke%26aufirst%3DR.%26aulast%3DSimon%26aufirst%3DB.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DGozani%26aufirst%3DO.%26aulast%3DCarlomagno%26aufirst%3DT.%26aulast%3DBenitah%26aufirst%3DS.%2BA.%26aulast%3DDi%2BCroce%26aufirst%3DL.%26atitle%3DPhf19%2520Links%2520Methylated%2520Lys36%2520of%2520Histone%2520H3%2520to%2520Regulation%2520of%2520Polycomb%2520Activity%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2012%26volume%3D19%26issue%3D12%26spage%3D1257%26epage%3D1265%26doi%3D10.1038%2Fnsmb.2434" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brien, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gambero, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jerman, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Egan, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunne, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jurgens, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wynne, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohan, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sinha, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loftus, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cagney, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bracken, A. P.</span></span> <span> </span><span class="NLM_article-title">Polycomb PHF19 Binds H3K36me3 and Recruits PRC2 and Demethylase NO66 to Embryonic Stem Cell Genes during Differentiation</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>19</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">1273</span>– <span class="NLM_lpage">1281</span>, <span class="refDoi"> DOI: 10.1038/nsmb.2449</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1038%2Fnsmb.2449" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=23160351" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1ymtr%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=1273-1281&issue=12&author=G.+L.+Brienauthor=G.+Gamberoauthor=D.+J.+O%E2%80%99Connellauthor=E.+Jermanauthor=S.+A.+Turnerauthor=C.+M.+Eganauthor=E.+J.+Dunneauthor=M.+C.+Jurgensauthor=K.+Wynneauthor=L.+Piaoauthor=A.+J.+Lohanauthor=N.+Fergusonauthor=X.+Shiauthor=K.+M.+Sinhaauthor=B.+J.+Loftusauthor=G.+Cagneyauthor=A.+P.+Bracken&title=Polycomb+PHF19+Binds+H3K36me3+and+Recruits+PRC2+and+Demethylase+NO66+to+Embryonic+Stem+Cell+Genes+during+Differentiation&doi=10.1038%2Fnsmb.2449"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Polycomb PHF19 binds H3K36me3 and recruits PRC2 and demethylase NO66 to embryonic stem cell genes during differentiation</span></div><div class="casAuthors">Brien, Gerard L.; Gambero, Guillermo; O'Connell, David J.; Jerman, Emilia; Turner, Siobhan A.; Egan, Chris M.; Dunne, Eiseart J.; Jurgens, Maike C.; Wynne, Kieran; Piao, Lianhua; Lohan, Amanda J.; Ferguson, Neil; Shi, Xiaobing; Sinha, Krishna M.; Loftus, Brendan J.; Cagney, Gerard; Bracken, Adrian P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1273-1281</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Polycomb group proteins are repressive chromatin modifiers with essential roles in metazoan development, cellular differentiation and cell fate maintenance.  How Polycomb proteins access active chromatin to confer transcriptional silencing during lineage transitions remains unclear.  Here we show that the Polycomb repressive complex 2 (PRC2) component PHF19 binds trimethylated histone H3 Lys36 (H3K36me3), a mark of active chromatin, via its Tudor domain.  PHF19 assocs. with the H3K36me3 demethylase NO66, and it is required to recruit the PRC2 complex and NO66 to stem cell genes during differentiation, leading to PRC2-mediated trimethylation of histone H3 Lys27 (H3K27), loss of H3K36me3 and transcriptional silencing.  We propose a model whereby PHF19 functions during mouse embryonic stem cell differentiation to transiently bind the H3K36me3 mark via its Tudor domain, forming essential contact points that allow recruitment of PRC2 and H3K36me3 demethylase activity to active gene loci during their transition to a Polycomb-repressed state.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5U3MbFaISUbVg90H21EOLACvtfcHk0lgvQgPWODfEWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1ymtr%252FF&md5=7ae44af5f842916fb16c07361a5b5cef</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.2449&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.2449%26sid%3Dliteratum%253Aachs%26aulast%3DBrien%26aufirst%3DG.%2BL.%26aulast%3DGambero%26aufirst%3DG.%26aulast%3DO%25E2%2580%2599Connell%26aufirst%3DD.%2BJ.%26aulast%3DJerman%26aufirst%3DE.%26aulast%3DTurner%26aufirst%3DS.%2BA.%26aulast%3DEgan%26aufirst%3DC.%2BM.%26aulast%3DDunne%26aufirst%3DE.%2BJ.%26aulast%3DJurgens%26aufirst%3DM.%2BC.%26aulast%3DWynne%26aufirst%3DK.%26aulast%3DPiao%26aufirst%3DL.%26aulast%3DLohan%26aufirst%3DA.%2BJ.%26aulast%3DFerguson%26aufirst%3DN.%26aulast%3DShi%26aufirst%3DX.%26aulast%3DSinha%26aufirst%3DK.%2BM.%26aulast%3DLoftus%26aufirst%3DB.%2BJ.%26aulast%3DCagney%26aufirst%3DG.%26aulast%3DBracken%26aufirst%3DA.%2BP.%26atitle%3DPolycomb%2520PHF19%2520Binds%2520H3K36me3%2520and%2520Recruits%2520PRC2%2520and%2520Demethylase%2520NO66%2520to%2520Embryonic%2520Stem%2520Cell%2520Genes%2520during%2520Differentiation%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2012%26volume%3D19%26issue%3D12%26spage%3D1273%26epage%3D1281%26doi%3D10.1038%2Fnsmb.2449" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gatchalian, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kingsley, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moslet, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosas Ospina, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutateladze, T. G.</span></span> <span> </span><span class="NLM_article-title">An Aromatic Cage Is Required but Not Sufficient for Binding of Tudor Domains of the Polycomblike Protein Family to H3K36me3</span>. <i>Epigenetics</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">467</span>– <span class="NLM_lpage">473</span>, <span class="refDoi"> DOI: 10.1080/15592294.2015.1042646</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1080%2F15592294.2015.1042646" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=25923537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A280%3ADC%252BC2MjoslCmsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=467-473&issue=6&author=J.+Gatchalianauthor=M.+C.+Kingsleyauthor=S.+D.+Mosletauthor=R.+D.+Rosas+Ospinaauthor=T.+G.+Kutateladze&title=An+Aromatic+Cage+Is+Required+but+Not+Sufficient+for+Binding+of+Tudor+Domains+of+the+Polycomblike+Protein+Family+to+H3K36me3&doi=10.1080%2F15592294.2015.1042646"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">An aromatic cage is required but not sufficient for binding of Tudor domains of the Polycomblike protein family to H3K36me3</span></div><div class="casAuthors">Gatchalian Jovylyn; Kingsley Molly C; Moslet Stacey D; Rosas Ospina Ruben D; Kutateladze Tatiana G</div><div class="citationInfo"><span class="NLM_cas:title">Epigenetics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">467-73</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Polycomblike (Pcl) proteins are important transcriptional regulators and components of the Polycomb Repressive Complex 2 (PRC2).  The Tudor domains of human homologs PHF1 and PHF19 have been found to recognize trimethylated lysine 36 of histone H3 (H3K36me3); however, the biological role of Tudor domains of other Pcl proteins remains poorly understood.  Here, we characterize the molecular basis underlying histone binding activities of the Tudor domains of the Pcl family.  In contrast to a predominant view, we found that the methyl lysine-binding aromatic cage is necessary but not sufficient for recognition of H3K36me3 by these Tudor domains and that a hydrophobic patch, adjacent to the aromatic cage, is also required.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRqFf2GlRp7bRfKb6ELK8t0fW6udTcc2eaSSF9RtjMdgbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MjoslCmsw%253D%253D&md5=ba1a1d106047361fd65ccef09236ea37</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1080%2F15592294.2015.1042646&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F15592294.2015.1042646%26sid%3Dliteratum%253Aachs%26aulast%3DGatchalian%26aufirst%3DJ.%26aulast%3DKingsley%26aufirst%3DM.%2BC.%26aulast%3DMoslet%26aufirst%3DS.%2BD.%26aulast%3DRosas%2BOspina%26aufirst%3DR.%2BD.%26aulast%3DKutateladze%26aufirst%3DT.%2BG.%26atitle%3DAn%2520Aromatic%2520Cage%2520Is%2520Required%2520but%2520Not%2520Sufficient%2520for%2520Binding%2520of%2520Tudor%2520Domains%2520of%2520the%2520Polycomblike%2520Protein%2520Family%2520to%2520H3K36me3%26jtitle%3DEpigenetics%26date%3D2015%26volume%3D10%26issue%3D6%26spage%3D467%26epage%3D473%26doi%3D10.1080%2F15592294.2015.1042646" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nekrasov, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klymenko, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraterman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oktaba, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Köcher, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stunnenberg, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilm, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, J.</span></span> <span> </span><span class="NLM_article-title">Pcl-PRC2 Is Needed to Generate High Levels of H3-K27 Trimethylation at Polycomb Target Genes</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>26</i></span> (<span class="NLM_issue">18</span>),  <span class="NLM_fpage">4078</span>– <span class="NLM_lpage">4088</span>, <span class="refDoi"> DOI: 10.1038/sj.emboj.7601837</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1038%2Fsj.emboj.7601837" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=17762866" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVKhtrvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2007&pages=4078-4088&issue=18&author=M.+Nekrasovauthor=T.+Klymenkoauthor=S.+Fratermanauthor=B.+Pappauthor=K.+Oktabaauthor=T.+K%C3%B6cherauthor=A.+Cohenauthor=H.+G.+Stunnenbergauthor=M.+Wilmauthor=J.+M%C3%BCller&title=Pcl-PRC2+Is+Needed+to+Generate+High+Levels+of+H3-K27+Trimethylation+at+Polycomb+Target+Genes&doi=10.1038%2Fsj.emboj.7601837"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Pcl-PRC2 is needed to generate high levels of H3-K27 trimethylation at Polycomb target genes</span></div><div class="casAuthors">Nekrasov, Maxim; Klymenko, Tetyana; Fraterman, Sven; Papp, Bernadett; Oktaba, Katarzyna; Koecher, Thomas; Cohen, Adrian; Stunnenberg, Hendrik G.; Wilm, Matthias; Mueller, Juerg</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">4078-4088</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">PRC2 is thought to be the histone methyltransferase (HMTase) responsible for H3-K27 trimethylation at Polycomb target genes.  Here we report the biochem. purifn. and characterization of a distinct form of Drosophila PRC2 that contains the Polycomb group protein polycomblike (Pcl).  Like PRC2, Pcl-PRC2 is an H3-K27-specific HMTase that mono-, di- and trimethylates H3-K27 in nucleosomes in vitro.  Anal. of Drosophila mutants that lack Pcl unexpectedly reveals that Pcl-PRC2 is required to generate high levels of H3-K27 trimethylation at Polycomb target genes but is dispensable for the genome-wide H3-K27 mono- and dimethylation that is generated by PRC2.  In Pcl mutants, Polycomb target genes become derepressed even though H3-K27 trimethylation at these genes is only reduced and not abolished, and even though targeting of the Polycomb protein complexes PhoRC and PRC1 to Polycomb response elements is not affected.  Pcl-PRC2 is thus the HMTase that generates the high levels of H3-K27 trimethylation in Polycomb target genes that are needed to maintain a Polycomb-repressed chromatin state.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVZCMxfevbBLVg90H21EOLACvtfcHk0limLVKs-sbtLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVKhtrvJ&md5=75db8cee2dc169ccec48065b29d4380e</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fsj.emboj.7601837&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.emboj.7601837%26sid%3Dliteratum%253Aachs%26aulast%3DNekrasov%26aufirst%3DM.%26aulast%3DKlymenko%26aufirst%3DT.%26aulast%3DFraterman%26aufirst%3DS.%26aulast%3DPapp%26aufirst%3DB.%26aulast%3DOktaba%26aufirst%3DK.%26aulast%3DK%25C3%25B6cher%26aufirst%3DT.%26aulast%3DCohen%26aufirst%3DA.%26aulast%3DStunnenberg%26aufirst%3DH.%2BG.%26aulast%3DWilm%26aufirst%3DM.%26aulast%3DM%25C3%25BCller%26aufirst%3DJ.%26atitle%3DPcl-PRC2%2520Is%2520Needed%2520to%2520Generate%2520High%2520Levels%2520of%2520H3-K27%2520Trimethylation%2520at%2520Polycomb%2520Target%2520Genes%26jtitle%3DEMBO%2520J.%26date%3D2007%26volume%3D26%26issue%3D18%26spage%3D4078%26epage%3D4088%26doi%3D10.1038%2Fsj.emboj.7601837" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sarma, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Margueron, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ivanov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pirrotta, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinberg, D.</span></span> <span> </span><span class="NLM_article-title">Ezh2 Requires PHF1 To Efficiently Catalyze H3 Lysine 27 Trimethylation In Vivo</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>28</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">2718</span>– <span class="NLM_lpage">2731</span>, <span class="refDoi"> DOI: 10.1128/MCB.02017-07</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1128%2FMCB.02017-07" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=18285464" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BD1cXksFWqtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2008&pages=2718-2731&issue=8&author=K.+Sarmaauthor=R.+Margueronauthor=A.+Ivanovauthor=V.+Pirrottaauthor=D.+Reinberg&title=Ezh2+Requires+PHF1+To+Efficiently+Catalyze+H3+Lysine+27+Trimethylation+In+Vivo&doi=10.1128%2FMCB.02017-07"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Ezh2 requires PHF1 to efficiently catalyze H3 lysine 27 trimethylation in vivo</span></div><div class="casAuthors">Sarma, Kavitha; Margueron, Raphael; Ivanov, Alexey; Pirrotta, Vincenzo; Reinberg, Danny</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2718-2731</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The mammalian Polycomblike protein PHF1 was previously shown to interact with the Polycomb group (PcG) protein Ezh2, a histone methyltransferase whose activity is pivotal in sustaining gene repression during development and in adulthood.  As Ezh2 is active only when part of the Polycomb Repressive Complexes (PRC2-PRC4), we examd. the functional role of its interaction with PHF1.  Chromatin immunopptn. expts. revealed that PHF1 resides along with Ezh2 at Ezh2-regulated genes such as the HoxA loci and the non-Hox MYT1 and WNT1 genes.  Knockdown of PHF1 or of Ezh2 led to up-regulated HoxA gene expression.  Interestingly, depletion of PHF1 did correlate with reduced occupancy of Bmi-1, a PRC1 component.  As expected, knockdown of Ezh2 led to reduced levels of its catalytic products H3K27me2/H3K27me3.  However, reduced levels of PHF1 also led to decreased global levels of H3K27me3.  Notably, the levels of H3K27me3 decreased while those of H3K27me2 increased at the up-regulated HoxA loci tested.  Consistent with this, the addn. of PHF1 specifically stimulated the ability of Ezh2 to catalyze H3K27me3 but not H3K27me1/H3K27me2 in vitro.  We conclude that PHF1 modulates the activity of Ezh2 in favor of the repressive H3K27me3 mark.  Thus, we propose that PHF1 is a determinant in PcG-mediated gene repression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsiWF9xVC8TbVg90H21EOLACvtfcHk0limLVKs-sbtLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXksFWqtrc%253D&md5=ce25d19e315ef42f53c34f199e81cfcb</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1128%2FMCB.02017-07&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.02017-07%26sid%3Dliteratum%253Aachs%26aulast%3DSarma%26aufirst%3DK.%26aulast%3DMargueron%26aufirst%3DR.%26aulast%3DIvanov%26aufirst%3DA.%26aulast%3DPirrotta%26aufirst%3DV.%26aulast%3DReinberg%26aufirst%3DD.%26atitle%3DEzh2%2520Requires%2520PHF1%2520To%2520Efficiently%2520Catalyze%2520H3%2520Lysine%252027%2520Trimethylation%2520In%2520Vivo%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2008%26volume%3D28%26issue%3D8%26spage%3D2718%26epage%3D2731%26doi%3D10.1128%2FMCB.02017-07" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Choi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bachmann, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tauscher, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benda, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fierz, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, J.</span></span> <span> </span><span class="NLM_article-title">DNA Binding by PHF1 Prolongs PRC2 Residence Time on Chromatin and Thereby Promotes H3K27 Methylation</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>24</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">1039</span>– <span class="NLM_lpage">1047</span>, <span class="refDoi"> DOI: 10.1038/nsmb.3488</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1038%2Fnsmb.3488" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=29058710" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslehtrvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2017&pages=1039-1047&issue=12&author=J.+Choiauthor=A.+L.+Bachmannauthor=K.+Tauscherauthor=C.+Bendaauthor=B.+Fierzauthor=J.+M%C3%BCller&title=DNA+Binding+by+PHF1+Prolongs+PRC2+Residence+Time+on+Chromatin+and+Thereby+Promotes+H3K27+Methylation&doi=10.1038%2Fnsmb.3488"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">DNA binding by PHF1 prolongs PRC2 residence time on chromatin and thereby promotes H3K27 methylation</span></div><div class="casAuthors">Choi, Jeongyoon; Bachmann, Andreas Linus; Tauscher, Katharina; Benda, Christian; Fierz, Beat; Mueller, Juerg</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1039-1047</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Polycomb repressive complex 2 (PRC2) trimethylates histone H3 at lysine 27 to mark genes for repression.  We measured the dynamics of PRC2 binding on recombinant chromatin and free DNA at the single-mol. level using total internal reflection fluorescence (TIRF) microscopy.  PRC2 preferentially binds free DNA with multisecond residence time and midnanomolar affinity.  PHF1, a PRC2 accessory protein of the Polycomblike family, extends PRC2 residence time on DNA and chromatin.  Crystallog. and functional studies reveal that Polycomblike proteins contain a winged-helix domain that binds DNA in a sequence-nonspecific fashion.  DNA binding by this winged-helix domain accounts for the prolonged residence time of PHF1-PRC2 on chromatin and makes it a more efficient H3K27 methyltranferase than PRC2 alone.  Together, these studies establish that interactions with DNA provide the predominant binding affinity of PRC2 for chromatin.  Moreover, they reveal the mol. basis for how Polycomblike proteins stabilize PRC2 on chromatin and stimulate its activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_zfe3Wangt7Vg90H21EOLACvtfcHk0limLVKs-sbtLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslehtrvK&md5=b58c890cba19c8aaf9f624cadf0f8139</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.3488&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.3488%26sid%3Dliteratum%253Aachs%26aulast%3DChoi%26aufirst%3DJ.%26aulast%3DBachmann%26aufirst%3DA.%2BL.%26aulast%3DTauscher%26aufirst%3DK.%26aulast%3DBenda%26aufirst%3DC.%26aulast%3DFierz%26aufirst%3DB.%26aulast%3DM%25C3%25BCller%26aufirst%3DJ.%26atitle%3DDNA%2520Binding%2520by%2520PHF1%2520Prolongs%2520PRC2%2520Residence%2520Time%2520on%2520Chromatin%2520and%2520Thereby%2520Promotes%2520H3K27%2520Methylation%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2017%26volume%3D24%26issue%3D12%26spage%3D1039%26epage%3D1047%26doi%3D10.1038%2Fnsmb.3488" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liefke, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurland, J. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bulyk, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span> <span> </span><span class="NLM_article-title">Polycomb-like Proteins Link the PRC2 Complex to CpG Islands</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>549</i></span> (<span class="NLM_issue">7671</span>),  <span class="NLM_fpage">287</span>– <span class="NLM_lpage">291</span>, <span class="refDoi"> DOI: 10.1038/nature23881</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1038%2Fnature23881" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=28869966" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVers77N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=549&publication_year=2017&pages=287-291&issue=7671&author=H.+Liauthor=R.+Liefkeauthor=J.+Jiangauthor=J.+V.+Kurlandauthor=W.+Tianauthor=P.+Dengauthor=W.+Zhangauthor=Q.+Heauthor=D.+J.+Patelauthor=M.+L.+Bulykauthor=Y.+Shiauthor=Z.+Wang&title=Polycomb-like+Proteins+Link+the+PRC2+Complex+to+CpG+Islands&doi=10.1038%2Fnature23881"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Polycomb-like proteins link the PRC2 complex to CpG islands</span></div><div class="casAuthors">Li, Haojie; Liefke, Robert; Jiang, Junyi; Kurland, Jesse Vigoda; Tian, Wei; Deng, Pujuan; Zhang, Weidi; He, Qian; Patel, Dinshaw J.; Bulyk, Martha L.; Shi, Yang; Wang, Zhanxin</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">549</span>
        (<span class="NLM_cas:issue">7671</span>),
    <span class="NLM_cas:pages">287-291</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The Polycomb repressive complex 2 (PRC2) mainly mediates transcriptional repression and has essential roles in various biol. processes including the maintenance of cell identity and proper differentiation.  Polycomb-like (PCL) proteins, such as PHF1, MTF2 and PHF19, are PRC2-assocd. factors that form sub-complexes with PRC2 core components, and have been proposed to modulate the enzymic activity of PRC2 or the recruitment of PRC2 to specific genomic loci.  Mammalian PRC2-binding sites are enriched in CG content, which correlates with CpG islands that display a low level of DNA methylation.  However, the mechanism of PRC2 recruitment to CpG islands is not fully understood.  Here we solve the crystal structures of the N-terminal domains of PHF1 and MTF2 with bound CpG-contg. DNAs in the presence of H3K36me3-contg. histone peptides.  We show that the extended homologous regions of both proteins fold into a winged-helix structure, which specifically binds to the unmethylated CpG motif but in a completely different manner from the canonical winged-helix DNA recognition motif.  We also show that the PCL extended homologous domains are required for efficient recruitment of PRC2 to CpG island-contg. promoters in mouse embryonic stem cells.  Our research provides the first, to our knowledge, direct evidence to demonstrate that PCL proteins are crucial for PRC2 recruitment to CpG islands, and further clarifies the roles of these proteins in transcriptional regulation in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLtwwW3WZV8bVg90H21EOLACvtfcHk0ljBD9qGQ-fetQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVers77N&md5=4ac008b821e5def7edad924d1c73ff8d</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fnature23881&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature23881%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DH.%26aulast%3DLiefke%26aufirst%3DR.%26aulast%3DJiang%26aufirst%3DJ.%26aulast%3DKurland%26aufirst%3DJ.%2BV.%26aulast%3DTian%26aufirst%3DW.%26aulast%3DDeng%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DHe%26aufirst%3DQ.%26aulast%3DPatel%26aufirst%3DD.%2BJ.%26aulast%3DBulyk%26aufirst%3DM.%2BL.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DZ.%26atitle%3DPolycomb-like%2520Proteins%2520Link%2520the%2520PRC2%2520Complex%2520to%2520CpG%2520Islands%26jtitle%3DNature%26date%3D2017%26volume%3D549%26issue%3D7671%26spage%3D287%26epage%3D291%26doi%3D10.1038%2Fnature23881" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">van
Mierlo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veenstra, G. J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vermeulen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marks, H.</span></span> <span> </span><span class="NLM_article-title">The Complexity of PRC2 Subcomplexes</span>. <i>Trends Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>29</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">660</span>– <span class="NLM_lpage">671</span>, <span class="refDoi"> DOI: 10.1016/j.tcb.2019.05.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1016%2Fj.tcb.2019.05.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=31178244" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtVKju7jJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=660-671&issue=8&author=G.+van%0AMierloauthor=G.+J.+C.+Veenstraauthor=M.+Vermeulenauthor=H.+Marks&title=The+Complexity+of+PRC2+Subcomplexes&doi=10.1016%2Fj.tcb.2019.05.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">The Complexity of PRC2 Subcomplexes</span></div><div class="casAuthors">van Mierlo, Guido; Veenstra, Gert Jan C.; Vermeulen, Michiel; Marks, Hendrik</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Cell Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">660-671</span>CODEN:
                <span class="NLM_cas:coden">TCBIEK</span>;
        ISSN:<span class="NLM_cas:issn">0962-8924</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Polycomb repressive complex 2 (PRC2) is a multisubunit protein complex essential for the development of multicellular organisms.  Recruitment of PRC2 to target genes, followed by deposition and propagation of its catalytic product histone H3 lysine 27 trimethylation (H3K27me3), are key to the spatiotemporal control of developmental gene expression.  Recent breakthrough studies have uncovered unexpected roles for substoichiometric PRC2 subunits in these processes.  Here, we elaborate on how the facultative PRC2 subunits regulate catalytic activity, locus-specific PRC2 binding, and propagation of H3K27me3, and how this affects chromatin structure, gene expression, and cell fate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSF3aGmH5JtbVg90H21EOLACvtfcHk0ljBD9qGQ-fetQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtVKju7jJ&md5=520c1f526a29c9a13a816c70f216a7a4</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.tcb.2019.05.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tcb.2019.05.004%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BMierlo%26aufirst%3DG.%26aulast%3DVeenstra%26aufirst%3DG.%2BJ.%2BC.%26aulast%3DVermeulen%26aufirst%3DM.%26aulast%3DMarks%26aufirst%3DH.%26atitle%3DThe%2520Complexity%2520of%2520PRC2%2520Subcomplexes%26jtitle%3DTrends%2520Cell%2520Biol.%26date%3D2019%26volume%3D29%26issue%3D8%26spage%3D660%26epage%3D671%26doi%3D10.1016%2Fj.tcb.2019.05.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Healy, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mucha, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glancy, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fitzpatrick, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conway, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neikes, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monger, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Mierlo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baltissen, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koseki, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vermeulen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koseki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bracken, A. P.</span></span> <span> </span><span class="NLM_article-title">PRC2.1 and PRC2.2 Synergize to Coordinate H3K27 Trimethylation</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>76</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">437</span>– <span class="NLM_lpage">452.e6</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2019.08.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1016%2Fj.molcel.2019.08.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=31521505" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhslOgs7zK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2019&pages=437-452.e6&issue=3&author=E.+Healyauthor=M.+Muchaauthor=E.+Glancyauthor=D.+J.+Fitzpatrickauthor=E.+Conwayauthor=H.+K.+Neikesauthor=C.+Mongerauthor=G.+Van+Mierloauthor=M.+P.+Baltissenauthor=Y.+Kosekiauthor=M.+Vermeulenauthor=H.+Kosekiauthor=A.+P.+Bracken&title=PRC2.1+and+PRC2.2+Synergize+to+Coordinate+H3K27+Trimethylation&doi=10.1016%2Fj.molcel.2019.08.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">PRC2.1 and PRC2.2 Synergize to Coordinate H3K27 Trimethylation</span></div><div class="casAuthors">Healy, Evan; Mucha, Marlena; Glancy, Eleanor; Fitzpatrick, Darren J.; Conway, Eric; Neikes, Hannah K.; Monger, Craig; Van Mierlo, Guido; Baltissen, Marijke P.; Koseki, Yoko; Vermeulen, Michiel; Koseki, Haruhiko; Bracken, Adrian P.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">437-452.e6</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Polycomb repressive complex 2 (PRC2) is composed of EED, SUZ12, and EZH1/2 and mediates mono-, di-, and trimethylation of histone H3 at lysine 27.  At least two independent subcomplexes exist, defined by their specific accessory proteins: PRC2.1 (PCL1-3, EPOP, and PALI1/2) and PRC2.2 (AEBP2 and JARID2).  We show that PRC2.1 and PRC2.2 share the majority of target genes in mouse embryonic stem cells.  The loss of PCL1-3 is sufficient to evict PRC2.1 from Polycomb target genes but only leads to a partial redn. of PRC2.2 and H3K27me3.  Conversely, disruption of PRC2.2 function through the loss of either JARID2 or RING1A/B is insufficient to completely disrupt targeting of SUZ12 by PCLs.  Instead, the combined loss of both PRC2.1 and PRC2.2 is required, leading to the global mislocalization of SUZ12.  This supports a model in which the specific accessory proteins within PRC2.1 and PRC2.2 cooperate to direct H3K27me3 via both synergistic and independent mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOH_8vEpojlbVg90H21EOLACvtfcHk0ljBD9qGQ-fetQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhslOgs7zK&md5=b61b34ab7fbb78caa0c8daea6d12e88c</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2019.08.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2019.08.012%26sid%3Dliteratum%253Aachs%26aulast%3DHealy%26aufirst%3DE.%26aulast%3DMucha%26aufirst%3DM.%26aulast%3DGlancy%26aufirst%3DE.%26aulast%3DFitzpatrick%26aufirst%3DD.%2BJ.%26aulast%3DConway%26aufirst%3DE.%26aulast%3DNeikes%26aufirst%3DH.%2BK.%26aulast%3DMonger%26aufirst%3DC.%26aulast%3DVan%2BMierlo%26aufirst%3DG.%26aulast%3DBaltissen%26aufirst%3DM.%2BP.%26aulast%3DKoseki%26aufirst%3DY.%26aulast%3DVermeulen%26aufirst%3DM.%26aulast%3DKoseki%26aufirst%3DH.%26aulast%3DBracken%26aufirst%3DA.%2BP.%26atitle%3DPRC2.1%2520and%2520PRC2.2%2520Synergize%2520to%2520Coordinate%2520H3K27%2520Trimethylation%26jtitle%3DMol.%2520Cell%26date%3D2019%26volume%3D76%26issue%3D3%26spage%3D437%26epage%3D452.e6%26doi%3D10.1016%2Fj.molcel.2019.08.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Perino, M.</span>; <span class="NLM_string-name">van Mierlo, G.</span>; <span class="NLM_string-name">Wardle, S. M. T.</span>; <span class="NLM_string-name">Marks, H.</span>; <span class="NLM_string-name">Veenstra, G. J. C.</span></span> <span> </span><span class="NLM_article-title">Two Distinct Functional Axes of Positive Feedback-Enforced PRC2 Recruitment in Mouse Embryonic Stem Cells</span>. <span class="NLM_year">2019</span>, bioRxiv 669960. biorxiv.org e-Print archive. <a href="https://www.biorxiv.org/content/10.1101/669960v1" class="extLink">https://www.biorxiv.org/content/10.1101/669960v1</a>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1101%2F669960" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=M.+Perino&author=G.+van+Mierlo&author=S.+M.+T.+Wardle&author=H.+Marks&author=G.+J.+C.+Veenstra&title=Two+Distinct+Functional+Axes+of+Positive+Feedback-Enforced+PRC2+Recruitment+in+Mouse+Embryonic+Stem+Cells"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1101%2F669960&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26id%3Ddoi%3A10.1101%252F669960%26sid%3Dliteratum%253Aachs%26aulast%3DPerino%26aufirst%3DM.%26atitle%3DTwo%2520Distinct%2520Functional%2520Axes%2520of%2520Positive%2520Feedback-Enforced%2520PRC2%2520Recruitment%2520in%2520Mouse%2520Embryonic%2520Stem%2520Cells%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Micci, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunetti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dal Cin, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nucci, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorunova, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panagopoulos, I.</span></span> <span> </span><span class="NLM_article-title">Fusion of the Genes BRD8 and PHF1 in Endometrial Stromal Sarcoma</span>. <i>Genes, Chromosomes Cancer</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">841</span>– <span class="NLM_lpage">845</span>, <span class="refDoi"> DOI: 10.1002/gcc.22485</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1002%2Fgcc.22485" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=28758277" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVaqsrrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=841-845&issue=12&author=F.+Micciauthor=M.+Brunettiauthor=P.+Dal+Cinauthor=M.+R.+Nucciauthor=L.+Gorunovaauthor=S.+Heimauthor=I.+Panagopoulos&title=Fusion+of+the+Genes+BRD8+and+PHF1+in+Endometrial+Stromal+Sarcoma&doi=10.1002%2Fgcc.22485"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Fusion of the genes BRD8 and PHF1 in endometrial stromal sarcoma</span></div><div class="casAuthors">Micci, Francesca; Brunetti, Marta; Dal Cin, Paola; Nucci, Marisa R.; Gorunova, Ludmila; Heim, Sverre; Panagopoulos, Ioannis</div><div class="citationInfo"><span class="NLM_cas:title">Genes, Chromosomes & Cancer</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">841-845</span>CODEN:
                <span class="NLM_cas:coden">GCCAES</span>;
        ISSN:<span class="NLM_cas:issn">1045-2257</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">We present a new endometrial stromal sarcoma (ESS)-assocd. genomic rearrangement involving chromosome arms 5p and 6p and leading to the formation of a BRD8-PHF1 fusion gene.  The PHF1 (PHD finger protein 1) gene, from 6p21, is known to be rearranged in ESS in a promiscuous way inasmuch as it has been shown to recombine with JAZF1, EPC1, MEAF6, and now also with BRD8, in tumors of this type.  In all rearrangements of PHF1, including the present one, a recurrent theme is that the entire coding part of PHF1 constitutes the 3' end of the fusion.  BRD8 (bromodomain contg. 8) encodes a protein which is involved in regulation of protein acetylation and/or histone acetyl transferase activity.  All the genetic fusions identified so far in ESS appear to recombine genes involved in transcriptional regulation, i.e., polycomb group complex-mediated and aberrant methylation/acetylation genes.  This adds to the likelihood that the new BRD8-PHF1 shares the same pathogenetic mechanism as the other ESS-specific rearrangements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNmV1SZ_bWx7Vg90H21EOLACvtfcHk0lgATxw8H5rn-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVaqsrrI&md5=2c516c6966038de383f9cc4005576aa5</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1002%2Fgcc.22485&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fgcc.22485%26sid%3Dliteratum%253Aachs%26aulast%3DMicci%26aufirst%3DF.%26aulast%3DBrunetti%26aufirst%3DM.%26aulast%3DDal%2BCin%26aufirst%3DP.%26aulast%3DNucci%26aufirst%3DM.%2BR.%26aulast%3DGorunova%26aufirst%3DL.%26aulast%3DHeim%26aufirst%3DS.%26aulast%3DPanagopoulos%26aufirst%3DI.%26atitle%3DFusion%2520of%2520the%2520Genes%2520BRD8%2520and%2520PHF1%2520in%2520Endometrial%2520Stromal%2520Sarcoma%26jtitle%3DGenes%252C%2520Chromosomes%2520Cancer%26date%3D2017%26volume%3D56%26issue%3D12%26spage%3D841%26epage%3D845%26doi%3D10.1002%2Fgcc.22485" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Panagopoulos, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Micci, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thorsen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorunova, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eibak, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bjerkehagen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davidson, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heim, S.</span></span> <span> </span><span class="NLM_article-title">Novel Fusion of MYST/Esa1-Associated Factor 6 and PHF1 in Endometrial Stromal Sarcoma</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">e39354</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0039354</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1371%2Fjournal.pone.0039354" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=22761769" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC38XpvVWltrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=e39354&issue=6&author=I.+Panagopoulosauthor=F.+Micciauthor=J.+Thorsenauthor=L.+Gorunovaauthor=A.+M.+Eibakauthor=B.+Bjerkehagenauthor=B.+Davidsonauthor=S.+Heim&title=Novel+Fusion+of+MYST%2FEsa1-Associated+Factor+6+and+PHF1+in+Endometrial+Stromal+Sarcoma&doi=10.1371%2Fjournal.pone.0039354"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Novel fusion of MYST/Esa1-associated factor 6 and PHF1 in endometrial stromal sarcoma</span></div><div class="casAuthors">Panagopoulos, Ioannis; Micci, Francesca; Thorsen, Jim; Gorunova, Ludmila; Eibak, Anne Mette; Bjerkehagen, Bodil; Davidson, Ben; Heim, Sverre</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">e39354</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Rearrangement of chromosome band 6p21 is recurrent in endometrial stromal sarcoma (ESS) and targets the PHF1 gene.  So far, PHF1 was found to be the 3' partner in the JAZF1-PHF1 and EPC1-PHF1 chimeras but since the 6p21 rearrangements involve also other chromosomal translocation partners, other PHF1-fusions seem likely.  Here, we show that PHF1 is recombined with a novel fusion partner, MEAF6 from 1p34, in an ESS carrying a t(1;6)(p34;p21) translocation as the sole karyotypic anomaly.  5'-RACE, RT-PCR, and sequencing showed the presence of an MEAF6-PHF1 chimera in the tumor with exon 5 of MEAF6 being fused in-frame to exon 2 of PHF1 so that the entire PHF1 coding region becomes the 3' terminal part of the MEAF6-PHF1 fusion.  The predicted fusion protein is composed of 750 amino acids and contains the histone acetyltransferase subunit NuA4 domain of MEAF6 and the tudor, PHD zinc finger, and MTF2 domains of PHF1.  Although the specific functions of the MEAF6 and PHF1 proteins and why they are targeted by a neoplasia-specific gene fusion are not directly apparent, it seems that rearrangement of genes involved in acetylation (EPC1, MEAF6) and methylation (PHF1), resulting in aberrant gene expression, is a common theme in ESS pathogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkx5ulZ__TY7Vg90H21EOLACvtfcHk0lgATxw8H5rn-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XpvVWltrg%253D&md5=d4984aef237ec87a0b1d2064dda4a4ba</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0039354&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0039354%26sid%3Dliteratum%253Aachs%26aulast%3DPanagopoulos%26aufirst%3DI.%26aulast%3DMicci%26aufirst%3DF.%26aulast%3DThorsen%26aufirst%3DJ.%26aulast%3DGorunova%26aufirst%3DL.%26aulast%3DEibak%26aufirst%3DA.%2BM.%26aulast%3DBjerkehagen%26aufirst%3DB.%26aulast%3DDavidson%26aufirst%3DB.%26aulast%3DHeim%26aufirst%3DS.%26atitle%3DNovel%2520Fusion%2520of%2520MYST%252FEsa1-Associated%2520Factor%25206%2520and%2520PHF1%2520in%2520Endometrial%2520Stromal%2520Sarcoma%26jtitle%3DPLoS%2520One%26date%3D2012%26volume%3D7%26issue%3D6%26spage%3De39354%26doi%3D10.1371%2Fjournal.pone.0039354" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gebre-Medhin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nord, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Möller, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mandahl, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnusson, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nilsson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jo, V. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vult Von Steyern, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brosjö, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsson, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Domanski, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sciot, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Debiec-Rychter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fletcher, C. D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mertens, F.</span></span> <span> </span><span class="NLM_article-title">Recurrent Rearrangement of the PHF1 Gene in Ossifying Fibromyxoid Tumors</span>. <i>Am. J. Pathol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>181</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">1069</span>– <span class="NLM_lpage">1077</span>, <span class="refDoi"> DOI: 10.1016/j.ajpath.2012.05.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1016%2Fj.ajpath.2012.05.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=22796436" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFSns7nJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=181&publication_year=2012&pages=1069-1077&issue=3&author=S.+Gebre-Medhinauthor=K.+H.+Nordauthor=E.+M%C3%B6llerauthor=N.+Mandahlauthor=L.+Magnussonauthor=J.+Nilssonauthor=V.+Y.+Joauthor=F.+Vult+Von+Steyernauthor=O.+Brosj%C3%B6author=O.+Larssonauthor=H.+A.+Domanskiauthor=R.+Sciotauthor=M.+Debiec-Rychterauthor=C.+D.+M.+Fletcherauthor=F.+Mertens&title=Recurrent+Rearrangement+of+the+PHF1+Gene+in+Ossifying+Fibromyxoid+Tumors&doi=10.1016%2Fj.ajpath.2012.05.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Recurrent rearrangement of the PHF1 gene in ossifying fibromyxoid tumors</span></div><div class="casAuthors">Gebre-Medhin, Samuel; Nord, Karolin H.; Moller, Emely; Mandahl, Nils; Magnusson, Linda; Nilsson, Jenny; Jo, Vickie Y.; Vult von Steyern, Fredrik; Brosjo, Otte; Larsson, Olle; Domanski, Henryk A.; Sciot, Raf; Debiec-Rychter, Maria; Fletcher, Christopher D. M.; Mertens, Fredrik</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Pathology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">181</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1069-1077</span>CODEN:
                <span class="NLM_cas:coden">AJPAA4</span>;
        ISSN:<span class="NLM_cas:issn">0002-9440</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Ossifying fibromyxoid tumor (OFMT) is a soft tissue tumor of unknown lineage.  Although most cases are histol. and clin. benign, some show malignant morphol. features and local recurrences are not uncommon; a few may even metastasize.  In the present study, cytogenetic anal. identified different structural rearrangements of chromosome band 6p21 in tumor cells from three cases of OFMT, including one with typical, one with atypical, and one with malignant morphol. features.  Mapping of the 6p21 breakpoint by fluorescence in situ hybridization (FISH) indicated that the PHF1 gene was rearranged in all three cases.  Further FISH, 5'-rapid amplification of cDNA ends, and RT-PCR analyses disclosed an EP400/PHF1 fusion transcript in one of the cases.  Interphase FISH on tumor sections from 13 addnl. cases of OFMT showed rearrangement of the PHF1 locus in four of four typical, two of three atypical, and one of six malignant lesions.  Thus, the PHF1 gene, previously shown to be the 3'-partner of fusion genes in endometrial stromal tumors, is also recurrently involved in the pathogenesis of OFMTs, irresp. of whether they are diagnosed as typical, atypical, or malignant lesions.  The PHF1 protein interacts with the polycomb-repressive complex 2 (PRC2), which, in turn, regulates the expression of a variety of developmental genes.  Thus, the results indicate that deregulation of PRC2 target genes is crucial for OFMT development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOKDxTyWmFHrVg90H21EOLACvtfcHk0lgATxw8H5rn-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFSns7nJ&md5=798b60bce3f6e100b9420fd07da9aa1b</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.ajpath.2012.05.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ajpath.2012.05.030%26sid%3Dliteratum%253Aachs%26aulast%3DGebre-Medhin%26aufirst%3DS.%26aulast%3DNord%26aufirst%3DK.%2BH.%26aulast%3DM%25C3%25B6ller%26aufirst%3DE.%26aulast%3DMandahl%26aufirst%3DN.%26aulast%3DMagnusson%26aufirst%3DL.%26aulast%3DNilsson%26aufirst%3DJ.%26aulast%3DJo%26aufirst%3DV.%2BY.%26aulast%3DVult%2BVon%2BSteyern%26aufirst%3DF.%26aulast%3DBrosj%25C3%25B6%26aufirst%3DO.%26aulast%3DLarsson%26aufirst%3DO.%26aulast%3DDomanski%26aufirst%3DH.%2BA.%26aulast%3DSciot%26aufirst%3DR.%26aulast%3DDebiec-Rychter%26aufirst%3DM.%26aulast%3DFletcher%26aufirst%3DC.%2BD.%2BM.%26aulast%3DMertens%26aufirst%3DF.%26atitle%3DRecurrent%2520Rearrangement%2520of%2520the%2520PHF1%2520Gene%2520in%2520Ossifying%2520Fibromyxoid%2520Tumors%26jtitle%3DAm.%2520J.%2520Pathol.%26date%3D2012%26volume%3D181%26issue%3D3%26spage%3D1069%26epage%3D1077%26doi%3D10.1016%2Fj.ajpath.2012.05.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Endo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohashi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishii, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshida, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsunobu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwamoto, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oda, Y.</span></span> <span> </span><span class="NLM_article-title">Ossifying Fibromyxoid Tumor Presenting EP400-PHF1 Fusion Gene</span>. <i>Hum. Pathol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>44</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">2603</span>– <span class="NLM_lpage">2608</span>, <span class="refDoi"> DOI: 10.1016/j.humpath.2013.04.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1016%2Fj.humpath.2013.04.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=23806526" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVSltr3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2013&pages=2603-2608&issue=11&author=M.+Endoauthor=K.+Kohashiauthor=H.+Yamamotoauthor=T.+Ishiiauthor=T.+Yoshidaauthor=T.+Matsunobuauthor=Y.+Iwamotoauthor=Y.+Oda&title=Ossifying+Fibromyxoid+Tumor+Presenting+EP400-PHF1+Fusion+Gene&doi=10.1016%2Fj.humpath.2013.04.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Ossifying fibromyxoid tumor presenting EP400-PHF1 fusion gene</span></div><div class="casAuthors">Endo, Makoto; Kohashi, Kenichi; Yamamoto, Hidetaka; Ishii, Takeaki; Yoshida, Tatsuya; Matsunobu, Tomoya; Iwamoto, Yukihide; Oda, Yoshinao</div><div class="citationInfo"><span class="NLM_cas:title">Human Pathology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2603-2608</span>CODEN:
                <span class="NLM_cas:coden">HPCQA4</span>;
        ISSN:<span class="NLM_cas:issn">0046-8177</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Ossifying fibromyxoid tumor is a rare soft tissue tumor of borderline malignancy and uncertain differentiation.  Recently, a novel fusion gene, EP400-PHF1, was discovered in ossifying fibromyxoid tumor; however, its relation to this type of tumor has been uncertain because the EP400-PHF1 fusion gene has been successfully detected in only 1 case.  We present an ossifying fibromyxoid tumor case with the EP400-PHF1 fusion gene detected by reverse transcriptase polymerase chain reaction, along with compatible cytogenetic data showing a t(6;12)(p21;q24.3) translocation.  Our results suggest that the EP400-PHF1 fusion gene is a reproducible finding in ossifying fibromyxoid tumor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoo6yctwzQ5cLVg90H21EOLACvtfcHk0lhSXVO9M5H-Jw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVSltr3E&md5=e5feac80be06543b8d4e70d55d7fd12b</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.humpath.2013.04.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.humpath.2013.04.003%26sid%3Dliteratum%253Aachs%26aulast%3DEndo%26aufirst%3DM.%26aulast%3DKohashi%26aufirst%3DK.%26aulast%3DYamamoto%26aufirst%3DH.%26aulast%3DIshii%26aufirst%3DT.%26aulast%3DYoshida%26aufirst%3DT.%26aulast%3DMatsunobu%26aufirst%3DT.%26aulast%3DIwamoto%26aufirst%3DY.%26aulast%3DOda%26aufirst%3DY.%26atitle%3DOssifying%2520Fibromyxoid%2520Tumor%2520Presenting%2520EP400-PHF1%2520Fusion%2520Gene%26jtitle%3DHum.%2520Pathol.%26date%3D2013%26volume%3D44%26issue%3D11%26spage%3D2603%26epage%3D2608%26doi%3D10.1016%2Fj.humpath.2013.04.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shokat, K. M.</span></span> <span> </span><span class="NLM_article-title">Recognizing and Exploiting Differences between RNAi and Small-Molecule Inhibitors</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">739</span>– <span class="NLM_lpage">744</span>, <span class="refDoi"> DOI: 10.1038/nchembio1207-739</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1038%2Fnchembio1207-739" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=18007642" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlWqurjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2007&pages=739-744&author=W.+A.+Weissauthor=S.+S.+Taylorauthor=K.+M.+Shokat&title=Recognizing+and+Exploiting+Differences+between+RNAi+and+Small-Molecule+Inhibitors&doi=10.1038%2Fnchembio1207-739"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Recognizing and exploiting differences between RNAi and small-molecule inhibitors</span></div><div class="casAuthors">Weiss, William A.; Taylor, Stephen S.; Shokat, Kevan M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">739-744</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The biol. of RNA interference has greatly facilitated anal. of loss-of-function phenotypes, but correlating these phenotypes with small-mol. inhibition profiles is not always straightforward.  The authors examine the rationale of comparing RNA interference to pharmacol. intervention in chem. biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9JPzqVyshS7Vg90H21EOLACvtfcHk0lhSXVO9M5H-Jw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlWqurjO&md5=9a26c990b358a6b2c717d1e361cac2da</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1038%2Fnchembio1207-739&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio1207-739%26sid%3Dliteratum%253Aachs%26aulast%3DWeiss%26aufirst%3DW.%2BA.%26aulast%3DTaylor%26aufirst%3DS.%2BS.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26atitle%3DRecognizing%2520and%2520Exploiting%2520Differences%2520between%2520RNAi%2520and%2520Small-Molecule%2520Inhibitors%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2007%26volume%3D3%26spage%3D739%26epage%3D744%26doi%3D10.1038%2Fnchembio1207-739" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kycia, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kudithipudi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamas, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kungulovski, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhayalan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeltsch, A.</span></span> <span> </span><span class="NLM_article-title">The Tudor Domain of the PHD Finger Protein 1 Is a Dual Reader of Lysine Trimethylation at Lysine 36 of Histone H3 and Lysine 27 of Histone Variant H3t</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>426</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">1651</span>– <span class="NLM_lpage">1660</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2013.08.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1016%2Fj.jmb.2013.08.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=23954330" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVSksrnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=426&publication_year=2014&pages=1651-1660&issue=8&author=I.+Kyciaauthor=S.+Kudithipudiauthor=R.+Tamasauthor=G.+Kungulovskiauthor=A.+Dhayalanauthor=A.+Jeltsch&title=The+Tudor+Domain+of+the+PHD+Finger+Protein+1+Is+a+Dual+Reader+of+Lysine+Trimethylation+at+Lysine+36+of+Histone+H3+and+Lysine+27+of+Histone+Variant+H3t&doi=10.1016%2Fj.jmb.2013.08.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">The Tudor Domain of the PHD Finger Protein 1 Is a Dual Reader of Lysine Trimethylation at Lysine 36 of Histone H3 and Lysine 27 of Histone Variant H3t</span></div><div class="casAuthors">Kycia, Ina; Kudithipudi, Srikanth; Tamas, Raluca; Kungulovski, Goran; Dhayalan, Arunkumar; Jeltsch, Albert</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">426</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1651-1660</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">PHF1 assocs. with the Polycomb repressive complex 2 and it was demonstrated to stimulate its H3K27-trimethylation activity.  We studied the interaction of the PHF1 Tudor domain with modified histone peptides and found that it recognizes H3K36me3 and H3tK27me3 (on the histone variant H3t) and that it uses the same trimethyllysine binding pocket for the interaction with both peptides.  Since both peptide sequences are very different, this result indicates that reading domains can have dual specificities.  Sub-nuclear localization studies of full-length PHF1 in human HEK293 cells revealed that it co-localizes with K27me3, but not with K36me3, and that this co-localization depends on the trimethyllysine binding pocket indicating that K27me3 is an in vivo target for the PHF1 Tudor domain.  Our data suggest that PHF1 binds to H3tK27me3 in human chromatin, and H3t has a more general role in Polycomb regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq85AlKyIAm3LVg90H21EOLACvtfcHk0lhSXVO9M5H-Jw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVSksrnK&md5=c7e974212da3bf9f929f3ce9726f7b88</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2013.08.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2013.08.009%26sid%3Dliteratum%253Aachs%26aulast%3DKycia%26aufirst%3DI.%26aulast%3DKudithipudi%26aufirst%3DS.%26aulast%3DTamas%26aufirst%3DR.%26aulast%3DKungulovski%26aufirst%3DG.%26aulast%3DDhayalan%26aufirst%3DA.%26aulast%3DJeltsch%26aufirst%3DA.%26atitle%3DThe%2520Tudor%2520Domain%2520of%2520the%2520PHD%2520Finger%2520Protein%25201%2520Is%2520a%2520Dual%2520Reader%2520of%2520Lysine%2520Trimethylation%2520at%2520Lysine%252036%2520of%2520Histone%2520H3%2520and%2520Lysine%252027%2520of%2520Histone%2520Variant%2520H3t%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2014%26volume%3D426%26issue%3D8%26spage%3D1651%26epage%3D1660%26doi%3D10.1016%2Fj.jmb.2013.08.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dong, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakagawa, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oyama, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshimura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamiya, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakayama, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueda, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, J.</span></span> <span> </span><span class="NLM_article-title">Structural Basis for Histone Variant H3tk27me3 Recognition by Phf1 and Phf19</span>. <i>eLife</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">21</span>, <span class="refDoi"> DOI: 10.7554/eLife.58675</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.7554%2FeLife.58675" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2020&pages=1-21&author=C.+Dongauthor=R.+Nakagawaauthor=K.+Oyamaauthor=Y.+Yamamotoauthor=W.+Zhangauthor=A.+Dongauthor=Y.+Liauthor=Y.+Yoshimuraauthor=H.+Kamiyaauthor=J.+I.+Nakayamaauthor=J.+Uedaauthor=J.+Min&title=Structural+Basis+for+Histone+Variant+H3tk27me3+Recognition+by+Phf1+and+Phf19&doi=10.7554%2FeLife.58675"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.7554%2FeLife.58675&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.58675%26sid%3Dliteratum%253Aachs%26aulast%3DDong%26aufirst%3DC.%26aulast%3DNakagawa%26aufirst%3DR.%26aulast%3DOyama%26aufirst%3DK.%26aulast%3DYamamoto%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DYoshimura%26aufirst%3DY.%26aulast%3DKamiya%26aufirst%3DH.%26aulast%3DNakayama%26aufirst%3DJ.%2BI.%26aulast%3DUeda%26aufirst%3DJ.%26aulast%3DMin%26aufirst%3DJ.%26atitle%3DStructural%2520Basis%2520for%2520Histone%2520Variant%2520H3tk27me3%2520Recognition%2520by%2520Phf1%2520and%2520Phf19%26jtitle%3DeLife%26date%3D2020%26volume%3D9%26spage%3D1%26epage%3D21%26doi%3D10.7554%2FeLife.58675" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Simhadri, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daze, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Douglas, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quon, T. T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gignac, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heller, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boulanger, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wulff, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hof, F.</span></span> <span> </span><span class="NLM_article-title">Chromodomain Antagonists That Target the Polycomb-Group Methyllysine Reader Protein Chromobox Homolog 7 (CBX7)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">2874</span>– <span class="NLM_lpage">2883</span>, <span class="refDoi"> DOI: 10.1021/jm401487x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401487x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC2cXktlenu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=2874-2883&issue=7&author=C.+Simhadriauthor=K.+D.+Dazeauthor=S.+F.+Douglasauthor=T.+T.+H.+Quonauthor=A.+Devauthor=M.+C.+Gignacauthor=F.+Pengauthor=M.+Hellerauthor=M.+J.+Boulangerauthor=J.+E.+Wulffauthor=F.+Hof&title=Chromodomain+Antagonists+That+Target+the+Polycomb-Group+Methyllysine+Reader+Protein+Chromobox+Homolog+7+%28CBX7%29&doi=10.1021%2Fjm401487x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Chromodomain Antagonists That Target the Polycomb-Group Methyllysine Reader Protein Chromobox Homolog 7 (CBX7)</span></div><div class="casAuthors">Simhadri, Chakravarthi; Daze, Kevin D.; Douglas, Sarah F.; Quon, Taylor T. H.; Dev, Amarjot; Gignac, Michael C.; Peng, Fangni; Heller, Markus; Boulanger, Martin J.; Wulff, Jeremy E.; Hof, Fraser</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2874-2883</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors report a peptide-driven approach to create the first inhibitors of the chromobox homolog 7 (CBX7), a methyllysine reader protein.  CBX7 uses its chromodomain to bind histone 3, lysine 27 trimethylated (H3K27me3), and this recognition event is implicated in silencing multiple tumor suppressors.  Small trimethyllysine contg.-peptides were used as the basic scaffold from which potent ligands for disruption of CBX7-H3K27me3 complex were developed.  Potency of ligands was detd. by fluorescence polarization and/or isothermal titrn. calorimetry.  Binding of one ligand was characterized in detail using 2D NMR and x-ray crystallog., revealing a structural motif unique among human CBX proteins.  Inhibitors with a ∼200 nM potency for CBX7 binding and 10-fold/400-fold selectivity over related CBX8/CBX1 proteins were identified.  These are the first reported inhibitors of any chromodomain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgJsmWqbn71bVg90H21EOLACvtfcHk0ljd3s6u-6rtVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXktlenu7s%253D&md5=d3f5c56c736a6de5cbb742755e3ec110</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fjm401487x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401487x%26sid%3Dliteratum%253Aachs%26aulast%3DSimhadri%26aufirst%3DC.%26aulast%3DDaze%26aufirst%3DK.%2BD.%26aulast%3DDouglas%26aufirst%3DS.%2BF.%26aulast%3DQuon%26aufirst%3DT.%2BT.%2BH.%26aulast%3DDev%26aufirst%3DA.%26aulast%3DGignac%26aufirst%3DM.%2BC.%26aulast%3DPeng%26aufirst%3DF.%26aulast%3DHeller%26aufirst%3DM.%26aulast%3DBoulanger%26aufirst%3DM.%2BJ.%26aulast%3DWulff%26aufirst%3DJ.%2BE.%26aulast%3DHof%26aufirst%3DF.%26atitle%3DChromodomain%2520Antagonists%2520That%2520Target%2520the%2520Polycomb-Group%2520Methyllysine%2520Reader%2520Protein%2520Chromobox%2520Homolog%25207%2520%2528CBX7%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26issue%3D7%26spage%3D2874%26epage%3D2883%26doi%3D10.1021%2Fjm401487x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Milosevich, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gignac, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McFarlane, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simhadri, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horvath, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daze, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Croft, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dheri, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quon, T. T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Douglas, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wulff, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paci, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hof, F.</span></span> <span> </span><span class="NLM_article-title">Selective Inhibition of CBX6: A Methyllysine Reader Protein in the Polycomb Family</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">139</span>– <span class="NLM_lpage">144</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.5b00378</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00378" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFKjtb%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=139-144&issue=2&author=N.+Milosevichauthor=M.+C.+Gignacauthor=J.+McFarlaneauthor=C.+Simhadriauthor=S.+Horvathauthor=K.+D.+Dazeauthor=C.+S.+Croftauthor=A.+Dheriauthor=T.+T.+H.+Quonauthor=S.+F.+Douglasauthor=J.+E.+Wulffauthor=I.+Paciauthor=F.+Hof&title=Selective+Inhibition+of+CBX6%3A+A+Methyllysine+Reader+Protein+in+the+Polycomb+Family&doi=10.1021%2Facsmedchemlett.5b00378"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Inhibition of CBX6: A Methyllysine Reader Protein in the Polycomb Family</span></div><div class="casAuthors">Milosevich, Natalia; Gignac, Michael C.; McFarlane, James; Simhadri, Chakravarthi; Horvath, Shanti; Daze, Kevin D.; Croft, Caitlin S.; Dheri, Aman; Quon, Taylor T. H.; Douglas, Sarah F.; Wulff, Jeremy E.; Paci, Irina; Hof, Fraser</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">139-144</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The polycomb paralogs CBX2, CBX4, CBX6, CBX7, and CBX8 are epigenetic readers that rely on "arom. cage" motifs to engage their partners' methyllysine side chains.  Each CBX carries out distinct functions, yet each includes a highly similar methyllysine-reading chromodomain as a key element.  CBX7 is the only chromodomain that has yet been targeted by chem. inhibition.  We report a small set of peptidomimetic agents in which a simple chem. modification switches the ligands from one with promiscuity across all polycomb paralogs to one that provides selective inhibition of CBX6.  The structural basis for this selectivity, which involves occupancy of a small hydrophobic pocket adjacent to the arom. cage, was confirmed through mol. dynamics simulations.  Our results demonstrate the increases in affinity and selectivity generated by ligands that engage extended regions of chromodomain binding surfaces.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqO9JIy3HQmrrVg90H21EOLACvtfcHk0ljd3s6u-6rtVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFKjtb%252FP&md5=905994e530d26186495e94b923279d1f</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00378&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00378%26sid%3Dliteratum%253Aachs%26aulast%3DMilosevich%26aufirst%3DN.%26aulast%3DGignac%26aufirst%3DM.%2BC.%26aulast%3DMcFarlane%26aufirst%3DJ.%26aulast%3DSimhadri%26aufirst%3DC.%26aulast%3DHorvath%26aufirst%3DS.%26aulast%3DDaze%26aufirst%3DK.%2BD.%26aulast%3DCroft%26aufirst%3DC.%2BS.%26aulast%3DDheri%26aufirst%3DA.%26aulast%3DQuon%26aufirst%3DT.%2BT.%2BH.%26aulast%3DDouglas%26aufirst%3DS.%2BF.%26aulast%3DWulff%26aufirst%3DJ.%2BE.%26aulast%3DPaci%26aufirst%3DI.%26aulast%3DHof%26aufirst%3DF.%26atitle%3DSelective%2520Inhibition%2520of%2520CBX6%253A%2520A%2520Methyllysine%2520Reader%2520Protein%2520in%2520the%2520Polycomb%2520Family%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D7%26issue%3D2%26spage%3D139%26epage%3D144%26doi%3D10.1021%2Facsmedchemlett.5b00378" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denton, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hobbs, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weaver, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McFarlane, J. M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Connelly, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gignac, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milosevich, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hof, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paci, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Musselman, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dykhuizen, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krusemark, C. J.</span></span> <span> </span><span class="NLM_article-title">Optimization of Ligands Using Focused DNA-Encoded Libraries to Develop a Selective, Cell-Permeable CBX8 Chromodomain Inhibitor</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>15</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">112</span>– <span class="NLM_lpage">131</span>, <span class="refDoi"> DOI: 10.1021/acschembio.9b00654</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.9b00654" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1WktL7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2020&pages=112-131&issue=1&author=S.+Wangauthor=K.+E.+Dentonauthor=K.+F.+Hobbsauthor=T.+Weaverauthor=J.+M.+B.+McFarlaneauthor=K.+E.+Connellyauthor=M.+C.+Gignacauthor=N.+Milosevichauthor=F.+Hofauthor=I.+Paciauthor=C.+A.+Musselmanauthor=E.+C.+Dykhuizenauthor=C.+J.+Krusemark&title=Optimization+of+Ligands+Using+Focused+DNA-Encoded+Libraries+to+Develop+a+Selective%2C+Cell-Permeable+CBX8+Chromodomain+Inhibitor&doi=10.1021%2Facschembio.9b00654"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of Ligands Using Focused DNA-Encoded Libraries To Develop a Selective, Cell-Permeable CBX8 Chromodomain Inhibitor</span></div><div class="casAuthors">Wang, Sijie; Denton, Kyle E.; Hobbs, Kathryn F.; Weaver, Tyler; McFarlane, James M. B.; Connelly, Katelyn E.; Gignac, Michael C.; Milosevich, Natalia; Hof, Fraser; Paci, Irina; Musselman, Catherine A.; Dykhuizen, Emily C.; Krusemark, Casey J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">112-131</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Polycomb repressive complex 1 (PRC1) is crit. for mediating gene expression during development.  Five chromobox (CBX) homolog proteins, CBX2, CBX4, CBX6, CBX7, and CBX8, are incorporated into PRC1 complexes, where they mediate targeting to trimethylated lysine 27 of histone H3 (H3K27me3) via the N-terminal chromodomain (ChD).  Individual CBX paralogs have been implicated as drug targets in cancer; however, high similarities in sequence and structure among the CBX ChDs provide a major obstacle in developing selective CBX ChD inhibitors.  Here we report the selection of small, focused, DNA-encoded libraries (DELs) against multiple homologous ChDs to identify modifications to a parental ligand that confer both selectivity and potency for the ChD of CBX8.  This on-DNA, medicinal chem. approach enabled the development of SW2_110A, a selective, cell-permeable inhibitor of the CBX8 ChD.  SW2_110A binds CBX8 ChD with a Kd of 800 nM, with minimal 5-fold selectivity for CBX8 ChD over all other CBX paralogs in vitro.  SW2_110A specifically inhibits the assocn. of CBX8 with chromatin in cells and inhibits the proliferation of THP1 leukemia cells driven by the MLL-AF9 translocation.  In THP1 cells, SW2_110A treatment results in a significant decrease in the expression of MLL-AF9 target genes, including HOXA9, validating the previously established role for CBX8 in MLL-AF9 transcriptional activation, and defining the ChD as necessary for this function.  The success of SW2_110A provides great promise for the development of highly selective and cell-permeable probes for the full CBX family.  In addn., the approach taken provides a proof-of-principle demonstration of how DELs can be used iteratively for optimization of both ligand potency and selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpG4CHEZDuLrLVg90H21EOLACvtfcHk0lhf93MdDciA3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1WktL7O&md5=69aa429f81136431674a6c704a7ab388</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Facschembio.9b00654&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.9b00654%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DS.%26aulast%3DDenton%26aufirst%3DK.%2BE.%26aulast%3DHobbs%26aufirst%3DK.%2BF.%26aulast%3DWeaver%26aufirst%3DT.%26aulast%3DMcFarlane%26aufirst%3DJ.%2BM.%2BB.%26aulast%3DConnelly%26aufirst%3DK.%2BE.%26aulast%3DGignac%26aufirst%3DM.%2BC.%26aulast%3DMilosevich%26aufirst%3DN.%26aulast%3DHof%26aufirst%3DF.%26aulast%3DPaci%26aufirst%3DI.%26aulast%3DMusselman%26aufirst%3DC.%2BA.%26aulast%3DDykhuizen%26aufirst%3DE.%2BC.%26aulast%3DKrusemark%26aufirst%3DC.%2BJ.%26atitle%3DOptimization%2520of%2520Ligands%2520Using%2520Focused%2520DNA-Encoded%2520Libraries%2520to%2520Develop%2520a%2520Selective%252C%2520Cell-Permeable%2520CBX8%2520Chromodomain%2520Inhibitor%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2020%26volume%3D15%26issue%3D1%26spage%3D112%26epage%3D131%26doi%3D10.1021%2Facschembio.9b00654" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Højfeldt, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hedehus, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laugesen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tatar, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiehle, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helin, K.</span></span> <span> </span><span class="NLM_article-title">Non-Core Subunits of the PRC2 Complex Are Collectively Required for Its Target-Site Specificity</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>76</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">423</span>– <span class="NLM_lpage">436.e3</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2019.07.031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1016%2Fj.molcel.2019.07.031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=31521506" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A280%3ADC%252BB3MrovVWqsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2019&pages=423-436.e3&issue=3&author=J.+W.+H%C3%B8jfeldtauthor=L.+Hedehusauthor=A.+Laugesenauthor=T.+Tatarauthor=L.+Wiehleauthor=K.+Helin&title=Non-Core+Subunits+of+the+PRC2+Complex+Are+Collectively+Required+for+Its+Target-Site+Specificity&doi=10.1016%2Fj.molcel.2019.07.031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Non-core Subunits of the PRC2 Complex Are Collectively Required for Its Target-Site Specificity</span></div><div class="casAuthors">Hojfeldt Jonas Westergaard; Hedehus Lin; Laugesen Anne; Tatar Tulin; Wiehle Laura; Helin Kristian</div><div class="citationInfo"><span class="NLM_cas:title">Molecular cell</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">423-436.e3</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The Polycomb repressive complex 2 (PRC2) catalyzes H3K27 methylation across the genome, which impacts transcriptional regulation and is critical for establishment of cell identity.  Because of its essential function during development and in cancer, understanding the delineation of genome-wide H3K27 methylation patterns has been the focus of intense investigation.  PRC2 methylation activity is abundant and dispersed throughout the genome, but the highest activity is specifically directed to a subset of target sites that are stably occupied by the complex and highly enriched for H3K27me3.  Here, we show, by systematically knocking out single and multiple non-core subunits of the PRC2 complex in mouse embryonic stem cells, that they each contribute to directing PRC2 activity to target sites.  Furthermore, combined knockout of six non-core subunits reveals that, while dispensable for global H3K27 methylation levels, the non-core PRC2 subunits are collectively required for focusing H3K27me3 activity to specific sites in the genome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQdVVBpmAH1NgWRJ3OJdH_qfW6udTcc2eaQy7Andn2M07ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MrovVWqsQ%253D%253D&md5=eedb372cd8f66a19919b4ea22d9000c3</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2019.07.031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2019.07.031%26sid%3Dliteratum%253Aachs%26aulast%3DH%25C3%25B8jfeldt%26aufirst%3DJ.%2BW.%26aulast%3DHedehus%26aufirst%3DL.%26aulast%3DLaugesen%26aufirst%3DA.%26aulast%3DTatar%26aufirst%3DT.%26aulast%3DWiehle%26aufirst%3DL.%26aulast%3DHelin%26aufirst%3DK.%26atitle%3DNon-Core%2520Subunits%2520of%2520the%2520PRC2%2520Complex%2520Are%2520Collectively%2520Required%2520for%2520Its%2520Target-Site%2520Specificity%26jtitle%3DMol.%2520Cell%26date%3D2019%26volume%3D76%26issue%3D3%26spage%3D423%26epage%3D436.e3%26doi%3D10.1016%2Fj.molcel.2019.07.031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brunetti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorunova, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davidson, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panagopoulos, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Micci, F.</span></span> <span> </span><span class="NLM_article-title">Identification of an EPC2-PHF1 Fusion Transcript in Low-Grade Endometrial Stromal Sarcoma</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">27</span>),  <span class="NLM_fpage">19203</span>– <span class="NLM_lpage">19208</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.24969</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.18632%2Foncotarget.24969" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=29721194" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A280%3ADC%252BC1Mjpt1Ojsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=19203-19208&issue=27&author=M.+Brunettiauthor=L.+Gorunovaauthor=B.+Davidsonauthor=S.+Heimauthor=I.+Panagopoulosauthor=F.+Micci&title=Identification+of+an+EPC2-PHF1+Fusion+Transcript+in+Low-Grade+Endometrial+Stromal+Sarcoma&doi=10.18632%2Foncotarget.24969"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of an EPC2-PHF1 fusion transcript in low-grade endometrial stromal sarcoma</span></div><div class="casAuthors">Brunetti Marta; Gorunova Ludmila; Heim Sverre; Panagopoulos Ioannis; Micci Francesca; Davidson Ben; Davidson Ben; Heim Sverre</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">19203-19208</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Recurrent chromosomal translocations leading to gene fusion formation have been described in uterine sarcomas, including low-grade endometrial stromal sarcoma (LG-ESS).  Involvement of the PHF1 gene in chromosomal rearrangements targeting band 6p21 has been found in LG-ESS with different partners from JAZF1 mapping in 7p15, to EPC1 from 10p11, MEAF6 from 1p34, and BRD8 from 5q31.  In the present study, RNA sequencing of a LG-ESS showed a novel recombination of PHF1 with the Enhancer of Polycomb homolog 2 (EPC2).  RT-PCR followed by Sanger sequencing and FISH analysis confirmed the EPC2-PHF1 fusion transcript.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSCiJg2QydNO691pGlMMoT5fW6udTcc2eaQy7Andn2M07ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1Mjpt1Ojsw%253D%253D&md5=5e7b3dd5d69a418da3f130fe2fbae206</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.24969&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.24969%26sid%3Dliteratum%253Aachs%26aulast%3DBrunetti%26aufirst%3DM.%26aulast%3DGorunova%26aufirst%3DL.%26aulast%3DDavidson%26aufirst%3DB.%26aulast%3DHeim%26aufirst%3DS.%26aulast%3DPanagopoulos%26aufirst%3DI.%26aulast%3DMicci%26aufirst%3DF.%26atitle%3DIdentification%2520of%2520an%2520EPC2-PHF1%2520Fusion%2520Transcript%2520in%2520Low-Grade%2520Endometrial%2520Stromal%2520Sarcoma%26jtitle%3DOncotarget%26date%3D2018%26volume%3D9%26issue%3D27%26spage%3D19203%26epage%3D19208%26doi%3D10.18632%2Foncotarget.24969" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hoang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C.-H.</span></span> <span> </span><span class="NLM_article-title">Endometrial Stromal Sarcomas and Related Neoplasms: New Developments and Diagnostic Considerations</span>. <i>Pathology</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>50</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">162</span>– <span class="NLM_lpage">177</span>, <span class="refDoi"> DOI: 10.1016/j.pathol.2017.11.086</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1016%2Fj.pathol.2017.11.086" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=29275929" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitVaktL%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2018&pages=162-177&issue=2&author=L.+Hoangauthor=S.+Chiangauthor=C.-H.+Lee&title=Endometrial+Stromal+Sarcomas+and+Related+Neoplasms%3A+New+Developments+and+Diagnostic+Considerations&doi=10.1016%2Fj.pathol.2017.11.086"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Endometrial stromal sarcomas and related neoplasms: new developments and diagnostic considerations</span></div><div class="casAuthors">Hoang, Lien; Chiang, Sarah; Lee, Cheng-Han</div><div class="citationInfo"><span class="NLM_cas:title">Pathology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">162-177</span>CODEN:
                <span class="NLM_cas:coden">PTLGAX</span>;
        ISSN:<span class="NLM_cas:issn">0031-3025</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Our understanding of endometrial stromal sarcomas has evolved dramatically since their earliest descriptions from over a century ago.  Initial studies focused on establishing the relationship between histol. appearances of endometrial stromal sarcomas and their clin. outcomes.  Studies performed in the last decade have uncovered several recurrent cytogenetic aberrations occurring in low- and high-grade endometrial stromal sarcomas.  Low-grade endometrial stromal sarcomas bear close histopathol. resemblance to proliferative-type endometrial stroma, and approx. half harbor t(7;17)(p15;q21) resulting in JAZF1-SUZ12 gene fusion.  Less common JAZF1-PHF1, EPC1-PHF1, MEAF6-PHF1, and MBTD1-CXorf67 fusions have also been reported.  The term 'high-grade endometrial stromal sarcoma' was recently re-introduced in the classification of endometrial stromal tumors after the discovery of t(10;17)(q22;p13) resulting in YWHAE-NUTM2A/B fusion and is assocd. with distinct morphol. characteristics.  This review highlights the evolution of endometrial stromal sarcoma classification schemes over time and describes the salient clinicopathol. and mol. features of endometrial stromal nodule, low-grade endometrial stromal sarcoma, high-grade endometrial stromal sarcoma, and undifferentiated uterine sarcoma.  It also describes the recent characterization of endometrial stromal sarcoma with t(X;22)(p11;q13) resulting in ZC3H7B-BCOR fusion, a noteworthy entity due to its close histol. resemblance to myxoid leiomyosarcoma.  We also provide insights into common challenging scenarios encountered when assessing endometrial stromal lesions in daily surgical pathol. practice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrz3Puf7lZGP7Vg90H21EOLACvtfcHk0lhf93MdDciA3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitVaktL%252FM&md5=5b53b2d33b381b336a78f1e22fe752b3</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.pathol.2017.11.086&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pathol.2017.11.086%26sid%3Dliteratum%253Aachs%26aulast%3DHoang%26aufirst%3DL.%26aulast%3DChiang%26aufirst%3DS.%26aulast%3DLee%26aufirst%3DC.-H.%26atitle%3DEndometrial%2520Stromal%2520Sarcomas%2520and%2520Related%2520Neoplasms%253A%2520New%2520Developments%2520and%2520Diagnostic%2520Considerations%26jtitle%3DPathology%26date%3D2018%26volume%3D50%26issue%3D2%26spage%3D162%26epage%3D177%26doi%3D10.1016%2Fj.pathol.2017.11.086" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bae, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viviano, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lv, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sagum, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castellano, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khalil, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sbardella, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span> <span> </span><span class="NLM_article-title">Developing Spindlin1 Small-Molecule Inhibitors by Using Protein Microarrays</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">750</span>– <span class="NLM_lpage">756</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2377</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1038%2Fnchembio.2377" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=28504676" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnslKktrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=750-756&issue=7&author=N.+Baeauthor=M.+Vivianoauthor=X.+Suauthor=J.+Lvauthor=D.+Chengauthor=C.+Sagumauthor=S.+Castellanoauthor=X.+Baiauthor=C.+Johnsonauthor=M.+I.+Khalilauthor=J.+Shenauthor=K.+Chenauthor=H.+Liauthor=G.+Sbardellaauthor=M.+T.+Bedford&title=Developing+Spindlin1+Small-Molecule+Inhibitors+by+Using+Protein+Microarrays&doi=10.1038%2Fnchembio.2377"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Developing Spindlin1 small-molecule inhibitors by using protein microarrays</span></div><div class="casAuthors">Bae, Narkhyun; Viviano, Monica; Su, Xiaonan; Lv, Jie; Cheng, Donghang; Sagum, Cari; Castellano, Sabrina; Bai, Xue; Johnson, Claire; Khalil, Mahmoud Ibrahim; Shen, Jianjun; Chen, Kaifu; Li, Haitao; Sbardella, Gianluca; Bedford, Mark T.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">750-756</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The discovery of inhibitors of methyl- and acetyl-binding domains has provided evidence for the 'druggability' of epigenetic effector mols.  The small-mol. probe, UNC1215, prevents methyl-dependent protein-protein interactions by engaging the arom. cage of MBT domains and, with lower affinity, Tudor domains.  Here, using a library of tagged UNC1215 analogs, we screened a protein-domain microarray of human methyllysine effector mols. to rapidly detect compds. with new binding profiles with either increased or decreased specificity.  Using this approach, we identified a compd. (EML405) that acquired a novel interaction with Tudor-domain-contg. protein Spindlin1 (SPIN1).  Structural studies facilitated the rational synthesis of SPIN1 inhibitors with increased selectivity (EML631-633), which engaged SPIN1 in cells, blocked its ability to 'read' H3K4me3 marks, and inhibited its transcriptional-coactivator activity.  Protein microarrays can thus be used as a platform to 'target-hop' and identify small mols. that bind and compete with domain-motif interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6diBsmZCcm7Vg90H21EOLACvtfcHk0ljxfcX_aCfLjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnslKktrc%253D&md5=5ac8aab47d4a67cbe0f7c1f04d4ae410</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2377&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2377%26sid%3Dliteratum%253Aachs%26aulast%3DBae%26aufirst%3DN.%26aulast%3DViviano%26aufirst%3DM.%26aulast%3DSu%26aufirst%3DX.%26aulast%3DLv%26aufirst%3DJ.%26aulast%3DCheng%26aufirst%3DD.%26aulast%3DSagum%26aufirst%3DC.%26aulast%3DCastellano%26aufirst%3DS.%26aulast%3DBai%26aufirst%3DX.%26aulast%3DJohnson%26aufirst%3DC.%26aulast%3DKhalil%26aufirst%3DM.%2BI.%26aulast%3DShen%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DSbardella%26aufirst%3DG.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DDeveloping%2520Spindlin1%2520Small-Molecule%2520Inhibitors%2520by%2520Using%2520Protein%2520Microarrays%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26issue%3D7%26spage%3D750%26epage%3D756%26doi%3D10.1038%2Fnchembio.2377" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fagan, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johansson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gileadi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monteiro, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunford, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nibhani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malzahn, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faram, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cribbs, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halidi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chau, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greschik, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velupillai, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allali-Hassani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christott, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giroud, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Athanasou, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bountra, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schüle, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farnie, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oppermann, U.</span></span> <span> </span><span class="NLM_article-title">A Chemical Probe for Tudor Domain Protein Spindlin1 to Investigate Chromatin Function</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span> (<span class="NLM_issue">20</span>),  <span class="NLM_fpage">9008</span>– <span class="NLM_lpage">9025</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00562</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00562" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvVCntrvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=9008-9025&issue=20&author=V.+Faganauthor=C.+Johanssonauthor=C.+Gileadiauthor=O.+Monteiroauthor=J.+E.+Dunfordauthor=R.+Nibhaniauthor=M.+Philpottauthor=J.+Malzahnauthor=G.+Wellsauthor=R.+Faramauthor=A.+P.+Cribbsauthor=N.+Halidiauthor=F.+Liauthor=I.+Chauauthor=H.+Greschikauthor=S.+Velupillaiauthor=A.+Allali-Hassaniauthor=J.+Bennettauthor=T.+Christottauthor=C.+Giroudauthor=A.+M.+Lewisauthor=K.+V.+M.+Huberauthor=N.+Athanasouauthor=C.+Bountraauthor=M.+Jungauthor=R.+Sch%C3%BCleauthor=M.+Vedadiauthor=C.+Arrowsmithauthor=Y.+Xiongauthor=J.+Jinauthor=O.+Fedorovauthor=G.+Farnieauthor=P.+E.+Brennanauthor=U.+Oppermann&title=A+Chemical+Probe+for+Tudor+Domain+Protein+Spindlin1+to+Investigate+Chromatin+Function&doi=10.1021%2Facs.jmedchem.9b00562"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">A Chemical Probe for Tudor Domain Protein Spindlin1 to Investigate Chromatin Function</span></div><div class="casAuthors">Fagan, Vincent; Johansson, Catrine; Gileadi, Carina; Monteiro, Octovia; Dunford, James E.; Nibhani, Reshma; Philpott, Martin; Malzahn, Jessica; Wells, Graham; Faram, Ruth; Cribbs, Adam P.; Halidi, Nadia; Li, Fengling; Chau, Irene; Greschik, Holger; Velupillai, Srikannathasan; Allali-Hassani, Abdellah; Bennett, James; Christott, Thomas; Giroud, Charline; Lewis, Andrew M.; Huber, Kilian V. M.; Athanasou, Nick; Bountra, Chas; Jung, Manfred; Schule, Roland; Vedadi, Masoud; Arrowsmith, Cheryl; Xiong, Yan; Jin, Jian; Fedorov, Oleg; Farnie, Gillian; Brennan, Paul E.; Oppermann, Udo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">9008-9025</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Modifications of histone tails, including lysine/arginine methylation, provide the basis of a 'chromatin or histone code'.  Proteins that contain 'reader' domains can bind to these modifications and form specific effector complexes, which ultimately mediate chromatin function.  The spindlin1 (SPIN1) protein contains three Tudor methyl-lysine/arginine reader domains and was identified as a putative oncogene and transcriptional co-activator.  Here we report a SPIN1 chem. probe inhibitor with low nanomolar in vitro activity, exquisite selectivity on a panel of Me reader and writer proteins, and with submicromolar cellular activity.  X-ray crystallog. showed that this Tudor domain chem. probe simultaneously engages Tudor domains 1 and 2 via a bidentate binding mode.  Small mol. inhibition and siRNA knockdown of SPIN1, as well as chemoproteomic studies, identified genes which are transcriptionally regulated by SPIN1 in squamous cell carcinoma and suggest that SPIN1 may have a roll in cancer related inflammation and/or cancer metastasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzjLXYszX5ZrVg90H21EOLACvtfcHk0ljxfcX_aCfLjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvVCntrvK&md5=cd05be7d6d6ec3cc200a4fdb39f2a7f2</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00562%26sid%3Dliteratum%253Aachs%26aulast%3DFagan%26aufirst%3DV.%26aulast%3DJohansson%26aufirst%3DC.%26aulast%3DGileadi%26aufirst%3DC.%26aulast%3DMonteiro%26aufirst%3DO.%26aulast%3DDunford%26aufirst%3DJ.%2BE.%26aulast%3DNibhani%26aufirst%3DR.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DMalzahn%26aufirst%3DJ.%26aulast%3DWells%26aufirst%3DG.%26aulast%3DFaram%26aufirst%3DR.%26aulast%3DCribbs%26aufirst%3DA.%2BP.%26aulast%3DHalidi%26aufirst%3DN.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DGreschik%26aufirst%3DH.%26aulast%3DVelupillai%26aufirst%3DS.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DBennett%26aufirst%3DJ.%26aulast%3DChristott%26aufirst%3DT.%26aulast%3DGiroud%26aufirst%3DC.%26aulast%3DLewis%26aufirst%3DA.%2BM.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DAthanasou%26aufirst%3DN.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DJung%26aufirst%3DM.%26aulast%3DSch%25C3%25BCle%26aufirst%3DR.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DFarnie%26aufirst%3DG.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DOppermann%26aufirst%3DU.%26atitle%3DA%2520Chemical%2520Probe%2520for%2520Tudor%2520Domain%2520Protein%2520Spindlin1%2520to%2520Investigate%2520Chromatin%2520Function%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26issue%3D20%26spage%3D9008%26epage%3D9025%26doi%3D10.1021%2Facs.jmedchem.9b00562" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leng, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span> <span> </span><span class="NLM_article-title">PHD Finger Protein 1 (PHF1) Is a Novel Reader for Histone H4R3 Symmetric Dimethylation and Coordinates with PRMT5-WDR77/CRL4B Complex to Promote Tumorigenesis</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>46</i></span> (<span class="NLM_issue">13</span>),  <span class="NLM_fpage">6608</span>– <span class="NLM_lpage">6626</span>, <span class="refDoi"> DOI: 10.1093/nar/gky461</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1093%2Fnar%2Fgky461" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=29846670" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC1MXosVykt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=6608-6626&issue=13&author=R.+Liuauthor=J.+Gaoauthor=Y.+Yangauthor=R.+Qiuauthor=Y.+Zhengauthor=W.+Huangauthor=Y.+Zengauthor=Y.+Houauthor=S.+Wangauthor=S.+Lengauthor=D.+Fengauthor=W.+Yuauthor=G.+Sunauthor=H.+Shiauthor=X.+Tengauthor=Y.+Wang&title=PHD+Finger+Protein+1+%28PHF1%29+Is+a+Novel+Reader+for+Histone+H4R3+Symmetric+Dimethylation+and+Coordinates+with+PRMT5-WDR77%2FCRL4B+Complex+to+Promote+Tumorigenesis&doi=10.1093%2Fnar%2Fgky461"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">PHD finger protein 1 (PHF1) is a novel reader for histone H4R3 symmetric dimethylation and coordinates with PRMT5-WDR77/CRL4B complex to promote tumorigenesis</span></div><div class="casAuthors">Liu, Ruiqiong; Gao, Jie; Yang, Yang; Qiu, Rongfang; Zheng, Yu; Huang, Wei; Zeng, Yi; Hou, Yongqiang; Wang, Shuang; Leng, Shuai; Feng, Dandan; Yu, Wenqian; Sun, Gancheng; Shi, Hang; Teng, Xu; Wang, Yan</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">6608-6626</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Histone post-translational modifications regulate chromatin structure and function largely through interactions with effector proteins that often contain multiple histone-binding domains.  PHF1 [plant homeodomain (PHD) finger protein 1], which contains two kinds of histone reader modules, a Tudor domain and two PHD fingers, is an essential factor for epigenetic regulation and genome maintenance.  While significant progress has been made in characterizing the function of the Tudor domain, the roles of the two PHD fingers are poorly defined.  Here, we demonstrated that the N-terminal PHD finger of PHF1 recognizes sym. dimethylation of H4R3 (H4R3me2s) catalyzed by PRMT5-WDR77.  However, the C-terminal PHD finger of PHF1, instead of binding to modified histones, directly interacts with DDB1, the main component of the CUL4B-Ring E3 ligase complex (CRL4B), which is responsible for H2AK119 mono-ubiquitination (H2AK119ub1).  We showed that PHF1, PRMT5-WDR77, and CRL4B reciprocally interact with one another and collaborate as a functional unit.  Genome-wide anal. of PHF1/PRMT5/CUL4B targets identified a cohort of genes including Ecadherin and FBXW7, which are critically involved in cell growth and migration.  We demonstrated that PHF1 promotes cell proliferation, invasion, and tumorigenesis in vivo and in vitro and found that its expression is markedly upregulated in a variety of human cancers.  Our data identified a new reader for H4R3me2s and provided a mol. basis for the functional interplay between histone arginine methylation and ubiquitination.  The results also indicated that PHF1 is a key factor in cancer progression, supporting the pursuit of PHF1 as a target for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhfGrXwspki7Vg90H21EOLACvtfcHk0lhitFECvCcAtQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXosVykt7c%253D&md5=d7ed750a8532e93dafef1e78cc864a7e</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgky461&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgky461%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DR.%26aulast%3DGao%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DQiu%26aufirst%3DR.%26aulast%3DZheng%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DW.%26aulast%3DZeng%26aufirst%3DY.%26aulast%3DHou%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DLeng%26aufirst%3DS.%26aulast%3DFeng%26aufirst%3DD.%26aulast%3DYu%26aufirst%3DW.%26aulast%3DSun%26aufirst%3DG.%26aulast%3DShi%26aufirst%3DH.%26aulast%3DTeng%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DY.%26atitle%3DPHD%2520Finger%2520Protein%25201%2520%2528PHF1%2529%2520Is%2520a%2520Novel%2520Reader%2520for%2520Histone%2520H4R3%2520Symmetric%2520Dimethylation%2520and%2520Coordinates%2520with%2520PRMT5-WDR77%252FCRL4B%2520Complex%2520to%2520Promote%2520Tumorigenesis%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2018%26volume%3D46%26issue%3D13%26spage%3D6608%26epage%3D6626%26doi%3D10.1093%2Fnar%2Fgky461" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gatchalian, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ikebe, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tencer, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burge, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibson, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Musselman, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poirier, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kono, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutateladze, T. G.</span></span> <span> </span><span class="NLM_article-title">Accessibility of the Histone H3 Tail in the Nucleosome for Binding of Paired Readers</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">10</span>, <span class="refDoi"> DOI: 10.1038/s41467-017-01598-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1038%2Fs41467-017-01598-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=28232747" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtV2ksbvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=1-10&issue=1&author=J.+Gatchalianauthor=X.+Wangauthor=J.+Ikebeauthor=K.+L.+Coxauthor=A.+H.+Tencerauthor=Y.+Zhangauthor=N.+L.+Burgeauthor=L.+Diauthor=M.+D.+Gibsonauthor=C.+A.+Musselmanauthor=M.+G.+Poirierauthor=H.+Konoauthor=J.+J.+Hayesauthor=T.+G.+Kutateladze&title=Accessibility+of+the+Histone+H3+Tail+in+the+Nucleosome+for+Binding+of+Paired+Readers&doi=10.1038%2Fs41467-017-01598-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Accessibility of the histone H3 tail in the nucleosome for binding of paired readers</span></div><div class="casAuthors">Gatchalian, Jovylyn; Wang, Xiaodong; Ikebe, Jinzen; Cox, Khan L.; Tencer, Adam H.; Zhang, Yi; Burge, Nathaniel L.; Di, Luo; Gibson, Matthew D.; Musselman, Catherine A.; Poirier, Michael G.; Kono, Hidetoshi; Hayes, Jeffrey J.; Kutateladze, Tatiana G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-10</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Combinatorial polyvalent contacts of histone-binding domains or readers commonly mediate localization and activities of chromatin-assocd. proteins.  A pair of readers, the PHD fingers of the protein CHD4, has been shown to bivalently recognize histone H3 tails.  Here we describe a mechanism by which these linked but independent readers bind to the intact nucleosome core particle (NCP).  Comprehensive NMR, chem. reactivity, mol. dynamics, and fluorescence analyses point to the crit. roles of intra-nucleosomal histone-DNA interactions that reduce the accessibility of H3 tails in NCP, the nucleosomal DNA, and the linker between readers in modulating nucleosome- and/or histone-binding activities of the readers.  We show that the second PHD finger of CHD4 initiates recruitment to the nucleosome, however both PHDs are required to alter the NCP dynamics.  Our findings reveal a distinctive regulatory mechanism for the assocn. of paired readers with the nucleosome that provides an intricate balance between cooperative and individual activities of the readers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqcRg-zJejfdLVg90H21EOLACvtfcHk0lhitFECvCcAtQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtV2ksbvJ&md5=2047a2e726b65c1ed10e009a568b1b42</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1038%2Fs41467-017-01598-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-017-01598-x%26sid%3Dliteratum%253Aachs%26aulast%3DGatchalian%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DIkebe%26aufirst%3DJ.%26aulast%3DCox%26aufirst%3DK.%2BL.%26aulast%3DTencer%26aufirst%3DA.%2BH.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DBurge%26aufirst%3DN.%2BL.%26aulast%3DDi%26aufirst%3DL.%26aulast%3DGibson%26aufirst%3DM.%2BD.%26aulast%3DMusselman%26aufirst%3DC.%2BA.%26aulast%3DPoirier%26aufirst%3DM.%2BG.%26aulast%3DKono%26aufirst%3DH.%26aulast%3DHayes%26aufirst%3DJ.%2BJ.%26aulast%3DKutateladze%26aufirst%3DT.%2BG.%26atitle%3DAccessibility%2520of%2520the%2520Histone%2520H3%2520Tail%2520in%2520the%2520Nucleosome%2520for%2520Binding%2520of%2520Paired%2520Readers%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26issue%3D1%26spage%3D1%26epage%3D10%26doi%3D10.1038%2Fs41467-017-01598-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Delaglio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grzesiek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vuister, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfeifer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bax, A.</span></span> <span> </span><span class="NLM_article-title">NMRPipe: A Multidimensional Spectral Processing System Based on UNIX Pipes</span>. <i>J. Biomol. NMR</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>6</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">277</span>– <span class="NLM_lpage">293</span>, <span class="refDoi"> DOI: 10.1007/BF00197809</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1007%2FBF00197809" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=8520220" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADyaK2MXhtVSmurfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1995&pages=277-293&issue=3&author=F.+Delaglioauthor=S.+Grzesiekauthor=G.+W.+Vuisterauthor=G.+Zhuauthor=J.+Pfeiferauthor=A.+Bax&title=NMRPipe%3A+A+Multidimensional+Spectral+Processing+System+Based+on+UNIX+Pipes&doi=10.1007%2FBF00197809"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">NMRPipe: a multidimensional spectral processing system based on UNIX pipes</span></div><div class="casAuthors">Delaglio, Frank; Grzesiek, Stephan; Vuister, Geerten W.; Zhu, Guang; Pfeifer, John; Bax, Ad</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular NMR</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">277-93</span>CODEN:
                <span class="NLM_cas:coden">JBNME9</span>;
        ISSN:<span class="NLM_cas:issn">0925-2738</span>.
    
            (<span class="NLM_cas:orgname">ESCOM</span>)
        </div><div class="casAbstract">The NMRPipe system is a UNIX software environment of processing, graphics, and anal. tools designed to meet current routine and research-oriented multidimensional processing requirements, and to anticipate and accommodate future demands and developments.  The system is based on UNIX pipes, which allow programs running simultaneously to exchange streams of data under user control.  In an NMRPipe processing scheme, a stream of spectral data flows through a pipeline of processing programs, each of which performs one component of the overall scheme, such as Fourier transformation or linear prediction.  Complete multidimensional processing schemes are constructed as simple UNIX shell scripts.  The processing modules themselves maintain and exploit accurate records of data sizes, detection modes, and calibration information in all dimensions, so that schemes can be constructed without the need to explicitly define or anticipate data sizes or storage details of real and imaginary channels during processing.  The asynchronous pipeline scheme provides other substantial advantages, including high flexibility, favorable processing speeds, choice of both all-in-memory and disk-bound processing, easy adaptation to different data formats, simpler software development and maintenance, and the ability to distribute processing tasks on multi-CPU computers and computer networks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSIUdXrdcEgbVg90H21EOLACvtfcHk0lhitFECvCcAtQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXhtVSmurfK&md5=a670fca5b164083e2178fafd2fb951ff</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1007%2FBF00197809&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF00197809%26sid%3Dliteratum%253Aachs%26aulast%3DDelaglio%26aufirst%3DF.%26aulast%3DGrzesiek%26aufirst%3DS.%26aulast%3DVuister%26aufirst%3DG.%2BW.%26aulast%3DZhu%26aufirst%3DG.%26aulast%3DPfeifer%26aufirst%3DJ.%26aulast%3DBax%26aufirst%3DA.%26atitle%3DNMRPipe%253A%2520A%2520Multidimensional%2520Spectral%2520Processing%2520System%2520Based%2520on%2520UNIX%2520Pipes%26jtitle%3DJ.%2520Biomol.%2520NMR%26date%3D1995%26volume%3D6%26issue%3D3%26spage%3D277%26epage%3D293%26doi%3D10.1007%2FBF00197809" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Kabsch%2C+W.+Acta+Crystallogr.%2C+Sect.+D%3A+Biol.+Crystallogr.+2010%2C+66+%282%29%2C+125%E2%80%93132%2C+10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0ljjnk80ZwSuQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26issue%3D2%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span> <span> </span><span class="NLM_article-title">Electronic Ligand Builder and Optimization Workbench (ELBOW): A Tool for Ligand Coordinate and Restraint Generation</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>65</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">1074</span>– <span class="NLM_lpage">1080</span>, <span class="refDoi"> DOI: 10.1107/S0907444909029436</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1107%2FS0907444909029436" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=19770504" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFKnsLnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2009&pages=1074-1080&issue=10&author=N.+W.+Moriartyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adams&title=Electronic+Ligand+Builder+and+Optimization+Workbench+%28ELBOW%29%3A+A+Tool+for+Ligand+Coordinate+and+Restraint+Generation&doi=10.1107%2FS0907444909029436"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">electronic Ligand Builder and Optimization Workbench (eLBOW): a tool for ligand coordinate and restraint generation</span></div><div class="casAuthors">Moriarty, Nigel W.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1074-1080</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The electronic Ligand Builder and Optimization Workbench (eLBOW) is a program module of the PHENIX suite of computational crystallog. software.  It is designed to be a flexible procedure that uses simple and fast quantum-chem. techniques to provide chem. accurate information for novel and known ligands alike.  A variety of input formats and options allow the attainment of a no. of diverse goals including geometry optimization and generation of restraints.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojnN4AVY_DwrVg90H21EOLACvtfcHk0ljjnk80ZwSuQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFKnsLnN&md5=3f9a9c6c31c9a4b014a9f151fb6aa8d9</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1107%2FS0907444909029436&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909029436%26sid%3Dliteratum%253Aachs%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26atitle%3DElectronic%2520Ligand%2520Builder%2520and%2520Optimization%2520Workbench%2520%2528ELBOW%2529%253A%2520A%2520Tool%2520for%2520Ligand%2520Coordinate%2520and%2520Restraint%2520Generation%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2009%26volume%3D65%26issue%3D10%26spage%3D1074%26epage%3D1080%26doi%3D10.1107%2FS0907444909029436" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and Development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&issue=4&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+Development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0ljjnk80ZwSuQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520Development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26issue%3D4%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkóczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title">PHENIX: A Comprehensive Python-Based System for Macromolecular Structure Solution</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&issue=2&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.+W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+A+Comprehensive+Python-Based+System+for+Macromolecular+Structure+Solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0ljjnk80ZwSuQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.%2BW.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520A%2520Comprehensive%2520Python-Based%2520System%2520for%2520Macromolecular%2520Structure%2520Solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26issue%3D2%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00430/20210616/images/medium/jm1c00430_0006.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00430/20210616/images/large/jm1c00430_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00430&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00430/20210616/images/medium/jm1c00430_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00430/20210616/images/large/jm1c00430_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Characterization of the interaction between the PHF1 Tudor domain and UNC6641 by NMR. (a) Overlay of <sup>1</sup>H,<sup>15</sup>N HSQC spectra of the <sup>15</sup>N-labeled PHF1 Tudor domain collected before and after the addition of UNC6641. Spectra are color coded according to the protein–peptide molar ratio. (b) Histogram showing chemical shift perturbations of backbone amides of the PHF1 Tudor domain at a 1:4 molar ratio of protein to UNC6641. The dashed line indicates an averaged chemical shift change plus one standard deviation. (c) The surface representation of the PHF1 Tudor domain (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HCZ">4HCZ</a>). Residues that exhibit substantial changes in part b are colored purple. (d) The PHF1 Tudor domain (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HCZ">4HCZ</a>) is depicted as a gray ribbon with the aromatic cage residues W41, Y47, F65, and F71 shown as pink sticks and labeled.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00430/20210616/images/large/jm1c00430_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00430&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00430/20210616/images/medium/jm1c00430_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00430/20210616/images/large/jm1c00430_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Molecular mechanism for the association of the PHF1 Tudor domain with UNC6641 (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7LKY">7LKY</a>). (a) The electrostatic surface potential of the PHF1 Tudor domain. Bound UNC6641 is colored green, and the phenyl substituent of K4 is indicated as Lig. (b) The 2Fo-Fc electron-density map of UNC6641 contoured at 1 σ shown as a gray mesh. (c) Structure of the PHF1 Tudor domain in complex with UNC6641. The PHF1 Tudor domain is shown as a gray ribbon, UNC6641 is shown as green sticks, and the protein residues interacting with UNC6641 are shown as gray sticks (except for W41, Y47, F65, and F71, which are colored yellow). The hydrogen bonds are indicated by yellow dashed lines. (d) Zoomed-in view of the UNC6641 binding site. (e) Overlay of the structures of the of PHF1 Tudor domain in complex with the H3K36me3 peptide (purple sticks) (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HCZ">4HCZ</a>) and UNC6641 (green sticks).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00430/20210616/images/large/jm1c00430_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00430&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00430/20210616/images/medium/jm1c00430_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00430/20210616/images/large/jm1c00430_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Selectivity of UNC6641 across closely related Tudor domains. (a) Sequence alignment of Tudor domains most closely related to PHF1. Aromatic cage residues are in orange. * = Fully conserved. (b) Binding affinities of UNC6641 for closely related Tudor domains. <i>K</i><sub>d</sub> values were determined by two independent ITC experiments (mean ± SD).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00430/20210616/images/large/jm1c00430_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00430&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i63">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_48461" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_48461" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 56 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Macdonald, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hathaway, N. A.</span></span> <span> </span><span class="NLM_article-title">Epigenetic Roots of Immunologic Disease and New Methods for Examining Chromatin Regulatory Pathways</span>. <i>Immunol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>93</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">261</span>– <span class="NLM_lpage">270</span>, <span class="refDoi"> DOI: 10.1038/icb.2014.105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1038%2Ficb.2014.105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=25533290" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFyhu7rF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=2015&pages=261-270&issue=3&author=I.+A.+Macdonaldauthor=N.+A.+Hathaway&title=Epigenetic+Roots+of+Immunologic+Disease+and+New+Methods+for+Examining+Chromatin+Regulatory+Pathways&doi=10.1038%2Ficb.2014.105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic roots of immunologic disease and new methods for examining chromatin regulatory pathways</span></div><div class="casAuthors">MacDonald, Ian A.; Hathaway, Nathaniel A.</div><div class="citationInfo"><span class="NLM_cas:title">Immunology & Cell Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">261-270</span>CODEN:
                <span class="NLM_cas:coden">ICBIEZ</span>;
        ISSN:<span class="NLM_cas:issn">0818-9641</span>.
    
            (<span class="NLM_cas:orgname">NPG Nature Asia-Pacific</span>)
        </div><div class="casAbstract">A review.  The ability to accurately quantitate and exptl. examine epigenetic modifications across the human genome has exploded in the past decade.  This has given rise to a wealth of new information concerning the contributions of epigenetic regulatory networks to the pathogenesis of human disease.  In particular, immunol. disorders have strong developmental roots in chromatin regulatory pathways.  In this review, we focus on the epigenetic signatures and new discoveries revealing the epigenetic compns. of specific immunol. cancers and autoimmune diseases.  We also comment on the conserved epigenetic roots among diverse immunol. disorders and suggest inhibition strategies that may be relevant for future treatment.  Finally, we highlight emerging exptl. tools with the capability to examine the mechanisms of chromatin regulatory enzymes with a high level of temporal control.  The knowledge of genetic and epigenetic defects in immunol. disease combined with new exptl. approaches will elucidate the contribution of individual enzymes in complex epigenetic regulatory networks.  This could lead to new diagnostic and therapeutic approaches for some very diverse and difficult to treat human diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9pqOiMk4yurVg90H21EOLACvtfcHk0lit3jnBqkd_8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFyhu7rF&md5=8b76e7fd5b3ec62d39d69cc3cb0b37bd</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Ficb.2014.105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Ficb.2014.105%26sid%3Dliteratum%253Aachs%26aulast%3DMacdonald%26aufirst%3DI.%2BA.%26aulast%3DHathaway%26aufirst%3DN.%2BA.%26atitle%3DEpigenetic%2520Roots%2520of%2520Immunologic%2520Disease%2520and%2520New%2520Methods%2520for%2520Examining%2520Chromatin%2520Regulatory%2520Pathways%26jtitle%3DImmunol.%2520Cell%2520Biol.%26date%3D2015%26volume%3D93%26issue%3D3%26spage%3D261%26epage%3D270%26doi%3D10.1038%2Ficb.2014.105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mirabella, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foster, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartke, T.</span></span> <span> </span><span class="NLM_article-title">Chromatin Deregulation in Disease</span>. <i>Chromosoma</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>125</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">75</span>– <span class="NLM_lpage">93</span>, <span class="refDoi"> DOI: 10.1007/s00412-015-0530-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1007%2Fs00412-015-0530-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=26188466" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1WqtrfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2016&pages=75-93&issue=1&author=A.+C.+Mirabellaauthor=B.+M.+Fosterauthor=T.+Bartke&title=Chromatin+Deregulation+in+Disease&doi=10.1007%2Fs00412-015-0530-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatin deregulation in disease</span></div><div class="casAuthors">Mirabella, Anne C.; Foster, Benjamin M.; Bartke, Till</div><div class="citationInfo"><span class="NLM_cas:title">Chromosoma</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">75-93</span>CODEN:
                <span class="NLM_cas:coden">CHROAU</span>;
        ISSN:<span class="NLM_cas:issn">0009-5915</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  The regulation of chromatin by epigenetic mechanisms plays a central role in gene expression and is essential for development and maintenance of cell identity and function.  Aberrant chromatin regulation is obsd. in many diseases where it leads to defects in epigenetic gene regulation resulting in pathol. gene expression programs.  These defects are caused by inherited or acquired mutations in genes encoding enzymes that deposit or remove DNA and histone modifications and that shape chromatin architecture.  Chromatin deregulation often results in neurodevelopmental disorders and intellectual disabilities, frequently linked to phys. and developmental abnormalities, but can also cause neurodegenerative diseases, immunodeficiency, or muscle wasting syndromes.  Epigenetic diseases can either be of monogenic origin or manifest themselves as complex multifactorial diseases such as in congenital heart disease, autism spectrum disorders, or cancer in which mutations in chromatin regulators are contributing factors.  The environment directly influences the epigenome and can induce changes that cause or predispose to diseases through risk factors such as stress, malnutrition or exposure to harmful chems.  The plasticity of chromatin regulation makes targeting the enzymic machinery an attractive strategy for therapeutic intervention and an increasing no. of small mol. inhibitors against a variety of epigenetic regulators are in clin. use or under development.  In this review, we will give an overview of the mol. lesions that underlie epigenetic diseases, and we will discuss the impact of the environment and prospects for epigenetic therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrW99ySZUDhtLVg90H21EOLACvtfcHk0liC2qSYe3OL5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1WqtrfK&md5=f19871296cb8383443499d2c7d78f527</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1007%2Fs00412-015-0530-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00412-015-0530-0%26sid%3Dliteratum%253Aachs%26aulast%3DMirabella%26aufirst%3DA.%2BC.%26aulast%3DFoster%26aufirst%3DB.%2BM.%26aulast%3DBartke%26aufirst%3DT.%26atitle%3DChromatin%2520Deregulation%2520in%2520Disease%26jtitle%3DChromosoma%26date%3D2016%26volume%3D125%26issue%3D1%26spage%3D75%26epage%3D93%26doi%3D10.1007%2Fs00412-015-0530-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Valencia, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadoch, C.</span></span> <span> </span><span class="NLM_article-title">Chromatin Regulatory Mechanisms and Therapeutic Opportunities in Cancer</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>21</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">152</span>– <span class="NLM_lpage">161</span>, <span class="refDoi"> DOI: 10.1038/s41556-018-0258-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1038%2Fs41556-018-0258-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=30602726" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC1MXlslKmsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2019&pages=152-161&issue=2&author=A.+M.+Valenciaauthor=C.+Kadoch&title=Chromatin+Regulatory+Mechanisms+and+Therapeutic+Opportunities+in+Cancer&doi=10.1038%2Fs41556-018-0258-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatin regulatory mechanisms and therapeutic opportunities in cancer</span></div><div class="casAuthors">Valencia, Alfredo M.; Kadoch, Cigall</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">152-161</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  Research over the past several decades has unmasked a major contribution of disrupted chromatin regulatory processes to human disease, particularly cancer.  Advances in genome-wide technologies have highlighted frequent mutations in genes encoding chromatin-assocd. proteins, identified unexpected synthetic lethal opportunities and enabled increasingly comprehensive structural and functional dissection.  Here, we review recent progress in our understanding of oncogenic mechanisms at each level of chromatin organization and regulation, and discuss new strategies towards therapeutic intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGog3wawAVdsr7Vg90H21EOLACvtfcHk0liC2qSYe3OL5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXlslKmsrY%253D&md5=fe9df9ef3fe0ffcf7e0e8ba377543251</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fs41556-018-0258-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41556-018-0258-1%26sid%3Dliteratum%253Aachs%26aulast%3DValencia%26aufirst%3DA.%2BM.%26aulast%3DKadoch%26aufirst%3DC.%26atitle%3DChromatin%2520Regulatory%2520Mechanisms%2520and%2520Therapeutic%2520Opportunities%2520in%2520Cancer%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2019%26volume%3D21%26issue%3D2%26spage%3D152%26epage%3D161%26doi%3D10.1038%2Fs41556-018-0258-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bird, A.</span></span> <span> </span><span class="NLM_article-title">The Methyl-CpG-Binding Protein MeCP2 and Neurological Disease</span>. <i>Biochem. Soc. Trans.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>36</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">575</span>– <span class="NLM_lpage">583</span>, <span class="refDoi"> DOI: 10.1042/BST0360575</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1042%2FBST0360575" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=18631120" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BD1cXoslCltrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2008&pages=575-583&issue=4&author=A.+Bird&title=The+Methyl-CpG-Binding+Protein+MeCP2+and+Neurological+Disease&doi=10.1042%2FBST0360575"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The methyl-CpG-binding protein MeCP2 and neurological disease</span></div><div class="casAuthors">Bird, Adrian</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Society Transactions</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">575-583</span>CODEN:
                <span class="NLM_cas:coden">BCSTB5</span>;
        ISSN:<span class="NLM_cas:issn">0300-5127</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">A review.  The methyl-CpG-binding protein MeCP2 was discovered over 15 years ago as part of a search for proteins that selectively bind methylated DNA.  It is a nuclear protein that is largely chromatin-bound and has a strong preference for binding to methylated DNA sequences in vivo.  Evidence from model systems shows that MeCP2 can recruit the Sin3a co-repressor complex to promoters leading to transcriptional repression, therefore suggesting that MeCP2 can interpret the DNA methylation signal to bring about gene silencing.  Mutations in the human MECP2 gene cause the autism spectrum disorder Rett Syndrome.  MeCP2 is most highly expressed in neurons, and mice lacking this protein show symptoms that strikingly parallel those of Rett patients.  Surprisingly, these symptoms are efficiently reversed by delayed activation of a stopped' Mecp2 gene, raising hopes that human Rett syndrome may also be reversible.  Future studies of MeCP2 promise to shed light upon brain function, neurol. disease and the biol. of DNA methylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPOd305g9ETrVg90H21EOLACvtfcHk0liC2qSYe3OL5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXoslCltrY%253D&md5=281fbc5aeb5d81318abad292ce8f9510</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1042%2FBST0360575&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBST0360575%26sid%3Dliteratum%253Aachs%26aulast%3DBird%26aufirst%3DA.%26atitle%3DThe%2520Methyl-CpG-Binding%2520Protein%2520MeCP2%2520and%2520Neurological%2520Disease%26jtitle%3DBiochem.%2520Soc.%2520Trans.%26date%3D2008%26volume%3D36%26issue%3D4%26spage%3D575%26epage%3D583%26doi%3D10.1042%2FBST0360575" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Strahl, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allis, C. D.</span></span> <span> </span><span class="NLM_article-title">The Language of Covalent Histone Modifications</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>403</i></span>,  <span class="NLM_fpage">41</span>– <span class="NLM_lpage">45</span>, <span class="refDoi"> DOI: 10.1038/47412</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1038%2F47412" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10638745" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A280%3ADC%252BD3c7gt1arsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=403&publication_year=2000&pages=41-45&author=B.+D.+Strahlauthor=C.+D.+Allis&title=The+Language+of+Covalent+Histone+Modifications&doi=10.1038%2F47412"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The language of covalent histone modifications</span></div><div class="casAuthors">Strahl B D; Allis C D</div><div class="citationInfo"><span class="NLM_cas:title">Nature</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">403</span>
        (<span class="NLM_cas:issue">6765</span>),
    <span class="NLM_cas:pages">41-5</span>
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">Histone proteins and the nucleosomes they form with DNA are the fundamental building blocks of eukaryotic chromatin.  A diverse array of post-translational modifications that often occur on tail domains of these proteins has been well documented.  Although the function of these highly conserved modifications has remained elusive, converging biochemical and genetic evidence suggests functions in several chromatin-based processes.  We propose that distinct histone modifications, on one or more tails, act sequentially or in combination to form a 'histone code' that is, read by other proteins to bring about distinct downstream events.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSyzhdlYh9sX7t7lRqEFUJVfW6udTcc2ebWiGUsnOlLjLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3c7gt1arsQ%253D%253D&md5=2eb7d093151c6122ccd70fa2adcfd94b</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2F47412&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F47412%26sid%3Dliteratum%253Aachs%26aulast%3DStrahl%26aufirst%3DB.%2BD.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DThe%2520Language%2520of%2520Covalent%2520Histone%2520Modifications%26jtitle%3DNature%26date%3D2000%26volume%3D403%26spage%3D41%26epage%3D45%26doi%3D10.1038%2F47412" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jenuwein, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allis, C. D.</span></span> <span> </span><span class="NLM_article-title">Translating the Histone Code</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>293</i></span>,  <span class="NLM_fpage">1074</span>– <span class="NLM_lpage">1080</span>, <span class="refDoi"> DOI: 10.1126/science.1063127</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1126%2Fscience.1063127" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=11498575" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BD3MXmtVWltro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=293&publication_year=2001&pages=1074-1080&author=T.+Jenuweinauthor=C.+D.+Allis&title=Translating+the+Histone+Code&doi=10.1126%2Fscience.1063127"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Translating the histone code</span></div><div class="casAuthors">Jenuwein, Thomas; Allis, C. David</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">293</span>
        (<span class="NLM_cas:issue">5532</span>),
    <span class="NLM_cas:pages">1074-1080</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The review.  Chromatin, the physiol. template of all eukaryotic genetic information, is subject to a diverse array of posttranslational modifications that largely impinge on histone amino termini, thereby regulating access to the underlying DNA.  Distinct histone amino-terminal modifications can generate synergistic or antagonistic interaction affinities for chromatin-assocd. proteins, which in turn dictate dynamic transitions between transcriptionally active or transcriptionally silent chromatin states.  The combinatorial nature of histone amino-terminal modifications thus reveals a "histone code" that considerably extends the information potential of the genetic code.  We propose that this epigenetic marking system represents a fundamental regulatory mechanism that has an impact on most, if not all; chromatin-templated processes, with far-reaching consequences for cell fate decisions and both normal and pathol. development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUFyHNepKcUbVg90H21EOLACvtfcHk0lithdGOsWQGPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXmtVWltro%253D&md5=021dcd98a5e5bf2ed7eedef1897b2269</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1126%2Fscience.1063127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1063127%26sid%3Dliteratum%253Aachs%26aulast%3DJenuwein%26aufirst%3DT.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DTranslating%2520the%2520Histone%2520Code%26jtitle%3DScience%26date%3D2001%26volume%3D293%26spage%3D1074%26epage%3D1080%26doi%3D10.1126%2Fscience.1063127" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Musselman, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khorasanizadeh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutateladze, T. G.</span></span> <span> </span><span class="NLM_article-title">Towards Understanding Methyllysine Readout</span>. <i>Biochim. Biophys. Acta, Gene Regul. Mech.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>1839</i></span>,  <span class="NLM_fpage">686</span>– <span class="NLM_lpage">693</span>, <span class="refDoi"> DOI: 10.1016/j.bbagrm.2014.04.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1016%2Fj.bbagrm.2014.04.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=24727128" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC2cXntFGjtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1839&publication_year=2014&pages=686-693&author=C.+A.+Musselmanauthor=S.+Khorasanizadehauthor=T.+G.+Kutateladze&title=Towards+Understanding+Methyllysine+Readout&doi=10.1016%2Fj.bbagrm.2014.04.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Towards understanding methyllysine readout</span></div><div class="casAuthors">Musselman, Catherine A.; Khorasanizadeh, Sepideh; Kutateladze, Tatiana G.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Gene Regulatory Mechanisms</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">1839</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">686-693</span>CODEN:
                <span class="NLM_cas:coden">BBAGC6</span>;
        ISSN:<span class="NLM_cas:issn">1874-9399</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Lysine methylation is the most versatile covalent posttranslational modification (PTM) found in histones and non-histone proteins.  Over the past decade a no. of methyllysine-specific readers have been discovered and their interactions with histone tails have been structurally and biochem. characterized.  More recently innovative exptl. approaches have emerged that allow for studying reader interactions in the context of the full nucleosome and nucleosomal arrays.  In this review, we give a brief overview of the known mechanisms of histone lysine methylation readout, summarize progress recently made in exploring interactions with methylated nucleosomes, and discuss the latest advances in the development of small mol. inhibitors of the methyllysine-specific readers.  New studies reveal various reader-nucleosome contacts outside the methylated histone tail, thus offering a better model for assocn. of histone readers to chromatin and broadening our understanding of the functional implications of these interactions.  In addn., some progress has been made in the design of antagonists of these interactions.Specific lysine methylation patterns are commonly assocd. with certain chromatin states and genomic elements, and are linked to distinct biol. outcomes such as transcription activation or repression.  Disruption of patterns of histone modifications is assocd. with a no. of diseases, and there is tremendous therapeutic potential in targeting histone modification pathways.  Thus, investigating binding of readers of these modifications is not only important for elucidating fundamental mechanisms of chromatin regulation, but also necessary for the design of targeted therapeutics.  This article is part of a Special Issue entitled: Mol. mechanisms of histone modification function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoC_YadxhKu5LVg90H21EOLACvtfcHk0lithdGOsWQGPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXntFGjtro%253D&md5=10c99cb631ecf00d8c7d941ed4d40fbb</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagrm.2014.04.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagrm.2014.04.001%26sid%3Dliteratum%253Aachs%26aulast%3DMusselman%26aufirst%3DC.%2BA.%26aulast%3DKhorasanizadeh%26aufirst%3DS.%26aulast%3DKutateladze%26aufirst%3DT.%2BG.%26atitle%3DTowards%2520Understanding%2520Methyllysine%2520Readout%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Gene%2520Regul.%2520Mech.%26date%3D2014%26volume%3D1839%26spage%3D686%26epage%3D693%26doi%3D10.1016%2Fj.bbagrm.2014.04.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Beaver, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waters, M. L.</span></span> <span> </span><span class="NLM_article-title">Molecular Recognition of Lys and Arg Methylation</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_fpage">643</span>– <span class="NLM_lpage">653</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00996</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00996" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVGitbjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&pages=643-653&author=J.+E.+Beaverauthor=M.+L.+Waters&title=Molecular+Recognition+of+Lys+and+Arg+Methylation&doi=10.1021%2Facschembio.5b00996"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Recognition of Lys and Arg Methylation</span></div><div class="casAuthors">Beaver, Joshua E.; Waters, Marcey L.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">643-653</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A network of reader proteins and enzymes precisely controls gene transcription through the dynamic addn., removal, and recognition of post-translational modifications (PTMs) of histone tails.  Histone PTMs work in concert with this network to regulate gene transcription through the histone code, and the dysregulation of PTM maintenance is linked to a large no. of diseases, including many types of cancer.  A wealth of research aims to elucidate the functions of this code, but our understanding of the effects of PTMs, specifically the methylation of lysine (Lys) and arginine (Arg), is lacking.  The development of new tools to study PTMs relies on a sophisticated understanding of the mechanisms that drive protein and small mol. recognition in water.  In this review, we outline the phys. org. concepts that drive the mol. recognition of Lys and Arg methylation by reader proteins and draw comparisons to the binding mechanisms of small mol. receptors for methylated Lys and Arg that have been developed recently.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPpKlWThh4vLVg90H21EOLACvtfcHk0lithdGOsWQGPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVGitbjF&md5=f2bca0418453cad1a65626c2867617ba</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00996&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00996%26sid%3Dliteratum%253Aachs%26aulast%3DBeaver%26aufirst%3DJ.%2BE.%26aulast%3DWaters%26aufirst%3DM.%2BL.%26atitle%3DMolecular%2520Recognition%2520of%2520Lys%2520and%2520Arg%2520Methylation%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26spage%3D643%26epage%3D653%26doi%3D10.1021%2Facschembio.5b00996" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Taverna, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruthenburg, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allis, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, D. J.</span></span> <span> </span><span class="NLM_article-title">How Chromatin-Binding Modules Interpret Histone Modifications: Lessons from Professional Pocket Pickers</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">1025</span>– <span class="NLM_lpage">1040</span>, <span class="refDoi"> DOI: 10.1038/nsmb1338</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1038%2Fnsmb1338" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=17984965" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1Kjs7vE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2007&pages=1025-1040&author=S.+D.+Tavernaauthor=H.+Liauthor=A.+J.+Ruthenburgauthor=C.+D.+Allisauthor=D.+J.+Patel&title=How+Chromatin-Binding+Modules+Interpret+Histone+Modifications%3A+Lessons+from+Professional+Pocket+Pickers&doi=10.1038%2Fnsmb1338"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">How chromatin-binding modules interpret histone modifications: lessons from professional pocket pickers</span></div><div class="casAuthors">Taverna, Sean D.; Li, Haitao; Ruthenburg, Alexander J.; Allis, C. David; Patel, Dinshaw J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1025-1040</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Histones comprise the major protein component of chromatin, the scaffold in which the eukaryotic genome is packaged, and are subject to many types of post-translational modifications (PTMs), esp. on their flexible tails.  These modifications may constitute a 'histone code' and could be used to manage epigenetic information that helps extend the genetic message beyond DNA sequences.  This proposed code, read in part by histone PTM-binding 'effector' modules and their assocd. complexes, has been predicted to define unique functional states of chromatin and/or regulate various chromatin-templated processes.  A wealth of structural and functional data show how chromatin effector modules target their cognate covalent histone modifications.  Here, the authors summarize key features in the mol. recognition of histone PTMs by a diverse family of 'reader pockets' (including bromodomains, Royal superfamily modules, PHD fingers, WD40 protein repeats, 14-3-3 proteins, and BRCT domains), highlighting specific readout mechanisms for individual marks, common themes, and insights into the downstream functional consequences of the interactions.  Changes in these interactions may have far-reaching implications for human biol. and disease, notably cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGroaoP5OiDUOLVg90H21EOLACvtfcHk0lilAYIeizXlMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1Kjs7vE&md5=4ff842f1ef5416b932a89c751707defd</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnsmb1338&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb1338%26sid%3Dliteratum%253Aachs%26aulast%3DTaverna%26aufirst%3DS.%2BD.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DRuthenburg%26aufirst%3DA.%2BJ.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26aulast%3DPatel%26aufirst%3DD.%2BJ.%26atitle%3DHow%2520Chromatin-Binding%2520Modules%2520Interpret%2520Histone%2520Modifications%253A%2520Lessons%2520from%2520Professional%2520Pocket%2520Pickers%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2007%26volume%3D14%26spage%3D1025%26epage%3D1040%26doi%3D10.1038%2Fnsmb1338" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Musselman, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lalonde, M.-E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Côté, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutateladze, T. G.</span></span> <span> </span><span class="NLM_article-title">Perceiving the Epigenetic Landscape through Histone Readers</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>19</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">1218</span>– <span class="NLM_lpage">1227</span>, <span class="refDoi"> DOI: 10.1038/nsmb.2436</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1038%2Fnsmb.2436" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=23211769" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslOktbbI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=1218-1227&issue=12&author=C.+A.+Musselmanauthor=M.-E.+Lalondeauthor=J.+C%C3%B4t%C3%A9author=T.+G.+Kutateladze&title=Perceiving+the+Epigenetic+Landscape+through+Histone+Readers&doi=10.1038%2Fnsmb.2436"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Perceiving the epigenetic landscape through histone readers</span></div><div class="casAuthors">Musselman, Catherine A.; Lalonde, Marie-Eve; Cote, Jacques; Kutateladze, Tatiana G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1218-1227</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Post-translational modifications (PTMs) of histones provide a fine-tuned mechanism for regulating chromatin structure and dynamics.  PTMs can alter direct interactions between histones and DNA and serve as docking sites for protein effectors, or readers, of these PTMs.  Binding of the readers recruits or stabilizes various components of the nuclear signaling machinery at specific genomic sites, mediating fundamental DNA-templated processes, including gene transcription and DNA recombination, replication and repair.  In this review, we highlight the latest advances in characterizing histone-binding mechanisms and identifying new epigenetic readers and summarize the functional significance of PTM recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIQ_OP4dJ93bVg90H21EOLACvtfcHk0lilAYIeizXlMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslOktbbI&md5=3eb922fae63c59ddea200c04f743a9ee</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.2436&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.2436%26sid%3Dliteratum%253Aachs%26aulast%3DMusselman%26aufirst%3DC.%2BA.%26aulast%3DLalonde%26aufirst%3DM.-E.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DJ.%26aulast%3DKutateladze%26aufirst%3DT.%2BG.%26atitle%3DPerceiving%2520the%2520Epigenetic%2520Landscape%2520through%2520Histone%2520Readers%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2012%26volume%3D19%26issue%3D12%26spage%3D1218%26epage%3D1227%26doi%3D10.1038%2Fnsmb.2436" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strahl, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutateladze, T. G.</span></span> <span> </span><span class="NLM_article-title">Insights into Newly Discovered Marks and Readers of Epigenetic Information</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">662</span>– <span class="NLM_lpage">668</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2149</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1038%2Fnchembio.2149" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=27538025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlOnsbbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=662-668&author=F.+H.+Andrewsauthor=B.+D.+Strahlauthor=T.+G.+Kutateladze&title=Insights+into+Newly+Discovered+Marks+and+Readers+of+Epigenetic+Information&doi=10.1038%2Fnchembio.2149"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Insights into newly discovered marks and readers of epigenetic information</span></div><div class="casAuthors">Andrews, Forest H.; Strahl, Brian D.; Kutateladze, Tatiana G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">662-668</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The field of chromatin biol. has been advancing at an accelerated pace.  Recent discoveries of previously uncharacterized sites and types of post-translational modifications (PTMs) and the identification of new sets of proteins responsible for the deposition, removal, and reading of these marks continue raising the complexity of an already exceedingly complicated biol. phenomenon.  Here, the authors examine the biol. importance of new types and sites of histone PTMs and summarize the mol. mechanisms of chromatin engagement by newly discovered epigenetic readers.  The authors also highlight the imperative role of structural insights in understanding PTM-reader interactions and discuss future directions to enhance the knowledge of PTM readout.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwyBPIk5aEQLVg90H21EOLACvtfcHk0lilAYIeizXlMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlOnsbbJ&md5=166d01c61bb022b393e24367afac17b8</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2149&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2149%26sid%3Dliteratum%253Aachs%26aulast%3DAndrews%26aufirst%3DF.%2BH.%26aulast%3DStrahl%26aufirst%3DB.%2BD.%26aulast%3DKutateladze%26aufirst%3DT.%2BG.%26atitle%3DInsights%2520into%2520Newly%2520Discovered%2520Marks%2520and%2520Readers%2520of%2520Epigenetic%2520Information%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3D662%26epage%3D668%26doi%3D10.1038%2Fnchembio.2149" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baril, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koenig, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krone, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albanese, K. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, C. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, G. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houk, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waters, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brustad, E. M.</span></span> <span> </span><span class="NLM_article-title">Investigation of Trimethyllysine Binding by the HP1 Chromodomain via Unnatural Amino Acid Mutagenesis</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>139</i></span> (<span class="NLM_issue">48</span>),  <span class="NLM_fpage">17253</span>– <span class="NLM_lpage">17256</span>, <span class="refDoi"> DOI: 10.1021/jacs.7b09223</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.7b09223" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslOns7fL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=139&publication_year=2017&pages=17253-17256&issue=48&author=S.+A.+Barilauthor=A.+L.+Koenigauthor=M.+W.+Kroneauthor=K.+I.+Albaneseauthor=C.+Q.+Heauthor=G.+Y.+Leeauthor=K.+N.+Houkauthor=M.+L.+Watersauthor=E.+M.+Brustad&title=Investigation+of+Trimethyllysine+Binding+by+the+HP1+Chromodomain+via+Unnatural+Amino+Acid+Mutagenesis&doi=10.1021%2Fjacs.7b09223"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Investigation of Trimethyllysine Binding by the HP1 Chromodomain via Unnatural Amino Acid Mutagenesis</span></div><div class="casAuthors">Baril, Stefanie A.; Koenig, Amber L.; Krone, Mackenzie W.; Albanese, Katherine I.; He, Cyndi Qixin; Lee, Ga Young; Houk, Kendall N.; Waters, Marcey L.; Brustad, Eric M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">139</span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">17253-17256</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Trimethyllysine (Kme3) reader proteins are targets for inhibition due to their role in mediating gene expression.  While all such reader proteins bind Kme3 in an arom. cage, the driving force for binding may differ; some readers exhibit evidence for cation-π interactions while others do not.  Here, we report a general unnatural amino acid mutagenesis (Tyr → Phe deriv. mutants) approach to quantify the contribution of individual Tyr residues to cation binding using the heterochromatin protein 1 (HP1) chromodomain as a model system.  We demonstrated that 2 Tyr residues (Tyr-24 and Tyr-48) bound to a Kme3-histone tail peptide via cation-π interactions, but linear free energy trends suggested they did not contribute equally to binding.  The x-ray crystal structures of the HP1 chromodomain Tyr → Phe deriv. mutants and computational anal. suggested that the distance and degree of contact between Tyr residues and Kme3 plays an important role in tuning cation-π-mediated Kme3 recognition.  While cation-π interactions have been studied in a no. of proteins, this work is the 1st to utilize direct binding assays, x-ray crystallog., and modeling, to pinpoint factors that influence the magnitude of the individual cation-π interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokN25CY4EicrVg90H21EOLACvtfcHk0lilAYIeizXlMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslOns7fL&md5=a19fe43332a270f2fc1ca4451477d4e0</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fjacs.7b09223&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.7b09223%26sid%3Dliteratum%253Aachs%26aulast%3DBaril%26aufirst%3DS.%2BA.%26aulast%3DKoenig%26aufirst%3DA.%2BL.%26aulast%3DKrone%26aufirst%3DM.%2BW.%26aulast%3DAlbanese%26aufirst%3DK.%2BI.%26aulast%3DHe%26aufirst%3DC.%2BQ.%26aulast%3DLee%26aufirst%3DG.%2BY.%26aulast%3DHouk%26aufirst%3DK.%2BN.%26aulast%3DWaters%26aufirst%3DM.%2BL.%26aulast%3DBrustad%26aufirst%3DE.%2BM.%26atitle%3DInvestigation%2520of%2520Trimethyllysine%2520Binding%2520by%2520the%2520HP1%2520Chromodomain%2520via%2520Unnatural%2520Amino%2520Acid%2520Mutagenesis%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2017%26volume%3D139%26issue%3D48%26spage%3D17253%26epage%3D17256%26doi%3D10.1021%2Fjacs.7b09223" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, Y.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tharp, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weber, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koenig, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waters, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Summerer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W. R.</span></span> <span> </span><span class="NLM_article-title">Genetically Encoded Fluorophenylalanines Enable Insights into the Recognition of Lysine Trimethylation by an Epigenetic Reader</span>. <i>Chem. Commun. (Cambridge, U. K.)</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>52</i></span> (<span class="NLM_issue">85</span>),  <span class="NLM_fpage">12606</span>– <span class="NLM_lpage">12609</span>, <span class="refDoi"> DOI: 10.1039/C6CC05959G</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1039%2FC6CC05959G" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=27711380" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFOrsLvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2016&pages=12606-12609&issue=85&author=Y.-J.+Leeauthor=M.+J.+Schmidtauthor=J.+M.+Tharpauthor=A.+Weberauthor=A.+L.+Koenigauthor=H.+Zhengauthor=J.+Gaoauthor=M.+L.+Watersauthor=D.+Summererauthor=W.+R.+Liu&title=Genetically+Encoded+Fluorophenylalanines+Enable+Insights+into+the+Recognition+of+Lysine+Trimethylation+by+an+Epigenetic+Reader&doi=10.1039%2FC6CC05959G"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Genetically encoded fluorophenylalanines enable insights into the recognition of lysine trimethylation by an epigenetic reader</span></div><div class="casAuthors">Lee, Yan-Jiun; Schmidt, M. J.; Tharp, Jeffery M.; Weber, Annemarie; Koenig, Amber L.; Zheng, Hong; Gao, Jianmin; Waters, Marcey L.; Summerer, Daniel; Liu, Wenshe R.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">85</span>),
    <span class="NLM_cas:pages">12606-12609</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Fluorophenylalanines bearing 2-5 fluorine atoms at the Ph ring have been genetically encoded by amber codon.  Replacement of F59, a phenylalanine residue that is directly involved in interactions with trimethylated K9 of histone H3, in the Mpp8 chromodomain recombinantly with fluorophenylalanines significantly impairs the binding to a K9-trimethylated H3 peptide.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqE6UCOJqwbPrVg90H21EOLACvtfcHk0lh3phLD9RpIew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFOrsLvJ&md5=ae08d74f4665b47dc7928dfa591717e2</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1039%2FC6CC05959G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6CC05959G%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DY.-J.%26aulast%3DSchmidt%26aufirst%3DM.%2BJ.%26aulast%3DTharp%26aufirst%3DJ.%2BM.%26aulast%3DWeber%26aufirst%3DA.%26aulast%3DKoenig%26aufirst%3DA.%2BL.%26aulast%3DZheng%26aufirst%3DH.%26aulast%3DGao%26aufirst%3DJ.%26aulast%3DWaters%26aufirst%3DM.%2BL.%26aulast%3DSummerer%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DW.%2BR.%26atitle%3DGenetically%2520Encoded%2520Fluorophenylalanines%2520Enable%2520Insights%2520into%2520the%2520Recognition%2520of%2520Lysine%2520Trimethylation%2520by%2520an%2520Epigenetic%2520Reader%26jtitle%3DChem.%2520Commun.%2520%2528Cambridge%252C%2520U.%2520K.%2529%26date%3D2016%26volume%3D52%26issue%3D85%26spage%3D12606%26epage%3D12609%26doi%3D10.1039%2FC6CC05959G" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Milosevich, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hof, F.</span></span> <span> </span><span class="NLM_article-title">Chemical Inhibitors of Epigenetic Methyllysine Reader Proteins</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">1570</span>– <span class="NLM_lpage">1583</span>, <span class="refDoi"> DOI: 10.1021/acs.biochem.5b01073</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biochem.5b01073" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFOmtrnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=1570-1583&issue=11&author=N.+Milosevichauthor=F.+Hof&title=Chemical+Inhibitors+of+Epigenetic+Methyllysine+Reader+Proteins&doi=10.1021%2Facs.biochem.5b01073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical Inhibitors of Epigenetic Methyllysine Reader Proteins</span></div><div class="casAuthors">Milosevich, Natalia; Hof, Fraser</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1570-1583</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein methylation is a common post-translational modification with diverse biol. functions.  Methyllysine reader proteins are increasingly a focus of epigenetics research and play important roles in regulating many cellular processes.  These reader proteins are vital players in development, cell cycle regulation, stress responses, oncogenesis, and other disease pathways.  The recent emergence of a small no. of chem. inhibitors for methyllysine reader proteins supports the viability of these proteins as targets for drug development.  This article introduces the biochem. and biol. of methyllysine reader proteins, provides an overview of functions for those families of readers that have been targeted to date (MBT, PHD, tudor, and chromodomains), and reviews the development of synthetic agents that directly block their methyllysine reading functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGps23HnAOk3T7Vg90H21EOLACvtfcHk0lh3phLD9RpIew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFOmtrnM&md5=0d9ec539ad7b345bec4ecb8a252a9ff8</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Facs.biochem.5b01073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biochem.5b01073%26sid%3Dliteratum%253Aachs%26aulast%3DMilosevich%26aufirst%3DN.%26aulast%3DHof%26aufirst%3DF.%26atitle%3DChemical%2520Inhibitors%2520of%2520Epigenetic%2520Methyllysine%2520Reader%2520Proteins%26jtitle%3DBiochemistry%26date%3D2016%26volume%3D55%26issue%3D11%26spage%3D1570%26epage%3D1583%26doi%3D10.1021%2Facs.biochem.5b01073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dickson, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norris, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cholensky, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tempel, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sagum, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Black, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baughman, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senisterra, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pattenden, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, L. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S. V.</span></span> <span> </span><span class="NLM_article-title">A Cellular Chemical Probe Targeting the Chromodomains of Polycomb Repressive Complex 1</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">180</span>– <span class="NLM_lpage">187</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1038%2Fnchembio.2007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=26807715" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFyis7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=180-187&issue=3&author=J.+I.+Stuckeyauthor=B.+M.+Dicksonauthor=N.+Chengauthor=Y.+Liuauthor=J.+L.+Norrisauthor=S.+H.+Cholenskyauthor=W.+Tempelauthor=S.+Qinauthor=K.+G.+Huberauthor=C.+Sagumauthor=K.+Blackauthor=F.+Liauthor=X.+P.+Huangauthor=B.+L.+Rothauthor=B.+M.+Baughmanauthor=G.+Senisterraauthor=S.+G.+Pattendenauthor=M.+Vedadiauthor=P.+J.+Brownauthor=M.+T.+Bedfordauthor=J.+Minauthor=C.+H.+Arrowsmithauthor=L.+I.+Jamesauthor=S.+V.+Frye&title=A+Cellular+Chemical+Probe+Targeting+the+Chromodomains+of+Polycomb+Repressive+Complex+1&doi=10.1038%2Fnchembio.2007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">A cellular chemical probe targeting the chromodomains of Polycomb repressive complex 1</span></div><div class="casAuthors">Stuckey, Jacob I.; Dickson, Bradley M.; Cheng, Nancy; Liu, Yanli; Norris, Jacqueline L.; Cholensky, Stephanie H.; Tempel, Wolfram; Qin, Su; Huber, Katherine G.; Sagum, Cari; Black, Karynne; Li, Fengling; Huang, Xi-Ping; Roth, Bryan L.; Baughman, Brandi M.; Senisterra, Guillermo; Pattenden, Samantha G.; Vedadi, Masoud; Brown, Peter J.; Bedford, Mark T.; Min, Jinrong; Arrowsmith, Cheryl H.; James, Lindsey I.; Frye, Stephen V.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">180-187</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We report the design and characterization of UNC3866, a potent antagonist of the methyllysine (Kme) reading function of the Polycomb CBX and CDY families of chromodomains.  Polycomb CBX proteins regulate gene expression by targeting Polycomb repressive complex 1 (PRC1) to sites of H3K27me3 via their chromodomains.  UNC3866 binds the chromodomains of CBX4 and CBX7 most potently, with a Kd of ∼100 nM for each, and is 6- to 18-fold selective as compared to seven other CBX and CDY chromodomains while being highly selective over >250 other protein targets.  X-ray crystallog. revealed that UNC3866's interactions with the CBX chromodomains closely mimic those of the methylated H3 tail.  UNC4195, a biotinylated deriv. of UNC3866, was used to demonstrate that UNC3866 engages intact PRC1 and that EED incorporation into PRC1 is isoform dependent in PC3 prostate cancer cells.  Finally, UNC3866 inhibits PC3 cell proliferation, consistent with the known ability of CBX7 overexpression to confer a growth advantage, whereas UNC4219, a methylated neg. control compd., has negligible effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobRZ8YoHZWk7Vg90H21EOLACvtfcHk0lh3phLD9RpIew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFyis7o%253D&md5=97fe8dd8e35094a8c91d5d4bdc265b4f</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2007%26sid%3Dliteratum%253Aachs%26aulast%3DStuckey%26aufirst%3DJ.%2BI.%26aulast%3DDickson%26aufirst%3DB.%2BM.%26aulast%3DCheng%26aufirst%3DN.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DCholensky%26aufirst%3DS.%2BH.%26aulast%3DTempel%26aufirst%3DW.%26aulast%3DQin%26aufirst%3DS.%26aulast%3DHuber%26aufirst%3DK.%2BG.%26aulast%3DSagum%26aufirst%3DC.%26aulast%3DBlack%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DHuang%26aufirst%3DX.%2BP.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DBaughman%26aufirst%3DB.%2BM.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DPattenden%26aufirst%3DS.%2BG.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DMin%26aufirst%3DJ.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DJames%26aufirst%3DL.%2BI.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26atitle%3DA%2520Cellular%2520Chemical%2520Probe%2520Targeting%2520the%2520Chromodomains%2520of%2520Polycomb%2520Repressive%2520Complex%25201%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26issue%3D3%26spage%3D180%26epage%3D187%26doi%3D10.1038%2Fnchembio.2007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simpson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norris-Drouin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cholensky, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasca, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dickson, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearce, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, L. I.</span></span> <span> </span><span class="NLM_article-title">Structure-Activity Relationships and Kinetic Studies of Peptidic Antagonists of CBX Chromodomains</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">19</span>),  <span class="NLM_fpage">8913</span>– <span class="NLM_lpage">8923</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00801</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00801" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVagu7zO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=8913-8923&issue=19&author=J.+I.+Stuckeyauthor=C.+Simpsonauthor=J.+L.+Norris-Drouinauthor=S.+H.+Cholenskyauthor=J.+Leeauthor=R.+Pascaauthor=N.+Chengauthor=B.+M.+Dicksonauthor=K.+H.+Pearceauthor=S.+V.+Fryeauthor=L.+I.+James&title=Structure-Activity+Relationships+and+Kinetic+Studies+of+Peptidic+Antagonists+of+CBX+Chromodomains&doi=10.1021%2Facs.jmedchem.6b00801"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Activity Relationships and Kinetic Studies of Peptidic Antagonists of CBX Chromodomains</span></div><div class="casAuthors">Stuckey, Jacob I.; Simpson, Catherine; Norris-Drouin, Jacqueline L.; Cholensky, Stephanie H.; Lee, Junghyun; Pasca, Ryan; Cheng, Nancy; Dickson, Bradley M.; Pearce, Kenneth H.; Frye, Stephen V.; James, Lindsey I.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8913-8923</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To better understand the contribution of methyl-lysine (Kme) binding proteins to various disease states, the authors recently developed and reported the discovery of I (UNC3866), a chem. probe that targets two families of Kme binding proteins, CBX and CDY chromodomains, with selectivity for CBX4 and -7.  The discovery of I was enabled in part by the use of mol. dynamics simulations performed with CBX7 and its endogenous substrate.  Herein, the authors describe the design, synthesis and structure-activity relationship studies that led to the development of I and provide support for the model of CBX7-ligand recognition by examg. the binding kinetics of the antagonists with CBX7 as detd. by surface-plasmon resonance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotQTqv5W7R6rVg90H21EOLACvtfcHk0ljMMFYrFsf92w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVagu7zO&md5=c53cfc1aeaf1d0ea12b67a949d74635b</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00801&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00801%26sid%3Dliteratum%253Aachs%26aulast%3DStuckey%26aufirst%3DJ.%2BI.%26aulast%3DSimpson%26aufirst%3DC.%26aulast%3DNorris-Drouin%26aufirst%3DJ.%2BL.%26aulast%3DCholensky%26aufirst%3DS.%2BH.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DPasca%26aufirst%3DR.%26aulast%3DCheng%26aufirst%3DN.%26aulast%3DDickson%26aufirst%3DB.%2BM.%26aulast%3DPearce%26aufirst%3DK.%2BH.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DJames%26aufirst%3DL.%2BI.%26atitle%3DStructure-Activity%2520Relationships%2520and%2520Kinetic%2520Studies%2520of%2520Peptidic%2520Antagonists%2520of%2520CBX%2520Chromodomains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26issue%3D19%26spage%3D8913%26epage%3D8923%26doi%3D10.1021%2Facs.jmedchem.6b00801" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lamb, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bsteh, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dishman, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moussa, H. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norris, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cholensky, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sagum, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stanton, B. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearce, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kenakin, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kireev, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, L. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S. V.</span></span> <span> </span><span class="NLM_article-title">Discovery and Characterization of a Cellular Potent Positive Allosteric Modulator of the Polycomb Repressive Complex 1 Chromodomain, CBX7</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>26</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">1365</span>– <span class="NLM_lpage">1379.e22</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2019.07.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1016%2Fj.chembiol.2019.07.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=31422906" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1SitLnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2019&pages=1365-1379.e22&issue=10&author=K.+N.+Lambauthor=D.+Bstehauthor=S.+N.+Dishmanauthor=H.+F.+Moussaauthor=H.+Fanauthor=J.+I.+Stuckeyauthor=J.+L.+Norrisauthor=S.+H.+Cholenskyauthor=D.+Liauthor=J.+Wangauthor=C.+Sagumauthor=B.+Z.+Stantonauthor=M.+T.+Bedfordauthor=K.+H.+Pearceauthor=T.+P.+Kenakinauthor=D.+B.+Kireevauthor=G.+G.+Wangauthor=L.+I.+Jamesauthor=O.+Bellauthor=S.+V.+Frye&title=Discovery+and+Characterization+of+a+Cellular+Potent+Positive+Allosteric+Modulator+of+the+Polycomb+Repressive+Complex+1+Chromodomain%2C+CBX7&doi=10.1016%2Fj.chembiol.2019.07.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Characterization of a Cellular Potent Positive Allosteric Modulator of the Polycomb Repressive Complex 1 Chromodomain, CBX7</span></div><div class="casAuthors">Lamb, Kelsey N.; Bsteh, Daniel; Dishman, Sarah N.; Moussa, Hagar F.; Fan, Huitao; Stuckey, Jacob I.; Norris, Jacqueline L.; Cholensky, Stephanie H.; Li, Dongxu; Wang, Jingkui; Sagum, Cari; Stanton, Benjamin Z.; Bedford, Mark T.; Pearce, Kenneth H.; Kenakin, Terry P.; Kireev, Dmitri B.; Wang, Gang Greg; James, Lindsey I.; Bell, Oliver; Frye, Stephen V.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1365-1379.e22</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Polycomb-directed repression of gene expression is frequently misregulated in human diseases.  A quant. and target-specific cellular assay was utilized to discover the first potent pos. allosteric modulator (PAM) peptidomimetic, UNC4976, of nucleic acid binding by CBX7, a chromodomain methyl-lysine reader of Polycomb repressive complex 1.  The PAM activity of UNC4976 resulted in enhanced efficacy across three orthogonal cellular assays by simultaneously antagonizing H3K27me3-specific recruitment of CBX7 to target genes while increasing non-specific binding to DNA and RNA.  PAM activity thereby reequilibrates PRC1 away from H3K27me3 target regions.  Together, our discovery and characterization of UNC4976 not only revealed the most cellularly potent PRC1-specific chem. probe to date, but also uncovers a potential mechanism of Polycomb regulation with implications for non-histone lysine methylated interaction partners.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXrcSgaffNibVg90H21EOLACvtfcHk0ljMMFYrFsf92w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhs1SitLnK&md5=81c0bc1a765f319637d8deb8e3d3e518</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2019.07.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2019.07.013%26sid%3Dliteratum%253Aachs%26aulast%3DLamb%26aufirst%3DK.%2BN.%26aulast%3DBsteh%26aufirst%3DD.%26aulast%3DDishman%26aufirst%3DS.%2BN.%26aulast%3DMoussa%26aufirst%3DH.%2BF.%26aulast%3DFan%26aufirst%3DH.%26aulast%3DStuckey%26aufirst%3DJ.%2BI.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DCholensky%26aufirst%3DS.%2BH.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DSagum%26aufirst%3DC.%26aulast%3DStanton%26aufirst%3DB.%2BZ.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DPearce%26aufirst%3DK.%2BH.%26aulast%3DKenakin%26aufirst%3DT.%2BP.%26aulast%3DKireev%26aufirst%3DD.%2BB.%26aulast%3DWang%26aufirst%3DG.%2BG.%26aulast%3DJames%26aufirst%3DL.%2BI.%26aulast%3DBell%26aufirst%3DO.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26atitle%3DDiscovery%2520and%2520Characterization%2520of%2520a%2520Cellular%2520Potent%2520Positive%2520Allosteric%2520Modulator%2520of%2520the%2520Polycomb%2520Repressive%2520Complex%25201%2520Chromodomain%252C%2520CBX7%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2019%26volume%3D26%26issue%3D10%26spage%3D1365%26epage%3D1379.e22%26doi%3D10.1016%2Fj.chembiol.2019.07.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barnash, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">The, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norris-Drouin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cholensky, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Worley, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, L. I.</span></span> <span> </span><span class="NLM_article-title">Discovery of Peptidomimetic Ligands of EED as Allosteric Inhibitors of PRC2</span>. <i>ACS Comb. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>19</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">161</span>– <span class="NLM_lpage">172</span>, <span class="refDoi"> DOI: 10.1021/acscombsci.6b00174</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscombsci.6b00174" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitFWgsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2017&pages=161-172&issue=3&author=K.+D.+Barnashauthor=J.+Theauthor=J.+L.+Norris-Drouinauthor=S.+H.+Cholenskyauthor=B.+M.+Worleyauthor=F.+Liauthor=J.+I.+Stuckeyauthor=P.+J.+Brownauthor=M.+Vedadiauthor=C.+H.+Arrowsmithauthor=S.+V.+Fryeauthor=L.+I.+James&title=Discovery+of+Peptidomimetic+Ligands+of+EED+as+Allosteric+Inhibitors+of+PRC2&doi=10.1021%2Facscombsci.6b00174"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Peptidomimetic Ligands of EED as Allosteric Inhibitors of PRC2</span></div><div class="casAuthors">Barnash, Kimberly D.; The, Juliana; Norris-Drouin, Jacqueline L.; Cholensky, Stephanie H.; Worley, Beau M.; Li, Fengling; Stuckey, Jacob I.; Brown, Peter J.; Vedadi, Masoud; Arrowsmith, Cheryl H.; Frye, Stephen V.; James, Lindsey I.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Combinatorial Science</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">161-172</span>CODEN:
                <span class="NLM_cas:coden">ACSCCC</span>;
        ISSN:<span class="NLM_cas:issn">2156-8944</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The function of EED within Polycomb repressive complex 2 (PRC2) is mediated by a complex network of protein-protein interactions.  Allosteric activation of PRC2 by binding of methylated proteins to EED's arom. cage is essential for full catalytic activity, but details of this regulation are not fully understood.  EED's recognition of the product of PRC2 activity, histone H3 lysine 27 trimethylation (H3K27me3), stimulates PRC2 methyltransferase activity at adjacent nucleosomes leading to H3K27me3 propagation and, ultimately, gene repression.  By coupling combinatorial chem. and structure-based design, we optimized a low affinity methylated Jarid2 peptide to a smaller, more potent peptidomimetic ligand (Kd = 1.14 ± 0.14 μM) of the arom. cage of EED.  Our strategy illustrates the effectiveness of applying combinatorial chem. to achieve both ligand potency and property optimization.  Furthermore, the resulting ligands, UNC5114 and UNC5115, demonstrate that targeted disruption of EED's reader function can lead to allosteric inhibition of PRC2 catalytic activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwOqOVJ8hSD7Vg90H21EOLACvtfcHk0lj7IcmBmRkpfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitFWgsrw%253D&md5=0e7da46fc40d89f4095ac979374338fc</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Facscombsci.6b00174&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscombsci.6b00174%26sid%3Dliteratum%253Aachs%26aulast%3DBarnash%26aufirst%3DK.%2BD.%26aulast%3DThe%26aufirst%3DJ.%26aulast%3DNorris-Drouin%26aufirst%3DJ.%2BL.%26aulast%3DCholensky%26aufirst%3DS.%2BH.%26aulast%3DWorley%26aufirst%3DB.%2BM.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DStuckey%26aufirst%3DJ.%2BI.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DJames%26aufirst%3DL.%2BI.%26atitle%3DDiscovery%2520of%2520Peptidomimetic%2520Ligands%2520of%2520EED%2520as%2520Allosteric%2520Inhibitors%2520of%2520PRC2%26jtitle%3DACS%2520Comb.%2520Sci.%26date%3D2017%26volume%3D19%26issue%3D3%26spage%3D161%26epage%3D172%26doi%3D10.1021%2Facscombsci.6b00174" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Suh, J. L. J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnash, K. D. K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abramyan, T. M. T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">The, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engelberg, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J. P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kireev, D. B. D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H. C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, L. I. L. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S. V. S. V.</span></span> <span> </span><span class="NLM_article-title">Discovery of Selective Activators of PRC2Mutant EED-I363M</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">6524</span>, <span class="refDoi"> DOI: 10.1038/s41598-019-43005-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1038%2Fs41598-019-43005-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=31024026" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A280%3ADC%252BB3M%252Fptlajtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2019&pages=6524&issue=1&author=J.+L.+J.+L.+Suhauthor=K.+D.+K.+D.+Barnashauthor=T.+M.+T.+M.+Abramyanauthor=F.+Liauthor=J.+Theauthor=I.+A.+Engelbergauthor=M.+Vedadiauthor=P.+J.+P.+J.+Brownauthor=D.+B.+D.+B.+Kireevauthor=C.+H.+C.+H.+Arrowsmithauthor=L.+I.+L.+I.+Jamesauthor=S.+V.+S.+V.+Frye&title=Discovery+of+Selective+Activators+of+PRC2Mutant+EED-I363M&doi=10.1038%2Fs41598-019-43005-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of selective activators of PRC2 mutant EED-I363M</span></div><div class="casAuthors">Suh Junghyun L; Barnash Kimberly D; Abramyan Tigran M; Engelberg Isabelle A; Kireev Dmitri B; James Lindsey I; Frye Stephen V; Barnash Kimberly D; Li Fengling; The Juliana; Vedadi Masoud; Brown Peter J; Arrowsmith Cheryl H; Arrowsmith Cheryl H</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">6524</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Many common disease-causing mutations result in loss-of-function (LOF) of the proteins in which they occur.  LOF mutations have proven recalcitrant to pharmacologic intervention, presenting a challenge for the development of targeted therapeutics.  Polycomb repressive complex 2 (PRC2), which contains core subunits (EZH2, EED, and SUZ12), regulates gene activity by trimethylation of histone 3 lysine 27.  The dysregulation of PRC2 catalytic activity by mutations has been implicated in cancer and other diseases.  Among the mutations that cause PRC2 malfunction, an I363M LOF mutation of EED has been identified in myeloid disorders, where it prevents allosteric activation of EZH2 catalysis.  We describe structure-based design and computational simulations of ligands created to ameliorate this LOF.  Notably, these compounds selectively stimulate the catalytic activity of PRC2-EED-I363M over wildtype-PRC2.  Overall, this work demonstrates the feasibility of developing targeted therapeutics for PRC2-EED-I363M that act as allosteric agonists, potentially correcting this LOF mutant phenotype.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTy_y3UKz_BfTQLBVDQn4bCfW6udTcc2eYeXFWIJ0OFArntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3M%252Fptlajtg%253D%253D&md5=92fb9e3260bdc49440a1b803c833ed9e</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fs41598-019-43005-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-019-43005-z%26sid%3Dliteratum%253Aachs%26aulast%3DSuh%26aufirst%3DJ.%2BL.%2BJ.%2BL.%26aulast%3DBarnash%26aufirst%3DK.%2BD.%2BK.%2BD.%26aulast%3DAbramyan%26aufirst%3DT.%2BM.%2BT.%2BM.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DThe%26aufirst%3DJ.%26aulast%3DEngelberg%26aufirst%3DI.%2BA.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%2BP.%2BJ.%26aulast%3DKireev%26aufirst%3DD.%2BB.%2BD.%2BB.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%2BC.%2BH.%26aulast%3DJames%26aufirst%3DL.%2BI.%2BL.%2BI.%26aulast%3DFrye%26aufirst%3DS.%2BV.%2BS.%2BV.%26atitle%3DDiscovery%2520of%2520Selective%2520Activators%2520of%2520PRC2Mutant%2520EED-I363M%26jtitle%3DSci.%2520Rep.%26date%3D2019%26volume%3D9%26issue%3D1%26spage%3D6524%26doi%3D10.1038%2Fs41598-019-43005-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barnash, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamb, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norris, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cholensky, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kireev, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, L. I.</span></span> <span> </span><span class="NLM_article-title">Chromodomain Ligand Optimization via Target-Class Directed Combinatorial Repurposing</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">2475</span>– <span class="NLM_lpage">2483</span>, <span class="refDoi"> DOI: 10.1021/acschembio.6b00415</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.6b00415" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVOgt7fK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=2475-2483&issue=9&author=K.+D.+Barnashauthor=K.+N.+Lambauthor=J.+I.+Stuckeyauthor=J.+L.+Norrisauthor=S.+H.+Cholenskyauthor=D.+B.+Kireevauthor=S.+V.+Fryeauthor=L.+I.+James&title=Chromodomain+Ligand+Optimization+via+Target-Class+Directed+Combinatorial+Repurposing&doi=10.1021%2Facschembio.6b00415"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Chromodomain Ligand Optimization via Target-Class Directed Combinatorial Repurposing</span></div><div class="casAuthors">Barnash, Kimberly D.; Lamb, Kelsey N.; Stuckey, Jacob I.; Norris, Jacqueline L.; Cholensky, Stephanie H.; Kireev, Dmitri B.; Frye, Stephen V.; James, Lindsey I.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2475-2483</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Efforts to develop strategies for small-mol. chem. probe discovery against the readers of the methyl-lysine (Kme) post-translational modification have been met with limited success.  Targeted disruption of these protein-protein interactions via peptidomimetic inhibitor optimization is a promising alternative to small-mol. hit discovery; however, recognition of identical peptide motifs by multiple Kme reader proteins presents a unique challenge in the development of selective Kme reader chem. probes.  These selectivity challenges are exemplified by the Polycomb repressive complex 1 (PRC1) chem. probe, UNC3866, which demonstrates submicromolar off-target affinity toward the non-PRC1 chromodomains CDYL2 and CDYL.  Moreover, since peptidomimetics are challenging subjects for structure-activity relationship (SAR) studies, traditional optimization of UNC3866 would prove costly and time-consuming.  Herein, we report a broadly applicable strategy for the affinity-based, target-class screening of chromodomains via the repurposing of UNC3866 in an efficient, combinatorial peptide library.  A first-generation library yielded UNC4991, a UNC3866 analog that exhibits a distinct selectivity profile while maintaining submicromolar affinity toward the CDYL chromodomains.  Addnl., in vitro pull-down expts. from HeLa nuclear lysates further demonstrate the selectivity and utility of this compd. for future elucidation of CDYL protein function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkyoB69iWV7bVg90H21EOLACvtfcHk0lgufa4dU1Kxcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVOgt7fK&md5=89613f3d7e6d533caa5f27f7b7c0629c</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facschembio.6b00415&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.6b00415%26sid%3Dliteratum%253Aachs%26aulast%3DBarnash%26aufirst%3DK.%2BD.%26aulast%3DLamb%26aufirst%3DK.%2BN.%26aulast%3DStuckey%26aufirst%3DJ.%2BI.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DCholensky%26aufirst%3DS.%2BH.%26aulast%3DKireev%26aufirst%3DD.%2BB.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DJames%26aufirst%3DL.%2BI.%26atitle%3DChromodomain%2520Ligand%2520Optimization%2520via%2520Target-Class%2520Directed%2520Combinatorial%2520Repurposing%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26issue%3D9%26spage%3D2475%26epage%3D2483%26doi%3D10.1021%2Facschembio.6b00415" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rectenwald, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hardy, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norris-Drouin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cholensky, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, L. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearce, K. H.</span></span> <span> </span><span class="NLM_article-title">A General TR-FRET Assay Platform for High-Throughput Screening and Characterizing Inhibitors of Methyl-Lysine Reader Proteins</span>. <i>SLAS Discovery Adv. Life Sci. R&D</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>24</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">693</span>– <span class="NLM_lpage">700</span>, <span class="refDoi"> DOI: 10.1177/2472555219844569</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1177%2F2472555219844569" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2019&pages=693-700&issue=6&author=J.+M.+Rectenwaldauthor=P.+B.+Hardyauthor=J.+L.+Norris-Drouinauthor=S.+H.+Cholenskyauthor=L.+I.+Jamesauthor=S.+V.+Fryeauthor=K.+H.+Pearce&title=A+General+TR-FRET+Assay+Platform+for+High-Throughput+Screening+and+Characterizing+Inhibitors+of+Methyl-Lysine+Reader+Proteins&doi=10.1177%2F2472555219844569"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1177%2F2472555219844569&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F2472555219844569%26sid%3Dliteratum%253Aachs%26aulast%3DRectenwald%26aufirst%3DJ.%2BM.%26aulast%3DHardy%26aufirst%3DP.%2BB.%26aulast%3DNorris-Drouin%26aufirst%3DJ.%2BL.%26aulast%3DCholensky%26aufirst%3DS.%2BH.%26aulast%3DJames%26aufirst%3DL.%2BI.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DPearce%26aufirst%3DK.%2BH.%26atitle%3DA%2520General%2520TR-FRET%2520Assay%2520Platform%2520for%2520High-Throughput%2520Screening%2520and%2520Characterizing%2520Inhibitors%2520of%2520Methyl-Lysine%2520Reader%2520Proteins%26jtitle%3DSLAS%2520Discovery%2520Adv.%2520Life%2520Sci.%2520R%2526D%26date%3D2019%26volume%3D24%26issue%3D6%26spage%3D693%26epage%3D700%26doi%3D10.1177%2F2472555219844569" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Musselman, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Avvakumov, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watanabe, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abraham, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lalonde, M.-E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nuñez, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nickoloff, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kulesza, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yasui, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Côté, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutateladze, T. G.</span></span> <span> </span><span class="NLM_article-title">Molecular Basis for H3K36me3 Recognition by the Tudor Domain of PHF1</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>19</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">1266</span>– <span class="NLM_lpage">1272</span>, <span class="refDoi"> DOI: 10.1038/nsmb.2435</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1038%2Fnsmb.2435" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=23142980" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1Ckt7rE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=1266-1272&issue=12&author=C.+A.+Musselmanauthor=N.+Avvakumovauthor=R.+Watanabeauthor=C.+G.+Abrahamauthor=M.-E.+E.+Lalondeauthor=Z.+Hongauthor=C.+Allenauthor=S.+Royauthor=J.+K.+Nu%C3%B1ezauthor=J.+Nickoloffauthor=C.+A.+Kuleszaauthor=A.+Yasuiauthor=J.+C%C3%B4t%C3%A9author=T.+G.+Kutateladze&title=Molecular+Basis+for+H3K36me3+Recognition+by+the+Tudor+Domain+of+PHF1&doi=10.1038%2Fnsmb.2435"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular basis for H3K36me3 recognition by the Tudor domain of PHF1</span></div><div class="casAuthors">Musselman, Catherine A.; Avvakumov, Nikita; Watanabe, Reiko; Abraham, Christopher G.; Lalonde, Marie-Eve; Hong, Zehui; Allen, Christopher; Roy, Siddhartha; Nunez, James K.; Nickoloff, Jac; Kulesza, Caroline A.; Yasui, Akira; Cote, Jacques; Kutateladze, Tatiana G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1266-1272</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The PHD finger protein 1 (PHF1) is essential in epigenetic regulation and genome maintenance.  Here, the authors show that the Tudor domain of human PHF1 binds to histone H3 trimethylated at Lys-36 (H3K36me3).  The authors report a 1.9-Å resoln. crystal structure of the Tudor domain in complex with H3K36me3 peptide (residues 31-40) and describe the mol. mechanism of H3K36me3 recognition using NMR.  The binding of PHF1 to H3K36me3 inhibited the ability of the Polycomb PRC2 complex to methylate Lys-27 of histone H3 in vitro and in vivo.  Laser microirradn. data showed that PHF1 was transiently recruited to DNA double-strand breaks, and PHF1 mutants impaired in the H3K36me3 interaction exhibited reduced retention at double-strand break sites.  Together, these findings suggest that PHF1 can mediate the deposition of the repressive H3K27me3 mark and acts as a cofactor in early DNA-damage response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAczVxJthG_7Vg90H21EOLACvtfcHk0lgufa4dU1Kxcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1Ckt7rE&md5=f8e6e578ef81a645b5123e6b51836643</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.2435&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.2435%26sid%3Dliteratum%253Aachs%26aulast%3DMusselman%26aufirst%3DC.%2BA.%26aulast%3DAvvakumov%26aufirst%3DN.%26aulast%3DWatanabe%26aufirst%3DR.%26aulast%3DAbraham%26aufirst%3DC.%2BG.%26aulast%3DLalonde%26aufirst%3DM.-E.%2BE.%26aulast%3DHong%26aufirst%3DZ.%26aulast%3DAllen%26aufirst%3DC.%26aulast%3DRoy%26aufirst%3DS.%26aulast%3DNu%25C3%25B1ez%26aufirst%3DJ.%2BK.%26aulast%3DNickoloff%26aufirst%3DJ.%26aulast%3DKulesza%26aufirst%3DC.%2BA.%26aulast%3DYasui%26aufirst%3DA.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DJ.%26aulast%3DKutateladze%26aufirst%3DT.%2BG.%26atitle%3DMolecular%2520Basis%2520for%2520H3K36me3%2520Recognition%2520by%2520the%2520Tudor%2520Domain%2520of%2520PHF1%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2012%26volume%3D19%26issue%3D12%26spage%3D1266%26epage%3D1272%26doi%3D10.1038%2Fnsmb.2435" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rothbart, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.-Y. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tripathy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rockowitz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allis, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strahl, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, G. G.</span></span> <span> </span><span class="NLM_article-title">An H3K36 Methylation-Engaging Tudor Motif of Polycomb-like Proteins Mediates PRC2 Complex Targeting</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>49</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">571</span>– <span class="NLM_lpage">582</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2012.11.026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1016%2Fj.molcel.2012.11.026" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=23273982" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC3sXotFOl" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2013&pages=571-582&issue=3&author=L.+Caiauthor=S.+B.+Rothbartauthor=R.+Luauthor=B.+Xuauthor=W.-Y.+Y.+Chenauthor=A.+Tripathyauthor=S.+Rockowitzauthor=D.+Zhengauthor=D.+J.+Patelauthor=C.+D.+Allisauthor=B.+D.+Strahlauthor=J.+Songauthor=G.+G.+Wang&title=An+H3K36+Methylation-Engaging+Tudor+Motif+of+Polycomb-like+Proteins+Mediates+PRC2+Complex+Targeting&doi=10.1016%2Fj.molcel.2012.11.026"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">An H3K36 Methylation-Engaging Tudor Motif of Polycomb-like Proteins Mediates PRC2 Complex Targeting</span></div><div class="casAuthors">Cai, Ling; Rothbart, Scott B.; Lu, Rui; Xu, Bowen; Chen, Wei-Yi; Tripathy, Ashutosh; Rockowitz, Shira; Zheng, Deyou; Patel, Dinshaw J.; Allis, C. David; Strahl, Brian D.; Song, Jikui; Wang, Gang Greg</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">571-582</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Polycomb repressive complex 2 (PRC2) regulates pluripotency, differentiation, and tumorigenesis through catalysis of histone H3 lysine 27 trimethylation (H3K27me3) on chromatin.  However, the mechanisms that underlie PRC2 recruitment and spreading on chromatin remain unclear.  Here we report that histone H3 lysine 36 trimethylation (H3K36me3) binding activity is harbored in the Tudor motifs of PRC2-assocd. polycomb-like (PCL) proteins PHF1/PCL1 and PHF19/PCL3.  Ectopically expressed PHF1 induced Tudor-dependent stabilization of PRC2 complexes on bulk chromatin and mediated spreading of PRC2 and H3K27me3 into H3K36me3-contg. chromatin regions.  In murine pluripotent stem cells, we identified coexistence of H3K36me3, H3K27me3, and PHF19/PCL3 at a subset of poised developmental genes and demonstrated that PHF19/PCL3 Tudor function is required for optimal H3K27me3 and repression of these loci.  Collectively, our data suggest that PCL recognition of H3K36me3 promotes intrusion of PRC2 complexes into active chromatin regions to promote gene silencing and modulate the chromatin landscape during development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpMX9XeqddRbVg90H21EOLACvtfcHk0lhDt3uHWwxyqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXotFOl&md5=1bcf5ff44f9588863a4b286917055753</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2012.11.026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2012.11.026%26sid%3Dliteratum%253Aachs%26aulast%3DCai%26aufirst%3DL.%26aulast%3DRothbart%26aufirst%3DS.%2BB.%26aulast%3DLu%26aufirst%3DR.%26aulast%3DXu%26aufirst%3DB.%26aulast%3DChen%26aufirst%3DW.-Y.%2BY.%26aulast%3DTripathy%26aufirst%3DA.%26aulast%3DRockowitz%26aufirst%3DS.%26aulast%3DZheng%26aufirst%3DD.%26aulast%3DPatel%26aufirst%3DD.%2BJ.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26aulast%3DStrahl%26aufirst%3DB.%2BD.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DG.%2BG.%26atitle%3DAn%2520H3K36%2520Methylation-Engaging%2520Tudor%2520Motif%2520of%2520Polycomb-like%2520Proteins%2520Mediates%2520PRC2%2520Complex%2520Targeting%26jtitle%3DMol.%2520Cell%26date%3D2013%26volume%3D49%26issue%3D3%26spage%3D571%26epage%3D582%26doi%3D10.1016%2Fj.molcel.2012.11.026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bian, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, J.</span></span> <span> </span><span class="NLM_article-title">Tudor Domains of the PRC2 Components PHF1 and PHF19 Selectively Bind to Histone H3K36me3</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>430</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">547</span>– <span class="NLM_lpage">553</span>, <span class="refDoi"> DOI: 10.1016/j.bbrc.2012.11.116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1016%2Fj.bbrc.2012.11.116" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=23228662" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvV2ktbfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=430&publication_year=2013&pages=547-553&issue=2&author=S.+Qinauthor=Y.+Guoauthor=C.+Xuauthor=C.+Bianauthor=M.+Fuauthor=S.+Gongauthor=J.+Min&title=Tudor+Domains+of+the+PRC2+Components+PHF1+and+PHF19+Selectively+Bind+to+Histone+H3K36me3&doi=10.1016%2Fj.bbrc.2012.11.116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Tudor domains of the PRC2 components PHF1 and PHF19 selectively bind to histone H3K36me3</span></div><div class="casAuthors">Qin, Su; Guo, Yahong; Xu, Chao; Bian, Chuanbing; Fu, Minfei; Gong, Sarah; Min, Jinrong</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">430</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">547-553</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">PRC2 is the major H3K27 methyltransferase and is responsible for maintaining repressed gene expression patterns throughout development.  It contains four core components: EZH2, EED, SUZ12 and RbAp46/48 and some cell-type specific components.  In this study, we focused on characterizing the histone binding domains of PHF1 and PHF19, and found that the Tudor domains of PHF1 and PHF19 selectively bind to histone H3K36me3.  Structural anal. of these Tudor domains also shed light on how these Tudor domains selectively bind to histone H3K36me3.  The histone H3K36me3 binding by the Tudor domains of PHF1, PHF19 and likely MTF2 provide another recruitment and regulatory mechanism for the PRC2 complex.  In addn., we found that the first PHD domains of PHF1 and PHF19 do not exhibit histone H3K4 binding ability, nor do they affect the Tudor domain binding to histones.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLQ6VOXMl3FbVg90H21EOLACvtfcHk0lhDt3uHWwxyqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvV2ktbfE&md5=90c224f4ec9068d18de5c01ca61962d3</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2012.11.116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2012.11.116%26sid%3Dliteratum%253Aachs%26aulast%3DQin%26aufirst%3DS.%26aulast%3DGuo%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DC.%26aulast%3DBian%26aufirst%3DC.%26aulast%3DFu%26aufirst%3DM.%26aulast%3DGong%26aufirst%3DS.%26aulast%3DMin%26aufirst%3DJ.%26atitle%3DTudor%2520Domains%2520of%2520the%2520PRC2%2520Components%2520PHF1%2520and%2520PHF19%2520Selectively%2520Bind%2520to%2520Histone%2520H3K36me3%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2013%26volume%3D430%26issue%3D2%26spage%3D547%26epage%3D553%26doi%3D10.1016%2Fj.bbrc.2012.11.116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ballaré, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lange, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lapinaite, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morey, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pascual, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liefke, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gozani, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlomagno, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benitah, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Croce, L.</span></span> <span> </span><span class="NLM_article-title">Phf19 Links Methylated Lys36 of Histone H3 to Regulation of Polycomb Activity</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>19</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">1257</span>– <span class="NLM_lpage">1265</span>, <span class="refDoi"> DOI: 10.1038/nsmb.2434</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1038%2Fnsmb.2434" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=23104054" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFOmtLrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=1257-1265&issue=12&author=C.+Ballar%C3%A9author=M.+Langeauthor=A.+Lapinaiteauthor=G.+M.+Martinauthor=L.+Moreyauthor=G.+Pascualauthor=R.+Liefkeauthor=B.+Simonauthor=Y.+Shiauthor=O.+Gozaniauthor=T.+Carlomagnoauthor=S.+A.+Benitahauthor=L.+Di+Croce&title=Phf19+Links+Methylated+Lys36+of+Histone+H3+to+Regulation+of+Polycomb+Activity&doi=10.1038%2Fnsmb.2434"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Phf19 links methylated Lys36 of histone H3 to regulation of Polycomb activity</span></div><div class="casAuthors">Ballare, Cecilia; Lange, Martin; Lapinaite, Audrone; Martin, Gloria Mas; Morey, Lluis; Pascual, Gloria; Liefke, Robert; Simon, Bernd; Shi, Yang; Gozani, Or; Carlomagno, Teresa; Benitah, Salvador Aznar; Di Croce, Luciano</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1257-1265</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Polycomb-group proteins are transcriptional repressors with essential roles in embryonic development.  Polycomb repressive complex 2 (PRC2) contains the methyltransferase activity for Lys27.  However, the role of other histone modifications in regulating PRC2 activity is just beginning to be understood.  Here we show that direct recognition of methylated histone H3 Lys36 (H3K36me), a mark assocd. with activation, by the PRC2 subunit Phf19 is required for the full enzymic activity of the PRC2 complex.  Using NMR spectroscopy, we provide structural evidence for this interaction.  Furthermore, we show that Phf19 binds to a subset of PRC2 targets in mouse embryonic stem cells and that this is required for their repression and for H3K27me3 deposition.  These findings show that the interaction of Phf19 with H3K36me2 and H3K36me3 is essential for PRC2 complex activity and for proper regulation of gene repression in embryonic stem cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTLpeahK4UbLVg90H21EOLACvtfcHk0lhDt3uHWwxyqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFOmtLrM&md5=e465ee81ea9643a80487739332ae8683</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.2434&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.2434%26sid%3Dliteratum%253Aachs%26aulast%3DBallar%25C3%25A9%26aufirst%3DC.%26aulast%3DLange%26aufirst%3DM.%26aulast%3DLapinaite%26aufirst%3DA.%26aulast%3DMartin%26aufirst%3DG.%2BM.%26aulast%3DMorey%26aufirst%3DL.%26aulast%3DPascual%26aufirst%3DG.%26aulast%3DLiefke%26aufirst%3DR.%26aulast%3DSimon%26aufirst%3DB.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DGozani%26aufirst%3DO.%26aulast%3DCarlomagno%26aufirst%3DT.%26aulast%3DBenitah%26aufirst%3DS.%2BA.%26aulast%3DDi%2BCroce%26aufirst%3DL.%26atitle%3DPhf19%2520Links%2520Methylated%2520Lys36%2520of%2520Histone%2520H3%2520to%2520Regulation%2520of%2520Polycomb%2520Activity%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2012%26volume%3D19%26issue%3D12%26spage%3D1257%26epage%3D1265%26doi%3D10.1038%2Fnsmb.2434" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brien, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gambero, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jerman, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Egan, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunne, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jurgens, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wynne, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohan, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sinha, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loftus, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cagney, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bracken, A. P.</span></span> <span> </span><span class="NLM_article-title">Polycomb PHF19 Binds H3K36me3 and Recruits PRC2 and Demethylase NO66 to Embryonic Stem Cell Genes during Differentiation</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>19</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">1273</span>– <span class="NLM_lpage">1281</span>, <span class="refDoi"> DOI: 10.1038/nsmb.2449</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1038%2Fnsmb.2449" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=23160351" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1ymtr%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=1273-1281&issue=12&author=G.+L.+Brienauthor=G.+Gamberoauthor=D.+J.+O%E2%80%99Connellauthor=E.+Jermanauthor=S.+A.+Turnerauthor=C.+M.+Eganauthor=E.+J.+Dunneauthor=M.+C.+Jurgensauthor=K.+Wynneauthor=L.+Piaoauthor=A.+J.+Lohanauthor=N.+Fergusonauthor=X.+Shiauthor=K.+M.+Sinhaauthor=B.+J.+Loftusauthor=G.+Cagneyauthor=A.+P.+Bracken&title=Polycomb+PHF19+Binds+H3K36me3+and+Recruits+PRC2+and+Demethylase+NO66+to+Embryonic+Stem+Cell+Genes+during+Differentiation&doi=10.1038%2Fnsmb.2449"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Polycomb PHF19 binds H3K36me3 and recruits PRC2 and demethylase NO66 to embryonic stem cell genes during differentiation</span></div><div class="casAuthors">Brien, Gerard L.; Gambero, Guillermo; O'Connell, David J.; Jerman, Emilia; Turner, Siobhan A.; Egan, Chris M.; Dunne, Eiseart J.; Jurgens, Maike C.; Wynne, Kieran; Piao, Lianhua; Lohan, Amanda J.; Ferguson, Neil; Shi, Xiaobing; Sinha, Krishna M.; Loftus, Brendan J.; Cagney, Gerard; Bracken, Adrian P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1273-1281</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Polycomb group proteins are repressive chromatin modifiers with essential roles in metazoan development, cellular differentiation and cell fate maintenance.  How Polycomb proteins access active chromatin to confer transcriptional silencing during lineage transitions remains unclear.  Here we show that the Polycomb repressive complex 2 (PRC2) component PHF19 binds trimethylated histone H3 Lys36 (H3K36me3), a mark of active chromatin, via its Tudor domain.  PHF19 assocs. with the H3K36me3 demethylase NO66, and it is required to recruit the PRC2 complex and NO66 to stem cell genes during differentiation, leading to PRC2-mediated trimethylation of histone H3 Lys27 (H3K27), loss of H3K36me3 and transcriptional silencing.  We propose a model whereby PHF19 functions during mouse embryonic stem cell differentiation to transiently bind the H3K36me3 mark via its Tudor domain, forming essential contact points that allow recruitment of PRC2 and H3K36me3 demethylase activity to active gene loci during their transition to a Polycomb-repressed state.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5U3MbFaISUbVg90H21EOLACvtfcHk0lhNW-NGDKBGJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1ymtr%252FF&md5=7ae44af5f842916fb16c07361a5b5cef</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.2449&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.2449%26sid%3Dliteratum%253Aachs%26aulast%3DBrien%26aufirst%3DG.%2BL.%26aulast%3DGambero%26aufirst%3DG.%26aulast%3DO%25E2%2580%2599Connell%26aufirst%3DD.%2BJ.%26aulast%3DJerman%26aufirst%3DE.%26aulast%3DTurner%26aufirst%3DS.%2BA.%26aulast%3DEgan%26aufirst%3DC.%2BM.%26aulast%3DDunne%26aufirst%3DE.%2BJ.%26aulast%3DJurgens%26aufirst%3DM.%2BC.%26aulast%3DWynne%26aufirst%3DK.%26aulast%3DPiao%26aufirst%3DL.%26aulast%3DLohan%26aufirst%3DA.%2BJ.%26aulast%3DFerguson%26aufirst%3DN.%26aulast%3DShi%26aufirst%3DX.%26aulast%3DSinha%26aufirst%3DK.%2BM.%26aulast%3DLoftus%26aufirst%3DB.%2BJ.%26aulast%3DCagney%26aufirst%3DG.%26aulast%3DBracken%26aufirst%3DA.%2BP.%26atitle%3DPolycomb%2520PHF19%2520Binds%2520H3K36me3%2520and%2520Recruits%2520PRC2%2520and%2520Demethylase%2520NO66%2520to%2520Embryonic%2520Stem%2520Cell%2520Genes%2520during%2520Differentiation%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2012%26volume%3D19%26issue%3D12%26spage%3D1273%26epage%3D1281%26doi%3D10.1038%2Fnsmb.2449" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gatchalian, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kingsley, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moslet, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosas Ospina, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutateladze, T. G.</span></span> <span> </span><span class="NLM_article-title">An Aromatic Cage Is Required but Not Sufficient for Binding of Tudor Domains of the Polycomblike Protein Family to H3K36me3</span>. <i>Epigenetics</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">467</span>– <span class="NLM_lpage">473</span>, <span class="refDoi"> DOI: 10.1080/15592294.2015.1042646</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1080%2F15592294.2015.1042646" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=25923537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A280%3ADC%252BC2MjoslCmsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=467-473&issue=6&author=J.+Gatchalianauthor=M.+C.+Kingsleyauthor=S.+D.+Mosletauthor=R.+D.+Rosas+Ospinaauthor=T.+G.+Kutateladze&title=An+Aromatic+Cage+Is+Required+but+Not+Sufficient+for+Binding+of+Tudor+Domains+of+the+Polycomblike+Protein+Family+to+H3K36me3&doi=10.1080%2F15592294.2015.1042646"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">An aromatic cage is required but not sufficient for binding of Tudor domains of the Polycomblike protein family to H3K36me3</span></div><div class="casAuthors">Gatchalian Jovylyn; Kingsley Molly C; Moslet Stacey D; Rosas Ospina Ruben D; Kutateladze Tatiana G</div><div class="citationInfo"><span class="NLM_cas:title">Epigenetics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">467-73</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Polycomblike (Pcl) proteins are important transcriptional regulators and components of the Polycomb Repressive Complex 2 (PRC2).  The Tudor domains of human homologs PHF1 and PHF19 have been found to recognize trimethylated lysine 36 of histone H3 (H3K36me3); however, the biological role of Tudor domains of other Pcl proteins remains poorly understood.  Here, we characterize the molecular basis underlying histone binding activities of the Tudor domains of the Pcl family.  In contrast to a predominant view, we found that the methyl lysine-binding aromatic cage is necessary but not sufficient for recognition of H3K36me3 by these Tudor domains and that a hydrophobic patch, adjacent to the aromatic cage, is also required.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRqFf2GlRp7bRfKb6ELK8t0fW6udTcc2eYHxWDpHVIZqLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MjoslCmsw%253D%253D&md5=ba1a1d106047361fd65ccef09236ea37</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1080%2F15592294.2015.1042646&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F15592294.2015.1042646%26sid%3Dliteratum%253Aachs%26aulast%3DGatchalian%26aufirst%3DJ.%26aulast%3DKingsley%26aufirst%3DM.%2BC.%26aulast%3DMoslet%26aufirst%3DS.%2BD.%26aulast%3DRosas%2BOspina%26aufirst%3DR.%2BD.%26aulast%3DKutateladze%26aufirst%3DT.%2BG.%26atitle%3DAn%2520Aromatic%2520Cage%2520Is%2520Required%2520but%2520Not%2520Sufficient%2520for%2520Binding%2520of%2520Tudor%2520Domains%2520of%2520the%2520Polycomblike%2520Protein%2520Family%2520to%2520H3K36me3%26jtitle%3DEpigenetics%26date%3D2015%26volume%3D10%26issue%3D6%26spage%3D467%26epage%3D473%26doi%3D10.1080%2F15592294.2015.1042646" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nekrasov, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klymenko, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraterman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oktaba, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Köcher, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stunnenberg, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilm, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, J.</span></span> <span> </span><span class="NLM_article-title">Pcl-PRC2 Is Needed to Generate High Levels of H3-K27 Trimethylation at Polycomb Target Genes</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>26</i></span> (<span class="NLM_issue">18</span>),  <span class="NLM_fpage">4078</span>– <span class="NLM_lpage">4088</span>, <span class="refDoi"> DOI: 10.1038/sj.emboj.7601837</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1038%2Fsj.emboj.7601837" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=17762866" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVKhtrvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2007&pages=4078-4088&issue=18&author=M.+Nekrasovauthor=T.+Klymenkoauthor=S.+Fratermanauthor=B.+Pappauthor=K.+Oktabaauthor=T.+K%C3%B6cherauthor=A.+Cohenauthor=H.+G.+Stunnenbergauthor=M.+Wilmauthor=J.+M%C3%BCller&title=Pcl-PRC2+Is+Needed+to+Generate+High+Levels+of+H3-K27+Trimethylation+at+Polycomb+Target+Genes&doi=10.1038%2Fsj.emboj.7601837"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Pcl-PRC2 is needed to generate high levels of H3-K27 trimethylation at Polycomb target genes</span></div><div class="casAuthors">Nekrasov, Maxim; Klymenko, Tetyana; Fraterman, Sven; Papp, Bernadett; Oktaba, Katarzyna; Koecher, Thomas; Cohen, Adrian; Stunnenberg, Hendrik G.; Wilm, Matthias; Mueller, Juerg</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">4078-4088</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">PRC2 is thought to be the histone methyltransferase (HMTase) responsible for H3-K27 trimethylation at Polycomb target genes.  Here we report the biochem. purifn. and characterization of a distinct form of Drosophila PRC2 that contains the Polycomb group protein polycomblike (Pcl).  Like PRC2, Pcl-PRC2 is an H3-K27-specific HMTase that mono-, di- and trimethylates H3-K27 in nucleosomes in vitro.  Anal. of Drosophila mutants that lack Pcl unexpectedly reveals that Pcl-PRC2 is required to generate high levels of H3-K27 trimethylation at Polycomb target genes but is dispensable for the genome-wide H3-K27 mono- and dimethylation that is generated by PRC2.  In Pcl mutants, Polycomb target genes become derepressed even though H3-K27 trimethylation at these genes is only reduced and not abolished, and even though targeting of the Polycomb protein complexes PhoRC and PRC1 to Polycomb response elements is not affected.  Pcl-PRC2 is thus the HMTase that generates the high levels of H3-K27 trimethylation in Polycomb target genes that are needed to maintain a Polycomb-repressed chromatin state.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVZCMxfevbBLVg90H21EOLACvtfcHk0lhNW-NGDKBGJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVKhtrvJ&md5=75db8cee2dc169ccec48065b29d4380e</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fsj.emboj.7601837&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.emboj.7601837%26sid%3Dliteratum%253Aachs%26aulast%3DNekrasov%26aufirst%3DM.%26aulast%3DKlymenko%26aufirst%3DT.%26aulast%3DFraterman%26aufirst%3DS.%26aulast%3DPapp%26aufirst%3DB.%26aulast%3DOktaba%26aufirst%3DK.%26aulast%3DK%25C3%25B6cher%26aufirst%3DT.%26aulast%3DCohen%26aufirst%3DA.%26aulast%3DStunnenberg%26aufirst%3DH.%2BG.%26aulast%3DWilm%26aufirst%3DM.%26aulast%3DM%25C3%25BCller%26aufirst%3DJ.%26atitle%3DPcl-PRC2%2520Is%2520Needed%2520to%2520Generate%2520High%2520Levels%2520of%2520H3-K27%2520Trimethylation%2520at%2520Polycomb%2520Target%2520Genes%26jtitle%3DEMBO%2520J.%26date%3D2007%26volume%3D26%26issue%3D18%26spage%3D4078%26epage%3D4088%26doi%3D10.1038%2Fsj.emboj.7601837" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sarma, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Margueron, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ivanov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pirrotta, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinberg, D.</span></span> <span> </span><span class="NLM_article-title">Ezh2 Requires PHF1 To Efficiently Catalyze H3 Lysine 27 Trimethylation In Vivo</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>28</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">2718</span>– <span class="NLM_lpage">2731</span>, <span class="refDoi"> DOI: 10.1128/MCB.02017-07</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1128%2FMCB.02017-07" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=18285464" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BD1cXksFWqtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2008&pages=2718-2731&issue=8&author=K.+Sarmaauthor=R.+Margueronauthor=A.+Ivanovauthor=V.+Pirrottaauthor=D.+Reinberg&title=Ezh2+Requires+PHF1+To+Efficiently+Catalyze+H3+Lysine+27+Trimethylation+In+Vivo&doi=10.1128%2FMCB.02017-07"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Ezh2 requires PHF1 to efficiently catalyze H3 lysine 27 trimethylation in vivo</span></div><div class="casAuthors">Sarma, Kavitha; Margueron, Raphael; Ivanov, Alexey; Pirrotta, Vincenzo; Reinberg, Danny</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2718-2731</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The mammalian Polycomblike protein PHF1 was previously shown to interact with the Polycomb group (PcG) protein Ezh2, a histone methyltransferase whose activity is pivotal in sustaining gene repression during development and in adulthood.  As Ezh2 is active only when part of the Polycomb Repressive Complexes (PRC2-PRC4), we examd. the functional role of its interaction with PHF1.  Chromatin immunopptn. expts. revealed that PHF1 resides along with Ezh2 at Ezh2-regulated genes such as the HoxA loci and the non-Hox MYT1 and WNT1 genes.  Knockdown of PHF1 or of Ezh2 led to up-regulated HoxA gene expression.  Interestingly, depletion of PHF1 did correlate with reduced occupancy of Bmi-1, a PRC1 component.  As expected, knockdown of Ezh2 led to reduced levels of its catalytic products H3K27me2/H3K27me3.  However, reduced levels of PHF1 also led to decreased global levels of H3K27me3.  Notably, the levels of H3K27me3 decreased while those of H3K27me2 increased at the up-regulated HoxA loci tested.  Consistent with this, the addn. of PHF1 specifically stimulated the ability of Ezh2 to catalyze H3K27me3 but not H3K27me1/H3K27me2 in vitro.  We conclude that PHF1 modulates the activity of Ezh2 in favor of the repressive H3K27me3 mark.  Thus, we propose that PHF1 is a determinant in PcG-mediated gene repression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsiWF9xVC8TbVg90H21EOLACvtfcHk0lhNW-NGDKBGJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXksFWqtrc%253D&md5=ce25d19e315ef42f53c34f199e81cfcb</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1128%2FMCB.02017-07&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.02017-07%26sid%3Dliteratum%253Aachs%26aulast%3DSarma%26aufirst%3DK.%26aulast%3DMargueron%26aufirst%3DR.%26aulast%3DIvanov%26aufirst%3DA.%26aulast%3DPirrotta%26aufirst%3DV.%26aulast%3DReinberg%26aufirst%3DD.%26atitle%3DEzh2%2520Requires%2520PHF1%2520To%2520Efficiently%2520Catalyze%2520H3%2520Lysine%252027%2520Trimethylation%2520In%2520Vivo%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2008%26volume%3D28%26issue%3D8%26spage%3D2718%26epage%3D2731%26doi%3D10.1128%2FMCB.02017-07" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Choi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bachmann, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tauscher, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benda, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fierz, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, J.</span></span> <span> </span><span class="NLM_article-title">DNA Binding by PHF1 Prolongs PRC2 Residence Time on Chromatin and Thereby Promotes H3K27 Methylation</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>24</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">1039</span>– <span class="NLM_lpage">1047</span>, <span class="refDoi"> DOI: 10.1038/nsmb.3488</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1038%2Fnsmb.3488" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=29058710" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslehtrvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2017&pages=1039-1047&issue=12&author=J.+Choiauthor=A.+L.+Bachmannauthor=K.+Tauscherauthor=C.+Bendaauthor=B.+Fierzauthor=J.+M%C3%BCller&title=DNA+Binding+by+PHF1+Prolongs+PRC2+Residence+Time+on+Chromatin+and+Thereby+Promotes+H3K27+Methylation&doi=10.1038%2Fnsmb.3488"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">DNA binding by PHF1 prolongs PRC2 residence time on chromatin and thereby promotes H3K27 methylation</span></div><div class="casAuthors">Choi, Jeongyoon; Bachmann, Andreas Linus; Tauscher, Katharina; Benda, Christian; Fierz, Beat; Mueller, Juerg</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1039-1047</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Polycomb repressive complex 2 (PRC2) trimethylates histone H3 at lysine 27 to mark genes for repression.  We measured the dynamics of PRC2 binding on recombinant chromatin and free DNA at the single-mol. level using total internal reflection fluorescence (TIRF) microscopy.  PRC2 preferentially binds free DNA with multisecond residence time and midnanomolar affinity.  PHF1, a PRC2 accessory protein of the Polycomblike family, extends PRC2 residence time on DNA and chromatin.  Crystallog. and functional studies reveal that Polycomblike proteins contain a winged-helix domain that binds DNA in a sequence-nonspecific fashion.  DNA binding by this winged-helix domain accounts for the prolonged residence time of PHF1-PRC2 on chromatin and makes it a more efficient H3K27 methyltranferase than PRC2 alone.  Together, these studies establish that interactions with DNA provide the predominant binding affinity of PRC2 for chromatin.  Moreover, they reveal the mol. basis for how Polycomblike proteins stabilize PRC2 on chromatin and stimulate its activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_zfe3Wangt7Vg90H21EOLACvtfcHk0lgemJPnkWvYaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslehtrvK&md5=b58c890cba19c8aaf9f624cadf0f8139</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.3488&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.3488%26sid%3Dliteratum%253Aachs%26aulast%3DChoi%26aufirst%3DJ.%26aulast%3DBachmann%26aufirst%3DA.%2BL.%26aulast%3DTauscher%26aufirst%3DK.%26aulast%3DBenda%26aufirst%3DC.%26aulast%3DFierz%26aufirst%3DB.%26aulast%3DM%25C3%25BCller%26aufirst%3DJ.%26atitle%3DDNA%2520Binding%2520by%2520PHF1%2520Prolongs%2520PRC2%2520Residence%2520Time%2520on%2520Chromatin%2520and%2520Thereby%2520Promotes%2520H3K27%2520Methylation%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2017%26volume%3D24%26issue%3D12%26spage%3D1039%26epage%3D1047%26doi%3D10.1038%2Fnsmb.3488" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liefke, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurland, J. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bulyk, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span> <span> </span><span class="NLM_article-title">Polycomb-like Proteins Link the PRC2 Complex to CpG Islands</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>549</i></span> (<span class="NLM_issue">7671</span>),  <span class="NLM_fpage">287</span>– <span class="NLM_lpage">291</span>, <span class="refDoi"> DOI: 10.1038/nature23881</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1038%2Fnature23881" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=28869966" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVers77N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=549&publication_year=2017&pages=287-291&issue=7671&author=H.+Liauthor=R.+Liefkeauthor=J.+Jiangauthor=J.+V.+Kurlandauthor=W.+Tianauthor=P.+Dengauthor=W.+Zhangauthor=Q.+Heauthor=D.+J.+Patelauthor=M.+L.+Bulykauthor=Y.+Shiauthor=Z.+Wang&title=Polycomb-like+Proteins+Link+the+PRC2+Complex+to+CpG+Islands&doi=10.1038%2Fnature23881"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Polycomb-like proteins link the PRC2 complex to CpG islands</span></div><div class="casAuthors">Li, Haojie; Liefke, Robert; Jiang, Junyi; Kurland, Jesse Vigoda; Tian, Wei; Deng, Pujuan; Zhang, Weidi; He, Qian; Patel, Dinshaw J.; Bulyk, Martha L.; Shi, Yang; Wang, Zhanxin</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">549</span>
        (<span class="NLM_cas:issue">7671</span>),
    <span class="NLM_cas:pages">287-291</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The Polycomb repressive complex 2 (PRC2) mainly mediates transcriptional repression and has essential roles in various biol. processes including the maintenance of cell identity and proper differentiation.  Polycomb-like (PCL) proteins, such as PHF1, MTF2 and PHF19, are PRC2-assocd. factors that form sub-complexes with PRC2 core components, and have been proposed to modulate the enzymic activity of PRC2 or the recruitment of PRC2 to specific genomic loci.  Mammalian PRC2-binding sites are enriched in CG content, which correlates with CpG islands that display a low level of DNA methylation.  However, the mechanism of PRC2 recruitment to CpG islands is not fully understood.  Here we solve the crystal structures of the N-terminal domains of PHF1 and MTF2 with bound CpG-contg. DNAs in the presence of H3K36me3-contg. histone peptides.  We show that the extended homologous regions of both proteins fold into a winged-helix structure, which specifically binds to the unmethylated CpG motif but in a completely different manner from the canonical winged-helix DNA recognition motif.  We also show that the PCL extended homologous domains are required for efficient recruitment of PRC2 to CpG island-contg. promoters in mouse embryonic stem cells.  Our research provides the first, to our knowledge, direct evidence to demonstrate that PCL proteins are crucial for PRC2 recruitment to CpG islands, and further clarifies the roles of these proteins in transcriptional regulation in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLtwwW3WZV8bVg90H21EOLACvtfcHk0lgemJPnkWvYaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVers77N&md5=4ac008b821e5def7edad924d1c73ff8d</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fnature23881&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature23881%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DH.%26aulast%3DLiefke%26aufirst%3DR.%26aulast%3DJiang%26aufirst%3DJ.%26aulast%3DKurland%26aufirst%3DJ.%2BV.%26aulast%3DTian%26aufirst%3DW.%26aulast%3DDeng%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DHe%26aufirst%3DQ.%26aulast%3DPatel%26aufirst%3DD.%2BJ.%26aulast%3DBulyk%26aufirst%3DM.%2BL.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DZ.%26atitle%3DPolycomb-like%2520Proteins%2520Link%2520the%2520PRC2%2520Complex%2520to%2520CpG%2520Islands%26jtitle%3DNature%26date%3D2017%26volume%3D549%26issue%3D7671%26spage%3D287%26epage%3D291%26doi%3D10.1038%2Fnature23881" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">van
Mierlo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veenstra, G. J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vermeulen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marks, H.</span></span> <span> </span><span class="NLM_article-title">The Complexity of PRC2 Subcomplexes</span>. <i>Trends Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>29</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">660</span>– <span class="NLM_lpage">671</span>, <span class="refDoi"> DOI: 10.1016/j.tcb.2019.05.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1016%2Fj.tcb.2019.05.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=31178244" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtVKju7jJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=660-671&issue=8&author=G.+van%0AMierloauthor=G.+J.+C.+Veenstraauthor=M.+Vermeulenauthor=H.+Marks&title=The+Complexity+of+PRC2+Subcomplexes&doi=10.1016%2Fj.tcb.2019.05.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">The Complexity of PRC2 Subcomplexes</span></div><div class="casAuthors">van Mierlo, Guido; Veenstra, Gert Jan C.; Vermeulen, Michiel; Marks, Hendrik</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Cell Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">660-671</span>CODEN:
                <span class="NLM_cas:coden">TCBIEK</span>;
        ISSN:<span class="NLM_cas:issn">0962-8924</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Polycomb repressive complex 2 (PRC2) is a multisubunit protein complex essential for the development of multicellular organisms.  Recruitment of PRC2 to target genes, followed by deposition and propagation of its catalytic product histone H3 lysine 27 trimethylation (H3K27me3), are key to the spatiotemporal control of developmental gene expression.  Recent breakthrough studies have uncovered unexpected roles for substoichiometric PRC2 subunits in these processes.  Here, we elaborate on how the facultative PRC2 subunits regulate catalytic activity, locus-specific PRC2 binding, and propagation of H3K27me3, and how this affects chromatin structure, gene expression, and cell fate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSF3aGmH5JtbVg90H21EOLACvtfcHk0lgemJPnkWvYaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtVKju7jJ&md5=520c1f526a29c9a13a816c70f216a7a4</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.tcb.2019.05.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tcb.2019.05.004%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BMierlo%26aufirst%3DG.%26aulast%3DVeenstra%26aufirst%3DG.%2BJ.%2BC.%26aulast%3DVermeulen%26aufirst%3DM.%26aulast%3DMarks%26aufirst%3DH.%26atitle%3DThe%2520Complexity%2520of%2520PRC2%2520Subcomplexes%26jtitle%3DTrends%2520Cell%2520Biol.%26date%3D2019%26volume%3D29%26issue%3D8%26spage%3D660%26epage%3D671%26doi%3D10.1016%2Fj.tcb.2019.05.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Healy, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mucha, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glancy, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fitzpatrick, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conway, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neikes, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monger, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Mierlo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baltissen, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koseki, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vermeulen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koseki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bracken, A. P.</span></span> <span> </span><span class="NLM_article-title">PRC2.1 and PRC2.2 Synergize to Coordinate H3K27 Trimethylation</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>76</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">437</span>– <span class="NLM_lpage">452.e6</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2019.08.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1016%2Fj.molcel.2019.08.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=31521505" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhslOgs7zK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2019&pages=437-452.e6&issue=3&author=E.+Healyauthor=M.+Muchaauthor=E.+Glancyauthor=D.+J.+Fitzpatrickauthor=E.+Conwayauthor=H.+K.+Neikesauthor=C.+Mongerauthor=G.+Van+Mierloauthor=M.+P.+Baltissenauthor=Y.+Kosekiauthor=M.+Vermeulenauthor=H.+Kosekiauthor=A.+P.+Bracken&title=PRC2.1+and+PRC2.2+Synergize+to+Coordinate+H3K27+Trimethylation&doi=10.1016%2Fj.molcel.2019.08.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">PRC2.1 and PRC2.2 Synergize to Coordinate H3K27 Trimethylation</span></div><div class="casAuthors">Healy, Evan; Mucha, Marlena; Glancy, Eleanor; Fitzpatrick, Darren J.; Conway, Eric; Neikes, Hannah K.; Monger, Craig; Van Mierlo, Guido; Baltissen, Marijke P.; Koseki, Yoko; Vermeulen, Michiel; Koseki, Haruhiko; Bracken, Adrian P.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">437-452.e6</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Polycomb repressive complex 2 (PRC2) is composed of EED, SUZ12, and EZH1/2 and mediates mono-, di-, and trimethylation of histone H3 at lysine 27.  At least two independent subcomplexes exist, defined by their specific accessory proteins: PRC2.1 (PCL1-3, EPOP, and PALI1/2) and PRC2.2 (AEBP2 and JARID2).  We show that PRC2.1 and PRC2.2 share the majority of target genes in mouse embryonic stem cells.  The loss of PCL1-3 is sufficient to evict PRC2.1 from Polycomb target genes but only leads to a partial redn. of PRC2.2 and H3K27me3.  Conversely, disruption of PRC2.2 function through the loss of either JARID2 or RING1A/B is insufficient to completely disrupt targeting of SUZ12 by PCLs.  Instead, the combined loss of both PRC2.1 and PRC2.2 is required, leading to the global mislocalization of SUZ12.  This supports a model in which the specific accessory proteins within PRC2.1 and PRC2.2 cooperate to direct H3K27me3 via both synergistic and independent mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOH_8vEpojlbVg90H21EOLACvtfcHk0liAt3sAzzBoIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhslOgs7zK&md5=b61b34ab7fbb78caa0c8daea6d12e88c</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2019.08.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2019.08.012%26sid%3Dliteratum%253Aachs%26aulast%3DHealy%26aufirst%3DE.%26aulast%3DMucha%26aufirst%3DM.%26aulast%3DGlancy%26aufirst%3DE.%26aulast%3DFitzpatrick%26aufirst%3DD.%2BJ.%26aulast%3DConway%26aufirst%3DE.%26aulast%3DNeikes%26aufirst%3DH.%2BK.%26aulast%3DMonger%26aufirst%3DC.%26aulast%3DVan%2BMierlo%26aufirst%3DG.%26aulast%3DBaltissen%26aufirst%3DM.%2BP.%26aulast%3DKoseki%26aufirst%3DY.%26aulast%3DVermeulen%26aufirst%3DM.%26aulast%3DKoseki%26aufirst%3DH.%26aulast%3DBracken%26aufirst%3DA.%2BP.%26atitle%3DPRC2.1%2520and%2520PRC2.2%2520Synergize%2520to%2520Coordinate%2520H3K27%2520Trimethylation%26jtitle%3DMol.%2520Cell%26date%3D2019%26volume%3D76%26issue%3D3%26spage%3D437%26epage%3D452.e6%26doi%3D10.1016%2Fj.molcel.2019.08.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Perino, M.</span>; <span class="NLM_string-name">van Mierlo, G.</span>; <span class="NLM_string-name">Wardle, S. M. T.</span>; <span class="NLM_string-name">Marks, H.</span>; <span class="NLM_string-name">Veenstra, G. J. C.</span></span> <span> </span><span class="NLM_article-title">Two Distinct Functional Axes of Positive Feedback-Enforced PRC2 Recruitment in Mouse Embryonic Stem Cells</span>. <span class="NLM_year">2019</span>, bioRxiv 669960. biorxiv.org e-Print archive. <a href="https://www.biorxiv.org/content/10.1101/669960v1" class="extLink">https://www.biorxiv.org/content/10.1101/669960v1</a>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1101%2F669960" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=M.+Perino&author=G.+van+Mierlo&author=S.+M.+T.+Wardle&author=H.+Marks&author=G.+J.+C.+Veenstra&title=Two+Distinct+Functional+Axes+of+Positive+Feedback-Enforced+PRC2+Recruitment+in+Mouse+Embryonic+Stem+Cells"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1101%2F669960&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26id%3Ddoi%3A10.1101%252F669960%26sid%3Dliteratum%253Aachs%26aulast%3DPerino%26aufirst%3DM.%26atitle%3DTwo%2520Distinct%2520Functional%2520Axes%2520of%2520Positive%2520Feedback-Enforced%2520PRC2%2520Recruitment%2520in%2520Mouse%2520Embryonic%2520Stem%2520Cells%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Micci, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunetti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dal Cin, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nucci, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorunova, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panagopoulos, I.</span></span> <span> </span><span class="NLM_article-title">Fusion of the Genes BRD8 and PHF1 in Endometrial Stromal Sarcoma</span>. <i>Genes, Chromosomes Cancer</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">841</span>– <span class="NLM_lpage">845</span>, <span class="refDoi"> DOI: 10.1002/gcc.22485</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1002%2Fgcc.22485" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=28758277" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVaqsrrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=841-845&issue=12&author=F.+Micciauthor=M.+Brunettiauthor=P.+Dal+Cinauthor=M.+R.+Nucciauthor=L.+Gorunovaauthor=S.+Heimauthor=I.+Panagopoulos&title=Fusion+of+the+Genes+BRD8+and+PHF1+in+Endometrial+Stromal+Sarcoma&doi=10.1002%2Fgcc.22485"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Fusion of the genes BRD8 and PHF1 in endometrial stromal sarcoma</span></div><div class="casAuthors">Micci, Francesca; Brunetti, Marta; Dal Cin, Paola; Nucci, Marisa R.; Gorunova, Ludmila; Heim, Sverre; Panagopoulos, Ioannis</div><div class="citationInfo"><span class="NLM_cas:title">Genes, Chromosomes & Cancer</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">841-845</span>CODEN:
                <span class="NLM_cas:coden">GCCAES</span>;
        ISSN:<span class="NLM_cas:issn">1045-2257</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">We present a new endometrial stromal sarcoma (ESS)-assocd. genomic rearrangement involving chromosome arms 5p and 6p and leading to the formation of a BRD8-PHF1 fusion gene.  The PHF1 (PHD finger protein 1) gene, from 6p21, is known to be rearranged in ESS in a promiscuous way inasmuch as it has been shown to recombine with JAZF1, EPC1, MEAF6, and now also with BRD8, in tumors of this type.  In all rearrangements of PHF1, including the present one, a recurrent theme is that the entire coding part of PHF1 constitutes the 3' end of the fusion.  BRD8 (bromodomain contg. 8) encodes a protein which is involved in regulation of protein acetylation and/or histone acetyl transferase activity.  All the genetic fusions identified so far in ESS appear to recombine genes involved in transcriptional regulation, i.e., polycomb group complex-mediated and aberrant methylation/acetylation genes.  This adds to the likelihood that the new BRD8-PHF1 shares the same pathogenetic mechanism as the other ESS-specific rearrangements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNmV1SZ_bWx7Vg90H21EOLACvtfcHk0liAt3sAzzBoIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVaqsrrI&md5=2c516c6966038de383f9cc4005576aa5</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1002%2Fgcc.22485&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fgcc.22485%26sid%3Dliteratum%253Aachs%26aulast%3DMicci%26aufirst%3DF.%26aulast%3DBrunetti%26aufirst%3DM.%26aulast%3DDal%2BCin%26aufirst%3DP.%26aulast%3DNucci%26aufirst%3DM.%2BR.%26aulast%3DGorunova%26aufirst%3DL.%26aulast%3DHeim%26aufirst%3DS.%26aulast%3DPanagopoulos%26aufirst%3DI.%26atitle%3DFusion%2520of%2520the%2520Genes%2520BRD8%2520and%2520PHF1%2520in%2520Endometrial%2520Stromal%2520Sarcoma%26jtitle%3DGenes%252C%2520Chromosomes%2520Cancer%26date%3D2017%26volume%3D56%26issue%3D12%26spage%3D841%26epage%3D845%26doi%3D10.1002%2Fgcc.22485" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Panagopoulos, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Micci, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thorsen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorunova, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eibak, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bjerkehagen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davidson, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heim, S.</span></span> <span> </span><span class="NLM_article-title">Novel Fusion of MYST/Esa1-Associated Factor 6 and PHF1 in Endometrial Stromal Sarcoma</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">e39354</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0039354</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1371%2Fjournal.pone.0039354" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=22761769" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC38XpvVWltrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=e39354&issue=6&author=I.+Panagopoulosauthor=F.+Micciauthor=J.+Thorsenauthor=L.+Gorunovaauthor=A.+M.+Eibakauthor=B.+Bjerkehagenauthor=B.+Davidsonauthor=S.+Heim&title=Novel+Fusion+of+MYST%2FEsa1-Associated+Factor+6+and+PHF1+in+Endometrial+Stromal+Sarcoma&doi=10.1371%2Fjournal.pone.0039354"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Novel fusion of MYST/Esa1-associated factor 6 and PHF1 in endometrial stromal sarcoma</span></div><div class="casAuthors">Panagopoulos, Ioannis; Micci, Francesca; Thorsen, Jim; Gorunova, Ludmila; Eibak, Anne Mette; Bjerkehagen, Bodil; Davidson, Ben; Heim, Sverre</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">e39354</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Rearrangement of chromosome band 6p21 is recurrent in endometrial stromal sarcoma (ESS) and targets the PHF1 gene.  So far, PHF1 was found to be the 3' partner in the JAZF1-PHF1 and EPC1-PHF1 chimeras but since the 6p21 rearrangements involve also other chromosomal translocation partners, other PHF1-fusions seem likely.  Here, we show that PHF1 is recombined with a novel fusion partner, MEAF6 from 1p34, in an ESS carrying a t(1;6)(p34;p21) translocation as the sole karyotypic anomaly.  5'-RACE, RT-PCR, and sequencing showed the presence of an MEAF6-PHF1 chimera in the tumor with exon 5 of MEAF6 being fused in-frame to exon 2 of PHF1 so that the entire PHF1 coding region becomes the 3' terminal part of the MEAF6-PHF1 fusion.  The predicted fusion protein is composed of 750 amino acids and contains the histone acetyltransferase subunit NuA4 domain of MEAF6 and the tudor, PHD zinc finger, and MTF2 domains of PHF1.  Although the specific functions of the MEAF6 and PHF1 proteins and why they are targeted by a neoplasia-specific gene fusion are not directly apparent, it seems that rearrangement of genes involved in acetylation (EPC1, MEAF6) and methylation (PHF1), resulting in aberrant gene expression, is a common theme in ESS pathogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkx5ulZ__TY7Vg90H21EOLACvtfcHk0liAt3sAzzBoIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XpvVWltrg%253D&md5=d4984aef237ec87a0b1d2064dda4a4ba</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0039354&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0039354%26sid%3Dliteratum%253Aachs%26aulast%3DPanagopoulos%26aufirst%3DI.%26aulast%3DMicci%26aufirst%3DF.%26aulast%3DThorsen%26aufirst%3DJ.%26aulast%3DGorunova%26aufirst%3DL.%26aulast%3DEibak%26aufirst%3DA.%2BM.%26aulast%3DBjerkehagen%26aufirst%3DB.%26aulast%3DDavidson%26aufirst%3DB.%26aulast%3DHeim%26aufirst%3DS.%26atitle%3DNovel%2520Fusion%2520of%2520MYST%252FEsa1-Associated%2520Factor%25206%2520and%2520PHF1%2520in%2520Endometrial%2520Stromal%2520Sarcoma%26jtitle%3DPLoS%2520One%26date%3D2012%26volume%3D7%26issue%3D6%26spage%3De39354%26doi%3D10.1371%2Fjournal.pone.0039354" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gebre-Medhin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nord, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Möller, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mandahl, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnusson, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nilsson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jo, V. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vult Von Steyern, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brosjö, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsson, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Domanski, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sciot, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Debiec-Rychter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fletcher, C. D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mertens, F.</span></span> <span> </span><span class="NLM_article-title">Recurrent Rearrangement of the PHF1 Gene in Ossifying Fibromyxoid Tumors</span>. <i>Am. J. Pathol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>181</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">1069</span>– <span class="NLM_lpage">1077</span>, <span class="refDoi"> DOI: 10.1016/j.ajpath.2012.05.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1016%2Fj.ajpath.2012.05.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=22796436" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFSns7nJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=181&publication_year=2012&pages=1069-1077&issue=3&author=S.+Gebre-Medhinauthor=K.+H.+Nordauthor=E.+M%C3%B6llerauthor=N.+Mandahlauthor=L.+Magnussonauthor=J.+Nilssonauthor=V.+Y.+Joauthor=F.+Vult+Von+Steyernauthor=O.+Brosj%C3%B6author=O.+Larssonauthor=H.+A.+Domanskiauthor=R.+Sciotauthor=M.+Debiec-Rychterauthor=C.+D.+M.+Fletcherauthor=F.+Mertens&title=Recurrent+Rearrangement+of+the+PHF1+Gene+in+Ossifying+Fibromyxoid+Tumors&doi=10.1016%2Fj.ajpath.2012.05.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Recurrent rearrangement of the PHF1 gene in ossifying fibromyxoid tumors</span></div><div class="casAuthors">Gebre-Medhin, Samuel; Nord, Karolin H.; Moller, Emely; Mandahl, Nils; Magnusson, Linda; Nilsson, Jenny; Jo, Vickie Y.; Vult von Steyern, Fredrik; Brosjo, Otte; Larsson, Olle; Domanski, Henryk A.; Sciot, Raf; Debiec-Rychter, Maria; Fletcher, Christopher D. M.; Mertens, Fredrik</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Pathology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">181</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1069-1077</span>CODEN:
                <span class="NLM_cas:coden">AJPAA4</span>;
        ISSN:<span class="NLM_cas:issn">0002-9440</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Ossifying fibromyxoid tumor (OFMT) is a soft tissue tumor of unknown lineage.  Although most cases are histol. and clin. benign, some show malignant morphol. features and local recurrences are not uncommon; a few may even metastasize.  In the present study, cytogenetic anal. identified different structural rearrangements of chromosome band 6p21 in tumor cells from three cases of OFMT, including one with typical, one with atypical, and one with malignant morphol. features.  Mapping of the 6p21 breakpoint by fluorescence in situ hybridization (FISH) indicated that the PHF1 gene was rearranged in all three cases.  Further FISH, 5'-rapid amplification of cDNA ends, and RT-PCR analyses disclosed an EP400/PHF1 fusion transcript in one of the cases.  Interphase FISH on tumor sections from 13 addnl. cases of OFMT showed rearrangement of the PHF1 locus in four of four typical, two of three atypical, and one of six malignant lesions.  Thus, the PHF1 gene, previously shown to be the 3'-partner of fusion genes in endometrial stromal tumors, is also recurrently involved in the pathogenesis of OFMTs, irresp. of whether they are diagnosed as typical, atypical, or malignant lesions.  The PHF1 protein interacts with the polycomb-repressive complex 2 (PRC2), which, in turn, regulates the expression of a variety of developmental genes.  Thus, the results indicate that deregulation of PRC2 target genes is crucial for OFMT development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOKDxTyWmFHrVg90H21EOLACvtfcHk0ljPqrBTUA5zeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFSns7nJ&md5=798b60bce3f6e100b9420fd07da9aa1b</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.ajpath.2012.05.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ajpath.2012.05.030%26sid%3Dliteratum%253Aachs%26aulast%3DGebre-Medhin%26aufirst%3DS.%26aulast%3DNord%26aufirst%3DK.%2BH.%26aulast%3DM%25C3%25B6ller%26aufirst%3DE.%26aulast%3DMandahl%26aufirst%3DN.%26aulast%3DMagnusson%26aufirst%3DL.%26aulast%3DNilsson%26aufirst%3DJ.%26aulast%3DJo%26aufirst%3DV.%2BY.%26aulast%3DVult%2BVon%2BSteyern%26aufirst%3DF.%26aulast%3DBrosj%25C3%25B6%26aufirst%3DO.%26aulast%3DLarsson%26aufirst%3DO.%26aulast%3DDomanski%26aufirst%3DH.%2BA.%26aulast%3DSciot%26aufirst%3DR.%26aulast%3DDebiec-Rychter%26aufirst%3DM.%26aulast%3DFletcher%26aufirst%3DC.%2BD.%2BM.%26aulast%3DMertens%26aufirst%3DF.%26atitle%3DRecurrent%2520Rearrangement%2520of%2520the%2520PHF1%2520Gene%2520in%2520Ossifying%2520Fibromyxoid%2520Tumors%26jtitle%3DAm.%2520J.%2520Pathol.%26date%3D2012%26volume%3D181%26issue%3D3%26spage%3D1069%26epage%3D1077%26doi%3D10.1016%2Fj.ajpath.2012.05.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Endo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohashi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishii, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshida, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsunobu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwamoto, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oda, Y.</span></span> <span> </span><span class="NLM_article-title">Ossifying Fibromyxoid Tumor Presenting EP400-PHF1 Fusion Gene</span>. <i>Hum. Pathol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>44</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">2603</span>– <span class="NLM_lpage">2608</span>, <span class="refDoi"> DOI: 10.1016/j.humpath.2013.04.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1016%2Fj.humpath.2013.04.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=23806526" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVSltr3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2013&pages=2603-2608&issue=11&author=M.+Endoauthor=K.+Kohashiauthor=H.+Yamamotoauthor=T.+Ishiiauthor=T.+Yoshidaauthor=T.+Matsunobuauthor=Y.+Iwamotoauthor=Y.+Oda&title=Ossifying+Fibromyxoid+Tumor+Presenting+EP400-PHF1+Fusion+Gene&doi=10.1016%2Fj.humpath.2013.04.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Ossifying fibromyxoid tumor presenting EP400-PHF1 fusion gene</span></div><div class="casAuthors">Endo, Makoto; Kohashi, Kenichi; Yamamoto, Hidetaka; Ishii, Takeaki; Yoshida, Tatsuya; Matsunobu, Tomoya; Iwamoto, Yukihide; Oda, Yoshinao</div><div class="citationInfo"><span class="NLM_cas:title">Human Pathology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2603-2608</span>CODEN:
                <span class="NLM_cas:coden">HPCQA4</span>;
        ISSN:<span class="NLM_cas:issn">0046-8177</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Ossifying fibromyxoid tumor is a rare soft tissue tumor of borderline malignancy and uncertain differentiation.  Recently, a novel fusion gene, EP400-PHF1, was discovered in ossifying fibromyxoid tumor; however, its relation to this type of tumor has been uncertain because the EP400-PHF1 fusion gene has been successfully detected in only 1 case.  We present an ossifying fibromyxoid tumor case with the EP400-PHF1 fusion gene detected by reverse transcriptase polymerase chain reaction, along with compatible cytogenetic data showing a t(6;12)(p21;q24.3) translocation.  Our results suggest that the EP400-PHF1 fusion gene is a reproducible finding in ossifying fibromyxoid tumor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoo6yctwzQ5cLVg90H21EOLACvtfcHk0ljPqrBTUA5zeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVSltr3E&md5=e5feac80be06543b8d4e70d55d7fd12b</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.humpath.2013.04.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.humpath.2013.04.003%26sid%3Dliteratum%253Aachs%26aulast%3DEndo%26aufirst%3DM.%26aulast%3DKohashi%26aufirst%3DK.%26aulast%3DYamamoto%26aufirst%3DH.%26aulast%3DIshii%26aufirst%3DT.%26aulast%3DYoshida%26aufirst%3DT.%26aulast%3DMatsunobu%26aufirst%3DT.%26aulast%3DIwamoto%26aufirst%3DY.%26aulast%3DOda%26aufirst%3DY.%26atitle%3DOssifying%2520Fibromyxoid%2520Tumor%2520Presenting%2520EP400-PHF1%2520Fusion%2520Gene%26jtitle%3DHum.%2520Pathol.%26date%3D2013%26volume%3D44%26issue%3D11%26spage%3D2603%26epage%3D2608%26doi%3D10.1016%2Fj.humpath.2013.04.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shokat, K. M.</span></span> <span> </span><span class="NLM_article-title">Recognizing and Exploiting Differences between RNAi and Small-Molecule Inhibitors</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">739</span>– <span class="NLM_lpage">744</span>, <span class="refDoi"> DOI: 10.1038/nchembio1207-739</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1038%2Fnchembio1207-739" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=18007642" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlWqurjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2007&pages=739-744&author=W.+A.+Weissauthor=S.+S.+Taylorauthor=K.+M.+Shokat&title=Recognizing+and+Exploiting+Differences+between+RNAi+and+Small-Molecule+Inhibitors&doi=10.1038%2Fnchembio1207-739"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Recognizing and exploiting differences between RNAi and small-molecule inhibitors</span></div><div class="casAuthors">Weiss, William A.; Taylor, Stephen S.; Shokat, Kevan M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">739-744</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The biol. of RNA interference has greatly facilitated anal. of loss-of-function phenotypes, but correlating these phenotypes with small-mol. inhibition profiles is not always straightforward.  The authors examine the rationale of comparing RNA interference to pharmacol. intervention in chem. biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9JPzqVyshS7Vg90H21EOLACvtfcHk0ljPqrBTUA5zeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlWqurjO&md5=9a26c990b358a6b2c717d1e361cac2da</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1038%2Fnchembio1207-739&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio1207-739%26sid%3Dliteratum%253Aachs%26aulast%3DWeiss%26aufirst%3DW.%2BA.%26aulast%3DTaylor%26aufirst%3DS.%2BS.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26atitle%3DRecognizing%2520and%2520Exploiting%2520Differences%2520between%2520RNAi%2520and%2520Small-Molecule%2520Inhibitors%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2007%26volume%3D3%26spage%3D739%26epage%3D744%26doi%3D10.1038%2Fnchembio1207-739" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kycia, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kudithipudi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamas, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kungulovski, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhayalan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeltsch, A.</span></span> <span> </span><span class="NLM_article-title">The Tudor Domain of the PHD Finger Protein 1 Is a Dual Reader of Lysine Trimethylation at Lysine 36 of Histone H3 and Lysine 27 of Histone Variant H3t</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>426</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">1651</span>– <span class="NLM_lpage">1660</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2013.08.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1016%2Fj.jmb.2013.08.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=23954330" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVSksrnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=426&publication_year=2014&pages=1651-1660&issue=8&author=I.+Kyciaauthor=S.+Kudithipudiauthor=R.+Tamasauthor=G.+Kungulovskiauthor=A.+Dhayalanauthor=A.+Jeltsch&title=The+Tudor+Domain+of+the+PHD+Finger+Protein+1+Is+a+Dual+Reader+of+Lysine+Trimethylation+at+Lysine+36+of+Histone+H3+and+Lysine+27+of+Histone+Variant+H3t&doi=10.1016%2Fj.jmb.2013.08.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">The Tudor Domain of the PHD Finger Protein 1 Is a Dual Reader of Lysine Trimethylation at Lysine 36 of Histone H3 and Lysine 27 of Histone Variant H3t</span></div><div class="casAuthors">Kycia, Ina; Kudithipudi, Srikanth; Tamas, Raluca; Kungulovski, Goran; Dhayalan, Arunkumar; Jeltsch, Albert</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">426</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1651-1660</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">PHF1 assocs. with the Polycomb repressive complex 2 and it was demonstrated to stimulate its H3K27-trimethylation activity.  We studied the interaction of the PHF1 Tudor domain with modified histone peptides and found that it recognizes H3K36me3 and H3tK27me3 (on the histone variant H3t) and that it uses the same trimethyllysine binding pocket for the interaction with both peptides.  Since both peptide sequences are very different, this result indicates that reading domains can have dual specificities.  Sub-nuclear localization studies of full-length PHF1 in human HEK293 cells revealed that it co-localizes with K27me3, but not with K36me3, and that this co-localization depends on the trimethyllysine binding pocket indicating that K27me3 is an in vivo target for the PHF1 Tudor domain.  Our data suggest that PHF1 binds to H3tK27me3 in human chromatin, and H3t has a more general role in Polycomb regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq85AlKyIAm3LVg90H21EOLACvtfcHk0lhp65X4ZkSCOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVSksrnK&md5=c7e974212da3bf9f929f3ce9726f7b88</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2013.08.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2013.08.009%26sid%3Dliteratum%253Aachs%26aulast%3DKycia%26aufirst%3DI.%26aulast%3DKudithipudi%26aufirst%3DS.%26aulast%3DTamas%26aufirst%3DR.%26aulast%3DKungulovski%26aufirst%3DG.%26aulast%3DDhayalan%26aufirst%3DA.%26aulast%3DJeltsch%26aufirst%3DA.%26atitle%3DThe%2520Tudor%2520Domain%2520of%2520the%2520PHD%2520Finger%2520Protein%25201%2520Is%2520a%2520Dual%2520Reader%2520of%2520Lysine%2520Trimethylation%2520at%2520Lysine%252036%2520of%2520Histone%2520H3%2520and%2520Lysine%252027%2520of%2520Histone%2520Variant%2520H3t%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2014%26volume%3D426%26issue%3D8%26spage%3D1651%26epage%3D1660%26doi%3D10.1016%2Fj.jmb.2013.08.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dong, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakagawa, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oyama, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshimura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamiya, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakayama, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueda, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, J.</span></span> <span> </span><span class="NLM_article-title">Structural Basis for Histone Variant H3tk27me3 Recognition by Phf1 and Phf19</span>. <i>eLife</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">21</span>, <span class="refDoi"> DOI: 10.7554/eLife.58675</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.7554%2FeLife.58675" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2020&pages=1-21&author=C.+Dongauthor=R.+Nakagawaauthor=K.+Oyamaauthor=Y.+Yamamotoauthor=W.+Zhangauthor=A.+Dongauthor=Y.+Liauthor=Y.+Yoshimuraauthor=H.+Kamiyaauthor=J.+I.+Nakayamaauthor=J.+Uedaauthor=J.+Min&title=Structural+Basis+for+Histone+Variant+H3tk27me3+Recognition+by+Phf1+and+Phf19&doi=10.7554%2FeLife.58675"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.7554%2FeLife.58675&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.58675%26sid%3Dliteratum%253Aachs%26aulast%3DDong%26aufirst%3DC.%26aulast%3DNakagawa%26aufirst%3DR.%26aulast%3DOyama%26aufirst%3DK.%26aulast%3DYamamoto%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DYoshimura%26aufirst%3DY.%26aulast%3DKamiya%26aufirst%3DH.%26aulast%3DNakayama%26aufirst%3DJ.%2BI.%26aulast%3DUeda%26aufirst%3DJ.%26aulast%3DMin%26aufirst%3DJ.%26atitle%3DStructural%2520Basis%2520for%2520Histone%2520Variant%2520H3tk27me3%2520Recognition%2520by%2520Phf1%2520and%2520Phf19%26jtitle%3DeLife%26date%3D2020%26volume%3D9%26spage%3D1%26epage%3D21%26doi%3D10.7554%2FeLife.58675" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Simhadri, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daze, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Douglas, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quon, T. T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gignac, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heller, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boulanger, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wulff, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hof, F.</span></span> <span> </span><span class="NLM_article-title">Chromodomain Antagonists That Target the Polycomb-Group Methyllysine Reader Protein Chromobox Homolog 7 (CBX7)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">2874</span>– <span class="NLM_lpage">2883</span>, <span class="refDoi"> DOI: 10.1021/jm401487x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401487x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC2cXktlenu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=2874-2883&issue=7&author=C.+Simhadriauthor=K.+D.+Dazeauthor=S.+F.+Douglasauthor=T.+T.+H.+Quonauthor=A.+Devauthor=M.+C.+Gignacauthor=F.+Pengauthor=M.+Hellerauthor=M.+J.+Boulangerauthor=J.+E.+Wulffauthor=F.+Hof&title=Chromodomain+Antagonists+That+Target+the+Polycomb-Group+Methyllysine+Reader+Protein+Chromobox+Homolog+7+%28CBX7%29&doi=10.1021%2Fjm401487x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Chromodomain Antagonists That Target the Polycomb-Group Methyllysine Reader Protein Chromobox Homolog 7 (CBX7)</span></div><div class="casAuthors">Simhadri, Chakravarthi; Daze, Kevin D.; Douglas, Sarah F.; Quon, Taylor T. H.; Dev, Amarjot; Gignac, Michael C.; Peng, Fangni; Heller, Markus; Boulanger, Martin J.; Wulff, Jeremy E.; Hof, Fraser</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2874-2883</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors report a peptide-driven approach to create the first inhibitors of the chromobox homolog 7 (CBX7), a methyllysine reader protein.  CBX7 uses its chromodomain to bind histone 3, lysine 27 trimethylated (H3K27me3), and this recognition event is implicated in silencing multiple tumor suppressors.  Small trimethyllysine contg.-peptides were used as the basic scaffold from which potent ligands for disruption of CBX7-H3K27me3 complex were developed.  Potency of ligands was detd. by fluorescence polarization and/or isothermal titrn. calorimetry.  Binding of one ligand was characterized in detail using 2D NMR and x-ray crystallog., revealing a structural motif unique among human CBX proteins.  Inhibitors with a ∼200 nM potency for CBX7 binding and 10-fold/400-fold selectivity over related CBX8/CBX1 proteins were identified.  These are the first reported inhibitors of any chromodomain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgJsmWqbn71bVg90H21EOLACvtfcHk0lhp65X4ZkSCOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXktlenu7s%253D&md5=d3f5c56c736a6de5cbb742755e3ec110</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fjm401487x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401487x%26sid%3Dliteratum%253Aachs%26aulast%3DSimhadri%26aufirst%3DC.%26aulast%3DDaze%26aufirst%3DK.%2BD.%26aulast%3DDouglas%26aufirst%3DS.%2BF.%26aulast%3DQuon%26aufirst%3DT.%2BT.%2BH.%26aulast%3DDev%26aufirst%3DA.%26aulast%3DGignac%26aufirst%3DM.%2BC.%26aulast%3DPeng%26aufirst%3DF.%26aulast%3DHeller%26aufirst%3DM.%26aulast%3DBoulanger%26aufirst%3DM.%2BJ.%26aulast%3DWulff%26aufirst%3DJ.%2BE.%26aulast%3DHof%26aufirst%3DF.%26atitle%3DChromodomain%2520Antagonists%2520That%2520Target%2520the%2520Polycomb-Group%2520Methyllysine%2520Reader%2520Protein%2520Chromobox%2520Homolog%25207%2520%2528CBX7%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26issue%3D7%26spage%3D2874%26epage%3D2883%26doi%3D10.1021%2Fjm401487x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Milosevich, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gignac, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McFarlane, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simhadri, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horvath, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daze, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Croft, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dheri, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quon, T. T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Douglas, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wulff, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paci, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hof, F.</span></span> <span> </span><span class="NLM_article-title">Selective Inhibition of CBX6: A Methyllysine Reader Protein in the Polycomb Family</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">139</span>– <span class="NLM_lpage">144</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.5b00378</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00378" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFKjtb%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=139-144&issue=2&author=N.+Milosevichauthor=M.+C.+Gignacauthor=J.+McFarlaneauthor=C.+Simhadriauthor=S.+Horvathauthor=K.+D.+Dazeauthor=C.+S.+Croftauthor=A.+Dheriauthor=T.+T.+H.+Quonauthor=S.+F.+Douglasauthor=J.+E.+Wulffauthor=I.+Paciauthor=F.+Hof&title=Selective+Inhibition+of+CBX6%3A+A+Methyllysine+Reader+Protein+in+the+Polycomb+Family&doi=10.1021%2Facsmedchemlett.5b00378"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Inhibition of CBX6: A Methyllysine Reader Protein in the Polycomb Family</span></div><div class="casAuthors">Milosevich, Natalia; Gignac, Michael C.; McFarlane, James; Simhadri, Chakravarthi; Horvath, Shanti; Daze, Kevin D.; Croft, Caitlin S.; Dheri, Aman; Quon, Taylor T. H.; Douglas, Sarah F.; Wulff, Jeremy E.; Paci, Irina; Hof, Fraser</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">139-144</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The polycomb paralogs CBX2, CBX4, CBX6, CBX7, and CBX8 are epigenetic readers that rely on "arom. cage" motifs to engage their partners' methyllysine side chains.  Each CBX carries out distinct functions, yet each includes a highly similar methyllysine-reading chromodomain as a key element.  CBX7 is the only chromodomain that has yet been targeted by chem. inhibition.  We report a small set of peptidomimetic agents in which a simple chem. modification switches the ligands from one with promiscuity across all polycomb paralogs to one that provides selective inhibition of CBX6.  The structural basis for this selectivity, which involves occupancy of a small hydrophobic pocket adjacent to the arom. cage, was confirmed through mol. dynamics simulations.  Our results demonstrate the increases in affinity and selectivity generated by ligands that engage extended regions of chromodomain binding surfaces.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqO9JIy3HQmrrVg90H21EOLACvtfcHk0lhp65X4ZkSCOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFKjtb%252FP&md5=905994e530d26186495e94b923279d1f</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00378&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00378%26sid%3Dliteratum%253Aachs%26aulast%3DMilosevich%26aufirst%3DN.%26aulast%3DGignac%26aufirst%3DM.%2BC.%26aulast%3DMcFarlane%26aufirst%3DJ.%26aulast%3DSimhadri%26aufirst%3DC.%26aulast%3DHorvath%26aufirst%3DS.%26aulast%3DDaze%26aufirst%3DK.%2BD.%26aulast%3DCroft%26aufirst%3DC.%2BS.%26aulast%3DDheri%26aufirst%3DA.%26aulast%3DQuon%26aufirst%3DT.%2BT.%2BH.%26aulast%3DDouglas%26aufirst%3DS.%2BF.%26aulast%3DWulff%26aufirst%3DJ.%2BE.%26aulast%3DPaci%26aufirst%3DI.%26aulast%3DHof%26aufirst%3DF.%26atitle%3DSelective%2520Inhibition%2520of%2520CBX6%253A%2520A%2520Methyllysine%2520Reader%2520Protein%2520in%2520the%2520Polycomb%2520Family%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D7%26issue%3D2%26spage%3D139%26epage%3D144%26doi%3D10.1021%2Facsmedchemlett.5b00378" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denton, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hobbs, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weaver, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McFarlane, J. M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Connelly, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gignac, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milosevich, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hof, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paci, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Musselman, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dykhuizen, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krusemark, C. J.</span></span> <span> </span><span class="NLM_article-title">Optimization of Ligands Using Focused DNA-Encoded Libraries to Develop a Selective, Cell-Permeable CBX8 Chromodomain Inhibitor</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>15</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">112</span>– <span class="NLM_lpage">131</span>, <span class="refDoi"> DOI: 10.1021/acschembio.9b00654</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.9b00654" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1WktL7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2020&pages=112-131&issue=1&author=S.+Wangauthor=K.+E.+Dentonauthor=K.+F.+Hobbsauthor=T.+Weaverauthor=J.+M.+B.+McFarlaneauthor=K.+E.+Connellyauthor=M.+C.+Gignacauthor=N.+Milosevichauthor=F.+Hofauthor=I.+Paciauthor=C.+A.+Musselmanauthor=E.+C.+Dykhuizenauthor=C.+J.+Krusemark&title=Optimization+of+Ligands+Using+Focused+DNA-Encoded+Libraries+to+Develop+a+Selective%2C+Cell-Permeable+CBX8+Chromodomain+Inhibitor&doi=10.1021%2Facschembio.9b00654"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of Ligands Using Focused DNA-Encoded Libraries To Develop a Selective, Cell-Permeable CBX8 Chromodomain Inhibitor</span></div><div class="casAuthors">Wang, Sijie; Denton, Kyle E.; Hobbs, Kathryn F.; Weaver, Tyler; McFarlane, James M. B.; Connelly, Katelyn E.; Gignac, Michael C.; Milosevich, Natalia; Hof, Fraser; Paci, Irina; Musselman, Catherine A.; Dykhuizen, Emily C.; Krusemark, Casey J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">112-131</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Polycomb repressive complex 1 (PRC1) is crit. for mediating gene expression during development.  Five chromobox (CBX) homolog proteins, CBX2, CBX4, CBX6, CBX7, and CBX8, are incorporated into PRC1 complexes, where they mediate targeting to trimethylated lysine 27 of histone H3 (H3K27me3) via the N-terminal chromodomain (ChD).  Individual CBX paralogs have been implicated as drug targets in cancer; however, high similarities in sequence and structure among the CBX ChDs provide a major obstacle in developing selective CBX ChD inhibitors.  Here we report the selection of small, focused, DNA-encoded libraries (DELs) against multiple homologous ChDs to identify modifications to a parental ligand that confer both selectivity and potency for the ChD of CBX8.  This on-DNA, medicinal chem. approach enabled the development of SW2_110A, a selective, cell-permeable inhibitor of the CBX8 ChD.  SW2_110A binds CBX8 ChD with a Kd of 800 nM, with minimal 5-fold selectivity for CBX8 ChD over all other CBX paralogs in vitro.  SW2_110A specifically inhibits the assocn. of CBX8 with chromatin in cells and inhibits the proliferation of THP1 leukemia cells driven by the MLL-AF9 translocation.  In THP1 cells, SW2_110A treatment results in a significant decrease in the expression of MLL-AF9 target genes, including HOXA9, validating the previously established role for CBX8 in MLL-AF9 transcriptional activation, and defining the ChD as necessary for this function.  The success of SW2_110A provides great promise for the development of highly selective and cell-permeable probes for the full CBX family.  In addn., the approach taken provides a proof-of-principle demonstration of how DELs can be used iteratively for optimization of both ligand potency and selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpG4CHEZDuLrLVg90H21EOLACvtfcHk0lhOPTYdNLqRiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1WktL7O&md5=69aa429f81136431674a6c704a7ab388</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Facschembio.9b00654&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.9b00654%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DS.%26aulast%3DDenton%26aufirst%3DK.%2BE.%26aulast%3DHobbs%26aufirst%3DK.%2BF.%26aulast%3DWeaver%26aufirst%3DT.%26aulast%3DMcFarlane%26aufirst%3DJ.%2BM.%2BB.%26aulast%3DConnelly%26aufirst%3DK.%2BE.%26aulast%3DGignac%26aufirst%3DM.%2BC.%26aulast%3DMilosevich%26aufirst%3DN.%26aulast%3DHof%26aufirst%3DF.%26aulast%3DPaci%26aufirst%3DI.%26aulast%3DMusselman%26aufirst%3DC.%2BA.%26aulast%3DDykhuizen%26aufirst%3DE.%2BC.%26aulast%3DKrusemark%26aufirst%3DC.%2BJ.%26atitle%3DOptimization%2520of%2520Ligands%2520Using%2520Focused%2520DNA-Encoded%2520Libraries%2520to%2520Develop%2520a%2520Selective%252C%2520Cell-Permeable%2520CBX8%2520Chromodomain%2520Inhibitor%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2020%26volume%3D15%26issue%3D1%26spage%3D112%26epage%3D131%26doi%3D10.1021%2Facschembio.9b00654" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Højfeldt, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hedehus, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laugesen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tatar, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiehle, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helin, K.</span></span> <span> </span><span class="NLM_article-title">Non-Core Subunits of the PRC2 Complex Are Collectively Required for Its Target-Site Specificity</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>76</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">423</span>– <span class="NLM_lpage">436.e3</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2019.07.031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1016%2Fj.molcel.2019.07.031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=31521506" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A280%3ADC%252BB3MrovVWqsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2019&pages=423-436.e3&issue=3&author=J.+W.+H%C3%B8jfeldtauthor=L.+Hedehusauthor=A.+Laugesenauthor=T.+Tatarauthor=L.+Wiehleauthor=K.+Helin&title=Non-Core+Subunits+of+the+PRC2+Complex+Are+Collectively+Required+for+Its+Target-Site+Specificity&doi=10.1016%2Fj.molcel.2019.07.031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Non-core Subunits of the PRC2 Complex Are Collectively Required for Its Target-Site Specificity</span></div><div class="casAuthors">Hojfeldt Jonas Westergaard; Hedehus Lin; Laugesen Anne; Tatar Tulin; Wiehle Laura; Helin Kristian</div><div class="citationInfo"><span class="NLM_cas:title">Molecular cell</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">423-436.e3</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The Polycomb repressive complex 2 (PRC2) catalyzes H3K27 methylation across the genome, which impacts transcriptional regulation and is critical for establishment of cell identity.  Because of its essential function during development and in cancer, understanding the delineation of genome-wide H3K27 methylation patterns has been the focus of intense investigation.  PRC2 methylation activity is abundant and dispersed throughout the genome, but the highest activity is specifically directed to a subset of target sites that are stably occupied by the complex and highly enriched for H3K27me3.  Here, we show, by systematically knocking out single and multiple non-core subunits of the PRC2 complex in mouse embryonic stem cells, that they each contribute to directing PRC2 activity to target sites.  Furthermore, combined knockout of six non-core subunits reveals that, while dispensable for global H3K27 methylation levels, the non-core PRC2 subunits are collectively required for focusing H3K27me3 activity to specific sites in the genome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQdVVBpmAH1NgWRJ3OJdH_qfW6udTcc2eZhpgOIqf_2QLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MrovVWqsQ%253D%253D&md5=eedb372cd8f66a19919b4ea22d9000c3</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2019.07.031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2019.07.031%26sid%3Dliteratum%253Aachs%26aulast%3DH%25C3%25B8jfeldt%26aufirst%3DJ.%2BW.%26aulast%3DHedehus%26aufirst%3DL.%26aulast%3DLaugesen%26aufirst%3DA.%26aulast%3DTatar%26aufirst%3DT.%26aulast%3DWiehle%26aufirst%3DL.%26aulast%3DHelin%26aufirst%3DK.%26atitle%3DNon-Core%2520Subunits%2520of%2520the%2520PRC2%2520Complex%2520Are%2520Collectively%2520Required%2520for%2520Its%2520Target-Site%2520Specificity%26jtitle%3DMol.%2520Cell%26date%3D2019%26volume%3D76%26issue%3D3%26spage%3D423%26epage%3D436.e3%26doi%3D10.1016%2Fj.molcel.2019.07.031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brunetti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorunova, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davidson, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panagopoulos, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Micci, F.</span></span> <span> </span><span class="NLM_article-title">Identification of an EPC2-PHF1 Fusion Transcript in Low-Grade Endometrial Stromal Sarcoma</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">27</span>),  <span class="NLM_fpage">19203</span>– <span class="NLM_lpage">19208</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.24969</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.18632%2Foncotarget.24969" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=29721194" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A280%3ADC%252BC1Mjpt1Ojsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=19203-19208&issue=27&author=M.+Brunettiauthor=L.+Gorunovaauthor=B.+Davidsonauthor=S.+Heimauthor=I.+Panagopoulosauthor=F.+Micci&title=Identification+of+an+EPC2-PHF1+Fusion+Transcript+in+Low-Grade+Endometrial+Stromal+Sarcoma&doi=10.18632%2Foncotarget.24969"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of an EPC2-PHF1 fusion transcript in low-grade endometrial stromal sarcoma</span></div><div class="casAuthors">Brunetti Marta; Gorunova Ludmila; Heim Sverre; Panagopoulos Ioannis; Micci Francesca; Davidson Ben; Davidson Ben; Heim Sverre</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">19203-19208</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Recurrent chromosomal translocations leading to gene fusion formation have been described in uterine sarcomas, including low-grade endometrial stromal sarcoma (LG-ESS).  Involvement of the PHF1 gene in chromosomal rearrangements targeting band 6p21 has been found in LG-ESS with different partners from JAZF1 mapping in 7p15, to EPC1 from 10p11, MEAF6 from 1p34, and BRD8 from 5q31.  In the present study, RNA sequencing of a LG-ESS showed a novel recombination of PHF1 with the Enhancer of Polycomb homolog 2 (EPC2).  RT-PCR followed by Sanger sequencing and FISH analysis confirmed the EPC2-PHF1 fusion transcript.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSCiJg2QydNO691pGlMMoT5fW6udTcc2eZhpgOIqf_2QLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1Mjpt1Ojsw%253D%253D&md5=5e7b3dd5d69a418da3f130fe2fbae206</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.24969&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.24969%26sid%3Dliteratum%253Aachs%26aulast%3DBrunetti%26aufirst%3DM.%26aulast%3DGorunova%26aufirst%3DL.%26aulast%3DDavidson%26aufirst%3DB.%26aulast%3DHeim%26aufirst%3DS.%26aulast%3DPanagopoulos%26aufirst%3DI.%26aulast%3DMicci%26aufirst%3DF.%26atitle%3DIdentification%2520of%2520an%2520EPC2-PHF1%2520Fusion%2520Transcript%2520in%2520Low-Grade%2520Endometrial%2520Stromal%2520Sarcoma%26jtitle%3DOncotarget%26date%3D2018%26volume%3D9%26issue%3D27%26spage%3D19203%26epage%3D19208%26doi%3D10.18632%2Foncotarget.24969" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hoang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C.-H.</span></span> <span> </span><span class="NLM_article-title">Endometrial Stromal Sarcomas and Related Neoplasms: New Developments and Diagnostic Considerations</span>. <i>Pathology</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>50</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">162</span>– <span class="NLM_lpage">177</span>, <span class="refDoi"> DOI: 10.1016/j.pathol.2017.11.086</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1016%2Fj.pathol.2017.11.086" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=29275929" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitVaktL%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2018&pages=162-177&issue=2&author=L.+Hoangauthor=S.+Chiangauthor=C.-H.+Lee&title=Endometrial+Stromal+Sarcomas+and+Related+Neoplasms%3A+New+Developments+and+Diagnostic+Considerations&doi=10.1016%2Fj.pathol.2017.11.086"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Endometrial stromal sarcomas and related neoplasms: new developments and diagnostic considerations</span></div><div class="casAuthors">Hoang, Lien; Chiang, Sarah; Lee, Cheng-Han</div><div class="citationInfo"><span class="NLM_cas:title">Pathology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">162-177</span>CODEN:
                <span class="NLM_cas:coden">PTLGAX</span>;
        ISSN:<span class="NLM_cas:issn">0031-3025</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Our understanding of endometrial stromal sarcomas has evolved dramatically since their earliest descriptions from over a century ago.  Initial studies focused on establishing the relationship between histol. appearances of endometrial stromal sarcomas and their clin. outcomes.  Studies performed in the last decade have uncovered several recurrent cytogenetic aberrations occurring in low- and high-grade endometrial stromal sarcomas.  Low-grade endometrial stromal sarcomas bear close histopathol. resemblance to proliferative-type endometrial stroma, and approx. half harbor t(7;17)(p15;q21) resulting in JAZF1-SUZ12 gene fusion.  Less common JAZF1-PHF1, EPC1-PHF1, MEAF6-PHF1, and MBTD1-CXorf67 fusions have also been reported.  The term 'high-grade endometrial stromal sarcoma' was recently re-introduced in the classification of endometrial stromal tumors after the discovery of t(10;17)(q22;p13) resulting in YWHAE-NUTM2A/B fusion and is assocd. with distinct morphol. characteristics.  This review highlights the evolution of endometrial stromal sarcoma classification schemes over time and describes the salient clinicopathol. and mol. features of endometrial stromal nodule, low-grade endometrial stromal sarcoma, high-grade endometrial stromal sarcoma, and undifferentiated uterine sarcoma.  It also describes the recent characterization of endometrial stromal sarcoma with t(X;22)(p11;q13) resulting in ZC3H7B-BCOR fusion, a noteworthy entity due to its close histol. resemblance to myxoid leiomyosarcoma.  We also provide insights into common challenging scenarios encountered when assessing endometrial stromal lesions in daily surgical pathol. practice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrz3Puf7lZGP7Vg90H21EOLACvtfcHk0lhRTIv3C8CyBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitVaktL%252FM&md5=5b53b2d33b381b336a78f1e22fe752b3</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.pathol.2017.11.086&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pathol.2017.11.086%26sid%3Dliteratum%253Aachs%26aulast%3DHoang%26aufirst%3DL.%26aulast%3DChiang%26aufirst%3DS.%26aulast%3DLee%26aufirst%3DC.-H.%26atitle%3DEndometrial%2520Stromal%2520Sarcomas%2520and%2520Related%2520Neoplasms%253A%2520New%2520Developments%2520and%2520Diagnostic%2520Considerations%26jtitle%3DPathology%26date%3D2018%26volume%3D50%26issue%3D2%26spage%3D162%26epage%3D177%26doi%3D10.1016%2Fj.pathol.2017.11.086" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bae, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viviano, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lv, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sagum, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castellano, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khalil, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sbardella, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span> <span> </span><span class="NLM_article-title">Developing Spindlin1 Small-Molecule Inhibitors by Using Protein Microarrays</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">750</span>– <span class="NLM_lpage">756</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2377</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1038%2Fnchembio.2377" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=28504676" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnslKktrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=750-756&issue=7&author=N.+Baeauthor=M.+Vivianoauthor=X.+Suauthor=J.+Lvauthor=D.+Chengauthor=C.+Sagumauthor=S.+Castellanoauthor=X.+Baiauthor=C.+Johnsonauthor=M.+I.+Khalilauthor=J.+Shenauthor=K.+Chenauthor=H.+Liauthor=G.+Sbardellaauthor=M.+T.+Bedford&title=Developing+Spindlin1+Small-Molecule+Inhibitors+by+Using+Protein+Microarrays&doi=10.1038%2Fnchembio.2377"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Developing Spindlin1 small-molecule inhibitors by using protein microarrays</span></div><div class="casAuthors">Bae, Narkhyun; Viviano, Monica; Su, Xiaonan; Lv, Jie; Cheng, Donghang; Sagum, Cari; Castellano, Sabrina; Bai, Xue; Johnson, Claire; Khalil, Mahmoud Ibrahim; Shen, Jianjun; Chen, Kaifu; Li, Haitao; Sbardella, Gianluca; Bedford, Mark T.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">750-756</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The discovery of inhibitors of methyl- and acetyl-binding domains has provided evidence for the 'druggability' of epigenetic effector mols.  The small-mol. probe, UNC1215, prevents methyl-dependent protein-protein interactions by engaging the arom. cage of MBT domains and, with lower affinity, Tudor domains.  Here, using a library of tagged UNC1215 analogs, we screened a protein-domain microarray of human methyllysine effector mols. to rapidly detect compds. with new binding profiles with either increased or decreased specificity.  Using this approach, we identified a compd. (EML405) that acquired a novel interaction with Tudor-domain-contg. protein Spindlin1 (SPIN1).  Structural studies facilitated the rational synthesis of SPIN1 inhibitors with increased selectivity (EML631-633), which engaged SPIN1 in cells, blocked its ability to 'read' H3K4me3 marks, and inhibited its transcriptional-coactivator activity.  Protein microarrays can thus be used as a platform to 'target-hop' and identify small mols. that bind and compete with domain-motif interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6diBsmZCcm7Vg90H21EOLACvtfcHk0lhRTIv3C8CyBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnslKktrc%253D&md5=5ac8aab47d4a67cbe0f7c1f04d4ae410</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2377&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2377%26sid%3Dliteratum%253Aachs%26aulast%3DBae%26aufirst%3DN.%26aulast%3DViviano%26aufirst%3DM.%26aulast%3DSu%26aufirst%3DX.%26aulast%3DLv%26aufirst%3DJ.%26aulast%3DCheng%26aufirst%3DD.%26aulast%3DSagum%26aufirst%3DC.%26aulast%3DCastellano%26aufirst%3DS.%26aulast%3DBai%26aufirst%3DX.%26aulast%3DJohnson%26aufirst%3DC.%26aulast%3DKhalil%26aufirst%3DM.%2BI.%26aulast%3DShen%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DSbardella%26aufirst%3DG.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DDeveloping%2520Spindlin1%2520Small-Molecule%2520Inhibitors%2520by%2520Using%2520Protein%2520Microarrays%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26issue%3D7%26spage%3D750%26epage%3D756%26doi%3D10.1038%2Fnchembio.2377" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fagan, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johansson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gileadi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monteiro, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunford, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nibhani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malzahn, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faram, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cribbs, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halidi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chau, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greschik, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velupillai, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allali-Hassani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christott, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giroud, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Athanasou, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bountra, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schüle, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farnie, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oppermann, U.</span></span> <span> </span><span class="NLM_article-title">A Chemical Probe for Tudor Domain Protein Spindlin1 to Investigate Chromatin Function</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span> (<span class="NLM_issue">20</span>),  <span class="NLM_fpage">9008</span>– <span class="NLM_lpage">9025</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00562</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00562" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvVCntrvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=9008-9025&issue=20&author=V.+Faganauthor=C.+Johanssonauthor=C.+Gileadiauthor=O.+Monteiroauthor=J.+E.+Dunfordauthor=R.+Nibhaniauthor=M.+Philpottauthor=J.+Malzahnauthor=G.+Wellsauthor=R.+Faramauthor=A.+P.+Cribbsauthor=N.+Halidiauthor=F.+Liauthor=I.+Chauauthor=H.+Greschikauthor=S.+Velupillaiauthor=A.+Allali-Hassaniauthor=J.+Bennettauthor=T.+Christottauthor=C.+Giroudauthor=A.+M.+Lewisauthor=K.+V.+M.+Huberauthor=N.+Athanasouauthor=C.+Bountraauthor=M.+Jungauthor=R.+Sch%C3%BCleauthor=M.+Vedadiauthor=C.+Arrowsmithauthor=Y.+Xiongauthor=J.+Jinauthor=O.+Fedorovauthor=G.+Farnieauthor=P.+E.+Brennanauthor=U.+Oppermann&title=A+Chemical+Probe+for+Tudor+Domain+Protein+Spindlin1+to+Investigate+Chromatin+Function&doi=10.1021%2Facs.jmedchem.9b00562"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">A Chemical Probe for Tudor Domain Protein Spindlin1 to Investigate Chromatin Function</span></div><div class="casAuthors">Fagan, Vincent; Johansson, Catrine; Gileadi, Carina; Monteiro, Octovia; Dunford, James E.; Nibhani, Reshma; Philpott, Martin; Malzahn, Jessica; Wells, Graham; Faram, Ruth; Cribbs, Adam P.; Halidi, Nadia; Li, Fengling; Chau, Irene; Greschik, Holger; Velupillai, Srikannathasan; Allali-Hassani, Abdellah; Bennett, James; Christott, Thomas; Giroud, Charline; Lewis, Andrew M.; Huber, Kilian V. M.; Athanasou, Nick; Bountra, Chas; Jung, Manfred; Schule, Roland; Vedadi, Masoud; Arrowsmith, Cheryl; Xiong, Yan; Jin, Jian; Fedorov, Oleg; Farnie, Gillian; Brennan, Paul E.; Oppermann, Udo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">9008-9025</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Modifications of histone tails, including lysine/arginine methylation, provide the basis of a 'chromatin or histone code'.  Proteins that contain 'reader' domains can bind to these modifications and form specific effector complexes, which ultimately mediate chromatin function.  The spindlin1 (SPIN1) protein contains three Tudor methyl-lysine/arginine reader domains and was identified as a putative oncogene and transcriptional co-activator.  Here we report a SPIN1 chem. probe inhibitor with low nanomolar in vitro activity, exquisite selectivity on a panel of Me reader and writer proteins, and with submicromolar cellular activity.  X-ray crystallog. showed that this Tudor domain chem. probe simultaneously engages Tudor domains 1 and 2 via a bidentate binding mode.  Small mol. inhibition and siRNA knockdown of SPIN1, as well as chemoproteomic studies, identified genes which are transcriptionally regulated by SPIN1 in squamous cell carcinoma and suggest that SPIN1 may have a roll in cancer related inflammation and/or cancer metastasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzjLXYszX5ZrVg90H21EOLACvtfcHk0lhRTIv3C8CyBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvVCntrvK&md5=cd05be7d6d6ec3cc200a4fdb39f2a7f2</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00562%26sid%3Dliteratum%253Aachs%26aulast%3DFagan%26aufirst%3DV.%26aulast%3DJohansson%26aufirst%3DC.%26aulast%3DGileadi%26aufirst%3DC.%26aulast%3DMonteiro%26aufirst%3DO.%26aulast%3DDunford%26aufirst%3DJ.%2BE.%26aulast%3DNibhani%26aufirst%3DR.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DMalzahn%26aufirst%3DJ.%26aulast%3DWells%26aufirst%3DG.%26aulast%3DFaram%26aufirst%3DR.%26aulast%3DCribbs%26aufirst%3DA.%2BP.%26aulast%3DHalidi%26aufirst%3DN.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DGreschik%26aufirst%3DH.%26aulast%3DVelupillai%26aufirst%3DS.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DBennett%26aufirst%3DJ.%26aulast%3DChristott%26aufirst%3DT.%26aulast%3DGiroud%26aufirst%3DC.%26aulast%3DLewis%26aufirst%3DA.%2BM.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DAthanasou%26aufirst%3DN.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DJung%26aufirst%3DM.%26aulast%3DSch%25C3%25BCle%26aufirst%3DR.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DFarnie%26aufirst%3DG.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DOppermann%26aufirst%3DU.%26atitle%3DA%2520Chemical%2520Probe%2520for%2520Tudor%2520Domain%2520Protein%2520Spindlin1%2520to%2520Investigate%2520Chromatin%2520Function%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26issue%3D20%26spage%3D9008%26epage%3D9025%26doi%3D10.1021%2Facs.jmedchem.9b00562" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leng, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span> <span> </span><span class="NLM_article-title">PHD Finger Protein 1 (PHF1) Is a Novel Reader for Histone H4R3 Symmetric Dimethylation and Coordinates with PRMT5-WDR77/CRL4B Complex to Promote Tumorigenesis</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>46</i></span> (<span class="NLM_issue">13</span>),  <span class="NLM_fpage">6608</span>– <span class="NLM_lpage">6626</span>, <span class="refDoi"> DOI: 10.1093/nar/gky461</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1093%2Fnar%2Fgky461" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=29846670" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC1MXosVykt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=6608-6626&issue=13&author=R.+Liuauthor=J.+Gaoauthor=Y.+Yangauthor=R.+Qiuauthor=Y.+Zhengauthor=W.+Huangauthor=Y.+Zengauthor=Y.+Houauthor=S.+Wangauthor=S.+Lengauthor=D.+Fengauthor=W.+Yuauthor=G.+Sunauthor=H.+Shiauthor=X.+Tengauthor=Y.+Wang&title=PHD+Finger+Protein+1+%28PHF1%29+Is+a+Novel+Reader+for+Histone+H4R3+Symmetric+Dimethylation+and+Coordinates+with+PRMT5-WDR77%2FCRL4B+Complex+to+Promote+Tumorigenesis&doi=10.1093%2Fnar%2Fgky461"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">PHD finger protein 1 (PHF1) is a novel reader for histone H4R3 symmetric dimethylation and coordinates with PRMT5-WDR77/CRL4B complex to promote tumorigenesis</span></div><div class="casAuthors">Liu, Ruiqiong; Gao, Jie; Yang, Yang; Qiu, Rongfang; Zheng, Yu; Huang, Wei; Zeng, Yi; Hou, Yongqiang; Wang, Shuang; Leng, Shuai; Feng, Dandan; Yu, Wenqian; Sun, Gancheng; Shi, Hang; Teng, Xu; Wang, Yan</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">6608-6626</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Histone post-translational modifications regulate chromatin structure and function largely through interactions with effector proteins that often contain multiple histone-binding domains.  PHF1 [plant homeodomain (PHD) finger protein 1], which contains two kinds of histone reader modules, a Tudor domain and two PHD fingers, is an essential factor for epigenetic regulation and genome maintenance.  While significant progress has been made in characterizing the function of the Tudor domain, the roles of the two PHD fingers are poorly defined.  Here, we demonstrated that the N-terminal PHD finger of PHF1 recognizes sym. dimethylation of H4R3 (H4R3me2s) catalyzed by PRMT5-WDR77.  However, the C-terminal PHD finger of PHF1, instead of binding to modified histones, directly interacts with DDB1, the main component of the CUL4B-Ring E3 ligase complex (CRL4B), which is responsible for H2AK119 mono-ubiquitination (H2AK119ub1).  We showed that PHF1, PRMT5-WDR77, and CRL4B reciprocally interact with one another and collaborate as a functional unit.  Genome-wide anal. of PHF1/PRMT5/CUL4B targets identified a cohort of genes including Ecadherin and FBXW7, which are critically involved in cell growth and migration.  We demonstrated that PHF1 promotes cell proliferation, invasion, and tumorigenesis in vivo and in vitro and found that its expression is markedly upregulated in a variety of human cancers.  Our data identified a new reader for H4R3me2s and provided a mol. basis for the functional interplay between histone arginine methylation and ubiquitination.  The results also indicated that PHF1 is a key factor in cancer progression, supporting the pursuit of PHF1 as a target for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhfGrXwspki7Vg90H21EOLACvtfcHk0liyy89o8B7yBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXosVykt7c%253D&md5=d7ed750a8532e93dafef1e78cc864a7e</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgky461&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgky461%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DR.%26aulast%3DGao%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DQiu%26aufirst%3DR.%26aulast%3DZheng%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DW.%26aulast%3DZeng%26aufirst%3DY.%26aulast%3DHou%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DLeng%26aufirst%3DS.%26aulast%3DFeng%26aufirst%3DD.%26aulast%3DYu%26aufirst%3DW.%26aulast%3DSun%26aufirst%3DG.%26aulast%3DShi%26aufirst%3DH.%26aulast%3DTeng%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DY.%26atitle%3DPHD%2520Finger%2520Protein%25201%2520%2528PHF1%2529%2520Is%2520a%2520Novel%2520Reader%2520for%2520Histone%2520H4R3%2520Symmetric%2520Dimethylation%2520and%2520Coordinates%2520with%2520PRMT5-WDR77%252FCRL4B%2520Complex%2520to%2520Promote%2520Tumorigenesis%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2018%26volume%3D46%26issue%3D13%26spage%3D6608%26epage%3D6626%26doi%3D10.1093%2Fnar%2Fgky461" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gatchalian, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ikebe, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tencer, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burge, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibson, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Musselman, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poirier, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kono, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutateladze, T. G.</span></span> <span> </span><span class="NLM_article-title">Accessibility of the Histone H3 Tail in the Nucleosome for Binding of Paired Readers</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">10</span>, <span class="refDoi"> DOI: 10.1038/s41467-017-01598-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1038%2Fs41467-017-01598-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=28232747" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtV2ksbvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=1-10&issue=1&author=J.+Gatchalianauthor=X.+Wangauthor=J.+Ikebeauthor=K.+L.+Coxauthor=A.+H.+Tencerauthor=Y.+Zhangauthor=N.+L.+Burgeauthor=L.+Diauthor=M.+D.+Gibsonauthor=C.+A.+Musselmanauthor=M.+G.+Poirierauthor=H.+Konoauthor=J.+J.+Hayesauthor=T.+G.+Kutateladze&title=Accessibility+of+the+Histone+H3+Tail+in+the+Nucleosome+for+Binding+of+Paired+Readers&doi=10.1038%2Fs41467-017-01598-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Accessibility of the histone H3 tail in the nucleosome for binding of paired readers</span></div><div class="casAuthors">Gatchalian, Jovylyn; Wang, Xiaodong; Ikebe, Jinzen; Cox, Khan L.; Tencer, Adam H.; Zhang, Yi; Burge, Nathaniel L.; Di, Luo; Gibson, Matthew D.; Musselman, Catherine A.; Poirier, Michael G.; Kono, Hidetoshi; Hayes, Jeffrey J.; Kutateladze, Tatiana G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-10</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Combinatorial polyvalent contacts of histone-binding domains or readers commonly mediate localization and activities of chromatin-assocd. proteins.  A pair of readers, the PHD fingers of the protein CHD4, has been shown to bivalently recognize histone H3 tails.  Here we describe a mechanism by which these linked but independent readers bind to the intact nucleosome core particle (NCP).  Comprehensive NMR, chem. reactivity, mol. dynamics, and fluorescence analyses point to the crit. roles of intra-nucleosomal histone-DNA interactions that reduce the accessibility of H3 tails in NCP, the nucleosomal DNA, and the linker between readers in modulating nucleosome- and/or histone-binding activities of the readers.  We show that the second PHD finger of CHD4 initiates recruitment to the nucleosome, however both PHDs are required to alter the NCP dynamics.  Our findings reveal a distinctive regulatory mechanism for the assocn. of paired readers with the nucleosome that provides an intricate balance between cooperative and individual activities of the readers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqcRg-zJejfdLVg90H21EOLACvtfcHk0liyy89o8B7yBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtV2ksbvJ&md5=2047a2e726b65c1ed10e009a568b1b42</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1038%2Fs41467-017-01598-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-017-01598-x%26sid%3Dliteratum%253Aachs%26aulast%3DGatchalian%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DIkebe%26aufirst%3DJ.%26aulast%3DCox%26aufirst%3DK.%2BL.%26aulast%3DTencer%26aufirst%3DA.%2BH.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DBurge%26aufirst%3DN.%2BL.%26aulast%3DDi%26aufirst%3DL.%26aulast%3DGibson%26aufirst%3DM.%2BD.%26aulast%3DMusselman%26aufirst%3DC.%2BA.%26aulast%3DPoirier%26aufirst%3DM.%2BG.%26aulast%3DKono%26aufirst%3DH.%26aulast%3DHayes%26aufirst%3DJ.%2BJ.%26aulast%3DKutateladze%26aufirst%3DT.%2BG.%26atitle%3DAccessibility%2520of%2520the%2520Histone%2520H3%2520Tail%2520in%2520the%2520Nucleosome%2520for%2520Binding%2520of%2520Paired%2520Readers%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26issue%3D1%26spage%3D1%26epage%3D10%26doi%3D10.1038%2Fs41467-017-01598-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Delaglio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grzesiek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vuister, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfeifer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bax, A.</span></span> <span> </span><span class="NLM_article-title">NMRPipe: A Multidimensional Spectral Processing System Based on UNIX Pipes</span>. <i>J. Biomol. NMR</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>6</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">277</span>– <span class="NLM_lpage">293</span>, <span class="refDoi"> DOI: 10.1007/BF00197809</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1007%2FBF00197809" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=8520220" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADyaK2MXhtVSmurfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1995&pages=277-293&issue=3&author=F.+Delaglioauthor=S.+Grzesiekauthor=G.+W.+Vuisterauthor=G.+Zhuauthor=J.+Pfeiferauthor=A.+Bax&title=NMRPipe%3A+A+Multidimensional+Spectral+Processing+System+Based+on+UNIX+Pipes&doi=10.1007%2FBF00197809"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">NMRPipe: a multidimensional spectral processing system based on UNIX pipes</span></div><div class="casAuthors">Delaglio, Frank; Grzesiek, Stephan; Vuister, Geerten W.; Zhu, Guang; Pfeifer, John; Bax, Ad</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular NMR</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">277-93</span>CODEN:
                <span class="NLM_cas:coden">JBNME9</span>;
        ISSN:<span class="NLM_cas:issn">0925-2738</span>.
    
            (<span class="NLM_cas:orgname">ESCOM</span>)
        </div><div class="casAbstract">The NMRPipe system is a UNIX software environment of processing, graphics, and anal. tools designed to meet current routine and research-oriented multidimensional processing requirements, and to anticipate and accommodate future demands and developments.  The system is based on UNIX pipes, which allow programs running simultaneously to exchange streams of data under user control.  In an NMRPipe processing scheme, a stream of spectral data flows through a pipeline of processing programs, each of which performs one component of the overall scheme, such as Fourier transformation or linear prediction.  Complete multidimensional processing schemes are constructed as simple UNIX shell scripts.  The processing modules themselves maintain and exploit accurate records of data sizes, detection modes, and calibration information in all dimensions, so that schemes can be constructed without the need to explicitly define or anticipate data sizes or storage details of real and imaginary channels during processing.  The asynchronous pipeline scheme provides other substantial advantages, including high flexibility, favorable processing speeds, choice of both all-in-memory and disk-bound processing, easy adaptation to different data formats, simpler software development and maintenance, and the ability to distribute processing tasks on multi-CPU computers and computer networks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSIUdXrdcEgbVg90H21EOLACvtfcHk0liyy89o8B7yBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXhtVSmurfK&md5=a670fca5b164083e2178fafd2fb951ff</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1007%2FBF00197809&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF00197809%26sid%3Dliteratum%253Aachs%26aulast%3DDelaglio%26aufirst%3DF.%26aulast%3DGrzesiek%26aufirst%3DS.%26aulast%3DVuister%26aufirst%3DG.%2BW.%26aulast%3DZhu%26aufirst%3DG.%26aulast%3DPfeifer%26aufirst%3DJ.%26aulast%3DBax%26aufirst%3DA.%26atitle%3DNMRPipe%253A%2520A%2520Multidimensional%2520Spectral%2520Processing%2520System%2520Based%2520on%2520UNIX%2520Pipes%26jtitle%3DJ.%2520Biomol.%2520NMR%26date%3D1995%26volume%3D6%26issue%3D3%26spage%3D277%26epage%3D293%26doi%3D10.1007%2FBF00197809" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Kabsch%2C+W.+Acta+Crystallogr.%2C+Sect.+D%3A+Biol.+Crystallogr.+2010%2C+66+%282%29%2C+125%E2%80%93132%2C+10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lipQouJbh2-Wg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26issue%3D2%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span> <span> </span><span class="NLM_article-title">Electronic Ligand Builder and Optimization Workbench (ELBOW): A Tool for Ligand Coordinate and Restraint Generation</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>65</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">1074</span>– <span class="NLM_lpage">1080</span>, <span class="refDoi"> DOI: 10.1107/S0907444909029436</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1107%2FS0907444909029436" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=19770504" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFKnsLnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2009&pages=1074-1080&issue=10&author=N.+W.+Moriartyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adams&title=Electronic+Ligand+Builder+and+Optimization+Workbench+%28ELBOW%29%3A+A+Tool+for+Ligand+Coordinate+and+Restraint+Generation&doi=10.1107%2FS0907444909029436"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">electronic Ligand Builder and Optimization Workbench (eLBOW): a tool for ligand coordinate and restraint generation</span></div><div class="casAuthors">Moriarty, Nigel W.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1074-1080</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The electronic Ligand Builder and Optimization Workbench (eLBOW) is a program module of the PHENIX suite of computational crystallog. software.  It is designed to be a flexible procedure that uses simple and fast quantum-chem. techniques to provide chem. accurate information for novel and known ligands alike.  A variety of input formats and options allow the attainment of a no. of diverse goals including geometry optimization and generation of restraints.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojnN4AVY_DwrVg90H21EOLACvtfcHk0lipQouJbh2-Wg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFKnsLnN&md5=3f9a9c6c31c9a4b014a9f151fb6aa8d9</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1107%2FS0907444909029436&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909029436%26sid%3Dliteratum%253Aachs%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26atitle%3DElectronic%2520Ligand%2520Builder%2520and%2520Optimization%2520Workbench%2520%2528ELBOW%2529%253A%2520A%2520Tool%2520for%2520Ligand%2520Coordinate%2520and%2520Restraint%2520Generation%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2009%26volume%3D65%26issue%3D10%26spage%3D1074%26epage%3D1080%26doi%3D10.1107%2FS0907444909029436" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and Development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&issue=4&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+Development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lipQouJbh2-Wg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520Development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26issue%3D4%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkóczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title">PHENIX: A Comprehensive Python-Based System for Macromolecular Structure Solution</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&issue=2&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.+W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+A+Comprehensive+Python-Based+System+for+Macromolecular+Structure+Solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0liL-Xli_F96kA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.%2BW.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520A%2520Comprehensive%2520Python-Based%2520System%2520for%2520Macromolecular%2520Structure%2520Solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26issue%3D2%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HCZ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HCZ','PDB','4HCZ'); return false;">PDB: 4HCZ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7LKY" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7LKY','PDB','7LKY'); return false;">PDB: 7LKY</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i56"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00430">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_72402"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.1c00430?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00430</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Supporting figures and tables (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00430/suppl_file/jm1c00430_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00430/suppl_file/jm1c00430_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00430/suppl_file/jm1c00430_si_001.pdf">jm1c00430_si_001.pdf (4.68 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00430/suppl_file/jm1c00430_si_002.csv">jm1c00430_si_002.csv (1.9 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Protein Data Bank ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7LKY">7LKY</a> (UNC6641). Authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.1c00430&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-12%3BpageGroup%3Astring%3APublication+Pages%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.1c00430%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.1c00430" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679927ea997f3ddd","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
