
<html lang="en"     class="pb-page"  data-request-id="a86c8dfa-aceb-4988-ae4c-1d1fc83fd735"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";article:article:10.1021/jm200903z;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;issue:issue:10.1021/jmcmar.2011.54.issue-17"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Optimization of Cellular Activity of G9a Inhibitors 7-Aminoalkoxy-quinazolines" /></meta><meta name="dc.Creator" content="Feng  Liu" /></meta><meta name="dc.Creator" content="Dalia  Barsyte-Lovejoy" /></meta><meta name="dc.Creator" content="Abdellah  Allali-Hassani" /></meta><meta name="dc.Creator" content="Yunlong  He" /></meta><meta name="dc.Creator" content="J. Martin  Herold" /></meta><meta name="dc.Creator" content="Xin  Chen" /></meta><meta name="dc.Creator" content="Christopher M.  Yates" /></meta><meta name="dc.Creator" content="Stephen V.  Frye" /></meta><meta name="dc.Creator" content="Peter J.  Brown" /></meta><meta name="dc.Creator" content="Jing  Huang" /></meta><meta name="dc.Creator" content="Masoud  Vedadi" /></meta><meta name="dc.Creator" content="Cheryl H.  Arrowsmith" /></meta><meta name="dc.Creator" content="Jian  Jin" /></meta><meta name="dc.Description" content="Protein lysine methyltransferase G9a plays key roles in the transcriptional repression of a variety of genes via dimethylation of lysine 9 on histone H3 (H3K9me2) of chromatin as well as dimethylat..." /></meta><meta name="Description" content="Protein lysine methyltransferase G9a plays key roles in the transcriptional repression of a variety of genes via dimethylation of lysine 9 on histone H3 (H3K9me2) of chromatin as well as dimethylat..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="August 5, 2011" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/jm200903z" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2011 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Fjm200903z" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Fjm200903z" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Fjm200903z" /></link>
        
    
    

<title>Optimization of Cellular Activity of G9a Inhibitors 7-Aminoalkoxy-quinazolines | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/jm200903z" /></meta><meta property="og:title" content="Optimization of Cellular Activity of G9a Inhibitors 7-Aminoalkoxy-quinazolines" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-17/jm200903z/production/images/large/jm-2011-00903z_0016.jpeg" /></meta><meta property="og:description" content="Protein lysine methyltransferase G9a plays key roles in the transcriptional repression of a variety of genes via dimethylation of lysine 9 on histone H3 (H3K9me2) of chromatin as well as dimethylation of nonhistone proteins including tumor suppressor p53. We previously reported the discovery of UNC0321 (3), the most potent G9a inhibitor to date, via structure-based design and structure–activity relationship (SAR) exploration of the quinazoline scaffold represented by BIX01294 (1). Despite its very high in vitro potency, compound 3 lacks sufficient cellular potency. The design and synthesis of several generations of new analogues aimed at improving cell membrane permeability while maintaining high in vitro potency resulted in the discovery of a number of novel G9a inhibitors such as UNC0646 (6) and UNC0631 (7) with excellent potency in a variety of cell lines and excellent separation of functional potency versus cell toxicity. The design, synthesis, and cellular SAR of these potent G9a inhibitors are described." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/jm200903z"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/jm200903z">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Fjm200903z&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Fjm200903z&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/jm200903z&amp;href=/doi/10.1021/jm200903z" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2011</span><span class="cit-fg-volume">, 54</span><span class="cit-fg-issue">, 17</span><span class="cit-fg-pageRange">, 6139-6150</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/54/17" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/jm2008387" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/jm2007722" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Optimization of Cellular Activity of G9a Inhibitors 7-Aminoalkoxy-quinazolines</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Feng++Liu">Feng Liu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Dalia++Barsyte-Lovejoy">Dalia Barsyte-Lovejoy</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Abdellah++Allali-Hassani">Abdellah Allali-Hassani</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Yunlong++He">Yunlong He</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=J.+Martin++Herold">J. Martin Herold</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Xin++Chen">Xin Chen</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Christopher+M.++Yates">Christopher M. Yates</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Stephen+V.++Frye">Stephen V. Frye</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Peter+J.++Brown">Peter J. Brown</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jing++Huang">Jing Huang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Masoud++Vedadi">Masoud Vedadi</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Cheryl+H.++Arrowsmith">Cheryl H. Arrowsmith</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jian++Jin">Jian Jin</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-text">Center for Integrative Chemical Biology and Drug Discovery, Division of Medicinal Chemistry and Natural Products, UNC Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span></div><div class="aff-info" id="aff2"><span class="aff-text">Structural Genomics Consortium, University of Toronto, Toronto, Ontario, M5G 1L7, Ontario, Canada</span></div><div class="aff-info" id="aff3"><span class="aff-text">Laboratory of Cancer Biology and Genetics, National Cancer Institute, National Institutes of Health, Bethesda, Maryland 20892, United States</span></div><div class="corresp-info">Phone: 919-843-8459. Fax: 919-843-8465. E-mail: <a href="/cdn-cgi/l/email-protection#afc5c6cec1c5c6c1efdac1cc81cacbda"><span class="__cf_email__" data-cfemail="b1dbd8d0dfdbd8dff1c4dfd29fd4d5c4">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Fjm200903z&amp;href=/doi/10.1021%2Fjm200903z" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2011</span></span><span class="cit-volume">, 54</span><span class="cit-issue">, 17</span><span class="cit-pageRange">, 6139–6150</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">July 22, 2011</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>10 July 2011</li><li><span class="item_label"><b>Published</b> online</span>5 August 2011</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 8 September 2011</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/jm200903z" title="DOI URL">https://doi.org/10.1021/jm200903z</a></div><div class="article_header-article-copyright"><strong>Copyright © 2011 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D6139%26pageCount%3D12%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DFeng%2BLiu%252C%2BDalia%2BBarsyte-Lovejoy%252C%2BAbdellah%2BAllali-Hassani%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D54%26issueNum%3D17%26contentID%3Djm200903z%26title%3DOptimization%2Bof%2BCellular%2BActivity%2Bof%2BG9a%2BInhibitors%2B7-Aminoalkoxy-quinazolines%26numPages%3D12%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D6150%26publicationDate%3DSeptember%2B2011">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/jm200903z"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">3856</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">92</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/jm200903z" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Optimization of Cellular Activity of G9a Inhibitors 7-Aminoalkoxy-quinazolines&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Feng&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Dalia&quot;,&quot;last_name&quot;:&quot;Barsyte-Lovejoy&quot;},{&quot;first_name&quot;:&quot;Abdellah&quot;,&quot;last_name&quot;:&quot;Allali-Hassani&quot;},{&quot;first_name&quot;:&quot;Yunlong&quot;,&quot;last_name&quot;:&quot;He&quot;},{&quot;first_name&quot;:&quot;J.&quot;,&quot;last_name&quot;:&quot;Martin Herold&quot;},{&quot;first_name&quot;:&quot;Xin&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Christopher&quot;,&quot;last_name&quot;:&quot;M. Yates&quot;},{&quot;first_name&quot;:&quot;Stephen&quot;,&quot;last_name&quot;:&quot;V. Frye&quot;},{&quot;first_name&quot;:&quot;Peter&quot;,&quot;last_name&quot;:&quot;J. Brown&quot;},{&quot;first_name&quot;:&quot;Jing&quot;,&quot;last_name&quot;:&quot;Huang&quot;},{&quot;first_name&quot;:&quot;Masoud&quot;,&quot;last_name&quot;:&quot;Vedadi&quot;},{&quot;first_name&quot;:&quot;Cheryl&quot;,&quot;last_name&quot;:&quot;H. Arrowsmith&quot;},{&quot;first_name&quot;:&quot;Jian&quot;,&quot;last_name&quot;:&quot;Jin&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2011&quot;,&quot;month&quot;:&quot;08&quot;,&quot;day&quot;:&quot;05&quot;,&quot;issue&quot;:&quot;17&quot;,&quot;volume&quot;:&quot;54&quot;,&quot;pages&quot;:&quot;6139-6150&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/jm200903z&quot;},&quot;abstract&quot;:&quot;Protein lysine methyltransferase G9a plays key roles in the transcriptional repression of a variety of genes via dimethylation of lysine 9 on histone H3 (H3K9me2) of chromatin as well as dimethylation of nonhistone proteins including tumor suppressor p53. We previously reported the discovery of UNC0321 (3), the most potent G9a inhibitor to date, via structure-based design and structure–activity relationship (SAR) exploration of the quinazoline scaffold represented by BIX01294 (1). Despite its very high in vitro potency, compound 3 lacks sufficient cellular potency. The design and synthesis of several generations of new analogues aimed at improving cell membrane permeability while maintaining high in vitro potency resulted in the discovery of a number of novel G9a inhibitors such as UNC0646 (6) and UNC0631 (7) with excellent potency in a variety of cell lines and excellent separation of functional potency versus cell toxicity. The design, synthesis, and cellular SAR of these potent G9a inhibitors are described&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Fjm200903z&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm200903z" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm200903z&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=jm200903z" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm200903z&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=jm200903z" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Fjm200903z&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm200903z&amp;href=/doi/10.1021/jm200903z" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/jm200903z" /></input><a href="/doi/pdf/10.1021/jm200903z" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (2 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/jm200903z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Fjm200903z%26sid%3Dliteratum%253Aachs%26pmid%3D21780790%26genre%3Darticle%26aulast%3DLiu%26date%3D2011%26atitle%3DOptimization%2Bof%2BCellular%2BActivity%2Bof%2BG9a%2BInhibitors%2B7-Aminoalkoxy-quinazolines%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D54%26issue%3D17%26spage%3D6139%26epage%3D6150%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291465" title="Structure activity relationship">Structure activity relationship</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=290808" title="Toxicity">Toxicity</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/54/17" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-17/jmcmar.2011.54.issue-17/production/jmcmar.2011.54.issue-17.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-17/jm200903z/production/images/medium/jm-2011-00903z_0016.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-17/jm200903z/production/images/large/jm-2011-00903z_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm200903z&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Protein lysine methyltransferase G9a plays key roles in the transcriptional repression of a variety of genes via dimethylation of lysine 9 on histone H3 (H3K9me2) of chromatin as well as dimethylation of nonhistone proteins including tumor suppressor p53. We previously reported the discovery of UNC0321 (<b>3</b>), the most potent G9a inhibitor to date, via structure-based design and structure–activity relationship (SAR) exploration of the quinazoline scaffold represented by BIX01294 (<b>1</b>). Despite its very high in vitro potency, compound <b>3</b> lacks sufficient cellular potency. The design and synthesis of several generations of new analogues aimed at improving cell membrane permeability while maintaining high in vitro potency resulted in the discovery of a number of novel G9a inhibitors such as UNC0646 (<b>6</b>) and UNC0631 (<b>7</b>) with excellent potency in a variety of cell lines and excellent separation of functional potency versus cell toxicity. The design, synthesis, and cellular SAR of these potent G9a inhibitors are described.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div class="extra-info-sec articleNote"><h4><span>∥</span> Author Contributions</h4><p class="last">These authors contributed equally to this work.</p></div><div class="extra-info-sec articleNote"><h4><span>⊥</span> Author Present Address</h4><p class="last">Viamet Pharmaceuticals, Inc., 2250 Perimeter Park Drive, Suite 320, Morrisville, North Carolina 27560, United States.</p></div><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_45248" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_45248" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The “writers” (the enzymes that produce post-translational modifications (PTMs)), the “readers” (the proteins that recognize PTMs), and the “erasers” (the enzymes that remove PTMs) of the histone code<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1, 2)</a> are critical targets for manipulation to understand the role of the histone code in human disease. Protein lysine methyltransferases (PKMTs), the methyl “writers” of the histone code, catalyze the transfer of a methyl group from <i>S</i>-adenosyl-<span class="smallcaps smallerCapital">l</span>-methionine (SAM) to the ε-amino group of lysine residues of various proteins including histones and nonhistone proteins.<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3, 4)</a> With the exception of DOT1L, all PKMTs contain the evolutionarily conserved SET domain, named after <i>Drosophila</i><u class="uu">S</u>u(var.)3–9 (suppressor of variegation 3–9), <u class="uu">E</u>(z) (enhancer of zeste), and <u class="uu">t</u>rithorax.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Since the first PKMT was characterized in 2000,<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> more than 50 human PKMTs have been identified<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3, 4)</a> and the mechanistic, biochemical, and biological functions of these proteins are still being elucidated. PKMTs display significant variations in protein substrate selectivity and the degree of methylation on lysine from mono- and di-, to trimethylation. In the context of epigenetic gene regulation, these different states of lysine methylation encode distinct signals and are recognized by a host of protein modules such as Tudor and chromodomains, MBT domains and PHD fingers, usually within larger proteins, and chromatin remodeling complexes that direct cellular processes such as chromatin packaging and transcriptional regulation, and DNA repair, replication and metabolism.<a onclick="showRef(event, 'ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref7 ref8">(7, 8)</a> PKMTs themselves are also often components of large regulatory complexes and act in concert with other epigenetic factors.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Selective inhibition of individual PKMT catalytic activity in cellular systems represents a valuable strategy to decipher the complex signaling mechanisms of histone and protein lysine methylation. A “toolkit” of potent, selective, and cell-penetrant chemical probes<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> of PKMTs will permit biological and therapeutic hypotheses to be tested with high confidence in cell-based models of human biology and disease. However, very few such small-molecule tools are available to the biomedical research community.<a onclick="showRef(event, 'ref11 ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref11 ref12 ref13">(11-13)</a></div><div class="NLM_p">G9a (also known as KMT1C or EHMT2) was initially identified as a H3K9 methyltransferase.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> It was later found that G9a could methylate various histone<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> and nonhistone proteins including the tumor suppressor p53.<a onclick="showRef(event, 'ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref16 ref17">(16, 17)</a> G9a is overexpressed in various human cancers,<a onclick="showRef(event, 'ref17 ref18 ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref17 ref18 ref19 ref20">(17-20)</a> and knockdown of G9a inhibits cancer cell growth.<a onclick="showRef(event, 'ref18 ref20 ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref18 ref20 ref21 ref22">(18, 20-22)</a> The dimethylation of p53 K373 by G9a and GLP (also known as KMT1D or EHMT1), a closely related methyltransferase that shares 80% sequence identity with G9a in their respective SET domains and forms a heterodimer with G9a, results in the inactivation of p53, whose loss of function is implicated in over 50% of cancers.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> In addition, G9a has been shown to play a role in cocaine addiction,<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> mental retardation,<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> maintenance of HIV-1 latency,<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> and DNA methylation in embryonic stem (ES) cells.<a onclick="showRef(event, 'ref26 ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref26 ref27 ref28">(26-28)</a></div><div class="NLM_p">The recent report by Jenuwein and colleagues of BIX01294 (<b>1</b>), the first selective small molecule inhibitor of G9a and GLP,<a onclick="showRef(event, 'ref29 ref30'); return false;" href="javascript:void(0);" class="ref ref29 ref30">(29, 30)</a> was an important advance in the PKMT probe discovery area (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>). We have explored the 2,4-diamino-6,7-dimethoxyquinazoline template represented by <b>1</b>. Our initial structure-based design and synthesis in combination with SAR exploration led to the discovery of potent and selective G9a inhibitors UNC0224 (<b>2</b>) and UNC0321 (<b>3</b>, the most potent G9a inhibitor (Morrison <i>K</i><sub>i</sub> = 63 pM) to date) and valuable SAR for the quinazoline template.<a onclick="showRef(event, 'ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref31 ref32">(31, 32)</a> Cheng and co-workers have also reported SAR of the quinazoline scaffold that resulted in the discovery of E72 (<b>4</b>) as a potent and selective GLP inhibitor.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> However, compounds <b>3</b><a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> and <b>4</b><a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> are less potent in cellular assays than <b>1</b> even though <b>3</b> and <b>4</b> are more potent than <b>1</b> in in vitro biochemical assays. Compound <b>1</b> is active at reducing the H3K9me2 levels in bulk histones and at G9a target genes,<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> but it has poor separation of functional potency versus cell toxicity.<a onclick="showRef(event, 'ref29 ref34'); return false;" href="javascript:void(0);" class="ref ref29 ref34">(29, 34)</a> We have recently reported UNC0638 (<b>5</b>), a chemical probe of G9a/GLP with robust on-target activities in cells and an excellent ratio of toxicity to functional potency (tox/function ratio, which is determined dividing the EC<sub>50</sub> value of the observed toxicity by the IC<sub>50</sub> value of the functional potency).<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> Here we report the design and synthesis of novel analogues of compound <b>3</b> aimed at improving cellular potency of this quinazoline series and the SAR of these compounds in cell-based assays that led to the discovery of compound <b>5</b> and novel G9a inhibitors such as UNC0646 (<b>6</b>) and UNC0631 (<b>7</b>), which have excellent tox/function ratios in a variety of cell lines.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-17/jm200903z/production/images/medium/jm-2011-00903z_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-17/jm200903z/production/images/large/jm-2011-00903z_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of known G9a/GLP inhibitors.<a onclick="showRef(event, 'ref29 ref31 ref32 ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref29 ref31 ref32 ref33 ref34">(29, 31-34)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-17/jm200903z/production/images/large/jm-2011-00903z_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm200903z&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_02700" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_02700" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">It was hypothesized that the poor cellular potency of compound <b>3</b> was likely due to its poor cell membrane permeability and low lipophilicity (ALogP = 1.9). To improve cell membrane permeability, and thus cellular potency of this chemical series, we exploited the SAR at the 2-amino, 4-amino, and 7-aminoalkoxy regions of this quinazoline scaffold discovered previously. For the 7-aminoalkoxy region, the four groups (marked in blue in Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>) that yield high in vitro potency<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> were selected as affinity anchors. Because few modifications to the 4-(piperidin-4-yl)amino moiety except the <i>N</i>-capping group (which does not make interactions with the protein) on the piperidine nitrogen are tolerated,<a onclick="showRef(event, 'ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref31 ref32">(31, 32)</a> the three <i>N</i>-capping groups (marked in red in Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>) were selected to increase lipophilicity while maintaining high in vitro potency. Because modifications to the 2-amino moiety are well tolerated,<a onclick="showRef(event, 'ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref31 ref32">(31, 32)</a> various homopiperazine, azepine, piperidine, and cyclohexyl groups (marked in green in Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>) were selected to further increase lipophilicity. These combination compounds were designed to achieve balanced in vitro potency and physicochemical properties aiding cell penetration.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-17/jm200903z/production/images/medium/jm-2011-00903z_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-17/jm200903z/production/images/large/jm-2011-00903z_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Compounds designed to achieve balanced in vitro potency and physicochemical properties aiding cell penetration to improve cellular potency.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-17/jm200903z/production/images/large/jm-2011-00903z_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm200903z&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The inhibitory activity of these compounds on G9a was evaluated using the fluorescence-based <i>S</i>-adenosyl-<span class="smallcaps smallerCapital">l</span>-homocysteine hydrolase (SAHH)-coupled assay<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> that we previously reported. The H3K9me2 antibody cell immunofluorescence In-Cell Western (ICW) assay,<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> recently developed by us, was used to assess the cellular potency of these compounds. This assay allows rapid processing of multiple compounds for H3K9me2 immunofluorescence signal and normalization to cell number via the use of the nucleic acid dye DRAQ5. The compounds were first evaluated in MDA-MB-231 cells because this cell line possesses robust H3K9me2 levels. Most promising compounds were subsequently evaluated in a number of other cancer and normal cell lines. Viability of cells after inhibitor treatment was assessed by an MTT (3-(4,5-dimethylthiazol-2-yl)->2,5-diphenyl tetrazolium bromide) assay, which measures the conversion of yellow MTT to purple formazan by cellular mitochondrial reductase.</div><div class="NLM_p">For the 7-aminoalkoxy region, compounds <b>2</b>, <b>3</b>, <b>8</b>, and <b>9</b>, the four most potent G9a inhibitors (in biochemical assays) that we previously reported,<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> were evaluated in the ICW and MTT assays (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>). Similar to compound <b>3</b>, compound <b>2</b> showed poor potency in reducing H3K9me2 levels in MDA-MB-231 cells even though it was highly potent in the G9a enzyme inhibitory assay. On the other hand, compounds <b>8</b> and <b>9</b>, which are more lipophilic than compounds <b>2</b> and <b>3</b> as evident by their ALogP values, showed higher cellular potency in the ICW assay. All four compounds had low cell toxicity (EC<sub>50</sub> > 10 μM) in the MTT assay. These initial findings validated our hypothesis that the poor cellular potency of compound <b>3</b> was due to its low lipophilicity, and high cellular potency could be achieved by increasing lipophilicity of this series. On the basis of these results, we selected the pyrrolidin-1-ylpropoxy (<b>8</b>) and piperidin-1-ylpropoxy (<b>9</b>) groups as the preferred 7-aminoalkoxy moieties for optimizing other regions of this quinazoline series.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. SAR of the 7-Aminoalkoxy Region<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-17/jm200903z/production/images/medium/jm-2011-00903z_0008.gif" alt="" id="fx1" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-17/jm200903z/production/images/medium/jm-2011-00903z_0009.gif" alt="" id="fx2" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>Table a</sup><p class="last">IC<sub>50</sub> or EC<sub>50</sub> values are the average of at least duplicate assay runs with standard deviation (SD) values that are 3-fold less than the average.</p></div><div class="footnote" id="t1fn2"><sup>Table b</sup><p class="last">ALogP values were calculated using Pipeline Pilot v7.5.</p></div></div><div></div></div><div class="NLM_p">To efficiently explore the 2-amino region, a new synthetic route (9 steps) was developed (Scheme <a class="ref internalNav" href="#sch1" aria-label="1">1</a>). Unlike our previously published synthetic route,<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> this new synthesis allows us to install various 2-amino groups at the last step of the synthetic sequence. Commercially available methyl 4-hydroxy-3-methoxybenzoate (<b>10</b>) was reacted with 1-chloro-3-iodopropane, followed by nitration, to afford the chloride <b>11</b>. Substitution of the chloride with pyrrolidine and subsequent reduction of the nitro group yielded the aniline <b>13</b>. Formation of urea from the aniline <b>13</b> and subsequent ring closure afforded the intermediate <b>15</b>, which was then converted to the 2,4-dichloroquinazoline <b>16</b>. Two consecutive chloro displacement reactions by various amines provided the desired final products <b>17a</b>–<b>17i</b> in good overall yields. Synthesis of compound <b>5</b>, the 2-cyclohexyl analogue of compounds <b>17a</b>–<b>17i</b>, from the intermediate <b>13</b> was reported by us previously.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a></div><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-17/jm200903z/production/images/medium/jm-2011-00903z_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-17/jm200903z/production/images/large/jm-2011-00903z_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compounds <b>17</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-17/jm200903z/production/images/large/jm-2011-00903z_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm200903z&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>(a) 1-Chloro-3-iodopropane, K<sub>2</sub>CO<sub>3</sub>, CH<sub>3</sub>CN, reflux; (b) HNO<sub>3</sub>, Ac<sub>2</sub>O, 0 °C to rt, 75% over two steps; (c) pyrrolidine, K<sub>2</sub>CO<sub>3</sub>, NaI, cat. tetrabutylammonium iodide, CH<sub>3</sub>CN, reflux, 81%; (d) Fe dust, NH<sub>4</sub>OAc, AcOEt–H<sub>2</sub>O, reflux, 63%; (e) NaOCN, AcOH, H<sub>2</sub>O, rt; (f) NaOH, H<sub>2</sub>O, MeOH, reflux; (g) <i>N</i>,<i>N</i>-diethylaniline, POCl<sub>3</sub>, reflux, 50% over three steps; (h) 1-isopropylpiperidin-4-amine, DIEA, THF, rt, 94%; (i) various amines, CF<sub>3</sub>COOH, <i>i</i>-PrOH, 160 °C, microwave, 74–82%.</p></p></figure><div class="NLM_p">Compounds <b>17a</b>–<b>17i</b> and <b>5</b> were highly potent inhibitors (IC<sub>50</sub> < 0.025 μM) in the G9a SAHH-coupled biochemical assay (Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>). Various <i>N</i>-capping groups (<b>17a</b>–<b>17g</b>) on the outer nitrogen of the homopiperazine moiety were well tolerated. In addition, azepane (<b>17i</b>), piperidine (<b>5</b>), and cyclohexyl groups (<b>5</b>) also showed high in vitro potency. Importantly, in addition to maintaining high in vitro potency, these new compounds have improved lipophilicity compared to compounds <b>2</b> and <b>3</b>. We were pleased to find that the combination of high in vitro potency and improved lipophilicity indeed resulted in high cellular potency (IC<sub>50</sub> < 0.5 μM in the ICW assay). In particular, more lipophilic compounds such as <b>17d</b> (ALogP 4.7), <b>17e</b> (ALogP 5.0), <b>17g</b> (ALogP 4.4), <b>17i</b> (ALogP 4.2), and <b>5</b> (ALogP 4.8) had cellular potency (IC<sub>50</sub>) equal to or less than 0.1 μM. Plotting cellular potency of the inhibitors in Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a> and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a> and inhibitors to be discussed below as a function of ALogP values of these inhibitors reveals that cellular potency strongly correlates with lipophilicity (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>). These results validated our design strategy for improving cellular potency. Furthermore, cell toxicity of these highly potent G9a inhibitors was assessed using the MTT assay. While most of these inhibitors had a relatively poor tox/function ratio (<10), compounds <b>17h</b>, <b>17i</b>, and <b>5</b> had a good separation of functional potency (reducing H3K9me2 levels) versus cell toxicity (tox/function ratio >45). In particular, compound <b>5</b> had an excellent tox/function ratio of 140.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. SAR of the 2-Amino Region<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-17/jm200903z/production/images/medium/jm-2011-00903z_0010.gif" alt="" id="fx3" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-17/jm200903z/production/images/medium/jm-2011-00903z_0011.gif" alt="" id="fx4" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>Table a</sup><p class="last">IC<sub>50</sub> or EC<sub>50</sub> values are the average of at least duplicate assay runs with standard deviation (SD) values that are 3-fold less than the average.</p></div><div class="footnote" id="t2fn2"><sup>Table b</sup><p class="last">ALogP values were calculated using Pipeline Pilot v7.5.</p></div></div><div></div></div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-17/jm200903z/production/images/medium/jm-2011-00903z_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-17/jm200903z/production/images/large/jm-2011-00903z_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Analysis of cellular potency of G9a inhibitors as a function of their ALogP values reveals that cellular potency increases as lipophilicity increases for inhibitors that have similar in vitro potency. The inhibitors are color-coded with their G9a in vitro potency.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-17/jm200903z/production/images/large/jm-2011-00903z_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm200903z&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Because compound <b>5</b> showed high cellular potency and excellent separation of functional potency versus cell toxicity, we further investigated this 2-cyclohexyl quinazoline subseries. Thus, a new 9-step synthesis (Scheme <a class="ref internalNav" href="#sch2" aria-label="2">2</a>) was developed to efficiently explore the 7-aminoalkoxy region. Benzyl protection of commercially available 4-hydroxy-3-methoxybenzonitrile (<b>18</b>), followed by nitration and subsequent reduction, afforded the aniline <b>19</b>. Amide formation from the aniline <b>19</b> and subsequent cyclization yielded the quinazolinone <b>20</b>, which was then converted to the 4-chloro quinazoline <b>21</b>. Displacement of the chloro intermediate <b>21</b> with 1-isopropylpiperidin-4-amine, followed by debenzylation, afforded the phenol <b>23</b>. Mitsunobu reactions of the phenol <b>23</b> with various amino alcohols yielded the desired compounds <b>24a</b>, <b>24b</b>, and <b>5</b>.</div><figure id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-17/jm200903z/production/images/medium/jm-2011-00903z_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-17/jm200903z/production/images/large/jm-2011-00903z_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>24</b> and <b>5</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-17/jm200903z/production/images/large/jm-2011-00903z_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm200903z&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>(a) BnBr, K<sub>2</sub>CO<sub>3</sub>, DMF, rt; (b) HNO<sub>3</sub>, Ac<sub>2</sub>O, 0 °C to rt; (c) Fe dust, NH<sub>4</sub>Cl, <i>i</i>-PrOH-H<sub>2</sub>O, reflux, 67% over 3 steps; d) cyclohexane carbonyl chloride, DIEA, DMF-DCM, 0 °C to rt; (e) NaOH, H<sub>2</sub>O<sub>2</sub>, EtOH, reflux, 48% over 2 steps; (f) <i>N</i>,<i>N</i>-diethylaniline, POCl<sub>3</sub>, reflux, 79%; (g) 1-isopropylpiperidin-4-amine, DIEA, <i>i</i>-PrOH, 160 °C, microwave, 94%; (h) Pd/C, 1,4-cyclohexadiene, EtOH, reflux; (i) various amino alcohols, PPh<sub>3</sub>, DIAD, THF, 0 °C to rt, 36–65% over 2 steps.</p></p></figure><div class="NLM_p">Compared to compound <b>5</b>, compound <b>24a</b> was less potent in the ICW assay (Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>). This is likely due to the about 10-fold lower potency of compound <b>24a</b> in the G9a biochemical assay although <b>24a</b> is slightly more lipophilic. The combination of lower in vitro potency and lower lipophilicity of compound <b>24b</b> compared to compound <b>5</b> resulted in lower cellular potency as expected (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>). Interestingly, both compounds <b>24a</b> and <b>24b</b> had low cell toxicity, similar to compound <b>5</b>.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. SAR of 2-Cyclohexyl Quinazoline Analogues<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-17/jm200903z/production/images/medium/jm-2011-00903z_0012.gif" alt="" id="fx5" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-17/jm200903z/production/images/medium/jm-2011-00903z_0013.gif" alt="" id="fx6" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup>Table a</sup><p class="last">IC<sub>50</sub> or EC<sub>50</sub> values are the average of at least duplicate assay runs with standard deviation (SD) values that are 3-fold less than the average.</p></div><div class="footnote" id="t3fn2"><sup>Table b</sup><p class="last">ALogP values were calculated using Pipeline Pilot v7.5.</p></div></div><div></div></div><div class="NLM_p">We next synthesized a number of combination compounds including <b>6</b>, <b>7</b>, and <b>25</b> – <b>27</b> using the synthetic route outlined in Scheme <a class="ref internalNav" href="#sch1" aria-label="1">1</a> and evaluated them in the G9a SAHH-coupled, ICW, and MTT assays. Compounds <b>6</b>, <b>7</b>, <b>25</b>, and <b>26</b>, which had high in vitro potency versus G9a and improved lipophilicity, were highly potent (IC<sub>50</sub> < 0.06 μM) in reducing H3K9me2 levels in MDA-MB-231 cells and had low cell toxicity (Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>). In particular, compounds <b>6</b> and <b>7</b> were more potent than compound <b>5</b> and had a similar tox/function ratio in MDA-MB-231 cells (Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a>). On the other hand, many combination compounds failed to improve cellular potency and reduce cell toxicity. For example, compound <b>27</b> was about 10-fold less potent in reducing cellular levels of H3K9me2 than compounds <b>6</b> and <b>7</b>. The lower cellular potency of compound <b>27</b> is likely due to the combination of its lower G9a in vitro potency and lower lipophilicity compared to compounds <b>6</b> and <b>7</b> (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>).</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Cellular Potency and Cell Toxicity of Combination Compounds<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-17/jm200903z/production/images/medium/jm-2011-00903z_0014.gif" alt="" id="fx7" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-17/jm200903z/production/images/medium/jm-2011-00903z_0015.gif" alt="" id="fx8" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup>Table a</sup><p class="last">IC<sub>50</sub> or EC<sub>50</sub> values are the average of at least duplicate assay runs with standard deviation (SD) values that are 3-fold less than the average.</p></div><div class="footnote" id="t4fn2"><sup>Table b</sup><p class="last">ALogP values were calculated using Pipeline Pilot v7.5.</p></div></div><div></div></div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Potency and Toxicity of Compounds <b>5</b>, <b>6</b>, and <b>7</b> in a Variety of Cell Lines<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="3" align="center">ICW IC<sub>50</sub> (μM)</th><th class="rowsep1 colsep0" colspan="3" align="center">MTT EC<sub>50</sub> (μM)</th><th class="rowsep1 colsep0" colspan="3" align="center">tox/function ratio (MTT EC<sub>50</sub>/ICW IC<sub>50</sub>)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"><b>5</b></th><th class="colsep0 rowsep0" align="center"><b>6</b></th><th class="colsep0 rowsep0" align="center"><b>7</b></th><th class="colsep0 rowsep0" align="center"><b>5</b></th><th class="colsep0 rowsep0" align="center"><b>6</b></th><th class="colsep0 rowsep0" align="center"><b>7</b></th><th class="colsep0 rowsep0" align="center"><b>5</b></th><th class="colsep0 rowsep0" align="center"><b>6</b></th><th class="colsep0 rowsep0" align="center"><b>7</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">breast carcinoma</td><td class="colsep0 rowsep0" align="center">MDA-MB-231</td><td class="colsep0 rowsep0" align="center">0.081</td><td class="colsep0 rowsep0" align="center">0.026</td><td class="colsep0 rowsep0" align="center">0.025</td><td class="colsep0 rowsep0" align="center">11</td><td class="colsep0 rowsep0" align="center">3.3</td><td class="colsep0 rowsep0" align="center">2.8</td><td class="colsep0 rowsep0" align="center">140</td><td class="colsep0 rowsep0" align="center">130</td><td class="colsep0 rowsep0" align="center">110</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">MCF7</td><td class="colsep0 rowsep0" align="center">0.070</td><td class="colsep0 rowsep0" align="center">0.010</td><td class="colsep0 rowsep0" align="center">0.018</td><td class="colsep0 rowsep0" align="center">7.6</td><td class="colsep0 rowsep0" align="center">4.7</td><td class="colsep0 rowsep0" align="center">1.7</td><td class="colsep0 rowsep0" align="center">110</td><td class="colsep0 rowsep0" align="center">470</td><td class="colsep0 rowsep0" align="center">94</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="11" align="center"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">prostate carcinoma</td><td class="colsep0 rowsep0" align="center">PC3</td><td class="colsep0 rowsep0" align="center">0.059</td><td class="colsep0 rowsep0" align="center">0.012</td><td class="colsep0 rowsep0" align="center">0.026</td><td class="colsep0 rowsep0" align="center">14</td><td class="colsep0 rowsep0" align="center">2.8</td><td class="colsep0 rowsep0" align="center">3.8</td><td class="colsep0 rowsep0" align="center">240</td><td class="colsep0 rowsep0" align="center">230</td><td class="colsep0 rowsep0" align="center">150</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">22RV1</td><td class="colsep0 rowsep0" align="center">0.048</td><td class="colsep0 rowsep0" align="center">0.014</td><td class="colsep0 rowsep0" align="center">0.024</td><td class="colsep0 rowsep0" align="center">4.5</td><td class="colsep0 rowsep0" align="center">7.2</td><td class="colsep0 rowsep0" align="center">2.5</td><td class="colsep0 rowsep0" align="center">94</td><td class="colsep0 rowsep0" align="center">510</td><td class="colsep0 rowsep0" align="center">100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="11" align="center"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">colon carcinoma</td><td class="colsep0 rowsep0" align="center">HCT116 wt</td><td class="colsep0 rowsep0" align="center">0.21</td><td class="colsep0 rowsep0" align="center">0.068</td><td class="colsep0 rowsep0" align="center">0.051</td><td class="colsep0 rowsep0" align="center">11</td><td class="colsep0 rowsep0" align="center">8.7</td><td class="colsep0 rowsep0" align="center">5.8</td><td class="colsep0 rowsep0" align="center">52</td><td class="colsep0 rowsep0" align="center">130</td><td class="colsep0 rowsep0" align="center">110</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">HCT116 p53–/–</td><td class="colsep0 rowsep0" align="center">0.24</td><td class="colsep0 rowsep0" align="center">0.086</td><td class="colsep0 rowsep0" align="center">0.072</td><td class="colsep0 rowsep0" align="center">11</td><td class="colsep0 rowsep0" align="center">12</td><td class="colsep0 rowsep0" align="center">6.4</td><td class="colsep0 rowsep0" align="center">46</td><td class="colsep0 rowsep0" align="center">140</td><td class="colsep0 rowsep0" align="center">89</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="11" align="center"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">human fibroblast</td><td class="colsep0 rowsep0" align="center">IMR90</td><td class="colsep0 rowsep0" align="center">0.12</td><td class="colsep0 rowsep0" align="center">0.010</td><td class="colsep0 rowsep0" align="center">0.046</td><td class="colsep0 rowsep0" align="center">2.3</td><td class="colsep0 rowsep0" align="center">3.6</td><td class="colsep0 rowsep0" align="center">1.8</td><td class="colsep0 rowsep0" align="center">19</td><td class="colsep0 rowsep0" align="center">360</td><td class="colsep0 rowsep0" align="center">39</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup>a</sup><p class="last">IC<sub>50</sub> or EC<sub>50</sub> values are the average of at least duplicate assay runs with standard deviation (SD) values that are 3-fold less than the average.</p></div></div></div><div class="NLM_p">Because compounds <b>6</b> and <b>7</b> showed excellent separation of functional potency versus cell toxicity in MDA-MB-231 cells, these two compounds were further evaluated in a variety of cell lines to characterize their cellular potency and cell toxicity. As shown in Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a>, compounds <b>6</b> and <b>7</b> were highly potent in reducing H3K9me2 levels in these cell lines and had low cell toxicity. Similar to compound <b>5</b>, compounds <b>6</b> and <b>7</b> had excellent tox/function ratios in these cell lines. In particular, compound <b>6</b> had an outstanding tox/function ratio in MCF7 (470), 22RV1 (510), and IMR90 (360) cells, making this compound potentially more useful for investigating G9a biology in these specific cell lines. In general, compounds <b>6</b> and <b>7</b> had higher cellular potency but also higher cell toxicity compared to compound <b>5</b>. While compound <b>5</b> has been selected as a chemical probe of G9a/GLP<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> because of its low cell toxicity and lower molecular weight, compounds <b>6</b> and <b>7</b> are valuable alternative tool compounds that may be more suitable for investigating specific cellular systems (vide supra).</div><div class="NLM_p">The excellent selectivity of compound <b>5</b> for G9a/GLP over a wide range of epigenetic and nonepigenetic targets has been reported previously.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> Consistent with these results, we found that compound <b>6</b> was also a potent GLP inhibitor (IC<sub>50</sub> < 0.015 μM in the GLP SAHH-coupled assay) and highly selective for G9a/GLP over several other PKMTs and PRMTs including SETD7 (a H3K4 PKMT), SUV39H2 (a H3K9 PKMT), SETD8 (a H4K20 PKMT), and PRMT3 (IC<sub>50</sub> > 10 μM in the SAHH-coupled assays against these four targets).</div><div class="NLM_p">In addition to methylating histone substrates,<a onclick="showRef(event, 'ref14 ref15 ref35'); return false;" href="javascript:void(0);" class="ref ref14 ref15 ref35">(14, 15, 35)</a> G9a and GLP dimethylate K373 of the tumor suppressor p53.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> We reported the robust effects of compound <b>5</b> on reducing H3K9me2 levels in various cellular systems.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> However, the effects of the quinazoline series on cellular levels of p53 K373me2 have not been reported previously. Thus, we evaluated the effects of compound <b>5</b> on reducing p53 K373me2 levels in H1299 cells (p53 negative), which were cotransfected with p53 and G9a or GLP. As shown in Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>, compound <b>5</b> at 0.25 μM effectively inhibited the dimethylation of p53 K373 without significant changes to levels of total p53 and G9a or GLP in H1299 cells. These results together with our previous H3K9me2 findings<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> strongly suggest that compound <b>5</b> is an effective inhibitor that blocks the methyltransferase activity of G9a and GLP in cells.</div><figure id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-17/jm200903z/production/images/medium/jm-2011-00903z_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-17/jm200903z/production/images/large/jm-2011-00903z_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Compound <b>5</b> at 0.25 μM effectively inhibited the dimethylation of p53 K373 in H1299 cells. p53 expression vector was cotransfected with or without expression vectors for G9a or GLP into H1299 cells. Immediately after transfection, cells were treated with compound <b>5</b> or DMSO for 24 h. Cellular levels of p53 K373me2, total p53 and G9a or GLP were subject to Western blot analysis using antibodies.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-17/jm200903z/production/images/large/jm-2011-00903z_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm200903z&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_16605" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_16605" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">To improve cellular potency of the quinazoline series, we designed several generations of novel analogues aimed at achieving balanced G9a in vitro potency and physicochemical properties aiding cell penetration. New synthetic routes to efficiently synthesize these novel compounds were developed. SAR studies of these newly synthesized compounds in the G9a biochemical (SAHH-coupled), cell-based functional (anti-H3K9me2 ICW), and cell toxicity (MTT) assays validated our inhibitor design strategy and resulted in the discovery of compounds <b>6</b> and <b>7</b>, which had high cellular potency and excellent separation of functional potency versus cell toxicity in a variety of cell lines. Compounds <b>6</b> and <b>7</b> together with compound <b>5</b>, the previously reported G9a chemical probe, are valuable small molecule tools for the biomedical research community to further investigate the biology of G9a and its role in chromatin remodeling as well as PTMs of other proteins.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_26342" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_26342" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Chemistry General Procedures</h3><div class="NLM_p last">HPLC spectra for all compounds were acquired using an Agilent 6110 series system with UV detector set to 254 nm. Samples were injected (5 μL) onto an Agilent Eclipse Plus 4.6 mm × 50 mm, 1.8 μM, C18 column at room temperature. A linear gradient from 10% to 100% B (MeOH + 0.1% acetic acid) in 5.0 min was followed by pumping 100% B for another 2 min with A being H<sub>2</sub>O + 0.1% acetic acid. The flow rate was 1.0 mL/min. Mass spectra (MS) data were acquired in positive ion mode using an Agilent 6110 single quadrupole mass spectrometer with an electrospray ionization (ESI) source. High-resolution mass spectra (HRMS) were acquired using a Shimadzu LCMS-IT-TOF time-of-flight mass spectrometer. Nuclear magnetic resonance (NMR) spectra were recorded at Varian Mercury spectrometer with 400 MHz for proton (<sup>1</sup>H NMR) and 100 MHz for carbon (<sup>13</sup>C NMR); chemical shifts are reported in ppm (δ).<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Preparative HPLC was performed on Agilent Prep 1200 series with UV detector set to 220 nm. Samples were injected onto a Phenomenex Luna 75 mm × 30 mm, 5 μM, C18 column at room temperature. The flow rate was 30 mL/min. A linear gradient was used with 10% of MeOH (A) in 0.1% TFA in H<sub>2</sub>O (B) to 100% of MeOH (A). All compounds that were evaluated in biological assays had >95% purity using the HPLC methods described above.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> 2,4-Dichloro-6-methoxy-7-(3-(pyrrolidin-1-yl)propoxy)quinazoline (<b>16</b>)</h3><div class="NLM_p last">A solution of compound <b>13</b> (3.6 g, 11.67 mmol) (prepared from commercially available methyl 3-methoxy-4-hydroxy benzoate (<b>10</b>) according to the procedures described previously<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a>) in acetic acid (20 mL) and water (10 mL) was treated with sodium cyanate (1.14 g, 17.5 mmol) while stirring, and then the mixture was stirred overnight at rt. After the resulting mixture was concentrated, the desired crude product <b>14</b> was obtained as a brown slurry oil. MS (ESI): 352 [M + H]<sup>+</sup>. A mixture of crude compound <b>14</b> in MeOH (30 mL) was neutralized with 10% NaOH aqueous solution, and then an additional 10% NaOH aqueous solution (10 mL) was added. The mixture was refluxed for 3 h, cooled to room temperature, and neutralized with concentrated HCl to give a precipitate. The resulting precipitate was collected and dried to provide the desired crude product <b>15</b> (2.60 g). MS (ESI): 320 [M + H]<sup>+</sup>. A mixture of the crude compound <b>15</b> (2.60 g, 8.14 mmol) and <i>N</i>,<i>N</i>-diethylaniline (1.3 mL, 8.14 mmol) in POCl<sub>3</sub> (45 mL) was heated at reflux for 7 h. The reaction mixture was concentrated in vacuo and saturated aq NaHCO<sub>3</sub> (50 mL) was added. The resulting mixture was extracted with chloroform (30 mL × 3). The combined organic layers were dried, concentrated, and purified by flash column chromatography on silica gel (0–20% MeOH/CH<sub>2</sub>Cl<sub>2</sub>) to afford the title compound <b>16</b> as a yellow solid (2.06 g, 50% over three steps). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.30 (s, 1H), 7.25 (s, 1H), 4.26 (t, <i>J</i> = 6.5 Hz, 2H), 4.02 (s, 3H), 2.74 (t, <i>J</i> = 7.3 Hz, 2H), 2.64 (m, 4H), 2.23–2.15 (m, 2H), 1.87–1.79 (m, 4H). MS (ESI): 356 [M + H]<sup>+</sup>.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> <i>N</i>-(1-Isopropylpiperidin-4-yl)-6-methoxy-2-(4-methyl-1,4-diazepan-1-yl)-7-(3-(pyrrolidin-1-yl)propoxy)quinazolin-4-amine (<b>17a</b>)</h3><div class="NLM_p last">A mixture of compound <b>16</b> (0.72 g, 2.02 mmol), 1-isopropylpiperidin-4-amine (0.87 g, 6.06 mmol) (this amine and other noncommercially available amines were prepared according to the literature procedures<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a>), and DIEA (0.67 mL, 4.04 mmol) in THF (10 mL) was stirred overnight at rt. After concentration in vacuo, the crude product was purified by silica gel chromatography (0–20% MeOH (1% NH<sub>3</sub>)/CH<sub>2</sub>Cl<sub>2</sub>) to afford the desired product 2-chloro-<i>N</i>-(1-isopropylpiperidin-4-yl)-6-methoxy-7-(3-(pyrrolidin-1-yl)propoxy)quinazolin-4-amine as a yellow solid (0.88 g, 94% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.12 (s, 1H), 6.78 (s, 1H), 5.36 (d, <i>J</i> = 7.7 Hz, 1H), 4.31–4.20 (m, 1H), 4.17 (t, <i>J</i> = 6.7 Hz, 2H), 3.96 (s, 3H), 2.90 (d, <i>J</i> = 11.9 Hz, 2H), 2.84–2.73 (m, 1H), 2.67–2.59 (m, 2H), 2.57–2.46 (m, 4H), 2.45–2.34 (m, 2H), 2.22–2.05 (m, 4H), 1.84–1.71 (m, 4H), 1.65–1.55 (m, 2H), 1.07 (d, <i>J</i> = 6.6 Hz, 6H). MS (ESI): 462 [M + H]<sup>+</sup>. Then, a mixture of the intermediate (71 mg, 0.149 mmol), 1-methylhomopiperazine (34 mg, 0.30 mmol), and TFA (68 mg, 0.60 mmol) in <i>i</i>-PrOH (0.25 mL) was heated by microwave irradiation to 160 °C for 15 min in a sealed tube. After concentration in vacuo, the crude product was purified by preparative HPLC with a gradient from 10% of MeOH (A) in 0.1% TFA in H<sub>2</sub>O (B) to 100% of MeOH (A). The resulting product was basified with saturated aq NaHCO<sub>3</sub> and extracted with CH<sub>2</sub>Cl<sub>2</sub> to afford the title compound <b>17a</b> as a white solid (64 mg, 79% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.87 (s, 1H), 6.71 (s, 1H), 4.98 (d, <i>J</i> = 7.1 Hz, 1H), 4.13 (t, <i>J</i> = 6.8 Hz, 2H), 4.08–3.98 (m, 1H), 3.97–3.92 (m, 2H), 3.89–3.82 (m, 5H), 2.91–2.84 (m, 2H), 2.78–2.69 (m, 1H), 2.69–2.63 (m, 2H), 2.61–2.56 (m, 2H), 2.55–2.50 (m, 2H), 2.51–2.44 (m, 4H), 2.34 (s, 3H), 2.29 (td, <i>J</i> = 11.5, 2.2 Hz, 2H), 2.19–2.11 (m, 2H), 2.11–2.02 (m, 2H), 2.01–1.94 (m, 2H), 1.79–1.70 (m, 4H), 1.54 (ddd, <i>J</i> = 14.9, 11.8, 3.7 Hz, 2H), 1.04 (d, <i>J</i> = 6.6 Hz, 6H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>, two overlapping peaks) δ 158.70, 158.10, 154.05, 149.76, 145.22, 107.08, 102.79, 101.67, 67.40, 59.07, 57.46, 56.76, 54.58, 54.25, 53.01, 48.77, 47.88, 46.83, 45.98, 32.75, 28.64, 27.98, 23.56, 18.60. HPLC: 97%; <i>t</i><sub>R</sub> 1.18 min. MS (ESI): 540 [M + H]<sup>+</sup>.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> 2-(4-Ethyl-1,4-diazepan-1-yl)-<i>N</i>-(1-isopropylpiperidin-4-yl)-6-methoxy-7-(3-(pyrrolidin-1-yl)propoxy)quinazolin-4-amine (<b>17b</b>)</h3><div class="NLM_p last">Compound <b>17b</b> was prepared as a white solid by the same procedures as the preparation of compound <b>17a</b> (80% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.89 (s, 1H), 6.71 (s, 1H), 4.95 (d, <i>J</i> = 7.1 Hz, 1H), 4.15 (t, <i>J</i> = 6.8 Hz, 2H), 4.10–4.01 (m, 1H), 3.98–3.92 (m, 2H), 3.90–3.82 (m, 5H), 2.94–2.85(m, 2H), 2.80–2.71 (m, 3H), 2.64–2.46 (m, 10H), 2.36–2.26 (m, 2H), 2.20–2.13 (m, 2H), 2.12–2.04 (m, 2H), 2.02–1.93 (m, 2H), 1.82–1.70 (m, 4H), 1.56 (ddd, <i>J</i> = 14.6, 12.0, 3.5 Hz, 2H), 1.13–0.99 (m, 9H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 158.75, 158.15, 154.10, 149.80, 145.26, 107.12, 102.81, 101.69, 67.43, 56.84, 55.94, 54.64, 54.47, 54.30, 53.08, 51.71, 48.76, 47.92, 46.28, 45.88, 32.77, 28.68, 27.97, 23.60, 18.63, 12.51. HPLC: 97%; <i>t</i><sub>R</sub> 1.14 min. MS (ESI): 554 [M + H]<sup>+</sup>.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> 2-(4-Isopropyl-1,4-diazepan-1-yl)-<i>N</i>-(1-isopropylpiperidin-4-yl)-6-methoxy-7-(3-(pyrrolidin-1-yl)propoxy)quinazolin-4-amine (<b>17c</b>)</h3><div class="NLM_p last">Compound <b>17c</b> was prepared as an off-white solid by the same procedures as the preparation of compound <b>17a</b> (82% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.87 (s, 1H), 6.72 (s, 1H), 4.98 (d, <i>J</i> = 7.1 Hz, 1H), 4.13 (t, <i>J</i> = 6.8 Hz, 2H), 4.09–3.99 (m, 1H), 3.93–3.87 (m, 2H), 3.88–3.80 (m, 5H), 2.95–2.83 (m, 3H), 2.78–2.69 (m, 3H), 2.62–2.53 (m, 4H), 2.53–2.43 (m, 4H), 2.33–2.23 (m, 2H), 2.20–2.11 (m, 2H), 2.10–2.02 (m, 2H), 1.94–1.85 (m, 2H), 1.81–1.68 (m, 4H), 1.54 (ddd, <i>J</i> = 14.6, 12.0, 3.6 Hz, 2H), 1.04 (d, <i>J</i> = 6.6 Hz, 6H), 0.98 (d, <i>J</i> = 6.5 Hz, 6H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>, two overlapping peaks) δ 158.74, 158.14, 154.00, 149.85, 145.12, 107.08, 102.78, 101.72, 67.39, 56.77, 55.20, 54.57, 54.25, 53.02, 52.26, 50.68, 48.76, 47.89, 47.80, 45.77, 32.76, 28.78, 28.65, 23.56, 18.61. HPLC: 99%; <i>t</i><sub>R</sub> 1.25 min. MS (ESI): 568 [M + H].</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> 2-(4-Cyclohexyl-1,4-diazepan-1-yl)-<i>N</i>-(1-isopropylpiperidin-4-yl)-6-methoxy-7-(3-(pyrrolidin-1-yl)propoxy)quinazolin-4-amine (<b>17d</b>)</h3><div class="NLM_p last">Compound <b>17d</b> was prepared as a white solid by the same procedures as the preparation of compound <b>17a</b> (78% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.87 (s, 1H), 6.72 (s, 1H), 4.98 (d, <i>J</i> = 7.1 Hz, 1H), 4.13 (t, <i>J</i> = 6.8 Hz, 2H), 4.09–3.97 (m, 1H), 3.92–3.78 (m, 7H), 2.91–2.84 (m, 2H), 2.83–2.77 (m, 2H), 2.77–2.68 (m, 1H), 2.65–2.60 (m, 2H), 2.58 (t, <i>J</i> = 7.4 Hz, 2H), 2.53–2.39 (m, 5H), 2.33–2.24 (m, 2H), 2.18–2.10 (m, 2H), 2.10–2.00 (m, 2H), 1.92–1.83 (m, 2H), 1.82–1.68 (m, 8H), 1.62–1.47 (m, 3H), 1.25–1.06 (m, 5H), 1.04 (d, <i>J</i> = 6.5 Hz, 6H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 158.73, 158.13, 153.98, 149.86, 145.09, 107.07, 102.76, 101.72, 67.37, 64.43, 56.75, 54.56, 54.24, 53.01, 52.64, 51.26, 48.75, 47.99, 47.89, 45.78, 32.75, 29.34, 28.85, 28.64, 26.49, 26.19, 23.55, 18.60. HPLC: 99%; <i>t</i><sub>R</sub> 2.16 min. MS (ESI): 608 [M + H]<sup>+</sup>.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> 2-(4-(Cyclohexylmethyl)-1,4-diazepan-1-yl)-<i>N</i>-(1-isopropylpiperidin-4-yl)-6-methoxy-7-(3-(pyrrolidin-1-yl)propoxy)quinazolin-4-amine (<b>17e</b>)</h3><div class="NLM_p last">Compound <b>17e</b> was prepared as a white solid by the same procedures as the preparation of compound <b>17a</b> (81% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.87 (s, 1H), 6.73 (s, 1H), 4.99 (d, <i>J</i> = 7.1 Hz, 1H), 4.13 (t, <i>J</i> = 6.8 Hz, 2H), 4.09–3.98 (m, 1H), 3.94–3.87 (m, 2H), 3.87–3.76 (m, 5H), 2.91–2.83 (m, 2H), 2.78–2.67 (m, 3H), 2.62–2.52 (m, 4H), 2.52–2.43 (m, 4H), 2.29 (t, <i>J</i> = 10.7 Hz, 2H), 2.22 (d, <i>J</i> = 6.9 Hz, 2H), 2.15 (d, <i>J</i> = 10.8 Hz, 2H), 2.11–2.01 (m, 2H), 1.96–1.86 (m, 2H), 1.81–1.48 (m, 11H), 1.47–1.34 (m, 1H), 1.26–1.10 (m, 3H), 1.04 (d, <i>J</i> = 6.5 Hz, 6H), 0.88–0.74 (m, 2H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>, two overlapping peaks) δ 158.75, 158.10, 153.99, 149.86, 145.10, 107.07, 102.76, 101.72, 67.38, 64.33, 56.75, 55.34, 54.56, 54.24, 53.01, 48.77, 47.89, 46.62, 45.96, 36.32, 32.74, 32.00, 28.64, 27.93, 26.96, 26.30, 23.56, 18.60. HPLC: 99%; <i>t</i><sub>R</sub> 2.39 min. MS (ESI): 622 [M + H]<sup>+</sup>.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> <i>N</i>-(1-Isopropylpiperidin-4-yl)-6-methoxy-2-(4-phenyl-1,4-diazepan-1-yl)-7-(3-(pyrrolidin-1-yl)propoxy)quinazolin-4-amine (<b>17f</b>)</h3><div class="NLM_p last">Compound <b>17f</b> was prepared as a yellow solid by the same procedures as the preparation of compound <b>17a</b> (74% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.22 (t, <i>J</i> = 7.9 Hz, 2H), 6.90 (s, 1H), 6.80–6.72 (m, 3H), 6.65 (t, <i>J</i> = 7.2 Hz, 1H), 4.99 (d, <i>J</i> = 7.1 Hz, 1H), 4.18 (t, <i>J</i> = 6.8 Hz, 2H), 4.14–3.99 (m, 3H), 3.90 (s, 3H), 3.71–3.61 (m, 4H), 3.51 (t, <i>J</i> = 6.2 Hz, 2H), 2.96–2.87 (m, 2H), 2.83–2.70 (m, 1H), 2.62 (t, <i>J</i> = 7.3 Hz, 2H), 2.58– 2.45 (m, 4H), 2.32 (t, <i>J</i> = 10.7 Hz, 2H), 2.22–2.05 (m, 6H), 1.85–1.72 (m, 4H), 1.58 (ddd, <i>J</i> = 14.7, 11.9, 3.5 Hz, 2H), 1.08 (d, <i>J</i> = 6.5 Hz, 6H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 158.40, 158.31, 154.12, 149.87, 147.69, 145.37, 129.52, 115.76, 111.66, 107.14, 102.93, 101.63, 67.42, 56.81, 54.60, 54.29, 53.06, 50.56, 48.83, 47.93, 47.78, 47.10, 46.12, 32.86, 28.67, 24.89, 23.59, 18.61. HPLC: 98%; <i>t</i><sub>R</sub> 3.29 min. MS (ESI): 602 [M + H]<sup>+</sup>.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> 2-(4-Benzyl-1,4-diazepan-1-yl)-<i>N</i>-(1-isopropylpiperidin-4-yl)-6-methoxy-7-(3-(pyrrolidin-1-yl)propoxy)quinazolin-4-amine (<b>17g</b>)</h3><div class="NLM_p last">Compound <b>17g</b> was prepared as an off-white solid by the same procedures as the preparation of compound <b>17a</b> (78% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.25–7.12 (m, 5H), 6.82 (s, 1H), 6.67 (s, 1H), 4.95 (d, <i>J</i> = 7.0 Hz, 1H), 4.08 (t, <i>J</i> = 6.8 Hz, 2H), 4.02–3.91 (m, 1H), 3.90–3.84 (m, 2H), 3.84–3.75 (m, 5H), 3.55 (s, 2H), 2.87–2.77 (m, 2H), 2.74–2.61 (m, 3H), 2.59–2.50 (m, 4H), 2.47–2.38 (m, 4H), 2.21 (t, <i>J</i> = 11.3 Hz, 2H), 2.15–1.95 (m, 4H), 1.94–1.83 (m, 2H), 1.73–1.63 (m, 4H), 1.48 (ddd, <i>J</i> = 14.6, 11.9, 3.5 Hz, 2H), 0.98 (d, <i>J</i> = 6.5 Hz, 6H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 158.71, 158.12, 154.04, 149.76, 145.19, 139.61, 128.91, 128.20, 126.87, 107.06, 102.79, 101.72, 67.39, 62.30, 56.76, 56.28, 54.98, 54.58, 54.25, 53.02, 48.76, 47.87, 46.58, 46.01, 32.72, 28.63, 28.15, 23.56, 18.58. HPLC: 98%; <i>t</i><sub>R</sub> 2.27 min. MS (ESI): 616 [M + H]<sup>+</sup>.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> <i>N</i>-(1-Isopropylpiperidin-4-yl)-6-methoxy-2-(piperidin-1-yl)-7-(3-(pyrrolidin-1-yl)propoxy)quinazolin-4-amine (<b>17h</b>)</h3><div class="NLM_p last">Compound <b>17h</b> was prepared as a white solid by the same procedures as the preparation of compound <b>17a</b> (76% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.90 (s, 1H), 6.75 (s, 1H), 5.10 (d, <i>J</i> = 7.2 Hz, 1H), 4.21–4.01 (m, 3H), 3.87 (s, 3H), 3.80–3.73 (m, 4H), 2.98–2.86 (m, 2H), 2.82–2.72 (m, 1H), 2.65–2.57 (m, 2H), 2.58–2.45 (m, 4H), 2.40–2.28 (m, 2H), 2.20–2.12 (m, 2H), 2.12–2.02 (m, 2H), 1.82–1.69 (m, 4H), 1.69–1.53 (m, 8H), 1.07 (d, <i>J</i> = 6.6 Hz, 6H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 158.95, 158.27, 154.01, 149.36, 145.46, 106.97, 102.93, 101.65, 67.35, 56.71, 54.74, 54.24, 53.02, 48.48, 47.89, 45.18, 32.47, 28.54, 26.07, 25.19, 23.56, 18.51. HPLC: 97%; <i>t</i><sub>R</sub> 2.92 min. MS (ESI): 511 [M + H]<sup>+</sup>.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> 2-(Azepan-1-yl)-<i>N</i>-(1-isopropylpiperidin-4-yl)-6-methoxy-7-(3-(pyrrolidin-1-yl)propoxy)quinazolin-4-amine (<b>17i</b>)</h3><div class="NLM_p last">Compound <b>17i</b> was prepared as a white solid by the same procedures as the preparation of compound <b>17a</b> (78% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.89 (s, 1H), 6.72 (s, 1H), 4.96 (d, <i>J</i> = 6.9 Hz, 1H), 4.15 (t, <i>J</i> = 6.8 Hz, 2H), 4.11–4.00 (m, 1H), 3.87 (s, 3H), 3.77 (t, <i>J</i> = 5.9 Hz, 4H), 2.93–2.85 (m, 2H), 2.80–2.68 (m, 1H), 2.60 (t, <i>J</i> = 7.4 Hz, 2H), 2.55–2.44 (m, 4H), 2.35–2.24 (m, 2H), 2.17 (d, <i>J</i> = 11.3 Hz, 2H), 2.13–2.02 (m, 2H), 1.85–1.67 (m, 8H), 1.63–1.47 (m, 6H), 1.05 (d, <i>J</i> = 6.5 Hz, 6H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 158.76, 158.15, 154.03, 149.91, 145.04, 107.06, 102.71, 101.81, 67.41, 56.84, 54.61, 54.29, 53.06, 48.83, 47.94, 46.99, 32.82, 28.69, 28.66, 27.53, 23.59, 18.62. HPLC: 98%; <i>t</i><sub>R</sub> 3.09 min. MS (ESI): 525 [M + H]<sup>+</sup>.</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> 7-(Benzyloxy)-2-cyclohexyl-6-methoxyquinazolin-4(3<i>H</i>)-one (<b>20</b>)</h3><div class="NLM_p last">Cyclohexanecarbonyl chloride (1.25 mL, 9.2 mmol) was added into a solution of compound <b>19</b> (1.95 g, 7.64 mmol) (prepared from commercially available 4-hydroxy-3-methoxybenzonitrile (<b>18</b>) according to the procedures described previously<a onclick="showRef(event, 'ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref31 ref32">(31, 32)</a>) and DIEA (1.90 mL, 11.5 mmol) in cosolvent DMF:DCM (10 mL:5 mL) at 0 °C. The reaction mixture was stirred overnight at rt, 25 mL of water was added, and the resulting mixture was extracted with CH<sub>2</sub>Cl<sub>2</sub> (15 mL × 3). The combined organic solvent was dried with sodium sulfate, concentrated, and used in the next step without further purification. A mixture of the product, 30 wt % H<sub>2</sub>O<sub>2</sub> (20 mL) and NaOH (400 mg, 10.0 mmol) in ethanol (100 mL) was stirred for 2 h under reflux. After adding 100 mL of water, the reaction mixture was cooled to rt and the resulting precipitate was collected, washed with water, and dried to afford the title compound <b>20</b> as a white solid (1.35 g, 48% yield over 2 steps). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 9.63 (s, 1H), 7.60 (s, 1H), 7.50–7.30 (m, 5H), 7.15 (s, 1H), 5.26 (s, 2H), 3.99 (s, 3H), 2.66–2.53 (m, 1H), 2.03 (d, <i>J</i> = 12.2 Hz, 2H), 1.96–1.86 (m, 2H), 1.83–1.74 (m, 1H), 1.68–1.52 (m, 2H), 1.49–1.24 (m, 3H).</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> 7-(Benzyloxy)-4-chloro-2-cyclohexyl-6-methoxyquinazoline (<b>21</b>)</h3><div class="NLM_p last">A mixture of the compound <b>20</b> (1.32 g, 3.61 mmol) and <i>N</i>,<i>N</i>-diethylaniline (0.30 mL, 1.80 mmol) in POCl<sub>3</sub> (20 mL) was heated at reflux for 4 h. The reaction mixture was concentrated in vacuo, and saturated aq NaHCO<sub>3</sub> (20 mL) was added. The resulting mixture was extracted with chloroform (20 mL × 3). The combined organic layers were dried, concentrated, and purified by flash column chromatography on silica gel (0–10% ethyl acetate/hexanes) to afford the title compound <b>21</b> as a yellow solid (1.10 g, 79% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.51–7.30 (m, 7H), 5.30 (s, 2H), 4.04 (s, 3H), 2.92 (tt, <i>J</i> = 11.7, 3.4 Hz, 1H), 2.10–1.99 (m, 2H), 1.93–1.84 (m, 2H), 1.79–1.64 (m, 3H), 1.49–1.27 (m, 3H).</div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> 7-(Benzyloxy)-2-cyclohexyl-<i>N</i>-(1-isopropylpiperidin-4-yl)-6-methoxyquinazolin-4-amine (<b>22</b>)</h3><div class="NLM_p last">A mixture of compound <b>21</b> (0.68 g, 1.77 mmol), 1-isopropylpiperidin-4-amine (0.77 g, 5.41 mmol), and DIEA (0.60 mL, 3.60 mmol) in <i>i</i>-PrOH (3.0 mL) was heated by microwave irradiation to 160 °C for 15 min in a sealed tube. After concentration in vacuo, the crude product was purified by silica gel chromatography (0–10% MeOH/CH<sub>2</sub>Cl<sub>2</sub>) to afford the title compound <b>22</b> as a white solid (0.81 g, 94% yield). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.52 (s, 1H), 7.45 (d, <i>J</i> = 7.2 Hz, 2H), 7.34 (t, <i>J</i> = 7.3 Hz, 2H), 7.31–7.24 (m, 1H), 7.11 (s, 1H), 5.18 (s, 2H), 4.27–4.19 (m, 1H), 3.94 (s, 3H), 2.98 (d, <i>J</i> = 11.7 Hz, 2H), 2.81–2.71 (m, 1H), 2.69–2.57 (m, 1H), 2.35 (t, <i>J</i> = 11.4 Hz, 2H), 2.13 (d, <i>J</i> = 11.4 Hz, 2H), 1.91–1.82 (m, 4H), 1.79–1.61 (m, 5H), 1.49–1.23 (m, 3H), 1.10 (d, <i>J</i> = 6.5 Hz, 6H). MS (ESI): 489 [M + H]<sup>+</sup>.</div></div><div id="sec4_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> 2-Cyclohexyl-<i>N</i>-(1-isopropylpiperidin-4-yl)-6-methoxy-7-(3-(pyrrolidin-1-yl)propoxy)quinazolin-4-amine (<b>5</b>)</h3><div class="NLM_p last">A mixture of compound <b>22</b> (0.81 g, 1.65 mmol), 1,4-cyclohexadiene (1.6 mL, 16.5 mmol), and 10 wt % Pd/C (250 mg) in ethanol (16 mL) was heated to reflux 2 h. The reaction mixture was filtered and concentrated to provide the debenzylated product <b>23</b> as a yellow solid which was used without purification. MS (ESI): 399 [M + H]<sup>+</sup>. Under nitrogen gas, DIAD (270 μL, 1.38 mmol) was added into an ice-cooled mixture of compound <b>21</b> (110 mg, 0.276 mmol), 3-(pyrrolidin-1-yl)propan-1-ol (142 mg, 1.10 mmol), and PPh<sub>3</sub> (399 mg, 1.52 mmol) in THF (4.0 mL). The reaction mixture was stirred overnight at rt. After concentration in vacuo, the crude product was purified by preparative HPLC with a gradient from 10% of MeOH (A) in 0.1% TFA in H<sub>2</sub>O (B) to 100% of MeOH (A). The resulting product was basified with a saturated solution of NaHCO<sub>3</sub> and extracted with CH<sub>2</sub>Cl<sub>2</sub> to afford 56 mg of the desired product <b>5</b> as a white solid (91 mg, 65% yield over 2 steps). The characterization data (<sup>1</sup>H and <sup>13</sup>C NMR, HPLC, HRMS) of the title compound match the reported.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a></div></div><div id="sec4_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> 2-Cyclohexyl-<i>N</i>-(1-isopropylpiperidin-4-yl)-6-methoxy-7-(3-(piperidin-1-yl)propoxy)quinazolin-4-amine (<b>24a</b>)</h3><div class="NLM_p last">Compound <b>24a</b> was prepared as a white solid from compound <b>23</b> and 3-(piperidin-1-yl)propan-1-ol via Mitsunobu reaction (52% over two steps). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.13 (s, 1H), 6.82 (s, 1H), 5.22 (d, <i>J</i> = 7.3 Hz, 1H), 4.25–4.07 (m, 3H), 3.88 (s, 3H), 2.93–2.84(m, 2H), 2.81–2.63 (m, 2H), 2.48–2.25 (m, 8H), 2.19 (d, <i>J</i> = 11.3 Hz, 2H), 2.08–1.90 (m, 4H), 1.81 (d, <i>J</i> = 12.6 Hz, 2H), 1.75–1.47 (m, 9H), 1.47–1.17 (m, 5H), 1.05 (d, <i>J</i> = 6.5 Hz, 6H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 169.44, 157.93, 153.68, 148.40, 147.38, 108.51, 106.73, 100.07, 67.73, 56.43, 55.78, 54.64, 54.62, 48.59, 47.97, 47.85, 32.67, 32.01, 26.49, 26.48, 26.32, 26.08, 24.53, 18.56. HPLC: 99%; <i>t</i><sub>R</sub> 3.10 min. MS (ESI): 524 [M + H]<sup>+</sup>.</div></div><div id="sec4_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> 2-Cyclohexyl-7-(2-(2-(dimethylamino)ethoxy)ethoxy)-<i>N</i>-(1-isopropylpiperidin-4-yl)-6-methoxyquinazolin-4-amine (<b>24b</b>)</h3><div class="NLM_p last">Compound <b>24b</b> was prepared as a white solid from compound <b>23</b> and 2-(2-(dimethylamino)ethoxy)ethanol via Mitsunobu reaction (36% over two steps). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.11 (s, 1H), 6.83 (s, 1H), 5.22 (d, <i>J</i> = 7.2 Hz, 1H), 4.27–4.13 (m, 3H), 3.92–3.83 (m, 5H), 3.61 (t, <i>J</i> = 5.8 Hz, 2H), 2.93–2.85 (m, 2H), 2.80–2.64 (m, 2H), 2.48 (t, <i>J</i> = 5.8 Hz, 2H), 2.40–2.31 (m, 2H), 2.22 (s, 6H), 2.19 (d, <i>J</i> = 10.5 Hz, 2H), 1.95 (d, <i>J</i> = 11.9 Hz, 2H), 1.86–1.76 (m, 2H), 1.75–1.50 (m, 5H), 1.46–1.21 (m, 3H), 1.05 (d, <i>J</i> = 6.5 Hz, 6H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 169.49, 157.94, 153.43, 148.39, 147.26, 108.52, 107.00, 100.20, 69.62, 69.01, 68.19, 58.84, 56.39, 54.61, 48.62, 47.98, 47.86, 45.91, 32.67, 32.01, 26.49, 26.32, 18.58. HPLC: 99%; <i>t</i><sub>R</sub> 2.98 min. MS (ESI): 514 [M + H]<sup>+</sup>.</div></div><div id="sec4_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> <i>N</i>-(1-Cyclohexylpiperidin-4-yl)-2-(4-isopropyl-1,4-diazepan-1-yl)-6-methoxy-7-(3-(piperidin-1-yl)propoxy)quinazolin-4-amine (<b>6</b>)</h3><div class="NLM_p last">Compound <b>6</b> was prepared as a white solid by the same procedures as the preparation of compound <b>17a</b> (81% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.87 (s, 1H), 6.72 (s, 1H), 4.99 (d, <i>J</i> = 7.1 Hz, 1H), 4.11 (t, <i>J</i> = 6.8 Hz, 2H), 4.08–3.98 (m, 1H), 3.93–3.88 (m, 2H), 3.87–3.78 (m, 5H), 3.17–3.00 (m, 1H), 2.96–2.85 (m, 3H), 2.78–2.70 (m, 2H), 2.60–2.52 (m, 2H), 2.49–2.23 (m, 8H), 2.14 (d, <i>J</i> = 11.1 Hz, 2H), 2.07–1.98 (m, 2H), 1.94–1.71 (m, 6H), 1.66–1.47 (m, 7H), 1.45–1.35 (m, 2H), 1.22 (p, <i>J</i> = 11.7 Hz, 4H), 1.15–1.03 (m, 1H), 0.98 (d, <i>J</i> = 6.6 Hz, 6H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 158.75, 158.12, 154.00, 149.85, 145.12, 107.05, 102.78, 101.72, 67.50, 63.86, 56.75, 55.86, 55.20, 54.66, 52.27, 50.69, 48.82, 48.27, 47.78, 45.77, 32.90, 29.05, 28.79, 26.59, 26.47, 26.14, 26.10, 24.56, 18.61. HPLC: 100%; <i>t</i><sub>R</sub> 2.25 min. MS (ESI): 622 [M + H]<sup>+</sup>. HRMS (ESI) calcd for C<sub>36</sub>H<sub>60</sub>N<sub>7</sub>O<sub>2</sub> [M + H]<sup>+</sup>, 622.4808; found, 622.4810.</div></div><div id="sec4_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> <i>N</i>-(1-(Cyclohexylmethyl)piperidin-4-yl)-2-(4-isopropyl-1,4-diazepan-1-yl)-6-methoxy-7-(3-(piperidin-1-yl)propoxy)quinazolin-4-amine (<b>7</b>)</h3><div class="NLM_p last">Compound <b>7</b> was prepared as a white solid by the same procedures as the preparation of compound <b>17a</b> (81% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.88 (s, 1H), 6.72 (s, 1H), 4.97 (d, <i>J</i> = 6.9 Hz, 1H), 4.13 (t, <i>J</i> = 6.8 Hz, 2H), 4.10–4.00 (m, 1H), 3.94–3.88 (m, 2H), 3.88–3.79 (m, 5H), 2.97–2.88 (m, 1H), 2.88–2.81 (m, 2H), 2.80–2.71 (m, 2H), 2.62–2.54 (m, 2H), 2.50–2.42 (m, 2H), 2.41–2.32 (m, 4H), 2.19–1.98 (m, 8H), 1.96–1.85 (m, 2H), 1.82–1.36 (m, 14H), 1.30–1.09 (m, 3H), 0.99 (d, <i>J</i> = 6.6 Hz, 6H), 0.93–0.80 (m, 2H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 158.79, 158.16, 154.05, 149.87, 145.17, 107.11, 102.81, 101.69, 67.55, 65.89, 56.78, 55.90, 55.23, 54.70, 53.30, 52.28, 50.73, 48.59, 47.80, 45.79, 35.57, 32.45, 32.10, 28.82, 26.95, 26.64, 26.31, 26.15, 24.61, 18.65. HPLC: 99%; <i>t</i><sub>R</sub> 2.56 min. MS (ESI): 636 [M + H]<sup>+</sup>. HRMS (ESI) calcd for C<sub>37</sub>H<sub>62</sub>N<sub>7</sub>O<sub>2</sub> [M + H]<sup>+</sup>, 636.4965; found, 636.4962.</div></div><div id="sec4_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> 2-(4-Isopropyl-1,4-diazepan-1-yl)-<i>N</i>-(1-isopropylpiperidin-4-yl)-6-methoxy-7-(3-(piperidin-1-yl)propoxy)quinazolin-4-amine (<b>25</b>)</h3><div class="NLM_p last">Compound <b>25</b> was prepared as a white solid by the same procedures as the preparation of compound <b>17a</b> (78% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.87 (s, 1H), 6.72 (s, 1H), 4.98 (d, <i>J</i> = 7.1 Hz, 1H), 4.12 (t, <i>J</i> = 6.8 Hz, 2H), 4.09–3.99 (m, 1H), 3.95–3.88 (m, 2H), 3.88–3.78 (m, 5H), 2.97–2.83 (m, 3H), 2.78–2.68 (m, 3H), 2.60–2.53 (m, 2H), 2.49–2.40 (m, 2H), 2.40–2.24 (m, 6H), 2.15 (d, <i>J</i> = 11.1 Hz, 2H), 2.08–1.98 (m, 2H), 1.96–1.84 (m, 2H), 1.61–1.48 (m, 6H), 1.45–1.36 (m, 2H), 1.05 (d, <i>J</i> = 6.5 Hz, 6H), 0.98 (d, <i>J</i> = 6.5 Hz, 6H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>, two overlapping peaks) δ 158.76, 158.14, 154.03, 149.86, 145.14, 107.07, 102.77, 101.74, 67.51, 56.77, 55.87, 55.21, 54.67, 54.58, 52.28, 50.70, 48.76, 47.90, 47.80, 45.78, 32.77, 28.80, 26.60, 26.11, 24.57, 18.61. HPLC: 98%; <i>t</i><sub>R</sub> 1.40 min. MS (ESI): 582 [M + H]<sup>+</sup>.</div></div><div id="sec4_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> <i>N</i>-(1-Cyclohexylpiperidin-4-yl)-6-methoxy-2-(4-methyl-1,4-diazepan-1-yl)-7-(3-(piperidin-1-yl)propoxy)quinazolin-4-amine (<b>26</b>)</h3><div class="NLM_p last">Compound <b>26</b> was prepared as a white solid by the same procedures as the preparation of compound <b>17a</b> (83% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.87 (s, 1H), 6.72 (s, 1H), 5.03 (d, <i>J</i> = 7.1 Hz, 1H), 4.11 (t, <i>J</i> = 6.8 Hz, 2H), 4.07–3.98 (m, 1H), 3.97–3.90 (m, 2H), 3.88–3.79 (m, 5H), 3.27–3.08 (m, 1H), 2.96–2.86 (m, 2H), 2.70–2.62 (m, 2H), 2.58–2.50 (m, 2H), 2.48–2.24 (m, 11H), 2.13 (d, <i>J</i> = 11.0 Hz, 2H), 2.06–1.93 (m, 4H), 1.90–1.71 (m, 4H), 1.66–1.48 (m, 7H), 1.46–1.33 (m, 2H), 1.22 (p, <i>J</i> = 12.0 Hz, 4H), 1.13–1.00 (m, 1H). HPLC: 99%; <i>t</i><sub>R</sub> 2.26 min. MS (ESI): 594 [M + H]<sup>+</sup>.</div></div><div id="sec4_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> 2-(Azepan-1-yl)-<i>N</i>-(1-isopropylpiperidin-4-yl)-6-methoxy-7-(3-(piperidin-1-yl)propoxy)quinazolin-4-amine (<b>27</b>)</h3><div class="NLM_p last">Compound <b>27</b> was prepared as a white solid by the same procedures as the preparation of compound <b>17a</b> (75% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.88 (s, 1H), 6.72 (s, 1H), 4.95 (d, <i>J</i> = 7.0 Hz, 1H), 4.13 (t, <i>J</i> = 6.8 Hz, 2H), 4.10–4.00 (m, 1H), 3.87 (s, 3H), 3.78 (t, <i>J</i> = 6.0 Hz, 4H), 2.93–2.85 (m, 2H), 2.80–2.68 (m, 1H), 2.50–2.42 (m, 2H), 2.41–2.24 (m, 6H), 2.17 (d, <i>J</i> = 10.8 Hz, 2H), 2.0–1.98 (m, 2H), 1.83–1.73 (m, 4H), 1.62–1.49 (m, 10H), 1.45–1.36 (m, 2H), 1.05 (d, <i>J</i> = 6.5 Hz, 6H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 158.81, 158.15, 154.03, 149.98, 145.03, 107.06, 102.70, 101.80, 67.52, 56.82, 55.90, 54.68, 54.60, 48.82, 47.94, 46.97, 32.83, 28.67, 27.54, 26.63, 26.13, 24.59, 18.62. HPLC: 100%; <i>t</i><sub>R</sub> 3.06 min. MS (ESI): 539 [M + H]<sup>+</sup>.</div></div><div id="sec4_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> ALogP Calculation</h3><div class="NLM_p last">ALogP values were calculated using Pipeline Pilot v7.5 (Accelrys Software Inc., San Diego, 2009).</div></div><div id="sec4_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> Experimental Procedures for SAHH-Coupled Assays</h3><div class="NLM_p last">The previously described enzyme-coupled assay<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> was optimized and employed to assay the activity of histone methyltransferases. This assay utilizes SAHH to hydrolyze the methyltransfer product SAH to homocysteine and adenosine in the presence of adenosine deaminase, which converts adenosine to inosine. The homocysteine concentration is then determined through conjugation of its free sulfhydryl moiety to a thiol-sensitive fluorophore, ThioGlo (Calbiochem). For IC<sub>50</sub> determinations, assay mixtures were prepared in 25 mM potassium phosphate buffer pH 7.5, 1 mM EDTA, 2 mM MgCl<sub>2</sub>, 0.01% Triton X-100 with 5 μM SAHH, 0.3 U/mL of adenosine deaminase from Sigma, 25 μM SAM, and 15 μM ThioGlo. G9a, GLP, SETD7, SETD8, PRMT3, and SUV39H2 were assayed at 25, 100, 200, 250, 500, and 100 nM, respectively. Inhibitors were added at concentrations ranging from 4 to 16 μM. After 2 min incubation, reactions were initiated by addition of the histone peptides: 10 μM H3(1–25) for G9a, 20 μM H3(1–25) for GLP, 100 μM H3(1–25) for SETD7, 500 μM H4(1–24) for SETD8, 10 μM H4(1–24) for PRMT3, and 200 μM H3K9Me1 (1–15) for SUV39H2. The methylation reaction was followed by monitoring the increase in fluorescence using Biotek Synergy2 plate reader with 360/40 nm excitation filter and 528/20 nm emission filter for 20 min in 384-well plate format. Activity values were corrected by subtracting background caused by the peptide or the protein. IC<sub>50</sub> values were calculated using Sigmaplot. Standard deviations were calculated from two independent experiments.</div></div><div id="sec4_25" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> Experimental Procedures for ICW and MTT Assays</h3><div class="NLM_p last">MDA-MB-231, PC3, and HCT116 cells were cultured in RPMI with 10% FBS, 22RV1 cells in alphaMEM and 10% FBS, MCF7 and IMR90 cells in DMEM with 10% FBS. Cells were treated with inhibitors for 48 h.</div></div><div id="sec4_26" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> In-Cell Western (ICW) Assay</h3><div class="NLM_p last">Cells were seeded at 5000–10000 cells in black-walled 96-well plates (Greiner) and exposed to various inhibitor concentrations for 48 h. The media was removed and 2% formaldehyde in PBS for fixation was added for 15 min. After five washes with 0.1% Triton X100 in PBS, cells were blocked for 1 h with 1% BSA in PBS. Three out of four replicates were exposed to the primary H3K9me2 antibody, Abcam no. 1220 at 1/800 dilution in 1% BSA, PBS for 2 h. One replicate was reserved for the background control. The wells were washed five times with 0.1% Tween 20 in PBS, then secondary IR800 conjugated antibody (LiCor) and nucleic acid-intercalating dye, DRAQ5 (LiCor) added for 1 h. After 5 washes with 0.1% Tween 20 PBS, the plates were read on an Odyssey (LiCor) scanner at 800 nm (H3K9me2 signal) and 700 nm (DRAQ5 signal). Fluorescence intensity was quantified, normalized to the background, then to the DRAQ5 signal, and expressed as a percentage of control. IC<sub>50</sub>s were calculated using GraphPad Prizm statistical package with sigmoidal variable slope dose response curve fit.</div></div><div id="sec4_27" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> MTT Assay</h3><div class="NLM_p last">The media was removed and replaced with DMEM 10% FBS without phenol red supplemented with 1 mg/mL of MTT and incubated for 1–2 h. Live cells reduce yellow MTT to purple formazan. The resulting formazan was solubilized in acidified isopropyl alcohol and 1% Triton. Formazan signal absorbance was measured at 570 nm and corrected for the 650 nm background. The data was analyzed as above.</div></div><div id="sec4_28" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> Experimental Procedures for the p53 K373me2 Assay</h3><div class="NLM_p last">H1299 cells were transfected with 1 μg p53 expressing vector and 11 μg G9a or GLP expressing vector. Immediately after transfection, cells were treated with 0.25 μM of compound <b>5</b> and incubated for 24 h. Cells were then lysed in NET buffer (50 mM Tris-HCl, pH 8.0, 0.1% NP40, 5 mM EDTA, 150 mM NaCl with protease inhibitors) and followed by immunoprecipitation with DO1 antibody, which recognizing the total p53. Protein A agarose beads were added to collect the immunoprecipitated complex. The immunoprecipitated complex was subject to the Western blot analysis using p53 K373me2 antibody. For p53 and G9a/GLP detection, whole cell lysate was used.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-3"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i45"><a href="/doi/suppl/10.1021/jm200903z">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_36377" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_36377" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last"><sup>1</sup>H and <sup>13</sup>C NMR spectra of compounds <b>6</b> and <b>7</b>. This material is available free of charge via the Internet at <a href="http://pubs.acs.org" class="extLink">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm200903z/suppl_file/jm200903z_si_001.pdf">jm200903z_si_001.pdf (691.26 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/jm200903z" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_86467" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_86467" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jian Jin</span> - <span class="hlFld-Affiliation affiliation">Center for Integrative Chemical Biology and Drug Discovery, Division of Medicinal Chemistry and Natural Products, UNC Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#472d2e26292d2e290732292469222332"><span class="__cf_email__" data-cfemail="88e2e1e9e6e2e1e6c8fde6eba6edecfd">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Feng Liu</span> - <span class="hlFld-Affiliation affiliation">Center for Integrative Chemical Biology and Drug Discovery, Division of Medicinal Chemistry and Natural Products, UNC Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dalia Barsyte-Lovejoy</span> - <span class="hlFld-Affiliation affiliation">Structural Genomics Consortium, University of Toronto, Toronto, Ontario, M5G 1L7, Ontario, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Abdellah Allali-Hassani</span> - <span class="hlFld-Affiliation affiliation">Structural Genomics Consortium, University of Toronto, Toronto, Ontario, M5G 1L7, Ontario, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yunlong He</span> - <span class="hlFld-Affiliation affiliation">Laboratory of Cancer Biology and Genetics, National Cancer Institute, National Institutes of Health, Bethesda, Maryland 20892, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">J. Martin Herold</span> - <span class="hlFld-Affiliation affiliation">Center for Integrative Chemical Biology and Drug Discovery, Division of Medicinal Chemistry and Natural Products, UNC Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xin Chen</span> - <span class="hlFld-Affiliation affiliation">Center for Integrative Chemical Biology and Drug Discovery, Division of Medicinal Chemistry and Natural Products, UNC Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christopher M. Yates</span> - <span class="hlFld-Affiliation affiliation">Center for Integrative Chemical Biology and Drug Discovery, Division of Medicinal Chemistry and Natural Products, UNC Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Structural Genomics Consortium, University of Toronto, Toronto, Ontario, M5G 1L7, Ontario, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Laboratory of Cancer Biology and Genetics, National Cancer Institute, National Institutes of Health, Bethesda, Maryland 20892, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stephen V. Frye</span> - <span class="hlFld-Affiliation affiliation">Center for Integrative Chemical Biology and Drug Discovery, Division of Medicinal Chemistry and Natural Products, UNC Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peter J. Brown</span> - <span class="hlFld-Affiliation affiliation">Structural Genomics Consortium, University of Toronto, Toronto, Ontario, M5G 1L7, Ontario, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jing Huang</span> - <span class="hlFld-Affiliation affiliation">Laboratory of Cancer Biology and Genetics, National Cancer Institute, National Institutes of Health, Bethesda, Maryland 20892, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Masoud Vedadi</span> - <span class="hlFld-Affiliation affiliation">Structural Genomics Consortium, University of Toronto, Toronto, Ontario, M5G 1L7, Ontario, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Cheryl H. Arrowsmith</span> - <span class="hlFld-Affiliation affiliation">Structural Genomics Consortium, University of Toronto, Toronto, Ontario, M5G 1L7, Ontario, Canada</span></div></li></ul></li><li></li><li></li><li></li></ul></div><div class="ack"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i42">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_74581" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_74581" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Dr. Ryan Trump for helpful discussions, Dr. Cen Gao for ALogP calculation, and Dr. R. Bristow for 22RV1 and PC3 cells. The research described here was supported by the grant RC1GM090732 from the NIH, the Carolina Partnership and University Cancer Research Fund from the University of North Carolina at Chapel Hill, and the Structural Genomics Consortium, a registered charity (no. 1097737) that receives funds from the Canadian Institutes of Health Research, the Canada Foundation for Innovation, Genome Canada through the Ontario Genomics Institute, GlaxoSmithKline, Karolinska Institutet, the Knut and Alice Wallenberg Foundation, the Ontario Innovation Trust, the Ontario Ministry for Research and Innovation, Merck & Co., Inc., the Novartis Research Foundation, the Swedish Agency for Innovation Systems, the Swedish Foundation for Strategic Research and the Wellcome Trust.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i43" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i43"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i44" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i44"> Abbreviations Used</h2><tr><td class="NLM_term">H3K9me2</td><td class="NLM_def"><p class="first last">dimethylation of lysine 9 on histone H3</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationships</p></td></tr><tr><td class="NLM_term">PTMs</td><td class="NLM_def"><p class="first last">post-translational modifications</p></td></tr><tr><td class="NLM_term">PKMTs</td><td class="NLM_def"><p class="first last">protein lysine methyltransferases</p></td></tr><tr><td class="NLM_term">SAM</td><td class="NLM_def"><p class="first last"><i>S</i>-adenosyl-<span class="smallcaps smallerCapital">l</span>-methionine</p></td></tr><tr><td class="NLM_term">SET</td><td class="NLM_def"><p class="first last">suppressor of variegation 3–9, enhancer of zeste, and trithorax</p></td></tr><tr><td class="NLM_term">MBT domains</td><td class="NLM_def"><p class="first last">malignant brain tumor domains</p></td></tr><tr><td class="NLM_term">PHD</td><td class="NLM_def"><p class="first last">plant homeo domain</p></td></tr><tr><td class="NLM_term">KMT1C</td><td class="NLM_def"><p class="first last">lysine methyltransferase 1C</p></td></tr><tr><td class="NLM_term">EHMT2</td><td class="NLM_def"><p class="first last">euchromatic histone lysine methyltransferase 2</p></td></tr><tr><td class="NLM_term">H3K9</td><td class="NLM_def"><p class="first last">histone H3 lysine 9</p></td></tr><tr><td class="NLM_term">p53 K373</td><td class="NLM_def"><p class="first last">lysine 373 of p53</p></td></tr><tr><td class="NLM_term">KMT1D</td><td class="NLM_def"><p class="first last">lysine methyltransferase 1D</p></td></tr><tr><td class="NLM_term">EHMT1</td><td class="NLM_def"><p class="first last">euchromatic histone lysine methyltransferase 1</p></td></tr><tr><td class="NLM_term">ES cells</td><td class="NLM_def"><p class="first last">embryonic stem cells</p></td></tr><tr><td class="NLM_term">tox/function ratio</td><td class="NLM_def"><p class="first last">ratio of toxicity to functional potency</p></td></tr><tr><td class="NLM_term">SAHH</td><td class="NLM_def"><p class="first last"><i>S</i>-adenosyl-<span class="smallcaps smallerCapital">l</span>-homocysteine hydrolase</p></td></tr><tr><td class="NLM_term">ICW</td><td class="NLM_def"><p class="first last">in-cell western</p></td></tr><tr><td class="NLM_term">MTT</td><td class="NLM_def"><p class="first last">3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyl tetrazolium bromide</p></td></tr><tr><td class="NLM_term">PRMT</td><td class="NLM_def"><p class="first last">protein arginine methyltransferase</p></td></tr><tr><td class="NLM_term">p53 K373me2</td><td class="NLM_def"><p class="first last">dimethylation of p53 lysine 373</p></td></tr><tr><td class="NLM_term">SAH</td><td class="NLM_def"><p class="first last"><i>S</i>-adenosyl-<span class="smallcaps smallerCapital">l</span>-homocysteine</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i46">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_44635" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_44635" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 38 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Strahl, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span> </span><span class="NLM_article-title">The language of covalent histone modifications</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">403</span><span class="NLM_x">, </span> <span class="NLM_fpage">41</span><span class="NLM_x">–</span> <span class="NLM_lpage">45</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjm200903z&amp;key=10.1038%2F47412" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Fjm200903z&amp;key=10638745" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm200903z&amp;key=1%3ACAS%3A280%3ADC%252BD3c7gt1arsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=403&publication_year=2000&pages=41-45&author=B.+D.+Strahlauthor=C.+D.+Allis&title=The+language+of+covalent+histone+modifications"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The language of covalent histone modifications</span></div><div class="casAuthors">Strahl B D; Allis C D</div><div class="citationInfo"><span class="NLM_cas:title">Nature</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">403</span>
        (<span class="NLM_cas:issue">6765</span>),
    <span class="NLM_cas:pages">41-5</span>
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">Histone proteins and the nucleosomes they form with DNA are the fundamental building blocks of eukaryotic chromatin.  A diverse array of post-translational modifications that often occur on tail domains of these proteins has been well documented.  Although the function of these highly conserved modifications has remained elusive, converging biochemical and genetic evidence suggests functions in several chromatin-based processes.  We propose that distinct histone modifications, on one or more tails, act sequentially or in combination to form a 'histone code' that is, read by other proteins to bring about distinct downstream events.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSyzhdlYh9sX7t7lRqEFUJVfW6udTcc2ebYLUImncil8Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3c7gt1arsQ%253D%253D&md5=2eb7d093151c6122ccd70fa2adcfd94b</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2F47412&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F47412%26sid%3Dliteratum%253Aachs%26aulast%3DStrahl%26aufirst%3DB.%2BD.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DThe%2520language%2520of%2520covalent%2520histone%2520modifications%26jtitle%3DNature%26date%3D2000%26volume%3D403%26spage%3D41%26epage%3D45" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Jenuwein, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span> </span><span class="NLM_article-title">Translating the histone code</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">293</span><span class="NLM_x">, </span> <span class="NLM_fpage">1074</span><span class="NLM_x">–</span> <span class="NLM_lpage">80</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Fjm200903z&amp;key=10.1126%2Fscience.1063127" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Fjm200903z&amp;key=11498575" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Fjm200903z&amp;key=1%3ACAS%3A528%3ADC%252BD3MXmtVWltro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=293&publication_year=2001&pages=1074-80&author=T.+Jenuweinauthor=C.+D.+Allis&title=Translating+the+histone+code"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Translating the histone code</span></div><div class="casAuthors">Jenuwein, Thomas; Allis, C. David</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">293</span>
        (<span class="NLM_cas:issue">5532</span>),
    <span class="NLM_cas:pages">1074-1080</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The review.  Chromatin, the physiol. template of all eukaryotic genetic information, is subject to a diverse array of posttranslational modifications that largely impinge on histone amino termini, thereby regulating access to the underlying DNA.  Distinct histone amino-terminal modifications can generate synergistic or antagonistic interaction affinities for chromatin-assocd. proteins, which in turn dictate dynamic transitions between transcriptionally active or transcriptionally silent chromatin states.  The combinatorial nature of histone amino-terminal modifications thus reveals a "histone code" that considerably extends the information potential of the genetic code.  We propose that this epigenetic marking system represents a fundamental regulatory mechanism that has an impact on most, if not all; chromatin-templated processes, with far-reaching consequences for cell fate decisions and both normal and pathol. development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUFyHNepKcUbVg90H21EOLACvtfcHk0lg48dYYuUdCwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXmtVWltro%253D&md5=021dcd98a5e5bf2ed7eedef1897b2269</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1126%2Fscience.1063127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1063127%26sid%3Dliteratum%253Aachs%26aulast%3DJenuwein%26aufirst%3DT.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DTranslating%2520the%2520histone%2520code%26jtitle%3DScience%26date%3D2001%26volume%3D293%26spage%3D1074%26epage%3D80" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Kouzarides, T.</span><span> </span><span class="NLM_article-title">Chromatin modifications and their function</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">128</span><span class="NLM_x">, </span> <span class="NLM_fpage">693</span><span class="NLM_x">–</span> <span class="NLM_lpage">705</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Fjm200903z&amp;key=10.1016%2Fj.cell.2007.02.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Fjm200903z&amp;key=17320507" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Fjm200903z&amp;key=1%3ACAS%3A528%3ADC%252BD2sXis12ju7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2007&pages=693-705&author=T.+Kouzarides&title=Chromatin+modifications+and+their+function"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatin modifications and their function</span></div><div class="casAuthors">Kouzarides, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">693-705</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The surface of nucleosomes is studded with a multiplicity of modifications.  At least eight different classes have been characterized to date and many different sites have been identified for each class.  Operationally, modifications function either by disrupting chromatin contacts or by affecting the recruitment of nonhistone proteins to chromatin.  Their presence on histones can dictate the higher-order chromatin structure in which DNA is packaged and can orchestrate the ordered recruitment of enzyme complexes to manipulate DNA.  In this way, histone modifications have the potential to influence many fundamental biol. processes, some of which may be epigenetically inherited.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAH9AX2dGicbVg90H21EOLACvtfcHk0lg48dYYuUdCwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXis12ju7Y%253D&md5=c5995fe68fd9473f4268d59c01d3b9de</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.02.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.02.005%26sid%3Dliteratum%253Aachs%26aulast%3DKouzarides%26aufirst%3DT.%26atitle%3DChromatin%2520modifications%2520and%2520their%2520function%26jtitle%3DCell%26date%3D2007%26volume%3D128%26spage%3D693%26epage%3D705" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Solomon, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richon, V. M.</span><span> </span><span class="NLM_article-title">Protein methyltransferases as a target class for drug discovery</span> <span class="citation_source-journal">Nature Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">724</span><span class="NLM_x">–</span> <span class="NLM_lpage">732</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=724-732&author=R.+A.+Copelandauthor=M.+E.+Solomonauthor=V.+M.+Richon&title=Protein+methyltransferases+as+a+target+class+for+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DSolomon%26aufirst%3DM.%2BE.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26atitle%3DProtein%2520methyltransferases%2520as%2520a%2520target%2520class%2520for%2520drug%2520discovery%26jtitle%3DNature%2520Rev.%2520Drug%2520Discovery%26date%3D2009%26volume%3D8%26spage%3D724%26epage%3D732" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Fog, C. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jensen, K. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lund, A. H.</span><span> </span><span class="NLM_article-title">Chromatin-modifying proteins in cancer</span> <span class="citation_source-journal">Acta Pathol. Microbiol. Immunol. Scand.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">115</span><span class="NLM_x">, </span> <span class="NLM_fpage">1060</span><span class="NLM_x">–</span> <span class="NLM_lpage">1089</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Fjm200903z&amp;key=10.1111%2Fj.1600-0463.2007.apm_776.xml.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Fjm200903z&amp;key=18042144" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Fjm200903z&amp;key=1%3ACAS%3A528%3ADC%252BD1cXkslOnsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2007&pages=1060-1089&author=C.+K.+Fogauthor=K.+T.+Jensenauthor=A.+H.+Lund&title=Chromatin-modifying+proteins+in+cancer"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatin-modifying proteins in cancer</span></div><div class="casAuthors">Fog, Cathrine K.; Jensen, Klaus T.; Lund, Anders H.</div><div class="citationInfo"><span class="NLM_cas:title">APMIS</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">115</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1060-1089</span>CODEN:
                <span class="NLM_cas:coden">APMSEL</span>;
        ISSN:<span class="NLM_cas:issn">0903-4641</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">A review.  Chromatin-modifying proteins mold the genome into areas that are accessible for transcriptional activity and areas that are transcriptionally silent.  This epigenetic gene regulation allows for different transcriptional programs to be conducted in different cell types at different timepoints-despite the fact that all cells in the organism contain the same genetic information.  A large amt. of data gathered over the last decades has demonstrated that deregulation of chromatin-modifying proteins is etiol. involved in the development and progression of cancer.  Here we discuss how epigenetic alterations influence cancer development and review known cancer-assocd. alterations in chromatin-modifying proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxhso-Ej5KY7Vg90H21EOLACvtfcHk0liiTwmo2FhNfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXkslOnsQ%253D%253D&md5=060c4f35c69924a72a16b316b773a215</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1111%2Fj.1600-0463.2007.apm_776.xml.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1600-0463.2007.apm_776.xml.x%26sid%3Dliteratum%253Aachs%26aulast%3DFog%26aufirst%3DC.%2BK.%26aulast%3DJensen%26aufirst%3DK.%2BT.%26aulast%3DLund%26aufirst%3DA.%2BH.%26atitle%3DChromatin-modifying%2520proteins%2520in%2520cancer%26jtitle%3DActa%2520Pathol.%2520Microbiol.%2520Immunol.%2520Scand.%26date%3D2007%26volume%3D115%26spage%3D1060%26epage%3D1089" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Rea, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eisenhaber, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Carroll, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strahl, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, Z. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmid, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Opravil, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mechtler, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ponting, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenuwein, T.</span><span> </span><span class="NLM_article-title">Regulation of chromatin structure by site-specific histone H3 methyltransferases</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">406</span><span class="NLM_x">, </span> <span class="NLM_fpage">593</span><span class="NLM_x">–</span> <span class="NLM_lpage">599</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Fjm200903z&amp;key=10.1038%2F35020506" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Fjm200903z&amp;key=10949293" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Fjm200903z&amp;key=1%3ACAS%3A528%3ADC%252BD3cXmtFegs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=406&publication_year=2000&pages=593-599&author=S.+Reaauthor=F.+Eisenhaberauthor=D.+O%E2%80%99Carrollauthor=B.+D.+Strahlauthor=Z.+W.+Sunauthor=M.+Schmidauthor=S.+Opravilauthor=K.+Mechtlerauthor=C.+P.+Pontingauthor=C.+D.+Allisauthor=T.+Jenuwein&title=Regulation+of+chromatin+structure+by+site-specific+histone+H3+methyltransferases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of chromatin structure by site-specific histone H3 methyltransferases</span></div><div class="casAuthors">Rea, Stephen; Elsenhaber, Frank; O'Carroll, Donal; Strahl, Brian D.; Sun, Zu-Wen; Schmid, Manfred; Opravil, Susanne; Mechtler, Karl; Ponting, Chris P.; Allis, C. David; Jenuwein, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">406</span>
        (<span class="NLM_cas:issue">6796</span>),
    <span class="NLM_cas:pages">593-599</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The organization of chromatin into higher-order structures influences chromosome function and epigenetic gene regulation.  Higher-order chromatin has been proposed to be nucleated by the covalent modification of histone tails and the subsequent establishment of chromosomal subdomains by non-histone modifier factors.  Here we show that human SUV39H1 and murine Suv39h1 - mammalian homologs of Drosophila Su(var)3-9 and of Schizosaccharomyces pombe clr4 - encode histone H3-specific methyltransferases that selectively methylate lysine 9 of the amino terminus of histone H3 in vitro.  We mapped the catalytic motif to the evolutionarily conserved SET domain, which requires adjacent cysteine-rich regions to confer histone methyltransferase activity.  Methylation of lysine 9 interferes with phosphorylation of serine 10, but is also influenced by pre-existing modifications in the amino terminus of H3.  In vivo, deregulated SUV39H1 or disrupted Suv39h activity modulate H3 serine 10 phosphorylation in native chromatin and induce aberrant mitotic divisions.  Our data reveal a functional interdependence of site-specific H3 tail modifications and suggest a dynamic mechanism for the regulation of higher-order chromatin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2yioyTahy77Vg90H21EOLACvtfcHk0ljv-uu8YfPoQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXmtFegs7s%253D&md5=65e14733d25f97fadaaeed602600bd23</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2F35020506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F35020506%26sid%3Dliteratum%253Aachs%26aulast%3DRea%26aufirst%3DS.%26aulast%3DEisenhaber%26aufirst%3DF.%26aulast%3DO%25E2%2580%2599Carroll%26aufirst%3DD.%26aulast%3DStrahl%26aufirst%3DB.%2BD.%26aulast%3DSun%26aufirst%3DZ.%2BW.%26aulast%3DSchmid%26aufirst%3DM.%26aulast%3DOpravil%26aufirst%3DS.%26aulast%3DMechtler%26aufirst%3DK.%26aulast%3DPonting%26aufirst%3DC.%2BP.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26aulast%3DJenuwein%26aufirst%3DT.%26atitle%3DRegulation%2520of%2520chromatin%2520structure%2520by%2520site-specific%2520histone%2520H3%2520methyltransferases%26jtitle%3DNature%26date%3D2000%26volume%3D406%26spage%3D593%26epage%3D599" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Ruthenburg, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span> </span><span class="NLM_article-title">Multivalent engagement of chromatin modifications by linked binding modules</span> <span class="citation_source-journal">Nature Rev. Mol. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">983</span><span class="NLM_x">–</span> <span class="NLM_lpage">994</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Fjm200903z&amp;key=10.1038%2Fnrm2298" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Fjm200903z&amp;key=18037899" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Fjm200903z&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlCrsb3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2007&pages=983-994&author=A.+J.+Ruthenburgauthor=H.+Liauthor=D.+J.+Patelauthor=C.+D.+Allis&title=Multivalent+engagement+of+chromatin+modifications+by+linked+binding+modules"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Multivalent engagement of chromatin modifications by linked binding modules</span></div><div class="casAuthors">Ruthenburg, Alexander J.; Li, Haitao; Patel, Dinshaw J.; Allis, C. David</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">983-994</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Histone post-translational modifications (PTMs) play crucial roles in genome management, in part by recruiting specific factors that alter the structural properties of chromatin.  These so-called effector complexes often comprise multiple histone-binding modules that may act in concert to regulate chromatin structure and DNA-related activities.  Various chem. modifications on histones and regions of assocd. DNA play crucial roles in genome management by binding specific factors that, in turn, serve to alter the structural properties of chromatin.  These so-called effector proteins have typically been studied with the biochemist's paring knife; the capacity to recognize specific chromatin PTMs has been mapped to an increasing no. of domains that frequently appear in the nuclear subset of the proteome, often present in large, multisubunit complexes that bristle with modification-dependent binding potential.  The authors propose that multivalent interactions on a single histone tail and beyond may have a significant, if not dominant, role in chromatin transactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkIrlnyvt64LVg90H21EOLACvtfcHk0liDtkJsaP-Jig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlCrsb3L&md5=0d27fc9ba2e2a009e0a9903e1c65bdf7</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnrm2298&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm2298%26sid%3Dliteratum%253Aachs%26aulast%3DRuthenburg%26aufirst%3DA.%2BJ.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DPatel%26aufirst%3DD.%2BJ.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DMultivalent%2520engagement%2520of%2520chromatin%2520modifications%2520by%2520linked%2520binding%2520modules%26jtitle%3DNature%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2007%26volume%3D8%26spage%3D983%26epage%3D994" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Wang, G. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dormann, H. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casadio, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span> </span><span class="NLM_article-title">Haematopoietic malignancies caused by dysregulation of a chromatin-binding PHD finger</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">459</span><span class="NLM_x">, </span> <span class="NLM_fpage">847</span><span class="NLM_x">–</span> <span class="NLM_lpage">851</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Fjm200903z&amp;key=10.1038%2Fnature08036" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Fjm200903z&amp;key=19430464" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Fjm200903z&amp;key=1%3ACAS%3A528%3ADC%252BD1MXls1CmtL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=459&publication_year=2009&pages=847-851&author=G.+G.+Wangauthor=J.+Songauthor=Z.+Wangauthor=H.+L.+Dormannauthor=F.+Casadioauthor=H.+Liauthor=J.+L.+Luoauthor=D.+J.+Patelauthor=C.+D.+Allis&title=Haematopoietic+malignancies+caused+by+dysregulation+of+a+chromatin-binding+PHD+finger"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Haematopoietic malignancies caused by dysregulation of a chromatin-binding PHD finger</span></div><div class="casAuthors">Wang, Gang G.; Song, Jikui; Wang, Zhanxin; Dormann, Holger L.; Casadio, Fabio; Li, Haitao; Luo, Jun-Li; Patel, Dinshaw J.; Allis, C. David</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">459</span>
        (<span class="NLM_cas:issue">7248</span>),
    <span class="NLM_cas:pages">847-851</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Histone H3 lysine 4 methylation (H3K4me) has been proposed as a crit. component in regulating gene expression, epigenetic states, and cellular identities.  The biol. meaning of H3K4me is interpreted by conserved modules including plant homeodomain (PHD) fingers that recognize varied H3K4me states.  The dysregulation of PHD fingers has been implicated in several human diseases, including cancers and immune or neurol. disorders.  Here we report that fusing an H3K4-trimethylation (H3K4me3)-binding PHD finger, such as the carboxy-terminal PHD finger of PHF23 or JARID1A (also known as KDM5A or RBBP2), to a common fusion partner nucleoporin-98 (NUP98) as identified in human leukemias, generated potent oncoproteins that arrested hematopoietic differentiation and induced acute myeloid leukemia in murine models.  In these processes, a PHD finger that specifically recognizes H3K4me3/2 marks was essential for leukemogenesis.  Mutations in PHD fingers that abrogated H3K4me3 binding also abolished leukemic transformation.  NUP98-PHD fusion prevented the differentiation-assocd. removal of H3K4me3 at many loci encoding lineage-specific transcription factors (Hox(s), Gata3, Meis1, Eya1 and Pbx1), and enforced their active gene transcription in murine hematopoietic stem/progenitor cells.  Mechanistically, NUP98-PHD fusions act as 'chromatin boundary factors', dominating over polycomb-mediated gene silencing to 'lock' developmentally crit. loci into an active chromatin state (H3K4me3 with induced histone acetylation), a state that defined leukemia stem cells.  Collectively, our studies represent, to our knowledge, the first report that deregulation of the PHD finger, an 'effector' of specific histone modification, perturbs the epigenetic dynamics on developmentally crit. loci, catastrophizes cellular fate decision-making, and even causes oncogenesis during mammalian development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpR1S9VoRYMRrVg90H21EOLACvtfcHk0liDtkJsaP-Jig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXls1CmtL0%253D&md5=3a647643c8a73639fcb332a774f1155e</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnature08036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08036%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DG.%2BG.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DDormann%26aufirst%3DH.%2BL.%26aulast%3DCasadio%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DLuo%26aufirst%3DJ.%2BL.%26aulast%3DPatel%26aufirst%3DD.%2BJ.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DHaematopoietic%2520malignancies%2520caused%2520by%2520dysregulation%2520of%2520a%2520chromatin-binding%2520PHD%2520finger%26jtitle%3DNature%26date%3D2009%26volume%3D459%26spage%3D847%26epage%3D851" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Fritsch, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robin, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mathieu, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Souidi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hinaux, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rougeulle, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harel-Bellan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ameyar-Zazoua, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ait-Si-Ali, S.</span><span> </span><span class="NLM_article-title">A subset of the histone H3 lysine 9 methyltransferases Suv39h1, G9a, GLP, and SETDB1 participate in a multimeric complex</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">37</span><span class="NLM_x">, </span> <span class="NLM_fpage">46</span><span class="NLM_x">–</span> <span class="NLM_lpage">56</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2010&pages=46-56&author=L.+Fritschauthor=P.+Robinauthor=J.+R.+Mathieuauthor=M.+Souidiauthor=H.+Hinauxauthor=C.+Rougeulleauthor=A.+Harel-Bellanauthor=M.+Ameyar-Zazouaauthor=S.+Ait-Si-Ali&title=A+subset+of+the+histone+H3+lysine+9+methyltransferases+Suv39h1%2C+G9a%2C+GLP%2C+and+SETDB1+participate+in+a+multimeric+complex"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFritsch%26aufirst%3DL.%26aulast%3DRobin%26aufirst%3DP.%26aulast%3DMathieu%26aufirst%3DJ.%2BR.%26aulast%3DSouidi%26aufirst%3DM.%26aulast%3DHinaux%26aufirst%3DH.%26aulast%3DRougeulle%26aufirst%3DC.%26aulast%3DHarel-Bellan%26aufirst%3DA.%26aulast%3DAmeyar-Zazoua%26aufirst%3DM.%26aulast%3DAit-Si-Ali%26aufirst%3DS.%26atitle%3DA%2520subset%2520of%2520the%2520histone%2520H3%2520lysine%25209%2520methyltransferases%2520Suv39h1%252C%2520G9a%252C%2520GLP%252C%2520and%2520SETDB1%2520participate%2520in%2520a%2520multimeric%2520complex%26jtitle%3DMol.%2520Cell%26date%3D2010%26volume%3D37%26spage%3D46%26epage%3D56" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Frye, S. V.</span><span> </span><span class="NLM_article-title">The art of the chemical probe</span> <span class="citation_source-journal">Nature Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">159</span><span class="NLM_x">–</span> <span class="NLM_lpage">161</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Fjm200903z&amp;key=10.1038%2Fnchembio.296" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Fjm200903z&amp;key=20154659" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Fjm200903z&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhvFGntbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=159-161&author=S.+V.+Frye&title=The+art+of+the+chemical+probe"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">The art of the chemical probe</span></div><div class="casAuthors">Frye, Stephen V.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">159-161</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Chem. biologists frequently aim to create small-mol. probes that interact with a specific protein in vitro in order to explore the role of the protein in a broader biol. context (cells or organisms), but a common understanding of what makes a high-quality probe is lacking.  Here I propose a set of principles to guide probe qualification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7vDfsTLhqWrVg90H21EOLACvtfcHk0lhLXzkmwjt-rg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhvFGntbg%253D&md5=b7b48c591a611b8686e11b3fc4f1b85c</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.296&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.296%26sid%3Dliteratum%253Aachs%26aulast%3DFrye%26aufirst%3DS.%2BV.%26atitle%3DThe%2520art%2520of%2520the%2520chemical%2520probe%26jtitle%3DNature%2520Chem.%2520Biol.%26date%3D2010%26volume%3D6%26spage%3D159%26epage%3D161" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Cole, P. A.</span><span> </span><span class="NLM_article-title">Chemical probes for histone-modifying enzymes</span> <span class="citation_source-journal">Nature Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">590</span><span class="NLM_x">–</span> <span class="NLM_lpage">597</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Fjm200903z&amp;key=10.1038%2Fnchembio.111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Fjm200903z&amp;key=18800048" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Fjm200903z&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtFamtbrL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2008&pages=590-597&author=P.+A.+Cole&title=Chemical+probes+for+histone-modifying+enzymes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical probes for histone-modifying enzymes</span></div><div class="casAuthors">Cole, Philip A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">590-597</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The histone-modifying enzymes that catalyze reversible lysine acetylation and methylation are central to the epigenetic regulation of chromatin remodeling.  From the early discovery of histone deacetylase inhibitors to the more recent identification of histone demethylase blockers, chem. approaches offer increasingly sophisticated tools for the investigation of the structure and function of these lysine-modifying enzymes.  This review summarizes progress to date on compds. identified from screens or by design that can modulate the activity of classical histone deacetylases, sirtuins, histone acetyltransferases, histone methyltransferases and histone demethylases.  We highlight applications of compds. to mechanistic and functional studies involving these enzymes and discuss future challenges regarding target specificity and general utility.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJZ5RKIrQsO7Vg90H21EOLACvtfcHk0lighroBawRNkg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtFamtbrL&md5=71fe323c232af42b67a7dfc134c4d49b</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.111%26sid%3Dliteratum%253Aachs%26aulast%3DCole%26aufirst%3DP.%2BA.%26atitle%3DChemical%2520probes%2520for%2520histone-modifying%2520enzymes%26jtitle%3DNature%2520Chem.%2520Biol.%26date%3D2008%26volume%3D4%26spage%3D590%26epage%3D597" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heightman, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Targeting Methyl Lysine</span> <span class="citation_source-journal">Annu. Rep. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">329</span><span class="NLM_x">–</span> <span class="NLM_lpage">343</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2010&pages=329-343&author=S.+V.+Fryeauthor=T.+Heightmanauthor=J.+Jin&title=Targeting+Methyl+Lysine"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DHeightman%26aufirst%3DT.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DTargeting%2520Methyl%2520Lysine%26jtitle%3DAnnu.%2520Rep.%2520Med.%2520Chem.%26date%3D2010%26volume%3D45%26spage%3D329%26epage%3D343" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Yost, J. M.; Korboukh, I.; Liu, F.; Gao, C.; Jin, J.</span><span> </span><span class="NLM_article-title">Targets in Epigenetics: Inhibiting the Methyl Writers of the Histone Code</span>. Curr. Chem. Genomics<span class="NLM_x"> </span><span class="NLM_year">2011</span><span class="NLM_x">, </span>in press. </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Fjm200903z&amp;key=21966347" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&author=J.+M.+Yost&author=I.+Korboukh&author=F.+Liu&author=C.+Gao&author=J.+Jin&title=Targets+in+Epigenetics%3A+Inhibiting+the+Methyl+Writers+of+the+Histone+Code"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DYost%26aufirst%3DJ.%2BM.%26atitle%3DTargets%2520in%2520Epigenetics%253A%2520Inhibiting%2520the%2520Methyl%2520Writers%2520of%2520the%2520Histone%2520Code%26jtitle%3DCurr.%2520Chem.%2520Genomics%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Tachibana, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sugimoto, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nozaki, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ueda, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohta, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohki, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fukuda, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takeda, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niida, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kato, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shinkai, Y.</span><span> </span><span class="NLM_article-title">G9a histone methyltransferase plays a dominant role in euchromatic histone H3 lysine 9 methylation and is essential for early embryogenesis</span> <span class="citation_source-journal">Genes Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">1779</span><span class="NLM_x">–</span> <span class="NLM_lpage">1791</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2002&pages=1779-1791&author=M.+Tachibanaauthor=K.+Sugimotoauthor=M.+Nozakiauthor=J.+Uedaauthor=T.+Ohtaauthor=M.+Ohkiauthor=M.+Fukudaauthor=N.+Takedaauthor=H.+Niidaauthor=H.+Katoauthor=Y.+Shinkai&title=G9a+histone+methyltransferase+plays+a+dominant+role+in+euchromatic+histone+H3+lysine+9+methylation+and+is+essential+for+early+embryogenesis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTachibana%26aufirst%3DM.%26aulast%3DSugimoto%26aufirst%3DK.%26aulast%3DNozaki%26aufirst%3DM.%26aulast%3DUeda%26aufirst%3DJ.%26aulast%3DOhta%26aufirst%3DT.%26aulast%3DOhki%26aufirst%3DM.%26aulast%3DFukuda%26aufirst%3DM.%26aulast%3DTakeda%26aufirst%3DN.%26aulast%3DNiida%26aufirst%3DH.%26aulast%3DKato%26aufirst%3DH.%26aulast%3DShinkai%26aufirst%3DY.%26atitle%3DG9a%2520histone%2520methyltransferase%2520plays%2520a%2520dominant%2520role%2520in%2520euchromatic%2520histone%2520H3%2520lysine%25209%2520methylation%2520and%2520is%2520essential%2520for%2520early%2520embryogenesis%26jtitle%3DGenes%2520Dev.%26date%3D2002%26volume%3D16%26spage%3D1779%26epage%3D1791" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Trojer, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yonezawa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmidt, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenuwein, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinberg, D.</span><span> </span><span class="NLM_article-title">Dynamic Histone H1 Isotype 4 Methylation and Demethylation by Histone Lysine Methyltransferase G9a/KMT1C and the Jumonji Domain-Containing JMJD2/KDM4 Proteins</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">284</span><span class="NLM_x">, </span> <span class="NLM_fpage">8395</span><span class="NLM_x">–</span> <span class="NLM_lpage">8405</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=8395-8405&author=P.+Trojerauthor=J.+Zhangauthor=M.+Yonezawaauthor=A.+Schmidtauthor=H.+Zhengauthor=T.+Jenuweinauthor=D.+Reinberg&title=Dynamic+Histone+H1+Isotype+4+Methylation+and+Demethylation+by+Histone+Lysine+Methyltransferase+G9a%2FKMT1C+and+the+Jumonji+Domain-Containing+JMJD2%2FKDM4+Proteins"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTrojer%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DYonezawa%26aufirst%3DM.%26aulast%3DSchmidt%26aufirst%3DA.%26aulast%3DZheng%26aufirst%3DH.%26aulast%3DJenuwein%26aufirst%3DT.%26aulast%3DReinberg%26aufirst%3DD.%26atitle%3DDynamic%2520Histone%2520H1%2520Isotype%25204%2520Methylation%2520and%2520Demethylation%2520by%2520Histone%2520Lysine%2520Methyltransferase%2520G9a%252FKMT1C%2520and%2520the%2520Jumonji%2520Domain-Containing%2520JMJD2%252FKDM4%2520Proteins%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2009%26volume%3D284%26spage%3D8395%26epage%3D8405" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Sampath, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marazzi, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yap, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krutchinsky, A. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mecklenbrauker, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Viale, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rudensky, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chait, B. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tarakhovsky, A.</span><span> </span><span class="NLM_article-title">Methylation of a histone mimic within the histone methyltransferase G9a regulates protein complex assembly</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">596</span><span class="NLM_x">–</span> <span class="NLM_lpage">608</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Fjm200903z&amp;key=10.1016%2Fj.molcel.2007.06.026" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Fjm200903z&amp;key=17707231" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Fjm200903z&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVSmtL3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2007&pages=596-608&author=S.+C.+Sampathauthor=I.+Marazziauthor=K.+L.+Yapauthor=A.+N.+Krutchinskyauthor=I.+Mecklenbraukerauthor=A.+Vialeauthor=E.+Rudenskyauthor=M.+M.+Zhouauthor=B.+T.+Chaitauthor=A.+Tarakhovsky&title=Methylation+of+a+histone+mimic+within+the+histone+methyltransferase+G9a+regulates+protein+complex+assembly"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Methylation of a histone mimic within the histone methyltransferase G9a regulates protein complex assembly</span></div><div class="casAuthors">Sampath, Srihari C.; Marazzi, Ivan; Yap, Kyoko L.; Sampath, Srinath C.; Krutchinsky, Andrew N.; Mecklenbrauker, Ingrid; Viale, Agnes; Rudensky, Eugene; Zhou, Ming-Ming; Chait, Brian T.; Tarakhovsky, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">596-608</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Epigenetic gene silencing in eukaryotes is regulated in part by lysine methylation of the core histone proteins.  While histone lysine methylation is known to control gene expression through the recruitment of modification-specific effector proteins, it remains unknown whether nonhistone chromatin proteins are targets for similar modification-recognition systems.  Here we show that the histone H3 methyltransferase G9a contains a conserved methylation motif with marked sequence similarity to H3 itself.  As with methylation of H3 lysine 9, autocatalytic G9a methylation is necessary and sufficient to mediate in vivo interaction with the epigenetic regulator heterochromatin protein 1 (HP1), and this methyl-dependent interaction can be reversed by adjacent G9a phosphorylation.  NMR anal. indicates that the HP1 chromodomain recognizes methyl-G9a through a binding mode similar to that used in recognition of methyl-H3K9, demonstrating that the chromodomain functions as a generalized methyl-lysine binding module.  These data reveal histone-like modification cassettes-or "histone mimics"-as a distinct class of nonhistone methylation targets and directly extend the principles of the histone code to the regulation of nonhistone proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFIcB4m0CyVbVg90H21EOLACvtfcHk0lipDleGGQtfJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVSmtL3J&md5=3002c1fd27ba6061da0cef626680b504</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2007.06.026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2007.06.026%26sid%3Dliteratum%253Aachs%26aulast%3DSampath%26aufirst%3DS.%2BC.%26aulast%3DMarazzi%26aufirst%3DI.%26aulast%3DYap%26aufirst%3DK.%2BL.%26aulast%3DKrutchinsky%26aufirst%3DA.%2BN.%26aulast%3DMecklenbrauker%26aufirst%3DI.%26aulast%3DViale%26aufirst%3DA.%26aulast%3DRudensky%26aufirst%3DE.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26aulast%3DChait%26aufirst%3DB.%2BT.%26aulast%3DTarakhovsky%26aufirst%3DA.%26atitle%3DMethylation%2520of%2520a%2520histone%2520mimic%2520within%2520the%2520histone%2520methyltransferase%2520G9a%2520regulates%2520protein%2520complex%2520assembly%26jtitle%3DMol.%2520Cell%26date%3D2007%26volume%3D27%26spage%3D596%26epage%3D608" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Huang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dorsey, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chuikov, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenuwein, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinberg, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berger, S. L.</span><span> </span><span class="NLM_article-title">G9A and GLP methylate lysine 373 in the tumor suppressor p53</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">285</span><span class="NLM_x">, </span> <span class="NLM_fpage">9636</span><span class="NLM_x">–</span> <span class="NLM_lpage">9641</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Fjm200903z&amp;key=10.1074%2Fjbc.M109.062588" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Fjm200903z&amp;key=20118233" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Fjm200903z&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjsFeht7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=9636-9641&author=J.+Huangauthor=J.+Dorseyauthor=S.+Chuikovauthor=X.+Zhangauthor=T.+Jenuweinauthor=D.+Reinbergauthor=S.+L.+Berger&title=G9A+and+GLP+methylate+lysine+373+in+the+tumor+suppressor+p53"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">G9a and Glp Methylate Lysine 373 in the Tumor Suppressor p53</span></div><div class="casAuthors">Huang, Jing; Dorsey, Jean; Chuikov, Sergei; Zhang, Xinyue; Jenuwein, Thomas; Reinberg, Danny; Berger, Shelley L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">285</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">9636-9641</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The tumor suppressor p53 is regulated by numerous post-translational modifications.  Lysine methylation has recently emerged as a key post-translational modification that alters the activity of p53.  Here, we describe a novel lysine methylation site in p53 that is carried out by two homologous histone methyltransferases, G9a and Glp.  G9a and Glp specifically methylate p53 at Lys373, resulting mainly in dimethylation.  During DNA damage, the overall level of p53 modified at Lys373me2 does not increase, despite the dramatic increase in total p53, indicating that Lys373me2 correlates with inactive p53.  Further, redn. of G9a and/or Glp levels leads to a larger population of apoptotic cells.  Examn. of the Oncomine data base shows that G9a and Glp are overexpressed in various cancers compared with corresponding normal tissues, suggesting that they are putative oncogenes.  These data reveal a new methylation site within p53 mediated by the methylases G9a and Glp and indicate that G9a is a potential inhibitory target for cancer treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozJXSsZlEE9LVg90H21EOLACvtfcHk0liRJi0jWqPv_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjsFeht7w%253D&md5=81adc2281f6344caf9a46ebcdf1c61e6</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.062588&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.062588%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DDorsey%26aufirst%3DJ.%26aulast%3DChuikov%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DJenuwein%26aufirst%3DT.%26aulast%3DReinberg%26aufirst%3DD.%26aulast%3DBerger%26aufirst%3DS.%2BL.%26atitle%3DG9A%2520and%2520GLP%2520methylate%2520lysine%2520373%2520in%2520the%2520tumor%2520suppressor%2520p53%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D9636%26epage%3D9641" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Kondo, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ahmed, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boumber, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sekido, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haddad, B. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Issa, J. P.</span><span> </span><span class="NLM_article-title">Downregulation of histone H3 lysine 9 methyltransferase G9a induces centrosome disruption and chromosome instability in cancer cells</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">e2037</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Fjm200903z&amp;key=10.1371%2Fjournal.pone.0002037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Fjm200903z&amp;key=18446223" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Fjm200903z&amp;key=1%3ACAS%3A280%3ADC%252BD1c3otlyhtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2008&pages=e2037&author=Y.+Kondoauthor=L.+Shenauthor=S.+Ahmedauthor=Y.+Boumberauthor=Y.+Sekidoauthor=B.+R.+Haddadauthor=J.+P.+Issa&title=Downregulation+of+histone+H3+lysine+9+methyltransferase+G9a+induces+centrosome+disruption+and+chromosome+instability+in+cancer+cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Downregulation of histone H3 lysine 9 methyltransferase G9a induces centrosome disruption and chromosome instability in cancer cells</span></div><div class="casAuthors">Kondo Yutaka; Shen Lanlan; Ahmed Saira; Boumber Yanis; Sekido Yoshitaka; Haddad Bassem R; Issa Jean-Pierre J</div><div class="citationInfo"><span class="NLM_cas:title">PloS one</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">e2037</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Modifications of the histone amino-terminal tails affect access of regulatory factors and complexes to chromatin and thereby influence biological processes.  Cancer cells are characterized by prominent epigenetic dysregulation, including histone modifications.  However, the functional roles of the histone methyltransferases (HMT) in cancer remain unclear.  METHODOLOGY/PRINCIPAL FINDINGS:  We studied RNAi-based inhibition (knockdown, KD) of 2 different H3K9 HMTs, SUV39H1 and G9a.  Knockdown of the 2 HMTs in PC3 cancer cell line markedly inhibited cell growth and caused profound morphological changes with loss of telomerase activity and shortened telomeres.  SUV39H1 KD cells showed substantial increase in G2/M fraction.  G9a KD cells showed increased DNA content (1.7-fold in 2 independent clones) compared with FACS analyses to control.  Karyotype analyses showed that this was due to an increased number of chromosomes (from 61 to 102) in G9a KD cells compared to parental PC3.  Intriguingly, we found abnormal centrosome morphology and number in about 25% of the G9a KD cells, while centrosomes were morphologically normal in control cells.  Microarray analyses after KD of SUV39H1 or G9a showed very few genes up-regulated among the 39,000 genes.  The silenced tumor-suppressor genes p16 and RASSF1A were not activated in KD cells.  CONCLUSIONS/SIGNIFICANCE:  These data suggest that the 2 HMTs, SUV39H1 and G9a are required to perpetuate the malignant phenotype.  Furthermore, G9a plays a critical role in regulating centrosome duplication presumably through chromatin structure rather than through affecting gene expression in cancer cells.  Targeting these histone methyltransferases may be of therapeutic benefit in cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQhnx1F8nyNwDEwO064Z95rfW6udTcc2eaTde_YNXZROrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1c3otlyhtw%253D%253D&md5=22936eeeffa4dc29e7254dfc546ebcfd</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0002037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0002037%26sid%3Dliteratum%253Aachs%26aulast%3DKondo%26aufirst%3DY.%26aulast%3DShen%26aufirst%3DL.%26aulast%3DAhmed%26aufirst%3DS.%26aulast%3DBoumber%26aufirst%3DY.%26aulast%3DSekido%26aufirst%3DY.%26aulast%3DHaddad%26aufirst%3DB.%2BR.%26aulast%3DIssa%26aufirst%3DJ.%2BP.%26atitle%3DDownregulation%2520of%2520histone%2520H3%2520lysine%25209%2520methyltransferase%2520G9a%2520induces%2520centrosome%2520disruption%2520and%2520chromosome%2520instability%2520in%2520cancer%2520cells%26jtitle%3DPLoS%2520One%26date%3D2008%26volume%3D3%26spage%3De2037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Kondo, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurokawa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Masuko, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanaka, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kato, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mizuno, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yokoe, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sugauchi, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirashima, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orito, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Osada, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ueda, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Issa, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sekido, Y.</span><span> </span><span class="NLM_article-title">Alterations of DNA methylation and histone modifications contribute to gene silencing in hepatocellular carcinomas</span> <span class="citation_source-journal">Hepatol. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">37</span><span class="NLM_x">, </span> <span class="NLM_fpage">974</span><span class="NLM_x">–</span> <span class="NLM_lpage">983</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Fjm200903z&amp;key=10.1111%2Fj.1872-034X.2007.00141.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Fjm200903z&amp;key=17584191" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Fjm200903z&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlCns7fK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2007&pages=974-983&author=Y.+Kondoauthor=L.+Shenauthor=S.+Suzukiauthor=T.+Kurokawaauthor=K.+Masukoauthor=Y.+Tanakaauthor=H.+Katoauthor=Y.+Mizunoauthor=M.+Yokoeauthor=F.+Sugauchiauthor=N.+Hirashimaauthor=E.+Oritoauthor=H.+Osadaauthor=R.+Uedaauthor=Y.+Guoauthor=X.+Chenauthor=J.+P.+Issaauthor=Y.+Sekido&title=Alterations+of+DNA+methylation+and+histone+modifications+contribute+to+gene+silencing+in+hepatocellular+carcinomas"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Alterations of DNA methylation and histone modifications contribute to gene silencing in hepatocellular carcinomas</span></div><div class="casAuthors">Kondo, Yutaka; Shen, Lanlan; Suzuki, Seiji; Kurokawa, Tsuyoshi; Masuko, Kazuo; Tanaka, Yasuhito; Kato, Hideaki; Mizuno, Yoshiki; Yokoe, Masamichi; Sugauchi, Fuminaka; Hirashima, Noboru; Orito, Etsuro; Osada, Hirotaka; Ueda, Ryuzo; Guo, Yi; Chen, Xinli; Issa, Jean-Pierre J.; Sekido, Yoshitaka</div><div class="citationInfo"><span class="NLM_cas:title">Hepatology Research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">974-983</span>CODEN:
                <span class="NLM_cas:coden">HPRSFM</span>;
        ISSN:<span class="NLM_cas:issn">1386-6346</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Asia Pty Ltd.</span>)
        </div><div class="casAbstract">Aim: The aim of the present study was to examine DNA methylation and histone modification changes in hepatocellular carcinomas (HCC).  Methods: DNA methylation in the P16, RASSF1a, progesterone receptor (PGR) and estrogen receptor α (ERα) promoters was detd. by quant. bisulfite-pyrosequencing technique in HCC patients.  Histone H3-lysine (K) 4, H3-K9 and H3-K27 modifications in all these four genes were examd. by chromatin immunopptn. (ChIP) assay in HCC cell lines.  Expression of two DNA methyltransferases (DNMT1 and DNMT3b) and three histone methyltransferases (SUV39H1, G9a and EZH2) in HCC patients was measured by real-time polymerase chain reaction.  Results: Aberrant DNA methylation was detected in all the HCC.  Patients with DNA methylation in the RASSF1a, PGR and ERα promoters in cancers also had substantial DNA methylation in their non-cancerous liver tissues, whereas DNA methylation in the P16 promoter was cancer specific.  Epigenetic states in HCC cell lines showed that silencing of P16 and RASSF1a depended on DNA methylation and histone H3-K9 methylation.  However, silencing of the PGR and ERα genes was more closely related to H3-K27 methylation rather than DNA methylation.  Consistent with the alteration of histone status, higher expression of G9a and EZH2 was found in HCC than in non-cancerous liver tissues (P < 0.01).  Conclusion: These data suggest that multiple epigenetic silencing mechanisms are inappropriately active in HCC cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwkWfvYpsUX7Vg90H21EOLACvtfcHk0lgW96ux4SyPYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlCns7fK&md5=afc7013e4adfd4dd30d0b0e07ce8d9ef</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1111%2Fj.1872-034X.2007.00141.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1872-034X.2007.00141.x%26sid%3Dliteratum%253Aachs%26aulast%3DKondo%26aufirst%3DY.%26aulast%3DShen%26aufirst%3DL.%26aulast%3DSuzuki%26aufirst%3DS.%26aulast%3DKurokawa%26aufirst%3DT.%26aulast%3DMasuko%26aufirst%3DK.%26aulast%3DTanaka%26aufirst%3DY.%26aulast%3DKato%26aufirst%3DH.%26aulast%3DMizuno%26aufirst%3DY.%26aulast%3DYokoe%26aufirst%3DM.%26aulast%3DSugauchi%26aufirst%3DF.%26aulast%3DHirashima%26aufirst%3DN.%26aulast%3DOrito%26aufirst%3DE.%26aulast%3DOsada%26aufirst%3DH.%26aulast%3DUeda%26aufirst%3DR.%26aulast%3DGuo%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DIssa%26aufirst%3DJ.%2BP.%26aulast%3DSekido%26aufirst%3DY.%26atitle%3DAlterations%2520of%2520DNA%2520methylation%2520and%2520histone%2520modifications%2520contribute%2520to%2520gene%2520silencing%2520in%2520hepatocellular%2520carcinomas%26jtitle%3DHepatol.%2520Res.%26date%3D2007%26volume%3D37%26spage%3D974%26epage%3D983" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Watanabe, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soejima, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yasuda, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawada, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakachi, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoda, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Naoki, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ishizaka, A.</span><span> </span><span class="NLM_article-title">Deregulation of histone lysine methyltransferases contributes to oncogenic transformation of human bronchoepithelial cells</span> <span class="citation_source-journal">Cancer Cell Int.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">15</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Fjm200903z&amp;key=10.1186%2F1475-2867-8-15" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Fjm200903z&amp;key=18980680" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Fjm200903z&amp;key=1%3ACAS%3A280%3ADC%252BD1cjltFSntg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2008&pages=15&author=H.+Watanabeauthor=K.+Soejimaauthor=H.+Yasudaauthor=I.+Kawadaauthor=I.+Nakachiauthor=S.+Yodaauthor=K.+Naokiauthor=A.+Ishizaka&title=Deregulation+of+histone+lysine+methyltransferases+contributes+to+oncogenic+transformation+of+human+bronchoepithelial+cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Deregulation of histone lysine methyltransferases contributes to oncogenic transformation of human bronchoepithelial cells</span></div><div class="casAuthors">Watanabe Hideo; Soejima Kenzo; Yasuda Hiroyuki; Kawada Ichiro; Nakachi Ichiro; Yoda Satoshi; Naoki Katsuhiko; Ishizaka Akitoshi</div><div class="citationInfo"><span class="NLM_cas:title">Cancer cell international</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">15</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Alterations in the processing of the genetic information in carcinogenesis result from stable genetic mutations or epigenetic modifications.  It is becoming clear that nucleosomal histones are central to proper gene expression and that aberrant DNA methylation of genes and histone methylation plays important roles in tumor progression.  To date, several histone lysine methyltransferases (HKMTs) have been identified and histone lysine methylation is now considered to be a critical regulator of transcription.  However, still relatively little is known about the role of HKMTs in tumorigenesis.  RESULTS:  We observed differential HKMT expression in a lung cancer model in which normal human bronchial epithelial (NHBE) cells expressing telomerase, SV40 large T antigen, and Ras were immortal, formed colonies in soft agar, and expressed specific HKMTs for H3 lysine 9 and 27 residues but not for H3 lysine 4 residue.  Modifications in the H3 tails affect the binding of proteins to the histone tails and regulate protein function and the position of lysine methylation marks a gene to be either activated or repressed.  In the present study, suppression by siRNA of HKMTs (EZH2, G9A, SETDB1 and SUV39H1) that are over-expressed in immortalized and transformed cells lead to reduced cell proliferation and much less anchorage-independent colony growth.  We also found that the suppression of H3-K9, G9A and SUV39H1 induced apoptosis and the suppression of H3-K27, EZH2 caused G1 arrest.  CONCLUSION:  Our results indicate the potential of these HKMTs in addition to the other targets for epigenetics such as DNMTs and HDACs to be interesting therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ8X-3k_fB763y-c-E1v-fmfW6udTcc2eY3DOYsIMDsAbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1cjltFSntg%253D%253D&md5=ac77cc1b0cca2a4d8bf111fb6463ee7e</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1186%2F1475-2867-8-15&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1475-2867-8-15%26sid%3Dliteratum%253Aachs%26aulast%3DWatanabe%26aufirst%3DH.%26aulast%3DSoejima%26aufirst%3DK.%26aulast%3DYasuda%26aufirst%3DH.%26aulast%3DKawada%26aufirst%3DI.%26aulast%3DNakachi%26aufirst%3DI.%26aulast%3DYoda%26aufirst%3DS.%26aulast%3DNaoki%26aufirst%3DK.%26aulast%3DIshizaka%26aufirst%3DA.%26atitle%3DDeregulation%2520of%2520histone%2520lysine%2520methyltransferases%2520contributes%2520to%2520oncogenic%2520transformation%2520of%2520human%2520bronchoepithelial%2520cells%26jtitle%3DCancer%2520Cell%2520Int.%26date%3D2008%26volume%3D8%26spage%3D15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">McGarvey, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fahrner, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greene, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martens, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenuwein, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baylin, S. B.</span><span> </span><span class="NLM_article-title">Silenced tumor suppressor genes reactivated by DNA demethylation do not return to a fully euchromatic chromatin state</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">3541</span><span class="NLM_x">–</span> <span class="NLM_lpage">3549</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Fjm200903z&amp;key=10.1158%2F0008-5472.CAN-05-2481" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Fjm200903z&amp;key=16585178" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Fjm200903z&amp;key=1%3ACAS%3A528%3ADC%252BD28XjtVOrsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2006&pages=3541-3549&author=K.+M.+McGarveyauthor=J.+A.+Fahrnerauthor=E.+Greeneauthor=J.+Martensauthor=T.+Jenuweinauthor=S.+B.+Baylin&title=Silenced+tumor+suppressor+genes+reactivated+by+DNA+demethylation+do+not+return+to+a+fully+euchromatic+chromatin+state"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Silenced Tumor Suppressor Genes Reactivated by DNA Demethylation Do Not Return to a Fully Euchromatic Chromatin State</span></div><div class="casAuthors">McGarvey, Kelly M.; Fahrner, Jill A.; Greene, Eriko; Martens, Joost; Jenuwein, Thomas; Baylin, Stephen B.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3541-3549</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Histone H3 lysine 9 (H3K9) and lysine 27 (H3K27) trimethylation are properties of stably silenced heterochromatin whereas H3K9 dimethylation (H3K9me2) is important for euchromatic gene repression.  In colorectal cancer cells, all of these marks, as well as the key enzymes which establish them, surround the hMLH1 promoter when it is DNA hypermethylated and aberrantly silenced, but are absent when the gene is unmethylated and fully expressed in a euchromatic state.  When the aberrantly silenced gene is DNA demethylated and reexpressed following 5-aza-2'-deoxycytidine treatment, H3K9me1 and H3K9me2 are the only silencing marks that are lost.  A series of other silenced and DNA hypermethylated gene promoters behave identically even when the genes are chronically DNA demethylated and reexpressed after genetic knockout of DNA methyltransferases.  Our data indicate that when transcription of DNA hypermethylated genes is activated in cancer cells, their promoters remain in an environment with certain heterochromatic characteristics.  This finding has important implications for the translational goal of reactivating aberrantly silenced cancer genes as a therapeutic maneuver.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEjoUm5wwMFbVg90H21EOLACvtfcHk0lj7UDhrHvgsmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XjtVOrsro%253D&md5=73b1830a5f7f839d53d5539c6e707462</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-05-2481&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-05-2481%26sid%3Dliteratum%253Aachs%26aulast%3DMcGarvey%26aufirst%3DK.%2BM.%26aulast%3DFahrner%26aufirst%3DJ.%2BA.%26aulast%3DGreene%26aufirst%3DE.%26aulast%3DMartens%26aufirst%3DJ.%26aulast%3DJenuwein%26aufirst%3DT.%26aulast%3DBaylin%26aufirst%3DS.%2BB.%26atitle%3DSilenced%2520tumor%2520suppressor%2520genes%2520reactivated%2520by%2520DNA%2520demethylation%2520do%2520not%2520return%2520to%2520a%2520fully%2520euchromatic%2520chromatin%2520state%26jtitle%3DCancer%2520Res.%26date%3D2006%26volume%3D66%26spage%3D3541%26epage%3D3549" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Goyama, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nitta, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshino, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kako, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watanabe-Okochi, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shimabe, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Imai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takahashi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurokawa, M.</span><span> </span><span class="NLM_article-title">EVI-1 interacts with histone methyltransferases SUV39H1 and G9a for transcriptional repression and bone marrow immortalization</span> <span class="citation_source-journal">Leukemia</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">81</span><span class="NLM_x">–</span> <span class="NLM_lpage">88</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2010&pages=81-88&author=S.+Goyamaauthor=E.+Nittaauthor=T.+Yoshinoauthor=S.+Kakoauthor=N.+Watanabe-Okochiauthor=M.+Shimabeauthor=Y.+Imaiauthor=K.+Takahashiauthor=M.+Kurokawa&title=EVI-1+interacts+with+histone+methyltransferases+SUV39H1+and+G9a+for+transcriptional+repression+and+bone+marrow+immortalization"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGoyama%26aufirst%3DS.%26aulast%3DNitta%26aufirst%3DE.%26aulast%3DYoshino%26aufirst%3DT.%26aulast%3DKako%26aufirst%3DS.%26aulast%3DWatanabe-Okochi%26aufirst%3DN.%26aulast%3DShimabe%26aufirst%3DM.%26aulast%3DImai%26aufirst%3DY.%26aulast%3DTakahashi%26aufirst%3DK.%26aulast%3DKurokawa%26aufirst%3DM.%26atitle%3DEVI-1%2520interacts%2520with%2520histone%2520methyltransferases%2520SUV39H1%2520and%2520G9a%2520for%2520transcriptional%2520repression%2520and%2520bone%2520marrow%2520immortalization%26jtitle%3DLeukemia%26date%3D2010%26volume%3D24%26spage%3D81%26epage%3D88" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Maze, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Covington, H. E.,  III</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dietz, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LaPlant, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Renthal, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Russo, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mechanic, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mouzon, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neve, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haggarty, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ren, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sampath, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurd, Y. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greengard, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tarakhovsky, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schaefer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nestler, E. J.</span><span> </span><span class="NLM_article-title">Essential role of the histone methyltransferase G9a in cocaine-induced plasticity</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">327</span><span class="NLM_x">, </span> <span class="NLM_fpage">213</span><span class="NLM_x">–</span> <span class="NLM_lpage">216</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Fjm200903z&amp;key=10.1126%2Fscience.1179438" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Fjm200903z&amp;key=20056891" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Fjm200903z&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1egtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=327&publication_year=2010&pages=213-216&author=I.+Mazeauthor=H.+E.+Covingtonauthor=D.+M.+Dietzauthor=Q.+LaPlantauthor=W.+Renthalauthor=S.+J.+Russoauthor=M.+Mechanicauthor=E.+Mouzonauthor=R.+L.+Neveauthor=S.+J.+Haggartyauthor=Y.+Renauthor=S.+C.+Sampathauthor=Y.+L.+Hurdauthor=P.+Greengardauthor=A.+Tarakhovskyauthor=A.+Schaeferauthor=E.+J.+Nestler&title=Essential+role+of+the+histone+methyltransferase+G9a+in+cocaine-induced+plasticity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Essential Role of the Histone Methyltransferase G9a in Cocaine-Induced Plasticity</span></div><div class="casAuthors">Maze, Ian; Covington, Herbert E., III; Dietz, David M.; LaPlant, Quincey; Renthal, William; Russo, Scott J.; Mechanic, Max; Mouzon, Ezekiell; Neve, Rachael L.; Haggarty, Stephen J.; Ren, Yanhua; Sampath, Srihari C.; Hurd, Yasmin L.; Greengard, Paul; Tarakhovsky, Alexander; Schaefer, Anne; Nestler, Eric J.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">327</span>
        (<span class="NLM_cas:issue">5962</span>),
    <span class="NLM_cas:pages">213-216</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Cocaine-induced alterations in gene expression cause changes in neuronal morphol. and behavior that may underlie cocaine addiction.  In mice, we identified an essential role for histone 3 lysine 9 (H3K9) dimethylation and the lysine dimethyltransferase G9a in cocaine-induced structural and behavioral plasticity.  Repeated cocaine administration reduced global levels of H3K9 dimethylation in the nucleus accumbens.  This redn. in histone methylation was mediated through the repression of G9a in this brain region, which was regulated by the cocaine-induced transcription factor ΔFosB.  Using conditional mutagenesis and viral-mediated gene transfer, we found that G9a down-regulation increased the dendritic spine plasticity of nucleus accumbens neurons and enhanced the preference for cocaine, thereby establishing a crucial role for histone methylation in the long-term actions of cocaine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8SPD826wEGrVg90H21EOLACvtfcHk0lj7UDhrHvgsmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1egtA%253D%253D&md5=a8347500f81862078787e50fff53b6b9</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1126%2Fscience.1179438&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1179438%26sid%3Dliteratum%253Aachs%26aulast%3DMaze%26aufirst%3DI.%26aulast%3DCovington%26aufirst%3DH.%2BE.%26aulast%3DDietz%26aufirst%3DD.%2BM.%26aulast%3DLaPlant%26aufirst%3DQ.%26aulast%3DRenthal%26aufirst%3DW.%26aulast%3DRusso%26aufirst%3DS.%2BJ.%26aulast%3DMechanic%26aufirst%3DM.%26aulast%3DMouzon%26aufirst%3DE.%26aulast%3DNeve%26aufirst%3DR.%2BL.%26aulast%3DHaggarty%26aufirst%3DS.%2BJ.%26aulast%3DRen%26aufirst%3DY.%26aulast%3DSampath%26aufirst%3DS.%2BC.%26aulast%3DHurd%26aufirst%3DY.%2BL.%26aulast%3DGreengard%26aufirst%3DP.%26aulast%3DTarakhovsky%26aufirst%3DA.%26aulast%3DSchaefer%26aufirst%3DA.%26aulast%3DNestler%26aufirst%3DE.%2BJ.%26atitle%3DEssential%2520role%2520of%2520the%2520histone%2520methyltransferase%2520G9a%2520in%2520cocaine-induced%2520plasticity%26jtitle%3DScience%26date%3D2010%26volume%3D327%26spage%3D213%26epage%3D216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Schaefer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sampath, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Intrator, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Min, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gertler, T. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Surmeier, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tarakhovsky, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greengard, P.</span><span> </span><span class="NLM_article-title">Control of Cognition and Adaptive Behavior by the GLP/G9a Epigenetic Suppressor Complex</span> <span class="citation_source-journal">Neuron</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">64</span><span class="NLM_x">, </span> <span class="NLM_fpage">678</span><span class="NLM_x">–</span> <span class="NLM_lpage">691</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Fjm200903z&amp;key=10.1016%2Fj.neuron.2009.11.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Fjm200903z&amp;key=20005824" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Fjm200903z&amp;key=1%3ACAS%3A528%3ADC%252BC3cXks1Cls7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2009&pages=678-691&author=A.+Schaeferauthor=S.+C.+Sampathauthor=A.+Intratorauthor=A.+Minauthor=T.+S.+Gertlerauthor=D.+J.+Surmeierauthor=A.+Tarakhovskyauthor=P.+Greengard&title=Control+of+Cognition+and+Adaptive+Behavior+by+the+GLP%2FG9a+Epigenetic+Suppressor+Complex"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Control of cognition and adaptive behavior by the GLP/G9a epigenetic suppressor complex</span></div><div class="casAuthors">Schaefer, Anne; Sampath, Srihari C.; Intrator, Adam; Min, Alice; Gertler, Tracy S.; Surmeier, D. James; Tarakhovsky, Alexander; Greengard, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Neuron</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">678-691</span>CODEN:
                <span class="NLM_cas:coden">NERNET</span>;
        ISSN:<span class="NLM_cas:issn">0896-6273</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The genetic basis of cognition and behavioral adaptation to the environment remains poorly understood.  Here we demonstrate that the histone methyltransferase complex GLP/G9a controls cognition and adaptive responses in a region-specific fashion in the adult brain.  Using conditional mutagenesis in mice, we show that postnatal, neuron-specific deficiency of GLP/G9a leads to derepression of numerous nonneuronal and neuron progenitor genes in adult neurons.  This transcriptional alteration is assocd. with complex behavioral abnormalities, including defects in learning, motivation, and environmental adaptation.  The behavioral changes triggered by GLP/G9a deficiency are similar to key symptoms of the human 9q34 mental retardation syndrome that is assocd. with structural alterations of the GLP/EHMT1 gene.  The likely causal role of GLP/G9a in mental retardation in mice and humans suggests a key role for the GLP/G9a-controlled histone H3K9 dimethylation in regulation of brain function through maintenance of the transcriptional homeostasis in adult neurons.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr22UcjjqkKdLVg90H21EOLACvtfcHk0lgKbIH1eisS7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXks1Cls7c%253D&md5=ece4e94ad097a8d0e67a5e8d27970e64</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.neuron.2009.11.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuron.2009.11.019%26sid%3Dliteratum%253Aachs%26aulast%3DSchaefer%26aufirst%3DA.%26aulast%3DSampath%26aufirst%3DS.%2BC.%26aulast%3DIntrator%26aufirst%3DA.%26aulast%3DMin%26aufirst%3DA.%26aulast%3DGertler%26aufirst%3DT.%2BS.%26aulast%3DSurmeier%26aufirst%3DD.%2BJ.%26aulast%3DTarakhovsky%26aufirst%3DA.%26aulast%3DGreengard%26aufirst%3DP.%26atitle%3DControl%2520of%2520Cognition%2520and%2520Adaptive%2520Behavior%2520by%2520the%2520GLP%252FG9a%2520Epigenetic%2520Suppressor%2520Complex%26jtitle%3DNeuron%26date%3D2009%26volume%3D64%26spage%3D678%26epage%3D691" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Imai, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Togami, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okamoto, T.</span><span> </span><span class="NLM_article-title">Involvement of histone H3 Lysine 9 (H3K9) methyl transferase G9a in the maintenance of HIV-1 latency and its reactivation by BIX01294</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">285</span><span class="NLM_x">, </span> <span class="NLM_fpage">16538</span><span class="NLM_x">–</span> <span class="NLM_lpage">16545</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=16538-16545&author=K.+Imaiauthor=H.+Togamiauthor=T.+Okamoto&title=Involvement+of+histone+H3+Lysine+9+%28H3K9%29+methyl+transferase+G9a+in+the+maintenance+of+HIV-1+latency+and+its+reactivation+by+BIX01294"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DImai%26aufirst%3DK.%26aulast%3DTogami%26aufirst%3DH.%26aulast%3DOkamoto%26aufirst%3DT.%26atitle%3DInvolvement%2520of%2520histone%2520H3%2520Lysine%25209%2520%2528H3K9%2529%2520methyl%2520transferase%2520G9a%2520in%2520the%2520maintenance%2520of%2520HIV-1%2520latency%2520and%2520its%2520reactivation%2520by%2520BIX01294%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D16538%26epage%3D16545" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Link, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gangisetty, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">James, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woloszynska-Read, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tachibana, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shinkai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karpf, A. R.</span><span> </span><span class="NLM_article-title">Distinct roles for histone methyltransferases G9a and GLP in cancer germ-line antigen gene regulation in human cancer cells and murine embryonic stem cells</span> <span class="citation_source-journal">Mol. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">851</span><span class="NLM_x">–</span> <span class="NLM_lpage">862</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Fjm200903z&amp;key=10.1158%2F1541-7786.MCR-08-0497" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Fjm200903z&amp;key=19531572" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Fjm200903z&amp;key=1%3ACAS%3A528%3ADC%252BD1MXns1Skt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2009&pages=851-862&author=P.+A.+Linkauthor=O.+Gangisettyauthor=S.+R.+Jamesauthor=A.+Woloszynska-Readauthor=M.+Tachibanaauthor=Y.+Shinkaiauthor=A.+R.+Karpf&title=Distinct+roles+for+histone+methyltransferases+G9a+and+GLP+in+cancer+germ-line+antigen+gene+regulation+in+human+cancer+cells+and+murine+embryonic+stem+cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Distinct Roles for Histone Methyltransferases G9a and GLP in Cancer Germ-Line Antigen Gene Regulation in Human Cancer Cells and Murine Embryonic Stem Cells</span></div><div class="casAuthors">Link, Petra A.; Gangisetty, Omkaram; James, Smitha R.; Woloszynska-Read, Anna; Tachibana, Makoto; Shinkai, Yoichi; Karpf, Adam R.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">851-862</span>CODEN:
                <span class="NLM_cas:coden">MCROC5</span>;
        ISSN:<span class="NLM_cas:issn">1541-7786</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The H3K9me2 histone methyltransferases G9a and GLP repress Mage-a class cancer germ-line (CG) antigen gene expression in murine embryonic stem (ES) cells, but the role of these enzymes in CG antigen gene regulation in human cancer cells is unknown.  Here we show that whereas independent or dual knockdown of G9a and GLP in human cancer cells leads to reduced global and CG antigen promoter-assocd. H3K9me2 levels, it does not activate CG antigen gene expression.  Moreover, CG antigen gene repression is maintained following pharmacol. targeting of G9a or treatment of G9a knockdown cells with the histone deacetylase inhibitor trichostatin A.  However, G9a knockdown cells display increased sensitivity to CG antigen gene activation mediated by the DNA methyltransferase inhibitor decitabine.  To account for these findings, we examd. DNA methylation at CG antigen gene promoters in both cell types.  We found robust DNA hypomethylation in G9a/GLP targeted murine ES cells but a lack of DNA methylation changes in G9a/GLP targeted human cancer cells; intriguingly, this distinction also extended to markers of global DNA methylation.  These data reveal that G9a/GLP is required for DNA methylation of CG antigen genes and genomic DNA in murine ES cells, but not human cancer cells, and implicate DNA methylation status as the key epigenetic mechanism involved in CG antigen gene repression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTDHnsCTv6qLVg90H21EOLACvtfcHk0lgKbIH1eisS7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXns1Skt74%253D&md5=fc1f607388e192a0768a22584f987cda</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1158%2F1541-7786.MCR-08-0497&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1541-7786.MCR-08-0497%26sid%3Dliteratum%253Aachs%26aulast%3DLink%26aufirst%3DP.%2BA.%26aulast%3DGangisetty%26aufirst%3DO.%26aulast%3DJames%26aufirst%3DS.%2BR.%26aulast%3DWoloszynska-Read%26aufirst%3DA.%26aulast%3DTachibana%26aufirst%3DM.%26aulast%3DShinkai%26aufirst%3DY.%26aulast%3DKarpf%26aufirst%3DA.%2BR.%26atitle%3DDistinct%2520roles%2520for%2520histone%2520methyltransferases%2520G9a%2520and%2520GLP%2520in%2520cancer%2520germ-line%2520antigen%2520gene%2520regulation%2520in%2520human%2520cancer%2520cells%2520and%2520murine%2520embryonic%2520stem%2520cells%26jtitle%3DMol.%2520Cancer%2520Res.%26date%3D2009%26volume%3D7%26spage%3D851%26epage%3D862" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Tachibana, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matsumura, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fukuda, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kimura, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shinkai, Y.</span><span> </span><span class="NLM_article-title">G9a/GLP complexes independently mediate H3K9 and DNA methylation to silence transcription</span> <span class="citation_source-journal">EMBO J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">2681</span><span class="NLM_x">–</span> <span class="NLM_lpage">2690</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2008&pages=2681-2690&author=M.+Tachibanaauthor=Y.+Matsumuraauthor=M.+Fukudaauthor=H.+Kimuraauthor=Y.+Shinkai&title=G9a%2FGLP+complexes+independently+mediate+H3K9+and+DNA+methylation+to+silence+transcription"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTachibana%26aufirst%3DM.%26aulast%3DMatsumura%26aufirst%3DY.%26aulast%3DFukuda%26aufirst%3DM.%26aulast%3DKimura%26aufirst%3DH.%26aulast%3DShinkai%26aufirst%3DY.%26atitle%3DG9a%252FGLP%2520complexes%2520independently%2520mediate%2520H3K9%2520and%2520DNA%2520methylation%2520to%2520silence%2520transcription%26jtitle%3DEMBO%2520J.%26date%3D2008%26volume%3D27%26spage%3D2681%26epage%3D2690" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Dong, K. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maksakova, I. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mohn, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leung, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Appanah, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, H. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lam, L. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mager, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schubeler, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tachibana, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shinkai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lorincz, M. C.</span><span> </span><span class="NLM_article-title">DNA methylation in ES cells requires the lysine methyltransferase G9a but not its catalytic activity</span> <span class="citation_source-journal">EMBO J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">2691</span><span class="NLM_x">–</span> <span class="NLM_lpage">2701</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2008&pages=2691-2701&author=K.+B.+Dongauthor=I.+A.+Maksakovaauthor=F.+Mohnauthor=D.+Leungauthor=R.+Appanahauthor=S.+Leeauthor=H.+W.+Yangauthor=L.+L.+Lamauthor=D.+L.+Magerauthor=D.+Schubelerauthor=M.+Tachibanaauthor=Y.+Shinkaiauthor=M.+C.+Lorincz&title=DNA+methylation+in+ES+cells+requires+the+lysine+methyltransferase+G9a+but+not+its+catalytic+activity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDong%26aufirst%3DK.%2BB.%26aulast%3DMaksakova%26aufirst%3DI.%2BA.%26aulast%3DMohn%26aufirst%3DF.%26aulast%3DLeung%26aufirst%3DD.%26aulast%3DAppanah%26aufirst%3DR.%26aulast%3DLee%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DH.%2BW.%26aulast%3DLam%26aufirst%3DL.%2BL.%26aulast%3DMager%26aufirst%3DD.%2BL.%26aulast%3DSchubeler%26aufirst%3DD.%26aulast%3DTachibana%26aufirst%3DM.%26aulast%3DShinkai%26aufirst%3DY.%26aulast%3DLorincz%26aufirst%3DM.%2BC.%26atitle%3DDNA%2520methylation%2520in%2520ES%2520cells%2520requires%2520the%2520lysine%2520methyltransferase%2520G9a%2520but%2520not%2520its%2520catalytic%2520activity%26jtitle%3DEMBO%2520J.%26date%3D2008%26volume%3D27%26spage%3D2691%26epage%3D2701" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Kubicek, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Sullivan, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">August, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hickey, E. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teodoro, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rea, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mechtler, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kowalski, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Homon, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kelly, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenuwein, T.</span><span> </span><span class="NLM_article-title">Reversal of H3K9me2 by a small-molecule inhibitor for the G9a histone methyltransferase</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">473</span><span class="NLM_x">–</span> <span class="NLM_lpage">481</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Fjm200903z&amp;key=10.1016%2Fj.molcel.2007.01.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Fjm200903z&amp;key=17289593" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Fjm200903z&amp;key=1%3ACAS%3A528%3ADC%252BD2sXitl2jurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2007&pages=473-481&author=S.+Kubicekauthor=R.+J.+O%E2%80%99Sullivanauthor=E.+M.+Augustauthor=E.+R.+Hickeyauthor=Q.+Zhangauthor=M.+L.+Teodoroauthor=S.+Reaauthor=K.+Mechtlerauthor=J.+A.+Kowalskiauthor=C.+A.+Homonauthor=T.+A.+Kellyauthor=T.+Jenuwein&title=Reversal+of+H3K9me2+by+a+small-molecule+inhibitor+for+the+G9a+histone+methyltransferase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Reversal of H3K9me2 by a small-molecule inhibitor for the G9a histone methyltransferase</span></div><div class="casAuthors">Kubicek, Stefan; O'Sullivan, Roderick J.; August, E. Michael; Hickey, Eugene R.; Zhang, Qiang; Teodoro, Miguel L.; Rea, Stephen; Mechtler, Karl; Kowalski, Jennifer A.; Homon, Carol Ann; Kelly, Terence A.; Jenuwein, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">473-481</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Histone lysine methylation has important roles in the organization of chromatin domains and the regulation of gene expression.  To analyze its function and modulate its activity, we screened for specific inhibitors against histone lysine methyltransferases (HMTases) using recombinant G9a as the target enzyme.  From a chem. library comprising 125,000 preselected compds., seven hits were identified.  Of those, one inhibitor, BIX-01294 (diazepinquinazolin-amine deriv.), does not compete with the cofactor S-adenosyl-methionine, and selectively impairs the G9a HMTase and the generation of H3K9me2 in vitro.  In cellular assays, transient incubation of several cell lines with BIX-01294 lowers bulk H3K9me2 levels that are restored upon removal of the inhibitor.  Importantly, chromatin immunopptn. at several G9a target genes demonstrates reversible redn. of promoter-proximal H3K9me2 in inhibitor-treated mouse ES cells and fibroblasts.  Our data identify a biol. active HMTase inhibitor that allows for the transient modulation of H3K9me2 marks in mammalian chromatin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3SW8czS4WjbVg90H21EOLACvtfcHk0ljj4y-BhzwzYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXitl2jurs%253D&md5=d2f5290f1252e630a9b05489b0775c2c</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2007.01.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2007.01.017%26sid%3Dliteratum%253Aachs%26aulast%3DKubicek%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Sullivan%26aufirst%3DR.%2BJ.%26aulast%3DAugust%26aufirst%3DE.%2BM.%26aulast%3DHickey%26aufirst%3DE.%2BR.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DTeodoro%26aufirst%3DM.%2BL.%26aulast%3DRea%26aufirst%3DS.%26aulast%3DMechtler%26aufirst%3DK.%26aulast%3DKowalski%26aufirst%3DJ.%2BA.%26aulast%3DHomon%26aufirst%3DC.%2BA.%26aulast%3DKelly%26aufirst%3DT.%2BA.%26aulast%3DJenuwein%26aufirst%3DT.%26atitle%3DReversal%2520of%2520H3K9me2%2520by%2520a%2520small-molecule%2520inhibitor%2520for%2520the%2520G9a%2520histone%2520methyltransferase%26jtitle%3DMol.%2520Cell%26date%3D2007%26volume%3D25%26spage%3D473%26epage%3D481" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Chang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horton, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Upadhyay, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spannhoff, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Snyder, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, X.</span><span> </span><span class="NLM_article-title">Structural basis for G9a-like protein lysine methyltransferase inhibition by BIX-01294</span> <span class="citation_source-journal">Nature Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">312</span><span class="NLM_x">–</span> <span class="NLM_lpage">317</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Fjm200903z&amp;key=10.1038%2Fnsmb.1560" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Fjm200903z&amp;key=19219047" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Fjm200903z&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhvFKjt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2009&pages=312-317&author=Y.+Changauthor=X.+Zhangauthor=J.+R.+Hortonauthor=A.+K.+Upadhyayauthor=A.+Spannhoffauthor=J.+Liuauthor=J.+P.+Snyderauthor=M.+T.+Bedfordauthor=X.+Cheng&title=Structural+basis+for+G9a-like+protein+lysine+methyltransferase+inhibition+by+BIX-01294"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for G9a-like protein lysine methyltransferase inhibition by BIX-01294</span></div><div class="casAuthors">Chang, Yanqi; Zhang, Xing; Horton, John R.; Upadhyay, Anup K.; Spannhoff, Astrid; Liu, Jin; Snyder, James P.; Bedford, Mark T.; Cheng, Xiaodong</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">312-317</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Histone lysine methylation is an important epigenetic mark that regulates gene expression and chromatin organization.  G9a and G9a-like protein (GLP) are euchromatin-assocd. methyltransferases that repress transcription by methylating histone H3 Lys9.  BIX-01294 was originally identified as a G9a inhibitor during a chem. library screen of small mols. and has previously been used in the generation of induced pluripotent stem cells.  Here we present the crystal structure of the catalytic SET domain of GLP in complex with BIX-01294 and S-adenosyl-L-homocysteine.  The inhibitor is bound in the substrate peptide groove at the location where the histone H3 residues N-terminal to the target lysine lie in the previously solved structure of the complex with histone peptide.  The inhibitor resembles the bound conformation of histone H3 Lys4 to Arg8, and is positioned in place by residues specific for G9a and GLP through specific interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhQepqN3zr_bVg90H21EOLACvtfcHk0ljj4y-BhzwzYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhvFKjt7g%253D&md5=6a16baaf33420fc1caf1df70d103d270</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.1560&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.1560%26sid%3Dliteratum%253Aachs%26aulast%3DChang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DHorton%26aufirst%3DJ.%2BR.%26aulast%3DUpadhyay%26aufirst%3DA.%2BK.%26aulast%3DSpannhoff%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DSnyder%26aufirst%3DJ.%2BP.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DCheng%26aufirst%3DX.%26atitle%3DStructural%2520basis%2520for%2520G9a-like%2520protein%2520lysine%2520methyltransferase%2520inhibition%2520by%2520BIX-01294%26jtitle%3DNature%2520Struct.%2520Mol.%2520Biol.%26date%3D2009%26volume%3D16%26spage%3D312%26epage%3D317" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quinn, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kozieradzki, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siarheyeva, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kireev, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jadhav, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herold, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Discovery of a 2,4-diamino-7-aminoalkoxyquinazoline as a potent and selective inhibitor of histone lysine methyltransferase G9a</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">7950</span><span class="NLM_x">–</span> <span class="NLM_lpage">7953</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm901543m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Fjm200903z&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlOiurzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=7950-7953&author=F.+Liuauthor=X.+Chenauthor=A.+Allali-Hassaniauthor=A.+M.+Quinnauthor=G.+A.+Wasneyauthor=A.+Dongauthor=D.+Barsyteauthor=I.+Kozieradzkiauthor=G.+Senisterraauthor=I.+Chauauthor=A.+Siarheyevaauthor=D.+B.+Kireevauthor=A.+Jadhavauthor=J.+M.+Heroldauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=A.+Simeonovauthor=M.+Vedadiauthor=J.+Jin&title=Discovery+of+a+2%2C4-diamino-7-aminoalkoxyquinazoline+as+a+potent+and+selective+inhibitor+of+histone+lysine+methyltransferase+G9a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a 2,4-Diamino-7-aminoalkoxyquinazoline as a Potent and Selective Inhibitor of Histone Lysine Methyltransferase G9a</span></div><div class="casAuthors">Liu, Feng; Chen, Xin; Allali-Hassani, Abdellah; Quinn, Amy M.; Wasney, Gregory A.; Dong, Aiping; Barsyte, Dalia; Kozieradzki, Ivona; Senisterra, Guillermo; Chau, Irene; Siarheyeva, Alena; Kireev, Dmitri B.; Jadhav, Ajit; Herold, J. Martin; Frye, Stephen V.; Arrowsmith, Cheryl H.; Brown, Peter J.; Simeonov, Anton; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">7950-7953</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">SAR exploration of the 2,4-diamino-6,7-dimethoxyquinazoline template led to the discovery of 8 (I) as a potent and selective G9a inhibitor.  A high resoln. X-ray crystal structure of the G9a-8 complex, the first cocrystal structure of G9a with a small mol. inhibitor, was obtained.  The cocrystal structure validated our binding hypothesis and will enable structure-based design of novel inhibitors.  8 Is a useful tool for investigating the biol. of G9a and its roles in chromatin remodeling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjWt5XsGXyj7Vg90H21EOLACvtfcHk0lgRLLu8PZIEyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlOiurzN&md5=2d7e34a8c4b5bf1cafd4e01ad2a86de1</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm901543m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901543m%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DQuinn%26aufirst%3DA.%2BM.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DBarsyte%26aufirst%3DD.%26aulast%3DKozieradzki%26aufirst%3DI.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DKireev%26aufirst%3DD.%2BB.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DHerold%26aufirst%3DJ.%2BM.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520a%25202%252C4-diamino-7-aminoalkoxyquinazoline%2520as%2520a%2520potent%2520and%2520selective%2520inhibitor%2520of%2520histone%2520lysine%2520methyltransferase%2520G9a%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D7950%26epage%3D7953" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quinn, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siarheyeva, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norris, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kireev, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jadhav, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herold, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzen, W. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Protein Lysine Methyltransferase G9a Inhibitors: Design, Synthesis, and Structure–Activity Relationships of 2,4-Diamino-7-aminoalkoxy-quinazolines</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">5844</span><span class="NLM_x">–</span> <span class="NLM_lpage">5857</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm100478y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=5844-5857&author=F.+Liuauthor=X.+Chenauthor=A.+Allali-Hassaniauthor=A.+M.+Quinnauthor=T.+J.+Wigleauthor=G.+A.+Wasneyauthor=A.+Dongauthor=G.+Senisterraauthor=I.+Chauauthor=A.+Siarheyevaauthor=J.+L.+Norrisauthor=D.+B.+Kireevauthor=A.+Jadhavauthor=J.+M.+Heroldauthor=W.+P.+Janzenauthor=C.+H.+Arrowsmithauthor=S.+V.+Fryeauthor=P.+J.+Brownauthor=A.+Simeonovauthor=M.+Vedadiauthor=J.+Jin&title=Protein+Lysine+Methyltransferase+G9a+Inhibitors%3A+Design%2C+Synthesis%2C+and+Structure%E2%80%93Activity+Relationships+of+2%2C4-Diamino-7-aminoalkoxy-quinazolines"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fjm100478y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100478y%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DQuinn%26aufirst%3DA.%2BM.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DKireev%26aufirst%3DD.%2BB.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DHerold%26aufirst%3DJ.%2BM.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DProtein%2520Lysine%2520Methyltransferase%2520G9a%2520Inhibitors%253A%2520Design%252C%2520Synthesis%252C%2520and%2520Structure%25E2%2580%2593Activity%2520Relationships%2520of%25202%252C4-Diamino-7-aminoalkoxy-quinazolines%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D5844%26epage%3D5857" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Chang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ganesh, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horton, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spannhoff, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shinkai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Snyder, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, X.</span><span> </span><span class="NLM_article-title">Adding a lysine mimic in the design of potent inhibitors of histone lysine methyltransferases</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">400</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">7</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=400&publication_year=2010&pages=1-7&author=Y.+Changauthor=T.+Ganeshauthor=J.+R.+Hortonauthor=A.+Spannhoffauthor=J.+Liuauthor=A.+Sunauthor=X.+Zhangauthor=M.+T.+Bedfordauthor=Y.+Shinkaiauthor=J.+P.+Snyderauthor=X.+Cheng&title=Adding+a+lysine+mimic+in+the+design+of+potent+inhibitors+of+histone+lysine+methyltransferases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChang%26aufirst%3DY.%26aulast%3DGanesh%26aufirst%3DT.%26aulast%3DHorton%26aufirst%3DJ.%2BR.%26aulast%3DSpannhoff%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DSun%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DShinkai%26aufirst%3DY.%26aulast%3DSnyder%26aufirst%3DJ.%2BP.%26aulast%3DCheng%26aufirst%3DX.%26atitle%3DAdding%2520a%2520lysine%2520mimic%2520in%2520the%2520design%2520of%2520potent%2520inhibitors%2520of%2520histone%2520lysine%2520methyltransferases%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2010%26volume%3D400%26spage%3D1%26epage%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rival-Gervier, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labrie, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DiMaggio, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siarheyeva, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tempel, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mangano, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simpson, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pattenden, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norris, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kireev, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tripathy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edwards, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzen, W. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petronis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellis, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J. A</span><span> </span><span class="NLM_article-title">Chemical Probe Selectively Inhibits G9a and GLP Methyltransferase Activity in Cells</span> <span class="citation_source-journal">Nature Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">566</span><span class="NLM_x">–</span> <span class="NLM_lpage">574</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Fjm200903z&amp;key=10.1038%2Fnchembio.599" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Fjm200903z&amp;key=21743462" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Fjm200903z&amp;key=1%3ACAS%3A528%3ADC%252BC3MXos1OmtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=566-574&author=M.+Vedadiauthor=D.+Barsyte-Lovejoyauthor=F.+Liuauthor=S.+Rival-Gervierauthor=A.+Allali-Hassaniauthor=V.+Labrieauthor=T.+J.+Wigleauthor=P.+A.+DiMaggioauthor=G.+A.+Wasneyauthor=A.+Siarheyevaauthor=A.+Dongauthor=W.+Tempelauthor=S.-C.+Wangauthor=X.+Chenauthor=I.+Chauauthor=T.+Manganoauthor=X.-P.+Huangauthor=C.+D.+Simpsonauthor=S.+G.+Pattendenauthor=J.+L.+Norrisauthor=D.+B.+Kireevauthor=A.+Tripathyauthor=A.+Edwardsauthor=B.+L.+Rothauthor=W.+P.+Janzenauthor=B.+A.+Garciaauthor=A.+Petronisauthor=J.+Ellisauthor=P.+J.+Brownauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=J.+A+Jin&title=Chemical+Probe+Selectively+Inhibits+G9a+and+GLP+Methyltransferase+Activity+in+Cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">A chemical probe selectively inhibits G9a and GLP methyltransferase activity in cells</span></div><div class="casAuthors">Vedadi, Masoud; Barsyte-Lovejoy, Dalia; Liu, Feng; Rival-Gervier, Sylvie; Allali-Hassani, Abdellah; Labrie, Viviane; Wigle, Tim J.; DiMaggio, Peter A.; Wasney, Gregory A.; Siarheyeva, Alena; Dong, Aiping; Tempel, Wolfram; Wang, Sun-Chong; Chen, Xin; Chau, Irene; Mangano, Thomas J.; Huang, Xi-ping; Simpson, Catherine D.; Pattenden, Samantha G.; Norris, Jacqueline L.; Kireev, Dmitri B.; Tripathy, Ashutosh; Edwards, Aled; Roth, Bryan L.; Janzen, William P.; Garcia, Benjamin A.; Petronis, Arturas; Ellis, James; Brown, Peter J.; Frye, Stephen V.; Arrowsmith, Cheryl H.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">566-574</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein lysine methyltransferases G9a and GLP modulate the transcriptional repression of a variety of genes via dimethylation of Lys9 on histone H3 (H3K9me2) as well as dimethylation of non-histone targets.  Here we report the discovery of UNC0638, an inhibitor of G9a and GLP with excellent potency and selectivity over a wide range of epigenetic and non-epigenetic targets.  UNC0638 treatment of a variety of cell lines resulted in lower global H3K9me2 levels, equiv. to levels obsd. for small hairpin RNA knockdown of G9a and GLP with the functional potency of UNC0638 being well sepd. from its toxicity.  UNC0638 markedly reduced the clonogenicity of MCF7 cells, reduced the abundance of H3K9me2 marks at promoters of known G9a-regulated endogenous genes and disproportionately affected several genomic loci encoding microRNAs.  In mouse embryonic stem cells, UNC0638 reactivated G9a-silenced genes and a retroviral reporter gene in a concn.-dependent manner without promoting differentiation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphv-2bZcBqwLVg90H21EOLACvtfcHk0li39gJBSzbIVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXos1OmtLs%253D&md5=a944e99a910194c1a15240fc6dc8f852</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.599&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.599%26sid%3Dliteratum%253Aachs%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DRival-Gervier%26aufirst%3DS.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DLabrie%26aufirst%3DV.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DDiMaggio%26aufirst%3DP.%2BA.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DTempel%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DS.-C.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DMangano%26aufirst%3DT.%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DSimpson%26aufirst%3DC.%2BD.%26aulast%3DPattenden%26aufirst%3DS.%2BG.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DKireev%26aufirst%3DD.%2BB.%26aulast%3DTripathy%26aufirst%3DA.%26aulast%3DEdwards%26aufirst%3DA.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26aulast%3DGarcia%26aufirst%3DB.%2BA.%26aulast%3DPetronis%26aufirst%3DA.%26aulast%3DEllis%26aufirst%3DJ.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DJin%26aufirst%3DJ.%2BA%26atitle%3DChemical%2520Probe%2520Selectively%2520Inhibits%2520G9a%2520and%2520GLP%2520Methyltransferase%2520Activity%2520in%2520Cells%26jtitle%3DNature%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D566%26epage%3D574" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Tachibana, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ueda, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fukuda, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takeda, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohta, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iwanari, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sakihama, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kodama, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamakubo, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shinkai, Y.</span><span> </span><span class="NLM_article-title">Histone methyltransferases G9a and GLP form heteromeric complexes and are both crucial for methylation of euchromatin at H3-K9</span> <span class="citation_source-journal">Genes Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">815</span><span class="NLM_x">–</span> <span class="NLM_lpage">826</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Fjm200903z&amp;key=10.1101%2Fgad.1284005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Fjm200903z&amp;key=15774718" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Fjm200903z&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjt1OjtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2005&pages=815-826&author=M.+Tachibanaauthor=J.+Uedaauthor=M.+Fukudaauthor=N.+Takedaauthor=T.+Ohtaauthor=H.+Iwanariauthor=T.+Sakihamaauthor=T.+Kodamaauthor=T.+Hamakuboauthor=Y.+Shinkai&title=Histone+methyltransferases+G9a+and+GLP+form+heteromeric+complexes+and+are+both+crucial+for+methylation+of+euchromatin+at+H3-K9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Histone methyltransferases G9a and GLP form heteromeric complexes and are both crucial for methylation of euchromatin at H3-K9</span></div><div class="casAuthors">Tachibana, Makoto; Ueda, Jun; Fukuda, Mikiko; Takeda, Naoki; Ohta, Tsutomu; Iwanari, Hiroko; Sakihama, Toshiko; Kodama, Tatsuhiko; Hamakubo, Takao; Shinkai, Yoichi</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">815-826</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Histone H3 Lys 9 (H3-K9) methylation is a crucial epigenetic mark for transcriptional silencing.  G9a is the major mammalian H3-K9 methyltransferase that targets euchromatic regions and is essential for murine embryogenesis.  There is a single G9a-related methyltransferase in mammals, called GLP/Eu-HMTase1.  Here the authors show that GLP is also important for H3-K9 methylation of mouse euchromatin.  GLP-deficiency led to embryonic lethality, a severe redn. of H3-K9 mono- and dimethylation, the induction of Mage-a gene expression, and HP1 relocalization in embryonic stem cells, all of which were phenotypes of G9a-deficiency.  Furthermore, the authors show that G9a and GLP formed a stoichiometric heteromeric complex in a wide variety of cell types.  Biochem. analyses revealed that formation of the G9a/GLP complex was dependent on their enzymic SET domains.  Taken together, these new findings revealed that G9a and GLP cooperatively exert H3-K9 methyltransferase function in vivo, likely through the formation of higher-order heteromeric complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUXBASXATfh7Vg90H21EOLACvtfcHk0li39gJBSzbIVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjt1OjtLc%253D&md5=cd0b7f8b9f62b4abe81b5ea590ab218c</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1101%2Fgad.1284005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.1284005%26sid%3Dliteratum%253Aachs%26aulast%3DTachibana%26aufirst%3DM.%26aulast%3DUeda%26aufirst%3DJ.%26aulast%3DFukuda%26aufirst%3DM.%26aulast%3DTakeda%26aufirst%3DN.%26aulast%3DOhta%26aufirst%3DT.%26aulast%3DIwanari%26aufirst%3DH.%26aulast%3DSakihama%26aufirst%3DT.%26aulast%3DKodama%26aufirst%3DT.%26aulast%3DHamakubo%26aufirst%3DT.%26aulast%3DShinkai%26aufirst%3DY.%26atitle%3DHistone%2520methyltransferases%2520G9a%2520and%2520GLP%2520form%2520heteromeric%2520complexes%2520and%2520are%2520both%2520crucial%2520for%2520methylation%2520of%2520euchromatin%2520at%2520H3-K9%26jtitle%3DGenes%2520Dev.%26date%3D2005%26volume%3D19%26spage%3D815%26epage%3D826" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Gottlieb, H. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kotlyar, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nudelman, A.</span><span> </span><span class="NLM_article-title">NMR Chemical Shifts of Common Laboratory Solvents as Trace Impurities</span> <span class="citation_source-journal">J. Org. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">62</span><span class="NLM_x">, </span> <span class="NLM_fpage">7512</span><span class="NLM_x">–</span> <span class="NLM_lpage">7515</span></span><div class="citationLinks">[<a href="/doi/10.1021/jo971176v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Fjm200903z&amp;key=1%3ACAS%3A528%3ADyaK2sXmvVClsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=1997&pages=7512-7515&author=H.+E.+Gottliebauthor=V.+Kotlyarauthor=A.+Nudelman&title=NMR+Chemical+Shifts+of+Common+Laboratory+Solvents+as+Trace+Impurities"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">NMR chemical shifts of common laboratory solvents as trace impurities</span></div><div class="casAuthors">Gottlieb, Hugo E.; Kotlyar, Vadim; Nudelman, Abraham</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">7512-7515</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The 1H (300 MHz) and 13C (75 MHz) NMR chem. shifts for >30 common impurities, such as lab. solvents, stabilizers, greases and water, are reported in 7 deuterated NMR solvents.  The temp. dependence of the chem. shift of the proton residual peak (HDO) in D2O is established for the entire liq. range of this sample.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocnoOemocS9LVg90H21EOLACvtfcHk0li39gJBSzbIVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXmvVClsbo%253D&md5=c297adda99f8c10e9f9b0ab9fe5f55be</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fjo971176v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo971176v%26sid%3Dliteratum%253Aachs%26aulast%3DGottlieb%26aufirst%3DH.%2BE.%26aulast%3DKotlyar%26aufirst%3DV.%26aulast%3DNudelman%26aufirst%3DA.%26atitle%3DNMR%2520Chemical%2520Shifts%2520of%2520Common%2520Laboratory%2520Solvents%2520as%2520Trace%2520Impurities%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1997%26volume%3D62%26spage%3D7512%26epage%3D7515" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Nazare, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Will, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matter, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schreuder, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ritter, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Urmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Essrich, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bauer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wagner, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Czech, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lorenz, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laux, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wehner, V.</span><span> </span><span class="NLM_article-title">Probing the subpockets of factor Xa reveals two binding modes for inhibitors based on a 2-carboxyindole scaffold: a study combining structure–activity relationship and X-ray crystallography</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">4511</span><span class="NLM_x">–</span> <span class="NLM_lpage">4525</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm0490540" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=4511-4525&author=M.+Nazareauthor=D.+W.+Willauthor=H.+Matterauthor=H.+Schreuderauthor=K.+Ritterauthor=M.+Urmannauthor=M.+Essrichauthor=A.+Bauerauthor=M.+Wagnerauthor=J.+Czechauthor=M.+Lorenzauthor=V.+Lauxauthor=V.+Wehner&title=Probing+the+subpockets+of+factor+Xa+reveals+two+binding+modes+for+inhibitors+based+on+a+2-carboxyindole+scaffold%3A+a+study+combining+structure%E2%80%93activity+relationship+and+X-ray+crystallography"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fjm0490540&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0490540%26sid%3Dliteratum%253Aachs%26aulast%3DNazare%26aufirst%3DM.%26aulast%3DWill%26aufirst%3DD.%2BW.%26aulast%3DMatter%26aufirst%3DH.%26aulast%3DSchreuder%26aufirst%3DH.%26aulast%3DRitter%26aufirst%3DK.%26aulast%3DUrmann%26aufirst%3DM.%26aulast%3DEssrich%26aufirst%3DM.%26aulast%3DBauer%26aufirst%3DA.%26aulast%3DWagner%26aufirst%3DM.%26aulast%3DCzech%26aufirst%3DJ.%26aulast%3DLorenz%26aufirst%3DM.%26aulast%3DLaux%26aufirst%3DV.%26aulast%3DWehner%26aufirst%3DV.%26atitle%3DProbing%2520the%2520subpockets%2520of%2520factor%2520Xa%2520reveals%2520two%2520binding%2520modes%2520for%2520inhibitors%2520based%2520on%2520a%25202-carboxyindole%2520scaffold%253A%2520a%2520study%2520combining%2520structure%25E2%2580%2593activity%2520relationship%2520and%2520X-ray%2520crystallography%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D4511%26epage%3D4525" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Collazo, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Couture, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bulfer, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trievel, R. C.</span><span> </span><span class="NLM_article-title">A coupled fluorescent assay for histone methyltransferases</span> <span class="citation_source-journal">Anal. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">342</span><span class="NLM_x">, </span> <span class="NLM_fpage">86</span><span class="NLM_x">–</span> <span class="NLM_lpage">92</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Fjm200903z&amp;key=10.1016%2Fj.ab.2005.04.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Fjm200903z&amp;key=15958184" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Fjm200903z&amp;key=1%3ACAS%3A528%3ADC%252BD2MXlt1GktL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=342&publication_year=2005&pages=86-92&author=E.+Collazoauthor=J.+F.+Coutureauthor=S.+Bulferauthor=R.+C.+Trievel&title=A+coupled+fluorescent+assay+for+histone+methyltransferases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">A coupled fluorescent assay for histone methyltransferases</span></div><div class="casAuthors">Collazo, Evys; Couture, Jean-Francois; Bulfer, Stacie; Trievel, Raymond C.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">342</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">86-92</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Histone methyltransferases (HMTs) catalyze the S-adenosylmethionine (AdoMet)-dependent methylation of lysines and arginines in the nucleosomal core histones H3 and H4 and the linker histone H1b.  Methylation of these residues regulates either transcriptional activation or silencing, depending on the residue modified and its degree of methylation.  Despite an intense interest in elucidating the functions of HMTs in transcriptional regulation, these enzymes have remained challenging to quant. assay.  To characterize the substrate specificity of HMTs, we have developed a coupled-fluorescence-based assay for AdoMet-dependent methyltransferases.  This assay utilizes S-adenosylhomocysteine hydrolase (SAHH) to hydrolyze the methyltransfer product S-adenosylhomocysteine (AdoHcy) to homocysteine (Hcy) and adenosine (Ado).  The Hcy concn. is then detd. through conjugation of its free sulfhydryl moiety to a thiol-sensitive fluorophore.  Using this assay, we have detd. the kinetic parameters for the methylation of a synthetic histone H3 peptide (corresponding to residues 1-15 of the native protein) by Schizosaccharomyces pombe CLR4, an H3 Lys-9-specific methyltransferase.  The fluorescent SAHH-coupled assay allows rapid and facile detn. of HMT kinetics and can be adapted to measure the enzymic activity of a wide variety of AdoMet-dependent methyltransferases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfHslBzi9obbVg90H21EOLACvtfcHk0liTKV2fFu_Hgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXlt1GktL4%253D&md5=2b949dcde263d15559202131cc74da1e</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2005.04.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2005.04.007%26sid%3Dliteratum%253Aachs%26aulast%3DCollazo%26aufirst%3DE.%26aulast%3DCouture%26aufirst%3DJ.%2BF.%26aulast%3DBulfer%26aufirst%3DS.%26aulast%3DTrievel%26aufirst%3DR.%2BC.%26atitle%3DA%2520coupled%2520fluorescent%2520assay%2520for%2520histone%2520methyltransferases%26jtitle%3DAnal.%2520Biochem.%26date%3D2005%26volume%3D342%26spage%3D86%26epage%3D92" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 92 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">M. Ramya Chandar Charles, Mu-Chun Li, Hsing-Pang Hsieh, <span class="NLM_string-name hlFld-ContribAuthor">Mohane Selvaraj Coumar</span>. </span><span class="cited-content_cbyCitation_article-title">Mimicking H3 Substrate Arginine in the Design of G9a Lysine Methyltransferase Inhibitors for Cancer Therapy: A Computational Study for Structure-Based Drug Design. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2021,</strong> <em>6 </em>
                                    (9)
                                     , 6100-6111. <a href="https://doi.org/10.1021/acsomega.0c04710" title="DOI URL">https://doi.org/10.1021/acsomega.0c04710</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.0c04710&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.0c04710%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DMimicking%252BH3%252BSubstrate%252BArginine%252Bin%252Bthe%252BDesign%252Bof%252BG9a%252BLysine%252BMethyltransferase%252BInhibitors%252Bfor%252BCancer%252BTherapy%25253A%252BA%252BComputational%252BStudy%252Bfor%252BStructure-Based%252BDrug%252BDesign%26aulast%3DChandar%2BCharles%26aufirst%3DM.%2BRamya%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D24092020%26date%3D09022021%26date%3D22022021%26volume%3D6%26issue%3D9%26spage%3D6100%26epage%3D6111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Mengjie Gu, Tan Boon Toh, Lissa Hooi, Jhin Jieh Lim, Xiyun Zhang, <span class="NLM_string-name hlFld-ContribAuthor">Edward Kai-Hua Chow</span>. </span><span class="cited-content_cbyCitation_article-title">Nanodiamond-Mediated Delivery of a G9a Inhibitor for Hepatocellular Carcinoma Therapy. </span><span class="cited-content_cbyCitation_journal-name">ACS Applied Materials & Interfaces</span><span> <strong>2019,</strong> <em>11 </em>
                                    (49)
                                     , 45427-45441. <a href="https://doi.org/10.1021/acsami.9b16323" title="DOI URL">https://doi.org/10.1021/acsami.9b16323</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsami.9b16323&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsami.9b16323%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Applied%2520Materials%2520%2526%2520Interfaces%26atitle%3DNanodiamond-Mediated%252BDelivery%252Bof%252Ba%252BG9a%252BInhibitor%252Bfor%252BHepatocellular%252BCarcinoma%252BTherapy%26aulast%3DGu%26aufirst%3DMengjie%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D10092019%26date%3D13112019%26date%3D26112019%26date%3D13112019%26volume%3D11%26issue%3D49%26spage%3D45427%26epage%3D45441" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Yan Xiong, Holger Greschik, Catrine Johansson, Ludwig Seifert, Johannes Bacher, Kwang-su Park, Nicolas Babault, Michael Martini, Vincent Fagan, Fengling Li, Irene Chau, Thomas Christott, David Dilworth, Dalia Barsyte-Lovejoy, Masoud Vedadi, Cheryl H. Arrowsmith, Paul Brennan, Oleg Fedorov, Manfred Jung, Gillian Farnie, Jing Liu, Udo Oppermann, Roland Schüle, <span class="NLM_string-name hlFld-ContribAuthor">Jian Jin</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of a Potent and Selective Fragment-like Inhibitor of Methyllysine Reader Protein Spindlin 1 (SPIN1). </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (20)
                                     , 8996-9007. <a href="https://doi.org/10.1021/acs.jmedchem.9b00522" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00522</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00522&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b00522%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Ba%252BPotent%252Band%252BSelective%252BFragment-like%252BInhibitor%252Bof%252BMethyllysine%252BReader%252BProtein%252BSpindlin%252B1%252B%252528SPIN1%252529%26aulast%3DXiong%26aufirst%3DYan%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D25032019%26date%3D24072019%26date%3D01072019%26volume%3D62%26issue%3D20%26spage%3D8996%26epage%3D9007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Ciro Milite, Alessandra Feoli, John R. Horton, Donatella Rescigno, Alessandra Cipriano, Vincenzo Pisapia, Monica Viviano, Giacomo Pepe, Giorgio Amendola, Ettore Novellino, Sandro Cosconati, Xiaodong Cheng, Sabrina Castellano, <span class="NLM_string-name hlFld-ContribAuthor">Gianluca Sbardella</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of a Novel Chemotype of Histone Lysine Methyltransferase EHMT1/2 (GLP/G9a) Inhibitors: Rational Design, Synthesis, Biological Evaluation, and Co-crystal Structure. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (5)
                                     , 2666-2689. <a href="https://doi.org/10.1021/acs.jmedchem.8b02008" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b02008</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b02008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b02008%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Ba%252BNovel%252BChemotype%252Bof%252BHistone%252BLysine%252BMethyltransferase%252BEHMT1%25252F2%252B%252528GLP%25252FG9a%252529%252BInhibitors%25253A%252BRational%252BDesign%25252C%252BSynthesis%25252C%252BBiological%252BEvaluation%25252C%252Band%252BCo-crystal%252BStructure%26aulast%3DMilite%26aufirst%3DCiro%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D21122018%26date%3D26022019%26date%3D12022019%26volume%3D62%26issue%3D5%26spage%3D2666%26epage%3D2689" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Obdulia Rabal, Juan Antonio Sánchez-Arias, Edurne San José-Enériz, Xabier Agirre, Irene de Miguel, Leire Garate, Estibaliz Miranda, Elena Sáez, Sergio Roa, José Angel Martínez-Climent, Yingying Liu, Wei Wu, Musheng Xu, Felipe Prosper, <span class="NLM_string-name hlFld-ContribAuthor">Julen Oyarzabal</span>. </span><span class="cited-content_cbyCitation_article-title">Detailed Exploration around 4-Aminoquinolines Chemical Space to Navigate the Lysine Methyltransferase G9a and DNA Methyltransferase Biological Spaces. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (15)
                                     , 6546-6573. <a href="https://doi.org/10.1021/acs.jmedchem.7b01925" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b01925</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b01925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b01925%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDetailed%252BExploration%252Baround%252B4-Aminoquinolines%252BChemical%252BSpace%252Bto%252BNavigate%252Bthe%252BLysine%252BMethyltransferase%252BG9a%252Band%252BDNA%252BMethyltransferase%252BBiological%252BSpaces%26aulast%3DRabal%26aufirst%3DObdulia%26date%3D2018%26date%3D2018%26date%3D2017%26date%3D30122017%26date%3D19072018%26date%3D11062018%26volume%3D61%26issue%3D15%26spage%3D6546%26epage%3D6573" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Obdulia Rabal, Edurne San José-Enériz, Xabier Agirre, Juan Antonio Sánchez-Arias, Amaia Vilas-Zornoza, Ana Ugarte, Irene de Miguel, Estíbaliz Miranda, Leire Garate, Mario Fraga, Pablo Santamarina, Raul Fernandez Perez, Raquel Ordoñez, Elena Sáez, Sergio Roa, María José García-Barchino, José Angel Martínez-Climent, Yingying Liu, Wei Wu, Musheng Xu, Felipe Prosper, <span class="NLM_string-name hlFld-ContribAuthor">Julen Oyarzabal</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of Reversible DNA Methyltransferase and Lysine Methyltransferase G9a Inhibitors with Antitumoral in Vivo Efficacy. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (15)
                                     , 6518-6545. <a href="https://doi.org/10.1021/acs.jmedchem.7b01926" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b01926</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b01926&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b01926%26sid%3Dliteratum%253Aachs%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26spage%3D15424%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BReversible%252BDNA%252BMethyltransferase%252Band%252BLysine%252BMethyltransferase%252BG9a%252BInhibitors%252Bwith%252BAntitumoral%252Bin%252BVivo%252BEfficacy%26aulast%3DRabal%26aufirst%3DObdulia%26date%3D2018%26date%3D2018%26date%3D2017%26date%3D30122017%26date%3D19072018%26date%3D28062018%26volume%3D61%26issue%3D15%26spage%3D6518%26epage%3D6545" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">H. Ümit  Kaniskan</span>, <span class="hlFld-ContribAuthor ">Michael L.  Martini</span>, and <span class="hlFld-ContribAuthor ">Jian  Jin</span>  . </span><span class="cited-content_cbyCitation_article-title">Inhibitors of Protein Methyltransferases and Demethylases. </span><span class="cited-content_cbyCitation_journal-name">Chemical Reviews</span><span> <strong>2018,</strong> <em>118 </em>
                                    (3)
                                     , 989-1068. <a href="https://doi.org/10.1021/acs.chemrev.6b00801" title="DOI URL">https://doi.org/10.1021/acs.chemrev.6b00801</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.chemrev.6b00801&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.chemrev.6b00801%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Reviews%26atitle%3DInhibitors%252Bof%252BProtein%252BMethyltransferases%252Band%252BDemethylases%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259Cmit%26date%3D2018%26date%3D2017%26date%3D2016%26date%3D30112016%26date%3D24032017%26date%3D14022018%26volume%3D118%26issue%3D3%26spage%3D989%26epage%3D1068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yan  Xiong</span>, <span class="hlFld-ContribAuthor ">Fengling  Li</span>, <span class="hlFld-ContribAuthor ">Nicolas  Babault</span>, <span class="hlFld-ContribAuthor ">Aiping  Dong</span>, <span class="hlFld-ContribAuthor ">Hong  Zeng</span>, <span class="hlFld-ContribAuthor ">Hong  Wu</span>, <span class="hlFld-ContribAuthor ">Xin  Chen</span>, <span class="hlFld-ContribAuthor ">Cheryl H.  Arrowsmith</span>, <span class="hlFld-ContribAuthor ">Peter J.  Brown</span>, <span class="hlFld-ContribAuthor ">Jing  Liu</span>, <span class="hlFld-ContribAuthor ">Masoud  Vedadi</span>, and <span class="hlFld-ContribAuthor ">Jian  Jin</span>  . </span><span class="cited-content_cbyCitation_article-title">Discovery of Potent and Selective Inhibitors for G9a-Like Protein (GLP) Lysine Methyltransferase. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>60 </em>
                                    (5)
                                     , 1876-1891. <a href="https://doi.org/10.1021/acs.jmedchem.6b01645" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b01645</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b01645&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.6b01645%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BPotent%252Band%252BSelective%252BInhibitors%252Bfor%252BG9a-Like%252BProtein%252B%252528GLP%252529%252BLysine%252BMethyltransferase%26aulast%3DXiong%26aufirst%3DYan%26date%3D2017%26date%3D2017%26date%3D2016%26date%3D15112016%26date%3D14022017%26date%3D09032017%26date%3D30012017%26volume%3D60%26issue%3D5%26spage%3D1876%26epage%3D1891" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kyle V.  Butler</span>, <span class="hlFld-ContribAuthor ">Anqi  Ma</span>, <span class="hlFld-ContribAuthor ">Wenyu  Yu</span>, <span class="hlFld-ContribAuthor ">Fengling  Li</span>, <span class="hlFld-ContribAuthor ">Wolfram  Tempel</span>, <span class="hlFld-ContribAuthor ">Nicolas  Babault</span>, <span class="hlFld-ContribAuthor ">Fabio  Pittella-Silva</span>, <span class="hlFld-ContribAuthor ">Jason  Shao</span>, <span class="hlFld-ContribAuthor ">Junyi  Wang</span>, <span class="hlFld-ContribAuthor ">Minkui  Luo</span>, <span class="hlFld-ContribAuthor ">Masoud  Vedadi</span>, <span class="hlFld-ContribAuthor ">Peter J.  Brown</span>, <span class="hlFld-ContribAuthor ">Cheryl H.  Arrowsmith</span>, and <span class="hlFld-ContribAuthor ">Jian  Jin</span>  . </span><span class="cited-content_cbyCitation_article-title">Structure-Based Design of a Covalent Inhibitor of the SET Domain-Containing Protein 8 (SETD8) Lysine Methyltransferase. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>59 </em>
                                    (21)
                                     , 9881-9889. <a href="https://doi.org/10.1021/acs.jmedchem.6b01244" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b01244</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b01244&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.6b01244%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure-Based%252BDesign%252Bof%252Ba%252BCovalent%252BInhibitor%252Bof%252Bthe%252BSET%252BDomain-Containing%252BProtein%252B8%252B%252528SETD8%252529%252BLysine%252BMethyltransferase%26aulast%3DButler%26aufirst%3DKyle%2BV.%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D16082016%26date%3D02112016%26date%3D10112016%26volume%3D59%26issue%3D21%26spage%3D9881%26epage%3D9889" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">H. Ümit  Kaniskan</span>, <span class="hlFld-ContribAuthor ">Kyle D.  Konze</span>, and <span class="hlFld-ContribAuthor ">Jian  Jin</span>  . </span><span class="cited-content_cbyCitation_article-title">Selective Inhibitors of Protein Methyltransferases. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2015,</strong> <em>58 </em>
                                    (4)
                                     , 1596-1629. <a href="https://doi.org/10.1021/jm501234a" title="DOI URL">https://doi.org/10.1021/jm501234a</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm501234a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm501234a%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSelective%252BInhibitors%252Bof%252BProtein%252BMethyltransferases%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259Cmit%26date%3D2015%26date%3D2014%26date%3D2014%26date%3D12082014%26date%3D02122014%26date%3D26022015%26date%3D19112014%26volume%3D58%26issue%3D4%26spage%3D1596%26epage%3D1629" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">H. Ümit  Kaniskan</span> and <span class="hlFld-ContribAuthor ">Jian  Jin</span>  . </span><span class="cited-content_cbyCitation_article-title">Chemical Probes of Histone Lysine Methyltransferases. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2015,</strong> <em>10 </em>
                                    (1)
                                     , 40-50. <a href="https://doi.org/10.1021/cb500785t" title="DOI URL">https://doi.org/10.1021/cb500785t</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/cb500785t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fcb500785t%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DChemical%252BProbes%252Bof%252BHistone%252BLysine%252BMethyltransferases%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259Cmit%26date%3D2015%26date%3D2014%26date%3D2014%26date%3D2014%26date%3D30092014%26date%3D17112014%26date%3D25112014%26date%3D16012015%26volume%3D10%26issue%3D1%26spage%3D40%26epage%3D50" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anqi  Ma</span>, <span class="hlFld-ContribAuthor ">Wenyu  Yu</span>, <span class="hlFld-ContribAuthor ">Fengling  Li</span>, <span class="hlFld-ContribAuthor ">Rachel M.  Bleich</span>, <span class="hlFld-ContribAuthor ">J. Martin  Herold</span>, <span class="hlFld-ContribAuthor ">Kyle V.  Butler</span>, <span class="hlFld-ContribAuthor ">Jacqueline L.  Norris</span>, <span class="hlFld-ContribAuthor ">Victoria  Korboukh</span>, <span class="hlFld-ContribAuthor ">Ashutosh  Tripathy</span>, <span class="hlFld-ContribAuthor ">William P.  Janzen</span>, <span class="hlFld-ContribAuthor ">Cheryl H.  Arrowsmith</span>, <span class="hlFld-ContribAuthor ">Stephen V.  Frye</span>, <span class="hlFld-ContribAuthor ">Masoud  Vedadi</span>, <span class="hlFld-ContribAuthor ">Peter J.  Brown</span>, and <span class="hlFld-ContribAuthor ">Jian  Jin</span>  . </span><span class="cited-content_cbyCitation_article-title">Discovery of a Selective, Substrate-Competitive Inhibitor of the Lysine Methyltransferase SETD8. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2014,</strong> <em>57 </em>
                                    (15)
                                     , 6822-6833. <a href="https://doi.org/10.1021/jm500871s" title="DOI URL">https://doi.org/10.1021/jm500871s</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm500871s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm500871s%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Ba%252BSelective%25252C%252BSubstrate-Competitive%252BInhibitor%252Bof%252Bthe%252BLysine%252BMethyltransferase%252BSETD8%26aulast%3DMa%26aufirst%3DAnqi%26date%3D2014%26date%3D2014%26date%3D2014%26date%3D09062014%26date%3D25072014%26date%3D14082014%26date%3D17072014%26volume%3D57%26issue%3D15%26spage%3D6822%26epage%3D6833" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kanchan  Devkota</span>, <span class="hlFld-ContribAuthor ">Brian  Lohse</span>, <span class="hlFld-ContribAuthor ">Qing  Liu</span>, <span class="hlFld-ContribAuthor ">Ming-Wei  Wang</span>, <span class="hlFld-ContribAuthor ">Dan  Stærk</span>, <span class="hlFld-ContribAuthor ">Jens  Berthelsen</span>, and <span class="hlFld-ContribAuthor ">Rasmus Prætorius  Clausen</span>  . </span><span class="cited-content_cbyCitation_article-title">Analogues of the Natural Product Sinefungin as Inhibitors of EHMT1 and EHMT2. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2014,</strong> <em>5 </em>
                                    (4)
                                     , 293-297. <a href="https://doi.org/10.1021/ml4002503" title="DOI URL">https://doi.org/10.1021/ml4002503</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ml4002503&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fml4002503%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DAnalogues%252Bof%252Bthe%252BNatural%252BProduct%252BSinefungin%252Bas%252BInhibitors%252Bof%252BEHMT1%252Band%252BEHMT2%26aulast%3DDevkota%26aufirst%3DKanchan%26date%3D2014%26date%3D2014%26date%3D2014%26date%3D2013%26date%3D03072013%26date%3D31012014%26date%3D04022014%26date%3D10042014%26date%3D31012014%26volume%3D5%26issue%3D4%26spage%3D293%26epage%3D297" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiao-Feng  Wang</span>, <span class="hlFld-ContribAuthor ">Fang  Guan</span>, <span class="hlFld-ContribAuthor ">Emika  Ohkoshi</span>, <span class="hlFld-ContribAuthor ">Wanjun  Guo</span>, <span class="hlFld-ContribAuthor ">Lili  Wang</span>, <span class="hlFld-ContribAuthor ">Dong-Qing  Zhu</span>, <span class="hlFld-ContribAuthor ">Sheng-Biao  Wang</span>, <span class="hlFld-ContribAuthor ">Li-Ting  Wang</span>, <span class="hlFld-ContribAuthor ">Ernest  Hamel</span>, <span class="hlFld-ContribAuthor ">Dexuan  Yang</span>, <span class="hlFld-ContribAuthor ">Linna  Li</span>, <span class="hlFld-ContribAuthor ">Keduo  Qian</span>, <span class="hlFld-ContribAuthor ">Susan L.  Morris-Natschke</span>, <span class="hlFld-ContribAuthor ">Shoujun  Yuan</span>, <span class="hlFld-ContribAuthor ">Kuo-Hsiung  Lee</span>, and <span class="hlFld-ContribAuthor ">Lan  Xie</span>  . </span><span class="cited-content_cbyCitation_article-title">Optimization of 4-(N-Cycloamino)phenylquinazolines as a Novel Class of Tubulin-Polymerization Inhibitors Targeting the Colchicine Site. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2014,</strong> <em>57 </em>
                                    (4)
                                     , 1390-1402. <a href="https://doi.org/10.1021/jm4016526" title="DOI URL">https://doi.org/10.1021/jm4016526</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm4016526&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm4016526%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DOptimization%252Bof%252B4-%252528N-Cycloamino%252529phenylquinazolines%252Bas%252Ba%252BNovel%252BClass%252Bof%252BTubulin-Polymerization%252BInhibitors%252BTargeting%252Bthe%252BColchicine%252BSite%26aulast%3DWang%26aufirst%3DXiao-Feng%26date%3D2014%26date%3D2014%26date%3D2013%26date%3D25102013%26date%3D06022014%26date%3D27022014%26volume%3D57%26issue%3D4%26spage%3D1390%26epage%3D1402" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Ramzi F. Sweis, Marina Pliushchev, Peter J. Brown, Jun Guo, Fengling Li, David Maag, Andrew M. Petros, Nirupama B. Soni, Chris Tse, Masoud Vedadi, Michael R. Michaelides, Gary G. Chiang, <span class="NLM_string-name hlFld-ContribAuthor">William N. Pappano</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery and Development of Potent and Selective Inhibitors of Histone Methyltransferase G9a. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2014,</strong> <em>5 </em>
                                    (2)
                                     , 205-209. <a href="https://doi.org/10.1021/ml400496h" title="DOI URL">https://doi.org/10.1021/ml400496h</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ml400496h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fml400496h%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDiscovery%252Band%252BDevelopment%252Bof%252BPotent%252Band%252BSelective%252BInhibitors%252Bof%252BHistone%252BMethyltransferase%252BG9a%26aulast%3DSweis%26aufirst%3DRamzi%2BF.%26date%3D2014%26date%3D2014%26date%3D2014%26date%3D2013%26date%3D03122013%26date%3D02012014%26date%3D09012014%26date%3D13022014%26date%3D02012014%26volume%3D5%26issue%3D2%26spage%3D205%26epage%3D209" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Feng  Liu</span>, <span class="hlFld-ContribAuthor ">Dalia  Barsyte-Lovejoy</span>, <span class="hlFld-ContribAuthor ">Fengling  Li</span>, <span class="hlFld-ContribAuthor ">Yan  Xiong</span>, <span class="hlFld-ContribAuthor ">Victoria  Korboukh</span>, <span class="hlFld-ContribAuthor ">Xi-Ping  Huang</span>, <span class="hlFld-ContribAuthor ">Abdellah  Allali-Hassani</span>, <span class="hlFld-ContribAuthor ">William P.  Janzen</span>, <span class="hlFld-ContribAuthor ">Bryan L.  Roth</span>, <span class="hlFld-ContribAuthor ">Stephen V.  Frye</span>, <span class="hlFld-ContribAuthor ">Cheryl H.  Arrowsmith</span>, <span class="hlFld-ContribAuthor ">Peter J.  Brown</span>, <span class="hlFld-ContribAuthor ">Masoud  Vedadi</span>, and <span class="hlFld-ContribAuthor ">Jian  Jin</span>  . </span><span class="cited-content_cbyCitation_article-title">Discovery of an in Vivo Chemical Probe of the Lysine Methyltransferases G9a and GLP. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2013,</strong> <em>56 </em>
                                    (21)
                                     , 8931-8942. <a href="https://doi.org/10.1021/jm401480r" title="DOI URL">https://doi.org/10.1021/jm401480r</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm401480r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm401480r%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Ban%252Bin%252BVivo%252BChemical%252BProbe%252Bof%252Bthe%252BLysine%252BMethyltransferases%252BG9a%252Band%252BGLP%26aulast%3DLiu%26aufirst%3DFeng%26date%3D2013%26date%3D2013%26date%3D2013%26date%3D24092013%26date%3D31102013%26date%3D14112013%26date%3D08102013%26volume%3D56%26issue%3D21%26spage%3D8931%26epage%3D8942" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kyle D.  Konze</span>, <span class="hlFld-ContribAuthor ">Anqi  Ma</span>, <span class="hlFld-ContribAuthor ">Fengling  Li</span>, <span class="hlFld-ContribAuthor ">Dalia  Barsyte-Lovejoy</span>, <span class="hlFld-ContribAuthor ">Trevor  Parton</span>, <span class="hlFld-ContribAuthor ">Christopher J.  MacNevin</span>, <span class="hlFld-ContribAuthor ">Feng  Liu</span>, <span class="hlFld-ContribAuthor ">Cen  Gao</span>, <span class="hlFld-ContribAuthor ">Xi-Ping  Huang</span>, <span class="hlFld-ContribAuthor ">Ekaterina  Kuznetsova</span>, <span class="hlFld-ContribAuthor ">Marie  Rougie</span>, <span class="hlFld-ContribAuthor ">Alice  Jiang</span>, <span class="hlFld-ContribAuthor ">Samantha G.  Pattenden</span>, <span class="hlFld-ContribAuthor ">Jacqueline L.  Norris</span>, <span class="hlFld-ContribAuthor ">Lindsey I.  James</span>, <span class="hlFld-ContribAuthor ">Bryan L.  Roth</span>, <span class="hlFld-ContribAuthor ">Peter J.  Brown</span>, <span class="hlFld-ContribAuthor ">Stephen V.  Frye</span>, <span class="hlFld-ContribAuthor ">Cheryl H.  Arrowsmith</span>, <span class="hlFld-ContribAuthor ">Klaus M.  Hahn</span>, <span class="hlFld-ContribAuthor ">Gang Greg  Wang</span>, <span class="hlFld-ContribAuthor ">Masoud  Vedadi</span>, and <span class="hlFld-ContribAuthor ">Jian  Jin</span>  . </span><span class="cited-content_cbyCitation_article-title">An Orally Bioavailable Chemical Probe of the Lysine Methyltransferases EZH2 and EZH1. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2013,</strong> <em>8 </em>
                                    (6)
                                     , 1324-1334. <a href="https://doi.org/10.1021/cb400133j" title="DOI URL">https://doi.org/10.1021/cb400133j</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/cb400133j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fcb400133j%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DAn%252BOrally%252BBioavailable%252BChemical%252BProbe%252Bof%252Bthe%252BLysine%252BMethyltransferases%252BEZH2%252Band%252BEZH1%26aulast%3DKonze%26aufirst%3DKyle%2BD.%26date%3D2013%26date%3D2013%26date%3D2013%26date%3D2013%26date%3D22022013%26date%3D08042013%26date%3D24042013%26date%3D21062013%26date%3D08042013%26volume%3D8%26issue%3D6%26spage%3D1324%26epage%3D1334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Feng  Liu</span>, <span class="hlFld-ContribAuthor ">Fengling  Li</span>, <span class="hlFld-ContribAuthor ">Anqi  Ma</span>, <span class="hlFld-ContribAuthor ">Elena  Dobrovetsky</span>, <span class="hlFld-ContribAuthor ">Aiping  Dong</span>, <span class="hlFld-ContribAuthor ">Cen  Gao</span>, <span class="hlFld-ContribAuthor ">Ilia  Korboukh</span>, <span class="hlFld-ContribAuthor ">Jing  Liu</span>, <span class="hlFld-ContribAuthor ">David  Smil</span>, <span class="hlFld-ContribAuthor ">Peter J.  Brown</span>, <span class="hlFld-ContribAuthor ">Stephen V.  Frye</span>, <span class="hlFld-ContribAuthor ">Cheryl H.  Arrowsmith</span>, <span class="hlFld-ContribAuthor ">Matthieu  Schapira</span>, <span class="hlFld-ContribAuthor ">Masoud  Vedadi</span>, and <span class="hlFld-ContribAuthor ">Jian  Jin</span>  . </span><span class="cited-content_cbyCitation_article-title">Exploiting an Allosteric Binding Site of PRMT3 Yields Potent and Selective Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2013,</strong> <em>56 </em>
                                    (5)
                                     , 2110-2124. <a href="https://doi.org/10.1021/jm3018332" title="DOI URL">https://doi.org/10.1021/jm3018332</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm3018332&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm3018332%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DExploiting%252Ban%252BAllosteric%252BBinding%252BSite%252Bof%252BPRMT3%252BYields%252BPotent%252Band%252BSelective%252BInhibitors%26aulast%3DLiu%26aufirst%3DFeng%26date%3D2013%26date%3D2013%26date%3D2012%26date%3D12122012%26date%3D27022013%26date%3D14032013%26volume%3D56%26issue%3D5%26spage%3D2110%26epage%3D2124" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Justin L.  Anglin</span>, <span class="hlFld-ContribAuthor ">Lisheng  Deng</span>, <span class="hlFld-ContribAuthor ">Yuan  Yao</span>, <span class="hlFld-ContribAuthor ">Guobin  Cai</span>, <span class="hlFld-ContribAuthor ">Zhen  Liu</span>, <span class="hlFld-ContribAuthor ">Hong  Jiang</span>, <span class="hlFld-ContribAuthor ">Gang  Cheng</span>, <span class="hlFld-ContribAuthor ">Pinhong  Chen</span>, <span class="hlFld-ContribAuthor ">Shuo  Dong</span>, and <span class="hlFld-ContribAuthor ">Yongcheng  Song</span>  . </span><span class="cited-content_cbyCitation_article-title">Synthesis and Structure–Activity Relationship Investigation of Adenosine-Containing Inhibitors of Histone Methyltransferase DOT1L. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2012,</strong> <em>55 </em>
                                    (18)
                                     , 8066-8074. <a href="https://doi.org/10.1021/jm300917h" title="DOI URL">https://doi.org/10.1021/jm300917h</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm300917h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm300917h%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSynthesis%252Band%252BStructure%2525E2%252580%252593Activity%252BRelationship%252BInvestigation%252Bof%252BAdenosine-Containing%252BInhibitors%252Bof%252BHistone%252BMethyltransferase%252BDOT1L%26aulast%3DAnglin%26aufirst%3DJustin%2BL.%26date%3D2012%26date%3D2012%26date%3D2012%26date%3D25062012%26date%3D06092012%26date%3D27092012%26date%3D27082012%26volume%3D55%26issue%3D18%26spage%3D8066%26epage%3D8074" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bo  Chao</span>, <span class="hlFld-ContribAuthor ">Xian-Kun  Tong</span>, <span class="hlFld-ContribAuthor ">Wei  Tang</span>, <span class="hlFld-ContribAuthor ">De-Wen  Li</span>, <span class="hlFld-ContribAuthor ">Pei-Lan  He</span>, <span class="hlFld-ContribAuthor ">Jean-Michel  Garcia</span>, <span class="hlFld-ContribAuthor ">Li-Min  Zeng</span>, <span class="hlFld-ContribAuthor ">An-Hui  Gao</span>, <span class="hlFld-ContribAuthor ">Li  Yang</span>, <span class="hlFld-ContribAuthor ">Jia  Li</span>, <span class="hlFld-ContribAuthor ">Fa-Jun  Nan</span>, <span class="hlFld-ContribAuthor ">Michael  Jacobs</span>, <span class="hlFld-ContribAuthor ">Ralf  Altmeyer</span>, <span class="hlFld-ContribAuthor ">Jian-Ping  Zuo</span>, and <span class="hlFld-ContribAuthor ">You-Hong  Hu</span>  . </span><span class="cited-content_cbyCitation_article-title">Discovery and Optimization of 2,4-Diaminoquinazoline Derivatives as a New Class of Potent Dengue Virus Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2012,</strong> <em>55 </em>
                                    (7)
                                     , 3135-3143. <a href="https://doi.org/10.1021/jm2015952" title="DOI URL">https://doi.org/10.1021/jm2015952</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm2015952&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm2015952%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Band%252BOptimization%252Bof%252B2%25252C4-Diaminoquinazoline%252BDerivatives%252Bas%252Ba%252BNew%252BClass%252Bof%252BPotent%252BDengue%252BVirus%252BInhibitors%26aulast%3DChao%26aufirst%3DBo%26date%3D2012%26date%3D2012%26date%3D2011%26date%3D25112011%26date%3D26032012%26date%3D12042012%26volume%3D55%26issue%3D7%26spage%3D3135%26epage%3D3143" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Minkui  Luo</span>  . </span><span class="cited-content_cbyCitation_article-title">Current Chemical Biology Approaches to Interrogate Protein Methyltransferases. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2012,</strong> <em>7 </em>
                                    (3)
                                     , 443-463. <a href="https://doi.org/10.1021/cb200519y" title="DOI URL">https://doi.org/10.1021/cb200519y</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/cb200519y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fcb200519y%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DCurrent%252BChemical%252BBiology%252BApproaches%252Bto%252BInterrogate%252BProtein%252BMethyltransferases%26aulast%3DLuo%26aufirst%3DMinkui%26date%3D2012%26date%3D2012%26date%3D2012%26date%3D2011%26date%3D07122011%26date%3D05012012%26date%3D01022012%26date%3D16032012%26date%3D05012012%26volume%3D7%26issue%3D3%26spage%3D443%26epage%3D463" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kaushik  Chanda</span>, <span class="hlFld-ContribAuthor ">Barnali  Maiti</span>, <span class="hlFld-ContribAuthor ">Chih-Chung  Tseng</span>, and <span class="hlFld-ContribAuthor ">Chung-Ming  Sun</span>  . </span><span class="cited-content_cbyCitation_article-title">Microwave-Assisted Linear Approach Toward Highly Substituted Benzo[d]oxazol-5-yl-1H-benzo[d]imidazole on Ionic Liquid Support. </span><span class="cited-content_cbyCitation_journal-name">ACS Combinatorial Science</span><span> <strong>2012,</strong> <em>14 </em>
                                    (2)
                                     , 115-123. <a href="https://doi.org/10.1021/co200188g" title="DOI URL">https://doi.org/10.1021/co200188g</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/co200188g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fco200188g%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Combinatorial%2520Science%26atitle%3DMicrowave-Assisted%252BLinear%252BApproach%252BToward%252BHighly%252BSubstituted%252BBenzo%25255Bd%25255Doxazol-5-yl-1H-benzo%25255Bd%25255Dimidazole%252Bon%252BIonic%252BLiquid%252BSupport%26aulast%3DChanda%26aufirst%3DKaushik%26date%3D2012%26date%3D2012%26date%3D2011%26date%3D2012%26date%3D09112011%26date%3D11012012%26date%3D20012012%26date%3D13022012%26volume%3D14%26issue%3D2%26spage%3D115%26epage%3D123" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Robert B. A.  Quinlan</span>, <span class="hlFld-ContribAuthor ">Paul E.  Brennan</span>. </span><span class="cited-content_cbyCitation_article-title">Chemogenomics for drug discovery: clinical molecules from open access chemical probes. </span><span class="cited-content_cbyCitation_journal-name">RSC Chemical Biology</span><span> <strong>2021,</strong> <em>2 </em>
                                    (3)
                                     , 759-795. <a href="https://doi.org/10.1039/D1CB00016K" title="DOI URL">https://doi.org/10.1039/D1CB00016K</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D1CB00016K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD1CB00016K%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Chemical%2520Biology%26atitle%3DChemogenomics%252Bfor%252Bdrug%252Bdiscovery%25253A%252Bclinical%252Bmolecules%252Bfrom%252Bopen%252Baccess%252Bchemical%252Bprobes%26aulast%3DQuinlan%26aufirst%3DRobert%2BB.%2BA.%26date%3D2021%26date%3D2021%26volume%3D2%26issue%3D3%26spage%3D759%26epage%3D795" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bratati  Saha</span>, <span class="hlFld-ContribAuthor ">Robin J.  Parks</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of human adenovirus replication inhibitors from a library of small molecules targeting cellular epigenetic regulators. </span><span class="cited-content_cbyCitation_journal-name">Virology</span><span> <strong>2021,</strong> <em>555 </em>, 102-110. <a href="https://doi.org/10.1016/j.virol.2020.07.007" title="DOI URL">https://doi.org/10.1016/j.virol.2020.07.007</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.virol.2020.07.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.virol.2020.07.007%26sid%3Dliteratum%253Aachs%26jtitle%3DVirology%26atitle%3DIdentification%252Bof%252Bhuman%252Badenovirus%252Breplication%252Binhibitors%252Bfrom%252Ba%252Blibrary%252Bof%252Bsmall%252Bmolecules%252Btargeting%252Bcellular%252Bepigenetic%252Bregulators%26aulast%3DSaha%26aufirst%3DBratati%26date%3D2021%26volume%3D555%26spage%3D102%26epage%3D110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Suraya  Jan</span>, <span class="hlFld-ContribAuthor ">Mohd Ishaq  Dar</span>, <span class="hlFld-ContribAuthor ">Rubiada  Wani</span>, <span class="hlFld-ContribAuthor ">Jagjeet  Sandey</span>, <span class="hlFld-ContribAuthor ">Iqra  Mushtaq</span>, <span class="hlFld-ContribAuthor ">Sammar  Lateef</span>, <span class="hlFld-ContribAuthor ">Sajad Hussain  Syed</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting EHMT2/ G9a for cancer therapy: Progress and perspective. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Pharmacology</span><span> <strong>2021,</strong> <em>893 </em>, 173827. <a href="https://doi.org/10.1016/j.ejphar.2020.173827" title="DOI URL">https://doi.org/10.1016/j.ejphar.2020.173827</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejphar.2020.173827&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejphar.2020.173827%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Pharmacology%26atitle%3DTargeting%252BEHMT2%25252F%252BG9a%252Bfor%252Bcancer%252Btherapy%25253A%252BProgress%252Band%252Bperspective%26aulast%3DJan%26aufirst%3DSuraya%26date%3D2021%26volume%3D893%26spage%3D173827" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alexander  Maus</span>, <span class="hlFld-ContribAuthor ">Lia  Strait</span>, <span class="hlFld-ContribAuthor ">Donghui  Zhu</span>. </span><span class="cited-content_cbyCitation_article-title">Nanoparticles as delivery vehicles for antiviral therapeutic drugs. </span><span class="cited-content_cbyCitation_journal-name">Engineered Regeneration</span><span> <strong>2021,</strong> <em>2 </em>, 31-46. <a href="https://doi.org/10.1016/j.engreg.2021.03.001" title="DOI URL">https://doi.org/10.1016/j.engreg.2021.03.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.engreg.2021.03.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.engreg.2021.03.001%26sid%3Dliteratum%253Aachs%26jtitle%3DEngineered%2520Regeneration%26atitle%3DNanoparticles%252Bas%252Bdelivery%252Bvehicles%252Bfor%252Bantiviral%252Btherapeutic%252Bdrugs%26aulast%3DMaus%26aufirst%3DAlexander%26date%3D2021%26volume%3D2%26spage%3D31%26epage%3D46" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Md. Sahab  Uddin</span>, <span class="hlFld-ContribAuthor ">Abdullah Al  Mamun</span>, <span class="hlFld-ContribAuthor ">Badrah S.  Alghamdi</span>, <span class="hlFld-ContribAuthor ">Devesh  Tewari</span>, <span class="hlFld-ContribAuthor ">Philippe  Jeandet</span>, <span class="hlFld-ContribAuthor ">Md. Shahid  Sarwar</span>, <span class="hlFld-ContribAuthor ">Ghulam Md.  Ashraf</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetics of glioblastoma multiforme: From molecular mechanisms to therapeutic approaches. </span><span class="cited-content_cbyCitation_journal-name">Seminars in Cancer Biology</span><span> <strong>2020,</strong> <em>18 </em><a href="https://doi.org/10.1016/j.semcancer.2020.12.015" title="DOI URL">https://doi.org/10.1016/j.semcancer.2020.12.015</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.semcancer.2020.12.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.semcancer.2020.12.015%26sid%3Dliteratum%253Aachs%26jtitle%3DSeminars%2520in%2520Cancer%2520Biology%26atitle%3DEpigenetics%252Bof%252Bglioblastoma%252Bmultiforme%25253A%252BFrom%252Bmolecular%252Bmechanisms%252Bto%252Btherapeutic%252Bapproaches%26aulast%3DUddin%26aufirst%3DMd.%2BSahab%26date%3D2020%26volume%3D18" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jieming  Nie</span>. </span><span class="cited-content_cbyCitation_article-title">UNC0321 inhibits high glucose induced apoptosis in HUVEC by targeting Rab4. </span><span class="cited-content_cbyCitation_journal-name">Biomedicine & Pharmacotherapy</span><span> <strong>2020,</strong> <em>131 </em>, 110662. <a href="https://doi.org/10.1016/j.biopha.2020.110662" title="DOI URL">https://doi.org/10.1016/j.biopha.2020.110662</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.biopha.2020.110662&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.biopha.2020.110662%26sid%3Dliteratum%253Aachs%26jtitle%3DBiomedicine%2520%2526%2520Pharmacotherapy%26atitle%3DUNC0321%252Binhibits%252Bhigh%252Bglucose%252Binduced%252Bapoptosis%252Bin%252BHUVEC%252Bby%252Btargeting%252BRab4%26aulast%3DNie%26aufirst%3DJieming%26date%3D2020%26volume%3D131%26spage%3D110662" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Katsushi  Katayama</span>, <span class="hlFld-ContribAuthor ">Ken  Ishii</span>, <span class="hlFld-ContribAuthor ">Eisuke  Tsuda</span>, <span class="hlFld-ContribAuthor ">Keiichi  Yotsumoto</span>, <span class="hlFld-ContribAuthor ">Kumiko  Hiramoto</span>, <span class="hlFld-ContribAuthor ">Makoto  Suzuki</span>, <span class="hlFld-ContribAuthor ">Isao  Yasumatsu</span>, <span class="hlFld-ContribAuthor ">Wataru  Igarashi</span>, <span class="hlFld-ContribAuthor ">Munefumi  Torihata</span>, <span class="hlFld-ContribAuthor ">Takashi  Ishiyama</span>, <span class="hlFld-ContribAuthor ">Takahiro  Katagiri</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of novel histone lysine methyltransferase G9a/GLP (EHMT2/1) inhibitors: Design, synthesis, and structure-activity relationships of 2,4-diamino-6-methylpyrimidines. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>30 </em>
                                    (20)
                                     , 127475. <a href="https://doi.org/10.1016/j.bmcl.2020.127475" title="DOI URL">https://doi.org/10.1016/j.bmcl.2020.127475</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2020.127475&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2020.127475%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDiscovery%252Bof%252Bnovel%252Bhistone%252Blysine%252Bmethyltransferase%252BG9a%25252FGLP%252B%252528EHMT2%25252F1%252529%252Binhibitors%25253A%252BDesign%25252C%252Bsynthesis%25252C%252Band%252Bstructure-activity%252Brelationships%252Bof%252B2%25252C4-diamino-6-methylpyrimidines%26aulast%3DKatayama%26aufirst%3DKatsushi%26date%3D2020%26volume%3D30%26issue%3D20%26spage%3D127475" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dong  Cao</span>, <span class="hlFld-ContribAuthor ">Xiaoyan  Wang</span>, <span class="hlFld-ContribAuthor ">Lei  Lei</span>, <span class="hlFld-ContribAuthor ">Liang  Ma</span>, <span class="hlFld-ContribAuthor ">Zhuang  Yang</span>, <span class="hlFld-ContribAuthor ">Fang  Wang</span>, <span class="hlFld-ContribAuthor ">Lijuan  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis and Biological Evaluation of Novel Substituted 4‐Anilinoquinazolines as Antitumor Agents. </span><span class="cited-content_cbyCitation_journal-name">Chemical Biology & Drug Design</span><span> <strong>2020,</strong> <em>96 </em>
                                    (4)
                                     , 1084-1094. <a href="https://doi.org/10.1111/cbdd.12706" title="DOI URL">https://doi.org/10.1111/cbdd.12706</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1111/cbdd.12706&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1111%2Fcbdd.12706%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Biology%2520%2526%2520Drug%2520Design%26atitle%3DSynthesis%252Band%252BBiological%252BEvaluation%252Bof%252BNovel%252BSubstituted%252B4%2525E2%252580%252590Anilinoquinazolines%252Bas%252BAntitumor%252BAgents%26aulast%3DCao%26aufirst%3DDong%26date%3D2020%26date%3D2020%26volume%3D96%26issue%3D4%26spage%3D1084%26epage%3D1094" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Iwona A.  Ciechomska</span>, <span class="hlFld-ContribAuthor ">Chinchu  Jayaprakash</span>, <span class="hlFld-ContribAuthor ">Marta  Maleszewska</span>, <span class="hlFld-ContribAuthor ">Bozena  Kaminska</span>. </span><span class="cited-content_cbyCitation_article-title">Histone Modifying Enzymes and Chromatin Modifiers in Glioma Pathobiology and Therapy Responses. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 259-279. <a href="https://doi.org/10.1007/978-3-030-30651-9_13" title="DOI URL">https://doi.org/10.1007/978-3-030-30651-9_13</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-3-030-30651-9_13&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-3-030-30651-9_13%26sid%3Dliteratum%253Aachs%26atitle%3DHistone%252BModifying%252BEnzymes%252Band%252BChromatin%252BModifiers%252Bin%252BGlioma%252BPathobiology%252Band%252BTherapy%252BResponses%26aulast%3DCiechomska%26aufirst%3DIwona%2BA.%26date%3D2020%26date%3D2020%26spage%3D259%26epage%3D279%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DGlioma%252BSignaling%26aulast%3DBara%25C5%2584ska%26aufirst%3DJolanta%26date%3D2020%26volume%3D1202" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">M. Ramya Chandar  Charles</span>, <span class="hlFld-ContribAuthor ">Arun  Mahesh</span>, <span class="hlFld-ContribAuthor ">Shu-Yu  Lin</span>, <span class="hlFld-ContribAuthor ">Hsing-Pang  Hsieh</span>, <span class="hlFld-ContribAuthor ">Arunkumar  Dhayalan</span>, <span class="hlFld-ContribAuthor ">Mohane Selvaraj  Coumar</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of novel quinoline inhibitor for EHMT2/G9a through virtual screening. </span><span class="cited-content_cbyCitation_journal-name">Biochimie</span><span> <strong>2020,</strong> <em>168 </em>, 220-230. <a href="https://doi.org/10.1016/j.biochi.2019.11.006" title="DOI URL">https://doi.org/10.1016/j.biochi.2019.11.006</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.biochi.2019.11.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.biochi.2019.11.006%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochimie%26atitle%3DIdentification%252Bof%252Bnovel%252Bquinoline%252Binhibitor%252Bfor%252BEHMT2%25252FG9a%252Bthrough%252Bvirtual%252Bscreening%26aulast%3DCharles%26aufirst%3DM.%2BRamya%2BChandar%26date%3D2020%26volume%3D168%26spage%3D220%26epage%3D230" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hani Mutlak A.  Hassan</span>, <span class="hlFld-ContribAuthor ">Iuliana  Denetiu</span>, <span class="hlFld-ContribAuthor ">Salman A.  Khan</span>, <span class="hlFld-ContribAuthor ">Mohd  Rehan</span>, <span class="hlFld-ContribAuthor ">Kaltoom  Sakkaf</span>, <span class="hlFld-ContribAuthor ">Kalamegam  Gauthaman</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis and biological evaluation of novel triazolyl 4-anilinoquinazolines as anticancer agents. </span><span class="cited-content_cbyCitation_journal-name">Medicinal Chemistry Research</span><span> <strong>2019,</strong> <em>28 </em>
                                    (10)
                                     , 1766-1772. <a href="https://doi.org/10.1007/s00044-019-02413-6" title="DOI URL">https://doi.org/10.1007/s00044-019-02413-6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00044-019-02413-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00044-019-02413-6%26sid%3Dliteratum%253Aachs%26jtitle%3DMedicinal%2520Chemistry%2520Research%26atitle%3DSynthesis%252Band%252Bbiological%252Bevaluation%252Bof%252Bnovel%252Btriazolyl%252B4-anilinoquinazolines%252Bas%252Banticancer%252Bagents%26aulast%3DHassan%26aufirst%3DHani%2BMutlak%2BA.%26date%3D2019%26date%3D2019%26volume%3D28%26issue%3D10%26spage%3D1766%26epage%3D1772" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hao  Cao</span>, <span class="hlFld-ContribAuthor ">Ling  Li</span>, <span class="hlFld-ContribAuthor ">Deying  Yang</span>, <span class="hlFld-ContribAuthor ">Liming  Zeng</span>, <span class="hlFld-ContribAuthor ">Xie  Yewei</span>, <span class="hlFld-ContribAuthor ">Bin  Yu</span>, <span class="hlFld-ContribAuthor ">Guochao  Liao</span>, <span class="hlFld-ContribAuthor ">Jianjun  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Recent progress in histone methyltransferase (G9a) inhibitors as anticancer agents. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>179 </em>, 537-546. <a href="https://doi.org/10.1016/j.ejmech.2019.06.072" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.06.072</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.06.072&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.06.072%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DRecent%252Bprogress%252Bin%252Bhistone%252Bmethyltransferase%252B%252528G9a%252529%252Binhibitors%252Bas%252Banticancer%252Bagents%26aulast%3DCao%26aufirst%3DHao%26date%3D2019%26volume%3D179%26spage%3D537%26epage%3D546" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ruben  Leenders</span>, <span class="hlFld-ContribAuthor ">Remco  Zijlmans</span>, <span class="hlFld-ContribAuthor ">Bart  van Bree</span>, <span class="hlFld-ContribAuthor ">Marc  van de Sande</span>, <span class="hlFld-ContribAuthor ">Federica  Trivarelli</span>, <span class="hlFld-ContribAuthor ">Eddy  Damen</span>, <span class="hlFld-ContribAuthor ">Anita  Wegert</span>, <span class="hlFld-ContribAuthor ">Daniel  Müller</span>, <span class="hlFld-ContribAuthor ">Jan Erik  Ehlert</span>, <span class="hlFld-ContribAuthor ">Daniel  Feger</span>, <span class="hlFld-ContribAuthor ">Carolin  Heidemann-Dinger</span>, <span class="hlFld-ContribAuthor ">Michael  Kubbutat</span>, <span class="hlFld-ContribAuthor ">Christoph  Schächtele</span>, <span class="hlFld-ContribAuthor ">Danny C.  Lenstra</span>, <span class="hlFld-ContribAuthor ">Jasmin  Mecinović</span>, <span class="hlFld-ContribAuthor ">Gerhard  Müller</span>. </span><span class="cited-content_cbyCitation_article-title">Novel SAR for quinazoline inhibitors of EHMT1 and EHMT2. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2019,</strong> <em>29 </em>
                                    (17)
                                     , 2516-2524. <a href="https://doi.org/10.1016/j.bmcl.2019.06.012" title="DOI URL">https://doi.org/10.1016/j.bmcl.2019.06.012</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2019.06.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2019.06.012%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DNovel%252BSAR%252Bfor%252Bquinazoline%252Binhibitors%252Bof%252BEHMT1%252Band%252BEHMT2%26aulast%3DLeenders%26aufirst%3DRuben%26date%3D2019%26volume%3D29%26issue%3D17%26spage%3D2516%26epage%3D2524" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mariasoosai Ramya Chandar  Charles</span>, <span class="hlFld-ContribAuthor ">Arunkumar  Dhayalan</span>, <span class="hlFld-ContribAuthor ">Hsing-Pang  Hsieh</span>, <span class="hlFld-ContribAuthor ">Mohane Selvaraj  Coumar</span>. </span><span class="cited-content_cbyCitation_article-title">Insights for the design of protein lysine methyltransferase G9a inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2019,</strong> <em>11 </em>
                                    (9)
                                     , 993-1014. <a href="https://doi.org/10.4155/fmc-2018-0396" title="DOI URL">https://doi.org/10.4155/fmc-2018-0396</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2018-0396&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2018-0396%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DInsights%252Bfor%252Bthe%252Bdesign%252Bof%252Bprotein%252Blysine%252Bmethyltransferase%252BG9a%252Binhibitors%26aulast%3DCharles%26aufirst%3DMariasoosai%2BRamya%2BChandar%26date%3D2019%26volume%3D11%26issue%3D9%26spage%3D993%26epage%3D1014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hui-Yi  Wang</span>, <span class="hlFld-ContribAuthor ">Qiao-Yun  Long</span>, <span class="hlFld-ContribAuthor ">Shan-Bo  Tang</span>, <span class="hlFld-ContribAuthor ">Qiong  Xiao</span>, <span class="hlFld-ContribAuthor ">Chuan  Gao</span>, <span class="hlFld-ContribAuthor ">Quan-Yi  Zhao</span>, <span class="hlFld-ContribAuthor ">Qing-Lan  Li</span>, <span class="hlFld-ContribAuthor ">Mei  Ye</span>, <span class="hlFld-ContribAuthor ">Lei  Zhang</span>, <span class="hlFld-ContribAuthor ">Lian-Yun  Li</span>, <span class="hlFld-ContribAuthor ">Min  Wu</span>. </span><span class="cited-content_cbyCitation_article-title">Histone demethylase KDM3A is required for enhancer activation of hippo target genes in colorectal cancer. </span><span class="cited-content_cbyCitation_journal-name">Nucleic Acids Research</span><span> <strong>2019,</strong> <em>47 </em>
                                    (5)
                                     , 2349-2364. <a href="https://doi.org/10.1093/nar/gky1317" title="DOI URL">https://doi.org/10.1093/nar/gky1317</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/nar/gky1317&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fnar%2Fgky1317%26sid%3Dliteratum%253Aachs%26jtitle%3DNucleic%2520Acids%2520Research%26atitle%3DHistone%252Bdemethylase%252BKDM3A%252Bis%252Brequired%252Bfor%252Benhancer%252Bactivation%252Bof%252Bhippo%252Btarget%252Bgenes%252Bin%252Bcolorectal%252Bcancer%26aulast%3DWang%26aufirst%3DHui-Yi%26date%3D2019%26date%3D2019%26volume%3D47%26issue%3D5%26spage%3D2349%26epage%3D2364" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Giulia  Stazi</span>, <span class="hlFld-ContribAuthor ">Clemens  Zwergel</span>, <span class="hlFld-ContribAuthor ">Sergio  Valente</span>. </span><span class="cited-content_cbyCitation_article-title">Lysine Methyltransferases and Their Inhibitors. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 123-157. <a href="https://doi.org/10.1007/7355_2019_72" title="DOI URL">https://doi.org/10.1007/7355_2019_72</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/7355_2019_72&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F7355_2019_72%26sid%3Dliteratum%253Aachs%26atitle%3DLysine%252BMethyltransferases%252Band%252BTheir%252BInhibitors%26aulast%3DStazi%26aufirst%3DGiulia%26date%3D2019%26date%3D2019%26spage%3D123%26epage%3D157%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DChemical%252BEpigenetics%26aulast%3DMai%26aufirst%3DAntonello%26date%3D2020%26volume%3D33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Carol A.  Eisenberg</span>, <span class="hlFld-ContribAuthor ">Leonard M.  Eisenberg</span>. </span><span class="cited-content_cbyCitation_article-title">G9a and G9a-Like Histone Methyltransferases and Their Effect on Cell Phenotype, Embryonic Development, and Human Disease. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 399-433. <a href="https://doi.org/10.1007/978-3-030-14792-1_16" title="DOI URL">https://doi.org/10.1007/978-3-030-14792-1_16</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-3-030-14792-1_16&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-3-030-14792-1_16%26sid%3Dliteratum%253Aachs%26atitle%3DG9a%252Band%252BG9a-Like%252BHistone%252BMethyltransferases%252Band%252BTheir%252BEffect%252Bon%252BCell%252BPhenotype%25252C%252BEmbryonic%252BDevelopment%25252C%252Band%252BHuman%252BDisease%26aulast%3DEisenberg%26aufirst%3DCarol%2BA.%26date%3D2019%26date%3D2019%26spage%3D399%26epage%3D433%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DThe%252BDNA%25252C%252BRNA%25252C%252Band%252BHistone%252BMethylomes%26aulast%3DJurga%26aufirst%3DStefan%26date%3D2019%26volume%3D10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Guoqiang  Chen</span>, <span class="hlFld-ContribAuthor ">Xiaqing  Yu</span>, <span class="hlFld-ContribAuthor ">Min  Zhang</span>, <span class="hlFld-ContribAuthor ">Aiwen  Zheng</span>, <span class="hlFld-ContribAuthor ">Zhennan  Wang</span>, <span class="hlFld-ContribAuthor ">Yufang  Zuo</span>, <span class="hlFld-ContribAuthor ">Qilian  Liang</span>, <span class="hlFld-ContribAuthor ">Danxian  Jiang</span>, <span class="hlFld-ContribAuthor ">Yaqing  Chen</span>, <span class="hlFld-ContribAuthor ">Lingqin  Zhao</span>, <span class="hlFld-ContribAuthor ">Liyuan  Jiang</span>, <span class="hlFld-ContribAuthor ">Dan  Li</span>, <span class="hlFld-ContribAuthor ">Sihai  Liao</span>. </span><span class="cited-content_cbyCitation_article-title">Inhibition of Euchromatic Histone Lysine Methyltransferase 2 (EHMT2) Suppresses the Proliferation and Invasion of Cervical Cancer Cells. </span><span class="cited-content_cbyCitation_journal-name">Cytogenetic and Genome Research</span><span> <strong>2019,</strong> <em>158 </em>
                                    (4)
                                     , 205-212. <a href="https://doi.org/10.1159/000502072" title="DOI URL">https://doi.org/10.1159/000502072</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1159/000502072&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1159%2F000502072%26sid%3Dliteratum%253Aachs%26jtitle%3DCytogenetic%2520and%2520Genome%2520Research%26atitle%3DInhibition%252Bof%252BEuchromatic%252BHistone%252BLysine%252BMethyltransferase%252B2%252B%252528EHMT2%252529%252BSuppresses%252Bthe%252BProliferation%252Band%252BInvasion%252Bof%252BCervical%252BCancer%252BCells%26aulast%3DChen%26aufirst%3DGuoqiang%26date%3D2019%26date%3D2019%26volume%3D158%26issue%3D4%26spage%3D205%26epage%3D212" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tomoya  Hirano</span>, <span class="hlFld-ContribAuthor ">Shuichi  Mori</span>, <span class="hlFld-ContribAuthor ">Hiroyuki  Kagechika</span>. </span><span class="cited-content_cbyCitation_article-title">Recent Advances in Chemical Tools for the Regulation and Study of Protein Lysine Methyltransferases. </span><span class="cited-content_cbyCitation_journal-name">The Chemical Record</span><span> <strong>2018,</strong> <em>18 </em>
                                    (12)
                                     , 1745-1759. <a href="https://doi.org/10.1002/tcr.201800034" title="DOI URL">https://doi.org/10.1002/tcr.201800034</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/tcr.201800034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Ftcr.201800034%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Chemical%2520Record%26atitle%3DRecent%252BAdvances%252Bin%252BChemical%252BTools%252Bfor%252Bthe%252BRegulation%252Band%252BStudy%252Bof%252BProtein%252BLysine%252BMethyltransferases%26aulast%3DHirano%26aufirst%3DTomoya%26date%3D2018%26date%3D2018%26volume%3D18%26issue%3D12%26spage%3D1745%26epage%3D1759" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">S. P.  Rowbotham</span>, <span class="hlFld-ContribAuthor ">F.  Li</span>, <span class="hlFld-ContribAuthor ">A. F. M.  Dost</span>, <span class="hlFld-ContribAuthor ">S. M.  Louie</span>, <span class="hlFld-ContribAuthor ">B. P.  Marsh</span>, <span class="hlFld-ContribAuthor ">P.  Pessina</span>, <span class="hlFld-ContribAuthor ">C. R.  Anbarasu</span>, <span class="hlFld-ContribAuthor ">C. F.  Brainson</span>, <span class="hlFld-ContribAuthor ">S. J.  Tuminello</span>, <span class="hlFld-ContribAuthor ">A.  Lieberman</span>, <span class="hlFld-ContribAuthor ">S.  Ryeom</span>, <span class="hlFld-ContribAuthor ">T. M.  Schlaeger</span>, <span class="hlFld-ContribAuthor ">B. J.  Aronow</span>, <span class="hlFld-ContribAuthor ">H.  Watanabe</span>, <span class="hlFld-ContribAuthor ">K. K.  Wong</span>, <span class="hlFld-ContribAuthor ">C. F.  Kim</span>. </span><span class="cited-content_cbyCitation_article-title">H3K9 methyltransferases and demethylases control lung tumor-propagating cells and lung cancer progression. </span><span class="cited-content_cbyCitation_journal-name">Nature Communications</span><span> <strong>2018,</strong> <em>9 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41467-018-07077-1" title="DOI URL">https://doi.org/10.1038/s41467-018-07077-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41467-018-07077-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41467-018-07077-1%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Communications%26atitle%3DH3K9%252Bmethyltransferases%252Band%252Bdemethylases%252Bcontrol%252Blung%252Btumor-propagating%252Bcells%252Band%252Blung%252Bcancer%252Bprogression%26aulast%3DRowbotham%26aufirst%3DS.%2BP.%26date%3D2018%26date%3D2018%26volume%3D9%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lanlan  Zang</span>, <span class="hlFld-ContribAuthor ">Shukkoor Muhammed  Kondengaden</span>, <span class="hlFld-ContribAuthor ">Fengyuan  Che</span>, <span class="hlFld-ContribAuthor ">Lijuan  Wang</span>, <span class="hlFld-ContribAuthor ">Xueyuan  Heng</span>. </span><span class="cited-content_cbyCitation_article-title">Potential Epigenetic-Based Therapeutic Targets for Glioma. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Molecular Neuroscience</span><span> <strong>2018,</strong> <em>11 </em><a href="https://doi.org/10.3389/fnmol.2018.00408" title="DOI URL">https://doi.org/10.3389/fnmol.2018.00408</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fnmol.2018.00408&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffnmol.2018.00408%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Molecular%2520Neuroscience%26atitle%3DPotential%252BEpigenetic-Based%252BTherapeutic%252BTargets%252Bfor%252BGlioma%26aulast%3DZang%26aufirst%3DLanlan%26date%3D2018%26date%3D2018%26volume%3D11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mohamed H.  Hekal</span>, <span class="hlFld-ContribAuthor ">Fatma S. M.  Abu El-Azm</span>. </span><span class="cited-content_cbyCitation_article-title">New potential antitumor quinazolinones derived from dynamic 2-undecyl benzoxazinone: Synthesis and cytotoxic evaluation. </span><span class="cited-content_cbyCitation_journal-name">Synthetic Communications</span><span> <strong>2018,</strong> <em>48 </em>
                                    (18)
                                     , 2391-2402. <a href="https://doi.org/10.1080/00397911.2018.1490433" title="DOI URL">https://doi.org/10.1080/00397911.2018.1490433</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/00397911.2018.1490433&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F00397911.2018.1490433%26sid%3Dliteratum%253Aachs%26jtitle%3DSynthetic%2520Communications%26atitle%3DNew%252Bpotential%252Bantitumor%252Bquinazolinones%252Bderived%252Bfrom%252Bdynamic%252B2-undecyl%252Bbenzoxazinone%25253A%252BSynthesis%252Band%252Bcytotoxic%252Bevaluation%26aulast%3DHekal%26aufirst%3DMohamed%2BH.%26date%3D2018%26date%3D2018%26volume%3D48%26issue%3D18%26spage%3D2391%26epage%3D2402" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gonçalo C.  Vilhais-Neto</span>, <span class="hlFld-ContribAuthor ">Marjorie  Fournier</span>, <span class="hlFld-ContribAuthor ">Jean-Luc  Plassat</span>, <span class="hlFld-ContribAuthor ">Mihaela E.  Sardiu</span>, <span class="hlFld-ContribAuthor ">Anita  Saraf</span>, <span class="hlFld-ContribAuthor ">Jean-Marie  Garnier</span>, <span class="hlFld-ContribAuthor ">Mitsuji  Maruhashi</span>, <span class="hlFld-ContribAuthor ">Laurence  Florens</span>, <span class="hlFld-ContribAuthor ">Michael P.  Washburn</span>, <span class="hlFld-ContribAuthor ">Olivier  Pourquié</span>. </span><span class="cited-content_cbyCitation_article-title">The WHHERE coactivator complex is required for retinoic acid-dependent regulation of embryonic symmetry. </span><span class="cited-content_cbyCitation_journal-name">Nature Communications</span><span> <strong>2017,</strong> <em>8 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41467-017-00593-6" title="DOI URL">https://doi.org/10.1038/s41467-017-00593-6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41467-017-00593-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41467-017-00593-6%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Communications%26atitle%3DThe%252BWHHERE%252Bcoactivator%252Bcomplex%252Bis%252Brequired%252Bfor%252Bretinoic%252Bacid-dependent%252Bregulation%252Bof%252Bembryonic%252Bsymmetry%26aulast%3DVilhais-Neto%26aufirst%3DGon%25C3%25A7alo%2BC.%26date%3D2017%26date%3D2017%26volume%3D8%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yiqun  Huang</span>, <span class="hlFld-ContribAuthor ">Yong  Zou</span>, <span class="hlFld-ContribAuthor ">Luhui  Lin</span>, <span class="hlFld-ContribAuthor ">Xudong  Ma</span>, <span class="hlFld-ContribAuthor ">Xiaohong  Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Effect of BIX-01294 on proliferation, apoptosis and histone methylation of acute T lymphoblastic leukemia cells. </span><span class="cited-content_cbyCitation_journal-name">Leukemia Research</span><span> <strong>2017,</strong> <em>62 </em>, 34-39. <a href="https://doi.org/10.1016/j.leukres.2017.09.015" title="DOI URL">https://doi.org/10.1016/j.leukres.2017.09.015</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.leukres.2017.09.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.leukres.2017.09.015%26sid%3Dliteratum%253Aachs%26jtitle%3DLeukemia%2520Research%26atitle%3DEffect%252Bof%252BBIX-01294%252Bon%252Bproliferation%25252C%252Bapoptosis%252Band%252Bhistone%252Bmethylation%252Bof%252Bacute%252BT%252Blymphoblastic%252Bleukemia%252Bcells%26aulast%3DHuang%26aufirst%3DYiqun%26date%3D2017%26volume%3D62%26spage%3D34%26epage%3D39" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">David A.  Vierra</span>, <span class="hlFld-ContribAuthor ">Jada L.  Garzon</span>, <span class="hlFld-ContribAuthor ">Meghan A.  Rego</span>, <span class="hlFld-ContribAuthor ">Morganne M.  Adroved</span>, <span class="hlFld-ContribAuthor ">Maurizio  Mauro</span>, <span class="hlFld-ContribAuthor ">Niall G.  Howlett</span>. </span><span class="cited-content_cbyCitation_article-title">Modulation of the Fanconi anemia pathway
              via
              chemically induced changes in chromatin structure. </span><span class="cited-content_cbyCitation_journal-name">Oncotarget</span><span> <strong>2017,</strong> <em>8 </em>
                                    (44)
                                     , 76443-76457. <a href="https://doi.org/10.18632/oncotarget.19470" title="DOI URL">https://doi.org/10.18632/oncotarget.19470</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.18632/oncotarget.19470&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.18632%2Foncotarget.19470%26sid%3Dliteratum%253Aachs%26jtitle%3DOncotarget%26atitle%3DModulation%252Bof%252Bthe%252BFanconi%252Banemia%252Bpathway%252Bvia%252Bchemically%252Binduced%252Bchanges%252Bin%252Bchromatin%252Bstructure%26aulast%3DVierra%26aufirst%3DDavid%2BA.%26date%3D2017%26date%3D2017%26volume%3D8%26issue%3D44%26spage%3D76443%26epage%3D76457" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Coralie  Poulard</span>, <span class="hlFld-ContribAuthor ">Danielle  Bittencourt</span>, <span class="hlFld-ContribAuthor ">Dai‐Ying  Wu</span>, <span class="hlFld-ContribAuthor ">Yixin  Hu</span>, <span class="hlFld-ContribAuthor ">Daniel S  Gerke</span>, <span class="hlFld-ContribAuthor ">Michael R  Stallcup</span>. </span><span class="cited-content_cbyCitation_article-title">A post‐translational modification switch controls coactivator function of histone methyltransferases G9a and GLP. </span><span class="cited-content_cbyCitation_journal-name">EMBO reports</span><span> <strong>2017,</strong> <em>18 </em>
                                    (8)
                                     , 1442-1459. <a href="https://doi.org/10.15252/embr.201744060" title="DOI URL">https://doi.org/10.15252/embr.201744060</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.15252/embr.201744060&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.15252%2Fembr.201744060%26sid%3Dliteratum%253Aachs%26jtitle%3DEMBO%2520reports%26atitle%3DA%252Bpost%2525E2%252580%252590translational%252Bmodification%252Bswitch%252Bcontrols%252Bcoactivator%252Bfunction%252Bof%252Bhistone%252Bmethyltransferases%252BG9a%252Band%252BGLP%26aulast%3DPoulard%26aufirst%3DCoralie%26date%3D2017%26date%3D2017%26volume%3D18%26issue%3D8%26spage%3D1442%26epage%3D1459" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yan  Xiong</span>, <span class="hlFld-ContribAuthor ">Fengling  Li</span>, <span class="hlFld-ContribAuthor ">Nicolas  Babault</span>, <span class="hlFld-ContribAuthor ">Hong  Wu</span>, <span class="hlFld-ContribAuthor ">Aiping  Dong</span>, <span class="hlFld-ContribAuthor ">Hong  Zeng</span>, <span class="hlFld-ContribAuthor ">Xin  Chen</span>, <span class="hlFld-ContribAuthor ">Cheryl H.  Arrowsmith</span>, <span class="hlFld-ContribAuthor ">Peter J.  Brown</span>, <span class="hlFld-ContribAuthor ">Jing  Liu</span>, <span class="hlFld-ContribAuthor ">Masoud  Vedadi</span>, <span class="hlFld-ContribAuthor ">Jian  Jin</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-activity relationship studies of G9a-like protein (GLP) inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2017,</strong> <em>25 </em>
                                    (16)
                                     , 4414-4423. <a href="https://doi.org/10.1016/j.bmc.2017.06.021" title="DOI URL">https://doi.org/10.1016/j.bmc.2017.06.021</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2017.06.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2017.06.021%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DStructure-activity%252Brelationship%252Bstudies%252Bof%252BG9a-like%252Bprotein%252B%252528GLP%252529%252Binhibitors%26aulast%3DXiong%26aufirst%3DYan%26date%3D2017%26volume%3D25%26issue%3D16%26spage%3D4414%26epage%3D4423" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jin  Zhang</span>, <span class="hlFld-ContribAuthor ">Dahong  Yao</span>, <span class="hlFld-ContribAuthor ">Yingnan  Jiang</span>, <span class="hlFld-ContribAuthor ">Jian  Huang</span>, <span class="hlFld-ContribAuthor ">Shilin  Yang</span>, <span class="hlFld-ContribAuthor ">Jinhui  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis and biological evaluation of benzimidazole derivatives as the G9a Histone Methyltransferase inhibitors that induce autophagy and apoptosis of breast cancer cells. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2017,</strong> <em>72 </em>, 168-181. <a href="https://doi.org/10.1016/j.bioorg.2017.04.005" title="DOI URL">https://doi.org/10.1016/j.bioorg.2017.04.005</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2017.04.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2017.04.005%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DSynthesis%252Band%252Bbiological%252Bevaluation%252Bof%252Bbenzimidazole%252Bderivatives%252Bas%252Bthe%252BG9a%252BHistone%252BMethyltransferase%252Binhibitors%252Bthat%252Binduce%252Bautophagy%252Band%252Bapoptosis%252Bof%252Bbreast%252Bcancer%252Bcells%26aulast%3DZhang%26aufirst%3DJin%26date%3D2017%26volume%3D72%26spage%3D168%26epage%3D181" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kayleigh L.  Brocklesby</span>, <span class="hlFld-ContribAuthor ">Jennifer S.  Waby</span>, <span class="hlFld-ContribAuthor ">Chris  Cawthorne</span>, <span class="hlFld-ContribAuthor ">Graham  Smith</span>. </span><span class="cited-content_cbyCitation_article-title">An alternative synthesis of Vandetanib (Caprelsa™) via a microwave accelerated Dimroth rearrangement. </span><span class="cited-content_cbyCitation_journal-name">Tetrahedron Letters</span><span> <strong>2017,</strong> <em>58 </em>
                                    (15)
                                     , 1467-1469. <a href="https://doi.org/10.1016/j.tetlet.2017.02.082" title="DOI URL">https://doi.org/10.1016/j.tetlet.2017.02.082</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tetlet.2017.02.082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tetlet.2017.02.082%26sid%3Dliteratum%253Aachs%26jtitle%3DTetrahedron%2520Letters%26atitle%3DAn%252Balternative%252Bsynthesis%252Bof%252BVandetanib%252B%252528Caprelsa%2525E2%252584%2525A2%252529%252Bvia%252Ba%252Bmicrowave%252Baccelerated%252BDimroth%252Brearrangement%26aulast%3DBrocklesby%26aufirst%3DKayleigh%2BL.%26date%3D2017%26volume%3D58%26issue%3D15%26spage%3D1467%26epage%3D1469" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jake  Shortt</span>, <span class="hlFld-ContribAuthor ">Christopher J.  Ott</span>, <span class="hlFld-ContribAuthor ">Ricky W.  Johnstone</span>, <span class="hlFld-ContribAuthor ">James E.  Bradner</span>. </span><span class="cited-content_cbyCitation_article-title">A chemical probe toolbox for dissecting the cancer epigenome. </span><span class="cited-content_cbyCitation_journal-name">Nature Reviews Cancer</span><span> <strong>2017,</strong> <em>17 </em>
                                    (3)
                                     , 160-183. <a href="https://doi.org/10.1038/nrc.2016.148" title="DOI URL">https://doi.org/10.1038/nrc.2016.148</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/nrc.2016.148&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fnrc.2016.148%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Reviews%2520Cancer%26atitle%3DA%252Bchemical%252Bprobe%252Btoolbox%252Bfor%252Bdissecting%252Bthe%252Bcancer%252Bepigenome%26aulast%3DShortt%26aufirst%3DJake%26date%3D2017%26date%3D2017%26volume%3D17%26issue%3D3%26spage%3D160%26epage%3D183" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yuna  Kim</span>, <span class="hlFld-ContribAuthor ">Hyeong-Min  Lee</span>, <span class="hlFld-ContribAuthor ">Yan  Xiong</span>, <span class="hlFld-ContribAuthor ">Noah  Sciaky</span>, <span class="hlFld-ContribAuthor ">Samuel W  Hulbert</span>, <span class="hlFld-ContribAuthor ">Xinyu  Cao</span>, <span class="hlFld-ContribAuthor ">Jeffrey I  Everitt</span>, <span class="hlFld-ContribAuthor ">Jian  Jin</span>, <span class="hlFld-ContribAuthor ">Bryan L  Roth</span>, <span class="hlFld-ContribAuthor ">Yong-hui  Jiang</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting the histone methyltransferase G9a activates imprinted genes and improves survival of a mouse model of Prader–Willi syndrome. </span><span class="cited-content_cbyCitation_journal-name">Nature Medicine</span><span> <strong>2017,</strong> <em>23 </em>
                                    (2)
                                     , 213-222. <a href="https://doi.org/10.1038/nm.4257" title="DOI URL">https://doi.org/10.1038/nm.4257</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/nm.4257&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fnm.4257%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Medicine%26atitle%3DTargeting%252Bthe%252Bhistone%252Bmethyltransferase%252BG9a%252Bactivates%252Bimprinted%252Bgenes%252Band%252Bimproves%252Bsurvival%252Bof%252Ba%252Bmouse%252Bmodel%252Bof%252BPrader%2525E2%252580%252593Willi%252Bsyndrome%26aulast%3DKim%26aufirst%3DYuna%26date%3D2017%26date%3D2016%26volume%3D23%26issue%3D2%26spage%3D213%26epage%3D222" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">D.  Harvey</span>, <span class="hlFld-ContribAuthor ">M.  Foley</span>. </span><span class="cited-content_cbyCitation_article-title">Inhibitors of Epigenetic Regulation in Cancer. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2017,</strong>,, 281-307. <a href="https://doi.org/10.1016/B978-0-12-409547-2.12393-5" title="DOI URL">https://doi.org/10.1016/B978-0-12-409547-2.12393-5</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-409547-2.12393-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-409547-2.12393-5%26sid%3Dliteratum%253Aachs%26atitle%3DInhibitors%252Bof%252BEpigenetic%252BRegulation%252Bin%252BCancer%26aulast%3DHarvey%26aufirst%3DD.%26date%3D2017%26spage%3D281%26epage%3D307%26pub%3DElsevier%26atitle%3DComprehensive%252BMedicinal%252BChemistry%252BIII%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sandeep  Sundriyal</span>, <span class="hlFld-ContribAuthor ">Patty B.  Chen</span>, <span class="hlFld-ContribAuthor ">Alexandra S.  Lubin</span>, <span class="hlFld-ContribAuthor ">Gregor A.  Lueg</span>, <span class="hlFld-ContribAuthor ">Fengling  Li</span>, <span class="hlFld-ContribAuthor ">Andrew J. P.  White</span>, <span class="hlFld-ContribAuthor ">Nicholas A.  Malmquist</span>, <span class="hlFld-ContribAuthor ">Masoud  Vedadi</span>, <span class="hlFld-ContribAuthor ">Artur  Scherf</span>, <span class="hlFld-ContribAuthor ">Matthew J.  Fuchter</span>. </span><span class="cited-content_cbyCitation_article-title">Histone lysine methyltransferase structure activity relationships that allow for segregation of G9a inhibition and anti-Plasmodium activity. </span><span class="cited-content_cbyCitation_journal-name">MedChemComm</span><span> <strong>2017,</strong> <em>8 </em>
                                    (5)
                                     , 1069-1092. <a href="https://doi.org/10.1039/C7MD00052A" title="DOI URL">https://doi.org/10.1039/C7MD00052A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C7MD00052A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC7MD00052A%26sid%3Dliteratum%253Aachs%26jtitle%3DMedChemComm%26atitle%3DHistone%252Blysine%252Bmethyltransferase%252Bstructure%252Bactivity%252Brelationships%252Bthat%252Ballow%252Bfor%252Bsegregation%252Bof%252BG9a%252Binhibition%252Band%252Banti-Plasmodium%252Bactivity%26aulast%3DSundriyal%26aufirst%3DSandeep%26date%3D2017%26date%3D2017%26volume%3D8%26issue%3D5%26spage%3D1069%26epage%3D1092" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shagufta  Shagufta</span>, <span class="hlFld-ContribAuthor ">Irshad  Ahmad</span>. </span><span class="cited-content_cbyCitation_article-title">An insight into the therapeutic potential of quinazoline derivatives as anticancer agents. </span><span class="cited-content_cbyCitation_journal-name">MedChemComm</span><span> <strong>2017,</strong> <em>8 </em>
                                    (5)
                                     , 871-885. <a href="https://doi.org/10.1039/C7MD00097A" title="DOI URL">https://doi.org/10.1039/C7MD00097A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C7MD00097A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC7MD00097A%26sid%3Dliteratum%253Aachs%26jtitle%3DMedChemComm%26atitle%3DAn%252Binsight%252Binto%252Bthe%252Btherapeutic%252Bpotential%252Bof%252Bquinazoline%252Bderivatives%252Bas%252Banticancer%252Bagents%26aulast%3DShagufta%26aufirst%3DShagufta%26date%3D2017%26date%3D2017%26volume%3D8%26issue%3D5%26spage%3D871%26epage%3D885" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ciro  Milite</span>, <span class="hlFld-ContribAuthor ">Alessandra  Feoli</span>, <span class="hlFld-ContribAuthor ">Monica  Viviano</span>, <span class="hlFld-ContribAuthor ">Donatella  Rescigno</span>, <span class="hlFld-ContribAuthor ">Agostino  Cianciulli</span>, <span class="hlFld-ContribAuthor ">Amodio Luca  Balzano</span>, <span class="hlFld-ContribAuthor ">Antonello  Mai</span>, <span class="hlFld-ContribAuthor ">Sabrina  Castellano</span>, <span class="hlFld-ContribAuthor ">Gianluca  Sbardella</span>. </span><span class="cited-content_cbyCitation_article-title">The emerging role of lysine methyltransferase SETD8 in human diseases. </span><span class="cited-content_cbyCitation_journal-name">Clinical Epigenetics</span><span> <strong>2016,</strong> <em>8 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13148-016-0268-4" title="DOI URL">https://doi.org/10.1186/s13148-016-0268-4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13148-016-0268-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13148-016-0268-4%26sid%3Dliteratum%253Aachs%26jtitle%3DClinical%2520Epigenetics%26atitle%3DThe%252Bemerging%252Brole%252Bof%252Blysine%252Bmethyltransferase%252BSETD8%252Bin%252Bhuman%252Bdiseases%26aulast%3DMilite%26aufirst%3DCiro%26date%3D2016%26date%3D2016%26volume%3D8%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shukkoor M.  Kondengaden</span>, <span class="hlFld-ContribAuthor ">Liu-fei  Luo</span>, <span class="hlFld-ContribAuthor ">Kenneth  Huang</span>, <span class="hlFld-ContribAuthor ">Mengyuan  Zhu</span>, <span class="hlFld-ContribAuthor ">Lanlan  Zang</span>, <span class="hlFld-ContribAuthor ">Eudoxie  Bataba</span>, <span class="hlFld-ContribAuthor ">Runling  Wang</span>, <span class="hlFld-ContribAuthor ">Cheng  Luo</span>, <span class="hlFld-ContribAuthor ">Binghe  Wang</span>, <span class="hlFld-ContribAuthor ">Keqin Kathy  Li</span>, <span class="hlFld-ContribAuthor ">Peng George  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of novel small molecule inhibitors of lysine methyltransferase G9a and their mechanism in leukemia cell lines. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>122 </em>, 382-393. <a href="https://doi.org/10.1016/j.ejmech.2016.06.028" title="DOI URL">https://doi.org/10.1016/j.ejmech.2016.06.028</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2016.06.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2016.06.028%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Bnovel%252Bsmall%252Bmolecule%252Binhibitors%252Bof%252Blysine%252Bmethyltransferase%252BG9a%252Band%252Btheir%252Bmechanism%252Bin%252Bleukemia%252Bcell%252Blines%26aulast%3DKondengaden%26aufirst%3DShukkoor%2BM.%26date%3D2016%26volume%3D122%26spage%3D382%26epage%3D393" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Qing  Liu</span>, <span class="hlFld-ContribAuthor ">Ming-wei  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Histone lysine methyltransferases as anti-cancer targets for drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Acta Pharmacologica Sinica</span><span> <strong>2016,</strong> <em>37 </em>
                                    (10)
                                     , 1273-1280. <a href="https://doi.org/10.1038/aps.2016.64" title="DOI URL">https://doi.org/10.1038/aps.2016.64</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/aps.2016.64&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Faps.2016.64%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Pharmacologica%2520Sinica%26atitle%3DHistone%252Blysine%252Bmethyltransferases%252Bas%252Banti-cancer%252Btargets%252Bfor%252Bdrug%252Bdiscovery%26aulast%3DLiu%26aufirst%3DQing%26date%3D2016%26date%3D2016%26volume%3D37%26issue%3D10%26spage%3D1273%26epage%3D1280" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Uttara  Soumyanarayanan</span>, <span class="hlFld-ContribAuthor ">Brian W  Dymock</span>. </span><span class="cited-content_cbyCitation_article-title">Recently discovered EZH2 and EHMT2 (G9a) inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2016,</strong> <em>8 </em>
                                    (13)
                                     , 1635-1654. <a href="https://doi.org/10.4155/fmc-2016-0096" title="DOI URL">https://doi.org/10.4155/fmc-2016-0096</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2016-0096&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2016-0096%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DRecently%252Bdiscovered%252BEZH2%252Band%252BEHMT2%252B%252528G9a%252529%252Binhibitors%26aulast%3DSoumyanarayanan%26aufirst%3DUttara%26date%3D2016%26volume%3D8%26issue%3D13%26spage%3D1635%26epage%3D1654" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ciro  Milite</span>, <span class="hlFld-ContribAuthor ">Alessandra  Feoli</span>, <span class="hlFld-ContribAuthor ">Monica  Viviano</span>, <span class="hlFld-ContribAuthor ">Donatella  Rescigno</span>, <span class="hlFld-ContribAuthor ">Antonello  Mai</span>, <span class="hlFld-ContribAuthor ">Sabrina  Castellano</span>, <span class="hlFld-ContribAuthor ">Gianluca  Sbardella</span>. </span><span class="cited-content_cbyCitation_article-title">Progress in the Development of Lysine Methyltransferase SETD8 Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2016,</strong> <em>11 </em>
                                    (16)
                                     , 1680-1685. <a href="https://doi.org/10.1002/cmdc.201600272" title="DOI URL">https://doi.org/10.1002/cmdc.201600272</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201600272&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201600272%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DProgress%252Bin%252Bthe%252BDevelopment%252Bof%252BLysine%252BMethyltransferase%252BSETD8%252BInhibitors%26aulast%3DMilite%26aufirst%3DCiro%26date%3D2016%26date%3D2016%26volume%3D11%26issue%3D16%26spage%3D1680%26epage%3D1685" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dong  Cao</span>, <span class="hlFld-ContribAuthor ">Xiaoyan  Wang</span>, <span class="hlFld-ContribAuthor ">Lei  Lei</span>, <span class="hlFld-ContribAuthor ">Liang  Ma</span>, <span class="hlFld-ContribAuthor ">Fang  Wang</span>, <span class="hlFld-ContribAuthor ">Chunyu  Wang</span>, <span class="hlFld-ContribAuthor ">Minghai  Tang</span>, <span class="hlFld-ContribAuthor ">Wei  Xiang</span>, <span class="hlFld-ContribAuthor ">Taijin  Wang</span>, <span class="hlFld-ContribAuthor ">Hongyang  Li</span>, <span class="hlFld-ContribAuthor ">Lijuan  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis, in vitro and in vivo evaluation of novel substituted N-(4-(2-(4-benzylpiperazin-1-yl)ethoxy)phenyl)-N-methyl-quinazolin-4-amines as potent antitumor agents. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2016,</strong> <em>26 </em>
                                    (8)
                                     , 1931-1935. <a href="https://doi.org/10.1016/j.bmcl.2016.03.016" title="DOI URL">https://doi.org/10.1016/j.bmcl.2016.03.016</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2016.03.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2016.03.016%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DSynthesis%25252C%252Bin%252Bvitro%252Band%252Bin%252Bvivo%252Bevaluation%252Bof%252Bnovel%252Bsubstituted%252BN-%2525284-%2525282-%2525284-benzylpiperazin-1-yl%252529ethoxy%252529phenyl%252529-N-methyl-quinazolin-4-amines%252Bas%252Bpotent%252Bantitumor%252Bagents%26aulast%3DCao%26aufirst%3DDong%26date%3D2016%26volume%3D26%26issue%3D8%26spage%3D1931%26epage%3D1935" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Marie  Lopez</span>, <span class="hlFld-ContribAuthor ">Maxime  Leroy</span>, <span class="hlFld-ContribAuthor ">Chantal  Etievant</span>, <span class="hlFld-ContribAuthor ">Fréderic  Ausseil</span>, <span class="hlFld-ContribAuthor ">Alexandre  Gagnon</span>, <span class="hlFld-ContribAuthor ">Paola  Arimondo</span>. </span><span class="cited-content_cbyCitation_article-title">Drug Discovery Methods. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2016,</strong>,, 63-95. <a href="https://doi.org/10.1016/B978-0-12-802208-5.00003-5" title="DOI URL">https://doi.org/10.1016/B978-0-12-802208-5.00003-5</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-802208-5.00003-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-802208-5.00003-5%26sid%3Dliteratum%253Aachs%26atitle%3DDrug%252BDiscovery%252BMethods%26aulast%3DLopez%26aufirst%3DMarie%26date%3D2016%26spage%3D63%26epage%3D95%26pub%3DElsevier%26atitle%3DDrug%252BDiscovery%252Bin%252BCancer%252BEpigenetics%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kyle V.  Butler</span>, <span class="hlFld-ContribAuthor ">Kelsey  Bohn</span>, <span class="hlFld-ContribAuthor ">Christine A.  Hrycyna</span>, <span class="hlFld-ContribAuthor ">Jian  Jin</span>. </span><span class="cited-content_cbyCitation_article-title">Non-substrate based, small molecule inhibitors of the human isoprenylcysteine carboxyl methyltransferase. </span><span class="cited-content_cbyCitation_journal-name">MedChemComm</span><span> <strong>2016,</strong> <em>7 </em>
                                    (5)
                                     , 1016-1021. <a href="https://doi.org/10.1039/C6MD00130K" title="DOI URL">https://doi.org/10.1039/C6MD00130K</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C6MD00130K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC6MD00130K%26sid%3Dliteratum%253Aachs%26jtitle%3DMedChemComm%26atitle%3DNon-substrate%252Bbased%25252C%252Bsmall%252Bmolecule%252Binhibitors%252Bof%252Bthe%252Bhuman%252Bisoprenylcysteine%252Bcarboxyl%252Bmethyltransferase%26aulast%3DButler%26aufirst%3DKyle%2BV.%26date%3D2016%26date%3D2016%26volume%3D7%26issue%3D5%26spage%3D1016%26epage%3D1021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Edward  Curry</span>, <span class="hlFld-ContribAuthor ">Ian  Green</span>, <span class="hlFld-ContribAuthor ">Nadine  Chapman-Rothe</span>, <span class="hlFld-ContribAuthor ">Elham  Shamsaei</span>, <span class="hlFld-ContribAuthor ">Sarah  Kandil</span>, <span class="hlFld-ContribAuthor ">Fanny L  Cherblanc</span>, <span class="hlFld-ContribAuthor ">Luke  Payne</span>, <span class="hlFld-ContribAuthor ">Emma  Bell</span>, <span class="hlFld-ContribAuthor ">Thota  Ganesh</span>, <span class="hlFld-ContribAuthor ">Nitipol  Srimongkolpithak</span>, <span class="hlFld-ContribAuthor ">Joachim  Caron</span>, <span class="hlFld-ContribAuthor ">Fengling  Li</span>, <span class="hlFld-ContribAuthor ">Anthony G.  Uren</span>, <span class="hlFld-ContribAuthor ">James P.  Snyder</span>, <span class="hlFld-ContribAuthor ">Masoud  Vedadi</span>, <span class="hlFld-ContribAuthor ">Matthew J.  Fuchter</span>, <span class="hlFld-ContribAuthor ">Robert  Brown</span>. </span><span class="cited-content_cbyCitation_article-title">Dual EZH2 and EHMT2 histone methyltransferase inhibition increases biological efficacy in breast cancer cells. </span><span class="cited-content_cbyCitation_journal-name">Clinical Epigenetics</span><span> <strong>2015,</strong> <em>7 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13148-015-0118-9" title="DOI URL">https://doi.org/10.1186/s13148-015-0118-9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13148-015-0118-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13148-015-0118-9%26sid%3Dliteratum%253Aachs%26jtitle%3DClinical%2520Epigenetics%26atitle%3DDual%252BEZH2%252Band%252BEHMT2%252Bhistone%252Bmethyltransferase%252Binhibition%252Bincreases%252Bbiological%252Befficacy%252Bin%252Bbreast%252Bcancer%252Bcells%26aulast%3DCurry%26aufirst%3DEdward%26date%3D2015%26date%3D2015%26volume%3D7%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Minkui  Luo</span>. </span><span class="cited-content_cbyCitation_article-title">Inhibitors of protein methyltransferases as chemical tools. </span><span class="cited-content_cbyCitation_journal-name">Epigenomics</span><span> <strong>2015,</strong> <em>7 </em>
                                    (8)
                                     , 1327-1338. <a href="https://doi.org/10.2217/epi.15.87" title="DOI URL">https://doi.org/10.2217/epi.15.87</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2217/epi.15.87&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2217%2Fepi.15.87%26sid%3Dliteratum%253Aachs%26jtitle%3DEpigenomics%26atitle%3DInhibitors%252Bof%252Bprotein%252Bmethyltransferases%252Bas%252Bchemical%252Btools%26aulast%3DLuo%26aufirst%3DMinkui%26date%3D2015%26volume%3D7%26issue%3D8%26spage%3D1327%26epage%3D1338" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Natalia J.  Martinez</span>, <span class="hlFld-ContribAuthor ">Anton  Simeonov</span>. </span><span class="cited-content_cbyCitation_article-title">Cell-based assays to support the profiling of small molecules with histone methyltransferase and demethylase modulatory activity. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today: Technologies</span><span> <strong>2015,</strong> <em>18 </em>, 9-17. <a href="https://doi.org/10.1016/j.ddtec.2015.10.004" title="DOI URL">https://doi.org/10.1016/j.ddtec.2015.10.004</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ddtec.2015.10.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ddtec.2015.10.004%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technologies%26atitle%3DCell-based%252Bassays%252Bto%252Bsupport%252Bthe%252Bprofiling%252Bof%252Bsmall%252Bmolecules%252Bwith%252Bhistone%252Bmethyltransferase%252Band%252Bdemethylase%252Bmodulatory%252Bactivity%26aulast%3DMartinez%26aufirst%3DNatalia%2BJ.%26date%3D2015%26volume%3D18%26spage%3D9%26epage%3D17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Juan  Bayo</span>, <span class="hlFld-ContribAuthor ">Maithili P  Dalvi</span>, <span class="hlFld-ContribAuthor ">Elisabeth D  Martinez</span>. </span><span class="cited-content_cbyCitation_article-title">Successful strategies in the discovery of small-molecule epigenetic modulators with anticancer potential. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2015,</strong> <em>7 </em>
                                    (16)
                                     , 2243-2261. <a href="https://doi.org/10.4155/fmc.15.140" title="DOI URL">https://doi.org/10.4155/fmc.15.140</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc.15.140&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc.15.140%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DSuccessful%252Bstrategies%252Bin%252Bthe%252Bdiscovery%252Bof%252Bsmall-molecule%252Bepigenetic%252Bmodulators%252Bwith%252Banticancer%252Bpotential%26aulast%3DBayo%26aufirst%3DJuan%26date%3D2015%26volume%3D7%26issue%3D16%26spage%3D2243%26epage%3D2261" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chunngai  Hui</span>, <span class="hlFld-ContribAuthor ">Tao  Ye</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis of lysine methyltransferase inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Chemistry</span><span> <strong>2015,</strong> <em>3 </em><a href="https://doi.org/10.3389/fchem.2015.00044" title="DOI URL">https://doi.org/10.3389/fchem.2015.00044</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fchem.2015.00044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffchem.2015.00044%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Chemistry%26atitle%3DSynthesis%252Bof%252Blysine%252Bmethyltransferase%252Binhibitors%26aulast%3DHui%26aufirst%3DChunngai%26date%3D2015%26date%3D2015%26volume%3D3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">John  McGrath</span>, <span class="hlFld-ContribAuthor ">Patrick  Trojer</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting histone lysine methylation in cancer. </span><span class="cited-content_cbyCitation_journal-name">Pharmacology & Therapeutics</span><span> <strong>2015,</strong> <em>150 </em>, 1-22. <a href="https://doi.org/10.1016/j.pharmthera.2015.01.002" title="DOI URL">https://doi.org/10.1016/j.pharmthera.2015.01.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.pharmthera.2015.01.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.pharmthera.2015.01.002%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmacology%2520%2526%2520Therapeutics%26atitle%3DTargeting%252Bhistone%252Blysine%252Bmethylation%252Bin%252Bcancer%26aulast%3DMcGrath%26aufirst%3DJohn%26date%3D2015%26volume%3D150%26spage%3D1%26epage%3D22" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Steven A.  Belinsky</span>. </span><span class="cited-content_cbyCitation_article-title">Unmasking the Lung Cancer Epigenome. </span><span class="cited-content_cbyCitation_journal-name">Annual Review of Physiology</span><span> <strong>2015,</strong> <em>77 </em>
                                    (1)
                                     , 453-474. <a href="https://doi.org/10.1146/annurev-physiol-021014-072018" title="DOI URL">https://doi.org/10.1146/annurev-physiol-021014-072018</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1146/annurev-physiol-021014-072018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1146%2Fannurev-physiol-021014-072018%26sid%3Dliteratum%253Aachs%26jtitle%3DAnnual%2520Review%2520of%2520Physiology%26atitle%3DUnmasking%252Bthe%252BLung%252BCancer%252BEpigenome%26aulast%3DBelinsky%26aufirst%3DSteven%2BA.%26date%3D2015%26volume%3D77%26issue%3D1%26spage%3D453%26epage%3D474" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Aili  Chen</span>, <span class="hlFld-ContribAuthor ">Gang  Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Animal Model Study of Epigenetic Inhibitors. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2015,</strong>,, 447-477. <a href="https://doi.org/10.1016/B978-0-12-801080-8.00020-X" title="DOI URL">https://doi.org/10.1016/B978-0-12-801080-8.00020-X</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-801080-8.00020-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-801080-8.00020-X%26sid%3Dliteratum%253Aachs%26atitle%3DAnimal%252BModel%252BStudy%252Bof%252BEpigenetic%252BInhibitors%26aulast%3DChen%26aufirst%3DAili%26date%3D2015%26spage%3D447%26epage%3D477%26pub%3DElsevier%26atitle%3DEpigenetic%252BTechnological%252BApplications%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dashyant  Dhanak</span>, <span class="hlFld-ContribAuthor ">Paul  Jackson</span>. </span><span class="cited-content_cbyCitation_article-title">Development and classes of epigenetic drugs for cancer. </span><span class="cited-content_cbyCitation_journal-name">Biochemical and Biophysical Research Communications</span><span> <strong>2014,</strong> <em>455 </em>
                                    (1-2)
                                     , 58-69. <a href="https://doi.org/10.1016/j.bbrc.2014.07.006" title="DOI URL">https://doi.org/10.1016/j.bbrc.2014.07.006</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bbrc.2014.07.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bbrc.2014.07.006%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemical%2520and%2520Biophysical%2520Research%2520Communications%26atitle%3DDevelopment%252Band%252Bclasses%252Bof%252Bepigenetic%252Bdrugs%252Bfor%252Bcancer%26aulast%3DDhanak%26aufirst%3DDashyant%26date%3D2014%26volume%3D455%26issue%3D1-2%26spage%3D58%26epage%3D69" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nitipol  Srimongkolpithak</span>, <span class="hlFld-ContribAuthor ">Sandeep  Sundriyal</span>, <span class="hlFld-ContribAuthor ">Fengling  Li</span>, <span class="hlFld-ContribAuthor ">Masoud  Vedadi</span>, <span class="hlFld-ContribAuthor ">Matthew J.  Fuchter</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of 2,4-diamino-6,7-dimethoxyquinoline derivatives as G9a inhibitors. </span><span class="cited-content_cbyCitation_journal-name">MedChemComm</span><span> <strong>2014,</strong> <em>5 </em>
                                    (12)
                                     , 1821-1828. <a href="https://doi.org/10.1039/C4MD00274A" title="DOI URL">https://doi.org/10.1039/C4MD00274A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C4MD00274A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC4MD00274A%26sid%3Dliteratum%253Aachs%26jtitle%3DMedChemComm%26atitle%3DIdentification%252Bof%252B2%25252C4-diamino-6%25252C7-dimethoxyquinoline%252Bderivatives%252Bas%252BG9a%252Binhibitors%26aulast%3DSrimongkolpithak%26aufirst%3DNitipol%26date%3D2014%26date%3D2014%26volume%3D5%26issue%3D12%26spage%3D1821%26epage%3D1828" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anqi  Ma</span>, <span class="hlFld-ContribAuthor ">Wenyu  Yu</span>, <span class="hlFld-ContribAuthor ">Yan  Xiong</span>, <span class="hlFld-ContribAuthor ">Kyle V.  Butler</span>, <span class="hlFld-ContribAuthor ">Peter J.  Brown</span>, <span class="hlFld-ContribAuthor ">Jian  Jin</span>. </span><span class="cited-content_cbyCitation_article-title">Structure–activity relationship studies of SETD8 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">MedChemComm</span><span> <strong>2014,</strong> <em>5 </em>
                                    (12)
                                     , 1892-1898. <a href="https://doi.org/10.1039/C4MD00317A" title="DOI URL">https://doi.org/10.1039/C4MD00317A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C4MD00317A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC4MD00317A%26sid%3Dliteratum%253Aachs%26jtitle%3DMedChemComm%26atitle%3DStructure%2525E2%252580%252593activity%252Brelationship%252Bstudies%252Bof%252BSETD8%252Binhibitors%26aulast%3DMa%26aufirst%3DAnqi%26date%3D2014%26date%3D2014%26volume%3D5%26issue%3D12%26spage%3D1892%26epage%3D1898" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sandeep  Sundriyal</span>, <span class="hlFld-ContribAuthor ">Nicholas A.  Malmquist</span>, <span class="hlFld-ContribAuthor ">Joachim  Caron</span>, <span class="hlFld-ContribAuthor ">Scott  Blundell</span>, <span class="hlFld-ContribAuthor ">Feng  Liu</span>, <span class="hlFld-ContribAuthor ">Xin  Chen</span>, <span class="hlFld-ContribAuthor ">Nitipol  Srimongkolpithak</span>, <span class="hlFld-ContribAuthor ">Jian  Jin</span>, <span class="hlFld-ContribAuthor ">Susan A.  Charman</span>, <span class="hlFld-ContribAuthor ">Artur  Scherf</span>, <span class="hlFld-ContribAuthor ">Matthew J.  Fuchter</span>. </span><span class="cited-content_cbyCitation_article-title">Development of Diaminoquinazoline Histone Lysine Methyltransferase Inhibitors as Potent Blood-Stage Antimalarial Compounds. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2014,</strong> <em>9 </em>
                                    (10)
                                     , 2360-2373. <a href="https://doi.org/10.1002/cmdc.201402098" title="DOI URL">https://doi.org/10.1002/cmdc.201402098</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201402098&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201402098%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DDevelopment%252Bof%252BDiaminoquinazoline%252BHistone%252BLysine%252BMethyltransferase%252BInhibitors%252Bas%252BPotent%252BBlood-Stage%252BAntimalarial%252BCompounds%26aulast%3DSundriyal%26aufirst%3DSandeep%26date%3D2014%26date%3D2014%26volume%3D9%26issue%3D10%26spage%3D2360%26epage%3D2373" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiao-Xue  Ke</span>, <span class="hlFld-ContribAuthor ">Dunke  Zhang</span>, <span class="hlFld-ContribAuthor ">Shunqin  Zhu</span>, <span class="hlFld-ContribAuthor ">Qingyou  Xia</span>, <span class="hlFld-ContribAuthor ">Zhonghuai  Xiang</span>, <span class="hlFld-ContribAuthor ">Hongjuan  Cui</span>, . </span><span class="cited-content_cbyCitation_article-title">Inhibition of H3K9 Methyltransferase G9a Repressed Cell Proliferation and Induced Autophagy in Neuroblastoma Cells. </span><span class="cited-content_cbyCitation_journal-name">PLoS ONE</span><span> <strong>2014,</strong> <em>9 </em>
                                    (9)
                                     , e106962. <a href="https://doi.org/10.1371/journal.pone.0106962" title="DOI URL">https://doi.org/10.1371/journal.pone.0106962</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pone.0106962&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pone.0106962%26sid%3Dliteratum%253Aachs%26jtitle%3DPLoS%2520ONE%26atitle%3DInhibition%252Bof%252BH3K9%252BMethyltransferase%252BG9a%252BRepressed%252BCell%252BProliferation%252Band%252BInduced%252BAutophagy%252Bin%252BNeuroblastoma%252BCells%26aulast%3DKe%26aufirst%3DXiao-Xue%26date%3D2014%26date%3D2014%26volume%3D9%26issue%3D9%26spage%3De106962" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yanli  Liu</span>, <span class="hlFld-ContribAuthor ">Ke  Liu</span>, <span class="hlFld-ContribAuthor ">Su  Qin</span>, <span class="hlFld-ContribAuthor ">Chao  Xu</span>, <span class="hlFld-ContribAuthor ">Jinrong  Min</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetic targets and drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Pharmacology & Therapeutics</span><span> <strong>2014,</strong> <em>143 </em>
                                    (3)
                                     , 275-294. <a href="https://doi.org/10.1016/j.pharmthera.2014.03.007" title="DOI URL">https://doi.org/10.1016/j.pharmthera.2014.03.007</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.pharmthera.2014.03.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.pharmthera.2014.03.007%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmacology%2520%2526%2520Therapeutics%26atitle%3DEpigenetic%252Btargets%252Band%252Bdrug%252Bdiscovery%26aulast%3DLiu%26aufirst%3DYanli%26date%3D2014%26volume%3D143%26issue%3D3%26spage%3D275%26epage%3D294" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kyle D.  Konze</span>, <span class="hlFld-ContribAuthor ">Samantha G.  Pattenden</span>, <span class="hlFld-ContribAuthor ">Feng  Liu</span>, <span class="hlFld-ContribAuthor ">Dalia  Barsyte-Lovejoy</span>, <span class="hlFld-ContribAuthor ">Fengling  Li</span>, <span class="hlFld-ContribAuthor ">Jeremy M.  Simon</span>, <span class="hlFld-ContribAuthor ">Ian J.  Davis</span>, <span class="hlFld-ContribAuthor ">Masoud  Vedadi</span>, <span class="hlFld-ContribAuthor ">Jian  Jin</span>. </span><span class="cited-content_cbyCitation_article-title">A Chemical Tool for In Vitro and In Vivo Precipitation of Lysine Methyltransferase G9a. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2014,</strong> <em>9 </em>
                                    (3)
                                     , 549-553. <a href="https://doi.org/10.1002/cmdc.201300450" title="DOI URL">https://doi.org/10.1002/cmdc.201300450</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201300450&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201300450%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DA%252BChemical%252BTool%252Bfor%252BIn%252BVitro%252Band%252BIn%252BVivo%252BPrecipitation%252Bof%252BLysine%252BMethyltransferase%252BG9a%26aulast%3DKonze%26aufirst%3DKyle%2BD.%26date%3D2014%26date%3D2014%26volume%3D9%26issue%3D3%26spage%3D549%26epage%3D553" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Pasano  Bojang</span>, <span class="hlFld-ContribAuthor ">Kenneth S.  Ramos</span>. </span><span class="cited-content_cbyCitation_article-title">The promise and failures of epigenetic therapies for cancer treatment. </span><span class="cited-content_cbyCitation_journal-name">Cancer Treatment Reviews</span><span> <strong>2014,</strong> <em>40 </em>
                                    (1)
                                     , 153-169. <a href="https://doi.org/10.1016/j.ctrv.2013.05.009" title="DOI URL">https://doi.org/10.1016/j.ctrv.2013.05.009</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ctrv.2013.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ctrv.2013.05.009%26sid%3Dliteratum%253Aachs%26jtitle%3DCancer%2520Treatment%2520Reviews%26atitle%3DThe%252Bpromise%252Band%252Bfailures%252Bof%252Bepigenetic%252Btherapies%252Bfor%252Bcancer%252Btreatment%26aulast%3DBojang%26aufirst%3DPasano%26date%3D2014%26volume%3D40%26issue%3D1%26spage%3D153%26epage%3D169" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kamaldeep  Paul</span>, <span class="hlFld-ContribAuthor ">Alka  Sharma</span>, <span class="hlFld-ContribAuthor ">Vijay  Luxami</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis and in vitro antitumor evaluation of primary amine substituted quinazoline linked benzimidazole. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2014,</strong> <em>24 </em>
                                    (2)
                                     , 624-629. <a href="https://doi.org/10.1016/j.bmcl.2013.12.005" title="DOI URL">https://doi.org/10.1016/j.bmcl.2013.12.005</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2013.12.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2013.12.005%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DSynthesis%252Band%252Bin%252Bvitro%252Bantitumor%252Bevaluation%252Bof%252Bprimary%252Bamine%252Bsubstituted%252Bquinazoline%252Blinked%252Bbenzimidazole%26aulast%3DPaul%26aufirst%3DKamaldeep%26date%3D2014%26volume%3D24%26issue%3D2%26spage%3D624%26epage%3D629" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Stefan  Knapp</span>, <span class="hlFld-ContribAuthor ">Hilmar  Weinmann</span>. </span><span class="cited-content_cbyCitation_article-title">Small-Molecule Modulators for Epigenetics Targets. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2013,</strong> <em>8 </em>
                                    (11)
                                     , 1885-1891. <a href="https://doi.org/10.1002/cmdc.201300344" title="DOI URL">https://doi.org/10.1002/cmdc.201300344</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201300344&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201300344%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DSmall-Molecule%252BModulators%252Bfor%252BEpigenetics%252BTargets%26aulast%3DKnapp%26aufirst%3DStefan%26date%3D2013%26date%3D2013%26volume%3D8%26issue%3D11%26spage%3D1885%26epage%3D1891" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alka  Sharma</span>, <span class="hlFld-ContribAuthor ">Vijay  Luxami</span>, <span class="hlFld-ContribAuthor ">Kamaldeep  Paul</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis, single crystal and antitumor activities of benzimidazole–quinazoline hybrids. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2013,</strong> <em>23 </em>
                                    (11)
                                     , 3288-3294. <a href="https://doi.org/10.1016/j.bmcl.2013.03.107" title="DOI URL">https://doi.org/10.1016/j.bmcl.2013.03.107</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2013.03.107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2013.03.107%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DSynthesis%25252C%252Bsingle%252Bcrystal%252Band%252Bantitumor%252Bactivities%252Bof%252Bbenzimidazole%2525E2%252580%252593quinazoline%252Bhybrids%26aulast%3DSharma%26aufirst%3DAlka%26date%3D2013%26volume%3D23%26issue%3D11%26spage%3D3288%26epage%3D3294" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Pilar  Blancafort</span>, <span class="hlFld-ContribAuthor ">Jian  Jin</span>, <span class="hlFld-ContribAuthor ">Stephen  Frye</span>. </span><span class="cited-content_cbyCitation_article-title">Writing and Rewriting the Epigenetic Code of Cancer Cells: From Engineered Proteins to Small Molecules. </span><span class="cited-content_cbyCitation_journal-name">Molecular Pharmacology</span><span> <strong>2013,</strong> <em>83 </em>
                                    (3)
                                     , 563-576. <a href="https://doi.org/10.1124/mol.112.080697" title="DOI URL">https://doi.org/10.1124/mol.112.080697</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1124/mol.112.080697&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1124%2Fmol.112.080697%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Pharmacology%26atitle%3DWriting%252Band%252BRewriting%252Bthe%252BEpigenetic%252BCode%252Bof%252BCancer%252BCells%25253A%252BFrom%252BEngineered%252BProteins%252Bto%252BSmall%252BMolecules%26aulast%3DBlancafort%26aufirst%3DPilar%26date%3D2013%26date%3D2012%26volume%3D83%26issue%3D3%26spage%3D563%26epage%3D576" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yukihiro  Itoh</span>, <span class="hlFld-ContribAuthor ">Takayoshi  Suzuki</span>, <span class="hlFld-ContribAuthor ">Naoki  Miyata</span>. </span><span class="cited-content_cbyCitation_article-title">Small-molecular modulators of cancer-associated epigenetic mechanisms. </span><span class="cited-content_cbyCitation_journal-name">Molecular BioSystems</span><span> <strong>2013,</strong> <em>9 </em>
                                    (5)
                                     , 873. <a href="https://doi.org/10.1039/c3mb25410k" title="DOI URL">https://doi.org/10.1039/c3mb25410k</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/c3mb25410k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2Fc3mb25410k%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520BioSystems%26atitle%3DSmall-molecular%252Bmodulators%252Bof%252Bcancer-associated%252Bepigenetic%252Bmechanisms%26aulast%3DItoh%26aufirst%3DYukihiro%26date%3D2013%26volume%3D9%26issue%3D5%26spage%3D873" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">D.  Bittencourt</span>, <span class="hlFld-ContribAuthor ">D.-Y.  Wu</span>, <span class="hlFld-ContribAuthor ">K. W.  Jeong</span>, <span class="hlFld-ContribAuthor ">D. S.  Gerke</span>, <span class="hlFld-ContribAuthor ">L.  Herviou</span>, <span class="hlFld-ContribAuthor ">I.  Ianculescu</span>, <span class="hlFld-ContribAuthor ">R.  Chodankar</span>, <span class="hlFld-ContribAuthor ">K. D.  Siegmund</span>, <span class="hlFld-ContribAuthor ">M. R.  Stallcup</span>. </span><span class="cited-content_cbyCitation_article-title">G9a functions as a molecular scaffold for assembly of transcriptional coactivators on a subset of Glucocorticoid Receptor target genes. </span><span class="cited-content_cbyCitation_journal-name">Proceedings of the National Academy of Sciences</span><span> <strong>2012,</strong> <em>109 </em>
                                    (48)
                                     , 19673-19678. <a href="https://doi.org/10.1073/pnas.1211803109" title="DOI URL">https://doi.org/10.1073/pnas.1211803109</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1073/pnas.1211803109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1073%2Fpnas.1211803109%26sid%3Dliteratum%253Aachs%26jtitle%3DProceedings%2520of%2520the%2520National%2520Academy%2520of%2520Sciences%26atitle%3DG9a%252Bfunctions%252Bas%252Ba%252Bmolecular%252Bscaffold%252Bfor%252Bassembly%252Bof%252Btranscriptional%252Bcoactivators%252Bon%252Ba%252Bsubset%252Bof%252BGlucocorticoid%252BReceptor%252Btarget%252Bgenes%26aulast%3DBittencourt%26aufirst%3DD.%26date%3D2012%26date%3D2012%26volume%3D109%26issue%3D48%26spage%3D19673%26epage%3D19678" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">N. A.  Malmquist</span>, <span class="hlFld-ContribAuthor ">T. A.  Moss</span>, <span class="hlFld-ContribAuthor ">S.  Mecheri</span>, <span class="hlFld-ContribAuthor ">A.  Scherf</span>, <span class="hlFld-ContribAuthor ">M. J.  Fuchter</span>. </span><span class="cited-content_cbyCitation_article-title">Small-molecule histone methyltransferase inhibitors display rapid antimalarial activity against all blood stage forms in Plasmodium falciparum. </span><span class="cited-content_cbyCitation_journal-name">Proceedings of the National Academy of Sciences</span><span> <strong>2012,</strong> <em>109 </em>
                                    (41)
                                     , 16708-16713. <a href="https://doi.org/10.1073/pnas.1205414109" title="DOI URL">https://doi.org/10.1073/pnas.1205414109</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1073/pnas.1205414109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1073%2Fpnas.1205414109%26sid%3Dliteratum%253Aachs%26jtitle%3DProceedings%2520of%2520the%2520National%2520Academy%2520of%2520Sciences%26atitle%3DSmall-molecule%252Bhistone%252Bmethyltransferase%252Binhibitors%252Bdisplay%252Brapid%252Bantimalarial%252Bactivity%252Bagainst%252Ball%252Bblood%252Bstage%252Bforms%252Bin%252BPlasmodium%252Bfalciparum%26aulast%3DMalmquist%26aufirst%3DN.%2BA.%26date%3D2012%26date%3D2012%26volume%3D109%26issue%3D41%26spage%3D16708%26epage%3D16713" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">F  Cherblanc</span>, <span class="hlFld-ContribAuthor ">N  Chapman-Rothe</span>, <span class="hlFld-ContribAuthor ">R  Brown</span>, <span class="hlFld-ContribAuthor ">MJ  Fuchter</span>. </span><span class="cited-content_cbyCitation_article-title">Current limitations and future opportunities for epigenetic therapies. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2012,</strong> <em>4 </em>
                                    (4)
                                     , 425-446. <a href="https://doi.org/10.4155/fmc.12.7" title="DOI URL">https://doi.org/10.4155/fmc.12.7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc.12.7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc.12.7%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DCurrent%252Blimitations%252Band%252Bfuture%252Bopportunities%252Bfor%252Bepigenetic%252Btherapies%26aulast%3DCherblanc%26aufirst%3DF%26date%3D2012%26volume%3D4%26issue%3D4%26spage%3D425%26epage%3D446" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anup K.  Upadhyay</span>, <span class="hlFld-ContribAuthor ">Dante  Rotili</span>, <span class="hlFld-ContribAuthor ">Ji Woong  Han</span>, <span class="hlFld-ContribAuthor ">Ruogu  Hu</span>, <span class="hlFld-ContribAuthor ">Yanqi  Chang</span>, <span class="hlFld-ContribAuthor ">Donatella  Labella</span>, <span class="hlFld-ContribAuthor ">Xing  Zhang</span>, <span class="hlFld-ContribAuthor ">Young-sup  Yoon</span>, <span class="hlFld-ContribAuthor ">Antonello  Mai</span>, <span class="hlFld-ContribAuthor ">Xiaodong  Cheng</span>. </span><span class="cited-content_cbyCitation_article-title">An Analog of BIX-01294 Selectively Inhibits a Family of Histone H3 Lysine 9 Jumonji Demethylases. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Biology</span><span> <strong>2012,</strong> <em>416 </em>
                                    (3)
                                     , 319-327. <a href="https://doi.org/10.1016/j.jmb.2011.12.036" title="DOI URL">https://doi.org/10.1016/j.jmb.2011.12.036</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jmb.2011.12.036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jmb.2011.12.036%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Biology%26atitle%3DAn%252BAnalog%252Bof%252BBIX-01294%252BSelectively%252BInhibits%252Ba%252BFamily%252Bof%252BHistone%252BH3%252BLysine%252B9%252BJumonji%252BDemethylases%26aulast%3DUpadhyay%26aufirst%3DAnup%2BK.%26date%3D2012%26volume%3D416%26issue%3D3%26spage%3D319%26epage%3D327" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">John H.  Ryan</span>, <span class="hlFld-ContribAuthor ">Christopher  Hyland</span>, <span class="hlFld-ContribAuthor ">Adam G.  Meyer</span>, <span class="hlFld-ContribAuthor ">Jason A.  Smith</span>, <span class="hlFld-ContribAuthor ">JieXiang  Yin</span>. </span><span class="cited-content_cbyCitation_article-title">Seven-Membered Rings. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2012,</strong>,, 493-536. <a href="https://doi.org/10.1016/B978-0-08-096807-0.00016-6" title="DOI URL">https://doi.org/10.1016/B978-0-08-096807-0.00016-6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-08-096807-0.00016-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-08-096807-0.00016-6%26sid%3Dliteratum%253Aachs%26atitle%3DSeven-Membered%252BRings%26aulast%3DRyan%26aufirst%3DJohn%2BH.%26date%3D2012%26spage%3D493%26epage%3D536%26pub%3DElsevier%26date%3D2012%26volume%3D24" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yunlong  He</span>, <span class="hlFld-ContribAuthor ">Ilia  Korboukh</span>, <span class="hlFld-ContribAuthor ">Jian  Jin</span>, <span class="hlFld-ContribAuthor ">Jing  Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting protein lysine methylation and demethylation in cancers. </span><span class="cited-content_cbyCitation_journal-name">Acta Biochimica et Biophysica Sinica</span><span> <strong>2012,</strong> <em>44 </em>
                                    (1)
                                     , 70-79. <a href="https://doi.org/10.1093/abbs/gmr109" title="DOI URL">https://doi.org/10.1093/abbs/gmr109</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/abbs/gmr109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fabbs%2Fgmr109%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Biochimica%2520et%2520Biophysica%2520Sinica%26atitle%3DTargeting%252Bprotein%252Blysine%252Bmethylation%252Band%252Bdemethylation%252Bin%252Bcancers%26aulast%3DHe%26aufirst%3DYunlong%26date%3D2012%26date%3D2011%26volume%3D44%26issue%3D1%26spage%3D70%26epage%3D79" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Weihui  Zhong</span>, <span class="hlFld-ContribAuthor ">Danli  Wu</span>, <span class="hlFld-ContribAuthor ">Zhenhua  Li</span>. </span><span class="cited-content_cbyCitation_article-title">Facile and Efficient Cyclization of Anthranilonitrile to 2,4-Dichloroquinazoline by Bis(trichloromethyl) Carbonate and Catalytic Amount Triphenylphosphine Oxide. </span><span class="cited-content_cbyCitation_journal-name">HETEROCYCLES</span><span> <strong>2012,</strong> <em>85 </em>
                                    (6)
                                     , 1417. <a href="https://doi.org/10.3987/COM-12-12466" title="DOI URL">https://doi.org/10.3987/COM-12-12466</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3987/COM-12-12466&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3987%2FCOM-12-12466%26sid%3Dliteratum%253Aachs%26jtitle%3DHETEROCYCLES%26atitle%3DFacile%252Band%252BEfficient%252BCyclization%252Bof%252BAnthranilonitrile%252Bto%252B2%25252C4-Dichloroquinazoline%252Bby%252BBis%252528trichloromethyl%252529%252BCarbonate%252Band%252BCatalytic%252BAmount%252BTriphenylphosphine%252BOxide%26aulast%3DZhong%26aufirst%3DWeihui%26date%3D2012%26volume%3D85%26issue%3D6%26spage%3D1417" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-17/jm200903z/production/images/medium/jm-2011-00903z_0016.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-17/jm200903z/production/images/large/jm-2011-00903z_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm200903z&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-17/jm200903z/production/images/medium/jm-2011-00903z_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-17/jm200903z/production/images/large/jm-2011-00903z_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of known G9a/GLP inhibitors.<a onclick="showRef(event, 'ref29 ref31 ref32 ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref29 ref31 ref32 ref33 ref34">(29, 31-34)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-17/jm200903z/production/images/large/jm-2011-00903z_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm200903z&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-17/jm200903z/production/images/medium/jm-2011-00903z_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-17/jm200903z/production/images/large/jm-2011-00903z_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Compounds designed to achieve balanced in vitro potency and physicochemical properties aiding cell penetration to improve cellular potency.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-17/jm200903z/production/images/large/jm-2011-00903z_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm200903z&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-17/jm200903z/production/images/medium/jm-2011-00903z_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-17/jm200903z/production/images/large/jm-2011-00903z_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compounds <b>17</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-17/jm200903z/production/images/large/jm-2011-00903z_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm200903z&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>(a) 1-Chloro-3-iodopropane, K<sub>2</sub>CO<sub>3</sub>, CH<sub>3</sub>CN, reflux; (b) HNO<sub>3</sub>, Ac<sub>2</sub>O, 0 °C to rt, 75% over two steps; (c) pyrrolidine, K<sub>2</sub>CO<sub>3</sub>, NaI, cat. tetrabutylammonium iodide, CH<sub>3</sub>CN, reflux, 81%; (d) Fe dust, NH<sub>4</sub>OAc, AcOEt–H<sub>2</sub>O, reflux, 63%; (e) NaOCN, AcOH, H<sub>2</sub>O, rt; (f) NaOH, H<sub>2</sub>O, MeOH, reflux; (g) <i>N</i>,<i>N</i>-diethylaniline, POCl<sub>3</sub>, reflux, 50% over three steps; (h) 1-isopropylpiperidin-4-amine, DIEA, THF, rt, 94%; (i) various amines, CF<sub>3</sub>COOH, <i>i</i>-PrOH, 160 °C, microwave, 74–82%.</p></p></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-17/jm200903z/production/images/medium/jm-2011-00903z_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-17/jm200903z/production/images/large/jm-2011-00903z_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Analysis of cellular potency of G9a inhibitors as a function of their ALogP values reveals that cellular potency increases as lipophilicity increases for inhibitors that have similar in vitro potency. The inhibitors are color-coded with their G9a in vitro potency.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-17/jm200903z/production/images/large/jm-2011-00903z_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm200903z&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-17/jm200903z/production/images/medium/jm-2011-00903z_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-17/jm200903z/production/images/large/jm-2011-00903z_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>24</b> and <b>5</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-17/jm200903z/production/images/large/jm-2011-00903z_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm200903z&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>(a) BnBr, K<sub>2</sub>CO<sub>3</sub>, DMF, rt; (b) HNO<sub>3</sub>, Ac<sub>2</sub>O, 0 °C to rt; (c) Fe dust, NH<sub>4</sub>Cl, <i>i</i>-PrOH-H<sub>2</sub>O, reflux, 67% over 3 steps; d) cyclohexane carbonyl chloride, DIEA, DMF-DCM, 0 °C to rt; (e) NaOH, H<sub>2</sub>O<sub>2</sub>, EtOH, reflux, 48% over 2 steps; (f) <i>N</i>,<i>N</i>-diethylaniline, POCl<sub>3</sub>, reflux, 79%; (g) 1-isopropylpiperidin-4-amine, DIEA, <i>i</i>-PrOH, 160 °C, microwave, 94%; (h) Pd/C, 1,4-cyclohexadiene, EtOH, reflux; (i) various amino alcohols, PPh<sub>3</sub>, DIAD, THF, 0 °C to rt, 36–65% over 2 steps.</p></p></figure><figure data-id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-17/jm200903z/production/images/medium/jm-2011-00903z_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-17/jm200903z/production/images/large/jm-2011-00903z_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Compound <b>5</b> at 0.25 μM effectively inhibited the dimethylation of p53 K373 in H1299 cells. p53 expression vector was cotransfected with or without expression vectors for G9a or GLP into H1299 cells. Immediately after transfection, cells were treated with compound <b>5</b> or DMSO for 24 h. Cellular levels of p53 K373me2, total p53 and G9a or GLP were subject to Western blot analysis using antibodies.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-17/jm200903z/production/images/large/jm-2011-00903z_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm200903z&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i46">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_31157" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_31157" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 38 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Strahl, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span> </span><span class="NLM_article-title">The language of covalent histone modifications</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">403</span><span class="NLM_x">, </span> <span class="NLM_fpage">41</span><span class="NLM_x">–</span> <span class="NLM_lpage">45</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjm200903z&amp;key=10.1038%2F47412" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Fjm200903z&amp;key=10638745" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm200903z&amp;key=1%3ACAS%3A280%3ADC%252BD3c7gt1arsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=403&publication_year=2000&pages=41-45&author=B.+D.+Strahlauthor=C.+D.+Allis&title=The+language+of+covalent+histone+modifications"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The language of covalent histone modifications</span></div><div class="casAuthors">Strahl B D; Allis C D</div><div class="citationInfo"><span class="NLM_cas:title">Nature</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">403</span>
        (<span class="NLM_cas:issue">6765</span>),
    <span class="NLM_cas:pages">41-5</span>
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">Histone proteins and the nucleosomes they form with DNA are the fundamental building blocks of eukaryotic chromatin.  A diverse array of post-translational modifications that often occur on tail domains of these proteins has been well documented.  Although the function of these highly conserved modifications has remained elusive, converging biochemical and genetic evidence suggests functions in several chromatin-based processes.  We propose that distinct histone modifications, on one or more tails, act sequentially or in combination to form a 'histone code' that is, read by other proteins to bring about distinct downstream events.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSyzhdlYh9sX7t7lRqEFUJVfW6udTcc2ebkWxNIy6XqPrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3c7gt1arsQ%253D%253D&md5=2eb7d093151c6122ccd70fa2adcfd94b</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2F47412&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F47412%26sid%3Dliteratum%253Aachs%26aulast%3DStrahl%26aufirst%3DB.%2BD.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DThe%2520language%2520of%2520covalent%2520histone%2520modifications%26jtitle%3DNature%26date%3D2000%26volume%3D403%26spage%3D41%26epage%3D45" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Jenuwein, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span> </span><span class="NLM_article-title">Translating the histone code</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">293</span><span class="NLM_x">, </span> <span class="NLM_fpage">1074</span><span class="NLM_x">–</span> <span class="NLM_lpage">80</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Fjm200903z&amp;key=10.1126%2Fscience.1063127" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Fjm200903z&amp;key=11498575" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Fjm200903z&amp;key=1%3ACAS%3A528%3ADC%252BD3MXmtVWltro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=293&publication_year=2001&pages=1074-80&author=T.+Jenuweinauthor=C.+D.+Allis&title=Translating+the+histone+code"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Translating the histone code</span></div><div class="casAuthors">Jenuwein, Thomas; Allis, C. David</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">293</span>
        (<span class="NLM_cas:issue">5532</span>),
    <span class="NLM_cas:pages">1074-1080</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The review.  Chromatin, the physiol. template of all eukaryotic genetic information, is subject to a diverse array of posttranslational modifications that largely impinge on histone amino termini, thereby regulating access to the underlying DNA.  Distinct histone amino-terminal modifications can generate synergistic or antagonistic interaction affinities for chromatin-assocd. proteins, which in turn dictate dynamic transitions between transcriptionally active or transcriptionally silent chromatin states.  The combinatorial nature of histone amino-terminal modifications thus reveals a "histone code" that considerably extends the information potential of the genetic code.  We propose that this epigenetic marking system represents a fundamental regulatory mechanism that has an impact on most, if not all; chromatin-templated processes, with far-reaching consequences for cell fate decisions and both normal and pathol. development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUFyHNepKcUbVg90H21EOLACvtfcHk0lgx_noLTsNOHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXmtVWltro%253D&md5=021dcd98a5e5bf2ed7eedef1897b2269</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1126%2Fscience.1063127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1063127%26sid%3Dliteratum%253Aachs%26aulast%3DJenuwein%26aufirst%3DT.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DTranslating%2520the%2520histone%2520code%26jtitle%3DScience%26date%3D2001%26volume%3D293%26spage%3D1074%26epage%3D80" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Kouzarides, T.</span><span> </span><span class="NLM_article-title">Chromatin modifications and their function</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">128</span><span class="NLM_x">, </span> <span class="NLM_fpage">693</span><span class="NLM_x">–</span> <span class="NLM_lpage">705</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Fjm200903z&amp;key=10.1016%2Fj.cell.2007.02.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Fjm200903z&amp;key=17320507" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Fjm200903z&amp;key=1%3ACAS%3A528%3ADC%252BD2sXis12ju7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2007&pages=693-705&author=T.+Kouzarides&title=Chromatin+modifications+and+their+function"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatin modifications and their function</span></div><div class="casAuthors">Kouzarides, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">693-705</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The surface of nucleosomes is studded with a multiplicity of modifications.  At least eight different classes have been characterized to date and many different sites have been identified for each class.  Operationally, modifications function either by disrupting chromatin contacts or by affecting the recruitment of nonhistone proteins to chromatin.  Their presence on histones can dictate the higher-order chromatin structure in which DNA is packaged and can orchestrate the ordered recruitment of enzyme complexes to manipulate DNA.  In this way, histone modifications have the potential to influence many fundamental biol. processes, some of which may be epigenetically inherited.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAH9AX2dGicbVg90H21EOLACvtfcHk0lg30A_3J8Sh8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXis12ju7Y%253D&md5=c5995fe68fd9473f4268d59c01d3b9de</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.02.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.02.005%26sid%3Dliteratum%253Aachs%26aulast%3DKouzarides%26aufirst%3DT.%26atitle%3DChromatin%2520modifications%2520and%2520their%2520function%26jtitle%3DCell%26date%3D2007%26volume%3D128%26spage%3D693%26epage%3D705" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Solomon, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richon, V. M.</span><span> </span><span class="NLM_article-title">Protein methyltransferases as a target class for drug discovery</span> <span class="citation_source-journal">Nature Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">724</span><span class="NLM_x">–</span> <span class="NLM_lpage">732</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=724-732&author=R.+A.+Copelandauthor=M.+E.+Solomonauthor=V.+M.+Richon&title=Protein+methyltransferases+as+a+target+class+for+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DSolomon%26aufirst%3DM.%2BE.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26atitle%3DProtein%2520methyltransferases%2520as%2520a%2520target%2520class%2520for%2520drug%2520discovery%26jtitle%3DNature%2520Rev.%2520Drug%2520Discovery%26date%3D2009%26volume%3D8%26spage%3D724%26epage%3D732" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Fog, C. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jensen, K. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lund, A. H.</span><span> </span><span class="NLM_article-title">Chromatin-modifying proteins in cancer</span> <span class="citation_source-journal">Acta Pathol. Microbiol. Immunol. Scand.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">115</span><span class="NLM_x">, </span> <span class="NLM_fpage">1060</span><span class="NLM_x">–</span> <span class="NLM_lpage">1089</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Fjm200903z&amp;key=10.1111%2Fj.1600-0463.2007.apm_776.xml.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Fjm200903z&amp;key=18042144" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Fjm200903z&amp;key=1%3ACAS%3A528%3ADC%252BD1cXkslOnsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2007&pages=1060-1089&author=C.+K.+Fogauthor=K.+T.+Jensenauthor=A.+H.+Lund&title=Chromatin-modifying+proteins+in+cancer"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatin-modifying proteins in cancer</span></div><div class="casAuthors">Fog, Cathrine K.; Jensen, Klaus T.; Lund, Anders H.</div><div class="citationInfo"><span class="NLM_cas:title">APMIS</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">115</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1060-1089</span>CODEN:
                <span class="NLM_cas:coden">APMSEL</span>;
        ISSN:<span class="NLM_cas:issn">0903-4641</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">A review.  Chromatin-modifying proteins mold the genome into areas that are accessible for transcriptional activity and areas that are transcriptionally silent.  This epigenetic gene regulation allows for different transcriptional programs to be conducted in different cell types at different timepoints-despite the fact that all cells in the organism contain the same genetic information.  A large amt. of data gathered over the last decades has demonstrated that deregulation of chromatin-modifying proteins is etiol. involved in the development and progression of cancer.  Here we discuss how epigenetic alterations influence cancer development and review known cancer-assocd. alterations in chromatin-modifying proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxhso-Ej5KY7Vg90H21EOLACvtfcHk0liVZ-KGBXeyaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXkslOnsQ%253D%253D&md5=060c4f35c69924a72a16b316b773a215</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1111%2Fj.1600-0463.2007.apm_776.xml.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1600-0463.2007.apm_776.xml.x%26sid%3Dliteratum%253Aachs%26aulast%3DFog%26aufirst%3DC.%2BK.%26aulast%3DJensen%26aufirst%3DK.%2BT.%26aulast%3DLund%26aufirst%3DA.%2BH.%26atitle%3DChromatin-modifying%2520proteins%2520in%2520cancer%26jtitle%3DActa%2520Pathol.%2520Microbiol.%2520Immunol.%2520Scand.%26date%3D2007%26volume%3D115%26spage%3D1060%26epage%3D1089" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Rea, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eisenhaber, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Carroll, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strahl, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, Z. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmid, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Opravil, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mechtler, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ponting, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenuwein, T.</span><span> </span><span class="NLM_article-title">Regulation of chromatin structure by site-specific histone H3 methyltransferases</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">406</span><span class="NLM_x">, </span> <span class="NLM_fpage">593</span><span class="NLM_x">–</span> <span class="NLM_lpage">599</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Fjm200903z&amp;key=10.1038%2F35020506" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Fjm200903z&amp;key=10949293" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Fjm200903z&amp;key=1%3ACAS%3A528%3ADC%252BD3cXmtFegs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=406&publication_year=2000&pages=593-599&author=S.+Reaauthor=F.+Eisenhaberauthor=D.+O%E2%80%99Carrollauthor=B.+D.+Strahlauthor=Z.+W.+Sunauthor=M.+Schmidauthor=S.+Opravilauthor=K.+Mechtlerauthor=C.+P.+Pontingauthor=C.+D.+Allisauthor=T.+Jenuwein&title=Regulation+of+chromatin+structure+by+site-specific+histone+H3+methyltransferases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of chromatin structure by site-specific histone H3 methyltransferases</span></div><div class="casAuthors">Rea, Stephen; Elsenhaber, Frank; O'Carroll, Donal; Strahl, Brian D.; Sun, Zu-Wen; Schmid, Manfred; Opravil, Susanne; Mechtler, Karl; Ponting, Chris P.; Allis, C. David; Jenuwein, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">406</span>
        (<span class="NLM_cas:issue">6796</span>),
    <span class="NLM_cas:pages">593-599</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The organization of chromatin into higher-order structures influences chromosome function and epigenetic gene regulation.  Higher-order chromatin has been proposed to be nucleated by the covalent modification of histone tails and the subsequent establishment of chromosomal subdomains by non-histone modifier factors.  Here we show that human SUV39H1 and murine Suv39h1 - mammalian homologs of Drosophila Su(var)3-9 and of Schizosaccharomyces pombe clr4 - encode histone H3-specific methyltransferases that selectively methylate lysine 9 of the amino terminus of histone H3 in vitro.  We mapped the catalytic motif to the evolutionarily conserved SET domain, which requires adjacent cysteine-rich regions to confer histone methyltransferase activity.  Methylation of lysine 9 interferes with phosphorylation of serine 10, but is also influenced by pre-existing modifications in the amino terminus of H3.  In vivo, deregulated SUV39H1 or disrupted Suv39h activity modulate H3 serine 10 phosphorylation in native chromatin and induce aberrant mitotic divisions.  Our data reveal a functional interdependence of site-specific H3 tail modifications and suggest a dynamic mechanism for the regulation of higher-order chromatin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2yioyTahy77Vg90H21EOLACvtfcHk0lgQb8JZvkDF5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXmtFegs7s%253D&md5=65e14733d25f97fadaaeed602600bd23</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2F35020506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F35020506%26sid%3Dliteratum%253Aachs%26aulast%3DRea%26aufirst%3DS.%26aulast%3DEisenhaber%26aufirst%3DF.%26aulast%3DO%25E2%2580%2599Carroll%26aufirst%3DD.%26aulast%3DStrahl%26aufirst%3DB.%2BD.%26aulast%3DSun%26aufirst%3DZ.%2BW.%26aulast%3DSchmid%26aufirst%3DM.%26aulast%3DOpravil%26aufirst%3DS.%26aulast%3DMechtler%26aufirst%3DK.%26aulast%3DPonting%26aufirst%3DC.%2BP.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26aulast%3DJenuwein%26aufirst%3DT.%26atitle%3DRegulation%2520of%2520chromatin%2520structure%2520by%2520site-specific%2520histone%2520H3%2520methyltransferases%26jtitle%3DNature%26date%3D2000%26volume%3D406%26spage%3D593%26epage%3D599" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Ruthenburg, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span> </span><span class="NLM_article-title">Multivalent engagement of chromatin modifications by linked binding modules</span> <span class="citation_source-journal">Nature Rev. Mol. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">983</span><span class="NLM_x">–</span> <span class="NLM_lpage">994</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Fjm200903z&amp;key=10.1038%2Fnrm2298" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Fjm200903z&amp;key=18037899" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Fjm200903z&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlCrsb3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2007&pages=983-994&author=A.+J.+Ruthenburgauthor=H.+Liauthor=D.+J.+Patelauthor=C.+D.+Allis&title=Multivalent+engagement+of+chromatin+modifications+by+linked+binding+modules"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Multivalent engagement of chromatin modifications by linked binding modules</span></div><div class="casAuthors">Ruthenburg, Alexander J.; Li, Haitao; Patel, Dinshaw J.; Allis, C. David</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">983-994</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Histone post-translational modifications (PTMs) play crucial roles in genome management, in part by recruiting specific factors that alter the structural properties of chromatin.  These so-called effector complexes often comprise multiple histone-binding modules that may act in concert to regulate chromatin structure and DNA-related activities.  Various chem. modifications on histones and regions of assocd. DNA play crucial roles in genome management by binding specific factors that, in turn, serve to alter the structural properties of chromatin.  These so-called effector proteins have typically been studied with the biochemist's paring knife; the capacity to recognize specific chromatin PTMs has been mapped to an increasing no. of domains that frequently appear in the nuclear subset of the proteome, often present in large, multisubunit complexes that bristle with modification-dependent binding potential.  The authors propose that multivalent interactions on a single histone tail and beyond may have a significant, if not dominant, role in chromatin transactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkIrlnyvt64LVg90H21EOLACvtfcHk0liXD4_zA6ENiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlCrsb3L&md5=0d27fc9ba2e2a009e0a9903e1c65bdf7</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnrm2298&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm2298%26sid%3Dliteratum%253Aachs%26aulast%3DRuthenburg%26aufirst%3DA.%2BJ.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DPatel%26aufirst%3DD.%2BJ.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DMultivalent%2520engagement%2520of%2520chromatin%2520modifications%2520by%2520linked%2520binding%2520modules%26jtitle%3DNature%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2007%26volume%3D8%26spage%3D983%26epage%3D994" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Wang, G. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dormann, H. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casadio, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span> </span><span class="NLM_article-title">Haematopoietic malignancies caused by dysregulation of a chromatin-binding PHD finger</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">459</span><span class="NLM_x">, </span> <span class="NLM_fpage">847</span><span class="NLM_x">–</span> <span class="NLM_lpage">851</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Fjm200903z&amp;key=10.1038%2Fnature08036" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Fjm200903z&amp;key=19430464" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Fjm200903z&amp;key=1%3ACAS%3A528%3ADC%252BD1MXls1CmtL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=459&publication_year=2009&pages=847-851&author=G.+G.+Wangauthor=J.+Songauthor=Z.+Wangauthor=H.+L.+Dormannauthor=F.+Casadioauthor=H.+Liauthor=J.+L.+Luoauthor=D.+J.+Patelauthor=C.+D.+Allis&title=Haematopoietic+malignancies+caused+by+dysregulation+of+a+chromatin-binding+PHD+finger"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Haematopoietic malignancies caused by dysregulation of a chromatin-binding PHD finger</span></div><div class="casAuthors">Wang, Gang G.; Song, Jikui; Wang, Zhanxin; Dormann, Holger L.; Casadio, Fabio; Li, Haitao; Luo, Jun-Li; Patel, Dinshaw J.; Allis, C. David</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">459</span>
        (<span class="NLM_cas:issue">7248</span>),
    <span class="NLM_cas:pages">847-851</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Histone H3 lysine 4 methylation (H3K4me) has been proposed as a crit. component in regulating gene expression, epigenetic states, and cellular identities.  The biol. meaning of H3K4me is interpreted by conserved modules including plant homeodomain (PHD) fingers that recognize varied H3K4me states.  The dysregulation of PHD fingers has been implicated in several human diseases, including cancers and immune or neurol. disorders.  Here we report that fusing an H3K4-trimethylation (H3K4me3)-binding PHD finger, such as the carboxy-terminal PHD finger of PHF23 or JARID1A (also known as KDM5A or RBBP2), to a common fusion partner nucleoporin-98 (NUP98) as identified in human leukemias, generated potent oncoproteins that arrested hematopoietic differentiation and induced acute myeloid leukemia in murine models.  In these processes, a PHD finger that specifically recognizes H3K4me3/2 marks was essential for leukemogenesis.  Mutations in PHD fingers that abrogated H3K4me3 binding also abolished leukemic transformation.  NUP98-PHD fusion prevented the differentiation-assocd. removal of H3K4me3 at many loci encoding lineage-specific transcription factors (Hox(s), Gata3, Meis1, Eya1 and Pbx1), and enforced their active gene transcription in murine hematopoietic stem/progenitor cells.  Mechanistically, NUP98-PHD fusions act as 'chromatin boundary factors', dominating over polycomb-mediated gene silencing to 'lock' developmentally crit. loci into an active chromatin state (H3K4me3 with induced histone acetylation), a state that defined leukemia stem cells.  Collectively, our studies represent, to our knowledge, the first report that deregulation of the PHD finger, an 'effector' of specific histone modification, perturbs the epigenetic dynamics on developmentally crit. loci, catastrophizes cellular fate decision-making, and even causes oncogenesis during mammalian development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpR1S9VoRYMRrVg90H21EOLACvtfcHk0lhhXoccYAw_Rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXls1CmtL0%253D&md5=3a647643c8a73639fcb332a774f1155e</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnature08036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08036%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DG.%2BG.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DDormann%26aufirst%3DH.%2BL.%26aulast%3DCasadio%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DLuo%26aufirst%3DJ.%2BL.%26aulast%3DPatel%26aufirst%3DD.%2BJ.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DHaematopoietic%2520malignancies%2520caused%2520by%2520dysregulation%2520of%2520a%2520chromatin-binding%2520PHD%2520finger%26jtitle%3DNature%26date%3D2009%26volume%3D459%26spage%3D847%26epage%3D851" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Fritsch, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robin, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mathieu, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Souidi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hinaux, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rougeulle, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harel-Bellan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ameyar-Zazoua, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ait-Si-Ali, S.</span><span> </span><span class="NLM_article-title">A subset of the histone H3 lysine 9 methyltransferases Suv39h1, G9a, GLP, and SETDB1 participate in a multimeric complex</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">37</span><span class="NLM_x">, </span> <span class="NLM_fpage">46</span><span class="NLM_x">–</span> <span class="NLM_lpage">56</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2010&pages=46-56&author=L.+Fritschauthor=P.+Robinauthor=J.+R.+Mathieuauthor=M.+Souidiauthor=H.+Hinauxauthor=C.+Rougeulleauthor=A.+Harel-Bellanauthor=M.+Ameyar-Zazouaauthor=S.+Ait-Si-Ali&title=A+subset+of+the+histone+H3+lysine+9+methyltransferases+Suv39h1%2C+G9a%2C+GLP%2C+and+SETDB1+participate+in+a+multimeric+complex"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFritsch%26aufirst%3DL.%26aulast%3DRobin%26aufirst%3DP.%26aulast%3DMathieu%26aufirst%3DJ.%2BR.%26aulast%3DSouidi%26aufirst%3DM.%26aulast%3DHinaux%26aufirst%3DH.%26aulast%3DRougeulle%26aufirst%3DC.%26aulast%3DHarel-Bellan%26aufirst%3DA.%26aulast%3DAmeyar-Zazoua%26aufirst%3DM.%26aulast%3DAit-Si-Ali%26aufirst%3DS.%26atitle%3DA%2520subset%2520of%2520the%2520histone%2520H3%2520lysine%25209%2520methyltransferases%2520Suv39h1%252C%2520G9a%252C%2520GLP%252C%2520and%2520SETDB1%2520participate%2520in%2520a%2520multimeric%2520complex%26jtitle%3DMol.%2520Cell%26date%3D2010%26volume%3D37%26spage%3D46%26epage%3D56" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Frye, S. V.</span><span> </span><span class="NLM_article-title">The art of the chemical probe</span> <span class="citation_source-journal">Nature Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">159</span><span class="NLM_x">–</span> <span class="NLM_lpage">161</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Fjm200903z&amp;key=10.1038%2Fnchembio.296" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Fjm200903z&amp;key=20154659" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Fjm200903z&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhvFGntbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=159-161&author=S.+V.+Frye&title=The+art+of+the+chemical+probe"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">The art of the chemical probe</span></div><div class="casAuthors">Frye, Stephen V.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">159-161</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Chem. biologists frequently aim to create small-mol. probes that interact with a specific protein in vitro in order to explore the role of the protein in a broader biol. context (cells or organisms), but a common understanding of what makes a high-quality probe is lacking.  Here I propose a set of principles to guide probe qualification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7vDfsTLhqWrVg90H21EOLACvtfcHk0lh47HnZPI5DnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhvFGntbg%253D&md5=b7b48c591a611b8686e11b3fc4f1b85c</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.296&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.296%26sid%3Dliteratum%253Aachs%26aulast%3DFrye%26aufirst%3DS.%2BV.%26atitle%3DThe%2520art%2520of%2520the%2520chemical%2520probe%26jtitle%3DNature%2520Chem.%2520Biol.%26date%3D2010%26volume%3D6%26spage%3D159%26epage%3D161" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Cole, P. A.</span><span> </span><span class="NLM_article-title">Chemical probes for histone-modifying enzymes</span> <span class="citation_source-journal">Nature Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">590</span><span class="NLM_x">–</span> <span class="NLM_lpage">597</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Fjm200903z&amp;key=10.1038%2Fnchembio.111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Fjm200903z&amp;key=18800048" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Fjm200903z&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtFamtbrL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2008&pages=590-597&author=P.+A.+Cole&title=Chemical+probes+for+histone-modifying+enzymes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical probes for histone-modifying enzymes</span></div><div class="casAuthors">Cole, Philip A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">590-597</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The histone-modifying enzymes that catalyze reversible lysine acetylation and methylation are central to the epigenetic regulation of chromatin remodeling.  From the early discovery of histone deacetylase inhibitors to the more recent identification of histone demethylase blockers, chem. approaches offer increasingly sophisticated tools for the investigation of the structure and function of these lysine-modifying enzymes.  This review summarizes progress to date on compds. identified from screens or by design that can modulate the activity of classical histone deacetylases, sirtuins, histone acetyltransferases, histone methyltransferases and histone demethylases.  We highlight applications of compds. to mechanistic and functional studies involving these enzymes and discuss future challenges regarding target specificity and general utility.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJZ5RKIrQsO7Vg90H21EOLACvtfcHk0lgLlmQHCpW2IA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtFamtbrL&md5=71fe323c232af42b67a7dfc134c4d49b</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.111%26sid%3Dliteratum%253Aachs%26aulast%3DCole%26aufirst%3DP.%2BA.%26atitle%3DChemical%2520probes%2520for%2520histone-modifying%2520enzymes%26jtitle%3DNature%2520Chem.%2520Biol.%26date%3D2008%26volume%3D4%26spage%3D590%26epage%3D597" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heightman, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Targeting Methyl Lysine</span> <span class="citation_source-journal">Annu. Rep. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">329</span><span class="NLM_x">–</span> <span class="NLM_lpage">343</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2010&pages=329-343&author=S.+V.+Fryeauthor=T.+Heightmanauthor=J.+Jin&title=Targeting+Methyl+Lysine"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DHeightman%26aufirst%3DT.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DTargeting%2520Methyl%2520Lysine%26jtitle%3DAnnu.%2520Rep.%2520Med.%2520Chem.%26date%3D2010%26volume%3D45%26spage%3D329%26epage%3D343" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Yost, J. M.; Korboukh, I.; Liu, F.; Gao, C.; Jin, J.</span><span> </span><span class="NLM_article-title">Targets in Epigenetics: Inhibiting the Methyl Writers of the Histone Code</span>. Curr. Chem. Genomics<span class="NLM_x"> </span><span class="NLM_year">2011</span><span class="NLM_x">, </span>in press. </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Fjm200903z&amp;key=21966347" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&author=J.+M.+Yost&author=I.+Korboukh&author=F.+Liu&author=C.+Gao&author=J.+Jin&title=Targets+in+Epigenetics%3A+Inhibiting+the+Methyl+Writers+of+the+Histone+Code"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DYost%26aufirst%3DJ.%2BM.%26atitle%3DTargets%2520in%2520Epigenetics%253A%2520Inhibiting%2520the%2520Methyl%2520Writers%2520of%2520the%2520Histone%2520Code%26jtitle%3DCurr.%2520Chem.%2520Genomics%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Tachibana, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sugimoto, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nozaki, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ueda, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohta, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohki, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fukuda, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takeda, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niida, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kato, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shinkai, Y.</span><span> </span><span class="NLM_article-title">G9a histone methyltransferase plays a dominant role in euchromatic histone H3 lysine 9 methylation and is essential for early embryogenesis</span> <span class="citation_source-journal">Genes Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">1779</span><span class="NLM_x">–</span> <span class="NLM_lpage">1791</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2002&pages=1779-1791&author=M.+Tachibanaauthor=K.+Sugimotoauthor=M.+Nozakiauthor=J.+Uedaauthor=T.+Ohtaauthor=M.+Ohkiauthor=M.+Fukudaauthor=N.+Takedaauthor=H.+Niidaauthor=H.+Katoauthor=Y.+Shinkai&title=G9a+histone+methyltransferase+plays+a+dominant+role+in+euchromatic+histone+H3+lysine+9+methylation+and+is+essential+for+early+embryogenesis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTachibana%26aufirst%3DM.%26aulast%3DSugimoto%26aufirst%3DK.%26aulast%3DNozaki%26aufirst%3DM.%26aulast%3DUeda%26aufirst%3DJ.%26aulast%3DOhta%26aufirst%3DT.%26aulast%3DOhki%26aufirst%3DM.%26aulast%3DFukuda%26aufirst%3DM.%26aulast%3DTakeda%26aufirst%3DN.%26aulast%3DNiida%26aufirst%3DH.%26aulast%3DKato%26aufirst%3DH.%26aulast%3DShinkai%26aufirst%3DY.%26atitle%3DG9a%2520histone%2520methyltransferase%2520plays%2520a%2520dominant%2520role%2520in%2520euchromatic%2520histone%2520H3%2520lysine%25209%2520methylation%2520and%2520is%2520essential%2520for%2520early%2520embryogenesis%26jtitle%3DGenes%2520Dev.%26date%3D2002%26volume%3D16%26spage%3D1779%26epage%3D1791" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Trojer, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yonezawa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmidt, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenuwein, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinberg, D.</span><span> </span><span class="NLM_article-title">Dynamic Histone H1 Isotype 4 Methylation and Demethylation by Histone Lysine Methyltransferase G9a/KMT1C and the Jumonji Domain-Containing JMJD2/KDM4 Proteins</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">284</span><span class="NLM_x">, </span> <span class="NLM_fpage">8395</span><span class="NLM_x">–</span> <span class="NLM_lpage">8405</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=8395-8405&author=P.+Trojerauthor=J.+Zhangauthor=M.+Yonezawaauthor=A.+Schmidtauthor=H.+Zhengauthor=T.+Jenuweinauthor=D.+Reinberg&title=Dynamic+Histone+H1+Isotype+4+Methylation+and+Demethylation+by+Histone+Lysine+Methyltransferase+G9a%2FKMT1C+and+the+Jumonji+Domain-Containing+JMJD2%2FKDM4+Proteins"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTrojer%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DYonezawa%26aufirst%3DM.%26aulast%3DSchmidt%26aufirst%3DA.%26aulast%3DZheng%26aufirst%3DH.%26aulast%3DJenuwein%26aufirst%3DT.%26aulast%3DReinberg%26aufirst%3DD.%26atitle%3DDynamic%2520Histone%2520H1%2520Isotype%25204%2520Methylation%2520and%2520Demethylation%2520by%2520Histone%2520Lysine%2520Methyltransferase%2520G9a%252FKMT1C%2520and%2520the%2520Jumonji%2520Domain-Containing%2520JMJD2%252FKDM4%2520Proteins%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2009%26volume%3D284%26spage%3D8395%26epage%3D8405" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Sampath, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marazzi, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yap, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krutchinsky, A. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mecklenbrauker, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Viale, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rudensky, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chait, B. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tarakhovsky, A.</span><span> </span><span class="NLM_article-title">Methylation of a histone mimic within the histone methyltransferase G9a regulates protein complex assembly</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">596</span><span class="NLM_x">–</span> <span class="NLM_lpage">608</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Fjm200903z&amp;key=10.1016%2Fj.molcel.2007.06.026" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Fjm200903z&amp;key=17707231" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Fjm200903z&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVSmtL3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2007&pages=596-608&author=S.+C.+Sampathauthor=I.+Marazziauthor=K.+L.+Yapauthor=A.+N.+Krutchinskyauthor=I.+Mecklenbraukerauthor=A.+Vialeauthor=E.+Rudenskyauthor=M.+M.+Zhouauthor=B.+T.+Chaitauthor=A.+Tarakhovsky&title=Methylation+of+a+histone+mimic+within+the+histone+methyltransferase+G9a+regulates+protein+complex+assembly"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Methylation of a histone mimic within the histone methyltransferase G9a regulates protein complex assembly</span></div><div class="casAuthors">Sampath, Srihari C.; Marazzi, Ivan; Yap, Kyoko L.; Sampath, Srinath C.; Krutchinsky, Andrew N.; Mecklenbrauker, Ingrid; Viale, Agnes; Rudensky, Eugene; Zhou, Ming-Ming; Chait, Brian T.; Tarakhovsky, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">596-608</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Epigenetic gene silencing in eukaryotes is regulated in part by lysine methylation of the core histone proteins.  While histone lysine methylation is known to control gene expression through the recruitment of modification-specific effector proteins, it remains unknown whether nonhistone chromatin proteins are targets for similar modification-recognition systems.  Here we show that the histone H3 methyltransferase G9a contains a conserved methylation motif with marked sequence similarity to H3 itself.  As with methylation of H3 lysine 9, autocatalytic G9a methylation is necessary and sufficient to mediate in vivo interaction with the epigenetic regulator heterochromatin protein 1 (HP1), and this methyl-dependent interaction can be reversed by adjacent G9a phosphorylation.  NMR anal. indicates that the HP1 chromodomain recognizes methyl-G9a through a binding mode similar to that used in recognition of methyl-H3K9, demonstrating that the chromodomain functions as a generalized methyl-lysine binding module.  These data reveal histone-like modification cassettes-or "histone mimics"-as a distinct class of nonhistone methylation targets and directly extend the principles of the histone code to the regulation of nonhistone proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFIcB4m0CyVbVg90H21EOLACvtfcHk0liMKckxOhNhVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVSmtL3J&md5=3002c1fd27ba6061da0cef626680b504</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2007.06.026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2007.06.026%26sid%3Dliteratum%253Aachs%26aulast%3DSampath%26aufirst%3DS.%2BC.%26aulast%3DMarazzi%26aufirst%3DI.%26aulast%3DYap%26aufirst%3DK.%2BL.%26aulast%3DKrutchinsky%26aufirst%3DA.%2BN.%26aulast%3DMecklenbrauker%26aufirst%3DI.%26aulast%3DViale%26aufirst%3DA.%26aulast%3DRudensky%26aufirst%3DE.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26aulast%3DChait%26aufirst%3DB.%2BT.%26aulast%3DTarakhovsky%26aufirst%3DA.%26atitle%3DMethylation%2520of%2520a%2520histone%2520mimic%2520within%2520the%2520histone%2520methyltransferase%2520G9a%2520regulates%2520protein%2520complex%2520assembly%26jtitle%3DMol.%2520Cell%26date%3D2007%26volume%3D27%26spage%3D596%26epage%3D608" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Huang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dorsey, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chuikov, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenuwein, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinberg, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berger, S. L.</span><span> </span><span class="NLM_article-title">G9A and GLP methylate lysine 373 in the tumor suppressor p53</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">285</span><span class="NLM_x">, </span> <span class="NLM_fpage">9636</span><span class="NLM_x">–</span> <span class="NLM_lpage">9641</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Fjm200903z&amp;key=10.1074%2Fjbc.M109.062588" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Fjm200903z&amp;key=20118233" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Fjm200903z&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjsFeht7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=9636-9641&author=J.+Huangauthor=J.+Dorseyauthor=S.+Chuikovauthor=X.+Zhangauthor=T.+Jenuweinauthor=D.+Reinbergauthor=S.+L.+Berger&title=G9A+and+GLP+methylate+lysine+373+in+the+tumor+suppressor+p53"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">G9a and Glp Methylate Lysine 373 in the Tumor Suppressor p53</span></div><div class="casAuthors">Huang, Jing; Dorsey, Jean; Chuikov, Sergei; Zhang, Xinyue; Jenuwein, Thomas; Reinberg, Danny; Berger, Shelley L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">285</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">9636-9641</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The tumor suppressor p53 is regulated by numerous post-translational modifications.  Lysine methylation has recently emerged as a key post-translational modification that alters the activity of p53.  Here, we describe a novel lysine methylation site in p53 that is carried out by two homologous histone methyltransferases, G9a and Glp.  G9a and Glp specifically methylate p53 at Lys373, resulting mainly in dimethylation.  During DNA damage, the overall level of p53 modified at Lys373me2 does not increase, despite the dramatic increase in total p53, indicating that Lys373me2 correlates with inactive p53.  Further, redn. of G9a and/or Glp levels leads to a larger population of apoptotic cells.  Examn. of the Oncomine data base shows that G9a and Glp are overexpressed in various cancers compared with corresponding normal tissues, suggesting that they are putative oncogenes.  These data reveal a new methylation site within p53 mediated by the methylases G9a and Glp and indicate that G9a is a potential inhibitory target for cancer treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozJXSsZlEE9LVg90H21EOLACvtfcHk0lhD9R7M32yKuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjsFeht7w%253D&md5=81adc2281f6344caf9a46ebcdf1c61e6</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.062588&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.062588%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DDorsey%26aufirst%3DJ.%26aulast%3DChuikov%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DJenuwein%26aufirst%3DT.%26aulast%3DReinberg%26aufirst%3DD.%26aulast%3DBerger%26aufirst%3DS.%2BL.%26atitle%3DG9A%2520and%2520GLP%2520methylate%2520lysine%2520373%2520in%2520the%2520tumor%2520suppressor%2520p53%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D9636%26epage%3D9641" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Kondo, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ahmed, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boumber, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sekido, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haddad, B. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Issa, J. P.</span><span> </span><span class="NLM_article-title">Downregulation of histone H3 lysine 9 methyltransferase G9a induces centrosome disruption and chromosome instability in cancer cells</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">e2037</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Fjm200903z&amp;key=10.1371%2Fjournal.pone.0002037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Fjm200903z&amp;key=18446223" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Fjm200903z&amp;key=1%3ACAS%3A280%3ADC%252BD1c3otlyhtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2008&pages=e2037&author=Y.+Kondoauthor=L.+Shenauthor=S.+Ahmedauthor=Y.+Boumberauthor=Y.+Sekidoauthor=B.+R.+Haddadauthor=J.+P.+Issa&title=Downregulation+of+histone+H3+lysine+9+methyltransferase+G9a+induces+centrosome+disruption+and+chromosome+instability+in+cancer+cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Downregulation of histone H3 lysine 9 methyltransferase G9a induces centrosome disruption and chromosome instability in cancer cells</span></div><div class="casAuthors">Kondo Yutaka; Shen Lanlan; Ahmed Saira; Boumber Yanis; Sekido Yoshitaka; Haddad Bassem R; Issa Jean-Pierre J</div><div class="citationInfo"><span class="NLM_cas:title">PloS one</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">e2037</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Modifications of the histone amino-terminal tails affect access of regulatory factors and complexes to chromatin and thereby influence biological processes.  Cancer cells are characterized by prominent epigenetic dysregulation, including histone modifications.  However, the functional roles of the histone methyltransferases (HMT) in cancer remain unclear.  METHODOLOGY/PRINCIPAL FINDINGS:  We studied RNAi-based inhibition (knockdown, KD) of 2 different H3K9 HMTs, SUV39H1 and G9a.  Knockdown of the 2 HMTs in PC3 cancer cell line markedly inhibited cell growth and caused profound morphological changes with loss of telomerase activity and shortened telomeres.  SUV39H1 KD cells showed substantial increase in G2/M fraction.  G9a KD cells showed increased DNA content (1.7-fold in 2 independent clones) compared with FACS analyses to control.  Karyotype analyses showed that this was due to an increased number of chromosomes (from 61 to 102) in G9a KD cells compared to parental PC3.  Intriguingly, we found abnormal centrosome morphology and number in about 25% of the G9a KD cells, while centrosomes were morphologically normal in control cells.  Microarray analyses after KD of SUV39H1 or G9a showed very few genes up-regulated among the 39,000 genes.  The silenced tumor-suppressor genes p16 and RASSF1A were not activated in KD cells.  CONCLUSIONS/SIGNIFICANCE:  These data suggest that the 2 HMTs, SUV39H1 and G9a are required to perpetuate the malignant phenotype.  Furthermore, G9a plays a critical role in regulating centrosome duplication presumably through chromatin structure rather than through affecting gene expression in cancer cells.  Targeting these histone methyltransferases may be of therapeutic benefit in cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQhnx1F8nyNwDEwO064Z95rfW6udTcc2ear75-0WLqG2Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1c3otlyhtw%253D%253D&md5=22936eeeffa4dc29e7254dfc546ebcfd</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0002037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0002037%26sid%3Dliteratum%253Aachs%26aulast%3DKondo%26aufirst%3DY.%26aulast%3DShen%26aufirst%3DL.%26aulast%3DAhmed%26aufirst%3DS.%26aulast%3DBoumber%26aufirst%3DY.%26aulast%3DSekido%26aufirst%3DY.%26aulast%3DHaddad%26aufirst%3DB.%2BR.%26aulast%3DIssa%26aufirst%3DJ.%2BP.%26atitle%3DDownregulation%2520of%2520histone%2520H3%2520lysine%25209%2520methyltransferase%2520G9a%2520induces%2520centrosome%2520disruption%2520and%2520chromosome%2520instability%2520in%2520cancer%2520cells%26jtitle%3DPLoS%2520One%26date%3D2008%26volume%3D3%26spage%3De2037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Kondo, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurokawa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Masuko, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanaka, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kato, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mizuno, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yokoe, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sugauchi, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirashima, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orito, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Osada, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ueda, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Issa, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sekido, Y.</span><span> </span><span class="NLM_article-title">Alterations of DNA methylation and histone modifications contribute to gene silencing in hepatocellular carcinomas</span> <span class="citation_source-journal">Hepatol. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">37</span><span class="NLM_x">, </span> <span class="NLM_fpage">974</span><span class="NLM_x">–</span> <span class="NLM_lpage">983</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Fjm200903z&amp;key=10.1111%2Fj.1872-034X.2007.00141.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Fjm200903z&amp;key=17584191" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Fjm200903z&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlCns7fK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2007&pages=974-983&author=Y.+Kondoauthor=L.+Shenauthor=S.+Suzukiauthor=T.+Kurokawaauthor=K.+Masukoauthor=Y.+Tanakaauthor=H.+Katoauthor=Y.+Mizunoauthor=M.+Yokoeauthor=F.+Sugauchiauthor=N.+Hirashimaauthor=E.+Oritoauthor=H.+Osadaauthor=R.+Uedaauthor=Y.+Guoauthor=X.+Chenauthor=J.+P.+Issaauthor=Y.+Sekido&title=Alterations+of+DNA+methylation+and+histone+modifications+contribute+to+gene+silencing+in+hepatocellular+carcinomas"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Alterations of DNA methylation and histone modifications contribute to gene silencing in hepatocellular carcinomas</span></div><div class="casAuthors">Kondo, Yutaka; Shen, Lanlan; Suzuki, Seiji; Kurokawa, Tsuyoshi; Masuko, Kazuo; Tanaka, Yasuhito; Kato, Hideaki; Mizuno, Yoshiki; Yokoe, Masamichi; Sugauchi, Fuminaka; Hirashima, Noboru; Orito, Etsuro; Osada, Hirotaka; Ueda, Ryuzo; Guo, Yi; Chen, Xinli; Issa, Jean-Pierre J.; Sekido, Yoshitaka</div><div class="citationInfo"><span class="NLM_cas:title">Hepatology Research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">974-983</span>CODEN:
                <span class="NLM_cas:coden">HPRSFM</span>;
        ISSN:<span class="NLM_cas:issn">1386-6346</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Asia Pty Ltd.</span>)
        </div><div class="casAbstract">Aim: The aim of the present study was to examine DNA methylation and histone modification changes in hepatocellular carcinomas (HCC).  Methods: DNA methylation in the P16, RASSF1a, progesterone receptor (PGR) and estrogen receptor α (ERα) promoters was detd. by quant. bisulfite-pyrosequencing technique in HCC patients.  Histone H3-lysine (K) 4, H3-K9 and H3-K27 modifications in all these four genes were examd. by chromatin immunopptn. (ChIP) assay in HCC cell lines.  Expression of two DNA methyltransferases (DNMT1 and DNMT3b) and three histone methyltransferases (SUV39H1, G9a and EZH2) in HCC patients was measured by real-time polymerase chain reaction.  Results: Aberrant DNA methylation was detected in all the HCC.  Patients with DNA methylation in the RASSF1a, PGR and ERα promoters in cancers also had substantial DNA methylation in their non-cancerous liver tissues, whereas DNA methylation in the P16 promoter was cancer specific.  Epigenetic states in HCC cell lines showed that silencing of P16 and RASSF1a depended on DNA methylation and histone H3-K9 methylation.  However, silencing of the PGR and ERα genes was more closely related to H3-K27 methylation rather than DNA methylation.  Consistent with the alteration of histone status, higher expression of G9a and EZH2 was found in HCC than in non-cancerous liver tissues (P < 0.01).  Conclusion: These data suggest that multiple epigenetic silencing mechanisms are inappropriately active in HCC cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwkWfvYpsUX7Vg90H21EOLACvtfcHk0lj9CxlU8_ToEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlCns7fK&md5=afc7013e4adfd4dd30d0b0e07ce8d9ef</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1111%2Fj.1872-034X.2007.00141.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1872-034X.2007.00141.x%26sid%3Dliteratum%253Aachs%26aulast%3DKondo%26aufirst%3DY.%26aulast%3DShen%26aufirst%3DL.%26aulast%3DSuzuki%26aufirst%3DS.%26aulast%3DKurokawa%26aufirst%3DT.%26aulast%3DMasuko%26aufirst%3DK.%26aulast%3DTanaka%26aufirst%3DY.%26aulast%3DKato%26aufirst%3DH.%26aulast%3DMizuno%26aufirst%3DY.%26aulast%3DYokoe%26aufirst%3DM.%26aulast%3DSugauchi%26aufirst%3DF.%26aulast%3DHirashima%26aufirst%3DN.%26aulast%3DOrito%26aufirst%3DE.%26aulast%3DOsada%26aufirst%3DH.%26aulast%3DUeda%26aufirst%3DR.%26aulast%3DGuo%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DIssa%26aufirst%3DJ.%2BP.%26aulast%3DSekido%26aufirst%3DY.%26atitle%3DAlterations%2520of%2520DNA%2520methylation%2520and%2520histone%2520modifications%2520contribute%2520to%2520gene%2520silencing%2520in%2520hepatocellular%2520carcinomas%26jtitle%3DHepatol.%2520Res.%26date%3D2007%26volume%3D37%26spage%3D974%26epage%3D983" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Watanabe, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soejima, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yasuda, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawada, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakachi, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoda, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Naoki, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ishizaka, A.</span><span> </span><span class="NLM_article-title">Deregulation of histone lysine methyltransferases contributes to oncogenic transformation of human bronchoepithelial cells</span> <span class="citation_source-journal">Cancer Cell Int.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">15</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Fjm200903z&amp;key=10.1186%2F1475-2867-8-15" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Fjm200903z&amp;key=18980680" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Fjm200903z&amp;key=1%3ACAS%3A280%3ADC%252BD1cjltFSntg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2008&pages=15&author=H.+Watanabeauthor=K.+Soejimaauthor=H.+Yasudaauthor=I.+Kawadaauthor=I.+Nakachiauthor=S.+Yodaauthor=K.+Naokiauthor=A.+Ishizaka&title=Deregulation+of+histone+lysine+methyltransferases+contributes+to+oncogenic+transformation+of+human+bronchoepithelial+cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Deregulation of histone lysine methyltransferases contributes to oncogenic transformation of human bronchoepithelial cells</span></div><div class="casAuthors">Watanabe Hideo; Soejima Kenzo; Yasuda Hiroyuki; Kawada Ichiro; Nakachi Ichiro; Yoda Satoshi; Naoki Katsuhiko; Ishizaka Akitoshi</div><div class="citationInfo"><span class="NLM_cas:title">Cancer cell international</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">15</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Alterations in the processing of the genetic information in carcinogenesis result from stable genetic mutations or epigenetic modifications.  It is becoming clear that nucleosomal histones are central to proper gene expression and that aberrant DNA methylation of genes and histone methylation plays important roles in tumor progression.  To date, several histone lysine methyltransferases (HKMTs) have been identified and histone lysine methylation is now considered to be a critical regulator of transcription.  However, still relatively little is known about the role of HKMTs in tumorigenesis.  RESULTS:  We observed differential HKMT expression in a lung cancer model in which normal human bronchial epithelial (NHBE) cells expressing telomerase, SV40 large T antigen, and Ras were immortal, formed colonies in soft agar, and expressed specific HKMTs for H3 lysine 9 and 27 residues but not for H3 lysine 4 residue.  Modifications in the H3 tails affect the binding of proteins to the histone tails and regulate protein function and the position of lysine methylation marks a gene to be either activated or repressed.  In the present study, suppression by siRNA of HKMTs (EZH2, G9A, SETDB1 and SUV39H1) that are over-expressed in immortalized and transformed cells lead to reduced cell proliferation and much less anchorage-independent colony growth.  We also found that the suppression of H3-K9, G9A and SUV39H1 induced apoptosis and the suppression of H3-K27, EZH2 caused G1 arrest.  CONCLUSION:  Our results indicate the potential of these HKMTs in addition to the other targets for epigenetics such as DNMTs and HDACs to be interesting therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ8X-3k_fB763y-c-E1v-fmfW6udTcc2eYDKLFIv1wKEbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1cjltFSntg%253D%253D&md5=ac77cc1b0cca2a4d8bf111fb6463ee7e</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1186%2F1475-2867-8-15&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1475-2867-8-15%26sid%3Dliteratum%253Aachs%26aulast%3DWatanabe%26aufirst%3DH.%26aulast%3DSoejima%26aufirst%3DK.%26aulast%3DYasuda%26aufirst%3DH.%26aulast%3DKawada%26aufirst%3DI.%26aulast%3DNakachi%26aufirst%3DI.%26aulast%3DYoda%26aufirst%3DS.%26aulast%3DNaoki%26aufirst%3DK.%26aulast%3DIshizaka%26aufirst%3DA.%26atitle%3DDeregulation%2520of%2520histone%2520lysine%2520methyltransferases%2520contributes%2520to%2520oncogenic%2520transformation%2520of%2520human%2520bronchoepithelial%2520cells%26jtitle%3DCancer%2520Cell%2520Int.%26date%3D2008%26volume%3D8%26spage%3D15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">McGarvey, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fahrner, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greene, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martens, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenuwein, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baylin, S. B.</span><span> </span><span class="NLM_article-title">Silenced tumor suppressor genes reactivated by DNA demethylation do not return to a fully euchromatic chromatin state</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">3541</span><span class="NLM_x">–</span> <span class="NLM_lpage">3549</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Fjm200903z&amp;key=10.1158%2F0008-5472.CAN-05-2481" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Fjm200903z&amp;key=16585178" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Fjm200903z&amp;key=1%3ACAS%3A528%3ADC%252BD28XjtVOrsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2006&pages=3541-3549&author=K.+M.+McGarveyauthor=J.+A.+Fahrnerauthor=E.+Greeneauthor=J.+Martensauthor=T.+Jenuweinauthor=S.+B.+Baylin&title=Silenced+tumor+suppressor+genes+reactivated+by+DNA+demethylation+do+not+return+to+a+fully+euchromatic+chromatin+state"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Silenced Tumor Suppressor Genes Reactivated by DNA Demethylation Do Not Return to a Fully Euchromatic Chromatin State</span></div><div class="casAuthors">McGarvey, Kelly M.; Fahrner, Jill A.; Greene, Eriko; Martens, Joost; Jenuwein, Thomas; Baylin, Stephen B.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3541-3549</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Histone H3 lysine 9 (H3K9) and lysine 27 (H3K27) trimethylation are properties of stably silenced heterochromatin whereas H3K9 dimethylation (H3K9me2) is important for euchromatic gene repression.  In colorectal cancer cells, all of these marks, as well as the key enzymes which establish them, surround the hMLH1 promoter when it is DNA hypermethylated and aberrantly silenced, but are absent when the gene is unmethylated and fully expressed in a euchromatic state.  When the aberrantly silenced gene is DNA demethylated and reexpressed following 5-aza-2'-deoxycytidine treatment, H3K9me1 and H3K9me2 are the only silencing marks that are lost.  A series of other silenced and DNA hypermethylated gene promoters behave identically even when the genes are chronically DNA demethylated and reexpressed after genetic knockout of DNA methyltransferases.  Our data indicate that when transcription of DNA hypermethylated genes is activated in cancer cells, their promoters remain in an environment with certain heterochromatic characteristics.  This finding has important implications for the translational goal of reactivating aberrantly silenced cancer genes as a therapeutic maneuver.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEjoUm5wwMFbVg90H21EOLACvtfcHk0lgyUF_S5nPxjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XjtVOrsro%253D&md5=73b1830a5f7f839d53d5539c6e707462</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-05-2481&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-05-2481%26sid%3Dliteratum%253Aachs%26aulast%3DMcGarvey%26aufirst%3DK.%2BM.%26aulast%3DFahrner%26aufirst%3DJ.%2BA.%26aulast%3DGreene%26aufirst%3DE.%26aulast%3DMartens%26aufirst%3DJ.%26aulast%3DJenuwein%26aufirst%3DT.%26aulast%3DBaylin%26aufirst%3DS.%2BB.%26atitle%3DSilenced%2520tumor%2520suppressor%2520genes%2520reactivated%2520by%2520DNA%2520demethylation%2520do%2520not%2520return%2520to%2520a%2520fully%2520euchromatic%2520chromatin%2520state%26jtitle%3DCancer%2520Res.%26date%3D2006%26volume%3D66%26spage%3D3541%26epage%3D3549" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Goyama, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nitta, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshino, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kako, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watanabe-Okochi, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shimabe, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Imai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takahashi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurokawa, M.</span><span> </span><span class="NLM_article-title">EVI-1 interacts with histone methyltransferases SUV39H1 and G9a for transcriptional repression and bone marrow immortalization</span> <span class="citation_source-journal">Leukemia</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">81</span><span class="NLM_x">–</span> <span class="NLM_lpage">88</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2010&pages=81-88&author=S.+Goyamaauthor=E.+Nittaauthor=T.+Yoshinoauthor=S.+Kakoauthor=N.+Watanabe-Okochiauthor=M.+Shimabeauthor=Y.+Imaiauthor=K.+Takahashiauthor=M.+Kurokawa&title=EVI-1+interacts+with+histone+methyltransferases+SUV39H1+and+G9a+for+transcriptional+repression+and+bone+marrow+immortalization"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGoyama%26aufirst%3DS.%26aulast%3DNitta%26aufirst%3DE.%26aulast%3DYoshino%26aufirst%3DT.%26aulast%3DKako%26aufirst%3DS.%26aulast%3DWatanabe-Okochi%26aufirst%3DN.%26aulast%3DShimabe%26aufirst%3DM.%26aulast%3DImai%26aufirst%3DY.%26aulast%3DTakahashi%26aufirst%3DK.%26aulast%3DKurokawa%26aufirst%3DM.%26atitle%3DEVI-1%2520interacts%2520with%2520histone%2520methyltransferases%2520SUV39H1%2520and%2520G9a%2520for%2520transcriptional%2520repression%2520and%2520bone%2520marrow%2520immortalization%26jtitle%3DLeukemia%26date%3D2010%26volume%3D24%26spage%3D81%26epage%3D88" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Maze, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Covington, H. E.,  III</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dietz, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LaPlant, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Renthal, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Russo, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mechanic, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mouzon, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neve, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haggarty, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ren, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sampath, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurd, Y. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greengard, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tarakhovsky, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schaefer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nestler, E. J.</span><span> </span><span class="NLM_article-title">Essential role of the histone methyltransferase G9a in cocaine-induced plasticity</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">327</span><span class="NLM_x">, </span> <span class="NLM_fpage">213</span><span class="NLM_x">–</span> <span class="NLM_lpage">216</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Fjm200903z&amp;key=10.1126%2Fscience.1179438" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Fjm200903z&amp;key=20056891" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Fjm200903z&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1egtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=327&publication_year=2010&pages=213-216&author=I.+Mazeauthor=H.+E.+Covingtonauthor=D.+M.+Dietzauthor=Q.+LaPlantauthor=W.+Renthalauthor=S.+J.+Russoauthor=M.+Mechanicauthor=E.+Mouzonauthor=R.+L.+Neveauthor=S.+J.+Haggartyauthor=Y.+Renauthor=S.+C.+Sampathauthor=Y.+L.+Hurdauthor=P.+Greengardauthor=A.+Tarakhovskyauthor=A.+Schaeferauthor=E.+J.+Nestler&title=Essential+role+of+the+histone+methyltransferase+G9a+in+cocaine-induced+plasticity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Essential Role of the Histone Methyltransferase G9a in Cocaine-Induced Plasticity</span></div><div class="casAuthors">Maze, Ian; Covington, Herbert E., III; Dietz, David M.; LaPlant, Quincey; Renthal, William; Russo, Scott J.; Mechanic, Max; Mouzon, Ezekiell; Neve, Rachael L.; Haggarty, Stephen J.; Ren, Yanhua; Sampath, Srihari C.; Hurd, Yasmin L.; Greengard, Paul; Tarakhovsky, Alexander; Schaefer, Anne; Nestler, Eric J.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">327</span>
        (<span class="NLM_cas:issue">5962</span>),
    <span class="NLM_cas:pages">213-216</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Cocaine-induced alterations in gene expression cause changes in neuronal morphol. and behavior that may underlie cocaine addiction.  In mice, we identified an essential role for histone 3 lysine 9 (H3K9) dimethylation and the lysine dimethyltransferase G9a in cocaine-induced structural and behavioral plasticity.  Repeated cocaine administration reduced global levels of H3K9 dimethylation in the nucleus accumbens.  This redn. in histone methylation was mediated through the repression of G9a in this brain region, which was regulated by the cocaine-induced transcription factor ΔFosB.  Using conditional mutagenesis and viral-mediated gene transfer, we found that G9a down-regulation increased the dendritic spine plasticity of nucleus accumbens neurons and enhanced the preference for cocaine, thereby establishing a crucial role for histone methylation in the long-term actions of cocaine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8SPD826wEGrVg90H21EOLACvtfcHk0ljcNoV1t2pSrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1egtA%253D%253D&md5=a8347500f81862078787e50fff53b6b9</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1126%2Fscience.1179438&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1179438%26sid%3Dliteratum%253Aachs%26aulast%3DMaze%26aufirst%3DI.%26aulast%3DCovington%26aufirst%3DH.%2BE.%26aulast%3DDietz%26aufirst%3DD.%2BM.%26aulast%3DLaPlant%26aufirst%3DQ.%26aulast%3DRenthal%26aufirst%3DW.%26aulast%3DRusso%26aufirst%3DS.%2BJ.%26aulast%3DMechanic%26aufirst%3DM.%26aulast%3DMouzon%26aufirst%3DE.%26aulast%3DNeve%26aufirst%3DR.%2BL.%26aulast%3DHaggarty%26aufirst%3DS.%2BJ.%26aulast%3DRen%26aufirst%3DY.%26aulast%3DSampath%26aufirst%3DS.%2BC.%26aulast%3DHurd%26aufirst%3DY.%2BL.%26aulast%3DGreengard%26aufirst%3DP.%26aulast%3DTarakhovsky%26aufirst%3DA.%26aulast%3DSchaefer%26aufirst%3DA.%26aulast%3DNestler%26aufirst%3DE.%2BJ.%26atitle%3DEssential%2520role%2520of%2520the%2520histone%2520methyltransferase%2520G9a%2520in%2520cocaine-induced%2520plasticity%26jtitle%3DScience%26date%3D2010%26volume%3D327%26spage%3D213%26epage%3D216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Schaefer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sampath, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Intrator, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Min, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gertler, T. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Surmeier, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tarakhovsky, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greengard, P.</span><span> </span><span class="NLM_article-title">Control of Cognition and Adaptive Behavior by the GLP/G9a Epigenetic Suppressor Complex</span> <span class="citation_source-journal">Neuron</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">64</span><span class="NLM_x">, </span> <span class="NLM_fpage">678</span><span class="NLM_x">–</span> <span class="NLM_lpage">691</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Fjm200903z&amp;key=10.1016%2Fj.neuron.2009.11.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Fjm200903z&amp;key=20005824" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Fjm200903z&amp;key=1%3ACAS%3A528%3ADC%252BC3cXks1Cls7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2009&pages=678-691&author=A.+Schaeferauthor=S.+C.+Sampathauthor=A.+Intratorauthor=A.+Minauthor=T.+S.+Gertlerauthor=D.+J.+Surmeierauthor=A.+Tarakhovskyauthor=P.+Greengard&title=Control+of+Cognition+and+Adaptive+Behavior+by+the+GLP%2FG9a+Epigenetic+Suppressor+Complex"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Control of cognition and adaptive behavior by the GLP/G9a epigenetic suppressor complex</span></div><div class="casAuthors">Schaefer, Anne; Sampath, Srihari C.; Intrator, Adam; Min, Alice; Gertler, Tracy S.; Surmeier, D. James; Tarakhovsky, Alexander; Greengard, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Neuron</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">678-691</span>CODEN:
                <span class="NLM_cas:coden">NERNET</span>;
        ISSN:<span class="NLM_cas:issn">0896-6273</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The genetic basis of cognition and behavioral adaptation to the environment remains poorly understood.  Here we demonstrate that the histone methyltransferase complex GLP/G9a controls cognition and adaptive responses in a region-specific fashion in the adult brain.  Using conditional mutagenesis in mice, we show that postnatal, neuron-specific deficiency of GLP/G9a leads to derepression of numerous nonneuronal and neuron progenitor genes in adult neurons.  This transcriptional alteration is assocd. with complex behavioral abnormalities, including defects in learning, motivation, and environmental adaptation.  The behavioral changes triggered by GLP/G9a deficiency are similar to key symptoms of the human 9q34 mental retardation syndrome that is assocd. with structural alterations of the GLP/EHMT1 gene.  The likely causal role of GLP/G9a in mental retardation in mice and humans suggests a key role for the GLP/G9a-controlled histone H3K9 dimethylation in regulation of brain function through maintenance of the transcriptional homeostasis in adult neurons.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr22UcjjqkKdLVg90H21EOLACvtfcHk0ljcNoV1t2pSrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXks1Cls7c%253D&md5=ece4e94ad097a8d0e67a5e8d27970e64</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.neuron.2009.11.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuron.2009.11.019%26sid%3Dliteratum%253Aachs%26aulast%3DSchaefer%26aufirst%3DA.%26aulast%3DSampath%26aufirst%3DS.%2BC.%26aulast%3DIntrator%26aufirst%3DA.%26aulast%3DMin%26aufirst%3DA.%26aulast%3DGertler%26aufirst%3DT.%2BS.%26aulast%3DSurmeier%26aufirst%3DD.%2BJ.%26aulast%3DTarakhovsky%26aufirst%3DA.%26aulast%3DGreengard%26aufirst%3DP.%26atitle%3DControl%2520of%2520Cognition%2520and%2520Adaptive%2520Behavior%2520by%2520the%2520GLP%252FG9a%2520Epigenetic%2520Suppressor%2520Complex%26jtitle%3DNeuron%26date%3D2009%26volume%3D64%26spage%3D678%26epage%3D691" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Imai, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Togami, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okamoto, T.</span><span> </span><span class="NLM_article-title">Involvement of histone H3 Lysine 9 (H3K9) methyl transferase G9a in the maintenance of HIV-1 latency and its reactivation by BIX01294</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">285</span><span class="NLM_x">, </span> <span class="NLM_fpage">16538</span><span class="NLM_x">–</span> <span class="NLM_lpage">16545</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=16538-16545&author=K.+Imaiauthor=H.+Togamiauthor=T.+Okamoto&title=Involvement+of+histone+H3+Lysine+9+%28H3K9%29+methyl+transferase+G9a+in+the+maintenance+of+HIV-1+latency+and+its+reactivation+by+BIX01294"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DImai%26aufirst%3DK.%26aulast%3DTogami%26aufirst%3DH.%26aulast%3DOkamoto%26aufirst%3DT.%26atitle%3DInvolvement%2520of%2520histone%2520H3%2520Lysine%25209%2520%2528H3K9%2529%2520methyl%2520transferase%2520G9a%2520in%2520the%2520maintenance%2520of%2520HIV-1%2520latency%2520and%2520its%2520reactivation%2520by%2520BIX01294%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D16538%26epage%3D16545" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Link, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gangisetty, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">James, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woloszynska-Read, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tachibana, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shinkai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karpf, A. R.</span><span> </span><span class="NLM_article-title">Distinct roles for histone methyltransferases G9a and GLP in cancer germ-line antigen gene regulation in human cancer cells and murine embryonic stem cells</span> <span class="citation_source-journal">Mol. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">851</span><span class="NLM_x">–</span> <span class="NLM_lpage">862</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Fjm200903z&amp;key=10.1158%2F1541-7786.MCR-08-0497" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Fjm200903z&amp;key=19531572" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Fjm200903z&amp;key=1%3ACAS%3A528%3ADC%252BD1MXns1Skt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2009&pages=851-862&author=P.+A.+Linkauthor=O.+Gangisettyauthor=S.+R.+Jamesauthor=A.+Woloszynska-Readauthor=M.+Tachibanaauthor=Y.+Shinkaiauthor=A.+R.+Karpf&title=Distinct+roles+for+histone+methyltransferases+G9a+and+GLP+in+cancer+germ-line+antigen+gene+regulation+in+human+cancer+cells+and+murine+embryonic+stem+cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Distinct Roles for Histone Methyltransferases G9a and GLP in Cancer Germ-Line Antigen Gene Regulation in Human Cancer Cells and Murine Embryonic Stem Cells</span></div><div class="casAuthors">Link, Petra A.; Gangisetty, Omkaram; James, Smitha R.; Woloszynska-Read, Anna; Tachibana, Makoto; Shinkai, Yoichi; Karpf, Adam R.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">851-862</span>CODEN:
                <span class="NLM_cas:coden">MCROC5</span>;
        ISSN:<span class="NLM_cas:issn">1541-7786</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The H3K9me2 histone methyltransferases G9a and GLP repress Mage-a class cancer germ-line (CG) antigen gene expression in murine embryonic stem (ES) cells, but the role of these enzymes in CG antigen gene regulation in human cancer cells is unknown.  Here we show that whereas independent or dual knockdown of G9a and GLP in human cancer cells leads to reduced global and CG antigen promoter-assocd. H3K9me2 levels, it does not activate CG antigen gene expression.  Moreover, CG antigen gene repression is maintained following pharmacol. targeting of G9a or treatment of G9a knockdown cells with the histone deacetylase inhibitor trichostatin A.  However, G9a knockdown cells display increased sensitivity to CG antigen gene activation mediated by the DNA methyltransferase inhibitor decitabine.  To account for these findings, we examd. DNA methylation at CG antigen gene promoters in both cell types.  We found robust DNA hypomethylation in G9a/GLP targeted murine ES cells but a lack of DNA methylation changes in G9a/GLP targeted human cancer cells; intriguingly, this distinction also extended to markers of global DNA methylation.  These data reveal that G9a/GLP is required for DNA methylation of CG antigen genes and genomic DNA in murine ES cells, but not human cancer cells, and implicate DNA methylation status as the key epigenetic mechanism involved in CG antigen gene repression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTDHnsCTv6qLVg90H21EOLACvtfcHk0ljkmYE3vuNKFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXns1Skt74%253D&md5=fc1f607388e192a0768a22584f987cda</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1158%2F1541-7786.MCR-08-0497&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1541-7786.MCR-08-0497%26sid%3Dliteratum%253Aachs%26aulast%3DLink%26aufirst%3DP.%2BA.%26aulast%3DGangisetty%26aufirst%3DO.%26aulast%3DJames%26aufirst%3DS.%2BR.%26aulast%3DWoloszynska-Read%26aufirst%3DA.%26aulast%3DTachibana%26aufirst%3DM.%26aulast%3DShinkai%26aufirst%3DY.%26aulast%3DKarpf%26aufirst%3DA.%2BR.%26atitle%3DDistinct%2520roles%2520for%2520histone%2520methyltransferases%2520G9a%2520and%2520GLP%2520in%2520cancer%2520germ-line%2520antigen%2520gene%2520regulation%2520in%2520human%2520cancer%2520cells%2520and%2520murine%2520embryonic%2520stem%2520cells%26jtitle%3DMol.%2520Cancer%2520Res.%26date%3D2009%26volume%3D7%26spage%3D851%26epage%3D862" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Tachibana, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matsumura, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fukuda, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kimura, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shinkai, Y.</span><span> </span><span class="NLM_article-title">G9a/GLP complexes independently mediate H3K9 and DNA methylation to silence transcription</span> <span class="citation_source-journal">EMBO J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">2681</span><span class="NLM_x">–</span> <span class="NLM_lpage">2690</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2008&pages=2681-2690&author=M.+Tachibanaauthor=Y.+Matsumuraauthor=M.+Fukudaauthor=H.+Kimuraauthor=Y.+Shinkai&title=G9a%2FGLP+complexes+independently+mediate+H3K9+and+DNA+methylation+to+silence+transcription"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTachibana%26aufirst%3DM.%26aulast%3DMatsumura%26aufirst%3DY.%26aulast%3DFukuda%26aufirst%3DM.%26aulast%3DKimura%26aufirst%3DH.%26aulast%3DShinkai%26aufirst%3DY.%26atitle%3DG9a%252FGLP%2520complexes%2520independently%2520mediate%2520H3K9%2520and%2520DNA%2520methylation%2520to%2520silence%2520transcription%26jtitle%3DEMBO%2520J.%26date%3D2008%26volume%3D27%26spage%3D2681%26epage%3D2690" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Dong, K. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maksakova, I. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mohn, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leung, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Appanah, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, H. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lam, L. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mager, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schubeler, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tachibana, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shinkai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lorincz, M. C.</span><span> </span><span class="NLM_article-title">DNA methylation in ES cells requires the lysine methyltransferase G9a but not its catalytic activity</span> <span class="citation_source-journal">EMBO J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">2691</span><span class="NLM_x">–</span> <span class="NLM_lpage">2701</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2008&pages=2691-2701&author=K.+B.+Dongauthor=I.+A.+Maksakovaauthor=F.+Mohnauthor=D.+Leungauthor=R.+Appanahauthor=S.+Leeauthor=H.+W.+Yangauthor=L.+L.+Lamauthor=D.+L.+Magerauthor=D.+Schubelerauthor=M.+Tachibanaauthor=Y.+Shinkaiauthor=M.+C.+Lorincz&title=DNA+methylation+in+ES+cells+requires+the+lysine+methyltransferase+G9a+but+not+its+catalytic+activity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDong%26aufirst%3DK.%2BB.%26aulast%3DMaksakova%26aufirst%3DI.%2BA.%26aulast%3DMohn%26aufirst%3DF.%26aulast%3DLeung%26aufirst%3DD.%26aulast%3DAppanah%26aufirst%3DR.%26aulast%3DLee%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DH.%2BW.%26aulast%3DLam%26aufirst%3DL.%2BL.%26aulast%3DMager%26aufirst%3DD.%2BL.%26aulast%3DSchubeler%26aufirst%3DD.%26aulast%3DTachibana%26aufirst%3DM.%26aulast%3DShinkai%26aufirst%3DY.%26aulast%3DLorincz%26aufirst%3DM.%2BC.%26atitle%3DDNA%2520methylation%2520in%2520ES%2520cells%2520requires%2520the%2520lysine%2520methyltransferase%2520G9a%2520but%2520not%2520its%2520catalytic%2520activity%26jtitle%3DEMBO%2520J.%26date%3D2008%26volume%3D27%26spage%3D2691%26epage%3D2701" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Kubicek, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Sullivan, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">August, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hickey, E. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teodoro, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rea, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mechtler, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kowalski, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Homon, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kelly, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenuwein, T.</span><span> </span><span class="NLM_article-title">Reversal of H3K9me2 by a small-molecule inhibitor for the G9a histone methyltransferase</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">473</span><span class="NLM_x">–</span> <span class="NLM_lpage">481</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Fjm200903z&amp;key=10.1016%2Fj.molcel.2007.01.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Fjm200903z&amp;key=17289593" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Fjm200903z&amp;key=1%3ACAS%3A528%3ADC%252BD2sXitl2jurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2007&pages=473-481&author=S.+Kubicekauthor=R.+J.+O%E2%80%99Sullivanauthor=E.+M.+Augustauthor=E.+R.+Hickeyauthor=Q.+Zhangauthor=M.+L.+Teodoroauthor=S.+Reaauthor=K.+Mechtlerauthor=J.+A.+Kowalskiauthor=C.+A.+Homonauthor=T.+A.+Kellyauthor=T.+Jenuwein&title=Reversal+of+H3K9me2+by+a+small-molecule+inhibitor+for+the+G9a+histone+methyltransferase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Reversal of H3K9me2 by a small-molecule inhibitor for the G9a histone methyltransferase</span></div><div class="casAuthors">Kubicek, Stefan; O'Sullivan, Roderick J.; August, E. Michael; Hickey, Eugene R.; Zhang, Qiang; Teodoro, Miguel L.; Rea, Stephen; Mechtler, Karl; Kowalski, Jennifer A.; Homon, Carol Ann; Kelly, Terence A.; Jenuwein, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">473-481</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Histone lysine methylation has important roles in the organization of chromatin domains and the regulation of gene expression.  To analyze its function and modulate its activity, we screened for specific inhibitors against histone lysine methyltransferases (HMTases) using recombinant G9a as the target enzyme.  From a chem. library comprising 125,000 preselected compds., seven hits were identified.  Of those, one inhibitor, BIX-01294 (diazepinquinazolin-amine deriv.), does not compete with the cofactor S-adenosyl-methionine, and selectively impairs the G9a HMTase and the generation of H3K9me2 in vitro.  In cellular assays, transient incubation of several cell lines with BIX-01294 lowers bulk H3K9me2 levels that are restored upon removal of the inhibitor.  Importantly, chromatin immunopptn. at several G9a target genes demonstrates reversible redn. of promoter-proximal H3K9me2 in inhibitor-treated mouse ES cells and fibroblasts.  Our data identify a biol. active HMTase inhibitor that allows for the transient modulation of H3K9me2 marks in mammalian chromatin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3SW8czS4WjbVg90H21EOLACvtfcHk0liZ1iTPP9Oy6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXitl2jurs%253D&md5=d2f5290f1252e630a9b05489b0775c2c</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2007.01.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2007.01.017%26sid%3Dliteratum%253Aachs%26aulast%3DKubicek%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Sullivan%26aufirst%3DR.%2BJ.%26aulast%3DAugust%26aufirst%3DE.%2BM.%26aulast%3DHickey%26aufirst%3DE.%2BR.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DTeodoro%26aufirst%3DM.%2BL.%26aulast%3DRea%26aufirst%3DS.%26aulast%3DMechtler%26aufirst%3DK.%26aulast%3DKowalski%26aufirst%3DJ.%2BA.%26aulast%3DHomon%26aufirst%3DC.%2BA.%26aulast%3DKelly%26aufirst%3DT.%2BA.%26aulast%3DJenuwein%26aufirst%3DT.%26atitle%3DReversal%2520of%2520H3K9me2%2520by%2520a%2520small-molecule%2520inhibitor%2520for%2520the%2520G9a%2520histone%2520methyltransferase%26jtitle%3DMol.%2520Cell%26date%3D2007%26volume%3D25%26spage%3D473%26epage%3D481" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Chang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horton, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Upadhyay, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spannhoff, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Snyder, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, X.</span><span> </span><span class="NLM_article-title">Structural basis for G9a-like protein lysine methyltransferase inhibition by BIX-01294</span> <span class="citation_source-journal">Nature Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">312</span><span class="NLM_x">–</span> <span class="NLM_lpage">317</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Fjm200903z&amp;key=10.1038%2Fnsmb.1560" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Fjm200903z&amp;key=19219047" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Fjm200903z&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhvFKjt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2009&pages=312-317&author=Y.+Changauthor=X.+Zhangauthor=J.+R.+Hortonauthor=A.+K.+Upadhyayauthor=A.+Spannhoffauthor=J.+Liuauthor=J.+P.+Snyderauthor=M.+T.+Bedfordauthor=X.+Cheng&title=Structural+basis+for+G9a-like+protein+lysine+methyltransferase+inhibition+by+BIX-01294"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for G9a-like protein lysine methyltransferase inhibition by BIX-01294</span></div><div class="casAuthors">Chang, Yanqi; Zhang, Xing; Horton, John R.; Upadhyay, Anup K.; Spannhoff, Astrid; Liu, Jin; Snyder, James P.; Bedford, Mark T.; Cheng, Xiaodong</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">312-317</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Histone lysine methylation is an important epigenetic mark that regulates gene expression and chromatin organization.  G9a and G9a-like protein (GLP) are euchromatin-assocd. methyltransferases that repress transcription by methylating histone H3 Lys9.  BIX-01294 was originally identified as a G9a inhibitor during a chem. library screen of small mols. and has previously been used in the generation of induced pluripotent stem cells.  Here we present the crystal structure of the catalytic SET domain of GLP in complex with BIX-01294 and S-adenosyl-L-homocysteine.  The inhibitor is bound in the substrate peptide groove at the location where the histone H3 residues N-terminal to the target lysine lie in the previously solved structure of the complex with histone peptide.  The inhibitor resembles the bound conformation of histone H3 Lys4 to Arg8, and is positioned in place by residues specific for G9a and GLP through specific interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhQepqN3zr_bVg90H21EOLACvtfcHk0lhKgdYsC6iomA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhvFKjt7g%253D&md5=6a16baaf33420fc1caf1df70d103d270</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.1560&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.1560%26sid%3Dliteratum%253Aachs%26aulast%3DChang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DHorton%26aufirst%3DJ.%2BR.%26aulast%3DUpadhyay%26aufirst%3DA.%2BK.%26aulast%3DSpannhoff%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DSnyder%26aufirst%3DJ.%2BP.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DCheng%26aufirst%3DX.%26atitle%3DStructural%2520basis%2520for%2520G9a-like%2520protein%2520lysine%2520methyltransferase%2520inhibition%2520by%2520BIX-01294%26jtitle%3DNature%2520Struct.%2520Mol.%2520Biol.%26date%3D2009%26volume%3D16%26spage%3D312%26epage%3D317" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quinn, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kozieradzki, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siarheyeva, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kireev, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jadhav, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herold, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Discovery of a 2,4-diamino-7-aminoalkoxyquinazoline as a potent and selective inhibitor of histone lysine methyltransferase G9a</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">7950</span><span class="NLM_x">–</span> <span class="NLM_lpage">7953</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm901543m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Fjm200903z&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlOiurzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=7950-7953&author=F.+Liuauthor=X.+Chenauthor=A.+Allali-Hassaniauthor=A.+M.+Quinnauthor=G.+A.+Wasneyauthor=A.+Dongauthor=D.+Barsyteauthor=I.+Kozieradzkiauthor=G.+Senisterraauthor=I.+Chauauthor=A.+Siarheyevaauthor=D.+B.+Kireevauthor=A.+Jadhavauthor=J.+M.+Heroldauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=A.+Simeonovauthor=M.+Vedadiauthor=J.+Jin&title=Discovery+of+a+2%2C4-diamino-7-aminoalkoxyquinazoline+as+a+potent+and+selective+inhibitor+of+histone+lysine+methyltransferase+G9a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a 2,4-Diamino-7-aminoalkoxyquinazoline as a Potent and Selective Inhibitor of Histone Lysine Methyltransferase G9a</span></div><div class="casAuthors">Liu, Feng; Chen, Xin; Allali-Hassani, Abdellah; Quinn, Amy M.; Wasney, Gregory A.; Dong, Aiping; Barsyte, Dalia; Kozieradzki, Ivona; Senisterra, Guillermo; Chau, Irene; Siarheyeva, Alena; Kireev, Dmitri B.; Jadhav, Ajit; Herold, J. Martin; Frye, Stephen V.; Arrowsmith, Cheryl H.; Brown, Peter J.; Simeonov, Anton; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">7950-7953</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">SAR exploration of the 2,4-diamino-6,7-dimethoxyquinazoline template led to the discovery of 8 (I) as a potent and selective G9a inhibitor.  A high resoln. X-ray crystal structure of the G9a-8 complex, the first cocrystal structure of G9a with a small mol. inhibitor, was obtained.  The cocrystal structure validated our binding hypothesis and will enable structure-based design of novel inhibitors.  8 Is a useful tool for investigating the biol. of G9a and its roles in chromatin remodeling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjWt5XsGXyj7Vg90H21EOLACvtfcHk0ljIVPBqjXdeLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlOiurzN&md5=2d7e34a8c4b5bf1cafd4e01ad2a86de1</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm901543m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901543m%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DQuinn%26aufirst%3DA.%2BM.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DBarsyte%26aufirst%3DD.%26aulast%3DKozieradzki%26aufirst%3DI.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DKireev%26aufirst%3DD.%2BB.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DHerold%26aufirst%3DJ.%2BM.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520a%25202%252C4-diamino-7-aminoalkoxyquinazoline%2520as%2520a%2520potent%2520and%2520selective%2520inhibitor%2520of%2520histone%2520lysine%2520methyltransferase%2520G9a%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D7950%26epage%3D7953" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quinn, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siarheyeva, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norris, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kireev, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jadhav, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herold, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzen, W. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Protein Lysine Methyltransferase G9a Inhibitors: Design, Synthesis, and Structure–Activity Relationships of 2,4-Diamino-7-aminoalkoxy-quinazolines</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">5844</span><span class="NLM_x">–</span> <span class="NLM_lpage">5857</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm100478y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=5844-5857&author=F.+Liuauthor=X.+Chenauthor=A.+Allali-Hassaniauthor=A.+M.+Quinnauthor=T.+J.+Wigleauthor=G.+A.+Wasneyauthor=A.+Dongauthor=G.+Senisterraauthor=I.+Chauauthor=A.+Siarheyevaauthor=J.+L.+Norrisauthor=D.+B.+Kireevauthor=A.+Jadhavauthor=J.+M.+Heroldauthor=W.+P.+Janzenauthor=C.+H.+Arrowsmithauthor=S.+V.+Fryeauthor=P.+J.+Brownauthor=A.+Simeonovauthor=M.+Vedadiauthor=J.+Jin&title=Protein+Lysine+Methyltransferase+G9a+Inhibitors%3A+Design%2C+Synthesis%2C+and+Structure%E2%80%93Activity+Relationships+of+2%2C4-Diamino-7-aminoalkoxy-quinazolines"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fjm100478y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100478y%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DQuinn%26aufirst%3DA.%2BM.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DKireev%26aufirst%3DD.%2BB.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DHerold%26aufirst%3DJ.%2BM.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DProtein%2520Lysine%2520Methyltransferase%2520G9a%2520Inhibitors%253A%2520Design%252C%2520Synthesis%252C%2520and%2520Structure%25E2%2580%2593Activity%2520Relationships%2520of%25202%252C4-Diamino-7-aminoalkoxy-quinazolines%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D5844%26epage%3D5857" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Chang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ganesh, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horton, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spannhoff, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shinkai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Snyder, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, X.</span><span> </span><span class="NLM_article-title">Adding a lysine mimic in the design of potent inhibitors of histone lysine methyltransferases</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">400</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">7</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=400&publication_year=2010&pages=1-7&author=Y.+Changauthor=T.+Ganeshauthor=J.+R.+Hortonauthor=A.+Spannhoffauthor=J.+Liuauthor=A.+Sunauthor=X.+Zhangauthor=M.+T.+Bedfordauthor=Y.+Shinkaiauthor=J.+P.+Snyderauthor=X.+Cheng&title=Adding+a+lysine+mimic+in+the+design+of+potent+inhibitors+of+histone+lysine+methyltransferases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChang%26aufirst%3DY.%26aulast%3DGanesh%26aufirst%3DT.%26aulast%3DHorton%26aufirst%3DJ.%2BR.%26aulast%3DSpannhoff%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DSun%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DShinkai%26aufirst%3DY.%26aulast%3DSnyder%26aufirst%3DJ.%2BP.%26aulast%3DCheng%26aufirst%3DX.%26atitle%3DAdding%2520a%2520lysine%2520mimic%2520in%2520the%2520design%2520of%2520potent%2520inhibitors%2520of%2520histone%2520lysine%2520methyltransferases%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2010%26volume%3D400%26spage%3D1%26epage%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rival-Gervier, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labrie, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DiMaggio, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siarheyeva, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tempel, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mangano, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simpson, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pattenden, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norris, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kireev, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tripathy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edwards, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzen, W. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petronis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellis, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J. A</span><span> </span><span class="NLM_article-title">Chemical Probe Selectively Inhibits G9a and GLP Methyltransferase Activity in Cells</span> <span class="citation_source-journal">Nature Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">566</span><span class="NLM_x">–</span> <span class="NLM_lpage">574</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Fjm200903z&amp;key=10.1038%2Fnchembio.599" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Fjm200903z&amp;key=21743462" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Fjm200903z&amp;key=1%3ACAS%3A528%3ADC%252BC3MXos1OmtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=566-574&author=M.+Vedadiauthor=D.+Barsyte-Lovejoyauthor=F.+Liuauthor=S.+Rival-Gervierauthor=A.+Allali-Hassaniauthor=V.+Labrieauthor=T.+J.+Wigleauthor=P.+A.+DiMaggioauthor=G.+A.+Wasneyauthor=A.+Siarheyevaauthor=A.+Dongauthor=W.+Tempelauthor=S.-C.+Wangauthor=X.+Chenauthor=I.+Chauauthor=T.+Manganoauthor=X.-P.+Huangauthor=C.+D.+Simpsonauthor=S.+G.+Pattendenauthor=J.+L.+Norrisauthor=D.+B.+Kireevauthor=A.+Tripathyauthor=A.+Edwardsauthor=B.+L.+Rothauthor=W.+P.+Janzenauthor=B.+A.+Garciaauthor=A.+Petronisauthor=J.+Ellisauthor=P.+J.+Brownauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=J.+A+Jin&title=Chemical+Probe+Selectively+Inhibits+G9a+and+GLP+Methyltransferase+Activity+in+Cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">A chemical probe selectively inhibits G9a and GLP methyltransferase activity in cells</span></div><div class="casAuthors">Vedadi, Masoud; Barsyte-Lovejoy, Dalia; Liu, Feng; Rival-Gervier, Sylvie; Allali-Hassani, Abdellah; Labrie, Viviane; Wigle, Tim J.; DiMaggio, Peter A.; Wasney, Gregory A.; Siarheyeva, Alena; Dong, Aiping; Tempel, Wolfram; Wang, Sun-Chong; Chen, Xin; Chau, Irene; Mangano, Thomas J.; Huang, Xi-ping; Simpson, Catherine D.; Pattenden, Samantha G.; Norris, Jacqueline L.; Kireev, Dmitri B.; Tripathy, Ashutosh; Edwards, Aled; Roth, Bryan L.; Janzen, William P.; Garcia, Benjamin A.; Petronis, Arturas; Ellis, James; Brown, Peter J.; Frye, Stephen V.; Arrowsmith, Cheryl H.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">566-574</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein lysine methyltransferases G9a and GLP modulate the transcriptional repression of a variety of genes via dimethylation of Lys9 on histone H3 (H3K9me2) as well as dimethylation of non-histone targets.  Here we report the discovery of UNC0638, an inhibitor of G9a and GLP with excellent potency and selectivity over a wide range of epigenetic and non-epigenetic targets.  UNC0638 treatment of a variety of cell lines resulted in lower global H3K9me2 levels, equiv. to levels obsd. for small hairpin RNA knockdown of G9a and GLP with the functional potency of UNC0638 being well sepd. from its toxicity.  UNC0638 markedly reduced the clonogenicity of MCF7 cells, reduced the abundance of H3K9me2 marks at promoters of known G9a-regulated endogenous genes and disproportionately affected several genomic loci encoding microRNAs.  In mouse embryonic stem cells, UNC0638 reactivated G9a-silenced genes and a retroviral reporter gene in a concn.-dependent manner without promoting differentiation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphv-2bZcBqwLVg90H21EOLACvtfcHk0lge6sgxX6VuXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXos1OmtLs%253D&md5=a944e99a910194c1a15240fc6dc8f852</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.599&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.599%26sid%3Dliteratum%253Aachs%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DRival-Gervier%26aufirst%3DS.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DLabrie%26aufirst%3DV.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DDiMaggio%26aufirst%3DP.%2BA.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DTempel%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DS.-C.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DMangano%26aufirst%3DT.%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DSimpson%26aufirst%3DC.%2BD.%26aulast%3DPattenden%26aufirst%3DS.%2BG.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DKireev%26aufirst%3DD.%2BB.%26aulast%3DTripathy%26aufirst%3DA.%26aulast%3DEdwards%26aufirst%3DA.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26aulast%3DGarcia%26aufirst%3DB.%2BA.%26aulast%3DPetronis%26aufirst%3DA.%26aulast%3DEllis%26aufirst%3DJ.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DJin%26aufirst%3DJ.%2BA%26atitle%3DChemical%2520Probe%2520Selectively%2520Inhibits%2520G9a%2520and%2520GLP%2520Methyltransferase%2520Activity%2520in%2520Cells%26jtitle%3DNature%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D566%26epage%3D574" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Tachibana, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ueda, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fukuda, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takeda, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohta, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iwanari, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sakihama, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kodama, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamakubo, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shinkai, Y.</span><span> </span><span class="NLM_article-title">Histone methyltransferases G9a and GLP form heteromeric complexes and are both crucial for methylation of euchromatin at H3-K9</span> <span class="citation_source-journal">Genes Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">815</span><span class="NLM_x">–</span> <span class="NLM_lpage">826</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Fjm200903z&amp;key=10.1101%2Fgad.1284005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Fjm200903z&amp;key=15774718" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Fjm200903z&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjt1OjtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2005&pages=815-826&author=M.+Tachibanaauthor=J.+Uedaauthor=M.+Fukudaauthor=N.+Takedaauthor=T.+Ohtaauthor=H.+Iwanariauthor=T.+Sakihamaauthor=T.+Kodamaauthor=T.+Hamakuboauthor=Y.+Shinkai&title=Histone+methyltransferases+G9a+and+GLP+form+heteromeric+complexes+and+are+both+crucial+for+methylation+of+euchromatin+at+H3-K9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Histone methyltransferases G9a and GLP form heteromeric complexes and are both crucial for methylation of euchromatin at H3-K9</span></div><div class="casAuthors">Tachibana, Makoto; Ueda, Jun; Fukuda, Mikiko; Takeda, Naoki; Ohta, Tsutomu; Iwanari, Hiroko; Sakihama, Toshiko; Kodama, Tatsuhiko; Hamakubo, Takao; Shinkai, Yoichi</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">815-826</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Histone H3 Lys 9 (H3-K9) methylation is a crucial epigenetic mark for transcriptional silencing.  G9a is the major mammalian H3-K9 methyltransferase that targets euchromatic regions and is essential for murine embryogenesis.  There is a single G9a-related methyltransferase in mammals, called GLP/Eu-HMTase1.  Here the authors show that GLP is also important for H3-K9 methylation of mouse euchromatin.  GLP-deficiency led to embryonic lethality, a severe redn. of H3-K9 mono- and dimethylation, the induction of Mage-a gene expression, and HP1 relocalization in embryonic stem cells, all of which were phenotypes of G9a-deficiency.  Furthermore, the authors show that G9a and GLP formed a stoichiometric heteromeric complex in a wide variety of cell types.  Biochem. analyses revealed that formation of the G9a/GLP complex was dependent on their enzymic SET domains.  Taken together, these new findings revealed that G9a and GLP cooperatively exert H3-K9 methyltransferase function in vivo, likely through the formation of higher-order heteromeric complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUXBASXATfh7Vg90H21EOLACvtfcHk0ljOuSWt8Uy8-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjt1OjtLc%253D&md5=cd0b7f8b9f62b4abe81b5ea590ab218c</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1101%2Fgad.1284005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.1284005%26sid%3Dliteratum%253Aachs%26aulast%3DTachibana%26aufirst%3DM.%26aulast%3DUeda%26aufirst%3DJ.%26aulast%3DFukuda%26aufirst%3DM.%26aulast%3DTakeda%26aufirst%3DN.%26aulast%3DOhta%26aufirst%3DT.%26aulast%3DIwanari%26aufirst%3DH.%26aulast%3DSakihama%26aufirst%3DT.%26aulast%3DKodama%26aufirst%3DT.%26aulast%3DHamakubo%26aufirst%3DT.%26aulast%3DShinkai%26aufirst%3DY.%26atitle%3DHistone%2520methyltransferases%2520G9a%2520and%2520GLP%2520form%2520heteromeric%2520complexes%2520and%2520are%2520both%2520crucial%2520for%2520methylation%2520of%2520euchromatin%2520at%2520H3-K9%26jtitle%3DGenes%2520Dev.%26date%3D2005%26volume%3D19%26spage%3D815%26epage%3D826" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Gottlieb, H. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kotlyar, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nudelman, A.</span><span> </span><span class="NLM_article-title">NMR Chemical Shifts of Common Laboratory Solvents as Trace Impurities</span> <span class="citation_source-journal">J. Org. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">62</span><span class="NLM_x">, </span> <span class="NLM_fpage">7512</span><span class="NLM_x">–</span> <span class="NLM_lpage">7515</span></span><div class="citationLinks">[<a href="/doi/10.1021/jo971176v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Fjm200903z&amp;key=1%3ACAS%3A528%3ADyaK2sXmvVClsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=1997&pages=7512-7515&author=H.+E.+Gottliebauthor=V.+Kotlyarauthor=A.+Nudelman&title=NMR+Chemical+Shifts+of+Common+Laboratory+Solvents+as+Trace+Impurities"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">NMR chemical shifts of common laboratory solvents as trace impurities</span></div><div class="casAuthors">Gottlieb, Hugo E.; Kotlyar, Vadim; Nudelman, Abraham</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">7512-7515</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The 1H (300 MHz) and 13C (75 MHz) NMR chem. shifts for >30 common impurities, such as lab. solvents, stabilizers, greases and water, are reported in 7 deuterated NMR solvents.  The temp. dependence of the chem. shift of the proton residual peak (HDO) in D2O is established for the entire liq. range of this sample.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocnoOemocS9LVg90H21EOLACvtfcHk0ljhZftujXxL2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXmvVClsbo%253D&md5=c297adda99f8c10e9f9b0ab9fe5f55be</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fjo971176v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo971176v%26sid%3Dliteratum%253Aachs%26aulast%3DGottlieb%26aufirst%3DH.%2BE.%26aulast%3DKotlyar%26aufirst%3DV.%26aulast%3DNudelman%26aufirst%3DA.%26atitle%3DNMR%2520Chemical%2520Shifts%2520of%2520Common%2520Laboratory%2520Solvents%2520as%2520Trace%2520Impurities%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1997%26volume%3D62%26spage%3D7512%26epage%3D7515" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Nazare, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Will, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matter, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schreuder, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ritter, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Urmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Essrich, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bauer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wagner, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Czech, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lorenz, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laux, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wehner, V.</span><span> </span><span class="NLM_article-title">Probing the subpockets of factor Xa reveals two binding modes for inhibitors based on a 2-carboxyindole scaffold: a study combining structure–activity relationship and X-ray crystallography</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">4511</span><span class="NLM_x">–</span> <span class="NLM_lpage">4525</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm0490540" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=4511-4525&author=M.+Nazareauthor=D.+W.+Willauthor=H.+Matterauthor=H.+Schreuderauthor=K.+Ritterauthor=M.+Urmannauthor=M.+Essrichauthor=A.+Bauerauthor=M.+Wagnerauthor=J.+Czechauthor=M.+Lorenzauthor=V.+Lauxauthor=V.+Wehner&title=Probing+the+subpockets+of+factor+Xa+reveals+two+binding+modes+for+inhibitors+based+on+a+2-carboxyindole+scaffold%3A+a+study+combining+structure%E2%80%93activity+relationship+and+X-ray+crystallography"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fjm0490540&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0490540%26sid%3Dliteratum%253Aachs%26aulast%3DNazare%26aufirst%3DM.%26aulast%3DWill%26aufirst%3DD.%2BW.%26aulast%3DMatter%26aufirst%3DH.%26aulast%3DSchreuder%26aufirst%3DH.%26aulast%3DRitter%26aufirst%3DK.%26aulast%3DUrmann%26aufirst%3DM.%26aulast%3DEssrich%26aufirst%3DM.%26aulast%3DBauer%26aufirst%3DA.%26aulast%3DWagner%26aufirst%3DM.%26aulast%3DCzech%26aufirst%3DJ.%26aulast%3DLorenz%26aufirst%3DM.%26aulast%3DLaux%26aufirst%3DV.%26aulast%3DWehner%26aufirst%3DV.%26atitle%3DProbing%2520the%2520subpockets%2520of%2520factor%2520Xa%2520reveals%2520two%2520binding%2520modes%2520for%2520inhibitors%2520based%2520on%2520a%25202-carboxyindole%2520scaffold%253A%2520a%2520study%2520combining%2520structure%25E2%2580%2593activity%2520relationship%2520and%2520X-ray%2520crystallography%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D4511%26epage%3D4525" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Collazo, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Couture, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bulfer, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trievel, R. C.</span><span> </span><span class="NLM_article-title">A coupled fluorescent assay for histone methyltransferases</span> <span class="citation_source-journal">Anal. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">342</span><span class="NLM_x">, </span> <span class="NLM_fpage">86</span><span class="NLM_x">–</span> <span class="NLM_lpage">92</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Fjm200903z&amp;key=10.1016%2Fj.ab.2005.04.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Fjm200903z&amp;key=15958184" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Fjm200903z&amp;key=1%3ACAS%3A528%3ADC%252BD2MXlt1GktL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=342&publication_year=2005&pages=86-92&author=E.+Collazoauthor=J.+F.+Coutureauthor=S.+Bulferauthor=R.+C.+Trievel&title=A+coupled+fluorescent+assay+for+histone+methyltransferases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">A coupled fluorescent assay for histone methyltransferases</span></div><div class="casAuthors">Collazo, Evys; Couture, Jean-Francois; Bulfer, Stacie; Trievel, Raymond C.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">342</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">86-92</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Histone methyltransferases (HMTs) catalyze the S-adenosylmethionine (AdoMet)-dependent methylation of lysines and arginines in the nucleosomal core histones H3 and H4 and the linker histone H1b.  Methylation of these residues regulates either transcriptional activation or silencing, depending on the residue modified and its degree of methylation.  Despite an intense interest in elucidating the functions of HMTs in transcriptional regulation, these enzymes have remained challenging to quant. assay.  To characterize the substrate specificity of HMTs, we have developed a coupled-fluorescence-based assay for AdoMet-dependent methyltransferases.  This assay utilizes S-adenosylhomocysteine hydrolase (SAHH) to hydrolyze the methyltransfer product S-adenosylhomocysteine (AdoHcy) to homocysteine (Hcy) and adenosine (Ado).  The Hcy concn. is then detd. through conjugation of its free sulfhydryl moiety to a thiol-sensitive fluorophore.  Using this assay, we have detd. the kinetic parameters for the methylation of a synthetic histone H3 peptide (corresponding to residues 1-15 of the native protein) by Schizosaccharomyces pombe CLR4, an H3 Lys-9-specific methyltransferase.  The fluorescent SAHH-coupled assay allows rapid and facile detn. of HMT kinetics and can be adapted to measure the enzymic activity of a wide variety of AdoMet-dependent methyltransferases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfHslBzi9obbVg90H21EOLACvtfcHk0lhvE4I255t88Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXlt1GktL4%253D&md5=2b949dcde263d15559202131cc74da1e</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2005.04.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2005.04.007%26sid%3Dliteratum%253Aachs%26aulast%3DCollazo%26aufirst%3DE.%26aulast%3DCouture%26aufirst%3DJ.%2BF.%26aulast%3DBulfer%26aufirst%3DS.%26aulast%3DTrievel%26aufirst%3DR.%2BC.%26atitle%3DA%2520coupled%2520fluorescent%2520assay%2520for%2520histone%2520methyltransferases%26jtitle%3DAnal.%2520Biochem.%26date%3D2005%26volume%3D342%26spage%3D86%26epage%3D92" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-3"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i45"><a href="/doi/suppl/10.1021/jm200903z">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_83120"></div></div></div></div></div><hr /></hr><p class="last"><sup>1</sup>H and <sup>13</sup>C NMR spectra of compounds <b>6</b> and <b>7</b>. This material is available free of charge via the Internet at <a class="extLink" href="http://pubs.acs.org">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm200903z/suppl_file/jm200903z_si_001.pdf">jm200903z_si_001.pdf (691.26 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Fjm200903z&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Fjm200903z%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2011.54.issue-17" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/jm200903z" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                2MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67998500edd33c8e","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
