
<html lang="en"     class="pb-page"  data-request-id="f62f5560-39dd-4a3a-90f5-b4a8001e9f16"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2018.61.issue-6;wgroup:string:ACHS website Group;website:website:acspubs;article:article:10.1021/acs.jmedchem.7b01884;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Creation of a Novel Class of Potent and Selective MutT Homologue 1 (MTH1) Inhibitors Using Fragment-Based Screening and Structure-Based Drug Design" /></meta><meta name="dc.Creator" content="Fredrik  Rahm" /></meta><meta name="dc.Creator" content="Jenny  Viklund" /></meta><meta name="dc.Creator" content="Lionel  Trésaugues" /></meta><meta name="dc.Creator" content="Manuel  Ellermann" /></meta><meta name="dc.Creator" content="Anja  Giese" /></meta><meta name="dc.Creator" content="Ulrika  Ericsson" /></meta><meta name="dc.Creator" content="Rickard  Forsblom" /></meta><meta name="dc.Creator" content="Tobias  Ginman" /></meta><meta name="dc.Creator" content="Judith  Günther" /></meta><meta name="dc.Creator" content="Kenth  Hallberg" /></meta><meta name="dc.Creator" content="Johan  Lindström" /></meta><meta name="dc.Creator" content="Lars Boukharta  Persson" /></meta><meta name="dc.Creator" content="Camilla  Silvander" /></meta><meta name="dc.Creator" content="Antoine  Talagas" /></meta><meta name="dc.Creator" content="Laura  Díaz-Sáez" /></meta><meta name="dc.Creator" content="Oleg  Fedorov" /></meta><meta name="dc.Creator" content="Kilian V. M.  Huber" /></meta><meta name="dc.Creator" content="Ioanna  Panagakou" /></meta><meta name="dc.Creator" content="Paulina  Siejka" /></meta><meta name="dc.Creator" content="Mátyás  Gorjánácz" /></meta><meta name="dc.Creator" content="Marcus  Bauser" /></meta><meta name="dc.Creator" content="Martin  Andersson" /></meta><meta name="dc.Description" content="Recent literature has both suggested and questioned MTH1 as a novel cancer target. BAY-707 was just published as a target validation small molecule probe for assessing the effects of pharmacologica..." /></meta><meta name="Description" content="Recent literature has both suggested and questioned MTH1 as a novel cancer target. BAY-707 was just published as a target validation small molecule probe for assessing the effects of pharmacologica..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="February 27, 2018" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.7b01884" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2018 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b01884" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.7b01884" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b01884" /></link>
        
    
    

<title>Creation of a Novel Class of Potent and Selective MutT Homologue 1 (MTH1) Inhibitors Using Fragment-Based Screening and Structure-Based Drug Design | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.7b01884" /></meta><meta property="og:title" content="Creation of a Novel Class of Potent and Selective MutT Homologue 1 (MTH1) Inhibitors Using Fragment-Based Screening and Structure-Based Drug Design" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/large/jm-2017-01884d_0012.jpeg" /></meta><meta property="og:description" content="Recent literature has both suggested and questioned MTH1 as a novel cancer target. BAY-707 was just published as a target validation small molecule probe for assessing the effects of pharmacological inhibition of MTH1 on tumor cell survival, both in vitro and in vivo.(1) In this report, we describe the medicinal chemistry program creating BAY-707, where fragment-based methods were used to develop a series of highly potent and selective MTH1 inhibitors. Using structure-based drug design and rational medicinal chemistry approaches, the potency was increased over 10,000 times from the fragment starting point while maintaining high ligand efficiency and drug-like properties." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.7b01884"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.7b01884">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.7b01884&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.7b01884&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.7b01884&amp;href=/doi/10.1021/acs.jmedchem.7b01884" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2018</span><span class="cit-fg-volume">, 61</span><span class="cit-fg-issue">, 6</span><span class="cit-fg-pageRange">, 2533-2551</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/61/6" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.7b01854" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.8b00007" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Creation of a Novel Class of Potent and Selective MutT Homologue 1 (MTH1) Inhibitors Using Fragment-Based Screening and Structure-Based Drug Design</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Fredrik Rahm</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Fredrik Rahm</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Sprint Bioscience AB, Novum, 14157 Huddinge, Sweden</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Fredrik++Rahm">Fredrik Rahm</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jenny Viklund</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jenny Viklund</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Sprint Bioscience AB, Novum, 14157 Huddinge, Sweden</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#caa0afa4a4b3e4bca3a1a6bfa4ae8ab9bab8a3a4bea8a3a5b9a9a3afa4a9afe4a9a5a7"><span class="__cf_email__" data-cfemail="f298979c9c8bdc849b999e879c96b28182809b9c86909b9d81919b979c9197dc919d9f">[email protected]</span></a> or <a href="/cdn-cgi/l/email-protection#ef858a818196c1998684839a818baf88828e8683c18c8082"><span class="__cf_email__" data-cfemail="c9a3aca7a7b0e7bfa0a2a5bca7ad89aea4a8a0a5e7aaa6a4">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jenny++Viklund">Jenny Viklund</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4087-9721" title="Orcid link">http://orcid.org/0000-0002-4087-9721</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lionel Trésaugues</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lionel Trésaugues</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Sprint Bioscience AB, Novum, 14157 Huddinge, Sweden</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lionel++Tr%C3%A9saugues">Lionel Trésaugues</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Manuel Ellermann</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Manuel Ellermann</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Bayer AG, Muellerstrasse 178, 13353 Berlin, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Manuel++Ellermann">Manuel Ellermann</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Anja Giese</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Anja Giese</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Bayer AG, Muellerstrasse 178, 13353 Berlin, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Anja++Giese">Anja Giese</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ulrika Ericsson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ulrika Ericsson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Sprint Bioscience AB, Novum, 14157 Huddinge, Sweden</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ulrika++Ericsson">Ulrika Ericsson</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Rickard Forsblom</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rickard Forsblom</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Sprint Bioscience AB, Novum, 14157 Huddinge, Sweden</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rickard++Forsblom">Rickard Forsblom</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Tobias Ginman</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Tobias Ginman</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Sprint Bioscience AB, Novum, 14157 Huddinge, Sweden</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Tobias++Ginman">Tobias Ginman</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Judith Günther</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Judith Günther</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Bayer AG, Muellerstrasse 178, 13353 Berlin, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Judith++G%C3%BCnther">Judith Günther</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kenth Hallberg</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kenth Hallberg</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Sprint Bioscience AB, Novum, 14157 Huddinge, Sweden</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kenth++Hallberg">Kenth Hallberg</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Johan Lindström</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Johan Lindström</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Sprint Bioscience AB, Novum, 14157 Huddinge, Sweden</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Johan++Lindstr%C3%B6m">Johan Lindström</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lars Boukharta Persson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lars Boukharta Persson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Sprint Bioscience AB, Novum, 14157 Huddinge, Sweden</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lars+Boukharta++Persson">Lars Boukharta Persson</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Camilla Silvander</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Camilla Silvander</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Sprint Bioscience AB, Novum, 14157 Huddinge, Sweden</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Camilla++Silvander">Camilla Silvander</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Antoine Talagas</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Antoine Talagas</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Sprint Bioscience AB, Novum, 14157 Huddinge, Sweden</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Antoine++Talagas">Antoine Talagas</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Laura Díaz-Sáez</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Laura Díaz-Sáez</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, Nuffield Department of Medicine, University of Oxford, Oxford OX3 7DQ, United Kingdom</div><div class="loa-info-affiliations-info">Target Discovery Institute, Nuffield Department of Medicine, University of Oxford, Oxford OX3 7FZ, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Laura++D%C3%ADaz-S%C3%A1ez">Laura Díaz-Sáez</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Oleg Fedorov</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Oleg Fedorov</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, Nuffield Department of Medicine, University of Oxford, Oxford OX3 7DQ, United Kingdom</div><div class="loa-info-affiliations-info">Target Discovery Institute, Nuffield Department of Medicine, University of Oxford, Oxford OX3 7FZ, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Oleg++Fedorov">Oleg Fedorov</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kilian V. M. Huber</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kilian V. M. Huber</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, Nuffield Department of Medicine, University of Oxford, Oxford OX3 7DQ, United Kingdom</div><div class="loa-info-affiliations-info">Target Discovery Institute, Nuffield Department of Medicine, University of Oxford, Oxford OX3 7FZ, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kilian+V.+M.++Huber">Kilian V. M. Huber</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1103-5300" title="Orcid link">http://orcid.org/0000-0002-1103-5300</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ioanna Panagakou</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ioanna Panagakou</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, Nuffield Department of Medicine, University of Oxford, Oxford OX3 7DQ, United Kingdom</div><div class="loa-info-affiliations-info">Target Discovery Institute, Nuffield Department of Medicine, University of Oxford, Oxford OX3 7FZ, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ioanna++Panagakou">Ioanna Panagakou</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Paulina Siejka</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Paulina Siejka</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, Nuffield Department of Medicine, University of Oxford, Oxford OX3 7DQ, United Kingdom</div><div class="loa-info-affiliations-info">Target Discovery Institute, Nuffield Department of Medicine, University of Oxford, Oxford OX3 7FZ, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Paulina++Siejka">Paulina Siejka</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mátyás Gorjánácz</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mátyás Gorjánácz</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Bayer AG, Muellerstrasse 178, 13353 Berlin, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=M%C3%A1ty%C3%A1s++Gorj%C3%A1n%C3%A1cz">Mátyás Gorjánácz</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8398-410X" title="Orcid link">http://orcid.org/0000-0002-8398-410X</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Marcus Bauser</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Marcus Bauser</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Bayer AG, Muellerstrasse 178, 13353 Berlin, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Marcus++Bauser">Marcus Bauser</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Martin Andersson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Martin Andersson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Sprint Bioscience AB, Novum, 14157 Huddinge, Sweden</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Martin++Andersson">Martin Andersson</a></span></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b01884&amp;href=/doi/10.1021%2Facs.jmedchem.7b01884" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2018</span></span><span class="cit-volume">, 61</span><span class="cit-issue">, 6</span><span class="cit-pageRange">, 2533–2551</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">February 27, 2018</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>21 December 2017</li><li><span class="item_label"><b>Published</b> online</span>27 February 2018</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 22 March 2018</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.7b01884" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b01884</a></div><div class="article_header-article-copyright"><strong>Copyright © 2018 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D2533%26pageCount%3D19%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DFredrik%2BRahm%252C%2BJenny%2BViklund%252C%2BLionel%2BTr%25C3%25A9saugues%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D61%26issueNum%3D6%26contentID%3Dacs.jmedchem.7b01884%26title%3DCreation%2Bof%2Ba%2BNovel%2BClass%2Bof%2BPotent%2Band%2BSelective%2BMutT%2BHomologue%2B1%2B%2528MTH1%2529%2BInhibitors%2BUsing%2BFragment-Based%2BScreening%2Band%2BStructure-Based%2BDrug%2BDesign%26numPages%3D19%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D2551%26publicationDate%3DMarch%2B2018">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.7b01884"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2666</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">13</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.7b01884" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Creation of a Novel Class of Potent and Selective MutT Homologue 1 (MTH1) Inhibitors Using Fragment-Based Screening and Structure-Based Drug Design&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Fredrik&quot;,&quot;last_name&quot;:&quot;Rahm&quot;},{&quot;first_name&quot;:&quot;Jenny&quot;,&quot;last_name&quot;:&quot;Viklund&quot;},{&quot;first_name&quot;:&quot;Lionel&quot;,&quot;last_name&quot;:&quot;Trésaugues&quot;},{&quot;first_name&quot;:&quot;Manuel&quot;,&quot;last_name&quot;:&quot;Ellermann&quot;},{&quot;first_name&quot;:&quot;Anja&quot;,&quot;last_name&quot;:&quot;Giese&quot;},{&quot;first_name&quot;:&quot;Ulrika&quot;,&quot;last_name&quot;:&quot;Ericsson&quot;},{&quot;first_name&quot;:&quot;Rickard&quot;,&quot;last_name&quot;:&quot;Forsblom&quot;},{&quot;first_name&quot;:&quot;Tobias&quot;,&quot;last_name&quot;:&quot;Ginman&quot;},{&quot;first_name&quot;:&quot;Judith&quot;,&quot;last_name&quot;:&quot;Günther&quot;},{&quot;first_name&quot;:&quot;Kenth&quot;,&quot;last_name&quot;:&quot;Hallberg&quot;},{&quot;first_name&quot;:&quot;Johan&quot;,&quot;last_name&quot;:&quot;Lindström&quot;},{&quot;first_name&quot;:&quot;Lars&quot;,&quot;last_name&quot;:&quot;Boukharta Persson&quot;},{&quot;first_name&quot;:&quot;Camilla&quot;,&quot;last_name&quot;:&quot;Silvander&quot;},{&quot;first_name&quot;:&quot;Antoine&quot;,&quot;last_name&quot;:&quot;Talagas&quot;},{&quot;first_name&quot;:&quot;Laura&quot;,&quot;last_name&quot;:&quot;Díaz-Sáez&quot;},{&quot;first_name&quot;:&quot;Oleg&quot;,&quot;last_name&quot;:&quot;Fedorov&quot;},{&quot;first_name&quot;:&quot;Kilian&quot;,&quot;last_name&quot;:&quot;V. M. Huber&quot;},{&quot;first_name&quot;:&quot;Ioanna&quot;,&quot;last_name&quot;:&quot;Panagakou&quot;},{&quot;first_name&quot;:&quot;Paulina&quot;,&quot;last_name&quot;:&quot;Siejka&quot;},{&quot;first_name&quot;:&quot;Mátyás&quot;,&quot;last_name&quot;:&quot;Gorjánácz&quot;},{&quot;first_name&quot;:&quot;Marcus&quot;,&quot;last_name&quot;:&quot;Bauser&quot;},{&quot;first_name&quot;:&quot;Martin&quot;,&quot;last_name&quot;:&quot;Andersson&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2018&quot;,&quot;month&quot;:&quot;02&quot;,&quot;day&quot;:&quot;27&quot;,&quot;issue&quot;:&quot;6&quot;,&quot;volume&quot;:&quot;61&quot;,&quot;pages&quot;:&quot;2533-2551&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.7b01884&quot;},&quot;abstract&quot;:&quot;Recent literature has both suggested and questioned MTH1 as a novel cancer target. BAY-707 was just published as a target validation small molecule probe for assessing the effects of pharmacological inhibition of MTH1 on tumor cell survival, both in vitro and in vivo.(1) In this report, we describe the medicinal chemistry program creating BAY-707, where fragment-based methods were used to develop a series of highly potent and selective MTH1 inhibitors. Using structure-based drug design and rational medicinal chemistry approaches, the potency was increased over 10,000 times from the fragment starting point while maintaining high ligand efficiency and drug-like properties.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b01884&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01884" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b01884&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01884" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b01884&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01884" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b01884&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01884&amp;href=/doi/10.1021/acs.jmedchem.7b01884" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.7b01884" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.7b01884" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b01884&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.7b01884%26sid%3Dliteratum%253Aachs%26pmid%3D29485874%26genre%3Darticle%26aulast%3DRahm%26date%3D2018%26atitle%3DCreation%2Bof%2Ba%2BNovel%2BClass%2Bof%2BPotent%2Band%2BSelective%2BMutT%2BHomologue%2B1%2B%2528MTH1%2529%2BInhibitors%2BUsing%2BFragment-Based%2BScreening%2Band%2BStructure-Based%2BDrug%2BDesign%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D61%26issue%3D6%26spage%3D2533%26epage%3D2551%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (5)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=290658" title="Amides">Amides</a>,</li><li><a href="/action/doSearch?ConceptID=291880" title="Crystal structure">Crystal structure</a>,</li><li><a href="/action/doSearch?ConceptID=290676" title="Substituents">Substituents</a>,</li><li><a href="/action/doSearch?ConceptID=290896" title="Cells">Cells</a>,</li><li><a href="/action/doSearch?ConceptID=292504" title="Permeability">Permeability</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/61/6" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/jmcmar.2018.61.issue-6/20180322/jmcmar.2018.61.issue-6.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/medium/jm-2017-01884d_0012.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/large/jm-2017-01884d_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01884&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Recent literature has both suggested and questioned MTH1 as a novel cancer target. BAY-707 was just published as a target validation small molecule probe for assessing the effects of pharmacological inhibition of MTH1 on tumor cell survival, both <i>in vitro</i> and <i>in vivo.</i><a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> In this report, we describe the medicinal chemistry program creating BAY-707, where fragment-based methods were used to develop a series of highly potent and selective MTH1 inhibitors. Using structure-based drug design and rational medicinal chemistry approaches, the potency was increased over 10,000 times from the fragment starting point while maintaining high ligand efficiency and drug-like properties.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_56405" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_56405" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">One prominent feature of cancer is the increase in reactive oxygen species (ROS) in cancer cells as compared to normal cells.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> ROS are generated during cell metabolism and are highly reactive toward most biological macromolecules such as nucleic acids and proteins.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> Hence, cancer cells have a higher dependency than normal cells on various ROS-related defense mechanisms, and consequently, such pathways have been suggested as potential targets for cancer treatments.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a></div><div class="NLM_p">The nucleotide pool is also subject to oxidative damage by ROS, with the main oxidation products being 8-oxo-7,8-dihydro-2′-deoxyguanosine (8-oxo-dG) and its corresponding nucleotides.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Oxidized nucleotide triphosphates (such as 8-oxo-dGTP) are mutagenic and cytotoxic when incorporated into DNA.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> To clear the nucleotide pool of such damaged nucleotides, cells are equipped with enzymes, such as members of the NUDIX family (NUcleoside DIphosphate linked to another moiety X).<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> Recent studies have suggested a dependency of cancer cells to a particular enzyme of this family, Nudix hydrolase 1 (NUDT1), also called MutT Homolog1 (MTH1), which is upregulated in several cancer forms.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> MTH1 catalyzes the hydrolysis of 8-oxo-dGTP and scarcer oxidized nucleotide triphosphates to their monophosphate counterparts, thus preventing their incorporation into DNA. The 8-oxo-dGMP product is further hydrolyzed to 8-oxo-dG and excreted from the cell.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a></div><div class="NLM_p">The first discoveries relating MTH1 to a role in cancer biology was published in 2011, in a study that identified MTH1 as one of the genes induced by RAS overexpression.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Given its role in nucleotide pool sanitization and ROS stress defense, further studies by the same authors showed that MTH1 knockdown inhibits RAS-driven tumor cell growth <i>in vitro</i> and <i>in vivo.</i><a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> More data was published during 2014 lending further support to the hypothesis that MTH1 has a role in prevention of DNA damage and cancer cell survival. It was shown that a phenotypic screening hit, SCH51344, that inhibits the nonadherent growth of cancer cells is an MTH1 inhibitor.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> (<i>S</i>)-Crizotinib was also identified in this screen as a more selective MTH1 inhibitor, and it was shown that these compounds induce DNA damage and inhibit tumor cell growth both <i>in vitro</i> and <i>in vivo</i>. In a third study, pharmacological target validation also looked promising, both <i>in vitro</i> and <i>in vivo</i>, using the novel MTH1 inhibitor TH588.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a></div><div class="NLM_p">The first crystal structure of MTH1 became available in the Protein Data Bank (PDB) in 2011.<a onclick="showRef(event, 'ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref13 ref14">(13,14)</a> The global interest in this potential cancer target is revealed by the large number of structures of complexes that have been released since then. More than 20 complex structures with either nucleotide analogues or small molecule inhibitors have been published: the first one being with 8-oxo-dGMP, the product formed by hydrolysis of the main substrate, 8-oxo-dGTP.<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15,16)</a> These ligands are all competitive binders,<a onclick="showRef(event, 'ref8 ref12 ref17'); return false;" href="javascript:void(0);" class="ref ref8 ref12 ref17">(8,12,17)</a> and they share common binding motifs with the purine ring of the substrate, including one or more hydrogen-bonds to the “key Asp119-Asp120 recognition element”.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Even though the ligand interactions to these aspartates appear to be crucial, the number and positions of the atoms involved in hydrogen-bonding can vary significantly (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/medium/jm-2017-01884d_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/large/jm-2017-01884d_0001.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Representations of the key interactions of selected inhibitors of MTH1 (the corresponding PDB ID is shown in parentheses), with varying hydrogen bonding patterns to the Asp119-Asp120 recognition element (dotted lines): 8-oxo-dGMP (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ZR0">3ZR0</a>),<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> (<i>S</i>)-Crizotinib (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4C9X">4C9X</a>),<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> TH588 (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4N1U">4N1U</a>),<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> AZ compound 19 (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ANV">5ANV</a>).<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/large/jm-2017-01884d_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01884&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In 2016, reports started to appear that questioned the role of MTH1 in cancer cell survival.<a onclick="showRef(event, 'ref17 ref19'); return false;" href="javascript:void(0);" class="ref ref17 ref19">(17,19)</a> In one study, proteomic profiling was used to conclude that the proteome-wide fingerprints of cells exposed to some of the pharmacological tools described in the 2014 studies did not resemble the proteomic fingerprints of other DNA-damage-inducing agents or other MTH1 inhibitors, indicating that the observed effects of these compounds might be due to off-target effects.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> These opposing results compared to the previous reports from 2014 emphasize the importance of using high quality, orthogonal probes for target validation.<a onclick="showRef(event, 'ref21 ref22 ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref21 ref22 ref23 ref24">(21−24)</a></div><div class="NLM_p last">Since none of the above publications from 2016 reported results from <i>in vivo</i> studies, the hypothesis that MTH1 inhibition affects tumor growth <i>in vivo</i> could not be ruled out. To test this suggestion, a novel MTH1 inhibitor (BAY-707) was recently used to evaluate the effects of pharmacological target inhibition in a broad array of model systems, both <i>in vitro</i> and <i>in vivo</i>, confirming the devalidation results from 2016.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> This report describes the medicinal chemistry program that created BAY-707, using fragment-based drug discovery methods.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Chemistry</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_77736" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_77736" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Compounds described in this report are based on 7-azaindole, 7-azaindazole, or 7-azabenzimidazole core structures, substituted in the 4-position and, in the first, optionally in the 2-position. The compounds were made by following a few general synthetic routes, as illustrated by Method A and Method B outlined in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a> below. Depending on the nature of the substituent in the 4-position, the Suzuki–Miyaura<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> cross-coupling reaction, the Buchwald–Hartwig<a onclick="showRef(event, 'ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref26 ref27">(26,27)</a> cross-coupling reaction, or direct nucleophilic aromatic substitution was used to introduce this substituent. To enable further functionalization in the 2-position of the core structure, suitable handles were introduced by standard deprotonation procedures followed by reaction with an appropriate electrophile. Some compounds were synthesized by combining one of these routes with additional standard functional group transformations. Full details regarding the synthesis of all compounds can be found in the <a class="ref internalNav" href="#sec5" aria-label="Experimental Section">Experimental Section</a>.</div><figure id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/medium/jm-2017-01884d_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/large/jm-2017-01884d_0008.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 1. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/large/jm-2017-01884d_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01884&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Representative reagents and conditions: (i) LDA, EtOCOCl, 2-MeTHF, −78 °C to rt; (ii) R1 = aryl: boronic acid reagent, Pd(amphos)Cl<sub>2</sub>, K<sub>2</sub>CO<sub>3</sub>, MeCN·H<sub>2</sub>O, 120 °C; (iii) R1 = −NR2R3: cyclic secondary amine, NMP, 160 °C; (iv) LiOH·H<sub>2</sub>O, THF:MeOH:H<sub>2</sub>O, 50 °C; (v) (a) SOCl<sub>2</sub>, 80 °C, (b) R4R5NH, THF, 0 °C to rt; (vi) LDA, 2-MeTHF, CO<sub>2</sub>, −78 °C to rt; (vii) (a) SOCl<sub>2</sub>, 80 °C, (b) EtNH<sub>2</sub>, THF, 0 °C to rt.</p></p></figure></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_10533" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_10533" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Primary screening was performed using a fluorescence-based thermal shift assay (TSA) to screen a library of 723 fragments at a compound concentration of 1 mM. The TSA monitors ligand effects on temperature-dependent protein unfolding, where ligand binding would lead to a positive <i>T</i><sub>m</sub> shift, as compared to the <i>T</i><sub>m</sub> of the protein alone (control). Hits were determined based on a change in <i>T</i><sub>m</sub> of MTH1 equal to or greater than three standard deviations of the mean of the <i>T</i><sub>m</sub> for the control. Consequently, fragments with a Δ<i>T</i><sub>m</sub> value of more than +1.0 °C in both duplicates were defined as hits. Applying this threshold, 166 fragments were identified that significantly increased the thermal melting point of MTH1, with Δ<i>T</i><sub>m</sub> values ranging from 1.0 to 11.8 °C. Such high hit-rate (23%) is usually indicative of a druggable active site.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a></div><div class="NLM_p">Dose–response experiments were performed to verify the stabilization of MTH1 for the most promising hits from the single-point screening. The fragment concentrations were varied from 62.5 to 1000 μM, and it was determined that the shift in Δ<i>T</i><sub>m</sub> was dose-dependent for 48 of 49 tested initial hit fragments. One of the screening hits was a 7-azaindole compound, fragment hit <b>1</b>, with a single point Δ<i>T</i><sub>m</sub> of 3.9 °C at 1 mM (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). One-to-one binding was verified with ITC, with a <i>K</i><sub>d</sub> of 49.5 μM (<a class="ref internalNav" href="#notes-1" aria-label="Figure S1">Figure S1</a>). The azaindole was neither the most potent nor the most ligand efficient fragment among the verified hits. Still, it was selected to be explored considering its tractability from several other important perspectives: the azaindole core was providing synthetically reasonable attachment points to grow the fragment into interesting parts of the protein. Also, we were intentionally avoiding mainly lipophilic fragments containing a basic amine.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> Lastly, all fragments containing the same aspartate-binding amino-pyrimidine and amino-pyridine cores as the reference compounds TH588 and S-Crizotinib were deselected, to ensure the creation of a novel and orthogonal series (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).</div><figure id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/medium/jm-2017-01884d_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/large/jm-2017-01884d_0002.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (a) Two-dimensional structure of fragment hit <b>1</b>. Vectors from the 2- and 4-positions illustrate the attachment points for synthetic expansions presented in this work. (b,c) Overlay of complex between MTH1 and <b>1</b> (green, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6F20">6F20</a>) and complex between MTH1 and 8-oxo-dGMP (cyan, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ZR0">3ZR0</a>). Side-chains of Phe72, Trp117, Asp119, and Asp120 in the structure of MTH1–<b>1</b> complex are shown as sticks with their carbon atoms colored green. Water molecules present in the inhibitor binding site of MTH1–<b>1</b> complex are shown as red spheres. The 8-oxo-dGMP is displayed as thin sticks, with its carbon atoms colored cyan. The ester motif occupies the same area as the ribose of 8-oxo-dGMP, and we refer to this area as the ribose pocket. Clipped molecular surface of MTH1 is shown in the MTH1–<b>1</b> complex structure. Panel (c) is rotated 90 deg as compared to panel (b).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/large/jm-2017-01884d_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01884&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The fragment hit <b>1</b> was cocrystallized with human MTH1 protein and the 7-azaindole core was shown to make hydrogen bonds to both residues in the key recognition element mentioned above: the acceptor is interacting with Asp119 and the donor is interacting with Asp120 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>a). The 7-azaindole core is sandwiched through pi-stacking interactions between aromatic moieties of both Phe72 and Trp117 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>b,c). The ester motif of <b>1</b> occupies the same area as the ribose part of the product, indicating the 4-position of the 7-azaindole ring system as an interesting starting point for growing the fragment hit into a larger molecule, aiming to increase binding affinity. Analysis of the crystal structure of <b>1</b> in complex with MTH1 revealed the existence of a minor pocket nearby Asp120, which we hypothesized could be targeted from the 2-position of the 7-azaindole core structure (pocket 1, <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>b).</div><div class="NLM_p">With the crystal structure of <b>1</b> in hand, our first approach for improving the fragment was to investigate close analogues in our in-house compound collection. Based on the crystal structure, docking studies were performed, and two compounds (<b>2</b> and <b>3</b>, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) both with a substituent in the 4-position of the 7-azaindole core were predicted to maintain the binding mode of <b>1</b>. TSA data showed an increase in melting temperature for both compounds, suggesting an increased binding affinity as compared to compound <b>1</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Encouraged by these data, compound <b>4</b>, with a more lipophilic group in the 4-position, was synthesized and tested together with compounds <b>1</b>, <b>2</b>, and <b>3</b> in a biochemical MTH1 inhibition assay. All compounds showed an increased potency as compared to <b>1</b> with at most a five-fold increase in potency for <b>4</b>, while maintaining a high ligand efficiency (LE)<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>).</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Biophysical, Biochemical, and Physicochemical Data of Fragment Hit <b>1</b> and Close Analogues</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/medium/jm-2017-01884d_0009.gif" alt="" id="fx1" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">n.d., not determined.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">cLogP was calculated using the ChemAxon Partitioning Plugin v16.10.24.</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">Solubility was estimated by visual inspection after dilution of the DMSO stock solution in buffer. The compounds were diluted at least 1/100 to give a final DMSO concentration not higher than 1%.</p></div></div><div></div></div><div class="NLM_p">Next, we set out to explore whether pocket 1 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>b) could fit a substituent in the 2-position of the 7-azaindole core. By visual inspection of the pocket, we reasoned that compound <b>5</b> was the only representative from our in-house collection with a substituent (amide) that was small enough to have a chance to fit into this minor pocket. It was encouraging to see that the amide in the 2-position increased biochemical potency 20 times as compared to the matched pair compound <b>3</b>. In addition, compound <b>5</b> maintained a high LE, high solubility, and even lower lipophilicity (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>), indicating that exploring the 2-position would be an interesting option for optimization.</div><div class="NLM_p">Encouraged by the large potency gain from the amide of compound <b>5</b>, we synthesized a small set of five additional amide-containing compounds to further investigate the 2-position of the 7-azaindole core (<b>6</b>–<b>10</b>, <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). To generate matched-pair compounds with a balanced lipophilicity,<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a><i>m</i>-chlorophenyl (as used in <b>4</b>) was chosen as substituent in the 4-position.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Biochemical Potency and Physicochemical Properties for Variations in 2-Position</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/medium/jm-2017-01884d_0010.gif" alt="" id="fx2" /></img><div></div></div><div class="NLM_p">The crystal structure of the ethyl amide analogue <b>7</b> was solved in complex with MTH1, to understand the potency increase owing to the amide substituent in the 2-position. While the 7-azaindole core remains in the same position as seen for the fragment hit <b>1</b>, the amide makes two additional hydrogen bonds to the protein; the donor is interacting with Asp120, and the carbonyl acceptor is interacting with the backbone nitrogen of Gly34. In contrast to the crystal structure of <b>1</b>, Asn33 has moved into a different conformation and is now replaced by the amide carbonyl (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). A similar interaction pattern, with a reversed amide substituent, was later published by others (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>, AZ compound 19).<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a></div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/medium/jm-2017-01884d_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/large/jm-2017-01884d_0003.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Crystal structure of <b>7</b>, showing that the 7-azaindole core remains in the same position as the fragment hit <b>1</b>. MTH1–<b>1</b> and MTH1–<b>7</b> complexes are colored green and violet, respectively. Residues interacting with the compounds, and compound themselves, are shown as thin (complex with <b>1</b>) or thick (complex with <b>7</b>) sticks with their carbon atoms colored following the same code as the secondary structure elements. Water molecules present in the binding-site of MTH1–<b>7</b> complex are shown as red spheres. Hydrogen bonds are displayed as orange dashes. Clipped molecular surface of MTH1 is shown in the MTH1–<b>7</b> complex structure (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6F1X">6F1X</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/large/jm-2017-01884d_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01884&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">From a computational chemistry perspective, it is interesting to note that neither the binding mode of <b>7</b>, nor any of its analogues, could be predicted correctly via docking studies, using any of the crystal structures available at that time (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4C9X">4C9X</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4C9W">4C9W</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ZR0">3ZR0</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ZR1">3ZR1</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3Q93">3Q93</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4N1T">4N1T</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4N1U">4N1U</a>, compound <b>1</b>). In all but one of these structures the carbonyl of the flexible residue Asn33 is occupying the same position as the carbonyl of the amide of <b>7</b>. The only exception is <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4CX9">4CX9</a>, where Asn33 occupies a similar position as in the MTH1–<b>7</b> complex. However, even when modifying the protonation states of the <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4C9X">4C9X</a> aspartates to accommodate the hydrogen bonding pattern of <b>7</b>, the binding mode of the amide could still not be predicted correctly. We believe this is an interesting example of how computational selection methods of new compounds should be complemented by visual inspection of the crystal structures, to take advantage of the intuition of experienced medicinal chemists when doing structure-based design. In retrospect, however, we did try induced fit docking, allowing for the active site residues to be flexible, and with this more time-consuming protocol, the correct amide binding mode could indeed be predicted correctly.</div><div class="NLM_p">In combination with the crystal structure of <b>7</b>, the small set of five amide-containing compounds generated distinct knowledge for the structure–activity relationships (SAR) in 2-position: the importance of the hydrogen bond interaction with Asp120 is clearly seen in the severe drop in potency for the tertiary amide of <b>8</b>, lacking the donor. The equally severe drop in potency seen for the primary amide of <b>9</b> shows the importance of filling the remaining part of pocket 1 with a small alkyl substituent, which is then surrounded by the hydrophobic residues Leu11, Leu20, Met116, and Phe124. Thus, only secondary amides (<b>6</b>, <b>7</b>, and <b>10</b>) were well tolerated and resulted in compounds with low nanomolar potencies and maintained LE (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>).</div><div class="NLM_p">To further explore pocket 1, and particularly to investigate whether more diverse types of substituents other than amides were allowed in the 2-position, four additional matched-paired compounds were synthesized (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Attempts to gain potency by either simply filling the pocket with a methyl group (<b>11</b>) or by attempting to make polar interactions with the protein, as predicted by docking studies, were unsuccessful. Both compounds <b>12</b> and <b>13</b> contain other types of acceptors, aiming to mimic the amide carbonyl hydrogen bond to the backbone of Gly34. However, both the nitrile and the oxazole rather reduced potency, as compared to the matched pair compound <b>4</b>, without a substituent in the 2-position. Compound <b>14</b>, with a methylene hydroxyl substituent, was predicted by the docking studies to be able to either interact as an acceptor to Gly34 or as a donor to Asp120. However, despite both of these hydrogen-bonding possibilities, <b>14</b> was merely equally potent as the matched pair compound <b>4</b>. To summarize the optimization efforts in the 2-position, the secondary amide analogues <b>6</b>, <b>7</b>, and <b>10</b> were considered the most promising compounds so far, with low nanomolar biochemical potency and high solubility.</div><div class="NLM_p">Since our main objective was the development of both potent and highly selective MTH1 inhibitors, we wanted to assess the kinome selectivity profile already at this early stage of the program. We were especially concerned by the presence of the 7-azaindole core in our series since this substructure is commonly seen as a hinge binding motif in kinase inhibitors.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> Consequently, we assessed the kinome selectivity profile of <b>10</b>, as a representative structure of the azaindole series. Aiming for a selectivity window of at least 1000 times, compound <b>10</b> was tested in single point at 1 μM in a 97-member kinase panel [DiscoverX’s scanEDGE Kinase Assay Panel]. It was encouraging to see that the compound did not hit any of the included kinases at significant levels [S-score = 0, <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>].</div><div class="NLM_p">After the 2-position had been explored, we decided to continue the optimization work using the substituent in the 4-position, aiming to explore the ribose pocket further (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). A sitemap analysis using the crystal structure of MTH1–<b>7</b> proposed that this area would favor quite bulky, lipophilic substituents, possibly in combination with a donor to interact with any of the backbone carbonyls of Thr8, Phe27, or Gly34 (Figure S2, <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>). Consequently, we aimed to introduce larger lipophilic substituents with high 3D character, still retaining rigidity, in hopes of improving potency by filling the ribose pocket more optimally. In addition, increasing the fraction of sp<sup>3</sup> carbons is a general strategy to improve vital drug properties, such as metabolic stability, solubility, and selectivity.<a onclick="showRef(event, 'ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref33 ref34">(33,34)</a></div><div class="NLM_p">A new set of compounds (<b>15</b>–<b>19</b>) with cyclic amines in the 4-position were selected for synthesis based on docking studies and availability of starting materials. We were intentionally avoiding additional donors to maintain high permeability.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Compound <b>16</b>, with a 2-phenylpyrrolidine in the 4-position, resulted in the highest increase of potency and LE (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). The structure of the complex between <b>16</b> and MTH1 showed a maintained binding mode of the 7-azaindole core and the bulky phenyl tightly occupying the pocket between the lipophilic residues Tyr7, Leu9, and Met81 (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). The most potent compounds <b>15</b> and <b>16</b>, together with the previous representative compound <b>10</b>, were assessed for primary ADME properties. In general, solubility was high, Caco-2 permeability was promising, whereas mouse microsomal stability was low (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>).</div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/medium/jm-2017-01884d_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/large/jm-2017-01884d_0004.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Overlay between the structure of MTH1–<b>16</b> (white) and MTH1–<b>1</b> (green). Compounds <b>16</b> and <b>1</b> are displayed as thick and thin sticks, respectively, with their carbon atoms colored following the same code as the secondary structure elements. The side chains of MTH1 residues forming the cavity surrounding the phenyl moiety of <b>16</b> are displayed as sticks. Clipped molecular surface of MTH1 is shown in the MTH1–<b>16</b> complex structure (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6F23">6F23</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/large/jm-2017-01884d_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01884&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Biochemical Potency, Physicochemical Properties, Metabolic Stability, and Caco-2 Permeability Data for Variations in 2- and 4-Position Substituents on the Azaindole and Azaindazole Cores<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/medium/jm-2017-01884d_0011.gif" alt="" id="fx3" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">CaCo2-permeability measurements for compounds <b>31</b>–<b>34</b> were performed by an in-house method, as described in the <a class="ref internalNav" href="#sec5" aria-label="Experimental Section">Experimental Section</a>. For all the other compounds, the cell permeability was provided by Jubilant Life Sciences Ltd., as described in the <a class="ref internalNav" href="#sec5" aria-label="Experimental Section">Experimental Section</a>.</p></div></div><div></div></div><div class="NLM_p">To determine whether a parallel SAR with respect to potency could be established, a few pyrrolidine-based substituents in the 4-position were combined with a small number of secondary amide substituents in the 2-position. Indeed, all combinations generated compounds (<b>20</b>–<b>25</b>) that were highly potent, some of them even below the assay wall of 0.1 nM (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). When assessing Caco-2 permeability and metabolic stability, the most lipophilic representative (<b>20</b>) was clearly the most permeable but also the least metabolically stable analogue. Comparing to the more hydrophilic examples (<b>21</b>–<b>24</b>), it seemed reasonable to believe that microsomal stability was favored by decreasing lipophilicity, which in turn seemed unfavorable for permeability. This type of imbalance is commonly seen for small molecule drugs.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Still, we considered the compounds <b>24</b> and <b>25</b> as the most promising representatives in this series so far, being highly potent, fairly stable in microsomes, and with a medium Caco-2 permeability (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>).</div><div class="NLM_p">Considering the very high potency of our compounds, we decided to deliberately discard the affinity-increasing amide-substituent in the 2-position, aiming to investigate whether the permeability and/or microsomal stability could be improved at the expense of potency. In the design process, we were also bearing in mind that increasing the fraction of sp<sup>3</sup> carbons and decreasing molecular weight and the number of aromatic rings may be favorable to improve drug properties.<a onclick="showRef(event, 'ref31 ref36'); return false;" href="javascript:void(0);" class="ref ref31 ref36">(31,36)</a> We hypothesized that exchanging the carbon in the 2-position of the 7-azaindole core for a nitrogen would lead to a small decrease in lipophilicity, which might be beneficial for increasing metabolic stability.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> More importantly, this extra nitrogen would remove the C–C double bond of this core, which we hypothesized could be significant for improving metabolic stability.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> Thus, a set of the most promising 4-position substituents of earlier compounds and some additional new variations were transferred to the 7-azaindazole core, creating compounds <b>26</b>–<b>30</b> (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). Three of these showed biochemical potencies in the low nanomolar range, which is equivalent to the original 7-azaindole compounds, but as expected, neither of them were more potent than the corresponding 7-azaindole amide analogues. Permeability was very high for all of these small compounds (MW < 350), with no signs of efflux (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). However, the metabolic stability was still low, and in fact, all 7-azaindazole compounds were less stable than the 7-azaindole amide analogues. Also, the matched paired compounds <b>16</b> and <b>27</b>, with the 2-phenylpyrollidine substituent combined with the 7-azaindole and 7-azaindazole cores, respectively, were equipotent and showed similar permeability, stability, and solubility profiles.</div><div class="NLM_p">The crystal structure of <b>29</b> showed a structural water within hydrogen-bonding distance from the newly introduced 2-position nitrogen of the 7-azaindazole core. Interestingly, this water is also at hydrogen-bonding distance from both interaction points (Gly34 nitrogen and Asp120) of the earlier compounds with an amide substituent in the 2-position (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). While making hydrogen-bonding interactions with a structural water in some cases may increase potency,<a onclick="showRef(event, 'ref39 ref40'); return false;" href="javascript:void(0);" class="ref ref39 ref40">(39,40)</a> in our case, all matched-pair compounds (<b>9</b> and <b>26</b>, <b>20</b> and <b>27</b>, <b>22</b> and <b>28</b>) clearly showed that the amide interactions were more important for potency than the corresponding interactions made by the structural water.</div><figure id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/medium/jm-2017-01884d_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/large/jm-2017-01884d_0005.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Overlay between the complex structure of MTH1–<b>29</b> (yellow) and MTH1–<b>1</b> (green). Compounds <b>29</b> and <b>1</b> are displayed as thick and thin sticks, respectively, with their carbon atoms colored following the same code as the secondary structure elements. Side chain of MTH1 residues interacting with the compounds are shown as thin (<b>1</b>) and thick (<b>29</b>) sticks, respectively. Water molecules present in the binding site of <b>29</b> are depicted as red spheres. Hydrogen bonds between MTH1, water molecules, and <b>29</b> are displayed as orange dashes. Clipped molecular surface of MTH1 is shown in the MTH1–<b>29</b> complex structure (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6F22">6F22</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/large/jm-2017-01884d_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01884&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To ensure a maintained high selectivity profile for the most promising representatives of our chemical series, compound <b>24</b> was tested in the same 97-membered kinase panel as compound <b>10</b>. Again, excellent selectivity was achieved [S-score = 0, <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>]. In addition, <b>24</b> was also tested in a 215-membered ATP-ase panel, including the structurally related MTH1-analogues NUDT2, NUDT3, and NUDT7. In summary, <b>24</b> was more than 1000 times selective against all additional antitargets at 1 μM [<a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>].</div><div class="NLM_p">Cellular on-target engagement was another important property we wanted to verify for our MTH1-inhibitors. The most promising amide-containing compounds <b>20</b>, <b>24</b>, and <b>25</b> were selected for evaluation, together with two less potent compounds <b>16</b> and <b>28</b>. These compounds were selected both as representatives for the different subseries and to cover a biochemical potency range between 50 and 0.1 nM. TH588 was also included for comparisons.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> To produce a reliable tool compound, we were aiming at creating compounds with a target engagement EC<sub>50</sub> value that was more potent than the reference compound TH588. The compounds were evaluated in K562 cells in dose–response, using the cellular thermal shift assay (CETSA).<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> There was a strong correlation between biochemical potency and CETSA EC<sub>50</sub>. The three most potent compounds (<b>20</b>, <b>24</b>, and <b>25</b>) had significantly higher CETSA pEC<sub>50</sub> values than the reference, and compounds <b>20</b> and <b>24</b> were 100 times more potent in both the biochemical and CETSA assays (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>). Similar EC<sub>50</sub> values of TH588 have been reported previously.<a onclick="showRef(event, 'ref8 ref17'); return false;" href="javascript:void(0);" class="ref ref8 ref17">(8,17)</a></div><figure id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/medium/jm-2017-01884d_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/large/jm-2017-01884d_0006.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. There was a strong correlation between biochemical potency and target engagement in cells, using CETSA. Dotted line shows <i>y</i> = <i>x</i> – 1.5, indicating a drop-off from the biochemical assay to the cellular assay of about 30 times. Three compounds (<b>20</b>, <b>24</b>, and <b>25</b>) had significantly higher pEC<sub>50</sub> in CETSA than the reference TH588.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/large/jm-2017-01884d_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01884&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">During the design process, a computational model was iteratively developed for predicting the potency of compounds in the 7-azaindole and 7-azaindazole series. The docking protocols usually predicted the correct binding mode, as assessed by crystal structures. Initially, substituents in the 4-position were prioritized for synthesis based only upon the predicted binding mode of the final compound, primarily selecting the ones that allowed the core to maintain the same interaction mode to MTH1 as compound <b>1</b>. However, the docking score could not predict potency.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> Considering the importance of the hydrogen-bond between the pyridine-type nitrogen N7 of the 7-azaindole ring and the Asp119 carboxylate, we combined the docking score and predicted p<i>K</i><sub>a</sub> of N7 into a small partial-least-squares (PLS) model. This model was later fine-tuned by including a simple heavy atom count descriptor. During the optimization work, this model was continuously improved and used for selecting substituents in the 4-position, and the last iteration resulted in a predictive PLS model (R2 0.87), covering a potency range of more than five log-units [Figure S3 and Table S2, <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>]. In a recent study, another docking-based model was used to virtually optimize fragments for MTH1-binding.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a></div><div class="NLM_p">Inspired by the importance of the p<i>K</i><sub>a</sub> of N7 in the core structure, the high potency of the 7-azaindazole core, and considering that a large portion of the ligand affinity to a protein usually comes from electronic interactions with key residues, the core itself was also subjected to further investigation.<a onclick="showRef(event, 'ref44 ref45'); return false;" href="javascript:void(0);" class="ref ref44 ref45">(44,45)</a> Two matched pair variations of the azaindole and azaindazole cores, introducing an extra nitrogen in the 3- and 5-position, respectively, were synthesized. They were designed to investigate how these electronic variations would affect the hydrogen-bonding strength to the aspartates. None of them resulted in an improved potency (Table S3, <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>). To examine whether other nitrogen variations would be more favorable for affinity, another type of predictive potency model was developed. We were inspired by a previously published model, using density functional theory (DFT) calculations, where other types of core structures are interacting with two key aspartates of a protease.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> Our model used the same methodology, and was used to predict the potency of three additional, virtual variations of the azaindole. However, since all three virtual compounds were suggested to be less potent than the original azaindole and azaindazole cores, we decided not to synthesize any additional variations (Figure S4, <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>).</div><div class="NLM_p">To summarize the optimization-work so far, we decided to keep both the original 7-azaindole core and the secondary amide in the 2-position since compounds such as <b>24</b> were combining the highest potencies in both the biochemical and CETSA assays with high kinome and ATPase selectivity, as well as showing the most promising permeability and stability profiles. In the next optimization cycle, we were aiming to fine-tune the ADME properties, with respect to Caco-2 permeability and metabolic stability. Using a combination of the collected SAR knowledge from all previous compounds, the PLS potency model, and the above-mentioned medicinal chemistry rules of thumb, four new compounds (<b>31</b>–<b>34</b>) were designed with both new and previously used substituents in the 4-position. In these compounds, a combination of picomolar potency, high solubility, high permeability, and acceptable metabolic stability was finally realized (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>).</div><div class="NLM_p">Considering the promising properties of the four last compounds (<b>31</b>–<b>34</b>), they were nominated for additional <i>in vitro</i> studies, aiming at selecting one compound for <i>in vivo</i> efficacy studies. Complementary <i>in vitro</i> stability profiles were determined, using both human liver microsomes and rat hepatocytes. The overall lowest clearance values were shown for compounds <b>32</b> and <b>34</b> (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>), also being the most stable in mouse microsomes (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). The permeability profiles were also reevaluated internally, being aware of the commonly seen lab-to-lab variability in Caco-2 assays.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> Given the higher A–B permeability and lower efflux ratio of <b>32</b> as compared to <b>34</b> (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>), compound <b>32</b> was chosen as our probe, to be used in further <i>in vivo</i> studies. The <i>in vivo</i> bioavailability of <b>32</b> in mice was later shown to be high, and this probe molecule was extensively used to evaluate the potential of MTH1 inhibitors for the treatment of cancer.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a></div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. <i>In Vitro</i> Stability for the Four Most Optimized Compounds</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center">human liver microsomes</th><th class="rowsep1 colsep0" colspan="2" align="center">rat hepatocytes</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center">CL blood (L/h/kg)</th><th class="colsep0 rowsep0" align="center"><i>F</i><sub>max</sub> (%)</th><th class="colsep0 rowsep0" align="center">CL blood (L/h/kg)</th><th class="colsep0 rowsep0" align="center"><i>F</i><sub>max</sub> (%)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>31</b></td><td class="colsep0 rowsep0" align="center">0.3</td><td class="colsep0 rowsep0" align="center">76.4</td><td class="colsep0 rowsep0" align="center">2.8</td><td class="colsep0 rowsep0" align="center">32.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>32</b><a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="center">0.3</td><td class="colsep0 rowsep0" align="center">78.4</td><td class="colsep0 rowsep0" align="center">0.5</td><td class="colsep0 rowsep0" align="center">87.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>33</b></td><td class="colsep0 rowsep0" align="center">0.6</td><td class="colsep0 rowsep0" align="center">57.4</td><td class="colsep0 rowsep0" align="center">3.5</td><td class="colsep0 rowsep0" align="center">17.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>34</b></td><td class="colsep0 rowsep0" align="center">0.4</td><td class="colsep0 rowsep0" align="center">72.4</td><td class="colsep0 rowsep0" align="center">1.4</td><td class="colsep0 rowsep0" align="center">67.6</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">Compound <b>32</b> = BAY-707.</p></div></div></div><div class="NLM_p">In order to create a structurally related companion negative control probe, <b>32</b> was <i>N</i>-methylated in the 2-position, to disrupt the capability to interact with the key Asp120. Indeed, this methylated compound (<b>35</b>) demonstrated no activity in the MTH1 enzymatic assay until the highest concentration tested (20 μM). Furthermore, the physicochemical and pharmacokinetic properties of <b>32</b> (named as BAY-707) and <b>35</b> (named as BAY-604) were evaluated, demonstrating similar or better profiles as compared to the reference compound TH588 (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a> and Table S4, <a href="/doi/suppl/10.1021/acs.jmedchem.7b01884/suppl_file/jm7b01884_si_005.pdf" class="ext-link">Supporting Information</a>). Finally, the selectivity profiles across 20 members of the NUDIX family were assessed for both probe compounds, using TSA. A significant temperature shift (21 °C) was observed only for <b>32</b>, against its primary target MTH1, demonstrating the high selectivity of this new chemical matter (Figure S5, <a href="/doi/suppl/10.1021/acs.jmedchem.7b01884/suppl_file/jm7b01884_si_005.pdf" class="ext-link">Supporting Information</a>).</div><figure id="fig7" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/medium/jm-2017-01884d_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/large/jm-2017-01884d_0007.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Compounds <b>32</b> and <b>35</b> were created as positive and negative probes for MTH1.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/large/jm-2017-01884d_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01884&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i9">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_84280" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_84280" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">Starting from a generic, 10 μM fragment hit, a series of highly potent and selective MTH1 inhibitors were designed in a rational manner. Using fragment-based drug discovery methods and structure-based design, only 20 compounds were needed to improve biochemical potency by 10,000 times, while maintaining drug-like properties throughout the process, such as high ligand efficiency, high solubility, and low lipophilicity. An additional 15 compounds were designed to improve the ADME properties, thus generating both permeable and stable compounds. As a result, the probe compound BAY-707 was created, allowing for extensive target validation work both <i>in vitro</i> and <i>in vivo</i>, when evaluating the potential of MTH1 inhibitors as a novel treatment of cancer.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Both BAY-707 and its companion negative control BAY-604 are now available as probe molecules at the Structural Genomics Consortium (SGC).</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i10">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_62522" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_62522" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Production of MTH1 Enzyme</h3><div class="NLM_p last">Full length human MTH1 was expressed and purified as previously described.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a></div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Thermal Shift Assay</h3><div class="NLM_p">Purified MTH1 was screened against a library of 723 structurally diverse, soluble fragments. The thermal shift assay was performed in 96-well PCR plates (Bio-Rad) using a CFX96 Real-Time PCR Detection System (Bio-Rad). Protein unfolding was monitored using the fluorescent dye SYPRO Orange, which was purchased from Life Technologies as a 5000-times concentrated solution in 100% dimethyl sulfoxide (DMSO).</div><div class="NLM_p">Each well contained 24 μL of 0.1 mg/mL of MTH1 protein in TSA buffer (20 mM HEPES pH 7.5, 0.3 M NaCl, 10% glycerol, 0.5 mM TCEP) and 5× Sypro Orange dye. Fragments were added from a 25 mM master stock in 100% DMSO at a volume of 1 μL, yielding a final concentration of 1 mM in 4% (v/v) DMSO. Sixteen wells with MTH1 protein in 4% DMSO were used as reference samples in each plate. The plates were sealed with optically clear adhesive seals (Bio-Rad) and centrifuged at 1000 × <i>g</i> for 2 min.</div><div class="NLM_p">The fluorescence intensity data was acquired with excitation and emission wavelengths of 470 and 570 nm, respectively. The fluorescence was measured from 40 to 80 °C at a rate of 0.5 °C per 10 s. All measurements were carried out in duplicate, and average values are reported.</div><div class="NLM_p">The minimum and maximum fluorescence signal before and after the transition is used to determine the melting temperature, <i>T</i><sub>m</sub>, which is the inflection point of the sigmoidal denaturation curve. The raw data was exported to Microsoft Excel, and the <i>T</i><sub>m</sub> calculations were performed in XLfit using a Boltzmann Sigmoidal fit. The Δ<i>T</i><sub>m</sub> for each fragment is calculated as the difference between the <i>T</i><sub>m</sub> in the presence and absence of fragments, Δ<i>T</i><sub>m</sub> = <i>T</i><sub>m</sub>(fragment) – <i>T</i><sub>m</sub>(DMSO only). Significant thermal stabilization should be at least three times above the standard deviation of the DMSO controls, the initial hit cutoff value was set to +1.0 °C.</div><div class="NLM_p last">Forty-nine of the 166 initial hits were selected for retesting. Dilution series of the fragments were prepared in DMSO at 25 times the final assay concentration. MTH1 protein diluted in TSA buffer was added to the fragments as described above, giving final fragment concentrations from 62.5 to 1000 μM in 4% (v/v) DMSO (<i>n</i><sub>1</sub> = <i>n</i><sub>0</sub>/2 in five titration steps).</div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Isothermal Titration Calorimetry (ITC)</h3><div class="NLM_p">An ITC200 microcalorimeter (MicroCal/GE Healthcare) was used to obtain the thermodynamic information on binding between MTH1 and fragment <b>1</b>. All the measurements were performed in a buffer composed of 20 mM Tris pH 7.5, 150 mM NaCl, 2 mM MgCl<sub>2</sub>, 10% glycerol, 0.5 mM TCEP. The calorimeter cell (volume 200 μL) was loaded with MTH1 protein at a concentration of 326 μM, and the syringe was loaded with fragment <b>1</b> (dissolved in the same buffer) at a concentration of 3 mM in 1.5% DMSO (v/v). The DMSO concentrations in the cell and the syringe were carefully matched. Titrations were carried out at 25 °C with a stirring speed of 1000 rpm, and each reaction had an initial delay of 60 s. Spacing between injections was kept at 150 s and the reference power at 8 μcal/s. The injection protocol included a single 0.4 μL first injection followed by 38 × 2 μL injections of the compound into the calorimeter cell.</div><div class="NLM_p last">Data were analyzed using Origin 7 software package and fitted well to a one-site binding model.</div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Crystallization of Complex between Compounds and MTH1</h3><div class="NLM_p">Protein sample was thawed, diluted to 1.8 mg/mL in 20 mM Tris-HCl pH 7.5, 150 mM sodium chloride, 5% glycerol, 2 mM TCEP, and 6 mM magnesium chloride. For each complex preparation, the protein solution was then supplemented, respectively, with either 0.25 mM (<b>7</b> and <b>16</b>) or 1 mM (<b>1</b> and <b>29</b>) of compound. Incubation occurred overnight at 4 °C. Complexes were then concentrated 10 times in an Amicon Ultra (MWCO 10 kDa) (Millipore). Sample was centrifuged 30 min at 20000 × <i>g</i> (4 °C), and supernatant was used to setup crystallization trials.</div><div class="NLM_p last">In a 96-well plate (vapor diffusion using sitting-drop), 0.15 μL of complex was mixed with 0.15 μL of a reservoir solution composed with either 23–27% PEG 6000, 0.2–0.3 M lithium sulfate, and 0.1 M sodium acetate pH 3.6 (complexes with <b>7</b>, <b>16</b> and <b>1</b>) or 22% PEG 4000, 0.24 M ammonium sulfate, and 12.5% glycerol (complex with <b>29</b>). Equilibration occurs against 50 μL of reservoir solution at either 18 °C (complexes with <b>7</b>, <b>16</b>, and <b>1</b>) or 4 °C (complex with <b>29</b>). Rod-like crystals appeared within 6 days. Crystals were briefly soaked into a cryoprotecting-solution containing crystallization condition supplemented with 25% glycerol and then flash-frozen into liquid nitrogen.</div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Data Collection, Structure Determination, and refinement</h3><div class="NLM_p last">Diffraction data were collected at the ESRF on the beamlines indicated in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01884/suppl_file/jm7b01884_si_005.pdf" class="ext-link">Table S1</a>. Integration and scaling of data were performed using XDS.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> Merging and truncating data were performed using the CCP4 suite of program.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> Structures were solved by molecular replacement in Phaser<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> using either the BAY707-MTH1 structure (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5NHY">5NHY</a>)<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> (for complexes MTH1-29 and MTH1-1) or the apo-MTH1 structure (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3Q93">3Q93</a>)<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> (for complexes MTH1-7 and MTH1-16) as a model. The asymmetric unit was composed with two molecules, and compounds were manually placed in the difference Fourier electron density maps. Final models were produced using iterative cycles of manual model-building in Coot<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> and maximum-likelihood refinement in Refmac.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> Furthermore, for MTH1–<b>16</b> complex, TLS refinement (translation-libration-screw-rotation) were used during the latest stages of refinement using two TLS groups per monomer as identified by TLSMD<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> For MTH1–<b>29</b>, the last stage of refinement was performed using anisotropic refinement in Refmac. The respective accession codes together with data processing and refinement statistics are reported in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01884/suppl_file/jm7b01884_si_005.pdf" class="ext-link">Table S1</a>.</div></div><div id="sec5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> PPiLight Biochemical Assay</h3><div class="NLM_p last">Dilution series of the compounds were prepared in DMSO at 100 times the final assay concentration (<i>n</i><sub>1</sub> = <i>n</i><sub>0</sub>/3 in 10 points). The compounds were further diluted to 5 times the assay concentration in assay buffer (40 mM Tris-HCl pH 7.5, 40 mM sodium chloride, 10 mM magnesium chloride, 1 mM DTT, 0.005% Tween 20). Two microliters of the diluted compounds were added to a 384 well assay plate followed by 2 μL of 0.5 nM enzyme (in-house produced full length human MTH1). Enzyme and compounds were preincubated at room temperature for 15 min. Then 6 μL of a substrate-converting-detection mixture (1 μL of 132 μM substrate (8-Oxo-2′-dGTP, TriLink Biotechnologies N-2034), 2.5 μL of converting reagent and 2.5 μL of detection reagent (PPiLight pyrophosphate Detection Kit, Lonza LT07–610)) was added to the wells containing compound and enzyme, and mixing was performed by pipetting several times. The assay plate was incubated at room temperature for 1 h and then read in a Victor 3V 1420 multilabel counter (PerkinElmer), with measurement time of 0.1 s. Percent inhibition of the compounds as compared to dimethyl sulfoxide treated control samples was calculated. Compound concentration versus percent inhibition were fitted to generate IC<sub>50</sub> values. Final compound concentration intervals are ranging from approximately 100× below to 100× above the respective IC<sub>50</sub> value. The final concentration enzyme is 0.1 nM, which defines the limit for the IC<sub>50</sub> values that can be measured in the assay.</div></div><div id="sec5_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Computational Chemistry</h3><div class="NLM_p">All computational chemistry work was done using Maestro 2016–3 from Schrödinger. Prior to docking studies and grid generation, the protein crystal structures were curated by preprocessing them in the Protein Preparation Wizard, using default settings. As the second step, hydrogen-bond assignment was done using default settings, and lastly, a restrained minimization was performed on hydrogens only. Crystallographic waters were removed in the active site, except for those structures that were used to dock the 7-azaindazoles (making a hydrogen bond to the crystallographic water binding to Asp120).</div><div class="NLM_p">Docking grids and sitemaps were created using default settings, defining the active site automatically around the crystallized ligand, belonging to each crystallized protein, respectively. Asp 119 was protonated on the oxygen at the closest distance to Trp123, to allow for a hydrogen bond to the pyridine nitrogen of the core.</div><div class="NLM_p">Glide dockings were done using default settings, without constraints. The induced fit protocol was used for retrospectively predicting the correct binding mode of compound <b>5</b> using the crystal structure of compound <b>1</b>. Default settings were used, except for the box size, which was reduced to save time: “Dock ligands with length = <12 Å”.</div><div class="NLM_p">The PLS-based potency prediction model was created within the Partial Least-Squares Regression module in Canvas. Docking scores were generated as above, extracting the lowest glide score from each grid based on the crystal structures of <b>7</b>, <b>16</b>, or <b>29</b>, the last one keeping the crystallographic water binding to Asp120. Heavy atom count (HAC) was calculated within Canvas. Predicted p<i>K</i><sub>a</sub> values of the 7-azaindole acceptor-nitrogen were calculated with the Jaguar p<i>K</i><sub>a</sub>-module, using default settings and water as solvent. Loading factors for the PLS-model were −0.27 for glide score, 0.85 for HAC, and 0.45 for predicted p<i>K</i><sub>a</sub>. Predicted potencies, predicted p<i>K</i><sub>a</sub>, glide scores, and which compounds were used as training and test set are found in Table S2, <a href="/doi/suppl/10.1021/acs.jmedchem.7b01884/suppl_file/jm7b01884_si_005.pdf" class="ext-link">Supporting Information</a>.</div><div class="NLM_p last">The DFT-based potency model was based on calculating the difference in energy (dE) between the protein–ligand complex in vacuum minus the energy of the protein alone in vacuum and the ligand alone in water. The predicted energy difference (a negative value) was linearly fitted to the biochemical pIC<sub>50</sub> values for the four synthesized compounds, using the equation pred(pIC<sub>50</sub>) = 3.1–0.31<i>dE</i> (Table S3, <a href="/doi/suppl/10.1021/acs.jmedchem.7b01884/suppl_file/jm7b01884_si_005.pdf" class="ext-link">Supporting Information</a>). Parts of the surrounding residues Trp117, Trp123, and Phe72 were needed to keep the ligand in the correct position above the key residues Asp119 and Asp120 during energy optimization. Examples of the Jaguar input-files of how the energies were calculated for the ligand only, protein–ligand complex, and the apo-structure are found in the <a href="/doi/suppl/10.1021/acs.jmedchem.7b01884/suppl_file/jm7b01884_si_005.pdf" class="ext-link">Supporting Information</a>. Basis set was lacvp+ and DFT method B3LYP. The starting position of the ligands were created via superpositioning of the core onto the crystal structure of <b>7</b>.</div></div><div id="sec5_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> CETSA</h3><div class="NLM_p">Isothermal concentration response curves<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> were used to determine cellular target engagement potency of the compounds on MTH1.</div><div class="NLM_p">K562 cells were cultured at 37 °C, 5% carbon dioxide in RPMI-1640 medium supplemented with 10% fetal bovine serum, 100 units/mL penicillin, and 100 units/mL streptomycin. The cells were fed with half the volume of additional fresh media a day prior to measurements. Cells of passage 16–19 that reached a density of 1.0–1.4 million/mL were used in the experiments. Intact K562 cells were pelleted by centrifugation and washed and resuspended in HBSS at a cell density of 40 million cells/mL. The cell suspension was divided into 30 μL aliquots, and an equal volume of HBSS containing 2× the intended compound concentration was added, resulting in a final cell concentration of 20 million cells/mL. Up to 11-step dilution concentrations of the ligands in 1% DMSO were applied together with 1% DMSO only as control. The cells were incubated with ligand at 37 °C for 60 min, with gentle mixing every 10 min.</div><div class="NLM_p">The aliquots were heated to a single specific temperature, 54 °C, as determined from established MTH1 melt and shift curves, for 3 min, and lysed by three cycles of freeze–thawing. Precipitated protein and cell debris were pelleted by centrifugation at 20000 × <i>g</i> for 20 min. Protein amounts were detected using standard Western Blot techniques. For detection of specific MTH1 bands the primary antibody sc-271082 (Santa Cruz Biotechnology) and the secondary antibody sc-2055 (Santa Cruz Biotechnology) together with Clarity Western ECL substrate (Bio-Rad) were used.</div><div class="NLM_p last">The Western Blot intensities were obtained by measuring the chemiluminescence counts per square millimeter, and they were plotted normalized to a relevant control count. The isothermal concentration response data was analyzed and plotted using GraphPad Prism analysis and software. The obtained CETSA EC<sub>50</sub> concentration response values represent the ligand concentration required to achieve half of the maximal stabilization of MTH1 at 54 °C.</div></div><div id="sec5_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Caco-2 Permeability Assay</h3><div class="NLM_p">Caco-2 cells (purchased from ATCC) were seeded in DMEM 20% FBS at a density of 6 × 10<sup>4</sup> cells/well in a 24-well plate (Millicell Cell Culture) and cultured for 21 days with media replacement every 3 days. Monolayer integrity was assessed by measuring the transepithelial electrical resistance (TEER), and only wells with TEER values greater than 500 Ω cm<sup>2</sup> were considered for the permeability experiment. Digoxin was used as a control compound for efflux.</div><div class="NLM_p">Test compound working stock solution at 10 μM was prepared by spiking 10 μL of the compound solution (10 mM in DMSO) in 9.990 mL of transport media (TM) (HBSS buffer 10 mM HEPES, pH 7.4).</div><div class="NLM_p last">Experiments were performed in the apical (A) to basolateral (B) (A to B) as well as in the basolateral to apical (B to A). Cells were washed twice with TM, and the cell filter plate was transferred to a 24-well transporter analysis plate containing either 1.0 mL of TM per well (for A to B setup) or 10 μM test compound and 5 μM reference compound (for B to A setup) in the receiver chamber. Permeability study was initiated by adding 0.3 mL of TM 10 μM of test compound or 5 μM reference compound to the donor chamber (A to B setup) or TM alone (B to A setup) in duplicates. Plates were incubated at 100 rpm and 37 °C, and an aliquot of 100 μL was collected from donor as well as receiver chamber at intervals of time 0 and 60 min as indicated. After addition of equal volumes of acetonitrile containing IS (phenacetin, 200 ng/mL), samples were centrifuged, and the cleared supernatants were analyzed by LC–MS/MS.<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a></div></div><div id="sec5_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Caco-2 Cell Permeability Assay for Compounds <b>31</b>–<b>34</b></h3><div class="NLM_p last">Caco-2 cells (purchased from DSMZ Braunschweig, Germany) were seeded at a density of 4.5 × 10<sup>4</sup> cell per well on 24-well insert plates, 0.4 μm pore size, and grown for 15 days in DMEM medium supplemented with 10% fetal bovine serum, 1% GlutaMAX (100×, GIBCO), 100 U/mL penicillin, 100 μg/mL streptomycin (GIBCO), and 1% nonessential amino acids (100×) and cultured according to the providers specifications. Before running the permeation assay, the culture medium was replaced by a FBS-free HEPES-carbonate transport puffer (pH 7.2). Monolayer integrity was assessed by measuring the transepithelial electrical resistance (TEER). Test compounds were added either to the apical or basolateral compartment to achieve 2 μM final concentration. Before and after 2 h incubation at 37 °C, samples were taken from both compartments. Analysis of compound content was done after precipitation with methanol by LC/MS/MS analysis. The apparent permeability (<i>P</i><sub>app</sub>) was calculated using the following equation:<span class="NLM_disp-formula" id="ueq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/medium/jm-2017-01884d_m001.gif" alt="" id="_i21" /></img></span>where <i>V</i><sub>r</sub> is the volume of medium in the receiver chamber, <i>P</i><sub>o</sub> is the measured peak area of the test drug in the donor chamber at <i>t</i> = 0, <i>S</i> the surface area of the monolayer, <i>P</i><sub>2</sub> is the measured peak area of the test drug in the acceptor chamber after 2 h of incubation, and <i>t</i> is the incubation time. The efflux ratio basolateral (B) to apical (A) was calculated by dividing the <i>P</i><sub>app</sub> B–A by the <i>P</i><sub>app</sub> A–B.</div></div><div id="sec5_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Metabolic Stability Assay in Mouse Liver Microsomes</h3><div class="NLM_p">Assays were performed as previously described<a onclick="showRef(event, 'ref55 ref56'); return false;" href="javascript:void(0);" class="ref ref55 ref56">(55,56)</a> with some modifications. Test compound or verapamil hydrochloride (reference compound) was added to 1 mL of microsome suspension (Life Technologies [Gibco], Inchinnan, UK) (550 μg/mL in potassium phosphate buffer 66.7 mM, pH 7.4, PPB) to achieve a final concentration of 1 mM and 0.1% DMSO. Aliquots of 180 μL of the incubation mixture were taken at indicated times and incubated at 37 °C for 5 min. Then 20 μL of NAPDH solution (10 mM in PPB) were added to each tube, except for control samples, which received 20 μL of PPB. At the end of the incubation period, 200 μL of quenching solution (0.05% formic acid in acetonitrile with 200 ng/mL phenacetin as internal standard) was added to stop the reaction. The samples were centrifuged at 4000 rpm for 10 min, and the supernatant was analyzed by LC–MS/MS.</div><div class="NLM_p last">Calculation of microsomal intrinsic clearance (mCLint) was based on the test compound disappearance rate. The chromatographic peaks were integrated, and the resultant peak area ratios between test compound or reference compound and IS were used for the calculations. A linear plot of the natural logarithm (ln) or the area ratio vs time was created to visualize the data, and the slope was calculated for each data set. Subsequently, half-life (<i>T</i><sub>1/2</sub>) and mCLint were calculated according to the following equations:<span class="NLM_disp-formula" id="ueq2"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/medium/jm-2017-01884d_m002.gif" alt="" id="_i23" /></img></span>where <i>x</i> is 5, 15, and 30 min, respectively, and area ratio is analyte peak area/IS peak area.<span class="NLM_disp-formula" id="ueq3"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/medium/jm-2017-01884d_m003.gif" alt="" id="_i24" /></img></span><span class="NLM_disp-formula" id="ueq4"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/medium/jm-2017-01884d_m004.gif" alt="" id="_i25" /></img></span>where [microsomal protein] is 0.5 mg/1000 μL.</div></div><div id="sec5_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> Metabolic Stability in Rat Hepatocytes</h3><div class="NLM_p">Hepatocytes from Han Wistar rats were isolated via a two-step perfusion method. After perfusion, the liver was carefully removed from the rat: the liver capsule was opened, and the hepatocytes were gently shaken out into a Petri dish with ice-cold Williams’ medium E (WME). The resulting cell suspension was filtered through sterile gaze in 50 mL falcon tubes and centrifuged at 50 × <i>g</i> for 3 min at room temperature. The cell pellet was resuspended in 30 mL WME and centrifuged through a Percoll gradient twice at 100 × <i>g</i>. The hepatocytes were washed again with WME and resuspended in medium containing 5% FCS. Cell viability was determined by trypan blue exclusion.</div><div class="NLM_p">For the metabolic stability assay liver cells were distributed in WME containing 5% FCS to glass vials at a density of 1.0 × 10<sup>6</sup> vital cells/mL. The test compound was added to a final concentration of 1 μM. During incubation, the hepatocyte suspensions were continuously shaken at 580 rpm, and aliquots were taken at 2, 8, 16, 30, 45, and 90 min, to which equal volumes of cold methanol were immediately added. Samples were frozen at −20 °C overnight, after subsequently centrifuged for 15 min at 3000 rpm, and the supernatant was analyzed with an Agilent 1200 HPLC-system with LC–MS/MS detection.</div><div class="NLM_p last">The half-life of a test compound was determined from the concentration–time plot. From the half-life the intrinsic clearances were calculated using the well stirred liver model together with the additional parameters liver blood flow and amount of liver cells <i>in vivo</i> and <i>in vitro</i>. The hepatic <i>in vivo</i> blood clearance (CL) and the maximal oral bioavailability (<i>F</i><sub>max</sub>) was calculated. The following parameter values were used: liver blood flow = 4.2 L/h/kg rat; specific liver weight = 32 g/kg rat body weight; liver cells <i>in vivo</i> = 1.1 × 10<sup>8</sup> cells/g liver, liver cells <i>in vitro</i> = 1.0 × 10<sup>6</sup> cells/mL.</div></div><div id="sec5_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> Metabolic Stability in Human Liver Microsomes</h3><div class="NLM_p">The <i>in vitro</i> metabolic stability of test compounds was determined by incubating them at 1 μM in a suspension of liver microsomes in 100 mM phosphate buffer, pH 7.4 (NaH<sub>2</sub>PO<sub>4</sub>·H<sub>2</sub>O + Na<sub>2</sub>HPO<sub>4</sub>·2H<sub>2</sub>O) and at a protein concentration of 0.5 mg/mL at 37 °C. The microsomes were activated by adding a cofactor mix containing 8 mM glucose-6-phosphate, 4 mM MgCl<sub>2</sub>; 0.5 mM NADP and 1 IU/mL G-6-P-dehydrogenase in phosphate buffer, pH 7.4. The metabolic assay was started shortly afterward by adding the test compound to the incubation at a final volume of 1 mL. Organic solvent in the incubations was limited to ≤0.01% dimethyl sulfoxide (DMSO) and ≤1% acetonitrile. During incubation, the microsomal suspensions were continuously shaken at 580 rpm, and aliquots were taken at 2, 8, 16, 30, 45, and 60 min, to which equal volumes of cold methanol were immediately added. Samples were frozen at −20 °C overnight and subsequently centrifuged for 15 min at 3000 rpm, and the supernatant was analyzed with an Agilent 1200 HPLC-system with LC–MS/MS detection.</div><div class="NLM_p last">The half-life of a test compound was determined from the concentration–time plot. From the half-life the intrinsic clearances were calculated. Together with the additional parameters liver blood flow, specific liver weight, and microsomal protein content, the hepatic <i>in vivo</i> blood clearance (CL) and the maximal oral bioavailability (<i>F</i><sub>max</sub>) were calculated using the well stirred liver model. The following parameter values were used: Liver blood flow = 1.32 L/h/kg; specific liver weight = 21 g/kg; microsomal protein content = 40 mg/g.</div></div><div id="sec5_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> Chemistry</h3><div class="NLM_p">Unless otherwise stated, all solvents, reagents, and starting materials used were commercially available and were used without further purification. Reactions requiring anhydrous conditions were run in dried equipment using anhydrous solvents under an inert atmosphere of nitrogen or argon. Room temperature refers to 20–25 °C. Microwave heating was performed in a Biotage Initiator Microwave Synthesizer at the indicated temperature in the recommended microwave tubes.</div><div class="NLM_p"><sup>1</sup>H NMR spectra were recorded in the indicated deuterated solvent at 400 or 500 MHz, and the spectra were obtained unless stated otherwise, using a Bruker AV400 NMR or Bruker 500 MHz Avance NMR spectrometer with a probe of suitable configuration. Chemical shifts are reported in ppm from lowest to highest. Resonance multiplicities are denoted s, d, t, q, m, and br for singlet, doublet, triplet, quartet, multiplet, and broad, respectively.</div><div class="NLM_p">Preparative HPLC was performed on a Gilson HPLC system with a UV detector using an XBridge Prep C18 5 μm 30 × 75 mm column with acetonitrile/0.2% NH<sub>3</sub> in Milli-Q gradient or a YMC-Pack ODS-AQ C18 5 μm 30 × 150 mm column with acetonitrile/0.2% acetic acid in Milli-Q gradient. Flow rate: 35 mL/min. Flash chromatography was performed using a Combi Flash Companion system using Silicycle normal-phase flash columns.</div><div class="NLM_p">Purity and mass analyses were performed on an LC–MS consisting of an Agilent 1100 system with a G1322A Degasser, G1312A Binary Pump, G1313A Autosampler, G1316A Column Oven, G1365B Multi Wavelength Detector, and G1946D Mass Spectrometer. The mass spectrometer was equipped with an electrospray ion source (ES) operated in positive ion mode. For separation, a linear gradient was applied starting at 90% 0.1% NH<sub>3</sub> in Milli-Q ending at 90% acetonitrile on a XTerra RP18 3.5 μm 3.0 × 50 mm column at a flow rate of 1 mL/min for 3 min or a linear gradient starting at 90% 0.1% acetic acid in Milli-Q ending at 90% acetonitrile on a Symmetry C18 3.5 μm 3.0 × 50 mm column at a flow rate of 1 mL/min for 3 min. The purity of final compounds was assessed by reversed-phase HPLC with UV detection at 214 nm. All tested compounds demonstrated HPLC purity ≥95%.</div><div id="sec5_14_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> 4-(1<i>H</i>-Pyrrolo[2,3-<i>b</i>]pyridin-5-yl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine (<b>2</b>)</h4><div class="NLM_p last">5-Bromo-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine (100 mg, 0.51 mmol), <i>tert</i>-butyl 4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)pyrrolo[2,3-<i>b</i>]pyridine-1-carboxylate (192 mg, 0.56 mmol), potassium carbonate (140 mg, 1.0 mmol), and [1,1′-bis(diphenylphosphino)ferrocene]dichloropalladium(II) (4.1 mg, 0,0051 mmol) were dissolved in 1,2-dimethoxyethane (5 mL). The mixture was heated to 100 °C in a sealed tube. The suspension was filtered and purified by preparative RP-HPLC with ammonia buffer to give 10 mg (8%) of the title compound. <sup>1</sup>H NMR (500 MHz, methanol-<i>d</i><sub>4</sub>) δ ppm 6.65 (d, <i>J</i> = 3.47 Hz, 1 H), 6.74 (d, <i>J</i> = 3.47 Hz, 1 H), 7.30 (d, <i>J</i> = 5.04 Hz, 1 H), 7.51 (dd, <i>J</i> = 6.62, 3.47 Hz, 2 H), 8.29 (d, <i>J</i> = 4.73 Hz, 1 H), 8.42 (d, <i>J</i> = 1.89 Hz, 1 H), 8.61–8.64 (m, 1 H). MS (ES+) <i>m</i>/<i>z</i> 235 [M + H]<sup>+</sup>.</div></div><div id="sec5_14_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> 4-(3-Pyridyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine (<b>3</b>)</h4><div class="NLM_p last">4-Chloro-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine (250 mg, 1.6 mmol) and 1<i>H</i>-pyrazole (279 mg, 4.1 mmol) were taken up in 1,4-dioxane (1.5 mL). Then, saturated aqueous ammonium formate (1.5 mL) and iodine (1.04 g, 4.1 mmol) were added, and the resulting mixture was stirred at room temperature for 16 h. A saturated aqueous solution of sodium thiosulfate (3 mL) and ethyl acetate (10 mL) were added. The phases were separated, and the aqueous phase was extracted with ethyl acetate (2 × 10 mL). The combined organic phases were washed with brine, dried over sodium sulfate, filtered, and concentrated to give a yellow solid. The material was then recrystallized from methanol to give 240 mg (67%) of 4-chloro-2-pyrazol-1-yl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 6.64 (s, 1 H), 6.69 (s, 1 H), 7.24 (d, <i>J</i> = 5.0 Hz, 1 H), 7.87 (s, 1 H), 8.15 (d, <i>J</i> = 5.3 Hz, 1 H), 8.57 (s, 1 H), 12.97 (br. s., 1 H). MS (ES+) <i>m</i>/<i>z</i> 219 [M + H]<sup>+</sup>. A portion of this material (140 mg, 0.64 mmol), bis(di<i>tert</i>-butyl(4-dimethylaminophenyl)phosphine)dichloropalladium(II) (23 mg, 0.032 mmol), potassium carbonate (265 g, 1.9 mmol), and 3-pyridylboronic acid (94 mg, 0.77 mmol) were taken up in a mixture of 1,4-dioxane/H<sub>2</sub>O/EtOH (6:3:1) (2 mL), and the resulting mixture was heated in a microwave reactor at 150 °C for 75 min. The mixture was filtered and purified by preparative RP-HPLC with ammonia buffer to give 18 mg (14%) of the title compound as a beige solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 6.63 (br. s., 1 H), 7.26 (d, <i>J</i> = 4.8 Hz, 1 H), 7.64–7.52 (m, 2 H), 8.18 (d, <i>J</i> = 8.1 Hz, 1 H), 8.32 (d, <i>J</i> = 4.8 Hz, 1 H), 8.67 (d, <i>J</i> = 4.5 Hz, 1 H), 8.97 (s, 1 H), 11.89 (br. s., 1 H). MS (ES+) <i>m</i>/<i>z</i> 196 [M + H]<sup>+</sup>.</div></div><div id="sec5_14_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> 4-(3-Chlorophenyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine (<b>4</b>)</h4><div class="NLM_p last">4-Chloro-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine (100 mg, 0.66 mmol), (3-chlorophenyl)boronic acid (102 mg, 0.66 mmol), bis(di<i>tert</i>-butyl(4-dimethylaminophenyl)phosphine)dichloropalladium(II) (46 mg, 0.066 mmol), and potassium carbonate (181 mg, 1.3 mmol) were taken up into acetonitrile (6 mL) and water (1 mL). The mixture was heated in a microwave reactor at 120 °C for 30 min. The organic layer was collected, filtered, and purified by preparative RP-HPLC with ammonia buffer to give 20 mg (13%) of the title compound. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 6.54–6.68 (m, 1 H), 7.23 (d, <i>J</i> = 5.04 Hz, 1 H), 7.49–7.68 (m, 3 H), 7.69–7.86 (m, 2 H), 8.31 (d, <i>J</i> = 5.04 Hz, 1 H), 11.88 (br s, 1 H). MS (ES+) <i>m</i>/<i>z</i> 229 [M + H]<sup>+</sup>.</div></div><div id="sec5_14_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> <i>N</i>-Methyl-4-(3-pyridyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-2-carboxamide (<b>5</b>)</h4><div class="NLM_p last"><i>n</i>-Butyllitium (4.1 mL, 1.6 M in hexanes, 6.6 mmol) was added slowly to a solution of diisopropylamine (1 mL, 7.1 mmol) in 2-methyltetrahydrofuran (20 mL) at 0 °C. The reaction was stirred at 0 °C for 5 min and then cooled to −78 °C. A solution of 1-(benzenesulfonyl)-4-chloro-pyrrolo[2,3-<i>b</i>]pyridine (1.63 g, 5.6 mmol) in 2-methyltetrahydrofuran (10 mL) was added dropwise. The resulting mixture was stirred at −78 °C for 15 min and then allowed to heat to room temperature. After stirring for another 10 min, the mixture was again cooled to −78 °C. Ethyl chloroformate (0.7 mL, 7.3 mmol) was added dropwise, and the resulting mixture was stirred on the thawing cooling bath for 30 min and then at room temperature for 30 min. Saturated aqueous ammonium chloride was added followed by water and ethyl acetate. The organic layer was separated, and the aqueous layer was extracted twice with ethyl acetate. The combined organic phases were washed with aqueous hydrochloric acid (1 M) and brine, dried over sodium sulfate, and concentrated. The crude product was dissolved in dichloromethane, passed through a plug of silica gel, and concentrated to give 2.0 g of ethyl 1-(benzenesulfonyl)-4-chloro-pyrrolo[2,3-<i>b</i>]pyridine-2-carboxylate (MS (ES+) <i>m</i>/<i>z</i> = 365 [M + H]<sup>+</sup>). A portion of this material (150 mg, 0.41 mmol), bis(di<i>tert</i>-butyl(4-dimethylaminophenyl)phosphine)dichloropalladium(II) (15 mg, 0.02 mmol), 3-pyridylboronic acid (66 mg, 0.53 mmol), and potassium carbonate (170 mg, 1.2 mmol) were taken up in a mixture of 1,4-dioxane/H<sub>2</sub>O/EtOH (6:3:1) (2 mL), and the resulting mixture was heated in a microwave reactor at 150 °C for 15 min. Methylamine (1 mL, 40% in water, 12 mmol) was added, and the mixture was heated in a microwave reactor at 150 °C for 15 min. The reaction mixture was filtered and purified by preparative RP-HPLC with ammonia buffer to give 12 mg (12%) of the title compound over several steps from ethyl 1-(benzenesulfonyl)-4-chloro-pyrrolo[2,3-<i>b</i>]pyridine-2-carboxylate. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 2.82 (br d, <i>J</i> = 3.78 Hz, 3 H), 7.32 (br d, <i>J</i> = 4.53 Hz, 1 H), 7.35 (s, 1 H), 7.59–7.65 (m, 1 H), 8.20 (br d, <i>J</i> = 7.81 Hz, 1 H), 8.42 (d, <i>J</i> = 4.78 Hz, 1 H), 8.54–8.66 (m, 1 H), 8.71 (br d, <i>J</i> = 5.04 Hz, 1 H), 9.01 (s, 1 H). MS (ES+) <i>m</i>/<i>z</i> 253 [M + H]<sup>+</sup>.</div></div><div id="sec5_14_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> 4-(3-Chlorophenyl)-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-2-carboxamide (<b>6</b>)</h4><div class="NLM_p last">Ethyl 1-(benzenesulfonyl)-4-chloro-pyrrolo[2,3-<i>b</i>]pyridine-2-carboxylate (2.4 g, 6.6 mmol), (3-chlorophenyl)boronic acid (1.13 g, 7.2 mmol), potassium carbonate (2.27 g, 16 mmol), and bis(di<i>tert</i>-butyl(4-dimethylaminophenyl)phosphine)dichloropalladium(II) (0.23 g, 0.33 mmol) were dissolved in acetonitrile (10 mL) and water (5 mL). The reaction was heated in a microwave reactor at 120 °C for 2 h. The mixture was diluted with ethyl acetate and water. The phases were separated, and the organic phase was concentrated. The formed residue was recrystallized from 2-propanol to give 2.0 g of ethyl 1-(benzenesulfonyl)-4-(3-chlorophenyl)pyrrolo[2,3-<i>b</i>]pyridine-2-carboxylate (MS (ES+) <i>m</i>/<i>z</i> 441 [M + H]<sup>+</sup>), which was dissolved in methanol (20 mL), tetrahydrofuran (10 mL), and water (5 mL). Lithium hydroxide monohydrate (0.95 g, 23 mmol) was added, and the resulting mixture was stirred at 50 °C for 16 h. The mixture was concentrated, and the resulting residue was dissolved in water. Concentrated hydrochloric acid was added to acidify the solution, which was then kept in a refrigerator for 30 min. The resulting precipitate was collected by suction filtration, washed with water, and dried to give 700 mg of 4-(3-chlorophenyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-2-carboxylic acid as an orange solid. A portion of this material (50 mg, 0.18 mmol), methylamine (0.18 mL, 2 M in ethanol, 0.37 mmol), <i>O</i>-(benzotriazol-1-yl)-<i>N</i>,<i>N</i>,<i>N</i>′,<i>N</i>′-tetramethyluronium hexafluorophosphate (83 mg, 0.22 mmol), and triethylamine (0.051 mL, 0.37 mmol) were dissolved in <i>N</i>,<i>N</i>-dimethylformamide (1 mL). The reaction was stirred at room temperature for 16 h. The reaction mixture was diluted with methanol, filtered, and purified by preparative RP-HPLC with ammonia buffer to give 7 mg (5%) of the title compound over several steps from ethyl 1-(benzenesulfonyl)-4-chloro-pyrrolo[2,3-<i>b</i>]pyridine-2-carboxylate. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 2.83 (d, <i>J</i> = 4.41 Hz, 3 H), 7.27–7.30 (m, 1 H), 7.30–7.34 (m, 1 H), 7.57–7.67 (m, 2 H), 7.74–7.78 (m, 1 H), 7.80–7.84 (m, 1 H), 8.35–8.48 (m, 1 H), 8.56–8.66 (m, 1 H), 12.34 (br s, 1 H). MS ES+ 286 <i>m</i>/<i>z</i> [M + H]<sup>+</sup></div></div><div id="sec5_14_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> 4-(3-Chlorophenyl)-<i>N</i>-ethyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-2-carboxamide (<b>7</b>)</h4><div class="NLM_p last">4-(3-Chlorophenyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-2-carboxylic acid (120 mg, 0.44 mmol), ethylamine (0.44 mL, 2 M in tetrahydrofuran, 0.88 mmol), <i>O</i>-(benzotriazol-1-yl)-<i>N</i>,<i>N</i>,<i>N</i>′,<i>N</i>′-tetramethyluronium hexafluorophosphate (200 mg, 0.53 mmol), and triethylamine (0.12 mL, 0.88 mmol) were dissolved in <i>N</i>,<i>N</i>-dimethylformamide (2 mL). The reaction was stirred at room temperature for 16 h. The reaction mixture was filtered and purified by preparative RP-HPLC with ammonia buffer to give 9 mg (2%) of the title compound over several steps from ethyl 1-(benzenesulfonyl)-4-chloro-pyrrolo[2,3-<i>b</i>]pyridine-2-carboxylate. <sup>1</sup>H NMR (500 MHz, chloroform-<i>d</i>) δ ppm 1.33 (t, <i>J</i> = 7.25 Hz, 3 H), 3.60 (quin, <i>J</i> = 6.78 Hz, 2 H), 6.36 (br s, 1 H), 6.98 (s, 1 H), 7.23 (d, <i>J</i> = 4.73 Hz, 1 H), 7.45–7.54 (m, 2 H), 7.63 (br d, <i>J</i> = 6.94 Hz, 1 H), 7.74 (s, 1 H), 8.65 (d, <i>J</i> = 4.73 Hz, 1 H), 11.25 (br s, 1 H). MS ES+ 300 <i>m</i>/<i>z</i> [M + H]<sup>+</sup></div></div><div id="sec5_14_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> 4-(3-Chlorophenyl)-<i>N</i>,<i>N</i>-dimethyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-2-carboxamide (<b>8</b>)</h4><div class="NLM_p last">4-(3-Chlorophenyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-2-carboxylic acid (200 mg, 0.73 mmol), dimethylamine (0.73 mL, 2 M in tetrahydrofuran, 1.5 mmol), <i>O</i>-(benzotriazol-1-yl)-<i>N</i>,<i>N</i>,<i>N</i>′,<i>N</i>′-tetramethyluronium hexafluorophosphate (330 mg, 0.88 mmol), and triethylamine (0.2 mL, 1.5 mmol) were dissolved in <i>N</i>,<i>N</i>-dimethylformamide (1 mL). The reaction was stirred at room temperature for 16 h. The reaction mixture was diluted with methanol, filtered, and purified by preparative RP-HPLC with ammonia buffer to give 7 mg (1%) of the title compound over several steps from ethyl 1-(benzenesulfonyl)-4-chloro-pyrrolo[2,3-<i>b</i>]pyridine-2-carboxylate. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 3.06 (br s, 3 H), 3.21 (br s, 3 H), 6.88 (d, <i>J</i> = 2.01 Hz, 1 H), 7.28 (d, <i>J</i> = 4.78 Hz, 1 H), 7.55–7.63 (m, 2 H), 7.72–7.79 (m, 1 H), 7.80 (s, 1 H), 8.40 (d, <i>J</i> = 4.78 Hz, 1 H), 12.32 (br s, 1 H). MS ES+ <i>m</i>/<i>z</i> 300 [M + H]<sup>+</sup>.</div></div><div id="sec5_14_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> 4-(3-Chlorophenyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-2-carboxamide (<b>9</b>)</h4><div class="NLM_p last">4-(3-Chlorophenyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-2-carboxylic acid (300 mg, 1.1 mmol) and oxalyl dichloride (0.11 mL, 1.3 mmol) were dissolved in dichloromethane (5 mL). One drop of <i>N</i>,<i>N</i>-dimethylformamide was added, and the mixture was stirred at room temperature for 1 h. The mixture was concentrated under reduced pressure, and the residue was dissolved in dichloromethane. Ammonia was bubbled though the mixture for 30 min. The mixture was concentrated, and the crude solid was crystallized from 2-propanol to give 10 mg (1%) of the title compound over several steps from ethyl 1-(benzenesulfonyl)-4-chloro-pyrrolo[2,3-<i>b</i>]pyridine-2-carboxylate. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 7.29 (d, <i>J</i> = 4.73 Hz, 1 H), 7.34–7.40 (m, 1 H), 7.50 (br s, 1 H), 7.61 (s, 2 H), 7.77 (br d, <i>J</i> = 7.75 Hz, 1 H), 7.82 (s, 1 H), 8.00–8.15 (m, 1 H), 8.42 (d, <i>J</i> = 4.73 Hz, 1 H), 12.21–12.34 (m, 1 H). MS ES+ 272 <i>m</i>/<i>z</i> [M + H]<sup>+</sup></div></div><div id="sec5_14_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> 4-(3-Chlorophenyl)-<i>N</i>-cyclopropyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-2-carboxamide (<b>10</b>)</h4><div class="NLM_p last">4-(3-Chlorophenyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-2-carboxylic acid (50 mg, 0.18 mmol), cyclopropylamine (25 μL, 0.367 mmol), <i>O</i>-(benzotriazol-1-yl)-<i>N</i>,<i>N</i>,<i>N</i>′,<i>N</i>′-tetramethyluronium hexafluorophosphate (83 mg, 0.22 mmol), and triethylamine (0.051 mL, 0.37 mmol) were dissolved in <i>N</i>,<i>N</i>-dimethylformamide (1 mL). The reaction was stirred at room temperature for 16 h. The reaction mixture was diluted with methanol, filtered, and purified by preparative HPLC to give 16 mg (11%) of the title compound over several steps from ethyl 1-(benzenesulfonyl)-4-chloro-pyrrolo[2,3-<i>b</i>]pyridine-2-carboxylate. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 0.56–0.63 (m, 2 H), 0.69–0.79 (m, 2 H), 2.87 (br d, <i>J</i> = 4.41 Hz, 1 H), 7.28 (d, <i>J</i> = 4.73 Hz, 1 H), 7.31 (s, 1 H), 7.57–7.66 (m, 2 H), 7.75 (d, <i>J</i> = 7.57 Hz, 1 H), 7.80 (s, 1 H), 8.41 (d, <i>J</i> = 4.73 Hz, 1 H), 8.58 (br d, <i>J</i> = 4.10 Hz, 1 H), 12.30 (br s, 1 H). MS ES+ 312 <i>m</i>/<i>z</i> [M + H]<sup>+</sup></div></div><div id="sec5_14_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> 4-(3-Chlorophenyl)-2-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine (<b>11</b>)</h4><div class="NLM_p last">1-(Benzenesulfonyl)-4-chloro-2-methyl-pyrrolo[2,3-<i>b</i>]pyridine (740 mg, 2.4 mmol), bis(pinacolato)diboron (796 mg, 3.1 mmol), and potassium acetate (710 mg, 7.2 mmol) were taken up in 1,4-dioxane (10 mL), and the reaction was put under nitrogen atmosphere. 2-Dicyclohexylphosphino-2′,6′-dimethoxybiphenyl (59 mg, 0.14 mmol) and palladium(II) acetate (16 mg, 0.07 mmol) were added, and the resulting mixture was stirred at 100 °C for 16 h. LC–MS showed ∼50% conversion to product and some dechlorinated starting material. More bis(pinacolato)diboron (400 mg, 1.5 mmol), 2-dicyclohexylphosphino-2′,6′-dimethoxybiphenyl (59 mg, 0.14 mmol), and palladium(II) acetate (16 mg, 0.07 mmol) were added, and the resulting mixture was stirred at 100 °C for 5 h. When cooled to room temperature the mixture was filtered, concentrated, and purified on a silica gel column eluted with 0–50% ethyl acetate in <i>n</i>-heptane to give 890 mg of 1-(benzenesulfonyl)-2-methyl-4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)pyrrolo[2,3-<i>b</i>]pyridine (MS (ES+) 399 <i>m</i>/<i>z</i> [M + H]<sup>+</sup>). A portion of this material (150 mg, 0.38 mmol), 1-bromo-3-chloro-benzene (72 mg, 0.38 mmol), potassium carbonate (156 g, 1.13 mmol), and bis(di<i>tert</i>-butyl(4-dimethylaminophenyl)phosphine)dichloropalladium(II) (13 mg, 0.02 mmol) were dissolved in acetonitrile (2 mL) and water (1 mL). The mixture was heated in a microwave reactor at 120 °C for 40 min. After cooling lithium hydroxide monohydrate (95 mg, 23 mmol) was added, and the mixture was heated in a microwave reactor at 100 °C for 1 h. The phases were separated, and the organic phase was filtered and purified by preparative HPLC to give 3 mg (3%) of the title compound over several steps from 1-(benzenesulfonyl)-4-chloro-2-methyl-pyrrolo[2,3-<i>b</i>]pyridine. <sup>1</sup>H NMR (500 MHz, methanol-<i>d</i><sub>4</sub>) δ ppm 2.49 (s, 3 H), 6.34 (s, 1 H), 7.14 (d, <i>J</i> = 5.04 Hz, 1 H), 7.43–7.48 (m, 1 H), 7.49–7.54 (m, 1 H), 7.68 (d, <i>J</i> = 7.57 Hz, 1 H), 7.73 (s, 1 H), 8.14 (d, <i>J</i> = 5.04 Hz, 1 H). MS (ES+) 243 <i>m</i>/<i>z</i> [M + H]<sup>+</sup></div></div><div id="sec5_14_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> 4-(3-Chlorophenyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-2-carbonitrile (<b>12</b>)</h4><div class="NLM_p last">4-(3-Chlorophenyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-2-carboxylic acid (500 mg, 1.8 mmol) was taken up in thionyl chloride (3 mL, 41 mmol), and the resulting mixture was stirred at 80 °C for 15 min. The mixture was concentrated, and the resulting residue was dissolved in tetrahydrofuran (5 mL) and then slowly added to aqueous ammonia (10 mL, 25%) at 0 °C. The cooling bath was removed, and the resulting mixture was stirred at room temperature for 30 min. The mixture was concentrated under reduced pressure to remove ammonia and tetrahydrofuran to give a yellow aqueous suspension. Ethyl acetate (5 mL) was added, and the mixture was stirred at room temperature for 10 min. The resulting precipitate was collected by suction filtration, washed with ethyl acetate and pentane, and dried to give 300 mg of <b>9</b> as a yellow solid. This material was taken up in phosphorus oxychloride (1 mL, 11 mmol), and the resulting mixture was stirred at 100 °C for 1.5 h. When cooled to room temperature, ice was added followed by saturated aqueous sodium bicarbonate and aqueous ammonia (25%). The formed precipitate was collected by suction filtration, washed with water, and dried. The solid was suspended in dichloromethane (5 mL), and the precipitate was collected by suction filtration, washed with dichloromethane, and dried. The solid was purified by preparative RP-HPLC with ammonia buffer to give 10 mg (1%) of the title compound as a solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 8.53 (d, <i>J</i> = 4.5 Hz, 1 H), 7.82–7.78 (m, 1 H), 7.78–7.73 (m, 1 H), 7.62–7.55 (m, 2 H), 7.51 (s, 1 H), 7.36 (d, <i>J</i> = 4.5 Hz, 1 H). MS ES+ <i>m</i>/<i>z</i> 254 [M + H]<sup>+</sup>.</div></div><div id="sec5_14_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> 5-[4-(3-Chlorophenyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-2-yl]oxazole (<b>13</b>)</h4><div class="NLM_p last"><i>n</i>-Butyllithium (2.8 mL, 2.5 M in hexanes, 7.0 mmol) was added slowly to a solution of diisopropylamine (1 mL, 7.1 mmol) in 2-methyltetrahydrofuran (15 mL) at 0 °C. The resulting mixture was stirred at 0 °C for 5 min and was then cooled to −78 °C. A solution of 1-(benzenesulfonyl)-4-chloro-pyrrolo[2,3-<i>b</i>]pyridine (1.5 g, 5.1 mmol) in 2-methyltetrahydrofuran (10 mL) was added dropwise. The resulting mixture was stirred at −78 °C for 20 min, and then the mixture was allowed to reach room temperature. The mixture was stirred at room temperature for 10 min to give a suspension and then cooled to −78 °C again. <i>N</i>,<i>N</i>-Dimethylformamide (1.2 mL, 16 mmol) was added dropwise, and the resulting mixture was stirred on the thawing cooling bath for 30 min and then at room temperature for 30 min. Saturated aqueous ammonium chloride (12 mL) was added, and the phases were separated. The aqueous phase was extracted with ethyl acetate (2 × 10 mL), and the combined organic phases were washed with hydrochloric acid (20 mL, 0.5 M) followed by brine (15 mL). The solution was dried over sodium sulfate, filtered, and concentrated to give the intermediate 1-(benzenesulfonyl)-4-chloro-pyrrolo[2,3-<i>b</i>]pyridine-2-carbaldehyde as an orange gum, which was used without any further purification. This material, (3-chlorophenyl)boronic acid (205 mg, 1.3 mmol), bis(triphenylphosphine)dichloropalladium(II) (38 mg, 0.06 mmol), and potassium carbonate (452 mg, 3.3 mmol) were taken up in 1,4-dioxane (5 mL) and water (1 mL), and the resulting mixture was stirred at 90 °C for 2 h. The reaction was cooled to room temperature, and brine (5 mL) was added. The mixture was extracted with ethyl acetate (2 × 5 mL). The combined organic phases were concentrated and purified on a silica gel column eluted with 0–40% ethyl acetate in <i>n</i>-heptane to give 200 mg of the intermediate 1-(benzenesulfonyl)-4-(3-chlorophenyl)pyrrolo[2,3-<i>b</i>]pyridine-2-carbaldehyde as a yellow solid. This material, potassium carbonate (70 mg, 0.50 mmol), and <i>p</i>-toluenesulfonylmethyl isocyanide (98 mg, 0.50 mmol) were taken up in absolute ethanol (10 mL), and the resulting mixture was stirred at 70 °C for 15 min. Methanol (5 mL) was added to dissolve the starting material, and the resulting mixture was stirred at 70 °C for 1 h. Lithium hydroxide monohydrate (63 mg, 1.5 mmol) and water (1 mL) were added, and the mixture was stirred at 70 °C for 1.5 h. The reaction was cooled to room temperature and concentrated. The resulting residue was taken up in ethyl acetate (10 mL) and water (5 mL). The phases were separated, and the aqueous phase was extracted with ethyl acetate (2 × 10 mL). The combined organic phases were washed with saturated aqueous sodium bicarbonate followed by brine. The solution was dried over sodium sulfate, filtered, and concentrated. The crude product was recrystallized from methanol to give 45 mg (3%) of the title compound over several steps from 1-(benzenesulfonyl)-4-chloro-pyrrolo[2,3-<i>b</i>]pyridine. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 6.94 (d, <i>J</i> = 2.0 Hz, 1 H), 7.28 (d, <i>J</i> = 5.0 Hz, 1 H), 7.64–7.54 (m, 2 H), 7.73 (s, 1 H), 7.82–7.75 (m, 2 H), 8.35 (d, <i>J</i> = 4.8 Hz, 1 H), 8.53 (s, 1 H), 12.61 (s, 1 H). MS ES+ <i>m</i>/<i>z</i> 296 [M + H]<sup>+</sup>.</div></div><div id="sec5_14_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> [4-(3-Chlorophenyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-2-yl]methanol (<b>14</b>)</h4><div class="NLM_p last">4-(3-Chlorophenyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-2-carboxylic acid (200 mg, 0.73 mmol) was dissolved in dichloromethane (15 mL). Lithium aluminum hydride (111 mg, 2.9 mmol) was added to the solution, and the mixture was stirred for 1 h. The reaction was cooled to 0 °C and quenched with water (0.1 mL) followed by aqueous sodium hydroxide (15%) (0.1 mL) and water (0.3 mL). After stirring for 1 h at room temperature, the mixture was filtered, and the solid was rinsed with ether. The combined organic phases were concentrated and purified by preparative RP-HPLC with ammonia buffer to give 12 mg (2%) of the title compound over several steps from ethyl 1-(benzenesulfonyl)-4-chloro-pyrrolo[2,3-<i>b</i>]pyridine-2-carboxylate. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 4.64 (s, 2 H), 5.33 (br s, 1 H), 6.47 (s, 1 H), 7.18 (d, <i>J</i> = 5.04 Hz, 1 H), 7.51–7.56 (m, 1 H), 7.56–7.63 (m, 1 H), 7.73 (d, <i>J</i> = 7.57 Hz, 1 H), 7.76 (s, 1 H), 8.23 (d, <i>J</i> = 4.73 Hz, 1 H), 11.77 (br s, 1 H). MS ES+ <i>m</i>/<i>z</i> 259 [M + H]<sup>+</sup>.</div></div><div id="sec5_14_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> 4-(1<i>H</i>-Pyrrolo[2,3-<i>b</i>]pyridin-4-yl)-2,3-dihydro-1,4-benzoxazine (<b>15</b>)</h4><div class="NLM_p last">4-Chloro-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine (100 mg, 0.66 mmol), 3,4-dihydro-2<i>H</i>-1,4-benzoxazine (177 mg, 1.3 mmol), (2-dicyclohexylphosphino-2′,4′,6′-triisopropyl-1,1′-biphenyl)[2-(2-aminophenyl)phenyl)]palladium(II) chloride (25 mg, 0.033 mmol), and sodium <i>tert</i>-butoxide (126 mg, 1.3 mmol) were taken up in 1,4-dioxane (2 mL), and the resulting mixture was stirred at 100 °C for 16 h in a sealed tube. The reaction was cooled to room temperature, and brine (2 mL) and ethyl acetate (3 mL) were added. The phases were separated, and the aqueous phase was extracted with ethyl acetate (3 mL). The combined organic phases were concentrated and purified by preparative RP-HPLC with ammonia buffer to give 37 mg (23%) of the title compound as a white solid. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 3.97–3.88 (m, 2 H), 4.29–4.18 (m, 2 H), 6.20 (dd, <i>J</i> = 1.6, 3.2 Hz, 1 H), 6.78–6.73 (m, 1 H), 6.80 (d, <i>J</i> = 5.4 Hz, 1 H), 6.87–6.82 (m, 1 H), 6.94–6.88 (m, 2 H), 7.34 (t, <i>J</i> = 2.8 Hz, 1 H), 8.08 (d, <i>J</i> = 5.4 Hz, 1 H), 11.60 (br. s., 1 H). MS (ES+) <i>m</i>/<i>z</i> 252 [M + H]<sup>+</sup>.</div></div><div id="sec5_14_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> 4-(2-Phenylpyrrolidin-1-yl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine (<b>16</b>)</h4><div class="NLM_p last">4-Chloro-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine (100 mg, 0.66 mmol) and 2-phenylpyrrolidine (289 mg, 2.0 mmol) were dissolved in <i>N</i>-methyl-2-pyrrolidone (1 mL). The mixture was heated in a microwave reactor at 210 °C for 1 h. After cooling, the mixture was diluted with methanol, filtered, and purified by preparative RP-HPLC with ammonia buffer to give 60 mg (33%) of the title compound. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 1.80–1.88 (m, 1 H), 1.88–1.96 (m, 1 H), 1.96–2.04 (m, 1 H), 2.35–2.46 (m, 1 H), 3.76–3.86 (m, 1 H), 4.06 (br t, <i>J</i> = 7.25 Hz, 1 H), 5.19 (br d, <i>J</i> = 6.62 Hz, 1 H), 5.87 (d, <i>J</i> = 5.36 Hz, 1 H), 6.42 (br s, 1 H), 7.00 (br s, 1 H), 7.17–7.22 (m, 1 H), 7.22–7.27 (m, 2 H), 7.27–7.36 (m, 2 H), 7.67 (d, <i>J</i> = 5.67 Hz, 1 H), 11.13 (br s, 1 H). MS (ES+) <i>m</i>/<i>z</i> 264 [M + H]<sup>+</sup>.</div></div><div id="sec5_14_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> 2-(1<i>H</i>-Pyrrolo[2,3-<i>b</i>]pyridin-4-yl)-3,4-dihydro-1<i>H</i>-isoquinoline (<b>17</b>)</h4><div class="NLM_p last">4-Chloro-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine (100 mg, 0.66 mmol) and 1,2,3,4-tetrahydroisoquinoline (0.25 mL, 2.0 mmol) were taken up in <i>N</i>-methyl-2-pyrrolidone (1 mL) and heated in a microwave reactor at 200 °C for 1 h. The reaction was cooled to room temperature and diluted with methanol (1 mL). The mixture was left standing at room temperature for 1 h. The resulting precipitate was collected by suction filtration, washed with methanol, and dried to give the crude product as a white solid. It was recrystallized from 2-propanol to give 27 mg (17%) of the title compound as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 2.99 (t, <i>J</i> = 5.8 Hz, 2 H), 3.80 (t, <i>J</i> = 5.8 Hz, 2 H), 4.63 (s, 2 H), 6.45 (d, <i>J</i> = 5.5 Hz, 1 H), 6.58 (d, <i>J</i> = 3.5 Hz, 1 H), 7.24–7.16 (m, 4 H), 7.29–7.24 (m, 1 H), 7.93 (d, <i>J</i> = 5.5 Hz, 1 H), 11.37 (br. s., 1 H). MS (ES+) <i>m</i>/<i>z</i> 250 [M + H]<sup>+</sup>.</div></div><div id="sec5_14_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> 2-Phenyl-4-(1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-4-yl)morpholine (<b>18</b>)</h4><div class="NLM_p last">4-Chloro-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine (107 mg, 0.70 mmol), 2-phenylmorpholine (149 mg, 0.91 mmol), and <i>N</i>,<i>N</i>-diisopropylethylamine (0.18 mL, 1.0 mmol) were taken up in <i>N</i>-methyl-2-pyrrolidone (0.6 mL), and the resulting mixture was heated in a microwave reactor at 220 °C for 1 h. The reaction was cooled to room temperature and diluted with methanol (1 mL). The resulting mixture was filtered and purified by preparative RP-HPLC with ammonia buffer to give 97 mg (50%) of the title compound as a light-yellow solid. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 2.80–2.89 (m, 1 H), 3.07 (td, <i>J</i> = 11.98, 3.15 Hz, 1 H), 3.83–3.97 (m, 3 H), 4.12 (dd, <i>J</i> = 11.35, 2.21 Hz, 1 H), 4.69–4.81 (m, 1 H), 6.47 (d, <i>J</i> = 5.36 Hz, 1 H), 6.51 (d, <i>J</i> = 2.84 Hz, 1 H), 7.26 (br. s., 1 H), 7.29–7.36 (m, 1 H), 7.38 (t, <i>J</i> = 7.41 Hz, 2 H), 7.48 (d, <i>J</i> = 7.25 Hz, 2 H), 7.96 (d, <i>J</i> = 5.67 Hz, 1 H), 11.45 (br. s., 1 H). MS (ES+) <i>m</i>/<i>z</i> 280 [M + H]<sup>+</sup>.</div></div><div id="sec5_14_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> 4-(2,3-Dihydropyrrolo[2,3-<i>b</i>]pyridin-1-yl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine (<b>19</b>)</h4><div class="NLM_p last">4-Chloro-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine (78 mg, 0.51 mmol), 2,3-dihydro-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine (74 mg, 0.61 mmol), (2-dicyclohexylphosphino-2′,4′,6′-triisopropyl-1,1′-biphenyl)[2-(2-aminophenyl)phenyl)]palladium(II) chloride (2.0 mg, 2.6 μmol), and lithium bis(trimethylsilyl)amide (1.21 mL, 1.06 M in hexanes, 1.3 mmol) were dissolved in 2-methyltetrahydrofuran (5 mL). The mixture was heated in a microwave reactor at 100 °C for 1 h. After cooling, the mixture was concentrated. The residue was dissolved in methanol, filtered, and purified by preparative RP-HPLC with ammonia buffer to give 8 mg (7%) of the title compound. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 3.21 (t, <i>J</i> = 8.35 Hz, 2 H), 4.48 (t, <i>J</i> = 8.35 Hz, 2 H), 6.65 (dd, <i>J</i> = 3.47, 1.58 Hz, 1 H), 6.80 (dd, <i>J</i> = 7.09, 5.20 Hz, 1 H), 7.28–7.33 (m, 1 H), 7.56 (dd, <i>J</i> = 7.25, 1.26 Hz, 1 H), 7.97–8.06 (m, 3 H), 11.50 (br s, 1 H). MS (ES+) <i>m</i>/<i>z</i> 237 [M + H]<sup>+</sup>.</div></div><div id="sec5_14_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> <i>N</i>-Ethyl-4-(2-phenylpyrrolidin-1-yl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-2-carboxamide (<b>20</b>)</h4><div class="NLM_p last">Ethyl 1-(benzenesulfonyl)-4-chloro-pyrrolo[2,3-<i>b</i>]pyridine-2-carboxylate (700 mg, 1.9 mmol) and 2-phenylpyrrolidine (706 mg, 4.8 mmol) were dissolved in <i>N</i>-methyl-2-pyrrolidone (2 mL), and the resulting mixture was heated in a microwave reactor at 160 °C for 1 h. After being cooled to room temperature, the mixture was diluted with ethyl acetate and water. The phases were separated, and the aqueous phase was extracted twice with ethyl acetate. The combined organic phases were concentrated, and the crude product was recrystallized from ethyl acetate to give 350 mg (53%) of ethyl 4-(2-phenylpyrrolidin-1-yl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-2-carboxylate. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 1.29 (t, <i>J</i> = 7.09 Hz, 3 H), 1.82–1.90 (m, 1 H), 1.90–1.98 (m, 1 H), 1.98–2.07 (m, 1 H), 2.37–2.47 (m, 1 H), 3.86 (br. s., 1 H), 4.13 (br. s., 1 H), 4.26 (q, <i>J</i> = 6.94 Hz, 2 H), 5.19 (br. s., 1 H), 5.89 (br. s., 1 H), 7.17–7.27 (m, 4 H), 7.28–7.33 (m, 2 H), 7.81 (d, <i>J</i> = 5.36 Hz, 1 H), 12.02 (br. s., 1 H). MS (ES+) <i>m</i>/<i>z</i> 336 [M + H]<sup>+</sup>. A portion of this material (215 mg, 0.64 mmol) was suspended in methanol (10 mL), tetrahydrofuran (5 mL), and water (2 mL), and lithium hydroxide monohydrate (108 mg, 2.6 mmol) was added. The reaction mixture was stirred at 50 °C for 16 h and then concentrated. The resulting crude product was dissolved in water, and concentrated hydrochloric acid was added to acidify the solution. The formed precipitate was isolated by suction filtration and dried to give 190 mg (96%) of 4-(2-phenylpyrrolidin-1-yl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-2-carboxylic acid. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 1.76–2.12 (m, 3 H), 2.35–2.47 (m, 1 H), 3.85 (br. s., 1 H), 4.11 (br. s., 1 H), 5.21 (br. s., 1 H), 5.90 (br. s., 1 H), 7.13 (br. s., 1 H), 7.18–7.26 (m, 3 H), 7.28–7.34 (m, 2 H), 7.80 (d, <i>J</i> = 5.67 Hz, 1 H), 11.88 (br. s., 1 H), 12.69 (br. s., 1 H). MS (ES+) <i>m</i>/<i>z</i> 308 [M + H]<sup>+</sup>. A portion of this material (120 mg, 0.39 mmol) was taken up in thionyl chloride (2 mL, 28 mmol), and the resulting mixture was stirred at 80 °C for 1 h. The reaction mixture was concentrated, and the resulting residue was taken up in tetrahydrofuran (10 mL) and cooled to 0 °C. Ethylamine (6 mL, 1 M in tetrahydrofuran, 6.0 mmol) was added, and the mixture was stirred at room temperature for 1 h. The formed precipitate was isolated by suction filtration, washed with tetrahydrofuran, and recrystallized from 2-propanol. Purification was by preparative RP-HPLC with ammonia buffer gave 48 mg (36%) of the title compound. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 1.14 (t, <i>J</i> = 7.25 Hz, 3 H), 1.81–1.90 (m, 1 H), 1.93–2.10 (m, 2 H), 2.39–2.48 (m, 1 H), 3.25–3.31 (m, 2 H), 3.88–4.03 (m, 1 H), 4.13–4.28 (m, 1 H), 5.04–5.17 (m, 1 H), 5.79 (d, <i>J</i> = 5.67 Hz, 1 H), 7.17–7.27 (m, 3 H), 7.27–7.37 (m, 3 H), 7.71 (d, <i>J</i> = 5.67 Hz, 1 H), 8.19–8.26 (m, 1 H), 11.52 (br. s., 1 H). MS (ES+) <i>m</i>/<i>z</i> 335 [M + H]<sup>+</sup>.</div></div><div id="sec5_14_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> <i>N</i>-Ethyl-4-[2-(3-pyridyl)pyrrolidin-1-yl]-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-2-carboxamide (<b>21</b>)</h4><div class="NLM_p last">Ethyl 1-(benzenesulfonyl)-4-chloro-pyrrolo[2,3-<i>b</i>]pyridine-2-carboxylate (1 g, 2.7 mmol) and 3-pyrrolidin-2-ylpyridine (1 g, 6.8 mmol) were dissolved in <i>N</i>-methyl-2-pyrrolidone (2 mL), and the resulting mixture was heated in a microwave reactor at 160 °C for 3 h. After being cooled to room temperature, the reaction mixture was diluted with water and ethyl acetate. The phases were separated, and the aqueous phase was extracted twice with ethyl acetate. The combined organic phases were diluted with methanol and washed with brine, dried over sodium sulfate, and concentrated. The residue was passed through a silica gel column (0–10% methanol in dichloromethane with 7 mM ammonia added) and evaporated. The resulting material was suspended in tetrahydrofuran (15 mL) and water (5 mL), and lithium hydroxide monohydrate (474 mg, 11 mmol) was added. The reaction mixture was stirred at 50 °C. After 90 min, methanol (10 mL) was added, and the stirring was continued at 50 °C for 16 h. The solvents were evaporated, and the resulting crude product was dissolved in water. Concentrated hydrochloric acid was added to acidify the solution, and the mixture was kept at 5 °C for 3 h. The formed precipitate was isolated by suction filtration, washed with water, and dried to give 680 mg of 4-[2-(3-pyridyl)pyrrolidin-1-yl]-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-2-carboxylic acid (MS (ES+) <i>m</i>/<i>z</i> 309 [M + H]<sup>+</sup>). A portion of this material (120 mg, 0.39 mmol) was taken up in thionyl chloride (2 mL, 27 mmol), and the reaction was stirred at 80 °C for 2.5 h. The mixture was concentrated, and the resulting residue was dissolved in tetrahydrofuran (10 mL) and cooled to 0 °C. Ethylamine (6 mL, 1 M in tetrahydrofuran, 6.0 mmol) was added, and the mixture was stirred at room temperature for 15 min. Dimethyl sulfoxide (1 mL) was added, and the resulting mixture was stirred at room temperature for 16 h. The formed precipitate was isolated by suction filtration, washed with tetrahydrofuran, and dried. The crude product was recrystallized from 2-propanol to give 27 mg (16%) of the title compound over several steps from ethyl 1-(benzenesulfonyl)-4-chloro-pyrrolo[2,3-<i>b</i>]pyridine-2-carboxylate. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 1.14 (s, 3 H), 1.84–2.13 (m, 3 H), 2.42–2.49 (m, 1 H), 3.24–3.31 (m, 2 H), 3.86–4.03 (m, 1 H), 4.17–4.30 (m, 1 H), 5.11–5.28 (m, 1 H), 5.72–5.91 (m, 1 H), 7.28–7.38 (m, 2 H), 7.58–7. 66 (m, 1 H), 7.70–7.77 (m, 1 H), 8.19–8.28 (m, 1 H), 8.39–8.46 (m, 1 H), 8.49–8.57 (m, 1 H), 11.26–11.90 (m, 1 H). MS (ES+) <i>m</i>/<i>z</i> 336 [M + H]<sup>+</sup>.</div></div><div id="sec5_14_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> <i>N</i>-(Oxetan-3-yl)-4-[2-(3-pyridyl)pyrrolidin-1-yl]-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-2-carboxamide (<b>22</b>)</h4><div class="NLM_p last">4-[2-(3-Pyridyl)pyrrolidin-1-yl]-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-2-carboxylic acid (120 mg, 0.39 mmol) was taken up in thionyl chloride (2 mL, 27 mmol), and the resulting mixture was stirred at 80 °C for 3 h. The mixture was concentrated, and the resulting residue was taken up in tetrahydrofuran (10 mL) and cooled to 0 °C. A solution of oxetan-3-amine (0.14 mL, 2.0 mmol) and <i>N</i>,<i>N</i>-diisopropylethylamine (0.34 mL, 2.0 mmol) in tetrahydrofuran (2 mL) was added, and the mixture was stirred at room temperature for 15 min. Dimethyl sulfoxide (1 mL) was added, and the resulting mixture was stirred at room temperature for 16 h. The mixture was concentrated, and the resulting residue was diluted with ethyl acetate (10 mL) stirred at room temperature for 15 min and then kept at 5 °C for 1 h. The resulting precipitate was filtered off, washed with ethyl acetate, and discarded. The filtrate was concentrated and purified by preparative RP-HPLC with ammonia buffer to give 48 mg (27%) of the title compound over several steps from ethyl 1-(benzenesulfonyl)-4-chloro-pyrrolo[2,3-<i>b</i>]pyridine-2-carboxylate. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 1.91 (br d, <i>J</i> = 2.5 Hz, 1 H), 1.99–2.16 (m, 2 H), 2.44–2.50 (m, 1 H), 3.97 (br d, 1 H), 4.26 (br s, 1 H), 4.58 (q, <i>J</i> = 6.0 Hz, 2 H), 4.82 (t, <i>J</i> = 6.8 Hz, 2 H), 4.96–5.11 (m, 1 H), 5.19 (br d, <i>J</i> = 5.7 Hz, 1 H), 5.80 (br d, <i>J</i> = 5.4 Hz, 1 H), 7.32 (dd, <i>J</i> = 4.9, 7.7 Hz, 1 H), 7.47 (br s, 1 H), 7.63 (d, <i>J</i> = 7.7 Hz, 1 H), 7.75 (d, <i>J</i> = 5.7 Hz, 1 H), 8.43 (dd, <i>J</i> = 1.6, 4.7 Hz, 1 H), 8.52 (d, <i>J</i> = 1.6 Hz, 1 H), 8.93 (br d, <i>J</i> = 6.9 Hz, 1 H), 11.67 (br s, 1 H). MS (ES+) <i>m</i>/<i>z</i> 364 [M + H]<sup>+</sup>.</div></div><div id="sec5_14_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> <i>N</i>-Ethyl-4-[2-(2-pyridyl)pyrrolidin-1-yl]-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-2-carboxamide (<b>23</b>)</h4><div class="NLM_p last">Ethyl 1-(benzenesulfonyl)-4-chloro-pyrrolo[2,3-<i>b</i>]pyridine-2-carboxylate (1.0 g, 2.7 mmol), 2-pyrrolidin-2-ylpyridine (1.02 g, 6.8 mmol) and <i>N</i>,<i>N</i>-diisopropylethylamine (0.49 mL, 2.7 mmol) were dissolved in <i>N</i>-methyl-2-pyrrolidone (2 mL). The mixture was heated in a microwave reactor at 180 °C for 1 h. Water and ethyl acetate were added. The phases were separated, and the organic phase was washed with water, dried over magnesium sulfate, and evaporated to give 800 mg of ethyl 4-[2-(2-pyridyl)pyrrolidin-1-yl]-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-2-carboxylate (MS (ES+) <i>m</i>/<i>z</i> 337 [M + H]<sup>+</sup>). A portion of this material (460 mg, 1.4 mmol) was taken up in tetrahydrofuran (5 mL), methanol (10 mL), and water (5 mL), and lithium hydroxide monohydrate (230 mg, 5.5 mmol) was added. The resulting mixture was stirred at 50 °C for 2 h. The solvents were evaporated, and the resulting crude product was dissolved in water. Concentrated hydrochloric acid was added to acidify the solution, and the mixture was kept at 5 °C for 3 h. The formed precipitate was isolated by suction filtration, washed with water, and dried to give 360 mg of 4-[2-(2-pyridyl)pyrrolidin-1-yl]-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-2-carboxylic acid (MS (ES+) <i>m</i>/<i>z</i> 309 [M + H]<sup>+</sup>). A portion of this material (130 mg, 0.42 mmol) was taken up in thionyl chloride (2 mL, 27 mmol), and the resulting mixture was stirred at 80 °C for 2.5 h. The mixture was concentrated, and the resulting residue was taken up in tetrahydrofuran (10 mL) and cooled to 0 °C. Ethylamine (5 mL, 2 M in tetrahydrofuran, 10 mmol) was added, and the mixture was stirred at room temperature for 15 min. The reaction mixture was concentrated and purified by preparative RP-HPLC with ammonia buffer to give 46 mg (24%) of the title compound over several steps from ethyl 1-(benzenesulfonyl)-4-chloro-pyrrolo[2,3-<i>b</i>]pyridine-2-carboxylate. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 1.15 (t, <i>J</i> = 7.1 Hz, 3 H), 1.96–2.08 (m, 3 H), 2.41–2.49 (m, 1 H), 3.25–3.33 (m, 2 H), 3.96 (br d, 1 H), 4.21 (br s, 1 H), 5.06–5.20 (m, 1 H), 5.80 (br d, 1 H), 7.20 (d, 1 H), 7.25 (t, 1 H), 7.32 (br s, 1 H), 7.69 (t, 1 H), 7.75 (d, 1 H), 8.24 (br s, 1 H), 8.57 (d, 1 H), 11.63 (br s, 1 H). MS (ES+) <i>m</i>/<i>z</i> 336 [M + H]<sup>+</sup>.</div></div><div id="sec5_14_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> <i>N</i>-Cyclopropyl-4-[2-(2-pyridyl)pyrrolidin-1-yl]-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-2-carboxamide (<b>24</b>)</h4><div class="NLM_p last">4-[2-(2-Pyridyl)pyrrolidin-1-yl]-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-2-carboxylic acid (120 mg, 0.39 mmol) was taken up in thionyl chloride (2 mL, 27 mmol), and the resulting mixture was stirred at 80 °C for 1 h. The mixture was concentrated, and the resulting residue was taken up in tetrahydrofuran (10 mL) and cooled to 0 °C. Cyclopropylamine (156 mg, 2.7 mmol) was added, and the mixture was stirred at room temperature for 15 min. Dimethyl sulfoxide (1 mL) was added, and the mixture was stirred at room temperature for 16 h. The formed precipitate was isolated by suction filtration, washed with tetrahydrofuran, and dried. The crude product was purified by preparative RP-HPLC with ammonia buffer to give 62 mg (33%) of the title compound over several steps from ethyl 1-(benzenesulfonyl)-4-chloro-pyrrolo[2,3-<i>b</i>]pyridine-2-carboxylate. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 0.56 (br s, 2 H), 0.73 (br d, <i>J</i> = 5.04 Hz, 2 H), 1.96–2.11 (m, 3 H), 2.40–2.49 (m, 1 H), 2.82 (br d, <i>J</i> = 3.78 Hz, 1 H), 3.96 (br s, 1 H), 4.20 (br s, 1 H), 5.12 (br s, 1 H), 5.79 (br s, 1 H), 7.19 (d, <i>J</i> = 7.88 Hz, 1 H), 7.22–7.28 (m, 1 H), 7.30 (br s, 1 H), 7.69 (br t, <i>J</i> = 7.09 Hz, 1 H), 7.73 (br d, <i>J</i> = 5.67 Hz, 1 H), 8.24 (br s, 1 H), 8.56 (br d, <i>J</i> = 4.10 Hz, 1 H), 11.50 (br s, 1 H). MS (ES+) <i>m</i>/<i>z</i> 348 [M + H]<sup>+</sup>.</div></div><div id="sec5_14_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> <i>N</i>-(Oxetan-3-yl)-4-[2-(2-pyridyl)pyrrolidin-1-yl]-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-2-carboxamide (<b>25</b>)</h4><div class="NLM_p last">4-[2-(2-Pyridyl)pyrrolidin-1-yl]-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-2-carboxylic acid (50 mg, 0.16 mmol) was taken up in thionyl chloride (1 mL, 14 mmol), and the resulting mixture was stirred at 80 °C for 2 h. The mixture was concentrated, and the resulting residue was taken up in 1,4-dioxane (2 mL). A solution of oxetan-3-amine (70 μL, 1.0 mmol) and <i>N</i>,<i>N</i>-diisopropylethylamine (0.14 mL, 0.81 mmol) in 1,4-dioxane (1 mL) was added, and the reaction mixture was stirred at room temperature for 1 h. The reaction mixture was concentrated and purified by preparative RP-HPLC with ammonia buffer to give 16 mg (20%) of the title compound over several steps from ethyl 1-(benzenesulfonyl)-4-chloro-pyrrolo[2,3-<i>b</i>]pyridine-2-carboxylate. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 1.94–2.15 (m, 3 H), 2.39–2.49 (m, 1 H), 4.00 (br s, 1 H), 4.24 (br s, 1 H), 4.55–4.62 (m, 2 H), 4.82 (t, <i>J</i> = 6.78 Hz, 2 H), 4.99–5.08 (m, 1 H), 5.14 (br s, 1 H), 5.80 (br s, 1 H), 7.21 (d, <i>J</i> = 7.88 Hz, 1 H), 7.23–7.28 (m, 1 H), 7.44 (br s, 1 H), 7.69 (br t, <i>J</i> = 7.57 Hz, 1 H), 7.74 (br d, <i>J</i> = 5.36 Hz, 1 H), 8.57 (br d, <i>J</i> = 4.73 Hz, 1 H), 8.92 (br d, <i>J</i> = 6.94 Hz, 1 H), 11.61 (br s, 1 H). MS (ES+) <i>m</i>/<i>z</i> 364 [M + H]<sup>+</sup>.</div></div><div id="sec5_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> 4-(3-Chlorophenyl)-1<i>H</i>-pyrazolo[3,4-<i>b</i>]pyridine (<b>26</b>)</h4><div class="NLM_p last">4-Chloro-1<i>H</i>-pyrazolo[3,4-<i>b</i>]pyridine (100 mg, 0.65 mmol), (3-chlorophenyl)boronic acid (122 mg, 0.78 mmol), bis(triphenylphosphine)dichloropalladium(II) (23 mg, 0.033 mmol), and potassium carbonate (270 mg, 1.95 mmol) were taken up in a mixture of 1,4-dioxane/H<sub>2</sub>O/EtOH (2 mL, 6:3:1), and the resulting mixture was heated in a microwave reactor at 150 °C for 30 min. The mixture was left standing at room temperature for 1 h, and the resulting precipitate was collected by suction filtration, washed with water, 1,4-dioxane, and pentane and finally dried to give 29 mg (19%) of the title compound as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 7.41 (d, <i>J</i> = 4.78 Hz, 1 H), 7.59–7.66 (m, 2 H), 7.78–7.88 (m, 1 H), 7.89 (s, 1 H), 8.32 (s, 1 H), 8.60 (d, <i>J</i> = 4.78 Hz, 1 H), 13.86 (br s, 1 H). MS (ES+) <i>m</i>/<i>z</i> 230 [M + H]<sup>+</sup>.</div></div><div id="sec5_15_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> 4-(2-Phenylpyrrolidin-1-yl)-1<i>H</i>-pyrazolo[3,4-<i>b</i>]pyridine (<b>27</b>)</h4><div class="NLM_p last">4-Chloro-1<i>H</i>-pyrazolo[3,4-<i>b</i>]pyridine (70 mg, 0.46 mmol) and 2-phenylpyrrolidine (134 mg, 0.91 mmol) were mixed with <i>N</i>-methyl-2-pyrrolidone (0.3 mL). The mixture was heated in a microwave reactor at 180 °C for 1 h. The mixture was diluted with methanol (3 mL) and stirred for 10 min. The solid was collected by suction filtration and washed with 2-propanol and pentane and finally air-dried to give 40 mg (33%) of the title compound. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 1.89 (br d, <i>J</i> = 9.46 Hz, 1 H), 1.96 (br d, <i>J</i> = 7.25 Hz, 1 H), 2.03 (br s, 1 H), 2.40–2.48 (m, 1 H), 3.79 (br s, 1 H), 4.07 (br s, 1 H), 5.23 (br s, 1 H), 5.87 (br s, 1 H), 7.18–7.27 (m, 3 H), 7.31 (br t, <i>J</i> = 7.41 Hz, 2 H), 7.88 (br s, 2 H), 13.01 (br s, 1 H). MS (ES+) <i>m</i>/<i>z</i> 265 [M + H]<sup>+</sup>.</div></div><div id="sec5_15_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> 4-[2-(3-Pyridyl)pyrrolidin-1-yl]-1<i>H</i>-pyrazolo[3,4-<i>b</i>]pyridine (<b>28</b>)</h4><div class="NLM_p last">4-Chloro-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidine (78 mg, 0.50 mmol) and 3-pyrrolidin-2-ylpyridine (110 mg, 0.76 mmol) were dissolved in <i>N</i>-methyl-2-pyrrolidone (1 mL). The mixture was heated in a microwave reactor at 220 °C for 30 min and then concentrated. The residue was dissolved in methanol, filtered, and purified by preparative RP-HPLC with acetic acid buffer to give 20 mg (11%) of the title compound. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 1.89–2.03 (m, 2 H), 2.05–2.13 (m, 1 H), 2.45–2.54 (m, 1 H), 3.82 (br s, 1 H), 4.14 (br s, 1 H), 5.32 (br s, 1 H), 5.88 (br s, 1 H), 7.33 (dd, <i>J</i> = 7.72, 4.89 Hz, 1 H), 7.64 (br d, <i>J</i> = 7.88 Hz, 1 H), 7.85–7.92 (m, 1 H), 7.97 (br s, 1 H), 8.45 (dd, <i>J</i> = 4.73, 1.26 Hz, 1 H), 8.49–8.56 (m, 1 H), 13.08 (br s, 1 H). MS (ES+) <i>m</i>/<i>z</i> 266 [M + H]<sup>+</sup>.</div></div><div id="sec5_15_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> 3-Phenyl-4-(1<i>H</i>-pyrazolo[3,4-<i>b</i>]pyridin-4-yl)morpholine (<b>29</b>)</h4><div class="NLM_p last">4-Chloro-1<i>H</i>-pyrazolo[3,4-<i>b</i>]pyridine (100 mg, 0.65 mmol), 3-phenylmorpholine (128 mg, 0.78 mmol), and triethylamine (108 mL, 0.78 mmol) were dissolved in <i>N</i>-methyl-2-pyrrolidone (1 mL). The mixture was heated in a microwave reactor at 180 °C for 30 min. After cooling, the mixture was diluted with methanol, filtered, and purified by preparative RP-HPLC with ammonia buffer to give 30 mg (16%) of the title compound. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 3.64–3.85 (m, 2 H), 3.90 (br d, <i>J</i> = 12.30 Hz, 1 H), 4.05 (td, <i>J</i> = 7.88, 3.78 Hz, 2 H), 4.17–4.27 (m, 1 H), 5.34 (br s, 1 H), 6.39 (d, <i>J</i> = 5.67 Hz, 1 H), 7.15–7.25 (m, 1 H), 7.25–7.36 (m, 4 H), 8.04 (d, <i>J</i> = 5.67 Hz, 1 H), 8.14 (s, 1 H), 13.24 (br s, 1 H). MS (ES+) <i>m</i>/<i>z</i> 281 [M + H]<sup>+</sup>.</div></div><div id="sec5_15_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> 4-[2-(1-Methoxy-1-methyl-ethyl)pyrrolidin-1-yl]-1<i>H</i>-pyrazolo[3,4-<i>b</i>]pyridine acetate salt (<b>30</b>)</h4><div class="NLM_p last">4-Chloro-1<i>H</i>-pyrazolo[3,4-<i>b</i>]pyridine (100 mg, 0.65 mmol), 2-(1-methoxy-1-methyl-ethyl)pyrrolidine (121 mg, 0.85 mmol), and triethylamine (180 mL, 1.3 mmol) were dissolved in <i>N</i>-methyl-2-pyrrolidone (1 mL). The mixture was heated in a microwave reactor at 230 °C for 1 h. The reaction mixture was purified by preparative RP-HPLC with acetic acid buffer to give 20 mg (10%) of the title compound. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 1.10 (s, 3 H), 1.15 (s, 3 H), 1.75–1.90 (m, 2 H), 1.91 (s, 3 H), 2.01–2.19 (m, 2 H), 3.09 (s, 3 H), 3.35 (br s, 1 H), 3.52–3.77 (m, 1 H), 3.82–4.00 (m, 1 H), 4.28 (br d, <i>J</i> = 7.92 Hz, 1 H), 6.38 (d, <i>J</i> = 6.16 Hz, 1 H), 7.94 (d, <i>J</i> = 5.72 Hz, 1 H), 8.14 (s, 1 H). MS (ES+) <i>m</i>/<i>z</i> 261 [M + H]<sup>+</sup>.</div></div><div id="sec5_15_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> <i>N</i>-Ethyl-4-[2-(1-methoxy-1-methyl-ethyl)pyrrolidin-1-yl]-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-2-carboxamide (<b>31</b>)</h4><div class="NLM_p last"><i>n</i>-Butyllithium (3.2 mL, 2.5 M in hexanes, 7.9 mmol) was added slowly to a solution of diisopropylamine (1.1 mL, 7.9 mmol) in 2-methyltetrahydrofuran (10 mL) at 0 °C. The resulting mixture was stirred at 0 °C for 5 min and was then cooled to −78 °C. A solution of 1-(benzenesulfonyl)-4-chloro-pyrrolo[2,3-<i>b</i>]pyridine (1.8 g, 6.2 mmol) in 2-methyltetrahydrofuran (10 mL) was added dropwise. The resulting mixture was stirred at −78 °C for 15 min and was then allowed to warm to room temperature. After stirring at room temperature for 10 min, the reaction mixture was cooled to −78 °C again. Carbon dioxide (from dry ice) was bubbled through the reaction mixture and stirring was continued on the thawing ice bath for 1 h. Saturated aqueous ammonium chloride, water, and ethyl acetate were added. The formed precipitate was filtered off and washed with cold 2-propanol. The aqueous phase of the combined filtrate was made acidic by the addition of concentrated hydrochloric acid and left to rest overnight. The formed precipitate was collected by suction filtration, washed with cold 2-propanol and pentane, and dried. The two solids were combined to give 1.9 g of 1-(benzenesulfonyl)-4-chloro-pyrrolo[2,3-<i>b</i>]pyridine-2-carboxylic acid (MS (ES+) <i>m</i>/<i>z</i> 338 [M + H]<sup>+</sup>). A portion of this material (1.5 g, 4.4 mmol) was dissolved in thionyl chloride (3.68 mL, 51 mmol), and the mixture was heated to 80 °C for 20 min. The reaction mixture was concentrated, and the crude product was dissolved in dichloromethane (10 mL). Ethylamine (4.45 mL, 2 M in tetrahydrofuran, 8.9 mmol) was slowly added. The reaction mixture was stirred at room temperature for 30 min and then concentrated under vacuum. Ethanol and water were added to the crude product, and the formed precipitate was isolated by filtration, washed with 2-propanol and pentane, and dried to give 800 mg of 4-chloro-<i>N</i>-ethyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-2-carboxamide (MS (ES+) <i>m</i>/<i>z</i> 224 [M + H]<sup>+</sup>). A portion of this material (65 mg, 0.29 mmol), 2-(1-methoxy-1-methyl-ethyl)pyrrolidine (42 mg, 0.29 mmol), and <i>N</i>,<i>N</i>-diisopropylethylamine (80 μL, 0.45 mmol) were dissolved in <i>N</i>-methyl-2-pyrrolidone (1 mL). The mixture was heated in a microwave reactor at 220 °C for 1 h and then at 230 °C for 1 h. The reaction mixture was filtered and then purified by preparative RP-HPLC with ammonia buffer to give 10 mg (7%) of the title compound over several steps from 1-(benzenesulfonyl)-4-chloro-pyrrolo[2,3-<i>b</i>]pyridine. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 1.08 (s, 3 H), 1.12–1.18 (m, 6 H), 1.88 (br s, 2 H), 2.08 (br d, 2 H), 3.12 (s, 3 H), 3.25–3.32 (m, 2 H), 3.74–3.83 (m, 1 H), 3.86–4.01 (m, 1 H), 4.32 (br d, <i>J</i> = 7.25 Hz, 1 H), 6.44 (d, <i>J</i> = 6.31 Hz, 1 H), 7.33 (s, 1 H), 7.84 (d, <i>J</i> = 5.67 Hz, 1 H), 8.27 (br t, <i>J</i> = 5.36 Hz, 1 H), 11.53 (br s, 1 H). MS (ES+) <i>m</i>/<i>z</i> 331 [M + H]<sup>+</sup>.</div></div><div id="sec5_15_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> <i>N</i>-Ethyl-4-[(3<i>S</i>)-3-methylmorpholin-4-yl]-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-2-carboxamide Acetic Acid Salt (<b>32</b>)</h4><div class="NLM_p last">4-Chloro-<i>N</i>-ethyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-2-carboxamide (100 mg, 0.45 mmol), (3<i>S</i>)-3-methylmorpholine (59 mg, 0.58 mmol), and triethylamine (0.12 mL, 0.89 mmol) were dissolved in <i>N</i>-methyl-2-pyrrolidone (1 mL). The mixture was heated in a microwave reactor at 220 °C for 1 h. The reaction mixture was filtered and then purified in two steps by preparative RP-HPLC with ammonia buffer followed by preparative RP-HPLC with acetic acid buffer to give 5 mg (3%) of the title compound over several steps from 1-(benzenesulfonyl)-4-chloro-pyrrolo[2,3-<i>b</i>]pyridine. <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>): δ ppm 1.24–1.33 (m, 6 H), 1.98 (s, 9 H), 3.37–3.51 (m, 2 H), 3.54–3.64 (m, 2 H), 3.71–3.79 (m, 1 H), 3.82 (br d, <i>J</i> = 11.44 Hz, 1 H), 3.94 (dd, <i>J</i> = 11.44, 2.64 Hz, 1 H), 4.05 (br d, <i>J</i> = 11.44 Hz, 1 H), 4.40 (br d, <i>J</i> = 5.72 Hz, 1 H), 6.53 (d, <i>J</i> = 5.72 Hz, 1 H), 7.33 (s, 1 H), 8.03 (d, <i>J</i> = 5.72 Hz, 1 H). MS (ES+) <i>m</i>/<i>z</i> 289 [M + H]<sup>+</sup>. <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) d ppm 12.654, 14.966, 33.660, 42.688, 50.418, 66.241, 70.599, 101.601, 101.996, 109.18, 128.800, 146.211, 150.253, 150.826, 160.434.</div></div><div id="sec5_15_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> <i>N</i>-Ethyl-4-(3-phenylmorpholin-4-yl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-2-carboxamide (<b>33</b>)</h4><div class="NLM_p last">4-Chloro-<i>N</i>-ethyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-2-carboxamide (100 mg, 0.45 mmol), 3-phenylmorpholine (95 mg, 0.58 mmol), and triethylamine (0.12 mL, 0.89 mmol) were dissolved in <i>N</i>-methyl-2-pyrrolidone (1 mL). The mixture was heated in a microwave reactor at 220 °C for 1 h and at 230 °C for 1 h. The reaction mixture was filtered and then purified by preparative RP-HPLC with ammonia buffer to give 5 mg (2%) of the title compound over several steps from 1-(benzenesulfonyl)-4-chloro-pyrrolo[2,3-<i>b</i>]pyridine. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 1.16 (t, <i>J</i> = 7.26 Hz, 3 H), 3.26–3.32 (m, 2 H), 3.36–3.43 (m, 1 H), 3.74 (dt, <i>J</i> = 12.43, 3.69 Hz, 1 H), 3.81–4.07 (m, 4 H), 4.99 (br s, 1 H), 6.36 (d, <i>J</i> = 5.72 Hz, 1 H), 7.16–7.44 (m, 6 H), 7.92 (d, <i>J</i> = 5.72 Hz, 1 H), 8.39 (br t, <i>J</i> = 5.28 Hz, 1 H), 11.62 (br s, 1 H). MS (ES+) <i>m</i>/<i>z</i> 351 [M + H]<sup>+</sup>.</div></div><div id="sec5_15_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> 4-(3-Cyclopropylmorpholin-4-yl)-<i>N</i>-ethyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-2-carboxamide (<b>34</b>)</h4><div class="NLM_p last">4-Chloro-<i>N</i>-ethyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-2-carboxamide (120 mg, 0.54 mmol), 3-cyclopropylmorpholine (273 mg, 2.2 mmol), and <i>N</i>,<i>N</i>-diisopropylethylamine (0.37 mL, 2.2 mmol) were mixed in pyridine (0.5 mL) and heated in a microwave reactor at 230 °C for 4 h. Methanol was added, and the reaction mixture was filtered and then purified in two steps by preparative RP-HPLC with acetic acid buffer followed by preparative RP-HPLC with ammonia buffer to give 15 mg (6%) of the title compound over several steps from 1-(benzenesulfonyl)-4-chloro-pyrrolo[2,3-<i>b</i>]pyridine. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm −0.26 to −0.19 (m, 1 H), 0.12–0.23 (m, 2 H), 0.33–0.41 (m, 1 H), 1.15 (t, <i>J</i> = 7.25 Hz, 3 H), 1.43–1.51 (m, 1 H), 3.28–3.33 (m, 2 H), 3.34–3.38 (m, 1 H), 3.55 (br d, 1 H), 3.61–3.66 (m, 2 H), 3.84 (dd, <i>J</i> = 11.03, 2.84 Hz, 1 H), 3.90–3.95 (m, 1 H), 3.98–4.03 (m, 1 H), 6.46 (d, <i>J</i> = 5.36 Hz, 1 H), 7.14 (s, 1 H), 8.02 (d, <i>J</i> = 5.36 Hz, 1 H), 8.30 (t, <i>J</i> = 5.52 Hz, 1 H), 11.58 (br s, 1 H). MS (ES+) <i>m</i>/<i>z</i> 315 [M + H]<sup>+</sup>.</div></div><div id="sec5_15_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> <i>N</i>-Ethyl-1-methyl-4-[(3<i>S</i>)-3-methylmorpholin-4-yl]-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-2-carboxamide (<b>35</b>)</h4><div class="NLM_p last"><i>N</i>-Ethyl-4-[(3<i>S</i>)-3-methylmorpholin-4-yl]-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-2-carboxamide (68 mg, 0.24 mmol) was dissolved in tetrahydrofuran (2 mL) and cooled to 0 °C. Sodium hydride (60%) (9.4 mg, 0.24 mmol) was added, followed by iodomethane (15 μL, 0.24 mmol). After stirring for 10 min at 0 °C, water was added slowly. The aqueous phase was extracted three times with ethyl acetate. The combined organic phases were dried and separated via chromatography to obtain 51 mg (68%) of the title compound. <sup>1</sup>H NMR (400 MHz, DMSO-d6) δ [ppm]: 1.11 (t, <i>J</i> = 7.09 Hz, 3H), 1.26 (d, <i>J</i> = 6.84 Hz, 3H), 3.22–3.31 (m, 2H), 3.42–3.61 (m, 2H), 3.64–3.79 (m, 2H), 3.97 (m, 5H), 4.45–4.60 (m, 1H), 6.40 (d, <i>J</i> = 7.35 Hz, 1H), 7.10 (s, 1H), 7.81 (t, <i>J</i> = 6.08 Hz, 1H), 7.86 (d, <i>J</i> = 7.10 Hz, 1H). MS (ES+) <i>m</i>/<i>z</i> 303 [M + H]<sup>+</sup>. <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): d ppm 14.070, 15.447, 33.092, 38.213, 42.390, 49.996, 66.162, 70.302, 95.852, 103.154, 111.719, 135.521, 141.779, 147.527, 152.375, 164.280.</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i63"><a href="/doi/suppl/10.1021/acs.jmedchem.7b01884" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_64821" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_64821" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.7b01884" class="ext-link">10.1021/acs.jmedchem.7b01884</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Selectivity data of <b>10</b> and <b>24</b> in a 97-membered kinome panel (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01884/suppl_file/jm7b01884_si_001.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Selectivity data of <b>24</b> in a 215-membered ATP-ase panel (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01884/suppl_file/jm7b01884_si_002.xlsx" class="ext-link">XLSX</a>)</p></li><li><p class="inline">Selectivity data of <b>32</b> (BAY-707) and <b>35</b> (BAY-604) in a 20-membered NUDIX panel (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01884/suppl_file/jm7b01884_si_003.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Table of SMILES and data for all compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01884/suppl_file/jm7b01884_si_004.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Data collection and refinement statistics, data used for computational modeling, and methods for NUDIX selectivity assessment (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01884/suppl_file/jm7b01884_si_005.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01884/suppl_file/jm7b01884_si_001.csv">jm7b01884_si_001.csv (6.85 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01884/suppl_file/jm7b01884_si_002.xlsx">jm7b01884_si_002.xlsx (168.34 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01884/suppl_file/jm7b01884_si_003.csv">jm7b01884_si_003.csv (1.21 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01884/suppl_file/jm7b01884_si_004.csv">jm7b01884_si_004.csv (2.82 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01884/suppl_file/jm7b01884_si_005.pdf">jm7b01884_si_005.pdf (1.43 MB)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Accession Codes in the PDB for human MTH1 with bound ligands: compound <b>1</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6F20">6F20</a>), compound 7 (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6F1X">6F1X</a>), compound <b>16</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6F23">6F23</a>), and compound <b>29</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6F22">6F22</a>).</p><div class="testing" data-doi="10.1021/acs.jmedchem.7b01884" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_34032" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_34032" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jenny Viklund</span> - <span class="hlFld-Affiliation affiliation">Sprint
Bioscience AB, Novum, 14157 Huddinge, Sweden</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4087-9721" title="Orcid link">http://orcid.org/0000-0002-4087-9721</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#d1bbb4bfbfa8ffa7b8babda4bfb591a2a1a3b8bfa5b3b8bea2b2b8b4bfb2b4ffb2bebcf1bbb4bfbfa8ffa7b8babda4bfb591b6bcb0b8bdffb2bebc"><span class="__cf_email__" data-cfemail="7a101f141403540c1311160f141e3a090a0813140e1813150919131f14191f54191517">[email protected]</span> <span class="__cf_email__" data-cfemail="cca6a9a2a2b5e2baa5a7a0b9a2a88caba1ada5a0e2afa3a1">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fredrik Rahm</span> - <span class="hlFld-Affiliation affiliation">Sprint
Bioscience AB, Novum, 14157 Huddinge, Sweden</span>; 
    <span>Present Address:
                        Awapatent AB, Jakobsbergsgatan 36, 111 44 Stockholm</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lionel Trésaugues</span> - <span class="hlFld-Affiliation affiliation">Sprint
Bioscience AB, Novum, 14157 Huddinge, Sweden</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Manuel Ellermann</span> - <span class="hlFld-Affiliation affiliation">Bayer
AG, Muellerstrasse 178, 13353 Berlin, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Anja Giese</span> - <span class="hlFld-Affiliation affiliation">Bayer
AG, Muellerstrasse 178, 13353 Berlin, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ulrika Ericsson</span> - <span class="hlFld-Affiliation affiliation">Sprint
Bioscience AB, Novum, 14157 Huddinge, Sweden</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rickard Forsblom</span> - <span class="hlFld-Affiliation affiliation">Sprint
Bioscience AB, Novum, 14157 Huddinge, Sweden</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tobias Ginman</span> - <span class="hlFld-Affiliation affiliation">Sprint
Bioscience AB, Novum, 14157 Huddinge, Sweden</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Judith Günther</span> - <span class="hlFld-Affiliation affiliation">Bayer
AG, Muellerstrasse 178, 13353 Berlin, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kenth Hallberg</span> - <span class="hlFld-Affiliation affiliation">Sprint
Bioscience AB, Novum, 14157 Huddinge, Sweden</span>; 
    <span>Present Address:
                        SARomics Biostructures, Medicon Village, 223 81 Lund</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Johan Lindström</span> - <span class="hlFld-Affiliation affiliation">Sprint
Bioscience AB, Novum, 14157 Huddinge, Sweden</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lars Boukharta Persson</span> - <span class="hlFld-Affiliation affiliation">Sprint
Bioscience AB, Novum, 14157 Huddinge, Sweden</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Camilla Silvander</span> - <span class="hlFld-Affiliation affiliation">Sprint
Bioscience AB, Novum, 14157 Huddinge, Sweden</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Antoine Talagas</span> - <span class="hlFld-Affiliation affiliation">Sprint
Bioscience AB, Novum, 14157 Huddinge, Sweden</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Laura Díaz-Sáez</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, Nuffield Department of Medicine, University of Oxford, Oxford OX3 7DQ, United Kingdom</span>; 
    <span class="hlFld-Affiliation affiliation">Target
Discovery Institute, Nuffield Department of Medicine, University of Oxford, Oxford OX3 7FZ, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Oleg Fedorov</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, Nuffield Department of Medicine, University of Oxford, Oxford OX3 7DQ, United Kingdom</span>; 
    <span class="hlFld-Affiliation affiliation">Target
Discovery Institute, Nuffield Department of Medicine, University of Oxford, Oxford OX3 7FZ, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kilian V. M. Huber</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, Nuffield Department of Medicine, University of Oxford, Oxford OX3 7DQ, United Kingdom</span>; 
    <span class="hlFld-Affiliation affiliation">Target
Discovery Institute, Nuffield Department of Medicine, University of Oxford, Oxford OX3 7FZ, United Kingdom</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1103-5300" title="Orcid link">http://orcid.org/0000-0002-1103-5300</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ioanna Panagakou</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, Nuffield Department of Medicine, University of Oxford, Oxford OX3 7DQ, United Kingdom</span>; 
    <span class="hlFld-Affiliation affiliation">Target
Discovery Institute, Nuffield Department of Medicine, University of Oxford, Oxford OX3 7FZ, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paulina Siejka</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, Nuffield Department of Medicine, University of Oxford, Oxford OX3 7DQ, United Kingdom</span>; 
    <span class="hlFld-Affiliation affiliation">Target
Discovery Institute, Nuffield Department of Medicine, University of Oxford, Oxford OX3 7FZ, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mátyás Gorjánácz</span> - <span class="hlFld-Affiliation affiliation">Bayer
AG, Muellerstrasse 178, 13353 Berlin, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8398-410X" title="Orcid link">http://orcid.org/0000-0002-8398-410X</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Marcus Bauser</span> - <span class="hlFld-Affiliation affiliation">Bayer
AG, Muellerstrasse 178, 13353 Berlin, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Martin Andersson</span> - <span class="hlFld-Affiliation affiliation">Sprint
Bioscience AB, Novum, 14157 Huddinge, Sweden</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>F.R., Sprint Bioscience project coordinator; F.R., J.V, R.F, T.G, J.L, M.E., A.G., J.G., M.B., medicinal chemistry; J.V., L.B.P. computational chemistry; L.T., K.H., A.T., crystallography; U.E., C.S., protein science and screening; M.A., target selection; M.G., pharmacology, Bayer project coordinator; O.F., L.D.S., I.P., P.S., K.V.M.H., expression and purification of NUDIX selectivity panel proteins and TSA measurements. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d46e4760-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i68">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_51052" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_51052" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We acknowledge the European Synchrotron Radiation Facility for provision of synchrotron radiation facilities and the staff of beamlines ID23-1 and ID29 for their support. We also thank Santiago Parpal for support in proof-reading and preparing the manuscript. SGC is a registered charity (number 1097737) that receives funds from AbbVie, Bayer Pharma AG, Boehringer Ingelheim, Canada Foundation for Innovation, Eshelman Institute for Innovation, Genome Canada, Innovative Medicines Initiative (EU/EFPIA) [ULTRA-DD grant no. 115766], Janssen, Merck KGaA Darmstadt Germany, MSD, Novartis Pharma AG, Ontario Ministry of Economic Development and Innovation, Pfizer, São Paulo Research Foundation-FAPESP, Takeda, and Wellcome [106169/ZZ14/Z].</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">ABBREVIATIONS</td></tr><tr><td class="NLM_term">ITC</td><td class="NLM_def"><p class="first last">isothermal titration calorimetry</p></td></tr><tr><td class="NLM_term">NUDIX</td><td class="NLM_def"><p class="first last">NUcleoside DIphosphate linked to another moiety X</p></td></tr><tr><td class="NLM_term">MTH1</td><td class="NLM_def"><p class="first last">MutT Homolog1</p></td></tr><tr><td class="NLM_term">TSA</td><td class="NLM_def"><p class="first last">thermal shift assay</p></td></tr><tr><td class="NLM_term">CETSA</td><td class="NLM_def"><p class="first last">cellular thermal shift assay</p></td></tr><tr><td class="NLM_term">HAC</td><td class="NLM_def"><p class="first last">heavy atom count</p></td></tr><tr><td class="NLM_term">PLS</td><td class="NLM_def"><p class="first last">partial-least-squares</p></td></tr><tr><td class="NLM_term">DFT</td><td class="NLM_def"><p class="first last">density functional theory</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i70">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78638" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78638" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 56 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ellermann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eheim, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahm, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viklund, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guenther, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ericsson, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forsblom, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ginman, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindström, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silvander, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trésaugues, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giese, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunse, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neuhaus, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiske, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quanz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glasauer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak-Reppel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bader, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irlbacher, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Queisser, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauser, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haegebarth, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorjánácz, M.</span></span> <span> </span><span class="NLM_article-title">Novel Class of Potent and Cellularly Active Inhibitors Devalidates MTH1 as Broad-Spectrum Cancer Target</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">1986</span>– <span class="NLM_lpage">1992</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00370</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00370" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=1986-1992&issue=8&author=M.+Ellermannauthor=A.+Eheimauthor=F.+Rahmauthor=J.+Viklundauthor=J.+Guentherauthor=M.+Anderssonauthor=U.+Ericssonauthor=R.+Forsblomauthor=T.+Ginmanauthor=J.+Lindstr%C3%B6mauthor=C.+Silvanderauthor=L.+Tr%C3%A9sauguesauthor=A.+Gieseauthor=S.+Bunseauthor=R.+Neuhausauthor=J.+Weiskeauthor=M.+Quanzauthor=A.+Glasauerauthor=K.+Nowak-Reppelauthor=B.+Baderauthor=H.+Irlbacherauthor=H.+Meyerauthor=N.+Queisserauthor=M.+Bauserauthor=A.+Haegebarthauthor=M.+Gorj%C3%A1n%C3%A1cz&title=Novel+Class+of+Potent+and+Cellularly+Active+Inhibitors+Devalidates+MTH1+as+Broad-Spectrum+Cancer+Target&doi=10.1021%2Facschembio.7b00370"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00370&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00370%26sid%3Dliteratum%253Aachs%26aulast%3DEllermann%26aufirst%3DM.%26aulast%3DEheim%26aufirst%3DA.%26aulast%3DRahm%26aufirst%3DF.%26aulast%3DViklund%26aufirst%3DJ.%26aulast%3DGuenther%26aufirst%3DJ.%26aulast%3DAndersson%26aufirst%3DM.%26aulast%3DEricsson%26aufirst%3DU.%26aulast%3DForsblom%26aufirst%3DR.%26aulast%3DGinman%26aufirst%3DT.%26aulast%3DLindstr%25C3%25B6m%26aufirst%3DJ.%26aulast%3DSilvander%26aufirst%3DC.%26aulast%3DTr%25C3%25A9saugues%26aufirst%3DL.%26aulast%3DGiese%26aufirst%3DA.%26aulast%3DBunse%26aufirst%3DS.%26aulast%3DNeuhaus%26aufirst%3DR.%26aulast%3DWeiske%26aufirst%3DJ.%26aulast%3DQuanz%26aufirst%3DM.%26aulast%3DGlasauer%26aufirst%3DA.%26aulast%3DNowak-Reppel%26aufirst%3DK.%26aulast%3DBader%26aufirst%3DB.%26aulast%3DIrlbacher%26aufirst%3DH.%26aulast%3DMeyer%26aufirst%3DH.%26aulast%3DQueisser%26aufirst%3DN.%26aulast%3DBauser%26aufirst%3DM.%26aulast%3DHaegebarth%26aufirst%3DA.%26aulast%3DGorj%25C3%25A1n%25C3%25A1cz%26aufirst%3DM.%26atitle%3DNovel%2520Class%2520of%2520Potent%2520and%2520Cellularly%2520Active%2520Inhibitors%2520Devalidates%2520MTH1%2520as%2520Broad-Spectrum%2520Cancer%2520Target%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26issue%3D8%26spage%3D1986%26epage%3D1992%26doi%3D10.1021%2Facschembio.7b00370" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liou, M.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storz, P.</span></span> <i>Free Radical Res.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">479</span>, <span class="refDoi"> DOI: 10.3109/10715761003667554</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.3109%2F10715761003667554" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=20370557" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BC3cXkvVyhu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Liou%2C+M.-Y.%3B+Storz%2C+P.+Free+Radical+Res.+2010%2C+44%2C+479%2C+10.3109%2F10715761003667554"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Reactive oxygen species in cancer</span></div><div class="casAuthors">Liou, Geou-Yarh; Storz, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Free Radical Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">479-496</span>CODEN:
                <span class="NLM_cas:coden">FRARER</span>;
        ISSN:<span class="NLM_cas:issn">1071-5762</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Elevated rates of reactive oxygen species (ROS) have been detected in almost all cancers, where they promote many aspects of tumor development and progression.  However, tumor cells also express increased levels of antioxidant proteins to detoxify from ROS, suggesting that a delicate balance of intracellular ROS levels is required for cancer cell function.  Further, the radical generated, the location of its generation, as well as the local concn. is important for the cellular functions of ROS in cancer.  A challenge for novel therapeutic strategies will be the fine tuning of intracellular ROS signalling to effectively deprive cells from ROS-induced tumor promoting events, towards tipping the balance to ROS-induced apoptotic signalling.  Alternatively, therapeutic antioxidants may prevent early events in tumor development, where ROS are important.  However, to effectively target cancer cells specific ROS-sensing signalling pathways that mediate the diverse stress-regulated cellular functions need to be identified.  This review discusses the generation of ROS within tumor cells, their detoxification, their cellular effects, as well as the major signalling cascades they utilize, but also provides an outlook on their modulation in therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwtVSe56knA7Vg90H21EOLACvtfcHk0ljqOPK5rKpXCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXkvVyhu78%253D&md5=74d17b013387120ba165302bd7c282df</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.3109%2F10715761003667554&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F10715761003667554%26sid%3Dliteratum%253Aachs%26aulast%3DLiou%26aufirst%3DM.-Y.%26aulast%3DStorz%26aufirst%3DP.%26jtitle%3DFree%2520Radical%2520Res.%26date%3D2010%26volume%3D44%26spage%3D479%26doi%3D10.3109%2F10715761003667554" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klaunig, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamendulis, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hocevar, B. A.</span></span> <span> </span><span class="NLM_article-title">Oxidative Stress and Oxidative Damage in Carcinogenesis</span>. <i>Toxicol. Pathol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>38</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">96</span>– <span class="NLM_lpage">109</span>, <span class="refDoi"> DOI: 10.1177/0192623309356453</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1177%2F0192623309356453" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=20019356" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjt1ajtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=96-109&issue=1&author=J.+E.+Klaunigauthor=L.+M.+Kamendulisauthor=B.+A.+Hocevar&title=Oxidative+Stress+and+Oxidative+Damage+in+Carcinogenesis&doi=10.1177%2F0192623309356453"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Oxidative stress and oxidative damage in carcinogenesis</span></div><div class="casAuthors">Klaunig, James E.; Kamendulis, Lisa M.; Hocevar, Barbara A.</div><div class="citationInfo"><span class="NLM_cas:title">Toxicologic Pathology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">96-109</span>CODEN:
                <span class="NLM_cas:coden">TOPADD</span>;
        ISSN:<span class="NLM_cas:issn">0192-6233</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">A review.  Carcinogenesis is a multistep process involving mutation and the subsequent selective clonal expansion of the mutated cell.  Chem. and phys. agents including those that induce reactive oxygen species can induce and/or modulate this multistep process.  Several modes of action by which carcinogens induce cancer have been identified, including through prodn. of reactive oxygen species (ROS).  Oxidative damage to cellular macromols. can arise through overprodn. of ROS and faulty antioxidant and/or DNA repair mechanisms.  In addn., ROS can stimulate signal transduction pathways and lead to activation of key transcription factors such as Nrf2 and NF-κB.  The resultant altered gene expression patterns evoked by ROS contribute to the carcinogenesis process.  Recent evidence demonstrates an assocn. between a no. of single nucleotide polymorphisms (SNPs) in oxidative DNA repair genes and antioxidant genes with human cancer susceptibility.  These aspects of ROS biol. will be discussed in the context of their relationship to carcinogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGow0gFgYl9yd7Vg90H21EOLACvtfcHk0ljqOPK5rKpXCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjt1ajtb4%253D&md5=09e44cb1a6b772c7ce38987837d9ae8b</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1177%2F0192623309356453&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F0192623309356453%26sid%3Dliteratum%253Aachs%26aulast%3DKlaunig%26aufirst%3DJ.%2BE.%26aulast%3DKamendulis%26aufirst%3DL.%2BM.%26aulast%3DHocevar%26aufirst%3DB.%2BA.%26atitle%3DOxidative%2520Stress%2520and%2520Oxidative%2520Damage%2520in%2520Carcinogenesis%26jtitle%3DToxicol.%2520Pathol.%26date%3D2010%26volume%3D38%26issue%3D1%26spage%3D96%26epage%3D109%26doi%3D10.1177%2F0192623309356453" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gorrini, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, I. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mak, T. W.</span></span> <span> </span><span class="NLM_article-title">Modulation of Oxidative Stress as an Anticancer Strategy</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>12</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">931</span>– <span class="NLM_lpage">947</span>, <span class="refDoi"> DOI: 10.1038/nrd4002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1038%2Fnrd4002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=24287781" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVGqs7%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=931-947&issue=12&author=C.+Gorriniauthor=I.+S.+Harrisauthor=T.+W.+Mak&title=Modulation+of+Oxidative+Stress+as+an+Anticancer+Strategy&doi=10.1038%2Fnrd4002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Modulation of oxidative stress as an anticancer strategy</span></div><div class="casAuthors">Gorrini, Chiara; Harris, Isaac S.; Mak, Tak W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">931-947</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The regulation of oxidative stress is an important factor in both tumor development and responses to anticancer therapies.  Many signalling pathways that are linked to tumorigenesis can also regulate the metab. of reactive oxygen species (ROS) through direct or indirect mechanisms.  High ROS levels are generally detrimental to cells, and the redox status of cancer cells usually differs from that of normal cells.  Because of metabolic and signalling aberrations, cancer cells exhibit elevated ROS levels.  The observation that this is balanced by an increased antioxidant capacity suggests that high ROS levels may constitute a barrier to tumorigenesis.  However, ROS can also promote tumor formation by inducing DNA mutations and pro-oncogenic signalling pathways.  These contradictory effects have important implications for potential anticancer strategies that aim to modulate levels of ROS.  In this Review, we address the controversial role of ROS in tumor development and in responses to anticancer therapies, and elaborate on the idea that targeting the antioxidant capacity of tumor cells can have a pos. therapeutic impact.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVKYMoILuOfbVg90H21EOLACvtfcHk0lghvh45gHEKAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVGqs7%252FI&md5=7f135a1b32b1134a88cfa28b35c53de7</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnrd4002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4002%26sid%3Dliteratum%253Aachs%26aulast%3DGorrini%26aufirst%3DC.%26aulast%3DHarris%26aufirst%3DI.%2BS.%26aulast%3DMak%26aufirst%3DT.%2BW.%26atitle%3DModulation%2520of%2520Oxidative%2520Stress%2520as%2520an%2520Anticancer%2520Strategy%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2013%26volume%3D12%26issue%3D12%26spage%3D931%26epage%3D947%26doi%3D10.1038%2Fnrd4002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Haghdoost, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sjölander, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czene, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harms-Ringdahl, M.</span></span> <span> </span><span class="NLM_article-title">The Nucleotide Pool Is a Significant Target for Oxidative Stress</span>. <i>Free Radical Biol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>41</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">620</span>– <span class="NLM_lpage">626</span>, <span class="refDoi"> DOI: 10.1016/j.freeradbiomed.2006.05.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1016%2Fj.freeradbiomed.2006.05.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=16863995" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BD28XntlCqt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2006&pages=620-626&issue=4&author=S.+Haghdoostauthor=L.+Sj%C3%B6landerauthor=S.+Czeneauthor=M.+Harms-Ringdahl&title=The+Nucleotide+Pool+Is+a+Significant+Target+for+Oxidative+Stress&doi=10.1016%2Fj.freeradbiomed.2006.05.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The nucleotide pool is a significant target for oxidative stress</span></div><div class="casAuthors">Haghdoost, Siamak; Sjoelander, Lena; Czene, Stefan; Harms-Ringdahl, Mats</div><div class="citationInfo"><span class="NLM_cas:title">Free Radical Biology & Medicine</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">620-626</span>CODEN:
                <span class="NLM_cas:coden">FRBMEH</span>;
        ISSN:<span class="NLM_cas:issn">0891-5849</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Oxidative stress is considered to be one of the most important phenomena involved in the process of aging and age-related diseases. 8-Oxo-7,8-dihydro-2'-deoxyguanosine (8-oxo-dG) has been frequently used as a marker for oxidative stress.  However, the origin of extracellular 8-oxo-dG is not well understood.  The aim of this work was to investigate the nucleotide pool and the role of the human mutT homolog protein (hMTH1) in the appearance of extracellular 8-oxo-dG in a cellular model system.  For this purpose we used primary human fibroblast cells, which were transfected by siRNAs homologous to hMTH1.  Extracellular 8-oxo-dG in cell culture media after exposure of the cells to ionizing radiation was measured as ELISA reactivity.  Our results demonstrate the profound effect of both hMTH1 expression and nucleotide pool size on the cellular excretion of 8-oxo-dG, suggesting that the nucleotide pool is a significant target for the formation of extracellular 8-oxo-dG.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwAhG08kKjqLVg90H21EOLACvtfcHk0lghvh45gHEKAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XntlCqt7Y%253D&md5=a01efdb97485a13cebe95641d8efe287</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.freeradbiomed.2006.05.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.freeradbiomed.2006.05.003%26sid%3Dliteratum%253Aachs%26aulast%3DHaghdoost%26aufirst%3DS.%26aulast%3DSj%25C3%25B6lander%26aufirst%3DL.%26aulast%3DCzene%26aufirst%3DS.%26aulast%3DHarms-Ringdahl%26aufirst%3DM.%26atitle%3DThe%2520Nucleotide%2520Pool%2520Is%2520a%2520Significant%2520Target%2520for%2520Oxidative%2520Stress%26jtitle%3DFree%2520Radical%2520Biol.%2520Med.%26date%3D2006%26volume%3D41%26issue%3D4%26spage%3D620%26epage%3D626%26doi%3D10.1016%2Fj.freeradbiomed.2006.05.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pavlov, Y. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minnick, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Izuta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kunkel, T. A.</span></span> <span> </span><span class="NLM_article-title">DNA Replication Fidelity with 8-Oxodeoxyguanosine Triphosphate</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>33</i></span> (<span class="NLM_issue">15</span>),  <span class="NLM_fpage">4695</span>– <span class="NLM_lpage">4701</span>, <span class="refDoi"> DOI: 10.1021/bi00181a029</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi00181a029" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=1994&pages=4695-4701&issue=15&author=Y.+I.+Pavlovauthor=D.+T.+Minnickauthor=S.+Izutaauthor=T.+A.+Kunkel&title=DNA+Replication+Fidelity+with+8-Oxodeoxyguanosine+Triphosphate&doi=10.1021%2Fbi00181a029"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Fbi00181a029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00181a029%26sid%3Dliteratum%253Aachs%26aulast%3DPavlov%26aufirst%3DY.%2BI.%26aulast%3DMinnick%26aufirst%3DD.%2BT.%26aulast%3DIzuta%26aufirst%3DS.%26aulast%3DKunkel%26aufirst%3DT.%2BA.%26atitle%3DDNA%2520Replication%2520Fidelity%2520with%25208-Oxodeoxyguanosine%2520Triphosphate%26jtitle%3DBiochemistry%26date%3D1994%26volume%3D33%26issue%3D15%26spage%3D4695%26epage%3D4701%26doi%3D10.1021%2Fbi00181a029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bessman, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frick, D. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Handley, S. F.</span></span> <span> </span><span class="NLM_article-title">The MutT Proteins or ″Nudix″ Hydrolases, a Family of Versatile, Widely Distributed, ″housecleaning″ Enzymes</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>271</i></span> (<span class="NLM_issue">41</span>),  <span class="NLM_fpage">25059</span>– <span class="NLM_lpage">25062</span>, <span class="refDoi"> DOI: 10.1074/jbc.271.41.25059</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1074%2Fjbc.271.41.25059" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=271&publication_year=1996&pages=25059-25062&issue=41&author=M.+J.+Bessmanauthor=D.+N.+Frickauthor=S.+F.+O%E2%80%99Handley&title=The+MutT+Proteins+or+%E2%80%B3Nudix%E2%80%B3+Hydrolases%2C+a+Family+of+Versatile%2C+Widely+Distributed%2C+%E2%80%B3housecleaning%E2%80%B3+Enzymes&doi=10.1074%2Fjbc.271.41.25059"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1074%2Fjbc.271.41.25059&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.271.41.25059%26sid%3Dliteratum%253Aachs%26aulast%3DBessman%26aufirst%3DM.%2BJ.%26aulast%3DFrick%26aufirst%3DD.%2BN.%26aulast%3DO%25E2%2580%2599Handley%26aufirst%3DS.%2BF.%26atitle%3DThe%2520MutT%2520Proteins%2520or%2520%25E2%2580%25B3Nudix%25E2%2580%25B3%2520Hydrolases%252C%2520a%2520Family%2520of%2520Versatile%252C%2520Widely%2520Distributed%252C%2520%25E2%2580%25B3housecleaning%25E2%2580%25B3%2520Enzymes%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1996%26volume%3D271%26issue%3D41%26spage%3D25059%26epage%3D25062%26doi%3D10.1074%2Fjbc.271.41.25059" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gad, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koolmeister, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jemth, A.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eshtad, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacques, S. a</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ström, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svensson, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lundbäck, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Einarsdottir, B. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saleh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Göktürk, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baranczewski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svensson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berntsson, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafsson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strömberg, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanjiv, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacques-Cordonnier, M.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desroses, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustavsson, A.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olofsson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johansson, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Homan, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loseva, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bräutigam, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johansson, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Höglund, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hagenkort, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pham, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altun, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaugaz, F. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vikingsson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evers, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henriksson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vallin, K. S. a</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallner, O. a</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammarström, L. G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiita, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almlöf, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalderén, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Axelsson, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Djureinovic, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puigvert, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Häggblad, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeppsson, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martens, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lundin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lundgren, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Granelli, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Artursson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nilsson, J. a</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stenmark, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scobie, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berglund, U. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helleday, T.</span></span> <span> </span><span class="NLM_article-title">MTH1 Inhibition Eradicates Cancer by Preventing Sanitation of the dNTP Pool</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>508</i></span> (<span class="NLM_issue">7495</span>),  <span class="NLM_fpage">215</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1038/nature13181</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1038%2Fnature13181" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=24695224" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtlWrsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=508&publication_year=2014&pages=215-221&issue=7495&author=H.+Gadauthor=T.+Koolmeisterauthor=A.-S.+Jemthauthor=S.+Eshtadauthor=S.+a+Jacquesauthor=C.+E.+Str%C3%B6mauthor=L.+M.+Svenssonauthor=N.+Schultzauthor=T.+Lundb%C3%A4ckauthor=B.+O.+Einarsdottirauthor=A.+Salehauthor=C.+G%C3%B6kt%C3%BCrkauthor=P.+Baranczewskiauthor=R.+Svenssonauthor=R.+P.+Berntssonauthor=R.+Gustafssonauthor=K.+Str%C3%B6mbergauthor=K.+Sanjivauthor=M.-C.+Jacques-Cordonnierauthor=M.+Desrosesauthor=A.-L.+Gustavssonauthor=R.+Olofssonauthor=F.+Johanssonauthor=E.+J.+Homanauthor=O.+Losevaauthor=L.+Br%C3%A4utigamauthor=L.+Johanssonauthor=A.+H%C3%B6glundauthor=A.+Hagenkortauthor=T.+Phamauthor=M.+Altunauthor=F.+Z.+Gaugazauthor=S.+Vikingssonauthor=B.+Eversauthor=M.+Henrikssonauthor=K.+S.+a+Vallinauthor=O.+a+Wallnerauthor=L.+G.+J.+Hammarstr%C3%B6mauthor=E.+Wiitaauthor=I.+Alml%C3%B6fauthor=C.+Kalder%C3%A9nauthor=H.+Axelssonauthor=T.+Djureinovicauthor=J.+C.+Puigvertauthor=M.+H%C3%A4ggbladauthor=F.+Jeppssonauthor=U.+Martensauthor=C.+Lundinauthor=B.+Lundgrenauthor=I.+Granelliauthor=A.+J.+Jensenauthor=P.+Arturssonauthor=J.+a+Nilssonauthor=P.+Stenmarkauthor=M.+Scobieauthor=U.+W.+Berglundauthor=T.+Helleday&title=MTH1+Inhibition+Eradicates+Cancer+by+Preventing+Sanitation+of+the+dNTP+Pool&doi=10.1038%2Fnature13181"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">MTH1 inhibition eradicates cancer by preventing sanitation of the dNTP pool</span></div><div class="casAuthors">Gad, Helge; Koolmeister, Tobias; Jemth, Ann-Sofie; Eshtad, Saeed; Jacques, Sylvain A.; Stroem, Cecilia E.; Svensson, Linda M.; Schultz, Niklas; Lundbaeck, Thomas; Einarsdottir, Berglind Osk; Saleh, Aljona; Gokturk, Camilla; Baranczewski, Pawel; Svensson, Richard; Berntsson, Ronnie P.-A.; Gustafsson, Robert; Stroemberg, Kia; Sanjiv, Kumar; Jacques-Cordonnier, Marie-Caroline; Desroses, Matthieu; Gustavsson, Anna-Lena; Olofsson, Roger; Johansson, Fredrik; Homan, Evert J.; Loseva, Olga; Braeutigam, Lars; Johansson, Lars; Hoeglund, Andreas; Hagenkort, Anna; Pham, Therese; Altun, Mikael; Gaugaz, Fabienne Z.; Vikingsson, Svante; Evers, Bastiaan; Henriksson, Martin; Vallin, Karl S. A.; Wallner, Olov A.; Hammarstroem, Lars G. J.; Wiita, Elisee; Almloef, Ingrid; Kalderen, Christina; Axelsson, Hanna; Djureinovic, Tatjana; Puigvert, Jordi Carreras; Haeggblad, Maria; Jeppsson, Fredrik; Martens, Ulf; Lundin, Cecilia; Lundgren, Bo; Granelli, Ingrid; Jensen, Annika Jenmalm; Artursson, Per; Nilsson, Jonas A.; Stenmark, Pal; Scobie, Martin; Berglund, Ulrika Warpman; Helleday, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">508</span>
        (<span class="NLM_cas:issue">7495</span>),
    <span class="NLM_cas:pages">215-221</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cancers have dysfunctional redox regulation resulting in reactive oxygen species prodn., damaging both DNA and free dNTPs.  The MTH1 protein sanitizes oxidized dNTP pools to prevent incorporation of damaged bases during DNA replication.  Although MTH1 is non-essential in normal cells, the authors show that cancer cells require MTH1 activity to avoid incorporation of oxidized dNTPs, resulting in DNA damage and cell death.  The authors validate MTH1 as an anticancer target in vivo and describe small mols. TH287 (I) and TH588 (II) as first-in-class nudix hydrolase family inhibitors that potently and selectively engage and inhibit the MTH1 protein in cells.  Protein co-crystal structures demonstrate that the inhibitors bind in the active site of MTH1.  The inhibitors cause incorporation of oxidized dNTPs in cancer cells, leading to DNA damage, cytotoxicity and therapeutic responses in patient-derived mouse xenografts.  This study exemplifies the non-oncogene addiction concept for anticancer treatment and validates MTH1 as being cancer phenotypic lethal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-Xn3KhC40XbVg90H21EOLACvtfcHk0ljxovkguoZzHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtlWrsLc%253D&md5=49afb4522874f4f5e216ae38d0ff7cd2</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnature13181&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13181%26sid%3Dliteratum%253Aachs%26aulast%3DGad%26aufirst%3DH.%26aulast%3DKoolmeister%26aufirst%3DT.%26aulast%3DJemth%26aufirst%3DA.-S.%26aulast%3DEshtad%26aufirst%3DS.%26aulast%3DJacques%26aufirst%3DS.%2Ba%26aulast%3DStr%25C3%25B6m%26aufirst%3DC.%2BE.%26aulast%3DSvensson%26aufirst%3DL.%2BM.%26aulast%3DSchultz%26aufirst%3DN.%26aulast%3DLundb%25C3%25A4ck%26aufirst%3DT.%26aulast%3DEinarsdottir%26aufirst%3DB.%2BO.%26aulast%3DSaleh%26aufirst%3DA.%26aulast%3DG%25C3%25B6kt%25C3%25BCrk%26aufirst%3DC.%26aulast%3DBaranczewski%26aufirst%3DP.%26aulast%3DSvensson%26aufirst%3DR.%26aulast%3DBerntsson%26aufirst%3DR.%2BP.%26aulast%3DGustafsson%26aufirst%3DR.%26aulast%3DStr%25C3%25B6mberg%26aufirst%3DK.%26aulast%3DSanjiv%26aufirst%3DK.%26aulast%3DJacques-Cordonnier%26aufirst%3DM.-C.%26aulast%3DDesroses%26aufirst%3DM.%26aulast%3DGustavsson%26aufirst%3DA.-L.%26aulast%3DOlofsson%26aufirst%3DR.%26aulast%3DJohansson%26aufirst%3DF.%26aulast%3DHoman%26aufirst%3DE.%2BJ.%26aulast%3DLoseva%26aufirst%3DO.%26aulast%3DBr%25C3%25A4utigam%26aufirst%3DL.%26aulast%3DJohansson%26aufirst%3DL.%26aulast%3DH%25C3%25B6glund%26aufirst%3DA.%26aulast%3DHagenkort%26aufirst%3DA.%26aulast%3DPham%26aufirst%3DT.%26aulast%3DAltun%26aufirst%3DM.%26aulast%3DGaugaz%26aufirst%3DF.%2BZ.%26aulast%3DVikingsson%26aufirst%3DS.%26aulast%3DEvers%26aufirst%3DB.%26aulast%3DHenriksson%26aufirst%3DM.%26aulast%3DVallin%26aufirst%3DK.%2BS.%2Ba%26aulast%3DWallner%26aufirst%3DO.%2Ba%26aulast%3DHammarstr%25C3%25B6m%26aufirst%3DL.%2BG.%2BJ.%26aulast%3DWiita%26aufirst%3DE.%26aulast%3DAlml%25C3%25B6f%26aufirst%3DI.%26aulast%3DKalder%25C3%25A9n%26aufirst%3DC.%26aulast%3DAxelsson%26aufirst%3DH.%26aulast%3DDjureinovic%26aufirst%3DT.%26aulast%3DPuigvert%26aufirst%3DJ.%2BC.%26aulast%3DH%25C3%25A4ggblad%26aufirst%3DM.%26aulast%3DJeppsson%26aufirst%3DF.%26aulast%3DMartens%26aufirst%3DU.%26aulast%3DLundin%26aufirst%3DC.%26aulast%3DLundgren%26aufirst%3DB.%26aulast%3DGranelli%26aufirst%3DI.%26aulast%3DJensen%26aufirst%3DA.%2BJ.%26aulast%3DArtursson%26aufirst%3DP.%26aulast%3DNilsson%26aufirst%3DJ.%2Ba%26aulast%3DStenmark%26aufirst%3DP.%26aulast%3DScobie%26aufirst%3DM.%26aulast%3DBerglund%26aufirst%3DU.%2BW.%26aulast%3DHelleday%26aufirst%3DT.%26atitle%3DMTH1%2520Inhibition%2520Eradicates%2520Cancer%2520by%2520Preventing%2520Sanitation%2520of%2520the%2520dNTP%2520Pool%26jtitle%3DNature%26date%3D2014%26volume%3D508%26issue%3D7495%26spage%3D215%26epage%3D221%26doi%3D10.1038%2Fnature13181" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nakabeppu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kajitani, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaguchi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuchimoto, D.</span></span> <span> </span><span class="NLM_article-title">MTH1, an Oxidized Purine Nucleoside Triphosphatase, Prevents the Cytotoxicity and Neurotoxicity of Oxidized Purine Nucleotides</span>. <i>DNA Repair</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>5</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">761</span>– <span class="NLM_lpage">772</span>, <span class="refDoi"> DOI: 10.1016/j.dnarep.2006.03.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1016%2Fj.dnarep.2006.03.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=16621731" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BD28Xmt1OqsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2006&pages=761-772&issue=7&author=Y.+Nakabeppuauthor=K.+Kajitaniauthor=K.+Sakamotoauthor=H.+Yamaguchiauthor=D.+Tsuchimoto&title=MTH1%2C+an+Oxidized+Purine+Nucleoside+Triphosphatase%2C+Prevents+the+Cytotoxicity+and+Neurotoxicity+of+Oxidized+Purine+Nucleotides&doi=10.1016%2Fj.dnarep.2006.03.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">MTH1, an oxidized purine nucleoside triphosphatase, prevents the cytotoxicity and neurotoxicity of oxidized purine nucleotides</span></div><div class="casAuthors">Nakabeppu, Yusaku; Kajitani, Kosuke; Sakamoto, Katsumi; Yamaguchi, Hiroo; Tsuchimoto, Daisuke</div><div class="citationInfo"><span class="NLM_cas:title">DNA Repair</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">761-772</span>CODEN:
                <span class="NLM_cas:coden">DRNEAR</span>;
        ISSN:<span class="NLM_cas:issn">1568-7864</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  In human and rodent cells, MTH1, an oxidized purine nucleoside triphosphatase, efficiently hydrolyzes oxidized dGTP, GTP, dATP, and ATP such as 2'-deoxy-8-oxoguanosine triphosphate (8-oxo-dGTP) and 2'-deoxy-2-hydroxyadenosine triphosphate (2-OH-dATP) in nucleotide pools, thus avoiding their incorporation into DNA or RNA.  MTH1 is expressed in postmitotic neurons as well as in proliferative tissues, and it is localized both in the mitochondria and nucleus, thus suggesting that MTH1 plays an important role in the prevention of the mutagenicity and cytotoxicity of such oxidized purines as 8-oxoG which are known to accumulate in the cellular genome.  Our recent studies with MTH1-deficient mice or cells revealed that MTH1 efficiently minimizes accumulation of 8-oxoG in both nuclear and mitochondrial DNA in the mouse brain as well as in cultured cells, thus contributing to the protection of the brain from oxidative stress.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoe-LGRN37xcrVg90H21EOLACvtfcHk0ljxovkguoZzHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xmt1OqsLc%253D&md5=9a38c80c0ed62c8ada5bf1ac0dab9d9e</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.dnarep.2006.03.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.dnarep.2006.03.003%26sid%3Dliteratum%253Aachs%26aulast%3DNakabeppu%26aufirst%3DY.%26aulast%3DKajitani%26aufirst%3DK.%26aulast%3DSakamoto%26aufirst%3DK.%26aulast%3DYamaguchi%26aufirst%3DH.%26aulast%3DTsuchimoto%26aufirst%3DD.%26atitle%3DMTH1%252C%2520an%2520Oxidized%2520Purine%2520Nucleoside%2520Triphosphatase%252C%2520Prevents%2520the%2520Cytotoxicity%2520and%2520Neurotoxicity%2520of%2520Oxidized%2520Purine%2520Nucleotides%26jtitle%3DDNA%2520Repair%26date%3D2006%26volume%3D5%26issue%3D7%26spage%3D761%26epage%3D772%26doi%3D10.1016%2Fj.dnarep.2006.03.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rai, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burton, D. G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giribaldi, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Onder, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinberg, R. A.</span></span> <span> </span><span class="NLM_article-title">Enhanced Elimination of Oxidized Guanine Nucleotides Inhibits Oncogenic RAS-Induced DNA Damage and Premature Senescence</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>30</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">1489</span>– <span class="NLM_lpage">1496</span>, <span class="refDoi"> DOI: 10.1038/onc.2010.520</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1038%2Fonc.2010.520" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2011&pages=1489-1496&issue=12&author=P.+Raiauthor=J.+J.+Youngauthor=D.+G.+A.+Burtonauthor=M.+G.+Giribaldiauthor=T.+T.+Onderauthor=R.+A.+Weinberg&title=Enhanced+Elimination+of+Oxidized+Guanine+Nucleotides+Inhibits+Oncogenic+RAS-Induced+DNA+Damage+and+Premature+Senescence&doi=10.1038%2Fonc.2010.520"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fonc.2010.520&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2010.520%26sid%3Dliteratum%253Aachs%26aulast%3DRai%26aufirst%3DP.%26aulast%3DYoung%26aufirst%3DJ.%2BJ.%26aulast%3DBurton%26aufirst%3DD.%2BG.%2BA.%26aulast%3DGiribaldi%26aufirst%3DM.%2BG.%26aulast%3DOnder%26aufirst%3DT.%2BT.%26aulast%3DWeinberg%26aufirst%3DR.%2BA.%26atitle%3DEnhanced%2520Elimination%2520of%2520Oxidized%2520Guanine%2520Nucleotides%2520Inhibits%2520Oncogenic%2520RAS-Induced%2520DNA%2520Damage%2520and%2520Premature%2520Senescence%26jtitle%3DOncogene%26date%3D2011%26volume%3D30%26issue%3D12%26spage%3D1489%26epage%3D1496%26doi%3D10.1038%2Fonc.2010.520" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Patel, a</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burton, D. G. a</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halvorsen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balkan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reiner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez-Stable, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, a</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munoz, a</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giribaldi, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robbins, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rai, P.</span></span> <span> </span><span class="NLM_article-title">MutT Homolog 1 (MTH1) Maintains Multiple KRAS-Driven pro-Malignant Pathways</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">2586</span>, <span class="refDoi"> DOI: 10.1038/onc.2014.195</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1038%2Fonc.2014.195" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=25023700" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFGqtr%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2015&pages=2586&author=a+Patelauthor=D.+G.+a+Burtonauthor=K.+Halvorsenauthor=W.+Balkanauthor=T.+Reinerauthor=C.+Perez-Stableauthor=a+Cohenauthor=a+Munozauthor=M.+G.+Giribaldiauthor=S.+Singhauthor=D.+J.+Robbinsauthor=D.+M.+Nguyenauthor=P.+Rai&title=MutT+Homolog+1+%28MTH1%29+Maintains+Multiple+KRAS-Driven+pro-Malignant+Pathways&doi=10.1038%2Fonc.2014.195"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">MutT Homolog 1 (MTH1) maintains multiple KRAS-driven pro-malignant pathways</span></div><div class="casAuthors">Patel, A.; Burton, D. G. A.; Halvorsen, K.; Balkan, W.; Reiner, T.; Perez-Stable, C.; Cohen, A.; Munoz, A.; Giribaldi, M. G.; Singh, S.; Robbins, D. J.; Nguyen, D. M.; Rai, P.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">2586-2596</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Oncogenic RAS promotes prodn. of reactive oxygen species (ROS), which mediate pro-malignant signaling but can also trigger DNA damage-induced tumor suppression.  Thus RAS-driven tumor cells require redox-protective mechanisms to mitigate the damaging aspects of ROS.  Here, we show that MutT Homolog 1 (MTH1), the mammalian 8-oxodGTPase that sanitizes oxidative damage in the nucleotide pool, is important for maintaining several KRAS-driven pro-malignant traits in a nonsmall cell lung carcinoma (NSCLC) model.  MTH1 suppression in KRAS-mutant NSCLC cells impairs proliferation and xenograft tumor formation.  Furthermore, MTH1 levels modulate KRAS-induced transformation of immortalized lung epithelial cells.  MTH1 expression is upregulated by oncogenic KRAS and correlates pos. with high KRAS levels in NSCLC human tumors.  At a mol. level, in p53-competent KRAS-mutant cells, MTH1 loss provokes DNA damage and induction of oncogene-induced senescence.  In p53-nonfunctional KRAS-mutant cells, MTH1 suppression does not produce DNA damage but reduces proliferation and leads to an adaptive decrease in KRAS expression levels.  Thus, MTH1 not only enables evasion of oxidative DNA damage and its consequences, but can also function as a mol. rheostat for maintaining oncogene expression at optimal levels.  Accordingly, our results indicate MTH1 is a novel and crit. component of oncogenic KRAS-assocd. malignancy and its inhibition is likely to yield significant tumor-suppressive outcomes in KRAS-driven tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQuLF6p5AL87Vg90H21EOLACvtfcHk0lguUAmzzwPqtQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFGqtr%252FN&md5=bb949121fcc660bfe99dfccdea321c6f</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fonc.2014.195&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2014.195%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3Da%26aulast%3DBurton%26aufirst%3DD.%2BG.%2Ba%26aulast%3DHalvorsen%26aufirst%3DK.%26aulast%3DBalkan%26aufirst%3DW.%26aulast%3DReiner%26aufirst%3DT.%26aulast%3DPerez-Stable%26aufirst%3DC.%26aulast%3DCohen%26aufirst%3Da%26aulast%3DMunoz%26aufirst%3Da%26aulast%3DGiribaldi%26aufirst%3DM.%2BG.%26aulast%3DSingh%26aufirst%3DS.%26aulast%3DRobbins%26aufirst%3DD.%2BJ.%26aulast%3DNguyen%26aufirst%3DD.%2BM.%26aulast%3DRai%26aufirst%3DP.%26atitle%3DMutT%2520Homolog%25201%2520%2528MTH1%2529%2520Maintains%2520Multiple%2520KRAS-Driven%2520pro-Malignant%2520Pathways%26jtitle%3DOncogene%26date%3D2015%26volume%3D34%26spage%3D2586%26doi%3D10.1038%2Fonc.2014.195" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salah, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Radic, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gridling, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elkins, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stukalov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jemth, A.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Göktürk, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanjiv, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strömberg, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pham, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berglund, U. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colinge, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loizou, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helleday, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Superti-Furga, G.</span></span> <span> </span><span class="NLM_article-title">Stereospecific Targeting of MTH1 by (S)-Crizotinib as an Anticancer Strategy</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>508</i></span> (<span class="NLM_issue">7495</span>),  <span class="NLM_fpage">222</span>– <span class="NLM_lpage">227</span>, <span class="refDoi"> DOI: 10.1038/nature13194</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1038%2Fnature13194" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=24695225" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtlWrsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=508&publication_year=2014&pages=222-227&issue=7495&author=K.+V.+M+Huberauthor=E.+Salahauthor=B.+Radicauthor=M.+Gridlingauthor=J.+M.+Elkinsauthor=A.+Stukalovauthor=A.-S.+Jemthauthor=C.+G%C3%B6kt%C3%BCrkauthor=K.+Sanjivauthor=K.+Str%C3%B6mbergauthor=T.+Phamauthor=U.+W.+Berglundauthor=J.+Colingeauthor=K.+L.+Bennettauthor=J.+I.+Loizouauthor=T.+Helledayauthor=S.+Knappauthor=G.+Superti-Furga&title=Stereospecific+Targeting+of+MTH1+by+%28S%29-Crizotinib+as+an+Anticancer+Strategy&doi=10.1038%2Fnature13194"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Stereospecific targeting of MTH1 by (S)-crizotinib as an anticancer strategy</span></div><div class="casAuthors">Huber, Kilian V. M.; Salah, Eidarus; Radic, Branka; Gridling, Manuela; Elkins, Jonathan M.; Stukalov, Alexey; Jemth, Ann-Sofie; Gokturk, Camilla; Sanjiv, Kumar; Stroemberg, Kia; Pham, Therese; Berglund, Ulrika Warpman; Colinge, Jacques; Bennett, Keiryn L.; Loizou, Joanna I.; Helleday, Thomas; Knapp, Stefan; Superti-Furga, Giulio</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">508</span>
        (<span class="NLM_cas:issue">7495</span>),
    <span class="NLM_cas:pages">222-227</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Activated RAS GTPase signaling is a crit. driver of oncogenic transformation and malignant disease.  Cellular models of RAS-dependent cancers have been used to identify exptl. small mols., such as SCH51344, but their mol. mechanism of action remains generally unknown.  Here, using a chem. proteomic approach, the authors identify the target of SCH51344 as the human mutT homolog MTH1 (also known as NUDT1), a nucleotide pool sanitizing enzyme.  Loss-of-function of MTH1 impaired growth of KRAS tumor cells, whereas MTH1 overexpression mitigated sensitivity towards SCH51344.  Searching for more drug-like inhibitors, the authors identified the kinase inhibitor crizotinib as a nanomolar suppressor of MTH1 activity.  Surprisingly, the clin. used (R)-enantiomer of the drug was inactive, whereas the (S)-enantiomer selectively inhibited MTH1 catalytic activity.  Enzymic assays, chem. proteomic profiling, kinome-wide activity surveys and MTH1 co-crystal structures of both enantiomers provide a rationale for this remarkable stereospecificity.  Disruption of nucleotide pool homeostasis via MTH1 inhibition by (S)-crizotinib induced an increase in DNA single-strand breaks, activated DNA repair in human colon carcinoma cells, and effectively suppressed tumor growth in animal models.  The authors' results propose (S)-crizotinib as an attractive chem. entity for further pre-clin. evaluation, and small-mol. inhibitors of MTH1 in general as a promising novel class of anticancer agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrbUm7UP6kIrVg90H21EOLACvtfcHk0lguUAmzzwPqtQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtlWrsb4%253D&md5=ca3df11a6782fd9140b6d8ba0c16e5ab</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnature13194&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13194%26sid%3Dliteratum%253Aachs%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM%26aulast%3DSalah%26aufirst%3DE.%26aulast%3DRadic%26aufirst%3DB.%26aulast%3DGridling%26aufirst%3DM.%26aulast%3DElkins%26aufirst%3DJ.%2BM.%26aulast%3DStukalov%26aufirst%3DA.%26aulast%3DJemth%26aufirst%3DA.-S.%26aulast%3DG%25C3%25B6kt%25C3%25BCrk%26aufirst%3DC.%26aulast%3DSanjiv%26aufirst%3DK.%26aulast%3DStr%25C3%25B6mberg%26aufirst%3DK.%26aulast%3DPham%26aufirst%3DT.%26aulast%3DBerglund%26aufirst%3DU.%2BW.%26aulast%3DColinge%26aufirst%3DJ.%26aulast%3DBennett%26aufirst%3DK.%2BL.%26aulast%3DLoizou%26aufirst%3DJ.%2BI.%26aulast%3DHelleday%26aufirst%3DT.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DSuperti-Furga%26aufirst%3DG.%26atitle%3DStereospecific%2520Targeting%2520of%2520MTH1%2520by%2520%2528S%2529-Crizotinib%2520as%2520an%2520Anticancer%2520Strategy%26jtitle%3DNature%26date%3D2014%26volume%3D508%26issue%3D7495%26spage%3D222%26epage%3D227%26doi%3D10.1038%2Fnature13194" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berman, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Battistuz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhat, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bluhm, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourne, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burkhardt, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilliland, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iype, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fagan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marvin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Padilla, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ravichandran, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thanki, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissig, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westbrook, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zardecki, C.</span></span> <span> </span><span class="NLM_article-title">The Protein Data Bank</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">899</span>– <span class="NLM_lpage">907</span>, <span class="refDoi"> DOI: 10.1107/S0907444902003451</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1107%2FS0907444902003451" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=12037327" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BD38XktVKhtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2002&pages=899-907&author=H.+M.+Bermanauthor=T.+Battistuzauthor=T.+N.+Bhatauthor=W.+F.+Bluhmauthor=P.+E.+Bourneauthor=K.+Burkhardtauthor=Z.+Fengauthor=G.+L.+Gillilandauthor=L.+Iypeauthor=S.+Jainauthor=P.+Faganauthor=J.+Marvinauthor=D.+Padillaauthor=V.+Ravichandranauthor=B.+Schneiderauthor=N.+Thankiauthor=H.+Weissigauthor=J.+D.+Westbrookauthor=C.+Zardecki&title=The+Protein+Data+Bank&doi=10.1107%2FS0907444902003451"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Data Bank</span></div><div class="casAuthors">Berman, Helen M.; Battistuz, Tammy; Bhat, T. N.; Bluhm, Wolfgang F.; Bourne, Philip E.; Burkhardt, Kyle; Feng, Zukang; Gilliland, Gary L.; Iype, Lisa; Jain, Shri; Fagan, Phoebe; Marvin, Jessica; Padilla, David; Ravichandran, Veerasamy; Schneider, Bohdan; Thanki, Narmada; Weissig, Helge; Westbrook, John D.; Zardecki, Christine</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">D58</span>
        (<span class="NLM_cas:issue">6, No. 1</span>),
    <span class="NLM_cas:pages">899-907</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Munksgaard</span>)
        </div><div class="casAbstract">The Protein Data Bank [PDB; Berman, Westbrook et al. (2000), Nucleic Acids Res. 28, 235-242; http://www.pdb.org/] is the single worldwide archive of primary structural data of biol. macromols.  Many secondary sources of information are derived from PDB data.  It is the starting point for studies in structural bioinformatics.  This article describes the goals of the PDB, the systems in place for data deposition and access, how to obtain further information and plans for the future development of the resource.  The reader should come away with an understanding of the scope of the PDB and what is provided by the resource.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsUFcMyys-PrVg90H21EOLACvtfcHk0lguUAmzzwPqtQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XktVKhtLg%253D&md5=d7594ad00ec8e5bacc413d38bceaa360</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1107%2FS0907444902003451&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444902003451%26sid%3Dliteratum%253Aachs%26aulast%3DBerman%26aufirst%3DH.%2BM.%26aulast%3DBattistuz%26aufirst%3DT.%26aulast%3DBhat%26aufirst%3DT.%2BN.%26aulast%3DBluhm%26aufirst%3DW.%2BF.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26aulast%3DBurkhardt%26aufirst%3DK.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DGilliland%26aufirst%3DG.%2BL.%26aulast%3DIype%26aufirst%3DL.%26aulast%3DJain%26aufirst%3DS.%26aulast%3DFagan%26aufirst%3DP.%26aulast%3DMarvin%26aufirst%3DJ.%26aulast%3DPadilla%26aufirst%3DD.%26aulast%3DRavichandran%26aufirst%3DV.%26aulast%3DSchneider%26aufirst%3DB.%26aulast%3DThanki%26aufirst%3DN.%26aulast%3DWeissig%26aufirst%3DH.%26aulast%3DWestbrook%26aufirst%3DJ.%2BD.%26aulast%3DZardecki%26aufirst%3DC.%26atitle%3DThe%2520Protein%2520Data%2520Bank%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2002%26volume%3D58%26spage%3D899%26epage%3D907%26doi%3D10.1107%2FS0907444902003451" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tresaugues, L.</span>; <span class="NLM_string-name">Siponen, M. .</span>; <span class="NLM_string-name">Arrowsmith, C. .</span>; <span class="NLM_string-name">Berglund, H.</span>; <span class="NLM_string-name">Bountra, C.</span>; <span class="NLM_string-name">Collins, R.</span>; <span class="NLM_string-name">Edwards, A. .</span>; <span class="NLM_string-name">Ekblad, T.</span>; <span class="NLM_string-name">Flodin, S.</span>; <span class="NLM_string-name">Flores, A.</span>; <span class="NLM_string-name">Graslund, S.</span>; <span class="NLM_string-name">Hammarstrom, M.</span>; <span class="NLM_string-name">Johansson, I.</span>; <span class="NLM_string-name">Karlberg, T.</span>; <span class="NLM_string-name">Kol, S.</span>; <span class="NLM_string-name">Kotenyova, T.</span>; <span class="NLM_string-name">Kouznetsova, E.</span>; <span class="NLM_string-name">Moche, M.</span>; <span class="NLM_string-name">Nyman, T.</span>; <span class="NLM_string-name">Persson, C.</span>; <span class="NLM_string-name">Schuler, H.</span>; <span class="NLM_string-name">Schutz, P.</span>; <span class="NLM_string-name">Thorsell, A. .</span>; <span class="NLM_string-name">Van Der Berg, S.</span>; <span class="NLM_string-name">Wahlberg, E.</span>; <span class="NLM_string-name">Weigelt, J.</span>; <span class="NLM_string-name">Welin, M.</span>; <span class="NLM_string-name">Nordlund, P.</span></span> <span> </span><span class="NLM_article-title">Crystal Structure of Human 8-Oxo-dGTPase (MTH1)</span>. Deposited in PDB as 3Q93, <span class="NLM_year">2011</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&author=L.+Tresaugues&author=M.+.+Siponen&author=C.+.+Arrowsmith&author=H.+Berglund&author=C.+Bountra&author=R.+Collins&author=A.+.+Edwards&author=T.+Ekblad&author=S.+Flodin&author=A.+Flores&author=S.+Graslund&author=M.+Hammarstrom&author=I.+Johansson&author=T.+Karlberg&author=S.+Kol&author=T.+Kotenyova&author=E.+Kouznetsova&author=M.+Moche&author=T.+Nyman&author=C.+Persson&author=H.+Schuler&author=P.+Schutz&author=A.+.+Thorsell&author=S.+Van+Der+Berg&author=E.+Wahlberg&author=J.+Weigelt&author=M.+Welin&author=P.+Nordlund&title=Crystal+Structure+of+Human+8-Oxo-dGTPase+%28MTH1%29"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DTresaugues%26aufirst%3DL.%26atitle%3DCrystal%2520Structure%2520of%2520Human%25208-Oxo-dGTPase%2520%2528MTH1%2529%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Waz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirata, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koga-Ogawa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chirifu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arimori, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamada, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ikemizu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakabeppu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamagata, Y.</span></span> <span> </span><span class="NLM_article-title">Structural and Kinetic Studies of the Human Nudix Hydrolase MTH1 Reveal the Mechanism for Its Broad Substrate Specificity</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>292</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">2785</span>– <span class="NLM_lpage">2794</span>, <span class="refDoi"> DOI: 10.1074/jbc.M116.749713</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1074%2Fjbc.M116.749713" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=292&publication_year=2017&pages=2785-2794&issue=7&author=S.+Wazauthor=T.+Nakamuraauthor=K.+Hirataauthor=Y.+Koga-Ogawaauthor=M.+Chirifuauthor=T.+Arimoriauthor=T.+Tamadaauthor=S.+Ikemizuauthor=Y.+Nakabeppuauthor=Y.+Yamagata&title=Structural+and+Kinetic+Studies+of+the+Human+Nudix+Hydrolase+MTH1+Reveal+the+Mechanism+for+Its+Broad+Substrate+Specificity&doi=10.1074%2Fjbc.M116.749713"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M116.749713&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M116.749713%26sid%3Dliteratum%253Aachs%26aulast%3DWaz%26aufirst%3DS.%26aulast%3DNakamura%26aufirst%3DT.%26aulast%3DHirata%26aufirst%3DK.%26aulast%3DKoga-Ogawa%26aufirst%3DY.%26aulast%3DChirifu%26aufirst%3DM.%26aulast%3DArimori%26aufirst%3DT.%26aulast%3DTamada%26aufirst%3DT.%26aulast%3DIkemizu%26aufirst%3DS.%26aulast%3DNakabeppu%26aufirst%3DY.%26aulast%3DYamagata%26aufirst%3DY.%26atitle%3DStructural%2520and%2520Kinetic%2520Studies%2520of%2520the%2520Human%2520Nudix%2520Hydrolase%2520MTH1%2520Reveal%2520the%2520Mechanism%2520for%2520Its%2520Broad%2520Substrate%2520Specificity%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2017%26volume%3D292%26issue%3D7%26spage%3D2785%26epage%3D2794%26doi%3D10.1074%2Fjbc.M116.749713" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nissink, J. W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bista, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breed, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Embrey, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter-Holt, J. J.</span></span> <span> </span><span class="NLM_article-title">MTH1 Substrate Recognition—An Example of Specific Promiscuity</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">e0151154</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0151154</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1371%2Fjournal.pone.0151154" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=e0151154&issue=3&author=J.+W.+M.+Nissinkauthor=M.+Bistaauthor=J.+Breedauthor=N.+Carterauthor=K.+Embreyauthor=J.+Readauthor=J.+J.+Winter-Holt&title=MTH1+Substrate+Recognition%E2%80%94An+Example+of+Specific+Promiscuity&doi=10.1371%2Fjournal.pone.0151154"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0151154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0151154%26sid%3Dliteratum%253Aachs%26aulast%3DNissink%26aufirst%3DJ.%2BW.%2BM.%26aulast%3DBista%26aufirst%3DM.%26aulast%3DBreed%26aufirst%3DJ.%26aulast%3DCarter%26aufirst%3DN.%26aulast%3DEmbrey%26aufirst%3DK.%26aulast%3DRead%26aufirst%3DJ.%26aulast%3DWinter-Holt%26aufirst%3DJ.%2BJ.%26atitle%3DMTH1%2520Substrate%2520Recognition%25E2%2580%2594An%2520Example%2520of%2520Specific%2520Promiscuity%26jtitle%3DPLoS%2520One%26date%3D2016%26volume%3D11%26issue%3D3%26spage%3De0151154%26doi%3D10.1371%2Fjournal.pone.0151154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kettle, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alwan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bista, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breed, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eckersley, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fillery, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foote, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodwin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Käck, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nissink, J. W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wissler, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wylot, M.</span></span> <span> </span><span class="NLM_article-title">Potent and Selective Inhibitors of MTH1 Probe Its Role in Cancer Cell Survival</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">2346</span>– <span class="NLM_lpage">2361</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01760</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01760" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BC28XisFahsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=2346-2361&issue=6&author=J.+G.+Kettleauthor=H.+Alwanauthor=M.+Bistaauthor=J.+Breedauthor=N.+L.+Daviesauthor=K.+Eckersleyauthor=S.+Filleryauthor=K.+M.+Footeauthor=L.+Goodwinauthor=D.+R.+Jonesauthor=H.+K%C3%A4ckauthor=A.+Lauauthor=J.+W.+M.+Nissinkauthor=J.+Readauthor=J.+S.+Scottauthor=B.+Taylorauthor=G.+Walkerauthor=L.+Wisslerauthor=M.+Wylot&title=Potent+and+Selective+Inhibitors+of+MTH1+Probe+Its+Role+in+Cancer+Cell+Survival&doi=10.1021%2Facs.jmedchem.5b01760"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Potent and Selective Inhibitors of MTH1 Probe Its Role in Cancer Cell Survival</span></div><div class="casAuthors">Kettle, Jason G.; Alwan, Husam; Bista, Michal; Breed, Jason; Davies, Nichola L.; Eckersley, Kay; Fillery, Shaun; Foote, Kevin M.; Goodwin, Louise; Jones, David R.; Kack, Helena; Lau, Alan; Nissink, J. Willem M.; Read, Jon; Scott, James S.; Taylor, Ben; Walker, Graeme; Wissler, Lisa; Wylot, Marta</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2346-2361</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Recent literature has claimed that inhibition of the enzyme MTH1 can eradicate cancer.  MTH1 is one of the "housekeeping" enzymes that are responsible for hydrolyzing damaged nucleotides in cells and thus prevent them from being incorporated into DNA.  We have developed orthogonal and chem. distinct tool compds. to those published in the literature to allow us to test the hypothesis that inhibition of MTH1 has wide applicability in the treatment of cancer.  Here we present the work that led to the discovery of three structurally different series of MTH1 inhibitors with excellent potency, selectivity, and proven target engagement in cells.  None of these compds. elicited the reported cellular phenotype, and addnl. siRNA and CRISPR expts. further support these observations.  Critically, the difference between the responses of our highly selective inhibitors and published tool compds. suggests that the effect reported for the latter may be due to off-target cytotoxic effects.  As a result, we conclude that the role of MTH1 in carcinogenesis and utility of its inhibition is yet to be established.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWdM4dWZuW-bVg90H21EOLACvtfcHk0lgcvt7e73aWOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XisFahsrg%253D&md5=70ab2afcf27b69d03908a990daa825af</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01760%26sid%3Dliteratum%253Aachs%26aulast%3DKettle%26aufirst%3DJ.%2BG.%26aulast%3DAlwan%26aufirst%3DH.%26aulast%3DBista%26aufirst%3DM.%26aulast%3DBreed%26aufirst%3DJ.%26aulast%3DDavies%26aufirst%3DN.%2BL.%26aulast%3DEckersley%26aufirst%3DK.%26aulast%3DFillery%26aufirst%3DS.%26aulast%3DFoote%26aufirst%3DK.%2BM.%26aulast%3DGoodwin%26aufirst%3DL.%26aulast%3DJones%26aufirst%3DD.%2BR.%26aulast%3DK%25C3%25A4ck%26aufirst%3DH.%26aulast%3DLau%26aufirst%3DA.%26aulast%3DNissink%26aufirst%3DJ.%2BW.%2BM.%26aulast%3DRead%26aufirst%3DJ.%26aulast%3DScott%26aufirst%3DJ.%2BS.%26aulast%3DTaylor%26aufirst%3DB.%26aulast%3DWalker%26aufirst%3DG.%26aulast%3DWissler%26aufirst%3DL.%26aulast%3DWylot%26aufirst%3DM.%26atitle%3DPotent%2520and%2520Selective%2520Inhibitors%2520of%2520MTH1%2520Probe%2520Its%2520Role%2520in%2520Cancer%2520Cell%2520Survival%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26issue%3D6%26spage%3D2346%26epage%3D2361%26doi%3D10.1021%2Facs.jmedchem.5b01760" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Svensson, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jemth, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desroses, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loseva, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helleday, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Högbom, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stenmark, P.</span></span> <span> </span><span class="NLM_article-title">Crystal Structure of Human MTH1 and the 8-Oxo-dGMP Product Complex</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>585</i></span> (<span class="NLM_issue">16</span>),  <span class="NLM_fpage">2617</span>– <span class="NLM_lpage">2621</span>, <span class="refDoi"> DOI: 10.1016/j.febslet.2011.07.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1016%2Fj.febslet.2011.07.017" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=585&publication_year=2011&pages=2617-2621&issue=16&author=L.+M.+Svenssonauthor=A.+S.+Jemthauthor=M.+Desrosesauthor=O.+Losevaauthor=T.+Helledayauthor=M.+H%C3%B6gbomauthor=P.+Stenmark&title=Crystal+Structure+of+Human+MTH1+and+the+8-Oxo-dGMP+Product+Complex&doi=10.1016%2Fj.febslet.2011.07.017"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.febslet.2011.07.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.febslet.2011.07.017%26sid%3Dliteratum%253Aachs%26aulast%3DSvensson%26aufirst%3DL.%2BM.%26aulast%3DJemth%26aufirst%3DA.%2BS.%26aulast%3DDesroses%26aufirst%3DM.%26aulast%3DLoseva%26aufirst%3DO.%26aulast%3DHelleday%26aufirst%3DT.%26aulast%3DH%25C3%25B6gbom%26aufirst%3DM.%26aulast%3DStenmark%26aufirst%3DP.%26atitle%3DCrystal%2520Structure%2520of%2520Human%2520MTH1%2520and%2520the%25208-Oxo-dGMP%2520Product%2520Complex%26jtitle%3DFEBS%2520Lett.%26date%3D2011%26volume%3D585%26issue%3D16%26spage%3D2617%26epage%3D2621%26doi%3D10.1016%2Fj.febslet.2011.07.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Petrocchi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reyna, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamilton, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parker, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mseeh, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bardenhagen, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leonard, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cross, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardozo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Draetta, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marszalek, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toniatti, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, R. T.</span></span> <span> </span><span class="NLM_article-title">Identification of Potent and Selective MTH1 Inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>26</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">1503</span>– <span class="NLM_lpage">1507</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2016.02.026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1016%2Fj.bmcl.2016.02.026" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=1503-1507&issue=6&author=A.+Petrocchiauthor=E.+Leoauthor=N.+J.+Reynaauthor=M.+M.+Hamiltonauthor=X.+Shiauthor=C.+A.+Parkerauthor=F.+Mseehauthor=J.+P.+Bardenhagenauthor=P.+Leonardauthor=J.+B.+Crossauthor=S.+Huangauthor=Y.+Jiangauthor=M.+Cardozoauthor=G.+Draettaauthor=J.+R.+Marszalekauthor=C.+Toniattiauthor=P.+Jonesauthor=R.+T.+Lewis&title=Identification+of+Potent+and+Selective+MTH1+Inhibitors&doi=10.1016%2Fj.bmcl.2016.02.026"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2016.02.026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2016.02.026%26sid%3Dliteratum%253Aachs%26aulast%3DPetrocchi%26aufirst%3DA.%26aulast%3DLeo%26aufirst%3DE.%26aulast%3DReyna%26aufirst%3DN.%2BJ.%26aulast%3DHamilton%26aufirst%3DM.%2BM.%26aulast%3DShi%26aufirst%3DX.%26aulast%3DParker%26aufirst%3DC.%2BA.%26aulast%3DMseeh%26aufirst%3DF.%26aulast%3DBardenhagen%26aufirst%3DJ.%2BP.%26aulast%3DLeonard%26aufirst%3DP.%26aulast%3DCross%26aufirst%3DJ.%2BB.%26aulast%3DHuang%26aufirst%3DS.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DCardozo%26aufirst%3DM.%26aulast%3DDraetta%26aufirst%3DG.%26aulast%3DMarszalek%26aufirst%3DJ.%2BR.%26aulast%3DToniatti%26aufirst%3DC.%26aulast%3DJones%26aufirst%3DP.%26aulast%3DLewis%26aufirst%3DR.%2BT.%26atitle%3DIdentification%2520of%2520Potent%2520and%2520Selective%2520MTH1%2520Inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D26%26issue%3D6%26spage%3D1503%26epage%3D1507%26doi%3D10.1016%2Fj.bmcl.2016.02.026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kawamura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawatani, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muroi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kondoh, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Futamura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aono, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Honda, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osada, H.</span></span> <span> </span><span class="NLM_article-title">Proteomic Profiling of Small-Molecule Inhibitors Reveals Dispensability of MTH1 for Cancer Cell Survival</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">26521</span>, <span class="refDoi"> DOI: 10.1038/srep26521</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1038%2Fsrep26521" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=27210421" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BC28XosVSktLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=26521&author=T.+Kawamuraauthor=M.+Kawataniauthor=M.+Muroiauthor=Y.+Kondohauthor=Y.+Futamuraauthor=H.+Aonoauthor=M.+Tanakaauthor=K.+Hondaauthor=H.+Osada&title=Proteomic+Profiling+of+Small-Molecule+Inhibitors+Reveals+Dispensability+of+MTH1+for+Cancer+Cell+Survival&doi=10.1038%2Fsrep26521"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Proteomic profiling of small-molecule inhibitors reveals dispensability of MTH1 for cancer cell survival</span></div><div class="casAuthors">Kawamura, Tatsuro; Kawatani, Makoto; Muroi, Makoto; Kondoh, Yasumitsu; Futamura, Yushi; Aono, Harumi; Tanaka, Miho; Honda, Kaori; Osada, Hiroyuki</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">26521</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Since recent publications suggested that the survival of cancer cells depends on MTH1 to avoid incorporation of oxidized nucleotides into the cellular DNA, MTH1 has attracted attention as a potential cancer therapeutic target.  In this study, we identified new purine-based MTH1 inhibitors by chem. array screening.  However, although the MTH1 inhibitors identified in this study targeted cellular MTH1, they exhibited only weak cytotoxicity against cancer cells compared to recently reported first-in-class inhibitors.  We performed proteomic profiling to investigate the modes of action by which chem. distinct MTH1 inhibitors induce cancer cell death, and found mechanistic differences among the first-in-class MTH1 inhibitors.  In particular, we identified tubulin as the primary target of TH287 and TH588 responsible for the antitumor effects despite the nanomolar MTH1-inhibitory activity in vitro.  Furthermore, overexpression of MTH1 did not rescue cells from MTH1 inhibitor-induced cell death, and siRNA-mediated knockdown of MTH1 did not suppress cancer cell growth.  Taken together, we conclude that the cytotoxicity of MTH1 inhibitors is attributable to off-target effects and that MTH1 is not essential for cancer cell survival.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjRxqUq-o5ybVg90H21EOLACvtfcHk0ljp87J8p8fqHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XosVSktLc%253D&md5=9d0e0e002f680456fccdf426d8f23539</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fsrep26521&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep26521%26sid%3Dliteratum%253Aachs%26aulast%3DKawamura%26aufirst%3DT.%26aulast%3DKawatani%26aufirst%3DM.%26aulast%3DMuroi%26aufirst%3DM.%26aulast%3DKondoh%26aufirst%3DY.%26aulast%3DFutamura%26aufirst%3DY.%26aulast%3DAono%26aufirst%3DH.%26aulast%3DTanaka%26aufirst%3DM.%26aulast%3DHonda%26aufirst%3DK.%26aulast%3DOsada%26aufirst%3DH.%26atitle%3DProteomic%2520Profiling%2520of%2520Small-Molecule%2520Inhibitors%2520Reveals%2520Dispensability%2520of%2520MTH1%2520for%2520Cancer%2520Cell%2520Survival%26jtitle%3DSci.%2520Rep.%26date%3D2016%26volume%3D6%26spage%3D26521%26doi%3D10.1038%2Fsrep26521" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaelin, W. G.</span></span> <span> </span><span class="NLM_article-title">Common Pitfalls in Preclinical Cancer Target Validation</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>17</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">425</span>, <span class="refDoi"> DOI: 10.1038/nrc.2017.32</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1038%2Fnrc.2017.32" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=28524181" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BC2sXotVKhsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2017&pages=425&issue=7&author=W.+G.+Kaelin&title=Common+Pitfalls+in+Preclinical+Cancer+Target+Validation&doi=10.1038%2Fnrc.2017.32"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Common pitfalls in preclinical cancer target validation</span></div><div class="casAuthors">Kaelin, William G., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">425-440</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">An alarming no. of papers from labs. nominating new cancer drug targets contain findings that cannot be reproduced by others or are simply not robust enough to justify drug discovery efforts.  This problem probably has many causes, including an underappreciation of the danger of being misled by off-target effects when using pharmacol. or genetic perturbants in complex biol. assays.  This danger is particularly acute when, as is often the case in cancer pharmacol., the biol. phenotype being measured is a 'down' readout (such as decreased proliferation, decreased viability or decreased tumor growth) that could simply reflect a nonspecific loss of cellular fitness.  These problems are compounded by multiple hypothesis testing, such as when candidate targets emerge from high-throughput screens that interrogate multiple targets in parallel, and by a publication and promotion system that preferentially rewards pos. findings.  In this Perspective, I outline some of the common pitfalls in preclin. cancer target identification and some potential approaches to mitigate them.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoO7ub-MnhM8LVg90H21EOLACvtfcHk0ljp87J8p8fqHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXotVKhsrk%253D&md5=a4ae5a455831ecdd4f481b0f6f548c17</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnrc.2017.32&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc.2017.32%26sid%3Dliteratum%253Aachs%26aulast%3DKaelin%26aufirst%3DW.%2BG.%26atitle%3DCommon%2520Pitfalls%2520in%2520Preclinical%2520Cancer%2520Target%2520Validation%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2017%26volume%3D17%26issue%3D7%26spage%3D425%26doi%3D10.1038%2Fnrc.2017.32" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Samaranayake, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huynh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rai, P.</span></span> <span> </span><span class="NLM_article-title">MTH1 as a Chemotherapeutic Target: The Elephant in the Room</span>. <i>Cancers</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">47</span>, <span class="refDoi"> DOI: 10.3390/cancers9050047</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.3390%2Fcancers9050047" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=47&issue=5&author=G.+J.+Samaranayakeauthor=M.+Huynhauthor=P.+Rai&title=MTH1+as+a+Chemotherapeutic+Target%3A+The+Elephant+in+the+Room&doi=10.3390%2Fcancers9050047"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.3390%2Fcancers9050047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fcancers9050047%26sid%3Dliteratum%253Aachs%26aulast%3DSamaranayake%26aufirst%3DG.%2BJ.%26aulast%3DHuynh%26aufirst%3DM.%26aulast%3DRai%26aufirst%3DP.%26atitle%3DMTH1%2520as%2520a%2520Chemotherapeutic%2520Target%253A%2520The%2520Elephant%2520in%2520the%2520Room%26jtitle%3DCancers%26date%3D2017%26volume%3D9%26issue%3D5%26spage%3D47%26doi%3D10.3390%2Fcancers9050047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Audia, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Austin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blagg, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bountra, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buser-Doepner, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahlin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhanak, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimshaw, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hepworth, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kotz, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruger, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mader, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsden, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller-Fahrnow, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Hagan, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overington, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenberg, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundström, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Superti-Furga, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taunton, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toledo-Sherman, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walpole, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walters, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willson, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Workman, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuercher, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frederiksen, M.</span></span> <span> </span><span class="NLM_article-title">The Promise and Peril of Chemical Probes</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">536</span>– <span class="NLM_lpage">541</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1867</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1038%2Fnchembio.1867" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=26196764" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Wlu7zO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=536-541&issue=8&author=C.+H.+Arrowsmithauthor=J.+E.+Audiaauthor=C.+Austinauthor=J.+Baellauthor=J.+Bennettauthor=J.+Blaggauthor=C.+Bountraauthor=P.+E.+Brennanauthor=P.+J.+Brownauthor=M.+E.+Bunnageauthor=C.+Buser-Doepnerauthor=R.+M.+Campbellauthor=A.+J.+Carterauthor=P.+Cohenauthor=R.+A.+Copelandauthor=B.+Cravattauthor=J.+L.+Dahlinauthor=D.+Dhanakauthor=A.+M.+Edwardsauthor=S.+V.+Fryeauthor=N.+Grayauthor=C.+E.+Grimshawauthor=D.+Hepworthauthor=T.+Howeauthor=K.+V.+M+Huberauthor=J.+Jinauthor=S.+Knappauthor=J.+D.+Kotzauthor=R.+G.+Krugerauthor=D.+Loweauthor=M.+M.+Maderauthor=B.+Marsdenauthor=A.+Mueller-Fahrnowauthor=S.+M%C3%BCllerauthor=R.+C.+O%E2%80%99Haganauthor=J.+P.+Overingtonauthor=D.+R.+Owenauthor=S.+H.+Rosenbergauthor=B.+Rothauthor=R.+Rossauthor=M.+Schapiraauthor=S.+L.+Schreiberauthor=B.+Shoichetauthor=M.+Sundstr%C3%B6mauthor=G.+Superti-Furgaauthor=J.+Tauntonauthor=L.+Toledo-Shermanauthor=C.+Walpoleauthor=M.+A.+Waltersauthor=T.+M.+Willsonauthor=P.+Workmanauthor=R.+N.+Youngauthor=W.+J.+Zuercherauthor=M.+Frederiksen&title=The+Promise+and+Peril+of+Chemical+Probes&doi=10.1038%2Fnchembio.1867"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">The promise and peril of chemical probes</span></div><div class="casAuthors">Arrowsmith, Cheryl H.; Audia, James E.; Austin, Christopher; Baell, Jonathan; Bennett, Jonathan; Blagg, Julian; Bountra, Chas; Brennan, Paul E.; Brown, Peter J.; Bunnage, Mark E.; Buser-Doepner, Carolyn; Campbell, Robert M.; Carter, Adrian J.; Cohen, Philip; Copeland, Robert A.; Cravatt, Ben; Dahlin, Jayme L.; Dhanak, Dashyant; Edwards, Aled M.; Frye, Stephen V.; Gray, Nathanael; Grimshaw, Charles E.; Hepworth, David; Howe, Trevor; Huber, Kilian V. M.; Jin, Jian; Knapp, Stefan; Kotz, Joanne D.; Kruger, Ryan G.; Lowe, Derek; Mader, Mary M.; Marsden, Brian; Mueller-Fahrnow, Anke; Muller, Susanne; O'Hagan, Ronan C.; Overington, John P.; Owen, Dafydd R.; Rosenberg, Saul H.; Roth, Brian; Ross, Ruth; Schapira, Matthieu; Schreiber, Stuart L.; Shoichet, Brian; Sundstrom, Michael; Superti-Furga, Giulio; Taunton, Jack; Toledo-Sherman, Leticia; Walpole, Chris; Walters, Michael A.; Willson, Timothy M.; Workman, Paul; Young, Robert N.; Zuercher, William J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">536-541</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Chem. probes are powerful reagents with increasing impacts on biomedical research.  However, probes of poor quality or that are used incorrectly generate misleading results.  To help address these shortcomings, we will create a community-driven wiki resource to improve quality and convey current best practice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-li4dxq487rVg90H21EOLACvtfcHk0ljaJsjAg6Iv5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Wlu7zO&md5=d8c7a7af94921ab4cdcc871487cceadd</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1867&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1867%26sid%3Dliteratum%253Aachs%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26aulast%3DAustin%26aufirst%3DC.%26aulast%3DBaell%26aufirst%3DJ.%26aulast%3DBennett%26aufirst%3DJ.%26aulast%3DBlagg%26aufirst%3DJ.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DBuser-Doepner%26aufirst%3DC.%26aulast%3DCampbell%26aufirst%3DR.%2BM.%26aulast%3DCarter%26aufirst%3DA.%2BJ.%26aulast%3DCohen%26aufirst%3DP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DCravatt%26aufirst%3DB.%26aulast%3DDahlin%26aufirst%3DJ.%2BL.%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DEdwards%26aufirst%3DA.%2BM.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DGray%26aufirst%3DN.%26aulast%3DGrimshaw%26aufirst%3DC.%2BE.%26aulast%3DHepworth%26aufirst%3DD.%26aulast%3DHowe%26aufirst%3DT.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DKotz%26aufirst%3DJ.%2BD.%26aulast%3DKruger%26aufirst%3DR.%2BG.%26aulast%3DLowe%26aufirst%3DD.%26aulast%3DMader%26aufirst%3DM.%2BM.%26aulast%3DMarsden%26aufirst%3DB.%26aulast%3DMueller-Fahrnow%26aufirst%3DA.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Hagan%26aufirst%3DR.%2BC.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DRosenberg%26aufirst%3DS.%2BH.%26aulast%3DRoth%26aufirst%3DB.%26aulast%3DRoss%26aufirst%3DR.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DShoichet%26aufirst%3DB.%26aulast%3DSundstr%25C3%25B6m%26aufirst%3DM.%26aulast%3DSuperti-Furga%26aufirst%3DG.%26aulast%3DTaunton%26aufirst%3DJ.%26aulast%3DToledo-Sherman%26aufirst%3DL.%26aulast%3DWalpole%26aufirst%3DC.%26aulast%3DWalters%26aufirst%3DM.%2BA.%26aulast%3DWillson%26aufirst%3DT.%2BM.%26aulast%3DWorkman%26aufirst%3DP.%26aulast%3DYoung%26aufirst%3DR.%2BN.%26aulast%3DZuercher%26aufirst%3DW.%2BJ.%26aulast%3DFrederiksen%26aufirst%3DM.%26atitle%3DThe%2520Promise%2520and%2520Peril%2520of%2520Chemical%2520Probes%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26issue%3D8%26spage%3D536%26epage%3D541%26doi%3D10.1038%2Fnchembio.1867" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blagg, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Workman, P.</span></span> <span> </span><span class="NLM_article-title">Choose and Use Your Chemical Probe Wisely to Explore Cancer Biology</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>32</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">9</span>– <span class="NLM_lpage">25</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2017.06.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1016%2Fj.ccell.2017.06.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=28697345" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFGmsLvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2017&pages=9-25&issue=1&author=J.+Blaggauthor=P.+Workman&title=Choose+and+Use+Your+Chemical+Probe+Wisely+to+Explore+Cancer+Biology&doi=10.1016%2Fj.ccell.2017.06.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Choose and Use Your Chemical Probe Wisely to Explore Cancer Biology</span></div><div class="casAuthors">Blagg, Julian; Workman, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">9-25</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Small-mol. chem. probes or tools have become progressively more important in recent years as valuable reagents to investigate fundamental biol. mechanisms and processes causing disease, including cancer.  Chem. probes have also achieved greater prominence alongside complementary biol. reagents for target validation in drug discovery.  However, there is evidence of widespread continuing misuse and promulgation of poor-quality and insufficiently selective chem. probes, perpetuating a worrisome and misleading pollution of the scientific literature.  We discuss current challenges with the selection and use of chem. probes, and suggest how biologists can and should be more discriminating in the probes they employ.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcXzLHHyTscrVg90H21EOLACvtfcHk0ljaJsjAg6Iv5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFGmsLvI&md5=468848d484d5d91c52330949d5778b1d</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2017.06.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2017.06.005%26sid%3Dliteratum%253Aachs%26aulast%3DBlagg%26aufirst%3DJ.%26aulast%3DWorkman%26aufirst%3DP.%26atitle%3DChoose%2520and%2520Use%2520Your%2520Chemical%2520Probe%2520Wisely%2520to%2520Explore%2520Cancer%2520Biology%26jtitle%3DCancer%2520Cell%26date%3D2017%26volume%3D32%26issue%3D1%26spage%3D9%26epage%3D25%26doi%3D10.1016%2Fj.ccell.2017.06.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miyaura, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamada, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, A.</span></span> <span> </span><span class="NLM_article-title">A New Stereospecific Cross-Coupling by the Palladium-Catalyzed Reaction of 1-Alkenylboranes with 1-Alkenyl or 1-Alkynyl Halides</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">1979</span>,  <span class="NLM_volume"><i>20</i></span> (<span class="NLM_issue">36</span>),  <span class="NLM_fpage">3437</span>– <span class="NLM_lpage">3440</span>, <span class="refDoi"> DOI: 10.1016/S0040-4039(01)95429-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1016%2FS0040-4039%2801%2995429-2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=1979&pages=3437-3440&issue=36&author=N.+Miyauraauthor=K.+Yamadaauthor=A.+Suzuki&title=A+New+Stereospecific+Cross-Coupling+by+the+Palladium-Catalyzed+Reaction+of+1-Alkenylboranes+with+1-Alkenyl+or+1-Alkynyl+Halides&doi=10.1016%2FS0040-4039%2801%2995429-2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2FS0040-4039%2801%2995429-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0040-4039%252801%252995429-2%26sid%3Dliteratum%253Aachs%26aulast%3DMiyaura%26aufirst%3DN.%26aulast%3DYamada%26aufirst%3DK.%26aulast%3DSuzuki%26aufirst%3DA.%26atitle%3DA%2520New%2520Stereospecific%2520Cross-Coupling%2520by%2520the%2520Palladium-Catalyzed%2520Reaction%2520of%25201-Alkenylboranes%2520with%25201-Alkenyl%2520or%25201-Alkynyl%2520Halides%26jtitle%3DTetrahedron%2520Lett.%26date%3D1979%26volume%3D20%26issue%3D36%26spage%3D3437%26epage%3D3440%26doi%3D10.1016%2FS0040-4039%2801%2995429-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guram, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rennels, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchwald, S. L.</span></span> <span> </span><span class="NLM_article-title">A Simple Catalytic Method for the Conversion of Aryl Bromides to Arylamines</span>. <i>Angew. Chem., Int. Ed. Engl.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>34</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">1348</span>– <span class="NLM_lpage">1350</span>, <span class="refDoi"> DOI: 10.1002/anie.199513481</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1002%2Fanie.199513481" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADyaK2MXmvV2msr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=1995&pages=1348-1350&issue=12&author=A.+S.+Guramauthor=R.+A.+Rennelsauthor=S.+L.+Buchwald&title=A+Simple+Catalytic+Method+for+the+Conversion+of+Aryl+Bromides+to+Arylamines&doi=10.1002%2Fanie.199513481"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">A simple catalytic method for the conversion of aryl bromides to arylamines</span></div><div class="casAuthors">Guram, Anil S.; Rennels, Roger A.; Buchwald, Stephen L.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition in English</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1348-50</span>CODEN:
                <span class="NLM_cas:coden">ACIEAY</span>;
        ISSN:<span class="NLM_cas:issn">0570-0833</span>.
    
            (<span class="NLM_cas:orgname">VCH</span>)
        </div><div class="casAbstract">Aryl bromides reacted with amines in the presence of NaOCMe3 and a catalytic amt. of PdCl2[P(o-tolyl)3]2 or Pd(dba)2/2P(o-tolyl)3 to give arylamines in good yields.  Also, cyclization of 2-XC6H4CH2(CH2)nNHCH2Ph (X = Br, iodo; n = 1, 2, 3) in the presence of NaOCMe3 and a catalytic amt. of Pd(PPh3)4 gave good yields of the N heterocycles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXDC5Vhs2d57Vg90H21EOLACvtfcHk0lhfuJN83Sbfcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXmvV2msr8%253D&md5=d435e21c3f535757455d53e12417596b</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1002%2Fanie.199513481&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.199513481%26sid%3Dliteratum%253Aachs%26aulast%3DGuram%26aufirst%3DA.%2BS.%26aulast%3DRennels%26aufirst%3DR.%2BA.%26aulast%3DBuchwald%26aufirst%3DS.%2BL.%26atitle%3DA%2520Simple%2520Catalytic%2520Method%2520for%2520the%2520Conversion%2520of%2520Aryl%2520Bromides%2520to%2520Arylamines%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%2520Engl.%26date%3D1995%26volume%3D34%26issue%3D12%26spage%3D1348%26epage%3D1350%26doi%3D10.1002%2Fanie.199513481" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Louie, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartwig, J. F.</span></span> <span> </span><span class="NLM_article-title">Palladium-Catalyzed Synthesis of Arylamines from Aryl Halides. Mechanistic Studies Lead to Coupling in the Absence of Tin Reagents</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>36</i></span> (<span class="NLM_issue">21</span>),  <span class="NLM_fpage">3609</span>– <span class="NLM_lpage">3612</span>, <span class="refDoi"> DOI: 10.1016/0040-4039(95)00605-C</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1016%2F0040-4039%2895%2900605-C" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADyaK2MXmtFCjtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=1995&pages=3609-3612&issue=21&author=J.+Louieauthor=J.+F.+Hartwig&title=Palladium-Catalyzed+Synthesis+of+Arylamines+from+Aryl+Halides.+Mechanistic+Studies+Lead+to+Coupling+in+the+Absence+of+Tin+Reagents&doi=10.1016%2F0040-4039%2895%2900605-C"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Palladium-catalyzed synthesis of arylamines from aryl halides. Mechanistic studies lead to coupling in the absence of tin reagents</span></div><div class="casAuthors">Louie, Janis; Hartwig, John F.</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">3609-12</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The reaction of aryl halides with secondary amines in the presence of silylamide base and tri-o-tolyphospshine palladium complexes gives arylamine products.  This process provides a convenient method for performing these heterocross coupling reactions without the necessity for forming tin amides and disposing of tin halides.  This reaction follows from a mechanistic anal. of the coupling reaction with tin amides and occurs as a result of the cleavage of palladium aryl halide dimers with secondary amines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAKpHJAivoR7Vg90H21EOLACvtfcHk0lhfuJN83Sbfcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXmtFCjtLo%253D&md5=7c4567c3e54efc05049d539c682696b2</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2F0040-4039%2895%2900605-C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0040-4039%252895%252900605-C%26sid%3Dliteratum%253Aachs%26aulast%3DLouie%26aufirst%3DJ.%26aulast%3DHartwig%26aufirst%3DJ.%2BF.%26atitle%3DPalladium-Catalyzed%2520Synthesis%2520of%2520Arylamines%2520from%2520Aryl%2520Halides.%2520Mechanistic%2520Studies%2520Lead%2520to%2520Coupling%2520in%2520the%2520Absence%2520of%2520Tin%2520Reagents%26jtitle%3DTetrahedron%2520Lett.%26date%3D1995%26volume%3D36%26issue%3D21%26spage%3D3609%26epage%3D3612%26doi%3D10.1016%2F0040-4039%2895%2900605-C" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Edfeldt, F. N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Folmer, R. H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breeze, A. L.</span></span> <span> </span><span class="NLM_article-title">Fragment Screening to Predict Druggability (Ligandability) and Lead Discovery Success</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>16</i></span> (<span class="NLM_issue">7–8</span>),  <span class="NLM_fpage">284</span>– <span class="NLM_lpage">287</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2011.02.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1016%2Fj.drudis.2011.02.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=21315179" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BC3MXksFemsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2011&pages=284-287&issue=7%E2%80%938&author=F.+N.+B.+Edfeldtauthor=R.+H.+A.+Folmerauthor=A.+L.+Breeze&title=Fragment+Screening+to+Predict+Druggability+%28Ligandability%29+and+Lead+Discovery+Success&doi=10.1016%2Fj.drudis.2011.02.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment screening to predict druggability (ligandability) and lead discovery success</span></div><div class="casAuthors">Edfeldt, Fredrik N. B.; Folmer, Rutger H. A.; Breeze, Alexander L.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">7/8</span>),
    <span class="NLM_cas:pages">284-287</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">This article discusses the use of fragment screening to predict ligandability which will be of great interest to the pharmaceutical industry.  Data presented in this article clearly show how fragment-based ligandability screening can be used in a truly predictive fashion and could be a powerful tool in reducing agent attrition by filtering out non-ligandable targets before entry into the portfolio.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSXfOy9wAHFLVg90H21EOLACvtfcHk0lhfuJN83Sbfcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXksFemsbs%253D&md5=7ba5adfe3d6a1eb4c9eaa3ce42c60044</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2011.02.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2011.02.002%26sid%3Dliteratum%253Aachs%26aulast%3DEdfeldt%26aufirst%3DF.%2BN.%2BB.%26aulast%3DFolmer%26aufirst%3DR.%2BH.%2BA.%26aulast%3DBreeze%26aufirst%3DA.%2BL.%26atitle%3DFragment%2520Screening%2520to%2520Predict%2520Druggability%2520%2528Ligandability%2529%2520and%2520Lead%2520Discovery%2520Success%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2011%26volume%3D16%26issue%3D7%25E2%2580%25938%26spage%3D284%26epage%3D287%26doi%3D10.1016%2Fj.drudis.2011.02.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Meanwell, N. A.</span></span> <span> </span><span class="NLM_article-title">Improving Drug Candidates by Design: A Focus on Physicochemical Properties As a Means of Improving Compound Disposition and Safety</span>. <i>Chem. Res. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>24</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">1420</span>– <span class="NLM_lpage">1456</span>, <span class="refDoi"> DOI: 10.1021/tx200211v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/tx200211v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BC3MXptlCitLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2011&pages=1420-1456&issue=9&author=N.+A.+Meanwell&title=Improving+Drug+Candidates+by+Design%3A+A+Focus+on+Physicochemical+Properties+As+a+Means+of+Improving+Compound+Disposition+and+Safety&doi=10.1021%2Ftx200211v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Improving Drug Candidates by Design: A Focus on Physicochemical Properties As a Means of Improving Compound Disposition and Safety</span></div><div class="casAuthors">Meanwell, Nicholas A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Research in Toxicology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1420-1456</span>CODEN:
                <span class="NLM_cas:coden">CRTOEC</span>;
        ISSN:<span class="NLM_cas:issn">0893-228X</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The development of small mol. drug candidates from the discovery phase to a marketed product continues to be a challenging enterprise with very low success rates that have fostered the perception of poor productivity by the pharmaceutical industry.  Although there have been significant advances in preclin. profiling that have improved compd. triaging and altered the underlying reasons for compd. attrition, the failure rates have not appreciably changed.  As part of an effort to more deeply understand the reasons for candidate failure, there has been considerable interest in analyzing the physicochem. properties of marketed drugs for the purpose of comparing with drugs in discovery and development as a means capturing recent trends in drug design.  The scenario that has emerged is one in which contemporary drug discovery is thought to be focused too heavily on advancing candidates with profiles that are most easily satisfied by mols. with increased mol. wt. and higher overall lipophilicity.  The preponderance of mols. expressing these properties is frequently a function of increased arom. ring count when compared with that of the drugs launched in the latter half of the 20th century and may reflect a preoccupation with maximizing target affinity rather than taking a more holistic approach to drug design.  These attributes not only present challenges for formulation and absorption but also may influence the manifestation of toxicity during development.  By providing some definition around the optimal physicochem. properties assocd. with marketed drugs, guidelines for drug design have been developed that are based largely on calcd. parameters and which may readily be applied by medicinal chemists as an aid to understanding candidate quality.  The physicochem. properties of a mol. that are consistent with the potential for good oral absorption were initially defined by Lipinski, with addnl. insights allowing further refinement, while deeper analyses have explored the correlation with metabolic stability and toxicity.  These insights have been augmented by careful analyses of physicochem. aspects of drug-target interactions, with thermodn. profiling indicating that the signature of best-in-class drugs is a dependence on enthalpy to drive binding energetics rather than entropy, which is dependent on lipophilicity.  Optimization of the entropic contribution to the binding energy of a ligand to its target is generally much easier than refining the enthalpic element.  Consequently, in the absence of a fundamental understanding of the thermodn. complexion of an interaction, the design of mols. with increased lipophilicity becomes almost inevitable.  The application of ligand efficiency, a measure of affinity per heavy atom, group efficiency, which assesses affinity in the context of structural changes, and lipophilic ligand efficiency, which relates potency to lipophilicity, offer less sophisticated but practically useful anal. algorithms to assess the quality of drug-target interactions.  These parameters are readily calcd. and can be applied to lead optimization programs in a fashion that helps to maximize potency while minimizing the kind of lipophilic burden that has been dubbed "mol. obesity".  Several recently described lead optimization campaigns provide illustrative, informative, and productive examples of the effect of paying close attention to carefully controlling physicochem. properties by monitoring ligand efficiency and lipophilic ligand efficiency.  However, to be successful during the lead optimization phase, drug candidate identification programs will need to adopt a holistic approach that integrates multiple parameters, many of which will have unique dependencies on both the drug target and the specific chemotype under prosecution.  Nevertheless, there are many important drug targets that necessitate working in space beyond that which has been defined by the retrospective analyses of marketed drugs and which will require adaptation of some of the guideposts that are useful in directing lead optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQrUfIwZiIR7Vg90H21EOLACvtfcHk0ljm9uEmPyrerg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXptlCitLs%253D&md5=9ddcf6dc81bf8ab0441df5037979ff4e</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Ftx200211v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Ftx200211v%26sid%3Dliteratum%253Aachs%26aulast%3DMeanwell%26aufirst%3DN.%2BA.%26atitle%3DImproving%2520Drug%2520Candidates%2520by%2520Design%253A%2520A%2520Focus%2520on%2520Physicochemical%2520Properties%2520As%2520a%2520Means%2520of%2520Improving%2520Compound%2520Disposition%2520and%2520Safety%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D2011%26volume%3D24%26issue%3D9%26spage%3D1420%26epage%3D1456%26doi%3D10.1021%2Ftx200211v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groom, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alex, A.</span></span> <span> </span><span class="NLM_article-title">Ligand Efficiency: A Useful Metric for Lead Selection</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">430</span>– <span class="NLM_lpage">431</span>, <span class="refDoi"> DOI: 10.1016/S1359-6446(04)03069-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1016%2FS1359-6446%2804%2903069-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=15109945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A280%3ADC%252BD2c3ht1ygtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2004&pages=430-431&issue=10&author=A.+L.+Hopkinsauthor=C.+R.+Groomauthor=A.+Alex&title=Ligand+Efficiency%3A+A+Useful+Metric+for+Lead+Selection&doi=10.1016%2FS1359-6446%2804%2903069-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand efficiency: a useful metric for lead selection</span></div><div class="casAuthors">Hopkins Andrew L; Groom Colin R; Alex Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">430-1</span>
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSdMyKW_d6RPeIHSmpohpd0fW6udTcc2eYNKH3ii4kIfbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2c3ht1ygtw%253D%253D&md5=eb19a1bca53247d0bd9adcb99d5817e6</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2804%2903069-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252804%252903069-7%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26aulast%3DAlex%26aufirst%3DA.%26atitle%3DLigand%2520Efficiency%253A%2520A%2520Useful%2520Metric%2520for%2520Lead%2520Selection%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2004%26volume%3D9%26issue%3D10%26spage%3D430%26epage%3D431%26doi%3D10.1016%2FS1359-6446%2804%2903069-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hann, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keserü, G. M.</span></span> <span> </span><span class="NLM_article-title">Finding the Sweet Spot: The Role of Nature and Nurture in Medicinal Chemistry</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">355</span>– <span class="NLM_lpage">365</span>, <span class="refDoi"> DOI: 10.1038/nrd3701</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1038%2Fnrd3701" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=22543468" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BC38Xmt1GqsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=355-365&issue=5&author=M.+M.+Hannauthor=G.+M.+Keser%C3%BC&title=Finding+the+Sweet+Spot%3A+The+Role+of+Nature+and+Nurture+in+Medicinal+Chemistry&doi=10.1038%2Fnrd3701"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Finding the sweet spot: the role of nature and nurture in medicinal chemistry</span></div><div class="casAuthors">Hann, Michael M.; Keserue, Gyoergy M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">355-365</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Given its position at the heart of small-mol. drug discovery, medicinal chem. has an important role in tackling the well-known productivity challenges in pharmaceutical research and development.  In recent years, extensive analyses of successful and failed discovery compds. and drug candidates have improved our understanding of the role of physicochem. properties in drug attrition.  Based on the clarified challenges in finding the 'sweet spot' in medicinal chem. programs, we suggest that this goal can be achieved through a combination of first identifying chem. starting points with appropriate 'nature' and then rigorously 'nurturing' them during lead optimization.  Here, we discuss scientific, strategic, organizational and cultural considerations for medicinal chem. practices, with the aim of promoting more effective use of what is already known, as well as a wider appreciation of the risks of pursuing suboptimal compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrURciGjB2A3LVg90H21EOLACvtfcHk0ljm9uEmPyrerg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xmt1GqsLY%253D&md5=90170e3f4e69e52bfa38667df380516a</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fnrd3701&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3701%26sid%3Dliteratum%253Aachs%26aulast%3DHann%26aufirst%3DM.%2BM.%26aulast%3DKeser%25C3%25BC%26aufirst%3DG.%2BM.%26atitle%3DFinding%2520the%2520Sweet%2520Spot%253A%2520The%2520Role%2520of%2520Nature%2520and%2520Nurture%2520in%2520Medicinal%2520Chemistry%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2012%26volume%3D11%26issue%3D5%26spage%3D355%26epage%3D365%26doi%3D10.1038%2Fnrd3701" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mérour, J.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buron, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plé, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonnet, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Routier, S.</span></span> <span> </span><span class="NLM_article-title">The Azaindole Framework in the Design of Kinase Inhibitors</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>19</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">19935</span>– <span class="NLM_lpage">19979</span>, <span class="refDoi"> DOI: 10.3390/molecules191219935</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.3390%2Fmolecules191219935" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2014&pages=19935-19979&issue=12&author=J.-Y.+M%C3%A9rourauthor=F.+Buronauthor=K.+Pl%C3%A9author=P.+Bonnetauthor=S.+Routier&title=The+Azaindole+Framework+in+the+Design+of+Kinase+Inhibitors&doi=10.3390%2Fmolecules191219935"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.3390%2Fmolecules191219935&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules191219935%26sid%3Dliteratum%253Aachs%26aulast%3DM%25C3%25A9rour%26aufirst%3DJ.-Y.%26aulast%3DBuron%26aufirst%3DF.%26aulast%3DPl%25C3%25A9%26aufirst%3DK.%26aulast%3DBonnet%26aufirst%3DP.%26aulast%3DRoutier%26aufirst%3DS.%26atitle%3DThe%2520Azaindole%2520Framework%2520in%2520the%2520Design%2520of%2520Kinase%2520Inhibitors%26jtitle%3DMolecules%26date%3D2014%26volume%3D19%26issue%3D12%26spage%3D19935%26epage%3D19979%26doi%3D10.3390%2Fmolecules191219935" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lovering, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bikker, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humblet, C.</span></span> <span> </span><span class="NLM_article-title">Escape from Flatland: Increasing Saturation as an Approach to Improving Clinical Success</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>52</i></span> (<span class="NLM_issue">21</span>),  <span class="NLM_fpage">6752</span>– <span class="NLM_lpage">6756</span>, <span class="refDoi"> DOI: 10.1021/jm901241e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901241e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1KjtLvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=6752-6756&issue=21&author=F.+Loveringauthor=J.+Bikkerauthor=C.+Humblet&title=Escape+from+Flatland%3A+Increasing+Saturation+as+an+Approach+to+Improving+Clinical+Success&doi=10.1021%2Fjm901241e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Escape from Flatland: Increasing Saturation as an Approach to Improving Clinical Success</span></div><div class="casAuthors">Lovering, Frank; Bikker, Jack; Humblet, Christine</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6752-6756</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The medicinal chem. community has become increasingly aware of the value of tracking calcd. phys. properties such as mol. wt., topol. polar surface area, rotatable bonds, and hydrogen bond donors and acceptors.  The authors hypothesized that the shift to high-throughput synthetic practices over the past decade may be another factor that may predispose mols. to fail by steering discovery efforts toward achiral, arom. compds.  The authors have proposed two simple and interpretable measures of the complexity of mols. prepd. as potential drug candidates.  The first is carbon bond satn. as defined by fraction Sp3 (Fsp3) where Fsp3 = (no. of Sp3 hybridized carbons/total carbon count).  The second is simply whether a chiral carbon exists in the mol.  The authors demonstrate that both complexity (as measured by Fsp3) and the presence of chiral centers correlate with success as compds. transition from discovery, through clin. testing, to drugs.  To explain these observations, the authors further demonstrate that satn. correlates with soly., an exptl. phys. property important to success in the drug discovery setting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoC1f3Ed7nUBbVg90H21EOLACvtfcHk0lhXBMEVRRlGUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1KjtLvN&md5=4ca92c30c17c53d77ad376719bad951e</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fjm901241e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901241e%26sid%3Dliteratum%253Aachs%26aulast%3DLovering%26aufirst%3DF.%26aulast%3DBikker%26aufirst%3DJ.%26aulast%3DHumblet%26aufirst%3DC.%26atitle%3DEscape%2520from%2520Flatland%253A%2520Increasing%2520Saturation%2520as%2520an%2520Approach%2520to%2520Improving%2520Clinical%2520Success%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26issue%3D21%26spage%3D6752%26epage%3D6756%26doi%3D10.1021%2Fjm901241e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, D. V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luscombe, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, A. P.</span></span> <span> </span><span class="NLM_article-title">Getting Physical in Drug Discovery II: The Impact of Chromatographic Hydrophobicity Measurements and Aromaticity</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>16</i></span> (<span class="NLM_issue">17–18</span>),  <span class="NLM_fpage">822</span>– <span class="NLM_lpage">830</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2011.06.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1016%2Fj.drudis.2011.06.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=21704184" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFCru7vF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2011&pages=822-830&issue=17%E2%80%9318&author=R.+J.+Youngauthor=D.+V.+S.+Greenauthor=C.+N.+Luscombeauthor=A.+P.+Hill&title=Getting+Physical+in+Drug+Discovery+II%3A+The+Impact+of+Chromatographic+Hydrophobicity+Measurements+and+Aromaticity&doi=10.1016%2Fj.drudis.2011.06.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Getting physical in drug discovery II: the impact of chromatographic hydrophobicity measurements and aromaticity</span></div><div class="casAuthors">Young, Robert J.; Green, Darren V. S.; Luscombe, Christopher N.; Hill, Alan P.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">17/18</span>),
    <span class="NLM_cas:pages">822-830</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Here, we review the performance of chromatog. hydrophobicity measurements in a data set of 100 000 GlaxoSmithKline compds., demonstrating the advantages of the method over octanol-water partitioning and highlighting new insights for drug discovery.  The value of chromatog. measurements, vs. other hydrophobicity ests., was supported by improved relationships with soly., permeation, cytochrome P450s, intrinsic clearance, hERG binding and promiscuity.  We also obsd. marked differentiation of the relative influence of intrinsic and effective hydrophobicity.  The summing of hydrophobicity values plus arom. ring count [log D pH7.4 (or log P) + #Ar], indicated a wide relevance for simplistic property forecast indexes' in developability assays, clearly enhanced by chromatog. values; therefore establishing new foundations for enriching property-based drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmIJyWi-DBW7Vg90H21EOLACvtfcHk0lhXBMEVRRlGUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFCru7vF&md5=22788603a45aa77537e8666433473d14</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2011.06.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2011.06.001%26sid%3Dliteratum%253Aachs%26aulast%3DYoung%26aufirst%3DR.%2BJ.%26aulast%3DGreen%26aufirst%3DD.%2BV.%2BS.%26aulast%3DLuscombe%26aufirst%3DC.%2BN.%26aulast%3DHill%26aufirst%3DA.%2BP.%26atitle%3DGetting%2520Physical%2520in%2520Drug%2520Discovery%2520II%253A%2520The%2520Impact%2520of%2520Chromatographic%2520Hydrophobicity%2520Measurements%2520and%2520Aromaticity%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2011%26volume%3D16%26issue%3D17%25E2%2580%259318%26spage%3D822%26epage%3D830%26doi%3D10.1016%2Fj.drudis.2011.06.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winiwarter, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ax, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lennernäs, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hallberg, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pettersson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karlén, A.</span></span> <span> </span><span class="NLM_article-title">Hydrogen Bonding Descriptors in the Prediction of Human in Vivo Intestinal Permeability</span>. <i>J. Mol. Graphics Modell.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>21</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">273</span>– <span class="NLM_lpage">287</span>, <span class="refDoi"> DOI: 10.1016/S1093-3263(02)00163-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1016%2FS1093-3263%2802%2900163-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=12479927" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BD38XptlWmur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2003&pages=273-287&issue=4&author=S.+Winiwarterauthor=F.+Axauthor=H.+Lennern%C3%A4sauthor=A.+Hallbergauthor=C.+Petterssonauthor=A.+Karl%C3%A9n&title=Hydrogen+Bonding+Descriptors+in+the+Prediction+of+Human+in+Vivo+Intestinal+Permeability&doi=10.1016%2FS1093-3263%2802%2900163-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Hydrogen bonding descriptors in the prediction of human in vivo intestinal permeability</span></div><div class="casAuthors">Winiwarter, Susanne; Ax, Fredrik; Lennernas, Hans; Hallberg, Anders; Pettersson, Curt; Karlen, Anders</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">273-287</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Inc.</span>)
        </div><div class="casAbstract">Hydrogen bonding has been identified as an important parameter for describing drug permeability.  Recently, we derived models for predicting intestinal permeability using the hydrogen bonding descriptors polar surface area (PSA) and no. of hydrogen bond donors (HBD), and a lipophilicity descriptor [J. Med. hem. 41 (1998) 4939].  We have now explored other types of hydrogen bonding descriptors to see if these improve the models.  Both an exptl. hydrogen bonding descriptor, Δlog P, and calcd. descriptors, based either on semiempirical calcns. or on exptl. derived hydrogen bond strength values of small mols., were used.  Principal component analyses (PCA) were performed in order to characterize the different parameters, using both a drug data set and a data set of small non-drug-like mols. for which Δlog P-values had been published.For a set of diverse drug mols., for which human intestinal permeability data was available, a PLS-anal. was performed to study the correlation of permeability to the different hydrogen bonding parameters.  No correlation could be identified between Δlog P and human intestinal permeability in this data set.  However, the combination of a hydrogen bond donor descriptor, a general hydrogen bonding descriptor and a lipophilicity descriptor enabled the prediction of human intestinal permeability, whereas hydrogen bond acceptor descriptors were found to be less important.  The obtained models successfully predicted the intestinal permeability values of two external data sets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmZy8iv6rXkLVg90H21EOLACvtfcHk0lhXBMEVRRlGUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XptlWmur0%253D&md5=30a2bc60186daf6f649110816e3c323b</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2FS1093-3263%2802%2900163-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1093-3263%252802%252900163-8%26sid%3Dliteratum%253Aachs%26aulast%3DWiniwarter%26aufirst%3DS.%26aulast%3DAx%26aufirst%3DF.%26aulast%3DLennern%25C3%25A4s%26aufirst%3DH.%26aulast%3DHallberg%26aufirst%3DA.%26aulast%3DPettersson%26aufirst%3DC.%26aulast%3DKarl%25C3%25A9n%26aufirst%3DA.%26atitle%3DHydrogen%2520Bonding%2520Descriptors%2520in%2520the%2520Prediction%2520of%2520Human%2520in%2520Vivo%2520Intestinal%2520Permeability%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2003%26volume%3D21%26issue%3D4%26spage%3D273%26epage%3D287%26doi%3D10.1016%2FS1093-3263%2802%2900163-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leeson, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. J.</span></span> <span> </span><span class="NLM_article-title">Molecular Property Design: Does Everyone Get It?</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">722</span>– <span class="NLM_lpage">725</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.5b00157</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00157" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptVGksr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=722-725&issue=7&author=P.+D.+Leesonauthor=R.+J.+Young&title=Molecular+Property+Design%3A+Does+Everyone+Get+It%3F&doi=10.1021%2Facsmedchemlett.5b00157"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Property Design: Does Everyone Get It?</span></div><div class="casAuthors">Leeson, Paul D.; Young, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">722-725</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The principles of mol. property optimization in drug design have been understood for decades, yet much drug discovery activity today is conducted at the periphery of historical druglike property space.  Lead optimization trajectories aimed at reducing physicochem. risk, assisted by ligand efficiency metrics, could help to reduce clin. attrition rates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_t8gIn79arVg90H21EOLACvtfcHk0lhXBMEVRRlGUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptVGksr4%253D&md5=5a807a8d049ce8cf4adc186891a9234d</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00157&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00157%26sid%3Dliteratum%253Aachs%26aulast%3DLeeson%26aufirst%3DP.%2BD.%26aulast%3DYoung%26aufirst%3DR.%2BJ.%26atitle%3DMolecular%2520Property%2520Design%253A%2520Does%2520Everyone%2520Get%2520It%253F%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26issue%3D7%26spage%3D722%26epage%3D725%26doi%3D10.1021%2Facsmedchemlett.5b00157" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hann, M. M.</span></span> <span> </span><span class="NLM_article-title">Molecular Obesity, Potency and Other Addictions in Drug Discovery</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">349</span>– <span class="NLM_lpage">355</span>, <span class="refDoi"> DOI: 10.1039/c1md00017a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1039%2Fc1md00017a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BC3MXls1Kht7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2011&pages=349-355&author=M.+M.+Hann&title=Molecular+Obesity%2C+Potency+and+Other+Addictions+in+Drug+Discovery&doi=10.1039%2Fc1md00017a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular obesity, potency and other addictions in drug discovery</span></div><div class="casAuthors">Hann, Michael M.</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">349-355</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  Despite the increase in global biol. and chem. knowledge the discovery of effective and safe new drugs seems to become harder rather than easier.  Some of this challenge is due to increasing demands for safety and novelty, but some of the risk involved in this should be controllable if we had more effectively learned from our failures.  This perspective reflects on some of the learnings of recent years in relation to the causes of attrition.  The term Mol. Obesity is introduced to describe our tendency to build potency into mols. by the inappropriate use of lipophilicity which leads to the premature demise of drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSlF7TPy6fHLVg90H21EOLACvtfcHk0liMRKsc6JePxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXls1Kht7g%253D&md5=954870db4b1d9e846612c3bb256582f7</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1039%2Fc1md00017a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc1md00017a%26sid%3Dliteratum%253Aachs%26aulast%3DHann%26aufirst%3DM.%2BM.%26atitle%3DMolecular%2520Obesity%252C%2520Potency%2520and%2520Other%2520Addictions%2520in%2520Drug%2520Discovery%26jtitle%3DMedChemComm%26date%3D2011%26volume%3D2%26spage%3D349%26epage%3D355%26doi%3D10.1039%2Fc1md00017a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Testa, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krämer, S. D.</span></span> <span> </span><span class="NLM_article-title">The Biochemistry of Drug Metabolism – An Introduction</span>. <i>Chem. Biodiversity</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>4</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">2031</span>– <span class="NLM_lpage">2122</span>, <span class="refDoi"> DOI: 10.1002/cbdv.200790169</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1002%2Fcbdv.200790169" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2007&pages=2031-2122&issue=9&author=B.+Testaauthor=S.+D.+Kr%C3%A4mer&title=The+Biochemistry+of+Drug+Metabolism+%E2%80%93+An+Introduction&doi=10.1002%2Fcbdv.200790169"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1002%2Fcbdv.200790169&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbdv.200790169%26sid%3Dliteratum%253Aachs%26aulast%3DTesta%26aufirst%3DB.%26aulast%3DKr%25C3%25A4mer%26aufirst%3DS.%2BD.%26atitle%3DThe%2520Biochemistry%2520of%2520Drug%2520Metabolism%2520%25E2%2580%2593%2520An%2520Introduction%26jtitle%3DChem.%2520Biodiversity%26date%3D2007%26volume%3D4%26issue%3D9%26spage%3D2031%26epage%3D2122%26doi%3D10.1002%2Fcbdv.200790169" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mancera, R. L.</span></span> <span> </span><span class="NLM_article-title">Molecular Modeling of Hydration in Drug Design</span>. <i>Curr. Opin. Drug Discovery Devel.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>10</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">275</span>– <span class="NLM_lpage">280</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2007&pages=275-280&issue=3&author=R.+L.+Mancera&title=Molecular+Modeling+of+Hydration+in+Drug+Design"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMancera%26aufirst%3DR.%2BL.%26atitle%3DMolecular%2520Modeling%2520of%2520Hydration%2520in%2520Drug%2520Design%26jtitle%3DCurr.%2520Opin.%2520Drug%2520Discovery%2520Devel.%26date%3D2007%26volume%3D10%26issue%3D3%26spage%3D275%26epage%3D280" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berne, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span> <span> </span><span class="NLM_article-title">Ligand Binding to Protein-Binding Pockets with Wet and Dry Regions</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>108</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">1326</span>– <span class="NLM_lpage">1330</span>, <span class="refDoi"> DOI: 10.1073/pnas.1016793108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1073%2Fpnas.1016793108" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=1326-1330&issue=4&author=L.+Wangauthor=B.+J.+Berneauthor=R.+A.+Friesner&title=Ligand+Binding+to+Protein-Binding+Pockets+with+Wet+and+Dry+Regions&doi=10.1073%2Fpnas.1016793108"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1016793108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1016793108%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DBerne%26aufirst%3DB.%2BJ.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DLigand%2520Binding%2520to%2520Protein-Binding%2520Pockets%2520with%2520Wet%2520and%2520Dry%2520Regions%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26issue%3D4%26spage%3D1326%26epage%3D1330%26doi%3D10.1073%2Fpnas.1016793108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Molina, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jafari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ignatushchenko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsson, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sreekumar, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nordlund, P.</span></span> <span> </span><span class="NLM_article-title">Monitoring Drug Target Engagement in Cells and Tissues Using the Cellular Thermal Shift Assay</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>341</i></span> (<span class="NLM_issue">6141</span>),  <span class="NLM_fpage">84</span>– <span class="NLM_lpage">87</span>, <span class="refDoi"> DOI: 10.1126/science.1233606</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1126%2Fscience.1233606" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=23828940" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVCgs7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=341&publication_year=2013&pages=84-87&issue=6141&author=D.+M.+Molinaauthor=R.+Jafariauthor=M.+Ignatushchenkoauthor=T.+Sekiauthor=E.+A.+Larssonauthor=C.+Danauthor=L.+Sreekumarauthor=Y.+Caoauthor=P.+Nordlund&title=Monitoring+Drug+Target+Engagement+in+Cells+and+Tissues+Using+the+Cellular+Thermal+Shift+Assay&doi=10.1126%2Fscience.1233606"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Monitoring Drug Target Engagement in Cells and Tissues Using the Cellular Thermal Shift Assay</span></div><div class="casAuthors">Molina, Daniel Martinez; Jafari, Rozbeh; Ignatushchenko, Marina; Seki, Takahiro; Larsson, E. Andreas; Dan, Chen; Sreekumar, Lekshmy; Cao, Yihai; Nordlund, Paer</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">341</span>
        (<span class="NLM_cas:issue">6141</span>),
    <span class="NLM_cas:pages">84-87</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The efficacy of therapeutics is dependent on a drug binding to its cognate target.  Optimization of target engagement by drugs in cells is often challenging, because drug binding cannot be monitored inside cells.  We have developed a method for evaluating drug binding to target proteins in cells and tissue samples.  This cellular thermal shift assay (CETSA) is based on the biophys. principle of ligand-induced thermal stabilization of target proteins.  Using this assay, we validated drug binding for a set of important clin. targets and monitored processes of drug transport and activation, off-target effects and drug resistance in cancer cell lines, as well as drug distribution in tissues.  CETSA is likely to become a valuable tool for the validation and optimization of drug target engagement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaaWSoozY7ObVg90H21EOLACvtfcHk0liMRKsc6JePxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVCgs7rP&md5=f8ba686ba38b67dbdf366c49eb7375b4</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1126%2Fscience.1233606&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1233606%26sid%3Dliteratum%253Aachs%26aulast%3DMolina%26aufirst%3DD.%2BM.%26aulast%3DJafari%26aufirst%3DR.%26aulast%3DIgnatushchenko%26aufirst%3DM.%26aulast%3DSeki%26aufirst%3DT.%26aulast%3DLarsson%26aufirst%3DE.%2BA.%26aulast%3DDan%26aufirst%3DC.%26aulast%3DSreekumar%26aufirst%3DL.%26aulast%3DCao%26aufirst%3DY.%26aulast%3DNordlund%26aufirst%3DP.%26atitle%3DMonitoring%2520Drug%2520Target%2520Engagement%2520in%2520Cells%2520and%2520Tissues%2520Using%2520the%2520Cellular%2520Thermal%2520Shift%2520Assay%26jtitle%3DScience%26date%3D2013%26volume%3D341%26issue%3D6141%26spage%3D84%26epage%3D87%26doi%3D10.1126%2Fscience.1233606" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pierce, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krilov, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lupyan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahlgren, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenwood, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romero, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masse, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knight, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinbrecher, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beuming, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damm, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harder, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brewer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wester, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murcko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farid, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mobley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berne, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abel, R.</span></span> <span> </span><span class="NLM_article-title">Accurate and Reliable Prediction of Relative Ligand Binding Potency in Prospective Drug Discovery by Way of a Modern Free-Energy Calculation Protocol and Force Field</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>137</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">2695</span>– <span class="NLM_lpage">2703</span>, <span class="refDoi"> DOI: 10.1021/ja512751q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja512751q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsF2iuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2015&pages=2695-2703&issue=7&author=L.+Wangauthor=Y.+Wuauthor=Y.+Dengauthor=B.+Kimauthor=L.+Pierceauthor=G.+Krilovauthor=D.+Lupyanauthor=S.+Robinsonauthor=M.+K.+Dahlgrenauthor=J.+Greenwoodauthor=D.+L.+Romeroauthor=C.+Masseauthor=J.+L.+Knightauthor=T.+Steinbrecherauthor=T.+Beumingauthor=W.+Dammauthor=E.+Harderauthor=W.+Shermanauthor=M.+Brewerauthor=R.+Westerauthor=M.+Murckoauthor=L.+Fryeauthor=R.+Faridauthor=T.+Linauthor=D.+L.+Mobleyauthor=W.+L.+Jorgensenauthor=B.+J.+Berneauthor=R.+A.+Friesnerauthor=R.+Abel&title=Accurate+and+Reliable+Prediction+of+Relative+Ligand+Binding+Potency+in+Prospective+Drug+Discovery+by+Way+of+a+Modern+Free-Energy+Calculation+Protocol+and+Force+Field&doi=10.1021%2Fja512751q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Accurate and Reliable Prediction of Relative Ligand Binding Potency in Prospective Drug Discovery by Way of a Modern Free-Energy Calculation Protocol and Force Field</span></div><div class="casAuthors">Wang, Lingle; Wu, Yujie; Deng, Yuqing; Kim, Byungchan; Pierce, Levi; Krilov, Goran; Lupyan, Dmitry; Robinson, Shaughnessy; Dahlgren, Markus K.; Greenwood, Jeremy; Romero, Donna L.; Masse, Craig; Knight, Jennifer L.; Steinbrecher, Thomas; Beuming, Thijs; Damm, Wolfgang; Harder, Ed; Sherman, Woody; Brewer, Mark; Wester, Ron; Murcko, Mark; Frye, Leah; Farid, Ramy; Lin, Teng; Mobley, David L.; Jorgensen, William L.; Berne, Bruce J.; Friesner, Richard A.; Abel, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2695-2703</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Designing tight-binding ligands is a primary objective of small-mol. drug discovery.  Over the past few decades, free-energy calcns. have benefited from improved force fields and sampling algorithms, as well as the advent of low-cost parallel computing.  However, it has proven to be challenging to reliably achieve the level of accuracy that would be needed to guide lead optimization (∼5× in binding affinity) for a wide range of ligands and protein targets.  Not surprisingly, widespread com. application of free-energy simulations has been limited due to the lack of large-scale validation coupled with the tech. challenges traditionally assocd. with running these types of calcns.  Here, we report an approach that achieves an unprecedented level of accuracy across a broad range of target classes and ligands, with retrospective results encompassing 200 ligands and a wide variety of chem. perturbations, many of which involve significant changes in ligand chem. structures.  In addn., we have applied the method in prospective drug discovery projects and found a significant improvement in the quality of the compds. synthesized that have been predicted to be potent.  Compds. predicted to be potent by this approach have a substantial redn. in false positives relative to compds. synthesized on the basis of other computational or medicinal chem. approaches.  Furthermore, the results are consistent with those obtained from our retrospective studies, demonstrating the robustness and broad range of applicability of this approach, which can be used to drive decisions in lead optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGor9UtXEyLPpLVg90H21EOLACvtfcHk0lg0qryMLtFL8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsF2iuro%253D&md5=37a4f4a6c085f47ed531342643b6c33b</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fja512751q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja512751q%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DDeng%26aufirst%3DY.%26aulast%3DKim%26aufirst%3DB.%26aulast%3DPierce%26aufirst%3DL.%26aulast%3DKrilov%26aufirst%3DG.%26aulast%3DLupyan%26aufirst%3DD.%26aulast%3DRobinson%26aufirst%3DS.%26aulast%3DDahlgren%26aufirst%3DM.%2BK.%26aulast%3DGreenwood%26aufirst%3DJ.%26aulast%3DRomero%26aufirst%3DD.%2BL.%26aulast%3DMasse%26aufirst%3DC.%26aulast%3DKnight%26aufirst%3DJ.%2BL.%26aulast%3DSteinbrecher%26aufirst%3DT.%26aulast%3DBeuming%26aufirst%3DT.%26aulast%3DDamm%26aufirst%3DW.%26aulast%3DHarder%26aufirst%3DE.%26aulast%3DSherman%26aufirst%3DW.%26aulast%3DBrewer%26aufirst%3DM.%26aulast%3DWester%26aufirst%3DR.%26aulast%3DMurcko%26aufirst%3DM.%26aulast%3DFrye%26aufirst%3DL.%26aulast%3DFarid%26aufirst%3DR.%26aulast%3DLin%26aufirst%3DT.%26aulast%3DMobley%26aufirst%3DD.%2BL.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DBerne%26aufirst%3DB.%2BJ.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DAbel%26aufirst%3DR.%26atitle%3DAccurate%2520and%2520Reliable%2520Prediction%2520of%2520Relative%2520Ligand%2520Binding%2520Potency%2520in%2520Prospective%2520Drug%2520Discovery%2520by%2520Way%2520of%2520a%2520Modern%2520Free-Energy%2520Calculation%2520Protocol%2520and%2520Force%2520Field%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2015%26volume%3D137%26issue%3D7%26spage%3D2695%26epage%3D2703%26doi%3D10.1021%2Fja512751q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rudling, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafsson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almlöf, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Homan, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scobie, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warpman
Berglund, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helleday, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stenmark, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlsson, J.</span></span> <span> </span><span class="NLM_article-title">Fragment-Based Discovery and Optimization of Enzyme Inhibitors by Docking of Commercial Chemical Space</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span> (<span class="NLM_issue">19</span>),  <span class="NLM_fpage">8160</span>– <span class="NLM_lpage">8169</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01006</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01006" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=8160-8169&issue=19&author=A.+Rudlingauthor=R.+Gustafssonauthor=I.+Alml%C3%B6fauthor=E.+Homanauthor=M.+Scobieauthor=U.+Warpman%0ABerglundauthor=T.+Helledayauthor=P.+Stenmarkauthor=J.+Carlsson&title=Fragment-Based+Discovery+and+Optimization+of+Enzyme+Inhibitors+by+Docking+of+Commercial+Chemical+Space&doi=10.1021%2Facs.jmedchem.7b01006"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01006%26sid%3Dliteratum%253Aachs%26aulast%3DRudling%26aufirst%3DA.%26aulast%3DGustafsson%26aufirst%3DR.%26aulast%3DAlml%25C3%25B6f%26aufirst%3DI.%26aulast%3DHoman%26aufirst%3DE.%26aulast%3DScobie%26aufirst%3DM.%26aulast%3DWarpman%2BBerglund%26aufirst%3DU.%26aulast%3DHelleday%26aufirst%3DT.%26aulast%3DStenmark%26aufirst%3DP.%26aulast%3DCarlsson%26aufirst%3DJ.%26atitle%3DFragment-Based%2520Discovery%2520and%2520Optimization%2520of%2520Enzyme%2520Inhibitors%2520by%2520Docking%2520of%2520Commercial%2520Chemical%2520Space%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26issue%3D19%26spage%3D8160%26epage%3D8169%26doi%3D10.1021%2Facs.jmedchem.7b01006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">DeLano, W. L.</span></span> <span> </span><span class="NLM_article-title">Unraveling Hot Spots in Binding Interfaces: Progress and Challenges</span>. <i>Curr. Opin. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>12</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">14</span>– <span class="NLM_lpage">20</span>, <span class="refDoi"> DOI: 10.1016/S0959-440X(02)00283-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1016%2FS0959-440X%2802%2900283-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=11839484" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BD38XhtF2it70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2002&pages=14-20&issue=1&author=W.+L.+DeLano&title=Unraveling+Hot+Spots+in+Binding+Interfaces%3A+Progress+and+Challenges&doi=10.1016%2FS0959-440X%2802%2900283-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Unraveling hot spots in binding interfaces: progress and challenges</span></div><div class="casAuthors">DeLano, Warren L.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">14-20</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review with 49 refs.  Protein interface hot spots, as revealed by alanine scanning mutagenesis, continue to stimulate interest in the biophys. basis of mol. recognition.  Although these regions apparently constitute fertile grounds for intermol. interactions, no general algorithm has yet been developed that can predict hot spots based solely on their shape or compn.  The discovery of structural plasticity in hot spot regions indicates that dynamic simulation techniques may be essential for achieving a predictive understanding of binding interface energetics.  Future progress will depend as much on the application of new computational approaches for dissecting protein interfaces as on expanding the empirical databank of mutagenic substitutions and their effects.  Despite current theor. shortcomings, recent methodol. advances provide efficient exptl. means of probing hot spots and enable immediate applications for hot spots in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpk1HNI-4QfrrVg90H21EOLACvtfcHk0lg0qryMLtFL8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XhtF2it70%253D&md5=a297ae443e69c94b7cf2e342337f29ea</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2FS0959-440X%2802%2900283-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0959-440X%252802%252900283-X%26sid%3Dliteratum%253Aachs%26aulast%3DDeLano%26aufirst%3DW.%2BL.%26atitle%3DUnraveling%2520Hot%2520Spots%2520in%2520Binding%2520Interfaces%253A%2520Progress%2520and%2520Challenges%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2002%26volume%3D12%26issue%3D1%26spage%3D14%26epage%3D20%26doi%3D10.1016%2FS0959-440X%2802%2900283-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blundell, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abell, C.</span></span> <span> </span><span class="NLM_article-title">Probing Hot Spots at Protein-Ligand Binding Sites: A Fragment-Based Approach Using Biophysical Methods</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>49</i></span> (<span class="NLM_issue">16</span>),  <span class="NLM_fpage">4992</span>– <span class="NLM_lpage">5000</span>, <span class="refDoi"> DOI: 10.1021/jm060490r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm060490r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=4992-5000&issue=16&author=A.+Ciulliauthor=G.+Williamsauthor=A.+G.+Smithauthor=T.+L.+Blundellauthor=C.+Abell&title=Probing+Hot+Spots+at+Protein-Ligand+Binding+Sites%3A+A+Fragment-Based+Approach+Using+Biophysical+Methods&doi=10.1021%2Fjm060490r"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fjm060490r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm060490r%26sid%3Dliteratum%253Aachs%26aulast%3DCiulli%26aufirst%3DA.%26aulast%3DWilliams%26aufirst%3DG.%26aulast%3DSmith%26aufirst%3DA.%2BG.%26aulast%3DBlundell%26aufirst%3DT.%2BL.%26aulast%3DAbell%26aufirst%3DC.%26atitle%3DProbing%2520Hot%2520Spots%2520at%2520Protein-Ligand%2520Binding%2520Sites%253A%2520A%2520Fragment-Based%2520Approach%2520Using%2520Biophysical%2520Methods%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26issue%3D16%26spage%3D4992%26epage%3D5000%26doi%3D10.1021%2Fjm060490r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roos, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viklund, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meuller, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaspersson, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svensson, M.</span></span> <span> </span><span class="NLM_article-title">Potency Prediction of β-Secretase (BACE-1) Inhibitors Using Density Functional Methods</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>54</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">818</span>– <span class="NLM_lpage">825</span>, <span class="refDoi"> DOI: 10.1021/ci400374z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci400374z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFKmurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=818-825&issue=3&author=K.+Roosauthor=J.+Viklundauthor=J.+Meullerauthor=K.+Kasperssonauthor=M.+Svensson&title=Potency+Prediction+of+%CE%B2-Secretase+%28BACE-1%29+Inhibitors+Using+Density+Functional+Methods&doi=10.1021%2Fci400374z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Potency Prediction of β-Secretase (BACE-1) Inhibitors Using Density Functional Methods</span></div><div class="casAuthors">Roos, Katarina; Viklund, Jenny; Meuller, Johan; Kaspersson, Karin; Svensson, Mats</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">818-825</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Scoring potency is a main challenge for structure based drug design.  Inductive effects of subtle variations in the ligand are not possible to accurately predict by classical computational chem. methods.  In this study, the problem of predicting potency of ligands with electronic variations participating in key interactions with the protein was addressed.  The potency was predicted for a large set of cyclic amidine and guanidine cores extd. from β-secretase (BACE-1) inhibitors.  All cores were of similar size and had equal interaction motifs but were diverse with respect to electronic substitutions.  A d. functional theory approach, in combination with a representation of the active site of a protein using only key residues, was shown to be predictive.  This computational approach was used to guide and support drug design, within the time frame of a normal drug discovery design cycle.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAMRpKC-p8rLVg90H21EOLACvtfcHk0lgWaIy6guAvJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFKmurk%253D&md5=2b33ce3d50a7234f805723093936a681</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fci400374z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci400374z%26sid%3Dliteratum%253Aachs%26aulast%3DRoos%26aufirst%3DK.%26aulast%3DViklund%26aufirst%3DJ.%26aulast%3DMeuller%26aufirst%3DJ.%26aulast%3DKaspersson%26aufirst%3DK.%26aulast%3DSvensson%26aufirst%3DM.%26atitle%3DPotency%2520Prediction%2520of%2520%25CE%25B2-Secretase%2520%2528BACE-1%2529%2520Inhibitors%2520Using%2520Density%2520Functional%2520Methods%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26issue%3D3%26spage%3D818%26epage%3D825%26doi%3D10.1021%2Fci400374z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zgair, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taha, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kagan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yun, H.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gershkovich, P.</span></span> <span> </span><span class="NLM_article-title">Quantitative Analysis of Lab-to-Lab Variability in Caco-2 Permeability Assays</span>. <i>Eur. J. Pharm. Biopharm.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">38</span>– <span class="NLM_lpage">42</span>, <span class="refDoi"> DOI: 10.1016/j.ejpb.2016.12.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1016%2Fj.ejpb.2016.12.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=28088633" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1entrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=38-42&author=J.+B.+Leeauthor=A.+Zgairauthor=D.+A.+Tahaauthor=X.+Zangauthor=L.+Kaganauthor=T.+H.+Kimauthor=M.+G.+Kimauthor=H.-Y.+Yunauthor=P.+M.+Fischerauthor=P.+Gershkovich&title=Quantitative+Analysis+of+Lab-to-Lab+Variability+in+Caco-2+Permeability+Assays&doi=10.1016%2Fj.ejpb.2016.12.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative analysis of lab-to-lab variability in Caco-2 permeability assays</span></div><div class="casAuthors">Lee, Jong Bong; Zgair, Atheer; Taha, Dhiaa A.; Zang, Xiaowei; Kagan, Leonid; Kim, Tae Hwan; Kim, Min Gi; Yun, Hwi-yeol; Fischer, Peter M.; Gershkovich, Pavel</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Pharmaceutics and Biopharmaceutics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">38-42</span>CODEN:
                <span class="NLM_cas:coden">EJPBEL</span>;
        ISSN:<span class="NLM_cas:issn">0939-6411</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">In this study, Caco-2 permeability results from different labs. were compared.  Six different sets of apparent permeability coeff. (Papp) values reported in the literature were compared to exptl. Papp obtained in our lab.  The differences were assessed by detg. the root mean square error (RMSE) values between the datasets, which reached levels as high as 0.581 for the training set compds., i.e. ten compds. with known effective human permeability (Peff).  The consequences of these differences in Papp for prediction of oral drug absorption were demonstrated by introducing the Papp into the absorption and pharmacokinetics simulation software application GastroPlus for prediction of the fraction absorbed (Fa) in humans using calibrated "user-defined permeability models".  The RMSE were calcd. to assess the differences between the simulated Fa and exptl. values reported in the literature.  The RMSE for Fa simulated with the permeability model calibrated using exptl. Papp from our lab. was 0.128.  When the calibration was performed using Papp from literature datasets, the RMSE values for Fa were higher in all cases except one.  This study shows quant. lab-to-lab variability of Caco-2 permeability results and the potential consequences this can have in the use of these results for predicting intestinal absorption of drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohz3m9Q4Il07Vg90H21EOLACvtfcHk0lgWaIy6guAvJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1entrw%253D&md5=9d823fab5060b4711261ff9e48bf4850</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.ejpb.2016.12.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejpb.2016.12.027%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DJ.%2BB.%26aulast%3DZgair%26aufirst%3DA.%26aulast%3DTaha%26aufirst%3DD.%2BA.%26aulast%3DZang%26aufirst%3DX.%26aulast%3DKagan%26aufirst%3DL.%26aulast%3DKim%26aufirst%3DT.%2BH.%26aulast%3DKim%26aufirst%3DM.%2BG.%26aulast%3DYun%26aufirst%3DH.-Y.%26aulast%3DFischer%26aufirst%3DP.%2BM.%26aulast%3DGershkovich%26aufirst%3DP.%26atitle%3DQuantitative%2520Analysis%2520of%2520Lab-to-Lab%2520Variability%2520in%2520Caco-2%2520Permeability%2520Assays%26jtitle%3DEur.%2520J.%2520Pharm.%2520Biopharm.%26date%3D2017%26volume%3D114%26spage%3D38%26epage%3D42%26doi%3D10.1016%2Fj.ejpb.2016.12.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">XDS</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=XDS&doi=10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lgWaIy6guAvJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXDS%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballard, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keegan, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krissinel, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mccoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mcnicholas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potterton, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, K. S.</span></span> <span> </span><span class="NLM_article-title">Overview of the CCP4 Suite and Current Developments</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1107/S0907444910045749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=235-242&author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=R.+M.+Keeganauthor=E.+B.+Krissinelauthor=A.+G.+W.+Leslieauthor=A.+Mccoyauthor=S.+J.+Mcnicholasauthor=G.+N.+Murshudovauthor=N.+S.+Pannuauthor=E.+A.+Pottertonauthor=H.+R.+Powellauthor=R.+J.+Readauthor=A.+Vaginauthor=K.+S.+Wilson&title=Overview+of+the+CCP4+Suite+and+Current+Developments&doi=10.1107%2FS0907444910045749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0liaDx7ymirCcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DKeegan%26aufirst%3DR.%2BM.%26aulast%3DKrissinel%26aufirst%3DE.%2BB.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26aulast%3DMccoy%26aufirst%3DA.%26aulast%3DMcnicholas%26aufirst%3DS.%2BJ.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DPotterton%26aufirst%3DE.%2BA.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520the%2520CCP4%2520Suite%2520and%2520Current%2520Developments%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D235%26epage%3D242%26doi%3D10.1107%2FS0907444910045749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phaser Crystallographic Software</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+Crystallographic+Software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0liaDx7ymirCcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520Crystallographic%2520Software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and Development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+Development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0liaDx7ymirCcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520Development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span> <span> </span><span class="NLM_article-title">Refinement of Macromolecular Structures by the Maximum-Likelihood Method</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">240</span>– <span class="NLM_lpage">255</span>, <span class="refDoi"> DOI: 10.1107/S0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+Macromolecular+Structures+by+the+Maximum-Likelihood+Method&doi=10.1107%2FS0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0liaDx7ymirCcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520Macromolecular%2520Structures%2520by%2520the%2520Maximum-Likelihood%2520Method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255%26doi%3D10.1107%2FS0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Painter, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merritt, E. A.</span></span> <span> </span><span class="NLM_article-title">TLSMD Web Server for the Generation of Multi-Group TLS Models</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>39</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">109</span>– <span class="NLM_lpage">111</span>, <span class="refDoi"> DOI: 10.1107/S0021889805038987</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1107%2FS0021889805038987" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BD28XmvVyjsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2006&pages=109-111&issue=1&author=J.+Painterauthor=E.+A.+Merritt&title=TLSMD+Web+Server+for+the+Generation+of+Multi-Group+TLS+Models&doi=10.1107%2FS0021889805038987"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">TLSMD web server for the generation of multi-group TLS models</span></div><div class="casAuthors">Painter, Jay; Merritt, Ethan A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">109-111</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">The TLSMD web server exts. information about dynamic properties of a protein based on information derived from a single-crystal structure.  It does so by analyzing the spatial distribution of individual at. thermal parameters present in an input structural model.  The server partitions the protein structure into multiple, contiguous chain segments, each segment corresponding to one group in a multi-group description of the protein's overall dynamic motion.  For each polypeptide chain of the input protein, the anal. generates the optimal partition into two segments, three segments,... up to 20 segments.  Each such partition is optimal in the sense that it is the best approxn. of the overall spatial distribution of input thermal parameters in terms of N chain segments, each acting as a rigid group undergoing TLS (translation/libration/screw) motion.  This multi-group TLS model may be used as a starting point for further crystallog. refinement, or as the basis for analyzing inter-domain and other large-scale motions implied by the crystal structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhkwf2sopjVrVg90H21EOLACvtfcHk0ljhyTBHL7tOsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmvVyjsA%253D%253D&md5=5d16d5d6310baa5434064bdf0ec81e24</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1107%2FS0021889805038987&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889805038987%26sid%3Dliteratum%253Aachs%26aulast%3DPainter%26aufirst%3DJ.%26aulast%3DMerritt%26aufirst%3DE.%2BA.%26atitle%3DTLSMD%2520Web%2520Server%2520for%2520the%2520Generation%2520of%2520Multi-Group%2520TLS%2520Models%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2006%26volume%3D39%26issue%3D1%26spage%3D109%26epage%3D111%26doi%3D10.1107%2FS0021889805038987" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gad, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koolmeister, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jemth, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eshtad, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacques, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ström, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svensson, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lundbäck, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Einarsdottir, B. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saleh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Göktürk, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baranczewski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svensson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berntsson, R. P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafsson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strömberg, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanjiv, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacques-Cordonnier, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desroses, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustavsson, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olofsson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johansson, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Homan, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loseva, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bräutigam, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johansson, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Höglund, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hagenkort, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pham, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altun, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaugaz, F. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vikingsson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evers, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henriksson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vallin, K. S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallner, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammarström, L. G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiita, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almlöf, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalderén, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Axelsson, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Djureinovic, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puigvert, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Häggblad, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeppsson, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martens, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lundin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lundgren, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Granelli, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Artursson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nilsson, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stenmark, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scobie, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berglund, U. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helleday, T.</span></span> <span> </span><span class="NLM_article-title">MTH1 Inhibition Eradicates Cancer by Preventing Sanitation of the dNTP Pool</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>508</i></span> (<span class="NLM_issue">7495</span>),  <span class="NLM_fpage">215</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1038/nature13181</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1038%2Fnature13181" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=24695224" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtlWrsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=508&publication_year=2014&pages=215-221&issue=7495&author=H.+Gadauthor=T.+Koolmeisterauthor=A.+S.+Jemthauthor=S.+Eshtadauthor=S.+A.+Jacquesauthor=C.+E.+Str%C3%B6mauthor=L.+M.+Svenssonauthor=N.+Schultzauthor=T.+Lundb%C3%A4ckauthor=B.+O.+Einarsdottirauthor=A.+Salehauthor=C.+G%C3%B6kt%C3%BCrkauthor=P.+Baranczewskiauthor=R.+Svenssonauthor=R.+P.+A.+Berntssonauthor=R.+Gustafssonauthor=K.+Str%C3%B6mbergauthor=K.+Sanjivauthor=M.+C.+Jacques-Cordonnierauthor=M.+Desrosesauthor=A.+L.+Gustavssonauthor=R.+Olofssonauthor=F.+Johanssonauthor=E.+J.+Homanauthor=O.+Losevaauthor=L.+Br%C3%A4utigamauthor=L.+Johanssonauthor=A.+H%C3%B6glundauthor=A.+Hagenkortauthor=T.+Phamauthor=M.+Altunauthor=F.+Z.+Gaugazauthor=S.+Vikingssonauthor=B.+Eversauthor=M.+Henrikssonauthor=K.+S.+A.+Vallinauthor=O.+A.+Wallnerauthor=L.+G.+J.+Hammarstr%C3%B6mauthor=E.+Wiitaauthor=I.+Alml%C3%B6fauthor=C.+Kalder%C3%A9nauthor=H.+Axelssonauthor=T.+Djureinovicauthor=J.+C.+Puigvertauthor=M.+H%C3%A4ggbladauthor=F.+Jeppssonauthor=U.+Martensauthor=C.+Lundinauthor=B.+Lundgrenauthor=I.+Granelliauthor=A.+J.+Jensenauthor=P.+Arturssonauthor=J.+A.+Nilssonauthor=P.+Stenmarkauthor=M.+Scobieauthor=U.+W.+Berglundauthor=T.+Helleday&title=MTH1+Inhibition+Eradicates+Cancer+by+Preventing+Sanitation+of+the+dNTP+Pool&doi=10.1038%2Fnature13181"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">MTH1 inhibition eradicates cancer by preventing sanitation of the dNTP pool</span></div><div class="casAuthors">Gad, Helge; Koolmeister, Tobias; Jemth, Ann-Sofie; Eshtad, Saeed; Jacques, Sylvain A.; Stroem, Cecilia E.; Svensson, Linda M.; Schultz, Niklas; Lundbaeck, Thomas; Einarsdottir, Berglind Osk; Saleh, Aljona; Gokturk, Camilla; Baranczewski, Pawel; Svensson, Richard; Berntsson, Ronnie P.-A.; Gustafsson, Robert; Stroemberg, Kia; Sanjiv, Kumar; Jacques-Cordonnier, Marie-Caroline; Desroses, Matthieu; Gustavsson, Anna-Lena; Olofsson, Roger; Johansson, Fredrik; Homan, Evert J.; Loseva, Olga; Braeutigam, Lars; Johansson, Lars; Hoeglund, Andreas; Hagenkort, Anna; Pham, Therese; Altun, Mikael; Gaugaz, Fabienne Z.; Vikingsson, Svante; Evers, Bastiaan; Henriksson, Martin; Vallin, Karl S. A.; Wallner, Olov A.; Hammarstroem, Lars G. J.; Wiita, Elisee; Almloef, Ingrid; Kalderen, Christina; Axelsson, Hanna; Djureinovic, Tatjana; Puigvert, Jordi Carreras; Haeggblad, Maria; Jeppsson, Fredrik; Martens, Ulf; Lundin, Cecilia; Lundgren, Bo; Granelli, Ingrid; Jensen, Annika Jenmalm; Artursson, Per; Nilsson, Jonas A.; Stenmark, Pal; Scobie, Martin; Berglund, Ulrika Warpman; Helleday, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">508</span>
        (<span class="NLM_cas:issue">7495</span>),
    <span class="NLM_cas:pages">215-221</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cancers have dysfunctional redox regulation resulting in reactive oxygen species prodn., damaging both DNA and free dNTPs.  The MTH1 protein sanitizes oxidized dNTP pools to prevent incorporation of damaged bases during DNA replication.  Although MTH1 is non-essential in normal cells, the authors show that cancer cells require MTH1 activity to avoid incorporation of oxidized dNTPs, resulting in DNA damage and cell death.  The authors validate MTH1 as an anticancer target in vivo and describe small mols. TH287 (I) and TH588 (II) as first-in-class nudix hydrolase family inhibitors that potently and selectively engage and inhibit the MTH1 protein in cells.  Protein co-crystal structures demonstrate that the inhibitors bind in the active site of MTH1.  The inhibitors cause incorporation of oxidized dNTPs in cancer cells, leading to DNA damage, cytotoxicity and therapeutic responses in patient-derived mouse xenografts.  This study exemplifies the non-oncogene addiction concept for anticancer treatment and validates MTH1 as being cancer phenotypic lethal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-Xn3KhC40XbVg90H21EOLACvtfcHk0ljhyTBHL7tOsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtlWrsLc%253D&md5=49afb4522874f4f5e216ae38d0ff7cd2</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1038%2Fnature13181&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13181%26sid%3Dliteratum%253Aachs%26aulast%3DGad%26aufirst%3DH.%26aulast%3DKoolmeister%26aufirst%3DT.%26aulast%3DJemth%26aufirst%3DA.%2BS.%26aulast%3DEshtad%26aufirst%3DS.%26aulast%3DJacques%26aufirst%3DS.%2BA.%26aulast%3DStr%25C3%25B6m%26aufirst%3DC.%2BE.%26aulast%3DSvensson%26aufirst%3DL.%2BM.%26aulast%3DSchultz%26aufirst%3DN.%26aulast%3DLundb%25C3%25A4ck%26aufirst%3DT.%26aulast%3DEinarsdottir%26aufirst%3DB.%2BO.%26aulast%3DSaleh%26aufirst%3DA.%26aulast%3DG%25C3%25B6kt%25C3%25BCrk%26aufirst%3DC.%26aulast%3DBaranczewski%26aufirst%3DP.%26aulast%3DSvensson%26aufirst%3DR.%26aulast%3DBerntsson%26aufirst%3DR.%2BP.%2BA.%26aulast%3DGustafsson%26aufirst%3DR.%26aulast%3DStr%25C3%25B6mberg%26aufirst%3DK.%26aulast%3DSanjiv%26aufirst%3DK.%26aulast%3DJacques-Cordonnier%26aufirst%3DM.%2BC.%26aulast%3DDesroses%26aufirst%3DM.%26aulast%3DGustavsson%26aufirst%3DA.%2BL.%26aulast%3DOlofsson%26aufirst%3DR.%26aulast%3DJohansson%26aufirst%3DF.%26aulast%3DHoman%26aufirst%3DE.%2BJ.%26aulast%3DLoseva%26aufirst%3DO.%26aulast%3DBr%25C3%25A4utigam%26aufirst%3DL.%26aulast%3DJohansson%26aufirst%3DL.%26aulast%3DH%25C3%25B6glund%26aufirst%3DA.%26aulast%3DHagenkort%26aufirst%3DA.%26aulast%3DPham%26aufirst%3DT.%26aulast%3DAltun%26aufirst%3DM.%26aulast%3DGaugaz%26aufirst%3DF.%2BZ.%26aulast%3DVikingsson%26aufirst%3DS.%26aulast%3DEvers%26aufirst%3DB.%26aulast%3DHenriksson%26aufirst%3DM.%26aulast%3DVallin%26aufirst%3DK.%2BS.%2BA.%26aulast%3DWallner%26aufirst%3DO.%2BA.%26aulast%3DHammarstr%25C3%25B6m%26aufirst%3DL.%2BG.%2BJ.%26aulast%3DWiita%26aufirst%3DE.%26aulast%3DAlml%25C3%25B6f%26aufirst%3DI.%26aulast%3DKalder%25C3%25A9n%26aufirst%3DC.%26aulast%3DAxelsson%26aufirst%3DH.%26aulast%3DDjureinovic%26aufirst%3DT.%26aulast%3DPuigvert%26aufirst%3DJ.%2BC.%26aulast%3DH%25C3%25A4ggblad%26aufirst%3DM.%26aulast%3DJeppsson%26aufirst%3DF.%26aulast%3DMartens%26aufirst%3DU.%26aulast%3DLundin%26aufirst%3DC.%26aulast%3DLundgren%26aufirst%3DB.%26aulast%3DGranelli%26aufirst%3DI.%26aulast%3DJensen%26aufirst%3DA.%2BJ.%26aulast%3DArtursson%26aufirst%3DP.%26aulast%3DNilsson%26aufirst%3DJ.%2BA.%26aulast%3DStenmark%26aufirst%3DP.%26aulast%3DScobie%26aufirst%3DM.%26aulast%3DBerglund%26aufirst%3DU.%2BW.%26aulast%3DHelleday%26aufirst%3DT.%26atitle%3DMTH1%2520Inhibition%2520Eradicates%2520Cancer%2520by%2520Preventing%2520Sanitation%2520of%2520the%2520dNTP%2520Pool%26jtitle%3DNature%26date%3D2014%26volume%3D508%26issue%3D7495%26spage%3D215%26epage%3D221%26doi%3D10.1038%2Fnature13181" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Houston, J. B.</span></span> <span> </span><span class="NLM_article-title">Utility of in Vitro Drug Metabolism Data in Predicting in Vivo Metabolic Clearance</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>47</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">1469</span>– <span class="NLM_lpage">1479</span>, <span class="refDoi"> DOI: 10.1016/0006-2952(94)90520-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1016%2F0006-2952%2894%2990520-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=8185657" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADyaK2cXkt1Kju7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=1994&pages=1469-1479&issue=9&author=J.+B.+Houston&title=Utility+of+in+Vitro+Drug+Metabolism+Data+in+Predicting+in+Vivo+Metabolic+Clearance&doi=10.1016%2F0006-2952%2894%2990520-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Utility of in vitro drug metabolism data in predicting in vivo metabolic clearance</span></div><div class="casAuthors">Houston, J. Brian</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1469-79</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    </div><div class="casAbstract">A review with 54 refs.  The aim of the commentary is to present a strategy to achieve the prediction of in vivo clearance from in vitro data based on the current understanding of metabolite kinetics in vitro and in vivo.  Hepatic microsomes and hepatocyte suspension are the main in vitro system addressed.  The theor. basis of the strategy is outlined and its general applicability evaluated, using a data base of 25 drugs drawn from the literature.  Finally, the prospective value of the approach is discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFHqQ8gCJ6lrVg90H21EOLACvtfcHk0ljjrsgSS-U29g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXkt1Kju7c%253D&md5=a6d0abdbd2f0b238a6a01ba63ee67739</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1016%2F0006-2952%2894%2990520-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0006-2952%252894%252990520-7%26sid%3Dliteratum%253Aachs%26aulast%3DHouston%26aufirst%3DJ.%2BB.%26atitle%3DUtility%2520of%2520in%2520Vitro%2520Drug%2520Metabolism%2520Data%2520in%2520Predicting%2520in%2520Vivo%2520Metabolic%2520Clearance%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D1994%26volume%3D47%26issue%3D9%26spage%3D1469%26epage%3D1479%26doi%3D10.1016%2F0006-2952%2894%2990520-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baranczewski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stańczak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundberg, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svensson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jansson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garberg, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Postlind, H.</span></span> <span> </span><span class="NLM_article-title">Introduction to in Vitro Estimation of Metabolic Stability and Drug Interactions of New Chemical Entities in Drug Discovery and Development</span>. <i>Pharmacol. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>58</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">453</span>– <span class="NLM_lpage">472</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=16963792" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtlarsbnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2006&pages=453-472&issue=4&author=P.+Baranczewskiauthor=A.+Sta%C5%84czakauthor=K.+Sundbergauthor=R.+Svenssonauthor=A.+Wallinauthor=J.+Janssonauthor=P.+Garbergauthor=H.+Postlind&title=Introduction+to+in+Vitro+Estimation+of+Metabolic+Stability+and+Drug+Interactions+of+New+Chemical+Entities+in+Drug+Discovery+and+Development"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Introduction to in vitro estimation of metabolic stability and drug interactions of new chemical entities in drug discovery and development</span></div><div class="casAuthors">Baranczewski, Pawel; Stanczak, Andrzej; Sundberg, Kathrin; Svensson, Richard; Wallin, Aasa; Jansson, Jenny; Garberg, Per; Postlind, Hans</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Reports</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">453-472</span>CODEN:
                <span class="NLM_cas:coden">PRHEDU</span>;
        ISSN:<span class="NLM_cas:issn">1734-1140</span>.
    
            (<span class="NLM_cas:orgname">Polish Academy of Sciences, Institute of Pharmacology</span>)
        </div><div class="casAbstract">A review.  Detn. of metabolic properties of a new chem. entity (NCE) is one of the most important steps during the drug discovery and development process.  Nowadays, in vitro methods are used for early estn. and prediction of in vivo metab. of NCEs.  Using in vitro methods, it is possible to det. the metabolic stability of NCEs as well as the risk for drug-drug interactions (DDIs) related to inhibition and induction of drug metabolic enzymes.  Metabolic stability is defined as the susceptibility of a chem. compd. to biotransformation, and is expressed as in vitro half-life (t1/2) and intrinsic clearance (CLint).  Based on these values, in vivo pharmacokinetic parameters such as bioavailability and in vivo half-life can be calcd.  The drug metabolic enzymes possess broad substrate specificity and can metabolize multiple compds.  Therefore, the risk for metab.-based DDIs is always a potential problem during the drug development process.  For this reason, inhibition and induction in vitro screens are used early, before selection of a candidate drug (CD), to est. the risk for clin. significant DDIs.  At present, most pharmaceutical companies perform in vitro drug metab. studies together with in silica prediction software and automated high-throughput screens (HTS).  Available data suggest that in vitro methods are useful tools for identification and elimination of NCEs with unappreciated metabolic properties.  However, the quant. output of the methods has to be improved.  The aim of this review is to highlight the practical and theor. basis of the in vitro metabolic methods and the recent progress in the development of these assays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeZpDZgTbV2rVg90H21EOLACvtfcHk0ljjrsgSS-U29g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtlarsbnM&md5=fe9b22fe952f43aa6258cbf7d22afda7</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBaranczewski%26aufirst%3DP.%26aulast%3DSta%25C5%2584czak%26aufirst%3DA.%26aulast%3DSundberg%26aufirst%3DK.%26aulast%3DSvensson%26aufirst%3DR.%26aulast%3DWallin%26aufirst%3DA.%26aulast%3DJansson%26aufirst%3DJ.%26aulast%3DGarberg%26aufirst%3DP.%26aulast%3DPostlind%26aufirst%3DH.%26atitle%3DIntroduction%2520to%2520in%2520Vitro%2520Estimation%2520of%2520Metabolic%2520Stability%2520and%2520Drug%2520Interactions%2520of%2520New%2520Chemical%2520Entities%2520in%2520Drug%2520Discovery%2520and%2520Development%26jtitle%3DPharmacol.%2520Rep.%26date%3D2006%26volume%3D58%26issue%3D4%26spage%3D453%26epage%3D472" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 13 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Jinyoung Lee, <span class="NLM_string-name hlFld-ContribAuthor">Matthieu Schapira</span>. </span><span class="cited-content_cbyCitation_article-title">The Promise and Peril of Chemical Probe Negative Controls. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2021,</strong> <em>16 </em>
                                    (4)
                                     , 579-585. <a href="https://doi.org/10.1021/acschembio.1c00036" title="DOI URL">https://doi.org/10.1021/acschembio.1c00036</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.1c00036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.1c00036%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DThe%252BPromise%252Band%252BPeril%252Bof%252BChemical%252BProbe%252BNegative%252BControls%26aulast%3DLee%26aufirst%3DJinyoung%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D18012021%26date%3D05032021%26date%3D21032021%26volume%3D16%26issue%3D4%26spage%3D579%26epage%3D585" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Daniel A. Erlanson, Iwan J. P. de Esch, Wolfgang Jahnke, Christopher N. Johnson, <span class="NLM_string-name hlFld-ContribAuthor">Paul N. Mortenson</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment-to-Lead Medicinal Chemistry Publications in 2018. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (9)
                                     , 4430-4444. <a href="https://doi.org/10.1021/acs.jmedchem.9b01581" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01581</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01581&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01581%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DFragment-to-Lead%252BMedicinal%252BChemistry%252BPublications%252Bin%252B2018%26aulast%3DErlanson%26aufirst%3DDaniel%2BA.%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D22092019%26date%3D22012020%26date%3D08012020%26volume%3D63%26issue%3D9%26spage%3D4430%26epage%3D4444" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Julien Lefranc, Volker Klaus Schulze, Roman Christian Hillig, Hans Briem, Florian Prinz, Anne Mengel, Tobias Heinrich, Jozsef Balint, Srinivasan Rengachari, Horst Irlbacher, Detlef Stöckigt, Ulf Bömer, Benjamin Bader, Stefan Nikolaus Gradl, Carl Friedrich Nising, Franz von Nussbaum, Dominik Mumberg, Daniel Panne, <span class="NLM_string-name hlFld-ContribAuthor">Antje Margret Wengner</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of BAY-985, a Highly Selective TBK1/IKKε Inhibitor. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (2)
                                     , 601-612. <a href="https://doi.org/10.1021/acs.jmedchem.9b01460" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01460</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01460&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01460%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BBAY-985%25252C%252Ba%252BHighly%252BSelective%252BTBK1%25252FIKK%2525CE%2525B5%252BInhibitor%26aulast%3DLefranc%26aufirst%3DJulien%26date%3D2020%26date%3D2019%26date%3D2019%26date%3D02092019%26date%3D10012020%26date%3D20122019%26volume%3D63%26issue%3D2%26spage%3D601%26epage%3D612" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Maurice Michel, Torkild Visnes, Evert J. Homan, Brinton Seashore-Ludlow, Mattias Hedenström, Elisée Wiita, Karl Vallin, Cynthia B. J. Paulin, Jiaxi Zhang, Olov Wallner, Martin Scobie, Andreas Schmidt, Annika Jenmalm-Jensen, Ulrika Warpman Berglund, <span class="NLM_string-name hlFld-ContribAuthor">Thomas Helleday</span>. </span><span class="cited-content_cbyCitation_article-title">Computational and Experimental Druggability Assessment of Human DNA Glycosylases. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2019,</strong> <em>4 </em>
                                    (7)
                                     , 11642-11656. <a href="https://doi.org/10.1021/acsomega.9b00162" title="DOI URL">https://doi.org/10.1021/acsomega.9b00162</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.9b00162&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.9b00162%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DComputational%252Band%252BExperimental%252BDruggability%252BAssessment%252Bof%252BHuman%252BDNA%252BGlycosylases%26aulast%3DMichel%26aufirst%3DMaurice%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D17012019%26date%3D21052019%26volume%3D4%26issue%3D7%26spage%3D11642%26epage%3D11656" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Ye Wang, Wen-Yun Zhang, <span class="NLM_string-name hlFld-ContribAuthor">Shu-Li You</span>. </span><span class="cited-content_cbyCitation_article-title">Ketones and Aldehydes as O-Nucleophiles in Iridium-Catalyzed Intramolecular Asymmetric Allylic Substitution Reaction. </span><span class="cited-content_cbyCitation_journal-name">Journal of the American Chemical Society</span><span> <strong>2019,</strong> <em>141 </em>
                                    (6)
                                     , 2228-2232. <a href="https://doi.org/10.1021/jacs.8b13182" title="DOI URL">https://doi.org/10.1021/jacs.8b13182</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jacs.8b13182&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjacs.8b13182%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520the%2520American%2520Chemical%2520Society%26atitle%3DKetones%252Band%252BAldehydes%252Bas%252BO-Nucleophiles%252Bin%252BIridium-Catalyzed%252BIntramolecular%252BAsymmetric%252BAllylic%252BSubstitution%252BReaction%26aulast%3DWang%26aufirst%3DYe%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D10122018%26date%3D31012019%26date%3D28012019%26volume%3D141%26issue%3D6%26spage%3D2228%26epage%3D2232" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Cheng  Peng</span>, <span class="hlFld-ContribAuthor ">Yu-Hsuan  Li</span>, <span class="hlFld-ContribAuthor ">Chao-Wu  Yu</span>, <span class="hlFld-ContribAuthor ">Ze-Hua  Cheng</span>, <span class="hlFld-ContribAuthor ">Jia-Rong  Liu</span>, <span class="hlFld-ContribAuthor ">Jui-Ling  Hsu</span>, <span class="hlFld-ContribAuthor ">Ling-Wei  Hsin</span>, <span class="hlFld-ContribAuthor ">Chen-Tsung  Huang</span>, <span class="hlFld-ContribAuthor ">Hsueh-Fen  Juan</span>, <span class="hlFld-ContribAuthor ">Ji-Wang  Chern</span>, <span class="hlFld-ContribAuthor ">Yi-Sheng  Cheng</span>. </span><span class="cited-content_cbyCitation_article-title">Inhibitor development of MTH1 via high-throughput screening with fragment based library and MTH1 substrate binding cavity. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2021,</strong> <em>110 </em>, 104813. <a href="https://doi.org/10.1016/j.bioorg.2021.104813" title="DOI URL">https://doi.org/10.1016/j.bioorg.2021.104813</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2021.104813&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2021.104813%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DInhibitor%252Bdevelopment%252Bof%252BMTH1%252Bvia%252Bhigh-throughput%252Bscreening%252Bwith%252Bfragment%252Bbased%252Blibrary%252Band%252BMTH1%252Bsubstrate%252Bbinding%252Bcavity%26aulast%3DPeng%26aufirst%3DCheng%26date%3D2021%26volume%3D110%26spage%3D104813" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yizhen  Yin</span>, <span class="hlFld-ContribAuthor ">Fener  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting human MutT homolog 1 (MTH1) for cancer eradication: current progress and perspectives. </span><span class="cited-content_cbyCitation_journal-name">Acta Pharmaceutica Sinica B</span><span> <strong>2020,</strong> <em>10 </em>
                                    (12)
                                     , 2259-2271. <a href="https://doi.org/10.1016/j.apsb.2020.02.012" title="DOI URL">https://doi.org/10.1016/j.apsb.2020.02.012</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.apsb.2020.02.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.apsb.2020.02.012%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Pharmaceutica%2520Sinica%2520B%26atitle%3DTargeting%252Bhuman%252BMutT%252Bhomolog%252B1%252B%252528MTH1%252529%252Bfor%252Bcancer%252Beradication%25253A%252Bcurrent%252Bprogress%252Band%252Bperspectives%26aulast%3DYin%26aufirst%3DYizhen%26date%3D2020%26volume%3D10%26issue%3D12%26spage%3D2259%26epage%3D2271" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sean G.  Rudd</span>, <span class="hlFld-ContribAuthor ">Helge  Gad</span>, <span class="hlFld-ContribAuthor ">Kumar  Sanjiv</span>, <span class="hlFld-ContribAuthor ">Nuno  Amaral</span>, <span class="hlFld-ContribAuthor ">Anna  Hagenkort</span>, <span class="hlFld-ContribAuthor ">Petra  Groth</span>, <span class="hlFld-ContribAuthor ">Cecilia E.  Ström</span>, <span class="hlFld-ContribAuthor ">Oliver  Mortusewicz</span>, <span class="hlFld-ContribAuthor ">Ulrika Warpman  Berglund</span>, <span class="hlFld-ContribAuthor ">Thomas  Helleday</span>. </span><span class="cited-content_cbyCitation_article-title">MTH1 Inhibitor TH588 Disturbs Mitotic Progression and Induces Mitosis-Dependent Accumulation of Genomic 8-oxodG. </span><span class="cited-content_cbyCitation_journal-name">Cancer Research</span><span> <strong>2020,</strong> <em>80 </em>
                                    (17)
                                     , 3530-3541. <a href="https://doi.org/10.1158/0008-5472.CAN-19-0883" title="DOI URL">https://doi.org/10.1158/0008-5472.CAN-19-0883</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1158/0008-5472.CAN-19-0883&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1158%2F0008-5472.CAN-19-0883%26sid%3Dliteratum%253Aachs%26jtitle%3DCancer%2520Research%26atitle%3DMTH1%252BInhibitor%252BTH588%252BDisturbs%252BMitotic%252BProgression%252Band%252BInduces%252BMitosis-Dependent%252BAccumulation%252Bof%252BGenomic%252B8-oxodG%26aulast%3DRudd%26aufirst%3DSean%2BG.%26date%3D2020%26date%3D2020%26volume%3D80%26issue%3D17%26spage%3D3530%26epage%3D3541" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Govindi J.  Samaranayake</span>, <span class="hlFld-ContribAuthor ">Clara I.  Troccoli</span>, <span class="hlFld-ContribAuthor ">Ling  Zhang</span>, <span class="hlFld-ContribAuthor ">Mai  Huynh</span>, <span class="hlFld-ContribAuthor ">Christina J.  Jayaraj</span>, <span class="hlFld-ContribAuthor ">Debin  Ji</span>, <span class="hlFld-ContribAuthor ">Lisa  McPherson</span>, <span class="hlFld-ContribAuthor ">Yoshiyuki  Onishi</span>, <span class="hlFld-ContribAuthor ">Dao M.  Nguyen</span>, <span class="hlFld-ContribAuthor ">David J.  Robbins</span>, <span class="hlFld-ContribAuthor ">Mahsa  Karbaschi</span>, <span class="hlFld-ContribAuthor ">Marcus S.  Cooke</span>, <span class="hlFld-ContribAuthor ">Antonio  Barrientos</span>, <span class="hlFld-ContribAuthor ">Eric T.  Kool</span>, <span class="hlFld-ContribAuthor ">Priyamvada  Rai</span>. </span><span class="cited-content_cbyCitation_article-title">The Existence of MTH1-independent 8-oxodGTPase Activity in Cancer Cells as a Compensatory Mechanism against On-target Effects of MTH1 Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Molecular Cancer Therapeutics</span><span> <strong>2020,</strong> <em>19 </em>
                                    (2)
                                     , 432-446. <a href="https://doi.org/10.1158/1535-7163.MCT-19-0437" title="DOI URL">https://doi.org/10.1158/1535-7163.MCT-19-0437</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1158/1535-7163.MCT-19-0437&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1158%2F1535-7163.MCT-19-0437%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Cancer%2520Therapeutics%26atitle%3DThe%252BExistence%252Bof%252BMTH1-independent%252B8-oxodGTPase%252BActivity%252Bin%252BCancer%252BCells%252Bas%252Ba%252BCompensatory%252BMechanism%252Bagainst%252BOn-target%252BEffects%252Bof%252BMTH1%252BInhibitors%26aulast%3DSamaranayake%26aufirst%3DGovindi%2BJ.%26date%3D2020%26date%3D2019%26volume%3D19%26issue%3D2%26spage%3D432%26epage%3D446" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lars  Wiedmer</span>, <span class="hlFld-ContribAuthor ">Claude  Schärer</span>, <span class="hlFld-ContribAuthor ">Dimitrios  Spiliotopoulos</span>, <span class="hlFld-ContribAuthor ">Marianne  Hürzeler</span>, <span class="hlFld-ContribAuthor ">Paweł  Śledź</span>, <span class="hlFld-ContribAuthor ">Amedeo  Caflisch</span>. </span><span class="cited-content_cbyCitation_article-title">Ligand retargeting by binding site analogy. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>175 </em>, 107-113. <a href="https://doi.org/10.1016/j.ejmech.2019.04.037" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.04.037</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.04.037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.04.037%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DLigand%252Bretargeting%252Bby%252Bbinding%252Bsite%252Banalogy%26aulast%3DWiedmer%26aufirst%3DLars%26date%3D2019%26volume%3D175%26spage%3D107%26epage%3D113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Priyamvada  Rai</span>, <span class="hlFld-ContribAuthor ">Robert W.  Sobol</span>. </span><span class="cited-content_cbyCitation_article-title">Mechanisms of MTH1 inhibition-induced DNA strand breaks: The slippery slope from the oxidized nucleotide pool to genotoxic damage. </span><span class="cited-content_cbyCitation_journal-name">DNA Repair</span><span> <strong>2019,</strong> <em>77 </em>, 18-26. <a href="https://doi.org/10.1016/j.dnarep.2019.03.001" title="DOI URL">https://doi.org/10.1016/j.dnarep.2019.03.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.dnarep.2019.03.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.dnarep.2019.03.001%26sid%3Dliteratum%253Aachs%26jtitle%3DDNA%2520Repair%26atitle%3DMechanisms%252Bof%252BMTH1%252Binhibition-induced%252BDNA%252Bstrand%252Bbreaks%25253A%252BThe%252Bslippery%252Bslope%252Bfrom%252Bthe%252Boxidized%252Bnucleotide%252Bpool%252Bto%252Bgenotoxic%252Bdamage%26aulast%3DRai%26aufirst%3DPriyamvada%26date%3D2019%26volume%3D77%26spage%3D18%26epage%3D26" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Takeshi  Yokoyama</span>, <span class="hlFld-ContribAuthor ">Ryota  Kitakami</span>, <span class="hlFld-ContribAuthor ">Mineyuki  Mizuguchi</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of a new class of MTH1 inhibitor by X-ray crystallographic screening. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>167 </em>, 153-160. <a href="https://doi.org/10.1016/j.ejmech.2019.02.011" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.02.011</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.02.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.02.011%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Ba%252Bnew%252Bclass%252Bof%252BMTH1%252Binhibitor%252Bby%252BX-ray%252Bcrystallographic%252Bscreening%26aulast%3DYokoyama%26aufirst%3DTakeshi%26date%3D2019%26volume%3D167%26spage%3D153%26epage%3D160" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rui-Juan  Niu</span>, <span class="hlFld-ContribAuthor ">Qing-Chuan  Zheng</span>, <span class="hlFld-ContribAuthor ">Hong-Xing  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">The influence of residue in the position of 116 on the inhibitory potency of TH588 for MTH1. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Graphics and Modelling</span><span> <strong>2018,</strong> <em>85 </em>, 75-83. <a href="https://doi.org/10.1016/j.jmgm.2018.08.002" title="DOI URL">https://doi.org/10.1016/j.jmgm.2018.08.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jmgm.2018.08.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jmgm.2018.08.002%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Graphics%2520and%2520Modelling%26atitle%3DThe%252Binfluence%252Bof%252Bresidue%252Bin%252Bthe%252Bposition%252Bof%252B116%252Bon%252Bthe%252Binhibitory%252Bpotency%252Bof%252BTH588%252Bfor%252BMTH1%26aulast%3DNiu%26aufirst%3DRui-Juan%26date%3D2018%26volume%3D85%26spage%3D75%26epage%3D83" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/medium/jm-2017-01884d_0012.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/large/jm-2017-01884d_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01884&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/medium/jm-2017-01884d_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/large/jm-2017-01884d_0001.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Representations of the key interactions of selected inhibitors of MTH1 (the corresponding PDB ID is shown in parentheses), with varying hydrogen bonding patterns to the Asp119-Asp120 recognition element (dotted lines): 8-oxo-dGMP (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ZR0">3ZR0</a>),<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> (<i>S</i>)-Crizotinib (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4C9X">4C9X</a>),<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> TH588 (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4N1U">4N1U</a>),<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> AZ compound 19 (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ANV">5ANV</a>).<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/large/jm-2017-01884d_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01884&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/medium/jm-2017-01884d_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/large/jm-2017-01884d_0008.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 1. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/large/jm-2017-01884d_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01884&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Representative reagents and conditions: (i) LDA, EtOCOCl, 2-MeTHF, −78 °C to rt; (ii) R1 = aryl: boronic acid reagent, Pd(amphos)Cl<sub>2</sub>, K<sub>2</sub>CO<sub>3</sub>, MeCN·H<sub>2</sub>O, 120 °C; (iii) R1 = −NR2R3: cyclic secondary amine, NMP, 160 °C; (iv) LiOH·H<sub>2</sub>O, THF:MeOH:H<sub>2</sub>O, 50 °C; (v) (a) SOCl<sub>2</sub>, 80 °C, (b) R4R5NH, THF, 0 °C to rt; (vi) LDA, 2-MeTHF, CO<sub>2</sub>, −78 °C to rt; (vii) (a) SOCl<sub>2</sub>, 80 °C, (b) EtNH<sub>2</sub>, THF, 0 °C to rt.</p></p></figure><figure data-id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/medium/jm-2017-01884d_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/large/jm-2017-01884d_0002.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (a) Two-dimensional structure of fragment hit <b>1</b>. Vectors from the 2- and 4-positions illustrate the attachment points for synthetic expansions presented in this work. (b,c) Overlay of complex between MTH1 and <b>1</b> (green, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6F20">6F20</a>) and complex between MTH1 and 8-oxo-dGMP (cyan, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ZR0">3ZR0</a>). Side-chains of Phe72, Trp117, Asp119, and Asp120 in the structure of MTH1–<b>1</b> complex are shown as sticks with their carbon atoms colored green. Water molecules present in the inhibitor binding site of MTH1–<b>1</b> complex are shown as red spheres. The 8-oxo-dGMP is displayed as thin sticks, with its carbon atoms colored cyan. The ester motif occupies the same area as the ribose of 8-oxo-dGMP, and we refer to this area as the ribose pocket. Clipped molecular surface of MTH1 is shown in the MTH1–<b>1</b> complex structure. Panel (c) is rotated 90 deg as compared to panel (b).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/large/jm-2017-01884d_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01884&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/medium/jm-2017-01884d_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/large/jm-2017-01884d_0003.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Crystal structure of <b>7</b>, showing that the 7-azaindole core remains in the same position as the fragment hit <b>1</b>. MTH1–<b>1</b> and MTH1–<b>7</b> complexes are colored green and violet, respectively. Residues interacting with the compounds, and compound themselves, are shown as thin (complex with <b>1</b>) or thick (complex with <b>7</b>) sticks with their carbon atoms colored following the same code as the secondary structure elements. Water molecules present in the binding-site of MTH1–<b>7</b> complex are shown as red spheres. Hydrogen bonds are displayed as orange dashes. Clipped molecular surface of MTH1 is shown in the MTH1–<b>7</b> complex structure (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6F1X">6F1X</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/large/jm-2017-01884d_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01884&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/medium/jm-2017-01884d_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/large/jm-2017-01884d_0004.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Overlay between the structure of MTH1–<b>16</b> (white) and MTH1–<b>1</b> (green). Compounds <b>16</b> and <b>1</b> are displayed as thick and thin sticks, respectively, with their carbon atoms colored following the same code as the secondary structure elements. The side chains of MTH1 residues forming the cavity surrounding the phenyl moiety of <b>16</b> are displayed as sticks. Clipped molecular surface of MTH1 is shown in the MTH1–<b>16</b> complex structure (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6F23">6F23</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/large/jm-2017-01884d_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01884&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/medium/jm-2017-01884d_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/large/jm-2017-01884d_0005.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Overlay between the complex structure of MTH1–<b>29</b> (yellow) and MTH1–<b>1</b> (green). Compounds <b>29</b> and <b>1</b> are displayed as thick and thin sticks, respectively, with their carbon atoms colored following the same code as the secondary structure elements. Side chain of MTH1 residues interacting with the compounds are shown as thin (<b>1</b>) and thick (<b>29</b>) sticks, respectively. Water molecules present in the binding site of <b>29</b> are depicted as red spheres. Hydrogen bonds between MTH1, water molecules, and <b>29</b> are displayed as orange dashes. Clipped molecular surface of MTH1 is shown in the MTH1–<b>29</b> complex structure (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6F22">6F22</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/large/jm-2017-01884d_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01884&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/medium/jm-2017-01884d_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/large/jm-2017-01884d_0006.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. There was a strong correlation between biochemical potency and target engagement in cells, using CETSA. Dotted line shows <i>y</i> = <i>x</i> – 1.5, indicating a drop-off from the biochemical assay to the cellular assay of about 30 times. Three compounds (<b>20</b>, <b>24</b>, and <b>25</b>) had significantly higher pEC<sub>50</sub> in CETSA than the reference TH588.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/large/jm-2017-01884d_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01884&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/medium/jm-2017-01884d_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/large/jm-2017-01884d_0007.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Compounds <b>32</b> and <b>35</b> were created as positive and negative probes for MTH1.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-6/acs.jmedchem.7b01884/20180315/images/large/jm-2017-01884d_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01884&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i70">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_54137" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_54137" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 56 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ellermann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eheim, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahm, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viklund, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guenther, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ericsson, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forsblom, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ginman, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindström, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silvander, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trésaugues, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giese, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunse, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neuhaus, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiske, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quanz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glasauer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak-Reppel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bader, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irlbacher, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Queisser, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauser, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haegebarth, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorjánácz, M.</span></span> <span> </span><span class="NLM_article-title">Novel Class of Potent and Cellularly Active Inhibitors Devalidates MTH1 as Broad-Spectrum Cancer Target</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">1986</span>– <span class="NLM_lpage">1992</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00370</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00370" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=1986-1992&issue=8&author=M.+Ellermannauthor=A.+Eheimauthor=F.+Rahmauthor=J.+Viklundauthor=J.+Guentherauthor=M.+Anderssonauthor=U.+Ericssonauthor=R.+Forsblomauthor=T.+Ginmanauthor=J.+Lindstr%C3%B6mauthor=C.+Silvanderauthor=L.+Tr%C3%A9sauguesauthor=A.+Gieseauthor=S.+Bunseauthor=R.+Neuhausauthor=J.+Weiskeauthor=M.+Quanzauthor=A.+Glasauerauthor=K.+Nowak-Reppelauthor=B.+Baderauthor=H.+Irlbacherauthor=H.+Meyerauthor=N.+Queisserauthor=M.+Bauserauthor=A.+Haegebarthauthor=M.+Gorj%C3%A1n%C3%A1cz&title=Novel+Class+of+Potent+and+Cellularly+Active+Inhibitors+Devalidates+MTH1+as+Broad-Spectrum+Cancer+Target&doi=10.1021%2Facschembio.7b00370"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00370&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00370%26sid%3Dliteratum%253Aachs%26aulast%3DEllermann%26aufirst%3DM.%26aulast%3DEheim%26aufirst%3DA.%26aulast%3DRahm%26aufirst%3DF.%26aulast%3DViklund%26aufirst%3DJ.%26aulast%3DGuenther%26aufirst%3DJ.%26aulast%3DAndersson%26aufirst%3DM.%26aulast%3DEricsson%26aufirst%3DU.%26aulast%3DForsblom%26aufirst%3DR.%26aulast%3DGinman%26aufirst%3DT.%26aulast%3DLindstr%25C3%25B6m%26aufirst%3DJ.%26aulast%3DSilvander%26aufirst%3DC.%26aulast%3DTr%25C3%25A9saugues%26aufirst%3DL.%26aulast%3DGiese%26aufirst%3DA.%26aulast%3DBunse%26aufirst%3DS.%26aulast%3DNeuhaus%26aufirst%3DR.%26aulast%3DWeiske%26aufirst%3DJ.%26aulast%3DQuanz%26aufirst%3DM.%26aulast%3DGlasauer%26aufirst%3DA.%26aulast%3DNowak-Reppel%26aufirst%3DK.%26aulast%3DBader%26aufirst%3DB.%26aulast%3DIrlbacher%26aufirst%3DH.%26aulast%3DMeyer%26aufirst%3DH.%26aulast%3DQueisser%26aufirst%3DN.%26aulast%3DBauser%26aufirst%3DM.%26aulast%3DHaegebarth%26aufirst%3DA.%26aulast%3DGorj%25C3%25A1n%25C3%25A1cz%26aufirst%3DM.%26atitle%3DNovel%2520Class%2520of%2520Potent%2520and%2520Cellularly%2520Active%2520Inhibitors%2520Devalidates%2520MTH1%2520as%2520Broad-Spectrum%2520Cancer%2520Target%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26issue%3D8%26spage%3D1986%26epage%3D1992%26doi%3D10.1021%2Facschembio.7b00370" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liou, M.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storz, P.</span></span> <i>Free Radical Res.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">479</span>, <span class="refDoi"> DOI: 10.3109/10715761003667554</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.3109%2F10715761003667554" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=20370557" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BC3cXkvVyhu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Liou%2C+M.-Y.%3B+Storz%2C+P.+Free+Radical+Res.+2010%2C+44%2C+479%2C+10.3109%2F10715761003667554"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Reactive oxygen species in cancer</span></div><div class="casAuthors">Liou, Geou-Yarh; Storz, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Free Radical Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">479-496</span>CODEN:
                <span class="NLM_cas:coden">FRARER</span>;
        ISSN:<span class="NLM_cas:issn">1071-5762</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Elevated rates of reactive oxygen species (ROS) have been detected in almost all cancers, where they promote many aspects of tumor development and progression.  However, tumor cells also express increased levels of antioxidant proteins to detoxify from ROS, suggesting that a delicate balance of intracellular ROS levels is required for cancer cell function.  Further, the radical generated, the location of its generation, as well as the local concn. is important for the cellular functions of ROS in cancer.  A challenge for novel therapeutic strategies will be the fine tuning of intracellular ROS signalling to effectively deprive cells from ROS-induced tumor promoting events, towards tipping the balance to ROS-induced apoptotic signalling.  Alternatively, therapeutic antioxidants may prevent early events in tumor development, where ROS are important.  However, to effectively target cancer cells specific ROS-sensing signalling pathways that mediate the diverse stress-regulated cellular functions need to be identified.  This review discusses the generation of ROS within tumor cells, their detoxification, their cellular effects, as well as the major signalling cascades they utilize, but also provides an outlook on their modulation in therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwtVSe56knA7Vg90H21EOLACvtfcHk0lgOzuNdNJhyFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXkvVyhu78%253D&md5=74d17b013387120ba165302bd7c282df</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.3109%2F10715761003667554&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F10715761003667554%26sid%3Dliteratum%253Aachs%26aulast%3DLiou%26aufirst%3DM.-Y.%26aulast%3DStorz%26aufirst%3DP.%26jtitle%3DFree%2520Radical%2520Res.%26date%3D2010%26volume%3D44%26spage%3D479%26doi%3D10.3109%2F10715761003667554" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klaunig, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamendulis, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hocevar, B. A.</span></span> <span> </span><span class="NLM_article-title">Oxidative Stress and Oxidative Damage in Carcinogenesis</span>. <i>Toxicol. Pathol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>38</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">96</span>– <span class="NLM_lpage">109</span>, <span class="refDoi"> DOI: 10.1177/0192623309356453</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1177%2F0192623309356453" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=20019356" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjt1ajtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=96-109&issue=1&author=J.+E.+Klaunigauthor=L.+M.+Kamendulisauthor=B.+A.+Hocevar&title=Oxidative+Stress+and+Oxidative+Damage+in+Carcinogenesis&doi=10.1177%2F0192623309356453"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Oxidative stress and oxidative damage in carcinogenesis</span></div><div class="casAuthors">Klaunig, James E.; Kamendulis, Lisa M.; Hocevar, Barbara A.</div><div class="citationInfo"><span class="NLM_cas:title">Toxicologic Pathology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">96-109</span>CODEN:
                <span class="NLM_cas:coden">TOPADD</span>;
        ISSN:<span class="NLM_cas:issn">0192-6233</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">A review.  Carcinogenesis is a multistep process involving mutation and the subsequent selective clonal expansion of the mutated cell.  Chem. and phys. agents including those that induce reactive oxygen species can induce and/or modulate this multistep process.  Several modes of action by which carcinogens induce cancer have been identified, including through prodn. of reactive oxygen species (ROS).  Oxidative damage to cellular macromols. can arise through overprodn. of ROS and faulty antioxidant and/or DNA repair mechanisms.  In addn., ROS can stimulate signal transduction pathways and lead to activation of key transcription factors such as Nrf2 and NF-κB.  The resultant altered gene expression patterns evoked by ROS contribute to the carcinogenesis process.  Recent evidence demonstrates an assocn. between a no. of single nucleotide polymorphisms (SNPs) in oxidative DNA repair genes and antioxidant genes with human cancer susceptibility.  These aspects of ROS biol. will be discussed in the context of their relationship to carcinogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGow0gFgYl9yd7Vg90H21EOLACvtfcHk0litHc0ZjEfIiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjt1ajtb4%253D&md5=09e44cb1a6b772c7ce38987837d9ae8b</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1177%2F0192623309356453&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F0192623309356453%26sid%3Dliteratum%253Aachs%26aulast%3DKlaunig%26aufirst%3DJ.%2BE.%26aulast%3DKamendulis%26aufirst%3DL.%2BM.%26aulast%3DHocevar%26aufirst%3DB.%2BA.%26atitle%3DOxidative%2520Stress%2520and%2520Oxidative%2520Damage%2520in%2520Carcinogenesis%26jtitle%3DToxicol.%2520Pathol.%26date%3D2010%26volume%3D38%26issue%3D1%26spage%3D96%26epage%3D109%26doi%3D10.1177%2F0192623309356453" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gorrini, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, I. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mak, T. W.</span></span> <span> </span><span class="NLM_article-title">Modulation of Oxidative Stress as an Anticancer Strategy</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>12</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">931</span>– <span class="NLM_lpage">947</span>, <span class="refDoi"> DOI: 10.1038/nrd4002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1038%2Fnrd4002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=24287781" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVGqs7%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=931-947&issue=12&author=C.+Gorriniauthor=I.+S.+Harrisauthor=T.+W.+Mak&title=Modulation+of+Oxidative+Stress+as+an+Anticancer+Strategy&doi=10.1038%2Fnrd4002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Modulation of oxidative stress as an anticancer strategy</span></div><div class="casAuthors">Gorrini, Chiara; Harris, Isaac S.; Mak, Tak W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">931-947</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The regulation of oxidative stress is an important factor in both tumor development and responses to anticancer therapies.  Many signalling pathways that are linked to tumorigenesis can also regulate the metab. of reactive oxygen species (ROS) through direct or indirect mechanisms.  High ROS levels are generally detrimental to cells, and the redox status of cancer cells usually differs from that of normal cells.  Because of metabolic and signalling aberrations, cancer cells exhibit elevated ROS levels.  The observation that this is balanced by an increased antioxidant capacity suggests that high ROS levels may constitute a barrier to tumorigenesis.  However, ROS can also promote tumor formation by inducing DNA mutations and pro-oncogenic signalling pathways.  These contradictory effects have important implications for potential anticancer strategies that aim to modulate levels of ROS.  In this Review, we address the controversial role of ROS in tumor development and in responses to anticancer therapies, and elaborate on the idea that targeting the antioxidant capacity of tumor cells can have a pos. therapeutic impact.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVKYMoILuOfbVg90H21EOLACvtfcHk0litHc0ZjEfIiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVGqs7%252FI&md5=7f135a1b32b1134a88cfa28b35c53de7</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnrd4002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4002%26sid%3Dliteratum%253Aachs%26aulast%3DGorrini%26aufirst%3DC.%26aulast%3DHarris%26aufirst%3DI.%2BS.%26aulast%3DMak%26aufirst%3DT.%2BW.%26atitle%3DModulation%2520of%2520Oxidative%2520Stress%2520as%2520an%2520Anticancer%2520Strategy%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2013%26volume%3D12%26issue%3D12%26spage%3D931%26epage%3D947%26doi%3D10.1038%2Fnrd4002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Haghdoost, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sjölander, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czene, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harms-Ringdahl, M.</span></span> <span> </span><span class="NLM_article-title">The Nucleotide Pool Is a Significant Target for Oxidative Stress</span>. <i>Free Radical Biol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>41</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">620</span>– <span class="NLM_lpage">626</span>, <span class="refDoi"> DOI: 10.1016/j.freeradbiomed.2006.05.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1016%2Fj.freeradbiomed.2006.05.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=16863995" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BD28XntlCqt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2006&pages=620-626&issue=4&author=S.+Haghdoostauthor=L.+Sj%C3%B6landerauthor=S.+Czeneauthor=M.+Harms-Ringdahl&title=The+Nucleotide+Pool+Is+a+Significant+Target+for+Oxidative+Stress&doi=10.1016%2Fj.freeradbiomed.2006.05.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The nucleotide pool is a significant target for oxidative stress</span></div><div class="casAuthors">Haghdoost, Siamak; Sjoelander, Lena; Czene, Stefan; Harms-Ringdahl, Mats</div><div class="citationInfo"><span class="NLM_cas:title">Free Radical Biology & Medicine</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">620-626</span>CODEN:
                <span class="NLM_cas:coden">FRBMEH</span>;
        ISSN:<span class="NLM_cas:issn">0891-5849</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Oxidative stress is considered to be one of the most important phenomena involved in the process of aging and age-related diseases. 8-Oxo-7,8-dihydro-2'-deoxyguanosine (8-oxo-dG) has been frequently used as a marker for oxidative stress.  However, the origin of extracellular 8-oxo-dG is not well understood.  The aim of this work was to investigate the nucleotide pool and the role of the human mutT homolog protein (hMTH1) in the appearance of extracellular 8-oxo-dG in a cellular model system.  For this purpose we used primary human fibroblast cells, which were transfected by siRNAs homologous to hMTH1.  Extracellular 8-oxo-dG in cell culture media after exposure of the cells to ionizing radiation was measured as ELISA reactivity.  Our results demonstrate the profound effect of both hMTH1 expression and nucleotide pool size on the cellular excretion of 8-oxo-dG, suggesting that the nucleotide pool is a significant target for the formation of extracellular 8-oxo-dG.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwAhG08kKjqLVg90H21EOLACvtfcHk0li14loocw3wHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XntlCqt7Y%253D&md5=a01efdb97485a13cebe95641d8efe287</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.freeradbiomed.2006.05.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.freeradbiomed.2006.05.003%26sid%3Dliteratum%253Aachs%26aulast%3DHaghdoost%26aufirst%3DS.%26aulast%3DSj%25C3%25B6lander%26aufirst%3DL.%26aulast%3DCzene%26aufirst%3DS.%26aulast%3DHarms-Ringdahl%26aufirst%3DM.%26atitle%3DThe%2520Nucleotide%2520Pool%2520Is%2520a%2520Significant%2520Target%2520for%2520Oxidative%2520Stress%26jtitle%3DFree%2520Radical%2520Biol.%2520Med.%26date%3D2006%26volume%3D41%26issue%3D4%26spage%3D620%26epage%3D626%26doi%3D10.1016%2Fj.freeradbiomed.2006.05.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pavlov, Y. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minnick, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Izuta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kunkel, T. A.</span></span> <span> </span><span class="NLM_article-title">DNA Replication Fidelity with 8-Oxodeoxyguanosine Triphosphate</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>33</i></span> (<span class="NLM_issue">15</span>),  <span class="NLM_fpage">4695</span>– <span class="NLM_lpage">4701</span>, <span class="refDoi"> DOI: 10.1021/bi00181a029</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi00181a029" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=1994&pages=4695-4701&issue=15&author=Y.+I.+Pavlovauthor=D.+T.+Minnickauthor=S.+Izutaauthor=T.+A.+Kunkel&title=DNA+Replication+Fidelity+with+8-Oxodeoxyguanosine+Triphosphate&doi=10.1021%2Fbi00181a029"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Fbi00181a029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00181a029%26sid%3Dliteratum%253Aachs%26aulast%3DPavlov%26aufirst%3DY.%2BI.%26aulast%3DMinnick%26aufirst%3DD.%2BT.%26aulast%3DIzuta%26aufirst%3DS.%26aulast%3DKunkel%26aufirst%3DT.%2BA.%26atitle%3DDNA%2520Replication%2520Fidelity%2520with%25208-Oxodeoxyguanosine%2520Triphosphate%26jtitle%3DBiochemistry%26date%3D1994%26volume%3D33%26issue%3D15%26spage%3D4695%26epage%3D4701%26doi%3D10.1021%2Fbi00181a029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bessman, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frick, D. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Handley, S. F.</span></span> <span> </span><span class="NLM_article-title">The MutT Proteins or ″Nudix″ Hydrolases, a Family of Versatile, Widely Distributed, ″housecleaning″ Enzymes</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>271</i></span> (<span class="NLM_issue">41</span>),  <span class="NLM_fpage">25059</span>– <span class="NLM_lpage">25062</span>, <span class="refDoi"> DOI: 10.1074/jbc.271.41.25059</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1074%2Fjbc.271.41.25059" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=271&publication_year=1996&pages=25059-25062&issue=41&author=M.+J.+Bessmanauthor=D.+N.+Frickauthor=S.+F.+O%E2%80%99Handley&title=The+MutT+Proteins+or+%E2%80%B3Nudix%E2%80%B3+Hydrolases%2C+a+Family+of+Versatile%2C+Widely+Distributed%2C+%E2%80%B3housecleaning%E2%80%B3+Enzymes&doi=10.1074%2Fjbc.271.41.25059"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1074%2Fjbc.271.41.25059&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.271.41.25059%26sid%3Dliteratum%253Aachs%26aulast%3DBessman%26aufirst%3DM.%2BJ.%26aulast%3DFrick%26aufirst%3DD.%2BN.%26aulast%3DO%25E2%2580%2599Handley%26aufirst%3DS.%2BF.%26atitle%3DThe%2520MutT%2520Proteins%2520or%2520%25E2%2580%25B3Nudix%25E2%2580%25B3%2520Hydrolases%252C%2520a%2520Family%2520of%2520Versatile%252C%2520Widely%2520Distributed%252C%2520%25E2%2580%25B3housecleaning%25E2%2580%25B3%2520Enzymes%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1996%26volume%3D271%26issue%3D41%26spage%3D25059%26epage%3D25062%26doi%3D10.1074%2Fjbc.271.41.25059" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gad, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koolmeister, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jemth, A.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eshtad, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacques, S. a</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ström, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svensson, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lundbäck, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Einarsdottir, B. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saleh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Göktürk, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baranczewski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svensson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berntsson, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafsson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strömberg, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanjiv, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacques-Cordonnier, M.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desroses, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustavsson, A.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olofsson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johansson, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Homan, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loseva, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bräutigam, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johansson, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Höglund, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hagenkort, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pham, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altun, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaugaz, F. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vikingsson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evers, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henriksson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vallin, K. S. a</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallner, O. a</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammarström, L. G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiita, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almlöf, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalderén, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Axelsson, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Djureinovic, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puigvert, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Häggblad, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeppsson, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martens, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lundin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lundgren, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Granelli, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Artursson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nilsson, J. a</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stenmark, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scobie, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berglund, U. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helleday, T.</span></span> <span> </span><span class="NLM_article-title">MTH1 Inhibition Eradicates Cancer by Preventing Sanitation of the dNTP Pool</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>508</i></span> (<span class="NLM_issue">7495</span>),  <span class="NLM_fpage">215</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1038/nature13181</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1038%2Fnature13181" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=24695224" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtlWrsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=508&publication_year=2014&pages=215-221&issue=7495&author=H.+Gadauthor=T.+Koolmeisterauthor=A.-S.+Jemthauthor=S.+Eshtadauthor=S.+a+Jacquesauthor=C.+E.+Str%C3%B6mauthor=L.+M.+Svenssonauthor=N.+Schultzauthor=T.+Lundb%C3%A4ckauthor=B.+O.+Einarsdottirauthor=A.+Salehauthor=C.+G%C3%B6kt%C3%BCrkauthor=P.+Baranczewskiauthor=R.+Svenssonauthor=R.+P.+Berntssonauthor=R.+Gustafssonauthor=K.+Str%C3%B6mbergauthor=K.+Sanjivauthor=M.-C.+Jacques-Cordonnierauthor=M.+Desrosesauthor=A.-L.+Gustavssonauthor=R.+Olofssonauthor=F.+Johanssonauthor=E.+J.+Homanauthor=O.+Losevaauthor=L.+Br%C3%A4utigamauthor=L.+Johanssonauthor=A.+H%C3%B6glundauthor=A.+Hagenkortauthor=T.+Phamauthor=M.+Altunauthor=F.+Z.+Gaugazauthor=S.+Vikingssonauthor=B.+Eversauthor=M.+Henrikssonauthor=K.+S.+a+Vallinauthor=O.+a+Wallnerauthor=L.+G.+J.+Hammarstr%C3%B6mauthor=E.+Wiitaauthor=I.+Alml%C3%B6fauthor=C.+Kalder%C3%A9nauthor=H.+Axelssonauthor=T.+Djureinovicauthor=J.+C.+Puigvertauthor=M.+H%C3%A4ggbladauthor=F.+Jeppssonauthor=U.+Martensauthor=C.+Lundinauthor=B.+Lundgrenauthor=I.+Granelliauthor=A.+J.+Jensenauthor=P.+Arturssonauthor=J.+a+Nilssonauthor=P.+Stenmarkauthor=M.+Scobieauthor=U.+W.+Berglundauthor=T.+Helleday&title=MTH1+Inhibition+Eradicates+Cancer+by+Preventing+Sanitation+of+the+dNTP+Pool&doi=10.1038%2Fnature13181"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">MTH1 inhibition eradicates cancer by preventing sanitation of the dNTP pool</span></div><div class="casAuthors">Gad, Helge; Koolmeister, Tobias; Jemth, Ann-Sofie; Eshtad, Saeed; Jacques, Sylvain A.; Stroem, Cecilia E.; Svensson, Linda M.; Schultz, Niklas; Lundbaeck, Thomas; Einarsdottir, Berglind Osk; Saleh, Aljona; Gokturk, Camilla; Baranczewski, Pawel; Svensson, Richard; Berntsson, Ronnie P.-A.; Gustafsson, Robert; Stroemberg, Kia; Sanjiv, Kumar; Jacques-Cordonnier, Marie-Caroline; Desroses, Matthieu; Gustavsson, Anna-Lena; Olofsson, Roger; Johansson, Fredrik; Homan, Evert J.; Loseva, Olga; Braeutigam, Lars; Johansson, Lars; Hoeglund, Andreas; Hagenkort, Anna; Pham, Therese; Altun, Mikael; Gaugaz, Fabienne Z.; Vikingsson, Svante; Evers, Bastiaan; Henriksson, Martin; Vallin, Karl S. A.; Wallner, Olov A.; Hammarstroem, Lars G. J.; Wiita, Elisee; Almloef, Ingrid; Kalderen, Christina; Axelsson, Hanna; Djureinovic, Tatjana; Puigvert, Jordi Carreras; Haeggblad, Maria; Jeppsson, Fredrik; Martens, Ulf; Lundin, Cecilia; Lundgren, Bo; Granelli, Ingrid; Jensen, Annika Jenmalm; Artursson, Per; Nilsson, Jonas A.; Stenmark, Pal; Scobie, Martin; Berglund, Ulrika Warpman; Helleday, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">508</span>
        (<span class="NLM_cas:issue">7495</span>),
    <span class="NLM_cas:pages">215-221</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cancers have dysfunctional redox regulation resulting in reactive oxygen species prodn., damaging both DNA and free dNTPs.  The MTH1 protein sanitizes oxidized dNTP pools to prevent incorporation of damaged bases during DNA replication.  Although MTH1 is non-essential in normal cells, the authors show that cancer cells require MTH1 activity to avoid incorporation of oxidized dNTPs, resulting in DNA damage and cell death.  The authors validate MTH1 as an anticancer target in vivo and describe small mols. TH287 (I) and TH588 (II) as first-in-class nudix hydrolase family inhibitors that potently and selectively engage and inhibit the MTH1 protein in cells.  Protein co-crystal structures demonstrate that the inhibitors bind in the active site of MTH1.  The inhibitors cause incorporation of oxidized dNTPs in cancer cells, leading to DNA damage, cytotoxicity and therapeutic responses in patient-derived mouse xenografts.  This study exemplifies the non-oncogene addiction concept for anticancer treatment and validates MTH1 as being cancer phenotypic lethal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-Xn3KhC40XbVg90H21EOLACvtfcHk0lg4dLefpcUHew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtlWrsLc%253D&md5=49afb4522874f4f5e216ae38d0ff7cd2</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnature13181&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13181%26sid%3Dliteratum%253Aachs%26aulast%3DGad%26aufirst%3DH.%26aulast%3DKoolmeister%26aufirst%3DT.%26aulast%3DJemth%26aufirst%3DA.-S.%26aulast%3DEshtad%26aufirst%3DS.%26aulast%3DJacques%26aufirst%3DS.%2Ba%26aulast%3DStr%25C3%25B6m%26aufirst%3DC.%2BE.%26aulast%3DSvensson%26aufirst%3DL.%2BM.%26aulast%3DSchultz%26aufirst%3DN.%26aulast%3DLundb%25C3%25A4ck%26aufirst%3DT.%26aulast%3DEinarsdottir%26aufirst%3DB.%2BO.%26aulast%3DSaleh%26aufirst%3DA.%26aulast%3DG%25C3%25B6kt%25C3%25BCrk%26aufirst%3DC.%26aulast%3DBaranczewski%26aufirst%3DP.%26aulast%3DSvensson%26aufirst%3DR.%26aulast%3DBerntsson%26aufirst%3DR.%2BP.%26aulast%3DGustafsson%26aufirst%3DR.%26aulast%3DStr%25C3%25B6mberg%26aufirst%3DK.%26aulast%3DSanjiv%26aufirst%3DK.%26aulast%3DJacques-Cordonnier%26aufirst%3DM.-C.%26aulast%3DDesroses%26aufirst%3DM.%26aulast%3DGustavsson%26aufirst%3DA.-L.%26aulast%3DOlofsson%26aufirst%3DR.%26aulast%3DJohansson%26aufirst%3DF.%26aulast%3DHoman%26aufirst%3DE.%2BJ.%26aulast%3DLoseva%26aufirst%3DO.%26aulast%3DBr%25C3%25A4utigam%26aufirst%3DL.%26aulast%3DJohansson%26aufirst%3DL.%26aulast%3DH%25C3%25B6glund%26aufirst%3DA.%26aulast%3DHagenkort%26aufirst%3DA.%26aulast%3DPham%26aufirst%3DT.%26aulast%3DAltun%26aufirst%3DM.%26aulast%3DGaugaz%26aufirst%3DF.%2BZ.%26aulast%3DVikingsson%26aufirst%3DS.%26aulast%3DEvers%26aufirst%3DB.%26aulast%3DHenriksson%26aufirst%3DM.%26aulast%3DVallin%26aufirst%3DK.%2BS.%2Ba%26aulast%3DWallner%26aufirst%3DO.%2Ba%26aulast%3DHammarstr%25C3%25B6m%26aufirst%3DL.%2BG.%2BJ.%26aulast%3DWiita%26aufirst%3DE.%26aulast%3DAlml%25C3%25B6f%26aufirst%3DI.%26aulast%3DKalder%25C3%25A9n%26aufirst%3DC.%26aulast%3DAxelsson%26aufirst%3DH.%26aulast%3DDjureinovic%26aufirst%3DT.%26aulast%3DPuigvert%26aufirst%3DJ.%2BC.%26aulast%3DH%25C3%25A4ggblad%26aufirst%3DM.%26aulast%3DJeppsson%26aufirst%3DF.%26aulast%3DMartens%26aufirst%3DU.%26aulast%3DLundin%26aufirst%3DC.%26aulast%3DLundgren%26aufirst%3DB.%26aulast%3DGranelli%26aufirst%3DI.%26aulast%3DJensen%26aufirst%3DA.%2BJ.%26aulast%3DArtursson%26aufirst%3DP.%26aulast%3DNilsson%26aufirst%3DJ.%2Ba%26aulast%3DStenmark%26aufirst%3DP.%26aulast%3DScobie%26aufirst%3DM.%26aulast%3DBerglund%26aufirst%3DU.%2BW.%26aulast%3DHelleday%26aufirst%3DT.%26atitle%3DMTH1%2520Inhibition%2520Eradicates%2520Cancer%2520by%2520Preventing%2520Sanitation%2520of%2520the%2520dNTP%2520Pool%26jtitle%3DNature%26date%3D2014%26volume%3D508%26issue%3D7495%26spage%3D215%26epage%3D221%26doi%3D10.1038%2Fnature13181" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nakabeppu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kajitani, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaguchi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuchimoto, D.</span></span> <span> </span><span class="NLM_article-title">MTH1, an Oxidized Purine Nucleoside Triphosphatase, Prevents the Cytotoxicity and Neurotoxicity of Oxidized Purine Nucleotides</span>. <i>DNA Repair</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>5</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">761</span>– <span class="NLM_lpage">772</span>, <span class="refDoi"> DOI: 10.1016/j.dnarep.2006.03.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1016%2Fj.dnarep.2006.03.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=16621731" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BD28Xmt1OqsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2006&pages=761-772&issue=7&author=Y.+Nakabeppuauthor=K.+Kajitaniauthor=K.+Sakamotoauthor=H.+Yamaguchiauthor=D.+Tsuchimoto&title=MTH1%2C+an+Oxidized+Purine+Nucleoside+Triphosphatase%2C+Prevents+the+Cytotoxicity+and+Neurotoxicity+of+Oxidized+Purine+Nucleotides&doi=10.1016%2Fj.dnarep.2006.03.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">MTH1, an oxidized purine nucleoside triphosphatase, prevents the cytotoxicity and neurotoxicity of oxidized purine nucleotides</span></div><div class="casAuthors">Nakabeppu, Yusaku; Kajitani, Kosuke; Sakamoto, Katsumi; Yamaguchi, Hiroo; Tsuchimoto, Daisuke</div><div class="citationInfo"><span class="NLM_cas:title">DNA Repair</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">761-772</span>CODEN:
                <span class="NLM_cas:coden">DRNEAR</span>;
        ISSN:<span class="NLM_cas:issn">1568-7864</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  In human and rodent cells, MTH1, an oxidized purine nucleoside triphosphatase, efficiently hydrolyzes oxidized dGTP, GTP, dATP, and ATP such as 2'-deoxy-8-oxoguanosine triphosphate (8-oxo-dGTP) and 2'-deoxy-2-hydroxyadenosine triphosphate (2-OH-dATP) in nucleotide pools, thus avoiding their incorporation into DNA or RNA.  MTH1 is expressed in postmitotic neurons as well as in proliferative tissues, and it is localized both in the mitochondria and nucleus, thus suggesting that MTH1 plays an important role in the prevention of the mutagenicity and cytotoxicity of such oxidized purines as 8-oxoG which are known to accumulate in the cellular genome.  Our recent studies with MTH1-deficient mice or cells revealed that MTH1 efficiently minimizes accumulation of 8-oxoG in both nuclear and mitochondrial DNA in the mouse brain as well as in cultured cells, thus contributing to the protection of the brain from oxidative stress.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoe-LGRN37xcrVg90H21EOLACvtfcHk0liQ5qVxz1LawA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xmt1OqsLc%253D&md5=9a38c80c0ed62c8ada5bf1ac0dab9d9e</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.dnarep.2006.03.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.dnarep.2006.03.003%26sid%3Dliteratum%253Aachs%26aulast%3DNakabeppu%26aufirst%3DY.%26aulast%3DKajitani%26aufirst%3DK.%26aulast%3DSakamoto%26aufirst%3DK.%26aulast%3DYamaguchi%26aufirst%3DH.%26aulast%3DTsuchimoto%26aufirst%3DD.%26atitle%3DMTH1%252C%2520an%2520Oxidized%2520Purine%2520Nucleoside%2520Triphosphatase%252C%2520Prevents%2520the%2520Cytotoxicity%2520and%2520Neurotoxicity%2520of%2520Oxidized%2520Purine%2520Nucleotides%26jtitle%3DDNA%2520Repair%26date%3D2006%26volume%3D5%26issue%3D7%26spage%3D761%26epage%3D772%26doi%3D10.1016%2Fj.dnarep.2006.03.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rai, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burton, D. G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giribaldi, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Onder, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinberg, R. A.</span></span> <span> </span><span class="NLM_article-title">Enhanced Elimination of Oxidized Guanine Nucleotides Inhibits Oncogenic RAS-Induced DNA Damage and Premature Senescence</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>30</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">1489</span>– <span class="NLM_lpage">1496</span>, <span class="refDoi"> DOI: 10.1038/onc.2010.520</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1038%2Fonc.2010.520" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2011&pages=1489-1496&issue=12&author=P.+Raiauthor=J.+J.+Youngauthor=D.+G.+A.+Burtonauthor=M.+G.+Giribaldiauthor=T.+T.+Onderauthor=R.+A.+Weinberg&title=Enhanced+Elimination+of+Oxidized+Guanine+Nucleotides+Inhibits+Oncogenic+RAS-Induced+DNA+Damage+and+Premature+Senescence&doi=10.1038%2Fonc.2010.520"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fonc.2010.520&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2010.520%26sid%3Dliteratum%253Aachs%26aulast%3DRai%26aufirst%3DP.%26aulast%3DYoung%26aufirst%3DJ.%2BJ.%26aulast%3DBurton%26aufirst%3DD.%2BG.%2BA.%26aulast%3DGiribaldi%26aufirst%3DM.%2BG.%26aulast%3DOnder%26aufirst%3DT.%2BT.%26aulast%3DWeinberg%26aufirst%3DR.%2BA.%26atitle%3DEnhanced%2520Elimination%2520of%2520Oxidized%2520Guanine%2520Nucleotides%2520Inhibits%2520Oncogenic%2520RAS-Induced%2520DNA%2520Damage%2520and%2520Premature%2520Senescence%26jtitle%3DOncogene%26date%3D2011%26volume%3D30%26issue%3D12%26spage%3D1489%26epage%3D1496%26doi%3D10.1038%2Fonc.2010.520" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Patel, a</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burton, D. G. a</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halvorsen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balkan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reiner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez-Stable, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, a</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munoz, a</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giribaldi, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robbins, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rai, P.</span></span> <span> </span><span class="NLM_article-title">MutT Homolog 1 (MTH1) Maintains Multiple KRAS-Driven pro-Malignant Pathways</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">2586</span>, <span class="refDoi"> DOI: 10.1038/onc.2014.195</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1038%2Fonc.2014.195" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=25023700" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFGqtr%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2015&pages=2586&author=a+Patelauthor=D.+G.+a+Burtonauthor=K.+Halvorsenauthor=W.+Balkanauthor=T.+Reinerauthor=C.+Perez-Stableauthor=a+Cohenauthor=a+Munozauthor=M.+G.+Giribaldiauthor=S.+Singhauthor=D.+J.+Robbinsauthor=D.+M.+Nguyenauthor=P.+Rai&title=MutT+Homolog+1+%28MTH1%29+Maintains+Multiple+KRAS-Driven+pro-Malignant+Pathways&doi=10.1038%2Fonc.2014.195"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">MutT Homolog 1 (MTH1) maintains multiple KRAS-driven pro-malignant pathways</span></div><div class="casAuthors">Patel, A.; Burton, D. G. A.; Halvorsen, K.; Balkan, W.; Reiner, T.; Perez-Stable, C.; Cohen, A.; Munoz, A.; Giribaldi, M. G.; Singh, S.; Robbins, D. J.; Nguyen, D. M.; Rai, P.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">2586-2596</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Oncogenic RAS promotes prodn. of reactive oxygen species (ROS), which mediate pro-malignant signaling but can also trigger DNA damage-induced tumor suppression.  Thus RAS-driven tumor cells require redox-protective mechanisms to mitigate the damaging aspects of ROS.  Here, we show that MutT Homolog 1 (MTH1), the mammalian 8-oxodGTPase that sanitizes oxidative damage in the nucleotide pool, is important for maintaining several KRAS-driven pro-malignant traits in a nonsmall cell lung carcinoma (NSCLC) model.  MTH1 suppression in KRAS-mutant NSCLC cells impairs proliferation and xenograft tumor formation.  Furthermore, MTH1 levels modulate KRAS-induced transformation of immortalized lung epithelial cells.  MTH1 expression is upregulated by oncogenic KRAS and correlates pos. with high KRAS levels in NSCLC human tumors.  At a mol. level, in p53-competent KRAS-mutant cells, MTH1 loss provokes DNA damage and induction of oncogene-induced senescence.  In p53-nonfunctional KRAS-mutant cells, MTH1 suppression does not produce DNA damage but reduces proliferation and leads to an adaptive decrease in KRAS expression levels.  Thus, MTH1 not only enables evasion of oxidative DNA damage and its consequences, but can also function as a mol. rheostat for maintaining oncogene expression at optimal levels.  Accordingly, our results indicate MTH1 is a novel and crit. component of oncogenic KRAS-assocd. malignancy and its inhibition is likely to yield significant tumor-suppressive outcomes in KRAS-driven tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQuLF6p5AL87Vg90H21EOLACvtfcHk0li-m_hsiI4WNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFGqtr%252FN&md5=bb949121fcc660bfe99dfccdea321c6f</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fonc.2014.195&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2014.195%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3Da%26aulast%3DBurton%26aufirst%3DD.%2BG.%2Ba%26aulast%3DHalvorsen%26aufirst%3DK.%26aulast%3DBalkan%26aufirst%3DW.%26aulast%3DReiner%26aufirst%3DT.%26aulast%3DPerez-Stable%26aufirst%3DC.%26aulast%3DCohen%26aufirst%3Da%26aulast%3DMunoz%26aufirst%3Da%26aulast%3DGiribaldi%26aufirst%3DM.%2BG.%26aulast%3DSingh%26aufirst%3DS.%26aulast%3DRobbins%26aufirst%3DD.%2BJ.%26aulast%3DNguyen%26aufirst%3DD.%2BM.%26aulast%3DRai%26aufirst%3DP.%26atitle%3DMutT%2520Homolog%25201%2520%2528MTH1%2529%2520Maintains%2520Multiple%2520KRAS-Driven%2520pro-Malignant%2520Pathways%26jtitle%3DOncogene%26date%3D2015%26volume%3D34%26spage%3D2586%26doi%3D10.1038%2Fonc.2014.195" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salah, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Radic, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gridling, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elkins, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stukalov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jemth, A.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Göktürk, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanjiv, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strömberg, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pham, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berglund, U. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colinge, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loizou, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helleday, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Superti-Furga, G.</span></span> <span> </span><span class="NLM_article-title">Stereospecific Targeting of MTH1 by (S)-Crizotinib as an Anticancer Strategy</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>508</i></span> (<span class="NLM_issue">7495</span>),  <span class="NLM_fpage">222</span>– <span class="NLM_lpage">227</span>, <span class="refDoi"> DOI: 10.1038/nature13194</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1038%2Fnature13194" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=24695225" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtlWrsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=508&publication_year=2014&pages=222-227&issue=7495&author=K.+V.+M+Huberauthor=E.+Salahauthor=B.+Radicauthor=M.+Gridlingauthor=J.+M.+Elkinsauthor=A.+Stukalovauthor=A.-S.+Jemthauthor=C.+G%C3%B6kt%C3%BCrkauthor=K.+Sanjivauthor=K.+Str%C3%B6mbergauthor=T.+Phamauthor=U.+W.+Berglundauthor=J.+Colingeauthor=K.+L.+Bennettauthor=J.+I.+Loizouauthor=T.+Helledayauthor=S.+Knappauthor=G.+Superti-Furga&title=Stereospecific+Targeting+of+MTH1+by+%28S%29-Crizotinib+as+an+Anticancer+Strategy&doi=10.1038%2Fnature13194"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Stereospecific targeting of MTH1 by (S)-crizotinib as an anticancer strategy</span></div><div class="casAuthors">Huber, Kilian V. M.; Salah, Eidarus; Radic, Branka; Gridling, Manuela; Elkins, Jonathan M.; Stukalov, Alexey; Jemth, Ann-Sofie; Gokturk, Camilla; Sanjiv, Kumar; Stroemberg, Kia; Pham, Therese; Berglund, Ulrika Warpman; Colinge, Jacques; Bennett, Keiryn L.; Loizou, Joanna I.; Helleday, Thomas; Knapp, Stefan; Superti-Furga, Giulio</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">508</span>
        (<span class="NLM_cas:issue">7495</span>),
    <span class="NLM_cas:pages">222-227</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Activated RAS GTPase signaling is a crit. driver of oncogenic transformation and malignant disease.  Cellular models of RAS-dependent cancers have been used to identify exptl. small mols., such as SCH51344, but their mol. mechanism of action remains generally unknown.  Here, using a chem. proteomic approach, the authors identify the target of SCH51344 as the human mutT homolog MTH1 (also known as NUDT1), a nucleotide pool sanitizing enzyme.  Loss-of-function of MTH1 impaired growth of KRAS tumor cells, whereas MTH1 overexpression mitigated sensitivity towards SCH51344.  Searching for more drug-like inhibitors, the authors identified the kinase inhibitor crizotinib as a nanomolar suppressor of MTH1 activity.  Surprisingly, the clin. used (R)-enantiomer of the drug was inactive, whereas the (S)-enantiomer selectively inhibited MTH1 catalytic activity.  Enzymic assays, chem. proteomic profiling, kinome-wide activity surveys and MTH1 co-crystal structures of both enantiomers provide a rationale for this remarkable stereospecificity.  Disruption of nucleotide pool homeostasis via MTH1 inhibition by (S)-crizotinib induced an increase in DNA single-strand breaks, activated DNA repair in human colon carcinoma cells, and effectively suppressed tumor growth in animal models.  The authors' results propose (S)-crizotinib as an attractive chem. entity for further pre-clin. evaluation, and small-mol. inhibitors of MTH1 in general as a promising novel class of anticancer agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrbUm7UP6kIrVg90H21EOLACvtfcHk0li-m_hsiI4WNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtlWrsb4%253D&md5=ca3df11a6782fd9140b6d8ba0c16e5ab</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnature13194&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13194%26sid%3Dliteratum%253Aachs%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM%26aulast%3DSalah%26aufirst%3DE.%26aulast%3DRadic%26aufirst%3DB.%26aulast%3DGridling%26aufirst%3DM.%26aulast%3DElkins%26aufirst%3DJ.%2BM.%26aulast%3DStukalov%26aufirst%3DA.%26aulast%3DJemth%26aufirst%3DA.-S.%26aulast%3DG%25C3%25B6kt%25C3%25BCrk%26aufirst%3DC.%26aulast%3DSanjiv%26aufirst%3DK.%26aulast%3DStr%25C3%25B6mberg%26aufirst%3DK.%26aulast%3DPham%26aufirst%3DT.%26aulast%3DBerglund%26aufirst%3DU.%2BW.%26aulast%3DColinge%26aufirst%3DJ.%26aulast%3DBennett%26aufirst%3DK.%2BL.%26aulast%3DLoizou%26aufirst%3DJ.%2BI.%26aulast%3DHelleday%26aufirst%3DT.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DSuperti-Furga%26aufirst%3DG.%26atitle%3DStereospecific%2520Targeting%2520of%2520MTH1%2520by%2520%2528S%2529-Crizotinib%2520as%2520an%2520Anticancer%2520Strategy%26jtitle%3DNature%26date%3D2014%26volume%3D508%26issue%3D7495%26spage%3D222%26epage%3D227%26doi%3D10.1038%2Fnature13194" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berman, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Battistuz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhat, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bluhm, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourne, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burkhardt, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilliland, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iype, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fagan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marvin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Padilla, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ravichandran, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thanki, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissig, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westbrook, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zardecki, C.</span></span> <span> </span><span class="NLM_article-title">The Protein Data Bank</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">899</span>– <span class="NLM_lpage">907</span>, <span class="refDoi"> DOI: 10.1107/S0907444902003451</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1107%2FS0907444902003451" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=12037327" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BD38XktVKhtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2002&pages=899-907&author=H.+M.+Bermanauthor=T.+Battistuzauthor=T.+N.+Bhatauthor=W.+F.+Bluhmauthor=P.+E.+Bourneauthor=K.+Burkhardtauthor=Z.+Fengauthor=G.+L.+Gillilandauthor=L.+Iypeauthor=S.+Jainauthor=P.+Faganauthor=J.+Marvinauthor=D.+Padillaauthor=V.+Ravichandranauthor=B.+Schneiderauthor=N.+Thankiauthor=H.+Weissigauthor=J.+D.+Westbrookauthor=C.+Zardecki&title=The+Protein+Data+Bank&doi=10.1107%2FS0907444902003451"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Data Bank</span></div><div class="casAuthors">Berman, Helen M.; Battistuz, Tammy; Bhat, T. N.; Bluhm, Wolfgang F.; Bourne, Philip E.; Burkhardt, Kyle; Feng, Zukang; Gilliland, Gary L.; Iype, Lisa; Jain, Shri; Fagan, Phoebe; Marvin, Jessica; Padilla, David; Ravichandran, Veerasamy; Schneider, Bohdan; Thanki, Narmada; Weissig, Helge; Westbrook, John D.; Zardecki, Christine</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">D58</span>
        (<span class="NLM_cas:issue">6, No. 1</span>),
    <span class="NLM_cas:pages">899-907</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Munksgaard</span>)
        </div><div class="casAbstract">The Protein Data Bank [PDB; Berman, Westbrook et al. (2000), Nucleic Acids Res. 28, 235-242; http://www.pdb.org/] is the single worldwide archive of primary structural data of biol. macromols.  Many secondary sources of information are derived from PDB data.  It is the starting point for studies in structural bioinformatics.  This article describes the goals of the PDB, the systems in place for data deposition and access, how to obtain further information and plans for the future development of the resource.  The reader should come away with an understanding of the scope of the PDB and what is provided by the resource.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsUFcMyys-PrVg90H21EOLACvtfcHk0liguhf5y42N_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XktVKhtLg%253D&md5=d7594ad00ec8e5bacc413d38bceaa360</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1107%2FS0907444902003451&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444902003451%26sid%3Dliteratum%253Aachs%26aulast%3DBerman%26aufirst%3DH.%2BM.%26aulast%3DBattistuz%26aufirst%3DT.%26aulast%3DBhat%26aufirst%3DT.%2BN.%26aulast%3DBluhm%26aufirst%3DW.%2BF.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26aulast%3DBurkhardt%26aufirst%3DK.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DGilliland%26aufirst%3DG.%2BL.%26aulast%3DIype%26aufirst%3DL.%26aulast%3DJain%26aufirst%3DS.%26aulast%3DFagan%26aufirst%3DP.%26aulast%3DMarvin%26aufirst%3DJ.%26aulast%3DPadilla%26aufirst%3DD.%26aulast%3DRavichandran%26aufirst%3DV.%26aulast%3DSchneider%26aufirst%3DB.%26aulast%3DThanki%26aufirst%3DN.%26aulast%3DWeissig%26aufirst%3DH.%26aulast%3DWestbrook%26aufirst%3DJ.%2BD.%26aulast%3DZardecki%26aufirst%3DC.%26atitle%3DThe%2520Protein%2520Data%2520Bank%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2002%26volume%3D58%26spage%3D899%26epage%3D907%26doi%3D10.1107%2FS0907444902003451" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tresaugues, L.</span>; <span class="NLM_string-name">Siponen, M. .</span>; <span class="NLM_string-name">Arrowsmith, C. .</span>; <span class="NLM_string-name">Berglund, H.</span>; <span class="NLM_string-name">Bountra, C.</span>; <span class="NLM_string-name">Collins, R.</span>; <span class="NLM_string-name">Edwards, A. .</span>; <span class="NLM_string-name">Ekblad, T.</span>; <span class="NLM_string-name">Flodin, S.</span>; <span class="NLM_string-name">Flores, A.</span>; <span class="NLM_string-name">Graslund, S.</span>; <span class="NLM_string-name">Hammarstrom, M.</span>; <span class="NLM_string-name">Johansson, I.</span>; <span class="NLM_string-name">Karlberg, T.</span>; <span class="NLM_string-name">Kol, S.</span>; <span class="NLM_string-name">Kotenyova, T.</span>; <span class="NLM_string-name">Kouznetsova, E.</span>; <span class="NLM_string-name">Moche, M.</span>; <span class="NLM_string-name">Nyman, T.</span>; <span class="NLM_string-name">Persson, C.</span>; <span class="NLM_string-name">Schuler, H.</span>; <span class="NLM_string-name">Schutz, P.</span>; <span class="NLM_string-name">Thorsell, A. .</span>; <span class="NLM_string-name">Van Der Berg, S.</span>; <span class="NLM_string-name">Wahlberg, E.</span>; <span class="NLM_string-name">Weigelt, J.</span>; <span class="NLM_string-name">Welin, M.</span>; <span class="NLM_string-name">Nordlund, P.</span></span> <span> </span><span class="NLM_article-title">Crystal Structure of Human 8-Oxo-dGTPase (MTH1)</span>. Deposited in PDB as 3Q93, <span class="NLM_year">2011</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&author=L.+Tresaugues&author=M.+.+Siponen&author=C.+.+Arrowsmith&author=H.+Berglund&author=C.+Bountra&author=R.+Collins&author=A.+.+Edwards&author=T.+Ekblad&author=S.+Flodin&author=A.+Flores&author=S.+Graslund&author=M.+Hammarstrom&author=I.+Johansson&author=T.+Karlberg&author=S.+Kol&author=T.+Kotenyova&author=E.+Kouznetsova&author=M.+Moche&author=T.+Nyman&author=C.+Persson&author=H.+Schuler&author=P.+Schutz&author=A.+.+Thorsell&author=S.+Van+Der+Berg&author=E.+Wahlberg&author=J.+Weigelt&author=M.+Welin&author=P.+Nordlund&title=Crystal+Structure+of+Human+8-Oxo-dGTPase+%28MTH1%29"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DTresaugues%26aufirst%3DL.%26atitle%3DCrystal%2520Structure%2520of%2520Human%25208-Oxo-dGTPase%2520%2528MTH1%2529%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Waz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirata, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koga-Ogawa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chirifu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arimori, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamada, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ikemizu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakabeppu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamagata, Y.</span></span> <span> </span><span class="NLM_article-title">Structural and Kinetic Studies of the Human Nudix Hydrolase MTH1 Reveal the Mechanism for Its Broad Substrate Specificity</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>292</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">2785</span>– <span class="NLM_lpage">2794</span>, <span class="refDoi"> DOI: 10.1074/jbc.M116.749713</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1074%2Fjbc.M116.749713" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=292&publication_year=2017&pages=2785-2794&issue=7&author=S.+Wazauthor=T.+Nakamuraauthor=K.+Hirataauthor=Y.+Koga-Ogawaauthor=M.+Chirifuauthor=T.+Arimoriauthor=T.+Tamadaauthor=S.+Ikemizuauthor=Y.+Nakabeppuauthor=Y.+Yamagata&title=Structural+and+Kinetic+Studies+of+the+Human+Nudix+Hydrolase+MTH1+Reveal+the+Mechanism+for+Its+Broad+Substrate+Specificity&doi=10.1074%2Fjbc.M116.749713"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M116.749713&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M116.749713%26sid%3Dliteratum%253Aachs%26aulast%3DWaz%26aufirst%3DS.%26aulast%3DNakamura%26aufirst%3DT.%26aulast%3DHirata%26aufirst%3DK.%26aulast%3DKoga-Ogawa%26aufirst%3DY.%26aulast%3DChirifu%26aufirst%3DM.%26aulast%3DArimori%26aufirst%3DT.%26aulast%3DTamada%26aufirst%3DT.%26aulast%3DIkemizu%26aufirst%3DS.%26aulast%3DNakabeppu%26aufirst%3DY.%26aulast%3DYamagata%26aufirst%3DY.%26atitle%3DStructural%2520and%2520Kinetic%2520Studies%2520of%2520the%2520Human%2520Nudix%2520Hydrolase%2520MTH1%2520Reveal%2520the%2520Mechanism%2520for%2520Its%2520Broad%2520Substrate%2520Specificity%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2017%26volume%3D292%26issue%3D7%26spage%3D2785%26epage%3D2794%26doi%3D10.1074%2Fjbc.M116.749713" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nissink, J. W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bista, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breed, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Embrey, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter-Holt, J. J.</span></span> <span> </span><span class="NLM_article-title">MTH1 Substrate Recognition—An Example of Specific Promiscuity</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">e0151154</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0151154</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1371%2Fjournal.pone.0151154" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=e0151154&issue=3&author=J.+W.+M.+Nissinkauthor=M.+Bistaauthor=J.+Breedauthor=N.+Carterauthor=K.+Embreyauthor=J.+Readauthor=J.+J.+Winter-Holt&title=MTH1+Substrate+Recognition%E2%80%94An+Example+of+Specific+Promiscuity&doi=10.1371%2Fjournal.pone.0151154"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0151154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0151154%26sid%3Dliteratum%253Aachs%26aulast%3DNissink%26aufirst%3DJ.%2BW.%2BM.%26aulast%3DBista%26aufirst%3DM.%26aulast%3DBreed%26aufirst%3DJ.%26aulast%3DCarter%26aufirst%3DN.%26aulast%3DEmbrey%26aufirst%3DK.%26aulast%3DRead%26aufirst%3DJ.%26aulast%3DWinter-Holt%26aufirst%3DJ.%2BJ.%26atitle%3DMTH1%2520Substrate%2520Recognition%25E2%2580%2594An%2520Example%2520of%2520Specific%2520Promiscuity%26jtitle%3DPLoS%2520One%26date%3D2016%26volume%3D11%26issue%3D3%26spage%3De0151154%26doi%3D10.1371%2Fjournal.pone.0151154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kettle, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alwan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bista, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breed, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eckersley, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fillery, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foote, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodwin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Käck, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nissink, J. W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wissler, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wylot, M.</span></span> <span> </span><span class="NLM_article-title">Potent and Selective Inhibitors of MTH1 Probe Its Role in Cancer Cell Survival</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">2346</span>– <span class="NLM_lpage">2361</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01760</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01760" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BC28XisFahsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=2346-2361&issue=6&author=J.+G.+Kettleauthor=H.+Alwanauthor=M.+Bistaauthor=J.+Breedauthor=N.+L.+Daviesauthor=K.+Eckersleyauthor=S.+Filleryauthor=K.+M.+Footeauthor=L.+Goodwinauthor=D.+R.+Jonesauthor=H.+K%C3%A4ckauthor=A.+Lauauthor=J.+W.+M.+Nissinkauthor=J.+Readauthor=J.+S.+Scottauthor=B.+Taylorauthor=G.+Walkerauthor=L.+Wisslerauthor=M.+Wylot&title=Potent+and+Selective+Inhibitors+of+MTH1+Probe+Its+Role+in+Cancer+Cell+Survival&doi=10.1021%2Facs.jmedchem.5b01760"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Potent and Selective Inhibitors of MTH1 Probe Its Role in Cancer Cell Survival</span></div><div class="casAuthors">Kettle, Jason G.; Alwan, Husam; Bista, Michal; Breed, Jason; Davies, Nichola L.; Eckersley, Kay; Fillery, Shaun; Foote, Kevin M.; Goodwin, Louise; Jones, David R.; Kack, Helena; Lau, Alan; Nissink, J. Willem M.; Read, Jon; Scott, James S.; Taylor, Ben; Walker, Graeme; Wissler, Lisa; Wylot, Marta</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2346-2361</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Recent literature has claimed that inhibition of the enzyme MTH1 can eradicate cancer.  MTH1 is one of the "housekeeping" enzymes that are responsible for hydrolyzing damaged nucleotides in cells and thus prevent them from being incorporated into DNA.  We have developed orthogonal and chem. distinct tool compds. to those published in the literature to allow us to test the hypothesis that inhibition of MTH1 has wide applicability in the treatment of cancer.  Here we present the work that led to the discovery of three structurally different series of MTH1 inhibitors with excellent potency, selectivity, and proven target engagement in cells.  None of these compds. elicited the reported cellular phenotype, and addnl. siRNA and CRISPR expts. further support these observations.  Critically, the difference between the responses of our highly selective inhibitors and published tool compds. suggests that the effect reported for the latter may be due to off-target cytotoxic effects.  As a result, we conclude that the role of MTH1 in carcinogenesis and utility of its inhibition is yet to be established.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWdM4dWZuW-bVg90H21EOLACvtfcHk0ljoqHPloiVxWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XisFahsrg%253D&md5=70ab2afcf27b69d03908a990daa825af</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01760%26sid%3Dliteratum%253Aachs%26aulast%3DKettle%26aufirst%3DJ.%2BG.%26aulast%3DAlwan%26aufirst%3DH.%26aulast%3DBista%26aufirst%3DM.%26aulast%3DBreed%26aufirst%3DJ.%26aulast%3DDavies%26aufirst%3DN.%2BL.%26aulast%3DEckersley%26aufirst%3DK.%26aulast%3DFillery%26aufirst%3DS.%26aulast%3DFoote%26aufirst%3DK.%2BM.%26aulast%3DGoodwin%26aufirst%3DL.%26aulast%3DJones%26aufirst%3DD.%2BR.%26aulast%3DK%25C3%25A4ck%26aufirst%3DH.%26aulast%3DLau%26aufirst%3DA.%26aulast%3DNissink%26aufirst%3DJ.%2BW.%2BM.%26aulast%3DRead%26aufirst%3DJ.%26aulast%3DScott%26aufirst%3DJ.%2BS.%26aulast%3DTaylor%26aufirst%3DB.%26aulast%3DWalker%26aufirst%3DG.%26aulast%3DWissler%26aufirst%3DL.%26aulast%3DWylot%26aufirst%3DM.%26atitle%3DPotent%2520and%2520Selective%2520Inhibitors%2520of%2520MTH1%2520Probe%2520Its%2520Role%2520in%2520Cancer%2520Cell%2520Survival%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26issue%3D6%26spage%3D2346%26epage%3D2361%26doi%3D10.1021%2Facs.jmedchem.5b01760" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Svensson, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jemth, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desroses, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loseva, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helleday, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Högbom, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stenmark, P.</span></span> <span> </span><span class="NLM_article-title">Crystal Structure of Human MTH1 and the 8-Oxo-dGMP Product Complex</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>585</i></span> (<span class="NLM_issue">16</span>),  <span class="NLM_fpage">2617</span>– <span class="NLM_lpage">2621</span>, <span class="refDoi"> DOI: 10.1016/j.febslet.2011.07.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1016%2Fj.febslet.2011.07.017" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=585&publication_year=2011&pages=2617-2621&issue=16&author=L.+M.+Svenssonauthor=A.+S.+Jemthauthor=M.+Desrosesauthor=O.+Losevaauthor=T.+Helledayauthor=M.+H%C3%B6gbomauthor=P.+Stenmark&title=Crystal+Structure+of+Human+MTH1+and+the+8-Oxo-dGMP+Product+Complex&doi=10.1016%2Fj.febslet.2011.07.017"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.febslet.2011.07.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.febslet.2011.07.017%26sid%3Dliteratum%253Aachs%26aulast%3DSvensson%26aufirst%3DL.%2BM.%26aulast%3DJemth%26aufirst%3DA.%2BS.%26aulast%3DDesroses%26aufirst%3DM.%26aulast%3DLoseva%26aufirst%3DO.%26aulast%3DHelleday%26aufirst%3DT.%26aulast%3DH%25C3%25B6gbom%26aufirst%3DM.%26aulast%3DStenmark%26aufirst%3DP.%26atitle%3DCrystal%2520Structure%2520of%2520Human%2520MTH1%2520and%2520the%25208-Oxo-dGMP%2520Product%2520Complex%26jtitle%3DFEBS%2520Lett.%26date%3D2011%26volume%3D585%26issue%3D16%26spage%3D2617%26epage%3D2621%26doi%3D10.1016%2Fj.febslet.2011.07.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Petrocchi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reyna, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamilton, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parker, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mseeh, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bardenhagen, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leonard, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cross, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardozo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Draetta, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marszalek, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toniatti, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, R. T.</span></span> <span> </span><span class="NLM_article-title">Identification of Potent and Selective MTH1 Inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>26</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">1503</span>– <span class="NLM_lpage">1507</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2016.02.026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1016%2Fj.bmcl.2016.02.026" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=1503-1507&issue=6&author=A.+Petrocchiauthor=E.+Leoauthor=N.+J.+Reynaauthor=M.+M.+Hamiltonauthor=X.+Shiauthor=C.+A.+Parkerauthor=F.+Mseehauthor=J.+P.+Bardenhagenauthor=P.+Leonardauthor=J.+B.+Crossauthor=S.+Huangauthor=Y.+Jiangauthor=M.+Cardozoauthor=G.+Draettaauthor=J.+R.+Marszalekauthor=C.+Toniattiauthor=P.+Jonesauthor=R.+T.+Lewis&title=Identification+of+Potent+and+Selective+MTH1+Inhibitors&doi=10.1016%2Fj.bmcl.2016.02.026"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2016.02.026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2016.02.026%26sid%3Dliteratum%253Aachs%26aulast%3DPetrocchi%26aufirst%3DA.%26aulast%3DLeo%26aufirst%3DE.%26aulast%3DReyna%26aufirst%3DN.%2BJ.%26aulast%3DHamilton%26aufirst%3DM.%2BM.%26aulast%3DShi%26aufirst%3DX.%26aulast%3DParker%26aufirst%3DC.%2BA.%26aulast%3DMseeh%26aufirst%3DF.%26aulast%3DBardenhagen%26aufirst%3DJ.%2BP.%26aulast%3DLeonard%26aufirst%3DP.%26aulast%3DCross%26aufirst%3DJ.%2BB.%26aulast%3DHuang%26aufirst%3DS.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DCardozo%26aufirst%3DM.%26aulast%3DDraetta%26aufirst%3DG.%26aulast%3DMarszalek%26aufirst%3DJ.%2BR.%26aulast%3DToniatti%26aufirst%3DC.%26aulast%3DJones%26aufirst%3DP.%26aulast%3DLewis%26aufirst%3DR.%2BT.%26atitle%3DIdentification%2520of%2520Potent%2520and%2520Selective%2520MTH1%2520Inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D26%26issue%3D6%26spage%3D1503%26epage%3D1507%26doi%3D10.1016%2Fj.bmcl.2016.02.026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kawamura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawatani, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muroi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kondoh, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Futamura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aono, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Honda, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osada, H.</span></span> <span> </span><span class="NLM_article-title">Proteomic Profiling of Small-Molecule Inhibitors Reveals Dispensability of MTH1 for Cancer Cell Survival</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">26521</span>, <span class="refDoi"> DOI: 10.1038/srep26521</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1038%2Fsrep26521" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=27210421" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BC28XosVSktLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=26521&author=T.+Kawamuraauthor=M.+Kawataniauthor=M.+Muroiauthor=Y.+Kondohauthor=Y.+Futamuraauthor=H.+Aonoauthor=M.+Tanakaauthor=K.+Hondaauthor=H.+Osada&title=Proteomic+Profiling+of+Small-Molecule+Inhibitors+Reveals+Dispensability+of+MTH1+for+Cancer+Cell+Survival&doi=10.1038%2Fsrep26521"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Proteomic profiling of small-molecule inhibitors reveals dispensability of MTH1 for cancer cell survival</span></div><div class="casAuthors">Kawamura, Tatsuro; Kawatani, Makoto; Muroi, Makoto; Kondoh, Yasumitsu; Futamura, Yushi; Aono, Harumi; Tanaka, Miho; Honda, Kaori; Osada, Hiroyuki</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">26521</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Since recent publications suggested that the survival of cancer cells depends on MTH1 to avoid incorporation of oxidized nucleotides into the cellular DNA, MTH1 has attracted attention as a potential cancer therapeutic target.  In this study, we identified new purine-based MTH1 inhibitors by chem. array screening.  However, although the MTH1 inhibitors identified in this study targeted cellular MTH1, they exhibited only weak cytotoxicity against cancer cells compared to recently reported first-in-class inhibitors.  We performed proteomic profiling to investigate the modes of action by which chem. distinct MTH1 inhibitors induce cancer cell death, and found mechanistic differences among the first-in-class MTH1 inhibitors.  In particular, we identified tubulin as the primary target of TH287 and TH588 responsible for the antitumor effects despite the nanomolar MTH1-inhibitory activity in vitro.  Furthermore, overexpression of MTH1 did not rescue cells from MTH1 inhibitor-induced cell death, and siRNA-mediated knockdown of MTH1 did not suppress cancer cell growth.  Taken together, we conclude that the cytotoxicity of MTH1 inhibitors is attributable to off-target effects and that MTH1 is not essential for cancer cell survival.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjRxqUq-o5ybVg90H21EOLACvtfcHk0lhahalI6EjZAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XosVSktLc%253D&md5=9d0e0e002f680456fccdf426d8f23539</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fsrep26521&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep26521%26sid%3Dliteratum%253Aachs%26aulast%3DKawamura%26aufirst%3DT.%26aulast%3DKawatani%26aufirst%3DM.%26aulast%3DMuroi%26aufirst%3DM.%26aulast%3DKondoh%26aufirst%3DY.%26aulast%3DFutamura%26aufirst%3DY.%26aulast%3DAono%26aufirst%3DH.%26aulast%3DTanaka%26aufirst%3DM.%26aulast%3DHonda%26aufirst%3DK.%26aulast%3DOsada%26aufirst%3DH.%26atitle%3DProteomic%2520Profiling%2520of%2520Small-Molecule%2520Inhibitors%2520Reveals%2520Dispensability%2520of%2520MTH1%2520for%2520Cancer%2520Cell%2520Survival%26jtitle%3DSci.%2520Rep.%26date%3D2016%26volume%3D6%26spage%3D26521%26doi%3D10.1038%2Fsrep26521" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaelin, W. G.</span></span> <span> </span><span class="NLM_article-title">Common Pitfalls in Preclinical Cancer Target Validation</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>17</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">425</span>, <span class="refDoi"> DOI: 10.1038/nrc.2017.32</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1038%2Fnrc.2017.32" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=28524181" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BC2sXotVKhsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2017&pages=425&issue=7&author=W.+G.+Kaelin&title=Common+Pitfalls+in+Preclinical+Cancer+Target+Validation&doi=10.1038%2Fnrc.2017.32"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Common pitfalls in preclinical cancer target validation</span></div><div class="casAuthors">Kaelin, William G., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">425-440</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">An alarming no. of papers from labs. nominating new cancer drug targets contain findings that cannot be reproduced by others or are simply not robust enough to justify drug discovery efforts.  This problem probably has many causes, including an underappreciation of the danger of being misled by off-target effects when using pharmacol. or genetic perturbants in complex biol. assays.  This danger is particularly acute when, as is often the case in cancer pharmacol., the biol. phenotype being measured is a 'down' readout (such as decreased proliferation, decreased viability or decreased tumor growth) that could simply reflect a nonspecific loss of cellular fitness.  These problems are compounded by multiple hypothesis testing, such as when candidate targets emerge from high-throughput screens that interrogate multiple targets in parallel, and by a publication and promotion system that preferentially rewards pos. findings.  In this Perspective, I outline some of the common pitfalls in preclin. cancer target identification and some potential approaches to mitigate them.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoO7ub-MnhM8LVg90H21EOLACvtfcHk0lis-H1twkCM4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXotVKhsrk%253D&md5=a4ae5a455831ecdd4f481b0f6f548c17</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnrc.2017.32&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc.2017.32%26sid%3Dliteratum%253Aachs%26aulast%3DKaelin%26aufirst%3DW.%2BG.%26atitle%3DCommon%2520Pitfalls%2520in%2520Preclinical%2520Cancer%2520Target%2520Validation%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2017%26volume%3D17%26issue%3D7%26spage%3D425%26doi%3D10.1038%2Fnrc.2017.32" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Samaranayake, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huynh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rai, P.</span></span> <span> </span><span class="NLM_article-title">MTH1 as a Chemotherapeutic Target: The Elephant in the Room</span>. <i>Cancers</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">47</span>, <span class="refDoi"> DOI: 10.3390/cancers9050047</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.3390%2Fcancers9050047" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=47&issue=5&author=G.+J.+Samaranayakeauthor=M.+Huynhauthor=P.+Rai&title=MTH1+as+a+Chemotherapeutic+Target%3A+The+Elephant+in+the+Room&doi=10.3390%2Fcancers9050047"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.3390%2Fcancers9050047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fcancers9050047%26sid%3Dliteratum%253Aachs%26aulast%3DSamaranayake%26aufirst%3DG.%2BJ.%26aulast%3DHuynh%26aufirst%3DM.%26aulast%3DRai%26aufirst%3DP.%26atitle%3DMTH1%2520as%2520a%2520Chemotherapeutic%2520Target%253A%2520The%2520Elephant%2520in%2520the%2520Room%26jtitle%3DCancers%26date%3D2017%26volume%3D9%26issue%3D5%26spage%3D47%26doi%3D10.3390%2Fcancers9050047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Audia, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Austin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blagg, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bountra, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buser-Doepner, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahlin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhanak, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimshaw, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hepworth, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kotz, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruger, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mader, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsden, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller-Fahrnow, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Hagan, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overington, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenberg, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundström, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Superti-Furga, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taunton, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toledo-Sherman, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walpole, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walters, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willson, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Workman, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuercher, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frederiksen, M.</span></span> <span> </span><span class="NLM_article-title">The Promise and Peril of Chemical Probes</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">536</span>– <span class="NLM_lpage">541</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1867</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1038%2Fnchembio.1867" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=26196764" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Wlu7zO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=536-541&issue=8&author=C.+H.+Arrowsmithauthor=J.+E.+Audiaauthor=C.+Austinauthor=J.+Baellauthor=J.+Bennettauthor=J.+Blaggauthor=C.+Bountraauthor=P.+E.+Brennanauthor=P.+J.+Brownauthor=M.+E.+Bunnageauthor=C.+Buser-Doepnerauthor=R.+M.+Campbellauthor=A.+J.+Carterauthor=P.+Cohenauthor=R.+A.+Copelandauthor=B.+Cravattauthor=J.+L.+Dahlinauthor=D.+Dhanakauthor=A.+M.+Edwardsauthor=S.+V.+Fryeauthor=N.+Grayauthor=C.+E.+Grimshawauthor=D.+Hepworthauthor=T.+Howeauthor=K.+V.+M+Huberauthor=J.+Jinauthor=S.+Knappauthor=J.+D.+Kotzauthor=R.+G.+Krugerauthor=D.+Loweauthor=M.+M.+Maderauthor=B.+Marsdenauthor=A.+Mueller-Fahrnowauthor=S.+M%C3%BCllerauthor=R.+C.+O%E2%80%99Haganauthor=J.+P.+Overingtonauthor=D.+R.+Owenauthor=S.+H.+Rosenbergauthor=B.+Rothauthor=R.+Rossauthor=M.+Schapiraauthor=S.+L.+Schreiberauthor=B.+Shoichetauthor=M.+Sundstr%C3%B6mauthor=G.+Superti-Furgaauthor=J.+Tauntonauthor=L.+Toledo-Shermanauthor=C.+Walpoleauthor=M.+A.+Waltersauthor=T.+M.+Willsonauthor=P.+Workmanauthor=R.+N.+Youngauthor=W.+J.+Zuercherauthor=M.+Frederiksen&title=The+Promise+and+Peril+of+Chemical+Probes&doi=10.1038%2Fnchembio.1867"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">The promise and peril of chemical probes</span></div><div class="casAuthors">Arrowsmith, Cheryl H.; Audia, James E.; Austin, Christopher; Baell, Jonathan; Bennett, Jonathan; Blagg, Julian; Bountra, Chas; Brennan, Paul E.; Brown, Peter J.; Bunnage, Mark E.; Buser-Doepner, Carolyn; Campbell, Robert M.; Carter, Adrian J.; Cohen, Philip; Copeland, Robert A.; Cravatt, Ben; Dahlin, Jayme L.; Dhanak, Dashyant; Edwards, Aled M.; Frye, Stephen V.; Gray, Nathanael; Grimshaw, Charles E.; Hepworth, David; Howe, Trevor; Huber, Kilian V. M.; Jin, Jian; Knapp, Stefan; Kotz, Joanne D.; Kruger, Ryan G.; Lowe, Derek; Mader, Mary M.; Marsden, Brian; Mueller-Fahrnow, Anke; Muller, Susanne; O'Hagan, Ronan C.; Overington, John P.; Owen, Dafydd R.; Rosenberg, Saul H.; Roth, Brian; Ross, Ruth; Schapira, Matthieu; Schreiber, Stuart L.; Shoichet, Brian; Sundstrom, Michael; Superti-Furga, Giulio; Taunton, Jack; Toledo-Sherman, Leticia; Walpole, Chris; Walters, Michael A.; Willson, Timothy M.; Workman, Paul; Young, Robert N.; Zuercher, William J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">536-541</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Chem. probes are powerful reagents with increasing impacts on biomedical research.  However, probes of poor quality or that are used incorrectly generate misleading results.  To help address these shortcomings, we will create a community-driven wiki resource to improve quality and convey current best practice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-li4dxq487rVg90H21EOLACvtfcHk0lgnU-m8jgTHXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Wlu7zO&md5=d8c7a7af94921ab4cdcc871487cceadd</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1867&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1867%26sid%3Dliteratum%253Aachs%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26aulast%3DAustin%26aufirst%3DC.%26aulast%3DBaell%26aufirst%3DJ.%26aulast%3DBennett%26aufirst%3DJ.%26aulast%3DBlagg%26aufirst%3DJ.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DBuser-Doepner%26aufirst%3DC.%26aulast%3DCampbell%26aufirst%3DR.%2BM.%26aulast%3DCarter%26aufirst%3DA.%2BJ.%26aulast%3DCohen%26aufirst%3DP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DCravatt%26aufirst%3DB.%26aulast%3DDahlin%26aufirst%3DJ.%2BL.%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DEdwards%26aufirst%3DA.%2BM.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DGray%26aufirst%3DN.%26aulast%3DGrimshaw%26aufirst%3DC.%2BE.%26aulast%3DHepworth%26aufirst%3DD.%26aulast%3DHowe%26aufirst%3DT.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DKotz%26aufirst%3DJ.%2BD.%26aulast%3DKruger%26aufirst%3DR.%2BG.%26aulast%3DLowe%26aufirst%3DD.%26aulast%3DMader%26aufirst%3DM.%2BM.%26aulast%3DMarsden%26aufirst%3DB.%26aulast%3DMueller-Fahrnow%26aufirst%3DA.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Hagan%26aufirst%3DR.%2BC.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DRosenberg%26aufirst%3DS.%2BH.%26aulast%3DRoth%26aufirst%3DB.%26aulast%3DRoss%26aufirst%3DR.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DShoichet%26aufirst%3DB.%26aulast%3DSundstr%25C3%25B6m%26aufirst%3DM.%26aulast%3DSuperti-Furga%26aufirst%3DG.%26aulast%3DTaunton%26aufirst%3DJ.%26aulast%3DToledo-Sherman%26aufirst%3DL.%26aulast%3DWalpole%26aufirst%3DC.%26aulast%3DWalters%26aufirst%3DM.%2BA.%26aulast%3DWillson%26aufirst%3DT.%2BM.%26aulast%3DWorkman%26aufirst%3DP.%26aulast%3DYoung%26aufirst%3DR.%2BN.%26aulast%3DZuercher%26aufirst%3DW.%2BJ.%26aulast%3DFrederiksen%26aufirst%3DM.%26atitle%3DThe%2520Promise%2520and%2520Peril%2520of%2520Chemical%2520Probes%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26issue%3D8%26spage%3D536%26epage%3D541%26doi%3D10.1038%2Fnchembio.1867" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blagg, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Workman, P.</span></span> <span> </span><span class="NLM_article-title">Choose and Use Your Chemical Probe Wisely to Explore Cancer Biology</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>32</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">9</span>– <span class="NLM_lpage">25</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2017.06.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1016%2Fj.ccell.2017.06.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=28697345" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFGmsLvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2017&pages=9-25&issue=1&author=J.+Blaggauthor=P.+Workman&title=Choose+and+Use+Your+Chemical+Probe+Wisely+to+Explore+Cancer+Biology&doi=10.1016%2Fj.ccell.2017.06.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Choose and Use Your Chemical Probe Wisely to Explore Cancer Biology</span></div><div class="casAuthors">Blagg, Julian; Workman, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">9-25</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Small-mol. chem. probes or tools have become progressively more important in recent years as valuable reagents to investigate fundamental biol. mechanisms and processes causing disease, including cancer.  Chem. probes have also achieved greater prominence alongside complementary biol. reagents for target validation in drug discovery.  However, there is evidence of widespread continuing misuse and promulgation of poor-quality and insufficiently selective chem. probes, perpetuating a worrisome and misleading pollution of the scientific literature.  We discuss current challenges with the selection and use of chem. probes, and suggest how biologists can and should be more discriminating in the probes they employ.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcXzLHHyTscrVg90H21EOLACvtfcHk0lgnU-m8jgTHXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFGmsLvI&md5=468848d484d5d91c52330949d5778b1d</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2017.06.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2017.06.005%26sid%3Dliteratum%253Aachs%26aulast%3DBlagg%26aufirst%3DJ.%26aulast%3DWorkman%26aufirst%3DP.%26atitle%3DChoose%2520and%2520Use%2520Your%2520Chemical%2520Probe%2520Wisely%2520to%2520Explore%2520Cancer%2520Biology%26jtitle%3DCancer%2520Cell%26date%3D2017%26volume%3D32%26issue%3D1%26spage%3D9%26epage%3D25%26doi%3D10.1016%2Fj.ccell.2017.06.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miyaura, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamada, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, A.</span></span> <span> </span><span class="NLM_article-title">A New Stereospecific Cross-Coupling by the Palladium-Catalyzed Reaction of 1-Alkenylboranes with 1-Alkenyl or 1-Alkynyl Halides</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">1979</span>,  <span class="NLM_volume"><i>20</i></span> (<span class="NLM_issue">36</span>),  <span class="NLM_fpage">3437</span>– <span class="NLM_lpage">3440</span>, <span class="refDoi"> DOI: 10.1016/S0040-4039(01)95429-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1016%2FS0040-4039%2801%2995429-2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=1979&pages=3437-3440&issue=36&author=N.+Miyauraauthor=K.+Yamadaauthor=A.+Suzuki&title=A+New+Stereospecific+Cross-Coupling+by+the+Palladium-Catalyzed+Reaction+of+1-Alkenylboranes+with+1-Alkenyl+or+1-Alkynyl+Halides&doi=10.1016%2FS0040-4039%2801%2995429-2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2FS0040-4039%2801%2995429-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0040-4039%252801%252995429-2%26sid%3Dliteratum%253Aachs%26aulast%3DMiyaura%26aufirst%3DN.%26aulast%3DYamada%26aufirst%3DK.%26aulast%3DSuzuki%26aufirst%3DA.%26atitle%3DA%2520New%2520Stereospecific%2520Cross-Coupling%2520by%2520the%2520Palladium-Catalyzed%2520Reaction%2520of%25201-Alkenylboranes%2520with%25201-Alkenyl%2520or%25201-Alkynyl%2520Halides%26jtitle%3DTetrahedron%2520Lett.%26date%3D1979%26volume%3D20%26issue%3D36%26spage%3D3437%26epage%3D3440%26doi%3D10.1016%2FS0040-4039%2801%2995429-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guram, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rennels, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchwald, S. L.</span></span> <span> </span><span class="NLM_article-title">A Simple Catalytic Method for the Conversion of Aryl Bromides to Arylamines</span>. <i>Angew. Chem., Int. Ed. Engl.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>34</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">1348</span>– <span class="NLM_lpage">1350</span>, <span class="refDoi"> DOI: 10.1002/anie.199513481</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1002%2Fanie.199513481" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADyaK2MXmvV2msr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=1995&pages=1348-1350&issue=12&author=A.+S.+Guramauthor=R.+A.+Rennelsauthor=S.+L.+Buchwald&title=A+Simple+Catalytic+Method+for+the+Conversion+of+Aryl+Bromides+to+Arylamines&doi=10.1002%2Fanie.199513481"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">A simple catalytic method for the conversion of aryl bromides to arylamines</span></div><div class="casAuthors">Guram, Anil S.; Rennels, Roger A.; Buchwald, Stephen L.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition in English</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1348-50</span>CODEN:
                <span class="NLM_cas:coden">ACIEAY</span>;
        ISSN:<span class="NLM_cas:issn">0570-0833</span>.
    
            (<span class="NLM_cas:orgname">VCH</span>)
        </div><div class="casAbstract">Aryl bromides reacted with amines in the presence of NaOCMe3 and a catalytic amt. of PdCl2[P(o-tolyl)3]2 or Pd(dba)2/2P(o-tolyl)3 to give arylamines in good yields.  Also, cyclization of 2-XC6H4CH2(CH2)nNHCH2Ph (X = Br, iodo; n = 1, 2, 3) in the presence of NaOCMe3 and a catalytic amt. of Pd(PPh3)4 gave good yields of the N heterocycles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXDC5Vhs2d57Vg90H21EOLACvtfcHk0li_HWPh92zQsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXmvV2msr8%253D&md5=d435e21c3f535757455d53e12417596b</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1002%2Fanie.199513481&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.199513481%26sid%3Dliteratum%253Aachs%26aulast%3DGuram%26aufirst%3DA.%2BS.%26aulast%3DRennels%26aufirst%3DR.%2BA.%26aulast%3DBuchwald%26aufirst%3DS.%2BL.%26atitle%3DA%2520Simple%2520Catalytic%2520Method%2520for%2520the%2520Conversion%2520of%2520Aryl%2520Bromides%2520to%2520Arylamines%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%2520Engl.%26date%3D1995%26volume%3D34%26issue%3D12%26spage%3D1348%26epage%3D1350%26doi%3D10.1002%2Fanie.199513481" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Louie, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartwig, J. F.</span></span> <span> </span><span class="NLM_article-title">Palladium-Catalyzed Synthesis of Arylamines from Aryl Halides. Mechanistic Studies Lead to Coupling in the Absence of Tin Reagents</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>36</i></span> (<span class="NLM_issue">21</span>),  <span class="NLM_fpage">3609</span>– <span class="NLM_lpage">3612</span>, <span class="refDoi"> DOI: 10.1016/0040-4039(95)00605-C</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1016%2F0040-4039%2895%2900605-C" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADyaK2MXmtFCjtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=1995&pages=3609-3612&issue=21&author=J.+Louieauthor=J.+F.+Hartwig&title=Palladium-Catalyzed+Synthesis+of+Arylamines+from+Aryl+Halides.+Mechanistic+Studies+Lead+to+Coupling+in+the+Absence+of+Tin+Reagents&doi=10.1016%2F0040-4039%2895%2900605-C"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Palladium-catalyzed synthesis of arylamines from aryl halides. Mechanistic studies lead to coupling in the absence of tin reagents</span></div><div class="casAuthors">Louie, Janis; Hartwig, John F.</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">3609-12</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The reaction of aryl halides with secondary amines in the presence of silylamide base and tri-o-tolyphospshine palladium complexes gives arylamine products.  This process provides a convenient method for performing these heterocross coupling reactions without the necessity for forming tin amides and disposing of tin halides.  This reaction follows from a mechanistic anal. of the coupling reaction with tin amides and occurs as a result of the cleavage of palladium aryl halide dimers with secondary amines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAKpHJAivoR7Vg90H21EOLACvtfcHk0li_HWPh92zQsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXmtFCjtLo%253D&md5=7c4567c3e54efc05049d539c682696b2</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2F0040-4039%2895%2900605-C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0040-4039%252895%252900605-C%26sid%3Dliteratum%253Aachs%26aulast%3DLouie%26aufirst%3DJ.%26aulast%3DHartwig%26aufirst%3DJ.%2BF.%26atitle%3DPalladium-Catalyzed%2520Synthesis%2520of%2520Arylamines%2520from%2520Aryl%2520Halides.%2520Mechanistic%2520Studies%2520Lead%2520to%2520Coupling%2520in%2520the%2520Absence%2520of%2520Tin%2520Reagents%26jtitle%3DTetrahedron%2520Lett.%26date%3D1995%26volume%3D36%26issue%3D21%26spage%3D3609%26epage%3D3612%26doi%3D10.1016%2F0040-4039%2895%2900605-C" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Edfeldt, F. N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Folmer, R. H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breeze, A. L.</span></span> <span> </span><span class="NLM_article-title">Fragment Screening to Predict Druggability (Ligandability) and Lead Discovery Success</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>16</i></span> (<span class="NLM_issue">7–8</span>),  <span class="NLM_fpage">284</span>– <span class="NLM_lpage">287</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2011.02.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1016%2Fj.drudis.2011.02.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=21315179" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BC3MXksFemsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2011&pages=284-287&issue=7%E2%80%938&author=F.+N.+B.+Edfeldtauthor=R.+H.+A.+Folmerauthor=A.+L.+Breeze&title=Fragment+Screening+to+Predict+Druggability+%28Ligandability%29+and+Lead+Discovery+Success&doi=10.1016%2Fj.drudis.2011.02.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment screening to predict druggability (ligandability) and lead discovery success</span></div><div class="casAuthors">Edfeldt, Fredrik N. B.; Folmer, Rutger H. A.; Breeze, Alexander L.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">7/8</span>),
    <span class="NLM_cas:pages">284-287</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">This article discusses the use of fragment screening to predict ligandability which will be of great interest to the pharmaceutical industry.  Data presented in this article clearly show how fragment-based ligandability screening can be used in a truly predictive fashion and could be a powerful tool in reducing agent attrition by filtering out non-ligandable targets before entry into the portfolio.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSXfOy9wAHFLVg90H21EOLACvtfcHk0ljZCQ7P3JHiiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXksFemsbs%253D&md5=7ba5adfe3d6a1eb4c9eaa3ce42c60044</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2011.02.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2011.02.002%26sid%3Dliteratum%253Aachs%26aulast%3DEdfeldt%26aufirst%3DF.%2BN.%2BB.%26aulast%3DFolmer%26aufirst%3DR.%2BH.%2BA.%26aulast%3DBreeze%26aufirst%3DA.%2BL.%26atitle%3DFragment%2520Screening%2520to%2520Predict%2520Druggability%2520%2528Ligandability%2529%2520and%2520Lead%2520Discovery%2520Success%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2011%26volume%3D16%26issue%3D7%25E2%2580%25938%26spage%3D284%26epage%3D287%26doi%3D10.1016%2Fj.drudis.2011.02.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Meanwell, N. A.</span></span> <span> </span><span class="NLM_article-title">Improving Drug Candidates by Design: A Focus on Physicochemical Properties As a Means of Improving Compound Disposition and Safety</span>. <i>Chem. Res. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>24</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">1420</span>– <span class="NLM_lpage">1456</span>, <span class="refDoi"> DOI: 10.1021/tx200211v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/tx200211v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BC3MXptlCitLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2011&pages=1420-1456&issue=9&author=N.+A.+Meanwell&title=Improving+Drug+Candidates+by+Design%3A+A+Focus+on+Physicochemical+Properties+As+a+Means+of+Improving+Compound+Disposition+and+Safety&doi=10.1021%2Ftx200211v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Improving Drug Candidates by Design: A Focus on Physicochemical Properties As a Means of Improving Compound Disposition and Safety</span></div><div class="casAuthors">Meanwell, Nicholas A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Research in Toxicology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1420-1456</span>CODEN:
                <span class="NLM_cas:coden">CRTOEC</span>;
        ISSN:<span class="NLM_cas:issn">0893-228X</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The development of small mol. drug candidates from the discovery phase to a marketed product continues to be a challenging enterprise with very low success rates that have fostered the perception of poor productivity by the pharmaceutical industry.  Although there have been significant advances in preclin. profiling that have improved compd. triaging and altered the underlying reasons for compd. attrition, the failure rates have not appreciably changed.  As part of an effort to more deeply understand the reasons for candidate failure, there has been considerable interest in analyzing the physicochem. properties of marketed drugs for the purpose of comparing with drugs in discovery and development as a means capturing recent trends in drug design.  The scenario that has emerged is one in which contemporary drug discovery is thought to be focused too heavily on advancing candidates with profiles that are most easily satisfied by mols. with increased mol. wt. and higher overall lipophilicity.  The preponderance of mols. expressing these properties is frequently a function of increased arom. ring count when compared with that of the drugs launched in the latter half of the 20th century and may reflect a preoccupation with maximizing target affinity rather than taking a more holistic approach to drug design.  These attributes not only present challenges for formulation and absorption but also may influence the manifestation of toxicity during development.  By providing some definition around the optimal physicochem. properties assocd. with marketed drugs, guidelines for drug design have been developed that are based largely on calcd. parameters and which may readily be applied by medicinal chemists as an aid to understanding candidate quality.  The physicochem. properties of a mol. that are consistent with the potential for good oral absorption were initially defined by Lipinski, with addnl. insights allowing further refinement, while deeper analyses have explored the correlation with metabolic stability and toxicity.  These insights have been augmented by careful analyses of physicochem. aspects of drug-target interactions, with thermodn. profiling indicating that the signature of best-in-class drugs is a dependence on enthalpy to drive binding energetics rather than entropy, which is dependent on lipophilicity.  Optimization of the entropic contribution to the binding energy of a ligand to its target is generally much easier than refining the enthalpic element.  Consequently, in the absence of a fundamental understanding of the thermodn. complexion of an interaction, the design of mols. with increased lipophilicity becomes almost inevitable.  The application of ligand efficiency, a measure of affinity per heavy atom, group efficiency, which assesses affinity in the context of structural changes, and lipophilic ligand efficiency, which relates potency to lipophilicity, offer less sophisticated but practically useful anal. algorithms to assess the quality of drug-target interactions.  These parameters are readily calcd. and can be applied to lead optimization programs in a fashion that helps to maximize potency while minimizing the kind of lipophilic burden that has been dubbed "mol. obesity".  Several recently described lead optimization campaigns provide illustrative, informative, and productive examples of the effect of paying close attention to carefully controlling physicochem. properties by monitoring ligand efficiency and lipophilic ligand efficiency.  However, to be successful during the lead optimization phase, drug candidate identification programs will need to adopt a holistic approach that integrates multiple parameters, many of which will have unique dependencies on both the drug target and the specific chemotype under prosecution.  Nevertheless, there are many important drug targets that necessitate working in space beyond that which has been defined by the retrospective analyses of marketed drugs and which will require adaptation of some of the guideposts that are useful in directing lead optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQrUfIwZiIR7Vg90H21EOLACvtfcHk0ljZCQ7P3JHiiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXptlCitLs%253D&md5=9ddcf6dc81bf8ab0441df5037979ff4e</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Ftx200211v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Ftx200211v%26sid%3Dliteratum%253Aachs%26aulast%3DMeanwell%26aufirst%3DN.%2BA.%26atitle%3DImproving%2520Drug%2520Candidates%2520by%2520Design%253A%2520A%2520Focus%2520on%2520Physicochemical%2520Properties%2520As%2520a%2520Means%2520of%2520Improving%2520Compound%2520Disposition%2520and%2520Safety%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D2011%26volume%3D24%26issue%3D9%26spage%3D1420%26epage%3D1456%26doi%3D10.1021%2Ftx200211v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groom, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alex, A.</span></span> <span> </span><span class="NLM_article-title">Ligand Efficiency: A Useful Metric for Lead Selection</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">430</span>– <span class="NLM_lpage">431</span>, <span class="refDoi"> DOI: 10.1016/S1359-6446(04)03069-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1016%2FS1359-6446%2804%2903069-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=15109945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A280%3ADC%252BD2c3ht1ygtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2004&pages=430-431&issue=10&author=A.+L.+Hopkinsauthor=C.+R.+Groomauthor=A.+Alex&title=Ligand+Efficiency%3A+A+Useful+Metric+for+Lead+Selection&doi=10.1016%2FS1359-6446%2804%2903069-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand efficiency: a useful metric for lead selection</span></div><div class="casAuthors">Hopkins Andrew L; Groom Colin R; Alex Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">430-1</span>
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSdMyKW_d6RPeIHSmpohpd0fW6udTcc2eZ-Vn2N6itFYbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2c3ht1ygtw%253D%253D&md5=eb19a1bca53247d0bd9adcb99d5817e6</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2804%2903069-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252804%252903069-7%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26aulast%3DAlex%26aufirst%3DA.%26atitle%3DLigand%2520Efficiency%253A%2520A%2520Useful%2520Metric%2520for%2520Lead%2520Selection%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2004%26volume%3D9%26issue%3D10%26spage%3D430%26epage%3D431%26doi%3D10.1016%2FS1359-6446%2804%2903069-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hann, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keserü, G. M.</span></span> <span> </span><span class="NLM_article-title">Finding the Sweet Spot: The Role of Nature and Nurture in Medicinal Chemistry</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">355</span>– <span class="NLM_lpage">365</span>, <span class="refDoi"> DOI: 10.1038/nrd3701</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1038%2Fnrd3701" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=22543468" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BC38Xmt1GqsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=355-365&issue=5&author=M.+M.+Hannauthor=G.+M.+Keser%C3%BC&title=Finding+the+Sweet+Spot%3A+The+Role+of+Nature+and+Nurture+in+Medicinal+Chemistry&doi=10.1038%2Fnrd3701"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Finding the sweet spot: the role of nature and nurture in medicinal chemistry</span></div><div class="casAuthors">Hann, Michael M.; Keserue, Gyoergy M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">355-365</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Given its position at the heart of small-mol. drug discovery, medicinal chem. has an important role in tackling the well-known productivity challenges in pharmaceutical research and development.  In recent years, extensive analyses of successful and failed discovery compds. and drug candidates have improved our understanding of the role of physicochem. properties in drug attrition.  Based on the clarified challenges in finding the 'sweet spot' in medicinal chem. programs, we suggest that this goal can be achieved through a combination of first identifying chem. starting points with appropriate 'nature' and then rigorously 'nurturing' them during lead optimization.  Here, we discuss scientific, strategic, organizational and cultural considerations for medicinal chem. practices, with the aim of promoting more effective use of what is already known, as well as a wider appreciation of the risks of pursuing suboptimal compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrURciGjB2A3LVg90H21EOLACvtfcHk0lgwwloCI4GWCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xmt1GqsLY%253D&md5=90170e3f4e69e52bfa38667df380516a</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fnrd3701&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3701%26sid%3Dliteratum%253Aachs%26aulast%3DHann%26aufirst%3DM.%2BM.%26aulast%3DKeser%25C3%25BC%26aufirst%3DG.%2BM.%26atitle%3DFinding%2520the%2520Sweet%2520Spot%253A%2520The%2520Role%2520of%2520Nature%2520and%2520Nurture%2520in%2520Medicinal%2520Chemistry%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2012%26volume%3D11%26issue%3D5%26spage%3D355%26epage%3D365%26doi%3D10.1038%2Fnrd3701" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mérour, J.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buron, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plé, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonnet, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Routier, S.</span></span> <span> </span><span class="NLM_article-title">The Azaindole Framework in the Design of Kinase Inhibitors</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>19</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">19935</span>– <span class="NLM_lpage">19979</span>, <span class="refDoi"> DOI: 10.3390/molecules191219935</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.3390%2Fmolecules191219935" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2014&pages=19935-19979&issue=12&author=J.-Y.+M%C3%A9rourauthor=F.+Buronauthor=K.+Pl%C3%A9author=P.+Bonnetauthor=S.+Routier&title=The+Azaindole+Framework+in+the+Design+of+Kinase+Inhibitors&doi=10.3390%2Fmolecules191219935"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.3390%2Fmolecules191219935&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules191219935%26sid%3Dliteratum%253Aachs%26aulast%3DM%25C3%25A9rour%26aufirst%3DJ.-Y.%26aulast%3DBuron%26aufirst%3DF.%26aulast%3DPl%25C3%25A9%26aufirst%3DK.%26aulast%3DBonnet%26aufirst%3DP.%26aulast%3DRoutier%26aufirst%3DS.%26atitle%3DThe%2520Azaindole%2520Framework%2520in%2520the%2520Design%2520of%2520Kinase%2520Inhibitors%26jtitle%3DMolecules%26date%3D2014%26volume%3D19%26issue%3D12%26spage%3D19935%26epage%3D19979%26doi%3D10.3390%2Fmolecules191219935" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lovering, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bikker, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humblet, C.</span></span> <span> </span><span class="NLM_article-title">Escape from Flatland: Increasing Saturation as an Approach to Improving Clinical Success</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>52</i></span> (<span class="NLM_issue">21</span>),  <span class="NLM_fpage">6752</span>– <span class="NLM_lpage">6756</span>, <span class="refDoi"> DOI: 10.1021/jm901241e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901241e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1KjtLvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=6752-6756&issue=21&author=F.+Loveringauthor=J.+Bikkerauthor=C.+Humblet&title=Escape+from+Flatland%3A+Increasing+Saturation+as+an+Approach+to+Improving+Clinical+Success&doi=10.1021%2Fjm901241e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Escape from Flatland: Increasing Saturation as an Approach to Improving Clinical Success</span></div><div class="casAuthors">Lovering, Frank; Bikker, Jack; Humblet, Christine</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6752-6756</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The medicinal chem. community has become increasingly aware of the value of tracking calcd. phys. properties such as mol. wt., topol. polar surface area, rotatable bonds, and hydrogen bond donors and acceptors.  The authors hypothesized that the shift to high-throughput synthetic practices over the past decade may be another factor that may predispose mols. to fail by steering discovery efforts toward achiral, arom. compds.  The authors have proposed two simple and interpretable measures of the complexity of mols. prepd. as potential drug candidates.  The first is carbon bond satn. as defined by fraction Sp3 (Fsp3) where Fsp3 = (no. of Sp3 hybridized carbons/total carbon count).  The second is simply whether a chiral carbon exists in the mol.  The authors demonstrate that both complexity (as measured by Fsp3) and the presence of chiral centers correlate with success as compds. transition from discovery, through clin. testing, to drugs.  To explain these observations, the authors further demonstrate that satn. correlates with soly., an exptl. phys. property important to success in the drug discovery setting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoC1f3Ed7nUBbVg90H21EOLACvtfcHk0lhXG2o2mv9J7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1KjtLvN&md5=4ca92c30c17c53d77ad376719bad951e</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fjm901241e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901241e%26sid%3Dliteratum%253Aachs%26aulast%3DLovering%26aufirst%3DF.%26aulast%3DBikker%26aufirst%3DJ.%26aulast%3DHumblet%26aufirst%3DC.%26atitle%3DEscape%2520from%2520Flatland%253A%2520Increasing%2520Saturation%2520as%2520an%2520Approach%2520to%2520Improving%2520Clinical%2520Success%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26issue%3D21%26spage%3D6752%26epage%3D6756%26doi%3D10.1021%2Fjm901241e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, D. V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luscombe, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, A. P.</span></span> <span> </span><span class="NLM_article-title">Getting Physical in Drug Discovery II: The Impact of Chromatographic Hydrophobicity Measurements and Aromaticity</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>16</i></span> (<span class="NLM_issue">17–18</span>),  <span class="NLM_fpage">822</span>– <span class="NLM_lpage">830</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2011.06.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1016%2Fj.drudis.2011.06.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=21704184" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFCru7vF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2011&pages=822-830&issue=17%E2%80%9318&author=R.+J.+Youngauthor=D.+V.+S.+Greenauthor=C.+N.+Luscombeauthor=A.+P.+Hill&title=Getting+Physical+in+Drug+Discovery+II%3A+The+Impact+of+Chromatographic+Hydrophobicity+Measurements+and+Aromaticity&doi=10.1016%2Fj.drudis.2011.06.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Getting physical in drug discovery II: the impact of chromatographic hydrophobicity measurements and aromaticity</span></div><div class="casAuthors">Young, Robert J.; Green, Darren V. S.; Luscombe, Christopher N.; Hill, Alan P.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">17/18</span>),
    <span class="NLM_cas:pages">822-830</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Here, we review the performance of chromatog. hydrophobicity measurements in a data set of 100 000 GlaxoSmithKline compds., demonstrating the advantages of the method over octanol-water partitioning and highlighting new insights for drug discovery.  The value of chromatog. measurements, vs. other hydrophobicity ests., was supported by improved relationships with soly., permeation, cytochrome P450s, intrinsic clearance, hERG binding and promiscuity.  We also obsd. marked differentiation of the relative influence of intrinsic and effective hydrophobicity.  The summing of hydrophobicity values plus arom. ring count [log D pH7.4 (or log P) + #Ar], indicated a wide relevance for simplistic property forecast indexes' in developability assays, clearly enhanced by chromatog. values; therefore establishing new foundations for enriching property-based drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmIJyWi-DBW7Vg90H21EOLACvtfcHk0lhXG2o2mv9J7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFCru7vF&md5=22788603a45aa77537e8666433473d14</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2011.06.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2011.06.001%26sid%3Dliteratum%253Aachs%26aulast%3DYoung%26aufirst%3DR.%2BJ.%26aulast%3DGreen%26aufirst%3DD.%2BV.%2BS.%26aulast%3DLuscombe%26aufirst%3DC.%2BN.%26aulast%3DHill%26aufirst%3DA.%2BP.%26atitle%3DGetting%2520Physical%2520in%2520Drug%2520Discovery%2520II%253A%2520The%2520Impact%2520of%2520Chromatographic%2520Hydrophobicity%2520Measurements%2520and%2520Aromaticity%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2011%26volume%3D16%26issue%3D17%25E2%2580%259318%26spage%3D822%26epage%3D830%26doi%3D10.1016%2Fj.drudis.2011.06.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winiwarter, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ax, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lennernäs, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hallberg, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pettersson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karlén, A.</span></span> <span> </span><span class="NLM_article-title">Hydrogen Bonding Descriptors in the Prediction of Human in Vivo Intestinal Permeability</span>. <i>J. Mol. Graphics Modell.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>21</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">273</span>– <span class="NLM_lpage">287</span>, <span class="refDoi"> DOI: 10.1016/S1093-3263(02)00163-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1016%2FS1093-3263%2802%2900163-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=12479927" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BD38XptlWmur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2003&pages=273-287&issue=4&author=S.+Winiwarterauthor=F.+Axauthor=H.+Lennern%C3%A4sauthor=A.+Hallbergauthor=C.+Petterssonauthor=A.+Karl%C3%A9n&title=Hydrogen+Bonding+Descriptors+in+the+Prediction+of+Human+in+Vivo+Intestinal+Permeability&doi=10.1016%2FS1093-3263%2802%2900163-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Hydrogen bonding descriptors in the prediction of human in vivo intestinal permeability</span></div><div class="casAuthors">Winiwarter, Susanne; Ax, Fredrik; Lennernas, Hans; Hallberg, Anders; Pettersson, Curt; Karlen, Anders</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">273-287</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Inc.</span>)
        </div><div class="casAbstract">Hydrogen bonding has been identified as an important parameter for describing drug permeability.  Recently, we derived models for predicting intestinal permeability using the hydrogen bonding descriptors polar surface area (PSA) and no. of hydrogen bond donors (HBD), and a lipophilicity descriptor [J. Med. hem. 41 (1998) 4939].  We have now explored other types of hydrogen bonding descriptors to see if these improve the models.  Both an exptl. hydrogen bonding descriptor, Δlog P, and calcd. descriptors, based either on semiempirical calcns. or on exptl. derived hydrogen bond strength values of small mols., were used.  Principal component analyses (PCA) were performed in order to characterize the different parameters, using both a drug data set and a data set of small non-drug-like mols. for which Δlog P-values had been published.For a set of diverse drug mols., for which human intestinal permeability data was available, a PLS-anal. was performed to study the correlation of permeability to the different hydrogen bonding parameters.  No correlation could be identified between Δlog P and human intestinal permeability in this data set.  However, the combination of a hydrogen bond donor descriptor, a general hydrogen bonding descriptor and a lipophilicity descriptor enabled the prediction of human intestinal permeability, whereas hydrogen bond acceptor descriptors were found to be less important.  The obtained models successfully predicted the intestinal permeability values of two external data sets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmZy8iv6rXkLVg90H21EOLACvtfcHk0lgeWKmWS6aowA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XptlWmur0%253D&md5=30a2bc60186daf6f649110816e3c323b</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2FS1093-3263%2802%2900163-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1093-3263%252802%252900163-8%26sid%3Dliteratum%253Aachs%26aulast%3DWiniwarter%26aufirst%3DS.%26aulast%3DAx%26aufirst%3DF.%26aulast%3DLennern%25C3%25A4s%26aufirst%3DH.%26aulast%3DHallberg%26aufirst%3DA.%26aulast%3DPettersson%26aufirst%3DC.%26aulast%3DKarl%25C3%25A9n%26aufirst%3DA.%26atitle%3DHydrogen%2520Bonding%2520Descriptors%2520in%2520the%2520Prediction%2520of%2520Human%2520in%2520Vivo%2520Intestinal%2520Permeability%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2003%26volume%3D21%26issue%3D4%26spage%3D273%26epage%3D287%26doi%3D10.1016%2FS1093-3263%2802%2900163-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leeson, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. J.</span></span> <span> </span><span class="NLM_article-title">Molecular Property Design: Does Everyone Get It?</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">722</span>– <span class="NLM_lpage">725</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.5b00157</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00157" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptVGksr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=722-725&issue=7&author=P.+D.+Leesonauthor=R.+J.+Young&title=Molecular+Property+Design%3A+Does+Everyone+Get+It%3F&doi=10.1021%2Facsmedchemlett.5b00157"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Property Design: Does Everyone Get It?</span></div><div class="casAuthors">Leeson, Paul D.; Young, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">722-725</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The principles of mol. property optimization in drug design have been understood for decades, yet much drug discovery activity today is conducted at the periphery of historical druglike property space.  Lead optimization trajectories aimed at reducing physicochem. risk, assisted by ligand efficiency metrics, could help to reduce clin. attrition rates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_t8gIn79arVg90H21EOLACvtfcHk0lgeWKmWS6aowA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptVGksr4%253D&md5=5a807a8d049ce8cf4adc186891a9234d</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00157&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00157%26sid%3Dliteratum%253Aachs%26aulast%3DLeeson%26aufirst%3DP.%2BD.%26aulast%3DYoung%26aufirst%3DR.%2BJ.%26atitle%3DMolecular%2520Property%2520Design%253A%2520Does%2520Everyone%2520Get%2520It%253F%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26issue%3D7%26spage%3D722%26epage%3D725%26doi%3D10.1021%2Facsmedchemlett.5b00157" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hann, M. M.</span></span> <span> </span><span class="NLM_article-title">Molecular Obesity, Potency and Other Addictions in Drug Discovery</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">349</span>– <span class="NLM_lpage">355</span>, <span class="refDoi"> DOI: 10.1039/c1md00017a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1039%2Fc1md00017a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BC3MXls1Kht7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2011&pages=349-355&author=M.+M.+Hann&title=Molecular+Obesity%2C+Potency+and+Other+Addictions+in+Drug+Discovery&doi=10.1039%2Fc1md00017a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular obesity, potency and other addictions in drug discovery</span></div><div class="casAuthors">Hann, Michael M.</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">349-355</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  Despite the increase in global biol. and chem. knowledge the discovery of effective and safe new drugs seems to become harder rather than easier.  Some of this challenge is due to increasing demands for safety and novelty, but some of the risk involved in this should be controllable if we had more effectively learned from our failures.  This perspective reflects on some of the learnings of recent years in relation to the causes of attrition.  The term Mol. Obesity is introduced to describe our tendency to build potency into mols. by the inappropriate use of lipophilicity which leads to the premature demise of drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSlF7TPy6fHLVg90H21EOLACvtfcHk0lidbHI_43Wg4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXls1Kht7g%253D&md5=954870db4b1d9e846612c3bb256582f7</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1039%2Fc1md00017a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc1md00017a%26sid%3Dliteratum%253Aachs%26aulast%3DHann%26aufirst%3DM.%2BM.%26atitle%3DMolecular%2520Obesity%252C%2520Potency%2520and%2520Other%2520Addictions%2520in%2520Drug%2520Discovery%26jtitle%3DMedChemComm%26date%3D2011%26volume%3D2%26spage%3D349%26epage%3D355%26doi%3D10.1039%2Fc1md00017a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Testa, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krämer, S. D.</span></span> <span> </span><span class="NLM_article-title">The Biochemistry of Drug Metabolism – An Introduction</span>. <i>Chem. Biodiversity</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>4</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">2031</span>– <span class="NLM_lpage">2122</span>, <span class="refDoi"> DOI: 10.1002/cbdv.200790169</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1002%2Fcbdv.200790169" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2007&pages=2031-2122&issue=9&author=B.+Testaauthor=S.+D.+Kr%C3%A4mer&title=The+Biochemistry+of+Drug+Metabolism+%E2%80%93+An+Introduction&doi=10.1002%2Fcbdv.200790169"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1002%2Fcbdv.200790169&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbdv.200790169%26sid%3Dliteratum%253Aachs%26aulast%3DTesta%26aufirst%3DB.%26aulast%3DKr%25C3%25A4mer%26aufirst%3DS.%2BD.%26atitle%3DThe%2520Biochemistry%2520of%2520Drug%2520Metabolism%2520%25E2%2580%2593%2520An%2520Introduction%26jtitle%3DChem.%2520Biodiversity%26date%3D2007%26volume%3D4%26issue%3D9%26spage%3D2031%26epage%3D2122%26doi%3D10.1002%2Fcbdv.200790169" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mancera, R. L.</span></span> <span> </span><span class="NLM_article-title">Molecular Modeling of Hydration in Drug Design</span>. <i>Curr. Opin. Drug Discovery Devel.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>10</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">275</span>– <span class="NLM_lpage">280</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2007&pages=275-280&issue=3&author=R.+L.+Mancera&title=Molecular+Modeling+of+Hydration+in+Drug+Design"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMancera%26aufirst%3DR.%2BL.%26atitle%3DMolecular%2520Modeling%2520of%2520Hydration%2520in%2520Drug%2520Design%26jtitle%3DCurr.%2520Opin.%2520Drug%2520Discovery%2520Devel.%26date%3D2007%26volume%3D10%26issue%3D3%26spage%3D275%26epage%3D280" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berne, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span> <span> </span><span class="NLM_article-title">Ligand Binding to Protein-Binding Pockets with Wet and Dry Regions</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>108</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">1326</span>– <span class="NLM_lpage">1330</span>, <span class="refDoi"> DOI: 10.1073/pnas.1016793108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1073%2Fpnas.1016793108" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=1326-1330&issue=4&author=L.+Wangauthor=B.+J.+Berneauthor=R.+A.+Friesner&title=Ligand+Binding+to+Protein-Binding+Pockets+with+Wet+and+Dry+Regions&doi=10.1073%2Fpnas.1016793108"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1016793108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1016793108%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DBerne%26aufirst%3DB.%2BJ.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DLigand%2520Binding%2520to%2520Protein-Binding%2520Pockets%2520with%2520Wet%2520and%2520Dry%2520Regions%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26issue%3D4%26spage%3D1326%26epage%3D1330%26doi%3D10.1073%2Fpnas.1016793108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Molina, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jafari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ignatushchenko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsson, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sreekumar, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nordlund, P.</span></span> <span> </span><span class="NLM_article-title">Monitoring Drug Target Engagement in Cells and Tissues Using the Cellular Thermal Shift Assay</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>341</i></span> (<span class="NLM_issue">6141</span>),  <span class="NLM_fpage">84</span>– <span class="NLM_lpage">87</span>, <span class="refDoi"> DOI: 10.1126/science.1233606</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1126%2Fscience.1233606" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=23828940" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVCgs7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=341&publication_year=2013&pages=84-87&issue=6141&author=D.+M.+Molinaauthor=R.+Jafariauthor=M.+Ignatushchenkoauthor=T.+Sekiauthor=E.+A.+Larssonauthor=C.+Danauthor=L.+Sreekumarauthor=Y.+Caoauthor=P.+Nordlund&title=Monitoring+Drug+Target+Engagement+in+Cells+and+Tissues+Using+the+Cellular+Thermal+Shift+Assay&doi=10.1126%2Fscience.1233606"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Monitoring Drug Target Engagement in Cells and Tissues Using the Cellular Thermal Shift Assay</span></div><div class="casAuthors">Molina, Daniel Martinez; Jafari, Rozbeh; Ignatushchenko, Marina; Seki, Takahiro; Larsson, E. Andreas; Dan, Chen; Sreekumar, Lekshmy; Cao, Yihai; Nordlund, Paer</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">341</span>
        (<span class="NLM_cas:issue">6141</span>),
    <span class="NLM_cas:pages">84-87</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The efficacy of therapeutics is dependent on a drug binding to its cognate target.  Optimization of target engagement by drugs in cells is often challenging, because drug binding cannot be monitored inside cells.  We have developed a method for evaluating drug binding to target proteins in cells and tissue samples.  This cellular thermal shift assay (CETSA) is based on the biophys. principle of ligand-induced thermal stabilization of target proteins.  Using this assay, we validated drug binding for a set of important clin. targets and monitored processes of drug transport and activation, off-target effects and drug resistance in cancer cell lines, as well as drug distribution in tissues.  CETSA is likely to become a valuable tool for the validation and optimization of drug target engagement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaaWSoozY7ObVg90H21EOLACvtfcHk0lidbHI_43Wg4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVCgs7rP&md5=f8ba686ba38b67dbdf366c49eb7375b4</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1126%2Fscience.1233606&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1233606%26sid%3Dliteratum%253Aachs%26aulast%3DMolina%26aufirst%3DD.%2BM.%26aulast%3DJafari%26aufirst%3DR.%26aulast%3DIgnatushchenko%26aufirst%3DM.%26aulast%3DSeki%26aufirst%3DT.%26aulast%3DLarsson%26aufirst%3DE.%2BA.%26aulast%3DDan%26aufirst%3DC.%26aulast%3DSreekumar%26aufirst%3DL.%26aulast%3DCao%26aufirst%3DY.%26aulast%3DNordlund%26aufirst%3DP.%26atitle%3DMonitoring%2520Drug%2520Target%2520Engagement%2520in%2520Cells%2520and%2520Tissues%2520Using%2520the%2520Cellular%2520Thermal%2520Shift%2520Assay%26jtitle%3DScience%26date%3D2013%26volume%3D341%26issue%3D6141%26spage%3D84%26epage%3D87%26doi%3D10.1126%2Fscience.1233606" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pierce, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krilov, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lupyan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahlgren, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenwood, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romero, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masse, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knight, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinbrecher, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beuming, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damm, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harder, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brewer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wester, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murcko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farid, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mobley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berne, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abel, R.</span></span> <span> </span><span class="NLM_article-title">Accurate and Reliable Prediction of Relative Ligand Binding Potency in Prospective Drug Discovery by Way of a Modern Free-Energy Calculation Protocol and Force Field</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>137</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">2695</span>– <span class="NLM_lpage">2703</span>, <span class="refDoi"> DOI: 10.1021/ja512751q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja512751q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsF2iuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2015&pages=2695-2703&issue=7&author=L.+Wangauthor=Y.+Wuauthor=Y.+Dengauthor=B.+Kimauthor=L.+Pierceauthor=G.+Krilovauthor=D.+Lupyanauthor=S.+Robinsonauthor=M.+K.+Dahlgrenauthor=J.+Greenwoodauthor=D.+L.+Romeroauthor=C.+Masseauthor=J.+L.+Knightauthor=T.+Steinbrecherauthor=T.+Beumingauthor=W.+Dammauthor=E.+Harderauthor=W.+Shermanauthor=M.+Brewerauthor=R.+Westerauthor=M.+Murckoauthor=L.+Fryeauthor=R.+Faridauthor=T.+Linauthor=D.+L.+Mobleyauthor=W.+L.+Jorgensenauthor=B.+J.+Berneauthor=R.+A.+Friesnerauthor=R.+Abel&title=Accurate+and+Reliable+Prediction+of+Relative+Ligand+Binding+Potency+in+Prospective+Drug+Discovery+by+Way+of+a+Modern+Free-Energy+Calculation+Protocol+and+Force+Field&doi=10.1021%2Fja512751q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Accurate and Reliable Prediction of Relative Ligand Binding Potency in Prospective Drug Discovery by Way of a Modern Free-Energy Calculation Protocol and Force Field</span></div><div class="casAuthors">Wang, Lingle; Wu, Yujie; Deng, Yuqing; Kim, Byungchan; Pierce, Levi; Krilov, Goran; Lupyan, Dmitry; Robinson, Shaughnessy; Dahlgren, Markus K.; Greenwood, Jeremy; Romero, Donna L.; Masse, Craig; Knight, Jennifer L.; Steinbrecher, Thomas; Beuming, Thijs; Damm, Wolfgang; Harder, Ed; Sherman, Woody; Brewer, Mark; Wester, Ron; Murcko, Mark; Frye, Leah; Farid, Ramy; Lin, Teng; Mobley, David L.; Jorgensen, William L.; Berne, Bruce J.; Friesner, Richard A.; Abel, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2695-2703</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Designing tight-binding ligands is a primary objective of small-mol. drug discovery.  Over the past few decades, free-energy calcns. have benefited from improved force fields and sampling algorithms, as well as the advent of low-cost parallel computing.  However, it has proven to be challenging to reliably achieve the level of accuracy that would be needed to guide lead optimization (∼5× in binding affinity) for a wide range of ligands and protein targets.  Not surprisingly, widespread com. application of free-energy simulations has been limited due to the lack of large-scale validation coupled with the tech. challenges traditionally assocd. with running these types of calcns.  Here, we report an approach that achieves an unprecedented level of accuracy across a broad range of target classes and ligands, with retrospective results encompassing 200 ligands and a wide variety of chem. perturbations, many of which involve significant changes in ligand chem. structures.  In addn., we have applied the method in prospective drug discovery projects and found a significant improvement in the quality of the compds. synthesized that have been predicted to be potent.  Compds. predicted to be potent by this approach have a substantial redn. in false positives relative to compds. synthesized on the basis of other computational or medicinal chem. approaches.  Furthermore, the results are consistent with those obtained from our retrospective studies, demonstrating the robustness and broad range of applicability of this approach, which can be used to drive decisions in lead optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGor9UtXEyLPpLVg90H21EOLACvtfcHk0lgOLAzsun8XpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsF2iuro%253D&md5=37a4f4a6c085f47ed531342643b6c33b</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fja512751q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja512751q%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DDeng%26aufirst%3DY.%26aulast%3DKim%26aufirst%3DB.%26aulast%3DPierce%26aufirst%3DL.%26aulast%3DKrilov%26aufirst%3DG.%26aulast%3DLupyan%26aufirst%3DD.%26aulast%3DRobinson%26aufirst%3DS.%26aulast%3DDahlgren%26aufirst%3DM.%2BK.%26aulast%3DGreenwood%26aufirst%3DJ.%26aulast%3DRomero%26aufirst%3DD.%2BL.%26aulast%3DMasse%26aufirst%3DC.%26aulast%3DKnight%26aufirst%3DJ.%2BL.%26aulast%3DSteinbrecher%26aufirst%3DT.%26aulast%3DBeuming%26aufirst%3DT.%26aulast%3DDamm%26aufirst%3DW.%26aulast%3DHarder%26aufirst%3DE.%26aulast%3DSherman%26aufirst%3DW.%26aulast%3DBrewer%26aufirst%3DM.%26aulast%3DWester%26aufirst%3DR.%26aulast%3DMurcko%26aufirst%3DM.%26aulast%3DFrye%26aufirst%3DL.%26aulast%3DFarid%26aufirst%3DR.%26aulast%3DLin%26aufirst%3DT.%26aulast%3DMobley%26aufirst%3DD.%2BL.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DBerne%26aufirst%3DB.%2BJ.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DAbel%26aufirst%3DR.%26atitle%3DAccurate%2520and%2520Reliable%2520Prediction%2520of%2520Relative%2520Ligand%2520Binding%2520Potency%2520in%2520Prospective%2520Drug%2520Discovery%2520by%2520Way%2520of%2520a%2520Modern%2520Free-Energy%2520Calculation%2520Protocol%2520and%2520Force%2520Field%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2015%26volume%3D137%26issue%3D7%26spage%3D2695%26epage%3D2703%26doi%3D10.1021%2Fja512751q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rudling, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafsson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almlöf, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Homan, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scobie, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warpman
Berglund, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helleday, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stenmark, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlsson, J.</span></span> <span> </span><span class="NLM_article-title">Fragment-Based Discovery and Optimization of Enzyme Inhibitors by Docking of Commercial Chemical Space</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span> (<span class="NLM_issue">19</span>),  <span class="NLM_fpage">8160</span>– <span class="NLM_lpage">8169</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01006</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01006" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=8160-8169&issue=19&author=A.+Rudlingauthor=R.+Gustafssonauthor=I.+Alml%C3%B6fauthor=E.+Homanauthor=M.+Scobieauthor=U.+Warpman%0ABerglundauthor=T.+Helledayauthor=P.+Stenmarkauthor=J.+Carlsson&title=Fragment-Based+Discovery+and+Optimization+of+Enzyme+Inhibitors+by+Docking+of+Commercial+Chemical+Space&doi=10.1021%2Facs.jmedchem.7b01006"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01006%26sid%3Dliteratum%253Aachs%26aulast%3DRudling%26aufirst%3DA.%26aulast%3DGustafsson%26aufirst%3DR.%26aulast%3DAlml%25C3%25B6f%26aufirst%3DI.%26aulast%3DHoman%26aufirst%3DE.%26aulast%3DScobie%26aufirst%3DM.%26aulast%3DWarpman%2BBerglund%26aufirst%3DU.%26aulast%3DHelleday%26aufirst%3DT.%26aulast%3DStenmark%26aufirst%3DP.%26aulast%3DCarlsson%26aufirst%3DJ.%26atitle%3DFragment-Based%2520Discovery%2520and%2520Optimization%2520of%2520Enzyme%2520Inhibitors%2520by%2520Docking%2520of%2520Commercial%2520Chemical%2520Space%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26issue%3D19%26spage%3D8160%26epage%3D8169%26doi%3D10.1021%2Facs.jmedchem.7b01006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">DeLano, W. L.</span></span> <span> </span><span class="NLM_article-title">Unraveling Hot Spots in Binding Interfaces: Progress and Challenges</span>. <i>Curr. Opin. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>12</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">14</span>– <span class="NLM_lpage">20</span>, <span class="refDoi"> DOI: 10.1016/S0959-440X(02)00283-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1016%2FS0959-440X%2802%2900283-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=11839484" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BD38XhtF2it70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2002&pages=14-20&issue=1&author=W.+L.+DeLano&title=Unraveling+Hot+Spots+in+Binding+Interfaces%3A+Progress+and+Challenges&doi=10.1016%2FS0959-440X%2802%2900283-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Unraveling hot spots in binding interfaces: progress and challenges</span></div><div class="casAuthors">DeLano, Warren L.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">14-20</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review with 49 refs.  Protein interface hot spots, as revealed by alanine scanning mutagenesis, continue to stimulate interest in the biophys. basis of mol. recognition.  Although these regions apparently constitute fertile grounds for intermol. interactions, no general algorithm has yet been developed that can predict hot spots based solely on their shape or compn.  The discovery of structural plasticity in hot spot regions indicates that dynamic simulation techniques may be essential for achieving a predictive understanding of binding interface energetics.  Future progress will depend as much on the application of new computational approaches for dissecting protein interfaces as on expanding the empirical databank of mutagenic substitutions and their effects.  Despite current theor. shortcomings, recent methodol. advances provide efficient exptl. means of probing hot spots and enable immediate applications for hot spots in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpk1HNI-4QfrrVg90H21EOLACvtfcHk0liYcDEZ7OOtlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XhtF2it70%253D&md5=a297ae443e69c94b7cf2e342337f29ea</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2FS0959-440X%2802%2900283-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0959-440X%252802%252900283-X%26sid%3Dliteratum%253Aachs%26aulast%3DDeLano%26aufirst%3DW.%2BL.%26atitle%3DUnraveling%2520Hot%2520Spots%2520in%2520Binding%2520Interfaces%253A%2520Progress%2520and%2520Challenges%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2002%26volume%3D12%26issue%3D1%26spage%3D14%26epage%3D20%26doi%3D10.1016%2FS0959-440X%2802%2900283-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blundell, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abell, C.</span></span> <span> </span><span class="NLM_article-title">Probing Hot Spots at Protein-Ligand Binding Sites: A Fragment-Based Approach Using Biophysical Methods</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>49</i></span> (<span class="NLM_issue">16</span>),  <span class="NLM_fpage">4992</span>– <span class="NLM_lpage">5000</span>, <span class="refDoi"> DOI: 10.1021/jm060490r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm060490r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=4992-5000&issue=16&author=A.+Ciulliauthor=G.+Williamsauthor=A.+G.+Smithauthor=T.+L.+Blundellauthor=C.+Abell&title=Probing+Hot+Spots+at+Protein-Ligand+Binding+Sites%3A+A+Fragment-Based+Approach+Using+Biophysical+Methods&doi=10.1021%2Fjm060490r"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fjm060490r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm060490r%26sid%3Dliteratum%253Aachs%26aulast%3DCiulli%26aufirst%3DA.%26aulast%3DWilliams%26aufirst%3DG.%26aulast%3DSmith%26aufirst%3DA.%2BG.%26aulast%3DBlundell%26aufirst%3DT.%2BL.%26aulast%3DAbell%26aufirst%3DC.%26atitle%3DProbing%2520Hot%2520Spots%2520at%2520Protein-Ligand%2520Binding%2520Sites%253A%2520A%2520Fragment-Based%2520Approach%2520Using%2520Biophysical%2520Methods%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26issue%3D16%26spage%3D4992%26epage%3D5000%26doi%3D10.1021%2Fjm060490r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roos, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viklund, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meuller, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaspersson, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svensson, M.</span></span> <span> </span><span class="NLM_article-title">Potency Prediction of β-Secretase (BACE-1) Inhibitors Using Density Functional Methods</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>54</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">818</span>– <span class="NLM_lpage">825</span>, <span class="refDoi"> DOI: 10.1021/ci400374z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci400374z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFKmurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=818-825&issue=3&author=K.+Roosauthor=J.+Viklundauthor=J.+Meullerauthor=K.+Kasperssonauthor=M.+Svensson&title=Potency+Prediction+of+%CE%B2-Secretase+%28BACE-1%29+Inhibitors+Using+Density+Functional+Methods&doi=10.1021%2Fci400374z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Potency Prediction of β-Secretase (BACE-1) Inhibitors Using Density Functional Methods</span></div><div class="casAuthors">Roos, Katarina; Viklund, Jenny; Meuller, Johan; Kaspersson, Karin; Svensson, Mats</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">818-825</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Scoring potency is a main challenge for structure based drug design.  Inductive effects of subtle variations in the ligand are not possible to accurately predict by classical computational chem. methods.  In this study, the problem of predicting potency of ligands with electronic variations participating in key interactions with the protein was addressed.  The potency was predicted for a large set of cyclic amidine and guanidine cores extd. from β-secretase (BACE-1) inhibitors.  All cores were of similar size and had equal interaction motifs but were diverse with respect to electronic substitutions.  A d. functional theory approach, in combination with a representation of the active site of a protein using only key residues, was shown to be predictive.  This computational approach was used to guide and support drug design, within the time frame of a normal drug discovery design cycle.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAMRpKC-p8rLVg90H21EOLACvtfcHk0liYcDEZ7OOtlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFKmurk%253D&md5=2b33ce3d50a7234f805723093936a681</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fci400374z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci400374z%26sid%3Dliteratum%253Aachs%26aulast%3DRoos%26aufirst%3DK.%26aulast%3DViklund%26aufirst%3DJ.%26aulast%3DMeuller%26aufirst%3DJ.%26aulast%3DKaspersson%26aufirst%3DK.%26aulast%3DSvensson%26aufirst%3DM.%26atitle%3DPotency%2520Prediction%2520of%2520%25CE%25B2-Secretase%2520%2528BACE-1%2529%2520Inhibitors%2520Using%2520Density%2520Functional%2520Methods%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26issue%3D3%26spage%3D818%26epage%3D825%26doi%3D10.1021%2Fci400374z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zgair, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taha, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kagan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yun, H.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gershkovich, P.</span></span> <span> </span><span class="NLM_article-title">Quantitative Analysis of Lab-to-Lab Variability in Caco-2 Permeability Assays</span>. <i>Eur. J. Pharm. Biopharm.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">38</span>– <span class="NLM_lpage">42</span>, <span class="refDoi"> DOI: 10.1016/j.ejpb.2016.12.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1016%2Fj.ejpb.2016.12.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=28088633" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1entrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=38-42&author=J.+B.+Leeauthor=A.+Zgairauthor=D.+A.+Tahaauthor=X.+Zangauthor=L.+Kaganauthor=T.+H.+Kimauthor=M.+G.+Kimauthor=H.-Y.+Yunauthor=P.+M.+Fischerauthor=P.+Gershkovich&title=Quantitative+Analysis+of+Lab-to-Lab+Variability+in+Caco-2+Permeability+Assays&doi=10.1016%2Fj.ejpb.2016.12.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative analysis of lab-to-lab variability in Caco-2 permeability assays</span></div><div class="casAuthors">Lee, Jong Bong; Zgair, Atheer; Taha, Dhiaa A.; Zang, Xiaowei; Kagan, Leonid; Kim, Tae Hwan; Kim, Min Gi; Yun, Hwi-yeol; Fischer, Peter M.; Gershkovich, Pavel</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Pharmaceutics and Biopharmaceutics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">38-42</span>CODEN:
                <span class="NLM_cas:coden">EJPBEL</span>;
        ISSN:<span class="NLM_cas:issn">0939-6411</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">In this study, Caco-2 permeability results from different labs. were compared.  Six different sets of apparent permeability coeff. (Papp) values reported in the literature were compared to exptl. Papp obtained in our lab.  The differences were assessed by detg. the root mean square error (RMSE) values between the datasets, which reached levels as high as 0.581 for the training set compds., i.e. ten compds. with known effective human permeability (Peff).  The consequences of these differences in Papp for prediction of oral drug absorption were demonstrated by introducing the Papp into the absorption and pharmacokinetics simulation software application GastroPlus for prediction of the fraction absorbed (Fa) in humans using calibrated "user-defined permeability models".  The RMSE were calcd. to assess the differences between the simulated Fa and exptl. values reported in the literature.  The RMSE for Fa simulated with the permeability model calibrated using exptl. Papp from our lab. was 0.128.  When the calibration was performed using Papp from literature datasets, the RMSE values for Fa were higher in all cases except one.  This study shows quant. lab-to-lab variability of Caco-2 permeability results and the potential consequences this can have in the use of these results for predicting intestinal absorption of drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohz3m9Q4Il07Vg90H21EOLACvtfcHk0liYcDEZ7OOtlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1entrw%253D&md5=9d823fab5060b4711261ff9e48bf4850</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.ejpb.2016.12.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejpb.2016.12.027%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DJ.%2BB.%26aulast%3DZgair%26aufirst%3DA.%26aulast%3DTaha%26aufirst%3DD.%2BA.%26aulast%3DZang%26aufirst%3DX.%26aulast%3DKagan%26aufirst%3DL.%26aulast%3DKim%26aufirst%3DT.%2BH.%26aulast%3DKim%26aufirst%3DM.%2BG.%26aulast%3DYun%26aufirst%3DH.-Y.%26aulast%3DFischer%26aufirst%3DP.%2BM.%26aulast%3DGershkovich%26aufirst%3DP.%26atitle%3DQuantitative%2520Analysis%2520of%2520Lab-to-Lab%2520Variability%2520in%2520Caco-2%2520Permeability%2520Assays%26jtitle%3DEur.%2520J.%2520Pharm.%2520Biopharm.%26date%3D2017%26volume%3D114%26spage%3D38%26epage%3D42%26doi%3D10.1016%2Fj.ejpb.2016.12.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">XDS</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=XDS&doi=10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lhVMSUvef28gg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXDS%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballard, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keegan, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krissinel, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mccoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mcnicholas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potterton, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, K. S.</span></span> <span> </span><span class="NLM_article-title">Overview of the CCP4 Suite and Current Developments</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1107/S0907444910045749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=235-242&author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=R.+M.+Keeganauthor=E.+B.+Krissinelauthor=A.+G.+W.+Leslieauthor=A.+Mccoyauthor=S.+J.+Mcnicholasauthor=G.+N.+Murshudovauthor=N.+S.+Pannuauthor=E.+A.+Pottertonauthor=H.+R.+Powellauthor=R.+J.+Readauthor=A.+Vaginauthor=K.+S.+Wilson&title=Overview+of+the+CCP4+Suite+and+Current+Developments&doi=10.1107%2FS0907444910045749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0lhVMSUvef28gg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DKeegan%26aufirst%3DR.%2BM.%26aulast%3DKrissinel%26aufirst%3DE.%2BB.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26aulast%3DMccoy%26aufirst%3DA.%26aulast%3DMcnicholas%26aufirst%3DS.%2BJ.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DPotterton%26aufirst%3DE.%2BA.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520the%2520CCP4%2520Suite%2520and%2520Current%2520Developments%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D235%26epage%3D242%26doi%3D10.1107%2FS0907444910045749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phaser Crystallographic Software</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+Crystallographic+Software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0lhVMSUvef28gg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520Crystallographic%2520Software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and Development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+Development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0ljUCcKOIFYMLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520Development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span> <span> </span><span class="NLM_article-title">Refinement of Macromolecular Structures by the Maximum-Likelihood Method</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">240</span>– <span class="NLM_lpage">255</span>, <span class="refDoi"> DOI: 10.1107/S0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+Macromolecular+Structures+by+the+Maximum-Likelihood+Method&doi=10.1107%2FS0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0ljUCcKOIFYMLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520Macromolecular%2520Structures%2520by%2520the%2520Maximum-Likelihood%2520Method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255%26doi%3D10.1107%2FS0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Painter, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merritt, E. A.</span></span> <span> </span><span class="NLM_article-title">TLSMD Web Server for the Generation of Multi-Group TLS Models</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>39</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">109</span>– <span class="NLM_lpage">111</span>, <span class="refDoi"> DOI: 10.1107/S0021889805038987</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1107%2FS0021889805038987" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BD28XmvVyjsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2006&pages=109-111&issue=1&author=J.+Painterauthor=E.+A.+Merritt&title=TLSMD+Web+Server+for+the+Generation+of+Multi-Group+TLS+Models&doi=10.1107%2FS0021889805038987"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">TLSMD web server for the generation of multi-group TLS models</span></div><div class="casAuthors">Painter, Jay; Merritt, Ethan A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">109-111</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">The TLSMD web server exts. information about dynamic properties of a protein based on information derived from a single-crystal structure.  It does so by analyzing the spatial distribution of individual at. thermal parameters present in an input structural model.  The server partitions the protein structure into multiple, contiguous chain segments, each segment corresponding to one group in a multi-group description of the protein's overall dynamic motion.  For each polypeptide chain of the input protein, the anal. generates the optimal partition into two segments, three segments,... up to 20 segments.  Each such partition is optimal in the sense that it is the best approxn. of the overall spatial distribution of input thermal parameters in terms of N chain segments, each acting as a rigid group undergoing TLS (translation/libration/screw) motion.  This multi-group TLS model may be used as a starting point for further crystallog. refinement, or as the basis for analyzing inter-domain and other large-scale motions implied by the crystal structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhkwf2sopjVrVg90H21EOLACvtfcHk0ljUCcKOIFYMLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmvVyjsA%253D%253D&md5=5d16d5d6310baa5434064bdf0ec81e24</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1107%2FS0021889805038987&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889805038987%26sid%3Dliteratum%253Aachs%26aulast%3DPainter%26aufirst%3DJ.%26aulast%3DMerritt%26aufirst%3DE.%2BA.%26atitle%3DTLSMD%2520Web%2520Server%2520for%2520the%2520Generation%2520of%2520Multi-Group%2520TLS%2520Models%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2006%26volume%3D39%26issue%3D1%26spage%3D109%26epage%3D111%26doi%3D10.1107%2FS0021889805038987" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gad, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koolmeister, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jemth, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eshtad, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacques, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ström, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svensson, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lundbäck, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Einarsdottir, B. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saleh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Göktürk, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baranczewski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svensson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berntsson, R. P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafsson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strömberg, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanjiv, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacques-Cordonnier, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desroses, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustavsson, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olofsson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johansson, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Homan, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loseva, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bräutigam, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johansson, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Höglund, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hagenkort, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pham, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altun, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaugaz, F. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vikingsson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evers, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henriksson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vallin, K. S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallner, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammarström, L. G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiita, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almlöf, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalderén, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Axelsson, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Djureinovic, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puigvert, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Häggblad, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeppsson, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martens, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lundin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lundgren, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Granelli, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Artursson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nilsson, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stenmark, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scobie, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berglund, U. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helleday, T.</span></span> <span> </span><span class="NLM_article-title">MTH1 Inhibition Eradicates Cancer by Preventing Sanitation of the dNTP Pool</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>508</i></span> (<span class="NLM_issue">7495</span>),  <span class="NLM_fpage">215</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1038/nature13181</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1038%2Fnature13181" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=24695224" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtlWrsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=508&publication_year=2014&pages=215-221&issue=7495&author=H.+Gadauthor=T.+Koolmeisterauthor=A.+S.+Jemthauthor=S.+Eshtadauthor=S.+A.+Jacquesauthor=C.+E.+Str%C3%B6mauthor=L.+M.+Svenssonauthor=N.+Schultzauthor=T.+Lundb%C3%A4ckauthor=B.+O.+Einarsdottirauthor=A.+Salehauthor=C.+G%C3%B6kt%C3%BCrkauthor=P.+Baranczewskiauthor=R.+Svenssonauthor=R.+P.+A.+Berntssonauthor=R.+Gustafssonauthor=K.+Str%C3%B6mbergauthor=K.+Sanjivauthor=M.+C.+Jacques-Cordonnierauthor=M.+Desrosesauthor=A.+L.+Gustavssonauthor=R.+Olofssonauthor=F.+Johanssonauthor=E.+J.+Homanauthor=O.+Losevaauthor=L.+Br%C3%A4utigamauthor=L.+Johanssonauthor=A.+H%C3%B6glundauthor=A.+Hagenkortauthor=T.+Phamauthor=M.+Altunauthor=F.+Z.+Gaugazauthor=S.+Vikingssonauthor=B.+Eversauthor=M.+Henrikssonauthor=K.+S.+A.+Vallinauthor=O.+A.+Wallnerauthor=L.+G.+J.+Hammarstr%C3%B6mauthor=E.+Wiitaauthor=I.+Alml%C3%B6fauthor=C.+Kalder%C3%A9nauthor=H.+Axelssonauthor=T.+Djureinovicauthor=J.+C.+Puigvertauthor=M.+H%C3%A4ggbladauthor=F.+Jeppssonauthor=U.+Martensauthor=C.+Lundinauthor=B.+Lundgrenauthor=I.+Granelliauthor=A.+J.+Jensenauthor=P.+Arturssonauthor=J.+A.+Nilssonauthor=P.+Stenmarkauthor=M.+Scobieauthor=U.+W.+Berglundauthor=T.+Helleday&title=MTH1+Inhibition+Eradicates+Cancer+by+Preventing+Sanitation+of+the+dNTP+Pool&doi=10.1038%2Fnature13181"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">MTH1 inhibition eradicates cancer by preventing sanitation of the dNTP pool</span></div><div class="casAuthors">Gad, Helge; Koolmeister, Tobias; Jemth, Ann-Sofie; Eshtad, Saeed; Jacques, Sylvain A.; Stroem, Cecilia E.; Svensson, Linda M.; Schultz, Niklas; Lundbaeck, Thomas; Einarsdottir, Berglind Osk; Saleh, Aljona; Gokturk, Camilla; Baranczewski, Pawel; Svensson, Richard; Berntsson, Ronnie P.-A.; Gustafsson, Robert; Stroemberg, Kia; Sanjiv, Kumar; Jacques-Cordonnier, Marie-Caroline; Desroses, Matthieu; Gustavsson, Anna-Lena; Olofsson, Roger; Johansson, Fredrik; Homan, Evert J.; Loseva, Olga; Braeutigam, Lars; Johansson, Lars; Hoeglund, Andreas; Hagenkort, Anna; Pham, Therese; Altun, Mikael; Gaugaz, Fabienne Z.; Vikingsson, Svante; Evers, Bastiaan; Henriksson, Martin; Vallin, Karl S. A.; Wallner, Olov A.; Hammarstroem, Lars G. J.; Wiita, Elisee; Almloef, Ingrid; Kalderen, Christina; Axelsson, Hanna; Djureinovic, Tatjana; Puigvert, Jordi Carreras; Haeggblad, Maria; Jeppsson, Fredrik; Martens, Ulf; Lundin, Cecilia; Lundgren, Bo; Granelli, Ingrid; Jensen, Annika Jenmalm; Artursson, Per; Nilsson, Jonas A.; Stenmark, Pal; Scobie, Martin; Berglund, Ulrika Warpman; Helleday, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">508</span>
        (<span class="NLM_cas:issue">7495</span>),
    <span class="NLM_cas:pages">215-221</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cancers have dysfunctional redox regulation resulting in reactive oxygen species prodn., damaging both DNA and free dNTPs.  The MTH1 protein sanitizes oxidized dNTP pools to prevent incorporation of damaged bases during DNA replication.  Although MTH1 is non-essential in normal cells, the authors show that cancer cells require MTH1 activity to avoid incorporation of oxidized dNTPs, resulting in DNA damage and cell death.  The authors validate MTH1 as an anticancer target in vivo and describe small mols. TH287 (I) and TH588 (II) as first-in-class nudix hydrolase family inhibitors that potently and selectively engage and inhibit the MTH1 protein in cells.  Protein co-crystal structures demonstrate that the inhibitors bind in the active site of MTH1.  The inhibitors cause incorporation of oxidized dNTPs in cancer cells, leading to DNA damage, cytotoxicity and therapeutic responses in patient-derived mouse xenografts.  This study exemplifies the non-oncogene addiction concept for anticancer treatment and validates MTH1 as being cancer phenotypic lethal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-Xn3KhC40XbVg90H21EOLACvtfcHk0lgBBuqCmeb0jg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtlWrsLc%253D&md5=49afb4522874f4f5e216ae38d0ff7cd2</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1038%2Fnature13181&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13181%26sid%3Dliteratum%253Aachs%26aulast%3DGad%26aufirst%3DH.%26aulast%3DKoolmeister%26aufirst%3DT.%26aulast%3DJemth%26aufirst%3DA.%2BS.%26aulast%3DEshtad%26aufirst%3DS.%26aulast%3DJacques%26aufirst%3DS.%2BA.%26aulast%3DStr%25C3%25B6m%26aufirst%3DC.%2BE.%26aulast%3DSvensson%26aufirst%3DL.%2BM.%26aulast%3DSchultz%26aufirst%3DN.%26aulast%3DLundb%25C3%25A4ck%26aufirst%3DT.%26aulast%3DEinarsdottir%26aufirst%3DB.%2BO.%26aulast%3DSaleh%26aufirst%3DA.%26aulast%3DG%25C3%25B6kt%25C3%25BCrk%26aufirst%3DC.%26aulast%3DBaranczewski%26aufirst%3DP.%26aulast%3DSvensson%26aufirst%3DR.%26aulast%3DBerntsson%26aufirst%3DR.%2BP.%2BA.%26aulast%3DGustafsson%26aufirst%3DR.%26aulast%3DStr%25C3%25B6mberg%26aufirst%3DK.%26aulast%3DSanjiv%26aufirst%3DK.%26aulast%3DJacques-Cordonnier%26aufirst%3DM.%2BC.%26aulast%3DDesroses%26aufirst%3DM.%26aulast%3DGustavsson%26aufirst%3DA.%2BL.%26aulast%3DOlofsson%26aufirst%3DR.%26aulast%3DJohansson%26aufirst%3DF.%26aulast%3DHoman%26aufirst%3DE.%2BJ.%26aulast%3DLoseva%26aufirst%3DO.%26aulast%3DBr%25C3%25A4utigam%26aufirst%3DL.%26aulast%3DJohansson%26aufirst%3DL.%26aulast%3DH%25C3%25B6glund%26aufirst%3DA.%26aulast%3DHagenkort%26aufirst%3DA.%26aulast%3DPham%26aufirst%3DT.%26aulast%3DAltun%26aufirst%3DM.%26aulast%3DGaugaz%26aufirst%3DF.%2BZ.%26aulast%3DVikingsson%26aufirst%3DS.%26aulast%3DEvers%26aufirst%3DB.%26aulast%3DHenriksson%26aufirst%3DM.%26aulast%3DVallin%26aufirst%3DK.%2BS.%2BA.%26aulast%3DWallner%26aufirst%3DO.%2BA.%26aulast%3DHammarstr%25C3%25B6m%26aufirst%3DL.%2BG.%2BJ.%26aulast%3DWiita%26aufirst%3DE.%26aulast%3DAlml%25C3%25B6f%26aufirst%3DI.%26aulast%3DKalder%25C3%25A9n%26aufirst%3DC.%26aulast%3DAxelsson%26aufirst%3DH.%26aulast%3DDjureinovic%26aufirst%3DT.%26aulast%3DPuigvert%26aufirst%3DJ.%2BC.%26aulast%3DH%25C3%25A4ggblad%26aufirst%3DM.%26aulast%3DJeppsson%26aufirst%3DF.%26aulast%3DMartens%26aufirst%3DU.%26aulast%3DLundin%26aufirst%3DC.%26aulast%3DLundgren%26aufirst%3DB.%26aulast%3DGranelli%26aufirst%3DI.%26aulast%3DJensen%26aufirst%3DA.%2BJ.%26aulast%3DArtursson%26aufirst%3DP.%26aulast%3DNilsson%26aufirst%3DJ.%2BA.%26aulast%3DStenmark%26aufirst%3DP.%26aulast%3DScobie%26aufirst%3DM.%26aulast%3DBerglund%26aufirst%3DU.%2BW.%26aulast%3DHelleday%26aufirst%3DT.%26atitle%3DMTH1%2520Inhibition%2520Eradicates%2520Cancer%2520by%2520Preventing%2520Sanitation%2520of%2520the%2520dNTP%2520Pool%26jtitle%3DNature%26date%3D2014%26volume%3D508%26issue%3D7495%26spage%3D215%26epage%3D221%26doi%3D10.1038%2Fnature13181" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Houston, J. B.</span></span> <span> </span><span class="NLM_article-title">Utility of in Vitro Drug Metabolism Data in Predicting in Vivo Metabolic Clearance</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>47</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">1469</span>– <span class="NLM_lpage">1479</span>, <span class="refDoi"> DOI: 10.1016/0006-2952(94)90520-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=10.1016%2F0006-2952%2894%2990520-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=8185657" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADyaK2cXkt1Kju7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=1994&pages=1469-1479&issue=9&author=J.+B.+Houston&title=Utility+of+in+Vitro+Drug+Metabolism+Data+in+Predicting+in+Vivo+Metabolic+Clearance&doi=10.1016%2F0006-2952%2894%2990520-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Utility of in vitro drug metabolism data in predicting in vivo metabolic clearance</span></div><div class="casAuthors">Houston, J. Brian</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1469-79</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    </div><div class="casAbstract">A review with 54 refs.  The aim of the commentary is to present a strategy to achieve the prediction of in vivo clearance from in vitro data based on the current understanding of metabolite kinetics in vitro and in vivo.  Hepatic microsomes and hepatocyte suspension are the main in vitro system addressed.  The theor. basis of the strategy is outlined and its general applicability evaluated, using a data base of 25 drugs drawn from the literature.  Finally, the prospective value of the approach is discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFHqQ8gCJ6lrVg90H21EOLACvtfcHk0lgBBuqCmeb0jg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXkt1Kju7c%253D&md5=a6d0abdbd2f0b238a6a01ba63ee67739</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1016%2F0006-2952%2894%2990520-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0006-2952%252894%252990520-7%26sid%3Dliteratum%253Aachs%26aulast%3DHouston%26aufirst%3DJ.%2BB.%26atitle%3DUtility%2520of%2520in%2520Vitro%2520Drug%2520Metabolism%2520Data%2520in%2520Predicting%2520in%2520Vivo%2520Metabolic%2520Clearance%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D1994%26volume%3D47%26issue%3D9%26spage%3D1469%26epage%3D1479%26doi%3D10.1016%2F0006-2952%2894%2990520-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baranczewski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stańczak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundberg, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svensson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jansson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garberg, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Postlind, H.</span></span> <span> </span><span class="NLM_article-title">Introduction to in Vitro Estimation of Metabolic Stability and Drug Interactions of New Chemical Entities in Drug Discovery and Development</span>. <i>Pharmacol. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>58</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">453</span>– <span class="NLM_lpage">472</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=16963792" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtlarsbnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2006&pages=453-472&issue=4&author=P.+Baranczewskiauthor=A.+Sta%C5%84czakauthor=K.+Sundbergauthor=R.+Svenssonauthor=A.+Wallinauthor=J.+Janssonauthor=P.+Garbergauthor=H.+Postlind&title=Introduction+to+in+Vitro+Estimation+of+Metabolic+Stability+and+Drug+Interactions+of+New+Chemical+Entities+in+Drug+Discovery+and+Development"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Introduction to in vitro estimation of metabolic stability and drug interactions of new chemical entities in drug discovery and development</span></div><div class="casAuthors">Baranczewski, Pawel; Stanczak, Andrzej; Sundberg, Kathrin; Svensson, Richard; Wallin, Aasa; Jansson, Jenny; Garberg, Per; Postlind, Hans</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Reports</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">453-472</span>CODEN:
                <span class="NLM_cas:coden">PRHEDU</span>;
        ISSN:<span class="NLM_cas:issn">1734-1140</span>.
    
            (<span class="NLM_cas:orgname">Polish Academy of Sciences, Institute of Pharmacology</span>)
        </div><div class="casAbstract">A review.  Detn. of metabolic properties of a new chem. entity (NCE) is one of the most important steps during the drug discovery and development process.  Nowadays, in vitro methods are used for early estn. and prediction of in vivo metab. of NCEs.  Using in vitro methods, it is possible to det. the metabolic stability of NCEs as well as the risk for drug-drug interactions (DDIs) related to inhibition and induction of drug metabolic enzymes.  Metabolic stability is defined as the susceptibility of a chem. compd. to biotransformation, and is expressed as in vitro half-life (t1/2) and intrinsic clearance (CLint).  Based on these values, in vivo pharmacokinetic parameters such as bioavailability and in vivo half-life can be calcd.  The drug metabolic enzymes possess broad substrate specificity and can metabolize multiple compds.  Therefore, the risk for metab.-based DDIs is always a potential problem during the drug development process.  For this reason, inhibition and induction in vitro screens are used early, before selection of a candidate drug (CD), to est. the risk for clin. significant DDIs.  At present, most pharmaceutical companies perform in vitro drug metab. studies together with in silica prediction software and automated high-throughput screens (HTS).  Available data suggest that in vitro methods are useful tools for identification and elimination of NCEs with unappreciated metabolic properties.  However, the quant. output of the methods has to be improved.  The aim of this review is to highlight the practical and theor. basis of the in vitro metabolic methods and the recent progress in the development of these assays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeZpDZgTbV2rVg90H21EOLACvtfcHk0lgBBuqCmeb0jg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtlarsbnM&md5=fe9b22fe952f43aa6258cbf7d22afda7</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBaranczewski%26aufirst%3DP.%26aulast%3DSta%25C5%2584czak%26aufirst%3DA.%26aulast%3DSundberg%26aufirst%3DK.%26aulast%3DSvensson%26aufirst%3DR.%26aulast%3DWallin%26aufirst%3DA.%26aulast%3DJansson%26aufirst%3DJ.%26aulast%3DGarberg%26aufirst%3DP.%26aulast%3DPostlind%26aufirst%3DH.%26atitle%3DIntroduction%2520to%2520in%2520Vitro%2520Estimation%2520of%2520Metabolic%2520Stability%2520and%2520Drug%2520Interactions%2520of%2520New%2520Chemical%2520Entities%2520in%2520Drug%2520Discovery%2520and%2520Development%26jtitle%3DPharmacol.%2520Rep.%26date%3D2006%26volume%3D58%26issue%3D4%26spage%3D453%26epage%3D472" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ZR0" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ZR0','PDB','3ZR0'); return false;">PDB: 3ZR0</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4C9X" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4C9X','PDB','4C9X'); return false;">PDB: 4C9X</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4N1U" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4N1U','PDB','4N1U'); return false;">PDB: 4N1U</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ANV" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ANV','PDB','5ANV'); return false;">PDB: 5ANV</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6F20" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6F20','PDB','6F20'); return false;">PDB: 6F20</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6F1X" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6F1X','PDB','6F1X'); return false;">PDB: 6F1X</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4C9W" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4C9W','PDB','4C9W'); return false;">PDB: 4C9W</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ZR1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ZR1','PDB','3ZR1'); return false;">PDB: 3ZR1</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3Q93" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3Q93','PDB','3Q93'); return false;">PDB: 3Q93</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4N1T" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4N1T','PDB','4N1T'); return false;">PDB: 4N1T</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4CX9" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4CX9','PDB','4CX9'); return false;">PDB: 4CX9</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6F23" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6F23','PDB','6F23'); return false;">PDB: 6F23</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6F22" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6F22','PDB','6F22'); return false;">PDB: 6F22</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5NHY" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5NHY','PDB','5NHY'); return false;">PDB: 5NHY</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i63"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b01884">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_16817"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.7b01884">10.1021/acs.jmedchem.7b01884</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Selectivity data of <b>10</b> and <b>24</b> in a 97-membered kinome panel (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b01884/suppl_file/jm7b01884_si_001.csv">CSV</a>)</p></li><li><p class="inline">Selectivity data of <b>24</b> in a 215-membered ATP-ase panel (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b01884/suppl_file/jm7b01884_si_002.xlsx">XLSX</a>)</p></li><li><p class="inline">Selectivity data of <b>32</b> (BAY-707) and <b>35</b> (BAY-604) in a 20-membered NUDIX panel (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b01884/suppl_file/jm7b01884_si_003.csv">CSV</a>)</p></li><li><p class="inline">Table of SMILES and data for all compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b01884/suppl_file/jm7b01884_si_004.csv">CSV</a>)</p></li><li><p class="inline">Data collection and refinement statistics, data used for computational modeling, and methods for NUDIX selectivity assessment (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b01884/suppl_file/jm7b01884_si_005.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01884/suppl_file/jm7b01884_si_001.csv">jm7b01884_si_001.csv (6.85 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01884/suppl_file/jm7b01884_si_002.xlsx">jm7b01884_si_002.xlsx (168.34 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01884/suppl_file/jm7b01884_si_003.csv">jm7b01884_si_003.csv (1.21 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01884/suppl_file/jm7b01884_si_004.csv">jm7b01884_si_004.csv (2.82 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01884/suppl_file/jm7b01884_si_005.pdf">jm7b01884_si_005.pdf (1.43 MB)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Accession Codes in the PDB for human MTH1 with bound ligands: compound <b>1</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6F20">6F20</a>), compound 7 (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6F1X">6F1X</a>), compound <b>16</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6F23">6F23</a>), and compound <b>29</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6F22">6F22</a>).</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.7b01884&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2018.61.issue-6%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.7b01884%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.7b01884" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799c3142dd93cc4","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
