
<html lang="en"     class="pb-page"  data-request-id="d781856d-e2b6-4d41-af81-10cbf8dae7cf"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";article:article:10.1021/acs.jmedchem.5b01973;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2016.59.issue-6;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Isoquinoline-1,3-diones as Selective Inhibitors of Tyrosyl DNA Phosphodiesterase II (TDP2)" /></meta><meta name="dc.Creator" content="Jayakanth  Kankanala" /></meta><meta name="dc.Creator" content="Christophe  Marchand" /></meta><meta name="dc.Creator" content="Monica  Abdelmalak" /></meta><meta name="dc.Creator" content="Hideki  Aihara" /></meta><meta name="dc.Creator" content="Yves  Pommier" /></meta><meta name="dc.Creator" content="Zhengqiang  Wang" /></meta><meta name="dc.Description" content="Tyrosyl DNA phosphodiesterase II (TDP2) is a recently discovered enzyme that specifically repairs DNA damages induced by topoisomerase II (Top2) poisons and causes resistance to these drugs. Inhibi..." /></meta><meta name="Description" content="Tyrosyl DNA phosphodiesterase II (TDP2) is a recently discovered enzyme that specifically repairs DNA damages induced by topoisomerase II (Top2) poisons and causes resistance to these drugs. Inhibi..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="March 7, 2016" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.5b01973" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2016 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.5b01973" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.5b01973" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.5b01973" /></link>
        
    
    

<title>Isoquinoline-1,3-diones as Selective Inhibitors of Tyrosyl DNA Phosphodiesterase II (TDP2) | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.5b01973" /></meta><meta property="og:title" content="Isoquinoline-1,3-diones as Selective Inhibitors of Tyrosyl DNA Phosphodiesterase II (TDP2)" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01973/20160318/images/large/jm-2015-01973q_0005.jpeg" /></meta><meta property="og:description" content="Tyrosyl DNA phosphodiesterase II (TDP2) is a recently discovered enzyme that specifically repairs DNA damages induced by topoisomerase II (Top2) poisons and causes resistance to these drugs. Inhibiting TDP2 is expected to enhance the efficacy of clinically important Top2-targeting anticancer drugs. However, TDP2 as a therapeutic target remains poorly understood. We report herein the discovery of isoquinoline-1,3-dione as a viable chemotype for selectively inhibiting TDP2. The initial hit compound 43 was identified by screening our in-house collection of synthetic compounds. Further structure–activity relationship (SAR) studies identified numerous analogues inhibiting TDP2 in low micromolar range without appreciable inhibition against the homologous TDP1 at the highest testing concentration (111 μM). The best compound 64 inhibited recombinant TDP2 with an IC50 of 1.9 μM. The discovery of this chemotype may provide a platform toward understanding TDP2 as a drug target." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.5b01973"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.5b01973">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.5b01973&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.5b01973&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.5b01973&amp;href=/doi/10.1021/acs.jmedchem.5b01973" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2016</span><span class="cit-fg-volume">, 59</span><span class="cit-fg-issue">, 6</span><span class="cit-fg-pageRange">, 2734-2746</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/59/6" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.5b01948" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.5b01979" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Isoquinoline-1,3-diones as Selective Inhibitors of Tyrosyl DNA Phosphodiesterase II (TDP2)</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jayakanth++Kankanala">Jayakanth Kankanala</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Christophe++Marchand">Christophe Marchand</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Monica++Abdelmalak">Monica Abdelmalak</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Hideki++Aihara">Hideki Aihara</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Yves++Pommier">Yves Pommier</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Zhengqiang++Wang">Zhengqiang Wang</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Center for Drug Design, Academic Health Center, University of Minnesota, Minneapolis, Minnesota 55455, United States</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Developmental Therapeutics Branch, Center for Cancer Research, National Cancer Institute, National Institutes of Health, Bethesda, Maryland 20892, United States</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">§</span> <span class="aff-text">Department of Biochemistry, Molecular Biology and Biophysics, University of Minnesota, Minneapolis, Minnesota 55455, United States</span></div><div class="corresp-info"><strong>*</strong>Phone: 612-626-7025. Fax: 612-625-8154. E-mail: <a href="/cdn-cgi/l/email-protection#82f5e3ece5fab6b5b0c2f7efecace7e6f7"><span class="__cf_email__" data-cfemail="88ffe9e6eff0bcbfbac8fde5e6a6edecfd">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.5b01973&amp;href=/doi/10.1021%2Facs.jmedchem.5b01973" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2016</span></span><span class="cit-volume">, 59</span><span class="cit-issue">, 6</span><span class="cit-pageRange">, 2734–2746</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">February 24, 2016</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>21 December 2015</li><li><span class="item_label"><b>Published</b> online</span>7 March 2016</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 24 March 2016</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.5b01973" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b01973</a></div><div class="article_header-article-copyright"><strong>Copyright © 2016 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D2734%26pageCount%3D13%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DJayakanth%2BKankanala%252C%2BChristophe%2BMarchand%252C%2BMonica%2BAbdelmalak%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D59%26issueNum%3D6%26contentID%3Dacs.jmedchem.5b01973%26title%3DIsoquinoline-1%252C3-diones%2Bas%2BSelective%2BInhibitors%2Bof%2BTyrosyl%2BDNA%2BPhosphodiesterase%2BII%2B%2528TDP2%2529%26numPages%3D13%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D2746%26publicationDate%3DMarch%2B2016">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.5b01973"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1798</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">31</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.5b01973" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Isoquinoline-1,3-diones as Selective Inhibitors of Tyrosyl DNA Phosphodiesterase II (TDP2)&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Jayakanth&quot;,&quot;last_name&quot;:&quot;Kankanala&quot;},{&quot;first_name&quot;:&quot;Christophe&quot;,&quot;last_name&quot;:&quot;Marchand&quot;},{&quot;first_name&quot;:&quot;Monica&quot;,&quot;last_name&quot;:&quot;Abdelmalak&quot;},{&quot;first_name&quot;:&quot;Hideki&quot;,&quot;last_name&quot;:&quot;Aihara&quot;},{&quot;first_name&quot;:&quot;Yves&quot;,&quot;last_name&quot;:&quot;Pommier&quot;},{&quot;first_name&quot;:&quot;Zhengqiang&quot;,&quot;last_name&quot;:&quot;Wang&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2016&quot;,&quot;month&quot;:&quot;03&quot;,&quot;day&quot;:&quot;07&quot;,&quot;issue&quot;:&quot;6&quot;,&quot;volume&quot;:&quot;59&quot;,&quot;pages&quot;:&quot;2734-2746&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.5b01973&quot;},&quot;abstract&quot;:&quot;Tyrosyl DNA phosphodiesterase II (TDP2) is a recently discovered enzyme that specifically repairs DNA damages induced by topoisomerase II (Top2) poisons and causes resistance to these drugs. Inhibiting TDP2 is expected to enhance the efficacy of clinically important Top2-targeting anticancer drugs. However, TDP2 as a therapeutic target remains poorly understood. We report herein the discovery of isoquinoline-1,3-dione as a viable chemotype for selectively inhibiting TDP2. The initial hit compound 43 was identified by screening our in-house collection of synthetic compounds. Further structure–activity relationship (SAR) studies identified numerous analogues inhibiting TDP2 in low micromolar range without appreciable inhibition against the homologous TDP1 at the highest testing concentration (111 μM). The best compound 64 inhibited recombinant TDP2 with an IC50 of 1.9 μM. The discovery of this chemotype may provide a platform toward understanding TDP2 as a drug target.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01973&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01973" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01973&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01973" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01973&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01973" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.5b01973&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01973&amp;href=/doi/10.1021/acs.jmedchem.5b01973" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.5b01973" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.5b01973" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b01973&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.5b01973%26sid%3Dliteratum%253Aachs%26pmid%3D26910725%26genre%3Darticle%26aulast%3DKankanala%26date%3D2016%26atitle%3DIsoquinoline-1%252C3-diones%2Bas%2BSelective%2BInhibitors%2Bof%2BTyrosyl%2BDNA%2BPhosphodiesterase%2BII%2B%2528TDP2%2529%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D59%26issue%3D6%26spage%3D2734%26epage%3D2746%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=291465" title="Structure activity relationship">Structure activity relationship</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/59/6" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/jmcmar.2016.59.issue-6/20160324/jmcmar.2016.59.issue-6.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01973/20160318/images/medium/jm-2015-01973q_0005.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01973/20160318/images/large/jm-2015-01973q_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01973&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Tyrosyl DNA phosphodiesterase II (TDP2) is a recently discovered enzyme that specifically repairs DNA damages induced by topoisomerase II (Top2) poisons and causes resistance to these drugs. Inhibiting TDP2 is expected to enhance the efficacy of clinically important Top2-targeting anticancer drugs. However, TDP2 as a therapeutic target remains poorly understood. We report herein the discovery of isoquinoline-1,3-dione as a viable chemotype for selectively inhibiting TDP2. The initial hit compound <b>43</b> was identified by screening our in-house collection of synthetic compounds. Further structure–activity relationship (SAR) studies identified numerous analogues inhibiting TDP2 in low micromolar range without appreciable inhibition against the homologous TDP1 at the highest testing concentration (111 μM). The best compound <b>64</b> inhibited recombinant TDP2 with an IC<sub>50</sub> of 1.9 μM. The discovery of this chemotype may provide a platform toward understanding TDP2 as a drug target.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_03786" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_03786" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Many cancer chemotherapies work by inducing excessive DNA damage that causes cell death. The survival of damaged cancer cells hinges on the repair of these DNA lesions through various pathways which ultimately cause therapeutic resistance. Therefore, targeting DNA repair pathways can enhance the efficacy of DNA-damaging anticancer drugs.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Among DNA damaging drugs are Top2 poisons, such as etoposide (ETP), teniposide, doxorubicin, epirubicin, and idarubicin, which are widely used for treating a broad spectrum of cancers.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Mechanistically, Top2 (<b>1</b>, <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>) relaxes DNA torsional strains by cutting DNA substrate (<b>2</b>), using its tyrosine residue to form the covalent Top2 cleavage complex  (<b>4</b>) along with the released 3′ DNA-end (<b>3</b>) (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>). The DNA break is normally resealed when the 3′-end (<b>3</b>) attacks back to resolve Top2 cleavage complex , which allows the continuation of functional DNA transcription and replication. However, the dynamic equilibrium between DNA and Top2 cleavage complex  can shift toward the accumulation of abortive Top2 cleavage complex (<b>5</b>) in which DNA breaks fail to rejoin. Clinical Top2 poisons work by this exact mechanism as they bind to and stabilize the Top2 cleavage complex to prevent religation (<b>5</b>, <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>). TDP2 is the only known human enzyme that can cleave the unique 5′ phosphotyrosyl covalent bond featured in abortive Top2 cleavage complex,<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> demonstrating that TDP2 plays a critical role in the repair of Top2-mediated DNA damage and in resistance development to clinical Top2 poisons. This is supported by observations both in cultured cells and animal models that lack of TDP2 leads to enhanced sensitivity to Top2-induced DNA breaks caused by Top2 poisons.<a onclick="showRef(event, 'ref3 ref4 ref5 ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref3 ref4 ref5 ref6 ref7">(3-7)</a> Importantly, up-regulation of TDP2 transcription through a gain-of-function p53 mutation has indeed been linked to etoposide resistance in human lung cancer.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> These observations provide strong rationales for developing TDP2 inhibitors to sensitize cancer cells toward clinical Top2 poisons.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01973/20160318/images/medium/jm-2015-01973q_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01973/20160318/images/large/jm-2015-01973q_0007.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. TDP2 repairs abortive Top2–DNA cleavage complex: <b>2</b>, DNA substrate; <b>4</b>, transient Top2–DNA complex; <b>5</b>, Top2–DNA complex stabilized by a Top2 poison; cyan, Top2; green, TDP2; yellow, Top2 poison.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01973/20160318/images/large/jm-2015-01973q_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01973&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In addition, recent studies have also linked the unique function of TDP2 to the replication of picornaviruses<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> and particularly hepatitis B virus (HBV).<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> HBV chronically infects over 350 million people globally. Current antivirals against HBV can not eradicate the virus to achieve disease cure. This is mainly due to the persistence of viral covalently closed circular DNA (cccDNA)<a onclick="showRef(event, 'ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref11 ref12">(11, 12)</a> produced from a precursor relaxed circular DNA (RC-DNA) during a key step of HBV replication. A recent report<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> found that this conversion is critically dependent on cellular TDP2 for the cleavage of a protein–DNA adduct similar to the Top2 cleavage complex. Inhibiting cellular TDP2 thus may lead to the depletion of HBV reservoir cccDNA and may provide a path toward viral eradication, although the role of TDP2 in HBV cccDNA formation in vivo remains debatable.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a></div><div class="NLM_p">Since the discovery of TDP2 in 2009,<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> efforts in biochemistry and crystallography have generated critical knowledge to allow basic understanding on TDP2 active site structure, mode of substrate recognition, enzyme kinetics, and mechanism of catalysis.<a onclick="showRef(event, 'ref14 ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref14 ref15 ref16">(14-16)</a> These studies support either a one- or two-metal catalytic mechanism characteristic of the exonuclease-endonuclease-phosphatase (EEP) nuclease superfamily. On the basis of these mechanisms, TDP2 repairs abortive Top2 cleavage complex through a hydrolytic reaction promoted by either one or two Mg<sup>2+</sup> ion(s) and a few key residues at the active site. Particularly important to catalysis are residues D262, E152, H351, and N120, which are highly conserved across species. Knowledge on substrate binding and mechanism of catalysis could assist inhibitor discovery, particularly the structural and biochemical kinship of TDP2 to another DNA repair enzyme, human apurinic/apyrimidinic endonuclease 1 (hAPE1),<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> could allow ligand-based inhibitor design. However, full-fledged medicinal chemistry efforts targeting TDP2 require good lead compounds and the understanding of their inhibition mechanisms, both of which are still lacking. As a result, TDP2 inhibitors remain largely unexplored. Toxoflavins (<b>6</b>) and deazaflavins (<b>7</b>) (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>a) identified through a high-throughput screening (HTS) are the only known inhibitor scaffolds of TDP2.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> While biochemically inhibiting TDP2, these scaffolds are flawed as drug candidates because <b>6</b> poses a redox liability and shows poor in vitro pharmacokinetics (PK) profiles due to its inhibition of metabolic enzyme CYP450, whereas <b>7</b> lacks cell permeability.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> Therefore, there is a pressing need to identify more drug-like TDP2 inhibitors that can also be used as tool compounds toward understanding the mechanism of TDP2 inhibition.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01973/20160318/images/medium/jm-2015-01973q_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01973/20160318/images/large/jm-2015-01973q_0008.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Chemotypes of interest. (a) Reported inhibitor types of TDP2; (b) in-house synthetic chemotypes screened against TDP2.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01973/20160318/images/large/jm-2015-01973q_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01973&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_27134" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_27134" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Hit Identification</h3><div class="NLM_p">Our efforts began with screening a carefully curated library comprising our in-house synthetic compounds. The screen used a biochemical assay with recombinant human TDP2.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> The library comprises compounds of various chemotypes (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>b), many featuring a metal chelating functionality, such as <b>8</b>–<b>9</b>, <b>12</b>–<b>14</b>, <b>17</b>, and <b>19b</b> (X = OH). The rationale is that the catalysis of TDP2 requires at least one Mg<sup>2+</sup>. From this initial screening, one analogue of the isoquinoline-1,3-dione chemotype (<b>19a</b>), namely 6-furanoisoquinoline-1,3-dione (<b>43</b>, <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>a) was found to selectively inhibit TDP2 (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>c). Dose–response testing revealed that the inhibitory IC<sub>50</sub> for <b>43</b> was 10 μM (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>d). This potency was confirmed through a secondary assay using a more robust assay where endogenous TDP2 from whole cell extracts was used (WCE, IC<sub>50</sub> = 12 μM, <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>b). Remarkably, none of any other screened chemotypes showed significant TDP2 inhibition, including the highly homologous <b>19b</b> where the only structural difference is the presence of the N-2 OH group. That the 2-hydroxyisoquinoline-1,3-dione (HID) analogue <b>46</b> did not inhibit TDP2 at concentrations up to 111 μM was remarkable. It implies that unlike the inhibition of some other Mg<sup>2+</sup> dependent DNA processing enzymes, TDP2 inhibition may not require a chelating functionality. On the other hand, counter assays against the homologous TDP1 showed no inhibition by <b>43</b> (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>b), strongly suggesting that the observed TDP2 inhibition was highly specific. In addition, the small size (mw = 227) and the lack of apparent PK liabilities render <b>43</b> an attractive hit, which is amenable to further SAR.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01973/20160318/images/medium/jm-2015-01973q_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01973/20160318/images/large/jm-2015-01973q_0009.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. SAR and selective TDP2 inhibition of the isoquinolinedione chemotype. (a) Structure of isoquinoline-1,3-dione <b>43</b> and HID <b>46</b>. (b) Inhibition profile: <b>43</b> selectively inhibits TDP2 in assays using either recombinant (Rec) or endogenous TDP2 from whole cell extracts (WCE), while <b>46</b> was inactive in either condition. (c) Gel images of dose response testing of <b>43</b> in Rec TDP2 biochemical assays. Conc (μM): 1.4, 4.1, 12.3, 37, 111. (d) Dose–response curve for <b>43</b> in Rec TDP2 assay expressed as mean ± SD from at least three independent experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01973/20160318/images/large/jm-2015-01973q_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01973&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Chemistry</h3><div class="NLM_p">The SAR of compound <b>43</b> concerns primarily three distinct series of compounds: (1) isoquinoline-1,3-diones with a simple functional group (Br, I, CO<sub>2</sub>H, CF<sub>3</sub>, NO<sub>2</sub>) around the left ring (<b>33</b>–<b>42</b>, <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>), (2) isoquinoline-1,3-diones with an aromatic ring substituted around the left ring (<b>43</b>–<b>88</b>, <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>), and (3) scaffold analogues (<b>89</b>–<b>96</b>, <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). Synthetically, all isoquinoline-1,3-dione analogues of the first two series (<b>43</b>–<b>88</b>) were accessed by either of the two routes delineated in <a class="ref internalNav" href="#sch1" aria-label="Schemes 1">Schemes 1</a>–<a class="ref internalNav" href="#sch2" aria-label="2">2</a>. The first approach (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>) was adapted based on our previously reported route for the synthesis of HID.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> This approach features key iodo homophthalic acid intermediates (<b>20</b>) which were synthesized using a Hurtley reaction.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> The structural diversity for analogues of series two was subsequently introduced via a Suzuki coupling reaction to afford intermediates <b>21</b>, which were condensed with urea in 1,2-dichlorobenzene to yield desired isoquinoline-1,3-dione analogues (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>).<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a></div><figure id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01973/20160318/images/medium/jm-2015-01973q_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01973/20160318/images/large/jm-2015-01973q_0011.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 1. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01973/20160318/images/large/jm-2015-01973q_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01973&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) arylboronic acid, K<sub>2</sub>CO<sub>3</sub>, Pd(PPh<sub>3</sub>)<sub>4</sub>, EtOH/H<sub>2</sub>O (1:1), 150 °C, 30 min, MW, 60–85%; (b) urea, 1,2-dichlorobenzene, 170 °C, 45 min, MW, 55–80%.</p></p></figure><div class="NLM_p">To limit the usage of 1,2-dichlorobenzene and to produce a range of substituted isoquinoline-1,3-diones, an alternative method was employed (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). This method began with the S<sub>N</sub>Ar reaction of substituted-2-fluorobenzonitrile (<b>22</b>) with methyl cyanoacetate at 90 °C to produce the ester intermediate (<b>23</b>), which upon heating in a mixture of DMSO/H<sub>2</sub>O (9:1) at 120 °C resulted in the smooth decarboxylation to furnish the halogen (I or Br) substituted dinitrile compounds (<b>24</b>). At this stage, structural diversity was introduced via either a Suzuki coupling to afford various aryl substituted dinitrile intermediates (<b>25</b>) or a palladium assisted arylation and carbonyl insertion to produce benzoyl substituted dinitrile compounds (<b>26</b>). Treatment of dinitrile intermediates <b>24</b>–<b>26</b> with conc HCl at 70 °C afforded the desired cyclized products (<b>33</b>–<b>88</b>, <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>a). The sulfonamide (<b>62</b>–<b>64</b>) or acrylamide (<b>65</b>–<b>66</b>) analogues were synthesized by derivatizing the aniline analogues (<b>59</b>–<b>61</b>) with sulfonyl or acryloyl chloride (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>b). The benzothiazole derivative (<b>84</b>) was synthesized using a T3P mediated coupling of <b>39</b> with 2-aminothiophenol (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>c).</div><figure id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01973/20160318/images/medium/jm-2015-01973q_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01973/20160318/images/large/jm-2015-01973q_0001.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 2. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01973/20160318/images/large/jm-2015-01973q_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01973&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) methyl cyanoacetate, NaH, DMSO, 90 °C; (b) DMSO/H<sub>2</sub>O (9:1), 120 °C, 16 h, 70–85%; (c) arylboronic acid, K<sub>2</sub>CO<sub>3</sub>, Pd(PPh<sub>3</sub>)<sub>4</sub>, DME/H<sub>2</sub>O (4:1), 110 °C, 40 min, MW, 55–87%; (d) Pd(OAc)<sub>2</sub>, Mo(CO)<sub>6</sub>, K<sub>2</sub>CO<sub>3</sub>, anisole (0.2 M), 140 °C, 30 min, MW, 50–62%; (e) conc HCl, 70 °C, 4 h, 60–90%; (f) sulfonyl or acryloyl chloride, pyridine, dioxane, rt; 65–79%; (g) 2-aminothiophenol, T3P (50% in EtOAc), DIPEA, 100 °C, 30 min, MW, 69%.</p></p></figure><div class="NLM_p">The methods for synthesizing scaffold analogues are depicted in <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>. The synthesis of quinazoline-2,4-dione (<b>90</b>) involved a Suzuki coupling reaction of the starting 4-bromomethyl anthranilate <b>27</b>. The resulting intermediate <b>28</b> was then converted to the desired cyclized product <b>90</b> via the treatment with potassium cyanate and the subsequent saponification (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>a).<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> The 1,8-naphthalimide derivatives (<b>91</b>–<b>92</b>) were synthesized by the condensation of 1,8-naphthalic anhydrides (<b>29</b>–<b>30</b>) with ammonium hydroxide (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>b).<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> The 5,6-fused heterocyclic core was obtained by treating 2-amino-1-propene-1,1,3-tricarbonitrile (<b>31</b>) with various substituted hydrazine in refluxing ethanol to yield dinitrile compounds (<b>32</b>), which were then cyclized with the aid of acid to furnish compounds <b>93</b>–<b>95</b>. Compound <b>95</b> was acetylated to produce <b>96</b> (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>c).</div><figure id="sch3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01973/20160318/images/medium/jm-2015-01973q_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01973/20160318/images/large/jm-2015-01973q_0002.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 3. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01973/20160318/images/large/jm-2015-01973q_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01973&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) phenylboronic acid, K<sub>2</sub>CO<sub>3</sub>, Pd(PPh<sub>3</sub>)<sub>4</sub>, DME/H<sub>2</sub>O (4:1), 110 °C, 45 min, MW, 50%; (b) (i) KOCN, toluene, rt, (ii) NaOH, EtOH, reflux, 68%; (c) NH<sub>4</sub>OH, 40 °C, 4 h, 70–90%; (d) 2-(tributylstannyl)furan, Pd(PPh<sub>3</sub>)<sub>4</sub>, toluene, 150 °C, 30 min, 72%; (e) R-NHNH<sub>2</sub>, EtOH, reflux, 70–95%; (f) conc HCl, 70 °C, 4 h, 65–85%; (g) acetyl chloride, AcOH, 70 °C, 62%.</p></p></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Biology</h3><div class="NLM_p">All final compounds <b>33</b>–<b>96</b> were evaluated biochemically for inhibition of human TDP2. The primary assay used recombinant TDP2, and all compounds were tested in dose–response fashion at concentrations up to 111 μM. A counter assay was also run with each compound against the homologous TDP1, and none of the tested compounds showed significant inhibition at the highest concentration (111 μM). The testing results are summarized in <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>. The most striking SAR trend was that while the vast majority of the quinolone-1,3-dione analogues (<b>33</b>–<b>89</b>) potently inhibited recombinant human TDP2 in the low micromolar range, none of the scaffold analogues (<b>90</b>–<b>96</b>) showed considerable TDP2 inhibition in the same assay. This observation strongly validates quinolone-1,3-dione as an important TDP2 inhibitor type. In addition, the dramatic OH effect at the N-2 position initially observed with compound <b>46</b> was confirmed with an additional pair of analogues (<b>44</b> and <b>47</b>). Consistent with the SAR trend, the 2-NH analogue <b>44</b> inhibited TDP2 with an IC<sub>50</sub> of 13 μM, whereas the 2-NOH analogue <b>47</b> was completely inactive (IC<sub>50</sub> > 111 μM). This confirmed SAR trend suggests that compounds with a chelating triad, such as the HID chemotype, should not be considered as favorable TDP2 inhibitor types. The third major SAR observation concerns the substitution position on the left ring of the isoquinoline-1,3-dione core. Notably, most of the C6 or C7 substituted analogues potently inhibited recombinant TDP2, whereas C8 analogues (<b>53</b> and <b>57</b>) showed substantially reduced inhibitory activity (compared to <b>51</b>–<b>52</b> and <b>55</b>–<b>56</b>), and more dramatically, C5 substituted analogues (<b>34</b>, <b>45</b>, and <b>58</b>, <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>) were all inactive regardless of the nature of the substituent.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Biochemical Inhibition of Synthetic Isoquinolinedione Analogues against Recombinant hTDP2</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01973/20160318/images/medium/jm-2015-01973q_0003.gif" alt="" id="fx1" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>Table a</sup><p class="last">IC<sub>50</sub>: concentration of a compound producing 50% inhibition, expressed as mean ± standard deviation from at least three independent experiments.</p></div><div class="footnote" id="t1fn2"><sup>Table b</sup><p class="last">Value from a second independent experiment.</p></div></div><div></div></div><div class="NLM_p">To confirm the observed potency with C6 or C7 analogues, selected compounds were also tested in a secondary assay using the whole cell extracts of the TDP2 expressing cells. As shown in <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>, with the exception of <b>48</b> and <b>85</b>, all compounds tested retained activity in this assay with IC<sub>50</sub> values consistent with those obtained from the assays using recombinant TDP2. These data strongly support a genuine and selective TDP2 inhibitory profile of the C6 or C7 substituted isoqunioline-1,3-dione chemotype. In addition, a few compounds (<b>36</b>, <b>40</b>, <b>43</b>, and <b>84</b>) were also tested for sensitivity against DT40 knockout cells complemented with human hTDP2 using a previously reported assay.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> Although synergy to Top2 inhibitor ETP was not observed with any of these compounds, three of them (<b>36</b>, <b>43</b>, and <b>84</b>) exhibited potent cytotoxicity at low micromolar concentrations (CC<sub>90</sub> = 2.5—12 μM). It is likely that further improved TDP2 inhibitors will produce the desired synergism.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Inhibitory Activity of Selected Compounds against hTDP2 Whole Cell Extract (WCE)</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">Rec. TDP2 IC<sub>50</sub><a class="ref internalNav" href="#t2fn1" aria-label="a">a</a> (μM)</th><th class="colsep0 rowsep0" align="center">WCE IC<sub>50</sub><a class="ref internalNav" href="#t2fn2" aria-label="b">b</a> (μM)</th><th class="colsep0 rowsep0" align="center">CC<sub>90</sub><a class="ref internalNav" href="#t2fn3" aria-label="c">c</a>(μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>36</b><a class="ref internalNav" href="#t2fn4" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="center">7.6 ± 4.1</td><td class="colsep0 rowsep0" align="center">9.3</td><td class="colsep0 rowsep0" align="center">9.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>38</b></td><td class="colsep0 rowsep0" align="center">7.5 ± 3.0</td><td class="colsep0 rowsep0" align="center">2.6</td><td class="colsep0 rowsep0" align="center"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>40</b><a class="ref internalNav" href="#t2fn4" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="center">3.0 ± 1.0</td><td class="colsep0 rowsep0" align="center">5.2</td><td class="colsep0 rowsep0" align="center"> >50</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>41</b></td><td class="colsep0 rowsep0" align="center">7.0 ± 1.7</td><td class="colsep0 rowsep0" align="center">3.5</td><td class="colsep0 rowsep0" align="center"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>43</b><a class="ref internalNav" href="#t2fn4" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="center">8.5</td><td class="colsep0 rowsep0" align="center">25</td><td class="colsep0 rowsep0" align="center">2.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>44</b></td><td class="colsep0 rowsep0" align="center">13</td><td class="colsep0 rowsep0" align="center">50</td><td class="colsep0 rowsep0" align="center"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>48</b></td><td class="colsep0 rowsep0" align="center">23</td><td class="colsep0 rowsep0" align="center"> >111</td><td class="colsep0 rowsep0" align="center"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>49</b></td><td class="colsep0 rowsep0" align="center">16 ± 5.0</td><td class="colsep0 rowsep0" align="center">23</td><td class="colsep0 rowsep0" align="center"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>51</b></td><td class="colsep0 rowsep0" align="center">13 ± 3.0</td><td class="colsep0 rowsep0" align="center">35</td><td class="colsep0 rowsep0" align="center"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>55</b></td><td class="colsep0 rowsep0" align="center">10 ± 2.0</td><td class="colsep0 rowsep0" align="center">25</td><td class="colsep0 rowsep0" align="center"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>59</b></td><td class="colsep0 rowsep0" align="center">42 ± 2.0</td><td class="colsep0 rowsep0" align="center">55</td><td class="colsep0 rowsep0" align="center"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>61</b></td><td class="colsep0 rowsep0" align="center">36 ± 3.0</td><td class="colsep0 rowsep0" align="center">40</td><td class="colsep0 rowsep0" align="center"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>62</b></td><td class="colsep0 rowsep0" align="center">18 ± 8.0</td><td class="colsep0 rowsep0" align="center">25</td><td class="colsep0 rowsep0" align="center"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>64</b></td><td class="colsep0 rowsep0" align="center">1.9 ± 0.90</td><td class="colsep0 rowsep0" align="center">2.2</td><td class="colsep0 rowsep0" align="center"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>69</b></td><td class="colsep0 rowsep0" align="center">21 ± 8.0</td><td class="colsep0 rowsep0" align="center">42</td><td class="colsep0 rowsep0" align="center"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>71</b></td><td class="colsep0 rowsep0" align="center">5.5 ± 2.0</td><td class="colsep0 rowsep0" align="center">15</td><td class="colsep0 rowsep0" align="center"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>72</b></td><td class="colsep0 rowsep0" align="center">5.6 ± 2.6</td><td class="colsep0 rowsep0" align="center">3.3</td><td class="colsep0 rowsep0" align="center"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>73</b></td><td class="colsep0 rowsep0" align="center">1.7 ± 0.30</td><td class="colsep0 rowsep0" align="center">3.2</td><td class="colsep0 rowsep0" align="center"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>74</b></td><td class="colsep0 rowsep0" align="center">6.4 ± 4.2</td><td class="colsep0 rowsep0" align="center">4.5</td><td class="colsep0 rowsep0" align="center"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>75</b></td><td class="colsep0 rowsep0" align="center">8.0 ± 5.1</td><td class="colsep0 rowsep0" align="center">6.5</td><td class="colsep0 rowsep0" align="center"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>79</b></td><td class="colsep0 rowsep0" align="center">3.0 ± 2.3</td><td class="colsep0 rowsep0" align="center">4.3</td><td class="colsep0 rowsep0" align="center"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>80</b></td><td class="colsep0 rowsep0" align="center">4.1 ± 1.9</td><td class="colsep0 rowsep0" align="center">5.3</td><td class="colsep0 rowsep0" align="center"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>81</b></td><td class="colsep0 rowsep0" align="center">4.5 ± 1.5</td><td class="colsep0 rowsep0" align="center">5.5</td><td class="colsep0 rowsep0" align="center"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>82</b></td><td class="colsep0 rowsep0" align="center">3.2 ± 2.5</td><td class="colsep0 rowsep0" align="center">2.7</td><td class="colsep0 rowsep0" align="center"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>84</b><a class="ref internalNav" href="#t2fn4" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="center">5.1 ± 2.0</td><td class="colsep0 rowsep0" align="center">11</td><td class="colsep0 rowsep0" align="center">12</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>85</b></td><td class="colsep0 rowsep0" align="center">9.3</td><td class="colsep0 rowsep0" align="center"> >111</td><td class="colsep0 rowsep0" align="center"> </td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>a</sup><p class="last">IC<sub>50</sub>: concentration of a compound producing 50% inhibition, expressed as mean ± standard deviation from at least three independent experiments.</p></div><div class="footnote" id="t2fn2"><sup>b</sup><p class="last">IC<sub>50</sub>: concentration of a compound producing 50% inhibition.</p></div><div class="footnote" id="t2fn3"><sup>c</sup><p class="last">CC<sub>90</sub>: concentration of a compound killing 90% of the cells. Assay was done in the absence of EPT against hTDP2 DT40 cells.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a></p></div><div class="footnote" id="t2fn4"><sup>d</sup><p class="last">Synergy was not observed when tested in combination with different concentrations of EPT.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a></p></div></div></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Molecular Modeling</h3><div class="NLM_p">With the lack of a TDP2/inhibitor cocrystal structure, understanding on the inhibitor binding mode and mechanism of inhibition remains elusive. With reported inhibitors <b>6</b> and <b>7</b>, it is unclear exactly how they inhibit TDP2 as molecular modeling revealed little about their binding. To provide a predicted binding mode of our new inhibitors, we have constructed a homology model of hTDP2 based on the available crystal structure of mouse TDP2 (mTDP2; PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GZ1">4GZ1</a><a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a>), which shares a high degree of sequence identity (65%) with hTDP2. With this model, docking analysis was performed using Glide XP (v.6.4).<a onclick="showRef(event, 'ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref23 ref24">(23, 24)</a> The predicted binding mode of compound <b>56</b> in the active site of hTDP2 suggests an interaction between the carbonyl at position 1 of isoquinoline-1,3-dione core and the residues, H351 and the backbone NH of N264 of TDP2 (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>a). The NH at position 2 and carbonyl group at position 3 of <b>56</b> can coordinate to the magnesium ion (Mg<sup>2+</sup>). The carbonyl group at position 7 of <b>56</b> forms a salt bridge with the R266 within the active site of hTDP2. Whereas in compound <b>40</b>, the carbonyl group at position 1 forms an H-bond interaction with the backbone NH of N264. The NH at position 2 and carbonyl group at position 3 within <b>40</b> coordinate to the E152 and magnesium ion (Mg<sup>2+</sup>), respectively (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>b).</div><figure id="fig4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01973/20160318/images/medium/jm-2015-01973q_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01973/20160318/images/large/jm-2015-01973q_0010.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Predicted binding mode of (a) <b>56</b> and (b) <b>40</b> within active site of hTDP2 (numbering was based on hTDP2 sequence). The active site residues are shown as green sticks and metal ions as magenta sphere. The H-bond interactions are depicted as black dotted lines, and the distance between bonded atoms are represented in red. Homology model (hTDP2) was built using the crystal structure of mTDP2 (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GZ1">4GZ1</a><a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01973/20160318/images/large/jm-2015-01973q_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01973&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i9">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_17730" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_17730" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">By screening our in-house collection of synthetic small molecules, isoquinoline-1,3-dione was identified as a novel inhibitor type of TDP2. The selective and potent inhibitory profile of this chemotype against TDP2 was further established through the chemical synthesis of 63 functional and scaffold analogues and validated via a secondary assay using WCE as well as a counter assay against TDP1, which shares some substrates with TDP2.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Analogue synthesis identified compound <b>64</b> as the best of the series with an IC<sub>50</sub> of 1.9 μM against recombinant TDP2 and 2.2 μM against TDP2 in WCE assay. The main SAR observations also suggest that the C6 or C7 substitution is strongly preferred and that the OH substitution at the N2 position resulting in a CO-NOH-CO chelating triad is undesired. While efforts toward obtaining the TDP2/inhibitor cocrystal structures are still underway, homology modeling did reveal a few specific interactions within the TDP2 active site that may contribute to understanding the molecular mechanism of TDP2 inhibition. Identifying this novel and selective TDP2 inhibitor type may provide an important path toward employing TDP2 as a drug target and developing TDP2 inhibitors as molecular probes.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i10">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_37157" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_37157" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Chemistry</h3><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i12" class="anchor-spacer"></div><h4 class="article-section__title" id="_i12"> General Procedures</h4><div class="NLM_p">All commercial chemicals were used as supplied unless otherwise indicated. Flash chromatography was performed on a Teledyne Combiflash RF-200 with RediSep columns (silica) and indicated mobile phase. All moisture sensitive reactions were performed under an inert atmosphere of ultrapure argon with oven-dried glassware. <sup>1</sup>H and <sup>13</sup>C NMR spectra were recorded on a Varian 600 MHz spectrometer. Mass data were acquired on an Agilent TOF II TOS/MS spectrometer capable of ESI and APCI ion sources. Analysis of sample purity was performed on a Varian Prepstar SD-1 HPLC system with a Phenomenex Gemini, 5 μm C18 column (250 mm × 4.6 mm). HPLC conditions: solvent A = H<sub>2</sub>O, solvent B = MeCN; flow rate =1.0 mL/min; compounds were eluted with a gradient of 10% MeCN/H<sub>2</sub>O for 0–5 min and to 80% MeCN/H<sub>2</sub>O from 5 to 30 min followed by 100% MeCN from 30 to 35 min and then 10% MeCN/H<sub>2</sub>O from 35 to 40 min. Purity was determined by total absorbance at 254 nm. All tested compounds have a purity ≥98%.</div><div id="sec4_1_1_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i13" class="anchor-spacer"></div><h5 class="article-section__title" id="_i13"> General Procedure for Suzuki Coupling (<b>21</b>, <b>25</b>)</h5><div class="NLM_p last">The mixture of compound <b>20</b> or <b>24</b> (1.0 equiv), aryl boronic acid (1.6 equiv), Pd(PPh<sub>3</sub>)<sub>4</sub> (0.065 equiv), and K<sub>2</sub>CO<sub>3</sub> (3.6 equiv) in EtOH/H<sub>2</sub>O (1:1) was irradiated at 150 °C for 30 min under microwave conditions. The black residue formed was filtered through Celite and the solvent was concentrated in vacuo. The resulting aqueous solution was acidified (pH = 3) using 2 N HCl. A white precipitate was obtained via filtration, which was washed with water and dried under vacuum to furnish desired compounds (<b>21</b>, <b>25</b>) as colorless solid.</div></div><div id="sec4_1_1_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i14" class="anchor-spacer"></div><h5 class="article-section__title" id="_i14"> General Procedure for the S<sub>N</sub>Ar Reaction (<b>23</b>)</h5><div class="NLM_p last">To a suspension of NaH (32 mmol, 2.0 equiv, 60% dispersion in oil) in DMSO (10 mL) at 0 °C was added methyl cyanoacetate (32 mmol, 2.0 equiv) slowly, and the mixture was stirred at rt for 30 min before 2-fluoro-4-iodobenzonitrile (<b>22</b>, 16 mmol, 1.0 equiv) in DMSO (16 mL) was added. The resulting solution was stirred at 90 °C for 8 h and quenched by adding 2 N HCl (20 mL). After extraction with the EtOAc (2 × 30 mL), the combined organics were washed with NaHCO<sub>3</sub> (2 × 20 mL), brine (30 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated in vacuo to leave brown oil. The crude material was used for next step without further purification.</div></div><div id="sec4_1_1_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i15" class="anchor-spacer"></div><h5 class="article-section__title" id="_i15"> General Procedure for the Synthesis of Dinitriles (<b>24</b>)</h5><div class="NLM_p last">A solution of compound <b>23</b> in DMSO/H<sub>2</sub>O (9:1) mixture was stirred at 120 °C for 16 h before being quenched with water. The aqueous solution was extracted using EtOAc (2 × 30 mL). The combined organics were washed with water (2 × 20 mL) and brine (30 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated in vacuo. The crude mixture was purified using CombiFlash with 0–20% EtOAc in hexane as an eluent to yield the desired products (<b>24</b>) as yellow solid. Yield: 70–85% over two steps.</div></div><div id="sec4_1_1_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i16" class="anchor-spacer"></div><h5 class="article-section__title" id="_i16"> General Procedure for the Synthesis of Arylketone (<b>26</b>)</h5><div class="NLM_p last">A mixture of compound <b>24</b> (1.0 equiv), arylboronic acid (2.5 equiv), Pd(OAC)<sub>2</sub> (0.1 equiv), Mo(CO)<sub>6</sub> (1.5 equiv), and K<sub>2</sub>CO<sub>3</sub> (3.0 equiv) in anisole (0.2M, 5 mL) was irradiated at 140 °C for 30 min under microwave conditions. The black residue formed was filtered through Celite, and the solvent was concentrated in vacuo. The crude mixture was purified using CombiFlash with 0–50% EtOAc in hexane as an eluent to yield the desired product <b>26</b> as colorless solid. Yield: 50–62%.</div></div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17"> General Procedure for Isoquinoline-1,3-dione (<b>33</b>–<b>88</b>)</h4><div class="NLM_p">Method A: To a suspension of compound <b>21</b> (1.0 equiv) in 1,2-dichlorobenzene was added urea (2.0 equiv), and the resulting mixture was irradiated at 175 °C for 45 min under microwave conditions. The solvent was removed in vacuo to produce the crude product as brown solid which was purified using CombiFlash with 0–50% EtOAc in hexane as an eluent to furnish titled compounds (<b>33</b>–<b>88</b>) as yellow solid. Method B: A solution of compound <b>24</b>/<b>25</b>/<b>26</b> (0.45 mmol) in conc HCl (2 mL) was stirred at 70 °C for 4 h. After the solvent was removed in vacuo, the residue was purified using CombiFlash with 0–30% EtOAc in hexane as an eluent to yield desired products (<b>33</b>–<b>88</b>) as colorless solid.</div><div id="sec4_1_2_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i18" class="anchor-spacer"></div><h5 class="article-section__title" id="_i18"> Isoquinoline-1,3(2<i>H</i>,4<i>H</i>)-dione (<b>33</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.28 (s, 1H), 8.01 (dd, <i>J</i> = 7.8, 1.4 Hz, 1H), 7.64 (td, <i>J</i> = 7.5, 1.4 Hz, 1H), 7.45 (t, <i>J</i> = 7.6 Hz, 1H), 7.38 (d, <i>J</i> = 7.6 Hz, 1H), 4.03 (s, 2H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 171.1, 165.5, 136.8, 133.6, 127.9, 127.7, 127.5, 125.1, 36.1. HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>9</sub>H<sub>6</sub>NO<sub>2</sub>, 160.0404 [M – H]<sup>−</sup>; found, 160.0400.</div></div><div id="sec4_1_2_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i19" class="anchor-spacer"></div><h5 class="article-section__title" id="_i19"> 5-Bromoisoquinoline-1,3(2<i>H</i>,4<i>H</i>)-dione (<b>34</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.48 (s, 1H), 8.06 (d, <i>J</i> = 7.7 Hz, 1H), 7.95 (d, <i>J</i> = 7.8 Hz, 1H), 7.43 (t, <i>J</i> = 7.8 Hz, 1H), 3.87 (s, 2H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 169.2, 164.1, 136.6, 135.7, 128.5, 126.9, 126.7, 122.2, 37.1. HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>9</sub>H<sub>5</sub>BrNO<sub>2</sub>, 237.9509 [M – H]<sup>−</sup>; found, 237.9506.</div></div><div id="sec4_1_2_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i20" class="anchor-spacer"></div><h5 class="article-section__title" id="_i20"> 6-Bromoisoquinoline-1,3(2<i>H</i>,4<i>H</i>)-dione (<b>35</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.37 (s, 1H), 7.92 (d, <i>J</i> = 8.1 Hz, 1H), 7.67–7.65 (m, 2H), 4.04 (s, 2H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 174.0, 162.4, 156.9, 137.1, 133.6, 129.9, 128.6, 127.6, 35.9. HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>9</sub>H<sub>5</sub>BrNO<sub>2</sub>, 237.9509 [M – H]<sup>−</sup>; found, 237.9515.</div></div><div id="sec4_1_2_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i21" class="anchor-spacer"></div><h5 class="article-section__title" id="_i21"> 6-Iodoisoquinoline-1,3(2<i>H</i>,4<i>H</i>)-dione (<b>36</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.35 (s, 1H), 7.83 (m, 2H), 7.74 (d, <i>J</i> = 8.7 Hz, 1H), 4.01 (s, 2H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 170.9, 165.5, 139.1, 136.9, 136.8, 136.4, 125.0, 102.5, 35.9. HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>9</sub>H<sub>5</sub>INO<sub>2</sub>, 285.9370 [M – H]<sup>−</sup>; found, 285.9376.</div></div><div id="sec4_1_2_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i22" class="anchor-spacer"></div><h5 class="article-section__title" id="_i22"> 7-Iodoisoquinoline-1,3(2<i>H</i>,4<i>H</i>)-dione (<b>37</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.38 (s, 1H), 8.25 (d, <i>J</i> = 1.8 Hz, 1H), 7.97 (dd, <i>J</i> = 8.1, 1.8 Hz, 1H), 7.19 (d, <i>J</i> = 8.1 Hz, 1H), 3.96 (s, 2H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 170.4, 163.9, 141.5, 136.1, 135.3, 129.9, 126.8, 92.1, 35.5. HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>9</sub>H<sub>5</sub>INO<sub>2</sub>, 285.9370 [M – H]<sup>−</sup>; found, 285.9367.</div></div><div id="sec4_1_2_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i23" class="anchor-spacer"></div><h5 class="article-section__title" id="_i23"> 8-Iodoisoquinoline-1,3(2<i>H</i>,4<i>H</i>)-dione (<b>38</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.28 (s, 1H), 8.06 (d, <i>J</i> = 7.7 Hz, 1H), 7.40 (d, <i>J</i> = 7.7 Hz, 1H), 7.26 (t, <i>J</i> = 7.7 Hz, 1H), 4.05 (s, 2H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 169.7, 163.6, 141.8, 134.4, 133.4, 128.2, 124.6, 94.9, 37.5. HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>9</sub>H<sub>5</sub>INO<sub>2</sub>, 285.9370 [M – H]<sup>−</sup>; found, 285.9372.</div></div><div id="sec4_1_2_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i24" class="anchor-spacer"></div><h5 class="article-section__title" id="_i24"> 1,3-Dioxo-1,2,3,4-tetrahydroisoquinoline-6-carboxylic Acid (<b>39</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.37 (s, 1H), 11.40 (s, 1H), 8.08 (d, <i>J</i> = 8.4 Hz, 1H), 7.92 (d, <i>J</i> = 8.4 Hz, 1H), 7.91 (s, 1H), 4.08 (s, 2H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 170.6, 166.3, 164.6, 136.8, 134.6, 128.6, 128.1, 127.6, 127.4, 35.8. HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>10</sub>H<sub>6</sub>NO<sub>4</sub>, 204.0302 [M – H]<sup>−</sup>; found, 204.0301.</div></div><div id="sec4_1_2_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i25" class="anchor-spacer"></div><h5 class="article-section__title" id="_i25"> 1,3-Dioxo-1,2,3,4-tetrahydroisoquinoline-7-carboxylic Acid (<b>40</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.26 (s, 1H), 11.42 (s, 1H), 8.53 (d, <i>J</i> = 1.8 Hz, 1H), 8.14 (dd, <i>J</i> = 8.1, 1.8 Hz, 1H), 7.51 (d, <i>J</i> = 8.1 Hz, 1H), 4.11 (s, 2H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 175.0, 165.9, 162.8, 135.9, 135.2, 134.1, 130.2, 128.7, 127.2, 36.1. HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>10</sub>H<sub>6</sub>NO<sub>4</sub>, 204.0302 [M – H]–; found, 204.0308.</div></div><div id="sec4_1_2_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i26" class="anchor-spacer"></div><h5 class="article-section__title" id="_i26"> 6-(Trifluoromethyl)isoquinoline-1,3(2<i>H</i>,4<i>H</i>)-dione (<b>41</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.50 (s, 1H), 8.19 (d, <i>J</i> = 8.1 Hz, 1H), 7.82 (s, 1H), 7.80 (d, <i>J</i> = 8.3 Hz, 1H), 4.12 (s, 2H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 170.5, 164.4, 138.0, 132.7, 128.5, 124.9, 124.6, 123.7, 122.7, 36.0. HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>10</sub>H<sub>5</sub>F<sub>3</sub>NO<sub>2</sub>, 228.0278 [M – H]<sup>−</sup>; found, 228.0285.</div></div><div id="sec4_1_2_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i27" class="anchor-spacer"></div><h5 class="article-section__title" id="_i27"> 6-Nitroisoquinoline-1,3(2<i>H</i>,4<i>H</i>)-dione (<b>42</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.59 (s, 1H), 8.29 (s, 1H), 8.25–8.25 (m, 2H), 4.17 (s, 2H). HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>9</sub>H<sub>5</sub>N<sub>2</sub>O<sub>4</sub>, 205.0255 [M – H]<sup>−</sup>; found, 205.0252.</div></div><div id="sec4_1_2_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i28" class="anchor-spacer"></div><h5 class="article-section__title" id="_i28"> 6-(Furan-2-yl)isoquinoline-1,3 (2<i>H</i>,4<i>H</i>)-dione (<b>43</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.28 (s, 1H), 8.03 (d, <i>J</i> = 8.2 Hz, 1H), 7.86 (s, 1H), 7.77 (d, <i>J</i> = 8.2 Hz, 1H), 7.70 (s, 1H), 7.18 (d, <i>J</i> = 3.3 Hz, 1H), 6.67 (dd, <i>J</i> = 3.3, 1.7 Hz, 1H), 4.07 (s, 2H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 175.4, 163.0, 150.9, 145.2, 135.0, 133.2, 129.3, 128.9, 128.0, 120.57, 112.9, 110.1, 39.2. HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>13</sub>H<sub>8</sub>NO<sub>3</sub>, 226.0510 [M – H]<sup>−</sup>; found, 226.0506.</div></div><div id="sec4_1_2_12" class="NLM_sec NLM_sec_level_4"><div id="ac_i29" class="anchor-spacer"></div><h5 class="article-section__title" id="_i29"> 7-(Furan-2-yl)isoquinoline-1,3(2<i>H</i>,4<i>H</i>)-dione (<b>44</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.36 (s, 1H), 8.27 (s, 1H), 7.97 (dd, <i>J</i> = 8.1, 1.7 Hz, 1H), 7.79 (s, 1H), 7.44 (d, <i>J</i> = 8.1 Hz, 1H), 7.09 (d, <i>J</i> = 3.3 Hz, 1H), 6.63 (dd, <i>J</i> = 3.3, 1.7 Hz, 1H), 4.04 (s, 2H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 171.0, 165.2, 151.9, 143.5, 135.6, 129.5, 128.8, 128.4, 125.6, 121.7, 112.4, 106.8, 35.9. HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>13</sub>H<sub>8</sub>NO<sub>3</sub>, 226.0510 [M – H]<sup>−</sup>; found, 226.0504.</div></div><div id="sec4_1_2_13" class="NLM_sec NLM_sec_level_4"><div id="ac_i30" class="anchor-spacer"></div><h5 class="article-section__title" id="_i30"> 5-(Furan-2-yl)isoquinoline-1,3(2<i>H</i>,4<i>H</i>)-dione (<b>45</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.45 (s, 1H), 8.06 (d, <i>J</i> = 7.7 Hz, 1H), 8.00 (d, <i>J</i> = 7.7 Hz, 1H), 7.88 (d, <i>J</i> = 1.7 Hz, 1H), 7.56 (t, <i>J</i> = 7.8 Hz, 1H), 6.95 (d, <i>J</i> = 3.4 Hz, 1H), 6.69 (dd, <i>J</i> = 3.4, 1.7 Hz, 1H), 4.13 (s, 2H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 170.5, 165.3, 150.6, 143.6, 132.3, 131.6, 128.8, 127.5, 127.3, 126.1, 112.2, 110.6, 35.8. HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>13</sub>H<sub>8</sub>NO<sub>3</sub>, 226.0510 [M – H]<sup>−</sup>; found, 226.0514.</div></div><div id="sec4_1_2_14" class="NLM_sec NLM_sec_level_4"><div id="ac_i31" class="anchor-spacer"></div><h5 class="article-section__title" id="_i31"> 6-(Furan-2-yl)-2-hydroxyisoquinoline-1,3(2<i>H</i>,4<i>H</i>)-dione (<b>46</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.39 (s, 1H), 8.05 (d, <i>J</i> = 8.3 Hz, 1H), 7.87 (d, <i>J</i> = 0.7 Hz, 1H), 7.80 (d, <i>J</i> = 8.3 Hz, 1H), 7.71 (s, 1H), 7.19 (d, <i>J</i> = 3.4 Hz, 1H), 6.69–6.66 (m, 1H), 4.30 (s, 2H). HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>13</sub>H<sub>8</sub>NO<sub>4</sub>, 242.0459 [M – H]<sup>−</sup>; found, 242.0464.</div></div><div id="sec4_1_2_15" class="NLM_sec NLM_sec_level_4"><div id="ac_i32" class="anchor-spacer"></div><h5 class="article-section__title" id="_i32"> 7-(Furan-2-yl)-2-hydroxyisoquinoline-1,3(2<i>H</i>,4<i>H</i>)-dione (<b>47</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 8.40 (d, <i>J</i> = 1.8 Hz, 1H), 7.95 (dd, <i>J</i> = 8.1, 1.8 Hz, 1H), 7.61 (d, <i>J</i> = 1.3 Hz, 1H), 7.42 (d, <i>J</i> = 8.1 Hz, 1H), 6.89 (d, <i>J</i> = 3.3 Hz, 1H), 6.55 (dd, <i>J</i> = 3.3, 1.8 Hz, 1H). HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>13</sub>H<sub>8</sub>NO<sub>4</sub>, 242.0459 [M – H]<sup>−</sup>; found, 242.0466.</div></div><div id="sec4_1_2_16" class="NLM_sec NLM_sec_level_4"><div id="ac_i33" class="anchor-spacer"></div><h5 class="article-section__title" id="_i33"> 6-Phenylisoquinoline-1,3(2<i>H</i>,4<i>H</i>)-dione (<b>48</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.31 (s, 1H), 8.08 (d, <i>J</i> = 8.1 Hz, 1H), 7.76–7.74 (m, 3H), 7.71 (s, 1H), 7.52 (t, <i>J</i> = 7.6 Hz, 2H), 7.44 (t, <i>J</i> = 7.3 Hz, 1H), 4.10 (s, 2H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 175.7, 163.1, 157.6, 145.3, 137.8, 132.9, 132.9, 129.5, 129.2, 128.6, 127.2, 124.2, 39.1. HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>15</sub>H<sub>10</sub>NO<sub>2</sub>, 236.0717 [M – H]<sup>−</sup>; found, 236.0712.</div></div><div id="sec4_1_2_17" class="NLM_sec NLM_sec_level_4"><div id="ac_i34" class="anchor-spacer"></div><h5 class="article-section__title" id="_i34"> 6-(Pyridin-3-yl)isoquinoline-1,3(2<i>H</i>,4<i>H</i>)-dione (<b>49</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.40 (s, 1H), 9.18 (s, 1H), 8.83 (d, <i>J</i> = 4.4 Hz, 1H), 8.62 (d, <i>J</i> = 7.8 Hz, 1H), 8.14 (d, <i>J</i> = 8.0 Hz, 1H), 7.96–7.84 (m, 3H), 4.11 (s, 2H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 170.6, 164.8, 144.7, 143.4, 139.6, 139.5, 137.4, 135.8, 128.2, 126.5, 125.8, 125.8, 125.2, 35.9. HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>14</sub>H<sub>9</sub>N<sub>2</sub>O<sub>2</sub>, 237.0670 [M – H]; found, 237.0672.</div></div><div id="sec4_1_2_18" class="NLM_sec NLM_sec_level_4"><div id="ac_i35" class="anchor-spacer"></div><h5 class="article-section__title" id="_i35"> 7-(Pyridin-3-yl)isoquinoline-1,3(2<i>H</i>,4<i>H</i>)-dione (<b>50</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.44 (s, 1H), 9.29 (s, 1H), 8.87 (d, <i>J</i> = 5.5 Hz, 1H), 8.82 (d, <i>J</i> = 8.1 Hz, 1H), 8.42 (d, <i>J</i> = 2.1 Hz, 1H), 8.15 (dd, <i>J</i> = 8.0, 2.1 Hz, 1H), 8.04 (dd, <i>J</i> = 8.0, 5.5 Hz, 1H), 7.59 (d, <i>J</i> = 8.1 Hz, 1H), 4.13 (s, 2H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 171.4, 165.6, 143.5, 142.8, 141.7, 138.5, 137.4, 134.3, 132.8, 129.7, 127.1, 126.7, 126.6, 36.6. HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>14</sub>H<sub>9</sub>N<sub>2</sub>O<sub>2</sub>, 237.0670 [M – H]<sup>−</sup>; found, 237.0673.</div></div><div id="sec4_1_2_19" class="NLM_sec NLM_sec_level_4"><div id="ac_i36" class="anchor-spacer"></div><h5 class="article-section__title" id="_i36"> 6-(3-Hydroxyphenyl)isoquinoline-1,3(2<i>H</i>,4<i>H</i>)-dione (<b>51</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.30 (s, 1H), 9.64 (s, 1H), 8.07 (d, <i>J</i> = 8.2 Hz, 1H), 7.69 (dd, <i>J</i> = 8.2, 1.7 Hz, 1H), 7.63 (d, <i>J</i> = 0.9 Hz, 1H), 7.31 (t, <i>J</i> = 7.9 Hz, 1H), 7.17–7.13 (m, 1H), 7.09 (t, <i>J</i> = 2.3 Hz, 1H), 6.85 (ddd, <i>J</i> = 8.2, 2.3, 0.9 Hz, 1H), 4.10 (s, 2H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 171.2, 165.3, 158.1, 145.1, 140.2, 137.4, 130.3, 128.2, 126.0, 125.6, 124.0, 117.9, 115.7, 113.9, 36.3. HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>15</sub>H<sub>10</sub>NO<sub>3</sub>, 252.0666 [M – H]<sup>−</sup>; found, 252.0670.</div></div><div id="sec4_1_2_20" class="NLM_sec NLM_sec_level_4"><div id="ac_i37" class="anchor-spacer"></div><h5 class="article-section__title" id="_i37"> 7-(3-Hydroxyphenyl)isoquinoline-1,3(2<i>H</i>,4<i>H</i>)-dione (<b>52</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.35 (s, 1H), 9.58 (s, 1H), 8.17 (d, <i>J</i> = 1.9 Hz, 1H), 7.90 (dd, <i>J</i> = 7.9, 1.9 Hz, 1H), 7.47 (d, <i>J</i> = 8.0 Hz, 1H), 7.29 (t, <i>J</i> = 7.9 Hz, 1H), 7.12 (d, <i>J</i> = 7.7 Hz, 1H), 7.07 (s, 1H), 6.87–6.75 (m, 1H), 4.07 (s, 2H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 170.7, 165.1, 157.8, 139.9, 138.9, 135.5, 131.4, 129.9, 128.4, 125.2, 124.7, 117.1, 114.7, 113.1, 35.5. HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>15</sub>H<sub>10</sub>NO<sub>3</sub>, 252.0666 [M – H]<sup>−</sup>; found, 252.0667.</div></div><div id="sec4_1_2_21" class="NLM_sec NLM_sec_level_4"><div id="ac_i38" class="anchor-spacer"></div><h5 class="article-section__title" id="_i38"> 8-(3-Hydroxyphenyl)isoquinoline-1,3(2<i>H</i>,4<i>H</i>)-dione (<b>53</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.00 (s, 1H), 9.32 (s, 1H), 7.60 (t, <i>J</i> = 7.6 Hz, 1H), 7.37 (d, <i>J</i> = 7.3 Hz, 1H), 7.15 (d, <i>J</i> = 7.5 Hz, 1H), 7.12 (t, <i>J</i> = 7.8 Hz, 1H), 6.71 (dd, <i>J</i> = 8.2, 1.7 Hz, 1H), 6.61 (d, <i>J</i> = 7.5 Hz, 1H), 6.60–6.58 (m, 1H), 4.08 (s, 2H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 170.9, 164.6, 156.8, 144.3, 143.8, 137.9, 132.4, 130.7, 128.7, 127.6, 123.0, 119.44, 115.6, 113.8, 37.0. HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>15</sub>H<sub>10</sub>NO<sub>3</sub>, 252.0666 [M – H]<sup>−</sup>; found, 252.0670.</div></div><div id="sec4_1_2_22" class="NLM_sec NLM_sec_level_4"><div id="ac_i39" class="anchor-spacer"></div><h5 class="article-section__title" id="_i39"> 7-(4-Hydroxyphenyl)isoquinoline-1,3(2<i>H</i>,4<i>H</i>)-dione (<b>54</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.32 (s, 1H), 9.63 (s, 1H), 8.15 (d, <i>J</i> = 2.0 Hz, 1H), 7.87 (dd, <i>J</i> = 8.0, 2.0 Hz, 1H), 7.54 (d, <i>J</i> = 8.6 Hz, 2H), 7.42 (d, <i>J</i> = 8.1 Hz, 1H), 6.87 (d, <i>J</i> = 8.6 Hz, 2H), 4.04 (s, 2H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 170.9, 165.2, 157.4, 139.0, 134.4, 130.9, 129.4, 128.4, 127.6, 125.2, 124.0, 115.8, 35.5. HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>15</sub>H<sub>10</sub>NO<sub>3</sub>, 252.0666 [M – H]<sup>−</sup>; found, 252.0669.</div></div><div id="sec4_1_2_23" class="NLM_sec NLM_sec_level_4"><div id="ac_i40" class="anchor-spacer"></div><h5 class="article-section__title" id="_i40"> 6-(3-Nitrophenyl)isoquinoline-1,3(2<i>H</i>,4<i>H</i>)-dione (<b>55</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.36 (s, 1H), 8.53 (t, <i>J</i> = 1.9 Hz, 1H), 8.29 (dd, <i>J</i> = 8.2, 1.9 Hz, 1H), 8.23 (d, <i>J</i> = 8.2 Hz, 1H), 8.12 (d, <i>J</i> = 8.1 Hz, 1H), 7.88 (d, <i>J</i> = 8.2 Hz, 1H), 7.86 (s, 1H), 7.81 (t, <i>J</i> = 8.1 Hz, 1H), 4.12 (s, 2H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 171.4, 165.5, 148.9, 142.7, 140.7, 138.0, 134.1, 131.1, 128.9, 126.9, 126.2, 125.3, 123.7, 122.0, 36.6. HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>15</sub>H<sub>9</sub>N<sub>2</sub>O<sub>4</sub>, 281.0568 [M – H]<sup>−</sup>; found, 281.0561.</div></div><div id="sec4_1_2_24" class="NLM_sec NLM_sec_level_4"><div id="ac_i41" class="anchor-spacer"></div><h5 class="article-section__title" id="_i41"> 7-(3-Nitrophenyl)isoquinoline-1,3(2<i>H</i>,4<i>H</i>)-dione (<b>56</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.41 (s, 1H), 8.49 (s, 1H), 8.33 (s, 1H), 8.26 (d, <i>J</i> = 8.1 Hz, 1H), 8.22 (d, <i>J</i> = 8.1 Hz, 1H), 8.10 (d, <i>J</i> = 9.8 Hz, 1H), 7.79 (t, <i>J</i> = 7.9 Hz, 1H), 7.55 (d, <i>J</i> = 8.1 Hz, 1H), 4.11 (s, 2H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 171.3, 165.5, 148.9, 140.8, 137.4, 134.2, 133.74, 132.4, 131.2, 129.4, 125.9, 123.1, 122.3, 121.6, 36.3. HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>15</sub>H<sub>9</sub>N<sub>2</sub>O<sub>4</sub>, 281.0568 [M – H]<sup>−</sup>; found, 281.0570.</div></div><div id="sec4_1_2_25" class="NLM_sec NLM_sec_level_4"><div id="ac_i42" class="anchor-spacer"></div><h5 class="article-section__title" id="_i42"> 8-(3-Nitrophenyl)isoquinoline-1,3(2<i>H</i>,4<i>H</i>)-dione (<b>57</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.12 (s, 1H), 8.23–8.19 (m, 1H), 8.08 (s, 1H), 7.72 (d, <i>J</i> = 7.6 Hz, 1H), 7.70–7.64 (m, 2H), 7.48 (d, <i>J</i> = 7.7 Hz, 1H), 7.26 (d, <i>J</i> = 7.5 Hz, 1H), 4.12 (s, 2H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 170.8, 165.0, 147.4, 144.1, 141.5, 138.5, 135.5, 132.8, 130.9, 130.7, 129.3, 123.6, 123.3, 121.8, 36.9. HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>15</sub>H<sub>9</sub>N<sub>2</sub>O<sub>4</sub>, 281.0568 [M – H]<sup>−</sup>; found, 281.0575</div></div><div id="sec4_1_2_26" class="NLM_sec NLM_sec_level_4"><div id="ac_i43" class="anchor-spacer"></div><h5 class="article-section__title" id="_i43"> 5-(3-Nitrophenyl)isoquinoline-1,3(2<i>H</i>,4<i>H</i>)-dione (<b>58</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.42 (s, 1H), 8.30 (d, <i>J</i> = 7.8 Hz, 1H), 8.23 (s, 1H), 8.15 (d, <i>J</i> = 7.5 Hz, 1H), 7.89 (d, <i>J</i> = 7.5 Hz, 1H), 7.80 (t, <i>J</i> = 7.8 Hz, 1H), 7.64 (d, <i>J</i> = 7.1 Hz, 1H), 7.60 (t, <i>J</i> = 7.5 Hz, 1H), 3.85 (s, 2H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 170.1, 165.0, 147.7, 140.1, 138.0, 135.5, 134.4, 133.9, 129.9, 127.6, 127.2, 125.4, 123.5, 122.5, 35.0. HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>15</sub>H<sub>9</sub>N<sub>2</sub>O<sub>4</sub>, 281.0568 [M – H]<sup>−</sup>; found, 281.0572.</div></div><div id="sec4_1_2_27" class="NLM_sec NLM_sec_level_4"><div id="ac_i44" class="anchor-spacer"></div><h5 class="article-section__title" id="_i44"> 6-(3-Aminophenyl)isoquinoline-1,3(2<i>H</i>,4<i>H</i>)-dione (<b>59</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.34 (s, 1H), 9.84 (s, 1H), 8.11 (d, <i>J</i> = 8.2 Hz, 1H), 7.71 (dd, <i>J</i> = 8.2, 1.7 Hz, 1H), 7.65–7.63 (m, 2H), 7.61–7.52 (m, 2H), 7.32 (s, 1H), 4.11 (s, 2H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 171.4, 165.5, 144.1, 140.4, 137.9, 130.9, 128.8, 126.4, 125.9, 124.9, 122.2, 120.6, 117.5, 114.2, 36.6. HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>15</sub>H<sub>11</sub>N<sub>2</sub>O<sub>2</sub>, 251.0826 [M – H]<sup>−</sup>; found, 251.0829.</div></div><div id="sec4_1_2_28" class="NLM_sec NLM_sec_level_4"><div id="ac_i45" class="anchor-spacer"></div><h5 class="article-section__title" id="_i45"> 7-(3-Aminophenyl)isoquinoline-1,3(2<i>H</i>,4<i>H</i>)-dione (<b>60</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.39 (s, 1H), 10.19 (s, 2H), 8.23 (d, <i>J</i> = 2.0 Hz, 1H), 7.95 (dd, <i>J</i> = 8.0, 2.0 Hz, 1H), 7.69 (d, <i>J</i> = 7.9 Hz, 1H), 7.67 (s, 1H), 7.57 (t, <i>J</i> = 7.9 Hz, 1H), 7.52 (d, <i>J</i> = 8.0 Hz, 1H), 7.38–7.34 (m, 1H), 4.09 (s, 2H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 171.3, 165.6, 140.5, 138.3, 136.9, 134.9, 131.9, 130.9, 129.4, 126.1, 125.5, 122.3, 120.9, 36.3. HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>15</sub>H<sub>11</sub>N<sub>2</sub>O<sub>2</sub>, 251.0826 [M – H]<sup>−</sup>; found, 251.0829.</div></div><div id="sec4_1_2_29" class="NLM_sec NLM_sec_level_4"><div id="ac_i46" class="anchor-spacer"></div><h5 class="article-section__title" id="_i46"> 6-(4-Aminophenyl)isoquinoline-1,3(2<i>H</i>,4<i>H</i>)-dione (<b>61</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.30 (s, 1H), 9.59 (s, 2H), 8.06 (d, <i>J</i> = 8.2 Hz, 1H), 7.81–7.77 (m, 2H), 7.74 (dd, <i>J</i> = 8.2, 1.7 Hz, 1H), 7.69 (d, <i>J</i> = 1.7 Hz, 1H), 7.35 (d, <i>J</i> = 8.4 Hz, 2H), 4.08 (s, 2H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 171.4, 165.6, 145.7, 144.3, 137.8, 133.5, 128.6, 126.1, 125.7, 124.3, 122.4, 119.2, 36.6. HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>15</sub>H<sub>11</sub>N<sub>2</sub>O<sub>2</sub>, 251.0826 [M – H]<sup>−</sup>; found, 251.0830.</div></div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> General Procedure for the Synthesis of Methanesulfonamide Derivative (<b>62</b>–<b>64</b>)</h4><div class="NLM_p">To a solution of <b>59</b>/<b>60</b>/<b>61</b> (0.39 mmol, 1.0 equiv) in dioxane (5 mL) was added pyridine (0.99 mmol, 2.5 equiv) and methanesulfonyl chloride (0.79 mmol, 2.0 equiv), and the resulting solution was stirred at rt for 1 h before being quenched with water. After extraction with EtOAc (2 × 20 mL), the combined organic layers were washed with 1 N HCl (2 × 20 mL) and brine (20 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated in vacuo. The crude mixture was purified using CombiFlash with 0–5% methanol in DCM as an eluent to yield the desired product as colorless solid. Yield: 65–77%.</div><div id="sec4_1_3_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i48" class="anchor-spacer"></div><h5 class="article-section__title" id="_i48"> <i>N</i>-(3-(1,3-Dioxo-1,2,3,4-tetrahydroisoquinolin-6-yl)phenyl)methanesulfonamide (<b>62</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.32 (s, 1H), 9.89 (s, 1H), 8.10 (d, <i>J</i> = 8.2 Hz, 1H), 7.69 (d, <i>J</i> = 8.5 Hz, 1H), 7.64 (s, 1H), 7.53 (s, 1H), 7.50–7.45 (m, 2H), 7.28 (d, <i>J</i> = 7.0 Hz, 1H), 3.05 (s, 3H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 171.4, 165.6, 144.9, 140.4, 139.6, 137.9, 130.6, 128.7, 126.5, 126.0, 124.7, 123.1, 120.2, 118.6, 39.9, 36.6. HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>16</sub>H<sub>13</sub>N<sub>2</sub>O<sub>4</sub>S, 329.0602 [M – H]<sup>−</sup>; found, 329.0595.</div></div><div id="sec4_1_3_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i49" class="anchor-spacer"></div><h5 class="article-section__title" id="_i49"> <i>N</i>-(3-(1,3-Dioxo-1,2,3,4-tetrahydroisoquinolin-7-yl)phenyl)methanesulfonamide (<b>63</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.37 (s, 1H), 9.84 (s, 1H), 8.20 (d, <i>J</i> = 2.0 Hz, 1H), 7.91 (dd, <i>J</i> = 8.1, 2.0 Hz, 1H), 7.52 (s, 1H), 7.49 (d, <i>J</i> = 8.1 Hz, 1H), 7.47–7.44 (m, 2H), 7.36–7.18 (m, 1H), 4.07 (s, 2H), 3.04 (s, 3H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 171.4, 165.7, 140.4, 139.6, 139.1, 136.5, 132.1, 130.6, 129.2, 126.0, 125.4, 122.6, 119.5, 118.2, 39.8, 36.2. HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>16</sub>H<sub>13</sub>N<sub>2</sub>O<sub>4</sub>S, 329.0602 [M – H]<sup>−</sup>; found, 329.0598.</div></div><div id="sec4_1_3_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i50" class="anchor-spacer"></div><h5 class="article-section__title" id="_i50"> <i>N</i>-(4-(1,3-Dioxo-1,2,3,4-tetrahydroisoquinolin-6-yl)phenyl)methanesulfonamide (<b>64</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.29 (s, 1H), 9.97 (s, 1H), 8.06 (d, <i>J</i> = 8.2 Hz, 1H), 7.76–7.72 (m, 3H), 7.67 (s, 1H), 7.33 (d, <i>J</i> = 8.6 Hz, 2H), 4.08 (s, 2H), 3.05 (s, 3H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 175.5, 163.1, 157.6, 144.7, 139.5, 132.9, 132.7, 132.2, 129.2, 128.3, 123.6, 119.6, 40.2, 36.5. HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>16</sub>H<sub>13</sub>N<sub>2</sub>O<sub>4</sub>S, 329.0602 [M – H]<sup>−</sup>; found, 329.0602.</div></div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> General Procedure for the Synthesis of Acrylamide Derivative (<b>65</b>–<b>66</b>)</h4><div class="NLM_p">To a solution of <b>60</b><b>/61</b> (0.39 mmol, 1.0 equiv) in dioxane (5 mL) was added pyridine (0.99 mmol, 2.5 equiv) and acryloyl chloride (0.79 mmol, 2.0 equiv), and the resulting solution was stirred at rt for 1 h before being quenched with water. After extraction with EtOAc (2 × 20 mL), the combined organic layers were washed with 1 N HCl (2 × 20 mL) and brine (20 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated in vacuo. The crude mixture was purified using CombiFlash with 0–10% methanol in DCM as an eluent to yield the desired product as colorless solid. Yield: 70–79%.</div><div id="sec4_1_4_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i52" class="anchor-spacer"></div><h5 class="article-section__title" id="_i52"> <i>N</i>-(4-(1,3-Dioxo-1,2,3,4-tetrahydroisoquinolin-6-yl)phenyl)acrylamide (<b>65</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.28 (s, 1H), 10.30 (s, 1H), 8.05 (d, <i>J</i> = 8.2 Hz, 1H), 7.81 (d, <i>J</i> = 8.7 Hz, 2H), 7.75 (dd, <i>J</i> = 8.7, 2.2 Hz, 3H), 7.69 (s, 1H), 6.46 (dd, <i>J</i> = 17.0, 10.2 Hz, 1H), 6.28 (dd, <i>J</i> = 17.0, 1.8 Hz, 1H), 5.78 (dd, <i>J</i> = 10.2, 1.8 Hz, 1H), 4.08 (s, 2H). HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>18</sub>H<sub>13</sub>N<sub>2</sub>O<sub>3</sub>, 305.0932 [M – H]<sup>−</sup>; found, 305.0928.</div></div><div id="sec4_1_4_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i53" class="anchor-spacer"></div><h5 class="article-section__title" id="_i53"> <i>N</i>-(3-(1,3-Dioxo-1,2,3,4-tetrahydroisoquinolin-7-yl)phenyl)acrylamide (<b>66</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.38 (s, 1H), 10.29 (s, 1H), 8.24 (s, 1H), 8.07 (s, 1H), 7.94 (d, <i>J</i> = 7.9 Hz, 1H), 7.72 (d, <i>J</i> = 6.6 Hz, 1H), 7.50 (d, <i>J</i> = 7.9 Hz, 1H), 7.45 (d, <i>J</i> = 7.4 Hz, 2H), 6.46 (dd, <i>J</i> = 16.8, 10.2 Hz, 1H), 6.29 (d, <i>J</i> = 16.8 Hz, 1H), 5.79 (d, <i>J</i> = 10.2 Hz, 1H), 4.08 (s, 2H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 171.1, 165.4, 163.5, 139.9, 139.41, 138.9, 136.1, 131.9, 131.7, 129.8, 128.9, 127.3, 125.7, 125.0, 121.9, 118.9, 117.5, 35.9. HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>18</sub>H<sub>13</sub>N<sub>2</sub>O<sub>3</sub>, 305.0932 [M – H]<sup>−</sup>; found, 305.0936.</div></div></div><div id="sec4_1_4_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> 2-(1,3-Dioxo-1,2,3,4-tetrahydroisoquinolin-6-yl)benzoic Acid (<b>67</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.90 (s, 1H), 11.31 (s, 1H), 8.02 (d, <i>J</i> = 7.8 Hz, 1H), 7.82 (d, <i>J</i> = 7.5 Hz, 1H), 7.63 (t, <i>J</i> = 7.4 Hz, 1H), 7.52 (t, <i>J</i> = 7.6 Hz, 1H), 7.43–7.36 (m, 3H), 4.06 (s, 2H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 171.1, 169.1, 165.3, 146.3, 140.2, 136.6, 131.9, 131.4, 130.6, 129.7, 128.3, 127.8, 127.6, 127.2, 123.9, 36.2. HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>16</sub>H<sub>10</sub>NO<sub>4</sub>, 280.0615 [M – H]<sup>−</sup>; found, 280.0608.</div></div><div id="sec4_1_4_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> 4-(1,3-Dioxo-1,2,3,4-tetrahydroisoquinolin-6-yl)benzoic Acid (<b>68</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.05 (s, 1H), 11.34 (s, 1H), 8.11 (d, <i>J</i> = 8.2 Hz, 1H), 8.06 (d, <i>J</i> = 8.2 Hz, 2H), 7.88 (d, <i>J</i> = 8.2 Hz, 2H), 7.83 (d, <i>J</i> = 8.3 Hz, 1H), 7.78 (s, 1H), 4.11 (s, 2H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 170.8, 166.8, 164.9, 143.4, 142.6, 137.3, 130.4, 129.9, 128.1, 127.1, 126.2, 125.6, 124.5, 35.9. HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>16</sub>H<sub>10</sub>NO<sub>4</sub>, 280.0615 [M – H]<sup>−</sup>; found, 280.0620.</div></div><div id="sec4_1_4_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> 3-(1,3-Dioxo-1,2,3,4-tetrahydroisoquinolin-6-yl)benzoic Acid (<b>69</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.16 (s, 1H), 11.32 (s, 1H), 8.26 (t, <i>J</i> = 1.6 Hz, 1H), 8.10 (d, <i>J</i> = 8.2 Hz, 1H), 8.00 (dd, <i>J</i> = 7.8, 1.8 Hz, 2H), 7.80 (dd, <i>J</i> = 8.2, 1.6 Hz, 1H), 7.76 (s, 1H), 7.65 (t, <i>J</i> = 7.8 Hz, 1H), 4.12 (s, 2H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 171.1, 167.2, 165.2, 143.9, 139.1, 137.7, 131.8, 131.5, 129.7, 129.3, 128.4, 127.7, 126.3, 125.8, 124.5, 36.2. HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>16</sub>H<sub>10</sub>NO<sub>4</sub>, 280.0615 [M – H]<sup>−</sup>; found, 280.0616.</div></div><div id="sec4_1_4_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> 3-(1,3-Dioxo-1,2,3,4-tetrahydroisoquinolin-6-yl)benzamide (<b>70</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.33 (s, 1H), 8.25 (s, 1H), 8.17 (s, 1H), 8.10 (d, <i>J</i> = 8.2 Hz, 1H), 7.93 (d, <i>J</i> = 7.7 Hz, 1H), 7.90 (d, <i>J</i> = 8.0 Hz, 1H), 7.84 (d, <i>J</i> = 8.2 Hz, 1H), 7.79 (s, 1H), 7.59 (t, <i>J</i> = 7.7 Hz, 1H), 7.48 (s, 1H), 4.11 (s, 2H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 175.7, 167.7, 163.3, 157.8, 144.9, 137.9, 135.5, 133.2, 130.1, 129.7, 129.3, 129.1, 128.5, 126.2, 124.7, 37.8. HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>16</sub>H<sub>11</sub>N<sub>2</sub>O<sub>3</sub>, 279.0775 [M – H]<sup>−</sup>; found, 279.0778.</div></div><div id="sec4_1_4_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> 6-(3-Acetylphenyl)isoquinoline-1,3(2<i>H</i>,4<i>H</i>)-dione (<b>71</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.16 (s, 1H), 8.09 (s, 1H), 7.94 (d, <i>J</i> = 8.1 Hz, 1H), 7.87–7.83 (m, 2H), 7.67 (d, <i>J</i> = 8.1 Hz, 1H), 7.63 (s, 1H), 7.50 (t, <i>J</i> = 7.7 Hz, 1H), 3.95 (s, 2H), 2.33 (s, 3H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 198.3, 171.4, 165.6, 144.3, 139.5, 138.0, 137.9, 132.0, 130.1, 128.7, 127.0, 126.7, 126.2, 125.6, 124.8, 36.6, 27.4. HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>17</sub>H<sub>12</sub>NO<sub>3</sub>, 278.0823 [M – H]<sup>−</sup>; found, 278.0816.</div></div><div id="sec4_1_4_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> 6-(3-Chlorophenyl)isoquinoline-1,3(2<i>H</i>,4<i>H</i>)-dione (<b>72</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.33 (s, 1H), 8.08 (d, <i>J</i> = 8.2 Hz, 1H), 7.82 (t, <i>J</i> = 1.8 Hz, 1H), 7.80 (d, <i>J</i> = 8.2 Hz, 1H), 7.76 (s, 1H), 7.73 (d, <i>J</i> = 7.7 Hz, 1H), 7.54 (t, <i>J</i> = 7.8 Hz, 1H), 7.51 (d, <i>J</i> = 8.5 Hz, 1H), 4.09 (s, 2H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 171.1, 165.2, 143.2, 140.9, 137.5, 134.1, 131.1, 128.5, 128.3, 126.9, 126.4, 125.8, 125.8, 124.6, 36.2. HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>15</sub>H<sub>9</sub>ClNO<sub>2</sub>, 270.0327 [M – H]<sup>−</sup>; found, 270.0330.</div></div><div id="sec4_1_4_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> 7-(3-Chlorophenyl)isoquinoline-1,3(2<i>H</i>,4<i>H</i>)-dione (<b>73</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.38 (s, 1H), 8.24 (d, <i>J</i> = 1.8 Hz, 1H), 8.00 (dd, <i>J</i> = 8.1, 1.8 Hz, 1H), 7.78 (d, <i>J</i> = 1.8 Hz, 1H), 7.70 (d, <i>J</i> = 7.8 Hz, 1H), 7.52 (t, <i>J</i> = 7.8 Hz, 1H), 7.50 (d, <i>J</i> = 8.1 Hz, 1H), 7.47 (d, <i>J</i> = 8.8 Hz, 1H), 4.08 (s, 2H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 171.1, 165.4, 141.3, 137.8, 136.8, 134.2, 132.1, 131.2, 129.1, 128.1, 126.7, 125.9, 125.7, 125.6, 36.1. HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>15</sub>H<sub>9</sub>ClNO<sub>2</sub>, 270.0327 [M – H]<sup>−</sup>; found, 270.0330.</div></div><div id="sec4_1_4_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> 6-(4-Chlorophenyl)isoquinoline-1,3(2<i>H</i>,4<i>H</i>)-dione (<b>74</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.33 (s, 1H), 8.08 (d, <i>J</i> = 8.2 Hz, 1H), 7.79–7.75 (m, 3H), 7.72 (s, 1H), 7.58 (d, <i>J</i> = 8.5 Hz, 2H), 4.09 (s, 2H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 171.2, 165.4, 143.7, 137.7, 133.8, 129.4, 129.1, 128.5, 126.2, 125.7, 124.5, 123.3, 36.4. HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>15</sub>H<sub>9</sub>BrNO<sub>2</sub>, 313.9822 [M – H]<sup>−</sup>; found, 313.9826.</div></div><div id="sec4_1_4_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> 6-(3-Bromophenyl)isoquinoline-1,3(2<i>H</i>,4<i>H</i>)-dione (<b>75</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.32 (s, 1H), 8.09–8.01 (m, 2H), 7.91–7.78 (m, 3H), 7.62–7.47 (m, 2H), 4.12 (s, 2H). HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>15</sub>H<sub>9</sub>ClNO<sub>2</sub>, 270.0327 [M – H]<sup>−</sup>; found, 270.0329.</div></div><div id="sec4_1_4_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> 7-(4-Bromophenyl)isoquinoline-1,3(2<i>H</i>, 4<i>H</i>)-dione (<b>76</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.38 (s, 1H), 8.23 (s, 1H), 7.96–7.82 (m, 2H), 7.69–7.64 (m, 4H), 4.07 (s, 2H). HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>15</sub>H<sub>9</sub>ClNO<sub>2</sub>, 270.0327 [M – H]<sup>−</sup>; found, 270.0326.</div></div><div id="sec4_1_4_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> 6-(2,6-Dimethoxyphenyl)isoquinoline-1,3(2<i>H</i>,4<i>H</i>)-dione (<b>77</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.23 (s, 1H), 7.99 (d, <i>J</i> = 8.0 Hz, 1H), 7.35 (t, <i>J</i> = 8.4 Hz, 1H), 7.28 (d, <i>J</i> = 8.0 Hz, 1H), 7.21 (s, 1H), 6.77 (d, <i>J</i> = 8.4 Hz, 2H), 4.02 (s, 2H), 3.66 (s, 6H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 171.7, 165.9, 157.6, 140.6, 136.5, 130.6, 130.4, 127.3, 124.0, 118.1, 117.8, 105.0, 56.4, 36.6. HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>17</sub>H<sub>14</sub>NO<sub>4</sub>, 296.0928 [M – H]<sup>−</sup>; found, 296.0931.</div></div><div id="sec4_1_4_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> 6-(2,5-Dimethoxyphenyl)isoquinoline-1,3(2<i>H</i>,4<i>H</i>)-dione (<b>78</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.27 (s, 1H), 8.01 (d, <i>J</i> = 8.1 Hz, 1H), 7.57 (dd, <i>J</i> = 8.1, 1.6 Hz, 1H), 7.48 (d, <i>J</i> = 1.6 Hz, 1H), 7.08 (d, <i>J</i> = 9.0 Hz, 1H), 6.96 (dd, <i>J</i> = 9.0, 3.1 Hz, 1H), 6.90 (d, <i>J</i> = 3.1 Hz, 1H), 4.05 (s, 2H), 3.75 (s, 3H), 3.71 (s, 3H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 171.2, 165.3, 153.5, 150.4, 143.3, 136.5, 129.3, 128.6, 128.4, 127.2, 123.7, 116.1, 114.6, 113.4, 56.3, 55.7, 36.2. HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>17</sub>H<sub>14</sub>NO<sub>4</sub>, 296.0928 [M – H]<sup>−</sup>; found, 296.0931.</div></div><div id="sec4_1_4_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> 6-(3-Fluoro-4-methoxyphenyl)isoquinoline-1,3(2<i>H</i>,4<i>H</i>)-dione (<b>79</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.29 (s, 1H), 8.03 (d, <i>J</i> = 8.2 Hz, 1H), 7.75 (dd, <i>J</i> = 8.2, 1.7 Hz, 1H), 7.70 (s, 1H), 7.66 (dd, <i>J</i> = 12.9, 2.2 Hz, 1H), 7.63–7.52 (m, 1H), 7.29 (t, <i>J</i> = 8.8 Hz, 1H), 4.06 (s, 2H), 3.89 (s, 3H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 171.4, 165.6, 152.9, 151.4, 148.0, 147.9, 143.6, 137.7, 131.7, 128.5, 125.9, 124.1, 123.7, 114.9, 56.6, 36.6. HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>16</sub>H<sub>11</sub>FNO<sub>3</sub>, 284.0728 [M – H]<sup>−</sup>; found, 284.0726.</div></div><div id="sec4_1_4_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> 6-(3-Chloro-4-fluorophenyl)isoquinoline-1,3(2<i>H</i>,4<i>H</i>)-dione (<b>80</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.35 (s, 1H), 8.14–7.88 (m, 2H), 7.87–7.64 (m, 3H), 7.58 (s, 1H), 4.10 (s, 2H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 171.4, 165.5, 142.7, 137.8, 136.9, 133.9, 129.5, 128.6, 128.2, 128.2, 126.6, 126.0, 124.8, 117.8, 36.5. HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>15</sub>H<sub>8</sub>ClFNO<sub>2</sub>, 288.0233 [M – H]<sup>−</sup>; found, 288.0237.</div></div><div id="sec4_1_4_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> 6-(2,4-Difluorophenyl)isoquinoline-1,3(2<i>H</i>,4<i>H</i>)-dione (<b>81</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.34 (s, 1H), 8.09 (d, <i>J</i> = 8.1 Hz, 1H), 7.65 (dd, <i>J</i> = 12.1, 5.4 Hz, 1H), 7.63–7.60 (m, 1H), 7.56 (s, 1H), 7.47–7.39 (m, 1H), 7.25 (td, <i>J</i> = 8.4, 2.3 Hz, 1H), 4.09 (s, 2H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 170.7, 164.8, 158.1, 138.9, 136.9, 131.8, 127.9, 127.6, 127.4, 124.2, 112.2, 112.1, 104.7, 104.5, 35.8. HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>15</sub>H<sub>8</sub>F<sub>2</sub>NO<sub>2</sub>, 272.0529 [M – H]<sup>−</sup>; found, 272.0528.</div></div><div id="sec4_1_4_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> 6-(4-(Trifluoromethyl)phenyl)isoquinoline-1,3(2<i>H</i>,4<i>H</i>)-dione (<b>82</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.36 (s, 1H), 8.12 (d, <i>J</i> = 8.2 Hz, 1H), 7.97 (d, <i>J</i> = 8.2 Hz, 2H), 7.87 (d, <i>J</i> = 8.2 Hz, 2H), 7.83 (dd, <i>J</i> = 8.2, 1.7 Hz, 1H), 7.79 (s, 1H), 4.11 (s, 2H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 171.2, 165.3, 143.4, 142.9, 137.8, 128.5, 128.4, 128.2, 126.8, 126.2, 126.2, 125.1, 124.9, 36.4. HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>16</sub>H<sub>9</sub>F<sub>3</sub>NO<sub>2</sub>, 304.0591 [M – H]<sup>−</sup>; found, 304.0586.</div></div><div id="sec4_1_4_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> 6-([1,1′-Biphenyl]-3-yl)isoquinoline-1,3(2<i>H</i>, 4<i>H</i>)-dione (<b>83</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.32 (s, 1H), 8.10 (d, <i>J</i> = 8.3 Hz, 1H), 7.99 (s, 1H), 7.88 (d, <i>J</i> = 7.8 Hz, 1H), 7.84 (s, 1H), 7.78 (d, <i>J</i> = 7.6 Hz, 2H), 7.74 (t, <i>J</i> = 7.2 Hz, 2H), 7.61 (t, <i>J</i> = 7.8 Hz, 1H), 7.51 (t, <i>J</i> = 7.6 Hz, 2H), 7.42 (d, <i>J</i> = 7.2 Hz, 1H), 4.11 (s, 2H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 175.8, 163.6, 158.1, 145.7, 141.8, 140.1, 139.0, 133.3, 130.4, 129.3, 129.1, 127.6, 127.4, 126.1, 125.9, 124.8, 124.6, 110.0, 35.8. HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>21</sub>H<sub>14</sub>NO<sub>2</sub>, 312.1030 [M – H]<sup>−</sup>; found, 312.1020.</div></div><div id="sec4_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> Synthesis of Benzothiazole (<b>84</b>)</h4><div class="NLM_p last">To the mixture of <i>o</i>-phenylenediamine (1 mmol) was added carboxylic acid <b>39</b> (1.0 mmol), <i>N</i>,<i>N</i>-diisopropylethylamine (1.5 mmol), and propylphosphonic anhydride (1.0 mmol, 50% w/w in EtOAc), and the resulting solution was irradiated at 100 °C for 30 min under microwave conditions before it was diluted with H<sub>2</sub>O. The precipitate was collected via filtration and washed thoroughly with H<sub>2</sub>O. Recrystallization of the crude sample in a mixture of EtOH/H<sub>2</sub>O furnished the title compound as pale-brown solid. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.43 (s, 1H), 8.22 (d, <i>J</i> = 7.9 Hz, 1H), 8.18 (d, <i>J</i> = 8.1 Hz, 1H), 8.17–8.10 (m, 3H), 7.60 (t, <i>J</i> = 7.6 Hz, 1H), 7.52 (t, <i>J</i> = 7.6 Hz, 1H), 4.18 (s, 2H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 170.6, 165.6, 164.5, 153.2, 137.7, 136.5, 134.6, 128.4, 126.8, 126.8, 126.1, 125.9, 125.5, 123.0, 122.4, 35.8. HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>16</sub>H<sub>9</sub>N<sub>2</sub>O<sub>2</sub>S, 293.0390 [M – H]<sup>−</sup>; found, 293.0384.</div></div><div id="sec4_1_5_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72"> 6-(Quinolin-3-yl)isoquinoline-1,3(2<i>H</i>,4<i>H</i>)-dione (<b>85</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.37 (s, 1H), 9.32 (d, <i>J</i> = 2.2 Hz, 1H), 8.78 (d, <i>J</i> = 2.2 Hz, 1H), 8.16 (d, <i>J</i> = 8.3 Hz, 1H), 8.11–8.06 (m, 2H), 8.00 (d, <i>J</i> = 8.3 Hz, 1H), 7.97 (s, 1H), 7.82 (t, <i>J</i> = 7.6 Hz, 1H), 7.69 (t, <i>J</i> = 7.6 Hz, 1H), 4.14 (s, 2H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 175.3, 163.0, 157.6, 148.9, 146.9, 142.2, 134.8, 133.2, 133.1, 130.7, 130.6, 129.2, 129.2, 128.9, 128.36, 127.6, 124.8, 39.1. HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>18</sub>H<sub>11</sub>N<sub>2</sub>O<sub>2</sub>, 287.0826 [M – H]<sup>−</sup>; found, 287.0826.</div></div><div id="sec4_1_5_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i73" class="anchor-spacer"></div><h4 class="article-section__title" id="_i73"> 6-Benzylisoquinoline-1,3(2<i>H</i>,4<i>H</i>)-dione (<b>86</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.20 (s, 1H), 7.92 (d, <i>J</i> = 8.0 Hz, 1H), 7.33–7.28 (m, 3H), 7.25 (d, <i>J</i> = 6.2 Hz, 3H), 7.19 (t, <i>J</i> = 7.2 Hz, 1H), 4.00 (s, 2H), 3.97 (s, 2H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 171.4, 165.6, 147.8, 135.6, 129.3, 129.3, 129.1, 129.0, 128.3, 128.2, 126.9, 126.7, 41.4, 36.4. HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>16</sub>H<sub>12</sub>NO<sub>2</sub>, 250.0874 [M – H]<sup>−</sup>; found, 250.0868.</div></div><div id="sec4_1_5_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i74" class="anchor-spacer"></div><h4 class="article-section__title" id="_i74"> 6-(3-Nitrobenzoyl)isoquinoline-1,3(2<i>H</i>,4<i>H</i>)-dione (<b>87</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.49 (s, 1H), 8.54 (dd, <i>J</i> = 7.9, 1.8 Hz, 1H), 8.47–8.44 (m, 1H), 8.20 (s, 1H), 8.18 (s, 1H), 7.89 (t, <i>J</i> = 8.0 Hz, 1H), 7.83–7.79 (m, 2H), 4.13 (s, 2H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 193.8, 171.1, 165.1, 148.2, 140.2, 138.1, 137.7, 136.2, 131.0, 129.5, 128.9, 128.4, 128.3, 127.8, 124.6, 36.5. HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>16</sub>H<sub>9</sub>N<sub>2</sub>O<sub>5</sub>, 309.0517 [M – H]<sup>−</sup>; found, 309.0510.</div></div><div id="sec4_1_5_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i75" class="anchor-spacer"></div><h4 class="article-section__title" id="_i75"> 7-(3-Nitrobenzoyl)isoquinoline-1,3(2<i>H</i>,4<i>H</i>)-dione (<b>88</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.49 (s, 1H), 8.53 (d, <i>J</i> = 8.5 Hz, 1H), 8.45 (s, 1H), 8.32 (s, 1H), 8.17 (d, <i>J</i> = 7.6 Hz, 1H), 8.06 (d, <i>J</i> = 7.9 Hz, 1H), 7.89 (t, <i>J</i> = 8.0 Hz, 1H), 7.61 (d, <i>J</i> = 7.9 Hz, 1H), 4.17 (s, 2H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 192.7, 170.3, 164.5, 147.6, 141.8, 137.9, 135.4, 134.8, 133.8, 130.4, 128.7, 128.5, 126.9, 125.3, 123.8, 36.2. HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>16</sub>H<sub>9</sub>N<sub>2</sub>O<sub>5</sub>, 309.0517 [M – H]<sup>−</sup>; found, 309.0523.</div></div><div id="sec4_1_5_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i76" class="anchor-spacer"></div><h4 class="article-section__title" id="_i76"> Ethyl-6-bromo-1,3-dioxo-1,2,3,4-tetrahydroisoquinoline-4-carboxylate (<b>89</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.63 (s, 1H), 11.37 (s, 1H, enol isomer), 8.63 (s, 1H), 8.04 (d, <i>J</i> = 8.4 Hz, 1H), 7.91 (d, <i>J</i> = 8.4 Hz, 1H, enol isomer), 7.66–7.61 (m, 2H, enol isomer), 7.51 (d, <i>J</i> = 8.0 Hz, 1H), 4.47 (q, <i>J</i> = 6.9 Hz, 2H), 4.03 (s, 1H), 1.42 (t, <i>J</i> = 6.9 Hz, 3H). HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>12</sub>H<sub>9</sub>BrNO<sub>4</sub>, 309.9720 [M – H]<sup>−</sup>; found, 309.9722.</div></div><div id="sec4_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i77" class="anchor-spacer"></div><h4 class="article-section__title" id="_i77"> Synthesis of Compound <b>90</b></h4><div class="NLM_p">The mixture of methyl-4-bromo-2-aminobenzoate (<b>27</b>,1.0 equiv), phenylboronic acid (1.6 equiv), Pd(PPh<sub>3</sub>)<sub>4</sub> (0.065 equiv), and K<sub>2</sub>CO<sub>3</sub> (3.6 equiv) in DME/H<sub>2</sub>O (4:1) was irradiated at 110 °C for 45 min under microwave conditions. The black residue formed was filtered through Celite, and the solvent was concentrated in vacuo. The crude mixture was purified using CombiFlash with 0–50% EtOAc in hexane as an eluent to yield the desired product (<b>28</b>) as colorless solid. Yield: 50%.</div><div class="NLM_p last">To a solution of potassium isocyanate (1 mmol) in 1 mL of dry toluene was added compound <b>28</b>, and the resulting mixture was stirred at rt for 3 h before water was added. The aqueous solution was extracted with DCM (2 × 10 mL), and the combined organics were washed with NaHCO<sub>3</sub> (2 × 10 mL) and brine (15 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated in vacuo to leave colorless oil. To the resulting oil in EtOH (2 mL) was added 1N NaOH (2 mL), and the reaction mixture was stirred under reflux for 1 h before it was acidified (pH = 3) using 1 N HCl. The precipitate was collected via filtration, washed with water, and then dried to obtain the titled compound (<b>90</b>) as colorless solid. Yield: 68%. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.30 (s, 1H), 11.20 (s, 1H), 7.96 (d, <i>J</i> = 8.2 Hz, 1H), 7.67 (s, 1H), 7.66 (s, 1H), 7.53 (t, <i>J</i> = 7.6 Hz, 2H), 7.49–7.44 (m, 2H), 7.38 (d, <i>J</i> = 1.3 Hz, 1H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 162.8, 150.5, 146.7, 141.5, 138.9, 129.3, 127.9, 127.8, 127.1, 113.5, 113.3, 113.0, 39.2. HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>14</sub>H<sub>9</sub>N<sub>2</sub>O<sub>2</sub>, 237.0670 [M – H]<sup>−</sup>; found, 237.0672.</div></div><div id="sec4_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i78" class="anchor-spacer"></div><h4 class="article-section__title" id="_i78"> Synthesis of Naphthalimides <b>91</b>–<b>92</b></h4><div class="NLM_p">A mixture of 4-bromo-1,8-naphthalic anhydride (<b>29</b>, 1.0 equiv), 2-(tributylstannyl)furan (1.2 equiv), and Pd(PPh<sub>3</sub>)<sub>4</sub> (0.03 equiv) in toluene (5.0 mL) was stirred at 150 °C for 30 min under microwave conditions. The black residue formed was filtered through Celite, and the solvent was concentrated in vacuo. The crude mixture was purified using CombiFlash with 0–30% EtOAc in hexane as an eluent to yield 4-furylnaphthalic anhydride (<b>30</b>) as colorless solid. Yield: 70%.</div><div class="NLM_p">A mixture of <b>29</b>/<b>30</b> (0.72 mmol) in ammonium hydroxide (15 mL) was stirred at 40 °C for 4 h. The resulting solution was extracted with EtOAc (2 × 20 mL), and the combined organic extracts were washed with H<sub>2</sub>O (2 × 20 mL) and brine (20 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated in vacuo. Recrystallization of the crude sample from EtOH produced desired compounds (<b>91</b>–<b>92</b>) as colorless solid. Yield: 70–90%.</div><div id="sec4_1_7_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i79" class="anchor-spacer"></div><h5 class="article-section__title" id="_i79"> 6-Bromo-1<i>H</i>-benzo[<i>d</i>]isoquinoline-1,3(2<i>H</i>)-dione (<b>91</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.84 (s, 1H), 8.53–8.48 (m, 2H), 8.25 (d, <i>J</i> = 7.8 Hz, 1H), 8.18 (d, <i>J</i> = 7.8 Hz, 1H), 7.96 (dd, <i>J</i> = 8.3, 7.4 Hz, 1H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 163.7, 163.6, 132.7, 131.4, 130.9, 130.4, 130.2, 129.7, 129.2, 128.8, 123.4, 122.6. HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>12</sub>H<sub>5</sub>BrNO<sub>2</sub>, 273.9509 [M – H]<sup>−</sup>; found, 273.9504.</div></div><div id="sec4_1_7_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i80" class="anchor-spacer"></div><h5 class="article-section__title" id="_i80"> 6-(Furan-2-yl)-1<i>H</i>-benzo[<i>d</i>]isoquinoline-1,3(2<i>H</i>)-dione (<b>92</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.80 (s, 1H), 8.93 (d, <i>J</i> = 8.6 Hz, 1H), 8.52 (d, <i>J</i> = 7.1 Hz, 1H), 8.48 (d, <i>J</i> = 7.7 Hz, 1H), 8.12 (d, <i>J</i> = 7.7 Hz, 1H), 8.07 (d, <i>J</i> = 1.8 Hz, 1H), 7.98–7.87 (m, 1H), 7.30 (d, <i>J</i> = 3.4 Hz, 1H), 6.83 (dd, <i>J</i> = 3.4, 1.8 Hz, 1H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 164.5, 164.1, 151.2, 145.5, 133.7, 132.3, 130.6, 130.1, 129.8, 128.1, 127.8, 125.9, 123.1, 113.3, 112.9, 39.1. HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>16</sub>H<sub>8</sub>NO<sub>3</sub>, 262.0510 [M – H]<sup>−</sup>; found, 262.0512.</div></div></div><div id="sec4_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i81" class="anchor-spacer"></div><h4 class="article-section__title" id="_i81"> Synthesis of <b>93</b>–<b>95</b></h4><div class="NLM_p">To a suspension of 2-amino-1-propene-1,1,3-tricarbonitrile (<b>31</b>, 7.6 mmol, 1.0 equiv) in EtOH (10 mL) was added arylhydrazine (8.4 mmol, 1.1 equiv) slowly and the solution was heated at reflux for 30 min before it was cooled. The solvent was removed in vacuo to leave pale-brown solid. Recrystallization of the crude mixture from EtOH furnished intermediates <b>32</b> as brown needles. Yield: 70–95%.</div><div class="NLM_p">A solution of compound <b>32</b> (0.45 mmol) in conc HCl (2 mL) was stirred at 70 °C for 4 h. After the solvent was removed in vacuo, the residue was purified using CombiFlash with 0–100% EtOAc in hexane as an eluent to yield desired products (<b>93</b>–<b>95</b>) as colorless solid. Yield: 65–85%.</div><div id="sec4_1_8_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i82" class="anchor-spacer"></div><h5 class="article-section__title" id="_i82"> 3-Amino-2-(4-fluorophenyl)-2<i>H</i>-pyrazolo[4,3-<i>c</i>]pyridine-4,6(5<i>H</i>,7<i>H</i>)dione (<b>93</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.60 (s, 1H), 7.55 (dd, <i>J</i> = 8.9, 4.9 Hz, 2H), 7.36 (t, <i>J</i> = 8.8 Hz, 2H), 6.48 (s, 2H), 3.75 (s, 2H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 171.4, 162.5, 160.5, 152.9, 148.4, 148.2, 134.2, 126.6, 126.5, 116.7, 116.5, 92.9, 32.2. HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>12</sub>H<sub>8</sub>FN<sub>4</sub>O<sub>2</sub>, 259.0637 [M – H]<sup>−</sup>; found, 259.0636.</div></div><div id="sec4_1_8_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i83" class="anchor-spacer"></div><h5 class="article-section__title" id="_i83"> 3-Amino-2-(pyridin-2-yl)-2<i>H</i>-pyrazolo[4,3-<i>c</i>]pyridine-4,6(5<i>H</i>,7<i>H</i>)dione (<b>94</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.64 (s, 1H), 8.51–8.43 (m, 1H), 8.02–7.96 (m, 1H), 7.82 (d, <i>J</i> = 8.4 Hz, 1H), 7.77 (s, 2H), 7.35–7.29 (m, 1H), 3.80 (s, 2H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 171.1, 162.2, 153.8, 149.7, 149.6, 147.5, 140.0, 121.1, 113.2, 92.8, 32.2. HRMS-ESI (+) <i>m</i>/<i>z</i> calculated for C<sub>11</sub>H<sub>10</sub>N<sub>5</sub>O<sub>2</sub>, 244.0829 [M + H]<sup>−</sup>; found, 244.0832.</div></div><div id="sec4_1_8_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i84" class="anchor-spacer"></div><h5 class="article-section__title" id="_i84"> 3-Amino-2-phenyl-2<i>H</i>-pyrazolo[4,3-<i>c</i>]pyridine-4,6(5<i>H</i>,7<i>H</i>)dione (<b>95</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.60 (s, 1H), 7.76–7.45 (m, 4H), 7.41 (dd, <i>J</i> = 5.6, 3.0 Hz, 1H), 6.45 (s, 2H), 3.76 (s, 2H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 170.8, 161.9, 147.8, 147.4, 137.3, 129.2, 123.2, 92.5, 31.7. HRMS-ESI (+) <i>m</i>/<i>z</i> calculated for C<sub>12</sub>H<sub>11</sub>N<sub>4</sub>O<sub>2</sub>, 243.0882 [M + H]<sup>−</sup>; found, 243.0890.</div></div><div id="sec4_1_8_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i85" class="anchor-spacer"></div><h5 class="article-section__title" id="_i85"> <i>N</i>-(4,6-Dioxo-2-phenyl-4,5,6,7-tetrahydro-2<i>H</i>-pyrazolo[4,3-<i>c</i>]pyridine-3-yl)acetamide (<b>96</b>)</h5><div class="NLM_p last">To a solution of compound <b>95</b> (1.0 equiv) in AcOH (5.0 mL) was added acetic anhydride (1.2 equiv) and stirred at 70 °C for 5 h. The solvent was evaporated in vacuo to leave pale solid. Recrystallization of crude sample from MeOH produced the title compound (<b>96</b>) as colorless solid. Yield: 62%. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.94 (s, 1H), 10.32 (s, 1H), 7.64–7.36 (m, 5H), 3.95 (s, 2H), 1.97 (s, 3H). HRMS-ESI (−) <i>m</i>/<i>z</i> calculated for C<sub>14</sub>H<sub>11</sub>N<sub>4</sub>O<sub>3</sub>, 283.0837 [M – H]<sup>−</sup>; found, 283.0831.</div></div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i86" class="anchor-spacer"></div><h3 class="article-section__title" id="_i86"> Biology</h3><div id="sec4_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i87" class="anchor-spacer"></div><h4 class="article-section__title" id="_i87"> Recombinant TDP1 Assay</h4><div class="NLM_p last">A 5′-[<sup>32</sup>P]-labeled single-stranded DNA oligonucleotide containing a 3′-phosphotyrosine (N14Y)<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> was incubated at 1 nM with 10 pM recombinant TDP1 in the absence or presence of inhibitor for 15 min at room temperature in WCE buffer. When indicated, parallel reactions were performed in the LMP1 assay buffer containing 50 mM Tris HCl, pH 7.5, 80 mM KCl, 2 mM EDTA, 1 mM DTT, 40 μg/mL BSA, and 0.01% Tween-20.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Reactions were terminated by the addition of 1 volume of gel loading buffer [99.5% (v/v) formamide, 5 mM EDTA, 0.01% (w/v) xylene cyanol, and 0.01% (w/v) bromophenol blue]. Samples were subjected to a 16% denaturing PAGE with multiple loadings at 12 min intervals. Gels were dried and exposed to a PhosphorImager screen (GE Healthcare). Gel images were scanned using a Typhoon 8600 (GE Healthcare), and densitometry analyses were performed using the ImageQuant software (GE Healthcare).</div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i88" class="anchor-spacer"></div><h4 class="article-section__title" id="_i88"> Whole Cell Extract TDP1 Assay</h4><div class="NLM_p last">DT40 knockout cells (1 × 10<sup>7</sup>) for TDP1 (TDP1–/−) complemented with human TDP1 (hTDP1) were collected, washed, and centrifuged. Cell pellets were then resuspended in 100 μL of CellLytic M cell lysis reagent (SIGMA-Aldrich C2978). After 15 min on ice, lysates were centrifuged at 12000<i>g</i> for 10 min, and supernatants were transferred to a new tube. Protein concentrations were determined using a Nanodrop spectrophotometer (Invitrogen), and whole cell extracts were stored at −80 °C. The N14Y DNA substrate was incubated at 1 nM with 5 μg/mL of whole cell extracts in the absence or presence of inhibitor for 15 min at room temperature in the LMP1 assay buffer. Samples were then analyzed similarly to the recombinant TDP1 assay (see above).</div></div><div id="sec4_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i89" class="anchor-spacer"></div><h4 class="article-section__title" id="_i89"> Recombinant TDP2 Assay</h4><div class="NLM_p last">TDP2 reactions were carried out as described previously<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> with the following modifications. The 18-mer single-stranded oligonucleotide DNA substrate (TY18, α<sup>32</sup>P-cordycepin-3′-labeled) was incubated at 1 nM with 25 pM recombinant human TDP2 in the absence or presence of inhibitor for 15 min at room temperature in the LMP2 assay buffer containing 50 mM Tris-HCl, pH 7.5, 80 mM KCl, 5 mM MgCl<sub>2</sub>, 0.1 mM EDTA, 1 mM DTT, 40 μg/mL BSA, and 0.01% Tween 20. Reactions were terminated and treated similarly to WCE and recombinant TDP1 reactions (see above).</div></div><div id="sec4_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i90" class="anchor-spacer"></div><h4 class="article-section__title" id="_i90"> Whole Cell Extract TDP2 Assay</h4><div class="NLM_p last">DT40 knockout cells (1 × 10<sup>7</sup>) for TDP1 (TDP2–/−) complemented with human TDP2 (hTDP2) were collected, washed, and centrifuged. hTDP2 whole cell extracts were prepared and stored similarly to hTDP1 whole cell extracts. The TY18 DNA substrate was incubated at 1 nM with 5 μg/mL of whole cell extracts in the absence or presence of inhibitor for 15 min at room temperature in the LMP2 assay buffer. Reactions were terminated and treated similarly to WCE and recombinant TDP1 reactions (see above).</div></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i91" class="anchor-spacer"></div><h3 class="article-section__title" id="_i91"> Molecular Modeling</h3><div class="NLM_p">Molecular modeling was performed using the Schrodinger small molecule drug discovery suite 2014-3. Homology models generated from templates having a higher degree of sequence identity with the target are found to be reliable and accurate for the utilization in structure-based drug design. Because of the high sequence homology between hTDP2 and mTDP2 (65%), an available crystal structure of mTDP2 (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GZ1">4GZ1</a><a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a>) was used to build a homology model of hTDP2. The sequence of hTDP2 was imported into the program Prime<a onclick="showRef(event, 'ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref28 ref29">(28, 29)</a> (Schrödinger Inc.), and the model was built using mTDP2 (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GZ1">4GZ1</a><a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a>) as a template. This model was subjected to the loop refinement and energy minimization to improve accuracy and quality of the model. This model was subjected to protein preparation wizard<a onclick="showRef(event, 'ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref30 ref31">(30, 31)</a> (Schrodinger Inc.) in which missing hydrogens were added and zero-order bonds to metals were created followed by the generation of metal binding states. The structure of protein was minimized using OPLS 2005 force field<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> to optimize hydrogen bonding network and converge heavy atoms to the RMSD of 0.3 Å.</div><div class="NLM_p last">The receptor grid generation tool in Maestro (Schrodinger Inc.) was used to define an active site around the metal (Mg<sup>2+</sup>) to cover all the residues within 12 Å from it. Compound <b>40</b> and <b>56</b> were drawn using Maestro and subjected to Lig Prep<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> to generate conformers, possible protonation at pH of 7 ± 3, and metal binding states which serves as an input for docking process. All the dockings were performed using Glide XP<a onclick="showRef(event, 'ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref23 ref24">(23, 24)</a> (Glide, version 6.4) mode. The van der Waals radii of nonpolar atoms for each of the ligands were scaled by a factor of 0.8. The predicted favorable binding mode of compounds <b>40</b> and <b>56</b> features the critical interaction between the ligands and E152, T230, D262, N264, R266, H351 (residues are labeled according to hTDP2 sequence), and the metal ion within the active site of hTDP2. All the ligands docked were further refined post docking by minimizing under implicit solvent to account for the local protein flexibility.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i92"><a href="/doi/suppl/10.1021/acs.jmedchem.5b01973">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78152" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78152" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.5b01973" class="ext-link">10.1021/acs.jmedchem.5b01973</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Characterization data, including <sup>1</sup>H NMR data of intermediate chemotypes <b>21</b>, <b>24</b>, <b>25</b>, and <b>32</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01973/suppl_file/jm5b01973_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01973/suppl_file/jm5b01973_si_003.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01973/suppl_file/jm5b01973_si_001.pdf">jm5b01973_si_001.pdf (385.1 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01973/suppl_file/jm5b01973_si_003.csv">jm5b01973_si_003.csv (3.67 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.5b01973" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_88868" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_88868" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zhengqiang Wang</span> - <span class="hlFld-Affiliation affiliation">Center
for Drug Design, Academic Health Center, University of Minnesota, Minneapolis, Minnesota 55455, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#3c4b5d525b44080b0e7c49515212595849"><span class="__cf_email__" data-cfemail="087f69666f703c3f3a487d6566266d6c7d">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jayakanth Kankanala</span> - <span class="hlFld-Affiliation affiliation">Center
for Drug Design, Academic Health Center, University of Minnesota, Minneapolis, Minnesota 55455, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christophe Marchand</span> - <span class="hlFld-Affiliation affiliation">Developmental
Therapeutics Branch, Center for Cancer Research, National Cancer Institute, National Institutes of Health, Bethesda, Maryland 20892, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Monica Abdelmalak</span> - <span class="hlFld-Affiliation affiliation">Developmental
Therapeutics Branch, Center for Cancer Research, National Cancer Institute, National Institutes of Health, Bethesda, Maryland 20892, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hideki Aihara</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, Molecular Biology and Biophysics, University of Minnesota, Minneapolis, Minnesota 55455, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yves Pommier</span> - <span class="hlFld-Affiliation affiliation">Developmental
Therapeutics Branch, Center for Cancer Research, National Cancer Institute, National Institutes of Health, Bethesda, Maryland 20892, United States</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e5810-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i93">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_25636" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_25636" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This research was supported by the Faculty Research Development Grant (to H.A. and Z.W.) of the Academic Health Center, University of Minnesota, and in part by the Intramural Research Program of the NIH, Center for Cancer Research, National Cancer Institute (Z01 BC 006161-17), and the Minnesota Supercomputing Institute (MSI) for computational resources involving modeling.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i94" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i94"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i95" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i95"> Abbreviations Used</h2><tr><td class="NLM_term">TDP2</td><td class="NLM_def"><p class="first last">tyrosyl DNA phosphodiesterase 2</p></td></tr><tr><td class="NLM_term">TDP1</td><td class="NLM_def"><p class="first last">tyrosyl DNA phosphodiesterase 1</p></td></tr><tr><td class="NLM_term">Top2</td><td class="NLM_def"><p class="first last">topoisomerase II</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">ETP</td><td class="NLM_def"><p class="first last">etoposide</p></td></tr><tr><td class="NLM_term">HBV</td><td class="NLM_def"><p class="first last">hepatitis B virus</p></td></tr><tr><td class="NLM_term">cccDNA</td><td class="NLM_def"><p class="first last">covalently closed circular DNA</p></td></tr><tr><td class="NLM_term">RC-DNA</td><td class="NLM_def"><p class="first last">relaxed circular DNA</p></td></tr><tr><td class="NLM_term">EEP</td><td class="NLM_def"><p class="first last">the exonuclease–endonuclease phosphatase</p></td></tr><tr><td class="NLM_term">hAPE1</td><td class="NLM_def"><p class="first last">human apurinic/apyrimidinic endonuclease 1</p></td></tr><tr><td class="NLM_term">HTS</td><td class="NLM_def"><p class="first last">high-throughput screening</p></td></tr><tr><td class="NLM_term">PK</td><td class="NLM_def"><p class="first last">pharmacokinetics</p></td></tr><tr><td class="NLM_term">WCE</td><td class="NLM_def"><p class="first last">whole cell extract</p></td></tr><tr><td class="NLM_term">HID</td><td class="NLM_def"><p class="first last">2-hydroxyisoquinoline-1,3-dione</p></td></tr><tr><td class="NLM_term">MW</td><td class="NLM_def"><p class="first last">microwave irradiation</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i96">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_76425" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_76425" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 33 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Helleday, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petermann, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lundin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hodgson, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharma, R. A.</span><span> </span><span class="NLM_article-title">DNA repair pathways as targets for cancer therapy</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">193</span><span class="NLM_x">–</span> <span class="NLM_lpage">204</span><span class="refDoi"> DOI: 10.1038/nrc2342</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=10.1038%2Fnrc2342" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=18256616" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=1%3ACAS%3A528%3ADC%252BD1cXitlOisr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2008&pages=193-204&author=T.+Helledayauthor=E.+Petermannauthor=C.+Lundinauthor=B.+Hodgsonauthor=R.+A.+Sharma&title=DNA+repair+pathways+as+targets+for+cancer+therapy&doi=10.1038%2Fnrc2342"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">DNA repair pathways as targets for cancer therapy</span></div><div class="casAuthors">Helleday, Thomas; Petermann, Eva; Lundin, Cecilia; Hodgson, Ben; Sharma, Ricky A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">193-204</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  DNA repair pathways can enable tumor cells to survive DNA damage that is induced by chemotherapeutic treatments; therefore, inhibitors of specific DNA repair pathways might prove efficacious when used in combination with DNA-damaging chemotherapeutic drugs.  In addn., alterations in DNA repair pathways that arise during tumor development can make some cancer cells reliant on a reduced set of DNA repair pathways for survival.  There is evidence that drugs that inhibit one of these pathways in such tumors could prove useful as single-agent therapies, with the potential advantage that this approach could be selective for tumor cells and have fewer side effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3W53LJmdCPrVg90H21EOLACvtfcHk0lhSlZ0zlYKumA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXitlOisr8%253D&md5=8592af6d815e5ca9c80ee256791bafb7</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrc2342&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2342%26sid%3Dliteratum%253Aachs%26aulast%3DHelleday%26aufirst%3DT.%26aulast%3DPetermann%26aufirst%3DE.%26aulast%3DLundin%26aufirst%3DC.%26aulast%3DHodgson%26aufirst%3DB.%26aulast%3DSharma%26aufirst%3DR.%2BA.%26atitle%3DDNA%2520repair%2520pathways%2520as%2520targets%2520for%2520cancer%2520therapy%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2008%26volume%3D8%26spage%3D193%26epage%3D204%26doi%3D10.1038%2Fnrc2342" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Nitiss, J. L.</span><span> </span><span class="NLM_article-title">Targeting DNA topoisomerase II in cancer chemotherapy</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">338</span><span class="NLM_x">–</span> <span class="NLM_lpage">350</span><span class="refDoi"> DOI: 10.1038/nrc2607</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=10.1038%2Fnrc2607" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=19377506" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=1%3ACAS%3A528%3ADC%252BD1MXks12kur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2009&pages=338-350&author=J.+L.+Nitiss&title=Targeting+DNA+topoisomerase+II+in+cancer+chemotherapy&doi=10.1038%2Fnrc2607"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting DNA topoisomerase II in cancer chemotherapy</span></div><div class="casAuthors">Nitiss, John L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">338-350</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Recent mol. studies have expanded the biol. contexts in which topoisomerase II (TOP2) has crucial functions, including DNA replication, transcription and chromosome segregation.  Although the biol. functions of TOP2 are important for ensuring genomic integrity, the ability to interfere with TOP2 and generate enzyme-mediated DNA damage is an effective strategy for cancer chemotherapy.  The mol. tools that have allowed an understanding of the biol. functions of TOP2 are also being applied to understanding the details of drug action.  These studies promise refined targeting of TOP2 as an effective anticancer strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_7Nq6nIe1RbVg90H21EOLACvtfcHk0lhSlZ0zlYKumA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXks12kur0%253D&md5=a787de602687580188cda6a2af7b6a07</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrc2607&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2607%26sid%3Dliteratum%253Aachs%26aulast%3DNitiss%26aufirst%3DJ.%2BL.%26atitle%3DTargeting%2520DNA%2520topoisomerase%2520II%2520in%2520cancer%2520chemotherapy%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2009%26volume%3D9%26spage%3D338%26epage%3D350%26doi%3D10.1038%2Fnrc2607" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Ledesma, F. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">El Khamisy, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zuma, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Osborn, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caldecott, K. W.</span><span> </span><span class="NLM_article-title">A human 5′-tyrosyl DNA phosphodiesterase that repairs topoisomerase-mediated DNA damage</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">461</span><span class="NLM_x">, </span> <span class="NLM_fpage">674</span><span class="NLM_x">–</span> <span class="NLM_lpage">U125</span><span class="refDoi"> DOI: 10.1038/nature08444</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=10.1038%2Fnature08444" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=19794497" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1SqtrzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=461&publication_year=2009&pages=674-U125&author=F.+C.+Ledesmaauthor=S.+F.+El+Khamisyauthor=M.+C.+Zumaauthor=K.+Osbornauthor=K.+W.+Caldecott&title=A+human+5%E2%80%B2-tyrosyl+DNA+phosphodiesterase+that+repairs+topoisomerase-mediated+DNA+damage&doi=10.1038%2Fnature08444"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">A human 5'-tyrosyl DNA phosphodiesterase that repairs topoisomerase-mediated DNA damage</span></div><div class="casAuthors">Ledesma, Felipe Cortes; El Khamisy, Sherif F.; Zuma, Maria C.; Osborn, Kay; Caldecott, Keith W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">461</span>
        (<span class="NLM_cas:issue">7264</span>),
    <span class="NLM_cas:pages">674-678</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Topoisomerases regulate DNA topol. and are fundamental to many aspects of chromosome metab.  Their activity involves the transient cleavage of DNA, which, if it occurs near sites of endogenous DNA damage or in the presence of topoisomerase poisons, can result in abortive topoisomerase-induced DNA strand breaks.  These breaks feature covalent linkage of the enzyme to the DNA termini by a 3'- or 5'-phosphotyrosyl bond and are implicated in hereditary human disease, chromosomal instability, and cancer, and underlie the clin. efficacy of an important class of anti-tumor poisons.  The importance of liberating DNA termini from trapped topoisomerase is illustrated by the progressive neurodegenerative disease obsd. in individuals contg. a mutation in tyrosyl-DNA phosphodiesterase 1 (TDP1), an enzyme that cleaves 3'-phosphotyrosyl bonds.  However, a complementary human enzyme that cleaves 5'-phosphotyrosyl bonds has not been reported, despite the effect of DNA double-strand breaks contg. such termini on chromosome instability and cancer.  Here, the authors identify such an enzyme in human cells and show that this activity efficiently restores 5'-phosphate termini at DNA double-strand breaks in prepn. for DNA ligation.  This enzyme, TTRAP (TRAF and TNF receptor-assocd. protein), is a member of the Mg2+/Mn2+-dependent family of phosphodiesterases.  Cellular depletion of TTRAP results in increased susceptibility and sensitivity to topoisomerase-II-induced DNA double-strand breaks.  Thus, TTRAP is, to the authors' knowledge, the 1st human 5'-tyrosyl DNA phosphodiesterase to be identified, and it is suggested that this enzyme is denoted tyrosyl DNA phosphodiesterase-2 (TDP2).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_IrEo1DjZabVg90H21EOLACvtfcHk0lhSlZ0zlYKumA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1SqtrzO&md5=a015dfe2e1b01c78455f67215ddb5b98</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnature08444&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08444%26sid%3Dliteratum%253Aachs%26aulast%3DLedesma%26aufirst%3DF.%2BC.%26aulast%3DEl%2BKhamisy%26aufirst%3DS.%2BF.%26aulast%3DZuma%26aufirst%3DM.%2BC.%26aulast%3DOsborn%26aufirst%3DK.%26aulast%3DCaldecott%26aufirst%3DK.%2BW.%26atitle%3DA%2520human%25205%25E2%2580%25B2-tyrosyl%2520DNA%2520phosphodiesterase%2520that%2520repairs%2520topoisomerase-mediated%2520DNA%2520damage%26jtitle%3DNature%26date%3D2009%26volume%3D461%26spage%3D674%26epage%3DU125%26doi%3D10.1038%2Fnature08444" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Gomez-Herreros, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romero-Granados, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alvarez-Quilon, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quintero, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ju, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Umans, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vermeire, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huylebroeck, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caldecott, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cortes-Ledesma, F.</span><span> </span><span class="NLM_article-title">TDP2-dependent non-homologous end-joining protects against topoisomerase II-induced DNA breaks and genome instability in cells and in vivo</span> <span class="citation_source-journal">PLoS Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1003226</span><span class="refDoi"> DOI: 10.1371/journal.pgen.1003226</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=10.1371%2Fjournal.pgen.1003226" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=e1003226&author=F.+Gomez-Herrerosauthor=R.+Romero-Granadosauthor=Z.+Zengauthor=A.+Alvarez-Quilonauthor=C.+Quinteroauthor=L.+Juauthor=L.+Umansauthor=L.+Vermeireauthor=D.+Huylebroeckauthor=K.+W.+Caldecottauthor=F.+Cortes-Ledesma&title=TDP2-dependent+non-homologous+end-joining+protects+against+topoisomerase+II-induced+DNA+breaks+and+genome+instability+in+cells+and+in+vivo&doi=10.1371%2Fjournal.pgen.1003226"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pgen.1003226&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pgen.1003226%26sid%3Dliteratum%253Aachs%26aulast%3DGomez-Herreros%26aufirst%3DF.%26aulast%3DRomero-Granados%26aufirst%3DR.%26aulast%3DZeng%26aufirst%3DZ.%26aulast%3DAlvarez-Quilon%26aufirst%3DA.%26aulast%3DQuintero%26aufirst%3DC.%26aulast%3DJu%26aufirst%3DL.%26aulast%3DUmans%26aufirst%3DL.%26aulast%3DVermeire%26aufirst%3DL.%26aulast%3DHuylebroeck%26aufirst%3DD.%26aulast%3DCaldecott%26aufirst%3DK.%2BW.%26aulast%3DCortes-Ledesma%26aufirst%3DF.%26atitle%3DTDP2-dependent%2520non-homologous%2520end-joining%2520protects%2520against%2520topoisomerase%2520II-induced%2520DNA%2520breaks%2520and%2520genome%2520instability%2520in%2520cells%2520and%2520in%2520vivo%26jtitle%3DPLoS%2520Genet.%26date%3D2013%26volume%3D9%26spage%3De1003226%26doi%3D10.1371%2Fjournal.pgen.1003226" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Gomez-Herreros, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schuurs-Hoeijmakers, J. H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCormack, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greally, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rulten, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romero-Granados, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Counihan, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chaila, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Conroy, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ennis, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delanty, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cortes-Ledesma, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Brouwer, A. P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavalleri, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">El-Khamisy, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Vries, B. B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caldecott, K. W.</span><span> </span><span class="NLM_article-title">TDP2 protects transcription from abortive topoisomerase activity and is required for normal neural function</span> <span class="citation_source-journal">Nat. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">516</span><span class="NLM_x">–</span> <span class="NLM_lpage">521</span><span class="refDoi"> DOI: 10.1038/ng.2929</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=10.1038%2Fng.2929" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=24658003" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=1%3ACAS%3A528%3ADC%252BC2cXks12ksb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2014&pages=516-521&author=F.+Gomez-Herrerosauthor=J.+H.+M.+Schuurs-Hoeijmakersauthor=M.+McCormackauthor=M.+T.+Greallyauthor=S.+Rultenauthor=R.+Romero-Granadosauthor=T.+J.+Counihanauthor=E.+Chailaauthor=J.+Conroyauthor=S.+Ennisauthor=N.+Delantyauthor=F.+Cortes-Ledesmaauthor=A.+P.+M.+de+Brouwerauthor=G.+L.+Cavalleriauthor=S.+F.+El-Khamisyauthor=B.+B.+A.+de+Vriesauthor=K.+W.+Caldecott&title=TDP2+protects+transcription+from+abortive+topoisomerase+activity+and+is+required+for+normal+neural+function&doi=10.1038%2Fng.2929"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">TDP2 protects transcription from abortive topoisomerase activity and is required for normal neural function</span></div><div class="casAuthors">Gomez-Herreros, Fernando; Schuurs-Hoeijmakers, Janneke H. M.; McCormack, Mark; Greally, Marie T.; Rulten, Stuart; Romero-Granados, Rocio; Counihan, Timothy J.; Chaila, Elijah; Conroy, Judith; Ennis, Sean; Delanty, Norman; Cortes-Ledesma, Felipe; de Brouwer, Arjan P. M.; Cavalleri, Gianpiero L.; El-Khamisy, Sherif F.; de Vries, Bert B. A.; Caldecott, Keith W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">516-521</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Topoisomerase II (TOP2) removes torsional stress from DNA and facilitates gene transcription by introducing transient DNA double-strand breaks (DSBs).  Such DSBs are normally rejoined by TOP2 but on occasion can become abortive and remain unsealed.  Here we identify homozygous mutations in the TDP2 gene encoding tyrosyl DNA phosphodiesterase-2, an enzyme that repairs 'abortive' TOP2-induced DSBs, in individuals with intellectual disability, seizures and ataxia.  We show that cells from affected individuals are hypersensitive to TOP2-induced DSBs and that loss of TDP2 inhibits TOP2-dependent gene transcription in cultured human cells and in mouse post-mitotic neurons following abortive TOP2 activity.  Notably, TDP2 is also required for normal levels of many gene transcripts in developing mouse brain, including numerous gene transcripts assocd. with neurol. function and/or disease, and for normal interneuron d. in mouse cerebellum.  Collectively, these data implicate chromosome breakage by TOP2 as an endogenous threat to gene transcription and to normal neuronal development and maintenance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDwc9VhWlS-rVg90H21EOLACvtfcHk0lhA4acjyDuvwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXks12ksb4%253D&md5=30364c8b473081a7486413b1bcd3e680</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fng.2929&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng.2929%26sid%3Dliteratum%253Aachs%26aulast%3DGomez-Herreros%26aufirst%3DF.%26aulast%3DSchuurs-Hoeijmakers%26aufirst%3DJ.%2BH.%2BM.%26aulast%3DMcCormack%26aufirst%3DM.%26aulast%3DGreally%26aufirst%3DM.%2BT.%26aulast%3DRulten%26aufirst%3DS.%26aulast%3DRomero-Granados%26aufirst%3DR.%26aulast%3DCounihan%26aufirst%3DT.%2BJ.%26aulast%3DChaila%26aufirst%3DE.%26aulast%3DConroy%26aufirst%3DJ.%26aulast%3DEnnis%26aufirst%3DS.%26aulast%3DDelanty%26aufirst%3DN.%26aulast%3DCortes-Ledesma%26aufirst%3DF.%26aulast%3Dde%2BBrouwer%26aufirst%3DA.%2BP.%2BM.%26aulast%3DCavalleri%26aufirst%3DG.%2BL.%26aulast%3DEl-Khamisy%26aufirst%3DS.%2BF.%26aulast%3Dde%2BVries%26aufirst%3DB.%2BB.%2BA.%26aulast%3DCaldecott%26aufirst%3DK.%2BW.%26atitle%3DTDP2%2520protects%2520transcription%2520from%2520abortive%2520topoisomerase%2520activity%2520and%2520is%2520required%2520for%2520normal%2520neural%2520function%26jtitle%3DNat.%2520Genet.%26date%3D2014%26volume%3D46%26spage%3D516%26epage%3D521%26doi%3D10.1038%2Fng.2929" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Maede, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shimizu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fukushima, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kogame, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takeda, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pommier, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murai, J.</span><span> </span><span class="NLM_article-title">Differential and common DNA repair pathways for topoisomerase I- and II-targeted drugs in a genetic DT40 repair cell screen panel</span> <span class="citation_source-journal">Mol. Cancer Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">214</span><span class="NLM_x">–</span> <span class="NLM_lpage">220</span><span class="refDoi"> DOI: 10.1158/1535-7163.MCT-13-0551</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=10.1158%2F1535-7163.MCT-13-0551" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=24130054" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=1%3ACAS%3A528%3ADC%252BC2cXnsFSmsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=214-220&author=Y.+Maedeauthor=H.+Shimizuauthor=T.+Fukushimaauthor=T.+Kogameauthor=T.+Nakamuraauthor=T.+Mikiauthor=S.+Takedaauthor=Y.+Pommierauthor=J.+Murai&title=Differential+and+common+DNA+repair+pathways+for+topoisomerase+I-+and+II-targeted+drugs+in+a+genetic+DT40+repair+cell+screen+panel&doi=10.1158%2F1535-7163.MCT-13-0551"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Differential and Common DNA Repair Pathways for Topoisomerase I- and II-Targeted Drugs in a Genetic DT40 Repair Cell Screen Panel</span></div><div class="casAuthors">Maede, Yuko; Shimizu, Hiroyasu; Fukushima, Toru; Kogame, Toshiaki; Nakamura, Terukazu; Miki, Tsuneharu; Takeda, Shunichi; Pommier, Yves; Murai, Junko</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">214-220</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Clin. topoisomerase I (Top1) and II (Top2) inhibitors trap topoisomerases on DNA, thereby inducing protein-linked DNA breaks.  Cancer cells resist the drugs by removing topoisomerase-DNA complexes, and repairing the drug-induced DNA double-strand breaks (DSB) by homologous recombination and nonhomologous end joining (NHEJ).  Because numerous enzymes and cofactors are involved in the removal of the topoisomerase-DNA complexes and DSB repair, it has been challenging to comprehensively analyze the relative contribution of multiple genetic pathways in vertebrate cells.  Comprehending the relative contribution of individual repair factors would give insights into the lesions induced by the inhibitors and genetic determinants of response.  Ultimately, this information would be useful to target specific pathways to augment the therapeutic activity of topoisomerase inhibitors.  To this end, we put together 48 isogenic DT40 mutant cells deficient in DNA repair and generated one cell line deficient in autophagy (ATG5).  Sensitivity profiles were established for three clin. relevant Top1 inhibitors (camptothecin and the indenoisoquinolines LMP400 and LMP776) and three Top2 inhibitors (etoposide, doxorubicin, and ICRF-193).  Highly significant correlations were found among Top1 inhibitors as well as Top2 inhibitors, whereas the profiles of Top1 inhibitors were different from those of Top2 inhibitors.  Most distinct repair pathways between Top1 and Top2 inhibitors include NHEJ, TDP1, TDP2, PARP1, and Fanconi Anemia genes, whereas homologous recombination seems relevant esp. for Top1 and, to a lesser extent, for Top2 inhibitors.  We also found and discuss differential pathways among Top1 inhibitors and Top2 inhibitors.  Mol Cancer Ther; 13(1); 214-20. ©2013 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrLkuEffSz27Vg90H21EOLACvtfcHk0lhA4acjyDuvwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXnsFSmsg%253D%253D&md5=346456ce64a6e142fb9df968cdc9f05e</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-13-0551&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-13-0551%26sid%3Dliteratum%253Aachs%26aulast%3DMaede%26aufirst%3DY.%26aulast%3DShimizu%26aufirst%3DH.%26aulast%3DFukushima%26aufirst%3DT.%26aulast%3DKogame%26aufirst%3DT.%26aulast%3DNakamura%26aufirst%3DT.%26aulast%3DMiki%26aufirst%3DT.%26aulast%3DTakeda%26aufirst%3DS.%26aulast%3DPommier%26aufirst%3DY.%26aulast%3DMurai%26aufirst%3DJ.%26atitle%3DDifferential%2520and%2520common%2520DNA%2520repair%2520pathways%2520for%2520topoisomerase%2520I-%2520and%2520II-targeted%2520drugs%2520in%2520a%2520genetic%2520DT40%2520repair%2520cell%2520screen%2520panel%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2014%26volume%3D13%26spage%3D214%26epage%3D220%26doi%3D10.1158%2F1535-7163.MCT-13-0551" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Zeng, Z. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cortes-Ledesma, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">El Khamisy, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caldecott, K. W.</span><span> </span><span class="NLM_article-title">TDP2/TTRAP is the major 5 ′-tyrosyl DNA phosphodiesterase activity in vertebrate cells and is critical for cellular resistance to topoisomerase II-induced DNA damage</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">286</span><span class="NLM_x">, </span> <span class="NLM_fpage">403</span><span class="NLM_x">–</span> <span class="NLM_lpage">409</span><span class="refDoi"> DOI: 10.1074/jbc.M110.181016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=10.1074%2Fjbc.M110.181016" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2011&pages=403-409&author=Z.+H.+Zengauthor=F.+Cortes-Ledesmaauthor=S.+F.+El+Khamisyauthor=K.+W.+Caldecott&title=TDP2%2FTTRAP+is+the+major+5+%E2%80%B2-tyrosyl+DNA+phosphodiesterase+activity+in+vertebrate+cells+and+is+critical+for+cellular+resistance+to+topoisomerase+II-induced+DNA+damage&doi=10.1074%2Fjbc.M110.181016"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M110.181016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M110.181016%26sid%3Dliteratum%253Aachs%26aulast%3DZeng%26aufirst%3DZ.%2BH.%26aulast%3DCortes-Ledesma%26aufirst%3DF.%26aulast%3DEl%2BKhamisy%26aufirst%3DS.%2BF.%26aulast%3DCaldecott%26aufirst%3DK.%2BW.%26atitle%3DTDP2%252FTTRAP%2520is%2520the%2520major%25205%2520%25E2%2580%25B2-tyrosyl%2520DNA%2520phosphodiesterase%2520activity%2520in%2520vertebrate%2520cells%2520and%2520is%2520critical%2520for%2520cellular%2520resistance%2520to%2520topoisomerase%2520II-induced%2520DNA%2520damage%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2011%26volume%3D286%26spage%3D403%26epage%3D409%26doi%3D10.1074%2Fjbc.M110.181016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Do, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varanasi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kubacka, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newman, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chauhan, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daniels, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boccetta, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garrett, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinez, L. A.</span><span> </span><span class="NLM_article-title">Mutant p53 cooperates with ETS2 to promote etoposide resistance</span> <span class="citation_source-journal">Genes Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">830</span><span class="NLM_x">–</span> <span class="NLM_lpage">845</span><span class="refDoi"> DOI: 10.1101/gad.181685.111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=10.1101%2Fgad.181685.111" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2012&pages=830-845&author=P.+M.+Doauthor=L.+Varanasiauthor=S.+Fanauthor=C.+Liauthor=I.+Kubackaauthor=V.+Newmanauthor=K.+Chauhanauthor=S.+R.+Danielsauthor=M.+Boccettaauthor=M.+R.+Garrettauthor=R.+Liauthor=L.+A.+Martinez&title=Mutant+p53+cooperates+with+ETS2+to+promote+etoposide+resistance&doi=10.1101%2Fgad.181685.111"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1101%2Fgad.181685.111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.181685.111%26sid%3Dliteratum%253Aachs%26aulast%3DDo%26aufirst%3DP.%2BM.%26aulast%3DVaranasi%26aufirst%3DL.%26aulast%3DFan%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DKubacka%26aufirst%3DI.%26aulast%3DNewman%26aufirst%3DV.%26aulast%3DChauhan%26aufirst%3DK.%26aulast%3DDaniels%26aufirst%3DS.%2BR.%26aulast%3DBoccetta%26aufirst%3DM.%26aulast%3DGarrett%26aufirst%3DM.%2BR.%26aulast%3DLi%26aufirst%3DR.%26aulast%3DMartinez%26aufirst%3DL.%2BA.%26atitle%3DMutant%2520p53%2520cooperates%2520with%2520ETS2%2520to%2520promote%2520etoposide%2520resistance%26jtitle%3DGenes%2520Dev.%26date%3D2012%26volume%3D26%26spage%3D830%26epage%3D845%26doi%3D10.1101%2Fgad.181685.111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Virgen-Slane, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rozovics, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fitzgerald, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ngo, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chou, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van der Heden van Noort, G. J. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filippov, D. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gershon, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Semler, B. L.</span><span> </span><span class="NLM_article-title">An RNA virus hijacks an incognito function of a DNA repair enzyme</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">109</span><span class="NLM_x">, </span> <span class="NLM_fpage">14634</span><span class="NLM_x">–</span> <span class="NLM_lpage">14639</span><span class="refDoi"> DOI: 10.1073/pnas.1208096109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=10.1073%2Fpnas.1208096109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=22908287" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVaqu7zF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=14634-14639&author=R.+Virgen-Slaneauthor=J.+M.+Rozovicsauthor=K.+D.+Fitzgeraldauthor=T.+Ngoauthor=W.+Chouauthor=G.+J.+V.+van+der+Heden+van+Noortauthor=D.+V.+Filippovauthor=P.+D.+Gershonauthor=B.+L.+Semler&title=An+RNA+virus+hijacks+an+incognito+function+of+a+DNA+repair+enzyme&doi=10.1073%2Fpnas.1208096109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">An RNA virus hijacks an incognito function of a DNA repair enzyme</span></div><div class="casAuthors">Virgen-Slane, Richard; Rozovics, Janet M.; Fitzgerald, Kerry D.; Ngo, Tuan; Chou, Wayne; van der Heden van Noort, Gerbrand J.; Filippov, Dmitri V.; Gershon, Paul D.; Semler, Bert L.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">14634-14639, S14634/1-S14634/3</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">A previously described mammalian cell activity, called VPg unlinkase, specifically cleaves a unique protein-RNA covalent linkage generated during the viral genomic RNA replication steps of a picornavirus infection.  For over three decades, the identity of this cellular activity and its normal role in the uninfected cell had remained elusive.  Here we report the purifn. and identification of VPg unlinkase as the DNA repair enzyme, 5'-tyrosyl-DNA phosphodiesterase-2 (TDP2).  Our data show that VPg unlinkase activity in different mammalian cell lines correlates with their differential expression of TDP2.  Furthermore, we show that recombinant TDP2 can cleave the protein-RNA linkage generated by different picornaviruses without impairing the integrity of viral RNA.  Our results reveal a unique RNA repair-like function for TDP2 and suggest an unusual role in host-pathogen interactions for this cellular enzyme.  On the basis of the identification of TDP2 as a potential antiviral target, our findings may lead to the development of universal therapeutics to treat the millions of individuals afflicted annually with diseases caused by picornaviruses, including myocarditis, aseptic meningitis, encephalitis, hepatitis, and the common cold.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPMlLwsqL-cLVg90H21EOLACvtfcHk0liA2t1Q0XAN1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVaqu7zF&md5=3af24e75947bd5cd81dcc21b79a67541</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1208096109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1208096109%26sid%3Dliteratum%253Aachs%26aulast%3DVirgen-Slane%26aufirst%3DR.%26aulast%3DRozovics%26aufirst%3DJ.%2BM.%26aulast%3DFitzgerald%26aufirst%3DK.%2BD.%26aulast%3DNgo%26aufirst%3DT.%26aulast%3DChou%26aufirst%3DW.%26aulast%3Dvan%2Bder%2BHeden%2Bvan%2BNoort%26aufirst%3DG.%2BJ.%2BV.%26aulast%3DFilippov%26aufirst%3DD.%2BV.%26aulast%3DGershon%26aufirst%3DP.%2BD.%26aulast%3DSemler%26aufirst%3DB.%2BL.%26atitle%3DAn%2520RNA%2520virus%2520hijacks%2520an%2520incognito%2520function%2520of%2520a%2520DNA%2520repair%2520enzyme%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26spage%3D14634%26epage%3D14639%26doi%3D10.1073%2Fpnas.1208096109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Koniger, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wingert, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosler, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beck, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nassal, M.</span><span> </span><span class="NLM_article-title">Involvement of the host DNA-repair enzyme TDP2 in formation of the covalently closed circular DNA persistence reservoir of hepatitis B viruses</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">111</span><span class="NLM_x">, </span> <span class="NLM_fpage">E4244</span><span class="NLM_x">–</span> <span class="NLM_lpage">4253</span><span class="refDoi"> DOI: 10.1073/pnas.1409986111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=10.1073%2Fpnas.1409986111" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=E4244-4253&author=C.+Konigerauthor=I.+Wingertauthor=M.+Marsmannauthor=C.+Roslerauthor=J.+Beckauthor=M.+Nassal&title=Involvement+of+the+host+DNA-repair+enzyme+TDP2+in+formation+of+the+covalently+closed+circular+DNA+persistence+reservoir+of+hepatitis+B+viruses&doi=10.1073%2Fpnas.1409986111"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1409986111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1409986111%26sid%3Dliteratum%253Aachs%26aulast%3DKoniger%26aufirst%3DC.%26aulast%3DWingert%26aufirst%3DI.%26aulast%3DMarsmann%26aufirst%3DM.%26aulast%3DRosler%26aufirst%3DC.%26aulast%3DBeck%26aufirst%3DJ.%26aulast%3DNassal%26aufirst%3DM.%26atitle%3DInvolvement%2520of%2520the%2520host%2520DNA-repair%2520enzyme%2520TDP2%2520in%2520formation%2520of%2520the%2520covalently%2520closed%2520circular%2520DNA%2520persistence%2520reservoir%2520of%2520hepatitis%2520B%2520viruses%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2014%26volume%3D111%26spage%3DE4244%26epage%3D4253%26doi%3D10.1073%2Fpnas.1409986111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Beck, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nassal, M.</span><span> </span><span class="NLM_article-title">Hepatitis B virus replication</span> <span class="citation_source-journal">World J. Gastroenterol</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">48</span><span class="NLM_x">–</span> <span class="NLM_lpage">64</span><span class="refDoi"> DOI: 10.3748/wjg.v13.i1.48</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=10.3748%2Fwjg.v13.i1.48" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=17206754" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=1%3ACAS%3A528%3ADC%252BD2sXitFSgt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2007&pages=48-64&author=J.+Beckauthor=M.+Nassal&title=Hepatitis+B+virus+replication&doi=10.3748%2Fwjg.v13.i1.48"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Hepatitis B virus replication</span></div><div class="casAuthors">Beck, Juergen; Nassal, Michael</div><div class="citationInfo"><span class="NLM_cas:title">World Journal of Gastroenterology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">48-64</span>CODEN:
                <span class="NLM_cas:coden">WJGAF2</span>;
        ISSN:<span class="NLM_cas:issn">1007-9327</span>.
    
            (<span class="NLM_cas:orgname">World Journal of Gastroenterology</span>)
        </div><div class="casAbstract">A review.  Hepadnaviruses, including human hepatitis B virus (HBV), replicate through reverse transcription of an RNA intermediate, the pregenomic RNA (pgRNA).  Despite this kinship to retroviruses, there are fundamental differences beyond the fact that hepadnavirions contain DNA instead of RNA.  Most peculiar is the initiation of reverse transcription: it occurs by protein-priming, is strictly committed to using an RNA hairpin on the pgRNA, ε, as template, and depends on cellular chaperones; moreover, proper replication can apparently occur only in the specialized environment of intact nucleocapsids.  This complexity has hampered an in-depth mechanistic understanding.  The recent successful reconstitution in the test tube of active replication initiation complexes from purified components, for duck HBV (DHBV), now allows for the anal. of the biochem. of hepadnaviral replication at the mol. level.  Here we review the current state of knowledge at all steps of the hepadnaviral genome replication cycle, with emphasis on new insights that turned up by the use of such cell-free systems.  At this time, they can, unfortunately, not be complemented by three-dimensional structural information on the involved components.  However, at least for the ε RNA element such information is emerging, raising expectations that combining biophysics with biochem. and genetics will soon provide a powerful integrated approach for solving the many outstanding questions.  The ultimate, though most challenging goal, will be to visualize the hepadnaviral reverse transcriptase in the act of synthesizing DNA, which will also have strong implications for drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeq4GL5xysiLVg90H21EOLACvtfcHk0liA2t1Q0XAN1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXitFSgt7w%253D&md5=a32783744d5f055abfdf9307bd883513</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.3748%2Fwjg.v13.i1.48&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3748%252Fwjg.v13.i1.48%26sid%3Dliteratum%253Aachs%26aulast%3DBeck%26aufirst%3DJ.%26aulast%3DNassal%26aufirst%3DM.%26atitle%3DHepatitis%2520B%2520virus%2520replication%26jtitle%3DWorld%2520J.%2520Gastroenterol%26date%3D2007%26volume%3D13%26spage%3D48%26epage%3D64%26doi%3D10.3748%2Fwjg.v13.i1.48" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Nassal, M.</span><span> </span><span class="NLM_article-title">HBV cccDNA: viral persistence reservoir and key obstacle for a cure of chronic hepatitis B</span> <span class="citation_source-journal">Gut</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">64</span><span class="NLM_x">, </span> <span class="NLM_fpage">1972</span><span class="NLM_x">–</span> <span class="NLM_lpage">1984</span><span class="refDoi"> DOI: 10.1136/gutjnl-2015-309809</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=10.1136%2Fgutjnl-2015-309809" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=26048673" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=1%3ACAS%3A528%3ADC%252BC28XmvVSrsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2015&pages=1972-1984&author=M.+Nassal&title=HBV+cccDNA%3A+viral+persistence+reservoir+and+key+obstacle+for+a+cure+of+chronic+hepatitis+B&doi=10.1136%2Fgutjnl-2015-309809"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">HBV cccDNA: viral persistence reservoir and key obstacle for a cure of chronic hepatitis B</span></div><div class="casAuthors">Nassal, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Gut</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1972-1984</span>CODEN:
                <span class="NLM_cas:coden">GUTTAK</span>;
        ISSN:<span class="NLM_cas:issn">0017-5749</span>.
    
            (<span class="NLM_cas:orgname">BMJ Publishing Group</span>)
        </div><div class="casAbstract">At least 250 million people worldwide are chronically infected with HBV, a small hepatotropic DNA virus that replicates through reverse transcription.  Chronic infection greatly increases the risk for terminal liver disease.  Current therapies rarely achieve a cure due to the refractory nature of an intracellular viral replication intermediate termed covalently closed circular (ccc) DNA.  Upon infection, cccDNA is generated as a plasmid-like episome in the host cell nucleus from the protein-linked relaxed circular (RC) DNA genome in incoming virions.  Its fundamental role is that as template for all viral RNAs, and in consequence new virions.  Biosynthesis of RC-DNA by reverse transcription of the viral pregenomic RNA is now understood in considerable detail, yet conversion of RC-DNA to cccDNA is still obscure, foremostly due to the lack of feasible, cccDNA-dependent assay systems.  Conceptual and recent exptl. data link cccDNA formation to cellular DNA repair, which is increasingly appreciated as a crit. interface between cells and viruses.  Together with new in vitro HBV infection systems, based on the identification of the bile acid transporter sodium taurocholate cotransporting polypeptide as an HBV entry receptor, this offers novel opportunities to decipher, and eventually interfere with, formation of the HBV persistence reservoir.  After a brief overview of the role of cccDNA in the HBV infectious cycle, this review aims to summarize current knowledge on cccDNA mol. biol., to highlight the exptl. restrictions that have hitherto hampered faster progress and to discuss cccDNA as target for new, potentially curative therapies of chronic hepatitis B.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQaHzOzO1JXLVg90H21EOLACvtfcHk0limQqY3wgzxyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmvVSrsbg%253D&md5=7cbbbccef38636fd9bbcae0a9b869bc9</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1136%2Fgutjnl-2015-309809&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Fgutjnl-2015-309809%26sid%3Dliteratum%253Aachs%26aulast%3DNassal%26aufirst%3DM.%26atitle%3DHBV%2520cccDNA%253A%2520viral%2520persistence%2520reservoir%2520and%2520key%2520obstacle%2520for%2520a%2520cure%2520of%2520chronic%2520hepatitis%2520B%26jtitle%3DGut%26date%3D2015%26volume%3D64%26spage%3D1972%26epage%3D1984%26doi%3D10.1136%2Fgutjnl-2015-309809" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Cui, X. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McAllister, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boregowda, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sohn, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ledesma, F. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caldecott, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seeger, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, J. M.</span><span> </span><span class="NLM_article-title">Does tyrosyl DNA phosphodiesterase-2 play a role in hepatitis B virus genome repair?</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">e0128401</span><span class="refDoi"> DOI: 10.1371/journal.pone.0128401</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=10.1371%2Fjournal.pone.0128401" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=26079492" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlWmur%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=e0128401&author=X.+J.+Cuiauthor=R.+McAllisterauthor=R.+Boregowdaauthor=J.+A.+Sohnauthor=F.+C.+Ledesmaauthor=K.+W.+Caldecottauthor=C.+Seegerauthor=J.+M.+Hu&title=Does+tyrosyl+DNA+phosphodiesterase-2+play+a+role+in+hepatitis+B+virus+genome+repair%3F&doi=10.1371%2Fjournal.pone.0128401"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Does tyrosyl DNA phosphodiesterase-2 play a role in hepatitis B virus genome repair?</span></div><div class="casAuthors">Cui, Xiuji; McAllister, Rebecca; Boregowda, Rajeev; Sohn, Ji A.; Ledesma, Felipe Cortes; Caldecott, Keith W.; Seeger, Christoph; Hu, Jianming</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">e0128401/1-e0128401/15</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Hepatitis B virus (HBV) replication and persistence are sustained by a nuclear episome, the covalently closed circular (CCC) DNA, which serves as the transcriptional template for all viral RNAs.  CCC DNA is converted from a relaxed circular (RC) DNA in the virion early during infection as well as from RC DNA in intracellular progeny nucleocapsids via an intracel- lular amplification pathway.  Current antiviral therapies suppress viral replication but cannot eliminate CCC DNA.  Thus, persistence of CCC DNA remains an obstacle toward curing chronic HBV infection.  Unfortunately, very little is known about how CCC DNA is formed.  CCC DNA formation requires removal of the virally encoded reverse transcriptase (RT) pro- tein from the 5' end of the minus strand of RC DNA.  Tyrosyl DNA phosphodiesterase-2 (Tdp2) was recently identified as the enzyme responsible for cleavage of tyrosyl-5' DNA linkages formed between topoisomerase II and cellular DNA.  Because the RT-DNA linkage is also a 5' DNA-phosphotyrosyl bond, it has been hypothesized that Tdp2 might be one of several elusive host factors required for CCC DNA formation.  Therefore, we examd. the role of Tdp2 in RC DNA deproteination and CCC DNA formation.  We demonstrated Tdp2 can cleave the tyrosyl-minus strand DNA linkage using authentic HBV RC DNA isolated from nucleocapsids and using RT covalently linked to short minus strand DNA produced in vitro.  On the other hand, our results showed that Tdp2 gene knockout did not block CCC DNA formation during HBV infection of permissive human hepatoma cells and did not prevent intracellular amplification of duck hepatitis B virus CCC DNA.  These results indicate that although Tdp2 can remove the RT covalently linked to the 5' end of the HBV minus strand DNA in vitro, this protein might not be required for CCC DNA formation in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhjQ8pGys-lLVg90H21EOLACvtfcHk0limQqY3wgzxyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlWmur%252FO&md5=76ed1ebb188d7d10a45523e65f2bd57a</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0128401&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0128401%26sid%3Dliteratum%253Aachs%26aulast%3DCui%26aufirst%3DX.%2BJ.%26aulast%3DMcAllister%26aufirst%3DR.%26aulast%3DBoregowda%26aufirst%3DR.%26aulast%3DSohn%26aufirst%3DJ.%2BA.%26aulast%3DLedesma%26aufirst%3DF.%2BC.%26aulast%3DCaldecott%26aufirst%3DK.%2BW.%26aulast%3DSeeger%26aufirst%3DC.%26aulast%3DHu%26aufirst%3DJ.%2BM.%26atitle%3DDoes%2520tyrosyl%2520DNA%2520phosphodiesterase-2%2520play%2520a%2520role%2520in%2520hepatitis%2520B%2520virus%2520genome%2520repair%253F%26jtitle%3DPLoS%2520One%26date%3D2015%26volume%3D10%26spage%3De0128401%26doi%3D10.1371%2Fjournal.pone.0128401" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Gao, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, S. Y. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marchand, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pommier, Y.</span><span> </span><span class="NLM_article-title">Biochemical characterization of human tyrosyl-DNA phosphodiesterase 2 (TDP2/TTRAP) a Mg<sup>2+</sup>/Mn<sup>2+</sup>-dependent phosphodiesterase specific for the repair of topoisomerase cleavage complexes</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">287</span><span class="NLM_x">, </span> <span class="NLM_fpage">30842</span><span class="NLM_x">–</span> <span class="NLM_lpage">30852</span><span class="refDoi"> DOI: 10.1074/jbc.M112.393983</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=10.1074%2Fjbc.M112.393983" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=30842-30852&author=R.+Gaoauthor=S.+Y.+N.+Huangauthor=C.+Marchandauthor=Y.+Pommier&title=Biochemical+characterization+of+human+tyrosyl-DNA+phosphodiesterase+2+%28TDP2%2FTTRAP%29+a+Mg2%2B%2FMn2%2B-dependent+phosphodiesterase+specific+for+the+repair+of+topoisomerase+cleavage+complexes&doi=10.1074%2Fjbc.M112.393983"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M112.393983&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M112.393983%26sid%3Dliteratum%253Aachs%26aulast%3DGao%26aufirst%3DR.%26aulast%3DHuang%26aufirst%3DS.%2BY.%2BN.%26aulast%3DMarchand%26aufirst%3DC.%26aulast%3DPommier%26aufirst%3DY.%26atitle%3DBiochemical%2520characterization%2520of%2520human%2520tyrosyl-DNA%2520phosphodiesterase%25202%2520%2528TDP2%252FTTRAP%2529%2520a%2520Mg2%252B%252FMn2%252B-dependent%2520phosphodiesterase%2520specific%2520for%2520the%2520repair%2520of%2520topoisomerase%2520cleavage%2520complexes%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26spage%3D30842%26epage%3D30852%26doi%3D10.1074%2Fjbc.M112.393983" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Schellenberg, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Appel, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adhikari, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robertson, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramsden, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, R. S.</span><span> </span><span class="NLM_article-title">Mechanism of repair of 5′-topoisomerase II-DNA adducts by mammalian tyrosyl- DNA phosphodiesterase 2</span> <span class="citation_source-journal">Nat. Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">1363</span><span class="NLM_x">–</span> <span class="NLM_lpage">1371</span><span class="refDoi"> DOI: 10.1038/nsmb.2418</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=10.1038%2Fnsmb.2418" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=23104055" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFOmt7jF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=1363-1371&author=M.+J.+Schellenbergauthor=C.+D.+Appelauthor=S.+Adhikariauthor=P.+D.+Robertsonauthor=D.+A.+Ramsdenauthor=R.+S.+Williams&title=Mechanism+of+repair+of+5%E2%80%B2-topoisomerase+II-DNA+adducts+by+mammalian+tyrosyl-+DNA+phosphodiesterase+2&doi=10.1038%2Fnsmb.2418"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism of repair of 5'-topoisomerase II-DNA adducts by mammalian tyrosyl-DNA phosphodiesterase 2</span></div><div class="casAuthors">Schellenberg, Matthew J.; Appel, C. Denise; Adhikari, Sanjay; Robertson, Patrick D.; Ramsden, Dale A.; Williams, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1363-1371</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The topoisomerase II (topo II) DNA incision-and-ligation cycle can be poisoned (for example following treatment with cancer chemotherapeutics) to generate cytotoxic DNA double-strand breaks (DSBs) with topo II covalently conjugated to DNA.  Tyrosyl-DNA phosphodiesterase 2 (Tdp2) protects genomic integrity by reversing 5'-phosphotyrosyl-linked topo II-DNA adducts.  Here, X-ray structures of mouse Tdp2-DNA complexes reveal that Tdp2 β-2-helix-β DNA damage-binding grasp, helical cap and DNA lesion-binding elements fuse to form an elongated protein-DNA conjugate substrate-interaction groove.  The Tdp2 DNA-binding surface is highly tailored for engagement of 5'-adducted single-stranded DNA ends and restricts non-specific endonucleolytic or exonucleolytic processing.  Structural, mutational and functional analyses support a single-metal ion catalytic mechanism for the exonuclease-endonuclease-phosphatase (EEP) nuclease superfamily and establish a mol. framework for targeted small-mol. blockade of Tdp2-mediated resistance to anticancer topoisomerase drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBb8RLbBMuDLVg90H21EOLACvtfcHk0limQqY3wgzxyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFOmt7jF&md5=eacf558a24e0e77de0577fe8a15b3e06</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.2418&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.2418%26sid%3Dliteratum%253Aachs%26aulast%3DSchellenberg%26aufirst%3DM.%2BJ.%26aulast%3DAppel%26aufirst%3DC.%2BD.%26aulast%3DAdhikari%26aufirst%3DS.%26aulast%3DRobertson%26aufirst%3DP.%2BD.%26aulast%3DRamsden%26aufirst%3DD.%2BA.%26aulast%3DWilliams%26aufirst%3DR.%2BS.%26atitle%3DMechanism%2520of%2520repair%2520of%25205%25E2%2580%25B2-topoisomerase%2520II-DNA%2520adducts%2520by%2520mammalian%2520tyrosyl-%2520DNA%2520phosphodiesterase%25202%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2012%26volume%3D19%26spage%3D1363%26epage%3D1371%26doi%3D10.1038%2Fnsmb.2418" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Shi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurahashi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsutakawa, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tainer, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pommier, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aihara, H.</span><span> </span><span class="NLM_article-title">Structural basis for recognition of 5′-phosphotyrosine adducts by Tdp2</span> <span class="citation_source-journal">Nat. Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">1372</span><span class="NLM_x">–</span> <span class="NLM_lpage">1377</span><span class="refDoi"> DOI: 10.1038/nsmb.2423</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=10.1038%2Fnsmb.2423" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=23104058" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFOmtL3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=1372-1377&author=K.+Shiauthor=K.+Kurahashiauthor=R.+Gaoauthor=S.+E.+Tsutakawaauthor=J.+A.+Tainerauthor=Y.+Pommierauthor=H.+Aihara&title=Structural+basis+for+recognition+of+5%E2%80%B2-phosphotyrosine+adducts+by+Tdp2&doi=10.1038%2Fnsmb.2423"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for recognition of 5'-phosphotyrosine adducts by Tdp2</span></div><div class="casAuthors">Shi, Ke; Kurahashi, Kayo; Gao, Rui; Tsutakawa, Susan E.; Tainer, John A.; Pommier, Yves; Aihara, Hideki</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1372-1377</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The DNA-repair enzyme Tdp2 resolves 5'-phosphotyrosyl DNA adducts and mediates resistance to anticancer drugs that target covalent topoisomerase-DNA complexes.  Tdp2 also participates in key signaling pathways during development and tumorigenesis and cleaves a protein-RNA linkage during picornavirus replication.  The crystal structure of zebrafish Tdp2 bound to DNA reveals a deep, narrow basic groove that selectively accommodates the 5' end of single-stranded DNA in a stretched conformation.  The crystal structure of the full-length Caenorhabditis elegansTdp2 shows that this groove can also accommodate an acidic peptide stretch in vitro, with glutamate and aspartate side chains occupying the DNA backbone phosphate-binding sites.  This extensive mol. mimicry suggests a potential mechanism for autoregulation and interaction of Tdp2 with phosphorylated proteins in signaling.  Our study provides a framework to interrogate functions of Tdp2 and develop inhibitors for chemotherapeutic and antiviral applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoExrAgpVGQ8LVg90H21EOLACvtfcHk0li3kJJWVVLRKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFOmtL3F&md5=c49681f32d5b045ec8570358869287d7</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.2423&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.2423%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DK.%26aulast%3DKurahashi%26aufirst%3DK.%26aulast%3DGao%26aufirst%3DR.%26aulast%3DTsutakawa%26aufirst%3DS.%2BE.%26aulast%3DTainer%26aufirst%3DJ.%2BA.%26aulast%3DPommier%26aufirst%3DY.%26aulast%3DAihara%26aufirst%3DH.%26atitle%3DStructural%2520basis%2520for%2520recognition%2520of%25205%25E2%2580%25B2-phosphotyrosine%2520adducts%2520by%2520Tdp2%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2012%26volume%3D19%26spage%3D1372%26epage%3D1377%26doi%3D10.1038%2Fnsmb.2423" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Garcin, E. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hosfield, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Desai, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haas, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bjoras, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cunningham, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tainer, J. A.</span><span> </span><span class="NLM_article-title">DNA apurinic-apyrimidinic site binding and excision by endonuclease IV</span> <span class="citation_source-journal">Nat. Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">515</span><span class="NLM_x">–</span> <span class="NLM_lpage">522</span><span class="refDoi"> DOI: 10.1038/nsmb.1414</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=10.1038%2Fnsmb.1414" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=18408731" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlsFSltb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2008&pages=515-522&author=E.+D.+Garcinauthor=D.+J.+Hosfieldauthor=S.+A.+Desaiauthor=B.+J.+Haasauthor=M.+Bjorasauthor=R.+P.+Cunninghamauthor=J.+A.+Tainer&title=DNA+apurinic-apyrimidinic+site+binding+and+excision+by+endonuclease+IV&doi=10.1038%2Fnsmb.1414"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">DNA apurinic-apyrimidinic site binding and excision by endonuclease IV</span></div><div class="casAuthors">Garcin, Elsa D.; Hosfield, David J.; Desai, Sunil A.; Haas, Brian J.; Bjoeras, Magnar; Cunningham, Richard P.; Tainer, John A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">515-522</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Escherichia coli endonuclease IV is an archetype for an abasic or apurinic-apyrimidinic endonuclease superfamily crucial for DNA base excision repair.  Here biochem., mutational and crystallog. characterizations reveal a three-metal ion mechanism for damage binding and incision.  The 1.10-Å resoln. DNA-free and the 2.45-Å resoln. DNA-substrate complex structures capture substrate stabilization by Arg37 and reveal a distorted Zn3-ligand arrangement that reverts, after catalysis, to an ideal geometry suitable to hold rather than release cleaved DNA product.  The 1.45-Å resoln. DNA-product complex structure shows how Tyr72 caps the active site, tunes its dielec. environment and promotes catalysis by Glu261-activated hydroxide, bound to two Zn2+ ions throughout catalysis.  These structural, mutagenesis and biochem. results suggest general requirements for abasic site removal in contrast to features specific to the distinct endonuclease IV α-β triose phosphate isomerase (TIM) barrel and APE1 four-layer α-β folds of the apurinic-apyrimidinic endonuclease families.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqyM9U4gyIqJrVg90H21EOLACvtfcHk0li3kJJWVVLRKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlsFSltb4%253D&md5=ca46b5de6944cd3bef5c7213fdba7c41</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.1414&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.1414%26sid%3Dliteratum%253Aachs%26aulast%3DGarcin%26aufirst%3DE.%2BD.%26aulast%3DHosfield%26aufirst%3DD.%2BJ.%26aulast%3DDesai%26aufirst%3DS.%2BA.%26aulast%3DHaas%26aufirst%3DB.%2BJ.%26aulast%3DBjoras%26aufirst%3DM.%26aulast%3DCunningham%26aufirst%3DR.%2BP.%26aulast%3DTainer%26aufirst%3DJ.%2BA.%26atitle%3DDNA%2520apurinic-apyrimidinic%2520site%2520binding%2520and%2520excision%2520by%2520endonuclease%2520IV%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2008%26volume%3D15%26spage%3D515%26epage%3D522%26doi%3D10.1038%2Fnsmb.1414" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Raoof, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Depledge, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamilton, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamilton, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hitchin, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopkins, G. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jordan, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maguire, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGonagle, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mould, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rushbrooke, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Small, H. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomson, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turlais, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waddell, I. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waszkowycz, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watson, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ogilvie, D. J.</span><span> </span><span class="NLM_article-title">Toxoflavins and deazaflavins as the first reported selective small molecule inhibitors of tyrosyl-DNA phosphodiesterase II</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">6352</span><span class="NLM_x">–</span> <span class="NLM_lpage">6370</span><span class="refDoi"> DOI: 10.1021/jm400568p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400568p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFanu7zF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=6352-6370&author=A.+Raoofauthor=P.+Depledgeauthor=N.+M.+Hamiltonauthor=N.+S.+Hamiltonauthor=J.+R.+Hitchinauthor=G.+V.+Hopkinsauthor=A.+M.+Jordanauthor=L.+A.+Maguireauthor=A.+E.+McGonagleauthor=D.+P.+Mouldauthor=M.+Rushbrookeauthor=H.+F.+Smallauthor=K.+M.+Smithauthor=G.+J.+Thomsonauthor=F.+Turlaisauthor=I.+D.+Waddellauthor=B.+Waszkowyczauthor=A.+J.+Watsonauthor=D.+J.+Ogilvie&title=Toxoflavins+and+deazaflavins+as+the+first+reported+selective+small+molecule+inhibitors+of+tyrosyl-DNA+phosphodiesterase+II&doi=10.1021%2Fjm400568p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Toxoflavins and Deazaflavins as the First Reported Selective Small Molecule Inhibitors of Tyrosyl-DNA Phosphodiesterase II</span></div><div class="casAuthors">Raoof, Ali; Depledge, Paul; Hamilton, Niall M.; Hamilton, Nicola S.; Hitchin, James R.; Hopkins, Gemma V.; Jordan, Allan M.; Maguire, Laura A.; McGonagle, Alison E.; Mould, Daniel P.; Rushbrooke, Mathew; Small, Helen F.; Smith, Kate M.; Thomson, Graeme J.; Turlais, Fabrice; Waddell, Ian D.; Waszkowycz, Bohdan; Watson, Amanda J.; Ogilvie, Donald J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">6352-6370</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The recently discovered enzyme tyrosyl-DNA phosphodiesterase 2 (TDP2) has been implicated in the topoisomerase-mediated repair of DNA damage.  In the clin. setting, it has been hypothesized that TDP2 may mediate drug resistance to topoisomerase II (topo II) inhibition by etoposide.  Therefore, selective pharmacol. inhibition of TDP2 is proposed as a novel approach to overcome intrinsic or acquired resistance to topo II-targeted drug therapy.  Following a high-throughput screening (HTS) campaign, toxoflavins and deazaflavins were identified as the first reported sub-micromolar and selective inhibitors of this enzyme.  Toxoflavin derivs., e.g., I (R1 = 4-EtC6H4, 4-F3CC6H4, 2-MeC6H4, etc.), appeared to exhibit a clear structure-activity relationship (SAR) for TDP2 enzymic inhibition.  However, we obsd. a key redox liability of this series, and this, alongside early in vitro drug metab. and pharmacokinetics (DMPK) issues, precluded further exploration.  The deazaflavins, II (R1 = Ph, cyclopropyl, 1H-indazol-6-yl, etc., R2 = H, Cl, R3 = H, Cl, CF3, CN, R4, R5 = H, Cl), were developed from a singleton HTS hit.  This series showed distinct SAR and did not display redox activity; however low cell permeability proved to be a challenge.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRqCoHvrkdmrVg90H21EOLACvtfcHk0lgV9VuGPTHtjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFanu7zF&md5=ba84b5ce65959a1967e99d40a2fb7825</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fjm400568p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400568p%26sid%3Dliteratum%253Aachs%26aulast%3DRaoof%26aufirst%3DA.%26aulast%3DDepledge%26aufirst%3DP.%26aulast%3DHamilton%26aufirst%3DN.%2BM.%26aulast%3DHamilton%26aufirst%3DN.%2BS.%26aulast%3DHitchin%26aufirst%3DJ.%2BR.%26aulast%3DHopkins%26aufirst%3DG.%2BV.%26aulast%3DJordan%26aufirst%3DA.%2BM.%26aulast%3DMaguire%26aufirst%3DL.%2BA.%26aulast%3DMcGonagle%26aufirst%3DA.%2BE.%26aulast%3DMould%26aufirst%3DD.%2BP.%26aulast%3DRushbrooke%26aufirst%3DM.%26aulast%3DSmall%26aufirst%3DH.%2BF.%26aulast%3DSmith%26aufirst%3DK.%2BM.%26aulast%3DThomson%26aufirst%3DG.%2BJ.%26aulast%3DTurlais%26aufirst%3DF.%26aulast%3DWaddell%26aufirst%3DI.%2BD.%26aulast%3DWaszkowycz%26aufirst%3DB.%26aulast%3DWatson%26aufirst%3DA.%2BJ.%26aulast%3DOgilvie%26aufirst%3DD.%2BJ.%26atitle%3DToxoflavins%2520and%2520deazaflavins%2520as%2520the%2520first%2520reported%2520selective%2520small%2520molecule%2520inhibitors%2520of%2520tyrosyl-DNA%2520phosphodiesterase%2520II%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D6352%26epage%3D6370%26doi%3D10.1021%2Fjm400568p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Chen, Y. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kesler, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sham, Y. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vince, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geraghty, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Z. Q.</span><span> </span><span class="NLM_article-title">The design, synthesis and biological evaluations of C-6 or C-7 substituted 2-hydroxyisoquinoline-1,3-diones as inhibitors of hepatitis C virus</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">467</span><span class="NLM_x">–</span> <span class="NLM_lpage">479</span><span class="refDoi"> DOI: 10.1016/j.bmc.2011.10.058</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=10.1016%2Fj.bmc.2011.10.058" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=22100256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=1%3ACAS%3A528%3ADC%252BC38XitlGlsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2012&pages=467-479&author=Y.+L.+Chenauthor=J.+Tangauthor=M.+J.+Keslerauthor=Y.+Y.+Shamauthor=R.+Vinceauthor=R.+J.+Geraghtyauthor=Z.+Q.+Wang&title=The+design%2C+synthesis+and+biological+evaluations+of+C-6+or+C-7+substituted+2-hydroxyisoquinoline-1%2C3-diones+as+inhibitors+of+hepatitis+C+virus&doi=10.1016%2Fj.bmc.2011.10.058"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">The design, synthesis and biological evaluations of C-6 or C-7 substituted 2-hydroxyisoquinoline-1,3-diones as inhibitors of hepatitis C virus</span></div><div class="casAuthors">Chen, Yue-Lei; Tang, Jing; Kesler, Matthew J.; Sham, Yuk Y.; Vince, Robert; Geraghty, Robert J.; Wang, Zhengqiang</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">467-479</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">C7-Substituted 2-hydroxyisoquinoline-1,3-diones inhibit the strand transfer of HIV integrase (IN) and the reverse-transcriptase-assocd. RNase H (RNH).  Hepatitis C virus (HCV) NS5B polymerase shares a similar active site fold to RNH and IN, suggesting that N-hydroxyimides could be useful inhibitor scaffolds of HCV via targeting the NS5B.  Herein is described the design, chem. synthesis, replicon and biochem. assays, and mol. docking of C-6 or C-7 aryl substituted 2-hydroxyisoquinoline-1,3-diones as novel HCV inhibitors.  The synthesis involved an improved and clean cyclization method, which allowed the convenient prepn. of various analogs.  Biol. studies revealed that the C-6 analogs, a previously unknown chemotype, consistently inhibit both HCV replicon and recombinant NS5B at low micromolar range.  Mol. modeling studies suggest that these inhibitors may bind to the NS5B active site.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnGy-yoKIh_7Vg90H21EOLACvtfcHk0lgV9VuGPTHtjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XitlGlsA%253D%253D&md5=9d03fdf85f6df43aa80eb51611f572e3</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2011.10.058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2011.10.058%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%2BL.%26aulast%3DTang%26aufirst%3DJ.%26aulast%3DKesler%26aufirst%3DM.%2BJ.%26aulast%3DSham%26aufirst%3DY.%2BY.%26aulast%3DVince%26aufirst%3DR.%26aulast%3DGeraghty%26aufirst%3DR.%2BJ.%26aulast%3DWang%26aufirst%3DZ.%2BQ.%26atitle%3DThe%2520design%252C%2520synthesis%2520and%2520biological%2520evaluations%2520of%2520C-6%2520or%2520C-7%2520substituted%25202-hydroxyisoquinoline-1%252C3-diones%2520as%2520inhibitors%2520of%2520hepatitis%2520C%2520virus%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2012%26volume%3D20%26spage%3D467%26epage%3D479%26doi%3D10.1016%2Fj.bmc.2011.10.058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Tsou, H. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Otteng, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tran, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floyd, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reich, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Birnberg, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kutterer, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ayral-Kaloustian, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ravi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nilakantan, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grillo, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGinnis, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rabindran, S. K.</span><span> </span><span class="NLM_article-title">4-(Phenylaminomethylene)isoquinoline-1,3(2<i>H</i>,4<i>H</i>)-diones as potent and selective inhibitors of the cyclin-dependent kinase 4 (CDK4)</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">3507</span><span class="NLM_x">–</span> <span class="NLM_lpage">3525</span><span class="refDoi"> DOI: 10.1021/jm800072z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm800072z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=1%3ACAS%3A528%3ADC%252BD1cXmtF2rtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=3507-3525&author=H.+R.+Tsouauthor=M.+Ottengauthor=T.+Tranauthor=M.+B.+Floydauthor=M.+Reichauthor=G.+Birnbergauthor=K.+Kuttererauthor=S.+Ayral-Kaloustianauthor=M.+Raviauthor=R.+Nilakantanauthor=M.+Grilloauthor=J.+P.+McGinnisauthor=S.+K.+Rabindran&title=4-%28Phenylaminomethylene%29isoquinoline-1%2C3%282H%2C4H%29-diones+as+potent+and+selective+inhibitors+of+the+cyclin-dependent+kinase+4+%28CDK4%29&doi=10.1021%2Fjm800072z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">4-(Phenylaminomethylene)isoquinoline-1,3(2H,4H)-diones as Potent and Selective Inhibitors of the Cyclin-Dependent Kinase 4 (CDK4)</span></div><div class="casAuthors">Tsou, Hwei-Ru; Otteng, Mercy; Tran, Tritin; Floyd, M. Brawner, Jr.; Reich, Marvin; Birnberg, Gary; Kutterer, Kristina; Ayral-Kaloustian, Semiramis; Ravi, Malini; Nilakantan, Ramaswamy; Grillo, Mary; McGinnis, John P.; Rabindran, Sridhar K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3507-3525</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The cyclin-dependent kinases (CDKs), as complexes with their resp. partners, the cyclins, are crit. regulators of cell cycle progression.  Because aberrant regulations of CDK4/cyclin D1 lead to uncontrolled cell proliferation, a hallmark of cancer, small-mol. inhibitors of CDK4/cyclin D1 are attractive as prospective antitumor agents.  The series of 4-(phenylaminomethylene)isoquinoline-1,3(2H,4H)-dione derivs. reported here represents a novel class of potent inhibitors that selectively inhibit CDK4 over CDK2 and CDK1 activities.  In the headpiece of the 4-(phenylaminomethylene)isoquinoline-1,3(2H,4H)-dione, a basic amine substitutent is required on the aniline ring for the CDK4 inhibitory activity.  The inhibitory activity is further enhanced when an aryl or heteroaryl substituent is introduced at the C-6 position of the isoquinoline-1,3(2H,4H)-dione core.  We present here SAR data and a CDK4 mimic model that explains the binding, potency, and selectivity of our CDK4 selective inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq09yU47pubWbVg90H21EOLACvtfcHk0lgV9VuGPTHtjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXmtF2rtrc%253D&md5=35083ce22416a584311a810ecf2b624d</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm800072z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm800072z%26sid%3Dliteratum%253Aachs%26aulast%3DTsou%26aufirst%3DH.%2BR.%26aulast%3DOtteng%26aufirst%3DM.%26aulast%3DTran%26aufirst%3DT.%26aulast%3DFloyd%26aufirst%3DM.%2BB.%26aulast%3DReich%26aufirst%3DM.%26aulast%3DBirnberg%26aufirst%3DG.%26aulast%3DKutterer%26aufirst%3DK.%26aulast%3DAyral-Kaloustian%26aufirst%3DS.%26aulast%3DRavi%26aufirst%3DM.%26aulast%3DNilakantan%26aufirst%3DR.%26aulast%3DGrillo%26aufirst%3DM.%26aulast%3DMcGinnis%26aufirst%3DJ.%2BP.%26aulast%3DRabindran%26aufirst%3DS.%2BK.%26atitle%3D4-%2528Phenylaminomethylene%2529isoquinoline-1%252C3%25282H%252C4H%2529-diones%2520as%2520potent%2520and%2520selective%2520inhibitors%2520of%2520the%2520cyclin-dependent%2520kinase%25204%2520%2528CDK4%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D3507%26epage%3D3525%26doi%3D10.1021%2Fjm800072z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Castro, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jerez, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gil, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Calderon, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Domenech, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nueda, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinez, A.</span><span> </span><span class="NLM_article-title">CODES, a novel procedure for ligand-based virtual screening: PDE7 inhibitors as an application example</span> <span class="citation_source-journal">Eur. J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">1349</span><span class="NLM_x">–</span> <span class="NLM_lpage">1359</span><span class="refDoi"> DOI: 10.1016/j.ejmech.2007.10.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=10.1016%2Fj.ejmech.2007.10.027" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2008&pages=1349-1359&author=A.+Castroauthor=M.+J.+Jerezauthor=C.+Gilauthor=F.+Calderonauthor=T.+Domenechauthor=A.+Nuedaauthor=A.+Martinez&title=CODES%2C+a+novel+procedure+for+ligand-based+virtual+screening%3A+PDE7+inhibitors+as+an+application+example&doi=10.1016%2Fj.ejmech.2007.10.027"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2007.10.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2007.10.027%26sid%3Dliteratum%253Aachs%26aulast%3DCastro%26aufirst%3DA.%26aulast%3DJerez%26aufirst%3DM.%2BJ.%26aulast%3DGil%26aufirst%3DC.%26aulast%3DCalderon%26aufirst%3DF.%26aulast%3DDomenech%26aufirst%3DT.%26aulast%3DNueda%26aufirst%3DA.%26aulast%3DMartinez%26aufirst%3DA.%26atitle%3DCODES%252C%2520a%2520novel%2520procedure%2520for%2520ligand-based%2520virtual%2520screening%253A%2520PDE7%2520inhibitors%2520as%2520an%2520application%2520example%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2008%26volume%3D43%26spage%3D1349%26epage%3D1359%26doi%3D10.1016%2Fj.ejmech.2007.10.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Jin, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, D. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gong, Y. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, R. Q.</span><span> </span><span class="NLM_article-title">Clicking fluoroionophores onto mesoporous silicas: a universal strategy toward efficient fluorescent surface sensors for metal ions</span> <span class="citation_source-journal">Anal. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">82</span><span class="NLM_x">, </span> <span class="NLM_fpage">6343</span><span class="NLM_x">–</span> <span class="NLM_lpage">6346</span><span class="refDoi"> DOI: 10.1021/ac101305e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ac101305e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=82&publication_year=2010&pages=6343-6346&author=Z.+Jinauthor=X.+B.+Zhangauthor=D.+X.+Xieauthor=Y.+J.+Gongauthor=J.+Zhangauthor=X.+Chenauthor=G.+L.+Shenauthor=R.+Q.+Yu&title=Clicking+fluoroionophores+onto+mesoporous+silicas%3A+a+universal+strategy+toward+efficient+fluorescent+surface+sensors+for+metal+ions&doi=10.1021%2Fac101305e"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fac101305e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac101305e%26sid%3Dliteratum%253Aachs%26aulast%3DJin%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DX.%2BB.%26aulast%3DXie%26aufirst%3DD.%2BX.%26aulast%3DGong%26aufirst%3DY.%2BJ.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DShen%26aufirst%3DG.%2BL.%26aulast%3DYu%26aufirst%3DR.%2BQ.%26atitle%3DClicking%2520fluoroionophores%2520onto%2520mesoporous%2520silicas%253A%2520a%2520universal%2520strategy%2520toward%2520efficient%2520fluorescent%2520surface%2520sensors%2520for%2520metal%2520ions%26jtitle%3DAnal.%2520Chem.%26date%3D2010%26volume%3D82%26spage%3D6343%26epage%3D6346%26doi%3D10.1021%2Fac101305e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Ali, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jackson, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howells, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maxwell, A.</span><span> </span><span class="NLM_article-title">The 43-kilodalton N-terminal fragment of the DNA gyrase B protein hydrolyzes ATP and binds coumarin drugs</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">2717</span><span class="NLM_x">–</span> <span class="NLM_lpage">2724</span><span class="refDoi"> DOI: 10.1021/bi00061a033</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi00061a033" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=1%3ACAS%3A528%3ADyaK3sXhtF2lt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=1993&pages=2717-2724&author=J.+A.+Aliauthor=A.+P.+Jacksonauthor=A.+J.+Howellsauthor=A.+Maxwell&title=The+43-kilodalton+N-terminal+fragment+of+the+DNA+gyrase+B+protein+hydrolyzes+ATP+and+binds+coumarin+drugs&doi=10.1021%2Fbi00061a033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">The 43-kilodalton N-terminal fragment of the DNA gyrase B protein hydrolyzes ATP and binds coumarin drugs</span></div><div class="casAuthors">Ali, Janid A.; Jackson, Andrew P.; Howells, Alison J.; Maxwell, Anthony</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2717-24</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    </div><div class="casAbstract">A gene encoding a 43-kDa protein corresponding to the N-terminal fragment of the DNA gyrase B subunit was cloned and overexpressed.  It was shown that this protein hydrolyzes ATP and binds coumarin drugs.  The hydrolysis of ATP showed distinctly non-Michaelis-Menten kinetics and was consistent with a scheme in which the active form of the protein was a dimer, a conclusion supported by mol. wt. studies.  The coumarin drugs bound very tightly to the 43-kDa fragment, with novobiocin binding to the protein monomer and coumermycin A1 apparently inducing the formation of a dimer.  The implications of these results with respect to the mechanism of supercoiling by DNA gyrase and the inhibition of gyrase by coumarin drugs were discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqKDubLhzserVg90H21EOLACvtfcHk0lgeodef39cKSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXhtF2lt74%253D&md5=c0fb5f7a4c5aa653cb0a26473afc3d74</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fbi00061a033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00061a033%26sid%3Dliteratum%253Aachs%26aulast%3DAli%26aufirst%3DJ.%2BA.%26aulast%3DJackson%26aufirst%3DA.%2BP.%26aulast%3DHowells%26aufirst%3DA.%2BJ.%26aulast%3DMaxwell%26aufirst%3DA.%26atitle%3DThe%252043-kilodalton%2520N-terminal%2520fragment%2520of%2520the%2520DNA%2520gyrase%2520B%2520protein%2520hydrolyzes%2520ATP%2520and%2520binds%2520coumarin%2520drugs%26jtitle%3DBiochemistry%26date%3D1993%26volume%3D32%26spage%3D2717%26epage%3D2724%26doi%3D10.1021%2Fbi00061a033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Repasky, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, L. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenwood, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halgren, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanschagrin, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mainz, D. T.</span><span> </span><span class="NLM_article-title">Extra precision glide: docking and scoring incorporating a model of hydrophobic enclosure for protein-ligand complexes</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">6177</span><span class="NLM_x">–</span> <span class="NLM_lpage">6196</span><span class="refDoi"> DOI: 10.1021/jm051256o</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm051256o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=1%3ACAS%3A528%3ADC%252BD28XpvVGmurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=6177-6196&author=R.+A.+Friesnerauthor=R.+B.+Murphyauthor=M.+P.+Repaskyauthor=L.+L.+Fryeauthor=J.+R.+Greenwoodauthor=T.+A.+Halgrenauthor=P.+C.+Sanschagrinauthor=D.+T.+Mainz&title=Extra+precision+glide%3A+docking+and+scoring+incorporating+a+model+of+hydrophobic+enclosure+for+protein-ligand+complexes&doi=10.1021%2Fjm051256o"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Extra Precision Glide: Docking and Scoring Incorporating a Model of Hydrophobic Enclosure for Protein-Ligand Complexes</span></div><div class="casAuthors">Friesner, Richard A.; Murphy, Robert B.; Repasky, Matthew P.; Frye, Leah L.; Greenwood, Jeremy R.; Halgren, Thomas A.; Sanschagrin, Paul C.; Mainz, Daniel T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6177-6196</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel scoring function to est. protein-ligand binding affinities has been developed and implemented as the Glide 4.0 XP scoring function and docking protocol.  In addn. to unique water desolvation energy terms, protein-ligand structural motifs leading to enhanced binding affinity are included:(1) hydrophobic enclosure where groups of lipophilic ligand atoms are enclosed on opposite faces by lipophilic protein atoms, (2) neutral-neutral single or correlated hydrogen bonds in a hydrophobically enclosed environment, and (3) five categories of charged-charged hydrogen bonds.  The XP scoring function and docking protocol have been developed to reproduce exptl. binding affinities for a set of 198 complexes (RMSDs of 2.26 and 1.73 kcal/mol over all and well-docked ligands, resp.) and to yield quality enrichments for a set of fifteen screens of pharmaceutical importance.  Enrichment results demonstrate the importance of the novel XP mol. recognition and water scoring in sepg. active and inactive ligands and avoiding false positives.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohZiwN12vaY7Vg90H21EOLACvtfcHk0lgeodef39cKSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XpvVGmurg%253D&md5=ea428c82ead0d8c27f8c1a7b694a1edf</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm051256o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm051256o%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DGreenwood%26aufirst%3DJ.%2BR.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DSanschagrin%26aufirst%3DP.%2BC.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26atitle%3DExtra%2520precision%2520glide%253A%2520docking%2520and%2520scoring%2520incorporating%2520a%2520model%2520of%2520hydrophobic%2520enclosure%2520for%2520protein-ligand%2520complexes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D6177%26epage%3D6196%26doi%3D10.1021%2Fjm051256o" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Pommier, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, S. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Das, B. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murai, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marchand, C.</span><span> </span><span class="NLM_article-title">Tyrosyl-DNA-phosphodiesterases (TDP1 and TDP2)</span> <span class="citation_source-journal">DNA Repair</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">114</span><span class="NLM_x">–</span> <span class="NLM_lpage">129</span><span class="refDoi"> DOI: 10.1016/j.dnarep.2014.03.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=10.1016%2Fj.dnarep.2014.03.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=24856239" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=1%3ACAS%3A528%3ADC%252BC2cXoslCnurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2014&pages=114-129&author=Y.+Pommierauthor=S.+Y.+Huangauthor=R.+Gaoauthor=B.+B.+Dasauthor=J.+Muraiauthor=C.+Marchand&title=Tyrosyl-DNA-phosphodiesterases+%28TDP1+and+TDP2%29&doi=10.1016%2Fj.dnarep.2014.03.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Tyrosyl-DNA-phosphodiesterases (TDP1 and TDP2)</span></div><div class="casAuthors">Pommier, Yves; Huang, Shar-yin N.; Gao, Rui; Das, Benu Brata; Murai, Junko; Marchand, Christophe</div><div class="citationInfo"><span class="NLM_cas:title">DNA Repair</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">114-129</span>CODEN:
                <span class="NLM_cas:coden">DRNEAR</span>;
        ISSN:<span class="NLM_cas:issn">1568-7864</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  TDP1 and TDP2 were discovered and named based on the fact they process 3'- and 5'-DNA ends by excising irreversible protein tyrosyl-DNA complexes involving topoisomerases I and II, resp.  Yet, both enzymes have an extended spectrum of activities.  TDP1 not only excises trapped topoisomerases I (Top1 in the nucleus and Top1mt in mitochondria), but also repairs oxidative damage-induced 3'-phosphoglycolates and alkylation damage-induced DNA breaks, and excises chain terminating anticancer and antiviral nucleosides in the nucleus and mitochondria.  The repair function of TDP2 is devoted to the excision of topoisomerase II- and potentially topoisomerases III-DNA adducts.  TDP2 is also essential for the life cycle of picornaviruses (important human and bovine pathogens) as it unlinks VPg proteins from the 5'-end of the viral RNA genome.  Moreover, TDP2 has been involved in signal transduction (under the former names of TTRAP or EAPII).  The DNA repair partners of TDP1 include PARP1, XRCC1, ligase III and PNKP from the base excision repair (BER) pathway.  By contrast, TDP2 repair functions are coordinated with Ku and ligase IV in the non-homologous end joining pathway (NHEJ).  This article summarizes and compares the biochem., functions, and post-translational regulation of TDP1 and TDP2, as well as the relevance of TDP1 and TDP2 as determinants of response to anticancer agents.  The authors discuss the rationale for developing TDP inhibitors for combinations with topoisomerase inhibitors (topotecan, irinotecan, doxorubicin, etoposide, mitoxantrone) and DNA damaging agents (temozolomide, bleomycin, cytarabine, and ionizing radiation), and as novel antiviral agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDBH91msgKDbVg90H21EOLACvtfcHk0lgY69kw-dYyjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXoslCnurc%253D&md5=8d0487ce63d11bd7312583c141b2269b</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.dnarep.2014.03.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.dnarep.2014.03.020%26sid%3Dliteratum%253Aachs%26aulast%3DPommier%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DS.%2BY.%26aulast%3DGao%26aufirst%3DR.%26aulast%3DDas%26aufirst%3DB.%2BB.%26aulast%3DMurai%26aufirst%3DJ.%26aulast%3DMarchand%26aufirst%3DC.%26atitle%3DTyrosyl-DNA-phosphodiesterases%2520%2528TDP1%2520and%2520TDP2%2529%26jtitle%3DDNA%2520Repair%26date%3D2014%26volume%3D19%26spage%3D114%26epage%3D129%26doi%3D10.1016%2Fj.dnarep.2014.03.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Marchand, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lea, W. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jadhav, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dexheimer, T. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Austin, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inglese, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pommier, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span> </span><span class="NLM_article-title">Identification of phosphotyrosine mimetic inhibitors of human tyrosyl-DNA phosphodiesterase I by a novel alphaScreen high-throughput assay</span> <span class="citation_source-journal">Mol. Cancer Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">240</span><span class="NLM_x">–</span> <span class="NLM_lpage">248</span><span class="refDoi"> DOI: 10.1158/1535-7163.MCT-08-0878</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=10.1158%2F1535-7163.MCT-08-0878" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=19139134" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjvFSltw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=240-248&author=C.+Marchandauthor=W.+A.+Leaauthor=A.+Jadhavauthor=T.+S.+Dexheimerauthor=C.+P.+Austinauthor=J.+Ingleseauthor=Y.+Pommierauthor=A.+Simeonov&title=Identification+of+phosphotyrosine+mimetic+inhibitors+of+human+tyrosyl-DNA+phosphodiesterase+I+by+a+novel+alphaScreen+high-throughput+assay&doi=10.1158%2F1535-7163.MCT-08-0878"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of phosphotyrosine mimetic inhibitors of human tyrosyl-DNA phosphodiesterase I by a novel AlphaScreen high-throughput assay</span></div><div class="casAuthors">Marchand, Christophe; Lea, Wendy A.; Jadhav, Ajit; Dexheimer, Thomas S.; Austin, Christopher P.; Inglese, James; Pommier, Yves; Simeonov, Anton</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">240-248</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Tyrosyl-DNA phosphodiesterase I (Tdp1) resolves topoisomerase I (Top1)-DNA adducts accumulated from natural DNA damage as well as from the action of certain anticancer drugs.  Tdp1 catalyzes the hydrolysis of the phosphodiester bond between the catalytic tyrosine residue of topoisomerase I and the DNA 3'-phosphate.  Only a limited no. of weak inhibitors have been reported for Tdp1, and there is an unmet need to identify novel chemotypes through screening of chem. libraries.  Herein, we present an easily configured, highly miniaturized, and robust Tdp1 assay using the AlphaScreen technol.  Uninhibited enzyme reaction is assocd. with low signal, whereas inhibition leads to a gain of signal, making the present assay format esp. attractive for automated large-collection high-throughput screening.  We report the identification and initial characterization of four previously unreported inhibitors of Tdp1.  Among them, suramin, NF449, and methyl-3,4-dephostatin are phosphotyrosine mimetics that may act as Tdp1 substrate decoys.  We also report a novel biochem. assay using the SCAN1 Tdp1 mutant to study the mechanism of action of methyl-3,4-dephostatin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqw2O21mqWopLVg90H21EOLACvtfcHk0lgY69kw-dYyjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjvFSltw%253D%253D&md5=05a32d97be998b397a5dc3586b81bef4</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-08-0878&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-08-0878%26sid%3Dliteratum%253Aachs%26aulast%3DMarchand%26aufirst%3DC.%26aulast%3DLea%26aufirst%3DW.%2BA.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DDexheimer%26aufirst%3DT.%2BS.%26aulast%3DAustin%26aufirst%3DC.%2BP.%26aulast%3DInglese%26aufirst%3DJ.%26aulast%3DPommier%26aufirst%3DY.%26aulast%3DSimeonov%26aufirst%3DA.%26atitle%3DIdentification%2520of%2520phosphotyrosine%2520mimetic%2520inhibitors%2520of%2520human%2520tyrosyl-DNA%2520phosphodiesterase%2520I%2520by%2520a%2520novel%2520alphaScreen%2520high-throughput%2520assay%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2009%26volume%3D8%26spage%3D240%26epage%3D248%26doi%3D10.1158%2F1535-7163.MCT-08-0878" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Nguyen, T. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morrell, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Conda-Sheridan, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marchand, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Agama, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bermingam, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stephen, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chergui, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Naumova, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fisher, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Keefe, B. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pommier, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cushman, M.</span><span> </span><span class="NLM_article-title">Synthesis and biological evaluation of the first dual tyrosyl-DNA phosphodiesterase I (Tdp1)-topoisomerase I (Top1) inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">4457</span><span class="NLM_x">–</span> <span class="NLM_lpage">4478</span><span class="refDoi"> DOI: 10.1021/jm300335n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm300335n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=1%3ACAS%3A528%3ADC%252BC38XmtFartrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=4457-4478&author=T.+X.+Nguyenauthor=A.+Morrellauthor=M.+Conda-Sheridanauthor=C.+Marchandauthor=K.+Agamaauthor=A.+Bermingamauthor=A.+G.+Stephenauthor=A.+Cherguiauthor=A.+Naumovaauthor=R.+Fisherauthor=B.+R.+O%E2%80%99Keefeauthor=Y.+Pommierauthor=M.+Cushman&title=Synthesis+and+biological+evaluation+of+the+first+dual+tyrosyl-DNA+phosphodiesterase+I+%28Tdp1%29-topoisomerase+I+%28Top1%29+inhibitors&doi=10.1021%2Fjm300335n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and Biological Evaluation of the First Dual Tyrosyl-DNA Phosphodiesterase I (Tdp1)-Topoisomerase I (Top1) Inhibitors</span></div><div class="casAuthors">Nguyen, Trung Xuan; Morrell, Andrew; Conda-Sheridan, Martin; Marchand, Christophe; Agama, Keli; Berminham, Alun; Stephen, Andrew G.; Chergui, Adel; Naumova, Alena; Fisher, Robert; O'Keefe, Barry R.; Pommier, Yves; Cushman, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4457-4478</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Substances with dual tyrosyl-DNA phosphodiesterase I-topoisomerase I inhibitory activity in one low mol. wt. compd. would constitute a unique class of anticancer agents that could potentially have significant advantages over drugs that work against the individual enzymes.  The present study demonstrates the successful synthesis and evaluation of the first dual Top1-Tdp1 inhibitors, which are based on the indenoisoquinoline chemotype.  One bis(indenoisoquinoline) (I.2TFA) had significant activity against human Tdp1 (IC50 = 1.52 ± 0.05 μM), and it was also equipotent to camptothecin as a Top1 inhibitor.  Significant insights into enzyme-drug interactions were gained via structure-activity relationship studies of the series.  The present results also document the failure of the previously reported sulfonyl ester pharmacophore to confer Tdp1 inhibition in this indenoisoquinoline class of inhibitors even though it was demonstrated to work well for the steroid NSC 88915 (7).  The current study will facilitate future efforts to optimize dual Top1-Tdp1 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHBLLYpVMf5rVg90H21EOLACvtfcHk0lgY69kw-dYyjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmtFartrY%253D&md5=c6f74f01e36da72fe9d5e0c435b9e3dd</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fjm300335n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300335n%26sid%3Dliteratum%253Aachs%26aulast%3DNguyen%26aufirst%3DT.%2BX.%26aulast%3DMorrell%26aufirst%3DA.%26aulast%3DConda-Sheridan%26aufirst%3DM.%26aulast%3DMarchand%26aufirst%3DC.%26aulast%3DAgama%26aufirst%3DK.%26aulast%3DBermingam%26aufirst%3DA.%26aulast%3DStephen%26aufirst%3DA.%2BG.%26aulast%3DChergui%26aufirst%3DA.%26aulast%3DNaumova%26aufirst%3DA.%26aulast%3DFisher%26aufirst%3DR.%26aulast%3DO%25E2%2580%2599Keefe%26aufirst%3DB.%2BR.%26aulast%3DPommier%26aufirst%3DY.%26aulast%3DCushman%26aufirst%3DM.%26atitle%3DSynthesis%2520and%2520biological%2520evaluation%2520of%2520the%2520first%2520dual%2520tyrosyl-DNA%2520phosphodiesterase%2520I%2520%2528Tdp1%2529-topoisomerase%2520I%2520%2528Top1%2529%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D4457%26epage%3D4478%26doi%3D10.1021%2Fjm300335n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">McRobb, F. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Capuano, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crosby, I. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chalmers, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuriev, E.</span><span> </span><span class="NLM_article-title">Homology modeling and docking evaluation of aminergic G protein-coupled receptors</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">626</span><span class="NLM_x">–</span> <span class="NLM_lpage">637</span><span class="refDoi"> DOI: 10.1021/ci900444q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci900444q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=1%3ACAS%3A528%3ADC%252BC3cXisFals78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=626-637&author=F.+M.+McRobbauthor=B.+Capuanoauthor=I.+T.+Crosbyauthor=D.+K.+Chalmersauthor=E.+Yuriev&title=Homology+modeling+and+docking+evaluation+of+aminergic+G+protein-coupled+receptors&doi=10.1021%2Fci900444q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Homology Modeling and Docking Evaluation of Aminergic G Protein-Coupled Receptors</span></div><div class="casAuthors">McRobb, Fiona M.; Capuano, Ben; Crosby, Ian T.; Chalmers, David K.; Yuriev, Elizabeth</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">626-637</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We report the development of homol. models of dopamine (D2, D3, and D4), serotonin (5-HT1B, 5-HT2A, 5-HT2B, and 5-HT2C), histamine (H1), and muscarinic (M1) receptors, based on the high-resoln. structure of the β2-adrenergic receptor.  The homol. models were built and refined using Prime.  We have addressed the required modeling of extracellular loop 2, which is often implicated in ligand binding.  The orthosteric sites of the models were optimized using induced fit docking, to allow for side-chain flexibility, and the resulting receptor models have been evaluated using protein validation tools.  Of the nine homol. models developed, six models showed moderate to good enrichment in virtual screening expts. (5-HT2A, 5-HT1B, D2, 5-HT2C, D3, and M1).  The 5-HT2A receptor displayed the highest enrichment in virtual screening expts. with enrichment factors of 6.1, 6.9, and 5.9 at 2%, 5%, and 10%, resp., of the screened database.  However, three of the models require further refinement (5-HT2B, D4, and H1), due to difficulties in modeling some of the binding site residues as well as the extracellular loop 2.  Our effort also aims to supplement the limited no. of tested G protein-coupled receptor homol. models based on the β2 crystal structure that are freely available to the research community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpe67bPU5vSUrVg90H21EOLACvtfcHk0lhECYgW_v6L2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXisFals78%253D&md5=e9435245054966fd162f2cff4285ab11</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fci900444q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci900444q%26sid%3Dliteratum%253Aachs%26aulast%3DMcRobb%26aufirst%3DF.%2BM.%26aulast%3DCapuano%26aufirst%3DB.%26aulast%3DCrosby%26aufirst%3DI.%2BT.%26aulast%3DChalmers%26aufirst%3DD.%2BK.%26aulast%3DYuriev%26aufirst%3DE.%26atitle%3DHomology%2520modeling%2520and%2520docking%2520evaluation%2520of%2520aminergic%2520G%2520protein-coupled%2520receptors%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D626%26epage%3D637%26doi%3D10.1021%2Fci900444q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Goldfeld, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beuming, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span> </span><span class="NLM_article-title">Successful prediction of the intra and extracellular loops of four G-protein-coupled receptors</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">108</span><span class="NLM_x">, </span> <span class="NLM_fpage">8275</span><span class="NLM_x">–</span> <span class="NLM_lpage">8280</span><span class="refDoi"> DOI: 10.1073/pnas.1016951108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=10.1073%2Fpnas.1016951108" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=8275-8280&author=D.+A.+Goldfeldauthor=K.+Zhuauthor=T.+Beumingauthor=R.+A.+Friesner&title=Successful+prediction+of+the+intra+and+extracellular+loops+of+four+G-protein-coupled+receptors&doi=10.1073%2Fpnas.1016951108"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1016951108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1016951108%26sid%3Dliteratum%253Aachs%26aulast%3DGoldfeld%26aufirst%3DD.%2BA.%26aulast%3DZhu%26aufirst%3DK.%26aulast%3DBeuming%26aufirst%3DT.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DSuccessful%2520prediction%2520of%2520the%2520intra%2520and%2520extracellular%2520loops%2520of%2520four%2520G-protein-coupled%2520receptors%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26spage%3D8275%26epage%3D8280%26doi%3D10.1073%2Fpnas.1016951108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Madhavi Sastry, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adzhigirey, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Day, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Annabhimoju, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sherman, W.</span><span> </span><span class="NLM_article-title">Protein and ligand preparation: parameters, protocols, and influence on virtual screening enrichments</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">221</span><span class="NLM_x">–</span> <span class="NLM_lpage">234</span><span class="refDoi"> DOI: 10.1007/s10822-013-9644-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=10.1007%2Fs10822-013-9644-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=23579614" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmslalu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2013&pages=221-234&author=G.+Madhavi+Sastryauthor=M.+Adzhigireyauthor=T.+Dayauthor=R.+Annabhimojuauthor=W.+Sherman&title=Protein+and+ligand+preparation%3A+parameters%2C+protocols%2C+and+influence+on+virtual+screening+enrichments&doi=10.1007%2Fs10822-013-9644-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Protein and ligand preparation: parameters, protocols, and influence on virtual screening enrichments</span></div><div class="casAuthors">Madhavi Sastry, G.; Adzhigirey, Matvey; Day, Tyler; Annabhimoju, Ramakrishna; Sherman, Woody</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">221-234</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Structure-based virtual screening plays an important role in drug discovery and complements other screening approaches.  In general, protein crystal structures are prepd. prior to docking in order to add hydrogen atoms, optimize hydrogen bonds, remove at. clashes, and perform other operations that are not part of the x-ray crystal structure refinement process.  In addn., ligands must be prepd. to create 3-dimensional geometries, assign proper bond orders, and generate accessible tautomer and ionization states prior to virtual screening.  While the prerequisite for proper system prepn. is generally accepted in the field, an extensive study of the prepn. steps and their effect on virtual screening enrichments has not been performed.  In this work, we systematically explore each of the steps involved in prepg. a system for virtual screening.  We first explore a large no. of parameters using the Glide validation set of 36 crystal structures and 1,000 decoys.  We then apply a subset of protocols to the DUD database.  We show that database enrichment is improved with proper prepn. and that neglecting certain steps of the prepn. process produces a systematic degrdn. in enrichments, which can be large for some targets.  We provide examples illustrating the structural changes introduced by the prepn. that impact database enrichment.  While the work presented here was performed with the Protein Prepn. Wizard and Glide, the insights and guidance are expected to be generalizable to structure-based virtual screening with other docking methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruqN5nIOJtR7Vg90H21EOLACvtfcHk0lhECYgW_v6L2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmslalu7c%253D&md5=259a6d547ef3e1310e091fb50fe8de16</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1007%2Fs10822-013-9644-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-013-9644-8%26sid%3Dliteratum%253Aachs%26aulast%3DMadhavi%2BSastry%26aufirst%3DG.%26aulast%3DAdzhigirey%26aufirst%3DM.%26aulast%3DDay%26aufirst%3DT.%26aulast%3DAnnabhimoju%26aufirst%3DR.%26aulast%3DSherman%26aufirst%3DW.%26atitle%3DProtein%2520and%2520ligand%2520preparation%253A%2520parameters%252C%2520protocols%252C%2520and%2520influence%2520on%2520virtual%2520screening%2520enrichments%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2013%26volume%3D27%26spage%3D221%26epage%3D234%26doi%3D10.1007%2Fs10822-013-9644-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="note"><p class="first last">Schrödinger Release 2014-3: Schrödinger Suite 2014-3.</p></div><div class="NLM_citation" id="cit31a"><span><span class="NLM_label">(a) </span><div class="note"><p class="first last">Protein Preparation Wizard</p></div>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Protein+Preparation+Wizard."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31a&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit31b"><span><span class="NLM_label">(b) </span>Epik, <span class="NLM_edition">version 2.9</span>; <span class="NLM_publisher-name">Schrödinger, LLC</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Epik%2C+version+2.9%3B+Schr%C3%B6dinger%2C+LLC%3A+New+York%2C+2014."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31b&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DEpik%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit31c"><span><span class="NLM_label">(c) </span>Impact, <span class="NLM_edition">version 6.4</span>; <span class="NLM_publisher-name">Schrödinger, LLC</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Impact%2C+version+6.4%3B+Schr%C3%B6dinger%2C+LLC%3A+New+York%2C+2014."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31c&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DImpact%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit31d"><span><span class="NLM_label">(d) </span>Prime, <span class="NLM_edition">version 3.7</span>; <span class="NLM_publisher-name">Schrödinger, LLC</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Prime%2C+version+3.7%3B+Schr%C3%B6dinger%2C+LLC%3A+New+York%2C+2014."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31d&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DPrime%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maxwell, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">TiradoRives, J.</span><span> </span><span class="NLM_article-title">Development and testing of the OPLS all-atom force field on conformational energetics and properties of organic liquids</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">118</span><span class="NLM_x">, </span> <span class="NLM_fpage">11225</span><span class="NLM_x">–</span> <span class="NLM_lpage">11236</span><span class="refDoi"> DOI: 10.1021/ja9621760</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja9621760" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=1%3ACAS%3A528%3ADyaK28XmtlOitrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=1996&pages=11225-11236&author=W.+L.+Jorgensenauthor=D.+S.+Maxwellauthor=J.+TiradoRives&title=Development+and+testing+of+the+OPLS+all-atom+force+field+on+conformational+energetics+and+properties+of+organic+liquids&doi=10.1021%2Fja9621760"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Development and Testing of the OPLS All-Atom Force Field on Conformational Energetics and Properties of Organic Liquids</span></div><div class="casAuthors">Jorgensen, William L.; Maxwell, David S.; Tirado-Rives, Julian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">11225-11236</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The parametrization and testing of the OPLS all-atom force field for org. mols. and peptides are described.  Parameters for both torsional and nonbonded energetics have been derived, while the bond stretching and angle bending parameters have been adopted mostly from the AMBER all-atom force field.  The torsional parameters were detd. by fitting to rotational energy profiles obtained from ab initio MO calcns. at the RHF/6-31G*//RHF/6-31G* level for more than 50 org. mols. and ions.  The quality of the fits was high with av. errors for conformational energies of less than 0.2 kcal/mol.  The force-field results for mol. structures are also demonstrated to closely match the ab initio predictions.  The nonbonded parameters were developed in conjunction with Monte Carlo statistical mechanics simulations by computing thermodn. and structural properties for 34 pure org. liqs. including alkanes, alkenes, alcs., ethers, acetals, thiols, sulfides, disulfides, aldehydes, ketones, and amides.  Av. errors in comparison with exptl. data are 2% for heats of vaporization and densities.  The Monte Carlo simulations included sampling all internal and intermol. degrees of freedom.  It is found that such non-polar and monofunctional systems do not show significant condensed-phase effects on internal energies in going from the gas phase to the pure liqs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhGotSev_b1LVg90H21EOLACvtfcHk0lijVg72CD2KJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XmtlOitrs%253D&md5=fef2924a69421881390282aa309ae91b</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fja9621760&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja9621760%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DMaxwell%26aufirst%3DD.%2BS.%26aulast%3DTiradoRives%26aufirst%3DJ.%26atitle%3DDevelopment%2520and%2520testing%2520of%2520the%2520OPLS%2520all-atom%2520force%2520field%2520on%2520conformational%2520energetics%2520and%2520properties%2520of%2520organic%2520liquids%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1996%26volume%3D118%26spage%3D11225%26epage%3D11236%26doi%3D10.1021%2Fja9621760" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><div class="note"><p class="first last">Schrödinger Release 2014-3:</p></div> <span class="citation_source-book">LigPrep</span>, <span class="NLM_edition">version 3.1</span>; <span class="NLM_publisher-name">Schrödinger, LLC</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Schr%C3%B6dinger+Release+2014-3%3ALigPrep%2C+version+3.1%3B+Schr%C3%B6dinger%2C+LLC%3A+New+York%2C+2014."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DLigPrep%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31a','cit31b','cit31c','cit31d'],'ref32':['cit32'],'ref33':['cit33']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 31 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Sean H. Kennedy, Mark N. Schaeff, <span class="NLM_string-name hlFld-ContribAuthor">Douglas A. Klumpp</span>. </span><span class="cited-content_cbyCitation_article-title">Superelectrophiles in Synthesis: Preparation of Aromatic Imides. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Organic Chemistry</span><span> <strong>2019,</strong> <em>84 </em>
                                    (21)
                                     , 14133-14140. <a href="https://doi.org/10.1021/acs.joc.9b01437" title="DOI URL">https://doi.org/10.1021/acs.joc.9b01437</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.joc.9b01437&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.joc.9b01437%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DSuperelectrophiles%252Bin%252BSynthesis%25253A%252BPreparation%252Bof%252BAromatic%252BImides%26aulast%3DKennedy%26aufirst%3DSean%2BH.%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D31052019%26date%3D09102019%26date%3D01102019%26volume%3D84%26issue%3D21%26spage%3D14133%26epage%3D14140" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jayakanth Kankanala, Carlos J. A. Ribeiro, Evgeny Kiselev, Azhar Ravji, Jessica Williams, Jiashu Xie, Hideki Aihara, Yves Pommier, <span class="NLM_string-name hlFld-ContribAuthor">Zhengqiang Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Novel Deazaflavin Analogues Potently Inhibited Tyrosyl DNA Phosphodiesterase 2 (TDP2) and Strongly Sensitized Cancer Cells toward Treatment with Topoisomerase II (TOP2) Poison Etoposide. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (9)
                                     , 4669-4682. <a href="https://doi.org/10.1021/acs.jmedchem.9b00274" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00274</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00274&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b00274%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DNovel%252BDeazaflavin%252BAnalogues%252BPotently%252BInhibited%252BTyrosyl%252BDNA%252BPhosphodiesterase%252B2%252B%252528TDP2%252529%252Band%252BStrongly%252BSensitized%252BCancer%252BCells%252Btoward%252BTreatment%252Bwith%252BTopoisomerase%252BII%252B%252528TOP2%252529%252BPoison%252BEtoposide%26aulast%3DKankanala%26aufirst%3DJayakanth%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D11022019%26date%3D30042019%26date%3D18042019%26volume%3D62%26issue%3D9%26spage%3D4669%26epage%3D4682" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Yuanyong Yang, Yingxian Li, Cheng Cheng, Guo Yang, Shuiying Wan, Jiquan Zhang, Yuanhu Mao, Yonglong Zhao, Lin Zhang, Chun Li, <span class="NLM_string-name hlFld-ContribAuthor">Lei Tang</span>. </span><span class="cited-content_cbyCitation_article-title">Reductant-Free Aerobic Hydroxylation of Isoquinoline-1,3(2H,4H)-dione Derivatives. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Organic Chemistry</span><span> <strong>2019,</strong> <em>84 </em>
                                    (4)
                                     , 2316-2324. <a href="https://doi.org/10.1021/acs.joc.8b02977" title="DOI URL">https://doi.org/10.1021/acs.joc.8b02977</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.joc.8b02977&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.joc.8b02977%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DReductant-Free%252BAerobic%252BHydroxylation%252Bof%252BIsoquinoline-1%25252C3%2525282H%25252C4H%252529-dione%252BDerivatives%26aulast%3DYang%26aufirst%3DYuanyong%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D22112018%26date%3D30012019%26date%3D15012019%26volume%3D84%26issue%3D4%26spage%3D2316%26epage%3D2324" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Xicheng Yang, Haihua Yu, Yulong Xu, <span class="NLM_string-name hlFld-ContribAuthor">Liming Shao</span>. </span><span class="cited-content_cbyCitation_article-title">Regioselective Access to 3-Aryl-1-aminoisoquinolines via Nickel(I)-Catalyzed C–C and C–N Cascade Coupling Reactions from the Substituted 2-(Cyanomethyl)benzonitriles. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Organic Chemistry</span><span> <strong>2018,</strong> <em>83 </em>
                                    (17)
                                     , 9682-9695. <a href="https://doi.org/10.1021/acs.joc.8b01159" title="DOI URL">https://doi.org/10.1021/acs.joc.8b01159</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.joc.8b01159&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.joc.8b01159%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DRegioselective%252BAccess%252Bto%252B3-Aryl-1-aminoisoquinolines%252Bvia%252BNickel%252528I%252529-Catalyzed%252BC%2525E2%252580%252593C%252Band%252BC%2525E2%252580%252593N%252BCascade%252BCoupling%252BReactions%252Bfrom%252Bthe%252BSubstituted%252B2-%252528Cyanomethyl%252529benzonitriles%26aulast%3DYang%26aufirst%3DXicheng%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D07052018%26date%3D27082018%26date%3D14082018%26volume%3D83%26issue%3D17%26spage%3D9682%26epage%3D9695" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yu  Zhang</span>, <span class="hlFld-ContribAuthor ">Xiao‐Zhi  He</span>, <span class="hlFld-ContribAuthor ">Hao  Yang</span>, <span class="hlFld-ContribAuthor ">Hai‐Yang  Liu</span>, <span class="hlFld-ContribAuthor ">Lin‐Kun  An</span>. </span><span class="cited-content_cbyCitation_article-title">Robustadial A and B from
              
                Eucalyptus globulus
              
              Labill. and their anticancer activity as selective
              tyrosyl‐DNA
              phosphodiesterase 2 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Phytotherapy Research</span><span> <strong>2021,</strong> <em>111 </em><a href="https://doi.org/10.1002/ptr.7207" title="DOI URL">https://doi.org/10.1002/ptr.7207</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ptr.7207&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fptr.7207%26sid%3Dliteratum%253Aachs%26jtitle%3DPhytotherapy%2520Research%26atitle%3DRobustadial%252BA%252Band%252BB%252Bfrom%252BEucalyptus%252Bglobulus%252BLabill.%252Band%252Btheir%252Banticancer%252Bactivity%252Bas%252Bselective%252Btyrosyl%2525E2%252580%252590DNA%252Bphosphodiesterase%252B2%252Binhibitors%26aulast%3DZhang%26aufirst%3DYu%26date%3D2021%26date%3D2021%26volume%3D111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hao  Yang</span>, <span class="hlFld-ContribAuthor ">Xiao-Qing  Zhu</span>, <span class="hlFld-ContribAuthor ">Wenjie  Wang</span>, <span class="hlFld-ContribAuthor ">Yu  Chen</span>, <span class="hlFld-ContribAuthor ">Zhu  Hu</span>, <span class="hlFld-ContribAuthor ">Yu  Zhang</span>, <span class="hlFld-ContribAuthor ">De-Xuan  Hu</span>, <span class="hlFld-ContribAuthor ">Le-Mao  Yu</span>, <span class="hlFld-ContribAuthor ">Keli  Agama</span>, <span class="hlFld-ContribAuthor ">Yves  Pommier</span>, <span class="hlFld-ContribAuthor ">Lin-Kun  An</span>. </span><span class="cited-content_cbyCitation_article-title">The synthesis of furoquinolinedione and isoxazoloquinolinedione derivatives as selective Tyrosyl-DNA phosphodiesterase 2 (TDP2) inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2021,</strong> <em>111 </em>, 104881. <a href="https://doi.org/10.1016/j.bioorg.2021.104881" title="DOI URL">https://doi.org/10.1016/j.bioorg.2021.104881</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2021.104881&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2021.104881%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DThe%252Bsynthesis%252Bof%252Bfuroquinolinedione%252Band%252Bisoxazoloquinolinedione%252Bderivatives%252Bas%252Bselective%252BTyrosyl-DNA%252Bphosphodiesterase%252B2%252B%252528TDP2%252529%252Binhibitors%26aulast%3DYang%26aufirst%3DHao%26date%3D2021%26volume%3D111%26spage%3D104881" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ying‐Xian  Li</span>, <span class="hlFld-ContribAuthor ">Xiao‐Bing  Feng</span>, <span class="hlFld-ContribAuthor ">Xue‐Min  Jia</span>, <span class="hlFld-ContribAuthor ">Huang  Jin</span>, <span class="hlFld-ContribAuthor ">Fei  Chen</span>, <span class="hlFld-ContribAuthor ">Yong‐Long  Zhao</span>, <span class="hlFld-ContribAuthor ">Ji‐Quan  Zhang</span>, <span class="hlFld-ContribAuthor ">Jian‐Ta  Wang</span>, <span class="hlFld-ContribAuthor ">Bing  Guo</span>, <span class="hlFld-ContribAuthor ">Lei  Tang</span>, <span class="hlFld-ContribAuthor ">Yuan‐Yong  Yang</span>. </span><span class="cited-content_cbyCitation_article-title">Regiodivergent Functionalization of Isoquinoline‐1,3(2
              H
              ,4
              H
              )‐dione Derivatives via Aerobic Umpolung. </span><span class="cited-content_cbyCitation_journal-name">Asian Journal of Organic Chemistry</span><span> <strong>2021,</strong> <em>10 </em>
                                    (6)
                                     , 1508-1513. <a href="https://doi.org/10.1002/ajoc.202100194" title="DOI URL">https://doi.org/10.1002/ajoc.202100194</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ajoc.202100194&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fajoc.202100194%26sid%3Dliteratum%253Aachs%26jtitle%3DAsian%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DRegiodivergent%252BFunctionalization%252Bof%252BIsoquinoline%2525E2%252580%2525901%25252C3%2525282%252BH%252B%25252C4%252BH%252B%252529%2525E2%252580%252590dione%252BDerivatives%252Bvia%252BAerobic%252BUmpolung%26aulast%3DLi%26aufirst%3DYing%25E2%2580%2590Xian%26date%3D2021%26date%3D2021%26volume%3D10%26issue%3D6%26spage%3D1508%26epage%3D1513" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shar-yin N.  Huang</span>, <span class="hlFld-ContribAuthor ">Stephanie A.  Michaels</span>, <span class="hlFld-ContribAuthor ">Brianna B.  Mitchell</span>, <span class="hlFld-ContribAuthor ">Nadim  Majdalani</span>, <span class="hlFld-ContribAuthor ">Arnaud  Vanden Broeck</span>, <span class="hlFld-ContribAuthor ">Andres  Canela</span>, <span class="hlFld-ContribAuthor ">Yuk-Ching  Tse-Dinh</span>, <span class="hlFld-ContribAuthor ">Valerie  Lamour</span>, <span class="hlFld-ContribAuthor ">Yves  Pommier</span>. </span><span class="cited-content_cbyCitation_article-title">Exonuclease VII repairs quinolone-induced damage by resolving DNA gyrase cleavage complexes. </span><span class="cited-content_cbyCitation_journal-name">Science Advances</span><span> <strong>2021,</strong> <em>7 </em>
                                    (10)
                                     , eabe0384. <a href="https://doi.org/10.1126/sciadv.abe0384" title="DOI URL">https://doi.org/10.1126/sciadv.abe0384</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1126/sciadv.abe0384&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1126%2Fsciadv.abe0384%26sid%3Dliteratum%253Aachs%26jtitle%3DScience%2520Advances%26atitle%3DExonuclease%252BVII%252Brepairs%252Bquinolone-induced%252Bdamage%252Bby%252Bresolving%252BDNA%252Bgyrase%252Bcleavage%252Bcomplexes%26aulast%3DHuang%26aufirst%3DShar-yin%2BN.%26date%3D2021%26date%3D2021%26volume%3D7%26issue%3D10%26spage%3Deabe0384" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sameera  Senaweera</span>, <span class="hlFld-ContribAuthor ">Tianyu  He</span>, <span class="hlFld-ContribAuthor ">Haixi  Cui</span>, <span class="hlFld-ContribAuthor ">Hideki  Aihara</span>, <span class="hlFld-ContribAuthor ">Zhengqiang  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">4-benzylideneisoquinoline-1,3(2H,4H)-diones as tyrosyl DNA phosphodiesterase 2 (TDP2) inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Medicinal Chemistry Research</span><span> <strong>2021,</strong> <em>30 </em>
                                    (2)
                                     , 371-386. <a href="https://doi.org/10.1007/s00044-020-02662-w" title="DOI URL">https://doi.org/10.1007/s00044-020-02662-w</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00044-020-02662-w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00044-020-02662-w%26sid%3Dliteratum%253Aachs%26jtitle%3DMedicinal%2520Chemistry%2520Research%26atitle%3D4-benzylideneisoquinoline-1%25252C3%2525282H%25252C4H%252529-diones%252Bas%252Btyrosyl%252BDNA%252Bphosphodiesterase%252B2%252B%252528TDP2%252529%252Binhibitors%26aulast%3DSenaweera%26aufirst%3DSameera%26date%3D2021%26date%3D2020%26volume%3D30%26issue%3D2%26spage%3D371%26epage%3D386" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Cheng  Cheng</span>, <span class="hlFld-ContribAuthor ">Ying-Xian  Li</span>, <span class="hlFld-ContribAuthor ">Xue-Min  Jia</span>, <span class="hlFld-ContribAuthor ">Ji-Quan  Zhang</span>, <span class="hlFld-ContribAuthor ">Yong-Long  Zhao</span>, <span class="hlFld-ContribAuthor ">Wei  Feng</span>, <span class="hlFld-ContribAuthor ">Lei  Tang</span>, <span class="hlFld-ContribAuthor ">Yuan-Yong  Yang</span>. </span><span class="cited-content_cbyCitation_article-title">Enantioselective amination of 4-alkylisoquinoline-1,3(2
              H
              ,4
              H
              )-dione derivatives. </span><span class="cited-content_cbyCitation_journal-name">RSC Advances</span><span> <strong>2020,</strong> <em>10 </em>
                                    (70)
                                     , 42912-42915. <a href="https://doi.org/10.1039/D0RA07806A" title="DOI URL">https://doi.org/10.1039/D0RA07806A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0RA07806A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0RA07806A%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Advances%26atitle%3DEnantioselective%252Bamination%252Bof%252B4-alkylisoquinoline-1%25252C3%2525282%252BH%252B%25252C4%252BH%252B%252529-dione%252Bderivatives%26aulast%3DCheng%26aufirst%3DCheng%26date%3D2020%26date%3D2020%26volume%3D10%26issue%3D70%26spage%3D42912%26epage%3D42915" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Masataka  Tsuda</span>, <span class="hlFld-ContribAuthor ">Kaito  Kitamasu</span>, <span class="hlFld-ContribAuthor ">Chiho  Kumagai</span>, <span class="hlFld-ContribAuthor ">Kazuya  Sugiyama</span>, <span class="hlFld-ContribAuthor ">Toshiaki  Nakano</span>, <span class="hlFld-ContribAuthor ">Hiroshi  Ide</span>. </span><span class="cited-content_cbyCitation_article-title">Tyrosyl-DNA phosphodiesterase 2 (TDP2) repairs topoisomerase 1 DNA-protein crosslinks and 3′-blocking lesions in the absence of tyrosyl-DNA phosphodiesterase 1 (TDP1). </span><span class="cited-content_cbyCitation_journal-name">DNA Repair</span><span> <strong>2020,</strong> <em>91-92 </em>, 102849. <a href="https://doi.org/10.1016/j.dnarep.2020.102849" title="DOI URL">https://doi.org/10.1016/j.dnarep.2020.102849</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.dnarep.2020.102849&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.dnarep.2020.102849%26sid%3Dliteratum%253Aachs%26jtitle%3DDNA%2520Repair%26atitle%3DTyrosyl-DNA%252Bphosphodiesterase%252B2%252B%252528TDP2%252529%252Brepairs%252Btopoisomerase%252B1%252BDNA-protein%252Bcrosslinks%252Band%252B3%2525E2%252580%2525B2-blocking%252Blesions%252Bin%252Bthe%252Babsence%252Bof%252Btyrosyl-DNA%252Bphosphodiesterase%252B1%252B%252528TDP1%252529%26aulast%3DTsuda%26aufirst%3DMasataka%26date%3D2020%26volume%3D91-92%26spage%3D102849" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yingpeng  Su</span>, <span class="hlFld-ContribAuthor ">Rong  Zhang</span>, <span class="hlFld-ContribAuthor ">Wenxuan  Xue</span>, <span class="hlFld-ContribAuthor ">Xuan  Liu</span>, <span class="hlFld-ContribAuthor ">Yanan  Zhao</span>, <span class="hlFld-ContribAuthor ">Ke-Hu  Wang</span>, <span class="hlFld-ContribAuthor ">Danfeng  Huang</span>, <span class="hlFld-ContribAuthor ">Congde  Huo</span>, <span class="hlFld-ContribAuthor ">Yulai  Hu</span>. </span><span class="cited-content_cbyCitation_article-title">Visible-light-promoted acyl radical cascade reaction for accessing acylated isoquinoline-1,3(2
              H
              ,4
              H
              )-dione derivatives. </span><span class="cited-content_cbyCitation_journal-name">Organic & Biomolecular Chemistry</span><span> <strong>2020,</strong> <em>18 </em>
                                    (10)
                                     , 1940-1948. <a href="https://doi.org/10.1039/D0OB00086H" title="DOI URL">https://doi.org/10.1039/D0OB00086H</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0OB00086H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0OB00086H%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520%2526%2520Biomolecular%2520Chemistry%26atitle%3DVisible-light-promoted%252Bacyl%252Bradical%252Bcascade%252Breaction%252Bfor%252Baccessing%252Bacylated%252Bisoquinoline-1%25252C3%2525282%252BH%252B%25252C4%252BH%252B%252529-dione%252Bderivatives%26aulast%3DSu%26aufirst%3DYingpeng%26date%3D2020%26date%3D2020%26volume%3D18%26issue%3D10%26spage%3D1940%26epage%3D1948" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Qianqian  Zhen</span>, <span class="hlFld-ContribAuthor ">Lepeng  Chen</span>, <span class="hlFld-ContribAuthor ">Linjun  Qi</span>, <span class="hlFld-ContribAuthor ">Kun  Hu</span>, <span class="hlFld-ContribAuthor ">Yinlin  Shao</span>, <span class="hlFld-ContribAuthor ">Renhao  Li</span>, <span class="hlFld-ContribAuthor ">Jiuxi  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Nickel‐Catalyzed Tandem Reaction of Functionalized Arylacetonitriles with Arylboronic Acids in 2‐MeTHF: Eco‐Friendly Synthesis of Aminoisoquinolines and Isoquinolones. </span><span class="cited-content_cbyCitation_journal-name">Chemistry – An Asian Journal</span><span> <strong>2020,</strong> <em>15 </em>
                                    (1)
                                     , 106-111. <a href="https://doi.org/10.1002/asia.201901442" title="DOI URL">https://doi.org/10.1002/asia.201901442</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/asia.201901442&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fasia.201901442%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistry%2520%25E2%2580%2593%2520An%2520Asian%2520Journal%26atitle%3DNickel%2525E2%252580%252590Catalyzed%252BTandem%252BReaction%252Bof%252BFunctionalized%252BArylacetonitriles%252Bwith%252BArylboronic%252BAcids%252Bin%252B2%2525E2%252580%252590MeTHF%25253A%252BEco%2525E2%252580%252590Friendly%252BSynthesis%252Bof%252BAminoisoquinolines%252Band%252BIsoquinolones%26aulast%3DZhen%26aufirst%3DQianqian%26date%3D2020%26date%3D2019%26volume%3D15%26issue%3D1%26spage%3D106%26epage%3D111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Evgeny  Kiselev</span>, <span class="hlFld-ContribAuthor ">Azhar  Ravji</span>, <span class="hlFld-ContribAuthor ">Jayakanth  Kankanala</span>, <span class="hlFld-ContribAuthor ">Jiashu  Xie</span>, <span class="hlFld-ContribAuthor ">Zhengqiang  Wang</span>, <span class="hlFld-ContribAuthor ">Yves  Pommier</span>. </span><span class="cited-content_cbyCitation_article-title">Novel deazaflavin tyrosyl-DNA phosphodiesterase 2 (TDP2) inhibitors. </span><span class="cited-content_cbyCitation_journal-name">DNA Repair</span><span> <strong>2020,</strong> <em>85 </em>, 102747. <a href="https://doi.org/10.1016/j.dnarep.2019.102747" title="DOI URL">https://doi.org/10.1016/j.dnarep.2019.102747</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.dnarep.2019.102747&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.dnarep.2019.102747%26sid%3Dliteratum%253Aachs%26jtitle%3DDNA%2520Repair%26atitle%3DNovel%252Bdeazaflavin%252Btyrosyl-DNA%252Bphosphodiesterase%252B2%252B%252528TDP2%252529%252Binhibitors%26aulast%3DKiselev%26aufirst%3DEvgeny%26date%3D2020%26volume%3D85%26spage%3D102747" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yanfei  Liu</span>, <span class="hlFld-ContribAuthor ">Jiaping  Wu</span>, <span class="hlFld-ContribAuthor ">Baiyang  Qian</span>, <span class="hlFld-ContribAuthor ">Yongjia  Shang</span>. </span><span class="cited-content_cbyCitation_article-title">Controllable construction of isoquinolinedione and isocoumarin scaffolds
              via
              Rh
              III
              -catalyzed C–H annulation of
              N
              -tosylbenzamides with diazo compounds. </span><span class="cited-content_cbyCitation_journal-name">Organic & Biomolecular Chemistry</span><span> <strong>2019,</strong> <em>17 </em>
                                    (38)
                                     , 8768-8777. <a href="https://doi.org/10.1039/C9OB01789E" title="DOI URL">https://doi.org/10.1039/C9OB01789E</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C9OB01789E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC9OB01789E%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520%2526%2520Biomolecular%2520Chemistry%26atitle%3DControllable%252Bconstruction%252Bof%252Bisoquinolinedione%252Band%252Bisocoumarin%252Bscaffolds%252Bvia%252BRh%252BIII%252B-catalyzed%252BC%2525E2%252580%252593H%252Bannulation%252Bof%252BN%252B-tosylbenzamides%252Bwith%252Bdiazo%252Bcompounds%26aulast%3DLiu%26aufirst%3DYanfei%26date%3D2019%26date%3D2019%26volume%3D17%26issue%3D38%26spage%3D8768%26epage%3D8777" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zh. S.  Arustamyan</span>, <span class="hlFld-ContribAuthor ">R. E.  Markaryan</span>, <span class="hlFld-ContribAuthor ">A. A.  Aghekyan</span>, <span class="hlFld-ContribAuthor ">I. M.  Nazaryan</span>, <span class="hlFld-ContribAuthor ">A. G.  Hakobyan</span>, <span class="hlFld-ContribAuthor ">R. G.  Paronikyan</span>, <span class="hlFld-ContribAuthor ">N. S.  Minasyan</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis and Anticonvulsant Activity of Amino Amides and Amino Esters Based on 1-(2,3-Dihydro-1,4-benzodioxin-6-yl)-cyclopentane-1-carboxylic Acid. </span><span class="cited-content_cbyCitation_journal-name">Russian Journal of Organic Chemistry</span><span> <strong>2019,</strong> <em>55 </em>
                                    (6)
                                     , 796-799. <a href="https://doi.org/10.1134/S1070428019060095" title="DOI URL">https://doi.org/10.1134/S1070428019060095</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1134/S1070428019060095&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1134%2FS1070428019060095%26sid%3Dliteratum%253Aachs%26jtitle%3DRussian%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DSynthesis%252Band%252BAnticonvulsant%252BActivity%252Bof%252BAmino%252BAmides%252Band%252BAmino%252BEsters%252BBased%252Bon%252B1-%2525282%25252C3-Dihydro-1%25252C4-benzodioxin-6-yl%252529-cyclopentane-1-carboxylic%252BAcid%26aulast%3DArustamyan%26aufirst%3DZh.%2BS.%26date%3D2019%26date%3D2019%26volume%3D55%26issue%3D6%26spage%3D796%26epage%3D799" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">A. A.  Aghekyan</span>, <span class="hlFld-ContribAuthor ">G. G.  Mkryan</span>, <span class="hlFld-ContribAuthor ">A. S.  Tsatinyan</span>, <span class="hlFld-ContribAuthor ">G. V.  Gasparyan</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis and Biological Activity of Arylcyclopentane-1-carboxylic Acids Aminoesters. </span><span class="cited-content_cbyCitation_journal-name">Russian Journal of General Chemistry</span><span> <strong>2019,</strong> <em>89 </em>
                                    (5)
                                     , 1051-1054. <a href="https://doi.org/10.1134/S107036321905027X" title="DOI URL">https://doi.org/10.1134/S107036321905027X</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1134/S107036321905027X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1134%2FS107036321905027X%26sid%3Dliteratum%253Aachs%26jtitle%3DRussian%2520Journal%2520of%2520General%2520Chemistry%26atitle%3DSynthesis%252Band%252BBiological%252BActivity%252Bof%252BArylcyclopentane-1-carboxylic%252BAcids%252BAminoesters%26aulast%3DAghekyan%26aufirst%3DA.%2BA.%26date%3D2019%26date%3D2019%26volume%3D89%26issue%3D5%26spage%3D1051%26epage%3D1054" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Carlos J.A.  Ribeiro</span>, <span class="hlFld-ContribAuthor ">Jayakanth  Kankanala</span>, <span class="hlFld-ContribAuthor ">Jiashu  Xie</span>, <span class="hlFld-ContribAuthor ">Jessica  Williams</span>, <span class="hlFld-ContribAuthor ">Hideki  Aihara</span>, <span class="hlFld-ContribAuthor ">Zhengqiang  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Triazolopyrimidine and triazolopyridine scaffolds as TDP2 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2019,</strong> <em>29 </em>
                                    (2)
                                     , 257-261. <a href="https://doi.org/10.1016/j.bmcl.2018.11.044" title="DOI URL">https://doi.org/10.1016/j.bmcl.2018.11.044</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2018.11.044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2018.11.044%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DTriazolopyrimidine%252Band%252Btriazolopyridine%252Bscaffolds%252Bas%252BTDP2%252Binhibitors%26aulast%3DRibeiro%26aufirst%3DCarlos%2BJ.A.%26date%3D2019%26volume%3D29%26issue%3D2%26spage%3D257%26epage%3D261" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Victor V.  Dotsenko</span>, <span class="hlFld-ContribAuthor ">Sergey G.  Krivokolysko</span>, <span class="hlFld-ContribAuthor ">Aminat M.  Semenova</span>. </span><span class="cited-content_cbyCitation_article-title">Heterocyclization reactions using malononitrile dimer (2-aminopropene-1,1,3-tricarbonitrile). </span><span class="cited-content_cbyCitation_journal-name">Chemistry of Heterocyclic Compounds</span><span> <strong>2018,</strong> <em>54 </em>
                                    (11)
                                     , 989-1019. <a href="https://doi.org/10.1007/s10593-018-2383-y" title="DOI URL">https://doi.org/10.1007/s10593-018-2383-y</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10593-018-2383-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10593-018-2383-y%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistry%2520of%2520Heterocyclic%2520Compounds%26atitle%3DHeterocyclization%252Breactions%252Busing%252Bmalononitrile%252Bdimer%252B%2525282-aminopropene-1%25252C1%25252C3-tricarbonitrile%252529%26aulast%3DDotsenko%26aufirst%3DVictor%2BV.%26date%3D2018%26date%3D2018%26volume%3D54%26issue%3D11%26spage%3D989%26epage%3D1019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Carlos J.A.  Ribeiro</span>, <span class="hlFld-ContribAuthor ">Jayakanth  Kankanala</span>, <span class="hlFld-ContribAuthor ">Ke  Shi</span>, <span class="hlFld-ContribAuthor ">Kayo  Kurahashi</span>, <span class="hlFld-ContribAuthor ">Evgeny  Kiselev</span>, <span class="hlFld-ContribAuthor ">Azhar  Ravji</span>, <span class="hlFld-ContribAuthor ">Yves  Pommier</span>, <span class="hlFld-ContribAuthor ">Hideki  Aihara</span>, <span class="hlFld-ContribAuthor ">Zhengqiang  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">New fluorescence-based high-throughput screening assay for small molecule inhibitors of tyrosyl-DNA phosphodiesterase 2 (TDP2). </span><span class="cited-content_cbyCitation_journal-name">European Journal of Pharmaceutical Sciences</span><span> <strong>2018,</strong> <em>118 </em>, 67-79. <a href="https://doi.org/10.1016/j.ejps.2018.03.021" title="DOI URL">https://doi.org/10.1016/j.ejps.2018.03.021</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejps.2018.03.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejps.2018.03.021%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Pharmaceutical%2520Sciences%26atitle%3DNew%252Bfluorescence-based%252Bhigh-throughput%252Bscreening%252Bassay%252Bfor%252Bsmall%252Bmolecule%252Binhibitors%252Bof%252Btyrosyl-DNA%252Bphosphodiesterase%252B2%252B%252528TDP2%252529%26aulast%3DRibeiro%26aufirst%3DCarlos%2BJ.A.%26date%3D2018%26volume%3D118%26spage%3D67%26epage%3D79" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Le-Mao  Yu</span>, <span class="hlFld-ContribAuthor ">Zhu  Hu</span>, <span class="hlFld-ContribAuthor ">Yu  Chen</span>, <span class="hlFld-ContribAuthor ">Azhar  Ravji</span>, <span class="hlFld-ContribAuthor ">Sophia  Lopez</span>, <span class="hlFld-ContribAuthor ">Caroline B.  Plescia</span>, <span class="hlFld-ContribAuthor ">Qian  Yu</span>, <span class="hlFld-ContribAuthor ">Hui  Yang</span>, <span class="hlFld-ContribAuthor ">Monica  Abdelmalak</span>, <span class="hlFld-ContribAuthor ">Sourav  Saha</span>, <span class="hlFld-ContribAuthor ">Keli  Agama</span>, <span class="hlFld-ContribAuthor ">Evgeny  Kiselev</span>, <span class="hlFld-ContribAuthor ">Christophe  Marchand</span>, <span class="hlFld-ContribAuthor ">Yves  Pommier</span>, <span class="hlFld-ContribAuthor ">Lin-Kun  An</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis and structure-activity relationship of furoquinolinediones as inhibitors of Tyrosyl-DNA phosphodiesterase 2 (TDP2). </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>151 </em>, 777-796. <a href="https://doi.org/10.1016/j.ejmech.2018.04.024" title="DOI URL">https://doi.org/10.1016/j.ejmech.2018.04.024</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2018.04.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2018.04.024%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSynthesis%252Band%252Bstructure-activity%252Brelationship%252Bof%252Bfuroquinolinediones%252Bas%252Binhibitors%252Bof%252BTyrosyl-DNA%252Bphosphodiesterase%252B2%252B%252528TDP2%252529%26aulast%3DYu%26aufirst%3DLe-Mao%26date%3D2018%26volume%3D151%26spage%3D777%26epage%3D796" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anne-Katrin  Bachon</span>, <span class="hlFld-ContribAuthor ">Anne-Dorothee  Steinkamp</span>, <span class="hlFld-ContribAuthor ">Carsten  Bolm</span>. </span><span class="cited-content_cbyCitation_article-title">N-Arylated Sulfoximines as Cross-Coupling Building Blocks. </span><span class="cited-content_cbyCitation_journal-name">Advanced Synthesis & Catalysis</span><span> <strong>2018,</strong> <em>360 </em>
                                    (6)
                                     , 1088-1093. <a href="https://doi.org/10.1002/adsc.201701394" title="DOI URL">https://doi.org/10.1002/adsc.201701394</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/adsc.201701394&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fadsc.201701394%26sid%3Dliteratum%253Aachs%26jtitle%3DAdvanced%2520Synthesis%2520%2526%2520Catalysis%26atitle%3DN-Arylated%252BSulfoximines%252Bas%252BCross-Coupling%252BBuilding%252BBlocks%26aulast%3DBachon%26aufirst%3DAnne-Katrin%26date%3D2018%26date%3D2018%26volume%3D360%26issue%3D6%26spage%3D1088%26epage%3D1093" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ajinkya S  Kawale</span>, <span class="hlFld-ContribAuthor ">Lawrence F  Povirk</span>. </span><span class="cited-content_cbyCitation_article-title">Tyrosyl–DNA phosphodiesterases: rescuing the genome from the risks of relaxation. </span><span class="cited-content_cbyCitation_journal-name">Nucleic Acids Research</span><span> <strong>2018,</strong> <em>46 </em>
                                    (2)
                                     , 520-537. <a href="https://doi.org/10.1093/nar/gkx1219" title="DOI URL">https://doi.org/10.1093/nar/gkx1219</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/nar/gkx1219&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fnar%2Fgkx1219%26sid%3Dliteratum%253Aachs%26jtitle%3DNucleic%2520Acids%2520Research%26atitle%3DTyrosyl%2525E2%252580%252593DNA%252Bphosphodiesterases%25253A%252Brescuing%252Bthe%252Bgenome%252Bfrom%252Bthe%252Brisks%252Bof%252Brelaxation%26aulast%3DKawale%26aufirst%3DAjinkya%2BS%26date%3D2018%26date%3D2017%26volume%3D46%26issue%3D2%26spage%3D520%26epage%3D537" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jayakanth  Kankanala</span>, <span class="hlFld-ContribAuthor ">Karen A.  Kirby</span>, <span class="hlFld-ContribAuthor ">Andrew D.  Huber</span>, <span class="hlFld-ContribAuthor ">Mary C.  Casey</span>, <span class="hlFld-ContribAuthor ">Daniel J.  Wilson</span>, <span class="hlFld-ContribAuthor ">Stefan G.  Sarafianos</span>, <span class="hlFld-ContribAuthor ">Zhengqiang  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis and biological evaluations of N-Hydroxy thienopyrimidine-2,4-diones as inhibitors of HIV reverse transcriptase-associated RNase H. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>141 </em>, 149-161. <a href="https://doi.org/10.1016/j.ejmech.2017.09.054" title="DOI URL">https://doi.org/10.1016/j.ejmech.2017.09.054</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2017.09.054&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2017.09.054%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%252Band%252Bbiological%252Bevaluations%252Bof%252BN-Hydroxy%252Bthienopyrimidine-2%25252C4-diones%252Bas%252Binhibitors%252Bof%252BHIV%252Breverse%252Btranscriptase-associated%252BRNase%252BH%26aulast%3DKankanala%26aufirst%3DJayakanth%26date%3D2017%26volume%3D141%26spage%3D149%26epage%3D161" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Qiujie  Tang</span>, <span class="hlFld-ContribAuthor ">Ping  Xie</span>, <span class="hlFld-ContribAuthor ">Jin  Wang</span>, <span class="hlFld-ContribAuthor ">Jiajun  Lin</span>, <span class="hlFld-ContribAuthor ">Chunlai  Feng</span>, <span class="hlFld-ContribAuthor ">Charles U.  Pittman</span>, <span class="hlFld-ContribAuthor ">Aihua  Zhou</span>. </span><span class="cited-content_cbyCitation_article-title">“One-pot” sequential preparation of isoquinoline-1,3(2H,4H)-dione derivatives by reacting N-alkyl(aryl)-N-methacryloyl benzamides with benzyl alcohols and sodium benzenesulfinates. </span><span class="cited-content_cbyCitation_journal-name">Tetrahedron</span><span> <strong>2017,</strong> <em>73 </em>
                                    (36)
                                     , 5436-5443. <a href="https://doi.org/10.1016/j.tet.2017.07.055" title="DOI URL">https://doi.org/10.1016/j.tet.2017.07.055</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tet.2017.07.055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tet.2017.07.055%26sid%3Dliteratum%253Aachs%26jtitle%3DTetrahedron%26atitle%3D%2525E2%252580%25259COne-pot%2525E2%252580%25259D%252Bsequential%252Bpreparation%252Bof%252Bisoquinoline-1%25252C3%2525282H%25252C4H%252529-dione%252Bderivatives%252Bby%252Breacting%252BN-alkyl%252528aryl%252529-N-methacryloyl%252Bbenzamides%252Bwith%252Bbenzyl%252Balcohols%252Band%252Bsodium%252Bbenzenesulfinates%26aulast%3DTang%26aufirst%3DQiujie%26date%3D2017%26volume%3D73%26issue%3D36%26spage%3D5436%26epage%3D5443" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Peter J.  Lindsay-Scott</span>, <span class="hlFld-ContribAuthor ">Peter T.  Gallagher</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis of heterocycles from arylacetonitriles: Powerful tools for medicinal chemists. </span><span class="cited-content_cbyCitation_journal-name">Tetrahedron Letters</span><span> <strong>2017,</strong> <em>58 </em>
                                    (27)
                                     , 2629-2635. <a href="https://doi.org/10.1016/j.tetlet.2017.05.089" title="DOI URL">https://doi.org/10.1016/j.tetlet.2017.05.089</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tetlet.2017.05.089&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tetlet.2017.05.089%26sid%3Dliteratum%253Aachs%26jtitle%3DTetrahedron%2520Letters%26atitle%3DSynthesis%252Bof%252Bheterocycles%252Bfrom%252Barylacetonitriles%25253A%252BPowerful%252Btools%252Bfor%252Bmedicinal%252Bchemists%26aulast%3DLindsay-Scott%26aufirst%3DPeter%2BJ.%26date%3D2017%26volume%3D58%26issue%3D27%26spage%3D2629%26epage%3D2635" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">A. A.  Aghekyan</span>, <span class="hlFld-ContribAuthor ">G. G.  Mkryan</span>, <span class="hlFld-ContribAuthor ">A. S.  Tsatinyan</span>, <span class="hlFld-ContribAuthor ">O. S.  Noravyan</span>, <span class="hlFld-ContribAuthor ">G. V.  Gasparyan</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis and biological activity of 1-(aryloxy)-3-(6-chloro-4,4-dimethyl-1,2,3,4-tetrahydroisoquinolin-2-yl)propan-2-ols. </span><span class="cited-content_cbyCitation_journal-name">Russian Journal of Organic Chemistry</span><span> <strong>2017,</strong> <em>53 </em>
                                    (3)
                                     , 362-365. <a href="https://doi.org/10.1134/S1070428017030083" title="DOI URL">https://doi.org/10.1134/S1070428017030083</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1134/S1070428017030083&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1134%2FS1070428017030083%26sid%3Dliteratum%253Aachs%26jtitle%3DRussian%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DSynthesis%252Band%252Bbiological%252Bactivity%252Bof%252B1-%252528aryloxy%252529-3-%2525286-chloro-4%25252C4-dimethyl-1%25252C2%25252C3%25252C4-tetrahydroisoquinolin-2-yl%252529propan-2-ols%26aulast%3DAghekyan%26aufirst%3DA.%2BA.%26date%3D2017%26date%3D2017%26volume%3D53%26issue%3D3%26spage%3D362%26epage%3D365" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Arwa A.  Abugable</span>, <span class="hlFld-ContribAuthor ">Dahlia A.  Awwad</span>, <span class="hlFld-ContribAuthor ">Dalia  Fleifel</span>, <span class="hlFld-ContribAuthor ">Mohamed M.  Ali</span>, <span class="hlFld-ContribAuthor ">Sherif  El-Khamisy</span>, <span class="hlFld-ContribAuthor ">Menattallah  Elserafy</span>. </span><span class="cited-content_cbyCitation_article-title">Personalised Medicine: Genome Maintenance Lessons Learned from Studies in Yeast as a Model Organism. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2017,</strong>,, 157-178. <a href="https://doi.org/10.1007/978-3-319-60733-7_9" title="DOI URL">https://doi.org/10.1007/978-3-319-60733-7_9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-3-319-60733-7_9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-3-319-60733-7_9%26sid%3Dliteratum%253Aachs%26atitle%3DPersonalised%252BMedicine%25253A%252BGenome%252BMaintenance%252BLessons%252BLearned%252Bfrom%252BStudies%252Bin%252BYeast%252Bas%252Ba%252BModel%252BOrganism%26aulast%3DAbugable%26aufirst%3DArwa%2BA.%26date%3D2017%26date%3D2017%26spage%3D157%26epage%3D178%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DPersonalised%252BMedicine%26aulast%3DEl-Khamisy%26aufirst%3DSherif%26date%3D2017%26volume%3D1007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sergey S.  Laev</span>, <span class="hlFld-ContribAuthor ">Nariman F.  Salakhutdinov</span>, <span class="hlFld-ContribAuthor ">Olga I.  Lavrik</span>. </span><span class="cited-content_cbyCitation_article-title">Tyrosyl-DNA phosphodiesterase inhibitors: Progress and potential. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2016,</strong> <em>24 </em>
                                    (21)
                                     , 5017-5027. <a href="https://doi.org/10.1016/j.bmc.2016.09.045" title="DOI URL">https://doi.org/10.1016/j.bmc.2016.09.045</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2016.09.045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2016.09.045%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DTyrosyl-DNA%252Bphosphodiesterase%252Binhibitors%25253A%252BProgress%252Band%252Bpotential%26aulast%3DLaev%26aufirst%3DSergey%2BS.%26date%3D2016%26volume%3D24%26issue%3D21%26spage%3D5017%26epage%3D5027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Waheba  Elsayed</span>, <span class="hlFld-ContribAuthor ">Lamia  El-Shafie</span>, <span class="hlFld-ContribAuthor ">Mohamed K.  Hassan</span>, <span class="hlFld-ContribAuthor ">Mohamed A.  Farag</span>, <span class="hlFld-ContribAuthor ">Sherif F.  El-Khamisy</span>. </span><span class="cited-content_cbyCitation_article-title">Isoeugenol is a selective potentiator of camptothecin cytotoxicity in vertebrate cells lacking TDP1. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2016,</strong> <em>6 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/srep26626" title="DOI URL">https://doi.org/10.1038/srep26626</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/srep26626&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fsrep26626%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DIsoeugenol%252Bis%252Ba%252Bselective%252Bpotentiator%252Bof%252Bcamptothecin%252Bcytotoxicity%252Bin%252Bvertebrate%252Bcells%252Blacking%252BTDP1%26aulast%3DElsayed%26aufirst%3DWaheba%26date%3D2016%26date%3D2016%26volume%3D6%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hong  Yan</span>, <span class="hlFld-ContribAuthor ">Margaret  Tammaro</span>, <span class="hlFld-ContribAuthor ">Shuren  Liao</span>. </span><span class="cited-content_cbyCitation_article-title">Collision of Trapped Topoisomerase 2 with Transcription and Replication: Generation and Repair of DNA Double-Strand Breaks with 5′ Adducts. </span><span class="cited-content_cbyCitation_journal-name">Genes</span><span> <strong>2016,</strong> <em>7 </em>
                                    (7)
                                     , 32. <a href="https://doi.org/10.3390/genes7070032" title="DOI URL">https://doi.org/10.3390/genes7070032</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/genes7070032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fgenes7070032%26sid%3Dliteratum%253Aachs%26jtitle%3DGenes%26atitle%3DCollision%252Bof%252BTrapped%252BTopoisomerase%252B2%252Bwith%252BTranscription%252Band%252BReplication%25253A%252BGeneration%252Band%252BRepair%252Bof%252BDNA%252BDouble-Strand%252BBreaks%252Bwith%252B5%2525E2%252580%2525B2%252BAdducts%26aulast%3DYan%26aufirst%3DHong%26date%3D2016%26date%3D2016%26volume%3D7%26issue%3D7%26spage%3D32" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01973/20160318/images/medium/jm-2015-01973q_0005.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01973/20160318/images/large/jm-2015-01973q_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01973&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01973/20160318/images/medium/jm-2015-01973q_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01973/20160318/images/large/jm-2015-01973q_0007.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. TDP2 repairs abortive Top2–DNA cleavage complex: <b>2</b>, DNA substrate; <b>4</b>, transient Top2–DNA complex; <b>5</b>, Top2–DNA complex stabilized by a Top2 poison; cyan, Top2; green, TDP2; yellow, Top2 poison.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01973/20160318/images/large/jm-2015-01973q_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01973&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01973/20160318/images/medium/jm-2015-01973q_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01973/20160318/images/large/jm-2015-01973q_0008.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Chemotypes of interest. (a) Reported inhibitor types of TDP2; (b) in-house synthetic chemotypes screened against TDP2.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01973/20160318/images/large/jm-2015-01973q_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01973&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01973/20160318/images/medium/jm-2015-01973q_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01973/20160318/images/large/jm-2015-01973q_0009.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. SAR and selective TDP2 inhibition of the isoquinolinedione chemotype. (a) Structure of isoquinoline-1,3-dione <b>43</b> and HID <b>46</b>. (b) Inhibition profile: <b>43</b> selectively inhibits TDP2 in assays using either recombinant (Rec) or endogenous TDP2 from whole cell extracts (WCE), while <b>46</b> was inactive in either condition. (c) Gel images of dose response testing of <b>43</b> in Rec TDP2 biochemical assays. Conc (μM): 1.4, 4.1, 12.3, 37, 111. (d) Dose–response curve for <b>43</b> in Rec TDP2 assay expressed as mean ± SD from at least three independent experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01973/20160318/images/large/jm-2015-01973q_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01973&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01973/20160318/images/medium/jm-2015-01973q_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01973/20160318/images/large/jm-2015-01973q_0011.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 1. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01973/20160318/images/large/jm-2015-01973q_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01973&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) arylboronic acid, K<sub>2</sub>CO<sub>3</sub>, Pd(PPh<sub>3</sub>)<sub>4</sub>, EtOH/H<sub>2</sub>O (1:1), 150 °C, 30 min, MW, 60–85%; (b) urea, 1,2-dichlorobenzene, 170 °C, 45 min, MW, 55–80%.</p></p></figure><figure data-id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01973/20160318/images/medium/jm-2015-01973q_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01973/20160318/images/large/jm-2015-01973q_0001.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 2. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01973/20160318/images/large/jm-2015-01973q_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01973&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) methyl cyanoacetate, NaH, DMSO, 90 °C; (b) DMSO/H<sub>2</sub>O (9:1), 120 °C, 16 h, 70–85%; (c) arylboronic acid, K<sub>2</sub>CO<sub>3</sub>, Pd(PPh<sub>3</sub>)<sub>4</sub>, DME/H<sub>2</sub>O (4:1), 110 °C, 40 min, MW, 55–87%; (d) Pd(OAc)<sub>2</sub>, Mo(CO)<sub>6</sub>, K<sub>2</sub>CO<sub>3</sub>, anisole (0.2 M), 140 °C, 30 min, MW, 50–62%; (e) conc HCl, 70 °C, 4 h, 60–90%; (f) sulfonyl or acryloyl chloride, pyridine, dioxane, rt; 65–79%; (g) 2-aminothiophenol, T3P (50% in EtOAc), DIPEA, 100 °C, 30 min, MW, 69%.</p></p></figure><figure data-id="sch3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01973/20160318/images/medium/jm-2015-01973q_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01973/20160318/images/large/jm-2015-01973q_0002.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 3. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01973/20160318/images/large/jm-2015-01973q_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01973&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) phenylboronic acid, K<sub>2</sub>CO<sub>3</sub>, Pd(PPh<sub>3</sub>)<sub>4</sub>, DME/H<sub>2</sub>O (4:1), 110 °C, 45 min, MW, 50%; (b) (i) KOCN, toluene, rt, (ii) NaOH, EtOH, reflux, 68%; (c) NH<sub>4</sub>OH, 40 °C, 4 h, 70–90%; (d) 2-(tributylstannyl)furan, Pd(PPh<sub>3</sub>)<sub>4</sub>, toluene, 150 °C, 30 min, 72%; (e) R-NHNH<sub>2</sub>, EtOH, reflux, 70–95%; (f) conc HCl, 70 °C, 4 h, 65–85%; (g) acetyl chloride, AcOH, 70 °C, 62%.</p></p></figure><figure data-id="fig4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01973/20160318/images/medium/jm-2015-01973q_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01973/20160318/images/large/jm-2015-01973q_0010.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Predicted binding mode of (a) <b>56</b> and (b) <b>40</b> within active site of hTDP2 (numbering was based on hTDP2 sequence). The active site residues are shown as green sticks and metal ions as magenta sphere. The H-bond interactions are depicted as black dotted lines, and the distance between bonded atoms are represented in red. Homology model (hTDP2) was built using the crystal structure of mTDP2 (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GZ1">4GZ1</a><a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01973/20160318/images/large/jm-2015-01973q_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01973&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i96">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_77728" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_77728" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 33 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Helleday, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petermann, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lundin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hodgson, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharma, R. A.</span><span> </span><span class="NLM_article-title">DNA repair pathways as targets for cancer therapy</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">193</span><span class="NLM_x">–</span> <span class="NLM_lpage">204</span><span class="refDoi"> DOI: 10.1038/nrc2342</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=10.1038%2Fnrc2342" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=18256616" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=1%3ACAS%3A528%3ADC%252BD1cXitlOisr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2008&pages=193-204&author=T.+Helledayauthor=E.+Petermannauthor=C.+Lundinauthor=B.+Hodgsonauthor=R.+A.+Sharma&title=DNA+repair+pathways+as+targets+for+cancer+therapy&doi=10.1038%2Fnrc2342"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">DNA repair pathways as targets for cancer therapy</span></div><div class="casAuthors">Helleday, Thomas; Petermann, Eva; Lundin, Cecilia; Hodgson, Ben; Sharma, Ricky A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">193-204</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  DNA repair pathways can enable tumor cells to survive DNA damage that is induced by chemotherapeutic treatments; therefore, inhibitors of specific DNA repair pathways might prove efficacious when used in combination with DNA-damaging chemotherapeutic drugs.  In addn., alterations in DNA repair pathways that arise during tumor development can make some cancer cells reliant on a reduced set of DNA repair pathways for survival.  There is evidence that drugs that inhibit one of these pathways in such tumors could prove useful as single-agent therapies, with the potential advantage that this approach could be selective for tumor cells and have fewer side effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3W53LJmdCPrVg90H21EOLACvtfcHk0ljAvEmcgqvrZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXitlOisr8%253D&md5=8592af6d815e5ca9c80ee256791bafb7</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrc2342&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2342%26sid%3Dliteratum%253Aachs%26aulast%3DHelleday%26aufirst%3DT.%26aulast%3DPetermann%26aufirst%3DE.%26aulast%3DLundin%26aufirst%3DC.%26aulast%3DHodgson%26aufirst%3DB.%26aulast%3DSharma%26aufirst%3DR.%2BA.%26atitle%3DDNA%2520repair%2520pathways%2520as%2520targets%2520for%2520cancer%2520therapy%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2008%26volume%3D8%26spage%3D193%26epage%3D204%26doi%3D10.1038%2Fnrc2342" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Nitiss, J. L.</span><span> </span><span class="NLM_article-title">Targeting DNA topoisomerase II in cancer chemotherapy</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">338</span><span class="NLM_x">–</span> <span class="NLM_lpage">350</span><span class="refDoi"> DOI: 10.1038/nrc2607</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=10.1038%2Fnrc2607" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=19377506" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=1%3ACAS%3A528%3ADC%252BD1MXks12kur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2009&pages=338-350&author=J.+L.+Nitiss&title=Targeting+DNA+topoisomerase+II+in+cancer+chemotherapy&doi=10.1038%2Fnrc2607"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting DNA topoisomerase II in cancer chemotherapy</span></div><div class="casAuthors">Nitiss, John L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">338-350</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Recent mol. studies have expanded the biol. contexts in which topoisomerase II (TOP2) has crucial functions, including DNA replication, transcription and chromosome segregation.  Although the biol. functions of TOP2 are important for ensuring genomic integrity, the ability to interfere with TOP2 and generate enzyme-mediated DNA damage is an effective strategy for cancer chemotherapy.  The mol. tools that have allowed an understanding of the biol. functions of TOP2 are also being applied to understanding the details of drug action.  These studies promise refined targeting of TOP2 as an effective anticancer strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_7Nq6nIe1RbVg90H21EOLACvtfcHk0ljAvEmcgqvrZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXks12kur0%253D&md5=a787de602687580188cda6a2af7b6a07</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrc2607&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2607%26sid%3Dliteratum%253Aachs%26aulast%3DNitiss%26aufirst%3DJ.%2BL.%26atitle%3DTargeting%2520DNA%2520topoisomerase%2520II%2520in%2520cancer%2520chemotherapy%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2009%26volume%3D9%26spage%3D338%26epage%3D350%26doi%3D10.1038%2Fnrc2607" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Ledesma, F. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">El Khamisy, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zuma, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Osborn, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caldecott, K. W.</span><span> </span><span class="NLM_article-title">A human 5′-tyrosyl DNA phosphodiesterase that repairs topoisomerase-mediated DNA damage</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">461</span><span class="NLM_x">, </span> <span class="NLM_fpage">674</span><span class="NLM_x">–</span> <span class="NLM_lpage">U125</span><span class="refDoi"> DOI: 10.1038/nature08444</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=10.1038%2Fnature08444" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=19794497" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1SqtrzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=461&publication_year=2009&pages=674-U125&author=F.+C.+Ledesmaauthor=S.+F.+El+Khamisyauthor=M.+C.+Zumaauthor=K.+Osbornauthor=K.+W.+Caldecott&title=A+human+5%E2%80%B2-tyrosyl+DNA+phosphodiesterase+that+repairs+topoisomerase-mediated+DNA+damage&doi=10.1038%2Fnature08444"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">A human 5'-tyrosyl DNA phosphodiesterase that repairs topoisomerase-mediated DNA damage</span></div><div class="casAuthors">Ledesma, Felipe Cortes; El Khamisy, Sherif F.; Zuma, Maria C.; Osborn, Kay; Caldecott, Keith W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">461</span>
        (<span class="NLM_cas:issue">7264</span>),
    <span class="NLM_cas:pages">674-678</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Topoisomerases regulate DNA topol. and are fundamental to many aspects of chromosome metab.  Their activity involves the transient cleavage of DNA, which, if it occurs near sites of endogenous DNA damage or in the presence of topoisomerase poisons, can result in abortive topoisomerase-induced DNA strand breaks.  These breaks feature covalent linkage of the enzyme to the DNA termini by a 3'- or 5'-phosphotyrosyl bond and are implicated in hereditary human disease, chromosomal instability, and cancer, and underlie the clin. efficacy of an important class of anti-tumor poisons.  The importance of liberating DNA termini from trapped topoisomerase is illustrated by the progressive neurodegenerative disease obsd. in individuals contg. a mutation in tyrosyl-DNA phosphodiesterase 1 (TDP1), an enzyme that cleaves 3'-phosphotyrosyl bonds.  However, a complementary human enzyme that cleaves 5'-phosphotyrosyl bonds has not been reported, despite the effect of DNA double-strand breaks contg. such termini on chromosome instability and cancer.  Here, the authors identify such an enzyme in human cells and show that this activity efficiently restores 5'-phosphate termini at DNA double-strand breaks in prepn. for DNA ligation.  This enzyme, TTRAP (TRAF and TNF receptor-assocd. protein), is a member of the Mg2+/Mn2+-dependent family of phosphodiesterases.  Cellular depletion of TTRAP results in increased susceptibility and sensitivity to topoisomerase-II-induced DNA double-strand breaks.  Thus, TTRAP is, to the authors' knowledge, the 1st human 5'-tyrosyl DNA phosphodiesterase to be identified, and it is suggested that this enzyme is denoted tyrosyl DNA phosphodiesterase-2 (TDP2).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_IrEo1DjZabVg90H21EOLACvtfcHk0ljAvEmcgqvrZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1SqtrzO&md5=a015dfe2e1b01c78455f67215ddb5b98</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnature08444&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08444%26sid%3Dliteratum%253Aachs%26aulast%3DLedesma%26aufirst%3DF.%2BC.%26aulast%3DEl%2BKhamisy%26aufirst%3DS.%2BF.%26aulast%3DZuma%26aufirst%3DM.%2BC.%26aulast%3DOsborn%26aufirst%3DK.%26aulast%3DCaldecott%26aufirst%3DK.%2BW.%26atitle%3DA%2520human%25205%25E2%2580%25B2-tyrosyl%2520DNA%2520phosphodiesterase%2520that%2520repairs%2520topoisomerase-mediated%2520DNA%2520damage%26jtitle%3DNature%26date%3D2009%26volume%3D461%26spage%3D674%26epage%3DU125%26doi%3D10.1038%2Fnature08444" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Gomez-Herreros, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romero-Granados, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alvarez-Quilon, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quintero, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ju, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Umans, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vermeire, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huylebroeck, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caldecott, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cortes-Ledesma, F.</span><span> </span><span class="NLM_article-title">TDP2-dependent non-homologous end-joining protects against topoisomerase II-induced DNA breaks and genome instability in cells and in vivo</span> <span class="citation_source-journal">PLoS Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1003226</span><span class="refDoi"> DOI: 10.1371/journal.pgen.1003226</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=10.1371%2Fjournal.pgen.1003226" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=e1003226&author=F.+Gomez-Herrerosauthor=R.+Romero-Granadosauthor=Z.+Zengauthor=A.+Alvarez-Quilonauthor=C.+Quinteroauthor=L.+Juauthor=L.+Umansauthor=L.+Vermeireauthor=D.+Huylebroeckauthor=K.+W.+Caldecottauthor=F.+Cortes-Ledesma&title=TDP2-dependent+non-homologous+end-joining+protects+against+topoisomerase+II-induced+DNA+breaks+and+genome+instability+in+cells+and+in+vivo&doi=10.1371%2Fjournal.pgen.1003226"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pgen.1003226&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pgen.1003226%26sid%3Dliteratum%253Aachs%26aulast%3DGomez-Herreros%26aufirst%3DF.%26aulast%3DRomero-Granados%26aufirst%3DR.%26aulast%3DZeng%26aufirst%3DZ.%26aulast%3DAlvarez-Quilon%26aufirst%3DA.%26aulast%3DQuintero%26aufirst%3DC.%26aulast%3DJu%26aufirst%3DL.%26aulast%3DUmans%26aufirst%3DL.%26aulast%3DVermeire%26aufirst%3DL.%26aulast%3DHuylebroeck%26aufirst%3DD.%26aulast%3DCaldecott%26aufirst%3DK.%2BW.%26aulast%3DCortes-Ledesma%26aufirst%3DF.%26atitle%3DTDP2-dependent%2520non-homologous%2520end-joining%2520protects%2520against%2520topoisomerase%2520II-induced%2520DNA%2520breaks%2520and%2520genome%2520instability%2520in%2520cells%2520and%2520in%2520vivo%26jtitle%3DPLoS%2520Genet.%26date%3D2013%26volume%3D9%26spage%3De1003226%26doi%3D10.1371%2Fjournal.pgen.1003226" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Gomez-Herreros, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schuurs-Hoeijmakers, J. H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCormack, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greally, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rulten, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romero-Granados, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Counihan, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chaila, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Conroy, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ennis, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delanty, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cortes-Ledesma, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Brouwer, A. P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavalleri, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">El-Khamisy, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Vries, B. B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caldecott, K. W.</span><span> </span><span class="NLM_article-title">TDP2 protects transcription from abortive topoisomerase activity and is required for normal neural function</span> <span class="citation_source-journal">Nat. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">516</span><span class="NLM_x">–</span> <span class="NLM_lpage">521</span><span class="refDoi"> DOI: 10.1038/ng.2929</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=10.1038%2Fng.2929" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=24658003" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=1%3ACAS%3A528%3ADC%252BC2cXks12ksb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2014&pages=516-521&author=F.+Gomez-Herrerosauthor=J.+H.+M.+Schuurs-Hoeijmakersauthor=M.+McCormackauthor=M.+T.+Greallyauthor=S.+Rultenauthor=R.+Romero-Granadosauthor=T.+J.+Counihanauthor=E.+Chailaauthor=J.+Conroyauthor=S.+Ennisauthor=N.+Delantyauthor=F.+Cortes-Ledesmaauthor=A.+P.+M.+de+Brouwerauthor=G.+L.+Cavalleriauthor=S.+F.+El-Khamisyauthor=B.+B.+A.+de+Vriesauthor=K.+W.+Caldecott&title=TDP2+protects+transcription+from+abortive+topoisomerase+activity+and+is+required+for+normal+neural+function&doi=10.1038%2Fng.2929"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">TDP2 protects transcription from abortive topoisomerase activity and is required for normal neural function</span></div><div class="casAuthors">Gomez-Herreros, Fernando; Schuurs-Hoeijmakers, Janneke H. M.; McCormack, Mark; Greally, Marie T.; Rulten, Stuart; Romero-Granados, Rocio; Counihan, Timothy J.; Chaila, Elijah; Conroy, Judith; Ennis, Sean; Delanty, Norman; Cortes-Ledesma, Felipe; de Brouwer, Arjan P. M.; Cavalleri, Gianpiero L.; El-Khamisy, Sherif F.; de Vries, Bert B. A.; Caldecott, Keith W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">516-521</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Topoisomerase II (TOP2) removes torsional stress from DNA and facilitates gene transcription by introducing transient DNA double-strand breaks (DSBs).  Such DSBs are normally rejoined by TOP2 but on occasion can become abortive and remain unsealed.  Here we identify homozygous mutations in the TDP2 gene encoding tyrosyl DNA phosphodiesterase-2, an enzyme that repairs 'abortive' TOP2-induced DSBs, in individuals with intellectual disability, seizures and ataxia.  We show that cells from affected individuals are hypersensitive to TOP2-induced DSBs and that loss of TDP2 inhibits TOP2-dependent gene transcription in cultured human cells and in mouse post-mitotic neurons following abortive TOP2 activity.  Notably, TDP2 is also required for normal levels of many gene transcripts in developing mouse brain, including numerous gene transcripts assocd. with neurol. function and/or disease, and for normal interneuron d. in mouse cerebellum.  Collectively, these data implicate chromosome breakage by TOP2 as an endogenous threat to gene transcription and to normal neuronal development and maintenance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDwc9VhWlS-rVg90H21EOLACvtfcHk0lick6TAMt-jIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXks12ksb4%253D&md5=30364c8b473081a7486413b1bcd3e680</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fng.2929&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng.2929%26sid%3Dliteratum%253Aachs%26aulast%3DGomez-Herreros%26aufirst%3DF.%26aulast%3DSchuurs-Hoeijmakers%26aufirst%3DJ.%2BH.%2BM.%26aulast%3DMcCormack%26aufirst%3DM.%26aulast%3DGreally%26aufirst%3DM.%2BT.%26aulast%3DRulten%26aufirst%3DS.%26aulast%3DRomero-Granados%26aufirst%3DR.%26aulast%3DCounihan%26aufirst%3DT.%2BJ.%26aulast%3DChaila%26aufirst%3DE.%26aulast%3DConroy%26aufirst%3DJ.%26aulast%3DEnnis%26aufirst%3DS.%26aulast%3DDelanty%26aufirst%3DN.%26aulast%3DCortes-Ledesma%26aufirst%3DF.%26aulast%3Dde%2BBrouwer%26aufirst%3DA.%2BP.%2BM.%26aulast%3DCavalleri%26aufirst%3DG.%2BL.%26aulast%3DEl-Khamisy%26aufirst%3DS.%2BF.%26aulast%3Dde%2BVries%26aufirst%3DB.%2BB.%2BA.%26aulast%3DCaldecott%26aufirst%3DK.%2BW.%26atitle%3DTDP2%2520protects%2520transcription%2520from%2520abortive%2520topoisomerase%2520activity%2520and%2520is%2520required%2520for%2520normal%2520neural%2520function%26jtitle%3DNat.%2520Genet.%26date%3D2014%26volume%3D46%26spage%3D516%26epage%3D521%26doi%3D10.1038%2Fng.2929" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Maede, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shimizu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fukushima, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kogame, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takeda, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pommier, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murai, J.</span><span> </span><span class="NLM_article-title">Differential and common DNA repair pathways for topoisomerase I- and II-targeted drugs in a genetic DT40 repair cell screen panel</span> <span class="citation_source-journal">Mol. Cancer Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">214</span><span class="NLM_x">–</span> <span class="NLM_lpage">220</span><span class="refDoi"> DOI: 10.1158/1535-7163.MCT-13-0551</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=10.1158%2F1535-7163.MCT-13-0551" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=24130054" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=1%3ACAS%3A528%3ADC%252BC2cXnsFSmsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=214-220&author=Y.+Maedeauthor=H.+Shimizuauthor=T.+Fukushimaauthor=T.+Kogameauthor=T.+Nakamuraauthor=T.+Mikiauthor=S.+Takedaauthor=Y.+Pommierauthor=J.+Murai&title=Differential+and+common+DNA+repair+pathways+for+topoisomerase+I-+and+II-targeted+drugs+in+a+genetic+DT40+repair+cell+screen+panel&doi=10.1158%2F1535-7163.MCT-13-0551"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Differential and Common DNA Repair Pathways for Topoisomerase I- and II-Targeted Drugs in a Genetic DT40 Repair Cell Screen Panel</span></div><div class="casAuthors">Maede, Yuko; Shimizu, Hiroyasu; Fukushima, Toru; Kogame, Toshiaki; Nakamura, Terukazu; Miki, Tsuneharu; Takeda, Shunichi; Pommier, Yves; Murai, Junko</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">214-220</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Clin. topoisomerase I (Top1) and II (Top2) inhibitors trap topoisomerases on DNA, thereby inducing protein-linked DNA breaks.  Cancer cells resist the drugs by removing topoisomerase-DNA complexes, and repairing the drug-induced DNA double-strand breaks (DSB) by homologous recombination and nonhomologous end joining (NHEJ).  Because numerous enzymes and cofactors are involved in the removal of the topoisomerase-DNA complexes and DSB repair, it has been challenging to comprehensively analyze the relative contribution of multiple genetic pathways in vertebrate cells.  Comprehending the relative contribution of individual repair factors would give insights into the lesions induced by the inhibitors and genetic determinants of response.  Ultimately, this information would be useful to target specific pathways to augment the therapeutic activity of topoisomerase inhibitors.  To this end, we put together 48 isogenic DT40 mutant cells deficient in DNA repair and generated one cell line deficient in autophagy (ATG5).  Sensitivity profiles were established for three clin. relevant Top1 inhibitors (camptothecin and the indenoisoquinolines LMP400 and LMP776) and three Top2 inhibitors (etoposide, doxorubicin, and ICRF-193).  Highly significant correlations were found among Top1 inhibitors as well as Top2 inhibitors, whereas the profiles of Top1 inhibitors were different from those of Top2 inhibitors.  Most distinct repair pathways between Top1 and Top2 inhibitors include NHEJ, TDP1, TDP2, PARP1, and Fanconi Anemia genes, whereas homologous recombination seems relevant esp. for Top1 and, to a lesser extent, for Top2 inhibitors.  We also found and discuss differential pathways among Top1 inhibitors and Top2 inhibitors.  Mol Cancer Ther; 13(1); 214-20. ©2013 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrLkuEffSz27Vg90H21EOLACvtfcHk0lick6TAMt-jIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXnsFSmsg%253D%253D&md5=346456ce64a6e142fb9df968cdc9f05e</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-13-0551&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-13-0551%26sid%3Dliteratum%253Aachs%26aulast%3DMaede%26aufirst%3DY.%26aulast%3DShimizu%26aufirst%3DH.%26aulast%3DFukushima%26aufirst%3DT.%26aulast%3DKogame%26aufirst%3DT.%26aulast%3DNakamura%26aufirst%3DT.%26aulast%3DMiki%26aufirst%3DT.%26aulast%3DTakeda%26aufirst%3DS.%26aulast%3DPommier%26aufirst%3DY.%26aulast%3DMurai%26aufirst%3DJ.%26atitle%3DDifferential%2520and%2520common%2520DNA%2520repair%2520pathways%2520for%2520topoisomerase%2520I-%2520and%2520II-targeted%2520drugs%2520in%2520a%2520genetic%2520DT40%2520repair%2520cell%2520screen%2520panel%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2014%26volume%3D13%26spage%3D214%26epage%3D220%26doi%3D10.1158%2F1535-7163.MCT-13-0551" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Zeng, Z. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cortes-Ledesma, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">El Khamisy, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caldecott, K. W.</span><span> </span><span class="NLM_article-title">TDP2/TTRAP is the major 5 ′-tyrosyl DNA phosphodiesterase activity in vertebrate cells and is critical for cellular resistance to topoisomerase II-induced DNA damage</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">286</span><span class="NLM_x">, </span> <span class="NLM_fpage">403</span><span class="NLM_x">–</span> <span class="NLM_lpage">409</span><span class="refDoi"> DOI: 10.1074/jbc.M110.181016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=10.1074%2Fjbc.M110.181016" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2011&pages=403-409&author=Z.+H.+Zengauthor=F.+Cortes-Ledesmaauthor=S.+F.+El+Khamisyauthor=K.+W.+Caldecott&title=TDP2%2FTTRAP+is+the+major+5+%E2%80%B2-tyrosyl+DNA+phosphodiesterase+activity+in+vertebrate+cells+and+is+critical+for+cellular+resistance+to+topoisomerase+II-induced+DNA+damage&doi=10.1074%2Fjbc.M110.181016"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M110.181016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M110.181016%26sid%3Dliteratum%253Aachs%26aulast%3DZeng%26aufirst%3DZ.%2BH.%26aulast%3DCortes-Ledesma%26aufirst%3DF.%26aulast%3DEl%2BKhamisy%26aufirst%3DS.%2BF.%26aulast%3DCaldecott%26aufirst%3DK.%2BW.%26atitle%3DTDP2%252FTTRAP%2520is%2520the%2520major%25205%2520%25E2%2580%25B2-tyrosyl%2520DNA%2520phosphodiesterase%2520activity%2520in%2520vertebrate%2520cells%2520and%2520is%2520critical%2520for%2520cellular%2520resistance%2520to%2520topoisomerase%2520II-induced%2520DNA%2520damage%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2011%26volume%3D286%26spage%3D403%26epage%3D409%26doi%3D10.1074%2Fjbc.M110.181016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Do, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varanasi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kubacka, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newman, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chauhan, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daniels, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boccetta, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garrett, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinez, L. A.</span><span> </span><span class="NLM_article-title">Mutant p53 cooperates with ETS2 to promote etoposide resistance</span> <span class="citation_source-journal">Genes Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">830</span><span class="NLM_x">–</span> <span class="NLM_lpage">845</span><span class="refDoi"> DOI: 10.1101/gad.181685.111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=10.1101%2Fgad.181685.111" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2012&pages=830-845&author=P.+M.+Doauthor=L.+Varanasiauthor=S.+Fanauthor=C.+Liauthor=I.+Kubackaauthor=V.+Newmanauthor=K.+Chauhanauthor=S.+R.+Danielsauthor=M.+Boccettaauthor=M.+R.+Garrettauthor=R.+Liauthor=L.+A.+Martinez&title=Mutant+p53+cooperates+with+ETS2+to+promote+etoposide+resistance&doi=10.1101%2Fgad.181685.111"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1101%2Fgad.181685.111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.181685.111%26sid%3Dliteratum%253Aachs%26aulast%3DDo%26aufirst%3DP.%2BM.%26aulast%3DVaranasi%26aufirst%3DL.%26aulast%3DFan%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DKubacka%26aufirst%3DI.%26aulast%3DNewman%26aufirst%3DV.%26aulast%3DChauhan%26aufirst%3DK.%26aulast%3DDaniels%26aufirst%3DS.%2BR.%26aulast%3DBoccetta%26aufirst%3DM.%26aulast%3DGarrett%26aufirst%3DM.%2BR.%26aulast%3DLi%26aufirst%3DR.%26aulast%3DMartinez%26aufirst%3DL.%2BA.%26atitle%3DMutant%2520p53%2520cooperates%2520with%2520ETS2%2520to%2520promote%2520etoposide%2520resistance%26jtitle%3DGenes%2520Dev.%26date%3D2012%26volume%3D26%26spage%3D830%26epage%3D845%26doi%3D10.1101%2Fgad.181685.111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Virgen-Slane, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rozovics, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fitzgerald, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ngo, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chou, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van der Heden van Noort, G. J. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filippov, D. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gershon, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Semler, B. L.</span><span> </span><span class="NLM_article-title">An RNA virus hijacks an incognito function of a DNA repair enzyme</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">109</span><span class="NLM_x">, </span> <span class="NLM_fpage">14634</span><span class="NLM_x">–</span> <span class="NLM_lpage">14639</span><span class="refDoi"> DOI: 10.1073/pnas.1208096109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=10.1073%2Fpnas.1208096109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=22908287" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVaqu7zF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=14634-14639&author=R.+Virgen-Slaneauthor=J.+M.+Rozovicsauthor=K.+D.+Fitzgeraldauthor=T.+Ngoauthor=W.+Chouauthor=G.+J.+V.+van+der+Heden+van+Noortauthor=D.+V.+Filippovauthor=P.+D.+Gershonauthor=B.+L.+Semler&title=An+RNA+virus+hijacks+an+incognito+function+of+a+DNA+repair+enzyme&doi=10.1073%2Fpnas.1208096109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">An RNA virus hijacks an incognito function of a DNA repair enzyme</span></div><div class="casAuthors">Virgen-Slane, Richard; Rozovics, Janet M.; Fitzgerald, Kerry D.; Ngo, Tuan; Chou, Wayne; van der Heden van Noort, Gerbrand J.; Filippov, Dmitri V.; Gershon, Paul D.; Semler, Bert L.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">14634-14639, S14634/1-S14634/3</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">A previously described mammalian cell activity, called VPg unlinkase, specifically cleaves a unique protein-RNA covalent linkage generated during the viral genomic RNA replication steps of a picornavirus infection.  For over three decades, the identity of this cellular activity and its normal role in the uninfected cell had remained elusive.  Here we report the purifn. and identification of VPg unlinkase as the DNA repair enzyme, 5'-tyrosyl-DNA phosphodiesterase-2 (TDP2).  Our data show that VPg unlinkase activity in different mammalian cell lines correlates with their differential expression of TDP2.  Furthermore, we show that recombinant TDP2 can cleave the protein-RNA linkage generated by different picornaviruses without impairing the integrity of viral RNA.  Our results reveal a unique RNA repair-like function for TDP2 and suggest an unusual role in host-pathogen interactions for this cellular enzyme.  On the basis of the identification of TDP2 as a potential antiviral target, our findings may lead to the development of universal therapeutics to treat the millions of individuals afflicted annually with diseases caused by picornaviruses, including myocarditis, aseptic meningitis, encephalitis, hepatitis, and the common cold.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPMlLwsqL-cLVg90H21EOLACvtfcHk0lgZRZyw2Ztmsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVaqu7zF&md5=3af24e75947bd5cd81dcc21b79a67541</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1208096109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1208096109%26sid%3Dliteratum%253Aachs%26aulast%3DVirgen-Slane%26aufirst%3DR.%26aulast%3DRozovics%26aufirst%3DJ.%2BM.%26aulast%3DFitzgerald%26aufirst%3DK.%2BD.%26aulast%3DNgo%26aufirst%3DT.%26aulast%3DChou%26aufirst%3DW.%26aulast%3Dvan%2Bder%2BHeden%2Bvan%2BNoort%26aufirst%3DG.%2BJ.%2BV.%26aulast%3DFilippov%26aufirst%3DD.%2BV.%26aulast%3DGershon%26aufirst%3DP.%2BD.%26aulast%3DSemler%26aufirst%3DB.%2BL.%26atitle%3DAn%2520RNA%2520virus%2520hijacks%2520an%2520incognito%2520function%2520of%2520a%2520DNA%2520repair%2520enzyme%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26spage%3D14634%26epage%3D14639%26doi%3D10.1073%2Fpnas.1208096109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Koniger, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wingert, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosler, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beck, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nassal, M.</span><span> </span><span class="NLM_article-title">Involvement of the host DNA-repair enzyme TDP2 in formation of the covalently closed circular DNA persistence reservoir of hepatitis B viruses</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">111</span><span class="NLM_x">, </span> <span class="NLM_fpage">E4244</span><span class="NLM_x">–</span> <span class="NLM_lpage">4253</span><span class="refDoi"> DOI: 10.1073/pnas.1409986111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=10.1073%2Fpnas.1409986111" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=E4244-4253&author=C.+Konigerauthor=I.+Wingertauthor=M.+Marsmannauthor=C.+Roslerauthor=J.+Beckauthor=M.+Nassal&title=Involvement+of+the+host+DNA-repair+enzyme+TDP2+in+formation+of+the+covalently+closed+circular+DNA+persistence+reservoir+of+hepatitis+B+viruses&doi=10.1073%2Fpnas.1409986111"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1409986111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1409986111%26sid%3Dliteratum%253Aachs%26aulast%3DKoniger%26aufirst%3DC.%26aulast%3DWingert%26aufirst%3DI.%26aulast%3DMarsmann%26aufirst%3DM.%26aulast%3DRosler%26aufirst%3DC.%26aulast%3DBeck%26aufirst%3DJ.%26aulast%3DNassal%26aufirst%3DM.%26atitle%3DInvolvement%2520of%2520the%2520host%2520DNA-repair%2520enzyme%2520TDP2%2520in%2520formation%2520of%2520the%2520covalently%2520closed%2520circular%2520DNA%2520persistence%2520reservoir%2520of%2520hepatitis%2520B%2520viruses%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2014%26volume%3D111%26spage%3DE4244%26epage%3D4253%26doi%3D10.1073%2Fpnas.1409986111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Beck, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nassal, M.</span><span> </span><span class="NLM_article-title">Hepatitis B virus replication</span> <span class="citation_source-journal">World J. Gastroenterol</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">48</span><span class="NLM_x">–</span> <span class="NLM_lpage">64</span><span class="refDoi"> DOI: 10.3748/wjg.v13.i1.48</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=10.3748%2Fwjg.v13.i1.48" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=17206754" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=1%3ACAS%3A528%3ADC%252BD2sXitFSgt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2007&pages=48-64&author=J.+Beckauthor=M.+Nassal&title=Hepatitis+B+virus+replication&doi=10.3748%2Fwjg.v13.i1.48"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Hepatitis B virus replication</span></div><div class="casAuthors">Beck, Juergen; Nassal, Michael</div><div class="citationInfo"><span class="NLM_cas:title">World Journal of Gastroenterology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">48-64</span>CODEN:
                <span class="NLM_cas:coden">WJGAF2</span>;
        ISSN:<span class="NLM_cas:issn">1007-9327</span>.
    
            (<span class="NLM_cas:orgname">World Journal of Gastroenterology</span>)
        </div><div class="casAbstract">A review.  Hepadnaviruses, including human hepatitis B virus (HBV), replicate through reverse transcription of an RNA intermediate, the pregenomic RNA (pgRNA).  Despite this kinship to retroviruses, there are fundamental differences beyond the fact that hepadnavirions contain DNA instead of RNA.  Most peculiar is the initiation of reverse transcription: it occurs by protein-priming, is strictly committed to using an RNA hairpin on the pgRNA, ε, as template, and depends on cellular chaperones; moreover, proper replication can apparently occur only in the specialized environment of intact nucleocapsids.  This complexity has hampered an in-depth mechanistic understanding.  The recent successful reconstitution in the test tube of active replication initiation complexes from purified components, for duck HBV (DHBV), now allows for the anal. of the biochem. of hepadnaviral replication at the mol. level.  Here we review the current state of knowledge at all steps of the hepadnaviral genome replication cycle, with emphasis on new insights that turned up by the use of such cell-free systems.  At this time, they can, unfortunately, not be complemented by three-dimensional structural information on the involved components.  However, at least for the ε RNA element such information is emerging, raising expectations that combining biophysics with biochem. and genetics will soon provide a powerful integrated approach for solving the many outstanding questions.  The ultimate, though most challenging goal, will be to visualize the hepadnaviral reverse transcriptase in the act of synthesizing DNA, which will also have strong implications for drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeq4GL5xysiLVg90H21EOLACvtfcHk0lgAI_gCXqzMpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXitFSgt7w%253D&md5=a32783744d5f055abfdf9307bd883513</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.3748%2Fwjg.v13.i1.48&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3748%252Fwjg.v13.i1.48%26sid%3Dliteratum%253Aachs%26aulast%3DBeck%26aufirst%3DJ.%26aulast%3DNassal%26aufirst%3DM.%26atitle%3DHepatitis%2520B%2520virus%2520replication%26jtitle%3DWorld%2520J.%2520Gastroenterol%26date%3D2007%26volume%3D13%26spage%3D48%26epage%3D64%26doi%3D10.3748%2Fwjg.v13.i1.48" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Nassal, M.</span><span> </span><span class="NLM_article-title">HBV cccDNA: viral persistence reservoir and key obstacle for a cure of chronic hepatitis B</span> <span class="citation_source-journal">Gut</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">64</span><span class="NLM_x">, </span> <span class="NLM_fpage">1972</span><span class="NLM_x">–</span> <span class="NLM_lpage">1984</span><span class="refDoi"> DOI: 10.1136/gutjnl-2015-309809</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=10.1136%2Fgutjnl-2015-309809" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=26048673" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=1%3ACAS%3A528%3ADC%252BC28XmvVSrsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2015&pages=1972-1984&author=M.+Nassal&title=HBV+cccDNA%3A+viral+persistence+reservoir+and+key+obstacle+for+a+cure+of+chronic+hepatitis+B&doi=10.1136%2Fgutjnl-2015-309809"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">HBV cccDNA: viral persistence reservoir and key obstacle for a cure of chronic hepatitis B</span></div><div class="casAuthors">Nassal, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Gut</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1972-1984</span>CODEN:
                <span class="NLM_cas:coden">GUTTAK</span>;
        ISSN:<span class="NLM_cas:issn">0017-5749</span>.
    
            (<span class="NLM_cas:orgname">BMJ Publishing Group</span>)
        </div><div class="casAbstract">At least 250 million people worldwide are chronically infected with HBV, a small hepatotropic DNA virus that replicates through reverse transcription.  Chronic infection greatly increases the risk for terminal liver disease.  Current therapies rarely achieve a cure due to the refractory nature of an intracellular viral replication intermediate termed covalently closed circular (ccc) DNA.  Upon infection, cccDNA is generated as a plasmid-like episome in the host cell nucleus from the protein-linked relaxed circular (RC) DNA genome in incoming virions.  Its fundamental role is that as template for all viral RNAs, and in consequence new virions.  Biosynthesis of RC-DNA by reverse transcription of the viral pregenomic RNA is now understood in considerable detail, yet conversion of RC-DNA to cccDNA is still obscure, foremostly due to the lack of feasible, cccDNA-dependent assay systems.  Conceptual and recent exptl. data link cccDNA formation to cellular DNA repair, which is increasingly appreciated as a crit. interface between cells and viruses.  Together with new in vitro HBV infection systems, based on the identification of the bile acid transporter sodium taurocholate cotransporting polypeptide as an HBV entry receptor, this offers novel opportunities to decipher, and eventually interfere with, formation of the HBV persistence reservoir.  After a brief overview of the role of cccDNA in the HBV infectious cycle, this review aims to summarize current knowledge on cccDNA mol. biol., to highlight the exptl. restrictions that have hitherto hampered faster progress and to discuss cccDNA as target for new, potentially curative therapies of chronic hepatitis B.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQaHzOzO1JXLVg90H21EOLACvtfcHk0lgAI_gCXqzMpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmvVSrsbg%253D&md5=7cbbbccef38636fd9bbcae0a9b869bc9</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1136%2Fgutjnl-2015-309809&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Fgutjnl-2015-309809%26sid%3Dliteratum%253Aachs%26aulast%3DNassal%26aufirst%3DM.%26atitle%3DHBV%2520cccDNA%253A%2520viral%2520persistence%2520reservoir%2520and%2520key%2520obstacle%2520for%2520a%2520cure%2520of%2520chronic%2520hepatitis%2520B%26jtitle%3DGut%26date%3D2015%26volume%3D64%26spage%3D1972%26epage%3D1984%26doi%3D10.1136%2Fgutjnl-2015-309809" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Cui, X. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McAllister, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boregowda, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sohn, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ledesma, F. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caldecott, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seeger, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, J. M.</span><span> </span><span class="NLM_article-title">Does tyrosyl DNA phosphodiesterase-2 play a role in hepatitis B virus genome repair?</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">e0128401</span><span class="refDoi"> DOI: 10.1371/journal.pone.0128401</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=10.1371%2Fjournal.pone.0128401" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=26079492" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlWmur%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=e0128401&author=X.+J.+Cuiauthor=R.+McAllisterauthor=R.+Boregowdaauthor=J.+A.+Sohnauthor=F.+C.+Ledesmaauthor=K.+W.+Caldecottauthor=C.+Seegerauthor=J.+M.+Hu&title=Does+tyrosyl+DNA+phosphodiesterase-2+play+a+role+in+hepatitis+B+virus+genome+repair%3F&doi=10.1371%2Fjournal.pone.0128401"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Does tyrosyl DNA phosphodiesterase-2 play a role in hepatitis B virus genome repair?</span></div><div class="casAuthors">Cui, Xiuji; McAllister, Rebecca; Boregowda, Rajeev; Sohn, Ji A.; Ledesma, Felipe Cortes; Caldecott, Keith W.; Seeger, Christoph; Hu, Jianming</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">e0128401/1-e0128401/15</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Hepatitis B virus (HBV) replication and persistence are sustained by a nuclear episome, the covalently closed circular (CCC) DNA, which serves as the transcriptional template for all viral RNAs.  CCC DNA is converted from a relaxed circular (RC) DNA in the virion early during infection as well as from RC DNA in intracellular progeny nucleocapsids via an intracel- lular amplification pathway.  Current antiviral therapies suppress viral replication but cannot eliminate CCC DNA.  Thus, persistence of CCC DNA remains an obstacle toward curing chronic HBV infection.  Unfortunately, very little is known about how CCC DNA is formed.  CCC DNA formation requires removal of the virally encoded reverse transcriptase (RT) pro- tein from the 5' end of the minus strand of RC DNA.  Tyrosyl DNA phosphodiesterase-2 (Tdp2) was recently identified as the enzyme responsible for cleavage of tyrosyl-5' DNA linkages formed between topoisomerase II and cellular DNA.  Because the RT-DNA linkage is also a 5' DNA-phosphotyrosyl bond, it has been hypothesized that Tdp2 might be one of several elusive host factors required for CCC DNA formation.  Therefore, we examd. the role of Tdp2 in RC DNA deproteination and CCC DNA formation.  We demonstrated Tdp2 can cleave the tyrosyl-minus strand DNA linkage using authentic HBV RC DNA isolated from nucleocapsids and using RT covalently linked to short minus strand DNA produced in vitro.  On the other hand, our results showed that Tdp2 gene knockout did not block CCC DNA formation during HBV infection of permissive human hepatoma cells and did not prevent intracellular amplification of duck hepatitis B virus CCC DNA.  These results indicate that although Tdp2 can remove the RT covalently linked to the 5' end of the HBV minus strand DNA in vitro, this protein might not be required for CCC DNA formation in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhjQ8pGys-lLVg90H21EOLACvtfcHk0lgAI_gCXqzMpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlWmur%252FO&md5=76ed1ebb188d7d10a45523e65f2bd57a</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0128401&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0128401%26sid%3Dliteratum%253Aachs%26aulast%3DCui%26aufirst%3DX.%2BJ.%26aulast%3DMcAllister%26aufirst%3DR.%26aulast%3DBoregowda%26aufirst%3DR.%26aulast%3DSohn%26aufirst%3DJ.%2BA.%26aulast%3DLedesma%26aufirst%3DF.%2BC.%26aulast%3DCaldecott%26aufirst%3DK.%2BW.%26aulast%3DSeeger%26aufirst%3DC.%26aulast%3DHu%26aufirst%3DJ.%2BM.%26atitle%3DDoes%2520tyrosyl%2520DNA%2520phosphodiesterase-2%2520play%2520a%2520role%2520in%2520hepatitis%2520B%2520virus%2520genome%2520repair%253F%26jtitle%3DPLoS%2520One%26date%3D2015%26volume%3D10%26spage%3De0128401%26doi%3D10.1371%2Fjournal.pone.0128401" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Gao, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, S. Y. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marchand, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pommier, Y.</span><span> </span><span class="NLM_article-title">Biochemical characterization of human tyrosyl-DNA phosphodiesterase 2 (TDP2/TTRAP) a Mg<sup>2+</sup>/Mn<sup>2+</sup>-dependent phosphodiesterase specific for the repair of topoisomerase cleavage complexes</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">287</span><span class="NLM_x">, </span> <span class="NLM_fpage">30842</span><span class="NLM_x">–</span> <span class="NLM_lpage">30852</span><span class="refDoi"> DOI: 10.1074/jbc.M112.393983</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=10.1074%2Fjbc.M112.393983" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=30842-30852&author=R.+Gaoauthor=S.+Y.+N.+Huangauthor=C.+Marchandauthor=Y.+Pommier&title=Biochemical+characterization+of+human+tyrosyl-DNA+phosphodiesterase+2+%28TDP2%2FTTRAP%29+a+Mg2%2B%2FMn2%2B-dependent+phosphodiesterase+specific+for+the+repair+of+topoisomerase+cleavage+complexes&doi=10.1074%2Fjbc.M112.393983"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M112.393983&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M112.393983%26sid%3Dliteratum%253Aachs%26aulast%3DGao%26aufirst%3DR.%26aulast%3DHuang%26aufirst%3DS.%2BY.%2BN.%26aulast%3DMarchand%26aufirst%3DC.%26aulast%3DPommier%26aufirst%3DY.%26atitle%3DBiochemical%2520characterization%2520of%2520human%2520tyrosyl-DNA%2520phosphodiesterase%25202%2520%2528TDP2%252FTTRAP%2529%2520a%2520Mg2%252B%252FMn2%252B-dependent%2520phosphodiesterase%2520specific%2520for%2520the%2520repair%2520of%2520topoisomerase%2520cleavage%2520complexes%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26spage%3D30842%26epage%3D30852%26doi%3D10.1074%2Fjbc.M112.393983" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Schellenberg, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Appel, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adhikari, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robertson, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramsden, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, R. S.</span><span> </span><span class="NLM_article-title">Mechanism of repair of 5′-topoisomerase II-DNA adducts by mammalian tyrosyl- DNA phosphodiesterase 2</span> <span class="citation_source-journal">Nat. Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">1363</span><span class="NLM_x">–</span> <span class="NLM_lpage">1371</span><span class="refDoi"> DOI: 10.1038/nsmb.2418</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=10.1038%2Fnsmb.2418" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=23104055" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFOmt7jF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=1363-1371&author=M.+J.+Schellenbergauthor=C.+D.+Appelauthor=S.+Adhikariauthor=P.+D.+Robertsonauthor=D.+A.+Ramsdenauthor=R.+S.+Williams&title=Mechanism+of+repair+of+5%E2%80%B2-topoisomerase+II-DNA+adducts+by+mammalian+tyrosyl-+DNA+phosphodiesterase+2&doi=10.1038%2Fnsmb.2418"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism of repair of 5'-topoisomerase II-DNA adducts by mammalian tyrosyl-DNA phosphodiesterase 2</span></div><div class="casAuthors">Schellenberg, Matthew J.; Appel, C. Denise; Adhikari, Sanjay; Robertson, Patrick D.; Ramsden, Dale A.; Williams, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1363-1371</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The topoisomerase II (topo II) DNA incision-and-ligation cycle can be poisoned (for example following treatment with cancer chemotherapeutics) to generate cytotoxic DNA double-strand breaks (DSBs) with topo II covalently conjugated to DNA.  Tyrosyl-DNA phosphodiesterase 2 (Tdp2) protects genomic integrity by reversing 5'-phosphotyrosyl-linked topo II-DNA adducts.  Here, X-ray structures of mouse Tdp2-DNA complexes reveal that Tdp2 β-2-helix-β DNA damage-binding grasp, helical cap and DNA lesion-binding elements fuse to form an elongated protein-DNA conjugate substrate-interaction groove.  The Tdp2 DNA-binding surface is highly tailored for engagement of 5'-adducted single-stranded DNA ends and restricts non-specific endonucleolytic or exonucleolytic processing.  Structural, mutational and functional analyses support a single-metal ion catalytic mechanism for the exonuclease-endonuclease-phosphatase (EEP) nuclease superfamily and establish a mol. framework for targeted small-mol. blockade of Tdp2-mediated resistance to anticancer topoisomerase drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBb8RLbBMuDLVg90H21EOLACvtfcHk0lioEkI_HDdhYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFOmt7jF&md5=eacf558a24e0e77de0577fe8a15b3e06</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.2418&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.2418%26sid%3Dliteratum%253Aachs%26aulast%3DSchellenberg%26aufirst%3DM.%2BJ.%26aulast%3DAppel%26aufirst%3DC.%2BD.%26aulast%3DAdhikari%26aufirst%3DS.%26aulast%3DRobertson%26aufirst%3DP.%2BD.%26aulast%3DRamsden%26aufirst%3DD.%2BA.%26aulast%3DWilliams%26aufirst%3DR.%2BS.%26atitle%3DMechanism%2520of%2520repair%2520of%25205%25E2%2580%25B2-topoisomerase%2520II-DNA%2520adducts%2520by%2520mammalian%2520tyrosyl-%2520DNA%2520phosphodiesterase%25202%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2012%26volume%3D19%26spage%3D1363%26epage%3D1371%26doi%3D10.1038%2Fnsmb.2418" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Shi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurahashi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsutakawa, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tainer, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pommier, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aihara, H.</span><span> </span><span class="NLM_article-title">Structural basis for recognition of 5′-phosphotyrosine adducts by Tdp2</span> <span class="citation_source-journal">Nat. Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">1372</span><span class="NLM_x">–</span> <span class="NLM_lpage">1377</span><span class="refDoi"> DOI: 10.1038/nsmb.2423</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=10.1038%2Fnsmb.2423" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=23104058" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFOmtL3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=1372-1377&author=K.+Shiauthor=K.+Kurahashiauthor=R.+Gaoauthor=S.+E.+Tsutakawaauthor=J.+A.+Tainerauthor=Y.+Pommierauthor=H.+Aihara&title=Structural+basis+for+recognition+of+5%E2%80%B2-phosphotyrosine+adducts+by+Tdp2&doi=10.1038%2Fnsmb.2423"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for recognition of 5'-phosphotyrosine adducts by Tdp2</span></div><div class="casAuthors">Shi, Ke; Kurahashi, Kayo; Gao, Rui; Tsutakawa, Susan E.; Tainer, John A.; Pommier, Yves; Aihara, Hideki</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1372-1377</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The DNA-repair enzyme Tdp2 resolves 5'-phosphotyrosyl DNA adducts and mediates resistance to anticancer drugs that target covalent topoisomerase-DNA complexes.  Tdp2 also participates in key signaling pathways during development and tumorigenesis and cleaves a protein-RNA linkage during picornavirus replication.  The crystal structure of zebrafish Tdp2 bound to DNA reveals a deep, narrow basic groove that selectively accommodates the 5' end of single-stranded DNA in a stretched conformation.  The crystal structure of the full-length Caenorhabditis elegansTdp2 shows that this groove can also accommodate an acidic peptide stretch in vitro, with glutamate and aspartate side chains occupying the DNA backbone phosphate-binding sites.  This extensive mol. mimicry suggests a potential mechanism for autoregulation and interaction of Tdp2 with phosphorylated proteins in signaling.  Our study provides a framework to interrogate functions of Tdp2 and develop inhibitors for chemotherapeutic and antiviral applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoExrAgpVGQ8LVg90H21EOLACvtfcHk0lioEkI_HDdhYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFOmtL3F&md5=c49681f32d5b045ec8570358869287d7</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.2423&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.2423%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DK.%26aulast%3DKurahashi%26aufirst%3DK.%26aulast%3DGao%26aufirst%3DR.%26aulast%3DTsutakawa%26aufirst%3DS.%2BE.%26aulast%3DTainer%26aufirst%3DJ.%2BA.%26aulast%3DPommier%26aufirst%3DY.%26aulast%3DAihara%26aufirst%3DH.%26atitle%3DStructural%2520basis%2520for%2520recognition%2520of%25205%25E2%2580%25B2-phosphotyrosine%2520adducts%2520by%2520Tdp2%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2012%26volume%3D19%26spage%3D1372%26epage%3D1377%26doi%3D10.1038%2Fnsmb.2423" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Garcin, E. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hosfield, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Desai, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haas, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bjoras, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cunningham, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tainer, J. A.</span><span> </span><span class="NLM_article-title">DNA apurinic-apyrimidinic site binding and excision by endonuclease IV</span> <span class="citation_source-journal">Nat. Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">515</span><span class="NLM_x">–</span> <span class="NLM_lpage">522</span><span class="refDoi"> DOI: 10.1038/nsmb.1414</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=10.1038%2Fnsmb.1414" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=18408731" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlsFSltb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2008&pages=515-522&author=E.+D.+Garcinauthor=D.+J.+Hosfieldauthor=S.+A.+Desaiauthor=B.+J.+Haasauthor=M.+Bjorasauthor=R.+P.+Cunninghamauthor=J.+A.+Tainer&title=DNA+apurinic-apyrimidinic+site+binding+and+excision+by+endonuclease+IV&doi=10.1038%2Fnsmb.1414"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">DNA apurinic-apyrimidinic site binding and excision by endonuclease IV</span></div><div class="casAuthors">Garcin, Elsa D.; Hosfield, David J.; Desai, Sunil A.; Haas, Brian J.; Bjoeras, Magnar; Cunningham, Richard P.; Tainer, John A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">515-522</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Escherichia coli endonuclease IV is an archetype for an abasic or apurinic-apyrimidinic endonuclease superfamily crucial for DNA base excision repair.  Here biochem., mutational and crystallog. characterizations reveal a three-metal ion mechanism for damage binding and incision.  The 1.10-Å resoln. DNA-free and the 2.45-Å resoln. DNA-substrate complex structures capture substrate stabilization by Arg37 and reveal a distorted Zn3-ligand arrangement that reverts, after catalysis, to an ideal geometry suitable to hold rather than release cleaved DNA product.  The 1.45-Å resoln. DNA-product complex structure shows how Tyr72 caps the active site, tunes its dielec. environment and promotes catalysis by Glu261-activated hydroxide, bound to two Zn2+ ions throughout catalysis.  These structural, mutagenesis and biochem. results suggest general requirements for abasic site removal in contrast to features specific to the distinct endonuclease IV α-β triose phosphate isomerase (TIM) barrel and APE1 four-layer α-β folds of the apurinic-apyrimidinic endonuclease families.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqyM9U4gyIqJrVg90H21EOLACvtfcHk0ljhHF1gDrkj4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlsFSltb4%253D&md5=ca46b5de6944cd3bef5c7213fdba7c41</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.1414&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.1414%26sid%3Dliteratum%253Aachs%26aulast%3DGarcin%26aufirst%3DE.%2BD.%26aulast%3DHosfield%26aufirst%3DD.%2BJ.%26aulast%3DDesai%26aufirst%3DS.%2BA.%26aulast%3DHaas%26aufirst%3DB.%2BJ.%26aulast%3DBjoras%26aufirst%3DM.%26aulast%3DCunningham%26aufirst%3DR.%2BP.%26aulast%3DTainer%26aufirst%3DJ.%2BA.%26atitle%3DDNA%2520apurinic-apyrimidinic%2520site%2520binding%2520and%2520excision%2520by%2520endonuclease%2520IV%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2008%26volume%3D15%26spage%3D515%26epage%3D522%26doi%3D10.1038%2Fnsmb.1414" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Raoof, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Depledge, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamilton, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamilton, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hitchin, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopkins, G. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jordan, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maguire, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGonagle, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mould, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rushbrooke, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Small, H. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomson, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turlais, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waddell, I. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waszkowycz, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watson, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ogilvie, D. J.</span><span> </span><span class="NLM_article-title">Toxoflavins and deazaflavins as the first reported selective small molecule inhibitors of tyrosyl-DNA phosphodiesterase II</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">6352</span><span class="NLM_x">–</span> <span class="NLM_lpage">6370</span><span class="refDoi"> DOI: 10.1021/jm400568p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400568p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFanu7zF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=6352-6370&author=A.+Raoofauthor=P.+Depledgeauthor=N.+M.+Hamiltonauthor=N.+S.+Hamiltonauthor=J.+R.+Hitchinauthor=G.+V.+Hopkinsauthor=A.+M.+Jordanauthor=L.+A.+Maguireauthor=A.+E.+McGonagleauthor=D.+P.+Mouldauthor=M.+Rushbrookeauthor=H.+F.+Smallauthor=K.+M.+Smithauthor=G.+J.+Thomsonauthor=F.+Turlaisauthor=I.+D.+Waddellauthor=B.+Waszkowyczauthor=A.+J.+Watsonauthor=D.+J.+Ogilvie&title=Toxoflavins+and+deazaflavins+as+the+first+reported+selective+small+molecule+inhibitors+of+tyrosyl-DNA+phosphodiesterase+II&doi=10.1021%2Fjm400568p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Toxoflavins and Deazaflavins as the First Reported Selective Small Molecule Inhibitors of Tyrosyl-DNA Phosphodiesterase II</span></div><div class="casAuthors">Raoof, Ali; Depledge, Paul; Hamilton, Niall M.; Hamilton, Nicola S.; Hitchin, James R.; Hopkins, Gemma V.; Jordan, Allan M.; Maguire, Laura A.; McGonagle, Alison E.; Mould, Daniel P.; Rushbrooke, Mathew; Small, Helen F.; Smith, Kate M.; Thomson, Graeme J.; Turlais, Fabrice; Waddell, Ian D.; Waszkowycz, Bohdan; Watson, Amanda J.; Ogilvie, Donald J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">6352-6370</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The recently discovered enzyme tyrosyl-DNA phosphodiesterase 2 (TDP2) has been implicated in the topoisomerase-mediated repair of DNA damage.  In the clin. setting, it has been hypothesized that TDP2 may mediate drug resistance to topoisomerase II (topo II) inhibition by etoposide.  Therefore, selective pharmacol. inhibition of TDP2 is proposed as a novel approach to overcome intrinsic or acquired resistance to topo II-targeted drug therapy.  Following a high-throughput screening (HTS) campaign, toxoflavins and deazaflavins were identified as the first reported sub-micromolar and selective inhibitors of this enzyme.  Toxoflavin derivs., e.g., I (R1 = 4-EtC6H4, 4-F3CC6H4, 2-MeC6H4, etc.), appeared to exhibit a clear structure-activity relationship (SAR) for TDP2 enzymic inhibition.  However, we obsd. a key redox liability of this series, and this, alongside early in vitro drug metab. and pharmacokinetics (DMPK) issues, precluded further exploration.  The deazaflavins, II (R1 = Ph, cyclopropyl, 1H-indazol-6-yl, etc., R2 = H, Cl, R3 = H, Cl, CF3, CN, R4, R5 = H, Cl), were developed from a singleton HTS hit.  This series showed distinct SAR and did not display redox activity; however low cell permeability proved to be a challenge.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRqCoHvrkdmrVg90H21EOLACvtfcHk0ljhHF1gDrkj4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFanu7zF&md5=ba84b5ce65959a1967e99d40a2fb7825</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fjm400568p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400568p%26sid%3Dliteratum%253Aachs%26aulast%3DRaoof%26aufirst%3DA.%26aulast%3DDepledge%26aufirst%3DP.%26aulast%3DHamilton%26aufirst%3DN.%2BM.%26aulast%3DHamilton%26aufirst%3DN.%2BS.%26aulast%3DHitchin%26aufirst%3DJ.%2BR.%26aulast%3DHopkins%26aufirst%3DG.%2BV.%26aulast%3DJordan%26aufirst%3DA.%2BM.%26aulast%3DMaguire%26aufirst%3DL.%2BA.%26aulast%3DMcGonagle%26aufirst%3DA.%2BE.%26aulast%3DMould%26aufirst%3DD.%2BP.%26aulast%3DRushbrooke%26aufirst%3DM.%26aulast%3DSmall%26aufirst%3DH.%2BF.%26aulast%3DSmith%26aufirst%3DK.%2BM.%26aulast%3DThomson%26aufirst%3DG.%2BJ.%26aulast%3DTurlais%26aufirst%3DF.%26aulast%3DWaddell%26aufirst%3DI.%2BD.%26aulast%3DWaszkowycz%26aufirst%3DB.%26aulast%3DWatson%26aufirst%3DA.%2BJ.%26aulast%3DOgilvie%26aufirst%3DD.%2BJ.%26atitle%3DToxoflavins%2520and%2520deazaflavins%2520as%2520the%2520first%2520reported%2520selective%2520small%2520molecule%2520inhibitors%2520of%2520tyrosyl-DNA%2520phosphodiesterase%2520II%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D6352%26epage%3D6370%26doi%3D10.1021%2Fjm400568p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Chen, Y. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kesler, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sham, Y. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vince, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geraghty, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Z. Q.</span><span> </span><span class="NLM_article-title">The design, synthesis and biological evaluations of C-6 or C-7 substituted 2-hydroxyisoquinoline-1,3-diones as inhibitors of hepatitis C virus</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">467</span><span class="NLM_x">–</span> <span class="NLM_lpage">479</span><span class="refDoi"> DOI: 10.1016/j.bmc.2011.10.058</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=10.1016%2Fj.bmc.2011.10.058" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=22100256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=1%3ACAS%3A528%3ADC%252BC38XitlGlsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2012&pages=467-479&author=Y.+L.+Chenauthor=J.+Tangauthor=M.+J.+Keslerauthor=Y.+Y.+Shamauthor=R.+Vinceauthor=R.+J.+Geraghtyauthor=Z.+Q.+Wang&title=The+design%2C+synthesis+and+biological+evaluations+of+C-6+or+C-7+substituted+2-hydroxyisoquinoline-1%2C3-diones+as+inhibitors+of+hepatitis+C+virus&doi=10.1016%2Fj.bmc.2011.10.058"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">The design, synthesis and biological evaluations of C-6 or C-7 substituted 2-hydroxyisoquinoline-1,3-diones as inhibitors of hepatitis C virus</span></div><div class="casAuthors">Chen, Yue-Lei; Tang, Jing; Kesler, Matthew J.; Sham, Yuk Y.; Vince, Robert; Geraghty, Robert J.; Wang, Zhengqiang</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">467-479</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">C7-Substituted 2-hydroxyisoquinoline-1,3-diones inhibit the strand transfer of HIV integrase (IN) and the reverse-transcriptase-assocd. RNase H (RNH).  Hepatitis C virus (HCV) NS5B polymerase shares a similar active site fold to RNH and IN, suggesting that N-hydroxyimides could be useful inhibitor scaffolds of HCV via targeting the NS5B.  Herein is described the design, chem. synthesis, replicon and biochem. assays, and mol. docking of C-6 or C-7 aryl substituted 2-hydroxyisoquinoline-1,3-diones as novel HCV inhibitors.  The synthesis involved an improved and clean cyclization method, which allowed the convenient prepn. of various analogs.  Biol. studies revealed that the C-6 analogs, a previously unknown chemotype, consistently inhibit both HCV replicon and recombinant NS5B at low micromolar range.  Mol. modeling studies suggest that these inhibitors may bind to the NS5B active site.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnGy-yoKIh_7Vg90H21EOLACvtfcHk0ljhHF1gDrkj4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XitlGlsA%253D%253D&md5=9d03fdf85f6df43aa80eb51611f572e3</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2011.10.058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2011.10.058%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%2BL.%26aulast%3DTang%26aufirst%3DJ.%26aulast%3DKesler%26aufirst%3DM.%2BJ.%26aulast%3DSham%26aufirst%3DY.%2BY.%26aulast%3DVince%26aufirst%3DR.%26aulast%3DGeraghty%26aufirst%3DR.%2BJ.%26aulast%3DWang%26aufirst%3DZ.%2BQ.%26atitle%3DThe%2520design%252C%2520synthesis%2520and%2520biological%2520evaluations%2520of%2520C-6%2520or%2520C-7%2520substituted%25202-hydroxyisoquinoline-1%252C3-diones%2520as%2520inhibitors%2520of%2520hepatitis%2520C%2520virus%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2012%26volume%3D20%26spage%3D467%26epage%3D479%26doi%3D10.1016%2Fj.bmc.2011.10.058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Tsou, H. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Otteng, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tran, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floyd, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reich, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Birnberg, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kutterer, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ayral-Kaloustian, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ravi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nilakantan, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grillo, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGinnis, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rabindran, S. K.</span><span> </span><span class="NLM_article-title">4-(Phenylaminomethylene)isoquinoline-1,3(2<i>H</i>,4<i>H</i>)-diones as potent and selective inhibitors of the cyclin-dependent kinase 4 (CDK4)</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">3507</span><span class="NLM_x">–</span> <span class="NLM_lpage">3525</span><span class="refDoi"> DOI: 10.1021/jm800072z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm800072z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=1%3ACAS%3A528%3ADC%252BD1cXmtF2rtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=3507-3525&author=H.+R.+Tsouauthor=M.+Ottengauthor=T.+Tranauthor=M.+B.+Floydauthor=M.+Reichauthor=G.+Birnbergauthor=K.+Kuttererauthor=S.+Ayral-Kaloustianauthor=M.+Raviauthor=R.+Nilakantanauthor=M.+Grilloauthor=J.+P.+McGinnisauthor=S.+K.+Rabindran&title=4-%28Phenylaminomethylene%29isoquinoline-1%2C3%282H%2C4H%29-diones+as+potent+and+selective+inhibitors+of+the+cyclin-dependent+kinase+4+%28CDK4%29&doi=10.1021%2Fjm800072z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">4-(Phenylaminomethylene)isoquinoline-1,3(2H,4H)-diones as Potent and Selective Inhibitors of the Cyclin-Dependent Kinase 4 (CDK4)</span></div><div class="casAuthors">Tsou, Hwei-Ru; Otteng, Mercy; Tran, Tritin; Floyd, M. Brawner, Jr.; Reich, Marvin; Birnberg, Gary; Kutterer, Kristina; Ayral-Kaloustian, Semiramis; Ravi, Malini; Nilakantan, Ramaswamy; Grillo, Mary; McGinnis, John P.; Rabindran, Sridhar K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3507-3525</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The cyclin-dependent kinases (CDKs), as complexes with their resp. partners, the cyclins, are crit. regulators of cell cycle progression.  Because aberrant regulations of CDK4/cyclin D1 lead to uncontrolled cell proliferation, a hallmark of cancer, small-mol. inhibitors of CDK4/cyclin D1 are attractive as prospective antitumor agents.  The series of 4-(phenylaminomethylene)isoquinoline-1,3(2H,4H)-dione derivs. reported here represents a novel class of potent inhibitors that selectively inhibit CDK4 over CDK2 and CDK1 activities.  In the headpiece of the 4-(phenylaminomethylene)isoquinoline-1,3(2H,4H)-dione, a basic amine substitutent is required on the aniline ring for the CDK4 inhibitory activity.  The inhibitory activity is further enhanced when an aryl or heteroaryl substituent is introduced at the C-6 position of the isoquinoline-1,3(2H,4H)-dione core.  We present here SAR data and a CDK4 mimic model that explains the binding, potency, and selectivity of our CDK4 selective inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq09yU47pubWbVg90H21EOLACvtfcHk0lj8h8kNeSof6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXmtF2rtrc%253D&md5=35083ce22416a584311a810ecf2b624d</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm800072z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm800072z%26sid%3Dliteratum%253Aachs%26aulast%3DTsou%26aufirst%3DH.%2BR.%26aulast%3DOtteng%26aufirst%3DM.%26aulast%3DTran%26aufirst%3DT.%26aulast%3DFloyd%26aufirst%3DM.%2BB.%26aulast%3DReich%26aufirst%3DM.%26aulast%3DBirnberg%26aufirst%3DG.%26aulast%3DKutterer%26aufirst%3DK.%26aulast%3DAyral-Kaloustian%26aufirst%3DS.%26aulast%3DRavi%26aufirst%3DM.%26aulast%3DNilakantan%26aufirst%3DR.%26aulast%3DGrillo%26aufirst%3DM.%26aulast%3DMcGinnis%26aufirst%3DJ.%2BP.%26aulast%3DRabindran%26aufirst%3DS.%2BK.%26atitle%3D4-%2528Phenylaminomethylene%2529isoquinoline-1%252C3%25282H%252C4H%2529-diones%2520as%2520potent%2520and%2520selective%2520inhibitors%2520of%2520the%2520cyclin-dependent%2520kinase%25204%2520%2528CDK4%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D3507%26epage%3D3525%26doi%3D10.1021%2Fjm800072z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Castro, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jerez, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gil, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Calderon, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Domenech, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nueda, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinez, A.</span><span> </span><span class="NLM_article-title">CODES, a novel procedure for ligand-based virtual screening: PDE7 inhibitors as an application example</span> <span class="citation_source-journal">Eur. J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">1349</span><span class="NLM_x">–</span> <span class="NLM_lpage">1359</span><span class="refDoi"> DOI: 10.1016/j.ejmech.2007.10.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=10.1016%2Fj.ejmech.2007.10.027" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2008&pages=1349-1359&author=A.+Castroauthor=M.+J.+Jerezauthor=C.+Gilauthor=F.+Calderonauthor=T.+Domenechauthor=A.+Nuedaauthor=A.+Martinez&title=CODES%2C+a+novel+procedure+for+ligand-based+virtual+screening%3A+PDE7+inhibitors+as+an+application+example&doi=10.1016%2Fj.ejmech.2007.10.027"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2007.10.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2007.10.027%26sid%3Dliteratum%253Aachs%26aulast%3DCastro%26aufirst%3DA.%26aulast%3DJerez%26aufirst%3DM.%2BJ.%26aulast%3DGil%26aufirst%3DC.%26aulast%3DCalderon%26aufirst%3DF.%26aulast%3DDomenech%26aufirst%3DT.%26aulast%3DNueda%26aufirst%3DA.%26aulast%3DMartinez%26aufirst%3DA.%26atitle%3DCODES%252C%2520a%2520novel%2520procedure%2520for%2520ligand-based%2520virtual%2520screening%253A%2520PDE7%2520inhibitors%2520as%2520an%2520application%2520example%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2008%26volume%3D43%26spage%3D1349%26epage%3D1359%26doi%3D10.1016%2Fj.ejmech.2007.10.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Jin, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, D. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gong, Y. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, R. Q.</span><span> </span><span class="NLM_article-title">Clicking fluoroionophores onto mesoporous silicas: a universal strategy toward efficient fluorescent surface sensors for metal ions</span> <span class="citation_source-journal">Anal. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">82</span><span class="NLM_x">, </span> <span class="NLM_fpage">6343</span><span class="NLM_x">–</span> <span class="NLM_lpage">6346</span><span class="refDoi"> DOI: 10.1021/ac101305e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ac101305e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=82&publication_year=2010&pages=6343-6346&author=Z.+Jinauthor=X.+B.+Zhangauthor=D.+X.+Xieauthor=Y.+J.+Gongauthor=J.+Zhangauthor=X.+Chenauthor=G.+L.+Shenauthor=R.+Q.+Yu&title=Clicking+fluoroionophores+onto+mesoporous+silicas%3A+a+universal+strategy+toward+efficient+fluorescent+surface+sensors+for+metal+ions&doi=10.1021%2Fac101305e"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fac101305e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac101305e%26sid%3Dliteratum%253Aachs%26aulast%3DJin%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DX.%2BB.%26aulast%3DXie%26aufirst%3DD.%2BX.%26aulast%3DGong%26aufirst%3DY.%2BJ.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DShen%26aufirst%3DG.%2BL.%26aulast%3DYu%26aufirst%3DR.%2BQ.%26atitle%3DClicking%2520fluoroionophores%2520onto%2520mesoporous%2520silicas%253A%2520a%2520universal%2520strategy%2520toward%2520efficient%2520fluorescent%2520surface%2520sensors%2520for%2520metal%2520ions%26jtitle%3DAnal.%2520Chem.%26date%3D2010%26volume%3D82%26spage%3D6343%26epage%3D6346%26doi%3D10.1021%2Fac101305e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Ali, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jackson, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howells, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maxwell, A.</span><span> </span><span class="NLM_article-title">The 43-kilodalton N-terminal fragment of the DNA gyrase B protein hydrolyzes ATP and binds coumarin drugs</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">2717</span><span class="NLM_x">–</span> <span class="NLM_lpage">2724</span><span class="refDoi"> DOI: 10.1021/bi00061a033</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi00061a033" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=1%3ACAS%3A528%3ADyaK3sXhtF2lt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=1993&pages=2717-2724&author=J.+A.+Aliauthor=A.+P.+Jacksonauthor=A.+J.+Howellsauthor=A.+Maxwell&title=The+43-kilodalton+N-terminal+fragment+of+the+DNA+gyrase+B+protein+hydrolyzes+ATP+and+binds+coumarin+drugs&doi=10.1021%2Fbi00061a033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">The 43-kilodalton N-terminal fragment of the DNA gyrase B protein hydrolyzes ATP and binds coumarin drugs</span></div><div class="casAuthors">Ali, Janid A.; Jackson, Andrew P.; Howells, Alison J.; Maxwell, Anthony</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2717-24</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    </div><div class="casAbstract">A gene encoding a 43-kDa protein corresponding to the N-terminal fragment of the DNA gyrase B subunit was cloned and overexpressed.  It was shown that this protein hydrolyzes ATP and binds coumarin drugs.  The hydrolysis of ATP showed distinctly non-Michaelis-Menten kinetics and was consistent with a scheme in which the active form of the protein was a dimer, a conclusion supported by mol. wt. studies.  The coumarin drugs bound very tightly to the 43-kDa fragment, with novobiocin binding to the protein monomer and coumermycin A1 apparently inducing the formation of a dimer.  The implications of these results with respect to the mechanism of supercoiling by DNA gyrase and the inhibition of gyrase by coumarin drugs were discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqKDubLhzserVg90H21EOLACvtfcHk0lj8h8kNeSof6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXhtF2lt74%253D&md5=c0fb5f7a4c5aa653cb0a26473afc3d74</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fbi00061a033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00061a033%26sid%3Dliteratum%253Aachs%26aulast%3DAli%26aufirst%3DJ.%2BA.%26aulast%3DJackson%26aufirst%3DA.%2BP.%26aulast%3DHowells%26aufirst%3DA.%2BJ.%26aulast%3DMaxwell%26aufirst%3DA.%26atitle%3DThe%252043-kilodalton%2520N-terminal%2520fragment%2520of%2520the%2520DNA%2520gyrase%2520B%2520protein%2520hydrolyzes%2520ATP%2520and%2520binds%2520coumarin%2520drugs%26jtitle%3DBiochemistry%26date%3D1993%26volume%3D32%26spage%3D2717%26epage%3D2724%26doi%3D10.1021%2Fbi00061a033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Repasky, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, L. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenwood, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halgren, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanschagrin, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mainz, D. T.</span><span> </span><span class="NLM_article-title">Extra precision glide: docking and scoring incorporating a model of hydrophobic enclosure for protein-ligand complexes</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">6177</span><span class="NLM_x">–</span> <span class="NLM_lpage">6196</span><span class="refDoi"> DOI: 10.1021/jm051256o</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm051256o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=1%3ACAS%3A528%3ADC%252BD28XpvVGmurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=6177-6196&author=R.+A.+Friesnerauthor=R.+B.+Murphyauthor=M.+P.+Repaskyauthor=L.+L.+Fryeauthor=J.+R.+Greenwoodauthor=T.+A.+Halgrenauthor=P.+C.+Sanschagrinauthor=D.+T.+Mainz&title=Extra+precision+glide%3A+docking+and+scoring+incorporating+a+model+of+hydrophobic+enclosure+for+protein-ligand+complexes&doi=10.1021%2Fjm051256o"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Extra Precision Glide: Docking and Scoring Incorporating a Model of Hydrophobic Enclosure for Protein-Ligand Complexes</span></div><div class="casAuthors">Friesner, Richard A.; Murphy, Robert B.; Repasky, Matthew P.; Frye, Leah L.; Greenwood, Jeremy R.; Halgren, Thomas A.; Sanschagrin, Paul C.; Mainz, Daniel T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6177-6196</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel scoring function to est. protein-ligand binding affinities has been developed and implemented as the Glide 4.0 XP scoring function and docking protocol.  In addn. to unique water desolvation energy terms, protein-ligand structural motifs leading to enhanced binding affinity are included:(1) hydrophobic enclosure where groups of lipophilic ligand atoms are enclosed on opposite faces by lipophilic protein atoms, (2) neutral-neutral single or correlated hydrogen bonds in a hydrophobically enclosed environment, and (3) five categories of charged-charged hydrogen bonds.  The XP scoring function and docking protocol have been developed to reproduce exptl. binding affinities for a set of 198 complexes (RMSDs of 2.26 and 1.73 kcal/mol over all and well-docked ligands, resp.) and to yield quality enrichments for a set of fifteen screens of pharmaceutical importance.  Enrichment results demonstrate the importance of the novel XP mol. recognition and water scoring in sepg. active and inactive ligands and avoiding false positives.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohZiwN12vaY7Vg90H21EOLACvtfcHk0lgwxGP_0wAmug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XpvVGmurg%253D&md5=ea428c82ead0d8c27f8c1a7b694a1edf</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm051256o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm051256o%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DGreenwood%26aufirst%3DJ.%2BR.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DSanschagrin%26aufirst%3DP.%2BC.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26atitle%3DExtra%2520precision%2520glide%253A%2520docking%2520and%2520scoring%2520incorporating%2520a%2520model%2520of%2520hydrophobic%2520enclosure%2520for%2520protein-ligand%2520complexes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D6177%26epage%3D6196%26doi%3D10.1021%2Fjm051256o" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Pommier, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, S. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Das, B. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murai, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marchand, C.</span><span> </span><span class="NLM_article-title">Tyrosyl-DNA-phosphodiesterases (TDP1 and TDP2)</span> <span class="citation_source-journal">DNA Repair</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">114</span><span class="NLM_x">–</span> <span class="NLM_lpage">129</span><span class="refDoi"> DOI: 10.1016/j.dnarep.2014.03.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=10.1016%2Fj.dnarep.2014.03.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=24856239" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=1%3ACAS%3A528%3ADC%252BC2cXoslCnurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2014&pages=114-129&author=Y.+Pommierauthor=S.+Y.+Huangauthor=R.+Gaoauthor=B.+B.+Dasauthor=J.+Muraiauthor=C.+Marchand&title=Tyrosyl-DNA-phosphodiesterases+%28TDP1+and+TDP2%29&doi=10.1016%2Fj.dnarep.2014.03.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Tyrosyl-DNA-phosphodiesterases (TDP1 and TDP2)</span></div><div class="casAuthors">Pommier, Yves; Huang, Shar-yin N.; Gao, Rui; Das, Benu Brata; Murai, Junko; Marchand, Christophe</div><div class="citationInfo"><span class="NLM_cas:title">DNA Repair</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">114-129</span>CODEN:
                <span class="NLM_cas:coden">DRNEAR</span>;
        ISSN:<span class="NLM_cas:issn">1568-7864</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  TDP1 and TDP2 were discovered and named based on the fact they process 3'- and 5'-DNA ends by excising irreversible protein tyrosyl-DNA complexes involving topoisomerases I and II, resp.  Yet, both enzymes have an extended spectrum of activities.  TDP1 not only excises trapped topoisomerases I (Top1 in the nucleus and Top1mt in mitochondria), but also repairs oxidative damage-induced 3'-phosphoglycolates and alkylation damage-induced DNA breaks, and excises chain terminating anticancer and antiviral nucleosides in the nucleus and mitochondria.  The repair function of TDP2 is devoted to the excision of topoisomerase II- and potentially topoisomerases III-DNA adducts.  TDP2 is also essential for the life cycle of picornaviruses (important human and bovine pathogens) as it unlinks VPg proteins from the 5'-end of the viral RNA genome.  Moreover, TDP2 has been involved in signal transduction (under the former names of TTRAP or EAPII).  The DNA repair partners of TDP1 include PARP1, XRCC1, ligase III and PNKP from the base excision repair (BER) pathway.  By contrast, TDP2 repair functions are coordinated with Ku and ligase IV in the non-homologous end joining pathway (NHEJ).  This article summarizes and compares the biochem., functions, and post-translational regulation of TDP1 and TDP2, as well as the relevance of TDP1 and TDP2 as determinants of response to anticancer agents.  The authors discuss the rationale for developing TDP inhibitors for combinations with topoisomerase inhibitors (topotecan, irinotecan, doxorubicin, etoposide, mitoxantrone) and DNA damaging agents (temozolomide, bleomycin, cytarabine, and ionizing radiation), and as novel antiviral agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDBH91msgKDbVg90H21EOLACvtfcHk0lgwxGP_0wAmug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXoslCnurc%253D&md5=8d0487ce63d11bd7312583c141b2269b</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.dnarep.2014.03.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.dnarep.2014.03.020%26sid%3Dliteratum%253Aachs%26aulast%3DPommier%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DS.%2BY.%26aulast%3DGao%26aufirst%3DR.%26aulast%3DDas%26aufirst%3DB.%2BB.%26aulast%3DMurai%26aufirst%3DJ.%26aulast%3DMarchand%26aufirst%3DC.%26atitle%3DTyrosyl-DNA-phosphodiesterases%2520%2528TDP1%2520and%2520TDP2%2529%26jtitle%3DDNA%2520Repair%26date%3D2014%26volume%3D19%26spage%3D114%26epage%3D129%26doi%3D10.1016%2Fj.dnarep.2014.03.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Marchand, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lea, W. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jadhav, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dexheimer, T. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Austin, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inglese, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pommier, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span> </span><span class="NLM_article-title">Identification of phosphotyrosine mimetic inhibitors of human tyrosyl-DNA phosphodiesterase I by a novel alphaScreen high-throughput assay</span> <span class="citation_source-journal">Mol. Cancer Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">240</span><span class="NLM_x">–</span> <span class="NLM_lpage">248</span><span class="refDoi"> DOI: 10.1158/1535-7163.MCT-08-0878</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=10.1158%2F1535-7163.MCT-08-0878" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=19139134" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjvFSltw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=240-248&author=C.+Marchandauthor=W.+A.+Leaauthor=A.+Jadhavauthor=T.+S.+Dexheimerauthor=C.+P.+Austinauthor=J.+Ingleseauthor=Y.+Pommierauthor=A.+Simeonov&title=Identification+of+phosphotyrosine+mimetic+inhibitors+of+human+tyrosyl-DNA+phosphodiesterase+I+by+a+novel+alphaScreen+high-throughput+assay&doi=10.1158%2F1535-7163.MCT-08-0878"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of phosphotyrosine mimetic inhibitors of human tyrosyl-DNA phosphodiesterase I by a novel AlphaScreen high-throughput assay</span></div><div class="casAuthors">Marchand, Christophe; Lea, Wendy A.; Jadhav, Ajit; Dexheimer, Thomas S.; Austin, Christopher P.; Inglese, James; Pommier, Yves; Simeonov, Anton</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">240-248</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Tyrosyl-DNA phosphodiesterase I (Tdp1) resolves topoisomerase I (Top1)-DNA adducts accumulated from natural DNA damage as well as from the action of certain anticancer drugs.  Tdp1 catalyzes the hydrolysis of the phosphodiester bond between the catalytic tyrosine residue of topoisomerase I and the DNA 3'-phosphate.  Only a limited no. of weak inhibitors have been reported for Tdp1, and there is an unmet need to identify novel chemotypes through screening of chem. libraries.  Herein, we present an easily configured, highly miniaturized, and robust Tdp1 assay using the AlphaScreen technol.  Uninhibited enzyme reaction is assocd. with low signal, whereas inhibition leads to a gain of signal, making the present assay format esp. attractive for automated large-collection high-throughput screening.  We report the identification and initial characterization of four previously unreported inhibitors of Tdp1.  Among them, suramin, NF449, and methyl-3,4-dephostatin are phosphotyrosine mimetics that may act as Tdp1 substrate decoys.  We also report a novel biochem. assay using the SCAN1 Tdp1 mutant to study the mechanism of action of methyl-3,4-dephostatin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqw2O21mqWopLVg90H21EOLACvtfcHk0lgwxGP_0wAmug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjvFSltw%253D%253D&md5=05a32d97be998b397a5dc3586b81bef4</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-08-0878&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-08-0878%26sid%3Dliteratum%253Aachs%26aulast%3DMarchand%26aufirst%3DC.%26aulast%3DLea%26aufirst%3DW.%2BA.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DDexheimer%26aufirst%3DT.%2BS.%26aulast%3DAustin%26aufirst%3DC.%2BP.%26aulast%3DInglese%26aufirst%3DJ.%26aulast%3DPommier%26aufirst%3DY.%26aulast%3DSimeonov%26aufirst%3DA.%26atitle%3DIdentification%2520of%2520phosphotyrosine%2520mimetic%2520inhibitors%2520of%2520human%2520tyrosyl-DNA%2520phosphodiesterase%2520I%2520by%2520a%2520novel%2520alphaScreen%2520high-throughput%2520assay%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2009%26volume%3D8%26spage%3D240%26epage%3D248%26doi%3D10.1158%2F1535-7163.MCT-08-0878" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Nguyen, T. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morrell, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Conda-Sheridan, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marchand, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Agama, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bermingam, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stephen, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chergui, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Naumova, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fisher, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Keefe, B. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pommier, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cushman, M.</span><span> </span><span class="NLM_article-title">Synthesis and biological evaluation of the first dual tyrosyl-DNA phosphodiesterase I (Tdp1)-topoisomerase I (Top1) inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">4457</span><span class="NLM_x">–</span> <span class="NLM_lpage">4478</span><span class="refDoi"> DOI: 10.1021/jm300335n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm300335n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=1%3ACAS%3A528%3ADC%252BC38XmtFartrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=4457-4478&author=T.+X.+Nguyenauthor=A.+Morrellauthor=M.+Conda-Sheridanauthor=C.+Marchandauthor=K.+Agamaauthor=A.+Bermingamauthor=A.+G.+Stephenauthor=A.+Cherguiauthor=A.+Naumovaauthor=R.+Fisherauthor=B.+R.+O%E2%80%99Keefeauthor=Y.+Pommierauthor=M.+Cushman&title=Synthesis+and+biological+evaluation+of+the+first+dual+tyrosyl-DNA+phosphodiesterase+I+%28Tdp1%29-topoisomerase+I+%28Top1%29+inhibitors&doi=10.1021%2Fjm300335n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and Biological Evaluation of the First Dual Tyrosyl-DNA Phosphodiesterase I (Tdp1)-Topoisomerase I (Top1) Inhibitors</span></div><div class="casAuthors">Nguyen, Trung Xuan; Morrell, Andrew; Conda-Sheridan, Martin; Marchand, Christophe; Agama, Keli; Berminham, Alun; Stephen, Andrew G.; Chergui, Adel; Naumova, Alena; Fisher, Robert; O'Keefe, Barry R.; Pommier, Yves; Cushman, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4457-4478</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Substances with dual tyrosyl-DNA phosphodiesterase I-topoisomerase I inhibitory activity in one low mol. wt. compd. would constitute a unique class of anticancer agents that could potentially have significant advantages over drugs that work against the individual enzymes.  The present study demonstrates the successful synthesis and evaluation of the first dual Top1-Tdp1 inhibitors, which are based on the indenoisoquinoline chemotype.  One bis(indenoisoquinoline) (I.2TFA) had significant activity against human Tdp1 (IC50 = 1.52 ± 0.05 μM), and it was also equipotent to camptothecin as a Top1 inhibitor.  Significant insights into enzyme-drug interactions were gained via structure-activity relationship studies of the series.  The present results also document the failure of the previously reported sulfonyl ester pharmacophore to confer Tdp1 inhibition in this indenoisoquinoline class of inhibitors even though it was demonstrated to work well for the steroid NSC 88915 (7).  The current study will facilitate future efforts to optimize dual Top1-Tdp1 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHBLLYpVMf5rVg90H21EOLACvtfcHk0lhORVHRiosLcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmtFartrY%253D&md5=c6f74f01e36da72fe9d5e0c435b9e3dd</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fjm300335n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300335n%26sid%3Dliteratum%253Aachs%26aulast%3DNguyen%26aufirst%3DT.%2BX.%26aulast%3DMorrell%26aufirst%3DA.%26aulast%3DConda-Sheridan%26aufirst%3DM.%26aulast%3DMarchand%26aufirst%3DC.%26aulast%3DAgama%26aufirst%3DK.%26aulast%3DBermingam%26aufirst%3DA.%26aulast%3DStephen%26aufirst%3DA.%2BG.%26aulast%3DChergui%26aufirst%3DA.%26aulast%3DNaumova%26aufirst%3DA.%26aulast%3DFisher%26aufirst%3DR.%26aulast%3DO%25E2%2580%2599Keefe%26aufirst%3DB.%2BR.%26aulast%3DPommier%26aufirst%3DY.%26aulast%3DCushman%26aufirst%3DM.%26atitle%3DSynthesis%2520and%2520biological%2520evaluation%2520of%2520the%2520first%2520dual%2520tyrosyl-DNA%2520phosphodiesterase%2520I%2520%2528Tdp1%2529-topoisomerase%2520I%2520%2528Top1%2529%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D4457%26epage%3D4478%26doi%3D10.1021%2Fjm300335n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">McRobb, F. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Capuano, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crosby, I. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chalmers, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuriev, E.</span><span> </span><span class="NLM_article-title">Homology modeling and docking evaluation of aminergic G protein-coupled receptors</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">626</span><span class="NLM_x">–</span> <span class="NLM_lpage">637</span><span class="refDoi"> DOI: 10.1021/ci900444q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci900444q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=1%3ACAS%3A528%3ADC%252BC3cXisFals78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=626-637&author=F.+M.+McRobbauthor=B.+Capuanoauthor=I.+T.+Crosbyauthor=D.+K.+Chalmersauthor=E.+Yuriev&title=Homology+modeling+and+docking+evaluation+of+aminergic+G+protein-coupled+receptors&doi=10.1021%2Fci900444q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Homology Modeling and Docking Evaluation of Aminergic G Protein-Coupled Receptors</span></div><div class="casAuthors">McRobb, Fiona M.; Capuano, Ben; Crosby, Ian T.; Chalmers, David K.; Yuriev, Elizabeth</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">626-637</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We report the development of homol. models of dopamine (D2, D3, and D4), serotonin (5-HT1B, 5-HT2A, 5-HT2B, and 5-HT2C), histamine (H1), and muscarinic (M1) receptors, based on the high-resoln. structure of the β2-adrenergic receptor.  The homol. models were built and refined using Prime.  We have addressed the required modeling of extracellular loop 2, which is often implicated in ligand binding.  The orthosteric sites of the models were optimized using induced fit docking, to allow for side-chain flexibility, and the resulting receptor models have been evaluated using protein validation tools.  Of the nine homol. models developed, six models showed moderate to good enrichment in virtual screening expts. (5-HT2A, 5-HT1B, D2, 5-HT2C, D3, and M1).  The 5-HT2A receptor displayed the highest enrichment in virtual screening expts. with enrichment factors of 6.1, 6.9, and 5.9 at 2%, 5%, and 10%, resp., of the screened database.  However, three of the models require further refinement (5-HT2B, D4, and H1), due to difficulties in modeling some of the binding site residues as well as the extracellular loop 2.  Our effort also aims to supplement the limited no. of tested G protein-coupled receptor homol. models based on the β2 crystal structure that are freely available to the research community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpe67bPU5vSUrVg90H21EOLACvtfcHk0lhORVHRiosLcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXisFals78%253D&md5=e9435245054966fd162f2cff4285ab11</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fci900444q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci900444q%26sid%3Dliteratum%253Aachs%26aulast%3DMcRobb%26aufirst%3DF.%2BM.%26aulast%3DCapuano%26aufirst%3DB.%26aulast%3DCrosby%26aufirst%3DI.%2BT.%26aulast%3DChalmers%26aufirst%3DD.%2BK.%26aulast%3DYuriev%26aufirst%3DE.%26atitle%3DHomology%2520modeling%2520and%2520docking%2520evaluation%2520of%2520aminergic%2520G%2520protein-coupled%2520receptors%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D626%26epage%3D637%26doi%3D10.1021%2Fci900444q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Goldfeld, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beuming, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span> </span><span class="NLM_article-title">Successful prediction of the intra and extracellular loops of four G-protein-coupled receptors</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">108</span><span class="NLM_x">, </span> <span class="NLM_fpage">8275</span><span class="NLM_x">–</span> <span class="NLM_lpage">8280</span><span class="refDoi"> DOI: 10.1073/pnas.1016951108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=10.1073%2Fpnas.1016951108" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=8275-8280&author=D.+A.+Goldfeldauthor=K.+Zhuauthor=T.+Beumingauthor=R.+A.+Friesner&title=Successful+prediction+of+the+intra+and+extracellular+loops+of+four+G-protein-coupled+receptors&doi=10.1073%2Fpnas.1016951108"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1016951108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1016951108%26sid%3Dliteratum%253Aachs%26aulast%3DGoldfeld%26aufirst%3DD.%2BA.%26aulast%3DZhu%26aufirst%3DK.%26aulast%3DBeuming%26aufirst%3DT.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DSuccessful%2520prediction%2520of%2520the%2520intra%2520and%2520extracellular%2520loops%2520of%2520four%2520G-protein-coupled%2520receptors%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26spage%3D8275%26epage%3D8280%26doi%3D10.1073%2Fpnas.1016951108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Madhavi Sastry, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adzhigirey, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Day, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Annabhimoju, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sherman, W.</span><span> </span><span class="NLM_article-title">Protein and ligand preparation: parameters, protocols, and influence on virtual screening enrichments</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">221</span><span class="NLM_x">–</span> <span class="NLM_lpage">234</span><span class="refDoi"> DOI: 10.1007/s10822-013-9644-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=10.1007%2Fs10822-013-9644-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=23579614" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmslalu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2013&pages=221-234&author=G.+Madhavi+Sastryauthor=M.+Adzhigireyauthor=T.+Dayauthor=R.+Annabhimojuauthor=W.+Sherman&title=Protein+and+ligand+preparation%3A+parameters%2C+protocols%2C+and+influence+on+virtual+screening+enrichments&doi=10.1007%2Fs10822-013-9644-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Protein and ligand preparation: parameters, protocols, and influence on virtual screening enrichments</span></div><div class="casAuthors">Madhavi Sastry, G.; Adzhigirey, Matvey; Day, Tyler; Annabhimoju, Ramakrishna; Sherman, Woody</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">221-234</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Structure-based virtual screening plays an important role in drug discovery and complements other screening approaches.  In general, protein crystal structures are prepd. prior to docking in order to add hydrogen atoms, optimize hydrogen bonds, remove at. clashes, and perform other operations that are not part of the x-ray crystal structure refinement process.  In addn., ligands must be prepd. to create 3-dimensional geometries, assign proper bond orders, and generate accessible tautomer and ionization states prior to virtual screening.  While the prerequisite for proper system prepn. is generally accepted in the field, an extensive study of the prepn. steps and their effect on virtual screening enrichments has not been performed.  In this work, we systematically explore each of the steps involved in prepg. a system for virtual screening.  We first explore a large no. of parameters using the Glide validation set of 36 crystal structures and 1,000 decoys.  We then apply a subset of protocols to the DUD database.  We show that database enrichment is improved with proper prepn. and that neglecting certain steps of the prepn. process produces a systematic degrdn. in enrichments, which can be large for some targets.  We provide examples illustrating the structural changes introduced by the prepn. that impact database enrichment.  While the work presented here was performed with the Protein Prepn. Wizard and Glide, the insights and guidance are expected to be generalizable to structure-based virtual screening with other docking methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruqN5nIOJtR7Vg90H21EOLACvtfcHk0lhORVHRiosLcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmslalu7c%253D&md5=259a6d547ef3e1310e091fb50fe8de16</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1007%2Fs10822-013-9644-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-013-9644-8%26sid%3Dliteratum%253Aachs%26aulast%3DMadhavi%2BSastry%26aufirst%3DG.%26aulast%3DAdzhigirey%26aufirst%3DM.%26aulast%3DDay%26aufirst%3DT.%26aulast%3DAnnabhimoju%26aufirst%3DR.%26aulast%3DSherman%26aufirst%3DW.%26atitle%3DProtein%2520and%2520ligand%2520preparation%253A%2520parameters%252C%2520protocols%252C%2520and%2520influence%2520on%2520virtual%2520screening%2520enrichments%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2013%26volume%3D27%26spage%3D221%26epage%3D234%26doi%3D10.1007%2Fs10822-013-9644-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="note"><p class="first last">Schrödinger Release 2014-3: Schrödinger Suite 2014-3.</p></div><div class="NLM_citation" id="rightTab-cit31a"><span><span class="NLM_label">(a) </span><div class="note"><p class="first last">Protein Preparation Wizard</p></div>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Protein+Preparation+Wizard."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31a&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit31b"><span><span class="NLM_label">(b) </span>Epik, <span class="NLM_edition">version 2.9</span>; <span class="NLM_publisher-name">Schrödinger, LLC</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Epik%2C+version+2.9%3B+Schr%C3%B6dinger%2C+LLC%3A+New+York%2C+2014."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31b&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DEpik%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit31c"><span><span class="NLM_label">(c) </span>Impact, <span class="NLM_edition">version 6.4</span>; <span class="NLM_publisher-name">Schrödinger, LLC</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Impact%2C+version+6.4%3B+Schr%C3%B6dinger%2C+LLC%3A+New+York%2C+2014."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31c&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DImpact%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit31d"><span><span class="NLM_label">(d) </span>Prime, <span class="NLM_edition">version 3.7</span>; <span class="NLM_publisher-name">Schrödinger, LLC</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Prime%2C+version+3.7%3B+Schr%C3%B6dinger%2C+LLC%3A+New+York%2C+2014."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31d&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DPrime%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maxwell, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">TiradoRives, J.</span><span> </span><span class="NLM_article-title">Development and testing of the OPLS all-atom force field on conformational energetics and properties of organic liquids</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">118</span><span class="NLM_x">, </span> <span class="NLM_fpage">11225</span><span class="NLM_x">–</span> <span class="NLM_lpage">11236</span><span class="refDoi"> DOI: 10.1021/ja9621760</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja9621760" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;key=1%3ACAS%3A528%3ADyaK28XmtlOitrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=1996&pages=11225-11236&author=W.+L.+Jorgensenauthor=D.+S.+Maxwellauthor=J.+TiradoRives&title=Development+and+testing+of+the+OPLS+all-atom+force+field+on+conformational+energetics+and+properties+of+organic+liquids&doi=10.1021%2Fja9621760"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Development and Testing of the OPLS All-Atom Force Field on Conformational Energetics and Properties of Organic Liquids</span></div><div class="casAuthors">Jorgensen, William L.; Maxwell, David S.; Tirado-Rives, Julian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">11225-11236</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The parametrization and testing of the OPLS all-atom force field for org. mols. and peptides are described.  Parameters for both torsional and nonbonded energetics have been derived, while the bond stretching and angle bending parameters have been adopted mostly from the AMBER all-atom force field.  The torsional parameters were detd. by fitting to rotational energy profiles obtained from ab initio MO calcns. at the RHF/6-31G*//RHF/6-31G* level for more than 50 org. mols. and ions.  The quality of the fits was high with av. errors for conformational energies of less than 0.2 kcal/mol.  The force-field results for mol. structures are also demonstrated to closely match the ab initio predictions.  The nonbonded parameters were developed in conjunction with Monte Carlo statistical mechanics simulations by computing thermodn. and structural properties for 34 pure org. liqs. including alkanes, alkenes, alcs., ethers, acetals, thiols, sulfides, disulfides, aldehydes, ketones, and amides.  Av. errors in comparison with exptl. data are 2% for heats of vaporization and densities.  The Monte Carlo simulations included sampling all internal and intermol. degrees of freedom.  It is found that such non-polar and monofunctional systems do not show significant condensed-phase effects on internal energies in going from the gas phase to the pure liqs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhGotSev_b1LVg90H21EOLACvtfcHk0lgbaIHcKMW9SQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XmtlOitrs%253D&md5=fef2924a69421881390282aa309ae91b</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fja9621760&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja9621760%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DMaxwell%26aufirst%3DD.%2BS.%26aulast%3DTiradoRives%26aufirst%3DJ.%26atitle%3DDevelopment%2520and%2520testing%2520of%2520the%2520OPLS%2520all-atom%2520force%2520field%2520on%2520conformational%2520energetics%2520and%2520properties%2520of%2520organic%2520liquids%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1996%26volume%3D118%26spage%3D11225%26epage%3D11236%26doi%3D10.1021%2Fja9621760" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><div class="note"><p class="first last">Schrödinger Release 2014-3:</p></div> <span class="citation_source-book">LigPrep</span>, <span class="NLM_edition">version 3.1</span>; <span class="NLM_publisher-name">Schrödinger, LLC</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Schr%C3%B6dinger+Release+2014-3%3ALigPrep%2C+version+3.1%3B+Schr%C3%B6dinger%2C+LLC%3A+New+York%2C+2014."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DLigPrep%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GZ1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GZ1','PDB','4GZ1'); return false;">PDB: 4GZ1</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i92"><a href="/doi/suppl/10.1021/acs.jmedchem.5b01973">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_52264"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.5b01973">10.1021/acs.jmedchem.5b01973</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Characterization data, including <sup>1</sup>H NMR data of intermediate chemotypes <b>21</b>, <b>24</b>, <b>25</b>, and <b>32</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.5b01973/suppl_file/jm5b01973_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.5b01973/suppl_file/jm5b01973_si_003.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01973/suppl_file/jm5b01973_si_001.pdf">jm5b01973_si_001.pdf (385.1 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01973/suppl_file/jm5b01973_si_003.csv">jm5b01973_si_003.csv (3.67 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.5b01973&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.5b01973%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2016.59.issue-6%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.5b01973" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799938ddcd63c58","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
