
<html lang="en"     class="pb-page"  data-request-id="6dcef5bc-4793-4c26-a0e0-ff597107344d"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.0c00075;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;issue:issue:10.1021/jmcmar.2020.63.issue-11"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Application of Atypical Acetyl-lysine Methyl Mimetics in the Development of Selective Inhibitors of the Bromodomain-Containing Protein 7 (BRD7)/Bromodomain-Containing Protein 9 (BRD9) Bromodomains" /></meta><meta name="dc.Creator" content="Michael A.  Clegg" /></meta><meta name="dc.Creator" content="Paul  Bamborough" /></meta><meta name="dc.Creator" content="Chun-wa  Chung" /></meta><meta name="dc.Creator" content="Peter D.  Craggs" /></meta><meta name="dc.Creator" content="Laurie  Gordon" /></meta><meta name="dc.Creator" content="Paola  Grandi" /></meta><meta name="dc.Creator" content="Melanie  Leveridge" /></meta><meta name="dc.Creator" content="Matthew  Lindon" /></meta><meta name="dc.Creator" content="Gemma M.  Liwicki" /></meta><meta name="dc.Creator" content="Anne-Marie  Michon" /></meta><meta name="dc.Creator" content="Judit  Molnar" /></meta><meta name="dc.Creator" content="Inmaculada  Rioja" /></meta><meta name="dc.Creator" content="Peter E.  Soden" /></meta><meta name="dc.Creator" content="Natalie H.  Theodoulou" /></meta><meta name="dc.Creator" content="Thilo  Werner" /></meta><meta name="dc.Creator" content="Nicholas C. O.  Tomkinson" /></meta><meta name="dc.Creator" content="Rab K.  Prinjha" /></meta><meta name="dc.Creator" content="Philip G.  Humphreys" /></meta><meta name="dc.Description" content="Non-BET bromodomain-containing proteins have become attractive targets for the development of novel therapeutics targeting epigenetic pathways. To help facilitate the target validation of this clas..." /></meta><meta name="Description" content="Non-BET bromodomain-containing proteins have become attractive targets for the development of novel therapeutics targeting epigenetic pathways. To help facilitate the target validation of this clas..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="May 15, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c00075" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00075" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c00075" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00075" /></link>
        
    
    

<title>Application of Atypical Acetyl-lysine Methyl Mimetics in the Development of Selective Inhibitors of the Bromodomain-Containing Protein 7 (BRD7)/Bromodomain-Containing Protein 9 (BRD9) Bromodomains | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c00075" /></meta><meta property="og:title" content="Application of Atypical Acetyl-lysine Methyl Mimetics in the Development of Selective Inhibitors of the Bromodomain-Containing Protein 7 (BRD7)/Bromodomain-Containing Protein 9 (BRD9) Bromodomains" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0030.jpeg" /></meta><meta property="og:description" content="Non-BET bromodomain-containing proteins have become attractive targets for the development of novel therapeutics targeting epigenetic pathways. To help facilitate the target validation of this class of proteins, structurally diverse small-molecule ligands and methodologies to produce selective inhibitors in a predictable fashion are in high demand. Herein, we report the development and application of atypical acetyl-lysine (KAc) methyl mimetics to take advantage of the differential stability of conserved water molecules in the bromodomain binding site. Discovery of the n-butyl group as an atypical KAc methyl mimetic allowed generation of 31 (GSK6776) as a soluble, permeable, and selective BRD7/9 inhibitor from a pyridazinone template. The n-butyl group was then used to enhance the bromodomain selectivity of an existing BRD9 inhibitor and to transform pan-bromodomain inhibitors into BRD7/9 selective compounds. Finally, a solvent-exposed vector was defined from the pyridazinone template to enable bifunctional molecule synthesis, and affinity enrichment chemoproteomic experiments were used to confirm several of the endogenous protein partners of BRD7 and BRD9, which form part of the chromatin remodeling PBAF and BAF complexes, respectively." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c00075"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00075">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c00075&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c00075&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c00075&amp;href=/doi/10.1021/acs.jmedchem.0c00075" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 11</span><span class="cit-fg-pageRange">, 5816-5840</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/11" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c00053" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c00088" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Application of Atypical Acetyl-lysine Methyl Mimetics in the Development of Selective Inhibitors of the Bromodomain-Containing Protein 7 (BRD7)/Bromodomain-Containing Protein 9 (BRD9) Bromodomains</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Michael A. Clegg</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Michael A. Clegg</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GlaxoSmithKline R&D, Stevenage SG1 2NY, Hertfordshire, United Kingdom</div><div class="loa-info-affiliations-info">WestCHEM, Department of Pure and Applied Chemistry, Thomas Graham Building, University of Strathclyde, 295 Cathedral Street, Glasgow G1 1XL, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Michael+A.++Clegg">Michael A. Clegg</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Paul Bamborough</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Paul Bamborough</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GlaxoSmithKline R&D, Stevenage SG1 2NY, Hertfordshire, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Paul++Bamborough">Paul Bamborough</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9479-2894" title="Orcid link">http://orcid.org/0000-0001-9479-2894</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Chun-wa Chung</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chun-wa Chung</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GlaxoSmithKline R&D, Stevenage SG1 2NY, Hertfordshire, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chun-wa++Chung">Chun-wa Chung</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2480-3110" title="Orcid link">http://orcid.org/0000-0002-2480-3110</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Peter D. Craggs</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Peter D. Craggs</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GlaxoSmithKline R&D, Stevenage SG1 2NY, Hertfordshire, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Peter+D.++Craggs">Peter D. Craggs</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Laurie Gordon</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Laurie Gordon</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GlaxoSmithKline R&D, Stevenage SG1 2NY, Hertfordshire, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Laurie++Gordon">Laurie Gordon</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Paola Grandi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Paola Grandi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cellzome GmbH, R&D MST GlaxoSmithKline, Meyerhofstrasse 1 69117 Heidelberg, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Paola++Grandi">Paola Grandi</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Melanie Leveridge</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Melanie Leveridge</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GlaxoSmithKline R&D, Stevenage SG1 2NY, Hertfordshire, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Melanie++Leveridge">Melanie Leveridge</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Matthew Lindon</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Matthew Lindon</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GlaxoSmithKline R&D, Stevenage SG1 2NY, Hertfordshire, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Matthew++Lindon">Matthew Lindon</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Gemma M. Liwicki</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Gemma M. Liwicki</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GlaxoSmithKline R&D, Stevenage SG1 2NY, Hertfordshire, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Gemma+M.++Liwicki">Gemma M. Liwicki</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3508-8711" title="Orcid link">http://orcid.org/0000-0003-3508-8711</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Anne-Marie Michon</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Anne-Marie Michon</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cellzome GmbH, R&D MST GlaxoSmithKline, Meyerhofstrasse 1 69117 Heidelberg, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Anne-Marie++Michon">Anne-Marie Michon</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Judit Molnar</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Judit Molnar</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GlaxoSmithKline R&D, Stevenage SG1 2NY, Hertfordshire, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Judit++Molnar">Judit Molnar</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Inmaculada Rioja</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Inmaculada Rioja</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GlaxoSmithKline R&D, Stevenage SG1 2NY, Hertfordshire, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Inmaculada++Rioja">Inmaculada Rioja</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Peter E. Soden</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Peter E. Soden</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GlaxoSmithKline R&D, Stevenage SG1 2NY, Hertfordshire, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Peter+E.++Soden">Peter E. Soden</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Natalie H. Theodoulou</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Natalie H. Theodoulou</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GlaxoSmithKline R&D, Stevenage SG1 2NY, Hertfordshire, United Kingdom</div><div class="loa-info-affiliations-info">WestCHEM, Department of Pure and Applied Chemistry, Thomas Graham Building, University of Strathclyde, 295 Cathedral Street, Glasgow G1 1XL, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Natalie+H.++Theodoulou">Natalie H. Theodoulou</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Thilo Werner</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Thilo Werner</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cellzome GmbH, R&D MST GlaxoSmithKline, Meyerhofstrasse 1 69117 Heidelberg, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Thilo++Werner">Thilo Werner</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Nicholas C. O. Tomkinson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nicholas C. O. Tomkinson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">WestCHEM, Department of Pure and Applied Chemistry, Thomas Graham Building, University of Strathclyde, 295 Cathedral Street, Glasgow G1 1XL, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nicholas+C.+O.++Tomkinson">Nicholas C. O. Tomkinson</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-5509-0133" title="Orcid link">http://orcid.org/0000-0002-5509-0133</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Rab K. Prinjha</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rab K. Prinjha</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GlaxoSmithKline R&D, Stevenage SG1 2NY, Hertfordshire, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rab+K.++Prinjha">Rab K. Prinjha</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Philip G. Humphreys</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Philip G. Humphreys</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GlaxoSmithKline R&D, Stevenage SG1 2NY, Hertfordshire, United Kingdom</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#413129282d28316f266f29342c3129332438320126322a6f222e2c"><span class="__cf_email__" data-cfemail="cfbfa7a6a3a6bfe1a8e1a7baa2bfa7bdaab6bc8fa8bca4e1aca0a2">[email protected]</span></a>. Phone: +44 (0)1438 764252.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Philip+G.++Humphreys">Philip G. Humphreys</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8614-7155" title="Orcid link">http://orcid.org/0000-0002-8614-7155</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00075&amp;href=/doi/10.1021%2Facs.jmedchem.0c00075" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 11</span><span class="cit-pageRange">, 5816–5840</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">May 15, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>15 January 2020</li><li><span class="item_label"><b>Published</b> online</span>15 May 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 11 June 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c00075" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00075</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D5816%26pageCount%3D25%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DMichael%2BA.%2BClegg%252C%2BPaul%2BBamborough%252C%2BChun-wa%2BChung%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D11%26contentID%3Dacs.jmedchem.0c00075%26title%3DApplication%2Bof%2BAtypical%2BAcetyl-lysine%2BMethyl%2BMimetics%2Bin%2Bthe%2BDevelopment%2Bof%2BSelective%2BInhibitors%2Bof%2Bthe%2BBromodomain-Containing%2BProtein%2B7%2B%2528BRD7%2529%252FBromodomain-Containing%2BProtein%2B9%2B%2528BRD9%2529%2BBromodomains%26numPages%3D25%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D5840%26publicationDate%3DJune%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c00075"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1634</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">3</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c00075" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Application of Atypical Acetyl-lysine Methyl Mimetics in the Development of Selective Inhibitors of the Bromodomain-Containing Protein 7 (BRD7)/Bromodomain-Containing Protein 9 (BRD9) Bromodomains&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Michael&quot;,&quot;last_name&quot;:&quot;A. Clegg&quot;},{&quot;first_name&quot;:&quot;Paul&quot;,&quot;last_name&quot;:&quot;Bamborough&quot;},{&quot;first_name&quot;:&quot;Chun-wa&quot;,&quot;last_name&quot;:&quot;Chung&quot;},{&quot;first_name&quot;:&quot;Peter&quot;,&quot;last_name&quot;:&quot;D. Craggs&quot;},{&quot;first_name&quot;:&quot;Laurie&quot;,&quot;last_name&quot;:&quot;Gordon&quot;},{&quot;first_name&quot;:&quot;Paola&quot;,&quot;last_name&quot;:&quot;Grandi&quot;},{&quot;first_name&quot;:&quot;Melanie&quot;,&quot;last_name&quot;:&quot;Leveridge&quot;},{&quot;first_name&quot;:&quot;Matthew&quot;,&quot;last_name&quot;:&quot;Lindon&quot;},{&quot;first_name&quot;:&quot;Gemma&quot;,&quot;last_name&quot;:&quot;M. Liwicki&quot;},{&quot;first_name&quot;:&quot;Anne-Marie&quot;,&quot;last_name&quot;:&quot;Michon&quot;},{&quot;first_name&quot;:&quot;Judit&quot;,&quot;last_name&quot;:&quot;Molnar&quot;},{&quot;first_name&quot;:&quot;Inmaculada&quot;,&quot;last_name&quot;:&quot;Rioja&quot;},{&quot;first_name&quot;:&quot;Peter&quot;,&quot;last_name&quot;:&quot;E. Soden&quot;},{&quot;first_name&quot;:&quot;Natalie&quot;,&quot;last_name&quot;:&quot;H. Theodoulou&quot;},{&quot;first_name&quot;:&quot;Thilo&quot;,&quot;last_name&quot;:&quot;Werner&quot;},{&quot;first_name&quot;:&quot;Nicholas&quot;,&quot;last_name&quot;:&quot;C. O. Tomkinson&quot;},{&quot;first_name&quot;:&quot;Rab&quot;,&quot;last_name&quot;:&quot;K. Prinjha&quot;},{&quot;first_name&quot;:&quot;Philip&quot;,&quot;last_name&quot;:&quot;G. Humphreys&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;05&quot;,&quot;day&quot;:&quot;15&quot;,&quot;issue&quot;:&quot;11&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;5816-5840&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c00075&quot;},&quot;abstract&quot;:&quot;Non-BET bromodomain-containing proteins have become attractive targets for the development of novel therapeutics targeting epigenetic pathways. To help facilitate the target validation of this class of proteins, structurally diverse small-molecule ligands and methodologies to produce selective inhibitors in a predictable fashion are in high demand. Herein, we report the development and application of atypical acetyl-lysine (KAc) methyl mimetics to take advantage of the differential stability of conserved water molecules in the bromodomain binding site. Discovery of the n-butyl group as an atypical KAc methyl mimetic allowed generation of 31 (GSK6776) as a soluble, permeable, and selective BRD7/9 inhibitor from a pyridazinone template. The n-butyl group was then used to enhance the bromodomain selectivity of an existing BRD9 inhibitor and to transform pan-bromodomain inhibitors into BRD7/9 selective compounds. Finally, a solvent-exposed vector was defined from the pyridazinone template to enable bifunctional m&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00075&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00075" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00075&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00075" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00075&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00075" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00075&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00075&amp;href=/doi/10.1021/acs.jmedchem.0c00075" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c00075" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c00075" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (12 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00075&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c00075%26sid%3Dliteratum%253Aachs%26pmid%3D32410449%26genre%3Darticle%26aulast%3DClegg%26date%3D2020%26atitle%3DApplication%2Bof%2BAtypical%2BAcetyl-lysine%2BMethyl%2BMimetics%2Bin%2Bthe%2BDevelopment%2Bof%2BSelective%2BInhibitors%2Bof%2Bthe%2BBromodomain-Containing%2BProtein%2B7%2B%2528BRD7%2529%252FBromodomain-Containing%2BProtein%2B9%2B%2528BRD9%2529%2BBromodomains%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D11%26spage%3D5816%26epage%3D5840%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=292349" title="Alkyls">Alkyls</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a>,</li><li><a href="/action/doSearch?ConceptID=291754" title="Molecules">Molecules</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/11" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/jmcmar.2020.63.issue-11/20200611/jmcmar.2020.63.issue-11.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/medium/jm0c00075_0030.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0030.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00075&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Non-BET bromodomain-containing proteins have become attractive targets for the development of novel therapeutics targeting epigenetic pathways. To help facilitate the target validation of this class of proteins, structurally diverse small-molecule ligands and methodologies to produce selective inhibitors in a predictable fashion are in high demand. Herein, we report the development and application of atypical acetyl-lysine (KAc) methyl mimetics to take advantage of the differential stability of conserved water molecules in the bromodomain binding site. Discovery of the <i>n</i>-butyl group as an atypical KAc methyl mimetic allowed generation of <b>31</b> (GSK6776) as a soluble, permeable, and selective BRD7/9 inhibitor from a pyridazinone template. The <i>n</i>-butyl group was then used to enhance the bromodomain selectivity of an existing BRD9 inhibitor and to transform pan-bromodomain inhibitors into BRD7/9 selective compounds. Finally, a solvent-exposed vector was defined from the pyridazinone template to enable bifunctional molecule synthesis, and affinity enrichment chemoproteomic experiments were used to confirm several of the endogenous protein partners of BRD7 and BRD9, which form part of the chromatin remodeling PBAF and BAF complexes, respectively.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_41137" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_41137" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Bromodomains are structurally conserved epigenetic reader modules found within bromodomain-containing proteins (BCPs), which selectively bind to context-sensitive acetylated lysine residues, thereby recruiting cellular transcriptional machinery to a specific histone mark and regulating gene expression.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> The structure of the 61 human bromodomains is composed of four antiparallel alpha helices (αZ, αA, αB, and αC) connected by two flexible loop regions (ZA and BC), which together form the hydrophobic pocket for acetylated lysine binding (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). Within the binding site of typical human bromodomains (48/61) are two conserved amino acid residues, Asn and Tyr, which form a direct and water-mediated hydrogen bond to the acetyl-lysine (KAc) residue. The water molecule involved in the latter interaction is one of four highly conserved water molecules also located in the binding site of the bromodomain.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/medium/jm0c00075_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. X-ray crystal structure of the N-terminal bromodomain of BRD4, BRD4(1), bound to KAc (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3uvy">3uvy</a>),<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> highlighting the four antiparallel alpha helices, loop regions, four conserved water molecules, and key interactions formed between typical bromodomains and acetylated lysines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00075&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Over the past decade, bromodomains have emerged as tractable therapeutic targets for immuno-inflammation, neurological disorders, metabolic disease, and oncology.<a onclick="showRef(event, 'ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref4 ref5">(4,5)</a> The bromodomain drug discovery landscape has been dominated by inhibitors of the bromodomain and extra terminal (BET) subfamily of proteins,<a onclick="showRef(event, 'ref6 ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref6 ref7 ref8">(6−8)</a> for which a strong and exciting biological phenotype is known. The remaining 53 non-BET BCPs have also been linked with a multitude of diseases and, as demonstrated by the first non-BET bromodomain inhibitor recently entering the clinic (CREBBP inhibitor CCS1477),<a onclick="showRef(event, 'ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref9 ref10">(9,10)</a> are attractive targets for future therapeutics. To help elucidate the role that BCPs play in modulating healthy and disease states, a variety of chemical and chemical biology approaches are being utilized for the preclinical target validation of their bromodomain regions.<a onclick="showRef(event, 'ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref11 ref12">(11,12)</a> One such approach is chemical probes, small molecules that selectively bind to a target and are used to interrogate its biological function.<a onclick="showRef(event, 'ref13 ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref13 ref14 ref15">(13−15)</a> Used in conjunction with techniques such as gene knockdown, chemical probes can provide complementary information into the modulation of individual protein domains, particularly prevalent with multidomain BCPs.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Fundamental to the success of target validation is the assignment of an observed phenotype to inhibition of the target in question, so access to multiple ligands of divergent chemotypes and structurally related negative controls helps facilitate this. Although significant advances have been made in populating the bromodomain small-molecule toolbox,<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> several BCPs lack the sufficient chemical equity required for robust target validation and would benefit from a deeper more diverse selection.</div><div class="NLM_p">BRD9 and BRD7 have been reported to be components of the mammalian chromatin remodeling complexes BRG1/BRM-associated factor (BAF) and polybromo-associated BAF (PBAF), respectively, two of the most frequently mutated complexes in cancer malignancies,<a onclick="showRef(event, 'ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref18 ref19">(18,19)</a> making BRD7 and BRD9 attractive targets for oncology.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> BRD9 has been shown to be involved in multiple cancers<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> and has recently emerged as a therapeutic target for rhabdoid tumors,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> while BRD7 has been found to function as a tumor suppressor across various cancers.<a onclick="showRef(event, 'ref23 ref24 ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref23 ref24 ref25 ref26">(23−26)</a> To enable investigations into the biology of the BRD7 and BRD9 bromodomains, in 2015, the Structural Genomics Consortium (SGC) and the University of Oxford reported the first micromolar BRD7/9 bromodomain inhibitor, LP99 (<b>1</b>) (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>).<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Since then, several BRD7/9 small-molecule tools have been reported (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>), spanning multiple chemotypes, although an unsaturated bicyclic system is common throughout. iBRD9 (<b>2</b>) was the first BRD9 inhibitor disclosed to display selectivity over the closely related BRD7 with nanomolar potency for BRD9.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> TP-472 (<b>3</b>),<a onclick="showRef(event, 'ref29 ref30'); return false;" href="javascript:void(0);" class="ref ref29 ref30">(29,30)</a> BI-9564 (<b>4</b>), and BI-7273 (<b>5</b>) have since been developed by SGC/Takeda and Boehringer Ingelheim, respectively.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> dBRD9 (<b>6</b>), the first BRD9 degrader, has also been disclosed utilizing <b>5</b> as a BRD9 selective binder.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> Degradation of BRD9 may find application in the treatment of strongly BRD9-dependent cancers such as synovial sarcoma.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> Finally, the selective BRD9 inhibitor GNE-375 (<b>7</b>) has recently been developed by Genentech and Constellation Pharmaceuticals.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> Small-molecule bromodomain inhibitors typically mimic the hydrogen bond acceptor motif and methyl group of the endogenous KAc ligand. The KAc methyl mimetic found most commonly in bromodomain small-molecule binders is the methyl group (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>, methyl mimetic highlighted in blue).<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> Chlorine and bromine groups have been used as methyl mimetics; however, the <i>E</i>-crotyl group found in GNE-375 (<b>7</b>) is particularly atypical as a KAc methyl mimetic, as is the mechanism by which it drives BRD7/9 selectivity (vide infra).<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a></div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/medium/jm0c00075_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0004.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Existing BRD7/9 inhibitors and their reported BRD7/9 potencies. The KAc methyl mimetic is highlighted in blue.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00075&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Selectivity is heavily scrutinized in medicinal chemistry and is particularly important when considering linking a biological phenotype induced by a small molecule to a target. As such, methods to obtain selectivity for a given target in a predictable and facile manner are valuable. Herein, we report the discovery and application of the <i>n</i>-butyl group as an atypical KAc methyl mimetic during a research program to obtain BRD7/9 inhibitors. The <i>n</i>-butyl group was critical to achieve the desired bromodomain selectivity profile and enabled delivery of a potent, selective, soluble, and cell-penetrant BRD7/9 inhibitor from an unselective pyridazinone start point bearing a chlorine KAc methyl mimetic. The <i>n</i>-butyl KAc methyl mimetic was then readily applied across diverse chemotypes to identify BRD7/9 selective inhibitors. Finally, a solvent-exposed vector was identified from the pyridazinone template to create a linkable analogue for bifunctional applications, and chemoproteomic affinity enrichment experiments were used to confirm the endogenous protein partners of BRD7 and BRD9, which form part of the PBAF and BAF complexes.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_08408" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_08408" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Discovery of the Pyridazinone Series as BRD9 Inhibitors</h3><div class="NLM_p">As reported previously, pyridazinones <b>8</b> and <b>9</b> were identified via a biochemical screen of molecules that were predicted to contain acetyl-lysine mimetics during work developing PCAF/GCN5 bromodomain inhibitors.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> During the course of this work, activity against the BET family of bromodomains, using the N-terminal bromodomain of BRD4, BRD4(1), as a representative example, was monitored as an antitarget due to the profound biological phenotype associated with BET inhibition. Fragment <b>8</b> is 5-fold biased over BRD4(1) and is equipotent against both PCAF and BRD9 (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). In contrast, benzyl <b>9</b> demonstrates increased activity and selectivity for BRD9 over both PCAF and BRD4(1). Although both BRD9 and PCAF possess Tyr gatekeeper residues, producing similar deep and narrow binding pockets, they possess significantly different WPF shelf motifs (PCAF: Trp746, Pro747, and Phe748; BRD9: Gly43, Phe44, and Phe45) (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>b). The impact on the binding site is manifested with Phe44 in BRD9, which sits almost perpendicular to Trp746 in PCAF and provides a more sterically confined pocket in BRD9 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>a).</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/medium/jm0c00075_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0005.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (a) Crystal structures of pyridazinone fragment <b>8</b> (yellow) bound to BRD9 (gray; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YQW">6YQW</a>) and PCAF (blue; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5MKX">5MKX</a>) highlighting the different “WPF” shelf motif residues. BRD9 amino acid residues are labeled in black, and PCAF amino acid residues are labeled in blue. (b) Sequence alignment of BRD9, BRD7, PCAF, and BRD4(1). The WPF moiety is colored orange, the conserved acetyl-lysine-binding asparagine and tyrosine are colored red and green, respectively, and the gatekeeper residue is colored blue.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00075&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Biochemical Profiling of Pyridazinones <b>8</b> and <b>9</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/medium/jm0c00075_0021.gif" alt="" id="gr21" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">pIC<sub>50</sub></th><th class="colsep0 rowsep0" align="center" char="."><b>8</b></th><th class="colsep0 rowsep0" align="center" char="."><b>9</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD9 pIC<sub>50</sub></td><td class="colsep0 rowsep0" align="char" char=".">4.6</td><td class="colsep0 rowsep0" align="char" char=".">5.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PCAF pIC<sub>50</sub> (selectivity)<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="char" char=".">4.7 (−)</td><td class="colsep0 rowsep0" align="char" char=".">4.9 (×6)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD4(1) pIC<sub>50</sub> (selectivity)<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="char" char=".">3.9 (×5)</td><td class="colsep0 rowsep0" align="char" char=".">4.7 (×10)</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Selectivity determined from the difference between PCAF or BRD4(1) pIC<sub>50</sub> and BRD9 pIC<sub>50</sub>.</p></div></div></div><div class="NLM_p last">With numerous pyridazinones synthesized and profiled across several bromodomain assays from PCAF/GCN5 studies,<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> interrogation of the unpublished data revealed a trend between BRD9 selectivity over both PCAF and BRD4(1) and sp<sup>2</sup> hybridization at the amine β-position carbon. On the assumption that selectivity between BRD7 and BRD9 was unlikely due to the high bromodomain homology (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>b), it was hypothesized that selective BRD7/9 inhibitors could be developed from the same pyridazinone template using sp<sup>2</sup> hybridization at the amine β-position as the start point, e.g., compound <b>9</b>.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Benzyl Ring SAR Investigation</h3><div class="NLM_p">SAR was initially focused around exploring the different vectors of the benzene ring in compound <b>9</b> to gain further interactions with BRD9 and improve potency (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Bromodomain potency for BRD9 and BRD4(1) was measured via a time-resolved fluorescence resonance energy transfer (TR-FRET) competition assay, where displacement of a fluorescent ligand from the bromodomain binding site provided pIC<sub>50</sub> values as measurement of affinity.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. BRD9/BRD4(1) SAR Table<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/medium/jm0c00075_0022.gif" alt="" id="gr22" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/medium/jm0c00075_0023.gif" alt="" id="gr23" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">pIC<sub>50</sub> values refer to potency within TR-FRET assays.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">LLE = pIC<sub>50</sub> – ChromLog<i>D</i><sub>pH7.4</sub>.</p></div><div class="footnote" id="t2fn3"><sup><sup>c</sup></sup><p class="last">Selectivity determined from the difference between BRD4(1) pIC<sub>50</sub> and BRD9 pIC<sub>50</sub>.</p></div></div><div></div></div><div class="NLM_p">As shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>a, the chloro group of the pyridazinone core functioned as the KAc methyl mimetic in compound <b>8</b>. On the assumption that the same was true for compound <b>9</b>, the Cl and Me KAc methyl mimetic matched molecular pair (<b>9</b> and <b>10</b>) demonstrated broadly similar activity at BRD9 and BRD4(1), although the Me group showed improved BRD9 lipophilic ligand efficiency (LLE) as a result of the reduced lipophilicity. Methyl substitution at the <i>ortho</i>, <i>meta</i>, and <i>para</i> positions of the benzene ring was tolerated (compounds <b>11</b>–<b>13</b>), with <i>ortho</i>-substituted <b>11</b> providing the most attractive combination of increased BRD9 potency, LLE, and selectivity over BRD4(1). Substitution of the <i>ortho</i>-methyl group for a benzene ring with <b>14</b> provided an increase in both potency against BRD9 (pIC<sub>50</sub> = 7.0) and lipophilicity. Although activity against the BET bromodomains increased slightly (BRD4(1) pIC<sub>50</sub> = 4.9), >100-fold selectivity was achieved for the first time.</div><div class="NLM_p">To understand the binding modes of compound <b>14</b> in both proteins and to gain insights into avenues for further optimization, X-ray crystallography in both BRD9 and BRD4(1) was obtained (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). In BRD9, the gatekeeper residue Tyr106 rotates relative to the published apo structure (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3hme">3hme</a>)<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> to make a face-to-face aromatic interaction with the pyridazinone core (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>a), an interaction not possible in BRD4(1) due to the isoleucine gatekeeper (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>b). The pyridazinone carbonyl group forms the canonical direct and water-mediated hydrogen bond interactions to Asn100 and Tyr57, respectively. The pyridazinone C–Me group functions as the KAc methyl mimetic, positioning the phenyl ring in the narrow cleft discussed previously with the <i>ortho</i>-phenyl ring twisted 67.6° to sit on the protein surface (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>b). The pyridazinone NH forms a direct hydrogen bond to the backbone carbonyl of Pro44, while the pyridazinone sp<sup>2</sup>-hybridized N forms a water-mediated hydrogen bond to the backbone carbonyl of Ile53. In contrast, crystallography of <b>14</b> bound to BRD4(1) revealed a dramatically different binding mode (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>c). While the canonical hydrogen bond interactions between the pyridazinone carbonyl and Asn140 together with a through-water hydrogen bond to Tyr97 are present, the pyridazinone core flips 180°, resulting in the N–Me group functioning as the KAc methyl mimetic. This allows the benzyl group to make partial hydrophobic interactions with the lipophilic WPF shelf. The <i>ortho</i>-phenyl group is directed into the narrow ZA channel formed between Leu92 and Trp81 with a 60.8° dihedral angle to avoid steric clashes with the protein (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>d). Presumably, the lower activity of <b>14</b> against BRD4(1) is driven by the lack of pyridazinone core face-to-face aromatic stacking with an aromatic gatekeeper residue, lack of NH hydrogen bond to the protein, and incomplete occupancy of the WPF shelf.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/medium/jm0c00075_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0006.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (a) X-ray crystallography of compound <b>14</b> (yellow) bound to (a) BRD9 (gray; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YQR">6YQR</a>), (b) BRD9 (gray; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YQR">6YQR</a>) highlighting the protein surface, (c) BRD4(1) (cyan; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YQZ">6YQZ</a>), and (d) BRD4(1) (cyan; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YQZ">6YQZ</a>) highlighting the protein surface.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00075&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">While <b>14</b> demonstrated that >100-fold selectivity over BRD4(1) was obtainable from the pyridazinone template, the molecule was unacceptably lipophilic (ChromLog<i>D</i><sub>pH7.4</sub> = 4.8) with a less than desirable LLE (BRD9 LLE, 2.2). As the <i>ortho</i>-phenyl ring sat in the narrow ZA cleft in BRD4(1), its replacement with polar saturated heterocycles was considered to not only further improve the selectivity over BRD4(1) via steric clashing but also improve BRD9 LLE.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> This work package was not anticipated to impact BRD9 potency drastically as the <i>ortho</i>-vector placed such substituents onto the BRD9 protein surface (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>b). Supporting the design hypothesis, morpholine <b>15</b> demonstrated improved BET selectivity (×200) while maintaining potency against BRD9 (pIC<sub>50</sub> = 7.0) and lowering lipophilicity (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Expansion to the seven-membered oxazepane <b>16</b> showed similar properties, whereas basic piperazine <b>17</b> showed further reduction in BET activity (pIC<sub>50</sub> < 4.3), together with a dramatic drop in lipophilicity and concomitant improvement in the BRD9 LLE to 6.0. Due to concerns over the low lipophilicity of <b>17</b> (ChromLog<i>D</i><sub>pH7.4</sub> = 0.7) and the likely impact on passive permeability,<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> methyl piperazine <b>18</b> was accessed to remove the additional hydrogen bond donor present in <b>17</b> and improve permeability while maintaining BRD9 potency, BRD4(1) selectivity, and lipophilicity within acceptable ranges. Compound <b>18</b> achieved these goals with no change in BRD9 potency, a small drop in BRD9 LLE, and desirable chemiluminescent nitrogen detection (CLND) solubility and artificial membrane permeability (AMP), indicating likely cell penetration via passive permeability (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). Further selectivity screening of piperazine <b>18</b> against the DiscoverX BROMOscan panel of 34 bromodomains not only confirmed the expected comparable potency between BRD9 and BRD7 but also revealed generally desirable selectivity against the BET (630-fold) and non-BET bromodomains (50-fold) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00075/suppl_file/jm0c00075_si_001.pdf" class="ext-link">Table S3</a>).<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> The notable exception to this was BRPF1 where almost equipotency was observed (BRPF1 p<i>K</i><sub>i</sub> = 6.6), resulting in only 6-fold bias.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. BROMOscan Activity and Physicochemical Profile for Compound <b>18</b><a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/medium/jm0c00075_0024.gif" alt="" id="gr24" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">parameter</th><th class="colsep0 rowsep0" align="center"><b>18</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD9 pIC<sub>50</sub></td><td class="colsep0 rowsep0" align="left">6.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD9 LE<a class="ref internalNav" href="#t3fn2" aria-label="b">b</a>/LLE<a class="ref internalNav" href="#t3fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">0.38/5.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD9 p<i>K</i><sub>i</sub></td><td class="colsep0 rowsep0" align="left">7.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD7 p<i>K</i><sub>i</sub></td><td class="colsep0 rowsep0" align="left">7.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD4(1) p<i>K</i><sub>i</sub></td><td class="colsep0 rowsep0" align="left">4.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRPF1 p<i>K</i><sub>i</sub></td><td class="colsep0 rowsep0" align="left">6.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="left">BROMOscan selectivity<a class="ref internalNav" href="#t3fn4" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="left">BET, ≥×630</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">all others, ≥×50</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ChromLog<i>D</i><sub>pH7.4</sub></td><td class="colsep0 rowsep0" align="left">1.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CLND solubility (μg/mL)</td><td class="colsep0 rowsep0" align="left">178</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AMP (nm/s)</td><td class="colsep0 rowsep0" align="left">330</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">pIC<sub>50</sub> values refer to potency within a TR-FRET assays. p<i>K</i><sub>i</sub> values refer to potency within the DiscoverX BROMOscan assay.</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last">LE = (1.37 × pIC<sub>50</sub>)/heavy atom count.</p></div><div class="footnote" id="t3fn3"><sup><sup>c</sup></sup><p class="last">LLE = pIC<sub>50</sub> – ChromLogD.</p></div><div class="footnote" id="t3fn4"><sup><sup>d</sup></sup><p class="last">Selectivity determined from the difference between potencies.</p></div></div></div><div class="NLM_p">X-ray crystallography of <b>18</b> bound to BRD9 enabled a better understanding of the interactions being made with the protein (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). Despite the pyridazinone core of <b>18</b> overlaying almost perfectly with that of compound <b>14</b> to make identical interactions with the protein, the more flexible benzylamine group shifts the phenyl ring of <b>18</b> ∼1.6 Å closer to Ile53. This movement results in a dramatic shift of the attached piperazine ring ∼3.7 Å away from the lipophilic Phe44 toward the more polar Tyr106 gatekeeper (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>b). Presumably, the more favorable dipole interactions of the weakly basic piperazine (p<i>K</i><sub>a</sub> = 8.1) and Tyr106 phenol drive this shift. Additionally, the crystallography highlighted the piperazine methyl group as an exit vector from the protein to enable linkable analogue synthesis (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>c). Confirming this hypothesis, linkable analogue <b>19</b> bearing a pendant amine retained activity against BRD9, together with 250-fold selectivity over BRD4(1) (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). The identification and demonstration of a solvent-exposed exit vector with compound <b>19</b> bearing a primary amine enable the development of bifunctional tools such as PROTACSs and attachment to a solid support for chemoproteomic experiments (vide infra).</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/medium/jm0c00075_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0007.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. X-ray crystallography of (a) compound <b>18</b> (green) bound to BRD9 (gray; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YQS">6YQS</a>), (b) overlay of compound <b>18</b> (green; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YQS">6YQS</a>) and compound <b>14</b> (yellow; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YQR">6YQR</a>) bound to BRD9 (gray), and (c) compound <b>18</b> (green) bound to BRD9 (gray; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YQS">6YQS</a>) highlighting the protein surface.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00075&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Atypical KAc Methyl Mimetics as a Path for Selectivity</h3><div class="NLM_p">With BROMOscan data demonstrating the potency of compound <b>18</b> at BRPF1, we sought to improve the selectivity against this bromodomain off-target. Attempts to deliver selectivity over BRPF1 through modification of the substituted benzyl group were unsuccessful with BRPF1 potency broadly tracking with BRD9. As an alternative approach, interactions with the conserved waters found in the bromodomain binding site were considered. The displacement of water molecules in small-molecule binding sites is a commonly used method in medicinal chemistry to produce higher-affinity ligands and impact target selectivity.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> As mentioned previously, the bromodomain binding site contains four water molecules conserved across all bromodomains adjacent to the methyl group of the KAc and has been the subject of great interest in recent years (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a> and <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>a). The relative stabilities of these water networks vary between different bromodomains and offer a route to drastically impact the bromodomain selectivity profile of small molecules. Indeed, multiple groups have estimated the stability of each bromodomain water network to help explain and predict such transformations.<a onclick="showRef(event, 'ref40 ref41'); return false;" href="javascript:void(0);" class="ref ref40 ref41">(40,41)</a> The displacement of the bromodomain conserved water network to engender selectivity has been exploited in the development of chemical probes for ATAD2,<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> TAF1(2),<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> and the subfamily VIII bromodomains.<a onclick="showRef(event, 'ref44 ref45 ref46'); return false;" href="javascript:void(0);" class="ref ref44 ref45 ref46">(44−46)</a> In direct contrast to water displacement, Crawford and co-workers at Genentech and Constellation Pharmaceuticals elegantly demonstrated that the stability of the conserved water network in the BRD9 and CECR2 bromodomains is such that hydrophobic pockets are preferentially induced by longer lipophilic KAc methyl mimetics to avoid displacing or clashing with the water network.<a onclick="showRef(event, 'ref35 ref47'); return false;" href="javascript:void(0);" class="ref ref35 ref47">(35,47)</a> This fundamental observation, which profoundly impacts the bromodomain selectivity of molecules incorporating larger lipophilic KAc methyl mimetics such as allyl and <i>E</i>-crotyl groups, was central to Genentech and Constellation Pharmaceuticals development of chemical probes for CECR2 and BRD7/9.<a onclick="showRef(event, 'ref34 ref48'); return false;" href="javascript:void(0);" class="ref ref34 ref48">(34,48)</a> Inspired by the selectivity profile of GNE-375 (<b>7</b>), attributable, in part, to the <i>E</i>-crotyl KAc methyl mimetic, it was hypothesized that atypical KAc methyl mimetics may offer a route to selectivity over BRPF1 for the pyridazinone template. Overlaying the crystal structures of <b>18</b> and <b>20</b> (an early derivative of <b>7</b>) revealed the pyridazinone C–Me group as a suitable vector for accessing the induced hydrophobic pocket (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>).<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a></div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/medium/jm0c00075_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0008.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (a) Structure of BRD9 inhibitors <b>18</b> and <b>20</b>. The methyl mimetic is colored blue. (b, c) Overlay of the crystal structures of <b>18</b> (green; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YQS">6YQS</a>) and <b>20</b> (cyan; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5I7Y">5I7Y</a>) bound to BRD9 (gray; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5I7Y">5I7Y</a>), highlighting the vector to access the induced pocket from different perspectives.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00075&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To test this design hypothesis, compound <b>22</b> bearing the same <i>E</i>-crotyl KAc methyl mimetic as compound <b>7</b> was accessed (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). To rapidly generate data across several bromodomains, compounds were screened in the BROMOscan assay (p<i>K</i><sub>i</sub> values) as opposed to internal TR-FRET assays (pIC<sub>50</sub> values) that had been used to guide the chemistry to this point.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> Validating the scaffold-hopping approach outlined in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>, a profound ≥40-fold reduction in BRPF1 potency was observed while retaining BRD9 activity with compound <b>22</b>. Interestingly, a 6-fold rise in BRD4(1) potency was also observed with the <i>E</i>-crotyl KAc methyl mimetic. To further explore the potential of hydrophobic KAc methyl mimetics on the pyridazinone scaffold, a range of unsaturated groups were accessed (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>).</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Unsaturated and Saturated Alkyl Chain KAc Methyl Mimetic BRD9, BRD4(1), and BRPF1 SAR Table<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/medium/jm0c00075_0025.gif" alt="" id="gr25" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/medium/jm0c00075_0026.gif" alt="" id="gr26" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">p<i>K</i><sub>i</sub> values refer to potency within the DiscoverX BROMOscan assay. pIC<sub>50</sub> values refer to potency within TR-FRET assays.</p></div><div class="footnote" id="t4fn2"><sup><sup>b</sup></sup><p class="last">LLE = p<i>K</i><sub>i</sub> – ChromLogD.</p></div><div class="footnote" id="t4fn3"><sup><sup>c</sup></sup><p class="last">Selectivity determined from the difference in measured biochemical potencies.</p></div><div class="footnote" id="t4fn4"><sup><sup>d</sup></sup><p class="last">Single unassigned enantiomer.</p></div></div><div></div></div><div class="NLM_p">Truncation of the crotyl group to an allyl group with compound <b>21</b> maintained the reduced BRPF1 activity while also reducing BRD9 and BRD4(1) potency. Changing the crotyl group geometric isomer from <i>E</i> to <i>Z</i> resulted in a reduction in potency against both BRD9 and BRD4(1), suggesting that the <i>Z</i> orientation cannot access the induced pocket as readily (compare compounds <b>22</b> and <b>23</b>). Moving the double bond into conjugation with the pyridazinone ring with <b>24</b> drastically reduced potency against BRD9, suggesting that the alkyl chain was no longer directed toward the induced pocket and was instead clashing with other residues within the binding site. Butenyl <b>25</b> with a terminal alkene also showed a reduction in BRD9 potency. Methyl branching of the allyl group with enantiomers <b>26</b> and <b>27</b> revealed a reduction of BRD9 activity, suggesting that increased steric bulk was not tolerated at this position.</div><div class="NLM_p">Although allyl <b>21</b> and <i>E</i>-crotyl <b>22</b> demonstrated an intriguing level of bromodomain selectivity, there was considerable concern around the risk of olefin isomerization into more thermodynamically stable conjugation with the pyridazinone ring. Such isomerization would likely lead to a large drop in BRD9 activity (compare nonconjugated <b>22</b> and <b>25</b> with conjugated <b>24</b>, <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). The potential for more flexible saturated analogues was considered to remove the isomerization risk while still accessing the induced BRD9 hydrophobic pocket.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Accordingly, saturated KAc methyl mimetic derivatives <b>28</b>–<b>32</b> were accessed and profiled. As also observed with the unsaturated KAc methyl mimetic studies (compounds <b>21</b>–<b>27</b>), at least a 25-fold reduction in BRPF1 potency was obtained across all saturated alkyl chains tested, highlighting the sensitivity of the BRPF1 bromodomain to substitution in this region (compare <b>18</b> with <b>28</b> and <b>29</b>, <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). Both ethyl <b>28</b> and propyl <b>29</b> profoundly reduced BRPF1 potency while showing a small decrease in BRD9 activity. Introduction of branching at the linker carbon in <b>30</b> remained detrimental to the BRD9 potency, confirming the poor tolerance for steric bulk at this position, as seen with compounds <b>26</b> and <b>27</b>. Moving from a three-carbon chain to a four-carbon chain with <i>n</i>-butyl <b>31</b> showed an increase in BRD9 potency (p<i>K</i><sub>i</sub> = 7.2), as observed for the unsaturated chains. Although <b>31</b> was more lipophilic than most other analogues, the compound remained inactive at BRD4(1) at the concentrations tested, maintaining high BET bromodomain selectivity (>×500). Extending the chain further with <i>n</i>-pentyl <b>32</b> resulted in a drop in BRD9 potency, together with an unexpected rise of BRD4(1) activity, suggesting that four-atom chains are optimal for occupying the induced BRD9 hydrophobic pocket. Complete removal of the <i>n</i>-butyl group with compound <b>33</b> ablated activity against BRD9 and BRD4(1), providing further evidence of the critical importance of the KAc methyl mimetic to potency against these bromodomains.</div><div class="NLM_p">On the assumption that molecules with lipophilic KAc mimetics were inducing the same hydrophobic pocket in BRD9, as described by Crawford and co-workers,<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> compounds <b>21</b>–<b>32</b> were docked into a crystal structure of the BRD9 protein with the induced hydrophobic pocket derived from <b>20</b> bound to BRD9 (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5I7Y">5I7Y</a>, <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>). Consistent with the biochemical data, the <i>E</i>-crotyl chain of <b>22</b> showed optimal occupancy of the induced pocket (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>b), whereas reducing the carbon chain length resulted in partial occupancy of the pocket with <b>21</b> (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>a). Alteration of the double bond geometry (compound <b>23</b>) and positioning (compounds <b>24</b> and <b>25</b>) appeared detrimental to binding, forcing the molecules to twist to accommodate the chain conformations (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>c–e). Similarly, branching at the adjoining carbon with <b>27</b> caused a shift of the core to accommodate the additional steric demand (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>f). Docking the saturated analogues revealed a similar pattern with the four-atom <i>n</i>-butyl chain <b>31</b> appearing optimal for occupying the induced pocket (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>k,l). Shorter chains (<b>28</b> and <b>29</b>) again showed only partial occupancy of the induced pocket (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>g,h), and branching with isopropyl <b>30</b> appeared to cause the molecule to twist within the binding site (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>i). The larger pentyl chain of <b>32</b> was also poorly accommodated as expected (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>j).</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/medium/jm0c00075_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0009.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Docking of pyridazinone analogues bearing different KAc methyl mimetics (a) allyl <b>21</b>, (b) <i>E</i>-crotyl <b>22</b>, (c) <i>Z</i>-crotyl <b>23</b>, (d) 1,2-butenyl <b>24</b>, (e) 3,4-butenyl <b>25</b>, (f) <i>sec</i>-butenyl <b>27</b>, (g) ethyl <b>28</b>, (h) propyl <b>29</b>, (i) isopropyl <b>30</b>, (j) pentyl <b>32</b>, (k) <i>n</i>-butyl <b>31</b>, and (l) <i>n</i>-butyl <b>31</b> space filled into the BRD9 bromodomain (gray) using Glide (Schrodinger Inc.) 2018.3 with hydrogen-bonding constraints between docked ligands and both the W1 conserved water molecule and the carbonyl group of Phe44. Tyr106 has been removed from the visualization to help show the induced pocket. PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5I7Y">5I7Y</a> was used for all dockings.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00075&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">A comparison of matched molecular pair profiles of methyl, <i>E</i>-crotyl, and <i>n</i>-butyl KAc methyl mimetics <b>18</b>, <b>22</b>, and <b>31</b> highlighted some key differences (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>). As discussed above, <b>18</b> shows substantial activity for BRPF1 in the BROMOscan assay, which is removed when moving to longer lipophilic chains <b>22</b> and <b>31</b>. The more flexible <i>n</i>-butyl chain drives reduced BRD4(1) potency and, as a result, improved BRD4(1) selectivity compared to the <i>E</i>-crotyl chain. Additionally, inhibition of TAF1(2) was seen at 10 μM for <i>E</i>-crotyl <b>22</b>, which was ablated with <i>n</i>-butyl <b>31</b>. Although there is an expected increase in lipophilicity upon addition of the <i>n</i>-butyl chain, together with a concomitant decrease in LLE compared to the other KAc methyl mimetics profiled, the improvement in bromodomain selectivity warranted further profiling of compound <b>31</b>.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Comparison of Methyl Mimetic SAR<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/medium/jm0c00075_0027.gif" alt="" id="gr27" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/medium/jm0c00075_0028.gif" alt="" id="gr28" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">p<i>K</i><sub>i</sub> and %<i>I</i> values refer to potency within the DiscoverX BROMOscan assay.</p></div><div class="footnote" id="t5fn2"><sup><sup>b</sup></sup><p class="last">LLE = p<i>K</i><sub>i</sub> – ChromLogD.</p></div><div class="footnote" id="t5fn3"><sup><sup>c</sup></sup><p class="last">Selectivity determined from the difference in measured biochemical potencies.</p></div></div><div></div></div><div class="NLM_p">The selectivity of <b>31</b> against the wider bromodomain family was determined using the DiscoverX BROMOscan panel (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.0c00075/suppl_file/jm0c00075_si_001.pdf" class="ext-link">Table S5</a>).<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> Excluding high potency against the highly homologous BRD7 (p<i>K</i><sub>i</sub> = 6.3), excellent selectivity against the BET (>×500) and non-BET bromodomains (≥×280) was observed. Compound <b>31</b> was also screened against an internal panel of 40 biological off-targets showing pXC<sub>50</sub> ≤ 5.0, excluding G protein-coupled receptor MrgX2 (pEC<sub>50</sub> = 5.3) and serotonin 1B (pEC<sub>50</sub> = 5.7) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00075/suppl_file/jm0c00075_si_001.pdf" class="ext-link">Table S6</a>). Compound <b>31</b> was soluble (≥228 μg/mL) in a charged-aerosol detection (CAD) assay<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> and permeable (250 nm/s) in an artificial membrane permeability assay, indicating likely cell permeability by passive diffusion. Supporting this hypothesis, the concentration of compound <b>31</b> outside and inside HeLa cells was monitored using an intracellular concentration assay.<a onclick="showRef(event, 'ref50 ref51'); return false;" href="javascript:void(0);" class="ref ref50 ref51">(50,51)</a> The cellular influx/efflux ratio was measured (pΔ<i>C</i> = 0.54) and demonstrated good cellular permeability with slight accumulation within the cell (<a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>).</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/medium/jm0c00075_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0010.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. DiscoverX BROMOscan activity profile for compound <b>31</b> screened against 40 bromodomains.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00075&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. Summary of Properties for <b>31</b> and <b>33</b><a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/medium/jm0c00075_0029.gif" alt="" id="gr29" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">parameter</th><th class="colsep0 rowsep0" align="center"><b>31</b></th><th class="colsep0 rowsep0" align="center"><b>33</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD9 p<i>K</i><sub>i</sub></td><td class="colsep0 rowsep0" align="left">7.2</td><td class="colsep0 rowsep0" align="left"><4.3 (pIC<sub>50</sub>)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD9 LE<a class="ref internalNav" href="#t6fn2" aria-label="b">b</a>/LLE<a class="ref internalNav" href="#t6fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">0.37/3.7</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD4(1) p<i>K</i><sub>i</sub></td><td class="colsep0 rowsep0" align="left"><4.5</td><td class="colsep0 rowsep0" align="left"><4.3 (pIC<sub>50</sub>)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD7 p<i>K</i><sub>i</sub></td><td class="colsep0 rowsep0" align="left">6.3</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="left">BROMOscan selectivity<a class="ref internalNav" href="#t6fn4" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="left">BET, >×500</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">all others, ≥×280</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ChromLog<i>D</i><sub>pH7.4</sub>/cLogP</td><td class="colsep0 rowsep0" align="left">3.5/3.2</td><td class="colsep0 rowsep0" align="left">1.1/1.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CAD solubility (μg/mL)</td><td class="colsep0 rowsep0" align="left">≥228</td><td class="colsep0 rowsep0" align="left">≥150</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AMP (nm/s)</td><td class="colsep0 rowsep0" align="left">250</td><td class="colsep0 rowsep0" align="left">140</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">intracellular influx/efflux ratio (pΔ<i>C</i>)</td><td class="colsep0 rowsep0" align="left">0.52</td><td class="colsep0 rowsep0" align="left"> </td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup><sup>a</sup></sup><p class="last">pIC<sub>50</sub> values refer to potency within TR-FRET assays. p<i>K</i><sub>i</sub> values refer to potency within the DiscoverX BROMOscan assay.</p></div><div class="footnote" id="t6fn2"><sup><sup>b</sup></sup><p class="last">LE = (1.37 × p<i>K</i><sub>i</sub>)/heavy atom count.</p></div><div class="footnote" id="t6fn3"><sup><sup>c</sup></sup><p class="last">LLE = pIC<sub>50</sub> – ChromLogD.</p></div><div class="footnote" id="t6fn4"><sup><sup>d</sup></sup><p class="last">Selectivity determined from the difference between potencies.</p></div></div></div><div class="NLM_p last">The combination of <b>31</b> and <b>33</b> as inhibitor and negative control, respectively, represent two potential additions to the small-molecule BRD7/9 target validation toolbox. However, assays to demonstrate cellular target engagement with compound <b>31</b> were not available within GlaxoSmithKline, and the data to support claiming compound <b>31</b> as a chemical probe for the BRD7/9 bromodomains are unavailable. It is of note that the profile outlined in <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a> is consistent with a soluble, cell-permeable compound, but the lack of BRD7/9 cellular target engagement data is an important caveat that should be acknowledged and considered before any use.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Application of <i>n</i>-Butyl KAc Methyl Mimetic across Multiple Chemotypes</h3><div class="NLM_p">Having demonstrated the possibility of bromodomain selectivity through the use of an <i>n</i>-butyl KAc methyl mimetic in the development of <b>31</b>, the broader applicability of this group was considered. Three molecules from different scaffolds were selected to test the utility and translatability of this atypical KAc methyl mimetic: (1) compound <b>34</b>,<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> an unselective compound from the pyridazinone template; (2) iBRD9 (<b>2</b>),<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> a BRD9 selective compound with a pyridinone KAc mimetic; and (3) bromosporine (<b>37</b>),<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> a pan-bromodomain inhibitor with a triazole KAc mimetic (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>). Published X-ray crystallographic data were used to identify the KAc methyl mimetic and appropriate vector to introduce the <i>n</i>-butyl group.</div><figure id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/medium/jm0c00075_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0011.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Three templates selected to test the broader applicability of the <i>n</i>-butyl KAc methyl mimetic methodology and their corresponding <i>n</i>-butyl derivatives. Full data shown in <a href="/doi/suppl/10.1021/acs.jmedchem.0c00075/suppl_file/jm0c00075_si_001.pdf" class="ext-link">Table S8</a>. Data for compound <b>2</b> can be found in ref <a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00075&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Pyridazinone <b>34</b> was identified during work developing PCAF/GCN5 inhibitors as a promiscuous pan-bromodomain inhibitor: screening at 10 μM concentration against 32 bromodomains revealed 10 bromodomains with >95% inhibition.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> The <i>n</i>-butyl analogue <b>35</b> showed far improved selectivity for BRD7/9 at 10 μM across the same panel of 32 bromodomains and retained activity against BRD9 (p<i>K</i><sub>i</sub> = 7.1) (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>b and <a href="/doi/suppl/10.1021/acs.jmedchem.0c00075/suppl_file/jm0c00075_si_001.pdf" class="ext-link">Table S7</a>).<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> Similarly, <b>36</b>, the <i>n</i>-butyl derivative of iBRD9 (<b>2</b>), showed improved selectivity at 10 μM (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>d and <a href="/doi/suppl/10.1021/acs.jmedchem.0c00075/suppl_file/jm0c00075_si_001.pdf" class="ext-link">Table S7</a>) and an overall increase in non-BET bromodomain selectivity (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00075/suppl_file/jm0c00075_si_001.pdf" class="ext-link">Table S8</a>). Interestingly, the CECR2 potency dropped upon moving from ethyl to <i>n</i>-butyl (CECR2 p<i>K</i><sub>i</sub> of <b>2</b> = 6.9 to p<i>K</i><sub>i</sub> of <b>36</b> = 5.6), whereas the TAF1(2) potency increased (TAF1(2) p<i>K</i><sub>i</sub> of <b>2</b> = 5.1 to p<i>K</i><sub>i</sub> of <b>36</b> = 5.7). However, the overall selectivity to the closest non-BET bromodomain increased from 70- to 200-fold. Bromosporine (<b>37</b>) is a highly promiscuous bromodomain inhibitor as demonstrated by 19 bromodomains showing ≥95% inhibition at 10 μM in the bromodomain panel (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>e). In comparison, at 10 μM, <i>n</i>-butyl analogue <b>38</b> only showed a single bromodomain (BRD9) at ≥95% inhibition (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>f), highlighting the dramatic shift in the selectivity profile possible across multiple chemotypes with different KAc mimetics.</div><figure id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/medium/jm0c00075_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0002.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. DiscoverX BROMOscan profiling at 10 μM for (a) compound <b>34</b>, (b) compound <b>35</b>, (c) iBRD9 (<b>2</b>), (d) compound <b>36</b>, (e) bromosporine (<b>37</b>), and (f) compound <b>38</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00075&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Investigation of the PBAF and BAF Complexes Using Chemoproteomics</h3><div class="NLM_p">Affinity enrichment chemoproteomics is a powerful method to not only identify small molecule protein targets but also to interrogate endogenous protein complexes.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> Linkable pyridazinone analogue <b>19</b> bearing a solvent-exposed primary amine facilitated such experiments and allowed interrogation of the endogenous BRD7- and BRD9-associated chromatin remodeling PBAF and BAF complexes, which have been explored via proteomic and bioinformatic approaches previously. Nuclear and chromatin lysates from Hut-78 cells were incubated with Sepharose-immobilized <b>19</b> as an affinity resin and either 10 μM compound <b>18</b> as the binding competitor or DMSO as a control. Following washing, the residual proteins still bound on the affinity resin were eluted, identified, and quantified by mass spectrometry (<a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.0c00075/suppl_file/jm0c00075_si_001.pdf" class="ext-link">Table S10</a>). The data were plotted as a Log<sub>2</sub> fold change (FC) versus the DMSO-only control, and those proteins specifically competed by compound <b>18</b> in both replicates are labeled in <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>. Together with the expected BRD7 and BRD9 bromodomain containing proteins, 17 other proteins were identified, all of which are reported members of the PBAF or BAF complexes.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Of note is the enrichment of BCL7A and BCL7B together with BRD9, which supports previous proteomic studies indicating that these are dedicated, nonexchangeable subunits of the BAF complex.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Bromodomain-containing proteins PBRM1, SMARCA2, and SMARCA4 were also identified as being specifically competed in this experiment. However, the low affinity of compound <b>18</b> to these bromodomain-containing proteins as judged by BROMOscan (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00075/suppl_file/jm0c00075_si_001.pdf" class="ext-link">Table S3</a>) further supports evidence that these proteins are enriched in the affinity-purification experiment because they are components of the PBAF and BAF complexes.</div><figure id="fig11" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/medium/jm0c00075_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0003.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Affinity enrichment chemoproteomic experiments in Hut-78 nuclear and chromatin lysate.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00075&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Chemistry: Synthesis of Compounds</h3><div class="NLM_p">Pyridazinones <b>12</b> and <b>13</b> were synthesized from commercially available 4,5-dichloro-2-methylpyridazin-3(2<i>H</i>)-one (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>).<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> S<sub>N</sub>Ar with the corresponding amine occurred preferentially at the 5-position, yielding compounds <b>12</b> and <b>13</b>.</div><figure id="sch1" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/medium/jm0c00075_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0012.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 1. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00075&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Amine, DMSO, 120 °C, 1 h, 23–75%</p></p></figure><div class="NLM_p">Compounds <b>10</b> and <b>11</b> were synthesized from commercially available <b>39</b> (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). Deprotonation followed by the S<sub>N</sub>Ar reaction with excess 3.4 M MeMgBr in THF gave 4-methyl pyridazinone <b>40</b>. Subsequent methylation with MeI gave key intermediate <b>41</b>, which was reacted under a variety of Buchwald–Hartwig amination conditions using different Pd precatalysts and ligands with the requisite amine to give compounds <b>10</b>, <b>11</b>, <b>14</b>–<b>16</b>, and <b>18</b>.</div><figure id="sch2" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/medium/jm0c00075_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0013.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 2. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00075&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 3.4 M MeMgBr, THF, rt, 5.5 h, 81%; (b) MeI, TBAB, K<sub>2</sub>CO<sub>3</sub>, MeCN, rt to 60 °C, 4 h, 55%; (c) Pd-Catalyzed Buchwald–Hartwig Amination, NaO<i><sup>t</sup></i>Bu or KO<i>t</i>-Bu, 1,4-Dioxane, 100–120 °C, 30 min to 1 h, 13–61%</p></p></figure><div class="NLM_p">Similarly, compound <b>17</b> was synthesized via Buchwald–Hartwig amination followed by acid-mediated Boc deprotection to unveil the target compound (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>). Cesium carbonate-mediated alkylation of <b>17</b> was then followed by Boc deprotection to yield linkable analogue <b>19</b>.</div><figure id="sch3" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/medium/jm0c00075_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0014.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 3. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00075&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <i>tert</i>-Butyl 4-(2-(aminomethyl)phenyl)piperazine-1-carboxylate, BrettPhos-Pd-G1, BrettPhos, NaO<i><sup>t</sup></i>Bu, 1,4-Dioxane, Microwave 120 °C, 30 min; (b) 4 M HCl, 1,4-Dioxane, DCM, rt, 18 h, 10% over Two Steps; (c) BocNHCH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>Br, Cs<sub>2</sub>CO<sub>3</sub>, DMF, rt to 50 °C, 36 h, 20%; (d) HCl, 1,4-Dioxane, rt, 18 h, 18%</p></p></figure><div class="NLM_p">The synthesis of compounds <b>21–24</b>, <b>26</b>, <b>27</b>, <b>29</b>, <b>32</b>, and <b>33</b> began from commercially available 4,5-dichloro-2-methylpyridazin-3(2<i>H</i>)-one (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>). S<sub>N</sub>Ar with (2-(4-methylpiperazin-1-yl)phenyl)methanamine occurred preferentially at the 5-position to yield pivotal intermediate <b>42</b>. Standard Suzuki cross-coupling introduced the desired alkenyl or alkyl chain to give <b>21</b>, <b>24</b>, <b>29</b>, and <b>32</b>. Cross-coupling of <b>42</b> with (<i>E</i>)-2-(but-2-en-1-yl)-4,4,5,5-tetramethyl-1,3,2-dioxaborolane yielded a mixture of <i>Z</i>- and <i>E</i>-crotyl isomers (<b>22</b> and <b>23</b>) and <i>sec</i>-butenyl enantiomers <b>26</b> and <b>27</b>, which were separated into a pair of unassigned enantiomers via chiral chromatography. Compound <b>33</b> was observed as the major by-product of the Suzuki reactions formed via proto-dechlorination of the starting material <b>42</b>. This by-product was particularly prevalent in the attempted Suzuki reaction with potassium butyltrifluoroborate and was isolated in 43% yield.</div><figure id="sch4" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/medium/jm0c00075_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0015.jpeg" id="gr15" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 4. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00075&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (2-(4-Methylpiperazin-1-yl)phenyl)methanamine, DIPEA, DMSO, 120 °C, 1 h, 63%; (b) Allyl-Bpin, RuPhos Pd G2, K<sub>2</sub>CO<sub>3</sub>, 1,4-Dioxane/H<sub>2</sub>O, 100 °C, 1 h, 28%; (c) (<i>E</i>)-But-1-en-1-ylboronic Acid, Pd(OAc)<sub>2</sub>, CataCXium A, K<sub>2</sub>CO<sub>3</sub>, 1,4-Dioxane/H<sub>2</sub>O, 100 °C, 5 h, 47%; (d) (<i>E</i>)-2-(But-2-en-1-yl)-Bpin, Pd(OAc)<sub>2</sub>, CataCXium A, K<sub>2</sub>CO<sub>3</sub>, 1,4-Dioxane/H<sub>2</sub>O, 100 °C, 1.5 h, 30% Combined Yield; (e) Propylboronic Acid, Pd(OAc)<sub>2</sub>, CataCXium A, K<sub>2</sub>CO<sub>3</sub>, 1,4-Dioxane/H<sub>2</sub>O, 100 °C, 1 h, 29%; (f) Pentylboronic Acid, Pd(OAc)<sub>2</sub>, CataCXium A, K<sub>2</sub>CO<sub>3</sub>, 1,4-Dioxane/H<sub>2</sub>O, 100 °C, 1 h, 25%; (g) Potassium Butyltrifluoroborate, Pd(OAc)<sub>2</sub>,CataCXium A, K<sub>2</sub>CO<sub>3</sub>, 1,4-Dioxane/H<sub>2</sub>O, 100 °C, 1 h, 43%</p></p></figure><div class="NLM_p">Terminal alkene <b>25</b> was synthesized from key intermediate <b>41</b> (<a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>). Deprotonation of the ene-imine γ-proton with LiHMDS and alkylation with allyl bromine appended the butenyl chain to give <b>43</b>. Standard Buchwald–Hartwig amination with the substituted benzylamine gave the target compound <b>25</b>.</div><figure id="sch5" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/medium/jm0c00075_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0016.jpeg" id="gr16" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 5. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00075&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Allylbromide, LiHMDS, THF, −78 °C, 45 min, 24%; (b) (2-(4-Methylpiperazin-1-yl)phenyl)methanamine, Pd<sub>2</sub>(dba)<sub>3</sub>, BrettPhos, NaO<i><sup>t</sup></i>Bu, THF, 100 °C, 1 h, 20%</p></p></figure><div class="NLM_p">Ethyl and isopropyl derivatives were synthesized from commercially available <b>39</b> in a similar manner to that described in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. A chemoselective S<sub>N</sub>Ar with ethyl or isopropyl Grignard reagents yielded intermediates <b>44</b> and <b>46</b>, respectively (<a class="ref internalNav" href="#sch6" aria-label="Scheme 6">Scheme 6</a>). Straightforward alkylation with MeI of both intermediates was followed by a Buchwald–Hartwig reaction to give compounds <b>28</b> and <b>30</b>.</div><figure id="sch6" data-index="17" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/medium/jm0c00075_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0017.jpeg" id="gr17" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 6. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00075&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) THF, 0 °C to rt, 1.5–3 h, 26–27%; (b) MeI, TBAB, K<sub>2</sub>CO<sub>3</sub>, MeCN, rt, 2.5–3 h, 19–63%; (c) (2-(4-Methylpiperazin-1-yl)phenyl)methanamine, Pd<sub>2</sub>(dba)<sub>3</sub>, BrettPhos, NaO<i><sup>t</sup></i>Bu, THF, 100 °C, 1 h, 36–46%</p></p></figure><div class="NLM_p">Butyl derivatives <b>31</b> and <b>35</b> were synthesized from commercially available <b>39</b> in a similar fashion (<a class="ref internalNav" href="#sch7" aria-label="Scheme 7">Scheme 7</a>). A chemoselective S<sub>N</sub>Ar with 1.6 M <i>n</i>-BuLi in hexanes was followed by NH methylation under standard conditions to give intermediate <b>49</b>. A Buchwald–Hartwig reaction with the appropriate amine gave target <b>31</b>. Alternatively, Buchwald–Hartwig coupling with <i>tert</i>-butyl 5-amino-3,4-dihydroisoquinoline-2(1<i>H</i>)-carboxylate was then followed by Boc deprotection under standard acidic conditions and then methylation via an Eschweiler–Clarke reaction to give <b>35</b>.</div><figure id="sch7" data-index="18" class="article__inlineFigure"><h2 class="fig-label">Scheme 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/medium/jm0c00075_0018.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0018.jpeg" id="gr18" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 7. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00075&amp;id=sch7"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 1.6 M <i>n</i>-BuLi, THF, −78 °C, 15 min, 63%; (b) MeI, TBAB, K<sub>2</sub>CO<sub>3</sub>, MeCN, 60 °C, 5 h, 77%; (c) (2-(4-Methylpiperazin-1-yl)phenyl)methanamine, Pd<sub>2</sub>(dba)<sub>3</sub>, BrettPhos, NaO<i><sup>t</sup></i>Bu, THF, 100 °C, 1 h, 22%; (d) <i>tert</i>-Butyl 5-Amino-3,4-dihydroisoquinoline-2(1<i>H</i>)-carboxylate, Pd<sub>2</sub>(dba)<sub>3</sub>, BrettPhos, NaO<i><sup>t</sup></i>Bu, THF, Microwave 100 °C, 1 h, 25%; (e) 4 M HCl, 1,4-Dioxane, rt, 45 min, Quant.; (f) 37 wt % Formaldehyde, Formic Acid, 80 °C, 6 h, 59%</p></p></figure><div class="NLM_p">Synthesis of <b>36</b> began from commercially available thienopyridinone <b>50</b> in a similar fashion to that described previously (<a class="ref internalNav" href="#sch8" aria-label="Scheme 8">Scheme 8</a>).<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Alkylation with 1-iodobutane followed by Negishi cross-coupling with zinc cyanide gave intermediate <b>51</b>. Bromination with NBS followed by a Pinner reaction produced intermediate <b>52</b>. The reaction with the required amine was followed by Suzuki cross-coupling to append the aryl ring and deliver target <b>36</b>.</div><figure id="sch8" data-index="19" class="article__inlineFigure"><h2 class="fig-label">Scheme 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/medium/jm0c00075_0019.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0019.jpeg" id="gr19" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 8. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00075&amp;id=sch8"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 1-Iodobutane, Cs<sub>2</sub>CO<sub>3</sub>, THF, 60 °C, 19 h, 71%; (b) Zn(CN)<sub>2</sub>, Pd(PPh<sub>3</sub>)<sub>4</sub>, DMF, 115 °C, 4.5 h, 61%; (c) NBS, THF, rt, 65 h, 86%; (d) 4-Aminotetrahydro-2<i>H</i>-thiopyran 1,1-Dioxide, 25 wt % NaOMe, MeOH, 75 °C, 26 h, 27%; (e) 4-Aminotetrahydro-2<i>H</i>-thiopyran 1,1-Dioxide, Triethylamine, DMF, 120 °C, 17 h, 21%; (f) (3-(Trifluoromethyl)phenyl)boronic Acid, PEPPSI-<sup>i</sup>Pr, K<sub>2</sub>CO<sub>3</sub>, IPA/H<sub>2</sub>O, 120 °C, 30 min, 52%</p></p></figure><div class="NLM_p">Synthesis of compound <b>38</b> began from commercially available dichloro pyridazine <b>53</b> in a similar fashion to that reported elsewhere (<a class="ref internalNav" href="#sch9" aria-label="Scheme 9">Scheme 9</a>).<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> S<sub>N</sub>Ar with hydrazine hydrate and subsequent cyclization with pentanoic acid gave butyl intermediate <b>54</b>. The reaction with ethyl chloroformate was followed by Suzuki-cross coupling to give intermediate <b>55</b>. Then, reduction of the nitro group with powdered iron in acetic acid was followed by amine mesylation to deliver target compound <b>38</b>.</div><figure id="sch9" data-index="20" class="article__inlineFigure"><h2 class="fig-label">Scheme 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/medium/jm0c00075_0020.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0020.jpeg" id="gr20" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 9. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00075&amp;id=sch9"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (i) Hydrazine Hydrate, 135 °C, 30 min; (ii) Pentanoic Acid, 100 °C, 3 h, 43% over Two Steps; (b) Ethylchloroformate, NMM, DCM, rt, 5 min, 67%; (c) (4-Methyl-3-nitrophenyl)boronic Acid, PdCl<sub>2</sub>(dppf), NaCO<sub>3</sub>, PhMe/EtOH, 100 °C, 5 h, 38%; (d) Fe, AcOH, EtOH/H<sub>2</sub>O, 80 °C, 2 h, 50%; (e) MsCl, Pyridine, DCM, rt, 3 h, 74%</p></p></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_63786" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_63786" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In conclusion, we report the development and utility of the atypical <i>n</i>-butyl KAc methyl mimetic as a straightforward approach to identifying selective BRD7/9 small-molecule inhibitors. The <i>n</i>-butyl group was identified during structure-guided optimization efforts that began from an unselective pyridazinone template and enabled delivery of <b>31</b> as a soluble, permeable, and selective BRD7/9 inhibitor. The atypical KAc methyl mimetic was then applied across three diverse chemotypes to enhance the bromodomain selectivity of an existing BRD9 inhibitor and to transform pan-bromodomain inhibitors into BRD7/9 selective compounds. Additionally, affinity enrichment chemoproteomic experiments enabled with linkable pyridazinone analogue <b>19</b> were used to interrogate the endogenous BRD7 and BRD9 protein complexes in Hut-78 cells. These experiments demonstrated specific enrichment of 19 proteins, all of which are reported to be members of the chromatin remodeling PBAF and BAF protein complexes.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i17">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_63488" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_63488" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Physicochemical Properties</h3><div class="NLM_p last">Artificial membrane permeability, chromatographicLog<i>D</i> at pH 7.4, CLND, and CAD aqueous solubility were measured using published protocols.<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Chemistry Methods</h3><div class="NLM_p">All solvents were purchased from Sigma Aldrich (anhydrous solvents) and commercially available reagents were used as received. All reactions were followed by TLC analysis (TLC plates GF254, Merck) or LCMS (liquid chromatography mass spectrometry) using a Waters ZQ instrument. The NMR spectra were recorded at ambient temperature unless otherwise stated using standard pulse methods on any of the following spectrometers and signal frequencies: Bruker AV-400 (<sup>1</sup>H = 400 MHz, <sup>13</sup>C = 100.6 MHz), Bruker AV-500 (<sup>1</sup>H = 500 MHz, <sup>13</sup>C = 125.8 MHz), and Bruker AVII+ 600 (<sup>1</sup>H = 600 MHz, <sup>13</sup>C = 150.9 MHz). Chemical shifts are reported in ppm and referenced to tetramethylsilane (TMS) or the following solvent peaks: CDCl<sub>3</sub> (<sup>1</sup>H = 7.27 ppm, <sup>13</sup>C = 77.00 ppm), DMSO-<i>d</i><sub>6</sub> (<sup>1</sup>H = 2.50 ppm, <sup>13</sup>C = 39.51 ppm), and CD<sub>3</sub>OD (<sup>1</sup>H = 3.31 ppm, <sup>13</sup>C = 49.15 ppm). Coupling constants are quoted to the nearest 0.1 Hz and multiplicities are given by the following abbreviations and combinations thereof: s (singlet), d (doublet), t (triplet), q (quartet), m (multiplet), and br (broad). Column chromatography was performed on prepacked silica gel columns using Biotage SP4, Isolera One, or Teledyne ISCO apparatus. The high-resolution mass spectra (HRMS) were recorded on a Micromass Q-Tof Ultima hybrid quadrupole time-of-flight mass spectrometer, with analytes separated on an Agilent 1100 Liquid Chromatograph equipped with a Phenomenex Luna C18(2) reversed-phase column (100 mm × 2.1 mm, 3 μm packing diameter). LC conditions were 0.5 mL/min flow rate, 35 °C, and injection volume of 2–5 μL. Gradient elution was composed of (A) H<sub>2</sub>O containing 0.1% (v/v) formic acid and (B) acetonitrile containing 0.1% (v/v) formic acid. Gradient conditions were initially 5% B, increasing linearly to 100% B over 6 min, remaining at 100% B for 2.5 min, and then decreasing linearly to 5% B over 1 min followed by an equilibration period of 2.5 min prior to the next injection. LCMS analysis was carried out on a H<sub>2</sub>Os Acquity UPLC instrument equipped with a BEH column (50 mm × 2.1 mm, 1.7 μm packing diameter) and H<sub>2</sub>Os micromass ZQ MS using alternate-scan positive and negative electrospray. Analytes were detected as a summed UV wavelength of 210–350 nm. Three liquid-phase methods were used:<ul class="NLM_list-list_type-bullet"><li><p class="inline">Formic acid, 40 °C, 1 mL/min flow rate. Gradient elution with the mobile phases as (A) H<sub>2</sub>O containing 0.1% (v/v) formic acid and (B) acetonitrile containing 0.1% (v/v) formic acid. Gradient conditions were initially 1% B, increasing linearly to 97% B over 1.5 min, remaining at 97% B for 0.4 min, and then increasing to 100% B over 0.1 min.</p></li><li><p class="inline">High pH, 40 °C, 1 mL/min flow rate. Gradient elution with the mobile phases as (A) 10 mM aqueous ammonium bicarbonate solution, adjusted to pH 10 with 0.88 M aqueous ammonia and (B) acetonitrile. Gradient conditions were initially 1% B, increasing linearly to 97% B over 1.5 min, remaining at 97% B for 0.4 min, and then increasing to 100% B over 0.1 min.</p></li><li><p class="inline">TFA, 40 °C, 1 mL/min flow rate. Gradient elution with the mobile phases as (A) H<sub>2</sub>O containing 0.1% (v/v) TFA and (B) acetonitrile containing 0.1% (v/v) TFA. Gradient conditions were initially 1% B, increasing linearly to 97% B over 1.5 min, remaining at 97% B for 0.4 min, and then increasing to 100% B over 0.1 min.</p></li></ul></div><div class="NLM_p">Mass-directed automatic purification (MDAP) was carried out using a H<sub>2</sub>Os ZQ MS using alternate-scan positive and negative electrospray and a summed UV wavelength of 210–350 nm. Two liquid-phase methods were used:<ul class="NLM_list-list_type-bullet"><li><p class="inline">Formic acid, Sunfire C18 column (100 mm x 19 mm, 5 μm packing diameter, 20 mL/min flow rate) or Sunfire C18 column (150 mm x 30 mm, 5 μm packing diameter, 40 mL/min flow rate). Gradient elution at ambient temperature with the mobile phases as (A) H<sub>2</sub>O containing 0.1% (v/v) formic acid and (B) acetonitrile containing 0.1% (v/v) formic acid.</p></li><li><p class="inline">High pH, Xbridge C18 column (100 mm × 19 mm, 5 μm packing diameter, 20 mL/min flow rate) or Xbridge C18 column (150 mm × 30 mm, 5 μm packing diameter, 40 mL/min flow rate). Gradient elution at ambient temperature with the mobile phases as (A) 10 mM aqueous ammonium bicarbonate solution, adjusted to pH 10 with 0.88 M aqueous ammonia, and (B) acetonitrile.</p></li></ul></div><div class="NLM_p">Melting point analysis was carried out using a Buchi M-565 melting point apparatus. Melting points are uncorrected. The IR spectra were obtained on a PerkinElmer Spectrum 1 FTIR apparatus, with major peaks reported. The purity of all compounds tested was determined by LCMS and <sup>1</sup>H NMR to be >95%.</div><div class="NLM_p last">The synthesis of compounds <b>8</b> and <b>9</b> has been described previously.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> 4-Chloro-2-methyl-5-((3-methylbenzyl)amino)pyridazin-3(2<i>H</i>)-one (<b>12</b>)</h3><div class="NLM_p last"><i>m</i>-Tolylmethanamine (0.11 mL, 0.84 mmol) and 4,5-dichloro-2-methylpyridazin-3(2H)-one (100 mg, 0.56 mmol) were dissolved in DMSO (1.5 mL) at rt. The reaction mixture was then heated to 120 °C in a microwave reactor for 1 h. The reaction mixture was allowed to cool down to rt before being diluted with DMSO (0.5 mL) and purified by MDAP (formic acid). The desired fractions were combined and concentrated <i>in vacuo</i>, yielding 4-chloro-2-methyl-5-((3-methylbenzyl)amino)pyridazin-3(2<i>H</i>)-one (34 mg, 0.13 mmol, 23%) as a white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm 7.53 (s, 1H), 7.33–7.29 (m, 1H), 7.16 (d, <i>J</i> = 7.6 Hz, 1H), 7.14–7.09 (m, 2H), 4.52 (d, <i>J</i> = 5.8 Hz, 2H), 3.77 (s, 3H), 2.39 (s, 3H) (N.B., exchangeable amine proton not visible); LC/MS (formic acid): <i>R</i><sub>t</sub> = 0.88 min (100%); [M + H]<sup>+</sup> = 264.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> 4-Chloro-2-methyl-5-((4-methylbenzyl)amino)pyridazin-3(2<i>H</i>)-one (<b>13</b>)</h3><div class="NLM_p last"><i>p</i>-Tolylmethanamine (0.18 mL, 1.40 mmol) and 4,5-dichloro-2-methylpyridazin-3(2<i>H</i>)-one (100 mg, 0.56 mmol) were dissolved in DMSO (1.5 mL) at rt. The reaction mixture was then heated to 120 °C in a microwave reactor for 1 h. The reaction mixture was allowed to cool down to rt before being diluted with DMSO (0.5 mL) and purified by MDAP (formic acid). The desired fractions were combined and concentrated <i>in vacuo</i>, yielding 4-chloro-2-methyl-5-((4-methylbenzyl)amino)pyridazin-3(2<i>H</i>)-one (110 mg, 0.42 mmol, 75%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 7.67 (s, 1H), 7.27 (t, <i>J</i> = 6.6 Hz, 1H), 7.21 (d, <i>J</i> = 8.1 Hz, 2H), 7.15 (d, <i>J</i> = 8.1 Hz, 2H), 4.52 (d, <i>J</i> = 6.6 Hz, 2H), 3.54 (s, 3H), 2.27 (s, 3H); LC/MS (formic acid): <i>R</i><sub>t</sub> = 0.89 min (100%); [M + H]<sup>+</sup> = 264.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> 5-Chloro-4-methylpyridazin-3(2<i>H</i>)-one (<b>40</b>)</h3><div class="NLM_p last">Methylmagnesium bromide (3.4 M) in THF solution (13.40 mL, 45.50 mmol) was added dropwise to a stirred suspension of 4,5-dichloropyridazin-3(2<i>H</i>)-one (2.5 g, 15.15 mmol) in THF (50 mL) at 0 °C under nitrogen. The resultant solution was stirred at 0 °C for 10 min before being allowed to warm to rt and stirred for 5.5 h. Saturated aq NH<sub>4</sub>Cl solution (25 mL) was added dropwise. EtOAc (25 mL) and 2 M aq HCl (25 mL) were added, the mixture was shaken, and the organic layer was removed. The aqueous layer was extracted further with EtOAc (25 mL). The organic layers were combined, washed with brine (25 mL), passed through a hydrophobic frit, and concentrated <i>in vacuo</i>. The resulting solid was triturated and then sonicated in diethyl ether (75 mL). The solid was removed under reduced pressure and washed with Et<sub>2</sub>O. The filtrate was concentrated <i>in vacuo</i>, yielding 5-chloro-4-methylpyridazin-3(2<i>H</i>)-one (1.785 g, 12.35 mmol, 81%) as a yellow solid. m.p. 124–126 °C; ν<sub>max</sub> (solid)/cm<sup>–1</sup>: 2873, 1634 (C═O), 1376, 1177, 1024, 884, 564; <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ ppm 7.88 (s, 1H), 2.24 (s, 3H); <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>OD) δ ppm 161.9, 137.6, 137.5, 136.9, 11.4; HRMS (M + H)<sup>+</sup> calculated for C<sub>5</sub>H<sub>6</sub>ClN<sub>2</sub>O, 145.0169; found, 145.0168; LC/MS (formic acid): <i>R</i><sub>t</sub> = 0.53 min (98%); [M + H]<sup>+</sup> = 145.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> 5-Chloro-2,4-dimethylpyridazin-3(2<i>H</i>)-one (<b>41</b>)</h3><div class="NLM_p last">Methyl iodide (1.14 mL, 18.16 mmol) was added to a stirred mixture of tetrabutylammonium bromide (6.630 g, 20.58 mmol), 5-chloro-4-methylpyridazin-3(2<i>H</i>)-one (1.750 g, 12.11 mmol), and potassium carbonate (3.35 g, 24.21 mmol) in MeCN (20 mL) at rt under nitrogen. The reaction mixture was stirred for 3 h at rt before being heated to 60 °C and stirred for 1 h. The resultant mixture was allowed to cool down to rt before being filtered under reduced pressure, and the filtrate was concentrated <i>in vacuo</i>. The resultant solid was dissolved in EtOAc (60 mL) and washed with 2 M aq HCl (40 mL). Methanol was added dropwise to dissolve a white precipitate. The organic layer was then washed with brine (40 mL), passed through a hydrophobic frit, and concentrated <i>in vacuo</i>. The resultant solid was dissolved in minimal DCM and purified by silica chromatography (0–50% EtOAc in cyclohexane). The desired fractions were concentrated <i>in vacuo</i>, yielding 5-chloro-2,4-dimethylpyridazin-3(2<i>H</i>)-one (1.062 g, 6.70 mmol, 55%) as a white solid. m.p. 73–74 °C; ν<sub>max</sub> (solid)/cm<sup>–1</sup>: 3032, 1634 (C═O), 1591, 1016, 887, 501; <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ ppm 7.89 (s, 1H), 3.76 (s, 3H), 2.26 (s, 3H); <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>OD) δ ppm 160.7, 136.6, 136.0, 135.9, 39.4, 12.0; HRMS (M + H)<sup>+</sup> calculated for C<sub>6</sub>H<sub>8</sub>ClN<sub>2</sub>O, 159.0325; found, 159.0325; LC/MS (TFA): <i>R</i><sub>t</sub> = 0.63 min (100%); [M + H]<sup>+</sup> = 159.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> 5-(Benzylamino)-2,4-dimethylpyridazin-3(2<i>H</i>)-one (<b>10</b>)</h3><div class="NLM_p last">Phenylmethanamine (0.10 mL, 0.95 mmol), 5-chloro-2,4-dimethylpyridazin-3(2<i>H</i>)-one (151 mg, 0.95 mmol), BrettPhos-Pd-G1 (76 mg, 0.10 mmol), BrettPhos (51 mg, 0.10 mmol), and sodium <i>tert</i>-butoxide (137 mg, 1.43 mmol) were dissolved in 1,4-dioxane (5 mL) at rt under nitrogen. The reaction mixture was then heated to 120 °C in a microwave reactor for 30 min. The reaction mixture was passed through Celite and the Celite was washed with EtOAc (3 × 10 mL). The resultant solution was then diluted with water (30 mL) and extracted with EtOAc (3 × 20 mL). The combined organic phases were passed through a hydrophobic frit and concentrated <i>in vacuo</i>. The resultant oil was then dissolved in 1:1 MeOH/DMSO, filtered, and purified via MDAP (high pH). The desired fractions were combined and concentrated <i>in vacuo</i>, yielding 5-(benzylamino)-2,4-dimethylpyridazin-3(2<i>H</i>)-one (108 mg, 0.47 mmol, 50%) as a white solid. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ ppm 7.57 (s, 1H), 7.39–7.31 (m, 4H), 7.30–7.23 (m, 1H), 4.57 (s, 2H), 3.66 (s, 3H), 2.04 (s, 3H); LC/MS (high pH): <i>R</i><sub>t</sub> = 0.82 min (100%); [M + H]<sup>+</sup> = 230.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> 2,4-Dimethyl-5-((2-methylbenzyl)amino)pyridazin-3(2<i>H</i>)-one (<b>11</b>)</h3><div class="NLM_p last"><i>o</i>-Tolylmethanamine (24 mg, 0.20 mmol), 5-chloro-2,4-dimethylpyridazin-3(2<i>H</i>)-one (24 mg, 0.15 mmol), BrettPhos-Pd-G1 (12 mg, 0.02 mmol), BrettPhos (8 mg, 0.02 mmol), and sodium <i>tert</i>-butoxide (22 mg, 0.23 mmol) were dissolved in 1,4-dioxane (0.6 mL) and heated to 120 °C in a microwave reactor for 30 min. The reaction mixture was loaded onto a preconditioned (MeCN × 2 mL) C18 SPE cartridge and eluted with 1:1 MeOH/DMSO. The resultant solution was concentrated under a positive pressure of nitrogen, redissolved in DMSO, and purified via MDAP (high pH). The desired fractions were combined and concentrated under a positive pressure of nitrogen, yielding 2,4-dimethyl-5-((2-methylbenzyl)amino)pyridazin-3(2<i>H</i>)-one (9 mg, 0.04 mmol, 23%). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 7.42 (s, 1H), 7.20–7.08 (m, 4H), 6.51 (t, <i>J</i> = 6.0 Hz, 1H), 4.46 (d, <i>J</i> = 6.0 Hz, 2H), 3.50 (s, 3H), 2.31 (s, 3H), 1.94 (s, 3H); LC/MS (formic acid): <i>R</i><sub>t</sub> = 0.85 min (100%); [M + H]<sup>+</sup> = 244.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> 5-(([1,1′-Biphenyl]-2-ylmethyl)amino)-2,4-dimethylpyridazin-3(2<i>H</i>)-one (<b>14</b>)</h3><div class="NLM_p last">[1,1′-Biphenyl]-2-ylmethanamine (0.08 mL, 0.47 mmol), 5-chloro-2,4-dimethylpyridazin-3(2<i>H</i>)-one (50 mg, 0.32 mmol), BrettPhos (17 mg, 0.03 mmol), Brettphos-Pd-G1 (25 mg, 0.03 mmol), and sodium <i>tert</i>-butoxide (35 mg, 0.36 mmol) were dissolved in 1,4-dioxane (1 mL). The reaction mixture was heated to 100 °C in a microwave reactor for 30 min. The reaction mixture was allowed to cool to rt before being diluted with saturated sodium bicarbonate solution (15 mL), and the aqueous phase was extracted with DCM (3 × 15 mL). The organic layers were combined, passed through a hydrophobic frit, and concentrated <i>in vacuo</i>. The resultant residue was dissolved in DMSO (2 mL) and purified by MDAP (formic acid). The desired fractions were combined and concentrated <i>in vacuo</i>, yielding 5-(([1,1′-biphenyl]-2-ylmethyl)amino)-2,4-dimethylpyridazin-3(2<i>H</i>)-one (58 mg, 0.19 mmol, 61%) as a beige solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 7.51–7.46 (m, 2H), 7.44–7.39 (m, 3H), 7.38–7.32 (m, 3H), 7.27–7.23 (m, 1H), 7.15 (s, 1H), 6.46 (t, <i>J</i> = 6.1 Hz, 1H), 4.40 (d, <i>J</i> = 6.1 Hz, 2H), 3.47 (s, 3H), 1.84 (s, 3H); LC/MS (formic acid): <i>R</i><sub>t</sub> = 1.01 min (100%); [M + H]<sup>+</sup> = 306.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> 2,4-Dimethyl-5-((2-morpholinobenzyl)amino)pyridazin-3(2<i>H</i>)-one (<b>15</b>)</h3><div class="NLM_p last">(2-Morpholinophenyl)methanamine (145 mg, 0.76 mmol), 5-chloro-2,4-dimethylpyridazin-3(2<i>H</i>)-one (50 mg, 0.32 mmol), XPhos-Pd-G1 (47 mg, 0.06 mmol), XPhos (30 mg, 0.06 mmol), and potassium <i>tert</i>-butoxide (35 mg, 0.36 mmol) were dissolved in 1,4-dioxane (4 mL). The reaction mixture was heated to 110 °C in a microwave reactor for 1 h. The reaction mixture was allowed to cool to rt before being diluted with EtOAc and washed with water and brine. The organic layer was passed through a hydrophobic frit and concentrated <i>in vacuo</i>. The resultant residue was dissolved in 1:1 MeOH/DMSO and purified via MDAP (high pH). The desired fractions were combined and concentrated <i>in vacuo</i>, yielding 2,4-dimethyl-5-((2-morpholinobenzyl)amino)pyridazin-3(2<i>H</i>)-one (33 mg, 0.10 mmol, 16%) as a white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm 7.51 (s, 1H), 7.36–7.29 (m, 2H), 7.20 (d, <i>J</i> = 7.8 Hz, 1H), 7.17–7.10 (m, 1H), 4.84–4.74 (m, 1H), 4.56 (d, <i>J</i> = 5.6 Hz, 2H), 3.91–3.85 (m, 4H), 3.73–3.66 (m, 3H), 2.98–2.92 (m, 4H), 2.02 (s, 3H); LC/MS (high pH): <i>R</i><sub>t</sub> = 0.82 min (100%); [M + H]<sup>+</sup> = 315.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> 5-((2-(1,4-Oxazepan-4-yl)benzyl)amino)-2,4-dimethylpyridazin-3(2<i>H</i>)-one, Hydrochloride (<b>16</b>)</h3><div class="NLM_p last">(2-(1,4-Oxazepan-4-yl)phenyl)methanamine (400 mg, 1.94 mmol), 5-chloro-2,4-dimethylpyridazin-3(2<i>H</i>)-one (369 mg, 2.33 mmol), RuPhos (277 mg, 0.58 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (178 mg, 0.19 mmol), and sodium <i>tert</i>-butoxide (559 mg, 5.82 mmol) were dissolved in 1,4-dioxane (2 mL) at 20 °C under nitrogen. The system was then sparged with nitrogen for 5 min. The reaction mixture was then heated to 110 °C in a microwave reactor for 1 h. The reaction mixture was allowed to cool down to rt and filtered, and the filtrate was concentrated <i>in vacuo</i>. The resultant residue was purified via prep-HPLC (high pH). The desired fractions were combined along with a drop of 2 M HCl and concentrated <i>in vacuo</i>, yielding 5-((2-(1,4-oxazepan-4-yl)benzyl)amino)-2,4-dimethylpyridazin-3(2<i>H</i>)-one, hydrochloride (90 mg, 0.25 mmol, 13%), as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 7.43 (s, 1H), 7.29–7.14 (m, 3H), 7.08–6.99 (m, 1H), 6.72 (t, <i>J</i> = 6.4 Hz, 1H), 4.49 (d, <i>J</i> = 6.4 Hz, 2H), 3.85 (t, <i>J</i> = 6.0 Hz, 2H), 3.82–3.74 (m, 2H), 3.48 (s, 3H), 3.17–3.07 (m, 4H), 1.99 (quin., <i>J</i> = 5.8 Hz, 2H), 1.91 (s, 3H); LC/MS (formic acid): <i>R</i><sub>t</sub> = 0.82 min (100%); [M + H]<sup>+</sup> = 328.</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> 2,4-Dimethyl-5-((2-(piperazin-1-yl)benzyl)amino)pyridazin-3(2<i>H</i>)-one (<b>17</b>)</h3><div class="NLM_p last"><i>tert-</i>Butyl 4-(2-(aminomethyl)phenyl)piperazine-1-carboxylate (58 mg, 0.2 mmol), 5-chloro-2,4-dimethylpyridazin-3(2<i>H</i>)-one (24 mg, 0.15 mmol), BrettPhos-Pd-G1 (12 mg, 0.02 mmol), BrettPhos (8 mg, 0.02 mmol), and sodium <i>tert</i>-butoxide (22 mg, 0.23 mmol) were dissolved in 1,4-dioxane and heated to 120 °C in a microwave reactor for 30 min. The reaction mixture was loaded on to a preconditioned (MeCN × 2 mL) C18 SPE cartridge and eluted with 1:1 MeOH/DMSO. The resultant solution was concentrated under a positive pressure of nitrogen, redissolved in DMSO, and purified via MDAP (high pH). The desired fractions were combined and concentrated under a positive pressure of nitrogen. The resultant residue was dissolved in DCM (0.5 mL) at rt and 4 M HCl in 1,4-dioxane was added (0.5 mL). The reaction mixture was stood at rt for 18 h before being concentrated <i>in vacuo</i>. The residue was dissolved in DMSO (0.6 mL) and purified via MDAP (high pH). The desired fractions were combined and concentrated <i>in vacuo</i>, yielding 2,4-dimethyl-5-((2-(piperazin-1-yl)benzyl)amino)pyridazin-3(2<i>H</i>)-one (5 mg, 0.02 mmol, 10%). <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ ppm 7.59 (s, 1H), 7.34 (dd, <i>J</i> = 7.6, 1.3 Hz, 1H), 7.30–7.24 (m, 1H), 7.22–7.17 (m, 1H), 7.09 (td, <i>J</i> = 7.6, 1.3 Hz, 1H), 4.58 (s, 2H), 3.64 (s, 3H), 3.07–3.01 (m, 4H), 2.96–2.89 (m, 4H), 2.02 (s, 3H); LC/MS (formic acid): <i>R</i><sub>t</sub> = 0.49 min (100%); [M + H]<sup>+</sup> = 314.</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> 2,4-Dimethyl-5-((2-(4-methylpiperazin-1-yl)benzyl)amino)pyridazin-3(2<i>H</i>)-one (<b>18</b>)</h3><div class="NLM_p last">(2-(4-Methylpiperazin-1-yl)phenyl)methanamine (200 mg, 0.97 mmol), 5-chloro-2,4-dimethylpyridazin-3(2<i>H</i>)-one (103 mg, 0.65 mmol), XPhos (31 mg, 0.07 mmol), XPhos-Pd-G1 (48 mg, 0.07 mmol), and sodium <i>tert</i>-butoxide (75 mg, 0.78 mmol) were dissolved in 1,4-dioxane (4 mL) at rt. The reaction mixture was heated to 100 °C in a microwave reactor for 30 min. The reaction mixture was allowed to cool down to rt before being diluted with water (10 mL) and extracted with EtOAc (3 × 10 mL). The combined organic phases were passed through a hydrophobic frit and concentrated <i>in vacuo</i>. The resultant oil was dissolved in 1:1 MeOH/DMSO purified via MDAP (high pH). The desired fractions were combined and concentrated <i>in vacuo</i>. The colorless oil was dissolved in MeOH (10 mL) and loaded onto a preconditioned (MeOH × 10 mL) amino propyl (2 g) column. The product was eluted with MeOH (10 mL) and the desired fractions were combined and concentrated <i>in vacuo</i>, yielding 2,4-dimethyl-5-((2-(4-methylpiperazin-1-yl)benzyl)amino)pyridazin-3(2<i>H</i>)-one (118 mg, 0.36 mmol, 56%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 7.43 (s, 1H), 7.29 (dd, <i>J</i> = 7.6, 1.5 Hz, 1H), 7.25–7.18 (m, 1H), 7.15–7.10 (m, 1H), 7.05 (td, <i>J</i> = 7.4, 1.1 Hz, 1H), 6.71 (t, <i>J</i> = 6.4 Hz, 1H), 4.44 (d, <i>J</i> = 6.4 Hz, 2H), 3.47 (s, 3H), 2.92–2.83 (m, 4H), 2.26 (s, 3H), 1.89 (s, 3H) (N.B., signal obscured by DMSO solvent peak); LC/MS (high pH): <i>R</i><sub>t</sub> = 0.77 min (98%); [M + H]<sup>+</sup> = 328.</div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> <i>tert</i>-Butyl (3-(4-(2-(((1,5-Dimethyl-6-oxo-1,6-dihydropyridazin-4-yl)amino)methyl)phenyl)piperazin-1-yl)propyl)carbamate</h3><div class="NLM_p last"><i>tert</i>-Butyl (3-bromopropyl)carbamate (449 mg, 1.886 mmol) was added to a solution of 2,4-dimethyl-5-((2-(piperazin-1-yl)benzyl)amino)pyridazin-3(2<i>H</i>)-one hydrochloride (600 mg, 1.715 mmol) and cesium carbonate (2235 mg, 6.860 mmol) in DMF (13 mL). The reaction mixture was stirred at rt for 18 h before being heated to 50 °C and stirred for an additional 18 h. The reaction mixture was concentrated <i>in vacuo</i> and the resultant residue was dissolved in EtOAc (50 mL) and washed with water (50 mL). The layers were separated, and the aqueous layer was extracted further with EtOAc (3 × 50 mL). The organic layers were combined, washed with brine (50 mL), passed through a hydrophobic frit, and concentrated <i>in vacuo</i>. The resulting residue was purified by silica chromatography (0–10% 2 M NH<sub>3</sub> in MeOH in DCM). The desired fractions were combined and concentrated <i>in vacuo</i>, yielding <i>tert</i>-butyl (3-(4-(2-(((1,5-dimethyl-6-oxo-1,6-dihydropyridazin-4-yl)amino)methyl)phenyl)piperazin-1-yl)propyl)carbamate (236 mg, 501 mmol, 20%) as a yellow oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm 7.54 (s, 1H), 7.36–7.29 (m, 2H), 7.22–7.08 (m, 2H), 4.90–4.84 (m, 1H), 4.54 (m, 2H), 3.71 (s, 3H), 3.28–3.19 (m, 2H), 2.99 (t, <i>J</i> = 4.6 Hz, 4H), 2.65 (br. s, 4H), 2.51 (t, <i>J</i> = 6.7 Hz, 2H), 2.01 (s, 3H), 1.72 (quin., <i>J</i> = 6.7 Hz, 2H), 1.47 (s, 9H) (N.B., exchangeable amine proton not visible); LC/MS (formic acid): <i>R</i><sub>t</sub> = 0.70 min (72%); [M + H]<sup>+</sup> = 471.</div></div><div id="sec4_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> 5-((2-(4-(3-Aminopropyl)piperazin-1-yl)benzyl)amino)-2,4-dimethylpyridazin-3(2<i>H</i>)-one (<b>19</b>)</h3><div class="NLM_p last"><i>tert</i>-Butyl (3-(4-(2-(((1,5-dimethyl-6-oxo-1,6-dihydropyridazin-4-yl)amino)methyl)phenyl)piperazin-1-yl)propyl)carbamate (236 mg, 0.501 mmol) was dissolved in 4 M HCl in 1,4-dioxane (5 mL, 20.00 mmol) and left to stir at rt for 18 h. The reaction mixture was concentrated <i>in vacuo</i> and the resultant residue was suspended in diethyl ether (30 mL), filtered under reduced pressure, rinsed with diethyl ether (20 mL), and collected. The residue was purified by MDAP (high pH). The desired fractions were combined and concentrated <i>in vacuo</i>, yielding 5-((2-(4-(3-aminopropyl)piperazin-1-yl)benzyl)amino)-2,4-dimethylpyridazin-3(2<i>H</i>)-one (33 mg, 0.089 mmol, 18%) as a pale yellow solid. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ ppm 7.58 (s, 1H), 7.35 (d, <i>J</i> = 7.6 Hz, 1H), 7.30–7.24 (m, 1H), 7.21 (d, <i>J</i> = 7.8 Hz, 1H), 7.12–7.06 (m, 1H), 4.57 (s, 2H), 3.63 (s, 3H), 3.02 (t, <i>J</i> = 4.6 Hz, 4H), 2.93 (t, <i>J</i> = 7.0 Hz, 2H), 2.74 (br. s, 4H), 2.59 (t, <i>J</i> = 7.0 Hz, 2H), 2.01 (s, 3H), 1.84 (quin., <i>J</i> = 7.0 Hz, 2H) (N.B., exchangeable amine protons not visible); LC/MS (high pH): <i>R</i><sub>t</sub> = 0.65 min (100%); [M + H]<sup>+</sup> = 371.</div></div><div id="sec4_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> 4-Chloro-2-methyl-5-((2-(4-methylpiperazin-1-yl)benzyl)amino)pyridazin-3(2<i>H</i>)-one (<b>42</b>)</h3><div class="NLM_p last">DIPEA (0.73 mL, 4.19 mmol) was added to a stirred solution of 4,5-dichloro-2-methylpyridazin-3(2<i>H</i>)-one (500 mg, 2.79 mmol) and (2-(4-methylpiperazin-1-yl)phenyl)methanamine (0.66 mL, 3.41 mmol) in DMSO (10 mL) at rt. The resultant solution was heated to 120 °C and stirred for 1 h in a microwave reactor. The resultant solution was diluted with saturated aq NH<sub>4</sub>Cl solution (15 mL) and EtOAc (15 mL) and the layers were separated. The aqueous layer was then extracted further with EtOAc (5 mL × 2). The organic fractions were combined and washed with brine before being passed through a hydrophobic frit and concentrated <i>in vacuo</i>. The resultant residue was dissolved in minimal DCM and purification by silica chromatography was attempted (0–20% 2 M NH<sub>3</sub> in MeOH in DCM). The desired fractions were combined and concentrated <i>in vacuo</i>. The resultant solid was dissolved in 1:1 DMSO and MeOH and purified by MDAP (high pH). The desired fractions were combined, yielding 4-chloro-2-methyl-5-((2-(4-methylpiperazin-1-yl)benzyl)amino)pyridazin-3(2<i>H</i>)-one (607 mg, 1.75 mmol, 63%) as a cream solid. ν<sub>max</sub> (solid)/cm<sup>–1</sup>: 3322 (N–H), 1609 (C═O), 1201, 1118, 720; <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ ppm 7.63 (s, 1H), 7.41–7.31 (m, 2H), 7.30–7.26 (m, 1H), 7.19 (td, <i>J</i> = 7.4, 1.3 Hz, 1H), 4.71–4.63 (m, 2H), 3.68 (s, 3H), 3.39 (app. br. s, 4H), 3.20 (t, <i>J</i> = 4.9 Hz, 4H), 2.91 (s, 3H) (N.B., exchangeable amine proton not visible); <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>OD) δ ppm 158.5, 149.3, 145.2, 133.2, 128.5, 128.2, 127.0, 125.4, 120.2, 105.9, 54.2, 50.3, 42.8, 41.3, 39.1; LC/MS (formic acid): <i>R</i><sub>t</sub> = 0.44 min (99%); [M + H]<sup>+</sup> = 348.</div></div><div id="sec4_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> 4-Allyl-2-methyl-5-((2-(4-methylpiperazin-1-yl)benzyl)amino)pyridazin-3(2<i>H</i>)-one (<b>21</b>)</h3><div class="NLM_p last">The solvent system was sparged with nitrogen for 30 min prior to use. 2-Allyl-4,4,5,5-tetramethyl-1,3,2-dioxaborolane (0.05 mL, 0.29 mmol) was added to a stirred solution of 4-chloro-2-methyl-5-((2-(4-methylpiperazin-1-yl)benzyl)amino)pyridazin-3(2<i>H</i>)-one (50 mg, 0.14 mmol), potassium carbonate (50 mg, 0.36 mmol), and RuPhos-Pd-G2 (17 mg, 0.02 mmol) in 1,4-dioxane (2 mL) and water (1 mL) under nitrogen at rt. The resultant mixture was heated to 100 °C and stirred for 1 h. The resultant solution was diluted with EtOAc and filtered through Celite before being concentrated <i>in vacuo</i>. The resultant residue was dissolved in EtOAc (10 mL) and washed with water (10 mL). The organic layer was passed through a hydrophobic frit and concentrated <i>in vacuo</i>. The resultant residue was dissolved in 1:1 MeOH/DMSO and purified by MDAP (high pH: extended). The desired fractions were combined and concentrated <i>in vacuo</i>, yielding 4-allyl-2-methyl-5-((2-(4-methylpiperazin-1-yl)benzyl)amino)pyridazin-3(2<i>H</i>)-one (14 mg, 0.04 mmol, 28%) as a colorless oil. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ ppm 7.59 (s, 1H), 7.34–7.30 (m, 1H), 7.26 (td, <i>J</i> = 7.6, 1.2 Hz, 1H), 7.20 (dd, <i>J</i> = 7.6, 1.2 Hz, 1H), 7.08 (td, <i>J</i> = 7.6, 1.2 Hz, 1H), 5.87 (ddt, <i>J</i> = 17.1, 10.3, 5.7 Hz, 1H), 5.11–5.01 (m, 2H), 4.56 (s, 2H), 3.63 (s, 3H), 3.00 (t, <i>J</i> = 4.8 Hz, 4H), 2.70 (app. br. s, 4H), 2.40 (s, 3H) (N.B., amine proton not visible and CH<sub>2</sub> allyl proton signal obscured by MeOH solvent peak); <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>OD) δ ppm 161.7, 150.5, 146.1, 133.3, 132.9, 128.3, 128.1, 128.0, 124.1, 119.5, 114.4, 109.1, 55.1, 52.1, 44.8, 40.9, 38.9, 26.6; HRMS (M + H)<sup>+</sup> calculated for C<sub>20</sub>H<sub>28</sub>N<sub>5</sub>O, 354.2294; found, 354.2299; LC/MS (high pH): <i>R</i><sub>t</sub> = 0.90 min (100%); [M + H]<sup>+</sup> = 354.</div></div><div id="sec4_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> (<i>E</i>)-4-(But-1-en-1-yl)-2-methyl-5-((2-(4-methylpiperazin-1-yl)benzyl)amino)pyridazin-3(2<i>H</i>)-one (<b>24</b>)</h3><div class="NLM_p last">The solvent system was sparged with nitrogen for 20 min prior to use. (<i>E</i>)-But-1-en-1-ylboronic acid (15 mg, 0.15 mmol), 4-chloro-2-methyl-5-((2-(4-methylpiperazin-1-yl)benzyl)amino)pyridazin-3(2<i>H</i>)-one (24 mg, 0.07 mmol), potassium carbonate (24 mg, 0.17 mmol), Pd(OAc)<sub>2</sub> (2 mg, 0.01 mmol), and butyldi-1-adamantylphosphine (4 mg, 0.01 mmol) were dissolved in 1,4-dioxane (2 mL) and water (1 mL) under nitrogen at rt. The resultant mixture was heated to 100 °C and stirred for 1 h in a microwave reactor. A further portion of (<i>E</i>)-but-1-en-1-ylboronic acid (15 mg, 0.15 mmol), potassium carbonate (24 mg, 0.17 mmol), Pd(OAc)<sub>2</sub> (2 mg, 0.01 mmol), and butyldi-1-adamantylphosphine (4 mg, 0.01 mmol) were added under nitrogen and the reaction was stirred at 100 °C for 2 h in a microwave reactor. A further portion of (<i>E</i>)-but-1-en-1-ylboronic acid (15 mg, 0.15 mmol), potassium carbonate (24 mg, 0.17 mmol), Pd(OAc)<sub>2</sub> (2 mg, 0.01 mmol), and butyldi-1-adamantylphosphine (4 mg, 0.01 mmol) were added under nitrogen and the reaction was stirred at 100 °C for 2 h. The solution was allowed to cool down to rt before being diluted with EtOAc (10 mL), filtered through Celite, and concentrated <i>in vacuo</i>. The resultant residue was dissolved in EtOAc (5 mL) and washed sequentially with water (5 mL) and brine (5 mL) before being passed through a hydrophobic frit and concentrated <i>in vacuo</i>. The resultant residue was dissolved in 1:1 MeOH/DMSO and purified by MDAP (high pH). The desired fractions were combined, yielding (<i>E</i>)-4-(but-1-en-1-yl)-2-methyl-5-((2-(4-methylpiperazin-1-yl)benzyl)amino)pyridazin-3(2<i>H</i>)-one (12 mg, 0.03 mmol, 47%) as a colorless oil. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ ppm 7.60 (s, 1H), 7.34 (dd, <i>J</i> = 7.6, 1.2 Hz, 1H), 7.31–7.25 (m, 1H), 7.24–7.19 (m, 1H), 7.10 (td, <i>J</i> = 7.6, 1.2 Hz, 1H), 6.77 (dt, <i>J</i> = 16.0, 6.6 Hz, 1H), 6.30 (dt, <i>J</i> = 16.0, 1.7 Hz, 1H), 4.57 (s, 2H), 3.63 (s, 3H), 3.00 (t, <i>J</i> = 4.8 Hz, 4H), 2.70 (app. br. s, 4H), 2.39 (s, 3H), 2.36–2.27 (m, 2H), 1.14 (t, <i>J</i> = 7.5 Hz, 3H) (N.B., amine proton not visible); <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>OD) δ ppm 160.8, 150.6, 144.6, 139.3, 133.2, 128.4, 128.2, 128.1, 124.3, 119.7, 118.1, 108.5, 55.2, 52.1, 44.8, 41.6, 38.8, 27.0, 12.6; HRMS (M + H)<sup>+</sup> calculated for C<sub>21</sub>H<sub>30</sub>N<sub>5</sub>O, 368.2450; found, 368.2461; LC/MS (high pH): <i>R</i><sub>t</sub> = 1.01 min (98%); [M + H]<sup>+</sup> = 368.</div></div><div id="sec4_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> (<i>E</i>)-4-(But-2-en-1-yl)-2-methyl-5-((2-(4-methylpiperazin-1-yl)benzyl)amino)pyridazin-3(2<i>H</i>)-one (<b>22</b>), (<i>Z</i>)-4-(But-2-en-1-yl)-2-methyl-5-((2-(4-methylpiperazin-1-yl)benzyl)amino)pyridazin-3(2<i>H</i>)-one (<b>23</b>), Single Unknown Enantiomer (<i>R</i>)-4-(But-3-en-2-yl)-2-methyl-5-((2-(4-methylpiperazin-1-yl)benzyl)amino)pyridazin-3(2<i>H</i>)-one (<b>26</b>), and Single Unknown Enantiomer (<i>S</i>)-4-(But-3-en-2-yl)-2-methyl-5-((2-(4-methylpiperazin-1-yl)benzyl)amino)pyridazin-3(2<i>H</i>)-one (<b>27</b>)</h3><div class="NLM_p">The solvent system was sparged with nitrogen for 2 h prior to use. (<i>E</i>)-2-(But-2-en-1-yl)-4,4,5,5-tetramethyl-1,3,2-dioxaborolane (0.34 mL, 1.67 mmol) was added to a stirred solution of 4-chloro-2-methyl-5-((2-(4-methylpiperazin-1-yl)benzyl)amino)pyridazin-3(2<i>H</i>)-one (290 mg, 0.834 mmol), potassium carbonate (288 mg, 2.08 mmol), Pd(OAc)<sub>2</sub> (37 mg, 0.17 mmol), and butyldi-1-adamantylphosphine (60 mg, 0.17 mmol) in 1,4-dioxane (1 mL) and water (0.5 mL) under nitrogen at rt. The resultant mixture was heated to 100 °C and stirred for 1.5 h in a microwave reactor. The resultant solution was diluted with EtOAc (3 mL) and passed through Celite before being concentrated <i>in vacuo</i>. The resultant residue was dissolved in EtOAc (3 mL) and washed with H<sub>2</sub>O (3 mL) before being concentrated <i>in vacuo</i>, dissolved in 1:1 MeOH/DMSO, and purified by MDAP (high pH). MDAP failed to separate the isomers. The desired fractions were combined and concentrated <i>in vacuo</i>, yielding a mixture of 4-(but-3-en-2-yl)-2-methyl-5-((2-(4-methylpiperazin-1-yl)benzyl)amino)pyridazin-3(2<i>H</i>)-one, (<i>E</i>)-4-(but-2-en-1-yl)-2-methyl-5-((2-(4-methylpiperazin-1-yl)benzyl)amino)pyridazin-3(2<i>H</i>)-one, and (<i>Z</i>)-4-(but-2-en-1-yl)-2-methyl-5-((2-(4-methylpiperazin-1-yl)benzyl)amino)pyridazin-3(2<i>H</i>)-one as a combined off-white gum. The gum was dissolved in EtOH (4.5 mL) and purified via chiral chromatography: Chiralpak AD-H column, 30 mm × 25 cm, 10%EtOH (+0.2% isopropylamine)/heptane (+0.2% isopropylamine), 30 mL/min. The desired fractions were combined and concentrated <i>in vacuo</i>, yielding the four products:</div><div class="NLM_p">(<i>E</i>)-4-(But-2-en-1-yl)-2-methyl-5-((2-(4-methylpiperazin-1-yl)benzyl)amino)pyridazin-3(2<i>H</i>)-one (<b>22</b>) as a white solid (26 mg, 0.07 mmol, 8%) as a colorless oil. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ ppm 7.60 (s, 1H), 7.33–7.24 (m, 2H), 7.21 (dd, <i>J</i> = 7.2, 1.2 Hz, 1H), 7.09 (td, <i>J</i> = 7.2, 1.2 Hz, 1H), 5.49 (dqt, <i>J</i> = 16.0, 6.0, 1.0 Hz, 1H), 5.45 (dtq, <i>J</i> = 16.0, 6.0, 1.0 Hz, 1H), 4.56 (s, 2H), 3.63 (s, 3H), 3.26–3.20 (m, 2H), 3.00 (t, <i>J</i> = 4.8 Hz, 4H), 2.71 (app. br. s, 4H), 2.40 (s, 3H), 1.70–1.63 (m, 3H); <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>OD) δ ppm 161.8, 150.6, 145.9, 133.3, 128.4, 128.1, 128.0, 125.6, 125.3, 124.1, 119.6, 110.1, 55.1, 52.1, 44.8, 40.9, 38.9, 25.4, 16.6; HRMS (M + H)<sup>+</sup> calculated for C<sub>21</sub>H<sub>30</sub>N<sub>5</sub>O, 368.2450; found, 368.2447; LC/MS (high pH): <i>R</i><sub>t</sub> = 0.96 min (100%); [M + H]<sup>+</sup> = 368.</div><div class="NLM_p">(<i>Z</i>)-4-(But-2-en-1-yl)-2-methyl-5-((2-(4-methylpiperazin-1-yl)benzyl)amino)pyridazin-3(2<i>H</i>)-one (<b>23</b>) (15 mg, 0.04 mmol, 5%) as a colorless oil. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ ppm 7.61 (s, 1H), 7.37–7.24 (m, 2H), 7.24–7.19 (m, 1H), 7.13–7.07 (m, 1H), 5.59 (dqt, <i>J</i> = 10.5, 6.8, 2.0 Hz, 1H), 5.34 (dtq, <i>J</i> = 10.5, 6.8, 1.8 Hz, 1H), 4.57 (s, 2H), 3.64 (s, 3H), 3.04–2.96 (m, 4H), 2.70 (app. br. s, 4H), 2.40 (s, 3H), 1.77–1.73 (m, 3H); <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>OD) δ ppm 161.7, 150.6, 145.8, 133.2, 128.4, 128.2, 128.1, 125.8, 125.3, 124.7, 119.6, 111.1, 55.1, 52.1, 44.7, 41.2, 38.9, 20.7, 11.9; HRMS (M + H)<sup>+</sup> calculated for C<sub>21</sub>H<sub>30</sub>N<sub>5</sub>O, 368.2450; found, 368.2444; LC/MS (high pH): <i>R</i><sub>t</sub> = 0.97 min (100%); [M + H]<sup>+</sup> = 368.</div><div class="NLM_p">Single unknown enantiomer (<i>R</i>)-4-(but-3-en-2-yl)-2-methyl-5-((2-(4-methylpiperazin-1-yl)benzyl)amino)pyridazin-3(2<i>H</i>)-one (<b>26</b>) (27 mg, 0.07 mmol, 9%) as a colorless oil. HPLC (Chiralpak AD-H column, 4.6 mm × 25 cm, 10% EtOH (+0.2% isopropylamine)/heptane, 1 mL/min): 10.5 min (major enantiomer), 11.6 (minor enantiomer), 97% ee; [α<sub>D</sub>]<sup>23</sup> (<i>c</i> = 10 mg/mL, MeOH): +21°; <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ ppm 7.59 (s, 1H), 7.35–7.24 (m, 2H), 7.23–7.18 (m, 1H), 7.10 (td, <i>J</i> = 7.6, 1.2 Hz, 1H), 6.18 (ddd, <i>J</i> = 17.4, 10.5, 5.1 Hz, 1H), 5.17 (dt, <i>J</i> = 17.4, 1.8 Hz, 1H), 5.13 (dt, <i>J</i> = 10.5, 1.8 Hz, 1H), 4.59–4.49 (m, 2H), 4.05–3.97 (m, 1H), 3.62 (s, 3H), 3.05–2.95 (m, 4H), 2.80–2.63 (m, 4H), 2.40 (s, 3H), 1.38 (d, <i>J</i> = 7.3 Hz, 3H); <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>OD) δ ppm 161.3, 150.5, 145.4, 139.8, 133.2, 128.7, 128.2, 128.1, 124.2, 119.6, 113.8, 113.4, 55.1, 52.2, 44.7, 41.0, 38.9, 32.6, 14.1; HRMS (M + H)<sup>+</sup> calculated for C<sub>21</sub>H<sub>30</sub>N<sub>5</sub>O, 368.2450; found, 368.2444; LC/MS (high pH): <i>R</i><sub>t</sub> = 0.98 min (100%); [M + H]<sup>+</sup> = 368.</div><div class="NLM_p last">Single unknown enantiomer (<i>S</i>)-4-(but-3-en-2-yl)-2-methyl-5-((2-(4-methylpiperazin-1-yl)benzyl)amino)pyridazin-3(2<i>H</i>)-one (<b>27</b>) (23 mg, 0.06 mmol, 8%) as a colorless oil. Analytical data as <b>26</b> above except HPLC (Chiralpak AD-H column, 4.6 mm × 25 cm, 10% EtOH (+0.2% isopropylamine)/heptane, 1 mL/min): 11.3 (major enantiomer), 10.4 (minor enantiomer), 95% ee; [α<sub>D</sub>]<sup>23</sup> (<i>c</i> = 10 mg/mL, MeOH): −21°.</div></div><div id="sec4_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> 4-(But-3-en-1-yl)-5-chloro-2-methylpyridazin-3(2<i>H</i>)-one (<b>43</b>)</h3><div class="NLM_p last">5-Chloro-2,4-dimethylpyridazin-3(2<i>H</i>)-one (500 mg, 3.15 mmol) was dissolved in THF (10 mL) at rt under nitrogen. The solution was cooled down to −78 °C and LiHMDS (1 M in THF, 4.73 mL, 4.73 mmol) and 3-bromoprop-1-ene (0.36 mL, 4.10 mmol) were added under nitrogen. The resultant solution was stirred at −78 °C under nitrogen for 45 min before being diluted with MeOH (5 mL), allowed to warm to rt, and concentrated <i>in vacuo</i>. The resultant residue was dissolved in EtOAc (10 mL) and washed with water. The aqueous layer was separated and extracted further with EtOAc (10 mL × 2). The organic layers were combined, washed with brine (15 mL), and passed through a hydrophobic frit before being concentrated <i>in vacuo</i>. The resultant residue was dissolved in minimal DCM and purified by silica chromatography (0–50% EtOAc in cyclohexane). The desired fractions were combined, yielding 4-(but-3-en-1-yl)-5-chloro-2-methylpyridazin-3(2<i>H</i>)-one (150 mg, 0.76 mmol, 24%) as a colorless oil. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ ppm 7.89 (s, 1H), 5.89 (ddt, <i>J</i> = 17.0, 10.2, 6.8 Hz, 1H), 5.04 (app. dq, <i>J</i> = 17.1, 1.8 Hz, 1H), 4.98 (ddt, <i>J</i> = 10.1, 1.8, 1.1 Hz, 1H), 3.75 (s, 3H), 2.84 (t, <i>J</i> = 7.3 Hz, 2H), 2.39–2.30 (m, 2H); <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>OD) δ ppm 160.3, 138.9, 136.7, 136.6, 114.7, 39.4, 30.5, 26.7 (N.B., one sp<sup>2</sup> carbon signal missing, 136.7 broad and intense possible overlap of two signals); HRMS (M + H)<sup>+</sup> calculated for C<sub>9</sub>H<sub>12</sub>ClN<sub>2</sub>O, 199.0638; found, 199.0638; LC/MS (formic acid): <i>R</i><sub>t</sub> = 0.96 min (91%); [M + H]<sup>+</sup> = 199.</div></div><div id="sec4_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> 4-(But-3-en-1-yl)-2-methyl-5-((2-(4-methylpiperazin-1-yl)benzyl)amino)pyridazin-3(2<i>H</i>)-one (<b>25</b>)</h3><div class="NLM_p last">(2-(4-Methylpiperazin-1-yl)phenyl)methanamine (0.17 mL, 0.88 mmol), 4-(but-3-en-1-yl)-5-chloro-2-methylpyridazin-3(2<i>H</i>)-one (145 mg, 0.73 mmol), sodium <i>tert</i>-butoxide (140 mg, 1.46 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (54 mg, 0.07 mmol), and BrettPhos (78 mg, 0.15 mmol) were dissolved in THF (0.5 mL). The resultant solution was stirred at 100 °C for 1 h in a microwave reactor. The reaction was allowed to cool down to rt before being concentrated <i>in vacuo</i>. The resultant residue was dissolved in 1:1 MeOH/DMSO and purified by MDAP (high pH). The desired fractions were combined, yielding 4-(but-3-en-1-yl)-2-methyl-5-((2-(4-methylpiperazin-1-yl)benzyl)amino)pyridazin-3(2<i>H</i>)-one (54 mg, 0.15 mmol, 20%) as a white solid. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ ppm 7.56 (s, 1H), 7.35 (dd, <i>J</i> = 7.5, 1.2 Hz, 1H), 7.30–7.24 (m, 1H), 7.20 (dd, <i>J</i> = 7.5, 1.2 Hz, 1H), 7.09 (td, <i>J</i> = 7.5, 1.2 Hz, 1H), 5.93 (ddt, <i>J</i> = 17.0, 10.2, 6.8 Hz, 1H), 5.09–5.01 (m, 1H), 4.99–4.94 (m, 1H), 4.55 (s, 2H), 3.62 (s, 3H), 3.01 (t, <i>J</i> = 4.8 Hz, 4H), 2.78–2.60 (m, 6H), 2.40 (s, 3H), 2.27 (app. q, <i>J</i> = 7.4 Hz, 2H) (N.B., exchangeable amine proton not visible); <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>OD) δ ppm 161.9, 150.5, 145.9, 137.7, 133.4, 128.4, 128.1, 128.0, 124.1, 119.5, 114.2, 111.6, 55.2, 52.1, 44.8, 40.9, 38.8, 30.6, 22.4; HRMS (M + H)<sup>+</sup> calculated for C<sub>21</sub>H<sub>30</sub>N<sub>5</sub>O, 368.2450; found, 368.2451; LC/MS (high pH): <i>R</i><sub>t</sub> = 0.93 min (100%); [M + H]<sup>+</sup> = 368.</div></div><div id="sec4_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> 5-Chloro-4-ethylpyridazin-3(2<i>H</i>)-one (<b>44</b>)</h3><div class="NLM_p last">Ethylmagnesium bromide (3 M) in Et<sub>2</sub>O solution (15.15 mL, 45.50 mmol) was added dropwise to a stirred suspension of 4,5-dichloropyridazin-3(2<i>H</i>)-one (2.500 g, 15.15 mmol) in THF (50 mL) at 0 °C under nitrogen. The resultant solution was stirred at 0 °C for 10 min before being allowed to warm to rt and stirred for 3 h. Saturated aq NH<sub>4</sub>Cl solution (25 mL) was added slowly over 10 min. EtOAc (50 mL) and 2 M aqueous HCl (50 mL) were added, the mixture was shaken, and the organic layer was removed. The aqueous layer was extracted further with EtOAc (25 mL). The organic layers were combined, washed with brine (25 mL), passed through a hydrophobic frit, and concentrated <i>in vacuo</i>. The resultant oil was dissolved in minimal DCM and purified by silica chromatography (0–40% EtOAc in cyclohexane). The desired fractions were combined, yielding 5-chloro-4-ethylpyridazin-3(2<i>H</i>)-one (625 mg, 3.94 mmol, 26%) as a white solid. m.p. 135–138 °C; ν<sub>max</sub> (solid)/cm<sup>–1</sup>: 2864, 1638 (C═O), 1177, 914, 575; <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ ppm 7.88 (s, 1H), 2.75 (q, <i>J</i> = 7.6 Hz, 2H), 1.17 (t, <i>J</i> = 7.6 Hz, 3H) (N.B., exchangeable lactam proton not visible); <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>OD) δ ppm 161.4, 141.7, 137.9, 137.0, 20.0, 10.0; HRMS (M + H)<sup>+</sup> calculated for C<sub>6</sub>H<sub>8</sub>ClN<sub>2</sub>O, 159.0325; found, 159.0325; LC/MS (formic acid): <i>R</i><sub>t</sub> = 0.66 min (98%); [M + H]<sup>+</sup> = 159.</div></div><div id="sec4_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> 5-Chloro-4-ethyl-2-methylpyridazin-3(2<i>H</i>)-one (<b>45</b>)</h3><div class="NLM_p last">Methyl iodide (0.20 mL, 3.15 mmol) was added to a stirred mixture of tetrabutylammonium bromide (1.118 g, 3.47 mmol), 5-chloro-4-ethylpyridazin-3(2H)-one (250 mg, 1.58 mmol), and potassium carbonate (436 mg, 3.15 mmol) in MeCN (5 mL) at rt. The resultant mixture was stirred under nitrogen for 3 h at rt. The resultant mixture was filtered under reduced pressure and the filtrate was concentrated <i>in vacuo</i>. The resultant solid was dissolved in EtOAc (50 mL) and washed with 2 M aq HCl (30 mL). MeOH was added dropwise to dissolve a white precipitate. The organic layer was then washed with brine (30 mL), passed through a hydrophobic frit, and concentrated <i>in vacuo</i>. The resultant solid was dissolved in minimal DCM and purified by silica chromatography (0–50% EtOAc in cyclohexane). The desired fractions were concentrated <i>in vacuo</i>, yielding 5-chloro-4-ethyl-2-methylpyridazin-3(2<i>H</i>)-one (170 mg, 0.99 mmol, 63%) as a colorless oil. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ ppm 7.89 (s, 1H), 3.75 (s, 3H), 2.76 (q, <i>J</i> = 7.6 Hz, 2H), 1.16 (t, <i>J</i> = 7.6 Hz, 3H); <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>OD) δ ppm 160.2, 140.9, 136.8, 136.0, 39.4, 20.6, 9.9; HRMS (M + H)<sup>+</sup> calculated for C<sub>7</sub>H<sub>10</sub>ClN<sub>2</sub>O, 173.0482; found, 173.0478; LC/MS (formic acid): <i>R</i><sub>t</sub> = 0.80 min (99%); [M + H]<sup>+</sup> = 173.</div></div><div id="sec4_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> 4-Ethyl-2-methyl-5-((2-(4-methylpiperazin-1-yl)benzyl)amino)pyridazin-3(2<i>H</i>)-one (<b>28</b>)</h3><div class="NLM_p last">(2-(4-Methylpiperazin-1-yl)phenyl)methanamine (0.13 mL, 0.70 mmol), 5-chloro-4-ethyl-2-methylpyridazin-3(2<i>H</i>)-one (100 mg, 0.58 mmol), sodium <i>tert</i>-butoxide (111 mg, 1.16 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (53 mg, 0.058 mmol), and BrettPhos (62 mg, 0.12 mmol) were dissolved in THF (0.5 mL) and heated to 100 °C in a microwave reactor. The reaction mixture was stirred at 100 °C for 1 h. The resultant solution was allowed to cool down to rt, diluted with EtOAc (10 mL), filtered through Celite, and concentrated <i>in vacuo</i>. The resultant residue was dissolved in EtOAc (10 mL) and washed sequentially with water (10 mL) and brine (10 mL). The organic layer was passed through a hydrophobic frit and concentrated <i>in vacuo</i>. The resultant residue was dissolved in 1:1 MeOH/DMSO and purified by MDAP (high pH). The desired fractions were combined and concentrated <i>in vacuo</i>, yielding 4-ethyl-2-methyl-5-((2-(4-methylpiperazin-1-yl)benzyl)amino)pyridazin-3(2<i>H</i>)-one (91 mg, 0.27 mmol, 46%) as a white solid. m.p. 148–149 °C; ν<sub>max</sub> (solid)/cm<sup>–1</sup>: 3344, 2796, 1595 (C═O), 1225, 769; <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ ppm 7.56 (s, 1H), 7.37–7.31 (m, 1H), 7.30–7.23 (m, 1H), 7.21 (dd, <i>J</i> = 7.4, 1.3 Hz, 1H), 7.09 (td, <i>J</i> = 7.4, 1.3 Hz, 1H), 4.56 (s, 2H), 3.62 (s, 3H), 3.01 (t, <i>J</i> = 4.8 Hz, 4H), 2.71 (app. br. s, 4H), 2.57 (q, <i>J</i> = 7.5 Hz, 2H), 2.40 (s, 3H), 1.10 (t, <i>J</i> = 7.5 Hz, 3H) (N.B., exchangeable amine proton not visible); <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>OD) δ ppm 161.7, 150.5, 145.5, 133.4, 128.5, 128.0, 127.9, 124.1, 119.5, 113.8, 55.2, 52.2, 44.8, 40.8, 38.8, 16.0, 10.1; HRMS (M + H)<sup>+</sup> calculated for C<sub>19</sub>H<sub>28</sub>N<sub>5</sub>O, 342.2294; found, 342.2292; LC/MS (high pH): <i>R</i><sub>t</sub> = 0.84 min (100%); [M + H]<sup>+</sup> = 342.</div></div><div id="sec4_25" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> 5-Chloro-4-isopropylpyridazin-3(2<i>H</i>)-one (<b>46</b>)</h3><div class="NLM_p last">Isopropylmagnesium bromide (2.9 M) in 2-methyl THF solution (3.14 mL, 9.09 mmol) was added dropwise to a stirred suspension of 4,5-dichloropyridazin-3(2<i>H</i>)-one (500 mg, 3.03 mmol) in THF (10 mL) at 0 °C under nitrogen. The resultant solution was stirred at 0 °C for 10 min before being allowed to warm to rt and stirred for 1.5 h. Saturated aq NH<sub>4</sub>Cl solution (5 mL) was added dropwise. EtOAc (5 mL) and 2 M aq HCl (5 mL) were added, the mixture was shaken, and the organic layer was removed. The aqueous layer was extracted further with EtOAc (25 mL). The organic layers were combined, washed with brine (25 mL), passed through a hydrophobic frit, and concentrated <i>in vacuo</i>. The resultant oil was dissolved in minimal DCM and purified by silica chromatography (0–40% EtOAc in cyclohexane). The desired fractions were combined and concentrated <i>in vacuo</i>, yielding 5-chloro-4-isopropylpyridazin-3(2<i>H</i>)-one (140 mg, 0.81 mmol, 27%) as an off-white solid. m.p. 98–102 °C; ν<sub>max</sub> (solid)/cm<sup>–1</sup>: 2966, 2876, 1645 (C═O), 1180, 1055, 918, 593; <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ ppm 7.84 (s, 1H), 3.50 (sept., <i>J</i> = 7.1 Hz, 1H), 1.36 (d, <i>J</i> = 7.1 Hz, 6H); <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>OD) δ ppm 160.9, 143.8, 138.0, 136.2, 28.9, 17.4; HRMS (M + H)<sup>+</sup> calculated for C<sub>7</sub>H<sub>10</sub>ClN<sub>2</sub>O, 173.0482; found, 173.0477; LC/MS (formic acid): <i>R</i><sub>t</sub> = 0.81 min (85%); [M + H]<sup>+</sup> = 173.</div></div><div id="sec4_26" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> 5-Chloro-4-isopropyl-2-methylpyridazin-3(2<i>H</i>)-one (<b>47</b>)</h3><div class="NLM_p last">Methyl iodide (0.07 mL, 1.11 mmol) was added to a stirred mixture of tetrabutylammonium bromide (406 mg, 1.26 mmol), 5-chloro-4-isopropylpyridazin-3(2<i>H</i>)-one (128 mg, 0.74 mmol), and potassium carbonate (205 mg, 1.48 mmol) in MeCN (5 mL) at rt. The resultant mixture was stirred under nitrogen for 2.5 h at rt. The reaction mixture was filtered under reduced pressure and the filtrate was concentrated <i>in vacuo</i>. The resultant solid was dissolved in EtOAc (10 mL) and washed with 2 M aq HCl (10 mL). MeOH was added dropwise to dissolve a white precipitate. The organic layer was then washed with brine (10 mL), passed through a hydrophobic frit, and concentrated <i>in vacuo</i>. The resultant solid was dissolved in minimal DCM and purified by silica chromatography (0–50% EtOAc in cyclohexane). The desired fractions were concentrated <i>in vacuo</i>, yielding 5-chloro-4-isopropyl-2-methylpyridazin-3(2<i>H</i>)-one (26 mg, 0.139 mmol, 19%) as a colorless oil. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 7.94 (s, 1H), 3.63 (s, 3H), 3.38 (sept., <i>J</i> = 7.1 Hz, 1H), 1.28 (d, <i>J</i> = 7.1 Hz, 6H); <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 158.9, 142.8, 136.8, 134.7, 29.3, 18.7 (N.B., one sp<sup>3</sup> carbon signal obscured by solvent peak at 40.2 ppm); HRMS (M + H)<sup>+</sup> calculated for C<sub>8</sub>H<sub>12</sub>ClN<sub>2</sub>O, 187.0638; found, 187.0638 LC/MS (formic acid): <i>R</i><sub>t</sub> = 0.97 min (100%); [M + H]<sup>+</sup> = 187.</div></div><div id="sec4_27" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> 4-Isopropyl-2-methyl-5-((2-(4-methylpiperazin-1-yl)benzyl)amino)pyridazin-3(2<i>H</i>)-one (<b>30</b>)</h3><div class="NLM_p last">(2-(4-Methylpiperazin-1-yl)phenyl)methanamine (0.09 mL, 0.48 mmol), 5-chloro-4-isopropyl-2-methylpyridazin-3(2<i>H</i>)-one (75 mg, 0.40 mmol), sodium <i>tert</i>-butoxide (77 mg, 0.80 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (37 mg, 0.04 mmol), and BrettPhos (43 mg, 0.08 mmol) were dissolved in THF (4 mL). The reaction mixture was stirred at 100 °C for 1 h in a microwave reactor. The resultant solution was allowed to cool to rt, diluted with EtOAc (10 mL), filtered through Celite, and concentrated <i>in vacuo</i>. The resultant residue was dissolved in EtOAc (10 mL) and washed sequentially with water (15 mL) and brine (15 mL). The organic layer was passed through a hydrophobic frit and concentrated <i>in vacuo</i>. The resultant residue was dissolved in 1:1 MeOH/DMSO and purified by MDAP (high pH). The desired fractions were combined and concentrated <i>in vacuo</i>, yielding 4-isopropyl-2-methyl-5-((2-(4-methylpiperazin-1-yl)benzyl)amino)pyridazin-3(2<i>H</i>)-one (51 mg, 0.14 mmol, 36%) as a white solid. m.p. 163–165 °C; ν<sub>max</sub> (solid)/cm<sup>–1</sup>: 3308, 2794, 1591 (C═O), 1450, 1141, 772; <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ ppm 7.52 (s, 1H), 7.33 (dd, <i>J</i> = 7.7, 1.1 Hz, 1H), 7.29–7.23 (m, 1H), 7.23–7.19 (dd, <i>J</i> = 7.3, 1.2 Hz, 1H), 7.10 (td, <i>J</i> = 7.3, 1.2 Hz, 1H), 4.55 (s, 2H), 3.58 (s, 3H), 3.23–3.14 (m, 1H), 3.01 (t, <i>J</i> = 4.8 Hz, 4H), 2.69 (app. br. s, 4H), 2.40 (s, 3H), 1.35 (d, <i>J</i> = 7.1 Hz, 6H) (N.B., exchangeable amine proton not visible); <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>OD) δ ppm 161.3, 150.5, 145.2, 133.4, 128.6, 128.1, 128.0, 124.2, 119.5, 116.6, 55.1, 52.1, 44.8, 41.1, 38.7, 24.6, 17.8; HRMS (M + H)<sup>+</sup> calculated for C<sub>20</sub>H<sub>30</sub>N<sub>5</sub>O, 356.2450; found, 356.2449.; LC/MS (high pH): <i>R</i><sub>t</sub> = 0.92 min (98%); [M + H]<sup>+</sup> = 356.</div></div><div id="sec4_28" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> 4-Butyl-5-chloropyridazin-3(2<i>H</i>)-one (<b>48</b>)</h3><div class="NLM_p last">4,5-Dichloropyridazin-3(2<i>H</i>)-one (1000 mg, 6.06 mmol) was dissolved in THF (30 mL) at rt under nitrogen. The solution was allowed to cool down to −78 °C before 1.6 M BuLi in hexanes (8.90 mL, 14.24 mmol) was added dropwise over 10 min. The resultant solution was stirred at −78 °C for 15 min. The solution was diluted with IPA (20 mL) and allowed to warm to rt before being concentrated <i>in vacuo</i>. The resultant residue was dissolved in EtOAc (40 mL) and washed with water (40 mL). The layers were separated and the aqueous phase was extracted further with EtOAc (30 mL). The combined organic fractions were washed with brine (30 mL) and passed through a hydrophobic frit before being concentrated <i>in vacuo</i>. The resultant residue was dissolved in minimal DCM and purified by silica chromatography (0–30% EtOAc in cyclohexane). The desired fractions were combined, yielding 4-butyl-5-chloropyridazin-3(2<i>H</i>)-one (715 mg, 3.83 mmol, 63%) as an off-white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm 11.88 (br. s, 1H), 7.76 (s, 1H), 2.80–2.69 (m, 2H), 1.64–1.54 (m, 2H), 1.50–1.38 (m, 2H), 0.97 (t, <i>J</i> = 7.2 Hz, 3H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ ppm 161.8, 141.6, 138.1, 137.3, 28.9, 27.0, 22.8, 13.8; HRMS (M + H)<sup>+</sup> calculated for C<sub>8</sub>H<sub>12</sub>ClN<sub>2</sub>O, 187.0638; found, 187.0644; LC/MS (formic acid): <i>R</i><sub>t</sub> = 0.90 min (100%); [M + H]<sup>+</sup> = 187.</div></div><div id="sec4_29" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> 4-Butyl-5-chloro-2-methylpyridazin-3(2<i>H</i>)-one (<b>49</b>)</h3><div class="NLM_p last">Methyl iodide (0.36 mL, 5.75 mmol) was added to a stirred mixture of tetrabutylammonium bromide (2.099 g, 6.51 mmol), 4-butyl-5-chloropyridazin-3(2<i>H</i>)-one (715 mg, 3.83 mmol), and potassium carbonate (1.059 g, 7.66 mmol) in MeCN (8 mL) at rt. The resultant solution was heated to 60 °C and stirred for 5 h. The resultant mixture was allowed to cool down to rt before being filtered under reduced pressure and the filtrate was concentrated <i>in vacuo</i>. The resultant solid was dissolved in EtOAc (30 mL) and washed with 1 M aq HCl (30 mL). The organic layer was then washed with brine (20 mL), passed through a hydrophobic frit, and concentrated <i>in vacuo</i>. The resultant solid was dissolved in minimal DCM and purified by silica chromatography (0–25% EtOAc in cyclohexane). The desired fractions were concentrated <i>in vacuo</i>, yielding 4-butyl-5-chloro-2-methylpyridazin-3(2<i>H</i>)-one (593 mg, 2.96 mmol, 77%) as a yellow oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm 7.68 (s, 1H), 3.77 (s, 3H), 2.79–2.68 (m, 2H), 1.60–1.52 (m, 2H), 1.49–1.37 (m, 2H), 0.97 (t, <i>J</i> = 7.3 Hz, 3H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ ppm 160.2, 140.6, 136.3, 135.9, 40.2, 28.9, 27.5, 22.8, 13.8; HRMS (M + H)<sup>+</sup> calculated for C<sub>9</sub>H<sub>14</sub>ClN<sub>2</sub>O, 201.0795; found, 201.0802; LC/MS (formic acid): <i>R</i><sub>t</sub> = 1.07 min (95%); [M + H]<sup>+</sup> = 201.</div></div><div id="sec4_30" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> 4-Butyl-2-methyl-5-((2-(4-methylpiperazin-1-yl)benzyl)amino)pyridazin-3(2<i>H</i>)-one (<b>31</b>)</h3><div class="NLM_p last">(2-(4-Methylpiperazin-1-yl)phenyl)methanamine (0.06 mL, 0.30 mmol), 4-butyl-5-chloro-2-methylpyridazin-3(2<i>H</i>)-one (50 mg, 0.25 mmol), sodium <i>tert</i>-butoxide (48 mg, 0.50 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (23 mg, 0.03 mmol), and BrettPhos (27 mg, 0.05 mmol) were dissolved in THF (0.5 mL). The reaction mixture was heated to 100 °C in a microwave reactor and stirred for 1 h. The resultant solution was allowed to cool down to rt, diluted with EtOAc (10 mL), filtered through Celite, and concentrated <i>in vacuo</i>. The resultant residue was dissolved in EtOAc (10 mL) and washed sequentially with water (10 mL) and brine (10 mL). The organic layer was passed through a hydrophobic frit and concentrated <i>in vacuo</i>. The resultant residue was dissolved in 1:1 MeOH/DMSO and purified by MDAP (high pH). The desired fractions were combined and concentrated <i>in vacuo</i>, yielding 4-butyl-2-methyl-5-((2-(4-methylpiperazin-1-yl)benzyl)amino)pyridazin-3(2<i>H</i>)-one (20 mg, 0.05 mmol, 22%) as an orange oil. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ ppm 7.55 (s, 1H), 7.36–7.31 (m, 1H), 7.30–7.24 (m, 1H), 7.21 (dd, <i>J</i> = 7.3, 1.2 Hz, 1H), 7.09 (td, <i>J</i> = 7.3, 1.2 Hz, 1H), 4.56 (s, 2H), 3.62 (s, 3H), 3.02 (t, <i>J</i> = 4.9 Hz, 4H), 2.73 (app. br. s, 4H), 2.55 (t, <i>J</i> = 7.2 Hz, 2H), 2.42 (s, 3H), 1.56–1.38 (m, 4H), 0.98 (t, <i>J</i> = 7.2 Hz, 3H) (N.B., exchangeable amine proton not visible); <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>OD) δ ppm 161.9, 150.5, 145.8, 133.4, 128.4, 128.0, 127.9, 124.1, 119.5, 112.6, 55.2, 52.1, 44.8, 40.9, 38.8, 28.8, 26.6, 22.5, 13.1; HRMS (M + H)<sup>+</sup> calculated for C<sub>21</sub>H<sub>32</sub>N<sub>5</sub>O, 370.2607; found, 370.2607; LC/MS (high pH): <i>R</i><sub>t</sub> = 1.01 min (100%); [M + H]<sup>+</sup> = 370.</div></div><div id="sec4_31" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> 2-Methyl-5-((2-(4-methylpiperazin-1-yl)benzyl)amino)-4-propylpyridazin-3(2<i>H</i>)-one (<b>29</b>)</h3><div class="NLM_p last">The solvent system was sparged with nitrogen for 1 h prior to use. 4-Chloro-2-methyl-5-((2-(4-methylpiperazin-1-yl)benzyl)amino)pyridazin-3(2<i>H</i>)-one (100 mg, 0.29 mmol), propylboronic acid (100 mg, 1.14 mmol), potassium carbonate (99 mg, 0.72 mmol), Pd(OAc)<sub>2</sub> (26 mg, 0.12 mmol), and butyldi-1-adamantylphosphine (52 mg, 0.14 mmol) were dissolved in in 1,4-dioxane (1.65 mL) and water (0.83 mL) under nitrogen at rt. The resultant mixture was heated to 100 °C and stirred for 1 h in a microwave reactor. The resultant solution was diluted with EtOAc (10 mL) and filtered through Celite before being concentrated <i>in vacuo</i>. The resultant residue was dissolved in 1:1 MeOH/DMSO and purified by MDAP (high pH). The desired fractions were combined, yielding 2-methyl-5-((2-(4-methylpiperazin-1-yl)benzyl)amino)-4-propylpyridazin-3(2<i>H</i>)-one (30 mg, 0.08 mmol, 29%) as an off-white solid. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ ppm 7.55 (s, 1H), 7.37–7.31 (m, 1H), 7.29–7.24 (m, 1H), 7.23–7.17 (m, 1H), 7.09 (td, <i>J</i> = 7.5, 1.2 Hz, 1H), 4.55 (s, 2H), 3.62 (s, 3H), 3.01 (t, <i>J</i> = 4.8 Hz, 4H), 2.71 (app. br. s, 4H), 2.57–2.48 (m, 2H), 2.40 (s, 3H), 1.61–1.46 (m, 2H), 1.02 (t, <i>J</i> = 7.3 Hz, 3H) (N.B., exchangeable amine proton not visible); <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>OD) δ ppm 162.0, 150.5, 145.9, 133.4, 128.4, 128.0, 127.9, 124.1, 119.5, 112.4, 55.1, 52.1, 44.8, 40.8, 38.8, 24.8, 19.7, 13.0; HRMS (M + H)<sup>+</sup> calculated for C<sub>20</sub>H<sub>30</sub>N<sub>5</sub>O, 356.2450; found, 356.2442; LC/MS (high pH): <i>R</i><sub>t</sub> = 0.92 min (100%); [M + H]<sup>+</sup> = 356.</div></div><div id="sec4_32" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49"> 2-Methyl-5-((2-(4-methylpiperazin-1-yl)benzyl)amino)-4-pentylpyridazin-3(2<i>H</i>)-one (<b>32</b>)</h3><div class="NLM_p last">The solvent system sparged with nitrogen for 1 h prior to use. 4-Chloro-2-methyl-5-((2-(4-methylpiperazin-1-yl)benzyl)amino)pyridazin-3(2<i>H</i>)-one (100 mg, 0.29 mmol), pentylboronic acid (100 mg, 0.86 mmol), potassium carbonate (99 mg, 0.72 mmol), Pd(OAc)<sub>2</sub> (26 mg, 0.12 mmol), and butyldi-1-adamantylphosphine (52 mg, 0.14 mmol) were dissolved in in 1,4-dioxane (1.65 mL) and water (0.83 mL) under nitrogen at rt. The reaction mixture was heated to 100 °C and stirred for 1 h in a microwave reactor. The resultant solution was diluted with EtOAc (5 mL) and filtered through Celite before being concentrated <i>in vacuo</i>. The resultant residue was dissolved in 1:1 MeOH/DMSO and purified by MDAP (high pH). The desired fractions were combined, yielding 2-methyl-5-((2-(4-methylpiperazin-1-yl)benzyl)amino)-4-pentylpyridazin-3(2<i>H</i>)-one (28 mg, 0.07 mmol, 25%) as an off-white solid. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ ppm 7.56 (s, 1H), 7.36–7.31 (m, 1H), 7.30–7.24 (m, 1H), 7.20 (dd, <i>J</i> = 7.5, 1.2 Hz, 1H), 7.09 (td, <i>J</i> = 7.5, 1.2 Hz, 1H), 4.55 (s, 2H), 3.62 (s, 3H), 3.01 (t, <i>J</i> = 4.8 Hz, 4H), 2.71 (app. br. s, 4H), 2.58–2.49 (m, 2H), 2.40 (s, 3H), 1.57–1.46 (m, 2H), 1.44–1.34 (m, 4H), 0.97–0.91 (m, 3H); <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>OD) δ ppm 162.0, 150.5, 145.8, 133.4, 128.4, 128.0, 127.9, 124.1, 119.5, 112.7, 55.2, 52.2, 44.8, 40.9, 38.8, 31.6, 26.3, 22.9, 22.4, 13.0; HRMS (M + H)<sup>+</sup> calculated for C<sub>22</sub>H<sub>34</sub>N<sub>5</sub>O, 384.2763; found, 384.2756; LC/MS (high pH): <i>R</i><sub>t</sub> = 1.08 min (96%); [M + H]<sup>+</sup> = 384.</div></div><div id="sec4_33" class="NLM_sec NLM_sec_level_2"><div id="ac_i50" class="anchor-spacer"></div><h3 class="article-section__title" id="_i50"> 2-Methyl-5-((2-(4-methylpiperazin-1-yl)benzyl)amino)pyridazin-3(2<i>H</i>)-one (<b>33</b>)</h3><div class="NLM_p last">The solvent system sparged with nitrogen for 20 min prior to use. 4-Chloro-2-methyl-5-((2-(4-methylpiperazin-1-yl)benzyl)amino)pyridazin-3(2<i>H</i>)-one (100 mg, 0.29 mmol), butyltrifluoro-l4-borane, potassium salt (94 mg, 0.58 mmol), potassium carbonate (99 mg, 0.72 mmol), Pd(OAc)<sub>2</sub> (13 mg, 0.06 mmol), and butyldi-1-adamantylphosphine (22 mg, 0.06 mmol) were dissolved in in 1,4-dioxane (3 mL) and water (1.5 mL) under nitrogen at rt. The resultant mixture was heated to 100 °C and stirred for 1 h in a microwave reactor. The resultant solution was diluted with EtOAc (5 mL) and filtered through Celite before being concentrated <i>in vacuo</i>. The resultant residue was dissolved in EtOAc (10 mL) and washed sequentially with water (10 mL) and brine (10 mL). The organic layer was then passed through a hydrophobic frit and concentrated <i>in vacuo</i>. The resultant residue was dissolved in 1:1 MeOH/DMSO and purified by MDAP (high pH). The desired fractions were combined, yielding 2-methyl-5-((2-(4-methylpiperazin-1-yl)benzyl)amino)pyridazin-3(2<i>H</i>)-one (39 mg, 0.12 mmol, 43%) as a white solid. m.p. 190–192 °C; ν<sub>max</sub> (solid)/cm<sup>–1</sup>: 3245, 2941, 2790, 1591 (C═O), 1450, 761; <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ ppm 7.57 (d, <i>J</i> = 2.7 Hz, 1H), 7.37–7.32 (m, 1H), 7.31–7.25 (m, 1H), 7.22 (dd, <i>J</i> = 7.5, 1.2 Hz, 1H), 7.11 (td, <i>J</i> = 7.5, 1.2 Hz, 1H), 5.62 (d, <i>J</i> = 2.7 Hz, 1H), 4.39 (s, 2H), 3.61 (s, 3H), 2.99 (t, <i>J</i> = 4.8 Hz, 4H), 2.68 (app. br. s, 4H), 2.38 (s, 3H); HRMS (M + H)<sup>+</sup> calculated for C<sub>17</sub>H<sub>24</sub>N<sub>5</sub>O, 314.1981; found, 314.1986; LC/MS (high pH): <i>R</i><sub>t</sub> = 0.76 min (100%); [M + H]<sup>+</sup> = 314.</div></div><div id="sec4_34" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> <i>tert</i>-Butyl 5-((5-butyl-1-methyl-6-oxo-1,6-dihydropyridazin-4-yl)amino)-3,4-dihydroisoquinoline-2(1<i>H</i>)-carboxylate (<b>50</b>)</h3><div class="NLM_p last"><i>tert</i>-Butyl 5-amino-3,4-dihydroisoquinoline-2(1<i>H</i>)-carboxylate (515 mg, 2.07 mmol), 4-butyl-5-chloro-2-methylpyridazin-3(2<i>H</i>)-one (320 mg, 1.60 mmol), sodium <i>tert</i>-butoxide (307 mg, 3.19 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (118 mg, 0.16 mmol), and BrettPhos (171 mg, 0.32 mmol) were dissolved in THF (10 mL). The reaction mixture was stirred at 100 °C for 1 h in a microwave reactor. The resultant solution was allowed to cool down to rt, diluted with EtOAc (25 mL), filtered through Celite, and concentrated <i>in vacuo</i>. The resultant residue was dissolved in EtOAc (30 mL) and washed sequentially with water (30 mL) and brine (20 mL). The organic layer was passed through a hydrophobic frit and concentrated <i>in vacuo</i>. The resultant residue was dissolved in minimal DCM and purified by silica chromatography (0–50% EtOAc in cyclohexane). The desired fractions were combined, yielding <i>tert</i>-butyl 5-((5-butyl-1-methyl-6-oxo-1,6-dihydropyridazin-4-yl)amino)-3,4-dihydroisoquinoline-2(1<i>H</i>)-carboxylate (167 mg, 0.41 mmol, 25%) as a pale orange solid; m.p. 60–65 °C; ν<sub>max</sub> (solid)/cm<sup>–1</sup>: 3295, 2930, 1694 (C═O), 1584 (C═O), 1395, 1163, 769; <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ ppm 7.29 (t, <i>J</i> = 7.6 Hz, 1H), 7.22 (s, 1H), 7.15 (d, <i>J</i> = 7.6 Hz, 1H), 7.09 (d, <i>J</i> = 7.6 Hz, 1H), 4.62 (s, 2H), 3.68 (s, 3H), 3.64 (t, <i>J</i> = 5.9 Hz, 2H), 2.74 (t, <i>J</i> = 5.9 Hz, 2H), 2.68–2.61 (m, 2H), 1.61–1.40 (m, 13H), 0.98 (t, <i>J</i> = 7.2 Hz, 3H) (N.B., exchangeable amine proton not visible); <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>OD) δ ppm 162.2, 155.1, 144.6, 137.4, 135.5, 131.9, 129.3, 126.9, 124.9, 124.6, 115.1, 80.1, 38.9, 28.9, 27.3, 26.6, 24.4, 22.9, 22.4, 13.1 (N.B., one sp<sup>3</sup> carbon missing); HRMS (M + H)<sup>+</sup> calculated for C<sub>23</sub>H<sub>33</sub>N<sub>4</sub>O<sub>3</sub>, 413.2555; found, 413.2557; LC/MS (formic acid): <i>R</i><sub>t</sub> = 1.24 min (98%); [M + H]<sup>+</sup> = 413.</div></div><div id="sec4_35" class="NLM_sec NLM_sec_level_2"><div id="ac_i52" class="anchor-spacer"></div><h3 class="article-section__title" id="_i52"> 4-Butyl-2-methyl-5-((1,2,3,4-tetrahydroisoquinolin-5-yl)amino)pyridazin-3(2<i>H</i>)-one (<b>51</b>)</h3><div class="NLM_p last"><i>tert</i>-Butyl 5-((5-butyl-1-methyl-6-oxo-1,6-dihydropyridazin-4-yl)amino)-3,4-dihydroisoquinoline-2(1<i>H</i>)-carboxylate (150 mg, 0.36 mmol) was dissolved in 4 M HCl in 1,4-dioxane (3.00 mL, 12.00 mmol) at rt and stirred for 45 min. The reaction mixture was concentrated <i>in vacuo</i>. The resultant residue was dissolved in MeOH and passed through a preconditioned (MeOH) amino propyl column (1 g) and eluted with MeOH (10 mL). The desired fractions were concentrated <i>in vacuo</i>, yielding 4-butyl-2-methyl-5-((1,2,3,4-tetrahydroisoquinolin-5-yl)amino)pyridazin-3(2<i>H</i>)-one (114 mg, 0.36 mmol, 100%) as a yellow solid. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ ppm 7.28 (t, <i>J</i> = 7.6 Hz, 1H), 7.23 (s, 1H), 7.14–7.08 (m, 2H), 4.14 (s, 2H), 3.68 (s, 3H), 3.22 (t, <i>J</i> = 6.1 Hz, 2H), 2.79 (t, <i>J</i> = 6.1 Hz, 2H), 2.69–2.61 (m, 2H), 1.61–1.51 (m, 2H), 1.51–1.41 (m, 2H), 0.98 (t, <i>J</i> = 7.2 Hz, 3H) (N.B., exchangeable amine protons not visible); <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>OD) δ ppm 162.2, 144.6, 137.7, 135.0, 131.2, 129.3, 126.8, 125.3, 124.9, 115.2, 46.4, 42.3, 39.0, 29.0, 23.5, 23.0, 22.4, 13.1; HRMS (M + H)<sup>+</sup> calculated for C<sub>18</sub>H<sub>25</sub>N<sub>4</sub>O, 313.2028; found, 313.2033; LC/MS (formic acid): <i>R</i><sub>t</sub> = 0.48 min (100%); [M + H]<sup>+</sup> = 313.</div></div><div id="sec4_36" class="NLM_sec NLM_sec_level_2"><div id="ac_i53" class="anchor-spacer"></div><h3 class="article-section__title" id="_i53"> 4-Butyl-2-methyl-5-((2-methyl-1,2,3,4-tetrahydroisoquinolin-5-yl)amino)pyridazin-3(2<i>H</i>)-one (<b>35</b>)</h3><div class="NLM_p last">Formaldehyde (37%) in water with 10–15% MeOH (0.25 mL, 3.36 mmol) and formic acid (1 mL, 26.10 mmol) were added to 4-butyl-2-methyl-5-((1,2,3,4-tetrahydroisoquinolin-5-yl)amino)pyridazin-3(2<i>H</i>)-one (105 mg, 0.34 mmol) in a sealed tube and heated to 80 °C for 6 h. The reaction mixture was allowed to cool down to rt and concentrated <i>in vacuo</i>. The resultant residue was dissolved in minimal DCM and purified by silica chromatography (0–10% MeOH in DCM). The desired fractions were combined and concentrated <i>in vacuo</i>. The resultant solid was dissolved in MeOH (1.5 mL) and loaded on to an SCX (1 g) column, eluting with 2 M ammonia in MeOH solution (4 mL). The desired fractions were combined, yielding 4-butyl-2-methyl-5-((2-methyl-1,2,3,4-tetrahydroisoquinolin-5-yl)amino)pyridazin-3(2<i>H</i>)-one (65 mg, 0.20 mmol, 59%) as a white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm 7.44 (s, 1H), 7.18 (t, <i>J</i> = 7.8 Hz, 1H), 7.00 (d, <i>J</i> = 7.8 Hz, 1H), 6.95 (d, <i>J</i> = 7.8 Hz, 1H), 5.41 (s, 1H), 3.73 (s, 3H), 3.62 (s, 2H), 2.79–2.71 (m, 4H), 2.66–2.59 (m, 2H), 2.49 (s, 3H), 1.63–1.54 (m, 2H), 1.52–1.45 (m, 2H), 0.99 (t, <i>J</i> = 7.3 Hz, 3H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ ppm 161.5, 142.4, 137.2, 137.1, 128.9, 128.7, 126.6, 124.0, 121.9, 117.7, 58.0, 52.5, 45.9, 39.8, 29.2, 25.5, 23.6, 23.0, 14.0; HRMS (M + H)<sup>+</sup> calculated for C<sub>19</sub>H<sub>27</sub>N<sub>4</sub>O, 327.2185; found, 327.2189; LC/MS (formic acid): <i>R</i><sub>t</sub> = 0.49 min (100%); [M + H]<sup>+</sup> = 327.</div></div><div id="sec4_37" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> 2-Bromo-5-butylthieno[3,2-<i>c</i>]pyridin-4(5<i>H</i>)-one</h3><div class="NLM_p last">To a solution of 2-bromothieno[3,2-<i>c</i>]pyridin-4(5<i>H</i>)-one (<b>50</b>) (1 g, 4.35 mmol) and cesium carbonate (4.25 g, 13.04 mmol) in THF (25 mL), 1-iodobutane (0.72 mL, 6.52 mmol) was added in a single portion and the mixture was heated to 60 °C for 19 h. The reaction mixture was allowed to cool down to room temperature and concentrated <i>in vacuo</i>. The resultant solid was suspended in water (15 mL) and filtered under reduced pressure. The solid was washed with water (10 mL), collected, and dried under vacuum at 40 °C, yielding 2-bromo-5-butylthieno[3,2-<i>c</i>]pyridin-4(5<i>H</i>)-one (884 mg, 3.09 mmol, 71%) as a brown solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm 7.63 (s, 1H), 7.13 (d, <i>J</i> = 7.1 Hz, 1H), 6.56 (d, <i>J</i> = 7.1 Hz, 1H), 4.01 (t, <i>J</i> = 7.4 Hz, 2H), 1.83–1.70 (m, 2H), 1.41 (sixt., <i>J</i> = 7.4 Hz, 2H), 0.98 (t, <i>J</i> = 7.4 Hz, 3H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ ppm 157.6, 148.4, 132.6, 131.3, 127.7, 112.2, 100.8, 49.2, 31.6, 19.9, 13.7; HRMS (M + H)<sup>+</sup> calculated for C<sub>11</sub>H<sub>13</sub>BrNOS, 285.9901; found, 285.9910; LC/MS (formic acid): <i>R</i><sub>t</sub> = 1.13 min (97%); [M + H]<sup>+</sup> = 286.</div></div><div id="sec4_38" class="NLM_sec NLM_sec_level_2"><div id="ac_i55" class="anchor-spacer"></div><h3 class="article-section__title" id="_i55"> 5-Butyl-4-oxo-4,5-dihydrothieno[3,2-<i>c</i>]pyridine-2-carbonitrile (<b>51</b>)</h3><div class="NLM_p last">Two batches of 2-bromo-5-butylthieno[3,2-<i>c</i>]pyridin-4(5<i>H</i>)-one (1 g, 3.50 mmol), Zn(CN)<sub>2</sub> (821 mg, 6.99 mmol), and Pd(PPh<sub>3</sub>)<sub>4</sub> (404 mg, 0.35 mmol) in DMF (5 mL) were heated to 115 °C in a microwave reactor for 4.5 h. The batches were diluted with EtOAc (50 mL each), combined, filtered through Celite, and concentrated <i>in vacuo</i>. The resulting residue was diluted with DCM and filtered under reduced pressure, and the filtrate was concentrated <i>in vacuo</i>. The resultant residue was dissolved in minimal DCM and purified by silica gel chromatography (0–40% EtOAc in cyclohexane). The desired fractions were combined and concentrated <i>in vacuo</i>, yielding 5-butyl-4-oxo-4,5-dihydrothieno[3,2-<i>c</i>]pyridine-2-carbonitrile (984 mg, 4.24 mmol, 61%) as a white solid. m.p. 95–99 °C; ν<sub>max</sub> (solid)/cm<sup>–1</sup>: 3103, 2960, 2871, 2209 (C≡N), 1631 (C═O), 1588, 769; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm 8.15 (s, 1H), 7.32 (d, <i>J</i> = 7.1 Hz, 1H), 6.65 (d, <i>J</i> = 7.1 Hz, 1H), 4.06–3.99 (m, 2H), 1.82–1.72 (m, 2H), 1.41 (sixt., <i>J</i> = 7.3 Hz, 2H), 0.99 (t, <i>J</i> = 7.3 Hz, 3H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ ppm 158.1, 150.6, 136.3, 135.9, 130.0, 113.8, 107.6, 100.6, 49.2, 31.5, 19.9, 13.7; HRMS (M + H)<sup>+</sup> calculated for C<sub>12</sub>H<sub>13</sub>N<sub>2</sub>OS, 233.0749; found, 233.0754; LC/MS (formic acid): <i>R</i><sub>t</sub> = 0.98 min (93%); [M + H]<sup>+</sup> = 233.</div></div><div id="sec4_39" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> 7-Bromo-5-butyl-4-oxo-4,5-dihydrothieno[3,2-<i>c</i>]pyridine-2-carbonitrile</h3><div class="NLM_p last">To a stirred solution of 5-butyl-4-oxo-4,5-dihydrothieno[3,2-<i>c</i>]pyridine-2-carbonitrile (984 mg, 4.24 mmol) in THF (20 mL) was added 1-bromopyrrolidine-2,5-dione (1131 mg, 6.35 mmol) at rt. The resultant solution was stirred at rt for 65 h before being concentrated <i>in vacuo</i>. The resultant solid was triturated with diethyl ether (15 mL) and filtered under reduced pressure. The collected solid was washed sequentially with diethyl ether (10 mL) and water (20 mL) before being dried, yielding 7-bromo-5-butyl-4-oxo-4,5-dihydrothieno[3,2-<i>c</i>]pyridine-2-carbonitrile (816 mg, 2.62 mmol, 62%) as a cream solid. The filtrate was diluted with EtOAc (25 mL) and washed sequentially with water (20 mL) and brine (20 mL). The organic layer was passed through a hydrophobic frit and concentrated <i>in vacuo</i>. The resultant solid was dissolved in minimal DCM and purified by silica chromatography (0–30% EtOAc in cyclohexane). The desired fractions were combined and concentrated <i>in vacuo</i>, yielding 7-bromo-5-butyl-4-oxo-4,5-dihydrothieno[3,2-<i>c</i>]pyridine-2-carbonitrile (313 mg, 1.006 mmol, 24%) as a cream solid. Both solids were combined, yielding 7-bromo-5-butyl-4-oxo-4,5-dihydrothieno[3,2-<i>c</i>]pyridine-2-carbonitrile (1129 mg, 3.63 mmol, 86%) as a cream solid. m.p. 115–116 °C; ν<sub>max</sub> (solid)/cm<sup>–1</sup>: 3043, 2957, 2872, 2212 (C≡N), 1651 (C═O), 1582, 764; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm 8.23 (s, 1H), 7.45 (s, 1H), 4.02 (t, <i>J</i> = 7.3 Hz, 2H), 1.83–1.72 (m, 2H), 1.42 (sixt., <i>J</i> = 7.3 Hz, 2H), 1.00 (t, <i>J</i> = 7.3 Hz, 3H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ ppm 160.8, 156.3, 140.8, 139.5, 133.2, 117.2, 112.6, 95.6, 53.3, 35.3, 23.6, 17.4; HRMS (M + H)<sup>+</sup> calculated for C<sub>12</sub>H<sub>12</sub>BrN<sub>2</sub>OS, 310.9854; found, 310.9855; LC/MS (formic acid): <i>R</i><sub>t</sub> = 1.18 min (95%); [M + H]<sup>+</sup> = 312.</div></div><div id="sec4_40" class="NLM_sec NLM_sec_level_2"><div id="ac_i57" class="anchor-spacer"></div><h3 class="article-section__title" id="_i57"> Methyl 7-bromo-5-butyl-4-oxo-4,5-dihydrothieno[3,2-<i>c</i>]pyridine-2-carbimidate (<b>52</b>)</h3><div class="NLM_p last">To a suspension of 7-bromo-5-butyl-4-oxo-4,5-dihydrothieno[3,2-<i>c</i>]pyridine-2-carbonitrile (1106 mg, 3.55 mmol) in MeOH (35 mL) was added sodium methoxide (25 wt % in MeOH) solution (0.813 mL, 3.55 mmol). The reaction mixture was heated to 75 °C when solvation was achieved. After 15 min, 4-aminotetrahydro-2<i>H</i>-thiopyran 1,1-dioxide (499 mg, 3.34 mmol) was added and the solution was heated for a further 18 h. A further equivalent of sodium methoxide (25% by weight solution in MeOH) (0.813 mL, 3.55 mmol) was added at rt before the reaction was heated to 85 °C and stirred for 3 h. No product was observed. The reaction was cooled down to 75 °C and a portion of 4-aminotetrahydro-2<i>H</i>-thiopyran 1,1-dioxide, hydrochloride (125 mg, 0.67 mmol), was added. The reaction was stirred at 75 °C for 5 h. The solution was allowed to cool down to rt before being concentrated <i>in vacuo</i>. The resultant residue was dry-loaded and purified by silica chromatography (0–75% EtOAc in cyclohexane). The desired fractions were combined, yielding methyl 7-bromo-5-butyl-4-oxo-4,5-dihydrothieno[3,2-<i>c</i>]pyridine-2-carbimidate (326 mg, 0.95 mmol, 27%) as a white solid. m.p. 152–154 °C; ν<sub>max</sub> (solid)/cm<sup>–1</sup>: 3295, 3077, 2953, 1648 (C═O), 1578, 1314, 1129, 702; <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ ppm 8.23 (s, 1H), 7.84 (s, 1H), 4.07 (t, <i>J</i> = 7.5 Hz, 2H), 3.97–3.87 (m, 3H), 1.82–1.71 (m, 2H), 1.41 (sixt., <i>J</i> = 7.3 Hz, 2H), 1.00 (t, <i>J</i> = 7.3 Hz, 3H); <sup>13</sup>C NMR (176 MHz, CD<sub>3</sub>OD) δ ppm 163.4, 159.8, 152.7, 136.5, 136.4, 135.7, 130.9, 128.7, 94.5, 50.6, 32.7, 21.0, 14.2; HRMS (M + H)<sup>+</sup> calculated for C<sub>13</sub>H<sub>16</sub>BrN<sub>2</sub>O<sub>2</sub>S, 343.0116; found, 343.0119; LC/MS (formic acid): <i>R</i><sub>t</sub> = 0.92 min (97%); [M + H]<sup>+</sup> = 343.</div></div><div id="sec4_41" class="NLM_sec NLM_sec_level_2"><div id="ac_i58" class="anchor-spacer"></div><h3 class="article-section__title" id="_i58"> 7-Bromo-5-butyl-<i>N</i>-(1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)-4-oxo-4,5-dihydrothieno[3,2-<i>c</i>]pyridine-2-carboximidamide</h3><div class="NLM_p last">Triethylamine (0.18 mL, 1.30 mmol) was added to a stirred solution of 4-aminotetrahydro-2<i>H</i>-thiopyran 1,1-dioxide (168 mg, 1.13 mmol) and methyl 7-bromo-5-butyl-4-oxo-4,5-dihydrothieno[3,2-<i>c</i>]pyridine-2-carbimidate (297 mg, 0.87 mmol) in DMF (5 mL). The resultant solution was heated to 120 °C and stirred for 17 h. The resultant solution was allowed to cool down to rt before being diluted with water (35 mL). The resultant precipitate was collected under reduced pressure, dissolved in minimal DCM, and purified by silica chromatography (0–50% MeOH (25%) in DCM in cyclohexane). The desired fractions were combined and concentrated <i>in vacuo</i>, yielding 7-bromo-5-butyl-<i>N</i>-(1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)-4-oxo-4,5-dihydrothieno[3,2-<i>c</i>]pyridine-2-carboximidamide (83 mg, 0.18 mmol, 21%) as a pale orange solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm 7.91 (s, 1H), 7.33 (s, 1H), 5.01 (br. s, 1H), 4.01 (t, <i>J</i> = 7.3 Hz, 2H), 3.77–3.53 (m, 2H), 3.05–2.88 (m, 2H), 2.49–2.33 (m, 2H), 2.26–2.12 (m, 2H), 1.80–1.72 (quint., <i>J</i> = 7.3 Hz, 2H), 1.49–1.36 (m, 2H), 0.99 (t, <i>J</i> = 7.3 Hz, 3H) (N.B., exchangeable amidine protons not visible); HRMS (M + H)<sup>+</sup> calculated for C<sub>17</sub>H<sub>23</sub>BrN<sub>3</sub>O<sub>3</sub>S<sub>2</sub>, 460.0364; found, 460.0368; LC/MS (formic acid): <i>R</i><sub>t</sub> = 0.56 min (91%); [M + H]<sup>+</sup> = 460.</div></div><div id="sec4_42" class="NLM_sec NLM_sec_level_2"><div id="ac_i59" class="anchor-spacer"></div><h3 class="article-section__title" id="_i59"> 5-Butyl-<i>N</i>-(1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)-4-oxo-7-(3-(trifluoromethyl)phenyl)-4,5-dihydrothieno[3,2-<i>c</i>]pyridine-2-carboximidamide (<b>36</b>)</h3><div class="NLM_p last">7-Bromo-5-butyl-<i>N</i>-(1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)-4-oxo-4,5-dihydrothieno[3,2-<i>c</i>]pyridine-2-carboximidamide (73 mg, 0.16 mmol), (3-(trifluoromethyl)phenyl)boronic acid (36 mg, 0.19 mmol), potassium carbonate (53 mg, 0.38 mmol), and PEPPSI-<sup>i</sup>Pr (10 mg, 0.01 mmol) were dissolved in water (0.13 mL) and IPA (0.38 mL). The resultant mixture was heated to 120 °C and stirred for 30 min in a microwave reactor. The reaction mixture was allowed to cool down to rt before being diluted with EtOAc (10 mL), filtered through Celite, and concentrated <i>in vacuo</i>. The resultant residue was dissolved in 1:1 MeOH/DMSO and purified by MDAP (high pH). The desired fractions were combined, yielding 5-butyl-<i>N</i>-(1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)-4-oxo-7-(3-(trifluoromethyl)phenyl)-4,5-dihydrothieno[3,2-<i>c</i>]pyridine-2-carboximidamide (43 mg, 0.08 mmol, 52%) as a white solid. m.p. 121–124 °C; ν<sub>max</sub> (solid)/cm<sup>–1</sup>: 3296, 2957, 1651 (C═O), 1583, 1118; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm 7.97 (s, 1H), 7.88–7.78 (m, 1H), 7.76–7.60 (m, 3H), 7.24 (s, 1H), 4.09 (t, <i>J</i> = 7.3 Hz, 2H), 3.76 (br. s, 1H), 3.57–3.45 (m, 2H), 3.01–2.86 (m, 2H), 2.49–2.29 (m, 2H), 2.26–2.13 (m, 2H), 1.82 (quin., <i>J</i> = 7.3 Hz, 2H), 1.44 (sixt., <i>J</i> = 7.3 Hz, 2H), 1.00 (t, <i>J</i> = 7.3 Hz, 3H) (N.B., exchangeable amidine protons not visible); <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>) δ ppm −62.64 (s); HRMS (M + H)<sup>+</sup> calculated for C<sub>24</sub>H<sub>27</sub>F<sub>3</sub>N<sub>3</sub>O<sub>3</sub>S<sub>2</sub>, 526.1446; found, 526.1448; LC/MS (high pH): <i>R</i><sub>t</sub> = 1.20 min (100%); [M + H]<sup>+</sup> = 526.</div></div><div id="sec4_43" class="NLM_sec NLM_sec_level_2"><div id="ac_i60" class="anchor-spacer"></div><h3 class="article-section__title" id="_i60"> 3-Butyl-6-chloro-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-8-amine (<b>54</b>)</h3><div class="NLM_p last">3,6-Dichloropyridazin-4-amine (<b>53</b>) (3000 mg, 18.29 mmol) was dissolved in hydrazine hydrate (22.25 mL, 274 mmol) at rt under nitrogen. The resultant mixture was heated to 135 °C and stirred under nitrogen for 30 min. The solution was allowed to cool down to rt before being diluted with crushed ice. The precipitate was collected under reduced pressure, washed with ice-cold water, and dried under vacuum at 40 °C, yielding a pale brown solid. The solid was dissolved in pentanoic acid (7.85 mL, 71.40 mmol) at rt under nitrogen. The resultant solution was heated to 100 °C and stirred for 3 h. The resultant solution was allowed to cool down to rt before being diluted with water (75 mL) and EtOAc (75 mL). The layers were separated, and the aqueous layer was extracted with EtOAc (50 mL). The combined organic fractions were washed with brine (75 mL), passed through a hydrophobic frit, and concentrated <i>in vacuo</i>. The resultant residue was loaded on to silica and purified by silica chromatography (0–50% 3:1 EtOAc/EtOH in cyclohexane). The desired fractions were combined, yielding 3-butyl-6-chloro-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-8-amine (1.754 g, 7.77 mmol, 43%) as a yellow solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 7.86 (s, 2H), 6.12 (s, 1H), 2.99 (t, <i>J</i> = 7.6 Hz, 2H), 1.77 (quint., <i>J</i> = 7.6 Hz, 2H), 1.38 (sixt., <i>J</i> = 7.6 Hz, 2H), 0.92 (t, <i>J</i> = 7.6 Hz, 3H); <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 150.3, 149.8, 144.4, 139.8, 94.1, 28.5, 23.7, 22.2, 14.0; HRMS (M + H)<sup>+</sup> calculated for C<sub>9</sub>H<sub>13</sub>ClN<sub>5</sub>, 226.0859; found, 226.0866; LC/MS (formic acid): <i>R</i><sub>t</sub> = 0.89 min (100%); [M + H]<sup>+</sup> = 226.</div></div><div id="sec4_44" class="NLM_sec NLM_sec_level_2"><div id="ac_i61" class="anchor-spacer"></div><h3 class="article-section__title" id="_i61"> Ethyl (3-Butyl-6-chloro-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-8-yl)carbamate</h3><div class="NLM_p last">4-Methylmorpholine (3.35 mL, 30.50 mmol) and ethyl chloroformate (1.83 mL, 19.06 mmol) were added to a stirred solution of 3-butyl-6-chloro-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-8-amine (1721 mg, 7.63 mmol) in DCM (15 mL) at 0 °C. The resultant solution was allowed to warm to rt and stirred for 5 min. The resultant solution was diluted with DCM (50 mL) and washed with water (2 × 50 mL). The organic layer was separated, washed with brine (50 mL), passed through a hydrophobic frit, and concentrated <i>in vacuo</i>. The resultant residue was dissolved in minimal DCM and purified by silica chromatography (0–80% EtOAc in cyclohexane). The desired fractions were combined, yielding ethyl (3-butyl-6-chloro-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-8-yl)carbamate (1.523 g, 5.12 mmol, 67%) as an orange oil. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 11.17 (br. s, 1H), 7.64 (s, 1H), 4.26 (q, <i>J</i> = 7.1 Hz, 2H), 3.05 (t, <i>J</i> = 7.3 Hz, 2H), 1.85–1.75 (m, 2H), 1.45–1.35 (m, 2H), 1.29 (t, <i>J</i> = 7.1 Hz, 3H), 0.93 (t, <i>J</i> = 7.3 Hz, 3H); <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 154.0, 150.8, 149.9, 138.9, 136.7, 103.5, 62.5, 28.4, 23.6, 22.2, 14.7, 14.0; HRMS (M + H)<sup>+</sup> calculated for C<sub>12</sub>H<sub>17</sub>ClN<sub>5</sub>O<sub>2</sub>, 298.1071; found, 298.1076; LC/MS (formic acid): <i>R</i><sub>t</sub> = 1.14 min (86%); [M + H]<sup>+</sup> = 298.</div></div><div id="sec4_45" class="NLM_sec NLM_sec_level_2"><div id="ac_i62" class="anchor-spacer"></div><h3 class="article-section__title" id="_i62"> Ethyl (3-Butyl-6-(4-methyl-3-nitrophenyl)-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-8-yl)carbamate (<b>55</b>)</h3><div class="NLM_p last">Solvent system was sparged with nitrogen for 1 h prior to use. Ethyl (3-butyl-6-chloro-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-8-yl)carbamate (480 mg, 1.61 mmol), (4-methyl-3-nitrophenyl)boronic acid (438 mg, 2.42 mmol), PdCl<sub>2</sub>(dppf) (590 mg, 0.81 mmol), and sodium carbonate (1709 mg, 16.12 mmol) were dissolved in toluene (15 mL) and EtOH (15 mL) under nitrogen at rt. The resultant solution was heated to 100 °C and stirred for 5 h under nitrogen before being allowed to cool down to rt and concentrated <i>in vacuo</i>. The resultant residue was dissolved in EtOAc (35 mL) and washed sequentially with water (35 mL) and brine (35 mL). The organic layer was passed through a hydrophobic frit and concentrated <i>in vacuo</i>. The resultant residue was dissolved in minimal DCM and purified by silica chromatography (0–100% EtOAc in cyclohexane). The desired fractions were combined, yielding ethyl (3-butyl-6-(4-methyl-3-nitrophenyl)-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-8-yl)carbamate (243 mg, 0.61 mmol, 38%) as an orange gum. m.p. 135–140 °C; ν<sub>max</sub> (solid)/cm<sup>–1</sup>: 3295, 2959, 2211, 1738, 1529 (N–O), 1220; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 10.91 (s, 1H), 8.50 (d, <i>J</i> = 2.0 Hz, 1H), 8.18 (dd, <i>J</i> = 8.1, 2.0 Hz, 1H), 8.13 (s, 1H), 7.70 (d, <i>J</i> = 8.1 Hz, 1H), 4.28 (q, <i>J</i> = 7.1 Hz, 2H), 3.17 (t, <i>J</i> = 7.5 Hz, 2H), 2.60 (s, 3H), 1.94–1.81 (m, 2H), 1.51–1.37 (m, 2H), 1.32 (t, <i>J</i> = 7.1 Hz, 3H), 0.96 (t, <i>J</i> = 7.5 Hz, 3H); <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 154.1, 152.4, 149.8, 139.2, 135.9, 135.5, 134.6, 134.3, 131.7, 123.0, 101.0, 62.2, 28.6, 23.7, 22.2, 19.9, 14.7, 14.0 (N.B., one sp<sup>2</sup> carbon signal not visible); HRMS (M + H)<sup>+</sup> calculated for C<sub>19</sub>H<sub>23</sub>N<sub>6</sub>O<sub>4</sub>, 399.1781; found, 399.1778; LC/MS (formic acid): <i>R</i><sub>t</sub> = 1.30 min (92%); [M + H]<sup>+</sup> = 399.</div></div><div id="sec4_46" class="NLM_sec NLM_sec_level_2"><div id="ac_i63" class="anchor-spacer"></div><h3 class="article-section__title" id="_i63"> Ethyl (6-(3-Amino-4-methylphenyl)-3-butyl-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-8-yl)carbamate</h3><div class="NLM_p last">Powdered iron (142 mg, 2.55 mmol) was added to a stirred solution of ethyl (3-butyl-6-(4-methyl-3-nitrophenyl)-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-8-yl)carbamate (203 mg, 0.51 mmol) and acetic acid (0.18 mL, 3.06 mmol) in EtOH (15 mL) and water (5 mL). The reaction mixture was then heated to 80 °C and stirred for 2 h. The resultant solution was allowed to cool down to rt before being concentrated <i>in vacuo</i>. The resultant residue was dissolved in saturated aq NaHCO<sub>3</sub> (15 mL) and extracted with EtOAc (3 × 20 mL). The organic fractions were combined, washed with brine (20 mL), passed through a hydrophobic frit, and concentrated <i>in vacuo</i>. The resultant residue was dissolved in minimal DCM and purified by silica chromatography (30–100% EtOAc in cyclohexane). The desired fractions were combined, yielding ethyl (6-(3-amino-4-methylphenyl)-3-butyl-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-8-yl)carbamate (93 mg, 0.25 mmol, 50%) as a pale brown solid. m.p. 79–83 °C; ν<sub>max</sub> (solid)/cm<sup>–1</sup>: 2959, 1738, 1561, 1530, 1220; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 10.71 (br. s, 1H), 8.08 (s, 1H), 7.25 (d, <i>J</i> = 1.5 Hz, 1H), 7.15–7.04 (m, 2H), 5.16 (br. s, 2H), 4.26 (q, <i>J</i> = 7.1 Hz, 2H), 3.15 (t, <i>J</i> = 7.5 Hz, 2H), 2.13 (s, 3H), 1.86 (quint., <i>J</i> = 7.5 Hz, 2H), 1.41 (sixt., <i>J</i> = 7.5 Hz, 2H), 1.30 (t, <i>J</i> = 7.1 Hz, 3H), 0.95 (t, <i>J</i> = 7.5 Hz, 3H); <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 155.2, 154.2, 150.9, 147.7, 139.4, 135.0, 133.8, 131.0, 124.4, 115.2, 112.4, 101.6, 62.1, 28.6, 23.8, 22.2, 17.8, 14.8, 14.0; HRMS (M + H)<sup>+</sup> calculated for C<sub>19</sub>H<sub>25</sub>N<sub>6</sub>O<sub>2</sub>, 369.2039; found, 369.2036; LC/MS (formic acid): <i>R</i><sub>t</sub> = 1.11 min (94%); [M + H]<sup>+</sup> = 369.</div></div><div id="sec4_47" class="NLM_sec NLM_sec_level_2"><div id="ac_i64" class="anchor-spacer"></div><h3 class="article-section__title" id="_i64"> Ethyl (3-Butyl-6-(4-methyl-3-(methylsulfonamido)phenyl)-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-8-yl)Carbamate (<b>38</b>)</h3><div class="NLM_p last">Mesyl-Cl (0.17 mL, 2.17 mmol) was added to a stirred solution of ethyl (6-(3-amino-4-methylphenyl)-3-butyl-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-8-yl)carbamate (80 mg, 0.22 mmol) and pyridine (0.06 mL, 0.76 mmol) in DCM (3 mL), and the resultant solution was stirred at rt for 3 h before being concentrated <i>in vacuo</i>. The resultant residue was dissolved in 1:1 MeOH/DMSO and purified by MDAP (high pH). The desired fractions were concentrated <i>in vacuo</i>, yielding ethyl (3-butyl-6-(4-methyl-3-(methylsulfonamido)phenyl)-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-8-yl)carbamate (72 mg, 0.16 mmol, 74%) as a white solid. m.p. 199–204 °C; ν<sub>max</sub> (solid)/cm<sup>–1</sup>: 3228, 2959, 1727, 1564, 1533, 1223; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 8.11 (s, 1H), 7.91 (d, <i>J</i> = 1.8 Hz, 1H), 7.75 (dd, <i>J</i> = 8.0, 1.8 Hz, 1H), 7.47 (d, <i>J</i> = 8.0 Hz, 1H), 4.27 (q, <i>J</i> = 7.1 Hz, 2H), 3.16 (t, <i>J</i> = 7.5 Hz, 2H), 3.03 (s, 3H), 2.40 (s, 3H), 1.86 (quint., <i>J</i> = 7.5 Hz, 2H), 1.42 (sixt., <i>J</i> = 7.5 Hz, 2H), 1.31 (t, <i>J</i> = 7.1 Hz, 3H), 0.95 (t, <i>J</i> = 7.5 Hz, 3H) (N.B., exchangeable sulfonamide and carbamate protons not visible); <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 154.2, 154.0, 151.1, 139.3, 137.1, 136.9, 135.5, 134.0, 132.1, 124.8, 124.7, 101.3, 62.2, 40.6, 28.6, 23.8, 22.2, 18.6, 14.8, 14.0. (N.B., the peak at 40.6 is obscured by the DMSO solvent peak but is clearly visible); HRMS (M + H)<sup>+</sup> calculated for C<sub>20</sub>H<sub>27</sub>N<sub>6</sub>O<sub>4</sub>S, 447.1814; found, 447.1809; LC/MS (formic acid): <i>R</i><sub>t</sub> = 1.07 min (99%); [M + H]<sup>+</sup> = 447.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i65"><a href="/doi/suppl/10.1021/acs.jmedchem.0c00075" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_20128" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_20128" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c00075?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00075</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Additional text describing all methods and results, all screening statistics, representative LCMS traces of target compounds, full BROMOscan and selectivity data, and X-ray data collection and refinement statistics (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00075/suppl_file/jm0c00075_si_001.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00075/suppl_file/jm0c00075_si_001.pdf">jm0c00075_si_001.pdf (977.06 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Coordinates have been deposited with the Protein Data Bank under accession codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YQW">6YQW</a> (BRD9/<b>8</b> complex), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YQR">6YQR</a> (BRD9/<b>14</b> complex), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YQZ">6YQZ</a> (BRD4(1)/<b>14</b> complex), and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YQS">6YQS</a> (BRD9/<b>18</b> complex). The authors will release atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.0c00075" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_48437" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_48437" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Philip G. Humphreys</span> - <span class="hlFld-Affiliation affiliation">GlaxoSmithKline
R&D, Stevenage SG1 2NY, Hertfordshire, United
Kingdom</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8614-7155" title="Orcid link">http://orcid.org/0000-0002-8614-7155</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#57273f3e3b3e277930793f223a273f25322e241730243c7934383a"><span class="__cf_email__" data-cfemail="36465e5f5a5f461851185e435b465e44534f457651455d1855595b">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michael A. Clegg</span> - <span class="hlFld-Affiliation affiliation">GlaxoSmithKline
R&D, Stevenage SG1 2NY, Hertfordshire, United
Kingdom</span>; 
    <span class="hlFld-Affiliation affiliation">WestCHEM,
Department of Pure and Applied Chemistry, Thomas Graham Building, University of Strathclyde, 295 Cathedral Street, Glasgow G1 1XL, United Kingdom</span>; 
    <span>Present Address:
                        Present address: MSD, Francis Crick Institute, 1 Midland Road, London NW1 1AT, U.K</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paul Bamborough</span> - <span class="hlFld-Affiliation affiliation">GlaxoSmithKline
R&D, Stevenage SG1 2NY, Hertfordshire, United
Kingdom</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9479-2894" title="Orcid link">http://orcid.org/0000-0001-9479-2894</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chun-wa Chung</span> - <span class="hlFld-Affiliation affiliation">GlaxoSmithKline
R&D, Stevenage SG1 2NY, Hertfordshire, United
Kingdom</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2480-3110" title="Orcid link">http://orcid.org/0000-0002-2480-3110</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peter D. Craggs</span> - <span class="hlFld-Affiliation affiliation">GlaxoSmithKline
R&D, Stevenage SG1 2NY, Hertfordshire, United
Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Laurie Gordon</span> - <span class="hlFld-Affiliation affiliation">GlaxoSmithKline
R&D, Stevenage SG1 2NY, Hertfordshire, United
Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paola Grandi</span> - <span class="hlFld-Affiliation affiliation">Cellzome
GmbH, R&D MST GlaxoSmithKline, Meyerhofstrasse 1 69117 Heidelberg, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Melanie Leveridge</span> - <span class="hlFld-Affiliation affiliation">GlaxoSmithKline
R&D, Stevenage SG1 2NY, Hertfordshire, United
Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Matthew Lindon</span> - <span class="hlFld-Affiliation affiliation">GlaxoSmithKline
R&D, Stevenage SG1 2NY, Hertfordshire, United
Kingdom</span>; 
    <span>Present Address:
                        Present address: Respiratory, Inflammation and Autoimmune (RIA) BioPharmaceuticals R&D, AstraZeneca, SE-431 83 Mölndal, Sweden</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gemma M. Liwicki</span> - <span class="hlFld-Affiliation affiliation">GlaxoSmithKline
R&D, Stevenage SG1 2NY, Hertfordshire, United
Kingdom</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3508-8711" title="Orcid link">http://orcid.org/0000-0003-3508-8711</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Anne-Marie Michon</span> - <span class="hlFld-Affiliation affiliation">Cellzome
GmbH, R&D MST GlaxoSmithKline, Meyerhofstrasse 1 69117 Heidelberg, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Judit Molnar</span> - <span class="hlFld-Affiliation affiliation">GlaxoSmithKline
R&D, Stevenage SG1 2NY, Hertfordshire, United
Kingdom</span>; 
    <span>Present Address:
                        Present address: Mission Therapeutics Ltd., McClintock Building, Granta Park, Great Abington, Cambridge CB21 6GP, U. K</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Inmaculada Rioja</span> - <span class="hlFld-Affiliation affiliation">GlaxoSmithKline
R&D, Stevenage SG1 2NY, Hertfordshire, United
Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peter E. Soden</span> - <span class="hlFld-Affiliation affiliation">GlaxoSmithKline
R&D, Stevenage SG1 2NY, Hertfordshire, United
Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Natalie H. Theodoulou</span> - <span class="hlFld-Affiliation affiliation">GlaxoSmithKline
R&D, Stevenage SG1 2NY, Hertfordshire, United
Kingdom</span>; 
    <span class="hlFld-Affiliation affiliation">WestCHEM,
Department of Pure and Applied Chemistry, Thomas Graham Building, University of Strathclyde, 295 Cathedral Street, Glasgow G1 1XL, United Kingdom</span>; 
    <span>Present Address:
                        Present address: The University of Manchester, Oxford Road, Manchester M13 9PL, U.K</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Thilo Werner</span> - <span class="hlFld-Affiliation affiliation">Cellzome
GmbH, R&D MST GlaxoSmithKline, Meyerhofstrasse 1 69117 Heidelberg, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nicholas C. O. Tomkinson</span> - <span class="hlFld-Affiliation affiliation">WestCHEM,
Department of Pure and Applied Chemistry, Thomas Graham Building, University of Strathclyde, 295 Cathedral Street, Glasgow G1 1XL, United Kingdom</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-5509-0133" title="Orcid link">http://orcid.org/0000-0002-5509-0133</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rab K. Prinjha</span> - <span class="hlFld-Affiliation affiliation">GlaxoSmithKline
R&D, Stevenage SG1 2NY, Hertfordshire, United
Kingdom</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>The manuscript was written through contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): All authors except M.A.C and N.C.O.T, are current or former employees of GlaxoSmithKline.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i72">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_24454" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_24454" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">M.A.C. and N.H.T. are grateful to GlaxoSmithKline R&D, Stevenage, and the University of Strathclyde for Ph.D. studentship, and we thank the EPSRC for funding via Prosperity Partnership EP/S035990/1. We also thank Heather Barnett for the array support, Sean Lynn and Richard Upton for assistance with NMR, Steve Jackson and Eric Hortense for chiral HPLC, and Tony Cook for high-resolution mass spectrometry. We appreciate helpful discussion with Emmanuel Demont during the course of this work.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">AMP</td><td class="NLM_def"><p class="first last">artificial membrane permeability</p></td></tr><tr><td class="NLM_term">ATAD2A</td><td class="NLM_def"><p class="first last">ATPase family, AAA domain-containing 2A</p></td></tr><tr><td class="NLM_term">ATAD2B</td><td class="NLM_def"><p class="first last">ATPase family, AAA domain-containing 2B</p></td></tr><tr><td class="NLM_term">BAF</td><td class="NLM_def"><p class="first last">BRG1/BRM-associated factor</p></td></tr><tr><td class="NLM_term">BAZ2A</td><td class="NLM_def"><p class="first last">bromodomain adjacent to zinc finger domain 2A</p></td></tr><tr><td class="NLM_term">BAZ2B</td><td class="NLM_def"><p class="first last">bromodomain adjacent to zinc finger domain 2B</p></td></tr><tr><td class="NLM_term">BCP</td><td class="NLM_def"><p class="first last">bromodomain-containing protein</p></td></tr><tr><td class="NLM_term">BD</td><td class="NLM_def"><p class="first last">bromodomain</p></td></tr><tr><td class="NLM_term">BET</td><td class="NLM_def"><p class="first last">bromodomain and extra terminal domain</p></td></tr><tr><td class="NLM_term">BRD1</td><td class="NLM_def"><p class="first last">bromodomain-containing protein 1</p></td></tr><tr><td class="NLM_term">BRD2</td><td class="NLM_def"><p class="first last">bromodomain-containing protein 2</p></td></tr><tr><td class="NLM_term">BRD3</td><td class="NLM_def"><p class="first last">bromodomain-containing protein 3</p></td></tr><tr><td class="NLM_term">BRD4</td><td class="NLM_def"><p class="first last">bromodomain-containing protein 4</p></td></tr><tr><td class="NLM_term">BRD7</td><td class="NLM_def"><p class="first last">bromodomain-containing protein 7</p></td></tr><tr><td class="NLM_term">BRD8</td><td class="NLM_def"><p class="first last">bromodomain-containing protein 8</p></td></tr><tr><td class="NLM_term">BRD9</td><td class="NLM_def"><p class="first last">bromodomain-containing protein 9</p></td></tr><tr><td class="NLM_term">BRDT</td><td class="NLM_def"><p class="first last">bromodomain-containing protein, testis-specific</p></td></tr><tr><td class="NLM_term">BRPF1</td><td class="NLM_def"><p class="first last">bromodomain and PHD finger-containing protein 1</p></td></tr><tr><td class="NLM_term">BRPF3</td><td class="NLM_def"><p class="first last">bromodomain and PHD finger-containing protein 3</p></td></tr><tr><td class="NLM_term">CAD</td><td class="NLM_def"><p class="first last">charged aerosol detection</p></td></tr><tr><td class="NLM_term">CECR2</td><td class="NLM_def"><p class="first last">cat eye syndrome chromosome region candidate 2</p></td></tr><tr><td class="NLM_term">CLND</td><td class="NLM_def"><p class="first last">chemiluminescent nitrogen detection</p></td></tr><tr><td class="NLM_term">CREBBP</td><td class="NLM_def"><p class="first last">CREB binding protein</p></td></tr><tr><td class="NLM_term">EP300</td><td class="NLM_def"><p class="first last">E1A-associated protein p300</p></td></tr><tr><td class="NLM_term">FALZ</td><td class="NLM_def"><p class="first last">bromodomain PHD finger transcription factor</p></td></tr><tr><td class="NLM_term">GCN5L2</td><td class="NLM_def"><p class="first last">general control nondepressible 5</p></td></tr><tr><td class="NLM_term">KAc</td><td class="NLM_def"><p class="first last">acetylated lysine</p></td></tr><tr><td class="NLM_term">LCMS</td><td class="NLM_def"><p class="first last">liquid chromatography mass spectrometry</p></td></tr><tr><td class="NLM_term">LE</td><td class="NLM_def"><p class="first last">ligand efficiency</p></td></tr><tr><td class="NLM_term">LLE</td><td class="NLM_def"><p class="first last">lipophilic ligand efficiency</p></td></tr><tr><td class="NLM_term">PBAF</td><td class="NLM_def"><p class="first last">polybromo-associated BAF</p></td></tr><tr><td class="NLM_term">PBRM1</td><td class="NLM_def"><p class="first last">polybromo 1</p></td></tr><tr><td class="NLM_term">PCAF</td><td class="NLM_def"><p class="first last">P300/CREBBP-associated factor</p></td></tr><tr><td class="NLM_term">PEPPSI</td><td class="NLM_def"><p class="first last">pyridine-enhanced precatalyst preparation stabilization and initiation</p></td></tr><tr><td class="NLM_term">PHD</td><td class="NLM_def"><p class="first last">plant homeodomain</p></td></tr><tr><td class="NLM_term">pIC<sub>50</sub></td><td class="NLM_def"><p class="first last">–log<sub>10</sub>(IC<sub>50</sub>)</p></td></tr><tr><td class="NLM_term">SMARCA2</td><td class="NLM_def"><p class="first last">SWI/SNF-related, matrix-associated, actin-dependent regulator of chromatin subfamily A, member 2</p></td></tr><tr><td class="NLM_term">SMARCA4</td><td class="NLM_def"><p class="first last">SWI/SNF-related, matrix-associated, actin-dependent regulator of chromatin, subfamily A, member 4</p></td></tr><tr><td class="NLM_term">TRIM24</td><td class="NLM_def"><p class="first last">tripartite motif-containing 24</p></td></tr><tr><td class="NLM_term">TRIM33</td><td class="NLM_def"><p class="first last">tripartite motif-containing 33</p></td></tr><tr><td class="NLM_term">WDR9</td><td class="NLM_def"><p class="first last">WD repeat-containing protein 9</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i74">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_28750" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_28750" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 54 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.-M.</span></span> <span> </span><span class="NLM_article-title">The Bromodomain: A New Target in Emerging Epigenetic Medicine</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">598</span>– <span class="NLM_lpage">608</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00831</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00831" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVyit7fJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=598-608&author=S.+G.+Smithauthor=M.-M.+Zhou&title=The+Bromodomain%3A+A+New+Target+in+Emerging+Epigenetic+Medicine&doi=10.1021%2Facschembio.5b00831"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The Bromodomain: A New Target in Emerging Epigenetic Medicine</span></div><div class="casAuthors">Smith, Steven G.; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">598-608</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromodomain (BrD) is a conserved protein modular domain found in many chromatin- and transcription-assocd. proteins that has the ability to recognize acetylated lysine residues.  This activity allows bromodomains to play a vital role in many acetylation-mediated protein-protein interactions in the cell, ranging from substrate recruitment for histone acetyltransferases (HATs) to aiding in multiple-protein complex assembly for gene transcriptional activation or suppression in chromatin.  In recent years, considerable efforts have been made to develop chem. inhibitors of these bromodomains in an effort to probe their cellular functions.  Potent and selective inhibitors have been extensively developed for one group of the bromodomain family termed BET proteins that consist of tandem bromodomains followed by an extra terminal domain.  Drug developers are actively designing inhibitors of other bromodomains and working to move the most successful inhibitors into the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTQF2Qjn_iuLVg90H21EOLACvtfcHk0liUyNvttTqfqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVyit7fJ&md5=dd34fc591550ff1528e6dccd1d04f1c1</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00831&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00831%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DS.%2BG.%26aulast%3DZhou%26aufirst%3DM.-M.%26atitle%3DThe%2520Bromodomain%253A%2520A%2520New%2520Target%2520in%2520Emerging%2520Epigenetic%2520Medicine%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D598%26epage%3D608%26doi%3D10.1021%2Facschembio.5b00831" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Targeting Bromodomains: Epigenetic Readers of Lysine Acetylation</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">337</span>– <span class="NLM_lpage">356</span>, <span class="refDoi"> DOI: 10.1038/nrd4286</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1038%2Fnrd4286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=24751816" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=337-356&author=P.+Filippakopoulosauthor=S.+Knapp&title=Targeting+Bromodomains%3A+Epigenetic+Readers+of+Lysine+Acetylation&doi=10.1038%2Fnrd4286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting bromodomains: epigenetic readers of lysine acetylation</span></div><div class="casAuthors">Filippakopoulos, Panagis; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">337-356</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Lysine acetylation is a key mechanism that regulates chromatin structure; aberrant acetylation levels have been linked to the development of several diseases.  Acetyl-lysine modifications create docking sites for bromodomains, which are small interaction modules found on diverse proteins, some of which have a key role in the acetylation-dependent assembly of transcriptional regulator complexes.  These complexes can then initiate transcriptional programs that result in phenotypic changes.  The recent discovery of potent and highly specific inhibitors for the BET (bromodomain and extra-terminal) family of bromodomains has stimulated intensive research activity in diverse therapeutic areas, particularly in oncol., where BET proteins regulate the expression of key oncogenes and anti-apoptotic proteins.  In addn., targeting BET bromodomains could hold potential for the treatment of inflammation and viral infection.  Here, we highlight recent progress in the development of bromodomain inhibitors, and their potential applications in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOSIcwBJKj4LVg90H21EOLACvtfcHk0liUyNvttTqfqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D&md5=77049c82ce6a19b4685e3b7950399a07</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrd4286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4286%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DTargeting%2520Bromodomains%253A%2520Epigenetic%2520Readers%2520of%2520Lysine%2520Acetylation%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D337%26epage%3D356%26doi%3D10.1038%2Fnrd4286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangos, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volkmer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pawson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingras, A.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Histone Recognition and Large-scale Structural Analysis of the Human Bromodomain Family</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>149</i></span>,  <span class="NLM_fpage">214</span>– <span class="NLM_lpage">231</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2012.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1016%2Fj.cell.2012.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=22464331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=149&publication_year=2012&pages=214-231&author=P.+Filippakopoulosauthor=S.+Picaudauthor=M.+Mangosauthor=T.+Keatesauthor=J.+P.+Lambertauthor=D.+Barsyte-Lovejoyauthor=I.+Felletarauthor=R.+Volkmerauthor=S.+M%C3%BCllerauthor=T.+Pawsonauthor=A.-C.+Gingrasauthor=C.+H.+Arrowsmithauthor=S.+Knapp&title=Histone+Recognition+and+Large-scale+Structural+Analysis+of+the+Human+Bromodomain+Family&doi=10.1016%2Fj.cell.2012.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Recognition and Large-Scale Structural Analysis of the Human Bromodomain Family</span></div><div class="casAuthors">Filippakopoulos, Panagis; Picaud, Sarah; Mangos, Maria; Keates, Tracy; Lambert, Jean-Philippe; Barsyte-Lovejoy, Dalia; Felletar, Ildiko; Volkmer, Rudolf; Muller, Susanne; Pawson, Tony; Gingras, Anne-Claude; Arrowsmith, Cheryl H.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">149</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">214-231</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) are protein interaction modules that specifically recognize ε-N-lysine acetylation motifs, a key event in the reading process of epigenetic marks.  The 61 BRDs in the human genome cluster into eight families based on structure/sequence similarity.  Here, we present 29 high-resoln. crystal structures, covering all BRD families.  These proteins are: ASH1L, ATAD2, BAZ2B, BPTF, BRD1, BRD3(1), BRD3(2), BRD4(1), BRD4(2), BRD9, BRDT(1), CECR2, EP300, CREBBP, GCN5L2, KIAA1240, PB1(1), PB1(2), PB1(3), PB1(4), PB1(5), PB1(6), PCAF, PHIP(2), TAF1(2), WDR9(2), BRD4(1).  Comprehensive crossfamily structural anal. identifies conserved and family-specific structural features that are necessary for specific acetylation-dependent substrate recognition.  Screening of more than 30 representative BRDs against systematic histone-peptide arrays identifies new BRD substrates and reveals a strong influence of flanking posttranslational modifications, such as acetylation and phosphorylation, suggesting that BRDs recognize combinations of marks rather than singly acetylated sequences.  We further uncovered a structural mechanism for the simultaneous binding and recognition of diverse diacetyl-contg. peptides by BRD4.  These data provide a foundation for structure-based drug design of specific inhibitors for this emerging target family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK2FSIcPoXZLVg90H21EOLACvtfcHk0lgyN4SplHo6EA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D&md5=05583a01a1cd4c3b59fc59c00e4c4777</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DMangos%26aufirst%3DM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DLambert%26aufirst%3DJ.%2BP.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DVolkmer%26aufirst%3DR.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DPawson%26aufirst%3DT.%26aulast%3DGingras%26aufirst%3DA.-C.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DHistone%2520Recognition%2520and%2520Large-scale%2520Structural%2520Analysis%2520of%2520the%2520Human%2520Bromodomain%2520Family%26jtitle%3DCell%26date%3D2012%26volume%3D149%26spage%3D214%26epage%3D231%26doi%3D10.1016%2Fj.cell.2012.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Bromodomains as Therapeutic Targets</span>. <i>Expert Rev. Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>13</i></span>, <span class="NLM_elocation-id">e29</span> <span class="refDoi"> DOI: 10.1017/S1462399411001992</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1017%2FS1462399411001992" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=21933453" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1OhurjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2011&author=S.+Mullerauthor=P.+Filippakopoulosauthor=S.+Knapp&title=Bromodomains+as+Therapeutic+Targets&doi=10.1017%2FS1462399411001992"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomains as therapeutic targets</span></div><div class="casAuthors">Muller, Susanne; Filippakopoulos, Panagis; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Expert Reviews in Molecular Medicine</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e29/1-e29/21</span>CODEN:
                <span class="NLM_cas:coden">ERMMFS</span>;
        ISSN:<span class="NLM_cas:issn">1462-3994</span>.
    
            (<span class="NLM_cas:orgname">Cambridge University Press</span>)
        </div><div class="casAbstract">A review.  Acetylation of lysine residues is a post-translational modification with broad relevance to cellular signalling and disease biol.  Enzymes that 'write' (histone acetyltransferases, HATs) and 'erase' (histone deacetylases, HDACs) acetylation sites are an area of extensive research in current drug development, but very few potent inhibitors that modulate the 'reading process' mediated by acetyl lysines have been described.  The principal readers of ε-N-acetyl lysine (Kac) marks are bromodomains (BRDs), which are a diverse family of evolutionary conserved protein-interaction modules.  The conserved BRD fold contains a deep, largely hydrophobic acetyl lysine binding site, which represents an attractive pocket for the development of small, pharmaceutically active mols.  Proteins that contain BRDs have been implicated in the development of a large variety of diseases.  Recently, two highly potent and selective inhibitors that target BRDs of the BET (bromodomains and extra-terminal) family provided compelling data supporting targeting of these BRDs in inflammation and in an aggressive type of squamous cell carcinoma.  It is likely that BRDs will emerge alongside HATs and HDACs as interesting targets for drug development for the large no. of diseases that are caused by aberrant acetylation of lysine residues.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSuKacpsr6xLVg90H21EOLACvtfcHk0lgyN4SplHo6EA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1OhurjI&md5=dc9383afcbe6aedc265041c21213aa0b</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1017%2FS1462399411001992&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1017%252FS1462399411001992%26sid%3Dliteratum%253Aachs%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DBromodomains%2520as%2520Therapeutic%2520Targets%26jtitle%3DExpert%2520Rev.%2520Mol.%2520Med.%26date%3D2011%26volume%3D13%26doi%3D10.1017%2FS1462399411001992" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jain, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, M. C.</span></span> <span> </span><span class="NLM_article-title">Bromodomain Histone Readers and Cancer</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>429</i></span>,  <span class="NLM_fpage">2003</span>– <span class="NLM_lpage">2010</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2016.11.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1016%2Fj.jmb.2016.11.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=27890782" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFOisbbI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=429&publication_year=2017&pages=2003-2010&author=A.+K.+Jainauthor=M.+C.+Barton&title=Bromodomain+Histone+Readers+and+Cancer&doi=10.1016%2Fj.jmb.2016.11.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomain histone readers and cancer</span></div><div class="casAuthors">Jain, Abhinav K.; Barton, Michelle C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">429</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">2003-2010</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Lysine acetylation of histone proteins is a fundamental post-translational modification that regulates chromatin structure and plays an important role in gene transcription.  Aberrant levels of histone lysine acetylation are assocd. with the development of several diseases.  Acetyl-lysine modifications create docking sites for bromodomains, which are structurally conserved modules present in transcription-assocd. proteins that are termed "reader" proteins.  Bromodomain-contg. reader proteins are part of multiprotein complexes that regulate transcription programs, which are often assocd. with profound phenotypic changes.  Many bromodomain-contg. proteins are aberrantly expressed in diseases, as best studied in cancers, where bromodomain proteins impact the expression of oncogenes and anti-apoptotic proteins.  Thus, bromodomain readers of histone acetylation have emerged as attractive targets for cancer drug discovery, prompting immense interest in epigenetic-focused, medicinal chem. to develop structurally guided chem. probes of bromodomains.  Here, we describe bromodomain-contg. proteins with defined roles in cancer and highlight recent progress in the development of bromodomain inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPuEwpy7dAA7Vg90H21EOLACvtfcHk0lgyN4SplHo6EA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFOisbbI&md5=2d1d3efa959f66eeb8789b48daf96325</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2016.11.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2016.11.020%26sid%3Dliteratum%253Aachs%26aulast%3DJain%26aufirst%3DA.%2BK.%26aulast%3DBarton%26aufirst%3DM.%2BC.%26atitle%3DBromodomain%2520Histone%2520Readers%2520and%2520Cancer%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2017%26volume%3D429%26spage%3D2003%26epage%3D2010%26doi%3D10.1016%2Fj.jmb.2016.11.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Noguchi-Yachide, T.</span></span> <span> </span><span class="NLM_article-title">BET Bromodomain as a Target of Epigenetic Therapy</span>. <i>Chem. Pharm. Bull. (Tokyo)</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>64</i></span>,  <span class="NLM_fpage">540</span>– <span class="NLM_lpage">547</span>, <span class="refDoi"> DOI: 10.1248/cpb.c16-00225</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1248%2Fcpb.c16-00225" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=27250788" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A280%3ADC%252BC2s%252FmtVShtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2016&pages=540-547&author=T.+Noguchi-Yachide&title=BET+Bromodomain+as+a+Target+of+Epigenetic+Therapy&doi=10.1248%2Fcpb.c16-00225"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">BET Bromodomain as a Target of Epigenetic Therapy</span></div><div class="casAuthors">Noguchi-Yachide Tomomi</div><div class="citationInfo"><span class="NLM_cas:title">Chemical & pharmaceutical bulletin</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">540-7</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Acetylation of histone is a key epigenetic modification, and contributes to many DNA-dependent cellular processes.  The bromodomain structure, which consists of approximately 110 amino acid residues, serves as a 'reader' that recognizes acetylated lysine in histones, leading to recruitment of positive transcriptional elongation factor b (P-TEFb), and thereby promoting transcriptional activity and chromatin remodeling.  Among bromodomain-containing proteins, members of the bromodomain and extra-terminal domain (BET) family contain tandem N-terminal bromodomains.  BET proteins, especially BRD4, have attracted interest as candidate therapeutic targets due to their putative involvement in the pathogenesis of various diseases, including cancer and inflammatory diseases.  Several BET inhibitors are under clinical trial for treatment of various cancers.  Furthermore, polypharmacological agents such as dual kinase/BET inhibitors and dual histone deacetylase (HDAC)/BET inhibitors have recently been developed, in addition to agents that degrade BET family proteins, such as proteolysis-targeting chimeras (PROTACs).  This paper reviews recent progress in epigenetic therapy targeting the BET bromodomain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRr5yqeTtyiLovKgEt20y1XfW6udTcc2eYGch88TZ-xT7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2s%252FmtVShtw%253D%253D&md5=7046d36daea4f693f1ebf6f19555db47</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1248%2Fcpb.c16-00225&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1248%252Fcpb.c16-00225%26sid%3Dliteratum%253Aachs%26aulast%3DNoguchi-Yachide%26aufirst%3DT.%26atitle%3DBET%2520Bromodomain%2520as%2520a%2520Target%2520of%2520Epigenetic%2520Therapy%26jtitle%3DChem.%2520Pharm.%2520Bull.%2520%2528Tokyo%2529%26date%3D2016%26volume%3D64%26spage%3D540%26epage%3D547%26doi%3D10.1248%2Fcpb.c16-00225" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Padmanabhan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathur, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manjula, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tripathi, S.</span></span> <span> </span><span class="NLM_article-title">Bromodomain and Extra-Terminal (BET) Family Proteins: New Therapeutic Targets in Major Diseases</span>. <i>J. Biosci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">295</span>– <span class="NLM_lpage">311</span>, <span class="refDoi"> DOI: 10.1007/s12038-016-9600-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1007%2Fs12038-016-9600-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=27240990" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC28XptFOqtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2016&pages=295-311&author=B.+Padmanabhanauthor=S.+Mathurauthor=R.+Manjulaauthor=S.+Tripathi&title=Bromodomain+and+Extra-Terminal+%28BET%29+Family+Proteins%3A+New+Therapeutic+Targets+in+Major+Diseases&doi=10.1007%2Fs12038-016-9600-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomain and extra-terminal (BET) family proteins: New therapeutic targets in major diseases</span></div><div class="casAuthors">Padmanabhan, Balasundaram; Mathur, Shruti; Manjula, Ramu; Tripathi, Shailesh</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biosciences (New Delhi, India)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">295-311</span>CODEN:
                <span class="NLM_cas:coden">JOBSDN</span>;
        ISSN:<span class="NLM_cas:issn">0250-5991</span>.
    
            (<span class="NLM_cas:orgname">Springer (India) Private Ltd.</span>)
        </div><div class="casAbstract">The bromodomains and extra-terminal domain (BET) family proteins recognize acetylated chromatin through their bromodomains (BDs) and help in regulating gene expression.  BDs are chromatin 'readers': by interacting with acetylated lysines on the histone tails, they recruit chromatin-regulating proteins on the promoter region to regulate gene expression and repression.  Extensive efforts have been employed by scientific communities worldwide to identify and develop potential inhibitors of BET family BDs to regulate protein expression by inhibiting acetylated histone (H3/H4) interactions.  Several small mol. inhibitors have been reported, which not only have high affinity but also have high specificity to BET BDs.  These developments make BET family proteins an important therapeutic targets for major diseases such as cancer, neurol. disorders, obesity and inflammation.  Here, we review and discuss the structural biol. of BET family BDs and their applications in major diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp87geC_AUDErVg90H21EOLACvtfcHk0lgJOoso6oXW1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XptFOqtbw%253D&md5=2555d6cb5db8e4151b63f7add86c7250</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1007%2Fs12038-016-9600-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12038-016-9600-6%26sid%3Dliteratum%253Aachs%26aulast%3DPadmanabhan%26aufirst%3DB.%26aulast%3DMathur%26aufirst%3DS.%26aulast%3DManjula%26aufirst%3DR.%26aulast%3DTripathi%26aufirst%3DS.%26atitle%3DBromodomain%2520and%2520Extra-Terminal%2520%2528BET%2529%2520Family%2520Proteins%253A%2520New%2520Therapeutic%2520Targets%2520in%2520Major%2520Diseases%26jtitle%3DJ.%2520Biosci.%26date%3D2016%26volume%3D41%26spage%3D295%26epage%3D311%26doi%3D10.1007%2Fs12038-016-9600-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, S.</span></span> <span> </span><span class="NLM_article-title">Disrupting Acetyl-Lysine Interactions: Recent Advance in the Development of BET Inhibitors</span>. <i>Curr. Drug Targets</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1148</span>– <span class="NLM_lpage">1165</span>, <span class="refDoi"> DOI: 10.2174/1389450119666171129165427</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.2174%2F1389450119666171129165427" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=29189147" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlCqt7bF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2018&pages=1148-1165&author=F.+Zhangauthor=S.+Ma&title=Disrupting+Acetyl-Lysine+Interactions%3A+Recent+Advance+in+the+Development+of+BET+Inhibitors&doi=10.2174%2F1389450119666171129165427"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Disrupting Acetyl-lysine Interactions: Recent Advance in the Development of BET Inhibitors</span></div><div class="casAuthors">Zhang, Fa; Ma, Shutao</div><div class="citationInfo"><span class="NLM_cas:title">Current Drug Targets</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1148-1165</span>CODEN:
                <span class="NLM_cas:coden">CDTUAU</span>;
        ISSN:<span class="NLM_cas:issn">1389-4501</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">Background: Histone acetylation is an essential approach of post-translational modification (PTM) and a significant component of epigenetic regulation that is mediated by Bromodomainscontaining protein (BRDs).  In recent years, many researchers have found that a variety of malignancy, inflammatory and other diseases occurrences and developments are assocd. with BRD4 expression disorders or dysfunction.  Meanwhile, many inhibitors of the extra-terminal (BET) family have been reported in many papers.  Objective: This review summarized those newly found BET inhibitors, their mechanism of action and bioactivity.  Secondly, those compds. were mainly classified based on their structures and their structure-activity relationship information was discussed.  Beyond that, every compd.'s design strategy was pointed out.  Results and Conclusion: Herein, the recent advances reported were reviewed for discovering more excellent small mol. inhibitors.  Currently, in addn. to compd. 4, compds. 7, 22 and 90, have also been into the clin. trial stage.  In the view of the outstanding performance of BET inhibitors in anti-tumor, anti-inflammatory and anti-drug resistance, we believe that more and more BET inhibitors will become the new epigenetic therapy for cancer, inflammation and autoimmune disease in clin. practice in the near future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2tTRPAV67lLVg90H21EOLACvtfcHk0lgJOoso6oXW1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlCqt7bF&md5=b70d378e6e49e10f91c1a3c8dcde1318</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.2174%2F1389450119666171129165427&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1389450119666171129165427%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DF.%26aulast%3DMa%26aufirst%3DS.%26atitle%3DDisrupting%2520Acetyl-Lysine%2520Interactions%253A%2520Recent%2520Advance%2520in%2520the%2520Development%2520of%2520BET%2520Inhibitors%26jtitle%3DCurr.%2520Drug%2520Targets%26date%3D2018%26volume%3D19%26spage%3D1148%26epage%3D1165%26doi%3D10.2174%2F1389450119666171129165427" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span> <i>CCS14 Scientific Posters</i>; <a href="https://www.cellcentric.com/ccs1477/Publications/" class="extLink">https://www.cellcentric.com/ccs1477/Publications/</a> (accessed 30 September 2019).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+CCS14+Scientific+Posters%3B+https%3A%2F%2Fwww.cellcentric.com%2Fccs1477%2FPublications%2F+%28accessed+30+September+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26jtitle%3DCCS14%2520Scientific%2520Posters" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span> <i>NCT03568656 -
Study to Evaluate CCS1477 in Advanced Tumours</i>; <a href="https://clinicaltrials.gov/ct2/show/NCT03568656?term=CCS1477&amp;rank=1" class="extLink">https://clinicaltrials.gov/ct2/show/NCT03568656?term=CCS1477&rank=1</a> (accessed 30 September 2019).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+NCT03568656+-%0AStudy+to+Evaluate+CCS1477+in+Advanced+Tumours%3B+https%3A%2F%2Fclinicaltrials.gov%2Fct2%2Fshow%2FNCT03568656%3Fterm%3DCCS1477%26rank%3D1+%28accessed+30+September+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26jtitle%3DNCT03568656%2520-%250AStudy%2520to%2520Evaluate%2520CCS1477%2520in%2520Advanced%2520Tumours" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zaware, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.-M.</span></span> <span> </span><span class="NLM_article-title">Bromodomain Biology and Drug Discovery</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">870</span>– <span class="NLM_lpage">879</span>, <span class="refDoi"> DOI: 10.1038/s41594-019-0309-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1038%2Fs41594-019-0309-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=31582847" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvFSnsL3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2019&pages=870-879&author=N.+Zawareauthor=M.-M.+Zhou&title=Bromodomain+Biology+and+Drug+Discovery&doi=10.1038%2Fs41594-019-0309-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomain biology and drug discovery</span></div><div class="casAuthors">Zaware, Nilesh; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">870-879</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The bromodomain (BrD) is a conserved structural module found in chromatin- and transcription-assocd. proteins that acts as the primary reader for acetylated lysine residues.  This basic activity endows BrD proteins with versatile functions in the regulation of protein-protein interactions mediating chromatin-templated gene transcription, DNA recombination, replication and repair.  Consequently, BrD proteins are involved in the pathogenesis of numerous human diseases.  In this Review, we highlight our current understanding of BrD biol., and discuss the latest development of small-mol. inhibitors targeting BrDs as emerging epigenetic therapies for cancer and inflammatory disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0q-3UgSbqULVg90H21EOLACvtfcHk0lgwGuSFXv-reA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvFSnsL3E&md5=7ccf3a3a2ff7147ead1148ee71fa2806</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fs41594-019-0309-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41594-019-0309-8%26sid%3Dliteratum%253Aachs%26aulast%3DZaware%26aufirst%3DN.%26aulast%3DZhou%26aufirst%3DM.-M.%26atitle%3DBromodomain%2520Biology%2520and%2520Drug%2520Discovery%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2019%26volume%3D26%26spage%3D870%26epage%3D879%26doi%3D10.1038%2Fs41594-019-0309-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schiedel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moroglu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ascough, D. M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chamberlain, A. E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamps, J. J. A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sekirnik, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conway, S. J.</span></span> <span> </span><span class="NLM_article-title">Chemical Epigenetics: The Impact of Chemical and Chemical Biology Techniques on Bromodomain Target Validation</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">17930</span>– <span class="NLM_lpage">17952</span>, <span class="refDoi"> DOI: 10.1002/anie.201812164</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1002%2Fanie.201812164" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitFKrt77N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2019&pages=17930-17952&author=M.+Schiedelauthor=M.+Morogluauthor=D.+M.+H.+Ascoughauthor=A.+E.+R.+Chamberlainauthor=J.+J.+A.+G.+Kampsauthor=A.+R.+Sekirnikauthor=S.+J.+Conway&title=Chemical+Epigenetics%3A+The+Impact+of+Chemical+and+Chemical+Biology+Techniques+on+Bromodomain+Target+Validation&doi=10.1002%2Fanie.201812164"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical Epigenetics: The Impact of Chemical and Chemical Biology Techniques on Bromodomain Target Validation</span></div><div class="casAuthors">Schiedel, Matthias; Moroglu, Mustafa; Ascough, David M. H.; Chamberlain, Anna E. R.; Kamps, Jos J. A. G.; Sekirnik, Angelina R.; Conway, Stuart J.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">50</span>),
    <span class="NLM_cas:pages">17930-17952</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Epigenetics is currently the focus of intense research interest across a broad range of disciplines due to its importance in a multitude of biol. processes and disease states.  Epigenetic functions result partly from modification of the nucleobases in DNA and RNA, and/or post-translational modifications of histone proteins.  These modifications are dynamic, with cellular machinery identified to modulate and interpret the marks.  The authors' focus is on bromodomains, which bind to acetylated lysine residues.  Progress in the study of bromodomains, and the development of bromodomain ligands, has been rapid.  These advances have been underpinned by many disciplines, but chem. and chem. biol. have undoubtedly played a significant role.  Herein, the authors review the key chem. and chem. biol. approaches that have furthered the authors' study of bromodomains, enabled the development of bromodomain ligands, and played a crit. role in the validation of bromodomains as therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq99ft2MpAlMLVg90H21EOLACvtfcHk0lgwGuSFXv-reA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitFKrt77N&md5=d26cbf7f78d9542b2c98b9b74d979761</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1002%2Fanie.201812164&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201812164%26sid%3Dliteratum%253Aachs%26aulast%3DSchiedel%26aufirst%3DM.%26aulast%3DMoroglu%26aufirst%3DM.%26aulast%3DAscough%26aufirst%3DD.%2BM.%2BH.%26aulast%3DChamberlain%26aufirst%3DA.%2BE.%2BR.%26aulast%3DKamps%26aufirst%3DJ.%2BJ.%2BA.%2BG.%26aulast%3DSekirnik%26aufirst%3DA.%2BR.%26aulast%3DConway%26aufirst%3DS.%2BJ.%26atitle%3DChemical%2520Epigenetics%253A%2520The%2520Impact%2520of%2520Chemical%2520and%2520Chemical%2520Biology%2520Techniques%2520on%2520Bromodomain%2520Target%2520Validation%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2019%26volume%3D58%26spage%3D17930%26epage%3D17952%26doi%3D10.1002%2Fanie.201812164" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oprea, T. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bologa, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curpan, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glen, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lipinski, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marshall, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ostopovici-Halip, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rishton, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ursu, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaz, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waller, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sklar, L. A.</span></span> <span> </span><span class="NLM_article-title">A Crowdsourcing Evaluation of the NIH Chemical Probes</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">441</span>– <span class="NLM_lpage">447</span>, <span class="refDoi"> DOI: 10.1038/nchembio0709-441</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1038%2Fnchembio0709-441" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=19536101" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BD1MXnsVags7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=441-447&author=T.+I.+Opreaauthor=C.+G.+Bologaauthor=S.+Boyerauthor=R.+F.+Curpanauthor=R.+C.+Glenauthor=A.+L.+Hopkinsauthor=C.+A.+Lipinskiauthor=G.+R.+Marshallauthor=Y.+C.+Martinauthor=L.+Ostopovici-Halipauthor=G.+Rishtonauthor=O.+Ursuauthor=R.+J.+Vazauthor=C.+Wallerauthor=H.+Waldmannauthor=L.+A.+Sklar&title=A+Crowdsourcing+Evaluation+of+the+NIH+Chemical+Probes&doi=10.1038%2Fnchembio0709-441"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">A crowdsourcing evaluation of the NIH chemical probes</span></div><div class="casAuthors">Oprea, Tudor I.; Bologa, Cristian G.; Boyer, Scott; Curpan, Ramona F.; Glen, Robert C.; Hopkins, Andrew L.; Lipinski, Christopher A.; Marshall, Garland R.; Martin, Yvonne C.; Ostopovici-Halip, Liliana; Rishton, Gilbert; Ursu, Oleg; Vaz, Roy J.; Waller, Chris; Waldmann, Herbert; Sklar, Larry A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">441-447</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review with commentary.  Between 2004 and 2008, the US National Institutes of Health Mol. Libraries and Imaging initiative pilot phase funded 10 high-throughput screening centers, resulting in the deposition of 691 assays into PubChem and the nomination of 64 chem. probes.  We crowdsourced the Mol. Libraries and Imaging initiative output to 11 experts, who expressed medium or high levels of confidence in 48 of these 64 probes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2vr16pYCeybVg90H21EOLACvtfcHk0liLd-2WuFgU_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXnsVags7k%253D&md5=adc3ed5b33cbd233ec6160c5465651b7</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnchembio0709-441&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio0709-441%26sid%3Dliteratum%253Aachs%26aulast%3DOprea%26aufirst%3DT.%2BI.%26aulast%3DBologa%26aufirst%3DC.%2BG.%26aulast%3DBoyer%26aufirst%3DS.%26aulast%3DCurpan%26aufirst%3DR.%2BF.%26aulast%3DGlen%26aufirst%3DR.%2BC.%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26aulast%3DMarshall%26aufirst%3DG.%2BR.%26aulast%3DMartin%26aufirst%3DY.%2BC.%26aulast%3DOstopovici-Halip%26aufirst%3DL.%26aulast%3DRishton%26aufirst%3DG.%26aulast%3DUrsu%26aufirst%3DO.%26aulast%3DVaz%26aufirst%3DR.%2BJ.%26aulast%3DWaller%26aufirst%3DC.%26aulast%3DWaldmann%26aufirst%3DH.%26aulast%3DSklar%26aufirst%3DL.%2BA.%26atitle%3DA%2520Crowdsourcing%2520Evaluation%2520of%2520the%2520NIH%2520Chemical%2520Probes%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2009%26volume%3D5%26spage%3D441%26epage%3D447%26doi%3D10.1038%2Fnchembio0709-441" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S. V.</span></span> <span> </span><span class="NLM_article-title">The Art of the Chemical Probe</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">159</span>– <span class="NLM_lpage">161</span>, <span class="refDoi"> DOI: 10.1038/nchembio.296</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1038%2Fnchembio.296" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=20154659" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhvFGntbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=159-161&author=S.+V.+Frye&title=The+Art+of+the+Chemical+Probe&doi=10.1038%2Fnchembio.296"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">The art of the chemical probe</span></div><div class="casAuthors">Frye, Stephen V.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">159-161</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Chem. biologists frequently aim to create small-mol. probes that interact with a specific protein in vitro in order to explore the role of the protein in a broader biol. context (cells or organisms), but a common understanding of what makes a high-quality probe is lacking.  Here I propose a set of principles to guide probe qualification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7vDfsTLhqWrVg90H21EOLACvtfcHk0liLd-2WuFgU_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhvFGntbg%253D&md5=b7b48c591a611b8686e11b3fc4f1b85c</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.296&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.296%26sid%3Dliteratum%253Aachs%26aulast%3DFrye%26aufirst%3DS.%2BV.%26atitle%3DThe%2520Art%2520of%2520the%2520Chemical%2520Probe%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2010%26volume%3D6%26spage%3D159%26epage%3D161%26doi%3D10.1038%2Fnchembio.296" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chekler, E. L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, L. H.</span></span> <span> </span><span class="NLM_article-title">Target Validation Using Chemical Probes</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">195</span>– <span class="NLM_lpage">199</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1197</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1038%2Fnchembio.1197" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=23508172" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktFagsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=195-199&author=M.+E.+Bunnageauthor=E.+L.+P.+Cheklerauthor=L.+H.+Jones&title=Target+Validation+Using+Chemical+Probes&doi=10.1038%2Fnchembio.1197"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Target validation using chemical probes</span></div><div class="casAuthors">Bunnage, Mark E.; Chekler, Eugene L. Piatnitski; Jones, Lyn H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">195-199</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review on the fully profiled chem. probes essential to support the unbiased interpretation of biol. expts. necessary for rigorous preclin. target validation.  We believe that by developing a 'chem. probe tool kit', and a framework for its use, chem. biol. can have a more central role in identifying targets of potential relevance to disease, avoiding many of the biases that complicate target validation as practiced currently.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLwRTT4pXGxbVg90H21EOLACvtfcHk0liLd-2WuFgU_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktFagsr4%253D&md5=af0c362882191de96a8e8698ff2c9ce8</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1197&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1197%26sid%3Dliteratum%253Aachs%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DChekler%26aufirst%3DE.%2BL.%2BP.%26aulast%3DJones%26aufirst%3DL.%2BH.%26atitle%3DTarget%2520Validation%2520Using%2520Chemical%2520Probes%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2013%26volume%3D9%26spage%3D195%26epage%3D199%26doi%3D10.1038%2Fnchembio.1197" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bassi, Z. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fillmore, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miah, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maller, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galwey, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waddington, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parravicini, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacMillan-Jones, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gongora, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphreys, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Churcher, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span> <span> </span><span class="NLM_article-title">Modulating PCAF/GCN5 Immune Cell Function Through a PROTAC Approach</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">2862</span>– <span class="NLM_lpage">2867</span>, <span class="refDoi"> DOI: 10.1021/acschembio.8b00705</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.8b00705" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1OktL7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=2862-2867&author=Z.+I.+Bassiauthor=M.+C.+Fillmoreauthor=A.+H.+Miahauthor=T.+D.+Chapmanauthor=C.+Mallerauthor=E.+J.+Robertsauthor=L.+C.+Davisauthor=D.+E.+Lewisauthor=N.+W.+Galweyauthor=K.+E.+Waddingtonauthor=V.+Parraviciniauthor=A.+L.+MacMillan-Jonesauthor=C.+Gongoraauthor=P.+G.+Humphreysauthor=I.+Churcherauthor=R.+K.+Prinjhaauthor=D.+F.+Tough&title=Modulating+PCAF%2FGCN5+Immune+Cell+Function+Through+a+PROTAC+Approach&doi=10.1021%2Facschembio.8b00705"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Modulating PCAF/GCN5 Immune Cell Function through a PROTAC Approach</span></div><div class="casAuthors">Bassi, Zuni I.; Fillmore, Martin C.; Miah, Afjal H.; Chapman, Trevor D.; Maller, Claire; Roberts, Emma J.; Davis, Lauren C.; Lewis, Darcy E.; Galwey, Nicholas W.; Waddington, Kirsty E.; Parravicini, Valentino; Macmillan-Jones, Abigail L.; Gongora, Celine; Humphreys, Philip G.; Churcher, Ian; Prinjha, Rab K.; Tough, David F.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2862-2867</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">P300/CBP-assocd. factor (PCAF) and general control nonderepressible 5 (GCN5) are closely related epigenetic proteins, each contg. an acetyltransferase domain and a bromodomain.  Consistent with reported roles for these proteins in immune function, we find that PCAF-deficient macrophages exhibit a markedly reduced ability to produce cytokines upon stimulation with lipopolysaccharide (LPS).  Investigating the potential to target this pathway pharmacol., we show that chem. inhibition of the PCAF/GCN5 bromodomains is insufficient to recapitulate the diminished inflammatory response of PCAF-deficient immune cells.  However, by generating the first PCAF/GCN5 proteolysis targeting chimera (PROTAC), we identify small mols. able to degrade PCAF/GCN5 and to potently modulate the expression of multiple inflammatory mediators in LPS-stimulated macrophages and dendritic cells.  Our data illustrate the power of the PROTAC approach in the context of multidomain proteins, revealing a novel anti-inflammatory therapeutic opportunity for targeting PCAF/GCN5.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUuwZ6JZtCirVg90H21EOLACvtfcHk0liLd-2WuFgU_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1OktL7I&md5=6783f39397381536ed40ac9e1b20fa5b</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Facschembio.8b00705&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.8b00705%26sid%3Dliteratum%253Aachs%26aulast%3DBassi%26aufirst%3DZ.%2BI.%26aulast%3DFillmore%26aufirst%3DM.%2BC.%26aulast%3DMiah%26aufirst%3DA.%2BH.%26aulast%3DChapman%26aufirst%3DT.%2BD.%26aulast%3DMaller%26aufirst%3DC.%26aulast%3DRoberts%26aufirst%3DE.%2BJ.%26aulast%3DDavis%26aufirst%3DL.%2BC.%26aulast%3DLewis%26aufirst%3DD.%2BE.%26aulast%3DGalwey%26aufirst%3DN.%2BW.%26aulast%3DWaddington%26aufirst%3DK.%2BE.%26aulast%3DParravicini%26aufirst%3DV.%26aulast%3DMacMillan-Jones%26aufirst%3DA.%2BL.%26aulast%3DGongora%26aufirst%3DC.%26aulast%3DHumphreys%26aufirst%3DP.%2BG.%26aulast%3DChurcher%26aufirst%3DI.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DTough%26aufirst%3DD.%2BF.%26atitle%3DModulating%2520PCAF%252FGCN5%2520Immune%2520Cell%2520Function%2520Through%2520a%2520PROTAC%2520Approach%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2018%26volume%3D13%26spage%3D2862%26epage%3D2867%26doi%3D10.1021%2Facschembio.8b00705" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clegg, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomkinson, N. C. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphreys, P. G.</span></span> <span> </span><span class="NLM_article-title">Advancements in the Development of Non-BET Bromodomain Chemical Probes</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">362</span>– <span class="NLM_lpage">385</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201800738</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1002%2Fcmdc.201800738" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=30624862" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1Ohs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=362-385&author=M.+A.+Cleggauthor=N.+C.+O.+Tomkinsonauthor=R.+K.+Prinjhaauthor=P.+G.+Humphreys&title=Advancements+in+the+Development+of+Non-BET+Bromodomain+Chemical+Probes&doi=10.1002%2Fcmdc.201800738"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Advancements in the Development of non-BET Bromodomain Chemical Probes</span></div><div class="casAuthors">Clegg, Michael A.; Tomkinson, Nicholas C. O.; Prinjha, Rab K.; Humphreys, Philip G.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">362-385</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The bromodomain and extra terminal (BET) family of bromodomain-contg. proteins (BCPs) have been the subject of extensive research over the past decade, resulting in a plethora of high-quality chem. probes for their tandem bromodomains.  In turn, these chem. probes have helped reveal the profound biol. role of the BET bromodomains and their role in disease, ultimately leading to a no. of mols. in active clin. development.  However, the BET subfamily represents just 8/61 of the known human bromodomains, and attention has now expanded to the biol. role of the remaining 53 non-BET bromodomains.  Rapid growth of this research area has been accompanied by a greater understanding of the requirements for an effective bromodomain chem. probe and has led to a no. of new non-BET bromodomain chem. probes being developed.  Advances since Dec. 2015 are discussed, highlighting the strengths/caveats of each mol., and the value they add toward validating the non-BET bromodomains as tractable therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_U9uvpaCEd7Vg90H21EOLACvtfcHk0ljWvWuwBohtvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1Ohs70%253D&md5=a93505759fe45ffd0352de2f0e17885a</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201800738&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201800738%26sid%3Dliteratum%253Aachs%26aulast%3DClegg%26aufirst%3DM.%2BA.%26aulast%3DTomkinson%26aufirst%3DN.%2BC.%2BO.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DHumphreys%26aufirst%3DP.%2BG.%26atitle%3DAdvancements%2520in%2520the%2520Development%2520of%2520Non-BET%2520Bromodomain%2520Chemical%2520Probes%26jtitle%3DChemMedChem%26date%3D2019%26volume%3D14%26spage%3D362%26epage%3D385%26doi%3D10.1002%2Fcmdc.201800738" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shain, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollack, J. R.</span></span> <span> </span><span class="NLM_article-title">The Spectrum of SWI/SNF Mutations, Ubiquitous in Human Cancers</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e55119</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0055119</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1371%2Fjournal.pone.0055119" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=23355908" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&author=A.+H.+Shainauthor=J.+R.+Pollack&title=The+Spectrum+of+SWI%2FSNF+Mutations%2C+Ubiquitous+in+Human+Cancers&doi=10.1371%2Fjournal.pone.0055119"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0055119&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0055119%26sid%3Dliteratum%253Aachs%26aulast%3DShain%26aufirst%3DA.%2BH.%26aulast%3DPollack%26aufirst%3DJ.%2BR.%26atitle%3DThe%2520Spectrum%2520of%2520SWI%252FSNF%2520Mutations%252C%2520Ubiquitous%2520in%2520Human%2520Cancers%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26doi%3D10.1371%2Fjournal.pone.0055119" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kadoch, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hargreaves, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodges, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elias, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ranish, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crabtree, G. R.</span></span> <span> </span><span class="NLM_article-title">Proteomic and Bioinformatic Analysis of Mammalian SWI/SNF Complexes Identifies Extensive Roles in Human Malignancy</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">592</span>– <span class="NLM_lpage">601</span>, <span class="refDoi"> DOI: 10.1038/ng.2628</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1038%2Fng.2628" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=23644491" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC3sXntVart7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2013&pages=592-601&author=C.+Kadochauthor=D.+C.+Hargreavesauthor=C.+Hodgesauthor=L.+Eliasauthor=L.+Hoauthor=J.+Ranishauthor=G.+R.+Crabtree&title=Proteomic+and+Bioinformatic+Analysis+of+Mammalian+SWI%2FSNF+Complexes+Identifies+Extensive+Roles+in+Human+Malignancy&doi=10.1038%2Fng.2628"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Proteomic and bioinformatic analysis of mammalian SWI/SNF complexes identifies extensive roles in human malignancy</span></div><div class="casAuthors">Kadoch, Cigall; Hargreaves, Diana C.; Hodges, Courtney; Elias, Laura; Ho, Lena; Ranish, Jeff; Crabtree, Gerald R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">592-601</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Subunits of mammalian SWI/SNF (mSWI/SNF or BAF) complexes have recently been implicated as tumor suppressors in human malignancies.  To understand the full extent of their involvement, we conducted a proteomic anal. of endogenous mSWI/SNF complexes, which identified several new dedicated, stable subunits not found in yeast SWI/SNF complexes, including BCL7A, BCL7B and BCL7C, BCL11A and BCL11B, BRD9 and SS18.  Incorporating these new members, we detd. mSWI/SNF subunit mutation frequency in exome and whole-genome sequencing studies of primary human tumors.  Notably, mSWI/SNF subunits are mutated in 19.6% of all human tumors reported in 44 studies.  Our anal. suggests that specific subunits protect against cancer in specific tissues.  In addn., mutations affecting more than one subunit, defined here as compd. heterozygosity, are prevalent in certain cancers.  Our studies demonstrate that mSWI/SNF is the most frequently mutated chromatin-regulatory complex (CRC) in human cancer, exhibiting a broad mutation pattern, similar to that of TP53.  Thus, proper functioning of polymorphic BAF complexes may constitute a major mechanism of tumor suppression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqP873DCpn7JrVg90H21EOLACvtfcHk0ljWvWuwBohtvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXntVart7s%253D&md5=0f49cf3c59e7bc04b74970fe656b1e17</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fng.2628&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng.2628%26sid%3Dliteratum%253Aachs%26aulast%3DKadoch%26aufirst%3DC.%26aulast%3DHargreaves%26aufirst%3DD.%2BC.%26aulast%3DHodges%26aufirst%3DC.%26aulast%3DElias%26aufirst%3DL.%26aulast%3DHo%26aufirst%3DL.%26aulast%3DRanish%26aufirst%3DJ.%26aulast%3DCrabtree%26aufirst%3DG.%2BR.%26atitle%3DProteomic%2520and%2520Bioinformatic%2520Analysis%2520of%2520Mammalian%2520SWI%252FSNF%2520Complexes%2520Identifies%2520Extensive%2520Roles%2520in%2520Human%2520Malignancy%26jtitle%3DNat.%2520Genet.%26date%3D2013%26volume%3D45%26spage%3D592%26epage%3D601%26doi%3D10.1038%2Fng.2628" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hodges, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirkland, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crabtree, G. R.</span></span> <span> </span><span class="NLM_article-title">The Many Roles of BAF (MSWI/SNF) and PBAF Complexes in Cancer</span>. <i>Cold Spring Harb. Perspect. Med.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">a026930</span>, <span class="refDoi"> DOI: 10.1101/cshperspect.a026930</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1101%2Fcshperspect.a026930" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=27413115" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC1cXns1ygsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=a026930&author=C.+Hodgesauthor=J.+G.+Kirklandauthor=G.+R.+Crabtree&title=The+Many+Roles+of+BAF+%28MSWI%2FSNF%29+and+PBAF+Complexes+in+Cancer&doi=10.1101%2Fcshperspect.a026930"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">The many roles of BAF (mSWI/SNF) and PBAF complexes in cancer</span></div><div class="casAuthors">Hodges, Courtney; Kirkland, Jacob G.; Crabtree, Gerald R.</div><div class="citationInfo"><span class="NLM_cas:title">Cold Spring Harbor Perspectives in Medicine</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">a026930/1-a026930/25</span>CODEN:
                <span class="NLM_cas:coden">CSHPFV</span>;
        ISSN:<span class="NLM_cas:issn">2157-1422</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">During the last decade, a host of epigenetic mechanisms were found to contribute to cancer and other human diseases.  Several genomic studies have revealed that ∼ 20% of malignancies have alterations of the subunits of polymorphic BRG-/BRM-assocd. factor (BAF) and Polybromo-assocd. BAF (PBAF) complexes, making them among the most frequently mutated complexes in cancer.  Recurrent mutations arise in genes encoding several BAF/ PBAF subunits, including ARID1A, ARID2, PBRM1, SMARCA4, and SMARCB1.  These subunits share some degree of conservation with subunits from related ATP (ATP)-dependent chromatin remodeling complexes in model organisms, in which a large body of work provides insight into their roles in cancer.  Here, we review the roles of BAF- and PBAF-like complexes in these organisms, and relate these findings to recent discoveries in cancer epigenomics.  We review several roles of BAF and PBAF complexes in cancer, including transcriptional regulation, DNA repair, and regulation of chromatin architecture and topol.  More recent results highlight the need for new techniquesto study these complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCynky-OxCc7Vg90H21EOLACvtfcHk0ljWvWuwBohtvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXns1ygsLs%253D&md5=71976f9f18bfad4c44fe26991679a194</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1101%2Fcshperspect.a026930&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fcshperspect.a026930%26sid%3Dliteratum%253Aachs%26aulast%3DHodges%26aufirst%3DC.%26aulast%3DKirkland%26aufirst%3DJ.%2BG.%26aulast%3DCrabtree%26aufirst%3DG.%2BR.%26atitle%3DThe%2520Many%2520Roles%2520of%2520BAF%2520%2528MSWI%252FSNF%2529%2520and%2520PBAF%2520Complexes%2520in%2520Cancer%26jtitle%3DCold%2520Spring%2520Harb.%2520Perspect.%2520Med.%26date%3D2016%26volume%3D6%26spage%3Da026930%26doi%3D10.1101%2Fcshperspect.a026930" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cleary, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeck, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Selitsky, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savich, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, T.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Getz, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parker, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powers, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guindi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghanekar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, D. Y.</span></span> <span> </span><span class="NLM_article-title">Identification of Driver Genes in Hepatocellular Carcinoma by Exome Sequencing</span>. <i>Hepatology</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">1693</span>– <span class="NLM_lpage">1702</span>, <span class="refDoi"> DOI: 10.1002/hep.26540</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1002%2Fhep.26540" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=23728943" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslWqsLbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2013&pages=1693-1702&author=S.+P.+Clearyauthor=W.+R.+Jeckauthor=X.+Zhaoauthor=K.+Chenauthor=S.+R.+Selitskyauthor=G.+L.+Savichauthor=T.-X.+Tanauthor=M.+C.+Wuauthor=G.+Getzauthor=M.+S.+Lawrenceauthor=J.+S.+Parkerauthor=J.+Liauthor=S.+Powersauthor=H.+Kimauthor=S.+Fischerauthor=M.+Guindiauthor=A.+Ghanekarauthor=D.+Y.+Chiang&title=Identification+of+Driver+Genes+in+Hepatocellular+Carcinoma+by+Exome+Sequencing&doi=10.1002%2Fhep.26540"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of driver genes in hepatocellular carcinoma by exome sequencing</span></div><div class="casAuthors">Cleary, Sean P.; Jeck, William R.; Zhao, Xiaobei; Chen, Kui; Selitsky, Sara R.; Savich, Gleb L.; Tan, Ting-Xu; Wu, Michael C.; Getz, Gad; Lawrence, Michael S.; Parker, Joel S.; Li, Jinyu; Powers, Scott; Kim, Hyeja; Fischer, Sandra; Guindi, Maha; Ghanekar, Anand; Chiang, Derek Y.</div><div class="citationInfo"><span class="NLM_cas:title">Hepatology (Hoboken, NJ, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1693-1702</span>CODEN:
                <span class="NLM_cas:coden">HPTLD9</span>;
        ISSN:<span class="NLM_cas:issn">0270-9139</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Genetic alterations in specific driver genes lead to disruption of cellular pathways and are crit. events in the instigation and progression of hepatocellular carcinoma (HCC).  As a prerequisite for individualized cancer treatment, we sought to characterize the landscape of recurrent somatic mutations in HCC.  We performed whole-exome sequencing on 87 HCCs and matched normal adjacent tissues to an av. coverage of 59×.  The overall mutation rate was roughly two mutations per Mb, with a median of 45 nonsynonymous mutations that altered the amino acid sequence (range, 2-381).  We found recurrent mutations in several genes with high transcript levels: TP53 (18%); CTNNB1 (10%); KEAP1 (8%); C16orf62 (8%); MLL4 (7%); and RAC2 (5%).  Significantly affected gene families include the nucleotide-binding domain and leucine-rich repeat-contg. family, calcium channel subunits, and histone methyltransferases.  In particular, the MLL family of methyltransferases for histone H3 lysine 4 were mutated in 20% of tumors.  Conclusion: The NFE2L2-KEAP1 and MLL pathways are recurrently mutated in multiple cohorts of HCC. (Hepatol. 2013;58:1693-1702).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYieayAje5DLVg90H21EOLACvtfcHk0lj0avbK6eDA7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslWqsLbF&md5=fd9f6d7220f85ac6f2cc337c1cd2ef99</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1002%2Fhep.26540&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fhep.26540%26sid%3Dliteratum%253Aachs%26aulast%3DCleary%26aufirst%3DS.%2BP.%26aulast%3DJeck%26aufirst%3DW.%2BR.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DSelitsky%26aufirst%3DS.%2BR.%26aulast%3DSavich%26aufirst%3DG.%2BL.%26aulast%3DTan%26aufirst%3DT.-X.%26aulast%3DWu%26aufirst%3DM.%2BC.%26aulast%3DGetz%26aufirst%3DG.%26aulast%3DLawrence%26aufirst%3DM.%2BS.%26aulast%3DParker%26aufirst%3DJ.%2BS.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DPowers%26aufirst%3DS.%26aulast%3DKim%26aufirst%3DH.%26aulast%3DFischer%26aufirst%3DS.%26aulast%3DGuindi%26aufirst%3DM.%26aulast%3DGhanekar%26aufirst%3DA.%26aulast%3DChiang%26aufirst%3DD.%2BY.%26atitle%3DIdentification%2520of%2520Driver%2520Genes%2520in%2520Hepatocellular%2520Carcinoma%2520by%2520Exome%2520Sequencing%26jtitle%3DHepatology%26date%3D2013%26volume%3D58%26spage%3D1693%26epage%3D1702%26doi%3D10.1002%2Fhep.26540" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Troisi, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howard, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haswell, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hawk, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oberlick, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tzvetkov, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vazquez, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahn, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, C. W. M.</span></span> <span> </span><span class="NLM_article-title">BRD9 Defines a SWI/SNF Sub-Complex and Constitutes a Specific Vulnerability in Malignant Rhabdoid Tumors</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">11</span>, <span class="refDoi"> DOI: 10.1038/s41467-019-09891-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1038%2Fs41467-019-09891-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=30602773" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlSqs7rI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1-11&author=X.+Wangauthor=S.+Wangauthor=E.+C.+Troisiauthor=T.+P.+Howardauthor=J.+R.+Haswellauthor=B.+K.+Wolfauthor=W.+H.+Hawkauthor=P.+Ramosauthor=E.+M.+Oberlickauthor=E.+P.+Tzvetkovauthor=F.+Vazquezauthor=W.+C.+Hahnauthor=P.+J.+Parkauthor=C.+W.+M.+Roberts&title=BRD9+Defines+a+SWI%2FSNF+Sub-Complex+and+Constitutes+a+Specific+Vulnerability+in+Malignant+Rhabdoid+Tumors&doi=10.1038%2Fs41467-019-09891-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insights into the formation and voltage degradation of lithium- and manganese-rich layered oxides</span></div><div class="casAuthors">Hua, Weibo; Wang, Suning; Knapp, Michael; Leake, Steven J.; Senyshyn, Anatoliy; Richter, Carsten; Yavuz, Murat; Binder, Joachim R.; Grey, Clare P.; Ehrenberg, Helmut; Indris, Sylvio; Schwarz, Bjoern</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-11</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">One major challenge in the field of lithium-ion batteries is to understand the degrdn. mechanism of high-energy lithium- and manganese-rich layered cathode materials.  Although they can deliver 30% excess capacity compared with today's com.- used cathodes, the so-called voltage decay has been restricting their practical application.  In order to unravel the nature of this phenomenon, we have investigated systematically the structural and compositional dependence of manganese-rich lithium insertion compds. on the lithium content provided during synthesis.  Structural, electronic and electrochem. characterizations of LixNi0.2Mn0.6Oy with a wide range of lithium contents (0.00 ≤x ≤1.52, 1.07 ≤y < 2.4) and an anal. of the complexity in the synthesis pathways of monoclinic-layered Li[Li0.2Ni0.2Mn0.6]O2 oxide provide insight into the underlying processes that cause voltage fading in these cathode materials, i.e. transformation of the lithium-rich layered phase to a lithium-poor spinel phase via an intermediate lithium-contg. rock-salt phase with release of lithium/oxygen.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpP-zV7jY0wWbVg90H21EOLACvtfcHk0lj0avbK6eDA7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlSqs7rI&md5=d4fbfd7183cf008ccb46b04d5cabb0a4</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fs41467-019-09891-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-019-09891-7%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DTroisi%26aufirst%3DE.%2BC.%26aulast%3DHoward%26aufirst%3DT.%2BP.%26aulast%3DHaswell%26aufirst%3DJ.%2BR.%26aulast%3DWolf%26aufirst%3DB.%2BK.%26aulast%3DHawk%26aufirst%3DW.%2BH.%26aulast%3DRamos%26aufirst%3DP.%26aulast%3DOberlick%26aufirst%3DE.%2BM.%26aulast%3DTzvetkov%26aufirst%3DE.%2BP.%26aulast%3DVazquez%26aufirst%3DF.%26aulast%3DHahn%26aufirst%3DW.%2BC.%26aulast%3DPark%26aufirst%3DP.%2BJ.%26aulast%3DRoberts%26aufirst%3DC.%2BW.%2BM.%26atitle%3DBRD9%2520Defines%2520a%2520SWI%252FSNF%2520Sub-Complex%2520and%2520Constitutes%2520a%2520Specific%2520Vulnerability%2520in%2520Malignant%2520Rhabdoid%2520Tumors%26jtitle%3DNat.%2520Commun.%26date%3D2019%26volume%3D10%26spage%3D1%26epage%3D11%26doi%3D10.1038%2Fs41467-019-09891-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.</span></span> <span> </span><span class="NLM_article-title">MiR-141 Is Involved in BRD7-Mediated Cell Proliferation and Tumor Formation Through Suppression of the PTEN/AKT Pathway in Nasopharyngeal Carcinoma</span>. <i>Cell Death Dis.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">e2156</span> <span class="refDoi"> DOI: 10.1038/cddis.2016.64</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1038%2Fcddis.2016.64" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=27010857" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC28XltFGmsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&author=Y.+Liuauthor=R.+Zhaoauthor=H.+Wangauthor=Y.+Luoauthor=X.+Wangauthor=W.+Niuauthor=Y.+Zhouauthor=Q.+Wenauthor=S.+Fanauthor=X.+Liauthor=W.+Xiongauthor=J.+Maauthor=X.+Liauthor=M.+Tanauthor=G.+Liauthor=M.+Zhou&title=MiR-141+Is+Involved+in+BRD7-Mediated+Cell+Proliferation+and+Tumor+Formation+Through+Suppression+of+the+PTEN%2FAKT+Pathway+in+Nasopharyngeal+Carcinoma&doi=10.1038%2Fcddis.2016.64"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">miR-141 is involved in BRD7-mediated cell proliferation and tumor formation through suppression of the PTEN/AKT pathway in nasopharyngeal carcinoma</span></div><div class="casAuthors">Liu, Y.; Zhao, R.; Wang, H.; Luo, Y.; Wang, X.; Niu, W.; Zhou, Y.; Wen, Q.; Fan, S.; Li, X.; Xiong, W.; Ma, J.; Li, X.; Tan, M.; Li, G.; Zhou, M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Death & Disease</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e2156</span>CODEN:
                <span class="NLM_cas:coden">CDDEA4</span>;
        ISSN:<span class="NLM_cas:issn">2041-4889</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Bromodomain contg. 7 (BRD7) was identified as a nuclear transcriptional regulatory factor.  BRD7 functions as a tumor suppressor in multiple cancers, including nasopharyngeal carcinoma (NPC).  In this study, we reported a novel mechanism of BRD7 in NPC progression.  We demonstrated that the expression of miR-141 was remarkably increased in NPC tissues and was neg. correlated with the expression of BRD7 and the survival rate of NPC patients.  Decreased expression levels of miR-141, including the primary, the precursor and the mature forms of miR-141, were found in BRD7-overexpressing HEK293, 5-8F and HNE1 cells compared the control cells, while there was no obvious effect on the expression levels of the two crit. enzymes Drosha and Dicer.  BRD7 can neg. regulate the promoter activity of miR-141, while no obvious binding site of BRD7 was found in the potential promoter region of miR-141.  Moreover, ectopic expression of miR-141 can significantly promote cell proliferation and inhibit apoptosis in NPC, and rescuing the expression of miR-141 in BRD7-overexpressing NPC cells could partially reverse the tumor suppressive effect of BRD7 on cell proliferation and tumor growth in vitro and in vivo.  Furthermore, the activation of the PTEN/AKT pathway mediated by the overexpression of BRD7 could be inhibited by rescuing the expression of miR-141, which accordingly results in the partial restoration of cell proliferation and tumor growth.  Our findings demonstrate that the BRD7/miR-141/PTEN/AKT axis has crit. roles in the progression of NPC and provide some promising targets for the diagnosis and treatment of NPC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZXqgJgJqtxbVg90H21EOLACvtfcHk0lg-xR__GSFT5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XltFGmsbw%253D&md5=63a7b76c9f8a4a3f51ede5086ee4c00f</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fcddis.2016.64&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcddis.2016.64%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DLuo%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DNiu%26aufirst%3DW.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DWen%26aufirst%3DQ.%26aulast%3DFan%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DXiong%26aufirst%3DW.%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DTan%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DG.%26aulast%3DZhou%26aufirst%3DM.%26atitle%3DMiR-141%2520Is%2520Involved%2520in%2520BRD7-Mediated%2520Cell%2520Proliferation%2520and%2520Tumor%2520Formation%2520Through%2520Suppression%2520of%2520the%2520PTEN%252FAKT%2520Pathway%2520in%2520Nasopharyngeal%2520Carcinoma%26jtitle%3DCell%2520Death%2520Dis.%26date%3D2016%26volume%3D7%26doi%3D10.1038%2Fcddis.2016.64" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J.</span></span> <span> </span><span class="NLM_article-title">BRD7: A Novel Tumor Suppressor Gene in Different Cancers</span>. <i>Am. J. Transl. Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">742</span>– <span class="NLM_lpage">748</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=27158366" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC1cXmtFWlsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2016&pages=742-748&author=X.+Yuauthor=Z.+Liauthor=J.+Shen&title=BRD7%3A+A+Novel+Tumor+Suppressor+Gene+in+Different+Cancers"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">BRD7: a novel tumor suppressor gene in different cancers</span></div><div class="casAuthors">Yu, Xin; Li, Zheng; Shen, Jianxiong</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Translational Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">742-748</span>CODEN:
                <span class="NLM_cas:coden">AJTRA7</span>;
        ISSN:<span class="NLM_cas:issn">1943-8141</span>.
    
            (<span class="NLM_cas:orgname">e-Century Publishing Corp.</span>)
        </div><div class="casAbstract">BRD7 (bromodomain 7), also known as celtix-1, was first identified in nasopharyngeal carcinoma (NPC) cells in 2000.  BRD7 is a crucial component of both functional p53 and BRCA1 (breast cancer 1, early onset) pathways.  Recently, the BRD7 tumor suppressor status has been fully established.  Previous studies demonstrated that BRD7 was downregulated in human breast cancer and the downregulation often assocs. with tumor progression.  The expression of BRD7 was downregulated in various cancers, including breast cancer, NPC, gastric cancer, colorectal carcinoma, ovarian cancer, and prostate cancer.  Moreover, BRD7 inhibited cancer cell growth and metastasis and promote apoptosis in vitro and in vivo via downregulating AKT pathway.  In addn., BRD7 may regulate many signaling pathways including ras-raf-MEK-ERK and RB/E2F.  In this review, we provide an overview of current knowledge concerning the role of BRD7 in tumor development and progression.  To our knowledge, this is the first review about the role of this novel tumor suppressor gene BRD7in tumor development and progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqL0hhv3IxTHbVg90H21EOLACvtfcHk0lg-xR__GSFT5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXmtFWlsLs%253D&md5=6d6244ad2621b551afd1fb4244db7d3d</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DShen%26aufirst%3DJ.%26atitle%3DBRD7%253A%2520A%2520Novel%2520Tumor%2520Suppressor%2520Gene%2520in%2520Different%2520Cancers%26jtitle%3DAm.%2520J.%2520Transl.%2520Res.%26date%3D2016%26volume%3D8%26spage%3D742%26epage%3D748" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Drost, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mantovani, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tocco, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elkon, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Comel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holstege, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerkhoven, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jonkers, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voorhoeve, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agami, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Del Sal, G.</span></span> <span> </span><span class="NLM_article-title">BRD7 Is a Candidate Tumour Suppressor Gene Required for P53 Function</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">380</span>– <span class="NLM_lpage">389</span>, <span class="refDoi"> DOI: 10.1038/ncb2038</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1038%2Fncb2038" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=20228809" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC3cXktVyjurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2010&pages=380-389&author=J.+Drostauthor=F.+Mantovaniauthor=F.+Toccoauthor=R.+Elkonauthor=A.+Comelauthor=H.+Holstegeauthor=R.+Kerkhovenauthor=J.+Jonkersauthor=P.+M.+Voorhoeveauthor=R.+Agamiauthor=G.+Del+Sal&title=BRD7+Is+a+Candidate+Tumour+Suppressor+Gene+Required+for+P53+Function&doi=10.1038%2Fncb2038"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">BRD7 is a candidate tumour suppressor gene required for p53 function</span></div><div class="casAuthors">Drost, Jarno; Mantovani, Fiamma; Tocco, Francesca; Elkon, Ran; Comel, Anna; Holstege, Henne; Kerkhoven, Ron; Jonkers, Jos; Voorhoeve, P. Mathijs; Agami, Reuven; Del Sal, Giannino</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">380-389</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Oncogene-induced senescence is a p53-dependent defense mechanism against uncontrolled proliferation.  Consequently, many human tumors harbor p53 mutations and others show a dysfunctional p53 pathway, frequently by unknown mechanisms.  Here the authors identify BRD7 (bromodomain-contg. 7) as a protein whose inhibition allows full neoplastic transformation in the presence of wild-type p53.  In human breast tumors harboring wild-type, but not mutant, p53 the BRD7 gene locus was frequently deleted and low BRD7 expression was found in a subgroup of tumors.  Functionally, BRD7 is required for efficient p53-mediated transcription of a subset of target genes.  BRD7 interacts with p53 and p300 and is recruited to target gene promoters, affecting histone acetylation, p53 acetylation and promoter activity.  Thus, BRD7 suppresses tumorigenicity by serving as a p53 cofactor required for the efficient induction of p53-dependent oncogene-induced senescence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1grCCyCF9MrVg90H21EOLACvtfcHk0lg-xR__GSFT5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXktVyjurw%253D&md5=676cb9cbc5a8d157ac85d76c42ba8007</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fncb2038&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncb2038%26sid%3Dliteratum%253Aachs%26aulast%3DDrost%26aufirst%3DJ.%26aulast%3DMantovani%26aufirst%3DF.%26aulast%3DTocco%26aufirst%3DF.%26aulast%3DElkon%26aufirst%3DR.%26aulast%3DComel%26aufirst%3DA.%26aulast%3DHolstege%26aufirst%3DH.%26aulast%3DKerkhoven%26aufirst%3DR.%26aulast%3DJonkers%26aufirst%3DJ.%26aulast%3DVoorhoeve%26aufirst%3DP.%2BM.%26aulast%3DAgami%26aufirst%3DR.%26aulast%3DDel%2BSal%26aufirst%3DG.%26atitle%3DBRD7%2520Is%2520a%2520Candidate%2520Tumour%2520Suppressor%2520Gene%2520Required%2520for%2520P53%2520Function%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2010%26volume%3D12%26spage%3D380%26epage%3D389%26doi%3D10.1038%2Fncb2038" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nie, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gui, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.</span></span> <span> </span><span class="NLM_article-title">Bromodomain-containing Protein 7 Sensitizes Breast Cancer Cells to Paclitaxel by Activating Bcl2-antagonist/Killer Protein</span>. <i>Oncol. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">1487</span>– <span class="NLM_lpage">1496</span>, <span class="refDoi"> DOI: 10.3892/or.2018.6951</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.3892%2For.2018.6951" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=30592293" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtFygsrrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2019&pages=1487-1496&author=J.+Maauthor=W.+Niuauthor=X.+Wangauthor=Y.+Zhouauthor=H.+Wangauthor=F.+Liuauthor=Y.+Liuauthor=J.+Guoauthor=W.+Xiongauthor=Z.+Zengauthor=S.+Fanauthor=X.+Liauthor=X.+Nieauthor=G.+Liauthor=R.+Guiauthor=Y.+Luoauthor=M.+Zhou&title=Bromodomain-containing+Protein+7+Sensitizes+Breast+Cancer+Cells+to+Paclitaxel+by+Activating+Bcl2-antagonist%2FKiller+Protein&doi=10.3892%2For.2018.6951"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomain-containing protein 7 sensitizes breast cancer cells to paclitaxel by activating Bcl2-antagonist/killer protein</span></div><div class="casAuthors">Ma, Jinqi; Niu, Weihong; Wang, Xinye; Zhou, Yao; Wang, Heran; Liu, Fengxia; Liu, Yukun; Guo, Jie; Xiong, Wei; Zeng, Zhaoyang; Fan, Songqing; Li, Xiaoling; Nie, Xinmin; Li, Guiyuan; Gui, Rong; Luo, Yanwei; Zhou, Ming</div><div class="citationInfo"><span class="NLM_cas:title">Oncology Reports</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1487-1496</span>CODEN:
                <span class="NLM_cas:coden">OCRPEW</span>;
        ISSN:<span class="NLM_cas:issn">1791-2431</span>.
    
            (<span class="NLM_cas:orgname">Spandidos Publications Ltd.</span>)
        </div><div class="casAbstract">Our previous study demonstrated that bromodomain-contg. protein 7 (BRD7) inhibits cell proliferation and tumor growth, restoring the expression of B-cell lymphoma 2 antagonist/killer (Bak) sensitized breast cancer cells to paclitaxel.  However, the assocn. between BRD7 and paclitaxel sensitization, as well as BRD7 and Bak in breast cancer remains unknown.  In the present study, immunochem. staining was performed to measure the expression of BRD7 and Bak in breast cancer tissues.  Cell Counting Kit-8 assay, flow cytometry and tumor xenograft procedures were performed to evaluate the biol. role of BRD7 and Bak in breast cancer cells.  Western blotting, reverse transcription-quant. polymerase chain reaction, chromatin immunopptn. and luciferase reporter assays were also performed.  BRD7 was pos. correlated with Bak levels in breast cancer tissues, and the survival rate of patients with low Bak and BRD7 expression was significantly lower than that of patients with high Bak and BRD7 expression.  In addn., BRD7 activated Bak promoter activity and induced Bak expression in an indirect manner.  Furthermore, ectopic expression of BRD7 inhibited cell proliferation, tumor growth and sensitized cancer cells to paclitaxel, while knockdown of Bak abolished BRD7-mediated inhibitory effects on cell proliferation and paclitaxel sensitization in breast cancer cells whether in vitro and in vivo.  The results demonstrated that BRD7 inhibits cell proliferation and sensitizes breast cancer cells to paclitaxel by activating Bak; they also provide promising targets for the diagnosis and treatment of breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLZNf6xmTCUrVg90H21EOLACvtfcHk0lgADyV4yqQI0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtFygsrrI&md5=be5d7bd82cc0eb5436473694aaf53357</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.3892%2For.2018.6951&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252For.2018.6951%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DNiu%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DGuo%26aufirst%3DJ.%26aulast%3DXiong%26aufirst%3DW.%26aulast%3DZeng%26aufirst%3DZ.%26aulast%3DFan%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DNie%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DG.%26aulast%3DGui%26aufirst%3DR.%26aulast%3DLuo%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DM.%26atitle%3DBromodomain-containing%2520Protein%25207%2520Sensitizes%2520Breast%2520Cancer%2520Cells%2520to%2520Paclitaxel%2520by%2520Activating%2520Bcl2-antagonist%252FKiller%2520Protein%26jtitle%3DOncol.%2520Rep.%26date%3D2019%26volume%3D41%26spage%3D1487%26epage%3D1496%26doi%3D10.3892%2For.2018.6951" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clark, P. G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vieira, L. C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorova, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singletone, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monteiro, O. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennet, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baronio, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Méndez, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, D. J.</span></span> <span> </span><span class="NLM_article-title">LP99: Discovery and Synthesis of the First Selective BRD7/9 Bromodomain Inhibitor</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">6217</span>– <span class="NLM_lpage">6221</span>, <span class="refDoi"> DOI: 10.1002/anie.201501394</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1002%2Fanie.201501394" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmtlGqsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2015&pages=6217-6221&author=P.+G.+K.+Clarkauthor=L.+C.+C.+Vieiraauthor=C.+Tallantauthor=O.+Fedorovaauthor=D.+C.+Singletoneauthor=C.+M.+Rogersauthor=O.+P.+Monteiroauthor=J.+M.+Bennetauthor=R.+Baronioauthor=S.+M%C3%BCllerauthor=D.+L.+Danielsauthor=J.+M%C3%A9ndezauthor=S.+Knappauthor=P.+E.+Brennanauthor=D.+J.+Dixon&title=LP99%3A+Discovery+and+Synthesis+of+the+First+Selective+BRD7%2F9+Bromodomain+Inhibitor&doi=10.1002%2Fanie.201501394"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">LP99: Discovery and Synthesis of the First Selective BRD7/9 Bromodomain Inhibitor</span></div><div class="casAuthors">Clark, Peter G. K.; Vieira, Lucas C. C.; Tallant, Cynthia; Fedorov, Oleg; Singleton, Dean C.; Rogers, Catherine M.; Monteiro, Octovia P.; Bennett, James M.; Baronio, Roberta; Mueller, Susanne; Daniels, Danette L.; Mendez, Jacqui; Knapp, Stefan; Brennan, Paul E.; Dixon, Darren J.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6217-6221</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The bromodomain-contg. proteins BRD9 and BRD7 are part of the human SWI/SNF chromatin-remodeling complexes BAF and PBAF.  To date, no selective inhibitor for BRD7/9 has been reported despite its potential value as a biol. tool or as a lead for future therapeutics.  The quinolone-fused lactam LP99 (I) is now reported as the first potent and selective inhibitor of the BRD7 and BRD9 bromodomains.  Development of LP99 from a fragment hit was expedited through balancing structure-based inhibitor design and biophys. characterization against tractable chem. synthesis: Complexity-building nitro-Mannich/lactamization cascade processes allowed for early structure-activity relationship studies whereas an enantioselective organocatalytic nitro-Mannich reaction enabled the synthesis of the lead scaffold in enantioenriched form and on scale.  This epigenetic probe was shown to inhibit the assocn. of BRD7 and BRD9 to acetylated histones in vitro and in cells.  Moreover, LP99 was used to demonstrate that BRD7/9 plays a role in regulating pro-inflammatory cytokine secretion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplfOmbNMpdPbVg90H21EOLACvtfcHk0lgADyV4yqQI0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmtlGqsrg%253D&md5=9dee2b352045f7ae5efcbdc83f240a3f</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1002%2Fanie.201501394&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201501394%26sid%3Dliteratum%253Aachs%26aulast%3DClark%26aufirst%3DP.%2BG.%2BK.%26aulast%3DVieira%26aufirst%3DL.%2BC.%2BC.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DFedorova%26aufirst%3DO.%26aulast%3DSingletone%26aufirst%3DD.%2BC.%26aulast%3DRogers%26aufirst%3DC.%2BM.%26aulast%3DMonteiro%26aufirst%3DO.%2BP.%26aulast%3DBennet%26aufirst%3DJ.%2BM.%26aulast%3DBaronio%26aufirst%3DR.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DDaniels%26aufirst%3DD.%2BL.%26aulast%3DM%25C3%25A9ndez%26aufirst%3DJ.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DDixon%26aufirst%3DD.%2BJ.%26atitle%3DLP99%253A%2520Discovery%2520and%2520Synthesis%2520of%2520the%2520First%2520Selective%2520BRD7%252F9%2520Bromodomain%2520Inhibitor%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2015%26volume%3D54%26spage%3D6217%26epage%3D6221%26doi%3D10.1002%2Fanie.201501394" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Theodoulou, N. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bannister, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becher, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bit, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dittmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewes, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewry, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leveridge, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molnar, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robson, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomkinson, N. C. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouzarides, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphreys, P. G.</span></span> <span> </span><span class="NLM_article-title">Discovery of I-BRD9, a Selective Cell Active Chemical Probe for Bromodomain Containing Protein 9 Inhibition</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1425</span>– <span class="NLM_lpage">1439</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00256</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00256" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmtlaktb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1425-1439&author=N.+H.+Theodoulouauthor=P.+Bamboroughauthor=A.+J.+Bannisterauthor=I.+Becherauthor=R.+A.+Bitauthor=K.+H.+Cheauthor=C.-w.+Chungauthor=A.+Dittmannauthor=G.+Drewesauthor=D.+H.+Drewryauthor=L.+Gordonauthor=P.+Grandiauthor=M.+Leveridgeauthor=M.+Lindonauthor=A.-M.+Michonauthor=J.+Molnarauthor=S.+C.+Robsonauthor=N.+C.+O.+Tomkinsonauthor=T.+Kouzaridesauthor=R.+K.+Prinjhaauthor=P.+G.+Humphreys&title=Discovery+of+I-BRD9%2C+a+Selective+Cell+Active+Chemical+Probe+for+Bromodomain+Containing+Protein+9+Inhibition&doi=10.1021%2Facs.jmedchem.5b00256"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of I-BRD9, a Selective Cell Active Chemical Probe for Bromodomain Containing Protein 9 Inhibition</span></div><div class="casAuthors">Theodoulou, Natalie H.; Bamborough, Paul; Bannister, Andrew J.; Becher, Isabelle; Bit, Rino A.; Che, Ka Hing; Chung, Chun-wa; Dittmann, Antje; Drewes, Gerard; Drewry, David H.; Gordon, Laurie; Grandi, Paola; Leveridge, Melanie; Lindon, Matthew; Michon, Anne-Marie; Molnar, Judit; Robson, Samuel C.; Tomkinson, Nicholas C. O.; Kouzarides, Tony; Prinjha, Rab K.; Humphreys, Philip G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1425-1439</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Acetylation of histone lysine residues is one of the most well-studied post-translational modifications of chromatin, selectively recognized by bromodomain "reader" modules.  Inhibitors of the bromodomain and extra terminal domain (BET) family of bromodomains have shown profound anticancer and anti-inflammatory properties, generating much interest in targeting other bromodomain-contg. proteins for disease treatment.  Herein, we report the discovery of I-BRD9, the first selective cellular chem. probe for bromodomain-contg. protein 9 (BRD9).  I-BRD9 was identified through structure-based design, leading to greater than 700-fold selectivity over the BET family and 200-fold over the highly homologous bromodomain-contg. protein 7 (BRD7).  I-BRD9 was used to identify genes regulated by BRD9 in Kasumi-1 cells involved in oncol. and immune response pathways and to the best of our knowledge, represents the first selective tool compd. available to elucidate the cellular phenotype of BRD9 bromodomain inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVmjhksIDyTrVg90H21EOLACvtfcHk0lgADyV4yqQI0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmtlaktb8%253D&md5=4e8f6aa664797fe54772b64c81cf2a83</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00256%26sid%3Dliteratum%253Aachs%26aulast%3DTheodoulou%26aufirst%3DN.%2BH.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DBannister%26aufirst%3DA.%2BJ.%26aulast%3DBecher%26aufirst%3DI.%26aulast%3DBit%26aufirst%3DR.%2BA.%26aulast%3DChe%26aufirst%3DK.%2BH.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DDittmann%26aufirst%3DA.%26aulast%3DDrewes%26aufirst%3DG.%26aulast%3DDrewry%26aufirst%3DD.%2BH.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DLeveridge%26aufirst%3DM.%26aulast%3DLindon%26aufirst%3DM.%26aulast%3DMichon%26aufirst%3DA.-M.%26aulast%3DMolnar%26aufirst%3DJ.%26aulast%3DRobson%26aufirst%3DS.%2BC.%26aulast%3DTomkinson%26aufirst%3DN.%2BC.%2BO.%26aulast%3DKouzarides%26aufirst%3DT.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DHumphreys%26aufirst%3DP.%2BG.%26atitle%3DDiscovery%2520of%2520I-BRD9%252C%2520a%2520Selective%2520Cell%2520Active%2520Chemical%2520Probe%2520for%2520Bromodomain%2520Containing%2520Protein%25209%2520Inhibition%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1425%26epage%3D1439%26doi%3D10.1021%2Facs.jmedchem.5b00256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span> <i>SGC | TP-472 A BRD9/7 Probe</i>; <a href="http://www.thesgc.org/chemical-probes/TP-472" class="extLink">http://www.thesgc.org/chemical-probes/TP-472</a>(accessed 3 October 2019).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+SGC+%7C+TP-472+A+BRD9%2F7+Probe%3B+http%3A%2F%2Fwww.thesgc.org%2Fchemical-probes%2FTP-472%28accessed+3+October+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26jtitle%3DSGC%2520%257C%2520TP-472%2520A%2520BRD9%252F7%2520Probe" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="note"><p class="first last">For alternative molecules from the same series, see:</p></div><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hay, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monteiro, O. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schofield, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span> <span> </span><span class="NLM_article-title">Design and Synthesis of Potent and Selective Inhibitors of BRD7 and BRD9 Bromodomains</span>. <i>Med. Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1381</span>– <span class="NLM_lpage">1386</span>, <span class="refDoi"> DOI: 10.1039/C5MD00152H</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1039%2FC5MD00152H" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=1381-1386&author=D.+A.+Hayauthor=C.+M.+Rogersauthor=O.+Fedorovauthor=C.+Tallantauthor=S.+Martinauthor=O.+P.+Monteiroauthor=S.+M%C3%BCllerauthor=S.+Knappauthor=C.+J.+Schofieldauthor=P.+E.+Brennan&title=Design+and+Synthesis+of+Potent+and+Selective+Inhibitors+of+BRD7+and+BRD9+Bromodomains&doi=10.1039%2FC5MD00152H"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1039%2FC5MD00152H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5MD00152H%26sid%3Dliteratum%253Aachs%26aulast%3DHay%26aufirst%3DD.%2BA.%26aulast%3DRogers%26aufirst%3DC.%2BM.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DMonteiro%26aufirst%3DO.%2BP.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26atitle%3DDesign%2520and%2520Synthesis%2520of%2520Potent%2520and%2520Selective%2520Inhibitors%2520of%2520BRD7%2520and%2520BRD9%2520Bromodomains%26jtitle%3DMed.%2520Chem.%2520Commun.%26date%3D2015%26volume%3D6%26spage%3D1381%26epage%3D1386%26doi%3D10.1039%2FC5MD00152H" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martin, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koegl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bader, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cockcroft, X.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiegen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstberger, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofmann, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hohmann, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knesl, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kornigg, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nar, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rumpel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaaf, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steurer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakoc, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeeb, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoephel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boehmelt, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McConnell, D.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Design of an in Vivo Active Selective BRD9 Inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">4462</span>– <span class="NLM_lpage">4475</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01865</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01865" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC28Xjt1Sit7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4462-4475&author=L.+J.+Martinauthor=M.+Koeglauthor=G.+Baderauthor=X.-L.+Cockcroftauthor=O.+Fedorovauthor=D.+Fiegenauthor=T.+Gerstbergerauthor=M.+H.+Hofmannauthor=A.+F.+Hohmannauthor=D.+Kesslerauthor=S.+Knappauthor=P.+Kneslauthor=S.+Korniggauthor=S.+M%C3%BCllerauthor=H.+Narauthor=C.+Rogersauthor=K.+Rumpelauthor=O.+Schaafauthor=S.+Steurerauthor=C.+Tallantauthor=C.+R.+Vakocauthor=M.+Zeebauthor=A.+Zoephelauthor=M.+Pearsonauthor=G.+Boehmeltauthor=D.+McConnell&title=Structure-Based+Design+of+an+in+Vivo+Active+Selective+BRD9+Inhibitor&doi=10.1021%2Facs.jmedchem.5b01865"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of an in Vivo Active Selective BRD9 Inhibitor</span></div><div class="casAuthors">Martin, Laetitia J.; Koegl, Manfred; Bader, Gerd; Cockcroft, Xiao-Ling; Fedorov, Oleg; Fiegen, Dennis; Gerstberger, Thomas; Hofmann, Marco H.; Hohmann, Anja F.; Kessler, Dirk; Knapp, Stefan; Knesl, Petr; Kornigg, Stefan; Mueller, Susanne; Nar, Herbert; Rogers, Catherine; Rumpel, Klaus; Schaaf, Otmar; Steurer, Steffen; Tallant, Cynthia; Vakoc, Christopher R.; Zeeb, Markus; Zoephel, Andreas; Pearson, Mark; Boehmelt, Guido; McConnell, Darryl</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4462-4475</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Components of the chromatin remodelling switch/sucrose nonfermentable (SWI/SNF) complex are recurrently mutated in tumors, suggesting that altering the activity of the complex plays a role in oncogenesis.  However, the role that the individual subunits play in this process is not clear.  We set out to develop an inhibitor compd. targeting the bromodomain of BRD9 in order to evaluate its function within the SWI/SNF complex.  Here, we present the discovery and development of a potent and selective BRD9 bromodomain inhibitor series based on a new pyridinone-like scaffold.  Crystallog. information on the inhibitors bound to BRD9 guided their development with respect to potency for BRD9 and selectivity against BRD4.  These compds. modulate BRD9 bromodomain cellular function and display antitumor activity in an AML xenograft model.  Two chem. probes, BI-7273 (1) and BI-9564 (2), were identified that should prove to be useful in further exploring BRD9 bromodomain biol. in both in vitro and in vivo settings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWHs4hA8bijbVg90H21EOLACvtfcHk0liHT_ZCg8p3vg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xjt1Sit7w%253D&md5=b1f4394a7a410273c800b65ee9697c7e</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01865&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01865%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DL.%2BJ.%26aulast%3DKoegl%26aufirst%3DM.%26aulast%3DBader%26aufirst%3DG.%26aulast%3DCockcroft%26aufirst%3DX.-L.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DFiegen%26aufirst%3DD.%26aulast%3DGerstberger%26aufirst%3DT.%26aulast%3DHofmann%26aufirst%3DM.%2BH.%26aulast%3DHohmann%26aufirst%3DA.%2BF.%26aulast%3DKessler%26aufirst%3DD.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DKnesl%26aufirst%3DP.%26aulast%3DKornigg%26aufirst%3DS.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DNar%26aufirst%3DH.%26aulast%3DRogers%26aufirst%3DC.%26aulast%3DRumpel%26aufirst%3DK.%26aulast%3DSchaaf%26aufirst%3DO.%26aulast%3DSteurer%26aufirst%3DS.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26aulast%3DZeeb%26aufirst%3DM.%26aulast%3DZoephel%26aufirst%3DA.%26aulast%3DPearson%26aufirst%3DM.%26aulast%3DBoehmelt%26aufirst%3DG.%26aulast%3DMcConnell%26aufirst%3DD.%26atitle%3DStructure-Based%2520Design%2520of%2520an%2520in%2520Vivo%2520Active%2520Selective%2520BRD9%2520Inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D4462%26epage%3D4475%26doi%3D10.1021%2Facs.jmedchem.5b01865" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Remillard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brien, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sonnett, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seo, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dastjerdi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wühr, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Degradation of the BAF Complex Factor BRD9 by Heterobifunctional Ligands</span>. <i>Angew. Chem. Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">5738</span>– <span class="NLM_lpage">5743</span>, <span class="refDoi"> DOI: 10.1002/anie.201611281</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1002%2Fanie.201611281" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmtFGrurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=5738-5743&author=D.+Remillardauthor=D.+L.+Buckleyauthor=J.+Paulkauthor=G.+L.+Brienauthor=M.+Sonnettauthor=H.-S.+Seoauthor=S.+Dastjerdiauthor=M.+W%C3%BChrauthor=S.+Dhe-Paganonauthor=S.+A.+Armstrongauthor=J.+E.+Bradner&title=Degradation+of+the+BAF+Complex+Factor+BRD9+by+Heterobifunctional+Ligands&doi=10.1002%2Fanie.201611281"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Degradation of the BAF Complex Factor BRD9 by Heterobifunctional Ligands</span></div><div class="casAuthors">Remillard, David; Buckley, Dennis L.; Paulk, Joshiawa; Brien, Gerard L.; Sonnett, Matthew; Seo, Hyuk-Soo; Dastierdi, Shiva; Wuehr, Martin; Dhe-Paganon, Sirano; Armstrong, Scott A.; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">5738-5743</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The bromodomain-contg. protein BRD9, a subunit of the human BAF (SWI/SNF) nucleosome remodeling complex, has emerged as an attractive therapeutic target in cancer.  Despite the development of chem. probes targeting the BRD9 bromodomain, there is a limited understanding of BRD9 function beyond acetyl-lysine recognition.  We have therefore created the first BRD9-directed chem. degraders, through iterative design and testing of heterobifunctional ligands that bridge the BRD9 bromodomain and the cereblon E3 ubiquitin ligase complex.  Degraders of BRD9 exhibit markedly enhanced potency compared to parental ligands (10- to 100-fold).  Parallel study of degraders with divergent BRD9-binding chemotypes in models of acute myeloid leukemia resolves bromodomain polypharmacol. in this emerging drug class.  Together, these findings reveal the tractability of non-BET bromodomain contg. proteins to chem. degrdn., and highlight lead compd. dBRD9 as a tool for the study of BRD9.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQ9FU2Ipaq27Vg90H21EOLACvtfcHk0liHT_ZCg8p3vg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmtFGrurk%253D&md5=2eb6562cc9d976c7ae6070c9a419e9ea</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1002%2Fanie.201611281&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201611281%26sid%3Dliteratum%253Aachs%26aulast%3DRemillard%26aufirst%3DD.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DBrien%26aufirst%3DG.%2BL.%26aulast%3DSonnett%26aufirst%3DM.%26aulast%3DSeo%26aufirst%3DH.-S.%26aulast%3DDastjerdi%26aufirst%3DS.%26aulast%3DW%25C3%25BChr%26aufirst%3DM.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DDegradation%2520of%2520the%2520BAF%2520Complex%2520Factor%2520BRD9%2520by%2520Heterobifunctional%2520Ligands%26jtitle%3DAngew.%2520Chem.%2520Int.%2520Ed.%26date%3D2017%26volume%3D56%26spage%3D5738%26epage%3D5743%26doi%3D10.1002%2Fanie.201611281" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brien, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Remillard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hemming, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chabon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wynne, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dillon, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cagney, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Mierlo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baltissen, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vermeulen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fröhling, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakoc, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, S. A.</span></span> <span> </span><span class="NLM_article-title">Targeted Degradation of BRD9 Reverses Oncogenic Gene Expression in Synovial Sarcoma</span>. <i>eLife</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">e41305</span> <span class="refDoi"> DOI: 10.7554/eLife.41305</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.7554%2FeLife.41305" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=30431433" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlyqs7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2018&author=G.+L.+Brienauthor=D.+Remillardauthor=J.+Shiauthor=M.+L.+Hemmingauthor=J.+Chabonauthor=K.+Wynneauthor=E.+T.+Dillonauthor=G.+Cagneyauthor=G.+Van+Mierloauthor=M.+P.+Baltissenauthor=M.+Vermeulenauthor=J.+Qiauthor=S.+Fr%C3%B6hlingauthor=N.+S.+Grayauthor=J.+E.+Bradnerauthor=C.+R.+Vakocauthor=S.+A.+Armstrong&title=Targeted+Degradation+of+BRD9+Reverses+Oncogenic+Gene+Expression+in+Synovial+Sarcoma&doi=10.7554%2FeLife.41305"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted degradation of BRD9 reverses oncogenic gene expression in synovial sarcoma</span></div><div class="casAuthors">Brien, Gerard L.; Remillard, David; Shi, Junwei; Hemming, Matthew L.; Chabon, Jonathon; Wynne, Kieran; Dillon, Euge Ne T.; Cagney, Gerard; Van Mierlo, Guido; Baltissen, Marijke P.; Vermeulen, Michiel; Qi, Jun; Froehling, Stefan; Gray, Nathanael S.; Bradner, James E.; Vakoc, Christopher R.; Armstrong, Scott A.</div><div class="citationInfo"><span class="NLM_cas:title">eLife</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e41305/1-e41305/26</span>CODEN:
                <span class="NLM_cas:coden">ELIFA8</span>;
        ISSN:<span class="NLM_cas:issn">2050-084X</span>.
    
            (<span class="NLM_cas:orgname">eLife Sciences Publications Ltd.</span>)
        </div><div class="casAbstract">Synovial sarcoma tumors contain a characteristic fusion protein, SS18-SSX, which drives disease development.  Targeting oncogenic fusion proteins presents an attractive therapeutic opportunity.  However, SS18-SSX has proven intractable for therapeutic intervention.  Using a domain-focused CRISPR screen we identified the bromodomain of BRD9 as a crit. functional dependency in synovial sarcoma.  BRD9 is a component of SS18-SSX contg. BAF complexes in synovial sarcoma cells; and integration of BRD9 into these complexes is crit. for cell growth.  Moreover BRD9 and SS18-SSX co-localize extensively on the synovial sarcoma genome.  Remarkably, synovial sarcoma cells are highly sensitive to a novel small mol. degrader of BRD9, while other sarcoma subtypes are unaffected.  Degrdn. of BRD9 induces downregulation of oncogenic transcriptional programs and inhibits tumor progression in vivo.  We demonstrate that BRD9 supports oncogenic mechanisms underlying the SS18-SSX fusion in synovial sarcoma and highlight targeted degrdn. of BRD9 as a potential therapeutic opportunity in this disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfd6-D9YlmrrVg90H21EOLACvtfcHk0lgZS7rJPGtZ_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlyqs7fF&md5=cbda64457406cd0d8fbd1211678b04b3</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.7554%2FeLife.41305&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.41305%26sid%3Dliteratum%253Aachs%26aulast%3DBrien%26aufirst%3DG.%2BL.%26aulast%3DRemillard%26aufirst%3DD.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DHemming%26aufirst%3DM.%2BL.%26aulast%3DChabon%26aufirst%3DJ.%26aulast%3DWynne%26aufirst%3DK.%26aulast%3DDillon%26aufirst%3DE.%2BT.%26aulast%3DCagney%26aufirst%3DG.%26aulast%3DVan%2BMierlo%26aufirst%3DG.%26aulast%3DBaltissen%26aufirst%3DM.%2BP.%26aulast%3DVermeulen%26aufirst%3DM.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DFr%25C3%25B6hling%26aufirst%3DS.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26atitle%3DTargeted%2520Degradation%2520of%2520BRD9%2520Reverses%2520Oncogenic%2520Gene%2520Expression%2520in%2520Synovial%2520Sarcoma%26jtitle%3DeLife%26date%3D2018%26volume%3D7%26doi%3D10.7554%2FeLife.41305" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vartanian, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Côté, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duplessis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flynn, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiefer, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nasveschuk, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romero, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandy, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsui, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zawadzke, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnuson, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cochran, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stokoe, D.</span></span> <span> </span><span class="NLM_article-title">Inhibition of Bromodomain-Containing Protein 9 for the Prevention of Epigenetically-Defined Drug Resistance</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">3534</span>– <span class="NLM_lpage">3541</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2017.05.063</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1016%2Fj.bmcl.2017.05.063" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=28606761" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVSjtb%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=3534-3541&author=T.+D.+Crawfordauthor=S.+Vartanianauthor=A.+C%C3%B4t%C3%A9author=S.+Bellonauthor=M.+Duplessisauthor=E.+M.+Flynnauthor=M.+Hewittauthor=H.-R.+Huangauthor=J.+R.+Kieferauthor=J.+Murrayauthor=C.+G.+Nasveschukauthor=E.+Pardoauthor=F.+A.+Romeroauthor=P.+Sandyauthor=Y.+Tangauthor=A.+M.+Taylorauthor=V.+Tsuiauthor=J.+Wangauthor=S.+Wangauthor=L.+Zawadzkeauthor=B.+K.+Albrechtauthor=S.+R.+Magnusonauthor=A.+G.+Cochranauthor=D.+Stokoe&title=Inhibition+of+Bromodomain-Containing+Protein+9+for+the+Prevention+of+Epigenetically-Defined+Drug+Resistance&doi=10.1016%2Fj.bmcl.2017.05.063"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of bromodomain-containing protein 9 for the prevention of epigenetically-defined drug resistance</span></div><div class="casAuthors">Crawford, Terry D.; Vartanian, Steffan; Cote, Alexandre; Bellon, Steve; Duplessis, Martin; Flynn, E. Megan; Hewitt, Michael; Huang, Hon-Ren; Kiefer, James R.; Murray, Jeremy; Nasveschuk, Christopher G.; Pardo, Eneida; Romero, F. Anthony; Sandy, Peter; Tang, Yong; Taylor, Alexander M.; Tsui, Vickie; Wang, Jian; Wang, Shumei; Zawadzke, Laura; Albrecht, Brian K.; Magnuson, Steven R.; Cochran, Andrea G.; Stokoe, David</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">3534-3541</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Bromodomain-contg. protein 9 (BRD9), an epigenetic "reader" of acetylated lysines on post-translationally modified histone proteins, is upregulated in multiple cancer cell lines.  To assess the functional role of BRD9 in cancer cell lines, the authors identified a small-mol. inhibitor of the BRD9 bromodomain.  Starting from a pyrrolopyridone lead, the authors used structure-based drug design to identify a potent and highly selective in vitro tool compd. 11, (GNE-375).  While this compd. showed minimal effects in cell viability or gene expression assays, it showed remarkable potency in preventing the emergence of a drug tolerant population in EGFR mutant PC9 cells treated with EGFR inhibitors.  Such tolerance has been linked to an altered epigenetic state, and 11 decreased BRD9 binding to chromatin, and this was assocd. with decreased expression of ALDH1A1, a gene previously shown to be important in drug tolerance.  BRD9 inhibitors may therefore show utility in preventing epigenetically-defined drug resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWdETG4wZY9rVg90H21EOLACvtfcHk0lgZS7rJPGtZ_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVSjtb%252FO&md5=f489e702cfb41cfce9d3d570bfde87ed</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2017.05.063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2017.05.063%26sid%3Dliteratum%253Aachs%26aulast%3DCrawford%26aufirst%3DT.%2BD.%26aulast%3DVartanian%26aufirst%3DS.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DA.%26aulast%3DBellon%26aufirst%3DS.%26aulast%3DDuplessis%26aufirst%3DM.%26aulast%3DFlynn%26aufirst%3DE.%2BM.%26aulast%3DHewitt%26aufirst%3DM.%26aulast%3DHuang%26aufirst%3DH.-R.%26aulast%3DKiefer%26aufirst%3DJ.%2BR.%26aulast%3DMurray%26aufirst%3DJ.%26aulast%3DNasveschuk%26aufirst%3DC.%2BG.%26aulast%3DPardo%26aufirst%3DE.%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DSandy%26aufirst%3DP.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DZawadzke%26aufirst%3DL.%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26aulast%3DMagnuson%26aufirst%3DS.%2BR.%26aulast%3DCochran%26aufirst%3DA.%2BG.%26aulast%3DStokoe%26aufirst%3DD.%26atitle%3DInhibition%2520of%2520Bromodomain-Containing%2520Protein%25209%2520for%2520the%2520Prevention%2520of%2520Epigenetically-Defined%2520Drug%2520Resistance%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D27%26spage%3D3534%26epage%3D3541%26doi%3D10.1016%2Fj.bmcl.2017.05.063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsui, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flynn, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Côteé, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Audia, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beresini, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burdick, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dakin, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duplessis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Good, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jayaram, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiefer, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nasveschuk, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poy, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romero, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zawadzke, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnuson, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cochran, A. G.</span></span> <span> </span><span class="NLM_article-title">Diving into the Water: Inducible Binding Conformations for BRD4, TAF1(2), BRD9, and CECR2 Bromodomains</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">5391</span>– <span class="NLM_lpage">5402</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00264</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00264" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC28XosFSnsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=5391-5402&author=T.+D.+Crawfordauthor=V.+Tsuiauthor=E.+M.+Flynnauthor=S.+Wangauthor=A.+M.+Taylorauthor=A.+C%C3%B4te%C3%A9author=J.+E.+Audiaauthor=M.+H.+Beresiniauthor=D.+J.+Burdickauthor=R.+Cummingsauthor=L.+A.+Dakinauthor=M.+Duplessisauthor=A.+C.+Goodauthor=M.+C.+Hewittauthor=H.-R.+Huangauthor=H.+Jayaramauthor=J.+R.+Kieferauthor=Y.+Jiangauthor=J.+Murrayauthor=C.+G.+Nasveschukauthor=E.+Pardoauthor=F.+Poyauthor=F.+A.+Romeroauthor=Y.+Tangauthor=J.+Wangauthor=Z.+Xuauthor=L.+E.+Zawadzkeauthor=X.+Zhuauthor=B.+K.+Albrechtauthor=S.+R.+Magnusonauthor=S.+Bellonauthor=A.+G.+Cochran&title=Diving+into+the+Water%3A+Inducible+Binding+Conformations+for+BRD4%2C+TAF1%282%29%2C+BRD9%2C+and+CECR2+Bromodomains&doi=10.1021%2Facs.jmedchem.6b00264"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Diving into the Water: Inducible Binding Conformations for BRD4, TAF1(2), BRD9, and CECR2 Bromodomains</span></div><div class="casAuthors">Crawford, Terry D.; Tsui, Vickie; Flynn, E. Megan; Wang, Shumei; Taylor, Alexander M.; Cote, Alexandre; Audia, James E.; Beresini, Maureen H.; Burdick, Daniel J.; Cummings, Richard; Dakin, Les A.; Duplessis, Martin; Good, Andrew C.; Hewitt, Michael C.; Huang, Hon-Ren; Jayaram, Hariharan; Kiefer, James R.; Jiang, Ying; Murray, Jeremy; Nasveschuk, Christopher G.; Pardo, Eneida; Poy, Florence; Romero, F. Anthony; Tang, Yong; Wang, Jian; Xu, Zhaowu; Zawadzke, Laura E.; Zhu, Xiaoyu; Albrecht, Brian K.; Magnuson, Steven R.; Bellon, Steve; Cochran, Andrea G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5391-5402</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The biol. role played by non-BET bromodomains remains poorly understood, and it is therefore imperative to identify potent and highly selective inhibitors to effectively explore the biol. of individual bromodomain proteins.  A ligand-efficient nonselective bromodomain inhibitor was identified from a 6-Me pyrrolopyridone fragment.  Small hydrophobic substituents replacing the N-Me group were designed directing toward the conserved bromodomain water pocket, and two distinct binding conformations were then obsd.  The substituents either directly displaced and rearranged the conserved solvent network, as in BRD4(1) and TAF1(2), or induced a narrow hydrophobic channel adjacent to the lipophilic shelf, as in BRD9 and CECR2.  The preference of distinct substituents for individual bromodomains provided selectivity handles useful for future lead optimization efforts for selective BRD9, CECR2, and TAF1(2) inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTLnuVCIKdHbVg90H21EOLACvtfcHk0liEdspR0MX4Yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XosFSnsbs%253D&md5=bf92e94c5fd3ee36bd8441294ca85185</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00264&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00264%26sid%3Dliteratum%253Aachs%26aulast%3DCrawford%26aufirst%3DT.%2BD.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DFlynn%26aufirst%3DE.%2BM.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DC%25C3%25B4te%25C3%25A9%26aufirst%3DA.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26aulast%3DBeresini%26aufirst%3DM.%2BH.%26aulast%3DBurdick%26aufirst%3DD.%2BJ.%26aulast%3DCummings%26aufirst%3DR.%26aulast%3DDakin%26aufirst%3DL.%2BA.%26aulast%3DDuplessis%26aufirst%3DM.%26aulast%3DGood%26aufirst%3DA.%2BC.%26aulast%3DHewitt%26aufirst%3DM.%2BC.%26aulast%3DHuang%26aufirst%3DH.-R.%26aulast%3DJayaram%26aufirst%3DH.%26aulast%3DKiefer%26aufirst%3DJ.%2BR.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DMurray%26aufirst%3DJ.%26aulast%3DNasveschuk%26aufirst%3DC.%2BG.%26aulast%3DPardo%26aufirst%3DE.%26aulast%3DPoy%26aufirst%3DF.%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DZawadzke%26aufirst%3DL.%2BE.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26aulast%3DMagnuson%26aufirst%3DS.%2BR.%26aulast%3DBellon%26aufirst%3DS.%26aulast%3DCochran%26aufirst%3DA.%2BG.%26atitle%3DDiving%2520into%2520the%2520Water%253A%2520Inducible%2520Binding%2520Conformations%2520for%2520BRD4%252C%2520TAF1%25282%2529%252C%2520BRD9%252C%2520and%2520CECR2%2520Bromodomains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D5391%26epage%3D5402%26doi%3D10.1021%2Facs.jmedchem.6b00264" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Humphreys, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craggs, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayhow, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hussain, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renaux, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suckling, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span> <span> </span><span class="NLM_article-title">Discovery of a Potent, Cell Penetrant, and Selective P300/CBP-Associated Factor (PCAF)/General Control Nonderepressible 5 (GCN5) Bromodomain Chemical Probe</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">695</span>– <span class="NLM_lpage">709</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01566</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01566" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFCgs77N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=695-709&author=P.+G.+Humphreysauthor=P.+Bamboroughauthor=C.+Chungauthor=P.+D.+Craggsauthor=L.+Gordonauthor=P.+Grandiauthor=T.+G.+Hayhowauthor=J.+Hussainauthor=K.+L.+Jonesauthor=M.+Lindonauthor=A.-M.+Michonauthor=J.+F.+Renauxauthor=C.+J.+Sucklingauthor=D.+F.+Toughauthor=R.+K.+Prinjha&title=Discovery+of+a+Potent%2C+Cell+Penetrant%2C+and+Selective+P300%2FCBP-Associated+Factor+%28PCAF%29%2FGeneral+Control+Nonderepressible+5+%28GCN5%29+Bromodomain+Chemical+Probe&doi=10.1021%2Facs.jmedchem.6b01566"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Potent, Cell Penetrant, and Selective p300/CBP-Associated Factor (PCAF)/General Control Nonderepressible 5 (GCN5) Bromodomain Chemical Probe</span></div><div class="casAuthors">Humphreys, Philip G.; Bamborough, Paul; Chung, Chun-wa; Craggs, Peter D.; Gordon, Laurie; Grandi, Paola; Hayhow, Thomas G.; Hussain, Jameed; Jones, Katherine L.; Lindon, Matthew; Michon, Anne-Marie; Renaux, Jessica F.; Suckling, Colin J.; Tough, David F.; Prinjha, Rab K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">695-709</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">P300/CREB binding protein assocd. factor (PCAF/KAT2B) and general control non-derepressible 5 (GCN5/KAT2A) are multidomain proteins that have been implicated in retroviral infection, inflammation pathways and cancer development.  However, outside of viral replication, little is known about the dependence of these effects on the C-terminal bromodomain.  Herein, the authors report GSK4027 as a chem. probe for the PCAF/GCN5 bromodomain, together with GSK4028 as an enantiomeric neg. control.  The probe was optimized from a weakly potent, non-selective pyridazinone hit to deliver high potency for the PCAF/GCN5 bromodomain, high soly., cellular target engagement and ≥18,000-fold selectivity over the BET family, together with ≥70-fold selectivity over the wider bromodomain families.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBkLwjcv1aZLVg90H21EOLACvtfcHk0liEdspR0MX4Yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFCgs77N&md5=fd4fdbe8827e26769fc17b09120c7826</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01566&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01566%26sid%3Dliteratum%253Aachs%26aulast%3DHumphreys%26aufirst%3DP.%2BG.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.%26aulast%3DCraggs%26aufirst%3DP.%2BD.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DHayhow%26aufirst%3DT.%2BG.%26aulast%3DHussain%26aufirst%3DJ.%26aulast%3DJones%26aufirst%3DK.%2BL.%26aulast%3DLindon%26aufirst%3DM.%26aulast%3DMichon%26aufirst%3DA.-M.%26aulast%3DRenaux%26aufirst%3DJ.%2BF.%26aulast%3DSuckling%26aufirst%3DC.%2BJ.%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26atitle%3DDiscovery%2520of%2520a%2520Potent%252C%2520Cell%2520Penetrant%252C%2520and%2520Selective%2520P300%252FCBP-Associated%2520Factor%2520%2528PCAF%2529%252FGeneral%2520Control%2520Nonderepressible%25205%2520%2528GCN5%2529%2520Bromodomain%2520Chemical%2520Probe%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D695%26epage%3D709%26doi%3D10.1021%2Facs.jmedchem.6b01566" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallego, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, M. P.</span></span> <span> </span><span class="NLM_article-title">Lipophilic Efficiency as an Important Metric in Drug Design</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">6401</span>– <span class="NLM_lpage">6420</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00077</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00077" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC1cXmsVahsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=6401-6420&author=T.+W.+Johnsonauthor=R.+A.+Gallegoauthor=M.+P.+Edwards&title=Lipophilic+Efficiency+as+an+Important+Metric+in+Drug+Design&doi=10.1021%2Facs.jmedchem.8b00077"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Lipophilic Efficiency as an Important Metric in Drug Design</span></div><div class="casAuthors">Johnson, Ted W.; Gallego, Rebecca A.; Edwards, Martin P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6401-6420</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Lipophilic efficiency (LipE) is an important metric that has been increasingly applied in drug discovery medicinal chem. lead optimization programs.  In this perspective, using literature drug discovery examples, we discuss the concept of rigorously applying LipE to guide medicinal chem. lead optimization toward drug candidates with potential for superior in vivo efficacy and safety, esp. when guided by physiochem. property-based optimization (PPBO).  Also highlighted are examples of small structural modifications such as addn. of single atoms, small functional groups, and cyclizations that produce large increases in LipE.  Understanding the factors that may contribute to LipE changes through anal. of ligand-protein crystal structures and using structure-based drug design (SBDD) to increase LipE by design is also discussed.  Herein we advocate for use of LipE anal. coupled with PPBO and SBDD as an efficient mechanism for drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXQ8DOcOj6GrVg90H21EOLACvtfcHk0liEdspR0MX4Yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXmsVahsbg%253D&md5=29f38880914fa75c33be3a640ace4309</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00077&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00077%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DT.%2BW.%26aulast%3DGallego%26aufirst%3DR.%2BA.%26aulast%3DEdwards%26aufirst%3DM.%2BP.%26atitle%3DLipophilic%2520Efficiency%2520as%2520an%2520Important%2520Metric%2520in%2520Drug%2520Design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D6401%26epage%3D6420%26doi%3D10.1021%2Facs.jmedchem.8b00077" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span> <i>BROMOscan recombinant protein binding assays were
carried out at
DiscoveRx</i>; <a href="http://www.discoverx.com" class="extLink">http://www.discoverx.com</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+BROMOscan+recombinant+protein+binding+assays+were%0Acarried+out+at%0ADiscoveRx%3B+http%3A%2F%2Fwww.discoverx.com."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26jtitle%3DBROMOscan%2520recombinant%2520protein%2520binding%2520assays%2520were%250Acarried%2520out%2520at%250ADiscoveRx" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nittinger, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibbons, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eigenbrot, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maurer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiefer, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuglstatter, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortwine, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsui, V.</span></span> <span> </span><span class="NLM_article-title">Water Molecules in Protein–Ligand Interfaces. Evaluation of Software Tools and SAR Comparison</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">307</span>– <span class="NLM_lpage">330</span>, <span class="refDoi"> DOI: 10.1007/s10822-019-00187-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1007%2Fs10822-019-00187-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=30756207" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC1MXmsVylsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2019&pages=307-330&author=E.+Nittingerauthor=P.+Gibbonsauthor=C.+Eigenbrotauthor=D.+R.+Daviesauthor=B.+Maurerauthor=C.+L.+Yuauthor=J.+R.+Kieferauthor=A.+Kuglstatterauthor=J.+Murrayauthor=D.+F.+Ortwineauthor=Y.+Tangauthor=V.+Tsui&title=Water+Molecules+in+Protein%E2%80%93Ligand+Interfaces.+Evaluation+of+Software+Tools+and+SAR+Comparison&doi=10.1007%2Fs10822-019-00187-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Water molecules in protein-ligand interfaces. Evaluation of software tools and SAR comparison</span></div><div class="casAuthors">Nittinger, Eva; Gibbons, Paul; Eigenbrot, Charles; Davies, Doug R.; Maurer, Brigitte; Yu, Christine L.; Kiefer, James R.; Kuglstatter, Andreas; Murray, Jeremy; Ortwine, Daniel F.; Tang, Yong; Tsui, Vickie</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">307-330</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Abstr.: Targeting the interaction with or displacement of the 'right' water mol. can significantly increase inhibitor potency in structure-guided drug design.  Multiple computational approaches exist to predict which waters should be targeted for displacement to achieve the largest gain in potency.  However, the relative success of different methods remains underexplored.  Here, we present a comparison of the ability of five water prediction programs (3D-RISM, SZMAP, WaterFLAP, WaterRank, and WaterMap) to predict crystallog. water locations, calc. their binding free energies, and to relate differences in these energies to obsd. changes in potency.  The structural cohort included nine Bruton's Tyrosine Kinase (BTK) structures, and nine bromodomain structures.  Each program accurately predicted the locations of most crystallog. water mols.  However, the predicted binding free energies correlated poorly with the obsd. changes in inhibitor potency when solvent atoms were displaced by chem. changes in closely related compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwiAvnyBQBhbVg90H21EOLACvtfcHk0lgpkvc_PeDgZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXmsVylsr8%253D&md5=4d44c3dbdddf732ea8030574f42c9ac1</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1007%2Fs10822-019-00187-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-019-00187-y%26sid%3Dliteratum%253Aachs%26aulast%3DNittinger%26aufirst%3DE.%26aulast%3DGibbons%26aufirst%3DP.%26aulast%3DEigenbrot%26aufirst%3DC.%26aulast%3DDavies%26aufirst%3DD.%2BR.%26aulast%3DMaurer%26aufirst%3DB.%26aulast%3DYu%26aufirst%3DC.%2BL.%26aulast%3DKiefer%26aufirst%3DJ.%2BR.%26aulast%3DKuglstatter%26aufirst%3DA.%26aulast%3DMurray%26aufirst%3DJ.%26aulast%3DOrtwine%26aufirst%3DD.%2BF.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DTsui%26aufirst%3DV.%26atitle%3DWater%2520Molecules%2520in%2520Protein%25E2%2580%2593Ligand%2520Interfaces.%2520Evaluation%2520of%2520Software%2520Tools%2520and%2520SAR%2520Comparison%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2019%26volume%3D33%26spage%3D307%26epage%3D330%26doi%3D10.1007%2Fs10822-019-00187-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiest, O.</span></span> <span> </span><span class="NLM_article-title">Protein Dynamics and Structural Waters in Bromodomains</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>, <span class="NLM_elocation-id">e0186570</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0186570</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1371%2Fjournal.pone.0186570" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=29077715" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFGms7bN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&author=X.+Zhangauthor=K.+Chenauthor=Y.-D.+Wuauthor=O.+Wiest&title=Protein+Dynamics+and+Structural+Waters+in+Bromodomains&doi=10.1371%2Fjournal.pone.0186570"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Protein dynamics and structural waters in bromodomains</span></div><div class="casAuthors">Zhang, Xiaoxiao; Chen, Kai; Wu, Yun-Dong; Wiest, Olaf</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">e0186570/1-e0186570/16</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Bromodomains are epigenetic readers of acetylated lysines that are integral parts of histone tails.  The 61 bromodomains in humans are structurally highly conserved but specifically bind to widely varying recognition motifs, suggesting that dynamic rather than static factors are responsible for recognition selectivity.  To test this hypothesis, the dynamics of the binding sites and structural water mols. of four bromodomains (ATAD2, BAZ2B, BRD2(1) and CREBBP) representing four different subtypes is studied with 1 μs MD simulations using the RSFF2 force field.  The different dynamics of the ZA-loops and BC-loops between the four bromodomains leads to distinct patterns for the opening and closing of the binding pocket.  This in turn dets. the structural and energetic properties of the structural waters in the binding pocket, suggesting that these waters are not only important for the recognition itself, as has been proposed previously, but also contribute to the selectivity of different bromodomains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdqzaoNSjS-7Vg90H21EOLACvtfcHk0lgpkvc_PeDgZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFGms7bN&md5=3ed02a6025d67e5c0e8b511c3dc6756e</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0186570&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0186570%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DWu%26aufirst%3DY.-D.%26aulast%3DWiest%26aufirst%3DO.%26atitle%3DProtein%2520Dynamics%2520and%2520Structural%2520Waters%2520in%2520Bromodomains%26jtitle%3DPLoS%2520One%26date%3D2017%26volume%3D12%26doi%3D10.1371%2Fjournal.pone.0186570" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aldeghi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bodkin, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Essex, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biggin, P. C.</span></span> <span> </span><span class="NLM_article-title">Large-Scale Analysis of Water Stability in Bromodomain Binding Pockets with Grand Canonical Monte Carlo</span>. <i>Commun. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">12</span>, <span class="refDoi"> DOI: 10.1038/s42004-018-0019-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1038%2Fs42004-018-0019-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC1MXpvVyqt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2018&pages=1-12&author=M.+Aldeghiauthor=G.+A.+Rossauthor=M.+J.+Bodkinauthor=J.+W.+Essexauthor=S.+Knappauthor=P.+C.+Biggin&title=Large-Scale+Analysis+of+Water+Stability+in+Bromodomain+Binding+Pockets+with+Grand+Canonical+Monte+Carlo&doi=10.1038%2Fs42004-018-0019-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Large-scale analysis of water stability in bromodomain binding pockets with grand canonical Monte Carlo</span></div><div class="casAuthors">Aldeghi, Matteo; Ross, Gregory A.; Bodkin, Michael J.; Essex, Jonathan W.; Knapp, Stefan; Biggin, Philip C.</div><div class="citationInfo"><span class="NLM_cas:title">Communications Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-12</span>CODEN:
                <span class="NLM_cas:coden">CCOHCT</span>;
        ISSN:<span class="NLM_cas:issn">2399-3669</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Conserved water mols. are of interest in drug design, as displacement of such waters can lead to higher affinity ligands, and in some cases, contribute towards selectivity.  Bromodomains, small protein domains involved in the epigenetic regulation of gene transcription, display a network of four conserved water mols. in their binding pockets and have recently been the focus of intense medicinal chem. efforts.  Understanding why certain bromodomains have displaceable water mols. and others do not is extremely challenging, and it remains unclear which water mols. in a given bromodomain can be targeted for displacement.  Here we est. the stability of the conserved water mols. in 35 bromodomains via binding free energy calcns. using all-atom grand canonical Monte Carlo simulations.  Encouraging quant. agreement to the available exptl. evidence is found.  We thus discuss the expected ease of water displacement in different bromodomains and the implications for ligand selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7J7hnRA65t7Vg90H21EOLACvtfcHk0ljHwZceAosX5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXpvVyqt7c%253D&md5=7af46c69fc53664ec719a178485457fe</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fs42004-018-0019-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs42004-018-0019-x%26sid%3Dliteratum%253Aachs%26aulast%3DAldeghi%26aufirst%3DM.%26aulast%3DRoss%26aufirst%3DG.%2BA.%26aulast%3DBodkin%26aufirst%3DM.%2BJ.%26aulast%3DEssex%26aufirst%3DJ.%2BW.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBiggin%26aufirst%3DP.%2BC.%26atitle%3DLarge-Scale%2520Analysis%2520of%2520Water%2520Stability%2520in%2520Bromodomain%2520Binding%2520Pockets%2520with%2520Grand%2520Canonical%2520Monte%2520Carlo%26jtitle%3DCommun.%2520Chem.%26date%3D2018%26volume%3D1%26spage%3D1%26epage%3D12%26doi%3D10.1038%2Fs42004-018-0019-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bridges, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craggs, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furze, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karamshi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Locke, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, S. C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pogány, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roa, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span> <span> </span><span class="NLM_article-title">A Qualified Success: Discovery of a New Series of ATAD2 Bromodomain Inhibitors with a Novel Binding Mode Using High-Throughput Screening and Hit Qualification</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">7506</span>– <span class="NLM_lpage">7525</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00673</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00673" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsFeqsL%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=7506-7525&author=P.+Bamboroughauthor=C.+Chungauthor=E.+H.+Demontauthor=A.+M.+Bridgesauthor=P.+D.+Craggsauthor=D.+P.+Dixonauthor=P.+Francisauthor=R.+C.+Furzeauthor=P.+Grandiauthor=E.+J.+Jonesauthor=B.+Karamshiauthor=K.+Lockeauthor=S.+C.+C.+Lucasauthor=A.-M.+Michonauthor=D.+J.+Mitchellauthor=P.+Pog%C3%A1nyauthor=R.+K.+Prinjhaauthor=C.+Rauauthor=A.+M.+Roaauthor=A.+D.+Robertsauthor=R.+J.+Sheppardauthor=R.+J.+Watson&title=A+Qualified+Success%3A+Discovery+of+a+New+Series+of+ATAD2+Bromodomain+Inhibitors+with+a+Novel+Binding+Mode+Using+High-Throughput+Screening+and+Hit+Qualification&doi=10.1021%2Facs.jmedchem.9b00673"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">A Qualified Success: Discovery of a New Series of ATAD2 Bromodomain Inhibitors with a Novel Binding Mode Using High-Throughput Screening and Hit Qualification</span></div><div class="casAuthors">Bamborough, Paul; Chung, Chun-wa; Demont, Emmanuel H.; Bridges, Angela M.; Craggs, Peter D.; Dixon, David P.; Francis, Peter; Furze, Rebecca C.; Grandi, Paola; Jones, Emma J.; Karamshi, Bhumika; Locke, Kelly; Lucas, Simon C. C.; Michon, Anne-Marie; Mitchell, Darren J.; Pogany, Peter; Prinjha, Rab K.; Rau, Christina; Roa, Ana Maria; Roberts, Andrew D.; Sheppard, Robert J.; Watson, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">7506-7525</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromodomain of ATAD2 has proved to be one of the least-tractable proteins within this target class.  Here, we describe the discovery of a new class of inhibitors by high-throughput screening and show how the difficulties encountered in establishing a screening triage capable of finding progressible hits were overcome by data-driven optimization.  Despite the prevalence of nonspecific hits and an exceptionally low progressible hit rate (0.001%), our optimized hit qualification strategy employing orthogonal biophys. methods enabled us to identify a single active series.  The compds. have a novel ATAD2 binding mode with noncanonical features including the displacement of all conserved water mols. within the active site and a halogen-bonding interaction.  In addn. to reporting this new series and preliminary structure-activity relationship, we demonstrate the value of diversity screening to complement the knowledge-based approach used in our previous ATAD2 work.  We also exemplify tactics that can increase the chance of success when seeking new chem. starting points for novel and less-tractable targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomDZyktXHrhbVg90H21EOLACvtfcHk0ljHwZceAosX5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsFeqsL%252FJ&md5=8132ccb7f63c3300bd173c42cd2ac026</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00673%26sid%3Dliteratum%253Aachs%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DBridges%26aufirst%3DA.%2BM.%26aulast%3DCraggs%26aufirst%3DP.%2BD.%26aulast%3DDixon%26aufirst%3DD.%2BP.%26aulast%3DFrancis%26aufirst%3DP.%26aulast%3DFurze%26aufirst%3DR.%2BC.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DJones%26aufirst%3DE.%2BJ.%26aulast%3DKaramshi%26aufirst%3DB.%26aulast%3DLocke%26aufirst%3DK.%26aulast%3DLucas%26aufirst%3DS.%2BC.%2BC.%26aulast%3DMichon%26aufirst%3DA.-M.%26aulast%3DMitchell%26aufirst%3DD.%2BJ.%26aulast%3DPog%25C3%25A1ny%26aufirst%3DP.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRau%26aufirst%3DC.%26aulast%3DRoa%26aufirst%3DA.%2BM.%26aulast%3DRoberts%26aufirst%3DA.%2BD.%26aulast%3DSheppard%26aufirst%3DR.%2BJ.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26atitle%3DA%2520Qualified%2520Success%253A%2520Discovery%2520of%2520a%2520New%2520Series%2520of%2520ATAD2%2520Bromodomain%2520Inhibitors%2520with%2520a%2520Novel%2520Binding%2520Mode%2520Using%2520High-Throughput%2520Screening%2520and%2520Hit%2520Qualification%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D7506%26epage%3D7525%26doi%3D10.1021%2Facs.jmedchem.9b00673" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsui, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Audia, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burdick, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beresini, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Côté, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duplessis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flynn, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jayaram, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joshi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nasveschuk, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poy, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romero, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zawadzke, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnuson, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cochran, A. G.</span></span> <span> </span><span class="NLM_article-title">GNE-371, a Potent and Selective Chemical Probe for the Second Bromodomains of Human Transcription-Initiation-Factor TFIID Subunit 1 and Transcription-Initiation-Factor TFIID Subunit 1-Like</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">9301</span>– <span class="NLM_lpage">9315</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01225</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01225" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVKgtLjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=9301-9315&author=S.+Wangauthor=V.+Tsuiauthor=T.+D.+Crawfordauthor=J.+E.+Audiaauthor=D.+J.+Burdickauthor=M.+H.+Beresiniauthor=A.+C%C3%B4t%C3%A9author=R.+Cummingsauthor=M.+Duplessisauthor=E.+M.+Flynnauthor=M.+C.+Hewittauthor=H.-R.+Huangauthor=H.+Jayaramauthor=Y.+Jiangauthor=S.+Joshiauthor=J.+Murrayauthor=C.+G.+Nasveschukauthor=E.+Pardoauthor=F.+Poyauthor=F.+A.+Romeroauthor=Y.+Tangauthor=A.+M.+Taylorauthor=J.+Wangauthor=Z.+Xuauthor=L.+E.+Zawadzkeauthor=X.+Zhuauthor=B.+K.+Albrechtauthor=S.+R.+Magnusonauthor=S.+Bellonauthor=A.+G.+Cochran&title=GNE-371%2C+a+Potent+and+Selective+Chemical+Probe+for+the+Second+Bromodomains+of+Human+Transcription-Initiation-Factor+TFIID+Subunit+1+and+Transcription-Initiation-Factor+TFIID+Subunit+1-Like&doi=10.1021%2Facs.jmedchem.8b01225"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">GNE-371, a Potent and Selective Chemical Probe for the Second Bromodomains of Human Transcription-Initiation-Factor TFIID Subunit 1 and Transcription-Initiation-Factor TFIID Subunit 1-like</span></div><div class="casAuthors">Wang, Shumei; Tsui, Vickie; Crawford, Terry D.; Audia, James E.; Burdick, Daniel J.; Beresini, Maureen H.; Cote, Alexandre; Cummings, Richard; Duplessis, Martin; Flynn, E. Megan; Hewitt, Michael C.; Huang, Hon-Ren; Jayaram, Hariharan; Jiang, Ying; Joshi, Shivangi; Murray, Jeremy; Nasveschuk, Christopher G.; Pardo, Eneida; Poy, Florence; Romero, F. Anthony; Tang, Yong; Taylor, Alexander M.; Wang, Jian; Xu, Zhaowu; Zawadzke, Laura E.; Zhu, Xiaoyu; Albrecht, Brian K.; Magnuson, Steven R.; Bellon, Steve; Cochran, Andrea G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">9301-9315</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The biol. functions of the dual bromodomains of human transcription-initiation-factor TFIID subunit 1 (TAF1(1,2)) remain unknown, although TAF1 has been identified as a potential target for oncol. research.  Here, the authors describe the discovery of a potent and selective in vitro tool compd. for TAF1(2), starting from a previously reported lead.  A cocrystal structure of lead compd. 2 bound to TAF1(2) enabled structure-based design and structure-activity-relation studies that ultimately led to the authors' in vitro tool compd., 27 (GNE-371).  Compd. 27 binds TAF1(2) with an IC50 of 10 nM while maintaining excellent selectivity over other bromodomain-family members.  Compd. 27 is also active in a cellular-TAF1(2) target-engagement assay (IC50 = 38 nM) and exhibits antiproliferative synergy with the BET inhibitor JQ1, suggesting engagement of endogenous TAF1 by 27 and further supporting the use of 27 in mechanistic and target-validation studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-B5cO-w0BOrVg90H21EOLACvtfcHk0ljHwZceAosX5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVKgtLjK&md5=6f13888f3d123f6c0119b047c44a11a7</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01225&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01225%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DS.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DCrawford%26aufirst%3DT.%2BD.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26aulast%3DBurdick%26aufirst%3DD.%2BJ.%26aulast%3DBeresini%26aufirst%3DM.%2BH.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DA.%26aulast%3DCummings%26aufirst%3DR.%26aulast%3DDuplessis%26aufirst%3DM.%26aulast%3DFlynn%26aufirst%3DE.%2BM.%26aulast%3DHewitt%26aufirst%3DM.%2BC.%26aulast%3DHuang%26aufirst%3DH.-R.%26aulast%3DJayaram%26aufirst%3DH.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DJoshi%26aufirst%3DS.%26aulast%3DMurray%26aufirst%3DJ.%26aulast%3DNasveschuk%26aufirst%3DC.%2BG.%26aulast%3DPardo%26aufirst%3DE.%26aulast%3DPoy%26aufirst%3DF.%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DZawadzke%26aufirst%3DL.%2BE.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26aulast%3DMagnuson%26aufirst%3DS.%2BR.%26aulast%3DBellon%26aufirst%3DS.%26aulast%3DCochran%26aufirst%3DA.%2BG.%26atitle%3DGNE-371%252C%2520a%2520Potent%2520and%2520Selective%2520Chemical%2520Probe%2520for%2520the%2520Second%2520Bromodomains%2520of%2520Human%2520Transcription-Initiation-Factor%2520TFIID%2520Subunit%25201%2520and%2520Transcription-Initiation-Factor%2520TFIID%2520Subunit%25201-Like%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D9301%26epage%3D9315%26doi%3D10.1021%2Facs.jmedchem.8b01225" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castex, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aldeghi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monteiro, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trzupek, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstenberger, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bountra, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willmann, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biggin, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schüle, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Günther, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span> <span> </span><span class="NLM_article-title">Selective Targeting of the BRG/PB1 Bromodomains Impairs Embryonic and Trophoblast Stem Cell Maintenance</span>. <i>Sci. Adv.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>1</i></span>, <span class="NLM_elocation-id">e1500723</span> <span class="refDoi"> DOI: 10.1126/sciadv.1500723</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1126%2Fsciadv.1500723" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=26702435" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC28XovFaktLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2015&author=O.+Fedorovauthor=J.+Castexauthor=C.+Tallantauthor=D.+R.+Owenauthor=S.+Martinauthor=M.+Aldeghiauthor=O.+Monteiroauthor=P.+Filippakopoulosauthor=S.+Picaudauthor=J.+D.+Trzupekauthor=B.+S.+Gerstenbergerauthor=C.+Bountraauthor=D.+Willmannauthor=C.+Wellsauthor=M.+Philpottauthor=C.+Rogersauthor=P.+C.+Bigginauthor=P.+E.+Brennanauthor=M.+E.+Bunnageauthor=R.+Sch%C3%BCleauthor=T.+G%C3%BCntherauthor=S.+Knappauthor=S.+M%C3%BCller&title=Selective+Targeting+of+the+BRG%2FPB1+Bromodomains+Impairs+Embryonic+and+Trophoblast+Stem+Cell+Maintenance&doi=10.1126%2Fsciadv.1500723"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Selective targeting of the BRG/PB1 bromodomains impairs embryonic and trophoblast stem cell maintenance</span></div><div class="casAuthors">Fedorov, Oleg; Castex, Josefina; Tallant, Cynthia; Owen, Dafydd R.; Martin, Sarah; Aldeghi, Matteo; Monteiro, Octovia; Filippakopoulos, Panagis; Picaud, Sarah; Trzupek, John D.; Gerstenberger, Brian S.; Bountra, Chas; Willmann, Dominica; Wells, Christopher; Philpott, Martin; Rogers, Catherine; Biggin, Philip C.; Brennan, Paul E.; Bunnage, Mark E.; Schule, Roland; Gunther, Thomas; Knapp, Stefan; Muller, Susanne</div><div class="citationInfo"><span class="NLM_cas:title">Science Advances</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">e1500723/1-e1500723/11</span>CODEN:
                <span class="NLM_cas:coden">SACDAF</span>;
        ISSN:<span class="NLM_cas:issn">2375-2548</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Mammalian SWI/SNF [also called Brg/Brahma-assocd. factors (BAFs)] are evolutionarily conserved chromatin remodeling complexes regulating gene transcription programs during development and stem cell differentiation.  BAF complexes contain an ATP (ATP)-driven remodeling enzyme (either BRG1 or BRM) and multiple protein interaction domains including bromodomains, an evolutionary conserved acetyl lysine-dependent protein interaction motif that recruits transcriptional regulators to acetylated chromatin.  We report a potent and cell active protein interaction inhibitor, PFI-3, that selectively binds to essential BAF bromodomains.  The high specificity of PFI-3 was achieved on the basis of a novel binding mode of a salicylic acid head group that led to the replacement of water mols. typically maintained in other bromodomain inhibitor complexes.  We show that exposure of embryonic stem cells to PFI-3 led to deprivation of stemness and deregulated lineage specification.  Furthermore, differentiation of trophoblast stem cells in the presence of PFI-3 was markedly enhanced.  The data present a key function of BAF bromodomains in stem cell maintenance and differentiation, introducing a novel versatile chem. probe for studies on acetylation-dependent cellular processes controlled by BAF remodeling complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiGyzl98dGOLVg90H21EOLACvtfcHk0liQpqsSK0KHtQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XovFaktLw%253D&md5=52757e667cc5afeb4c3a151fc8ef3442</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1126%2Fsciadv.1500723&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fsciadv.1500723%26sid%3Dliteratum%253Aachs%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DCastex%26aufirst%3DJ.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DAldeghi%26aufirst%3DM.%26aulast%3DMonteiro%26aufirst%3DO.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DTrzupek%26aufirst%3DJ.%2BD.%26aulast%3DGerstenberger%26aufirst%3DB.%2BS.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DWillmann%26aufirst%3DD.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DRogers%26aufirst%3DC.%26aulast%3DBiggin%26aufirst%3DP.%2BC.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DSch%25C3%25BCle%26aufirst%3DR.%26aulast%3DG%25C3%25BCnther%26aufirst%3DT.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26atitle%3DSelective%2520Targeting%2520of%2520the%2520BRG%252FPB1%2520Bromodomains%2520Impairs%2520Embryonic%2520and%2520Trophoblast%2520Stem%2520Cell%2520Maintenance%26jtitle%3DSci.%2520Adv.%26date%3D2015%26volume%3D1%26doi%3D10.1126%2Fsciadv.1500723" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gerstenberger, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trzupek, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedele, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parikh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samas, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Mahony, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berg, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallares, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torrey, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samardjiev, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nasipak, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Padilla-Benavides, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imbalzano, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nickerson, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen, D. R.</span></span> <span> </span><span class="NLM_article-title">Identification of a Chemical Probe for Family VIII Bromodomains Through Optimization of a Fragment Hit</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">4800</span>– <span class="NLM_lpage">4811</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00012</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00012" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC28Xmsl2rs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4800-4811&author=B.+S.+Gerstenbergerauthor=J.+D.+Trzupekauthor=C.+Tallantauthor=O.+Fedorovauthor=P.+Filippakopoulosauthor=P.+E.+Brennanauthor=V.+Fedeleauthor=S.+Martinauthor=S.+Picaudauthor=C.+Rogersauthor=M.+Parikhauthor=A.+Taylorauthor=B.+Samasauthor=A.+O%E2%80%99Mahonyauthor=E.+Bergauthor=G.+Pallaresauthor=A.+D.+Torreyauthor=D.+K.+Treiberauthor=I.+J.+Samardjievauthor=B.+T.+Nasipakauthor=T.+Padilla-Benavidesauthor=Q.+Wuauthor=A.+N.+Imbalzanoauthor=J.+A.+Nickersonauthor=M.+E.+Bunnageauthor=S.+M%C3%BCllerauthor=S.+Knappauthor=D.+R.+Owen&title=Identification+of+a+Chemical+Probe+for+Family+VIII+Bromodomains+Through+Optimization+of+a+Fragment+Hit&doi=10.1021%2Facs.jmedchem.6b00012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a Chemical Probe for Family VIII Bromodomains through Optimization of a Fragment Hit</span></div><div class="casAuthors">Gerstenberger, Brian S.; Trzupek, John D.; Tallant, Cynthia; Fedorov, Oleg; Filippakopoulos, Panagis; Brennan, Paul E.; Fedele, Vita; Martin, Sarah; Picaud, Sarah; Rogers, Catherine; Parikh, Mihir; Taylor, Alexandria; Samas, Brian; O'Mahony, Alison; Berg, Ellen; Pallares, Gabriel; Torrey, Adam V.; Treiber, Daniel K.; Samardjiev, Ivan J.; Nasipak, Brian T.; Padilla-Benavides, Teresita; Wu, Qiong; Imbalzano, Anthony N.; Nickerson, Jeffrey A.; Bunnage, Mark E.; Muller, Susanne; Knapp, Stefan; Owen, Dafydd R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4800-4811</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The acetyl post-translational modification of chromatin at selected histone lysine residues is interpreted by an acetyl-lysine specific interaction with bromodomain reader modules.  Here we report the discovery of the potent, acetyl-lysine-competitive, and cell active inhibitor PFI-3 that binds to certain family VIII bromodomains while displaying significant, broader bromodomain family selectivity.  The high specificity of PFI-3 for family VIII was achieved through a novel bromodomain binding mode of a phenolic headgroup that led to the unusual displacement of water mols. that are generally retained by most other bromodomain inhibitors reported to date.  The medicinal chem. program that led to PFI-3 from an initial fragment screening hit is described in detail, and addnl. analogs with differing family VIII bromodomain selectivity profiles are also reported.  We also describe the full pharmacol. characterization of PFI-3 as a chem. probe, along with phenotypic data on adipocyte and myoblast cell differentiation assays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZlrg_xjnZGbVg90H21EOLACvtfcHk0liQpqsSK0KHtQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xmsl2rs74%253D&md5=6cb73e76a19f7fb44153f8cbfeff9cf6</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00012%26sid%3Dliteratum%253Aachs%26aulast%3DGerstenberger%26aufirst%3DB.%2BS.%26aulast%3DTrzupek%26aufirst%3DJ.%2BD.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DFedele%26aufirst%3DV.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DRogers%26aufirst%3DC.%26aulast%3DParikh%26aufirst%3DM.%26aulast%3DTaylor%26aufirst%3DA.%26aulast%3DSamas%26aufirst%3DB.%26aulast%3DO%25E2%2580%2599Mahony%26aufirst%3DA.%26aulast%3DBerg%26aufirst%3DE.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DTorrey%26aufirst%3DA.%2BD.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DSamardjiev%26aufirst%3DI.%2BJ.%26aulast%3DNasipak%26aufirst%3DB.%2BT.%26aulast%3DPadilla-Benavides%26aufirst%3DT.%26aulast%3DWu%26aufirst%3DQ.%26aulast%3DImbalzano%26aufirst%3DA.%2BN.%26aulast%3DNickerson%26aufirst%3DJ.%2BA.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26atitle%3DIdentification%2520of%2520a%2520Chemical%2520Probe%2520for%2520Family%2520VIII%2520Bromodomains%2520Through%2520Optimization%2520of%2520a%2520Fragment%2520Hit%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D4800%26epage%3D4811%26doi%3D10.1021%2Facs.jmedchem.6b00012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sutherell, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monteiro, O. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yapp, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuchs, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siejka, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brenton, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ley, S. V.</span></span> <span> </span><span class="NLM_article-title">Identification and Development of 2,3-Dihydropyrrolo[1,2- a ]Quinazolin-5(1 H )-One Inhibitors Targeting Bromodomains Within the Switch/Sucrose Nonfermenting Complex</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">5095</span>– <span class="NLM_lpage">5101</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01997</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01997" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC28Xmsl2rt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=5095-5101&author=C.+L.+Sutherellauthor=C.+Tallantauthor=O.+P.+Monteiroauthor=C.+Yappauthor=J.+E.+Fuchsauthor=O.+Fedorovauthor=P.+Siejkaauthor=S.+M%C3%BCllerauthor=S.+Knappauthor=J.+D.+Brentonauthor=P.+E.+Brennanauthor=S.+V.+Ley&title=Identification+and+Development+of+2%2C3-Dihydropyrrolo%5B1%2C2-+a+%5DQuinazolin-5%281+H+%29-One+Inhibitors+Targeting+Bromodomains+Within+the+Switch%2FSucrose+Nonfermenting+Complex&doi=10.1021%2Facs.jmedchem.5b01997"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and Development of 2,3-Dihydropyrrolo[1,2-a]quinazolin-5(1H)-one Inhibitors Targeting Bromodomains within the Switch/Sucrose Nonfermenting Complex</span></div><div class="casAuthors">Sutherell, Charlotte L.; Tallant, Cynthia; Monteiro, Octovia P.; Yapp, Clarence; Fuchs, Julian E.; Fedorov, Oleg; Siejka, Paulina; Muller, Suzanne; Knapp, Stefan; Brenton, James D.; Brennan, Paul E.; Ley, Steven V.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5095-5101</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomain contg. proteins PB1, SMARCA4, and SMARCA2 are important components of SWI/SNF chromatin remodeling complexes.  We identified bromodomain inhibitors that target these proteins and display unusual binding modes involving water displacement from the KAc binding site.  The best compd. (I) binds the fifth bromodomain of PB1 with a KD of 124 nM, SMARCA2B and SMARCA4 with KD values of 262 and 417 nM, resp., and displays excellent selectivity over bromodomains other than PB1, SMARCA2, and SMARCA4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfnWlpl3TimLVg90H21EOLACvtfcHk0lg5PRu8uI4m1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xmsl2rt7s%253D&md5=a27afa2c97bd74462a09c9ec231f61ef</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01997&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01997%26sid%3Dliteratum%253Aachs%26aulast%3DSutherell%26aufirst%3DC.%2BL.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DMonteiro%26aufirst%3DO.%2BP.%26aulast%3DYapp%26aufirst%3DC.%26aulast%3DFuchs%26aufirst%3DJ.%2BE.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DSiejka%26aufirst%3DP.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBrenton%26aufirst%3DJ.%2BD.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DLey%26aufirst%3DS.%2BV.%26atitle%3DIdentification%2520and%2520Development%2520of%25202%252C3-Dihydropyrrolo%255B1%252C2-%2520a%2520%255DQuinazolin-5%25281%2520H%2520%2529-One%2520Inhibitors%2520Targeting%2520Bromodomains%2520Within%2520the%2520Switch%252FSucrose%2520Nonfermenting%2520Complex%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D5095%26epage%3D5101%26doi%3D10.1021%2Facs.jmedchem.5b01997" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="note"><p class="first last">Peptides substituted with propionylated and butyrylated and crotonylated lysines have also been shown to be capable of binding to human bromodomains, albeit with weak affinity; see</p></div><div class="NLM_citation" id="cit47a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Vollmuth, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geyer, M.</span></span> <span> </span><span class="NLM_article-title">Interaction of Propionylated and Butyrylated Histone H3 Lysine Marks with Brd4 Bromodomains</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">6768</span>– <span class="NLM_lpage">6772</span>, <span class="refDoi"> DOI: 10.1002/anie.201002724</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1002%2Fanie.201002724" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFGnsbfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2010&pages=6768-6772&author=F.+Vollmuthauthor=M.+Geyer&title=Interaction+of+Propionylated+and+Butyrylated+Histone+H3+Lysine+Marks+with+Brd4+Bromodomains&doi=10.1002%2Fanie.201002724"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47aR"><div class="casContent"><span class="casTitleNuber">47a</span><div class="casTitle"><span class="NLM_cas:atitle">Interaction of Propionylated and Butyrylated Histone H3 Lysine Marks with Brd4 Bromodomains</span></div><div class="casAuthors">Vollmuth, Friederike; Geyer, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">6768-6772, S6768/1-S6768/9</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">In this study, we analyzed the interaction of propionyl- and butyryllysine residues on histone chains with Brd4 bromodomains and the structural basis of this interaction.  The binding affinities of both short-chain acylations were similar to or weaker than those obsd. for lysine acetylation.  The addnl. carbon atoms aligned between a highly conserved Pro1Phe motif, which suggests a general mode of bromodomain recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCelcpDi5QebVg90H21EOLACvtfcHk0lg5PRu8uI4m1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFGnsbfI&md5=ad48736d13205ddc66f4207a64ea3b8e</span></div><a href="/servlet/linkout?suffix=cit47a&amp;dbid=16384&amp;doi=10.1002%2Fanie.201002724&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201002724%26sid%3Dliteratum%253Aachs%26aulast%3DVollmuth%26aufirst%3DF.%26aulast%3DGeyer%26aufirst%3DM.%26atitle%3DInteraction%2520of%2520Propionylated%2520and%2520Butyrylated%2520Histone%2520H3%2520Lysine%2520Marks%2520with%2520Brd4%2520Bromodomains%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2010%26volume%3D49%26spage%3D6768%26epage%3D6772%26doi%3D10.1002%2Fanie.201002724" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit47b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Flynn, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, O. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poy, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oppikofer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellon, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cochran, A. G.</span></span> <span> </span><span class="NLM_article-title">A Subset of Human Bromodomains Recognizes Butyryllysine and Crotonyllysine Histone Peptide Modifications</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1801</span>– <span class="NLM_lpage">1814</span>, <span class="refDoi"> DOI: 10.1016/j.str.2015.08.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1016%2Fj.str.2015.08.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=26365797" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsV2ls7jN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2015&pages=1801-1814&author=E.+M.+Flynnauthor=O.+W.+Huangauthor=F.+Poyauthor=M.+Oppikoferauthor=S.+F.+Bellonauthor=Y.+Tangauthor=A.+G.+Cochran&title=A+Subset+of+Human+Bromodomains+Recognizes+Butyryllysine+and+Crotonyllysine+Histone+Peptide+Modifications&doi=10.1016%2Fj.str.2015.08.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47bR"><div class="casContent"><span class="casTitleNuber">47b</span><div class="casTitle"><span class="NLM_cas:atitle">A Subset of Human Bromodomains Recognizes Butyryllysine and Crotonyllysine Histone Peptide Modifications</span></div><div class="casAuthors">Flynn, E. Megan; Huang, Oscar W.; Poy, Florence; Oppikofer, Mariano; Bellon, Steve F.; Tang, Yong; Cochran, Andrea G.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1801-1814</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Bromodomains are epigenetic readers that are recruited to acetyllysine residues in histone tails.  Recent studies have identified non-acetyl acyllysine modifications, raising the possibility that these might be read by bromodomains.  Profiling the nearly complete human bromodomain family revealed that while most human bromodomains bind only the shorter acetyl and propionyl marks, the bromodomains of BRD9, CECR2, and the second bromodomain of TAF1 also recognize the longer butyryl mark.  In addn., the TAF1 s bromodomain is capable of binding crotonyl marks.  None of the human bromodomains tested binds succinyl marks.  We characterized structurally and biochem. the binding to different acyl groups, identifying bromodomain residues and structural attributes that contribute to specificity.  These studies demonstrate a surprising degree of plasticity in some human bromodomains but no single factor controlling specificity across the family.  The identification of candidate butyryl- and crotonyllysine readers supports the idea that these marks could have specific physiol. functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrv2tMgdAAflbVg90H21EOLACvtfcHk0lg5PRu8uI4m1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsV2ls7jN&md5=c3b216ef2596ea21196f531a5da5f1f3</span></div><a href="/servlet/linkout?suffix=cit47b&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2015.08.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2015.08.004%26sid%3Dliteratum%253Aachs%26aulast%3DFlynn%26aufirst%3DE.%2BM.%26aulast%3DHuang%26aufirst%3DO.%2BW.%26aulast%3DPoy%26aufirst%3DF.%26aulast%3DOppikofer%26aufirst%3DM.%26aulast%3DBellon%26aufirst%3DS.%2BF.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DCochran%26aufirst%3DA.%2BG.%26atitle%3DA%2520Subset%2520of%2520Human%2520Bromodomains%2520Recognizes%2520Butyryllysine%2520and%2520Crotonyllysine%2520Histone%2520Peptide%2520Modifications%26jtitle%3DStructure%26date%3D2015%26volume%3D23%26spage%3D1801%26epage%3D1814%26doi%3D10.1016%2Fj.str.2015.08.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Audia, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burdick, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bommi-Reddy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Côté, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duplessis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flynn, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jayaram, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joshi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiefer, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nasveschuk, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neiss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romero, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandy, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sims, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsui, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zawadzke, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnuson, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cochran, A. G.</span></span> <span> </span><span class="NLM_article-title">GNE-886: A Potent and Selective Inhibitor of the Cat Eye Syndrome Chromosome Region Candidate 2 Bromodomain (CECR2)</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">737</span>– <span class="NLM_lpage">741</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.7b00132</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.7b00132" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpt1Shsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=737-741&author=T.+D.+Crawfordauthor=J.+E.+Audiaauthor=S.+Bellonauthor=D.+J.+Burdickauthor=A.+Bommi-Reddyauthor=A.+C%C3%B4t%C3%A9author=R.+T.+Cummingsauthor=M.+Duplessisauthor=E.+M.+Flynnauthor=M.+Hewittauthor=H.-R.+Huangauthor=H.+Jayaramauthor=Y.+Jiangauthor=S.+Joshiauthor=J.+R.+Kieferauthor=J.+Murrayauthor=C.+G.+Nasveschukauthor=A.+Neissauthor=E.+Pardoauthor=F.+A.+Romeroauthor=P.+Sandyauthor=R.+J.+Simsauthor=Y.+Tangauthor=A.+M.+Taylorauthor=V.+Tsuiauthor=J.+Wangauthor=S.+Wangauthor=Y.+Wangauthor=Z.+Xuauthor=L.+Zawadzkeauthor=X.+Zhuauthor=B.+K.+Albrechtauthor=S.+R.+Magnusonauthor=A.+G.+Cochran&title=GNE-886%3A+A+Potent+and+Selective+Inhibitor+of+the+Cat+Eye+Syndrome+Chromosome+Region+Candidate+2+Bromodomain+%28CECR2%29&doi=10.1021%2Facsmedchemlett.7b00132"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">GNE-886: A Potent and Selective Inhibitor of the Cat Eye Syndrome Chromosome Region Candidate 2 Bromodomain (CECR2)</span></div><div class="casAuthors">Crawford, Terry D.; Audia, James E.; Bellon, Steve; Burdick, Daniel J.; Bommi-Reddy, Archana; Cote, Alexandre; Cummings, Richard T.; Duplessis, Martin; Flynn, E. Megan; Hewitt, Michael; Huang, Hon-Ren; Jayaram, Hariharan; Jiang, Ying; Joshi, Shivangi; Kiefer, James R.; Murray, Jeremy; Nasveschuk, Christopher G.; Neiss, Arianne; Pardo, Eneida; Romero, F. Anthony; Sandy, Peter; Sims, Robert J.; Tang, Yong; Taylor, Alexander M.; Tsui, Vickie; Wang, Jian; Wang, Shumei; Wang, Yongyun; Xu, Zhaowu; Zawadzke, Laura; Zhu, Xiaoqin; Albrecht, Brian K.; Magnuson, Steven R.; Cochran, Andrea G.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">737-741</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The biol. function of bromodomains, epigenetic readers of acetylated lysine residues, remains largely unknown.  Herein we report our efforts to discover a potent and selective inhibitor of the bromodomain of cat eye syndrome chromosome region candidate 2 (CECR2).  Screening of our internal medicinal chem. collection led to the identification of a pyrrolopyridone chem. lead, and subsequent structure-based drug design led to a potent and selective CECR2 bromodomain inhibitor (GNE-886) suitable for use as an in vitro tool compd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxtzb-gQeFAbVg90H21EOLACvtfcHk0lheMxyITdzwkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpt1Shsrw%253D&md5=f92e17923136477fc36edefd3bdd4998</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.7b00132&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.7b00132%26sid%3Dliteratum%253Aachs%26aulast%3DCrawford%26aufirst%3DT.%2BD.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26aulast%3DBellon%26aufirst%3DS.%26aulast%3DBurdick%26aufirst%3DD.%2BJ.%26aulast%3DBommi-Reddy%26aufirst%3DA.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DA.%26aulast%3DCummings%26aufirst%3DR.%2BT.%26aulast%3DDuplessis%26aufirst%3DM.%26aulast%3DFlynn%26aufirst%3DE.%2BM.%26aulast%3DHewitt%26aufirst%3DM.%26aulast%3DHuang%26aufirst%3DH.-R.%26aulast%3DJayaram%26aufirst%3DH.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DJoshi%26aufirst%3DS.%26aulast%3DKiefer%26aufirst%3DJ.%2BR.%26aulast%3DMurray%26aufirst%3DJ.%26aulast%3DNasveschuk%26aufirst%3DC.%2BG.%26aulast%3DNeiss%26aufirst%3DA.%26aulast%3DPardo%26aufirst%3DE.%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DSandy%26aufirst%3DP.%26aulast%3DSims%26aufirst%3DR.%2BJ.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DZawadzke%26aufirst%3DL.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26aulast%3DMagnuson%26aufirst%3DS.%2BR.%26aulast%3DCochran%26aufirst%3DA.%2BG.%26atitle%3DGNE-886%253A%2520A%2520Potent%2520and%2520Selective%2520Inhibitor%2520of%2520the%2520Cat%2520Eye%2520Syndrome%2520Chromosome%2520Region%2520Candidate%25202%2520Bromodomain%2520%2528CECR2%2529%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D8%26spage%3D737%26epage%3D741%26doi%3D10.1021%2Facsmedchemlett.7b00132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Readshaw, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollerton, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Upton, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lynne, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Besley, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boughtflower, B. J.</span></span> <span> </span><span class="NLM_article-title">Use of Calculated Physicochemical Properties to Enhance Quantitative Response When Using Charged Aerosol Detection</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>89</i></span>,  <span class="NLM_fpage">1772</span>– <span class="NLM_lpage">1777</span>, <span class="refDoi"> DOI: 10.1021/acs.analchem.6b04060</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.analchem.6b04060" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFams7%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=2017&pages=1772-1777&author=M.+W.+Robinsonauthor=A.+P.+Hillauthor=S.+A.+Readshawauthor=J.+C.+Hollertonauthor=R.+J.+Uptonauthor=S.+M.+Lynneauthor=S.+C.+Besleyauthor=B.+J.+Boughtflower&title=Use+of+Calculated+Physicochemical+Properties+to+Enhance+Quantitative+Response+When+Using+Charged+Aerosol+Detection&doi=10.1021%2Facs.analchem.6b04060"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Use of Calculated Physicochemical Properties to Enhance Quantitative Response When Using Charged Aerosol Detection</span></div><div class="casAuthors">Robinson, Max W.; Hill, Alan P.; Readshaw, Simon A.; Hollerton, John C.; Upton, Richard J.; Lynn, Sean M.; Besley, Steve C.; Boughtflower, Bob J.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1772-1777</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Universal quant. detection without the need for analyte ref. stds. would offer substantial benefits in many areas of anal. science.  The quant. capability of high performance liq. chromatog. (HPLC) with charged aerosol detection (CAD) was investigated for 50 compds. with a wide range of phys. and chem. properties.  It is widely believed that CAD is a mass detector.  Quantification of the 50 compds. using a generic calibrant and mass calibration achieved an av. error of 11.4% relative to 1H NMR.  Correction factors are proposed that est. the relative surface area of particles in the detector, taking into account the effects of the d. and charge of analytes.  Performing these corrections and quantifying with surface area calibration, rather than mass, shows considerably improved linearity and uniformity of detection, reducing the av. error relative to 1H NMR to 7.1%.  The accuracy of CAD quantification was most significantly improved for highly dense compds., with traditional mass calibration showing an av. error of 34.7% and the newly proposed surface area calibration showing an av. error of 5.8%.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxWCD0M2yQRLVg90H21EOLACvtfcHk0lheMxyITdzwkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFams7%252FK&md5=46c11b7527157b76f1fbc8eaf5cef75a</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Facs.analchem.6b04060&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.analchem.6b04060%26sid%3Dliteratum%253Aachs%26aulast%3DRobinson%26aufirst%3DM.%2BW.%26aulast%3DHill%26aufirst%3DA.%2BP.%26aulast%3DReadshaw%26aufirst%3DS.%2BA.%26aulast%3DHollerton%26aufirst%3DJ.%2BC.%26aulast%3DUpton%26aufirst%3DR.%2BJ.%26aulast%3DLynne%26aufirst%3DS.%2BM.%26aulast%3DBesley%26aufirst%3DS.%2BC.%26aulast%3DBoughtflower%26aufirst%3DB.%2BJ.%26atitle%3DUse%2520of%2520Calculated%2520Physicochemical%2520Properties%2520to%2520Enhance%2520Quantitative%2520Response%2520When%2520Using%2520Charged%2520Aerosol%2520Detection%26jtitle%3DAnal.%2520Chem.%26date%3D2017%26volume%3D89%26spage%3D1772%26epage%3D1777%26doi%3D10.1021%2Facs.analchem.6b04060" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mateus, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wayne, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almqvist, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Axelsson, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seashore-Ludlow, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treyer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lundbäck, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hann, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Artursson, P.</span></span> <span> </span><span class="NLM_article-title">Prediction of Intracellular Exposure Bridges the Gap between Target- and Cell-Based Drug Discovery</span>. <i>Proc. Natl. Acad. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">E6231</span>– <span class="NLM_lpage">E6239</span>, <span class="refDoi"> DOI: 10.1073/pnas.1701848114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1073%2Fpnas.1701848114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=28701380" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFKmsr%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=E6231-E6239&author=A.+Mateusauthor=L.+J.+Gordonauthor=G.+J.+Wayneauthor=H.+Almqvistauthor=H.+Axelssonauthor=B.+Seashore-Ludlowauthor=A.+Treyerauthor=P.+Matssonauthor=T.+Lundb%C3%A4ckauthor=A.+Westauthor=M.+M.+Hannauthor=P.+Artursson&title=Prediction+of+Intracellular+Exposure+Bridges+the+Gap+between+Target-+and+Cell-Based+Drug+Discovery&doi=10.1073%2Fpnas.1701848114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of intracellular exposure bridges the gap between target- and cell-based drug discovery</span></div><div class="casAuthors">Mateus, Andre; Gordon, Laurie J.; Wayne, Gareth J.; Almqvist, Helena; Axelsson, Hanna; Seashore-Ludlow, Brinton; Treyer, Andrea; Matsson, Paer; Lundbaeck, Thomas; West, Andy; Hann, Michael M.; Artursson, Per</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">E6231-E6239</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Inadequate target exposure is a major cause of high attrition in drug discovery.  Here, we show that a label-free method for quantifying the intracellular bioavailability (Fic) of drug mols. predicts drug access to intracellular targets and hence, pharmacol. effect.  We detd. Fic in multiple cellular assays and cell types representing different targets from a no. of therapeutic areas, including cancer, inflammation, and dementia.  Both cytosolic targets and targets localized in subcellular compartments were investigated.  Fic gives insights on membrane-permeable compds. in terms of cellular potency and intracellular target engagement, compared with biochem. potency measurements alone.  Knowledge of the amt. of drug that is locally available to bind intracellular targets provides a powerful tool for compd. selection in early drug discovery. localization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqo1IEI-YIPNbVg90H21EOLACvtfcHk0lgu3GTBRTJVog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFKmsr%252FK&md5=ace62bf5bda6cfd214bac4b7c5bf9cf1</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1701848114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1701848114%26sid%3Dliteratum%253Aachs%26aulast%3DMateus%26aufirst%3DA.%26aulast%3DGordon%26aufirst%3DL.%2BJ.%26aulast%3DWayne%26aufirst%3DG.%2BJ.%26aulast%3DAlmqvist%26aufirst%3DH.%26aulast%3DAxelsson%26aufirst%3DH.%26aulast%3DSeashore-Ludlow%26aufirst%3DB.%26aulast%3DTreyer%26aufirst%3DA.%26aulast%3DMatsson%26aufirst%3DP.%26aulast%3DLundb%25C3%25A4ck%26aufirst%3DT.%26aulast%3DWest%26aufirst%3DA.%26aulast%3DHann%26aufirst%3DM.%2BM.%26aulast%3DArtursson%26aufirst%3DP.%26atitle%3DPrediction%2520of%2520Intracellular%2520Exposure%2520Bridges%2520the%2520Gap%2520between%2520Target-%2520and%2520Cell-Based%2520Drug%2520Discovery%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%26date%3D2017%26volume%3D114%26spage%3DE6231%26epage%3DE6239%26doi%3D10.1073%2Fpnas.1701848114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Artursson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hann, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leavens, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mateus, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Readshaw, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valko, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wayne, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, A.</span></span> <span> </span><span class="NLM_article-title">Direct Measurement of Intracellular Compound Concentration by RapidFire Mass Spectrometry Offers Insights into Cell Permeability</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">156</span>– <span class="NLM_lpage">164</span>, <span class="refDoi"> DOI: 10.1177/1087057115604141</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1177%2F1087057115604141" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=26336900" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=156-164&author=L.+J.+Gordonauthor=M.+Allenauthor=P.+Arturssonauthor=M.+M.+Hannauthor=B.+J.+Leavensauthor=A.+Mateusauthor=S.+Readshawauthor=K.+Valkoauthor=G.+J.+Wayneauthor=A.+West&title=Direct+Measurement+of+Intracellular+Compound+Concentration+by+RapidFire+Mass+Spectrometry+Offers+Insights+into+Cell+Permeability&doi=10.1177%2F1087057115604141"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1177%2F1087057115604141&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057115604141%26sid%3Dliteratum%253Aachs%26aulast%3DGordon%26aufirst%3DL.%2BJ.%26aulast%3DAllen%26aufirst%3DM.%26aulast%3DArtursson%26aufirst%3DP.%26aulast%3DHann%26aufirst%3DM.%2BM.%26aulast%3DLeavens%26aufirst%3DB.%2BJ.%26aulast%3DMateus%26aufirst%3DA.%26aulast%3DReadshaw%26aufirst%3DS.%26aulast%3DValko%26aufirst%3DK.%26aulast%3DWayne%26aufirst%3DG.%2BJ.%26aulast%3DWest%26aufirst%3DA.%26atitle%3DDirect%2520Measurement%2520of%2520Intracellular%2520Compound%2520Concentration%2520by%2520RapidFire%2520Mass%2520Spectrometry%2520Offers%2520Insights%2520into%2520Cell%2520Permeability%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2015%26volume%3D21%26spage%3D156%26epage%3D164%26doi%3D10.1177%2F1087057115604141" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leonards, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dovey, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monteiro, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujisawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingard, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikbin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guetzoyan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ingham, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ley, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samsonova, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingras, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwaller, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vassiliou, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span> <span> </span><span class="NLM_article-title">Promiscuous Targeting of Bromodomains by Bromosporine Identifies BET Proteins as Master Regulators of Primary Transcription Response in Leukemia</span>. <i>Sci. Adv.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">e1600760</span>– <span class="NLM_lpage">e1600760</span>, <span class="refDoi"> DOI: 10.1126/sciadv.1600760</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1126%2Fsciadv.1600760" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=27757418" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC1cXkvFyjtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2016&pages=e1600760-e1600760&author=S.+Picaudauthor=K.+Leonardsauthor=J.+P.+Lambertauthor=O.+Doveyauthor=C.+Wellsauthor=O.+Fedorovauthor=O.+Monteiroauthor=T.+Fujisawaauthor=C.+Y.+Wangauthor=H.+Lingardauthor=C.+Tallantauthor=N.+Nikbinauthor=L.+Guetzoyanauthor=R.+Inghamauthor=S.+V.+Leyauthor=P.+Brennanauthor=S.+Mullerauthor=A.+Samsonovaauthor=A.+C.+Gingrasauthor=J.+Schwallerauthor=G.+Vassiliouauthor=S.+Knappauthor=P.+Filippakopoulos&title=Promiscuous+Targeting+of+Bromodomains+by+Bromosporine+Identifies+BET+Proteins+as+Master+Regulators+of+Primary+Transcription+Response+in+Leukemia&doi=10.1126%2Fsciadv.1600760"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Promiscuous targeting of bromodomains by bromosporine identifies BET proteins as master regulators of primary transcription response in leukemia</span></div><div class="casAuthors">Picaud, Sarah; Leonards, Katharina; Lambert, Jean-Philippe; Dovey, Oliver; Wells, Christopher; Fedorov, Oleg; Monteiro, Octovia; Fujisawa, Takao; Wang, Chen-Yi; Lingard, Hannah; Tallant, Cynthia; Nikbin, Nikzad; Guetzoyan, Lucie; Ingham, Richard; Ley, Steven V.; Brennan, Paul; Muller, Susanne; Samsonova, Anastasia; Gingras, Anne-Claude; Schwaller, Juerg; Vassiliou, George; Knapp, Stefan; Filippakopoulos, Panagis</div><div class="citationInfo"><span class="NLM_cas:title">Science Advances</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">e1600760/1-e1600760/16</span>CODEN:
                <span class="NLM_cas:coden">SACDAF</span>;
        ISSN:<span class="NLM_cas:issn">2375-2548</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) have emerged as compelling targets for cancer therapy.  The development of selective and potent BET (bromo and extra-terminal) inhibitors and their significant activity in diverse tumor models have rapidly translated into clin. studies and have motivated drug development efforts targeting non-BET BRDs.  However, the complexmultidomain/subunit architecture of BRD protein complexes complicates predictions of the consequences of their pharmacol. targeting.  To address this issue, we developed a promiscuous BRD inhibitor [bromosporine (BSP)] that broadly targets BRDs (including BETs) with nanomolar affinity, creating a tool for the identification of cellular processes and diseases where BRDs have a regulatory function.  As a proof of principle, we studied the effects of BSP on leukemic cell lines known to be sensitive to BET inhibition and found, as expected, strong antiproliferative activity.  Comparison of the modulation of transcriptional profiles by BSP after a short exposure to the inhibitor resulted in a BET inhibitor signature but no significant addnl. changes in transcription that could account for inhibition of other BRDs.  Thus, nonselective targeting of BRDs identified BETs, but not other BRDs, as master regulators of contextdependent primary transcription response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr28artLYYsqrVg90H21EOLACvtfcHk0lgu3GTBRTJVog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXkvFyjtbw%253D&md5=3060e8fd66c657f70e0ec3acea273057</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1126%2Fsciadv.1600760&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fsciadv.1600760%26sid%3Dliteratum%253Aachs%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DLeonards%26aufirst%3DK.%26aulast%3DLambert%26aufirst%3DJ.%2BP.%26aulast%3DDovey%26aufirst%3DO.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMonteiro%26aufirst%3DO.%26aulast%3DFujisawa%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DC.%2BY.%26aulast%3DLingard%26aufirst%3DH.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DNikbin%26aufirst%3DN.%26aulast%3DGuetzoyan%26aufirst%3DL.%26aulast%3DIngham%26aufirst%3DR.%26aulast%3DLey%26aufirst%3DS.%2BV.%26aulast%3DBrennan%26aufirst%3DP.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DSamsonova%26aufirst%3DA.%26aulast%3DGingras%26aufirst%3DA.%2BC.%26aulast%3DSchwaller%26aufirst%3DJ.%26aulast%3DVassiliou%26aufirst%3DG.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26atitle%3DPromiscuous%2520Targeting%2520of%2520Bromodomains%2520by%2520Bromosporine%2520Identifies%2520BET%2520Proteins%2520as%2520Master%2520Regulators%2520of%2520Primary%2520Transcription%2520Response%2520in%2520Leukemia%26jtitle%3DSci.%2520Adv.%26date%3D2016%26volume%3D2%26spage%3De1600760%26epage%3De1600760%26doi%3D10.1126%2Fsciadv.1600760" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tyler, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vappiani, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cañeque, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, E. Y. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilan, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, Y.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hienzsch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rutkowska, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lugo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gregory, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramirez Molina, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garton, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wellaway, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacPherson, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Figueiredo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stolzenburg, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">House, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, S.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hawkins, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewes, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriquez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, M. A.</span></span> <span> </span><span class="NLM_article-title">Click Chemistry Enables Preclinical Evaluation of Targeted Epigenetic Therapies</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>356</i></span>,  <span class="NLM_fpage">1397</span>– <span class="NLM_lpage">1401</span>, <span class="refDoi"> DOI: 10.1126/science.aal2066</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1126%2Fscience.aal2066" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=28619718" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVKgsb%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=356&publication_year=2017&pages=1397-1401&author=D.+S.+Tylerauthor=J.+Vappianiauthor=T.+Ca%C3%B1equeauthor=E.+Y.+N.+Lamauthor=A.+Wardauthor=O.+Gilanauthor=Y.-C.+Chanauthor=A.+Hienzschauthor=A.+Rutkowskaauthor=T.+Wernerauthor=A.+J.+Wagnerauthor=D.+Lugoauthor=R.+Gregoryauthor=C.+Ramirez+Molinaauthor=N.+Gartonauthor=C.+R.+Wellawayauthor=S.+Jacksonauthor=L.+MacPhersonauthor=M.+Figueiredoauthor=S.+Stolzenburgauthor=C.+C.+Bellauthor=C.+Houseauthor=S.-J.+Dawsonauthor=E.+D.+Hawkinsauthor=G.+Drewesauthor=R.+K.+Prinjhaauthor=R.+Rodriquezauthor=P.+Grandiauthor=M.+A.+Dawson&title=Click+Chemistry+Enables+Preclinical+Evaluation+of+Targeted+Epigenetic+Therapies&doi=10.1126%2Fscience.aal2066"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Click chemistry enables preclinical evaluation of targeted epigenetic therapies</span></div><div class="casAuthors">Tyler, Dean S.; Vappiani, Johanna; Caneque, Tatiana; Lam, Enid Y. N.; Ward, Aoife; Gilan, Omer; Chan, Yih-Chih; Hienzsch, Antje; Rutkowska, Anna; Werner, Thilo; Wagner, Anne J.; Lugo, Dave; Gregory, Richard; Ramirez Molina, Cesar; Garton, Neil; Wellaway, Christopher R.; Jackson, Susan; MacPherson, Laura; Figueiredo, Margarida; Stolzenburg, Sabine; Bell, Charles C.; House, Colin; Dawson, Sarah-Jane; Hawkins, Edwin D.; Drewes, Gerard; Prinjha, Rab K.; Rodriguez, Raphael; Grandi, Paola; Dawson, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">356</span>
        (<span class="NLM_cas:issue">6345</span>),
    <span class="NLM_cas:pages">1397-1401</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The success of new therapies hinges on our ability to understand their mol. and cellular mechanisms of action.  We modified BET bromodomain inhibitors, an epigenetic-based therapy, to create functionally conserved compds. that are amenable to click chem. and can be used as mol. probes in vitro and in vivo.  We used click proteomics and click sequencing to explore the gene regulatory function of BRD4 (bromodomain contg. protein 4) and the transcriptional changes induced by BET inhibitors.  In our studies of mouse models of acute leukemia, we used high-resoln. microscopy and flow cytometry to highlight the heterogeneity of drug activity within tumor cells located in different tissue compartments.  We also demonstrate the differential distribution and effects of BET inhibitors in normal and malignant cells in vivo.  This study provides a potential framework for the preclin. assessment of a wide range of drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqf_08jpo_TP7Vg90H21EOLACvtfcHk0lhIhaJQcBEsYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVKgsb%252FN&md5=a6b837fb92c5fa3818b8eecd8274657d</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1126%2Fscience.aal2066&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aal2066%26sid%3Dliteratum%253Aachs%26aulast%3DTyler%26aufirst%3DD.%2BS.%26aulast%3DVappiani%26aufirst%3DJ.%26aulast%3DCa%25C3%25B1eque%26aufirst%3DT.%26aulast%3DLam%26aufirst%3DE.%2BY.%2BN.%26aulast%3DWard%26aufirst%3DA.%26aulast%3DGilan%26aufirst%3DO.%26aulast%3DChan%26aufirst%3DY.-C.%26aulast%3DHienzsch%26aufirst%3DA.%26aulast%3DRutkowska%26aufirst%3DA.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DWagner%26aufirst%3DA.%2BJ.%26aulast%3DLugo%26aufirst%3DD.%26aulast%3DGregory%26aufirst%3DR.%26aulast%3DRamirez%2BMolina%26aufirst%3DC.%26aulast%3DGarton%26aufirst%3DN.%26aulast%3DWellaway%26aufirst%3DC.%2BR.%26aulast%3DJackson%26aufirst%3DS.%26aulast%3DMacPherson%26aufirst%3DL.%26aulast%3DFigueiredo%26aufirst%3DM.%26aulast%3DStolzenburg%26aufirst%3DS.%26aulast%3DBell%26aufirst%3DC.%2BC.%26aulast%3DHouse%26aufirst%3DC.%26aulast%3DDawson%26aufirst%3DS.-J.%26aulast%3DHawkins%26aufirst%3DE.%2BD.%26aulast%3DDrewes%26aufirst%3DG.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRodriquez%26aufirst%3DR.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DDawson%26aufirst%3DM.%2BA.%26atitle%3DClick%2520Chemistry%2520Enables%2520Preclinical%2520Evaluation%2520of%2520Targeted%2520Epigenetic%2520Therapies%26jtitle%3DScience%26date%3D2017%26volume%3D356%26spage%3D1397%26epage%3D1401%26doi%3D10.1126%2Fscience.aal2066" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gosmini, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, V. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toum, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brusq, J.-M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krysa, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirguet, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riou-Eymard, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boursier, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trottet, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutler, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaur, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schilling, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soden, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicodème, E.</span></span> <span> </span><span class="NLM_article-title">The Discovery of I-BET726 (GSK1324726A), a Potent Tetrahydroquinoline ApoA1 Up-Regulator and Selective BET Bromodomain Inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">8111</span>– <span class="NLM_lpage">8131</span>, <span class="refDoi"> DOI: 10.1021/jm5010539</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5010539" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1Wlur7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=8111-8131&author=R.+Gosminiauthor=V.+L.+Nguyenauthor=J.+Toumauthor=C.+Simonauthor=J.-M.+G.+Brusqauthor=G.+Krysaauthor=O.+Mirguetauthor=A.+M.+Riou-Eymardauthor=E.+V.+Boursierauthor=L.+Trottetauthor=P.+Bamboroughauthor=H.+Clarkauthor=C.+Chungauthor=L.+Cutlerauthor=E.+H.+Demontauthor=R.+Kaurauthor=A.+J.+Lewisauthor=M.+B.+Schillingauthor=P.+E.+Sodenauthor=S.+Taylorauthor=A.+L.+Walkerauthor=M.+D.+Walkerauthor=R.+K.+Prinjhaauthor=E.+Nicod%C3%A8me&title=The+Discovery+of+I-BET726+%28GSK1324726A%29%2C+a+Potent+Tetrahydroquinoline+ApoA1+Up-Regulator+and+Selective+BET+Bromodomain+Inhibitor&doi=10.1021%2Fjm5010539"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">The Discovery of I-BET726 (GSK1324726A), a Potent Tetrahydroquinoline ApoA1 Up-Regulator and Selective BET Bromodomain Inhibitor</span></div><div class="casAuthors">Gosmini, Romain; Nguyen, Van Loc; Toum, Jerome; Simon, Christophe; Brusq, Jean-Marie G.; Krysa, Gael; Mirguet, Olivier; Riou-Eymard, Alizon M.; Boursier, Eric V.; Trottet, Lionel; Bamborough, Paul; Clark, Hugh; Chung, Chun-wa; Cutler, Leanne; Demont, Emmanuel H.; Kaur, Rejbinder; Lewis, Antonia J.; Schilling, Mark B.; Soden, Peter E.; Taylor, Simon; Walker, Ann L.; Walker, Matthew D.; Prinjha, Rab K.; Nicodeme, Edwige</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8111-8131</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Through their function as epigenetic readers of the histone code, the BET family of bromodomain-contg. proteins regulate expression of multiple genes of therapeutic relevance, including those involved in tumor cell growth and inflammation.  BET bromodomain inhibitors have profound antiproliferative and anti-inflammatory effects which translate into efficacy in oncol. and inflammation models, and the first compds. have now progressed into clin. trials.  The exciting biol. of the BETs has led to great interest in the discovery of novel inhibitor classes.  Here we describe the identification of a novel tetrahydroquinoline series through up-regulation of apolipoprotein A1 and the optimization into potent compds. active in murine models of septic shock and neuroblastoma.  At the mol. level, these effects are produced by inhibition of BET bromodomains.  X-ray crystallog. reveals the interactions explaining the structure-activity relationships of binding.  The resulting lead mol., I-BET726, represents a new, potent, and selective class of tetrahydroquinoline-based BET inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5OdMn_TjoFLVg90H21EOLACvtfcHk0lgZT6o0GZ_hjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1Wlur7J&md5=b1c9c9cae73c41252cf73908d561bd9b</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Fjm5010539&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5010539%26sid%3Dliteratum%253Aachs%26aulast%3DGosmini%26aufirst%3DR.%26aulast%3DNguyen%26aufirst%3DV.%2BL.%26aulast%3DToum%26aufirst%3DJ.%26aulast%3DSimon%26aufirst%3DC.%26aulast%3DBrusq%26aufirst%3DJ.-M.%2BG.%26aulast%3DKrysa%26aufirst%3DG.%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DRiou-Eymard%26aufirst%3DA.%2BM.%26aulast%3DBoursier%26aufirst%3DE.%2BV.%26aulast%3DTrottet%26aufirst%3DL.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DClark%26aufirst%3DH.%26aulast%3DChung%26aufirst%3DC.%26aulast%3DCutler%26aufirst%3DL.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DKaur%26aufirst%3DR.%26aulast%3DLewis%26aufirst%3DA.%2BJ.%26aulast%3DSchilling%26aufirst%3DM.%2BB.%26aulast%3DSoden%26aufirst%3DP.%2BE.%26aulast%3DTaylor%26aufirst%3DS.%26aulast%3DWalker%26aufirst%3DA.%2BL.%26aulast%3DWalker%26aufirst%3DM.%2BD.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DNicod%25C3%25A8me%26aufirst%3DE.%26atitle%3DThe%2520Discovery%2520of%2520I-BET726%2520%2528GSK1324726A%2529%252C%2520a%2520Potent%2520Tetrahydroquinoline%2520ApoA1%2520Up-Regulator%2520and%2520Selective%2520BET%2520Bromodomain%2520Inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D8111%26epage%3D8131%26doi%3D10.1021%2Fjm5010539" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47a','cit47b'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 3 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Michael A. Clegg, Natalie H. Theodoulou, Paul Bamborough, Chun-wa Chung, Peter D. Craggs, Emmanuel H. Demont, Laurie J. Gordon, Gemma M. Liwicki, Alex Phillipou, Nicholas C. O. Tomkinson, Rab K. Prinjha, <span class="NLM_string-name hlFld-ContribAuthor">Philip G. Humphreys</span>. </span><span class="cited-content_cbyCitation_article-title">Optimization of Naphthyridones into Selective TATA-Binding Protein Associated Factor 1 (TAF1) Bromodomain Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2021</strong>, Article ASAP.</span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.1c00294&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.1c00294%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DOptimization%252Bof%252BNaphthyridones%252Binto%252BSelective%252BTATA-Binding%252BProtein%252BAssociated%252BFactor%252B1%252B%252528TAF1%252529%252BBromodomain%252BInhibitors%26aulast%3DClegg%26aufirst%3DMichael%2BA.%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D21052021%26date%3D29072021%26volume%3D0%26issue%3D0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Francesca  D'Amico</span>, <span class="hlFld-ContribAuthor ">Rishov  Mukhopadhyay</span>, <span class="hlFld-ContribAuthor ">Huib  Ovaa</span>, <span class="hlFld-ContribAuthor ">Monique P. C.  Mulder</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting TRIM Proteins: A Quest towards Drugging an Emerging Protein Class. </span><span class="cited-content_cbyCitation_journal-name">ChemBioChem</span><span> <strong>2021,</strong> <em>22 </em>
                                    (12)
                                     , 2011-2031. <a href="https://doi.org/10.1002/cbic.202000787" title="DOI URL">https://doi.org/10.1002/cbic.202000787</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cbic.202000787&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcbic.202000787%26sid%3Dliteratum%253Aachs%26jtitle%3DChemBioChem%26atitle%3DTargeting%252BTRIM%252BProteins%25253A%252BA%252BQuest%252Btowards%252BDrugging%252Ban%252BEmerging%252BProtein%252BClass%26aulast%3DD%2527Amico%26aufirst%3DFrancesca%26date%3D2021%26date%3D2021%26volume%3D22%26issue%3D12%26spage%3D2011%26epage%3D2031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiuzuo  Zhu</span>, <span class="hlFld-ContribAuthor ">Yi  Liao</span>, <span class="hlFld-ContribAuthor ">Liling  Tang</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting BRD9 for Cancer Treatment: A New Strategy. </span><span class="cited-content_cbyCitation_journal-name">OncoTargets and Therapy</span><span> <strong>2020,</strong> <em>Volume 13 </em>, 13191-13200. <a href="https://doi.org/10.2147/OTT.S286867" title="DOI URL">https://doi.org/10.2147/OTT.S286867</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2147/OTT.S286867&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2147%2FOTT.S286867%26sid%3Dliteratum%253Aachs%26jtitle%3DOncoTargets%2520and%2520Therapy%26atitle%3DTargeting%252BBRD9%252Bfor%252BCancer%252BTreatment%25253A%252BA%252BNew%252BStrategy%26aulast%3DZhu%26aufirst%3DXiuzuo%26date%3D2020%26date%3D2020%26volume%3DVolume%252013%26spage%3D13191%26epage%3D13200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/medium/jm0c00075_0030.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0030.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00075&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/medium/jm0c00075_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. X-ray crystal structure of the N-terminal bromodomain of BRD4, BRD4(1), bound to KAc (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3uvy">3uvy</a>),<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> highlighting the four antiparallel alpha helices, loop regions, four conserved water molecules, and key interactions formed between typical bromodomains and acetylated lysines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00075&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/medium/jm0c00075_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0004.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Existing BRD7/9 inhibitors and their reported BRD7/9 potencies. The KAc methyl mimetic is highlighted in blue.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00075&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/medium/jm0c00075_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0005.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (a) Crystal structures of pyridazinone fragment <b>8</b> (yellow) bound to BRD9 (gray; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YQW">6YQW</a>) and PCAF (blue; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5MKX">5MKX</a>) highlighting the different “WPF” shelf motif residues. BRD9 amino acid residues are labeled in black, and PCAF amino acid residues are labeled in blue. (b) Sequence alignment of BRD9, BRD7, PCAF, and BRD4(1). The WPF moiety is colored orange, the conserved acetyl-lysine-binding asparagine and tyrosine are colored red and green, respectively, and the gatekeeper residue is colored blue.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00075&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/medium/jm0c00075_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0006.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (a) X-ray crystallography of compound <b>14</b> (yellow) bound to (a) BRD9 (gray; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YQR">6YQR</a>), (b) BRD9 (gray; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YQR">6YQR</a>) highlighting the protein surface, (c) BRD4(1) (cyan; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YQZ">6YQZ</a>), and (d) BRD4(1) (cyan; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YQZ">6YQZ</a>) highlighting the protein surface.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00075&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/medium/jm0c00075_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0007.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. X-ray crystallography of (a) compound <b>18</b> (green) bound to BRD9 (gray; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YQS">6YQS</a>), (b) overlay of compound <b>18</b> (green; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YQS">6YQS</a>) and compound <b>14</b> (yellow; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YQR">6YQR</a>) bound to BRD9 (gray), and (c) compound <b>18</b> (green) bound to BRD9 (gray; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YQS">6YQS</a>) highlighting the protein surface.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00075&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/medium/jm0c00075_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0008.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (a) Structure of BRD9 inhibitors <b>18</b> and <b>20</b>. The methyl mimetic is colored blue. (b, c) Overlay of the crystal structures of <b>18</b> (green; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YQS">6YQS</a>) and <b>20</b> (cyan; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5I7Y">5I7Y</a>) bound to BRD9 (gray; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5I7Y">5I7Y</a>), highlighting the vector to access the induced pocket from different perspectives.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00075&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/medium/jm0c00075_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0009.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Docking of pyridazinone analogues bearing different KAc methyl mimetics (a) allyl <b>21</b>, (b) <i>E</i>-crotyl <b>22</b>, (c) <i>Z</i>-crotyl <b>23</b>, (d) 1,2-butenyl <b>24</b>, (e) 3,4-butenyl <b>25</b>, (f) <i>sec</i>-butenyl <b>27</b>, (g) ethyl <b>28</b>, (h) propyl <b>29</b>, (i) isopropyl <b>30</b>, (j) pentyl <b>32</b>, (k) <i>n</i>-butyl <b>31</b>, and (l) <i>n</i>-butyl <b>31</b> space filled into the BRD9 bromodomain (gray) using Glide (Schrodinger Inc.) 2018.3 with hydrogen-bonding constraints between docked ligands and both the W1 conserved water molecule and the carbonyl group of Phe44. Tyr106 has been removed from the visualization to help show the induced pocket. PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5I7Y">5I7Y</a> was used for all dockings.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00075&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/medium/jm0c00075_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0010.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. DiscoverX BROMOscan activity profile for compound <b>31</b> screened against 40 bromodomains.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00075&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/medium/jm0c00075_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0011.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Three templates selected to test the broader applicability of the <i>n</i>-butyl KAc methyl mimetic methodology and their corresponding <i>n</i>-butyl derivatives. Full data shown in <a href="/doi/suppl/10.1021/acs.jmedchem.0c00075/suppl_file/jm0c00075_si_001.pdf" class="ext-link">Table S8</a>. Data for compound <b>2</b> can be found in ref <a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00075&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/medium/jm0c00075_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0002.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. DiscoverX BROMOscan profiling at 10 μM for (a) compound <b>34</b>, (b) compound <b>35</b>, (c) iBRD9 (<b>2</b>), (d) compound <b>36</b>, (e) bromosporine (<b>37</b>), and (f) compound <b>38</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00075&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig11" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/medium/jm0c00075_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0003.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Affinity enrichment chemoproteomic experiments in Hut-78 nuclear and chromatin lysate.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00075&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/medium/jm0c00075_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0012.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 1. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00075&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Amine, DMSO, 120 °C, 1 h, 23–75%</p></p></figure><figure data-id="sch2" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/medium/jm0c00075_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0013.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 2. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00075&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 3.4 M MeMgBr, THF, rt, 5.5 h, 81%; (b) MeI, TBAB, K<sub>2</sub>CO<sub>3</sub>, MeCN, rt to 60 °C, 4 h, 55%; (c) Pd-Catalyzed Buchwald–Hartwig Amination, NaO<i><sup>t</sup></i>Bu or KO<i>t</i>-Bu, 1,4-Dioxane, 100–120 °C, 30 min to 1 h, 13–61%</p></p></figure><figure data-id="sch3" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/medium/jm0c00075_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0014.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 3. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00075&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <i>tert</i>-Butyl 4-(2-(aminomethyl)phenyl)piperazine-1-carboxylate, BrettPhos-Pd-G1, BrettPhos, NaO<i><sup>t</sup></i>Bu, 1,4-Dioxane, Microwave 120 °C, 30 min; (b) 4 M HCl, 1,4-Dioxane, DCM, rt, 18 h, 10% over Two Steps; (c) BocNHCH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>Br, Cs<sub>2</sub>CO<sub>3</sub>, DMF, rt to 50 °C, 36 h, 20%; (d) HCl, 1,4-Dioxane, rt, 18 h, 18%</p></p></figure><figure data-id="sch4" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/medium/jm0c00075_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0015.jpeg" id="rightTab-gr15" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 4. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00075&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (2-(4-Methylpiperazin-1-yl)phenyl)methanamine, DIPEA, DMSO, 120 °C, 1 h, 63%; (b) Allyl-Bpin, RuPhos Pd G2, K<sub>2</sub>CO<sub>3</sub>, 1,4-Dioxane/H<sub>2</sub>O, 100 °C, 1 h, 28%; (c) (<i>E</i>)-But-1-en-1-ylboronic Acid, Pd(OAc)<sub>2</sub>, CataCXium A, K<sub>2</sub>CO<sub>3</sub>, 1,4-Dioxane/H<sub>2</sub>O, 100 °C, 5 h, 47%; (d) (<i>E</i>)-2-(But-2-en-1-yl)-Bpin, Pd(OAc)<sub>2</sub>, CataCXium A, K<sub>2</sub>CO<sub>3</sub>, 1,4-Dioxane/H<sub>2</sub>O, 100 °C, 1.5 h, 30% Combined Yield; (e) Propylboronic Acid, Pd(OAc)<sub>2</sub>, CataCXium A, K<sub>2</sub>CO<sub>3</sub>, 1,4-Dioxane/H<sub>2</sub>O, 100 °C, 1 h, 29%; (f) Pentylboronic Acid, Pd(OAc)<sub>2</sub>, CataCXium A, K<sub>2</sub>CO<sub>3</sub>, 1,4-Dioxane/H<sub>2</sub>O, 100 °C, 1 h, 25%; (g) Potassium Butyltrifluoroborate, Pd(OAc)<sub>2</sub>,CataCXium A, K<sub>2</sub>CO<sub>3</sub>, 1,4-Dioxane/H<sub>2</sub>O, 100 °C, 1 h, 43%</p></p></figure><figure data-id="sch5" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/medium/jm0c00075_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0016.jpeg" id="rightTab-gr16" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 5. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00075&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Allylbromide, LiHMDS, THF, −78 °C, 45 min, 24%; (b) (2-(4-Methylpiperazin-1-yl)phenyl)methanamine, Pd<sub>2</sub>(dba)<sub>3</sub>, BrettPhos, NaO<i><sup>t</sup></i>Bu, THF, 100 °C, 1 h, 20%</p></p></figure><figure data-id="sch6" data-index="17" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/medium/jm0c00075_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0017.jpeg" id="rightTab-gr17" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 6. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00075&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) THF, 0 °C to rt, 1.5–3 h, 26–27%; (b) MeI, TBAB, K<sub>2</sub>CO<sub>3</sub>, MeCN, rt, 2.5–3 h, 19–63%; (c) (2-(4-Methylpiperazin-1-yl)phenyl)methanamine, Pd<sub>2</sub>(dba)<sub>3</sub>, BrettPhos, NaO<i><sup>t</sup></i>Bu, THF, 100 °C, 1 h, 36–46%</p></p></figure><figure data-id="sch7" data-index="18" class="article__inlineFigure"><h2 class="fig-label">Scheme 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/medium/jm0c00075_0018.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0018.jpeg" id="rightTab-gr18" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 7. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00075&amp;id=sch7"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 1.6 M <i>n</i>-BuLi, THF, −78 °C, 15 min, 63%; (b) MeI, TBAB, K<sub>2</sub>CO<sub>3</sub>, MeCN, 60 °C, 5 h, 77%; (c) (2-(4-Methylpiperazin-1-yl)phenyl)methanamine, Pd<sub>2</sub>(dba)<sub>3</sub>, BrettPhos, NaO<i><sup>t</sup></i>Bu, THF, 100 °C, 1 h, 22%; (d) <i>tert</i>-Butyl 5-Amino-3,4-dihydroisoquinoline-2(1<i>H</i>)-carboxylate, Pd<sub>2</sub>(dba)<sub>3</sub>, BrettPhos, NaO<i><sup>t</sup></i>Bu, THF, Microwave 100 °C, 1 h, 25%; (e) 4 M HCl, 1,4-Dioxane, rt, 45 min, Quant.; (f) 37 wt % Formaldehyde, Formic Acid, 80 °C, 6 h, 59%</p></p></figure><figure data-id="sch8" data-index="19" class="article__inlineFigure"><h2 class="fig-label">Scheme 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/medium/jm0c00075_0019.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0019.jpeg" id="rightTab-gr19" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 8. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00075&amp;id=sch8"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 1-Iodobutane, Cs<sub>2</sub>CO<sub>3</sub>, THF, 60 °C, 19 h, 71%; (b) Zn(CN)<sub>2</sub>, Pd(PPh<sub>3</sub>)<sub>4</sub>, DMF, 115 °C, 4.5 h, 61%; (c) NBS, THF, rt, 65 h, 86%; (d) 4-Aminotetrahydro-2<i>H</i>-thiopyran 1,1-Dioxide, 25 wt % NaOMe, MeOH, 75 °C, 26 h, 27%; (e) 4-Aminotetrahydro-2<i>H</i>-thiopyran 1,1-Dioxide, Triethylamine, DMF, 120 °C, 17 h, 21%; (f) (3-(Trifluoromethyl)phenyl)boronic Acid, PEPPSI-<sup>i</sup>Pr, K<sub>2</sub>CO<sub>3</sub>, IPA/H<sub>2</sub>O, 120 °C, 30 min, 52%</p></p></figure><figure data-id="sch9" data-index="20" class="article__inlineFigure"><h2 class="fig-label">Scheme 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/medium/jm0c00075_0020.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0020.jpeg" id="rightTab-gr20" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 9. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00075/20200604/images/large/jm0c00075_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00075&amp;id=sch9"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (i) Hydrazine Hydrate, 135 °C, 30 min; (ii) Pentanoic Acid, 100 °C, 3 h, 43% over Two Steps; (b) Ethylchloroformate, NMM, DCM, rt, 5 min, 67%; (c) (4-Methyl-3-nitrophenyl)boronic Acid, PdCl<sub>2</sub>(dppf), NaCO<sub>3</sub>, PhMe/EtOH, 100 °C, 5 h, 38%; (d) Fe, AcOH, EtOH/H<sub>2</sub>O, 80 °C, 2 h, 50%; (e) MsCl, Pyridine, DCM, rt, 3 h, 74%</p></p></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i74">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_04856" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_04856" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 54 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.-M.</span></span> <span> </span><span class="NLM_article-title">The Bromodomain: A New Target in Emerging Epigenetic Medicine</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">598</span>– <span class="NLM_lpage">608</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00831</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00831" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVyit7fJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=598-608&author=S.+G.+Smithauthor=M.-M.+Zhou&title=The+Bromodomain%3A+A+New+Target+in+Emerging+Epigenetic+Medicine&doi=10.1021%2Facschembio.5b00831"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The Bromodomain: A New Target in Emerging Epigenetic Medicine</span></div><div class="casAuthors">Smith, Steven G.; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">598-608</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromodomain (BrD) is a conserved protein modular domain found in many chromatin- and transcription-assocd. proteins that has the ability to recognize acetylated lysine residues.  This activity allows bromodomains to play a vital role in many acetylation-mediated protein-protein interactions in the cell, ranging from substrate recruitment for histone acetyltransferases (HATs) to aiding in multiple-protein complex assembly for gene transcriptional activation or suppression in chromatin.  In recent years, considerable efforts have been made to develop chem. inhibitors of these bromodomains in an effort to probe their cellular functions.  Potent and selective inhibitors have been extensively developed for one group of the bromodomain family termed BET proteins that consist of tandem bromodomains followed by an extra terminal domain.  Drug developers are actively designing inhibitors of other bromodomains and working to move the most successful inhibitors into the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTQF2Qjn_iuLVg90H21EOLACvtfcHk0lj9fLrtl2ZJaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVyit7fJ&md5=dd34fc591550ff1528e6dccd1d04f1c1</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00831&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00831%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DS.%2BG.%26aulast%3DZhou%26aufirst%3DM.-M.%26atitle%3DThe%2520Bromodomain%253A%2520A%2520New%2520Target%2520in%2520Emerging%2520Epigenetic%2520Medicine%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D598%26epage%3D608%26doi%3D10.1021%2Facschembio.5b00831" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Targeting Bromodomains: Epigenetic Readers of Lysine Acetylation</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">337</span>– <span class="NLM_lpage">356</span>, <span class="refDoi"> DOI: 10.1038/nrd4286</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1038%2Fnrd4286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=24751816" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=337-356&author=P.+Filippakopoulosauthor=S.+Knapp&title=Targeting+Bromodomains%3A+Epigenetic+Readers+of+Lysine+Acetylation&doi=10.1038%2Fnrd4286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting bromodomains: epigenetic readers of lysine acetylation</span></div><div class="casAuthors">Filippakopoulos, Panagis; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">337-356</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Lysine acetylation is a key mechanism that regulates chromatin structure; aberrant acetylation levels have been linked to the development of several diseases.  Acetyl-lysine modifications create docking sites for bromodomains, which are small interaction modules found on diverse proteins, some of which have a key role in the acetylation-dependent assembly of transcriptional regulator complexes.  These complexes can then initiate transcriptional programs that result in phenotypic changes.  The recent discovery of potent and highly specific inhibitors for the BET (bromodomain and extra-terminal) family of bromodomains has stimulated intensive research activity in diverse therapeutic areas, particularly in oncol., where BET proteins regulate the expression of key oncogenes and anti-apoptotic proteins.  In addn., targeting BET bromodomains could hold potential for the treatment of inflammation and viral infection.  Here, we highlight recent progress in the development of bromodomain inhibitors, and their potential applications in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOSIcwBJKj4LVg90H21EOLACvtfcHk0lj9fLrtl2ZJaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D&md5=77049c82ce6a19b4685e3b7950399a07</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrd4286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4286%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DTargeting%2520Bromodomains%253A%2520Epigenetic%2520Readers%2520of%2520Lysine%2520Acetylation%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D337%26epage%3D356%26doi%3D10.1038%2Fnrd4286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangos, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volkmer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pawson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingras, A.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Histone Recognition and Large-scale Structural Analysis of the Human Bromodomain Family</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>149</i></span>,  <span class="NLM_fpage">214</span>– <span class="NLM_lpage">231</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2012.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1016%2Fj.cell.2012.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=22464331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=149&publication_year=2012&pages=214-231&author=P.+Filippakopoulosauthor=S.+Picaudauthor=M.+Mangosauthor=T.+Keatesauthor=J.+P.+Lambertauthor=D.+Barsyte-Lovejoyauthor=I.+Felletarauthor=R.+Volkmerauthor=S.+M%C3%BCllerauthor=T.+Pawsonauthor=A.-C.+Gingrasauthor=C.+H.+Arrowsmithauthor=S.+Knapp&title=Histone+Recognition+and+Large-scale+Structural+Analysis+of+the+Human+Bromodomain+Family&doi=10.1016%2Fj.cell.2012.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Recognition and Large-Scale Structural Analysis of the Human Bromodomain Family</span></div><div class="casAuthors">Filippakopoulos, Panagis; Picaud, Sarah; Mangos, Maria; Keates, Tracy; Lambert, Jean-Philippe; Barsyte-Lovejoy, Dalia; Felletar, Ildiko; Volkmer, Rudolf; Muller, Susanne; Pawson, Tony; Gingras, Anne-Claude; Arrowsmith, Cheryl H.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">149</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">214-231</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) are protein interaction modules that specifically recognize ε-N-lysine acetylation motifs, a key event in the reading process of epigenetic marks.  The 61 BRDs in the human genome cluster into eight families based on structure/sequence similarity.  Here, we present 29 high-resoln. crystal structures, covering all BRD families.  These proteins are: ASH1L, ATAD2, BAZ2B, BPTF, BRD1, BRD3(1), BRD3(2), BRD4(1), BRD4(2), BRD9, BRDT(1), CECR2, EP300, CREBBP, GCN5L2, KIAA1240, PB1(1), PB1(2), PB1(3), PB1(4), PB1(5), PB1(6), PCAF, PHIP(2), TAF1(2), WDR9(2), BRD4(1).  Comprehensive crossfamily structural anal. identifies conserved and family-specific structural features that are necessary for specific acetylation-dependent substrate recognition.  Screening of more than 30 representative BRDs against systematic histone-peptide arrays identifies new BRD substrates and reveals a strong influence of flanking posttranslational modifications, such as acetylation and phosphorylation, suggesting that BRDs recognize combinations of marks rather than singly acetylated sequences.  We further uncovered a structural mechanism for the simultaneous binding and recognition of diverse diacetyl-contg. peptides by BRD4.  These data provide a foundation for structure-based drug design of specific inhibitors for this emerging target family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK2FSIcPoXZLVg90H21EOLACvtfcHk0liy8mfixaKOWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D&md5=05583a01a1cd4c3b59fc59c00e4c4777</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DMangos%26aufirst%3DM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DLambert%26aufirst%3DJ.%2BP.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DVolkmer%26aufirst%3DR.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DPawson%26aufirst%3DT.%26aulast%3DGingras%26aufirst%3DA.-C.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DHistone%2520Recognition%2520and%2520Large-scale%2520Structural%2520Analysis%2520of%2520the%2520Human%2520Bromodomain%2520Family%26jtitle%3DCell%26date%3D2012%26volume%3D149%26spage%3D214%26epage%3D231%26doi%3D10.1016%2Fj.cell.2012.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Bromodomains as Therapeutic Targets</span>. <i>Expert Rev. Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>13</i></span>, <span class="NLM_elocation-id">e29</span> <span class="refDoi"> DOI: 10.1017/S1462399411001992</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1017%2FS1462399411001992" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=21933453" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1OhurjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2011&author=S.+Mullerauthor=P.+Filippakopoulosauthor=S.+Knapp&title=Bromodomains+as+Therapeutic+Targets&doi=10.1017%2FS1462399411001992"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomains as therapeutic targets</span></div><div class="casAuthors">Muller, Susanne; Filippakopoulos, Panagis; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Expert Reviews in Molecular Medicine</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e29/1-e29/21</span>CODEN:
                <span class="NLM_cas:coden">ERMMFS</span>;
        ISSN:<span class="NLM_cas:issn">1462-3994</span>.
    
            (<span class="NLM_cas:orgname">Cambridge University Press</span>)
        </div><div class="casAbstract">A review.  Acetylation of lysine residues is a post-translational modification with broad relevance to cellular signalling and disease biol.  Enzymes that 'write' (histone acetyltransferases, HATs) and 'erase' (histone deacetylases, HDACs) acetylation sites are an area of extensive research in current drug development, but very few potent inhibitors that modulate the 'reading process' mediated by acetyl lysines have been described.  The principal readers of ε-N-acetyl lysine (Kac) marks are bromodomains (BRDs), which are a diverse family of evolutionary conserved protein-interaction modules.  The conserved BRD fold contains a deep, largely hydrophobic acetyl lysine binding site, which represents an attractive pocket for the development of small, pharmaceutically active mols.  Proteins that contain BRDs have been implicated in the development of a large variety of diseases.  Recently, two highly potent and selective inhibitors that target BRDs of the BET (bromodomains and extra-terminal) family provided compelling data supporting targeting of these BRDs in inflammation and in an aggressive type of squamous cell carcinoma.  It is likely that BRDs will emerge alongside HATs and HDACs as interesting targets for drug development for the large no. of diseases that are caused by aberrant acetylation of lysine residues.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSuKacpsr6xLVg90H21EOLACvtfcHk0liy8mfixaKOWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1OhurjI&md5=dc9383afcbe6aedc265041c21213aa0b</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1017%2FS1462399411001992&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1017%252FS1462399411001992%26sid%3Dliteratum%253Aachs%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DBromodomains%2520as%2520Therapeutic%2520Targets%26jtitle%3DExpert%2520Rev.%2520Mol.%2520Med.%26date%3D2011%26volume%3D13%26doi%3D10.1017%2FS1462399411001992" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jain, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, M. C.</span></span> <span> </span><span class="NLM_article-title">Bromodomain Histone Readers and Cancer</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>429</i></span>,  <span class="NLM_fpage">2003</span>– <span class="NLM_lpage">2010</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2016.11.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1016%2Fj.jmb.2016.11.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=27890782" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFOisbbI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=429&publication_year=2017&pages=2003-2010&author=A.+K.+Jainauthor=M.+C.+Barton&title=Bromodomain+Histone+Readers+and+Cancer&doi=10.1016%2Fj.jmb.2016.11.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomain histone readers and cancer</span></div><div class="casAuthors">Jain, Abhinav K.; Barton, Michelle C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">429</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">2003-2010</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Lysine acetylation of histone proteins is a fundamental post-translational modification that regulates chromatin structure and plays an important role in gene transcription.  Aberrant levels of histone lysine acetylation are assocd. with the development of several diseases.  Acetyl-lysine modifications create docking sites for bromodomains, which are structurally conserved modules present in transcription-assocd. proteins that are termed "reader" proteins.  Bromodomain-contg. reader proteins are part of multiprotein complexes that regulate transcription programs, which are often assocd. with profound phenotypic changes.  Many bromodomain-contg. proteins are aberrantly expressed in diseases, as best studied in cancers, where bromodomain proteins impact the expression of oncogenes and anti-apoptotic proteins.  Thus, bromodomain readers of histone acetylation have emerged as attractive targets for cancer drug discovery, prompting immense interest in epigenetic-focused, medicinal chem. to develop structurally guided chem. probes of bromodomains.  Here, we describe bromodomain-contg. proteins with defined roles in cancer and highlight recent progress in the development of bromodomain inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPuEwpy7dAA7Vg90H21EOLACvtfcHk0liy8mfixaKOWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFOisbbI&md5=2d1d3efa959f66eeb8789b48daf96325</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2016.11.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2016.11.020%26sid%3Dliteratum%253Aachs%26aulast%3DJain%26aufirst%3DA.%2BK.%26aulast%3DBarton%26aufirst%3DM.%2BC.%26atitle%3DBromodomain%2520Histone%2520Readers%2520and%2520Cancer%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2017%26volume%3D429%26spage%3D2003%26epage%3D2010%26doi%3D10.1016%2Fj.jmb.2016.11.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Noguchi-Yachide, T.</span></span> <span> </span><span class="NLM_article-title">BET Bromodomain as a Target of Epigenetic Therapy</span>. <i>Chem. Pharm. Bull. (Tokyo)</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>64</i></span>,  <span class="NLM_fpage">540</span>– <span class="NLM_lpage">547</span>, <span class="refDoi"> DOI: 10.1248/cpb.c16-00225</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1248%2Fcpb.c16-00225" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=27250788" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A280%3ADC%252BC2s%252FmtVShtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2016&pages=540-547&author=T.+Noguchi-Yachide&title=BET+Bromodomain+as+a+Target+of+Epigenetic+Therapy&doi=10.1248%2Fcpb.c16-00225"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">BET Bromodomain as a Target of Epigenetic Therapy</span></div><div class="casAuthors">Noguchi-Yachide Tomomi</div><div class="citationInfo"><span class="NLM_cas:title">Chemical & pharmaceutical bulletin</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">540-7</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Acetylation of histone is a key epigenetic modification, and contributes to many DNA-dependent cellular processes.  The bromodomain structure, which consists of approximately 110 amino acid residues, serves as a 'reader' that recognizes acetylated lysine in histones, leading to recruitment of positive transcriptional elongation factor b (P-TEFb), and thereby promoting transcriptional activity and chromatin remodeling.  Among bromodomain-containing proteins, members of the bromodomain and extra-terminal domain (BET) family contain tandem N-terminal bromodomains.  BET proteins, especially BRD4, have attracted interest as candidate therapeutic targets due to their putative involvement in the pathogenesis of various diseases, including cancer and inflammatory diseases.  Several BET inhibitors are under clinical trial for treatment of various cancers.  Furthermore, polypharmacological agents such as dual kinase/BET inhibitors and dual histone deacetylase (HDAC)/BET inhibitors have recently been developed, in addition to agents that degrade BET family proteins, such as proteolysis-targeting chimeras (PROTACs).  This paper reviews recent progress in epigenetic therapy targeting the BET bromodomain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRr5yqeTtyiLovKgEt20y1XfW6udTcc2eb--wCibhfLSLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2s%252FmtVShtw%253D%253D&md5=7046d36daea4f693f1ebf6f19555db47</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1248%2Fcpb.c16-00225&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1248%252Fcpb.c16-00225%26sid%3Dliteratum%253Aachs%26aulast%3DNoguchi-Yachide%26aufirst%3DT.%26atitle%3DBET%2520Bromodomain%2520as%2520a%2520Target%2520of%2520Epigenetic%2520Therapy%26jtitle%3DChem.%2520Pharm.%2520Bull.%2520%2528Tokyo%2529%26date%3D2016%26volume%3D64%26spage%3D540%26epage%3D547%26doi%3D10.1248%2Fcpb.c16-00225" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Padmanabhan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathur, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manjula, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tripathi, S.</span></span> <span> </span><span class="NLM_article-title">Bromodomain and Extra-Terminal (BET) Family Proteins: New Therapeutic Targets in Major Diseases</span>. <i>J. Biosci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">295</span>– <span class="NLM_lpage">311</span>, <span class="refDoi"> DOI: 10.1007/s12038-016-9600-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1007%2Fs12038-016-9600-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=27240990" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC28XptFOqtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2016&pages=295-311&author=B.+Padmanabhanauthor=S.+Mathurauthor=R.+Manjulaauthor=S.+Tripathi&title=Bromodomain+and+Extra-Terminal+%28BET%29+Family+Proteins%3A+New+Therapeutic+Targets+in+Major+Diseases&doi=10.1007%2Fs12038-016-9600-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomain and extra-terminal (BET) family proteins: New therapeutic targets in major diseases</span></div><div class="casAuthors">Padmanabhan, Balasundaram; Mathur, Shruti; Manjula, Ramu; Tripathi, Shailesh</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biosciences (New Delhi, India)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">295-311</span>CODEN:
                <span class="NLM_cas:coden">JOBSDN</span>;
        ISSN:<span class="NLM_cas:issn">0250-5991</span>.
    
            (<span class="NLM_cas:orgname">Springer (India) Private Ltd.</span>)
        </div><div class="casAbstract">The bromodomains and extra-terminal domain (BET) family proteins recognize acetylated chromatin through their bromodomains (BDs) and help in regulating gene expression.  BDs are chromatin 'readers': by interacting with acetylated lysines on the histone tails, they recruit chromatin-regulating proteins on the promoter region to regulate gene expression and repression.  Extensive efforts have been employed by scientific communities worldwide to identify and develop potential inhibitors of BET family BDs to regulate protein expression by inhibiting acetylated histone (H3/H4) interactions.  Several small mol. inhibitors have been reported, which not only have high affinity but also have high specificity to BET BDs.  These developments make BET family proteins an important therapeutic targets for major diseases such as cancer, neurol. disorders, obesity and inflammation.  Here, we review and discuss the structural biol. of BET family BDs and their applications in major diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp87geC_AUDErVg90H21EOLACvtfcHk0lgzzzxMiKebTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XptFOqtbw%253D&md5=2555d6cb5db8e4151b63f7add86c7250</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1007%2Fs12038-016-9600-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12038-016-9600-6%26sid%3Dliteratum%253Aachs%26aulast%3DPadmanabhan%26aufirst%3DB.%26aulast%3DMathur%26aufirst%3DS.%26aulast%3DManjula%26aufirst%3DR.%26aulast%3DTripathi%26aufirst%3DS.%26atitle%3DBromodomain%2520and%2520Extra-Terminal%2520%2528BET%2529%2520Family%2520Proteins%253A%2520New%2520Therapeutic%2520Targets%2520in%2520Major%2520Diseases%26jtitle%3DJ.%2520Biosci.%26date%3D2016%26volume%3D41%26spage%3D295%26epage%3D311%26doi%3D10.1007%2Fs12038-016-9600-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, S.</span></span> <span> </span><span class="NLM_article-title">Disrupting Acetyl-Lysine Interactions: Recent Advance in the Development of BET Inhibitors</span>. <i>Curr. Drug Targets</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1148</span>– <span class="NLM_lpage">1165</span>, <span class="refDoi"> DOI: 10.2174/1389450119666171129165427</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.2174%2F1389450119666171129165427" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=29189147" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlCqt7bF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2018&pages=1148-1165&author=F.+Zhangauthor=S.+Ma&title=Disrupting+Acetyl-Lysine+Interactions%3A+Recent+Advance+in+the+Development+of+BET+Inhibitors&doi=10.2174%2F1389450119666171129165427"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Disrupting Acetyl-lysine Interactions: Recent Advance in the Development of BET Inhibitors</span></div><div class="casAuthors">Zhang, Fa; Ma, Shutao</div><div class="citationInfo"><span class="NLM_cas:title">Current Drug Targets</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1148-1165</span>CODEN:
                <span class="NLM_cas:coden">CDTUAU</span>;
        ISSN:<span class="NLM_cas:issn">1389-4501</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">Background: Histone acetylation is an essential approach of post-translational modification (PTM) and a significant component of epigenetic regulation that is mediated by Bromodomainscontaining protein (BRDs).  In recent years, many researchers have found that a variety of malignancy, inflammatory and other diseases occurrences and developments are assocd. with BRD4 expression disorders or dysfunction.  Meanwhile, many inhibitors of the extra-terminal (BET) family have been reported in many papers.  Objective: This review summarized those newly found BET inhibitors, their mechanism of action and bioactivity.  Secondly, those compds. were mainly classified based on their structures and their structure-activity relationship information was discussed.  Beyond that, every compd.'s design strategy was pointed out.  Results and Conclusion: Herein, the recent advances reported were reviewed for discovering more excellent small mol. inhibitors.  Currently, in addn. to compd. 4, compds. 7, 22 and 90, have also been into the clin. trial stage.  In the view of the outstanding performance of BET inhibitors in anti-tumor, anti-inflammatory and anti-drug resistance, we believe that more and more BET inhibitors will become the new epigenetic therapy for cancer, inflammation and autoimmune disease in clin. practice in the near future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2tTRPAV67lLVg90H21EOLACvtfcHk0lgzzzxMiKebTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlCqt7bF&md5=b70d378e6e49e10f91c1a3c8dcde1318</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.2174%2F1389450119666171129165427&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1389450119666171129165427%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DF.%26aulast%3DMa%26aufirst%3DS.%26atitle%3DDisrupting%2520Acetyl-Lysine%2520Interactions%253A%2520Recent%2520Advance%2520in%2520the%2520Development%2520of%2520BET%2520Inhibitors%26jtitle%3DCurr.%2520Drug%2520Targets%26date%3D2018%26volume%3D19%26spage%3D1148%26epage%3D1165%26doi%3D10.2174%2F1389450119666171129165427" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span> <i>CCS14 Scientific Posters</i>; <a href="https://www.cellcentric.com/ccs1477/Publications/" class="extLink">https://www.cellcentric.com/ccs1477/Publications/</a> (accessed 30 September 2019).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+CCS14+Scientific+Posters%3B+https%3A%2F%2Fwww.cellcentric.com%2Fccs1477%2FPublications%2F+%28accessed+30+September+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26jtitle%3DCCS14%2520Scientific%2520Posters" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span> <i>NCT03568656 -
Study to Evaluate CCS1477 in Advanced Tumours</i>; <a href="https://clinicaltrials.gov/ct2/show/NCT03568656?term=CCS1477&amp;rank=1" class="extLink">https://clinicaltrials.gov/ct2/show/NCT03568656?term=CCS1477&rank=1</a> (accessed 30 September 2019).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+NCT03568656+-%0AStudy+to+Evaluate+CCS1477+in+Advanced+Tumours%3B+https%3A%2F%2Fclinicaltrials.gov%2Fct2%2Fshow%2FNCT03568656%3Fterm%3DCCS1477%26rank%3D1+%28accessed+30+September+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26jtitle%3DNCT03568656%2520-%250AStudy%2520to%2520Evaluate%2520CCS1477%2520in%2520Advanced%2520Tumours" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zaware, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.-M.</span></span> <span> </span><span class="NLM_article-title">Bromodomain Biology and Drug Discovery</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">870</span>– <span class="NLM_lpage">879</span>, <span class="refDoi"> DOI: 10.1038/s41594-019-0309-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1038%2Fs41594-019-0309-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=31582847" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvFSnsL3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2019&pages=870-879&author=N.+Zawareauthor=M.-M.+Zhou&title=Bromodomain+Biology+and+Drug+Discovery&doi=10.1038%2Fs41594-019-0309-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomain biology and drug discovery</span></div><div class="casAuthors">Zaware, Nilesh; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">870-879</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The bromodomain (BrD) is a conserved structural module found in chromatin- and transcription-assocd. proteins that acts as the primary reader for acetylated lysine residues.  This basic activity endows BrD proteins with versatile functions in the regulation of protein-protein interactions mediating chromatin-templated gene transcription, DNA recombination, replication and repair.  Consequently, BrD proteins are involved in the pathogenesis of numerous human diseases.  In this Review, we highlight our current understanding of BrD biol., and discuss the latest development of small-mol. inhibitors targeting BrDs as emerging epigenetic therapies for cancer and inflammatory disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0q-3UgSbqULVg90H21EOLACvtfcHk0lgzzzxMiKebTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvFSnsL3E&md5=7ccf3a3a2ff7147ead1148ee71fa2806</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fs41594-019-0309-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41594-019-0309-8%26sid%3Dliteratum%253Aachs%26aulast%3DZaware%26aufirst%3DN.%26aulast%3DZhou%26aufirst%3DM.-M.%26atitle%3DBromodomain%2520Biology%2520and%2520Drug%2520Discovery%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2019%26volume%3D26%26spage%3D870%26epage%3D879%26doi%3D10.1038%2Fs41594-019-0309-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schiedel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moroglu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ascough, D. M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chamberlain, A. E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamps, J. J. A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sekirnik, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conway, S. J.</span></span> <span> </span><span class="NLM_article-title">Chemical Epigenetics: The Impact of Chemical and Chemical Biology Techniques on Bromodomain Target Validation</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">17930</span>– <span class="NLM_lpage">17952</span>, <span class="refDoi"> DOI: 10.1002/anie.201812164</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1002%2Fanie.201812164" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitFKrt77N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2019&pages=17930-17952&author=M.+Schiedelauthor=M.+Morogluauthor=D.+M.+H.+Ascoughauthor=A.+E.+R.+Chamberlainauthor=J.+J.+A.+G.+Kampsauthor=A.+R.+Sekirnikauthor=S.+J.+Conway&title=Chemical+Epigenetics%3A+The+Impact+of+Chemical+and+Chemical+Biology+Techniques+on+Bromodomain+Target+Validation&doi=10.1002%2Fanie.201812164"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical Epigenetics: The Impact of Chemical and Chemical Biology Techniques on Bromodomain Target Validation</span></div><div class="casAuthors">Schiedel, Matthias; Moroglu, Mustafa; Ascough, David M. H.; Chamberlain, Anna E. R.; Kamps, Jos J. A. G.; Sekirnik, Angelina R.; Conway, Stuart J.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">50</span>),
    <span class="NLM_cas:pages">17930-17952</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Epigenetics is currently the focus of intense research interest across a broad range of disciplines due to its importance in a multitude of biol. processes and disease states.  Epigenetic functions result partly from modification of the nucleobases in DNA and RNA, and/or post-translational modifications of histone proteins.  These modifications are dynamic, with cellular machinery identified to modulate and interpret the marks.  The authors' focus is on bromodomains, which bind to acetylated lysine residues.  Progress in the study of bromodomains, and the development of bromodomain ligands, has been rapid.  These advances have been underpinned by many disciplines, but chem. and chem. biol. have undoubtedly played a significant role.  Herein, the authors review the key chem. and chem. biol. approaches that have furthered the authors' study of bromodomains, enabled the development of bromodomain ligands, and played a crit. role in the validation of bromodomains as therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq99ft2MpAlMLVg90H21EOLACvtfcHk0lhR95dtIDfH3A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitFKrt77N&md5=d26cbf7f78d9542b2c98b9b74d979761</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1002%2Fanie.201812164&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201812164%26sid%3Dliteratum%253Aachs%26aulast%3DSchiedel%26aufirst%3DM.%26aulast%3DMoroglu%26aufirst%3DM.%26aulast%3DAscough%26aufirst%3DD.%2BM.%2BH.%26aulast%3DChamberlain%26aufirst%3DA.%2BE.%2BR.%26aulast%3DKamps%26aufirst%3DJ.%2BJ.%2BA.%2BG.%26aulast%3DSekirnik%26aufirst%3DA.%2BR.%26aulast%3DConway%26aufirst%3DS.%2BJ.%26atitle%3DChemical%2520Epigenetics%253A%2520The%2520Impact%2520of%2520Chemical%2520and%2520Chemical%2520Biology%2520Techniques%2520on%2520Bromodomain%2520Target%2520Validation%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2019%26volume%3D58%26spage%3D17930%26epage%3D17952%26doi%3D10.1002%2Fanie.201812164" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oprea, T. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bologa, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curpan, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glen, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lipinski, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marshall, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ostopovici-Halip, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rishton, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ursu, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaz, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waller, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sklar, L. A.</span></span> <span> </span><span class="NLM_article-title">A Crowdsourcing Evaluation of the NIH Chemical Probes</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">441</span>– <span class="NLM_lpage">447</span>, <span class="refDoi"> DOI: 10.1038/nchembio0709-441</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1038%2Fnchembio0709-441" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=19536101" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BD1MXnsVags7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=441-447&author=T.+I.+Opreaauthor=C.+G.+Bologaauthor=S.+Boyerauthor=R.+F.+Curpanauthor=R.+C.+Glenauthor=A.+L.+Hopkinsauthor=C.+A.+Lipinskiauthor=G.+R.+Marshallauthor=Y.+C.+Martinauthor=L.+Ostopovici-Halipauthor=G.+Rishtonauthor=O.+Ursuauthor=R.+J.+Vazauthor=C.+Wallerauthor=H.+Waldmannauthor=L.+A.+Sklar&title=A+Crowdsourcing+Evaluation+of+the+NIH+Chemical+Probes&doi=10.1038%2Fnchembio0709-441"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">A crowdsourcing evaluation of the NIH chemical probes</span></div><div class="casAuthors">Oprea, Tudor I.; Bologa, Cristian G.; Boyer, Scott; Curpan, Ramona F.; Glen, Robert C.; Hopkins, Andrew L.; Lipinski, Christopher A.; Marshall, Garland R.; Martin, Yvonne C.; Ostopovici-Halip, Liliana; Rishton, Gilbert; Ursu, Oleg; Vaz, Roy J.; Waller, Chris; Waldmann, Herbert; Sklar, Larry A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">441-447</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review with commentary.  Between 2004 and 2008, the US National Institutes of Health Mol. Libraries and Imaging initiative pilot phase funded 10 high-throughput screening centers, resulting in the deposition of 691 assays into PubChem and the nomination of 64 chem. probes.  We crowdsourced the Mol. Libraries and Imaging initiative output to 11 experts, who expressed medium or high levels of confidence in 48 of these 64 probes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2vr16pYCeybVg90H21EOLACvtfcHk0lhR95dtIDfH3A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXnsVags7k%253D&md5=adc3ed5b33cbd233ec6160c5465651b7</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnchembio0709-441&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio0709-441%26sid%3Dliteratum%253Aachs%26aulast%3DOprea%26aufirst%3DT.%2BI.%26aulast%3DBologa%26aufirst%3DC.%2BG.%26aulast%3DBoyer%26aufirst%3DS.%26aulast%3DCurpan%26aufirst%3DR.%2BF.%26aulast%3DGlen%26aufirst%3DR.%2BC.%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26aulast%3DMarshall%26aufirst%3DG.%2BR.%26aulast%3DMartin%26aufirst%3DY.%2BC.%26aulast%3DOstopovici-Halip%26aufirst%3DL.%26aulast%3DRishton%26aufirst%3DG.%26aulast%3DUrsu%26aufirst%3DO.%26aulast%3DVaz%26aufirst%3DR.%2BJ.%26aulast%3DWaller%26aufirst%3DC.%26aulast%3DWaldmann%26aufirst%3DH.%26aulast%3DSklar%26aufirst%3DL.%2BA.%26atitle%3DA%2520Crowdsourcing%2520Evaluation%2520of%2520the%2520NIH%2520Chemical%2520Probes%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2009%26volume%3D5%26spage%3D441%26epage%3D447%26doi%3D10.1038%2Fnchembio0709-441" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S. V.</span></span> <span> </span><span class="NLM_article-title">The Art of the Chemical Probe</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">159</span>– <span class="NLM_lpage">161</span>, <span class="refDoi"> DOI: 10.1038/nchembio.296</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1038%2Fnchembio.296" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=20154659" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhvFGntbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=159-161&author=S.+V.+Frye&title=The+Art+of+the+Chemical+Probe&doi=10.1038%2Fnchembio.296"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">The art of the chemical probe</span></div><div class="casAuthors">Frye, Stephen V.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">159-161</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Chem. biologists frequently aim to create small-mol. probes that interact with a specific protein in vitro in order to explore the role of the protein in a broader biol. context (cells or organisms), but a common understanding of what makes a high-quality probe is lacking.  Here I propose a set of principles to guide probe qualification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7vDfsTLhqWrVg90H21EOLACvtfcHk0ljmVRWMhM66Gg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhvFGntbg%253D&md5=b7b48c591a611b8686e11b3fc4f1b85c</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.296&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.296%26sid%3Dliteratum%253Aachs%26aulast%3DFrye%26aufirst%3DS.%2BV.%26atitle%3DThe%2520Art%2520of%2520the%2520Chemical%2520Probe%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2010%26volume%3D6%26spage%3D159%26epage%3D161%26doi%3D10.1038%2Fnchembio.296" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chekler, E. L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, L. H.</span></span> <span> </span><span class="NLM_article-title">Target Validation Using Chemical Probes</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">195</span>– <span class="NLM_lpage">199</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1197</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1038%2Fnchembio.1197" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=23508172" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktFagsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=195-199&author=M.+E.+Bunnageauthor=E.+L.+P.+Cheklerauthor=L.+H.+Jones&title=Target+Validation+Using+Chemical+Probes&doi=10.1038%2Fnchembio.1197"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Target validation using chemical probes</span></div><div class="casAuthors">Bunnage, Mark E.; Chekler, Eugene L. Piatnitski; Jones, Lyn H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">195-199</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review on the fully profiled chem. probes essential to support the unbiased interpretation of biol. expts. necessary for rigorous preclin. target validation.  We believe that by developing a 'chem. probe tool kit', and a framework for its use, chem. biol. can have a more central role in identifying targets of potential relevance to disease, avoiding many of the biases that complicate target validation as practiced currently.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLwRTT4pXGxbVg90H21EOLACvtfcHk0ljmVRWMhM66Gg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktFagsr4%253D&md5=af0c362882191de96a8e8698ff2c9ce8</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1197&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1197%26sid%3Dliteratum%253Aachs%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DChekler%26aufirst%3DE.%2BL.%2BP.%26aulast%3DJones%26aufirst%3DL.%2BH.%26atitle%3DTarget%2520Validation%2520Using%2520Chemical%2520Probes%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2013%26volume%3D9%26spage%3D195%26epage%3D199%26doi%3D10.1038%2Fnchembio.1197" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bassi, Z. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fillmore, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miah, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maller, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galwey, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waddington, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parravicini, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacMillan-Jones, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gongora, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphreys, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Churcher, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span> <span> </span><span class="NLM_article-title">Modulating PCAF/GCN5 Immune Cell Function Through a PROTAC Approach</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">2862</span>– <span class="NLM_lpage">2867</span>, <span class="refDoi"> DOI: 10.1021/acschembio.8b00705</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.8b00705" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1OktL7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=2862-2867&author=Z.+I.+Bassiauthor=M.+C.+Fillmoreauthor=A.+H.+Miahauthor=T.+D.+Chapmanauthor=C.+Mallerauthor=E.+J.+Robertsauthor=L.+C.+Davisauthor=D.+E.+Lewisauthor=N.+W.+Galweyauthor=K.+E.+Waddingtonauthor=V.+Parraviciniauthor=A.+L.+MacMillan-Jonesauthor=C.+Gongoraauthor=P.+G.+Humphreysauthor=I.+Churcherauthor=R.+K.+Prinjhaauthor=D.+F.+Tough&title=Modulating+PCAF%2FGCN5+Immune+Cell+Function+Through+a+PROTAC+Approach&doi=10.1021%2Facschembio.8b00705"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Modulating PCAF/GCN5 Immune Cell Function through a PROTAC Approach</span></div><div class="casAuthors">Bassi, Zuni I.; Fillmore, Martin C.; Miah, Afjal H.; Chapman, Trevor D.; Maller, Claire; Roberts, Emma J.; Davis, Lauren C.; Lewis, Darcy E.; Galwey, Nicholas W.; Waddington, Kirsty E.; Parravicini, Valentino; Macmillan-Jones, Abigail L.; Gongora, Celine; Humphreys, Philip G.; Churcher, Ian; Prinjha, Rab K.; Tough, David F.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2862-2867</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">P300/CBP-assocd. factor (PCAF) and general control nonderepressible 5 (GCN5) are closely related epigenetic proteins, each contg. an acetyltransferase domain and a bromodomain.  Consistent with reported roles for these proteins in immune function, we find that PCAF-deficient macrophages exhibit a markedly reduced ability to produce cytokines upon stimulation with lipopolysaccharide (LPS).  Investigating the potential to target this pathway pharmacol., we show that chem. inhibition of the PCAF/GCN5 bromodomains is insufficient to recapitulate the diminished inflammatory response of PCAF-deficient immune cells.  However, by generating the first PCAF/GCN5 proteolysis targeting chimera (PROTAC), we identify small mols. able to degrade PCAF/GCN5 and to potently modulate the expression of multiple inflammatory mediators in LPS-stimulated macrophages and dendritic cells.  Our data illustrate the power of the PROTAC approach in the context of multidomain proteins, revealing a novel anti-inflammatory therapeutic opportunity for targeting PCAF/GCN5.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUuwZ6JZtCirVg90H21EOLACvtfcHk0ljmVRWMhM66Gg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1OktL7I&md5=6783f39397381536ed40ac9e1b20fa5b</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Facschembio.8b00705&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.8b00705%26sid%3Dliteratum%253Aachs%26aulast%3DBassi%26aufirst%3DZ.%2BI.%26aulast%3DFillmore%26aufirst%3DM.%2BC.%26aulast%3DMiah%26aufirst%3DA.%2BH.%26aulast%3DChapman%26aufirst%3DT.%2BD.%26aulast%3DMaller%26aufirst%3DC.%26aulast%3DRoberts%26aufirst%3DE.%2BJ.%26aulast%3DDavis%26aufirst%3DL.%2BC.%26aulast%3DLewis%26aufirst%3DD.%2BE.%26aulast%3DGalwey%26aufirst%3DN.%2BW.%26aulast%3DWaddington%26aufirst%3DK.%2BE.%26aulast%3DParravicini%26aufirst%3DV.%26aulast%3DMacMillan-Jones%26aufirst%3DA.%2BL.%26aulast%3DGongora%26aufirst%3DC.%26aulast%3DHumphreys%26aufirst%3DP.%2BG.%26aulast%3DChurcher%26aufirst%3DI.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DTough%26aufirst%3DD.%2BF.%26atitle%3DModulating%2520PCAF%252FGCN5%2520Immune%2520Cell%2520Function%2520Through%2520a%2520PROTAC%2520Approach%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2018%26volume%3D13%26spage%3D2862%26epage%3D2867%26doi%3D10.1021%2Facschembio.8b00705" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clegg, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomkinson, N. C. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphreys, P. G.</span></span> <span> </span><span class="NLM_article-title">Advancements in the Development of Non-BET Bromodomain Chemical Probes</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">362</span>– <span class="NLM_lpage">385</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201800738</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1002%2Fcmdc.201800738" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=30624862" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1Ohs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=362-385&author=M.+A.+Cleggauthor=N.+C.+O.+Tomkinsonauthor=R.+K.+Prinjhaauthor=P.+G.+Humphreys&title=Advancements+in+the+Development+of+Non-BET+Bromodomain+Chemical+Probes&doi=10.1002%2Fcmdc.201800738"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Advancements in the Development of non-BET Bromodomain Chemical Probes</span></div><div class="casAuthors">Clegg, Michael A.; Tomkinson, Nicholas C. O.; Prinjha, Rab K.; Humphreys, Philip G.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">362-385</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The bromodomain and extra terminal (BET) family of bromodomain-contg. proteins (BCPs) have been the subject of extensive research over the past decade, resulting in a plethora of high-quality chem. probes for their tandem bromodomains.  In turn, these chem. probes have helped reveal the profound biol. role of the BET bromodomains and their role in disease, ultimately leading to a no. of mols. in active clin. development.  However, the BET subfamily represents just 8/61 of the known human bromodomains, and attention has now expanded to the biol. role of the remaining 53 non-BET bromodomains.  Rapid growth of this research area has been accompanied by a greater understanding of the requirements for an effective bromodomain chem. probe and has led to a no. of new non-BET bromodomain chem. probes being developed.  Advances since Dec. 2015 are discussed, highlighting the strengths/caveats of each mol., and the value they add toward validating the non-BET bromodomains as tractable therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_U9uvpaCEd7Vg90H21EOLACvtfcHk0ljmVRWMhM66Gg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1Ohs70%253D&md5=a93505759fe45ffd0352de2f0e17885a</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201800738&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201800738%26sid%3Dliteratum%253Aachs%26aulast%3DClegg%26aufirst%3DM.%2BA.%26aulast%3DTomkinson%26aufirst%3DN.%2BC.%2BO.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DHumphreys%26aufirst%3DP.%2BG.%26atitle%3DAdvancements%2520in%2520the%2520Development%2520of%2520Non-BET%2520Bromodomain%2520Chemical%2520Probes%26jtitle%3DChemMedChem%26date%3D2019%26volume%3D14%26spage%3D362%26epage%3D385%26doi%3D10.1002%2Fcmdc.201800738" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shain, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollack, J. R.</span></span> <span> </span><span class="NLM_article-title">The Spectrum of SWI/SNF Mutations, Ubiquitous in Human Cancers</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e55119</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0055119</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1371%2Fjournal.pone.0055119" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=23355908" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&author=A.+H.+Shainauthor=J.+R.+Pollack&title=The+Spectrum+of+SWI%2FSNF+Mutations%2C+Ubiquitous+in+Human+Cancers&doi=10.1371%2Fjournal.pone.0055119"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0055119&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0055119%26sid%3Dliteratum%253Aachs%26aulast%3DShain%26aufirst%3DA.%2BH.%26aulast%3DPollack%26aufirst%3DJ.%2BR.%26atitle%3DThe%2520Spectrum%2520of%2520SWI%252FSNF%2520Mutations%252C%2520Ubiquitous%2520in%2520Human%2520Cancers%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26doi%3D10.1371%2Fjournal.pone.0055119" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kadoch, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hargreaves, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodges, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elias, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ranish, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crabtree, G. R.</span></span> <span> </span><span class="NLM_article-title">Proteomic and Bioinformatic Analysis of Mammalian SWI/SNF Complexes Identifies Extensive Roles in Human Malignancy</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">592</span>– <span class="NLM_lpage">601</span>, <span class="refDoi"> DOI: 10.1038/ng.2628</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1038%2Fng.2628" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=23644491" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC3sXntVart7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2013&pages=592-601&author=C.+Kadochauthor=D.+C.+Hargreavesauthor=C.+Hodgesauthor=L.+Eliasauthor=L.+Hoauthor=J.+Ranishauthor=G.+R.+Crabtree&title=Proteomic+and+Bioinformatic+Analysis+of+Mammalian+SWI%2FSNF+Complexes+Identifies+Extensive+Roles+in+Human+Malignancy&doi=10.1038%2Fng.2628"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Proteomic and bioinformatic analysis of mammalian SWI/SNF complexes identifies extensive roles in human malignancy</span></div><div class="casAuthors">Kadoch, Cigall; Hargreaves, Diana C.; Hodges, Courtney; Elias, Laura; Ho, Lena; Ranish, Jeff; Crabtree, Gerald R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">592-601</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Subunits of mammalian SWI/SNF (mSWI/SNF or BAF) complexes have recently been implicated as tumor suppressors in human malignancies.  To understand the full extent of their involvement, we conducted a proteomic anal. of endogenous mSWI/SNF complexes, which identified several new dedicated, stable subunits not found in yeast SWI/SNF complexes, including BCL7A, BCL7B and BCL7C, BCL11A and BCL11B, BRD9 and SS18.  Incorporating these new members, we detd. mSWI/SNF subunit mutation frequency in exome and whole-genome sequencing studies of primary human tumors.  Notably, mSWI/SNF subunits are mutated in 19.6% of all human tumors reported in 44 studies.  Our anal. suggests that specific subunits protect against cancer in specific tissues.  In addn., mutations affecting more than one subunit, defined here as compd. heterozygosity, are prevalent in certain cancers.  Our studies demonstrate that mSWI/SNF is the most frequently mutated chromatin-regulatory complex (CRC) in human cancer, exhibiting a broad mutation pattern, similar to that of TP53.  Thus, proper functioning of polymorphic BAF complexes may constitute a major mechanism of tumor suppression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqP873DCpn7JrVg90H21EOLACvtfcHk0lhj5xWGhu2i0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXntVart7s%253D&md5=0f49cf3c59e7bc04b74970fe656b1e17</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fng.2628&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng.2628%26sid%3Dliteratum%253Aachs%26aulast%3DKadoch%26aufirst%3DC.%26aulast%3DHargreaves%26aufirst%3DD.%2BC.%26aulast%3DHodges%26aufirst%3DC.%26aulast%3DElias%26aufirst%3DL.%26aulast%3DHo%26aufirst%3DL.%26aulast%3DRanish%26aufirst%3DJ.%26aulast%3DCrabtree%26aufirst%3DG.%2BR.%26atitle%3DProteomic%2520and%2520Bioinformatic%2520Analysis%2520of%2520Mammalian%2520SWI%252FSNF%2520Complexes%2520Identifies%2520Extensive%2520Roles%2520in%2520Human%2520Malignancy%26jtitle%3DNat.%2520Genet.%26date%3D2013%26volume%3D45%26spage%3D592%26epage%3D601%26doi%3D10.1038%2Fng.2628" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hodges, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirkland, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crabtree, G. R.</span></span> <span> </span><span class="NLM_article-title">The Many Roles of BAF (MSWI/SNF) and PBAF Complexes in Cancer</span>. <i>Cold Spring Harb. Perspect. Med.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">a026930</span>, <span class="refDoi"> DOI: 10.1101/cshperspect.a026930</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1101%2Fcshperspect.a026930" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=27413115" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC1cXns1ygsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=a026930&author=C.+Hodgesauthor=J.+G.+Kirklandauthor=G.+R.+Crabtree&title=The+Many+Roles+of+BAF+%28MSWI%2FSNF%29+and+PBAF+Complexes+in+Cancer&doi=10.1101%2Fcshperspect.a026930"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">The many roles of BAF (mSWI/SNF) and PBAF complexes in cancer</span></div><div class="casAuthors">Hodges, Courtney; Kirkland, Jacob G.; Crabtree, Gerald R.</div><div class="citationInfo"><span class="NLM_cas:title">Cold Spring Harbor Perspectives in Medicine</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">a026930/1-a026930/25</span>CODEN:
                <span class="NLM_cas:coden">CSHPFV</span>;
        ISSN:<span class="NLM_cas:issn">2157-1422</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">During the last decade, a host of epigenetic mechanisms were found to contribute to cancer and other human diseases.  Several genomic studies have revealed that ∼ 20% of malignancies have alterations of the subunits of polymorphic BRG-/BRM-assocd. factor (BAF) and Polybromo-assocd. BAF (PBAF) complexes, making them among the most frequently mutated complexes in cancer.  Recurrent mutations arise in genes encoding several BAF/ PBAF subunits, including ARID1A, ARID2, PBRM1, SMARCA4, and SMARCB1.  These subunits share some degree of conservation with subunits from related ATP (ATP)-dependent chromatin remodeling complexes in model organisms, in which a large body of work provides insight into their roles in cancer.  Here, we review the roles of BAF- and PBAF-like complexes in these organisms, and relate these findings to recent discoveries in cancer epigenomics.  We review several roles of BAF and PBAF complexes in cancer, including transcriptional regulation, DNA repair, and regulation of chromatin architecture and topol.  More recent results highlight the need for new techniquesto study these complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCynky-OxCc7Vg90H21EOLACvtfcHk0lhj5xWGhu2i0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXns1ygsLs%253D&md5=71976f9f18bfad4c44fe26991679a194</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1101%2Fcshperspect.a026930&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fcshperspect.a026930%26sid%3Dliteratum%253Aachs%26aulast%3DHodges%26aufirst%3DC.%26aulast%3DKirkland%26aufirst%3DJ.%2BG.%26aulast%3DCrabtree%26aufirst%3DG.%2BR.%26atitle%3DThe%2520Many%2520Roles%2520of%2520BAF%2520%2528MSWI%252FSNF%2529%2520and%2520PBAF%2520Complexes%2520in%2520Cancer%26jtitle%3DCold%2520Spring%2520Harb.%2520Perspect.%2520Med.%26date%3D2016%26volume%3D6%26spage%3Da026930%26doi%3D10.1101%2Fcshperspect.a026930" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cleary, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeck, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Selitsky, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savich, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, T.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Getz, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parker, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powers, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guindi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghanekar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, D. Y.</span></span> <span> </span><span class="NLM_article-title">Identification of Driver Genes in Hepatocellular Carcinoma by Exome Sequencing</span>. <i>Hepatology</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">1693</span>– <span class="NLM_lpage">1702</span>, <span class="refDoi"> DOI: 10.1002/hep.26540</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1002%2Fhep.26540" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=23728943" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslWqsLbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2013&pages=1693-1702&author=S.+P.+Clearyauthor=W.+R.+Jeckauthor=X.+Zhaoauthor=K.+Chenauthor=S.+R.+Selitskyauthor=G.+L.+Savichauthor=T.-X.+Tanauthor=M.+C.+Wuauthor=G.+Getzauthor=M.+S.+Lawrenceauthor=J.+S.+Parkerauthor=J.+Liauthor=S.+Powersauthor=H.+Kimauthor=S.+Fischerauthor=M.+Guindiauthor=A.+Ghanekarauthor=D.+Y.+Chiang&title=Identification+of+Driver+Genes+in+Hepatocellular+Carcinoma+by+Exome+Sequencing&doi=10.1002%2Fhep.26540"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of driver genes in hepatocellular carcinoma by exome sequencing</span></div><div class="casAuthors">Cleary, Sean P.; Jeck, William R.; Zhao, Xiaobei; Chen, Kui; Selitsky, Sara R.; Savich, Gleb L.; Tan, Ting-Xu; Wu, Michael C.; Getz, Gad; Lawrence, Michael S.; Parker, Joel S.; Li, Jinyu; Powers, Scott; Kim, Hyeja; Fischer, Sandra; Guindi, Maha; Ghanekar, Anand; Chiang, Derek Y.</div><div class="citationInfo"><span class="NLM_cas:title">Hepatology (Hoboken, NJ, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1693-1702</span>CODEN:
                <span class="NLM_cas:coden">HPTLD9</span>;
        ISSN:<span class="NLM_cas:issn">0270-9139</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Genetic alterations in specific driver genes lead to disruption of cellular pathways and are crit. events in the instigation and progression of hepatocellular carcinoma (HCC).  As a prerequisite for individualized cancer treatment, we sought to characterize the landscape of recurrent somatic mutations in HCC.  We performed whole-exome sequencing on 87 HCCs and matched normal adjacent tissues to an av. coverage of 59×.  The overall mutation rate was roughly two mutations per Mb, with a median of 45 nonsynonymous mutations that altered the amino acid sequence (range, 2-381).  We found recurrent mutations in several genes with high transcript levels: TP53 (18%); CTNNB1 (10%); KEAP1 (8%); C16orf62 (8%); MLL4 (7%); and RAC2 (5%).  Significantly affected gene families include the nucleotide-binding domain and leucine-rich repeat-contg. family, calcium channel subunits, and histone methyltransferases.  In particular, the MLL family of methyltransferases for histone H3 lysine 4 were mutated in 20% of tumors.  Conclusion: The NFE2L2-KEAP1 and MLL pathways are recurrently mutated in multiple cohorts of HCC. (Hepatol. 2013;58:1693-1702).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYieayAje5DLVg90H21EOLACvtfcHk0lhSKzxpUCjOYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslWqsLbF&md5=fd9f6d7220f85ac6f2cc337c1cd2ef99</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1002%2Fhep.26540&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fhep.26540%26sid%3Dliteratum%253Aachs%26aulast%3DCleary%26aufirst%3DS.%2BP.%26aulast%3DJeck%26aufirst%3DW.%2BR.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DSelitsky%26aufirst%3DS.%2BR.%26aulast%3DSavich%26aufirst%3DG.%2BL.%26aulast%3DTan%26aufirst%3DT.-X.%26aulast%3DWu%26aufirst%3DM.%2BC.%26aulast%3DGetz%26aufirst%3DG.%26aulast%3DLawrence%26aufirst%3DM.%2BS.%26aulast%3DParker%26aufirst%3DJ.%2BS.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DPowers%26aufirst%3DS.%26aulast%3DKim%26aufirst%3DH.%26aulast%3DFischer%26aufirst%3DS.%26aulast%3DGuindi%26aufirst%3DM.%26aulast%3DGhanekar%26aufirst%3DA.%26aulast%3DChiang%26aufirst%3DD.%2BY.%26atitle%3DIdentification%2520of%2520Driver%2520Genes%2520in%2520Hepatocellular%2520Carcinoma%2520by%2520Exome%2520Sequencing%26jtitle%3DHepatology%26date%3D2013%26volume%3D58%26spage%3D1693%26epage%3D1702%26doi%3D10.1002%2Fhep.26540" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Troisi, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howard, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haswell, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hawk, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oberlick, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tzvetkov, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vazquez, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahn, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, C. W. M.</span></span> <span> </span><span class="NLM_article-title">BRD9 Defines a SWI/SNF Sub-Complex and Constitutes a Specific Vulnerability in Malignant Rhabdoid Tumors</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">11</span>, <span class="refDoi"> DOI: 10.1038/s41467-019-09891-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1038%2Fs41467-019-09891-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=30602773" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlSqs7rI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1-11&author=X.+Wangauthor=S.+Wangauthor=E.+C.+Troisiauthor=T.+P.+Howardauthor=J.+R.+Haswellauthor=B.+K.+Wolfauthor=W.+H.+Hawkauthor=P.+Ramosauthor=E.+M.+Oberlickauthor=E.+P.+Tzvetkovauthor=F.+Vazquezauthor=W.+C.+Hahnauthor=P.+J.+Parkauthor=C.+W.+M.+Roberts&title=BRD9+Defines+a+SWI%2FSNF+Sub-Complex+and+Constitutes+a+Specific+Vulnerability+in+Malignant+Rhabdoid+Tumors&doi=10.1038%2Fs41467-019-09891-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insights into the formation and voltage degradation of lithium- and manganese-rich layered oxides</span></div><div class="casAuthors">Hua, Weibo; Wang, Suning; Knapp, Michael; Leake, Steven J.; Senyshyn, Anatoliy; Richter, Carsten; Yavuz, Murat; Binder, Joachim R.; Grey, Clare P.; Ehrenberg, Helmut; Indris, Sylvio; Schwarz, Bjoern</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-11</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">One major challenge in the field of lithium-ion batteries is to understand the degrdn. mechanism of high-energy lithium- and manganese-rich layered cathode materials.  Although they can deliver 30% excess capacity compared with today's com.- used cathodes, the so-called voltage decay has been restricting their practical application.  In order to unravel the nature of this phenomenon, we have investigated systematically the structural and compositional dependence of manganese-rich lithium insertion compds. on the lithium content provided during synthesis.  Structural, electronic and electrochem. characterizations of LixNi0.2Mn0.6Oy with a wide range of lithium contents (0.00 ≤x ≤1.52, 1.07 ≤y < 2.4) and an anal. of the complexity in the synthesis pathways of monoclinic-layered Li[Li0.2Ni0.2Mn0.6]O2 oxide provide insight into the underlying processes that cause voltage fading in these cathode materials, i.e. transformation of the lithium-rich layered phase to a lithium-poor spinel phase via an intermediate lithium-contg. rock-salt phase with release of lithium/oxygen.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpP-zV7jY0wWbVg90H21EOLACvtfcHk0lhSKzxpUCjOYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlSqs7rI&md5=d4fbfd7183cf008ccb46b04d5cabb0a4</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fs41467-019-09891-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-019-09891-7%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DTroisi%26aufirst%3DE.%2BC.%26aulast%3DHoward%26aufirst%3DT.%2BP.%26aulast%3DHaswell%26aufirst%3DJ.%2BR.%26aulast%3DWolf%26aufirst%3DB.%2BK.%26aulast%3DHawk%26aufirst%3DW.%2BH.%26aulast%3DRamos%26aufirst%3DP.%26aulast%3DOberlick%26aufirst%3DE.%2BM.%26aulast%3DTzvetkov%26aufirst%3DE.%2BP.%26aulast%3DVazquez%26aufirst%3DF.%26aulast%3DHahn%26aufirst%3DW.%2BC.%26aulast%3DPark%26aufirst%3DP.%2BJ.%26aulast%3DRoberts%26aufirst%3DC.%2BW.%2BM.%26atitle%3DBRD9%2520Defines%2520a%2520SWI%252FSNF%2520Sub-Complex%2520and%2520Constitutes%2520a%2520Specific%2520Vulnerability%2520in%2520Malignant%2520Rhabdoid%2520Tumors%26jtitle%3DNat.%2520Commun.%26date%3D2019%26volume%3D10%26spage%3D1%26epage%3D11%26doi%3D10.1038%2Fs41467-019-09891-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.</span></span> <span> </span><span class="NLM_article-title">MiR-141 Is Involved in BRD7-Mediated Cell Proliferation and Tumor Formation Through Suppression of the PTEN/AKT Pathway in Nasopharyngeal Carcinoma</span>. <i>Cell Death Dis.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">e2156</span> <span class="refDoi"> DOI: 10.1038/cddis.2016.64</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1038%2Fcddis.2016.64" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=27010857" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC28XltFGmsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&author=Y.+Liuauthor=R.+Zhaoauthor=H.+Wangauthor=Y.+Luoauthor=X.+Wangauthor=W.+Niuauthor=Y.+Zhouauthor=Q.+Wenauthor=S.+Fanauthor=X.+Liauthor=W.+Xiongauthor=J.+Maauthor=X.+Liauthor=M.+Tanauthor=G.+Liauthor=M.+Zhou&title=MiR-141+Is+Involved+in+BRD7-Mediated+Cell+Proliferation+and+Tumor+Formation+Through+Suppression+of+the+PTEN%2FAKT+Pathway+in+Nasopharyngeal+Carcinoma&doi=10.1038%2Fcddis.2016.64"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">miR-141 is involved in BRD7-mediated cell proliferation and tumor formation through suppression of the PTEN/AKT pathway in nasopharyngeal carcinoma</span></div><div class="casAuthors">Liu, Y.; Zhao, R.; Wang, H.; Luo, Y.; Wang, X.; Niu, W.; Zhou, Y.; Wen, Q.; Fan, S.; Li, X.; Xiong, W.; Ma, J.; Li, X.; Tan, M.; Li, G.; Zhou, M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Death & Disease</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e2156</span>CODEN:
                <span class="NLM_cas:coden">CDDEA4</span>;
        ISSN:<span class="NLM_cas:issn">2041-4889</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Bromodomain contg. 7 (BRD7) was identified as a nuclear transcriptional regulatory factor.  BRD7 functions as a tumor suppressor in multiple cancers, including nasopharyngeal carcinoma (NPC).  In this study, we reported a novel mechanism of BRD7 in NPC progression.  We demonstrated that the expression of miR-141 was remarkably increased in NPC tissues and was neg. correlated with the expression of BRD7 and the survival rate of NPC patients.  Decreased expression levels of miR-141, including the primary, the precursor and the mature forms of miR-141, were found in BRD7-overexpressing HEK293, 5-8F and HNE1 cells compared the control cells, while there was no obvious effect on the expression levels of the two crit. enzymes Drosha and Dicer.  BRD7 can neg. regulate the promoter activity of miR-141, while no obvious binding site of BRD7 was found in the potential promoter region of miR-141.  Moreover, ectopic expression of miR-141 can significantly promote cell proliferation and inhibit apoptosis in NPC, and rescuing the expression of miR-141 in BRD7-overexpressing NPC cells could partially reverse the tumor suppressive effect of BRD7 on cell proliferation and tumor growth in vitro and in vivo.  Furthermore, the activation of the PTEN/AKT pathway mediated by the overexpression of BRD7 could be inhibited by rescuing the expression of miR-141, which accordingly results in the partial restoration of cell proliferation and tumor growth.  Our findings demonstrate that the BRD7/miR-141/PTEN/AKT axis has crit. roles in the progression of NPC and provide some promising targets for the diagnosis and treatment of NPC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZXqgJgJqtxbVg90H21EOLACvtfcHk0lhSKzxpUCjOYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XltFGmsbw%253D&md5=63a7b76c9f8a4a3f51ede5086ee4c00f</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fcddis.2016.64&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcddis.2016.64%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DLuo%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DNiu%26aufirst%3DW.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DWen%26aufirst%3DQ.%26aulast%3DFan%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DXiong%26aufirst%3DW.%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DTan%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DG.%26aulast%3DZhou%26aufirst%3DM.%26atitle%3DMiR-141%2520Is%2520Involved%2520in%2520BRD7-Mediated%2520Cell%2520Proliferation%2520and%2520Tumor%2520Formation%2520Through%2520Suppression%2520of%2520the%2520PTEN%252FAKT%2520Pathway%2520in%2520Nasopharyngeal%2520Carcinoma%26jtitle%3DCell%2520Death%2520Dis.%26date%3D2016%26volume%3D7%26doi%3D10.1038%2Fcddis.2016.64" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J.</span></span> <span> </span><span class="NLM_article-title">BRD7: A Novel Tumor Suppressor Gene in Different Cancers</span>. <i>Am. J. Transl. Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">742</span>– <span class="NLM_lpage">748</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=27158366" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC1cXmtFWlsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2016&pages=742-748&author=X.+Yuauthor=Z.+Liauthor=J.+Shen&title=BRD7%3A+A+Novel+Tumor+Suppressor+Gene+in+Different+Cancers"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">BRD7: a novel tumor suppressor gene in different cancers</span></div><div class="casAuthors">Yu, Xin; Li, Zheng; Shen, Jianxiong</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Translational Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">742-748</span>CODEN:
                <span class="NLM_cas:coden">AJTRA7</span>;
        ISSN:<span class="NLM_cas:issn">1943-8141</span>.
    
            (<span class="NLM_cas:orgname">e-Century Publishing Corp.</span>)
        </div><div class="casAbstract">BRD7 (bromodomain 7), also known as celtix-1, was first identified in nasopharyngeal carcinoma (NPC) cells in 2000.  BRD7 is a crucial component of both functional p53 and BRCA1 (breast cancer 1, early onset) pathways.  Recently, the BRD7 tumor suppressor status has been fully established.  Previous studies demonstrated that BRD7 was downregulated in human breast cancer and the downregulation often assocs. with tumor progression.  The expression of BRD7 was downregulated in various cancers, including breast cancer, NPC, gastric cancer, colorectal carcinoma, ovarian cancer, and prostate cancer.  Moreover, BRD7 inhibited cancer cell growth and metastasis and promote apoptosis in vitro and in vivo via downregulating AKT pathway.  In addn., BRD7 may regulate many signaling pathways including ras-raf-MEK-ERK and RB/E2F.  In this review, we provide an overview of current knowledge concerning the role of BRD7 in tumor development and progression.  To our knowledge, this is the first review about the role of this novel tumor suppressor gene BRD7in tumor development and progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqL0hhv3IxTHbVg90H21EOLACvtfcHk0lhVisOFavucZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXmtFWlsLs%253D&md5=6d6244ad2621b551afd1fb4244db7d3d</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DShen%26aufirst%3DJ.%26atitle%3DBRD7%253A%2520A%2520Novel%2520Tumor%2520Suppressor%2520Gene%2520in%2520Different%2520Cancers%26jtitle%3DAm.%2520J.%2520Transl.%2520Res.%26date%3D2016%26volume%3D8%26spage%3D742%26epage%3D748" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Drost, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mantovani, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tocco, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elkon, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Comel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holstege, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerkhoven, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jonkers, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voorhoeve, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agami, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Del Sal, G.</span></span> <span> </span><span class="NLM_article-title">BRD7 Is a Candidate Tumour Suppressor Gene Required for P53 Function</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">380</span>– <span class="NLM_lpage">389</span>, <span class="refDoi"> DOI: 10.1038/ncb2038</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1038%2Fncb2038" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=20228809" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC3cXktVyjurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2010&pages=380-389&author=J.+Drostauthor=F.+Mantovaniauthor=F.+Toccoauthor=R.+Elkonauthor=A.+Comelauthor=H.+Holstegeauthor=R.+Kerkhovenauthor=J.+Jonkersauthor=P.+M.+Voorhoeveauthor=R.+Agamiauthor=G.+Del+Sal&title=BRD7+Is+a+Candidate+Tumour+Suppressor+Gene+Required+for+P53+Function&doi=10.1038%2Fncb2038"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">BRD7 is a candidate tumour suppressor gene required for p53 function</span></div><div class="casAuthors">Drost, Jarno; Mantovani, Fiamma; Tocco, Francesca; Elkon, Ran; Comel, Anna; Holstege, Henne; Kerkhoven, Ron; Jonkers, Jos; Voorhoeve, P. Mathijs; Agami, Reuven; Del Sal, Giannino</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">380-389</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Oncogene-induced senescence is a p53-dependent defense mechanism against uncontrolled proliferation.  Consequently, many human tumors harbor p53 mutations and others show a dysfunctional p53 pathway, frequently by unknown mechanisms.  Here the authors identify BRD7 (bromodomain-contg. 7) as a protein whose inhibition allows full neoplastic transformation in the presence of wild-type p53.  In human breast tumors harboring wild-type, but not mutant, p53 the BRD7 gene locus was frequently deleted and low BRD7 expression was found in a subgroup of tumors.  Functionally, BRD7 is required for efficient p53-mediated transcription of a subset of target genes.  BRD7 interacts with p53 and p300 and is recruited to target gene promoters, affecting histone acetylation, p53 acetylation and promoter activity.  Thus, BRD7 suppresses tumorigenicity by serving as a p53 cofactor required for the efficient induction of p53-dependent oncogene-induced senescence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1grCCyCF9MrVg90H21EOLACvtfcHk0lhVisOFavucZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXktVyjurw%253D&md5=676cb9cbc5a8d157ac85d76c42ba8007</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fncb2038&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncb2038%26sid%3Dliteratum%253Aachs%26aulast%3DDrost%26aufirst%3DJ.%26aulast%3DMantovani%26aufirst%3DF.%26aulast%3DTocco%26aufirst%3DF.%26aulast%3DElkon%26aufirst%3DR.%26aulast%3DComel%26aufirst%3DA.%26aulast%3DHolstege%26aufirst%3DH.%26aulast%3DKerkhoven%26aufirst%3DR.%26aulast%3DJonkers%26aufirst%3DJ.%26aulast%3DVoorhoeve%26aufirst%3DP.%2BM.%26aulast%3DAgami%26aufirst%3DR.%26aulast%3DDel%2BSal%26aufirst%3DG.%26atitle%3DBRD7%2520Is%2520a%2520Candidate%2520Tumour%2520Suppressor%2520Gene%2520Required%2520for%2520P53%2520Function%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2010%26volume%3D12%26spage%3D380%26epage%3D389%26doi%3D10.1038%2Fncb2038" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nie, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gui, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.</span></span> <span> </span><span class="NLM_article-title">Bromodomain-containing Protein 7 Sensitizes Breast Cancer Cells to Paclitaxel by Activating Bcl2-antagonist/Killer Protein</span>. <i>Oncol. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">1487</span>– <span class="NLM_lpage">1496</span>, <span class="refDoi"> DOI: 10.3892/or.2018.6951</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.3892%2For.2018.6951" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=30592293" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtFygsrrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2019&pages=1487-1496&author=J.+Maauthor=W.+Niuauthor=X.+Wangauthor=Y.+Zhouauthor=H.+Wangauthor=F.+Liuauthor=Y.+Liuauthor=J.+Guoauthor=W.+Xiongauthor=Z.+Zengauthor=S.+Fanauthor=X.+Liauthor=X.+Nieauthor=G.+Liauthor=R.+Guiauthor=Y.+Luoauthor=M.+Zhou&title=Bromodomain-containing+Protein+7+Sensitizes+Breast+Cancer+Cells+to+Paclitaxel+by+Activating+Bcl2-antagonist%2FKiller+Protein&doi=10.3892%2For.2018.6951"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomain-containing protein 7 sensitizes breast cancer cells to paclitaxel by activating Bcl2-antagonist/killer protein</span></div><div class="casAuthors">Ma, Jinqi; Niu, Weihong; Wang, Xinye; Zhou, Yao; Wang, Heran; Liu, Fengxia; Liu, Yukun; Guo, Jie; Xiong, Wei; Zeng, Zhaoyang; Fan, Songqing; Li, Xiaoling; Nie, Xinmin; Li, Guiyuan; Gui, Rong; Luo, Yanwei; Zhou, Ming</div><div class="citationInfo"><span class="NLM_cas:title">Oncology Reports</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1487-1496</span>CODEN:
                <span class="NLM_cas:coden">OCRPEW</span>;
        ISSN:<span class="NLM_cas:issn">1791-2431</span>.
    
            (<span class="NLM_cas:orgname">Spandidos Publications Ltd.</span>)
        </div><div class="casAbstract">Our previous study demonstrated that bromodomain-contg. protein 7 (BRD7) inhibits cell proliferation and tumor growth, restoring the expression of B-cell lymphoma 2 antagonist/killer (Bak) sensitized breast cancer cells to paclitaxel.  However, the assocn. between BRD7 and paclitaxel sensitization, as well as BRD7 and Bak in breast cancer remains unknown.  In the present study, immunochem. staining was performed to measure the expression of BRD7 and Bak in breast cancer tissues.  Cell Counting Kit-8 assay, flow cytometry and tumor xenograft procedures were performed to evaluate the biol. role of BRD7 and Bak in breast cancer cells.  Western blotting, reverse transcription-quant. polymerase chain reaction, chromatin immunopptn. and luciferase reporter assays were also performed.  BRD7 was pos. correlated with Bak levels in breast cancer tissues, and the survival rate of patients with low Bak and BRD7 expression was significantly lower than that of patients with high Bak and BRD7 expression.  In addn., BRD7 activated Bak promoter activity and induced Bak expression in an indirect manner.  Furthermore, ectopic expression of BRD7 inhibited cell proliferation, tumor growth and sensitized cancer cells to paclitaxel, while knockdown of Bak abolished BRD7-mediated inhibitory effects on cell proliferation and paclitaxel sensitization in breast cancer cells whether in vitro and in vivo.  The results demonstrated that BRD7 inhibits cell proliferation and sensitizes breast cancer cells to paclitaxel by activating Bak; they also provide promising targets for the diagnosis and treatment of breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLZNf6xmTCUrVg90H21EOLACvtfcHk0lhVisOFavucZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtFygsrrI&md5=be5d7bd82cc0eb5436473694aaf53357</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.3892%2For.2018.6951&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252For.2018.6951%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DNiu%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DGuo%26aufirst%3DJ.%26aulast%3DXiong%26aufirst%3DW.%26aulast%3DZeng%26aufirst%3DZ.%26aulast%3DFan%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DNie%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DG.%26aulast%3DGui%26aufirst%3DR.%26aulast%3DLuo%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DM.%26atitle%3DBromodomain-containing%2520Protein%25207%2520Sensitizes%2520Breast%2520Cancer%2520Cells%2520to%2520Paclitaxel%2520by%2520Activating%2520Bcl2-antagonist%252FKiller%2520Protein%26jtitle%3DOncol.%2520Rep.%26date%3D2019%26volume%3D41%26spage%3D1487%26epage%3D1496%26doi%3D10.3892%2For.2018.6951" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clark, P. G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vieira, L. C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorova, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singletone, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monteiro, O. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennet, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baronio, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Méndez, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, D. J.</span></span> <span> </span><span class="NLM_article-title">LP99: Discovery and Synthesis of the First Selective BRD7/9 Bromodomain Inhibitor</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">6217</span>– <span class="NLM_lpage">6221</span>, <span class="refDoi"> DOI: 10.1002/anie.201501394</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1002%2Fanie.201501394" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmtlGqsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2015&pages=6217-6221&author=P.+G.+K.+Clarkauthor=L.+C.+C.+Vieiraauthor=C.+Tallantauthor=O.+Fedorovaauthor=D.+C.+Singletoneauthor=C.+M.+Rogersauthor=O.+P.+Monteiroauthor=J.+M.+Bennetauthor=R.+Baronioauthor=S.+M%C3%BCllerauthor=D.+L.+Danielsauthor=J.+M%C3%A9ndezauthor=S.+Knappauthor=P.+E.+Brennanauthor=D.+J.+Dixon&title=LP99%3A+Discovery+and+Synthesis+of+the+First+Selective+BRD7%2F9+Bromodomain+Inhibitor&doi=10.1002%2Fanie.201501394"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">LP99: Discovery and Synthesis of the First Selective BRD7/9 Bromodomain Inhibitor</span></div><div class="casAuthors">Clark, Peter G. K.; Vieira, Lucas C. C.; Tallant, Cynthia; Fedorov, Oleg; Singleton, Dean C.; Rogers, Catherine M.; Monteiro, Octovia P.; Bennett, James M.; Baronio, Roberta; Mueller, Susanne; Daniels, Danette L.; Mendez, Jacqui; Knapp, Stefan; Brennan, Paul E.; Dixon, Darren J.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6217-6221</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The bromodomain-contg. proteins BRD9 and BRD7 are part of the human SWI/SNF chromatin-remodeling complexes BAF and PBAF.  To date, no selective inhibitor for BRD7/9 has been reported despite its potential value as a biol. tool or as a lead for future therapeutics.  The quinolone-fused lactam LP99 (I) is now reported as the first potent and selective inhibitor of the BRD7 and BRD9 bromodomains.  Development of LP99 from a fragment hit was expedited through balancing structure-based inhibitor design and biophys. characterization against tractable chem. synthesis: Complexity-building nitro-Mannich/lactamization cascade processes allowed for early structure-activity relationship studies whereas an enantioselective organocatalytic nitro-Mannich reaction enabled the synthesis of the lead scaffold in enantioenriched form and on scale.  This epigenetic probe was shown to inhibit the assocn. of BRD7 and BRD9 to acetylated histones in vitro and in cells.  Moreover, LP99 was used to demonstrate that BRD7/9 plays a role in regulating pro-inflammatory cytokine secretion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplfOmbNMpdPbVg90H21EOLACvtfcHk0liGQ8h-rrZqoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmtlGqsrg%253D&md5=9dee2b352045f7ae5efcbdc83f240a3f</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1002%2Fanie.201501394&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201501394%26sid%3Dliteratum%253Aachs%26aulast%3DClark%26aufirst%3DP.%2BG.%2BK.%26aulast%3DVieira%26aufirst%3DL.%2BC.%2BC.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DFedorova%26aufirst%3DO.%26aulast%3DSingletone%26aufirst%3DD.%2BC.%26aulast%3DRogers%26aufirst%3DC.%2BM.%26aulast%3DMonteiro%26aufirst%3DO.%2BP.%26aulast%3DBennet%26aufirst%3DJ.%2BM.%26aulast%3DBaronio%26aufirst%3DR.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DDaniels%26aufirst%3DD.%2BL.%26aulast%3DM%25C3%25A9ndez%26aufirst%3DJ.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DDixon%26aufirst%3DD.%2BJ.%26atitle%3DLP99%253A%2520Discovery%2520and%2520Synthesis%2520of%2520the%2520First%2520Selective%2520BRD7%252F9%2520Bromodomain%2520Inhibitor%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2015%26volume%3D54%26spage%3D6217%26epage%3D6221%26doi%3D10.1002%2Fanie.201501394" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Theodoulou, N. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bannister, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becher, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bit, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dittmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewes, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewry, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leveridge, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molnar, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robson, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomkinson, N. C. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouzarides, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphreys, P. G.</span></span> <span> </span><span class="NLM_article-title">Discovery of I-BRD9, a Selective Cell Active Chemical Probe for Bromodomain Containing Protein 9 Inhibition</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1425</span>– <span class="NLM_lpage">1439</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00256</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00256" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmtlaktb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1425-1439&author=N.+H.+Theodoulouauthor=P.+Bamboroughauthor=A.+J.+Bannisterauthor=I.+Becherauthor=R.+A.+Bitauthor=K.+H.+Cheauthor=C.-w.+Chungauthor=A.+Dittmannauthor=G.+Drewesauthor=D.+H.+Drewryauthor=L.+Gordonauthor=P.+Grandiauthor=M.+Leveridgeauthor=M.+Lindonauthor=A.-M.+Michonauthor=J.+Molnarauthor=S.+C.+Robsonauthor=N.+C.+O.+Tomkinsonauthor=T.+Kouzaridesauthor=R.+K.+Prinjhaauthor=P.+G.+Humphreys&title=Discovery+of+I-BRD9%2C+a+Selective+Cell+Active+Chemical+Probe+for+Bromodomain+Containing+Protein+9+Inhibition&doi=10.1021%2Facs.jmedchem.5b00256"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of I-BRD9, a Selective Cell Active Chemical Probe for Bromodomain Containing Protein 9 Inhibition</span></div><div class="casAuthors">Theodoulou, Natalie H.; Bamborough, Paul; Bannister, Andrew J.; Becher, Isabelle; Bit, Rino A.; Che, Ka Hing; Chung, Chun-wa; Dittmann, Antje; Drewes, Gerard; Drewry, David H.; Gordon, Laurie; Grandi, Paola; Leveridge, Melanie; Lindon, Matthew; Michon, Anne-Marie; Molnar, Judit; Robson, Samuel C.; Tomkinson, Nicholas C. O.; Kouzarides, Tony; Prinjha, Rab K.; Humphreys, Philip G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1425-1439</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Acetylation of histone lysine residues is one of the most well-studied post-translational modifications of chromatin, selectively recognized by bromodomain "reader" modules.  Inhibitors of the bromodomain and extra terminal domain (BET) family of bromodomains have shown profound anticancer and anti-inflammatory properties, generating much interest in targeting other bromodomain-contg. proteins for disease treatment.  Herein, we report the discovery of I-BRD9, the first selective cellular chem. probe for bromodomain-contg. protein 9 (BRD9).  I-BRD9 was identified through structure-based design, leading to greater than 700-fold selectivity over the BET family and 200-fold over the highly homologous bromodomain-contg. protein 7 (BRD7).  I-BRD9 was used to identify genes regulated by BRD9 in Kasumi-1 cells involved in oncol. and immune response pathways and to the best of our knowledge, represents the first selective tool compd. available to elucidate the cellular phenotype of BRD9 bromodomain inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVmjhksIDyTrVg90H21EOLACvtfcHk0liGQ8h-rrZqoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmtlaktb8%253D&md5=4e8f6aa664797fe54772b64c81cf2a83</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00256%26sid%3Dliteratum%253Aachs%26aulast%3DTheodoulou%26aufirst%3DN.%2BH.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DBannister%26aufirst%3DA.%2BJ.%26aulast%3DBecher%26aufirst%3DI.%26aulast%3DBit%26aufirst%3DR.%2BA.%26aulast%3DChe%26aufirst%3DK.%2BH.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DDittmann%26aufirst%3DA.%26aulast%3DDrewes%26aufirst%3DG.%26aulast%3DDrewry%26aufirst%3DD.%2BH.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DLeveridge%26aufirst%3DM.%26aulast%3DLindon%26aufirst%3DM.%26aulast%3DMichon%26aufirst%3DA.-M.%26aulast%3DMolnar%26aufirst%3DJ.%26aulast%3DRobson%26aufirst%3DS.%2BC.%26aulast%3DTomkinson%26aufirst%3DN.%2BC.%2BO.%26aulast%3DKouzarides%26aufirst%3DT.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DHumphreys%26aufirst%3DP.%2BG.%26atitle%3DDiscovery%2520of%2520I-BRD9%252C%2520a%2520Selective%2520Cell%2520Active%2520Chemical%2520Probe%2520for%2520Bromodomain%2520Containing%2520Protein%25209%2520Inhibition%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1425%26epage%3D1439%26doi%3D10.1021%2Facs.jmedchem.5b00256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span> <i>SGC | TP-472 A BRD9/7 Probe</i>; <a href="http://www.thesgc.org/chemical-probes/TP-472" class="extLink">http://www.thesgc.org/chemical-probes/TP-472</a>(accessed 3 October 2019).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+SGC+%7C+TP-472+A+BRD9%2F7+Probe%3B+http%3A%2F%2Fwww.thesgc.org%2Fchemical-probes%2FTP-472%28accessed+3+October+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26jtitle%3DSGC%2520%257C%2520TP-472%2520A%2520BRD9%252F7%2520Probe" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="note"><p class="first last">For alternative molecules from the same series, see:</p></div><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hay, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monteiro, O. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schofield, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span> <span> </span><span class="NLM_article-title">Design and Synthesis of Potent and Selective Inhibitors of BRD7 and BRD9 Bromodomains</span>. <i>Med. Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1381</span>– <span class="NLM_lpage">1386</span>, <span class="refDoi"> DOI: 10.1039/C5MD00152H</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1039%2FC5MD00152H" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=1381-1386&author=D.+A.+Hayauthor=C.+M.+Rogersauthor=O.+Fedorovauthor=C.+Tallantauthor=S.+Martinauthor=O.+P.+Monteiroauthor=S.+M%C3%BCllerauthor=S.+Knappauthor=C.+J.+Schofieldauthor=P.+E.+Brennan&title=Design+and+Synthesis+of+Potent+and+Selective+Inhibitors+of+BRD7+and+BRD9+Bromodomains&doi=10.1039%2FC5MD00152H"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1039%2FC5MD00152H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5MD00152H%26sid%3Dliteratum%253Aachs%26aulast%3DHay%26aufirst%3DD.%2BA.%26aulast%3DRogers%26aufirst%3DC.%2BM.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DMonteiro%26aufirst%3DO.%2BP.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26atitle%3DDesign%2520and%2520Synthesis%2520of%2520Potent%2520and%2520Selective%2520Inhibitors%2520of%2520BRD7%2520and%2520BRD9%2520Bromodomains%26jtitle%3DMed.%2520Chem.%2520Commun.%26date%3D2015%26volume%3D6%26spage%3D1381%26epage%3D1386%26doi%3D10.1039%2FC5MD00152H" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martin, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koegl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bader, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cockcroft, X.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiegen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstberger, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofmann, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hohmann, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knesl, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kornigg, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nar, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rumpel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaaf, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steurer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakoc, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeeb, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoephel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boehmelt, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McConnell, D.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Design of an in Vivo Active Selective BRD9 Inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">4462</span>– <span class="NLM_lpage">4475</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01865</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01865" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC28Xjt1Sit7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4462-4475&author=L.+J.+Martinauthor=M.+Koeglauthor=G.+Baderauthor=X.-L.+Cockcroftauthor=O.+Fedorovauthor=D.+Fiegenauthor=T.+Gerstbergerauthor=M.+H.+Hofmannauthor=A.+F.+Hohmannauthor=D.+Kesslerauthor=S.+Knappauthor=P.+Kneslauthor=S.+Korniggauthor=S.+M%C3%BCllerauthor=H.+Narauthor=C.+Rogersauthor=K.+Rumpelauthor=O.+Schaafauthor=S.+Steurerauthor=C.+Tallantauthor=C.+R.+Vakocauthor=M.+Zeebauthor=A.+Zoephelauthor=M.+Pearsonauthor=G.+Boehmeltauthor=D.+McConnell&title=Structure-Based+Design+of+an+in+Vivo+Active+Selective+BRD9+Inhibitor&doi=10.1021%2Facs.jmedchem.5b01865"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of an in Vivo Active Selective BRD9 Inhibitor</span></div><div class="casAuthors">Martin, Laetitia J.; Koegl, Manfred; Bader, Gerd; Cockcroft, Xiao-Ling; Fedorov, Oleg; Fiegen, Dennis; Gerstberger, Thomas; Hofmann, Marco H.; Hohmann, Anja F.; Kessler, Dirk; Knapp, Stefan; Knesl, Petr; Kornigg, Stefan; Mueller, Susanne; Nar, Herbert; Rogers, Catherine; Rumpel, Klaus; Schaaf, Otmar; Steurer, Steffen; Tallant, Cynthia; Vakoc, Christopher R.; Zeeb, Markus; Zoephel, Andreas; Pearson, Mark; Boehmelt, Guido; McConnell, Darryl</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4462-4475</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Components of the chromatin remodelling switch/sucrose nonfermentable (SWI/SNF) complex are recurrently mutated in tumors, suggesting that altering the activity of the complex plays a role in oncogenesis.  However, the role that the individual subunits play in this process is not clear.  We set out to develop an inhibitor compd. targeting the bromodomain of BRD9 in order to evaluate its function within the SWI/SNF complex.  Here, we present the discovery and development of a potent and selective BRD9 bromodomain inhibitor series based on a new pyridinone-like scaffold.  Crystallog. information on the inhibitors bound to BRD9 guided their development with respect to potency for BRD9 and selectivity against BRD4.  These compds. modulate BRD9 bromodomain cellular function and display antitumor activity in an AML xenograft model.  Two chem. probes, BI-7273 (1) and BI-9564 (2), were identified that should prove to be useful in further exploring BRD9 bromodomain biol. in both in vitro and in vivo settings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWHs4hA8bijbVg90H21EOLACvtfcHk0lgVhpmUY14nPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xjt1Sit7w%253D&md5=b1f4394a7a410273c800b65ee9697c7e</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01865&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01865%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DL.%2BJ.%26aulast%3DKoegl%26aufirst%3DM.%26aulast%3DBader%26aufirst%3DG.%26aulast%3DCockcroft%26aufirst%3DX.-L.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DFiegen%26aufirst%3DD.%26aulast%3DGerstberger%26aufirst%3DT.%26aulast%3DHofmann%26aufirst%3DM.%2BH.%26aulast%3DHohmann%26aufirst%3DA.%2BF.%26aulast%3DKessler%26aufirst%3DD.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DKnesl%26aufirst%3DP.%26aulast%3DKornigg%26aufirst%3DS.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DNar%26aufirst%3DH.%26aulast%3DRogers%26aufirst%3DC.%26aulast%3DRumpel%26aufirst%3DK.%26aulast%3DSchaaf%26aufirst%3DO.%26aulast%3DSteurer%26aufirst%3DS.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26aulast%3DZeeb%26aufirst%3DM.%26aulast%3DZoephel%26aufirst%3DA.%26aulast%3DPearson%26aufirst%3DM.%26aulast%3DBoehmelt%26aufirst%3DG.%26aulast%3DMcConnell%26aufirst%3DD.%26atitle%3DStructure-Based%2520Design%2520of%2520an%2520in%2520Vivo%2520Active%2520Selective%2520BRD9%2520Inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D4462%26epage%3D4475%26doi%3D10.1021%2Facs.jmedchem.5b01865" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Remillard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brien, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sonnett, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seo, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dastjerdi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wühr, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Degradation of the BAF Complex Factor BRD9 by Heterobifunctional Ligands</span>. <i>Angew. Chem. Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">5738</span>– <span class="NLM_lpage">5743</span>, <span class="refDoi"> DOI: 10.1002/anie.201611281</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1002%2Fanie.201611281" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmtFGrurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=5738-5743&author=D.+Remillardauthor=D.+L.+Buckleyauthor=J.+Paulkauthor=G.+L.+Brienauthor=M.+Sonnettauthor=H.-S.+Seoauthor=S.+Dastjerdiauthor=M.+W%C3%BChrauthor=S.+Dhe-Paganonauthor=S.+A.+Armstrongauthor=J.+E.+Bradner&title=Degradation+of+the+BAF+Complex+Factor+BRD9+by+Heterobifunctional+Ligands&doi=10.1002%2Fanie.201611281"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Degradation of the BAF Complex Factor BRD9 by Heterobifunctional Ligands</span></div><div class="casAuthors">Remillard, David; Buckley, Dennis L.; Paulk, Joshiawa; Brien, Gerard L.; Sonnett, Matthew; Seo, Hyuk-Soo; Dastierdi, Shiva; Wuehr, Martin; Dhe-Paganon, Sirano; Armstrong, Scott A.; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">5738-5743</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The bromodomain-contg. protein BRD9, a subunit of the human BAF (SWI/SNF) nucleosome remodeling complex, has emerged as an attractive therapeutic target in cancer.  Despite the development of chem. probes targeting the BRD9 bromodomain, there is a limited understanding of BRD9 function beyond acetyl-lysine recognition.  We have therefore created the first BRD9-directed chem. degraders, through iterative design and testing of heterobifunctional ligands that bridge the BRD9 bromodomain and the cereblon E3 ubiquitin ligase complex.  Degraders of BRD9 exhibit markedly enhanced potency compared to parental ligands (10- to 100-fold).  Parallel study of degraders with divergent BRD9-binding chemotypes in models of acute myeloid leukemia resolves bromodomain polypharmacol. in this emerging drug class.  Together, these findings reveal the tractability of non-BET bromodomain contg. proteins to chem. degrdn., and highlight lead compd. dBRD9 as a tool for the study of BRD9.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQ9FU2Ipaq27Vg90H21EOLACvtfcHk0lgVhpmUY14nPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmtFGrurk%253D&md5=2eb6562cc9d976c7ae6070c9a419e9ea</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1002%2Fanie.201611281&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201611281%26sid%3Dliteratum%253Aachs%26aulast%3DRemillard%26aufirst%3DD.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DBrien%26aufirst%3DG.%2BL.%26aulast%3DSonnett%26aufirst%3DM.%26aulast%3DSeo%26aufirst%3DH.-S.%26aulast%3DDastjerdi%26aufirst%3DS.%26aulast%3DW%25C3%25BChr%26aufirst%3DM.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DDegradation%2520of%2520the%2520BAF%2520Complex%2520Factor%2520BRD9%2520by%2520Heterobifunctional%2520Ligands%26jtitle%3DAngew.%2520Chem.%2520Int.%2520Ed.%26date%3D2017%26volume%3D56%26spage%3D5738%26epage%3D5743%26doi%3D10.1002%2Fanie.201611281" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brien, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Remillard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hemming, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chabon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wynne, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dillon, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cagney, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Mierlo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baltissen, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vermeulen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fröhling, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakoc, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, S. A.</span></span> <span> </span><span class="NLM_article-title">Targeted Degradation of BRD9 Reverses Oncogenic Gene Expression in Synovial Sarcoma</span>. <i>eLife</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">e41305</span> <span class="refDoi"> DOI: 10.7554/eLife.41305</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.7554%2FeLife.41305" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=30431433" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlyqs7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2018&author=G.+L.+Brienauthor=D.+Remillardauthor=J.+Shiauthor=M.+L.+Hemmingauthor=J.+Chabonauthor=K.+Wynneauthor=E.+T.+Dillonauthor=G.+Cagneyauthor=G.+Van+Mierloauthor=M.+P.+Baltissenauthor=M.+Vermeulenauthor=J.+Qiauthor=S.+Fr%C3%B6hlingauthor=N.+S.+Grayauthor=J.+E.+Bradnerauthor=C.+R.+Vakocauthor=S.+A.+Armstrong&title=Targeted+Degradation+of+BRD9+Reverses+Oncogenic+Gene+Expression+in+Synovial+Sarcoma&doi=10.7554%2FeLife.41305"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted degradation of BRD9 reverses oncogenic gene expression in synovial sarcoma</span></div><div class="casAuthors">Brien, Gerard L.; Remillard, David; Shi, Junwei; Hemming, Matthew L.; Chabon, Jonathon; Wynne, Kieran; Dillon, Euge Ne T.; Cagney, Gerard; Van Mierlo, Guido; Baltissen, Marijke P.; Vermeulen, Michiel; Qi, Jun; Froehling, Stefan; Gray, Nathanael S.; Bradner, James E.; Vakoc, Christopher R.; Armstrong, Scott A.</div><div class="citationInfo"><span class="NLM_cas:title">eLife</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e41305/1-e41305/26</span>CODEN:
                <span class="NLM_cas:coden">ELIFA8</span>;
        ISSN:<span class="NLM_cas:issn">2050-084X</span>.
    
            (<span class="NLM_cas:orgname">eLife Sciences Publications Ltd.</span>)
        </div><div class="casAbstract">Synovial sarcoma tumors contain a characteristic fusion protein, SS18-SSX, which drives disease development.  Targeting oncogenic fusion proteins presents an attractive therapeutic opportunity.  However, SS18-SSX has proven intractable for therapeutic intervention.  Using a domain-focused CRISPR screen we identified the bromodomain of BRD9 as a crit. functional dependency in synovial sarcoma.  BRD9 is a component of SS18-SSX contg. BAF complexes in synovial sarcoma cells; and integration of BRD9 into these complexes is crit. for cell growth.  Moreover BRD9 and SS18-SSX co-localize extensively on the synovial sarcoma genome.  Remarkably, synovial sarcoma cells are highly sensitive to a novel small mol. degrader of BRD9, while other sarcoma subtypes are unaffected.  Degrdn. of BRD9 induces downregulation of oncogenic transcriptional programs and inhibits tumor progression in vivo.  We demonstrate that BRD9 supports oncogenic mechanisms underlying the SS18-SSX fusion in synovial sarcoma and highlight targeted degrdn. of BRD9 as a potential therapeutic opportunity in this disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfd6-D9YlmrrVg90H21EOLACvtfcHk0ljSw5ajKF5fAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlyqs7fF&md5=cbda64457406cd0d8fbd1211678b04b3</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.7554%2FeLife.41305&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.41305%26sid%3Dliteratum%253Aachs%26aulast%3DBrien%26aufirst%3DG.%2BL.%26aulast%3DRemillard%26aufirst%3DD.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DHemming%26aufirst%3DM.%2BL.%26aulast%3DChabon%26aufirst%3DJ.%26aulast%3DWynne%26aufirst%3DK.%26aulast%3DDillon%26aufirst%3DE.%2BT.%26aulast%3DCagney%26aufirst%3DG.%26aulast%3DVan%2BMierlo%26aufirst%3DG.%26aulast%3DBaltissen%26aufirst%3DM.%2BP.%26aulast%3DVermeulen%26aufirst%3DM.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DFr%25C3%25B6hling%26aufirst%3DS.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26atitle%3DTargeted%2520Degradation%2520of%2520BRD9%2520Reverses%2520Oncogenic%2520Gene%2520Expression%2520in%2520Synovial%2520Sarcoma%26jtitle%3DeLife%26date%3D2018%26volume%3D7%26doi%3D10.7554%2FeLife.41305" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vartanian, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Côté, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duplessis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flynn, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiefer, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nasveschuk, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romero, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandy, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsui, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zawadzke, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnuson, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cochran, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stokoe, D.</span></span> <span> </span><span class="NLM_article-title">Inhibition of Bromodomain-Containing Protein 9 for the Prevention of Epigenetically-Defined Drug Resistance</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">3534</span>– <span class="NLM_lpage">3541</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2017.05.063</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1016%2Fj.bmcl.2017.05.063" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=28606761" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVSjtb%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=3534-3541&author=T.+D.+Crawfordauthor=S.+Vartanianauthor=A.+C%C3%B4t%C3%A9author=S.+Bellonauthor=M.+Duplessisauthor=E.+M.+Flynnauthor=M.+Hewittauthor=H.-R.+Huangauthor=J.+R.+Kieferauthor=J.+Murrayauthor=C.+G.+Nasveschukauthor=E.+Pardoauthor=F.+A.+Romeroauthor=P.+Sandyauthor=Y.+Tangauthor=A.+M.+Taylorauthor=V.+Tsuiauthor=J.+Wangauthor=S.+Wangauthor=L.+Zawadzkeauthor=B.+K.+Albrechtauthor=S.+R.+Magnusonauthor=A.+G.+Cochranauthor=D.+Stokoe&title=Inhibition+of+Bromodomain-Containing+Protein+9+for+the+Prevention+of+Epigenetically-Defined+Drug+Resistance&doi=10.1016%2Fj.bmcl.2017.05.063"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of bromodomain-containing protein 9 for the prevention of epigenetically-defined drug resistance</span></div><div class="casAuthors">Crawford, Terry D.; Vartanian, Steffan; Cote, Alexandre; Bellon, Steve; Duplessis, Martin; Flynn, E. Megan; Hewitt, Michael; Huang, Hon-Ren; Kiefer, James R.; Murray, Jeremy; Nasveschuk, Christopher G.; Pardo, Eneida; Romero, F. Anthony; Sandy, Peter; Tang, Yong; Taylor, Alexander M.; Tsui, Vickie; Wang, Jian; Wang, Shumei; Zawadzke, Laura; Albrecht, Brian K.; Magnuson, Steven R.; Cochran, Andrea G.; Stokoe, David</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">3534-3541</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Bromodomain-contg. protein 9 (BRD9), an epigenetic "reader" of acetylated lysines on post-translationally modified histone proteins, is upregulated in multiple cancer cell lines.  To assess the functional role of BRD9 in cancer cell lines, the authors identified a small-mol. inhibitor of the BRD9 bromodomain.  Starting from a pyrrolopyridone lead, the authors used structure-based drug design to identify a potent and highly selective in vitro tool compd. 11, (GNE-375).  While this compd. showed minimal effects in cell viability or gene expression assays, it showed remarkable potency in preventing the emergence of a drug tolerant population in EGFR mutant PC9 cells treated with EGFR inhibitors.  Such tolerance has been linked to an altered epigenetic state, and 11 decreased BRD9 binding to chromatin, and this was assocd. with decreased expression of ALDH1A1, a gene previously shown to be important in drug tolerance.  BRD9 inhibitors may therefore show utility in preventing epigenetically-defined drug resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWdETG4wZY9rVg90H21EOLACvtfcHk0ljSw5ajKF5fAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVSjtb%252FO&md5=f489e702cfb41cfce9d3d570bfde87ed</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2017.05.063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2017.05.063%26sid%3Dliteratum%253Aachs%26aulast%3DCrawford%26aufirst%3DT.%2BD.%26aulast%3DVartanian%26aufirst%3DS.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DA.%26aulast%3DBellon%26aufirst%3DS.%26aulast%3DDuplessis%26aufirst%3DM.%26aulast%3DFlynn%26aufirst%3DE.%2BM.%26aulast%3DHewitt%26aufirst%3DM.%26aulast%3DHuang%26aufirst%3DH.-R.%26aulast%3DKiefer%26aufirst%3DJ.%2BR.%26aulast%3DMurray%26aufirst%3DJ.%26aulast%3DNasveschuk%26aufirst%3DC.%2BG.%26aulast%3DPardo%26aufirst%3DE.%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DSandy%26aufirst%3DP.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DZawadzke%26aufirst%3DL.%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26aulast%3DMagnuson%26aufirst%3DS.%2BR.%26aulast%3DCochran%26aufirst%3DA.%2BG.%26aulast%3DStokoe%26aufirst%3DD.%26atitle%3DInhibition%2520of%2520Bromodomain-Containing%2520Protein%25209%2520for%2520the%2520Prevention%2520of%2520Epigenetically-Defined%2520Drug%2520Resistance%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D27%26spage%3D3534%26epage%3D3541%26doi%3D10.1016%2Fj.bmcl.2017.05.063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsui, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flynn, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Côteé, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Audia, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beresini, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burdick, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dakin, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duplessis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Good, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jayaram, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiefer, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nasveschuk, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poy, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romero, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zawadzke, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnuson, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cochran, A. G.</span></span> <span> </span><span class="NLM_article-title">Diving into the Water: Inducible Binding Conformations for BRD4, TAF1(2), BRD9, and CECR2 Bromodomains</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">5391</span>– <span class="NLM_lpage">5402</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00264</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00264" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC28XosFSnsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=5391-5402&author=T.+D.+Crawfordauthor=V.+Tsuiauthor=E.+M.+Flynnauthor=S.+Wangauthor=A.+M.+Taylorauthor=A.+C%C3%B4te%C3%A9author=J.+E.+Audiaauthor=M.+H.+Beresiniauthor=D.+J.+Burdickauthor=R.+Cummingsauthor=L.+A.+Dakinauthor=M.+Duplessisauthor=A.+C.+Goodauthor=M.+C.+Hewittauthor=H.-R.+Huangauthor=H.+Jayaramauthor=J.+R.+Kieferauthor=Y.+Jiangauthor=J.+Murrayauthor=C.+G.+Nasveschukauthor=E.+Pardoauthor=F.+Poyauthor=F.+A.+Romeroauthor=Y.+Tangauthor=J.+Wangauthor=Z.+Xuauthor=L.+E.+Zawadzkeauthor=X.+Zhuauthor=B.+K.+Albrechtauthor=S.+R.+Magnusonauthor=S.+Bellonauthor=A.+G.+Cochran&title=Diving+into+the+Water%3A+Inducible+Binding+Conformations+for+BRD4%2C+TAF1%282%29%2C+BRD9%2C+and+CECR2+Bromodomains&doi=10.1021%2Facs.jmedchem.6b00264"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Diving into the Water: Inducible Binding Conformations for BRD4, TAF1(2), BRD9, and CECR2 Bromodomains</span></div><div class="casAuthors">Crawford, Terry D.; Tsui, Vickie; Flynn, E. Megan; Wang, Shumei; Taylor, Alexander M.; Cote, Alexandre; Audia, James E.; Beresini, Maureen H.; Burdick, Daniel J.; Cummings, Richard; Dakin, Les A.; Duplessis, Martin; Good, Andrew C.; Hewitt, Michael C.; Huang, Hon-Ren; Jayaram, Hariharan; Kiefer, James R.; Jiang, Ying; Murray, Jeremy; Nasveschuk, Christopher G.; Pardo, Eneida; Poy, Florence; Romero, F. Anthony; Tang, Yong; Wang, Jian; Xu, Zhaowu; Zawadzke, Laura E.; Zhu, Xiaoyu; Albrecht, Brian K.; Magnuson, Steven R.; Bellon, Steve; Cochran, Andrea G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5391-5402</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The biol. role played by non-BET bromodomains remains poorly understood, and it is therefore imperative to identify potent and highly selective inhibitors to effectively explore the biol. of individual bromodomain proteins.  A ligand-efficient nonselective bromodomain inhibitor was identified from a 6-Me pyrrolopyridone fragment.  Small hydrophobic substituents replacing the N-Me group were designed directing toward the conserved bromodomain water pocket, and two distinct binding conformations were then obsd.  The substituents either directly displaced and rearranged the conserved solvent network, as in BRD4(1) and TAF1(2), or induced a narrow hydrophobic channel adjacent to the lipophilic shelf, as in BRD9 and CECR2.  The preference of distinct substituents for individual bromodomains provided selectivity handles useful for future lead optimization efforts for selective BRD9, CECR2, and TAF1(2) inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTLnuVCIKdHbVg90H21EOLACvtfcHk0lg8AjOdmLLkdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XosFSnsbs%253D&md5=bf92e94c5fd3ee36bd8441294ca85185</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00264&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00264%26sid%3Dliteratum%253Aachs%26aulast%3DCrawford%26aufirst%3DT.%2BD.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DFlynn%26aufirst%3DE.%2BM.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DC%25C3%25B4te%25C3%25A9%26aufirst%3DA.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26aulast%3DBeresini%26aufirst%3DM.%2BH.%26aulast%3DBurdick%26aufirst%3DD.%2BJ.%26aulast%3DCummings%26aufirst%3DR.%26aulast%3DDakin%26aufirst%3DL.%2BA.%26aulast%3DDuplessis%26aufirst%3DM.%26aulast%3DGood%26aufirst%3DA.%2BC.%26aulast%3DHewitt%26aufirst%3DM.%2BC.%26aulast%3DHuang%26aufirst%3DH.-R.%26aulast%3DJayaram%26aufirst%3DH.%26aulast%3DKiefer%26aufirst%3DJ.%2BR.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DMurray%26aufirst%3DJ.%26aulast%3DNasveschuk%26aufirst%3DC.%2BG.%26aulast%3DPardo%26aufirst%3DE.%26aulast%3DPoy%26aufirst%3DF.%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DZawadzke%26aufirst%3DL.%2BE.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26aulast%3DMagnuson%26aufirst%3DS.%2BR.%26aulast%3DBellon%26aufirst%3DS.%26aulast%3DCochran%26aufirst%3DA.%2BG.%26atitle%3DDiving%2520into%2520the%2520Water%253A%2520Inducible%2520Binding%2520Conformations%2520for%2520BRD4%252C%2520TAF1%25282%2529%252C%2520BRD9%252C%2520and%2520CECR2%2520Bromodomains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D5391%26epage%3D5402%26doi%3D10.1021%2Facs.jmedchem.6b00264" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Humphreys, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craggs, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayhow, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hussain, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renaux, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suckling, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span> <span> </span><span class="NLM_article-title">Discovery of a Potent, Cell Penetrant, and Selective P300/CBP-Associated Factor (PCAF)/General Control Nonderepressible 5 (GCN5) Bromodomain Chemical Probe</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">695</span>– <span class="NLM_lpage">709</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01566</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01566" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFCgs77N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=695-709&author=P.+G.+Humphreysauthor=P.+Bamboroughauthor=C.+Chungauthor=P.+D.+Craggsauthor=L.+Gordonauthor=P.+Grandiauthor=T.+G.+Hayhowauthor=J.+Hussainauthor=K.+L.+Jonesauthor=M.+Lindonauthor=A.-M.+Michonauthor=J.+F.+Renauxauthor=C.+J.+Sucklingauthor=D.+F.+Toughauthor=R.+K.+Prinjha&title=Discovery+of+a+Potent%2C+Cell+Penetrant%2C+and+Selective+P300%2FCBP-Associated+Factor+%28PCAF%29%2FGeneral+Control+Nonderepressible+5+%28GCN5%29+Bromodomain+Chemical+Probe&doi=10.1021%2Facs.jmedchem.6b01566"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Potent, Cell Penetrant, and Selective p300/CBP-Associated Factor (PCAF)/General Control Nonderepressible 5 (GCN5) Bromodomain Chemical Probe</span></div><div class="casAuthors">Humphreys, Philip G.; Bamborough, Paul; Chung, Chun-wa; Craggs, Peter D.; Gordon, Laurie; Grandi, Paola; Hayhow, Thomas G.; Hussain, Jameed; Jones, Katherine L.; Lindon, Matthew; Michon, Anne-Marie; Renaux, Jessica F.; Suckling, Colin J.; Tough, David F.; Prinjha, Rab K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">695-709</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">P300/CREB binding protein assocd. factor (PCAF/KAT2B) and general control non-derepressible 5 (GCN5/KAT2A) are multidomain proteins that have been implicated in retroviral infection, inflammation pathways and cancer development.  However, outside of viral replication, little is known about the dependence of these effects on the C-terminal bromodomain.  Herein, the authors report GSK4027 as a chem. probe for the PCAF/GCN5 bromodomain, together with GSK4028 as an enantiomeric neg. control.  The probe was optimized from a weakly potent, non-selective pyridazinone hit to deliver high potency for the PCAF/GCN5 bromodomain, high soly., cellular target engagement and ≥18,000-fold selectivity over the BET family, together with ≥70-fold selectivity over the wider bromodomain families.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBkLwjcv1aZLVg90H21EOLACvtfcHk0lg8AjOdmLLkdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFCgs77N&md5=fd4fdbe8827e26769fc17b09120c7826</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01566&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01566%26sid%3Dliteratum%253Aachs%26aulast%3DHumphreys%26aufirst%3DP.%2BG.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.%26aulast%3DCraggs%26aufirst%3DP.%2BD.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DHayhow%26aufirst%3DT.%2BG.%26aulast%3DHussain%26aufirst%3DJ.%26aulast%3DJones%26aufirst%3DK.%2BL.%26aulast%3DLindon%26aufirst%3DM.%26aulast%3DMichon%26aufirst%3DA.-M.%26aulast%3DRenaux%26aufirst%3DJ.%2BF.%26aulast%3DSuckling%26aufirst%3DC.%2BJ.%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26atitle%3DDiscovery%2520of%2520a%2520Potent%252C%2520Cell%2520Penetrant%252C%2520and%2520Selective%2520P300%252FCBP-Associated%2520Factor%2520%2528PCAF%2529%252FGeneral%2520Control%2520Nonderepressible%25205%2520%2528GCN5%2529%2520Bromodomain%2520Chemical%2520Probe%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D695%26epage%3D709%26doi%3D10.1021%2Facs.jmedchem.6b01566" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallego, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, M. P.</span></span> <span> </span><span class="NLM_article-title">Lipophilic Efficiency as an Important Metric in Drug Design</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">6401</span>– <span class="NLM_lpage">6420</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00077</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00077" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC1cXmsVahsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=6401-6420&author=T.+W.+Johnsonauthor=R.+A.+Gallegoauthor=M.+P.+Edwards&title=Lipophilic+Efficiency+as+an+Important+Metric+in+Drug+Design&doi=10.1021%2Facs.jmedchem.8b00077"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Lipophilic Efficiency as an Important Metric in Drug Design</span></div><div class="casAuthors">Johnson, Ted W.; Gallego, Rebecca A.; Edwards, Martin P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6401-6420</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Lipophilic efficiency (LipE) is an important metric that has been increasingly applied in drug discovery medicinal chem. lead optimization programs.  In this perspective, using literature drug discovery examples, we discuss the concept of rigorously applying LipE to guide medicinal chem. lead optimization toward drug candidates with potential for superior in vivo efficacy and safety, esp. when guided by physiochem. property-based optimization (PPBO).  Also highlighted are examples of small structural modifications such as addn. of single atoms, small functional groups, and cyclizations that produce large increases in LipE.  Understanding the factors that may contribute to LipE changes through anal. of ligand-protein crystal structures and using structure-based drug design (SBDD) to increase LipE by design is also discussed.  Herein we advocate for use of LipE anal. coupled with PPBO and SBDD as an efficient mechanism for drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXQ8DOcOj6GrVg90H21EOLACvtfcHk0lg8AjOdmLLkdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXmsVahsbg%253D&md5=29f38880914fa75c33be3a640ace4309</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00077&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00077%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DT.%2BW.%26aulast%3DGallego%26aufirst%3DR.%2BA.%26aulast%3DEdwards%26aufirst%3DM.%2BP.%26atitle%3DLipophilic%2520Efficiency%2520as%2520an%2520Important%2520Metric%2520in%2520Drug%2520Design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D6401%26epage%3D6420%26doi%3D10.1021%2Facs.jmedchem.8b00077" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span> <i>BROMOscan recombinant protein binding assays were
carried out at
DiscoveRx</i>; <a href="http://www.discoverx.com" class="extLink">http://www.discoverx.com</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+BROMOscan+recombinant+protein+binding+assays+were%0Acarried+out+at%0ADiscoveRx%3B+http%3A%2F%2Fwww.discoverx.com."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26jtitle%3DBROMOscan%2520recombinant%2520protein%2520binding%2520assays%2520were%250Acarried%2520out%2520at%250ADiscoveRx" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nittinger, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibbons, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eigenbrot, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maurer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiefer, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuglstatter, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortwine, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsui, V.</span></span> <span> </span><span class="NLM_article-title">Water Molecules in Protein–Ligand Interfaces. Evaluation of Software Tools and SAR Comparison</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">307</span>– <span class="NLM_lpage">330</span>, <span class="refDoi"> DOI: 10.1007/s10822-019-00187-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1007%2Fs10822-019-00187-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=30756207" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC1MXmsVylsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2019&pages=307-330&author=E.+Nittingerauthor=P.+Gibbonsauthor=C.+Eigenbrotauthor=D.+R.+Daviesauthor=B.+Maurerauthor=C.+L.+Yuauthor=J.+R.+Kieferauthor=A.+Kuglstatterauthor=J.+Murrayauthor=D.+F.+Ortwineauthor=Y.+Tangauthor=V.+Tsui&title=Water+Molecules+in+Protein%E2%80%93Ligand+Interfaces.+Evaluation+of+Software+Tools+and+SAR+Comparison&doi=10.1007%2Fs10822-019-00187-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Water molecules in protein-ligand interfaces. Evaluation of software tools and SAR comparison</span></div><div class="casAuthors">Nittinger, Eva; Gibbons, Paul; Eigenbrot, Charles; Davies, Doug R.; Maurer, Brigitte; Yu, Christine L.; Kiefer, James R.; Kuglstatter, Andreas; Murray, Jeremy; Ortwine, Daniel F.; Tang, Yong; Tsui, Vickie</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">307-330</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Abstr.: Targeting the interaction with or displacement of the 'right' water mol. can significantly increase inhibitor potency in structure-guided drug design.  Multiple computational approaches exist to predict which waters should be targeted for displacement to achieve the largest gain in potency.  However, the relative success of different methods remains underexplored.  Here, we present a comparison of the ability of five water prediction programs (3D-RISM, SZMAP, WaterFLAP, WaterRank, and WaterMap) to predict crystallog. water locations, calc. their binding free energies, and to relate differences in these energies to obsd. changes in potency.  The structural cohort included nine Bruton's Tyrosine Kinase (BTK) structures, and nine bromodomain structures.  Each program accurately predicted the locations of most crystallog. water mols.  However, the predicted binding free energies correlated poorly with the obsd. changes in inhibitor potency when solvent atoms were displaced by chem. changes in closely related compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwiAvnyBQBhbVg90H21EOLACvtfcHk0lgF2VErxDOzvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXmsVylsr8%253D&md5=4d44c3dbdddf732ea8030574f42c9ac1</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1007%2Fs10822-019-00187-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-019-00187-y%26sid%3Dliteratum%253Aachs%26aulast%3DNittinger%26aufirst%3DE.%26aulast%3DGibbons%26aufirst%3DP.%26aulast%3DEigenbrot%26aufirst%3DC.%26aulast%3DDavies%26aufirst%3DD.%2BR.%26aulast%3DMaurer%26aufirst%3DB.%26aulast%3DYu%26aufirst%3DC.%2BL.%26aulast%3DKiefer%26aufirst%3DJ.%2BR.%26aulast%3DKuglstatter%26aufirst%3DA.%26aulast%3DMurray%26aufirst%3DJ.%26aulast%3DOrtwine%26aufirst%3DD.%2BF.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DTsui%26aufirst%3DV.%26atitle%3DWater%2520Molecules%2520in%2520Protein%25E2%2580%2593Ligand%2520Interfaces.%2520Evaluation%2520of%2520Software%2520Tools%2520and%2520SAR%2520Comparison%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2019%26volume%3D33%26spage%3D307%26epage%3D330%26doi%3D10.1007%2Fs10822-019-00187-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiest, O.</span></span> <span> </span><span class="NLM_article-title">Protein Dynamics and Structural Waters in Bromodomains</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>, <span class="NLM_elocation-id">e0186570</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0186570</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1371%2Fjournal.pone.0186570" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=29077715" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFGms7bN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&author=X.+Zhangauthor=K.+Chenauthor=Y.-D.+Wuauthor=O.+Wiest&title=Protein+Dynamics+and+Structural+Waters+in+Bromodomains&doi=10.1371%2Fjournal.pone.0186570"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Protein dynamics and structural waters in bromodomains</span></div><div class="casAuthors">Zhang, Xiaoxiao; Chen, Kai; Wu, Yun-Dong; Wiest, Olaf</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">e0186570/1-e0186570/16</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Bromodomains are epigenetic readers of acetylated lysines that are integral parts of histone tails.  The 61 bromodomains in humans are structurally highly conserved but specifically bind to widely varying recognition motifs, suggesting that dynamic rather than static factors are responsible for recognition selectivity.  To test this hypothesis, the dynamics of the binding sites and structural water mols. of four bromodomains (ATAD2, BAZ2B, BRD2(1) and CREBBP) representing four different subtypes is studied with 1 μs MD simulations using the RSFF2 force field.  The different dynamics of the ZA-loops and BC-loops between the four bromodomains leads to distinct patterns for the opening and closing of the binding pocket.  This in turn dets. the structural and energetic properties of the structural waters in the binding pocket, suggesting that these waters are not only important for the recognition itself, as has been proposed previously, but also contribute to the selectivity of different bromodomains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdqzaoNSjS-7Vg90H21EOLACvtfcHk0lgF2VErxDOzvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFGms7bN&md5=3ed02a6025d67e5c0e8b511c3dc6756e</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0186570&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0186570%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DWu%26aufirst%3DY.-D.%26aulast%3DWiest%26aufirst%3DO.%26atitle%3DProtein%2520Dynamics%2520and%2520Structural%2520Waters%2520in%2520Bromodomains%26jtitle%3DPLoS%2520One%26date%3D2017%26volume%3D12%26doi%3D10.1371%2Fjournal.pone.0186570" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aldeghi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bodkin, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Essex, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biggin, P. C.</span></span> <span> </span><span class="NLM_article-title">Large-Scale Analysis of Water Stability in Bromodomain Binding Pockets with Grand Canonical Monte Carlo</span>. <i>Commun. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">12</span>, <span class="refDoi"> DOI: 10.1038/s42004-018-0019-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1038%2Fs42004-018-0019-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC1MXpvVyqt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2018&pages=1-12&author=M.+Aldeghiauthor=G.+A.+Rossauthor=M.+J.+Bodkinauthor=J.+W.+Essexauthor=S.+Knappauthor=P.+C.+Biggin&title=Large-Scale+Analysis+of+Water+Stability+in+Bromodomain+Binding+Pockets+with+Grand+Canonical+Monte+Carlo&doi=10.1038%2Fs42004-018-0019-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Large-scale analysis of water stability in bromodomain binding pockets with grand canonical Monte Carlo</span></div><div class="casAuthors">Aldeghi, Matteo; Ross, Gregory A.; Bodkin, Michael J.; Essex, Jonathan W.; Knapp, Stefan; Biggin, Philip C.</div><div class="citationInfo"><span class="NLM_cas:title">Communications Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-12</span>CODEN:
                <span class="NLM_cas:coden">CCOHCT</span>;
        ISSN:<span class="NLM_cas:issn">2399-3669</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Conserved water mols. are of interest in drug design, as displacement of such waters can lead to higher affinity ligands, and in some cases, contribute towards selectivity.  Bromodomains, small protein domains involved in the epigenetic regulation of gene transcription, display a network of four conserved water mols. in their binding pockets and have recently been the focus of intense medicinal chem. efforts.  Understanding why certain bromodomains have displaceable water mols. and others do not is extremely challenging, and it remains unclear which water mols. in a given bromodomain can be targeted for displacement.  Here we est. the stability of the conserved water mols. in 35 bromodomains via binding free energy calcns. using all-atom grand canonical Monte Carlo simulations.  Encouraging quant. agreement to the available exptl. evidence is found.  We thus discuss the expected ease of water displacement in different bromodomains and the implications for ligand selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7J7hnRA65t7Vg90H21EOLACvtfcHk0lgF2VErxDOzvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXpvVyqt7c%253D&md5=7af46c69fc53664ec719a178485457fe</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fs42004-018-0019-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs42004-018-0019-x%26sid%3Dliteratum%253Aachs%26aulast%3DAldeghi%26aufirst%3DM.%26aulast%3DRoss%26aufirst%3DG.%2BA.%26aulast%3DBodkin%26aufirst%3DM.%2BJ.%26aulast%3DEssex%26aufirst%3DJ.%2BW.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBiggin%26aufirst%3DP.%2BC.%26atitle%3DLarge-Scale%2520Analysis%2520of%2520Water%2520Stability%2520in%2520Bromodomain%2520Binding%2520Pockets%2520with%2520Grand%2520Canonical%2520Monte%2520Carlo%26jtitle%3DCommun.%2520Chem.%26date%3D2018%26volume%3D1%26spage%3D1%26epage%3D12%26doi%3D10.1038%2Fs42004-018-0019-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bridges, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craggs, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furze, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karamshi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Locke, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, S. C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pogány, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roa, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span> <span> </span><span class="NLM_article-title">A Qualified Success: Discovery of a New Series of ATAD2 Bromodomain Inhibitors with a Novel Binding Mode Using High-Throughput Screening and Hit Qualification</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">7506</span>– <span class="NLM_lpage">7525</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00673</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00673" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsFeqsL%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=7506-7525&author=P.+Bamboroughauthor=C.+Chungauthor=E.+H.+Demontauthor=A.+M.+Bridgesauthor=P.+D.+Craggsauthor=D.+P.+Dixonauthor=P.+Francisauthor=R.+C.+Furzeauthor=P.+Grandiauthor=E.+J.+Jonesauthor=B.+Karamshiauthor=K.+Lockeauthor=S.+C.+C.+Lucasauthor=A.-M.+Michonauthor=D.+J.+Mitchellauthor=P.+Pog%C3%A1nyauthor=R.+K.+Prinjhaauthor=C.+Rauauthor=A.+M.+Roaauthor=A.+D.+Robertsauthor=R.+J.+Sheppardauthor=R.+J.+Watson&title=A+Qualified+Success%3A+Discovery+of+a+New+Series+of+ATAD2+Bromodomain+Inhibitors+with+a+Novel+Binding+Mode+Using+High-Throughput+Screening+and+Hit+Qualification&doi=10.1021%2Facs.jmedchem.9b00673"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">A Qualified Success: Discovery of a New Series of ATAD2 Bromodomain Inhibitors with a Novel Binding Mode Using High-Throughput Screening and Hit Qualification</span></div><div class="casAuthors">Bamborough, Paul; Chung, Chun-wa; Demont, Emmanuel H.; Bridges, Angela M.; Craggs, Peter D.; Dixon, David P.; Francis, Peter; Furze, Rebecca C.; Grandi, Paola; Jones, Emma J.; Karamshi, Bhumika; Locke, Kelly; Lucas, Simon C. C.; Michon, Anne-Marie; Mitchell, Darren J.; Pogany, Peter; Prinjha, Rab K.; Rau, Christina; Roa, Ana Maria; Roberts, Andrew D.; Sheppard, Robert J.; Watson, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">7506-7525</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromodomain of ATAD2 has proved to be one of the least-tractable proteins within this target class.  Here, we describe the discovery of a new class of inhibitors by high-throughput screening and show how the difficulties encountered in establishing a screening triage capable of finding progressible hits were overcome by data-driven optimization.  Despite the prevalence of nonspecific hits and an exceptionally low progressible hit rate (0.001%), our optimized hit qualification strategy employing orthogonal biophys. methods enabled us to identify a single active series.  The compds. have a novel ATAD2 binding mode with noncanonical features including the displacement of all conserved water mols. within the active site and a halogen-bonding interaction.  In addn. to reporting this new series and preliminary structure-activity relationship, we demonstrate the value of diversity screening to complement the knowledge-based approach used in our previous ATAD2 work.  We also exemplify tactics that can increase the chance of success when seeking new chem. starting points for novel and less-tractable targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomDZyktXHrhbVg90H21EOLACvtfcHk0lhRcql9YEhfZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsFeqsL%252FJ&md5=8132ccb7f63c3300bd173c42cd2ac026</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00673%26sid%3Dliteratum%253Aachs%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DBridges%26aufirst%3DA.%2BM.%26aulast%3DCraggs%26aufirst%3DP.%2BD.%26aulast%3DDixon%26aufirst%3DD.%2BP.%26aulast%3DFrancis%26aufirst%3DP.%26aulast%3DFurze%26aufirst%3DR.%2BC.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DJones%26aufirst%3DE.%2BJ.%26aulast%3DKaramshi%26aufirst%3DB.%26aulast%3DLocke%26aufirst%3DK.%26aulast%3DLucas%26aufirst%3DS.%2BC.%2BC.%26aulast%3DMichon%26aufirst%3DA.-M.%26aulast%3DMitchell%26aufirst%3DD.%2BJ.%26aulast%3DPog%25C3%25A1ny%26aufirst%3DP.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRau%26aufirst%3DC.%26aulast%3DRoa%26aufirst%3DA.%2BM.%26aulast%3DRoberts%26aufirst%3DA.%2BD.%26aulast%3DSheppard%26aufirst%3DR.%2BJ.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26atitle%3DA%2520Qualified%2520Success%253A%2520Discovery%2520of%2520a%2520New%2520Series%2520of%2520ATAD2%2520Bromodomain%2520Inhibitors%2520with%2520a%2520Novel%2520Binding%2520Mode%2520Using%2520High-Throughput%2520Screening%2520and%2520Hit%2520Qualification%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D7506%26epage%3D7525%26doi%3D10.1021%2Facs.jmedchem.9b00673" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsui, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Audia, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burdick, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beresini, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Côté, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duplessis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flynn, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jayaram, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joshi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nasveschuk, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poy, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romero, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zawadzke, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnuson, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cochran, A. G.</span></span> <span> </span><span class="NLM_article-title">GNE-371, a Potent and Selective Chemical Probe for the Second Bromodomains of Human Transcription-Initiation-Factor TFIID Subunit 1 and Transcription-Initiation-Factor TFIID Subunit 1-Like</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">9301</span>– <span class="NLM_lpage">9315</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01225</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01225" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVKgtLjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=9301-9315&author=S.+Wangauthor=V.+Tsuiauthor=T.+D.+Crawfordauthor=J.+E.+Audiaauthor=D.+J.+Burdickauthor=M.+H.+Beresiniauthor=A.+C%C3%B4t%C3%A9author=R.+Cummingsauthor=M.+Duplessisauthor=E.+M.+Flynnauthor=M.+C.+Hewittauthor=H.-R.+Huangauthor=H.+Jayaramauthor=Y.+Jiangauthor=S.+Joshiauthor=J.+Murrayauthor=C.+G.+Nasveschukauthor=E.+Pardoauthor=F.+Poyauthor=F.+A.+Romeroauthor=Y.+Tangauthor=A.+M.+Taylorauthor=J.+Wangauthor=Z.+Xuauthor=L.+E.+Zawadzkeauthor=X.+Zhuauthor=B.+K.+Albrechtauthor=S.+R.+Magnusonauthor=S.+Bellonauthor=A.+G.+Cochran&title=GNE-371%2C+a+Potent+and+Selective+Chemical+Probe+for+the+Second+Bromodomains+of+Human+Transcription-Initiation-Factor+TFIID+Subunit+1+and+Transcription-Initiation-Factor+TFIID+Subunit+1-Like&doi=10.1021%2Facs.jmedchem.8b01225"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">GNE-371, a Potent and Selective Chemical Probe for the Second Bromodomains of Human Transcription-Initiation-Factor TFIID Subunit 1 and Transcription-Initiation-Factor TFIID Subunit 1-like</span></div><div class="casAuthors">Wang, Shumei; Tsui, Vickie; Crawford, Terry D.; Audia, James E.; Burdick, Daniel J.; Beresini, Maureen H.; Cote, Alexandre; Cummings, Richard; Duplessis, Martin; Flynn, E. Megan; Hewitt, Michael C.; Huang, Hon-Ren; Jayaram, Hariharan; Jiang, Ying; Joshi, Shivangi; Murray, Jeremy; Nasveschuk, Christopher G.; Pardo, Eneida; Poy, Florence; Romero, F. Anthony; Tang, Yong; Taylor, Alexander M.; Wang, Jian; Xu, Zhaowu; Zawadzke, Laura E.; Zhu, Xiaoyu; Albrecht, Brian K.; Magnuson, Steven R.; Bellon, Steve; Cochran, Andrea G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">9301-9315</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The biol. functions of the dual bromodomains of human transcription-initiation-factor TFIID subunit 1 (TAF1(1,2)) remain unknown, although TAF1 has been identified as a potential target for oncol. research.  Here, the authors describe the discovery of a potent and selective in vitro tool compd. for TAF1(2), starting from a previously reported lead.  A cocrystal structure of lead compd. 2 bound to TAF1(2) enabled structure-based design and structure-activity-relation studies that ultimately led to the authors' in vitro tool compd., 27 (GNE-371).  Compd. 27 binds TAF1(2) with an IC50 of 10 nM while maintaining excellent selectivity over other bromodomain-family members.  Compd. 27 is also active in a cellular-TAF1(2) target-engagement assay (IC50 = 38 nM) and exhibits antiproliferative synergy with the BET inhibitor JQ1, suggesting engagement of endogenous TAF1 by 27 and further supporting the use of 27 in mechanistic and target-validation studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-B5cO-w0BOrVg90H21EOLACvtfcHk0lhRcql9YEhfZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVKgtLjK&md5=6f13888f3d123f6c0119b047c44a11a7</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01225&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01225%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DS.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DCrawford%26aufirst%3DT.%2BD.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26aulast%3DBurdick%26aufirst%3DD.%2BJ.%26aulast%3DBeresini%26aufirst%3DM.%2BH.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DA.%26aulast%3DCummings%26aufirst%3DR.%26aulast%3DDuplessis%26aufirst%3DM.%26aulast%3DFlynn%26aufirst%3DE.%2BM.%26aulast%3DHewitt%26aufirst%3DM.%2BC.%26aulast%3DHuang%26aufirst%3DH.-R.%26aulast%3DJayaram%26aufirst%3DH.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DJoshi%26aufirst%3DS.%26aulast%3DMurray%26aufirst%3DJ.%26aulast%3DNasveschuk%26aufirst%3DC.%2BG.%26aulast%3DPardo%26aufirst%3DE.%26aulast%3DPoy%26aufirst%3DF.%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DZawadzke%26aufirst%3DL.%2BE.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26aulast%3DMagnuson%26aufirst%3DS.%2BR.%26aulast%3DBellon%26aufirst%3DS.%26aulast%3DCochran%26aufirst%3DA.%2BG.%26atitle%3DGNE-371%252C%2520a%2520Potent%2520and%2520Selective%2520Chemical%2520Probe%2520for%2520the%2520Second%2520Bromodomains%2520of%2520Human%2520Transcription-Initiation-Factor%2520TFIID%2520Subunit%25201%2520and%2520Transcription-Initiation-Factor%2520TFIID%2520Subunit%25201-Like%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D9301%26epage%3D9315%26doi%3D10.1021%2Facs.jmedchem.8b01225" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castex, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aldeghi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monteiro, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trzupek, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstenberger, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bountra, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willmann, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biggin, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schüle, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Günther, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span> <span> </span><span class="NLM_article-title">Selective Targeting of the BRG/PB1 Bromodomains Impairs Embryonic and Trophoblast Stem Cell Maintenance</span>. <i>Sci. Adv.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>1</i></span>, <span class="NLM_elocation-id">e1500723</span> <span class="refDoi"> DOI: 10.1126/sciadv.1500723</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1126%2Fsciadv.1500723" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=26702435" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC28XovFaktLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2015&author=O.+Fedorovauthor=J.+Castexauthor=C.+Tallantauthor=D.+R.+Owenauthor=S.+Martinauthor=M.+Aldeghiauthor=O.+Monteiroauthor=P.+Filippakopoulosauthor=S.+Picaudauthor=J.+D.+Trzupekauthor=B.+S.+Gerstenbergerauthor=C.+Bountraauthor=D.+Willmannauthor=C.+Wellsauthor=M.+Philpottauthor=C.+Rogersauthor=P.+C.+Bigginauthor=P.+E.+Brennanauthor=M.+E.+Bunnageauthor=R.+Sch%C3%BCleauthor=T.+G%C3%BCntherauthor=S.+Knappauthor=S.+M%C3%BCller&title=Selective+Targeting+of+the+BRG%2FPB1+Bromodomains+Impairs+Embryonic+and+Trophoblast+Stem+Cell+Maintenance&doi=10.1126%2Fsciadv.1500723"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Selective targeting of the BRG/PB1 bromodomains impairs embryonic and trophoblast stem cell maintenance</span></div><div class="casAuthors">Fedorov, Oleg; Castex, Josefina; Tallant, Cynthia; Owen, Dafydd R.; Martin, Sarah; Aldeghi, Matteo; Monteiro, Octovia; Filippakopoulos, Panagis; Picaud, Sarah; Trzupek, John D.; Gerstenberger, Brian S.; Bountra, Chas; Willmann, Dominica; Wells, Christopher; Philpott, Martin; Rogers, Catherine; Biggin, Philip C.; Brennan, Paul E.; Bunnage, Mark E.; Schule, Roland; Gunther, Thomas; Knapp, Stefan; Muller, Susanne</div><div class="citationInfo"><span class="NLM_cas:title">Science Advances</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">e1500723/1-e1500723/11</span>CODEN:
                <span class="NLM_cas:coden">SACDAF</span>;
        ISSN:<span class="NLM_cas:issn">2375-2548</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Mammalian SWI/SNF [also called Brg/Brahma-assocd. factors (BAFs)] are evolutionarily conserved chromatin remodeling complexes regulating gene transcription programs during development and stem cell differentiation.  BAF complexes contain an ATP (ATP)-driven remodeling enzyme (either BRG1 or BRM) and multiple protein interaction domains including bromodomains, an evolutionary conserved acetyl lysine-dependent protein interaction motif that recruits transcriptional regulators to acetylated chromatin.  We report a potent and cell active protein interaction inhibitor, PFI-3, that selectively binds to essential BAF bromodomains.  The high specificity of PFI-3 was achieved on the basis of a novel binding mode of a salicylic acid head group that led to the replacement of water mols. typically maintained in other bromodomain inhibitor complexes.  We show that exposure of embryonic stem cells to PFI-3 led to deprivation of stemness and deregulated lineage specification.  Furthermore, differentiation of trophoblast stem cells in the presence of PFI-3 was markedly enhanced.  The data present a key function of BAF bromodomains in stem cell maintenance and differentiation, introducing a novel versatile chem. probe for studies on acetylation-dependent cellular processes controlled by BAF remodeling complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiGyzl98dGOLVg90H21EOLACvtfcHk0lj8J4LBQ3Xf4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XovFaktLw%253D&md5=52757e667cc5afeb4c3a151fc8ef3442</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1126%2Fsciadv.1500723&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fsciadv.1500723%26sid%3Dliteratum%253Aachs%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DCastex%26aufirst%3DJ.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DAldeghi%26aufirst%3DM.%26aulast%3DMonteiro%26aufirst%3DO.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DTrzupek%26aufirst%3DJ.%2BD.%26aulast%3DGerstenberger%26aufirst%3DB.%2BS.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DWillmann%26aufirst%3DD.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DRogers%26aufirst%3DC.%26aulast%3DBiggin%26aufirst%3DP.%2BC.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DSch%25C3%25BCle%26aufirst%3DR.%26aulast%3DG%25C3%25BCnther%26aufirst%3DT.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26atitle%3DSelective%2520Targeting%2520of%2520the%2520BRG%252FPB1%2520Bromodomains%2520Impairs%2520Embryonic%2520and%2520Trophoblast%2520Stem%2520Cell%2520Maintenance%26jtitle%3DSci.%2520Adv.%26date%3D2015%26volume%3D1%26doi%3D10.1126%2Fsciadv.1500723" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gerstenberger, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trzupek, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedele, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parikh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samas, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Mahony, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berg, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallares, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torrey, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samardjiev, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nasipak, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Padilla-Benavides, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imbalzano, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nickerson, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen, D. R.</span></span> <span> </span><span class="NLM_article-title">Identification of a Chemical Probe for Family VIII Bromodomains Through Optimization of a Fragment Hit</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">4800</span>– <span class="NLM_lpage">4811</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00012</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00012" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC28Xmsl2rs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4800-4811&author=B.+S.+Gerstenbergerauthor=J.+D.+Trzupekauthor=C.+Tallantauthor=O.+Fedorovauthor=P.+Filippakopoulosauthor=P.+E.+Brennanauthor=V.+Fedeleauthor=S.+Martinauthor=S.+Picaudauthor=C.+Rogersauthor=M.+Parikhauthor=A.+Taylorauthor=B.+Samasauthor=A.+O%E2%80%99Mahonyauthor=E.+Bergauthor=G.+Pallaresauthor=A.+D.+Torreyauthor=D.+K.+Treiberauthor=I.+J.+Samardjievauthor=B.+T.+Nasipakauthor=T.+Padilla-Benavidesauthor=Q.+Wuauthor=A.+N.+Imbalzanoauthor=J.+A.+Nickersonauthor=M.+E.+Bunnageauthor=S.+M%C3%BCllerauthor=S.+Knappauthor=D.+R.+Owen&title=Identification+of+a+Chemical+Probe+for+Family+VIII+Bromodomains+Through+Optimization+of+a+Fragment+Hit&doi=10.1021%2Facs.jmedchem.6b00012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a Chemical Probe for Family VIII Bromodomains through Optimization of a Fragment Hit</span></div><div class="casAuthors">Gerstenberger, Brian S.; Trzupek, John D.; Tallant, Cynthia; Fedorov, Oleg; Filippakopoulos, Panagis; Brennan, Paul E.; Fedele, Vita; Martin, Sarah; Picaud, Sarah; Rogers, Catherine; Parikh, Mihir; Taylor, Alexandria; Samas, Brian; O'Mahony, Alison; Berg, Ellen; Pallares, Gabriel; Torrey, Adam V.; Treiber, Daniel K.; Samardjiev, Ivan J.; Nasipak, Brian T.; Padilla-Benavides, Teresita; Wu, Qiong; Imbalzano, Anthony N.; Nickerson, Jeffrey A.; Bunnage, Mark E.; Muller, Susanne; Knapp, Stefan; Owen, Dafydd R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4800-4811</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The acetyl post-translational modification of chromatin at selected histone lysine residues is interpreted by an acetyl-lysine specific interaction with bromodomain reader modules.  Here we report the discovery of the potent, acetyl-lysine-competitive, and cell active inhibitor PFI-3 that binds to certain family VIII bromodomains while displaying significant, broader bromodomain family selectivity.  The high specificity of PFI-3 for family VIII was achieved through a novel bromodomain binding mode of a phenolic headgroup that led to the unusual displacement of water mols. that are generally retained by most other bromodomain inhibitors reported to date.  The medicinal chem. program that led to PFI-3 from an initial fragment screening hit is described in detail, and addnl. analogs with differing family VIII bromodomain selectivity profiles are also reported.  We also describe the full pharmacol. characterization of PFI-3 as a chem. probe, along with phenotypic data on adipocyte and myoblast cell differentiation assays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZlrg_xjnZGbVg90H21EOLACvtfcHk0lj8J4LBQ3Xf4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xmsl2rs74%253D&md5=6cb73e76a19f7fb44153f8cbfeff9cf6</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00012%26sid%3Dliteratum%253Aachs%26aulast%3DGerstenberger%26aufirst%3DB.%2BS.%26aulast%3DTrzupek%26aufirst%3DJ.%2BD.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DFedele%26aufirst%3DV.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DRogers%26aufirst%3DC.%26aulast%3DParikh%26aufirst%3DM.%26aulast%3DTaylor%26aufirst%3DA.%26aulast%3DSamas%26aufirst%3DB.%26aulast%3DO%25E2%2580%2599Mahony%26aufirst%3DA.%26aulast%3DBerg%26aufirst%3DE.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DTorrey%26aufirst%3DA.%2BD.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DSamardjiev%26aufirst%3DI.%2BJ.%26aulast%3DNasipak%26aufirst%3DB.%2BT.%26aulast%3DPadilla-Benavides%26aufirst%3DT.%26aulast%3DWu%26aufirst%3DQ.%26aulast%3DImbalzano%26aufirst%3DA.%2BN.%26aulast%3DNickerson%26aufirst%3DJ.%2BA.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26atitle%3DIdentification%2520of%2520a%2520Chemical%2520Probe%2520for%2520Family%2520VIII%2520Bromodomains%2520Through%2520Optimization%2520of%2520a%2520Fragment%2520Hit%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D4800%26epage%3D4811%26doi%3D10.1021%2Facs.jmedchem.6b00012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sutherell, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monteiro, O. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yapp, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuchs, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siejka, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brenton, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ley, S. V.</span></span> <span> </span><span class="NLM_article-title">Identification and Development of 2,3-Dihydropyrrolo[1,2- a ]Quinazolin-5(1 H )-One Inhibitors Targeting Bromodomains Within the Switch/Sucrose Nonfermenting Complex</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">5095</span>– <span class="NLM_lpage">5101</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01997</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01997" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC28Xmsl2rt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=5095-5101&author=C.+L.+Sutherellauthor=C.+Tallantauthor=O.+P.+Monteiroauthor=C.+Yappauthor=J.+E.+Fuchsauthor=O.+Fedorovauthor=P.+Siejkaauthor=S.+M%C3%BCllerauthor=S.+Knappauthor=J.+D.+Brentonauthor=P.+E.+Brennanauthor=S.+V.+Ley&title=Identification+and+Development+of+2%2C3-Dihydropyrrolo%5B1%2C2-+a+%5DQuinazolin-5%281+H+%29-One+Inhibitors+Targeting+Bromodomains+Within+the+Switch%2FSucrose+Nonfermenting+Complex&doi=10.1021%2Facs.jmedchem.5b01997"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and Development of 2,3-Dihydropyrrolo[1,2-a]quinazolin-5(1H)-one Inhibitors Targeting Bromodomains within the Switch/Sucrose Nonfermenting Complex</span></div><div class="casAuthors">Sutherell, Charlotte L.; Tallant, Cynthia; Monteiro, Octovia P.; Yapp, Clarence; Fuchs, Julian E.; Fedorov, Oleg; Siejka, Paulina; Muller, Suzanne; Knapp, Stefan; Brenton, James D.; Brennan, Paul E.; Ley, Steven V.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5095-5101</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomain contg. proteins PB1, SMARCA4, and SMARCA2 are important components of SWI/SNF chromatin remodeling complexes.  We identified bromodomain inhibitors that target these proteins and display unusual binding modes involving water displacement from the KAc binding site.  The best compd. (I) binds the fifth bromodomain of PB1 with a KD of 124 nM, SMARCA2B and SMARCA4 with KD values of 262 and 417 nM, resp., and displays excellent selectivity over bromodomains other than PB1, SMARCA2, and SMARCA4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfnWlpl3TimLVg90H21EOLACvtfcHk0ljlFrr9EEkBng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xmsl2rt7s%253D&md5=a27afa2c97bd74462a09c9ec231f61ef</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01997&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01997%26sid%3Dliteratum%253Aachs%26aulast%3DSutherell%26aufirst%3DC.%2BL.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DMonteiro%26aufirst%3DO.%2BP.%26aulast%3DYapp%26aufirst%3DC.%26aulast%3DFuchs%26aufirst%3DJ.%2BE.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DSiejka%26aufirst%3DP.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBrenton%26aufirst%3DJ.%2BD.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DLey%26aufirst%3DS.%2BV.%26atitle%3DIdentification%2520and%2520Development%2520of%25202%252C3-Dihydropyrrolo%255B1%252C2-%2520a%2520%255DQuinazolin-5%25281%2520H%2520%2529-One%2520Inhibitors%2520Targeting%2520Bromodomains%2520Within%2520the%2520Switch%252FSucrose%2520Nonfermenting%2520Complex%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D5095%26epage%3D5101%26doi%3D10.1021%2Facs.jmedchem.5b01997" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="note"><p class="first last">Peptides substituted with propionylated and butyrylated and crotonylated lysines have also been shown to be capable of binding to human bromodomains, albeit with weak affinity; see</p></div><div class="NLM_citation" id="rightTab-cit47a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Vollmuth, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geyer, M.</span></span> <span> </span><span class="NLM_article-title">Interaction of Propionylated and Butyrylated Histone H3 Lysine Marks with Brd4 Bromodomains</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">6768</span>– <span class="NLM_lpage">6772</span>, <span class="refDoi"> DOI: 10.1002/anie.201002724</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1002%2Fanie.201002724" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFGnsbfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2010&pages=6768-6772&author=F.+Vollmuthauthor=M.+Geyer&title=Interaction+of+Propionylated+and+Butyrylated+Histone+H3+Lysine+Marks+with+Brd4+Bromodomains&doi=10.1002%2Fanie.201002724"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47aR"><div class="casContent"><span class="casTitleNuber">47a</span><div class="casTitle"><span class="NLM_cas:atitle">Interaction of Propionylated and Butyrylated Histone H3 Lysine Marks with Brd4 Bromodomains</span></div><div class="casAuthors">Vollmuth, Friederike; Geyer, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">6768-6772, S6768/1-S6768/9</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">In this study, we analyzed the interaction of propionyl- and butyryllysine residues on histone chains with Brd4 bromodomains and the structural basis of this interaction.  The binding affinities of both short-chain acylations were similar to or weaker than those obsd. for lysine acetylation.  The addnl. carbon atoms aligned between a highly conserved Pro1Phe motif, which suggests a general mode of bromodomain recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCelcpDi5QebVg90H21EOLACvtfcHk0ljlFrr9EEkBng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFGnsbfI&md5=ad48736d13205ddc66f4207a64ea3b8e</span></div><a href="/servlet/linkout?suffix=cit47a&amp;dbid=16384&amp;doi=10.1002%2Fanie.201002724&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201002724%26sid%3Dliteratum%253Aachs%26aulast%3DVollmuth%26aufirst%3DF.%26aulast%3DGeyer%26aufirst%3DM.%26atitle%3DInteraction%2520of%2520Propionylated%2520and%2520Butyrylated%2520Histone%2520H3%2520Lysine%2520Marks%2520with%2520Brd4%2520Bromodomains%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2010%26volume%3D49%26spage%3D6768%26epage%3D6772%26doi%3D10.1002%2Fanie.201002724" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit47b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Flynn, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, O. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poy, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oppikofer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellon, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cochran, A. G.</span></span> <span> </span><span class="NLM_article-title">A Subset of Human Bromodomains Recognizes Butyryllysine and Crotonyllysine Histone Peptide Modifications</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1801</span>– <span class="NLM_lpage">1814</span>, <span class="refDoi"> DOI: 10.1016/j.str.2015.08.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1016%2Fj.str.2015.08.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=26365797" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsV2ls7jN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2015&pages=1801-1814&author=E.+M.+Flynnauthor=O.+W.+Huangauthor=F.+Poyauthor=M.+Oppikoferauthor=S.+F.+Bellonauthor=Y.+Tangauthor=A.+G.+Cochran&title=A+Subset+of+Human+Bromodomains+Recognizes+Butyryllysine+and+Crotonyllysine+Histone+Peptide+Modifications&doi=10.1016%2Fj.str.2015.08.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47bR"><div class="casContent"><span class="casTitleNuber">47b</span><div class="casTitle"><span class="NLM_cas:atitle">A Subset of Human Bromodomains Recognizes Butyryllysine and Crotonyllysine Histone Peptide Modifications</span></div><div class="casAuthors">Flynn, E. Megan; Huang, Oscar W.; Poy, Florence; Oppikofer, Mariano; Bellon, Steve F.; Tang, Yong; Cochran, Andrea G.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1801-1814</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Bromodomains are epigenetic readers that are recruited to acetyllysine residues in histone tails.  Recent studies have identified non-acetyl acyllysine modifications, raising the possibility that these might be read by bromodomains.  Profiling the nearly complete human bromodomain family revealed that while most human bromodomains bind only the shorter acetyl and propionyl marks, the bromodomains of BRD9, CECR2, and the second bromodomain of TAF1 also recognize the longer butyryl mark.  In addn., the TAF1 s bromodomain is capable of binding crotonyl marks.  None of the human bromodomains tested binds succinyl marks.  We characterized structurally and biochem. the binding to different acyl groups, identifying bromodomain residues and structural attributes that contribute to specificity.  These studies demonstrate a surprising degree of plasticity in some human bromodomains but no single factor controlling specificity across the family.  The identification of candidate butyryl- and crotonyllysine readers supports the idea that these marks could have specific physiol. functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrv2tMgdAAflbVg90H21EOLACvtfcHk0lj5D2jzHAlDNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsV2ls7jN&md5=c3b216ef2596ea21196f531a5da5f1f3</span></div><a href="/servlet/linkout?suffix=cit47b&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2015.08.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2015.08.004%26sid%3Dliteratum%253Aachs%26aulast%3DFlynn%26aufirst%3DE.%2BM.%26aulast%3DHuang%26aufirst%3DO.%2BW.%26aulast%3DPoy%26aufirst%3DF.%26aulast%3DOppikofer%26aufirst%3DM.%26aulast%3DBellon%26aufirst%3DS.%2BF.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DCochran%26aufirst%3DA.%2BG.%26atitle%3DA%2520Subset%2520of%2520Human%2520Bromodomains%2520Recognizes%2520Butyryllysine%2520and%2520Crotonyllysine%2520Histone%2520Peptide%2520Modifications%26jtitle%3DStructure%26date%3D2015%26volume%3D23%26spage%3D1801%26epage%3D1814%26doi%3D10.1016%2Fj.str.2015.08.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Audia, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burdick, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bommi-Reddy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Côté, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duplessis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flynn, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jayaram, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joshi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiefer, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nasveschuk, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neiss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romero, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandy, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sims, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsui, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zawadzke, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnuson, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cochran, A. G.</span></span> <span> </span><span class="NLM_article-title">GNE-886: A Potent and Selective Inhibitor of the Cat Eye Syndrome Chromosome Region Candidate 2 Bromodomain (CECR2)</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">737</span>– <span class="NLM_lpage">741</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.7b00132</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.7b00132" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpt1Shsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=737-741&author=T.+D.+Crawfordauthor=J.+E.+Audiaauthor=S.+Bellonauthor=D.+J.+Burdickauthor=A.+Bommi-Reddyauthor=A.+C%C3%B4t%C3%A9author=R.+T.+Cummingsauthor=M.+Duplessisauthor=E.+M.+Flynnauthor=M.+Hewittauthor=H.-R.+Huangauthor=H.+Jayaramauthor=Y.+Jiangauthor=S.+Joshiauthor=J.+R.+Kieferauthor=J.+Murrayauthor=C.+G.+Nasveschukauthor=A.+Neissauthor=E.+Pardoauthor=F.+A.+Romeroauthor=P.+Sandyauthor=R.+J.+Simsauthor=Y.+Tangauthor=A.+M.+Taylorauthor=V.+Tsuiauthor=J.+Wangauthor=S.+Wangauthor=Y.+Wangauthor=Z.+Xuauthor=L.+Zawadzkeauthor=X.+Zhuauthor=B.+K.+Albrechtauthor=S.+R.+Magnusonauthor=A.+G.+Cochran&title=GNE-886%3A+A+Potent+and+Selective+Inhibitor+of+the+Cat+Eye+Syndrome+Chromosome+Region+Candidate+2+Bromodomain+%28CECR2%29&doi=10.1021%2Facsmedchemlett.7b00132"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">GNE-886: A Potent and Selective Inhibitor of the Cat Eye Syndrome Chromosome Region Candidate 2 Bromodomain (CECR2)</span></div><div class="casAuthors">Crawford, Terry D.; Audia, James E.; Bellon, Steve; Burdick, Daniel J.; Bommi-Reddy, Archana; Cote, Alexandre; Cummings, Richard T.; Duplessis, Martin; Flynn, E. Megan; Hewitt, Michael; Huang, Hon-Ren; Jayaram, Hariharan; Jiang, Ying; Joshi, Shivangi; Kiefer, James R.; Murray, Jeremy; Nasveschuk, Christopher G.; Neiss, Arianne; Pardo, Eneida; Romero, F. Anthony; Sandy, Peter; Sims, Robert J.; Tang, Yong; Taylor, Alexander M.; Tsui, Vickie; Wang, Jian; Wang, Shumei; Wang, Yongyun; Xu, Zhaowu; Zawadzke, Laura; Zhu, Xiaoqin; Albrecht, Brian K.; Magnuson, Steven R.; Cochran, Andrea G.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">737-741</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The biol. function of bromodomains, epigenetic readers of acetylated lysine residues, remains largely unknown.  Herein we report our efforts to discover a potent and selective inhibitor of the bromodomain of cat eye syndrome chromosome region candidate 2 (CECR2).  Screening of our internal medicinal chem. collection led to the identification of a pyrrolopyridone chem. lead, and subsequent structure-based drug design led to a potent and selective CECR2 bromodomain inhibitor (GNE-886) suitable for use as an in vitro tool compd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxtzb-gQeFAbVg90H21EOLACvtfcHk0lj5D2jzHAlDNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpt1Shsrw%253D&md5=f92e17923136477fc36edefd3bdd4998</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.7b00132&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.7b00132%26sid%3Dliteratum%253Aachs%26aulast%3DCrawford%26aufirst%3DT.%2BD.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26aulast%3DBellon%26aufirst%3DS.%26aulast%3DBurdick%26aufirst%3DD.%2BJ.%26aulast%3DBommi-Reddy%26aufirst%3DA.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DA.%26aulast%3DCummings%26aufirst%3DR.%2BT.%26aulast%3DDuplessis%26aufirst%3DM.%26aulast%3DFlynn%26aufirst%3DE.%2BM.%26aulast%3DHewitt%26aufirst%3DM.%26aulast%3DHuang%26aufirst%3DH.-R.%26aulast%3DJayaram%26aufirst%3DH.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DJoshi%26aufirst%3DS.%26aulast%3DKiefer%26aufirst%3DJ.%2BR.%26aulast%3DMurray%26aufirst%3DJ.%26aulast%3DNasveschuk%26aufirst%3DC.%2BG.%26aulast%3DNeiss%26aufirst%3DA.%26aulast%3DPardo%26aufirst%3DE.%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DSandy%26aufirst%3DP.%26aulast%3DSims%26aufirst%3DR.%2BJ.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DZawadzke%26aufirst%3DL.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26aulast%3DMagnuson%26aufirst%3DS.%2BR.%26aulast%3DCochran%26aufirst%3DA.%2BG.%26atitle%3DGNE-886%253A%2520A%2520Potent%2520and%2520Selective%2520Inhibitor%2520of%2520the%2520Cat%2520Eye%2520Syndrome%2520Chromosome%2520Region%2520Candidate%25202%2520Bromodomain%2520%2528CECR2%2529%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D8%26spage%3D737%26epage%3D741%26doi%3D10.1021%2Facsmedchemlett.7b00132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Readshaw, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollerton, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Upton, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lynne, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Besley, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boughtflower, B. J.</span></span> <span> </span><span class="NLM_article-title">Use of Calculated Physicochemical Properties to Enhance Quantitative Response When Using Charged Aerosol Detection</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>89</i></span>,  <span class="NLM_fpage">1772</span>– <span class="NLM_lpage">1777</span>, <span class="refDoi"> DOI: 10.1021/acs.analchem.6b04060</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.analchem.6b04060" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFams7%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=2017&pages=1772-1777&author=M.+W.+Robinsonauthor=A.+P.+Hillauthor=S.+A.+Readshawauthor=J.+C.+Hollertonauthor=R.+J.+Uptonauthor=S.+M.+Lynneauthor=S.+C.+Besleyauthor=B.+J.+Boughtflower&title=Use+of+Calculated+Physicochemical+Properties+to+Enhance+Quantitative+Response+When+Using+Charged+Aerosol+Detection&doi=10.1021%2Facs.analchem.6b04060"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Use of Calculated Physicochemical Properties to Enhance Quantitative Response When Using Charged Aerosol Detection</span></div><div class="casAuthors">Robinson, Max W.; Hill, Alan P.; Readshaw, Simon A.; Hollerton, John C.; Upton, Richard J.; Lynn, Sean M.; Besley, Steve C.; Boughtflower, Bob J.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1772-1777</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Universal quant. detection without the need for analyte ref. stds. would offer substantial benefits in many areas of anal. science.  The quant. capability of high performance liq. chromatog. (HPLC) with charged aerosol detection (CAD) was investigated for 50 compds. with a wide range of phys. and chem. properties.  It is widely believed that CAD is a mass detector.  Quantification of the 50 compds. using a generic calibrant and mass calibration achieved an av. error of 11.4% relative to 1H NMR.  Correction factors are proposed that est. the relative surface area of particles in the detector, taking into account the effects of the d. and charge of analytes.  Performing these corrections and quantifying with surface area calibration, rather than mass, shows considerably improved linearity and uniformity of detection, reducing the av. error relative to 1H NMR to 7.1%.  The accuracy of CAD quantification was most significantly improved for highly dense compds., with traditional mass calibration showing an av. error of 34.7% and the newly proposed surface area calibration showing an av. error of 5.8%.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxWCD0M2yQRLVg90H21EOLACvtfcHk0lj5D2jzHAlDNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFams7%252FK&md5=46c11b7527157b76f1fbc8eaf5cef75a</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Facs.analchem.6b04060&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.analchem.6b04060%26sid%3Dliteratum%253Aachs%26aulast%3DRobinson%26aufirst%3DM.%2BW.%26aulast%3DHill%26aufirst%3DA.%2BP.%26aulast%3DReadshaw%26aufirst%3DS.%2BA.%26aulast%3DHollerton%26aufirst%3DJ.%2BC.%26aulast%3DUpton%26aufirst%3DR.%2BJ.%26aulast%3DLynne%26aufirst%3DS.%2BM.%26aulast%3DBesley%26aufirst%3DS.%2BC.%26aulast%3DBoughtflower%26aufirst%3DB.%2BJ.%26atitle%3DUse%2520of%2520Calculated%2520Physicochemical%2520Properties%2520to%2520Enhance%2520Quantitative%2520Response%2520When%2520Using%2520Charged%2520Aerosol%2520Detection%26jtitle%3DAnal.%2520Chem.%26date%3D2017%26volume%3D89%26spage%3D1772%26epage%3D1777%26doi%3D10.1021%2Facs.analchem.6b04060" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mateus, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wayne, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almqvist, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Axelsson, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seashore-Ludlow, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treyer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lundbäck, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hann, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Artursson, P.</span></span> <span> </span><span class="NLM_article-title">Prediction of Intracellular Exposure Bridges the Gap between Target- and Cell-Based Drug Discovery</span>. <i>Proc. Natl. Acad. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">E6231</span>– <span class="NLM_lpage">E6239</span>, <span class="refDoi"> DOI: 10.1073/pnas.1701848114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1073%2Fpnas.1701848114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=28701380" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFKmsr%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=E6231-E6239&author=A.+Mateusauthor=L.+J.+Gordonauthor=G.+J.+Wayneauthor=H.+Almqvistauthor=H.+Axelssonauthor=B.+Seashore-Ludlowauthor=A.+Treyerauthor=P.+Matssonauthor=T.+Lundb%C3%A4ckauthor=A.+Westauthor=M.+M.+Hannauthor=P.+Artursson&title=Prediction+of+Intracellular+Exposure+Bridges+the+Gap+between+Target-+and+Cell-Based+Drug+Discovery&doi=10.1073%2Fpnas.1701848114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of intracellular exposure bridges the gap between target- and cell-based drug discovery</span></div><div class="casAuthors">Mateus, Andre; Gordon, Laurie J.; Wayne, Gareth J.; Almqvist, Helena; Axelsson, Hanna; Seashore-Ludlow, Brinton; Treyer, Andrea; Matsson, Paer; Lundbaeck, Thomas; West, Andy; Hann, Michael M.; Artursson, Per</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">E6231-E6239</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Inadequate target exposure is a major cause of high attrition in drug discovery.  Here, we show that a label-free method for quantifying the intracellular bioavailability (Fic) of drug mols. predicts drug access to intracellular targets and hence, pharmacol. effect.  We detd. Fic in multiple cellular assays and cell types representing different targets from a no. of therapeutic areas, including cancer, inflammation, and dementia.  Both cytosolic targets and targets localized in subcellular compartments were investigated.  Fic gives insights on membrane-permeable compds. in terms of cellular potency and intracellular target engagement, compared with biochem. potency measurements alone.  Knowledge of the amt. of drug that is locally available to bind intracellular targets provides a powerful tool for compd. selection in early drug discovery. localization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqo1IEI-YIPNbVg90H21EOLACvtfcHk0ljDHxxZiMdfLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFKmsr%252FK&md5=ace62bf5bda6cfd214bac4b7c5bf9cf1</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1701848114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1701848114%26sid%3Dliteratum%253Aachs%26aulast%3DMateus%26aufirst%3DA.%26aulast%3DGordon%26aufirst%3DL.%2BJ.%26aulast%3DWayne%26aufirst%3DG.%2BJ.%26aulast%3DAlmqvist%26aufirst%3DH.%26aulast%3DAxelsson%26aufirst%3DH.%26aulast%3DSeashore-Ludlow%26aufirst%3DB.%26aulast%3DTreyer%26aufirst%3DA.%26aulast%3DMatsson%26aufirst%3DP.%26aulast%3DLundb%25C3%25A4ck%26aufirst%3DT.%26aulast%3DWest%26aufirst%3DA.%26aulast%3DHann%26aufirst%3DM.%2BM.%26aulast%3DArtursson%26aufirst%3DP.%26atitle%3DPrediction%2520of%2520Intracellular%2520Exposure%2520Bridges%2520the%2520Gap%2520between%2520Target-%2520and%2520Cell-Based%2520Drug%2520Discovery%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%26date%3D2017%26volume%3D114%26spage%3DE6231%26epage%3DE6239%26doi%3D10.1073%2Fpnas.1701848114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Artursson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hann, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leavens, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mateus, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Readshaw, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valko, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wayne, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, A.</span></span> <span> </span><span class="NLM_article-title">Direct Measurement of Intracellular Compound Concentration by RapidFire Mass Spectrometry Offers Insights into Cell Permeability</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">156</span>– <span class="NLM_lpage">164</span>, <span class="refDoi"> DOI: 10.1177/1087057115604141</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1177%2F1087057115604141" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=26336900" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=156-164&author=L.+J.+Gordonauthor=M.+Allenauthor=P.+Arturssonauthor=M.+M.+Hannauthor=B.+J.+Leavensauthor=A.+Mateusauthor=S.+Readshawauthor=K.+Valkoauthor=G.+J.+Wayneauthor=A.+West&title=Direct+Measurement+of+Intracellular+Compound+Concentration+by+RapidFire+Mass+Spectrometry+Offers+Insights+into+Cell+Permeability&doi=10.1177%2F1087057115604141"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1177%2F1087057115604141&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057115604141%26sid%3Dliteratum%253Aachs%26aulast%3DGordon%26aufirst%3DL.%2BJ.%26aulast%3DAllen%26aufirst%3DM.%26aulast%3DArtursson%26aufirst%3DP.%26aulast%3DHann%26aufirst%3DM.%2BM.%26aulast%3DLeavens%26aufirst%3DB.%2BJ.%26aulast%3DMateus%26aufirst%3DA.%26aulast%3DReadshaw%26aufirst%3DS.%26aulast%3DValko%26aufirst%3DK.%26aulast%3DWayne%26aufirst%3DG.%2BJ.%26aulast%3DWest%26aufirst%3DA.%26atitle%3DDirect%2520Measurement%2520of%2520Intracellular%2520Compound%2520Concentration%2520by%2520RapidFire%2520Mass%2520Spectrometry%2520Offers%2520Insights%2520into%2520Cell%2520Permeability%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2015%26volume%3D21%26spage%3D156%26epage%3D164%26doi%3D10.1177%2F1087057115604141" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leonards, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dovey, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monteiro, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujisawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingard, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikbin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guetzoyan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ingham, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ley, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samsonova, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingras, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwaller, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vassiliou, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span> <span> </span><span class="NLM_article-title">Promiscuous Targeting of Bromodomains by Bromosporine Identifies BET Proteins as Master Regulators of Primary Transcription Response in Leukemia</span>. <i>Sci. Adv.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">e1600760</span>– <span class="NLM_lpage">e1600760</span>, <span class="refDoi"> DOI: 10.1126/sciadv.1600760</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1126%2Fsciadv.1600760" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=27757418" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC1cXkvFyjtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2016&pages=e1600760-e1600760&author=S.+Picaudauthor=K.+Leonardsauthor=J.+P.+Lambertauthor=O.+Doveyauthor=C.+Wellsauthor=O.+Fedorovauthor=O.+Monteiroauthor=T.+Fujisawaauthor=C.+Y.+Wangauthor=H.+Lingardauthor=C.+Tallantauthor=N.+Nikbinauthor=L.+Guetzoyanauthor=R.+Inghamauthor=S.+V.+Leyauthor=P.+Brennanauthor=S.+Mullerauthor=A.+Samsonovaauthor=A.+C.+Gingrasauthor=J.+Schwallerauthor=G.+Vassiliouauthor=S.+Knappauthor=P.+Filippakopoulos&title=Promiscuous+Targeting+of+Bromodomains+by+Bromosporine+Identifies+BET+Proteins+as+Master+Regulators+of+Primary+Transcription+Response+in+Leukemia&doi=10.1126%2Fsciadv.1600760"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Promiscuous targeting of bromodomains by bromosporine identifies BET proteins as master regulators of primary transcription response in leukemia</span></div><div class="casAuthors">Picaud, Sarah; Leonards, Katharina; Lambert, Jean-Philippe; Dovey, Oliver; Wells, Christopher; Fedorov, Oleg; Monteiro, Octovia; Fujisawa, Takao; Wang, Chen-Yi; Lingard, Hannah; Tallant, Cynthia; Nikbin, Nikzad; Guetzoyan, Lucie; Ingham, Richard; Ley, Steven V.; Brennan, Paul; Muller, Susanne; Samsonova, Anastasia; Gingras, Anne-Claude; Schwaller, Juerg; Vassiliou, George; Knapp, Stefan; Filippakopoulos, Panagis</div><div class="citationInfo"><span class="NLM_cas:title">Science Advances</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">e1600760/1-e1600760/16</span>CODEN:
                <span class="NLM_cas:coden">SACDAF</span>;
        ISSN:<span class="NLM_cas:issn">2375-2548</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) have emerged as compelling targets for cancer therapy.  The development of selective and potent BET (bromo and extra-terminal) inhibitors and their significant activity in diverse tumor models have rapidly translated into clin. studies and have motivated drug development efforts targeting non-BET BRDs.  However, the complexmultidomain/subunit architecture of BRD protein complexes complicates predictions of the consequences of their pharmacol. targeting.  To address this issue, we developed a promiscuous BRD inhibitor [bromosporine (BSP)] that broadly targets BRDs (including BETs) with nanomolar affinity, creating a tool for the identification of cellular processes and diseases where BRDs have a regulatory function.  As a proof of principle, we studied the effects of BSP on leukemic cell lines known to be sensitive to BET inhibition and found, as expected, strong antiproliferative activity.  Comparison of the modulation of transcriptional profiles by BSP after a short exposure to the inhibitor resulted in a BET inhibitor signature but no significant addnl. changes in transcription that could account for inhibition of other BRDs.  Thus, nonselective targeting of BRDs identified BETs, but not other BRDs, as master regulators of contextdependent primary transcription response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr28artLYYsqrVg90H21EOLACvtfcHk0lj6gtf_E709RQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXkvFyjtbw%253D&md5=3060e8fd66c657f70e0ec3acea273057</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1126%2Fsciadv.1600760&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fsciadv.1600760%26sid%3Dliteratum%253Aachs%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DLeonards%26aufirst%3DK.%26aulast%3DLambert%26aufirst%3DJ.%2BP.%26aulast%3DDovey%26aufirst%3DO.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMonteiro%26aufirst%3DO.%26aulast%3DFujisawa%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DC.%2BY.%26aulast%3DLingard%26aufirst%3DH.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DNikbin%26aufirst%3DN.%26aulast%3DGuetzoyan%26aufirst%3DL.%26aulast%3DIngham%26aufirst%3DR.%26aulast%3DLey%26aufirst%3DS.%2BV.%26aulast%3DBrennan%26aufirst%3DP.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DSamsonova%26aufirst%3DA.%26aulast%3DGingras%26aufirst%3DA.%2BC.%26aulast%3DSchwaller%26aufirst%3DJ.%26aulast%3DVassiliou%26aufirst%3DG.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26atitle%3DPromiscuous%2520Targeting%2520of%2520Bromodomains%2520by%2520Bromosporine%2520Identifies%2520BET%2520Proteins%2520as%2520Master%2520Regulators%2520of%2520Primary%2520Transcription%2520Response%2520in%2520Leukemia%26jtitle%3DSci.%2520Adv.%26date%3D2016%26volume%3D2%26spage%3De1600760%26epage%3De1600760%26doi%3D10.1126%2Fsciadv.1600760" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tyler, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vappiani, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cañeque, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, E. Y. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilan, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, Y.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hienzsch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rutkowska, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lugo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gregory, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramirez Molina, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garton, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wellaway, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacPherson, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Figueiredo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stolzenburg, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">House, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, S.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hawkins, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewes, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriquez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, M. A.</span></span> <span> </span><span class="NLM_article-title">Click Chemistry Enables Preclinical Evaluation of Targeted Epigenetic Therapies</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>356</i></span>,  <span class="NLM_fpage">1397</span>– <span class="NLM_lpage">1401</span>, <span class="refDoi"> DOI: 10.1126/science.aal2066</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=10.1126%2Fscience.aal2066" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=28619718" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVKgsb%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=356&publication_year=2017&pages=1397-1401&author=D.+S.+Tylerauthor=J.+Vappianiauthor=T.+Ca%C3%B1equeauthor=E.+Y.+N.+Lamauthor=A.+Wardauthor=O.+Gilanauthor=Y.-C.+Chanauthor=A.+Hienzschauthor=A.+Rutkowskaauthor=T.+Wernerauthor=A.+J.+Wagnerauthor=D.+Lugoauthor=R.+Gregoryauthor=C.+Ramirez+Molinaauthor=N.+Gartonauthor=C.+R.+Wellawayauthor=S.+Jacksonauthor=L.+MacPhersonauthor=M.+Figueiredoauthor=S.+Stolzenburgauthor=C.+C.+Bellauthor=C.+Houseauthor=S.-J.+Dawsonauthor=E.+D.+Hawkinsauthor=G.+Drewesauthor=R.+K.+Prinjhaauthor=R.+Rodriquezauthor=P.+Grandiauthor=M.+A.+Dawson&title=Click+Chemistry+Enables+Preclinical+Evaluation+of+Targeted+Epigenetic+Therapies&doi=10.1126%2Fscience.aal2066"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Click chemistry enables preclinical evaluation of targeted epigenetic therapies</span></div><div class="casAuthors">Tyler, Dean S.; Vappiani, Johanna; Caneque, Tatiana; Lam, Enid Y. N.; Ward, Aoife; Gilan, Omer; Chan, Yih-Chih; Hienzsch, Antje; Rutkowska, Anna; Werner, Thilo; Wagner, Anne J.; Lugo, Dave; Gregory, Richard; Ramirez Molina, Cesar; Garton, Neil; Wellaway, Christopher R.; Jackson, Susan; MacPherson, Laura; Figueiredo, Margarida; Stolzenburg, Sabine; Bell, Charles C.; House, Colin; Dawson, Sarah-Jane; Hawkins, Edwin D.; Drewes, Gerard; Prinjha, Rab K.; Rodriguez, Raphael; Grandi, Paola; Dawson, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">356</span>
        (<span class="NLM_cas:issue">6345</span>),
    <span class="NLM_cas:pages">1397-1401</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The success of new therapies hinges on our ability to understand their mol. and cellular mechanisms of action.  We modified BET bromodomain inhibitors, an epigenetic-based therapy, to create functionally conserved compds. that are amenable to click chem. and can be used as mol. probes in vitro and in vivo.  We used click proteomics and click sequencing to explore the gene regulatory function of BRD4 (bromodomain contg. protein 4) and the transcriptional changes induced by BET inhibitors.  In our studies of mouse models of acute leukemia, we used high-resoln. microscopy and flow cytometry to highlight the heterogeneity of drug activity within tumor cells located in different tissue compartments.  We also demonstrate the differential distribution and effects of BET inhibitors in normal and malignant cells in vivo.  This study provides a potential framework for the preclin. assessment of a wide range of drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqf_08jpo_TP7Vg90H21EOLACvtfcHk0lj6gtf_E709RQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVKgsb%252FN&md5=a6b837fb92c5fa3818b8eecd8274657d</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1126%2Fscience.aal2066&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aal2066%26sid%3Dliteratum%253Aachs%26aulast%3DTyler%26aufirst%3DD.%2BS.%26aulast%3DVappiani%26aufirst%3DJ.%26aulast%3DCa%25C3%25B1eque%26aufirst%3DT.%26aulast%3DLam%26aufirst%3DE.%2BY.%2BN.%26aulast%3DWard%26aufirst%3DA.%26aulast%3DGilan%26aufirst%3DO.%26aulast%3DChan%26aufirst%3DY.-C.%26aulast%3DHienzsch%26aufirst%3DA.%26aulast%3DRutkowska%26aufirst%3DA.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DWagner%26aufirst%3DA.%2BJ.%26aulast%3DLugo%26aufirst%3DD.%26aulast%3DGregory%26aufirst%3DR.%26aulast%3DRamirez%2BMolina%26aufirst%3DC.%26aulast%3DGarton%26aufirst%3DN.%26aulast%3DWellaway%26aufirst%3DC.%2BR.%26aulast%3DJackson%26aufirst%3DS.%26aulast%3DMacPherson%26aufirst%3DL.%26aulast%3DFigueiredo%26aufirst%3DM.%26aulast%3DStolzenburg%26aufirst%3DS.%26aulast%3DBell%26aufirst%3DC.%2BC.%26aulast%3DHouse%26aufirst%3DC.%26aulast%3DDawson%26aufirst%3DS.-J.%26aulast%3DHawkins%26aufirst%3DE.%2BD.%26aulast%3DDrewes%26aufirst%3DG.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRodriquez%26aufirst%3DR.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DDawson%26aufirst%3DM.%2BA.%26atitle%3DClick%2520Chemistry%2520Enables%2520Preclinical%2520Evaluation%2520of%2520Targeted%2520Epigenetic%2520Therapies%26jtitle%3DScience%26date%3D2017%26volume%3D356%26spage%3D1397%26epage%3D1401%26doi%3D10.1126%2Fscience.aal2066" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gosmini, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, V. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toum, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brusq, J.-M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krysa, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirguet, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riou-Eymard, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boursier, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trottet, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutler, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaur, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schilling, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soden, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicodème, E.</span></span> <span> </span><span class="NLM_article-title">The Discovery of I-BET726 (GSK1324726A), a Potent Tetrahydroquinoline ApoA1 Up-Regulator and Selective BET Bromodomain Inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">8111</span>– <span class="NLM_lpage">8131</span>, <span class="refDoi"> DOI: 10.1021/jm5010539</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5010539" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1Wlur7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=8111-8131&author=R.+Gosminiauthor=V.+L.+Nguyenauthor=J.+Toumauthor=C.+Simonauthor=J.-M.+G.+Brusqauthor=G.+Krysaauthor=O.+Mirguetauthor=A.+M.+Riou-Eymardauthor=E.+V.+Boursierauthor=L.+Trottetauthor=P.+Bamboroughauthor=H.+Clarkauthor=C.+Chungauthor=L.+Cutlerauthor=E.+H.+Demontauthor=R.+Kaurauthor=A.+J.+Lewisauthor=M.+B.+Schillingauthor=P.+E.+Sodenauthor=S.+Taylorauthor=A.+L.+Walkerauthor=M.+D.+Walkerauthor=R.+K.+Prinjhaauthor=E.+Nicod%C3%A8me&title=The+Discovery+of+I-BET726+%28GSK1324726A%29%2C+a+Potent+Tetrahydroquinoline+ApoA1+Up-Regulator+and+Selective+BET+Bromodomain+Inhibitor&doi=10.1021%2Fjm5010539"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">The Discovery of I-BET726 (GSK1324726A), a Potent Tetrahydroquinoline ApoA1 Up-Regulator and Selective BET Bromodomain Inhibitor</span></div><div class="casAuthors">Gosmini, Romain; Nguyen, Van Loc; Toum, Jerome; Simon, Christophe; Brusq, Jean-Marie G.; Krysa, Gael; Mirguet, Olivier; Riou-Eymard, Alizon M.; Boursier, Eric V.; Trottet, Lionel; Bamborough, Paul; Clark, Hugh; Chung, Chun-wa; Cutler, Leanne; Demont, Emmanuel H.; Kaur, Rejbinder; Lewis, Antonia J.; Schilling, Mark B.; Soden, Peter E.; Taylor, Simon; Walker, Ann L.; Walker, Matthew D.; Prinjha, Rab K.; Nicodeme, Edwige</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8111-8131</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Through their function as epigenetic readers of the histone code, the BET family of bromodomain-contg. proteins regulate expression of multiple genes of therapeutic relevance, including those involved in tumor cell growth and inflammation.  BET bromodomain inhibitors have profound antiproliferative and anti-inflammatory effects which translate into efficacy in oncol. and inflammation models, and the first compds. have now progressed into clin. trials.  The exciting biol. of the BETs has led to great interest in the discovery of novel inhibitor classes.  Here we describe the identification of a novel tetrahydroquinoline series through up-regulation of apolipoprotein A1 and the optimization into potent compds. active in murine models of septic shock and neuroblastoma.  At the mol. level, these effects are produced by inhibition of BET bromodomains.  X-ray crystallog. reveals the interactions explaining the structure-activity relationships of binding.  The resulting lead mol., I-BET726, represents a new, potent, and selective class of tetrahydroquinoline-based BET inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5OdMn_TjoFLVg90H21EOLACvtfcHk0lhm2LdFSbMusw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1Wlur7J&md5=b1c9c9cae73c41252cf73908d561bd9b</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Fjm5010539&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5010539%26sid%3Dliteratum%253Aachs%26aulast%3DGosmini%26aufirst%3DR.%26aulast%3DNguyen%26aufirst%3DV.%2BL.%26aulast%3DToum%26aufirst%3DJ.%26aulast%3DSimon%26aufirst%3DC.%26aulast%3DBrusq%26aufirst%3DJ.-M.%2BG.%26aulast%3DKrysa%26aufirst%3DG.%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DRiou-Eymard%26aufirst%3DA.%2BM.%26aulast%3DBoursier%26aufirst%3DE.%2BV.%26aulast%3DTrottet%26aufirst%3DL.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DClark%26aufirst%3DH.%26aulast%3DChung%26aufirst%3DC.%26aulast%3DCutler%26aufirst%3DL.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DKaur%26aufirst%3DR.%26aulast%3DLewis%26aufirst%3DA.%2BJ.%26aulast%3DSchilling%26aufirst%3DM.%2BB.%26aulast%3DSoden%26aufirst%3DP.%2BE.%26aulast%3DTaylor%26aufirst%3DS.%26aulast%3DWalker%26aufirst%3DA.%2BL.%26aulast%3DWalker%26aufirst%3DM.%2BD.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DNicod%25C3%25A8me%26aufirst%3DE.%26atitle%3DThe%2520Discovery%2520of%2520I-BET726%2520%2528GSK1324726A%2529%252C%2520a%2520Potent%2520Tetrahydroquinoline%2520ApoA1%2520Up-Regulator%2520and%2520Selective%2520BET%2520Bromodomain%2520Inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D8111%26epage%3D8131%26doi%3D10.1021%2Fjm5010539" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3uvy" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3uvy','PDB','3uvy'); return false;">PDB: 3uvy</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YQW" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YQW','PDB','6YQW'); return false;">PDB: 6YQW</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5MKX" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5MKX','PDB','5MKX'); return false;">PDB: 5MKX</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3hme" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3hme','PDB','3hme'); return false;">PDB: 3hme</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YQR" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YQR','PDB','6YQR'); return false;">PDB: 6YQR</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YQZ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YQZ','PDB','6YQZ'); return false;">PDB: 6YQZ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YQS" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YQS','PDB','6YQS'); return false;">PDB: 6YQS</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5I7Y" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5I7Y','PDB','5I7Y'); return false;">PDB: 5I7Y</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i65"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00075">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_15270"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c00075?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00075</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Additional text describing all methods and results, all screening statistics, representative LCMS traces of target compounds, full BROMOscan and selectivity data, and X-ray data collection and refinement statistics (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00075/suppl_file/jm0c00075_si_001.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00075/suppl_file/jm0c00075_si_001.pdf">jm0c00075_si_001.pdf (977.06 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Coordinates have been deposited with the Protein Data Bank under accession codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YQW">6YQW</a> (BRD9/<b>8</b> complex), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YQR">6YQR</a> (BRD9/<b>14</b> complex), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YQZ">6YQZ</a> (BRD4(1)/<b>14</b> complex), and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YQS">6YQS</a> (BRD9/<b>18</b> complex). The authors will release atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c00075&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c00075%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-11" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c00075" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                12MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799c3cd883b329d","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
