
<html lang="en"     class="pb-page"  data-request-id="64331701-9319-4639-b002-8d1cb07f1c19"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.0c01272;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;issue:issue:10.1021/jmcmar.2020.63.issue-21"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="[1,2,4]Triazolo[1,5-a]pyrimidine Phosphodiesterase 2A Inhibitors: Structure and Free-Energy Perturbation-Guided Exploration" /></meta><meta name="dc.Creator" content="Gary  Tresadern" /></meta><meta name="dc.Creator" content="Ingrid  Velter" /></meta><meta name="dc.Creator" content="Andrés A.  Trabanco" /></meta><meta name="dc.Creator" content="Frans  Van den Keybus" /></meta><meta name="dc.Creator" content="Gregor J.  Macdonald" /></meta><meta name="dc.Creator" content="Marijke V. F.  Somers" /></meta><meta name="dc.Creator" content="Greet  Vanhoof" /></meta><meta name="dc.Creator" content="Philip M.  Leonard" /></meta><meta name="dc.Creator" content="Marieke B. A. C.  Lamers" /></meta><meta name="dc.Creator" content="Yves E. M.  Van Roosbroeck" /></meta><meta name="dc.Creator" content="Peter J. J. A.  Buijnsters" /></meta><meta name="dc.Description" content="We describe the hit-to-lead exploration of a [1,2,4]triazolo[1,5-a]pyrimidine phosphodiesterase 2A (PDE2A) inhibitor arising from high-throughput screening. X-ray crystallography enabled structure-..." /></meta><meta name="Description" content="We describe the hit-to-lead exploration of a [1,2,4]triazolo[1,5-a]pyrimidine phosphodiesterase 2A (PDE2A) inhibitor arising from high-throughput screening. X-ray crystallography enabled structure-..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="October 27, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c01272" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01272" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c01272" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01272" /></link>
        
    
    

<title>[1,2,4]Triazolo[1,5-a]pyrimidine Phosphodiesterase 2A Inhibitors: Structure and Free-Energy Perturbation-Guided Exploration | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c01272" /></meta><meta property="og:title" content="[1,2,4]Triazolo[1,5-a]pyrimidine Phosphodiesterase 2A Inhibitors: Structure and Free-Energy Perturbation-Guided Exploration" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0027.jpeg" /></meta><meta property="og:description" content="We describe the hit-to-lead exploration of a [1,2,4]triazolo[1,5-a]pyrimidine phosphodiesterase 2A (PDE2A) inhibitor arising from high-throughput screening. X-ray crystallography enabled structure-guided design, leading to the identification of preferred substructural components. Further rounds of optimization used relative binding free-energy calculations to prioritize different substituents from the large accessible chemical space. The free-energy perturbation (FEP) calculations were performed for 265 putative PDE2A inhibitors, and 100 compounds were synthesized representing a relatively large prospective application providing unexpectedly active molecules with IC50′s from 2340 to 0.89 nM. Lead compound 46 originating from the FEP calculations showed PDE2A inhibition IC50 of 1.3 ± 0.39 nM, ∼100-fold selectivity versus other PDE enzymes, clean cytochrome P450 profile, in vivo target occupancy, and promise for further lead optimization." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c01272"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01272">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c01272&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c01272&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c01272&amp;href=/doi/10.1021/acs.jmedchem.0c01272" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 21</span><span class="cit-fg-pageRange">, 12887-12910</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/21" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c00774" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c01285" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">[1,2,4]Triazolo[1,5-<i>a</i>]pyrimidine Phosphodiesterase 2A Inhibitors: Structure and Free-Energy Perturbation-Guided Exploration</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Gary Tresadern</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Gary Tresadern</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Computational Chemistry, Janssen Pharmaceutica N. V., Turnhoutseweg 30, B-2340 Beerse, Belgium</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#ff988b8d9a8c9e9b9abf968b8cd1959195d19c9092"><span class="__cf_email__" data-cfemail="aec9dadccbddcfcacbeec7dadd80c4c0c480cdc1c3">[email protected]</span></a>. Phone: +32 1464 1569.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Gary++Tresadern">Gary Tresadern</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4801-1644" title="Orcid link">http://orcid.org/0000-0002-4801-1644</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ingrid Velter</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ingrid Velter</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Chemistry, Janssen Pharmaceutica N. V., Turnhoutseweg 30, B-2340 Beerse, Belgium</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ingrid++Velter">Ingrid Velter</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Andrés A. Trabanco</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Andrés A. Trabanco</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Chemistry, Janssen Research & Development, Janssen-Cilag S. A., Jarama 75A, 45007 Toledo, Spain</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Andr%C3%A9s+A.++Trabanco">Andrés A. Trabanco</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4225-758X" title="Orcid link">http://orcid.org/0000-0002-4225-758X</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Frans Van den Keybus</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Frans Van den Keybus</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Chemistry, Janssen Pharmaceutica N. V., Turnhoutseweg 30, B-2340 Beerse, Belgium</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Frans++Van+den+Keybus">Frans Van den Keybus</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Gregor J. Macdonald</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Gregor J. Macdonald</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Chemistry, Janssen Pharmaceutica N. V., Turnhoutseweg 30, B-2340 Beerse, Belgium</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Gregor+J.++Macdonald">Gregor J. Macdonald</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Marijke V. F. Somers</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Marijke V. F. Somers</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Discovery Sciences, Janssen Research & Development, Janssen Pharmaceutica N. V., Turnhoutseweg 30, B-2340 Beerse, Belgium</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Marijke+V.+F.++Somers">Marijke V. F. Somers</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Greet Vanhoof</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Greet Vanhoof</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Discovery Sciences, Janssen Research & Development, Janssen Pharmaceutica N. V., Turnhoutseweg 30, B-2340 Beerse, Belgium</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Greet++Vanhoof">Greet Vanhoof</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Philip M. Leonard</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Philip M. Leonard</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Biology, Charles River Discovery (Previously BioFocus), Chesterford Research Park, Saffron Walden, CB10 1XL Essex, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Philip+M.++Leonard">Philip M. Leonard</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Marieke B. A. C. Lamers</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Marieke B. A. C. Lamers</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Biology, Charles River Discovery (Previously BioFocus), Chesterford Research Park, Saffron Walden, CB10 1XL Essex, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Marieke+B.+A.+C.++Lamers">Marieke B. A. C. Lamers</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yves E. M. Van Roosbroeck</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yves E. M. Van Roosbroeck</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Chemistry, Janssen Pharmaceutica N. V., Turnhoutseweg 30, B-2340 Beerse, Belgium</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yves+E.+M.++Van+Roosbroeck">Yves E. M. Van Roosbroeck</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Peter J. J. A. Buijnsters</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Peter J. J. A. Buijnsters</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Chemistry, Janssen Pharmaceutica N. V., Turnhoutseweg 30, B-2340 Beerse, Belgium</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#4c3c2e392526223f380c25383f62262226622f2321"><span class="__cf_email__" data-cfemail="8cfceef9e5e6e2fff8cce5f8ffa2e6e2e6a2efe3e1">[email protected]</span></a>. Phone: +32 1460 7439.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Peter+J.+J.+A.++Buijnsters">Peter J. J. A. Buijnsters</a></span></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01272&amp;href=/doi/10.1021%2Facs.jmedchem.0c01272" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 21</span><span class="cit-pageRange">, 12887–12910</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">October 26, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>21 July 2020</li><li><span class="item_label"><b>Published</b> online</span>27 October 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 12 November 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c01272" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01272</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D12887%26pageCount%3D24%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DGary%2BTresadern%252C%2BIngrid%2BVelter%252C%2BAndr%25C3%25A9s%2BA.%2BTrabanco%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D21%26contentID%3Dacs.jmedchem.0c01272%26title%3D%255B1%252C2%252C4%255DTriazolo%255B1%252C5-a%255Dpyrimidine%2BPhosphodiesterase%2B2A%2BInhibitors%253A%2BStructure%2Band%2BFree-Energy%2BPerturbation-Guided%2BExploration%26numPages%3D24%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D12910%26publicationDate%3DNovember%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c01272"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1126</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c01272" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;[1,2,4]Triazolo[1,5-a]pyrimidine Phosphodiesterase 2A Inhibitors: Structure and Free-Energy Perturbation-Guided Exploration&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Gary&quot;,&quot;last_name&quot;:&quot;Tresadern&quot;},{&quot;first_name&quot;:&quot;Ingrid&quot;,&quot;last_name&quot;:&quot;Velter&quot;},{&quot;first_name&quot;:&quot;Andrés&quot;,&quot;last_name&quot;:&quot;A. Trabanco&quot;},{&quot;first_name&quot;:&quot;Frans&quot;,&quot;last_name&quot;:&quot;Van den Keybus&quot;},{&quot;first_name&quot;:&quot;Gregor&quot;,&quot;last_name&quot;:&quot;J. Macdonald&quot;},{&quot;first_name&quot;:&quot;Marijke&quot;,&quot;last_name&quot;:&quot;V. F. Somers&quot;},{&quot;first_name&quot;:&quot;Greet&quot;,&quot;last_name&quot;:&quot;Vanhoof&quot;},{&quot;first_name&quot;:&quot;Philip&quot;,&quot;last_name&quot;:&quot;M. Leonard&quot;},{&quot;first_name&quot;:&quot;Marieke&quot;,&quot;last_name&quot;:&quot;B. A. C. Lamers&quot;},{&quot;first_name&quot;:&quot;Yves&quot;,&quot;last_name&quot;:&quot;E. M. Van Roosbroeck&quot;},{&quot;first_name&quot;:&quot;Peter&quot;,&quot;last_name&quot;:&quot;J. J. A. Buijnsters&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;10&quot;,&quot;day&quot;:&quot;27&quot;,&quot;issue&quot;:&quot;21&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;12887-12910&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c01272&quot;},&quot;abstract&quot;:&quot;We describe the hit-to-lead exploration of a [1,2,4]triazolo[1,5-a]pyrimidine phosphodiesterase 2A (PDE2A) inhibitor arising from high-throughput screening. X-ray crystallography enabled structure-guided design, leading to the identification of preferred substructural components. Further rounds of optimization used relative binding free-energy calculations to prioritize different substituents from the large accessible chemical space. The free-energy perturbation (FEP) calculations were performed for 265 putative PDE2A inhibitors, and 100 compounds were synthesized representing a relatively large prospective application providing unexpectedly active molecules with IC50′s from 2340 to 0.89 nM. Lead compound 46 originating from the FEP calculations showed PDE2A inhibition IC50 of 1.3 ± 0.39 nM, ∼100-fold selectivity versus other PDE enzymes, clean cytochrome P450 profile, in vivo target occupancy, and promise for further lead optimization.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01272&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01272" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01272&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01272" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01272&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01272" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01272&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01272&amp;href=/doi/10.1021/acs.jmedchem.0c01272" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c01272" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c01272" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01272&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c01272%26sid%3Dliteratum%253Aachs%26pmid%3D33105987%26genre%3Darticle%26aulast%3DTresadern%26date%3D2020%26atitle%3D%255B1%252C2%252C4%255DTriazolo%255B1%252C5-a%255Dpyrimidine%2BPhosphodiesterase%2B2A%2BInhibitors%253A%2BStructure%2Band%2BFree-Energy%2BPerturbation-Guided%2BExploration%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D21%26spage%3D12887%26epage%3D12910%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=290658" title="Amides">Amides</a>,</li><li><a href="/action/doSearch?ConceptID=290676" title="Substituents">Substituents</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=291901" title="Chemical calculations">Chemical calculations</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/21" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/jmcmar.2020.63.issue-21/20201112/jmcmar.2020.63.issue-21.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/medium/jm0c01272_0027.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0027.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01272&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">We describe the hit-to-lead exploration of a [1,2,4]triazolo[1,5-<i>a</i>]pyrimidine phosphodiesterase 2A (PDE2A) inhibitor arising from high-throughput screening. X-ray crystallography enabled structure-guided design, leading to the identification of preferred substructural components. Further rounds of optimization used relative binding free-energy calculations to prioritize different substituents from the large accessible chemical space. The free-energy perturbation (FEP) calculations were performed for 265 putative PDE2A inhibitors, and 100 compounds were synthesized representing a relatively large prospective application providing unexpectedly active molecules with IC<sub>50</sub>′s from 2340 to 0.89 nM. Lead compound <b>46</b> originating from the FEP calculations showed PDE2A inhibition IC<sub>50</sub> of 1.3 ± 0.39 nM, ∼100-fold selectivity versus other PDE enzymes, clean cytochrome P450 profile, <i>in vivo</i> target occupancy, and promise for further lead optimization.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_07302" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_07302" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The ubiquitous nucleotides 3′,5′-cyclic adenosine monophosphate (cAMP) and 3′,5′-cyclic guanosine monophosphate (cGMP) are second messengers that play a critical role in many signal transduction pathways. Both cyclic nucleotides are hydrolyzed by phosphodiesterase (PDE) enzymes to the corresponding biologically inactive adenosine and guanosine monophosphates, AMP and GMP, respectively. PDE enzymes are encoded by 21 genes and are grouped into 12 families, PDE1 to PDE12.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Their expression is tissue- and cell-specific; moreover, the different isozymes and splice variants are expressed in specific cell compartments and targeted to particular protein complexes, resulting in a spatiotemporal integration of multiple signaling pathways.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a></div><div class="NLM_p">PDE2A is a homodimer of two monomers approximately 105 kDa each. The enzyme has two regulatory units for each monomer, a GAF-A and a GAF-B domain that have distinct roles in dimerization and activation, respectively. Binding of cGMP to the GAF-B domain allosterically activates the enzyme resulting in a 6- to 30-fold increase of cAMP hydrolysis.<a onclick="showRef(event, 'ref3 ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref3 ref4 ref5">(3−5)</a> Once the enzyme is activated, it hydrolyzes both nucleotides with similar <i>K</i><sub>M</sub> values restoring the cAMP and cGMP concentrations to basal levels. PDE2A is expressed in the brain and is most prevalent in cortex, hippocampus, and striatum,<a onclick="showRef(event, 'ref6 ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref6 ref7 ref8">(6−8)</a> supporting a role in cognitive processes, neuronal plasticity, and memory.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Inhibition of PDE2A will result in elevation of intracellular cyclic nucleotide levels that play a key role in processes of neuroplasticity such as long-term potentiation (LTP), which is considered as a neurophysiological correlate of memory and learning.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> The exact underlying mechanism of memory enhancement following PDE2A inhibition is not fully understood, but it is likely occurring via the nitric oxide synthase (NOS)/cGMP/protein kinase-G pathway rather than the cAMP/protein kinase A pathway.<a onclick="showRef(event, 'ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref11 ref12">(11,12)</a></div><div class="NLM_p">Cognitive deficits are prominent in neurodegenerative disorders such as Alzheimer’s disease (AD). The earliest cognitive symptoms of AD include a decline in memory function,<a onclick="showRef(event, 'ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref13 ref14">(13,14)</a> manifest as problems with planning and task execution (executive function),<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> described as an inability to successfully form new memory traces for integration into existing memory networks.<a onclick="showRef(event, 'ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref16 ref17">(16,17)</a> While the impairment of episodic memory is frequently the earliest detectable neurocognitive sign, the earliest symptoms often cooccur with neuropsychiatric behavioral symptoms.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> Upon progression of the disease, more cognitive function is lost<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> and patients develop concurrent neuropsychiatric symptoms. Disease-modifying treatments in AD are yet to be established, and currently available symptomatic treatments are only moderately effective or limited in their use because of their side effects. Therefore, improved treatments to alleviate or even reverse the symptoms of reduced cognitive function represent a significant unmet medical need. Thus, PDE2A inhibition may be procognitive and have a positive impact in the treatment of cognitive dysfunction in early stage AD,<a onclick="showRef(event, 'ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref20 ref21">(20,21)</a> and PDE2 inhibition has been the subject of recent medicinal chemistry reports.<a onclick="showRef(event, 'ref22 ref23 ref24 ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref22 ref23 ref24 ref25 ref26">(22−26)</a></div><div class="NLM_p last">We previously described the structure-based optimization of tricyclic PDE2A inhibitors such as <b>1</b> and <b>2</b>.<a onclick="showRef(event, 'ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref27 ref28">(27,28)</a> Although highly potent and selective, poor solubility and high plasma clearance could not be overcome, limiting their developability. Here, we report a series of [1,2,4]triazolo[1,5-<i>a</i>]pyrimidine PDE2A inhibitors arising from high throughput screening (HTS). X-ray crystallography of the initial hit <b>3</b> allowed structure-guided design, leading to improved target potency. An X-ray structure of a second example confirmed the binding mode, and we employed computational relative binding free-energy (RBFE) calculations to guide the molecular design. The accurate prediction of protein–ligand interaction energies has been, and remains, an important but very challenging goal for computational structure-based design.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> We applied the free-energy perturbation (FEP) approach that calculates the binding energy difference between structurally similar ligands, making it well suited for lead optimization (LO). The method is undergoing a resurgence of interest<a onclick="showRef(event, 'ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref30 ref31">(30,31)</a> and we have seen a good performance in drug discovery programs<a onclick="showRef(event, 'ref32 ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref32 ref33 ref34">(32−34)</a> including PDE2A<a onclick="showRef(event, 'ref28 ref35'); return false;" href="javascript:void(0);" class="ref ref28 ref35">(28,35)</a> and for other large protein–ligand data sets.<a onclick="showRef(event, 'ref36 ref37 ref38'); return false;" href="javascript:void(0);" class="ref ref36 ref37 ref38">(36−38)</a> Here, FEP calculations on 250 triazolopyrimidines were followed by the synthesis of 100 examples, giving a valuable demonstration of FEP-guided LO. Overall, the resulting lead compounds showed excellent PDE2A inhibition potency (IC<sub>50</sub> < 1 nM) and selectivity of up to 100- to 1000-fold versus other PDE enzymes. Brain penetration was confirmed for selected leads and the most promising examples, <b>42</b> and <b>46</b>, showed <i>in vivo</i> target engagement in a PDE2A occupancy assay.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_15076" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_15076" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">HTS hit <b>3</b> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) had a PDE2A inhibition IC<sub>50</sub> of 66 ± 3.2 nM and approximately eightfold selectivity versus PDE10A (the closest structural PDE family member to PDE2A). It was inactive (IC<sub>50</sub> > 10,000 nM) in all cases when tested for inhibition activity in a larger panel of PDE enzymes (PDE1B, PDE3A, PDE4D, PDE5A, PDE7A, PDE9A, and PDE11A). Compound <b>3</b> originated from commercial sources and was part of our internal screening library. It was tested in over 100 internal screens and was inactive (IC<sub>50</sub> > 10,000 nM) other than for PDE2A and PDE10A. We began our experimental follow-up by solving the X-ray crystallographic structure of <b>3</b> with the PDE2A catalytic domain, refined to 2.35 Å resolution (<i>R</i><sub>work</sub> = 21.5% and <i>R</i><sub>free</sub> = 29.1%). There were two PDE2A molecules in the asymmetric unit, but each active site overlaid well and showed minimal variation in ligand binding. Molecule <b>3</b> sits in the hydrophobic clamp between Phe830 and Phe862 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). Most active site interactions with the ligand are formed with the triazolopyrimidine core. Gln812 donates a 2.3 Å hydrogen bond from its NH side chain to the pyrimidine sp<sup>2</sup> nitrogen acceptor of the bicycle. Another hydrogen bond with a distance of 2.5 Å is donated from the NH side chain of Gln859 to the triazole sp<sup>2</sup> nitrogen and Phe862 forms an off-center π-stack above the scaffold. The linking amido piperidinyl group has an end-on π-interaction with Phe830 and is largely solvent exposed along with the pendant trimethoxyphenyl residue that is bound on the solvent exposed surface of helix 15. The para methoxy group forms a hydrogen bond with a water molecule. The X-ray structure settled the uncertainty of whether the alkoxyphenyl in <b>3</b> may interact with the “switch” Gln859 as seen for well-known PDE inhibitors such as Papaverine and Rolipram.<a onclick="showRef(event, 'ref39 ref40'); return false;" href="javascript:void(0);" class="ref ref39 ref40">(39,40)</a> Further examination of the binding mode suggested that there was space to substitute on the free positions of the pyrimido ring along with ample opportunity for amide replacements.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/medium/jm0c01272_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0003.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Previous tricyclic PDE2A inhibitor leads <b>1</b> and <b>2</b>, along with triazolopyrimidine HTS hit <b>3</b> which is the subject of this report.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01272&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/medium/jm0c01272_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0004.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. X-ray structure of HTS hit <b>3</b> with PDE2A viewed from the entrance to the active site (A) and top down (B), PDB accession code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZND">6ZND</a>. Important amino acids are highlighted in green, and catalytic Zn<sup>2+</sup> and Mg<sup>2+</sup> ions are shown, along with active site water molecules (red spheres); hydrogen bonds between Gln812 and Gln859 and the triazolopyrimidine scaffold are indicated with dashed yellow lines in panel B.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01272&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The chemistry exploration of the initial hit began with the preparation of the right-hand side (RHS, <a class="ref internalNav" href="#cht1" aria-label="Chart 1">Chart 1</a>) amides <b>3</b>–<b>18</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). The compounds were tested in a full-length human PDE2A inhibition assay measuring changes in levels of cAMP by a scintillation proximity approach. Consistent with the binding mode, a variety of benzamides were tolerated as substituents on the piperidine. Initial hit <b>3</b> was racemic and a 66 nM PDE2A inhibitor. The dimethoxy analogue <b>4</b> showed similar PDE2A inhibition activity (IC<sub>50</sub> = 47 ± 16 nM). Homologation via a methylene spacer between the amide carbonyl and the distal aromatic resulted in a ∼10-fold drop in activity for <b>5</b> compared to <b>4</b> (<b>5</b>, IC<sub>50</sub> = 479 ± 16 nM vs <b>4</b>, IC<sub>50</sub> = 47 ± 16 nM). The X-ray structure of <b>3</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>) suggests that this was due to a loss of the optimal interactions with Met847 and Phe862. Simplification to the unsubstituted phenyl <b>6</b> (IC<sub>50</sub> = 339 ± 31 nM) or replacement of the distal aromatic by small aliphatic groups, <b>7</b> (IC<sub>50</sub> = 246 ± 12 nM) and <b>8</b> (IC<sub>50</sub> = 1580 ± 180 nM), were detrimental for activity, again because of the loss of preferred aromatic interactions with Met847 and Phe862.</div><figure id="cht1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Chart 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/medium/jm0c01272_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0002.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Chart 1. Medicinal Chemistry Strategy for Exploration of HTS Hit <b>3</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01272&amp;id=cht1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. RHS Exploration of HTS Hit <b>3</b><a class="ref internalNav" href="#t1fn1" aria-label="a">a</a><sup>,</sup><a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/medium/jm0c01272_0018.gif" alt="" id="GRAPHIC-d7e631-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/medium/jm0c01272_0019.gif" alt="" id="gr20" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Primary inhibition activity versus PDE2A, and activity versus close structural family members PDE3A/B and PDE10A.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">PDE2A bioactivity measurements are the mean of at least two independent repeats and provided with standard deviations except <b>6</b>, <b>8</b>, and <b>12</b>, which are provided with the standard error on dose response for <i>n</i> = 1.</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">All data were generated with PDE3A except compounds <b>8</b>, <b>9</b>, and <b>15</b> that were tested versus PDE3B.</p></div></div><div></div></div><div class="NLM_p">The PDE2A activity was dependent on the position and type of substitution on the aromatic ring. For instance, meta and para substituents were preferred, exemplified with ortho methoxy <b>9</b> (IC<sub>50</sub> = 912 ± 200 nM) that was >12-fold less active than meta <b>10</b> (IC<sub>50</sub> = 74 ± 27 nM) or para <b>11</b> (IC<sub>50</sub> = 50 ± 22 nM). It was expected that the ortho substituent caused the rotation of the distal aromatic ring and disrupted its interaction with Phe862. Meanwhile, it was clear that electron donating groups were beneficial for activity compared to withdrawing, for instance, the comparison of <i>para</i> methoxy <b>11</b> (IC<sub>50</sub> = 50 ± 22 nM) to <i>para</i> cyano <b>12</b> (IC<sub>50</sub> = 589 ± 280 nM) and trifluoromethyl <b>13</b> (IC<sub>50</sub> = 324 ± 96 nM) substituents. The electron-rich aromatic rings were preferred as they form an energetically more favorable T-shaped π-interaction with Phe862, <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>. Conformational restriction of the methoxy groups into the benzodioxane derivative <b>14</b> (IC<sub>50</sub> = 49 ± 21 nM) maintained a similar range of activity compared to <b>4</b> (IC<sub>50</sub> = 47 ± 16 nM), although <b>4</b> was racemic. Fully aromatic bicyclic examples such as benzofuran <b>15</b> improved the inhibition activity to 24 ± 8.5 nM. This prompted us to expand the exploration around these bicyclic heteroaromatics, resulting in analogues such as <b>16</b> and <b>17</b> that reached PDE2A potencies of 2.1 ± 0.9 and 2.6 ± 1.1 nM, respectively. The tetrahydrobenzothiophene group attached via the five-membered ring in <b>18</b> was a 10 ± 5.2 nM PDE2A inhibitor and was selected as the template to explore central core and LHS variations. Finally, the observation of the <i>S</i> stereoisomer binding in the X-ray structure was formally confirmed by chiral separation of <b>17</b>. Indeed, the <i>S</i> stereoisomer retains most of the PDE2A inhibitory activity with <b>20</b> being 60-fold more active than its enantiomer <b>19</b> (<b>20</b>, IC<sub>50</sub> = 1.5 ± 0.1 nM vs <b>19</b>, IC<sub>50</sub> = 87 ± 27 nM).</div><div class="NLM_p">The identification of a selective inhibitor was a key aspect of our strategy to later validate the role of PDE2A inhibition for cognition. The main concern was selectivity versus PDE10A given its close structural similarity to PDE2A (34% sequence identity, 50% similarity for the catalytic domain) and its implication in other central nervous system disorders.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> Indeed, the selectivity data available for all the compounds in this article confirms only weak and sparse activity (range 3160–10,000 nM for 12 out of 140 IC<sub>50</sub> measurements) versus multiple other PDE family members (PDE1B, PDE4D, PDE5A, PDE7A, PDE9A, and PDE11A), <a href="/doi/suppl/10.1021/acs.jmedchem.0c01272/suppl_file/jm0c01272_si_001.pdf" class="ext-link">Table S1</a>. The HTS hit <b>3</b> showed a modest 8-fold selectivity, but this was substantially improved to approximately 30- to 50-fold with bicyclic amides such as <b>16</b>–<b>18</b>. This could be rationalized because of the interaction with Leu858 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>), which forms the boundary at the bottom of the benzamide pocket and interacts directly with the meta and para substituents of the distal aromatic ring, or in the case of <b>16</b>–<b>18</b>, the fused 5-membered ring. A glycine is found at the same position in PDE10A (Gly725), and because of the absence of the side chain, the pocket extends further, meaning that this region of the binding site, traditionally referred to as the Q2 pocket,<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> can be exploited for PDE selectivity.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a></div><div class="NLM_p">Turning to the core modifications (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>), replacement of the piperidine in <b>18</b> by a morpholine was detrimental for activity with <b>21</b> being ninefold less active than <b>18</b> (<b>21</b>, IC<sub>50</sub> 91 ± 29 vs <b>18</b>, 10 ± 5.2 nM). Although the unsubstituted piperazine was substantially less active (<b>22</b>, IC<sub>50</sub> = 1412 ± 196 nM), the <i>N</i>-methyl analogue <b>23</b> showed a double-digit nanomolar PDE2A inhibition activity (IC<sub>50</sub> = 87 ± 41 nM). Acylation of the piperazine nitrogen in <b>24</b> was inactive up to the 10 μM concentration limit. Ring enlargement was also detrimental for activity with the homopiperidine <b>25</b> being 26-fold less active than the corresponding piperidine matched pair (<b>25</b>, IC<sub>50</sub> = 263 ± 24 nM vs <b>18</b>, IC<sub>50</sub> = 10 ± 5.2 nM). Hence, the piperidine was optimal for the activity, something that could be understood from the X-ray structure because of its close lipophilic interaction with the side chains of Tyr655 and Ile826. The selectivity of the most active examples in this exploration, <b>21</b> and <b>23</b>, was similar to the parent piperidine central core, <b>18</b>. Indeed, all three examples maintained the same 28-fold selectivity versus PDE10A, demonstrating the origin of selectivity from the constant presence of the tetrahydrobenzothiophene amide. In contrast, <b>18</b> and <b>21</b> displayed only 1.4- to 4-fold selectivity versus PDE3A. The sequence similarity between PDE2A and PDE3A (20% sequence identity, 33% similarity for the catalytic domain) was lower than that for PDE10A, and the loss of selectivity versus PDE3A was initially a surprise. Nevertheless, the complete lack of PDE3A inhibition activity for some benzamide replacements (<b>3</b>, <b>7</b>, and <b>19</b><a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) offered promise that PDE3A selectivity could be modulated in the same Q2 pocket that was targeted for PDE10A selectivity.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> Comparison of PDE2A and PDE3B<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> crystal structures revealed differences, for instance, Met847 (PDE2A, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) was instead Phe976 (PDE3B) that also adopts a different conformation and changes the shape of the pocket” to “for instance, Met847 in PDE2A (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) was instead Phe976 in PDE3B. The Phe976 also adopts a different conformation compared to Met847 and changes the shape of the pocket.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Central Core Modifications<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a><sup>,</sup><a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/medium/jm0c01272_0020.gif" alt="" id="GRAPHIC-d7e922-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/medium/jm0c01272_0021.gif" alt="" id="gr17" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Primary inhibition activity versus PDE2A, and activity versus close structural family members PDE3A/B and PDE10A.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">PDE2A bioactivity measurements are the mean of at least two independent repeats and provided with standard deviations except <b>22</b> and <b>25</b> which are provided with the standard error on dose response for <i>n</i> = 1.</p></div><div class="footnote" id="t2fn3"><sup><sup>c</sup></sup><p class="last">All data were generated with PDE3A except compound <b>23</b> that was tested versus PDE3B.</p></div></div><div></div></div><div class="NLM_p">To complete the initial structure–activity relationship (SAR) exploration around the triazolopyrimidine hit series, the LHS group (<a class="ref internalNav" href="#cht1" aria-label="Chart 1">Chart 1</a>) was modified while maintaining the piperidine central core and RHS residue present in <b>18</b>. The introduction of a methyl substituent on the pyrimidine ring brought immediate improvements in potency and compound <b>26</b> showed a 3.5 ± 2.3 nM PDE2A inhibitor IC<sub>50</sub> activity (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). According to the crystal structure binding mode of <b>3</b>, the methyl group in <b>26</b> would occupy a small pocket deep in the binding site. The larger isopropyl group in <b>27</b> would not fit and hence the compound was much less active (<b>27</b>, IC<sub>50</sub> = 6031 ± 1943 nM). The smaller methoxy group in the same position recovered the activity (<b>28</b>, IC<sub>50</sub> = 195 ± 85 nM). Triazolopyrimidine compounds bearing electron-withdrawing cyano (<b>29</b>) and trifluoromethyl (<b>30</b>) substituents showed good PDE2A inhibition values (10 ± 5.9 and 50 ± 19 nM, respectively), although they were less potent than the methyl analogue <b>26</b>. Interestingly, the CHF<sub>2</sub> group in <b>31</b> delivered an equipotent PDE2A inhibitor (<b>31</b>, 3.4 ± 2.3 nM vs <b>26</b>, 3.5 ± 2.3 nM). Modification of the heterobicyclic ring (<b>32</b>–<b>34</b>) resulted in all cases in a substantial loss of activity. The IC<sub>50</sub> of pyrazolopyrimidine <b>32</b>, imidazolopyrimidine <b>33</b>, and azaindole <b>34</b> were 30 ± 22, 407 ± 66, and 6020 ± 1980 nM, respectively. Finally, the introduction of substituents in the triazolo ring in <b>26</b> was clearly detrimental for activity with methyl- (<b>35</b>, IC<sub>50</sub> = 562 ± 65 nM) or trifluoromethyl-substituted (<b>36</b>, IC<sub>50</sub> = 2951 ± 478 nM) analogues 160- and 842-fold less active than the PDE2A inhibitor <b>26</b>. In general, selectivity improved by the introduction of the small substituent on the pyrimidine ring, for instance <b>31</b> was nearly 200-fold and 16-fold selective versus PDE10A and PDE3A, respectively. Compound <b>31</b> was tested in a larger PDE inhibition panel including PDE1B, PDE4D, PDE5A, PDE7A, and PDE11A, where it showed >10,000 nM activity in all assays except for PDE4D, where a moderate activity (IC<sub>50</sub> = 6460 nM) was measured.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. LHS Exploration of HTS Hit <b>3</b><a class="ref internalNav" href="#t3fn1" aria-label="a">a</a><sup>,</sup><a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/medium/jm0c01272_0022.gif" alt="" id="GRAPHIC-d7e1076-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/medium/jm0c01272_0023.gif" alt="" id="gr18" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">Primary inhibition activity versus PDE2A, and activity versus close structural family members PDE3A/B and PDE10A.</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last">PDE2A bioactivity measurements are the mean of at least two independent repeats and provided with standard deviations except for <b>33</b>, <b>35</b>, and <b>36</b>, which are provided with the standard error on dose response for <i>n</i> = 1.</p></div><div class="footnote" id="t3fn3"><sup><sup>c</sup></sup><p class="last">All data were generated with PDE3A except for compounds <b>28</b>, <b>29</b>, <b>32</b>, and <b>33</b> that were tested versus PDE3B.</p></div></div><div></div></div><div class="NLM_p">From the initial SAR studies, it was clear that although the RHS substituent tolerated a diverse set of aromatic amides, the substitution pattern and electronic nature of the substituents were important for achieving increased PDE2A inhibition. This was due to forming optimal nearby interactions, particularly with Met847 and Phe862. All modifications of the central scaffold were less active than the parent piperidine example. Meanwhile, the LHS group that hydrogen bonds with the Gln812 switch and Gln859, permitted a CH<sub>3</sub> and CHF<sub>2</sub> substituent entering deep into the binding pocket. Therefore, using the optimal LHS along with the piperidine central ring, benzylic heteroaromatic groups were studied as RHS benzamide replacements (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). The SAR obtained did not correlate well with the phenyl amide subseries. First, both the dimethoxyphenyl- (<b>37</b>) and the benzofuran-containing (<b>38</b>) triazolopyrimidine analogues were less active than the equivalent amides <b>4</b> and <b>15</b> (<b>37</b>, IC<sub>50</sub> = 741 ± 345 nM vs <b>4</b>, IC<sub>50</sub> = 47 ± 16 nM and <b>38</b>, IC<sub>50</sub> = 1148 ± 207 nM vs <b>15</b>, IC<sub>50</sub> = 24 ± 8.5 nM, respectively). Furthermore, the relative order of activity was reversed. Interestingly, the benzyl-like pyridine derivatives <b>39</b> and <b>40</b> were 24 ± 6.3 and 69 ± 39 nM PDE2A inhibitors and recovered selectivity, being >120-fold selective for both PDE3A and PDE10A. This confirmed the comparison between PDE2A and PDE3B X-ray structures that revealed that the sequence and structural differences in the Q2 pocket could lead to PDE3A selectivity. It was expected that these electron-rich heteroaromatics would make favorable interactions with Phe862 as mentioned above, but the differences in activity suggested that different ligand conformations were possible compared to equivalent amides. Therefore, intrigued to understand if this would translate into structural differences when bound to the enzyme, we attempted and solved an X-ray crystal structure of <b>39</b> with the PDE2A catalytic domain.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Second Round RHS Modifications<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a><sup>,</sup><a class="ref internalNav" href="#t4fn2" aria-label="b">b</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/medium/jm0c01272_0024.gif" alt="" id="GRAPHIC-d7e1188-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/medium/jm0c01272_0025.gif" alt="" id="gr19" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">Primary inhibition activity versus PDE2A, and activity versus close structural family members PDE3B and PDE10A.</p></div><div class="footnote" id="t4fn2"><sup><sup>b</sup></sup><p class="last">PDE2A bioactivity measurements are the mean of at least two independent repeats and provided with standard deviations.</p></div></div><div></div></div><div class="NLM_p">The structure of PDE2A in complex with lead <b>39</b> was solved at 1.9 Å resolution (<i>R</i><sub>work</sub> = 19.6% and <i>R</i><sub>free</sub> = 24.4%), and the binding site is shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>. There were four PDE2A monomers per asymmetric unit and each had a single <b>39</b> molecule bound in its active site. Structural overlay indicated little conformational difference between the four subunits; the electron density was best defined for subunit D. The triazolopyrimidine core of <b>39</b> was again anchored by two hydrogen bonds to Gln859 and Gln812 while being held in a hydrophobic clamp by Ile826, Phe830 and Phe862. The difluoro methyl group sits in a small, well-buried pocket, as expected from the X-ray structure of <b>3</b>. The proton of the difluoro methyl group is electropositive and makes a favorable electrostatic interaction with the phenol oxygen of the neighboring Tyr655. The piperidine core is largely solvent exposed along with the pendant pyridine ring. Positive electron density peaks in the final difference density maps indicated a second alternative conformation of the pyridine ring related by a 180° rotation of the pyridine, positioning the chlorine substituent toward the solvent space. <i>B</i>-Factor refinement indicated less than 20% occupancy of this second conformation in each of the four subunits. The alternative orientation was therefore not included in the final model. Nevertheless, the second possible orientation was consistent with conformational effects contributing to lower inhibition activity of benzyl- compared to benzamide-substituents (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). The pyridyl ring was again sandwiched between Met847 and Phe862, and the nitrogen formed a hydrogen bond to a crystallographic water at the bottom of the Q2 pocket.</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/medium/jm0c01272_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0005.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. X-ray structure of lead <b>39</b> with PDE2A viewed from the entrance to the active site (A) and top down (B), PDB accession code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZQZ">6ZQZ</a>. Important amino acids are highlighted in green, and catalytic Zn<sup>2+</sup> and Mg<sup>2+</sup> ions are shown, along with active site water molecules (red spheres); hydrogen bonds are indicated with dashed yellow lines in panel B.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01272&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Racemization of the Central Scaffold</h3><div class="NLM_p">Up to this point, we had several promising leads, including <b>15</b>–<b>18</b>, <b>26</b>, and <b>31</b> that showed PDE2A inhibition <10 nM and in most cases >10-fold selectivity versus PDE3A and PDE10A. Unfortunately, it was noticed that after chiral separation, in some cases, enantiopure compounds in solution turned into mixtures after several days. Racemization presented a challenge for the progression of the series. Interestingly, the problem was overcome by introducing a methyl substituent into the 4-position of the piperidine ring. Several examples were synthesized and shown not only to maintain but even improve potency (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). This was likely because of the preference for hydrophobic interactions with the side chains of Tyr655 and Ile826, as seen for the central core modifications (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Compound <b>41</b> showed high PDE2A inhibition activity (IC<sub>50</sub> = 1.1 ± 0.4 nM) and >90-fold selectivity versus PDE3A (IC<sub>50</sub> = 2240 nM) and PDE10A (IC<sub>50</sub> = 97 nM). It was also tested for inhibition against a larger panel of PDE enzymes and was inactive (IC<sub>50</sub> > 10,000 nM) in all cases (PDE1B, PDE4D, PDE7A, and PDE11A) except for PDE5A, where a weak activity was detected (IC<sub>50</sub> = 6760 nM). The chiral example <b>42</b> showed sub-nanomolar PDE2A inhibition (IC<sub>50</sub> = 0.6 ± 0.3 nM) and was 2200- and 90-fold selective versus PDE3A and PDE10A, respectively.</div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/medium/jm0c01272_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0006.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. 4-Methyl-substituted piperidines <b>41</b>–<b>43</b>, introduction of the methyl group at the 4-position of the piperidine central scaffold to prevent racemization.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01272&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> RBFE Calculations</h3><div class="NLM_p">Up to this point, we had mostly used amide substitution on the piperidine nitrogen to explore the hit series. However, molecules with benzylic functionality such as <b>39</b> and <b>40</b> demonstrated that a large chemically accessible space may be possible. Hence, we turned to computational prioritization using RBFE calculations with FEP methodology. The method requires that perturbed molecules share an overlapping binding mode during the computational simulations. This was expected here as the two crystallographic binding orientations of <b>3</b> and <b>39</b> were highly similar, suggesting that different chemical decoration can be probed with FEP calculations. Others have used binding free-energy calculations for PDE10<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> and for other PDE enzymes.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> We have described accurate RBFE predictions for ligand modifications in a different subpocket of the PDE2A binding site.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Here, the ligand changes are relatively solvent exposed and that was expected to help the performance of the calculations by avoiding issues with kinetically trapped solvent.</div><div class="NLM_p">We began by testing the FEP performance with a small retrospective validation study using 20 existing molecules involving a mixture of structurally similar amides and benzyls covering a 4 log unit range of activity. A good correlation, <i>R</i><sup>2</sup> 0.69, and mean unsigned error (MUE) of 0.89 ± 0.35 kcal/mol was seen when comparing experimental and calculated binding affinities, <a href="/doi/suppl/10.1021/acs.jmedchem.0c01272/suppl_file/jm0c01272_si_001.pdf" class="ext-link">Figure S1</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.0c01272/suppl_file/jm0c01272_si_001.pdf" class="ext-link">Table S2</a>. This prompted us to continue and perform successive rounds of FEP to help prioritize on-going chemistry explorations on a roughly monthly basis. The calculations were applied to different virtual libraries containing varied substituents on the piperidine nitrogen. The input for the FEP calculations was guided using docking that performed well at reproducing the expected binding modes, <a href="/doi/suppl/10.1021/acs.jmedchem.0c01272/suppl_file/jm0c01272_si_001.pdf" class="ext-link">Figure S2</a>. The reagents for each virtual library were selected based on their similarity to the known actives, that is, aromatic amides, and their chemical feasibility and availability. These included amides, benzylic groups, and heteroaromatic halide ((het)aryl) reagents (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>). This study was part of a larger in-house evaluation<a onclick="showRef(event, 'ref28 ref32 ref33 ref35'); return false;" href="javascript:void(0);" class="ref ref28 ref32 ref33 ref35">(28,32,33,35)</a> of FEP, and therefore, we chose compounds from across the range of predicted affinity to avoid issues of selection bias and deliver a fair evaluation of the FEP performance.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a></div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Summary of the Rounds of FEP Calculations That Were Applied to Prospective Molecular Design</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/medium/jm0c01272_0026.gif" alt="" id="GRAPHIC-d7e1381-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">round</th><th class="colsep0 rowsep0" align="center">R-substituents</th><th class="colsep0 rowsep0" align="center" char=".">no calculated</th><th class="colsep0 rowsep0" align="center" char=".">no synthesized</th><th class="colsep0 rowsep0" align="center" char="."><i>R</i><sup>2</sup></th><th class="colsep0 rowsep0" align="center" char="±">MUE (kcal/mol)<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">amides & (het)aryls</td><td class="colsep0 rowsep0" align="char" char=".">59</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="char" char=".">0.77</td><td class="colsep0 rowsep0" align="char" char="±">1.02 ± 0.90</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">(het)aryls</td><td class="colsep0 rowsep0" align="char" char=".">57</td><td class="colsep0 rowsep0" align="char" char=".">29</td><td class="colsep0 rowsep0" align="char" char=".">0.51</td><td class="colsep0 rowsep0" align="char" char="±">0.78 ± 0.27</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">benzyls</td><td class="colsep0 rowsep0" align="char" char=".">58</td><td class="colsep0 rowsep0" align="char" char=".">29</td><td class="colsep0 rowsep0" align="char" char=".">0.28</td><td class="colsep0 rowsep0" align="char" char="±">1.01 ± 0.33</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">amides</td><td class="colsep0 rowsep0" align="char" char=".">39</td><td class="colsep0 rowsep0" align="char" char=".">10</td><td class="colsep0 rowsep0" align="char" char=".">0.41</td><td class="colsep0 rowsep0" align="char" char="±">0.47 ± 0.32</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">pyridyls</td><td class="colsep0 rowsep0" align="char" char=".">52</td><td class="colsep0 rowsep0" align="char" char=".">27</td><td class="colsep0 rowsep0" align="char" char=".">0.58</td><td class="colsep0 rowsep0" align="char" char="±">0.62 ± 0.29</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">±99% confidence interval. (Het)aryls—heteroaromatic halide reagents delivering compounds with a heteroaromatic ring directly bonded to the piperidine nitrogen.</p></div></div></div><div class="NLM_p">In total, 100 molecules were synthesized from the different libraries. The experimental and predicted binding free energies (δ<i>G</i>, kcal/mol) are compared in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>. The most active molecules are those in the bottom left of the plot with the largest negative favorable binding free energy. The correlation of <i>R</i><sup>2</sup> 0.57 and overall MUE of 0.79 ± 0.16 kcal/mol were consistent with the retrospective validation. A key reason for exploring quantitative RBFE predictions is the poor activity prediction for congeneric series by methods such as docking. This was also the case here because the docking, although able to reproduce the binding pose, was unable to predict binding affinity for the same set of compounds, <i>R</i><sup>2</sup> 0.01 and MUE of 2.27 ± 0.33 kcal/mol, <a href="/doi/suppl/10.1021/acs.jmedchem.0c01272/suppl_file/jm0c01272_si_001.pdf" class="ext-link">Figure S2</a>.</div><figure id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/medium/jm0c01272_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0007.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Correlation of experimental and calculated binding activity (∂<i>G</i>) for the compounds synthesized in each round of the FEP-guided design. The compound data points are colored and shaped according to the rounds of FEP and synthesis, see <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>. The diagonal lines of unity and ±1 or 2 kcal/mol errors are shown. The trendline (black, partly obscured by the line of unity), equation, and <i>R</i><sup>2</sup> were calculated based on all the data combined, showing almost linear slope and very little offset. Selected compounds are labeled.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01272&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">There were various expected and unexpected examples amongst the FEP results. The calculations predicted multiple amides as highly active. An analogue of <b>42</b> where the benzofuran was replaced by indole (changing only the oxygen atom for NH) was a high affinity 0.9 ± 0.4 nM PDE2A inhibitor but structurally very close to previous examples. Therefore, although this was an interesting example of successful activity prediction, that is itself a difficult achievement, it is a relatively obvious result. However, amongst a series of 4-substituted pyridyls, <b>44</b>–<b>46</b>, example <b>46</b> with the 4-phenyl substituted pyridine was a surprising 1.3 ± 0.4 nM PDE2A inhibitor that showed ∼100-fold increase in activity compared to analogues <b>44</b> and <b>45</b> (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>). The phenyl ring in the 4-pyridyl position would have been disregarded based on the relatively low activity of <b>44</b> that presents an amide with an aromatic character in this position and <b>45</b> with a lipophilic <i>t</i>-butyl group. Curiously, the FEP calculation for this molecule was performed twice in two separate libraries of compounds, as seen by the almost overlapping star and square labels highlighted in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>. It was part of a general library of heteroaromatic halide reagents (round 2), and again in the more focused halo-pyridyl reagent library (round 5). In both cases, it was predicted to be a potent example that was confirmed after synthesis and testing and without the calculations this example would have been overlooked.</div><figure id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/medium/jm0c01272_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0008.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Comparative examples of 4-substituted pyridine RHS substituents arising from the FEP calculations.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01272&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The difference in activity between <b>45</b> and <b>46</b> was correctly predicted by the FEP calculations, predicted to be 4.0 kcal/mol in favor of <b>46</b> compared to 2.9 kcal/mol experimentally. In other words, the calculations were capturing the multiple SAR effects that include the aromatic interaction with Phe862, the CH–pi interactions between the 4-phenyl substituent of the ligand and Ile866, and displacement of the crystallographic water at the bottom of the Q2 pocket from a relatively hydrophobic and exposed region of the protein surface (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a> and <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a> for binding site details). This is manifest in the mostly good correlation seen for the different FEP rounds. Overall, relatively substantial improvements in potency were seen for a pocket that while solvent exposed contains multiple hydrophobic side chains, Met847, Leu858, Phe862, and Ile866. This makes it an ideal pocket for improving binding affinity and also avoids difficulties for FEP calculations, such as trapped solvent in buried pockets or needing to introduce polar interactions with optimal distance and angles to compete with the solvent. Altogether these factors may explain the good performance of FEP in this case.</div><div class="NLM_p">The error between experimental and predicted binding affinity of the successive rounds of FEP improved, for instance, rounds 4 and 5 showed MUE of 0.47 ± 0.32 and 0.62 ± 0.29 kcal/mol, respectively, compared to the first rounds with MUE ∼1 kcal/mol. The first round included molecules with two different types of substitution on the piperidine nitrogen, all the subsequent rounds only used one type of chemical substituent. The benzyl library, round 3, performed the worst, <i>R</i><sup>2</sup> 0.28 and MUE 1.01 ± 0.33 kcal/mol.</div><div class="NLM_p">The lower correlation could be due to alternative binding conformations, although the low populated conformation in the X-ray structure of <b>39</b> was seen in the simulations, or it could be due to a change in the ionization state of the piperidine nitrogen, which was treated as neutral in all cases but may vary based on the benzylic substituent. The correlation of the amide library in round 4 was low, but this was due to a smaller range of the resulting experimental activity (1.3 log units), making it difficult to achieve high correlation but the MUE remained good.</div><div class="NLM_p">Overall, the improvement with each round was attributed to the closer structural similarity between the compounds (beneficial for the convergence of the calculations) because each library became more focused on analogues of previously identified actives. We did not include additional experimental reference molecules in the subsequent rounds, although this can be used as a strategy to improve results. In our experience, this application of FEP has been particularly successful for various reasons. On the one hand, the SAR is relatively easy to capture with important hydrophobic interactions in a solvent accessible region, where water displacement can be modeled. The site of perturbation is exposed, does not require protein movement, and is far away from the active site metal ions which can cause difficulties, all in contrast to our previous FEP studies on PDE2.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Practical issues also make this system amenable, the high-resolution crystal structures were convergent on a common binding mode for different substituents, the substituents themselves are distal and often unlikely to disturb the conformation of the remaining molecule, there is no expected change in formal charge of the ligands, and all perturbations took place in only one region of the molecule. Thus, many reasons combine to make this an ideal system for studying with RBFE calculations.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a></div><div class="NLM_p">Finally for the calculations, it is worthwhile to consider the value and impact that accurate binding energy predictions can have on drug discovery. An analysis of 50,000 medicinal chemistry transformations from 30 projects revealed only 8.5% having a potency gain of >1 log unit (∼1.4 kcal/mol).<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> For a small set of 10 compounds, a prediction with 1 kcal/mol of error increases the chance a log unit improvement in activity to 36%, a ∼4-fold gain.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> As more compounds are assessed, the impact improves. This simple model is informative but has various caveats, for instance not all transformations are intended to improve potency. More recent analysis for a prototypical drug discovery program has suggested a 2- to 4-fold increase in the number of potent molecules when assessing at least five ideas for each synthesized compound.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> In this study, we synthesized 100 molecules from across a range of activity prediction. FEP correctly identified 90% (9 out of 10) of the 10 most active amongst its best ranked 10, compared to 20% (2 out of 10) from docking. Although this represents an unexpectedly good performance, it highlights the benefits for increased computational assistance in molecular design and the value of RBFE predictions compared to classical approaches.</div><div class="NLM_p">The initial systematic exploration of the HTS hit <b>3</b>, followed by the computationally guided library exploration, led to a collection of early lead candidates with PDE2A inhibition activity in the range of 0.6–24 nM (<a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>). Selectivity was improved during the optimization with later examples such as <b>39</b>, <b>42</b>, <b>43</b>, and <b>46</b> showing approximately 100-fold selectivity versus either PDE3A or PDE10A. This set of promising compounds was further profiled <i>in vitro</i> for its ADMET properties. In general, solubility was good and no relevant cytochrome P450 (CYP450) inhibition was seen for most compounds up to 10 μM. The metabolic stability was an issue for the leads that contained the tetrahydrobenzothiophene amide such as <b>18</b> and <b>31</b> with both showing high turnover in human and rat liver microsomal stability assays. Meanwhile, examples such as the benzofuran-containing lead <b>42</b> suggested the high metabolic turnover could be overcome. This compared favorably with the previous tricycle <b>1</b> that was highly metabolized (compound was undetectable after 15 min of incubation in human and rat liver microsomes) in similar assays.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a></div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. Emerging Leads from the Initial Exploration of HTS Triazolopyrimidine Hit <b>3</b><a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="." /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center" char="±">PDE2A IC<sub>50</sub> (nM)</th><th class="colsep0 rowsep0" align="center" char=".">selectivity<a class="ref internalNav" href="#t6fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">solubility (μM pH 7.4)</th><th class="colsep0 rowsep0" align="center">CYP450<a class="ref internalNav" href="#t6fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center">rat Cl<sub>int</sub><a class="ref internalNav" href="#t6fn4" aria-label="d">d</a> (μL/min/mg)</th><th class="colsep0 rowsep0" align="center">hum Cl<sub>int</sub><a class="ref internalNav" href="#t6fn4" aria-label="d">d</a> (μL/min/mg)</th><th class="colsep0 rowsep0" align="center">rat <i>t</i><sub>1/2</sub><a class="ref internalNav" href="#t6fn4" aria-label="d">d</a> (min)</th><th class="colsep0 rowsep0" align="center">hum <i>t</i><sub>1/2</sub><a class="ref internalNav" href="#t6fn4" aria-label="d">d</a> (min)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15</b></td><td class="colsep0 rowsep0" align="char" char="±">24 ± 8.5</td><td class="colsep0 rowsep0" align="char" char=".">21</td><td class="colsep0 rowsep0" align="left">n.d.</td><td class="colsep0 rowsep0" align="left">n.d.</td><td class="colsep0 rowsep0" align="left">14</td><td class="colsep0 rowsep0" align="left">19</td><td class="colsep0 rowsep0" align="left">98</td><td class="colsep0 rowsep0" align="left">72</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17</b></td><td class="colsep0 rowsep0" align="char" char="±">2.6 ± 1.1</td><td class="colsep0 rowsep0" align="char" char=".">30</td><td class="colsep0 rowsep0" align="left">105</td><td class="colsep0 rowsep0" align="left">0, -, 2, 9, 0, 0</td><td class="colsep0 rowsep0" align="left">35</td><td class="colsep0 rowsep0" align="left">32</td><td class="colsep0 rowsep0" align="left">40</td><td class="colsep0 rowsep0" align="left">43</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18</b></td><td class="colsep0 rowsep0" align="char" char="±">10 ± 5.2</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">102</td><td class="colsep0 rowsep0" align="left">69, -, 44, 16, 0, 51</td><td class="colsep0 rowsep0" align="left">>347</td><td class="colsep0 rowsep0" align="left">>347</td><td class="colsep0 rowsep0" align="left"><4</td><td class="colsep0 rowsep0" align="left"><4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31</b></td><td class="colsep0 rowsep0" align="char" char="±">3.4 ± 2.3</td><td class="colsep0 rowsep0" align="char" char=".">11</td><td class="colsep0 rowsep0" align="left">92</td><td class="colsep0 rowsep0" align="left">28, -, 31, 7, 43, 41</td><td class="colsep0 rowsep0" align="left">>347</td><td class="colsep0 rowsep0" align="left">>347</td><td class="colsep0 rowsep0" align="left"><4</td><td class="colsep0 rowsep0" align="left"><4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>39</b></td><td class="colsep0 rowsep0" align="char" char="±">24 ± 6.3</td><td class="colsep0 rowsep0" align="char" char=".">234</td><td class="colsep0 rowsep0" align="left">n.d.</td><td class="colsep0 rowsep0" align="left">n.d.</td><td class="colsep0 rowsep0" align="left">74</td><td class="colsep0 rowsep0" align="left">49</td><td class="colsep0 rowsep0" align="left">19</td><td class="colsep0 rowsep0" align="left">28</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>42</b></td><td class="colsep0 rowsep0" align="char" char="±">0.6 ± 0.3</td><td class="colsep0 rowsep0" align="char" char=".">89</td><td class="colsep0 rowsep0" align="left">n.d.</td><td class="colsep0 rowsep0" align="left">n.d.</td><td class="colsep0 rowsep0" align="left">11</td><td class="colsep0 rowsep0" align="left">14</td><td class="colsep0 rowsep0" align="left">129</td><td class="colsep0 rowsep0" align="left">98</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>43</b></td><td class="colsep0 rowsep0" align="char" char="±">1.7 ± 2.0</td><td class="colsep0 rowsep0" align="char" char=".">112</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">n.d.</td><td class="colsep0 rowsep0" align="left">26</td><td class="colsep0 rowsep0" align="left">14</td><td class="colsep0 rowsep0" align="left">55</td><td class="colsep0 rowsep0" align="left">99</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>46</b></td><td class="colsep0 rowsep0" align="char" char="±">1.3 ± 0.4</td><td class="colsep0 rowsep0" align="char" char=".">99</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">>10, >10, 9, >10, >10, >10</td><td class="colsep0 rowsep0" align="left">157</td><td class="colsep0 rowsep0" align="left">38</td><td class="colsep0 rowsep0" align="left">9</td><td class="colsep0 rowsep0" align="left">37</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup><sup>a</sup></sup><p class="last">n.d. not determined.</p></div><div class="footnote" id="t6fn2"><sup><sup>b</sup></sup><p class="last">Minimum fold-selectivity calculated with respect to either PDE3A or PDE10A.</p></div><div class="footnote" id="t6fn3"><sup><sup>c</sup></sup><p class="last">Cytochrome P450 inhibition, all compounds measured as % inhibition at 10 μM compound concentration, except <b>46,</b> which corresponds to IC<sub>50</sub> in μM. All measurements follow the same order: 3A4, 2C8, 2C9, 2C19, 2D6, 1A2.</p></div><div class="footnote" id="t6fn4"><sup><sup>d</sup></sup><p class="last">All compounds incubated at 1 μM for 60 min in human and rat liver microsomes to determine the metabolic turnover (half-life, Cl<sub>int</sub>).</p></div></div></div><div class="NLM_p">We investigated the ability of a representative set of compounds to cross the blood–brain barrier (BBB). Plasma and brain levels measured 1 h after dosing at 10 mg/kg are listed in <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a>. The tetrahydrobenzothiophene amides (<b>18</b> and <b>31</b>) and the chiral benzofuran (<b>42</b>) were less brain penetrant (B/P = 0.4–0.5) than the 4-phenyl substituted pyridine (<b>46</b>) (B/P = 1.8). The better <i>in vitro</i> clearance seen for <b>42</b> permitted p.o. dosing with higher absolute levels in plasma and brain. Meanwhile, the better B/P ratio of <b>46</b> may be explained by its lower total polar surface area (TPSA) and higher lipophilicity, as shown in <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a>. On the basis of these results, compound <b>46</b> showed promise with the best brain to plasma ratio, but further optimization would be needed to resolve metabolic clearance and increase absolute levels.</div><div class="NLM_table-wrap" id="tbl7"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 7. Brain and Plasma Levels for Compounds after 1 h of a Single Dose at 10 mg/kg in Rats</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center" char="±">plasma levels (ng/mL)<a class="ref internalNav" href="#t7fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char="±">brain levels (ng/mL)<a class="ref internalNav" href="#t7fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">B/P ratio</th><th class="colsep0 rowsep0" align="center" char="."><i>c</i> log <i>P</i><a class="ref internalNav" href="#t7fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char=".">TPSA</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18</b><a class="ref internalNav" href="#t7fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="char" char="±">679 ± 28</td><td class="colsep0 rowsep0" align="char" char="±">240 ± 1</td><td class="colsep0 rowsep0" align="char" char=".">0.4</td><td class="colsep0 rowsep0" align="char" char=".">1.9</td><td class="colsep0 rowsep0" align="char" char=".">92</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31</b><a class="ref internalNav" href="#t7fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="char" char="±">831 ± 450</td><td class="colsep0 rowsep0" align="char" char="±">346 ± 260</td><td class="colsep0 rowsep0" align="char" char=".">0.4</td><td class="colsep0 rowsep0" align="char" char=".">2.1</td><td class="colsep0 rowsep0" align="char" char=".">92</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>42</b><a class="ref internalNav" href="#t7fn4" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="char" char="±">1640 ± 57</td><td class="colsep0 rowsep0" align="char" char="±">788 ± 4</td><td class="colsep0 rowsep0" align="char" char=".">0.5</td><td class="colsep0 rowsep0" align="char" char=".">1.8</td><td class="colsep0 rowsep0" align="char" char=".">77</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>46</b><a class="ref internalNav" href="#t7fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="char" char="±">338 ± 68</td><td class="colsep0 rowsep0" align="char" char="±">596 ± 75</td><td class="colsep0 rowsep0" align="char" char=".">1.8</td><td class="colsep0 rowsep0" align="char" char=".">3.4</td><td class="colsep0 rowsep0" align="char" char=".">59</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t7fn1"><div class="footnote" id="t7fn1"><sup><sup>a</sup></sup><p class="last">Data are expressed as the mean average of at least two runs and provided with standard deviations.</p></div><div class="footnote" id="t7fn2"><sup><sup>b</sup></sup><p class="last"><i>c</i> log <i>P</i> calculated with Biobyte software.</p></div><div class="footnote" id="t7fn3"><sup><sup>c</sup></sup><p class="last">Compound dosed s.c. at 1 mg/mL in 20% HP-β-CD solution in Sprague Dawley rat.</p></div><div class="footnote" id="t7fn4"><sup><sup>d</sup></sup><p class="last">Compound dosed p.o. at 1 mg/mL in a 20% HP-β-CD solution pH 3.6–3.7 in the Long–Evans rat.</p></div></div></div><div class="NLM_p last">An <i>in vivo</i> target occupancy assay was used to evaluate PDE2 target engagement of compounds <b>42</b> and <b>46</b> in rat brain slices. A tritiated version of compound <b>1</b><a onclick="showRef(event, 'ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref27 ref28">(27,28)</a> was used as a radioactive tracer (10 μCi i.v.). In this experiment, MP-10<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> (2.5 mg/kg s.c.), a potent and selective PDE10 inhibitor, was predosed because it was found to greatly increase the signal-to-noise ratio of the tracer.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> This is understood to be due to PDE10 inhibition, leading to increased intracellular cGMP that is known to activate PDE2 through its GAF domain. Using this protocol, <b>42</b> showed 92% occupancy of PDE2 after p.o. administration (10 mg/kg suspension in 20% HP-β-CD solution). Compound <b>46</b> showed 90% PDE2 occupancy after s.c. administration (10 mg/kg suspension in 20% HP-β-CD solution).</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Chemistry</h3><div class="NLM_p">A general retrosynthesis toward targeted amide compounds<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> is outlined in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. The final compounds <b>47</b> were synthesized via amide coupling between the cyclic amines <b>48</b> and readily accessible or commercially available (hetero)aryl carboxylic acids <b>49</b>. In turn, triazolopyrimidine-amines <b>48</b> were obtained from the acetylamines <b>50</b> or from the piperidin-3-one <b>51</b> via known procedures.</div><figure id="sch1" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/medium/jm0c01272_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. General Retrosynthetic Route</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01272&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">5-Unsubstituted triazolo[1,5-<i>a</i>]pyrimidine amines <b>48a–d</b> were prepared, as shown in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. Thus, condensation of the Boc-protected acetylamines <b>50a–d</b> with <i>N</i>,<i>N</i>-dimethylformamide dimethylacetal (DMF/DMA) provided the intermediate 3-dimethylaminobuten-2-ones <b>52a–d</b>, which were then cyclized with 1<i>H</i>-1,2,4-triazol-3-amine (<b>53</b>) in acetic acid at 100 °C, to provide triazolopyrimidines <b>54a–d</b>. Cleavage of the Boc-protecting group in <b>54a–d</b> gave the desired intermediates <b>48a–d</b>. Optionally, intermediates <b>48a–b</b> were also separated in their enantiomers via supercritical fluid chromatography (SFC) purification to provide intermediates <b>48aa</b> (<i>R</i>)–<b>48ab</b> (<i>S</i>) and <b>48ba</b> (<i>R</i>)–<b>48bb</b> (<i>S</i>), respectively.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a></div><figure id="sch2" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/medium/jm0c01272_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0010.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Intermediates <b>48a</b>–<b>d</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01272&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) DMF/DMA, 120 °C, 3 h; (b) AcOH, 100 °C, 1 h, 28–62% over 2 steps; (c) HCl, 1,4-dioxane, RT, 1 h, 70–93%.</p></p></figure><div class="NLM_p">The synthesis of 5-substituted triazolopyrimidine amines <b>48e–f</b> commenced with the condensation of 3-acetylpiperidine <b>50a</b> with carbon disulfide, followed by S-methylation using methyl iodide and sodium hydride, which gave the bismethylsulfanyl butenone intermediate <b>55</b> (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>). Cyclization of <b>55</b> with the aminotriazole <b>53</b> provided the 5-thiomethyl-triazolopirimidine <b>56</b>. Oxidation of the thiomethyl substituent in <b>56</b> with mCPBA gave the corresponding sulfone <b>57</b>, which was then converted via aromatic nucleophilic substitution with sodium methoxide and sodium cyanide to the respective methoxy (<b>48e</b>) and cyano (<b>48f</b>) triazolopyrimidine intermediate amines after Boc deprotection.</div><figure id="sch3" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/medium/jm0c01272_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0011.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Intermediates <b>48e</b>–<b>f</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01272&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NaH, CS<sub>2</sub>, CH<sub>3</sub>I; THF, 0 °C—RT, 16 h, 21%; (b) 150 °C, 3 h, 12%; (c) mCPBA, CHCl<sub>3</sub>, 60 °C, 2 h, 99%; (d) R = OMe: NaOMe, MeOH, RT, 30 min; (e) R = CN: NaCN, DMSO, RT, 30 min, 43%; (f) TFA, DCM, RT, 2 h; R = OMe: 99%, R = CN: 68%.</p></p></figure><div class="NLM_p">The synthetic sequence for the preparation of intermediates <b>48g</b>–<b>h</b> is shown in <a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a> and was similar to <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a> for the synthesis of intermediates <b>48a–d</b>. Condensation of Boc-protected 3-acetylpiperidine <b>50a</b> with <i>N</i>,<i>N</i>-dimethylacetamide dimethyl acetal afforded the enaminone <b>58</b>, which was then reacted with either 1<i>H</i>-imidazol-2-amine or with <b>53</b> to provide the corresponding 5- or 7-methyl-substituted [1,2,4]triazolo[1,5-<i>a</i>]pyrimidine (<b>48g</b>) or imidazo[1,2-<i>a</i>]pyrimidine (<b>48h</b>) intermediate amines. The 5-(difluoromethyl)-7-(3-piperidyl)-[1,2,4]triazolo[1,5-<i>a</i>]pyrimidine amine derivative <b>48i</b> was obtained in three steps starting from <b>50a</b>. First, the condensation of <b>50a</b> with ethyl difluoroacetate afforded the 1,3-dicarbonyl intermediate <b>59</b> in excellent yield. Then, reaction of <b>59</b> with <b>53</b> led to intermediate <b>48i</b> after Boc cleavage.</div><figure id="sch4" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/medium/jm0c01272_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0012.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Intermediates <b>48g</b>–<b>i</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01272&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 120 °C, 3 h, 95%; (b) AcOH, 120 °C, 30 min, for 53: 80%, for 2-aminoimidazole: 55%; (c) C<sub>6</sub>H<sub>6</sub>, KOtBu, 0 °C—RT, 5 h, 98%; (d) (i) <b>53</b>, DMF, 60 °C, 28 h; (ii) Et<sub>3</sub>N, di-<i>tert</i>-butyl dicarbonate, 0 °C—RT, 1 h, 86%; (e) HCl, iPrOH, MeOH, 1–24 h, RT (<b>48g</b>, 55%; <b>48h</b>, 95%; <b>48i</b>, 61%).</p></p></figure><div class="NLM_p">For the preparation of the piperidine intermediates <b>48j</b>–<b>o</b>, the boronic ester intermediates <b>62</b> and <b>63</b> were prepared from Boc-protected piperidine3-one <b>51,</b> as shown in <a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>. First, <b>51</b> was treated with LDA in the presence of <i>N</i>-phenyl-bis(trifluoromethanesulfonimide), affording a mixture of triflate enol ethers <b>60</b> and <b>61</b> that could be separated by column chromatography on a silica gel. Both compounds were converted to the corresponding boronate esters <b>62</b> and <b>63</b> in moderate yields via palladium-catalyzed reaction with bis(pinacolato)diboron.</div><figure id="sch5" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/medium/jm0c01272_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0013.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of 3,4- and 3,6-Dihydropyridyl Pinacol Boronic Ester Intermediates<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01272&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) LDA, THF, −78 °C—RT, 16 h; (b) AcOK, Pin<sub>2</sub>B<sub>2</sub>, Pd(OAc)<sub>2</sub>, PCy<sub>3</sub>, 2-MeTHF, 80 °C, 16 h.</p></p></figure><div class="NLM_p">Boronic ester <b>62</b> was used in Suzuki cross-coupling reactions with chloro-triazolopyrimidines <b>64</b> and tetrakis(triphenylphosphine)palladium(0) to give tetrahydropyridines <b>65a–d</b> in modest yields because of the instability of the chloro-heteroaromatic derivatives <b>64</b>, which hydrolyzed in the reaction conditions. The double bond in derivatives <b>65a–d</b> was satisfactorily reduced under catalytic hydrogenation conditions, leading to the corresponding <i>N</i>-Boc-protected piperidines using either Pd/C or Pt/C as catalysts. Boc removal under acidic conditions provided piperidines <b>48j–m</b> in nearly quantitative yields. In a similar fashion, the boronic ester <b>63</b> was coupled to 4-chloroazaindole (<b>66</b>) and chloro pyrazolopyrimidine (<b>68</b>) using the <i>N</i>-heterocyclic carbene complex allyl[1,3-bis(2,6-diisopropylphenyl)imidazol-2-ylidene]chloropalladium(II) to provide tetrahydropyridines <b>67</b> and <b>69</b>, respectively. Catalytic reduction of the double bond in <b>67</b> and <b>69</b>, followed by Boc cleavage afforded the piperidines <b>48n</b> and <b>48o</b>, respectively (<a class="ref internalNav" href="#sch6" aria-label="Scheme 6">Scheme 6</a>).</div><figure id="sch6" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/medium/jm0c01272_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0014.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis of Intermediates <b>48j</b>–<b>48o</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01272&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Pd(PPh<sub>3</sub>)<sub>4</sub>, Na<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane, EtOH, water, (9–27%). (b) H<sub>2</sub>, Pd/C, MeOH, (R<sup>1</sup> = iPr, R<sup>2</sup> = H, 99%) or Pt/C (47%–88%); (c) HCl, iPrOH. MeOH, RT, 16 h (94–100%); (d) (SIPr)Pd(allyl)Cl, Na<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane, EtOH, water (18%); (e) H<sub>2</sub>, Pt/C, MeOH (99%); (f) HCl, iPrOH. MeOH, RT, 16 h,91%; (h) (SIPr)Pd(allyl)Cl, Na<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane, EtOH, water,63%; (i) H<sub>2</sub>, Pd/C, MeOH, (R<sup>1</sup> = iPr, R<sup>2</sup> = H, 86%); (j) HCl, iPrOH. MeOH, RT, 16 h, 94%.</p></p></figure><div class="NLM_p">The piperidine intermediates <b>48a–o</b> were transformed into the final targets via amide coupling reaction conditions with the corresponding carboxylic acids of acid chlorides (<a class="ref internalNav" href="#sch7" aria-label="Scheme 7">Scheme 7</a>).</div><figure id="sch7" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Scheme 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/medium/jm0c01272_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0015.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 7. Synthesis of Compounds <b>3</b>, <b>7</b>–<b>18</b>, <b>21</b>–<b>36</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01272&amp;id=sch7"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: for compounds <b>3</b>–<b>7</b>, <b>10</b>–<b>12</b>, <b>14</b>–<b>18</b>, and <b>21</b>–<b>36</b> (a) (Het)ArCOOH, HBTU, DIPEA, DCM, 1 h, 15–95%; for compounds <b>8</b>, <b>9</b>, and <b>13</b> (b) (Het)ArCOCl, DIPEA, DCM, RT, 1 h, 11–80%.</p></p></figure><div class="NLM_p">Piperidines bearing benzyl-like substituents at the nitrogen atom <b>37</b>, <b>38</b>, and <b>40</b> were prepared by reductive amination with sodium triacetoxyborohydride of difluoromethyltriazolopyrimidine <b>48i</b> with readily available aldehydes <b>70</b> (<a class="ref internalNav" href="#sch8" aria-label="Scheme 8">Scheme 8</a>). Alternatively, substitution-alkylation of <b>48i</b> (<a class="ref internalNav" href="#sch7" aria-label="Scheme 7">Scheme 7</a>) with the derivative <b>71</b> provided the benzylamine <b>39</b> in 37% yield.</div><figure id="sch8" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Scheme 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/medium/jm0c01272_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0016.jpeg" id="gr15" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 8. Synthesis of Compounds <b>37</b>–<b>40</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01272&amp;id=sch8"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NaBH(OAc)<sub>3</sub>, DIPEA, DCM, RT, 4 h (36%–86%); (b) DIPEA, DMF, 60 °C, 4 h (37%).</p></p></figure><div class="NLM_p">Finally, the target compounds containing a trisubstituted piperidine were prepared following the synthetic route outlined in <a class="ref internalNav" href="#sch9" aria-label="Scheme 9">Scheme 9</a>. Thus, catalytic hydrogenation of commercially available methyl 4-methylpyridine-3-carboxylate <b>72</b> using PtO<sub>2</sub> in acetic acid (<a class="ref internalNav" href="#sch9" aria-label="Scheme 9">Scheme 9</a>), followed by Boc-protection of the resulting piperidine, provided methyl 4-methylpiperidine-3-carboxylateas a cis/trans mixture of diastereoisomers, with trans isomer <b>73</b> being the major component in the mixture. The desired diastereoisomer <b>73</b> could be separated by column chromatography. Saponification of the methyl ester in <b>73</b>, followed by CDI-mediated coupling with methoxymethylamine, provided Weinreb amide <b>74</b>. Addition of methylmagnesium bromide to amide <b>74</b> led to the methylketone <b>75</b>, which was then condensed with difluoroethylacetate to afford the 1,3-dicarbonylic compound <b>76</b>. In turn, <b>76</b> was reacted with 2-aminotriazole <b>53</b> to provide the triazolopyrimidine intermediate <b>77</b>. The Boc N-protecting group was partially cleaved in the cyclization reaction conditions; therefore, the crude reaction mixture (RM) was treated with Boc<sub>2</sub>O to reprotect the deprotected intermediate in order to enable a better separation of <b>77</b> from the reaction mixture. Deprotection of the Boc N-protecting group under acidic conditions provided piperidine <b>78</b>. SFC purification at this stage allowed isolation of the eutomer of <b>78</b>, which was then submitted to amide coupling or to nucleophilic aromatic substitution conditions to provide the target molecules <b>41</b>–<b>43</b> and <b>44</b>–<b>46</b>, respectively.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a></div><figure id="sch9" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Scheme 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/medium/jm0c01272_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0017.jpeg" id="gr16" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 9. Synthesis of Compounds <b>41</b>–<b>43</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01272&amp;id=sch9"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) H<sub>2</sub>, PtO<sub>2</sub>, AcOH, 50 °C, 5 d, 97%; (b) Boc<sub>2</sub>O, DCM, DIPEA, DMAP, RT, 1 h, 75%; (c) NaOH, MeOH, RT, 14 h, 100%; (d) CDI, MeNHOMe, Et<sub>3</sub>N, THF, ACN, RT—80 °C, 7 d,100%; (e) MeMgBr, THF, −15 °C—RT, 1 h, 90%; (f) KOtBu, toluene, 0 °C—RT, 2 h, 100%; (g) (i) <b>53</b>, DMF, 80 °C, 24 h; (ii) Boc<sub>2</sub>O, Et<sub>3</sub>N, RT, 30 min, 30%; (h) HCl, iPrOH, MeOH, 2 h, RT (72%); (i) (Het)ArCOOH, HBTU, DIPEA, DCM, 1 h, 15–95%; (j) (Het)Cl or (Het)Br, DIPEA, <i>n</i>BuOH, 160 °C, 4 h (6–41%).</p></p></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i24">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_23750" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_23750" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">The initial exploration of a triazolopyrimidine PDE2A inhibitor HTS hit (<b>3</b>) has been described. Guided by the X-ray structure and later with FEP relative binding energy calculations, we were able to rapidly identify highly potent PDE2A inhibitors with greater than 100-fold selectivity over PDE3A and PDE10A. Furthermore, several examples (<b>39</b>, <b>42</b>, and <b>46</b>) demonstrated that selectivity versus other members of the PDE enzyme family was resolved by targeting the Q2 pocket. Racemization issues were overcome by the introduction of a methyl substituent in the central core, leading to an optimal trisubstituted piperidine scaffold. Four lead compounds were evaluated to test BBB penetration with <b>42</b> and <b>46</b> showing most promise, and <i>in vivo</i> PDE2 target engagement was confirmed for these two compounds via p.o. and s.c. administration, respectively. The FEP calculations were useful for identifying unexpectedly active compounds from amongst the virtual libraries that were profiled. We consider the approach of FEP calculations on increasingly larger virtual libraries and chemical spaces to offer significant potential for future LO efforts.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> It provides a way to identify stand-out or unexpectedly active compounds, thereby having a larger impact on the LO process. Finally, despite the good results here, it is important to note that FEP can be problematic in real-world prospective applications as described in the recent large report of Schindler et al.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> We will follow up with a detailed report on the optimization of leads from this series.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i25">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_05755" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_05755" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> X-ray Crystallography</h3><div class="NLM_p last">Cocrystals of C-His tagged human 579-921 PDE2A with HTS hit <b>3</b> and lead <b>39</b> were grown in 0.2 M MgCl<sub>2</sub>, 0.1 M Tris-HCl pH 8.0 and 20% PEG 3350 and 0.2 M MgCl<sub>2</sub>, 0.1 M Tris-HCl pH 8.0 and 24% PEG 3350, respectively, both in the presence of 1 mM compound. Crystals were flash-frozen directly from the hanging drop without additional cryo-protectant by plunging into liquid nitrogen. The HTS hit <b>3</b> data set was collected in house on a Rigaku Saturn 944 CCD detector. The lead <b>39</b> data set was collected at the Diamond Light Source at beamline 102 on an ADSC Q315 CCD detector. Data sets were indexed, integrated, and scaled using MOSFLM and SCALA (CCP4). The PDE2A model with PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1Z1L">1Z1L</a> was used for molecular replacement with the data sets using PHASER (CCP4). The resulting models were then given 10 cycles of atomic refinement with tight geometric weights using REFMAC5. In each case, the difference electron density map calculated after molecular replacement and initial refinement was examined for the presence of possible ligands. Once the ligand electron density was located, a molecular structure file and refinement library file were produced using SKETCHER (CCP4). The ligands were fitted into the electron density using COOT and refined using REFMAC5 (CCP4). Water molecules were then added using the water placement option in COOT and refined using REFMAC5 (CCP4). Metal ions were also added and refined. The structural geometry of both the protein and ligand were finally checked using Maestro and MOE.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> Protocols for Measuring Inhibition of PDE’s <i>In Vitro</i></h3><div class="NLM_p">PDEs 1B1 and 11A4 were expressed in HEK cells from full-length human recombinant clones. Human recombinant PDEs 2A, 4D3, 5A3, 7A1, 9A1, 10A2, and rat PDE10A2 were expressed in Sf9 cells using a recombinant baculovirus construct containing the full-length sequence containing a a 6xHis sequence, following the start Met to allow metal affinity purification of the recombinant protein. Cells were harvested, and the PDE protein was purified by metal chelate chromatography on Ni-sepharose 6FF. PDE6A, PDE3A, and PDE8A were purchased as partially purified Sf9 cell lysates (Scottish Biomedical, UK). All enzymes were diluted in 50 mM Tris pH 7.8, 1.7 mM EGTA, and 8.3 mM MgCl<sub>2</sub> except for PDE9A, which was diluted in 50 mM Tris pH 7.8 and 5 mM MnCl<sub>2</sub>, and PDE1B was diluted in 50 mM Tris pH 7.8, 8.3 mM MgCl<sub>2</sub> complemented with 624 U/mL calmodulin and 800 μM CaCl<sub>2</sub>. The affinity of the compounds for PDEs was measured by a scintillation proximity assay (SPA). PDE yttrium silicate SPA beads allow the PDE activity to be measured by direct binding of the primary phosphate groups of noncyclic AMP or GMP to the beads via an iron complex chelation mechanism. The amount of bound tritiated product ([<sup>3</sup>H]-AMP or [<sup>3</sup>H]-GMP) was measured by liquid scintillation counting in a TopCount (Packard).</div><div class="NLM_p last">Compounds were dissolved and diluted in 100% dimethyl sulfoxide (DMSO) in polystyrene plates to a concentration of 100-fold the final concentration in the assay. Rat or human PDE10A or human PDE2A or human PDE3A or PDE3B enzyme solution (10 μL) was added to 20 μL of incubation buffer (50 mM Tris pH 7.8, 8.3 mM MgCl<sub>2</sub>, 1.7 mM EGTA), 10 μL of substrate solution consisting of a mixture of nontritiated and tritiated substrate (60 nM cAMP, 0.008 μCi <sup>3</sup>H-cAMP for human and rat PDE10A, 10 μM cGMP, 0.01 μCi <sup>3</sup>H-cGMP for PDE2A, 0.1 μM cAMP, 0.024 μCi <sup>3</sup>H-cAMP for PDE3A or PDE3B), and 0.4 μL of compound in 100% DMSO in a 384-well plate and incubated for 60 min at room temperature for human and rat PDE10A and PDE3A/B or for 40 min at room temperature for PDE2A. After incubation, the reaction was stopped with 20 μL of stop solution, consisting of PDE SPA beads (17.8 mg beads/mL in 18 mM zinc sulfate for human and rat PDE10A or in 200 mM zinc sulfate for PDE2A and PDE3A, and in 400 mM zinc sulfate for PDE3B). After sedimentation of the beads for 30 min, the radioactivity was measured in a PerkinElmer TopCount scintillation counter and results were expressed as counts per minute (cpm). To measure the low control, no enzyme was added to the reaction mixture. The same assay principle was applied for the measurement of the inhibition of other members of the PDE family, with appropriate modifications of enzyme concentration, incubation buffer, substrate solution, incubation time, and stop solution. Data were calculated as the percentage of inhibition of total activity measured in the absence of test compound (% control). A best-fit curve was fitted by a minimum sum of squares method to the plot of % control versus compound concentration, from which an IC<sub>50</sub> value (inhibitory concentration causing 50% inhibition of hydrolysis) was obtained. Compounds <b>1</b>–<b>40</b> were tested on rat PDE10A, and <b>41</b>–<b>46</b> were tested on human PDE10A.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> CYP450 Inhibition Assay</h3><div class="NLM_p last">The potential to reversibly inhibit the major human P450 isoforms (CYP450s 1A2, 2C9, 2C19, 2D6, and 3A4) was determined in the CYP450 inhibition cocktail assay. Test compounds are incubated across a concentration range (five concentrations) with human liver microsomes and probe substrates for each of the six isoforms (CYP450 1A2, 2C8, 2C9, 2C19, and 2D6 in cocktail format and 3A4 in the single probe format) to estimate the IC<sub>50</sub> for inhibition of the probe substrate by the test compound. After the defined incubation time, the reaction is quenched and samples are centrifuged and prepared for analysis by liquid chromatography/mass spectrometry (LC/MS). The percentage inhibition of the probe substrate metabolite formation is plotted against inhibitor concentration and an IC<sub>50</sub> calculated by curve fitting.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> Microsomal Metabolic Stability Assay</h3><div class="NLM_p last">The metabolic stability of a test compound was tested at Cyprotex by using liver microsomes (0.5 mg/mL protein) from human and preclinical species incubated up to 60 min at 37 °C with 1 μM of the test compound. The <i>in vitro</i> metabolic half-life (<i>t</i><sub>1/2</sub>) is calculated using the slope of the log–linear regression from the percentage parent compound remaining versus time relationship. The <i>in vitro</i> intrinsic clearance (Cl<sub>int</sub>) (mL/min/mg microsomal protein) is derived from the half-life taking into account the incubation volume and the weight of microsomal protein in the incubation.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> <i>In Vivo</i> PDE2 Occupancy Method</h3><div class="NLM_p last">Male Wistar rats were treated by systemic administration (s.c. or p.o.) of vehicle or single dose of selective PDE2A inhibitors (<i>n</i> = 3 per dose). After 15 min, every rat received a s.c. injection of the PDE10 inhibitor MP-10 at the dose of 2.5 mg/kg, followed 55 min later by an i.v. injection of the selective PDE2A radioligand [<sup>3</sup>H]-<b>1</b>.<a onclick="showRef(event, 'ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref27 ref28">(27,28)</a> Dosing rats with MP-10,<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> a potent and selective PDE10 inhibitor, prior to the radioligand injection allowed to increase the signal to noise ratio of [<sup>3</sup>H]-<b>1</b> because its binding is increased when intracellular cGMP concentration increased.<a onclick="showRef(event, 'ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref27 ref28">(27,28)</a> Rats were sacrificed 5 min after the tracer injection, and their brains were dissected and frozen. Then, 20 μm of thick coronal sections were cut using a cryostat, collected on glass slides, and dried. Brain sections were loaded in a β-imager for 12 h. The specific binding was determined as the difference between [<sup>3</sup>H]-<b>1</b> binding quantified in the striatum (a brain area showing a high density of PDE2A) and in the cerebellum (a brain area where PDE2A is virtually absent). Occupancy was calculated as the inhibition of specific [<sup>3</sup>H]-<b>1</b> binding in drug-treated animals relative to vehicle-treated animals. The percentage of PDE2A occupancy was measured at one dosage.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> Computational Docking Procedure</h3><div class="NLM_p last">Docking used the Glide software from Schrödinger Inc. The corresponding crystal structure solved with <b>39</b> was used. The protein was prepared using the protein preparation wizard<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> in Maestro.<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> Hydrogen-bond constraints were enforced with Gln812 and Gln859 to ensure accurate placement of the triazolopyrimidine scaffold. All molecules were prepared for docking using the LigPrep tool. All default settings were used, and ionization states were manually checked. The ligands were parameterized for use with the OPLSv2.1 force field up front using the tools available in Maestro. The Glide XP scoring function was used with expanded sampling, all other docking parameters were set to the defaults. Results were visually inspected to ensure that each molecule adopted the desired binding mode consistent with the crystal structure.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> Computational FEP Procedure</h3><div class="NLM_p last">All FEP calculations were conducted using the Schrodinger molecular modeling suite version 2015-1, the implementation is often referred to as FEP+. Calculations were set up using the mapper technology to define the perturbations. Default protocols were used with a 5 ns simulation length for ligands both in complex and in solution. As mentioned above, the FEP calculations were performed with the molecules in the chosen docked pose. Molecules were treated in a neutral form and missing OPLS force-field parameters were calculated and fitted up front using the force-field builder tool. The results of the simulations with 5 ns simulation time and 12 λ windows were used to define the molecules recommended for synthesis in the prospective application. The experimental pIC<sub>50</sub> were converted to expected binding free energies at 298 K using the relation <i>RT</i> ln(IC<sub>50</sub>); any offset with respect to binding free energy cancels when comparing relative differences. The calculations deliver the RBFE differences between compounds, and the resulting δδ<i>G</i>’s were fitted to the mean of the experimental δ<i>G</i> to provide the calculated δ<i>G</i> with reduced offset. MUEs are reported by comparing experimental and calculated δ<i>G</i> (kcal/mol) and shown with the corresponding ±99% confidence interval. The force field available at the time was OPLSv2.1. To compare with newer versions, the predicted Δ<i>G</i> for the retrospective dataset was recalculated using Schrodinger suite 2019–3 and OPLSv3e. A high degree of correlation was seen with OPLSv2.1, R 0.9. Input structures for FEP calculations are provided in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c01272/suppl_file/jm0c01272_si_001.pdf" class="ext-link">Supporting Information</a>.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> Experimental Procedures and Compound Characterization</h3><div class="NLM_p">All reactions were carried out by employing standard chemical techniques under an inert atmosphere. Solvents used for extraction, washing, and chromatography were of high-performance liquid chromatography (HPLC) grade. Unless otherwise noted, all solvents and chemicals used were of reagent grade. Anhydrous solvents tetrahydrofuran (THF), DCM, and DMF were purchased from Sigma-Aldrich or Acros. All final compounds were characterized by <sup>1</sup>H NMR and LC/MS. <sup>1</sup>H nuclear magnetic resonance spectra were recorded on Bruker spectrometers: DPX-360 MHz, DPX-400 MHz, Bruker Avance Neo spectrometer operating at 400 MHz, or on a Bruker Avance III-HD spectrometer operating at 600 MHz using chloroform-<i>d</i> (deuterated chloroform, CDCl<sub>3</sub>) or DMSO-<i>d</i><sub>6</sub> (deuterated DMSO, dimethyl-<i>d</i><sub>6</sub> sulfoxide) as solvents. Chemical shifts (d) are reported in parts per million relative to tetramethylsilane, which was used as the internal standard.</div><div class="NLM_p">Elemental analyses were performed with a Carlo-Erba EA1110 analyzer. Thin-layer chromatography (TLC) was carried out on silica gel 60 F254 plates (Merck) using reagent grade solvents. Open column chromatography was performed on a silica gel, mesh 230–400 particle size and 60 Å pore size (Merck) under standard techniques. Automated flash column chromatography was performed using ready-to-connect cartridges from Merck, on an irregular silica gel, particle size 15–40 pm (normal phase disposable flash columns) on a SPOT or LAFLASH system from Armen Instrument. All final compounds were confirmed to be >95% pure via HPLC methods. All the LC/MS analyses were performed using an Agilent G1956A LC/MS quadrupole coupled to an Agilent 1100 series LC system consisting of a binary pump with a degasser, autosampler, thermostated column compartment, and diode array detector. The mass spectrometer was operated with an atmospheric pressure electrospray ionization source in the positive ion mode. The capillary voltage was set to 3000 V, and the fragmentor voltage was set to 70 V, and the quadrupole temperature was maintained at 100 °C. The drying gas flow and temperature values were 12.0 L/min and 350 °C, respectively. Nitrogen was used as the nebulizer gas at a pressure of 35 psig. Data acquisition was performed with Agilent ChemStation software. Analyses were carried out on a YMC pack ODS-AQ C18 column (50 mm long × 4.6 mm I.D.; 3 μm particle size) at 35 °C, with a flow rate of 2.6 mL/min. A gradient elution was performed from 95% (water + 0.1% formic acid)/5% acetonitrile to 5% (water + 0.1% formic acid)/95% acetonitrile in 4.8 min; the resulting composition was held for 1.0 min; from 5% (water + 0.1% formic acid)/95% acetonitrile to 95% (water + 0.1% formic acid)/5% acetonitrile in 0.2 min. The standard injection volume was 2 μL. Acquisition ranges were set to 190–400 nm for the UV-PDA detector and 100–1400 <i>m</i>/<i>z</i> for the MS detector. Optical rotations measurements were carried out on a 341 PerkinElmer polarimeter in the indicated solvents. Melting points were determined with a DSC823e (Mettler Toledo) and were measured with a temperature gradient of 30 °C/min. The reported values are peak values. Purities of all new compounds were determined by analytical reverse phase (RP)-HPLC using the area percentage method on the UV trace recorded at a wavelength of 254 nm, and compounds were found to have ≥95% purity unless otherwise specified.</div><div class="NLM_p">The absolute stereochemical configuration for some of the compounds was determined using vibrational circular dichroism (VCD). They were measured on a Bruker Equinox 55 equipped with a PMA 37 in a KBr liquid cell using CD<sub>2</sub>CI<sub>2</sub> as the solvent (PEM: 1350cm<sup>–l</sup>, LIA: l mV, resolution: 4 cm<sup>–1</sup>). A description on the use of VCD for the WO 2018/083101 PCT/EP2017/077918-29-5 10 15 20 25 30 determination of absolute configuration can be found in Dyatkin A.B. et al., <i>Chirality, 14,</i> 215–219 (2002). Ab initio calculations: A thorough conformational search was performed at molecular mechanics level using a Macromodel to do a mixed torsional/low-mode sampling with the OPLS-2005 force field. The located minima were optimized using Jaguar at the B3LYP/6-31G** level with a Poisson–Boltzmann continuum solvation model to mimic a dichloromethane solvent. All conformations within 10 kJ/mol interval were used to simulate VCD and IR spectra. Dipole and rotational strengths were calculated at the same B3LYP/6-31G** level using Jaguar. The calculated VCD spectra, generated after scaling the frequencies with a factor of 0.97, converting to a Lorentzian bandshape, and summing up the contribution of each conformer assuming a Boltzmann ensemble, were visually compared with the experimental spectra for assigning the correct stereo chemistry.</div><div id="sec4_8_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> Intermediate <b>52b</b>, <i>tert</i>-Butyl 2-[3-(Dimethylamino)prop-2-enoyl]morpholine-4-carboxylate</h4><div class="NLM_p last">A stirred mixture of <i>t</i>-butyl 2-acetylmorpholine-4 carboxylate (25 g, 0.11 mol) in <i>N</i>,<i>N</i>-dimethylacetamide dimethyl acetal (58 mL) was heated at 100 °C for 16 h. The reaction mixture was concentrated <i>in vacuo</i> to give the title compound (30 g), which was used in the next step without further purification.</div></div><div id="sec4_8_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> Intermediate <b>52c</b>, <i>tert</i>-Butyl 3-[3-(Dimethylamino)prop-2-enoyl]azepane-1-carboxylate</h4><div class="NLM_p last">Prepared with the same procedure for <b>52b</b>, starting from <i>tert</i>-butyl 3-acetylazepane-1-carboxylate [1782629-29-1] (18.2 g) to give to give the title compound, (22 g), which was used in the next step without further purification.</div></div><div id="sec4_8_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> Intermediate <b>52d</b>, Di-<i>tert</i>-butyl 2-Acetylpiperazine-1,4-dicarboxylate, Prepared in Three Steps</h4><div class="NLM_p last">Step 1, intermediate <b>79</b>, di-<i>tert</i>-butyl 2-[methoxy(methyl)carbamoyl]piperazine-1,4-dicarboxylate. To a suspension of 1,4-bis[(<i>t</i>-butoxy)carbonyl]piperazine-2-carboxylic acid [173774-48-6] (60 g, 0.18 mol) in THF (285 mL) was added di-1<i>H</i>-imidazol-1-yl-methanone (CDI, 40 g, 0.25 mol), and the mixture was stirred at RT for 2 h. In another flask, to a suspension of <i>N</i>-methoxy-methanamine hydrochloride (24.45 g, 0.25 mol) in ACN (190 mL) was added trimethylamine (36.35 mL, 0.26 mol). Both mixtures were combined and stirred at RT for 16 h. The solvents were evaporated. The residue was dissolved in DCM and washed with water, 20% AcOH solution, and finally an aqueous NaOH (1 N) solution. The organic layer was dried over MgSO<sub>4</sub>, filtered, and evaporated. The product was purified by column chromatography (silica gel, eluent: a gradient DCM 100 to 3% MeOH in DCM). The pure fractions were evaporated, yielding intermediate <b>79</b> (56.8 g, 83.7% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 1.37 (d, <i>J</i> = 6.06 Hz, 18H), 2.86–2.96 (m, 1H), 3.06–3.12 (m, 3H), 3.22–3.34 (m, 1H), 3.39–3.55 (m, 1H), 3.59–3.63 (m, 1H), 3.73 (s, 3H), 3.76–3.84 (m, 1H), 4.11–4.19 (m, 1H), 4.70 (br d, <i>J</i> = 2.83 Hz, 1H). Step 2, intermediate <b>80</b>, di-<i>tert</i>-butyl 2-acetylpiperazine-1,4-dicarboxylate. Intermediate <b>77</b> (56.8 g, 0.15 mol) in THF (200 mL) was charged in a flask under N<sub>2</sub> and cooled to 0 °C. Methylmagnesium bromide (265 mL, 1.4 M in toluene/THF 75/25) was added dropwise, with the temperature not exceeding 15 °C. After addition, the reaction mixture was stirred at RT for 1 h. Then, the reaction mixture was poured on ice with 100 mL of AcOH. The product was extracted with Et<sub>2</sub>O, and the organic layer was washed with a 5% NaHCO<sub>3</sub> solution. The organic layer was dried over MgSO<sub>4</sub>, filtered, and evaporated to give intermediate <b>80</b> (36.6 g, 73%), used as such in the next step. Gas chromatography–MS <i>m</i>/<i>z</i> 328 (M). Step 3, intermediate <b>52d</b>, di-<i>tert</i>-butyl 2-acetylpiperazine-1,4-dicarboxylate. That was prepared with the same procedure as <b>52b</b>, from di-<i>tert</i>-butyl 2-acetylpiperazine-1,4-dicarboxylate (intermediate <b>78</b>) (10 g) to give the title compound (11 g, 94%). MS (ESI) <i>m</i>/<i>z</i>: 384 [M + H]<sup>+</sup>.</div></div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> Representative Procedure for the Preparation of Intermediates <b>54a</b>–<b>54d</b></h3><div id="sec4_9_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> Intermediate <b>54a</b>, <i>tert</i>-Butyl 3-([1,2,4]Triazolo[1,5-<i>a</i>]pyrimidin-7-yl)piperidine-1-carboxylate</h4><div class="NLM_p last">Intermediate <b>52a</b> (33.0 g, 0.12 mol), 1<i>H</i>-1,2,4-triazol-5-amine hydrochloride (1:1) (12.3 g, 0.14 mol), and AcOH (75 mL) were stirred at 130 °C for 1 h. The reaction mixture was cooled on an ice bath and left under stirring overnight at RT. The reaction mixture was concentrated <i>in vacuo</i>, co-evaporated with toluene, diluted with Et<sub>2</sub>O (0.7 L), and then poured onto ice (0.5 L). The layers were separated, and the aqueous layer was back extracted with Et<sub>2</sub>O (3 × 0.3 L). The combined organic layers were treated with brine, dried over MgSO<sub>4</sub>, filtered, and evaporated to give the title compound (racemic) as a brown oil (25 g, 70.5%). MS (ESI) <i>m</i>/<i>z</i>: 304 [M + H]<sup>+</sup>. This material could be used as such. Alternatively, <b>54a</b> was also separated into enantiomers <b>54aa</b> and <b>54ab</b> by purification by Prep SFC (stationary phase: CHIRALPAK Daicel OJ 20 × 250 mm; mobile phase: CO<sub>2</sub>, MeOH) yielding intermediate <b>54aa</b> (<b>R!</b>, 10 g , 28%, <i>R</i><sub>t</sub> = 5.59 min) and intermediate <b>54ab</b> (<b>S!</b>, 10 g, 28%, <i>R</i><sub>t</sub> = 7.4 min).</div></div><div id="sec4_9_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> Intermediate <b>54b</b>, <i>tert</i>-Butyl 2-([1,2,4]Triazolo[1,5-<i>a</i>]pyrimidin-7-yl)morpholine-4-carboxylate</h4><div class="NLM_p last">Prepared with the same procedure for <b>54a</b>, starting from intermediate <b>52b</b> (30 g, 0.11 mol) to give the title compound <b>54b</b> (racemic mixture, 19 g, 62%) after purification on a silica gel (eluent: DCM, 1% MeOH in DCM, 2%, 4%). MS (ESI) <i>m</i>/<i>z</i>: 305 [M + H]<sup>+</sup>.</div></div><div id="sec4_9_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> Intermediate <b>54c</b>, <i>tert</i>-Butyl 3-([1,2,4]Triazolo[1,5-<i>a</i>]pyrimidin-7-yl)azepane-1-carboxylate</h4><div class="NLM_p last">Prepared with the same procedure for <b>54a</b>, starting from intermediate <b>52c</b> (5.0 g, 16.87 mmol) to give 1.69 g of the title compound <b>54c</b> (racemic mixture, 31.56%) after purification on a silica gel (eluent: DCM, 1% MeOH in DCM, 2%). MS (ESI) <i>m</i>/<i>z</i>: 318 [M + H]<sup>+</sup>.</div></div><div id="sec4_9_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> Intermediate <b>54d</b>, Di-<i>tert</i>-butyl 2-([1,2,4]Triazolo[1,5-<i>a</i>]pyrimidin-7-yl)piperazine-1,4-dicarboxylate</h4><div class="NLM_p last">Prepared with the same procedure for <b>54a</b>, starting from intermediate <b>52d</b> (5 g, 13.04 mmol) to give 2.42 g of the title compound <b>54d</b> (racemic mixture, 40%) after purification on a silica gel (eluent: DCM, 1% MeOH in DCM, 2%). MS (ESI) <i>m</i>/<i>z</i>: 405 [M + H]<sup>+</sup>.</div></div><div id="sec4_9_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> Intermediate <b>48a</b>, 7-(3-Piperidyl)-[1,2,4]triazolo[1,5-<i>a</i>]pyrimidine</h4><div class="NLM_p last">To a solution of intermediate <b>54a</b> (53.8 g, 0.18 mol) in 1,4 dioxane (600 mL) was added HCl-1,4 dioxane (200 mL, 4 M), and the reaction mixture was stirred for 2 h at RT. The product crystallized out as a hydrochloric salt, and it was filtered and washed with ether to give very hygroscopic crystals. These were then triturated in ether, filtered, and dried to give the title compound <b>48a</b> as a HCl salt (racemic, 39.7 g, 93%). MS (ESI) 204 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (360 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 1.74–2.00 (m, 3H), 2.13–2.23 (m, 1H), 2.76–3.01 (m, 1H), 3.22–3.40 (m, 2H), 3.58–3.66 (m, 1H), 3.94–4.04 (m, 1H), 7.34 (d, <i>J</i> = 4.39 Hz, 1H), 8.75 (s, 1H), 8.90 (d, <i>J</i> = 4.76 Hz, 1H), 9.27–9.51 (m, 1H), 9.65 (br d, <i>J</i> = 9.15 Hz, 1H). Anal. Calcd (C<sub>25</sub>H<sub>23</sub>N<sub>5</sub>O): C, H, N. Alternatively, <b>48a</b> (4 g) was also separated into enantiomers by purification by Prep SFC (Stationary phase: CHIRALPAK Daicel OJ 20 × 250 mm; mobile phase: CO<sub>2</sub>, EtOH with 0.5% iPr NH<sub>2</sub>) yielding intermediates: <b>48aa</b> (<b>R!</b>, 2g), <i>R</i><sub>t</sub> = 6.43 min, OR = +56.59° (589 nm, 20 °C, 0.42 w/v %; MeOH) and intermediate <b>48ab</b> (<b>S!</b>, 3 g), <i>R</i><sub>t</sub> = 9.15 min, OR = −53.9° (589 nm, 20 °C, 0.43 w/v %; MeOH).</div></div><div id="sec4_9_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> Intermediate <b>48b</b>, 2-([1,2,4]Triazolo[1,5-<i>a</i>]pyrimidin-7-yl)morpholine</h4><div class="NLM_p last">To a solution of intermediate <b>54b</b> (19.0 g, 62.2 mmol) in MeOH (100 mL) was added HCl–Et<sub>2</sub>O (200 mL, 1 M), and the reaction mixture was stirred for 16 h at RT. The solvent was evaporated under reduced pressure, and the residue was placed in DCM and treated with a saturated aq sol of NaHCO<sub>3</sub>. The biphasic layer was separated, and then the aqueous layer was extracted 2× DCM. The combined OL was dried over MgSO<sub>4</sub>, filtered, and evaporated to provide <b>48b</b> (racemic, 9 g, 71%). MS (ESI) <i>m</i>/<i>z</i>: 206 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (360 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 2.94–3.25 (m, 3H) 3.28–3.41 (m, 1H) 3.85 (br d, <i>J</i> = 12.64 Hz, 1H) 4.08–4.20 (m, 1H) 4.21–4.31 (m, 1H) 5.65 (dd, <i>J</i> = 11.03, 2.45 Hz, 1H) 7.40 (d, <i>J</i> = 4.53 Hz, 1H) 8.78 (s, 1H) 8.96 (d, <i>J</i> = 4.53 Hz, 1H) 10.03 (d, <i>J</i> = 9.70 Hz, 1H) 10.24 (br d, <i>J</i> = 10.69 Hz, 1H). This material was also separated into enantiomers by purification by Prep SFC (stationary phase: CHIRALPAK Daicel OJ 20 × 250 mm; mobile phase: CO<sub>2</sub>, MeOH) yielding intermediates: <b>54ba</b> (<b>R!</b>, 1.52 g, 8%), <i>R</i><sub>t</sub> = 8.19 min and <b>54bb</b> (<b>S!</b>, 1.77 g), <i>R</i><sub>t</sub> = 11.6 min.</div></div><div id="sec4_9_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> Intermediate <b>48c</b>, 7-(Azepan-3-yl)-[1,2,4]triazolo[1,5-<i>a</i>]pyrimidine</h4><div class="NLM_p last">Prepared as in the procedure for <b>48b</b>, starting from intermediate <b>54c</b> (600 mg, 1.89 mmol) to give the title compound <b>48c</b> (400 mg, 73%), and used in the next step without further purification. MS (ESI) <i>m</i>/<i>z</i>: 218 [M + H]<sup>+</sup>.</div></div><div id="sec4_9_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> Intermediate <b>48d</b>, 7-Piperazin-2-yl-[1,2,4]triazolo[1,5-<i>a</i>]pyrimidine</h4><div class="NLM_p last">Prepared as in the procedure for <b>48b</b>, starting from intermediate <b>54d</b> (2.42 g, 5.98 mmol) to give the title compound <b>48d</b> (1.85 g, 98%), and used in the next step without further purification.</div></div><div id="sec4_9_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> Intermediate <b>55</b>, <i>tert</i>-Butyl 3-[3,3-Bis(methylsulfanyl)prop-2-enoyl]piperidine-1-carboxylate</h4><div class="NLM_p last">NaH (60% dispersion in mineral oil, 8.77 g, 0.22 mol) was added to a solution of <i>tert</i>-butyl 3-acetylpiperidine-1-carboxylate (<b>50a</b>, 45.3 g, 0.20 mol) in anhydrous THF (300 mL) at 0 °C under an atmosphere of N<sub>2</sub>, and the resulting mixture was stirred at this temperature for 10 min. After this time, carbon disulfide (16.7 g, 0.22 mol) and then methyl iodide (59.4 g, 0.42 mol) were added dropwise. The reaction mixture was allowed to reach RT and stirred for 4 h. Water was added, and the resulting mixture was extracted with EtOAc. The combined organic layer was washed with brine, dried over MgSO<sub>4</sub>, filtered, and evaporated under reduced pressure. The resulting residue was purified via flash column chromatography on a silica gel using a gradient (DCM, 0.25% MeOH in DCM, 0.5, 1%) as the eluent to give the title compound (13.88 g, 21%). MS (ESI) <i>m</i>/<i>z</i>: 332 [M + H]<sup>+</sup>.</div></div><div id="sec4_9_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> Intermediate <b>56</b>, <i>tert</i>-Butyl 3-(5-Methylsulfanyl-[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)piperidine-1-carboxylate</h4><div class="NLM_p last">Intermediate <b>55</b> (8 g, 24.1 mmol) and 1<i>H</i>-1,2,4-triazol-5-amine hydrochloride (1:1) (2.03 g, 24.1 mmol) were stirred in melt at 150 °C for 3 h. The reaction mixture was cooled to RT and dissolved in DCM. The organic layer was washed with water, dried over MgSO<sub>4</sub>, filtered, and evaporated under reduced pressure. The resulting residue was purified by column chromatography on a silica gel using a gradient (DCM, 0.25% MeOH in DCM, 0.5, 1%) as the eluent to provide the title compound (1 g, 12%) MS (ESI) <i>m</i>/<i>z</i>: 350 [M + H]<sup>+</sup></div></div><div id="sec4_9_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> Intermediate <b>57</b>, <i>tert</i>-Butyl 3-(5-Methylsulfonyl-[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)piperidine-1-carboxylate</h4><div class="NLM_p last">A solution of intermediate <b>56</b> (1.39 g, 3.98 mmol) and mCPBA (2.75 g, 15.9 mmol) in chloroform (CHCl<sub>3</sub>, 50 mL) was stirred at reflux for 2 h. After this time, the reaction mixture was cooled to RT, then washed with NaOH 1 N, dried over MgSO<sub>4</sub>, filtered, and evaporated <i>in vacuo</i> to yield the title compound (1.50 g, 99%), which was used as such in the next step. MS (ESI) <i>m</i>/<i>z</i>: 380 [M – H]<sup>+</sup>.</div></div><div id="sec4_9_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> Intermediate <b>81e</b>, <i>tert</i>-Butyl 3-(5-Methoxy-[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)piperidine-1-carboxylate</h4><div class="NLM_p last">Sodium methoxide (NaOMe, 234 mg, 4.33 mmol) was added at RT to a solution of intermediate <b>57</b> (1.5 g, 3.93 mmol) in methanol (MeOH, 50 mL), and the resulting mixture was stirred at this temperature for 12 h. After this time, the solvent was evaporated <i>in vacuo</i> and the resulting residue was partitioned between water and DCM. The organic layer was separated, and then the aqueous layer was back extracted with DCM (2×). The combined organic layer was dried over MgSO<sub>4</sub>, filtered, and evaporated under reduced pressure. The resulting residue was purified by column chromatography on a silica gel using a gradient (DCM, 1% MeOH in DCM, 2%) as the eluent to provide the title compound (560 mg, 43%). MS (ESI) <i>m</i>/<i>z</i>: 334 [M + H]<sup>+</sup>.</div></div><div id="sec4_9_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> Intermediate <b>81f</b>, <i>tert</i>-Butyl 3-(5-Cyano-[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)piperidine-1-carboxylate</h4><div class="NLM_p last">Intermediate <b>57</b> (950 mg, 2.50 mmol) and sodium cyanide (249 mg, 4.98 mmol) were stirred in DMSO (5 mL) at RT for 30 min. The reaction mixture was poured in water and extracted with EtOAc. The organic layer was washed with brine, dried on MgSO<sub>4</sub>, filtered, and evaporated under reduced pressure. The resulting residue was purified via flash column chromatography on a silica gel using a gradient (DCM, 1% MeOH in DCM, 2%) as the eluent to give intermediate <b>79f</b> (820 mg, 100%). MS (ESI) <i>m</i>/<i>z</i>: 329 [M + H]<sup>+</sup>.</div></div><div id="sec4_9_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> Intermediate <b>48e</b>, 5-Methoxy-7-(3-piperidyl)-[1,2,4]triazolo[1,5-<i>a</i>]pyrimidine</h4><div class="NLM_p last">TFA (0.626 mL, 8.18 mmol) was added at RT to a solution of intermediate <b>81e</b> (560 mg, 1.68 mmol) in DCM (50 mL), and the resulting mixture was stirred for 2 h at RT. Saturated aqueous NaHCO<sub>3</sub> solution was then added to the reaction mixture and stirred for 10 min. The organic layer was separated, dried on MgSO<sub>4</sub>, filtered, and evaporated under reduced pressure to give intermediate <b>48e</b> (390 mg, 99.5%), which was used in the next step without further purification.</div></div><div id="sec4_9_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> Intermediate <b>48f</b>, 7-(3-Piperidyl)-[1,2,4]triazolo[1,5-<i>a</i>]pyrimidine-5-carbonitrile</h4><div class="NLM_p last">Prepared in a similar way to procedure <b>3b</b>, starting from intermediate <b>81f</b> (820 mg, 2.5 mmol) to give the title compound <b>48f</b> (390 mg, 68%), which was used in the next step without further purification.</div></div><div id="sec4_9_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> Intermediate <b>81g</b>, <i>tert</i>-Butyl 3-(5-Methyl-[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)piperidine-1-carboxylate</h4><div class="NLM_p last"><b>81g</b> is prepared in two steps. Step 1, intermediate <b>58</b>, <i>tert</i>-butyl 3-[(<i>Z</i>)-3-(dimethylamino)but-2-enoyl]piperidine-1-carboxylate. Intermediate <b>50a</b> (2.0 g, 8.8 mmol) was added to <i>N</i>,<i>N</i>-dimethylacetamide dimethyl acetal (3.52 g), and the mixture was refluxed for 3 h. The reaction mixture was evaporated to give the crude product, which was used as such in the next step (2.5 g, 95%). Step 2, intermediate <b>81g</b>, <i>tert</i>-butyl 3-(5-methyl-[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)piperidine-1-carboxylate. A solution of intermediate <b>58</b> (2.5 g, 8.43 mmol) and 1<i>H</i>-1,2,4-triazol-5-amine hydrochloride (0.71 g, 8.43 mmol) in acetic acid (50 mL) was stirred at reflux during 30 min. Saturated aqueous Na<sub>2</sub>CO<sub>3</sub> solution was added, and the product was extracted with DCM. The organic layer was dried on MgSO<sub>4</sub>, filtered, and evaporated. The resulting residue was purified by flash column chromatography on a silica gel using a gradient (DCM, 1% MeOH in DCM, 2%) as the eluent to give intermediate <b>81g</b> (2.14 g, 80%). MS (ESI) <i>m</i>/<i>z</i>: 318 [M + H]<sup>+</sup>.</div></div><div id="sec4_9_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> Intermediate <b>48g</b>, 5-Methyl-7-(3-piperidyl)-[1,2,4]triazolo[1,5-<i>a</i>]pyrimidine and <b>54a</b></h4><div class="NLM_p last">Prepared according to procedure 3, starting from intermediate <b>81g</b> (2.14 g, 6.74 mmol), to give the title compound as a mixture with <b>54a</b> (1.07 g, 54.6%). <sup>1</sup>H NMR (360 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 1.76–2.01 (m, 3H) 2.12–2.23 (m, 1H) 2.64 (s, 3H) 2.87–2.98 (m, 1H) 3.22–3.41 (m, 2H) 3.59 (br d, <i>J</i> = 11.71 Hz, 1H) 3.89–4.00 (m, 1H) 7.31 (s, 1H) 8.69 (s, 1H) 9.55 (br d, <i>J</i> = 10.25 Hz, 1H) 9.80 (br d, <i>J</i> = 10.25 Hz, 1H). MS (ESI) <i>m</i>/<i>z</i>: 218 [M + H]<sup>+</sup>.</div></div><div id="sec4_9_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> Intermediate <b>82</b>, <i>tert</i>-Butyl 3-(7-Methylimidazo[1,2-<i>a</i>]pyrimidin-5-yl)piperidine-1-carboxylate</h4><div class="NLM_p last">Prepared in a similar way to intermediate <b>81g</b>, starting from intermediate <b>58</b> (2.5 g, 8.43 mmol) and 1<i>H</i>-imidazol-2-amine (0.71 g, 8.43 mmol) to give the title compound <b>82</b> (0.86 g, 32%). MS (ESI) <i>m</i>/<i>z</i>: 317 [M + H]<sup>+</sup>.</div></div><div id="sec4_9_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> Intermediate <b>48h</b>, 7-Methyl-5-(3-piperidyl)imidazo[1,2-<i>a</i>]pyrimidine</h4><div class="NLM_p last">Prepared according to the procedure for <b>48a</b>, starting from intermediate <b>82</b> (0.86 g, 2.72 mmol), to give the title compound <b>48h</b> (0.75 g, 95%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 1.96–2.05 (m, 3H), 2.05–2.19 (m, 1H), 2.71 (s, 3H), 2.92–3.13 (m, 1H), 3.35–3.54 (m, 1H), 3.99–4.2 (m, 1H), 7.53 (s, 1H), 8.27 (d, <i>J</i> = 2.42 Hz, 1H), 8.69 (d, <i>J</i> = 2.83 Hz, 1H), 9.78 (br s, 1H), 9.95 (br s, 1H).</div></div><div id="sec4_9_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> Intermediate <b>59</b>, <i>tert</i>-Butyl 3-(4,4-Difluoro-3-oxo-butanoyl)piperidine-1-carboxylate</h4><div class="NLM_p last">Intermediate <b>50a</b> (17.4 g, 0.077 mol) was stirred in benzene (200 mL) at 0–5 °C under N<sub>2</sub>. Potassium <i>tert</i>-butoxide (10.5 g, 93.6 mol) was added at 0–5 °C and 2,2-difluoro-acetic acid ethyl ester (11.43 g, 0.92 mol) was added dropwise at 0–5 °C. The reaction mixture was stirred at RT for 5 h, then it was brought to pH = 7 with the aid of an aqueous H<sub>2</sub>SO<sub>4</sub> solution (10%, 10 mL). The reaction mixture was extracted with EtOAc (2 × 50 mL), then the combined organic layer was washed with brine, dried over MgSO<sub>4</sub>, filtered and evaporated under reduced pressure, to give intermediate <b>59</b> (23 g, 98%), which was pure enough to be used in the next step. MS (ESI) <i>m</i>/<i>z</i>: 304 [M – H]<sup>+</sup>.</div></div><div id="sec4_9_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> Intermediate <b>81i</b>, <i>tert</i>-Butyl 3-[5-(Difluoromethyl)-[1,2, 4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl]piperidine-1-carboxylate</h4><div class="NLM_p last">Intermediate <b>59</b> (23 g, 75.3 mmol), 1<i>H</i>-1,2,4-triazol-5-amine hydrochloride (1:1) (18.16 g, 150.7 mmol), and DMF (270 mL) were stirred at 60 °C for 28 h, then at 50 °C for 20 h. The reaction mixture was cooled to 0 °C, then Et<sub>3</sub>N (52 mL, 376 mmol) and di<i>tert</i>-butyl dicarbonate (16 g, 75.3 mmol) were added. The mixture was stirred for 1 h at RT, then concentrated <i>in vacuo</i>. The resulting residue was partitioned between DCM and an aqueous NH<sub>4</sub>OH solution (20%) and stirred for 30 min. The organic layer was separated, then dried over MgSO<sub>4</sub>, filtered and concentrated under reduced pressure. The resulting residue was purified by flash column chromatography using a gradient heptane/EtOAc, 100/0 to 60/40, as the eluent to give the title compound (23 g, 86%). MS (ESI) <i>m</i>/<i>z</i>: 354 [M + H]<sup>+</sup>.</div></div><div id="sec4_9_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> Intermediate <b>48i</b>, 5-(Difluoromethyl)-7-(3-piperidyl)-[1,2,4]triazolo[1,5-<i>a</i>]pyrimidine</h4><div class="NLM_p last">Prepared according to the procedure for <b>48a</b>, starting from intermediate <b>81i</b> (23 g, 65.09 mmol), to give the title compound as an HCl salt (13 g, 61%). <sup>1</sup>H NMR (360 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 1.86–2.04 (m, 3H), 2.18 (br d, <i>J</i> = 7.48 Hz, 1H), 2.94 (br d, <i>J</i> = 10.79 Hz, 1H), 3.29–3.49 (m, 2H), 3.59 (br d, <i>J</i> = 12.14 Hz, 1H), 3.98–4.14 (m, 2H), 7.15 (t, <i>J</i> = 54.52 Hz, 1H), 7.62 (s, 1H), 8.89 (s, 1H), 9.46 (br d, <i>J</i> = 11.57 Hz, 1H), 9.74 (br d, <i>J</i> = 10.70 Hz, 1H). MS (ESI) <i>m</i>/<i>z</i>: 254 [M + H]<sup>+</sup>.</div></div><div id="sec4_9_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> Intermediates <b>60</b>, <i>tert</i>-Butyl 5-(Trifluoromethylsulfonyloxy)-3,4-dihydro-2<i>H</i>-pyridine-1-carboxylate, and <b>61</b>, <i>tert</i>-Butyl 5-(Trifluoromethylsulfonyloxy)-3,6-dihydro-2<i>H</i>-pyridine-1-carboxylate</h4><div class="NLM_p last">A solution of commercially available 1-Boc-3-piperidinone (50 g, 250.9 mmol) in THF (125 mL) was added dropwise at −78 °C to a stirred solution of LDA in a mixture of THF/heptane/ethylbenzene (179 mL, 2 M, 358 mmol) in THF (625 mL) under a nitrogen atmosphere. After 15 min, a solution of <i>N</i>,<i>N</i>-bis(trifluoromethylsulfonyl)aniline (107.5 g, 300.9 mmol) in THF (250 mL) was added dropwise to the reaction mixture at −78 °C, and the resulting mixture was stirred for another 15 min. The reaction mixture was allowed to warm to RT overnight, and then it was cooled to 0 °C and quenched with a saturated aqueous NH<sub>4</sub>Cl solution (190 mL), diluted with water, and extracted with DCM. The organic layer was dried over MgSO<sub>4</sub>, filtered, and concentrated under reduced pressure. The resulting residue was purified by flash column chromatography on a silica gel using a gradient as the eluent (DCM, 1% CH<sub>3</sub>CN in DCM, 5%) to provide the intermediate <b>60</b> (42 g, 50%) and intermediate <b>61</b> (24.4 g, 29%). Intermediate <b>60</b>, <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 1.45 (s, 9H), 1.88 (quin, <i>J</i> = 6.06 Hz, 2H), 2.36–2.44 (m, 2H), 3.43–3.50 (m, 2H), 7.11 (s, 1H).</div></div><div id="sec4_9_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> Intermediate <b>62</b>, <i>tert</i>-Butyl 5-(4,4,5,5-Tetramethyl-1,3,2-dioxaborolan-2-yl)-3,4-dihydro-2<i>H</i>-pyridine-1-carboxylate</h4><div class="NLM_p last">A degassed solution of intermediate <b>60</b> (24.4 g, 73.65 mmol) in 1,4-dioxane (700 mL) was added to a previously degassed mixture of PdCl<sub>2</sub>(dppf) (1.61 g, 2.19 mmol), dppf (1.22 g, 2.2 mmol), KOAc (21.552 g, 219.6 mmol), and bis(pinacolato)diboron (20.53 g, 80.84 mmol) at RT and under a nitrogen atmosphere. The reaction mixture was stirred during 16 h at 80 °C, after which time it was filtered over Celite. The filtrate was evaporated under reduced pressure, and the resulting residue was purified via flash column chromatography on a silica gel using a gradient (DCM, 2% CH3CN in DCM, 5%) as the eluent to give the title compound <b>62</b> as a white solid after recrystallization from heptane (10.8 g, 47%). MS (ESI) <i>m</i>/<i>z</i>: 209 [M-Boc + H]<sup>+</sup>.</div></div><div id="sec4_9_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> Intermediate <b>63</b>, <i>tert</i>-Butyl 5-(4,4,5,5-Tetramethyl-1,3,2-dioxaborolan-2-yl)-3,6-dihydro-2<i>H</i>-pyridine-1-carboxylate</h4><div class="NLM_p last">Prepared in a similar way to intermediate <b>62</b>, starting from intermediate <b>61</b> (24.4 g, 73.6 mmol), to give the title compound <b>63</b> (10.8 g, 35% yield). MS (ESI) <i>m</i>/<i>z</i>: 209 [M-Boc + H]<sup>+</sup>.</div></div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i63" class="anchor-spacer"></div><h3 class="article-section__title" id="_i63"> General Procedure for the Suzuki Couplings of Intermediates <b>62</b> and <b>63</b> with the Corresponding Bicyclic Heteroaromatic Chlorides</h3><div id="sec4_10_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> Intermediate <b>65a</b>, <i>tert</i>-Butyl 5-(5-Isopropyl-[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)-3,4-dihydro-2<i>H</i>-pyridine-1-carboxylate</h4><div class="NLM_p last">A mixture of intermediate <b>62</b> (2.04 g, 6.61 mmol), commercially available 7-chloro-5-(propan-2-yl)-[1,2,4]triazolo[1,5-<i>a</i>]pyrimidine (1 g, 5.08 mmol), allylchloro[1,3-bis(2,6-di-<i>i</i>-propylphenyl)-4,5-dihydroimidazol-2-ylidene]palladium(II) (584 mg, 1.02 mmol), Na<sub>2</sub>CO<sub>3</sub> (1.08 g, 10.17 mmol), 1,4-dioxane (5 mL), EtOH (5 mL), and distilled water (1 mL) was stirred under microwave irradiation at 120 °C 2 h. The volatiles were removed under reduced pressure, and the resulting residue was partitioned between water and EtOAc. The organic layer was washed with brine, dried over MgSO<sub>4</sub>, filtered, and concentrated <i>in vacuo</i>. The resulting residue was purified by Prep HPLC using as the stationary phase a column Vydac Denali C18—10 μm, 250 g, 5 cm, and as the mobile phase a mixture of 0.25% NH<sub>4</sub>HCO<sub>3</sub> solution in water and MeOH to give the title compound <b>65a</b> (180 mg, 10%). MS (ESI) <i>m</i>/<i>z</i>: 344 [M + H]<sup>+</sup>.</div></div><div id="sec4_10_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> Intermediate <b>65b</b>, <i>tert</i>-Butyl 5-[5-(Trifluoromethyl)-[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl]-3,4-dihydro-2<i>H</i>-pyridine-1-carboxylate</h4><div class="NLM_p last">Prepared in a similar way to intermediate <b>65a</b>, starting from intermediate <b>62</b> (1.00 g, 3.23 mmol) and commercially available 7-chloro-5-(trifluoromethyl)-[1,2,4]triazolo[1,5-<i>a</i>]pyrimidine (0.72 g, 3.23 mmol) and employing Pd(PPh<sub>3</sub>)<sub>4</sub> (0.75 g, 0.65 mmol) as the catalyst to give the title compound <b>65b</b> (0.32 g, 26.8%). MS (ESI) <i>m</i>/<i>z</i>: 370 [M + H]<sup>+</sup>.</div></div><div id="sec4_10_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> Intermediate <b>65c</b>, <i>tert</i>-Butyl 5-(2,5-Dimethyl-[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)-3,4-dihydro-2<i>H</i>-pyridine-1-carboxylate</h4><div class="NLM_p last">Prepared in a similar way to intermediate <b>65a</b>, starting from intermediate <b>62</b> (2.2 g, 7.12 mmol) and commercially available 7-chloro-2,5-dimethyl-[1,2,4]triazolo[1,5-<i>a</i>]pyrimidine (1.0 g, 5.48 mmol) to give the title compound <b>65c</b> (0.17 g, 9.4%). MS (ESI) <i>m</i>/<i>z</i>: 330 [M + H]<sup>+</sup>.</div></div><div id="sec4_10_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> Intermediate <b>65d</b>, <i>tert</i>-Butyl 5-[5-Methyl-2-(trifluoromethyl)-[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl]-3,4-dihydro-2<i>H</i>-pyridine-1-carboxylate</h4><div class="NLM_p last">Prepared in a similar way to intermediate <b>65a</b>, starting from intermediate <b>62</b> (1.31 g, 4.23 mmol) and commercially available, 7-chloro-5-methyl-2-(trifluoromethyl)-[1,2,4]triazolo[1,5-<i>a</i>]pyrimidine (1.00 g, 4.23 mmol) and employing Pd(PPh<sub>3</sub>)<sub>4</sub> (0.97 g, 0.85 mmol) as the catalyst to give the title compound <b>65d</b> (0.39 g, 9.26%). MS (ESI) <i>m</i>/<i>z</i>: 384 [M + H]<sup>+</sup>.</div></div><div id="sec4_10_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> Intermediate <b>83a</b>, <i>tert</i>-Butyl 3-(5-Isopropyl-[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)piperidine-1-carboxylate</h4><div class="NLM_p last">Intermediate <b>65a</b> (180 mg, 0.52 mmol) and a solution of thiophenol (0.1 mL, 4%, 0.039 mmol) was added to a suspension of Pd/C (10%) (100 mg, 0.094 mmol) in methanol (40 mL) under N<sub>2</sub> flow. The reaction mixture was stirred at RT under H<sub>2</sub> atmosphere until 1 equiv hydrogen was absorbed. The catalyst was removed by filtration over Celite and the filtrate was evaporated to give the title compound <b>83a</b> (180 mg, 99.4%), which was used in the next step without further purification.</div></div><div id="sec4_10_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> Intermediate <b>83b</b>, <i>tert</i>-Butyl 3-[5-(Trifluoromethyl)-[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl]piperidine-1-carboxylate</h4><div class="NLM_p last">Intermediate <b>62b</b> (320 mg, 0.87 mmol) and a solution of thiophenol (0.1 mL, 4%, 0.039 mmol) was added to a suspension of Pt/C (5%) (100 mg) in methanol (40 mL) under N<sub>2</sub> flow. The reaction mixture was stirred at 50 °C under a H<sub>2</sub> atmosphere until 1 equiv hydrogen was absorbed. The catalyst was removed by filtration over Celite, and the filtrate was evaporated. The resulting residue was purified via flash column chromatography on a silica gel using a gradient (DCM, 2% MeOH in DCM) as the eluent to give the title compound <b>83b</b> (150 mg, 46.6%). MS (ESI) <i>m</i>/<i>z</i>: 372 [M + H]<sup>+</sup>.</div></div><div id="sec4_10_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> Intermediate <b>83c</b>, <i>tert</i>-Butyl 3-(2,5-Dimethyl-[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)piperidine-1-carboxylate</h4><div class="NLM_p last">Intermediate <b>65c</b> (170 mg, 0.52 mmol) and a solution of thiophenol (0.1 mL, 4%, 0.039 mmol) were added to a suspension of Pt/C (5%) (100 mg) in methanol (40 mL) under N<sub>2</sub> flow. The reaction mixture was stirred at 50 °C under a H<sub>2</sub> atmosphere until 1 equiv hydrogen was absorbed. The catalyst was removed by filtration over Celite, and the filtrate was evaporated. The resulting residue was dissolved in DCM (10 mL), and then it was treated with MnO<sub>2</sub> (45 mg, 0.52 mmol) and stirred at RT for 1 h. The reaction mixture was filtered over Celite, and the solvent was evaporated under reduced pressure to provide the title compound <b>83c</b> (150 mg, 87.7%), which was used in the next step without further purification.</div></div><div id="sec4_10_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> Intermediate <b>83d</b>, <i>tert</i>-Butyl 3-[5-Methyl-2-(trifluoromethyl)-[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl]piperidine-1-carboxylate</h4><div class="NLM_p last">Prepared in a similar way to intermediate <b>83c</b>, starting from intermediate <b>65d</b> (150 mg, 0.39 mmol) to give the title compound <b>83d</b> (100 mg, 66%). MS (ESI) <i>m</i>/<i>z</i>: 386 [M + H]<sup>+</sup>.</div></div><div id="sec4_10_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72"> Intermediate <b>48j</b>, 5-Isopropyl-7-(3-piperidyl)-[1,2,4]triazolo[1,5-<i>a</i>]pyrimidine</h4><div class="NLM_p last">To a solution of intermediate <b>83a</b> (180 mg, 0.52 mmol) in MeOH (50 mL) was added HCl–iPrOH (50 mL), and the reaction mixture was stirred for 16 h at RT. The reaction mixture was evaporated to give the title compound as an HCl salt (160 mg, 96%) MS (ESI) <i>m</i>/<i>z</i>: 246 [M + H]<sup>+</sup>.</div></div><div id="sec4_10_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i73" class="anchor-spacer"></div><h4 class="article-section__title" id="_i73"> Intermediate <b>48k</b>, 7-(3-Piperidyl)-5-(trifluoromethyl)-[1,2,4]triazolo[1,5-<i>a</i>]pyrimidine</h4><div class="NLM_p last">Prepared in a similar way to <b>48j</b>, starting from intermediate <b>83b</b> (150 mg, 0.40 mmol) to give the title compound <b>48k</b> as an HCl salt (140 mg, quantitative) MS (ESI) <i>m</i>/<i>z</i>: 272 [M + H]<sup>+</sup>.</div></div><div id="sec4_10_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i74" class="anchor-spacer"></div><h4 class="article-section__title" id="_i74"> Intermediate <b>48l</b>, 2,5-Dimethyl-7-(3-piperidyl)-[1,2,4]triazolo[1,5-<i>a</i>]pyrimidine</h4><div class="NLM_p last">Prepared in a similar way to <b>48j</b>, starting from intermediate <b>83c</b> (150 mg, 0.45 mmol) to give the title compound <b>48l</b> as an HCl salt (130 mg, 94%). MS (ESI) <i>m</i>/<i>z</i>: 232 [M + H]<sup>+</sup>.</div></div><div id="sec4_10_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i75" class="anchor-spacer"></div><h4 class="article-section__title" id="_i75"> Intermediate <b>48m</b>, 5-Methyl-7-(3-piperidyl)-2-(trifluoromethyl)-[1,2,4]triazolo[1,5-<i>a</i>]pyrimidine</h4><div class="NLM_p last">Prepared in a similar way to <b>48j</b>, starting from intermediate <b>83d</b> (100 mg, 0.45 mmol) to give the title compound <b>48m</b> as an HCl salt (90 mg, 97%). MS (ESI) <i>m</i>/<i>z</i>: 286 [M + H]<sup>+</sup>.</div></div><div id="sec4_10_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i76" class="anchor-spacer"></div><h4 class="article-section__title" id="_i76"> Intermediate <b>67</b>, <i>tert</i>-Butyl 5-(1<i>H</i>-Pyrrolo[2,3-<i>b</i>]pyridin-4-yl)-3,4-dihydro-2<i>H</i>-pyridine-1-carboxylate</h4><div class="NLM_p last">Prepared in a similar way to <b>65a</b>, starting from intermediate <b>63</b> (1.32 g, 4.26 mmol) and commercially available 4-chloro-7-azaindole (0.5 g, 3.28 mmol). Purification via flash column chromatography on a silica gel using a gradient (DCM, 1% MeOH in DCM, 2%) as the eluent to give the title compound <b>67</b> (180 mg, 18%). MS (ESI) <i>m</i>/<i>z</i>: 300 [M + H]<sup>+</sup>.</div></div><div id="sec4_10_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i77" class="anchor-spacer"></div><h4 class="article-section__title" id="_i77"> Intermediate <b>84</b>, <i>tert</i>-Butyl 3-(1<i>H</i>-Pyrrolo[2,3-<i>b</i>]pyridin-4-yl)piperidine-1-carboxylate</h4><div class="NLM_p last">Prepared in a similar way to <b>83b</b>, starting from intermediate <b>67</b> (180 mg, 0.60 mmol) to give the title compound <b>84</b> (180 mg, 99%), which used as such in the next step.</div></div><div id="sec4_10_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i78" class="anchor-spacer"></div><h4 class="article-section__title" id="_i78"> Intermediate <b>48n</b>, 4-(3-Piperidyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine</h4><div class="NLM_p last">Prepared in a similar way to <b>48a</b>, starting from intermediate <b>84</b> (180 mg, 0.59 mmol) to give the title compound <b>48n</b> as an HCl salt (150 mg, 91.6%), which was used directly in the next step.</div></div><div id="sec4_10_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i79" class="anchor-spacer"></div><h4 class="article-section__title" id="_i79"> Intermediate <b>69</b>, <i>tert</i>-Butyl 5-[5-(Difluoromethyl)pyrazolo[1,5-<i>a</i>]pyrimidin-7-yl]-3,4-dihydro-2<i>H</i>-pyridine-1-carboxylate</h4><div class="NLM_p last">Prepared in a similar way to <b>65a</b>, starting from intermediate <b>62</b> (0.59 g, 1.92 mmol) and 7-chloro-5-(difluoromethyl)pyrazolo[1,5-<i>a</i>]pyrimidine (0.30 g, 1.5 mmol). Purification via flash column chromatography on a silica gel using a mixture heptane/EtOAc, 80/20, as the eluent gave the title compound <b>69</b> (1.3 g) MS (ESI) <i>m</i>/<i>z</i>: 351 [M + H]<sup>+</sup>.</div></div><div id="sec4_10_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i80" class="anchor-spacer"></div><h4 class="article-section__title" id="_i80"> Intermediate <b>85</b>, <i>tert</i>-Butyl 3-[5-(Difluoromethyl)pyrazolo[1,5-<i>a</i>]pyrimidin-7-yl]piperidine-1-carboxylate</h4><div class="NLM_p last">Prepared in a similar way to <b>83a</b>, starting from intermediate <b>69</b> (1.3 g, 3.71 mmol) to give the title compound <b>85</b> (820 mg, 63%), MS (ESI) <i>m</i>/<i>z</i>: 351 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 1.36 (br s, 9H), 1.49–1.63 (m, 1H), 1.74 (d, <i>J</i> = 11.3 Hz, 1H), 1.93–2.07 (m, 1H), 3.10 (ddd, <i>J</i> = 13.2, 9.7, 3.1 Hz, 1H), 3.40 (br s, 1H), 3.66–3.74 (m, 1H), 3.79 (d, <i>J</i> = 13.1 Hz, 1H), 4.15 (d, <i>J</i> = 13.2 Hz, 1H), 7.00 (t, <i>J</i> = 54.4 Hz, 1H), 6.94 (<i>J</i> = 2.5 Hz, 1H), 7.21 (s, 1H), 8.40 (d, <i>J</i> = 2.4 Hz, 1H).</div></div><div id="sec4_10_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i81" class="anchor-spacer"></div><h4 class="article-section__title" id="_i81"> Intermediate <b>48o</b>, 5-(Difluoromethyl)-7-(3-piperidyl)pyrazolo[1,5-<i>a</i>]pyrimidine</h4><div class="NLM_p last">Prepared in a similar way to <b>48a</b>, starting from intermediate <b>85</b> (820 mg, 2.33 mmol) to give the title compound <b>48o</b> as an HCl salt (520 mg, 88%), which used as such in the next step.</div></div><div id="sec4_10_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i82" class="anchor-spacer"></div><h4 class="article-section__title" id="_i82"> Final Compound <b>4</b>, (3,4-Dimethoxyphenyl)-[3-([1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)-1-piperidyl]methanone</h4><div class="NLM_p last">3,4-Dimethoxybenzoic acid (760 mg, 4.17 mmol) was stirred in THF (40 mL). DIPEA (2.16 g, 0.75 g/mL, 16.7 mmol) and HBTU (1.74 g, 4.59 mmol) were added, stirring was continued for 20 min at RT. A solution of intermediate <b>48a</b> (1.38 g, 5.00 mmol) in DMF (10 mL) was added, and stirring was continued for 2 h at RT. The solvents were evaporated under reduced pressure, and the resulting residue was diluted with ice and water and then extracted with 2× DCM. The combined organic layers were washed with an aqueous solution of NaOH (1 N) and brine, separated, dried over MgSO<sub>4</sub>, filtered, and then concentrated <i>in vacuo</i>. This material was purified by recrystallization from EtOAc, yielding compound <b>4</b> (1.04 g, 67.8% yield). MS (ESI) <i>m</i>/<i>z</i>: 368 [M + H]<sup>+</sup>. mp 159.2 C; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 1.62–1.74 (m, 1H) 1.84 (dt, <i>J</i> = 13.62, 3.89 Hz, 1H) 1.96–2.06 (m, 1H) 2.20–2.27 (m, 1H) 3.08–3.18 (m, 1H) 3.39 (dd, <i>J</i> = 12.92, 10.09 Hz, 1H) 3.68–3.74 (m, 1H) 3.79 (s, 6H) 3.96–4.08 (m, 1H) 4.38 (br d, <i>J</i> = 12.51 Hz, 1H) 6.93–7.00 (m, 3H) 7.26 (d, <i>J</i> = 4.84 Hz, 1H) 8.56 (s, 1H) 8.81 (d, <i>J</i> = 4.44 Hz, 1H).</div></div><div id="sec4_10_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i83" class="anchor-spacer"></div><h4 class="article-section__title" id="_i83"> Final Compound <b>5</b>, 2-(3,4-Dimethoxyphenyl)-1-[3-([1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)-1-piperidyl]ethenone</h4><div class="NLM_p last">3,4-Dimethoxyphenylacetic acid (0.047 g, 0.24 mmol) was stirred in DCM (5 mL). DIPEA (0.14 mL, 0.86 mmol) and HBTU (90 mg, 0.24 mmol) were added, and stirring was continued for 30 min at RT. Intermediate <b>48a</b> (0.060 g, 0.22 mmol) was added, and stirring was continued for 2 h at RT. The RM was quenched with water (1 mL), and then it was filtered on an Extrelut column. The OL was evaporated <i>in vacuo</i>. This material was purified by Prep HPLC (stationary phase: RP Vydac Denali C18—10 μm, 250 × 5 cm, mobile phase: 0.25% NH<sub>4</sub>HCO<sub>3</sub> solution in water, MeOH), to give compound <b>5</b> (57 mg, 69 % yield). MS (ESI) <i>m</i>/<i>z</i>: 382 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6,</sub> 100 °C): δ ppm 1.46–1.56 (m, 1H); 1.77 (m, 1H); 1.93 (dtd, <i>J</i> = 13.00, 11.21, 11.21, 4.05 Hz, 1H); 2.15–2.21 (m, 1H); 3.26–3.34 (m, 1H); 3.51 (br t, <i>J</i> = 9.91 Hz, 1H); 3.67 (m, 2H); 3.71 (s, 3H); 3.73 (m, 4H); 4.07–4.26 (m, 1H); 4.38–4.54 (m, 1H); 6.75 (br d, <i>J</i> = 7.60 Hz, 1H); 6.79–6.89 (m, 2H); 7.17 (d, <i>J</i> = 4.62 Hz, 1H); 8.59 (s, 1H); 8.78 (d, <i>J</i> = 4.62 Hz, 1H).</div></div><div id="sec4_10_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i84" class="anchor-spacer"></div><h4 class="article-section__title" id="_i84"> Final Compound <b>6</b>, Phenyl-[3-([1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)-1-piperidyl]methanone</h4><div class="NLM_p last">Benzoyl chloride (0.03 g, 0.22 mmol) was added at RT to a solution of intermediate <b>48a</b> (0.06 g, 0.22 mmol) and DIPEA (0.084 g, 0.75 g/mL, 0.65 mmol) in DCM (2 mL), and the resulting mixture was stirred at RT for 1 h. The RM was quenched with water (1 mL), and then it was filtered on an Extrelut column. The OL was evaporated <i>in vacuo</i>. This material was purified by flash column chromatography on a silica gel, eluting with 0–2% MeOH in DCM, to give compound <b>6</b> (47 mg, 70% yield). MS (ESI) <i>m</i>/<i>z</i>: 308 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6.</sub> 100 °C): δ ppm 1.64–1.72 (m, 1H); 1.79–1.88 (m, 1H) 2.03 (dtd, <i>J</i> = 13.11, 10.95, 10.95, 3.96 Hz, 1H); 2.21–2.28 (m, 1H); 3.13–3.19 (m, 1H); 3.42 (dd, <i>J</i> = 12.96, 9.99 Hz, 1H); 3.65–3.76 (m, 1H); 3.91–4.03 (m, 1H); 4.31–4.42 (m, 1H); 7.24 (d, <i>J</i> = 4.62 Hz, 1H); 7.34–7.44 (m, 5H); 8.53 (s, 1H); 8.80 (d, <i>J</i> = 4.62 Hz, 1H).</div></div><div id="sec4_10_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i85" class="anchor-spacer"></div><h4 class="article-section__title" id="_i85"> Final Compound <b>7</b>, Cyclopropyl-[3-([1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)-1-piperidyl]methanone</h4><div class="NLM_p last">Prepared in a similar manner as compound <b>5</b>, starting from cyclopropanecarboxylic acid (0.02 g, 0.24 mmol) and intermediate <b>48a</b> (0.06 g, 0.22 mmol) to give compound <b>7</b> (49 mg, 84% yield). MS (ESI) <i>m</i>/<i>z</i>: 272 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 0.62–0.69 (m, 1H) 0.69–0.75 (m, 2H) 0.76–0.81 (m, 1H) 1.56–1.67 (m, 1H) 1.84 (dt, <i>J</i> = 13.67, 3.90 Hz, 1H) 1.90–1.95 (m, 1H) 2.00 (dtd, <i>J</i> = 13.07, 11.05, 11.05, 4.13 Hz, 1H) 2.22 (dq, <i>J</i> = 12.96, 4.16 Hz, 1H) 3.10–3.15 (m, 1H) 3.38 (br t, <i>J</i> = 11.31 Hz, 2H) 3.55–3.65 (m, 1H) 4.14–4.28 (m, 1H) 4.54 (ddt, <i>J</i> = 13.03, 3.61, 1.59, 1.59 Hz, 1H) 7.24 (d, <i>J</i> = 4.62 Hz, 1H) 8.58 (s, 1H) 8.81 (d, <i>J</i> = 4.62 Hz, 1H).</div></div><div id="sec4_10_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i86" class="anchor-spacer"></div><h4 class="article-section__title" id="_i86"> Final Compound <b>8</b>, Cyclohexyl-[3-([1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)-1-piperidyl]methanone</h4><div class="NLM_p last">Cyclohexanecarbonyl chloride (0.116 mL, 0.87 mmol) was added at RT to a solution of intermediate <b>48a</b> (250 mg, 0.87 mmol) and DIPEA (0.574 mL, 0.75 g/mL, 3.47 mmol), and the resulting mixture was stirred at RT for 1 h. The reaction mixture was quenched with water (1 mL) and saturated aqueous solution of Na<sub>2</sub>CO<sub>3</sub> (1 mL), and then the mixture was filtered through an Extrelut filter. The organic layer was evaporated under reduced pressure, and the resulting residue was purified via Prep HPLC (stationary phase: RP Vydac Denali C18—10 μm, 250 × 5 cm, mobile phase: 0.25% NH<sub>4</sub>HCO<sub>3</sub> solution in water, MeOH) to afford compound <b>8</b> (30 mg, 11%). MS (ESI) <i>m</i>/<i>z</i>: 312 [M – H]<sup>+</sup>. <sup>1</sup>H NMR (360 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 1.15 (br s, 1H) 1.28–1.39 (m, 2H) 1.47 (br s, 1H) 1.58–1.73 (m, 4H) 1.75–1.85 (m, 1H) 1.87–2.02 (m, 2H) 2.17 (br d, <i>J</i> = 12.81 Hz, 1H) 2.52–2.83 (m, 2H) 3.04–3.32 (m, 3H) 3.51 (br d, <i>J</i> = 11.34 Hz, 1H) 3.72–4.01 (m, 1H) 4.39 (br d, <i>J</i> = 7.68 Hz, 1H) 4.47–4.66 (m, 1H) 7.31 (dd, <i>J</i> = 16.10, 4.39 Hz, 1H) 8.72 (d, <i>J</i> = 13.54 Hz, 1H) 8.86 (dd, <i>J</i> = 10.25, 4.39 Hz, 1H).</div></div><div id="sec4_10_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i87" class="anchor-spacer"></div><h4 class="article-section__title" id="_i87"> Final Compound <b>9</b>, (2-Methoxyphenyl)-[3(<i>S</i>)-([1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)-1-piperidyl]methanone</h4><div class="NLM_p last">Prepared in a similar manner as compound <b>8</b>, starting from intermediate <b>48ab</b> (<b>S!</b>, 0.20 g, 0.72 mmol) and 2-methoxybenzoyl chloride (0.281 g, 2.17 mmol). Purification by flash chromatography on a silica gel, eluting with 0–2% MeOH in DCM, afforded compound <b>9</b> (167 mg, 68%). MS (ESI) <i>m</i>/<i>z</i>: 338 [M + H]<sup>+</sup>. (DMSO-<i>d</i><sub>6</sub>): δ ppm 1.49 (br d, <i>J</i> = 9.51 Hz, 1H) 1.56–1.73 (m, 2H) 1.87–2.05 (m, 3H) 2.16 (br s, 2H) 2.96–3.21 (m, 2H) 3.25–3.33 (m, 2H) 3.36–3.46 (m, 1H) 3.50–3.75 (m, 5H) 3.81 (s, 1H) 3.85 (s, 3H) 4.37–4.49 (m, 1H) 4.54 (br d, <i>J</i> = 13.17 Hz, 1H) 4.74 (br d, <i>J</i> = 11.34 Hz, 1H) 6.77–6.83 (m, 1H) 6.87–7.18 (m, 5H) 7.24–7.33 (m, 3H) 7.33–7.44 (m, 2H) 8.49 (s, 1H) 8.74 (d, <i>J</i> = 1.83 Hz, 1H) 8.80 (d, <i>J</i> = 4.76 Hz, 1H) 8.90 (d, <i>J</i> = 4.39 Hz, 1H).</div></div><div id="sec4_10_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i88" class="anchor-spacer"></div><h4 class="article-section__title" id="_i88"> Final Compound <b>10</b></h4><div class="NLM_p last">Prepared in a similar manner as compound <b>8</b>, starting from intermediate <b>48ab</b> (<b>S!</b>, 1 equiv) and 3-methoxybenzoyl chloride (1 equiv). MS (ESI) <i>m</i>/<i>z</i>: 338 [M + H]<sup>+</sup>; <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6,</sub> 100 °C): δ ppm 1.64–1.73 (m, 1H) 1.79–1.87 (m, 1H) 1.96–2.08 (m, 1H) 2.18–2.28 (m, 1H) 3.11–3.21 (m, 1H) 3.41 (dd, <i>J</i> = 12.96, 9.99 Hz, 1H) 3.71 (tt, <i>J</i> = 10.28, 3.92 Hz, 1H) 3.79 (s, 3H) 3.93–4.04 (m, 1H) 4.30–4.42 (m, 1H) 6.90–6.93 (m, 2H) 6.95–6.98 (m, 1H) 7.24 (d, <i>J</i> = 4.46 Hz, 1H) 7.27–7.33 (m, 1H) 8.52 (s, 1H) 8.80 (d, <i>J</i> = 4.62 Hz, 1H).</div></div><div id="sec4_10_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i89" class="anchor-spacer"></div><h4 class="article-section__title" id="_i89"> Final Compound <b>11</b>, (4-Methoxyphenyl)-[3(<i>S</i>)-([1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)-1-piperidyl]methanone</h4><div class="NLM_p last">Prepared in a similar manner to compound <b>4</b>, starting from 4-methoxybenzoic acid (0.100 g, 0.66 mmol) and intermediate <b>48ab</b> (<b>S!</b>, 0.218 g, 0.79 mmol) and using DCM as the solvent. Purification via Prep HPLC (stationary phase: RP Vydac Denali C18—10 μm, 250 × 5 cm, mobile phase: 0.25% NH<sub>4</sub>HCO<sub>3</sub> solution in water, MeOH) provided compound <b>11</b> (180 mg, 81%). MS (ESI) <i>m</i>/<i>z</i>: 338 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 1.56–1.90 (m, 2H); 1.99 (m, 1H); 2.18–2.25 (m, 1H); 2.93–3.27 (m, 2H); 3.64–3.74 (m, 1H); 3.79 (s, 3H); 4.03–4.83 (m, 2H); 6.97 (br s, 2H); 7.32 (br s, 1H); 7.39 (br d, <i>J</i> = 7.93 Hz, 2H); 8.68 (br s, 1H); 8.85 (br s, 1H).</div></div><div id="sec4_10_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i90" class="anchor-spacer"></div><h4 class="article-section__title" id="_i90"> Final Compound <b>12</b>, 4-[3(<i>S</i>)-([1,2,4]Triazolo[1,5-<i>a</i>]pyrimidin-7-yl)piperidine-1-carbonyl]benzonitrile</h4><div class="NLM_p last">Prepared in a similar manner to compound <b>11</b>, starting from 4-cyanobenzoic acid (0.100 g, 0.68 mmol) and intermediate <b>48ab</b> (<b>S!</b>, 0.225 g, 0.82 mmol). Purification via Prep HPLC (stationary phase: RP Vydac Denali C18—10 μm, 250 × 5 cm, mobile phase: 0.25% NH<sub>4</sub>HCO<sub>3</sub> solution in water, MeOH) provided compound <b>12</b> (35 mg, 15%). MS (ESI) <i>m</i>/<i>z</i>: 333 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6,</sub> 100 °C): δ ppm 1.65–1.75 (m, 1H); 1.81–1.90 (m, 1H); 2.04 (dtd, <i>J</i> = 13.11, 11.03, 11.03, 3.96 Hz, 1H); 2.21–2.27 (m, 1H); 3.13–3.20 (m, 1H); 3.41 (dd, <i>J</i> = 13.05, 10.07 Hz, 1H); 3.67–3.76 (m, 1H); 3.86–4.06 (m, 1H); 4.18–4.42 (m, 1H); 7.23 (d, <i>J</i> = 4.46 Hz, 1H); 7.56–7.58 (m, 2H); 7.83 (d, <i>J</i> = 8.26 Hz, 2H); 8.54 (s, 1H); 8.80 (d, <i>J</i> = 4.62 Hz, 1H).</div></div><div id="sec4_10_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i91" class="anchor-spacer"></div><h4 class="article-section__title" id="_i91"> Final Compound <b>13</b>, [(3<i>S</i>)-([1,2,4]Triazolo[1,5-<i>a</i>]pyrimidin-7-yl)-1-piperidyl]-[4-(trifluoromethyl)phenyl]methanone</h4><div class="NLM_p last">Prepared in a similar manner as compound <b>8</b>, starting from intermediate <b>48ab</b> (<b>S!</b>, 0.06 g, 0.22 mmol) and 4-(trifluoromethyl)benzoyl chloride (0.045 g, 0.22 mmol). Purification by flash chromatography on a silica gel, eluting with 0–2% MeOH in DCM, afforded compound <b>9</b> (65 mg, 80%). MS (ESI) <i>m</i>/<i>z</i>: 376 [M + H]<sup>+</sup>. SFC [CHIRALCEL Daicel OD 4.6 × 500 mm; mobile phase: CO<sub>2</sub>, MeOH with 0.2% 2-propylamine] <i>R</i><sub>t</sub> = 12.68 min. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6,</sub> 100 °C): δ ppm 1.62–1.78 (m, 2H) 1.81–1.91 (m, 1H) 2.04 (dtd, <i>J</i> = 13.13, 10.98, 10.98, 4.05 Hz, 2H) 2.20–2.29 (m, 1H) 3.13–3.23 (m, 1H) 3.42 (dd, <i>J</i> = 12.96, 9.99 Hz, 1H) 3.66–3.76 (m, 1H) 3.85–4.08 (m, 1H) 4.23–4.47 (m, 1H) 7.24 (d, <i>J</i> = 4.46 Hz, 1H) 7.60 (d, <i>J</i> = 7.93 Hz, 2H) 7.74 (d, <i>J</i> = 7.93 Hz, 2H) 8.50 (s, 1H) 8.80 (d, <i>J</i> = 4.62 Hz, 1H).</div></div><div id="sec4_10_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i92" class="anchor-spacer"></div><h4 class="article-section__title" id="_i92"> Final compound <b>14</b>, 2,3-Dihydro-1,4-benzodioxin-6-yl-[(3<i>S</i>)-([1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)-1-piperidyl]methanone</h4><div class="NLM_p last">Prepared in a similar manner to compound <b>11</b>, starting from 2,3-dihydro-1,4-benzodioxine-6-carboxylic acid (0.100 g, 0.56 mmol) and intermediate <b>48ab</b> (<b>S!</b>, 0.184 g, 0.67 mmol). Purification via Prep HPLC (stationary phase: RP Vydac Denali C18—10 μm, 250 × 5 cm, mobile phase: 0.25% NH<sub>4</sub>HCO<sub>3</sub> solution in water, MeOH) provided compound <b>14</b> (114 mg, 56%). MS (ESI) <i>m</i>/<i>z</i>: 366 [M + H]<sup>+</sup>. SFC [CHIRALCEL Daicel OJ 4.6 × 500 mm; mobile phase: CO<sub>2</sub>, MeOH with 0.2% 2-propylamine] <i>R</i><sub>t</sub> = 13.5 min. <sup>1</sup>H NMR (360 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 1.67 (br s, 1H) 1.82 (br s, 1H) 1.99 (q, <i>J</i> = 10.49 Hz, 1H) 2.21 (br dd, <i>J</i> = 12.99, 3.48 Hz, 1H) 3.12 (br s, 1H) 3.26–3.32 (m, 1H) 3.62–3.74 (m, 1H) 4.27 (s, 5H) 4.36–4.98 (m, 1H) 6.91 (br d, <i>J</i> = 9.51 Hz, 3H) 7.32 (br s, 1H) 8.68 (br s, 1H) 8.86 (br d, <i>J</i> = 4.39 Hz, 1H).</div></div><div id="sec4_10_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i93" class="anchor-spacer"></div><h4 class="article-section__title" id="_i93"> Final Compound <b>15</b>, Benzofuran-5-yl-[3-([1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)-1-piperidyl]methanone</h4><div class="NLM_p last">Prepared in a similar manner to compound <b>11</b>, starting from 1-benzofuran-5-carboxylic acid (0.150 g, 0.93 mmol) and intermediate <b>48a</b> (0.207 g, 1.02 mmol). Purification by flash chromatography on a silica gel, eluting with 0–2% MeOH in DCM, followed by crystallization from diethyl ether, afforded compound <b>15</b> (241 mg, 75%). MS (ESI) <i>m</i>/<i>z</i>: 348 [M + H]<sup>+</sup>. mp 216.2 C. <sup>1</sup>H NMR (360 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 1.71 (br s, 1H) 2.01 (br d, <i>J</i> = 11.34 Hz, 2H) 2.16–2.28 (m, 1H) 3.17–3.34 (m, 2H) 3.60–3.82 (m, 1H) 3.87–4.27 (m, 1H) 4.29–5.00 (m, 1H) 7.04 (dd, <i>J</i> = 2.20, 0.73 Hz, 1H) 7.38 (br d, <i>J</i> = 8.42 Hz, 2H) 7.64 (br s, 1H) 7.77 (s, 1H) 8.08 (s, 1H) 8.48–8.77 (m, 1H) 8.85 (br s, 1H).</div></div><div id="sec4_10_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i94" class="anchor-spacer"></div><h4 class="article-section__title" id="_i94"> Final Compound <b>16</b></h4><div class="NLM_p last">Step 1, intermediate <b>16a</b>, 1<i>H</i>-indol-5-yl-[(3<i>S</i>)-([1,2,4]triazolo[1,5-a]pyrimidin-7-yl)-1-piperidyl]methanone. Intermediate <b>16a</b> was prepared in a similar manner to compound <b>11</b>, starting from indole-5-carboxylic acid (0.4 g, 2.48 mmol) and intermediate <b>48ab</b> (<b>S!</b>, 0.823 g, 2.98 mmol). Purification by flash chromatography on a silica gel, eluting with 0–2% MeOH in DCM, afforded intermediate <b>16a</b> (800 mg, 93%). MS (ESI) <i>m</i>/<i>z</i>: 347 [M + H]<sup>+</sup>. SFC [CHIRALCEL Daicel OJ 4.6 × 500 mm; mobile phase: CO<sub>2</sub>, MeOH with 0.2% 2-propylamine] <i>R</i><sub>t</sub> = 17.37 min. <sup>1</sup>H NMR (360 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 1.68 (br d, <i>J</i> = 12.44 Hz, 1H) 1.82 (br s, 1H) 1.88–2.07 (m, 1H) 2.23 (br d, <i>J</i> = 9.51 Hz, 1H) 3.13 (br s, 1H) 3.25–3.36 (m, 2H) 3.63–3.79 (m, 1H) 4.02 (br s, 1H) 4.28–4.75 (m, 1H) 6.52 (br s, 1H) 7.16 (br d, <i>J</i> = 8.05 Hz, 1H) 7.28–7.37 (m, 1H) 7.37–7.47 (m, 2H) 7.69 (s, 1H) 8.67 (br s, 1H) 8.85 (br d, <i>J</i> = 4.02 Hz, 1H) 11.29 (br s, 1H).</div></div><div id="sec4_10_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i95" class="anchor-spacer"></div><h4 class="article-section__title" id="_i95"> Final Compound <b>16</b>, 1-[5-[(3<i>S</i>)-([1,2,4]Triazolo[1,5-<i>a</i>]pyrimidin-7-yl)piperidine-1-carbonyl]indol-1-yl]ethenone</h4><div class="NLM_p last">NaH (0.012 mg, 0.29 mmol, 60% dispersion in mineral oil) was added at RT to a solution of intermediate <b>16a</b> (100 mg, 0.29 mmol) in DMF (5 mL), and the reaction mixture was stirred at RT for 30 min. After this time, acetyl chloride (0.023 mg, 0.29 mmol) was added to the reaction mixture, which was stirred at RT for another 30 min before being quenched with water. The mixture was extracted with DCM, and then the organic layer was dried over MgSO<sub>4</sub>, filtered, and evaporated under reduced pressure. The resulting residue was purified via Prep HPLC (stationary phase: RP Vydac Denali C18—10 μm, 250 × 5 cm, mobile phase: 0.25% NH<sub>4</sub>HCO<sub>3</sub> solution in water, MeOH), which provided compound <b>16</b> (82 mg, 73%). MS (ESI) <i>m</i>/<i>z</i>: 389 [M + H]<sup>+</sup>. SFC [CHIRALCEL Daicel OJ 4.6 × 500 mm; mobile phase: CO<sub>2</sub>, EtOH with 0.2% 2-propylamine] <i>R</i><sub>t</sub> = 18.02 min and <i>R</i><sub>t</sub> = 18.4 min. <sup>1</sup>H NMR (360 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 1.73 (m, 1H) 1.94–2.07 (m, 1H) 2.23 (br d, <i>J</i> = 9.15 Hz, 1H) 2.67 (s, 3H) 3.19 (br s, 1H) 3.24–3.33 (m, 1H) 3.56–3.82 (m, 2H) 3.91–4.22 (m, 1H) 4.36–4.95 (m, 1H) 6.82 (d, <i>J</i> = 3.66 Hz, 1H) 7.37 (br s, 2H) 7.71–7.75 (m, 1H) 7.94 (br d, <i>J</i> = 3.29 Hz, 1H) 8.35 (br s, 1H) 8.72 (br s, 1H) 8.85 (br s, 1H).</div></div><div id="sec4_10_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i96" class="anchor-spacer"></div><h4 class="article-section__title" id="_i96"> Final Compound <b>17</b>, (1-Methylsulfonylindol-5-yl)-[3-([1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)-1-piperidyl]methanone</h4><div class="NLM_p last">Prepared in a similar manner to compound <b>11</b>, starting from 1-methanesulfonyl-2,3-dihydro-1<i>H</i>-indole-5-carboxylic acid (0.500 g, 2.07 mmol) and intermediate <b>48a</b> (0.687 g, 2.49 mmol). Purification by flash chromatography on a silica gel, eluting with 0–2% MeOH in DCM, followed by crystallization from diethyl ether, afforded compound <b>17</b> (511 mg, 58%). MS (ESI) <i>m</i>/<i>z</i>: 427 [M + H]<sup>+</sup>. mp 238.2 C. SFC [CHIRALCEL Daicel OJ 4.6 × 500 mm; mobile phase: CO<sub>2</sub>, MeOH with 0.2% 2-propylamine] <i>R</i><sub>t</sub> = 17.87 min and <i>R</i><sub>t</sub> = 20.28 min. <sup>1</sup>H NMR (360 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 1.68 (br s, 1H) 1.76–1.92 (m, 1H) 2.02 (br d, <i>J</i> = 9.51 Hz, 1H) 2.22 (br dd, <i>J</i> = 12.99, 3.48 Hz, 1H) 3.05 (s, 3H) 3.14 (br t, <i>J</i> = 8.05 Hz, 3H) 3.37–3.52 (m, 1H) 3.62–3.75 (m, 1H) 3.98 (br t, <i>J</i> = 8.60 Hz, 2H) 4.18 (br s, 1H) 4.67 (br s, 1H) 7.04–7.31 (m, 3H) 7.33 (br s, 1H) 8.64 (br d, <i>J</i> = 11.34 Hz, 1H) 8.86 (br d, <i>J</i> = 4.03 Hz, 1H).</div></div><div id="sec4_10_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i97" class="anchor-spacer"></div><h4 class="article-section__title" id="_i97"> Final Compound <b>18</b>, 4,5,6,7-Tetrahydrobenzothiophen-2-yl-[(3<i>S</i>)-([1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)-1-piperidyl]methanone</h4><div class="NLM_p last">Prepared in a similar manner to compound <b>11</b>, starting from 4,5,6,7-tetrahydrobenzo[<i>b</i>]thiophene-2-carboxylic acid (0.100 g, 0.55 mmol) and intermediate <b>48ab</b> (0.687 g, 2.49 mmol). Purification by flash chromatography on a silica gel, eluting with 0–2% MeOH in DCM, followed by crystallization from diethyl ether, afforded compound <b>17</b> (511 mg, 58%). MS (ESI) <i>m</i>/<i>z</i>: 427 [M + H]<sup>+</sup>. mp 238.2 C. SFC [CHIRALCEL Daicel OJ 4.6 × 500 mm; mobile phase: CO<sub>2</sub>, EtOH with 0.2% 2-propylamine] <i>R</i><sub>t</sub> = 17.87 min and <i>R</i><sub>t</sub> = 20.28 min. <sup>1</sup>H NMR (360 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 1.68 (br s, 1H) 1.76–1.92 (m, 1H) 2.02 (br d, <i>J</i> = 9.51 Hz, 1H) 2.22 (br dd, <i>J</i> = 12.99, 3.48 Hz, 1H) 3.05 (s, 3H) 3.14 (br t, <i>J</i> = 8.05 Hz, 3H) 3.37–3.52 (m, 1H) 3.62–3.75 (m, 1H) 3.98 (br t, <i>J</i> = 8.60 Hz, 2H) 4.18 (br s, 1H) 4.67 (br s, 1H) 7.04–7.31 (m, 3H) 7.33 (br s, 1H) 8.64 (br d, <i>J</i> = 11.34 Hz, 1H) 8.86 (br d, <i>J</i> = 4.03 Hz, 1H).</div></div><div id="sec4_10_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i98" class="anchor-spacer"></div><h4 class="article-section__title" id="_i98"> Final Compound <b>19</b>, (1-Methylsulfonylindol-5-yl)-[(3<i>R</i>)-3-([1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)-1-piperidyl]methanone</h4><div class="NLM_p last">Prepared in a similar manner to compound <b>18</b>, starting from 1-methanesulfonyl-2,3-dihydro-1<i>H</i>-indole-5-carboxylic acid (0.145 g, 0.60 mmol) and intermediate <b>48aa</b> (<b>R!</b>, 0.2 g, 0.72 mmol). Purification by flash chromatography on a silica gel, eluting with 0–4% MeOH in DCM afforded compound <b>19</b> (120 mg, 46.6%). MS (ESI). SFC [CHIRALCEL Daicel OJ 4.6 × 500 mm; mobile phase: CO<sub>2</sub>, MeOH with 0.2% 2-propylamine] <i>R</i><sub>t</sub> = 20.21 min. OR = −10.5° (589 nm, 20 °C, 0.438 w/v %; MeOH). <sup>1</sup>H NMR (360 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 1.68 (br s, 1H) 1.76–1.92 (m, 1H) 2.02 (br d, <i>J</i> = 9.51 Hz, 1H) 2.22 (br dd, <i>J</i> = 12.99, 3.48 Hz, 1H) 3.05 (s, 3H) 3.14 (br t, <i>J</i> = 8.05 Hz, 3H) 3.37–3.52 (m, 1H) 3.62–3.75 (m, 1H) 3.98 (br t, <i>J</i> = 8.60 Hz, 2H) 4.18 (br s, 1H) 4.67 (br s, 1H) 7.04–7.31 (m, 3H) 7.33 (br s, 1H) 8.64 (br d, <i>J</i> = 11.34 Hz, 1H) 8.86 (br d, <i>J</i> = 4.03 Hz, 1H).</div></div><div id="sec4_10_36" class="NLM_sec NLM_sec_level_3"><div id="ac_i99" class="anchor-spacer"></div><h4 class="article-section__title" id="_i99"> Final Compound <b>20</b>, (1-Methylsulfonylindol-5-yl)-[(3<i>S</i>)-3-([1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)-1-piperidyl]methanone</h4><div class="NLM_p last">Prepared in a similar manner to compound <b>18</b>, starting from 1-methanesulfonyl-2,3-dihydro-1<i>H</i>-indole-5-carboxylic acid (0.145 g, 0.60 mmol) and intermediate <b>48ab</b> (<b>S!</b>, 0.2 g, 0.72 mmol). Purification by flash chromatography on a silica gel, eluting with 0–4% MeOH in DCM, afforded compound <b>20</b> (128 mg, 46.6%). MS (ESI). SFC [CHIRALCEL Daicel OJ 4.6 × 500 mm; mobile phase: CO<sub>2</sub>, MeOH with 0.2% 2-propylamine] <i>R</i><sub>t</sub> = 17.88 min. OR = + 9.36° (589 nm, 20 °C, 0.417 w/v %; MeOH). <sup>1</sup>H NMR (360 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 1.68 (br s, 1H) 1.76–1.92 (m, 1H) 2.02 (br d, <i>J</i> = 9.51 Hz, 1H) 2.22 (br dd, <i>J</i> = 12.99, 3.48 Hz, 1H) 3.05 (s, 3H) 3.14 (br t, <i>J</i> = 8.05 Hz, 3H) 3.37–3.52 (m, 1H) 3.62–3.75 (m, 1H) 3.98 (br t, <i>J</i> = 8.60 Hz, 2H) 4.18 (br s, 1H) 4.67 (br s, 1H) 7.04–7.31 (m, 3H) 7.33 (br s, 1H) 8.64 (br d, <i>J</i> = 11.34 Hz, 1H) 8.86 (br d, <i>J</i> = 4.03 Hz, 1H).</div></div><div id="sec4_10_37" class="NLM_sec NLM_sec_level_3"><div id="ac_i100" class="anchor-spacer"></div><h4 class="article-section__title" id="_i100"> Final Compound <b>21</b>, 4,5,6,7-Tetrahydrobenzothiophen-2-yl-[(2<i>S</i>)-([1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)morpholin-4-yl]methanone</h4><div class="NLM_p last">Prepared in a similar manner to compound <b>11</b>, starting from 4,5,6,7-tetrahydrobenzo[<i>b</i>]thiophene-2-carboxylic acid (0.175 g, 0.96 mmol) and intermediate <b>48bb</b> (<b>S!</b> 0.32 g, 1.15 mmol). Purification by flash chromatography on a silica gel, eluting with 0–2% MeOH in DCM, followed by SFC purification [CHIRALCEL Daicel OJ 20 × 250 mm; mobile phase: CO<sub>2</sub>, MeOH with 0.2% 2-propylamine] provided compound <b>21</b> (205 mg, 58%). <i>R</i><sub>t</sub> = 8.06 min. (ESI) <i>m</i>/<i>z</i>: 370 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (360 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 1.78 (br s, 4H) 2.63 (br s, 2H) 2.74 (br s, 2H) 3.08 (br s, 1H) 3.13–3.21 (m, 1H) 3.88 (br t, <i>J</i> = 11.16 Hz, 1H) 4.05–4.22 (m, 1H) 4.27 (br d, <i>J</i> = 13.17 Hz, 1H) 4.99 (br d, <i>J</i> = 12.44 Hz, 1H) 5.35 (br d, <i>J</i> = 9.51 Hz, 1H) 7.40 (br d, <i>J</i> = 4.03 Hz, 2H) 8.78 (s, 1H) 8.94 (br d, <i>J</i> = 4.02 Hz, 1H).</div></div><div id="sec4_10_38" class="NLM_sec NLM_sec_level_3"><div id="ac_i101" class="anchor-spacer"></div><h4 class="article-section__title" id="_i101"> Final Compound <b>22</b>, 4,5,6,7-Tetrahydrobenzothiophen-2-yl-[3-([1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)piperazin-1-yl]methanone</h4><div class="NLM_p last">Prepared in a similar manner to compound <b>11</b>, starting from 4,5,6,7-tetrahydrobenzo[<i>b</i>]thiophene-2-carboxylic acid (0.484 g, 2.66 mmol) and intermediate <b>48d</b> (1.00 g, 3.19 mmol). Purification by flash chromatography on a silica gel, eluting with 0–6% MeOH in DCM, followed by crystallization from EtOAc, provided compound <b>22</b> (707 mg, 72%). (ESI) <i>m</i>/<i>z</i>: 369 [M + H]<sup>+</sup>. mp 193.45 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 1.72–1.83 (m, 4H) 2.57 (br t, <i>J</i> = 6.06 Hz, 2H) 2.71 (br t, <i>J</i> = 5.65 Hz, 3H) 2.89–2.98 (m, 1H) 3.13 (dt, <i>J</i> = 12.41, 3.68 Hz, 1H) 3.32 (ddd, <i>J</i> = 13.02, 9.79, 3.43 Hz, 1H) 3.39 (dd, <i>J</i> = 12.92, 8.48 Hz, 1H) 4.02–4.10 (m, 1H) 4.57 (ddd, <i>J</i> = 12.92, 3.63, 1.21 Hz, 1H) 4.67 (dd, <i>J</i> = 8.48, 3.63 Hz, 1H) 7.06 (s, 1H) 7.38 (d, <i>J</i> = 4.44 Hz, 1H) 8.52 (s, 1H) 8.82 (d, <i>J</i> = 4.44 Hz, 1H).</div></div><div id="sec4_10_39" class="NLM_sec NLM_sec_level_3"><div id="ac_i102" class="anchor-spacer"></div><h4 class="article-section__title" id="_i102"> Final Compound <b>23</b>, [4-Methyl-3-([1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)piperazin-1-yl]-(4,5,6,7-tetrahydrobenzothiophen-2-yl)methanone</h4><div class="NLM_p last">A solution of compound <b>22</b> (0.2 g, 0.54 mmol), paraformaldehyde (0.020 g, 0.66 mmol), and NaBH(OAc)<sub>3</sub> (0.69 g, 3.25 mmol) in DCM (20 mL) was stirred at RT for 32 h. The mixture was then filtered through Celite, and the filtrate was washed with water. The organic layer was filtered over Extrelut, and the filtrate was evaporated under reduced pressure. The resulting residue was purified by recrystallization from DIPE to provide compound <b>23</b> (192 mg, 92%). (ESI) <i>m</i>/<i>z</i>: 383 [M + H]<sup>+</sup>. mp 174.6 °C. <sup>1</sup>H NMR (360 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 1.68–1.81 (m, 4H) 2.19 (s, 3H) 2.35–2.46 (m, 1H) 2.56–2.62 (m, 2H) 2.71 (br s, 2H) 3.10 (br d, <i>J</i> = 11.71 Hz, 2H) 3.21–3.32 (m, 1H) 4.19 (dd, <i>J</i> = 9.88, 2.93 Hz, 1H) 4.28 (br d, <i>J</i> = 12.81 Hz, 1H) 4.51 (br d, <i>J</i> = 13.17 Hz, 1H) 7.28 (s, 1H) 7.41 (d, <i>J</i> = 4.39 Hz, 1H) 8.76 (s, 1H) 8.91 (d, <i>J</i> = 4.39 Hz, 1H).</div></div><div id="sec4_10_40" class="NLM_sec NLM_sec_level_3"><div id="ac_i103" class="anchor-spacer"></div><h4 class="article-section__title" id="_i103"> Final Compound <b>24</b>, 1-[4-(4,5,6,7-Tetrahydrobenzothiophene-2-carbonyl)-2-([1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)piperazin-1-yl]ethenone</h4><div class="NLM_p last">Acetyl chloride (0.07 g, 0.89 mmol) was added at 0 °C to a solution of compound <b>22</b> (0.30 g, 0.81 mmol) and DIPEA (0.127 g, 0.98 mmol, 0.75 g/mL) in DCM (50 mL). The reaction mixture was allowed to reach RT and stirred until the starting material was consumed, as indicated by LC/MS and TLC. The reaction mixture was quenched with a saturated aqueous NaHCO<sub>3</sub> solution, and then the organic layer was separated, dried over MgSO<sub>4</sub>, filtered, and concentrated under reduced pressure. The resulting residue was purified by flash chromatography on a silica gel, eluting with 0–2% MeOH in DCM, and the resulting material triturated in diethyl ether to provide compound <b>24</b> (293 mg, 87.6%). (ESI) <i>m</i>/<i>z</i>: 411 [M + H]<sup>+</sup>. mp 186 °C. <sup>1</sup>H NMR (360 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 1.61–1.78 (m, 5H) 1.98 (s, 1H) 2.20 (s, 3H) 2.37–2.47 (m, 2H) 2.52–2.66 (m, 2H) 3.37–3.47 (m, 1H) 3.85–3.99 (m, 1H) 3.99–4.09 (m, 2H) 4.12–4.22 (m, 1H) 4.80 (br s, 1H) 5.01 (br s, 1H) 5.66–5.75 (m, 1H) 5.81 (br s, 1H) 6.21–6.98 (m, 1H) 7.25 (br d, <i>J</i> = 4.03 Hz, 1H) 7.35 (d, <i>J</i> = 4.39 Hz, 1H) 8.71 (br s, 1H) 8.81–8.88 (m, 1H) (80/20 mixture of rotamers).</div></div><div id="sec4_10_41" class="NLM_sec NLM_sec_level_3"><div id="ac_i104" class="anchor-spacer"></div><h4 class="article-section__title" id="_i104"> Final Compound <b>25</b>, 4,5,6,7-Tetrahydrobenzothiophen-2-yl-[3-([1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)azepan-1-yl]methanone</h4><div class="NLM_p last">Prepared in a similar manner to compound <b>5</b>, starting from 4,5,6,7-tetrahydrobenzo[<i>b</i>]thiophene-2-carboxylic acid [40133-07-1] (1.1 equiv) and intermediate <b>48c</b> (1 equiv). (ESI) <i>m</i>/<i>z</i>: 382 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6,</sub> 100 °C, mixture of rotamers): δ ppm 1.69–1.87 (m, 6H) 1.90–2.00 (m, 3H) 2.06–2.14 (m, 1H) 2.66–2.78 (m, 4H) 3.68–3.74 (m, 1H) 3.80–3.86 (m, 1H) 3.89–3.99 (m, 2H) 4.17–4.23 (m, 1H) 7.06 (s, 1H) 7.22–7.26 (m, 1H) 8.56–8.58 (m, 1H) 8.76–8.85 (m, 1H).</div></div><div id="sec4_10_42" class="NLM_sec NLM_sec_level_3"><div id="ac_i105" class="anchor-spacer"></div><h4 class="article-section__title" id="_i105"> Final Compound <b>26</b>, [3-(5-Methyl-[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)-1-piperidyl]-(4,5,6,7-tetrahydrobenzothiophen-2-yl)methanone</h4><div class="NLM_p last">Prepared in a similar manner to compound <b>11</b>, starting from 4,5,6,7-tetrahydrobenzo[<i>b</i>]thiophene-2-carboxylic acid (0.100 g, 0.55 mmol) and intermediate <b>48g</b> (0.687 g, 2.49 mmol). Purification by flash chromatography on a silica gel, eluting with 0–2% MeOH in DCM, followed by crystallization from diethyl ether, afforded compound <b>26</b> (511 mg, 58%). (ESI) <i>m</i>/<i>z</i>: 382 [M + H]<sup>+</sup>. mp 201.2 °C. <sup>1</sup>H NMR (360 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 1.60–1.81 (m, 6H) 1.88 (br d, <i>J</i> = 13.54 Hz, 1H) 1.99 (br d, <i>J</i> = 12.08 Hz, 1H) 2.20 (br d, <i>J</i> = 13.17 Hz, 1H) 2.58 (br t, <i>J</i> = 5.67 Hz, 2H) 2.62 (s, 3H) 2.70 (br d, <i>J</i> = 5.12 Hz, 2H) 3.11 (br s, 1H) 3.58–3.67 (m, 1H) 4.29 (br d, <i>J</i> = 12.08 Hz, 1H) 4.60 (br d, <i>J</i> = 12.81 Hz, 1H) 7.26 (s, 2H) 8.60 (s, 1H).</div></div><div id="sec4_10_43" class="NLM_sec NLM_sec_level_3"><div id="ac_i106" class="anchor-spacer"></div><h4 class="article-section__title" id="_i106"> Final Compound <b>27</b>, [3-(5-Isopropyl-[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)-1-piperidyl]-(4,5,6,7-tetrahydrobenzothiophen-2-yl)methanone</h4><div class="NLM_p last">Prepared in a similar manner to compound <b>11</b>, starting from 4,5,6,7-tetrahydrobenzo[<i>b</i>]thiophene-2-carboxylic acid (0.076 g, 0.42 mmol) and intermediate <b>48j</b> (0.160 g, 0.5 mmol). Purification by flash chromatography on a silica gel, eluting with 0.5–1% MeOH in DCM, followed by crystallization from diisopropyl ether, afforded compound <b>27</b> (54 mg, 31%). ESI <i>m</i>/<i>z</i>: 410 [M + H]<sup>+</sup>. mp 158 °C. <sup>1</sup>H NMR (360 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 1.30 (d, <i>J</i> = 6.95 Hz, 6H) 1.74 (br s, 5H) 1.96–2.31 (m, 3H) 2.54–2.59 (m, 2H) 2.70 (br d, <i>J</i> = 4.76 Hz, 2H) 3.11–3.24 (m, 2H) 4.11–4.36 (m, 1H) 4.45–4.63 (m, 1H) 7.24 (s, 1H) 7.28 (s, 1H) 8.62 (s, 1H).</div></div><div id="sec4_10_44" class="NLM_sec NLM_sec_level_3"><div id="ac_i107" class="anchor-spacer"></div><h4 class="article-section__title" id="_i107"> Final Compound <b>28</b>, [3-(5-Methoxy-[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)-1-piperidyl]-(4,5,6,7-tetrahydrobenzothiophen-2-yl)methanone</h4><div class="NLM_p last">Prepared in a similar manner to compound <b>11</b>, starting from 4,5,6,7-tetrahydrobenzo[<i>b</i>]thiophene-2-carboxylic acid (0.101 g, 0.56 mmol) and intermediate <b>48e</b> (0.130 g, 0.56 mmol). Purification by flash chromatography on a silica gel, eluting with 0.5–1% MeOH in DCM, followed by crystallization from diethyl ether, afforded compound <b>28</b> (86 mg, 39%). ESI <i>m</i>/<i>z</i>: 398 [M + H]<sup>+</sup>. mp 165.5 °C. <sup>1</sup>H NMR (360 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 1.58–1.88 (m, 7H) 1.91–2.06 (m, 1H) 2.16 (br d, <i>J</i> = 12.81 Hz, 1H) 2.53–2.61 (m, 2H) 2.65–2.76 (m, 2H) 3.10 (br s, 1H) 3.20–3.30 (m, 1H) 3.51–3.60 (m, 1H) 3.99 (s, 3H) 4.26 (br d, <i>J</i> = 12.08 Hz, 1H) 4.55 (br d, <i>J</i> = 13.17 Hz, 1H) 6.80 (s, 1H) 7.25 (br s, 1H) 8.46 (s, 1H).</div></div><div id="sec4_10_45" class="NLM_sec NLM_sec_level_3"><div id="ac_i108" class="anchor-spacer"></div><h4 class="article-section__title" id="_i108"> Final Compound <b>29</b>, 7-[1-(4,5,6,7-Tetrahydrobenzothiophene-2-carbonyl)-3-piperidyl]-[1,2,4]triazolo[1,5-<i>a</i>]pyrimidine-5-carbonitrile</h4><div class="NLM_p last">Prepared in a similar manner to compound <b>11</b>, starting from 4,5,6,7-tetrahydrobenzo[<i>b</i>]thiophene-2-carboxylic acid (0.157 g, 0.86 mmol) and intermediate <b>48f</b> (0.135 g, 1.04 mmol). Purification by flash chromatography on a silica gel, eluting with 0–1.5% MeOH in DCM, followed by crystallization from diethyl ether, afforded compound <b>29</b> (73 mg, 21%). ESI <i>m</i>/<i>z</i>: 393 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (360 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 1.58–1.82 (m, 5H) 1.89 (br d, <i>J</i> = 13.17 Hz, 1H) 1.97–2.14 (m, 1H) 2.20 (br d, <i>J</i> = 13.17 Hz, 1H) 2.52–2.61 (m, 2H) 2.70 (br d, <i>J</i> = 5.85 Hz, 2H) 3.11 (br s, 1H) 3.25–3.32 (m, 1H) 3.69–3.79 (m, 1H) 4.31 (br d, <i>J</i> = 12.44 Hz, 1H) 4.61 (br d, <i>J</i> = 12.81 Hz, 1H) 7.23 (s, 1H) 8.00 (s, 1H) 8.99 (s, 1H).</div></div><div id="sec4_10_46" class="NLM_sec NLM_sec_level_3"><div id="ac_i109" class="anchor-spacer"></div><h4 class="article-section__title" id="_i109"> Final Compound <b>30</b>, 4,5,6,7-Tetrahydrobenzothiophen-2-yl-[3-[5-(trifluoromethyl)-[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl]-1-piperidyl]methanone</h4><div class="NLM_p last">Prepared in a similar manner to compound <b>11</b>, starting from 4,5,6,7-tetrahydrobenzo[<i>b</i>]thiophene-2-carboxylic acid (0.062 g, 0.34 mmol) and intermediate <b>48k</b> (0.140 g, 0.41 mmol). Purification by flash chromatography on a silica gel, eluting with 0–0.5% MeOH in DCM, afforded compound <b>30</b> (110 mg, 74%). ESI <i>m</i>/<i>z</i>: 436 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (360 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 1.04 (d, <i>J</i> = 6.22 Hz, 1H) 1.61–1.80 (m, 5H) 1.87 (br d, <i>J</i> = 12.81 Hz, 1H) 2.10–2.25 (m, 2H) 2.52–2.61 (m, 3H) 2.65–2.76 (m, 2H) 3.02–3.24 (m, 1H) 3.36–3.53 (m, 1H) 3.75–3.84 (m, 1H) 4.30 (br d, <i>J</i> = 13.90 Hz, 1H) 4.59 (br d, <i>J</i> = 10.25 Hz, 1H) 7.24 (s, 1H) 7.80 (s, 1H) 8.96 (s, 1H).</div></div><div id="sec4_10_47" class="NLM_sec NLM_sec_level_3"><div id="ac_i110" class="anchor-spacer"></div><h4 class="article-section__title" id="_i110"> Final Compound <b>31</b>, [3-[5-(Difluoromethyl)-[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl]-1-piperidyl]-(4,5,6,7-tetrahydrobenzothiophen-2-yl)methanone</h4><div class="NLM_p last">Prepared in a similar manner to compound <b>11</b>, starting from 4,5,6,7-tetrahydrobenzo[<i>b</i>]thiophene-2-carboxylic acid (0.060 g, 0.33 mmol) and intermediate <b>48i</b> (0.100 g, 0.39 mmol). Purification by prep HPLC on a RP Vydac Denali column (C18—10 μm, 250 g, 5 cm) and using as the mobile phase a mixture of a 0.25% NH<sub>4</sub>HCO<sub>3</sub> solution in water and MeOH afforded compound <b>31</b> (130 mg, 95%). ESI <i>m</i>/<i>z</i>: 418 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 1.68–1.82 (m, 5H) 1.88 (dt, <i>J</i> = 13.52, 3.73 Hz, 1H) 2.06–2.17 (m, 1H) 2.21–2.28 (m, 1H) 2.56 (t, <i>J</i> = 5.85 Hz, 2H) 2.71 (t, <i>J</i> = 5.85 Hz, 2H) 3.19–3.28 (m, 1H) 3.54 (dd, <i>J</i> = 13.12, 9.89 Hz, 1H) 3.75–3.82 (m, 1H) 4.23 (br d, <i>J</i> = 13.32 Hz, 1H) 4.56 (dd, <i>J</i> = 13.32, 3.63 Hz, 1H) 7.03 (t, <i>J</i> = 54.50 Hz, 1H) 7.13 (s, 1H) 7.51 (s, 1H) 8.75 (s, 1H).</div></div><div id="sec4_10_48" class="NLM_sec NLM_sec_level_3"><div id="ac_i111" class="anchor-spacer"></div><h4 class="article-section__title" id="_i111"> Final Compound <b>32</b>, 3-[5-(Difluoromethyl)pyrazolo[1,5-<i>a</i>]pyrimidin-7-yl]-1-piperidyl]-(4,5,6,7-tetrahydrobenzothiophen-2-yl)methanone</h4><div class="NLM_p last">Prepared in a similar manner to compound <b>11</b>, starting from 4,5,6,7-tetrahydrobenzo[<i>b</i>]thiophene-2-carboxylic acid (0.108 g, 0.59 mmol) and intermediate <b>48o</b> (0.150 g, 0.59 mmol). Purification by flash chromatography on a silica gel, eluting with 0–30% EtOAc in heptane, followed by crystallization from diisopropyl ether, afforded compound <b>32</b> (190 mg, 77%). ESI <i>m</i>/<i>z</i> 417 [M + H]<sup>+</sup>. mp 192.8 °C. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 1.64–1.81 (m, 5H) 1.84–1.91 (m, 1H) 2.05–2.12 (m, 1H) 2.21–2.26 (m, 1H) 2.54–2.61 (m, 2H) 2.67–2.75 (m, 2H) 3.17–3.23 (m, 1H) 3.46 (dd, <i>J</i> = 12.76, 10.12 Hz, 1H) 3.83–3.88 (m, 1H) 4.26 (br d, <i>J</i> = 13.21 Hz, 1H) 4.57 (dd, <i>J</i> = 12.76, 3.67 Hz, 1H) 6.90 (t, <i>J</i> = 54.87 Hz, 1H) 6.87–6.90 (m, 1H) 7.19 (d, <i>J</i> = 8.80 Hz, 2H) 8.34 (dd, <i>J</i> = 2.35, 0.88 Hz, 1H).</div></div><div id="sec4_10_49" class="NLM_sec NLM_sec_level_3"><div id="ac_i112" class="anchor-spacer"></div><h4 class="article-section__title" id="_i112"> Final Compound <b>33</b>, [3-(7-Methylimidazo[1,2-<i>a</i>]pyrimidin-5-yl)-1-piperidyl]-(4,5,6,7-tetrahydrobenzothiophen-2-yl)methanone</h4><div class="NLM_p last">Prepared in a similar manner to compound <b>11</b>, starting from 4,5,6,7-tetrahydrobenzo[<i>b</i>]thiophene-2-carboxylic acid (0.131 g, 0.72 mmol) and intermediate <b>48h</b> (0.250 g, 0.86 mmol). Purification by flash chromatography on a silica gel, eluting with 0–3% MeOH in DCM, followed by crystallization from diethyl ether, afforded compound <b>33</b> (168 mg, 61%). ESI <i>m</i>/<i>z</i>: 381 [M + H]<sup>+</sup>. mp 192.7 °C. <sup>1</sup>H NMR (360 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 1.63–1.91 (m, 8H); 2.17 (br d, <i>J</i> = 12.44 Hz, 1H); 2.52 (s, 3H); 2.53–2.59 (m, 2H); 2.64–2.81 (m, 2H); 3.22 (br d, <i>J</i> = 16.10 Hz, 2H); 3.27–3.31 (m, 1H); 4.32 (br d, <i>J</i> = 12.81 Hz, 1H); 4.51 (br d, <i>J</i> = 11.34 Hz, 1H); 6.90 (s, 1H); 7.14 (s, 1H); 7.67 (s, 1H); 8.01 (br s, 1H).</div></div><div id="sec4_10_50" class="NLM_sec NLM_sec_level_3"><div id="ac_i113" class="anchor-spacer"></div><h4 class="article-section__title" id="_i113"> Final Compound <b>34</b>, [3-(1<i>H</i>-Pyrrolo[2,3-<i>b</i>]pyridin-4-yl)-1-piperidyl]-(4,5,6,7-tetrahydrobenzothiophen-2-yl)methanone</h4><div class="NLM_p last">Prepared in a similar manner to compound <b>11</b>, starting from 4,5,6,7-tetrahydrobenzo[<i>b</i>]thiophene-2-carboxylic acid (0.083 g, 0.46 mmol) and intermediate <b>48n</b> (0.150 g, 0.55 mmol). Purification by crystallization from methanol afforded compound <b>34</b> (114 mg, 68%). ESI <i>m</i>/<i>z</i>: 366 [M + H]<sup>+</sup>. mp 275.7 °C. <sup>1</sup>H NMR (360 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 1.59–1.87 (m, 6H); 1.89–2.07 (m, 2H); 2.52–2.58 (m, 2H); 2.67–2.75 (m, 2H); 3.02–3.32 (m, 3H); 4.37 (br d, <i>J</i> = 11.71 Hz, 2H); 6.57 (br s, 1H); 6.97 (d, <i>J</i> = 5.12 Hz, 1H); 7.10 (s, 1H); 7.44 (t, <i>J</i> = 3.11 Hz, 1H); 8.15 (d, <i>J</i> = 4.76 Hz, 1H); 11.65 (br s, 1H).</div></div><div id="sec4_10_51" class="NLM_sec NLM_sec_level_3"><div id="ac_i114" class="anchor-spacer"></div><h4 class="article-section__title" id="_i114"> Final Compound <b>35</b>, [3-(2,5-Dimethyl-[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)-1-piperidyl]-(4,5,6,7-tetrahydrobenzothiophen-2-yl)methanone</h4><div class="NLM_p last">Prepared in a similar manner to compound <b>11</b>, starting from 4,5,6,7-tetrahydrobenzo[<i>b</i>]thiophene-2-carboxylic acid (0.065 g, 0.36 mmol) and intermediate <b>48l</b> (0.130 g, 0.43 mmol). Purification by flash chromatography on a silica gel, eluting with 0.5–1% MeOH in DCM, followed by crystallization from diisopropyl ether, afforded compound <b>35</b> (12 mg, 39%). ESI <i>m</i>/<i>z</i>: 396 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 1.62–1.70 (m, 1H) 1.81 (br s, 6H) 1.86–1.91 (m, 1H) 1.96–2.04 (m, 1H) 2.17–2.24 (m, 1H) 2.46 (s, 3H) 2.54–2.56 (m, 2H) 2.57 (s, 3H) 2.71 (br t, <i>J</i> = 6.19 Hz, 2H) 3.19 (ddd, <i>J</i> = 13.33, 11.11, 3.30 Hz, 1H) 3.43 (dd, <i>J</i> = 13.05, 10.07 Hz, 1H) 3.59 (tt, <i>J</i> = 10.30, 3.90 Hz, 1H) 4.21 (dt, <i>J</i> = 13.29, 4.00 Hz, 1H) 4.53 (ddt, <i>J</i> = 12.98, 3.65, 1.59, 1.59 Hz, 1H) 7.04 (s, 1H) 7.05 (s, 1H).</div></div><div id="sec4_10_52" class="NLM_sec NLM_sec_level_3"><div id="ac_i115" class="anchor-spacer"></div><h4 class="article-section__title" id="_i115"> Final Compound <b>36</b>, [3-[5-Methyl-2-(trifluoromethyl)-[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl]-1-piperidyl]-(4,5,6,7-tetrahydrobenzothiophen-2-yl)methanone</h4><div class="NLM_p last">Prepared in a similar manner to compound <b>11</b>, starting from 4,5,6,7-tetrahydrobenzo[<i>b</i>]thiophene-2-carboxylic acid (0.038 g, 0.21 mmol) and intermediate <b>48m</b> (0.09 g, 0.25 mmol). Purification by flash chromatography on a silica gel, eluting with 0.5–1% MeOH in DCM, afforded compound <b>36</b> (75 mg, 78%). ESI <i>m</i>/<i>z</i>: 450 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 1.63–1.81 (m, 5H); 1.85–2.06 (m, 2H); 2.19–2.26 (m, 1H); 2.50–2.55 (m, 2H); 2.64–2.72 (m, 5H); 3.16–3.26 (m, 1H); 3.44 (dd, <i>J</i> = 12.9, 10.1 Hz, 1H); 3.57–3.67 (m, 1H); 4.18–4.25 (m, 1H); 4.52–4.57 (m, 1H); 7.02 (s, 1H); 7.38 (s, 1H).</div></div><div id="sec4_10_53" class="NLM_sec NLM_sec_level_3"><div id="ac_i116" class="anchor-spacer"></div><h4 class="article-section__title" id="_i116"> Final Compound <b>37</b>, 5-(Difluoromethyl)-7-[1-[(3,4-dimethoxyphenyl)methyl]-3-piperidyl]-[1,2,4]triazolo[1,5-<i>a</i>]pyrimidine</h4><div class="NLM_p last">DIPEA (0.52 mL, 3.02 mmol) and 3,4-dimethoxybenzaldehyde [120-14-9] (0.076 g, 0.46 mmol) were added to a solution of intermediate <b>48i</b> (0.15 g, 0.46 mmol) in DCM (30 mL). The reaction mixture was stirred at RT for 1 h, and then sodiumtriacetoxyborohydride [56553-60-7] (0.15 g, 1.84) was added. The reaction mixture was stirred at RT for 2 h, and then it was washed with sat. aq NaHCO<sub>3</sub> solution, dried over MgSO<sub>4</sub>, filtered, and concentrated by evaporation. The resulting residue was purified by flash column chromatography on a silica gel using a mixture of EtOAc and heptane, 80/20, as the eluent to provide compound 37 (160 mg, 86%). ESI <i>m</i>/<i>z</i>: 404 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, 80 °C): δ ppm 1.60–1.71 (m, 2H); 1.79–1.89 (m, 1H); 2.00–2.08 (m, 1H); 2.35–2.43 (m, 1H); 2.59–2.68 (m, 2H); 2.96–3.01 (m, 1H); 3.50 (s, 2H); 3.73 (s, 3H); 3.76 (s, 3H); 3.78–3.88 (m, 1H); 6.83 (dd, <i>J</i> = 8.1, 1.61 Hz, 1H); 6.85–6.89 (d, <i>J</i> = 8.5 Hz, 1H); 7.09 (t, <i>J</i> = 54.5 Hz, 1H); 6.93 (d, <i>J</i> = 1.84 Hz, 1H); 7.66 (s, 1H) 8.72 (s, 1H).</div></div><div id="sec4_10_54" class="NLM_sec NLM_sec_level_3"><div id="ac_i117" class="anchor-spacer"></div><h4 class="article-section__title" id="_i117"> Final Compound <b>38</b>, 7-[1-(Benzofuran-5-ylmethyl)-3-piperidyl]-5-(difluoromethyl)-[1,2,4]triazolo[1,5-<i>a</i>]pyrimidine</h4><div class="NLM_p last">Prepared in a similar manner to compound <b>37</b>, starting from benzo[<i>b</i>]furan-5-carboxaldehyde (0.10 g, 0.69 mmol) and intermediate <b>48i</b> (0.15 g, 0.46 mmol). Purification by Prep HPLC on a RP Vydac Denali column (C18—10 μm, 250 g, 5 cm) and using as mobile phase a mixture of a 0.25% NH<sub>4</sub>HCO<sub>3</sub> solution in water and ACN afforded the title compound as a free base, which was then dissolved in DIPE and treated with HCl to afford compound <b>38</b> as a HCl salt (180 mg, 86%). ESI <i>m</i>/<i>z</i>: 384 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (360 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 1.80–2.17 (m, 4H); 3.03 (br d, <i>J</i> = 11.71 Hz, 1H); 3.43–3.55 (m, 3H); 4.24 (br t, <i>J</i> = 12.08 Hz, 1H); 4.38–4.52 (m, 2H); 7.14 (t, <i>J</i> = 54.52 Hz, 1H); 7.01–7.05 (m, 1H); 7.55 (s, 1H); 7.57 (d, <i>J</i> = 9.88 Hz, 1H); 7.69 (d, <i>J</i> = 8.42 Hz, 1H); 7.91 (s, 1H); 8.08 (d, <i>J</i> = 2.20 Hz, 1H); 8.85 (s, 1H); 11.06 (br s, 1H). Anal. Calcd (C<sub>20</sub>H<sub>19</sub>F<sub>2</sub>N<sub>5</sub>O): C, H, N.</div></div><div id="sec4_10_55" class="NLM_sec NLM_sec_level_3"><div id="ac_i118" class="anchor-spacer"></div><h4 class="article-section__title" id="_i118"> Final Compound <b>39</b>, 7-[1-[(2-Chloro-6-methyl-4-pyridyl)methyl]-3-piperidyl]-5-(difluoromethyl)-[1,2,4]triazolo[1,5-<i>a</i>]pyrimidine</h4><div class="NLM_p last">A solution of 2-chloro-4-chloromethyl-6-methylpyridine [162046-59-5] (0.81 g, 4.6 mmol), intermediate <b>48i</b> (1.00 g, 3.07 mmol) and DIPEA (2.13 mL, 12.4 mmol) in DMF (100 mL) was stirred and heated at 40 °C for 4 h. The reaction mixture was cooled to RT, and then it was evaporated under reduced pressure. The resulting residue was dissolved in DCM, washed with a satd aq NaHCO<sub>3</sub> soln (1×), dried over MgSO<sub>4</sub>, filtered, and evaporated <i>in vacuo</i>. The resulting residue was purified by prep HPLC on a RP Vydac Denali column (C18—10 μm, 250 g, 5 cm) and using as a mobile phase a mixture of a 0.25% NH<sub>4</sub>HCO<sub>3</sub> solution in water and ACN afforded the title compound as a racemic mixture (1.15 g, 95%). ESI <i>m</i>/<i>z</i>: 393 [M + H]<sup>+</sup>. Separation in enantiomers by preparative SFC [CHIRALPAK Daicel AD 20 × 250 mm; mobile phase: CO<sub>2</sub>, EtOH with 0.2% 2-propylamine] afforded compounds <b>39</b> (R*) (430 mg, 37%) <i>R</i><sub>t</sub> = 3.75 min, OR = +39.71° (589 nm, 20 °C, 0.345 w/v %; DMF), mp 164.56 °C, <sup>1</sup>H NMR (360 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 1.62–1.73 (m, 2H); 1.74–1.88 (m, 1H); 2.04 (br dd, <i>J</i> = 12.62, 4.21 Hz, 1H); 2.37 (br s, 1H); 2.44 (s, 3H); 2.53–2.71 (m, 2H); 2.99 (br d, <i>J</i> = 10.98 Hz, 1H); 3.54–3.62 (m, 2H); 3.82 (dt, <i>J</i> = 8.51, 4.35 Hz, 1H); 7.04 (t, <i>J</i> = 55.30 Hz, 1H); 7.26 (d, <i>J</i> = 6.59 Hz, 2H); 7.70 (s, 1H); 8.82 (s, 1H) and <b>41</b> (S*) (407 mg, 35%). <i>R</i><sub>t</sub> = 4.67 min, OR = −42.56° (589 nm, 20 °C, 0.43 w/v %; DMF), mp 165.83 °C.</div></div><div id="sec4_10_56" class="NLM_sec NLM_sec_level_3"><div id="ac_i119" class="anchor-spacer"></div><h4 class="article-section__title" id="_i119"> Final Compound <b>40</b>, 5-(Difluoromethyl)-7-[1-[(2,6-dimethyl-4-pyridyl)methyl]-3-piperidyl]-[1,2,4]triazolo[1,5-<i>a</i>]pyrimidine</h4><div class="NLM_p last">Prepared in a similar manner to compound <b>37</b>, starting from 2,6-dimethylisonicotinaldehyde [18206-06-9] (0.99 g, 7.36 mmol) and intermediate <b>48i</b> (1.20 g, 3.68 mmol). Purification by prep HPLC on a Uptisphere column (C18—10 μm, 200 g, 5 cm) and using as the mobile phase a mixture of a 0.25% NH<sub>4</sub>HCO<sub>3</sub> solution in water and MeOH afforded the title compound as a racemic mixture (1.04 g, 76%). ESI <i>m</i>/<i>z</i>: 373 [M + H]<sup>+</sup>. Separation in enantiomers by preparative SFC [CHIRALPAK Daicel AD 30 × 250 mm; mobile phase: CO<sub>2</sub>, EtOH with 0.2% 2-propylamine] afforded compounds <b>40</b> (R*) (373 mg, 36%) <i>R</i><sub>t</sub> = 3.53 min, OR = +48.93° (589 nm, 20 °C, 0.515 w/v %; DMF), mp 153.3 °C, <sup>1</sup>H NMR (360 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 1.62–1.73 (m, 2H); 1.74–1.88 (m, 1H); 2.04 (br dd, <i>J</i> = 12.62, 4.21 Hz, 1H); 2.37 (br s, 1H); 2.44 (s, 3H); 2.53–2.71 (m, 2H); 2.99 (br d, <i>J</i> = 10.98 Hz, 1H); 3.54–3.62 (m, 2H); 3.82 (dt, <i>J</i> = 8.51, 4.35 Hz, 1H); 7.04 (t, <i>J</i> = 55.30 Hz, 1H); 7.26 (d, <i>J</i> = 6.59 Hz, 2H); 7.70 (s, 1H); 8.82 (s, 1H) and <b>42</b> (S*) (348 mg, 33%). <i>R</i><sub>t</sub> = 4.66 min, OR = −48.46° (589 nm, 20 °C, 0.43 w/v %; DMF), mp 154.4 °C.</div></div><div id="sec4_10_57" class="NLM_sec NLM_sec_level_3"><div id="ac_i120" class="anchor-spacer"></div><h4 class="article-section__title" id="_i120"> Final Compound <b>41</b>, Benzofuran-5-yl-[3-[5-(difluoromethyl)-[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl]-4-methyl-1-piperidyl]methanone</h4><div class="NLM_p last">Prepared in a similar manner to compound <b>42</b>, starting from intermediate <b>78</b> (rac, 220 mg, 0.82 mmol). Purification by flash chromatography on a silica gel, eluting with 0–1% MeOH in DCM, afforded compound <b>41</b> %. ESI <i>m</i>/<i>z</i>: 412 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 0.82 (d, <i>J</i> = 6.46 Hz, 3H) 1.37–1.50 (m, 1H) 1.84–1.92 (m, 1H) 2.41–2.46 (m, 1H) 2.50–2.53 (m, 1H) 3.15 (td, <i>J</i> = 12.92, 2.83 Hz, 1H) 3.34–3.40 (m, 1H) 3.56–3.64 (m, 1H) 4.17 (br d, <i>J</i> = 12.92 Hz, 1H) 4.32 (br d, <i>J</i> = 12.51 Hz, 1H) 6.84–7.14 (m, 1H) 6.96 (dd, <i>J</i> = 2.42, 0.81 Hz, 1H) 7.38 (dd, <i>J</i> = 8.48, 1.61 Hz, 1H) 7.55–7.56 (m, 1H) 7.58 (d, <i>J</i> = 8.48 Hz, 1H) 7.75 (d, <i>J</i> = 1.61 Hz, 1H) 7.94 (d, <i>J</i> = 2.02 Hz, 1H) 8.70 (s, 1H).</div></div><div id="sec4_10_58" class="NLM_sec NLM_sec_level_3"><div id="ac_i121" class="anchor-spacer"></div><h4 class="article-section__title" id="_i121"> Final Compound <b>42</b>, Benzofuran-5-yl-[(3<i>S</i>,4<i>R</i>)-3-[5-(difluoromethyl)-[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl]-4-methyl-1-piperidyl]methanone</h4><div class="NLM_p last">1-Benzofuran-5-carboxylic acid (214 mg, 1.32 mmol) was stirred in DCM (10 mL). DIPEA (851 mg, 0.75 g/mL, 6.58 mmol) and HBTU (499 mg, 1.32 mmol) were added, and stirring was continued for 20 min at RT. Intermediate <b>78</b> (RS!, 400 mg, 1.32 mmol; WO 2018/083098) was added, and stirring was continued for 1 h at RT. The RM was quenched with 1 N aqueous NaOH (1 mL) solution and stirred for 5 min, then the OL was separated and dried over MgSO<sub>4</sub>, filtered, and concentrated <i>in vacuo</i>. This material was purified by silica gel flash chromatography eluting with 0–3% MeOH in DCM to afford compound <b>42</b> (120 mg, yield 52.931%). ESI <i>m</i>/<i>z</i>: 412 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 0.82 (d, <i>J</i> = 6.46 Hz, 3H) 1.37–1.50 (m, 1H) 1.84–1.92 (m, 1H) 2.41–2.46 (m, 1H) 2.50–2.53 (m, 1H) 3.15 (td, <i>J</i> = 12.92, 2.83 Hz, 1H) 3.34–3.40 (m, 1H) 3.56–3.64 (m, 1H) 4.17 (br d, <i>J</i> = 12.92 Hz, 1H) 4.32 (br d, <i>J</i> = 12.51 Hz, 1H) 6.84–7.14 (m, 1H) 6.96 (dd, <i>J</i> = 2.42, 0.81 Hz, 1H) 7.38 (dd, <i>J</i> = 8.48, 1.61 Hz, 1H) 7.55–7.56 (m, 1H) 7.58 (d, <i>J</i> = 8.48 Hz, 1H) 7.75 (d, <i>J</i> = 1.61 Hz, 1H) 7.94 (d, <i>J</i> = 2.02 Hz, 1H) 8.70 (s, 1H).</div></div><div id="sec4_10_59" class="NLM_sec NLM_sec_level_3"><div id="ac_i122" class="anchor-spacer"></div><h4 class="article-section__title" id="_i122"> Final Compound <b>43</b>, (3,4-Dimethoxyphenyl)-[(3<i>S</i>,4<i>R</i>)-3-[5-(difluoromethyl)-[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl]-4-methyl-1-piperidyl]methanone</h4><div class="NLM_p last">Prepared in a similar manner to compound <b>42</b>, starting from 3,4-dimethoxybenzoic acid (120 mg, 0.66 mmol) and intermediate <b>78</b> (SR!, 200 mg, 0.66 mmol). Purification by flash chromatography on a silica gel, eluting with 0–2% MeOH in DCM, afforded compound <b>41</b> %. ESI <i>m</i>/<i>z</i>: 432 [M + H]<sup>+</sup>. mp 227 °C. <sup>1</sup>H NMR (360 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 0.00 (s, 1H) 0.77 (d, <i>J</i> = 6.59 Hz, 3H); 1.35–1.47 (m, 1H); 1.79–1.95 (m, 1H); 2.40–2.47 (m, 1H); 3.55–3.63 (m, 1H); 3.77 (br s, 3H); 3.79 (s, 4H); 3.93–4.65 (m, 1H); 6.95 (br s, 1H); 7.01 (br s, 3H); 7.10 (s, 1H); 7.68 (br s, 1H); 8.83 (s, 1H).</div></div><div id="sec4_10_60" class="NLM_sec NLM_sec_level_3"><div id="ac_i123" class="anchor-spacer"></div><h4 class="article-section__title" id="_i123"> Final Compound <b>44</b>, [3-[2-(5-Bromo-2-methyl-pyrazol-3-yl)ethyl]-6-fluoro-1-naphthyl]oxy-<i>tert</i>-butyl-diphenyl-silane; 2-[3-[5-(Difluoromethyl)-[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl]-4-methyl-1-piperidyl]-<i>N</i>-methyl-pyridine-4-carboxamide; Methane</h4><div class="NLM_p last">DIPEA [7087-68-5] (372 mg, 2.88 mmol) was added at RT to a solution of intermediate <b>78</b> (RS!, 175 mg, 0.58 mmol) and 2-bromo-<i>n</i>-methyl-4-pyridinecarboxamide [337536-01-3] (372 mg, 1.73 mmol) in <i>n</i>BuOH [71-36-3] (3 mL). The resulting mixture was then stirred for 4 h at 180 °C under microwave irradiation, then it was cooled to RT, and the solvent was evaporated under reduced pressure. The resulting residue was dissolved in DCM (5 mL) and washed with a saturated aqueous NaHCO<sub>3</sub> solution (1 mL). The biphasic mixture was passed through an Extrelut filter, which was afterward washed with DCM (×2). The combined OL was evaporated under reduced pressure. The resulting residue was purified by Prep HPLC using as the stationary phase a Uptisphere C18 ODB—10 μm, 200 g, 5 cm column and as the mobile phase a mixture of 0.25% NH<sub>4</sub>HCO<sub>3</sub> solution in water and CH<sub>3</sub>CN, to provide the title compound. This was recrystallized from ether, to give compound <b>44</b> (95 mg, 41%). ESI <i>m</i>/<i>z</i>: 402 [M + H]<sup>+</sup>. mp 180.87 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 0.78 (d, <i>J</i> = 6.53 Hz, 3H) 1.42 (br dd, <i>J</i> = 12.42, 3.89 Hz, 1H) 1.91 (br dd, <i>J</i> = 13.55, 3.51 Hz, 1H) 2.44 (br s, 2H) 2.78 (d, <i>J</i> = 4.52 Hz, 3H) 3.11 (td, <i>J</i> = 12.80, 3.01 Hz, 1H) 3.36 (dd, <i>J</i> = 12.80, 11.04 Hz, 1H) 3.51–3.58 (m, 1H) 4.45 (br d, <i>J</i> = 13.30 Hz, 1H) 4.53–4.59 (m, 1H) 6.95 (dd, <i>J</i> = 5.14, 1.13 Hz, 1H) 7.16–7.19 (m, 1H) 7.61 (s, 1H) 8.16 (d, <i>J</i> = 5.02 Hz, 2H) 8.74 (s, 1H).</div></div><div id="sec4_10_61" class="NLM_sec NLM_sec_level_3"><div id="ac_i124" class="anchor-spacer"></div><h4 class="article-section__title" id="_i124"> Final Compound <b>45</b>, 7-[1-(4-<i>tert</i>-Butyl-2-pyridyl)-4-methyl-3-piperidyl]-5-(difluoromethyl)-[1,2,4]triazolo[1,5-<i>a</i>]pyrimidine</h4><div class="NLM_p last">Prepared in a similar manner to compound <b>44</b>, starting from 4-<i>tert</i>-butyl-2-chloropyridine (112 mg, 0.66 mmol) and intermediate <b>78</b> (SR!, 100 mg, 0.33 mmol) and using NMP (3 mL) as the solvent. Purification by flash chromatography on a silica gel, eluting with 0–2% MeOH in DCM, followed by crystallization from diisopropyl ether, afforded compound <b>45</b> (37.6 mg, 29%). ESI <i>m</i>/<i>z</i>: 401 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (360 MHz, chloroform-<i>d</i>): δ ppm 0.80–0.98 (m, 5H); 1.21–1.35 (m, 11H); 1.41–1.56 (m, 1H); 1.61–1.79 (m, 1H); 1.91–1.99 (m, 1H); 2.36 (br s, 1H); 3.03–3.23 (m, 2H); 3.66 (br s, 1H); 4.55 (br dd, <i>J</i> = 12.26, 2.74 Hz, 2H); 6.53–6.85 (m, 2H); 6.86 (s, 1H); 7.33 (s, 1H); 8.10 (d, <i>J</i> = 5.12 Hz, 1H); 8.60 (s, 1H).</div></div><div id="sec4_10_62" class="NLM_sec NLM_sec_level_3"><div id="ac_i125" class="anchor-spacer"></div><h4 class="article-section__title" id="_i125"> Final Compound <b>46</b>, 5-(Difluoromethyl)-7-[4-methyl-1-(4-phenyl-2-pyridyl)-3-piperidyl]-[1,2,4]triazolo[1,5-<i>a</i>]pyrimidine</h4><div class="NLM_p last">Prepared in a similar manner to compound <b>44</b>, starting from 2-chloro-4-phenylpyridine [42260-39-9] (139 mg, 0.74 mmol) and intermediate <b>78</b> (SR!, 112 mg, 0.37 mmol) and using NMP (3.5 mL) as the solvent. Purification by flash chromatography on a silica gel, eluting with 0–2% MeOH in DCM, followed by crystallization from diisopropyl ether, afforded compound <b>46</b> (8.6 mg, 6%). ESI <i>m</i>/<i>z</i>: 421 [M + H]<sup>+</sup>. mp 127.58 °C. <sup>1</sup>H NMR (360 MHz, chloroform-<i>d</i>): δ ppm 0.76–0.96 (m, 4H) 1.40–1.61 (m, 1H) 1.93–2.01 (m, 1H) 2.41 (br s, 1H) 3.09–3.30 (m, 2H) 3.61–3.77 (m, 1H) 4.58–4.67 (m, 2H) 6.52–6.87 (m, 1H) 6.91 (dd, <i>J</i> = 5.31, 1.28 Hz, 1H) 7.12 (s, 1H) 7.34 (s, 1H) 7.41–7.53 (m, 3H) 7.64–7.72 (m, 2H) 8.25 (d, <i>J</i> = 5.49 Hz, 1H) 8.62 (s, 1H).</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i126"><a href="/doi/suppl/10.1021/acs.jmedchem.0c01272" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_17067" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_17067" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c01272?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01272</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Selectivity data for all compounds; data for retrospective validation of FEP approach; comparison of binding affinity predictions from docking with experiment; and additional chemistry details (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01272/suppl_file/jm0c01272_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings for <b>3</b>–<b>46</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01272/suppl_file/jm0c01272_si_002.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Input structures for FEP calculations (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01272/suppl_file/jm0c01272_si_003.zip" class="ext-link">ZIP</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01272/suppl_file/jm0c01272_si_001.pdf">jm0c01272_si_001.pdf (527.64 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01272/suppl_file/jm0c01272_si_002.csv">jm0c01272_si_002.csv (3.31 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01272/suppl_file/jm0c01272_si_003.zip">jm0c01272_si_003.zip (796.92 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Atomic coordinates and structure factors for compounds <b>3</b> and <b>39</b> have been deposited in the RCSB Protein Data Bank under the accession code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZND">6ZND</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZQZ">6ZQZ</a> respectively. Authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.0c01272" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_68568" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_68568" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gary Tresadern</span> - <span class="hlFld-Affiliation affiliation">Computational
Chemistry, Janssen Pharmaceutica N. V., Turnhoutseweg 30, B-2340 Beerse, Belgium</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4801-1644" title="Orcid link">http://orcid.org/0000-0002-4801-1644</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#b0d7c4c2d5c3d1d4d5f0d9c4c39edadeda9ed3dfdd"><span class="__cf_email__" data-cfemail="82e5f6f0e7f1e3e6e7c2ebf6f1ace8ece8ace1edef">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peter J. J. A. Buijnsters</span> - <span class="hlFld-Affiliation affiliation">Medicinal
Chemistry, Janssen Pharmaceutica N. V., Turnhoutseweg 30, B-2340 Beerse, Belgium</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#0b7b697e626165787f4b627f782561656125686466"><span class="__cf_email__" data-cfemail="31415344585b5f4245715845421f5b5f5b1f525e5c">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ingrid Velter</span> - <span class="hlFld-Affiliation affiliation">Medicinal
Chemistry, Janssen Pharmaceutica N. V., Turnhoutseweg 30, B-2340 Beerse, Belgium</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andrés A. Trabanco</span> - <span class="hlFld-Affiliation affiliation">Medicinal
Chemistry, Janssen Research & Development, Janssen-Cilag S. A., Jarama 75A, 45007 Toledo, Spain</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4225-758X" title="Orcid link">http://orcid.org/0000-0002-4225-758X</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Frans Van den Keybus</span> - <span class="hlFld-Affiliation affiliation">Medicinal
Chemistry, Janssen Pharmaceutica N. V., Turnhoutseweg 30, B-2340 Beerse, Belgium</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gregor J. Macdonald</span> - <span class="hlFld-Affiliation affiliation">Medicinal
Chemistry, Janssen Pharmaceutica N. V., Turnhoutseweg 30, B-2340 Beerse, Belgium</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Marijke V. F. Somers</span> - <span class="hlFld-Affiliation affiliation">Discovery
Sciences, Janssen Research & Development, Janssen Pharmaceutica N. V., Turnhoutseweg 30, B-2340 Beerse, Belgium</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Greet Vanhoof</span> - <span class="hlFld-Affiliation affiliation">Discovery
Sciences, Janssen Research & Development, Janssen Pharmaceutica N. V., Turnhoutseweg 30, B-2340 Beerse, Belgium</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Philip M. Leonard</span> - <span class="hlFld-Affiliation affiliation">Structural
Biology, Charles River Discovery (Previously BioFocus), Chesterford Research Park, Saffron Walden, CB10 1XL Essex, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Marieke B. A. C. Lamers</span> - <span class="hlFld-Affiliation affiliation">Structural
Biology, Charles River Discovery (Previously BioFocus), Chesterford Research Park, Saffron Walden, CB10 1XL Essex, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yves E. M. Van Roosbroeck</span> - <span class="hlFld-Affiliation affiliation">Medicinal
Chemistry, Janssen Pharmaceutica N. V., Turnhoutseweg 30, B-2340 Beerse, Belgium</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>The manuscript was written through contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i129">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_65833" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_65833" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Thomas Steinbrecher and Amr Ragab at Schrodinger for assistance with the free-energy perturbation calculations. We also thank Ilse Lenaerts for performing the occupancy experiments. We thank the members of analysis and purification department, in particular Michel Carpentier and Alex de Groot for help with data analysis.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">AD</td><td class="NLM_def"><p class="first last">Alzheimer’s disease</p></td></tr><tr><td class="NLM_term">cAMP</td><td class="NLM_def"><p class="first last">3′,5′-cyclic adenosine monophosphate</p></td></tr><tr><td class="NLM_term">cGMP</td><td class="NLM_def"><p class="first last">3′,5′-cyclic guanosine monophosphate</p></td></tr><tr><td class="NLM_term">CYP450</td><td class="NLM_def"><p class="first last">cytochrome P450</p></td></tr><tr><td class="NLM_term">HTS</td><td class="NLM_def"><p class="first last">high-throughput screening</p></td></tr><tr><td class="NLM_term">FEP</td><td class="NLM_def"><p class="first last">free-energy perturbation</p></td></tr><tr><td class="NLM_term">HP-β-CD</td><td class="NLM_def"><p class="first last">(2-hydroxypropyl)-beta-cyclodextrin</p></td></tr><tr><td class="NLM_term">LHS</td><td class="NLM_def"><p class="first last">left-hand side</p></td></tr><tr><td class="NLM_term">LO</td><td class="NLM_def"><p class="first last">lead optimization</p></td></tr><tr><td class="NLM_term">LTP</td><td class="NLM_def"><p class="first last">long-term potentiation</p></td></tr><tr><td class="NLM_term">MUE</td><td class="NLM_def"><p class="first last">mean unsigned error</p></td></tr><tr><td class="NLM_term">NOS</td><td class="NLM_def"><p class="first last">nitric oxide synthase</p></td></tr><tr><td class="NLM_term">PDB</td><td class="NLM_def"><p class="first last">protein data bank</p></td></tr><tr><td class="NLM_term">PDE</td><td class="NLM_def"><p class="first last">phosphodiesterase</p></td></tr><tr><td class="NLM_term">RBFE</td><td class="NLM_def"><p class="first last">relative binding free energy</p></td></tr><tr><td class="NLM_term">RHS</td><td class="NLM_def"><p class="first last">right-hand side</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">SD</td><td class="NLM_def"><p class="first last">standard deviation</p></td></tr><tr><td class="NLM_term">SFC</td><td class="NLM_def"><p class="first last">supercritical fluid chromatography</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i131">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_35524" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_35524" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 55 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maurice, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ke, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahmad, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manganiello, V. C.</span></span> <span> </span><span class="NLM_article-title">Advances in targeting cyclic nucleotide phosphodiesterases</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">290</span>– <span class="NLM_lpage">314</span>, <span class="refDoi"> DOI: 10.1038/nrd4228</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.1038%2Fnrd4228" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=24687066" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC2cXltFOiu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=290-314&author=D.+H.+Mauriceauthor=H.+Keauthor=F.+Ahmadauthor=Y.+Wangauthor=J.+Chungauthor=V.+C.+Manganiello&title=Advances+in+targeting+cyclic+nucleotide+phosphodiesterases&doi=10.1038%2Fnrd4228"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Advances in targeting cyclic nucleotide phosphodiesterases</span></div><div class="casAuthors">Maurice, Donald H.; Ke, Hengming; Ahmad, Faiyaz; Wang, Yousheng; Chung, Jay; Manganiello, Vincent C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">290-314</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Cyclic nucleotide phosphodiesterases (PDEs) catalyze the hydrolysis of cAMP and cyclic GMP, thereby regulating the intracellular concns. of these cyclic nucleotides, their signalling pathways and, consequently, myriad biol. responses in health and disease.  Currently, a small no. of PDE inhibitors are used clin. for treating the pathophysiol. dysregulation of cyclic nucleotide signalling in several disorders, including erectile dysfunction, pulmonary hypertension, acute refractory cardiac failure, intermittent claudication and chronic obstructive pulmonary disease.  However, pharmaceutical interest in PDEs has been reignited by the increasing understanding of the roles of individual PDEs in regulating the subcellular compartmentalization of specific cyclic nucleotide signalling pathways, by the structure-based design of novel specific inhibitors and by the development of more sophisticated strategies to target individual PDE variants.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwMRfUs-zUmLVg90H21EOLACvtfcHk0lhSjz8hLFLOeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXltFOiu7s%253D&md5=410c8d5bb5fbd7d2906e30dc1b5f2876</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd4228&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4228%26sid%3Dliteratum%253Aachs%26aulast%3DMaurice%26aufirst%3DD.%2BH.%26aulast%3DKe%26aufirst%3DH.%26aulast%3DAhmad%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DChung%26aufirst%3DJ.%26aulast%3DManganiello%26aufirst%3DV.%2BC.%26atitle%3DAdvances%2520in%2520targeting%2520cyclic%2520nucleotide%2520phosphodiesterases%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D290%26epage%3D314%26doi%3D10.1038%2Fnrd4228" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Francis, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blount, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corbin, J. D.</span></span> <span> </span><span class="NLM_article-title">Mammalian cyclic nucleotide phosphodiesterases: molecular mechanisms and physiological functions</span>. <i>Physiol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>91</i></span>,  <span class="NLM_fpage">651</span>– <span class="NLM_lpage">690</span>, <span class="refDoi"> DOI: 10.1152/physrev.00030.2010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.1152%2Fphysrev.00030.2010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=21527734" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC3MXns1Cms7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=91&publication_year=2011&pages=651-690&author=S.+H.+Francisauthor=M.+A.+Blountauthor=J.+D.+Corbin&title=Mammalian+cyclic+nucleotide+phosphodiesterases%3A+molecular+mechanisms+and+physiological+functions&doi=10.1152%2Fphysrev.00030.2010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Mammalian cyclic nucleotide phosphodiesterases: molecular mechanisms and physiological functions</span></div><div class="casAuthors">Francis, Sharron H.; Blount, Mitsi A.; Corbin, Jackie D.</div><div class="citationInfo"><span class="NLM_cas:title">Physiological Reviews</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">91</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">651-690</span>CODEN:
                <span class="NLM_cas:coden">PHREA7</span>;
        ISSN:<span class="NLM_cas:issn">0031-9333</span>.
    
            (<span class="NLM_cas:orgname">American Physiological Society</span>)
        </div><div class="casAbstract">A review.  The superfamily of cyclic nucleotide (cN) phosphodiesterases (PDEs) is comprised of 11 families of enzymes.  PDEs break down cAMP and/or cGMP and are major determinants of cellular cN levels and, consequently, the actions of cN-signaling pathways.  PDEs exhibit a range of catalytic efficiencies for breakdown of cAMP and/or cGMP and are regulated by myriad processes including phosphorylation, cN binding to allosteric GAF domains, changes in expression levels, interaction with regulatory or anchoring proteins, and reversible translocation among subcellular compartments.  Selective PDE inhibitors are currently in clin. use for treatment of erectile dysfunction, pulmonary hypertension, intermittent claudication, and chronic pulmonary obstructive disease; many new inhibitors are being developed for treatment of these and other maladies.  Recently reported x-ray crystallog. structures have defined features that provide for specificity for cAMP or cGMP in PDE catalytic sites or their GAF domains, as well as mechanisms involved in catalysis, oligomerization, autoinhibition, and interactions with inhibitors.  In addn., major advances have been made in understanding the physiol. impact and the biochem. basis for selective localization and/or recruitment of specific PDE isoenzymes to particular subcellular compartments.  The many recent advances in understanding PDE structures, functions, and physiol. actions are discussed in this review.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrN99wWYsCXRbVg90H21EOLACvtfcHk0lhSjz8hLFLOeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXns1Cms7o%253D&md5=4aae757704a6243c9aa2648c32d5c759</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1152%2Fphysrev.00030.2010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1152%252Fphysrev.00030.2010%26sid%3Dliteratum%253Aachs%26aulast%3DFrancis%26aufirst%3DS.%2BH.%26aulast%3DBlount%26aufirst%3DM.%2BA.%26aulast%3DCorbin%26aufirst%3DJ.%2BD.%26atitle%3DMammalian%2520cyclic%2520nucleotide%2520phosphodiesterases%253A%2520molecular%2520mechanisms%2520and%2520physiological%2520functions%26jtitle%3DPhysiol.%2520Rev.%26date%3D2011%26volume%3D91%26spage%3D651%26epage%3D690%26doi%3D10.1152%2Fphysrev.00030.2010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pandit, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forman, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fennell, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dillman, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menniti, F. S.</span></span> <span> </span><span class="NLM_article-title">Mechanism for the allosteric regulation of phosphodiesterase 2A deduced from the X-ray structure of a near full-length construct</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">18225</span>– <span class="NLM_lpage">18230</span>, <span class="refDoi"> DOI: 10.1073/pnas.0907635106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.1073%2Fpnas.0907635106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=19828435" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVags7vO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=18225-18230&author=J.+Panditauthor=M.+D.+Formanauthor=K.+F.+Fennellauthor=K.+S.+Dillmanauthor=F.+S.+Menniti&title=Mechanism+for+the+allosteric+regulation+of+phosphodiesterase+2A+deduced+from+the+X-ray+structure+of+a+near+full-length+construct&doi=10.1073%2Fpnas.0907635106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism for the allosteric regulation of phosphodiesterase 2A deduced from the X-ray structure of a near full-length construct</span></div><div class="casAuthors">Pandit, Jayvardhan; Forman, Michael D.; Fennell, Kimberly F.; Dillman, Keith S.; Menniti, Frank S.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">18225-18230, S18225/1-S18225/8</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">We report the x-ray crystal structure of a phosphodiesterase (PDE) that includes both catalytic and regulatory domains.  PDE2A (215-900) crystd. as a dimer in which each subunit had an extended organization of regulatory GAF-A and GAF-B and catalytic domains connected by long α-helixes.  The subunits cross at the GAF-B/catalytic domain linker, and each side of the dimer contains in series the GAF-A and GAF-B of one subunit and the catalytic domain of the other subunit.  A dimer interface extends over the entire length of the mol.  The substrate binding pocket of each catalytic domain is occluded by the H-loop.  We deduced from comparisons with structures of isolated, ligand-bound catalytic subunits that the H-loop swings out to allow substrate access.  However, in dimeric PDE2A (215-900), the H-loops of the two catalytic subunits pack against each other at the dimer interface, necessitating movement of the catalytic subunits to allow for H-loop movement.  Comparison of the unliganded GAF-B of PDE2A (215-900) with previous structures of isolated, cGMP-bound GAF domains indicates that cGMP binding induces a significant shift in the GAF-B/catalytic domain linker.  We propose that cGMP binding to GAF-B causes movement, through this linker region, of the catalytic domains, such that the H-loops no longer pack at the dimer interface and are, instead, free to swing out to allow substrate access.  This increase in substrate access is proposed as the basis for PDE2A activation by cGMP and may be a general mechanism for regulation of all PDEs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqh-kgJc6VX2bVg90H21EOLACvtfcHk0liBLKlTWmUGww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVags7vO&md5=73ddfdb3ffb861674744a1fbeef7c881</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0907635106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0907635106%26sid%3Dliteratum%253Aachs%26aulast%3DPandit%26aufirst%3DJ.%26aulast%3DForman%26aufirst%3DM.%2BD.%26aulast%3DFennell%26aufirst%3DK.%2BF.%26aulast%3DDillman%26aufirst%3DK.%2BS.%26aulast%3DMenniti%26aufirst%3DF.%2BS.%26atitle%3DMechanism%2520for%2520the%2520allosteric%2520regulation%2520of%2520phosphodiesterase%25202A%2520deduced%2520from%2520the%2520X-ray%2520structure%2520of%2520a%2520near%2520full-length%2520construct%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2009%26volume%3D106%26spage%3D18225%26epage%3D18230%26doi%3D10.1073%2Fpnas.0907635106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, A. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, X.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ikeda, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beavo, J. A.</span></span> <span> </span><span class="NLM_article-title">Molecular determinants for cyclic nucleotide binding to the regulatory domains of phosphodiesterase 2A</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>279</i></span>,  <span class="NLM_fpage">37928</span>– <span class="NLM_lpage">37938</span>, <span class="refDoi"> DOI: 10.1074/jbc.m404287200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.1074%2Fjbc.M404287200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=15210692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BD2cXntFSntrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=279&publication_year=2004&pages=37928-37938&author=A.+Y.+Wuauthor=X.-B.+Tangauthor=S.+E.+Martinezauthor=K.+Ikedaauthor=J.+A.+Beavo&title=Molecular+determinants+for+cyclic+nucleotide+binding+to+the+regulatory+domains+of+phosphodiesterase+2A&doi=10.1074%2Fjbc.m404287200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Determinants for Cyclic Nucleotide Binding to the Regulatory Domains of Phosphodiesterase 2A</span></div><div class="casAuthors">Wu, Albert Y.; Tang, Xiao-Bo; Martinez, Sergio E.; Ikeda, Kaori; Beavo, Joseph A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">279</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">37928-37938</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Binding of cGMP to the GAF-B domain of phosphodiesterase 2A allosterically activates catalytic activity.  We report here a series of mutagenesis studies on the GAF-B domain of PDE2A that support a novel mechanism for mol. recognition of cGMP.  Alanine mutations of Phe-438, Asp-439, and Thr-488, amino acids that interact with the pyrimidine ring, decrease cGMP affinity slightly but increase cAMP affinity by up to 8-fold.  Each interaction is required to provide for cAMP/cGMP specificity.  Mutations of any of the residues that interact with the phosphate-ribose moiety or the imidazole ring abolish cGMP binding.  Thus, residues that interact with the pyrimidine ring collectively control cAMP/cGMP specificity, whereas residues that bind the phosphate-ribose moiety and imidazole ring are crit. for high affinity binding.  Similar decreases in binding were found for mutations made in a bacterially expressed GAF-A/B plus catalytic domain construct.  Because these constructs had very high catalytic activity, it appears that these mutations did not cause a global denaturation.  The affinities of cAMP and cGMP for wild-type GAF-B alone were ∼4-fold greater than for the holoenzyme, suggesting that the presence of neighboring domains alters the conformation of GAF-B.  More importantly, the PDE2A GAF-B, GAF-A/B, GAF-A/B+C domains, and holoenzyme all bind cGMP with much higher affinity than has previously been reported.  This high affinity suggests that cGMP binding to PDE2 GAF-B activates the enzyme rapidly, stoichiometrically, and in an all or none fashion, rather than variably over a large range of cyclic nucleotide concns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUnZDzjQ-ZNLVg90H21EOLACvtfcHk0liBLKlTWmUGww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXntFSntrs%253D&md5=e1b95613e37644fd8e87c8553db84491</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M404287200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M404287200%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DA.%2BY.%26aulast%3DTang%26aufirst%3DX.-B.%26aulast%3DMartinez%26aufirst%3DS.%2BE.%26aulast%3DIkeda%26aufirst%3DK.%26aulast%3DBeavo%26aufirst%3DJ.%2BA.%26atitle%3DMolecular%2520determinants%2520for%2520cyclic%2520nucleotide%2520binding%2520to%2520the%2520regulatory%2520domains%2520of%2520phosphodiesterase%25202A%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2004%26volume%3D279%26spage%3D37928%26epage%3D37938%26doi%3D10.1074%2Fjbc.m404287200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Erneux, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Couchie, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumont, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baraniak, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stec, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abbad, E. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petridis, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jastorff, B.</span></span> <span> </span><span class="NLM_article-title">Specificity of cyclic GMP activation of a multi-substrate cyclic nucleotide phosphodiesterase from rat liver</span>. <i>Eur. J. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">1981</span>,  <span class="NLM_volume"><i>115</i></span>,  <span class="NLM_fpage">503</span>– <span class="NLM_lpage">510</span>, <span class="refDoi"> DOI: 10.1111/j.1432-1033.1981.tb06231.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.1111%2Fj.1432-1033.1981.tb06231.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=6263632" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADyaL3MXktVamsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=1981&pages=503-510&author=C.+Erneuxauthor=D.+Couchieauthor=J.+E.+Dumontauthor=J.+Baraniakauthor=W.+J.+Stecauthor=E.+G.+Abbadauthor=G.+Petridisauthor=B.+Jastorff&title=Specificity+of+cyclic+GMP+activation+of+a+multi-substrate+cyclic+nucleotide+phosphodiesterase+from+rat+liver&doi=10.1111%2Fj.1432-1033.1981.tb06231.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Specificity of cyclic GMP activation of a multi-substrate cyclic nucleotide phosphodiesterase from rat liver</span></div><div class="casAuthors">Erneux, Christophe; Couchie, Dominique; Dumont, Jacques E.; Baraniak, Janina; Stec, Woljciech J.; Abbad, Emilio Garcia; Petridis, Georg; Jastorff, Bernd</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Biochemistry</span>
        (<span class="NLM_cas:date">1981</span>),
    <span class="NLM_cas:volume">115</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">503-10</span>CODEN:
                <span class="NLM_cas:coden">EJBCAI</span>;
        ISSN:<span class="NLM_cas:issn">0014-2956</span>.
    </div><div class="casAbstract">Cyclic nucleotide derivs. have been used as a tool to investigate the existence of distinctive activating and hydrolytic sites on the cyclic nucleotide phosphodiesterase from rat liver activated by cGMP.  This pos. cooperative enzyme was stimulated up to 30-fold by 3 μM cGMP when 3 μM cAMP was used as substrate.  All analogs were less potent activators than cGMP.  Most cAMP derivs. were inactive, with 2 exceptions, 7-deazaadenosine 3',5'-monophosphate and 3'-amino-3'-deoxyadenosine 3',5'-monophosphate.  Benzimidazole ribonucleoside 3',5'-monophosphate, where the 2 N atoms of the pyrimidine ring are missing, was a better stimulator than the intact purine-related cyclic deriv.  When cAMP and cGMP with identical chem. ligands substituted at the same position were compared, the cGMP analog was always the more potent activator suggesting that the activating site is sensitive to a guanine-type cyclic nucleotide structure.  Degrdn. of the derivs. by the enzyme was measured by high-performance liq. chromatog.; no relation could be established between hydrolysis and effectiveness of activation.  In addn., there was no parallelism between inhibitory and activating potency for 10 cyclic nucleotide derivs.  Since the chem. interactions between the analogs at the activating site on the one hand and at the catalytic site on the other, are different, it is proposed that the sites are distinct.  Consequently, it is suggested that the enzyme operates in steps.  In the 1st activating step, cGMP is fixed by at least 2 H-bonds at a specific binding site of the enzyme.  This is followed by a conformational change of the protein and subsequently a change of the kinetic parameters.  In a rather unspecific process and in a 2nd hydrolytic step, several purine-related cyclic nucleotides are converted to the corresponding 5'-nucleotides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-pwbda4flLbVg90H21EOLACvtfcHk0liBLKlTWmUGww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3MXktVamsrY%253D&md5=c914a884a8486cb57531212c0d93641c</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1111%2Fj.1432-1033.1981.tb06231.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1432-1033.1981.tb06231.x%26sid%3Dliteratum%253Aachs%26aulast%3DErneux%26aufirst%3DC.%26aulast%3DCouchie%26aufirst%3DD.%26aulast%3DDumont%26aufirst%3DJ.%2BE.%26aulast%3DBaraniak%26aufirst%3DJ.%26aulast%3DStec%26aufirst%3DW.%2BJ.%26aulast%3DAbbad%26aufirst%3DE.%2BG.%26aulast%3DPetridis%26aufirst%3DG.%26aulast%3DJastorff%26aufirst%3DB.%26atitle%3DSpecificity%2520of%2520cyclic%2520GMP%2520activation%2520of%2520a%2520multi-substrate%2520cyclic%2520nucleotide%2520phosphodiesterase%2520from%2520rat%2520liver%26jtitle%3DEur.%2520J.%2520Biochem.%26date%3D1981%26volume%3D115%26spage%3D503%26epage%3D510%26doi%3D10.1111%2Fj.1432-1033.1981.tb06231.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lakics, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karran, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boess, F. G.</span></span> <span> </span><span class="NLM_article-title">Quantitative comparison of phosphodiesterase mRNA distribution in human brain and peripheral tissues</span>. <i>Neuropharmacology</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">367</span>– <span class="NLM_lpage">374</span>, <span class="refDoi"> DOI: 10.1016/j.neuropharm.2010.05.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.1016%2Fj.neuropharm.2010.05.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=20493887" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1SqsLbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2010&pages=367-374&author=V.+Lakicsauthor=E.+H.+Karranauthor=F.+G.+Boess&title=Quantitative+comparison+of+phosphodiesterase+mRNA+distribution+in+human+brain+and+peripheral+tissues&doi=10.1016%2Fj.neuropharm.2010.05.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative comparison of phosphodiesterase mRNA distribution in human brain and peripheral tissues</span></div><div class="casAuthors">Lakics, Viktor; Karran, Eric H.; Boess, Frank G.</div><div class="citationInfo"><span class="NLM_cas:title">Neuropharmacology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">367-374</span>CODEN:
                <span class="NLM_cas:coden">NEPHBW</span>;
        ISSN:<span class="NLM_cas:issn">0028-3908</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Cyclic nucleotide-specific phosphodiesterases (PDEs) play a crit. role in signal transduction by regulating the level of adenosine 3',5'-cyclic monophosphate (cAMP) and guanosine 3',5'-cyclic monophosphate (cGMP) in cells.  The gene expression pattern of a PDE provides important information regarding its role in physiol. and pathol. processes.  In this study, we have established the mRNA expression profile all PDE isoenzymes (PDE1A/B/C, 2A, 3A/B, 4A/B/C/D, 5A, 6A/B/C, 7A/B, 8A/B, 9A, 10A, 11A) in a human cDNA collection consisting of 10 brain regions (parietal, frontal, temporal cortex, hippocampus, striatum, thalamus, hypothalamus, substantia nigra, nucleus accumbens, cerebellum), spinal cord, dorsal root ganglia and 12 peripheral tissues (skeletal muscle, heart, thyroid, adrenal gland, pancreas, bladder, kidney, liver, lung, small intestine, spleen, and stomach).  Using quant. real-time polymerase chain reaction and parallel anal. of a carefully selected group of ref. genes, we have detd. the relative expression of each PDE isoenzyme across the 24 selected tissues, and also compared the expression of selected PDEs to each other within a given tissue type.  Several PDEs show strikingly selective expression (e.g. PDE10A and PDE1B mRNA levels in the caudate nucleus are 20-fold higher than in most other tissues; PDE1C and PDE3A are highly expressed in the heart and PDE8B is expressed very strongly in the thyroid gland).  This comprehensive approach provides a coherent and quant. view of the mRNA expression of the PDE gene family and enables an integration of data obtained with other non-quant. methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprGz6OEkRjbLVg90H21EOLACvtfcHk0ljfSTEwMioHgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1SqsLbJ&md5=ee55772f572818eaa7194a1ae3f52f45</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.neuropharm.2010.05.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuropharm.2010.05.004%26sid%3Dliteratum%253Aachs%26aulast%3DLakics%26aufirst%3DV.%26aulast%3DKarran%26aufirst%3DE.%2BH.%26aulast%3DBoess%26aufirst%3DF.%2BG.%26atitle%3DQuantitative%2520comparison%2520of%2520phosphodiesterase%2520mRNA%2520distribution%2520in%2520human%2520brain%2520and%2520peripheral%2520tissues%26jtitle%3DNeuropharmacology%26date%3D2010%26volume%3D59%26spage%3D367%26epage%3D374%26doi%3D10.1016%2Fj.neuropharm.2010.05.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stephenson, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coskran, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilhelms, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adamowicz, W. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Donnell, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muravnick, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menniti, F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kleiman, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morton, D.</span></span> <span> </span><span class="NLM_article-title">Immunohistochemical localization of phosphodiesterase 2A in multiple mammalian species</span>. <i>J. Histochem. Cytochem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">933</span>– <span class="NLM_lpage">949</span>, <span class="refDoi"> DOI: 10.1369/jhc.2009.953471</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.1369%2Fjhc.2009.953471" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=19506089" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFOgtrrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2009&pages=933-949&author=D.+T.+Stephensonauthor=T.+M.+Coskranauthor=M.+B.+Wilhelmsauthor=W.+O.+Adamowiczauthor=M.+M.+O%E2%80%99Donnellauthor=K.+B.+Muravnickauthor=F.+S.+Mennitiauthor=R.+J.+Kleimanauthor=D.+Morton&title=Immunohistochemical+localization+of+phosphodiesterase+2A+in+multiple+mammalian+species&doi=10.1369%2Fjhc.2009.953471"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Immunohistochemical localization of phosphodiesterase 2A in multiple mammalian species</span></div><div class="casAuthors">Stephenson, Diane T.; Coskran, Tim M.; Wilhelms, Margaret B.; Adamowicz, Wendy O.; O'Donnell, Michele M.; Muravnick, Kathleen B.; Menniti, Frank S.; Kleiman, Robin J.; Morton, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Histochemistry and Cytochemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">933-949</span>CODEN:
                <span class="NLM_cas:coden">JHCYAS</span>;
        ISSN:<span class="NLM_cas:issn">0022-1554</span>.
    
            (<span class="NLM_cas:orgname">Histochemical Society, Inc.</span>)
        </div><div class="casAbstract">Phosphodiesterases (PDEs) comprise a family of enzymes that regulate the levels of cyclic nucleotides, key second messengers that mediate a diverse array of functions.  PDE2A is an evolutionarily conserved cGMP-stimulated cAMP and cGMP PDE.  In the present study, the regional and cellular distribution of PDE2A in tissues of rats, mice, cynomolgus monkeys, dogs, and humans was evaluated by immunohistochem.  A polyclonal antibody directed to the C-terminal portion of PDE2A specifically detected PDE2A by Western blotting and by immunohistochem.  The pattern of PDE2A immunoreactivity (ir) was consistent across all species.  Western blot anal. demonstrated that PDE2A was most abundant in the brain relative to peripheral tissues.  PDE2A ir was heterogeneously distributed within brain and was selectively expressed in particular peripheral tissues.  In the brain, prominent immunoreactivity was apparent in components of the limbic system, including the isocortex, hippocampus, amygdala, habenula, basal ganglia, and interpeduncular nucleus.  Cytoplasmic PDE2A staining was prominent in several peripheral tissues, including the adrenal zona glomerulosa, neurons of enteric ganglia, endothelial cells in all organs, lymphocytes of spleen and lymph nodes, and pituitary.  These studies suggest that PDE2A is evolutionarily conserved across mammalian species and support the hypothesis that the enzyme plays a fundamental role in signal transduction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHVQCCm4loO7Vg90H21EOLACvtfcHk0ljfSTEwMioHgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFOgtrrJ&md5=ce1dfa99cc99c02f31ab8071a9e438fd</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1369%2Fjhc.2009.953471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1369%252Fjhc.2009.953471%26sid%3Dliteratum%253Aachs%26aulast%3DStephenson%26aufirst%3DD.%2BT.%26aulast%3DCoskran%26aufirst%3DT.%2BM.%26aulast%3DWilhelms%26aufirst%3DM.%2BB.%26aulast%3DAdamowicz%26aufirst%3DW.%2BO.%26aulast%3DO%25E2%2580%2599Donnell%26aufirst%3DM.%2BM.%26aulast%3DMuravnick%26aufirst%3DK.%2BB.%26aulast%3DMenniti%26aufirst%3DF.%2BS.%26aulast%3DKleiman%26aufirst%3DR.%2BJ.%26aulast%3DMorton%26aufirst%3DD.%26atitle%3DImmunohistochemical%2520localization%2520of%2520phosphodiesterase%25202A%2520in%2520multiple%2520mammalian%2520species%26jtitle%3DJ.%2520Histochem.%2520Cytochem.%26date%3D2009%26volume%3D57%26spage%3D933%26epage%3D949%26doi%3D10.1369%2Fjhc.2009.953471" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stephenson, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coskran, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kleiman, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morton, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Neill, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinberg, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menniti, F. S.</span></span> <span> </span><span class="NLM_article-title">The distribution of phosphodiesterase 2A in the rat brain</span>. <i>Neuroscience</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>226</i></span>,  <span class="NLM_fpage">145</span>– <span class="NLM_lpage">155</span>, <span class="refDoi"> DOI: 10.1016/j.neuroscience.2012.09.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.1016%2Fj.neuroscience.2012.09.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=23000621" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1ahur7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=226&publication_year=2012&pages=145-155&author=D.+T.+Stephensonauthor=T.+M.+Coskranauthor=M.+P.+Kellyauthor=R.+J.+Kleimanauthor=D.+Mortonauthor=S.+M.+O%E2%80%99Neillauthor=C.+J.+Schmidtauthor=R.+J.+Weinbergauthor=F.+S.+Menniti&title=The+distribution+of+phosphodiesterase+2A+in+the+rat+brain&doi=10.1016%2Fj.neuroscience.2012.09.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">The distribution of phosphodiesterase 2A in the rat brain</span></div><div class="casAuthors">Stephenson, D. T.; Coskran, T. M.; Kelly, M. P.; Kleiman, R. J.; Morton, D.; O'Neill, S. M.; Schmidt, C. J.; Weinberg, R. J.; Menniti, F. S.</div><div class="citationInfo"><span class="NLM_cas:title">Neuroscience (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">226</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">145-155</span>CODEN:
                <span class="NLM_cas:coden">NRSCDN</span>;
        ISSN:<span class="NLM_cas:issn">0306-4522</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The phosphodiesterases (PDEs) are a superfamily of enzymes that regulate spatio-temporal signaling by the intracellular second messengers cAMP and cGMP.  PDE2A is expressed at high levels in the mammalian brain.  To advance our understanding of the role of this enzyme in regulation of neuronal signaling, we here describe the distribution of PDE2A in the rat brain.  PDE2A mRNA was prominently expressed in glutamatergic pyramidal cells in cortex, and in pyramidal and dentate granule cells in the hippocampus.  Protein concd. in the axons and nerve terminals of these neurons; staining was markedly weaker in the cell bodies and proximal dendrites.  In addn., in both hippocampus and cortex, small populations of non-pyramidal cells, presumed to be interneurons, were strongly immunoreactive.  PDE2A mRNA was expressed in medium spiny neurons in neostriatum.  Little immunoreactivity was obsd. in cell bodies, whereas dense immunoreactivity was found in the axon tracts of these neurons and their terminal regions in globus pallidus and substantia nigra pars reticulata.  Immunostaining was dense in the medial habenula, but weak in other diencephalic regions.  In midbrain and hindbrain, immunostaining was restricted to discrete regions of the neuropil or clusters of cell bodies.  These results suggest that PDE2A may modulate cortical, hippocampal and striatal networks at several levels.  Preferential distribution of PDE2A into axons and terminals of the principal neurons suggests roles in regulation of axonal excitability or transmitter release.  The enzyme is also in forebrain interneurons, and in mid- and hindbrain neurons that may modulate forebrain networks and circuits.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxNSYsU1HXSrVg90H21EOLACvtfcHk0lgNG77QlgDBUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1ahur7F&md5=3bb5ae07b9fcae2a917cc4aed4fef4a1</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.neuroscience.2012.09.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuroscience.2012.09.011%26sid%3Dliteratum%253Aachs%26aulast%3DStephenson%26aufirst%3DD.%2BT.%26aulast%3DCoskran%26aufirst%3DT.%2BM.%26aulast%3DKelly%26aufirst%3DM.%2BP.%26aulast%3DKleiman%26aufirst%3DR.%2BJ.%26aulast%3DMorton%26aufirst%3DD.%26aulast%3DO%25E2%2580%2599Neill%26aufirst%3DS.%2BM.%26aulast%3DSchmidt%26aufirst%3DC.%2BJ.%26aulast%3DWeinberg%26aufirst%3DR.%2BJ.%26aulast%3DMenniti%26aufirst%3DF.%2BS.%26atitle%3DThe%2520distribution%2520of%2520phosphodiesterase%25202A%2520in%2520the%2520rat%2520brain%26jtitle%3DNeuroscience%26date%3D2012%26volume%3D226%26spage%3D145%26epage%3D155%26doi%3D10.1016%2Fj.neuroscience.2012.09.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blokland, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menniti, F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prickaerts, J.</span></span> <span> </span><span class="NLM_article-title">PDE Inhibition and cognition enhancement</span>. <i>Expert Opin. Ther. Pat.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">349</span>– <span class="NLM_lpage">354</span>, <span class="refDoi"> DOI: 10.1517/13543776.2012.674514</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.1517%2F13543776.2012.674514" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=22475506" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC38XltVKis70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=349-354&author=A.+Bloklandauthor=F.+S.+Mennitiauthor=J.+Prickaerts&title=PDE+Inhibition+and+cognition+enhancement&doi=10.1517%2F13543776.2012.674514"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">PDE Inhibition and cognition enhancement</span></div><div class="casAuthors">Blokland, Arjan; Menniti, Frank S.; Prickaerts, Jos</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Patents</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">349-354</span>CODEN:
                <span class="NLM_cas:coden">EOTPEG</span>;
        ISSN:<span class="NLM_cas:issn">1354-3776</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  There has been an increasing interest in the development of phosphodiesterase (PDE) inhibitors for the treatment of cognitive dysfunctions.  In this editorial, the mechanism of action of PDEs is briefly described, while the effects of different PDE inhibitors in preclin. models are reviewed.  Based on the expression of PDE mRNA in the human brain, it is suggested that PDE1 and PDE10 inhibitors are strong candidates for the development of cognition enhancers.  However, the complex nature of the expression of PDEs in the brain warrants further research into the role of PDEs in the signaling pathways in brain circuits.  The development of PDE inhibitors, which are selective for PDE splicing isoforms, may be promising for future drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreP8gLClAGx7Vg90H21EOLACvtfcHk0lgNG77QlgDBUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XltVKis70%253D&md5=368b04ba842f031cc8c67ac55efa1b4b</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1517%2F13543776.2012.674514&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F13543776.2012.674514%26sid%3Dliteratum%253Aachs%26aulast%3DBlokland%26aufirst%3DA.%26aulast%3DMenniti%26aufirst%3DF.%2BS.%26aulast%3DPrickaerts%26aufirst%3DJ.%26atitle%3DPDE%2520Inhibition%2520and%2520cognition%2520enhancement%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Pat.%26date%3D2012%26volume%3D22%26spage%3D349%26epage%3D354%26doi%3D10.1517%2F13543776.2012.674514" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bliss, T. V. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collingridge, G. L.</span></span> <span> </span><span class="NLM_article-title">A synaptic model of memory: long-term potentiation in the hippocampus</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>361</i></span>,  <span class="NLM_fpage">31</span>– <span class="NLM_lpage">39</span>, <span class="refDoi"> DOI: 10.1038/361031a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.1038%2F361031a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=8421494" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADyaK3sXntlOntw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=361&publication_year=1993&pages=31-39&author=T.+V.+P.+Blissauthor=G.+L.+Collingridge&title=A+synaptic+model+of+memory%3A+long-term+potentiation+in+the+hippocampus&doi=10.1038%2F361031a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">A synaptic model of memory:  long-term potentiation in the hippocampus</span></div><div class="casAuthors">Bliss, T. V. P.; Collingridge, G. L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">361</span>
        (<span class="NLM_cas:issue">6407</span>),
    <span class="NLM_cas:pages">31-9</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">A review, with 174 refs., of long-term potentiation (LTP) of synaptic transmission in the hippocampus as the primary exptl. model for investigating the synaptic basis of learning and memory in vertebrates.  Topics discussed were: properties of hippocampal LTP; induction of LTP; and expression of LTP.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoD1tHZif2aP7Vg90H21EOLACvtfcHk0lgNG77QlgDBUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXntlOntw%253D%253D&md5=285f5a175da5a77f4f57a3f5236e3e0a</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2F361031a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F361031a0%26sid%3Dliteratum%253Aachs%26aulast%3DBliss%26aufirst%3DT.%2BV.%2BP.%26aulast%3DCollingridge%26aufirst%3DG.%2BL.%26atitle%3DA%2520synaptic%2520model%2520of%2520memory%253A%2520long-term%2520potentiation%2520in%2520the%2520hippocampus%26jtitle%3DNature%26date%3D1993%26volume%3D361%26spage%3D31%26epage%3D39%26doi%3D10.1038%2F361031a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bollen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puzzo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rutten, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Privitera, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Vry, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vanmierlo, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kenis, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmeri, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Hooge, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balschun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinbusch, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blokland, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prickaerts, J.</span></span> <span> </span><span class="NLM_article-title">Improved long-term memory via enhancing cGMP-PKG signaling requires cAMP-PKA signaling</span>. <i>Neuropsychopharmacology</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">2497</span>– <span class="NLM_lpage">2505</span>, <span class="refDoi"> DOI: 10.1038/npp.2014.106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.1038%2Fnpp.2014.106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=24813825" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVKls7zF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2014&pages=2497-2505&author=E.+Bollenauthor=D.+Puzzoauthor=K.+Ruttenauthor=L.+Priviteraauthor=J.+De+Vryauthor=T.+Vanmierloauthor=G.+Kenisauthor=A.+Palmeriauthor=R.+D%E2%80%99Hoogeauthor=D.+Balschunauthor=H.+M.+Steinbuschauthor=A.+Bloklandauthor=J.+Prickaerts&title=Improved+long-term+memory+via+enhancing+cGMP-PKG+signaling+requires+cAMP-PKA+signaling&doi=10.1038%2Fnpp.2014.106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Improved Long-Term Memory via Enhancing cGMP-PKG Signaling Requires cAMP-PKA Signaling</span></div><div class="casAuthors">Bollen, Eva; Puzzo, Daniela; Rutten, Kris; Privitera, Lucia; De Vry, Jochen; Vanmierlo, Tim; Kenis, Gunter; Palmeri, Agostino; D'Hooge, Rudi; Balschun, Detlef; Steinbusch, Harry M. W.; Blokland, Arjan; Prickaerts, Jos</div><div class="citationInfo"><span class="NLM_cas:title">Neuropsychopharmacology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2497-2505</span>CODEN:
                <span class="NLM_cas:coden">NEROEW</span>;
        ISSN:<span class="NLM_cas:issn">0893-133X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Memory consolidation is defined by the stabilization of a memory trace after acquisition, and consists of numerous mol. cascades that mediate synaptic plasticity.  Commonly, a distinction is made between an early and a late consolidation phase, in which early refers to the first hours in which labile synaptic changes occur, whereas late consolidation relates to stable and long-lasting synaptic changes induced by de novo protein synthesis.  How these phases are linked at a mol. level is not yet clear.  Here we studied the interaction of the cyclic nucleotide-mediated pathways during the different phases of memory consolidation in rodents.  In addn., the same pathways were studied in a model of neuronal plasticity, long-term potentiation (LTP).  We demonstrated that cGMP/protein kinase G (PKG) signaling mediates early memory consolidation as well as early-phase LTP, whereas cAMP/protein kinase A (PKA) signaling mediates late consolidation and late-phase-like LTP.  In addn., we show for the first time that early-phase cGMP/PKG signaling requires late-phase cAMP/PKA-signaling in both LTP and long-term memory formation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2NJct-ZSPAbVg90H21EOLACvtfcHk0lgNG77QlgDBUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVKls7zF&md5=d8dc4c85e501b91bc918950d7dbf0f1f</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnpp.2014.106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnpp.2014.106%26sid%3Dliteratum%253Aachs%26aulast%3DBollen%26aufirst%3DE.%26aulast%3DPuzzo%26aufirst%3DD.%26aulast%3DRutten%26aufirst%3DK.%26aulast%3DPrivitera%26aufirst%3DL.%26aulast%3DDe%2BVry%26aufirst%3DJ.%26aulast%3DVanmierlo%26aufirst%3DT.%26aulast%3DKenis%26aufirst%3DG.%26aulast%3DPalmeri%26aufirst%3DA.%26aulast%3DD%25E2%2580%2599Hooge%26aufirst%3DR.%26aulast%3DBalschun%26aufirst%3DD.%26aulast%3DSteinbusch%26aufirst%3DH.%2BM.%26aulast%3DBlokland%26aufirst%3DA.%26aulast%3DPrickaerts%26aufirst%3DJ.%26atitle%3DImproved%2520long-term%2520memory%2520via%2520enhancing%2520cGMP-PKG%2520signaling%2520requires%2520cAMP-PKA%2520signaling%26jtitle%3DNeuropsychopharmacology%26date%3D2014%26volume%3D39%26spage%3D2497%26epage%3D2505%26doi%3D10.1038%2Fnpp.2014.106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lueptow, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhan, C.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Donnell, J. M.</span></span> <span> </span><span class="NLM_article-title">Cylic cGMP-mediated memory enhancement in the object recognition test by inhibitors of phosphodiesterase-2 in mice</span>. <i>Psycopharmacology</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>233</i></span>,  <span class="NLM_fpage">447</span>– <span class="NLM_lpage">456</span>, <span class="refDoi"> DOI: 10.1007/s00213-015-4129-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.1007%2Fs00213-015-4129-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=26525565" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=233&publication_year=2016&pages=447-456&author=L.+M.+Lueptowauthor=C.-G.+Zhanauthor=J.+M.+O%E2%80%99Donnell&title=Cylic+cGMP-mediated+memory+enhancement+in+the+object+recognition+test+by+inhibitors+of+phosphodiesterase-2+in+mice&doi=10.1007%2Fs00213-015-4129-1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1007%2Fs00213-015-4129-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00213-015-4129-1%26sid%3Dliteratum%253Aachs%26aulast%3DLueptow%26aufirst%3DL.%2BM.%26aulast%3DZhan%26aufirst%3DC.-G.%26aulast%3DO%25E2%2580%2599Donnell%26aufirst%3DJ.%2BM.%26atitle%3DCylic%2520cGMP-mediated%2520memory%2520enhancement%2520in%2520the%2520object%2520recognition%2520test%2520by%2520inhibitors%2520of%2520phosphodiesterase-2%2520in%2520mice%26jtitle%3DPsycopharmacology%26date%3D2016%26volume%3D233%26spage%3D447%26epage%3D456%26doi%3D10.1007%2Fs00213-015-4129-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Elias, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beiser, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Au, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Agostino, R. B.</span></span> <span> </span><span class="NLM_article-title">The preclinical phase of Alzheimer disease: A 22-year prospective study of the Framingham Cohort</span>. <i>Arch. Neurol.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">808</span>– <span class="NLM_lpage">813</span>, <span class="refDoi"> DOI: 10.1001/archneur.57.6.808</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.1001%2Farchneur.57.6.808" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10867777" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A280%3ADC%252BD3czivVWgtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2000&pages=808-813&author=M.+F.+Eliasauthor=A.+Beiserauthor=P.+A.+Wolfauthor=R.+Auauthor=R.+F.+Whiteauthor=R.+B.+D%E2%80%99Agostino&title=The+preclinical+phase+of+Alzheimer+disease%3A+A+22-year+prospective+study+of+the+Framingham+Cohort&doi=10.1001%2Farchneur.57.6.808"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">The preclinical phase of alzheimer disease: A 22-year prospective study of the Framingham Cohort</span></div><div class="casAuthors">Elias M F; Beiser A; Wolf P A; Au R; White R F; D'Agostino R B</div><div class="citationInfo"><span class="NLM_cas:title">Archives of neurology</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">808-13</span>
        ISSN:<span class="NLM_cas:issn">0003-9942</span>.
    </div><div class="casAbstract">OBJECTIVES:  To relate performance on tests of cognitive ability to the subsequent development of probable Alzheimer disease (pAD) and to identify the pattern of earliest changes in cognitive functioning associated with a diagnosis of pAD.  DESIGN:  From May 1975 to November 1979, a screening neuropsychological battery was administered to Framingham Study participants.  They were followed up prospectively for 22 years and examined at least every 2 years for the development of pAD.  SETTING:  A community-based center for epidemiological research.  PARTICIPANTS:  Subjects were 1076 participants of the Framingham Study aged 65 to 94 years who were free of dementia and stroke at baseline (initial) neuropsychological testing.  MAIN OUTCOME MEASURE:  Presence or absence of pAD during a 22-year surveillance period was related to test performance at initial neuropsychological testing.  RESULTS:  Lower scores for measures of new learning, recall, retention, and abstract reasoning obtained during a dementia-free period were associated with the development of pAD.  Lower scores for measures of abstract reasoning and retention predicted pAD after a dementia-free period of 10 years.  CONCLUSIONS:  The "preclinical phase" of detectable lowering of cognitive functioning precedes the appearance of pAD by many years.  Measures of retention of information and abstract reasoning are among the strongest predictors of pAD when the interval between initial assessment and the development of pAD is long.  Arch Neurol. 2000.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSXl6yMaOTr2KNc3HjJRoFAfW6udTcc2ea8-85IywAUXLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3czivVWgtA%253D%253D&md5=4d9dcc434215c236965c6558eb998cb7</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1001%2Farchneur.57.6.808&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1001%252Farchneur.57.6.808%26sid%3Dliteratum%253Aachs%26aulast%3DElias%26aufirst%3DM.%2BF.%26aulast%3DBeiser%26aufirst%3DA.%26aulast%3DWolf%26aufirst%3DP.%2BA.%26aulast%3DAu%26aufirst%3DR.%26aulast%3DWhite%26aufirst%3DR.%2BF.%26aulast%3DD%25E2%2580%2599Agostino%26aufirst%3DR.%2BB.%26atitle%3DThe%2520preclinical%2520phase%2520of%2520Alzheimer%2520disease%253A%2520A%252022-year%2520prospective%2520study%2520of%2520the%2520Framingham%2520Cohort%26jtitle%3DArch.%2520Neurol.%26date%3D2000%26volume%3D57%26spage%3D808%26epage%3D813%26doi%3D10.1001%2Farchneur.57.6.808" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Newman, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warrington, E. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossor, M. N.</span></span> <span> </span><span class="NLM_article-title">The earliest cognitive change in a person with familial Alzheimer’s disease: presymptomatic neuropsychological features in a pedigree with familial Alzheimer’s disease confirmed at necropsy</span>. <i>J. Neurol., Neurosurg. Psychiatry</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">967</span>– <span class="NLM_lpage">972</span>, <span class="refDoi"> DOI: 10.1136/jnnp.57.8.967</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.1136%2Fjnnp.57.8.967" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=8057122" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A280%3ADyaK2czjsVKqug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=1994&pages=967-972&author=S.+K.+Newmanauthor=E.+K.+Warringtonauthor=A.+M.+Kennedyauthor=M.+N.+Rossor&title=The+earliest+cognitive+change+in+a+person+with+familial+Alzheimer%E2%80%99s+disease%3A+presymptomatic+neuropsychological+features+in+a+pedigree+with+familial+Alzheimer%E2%80%99s+disease+confirmed+at+necropsy&doi=10.1136%2Fjnnp.57.8.967"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">The earliest cognitive change in a person with familial Alzheimer's disease: presymptomatic neuropsychological features in a pedigree with familial Alzheimer's disease confirmed at necropsy</span></div><div class="casAuthors">Newman S K; Warrington E K; Kennedy A M; Rossor M N</div><div class="citationInfo"><span class="NLM_cas:title">Journal of neurology, neurosurgery, and psychiatry</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">967-72</span>
        ISSN:<span class="NLM_cas:issn">0022-3050</span>.
    </div><div class="casAbstract">Comprehensive, longitudinal neuropsychological assessments are reported in a person "at risk" from autosomal dominant, necropsy confirmed familial Alzheimer's disease.  The first assessment showed a moderately selective verbal memory deficit in the context of mild general intellectual impairment.  Subsequent testing showed the progressive deterioration of visual memory and a mild decline of perceptual and spatial skills.  Language and literacy skills, however, remained comparatively intact.  The neuropsychological profiles obtained at each assessment are presented in profile maps.  These permit direct longitudinal comparison of cognitive function, and may serve in the comparison of different potential cases of familial Alzheimer's disease.  This case sought medical attention for memory difficulties 26 months after the first neuropsychological assessment.  These results mark the first cognitive manifestations in a pedigree with familial Alzheimer's disease which, in this case, were seen presymptomatically.  The findings are discussed in relation to neuropsychological studies of affected cases, and in terms of their reflecting the heterogeneous nature of familial Alzheimer's disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ5RO_FBTlCYHELvajU9pjPfW6udTcc2ea8-85IywAUXLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK2czjsVKqug%253D%253D&md5=85b544e27364de30a6c7a4cf6c0e857e</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1136%2Fjnnp.57.8.967&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Fjnnp.57.8.967%26sid%3Dliteratum%253Aachs%26aulast%3DNewman%26aufirst%3DS.%2BK.%26aulast%3DWarrington%26aufirst%3DE.%2BK.%26aulast%3DKennedy%26aufirst%3DA.%2BM.%26aulast%3DRossor%26aufirst%3DM.%2BN.%26atitle%3DThe%2520earliest%2520cognitive%2520change%2520in%2520a%2520person%2520with%2520familial%2520Alzheimer%25E2%2580%2599s%2520disease%253A%2520presymptomatic%2520neuropsychological%2520features%2520in%2520a%2520pedigree%2520with%2520familial%2520Alzheimer%25E2%2580%2599s%2520disease%2520confirmed%2520at%2520necropsy%26jtitle%3DJ.%2520Neurol.%252C%2520Neurosurg.%2520Psychiatry%26date%3D1994%26volume%3D57%26spage%3D967%26epage%3D972%26doi%3D10.1136%2Fjnnp.57.8.967" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mayeux, R.</span></span> <span> </span><span class="NLM_article-title">Early Alzheimer’s Disease</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>362</i></span>,  <span class="NLM_fpage">2194</span>– <span class="NLM_lpage">2201</span>, <span class="refDoi"> DOI: 10.1056/nejmcp0910236</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.1056%2FNEJMcp0910236" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=20558370" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnsV2htrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=362&publication_year=2010&pages=2194-2201&author=R.+Mayeux&title=Early+Alzheimer%E2%80%99s+Disease&doi=10.1056%2Fnejmcp0910236"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Early Alzheimer's disease</span></div><div class="casAuthors">Mayeux, Richard</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">362</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">2194-2201</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">A 72-yr-old man who is still managing investments at a brokerage firm seeks consultation at the urging of his wife for increasing difficulty with memory over the past 2 years.  Clients have expressed concern about his occasional lapses in memory.  His wife reports that he frequently repeats questions about social appointments and becomes angry when she points this out.  The phys. examn. is normal, but the patient has difficulty remembering elements of a brief story and adding a small amt. of change.  He has a score of 28 out of 30 on the Mini-Mental State Examn., indicating slightly impaired cognitive function.  Early Alzheimer's disease is suspected.  How should the patient be further evaluated and treated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2kgZeBUNV27Vg90H21EOLACvtfcHk0lgu2ytkvk7g6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXnsV2htrw%253D&md5=6f8db845fbe83e95f07c1006b80c564b</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1056%2FNEJMcp0910236&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMcp0910236%26sid%3Dliteratum%253Aachs%26aulast%3DMayeux%26aufirst%3DR.%26atitle%3DEarly%2520Alzheimer%25E2%2580%2599s%2520Disease%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2010%26volume%3D362%26spage%3D2194%26epage%3D2201%26doi%3D10.1056%2Fnejmcp0910236" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gabrieli, J. D. E.</span></span> <span> </span><span class="NLM_article-title">Memory systems analyses of mnemonic disorders in aging and age-related diseases</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">13534</span>– <span class="NLM_lpage">13540</span>, <span class="refDoi"> DOI: 10.1073/pnas.93.24.13534</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.1073%2Fpnas.93.24.13534" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=8942968" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADyaK28Xnt1GhsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=1996&pages=13534-13540&author=J.+D.+E.+Gabrieli&title=Memory+systems+analyses+of+mnemonic+disorders+in+aging+and+age-related+diseases&doi=10.1073%2Fpnas.93.24.13534"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Memory systems analyses of mnemonic disorders in aging and age-related diseases</span></div><div class="casAuthors">Gabrieli, John D. E.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">13534-13540</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The effects upon memory of normal aging and two age-related neurodegenerative diseases, Alzheimer disease (AD) and Parkinson disease, are analyzed in terms of memory systems, specific neural networks that mediate specific mnemonic processes.  An occipital memory system mediating implicit visual-perceptual memory appears to be unaffected by aging or AD.  A frontal system that may mediate implicit conceptual memory is affected by Ad but not by normal aging.  Another frontal system that mediates aspects of working and strategic memory is affected by Parkinson disease and, to a lesser extent, by aging.  The aging effect appears to occur during all ages of the adult life-span.  Finally, a medial-temporal system that mediates declarative memory is affected by the late onset of AD.  Studies of intact and impaired memory in age-related diseases suggest that normal aging has markedly different effects upon different memory systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJU9ERkAGVH7Vg90H21EOLACvtfcHk0lglXsqn95i52g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xnt1GhsL8%253D&md5=7c0fbfbf0f7bd6040698e792b98f817d</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1073%2Fpnas.93.24.13534&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.93.24.13534%26sid%3Dliteratum%253Aachs%26aulast%3DGabrieli%26aufirst%3DJ.%2BD.%2BE.%26atitle%3DMemory%2520systems%2520analyses%2520of%2520mnemonic%2520disorders%2520in%2520aging%2520and%2520age-related%2520diseases%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1996%26volume%3D93%26spage%3D13534%26epage%3D13540%26doi%3D10.1073%2Fpnas.93.24.13534" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Belleville, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sylvain-Roy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Boysson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ménard, M.-C.</span></span> <span> </span><span class="NLM_article-title">Chapter 23 Characterizing the memory changes in persons with mild cognitive impairment</span>. <i>Prog. Brain Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>169</i></span>,  <span class="NLM_fpage">365</span>– <span class="NLM_lpage">375</span>, <span class="refDoi"> DOI: 10.1016/s0079-6123(07)00023-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.1016%2FS0079-6123%2807%2900023-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=18394487" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A280%3ADC%252BD1c3jsFyktw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=169&publication_year=2008&pages=365-375&author=S.+Bellevilleauthor=S.+Sylvain-Royauthor=C.+de+Boyssonauthor=M.-C.+M%C3%A9nard&title=Chapter+23+Characterizing+the+memory+changes+in+persons+with+mild+cognitive+impairment&doi=10.1016%2Fs0079-6123%2807%2900023-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Characterizing the memory changes in persons with mild cognitive impairment</span></div><div class="casAuthors">Belleville Sylvie; Sylvain-Roy Stephanie; de Boysson Chloe; Menard Marie-Claude</div><div class="citationInfo"><span class="NLM_cas:title">Progress in brain research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">169</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">365-75</span>
        ISSN:<span class="NLM_cas:issn">0079-6123</span>.
    </div><div class="casAbstract">Persons with mild cognitive impairment (MCI) do not meet criteria for Alzheimer's disease (AD) but are at high risk for developing the disease.  Presence of a memory deficit is a key component in the characterization of MCI.  This chapter presents empirical studies that attempt to describe and understand the nature of the memory deficit in MCI with a focus on episodic memory and working memory.  Cross-sectional studies report prominent deficits of episodic memory characterized by impaired encoding of the contextual information that makes up complex events.  This results in reduced free and cued recall, impaired recognition, and impaired associative learning.  Although semantic encoding is found to be impaired in conditions that rely on explicit and intentional retrieval, preserved semantic processing is found with automatic conditions of testing.  Studies indicate the presence of a partial deficit of working memory with the ability to divide attention being most severely impaired.  However, there appears to be heterogeneity as to the extent of the working memory impairment.  The presence of vascular anomalies on MRI, as well as being in a more advanced stage in the continuum from MCI to AD, are associated with more severe and more pervasive working memory deficits.  Finally, longitudinal studies indicate that the combination of episodic and working memory deficits represents a strong predictor of progression from MCI to AD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTbHR2uYMPv3eIHSmpohpd0fW6udTcc2eaGEENRxmYm27ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1c3jsFyktw%253D%253D&md5=7a02e44c01c5b7148bcf8e1e88fffbbc</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2FS0079-6123%2807%2900023-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0079-6123%252807%252900023-4%26sid%3Dliteratum%253Aachs%26aulast%3DBelleville%26aufirst%3DS.%26aulast%3DSylvain-Roy%26aufirst%3DS.%26aulast%3Dde%2BBoysson%26aufirst%3DC.%26aulast%3DM%25C3%25A9nard%26aufirst%3DM.-C.%26atitle%3DChapter%252023%2520Characterizing%2520the%2520memory%2520changes%2520in%2520persons%2520with%2520mild%2520cognitive%2520impairment%26jtitle%3DProg.%2520Brain%2520Res.%26date%3D2008%26volume%3D169%26spage%3D365%26epage%3D375%26doi%3D10.1016%2Fs0079-6123%2807%2900023-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lyketsos, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fitzpatrick, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breitner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeKosky, S.</span></span> <span> </span><span class="NLM_article-title">Prevalence of neuropsychiatric symptoms in dementia and mild cognitive impairment: results from the cardiovascular health study</span>. <i>JAMA</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>288</i></span>,  <span class="NLM_fpage">1475</span>– <span class="NLM_lpage">1483</span>, <span class="refDoi"> DOI: 10.1001/jama.288.12.1475</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.1001%2Fjama.288.12.1475" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=12243634" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A280%3ADC%252BD38vot1elsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=288&publication_year=2002&pages=1475-1483&author=C.+G.+Lyketsosauthor=O.+Lopezauthor=B.+Jonesauthor=A.+L.+Fitzpatrickauthor=J.+Breitnerauthor=S.+DeKosky&title=Prevalence+of+neuropsychiatric+symptoms+in+dementia+and+mild+cognitive+impairment%3A+results+from+the+cardiovascular+health+study&doi=10.1001%2Fjama.288.12.1475"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Prevalence of neuropsychiatric symptoms in dementia and mild cognitive impairment: results from the cardiovascular health study</span></div><div class="casAuthors">Lyketsos Constantine G; Lopez Oscar; Jones Beverly; Fitzpatrick Annette L; Breitner John; DeKosky Steven</div><div class="citationInfo"><span class="NLM_cas:title">JAMA</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">288</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1475-83</span>
        ISSN:<span class="NLM_cas:issn">0098-7484</span>.
    </div><div class="casAbstract">CONTEXT:  Mild cognitive impairment (MCI) may be a precursor to dementia, at least in some cases.  Dementia and MCI are associated with neuropsychiatric symptoms in clinical samples.  Only 2 population-based studies exist of the prevalence of these symptoms in dementia, and none exist for MCI.  OBJECTIVE:  To estimate the prevalence of neuropsychiatric symptoms in dementia and MCI in a population-based study.  DESIGN:  Cross-sectional study derived from the Cardiovascular Health Study, a longitudinal cohort study.  SETTING AND PARTICIPANTS:  A total of 3608 participants were cognitively evaluated using data collected longitudinally over 10 years and additional data collected in 1999-2000 in 4 US counties.  Dementia and MCI were classified using clinical criteria and adjudicated by committee review by expert neurologists and psychiatrists.  A total of 824 individuals completed the Neuropsychiatric Inventory (NPI); 362 were classified as having dementia, 320 as having MCI; and 142 did not meet criteria for MCI or dementia.  MAIN OUTCOME MEASURE:  Prevalence of neuropsychiatric symptoms, based on ratings on the NPI in the previous month and from the onset of cognitive symptoms.  RESULTS:  Of the 682 individuals with dementia or MCI, 43% of MCI participants (n = 138) exhibited neuropsychiatric symptoms in the previous month (29% rated as clinically significant) with depression (20%), apathy (15%), and irritability (15%) being most common.  Among the dementia participants, 75% (n = 270) had exhibited a neuropsychiatric symptom in the past month (62% were clinically significant); 55% (n = 199) reported 2 or more and 44% (n = 159) 3 or more disturbances in the past month.  In participants with dementia, the most frequent disturbances were apathy (36%), depression (32%), and agitation/aggression (30%).  Eighty percent of dementia participants (n = 233) and 50% of MCI participants (n = 139) exhibited at least 1 NPI symptom from the onset of cognitive symptoms.  There were no differences in prevalence of neuropsychiatric symptoms between participants with Alzheimer-type dementia and those with other dementias, with the exception of aberrant motor behavior, which was more frequent in Alzheimer-type dementia (5.4% vs 1%; P =.02).  CONCLUSIONS:  Neuropsychiatric symptoms occur in the majority of persons with dementia over the course of the disease.  These are the first population-based estimates for neuropsychiatric symptoms in MCI, indicating a high prevalence associated with this condition as well.  These symptoms have serious adverse consequences and should be inquired about and treated as necessary.  Study of neuropsychiatric symptoms in the context of dementia may improve our understanding of brain-behavior relationships.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRiqzKr_B3vlZHZzFRCXxVKfW6udTcc2eaGEENRxmYm27ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD38vot1elsg%253D%253D&md5=1dccd5468a3ba37bb56ba804694e29ff</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1001%2Fjama.288.12.1475&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1001%252Fjama.288.12.1475%26sid%3Dliteratum%253Aachs%26aulast%3DLyketsos%26aufirst%3DC.%2BG.%26aulast%3DLopez%26aufirst%3DO.%26aulast%3DJones%26aufirst%3DB.%26aulast%3DFitzpatrick%26aufirst%3DA.%2BL.%26aulast%3DBreitner%26aufirst%3DJ.%26aulast%3DDeKosky%26aufirst%3DS.%26atitle%3DPrevalence%2520of%2520neuropsychiatric%2520symptoms%2520in%2520dementia%2520and%2520mild%2520cognitive%2520impairment%253A%2520results%2520from%2520the%2520cardiovascular%2520health%2520study%26jtitle%3DJAMA%26date%3D2002%26volume%3D288%26spage%3D1475%26epage%3D1483%26doi%3D10.1001%2Fjama.288.12.1475" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blennow, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Leon, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zetterberg, H.</span></span> <span> </span><span class="NLM_article-title">Alzheimer’s disease</span>. <i>Lancet</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>368</i></span>,  <span class="NLM_fpage">387</span>– <span class="NLM_lpage">403</span>, <span class="refDoi"> DOI: 10.1016/s0140-6736(06)69113-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.1016%2Fs0140-6736%2806%2969113-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=16876668" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BD28XnsFymsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=368&publication_year=2006&pages=387-403&author=K.+Blennowauthor=M.+J.+de+Leonauthor=H.+Zetterberg&title=Alzheimer%E2%80%99s+disease&doi=10.1016%2Fs0140-6736%2806%2969113-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Alzheimer's disease</span></div><div class="casAuthors">Blennow, Kaj; de Leon, Mony J.; Zetterberg, Henrik</div><div class="citationInfo"><span class="NLM_cas:title">Lancet</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">368</span>
        (<span class="NLM_cas:issue">9533</span>),
    <span class="NLM_cas:pages">387-403</span>CODEN:
                <span class="NLM_cas:coden">LANCAO</span>;
        ISSN:<span class="NLM_cas:issn">0140-6736</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Alzheimer's disease is the most common cause of dementia.  Research advances have enabled detailed understanding of the mol. pathogenesis of the hallmarks of the disease-ie, plaques, composed of amyloid β (Aβ), and tangles, composed of hyperphosphorylated tau.  However, as our knowledge increases so does our appreciation for the pathogenic complexity of the disorder.  Alzheimer's disease is a very rare autosomal dominant disease with early onset, caused by mutations in the amyloid precursor protein and presenilin genes, both linked to Aβ metab.  By contrast with familial disease, sporadic Alzheimer's disease is very common with more than 15 million people affected worldwide.  The cause of the sporadic form of the disease is unknown, probably because the disease is heterogeneous, caused by aging in concert with a complex interaction of both genetic and environmental risk factors.  This seminar reviews the key aspects of the disease, including epidemiol., genetics, pathogenesis, diagnosis, and treatment, as well as recent developments and controversies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriOIYvvzbyybVg90H21EOLACvtfcHk0lhBxwp_Y36JCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XnsFymsbY%253D&md5=f70c374fbe01be1953bd0eea290745bd</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fs0140-6736%2806%2969113-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs0140-6736%252806%252969113-7%26sid%3Dliteratum%253Aachs%26aulast%3DBlennow%26aufirst%3DK.%26aulast%3Dde%2BLeon%26aufirst%3DM.%2BJ.%26aulast%3DZetterberg%26aufirst%3DH.%26atitle%3DAlzheimer%25E2%2580%2599s%2520disease%26jtitle%3DLancet%26date%3D2006%26volume%3D368%26spage%3D387%26epage%3D403%26doi%3D10.1016%2Fs0140-6736%2806%2969113-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gomez, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breitenbucher, J. G.</span></span> <span> </span><span class="NLM_article-title">PDE2A inhibition: potential for the treatment of cognitive disorders</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">6522</span>– <span class="NLM_lpage">6527</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2013.10.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.1016%2Fj.bmcl.2013.10.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=24189054" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslartbjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=6522-6527&author=L.+Gomezauthor=J.+G.+Breitenbucher&title=PDE2A+inhibition%3A+potential+for+the+treatment+of+cognitive+disorders&doi=10.1016%2Fj.bmcl.2013.10.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">PDE2 inhibition: Potential for the treatment of cognitive disorders</span></div><div class="casAuthors">Gomez, Laurent; Breitenbucher, J. Guy</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">6522-6527</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Phosphodiesterase inhibition has received much attention in the past 20 years for the potential treatment of CNS disorders.  A primary focus of this work is the enhancement of memory and/or cognitive functioning.  The role of PDEs in the augmentation of cyclic nucleotide signaling makes these enzymes attractive targets for enhancing the effects of neuronal communication.  This review focuses on recent findings with respect to the role of PDE2 inhibition in cognitive functioning.  Special attention is paid to recently disclosed, selective tool compds. and the use of these tool compds. to support the role of PDE2 inhibition in cognition.  Recently reported SAR and modeling work will be presented along with discussion of the entry of new PDE2 inhibitors into the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGUUFzh2-kTrVg90H21EOLACvtfcHk0lhBxwp_Y36JCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslartbjL&md5=04ca2bdc35dc0b436a228ba62934b91e</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2013.10.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2013.10.014%26sid%3Dliteratum%253Aachs%26aulast%3DGomez%26aufirst%3DL.%26aulast%3DBreitenbucher%26aufirst%3DJ.%2BG.%26atitle%3DPDE2A%2520inhibition%253A%2520potential%2520for%2520the%2520treatment%2520of%2520cognitive%2520disorders%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D23%26spage%3D6522%26epage%3D6527%26doi%3D10.1016%2Fj.bmcl.2013.10.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, H.-B.</span></span> <span> </span><span class="NLM_article-title">Novel phosphodiesterase inhibitors for cognitive improvement in Alzheimer’s disease</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">5467</span>– <span class="NLM_lpage">5483</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01370</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01370" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFGqu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=5467-5483&author=Y.+Wuauthor=Z.+Liauthor=Y.-Y.+Huangauthor=D.+Wuauthor=H.-B.+Luo&title=Novel+phosphodiesterase+inhibitors+for+cognitive+improvement+in+Alzheimer%E2%80%99s+disease&doi=10.1021%2Facs.jmedchem.7b01370"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Phosphodiesterase Inhibitors for Cognitive Improvement in Alzheimer's Disease</span></div><div class="casAuthors">Wu, Yinuo; Li, Zhe; Huang, Yi-You; Wu, Deyan; Luo, Hai-Bin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">5467-5483</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Alzheimer's disease (AD) is one of the greatest public health challenges.  Phosphodiesterases (PDEs) are a superenzyme family responsible for the hydrolysis of two second messengers: cyclic adenosine monophosphate (cAMP) and cyclic guanosine monophosphate (cGMP).  Since several PDE subfamilies are highly expressed in the human brain, the inhibition of PDEs is involved in neurodegenerative processes by regulating the concn. of cAMP and/or cGMP.  Currently, PDEs are considered as promising targets for the treatment of AD since many PDE inhibitors have exhibited remarkable cognitive improvement effects in preclin. studies and over 15 of them have been subjected to clin. trials.  The aim of this review is to summarize the outstanding progress that has been made by PDE inhibitors as anti-AD agents with encouraging results in preclin. studies and clin. trials.  The binding affinity, pharmacokinetics, underlying mechanisms, and limitations of these PDE inhibitors in the treatment of AD are also reviewed and discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGom1l9eMNKCt7Vg90H21EOLACvtfcHk0lhBxwp_Y36JCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFGqu7w%253D&md5=87cd22fa21c661f4cb5dc2155368bd7c</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01370&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01370%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DHuang%26aufirst%3DY.-Y.%26aulast%3DWu%26aufirst%3DD.%26aulast%3DLuo%26aufirst%3DH.-B.%26atitle%3DNovel%2520phosphodiesterase%2520inhibitors%2520for%2520cognitive%2520improvement%2520in%2520Alzheimer%25E2%2580%2599s%2520disease%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D5467%26epage%3D5483%26doi%3D10.1021%2Facs.jmedchem.7b01370" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gomez, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Massari, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vickers, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freestone, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vernier, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ly, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCarrick, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marrone, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Y. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoder, Z. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemus, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Broadbent, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barido, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warren, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmelzer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neul, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersen, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sebring, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aertgeerts, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tabatabaei, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breitenbucher, J. G.</span></span> <span> </span><span class="NLM_article-title">Design and synthesis of novel and selective phosphodiesterase 2 (PDE2a) inhibitors for the treatment of memory disorders</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2037</span>– <span class="NLM_lpage">2051</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01793</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01793" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC2sXit1Sjuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=2037-2051&author=L.+Gomezauthor=M.+E.+Massariauthor=T.+Vickersauthor=G.+Freestoneauthor=W.+Vernierauthor=K.+Lyauthor=R.+Xuauthor=M.+McCarrickauthor=T.+Marroneauthor=M.+Metzauthor=Y.+G.+Yanauthor=Z.+W.+Yoderauthor=R.+Lemusauthor=N.+J.+Broadbentauthor=R.+Baridoauthor=N.+Warrenauthor=K.+Schmelzerauthor=D.+Neulauthor=D.+Leeauthor=C.+B.+Andersenauthor=K.+Sebringauthor=K.+Aertgeertsauthor=X.+Zhouauthor=A.+Tabatabaeiauthor=M.+Petersauthor=J.+G.+Breitenbucher&title=Design+and+synthesis+of+novel+and+selective+phosphodiesterase+2+%28PDE2a%29+inhibitors+for+the+treatment+of+memory+disorders&doi=10.1021%2Facs.jmedchem.6b01793"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Synthesis of Novel and Selective Phosphodiesterase 2 (PDE2a) Inhibitors for the Treatment of Memory Disorders</span></div><div class="casAuthors">Gomez, Laurent; Massari, Mark Eben; Vickers, Troy; Freestone, Graeme; Vernier, William; Ly, Kiev; Xu, Rui; McCarrick, Margaret; Marrone, Tami; Metz, Markus; Yan, Yingzhou G.; Yoder, Zachary W.; Lemus, Robert; Broadbent, Nicola J.; Barido, Richard; Warren, Noelle; Schmelzer, Kara; Neul, David; Lee, Dong; Andersen, Carsten B.; Sebring, Kristen; Aertgeerts, Kathleen; Zhou, Xianbo; Tabatabaei, Ali; Peters, Marco; Breitenbucher, J. Guy</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2037-2051</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of potent and selective [1,2,4]triazolo[1,5-a]pyrimidine PDE2a inhibitors is reported.  The design and improvement of the binding properties of this series was achieved using X-ray crystal structures in conjunction with careful anal. of electronic and structural requirements for the PDE2a enzyme.  One of the lead compds., compd. I (DNS-8254), was identified as a potent and highly selective PDE2a enzyme inhibitor with favorable rat pharmacokinetic properties.  Interestingly, the increased potency of compd. I was facilitated by the formation of a halogen bond with the oxygen of Tyr827 present in the PDE2a active site.  In vivo, compd. I demonstrated significant memory enhancing effects in a rat model of novel object recognition.  Taken together, these data suggest that compd. I may be a useful tool to explore the pharmacol. of selective PDE2a inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6ML_QxZhnWLVg90H21EOLACvtfcHk0lgYU5uJXyHkmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXit1Sjuro%253D&md5=fe8df71fbd1f61423352dfcb862d9d53</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01793&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01793%26sid%3Dliteratum%253Aachs%26aulast%3DGomez%26aufirst%3DL.%26aulast%3DMassari%26aufirst%3DM.%2BE.%26aulast%3DVickers%26aufirst%3DT.%26aulast%3DFreestone%26aufirst%3DG.%26aulast%3DVernier%26aufirst%3DW.%26aulast%3DLy%26aufirst%3DK.%26aulast%3DXu%26aufirst%3DR.%26aulast%3DMcCarrick%26aufirst%3DM.%26aulast%3DMarrone%26aufirst%3DT.%26aulast%3DMetz%26aufirst%3DM.%26aulast%3DYan%26aufirst%3DY.%2BG.%26aulast%3DYoder%26aufirst%3DZ.%2BW.%26aulast%3DLemus%26aufirst%3DR.%26aulast%3DBroadbent%26aufirst%3DN.%2BJ.%26aulast%3DBarido%26aufirst%3DR.%26aulast%3DWarren%26aufirst%3DN.%26aulast%3DSchmelzer%26aufirst%3DK.%26aulast%3DNeul%26aufirst%3DD.%26aulast%3DLee%26aufirst%3DD.%26aulast%3DAndersen%26aufirst%3DC.%2BB.%26aulast%3DSebring%26aufirst%3DK.%26aulast%3DAertgeerts%26aufirst%3DK.%26aulast%3DZhou%26aufirst%3DX.%26aulast%3DTabatabaei%26aufirst%3DA.%26aulast%3DPeters%26aufirst%3DM.%26aulast%3DBreitenbucher%26aufirst%3DJ.%2BG.%26atitle%3DDesign%2520and%2520synthesis%2520of%2520novel%2520and%2520selective%2520phosphodiesterase%25202%2520%2528PDE2a%2529%2520inhibitors%2520for%2520the%2520treatment%2520of%2520memory%2520disorders%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D2037%26epage%3D2051%26doi%3D10.1021%2Facs.jmedchem.6b01793" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mikami, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashizawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nomura, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawasaki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasaki, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kokubo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uchiyama, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakashima, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamiguchi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imada, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwashita, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taniguchi, T.</span></span> <span> </span><span class="NLM_article-title">Discovery of clinical candidate N-((1S)-1-(3-fluoro-4-(trifluoromethoxy)phenyl)-2-methoxyethyl)-7-methoxy-2-oxo-2,3-dihydropyrido[2,3-b]pyrazine-4(1H)-carboxamide (TAK-915): a highly potent, selective, and brain-penetrating phosphodiesterase 2A inhibitor for the treatment of cognitive disorders</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">7677</span>– <span class="NLM_lpage">7702</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00807</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00807" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlSmtL3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=7677-7702&author=S.+Mikamiauthor=S.+Nakamuraauthor=T.+Ashizawaauthor=I.+Nomuraauthor=M.+Kawasakiauthor=S.+Sasakiauthor=H.+Okiauthor=H.+Kokuboauthor=I.+D.+Hoffmanauthor=H.+Zouauthor=N.+Uchiyamaauthor=K.+Nakashimaauthor=N.+Kamiguchiauthor=H.+Imadaauthor=N.+Suzukiauthor=H.+Iwashitaauthor=T.+Taniguchi&title=Discovery+of+clinical+candidate+N-%28%281S%29-1-%283-fluoro-4-%28trifluoromethoxy%29phenyl%29-2-methoxyethyl%29-7-methoxy-2-oxo-2%2C3-dihydropyrido%5B2%2C3-b%5Dpyrazine-4%281H%29-carboxamide+%28TAK-915%29%3A+a+highly+potent%2C+selective%2C+and+brain-penetrating+phosphodiesterase+2A+inhibitor+for+the+treatment+of+cognitive+disorders&doi=10.1021%2Facs.jmedchem.7b00807"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Clinical Candidate N-((1S)-1-(3-Fluoro-4-(trifluoromethoxy)phenyl)-2-methoxyethyl)-7-methoxy-2-oxo-2,3-dihydropyrido[2,3-b]pyrazine-4(1H)-carboxamide (TAK-915): A Highly Potent, Selective, and Brain-Penetrating Phosphodiesterase 2A Inhibitor for the Treatment of Cognitive Disorders</span></div><div class="casAuthors">Mikami, Satoshi; Nakamura, Shinji; Ashizawa, Tomoko; Nomura, Izumi; Kawasaki, Masanori; Sasaki, Shigekazu; Oki, Hideyuki; Kokubo, Hironori; Hoffman, Isaac D.; Zou, Hua; Uchiyama, Noriko; Nakashima, Kosuke; Kamiguchi, Naomi; Imada, Haruka; Suzuki, Noriko; Iwashita, Hiroki; Taniguchi, Takahiko</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">7677-7702</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Phosphodiesterase (PDE) 2A inhibitors have emerged as a novel mechanism with potential therapeutic option to ameliorate cognitive dysfunction in schizophrenia or Alzheimer's disease through upregulation of cyclic nucleotides in the brain, and thereby achieve potentiation of cyclic nucleotide signaling pathways.  This article details the expedited optimization of the authors' recently disclosed pyrazolo[1,5-a]pyrimidine lead compd. 4b, leading to the discovery of clin. candidate 36 (I) (TAK-915), which demonstrates an appropriate combination of potency, PDE selectivity, and favorable pharmacokinetic (PK) properties, including brain penetration.  Successful identification of 36 was realized through application of structure-based drug design (SBDD) to further improve potency and PDE selectivity, coupled with prospective design focused on physicochem. properties to deliver brain penetration.  Oral administration of 36 demonstrated significant elevation of 3',5'-cGMP levels in mouse brains, and improved cognitive performance in a novel object recognition task in rats.  Consequently, compd. 36 was advanced into human clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCUw5grC_66rVg90H21EOLACvtfcHk0lgYU5uJXyHkmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlSmtL3I&md5=f87cb3ac86b6299fc90e336b88ea2a21</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00807&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00807%26sid%3Dliteratum%253Aachs%26aulast%3DMikami%26aufirst%3DS.%26aulast%3DNakamura%26aufirst%3DS.%26aulast%3DAshizawa%26aufirst%3DT.%26aulast%3DNomura%26aufirst%3DI.%26aulast%3DKawasaki%26aufirst%3DM.%26aulast%3DSasaki%26aufirst%3DS.%26aulast%3DOki%26aufirst%3DH.%26aulast%3DKokubo%26aufirst%3DH.%26aulast%3DHoffman%26aufirst%3DI.%2BD.%26aulast%3DZou%26aufirst%3DH.%26aulast%3DUchiyama%26aufirst%3DN.%26aulast%3DNakashima%26aufirst%3DK.%26aulast%3DKamiguchi%26aufirst%3DN.%26aulast%3DImada%26aufirst%3DH.%26aulast%3DSuzuki%26aufirst%3DN.%26aulast%3DIwashita%26aufirst%3DH.%26aulast%3DTaniguchi%26aufirst%3DT.%26atitle%3DDiscovery%2520of%2520clinical%2520candidate%2520N-%2528%25281S%2529-1-%25283-fluoro-4-%2528trifluoromethoxy%2529phenyl%2529-2-methoxyethyl%2529-7-methoxy-2-oxo-2%252C3-dihydropyrido%255B2%252C3-b%255Dpyrazine-4%25281H%2529-carboxamide%2520%2528TAK-915%2529%253A%2520a%2520highly%2520potent%252C%2520selective%252C%2520and%2520brain-penetrating%2520phosphodiesterase%25202A%2520inhibitor%2520for%2520the%2520treatment%2520of%2520cognitive%2520disorders%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D7677%26epage%3D7702%26doi%3D10.1021%2Facs.jmedchem.7b00807" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Helal, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyden, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chappie, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fennell, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forman, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hajos, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harms, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphrey, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kleiman, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kormos, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maklad, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDowell, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mente, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connor, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandit, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piotrowski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueno, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verhoest, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, E. X.</span></span> <span> </span><span class="NLM_article-title">Application of structure-based design and parallel chemistry to identify a potent, selective, and brain penetrant Phosphodiesterase 2A inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">5673</span>– <span class="NLM_lpage">5698</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00397</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00397" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpt1Sitb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=5673-5698&author=C.+J.+Helalauthor=E.+P.+Arnoldauthor=T.+L.+Boydenauthor=C.+Changauthor=T.+A.+Chappieauthor=K.+F.+Fennellauthor=M.+D.+Formanauthor=M.+Hajosauthor=J.+F.+Harmsauthor=W.+E.+Hoffmanauthor=J.+M.+Humphreyauthor=Z.+Kangauthor=R.+J.+Kleimanauthor=B.+L.+Kormosauthor=C.-W.+Leeauthor=J.+Luauthor=N.+Makladauthor=L.+McDowellauthor=S.+Menteauthor=R.+E.+O%E2%80%99Connorauthor=J.+Panditauthor=M.+Piotrowskiauthor=A.+W.+Schmidtauthor=C.+J.+Schmidtauthor=H.+Uenoauthor=P.+R.+Verhoestauthor=E.+X.+Yang&title=Application+of+structure-based+design+and+parallel+chemistry+to+identify+a+potent%2C+selective%2C+and+brain+penetrant+Phosphodiesterase+2A+inhibitor&doi=10.1021%2Facs.jmedchem.7b00397"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Application of Structure-Based Design and Parallel Chemistry to Identify a Potent, Selective, and Brain Penetrant Phosphodiesterase 2A Inhibitor</span></div><div class="casAuthors">Helal, Christopher J.; Arnold, Eric P.; Boyden, Tracey L.; Chang, Cheng; Chappie, Thomas A.; Fennell, Kimberly F.; Forman, Michael D.; Hajos, Mihaly; Harms, John F.; Hoffman, William E.; Humphrey, John M.; Kang, Zhijun; Kleiman, Robin J.; Kormos, Bethany L.; Lee, Che-Wah; Lu, Jiemin; Maklad, Noha; McDowell, Laura; Mente, Scot; O'Connor, Rebecca E.; Pandit, Jayvardhan; Piotrowski, Mary; Schmidt, Anne W.; Schmidt, Christopher J.; Ueno, Hirokazu; Verhoest, Patrick R.; Yang, Edward X.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">5673-5698</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Phosphodiesterase 2A (PDE2A) inhibitors have been reported to demonstrate in vivo activity in preclin. models of cognition.  To more fully explore the biol. of PDE2A inhibition, we sought to identify potent PDE2A inhibitors with improved brain penetration as compared to current literature compds.  Applying estd. human dose calcns. while simultaneously leveraging synthetically enabled chem. and structure-based drug design has resulted in a highly potent, selective, brain penetrant compd. 71 (PF-05085727) that effects in vivo biochem. changes commensurate with PDE2A inhibition along with behavioral and electrophysiol. reversal of the effects of NMDA antagonists in rodents.  This data supports the ability of PDE2A inhibitors to potentiate NMDA signaling and their further development for clin. cognition indications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpZdZgTF6GMLVg90H21EOLACvtfcHk0ljNSGYgNo9TQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpt1Sitb4%253D&md5=e1c3c778fc622ac2f5917625a983323f</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00397&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00397%26sid%3Dliteratum%253Aachs%26aulast%3DHelal%26aufirst%3DC.%2BJ.%26aulast%3DArnold%26aufirst%3DE.%2BP.%26aulast%3DBoyden%26aufirst%3DT.%2BL.%26aulast%3DChang%26aufirst%3DC.%26aulast%3DChappie%26aufirst%3DT.%2BA.%26aulast%3DFennell%26aufirst%3DK.%2BF.%26aulast%3DForman%26aufirst%3DM.%2BD.%26aulast%3DHajos%26aufirst%3DM.%26aulast%3DHarms%26aufirst%3DJ.%2BF.%26aulast%3DHoffman%26aufirst%3DW.%2BE.%26aulast%3DHumphrey%26aufirst%3DJ.%2BM.%26aulast%3DKang%26aufirst%3DZ.%26aulast%3DKleiman%26aufirst%3DR.%2BJ.%26aulast%3DKormos%26aufirst%3DB.%2BL.%26aulast%3DLee%26aufirst%3DC.-W.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DMaklad%26aufirst%3DN.%26aulast%3DMcDowell%26aufirst%3DL.%26aulast%3DMente%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DR.%2BE.%26aulast%3DPandit%26aufirst%3DJ.%26aulast%3DPiotrowski%26aufirst%3DM.%26aulast%3DSchmidt%26aufirst%3DA.%2BW.%26aulast%3DSchmidt%26aufirst%3DC.%2BJ.%26aulast%3DUeno%26aufirst%3DH.%26aulast%3DVerhoest%26aufirst%3DP.%2BR.%26aulast%3DYang%26aufirst%3DE.%2BX.%26atitle%3DApplication%2520of%2520structure-based%2520design%2520and%2520parallel%2520chemistry%2520to%2520identify%2520a%2520potent%252C%2520selective%252C%2520and%2520brain%2520penetrant%2520Phosphodiesterase%25202A%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D5673%26epage%3D5698%26doi%3D10.1021%2Facs.jmedchem.7b00397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stachel, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nomland, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ginnetti, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paone, D. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puri, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lange, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drott, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcus, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dwyer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uslaner, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, S. M.</span></span> <span> </span><span class="NLM_article-title">Structure-Guided design and procognitive assessment of a potent and selective phosphodiesterase 2A inhibitor</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">815</span>– <span class="NLM_lpage">820</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.8b00214</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.8b00214" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlykurrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=815-820&author=S.+J.+Stachelauthor=R.+Bergerauthor=A.+B.+Nomlandauthor=A.+T.+Ginnettiauthor=D.+V.+Paoneauthor=D.+Wangauthor=V.+Puriauthor=H.+Langeauthor=J.+Drottauthor=J.+Luauthor=J.+Marcusauthor=M.+P.+Dwyerauthor=S.+Suonauthor=J.+M.+Uslanerauthor=S.+M.+Smith&title=Structure-Guided+design+and+procognitive+assessment+of+a+potent+and+selective+phosphodiesterase+2A+inhibitor&doi=10.1021%2Facsmedchemlett.8b00214"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Guided Design and Procognitive Assessment of a Potent and Selective Phosphodiesterase 2A Inhibitor</span></div><div class="casAuthors">Stachel, Shawn J.; Berger, Richard; Nomland, Ashley B.; Ginnetti, Anthony T.; Paone, Daniel V.; Wang, Deping; Puri, Vanita; Lange, Henry; Drott, Jason; Lu, Jun; Marcus, Jacob; Dwyer, Michael P.; Suon, Sokreine; Uslaner, Jason M.; Smith, Sean M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">815-820</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Herein the authors describe the development of a series of pyrazolopyrimidinone phosphodiesterase 2A (PDE2) inhibitors using structure-guided lead identification and design.  The series was derived from informed chemotype replacement based on previously identified internal leads.  The initially designed compd. 3 ((S)-3,6-dimethyl-1-(1-(4-(trifluoromethyl)phenyl)ethyl)-1,5-dihydro-4H-pyrazolo[3,4-d]pyrimidin-4-one), while potent on PDE2, displayed unsatisfactory selectivity against the other PDE2 isoforms.  Compd. 3 was subsequently optimized for improved PDE2 activity and isoform selectivity.  Insights into the origins of PDE2 selectivity are described and verified using cocrystallog.  An optimized lead, 4 ((S)-3-(hydroxymethyl)-6-methyl-1-(1-(4-(trifluoromethyl)phenyl)ethyl)-1,5-dihydro-4H-pyrazolo[3,4-d]pyrimidin-4-one), demonstrated improved performance in both a rodent and a nonhuman primate cognition model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAr9ZG63azTrVg90H21EOLACvtfcHk0ljNSGYgNo9TQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlykurrE&md5=049e2f26a35c71a2cdace40f2d6525b5</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.8b00214&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.8b00214%26sid%3Dliteratum%253Aachs%26aulast%3DStachel%26aufirst%3DS.%2BJ.%26aulast%3DBerger%26aufirst%3DR.%26aulast%3DNomland%26aufirst%3DA.%2BB.%26aulast%3DGinnetti%26aufirst%3DA.%2BT.%26aulast%3DPaone%26aufirst%3DD.%2BV.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DPuri%26aufirst%3DV.%26aulast%3DLange%26aufirst%3DH.%26aulast%3DDrott%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DMarcus%26aufirst%3DJ.%26aulast%3DDwyer%26aufirst%3DM.%2BP.%26aulast%3DSuon%26aufirst%3DS.%26aulast%3DUslaner%26aufirst%3DJ.%2BM.%26aulast%3DSmith%26aufirst%3DS.%2BM.%26atitle%3DStructure-Guided%2520design%2520and%2520procognitive%2520assessment%2520of%2520a%2520potent%2520and%2520selective%2520phosphodiesterase%25202A%2520inhibitor%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2018%26volume%3D9%26spage%3D815%26epage%3D820%26doi%3D10.1021%2Facsmedchemlett.8b00214" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Helal, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyden, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chappie, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fisher, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hajos, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harms, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphrey, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandit, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kleiman, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kormos, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maklad, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDowell, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGinnis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connor, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Donnell, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogden, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piotrowski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seymour, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueno, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vansell, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verhoest, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, E. X.</span></span> <span> </span><span class="NLM_article-title">Identification of a potent, highly selective, and brain penetrant phosphodiesterase 2A inhibitor clinical candidate</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">1001</span>– <span class="NLM_lpage">1018</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01466</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01466" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVWgsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=1001-1018&author=C.+J.+Helalauthor=E.+Arnoldauthor=T.+Boydenauthor=C.+Changauthor=T.+A.+Chappieauthor=E.+Fisherauthor=M.+Hajosauthor=J.+F.+Harmsauthor=W.+E.+Hoffmanauthor=J.+M.+Humphreyauthor=J.+Panditauthor=Z.+Kangauthor=R.+J.+Kleimanauthor=B.+L.+Kormosauthor=C.-W.+Leeauthor=J.+Luauthor=N.+Makladauthor=L.+McDowellauthor=D.+McGinnisauthor=R.+E.+O%E2%80%99Connorauthor=C.+J.+O%E2%80%99Donnellauthor=A.+Ogdenauthor=M.+Piotrowskiauthor=C.+J.+Schmidtauthor=P.+A.+Seymourauthor=H.+Uenoauthor=N.+Vansellauthor=P.+R.+Verhoestauthor=E.+X.+Yang&title=Identification+of+a+potent%2C+highly+selective%2C+and+brain+penetrant+phosphodiesterase+2A+inhibitor+clinical+candidate&doi=10.1021%2Facs.jmedchem.7b01466"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a Potent, Highly Selective, and Brain Penetrant Phosphodiesterase 2A Inhibitor Clinical Candidate</span></div><div class="casAuthors">Helal, Christopher J.; Arnold, Eric; Boyden, Tracey; Chang, Cheng; Chappie, Thomas A.; Fisher, Ethan; Hajos, Mihaly; Harms, John F.; Hoffman, William E.; Humphrey, John M.; Pandit, Jayvardhan; Kang, Zhijun; Kleiman, Robin J.; Kormos, Bethany L.; Lee, Che-Wah; Lu, Jiemin; Maklad, Noha; McDowell, Laura; McGinnis, Dina; OConnor, Rebecca E.; ODonnell, Christopher J.; Ogden, Adam; Piotrowski, Mary; Schmidt, Christopher J.; Seymour, Patricia A.; Ueno, Hirokazu; Vansell, Nichole; Verhoest, Patrick R.; Yang, Edward X.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1001-1018</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Computational modeling was used to direct the synthesis of analogs of previously reported phosphodiesterase 2A (PDE2A) inhibitor (I) with an imidazotriazine core to yield compds. of significantly enhanced potency.  The analog PF-05180999 (30) was subsequently identified as a preclin. candidate targeting cognitive impairment assocd. with schizophrenia.  Compd. 30 demonstrated potent binding to PDE2A in brain tissue, dose responsive mouse brain cGMP increases, and reversal of N-methyl-D-aspartate (NMDA) antagonist-induced (MK-801, ketamine) effects in electrophysiol. and working memory models in rats.  Preclin. pharmacokinetics revealed unbound brain/unbound plasma levels approaching unity and good oral bioavailability resulting in an av. concn. at steady state (Cav,ss) predicted human dose of 30 mg once daily (q.d.).  Modeling of a modified release formulation suggested that 25 mg twice daily (b.i.d.) could maintain plasma levels of 30 at or above targeted efficacious plasma levels for 24 h, which became part of the human clin. plan.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfbdIAg1Vd27Vg90H21EOLACvtfcHk0ljNSGYgNo9TQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVWgsg%253D%253D&md5=5424e50970acfe7acb0d29d1b750c380</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01466&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01466%26sid%3Dliteratum%253Aachs%26aulast%3DHelal%26aufirst%3DC.%2BJ.%26aulast%3DArnold%26aufirst%3DE.%26aulast%3DBoyden%26aufirst%3DT.%26aulast%3DChang%26aufirst%3DC.%26aulast%3DChappie%26aufirst%3DT.%2BA.%26aulast%3DFisher%26aufirst%3DE.%26aulast%3DHajos%26aufirst%3DM.%26aulast%3DHarms%26aufirst%3DJ.%2BF.%26aulast%3DHoffman%26aufirst%3DW.%2BE.%26aulast%3DHumphrey%26aufirst%3DJ.%2BM.%26aulast%3DPandit%26aufirst%3DJ.%26aulast%3DKang%26aufirst%3DZ.%26aulast%3DKleiman%26aufirst%3DR.%2BJ.%26aulast%3DKormos%26aufirst%3DB.%2BL.%26aulast%3DLee%26aufirst%3DC.-W.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DMaklad%26aufirst%3DN.%26aulast%3DMcDowell%26aufirst%3DL.%26aulast%3DMcGinnis%26aufirst%3DD.%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DR.%2BE.%26aulast%3DO%25E2%2580%2599Donnell%26aufirst%3DC.%2BJ.%26aulast%3DOgden%26aufirst%3DA.%26aulast%3DPiotrowski%26aufirst%3DM.%26aulast%3DSchmidt%26aufirst%3DC.%2BJ.%26aulast%3DSeymour%26aufirst%3DP.%2BA.%26aulast%3DUeno%26aufirst%3DH.%26aulast%3DVansell%26aufirst%3DN.%26aulast%3DVerhoest%26aufirst%3DP.%2BR.%26aulast%3DYang%26aufirst%3DE.%2BX.%26atitle%3DIdentification%2520of%2520a%2520potent%252C%2520highly%2520selective%252C%2520and%2520brain%2520penetrant%2520phosphodiesterase%25202A%2520inhibitor%2520clinical%2520candidate%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D1001%26epage%3D1018%26doi%3D10.1021%2Facs.jmedchem.7b01466" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buijnsters, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Angelis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langlois, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rombouts, F. J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanderson, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tresadern, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ritchie, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trabanco, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vanhoof, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roosbroeck, Y. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrés, J.-I.</span></span> <span> </span><span class="NLM_article-title">Structure-based design of a potent, selective, and brain penetrating PDE2A inhibitor with demonstrated target engagement</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">1049</span>– <span class="NLM_lpage">1053</span>, <span class="refDoi"> DOI: 10.1021/ml500262u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml500262u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFyqs7%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=1049-1053&author=P.+Buijnstersauthor=M.+De+Angelisauthor=X.+Langloisauthor=F.+J.+R.+Romboutsauthor=W.+Sandersonauthor=G.+Tresadernauthor=A.+Ritchieauthor=A.+A.+Trabancoauthor=G.+Vanhoofauthor=Y.+V.+Roosbroeckauthor=J.-I.+Andr%C3%A9s&title=Structure-based+design+of+a+potent%2C+selective%2C+and+brain+penetrating+PDE2A+inhibitor+with+demonstrated+target+engagement&doi=10.1021%2Fml500262u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of a Potent, Selective, and Brain Penetrating PDE2 Inhibitor with Demonstrated Target Engagement</span></div><div class="casAuthors">Buijnsters, Peter; De Angelis, Meri; Langlois, Xavier; Rombouts, Frederik J. R.; Sanderson, Wendy; Tresadern, Gary; Ritchie, Alison; Trabanco, Andres A.; Van Hoof, Greet; Van Roosbroeck, Yves; Andres, Jose-Ignacio</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1049-1053</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Structure-guided design led to the identification of the novel, potent, and selective phosphodiesterase 2 (PDE2) inhibitor 12.  Compd. 12 demonstrated a >210-fold selectivity vs. PDE10 and PDE11 and was inactive against all other PDE family members up to 10 μM.  In vivo evaluation of 12 provided evidence that it is able to engage the target and to increase cGMP levels in relevant brain regions.  Hence, 12 is a valuable tool compd. for the better understanding of the role of PDE2 in cognitive impairment and other central nervous system related disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonEHLDgf3R67Vg90H21EOLACvtfcHk0lh8NNcrSZSU1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFyqs7%252FL&md5=e7d10d94929444876909ebf89dbde404</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fml500262u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml500262u%26sid%3Dliteratum%253Aachs%26aulast%3DBuijnsters%26aufirst%3DP.%26aulast%3DDe%2BAngelis%26aufirst%3DM.%26aulast%3DLanglois%26aufirst%3DX.%26aulast%3DRombouts%26aufirst%3DF.%2BJ.%2BR.%26aulast%3DSanderson%26aufirst%3DW.%26aulast%3DTresadern%26aufirst%3DG.%26aulast%3DRitchie%26aufirst%3DA.%26aulast%3DTrabanco%26aufirst%3DA.%2BA.%26aulast%3DVanhoof%26aufirst%3DG.%26aulast%3DRoosbroeck%26aufirst%3DY.%2BV.%26aulast%3DAndr%25C3%25A9s%26aufirst%3DJ.-I.%26atitle%3DStructure-based%2520design%2520of%2520a%2520potent%252C%2520selective%252C%2520and%2520brain%2520penetrating%2520PDE2A%2520inhibitor%2520with%2520demonstrated%2520target%2520engagement%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D5%26spage%3D1049%26epage%3D1053%26doi%3D10.1021%2Fml500262u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rombouts, F. J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tresadern, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buijnsters, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langlois, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tovar, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinbrecher, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vanhoof, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Somers, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrés, J.-I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trabanco, A. A.-I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trabanco, A. A.</span></span> <span> </span><span class="NLM_article-title">Pyrido[4,3-e][1,2,4]triazolo[4,3-a]pyrazines as selective, brain penetrant phosphodiesterase 2 (PDE2A) inhibitors</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">282</span>– <span class="NLM_lpage">286</span>, <span class="refDoi"> DOI: 10.1021/ml500463t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml500463t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC2MXosVOltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=282-286&author=F.+J.+R.+Romboutsauthor=G.+Tresadernauthor=P.+Buijnstersauthor=X.+Langloisauthor=F.+Tovarauthor=T.+B.+Steinbrecherauthor=G.+Vanhoofauthor=M.+Somersauthor=J.-I.+Andr%C3%A9sauthor=A.+A.-I.+Trabancoauthor=A.+A.+Trabanco&title=Pyrido%5B4%2C3-e%5D%5B1%2C2%2C4%5Dtriazolo%5B4%2C3-a%5Dpyrazines+as+selective%2C+brain+penetrant+phosphodiesterase+2+%28PDE2A%29+inhibitors&doi=10.1021%2Fml500463t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Pyrido[4,3-e][1,2,4]triazolo[4,3-a]pyrazines as Selective, Brain Penetrant Phosphodiesterase 2 (PDE2) Inhibitors</span></div><div class="casAuthors">Rombouts, Frederik J. R.; Tresadern, Gary; Buijnsters, Peter; Langlois, Xavier; Tovar, Fulgencio; Steinbrecher, Thomas B.; Vanhoof, Greet; Somers, Marijke; Andres, Jose-Ignacio; Trabanco, Andres A.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">282-286</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel series of pyrido[4,3-e][1,2,4]triazolo[4,3-a]pyrazines is reported as potent PDE2/PDE10 inhibitors with drug-like properties.  Selectivity for PDE2 was obtained by introducing a linear, lipophilic moiety on the meta-position of the Ph ring pending from the triazole.  The SAR and protein flexibility were explored with free energy perturbation calcns.  Rat pharmacokinetic data and in vivo receptor occupancy data are given for two representative compds. 6 and 12.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbUtc8164aobVg90H21EOLACvtfcHk0lh8NNcrSZSU1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXosVOltA%253D%253D&md5=d92a3c22cee3c286c2608d5cf719b14c</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fml500463t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml500463t%26sid%3Dliteratum%253Aachs%26aulast%3DRombouts%26aufirst%3DF.%2BJ.%2BR.%26aulast%3DTresadern%26aufirst%3DG.%26aulast%3DBuijnsters%26aufirst%3DP.%26aulast%3DLanglois%26aufirst%3DX.%26aulast%3DTovar%26aufirst%3DF.%26aulast%3DSteinbrecher%26aufirst%3DT.%2BB.%26aulast%3DVanhoof%26aufirst%3DG.%26aulast%3DSomers%26aufirst%3DM.%26aulast%3DAndr%25C3%25A9s%26aufirst%3DJ.-I.%26aulast%3DTrabanco%26aufirst%3DA.%2BA.-I.%26aulast%3DTrabanco%26aufirst%3DA.%2BA.%26atitle%3DPyrido%255B4%252C3-e%255D%255B1%252C2%252C4%255Dtriazolo%255B4%252C3-a%255Dpyrazines%2520as%2520selective%252C%2520brain%2520penetrant%2520phosphodiesterase%25202%2520%2528PDE2A%2529%2520inhibitors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D282%26epage%3D286%26doi%3D10.1021%2Fml500463t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span> <span> </span><span class="NLM_article-title">Efficient drug lead discovery and optimization</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">724</span>– <span class="NLM_lpage">733</span>, <span class="refDoi"> DOI: 10.1021/ar800236t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar800236t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjsFentr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2009&pages=724-733&author=W.+L.+Jorgensen&title=Efficient+drug+lead+discovery+and+optimization&doi=10.1021%2Far800236t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Efficient Drug Lead Discovery and Optimization</span></div><div class="casAuthors">Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">724-733</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  During the 1980s, advances in the abilities to perform computer simulations of chem. and biomol. systems and to calc. free energy changes led to the expectation that such methodol. would soon show great utility for guiding mol. design.  Important potential applications included design of selective receptors, catalysts, and regulators of biol. function including enzyme inhibitors.  This time also saw the rise of high-throughput screening and combinatorial chem. along with complementary computational methods for de novo design and virtual screening including docking.  These technologies appeared poised to deliver diverse lead compds. for any biol. target.  As with many technol. advances, realization of the expectations required significant addnl. effort and time.  However, as summarized here, striking success has now been achieved for computer-aided drug lead generation and optimization.  De novo design using both mol. growing and docking are illustrated for lead generation, and lead optimization features free energy perturbation calcns. in conjunction with Monte Carlo statistical mechanics simulations for protein-inhibitor complexes in aq. soln.  The specific applications are to the discovery of non-nucleoside inhibitors of HIV reverse transcriptase (HIV-RT) and inhibitors of the binding of the proinflammatory cytokine MIF to its receptor CD74.  A std. protocol is presented that includes scans for possible addns. of small substituents to a mol. core, interchange of heterocycles, and focused optimization of substituents at one site.  Initial leads with activities at low-micromolar concns. have been advanced rapidly to low-nanomolar inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrn0YpWuo1jCLVg90H21EOLACvtfcHk0lh8NNcrSZSU1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjsFentr4%253D&md5=9641587769701c6541b38e18fa05538a</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Far800236t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far800236t%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DEfficient%2520drug%2520lead%2520discovery%2520and%2520optimization%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2009%26volume%3D42%26spage%3D724%26epage%3D733%26doi%3D10.1021%2Far800236t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harder, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berne, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span> <span> </span><span class="NLM_article-title">Advancing drug discovery through enhanced free energy calculations</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">1625</span>– <span class="NLM_lpage">1632</span>, <span class="refDoi"> DOI: 10.1021/acs.accounts.7b00083</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.accounts.7b00083" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFSiur3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2017&pages=1625-1632&author=R.+Abelauthor=L.+Wangauthor=E.+D.+Harderauthor=B.+J.+Berneauthor=R.+A.+Friesner&title=Advancing+drug+discovery+through+enhanced+free+energy+calculations&doi=10.1021%2Facs.accounts.7b00083"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Advancing Drug Discovery through Enhanced Free Energy Calculations</span></div><div class="casAuthors">Abel, Robert; Wang, Lingle; Harder, Edward D.; Berne, B. J.; Friesner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1625-1632</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  A principal goal of drug discovery project is to design mols. that can tightly and selectively bind to the target protein receptor.  Accurate prediction of protein-ligand binding free energies is therefore of central importance in computational chem. and computer aided drug design.  Multiple recent improvements in computing power, classical force field accuracy, enhanced sampling methods, and simulation setup, have enabled accurate and reliable calcns. of protein-ligands binding free energies, and position free energy calcns. to play a guiding role in small mol. drug discovery.  In this accounts, the authors outline the relevant methodol. advances, including the REST2 (Replica Exchange with Solute Temperting) enhanced sampling, the incorporation of REST2 sampling with conventional FEP (Free Energy Perturbation) through FEP/REST, the OPLS3 force fields, and the advanced simulation set up that constitute the authors' FEP+ approach, followed by the presentation of extensive comparisons with expt., demonstrating sufficient accuracy in potency prediction (better than 1 kcal/mol) to substantially impact lead optimization campaigns.  The limitations of the current FEP+ implementation and best practices in drug discovery applications are also discussed followed by the future methodol. development plans to address those limitations.  The authors then report results from a recent drug discovery project, in which several thousand FEP+ calcns. were successfully deployed to simultaneously optimize potency, selectivity, and soly., illustrating the power of the approach to solve challenging drug design problems.  The capabilities of free energy calcns. to accurately predict potency and selectivity have led to the advance of ongoing drug discovery projects, in challenging situations where alternative approaches would have great difficulties.  The ability to effectively carry out projects evaluating tens of thousands, or hundreds of thousands, of proposed drug candidates, is potentially transformative in enabling hard to drug targets to be attacked, and in facilitating the development of superior compds., in various dimensions, for a wide range of targets.  More effective integration of FEP+ calcns. into the drug discovery process will ensure that the results are deployed in an optimal fashion for yielding the best possible compds. entering the clinic; this is where the greatest payoff is in the exploitation of computer driven design capabilities.  A key conclusion from the work described is the surprisingly robust and accurate results that are attainable within the conventional classical simulation, fixed charge paradigm.  No doubt there are individual cases that would benefit from a more sophisticated energy model or dynamical treatment, and properties other than protein-ligand binding energies may be more sensitive to these approxns.  The authors conclude that an inflection point in the ability of MD simulations to impact drug discovery has now been attained, due to the confluence of hardware and software development along with the formulation of "good enough" theor. methods and models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_VeCPmDRlf7Vg90H21EOLACvtfcHk0lgZ2qfTv6oquA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFSiur3K&md5=46d07ee6f6f129a126f0fde40a62ffe2</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facs.accounts.7b00083&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.accounts.7b00083%26sid%3Dliteratum%253Aachs%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DHarder%26aufirst%3DE.%2BD.%26aulast%3DBerne%26aufirst%3DB.%2BJ.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DAdvancing%2520drug%2520discovery%2520through%2520enhanced%2520free%2520energy%2520calculations%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2017%26volume%3D50%26spage%3D1625%26epage%3D1632%26doi%3D10.1021%2Facs.accounts.7b00083" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cournia, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, W.</span></span> <span> </span><span class="NLM_article-title">Relative binding free energy calculations in drug discovery: Recent advances and practical considerations</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">2911</span>– <span class="NLM_lpage">2937</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.7b00564</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.7b00564" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFOhu7fM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2017&pages=2911-2937&author=Z.+Courniaauthor=B.+Allenauthor=W.+Sherman&title=Relative+binding+free+energy+calculations+in+drug+discovery%3A+Recent+advances+and+practical+considerations&doi=10.1021%2Facs.jcim.7b00564"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Relative Binding Free Energy Calculations in Drug Discovery: Recent Advances and Practical Considerations</span></div><div class="casAuthors">Cournia, Zoe; Allen, Bryce; Sherman, Woody</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2911-2937</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Accurate in silico prediction of protein-ligand binding affinities has been a primary objective of structure-based drug design for decades due to the putative value it would bring to the drug discovery process.  However, computational methods have historically failed to deliver value in real-world drug discovery applications due to a variety of scientific, tech., and practical challenges.  Recently, a family of approaches commonly referred to as relative binding free energy (RBFE) calcns., which rely on physics-based mol. simulations and statistical mechanics, have shown promise in reliably generating accurate predictions in the context of drug discovery projects.  This advance arises from accumulating developments in the underlying scientific methods (decades of research on force fields and sampling algorithms) coupled with vast increases in computational resources (graphics processing units and cloud infrastructures).  Mounting evidence from retrospective validation studies, blind challenge predictions, and prospective applications suggests that RBFE simulations can now predict the affinity differences for congeneric ligands with sufficient accuracy and throughput to deliver considerable value in hit-to-lead and lead optimization efforts.  Here, the authors present an overview of current RBFE implementations, highlighting recent advances and remaining challenges, along with examples that emphasize practical considerations for obtaining reliable RBFE results.  The authors focus specifically on relative binding free energies because the calcns. are less computationally intensive than abs. binding free energy (ABFE) calcns. and map directly onto the hit-to-lead and lead optimization processes, where the prediction of relative binding energies between a ref. mol. and new ideas (virtual mols.) can be used to prioritize mols. for synthesis.  The authors describe the crit. aspects of running RBFE calcns., from both theor. and applied perspectives, using a combination of retrospective literature examples and prospective studies from drug discovery projects.  This work is intended to provide a contemporary overview of the scientific, tech., and practical issues assocd. with running relative binding free energy simulations, with a focus on real-world drug discovery applications.  The authors offer guidelines for improving the accuracy of RBFE simulations, esp. for challenging cases, and emphasize unresolved issues that could be improved by further research in the field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokxZRvplrWFbVg90H21EOLACvtfcHk0lgZ2qfTv6oquA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFOhu7fM&md5=ffcc9f970ab660eced8d3343ccefeec6</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.7b00564&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.7b00564%26sid%3Dliteratum%253Aachs%26aulast%3DCournia%26aufirst%3DZ.%26aulast%3DAllen%26aufirst%3DB.%26aulast%3DSherman%26aufirst%3DW.%26atitle%3DRelative%2520binding%2520free%2520energy%2520calculations%2520in%2520drug%2520discovery%253A%2520Recent%2520advances%2520and%2520practical%2520considerations%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2017%26volume%3D57%26spage%3D2911%26epage%3D2937%26doi%3D10.1021%2Facs.jcim.7b00564" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ciordia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pérez-Benito, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delgado, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trabanco, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tresadern, G.</span></span> <span> </span><span class="NLM_article-title">Application of free energy perturbation for the design of BACE1 inhibitors</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">1856</span>– <span class="NLM_lpage">1871</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.6b00220</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.6b00220" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht12mu7%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2016&pages=1856-1871&author=M.+Ciordiaauthor=L.+P%C3%A9rez-Benitoauthor=F.+Delgadoauthor=A.+A.+Trabancoauthor=G.+Tresadern&title=Application+of+free+energy+perturbation+for+the+design+of+BACE1+inhibitors&doi=10.1021%2Facs.jcim.6b00220"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Application of Free Energy Perturbation for the Design of BACE1 Inhibitors</span></div><div class="casAuthors">Ciordia, Myriam; Perez-Benito, Laura; Delgado, Francisca; Trabanco, Andres A.; Tresadern, Gary</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1856-1871</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Novel spiroaminodihydropyrroles probing for optimized interactions at the P3 pocket of β-secretase 1 (BACE1) were designed with the use of free energy perturbation (FEP) calcns.  The resulting mols. showed pIC50 potencies in enzymic BACE1 inhibition assays ranging from approx. 5 to 7.  Good correlation was obsd. between the predicted activity from the FEP calcns. and exptl. activity.  Simulations run with a default 5 ns approach delivered a mean unsigned error (MUE) between prediction and expt. of 0.58 and 0.91 kcal/mol for retrospective and prospective applications, resp.  With longer simulations of 10 and 20 ns, the MUE was in both cases 0.57 kcal/mol for the retrospective application, and 0.69 and 0.59 kcal/mol for the prospective application.  Other considerations that impact the quality of the calcns. are discussed.  This work provides an example of the value of FEP as a computational tool for drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqpi-OiII-FrVg90H21EOLACvtfcHk0lgZ2qfTv6oquA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht12mu7%252FK&md5=b7577a89492867fc334cb1ec937a4eb6</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.6b00220&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.6b00220%26sid%3Dliteratum%253Aachs%26aulast%3DCiordia%26aufirst%3DM.%26aulast%3DP%25C3%25A9rez-Benito%26aufirst%3DL.%26aulast%3DDelgado%26aufirst%3DF.%26aulast%3DTrabanco%26aufirst%3DA.%2BA.%26aulast%3DTresadern%26aufirst%3DG.%26atitle%3DApplication%2520of%2520free%2520energy%2520perturbation%2520for%2520the%2520design%2520of%2520BACE1%2520inhibitors%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2016%26volume%3D56%26spage%3D1856%26epage%3D1871%26doi%3D10.1021%2Facs.jcim.6b00220" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keränen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pérez-Benito, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciordia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delgado, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinbrecher, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oehlrich, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Vlijmen, H. W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trabanco, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tresadern, G.</span></span> <span> </span><span class="NLM_article-title">Acylguanidine beta secretase 1 inhibitors: A combined experimental and free energy perturbation study</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">1439</span>– <span class="NLM_lpage">1453</span>, <span class="refDoi"> DOI: 10.1021/acs.jctc.6b01141</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.6b01141" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1ehsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=1439-1453&author=H.+Ker%C3%A4nenauthor=L.+P%C3%A9rez-Benitoauthor=M.+Ciordiaauthor=F.+Delgadoauthor=T.+B.+Steinbrecherauthor=D.+Oehlrichauthor=H.+W.+T.+van+Vlijmenauthor=A.+A.+Trabancoauthor=G.+Tresadern&title=Acylguanidine+beta+secretase+1+inhibitors%3A+A+combined+experimental+and+free+energy+perturbation+study&doi=10.1021%2Facs.jctc.6b01141"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Acylguanidine Beta Secretase 1 Inhibitors: A Combined Experimental and Free Energy Perturbation Study</span></div><div class="casAuthors">Keranen, Henrik; Perez-Benito, Laura; Ciordia, Myriam; Delgado, Francisca; Steinbrecher, Thomas B.; Oehlrich, Daniel; van Vlijmen, Herman W. T.; Trabanco, Andres A.; Tresadern, Gary</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1439-1453</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of acylguanidine beta secretase 1 (BACE1) inhibitors with modified scaffold and P3 pocket substituent was synthesized and studied with Free Energy Perturbation (FEP) calcns.  The resulting mols. showed potencies in enzymic BACE1 inhibition assays up to 1 nM.  The correlation between the predicted activity from the FEP calcns. and the exptl. activity was good for the P3 pocket substituents.  The av. mean unsigned error (MUE) between prediction and expt. was 0.68±0.17 kcal/mol for the default 5 ns lambda window simulation time improving to 0.35±0.13 kcal/mol for 40 ns.  FEP calcns. for the P2' pocket substituents on the same acylguanidine scaffold also showed good agreement with expt. and the results remained stable with repeated simulations and increased simulation time.  It proved more difficult to use FEP calcns. to study the scaffold modification from increasing 5 to 6 and 7 membered-rings.  Although prediction and expt. were in agreement for short 2 ns simulations, as the simulation time increased the results diverged.  This was improved by the use of a newly developed 'Core Hopping FEP+' approach, which also showed improved stability in repeat calcns.  The origins of these differences along with the value of repeat and longer simulation times are discussed.  This work provides a further example of the use of FEP as a computational tool for mol. design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmHZjygsk_PbVg90H21EOLACvtfcHk0lgD-9FC0T-01A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1ehsrg%253D&md5=987ab8fc475670d0ba19aa086519a4ba</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.6b01141&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.6b01141%26sid%3Dliteratum%253Aachs%26aulast%3DKer%25C3%25A4nen%26aufirst%3DH.%26aulast%3DP%25C3%25A9rez-Benito%26aufirst%3DL.%26aulast%3DCiordia%26aufirst%3DM.%26aulast%3DDelgado%26aufirst%3DF.%26aulast%3DSteinbrecher%26aufirst%3DT.%2BB.%26aulast%3DOehlrich%26aufirst%3DD.%26aulast%3Dvan%2BVlijmen%26aufirst%3DH.%2BW.%2BT.%26aulast%3DTrabanco%26aufirst%3DA.%2BA.%26aulast%3DTresadern%26aufirst%3DG.%26atitle%3DAcylguanidine%2520beta%2520secretase%25201%2520inhibitors%253A%2520A%2520combined%2520experimental%2520and%2520free%2520energy%2520perturbation%2520study%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2017%26volume%3D13%26spage%3D1439%26epage%3D1453%26doi%3D10.1021%2Facs.jctc.6b01141" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deflorian, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pérez-Benito, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lenselink, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Congreve, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Vlijmen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mason, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Graaf, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tresadern, G.</span></span> <span> </span><span class="NLM_article-title">Accurate prediction of GPCR ligand binding affinity with free energy perturbation</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">4311</span>– <span class="NLM_lpage">4325</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.0c00449</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.0c00449" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2020&pages=4311-4325&author=F.+Deflorianauthor=L.+P%C3%A9rez-Benitoauthor=E.+B.+Lenselinkauthor=M.+Congreveauthor=H.+Van+Vlijmenauthor=J.+S.+Masonauthor=C.+de+Graafauthor=G.+Tresadern&title=Accurate+prediction+of+GPCR+ligand+binding+affinity+with+free+energy+perturbation&doi=10.1021%2Facs.jcim.0c00449"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.0c00449&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.0c00449%26sid%3Dliteratum%253Aachs%26aulast%3DDeflorian%26aufirst%3DF.%26aulast%3DP%25C3%25A9rez-Benito%26aufirst%3DL.%26aulast%3DLenselink%26aufirst%3DE.%2BB.%26aulast%3DCongreve%26aufirst%3DM.%26aulast%3DVan%2BVlijmen%26aufirst%3DH.%26aulast%3DMason%26aufirst%3DJ.%2BS.%26aulast%3Dde%2BGraaf%26aufirst%3DC.%26aulast%3DTresadern%26aufirst%3DG.%26atitle%3DAccurate%2520prediction%2520of%2520GPCR%2520ligand%2520binding%2520affinity%2520with%2520free%2520energy%2520perturbation%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2020%26volume%3D60%26spage%3D4311%26epage%3D4325%26doi%3D10.1021%2Facs.jcim.0c00449" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pérez-Benito, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keränen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Vlijmen, H. W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tresadern, G.</span></span> <span> </span><span class="NLM_article-title">Predicting binding free energies of PDE2A inhibitors. The difficulties of protein conformation</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">4883</span>, <span class="refDoi"> DOI: 10.1038/s41598-018-23039-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.1038%2Fs41598-018-23039-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=29559702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A280%3ADC%252BC1MnjtFKlsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=4883&author=L.+P%C3%A9rez-Benitoauthor=H.+Ker%C3%A4nenauthor=H.+W.+T.+van+Vlijmenauthor=G.+Tresadern&title=Predicting+binding+free+energies+of+PDE2A+inhibitors.+The+difficulties+of+protein+conformation&doi=10.1038%2Fs41598-018-23039-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting Binding Free Energies of PDE2 Inhibitors. The Difficulties of Protein Conformation</span></div><div class="casAuthors">Perez-Benito Laura; Keranen Henrik; van Vlijmen Herman; Tresadern Gary; Keranen Henrik</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">4883</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">A congeneric series of 21 phosphodiesterase 2 (PDE2) inhibitors are reported.  Crystal structures show how the molecules can occupy a 'top-pocket' of the active site.  Molecules with small substituents do not enter the pocket, a critical leucine (Leu770) is closed and water molecules are present.  Large substituents enter the pocket, opening the Leu770 conformation and displacing the waters.  We also report an X-ray structure revealing a new conformation of the PDE2 active site domain.  The relative binding affinities of these compounds were studied with free energy perturbation (FEP) methods and it represents an attractive real-world test case.  In general, the calculations could predict the energy of small-to-small, or large-to-large molecule perturbations.  However, accurately capturing the transition from small-to-large proved challenging.  Only when using alternative protein conformations did results improve.  The new X-ray structure, along with a modelled dimer, conferred stability to the catalytic domain during the FEP molecular dynamics (MD) simulations, increasing the convergence and thereby improving the prediction of ΔΔG of binding for some small-to-large transitions.  In summary, we found the most significant improvement in results when using different protein structures, and this data set is useful for future free energy validation studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTMdLif4rzX6u_Q9URG6Dx8fW6udTcc2eZSlHXM6bv9_Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MnjtFKlsA%253D%253D&md5=44c26c5bd2fe3e35bba344abb8262eeb</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Fs41598-018-23039-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-018-23039-5%26sid%3Dliteratum%253Aachs%26aulast%3DP%25C3%25A9rez-Benito%26aufirst%3DL.%26aulast%3DKer%25C3%25A4nen%26aufirst%3DH.%26aulast%3Dvan%2BVlijmen%26aufirst%3DH.%2BW.%2BT.%26aulast%3DTresadern%26aufirst%3DG.%26atitle%3DPredicting%2520binding%2520free%2520energies%2520of%2520PDE2A%2520inhibitors.%2520The%2520difficulties%2520of%2520protein%2520conformation%26jtitle%3DSci.%2520Rep.%26date%3D2018%26volume%3D8%26spage%3D4883%26doi%3D10.1038%2Fs41598-018-23039-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pérez-Benito, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casajuana-Martin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiménez-Rosés, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Vlijmen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tresadern, G.</span></span> <span> </span><span class="NLM_article-title">Predicting activity cliffs with free-energy perturbation</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">1884</span>– <span class="NLM_lpage">1895</span>, <span class="refDoi"> DOI: 10.1021/acs.jctc.8b01290</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.8b01290" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjtFKktr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2019&pages=1884-1895&author=L.+P%C3%A9rez-Benitoauthor=N.+Casajuana-Martinauthor=M.+Jim%C3%A9nez-Ros%C3%A9sauthor=H.+van+Vlijmenauthor=G.+Tresadern&title=Predicting+activity+cliffs+with+free-energy+perturbation&doi=10.1021%2Facs.jctc.8b01290"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting Activity Cliffs with Free-Energy Perturbation</span></div><div class="casAuthors">Perez-Benito, Laura; Casajuana-Martin, Nil; Jimenez-Roses, Mireia; van Vlijmen, Herman; Tresadern, Gary</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1884-1895</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Activity cliffs (ACs) are an important type of structure-activity relationship in medicinal chem. where small structural changes result in unexpectedly large differences in biol. activity.  Being able to predict these changes would have a profound impact on lead optimization of drug candidates.  Free-energy perturbation is an ideal tool for predicting relative binding energy differences for small structural modifications, but its performance for ACs is unknown.  Here, we show that FEP can on av. predict ACs to within 1.39 kcal/mol of expt. (∼1 log unit of activity).  We performed FEP calcns. with two different software methods: Schr.ovrddot.odinger-Desmond FEP+ and GROMACS implementations.  There was qual. agreement in the results from the two methods, and quant. the error for one data set was identical, 1.43 kcal/mol, but FEP+ performed better in the second, with errors of 1.17 vs. 1.90 kcal/mol.  The results have far-reaching implications, suggesting well-implemented FEP calcns. can have a major impact on computational drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-1Z4ZWIjgcbVg90H21EOLACvtfcHk0lgD-9FC0T-01A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjtFKktr8%253D&md5=59a4c8d38efece18891da4266f6bc3d5</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.8b01290&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.8b01290%26sid%3Dliteratum%253Aachs%26aulast%3DP%25C3%25A9rez-Benito%26aufirst%3DL.%26aulast%3DCasajuana-Martin%26aufirst%3DN.%26aulast%3DJim%25C3%25A9nez-Ros%25C3%25A9s%26aufirst%3DM.%26aulast%3Dvan%2BVlijmen%26aufirst%3DH.%26aulast%3DTresadern%26aufirst%3DG.%26atitle%3DPredicting%2520activity%2520cliffs%2520with%2520free-energy%2520perturbation%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2019%26volume%3D15%26spage%3D1884%26epage%3D1895%26doi%3D10.1021%2Facs.jctc.8b01290" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gapsys, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pérez-Benito, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aldeghi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seeliger, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Vlijmen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tresadern, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Groot, B. L.</span></span> <span> </span><span class="NLM_article-title">Large scale relative protein ligand binding affinities using non-equilibrium alchemy</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1140</span>– <span class="NLM_lpage">1152</span>, <span class="refDoi"> DOI: 10.1039/c9sc03754c</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.1039%2Fc9sc03754c" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1OrsLbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=1140-1152&author=V.+Gapsysauthor=L.+P%C3%A9rez-Benitoauthor=M.+Aldeghiauthor=D.+Seeligerauthor=H.+van+Vlijmenauthor=G.+Tresadernauthor=B.+L.+de+Groot&title=Large+scale+relative+protein+ligand+binding+affinities+using+non-equilibrium+alchemy&doi=10.1039%2Fc9sc03754c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Large scale relative protein ligand binding affinities using non-equilibrium alchemy</span></div><div class="casAuthors">Gapsys, Vytautas; Perez-Benito, Laura; Aldeghi, Matteo; Seeliger, Daniel; van Vlijmen, Herman; Tresadern, Gary; de Groot, Bert L.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1140-1152</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Ligand binding affinity calcns. based on mol. dynamics (MD) simulations and non-phys. (alchem.) thermodn. cycles have shown great promise for structure-based drug design.  However, their broad uptake and impact is held back by the notoriously complex setup of the calcns.  Only a few tools other than the free energy perturbation approach by Schrodinger Inc. (referred to as FEP+) currently enable end-to-end application.  Here, we present for the first time an approach based on the open-source software pmx that allows to easily set up and run alchem. calcns. for diverse sets of small mols. using the GROMACS MD engine.  The method relies on theor. rigorous non-equil. thermodn. integration (TI) foundations, and its flexibility allows calcns. with multiple force fields.  In this study, results from the Amber and Charmm force fields were combined to yield a consensus outcome performing on par with the com. FEP+ approach.  A large dataset of 482 perturbations from 13 different protein-ligand datasets led to an av. unsigned error (AUE) of 3.64 ± 0.14 kJ mol-1, equiv. to Schrodinger's FEP+ AUE of 3.66 ± 0.14 kJ mol-1.  For the first time, a setup is presented for overall high precision and high accuracy relative protein-ligand alchem. free energy calcns. based on open-source software.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqM_Ma3c_2AGrVg90H21EOLACvtfcHk0lhTi_O1JqhrYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1OrsLbJ&md5=56ae345e5332032a3760776215fc53df</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1039%2Fc9sc03754c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc9sc03754c%26sid%3Dliteratum%253Aachs%26aulast%3DGapsys%26aufirst%3DV.%26aulast%3DP%25C3%25A9rez-Benito%26aufirst%3DL.%26aulast%3DAldeghi%26aufirst%3DM.%26aulast%3DSeeliger%26aufirst%3DD.%26aulast%3Dvan%2BVlijmen%26aufirst%3DH.%26aulast%3DTresadern%26aufirst%3DG.%26aulast%3Dde%2BGroot%26aufirst%3DB.%2BL.%26atitle%3DLarge%2520scale%2520relative%2520protein%2520ligand%2520binding%2520affinities%2520using%2520non-equilibrium%2520alchemy%26jtitle%3DChem.%2520Sci.%26date%3D2020%26volume%3D11%26spage%3D1140%26epage%3D1152%26doi%3D10.1039%2Fc9sc03754c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tresadern, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pérez-Benito, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vlijmen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coveney, P. V.</span></span> <span> </span><span class="NLM_article-title">Accuracy and precision of alchemical relative free energy predictions with and without replica-exchange</span>. <i>Adv. Theory Simul.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">1900195</span>, <span class="refDoi"> DOI: 10.1002/adts.201900195</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.1002%2Fadts.201900195" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2019&pages=1900195&author=S.+Wanauthor=G.+Tresadernauthor=L.+P%C3%A9rez-Benitoauthor=H.+Vlijmenauthor=P.+V.+Coveney&title=Accuracy+and+precision+of+alchemical+relative+free+energy+predictions+with+and+without+replica-exchange&doi=10.1002%2Fadts.201900195"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1002%2Fadts.201900195&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fadts.201900195%26sid%3Dliteratum%253Aachs%26aulast%3DWan%26aufirst%3DS.%26aulast%3DTresadern%26aufirst%3DG.%26aulast%3DP%25C3%25A9rez-Benito%26aufirst%3DL.%26aulast%3DVlijmen%26aufirst%3DH.%26aulast%3DCoveney%26aufirst%3DP.%2BV.%26atitle%3DAccuracy%2520and%2520precision%2520of%2520alchemical%2520relative%2520free%2520energy%2520predictions%2520with%2520and%2520without%2520replica-exchange%26jtitle%3DAdv.%2520Theory%2520Simul.%26date%3D2019%26volume%3D3%26spage%3D1900195%26doi%3D10.1002%2Fadts.201900195" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K. Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Card, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Artis, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fong, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gillette, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsieh, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neiman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milburn, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlessinger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bollag, G.</span></span> <span> </span><span class="NLM_article-title">Glutamine switch mechanism for nucleotide selectivity by phosphodiesterases</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">279</span>– <span class="NLM_lpage">286</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2004.07.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.1016%2Fj.molcel.2004.07.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=15260978" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BD2cXms1ektLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2004&pages=279-286&author=K.+Y.+J.+Zhangauthor=G.+L.+Cardauthor=Y.+Suzukiauthor=D.+R.+Artisauthor=D.+Fongauthor=S.+Gilletteauthor=D.+Hsiehauthor=J.+Neimanauthor=B.+L.+Westauthor=C.+Zhangauthor=M.+V.+Milburnauthor=S.-H.+Kimauthor=J.+Schlessingerauthor=G.+Bollag&title=Glutamine+switch+mechanism+for+nucleotide+selectivity+by+phosphodiesterases&doi=10.1016%2Fj.molcel.2004.07.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">A glutamine switch mechanism for nucleotide selectivity by phosphodiesterases</span></div><div class="casAuthors">Zhang, Kam Y. J.; Card, Graeme L.; Suzuki, Yoshihisa; Artis, D. Richard; Fong, Daniel; Gillette, Sam; Hsieh, Davin; Neiman, Joshua; West, Brian L.; Zhang, Chao; Milburn, Michael V.; Kim, Sung-Hou; Schlessinger, Joseph; Bollag, Gideon</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">279-286</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Phosphodiesterases (PDEs) comprise a family of enzymes that modulate the immune response, inflammation, and memory, among many other functions.  There are three types of PDEs: cAMP-specific, cGMP-specific, and dual-specific.  Here we describe the mechanism of nucleotide selectivity on the basis of high-resoln. co-crystal structures of the cAMP-specific PDE4B and PDE4D with AMP, the cGMP-specific PDE5A with GMP, and the apo-structure of the dual-specific PDE1B.  These structures show that an invariant glutamine functions as the key specificity determinant by a "glutamine switch" mechanism for recognizing the purine moiety in cAMP or cGMP.  The surrounding residues anchor the glutamine residue in different orientations for cAMP and for cGMP.  The PDE1B structure shows that in dual-specific PDEs a key histidine residue may enable the invariant glutamine to toggle between cAMP and cGMP.  The structural understanding of nucleotide binding enables the design of new PDE inhibitors that may treat diseases in which cyclic nucleotides play a crit. role.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorRXGTDdAa1bVg90H21EOLACvtfcHk0lhTi_O1JqhrYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXms1ektLs%253D&md5=32fab6a75031a67290ec5eda2c3e080b</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2004.07.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2004.07.005%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DK.%2BY.%2BJ.%26aulast%3DCard%26aufirst%3DG.%2BL.%26aulast%3DSuzuki%26aufirst%3DY.%26aulast%3DArtis%26aufirst%3DD.%2BR.%26aulast%3DFong%26aufirst%3DD.%26aulast%3DGillette%26aufirst%3DS.%26aulast%3DHsieh%26aufirst%3DD.%26aulast%3DNeiman%26aufirst%3DJ.%26aulast%3DWest%26aufirst%3DB.%2BL.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DMilburn%26aufirst%3DM.%2BV.%26aulast%3DKim%26aufirst%3DS.-H.%26aulast%3DSchlessinger%26aufirst%3DJ.%26aulast%3DBollag%26aufirst%3DG.%26atitle%3DGlutamine%2520switch%2520mechanism%2520for%2520nucleotide%2520selectivity%2520by%2520phosphodiesterases%26jtitle%3DMol.%2520Cell%26date%3D2004%26volume%3D15%26spage%3D279%26epage%3D286%26doi%3D10.1016%2Fj.molcel.2004.07.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jansen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kooistra, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanev, G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leurs, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Esch, I. J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Graaf, C.</span></span> <span> </span><span class="NLM_article-title">PDEStrIAn: A Phosphodiesterase structure and ligand interaction annotated database as a tool for structure-based drug design</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">7029</span>– <span class="NLM_lpage">7065</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01813</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01813" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC28XjtVCju7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=7029-7065&author=C.+Jansenauthor=A.+J.+Kooistraauthor=G.+K.+Kanevauthor=R.+Leursauthor=I.+J.+P.+de+Eschauthor=C.+de+Graaf&title=PDEStrIAn%3A+A+Phosphodiesterase+structure+and+ligand+interaction+annotated+database+as+a+tool+for+structure-based+drug+design&doi=10.1021%2Facs.jmedchem.5b01813"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">PDEStrIAn: A Phosphodiesterase Structure and Ligand Interaction Annotated Database As a Tool for Structure-Based Drug Design</span></div><div class="casAuthors">Jansen, Chimed; Kooistra, Albert J.; Kanev, Georgi K.; Leurs, Rob; de Esch, Iwan J. P.; de Graaf, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">7029-7065</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A systematic anal. is presented of the 220 phosphodiesterase (PDE) catalytic domain crystal structures present in the Protein Data Bank (PDB) with a focus on PDE-ligand interactions.  The consistent structural alignment of 57 PDE ligand binding site residues enables the systematic anal. of PDE-ligand interaction fingerprints (IFPs), the identification of subtype-specific PDE-ligand interaction features, and the classification of ligands according to their binding modes.  We illustrate how systematic mining of this phosphodiesterase structure and ligand interaction annotated (PDEStrIAn) database provides new insights into how conserved and selective PDE interaction hot spots can accommodate the large diversity of chem. scaffolds in PDE ligands.  A substructure anal. of the cocrystd. PDE ligands in combination with those in the ChEMBL database provides a toolbox for scaffold hopping and ligand design.  These analyses lead to an improved understanding of the structural requirements of PDE binding that will be useful in future drug discovery studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-1I6fmiCv-rVg90H21EOLACvtfcHk0lhl-xgOaEtKMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjtVCju7k%253D&md5=67669586d18089a73f4e7de21cf7c33b</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01813&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01813%26sid%3Dliteratum%253Aachs%26aulast%3DJansen%26aufirst%3DC.%26aulast%3DKooistra%26aufirst%3DA.%2BJ.%26aulast%3DKanev%26aufirst%3DG.%2BK.%26aulast%3DLeurs%26aufirst%3DR.%26aulast%3Dde%2BEsch%26aufirst%3DI.%2BJ.%2BP.%26aulast%3Dde%2BGraaf%26aufirst%3DC.%26atitle%3DPDEStrIAn%253A%2520A%2520Phosphodiesterase%2520structure%2520and%2520ligand%2520interaction%2520annotated%2520database%2520as%2520a%2520tool%2520for%2520structure-based%2520drug%2520design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D7029%26epage%3D7065%26doi%3D10.1021%2Facs.jmedchem.5b01813" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brandon, N. J.</span></span> <span> </span><span class="NLM_article-title">Emerging biology of PDE10A</span>. <i>Curr. Pharm. Des.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">378</span>– <span class="NLM_lpage">388</span>, <span class="refDoi"> DOI: 10.2174/1381612820666140826114744</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.2174%2F1381612820666140826114744" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=25159072" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFOntr%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=378-388&author=L.+S.+Wilsonauthor=N.+J.+Brandon&title=Emerging+biology+of+PDE10A&doi=10.2174%2F1381612820666140826114744"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging Biology of PDE10A</span></div><div class="casAuthors">Wilson, Lindsay S.; Brandon, Nicholas J.</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">378-388</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  CAMP and cyclic GMP are essential second messengers that regulate multiple signaling pathways in virtually all cell types.  Their accumulation in cells is finely regulated by cyclic nucleotide phosphodiesterases (PDEs), the only enzymes that can degrade these signaling mols. and thus provide exquisite control over intracellular signaling processes.  One PDE family, PDE10A, is highly enriched in the brain and its unique expression profile in specific brain regions of interest, in particular to antipsychotic treatment, has made it an attractive therapeutic target for the treatment of schizophrenia.  However, after a Phase II trial failure of a selective PDE10A inhibitor for the treatment of schizophrenia, it has encouraged the field to reexamine the role of this enzyme in the brain, and the possible CNS disorders in which PDE10A inhibition could be therapeutic.  We will review the localization of PDE10A, both within the brain and the neuron and discuss how its role in regulating cAMP and cGMP accumulation modulates intracellular signaling pathways.  Since this cellular signaling has best been documented in the striatum, we will focus our discussion of PDE10A in the context of disorders that affect the basal ganglia, including psychiatric disorders such as bipolar disorder and autism spectrum disorders and the movement disorders, including Parkinson's disease and Huntington's disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3nn_WW9cz37Vg90H21EOLACvtfcHk0lhl-xgOaEtKMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFOntr%252FJ&md5=852dbd753943a0ce1043c52085698c2a</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.2174%2F1381612820666140826114744&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1381612820666140826114744%26sid%3Dliteratum%253Aachs%26aulast%3DWilson%26aufirst%3DL.%2BS.%26aulast%3DBrandon%26aufirst%3DN.%2BJ.%26atitle%3DEmerging%2520biology%2520of%2520PDE10A%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2015%26volume%3D21%26spage%3D378%26epage%3D388%26doi%3D10.2174%2F1381612820666140826114744" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Verhoest, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapin, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fonseca, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harms, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marr, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menniti, F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connor, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandit, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Proulx-LaFrance, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suiciak, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liras, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of a novel class of phosphodiesterase 10A inhibitors and identification of clinical candidate 2-[4-(1-Methyl-4-pyridin-4-yl-1H-pyrazol-3-yl)-phenoxymethyl]-quinoline (PF-2545920) for the treatment of schizophrenia</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">5188</span>– <span class="NLM_lpage">5196</span>, <span class="refDoi"> DOI: 10.1021/jm900521k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm900521k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BD1MXovFKmu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=5188-5196&author=P.+R.+Verhoestauthor=D.+S.+Chapinauthor=M.+Cormanauthor=K.+Fonsecaauthor=J.+F.+Harmsauthor=X.+Houauthor=E.+S.+Marrauthor=F.+S.+Mennitiauthor=F.+Nelsonauthor=R.+O%E2%80%99Connorauthor=J.+Panditauthor=C.+Proulx-LaFranceauthor=A.+W.+Schmidtauthor=C.+J.+Schmidtauthor=J.+A.+Suiciakauthor=S.+Liras&title=Discovery+of+a+novel+class+of+phosphodiesterase+10A+inhibitors+and+identification+of+clinical+candidate+2-%5B4-%281-Methyl-4-pyridin-4-yl-1H-pyrazol-3-yl%29-phenoxymethyl%5D-quinoline+%28PF-2545920%29+for+the+treatment+of+schizophrenia&doi=10.1021%2Fjm900521k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Novel Class of Phosphodiesterase 10A Inhibitors and Identification of Clinical Candidate 2-[4-(1-Methyl-4-pyridin-4-yl-1H-pyrazol-3-yl)-phenoxymethyl]-quinoline (PF-2545920)(I) for the Treatment of Schizophrenia</span></div><div class="casAuthors">Verhoest, Patrick R.; Chapin, Douglas S.; Corman, Michael; Fonseca, Kari; Harms, John F.; Hou, Xinjun; Marr, Eric S.; Menniti, Frank S.; Nelson, Frederick; O'Connor, Rebecca; Pandit, Jayvardhan; Proulx-LaFrance, Caroline; Schmidt, Anne W.; Schmidt, Christopher J.; Suiciak, Judith A.; Liras, Spiros</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">5188-5196</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">By utilizing structure-based drug design (SBDD) knowledge, a novel class of phosphodiesterase (PDE) 10A inhibitors was identified.  The structure-based drug design efforts identified a unique "selectivity pocket" for PDE10A inhibitors, and interactions within this pocket allowed the design of highly selective and potent PDE10A inhibitors.  Further optimization of brain penetration and drug-like properties led to the discovery of 2-[4-(1-methyl-4-pyridin-4-yl-1H-pyrazol-3-yl)-phenoxymethyl]-quinoline (PF-2545920) (I).  This PDE10A inhibitor is the first reported clin. entry for this mechanism in the treatment of schizophrenia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwg5FBdhz5NbVg90H21EOLACvtfcHk0lhGV-ACm_8OyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXovFKmu7Y%253D&md5=acc9e8687684c4cef1537a15800c8317</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fjm900521k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm900521k%26sid%3Dliteratum%253Aachs%26aulast%3DVerhoest%26aufirst%3DP.%2BR.%26aulast%3DChapin%26aufirst%3DD.%2BS.%26aulast%3DCorman%26aufirst%3DM.%26aulast%3DFonseca%26aufirst%3DK.%26aulast%3DHarms%26aufirst%3DJ.%2BF.%26aulast%3DHou%26aufirst%3DX.%26aulast%3DMarr%26aufirst%3DE.%2BS.%26aulast%3DMenniti%26aufirst%3DF.%2BS.%26aulast%3DNelson%26aufirst%3DF.%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DR.%26aulast%3DPandit%26aufirst%3DJ.%26aulast%3DProulx-LaFrance%26aufirst%3DC.%26aulast%3DSchmidt%26aufirst%3DA.%2BW.%26aulast%3DSchmidt%26aufirst%3DC.%2BJ.%26aulast%3DSuiciak%26aufirst%3DJ.%2BA.%26aulast%3DLiras%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520a%2520novel%2520class%2520of%2520phosphodiesterase%252010A%2520inhibitors%2520and%2520identification%2520of%2520clinical%2520candidate%25202-%255B4-%25281-Methyl-4-pyridin-4-yl-1H-pyrazol-3-yl%2529-phenoxymethyl%255D-quinoline%2520%2528PF-2545920%2529%2520for%2520the%2520treatment%2520of%2520schizophrenia%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D5188%26epage%3D5196%26doi%3D10.1021%2Fjm900521k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scapin, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varnerin, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edmondson, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mastracchio, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parmee, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van der
Ploeg, L. H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tota, M. R.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of human phosphodiesterase 3B: Atomic basis for substrate and inhibitor specificity</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">6091</span>– <span class="NLM_lpage">6100</span>, <span class="refDoi"> DOI: 10.1021/bi049868i</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi049868i" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjsVShs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2004&pages=6091-6100&author=G.+Scapinauthor=S.+B.+Patelauthor=C.+Chungauthor=J.+P.+Varnerinauthor=S.+D.+Edmondsonauthor=A.+Mastracchioauthor=E.+R.+Parmeeauthor=S.+B.+Singhauthor=J.+W.+Beckerauthor=L.+H.+T.+Van+der%0APloegauthor=M.+R.+Tota&title=Crystal+structure+of+human+phosphodiesterase+3B%3A+Atomic+basis+for+substrate+and+inhibitor+specificity&doi=10.1021%2Fbi049868i"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal Structure of Human Phosphodiesterase 3B: Atomic Basis for Substrate and Inhibitor Specificity</span></div><div class="casAuthors">Scapin, Giovanna; Patel, Sangita B.; Chung, Christine; Varnerin, Jeffrey P.; Edmondson, Scott D.; Mastracchio, Anthony; Parmee, Emma R.; Singh, Suresh B.; Becker, Joseph W.; Van der Ploeg, Lex H. T.; Tota, Michael R.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">6091-6100</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Phosphodiesterases (PDEs) are enzymes that modulate cyclic nucleotide signaling and as such are clin. targets for a range of disorders including congestive heart failure, erectile dysfunction, and inflammation.  The PDE3 family comprises two highly homologous subtypes expressed in different tissues, and inhibitors of this family have been shown to increase lipolysis in adipocytes.  A specific PDE3B (the lipocyte-localized subtype) inhibitor would be a very useful tool to evaluate the effects of PDE3 inhibition on lipolysis and metabolic rate and might become a novel tool for treatment of obesity.  We report here the three-dimensional structures of the catalytic domain of human PDE3B in complex with a generic PDE inhibitor and a novel PDE3 selective inhibitor.  These structures explain the dual cAMP/cGMP binding capabilities of PDE3, provide the mol. basis for inhibitor specificity, and can supply a valid platform for the design of improved compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6lJKq3qzQGrVg90H21EOLACvtfcHk0lhGV-ACm_8OyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjsVShs7s%253D&md5=e39041b5c61c01b8b04b21e1d8515c22</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fbi049868i&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi049868i%26sid%3Dliteratum%253Aachs%26aulast%3DScapin%26aufirst%3DG.%26aulast%3DPatel%26aufirst%3DS.%2BB.%26aulast%3DChung%26aufirst%3DC.%26aulast%3DVarnerin%26aufirst%3DJ.%2BP.%26aulast%3DEdmondson%26aufirst%3DS.%2BD.%26aulast%3DMastracchio%26aufirst%3DA.%26aulast%3DParmee%26aufirst%3DE.%2BR.%26aulast%3DSingh%26aufirst%3DS.%2BB.%26aulast%3DBecker%26aufirst%3DJ.%2BW.%26aulast%3DVan%2Bder%2BPloeg%26aufirst%3DL.%2BH.%2BT.%26aulast%3DTota%26aufirst%3DM.%2BR.%26atitle%3DCrystal%2520structure%2520of%2520human%2520phosphodiesterase%25203B%253A%2520Atomic%2520basis%2520for%2520substrate%2520and%2520inhibitor%2520specificity%26jtitle%3DBiochemistry%26date%3D2004%26volume%3D43%26spage%3D6091%26epage%3D6100%26doi%3D10.1021%2Fbi049868i" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhan, C.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, H.-B.</span></span> <span> </span><span class="NLM_article-title">Absolute binding free energy calculation and design of a subnanomolar inhibitor of phosphodiesterase-10</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">2099</span>– <span class="NLM_lpage">2111</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01763</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01763" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvFKjsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=2099-2111&author=Z.+Liauthor=Y.+Huangauthor=Y.+Wuauthor=J.+Chenauthor=D.+Wuauthor=C.-G.+Zhanauthor=H.-B.+Luo&title=Absolute+binding+free+energy+calculation+and+design+of+a+subnanomolar+inhibitor+of+phosphodiesterase-10&doi=10.1021%2Facs.jmedchem.8b01763"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Absolute Binding Free Energy Calculation and Design of a Subnanomolar Inhibitor of Phosphodiesterase-10</span></div><div class="casAuthors">Li, Zhe; Huang, Yiyou; Wu, Yinuo; Chen, Jingyi; Wu, Deyan; Zhan, Chang-Guo; Luo, Hai-Bin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">2099-2111</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Accurate prediction of abs. protein-ligand binding free energy could considerably enhance the success rate of structure-based drug design but is extremely challenging and time-consuming.  Free energy perturbation (FEP) has been proven reliable but is limited to prediction of relative binding free energies of similar ligands (with only minor structural differences) in binding with a same drug target in practical drug design applications.  Herein, a Gaussian algorithm-enhanced FEP (GA-FEP) protocol has been developed to enhance the FEP simulation performance, enabling to efficiently carry out the FEP simulations on vanishing the whole ligand and, thus, predict the abs. binding free energies (ABFEs).  Using the GA-FEP protocol, the FEP simulations for the ABFE calcn. (denoted as GA-FEP/ABFE) can achieve a satisfactory accuracy for both structurally similar and diverse ligands in a dataset of more than 100 receptor-ligand systems.  Further, our GA-FEP/ABFE-guided lead optimization against phosphodiesterase-10 led to the discovery of a subnanomolar inhibitor (IC50 = 0.87 nM, ∼2000-fold improvement in potency) with cocrystal confirmation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKTyv17ShhKrVg90H21EOLACvtfcHk0lhGV-ACm_8OyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvFKjsLc%253D&md5=feaf32e0006d6a3cf2f2b7d4d98c2a25</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01763&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01763%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DD.%26aulast%3DZhan%26aufirst%3DC.-G.%26aulast%3DLuo%26aufirst%3DH.-B.%26atitle%3DAbsolute%2520binding%2520free%2520energy%2520calculation%2520and%2520design%2520of%2520a%2520subnanomolar%2520inhibitor%2520of%2520phosphodiesterase-10%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D2099%26epage%3D2111%26doi%3D10.1021%2Facs.jmedchem.8b01763" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moraca, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Negri, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Oliveira, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abel, R.</span></span> <span> </span><span class="NLM_article-title">Application of free energy perturbation (FEP+) to understanding ligand selectivity: A case study to assess selectivity between pairs of phosphodiesterases (PDE’s)</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">2729</span>– <span class="NLM_lpage">2740</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.9b00106</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.9b00106" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtVKgtLjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2019&pages=2729-2740&author=F.+Moracaauthor=A.+Negriauthor=C.+de+Oliveiraauthor=R.+Abel&title=Application+of+free+energy+perturbation+%28FEP%2B%29+to+understanding+ligand+selectivity%3A+A+case+study+to+assess+selectivity+between+pairs+of+phosphodiesterases+%28PDE%E2%80%99s%29&doi=10.1021%2Facs.jcim.9b00106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Application of Free Energy Perturbation (FEP+) to Understanding Ligand Selectivity: A Case Study to Assess Selectivity Between Pairs of Phosphodiesterases (PDE's)</span></div><div class="casAuthors">Moraca, Francesca; Negri, Ana; de Oliveira, Cesar; Abel, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2729-2740</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cyclic nucleotide phosphodiesterases (PDE's) are metalloenzymes that play a key role in regulating the levels of the ubiquitous second messengers, cyclic adenosine monophosphate (cAMP) and cyclic guanosine monophosphate (cGMP).  In humans, 11 PDE protein families mediate numerous biochem. pathways throughout the body and are effective drug targets for the treatment of diseases ranging from central nervous system disorders to heart and pulmonary diseases.  PDE's also share a highly conserved catalytic site (about 50%), thus making the design of selective drug candidates very challenging with classical structure-based design approaches given also the lack of publicly available co-crystal structures of pairs of PDE's with consistent biol. data to be compared, as we show in our work.  In this retrospective study, we apply free energy perturbation (FEP+) to predict the selectivity of inhibitors that bind two pairs of closely related PDE families: PDE9/1 and PDE5/6 where only 1 co-crystal structure per pair is publicly available.  As another challenge, the pKa of the PDE5/6 inhibitor is close to the exptl. pH, making unclear the exact protonation state that should be used in the computational workflow.  We demonstrate that running FEP+ on homol. models constructed for these metalloenzymes accurately reproduces exptl. obsd. selectivity profiles also addressing the unclear protonation state to be used during computation with our recently developed pKa-correction method.  Based on these data, we conclude that FEP+ is a robust method for prediction of selectivity for this target class and may be helpful to address related lead optimization challenges in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_z0VwaEW9krVg90H21EOLACvtfcHk0ljHRFHL-fy3lg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtVKgtLjJ&md5=b46b0d3d7dd802f4852af5253d6c629a</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.9b00106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.9b00106%26sid%3Dliteratum%253Aachs%26aulast%3DMoraca%26aufirst%3DF.%26aulast%3DNegri%26aufirst%3DA.%26aulast%3Dde%2BOliveira%26aufirst%3DC.%26aulast%3DAbel%26aufirst%3DR.%26atitle%3DApplication%2520of%2520free%2520energy%2520perturbation%2520%2528FEP%252B%2529%2520to%2520understanding%2520ligand%2520selectivity%253A%2520A%2520case%2520study%2520to%2520assess%2520selectivity%2520between%2520pairs%2520of%2520phosphodiesterases%2520%2528PDE%25E2%2580%2599s%2529%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2019%26volume%3D59%26spage%3D2729%26epage%3D2740%26doi%3D10.1021%2Facs.jcim.9b00106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mobley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span> <span> </span><span class="NLM_article-title">A critical review of validation, blind testing, and real-world use of alchemical protein-ligand binding free energy calculations</span>. <i>Curr. Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">2577</span>– <span class="NLM_lpage">2585</span>, <span class="refDoi"> DOI: 10.2174/1568026617666170414142131</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.2174%2F1568026617666170414142131" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=28413950" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlelsLnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2017&pages=2577-2585&author=R.+Abelauthor=L.+Wangauthor=D.+L.+Mobleyauthor=R.+A.+Friesner&title=A+critical+review+of+validation%2C+blind+testing%2C+and+real-world+use+of+alchemical+protein-ligand+binding+free+energy+calculations&doi=10.2174%2F1568026617666170414142131"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">A Critical Review of Validation, Blind Testing, and Real- World Use of Alchemical Protein-Ligand Binding Free Energy Calculations</span></div><div class="casAuthors">Abel, Robert; Wang, Lingle; Mobley, David L.; Friesner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">2577-2585</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">Protein-ligand binding is among the most fundamental phenomena underlying all mol. biol., and a greater ability to more accurately and robustly predict the binding free energy of a small mol. ligand for its cognate protein is expected to have vast consequences for improving the efficiency of pharmaceutical drug discovery.  We briefly reviewed a no. of scientific and tech. advances that have enabled alchem. free energy calcns. to recently emerge as a preferred approach, and critically considered proper validation and effective use of these techniques.  In particular, we characterized a selection bias effect which may be important in prospective free energy calcns., and introduced a strategy to improve the accuracy of the free energy predictions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYWt7IyeVNI7Vg90H21EOLACvtfcHk0ljHRFHL-fy3lg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlelsLnM&md5=1b13f6922117268e88b450f1012d140d</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.2174%2F1568026617666170414142131&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568026617666170414142131%26sid%3Dliteratum%253Aachs%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DMobley%26aufirst%3DD.%2BL.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DA%2520critical%2520review%2520of%2520validation%252C%2520blind%2520testing%252C%2520and%2520real-world%2520use%2520of%2520alchemical%2520protein-ligand%2520binding%2520free%2520energy%2520calculations%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2017%26volume%3D17%26spage%3D2577%26epage%3D2585%26doi%3D10.2174%2F1568026617666170414142131" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mey, A. S.</span>; <span class="NLM_string-name">Allen, B.</span>; <span class="NLM_string-name">Macdonald, H. E. B.</span>; <span class="NLM_string-name">Chodera, J. D.</span>; <span class="NLM_string-name">Kuhn, M.</span>; <span class="NLM_string-name">Michel, J.</span>; <span class="NLM_string-name">Mobley, D. L.</span>; <span class="NLM_string-name">Naden, L. N.</span>; <span class="NLM_string-name">Prasad, S.</span>; <span class="NLM_string-name">Rizzi, A.</span>; <span class="NLM_string-name">Scheen, J.</span>; <span class="NLM_string-name">Shirts, M. R.</span>; <span class="NLM_string-name">Tresadern, G.</span>; <span class="NLM_string-name">Xu, H.</span></span>. <span> </span><span class="NLM_article-title">Best practice for free energy calculations</span>. <span class="NLM_year">2020</span>, arXiv:2008.03067. Published Online: August 21, 2020, <a href="https://arxiv.org/abs/2008.03067" class="extLink">https://arxiv.org/abs/2008.03067</a> (accessed Sept 22, 2020).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=A.+S.+Mey&author=B.+Allen&author=H.+E.+B.+Macdonald&author=J.+D.+Chodera&author=M.+Kuhn&author=J.+Michel&author=D.+L.+Mobley&author=L.+N.+Naden&author=S.+Prasad&author=A.+Rizzi&author=J.+Scheen&author=M.+R.+Shirts&author=G.+Tresadern&author=H.+Xu&title=Best+practice+for+free+energy+calculations"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3DMey%26aufirst%3DA.%2BS.%26atitle%3DBest%2520practice%2520for%2520free%2520energy%2520calculations%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shirts, M. R.</span>; <span class="NLM_string-name">Mobley, D. L.</span>; <span class="NLM_string-name">Brown, S. P.</span></span> <span> </span><span class="NLM_article-title">Free-Energy Calculations in Structure-Based Drug Design</span>. In  <i>Drug Design. Structure- and Ligand-Based Approaches</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merz, K. M.,  Jr.</span>, <span class="NLM_string-name">Ringe, D.</span>, <span class="NLM_string-name">Reynolds, C. H.</span></span>, Eds.; <span class="NLM_publisher-name">Cambridge University Press</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">2010</span>; pp  <span class="NLM_fpage">61</span>– <span class="NLM_lpage">86</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.1017%2FCBO9780511730412.007" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&pages=61-86&author=M.+R.+Shirts&author=D.+L.+Mobley&author=S.+P.+Brownauthor=K.+M.+Merz&author=D.+Ringe&author=C.+H.+Reynolds&title=Drug+Design.+Structure-+and+Ligand-Based+Approaches"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1017%2FCBO9780511730412.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1017%252FCBO9780511730412.007%26sid%3Dliteratum%253Aachs%26aulast%3DShirts%26aufirst%3DM.%2BR.%26atitle%3DFree-Energy%2520Calculations%2520in%2520Structure-Based%2520Drug%2520Design%26btitle%3DDrug%2520Design.%2520Structure-%2520and%2520Ligand-Based%2520Approaches%26aulast%3DMerz%26aufirst%3DK.%2BM.%26pub%3DCambridge%2520University%2520Press%26date%3D2010%26spage%3D61%26epage%3D86" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manas, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farid, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span> <span> </span><span class="NLM_article-title">Modeling the value of predictive affinity scoring in preclinical drug discovery</span>. <i>Curr. Opin. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">103</span>– <span class="NLM_lpage">110</span>, <span class="refDoi"> DOI: 10.1016/j.sbi.2018.09.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.1016%2Fj.sbi.2018.09.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=30321805" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslyru73I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2018&pages=103-110&author=R.+Abelauthor=E.+S.+Manasauthor=R.+A.+Friesnerauthor=R.+S.+Faridauthor=L.+Wang&title=Modeling+the+value+of+predictive+affinity+scoring+in+preclinical+drug+discovery&doi=10.1016%2Fj.sbi.2018.09.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Modeling the value of predictive affinity scoring in preclinical drug discovery</span></div><div class="casAuthors">Abel, Robert; Manas, Eric S.; Friesner, Richard A.; Farid, Ramy S.; Wang, Lingle</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">103-110</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Drug discovery is widely recognized to be a difficult and costly activity in large part due to the challenge of identifying chem. matter which simultaneously optimizes multiple properties, one of which is affinity for the primary biol. target.  Further, many of these properties are difficult to predict ahead of expensive and time-consuming compd. synthesis and exptl. testing.  Here we highlight recent work to develop compd. affinity prediction models, and extensively investigate the value such models may provide to preclin. drug discovery.  We demonstrate that the ability of these models to improve the overall probability of success is crucially dependent on the shape of the error distribution, not just the root-mean-square error.  In particular, while scoring more mol. ideas generally improves the probability of project success when the error distribution is Gaussian, fat-tail distributions such as a Cauchy distribution, can lead to a situation where scoring more ideas actually decreases the overall probability of success.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIs1gJSX0VLbVg90H21EOLACvtfcHk0ljHRFHL-fy3lg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslyru73I&md5=2e996488928b652dd299d3e1425d4dd3</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2018.09.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2018.09.002%26sid%3Dliteratum%253Aachs%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DManas%26aufirst%3DE.%2BS.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DFarid%26aufirst%3DR.%2BS.%26aulast%3DWang%26aufirst%3DL.%26atitle%3DModeling%2520the%2520value%2520of%2520predictive%2520affinity%2520scoring%2520in%2520preclinical%2520drug%2520discovery%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2018%26volume%3D52%26spage%3D103%26epage%3D110%26doi%3D10.1016%2Fj.sbi.2018.09.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group"><span class="NLM_string-name">Van Roosbroeck, Y. E. M.</span>; <span class="NLM_string-name">Van den Keybus, F. A. M.</span>; <span class="NLM_string-name">Tresadern, G. J.</span>; <span class="NLM_string-name">Buijnsters, P. J. J. A.</span>; <span class="NLM_string-name">Velter, A. I.</span>; <span class="NLM_string-name">Jacoby, E.</span>; <span class="NLM_string-name">Macdonald, G. J.</span>; <span class="NLM_string-name">Gijsen, H. J. M.</span>; <span class="NLM_string-name">Ahnaou, A.</span>; <span class="NLM_string-name">Drinkenburg, W. H. I. M.</span></span> <span> </span><span class="NLM_article-title">[1,2,4]Triazolo[1,5-a]pyrimidine Derivatives as PDE2 Inhibitors</span>. <span class="NLM_patent">WO 2018083098 A1</span>, May 11, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=Y.+E.+M.+Van+Roosbroeck&author=F.+A.+M.+Van+den+Keybus&author=G.+J.+Tresadern&author=P.+J.+J.+A.+Buijnsters&author=A.+I.+Velter&author=E.+Jacoby&author=G.+J.+Macdonald&author=H.+J.+M.+Gijsen&author=A.+Ahnaou&author=W.+H.+I.+M.+Drinkenburg&title=%5B1%2C2%2C4%5DTriazolo%5B1%2C5-a%5Dpyrimidine+Derivatives+as+PDE2+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50a&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2BRoosbroeck%26aufirst%3DY.%2BE.%2BM.%26atitle%3D%255B1%252C2%252C4%255DTriazolo%255B1%252C5-a%255Dpyrimidine%2520Derivatives%2520as%2520PDE2%2520Inhibitors%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit50b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group"><span class="NLM_string-name">Van Roosbroeck, Y. E. M.</span>; <span class="NLM_string-name">Buijnsters, P. J. J. A.</span>; <span class="NLM_string-name">Tresadern, G. J.</span>; <span class="NLM_string-name">Jacoby, E.</span>; <span class="NLM_string-name">Oehlrich, D.</span>; <span class="NLM_string-name">Gijsen, H. J. M.</span></span> <span> </span><span class="NLM_article-title">[1,2,4]Triazolo[1,5-a]pyrimidine Compounds as PDE2 Inhibitors</span>. <span class="NLM_patent">WO 2018083103 A1</span>, May 11, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=Y.+E.+M.+Van+Roosbroeck&author=P.+J.+J.+A.+Buijnsters&author=G.+J.+Tresadern&author=E.+Jacoby&author=D.+Oehlrich&author=H.+J.+M.+Gijsen&title=%5B1%2C2%2C4%5DTriazolo%5B1%2C5-a%5Dpyrimidine+Compounds+as+PDE2+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50b&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2BRoosbroeck%26aufirst%3DY.%2BE.%2BM.%26atitle%3D%255B1%252C2%252C4%255DTriazolo%255B1%252C5-a%255Dpyrimidine%2520Compounds%2520as%2520PDE2%2520Inhibitors%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="note"><p class="first last">The absolute configuration was determined using vibrational VCD. See <a href="/doi/suppl/10.1021/acs.jmedchem.0c01272/suppl_file/jm0c01272_si_001.pdf" class="ext-link">Supporting Information</a>.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghanakota, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bos, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konze, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staker, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marques, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marshall, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leswing, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhat, S.</span></span> <span> </span><span class="NLM_article-title">Combining cloud-based free energy calculations, synthetically aware enumerations and goal-directed generative machine learning for rapid large scale chemical exploration and optimization</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.0c00120</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.0c00120" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=P.+Ghanakotaauthor=P.+H.+Bosauthor=K.+Konzeauthor=J.+Stakerauthor=G.+Marquesauthor=K.+Marshallauthor=K.+Leswingauthor=R.+Abelauthor=S.+Bhat&title=Combining+cloud-based+free+energy+calculations%2C+synthetically+aware+enumerations+and+goal-directed+generative+machine+learning+for+rapid+large+scale+chemical+exploration+and+optimization&doi=10.1021%2Facs.jcim.0c00120"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.0c00120&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.0c00120%26sid%3Dliteratum%253Aachs%26aulast%3DGhanakota%26aufirst%3DP.%26aulast%3DBos%26aufirst%3DP.%2BH.%26aulast%3DKonze%26aufirst%3DK.%26aulast%3DStaker%26aufirst%3DJ.%26aulast%3DMarques%26aufirst%3DG.%26aulast%3DMarshall%26aufirst%3DK.%26aulast%3DLeswing%26aufirst%3DK.%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DBhat%26aufirst%3DS.%26atitle%3DCombining%2520cloud-based%2520free%2520energy%2520calculations%252C%2520synthetically%2520aware%2520enumerations%2520and%2520goal-directed%2520generative%2520machine%2520learning%2520for%2520rapid%2520large%2520scale%2520chemical%2520exploration%2520and%2520optimization%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2020%26doi%3D10.1021%2Facs.jcim.0c00120" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schindler, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baumann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blum, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Böse, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchstaller, H.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgdorf, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cappel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chekler, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czodrowski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dorsch, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eguida, M. K. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Follows, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuchß, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grädler, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunera, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jorand Lebrun, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karra, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knehans, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kötzner, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krier, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leiendecker, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leuthner, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mochalkin, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Musil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neagu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rippmann, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiemann, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulz, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinbrecher, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanzer, E.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unzue
Lopez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viacava Follis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wegener, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, D.</span></span> <span> </span><span class="NLM_article-title">Large-Scale assessment of binding free energy calculations in active drug discovery projects</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.0c00900</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.0c00900" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=C.+Schindlerauthor=H.+Baumannauthor=A.+Blumauthor=D.+B%C3%B6seauthor=H.-P.+Buchstallerauthor=L.+Burgdorfauthor=D.+Cappelauthor=E.+Cheklerauthor=P.+Czodrowskiauthor=D.+Dorschauthor=M.+K.+I.+Eguidaauthor=B.+Followsauthor=T.+Fuch%C3%9Fauthor=U.+Gr%C3%A4dlerauthor=J.+Guneraauthor=T.+Johnsonauthor=C.+Jorand+Lebrunauthor=S.+Karraauthor=M.+Kleinauthor=T.+Knehansauthor=L.+K%C3%B6tznerauthor=M.+Krierauthor=M.+Leiendeckerauthor=B.+Leuthnerauthor=L.+Liauthor=I.+Mochalkinauthor=D.+Musilauthor=C.+Neaguauthor=F.+Rippmannauthor=K.+Schiemannauthor=R.+Schulzauthor=T.+B.+Steinbrecherauthor=E.-M.+Tanzerauthor=A.+Unzue%0ALopezauthor=A.+Viacava+Follisauthor=A.+Wegenerauthor=D.+Kuhn&title=Large-Scale+assessment+of+binding+free+energy+calculations+in+active+drug+discovery+projects&doi=10.1021%2Facs.jcim.0c00900"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.0c00900&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.0c00900%26sid%3Dliteratum%253Aachs%26aulast%3DSchindler%26aufirst%3DC.%26aulast%3DBaumann%26aufirst%3DH.%26aulast%3DBlum%26aufirst%3DA.%26aulast%3DB%25C3%25B6se%26aufirst%3DD.%26aulast%3DBuchstaller%26aufirst%3DH.-P.%26aulast%3DBurgdorf%26aufirst%3DL.%26aulast%3DCappel%26aufirst%3DD.%26aulast%3DChekler%26aufirst%3DE.%26aulast%3DCzodrowski%26aufirst%3DP.%26aulast%3DDorsch%26aufirst%3DD.%26aulast%3DEguida%26aufirst%3DM.%2BK.%2BI.%26aulast%3DFollows%26aufirst%3DB.%26aulast%3DFuch%25C3%259F%26aufirst%3DT.%26aulast%3DGr%25C3%25A4dler%26aufirst%3DU.%26aulast%3DGunera%26aufirst%3DJ.%26aulast%3DJohnson%26aufirst%3DT.%26aulast%3DJorand%2BLebrun%26aufirst%3DC.%26aulast%3DKarra%26aufirst%3DS.%26aulast%3DKlein%26aufirst%3DM.%26aulast%3DKnehans%26aufirst%3DT.%26aulast%3DK%25C3%25B6tzner%26aufirst%3DL.%26aulast%3DKrier%26aufirst%3DM.%26aulast%3DLeiendecker%26aufirst%3DM.%26aulast%3DLeuthner%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DMochalkin%26aufirst%3DI.%26aulast%3DMusil%26aufirst%3DD.%26aulast%3DNeagu%26aufirst%3DC.%26aulast%3DRippmann%26aufirst%3DF.%26aulast%3DSchiemann%26aufirst%3DK.%26aulast%3DSchulz%26aufirst%3DR.%26aulast%3DSteinbrecher%26aufirst%3DT.%2BB.%26aulast%3DTanzer%26aufirst%3DE.-M.%26aulast%3DUnzue%2BLopez%26aufirst%3DA.%26aulast%3DViacava%2BFollis%26aufirst%3DA.%26aulast%3DWegener%26aufirst%3DA.%26aulast%3DKuhn%26aufirst%3DD.%26atitle%3DLarge-Scale%2520assessment%2520of%2520binding%2520free%2520energy%2520calculations%2520in%2520active%2520drug%2520discovery%2520projects%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2020%26doi%3D10.1021%2Facs.jcim.0c00900" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Madhavi
Sastry, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adzhigirey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Annabhimoju, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, W.</span></span> <span> </span><span class="NLM_article-title">Protein and ligand preparation: Parameters, protocols, and influence on virtual screening enrichments</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">221</span>– <span class="NLM_lpage">234</span>, <span class="refDoi"> DOI: 10.1007/s10822-013-9644-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.1007%2Fs10822-013-9644-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=23579614" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmslalu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2013&pages=221-234&author=G.+Madhavi%0ASastryauthor=M.+Adzhigireyauthor=T.+Dayauthor=R.+Annabhimojuauthor=W.+Sherman&title=Protein+and+ligand+preparation%3A+Parameters%2C+protocols%2C+and+influence+on+virtual+screening+enrichments&doi=10.1007%2Fs10822-013-9644-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Protein and ligand preparation: parameters, protocols, and influence on virtual screening enrichments</span></div><div class="casAuthors">Madhavi Sastry, G.; Adzhigirey, Matvey; Day, Tyler; Annabhimoju, Ramakrishna; Sherman, Woody</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">221-234</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Structure-based virtual screening plays an important role in drug discovery and complements other screening approaches.  In general, protein crystal structures are prepd. prior to docking in order to add hydrogen atoms, optimize hydrogen bonds, remove at. clashes, and perform other operations that are not part of the x-ray crystal structure refinement process.  In addn., ligands must be prepd. to create 3-dimensional geometries, assign proper bond orders, and generate accessible tautomer and ionization states prior to virtual screening.  While the prerequisite for proper system prepn. is generally accepted in the field, an extensive study of the prepn. steps and their effect on virtual screening enrichments has not been performed.  In this work, we systematically explore each of the steps involved in prepg. a system for virtual screening.  We first explore a large no. of parameters using the Glide validation set of 36 crystal structures and 1,000 decoys.  We then apply a subset of protocols to the DUD database.  We show that database enrichment is improved with proper prepn. and that neglecting certain steps of the prepn. process produces a systematic degrdn. in enrichments, which can be large for some targets.  We provide examples illustrating the structural changes introduced by the prepn. that impact database enrichment.  While the work presented here was performed with the Protein Prepn. Wizard and Glide, the insights and guidance are expected to be generalizable to structure-based virtual screening with other docking methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruqN5nIOJtR7Vg90H21EOLACvtfcHk0li29erkccPAKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmslalu7c%253D&md5=259a6d547ef3e1310e091fb50fe8de16</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1007%2Fs10822-013-9644-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-013-9644-8%26sid%3Dliteratum%253Aachs%26aulast%3DMadhavi%2BSastry%26aufirst%3DG.%26aulast%3DAdzhigirey%26aufirst%3DM.%26aulast%3DDay%26aufirst%3DT.%26aulast%3DAnnabhimoju%26aufirst%3DR.%26aulast%3DSherman%26aufirst%3DW.%26atitle%3DProtein%2520and%2520ligand%2520preparation%253A%2520Parameters%252C%2520protocols%252C%2520and%2520influence%2520on%2520virtual%2520screening%2520enrichments%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2013%26volume%3D27%26spage%3D221%26epage%3D234%26doi%3D10.1007%2Fs10822-013-9644-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group">Schrodinger
LLC</span>. 120 West 45th Street 17th Floor, Tower
45 New York, NY 10036-4041, USA at <a href="http://www.schrodinger.com/Maestro/" class="extLink">http://www.schrodinger.com/Maestro/</a>, <span class="NLM_year">2014</span>. (accessed Sept 18, 2020).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Schrodinger%0ALLC.+120+West+45th+Street+17th+Floor%2C+Tower%0A45+New+York%2C+NY+10036-4041%2C+USA+at+http%3A%2F%2Fwww.schrodinger.com%2FMaestro%2F%2C+2014.+%28accessed+Sept+18%2C+2020%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3D%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50a','cit50b'],'ref51':[],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/medium/jm0c01272_0027.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0027.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01272&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/medium/jm0c01272_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0003.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Previous tricyclic PDE2A inhibitor leads <b>1</b> and <b>2</b>, along with triazolopyrimidine HTS hit <b>3</b> which is the subject of this report.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01272&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/medium/jm0c01272_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0004.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. X-ray structure of HTS hit <b>3</b> with PDE2A viewed from the entrance to the active site (A) and top down (B), PDB accession code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZND">6ZND</a>. Important amino acids are highlighted in green, and catalytic Zn<sup>2+</sup> and Mg<sup>2+</sup> ions are shown, along with active site water molecules (red spheres); hydrogen bonds between Gln812 and Gln859 and the triazolopyrimidine scaffold are indicated with dashed yellow lines in panel B.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01272&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="cht1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Chart 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/medium/jm0c01272_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0002.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Chart 1. Medicinal Chemistry Strategy for Exploration of HTS Hit <b>3</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01272&amp;id=cht1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/medium/jm0c01272_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0005.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. X-ray structure of lead <b>39</b> with PDE2A viewed from the entrance to the active site (A) and top down (B), PDB accession code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZQZ">6ZQZ</a>. Important amino acids are highlighted in green, and catalytic Zn<sup>2+</sup> and Mg<sup>2+</sup> ions are shown, along with active site water molecules (red spheres); hydrogen bonds are indicated with dashed yellow lines in panel B.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01272&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/medium/jm0c01272_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0006.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. 4-Methyl-substituted piperidines <b>41</b>–<b>43</b>, introduction of the methyl group at the 4-position of the piperidine central scaffold to prevent racemization.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01272&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/medium/jm0c01272_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0007.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Correlation of experimental and calculated binding activity (∂<i>G</i>) for the compounds synthesized in each round of the FEP-guided design. The compound data points are colored and shaped according to the rounds of FEP and synthesis, see <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>. The diagonal lines of unity and ±1 or 2 kcal/mol errors are shown. The trendline (black, partly obscured by the line of unity), equation, and <i>R</i><sup>2</sup> were calculated based on all the data combined, showing almost linear slope and very little offset. Selected compounds are labeled.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01272&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/medium/jm0c01272_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0008.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Comparative examples of 4-substituted pyridine RHS substituents arising from the FEP calculations.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01272&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/medium/jm0c01272_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. General Retrosynthetic Route</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01272&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch2" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/medium/jm0c01272_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0010.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Intermediates <b>48a</b>–<b>d</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01272&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) DMF/DMA, 120 °C, 3 h; (b) AcOH, 100 °C, 1 h, 28–62% over 2 steps; (c) HCl, 1,4-dioxane, RT, 1 h, 70–93%.</p></p></figure><figure data-id="sch3" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/medium/jm0c01272_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0011.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Intermediates <b>48e</b>–<b>f</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01272&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NaH, CS<sub>2</sub>, CH<sub>3</sub>I; THF, 0 °C—RT, 16 h, 21%; (b) 150 °C, 3 h, 12%; (c) mCPBA, CHCl<sub>3</sub>, 60 °C, 2 h, 99%; (d) R = OMe: NaOMe, MeOH, RT, 30 min; (e) R = CN: NaCN, DMSO, RT, 30 min, 43%; (f) TFA, DCM, RT, 2 h; R = OMe: 99%, R = CN: 68%.</p></p></figure><figure data-id="sch4" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/medium/jm0c01272_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0012.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Intermediates <b>48g</b>–<b>i</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01272&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 120 °C, 3 h, 95%; (b) AcOH, 120 °C, 30 min, for 53: 80%, for 2-aminoimidazole: 55%; (c) C<sub>6</sub>H<sub>6</sub>, KOtBu, 0 °C—RT, 5 h, 98%; (d) (i) <b>53</b>, DMF, 60 °C, 28 h; (ii) Et<sub>3</sub>N, di-<i>tert</i>-butyl dicarbonate, 0 °C—RT, 1 h, 86%; (e) HCl, iPrOH, MeOH, 1–24 h, RT (<b>48g</b>, 55%; <b>48h</b>, 95%; <b>48i</b>, 61%).</p></p></figure><figure data-id="sch5" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/medium/jm0c01272_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0013.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of 3,4- and 3,6-Dihydropyridyl Pinacol Boronic Ester Intermediates<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01272&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) LDA, THF, −78 °C—RT, 16 h; (b) AcOK, Pin<sub>2</sub>B<sub>2</sub>, Pd(OAc)<sub>2</sub>, PCy<sub>3</sub>, 2-MeTHF, 80 °C, 16 h.</p></p></figure><figure data-id="sch6" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/medium/jm0c01272_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0014.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis of Intermediates <b>48j</b>–<b>48o</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01272&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Pd(PPh<sub>3</sub>)<sub>4</sub>, Na<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane, EtOH, water, (9–27%). (b) H<sub>2</sub>, Pd/C, MeOH, (R<sup>1</sup> = iPr, R<sup>2</sup> = H, 99%) or Pt/C (47%–88%); (c) HCl, iPrOH. MeOH, RT, 16 h (94–100%); (d) (SIPr)Pd(allyl)Cl, Na<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane, EtOH, water (18%); (e) H<sub>2</sub>, Pt/C, MeOH (99%); (f) HCl, iPrOH. MeOH, RT, 16 h,91%; (h) (SIPr)Pd(allyl)Cl, Na<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane, EtOH, water,63%; (i) H<sub>2</sub>, Pd/C, MeOH, (R<sup>1</sup> = iPr, R<sup>2</sup> = H, 86%); (j) HCl, iPrOH. MeOH, RT, 16 h, 94%.</p></p></figure><figure data-id="sch7" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Scheme 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/medium/jm0c01272_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0015.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 7. Synthesis of Compounds <b>3</b>, <b>7</b>–<b>18</b>, <b>21</b>–<b>36</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01272&amp;id=sch7"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: for compounds <b>3</b>–<b>7</b>, <b>10</b>–<b>12</b>, <b>14</b>–<b>18</b>, and <b>21</b>–<b>36</b> (a) (Het)ArCOOH, HBTU, DIPEA, DCM, 1 h, 15–95%; for compounds <b>8</b>, <b>9</b>, and <b>13</b> (b) (Het)ArCOCl, DIPEA, DCM, RT, 1 h, 11–80%.</p></p></figure><figure data-id="sch8" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Scheme 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/medium/jm0c01272_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0016.jpeg" id="rightTab-gr15" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 8. Synthesis of Compounds <b>37</b>–<b>40</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01272&amp;id=sch8"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NaBH(OAc)<sub>3</sub>, DIPEA, DCM, RT, 4 h (36%–86%); (b) DIPEA, DMF, 60 °C, 4 h (37%).</p></p></figure><figure data-id="sch9" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Scheme 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/medium/jm0c01272_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0017.jpeg" id="rightTab-gr16" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 9. Synthesis of Compounds <b>41</b>–<b>43</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01272/20201105/images/large/jm0c01272_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01272&amp;id=sch9"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) H<sub>2</sub>, PtO<sub>2</sub>, AcOH, 50 °C, 5 d, 97%; (b) Boc<sub>2</sub>O, DCM, DIPEA, DMAP, RT, 1 h, 75%; (c) NaOH, MeOH, RT, 14 h, 100%; (d) CDI, MeNHOMe, Et<sub>3</sub>N, THF, ACN, RT—80 °C, 7 d,100%; (e) MeMgBr, THF, −15 °C—RT, 1 h, 90%; (f) KOtBu, toluene, 0 °C—RT, 2 h, 100%; (g) (i) <b>53</b>, DMF, 80 °C, 24 h; (ii) Boc<sub>2</sub>O, Et<sub>3</sub>N, RT, 30 min, 30%; (h) HCl, iPrOH, MeOH, 2 h, RT (72%); (i) (Het)ArCOOH, HBTU, DIPEA, DCM, 1 h, 15–95%; (j) (Het)Cl or (Het)Br, DIPEA, <i>n</i>BuOH, 160 °C, 4 h (6–41%).</p></p></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i131">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_03872" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_03872" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 55 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maurice, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ke, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahmad, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manganiello, V. C.</span></span> <span> </span><span class="NLM_article-title">Advances in targeting cyclic nucleotide phosphodiesterases</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">290</span>– <span class="NLM_lpage">314</span>, <span class="refDoi"> DOI: 10.1038/nrd4228</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.1038%2Fnrd4228" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=24687066" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC2cXltFOiu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=290-314&author=D.+H.+Mauriceauthor=H.+Keauthor=F.+Ahmadauthor=Y.+Wangauthor=J.+Chungauthor=V.+C.+Manganiello&title=Advances+in+targeting+cyclic+nucleotide+phosphodiesterases&doi=10.1038%2Fnrd4228"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Advances in targeting cyclic nucleotide phosphodiesterases</span></div><div class="casAuthors">Maurice, Donald H.; Ke, Hengming; Ahmad, Faiyaz; Wang, Yousheng; Chung, Jay; Manganiello, Vincent C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">290-314</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Cyclic nucleotide phosphodiesterases (PDEs) catalyze the hydrolysis of cAMP and cyclic GMP, thereby regulating the intracellular concns. of these cyclic nucleotides, their signalling pathways and, consequently, myriad biol. responses in health and disease.  Currently, a small no. of PDE inhibitors are used clin. for treating the pathophysiol. dysregulation of cyclic nucleotide signalling in several disorders, including erectile dysfunction, pulmonary hypertension, acute refractory cardiac failure, intermittent claudication and chronic obstructive pulmonary disease.  However, pharmaceutical interest in PDEs has been reignited by the increasing understanding of the roles of individual PDEs in regulating the subcellular compartmentalization of specific cyclic nucleotide signalling pathways, by the structure-based design of novel specific inhibitors and by the development of more sophisticated strategies to target individual PDE variants.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwMRfUs-zUmLVg90H21EOLACvtfcHk0lhRjxI8PMCtpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXltFOiu7s%253D&md5=410c8d5bb5fbd7d2906e30dc1b5f2876</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd4228&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4228%26sid%3Dliteratum%253Aachs%26aulast%3DMaurice%26aufirst%3DD.%2BH.%26aulast%3DKe%26aufirst%3DH.%26aulast%3DAhmad%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DChung%26aufirst%3DJ.%26aulast%3DManganiello%26aufirst%3DV.%2BC.%26atitle%3DAdvances%2520in%2520targeting%2520cyclic%2520nucleotide%2520phosphodiesterases%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D290%26epage%3D314%26doi%3D10.1038%2Fnrd4228" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Francis, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blount, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corbin, J. D.</span></span> <span> </span><span class="NLM_article-title">Mammalian cyclic nucleotide phosphodiesterases: molecular mechanisms and physiological functions</span>. <i>Physiol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>91</i></span>,  <span class="NLM_fpage">651</span>– <span class="NLM_lpage">690</span>, <span class="refDoi"> DOI: 10.1152/physrev.00030.2010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.1152%2Fphysrev.00030.2010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=21527734" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC3MXns1Cms7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=91&publication_year=2011&pages=651-690&author=S.+H.+Francisauthor=M.+A.+Blountauthor=J.+D.+Corbin&title=Mammalian+cyclic+nucleotide+phosphodiesterases%3A+molecular+mechanisms+and+physiological+functions&doi=10.1152%2Fphysrev.00030.2010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Mammalian cyclic nucleotide phosphodiesterases: molecular mechanisms and physiological functions</span></div><div class="casAuthors">Francis, Sharron H.; Blount, Mitsi A.; Corbin, Jackie D.</div><div class="citationInfo"><span class="NLM_cas:title">Physiological Reviews</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">91</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">651-690</span>CODEN:
                <span class="NLM_cas:coden">PHREA7</span>;
        ISSN:<span class="NLM_cas:issn">0031-9333</span>.
    
            (<span class="NLM_cas:orgname">American Physiological Society</span>)
        </div><div class="casAbstract">A review.  The superfamily of cyclic nucleotide (cN) phosphodiesterases (PDEs) is comprised of 11 families of enzymes.  PDEs break down cAMP and/or cGMP and are major determinants of cellular cN levels and, consequently, the actions of cN-signaling pathways.  PDEs exhibit a range of catalytic efficiencies for breakdown of cAMP and/or cGMP and are regulated by myriad processes including phosphorylation, cN binding to allosteric GAF domains, changes in expression levels, interaction with regulatory or anchoring proteins, and reversible translocation among subcellular compartments.  Selective PDE inhibitors are currently in clin. use for treatment of erectile dysfunction, pulmonary hypertension, intermittent claudication, and chronic pulmonary obstructive disease; many new inhibitors are being developed for treatment of these and other maladies.  Recently reported x-ray crystallog. structures have defined features that provide for specificity for cAMP or cGMP in PDE catalytic sites or their GAF domains, as well as mechanisms involved in catalysis, oligomerization, autoinhibition, and interactions with inhibitors.  In addn., major advances have been made in understanding the physiol. impact and the biochem. basis for selective localization and/or recruitment of specific PDE isoenzymes to particular subcellular compartments.  The many recent advances in understanding PDE structures, functions, and physiol. actions are discussed in this review.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrN99wWYsCXRbVg90H21EOLACvtfcHk0lhEGGHPUn_BeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXns1Cms7o%253D&md5=4aae757704a6243c9aa2648c32d5c759</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1152%2Fphysrev.00030.2010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1152%252Fphysrev.00030.2010%26sid%3Dliteratum%253Aachs%26aulast%3DFrancis%26aufirst%3DS.%2BH.%26aulast%3DBlount%26aufirst%3DM.%2BA.%26aulast%3DCorbin%26aufirst%3DJ.%2BD.%26atitle%3DMammalian%2520cyclic%2520nucleotide%2520phosphodiesterases%253A%2520molecular%2520mechanisms%2520and%2520physiological%2520functions%26jtitle%3DPhysiol.%2520Rev.%26date%3D2011%26volume%3D91%26spage%3D651%26epage%3D690%26doi%3D10.1152%2Fphysrev.00030.2010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pandit, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forman, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fennell, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dillman, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menniti, F. S.</span></span> <span> </span><span class="NLM_article-title">Mechanism for the allosteric regulation of phosphodiesterase 2A deduced from the X-ray structure of a near full-length construct</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">18225</span>– <span class="NLM_lpage">18230</span>, <span class="refDoi"> DOI: 10.1073/pnas.0907635106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.1073%2Fpnas.0907635106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=19828435" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVags7vO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=18225-18230&author=J.+Panditauthor=M.+D.+Formanauthor=K.+F.+Fennellauthor=K.+S.+Dillmanauthor=F.+S.+Menniti&title=Mechanism+for+the+allosteric+regulation+of+phosphodiesterase+2A+deduced+from+the+X-ray+structure+of+a+near+full-length+construct&doi=10.1073%2Fpnas.0907635106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism for the allosteric regulation of phosphodiesterase 2A deduced from the X-ray structure of a near full-length construct</span></div><div class="casAuthors">Pandit, Jayvardhan; Forman, Michael D.; Fennell, Kimberly F.; Dillman, Keith S.; Menniti, Frank S.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">18225-18230, S18225/1-S18225/8</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">We report the x-ray crystal structure of a phosphodiesterase (PDE) that includes both catalytic and regulatory domains.  PDE2A (215-900) crystd. as a dimer in which each subunit had an extended organization of regulatory GAF-A and GAF-B and catalytic domains connected by long α-helixes.  The subunits cross at the GAF-B/catalytic domain linker, and each side of the dimer contains in series the GAF-A and GAF-B of one subunit and the catalytic domain of the other subunit.  A dimer interface extends over the entire length of the mol.  The substrate binding pocket of each catalytic domain is occluded by the H-loop.  We deduced from comparisons with structures of isolated, ligand-bound catalytic subunits that the H-loop swings out to allow substrate access.  However, in dimeric PDE2A (215-900), the H-loops of the two catalytic subunits pack against each other at the dimer interface, necessitating movement of the catalytic subunits to allow for H-loop movement.  Comparison of the unliganded GAF-B of PDE2A (215-900) with previous structures of isolated, cGMP-bound GAF domains indicates that cGMP binding induces a significant shift in the GAF-B/catalytic domain linker.  We propose that cGMP binding to GAF-B causes movement, through this linker region, of the catalytic domains, such that the H-loops no longer pack at the dimer interface and are, instead, free to swing out to allow substrate access.  This increase in substrate access is proposed as the basis for PDE2A activation by cGMP and may be a general mechanism for regulation of all PDEs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqh-kgJc6VX2bVg90H21EOLACvtfcHk0lhEGGHPUn_BeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVags7vO&md5=73ddfdb3ffb861674744a1fbeef7c881</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0907635106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0907635106%26sid%3Dliteratum%253Aachs%26aulast%3DPandit%26aufirst%3DJ.%26aulast%3DForman%26aufirst%3DM.%2BD.%26aulast%3DFennell%26aufirst%3DK.%2BF.%26aulast%3DDillman%26aufirst%3DK.%2BS.%26aulast%3DMenniti%26aufirst%3DF.%2BS.%26atitle%3DMechanism%2520for%2520the%2520allosteric%2520regulation%2520of%2520phosphodiesterase%25202A%2520deduced%2520from%2520the%2520X-ray%2520structure%2520of%2520a%2520near%2520full-length%2520construct%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2009%26volume%3D106%26spage%3D18225%26epage%3D18230%26doi%3D10.1073%2Fpnas.0907635106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, A. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, X.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ikeda, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beavo, J. A.</span></span> <span> </span><span class="NLM_article-title">Molecular determinants for cyclic nucleotide binding to the regulatory domains of phosphodiesterase 2A</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>279</i></span>,  <span class="NLM_fpage">37928</span>– <span class="NLM_lpage">37938</span>, <span class="refDoi"> DOI: 10.1074/jbc.m404287200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.1074%2Fjbc.M404287200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=15210692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BD2cXntFSntrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=279&publication_year=2004&pages=37928-37938&author=A.+Y.+Wuauthor=X.-B.+Tangauthor=S.+E.+Martinezauthor=K.+Ikedaauthor=J.+A.+Beavo&title=Molecular+determinants+for+cyclic+nucleotide+binding+to+the+regulatory+domains+of+phosphodiesterase+2A&doi=10.1074%2Fjbc.m404287200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Determinants for Cyclic Nucleotide Binding to the Regulatory Domains of Phosphodiesterase 2A</span></div><div class="casAuthors">Wu, Albert Y.; Tang, Xiao-Bo; Martinez, Sergio E.; Ikeda, Kaori; Beavo, Joseph A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">279</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">37928-37938</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Binding of cGMP to the GAF-B domain of phosphodiesterase 2A allosterically activates catalytic activity.  We report here a series of mutagenesis studies on the GAF-B domain of PDE2A that support a novel mechanism for mol. recognition of cGMP.  Alanine mutations of Phe-438, Asp-439, and Thr-488, amino acids that interact with the pyrimidine ring, decrease cGMP affinity slightly but increase cAMP affinity by up to 8-fold.  Each interaction is required to provide for cAMP/cGMP specificity.  Mutations of any of the residues that interact with the phosphate-ribose moiety or the imidazole ring abolish cGMP binding.  Thus, residues that interact with the pyrimidine ring collectively control cAMP/cGMP specificity, whereas residues that bind the phosphate-ribose moiety and imidazole ring are crit. for high affinity binding.  Similar decreases in binding were found for mutations made in a bacterially expressed GAF-A/B plus catalytic domain construct.  Because these constructs had very high catalytic activity, it appears that these mutations did not cause a global denaturation.  The affinities of cAMP and cGMP for wild-type GAF-B alone were ∼4-fold greater than for the holoenzyme, suggesting that the presence of neighboring domains alters the conformation of GAF-B.  More importantly, the PDE2A GAF-B, GAF-A/B, GAF-A/B+C domains, and holoenzyme all bind cGMP with much higher affinity than has previously been reported.  This high affinity suggests that cGMP binding to PDE2 GAF-B activates the enzyme rapidly, stoichiometrically, and in an all or none fashion, rather than variably over a large range of cyclic nucleotide concns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUnZDzjQ-ZNLVg90H21EOLACvtfcHk0lhEGGHPUn_BeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXntFSntrs%253D&md5=e1b95613e37644fd8e87c8553db84491</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M404287200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M404287200%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DA.%2BY.%26aulast%3DTang%26aufirst%3DX.-B.%26aulast%3DMartinez%26aufirst%3DS.%2BE.%26aulast%3DIkeda%26aufirst%3DK.%26aulast%3DBeavo%26aufirst%3DJ.%2BA.%26atitle%3DMolecular%2520determinants%2520for%2520cyclic%2520nucleotide%2520binding%2520to%2520the%2520regulatory%2520domains%2520of%2520phosphodiesterase%25202A%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2004%26volume%3D279%26spage%3D37928%26epage%3D37938%26doi%3D10.1074%2Fjbc.m404287200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Erneux, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Couchie, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumont, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baraniak, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stec, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abbad, E. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petridis, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jastorff, B.</span></span> <span> </span><span class="NLM_article-title">Specificity of cyclic GMP activation of a multi-substrate cyclic nucleotide phosphodiesterase from rat liver</span>. <i>Eur. J. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">1981</span>,  <span class="NLM_volume"><i>115</i></span>,  <span class="NLM_fpage">503</span>– <span class="NLM_lpage">510</span>, <span class="refDoi"> DOI: 10.1111/j.1432-1033.1981.tb06231.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.1111%2Fj.1432-1033.1981.tb06231.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=6263632" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADyaL3MXktVamsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=1981&pages=503-510&author=C.+Erneuxauthor=D.+Couchieauthor=J.+E.+Dumontauthor=J.+Baraniakauthor=W.+J.+Stecauthor=E.+G.+Abbadauthor=G.+Petridisauthor=B.+Jastorff&title=Specificity+of+cyclic+GMP+activation+of+a+multi-substrate+cyclic+nucleotide+phosphodiesterase+from+rat+liver&doi=10.1111%2Fj.1432-1033.1981.tb06231.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Specificity of cyclic GMP activation of a multi-substrate cyclic nucleotide phosphodiesterase from rat liver</span></div><div class="casAuthors">Erneux, Christophe; Couchie, Dominique; Dumont, Jacques E.; Baraniak, Janina; Stec, Woljciech J.; Abbad, Emilio Garcia; Petridis, Georg; Jastorff, Bernd</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Biochemistry</span>
        (<span class="NLM_cas:date">1981</span>),
    <span class="NLM_cas:volume">115</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">503-10</span>CODEN:
                <span class="NLM_cas:coden">EJBCAI</span>;
        ISSN:<span class="NLM_cas:issn">0014-2956</span>.
    </div><div class="casAbstract">Cyclic nucleotide derivs. have been used as a tool to investigate the existence of distinctive activating and hydrolytic sites on the cyclic nucleotide phosphodiesterase from rat liver activated by cGMP.  This pos. cooperative enzyme was stimulated up to 30-fold by 3 μM cGMP when 3 μM cAMP was used as substrate.  All analogs were less potent activators than cGMP.  Most cAMP derivs. were inactive, with 2 exceptions, 7-deazaadenosine 3',5'-monophosphate and 3'-amino-3'-deoxyadenosine 3',5'-monophosphate.  Benzimidazole ribonucleoside 3',5'-monophosphate, where the 2 N atoms of the pyrimidine ring are missing, was a better stimulator than the intact purine-related cyclic deriv.  When cAMP and cGMP with identical chem. ligands substituted at the same position were compared, the cGMP analog was always the more potent activator suggesting that the activating site is sensitive to a guanine-type cyclic nucleotide structure.  Degrdn. of the derivs. by the enzyme was measured by high-performance liq. chromatog.; no relation could be established between hydrolysis and effectiveness of activation.  In addn., there was no parallelism between inhibitory and activating potency for 10 cyclic nucleotide derivs.  Since the chem. interactions between the analogs at the activating site on the one hand and at the catalytic site on the other, are different, it is proposed that the sites are distinct.  Consequently, it is suggested that the enzyme operates in steps.  In the 1st activating step, cGMP is fixed by at least 2 H-bonds at a specific binding site of the enzyme.  This is followed by a conformational change of the protein and subsequently a change of the kinetic parameters.  In a rather unspecific process and in a 2nd hydrolytic step, several purine-related cyclic nucleotides are converted to the corresponding 5'-nucleotides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-pwbda4flLbVg90H21EOLACvtfcHk0ljUIMAV2HCx_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3MXktVamsrY%253D&md5=c914a884a8486cb57531212c0d93641c</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1111%2Fj.1432-1033.1981.tb06231.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1432-1033.1981.tb06231.x%26sid%3Dliteratum%253Aachs%26aulast%3DErneux%26aufirst%3DC.%26aulast%3DCouchie%26aufirst%3DD.%26aulast%3DDumont%26aufirst%3DJ.%2BE.%26aulast%3DBaraniak%26aufirst%3DJ.%26aulast%3DStec%26aufirst%3DW.%2BJ.%26aulast%3DAbbad%26aufirst%3DE.%2BG.%26aulast%3DPetridis%26aufirst%3DG.%26aulast%3DJastorff%26aufirst%3DB.%26atitle%3DSpecificity%2520of%2520cyclic%2520GMP%2520activation%2520of%2520a%2520multi-substrate%2520cyclic%2520nucleotide%2520phosphodiesterase%2520from%2520rat%2520liver%26jtitle%3DEur.%2520J.%2520Biochem.%26date%3D1981%26volume%3D115%26spage%3D503%26epage%3D510%26doi%3D10.1111%2Fj.1432-1033.1981.tb06231.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lakics, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karran, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boess, F. G.</span></span> <span> </span><span class="NLM_article-title">Quantitative comparison of phosphodiesterase mRNA distribution in human brain and peripheral tissues</span>. <i>Neuropharmacology</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">367</span>– <span class="NLM_lpage">374</span>, <span class="refDoi"> DOI: 10.1016/j.neuropharm.2010.05.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.1016%2Fj.neuropharm.2010.05.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=20493887" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1SqsLbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2010&pages=367-374&author=V.+Lakicsauthor=E.+H.+Karranauthor=F.+G.+Boess&title=Quantitative+comparison+of+phosphodiesterase+mRNA+distribution+in+human+brain+and+peripheral+tissues&doi=10.1016%2Fj.neuropharm.2010.05.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative comparison of phosphodiesterase mRNA distribution in human brain and peripheral tissues</span></div><div class="casAuthors">Lakics, Viktor; Karran, Eric H.; Boess, Frank G.</div><div class="citationInfo"><span class="NLM_cas:title">Neuropharmacology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">367-374</span>CODEN:
                <span class="NLM_cas:coden">NEPHBW</span>;
        ISSN:<span class="NLM_cas:issn">0028-3908</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Cyclic nucleotide-specific phosphodiesterases (PDEs) play a crit. role in signal transduction by regulating the level of adenosine 3',5'-cyclic monophosphate (cAMP) and guanosine 3',5'-cyclic monophosphate (cGMP) in cells.  The gene expression pattern of a PDE provides important information regarding its role in physiol. and pathol. processes.  In this study, we have established the mRNA expression profile all PDE isoenzymes (PDE1A/B/C, 2A, 3A/B, 4A/B/C/D, 5A, 6A/B/C, 7A/B, 8A/B, 9A, 10A, 11A) in a human cDNA collection consisting of 10 brain regions (parietal, frontal, temporal cortex, hippocampus, striatum, thalamus, hypothalamus, substantia nigra, nucleus accumbens, cerebellum), spinal cord, dorsal root ganglia and 12 peripheral tissues (skeletal muscle, heart, thyroid, adrenal gland, pancreas, bladder, kidney, liver, lung, small intestine, spleen, and stomach).  Using quant. real-time polymerase chain reaction and parallel anal. of a carefully selected group of ref. genes, we have detd. the relative expression of each PDE isoenzyme across the 24 selected tissues, and also compared the expression of selected PDEs to each other within a given tissue type.  Several PDEs show strikingly selective expression (e.g. PDE10A and PDE1B mRNA levels in the caudate nucleus are 20-fold higher than in most other tissues; PDE1C and PDE3A are highly expressed in the heart and PDE8B is expressed very strongly in the thyroid gland).  This comprehensive approach provides a coherent and quant. view of the mRNA expression of the PDE gene family and enables an integration of data obtained with other non-quant. methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprGz6OEkRjbLVg90H21EOLACvtfcHk0ljUIMAV2HCx_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1SqsLbJ&md5=ee55772f572818eaa7194a1ae3f52f45</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.neuropharm.2010.05.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuropharm.2010.05.004%26sid%3Dliteratum%253Aachs%26aulast%3DLakics%26aufirst%3DV.%26aulast%3DKarran%26aufirst%3DE.%2BH.%26aulast%3DBoess%26aufirst%3DF.%2BG.%26atitle%3DQuantitative%2520comparison%2520of%2520phosphodiesterase%2520mRNA%2520distribution%2520in%2520human%2520brain%2520and%2520peripheral%2520tissues%26jtitle%3DNeuropharmacology%26date%3D2010%26volume%3D59%26spage%3D367%26epage%3D374%26doi%3D10.1016%2Fj.neuropharm.2010.05.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stephenson, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coskran, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilhelms, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adamowicz, W. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Donnell, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muravnick, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menniti, F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kleiman, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morton, D.</span></span> <span> </span><span class="NLM_article-title">Immunohistochemical localization of phosphodiesterase 2A in multiple mammalian species</span>. <i>J. Histochem. Cytochem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">933</span>– <span class="NLM_lpage">949</span>, <span class="refDoi"> DOI: 10.1369/jhc.2009.953471</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.1369%2Fjhc.2009.953471" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=19506089" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFOgtrrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2009&pages=933-949&author=D.+T.+Stephensonauthor=T.+M.+Coskranauthor=M.+B.+Wilhelmsauthor=W.+O.+Adamowiczauthor=M.+M.+O%E2%80%99Donnellauthor=K.+B.+Muravnickauthor=F.+S.+Mennitiauthor=R.+J.+Kleimanauthor=D.+Morton&title=Immunohistochemical+localization+of+phosphodiesterase+2A+in+multiple+mammalian+species&doi=10.1369%2Fjhc.2009.953471"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Immunohistochemical localization of phosphodiesterase 2A in multiple mammalian species</span></div><div class="casAuthors">Stephenson, Diane T.; Coskran, Tim M.; Wilhelms, Margaret B.; Adamowicz, Wendy O.; O'Donnell, Michele M.; Muravnick, Kathleen B.; Menniti, Frank S.; Kleiman, Robin J.; Morton, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Histochemistry and Cytochemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">933-949</span>CODEN:
                <span class="NLM_cas:coden">JHCYAS</span>;
        ISSN:<span class="NLM_cas:issn">0022-1554</span>.
    
            (<span class="NLM_cas:orgname">Histochemical Society, Inc.</span>)
        </div><div class="casAbstract">Phosphodiesterases (PDEs) comprise a family of enzymes that regulate the levels of cyclic nucleotides, key second messengers that mediate a diverse array of functions.  PDE2A is an evolutionarily conserved cGMP-stimulated cAMP and cGMP PDE.  In the present study, the regional and cellular distribution of PDE2A in tissues of rats, mice, cynomolgus monkeys, dogs, and humans was evaluated by immunohistochem.  A polyclonal antibody directed to the C-terminal portion of PDE2A specifically detected PDE2A by Western blotting and by immunohistochem.  The pattern of PDE2A immunoreactivity (ir) was consistent across all species.  Western blot anal. demonstrated that PDE2A was most abundant in the brain relative to peripheral tissues.  PDE2A ir was heterogeneously distributed within brain and was selectively expressed in particular peripheral tissues.  In the brain, prominent immunoreactivity was apparent in components of the limbic system, including the isocortex, hippocampus, amygdala, habenula, basal ganglia, and interpeduncular nucleus.  Cytoplasmic PDE2A staining was prominent in several peripheral tissues, including the adrenal zona glomerulosa, neurons of enteric ganglia, endothelial cells in all organs, lymphocytes of spleen and lymph nodes, and pituitary.  These studies suggest that PDE2A is evolutionarily conserved across mammalian species and support the hypothesis that the enzyme plays a fundamental role in signal transduction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHVQCCm4loO7Vg90H21EOLACvtfcHk0ljUIMAV2HCx_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFOgtrrJ&md5=ce1dfa99cc99c02f31ab8071a9e438fd</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1369%2Fjhc.2009.953471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1369%252Fjhc.2009.953471%26sid%3Dliteratum%253Aachs%26aulast%3DStephenson%26aufirst%3DD.%2BT.%26aulast%3DCoskran%26aufirst%3DT.%2BM.%26aulast%3DWilhelms%26aufirst%3DM.%2BB.%26aulast%3DAdamowicz%26aufirst%3DW.%2BO.%26aulast%3DO%25E2%2580%2599Donnell%26aufirst%3DM.%2BM.%26aulast%3DMuravnick%26aufirst%3DK.%2BB.%26aulast%3DMenniti%26aufirst%3DF.%2BS.%26aulast%3DKleiman%26aufirst%3DR.%2BJ.%26aulast%3DMorton%26aufirst%3DD.%26atitle%3DImmunohistochemical%2520localization%2520of%2520phosphodiesterase%25202A%2520in%2520multiple%2520mammalian%2520species%26jtitle%3DJ.%2520Histochem.%2520Cytochem.%26date%3D2009%26volume%3D57%26spage%3D933%26epage%3D949%26doi%3D10.1369%2Fjhc.2009.953471" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stephenson, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coskran, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kleiman, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morton, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Neill, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinberg, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menniti, F. S.</span></span> <span> </span><span class="NLM_article-title">The distribution of phosphodiesterase 2A in the rat brain</span>. <i>Neuroscience</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>226</i></span>,  <span class="NLM_fpage">145</span>– <span class="NLM_lpage">155</span>, <span class="refDoi"> DOI: 10.1016/j.neuroscience.2012.09.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.1016%2Fj.neuroscience.2012.09.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=23000621" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1ahur7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=226&publication_year=2012&pages=145-155&author=D.+T.+Stephensonauthor=T.+M.+Coskranauthor=M.+P.+Kellyauthor=R.+J.+Kleimanauthor=D.+Mortonauthor=S.+M.+O%E2%80%99Neillauthor=C.+J.+Schmidtauthor=R.+J.+Weinbergauthor=F.+S.+Menniti&title=The+distribution+of+phosphodiesterase+2A+in+the+rat+brain&doi=10.1016%2Fj.neuroscience.2012.09.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">The distribution of phosphodiesterase 2A in the rat brain</span></div><div class="casAuthors">Stephenson, D. T.; Coskran, T. M.; Kelly, M. P.; Kleiman, R. J.; Morton, D.; O'Neill, S. M.; Schmidt, C. J.; Weinberg, R. J.; Menniti, F. S.</div><div class="citationInfo"><span class="NLM_cas:title">Neuroscience (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">226</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">145-155</span>CODEN:
                <span class="NLM_cas:coden">NRSCDN</span>;
        ISSN:<span class="NLM_cas:issn">0306-4522</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The phosphodiesterases (PDEs) are a superfamily of enzymes that regulate spatio-temporal signaling by the intracellular second messengers cAMP and cGMP.  PDE2A is expressed at high levels in the mammalian brain.  To advance our understanding of the role of this enzyme in regulation of neuronal signaling, we here describe the distribution of PDE2A in the rat brain.  PDE2A mRNA was prominently expressed in glutamatergic pyramidal cells in cortex, and in pyramidal and dentate granule cells in the hippocampus.  Protein concd. in the axons and nerve terminals of these neurons; staining was markedly weaker in the cell bodies and proximal dendrites.  In addn., in both hippocampus and cortex, small populations of non-pyramidal cells, presumed to be interneurons, were strongly immunoreactive.  PDE2A mRNA was expressed in medium spiny neurons in neostriatum.  Little immunoreactivity was obsd. in cell bodies, whereas dense immunoreactivity was found in the axon tracts of these neurons and their terminal regions in globus pallidus and substantia nigra pars reticulata.  Immunostaining was dense in the medial habenula, but weak in other diencephalic regions.  In midbrain and hindbrain, immunostaining was restricted to discrete regions of the neuropil or clusters of cell bodies.  These results suggest that PDE2A may modulate cortical, hippocampal and striatal networks at several levels.  Preferential distribution of PDE2A into axons and terminals of the principal neurons suggests roles in regulation of axonal excitability or transmitter release.  The enzyme is also in forebrain interneurons, and in mid- and hindbrain neurons that may modulate forebrain networks and circuits.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxNSYsU1HXSrVg90H21EOLACvtfcHk0lghOD4zscecWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1ahur7F&md5=3bb5ae07b9fcae2a917cc4aed4fef4a1</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.neuroscience.2012.09.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuroscience.2012.09.011%26sid%3Dliteratum%253Aachs%26aulast%3DStephenson%26aufirst%3DD.%2BT.%26aulast%3DCoskran%26aufirst%3DT.%2BM.%26aulast%3DKelly%26aufirst%3DM.%2BP.%26aulast%3DKleiman%26aufirst%3DR.%2BJ.%26aulast%3DMorton%26aufirst%3DD.%26aulast%3DO%25E2%2580%2599Neill%26aufirst%3DS.%2BM.%26aulast%3DSchmidt%26aufirst%3DC.%2BJ.%26aulast%3DWeinberg%26aufirst%3DR.%2BJ.%26aulast%3DMenniti%26aufirst%3DF.%2BS.%26atitle%3DThe%2520distribution%2520of%2520phosphodiesterase%25202A%2520in%2520the%2520rat%2520brain%26jtitle%3DNeuroscience%26date%3D2012%26volume%3D226%26spage%3D145%26epage%3D155%26doi%3D10.1016%2Fj.neuroscience.2012.09.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blokland, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menniti, F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prickaerts, J.</span></span> <span> </span><span class="NLM_article-title">PDE Inhibition and cognition enhancement</span>. <i>Expert Opin. Ther. Pat.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">349</span>– <span class="NLM_lpage">354</span>, <span class="refDoi"> DOI: 10.1517/13543776.2012.674514</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.1517%2F13543776.2012.674514" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=22475506" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC38XltVKis70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=349-354&author=A.+Bloklandauthor=F.+S.+Mennitiauthor=J.+Prickaerts&title=PDE+Inhibition+and+cognition+enhancement&doi=10.1517%2F13543776.2012.674514"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">PDE Inhibition and cognition enhancement</span></div><div class="casAuthors">Blokland, Arjan; Menniti, Frank S.; Prickaerts, Jos</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Patents</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">349-354</span>CODEN:
                <span class="NLM_cas:coden">EOTPEG</span>;
        ISSN:<span class="NLM_cas:issn">1354-3776</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  There has been an increasing interest in the development of phosphodiesterase (PDE) inhibitors for the treatment of cognitive dysfunctions.  In this editorial, the mechanism of action of PDEs is briefly described, while the effects of different PDE inhibitors in preclin. models are reviewed.  Based on the expression of PDE mRNA in the human brain, it is suggested that PDE1 and PDE10 inhibitors are strong candidates for the development of cognition enhancers.  However, the complex nature of the expression of PDEs in the brain warrants further research into the role of PDEs in the signaling pathways in brain circuits.  The development of PDE inhibitors, which are selective for PDE splicing isoforms, may be promising for future drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreP8gLClAGx7Vg90H21EOLACvtfcHk0lghOD4zscecWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XltVKis70%253D&md5=368b04ba842f031cc8c67ac55efa1b4b</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1517%2F13543776.2012.674514&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F13543776.2012.674514%26sid%3Dliteratum%253Aachs%26aulast%3DBlokland%26aufirst%3DA.%26aulast%3DMenniti%26aufirst%3DF.%2BS.%26aulast%3DPrickaerts%26aufirst%3DJ.%26atitle%3DPDE%2520Inhibition%2520and%2520cognition%2520enhancement%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Pat.%26date%3D2012%26volume%3D22%26spage%3D349%26epage%3D354%26doi%3D10.1517%2F13543776.2012.674514" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bliss, T. V. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collingridge, G. L.</span></span> <span> </span><span class="NLM_article-title">A synaptic model of memory: long-term potentiation in the hippocampus</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>361</i></span>,  <span class="NLM_fpage">31</span>– <span class="NLM_lpage">39</span>, <span class="refDoi"> DOI: 10.1038/361031a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.1038%2F361031a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=8421494" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADyaK3sXntlOntw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=361&publication_year=1993&pages=31-39&author=T.+V.+P.+Blissauthor=G.+L.+Collingridge&title=A+synaptic+model+of+memory%3A+long-term+potentiation+in+the+hippocampus&doi=10.1038%2F361031a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">A synaptic model of memory:  long-term potentiation in the hippocampus</span></div><div class="casAuthors">Bliss, T. V. P.; Collingridge, G. L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">361</span>
        (<span class="NLM_cas:issue">6407</span>),
    <span class="NLM_cas:pages">31-9</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">A review, with 174 refs., of long-term potentiation (LTP) of synaptic transmission in the hippocampus as the primary exptl. model for investigating the synaptic basis of learning and memory in vertebrates.  Topics discussed were: properties of hippocampal LTP; induction of LTP; and expression of LTP.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoD1tHZif2aP7Vg90H21EOLACvtfcHk0lghOD4zscecWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXntlOntw%253D%253D&md5=285f5a175da5a77f4f57a3f5236e3e0a</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2F361031a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F361031a0%26sid%3Dliteratum%253Aachs%26aulast%3DBliss%26aufirst%3DT.%2BV.%2BP.%26aulast%3DCollingridge%26aufirst%3DG.%2BL.%26atitle%3DA%2520synaptic%2520model%2520of%2520memory%253A%2520long-term%2520potentiation%2520in%2520the%2520hippocampus%26jtitle%3DNature%26date%3D1993%26volume%3D361%26spage%3D31%26epage%3D39%26doi%3D10.1038%2F361031a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bollen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puzzo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rutten, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Privitera, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Vry, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vanmierlo, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kenis, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmeri, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Hooge, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balschun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinbusch, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blokland, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prickaerts, J.</span></span> <span> </span><span class="NLM_article-title">Improved long-term memory via enhancing cGMP-PKG signaling requires cAMP-PKA signaling</span>. <i>Neuropsychopharmacology</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">2497</span>– <span class="NLM_lpage">2505</span>, <span class="refDoi"> DOI: 10.1038/npp.2014.106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.1038%2Fnpp.2014.106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=24813825" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVKls7zF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2014&pages=2497-2505&author=E.+Bollenauthor=D.+Puzzoauthor=K.+Ruttenauthor=L.+Priviteraauthor=J.+De+Vryauthor=T.+Vanmierloauthor=G.+Kenisauthor=A.+Palmeriauthor=R.+D%E2%80%99Hoogeauthor=D.+Balschunauthor=H.+M.+Steinbuschauthor=A.+Bloklandauthor=J.+Prickaerts&title=Improved+long-term+memory+via+enhancing+cGMP-PKG+signaling+requires+cAMP-PKA+signaling&doi=10.1038%2Fnpp.2014.106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Improved Long-Term Memory via Enhancing cGMP-PKG Signaling Requires cAMP-PKA Signaling</span></div><div class="casAuthors">Bollen, Eva; Puzzo, Daniela; Rutten, Kris; Privitera, Lucia; De Vry, Jochen; Vanmierlo, Tim; Kenis, Gunter; Palmeri, Agostino; D'Hooge, Rudi; Balschun, Detlef; Steinbusch, Harry M. W.; Blokland, Arjan; Prickaerts, Jos</div><div class="citationInfo"><span class="NLM_cas:title">Neuropsychopharmacology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2497-2505</span>CODEN:
                <span class="NLM_cas:coden">NEROEW</span>;
        ISSN:<span class="NLM_cas:issn">0893-133X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Memory consolidation is defined by the stabilization of a memory trace after acquisition, and consists of numerous mol. cascades that mediate synaptic plasticity.  Commonly, a distinction is made between an early and a late consolidation phase, in which early refers to the first hours in which labile synaptic changes occur, whereas late consolidation relates to stable and long-lasting synaptic changes induced by de novo protein synthesis.  How these phases are linked at a mol. level is not yet clear.  Here we studied the interaction of the cyclic nucleotide-mediated pathways during the different phases of memory consolidation in rodents.  In addn., the same pathways were studied in a model of neuronal plasticity, long-term potentiation (LTP).  We demonstrated that cGMP/protein kinase G (PKG) signaling mediates early memory consolidation as well as early-phase LTP, whereas cAMP/protein kinase A (PKA) signaling mediates late consolidation and late-phase-like LTP.  In addn., we show for the first time that early-phase cGMP/PKG signaling requires late-phase cAMP/PKA-signaling in both LTP and long-term memory formation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2NJct-ZSPAbVg90H21EOLACvtfcHk0lh58Zg4KXpKGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVKls7zF&md5=d8dc4c85e501b91bc918950d7dbf0f1f</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnpp.2014.106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnpp.2014.106%26sid%3Dliteratum%253Aachs%26aulast%3DBollen%26aufirst%3DE.%26aulast%3DPuzzo%26aufirst%3DD.%26aulast%3DRutten%26aufirst%3DK.%26aulast%3DPrivitera%26aufirst%3DL.%26aulast%3DDe%2BVry%26aufirst%3DJ.%26aulast%3DVanmierlo%26aufirst%3DT.%26aulast%3DKenis%26aufirst%3DG.%26aulast%3DPalmeri%26aufirst%3DA.%26aulast%3DD%25E2%2580%2599Hooge%26aufirst%3DR.%26aulast%3DBalschun%26aufirst%3DD.%26aulast%3DSteinbusch%26aufirst%3DH.%2BM.%26aulast%3DBlokland%26aufirst%3DA.%26aulast%3DPrickaerts%26aufirst%3DJ.%26atitle%3DImproved%2520long-term%2520memory%2520via%2520enhancing%2520cGMP-PKG%2520signaling%2520requires%2520cAMP-PKA%2520signaling%26jtitle%3DNeuropsychopharmacology%26date%3D2014%26volume%3D39%26spage%3D2497%26epage%3D2505%26doi%3D10.1038%2Fnpp.2014.106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lueptow, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhan, C.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Donnell, J. M.</span></span> <span> </span><span class="NLM_article-title">Cylic cGMP-mediated memory enhancement in the object recognition test by inhibitors of phosphodiesterase-2 in mice</span>. <i>Psycopharmacology</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>233</i></span>,  <span class="NLM_fpage">447</span>– <span class="NLM_lpage">456</span>, <span class="refDoi"> DOI: 10.1007/s00213-015-4129-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.1007%2Fs00213-015-4129-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=26525565" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=233&publication_year=2016&pages=447-456&author=L.+M.+Lueptowauthor=C.-G.+Zhanauthor=J.+M.+O%E2%80%99Donnell&title=Cylic+cGMP-mediated+memory+enhancement+in+the+object+recognition+test+by+inhibitors+of+phosphodiesterase-2+in+mice&doi=10.1007%2Fs00213-015-4129-1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1007%2Fs00213-015-4129-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00213-015-4129-1%26sid%3Dliteratum%253Aachs%26aulast%3DLueptow%26aufirst%3DL.%2BM.%26aulast%3DZhan%26aufirst%3DC.-G.%26aulast%3DO%25E2%2580%2599Donnell%26aufirst%3DJ.%2BM.%26atitle%3DCylic%2520cGMP-mediated%2520memory%2520enhancement%2520in%2520the%2520object%2520recognition%2520test%2520by%2520inhibitors%2520of%2520phosphodiesterase-2%2520in%2520mice%26jtitle%3DPsycopharmacology%26date%3D2016%26volume%3D233%26spage%3D447%26epage%3D456%26doi%3D10.1007%2Fs00213-015-4129-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Elias, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beiser, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Au, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Agostino, R. B.</span></span> <span> </span><span class="NLM_article-title">The preclinical phase of Alzheimer disease: A 22-year prospective study of the Framingham Cohort</span>. <i>Arch. Neurol.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">808</span>– <span class="NLM_lpage">813</span>, <span class="refDoi"> DOI: 10.1001/archneur.57.6.808</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.1001%2Farchneur.57.6.808" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10867777" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A280%3ADC%252BD3czivVWgtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2000&pages=808-813&author=M.+F.+Eliasauthor=A.+Beiserauthor=P.+A.+Wolfauthor=R.+Auauthor=R.+F.+Whiteauthor=R.+B.+D%E2%80%99Agostino&title=The+preclinical+phase+of+Alzheimer+disease%3A+A+22-year+prospective+study+of+the+Framingham+Cohort&doi=10.1001%2Farchneur.57.6.808"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">The preclinical phase of alzheimer disease: A 22-year prospective study of the Framingham Cohort</span></div><div class="casAuthors">Elias M F; Beiser A; Wolf P A; Au R; White R F; D'Agostino R B</div><div class="citationInfo"><span class="NLM_cas:title">Archives of neurology</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">808-13</span>
        ISSN:<span class="NLM_cas:issn">0003-9942</span>.
    </div><div class="casAbstract">OBJECTIVES:  To relate performance on tests of cognitive ability to the subsequent development of probable Alzheimer disease (pAD) and to identify the pattern of earliest changes in cognitive functioning associated with a diagnosis of pAD.  DESIGN:  From May 1975 to November 1979, a screening neuropsychological battery was administered to Framingham Study participants.  They were followed up prospectively for 22 years and examined at least every 2 years for the development of pAD.  SETTING:  A community-based center for epidemiological research.  PARTICIPANTS:  Subjects were 1076 participants of the Framingham Study aged 65 to 94 years who were free of dementia and stroke at baseline (initial) neuropsychological testing.  MAIN OUTCOME MEASURE:  Presence or absence of pAD during a 22-year surveillance period was related to test performance at initial neuropsychological testing.  RESULTS:  Lower scores for measures of new learning, recall, retention, and abstract reasoning obtained during a dementia-free period were associated with the development of pAD.  Lower scores for measures of abstract reasoning and retention predicted pAD after a dementia-free period of 10 years.  CONCLUSIONS:  The "preclinical phase" of detectable lowering of cognitive functioning precedes the appearance of pAD by many years.  Measures of retention of information and abstract reasoning are among the strongest predictors of pAD when the interval between initial assessment and the development of pAD is long.  Arch Neurol. 2000.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSXl6yMaOTr2KNc3HjJRoFAfW6udTcc2eYi6guPsP_Udrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3czivVWgtA%253D%253D&md5=4d9dcc434215c236965c6558eb998cb7</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1001%2Farchneur.57.6.808&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1001%252Farchneur.57.6.808%26sid%3Dliteratum%253Aachs%26aulast%3DElias%26aufirst%3DM.%2BF.%26aulast%3DBeiser%26aufirst%3DA.%26aulast%3DWolf%26aufirst%3DP.%2BA.%26aulast%3DAu%26aufirst%3DR.%26aulast%3DWhite%26aufirst%3DR.%2BF.%26aulast%3DD%25E2%2580%2599Agostino%26aufirst%3DR.%2BB.%26atitle%3DThe%2520preclinical%2520phase%2520of%2520Alzheimer%2520disease%253A%2520A%252022-year%2520prospective%2520study%2520of%2520the%2520Framingham%2520Cohort%26jtitle%3DArch.%2520Neurol.%26date%3D2000%26volume%3D57%26spage%3D808%26epage%3D813%26doi%3D10.1001%2Farchneur.57.6.808" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Newman, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warrington, E. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossor, M. N.</span></span> <span> </span><span class="NLM_article-title">The earliest cognitive change in a person with familial Alzheimer’s disease: presymptomatic neuropsychological features in a pedigree with familial Alzheimer’s disease confirmed at necropsy</span>. <i>J. Neurol., Neurosurg. Psychiatry</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">967</span>– <span class="NLM_lpage">972</span>, <span class="refDoi"> DOI: 10.1136/jnnp.57.8.967</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.1136%2Fjnnp.57.8.967" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=8057122" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A280%3ADyaK2czjsVKqug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=1994&pages=967-972&author=S.+K.+Newmanauthor=E.+K.+Warringtonauthor=A.+M.+Kennedyauthor=M.+N.+Rossor&title=The+earliest+cognitive+change+in+a+person+with+familial+Alzheimer%E2%80%99s+disease%3A+presymptomatic+neuropsychological+features+in+a+pedigree+with+familial+Alzheimer%E2%80%99s+disease+confirmed+at+necropsy&doi=10.1136%2Fjnnp.57.8.967"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">The earliest cognitive change in a person with familial Alzheimer's disease: presymptomatic neuropsychological features in a pedigree with familial Alzheimer's disease confirmed at necropsy</span></div><div class="casAuthors">Newman S K; Warrington E K; Kennedy A M; Rossor M N</div><div class="citationInfo"><span class="NLM_cas:title">Journal of neurology, neurosurgery, and psychiatry</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">967-72</span>
        ISSN:<span class="NLM_cas:issn">0022-3050</span>.
    </div><div class="casAbstract">Comprehensive, longitudinal neuropsychological assessments are reported in a person "at risk" from autosomal dominant, necropsy confirmed familial Alzheimer's disease.  The first assessment showed a moderately selective verbal memory deficit in the context of mild general intellectual impairment.  Subsequent testing showed the progressive deterioration of visual memory and a mild decline of perceptual and spatial skills.  Language and literacy skills, however, remained comparatively intact.  The neuropsychological profiles obtained at each assessment are presented in profile maps.  These permit direct longitudinal comparison of cognitive function, and may serve in the comparison of different potential cases of familial Alzheimer's disease.  This case sought medical attention for memory difficulties 26 months after the first neuropsychological assessment.  These results mark the first cognitive manifestations in a pedigree with familial Alzheimer's disease which, in this case, were seen presymptomatically.  The findings are discussed in relation to neuropsychological studies of affected cases, and in terms of their reflecting the heterogeneous nature of familial Alzheimer's disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ5RO_FBTlCYHELvajU9pjPfW6udTcc2eYi6guPsP_Udrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK2czjsVKqug%253D%253D&md5=85b544e27364de30a6c7a4cf6c0e857e</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1136%2Fjnnp.57.8.967&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Fjnnp.57.8.967%26sid%3Dliteratum%253Aachs%26aulast%3DNewman%26aufirst%3DS.%2BK.%26aulast%3DWarrington%26aufirst%3DE.%2BK.%26aulast%3DKennedy%26aufirst%3DA.%2BM.%26aulast%3DRossor%26aufirst%3DM.%2BN.%26atitle%3DThe%2520earliest%2520cognitive%2520change%2520in%2520a%2520person%2520with%2520familial%2520Alzheimer%25E2%2580%2599s%2520disease%253A%2520presymptomatic%2520neuropsychological%2520features%2520in%2520a%2520pedigree%2520with%2520familial%2520Alzheimer%25E2%2580%2599s%2520disease%2520confirmed%2520at%2520necropsy%26jtitle%3DJ.%2520Neurol.%252C%2520Neurosurg.%2520Psychiatry%26date%3D1994%26volume%3D57%26spage%3D967%26epage%3D972%26doi%3D10.1136%2Fjnnp.57.8.967" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mayeux, R.</span></span> <span> </span><span class="NLM_article-title">Early Alzheimer’s Disease</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>362</i></span>,  <span class="NLM_fpage">2194</span>– <span class="NLM_lpage">2201</span>, <span class="refDoi"> DOI: 10.1056/nejmcp0910236</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.1056%2FNEJMcp0910236" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=20558370" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnsV2htrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=362&publication_year=2010&pages=2194-2201&author=R.+Mayeux&title=Early+Alzheimer%E2%80%99s+Disease&doi=10.1056%2Fnejmcp0910236"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Early Alzheimer's disease</span></div><div class="casAuthors">Mayeux, Richard</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">362</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">2194-2201</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">A 72-yr-old man who is still managing investments at a brokerage firm seeks consultation at the urging of his wife for increasing difficulty with memory over the past 2 years.  Clients have expressed concern about his occasional lapses in memory.  His wife reports that he frequently repeats questions about social appointments and becomes angry when she points this out.  The phys. examn. is normal, but the patient has difficulty remembering elements of a brief story and adding a small amt. of change.  He has a score of 28 out of 30 on the Mini-Mental State Examn., indicating slightly impaired cognitive function.  Early Alzheimer's disease is suspected.  How should the patient be further evaluated and treated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2kgZeBUNV27Vg90H21EOLACvtfcHk0lgX0tdL1lC1QA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXnsV2htrw%253D&md5=6f8db845fbe83e95f07c1006b80c564b</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1056%2FNEJMcp0910236&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMcp0910236%26sid%3Dliteratum%253Aachs%26aulast%3DMayeux%26aufirst%3DR.%26atitle%3DEarly%2520Alzheimer%25E2%2580%2599s%2520Disease%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2010%26volume%3D362%26spage%3D2194%26epage%3D2201%26doi%3D10.1056%2Fnejmcp0910236" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gabrieli, J. D. E.</span></span> <span> </span><span class="NLM_article-title">Memory systems analyses of mnemonic disorders in aging and age-related diseases</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">13534</span>– <span class="NLM_lpage">13540</span>, <span class="refDoi"> DOI: 10.1073/pnas.93.24.13534</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.1073%2Fpnas.93.24.13534" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=8942968" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADyaK28Xnt1GhsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=1996&pages=13534-13540&author=J.+D.+E.+Gabrieli&title=Memory+systems+analyses+of+mnemonic+disorders+in+aging+and+age-related+diseases&doi=10.1073%2Fpnas.93.24.13534"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Memory systems analyses of mnemonic disorders in aging and age-related diseases</span></div><div class="casAuthors">Gabrieli, John D. E.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">13534-13540</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The effects upon memory of normal aging and two age-related neurodegenerative diseases, Alzheimer disease (AD) and Parkinson disease, are analyzed in terms of memory systems, specific neural networks that mediate specific mnemonic processes.  An occipital memory system mediating implicit visual-perceptual memory appears to be unaffected by aging or AD.  A frontal system that may mediate implicit conceptual memory is affected by Ad but not by normal aging.  Another frontal system that mediates aspects of working and strategic memory is affected by Parkinson disease and, to a lesser extent, by aging.  The aging effect appears to occur during all ages of the adult life-span.  Finally, a medial-temporal system that mediates declarative memory is affected by the late onset of AD.  Studies of intact and impaired memory in age-related diseases suggest that normal aging has markedly different effects upon different memory systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJU9ERkAGVH7Vg90H21EOLACvtfcHk0lgX0tdL1lC1QA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xnt1GhsL8%253D&md5=7c0fbfbf0f7bd6040698e792b98f817d</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1073%2Fpnas.93.24.13534&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.93.24.13534%26sid%3Dliteratum%253Aachs%26aulast%3DGabrieli%26aufirst%3DJ.%2BD.%2BE.%26atitle%3DMemory%2520systems%2520analyses%2520of%2520mnemonic%2520disorders%2520in%2520aging%2520and%2520age-related%2520diseases%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1996%26volume%3D93%26spage%3D13534%26epage%3D13540%26doi%3D10.1073%2Fpnas.93.24.13534" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Belleville, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sylvain-Roy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Boysson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ménard, M.-C.</span></span> <span> </span><span class="NLM_article-title">Chapter 23 Characterizing the memory changes in persons with mild cognitive impairment</span>. <i>Prog. Brain Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>169</i></span>,  <span class="NLM_fpage">365</span>– <span class="NLM_lpage">375</span>, <span class="refDoi"> DOI: 10.1016/s0079-6123(07)00023-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.1016%2FS0079-6123%2807%2900023-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=18394487" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A280%3ADC%252BD1c3jsFyktw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=169&publication_year=2008&pages=365-375&author=S.+Bellevilleauthor=S.+Sylvain-Royauthor=C.+de+Boyssonauthor=M.-C.+M%C3%A9nard&title=Chapter+23+Characterizing+the+memory+changes+in+persons+with+mild+cognitive+impairment&doi=10.1016%2Fs0079-6123%2807%2900023-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Characterizing the memory changes in persons with mild cognitive impairment</span></div><div class="casAuthors">Belleville Sylvie; Sylvain-Roy Stephanie; de Boysson Chloe; Menard Marie-Claude</div><div class="citationInfo"><span class="NLM_cas:title">Progress in brain research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">169</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">365-75</span>
        ISSN:<span class="NLM_cas:issn">0079-6123</span>.
    </div><div class="casAbstract">Persons with mild cognitive impairment (MCI) do not meet criteria for Alzheimer's disease (AD) but are at high risk for developing the disease.  Presence of a memory deficit is a key component in the characterization of MCI.  This chapter presents empirical studies that attempt to describe and understand the nature of the memory deficit in MCI with a focus on episodic memory and working memory.  Cross-sectional studies report prominent deficits of episodic memory characterized by impaired encoding of the contextual information that makes up complex events.  This results in reduced free and cued recall, impaired recognition, and impaired associative learning.  Although semantic encoding is found to be impaired in conditions that rely on explicit and intentional retrieval, preserved semantic processing is found with automatic conditions of testing.  Studies indicate the presence of a partial deficit of working memory with the ability to divide attention being most severely impaired.  However, there appears to be heterogeneity as to the extent of the working memory impairment.  The presence of vascular anomalies on MRI, as well as being in a more advanced stage in the continuum from MCI to AD, are associated with more severe and more pervasive working memory deficits.  Finally, longitudinal studies indicate that the combination of episodic and working memory deficits represents a strong predictor of progression from MCI to AD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTbHR2uYMPv3eIHSmpohpd0fW6udTcc2ebsw4fvajkdYbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1c3jsFyktw%253D%253D&md5=7a02e44c01c5b7148bcf8e1e88fffbbc</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2FS0079-6123%2807%2900023-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0079-6123%252807%252900023-4%26sid%3Dliteratum%253Aachs%26aulast%3DBelleville%26aufirst%3DS.%26aulast%3DSylvain-Roy%26aufirst%3DS.%26aulast%3Dde%2BBoysson%26aufirst%3DC.%26aulast%3DM%25C3%25A9nard%26aufirst%3DM.-C.%26atitle%3DChapter%252023%2520Characterizing%2520the%2520memory%2520changes%2520in%2520persons%2520with%2520mild%2520cognitive%2520impairment%26jtitle%3DProg.%2520Brain%2520Res.%26date%3D2008%26volume%3D169%26spage%3D365%26epage%3D375%26doi%3D10.1016%2Fs0079-6123%2807%2900023-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lyketsos, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fitzpatrick, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breitner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeKosky, S.</span></span> <span> </span><span class="NLM_article-title">Prevalence of neuropsychiatric symptoms in dementia and mild cognitive impairment: results from the cardiovascular health study</span>. <i>JAMA</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>288</i></span>,  <span class="NLM_fpage">1475</span>– <span class="NLM_lpage">1483</span>, <span class="refDoi"> DOI: 10.1001/jama.288.12.1475</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.1001%2Fjama.288.12.1475" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=12243634" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A280%3ADC%252BD38vot1elsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=288&publication_year=2002&pages=1475-1483&author=C.+G.+Lyketsosauthor=O.+Lopezauthor=B.+Jonesauthor=A.+L.+Fitzpatrickauthor=J.+Breitnerauthor=S.+DeKosky&title=Prevalence+of+neuropsychiatric+symptoms+in+dementia+and+mild+cognitive+impairment%3A+results+from+the+cardiovascular+health+study&doi=10.1001%2Fjama.288.12.1475"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Prevalence of neuropsychiatric symptoms in dementia and mild cognitive impairment: results from the cardiovascular health study</span></div><div class="casAuthors">Lyketsos Constantine G; Lopez Oscar; Jones Beverly; Fitzpatrick Annette L; Breitner John; DeKosky Steven</div><div class="citationInfo"><span class="NLM_cas:title">JAMA</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">288</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1475-83</span>
        ISSN:<span class="NLM_cas:issn">0098-7484</span>.
    </div><div class="casAbstract">CONTEXT:  Mild cognitive impairment (MCI) may be a precursor to dementia, at least in some cases.  Dementia and MCI are associated with neuropsychiatric symptoms in clinical samples.  Only 2 population-based studies exist of the prevalence of these symptoms in dementia, and none exist for MCI.  OBJECTIVE:  To estimate the prevalence of neuropsychiatric symptoms in dementia and MCI in a population-based study.  DESIGN:  Cross-sectional study derived from the Cardiovascular Health Study, a longitudinal cohort study.  SETTING AND PARTICIPANTS:  A total of 3608 participants were cognitively evaluated using data collected longitudinally over 10 years and additional data collected in 1999-2000 in 4 US counties.  Dementia and MCI were classified using clinical criteria and adjudicated by committee review by expert neurologists and psychiatrists.  A total of 824 individuals completed the Neuropsychiatric Inventory (NPI); 362 were classified as having dementia, 320 as having MCI; and 142 did not meet criteria for MCI or dementia.  MAIN OUTCOME MEASURE:  Prevalence of neuropsychiatric symptoms, based on ratings on the NPI in the previous month and from the onset of cognitive symptoms.  RESULTS:  Of the 682 individuals with dementia or MCI, 43% of MCI participants (n = 138) exhibited neuropsychiatric symptoms in the previous month (29% rated as clinically significant) with depression (20%), apathy (15%), and irritability (15%) being most common.  Among the dementia participants, 75% (n = 270) had exhibited a neuropsychiatric symptom in the past month (62% were clinically significant); 55% (n = 199) reported 2 or more and 44% (n = 159) 3 or more disturbances in the past month.  In participants with dementia, the most frequent disturbances were apathy (36%), depression (32%), and agitation/aggression (30%).  Eighty percent of dementia participants (n = 233) and 50% of MCI participants (n = 139) exhibited at least 1 NPI symptom from the onset of cognitive symptoms.  There were no differences in prevalence of neuropsychiatric symptoms between participants with Alzheimer-type dementia and those with other dementias, with the exception of aberrant motor behavior, which was more frequent in Alzheimer-type dementia (5.4% vs 1%; P =.02).  CONCLUSIONS:  Neuropsychiatric symptoms occur in the majority of persons with dementia over the course of the disease.  These are the first population-based estimates for neuropsychiatric symptoms in MCI, indicating a high prevalence associated with this condition as well.  These symptoms have serious adverse consequences and should be inquired about and treated as necessary.  Study of neuropsychiatric symptoms in the context of dementia may improve our understanding of brain-behavior relationships.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRiqzKr_B3vlZHZzFRCXxVKfW6udTcc2ebcGnZt0YNydrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD38vot1elsg%253D%253D&md5=1dccd5468a3ba37bb56ba804694e29ff</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1001%2Fjama.288.12.1475&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1001%252Fjama.288.12.1475%26sid%3Dliteratum%253Aachs%26aulast%3DLyketsos%26aufirst%3DC.%2BG.%26aulast%3DLopez%26aufirst%3DO.%26aulast%3DJones%26aufirst%3DB.%26aulast%3DFitzpatrick%26aufirst%3DA.%2BL.%26aulast%3DBreitner%26aufirst%3DJ.%26aulast%3DDeKosky%26aufirst%3DS.%26atitle%3DPrevalence%2520of%2520neuropsychiatric%2520symptoms%2520in%2520dementia%2520and%2520mild%2520cognitive%2520impairment%253A%2520results%2520from%2520the%2520cardiovascular%2520health%2520study%26jtitle%3DJAMA%26date%3D2002%26volume%3D288%26spage%3D1475%26epage%3D1483%26doi%3D10.1001%2Fjama.288.12.1475" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blennow, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Leon, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zetterberg, H.</span></span> <span> </span><span class="NLM_article-title">Alzheimer’s disease</span>. <i>Lancet</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>368</i></span>,  <span class="NLM_fpage">387</span>– <span class="NLM_lpage">403</span>, <span class="refDoi"> DOI: 10.1016/s0140-6736(06)69113-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.1016%2Fs0140-6736%2806%2969113-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=16876668" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BD28XnsFymsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=368&publication_year=2006&pages=387-403&author=K.+Blennowauthor=M.+J.+de+Leonauthor=H.+Zetterberg&title=Alzheimer%E2%80%99s+disease&doi=10.1016%2Fs0140-6736%2806%2969113-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Alzheimer's disease</span></div><div class="casAuthors">Blennow, Kaj; de Leon, Mony J.; Zetterberg, Henrik</div><div class="citationInfo"><span class="NLM_cas:title">Lancet</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">368</span>
        (<span class="NLM_cas:issue">9533</span>),
    <span class="NLM_cas:pages">387-403</span>CODEN:
                <span class="NLM_cas:coden">LANCAO</span>;
        ISSN:<span class="NLM_cas:issn">0140-6736</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Alzheimer's disease is the most common cause of dementia.  Research advances have enabled detailed understanding of the mol. pathogenesis of the hallmarks of the disease-ie, plaques, composed of amyloid β (Aβ), and tangles, composed of hyperphosphorylated tau.  However, as our knowledge increases so does our appreciation for the pathogenic complexity of the disorder.  Alzheimer's disease is a very rare autosomal dominant disease with early onset, caused by mutations in the amyloid precursor protein and presenilin genes, both linked to Aβ metab.  By contrast with familial disease, sporadic Alzheimer's disease is very common with more than 15 million people affected worldwide.  The cause of the sporadic form of the disease is unknown, probably because the disease is heterogeneous, caused by aging in concert with a complex interaction of both genetic and environmental risk factors.  This seminar reviews the key aspects of the disease, including epidemiol., genetics, pathogenesis, diagnosis, and treatment, as well as recent developments and controversies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriOIYvvzbyybVg90H21EOLACvtfcHk0ljmHL2AiqefTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XnsFymsbY%253D&md5=f70c374fbe01be1953bd0eea290745bd</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fs0140-6736%2806%2969113-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs0140-6736%252806%252969113-7%26sid%3Dliteratum%253Aachs%26aulast%3DBlennow%26aufirst%3DK.%26aulast%3Dde%2BLeon%26aufirst%3DM.%2BJ.%26aulast%3DZetterberg%26aufirst%3DH.%26atitle%3DAlzheimer%25E2%2580%2599s%2520disease%26jtitle%3DLancet%26date%3D2006%26volume%3D368%26spage%3D387%26epage%3D403%26doi%3D10.1016%2Fs0140-6736%2806%2969113-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gomez, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breitenbucher, J. G.</span></span> <span> </span><span class="NLM_article-title">PDE2A inhibition: potential for the treatment of cognitive disorders</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">6522</span>– <span class="NLM_lpage">6527</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2013.10.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.1016%2Fj.bmcl.2013.10.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=24189054" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslartbjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=6522-6527&author=L.+Gomezauthor=J.+G.+Breitenbucher&title=PDE2A+inhibition%3A+potential+for+the+treatment+of+cognitive+disorders&doi=10.1016%2Fj.bmcl.2013.10.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">PDE2 inhibition: Potential for the treatment of cognitive disorders</span></div><div class="casAuthors">Gomez, Laurent; Breitenbucher, J. Guy</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">6522-6527</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Phosphodiesterase inhibition has received much attention in the past 20 years for the potential treatment of CNS disorders.  A primary focus of this work is the enhancement of memory and/or cognitive functioning.  The role of PDEs in the augmentation of cyclic nucleotide signaling makes these enzymes attractive targets for enhancing the effects of neuronal communication.  This review focuses on recent findings with respect to the role of PDE2 inhibition in cognitive functioning.  Special attention is paid to recently disclosed, selective tool compds. and the use of these tool compds. to support the role of PDE2 inhibition in cognition.  Recently reported SAR and modeling work will be presented along with discussion of the entry of new PDE2 inhibitors into the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGUUFzh2-kTrVg90H21EOLACvtfcHk0ljmHL2AiqefTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslartbjL&md5=04ca2bdc35dc0b436a228ba62934b91e</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2013.10.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2013.10.014%26sid%3Dliteratum%253Aachs%26aulast%3DGomez%26aufirst%3DL.%26aulast%3DBreitenbucher%26aufirst%3DJ.%2BG.%26atitle%3DPDE2A%2520inhibition%253A%2520potential%2520for%2520the%2520treatment%2520of%2520cognitive%2520disorders%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D23%26spage%3D6522%26epage%3D6527%26doi%3D10.1016%2Fj.bmcl.2013.10.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, H.-B.</span></span> <span> </span><span class="NLM_article-title">Novel phosphodiesterase inhibitors for cognitive improvement in Alzheimer’s disease</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">5467</span>– <span class="NLM_lpage">5483</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01370</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01370" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFGqu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=5467-5483&author=Y.+Wuauthor=Z.+Liauthor=Y.-Y.+Huangauthor=D.+Wuauthor=H.-B.+Luo&title=Novel+phosphodiesterase+inhibitors+for+cognitive+improvement+in+Alzheimer%E2%80%99s+disease&doi=10.1021%2Facs.jmedchem.7b01370"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Phosphodiesterase Inhibitors for Cognitive Improvement in Alzheimer's Disease</span></div><div class="casAuthors">Wu, Yinuo; Li, Zhe; Huang, Yi-You; Wu, Deyan; Luo, Hai-Bin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">5467-5483</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Alzheimer's disease (AD) is one of the greatest public health challenges.  Phosphodiesterases (PDEs) are a superenzyme family responsible for the hydrolysis of two second messengers: cyclic adenosine monophosphate (cAMP) and cyclic guanosine monophosphate (cGMP).  Since several PDE subfamilies are highly expressed in the human brain, the inhibition of PDEs is involved in neurodegenerative processes by regulating the concn. of cAMP and/or cGMP.  Currently, PDEs are considered as promising targets for the treatment of AD since many PDE inhibitors have exhibited remarkable cognitive improvement effects in preclin. studies and over 15 of them have been subjected to clin. trials.  The aim of this review is to summarize the outstanding progress that has been made by PDE inhibitors as anti-AD agents with encouraging results in preclin. studies and clin. trials.  The binding affinity, pharmacokinetics, underlying mechanisms, and limitations of these PDE inhibitors in the treatment of AD are also reviewed and discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGom1l9eMNKCt7Vg90H21EOLACvtfcHk0lh_WjiBNc3BUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFGqu7w%253D&md5=87cd22fa21c661f4cb5dc2155368bd7c</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01370&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01370%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DHuang%26aufirst%3DY.-Y.%26aulast%3DWu%26aufirst%3DD.%26aulast%3DLuo%26aufirst%3DH.-B.%26atitle%3DNovel%2520phosphodiesterase%2520inhibitors%2520for%2520cognitive%2520improvement%2520in%2520Alzheimer%25E2%2580%2599s%2520disease%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D5467%26epage%3D5483%26doi%3D10.1021%2Facs.jmedchem.7b01370" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gomez, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Massari, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vickers, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freestone, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vernier, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ly, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCarrick, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marrone, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Y. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoder, Z. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemus, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Broadbent, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barido, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warren, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmelzer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neul, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersen, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sebring, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aertgeerts, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tabatabaei, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breitenbucher, J. G.</span></span> <span> </span><span class="NLM_article-title">Design and synthesis of novel and selective phosphodiesterase 2 (PDE2a) inhibitors for the treatment of memory disorders</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2037</span>– <span class="NLM_lpage">2051</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01793</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01793" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC2sXit1Sjuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=2037-2051&author=L.+Gomezauthor=M.+E.+Massariauthor=T.+Vickersauthor=G.+Freestoneauthor=W.+Vernierauthor=K.+Lyauthor=R.+Xuauthor=M.+McCarrickauthor=T.+Marroneauthor=M.+Metzauthor=Y.+G.+Yanauthor=Z.+W.+Yoderauthor=R.+Lemusauthor=N.+J.+Broadbentauthor=R.+Baridoauthor=N.+Warrenauthor=K.+Schmelzerauthor=D.+Neulauthor=D.+Leeauthor=C.+B.+Andersenauthor=K.+Sebringauthor=K.+Aertgeertsauthor=X.+Zhouauthor=A.+Tabatabaeiauthor=M.+Petersauthor=J.+G.+Breitenbucher&title=Design+and+synthesis+of+novel+and+selective+phosphodiesterase+2+%28PDE2a%29+inhibitors+for+the+treatment+of+memory+disorders&doi=10.1021%2Facs.jmedchem.6b01793"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Synthesis of Novel and Selective Phosphodiesterase 2 (PDE2a) Inhibitors for the Treatment of Memory Disorders</span></div><div class="casAuthors">Gomez, Laurent; Massari, Mark Eben; Vickers, Troy; Freestone, Graeme; Vernier, William; Ly, Kiev; Xu, Rui; McCarrick, Margaret; Marrone, Tami; Metz, Markus; Yan, Yingzhou G.; Yoder, Zachary W.; Lemus, Robert; Broadbent, Nicola J.; Barido, Richard; Warren, Noelle; Schmelzer, Kara; Neul, David; Lee, Dong; Andersen, Carsten B.; Sebring, Kristen; Aertgeerts, Kathleen; Zhou, Xianbo; Tabatabaei, Ali; Peters, Marco; Breitenbucher, J. Guy</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2037-2051</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of potent and selective [1,2,4]triazolo[1,5-a]pyrimidine PDE2a inhibitors is reported.  The design and improvement of the binding properties of this series was achieved using X-ray crystal structures in conjunction with careful anal. of electronic and structural requirements for the PDE2a enzyme.  One of the lead compds., compd. I (DNS-8254), was identified as a potent and highly selective PDE2a enzyme inhibitor with favorable rat pharmacokinetic properties.  Interestingly, the increased potency of compd. I was facilitated by the formation of a halogen bond with the oxygen of Tyr827 present in the PDE2a active site.  In vivo, compd. I demonstrated significant memory enhancing effects in a rat model of novel object recognition.  Taken together, these data suggest that compd. I may be a useful tool to explore the pharmacol. of selective PDE2a inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6ML_QxZhnWLVg90H21EOLACvtfcHk0lh_WjiBNc3BUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXit1Sjuro%253D&md5=fe8df71fbd1f61423352dfcb862d9d53</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01793&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01793%26sid%3Dliteratum%253Aachs%26aulast%3DGomez%26aufirst%3DL.%26aulast%3DMassari%26aufirst%3DM.%2BE.%26aulast%3DVickers%26aufirst%3DT.%26aulast%3DFreestone%26aufirst%3DG.%26aulast%3DVernier%26aufirst%3DW.%26aulast%3DLy%26aufirst%3DK.%26aulast%3DXu%26aufirst%3DR.%26aulast%3DMcCarrick%26aufirst%3DM.%26aulast%3DMarrone%26aufirst%3DT.%26aulast%3DMetz%26aufirst%3DM.%26aulast%3DYan%26aufirst%3DY.%2BG.%26aulast%3DYoder%26aufirst%3DZ.%2BW.%26aulast%3DLemus%26aufirst%3DR.%26aulast%3DBroadbent%26aufirst%3DN.%2BJ.%26aulast%3DBarido%26aufirst%3DR.%26aulast%3DWarren%26aufirst%3DN.%26aulast%3DSchmelzer%26aufirst%3DK.%26aulast%3DNeul%26aufirst%3DD.%26aulast%3DLee%26aufirst%3DD.%26aulast%3DAndersen%26aufirst%3DC.%2BB.%26aulast%3DSebring%26aufirst%3DK.%26aulast%3DAertgeerts%26aufirst%3DK.%26aulast%3DZhou%26aufirst%3DX.%26aulast%3DTabatabaei%26aufirst%3DA.%26aulast%3DPeters%26aufirst%3DM.%26aulast%3DBreitenbucher%26aufirst%3DJ.%2BG.%26atitle%3DDesign%2520and%2520synthesis%2520of%2520novel%2520and%2520selective%2520phosphodiesterase%25202%2520%2528PDE2a%2529%2520inhibitors%2520for%2520the%2520treatment%2520of%2520memory%2520disorders%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D2037%26epage%3D2051%26doi%3D10.1021%2Facs.jmedchem.6b01793" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mikami, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashizawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nomura, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawasaki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasaki, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kokubo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uchiyama, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakashima, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamiguchi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imada, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwashita, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taniguchi, T.</span></span> <span> </span><span class="NLM_article-title">Discovery of clinical candidate N-((1S)-1-(3-fluoro-4-(trifluoromethoxy)phenyl)-2-methoxyethyl)-7-methoxy-2-oxo-2,3-dihydropyrido[2,3-b]pyrazine-4(1H)-carboxamide (TAK-915): a highly potent, selective, and brain-penetrating phosphodiesterase 2A inhibitor for the treatment of cognitive disorders</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">7677</span>– <span class="NLM_lpage">7702</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00807</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00807" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlSmtL3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=7677-7702&author=S.+Mikamiauthor=S.+Nakamuraauthor=T.+Ashizawaauthor=I.+Nomuraauthor=M.+Kawasakiauthor=S.+Sasakiauthor=H.+Okiauthor=H.+Kokuboauthor=I.+D.+Hoffmanauthor=H.+Zouauthor=N.+Uchiyamaauthor=K.+Nakashimaauthor=N.+Kamiguchiauthor=H.+Imadaauthor=N.+Suzukiauthor=H.+Iwashitaauthor=T.+Taniguchi&title=Discovery+of+clinical+candidate+N-%28%281S%29-1-%283-fluoro-4-%28trifluoromethoxy%29phenyl%29-2-methoxyethyl%29-7-methoxy-2-oxo-2%2C3-dihydropyrido%5B2%2C3-b%5Dpyrazine-4%281H%29-carboxamide+%28TAK-915%29%3A+a+highly+potent%2C+selective%2C+and+brain-penetrating+phosphodiesterase+2A+inhibitor+for+the+treatment+of+cognitive+disorders&doi=10.1021%2Facs.jmedchem.7b00807"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Clinical Candidate N-((1S)-1-(3-Fluoro-4-(trifluoromethoxy)phenyl)-2-methoxyethyl)-7-methoxy-2-oxo-2,3-dihydropyrido[2,3-b]pyrazine-4(1H)-carboxamide (TAK-915): A Highly Potent, Selective, and Brain-Penetrating Phosphodiesterase 2A Inhibitor for the Treatment of Cognitive Disorders</span></div><div class="casAuthors">Mikami, Satoshi; Nakamura, Shinji; Ashizawa, Tomoko; Nomura, Izumi; Kawasaki, Masanori; Sasaki, Shigekazu; Oki, Hideyuki; Kokubo, Hironori; Hoffman, Isaac D.; Zou, Hua; Uchiyama, Noriko; Nakashima, Kosuke; Kamiguchi, Naomi; Imada, Haruka; Suzuki, Noriko; Iwashita, Hiroki; Taniguchi, Takahiko</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">7677-7702</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Phosphodiesterase (PDE) 2A inhibitors have emerged as a novel mechanism with potential therapeutic option to ameliorate cognitive dysfunction in schizophrenia or Alzheimer's disease through upregulation of cyclic nucleotides in the brain, and thereby achieve potentiation of cyclic nucleotide signaling pathways.  This article details the expedited optimization of the authors' recently disclosed pyrazolo[1,5-a]pyrimidine lead compd. 4b, leading to the discovery of clin. candidate 36 (I) (TAK-915), which demonstrates an appropriate combination of potency, PDE selectivity, and favorable pharmacokinetic (PK) properties, including brain penetration.  Successful identification of 36 was realized through application of structure-based drug design (SBDD) to further improve potency and PDE selectivity, coupled with prospective design focused on physicochem. properties to deliver brain penetration.  Oral administration of 36 demonstrated significant elevation of 3',5'-cGMP levels in mouse brains, and improved cognitive performance in a novel object recognition task in rats.  Consequently, compd. 36 was advanced into human clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCUw5grC_66rVg90H21EOLACvtfcHk0lh_WjiBNc3BUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlSmtL3I&md5=f87cb3ac86b6299fc90e336b88ea2a21</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00807&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00807%26sid%3Dliteratum%253Aachs%26aulast%3DMikami%26aufirst%3DS.%26aulast%3DNakamura%26aufirst%3DS.%26aulast%3DAshizawa%26aufirst%3DT.%26aulast%3DNomura%26aufirst%3DI.%26aulast%3DKawasaki%26aufirst%3DM.%26aulast%3DSasaki%26aufirst%3DS.%26aulast%3DOki%26aufirst%3DH.%26aulast%3DKokubo%26aufirst%3DH.%26aulast%3DHoffman%26aufirst%3DI.%2BD.%26aulast%3DZou%26aufirst%3DH.%26aulast%3DUchiyama%26aufirst%3DN.%26aulast%3DNakashima%26aufirst%3DK.%26aulast%3DKamiguchi%26aufirst%3DN.%26aulast%3DImada%26aufirst%3DH.%26aulast%3DSuzuki%26aufirst%3DN.%26aulast%3DIwashita%26aufirst%3DH.%26aulast%3DTaniguchi%26aufirst%3DT.%26atitle%3DDiscovery%2520of%2520clinical%2520candidate%2520N-%2528%25281S%2529-1-%25283-fluoro-4-%2528trifluoromethoxy%2529phenyl%2529-2-methoxyethyl%2529-7-methoxy-2-oxo-2%252C3-dihydropyrido%255B2%252C3-b%255Dpyrazine-4%25281H%2529-carboxamide%2520%2528TAK-915%2529%253A%2520a%2520highly%2520potent%252C%2520selective%252C%2520and%2520brain-penetrating%2520phosphodiesterase%25202A%2520inhibitor%2520for%2520the%2520treatment%2520of%2520cognitive%2520disorders%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D7677%26epage%3D7702%26doi%3D10.1021%2Facs.jmedchem.7b00807" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Helal, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyden, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chappie, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fennell, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forman, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hajos, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harms, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphrey, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kleiman, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kormos, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maklad, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDowell, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mente, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connor, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandit, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piotrowski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueno, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verhoest, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, E. X.</span></span> <span> </span><span class="NLM_article-title">Application of structure-based design and parallel chemistry to identify a potent, selective, and brain penetrant Phosphodiesterase 2A inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">5673</span>– <span class="NLM_lpage">5698</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00397</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00397" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpt1Sitb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=5673-5698&author=C.+J.+Helalauthor=E.+P.+Arnoldauthor=T.+L.+Boydenauthor=C.+Changauthor=T.+A.+Chappieauthor=K.+F.+Fennellauthor=M.+D.+Formanauthor=M.+Hajosauthor=J.+F.+Harmsauthor=W.+E.+Hoffmanauthor=J.+M.+Humphreyauthor=Z.+Kangauthor=R.+J.+Kleimanauthor=B.+L.+Kormosauthor=C.-W.+Leeauthor=J.+Luauthor=N.+Makladauthor=L.+McDowellauthor=S.+Menteauthor=R.+E.+O%E2%80%99Connorauthor=J.+Panditauthor=M.+Piotrowskiauthor=A.+W.+Schmidtauthor=C.+J.+Schmidtauthor=H.+Uenoauthor=P.+R.+Verhoestauthor=E.+X.+Yang&title=Application+of+structure-based+design+and+parallel+chemistry+to+identify+a+potent%2C+selective%2C+and+brain+penetrant+Phosphodiesterase+2A+inhibitor&doi=10.1021%2Facs.jmedchem.7b00397"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Application of Structure-Based Design and Parallel Chemistry to Identify a Potent, Selective, and Brain Penetrant Phosphodiesterase 2A Inhibitor</span></div><div class="casAuthors">Helal, Christopher J.; Arnold, Eric P.; Boyden, Tracey L.; Chang, Cheng; Chappie, Thomas A.; Fennell, Kimberly F.; Forman, Michael D.; Hajos, Mihaly; Harms, John F.; Hoffman, William E.; Humphrey, John M.; Kang, Zhijun; Kleiman, Robin J.; Kormos, Bethany L.; Lee, Che-Wah; Lu, Jiemin; Maklad, Noha; McDowell, Laura; Mente, Scot; O'Connor, Rebecca E.; Pandit, Jayvardhan; Piotrowski, Mary; Schmidt, Anne W.; Schmidt, Christopher J.; Ueno, Hirokazu; Verhoest, Patrick R.; Yang, Edward X.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">5673-5698</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Phosphodiesterase 2A (PDE2A) inhibitors have been reported to demonstrate in vivo activity in preclin. models of cognition.  To more fully explore the biol. of PDE2A inhibition, we sought to identify potent PDE2A inhibitors with improved brain penetration as compared to current literature compds.  Applying estd. human dose calcns. while simultaneously leveraging synthetically enabled chem. and structure-based drug design has resulted in a highly potent, selective, brain penetrant compd. 71 (PF-05085727) that effects in vivo biochem. changes commensurate with PDE2A inhibition along with behavioral and electrophysiol. reversal of the effects of NMDA antagonists in rodents.  This data supports the ability of PDE2A inhibitors to potentiate NMDA signaling and their further development for clin. cognition indications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpZdZgTF6GMLVg90H21EOLACvtfcHk0lgprDynkqMtlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpt1Sitb4%253D&md5=e1c3c778fc622ac2f5917625a983323f</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00397&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00397%26sid%3Dliteratum%253Aachs%26aulast%3DHelal%26aufirst%3DC.%2BJ.%26aulast%3DArnold%26aufirst%3DE.%2BP.%26aulast%3DBoyden%26aufirst%3DT.%2BL.%26aulast%3DChang%26aufirst%3DC.%26aulast%3DChappie%26aufirst%3DT.%2BA.%26aulast%3DFennell%26aufirst%3DK.%2BF.%26aulast%3DForman%26aufirst%3DM.%2BD.%26aulast%3DHajos%26aufirst%3DM.%26aulast%3DHarms%26aufirst%3DJ.%2BF.%26aulast%3DHoffman%26aufirst%3DW.%2BE.%26aulast%3DHumphrey%26aufirst%3DJ.%2BM.%26aulast%3DKang%26aufirst%3DZ.%26aulast%3DKleiman%26aufirst%3DR.%2BJ.%26aulast%3DKormos%26aufirst%3DB.%2BL.%26aulast%3DLee%26aufirst%3DC.-W.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DMaklad%26aufirst%3DN.%26aulast%3DMcDowell%26aufirst%3DL.%26aulast%3DMente%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DR.%2BE.%26aulast%3DPandit%26aufirst%3DJ.%26aulast%3DPiotrowski%26aufirst%3DM.%26aulast%3DSchmidt%26aufirst%3DA.%2BW.%26aulast%3DSchmidt%26aufirst%3DC.%2BJ.%26aulast%3DUeno%26aufirst%3DH.%26aulast%3DVerhoest%26aufirst%3DP.%2BR.%26aulast%3DYang%26aufirst%3DE.%2BX.%26atitle%3DApplication%2520of%2520structure-based%2520design%2520and%2520parallel%2520chemistry%2520to%2520identify%2520a%2520potent%252C%2520selective%252C%2520and%2520brain%2520penetrant%2520Phosphodiesterase%25202A%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D5673%26epage%3D5698%26doi%3D10.1021%2Facs.jmedchem.7b00397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stachel, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nomland, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ginnetti, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paone, D. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puri, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lange, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drott, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcus, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dwyer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uslaner, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, S. M.</span></span> <span> </span><span class="NLM_article-title">Structure-Guided design and procognitive assessment of a potent and selective phosphodiesterase 2A inhibitor</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">815</span>– <span class="NLM_lpage">820</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.8b00214</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.8b00214" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlykurrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=815-820&author=S.+J.+Stachelauthor=R.+Bergerauthor=A.+B.+Nomlandauthor=A.+T.+Ginnettiauthor=D.+V.+Paoneauthor=D.+Wangauthor=V.+Puriauthor=H.+Langeauthor=J.+Drottauthor=J.+Luauthor=J.+Marcusauthor=M.+P.+Dwyerauthor=S.+Suonauthor=J.+M.+Uslanerauthor=S.+M.+Smith&title=Structure-Guided+design+and+procognitive+assessment+of+a+potent+and+selective+phosphodiesterase+2A+inhibitor&doi=10.1021%2Facsmedchemlett.8b00214"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Guided Design and Procognitive Assessment of a Potent and Selective Phosphodiesterase 2A Inhibitor</span></div><div class="casAuthors">Stachel, Shawn J.; Berger, Richard; Nomland, Ashley B.; Ginnetti, Anthony T.; Paone, Daniel V.; Wang, Deping; Puri, Vanita; Lange, Henry; Drott, Jason; Lu, Jun; Marcus, Jacob; Dwyer, Michael P.; Suon, Sokreine; Uslaner, Jason M.; Smith, Sean M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">815-820</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Herein the authors describe the development of a series of pyrazolopyrimidinone phosphodiesterase 2A (PDE2) inhibitors using structure-guided lead identification and design.  The series was derived from informed chemotype replacement based on previously identified internal leads.  The initially designed compd. 3 ((S)-3,6-dimethyl-1-(1-(4-(trifluoromethyl)phenyl)ethyl)-1,5-dihydro-4H-pyrazolo[3,4-d]pyrimidin-4-one), while potent on PDE2, displayed unsatisfactory selectivity against the other PDE2 isoforms.  Compd. 3 was subsequently optimized for improved PDE2 activity and isoform selectivity.  Insights into the origins of PDE2 selectivity are described and verified using cocrystallog.  An optimized lead, 4 ((S)-3-(hydroxymethyl)-6-methyl-1-(1-(4-(trifluoromethyl)phenyl)ethyl)-1,5-dihydro-4H-pyrazolo[3,4-d]pyrimidin-4-one), demonstrated improved performance in both a rodent and a nonhuman primate cognition model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAr9ZG63azTrVg90H21EOLACvtfcHk0lgprDynkqMtlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlykurrE&md5=049e2f26a35c71a2cdace40f2d6525b5</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.8b00214&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.8b00214%26sid%3Dliteratum%253Aachs%26aulast%3DStachel%26aufirst%3DS.%2BJ.%26aulast%3DBerger%26aufirst%3DR.%26aulast%3DNomland%26aufirst%3DA.%2BB.%26aulast%3DGinnetti%26aufirst%3DA.%2BT.%26aulast%3DPaone%26aufirst%3DD.%2BV.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DPuri%26aufirst%3DV.%26aulast%3DLange%26aufirst%3DH.%26aulast%3DDrott%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DMarcus%26aufirst%3DJ.%26aulast%3DDwyer%26aufirst%3DM.%2BP.%26aulast%3DSuon%26aufirst%3DS.%26aulast%3DUslaner%26aufirst%3DJ.%2BM.%26aulast%3DSmith%26aufirst%3DS.%2BM.%26atitle%3DStructure-Guided%2520design%2520and%2520procognitive%2520assessment%2520of%2520a%2520potent%2520and%2520selective%2520phosphodiesterase%25202A%2520inhibitor%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2018%26volume%3D9%26spage%3D815%26epage%3D820%26doi%3D10.1021%2Facsmedchemlett.8b00214" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Helal, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyden, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chappie, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fisher, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hajos, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harms, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphrey, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandit, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kleiman, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kormos, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maklad, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDowell, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGinnis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connor, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Donnell, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogden, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piotrowski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seymour, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueno, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vansell, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verhoest, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, E. X.</span></span> <span> </span><span class="NLM_article-title">Identification of a potent, highly selective, and brain penetrant phosphodiesterase 2A inhibitor clinical candidate</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">1001</span>– <span class="NLM_lpage">1018</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01466</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01466" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVWgsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=1001-1018&author=C.+J.+Helalauthor=E.+Arnoldauthor=T.+Boydenauthor=C.+Changauthor=T.+A.+Chappieauthor=E.+Fisherauthor=M.+Hajosauthor=J.+F.+Harmsauthor=W.+E.+Hoffmanauthor=J.+M.+Humphreyauthor=J.+Panditauthor=Z.+Kangauthor=R.+J.+Kleimanauthor=B.+L.+Kormosauthor=C.-W.+Leeauthor=J.+Luauthor=N.+Makladauthor=L.+McDowellauthor=D.+McGinnisauthor=R.+E.+O%E2%80%99Connorauthor=C.+J.+O%E2%80%99Donnellauthor=A.+Ogdenauthor=M.+Piotrowskiauthor=C.+J.+Schmidtauthor=P.+A.+Seymourauthor=H.+Uenoauthor=N.+Vansellauthor=P.+R.+Verhoestauthor=E.+X.+Yang&title=Identification+of+a+potent%2C+highly+selective%2C+and+brain+penetrant+phosphodiesterase+2A+inhibitor+clinical+candidate&doi=10.1021%2Facs.jmedchem.7b01466"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a Potent, Highly Selective, and Brain Penetrant Phosphodiesterase 2A Inhibitor Clinical Candidate</span></div><div class="casAuthors">Helal, Christopher J.; Arnold, Eric; Boyden, Tracey; Chang, Cheng; Chappie, Thomas A.; Fisher, Ethan; Hajos, Mihaly; Harms, John F.; Hoffman, William E.; Humphrey, John M.; Pandit, Jayvardhan; Kang, Zhijun; Kleiman, Robin J.; Kormos, Bethany L.; Lee, Che-Wah; Lu, Jiemin; Maklad, Noha; McDowell, Laura; McGinnis, Dina; OConnor, Rebecca E.; ODonnell, Christopher J.; Ogden, Adam; Piotrowski, Mary; Schmidt, Christopher J.; Seymour, Patricia A.; Ueno, Hirokazu; Vansell, Nichole; Verhoest, Patrick R.; Yang, Edward X.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1001-1018</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Computational modeling was used to direct the synthesis of analogs of previously reported phosphodiesterase 2A (PDE2A) inhibitor (I) with an imidazotriazine core to yield compds. of significantly enhanced potency.  The analog PF-05180999 (30) was subsequently identified as a preclin. candidate targeting cognitive impairment assocd. with schizophrenia.  Compd. 30 demonstrated potent binding to PDE2A in brain tissue, dose responsive mouse brain cGMP increases, and reversal of N-methyl-D-aspartate (NMDA) antagonist-induced (MK-801, ketamine) effects in electrophysiol. and working memory models in rats.  Preclin. pharmacokinetics revealed unbound brain/unbound plasma levels approaching unity and good oral bioavailability resulting in an av. concn. at steady state (Cav,ss) predicted human dose of 30 mg once daily (q.d.).  Modeling of a modified release formulation suggested that 25 mg twice daily (b.i.d.) could maintain plasma levels of 30 at or above targeted efficacious plasma levels for 24 h, which became part of the human clin. plan.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfbdIAg1Vd27Vg90H21EOLACvtfcHk0lghyoMDHyOw4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVWgsg%253D%253D&md5=5424e50970acfe7acb0d29d1b750c380</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01466&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01466%26sid%3Dliteratum%253Aachs%26aulast%3DHelal%26aufirst%3DC.%2BJ.%26aulast%3DArnold%26aufirst%3DE.%26aulast%3DBoyden%26aufirst%3DT.%26aulast%3DChang%26aufirst%3DC.%26aulast%3DChappie%26aufirst%3DT.%2BA.%26aulast%3DFisher%26aufirst%3DE.%26aulast%3DHajos%26aufirst%3DM.%26aulast%3DHarms%26aufirst%3DJ.%2BF.%26aulast%3DHoffman%26aufirst%3DW.%2BE.%26aulast%3DHumphrey%26aufirst%3DJ.%2BM.%26aulast%3DPandit%26aufirst%3DJ.%26aulast%3DKang%26aufirst%3DZ.%26aulast%3DKleiman%26aufirst%3DR.%2BJ.%26aulast%3DKormos%26aufirst%3DB.%2BL.%26aulast%3DLee%26aufirst%3DC.-W.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DMaklad%26aufirst%3DN.%26aulast%3DMcDowell%26aufirst%3DL.%26aulast%3DMcGinnis%26aufirst%3DD.%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DR.%2BE.%26aulast%3DO%25E2%2580%2599Donnell%26aufirst%3DC.%2BJ.%26aulast%3DOgden%26aufirst%3DA.%26aulast%3DPiotrowski%26aufirst%3DM.%26aulast%3DSchmidt%26aufirst%3DC.%2BJ.%26aulast%3DSeymour%26aufirst%3DP.%2BA.%26aulast%3DUeno%26aufirst%3DH.%26aulast%3DVansell%26aufirst%3DN.%26aulast%3DVerhoest%26aufirst%3DP.%2BR.%26aulast%3DYang%26aufirst%3DE.%2BX.%26atitle%3DIdentification%2520of%2520a%2520potent%252C%2520highly%2520selective%252C%2520and%2520brain%2520penetrant%2520phosphodiesterase%25202A%2520inhibitor%2520clinical%2520candidate%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D1001%26epage%3D1018%26doi%3D10.1021%2Facs.jmedchem.7b01466" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buijnsters, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Angelis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langlois, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rombouts, F. J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanderson, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tresadern, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ritchie, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trabanco, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vanhoof, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roosbroeck, Y. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrés, J.-I.</span></span> <span> </span><span class="NLM_article-title">Structure-based design of a potent, selective, and brain penetrating PDE2A inhibitor with demonstrated target engagement</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">1049</span>– <span class="NLM_lpage">1053</span>, <span class="refDoi"> DOI: 10.1021/ml500262u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml500262u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFyqs7%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=1049-1053&author=P.+Buijnstersauthor=M.+De+Angelisauthor=X.+Langloisauthor=F.+J.+R.+Romboutsauthor=W.+Sandersonauthor=G.+Tresadernauthor=A.+Ritchieauthor=A.+A.+Trabancoauthor=G.+Vanhoofauthor=Y.+V.+Roosbroeckauthor=J.-I.+Andr%C3%A9s&title=Structure-based+design+of+a+potent%2C+selective%2C+and+brain+penetrating+PDE2A+inhibitor+with+demonstrated+target+engagement&doi=10.1021%2Fml500262u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of a Potent, Selective, and Brain Penetrating PDE2 Inhibitor with Demonstrated Target Engagement</span></div><div class="casAuthors">Buijnsters, Peter; De Angelis, Meri; Langlois, Xavier; Rombouts, Frederik J. R.; Sanderson, Wendy; Tresadern, Gary; Ritchie, Alison; Trabanco, Andres A.; Van Hoof, Greet; Van Roosbroeck, Yves; Andres, Jose-Ignacio</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1049-1053</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Structure-guided design led to the identification of the novel, potent, and selective phosphodiesterase 2 (PDE2) inhibitor 12.  Compd. 12 demonstrated a >210-fold selectivity vs. PDE10 and PDE11 and was inactive against all other PDE family members up to 10 μM.  In vivo evaluation of 12 provided evidence that it is able to engage the target and to increase cGMP levels in relevant brain regions.  Hence, 12 is a valuable tool compd. for the better understanding of the role of PDE2 in cognitive impairment and other central nervous system related disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonEHLDgf3R67Vg90H21EOLACvtfcHk0lghyoMDHyOw4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFyqs7%252FL&md5=e7d10d94929444876909ebf89dbde404</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fml500262u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml500262u%26sid%3Dliteratum%253Aachs%26aulast%3DBuijnsters%26aufirst%3DP.%26aulast%3DDe%2BAngelis%26aufirst%3DM.%26aulast%3DLanglois%26aufirst%3DX.%26aulast%3DRombouts%26aufirst%3DF.%2BJ.%2BR.%26aulast%3DSanderson%26aufirst%3DW.%26aulast%3DTresadern%26aufirst%3DG.%26aulast%3DRitchie%26aufirst%3DA.%26aulast%3DTrabanco%26aufirst%3DA.%2BA.%26aulast%3DVanhoof%26aufirst%3DG.%26aulast%3DRoosbroeck%26aufirst%3DY.%2BV.%26aulast%3DAndr%25C3%25A9s%26aufirst%3DJ.-I.%26atitle%3DStructure-based%2520design%2520of%2520a%2520potent%252C%2520selective%252C%2520and%2520brain%2520penetrating%2520PDE2A%2520inhibitor%2520with%2520demonstrated%2520target%2520engagement%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D5%26spage%3D1049%26epage%3D1053%26doi%3D10.1021%2Fml500262u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rombouts, F. J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tresadern, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buijnsters, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langlois, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tovar, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinbrecher, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vanhoof, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Somers, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrés, J.-I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trabanco, A. A.-I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trabanco, A. A.</span></span> <span> </span><span class="NLM_article-title">Pyrido[4,3-e][1,2,4]triazolo[4,3-a]pyrazines as selective, brain penetrant phosphodiesterase 2 (PDE2A) inhibitors</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">282</span>– <span class="NLM_lpage">286</span>, <span class="refDoi"> DOI: 10.1021/ml500463t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml500463t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC2MXosVOltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=282-286&author=F.+J.+R.+Romboutsauthor=G.+Tresadernauthor=P.+Buijnstersauthor=X.+Langloisauthor=F.+Tovarauthor=T.+B.+Steinbrecherauthor=G.+Vanhoofauthor=M.+Somersauthor=J.-I.+Andr%C3%A9sauthor=A.+A.-I.+Trabancoauthor=A.+A.+Trabanco&title=Pyrido%5B4%2C3-e%5D%5B1%2C2%2C4%5Dtriazolo%5B4%2C3-a%5Dpyrazines+as+selective%2C+brain+penetrant+phosphodiesterase+2+%28PDE2A%29+inhibitors&doi=10.1021%2Fml500463t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Pyrido[4,3-e][1,2,4]triazolo[4,3-a]pyrazines as Selective, Brain Penetrant Phosphodiesterase 2 (PDE2) Inhibitors</span></div><div class="casAuthors">Rombouts, Frederik J. R.; Tresadern, Gary; Buijnsters, Peter; Langlois, Xavier; Tovar, Fulgencio; Steinbrecher, Thomas B.; Vanhoof, Greet; Somers, Marijke; Andres, Jose-Ignacio; Trabanco, Andres A.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">282-286</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel series of pyrido[4,3-e][1,2,4]triazolo[4,3-a]pyrazines is reported as potent PDE2/PDE10 inhibitors with drug-like properties.  Selectivity for PDE2 was obtained by introducing a linear, lipophilic moiety on the meta-position of the Ph ring pending from the triazole.  The SAR and protein flexibility were explored with free energy perturbation calcns.  Rat pharmacokinetic data and in vivo receptor occupancy data are given for two representative compds. 6 and 12.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbUtc8164aobVg90H21EOLACvtfcHk0lghyoMDHyOw4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXosVOltA%253D%253D&md5=d92a3c22cee3c286c2608d5cf719b14c</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fml500463t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml500463t%26sid%3Dliteratum%253Aachs%26aulast%3DRombouts%26aufirst%3DF.%2BJ.%2BR.%26aulast%3DTresadern%26aufirst%3DG.%26aulast%3DBuijnsters%26aufirst%3DP.%26aulast%3DLanglois%26aufirst%3DX.%26aulast%3DTovar%26aufirst%3DF.%26aulast%3DSteinbrecher%26aufirst%3DT.%2BB.%26aulast%3DVanhoof%26aufirst%3DG.%26aulast%3DSomers%26aufirst%3DM.%26aulast%3DAndr%25C3%25A9s%26aufirst%3DJ.-I.%26aulast%3DTrabanco%26aufirst%3DA.%2BA.-I.%26aulast%3DTrabanco%26aufirst%3DA.%2BA.%26atitle%3DPyrido%255B4%252C3-e%255D%255B1%252C2%252C4%255Dtriazolo%255B4%252C3-a%255Dpyrazines%2520as%2520selective%252C%2520brain%2520penetrant%2520phosphodiesterase%25202%2520%2528PDE2A%2529%2520inhibitors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D282%26epage%3D286%26doi%3D10.1021%2Fml500463t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span> <span> </span><span class="NLM_article-title">Efficient drug lead discovery and optimization</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">724</span>– <span class="NLM_lpage">733</span>, <span class="refDoi"> DOI: 10.1021/ar800236t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar800236t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjsFentr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2009&pages=724-733&author=W.+L.+Jorgensen&title=Efficient+drug+lead+discovery+and+optimization&doi=10.1021%2Far800236t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Efficient Drug Lead Discovery and Optimization</span></div><div class="casAuthors">Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">724-733</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  During the 1980s, advances in the abilities to perform computer simulations of chem. and biomol. systems and to calc. free energy changes led to the expectation that such methodol. would soon show great utility for guiding mol. design.  Important potential applications included design of selective receptors, catalysts, and regulators of biol. function including enzyme inhibitors.  This time also saw the rise of high-throughput screening and combinatorial chem. along with complementary computational methods for de novo design and virtual screening including docking.  These technologies appeared poised to deliver diverse lead compds. for any biol. target.  As with many technol. advances, realization of the expectations required significant addnl. effort and time.  However, as summarized here, striking success has now been achieved for computer-aided drug lead generation and optimization.  De novo design using both mol. growing and docking are illustrated for lead generation, and lead optimization features free energy perturbation calcns. in conjunction with Monte Carlo statistical mechanics simulations for protein-inhibitor complexes in aq. soln.  The specific applications are to the discovery of non-nucleoside inhibitors of HIV reverse transcriptase (HIV-RT) and inhibitors of the binding of the proinflammatory cytokine MIF to its receptor CD74.  A std. protocol is presented that includes scans for possible addns. of small substituents to a mol. core, interchange of heterocycles, and focused optimization of substituents at one site.  Initial leads with activities at low-micromolar concns. have been advanced rapidly to low-nanomolar inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrn0YpWuo1jCLVg90H21EOLACvtfcHk0lhULksKJAgndA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjsFentr4%253D&md5=9641587769701c6541b38e18fa05538a</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Far800236t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far800236t%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DEfficient%2520drug%2520lead%2520discovery%2520and%2520optimization%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2009%26volume%3D42%26spage%3D724%26epage%3D733%26doi%3D10.1021%2Far800236t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harder, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berne, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span> <span> </span><span class="NLM_article-title">Advancing drug discovery through enhanced free energy calculations</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">1625</span>– <span class="NLM_lpage">1632</span>, <span class="refDoi"> DOI: 10.1021/acs.accounts.7b00083</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.accounts.7b00083" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFSiur3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2017&pages=1625-1632&author=R.+Abelauthor=L.+Wangauthor=E.+D.+Harderauthor=B.+J.+Berneauthor=R.+A.+Friesner&title=Advancing+drug+discovery+through+enhanced+free+energy+calculations&doi=10.1021%2Facs.accounts.7b00083"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Advancing Drug Discovery through Enhanced Free Energy Calculations</span></div><div class="casAuthors">Abel, Robert; Wang, Lingle; Harder, Edward D.; Berne, B. J.; Friesner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1625-1632</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  A principal goal of drug discovery project is to design mols. that can tightly and selectively bind to the target protein receptor.  Accurate prediction of protein-ligand binding free energies is therefore of central importance in computational chem. and computer aided drug design.  Multiple recent improvements in computing power, classical force field accuracy, enhanced sampling methods, and simulation setup, have enabled accurate and reliable calcns. of protein-ligands binding free energies, and position free energy calcns. to play a guiding role in small mol. drug discovery.  In this accounts, the authors outline the relevant methodol. advances, including the REST2 (Replica Exchange with Solute Temperting) enhanced sampling, the incorporation of REST2 sampling with conventional FEP (Free Energy Perturbation) through FEP/REST, the OPLS3 force fields, and the advanced simulation set up that constitute the authors' FEP+ approach, followed by the presentation of extensive comparisons with expt., demonstrating sufficient accuracy in potency prediction (better than 1 kcal/mol) to substantially impact lead optimization campaigns.  The limitations of the current FEP+ implementation and best practices in drug discovery applications are also discussed followed by the future methodol. development plans to address those limitations.  The authors then report results from a recent drug discovery project, in which several thousand FEP+ calcns. were successfully deployed to simultaneously optimize potency, selectivity, and soly., illustrating the power of the approach to solve challenging drug design problems.  The capabilities of free energy calcns. to accurately predict potency and selectivity have led to the advance of ongoing drug discovery projects, in challenging situations where alternative approaches would have great difficulties.  The ability to effectively carry out projects evaluating tens of thousands, or hundreds of thousands, of proposed drug candidates, is potentially transformative in enabling hard to drug targets to be attacked, and in facilitating the development of superior compds., in various dimensions, for a wide range of targets.  More effective integration of FEP+ calcns. into the drug discovery process will ensure that the results are deployed in an optimal fashion for yielding the best possible compds. entering the clinic; this is where the greatest payoff is in the exploitation of computer driven design capabilities.  A key conclusion from the work described is the surprisingly robust and accurate results that are attainable within the conventional classical simulation, fixed charge paradigm.  No doubt there are individual cases that would benefit from a more sophisticated energy model or dynamical treatment, and properties other than protein-ligand binding energies may be more sensitive to these approxns.  The authors conclude that an inflection point in the ability of MD simulations to impact drug discovery has now been attained, due to the confluence of hardware and software development along with the formulation of "good enough" theor. methods and models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_VeCPmDRlf7Vg90H21EOLACvtfcHk0lhULksKJAgndA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFSiur3K&md5=46d07ee6f6f129a126f0fde40a62ffe2</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facs.accounts.7b00083&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.accounts.7b00083%26sid%3Dliteratum%253Aachs%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DHarder%26aufirst%3DE.%2BD.%26aulast%3DBerne%26aufirst%3DB.%2BJ.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DAdvancing%2520drug%2520discovery%2520through%2520enhanced%2520free%2520energy%2520calculations%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2017%26volume%3D50%26spage%3D1625%26epage%3D1632%26doi%3D10.1021%2Facs.accounts.7b00083" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cournia, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, W.</span></span> <span> </span><span class="NLM_article-title">Relative binding free energy calculations in drug discovery: Recent advances and practical considerations</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">2911</span>– <span class="NLM_lpage">2937</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.7b00564</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.7b00564" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFOhu7fM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2017&pages=2911-2937&author=Z.+Courniaauthor=B.+Allenauthor=W.+Sherman&title=Relative+binding+free+energy+calculations+in+drug+discovery%3A+Recent+advances+and+practical+considerations&doi=10.1021%2Facs.jcim.7b00564"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Relative Binding Free Energy Calculations in Drug Discovery: Recent Advances and Practical Considerations</span></div><div class="casAuthors">Cournia, Zoe; Allen, Bryce; Sherman, Woody</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2911-2937</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Accurate in silico prediction of protein-ligand binding affinities has been a primary objective of structure-based drug design for decades due to the putative value it would bring to the drug discovery process.  However, computational methods have historically failed to deliver value in real-world drug discovery applications due to a variety of scientific, tech., and practical challenges.  Recently, a family of approaches commonly referred to as relative binding free energy (RBFE) calcns., which rely on physics-based mol. simulations and statistical mechanics, have shown promise in reliably generating accurate predictions in the context of drug discovery projects.  This advance arises from accumulating developments in the underlying scientific methods (decades of research on force fields and sampling algorithms) coupled with vast increases in computational resources (graphics processing units and cloud infrastructures).  Mounting evidence from retrospective validation studies, blind challenge predictions, and prospective applications suggests that RBFE simulations can now predict the affinity differences for congeneric ligands with sufficient accuracy and throughput to deliver considerable value in hit-to-lead and lead optimization efforts.  Here, the authors present an overview of current RBFE implementations, highlighting recent advances and remaining challenges, along with examples that emphasize practical considerations for obtaining reliable RBFE results.  The authors focus specifically on relative binding free energies because the calcns. are less computationally intensive than abs. binding free energy (ABFE) calcns. and map directly onto the hit-to-lead and lead optimization processes, where the prediction of relative binding energies between a ref. mol. and new ideas (virtual mols.) can be used to prioritize mols. for synthesis.  The authors describe the crit. aspects of running RBFE calcns., from both theor. and applied perspectives, using a combination of retrospective literature examples and prospective studies from drug discovery projects.  This work is intended to provide a contemporary overview of the scientific, tech., and practical issues assocd. with running relative binding free energy simulations, with a focus on real-world drug discovery applications.  The authors offer guidelines for improving the accuracy of RBFE simulations, esp. for challenging cases, and emphasize unresolved issues that could be improved by further research in the field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokxZRvplrWFbVg90H21EOLACvtfcHk0lhULksKJAgndA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFOhu7fM&md5=ffcc9f970ab660eced8d3343ccefeec6</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.7b00564&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.7b00564%26sid%3Dliteratum%253Aachs%26aulast%3DCournia%26aufirst%3DZ.%26aulast%3DAllen%26aufirst%3DB.%26aulast%3DSherman%26aufirst%3DW.%26atitle%3DRelative%2520binding%2520free%2520energy%2520calculations%2520in%2520drug%2520discovery%253A%2520Recent%2520advances%2520and%2520practical%2520considerations%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2017%26volume%3D57%26spage%3D2911%26epage%3D2937%26doi%3D10.1021%2Facs.jcim.7b00564" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ciordia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pérez-Benito, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delgado, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trabanco, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tresadern, G.</span></span> <span> </span><span class="NLM_article-title">Application of free energy perturbation for the design of BACE1 inhibitors</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">1856</span>– <span class="NLM_lpage">1871</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.6b00220</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.6b00220" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht12mu7%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2016&pages=1856-1871&author=M.+Ciordiaauthor=L.+P%C3%A9rez-Benitoauthor=F.+Delgadoauthor=A.+A.+Trabancoauthor=G.+Tresadern&title=Application+of+free+energy+perturbation+for+the+design+of+BACE1+inhibitors&doi=10.1021%2Facs.jcim.6b00220"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Application of Free Energy Perturbation for the Design of BACE1 Inhibitors</span></div><div class="casAuthors">Ciordia, Myriam; Perez-Benito, Laura; Delgado, Francisca; Trabanco, Andres A.; Tresadern, Gary</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1856-1871</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Novel spiroaminodihydropyrroles probing for optimized interactions at the P3 pocket of β-secretase 1 (BACE1) were designed with the use of free energy perturbation (FEP) calcns.  The resulting mols. showed pIC50 potencies in enzymic BACE1 inhibition assays ranging from approx. 5 to 7.  Good correlation was obsd. between the predicted activity from the FEP calcns. and exptl. activity.  Simulations run with a default 5 ns approach delivered a mean unsigned error (MUE) between prediction and expt. of 0.58 and 0.91 kcal/mol for retrospective and prospective applications, resp.  With longer simulations of 10 and 20 ns, the MUE was in both cases 0.57 kcal/mol for the retrospective application, and 0.69 and 0.59 kcal/mol for the prospective application.  Other considerations that impact the quality of the calcns. are discussed.  This work provides an example of the value of FEP as a computational tool for drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqpi-OiII-FrVg90H21EOLACvtfcHk0lieNyOPU7tXVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht12mu7%252FK&md5=b7577a89492867fc334cb1ec937a4eb6</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.6b00220&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.6b00220%26sid%3Dliteratum%253Aachs%26aulast%3DCiordia%26aufirst%3DM.%26aulast%3DP%25C3%25A9rez-Benito%26aufirst%3DL.%26aulast%3DDelgado%26aufirst%3DF.%26aulast%3DTrabanco%26aufirst%3DA.%2BA.%26aulast%3DTresadern%26aufirst%3DG.%26atitle%3DApplication%2520of%2520free%2520energy%2520perturbation%2520for%2520the%2520design%2520of%2520BACE1%2520inhibitors%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2016%26volume%3D56%26spage%3D1856%26epage%3D1871%26doi%3D10.1021%2Facs.jcim.6b00220" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keränen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pérez-Benito, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciordia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delgado, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinbrecher, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oehlrich, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Vlijmen, H. W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trabanco, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tresadern, G.</span></span> <span> </span><span class="NLM_article-title">Acylguanidine beta secretase 1 inhibitors: A combined experimental and free energy perturbation study</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">1439</span>– <span class="NLM_lpage">1453</span>, <span class="refDoi"> DOI: 10.1021/acs.jctc.6b01141</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.6b01141" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1ehsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=1439-1453&author=H.+Ker%C3%A4nenauthor=L.+P%C3%A9rez-Benitoauthor=M.+Ciordiaauthor=F.+Delgadoauthor=T.+B.+Steinbrecherauthor=D.+Oehlrichauthor=H.+W.+T.+van+Vlijmenauthor=A.+A.+Trabancoauthor=G.+Tresadern&title=Acylguanidine+beta+secretase+1+inhibitors%3A+A+combined+experimental+and+free+energy+perturbation+study&doi=10.1021%2Facs.jctc.6b01141"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Acylguanidine Beta Secretase 1 Inhibitors: A Combined Experimental and Free Energy Perturbation Study</span></div><div class="casAuthors">Keranen, Henrik; Perez-Benito, Laura; Ciordia, Myriam; Delgado, Francisca; Steinbrecher, Thomas B.; Oehlrich, Daniel; van Vlijmen, Herman W. T.; Trabanco, Andres A.; Tresadern, Gary</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1439-1453</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of acylguanidine beta secretase 1 (BACE1) inhibitors with modified scaffold and P3 pocket substituent was synthesized and studied with Free Energy Perturbation (FEP) calcns.  The resulting mols. showed potencies in enzymic BACE1 inhibition assays up to 1 nM.  The correlation between the predicted activity from the FEP calcns. and the exptl. activity was good for the P3 pocket substituents.  The av. mean unsigned error (MUE) between prediction and expt. was 0.68±0.17 kcal/mol for the default 5 ns lambda window simulation time improving to 0.35±0.13 kcal/mol for 40 ns.  FEP calcns. for the P2' pocket substituents on the same acylguanidine scaffold also showed good agreement with expt. and the results remained stable with repeated simulations and increased simulation time.  It proved more difficult to use FEP calcns. to study the scaffold modification from increasing 5 to 6 and 7 membered-rings.  Although prediction and expt. were in agreement for short 2 ns simulations, as the simulation time increased the results diverged.  This was improved by the use of a newly developed 'Core Hopping FEP+' approach, which also showed improved stability in repeat calcns.  The origins of these differences along with the value of repeat and longer simulation times are discussed.  This work provides a further example of the use of FEP as a computational tool for mol. design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmHZjygsk_PbVg90H21EOLACvtfcHk0lieNyOPU7tXVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1ehsrg%253D&md5=987ab8fc475670d0ba19aa086519a4ba</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.6b01141&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.6b01141%26sid%3Dliteratum%253Aachs%26aulast%3DKer%25C3%25A4nen%26aufirst%3DH.%26aulast%3DP%25C3%25A9rez-Benito%26aufirst%3DL.%26aulast%3DCiordia%26aufirst%3DM.%26aulast%3DDelgado%26aufirst%3DF.%26aulast%3DSteinbrecher%26aufirst%3DT.%2BB.%26aulast%3DOehlrich%26aufirst%3DD.%26aulast%3Dvan%2BVlijmen%26aufirst%3DH.%2BW.%2BT.%26aulast%3DTrabanco%26aufirst%3DA.%2BA.%26aulast%3DTresadern%26aufirst%3DG.%26atitle%3DAcylguanidine%2520beta%2520secretase%25201%2520inhibitors%253A%2520A%2520combined%2520experimental%2520and%2520free%2520energy%2520perturbation%2520study%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2017%26volume%3D13%26spage%3D1439%26epage%3D1453%26doi%3D10.1021%2Facs.jctc.6b01141" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deflorian, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pérez-Benito, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lenselink, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Congreve, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Vlijmen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mason, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Graaf, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tresadern, G.</span></span> <span> </span><span class="NLM_article-title">Accurate prediction of GPCR ligand binding affinity with free energy perturbation</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">4311</span>– <span class="NLM_lpage">4325</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.0c00449</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.0c00449" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2020&pages=4311-4325&author=F.+Deflorianauthor=L.+P%C3%A9rez-Benitoauthor=E.+B.+Lenselinkauthor=M.+Congreveauthor=H.+Van+Vlijmenauthor=J.+S.+Masonauthor=C.+de+Graafauthor=G.+Tresadern&title=Accurate+prediction+of+GPCR+ligand+binding+affinity+with+free+energy+perturbation&doi=10.1021%2Facs.jcim.0c00449"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.0c00449&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.0c00449%26sid%3Dliteratum%253Aachs%26aulast%3DDeflorian%26aufirst%3DF.%26aulast%3DP%25C3%25A9rez-Benito%26aufirst%3DL.%26aulast%3DLenselink%26aufirst%3DE.%2BB.%26aulast%3DCongreve%26aufirst%3DM.%26aulast%3DVan%2BVlijmen%26aufirst%3DH.%26aulast%3DMason%26aufirst%3DJ.%2BS.%26aulast%3Dde%2BGraaf%26aufirst%3DC.%26aulast%3DTresadern%26aufirst%3DG.%26atitle%3DAccurate%2520prediction%2520of%2520GPCR%2520ligand%2520binding%2520affinity%2520with%2520free%2520energy%2520perturbation%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2020%26volume%3D60%26spage%3D4311%26epage%3D4325%26doi%3D10.1021%2Facs.jcim.0c00449" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pérez-Benito, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keränen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Vlijmen, H. W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tresadern, G.</span></span> <span> </span><span class="NLM_article-title">Predicting binding free energies of PDE2A inhibitors. The difficulties of protein conformation</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">4883</span>, <span class="refDoi"> DOI: 10.1038/s41598-018-23039-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.1038%2Fs41598-018-23039-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=29559702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A280%3ADC%252BC1MnjtFKlsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=4883&author=L.+P%C3%A9rez-Benitoauthor=H.+Ker%C3%A4nenauthor=H.+W.+T.+van+Vlijmenauthor=G.+Tresadern&title=Predicting+binding+free+energies+of+PDE2A+inhibitors.+The+difficulties+of+protein+conformation&doi=10.1038%2Fs41598-018-23039-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting Binding Free Energies of PDE2 Inhibitors. The Difficulties of Protein Conformation</span></div><div class="casAuthors">Perez-Benito Laura; Keranen Henrik; van Vlijmen Herman; Tresadern Gary; Keranen Henrik</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">4883</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">A congeneric series of 21 phosphodiesterase 2 (PDE2) inhibitors are reported.  Crystal structures show how the molecules can occupy a 'top-pocket' of the active site.  Molecules with small substituents do not enter the pocket, a critical leucine (Leu770) is closed and water molecules are present.  Large substituents enter the pocket, opening the Leu770 conformation and displacing the waters.  We also report an X-ray structure revealing a new conformation of the PDE2 active site domain.  The relative binding affinities of these compounds were studied with free energy perturbation (FEP) methods and it represents an attractive real-world test case.  In general, the calculations could predict the energy of small-to-small, or large-to-large molecule perturbations.  However, accurately capturing the transition from small-to-large proved challenging.  Only when using alternative protein conformations did results improve.  The new X-ray structure, along with a modelled dimer, conferred stability to the catalytic domain during the FEP molecular dynamics (MD) simulations, increasing the convergence and thereby improving the prediction of ΔΔG of binding for some small-to-large transitions.  In summary, we found the most significant improvement in results when using different protein structures, and this data set is useful for future free energy validation studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTMdLif4rzX6u_Q9URG6Dx8fW6udTcc2eY5T7vaQUNaZrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MnjtFKlsA%253D%253D&md5=44c26c5bd2fe3e35bba344abb8262eeb</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Fs41598-018-23039-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-018-23039-5%26sid%3Dliteratum%253Aachs%26aulast%3DP%25C3%25A9rez-Benito%26aufirst%3DL.%26aulast%3DKer%25C3%25A4nen%26aufirst%3DH.%26aulast%3Dvan%2BVlijmen%26aufirst%3DH.%2BW.%2BT.%26aulast%3DTresadern%26aufirst%3DG.%26atitle%3DPredicting%2520binding%2520free%2520energies%2520of%2520PDE2A%2520inhibitors.%2520The%2520difficulties%2520of%2520protein%2520conformation%26jtitle%3DSci.%2520Rep.%26date%3D2018%26volume%3D8%26spage%3D4883%26doi%3D10.1038%2Fs41598-018-23039-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pérez-Benito, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casajuana-Martin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiménez-Rosés, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Vlijmen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tresadern, G.</span></span> <span> </span><span class="NLM_article-title">Predicting activity cliffs with free-energy perturbation</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">1884</span>– <span class="NLM_lpage">1895</span>, <span class="refDoi"> DOI: 10.1021/acs.jctc.8b01290</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.8b01290" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjtFKktr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2019&pages=1884-1895&author=L.+P%C3%A9rez-Benitoauthor=N.+Casajuana-Martinauthor=M.+Jim%C3%A9nez-Ros%C3%A9sauthor=H.+van+Vlijmenauthor=G.+Tresadern&title=Predicting+activity+cliffs+with+free-energy+perturbation&doi=10.1021%2Facs.jctc.8b01290"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting Activity Cliffs with Free-Energy Perturbation</span></div><div class="casAuthors">Perez-Benito, Laura; Casajuana-Martin, Nil; Jimenez-Roses, Mireia; van Vlijmen, Herman; Tresadern, Gary</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1884-1895</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Activity cliffs (ACs) are an important type of structure-activity relationship in medicinal chem. where small structural changes result in unexpectedly large differences in biol. activity.  Being able to predict these changes would have a profound impact on lead optimization of drug candidates.  Free-energy perturbation is an ideal tool for predicting relative binding energy differences for small structural modifications, but its performance for ACs is unknown.  Here, we show that FEP can on av. predict ACs to within 1.39 kcal/mol of expt. (∼1 log unit of activity).  We performed FEP calcns. with two different software methods: Schr.ovrddot.odinger-Desmond FEP+ and GROMACS implementations.  There was qual. agreement in the results from the two methods, and quant. the error for one data set was identical, 1.43 kcal/mol, but FEP+ performed better in the second, with errors of 1.17 vs. 1.90 kcal/mol.  The results have far-reaching implications, suggesting well-implemented FEP calcns. can have a major impact on computational drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-1Z4ZWIjgcbVg90H21EOLACvtfcHk0lhAsGyH7y4gnw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjtFKktr8%253D&md5=59a4c8d38efece18891da4266f6bc3d5</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.8b01290&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.8b01290%26sid%3Dliteratum%253Aachs%26aulast%3DP%25C3%25A9rez-Benito%26aufirst%3DL.%26aulast%3DCasajuana-Martin%26aufirst%3DN.%26aulast%3DJim%25C3%25A9nez-Ros%25C3%25A9s%26aufirst%3DM.%26aulast%3Dvan%2BVlijmen%26aufirst%3DH.%26aulast%3DTresadern%26aufirst%3DG.%26atitle%3DPredicting%2520activity%2520cliffs%2520with%2520free-energy%2520perturbation%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2019%26volume%3D15%26spage%3D1884%26epage%3D1895%26doi%3D10.1021%2Facs.jctc.8b01290" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gapsys, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pérez-Benito, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aldeghi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seeliger, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Vlijmen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tresadern, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Groot, B. L.</span></span> <span> </span><span class="NLM_article-title">Large scale relative protein ligand binding affinities using non-equilibrium alchemy</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1140</span>– <span class="NLM_lpage">1152</span>, <span class="refDoi"> DOI: 10.1039/c9sc03754c</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.1039%2Fc9sc03754c" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1OrsLbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=1140-1152&author=V.+Gapsysauthor=L.+P%C3%A9rez-Benitoauthor=M.+Aldeghiauthor=D.+Seeligerauthor=H.+van+Vlijmenauthor=G.+Tresadernauthor=B.+L.+de+Groot&title=Large+scale+relative+protein+ligand+binding+affinities+using+non-equilibrium+alchemy&doi=10.1039%2Fc9sc03754c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Large scale relative protein ligand binding affinities using non-equilibrium alchemy</span></div><div class="casAuthors">Gapsys, Vytautas; Perez-Benito, Laura; Aldeghi, Matteo; Seeliger, Daniel; van Vlijmen, Herman; Tresadern, Gary; de Groot, Bert L.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1140-1152</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Ligand binding affinity calcns. based on mol. dynamics (MD) simulations and non-phys. (alchem.) thermodn. cycles have shown great promise for structure-based drug design.  However, their broad uptake and impact is held back by the notoriously complex setup of the calcns.  Only a few tools other than the free energy perturbation approach by Schrodinger Inc. (referred to as FEP+) currently enable end-to-end application.  Here, we present for the first time an approach based on the open-source software pmx that allows to easily set up and run alchem. calcns. for diverse sets of small mols. using the GROMACS MD engine.  The method relies on theor. rigorous non-equil. thermodn. integration (TI) foundations, and its flexibility allows calcns. with multiple force fields.  In this study, results from the Amber and Charmm force fields were combined to yield a consensus outcome performing on par with the com. FEP+ approach.  A large dataset of 482 perturbations from 13 different protein-ligand datasets led to an av. unsigned error (AUE) of 3.64 ± 0.14 kJ mol-1, equiv. to Schrodinger's FEP+ AUE of 3.66 ± 0.14 kJ mol-1.  For the first time, a setup is presented for overall high precision and high accuracy relative protein-ligand alchem. free energy calcns. based on open-source software.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqM_Ma3c_2AGrVg90H21EOLACvtfcHk0lhAsGyH7y4gnw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1OrsLbJ&md5=56ae345e5332032a3760776215fc53df</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1039%2Fc9sc03754c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc9sc03754c%26sid%3Dliteratum%253Aachs%26aulast%3DGapsys%26aufirst%3DV.%26aulast%3DP%25C3%25A9rez-Benito%26aufirst%3DL.%26aulast%3DAldeghi%26aufirst%3DM.%26aulast%3DSeeliger%26aufirst%3DD.%26aulast%3Dvan%2BVlijmen%26aufirst%3DH.%26aulast%3DTresadern%26aufirst%3DG.%26aulast%3Dde%2BGroot%26aufirst%3DB.%2BL.%26atitle%3DLarge%2520scale%2520relative%2520protein%2520ligand%2520binding%2520affinities%2520using%2520non-equilibrium%2520alchemy%26jtitle%3DChem.%2520Sci.%26date%3D2020%26volume%3D11%26spage%3D1140%26epage%3D1152%26doi%3D10.1039%2Fc9sc03754c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tresadern, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pérez-Benito, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vlijmen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coveney, P. V.</span></span> <span> </span><span class="NLM_article-title">Accuracy and precision of alchemical relative free energy predictions with and without replica-exchange</span>. <i>Adv. Theory Simul.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">1900195</span>, <span class="refDoi"> DOI: 10.1002/adts.201900195</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.1002%2Fadts.201900195" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2019&pages=1900195&author=S.+Wanauthor=G.+Tresadernauthor=L.+P%C3%A9rez-Benitoauthor=H.+Vlijmenauthor=P.+V.+Coveney&title=Accuracy+and+precision+of+alchemical+relative+free+energy+predictions+with+and+without+replica-exchange&doi=10.1002%2Fadts.201900195"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1002%2Fadts.201900195&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fadts.201900195%26sid%3Dliteratum%253Aachs%26aulast%3DWan%26aufirst%3DS.%26aulast%3DTresadern%26aufirst%3DG.%26aulast%3DP%25C3%25A9rez-Benito%26aufirst%3DL.%26aulast%3DVlijmen%26aufirst%3DH.%26aulast%3DCoveney%26aufirst%3DP.%2BV.%26atitle%3DAccuracy%2520and%2520precision%2520of%2520alchemical%2520relative%2520free%2520energy%2520predictions%2520with%2520and%2520without%2520replica-exchange%26jtitle%3DAdv.%2520Theory%2520Simul.%26date%3D2019%26volume%3D3%26spage%3D1900195%26doi%3D10.1002%2Fadts.201900195" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K. Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Card, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Artis, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fong, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gillette, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsieh, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neiman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milburn, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlessinger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bollag, G.</span></span> <span> </span><span class="NLM_article-title">Glutamine switch mechanism for nucleotide selectivity by phosphodiesterases</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">279</span>– <span class="NLM_lpage">286</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2004.07.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.1016%2Fj.molcel.2004.07.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=15260978" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BD2cXms1ektLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2004&pages=279-286&author=K.+Y.+J.+Zhangauthor=G.+L.+Cardauthor=Y.+Suzukiauthor=D.+R.+Artisauthor=D.+Fongauthor=S.+Gilletteauthor=D.+Hsiehauthor=J.+Neimanauthor=B.+L.+Westauthor=C.+Zhangauthor=M.+V.+Milburnauthor=S.-H.+Kimauthor=J.+Schlessingerauthor=G.+Bollag&title=Glutamine+switch+mechanism+for+nucleotide+selectivity+by+phosphodiesterases&doi=10.1016%2Fj.molcel.2004.07.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">A glutamine switch mechanism for nucleotide selectivity by phosphodiesterases</span></div><div class="casAuthors">Zhang, Kam Y. J.; Card, Graeme L.; Suzuki, Yoshihisa; Artis, D. Richard; Fong, Daniel; Gillette, Sam; Hsieh, Davin; Neiman, Joshua; West, Brian L.; Zhang, Chao; Milburn, Michael V.; Kim, Sung-Hou; Schlessinger, Joseph; Bollag, Gideon</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">279-286</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Phosphodiesterases (PDEs) comprise a family of enzymes that modulate the immune response, inflammation, and memory, among many other functions.  There are three types of PDEs: cAMP-specific, cGMP-specific, and dual-specific.  Here we describe the mechanism of nucleotide selectivity on the basis of high-resoln. co-crystal structures of the cAMP-specific PDE4B and PDE4D with AMP, the cGMP-specific PDE5A with GMP, and the apo-structure of the dual-specific PDE1B.  These structures show that an invariant glutamine functions as the key specificity determinant by a "glutamine switch" mechanism for recognizing the purine moiety in cAMP or cGMP.  The surrounding residues anchor the glutamine residue in different orientations for cAMP and for cGMP.  The PDE1B structure shows that in dual-specific PDEs a key histidine residue may enable the invariant glutamine to toggle between cAMP and cGMP.  The structural understanding of nucleotide binding enables the design of new PDE inhibitors that may treat diseases in which cyclic nucleotides play a crit. role.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorRXGTDdAa1bVg90H21EOLACvtfcHk0lhAsGyH7y4gnw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXms1ektLs%253D&md5=32fab6a75031a67290ec5eda2c3e080b</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2004.07.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2004.07.005%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DK.%2BY.%2BJ.%26aulast%3DCard%26aufirst%3DG.%2BL.%26aulast%3DSuzuki%26aufirst%3DY.%26aulast%3DArtis%26aufirst%3DD.%2BR.%26aulast%3DFong%26aufirst%3DD.%26aulast%3DGillette%26aufirst%3DS.%26aulast%3DHsieh%26aufirst%3DD.%26aulast%3DNeiman%26aufirst%3DJ.%26aulast%3DWest%26aufirst%3DB.%2BL.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DMilburn%26aufirst%3DM.%2BV.%26aulast%3DKim%26aufirst%3DS.-H.%26aulast%3DSchlessinger%26aufirst%3DJ.%26aulast%3DBollag%26aufirst%3DG.%26atitle%3DGlutamine%2520switch%2520mechanism%2520for%2520nucleotide%2520selectivity%2520by%2520phosphodiesterases%26jtitle%3DMol.%2520Cell%26date%3D2004%26volume%3D15%26spage%3D279%26epage%3D286%26doi%3D10.1016%2Fj.molcel.2004.07.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jansen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kooistra, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanev, G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leurs, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Esch, I. J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Graaf, C.</span></span> <span> </span><span class="NLM_article-title">PDEStrIAn: A Phosphodiesterase structure and ligand interaction annotated database as a tool for structure-based drug design</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">7029</span>– <span class="NLM_lpage">7065</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01813</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01813" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC28XjtVCju7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=7029-7065&author=C.+Jansenauthor=A.+J.+Kooistraauthor=G.+K.+Kanevauthor=R.+Leursauthor=I.+J.+P.+de+Eschauthor=C.+de+Graaf&title=PDEStrIAn%3A+A+Phosphodiesterase+structure+and+ligand+interaction+annotated+database+as+a+tool+for+structure-based+drug+design&doi=10.1021%2Facs.jmedchem.5b01813"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">PDEStrIAn: A Phosphodiesterase Structure and Ligand Interaction Annotated Database As a Tool for Structure-Based Drug Design</span></div><div class="casAuthors">Jansen, Chimed; Kooistra, Albert J.; Kanev, Georgi K.; Leurs, Rob; de Esch, Iwan J. P.; de Graaf, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">7029-7065</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A systematic anal. is presented of the 220 phosphodiesterase (PDE) catalytic domain crystal structures present in the Protein Data Bank (PDB) with a focus on PDE-ligand interactions.  The consistent structural alignment of 57 PDE ligand binding site residues enables the systematic anal. of PDE-ligand interaction fingerprints (IFPs), the identification of subtype-specific PDE-ligand interaction features, and the classification of ligands according to their binding modes.  We illustrate how systematic mining of this phosphodiesterase structure and ligand interaction annotated (PDEStrIAn) database provides new insights into how conserved and selective PDE interaction hot spots can accommodate the large diversity of chem. scaffolds in PDE ligands.  A substructure anal. of the cocrystd. PDE ligands in combination with those in the ChEMBL database provides a toolbox for scaffold hopping and ligand design.  These analyses lead to an improved understanding of the structural requirements of PDE binding that will be useful in future drug discovery studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-1I6fmiCv-rVg90H21EOLACvtfcHk0ljNHgeoSPkZfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjtVCju7k%253D&md5=67669586d18089a73f4e7de21cf7c33b</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01813&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01813%26sid%3Dliteratum%253Aachs%26aulast%3DJansen%26aufirst%3DC.%26aulast%3DKooistra%26aufirst%3DA.%2BJ.%26aulast%3DKanev%26aufirst%3DG.%2BK.%26aulast%3DLeurs%26aufirst%3DR.%26aulast%3Dde%2BEsch%26aufirst%3DI.%2BJ.%2BP.%26aulast%3Dde%2BGraaf%26aufirst%3DC.%26atitle%3DPDEStrIAn%253A%2520A%2520Phosphodiesterase%2520structure%2520and%2520ligand%2520interaction%2520annotated%2520database%2520as%2520a%2520tool%2520for%2520structure-based%2520drug%2520design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D7029%26epage%3D7065%26doi%3D10.1021%2Facs.jmedchem.5b01813" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brandon, N. J.</span></span> <span> </span><span class="NLM_article-title">Emerging biology of PDE10A</span>. <i>Curr. Pharm. Des.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">378</span>– <span class="NLM_lpage">388</span>, <span class="refDoi"> DOI: 10.2174/1381612820666140826114744</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.2174%2F1381612820666140826114744" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=25159072" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFOntr%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=378-388&author=L.+S.+Wilsonauthor=N.+J.+Brandon&title=Emerging+biology+of+PDE10A&doi=10.2174%2F1381612820666140826114744"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging Biology of PDE10A</span></div><div class="casAuthors">Wilson, Lindsay S.; Brandon, Nicholas J.</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">378-388</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  CAMP and cyclic GMP are essential second messengers that regulate multiple signaling pathways in virtually all cell types.  Their accumulation in cells is finely regulated by cyclic nucleotide phosphodiesterases (PDEs), the only enzymes that can degrade these signaling mols. and thus provide exquisite control over intracellular signaling processes.  One PDE family, PDE10A, is highly enriched in the brain and its unique expression profile in specific brain regions of interest, in particular to antipsychotic treatment, has made it an attractive therapeutic target for the treatment of schizophrenia.  However, after a Phase II trial failure of a selective PDE10A inhibitor for the treatment of schizophrenia, it has encouraged the field to reexamine the role of this enzyme in the brain, and the possible CNS disorders in which PDE10A inhibition could be therapeutic.  We will review the localization of PDE10A, both within the brain and the neuron and discuss how its role in regulating cAMP and cGMP accumulation modulates intracellular signaling pathways.  Since this cellular signaling has best been documented in the striatum, we will focus our discussion of PDE10A in the context of disorders that affect the basal ganglia, including psychiatric disorders such as bipolar disorder and autism spectrum disorders and the movement disorders, including Parkinson's disease and Huntington's disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3nn_WW9cz37Vg90H21EOLACvtfcHk0ljNHgeoSPkZfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFOntr%252FJ&md5=852dbd753943a0ce1043c52085698c2a</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.2174%2F1381612820666140826114744&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1381612820666140826114744%26sid%3Dliteratum%253Aachs%26aulast%3DWilson%26aufirst%3DL.%2BS.%26aulast%3DBrandon%26aufirst%3DN.%2BJ.%26atitle%3DEmerging%2520biology%2520of%2520PDE10A%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2015%26volume%3D21%26spage%3D378%26epage%3D388%26doi%3D10.2174%2F1381612820666140826114744" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Verhoest, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapin, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fonseca, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harms, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marr, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menniti, F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connor, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandit, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Proulx-LaFrance, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suiciak, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liras, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of a novel class of phosphodiesterase 10A inhibitors and identification of clinical candidate 2-[4-(1-Methyl-4-pyridin-4-yl-1H-pyrazol-3-yl)-phenoxymethyl]-quinoline (PF-2545920) for the treatment of schizophrenia</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">5188</span>– <span class="NLM_lpage">5196</span>, <span class="refDoi"> DOI: 10.1021/jm900521k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm900521k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BD1MXovFKmu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=5188-5196&author=P.+R.+Verhoestauthor=D.+S.+Chapinauthor=M.+Cormanauthor=K.+Fonsecaauthor=J.+F.+Harmsauthor=X.+Houauthor=E.+S.+Marrauthor=F.+S.+Mennitiauthor=F.+Nelsonauthor=R.+O%E2%80%99Connorauthor=J.+Panditauthor=C.+Proulx-LaFranceauthor=A.+W.+Schmidtauthor=C.+J.+Schmidtauthor=J.+A.+Suiciakauthor=S.+Liras&title=Discovery+of+a+novel+class+of+phosphodiesterase+10A+inhibitors+and+identification+of+clinical+candidate+2-%5B4-%281-Methyl-4-pyridin-4-yl-1H-pyrazol-3-yl%29-phenoxymethyl%5D-quinoline+%28PF-2545920%29+for+the+treatment+of+schizophrenia&doi=10.1021%2Fjm900521k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Novel Class of Phosphodiesterase 10A Inhibitors and Identification of Clinical Candidate 2-[4-(1-Methyl-4-pyridin-4-yl-1H-pyrazol-3-yl)-phenoxymethyl]-quinoline (PF-2545920)(I) for the Treatment of Schizophrenia</span></div><div class="casAuthors">Verhoest, Patrick R.; Chapin, Douglas S.; Corman, Michael; Fonseca, Kari; Harms, John F.; Hou, Xinjun; Marr, Eric S.; Menniti, Frank S.; Nelson, Frederick; O'Connor, Rebecca; Pandit, Jayvardhan; Proulx-LaFrance, Caroline; Schmidt, Anne W.; Schmidt, Christopher J.; Suiciak, Judith A.; Liras, Spiros</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">5188-5196</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">By utilizing structure-based drug design (SBDD) knowledge, a novel class of phosphodiesterase (PDE) 10A inhibitors was identified.  The structure-based drug design efforts identified a unique "selectivity pocket" for PDE10A inhibitors, and interactions within this pocket allowed the design of highly selective and potent PDE10A inhibitors.  Further optimization of brain penetration and drug-like properties led to the discovery of 2-[4-(1-methyl-4-pyridin-4-yl-1H-pyrazol-3-yl)-phenoxymethyl]-quinoline (PF-2545920) (I).  This PDE10A inhibitor is the first reported clin. entry for this mechanism in the treatment of schizophrenia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwg5FBdhz5NbVg90H21EOLACvtfcHk0ljNHgeoSPkZfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXovFKmu7Y%253D&md5=acc9e8687684c4cef1537a15800c8317</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fjm900521k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm900521k%26sid%3Dliteratum%253Aachs%26aulast%3DVerhoest%26aufirst%3DP.%2BR.%26aulast%3DChapin%26aufirst%3DD.%2BS.%26aulast%3DCorman%26aufirst%3DM.%26aulast%3DFonseca%26aufirst%3DK.%26aulast%3DHarms%26aufirst%3DJ.%2BF.%26aulast%3DHou%26aufirst%3DX.%26aulast%3DMarr%26aufirst%3DE.%2BS.%26aulast%3DMenniti%26aufirst%3DF.%2BS.%26aulast%3DNelson%26aufirst%3DF.%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DR.%26aulast%3DPandit%26aufirst%3DJ.%26aulast%3DProulx-LaFrance%26aufirst%3DC.%26aulast%3DSchmidt%26aufirst%3DA.%2BW.%26aulast%3DSchmidt%26aufirst%3DC.%2BJ.%26aulast%3DSuiciak%26aufirst%3DJ.%2BA.%26aulast%3DLiras%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520a%2520novel%2520class%2520of%2520phosphodiesterase%252010A%2520inhibitors%2520and%2520identification%2520of%2520clinical%2520candidate%25202-%255B4-%25281-Methyl-4-pyridin-4-yl-1H-pyrazol-3-yl%2529-phenoxymethyl%255D-quinoline%2520%2528PF-2545920%2529%2520for%2520the%2520treatment%2520of%2520schizophrenia%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D5188%26epage%3D5196%26doi%3D10.1021%2Fjm900521k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scapin, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varnerin, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edmondson, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mastracchio, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parmee, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van der
Ploeg, L. H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tota, M. R.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of human phosphodiesterase 3B: Atomic basis for substrate and inhibitor specificity</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">6091</span>– <span class="NLM_lpage">6100</span>, <span class="refDoi"> DOI: 10.1021/bi049868i</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi049868i" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjsVShs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2004&pages=6091-6100&author=G.+Scapinauthor=S.+B.+Patelauthor=C.+Chungauthor=J.+P.+Varnerinauthor=S.+D.+Edmondsonauthor=A.+Mastracchioauthor=E.+R.+Parmeeauthor=S.+B.+Singhauthor=J.+W.+Beckerauthor=L.+H.+T.+Van+der%0APloegauthor=M.+R.+Tota&title=Crystal+structure+of+human+phosphodiesterase+3B%3A+Atomic+basis+for+substrate+and+inhibitor+specificity&doi=10.1021%2Fbi049868i"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal Structure of Human Phosphodiesterase 3B: Atomic Basis for Substrate and Inhibitor Specificity</span></div><div class="casAuthors">Scapin, Giovanna; Patel, Sangita B.; Chung, Christine; Varnerin, Jeffrey P.; Edmondson, Scott D.; Mastracchio, Anthony; Parmee, Emma R.; Singh, Suresh B.; Becker, Joseph W.; Van der Ploeg, Lex H. T.; Tota, Michael R.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">6091-6100</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Phosphodiesterases (PDEs) are enzymes that modulate cyclic nucleotide signaling and as such are clin. targets for a range of disorders including congestive heart failure, erectile dysfunction, and inflammation.  The PDE3 family comprises two highly homologous subtypes expressed in different tissues, and inhibitors of this family have been shown to increase lipolysis in adipocytes.  A specific PDE3B (the lipocyte-localized subtype) inhibitor would be a very useful tool to evaluate the effects of PDE3 inhibition on lipolysis and metabolic rate and might become a novel tool for treatment of obesity.  We report here the three-dimensional structures of the catalytic domain of human PDE3B in complex with a generic PDE inhibitor and a novel PDE3 selective inhibitor.  These structures explain the dual cAMP/cGMP binding capabilities of PDE3, provide the mol. basis for inhibitor specificity, and can supply a valid platform for the design of improved compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6lJKq3qzQGrVg90H21EOLACvtfcHk0lizU7cFHMMF1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjsVShs7s%253D&md5=e39041b5c61c01b8b04b21e1d8515c22</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fbi049868i&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi049868i%26sid%3Dliteratum%253Aachs%26aulast%3DScapin%26aufirst%3DG.%26aulast%3DPatel%26aufirst%3DS.%2BB.%26aulast%3DChung%26aufirst%3DC.%26aulast%3DVarnerin%26aufirst%3DJ.%2BP.%26aulast%3DEdmondson%26aufirst%3DS.%2BD.%26aulast%3DMastracchio%26aufirst%3DA.%26aulast%3DParmee%26aufirst%3DE.%2BR.%26aulast%3DSingh%26aufirst%3DS.%2BB.%26aulast%3DBecker%26aufirst%3DJ.%2BW.%26aulast%3DVan%2Bder%2BPloeg%26aufirst%3DL.%2BH.%2BT.%26aulast%3DTota%26aufirst%3DM.%2BR.%26atitle%3DCrystal%2520structure%2520of%2520human%2520phosphodiesterase%25203B%253A%2520Atomic%2520basis%2520for%2520substrate%2520and%2520inhibitor%2520specificity%26jtitle%3DBiochemistry%26date%3D2004%26volume%3D43%26spage%3D6091%26epage%3D6100%26doi%3D10.1021%2Fbi049868i" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhan, C.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, H.-B.</span></span> <span> </span><span class="NLM_article-title">Absolute binding free energy calculation and design of a subnanomolar inhibitor of phosphodiesterase-10</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">2099</span>– <span class="NLM_lpage">2111</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01763</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01763" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvFKjsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=2099-2111&author=Z.+Liauthor=Y.+Huangauthor=Y.+Wuauthor=J.+Chenauthor=D.+Wuauthor=C.-G.+Zhanauthor=H.-B.+Luo&title=Absolute+binding+free+energy+calculation+and+design+of+a+subnanomolar+inhibitor+of+phosphodiesterase-10&doi=10.1021%2Facs.jmedchem.8b01763"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Absolute Binding Free Energy Calculation and Design of a Subnanomolar Inhibitor of Phosphodiesterase-10</span></div><div class="casAuthors">Li, Zhe; Huang, Yiyou; Wu, Yinuo; Chen, Jingyi; Wu, Deyan; Zhan, Chang-Guo; Luo, Hai-Bin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">2099-2111</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Accurate prediction of abs. protein-ligand binding free energy could considerably enhance the success rate of structure-based drug design but is extremely challenging and time-consuming.  Free energy perturbation (FEP) has been proven reliable but is limited to prediction of relative binding free energies of similar ligands (with only minor structural differences) in binding with a same drug target in practical drug design applications.  Herein, a Gaussian algorithm-enhanced FEP (GA-FEP) protocol has been developed to enhance the FEP simulation performance, enabling to efficiently carry out the FEP simulations on vanishing the whole ligand and, thus, predict the abs. binding free energies (ABFEs).  Using the GA-FEP protocol, the FEP simulations for the ABFE calcn. (denoted as GA-FEP/ABFE) can achieve a satisfactory accuracy for both structurally similar and diverse ligands in a dataset of more than 100 receptor-ligand systems.  Further, our GA-FEP/ABFE-guided lead optimization against phosphodiesterase-10 led to the discovery of a subnanomolar inhibitor (IC50 = 0.87 nM, ∼2000-fold improvement in potency) with cocrystal confirmation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKTyv17ShhKrVg90H21EOLACvtfcHk0lizU7cFHMMF1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvFKjsLc%253D&md5=feaf32e0006d6a3cf2f2b7d4d98c2a25</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01763&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01763%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DD.%26aulast%3DZhan%26aufirst%3DC.-G.%26aulast%3DLuo%26aufirst%3DH.-B.%26atitle%3DAbsolute%2520binding%2520free%2520energy%2520calculation%2520and%2520design%2520of%2520a%2520subnanomolar%2520inhibitor%2520of%2520phosphodiesterase-10%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D2099%26epage%3D2111%26doi%3D10.1021%2Facs.jmedchem.8b01763" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moraca, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Negri, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Oliveira, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abel, R.</span></span> <span> </span><span class="NLM_article-title">Application of free energy perturbation (FEP+) to understanding ligand selectivity: A case study to assess selectivity between pairs of phosphodiesterases (PDE’s)</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">2729</span>– <span class="NLM_lpage">2740</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.9b00106</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.9b00106" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtVKgtLjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2019&pages=2729-2740&author=F.+Moracaauthor=A.+Negriauthor=C.+de+Oliveiraauthor=R.+Abel&title=Application+of+free+energy+perturbation+%28FEP%2B%29+to+understanding+ligand+selectivity%3A+A+case+study+to+assess+selectivity+between+pairs+of+phosphodiesterases+%28PDE%E2%80%99s%29&doi=10.1021%2Facs.jcim.9b00106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Application of Free Energy Perturbation (FEP+) to Understanding Ligand Selectivity: A Case Study to Assess Selectivity Between Pairs of Phosphodiesterases (PDE's)</span></div><div class="casAuthors">Moraca, Francesca; Negri, Ana; de Oliveira, Cesar; Abel, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2729-2740</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cyclic nucleotide phosphodiesterases (PDE's) are metalloenzymes that play a key role in regulating the levels of the ubiquitous second messengers, cyclic adenosine monophosphate (cAMP) and cyclic guanosine monophosphate (cGMP).  In humans, 11 PDE protein families mediate numerous biochem. pathways throughout the body and are effective drug targets for the treatment of diseases ranging from central nervous system disorders to heart and pulmonary diseases.  PDE's also share a highly conserved catalytic site (about 50%), thus making the design of selective drug candidates very challenging with classical structure-based design approaches given also the lack of publicly available co-crystal structures of pairs of PDE's with consistent biol. data to be compared, as we show in our work.  In this retrospective study, we apply free energy perturbation (FEP+) to predict the selectivity of inhibitors that bind two pairs of closely related PDE families: PDE9/1 and PDE5/6 where only 1 co-crystal structure per pair is publicly available.  As another challenge, the pKa of the PDE5/6 inhibitor is close to the exptl. pH, making unclear the exact protonation state that should be used in the computational workflow.  We demonstrate that running FEP+ on homol. models constructed for these metalloenzymes accurately reproduces exptl. obsd. selectivity profiles also addressing the unclear protonation state to be used during computation with our recently developed pKa-correction method.  Based on these data, we conclude that FEP+ is a robust method for prediction of selectivity for this target class and may be helpful to address related lead optimization challenges in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_z0VwaEW9krVg90H21EOLACvtfcHk0lizU7cFHMMF1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtVKgtLjJ&md5=b46b0d3d7dd802f4852af5253d6c629a</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.9b00106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.9b00106%26sid%3Dliteratum%253Aachs%26aulast%3DMoraca%26aufirst%3DF.%26aulast%3DNegri%26aufirst%3DA.%26aulast%3Dde%2BOliveira%26aufirst%3DC.%26aulast%3DAbel%26aufirst%3DR.%26atitle%3DApplication%2520of%2520free%2520energy%2520perturbation%2520%2528FEP%252B%2529%2520to%2520understanding%2520ligand%2520selectivity%253A%2520A%2520case%2520study%2520to%2520assess%2520selectivity%2520between%2520pairs%2520of%2520phosphodiesterases%2520%2528PDE%25E2%2580%2599s%2529%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2019%26volume%3D59%26spage%3D2729%26epage%3D2740%26doi%3D10.1021%2Facs.jcim.9b00106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mobley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span> <span> </span><span class="NLM_article-title">A critical review of validation, blind testing, and real-world use of alchemical protein-ligand binding free energy calculations</span>. <i>Curr. Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">2577</span>– <span class="NLM_lpage">2585</span>, <span class="refDoi"> DOI: 10.2174/1568026617666170414142131</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.2174%2F1568026617666170414142131" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=28413950" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlelsLnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2017&pages=2577-2585&author=R.+Abelauthor=L.+Wangauthor=D.+L.+Mobleyauthor=R.+A.+Friesner&title=A+critical+review+of+validation%2C+blind+testing%2C+and+real-world+use+of+alchemical+protein-ligand+binding+free+energy+calculations&doi=10.2174%2F1568026617666170414142131"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">A Critical Review of Validation, Blind Testing, and Real- World Use of Alchemical Protein-Ligand Binding Free Energy Calculations</span></div><div class="casAuthors">Abel, Robert; Wang, Lingle; Mobley, David L.; Friesner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">2577-2585</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">Protein-ligand binding is among the most fundamental phenomena underlying all mol. biol., and a greater ability to more accurately and robustly predict the binding free energy of a small mol. ligand for its cognate protein is expected to have vast consequences for improving the efficiency of pharmaceutical drug discovery.  We briefly reviewed a no. of scientific and tech. advances that have enabled alchem. free energy calcns. to recently emerge as a preferred approach, and critically considered proper validation and effective use of these techniques.  In particular, we characterized a selection bias effect which may be important in prospective free energy calcns., and introduced a strategy to improve the accuracy of the free energy predictions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYWt7IyeVNI7Vg90H21EOLACvtfcHk0lgwADQIjweQ2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlelsLnM&md5=1b13f6922117268e88b450f1012d140d</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.2174%2F1568026617666170414142131&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568026617666170414142131%26sid%3Dliteratum%253Aachs%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DMobley%26aufirst%3DD.%2BL.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DA%2520critical%2520review%2520of%2520validation%252C%2520blind%2520testing%252C%2520and%2520real-world%2520use%2520of%2520alchemical%2520protein-ligand%2520binding%2520free%2520energy%2520calculations%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2017%26volume%3D17%26spage%3D2577%26epage%3D2585%26doi%3D10.2174%2F1568026617666170414142131" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mey, A. S.</span>; <span class="NLM_string-name">Allen, B.</span>; <span class="NLM_string-name">Macdonald, H. E. B.</span>; <span class="NLM_string-name">Chodera, J. D.</span>; <span class="NLM_string-name">Kuhn, M.</span>; <span class="NLM_string-name">Michel, J.</span>; <span class="NLM_string-name">Mobley, D. L.</span>; <span class="NLM_string-name">Naden, L. N.</span>; <span class="NLM_string-name">Prasad, S.</span>; <span class="NLM_string-name">Rizzi, A.</span>; <span class="NLM_string-name">Scheen, J.</span>; <span class="NLM_string-name">Shirts, M. R.</span>; <span class="NLM_string-name">Tresadern, G.</span>; <span class="NLM_string-name">Xu, H.</span></span>. <span> </span><span class="NLM_article-title">Best practice for free energy calculations</span>. <span class="NLM_year">2020</span>, arXiv:2008.03067. Published Online: August 21, 2020, <a href="https://arxiv.org/abs/2008.03067" class="extLink">https://arxiv.org/abs/2008.03067</a> (accessed Sept 22, 2020).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=A.+S.+Mey&author=B.+Allen&author=H.+E.+B.+Macdonald&author=J.+D.+Chodera&author=M.+Kuhn&author=J.+Michel&author=D.+L.+Mobley&author=L.+N.+Naden&author=S.+Prasad&author=A.+Rizzi&author=J.+Scheen&author=M.+R.+Shirts&author=G.+Tresadern&author=H.+Xu&title=Best+practice+for+free+energy+calculations"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3DMey%26aufirst%3DA.%2BS.%26atitle%3DBest%2520practice%2520for%2520free%2520energy%2520calculations%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shirts, M. R.</span>; <span class="NLM_string-name">Mobley, D. L.</span>; <span class="NLM_string-name">Brown, S. P.</span></span> <span> </span><span class="NLM_article-title">Free-Energy Calculations in Structure-Based Drug Design</span>. In  <i>Drug Design. Structure- and Ligand-Based Approaches</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merz, K. M.,  Jr.</span>, <span class="NLM_string-name">Ringe, D.</span>, <span class="NLM_string-name">Reynolds, C. H.</span></span>, Eds.; <span class="NLM_publisher-name">Cambridge University Press</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">2010</span>; pp  <span class="NLM_fpage">61</span>– <span class="NLM_lpage">86</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.1017%2FCBO9780511730412.007" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&pages=61-86&author=M.+R.+Shirts&author=D.+L.+Mobley&author=S.+P.+Brownauthor=K.+M.+Merz&author=D.+Ringe&author=C.+H.+Reynolds&title=Drug+Design.+Structure-+and+Ligand-Based+Approaches"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1017%2FCBO9780511730412.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1017%252FCBO9780511730412.007%26sid%3Dliteratum%253Aachs%26aulast%3DShirts%26aufirst%3DM.%2BR.%26atitle%3DFree-Energy%2520Calculations%2520in%2520Structure-Based%2520Drug%2520Design%26btitle%3DDrug%2520Design.%2520Structure-%2520and%2520Ligand-Based%2520Approaches%26aulast%3DMerz%26aufirst%3DK.%2BM.%26pub%3DCambridge%2520University%2520Press%26date%3D2010%26spage%3D61%26epage%3D86" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manas, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farid, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span> <span> </span><span class="NLM_article-title">Modeling the value of predictive affinity scoring in preclinical drug discovery</span>. <i>Curr. Opin. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">103</span>– <span class="NLM_lpage">110</span>, <span class="refDoi"> DOI: 10.1016/j.sbi.2018.09.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.1016%2Fj.sbi.2018.09.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=30321805" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslyru73I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2018&pages=103-110&author=R.+Abelauthor=E.+S.+Manasauthor=R.+A.+Friesnerauthor=R.+S.+Faridauthor=L.+Wang&title=Modeling+the+value+of+predictive+affinity+scoring+in+preclinical+drug+discovery&doi=10.1016%2Fj.sbi.2018.09.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Modeling the value of predictive affinity scoring in preclinical drug discovery</span></div><div class="casAuthors">Abel, Robert; Manas, Eric S.; Friesner, Richard A.; Farid, Ramy S.; Wang, Lingle</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">103-110</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Drug discovery is widely recognized to be a difficult and costly activity in large part due to the challenge of identifying chem. matter which simultaneously optimizes multiple properties, one of which is affinity for the primary biol. target.  Further, many of these properties are difficult to predict ahead of expensive and time-consuming compd. synthesis and exptl. testing.  Here we highlight recent work to develop compd. affinity prediction models, and extensively investigate the value such models may provide to preclin. drug discovery.  We demonstrate that the ability of these models to improve the overall probability of success is crucially dependent on the shape of the error distribution, not just the root-mean-square error.  In particular, while scoring more mol. ideas generally improves the probability of project success when the error distribution is Gaussian, fat-tail distributions such as a Cauchy distribution, can lead to a situation where scoring more ideas actually decreases the overall probability of success.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIs1gJSX0VLbVg90H21EOLACvtfcHk0lgwADQIjweQ2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslyru73I&md5=2e996488928b652dd299d3e1425d4dd3</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2018.09.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2018.09.002%26sid%3Dliteratum%253Aachs%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DManas%26aufirst%3DE.%2BS.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DFarid%26aufirst%3DR.%2BS.%26aulast%3DWang%26aufirst%3DL.%26atitle%3DModeling%2520the%2520value%2520of%2520predictive%2520affinity%2520scoring%2520in%2520preclinical%2520drug%2520discovery%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2018%26volume%3D52%26spage%3D103%26epage%3D110%26doi%3D10.1016%2Fj.sbi.2018.09.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group"><span class="NLM_string-name">Van Roosbroeck, Y. E. M.</span>; <span class="NLM_string-name">Van den Keybus, F. A. M.</span>; <span class="NLM_string-name">Tresadern, G. J.</span>; <span class="NLM_string-name">Buijnsters, P. J. J. A.</span>; <span class="NLM_string-name">Velter, A. I.</span>; <span class="NLM_string-name">Jacoby, E.</span>; <span class="NLM_string-name">Macdonald, G. J.</span>; <span class="NLM_string-name">Gijsen, H. J. M.</span>; <span class="NLM_string-name">Ahnaou, A.</span>; <span class="NLM_string-name">Drinkenburg, W. H. I. M.</span></span> <span> </span><span class="NLM_article-title">[1,2,4]Triazolo[1,5-a]pyrimidine Derivatives as PDE2 Inhibitors</span>. <span class="NLM_patent">WO 2018083098 A1</span>, May 11, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=Y.+E.+M.+Van+Roosbroeck&author=F.+A.+M.+Van+den+Keybus&author=G.+J.+Tresadern&author=P.+J.+J.+A.+Buijnsters&author=A.+I.+Velter&author=E.+Jacoby&author=G.+J.+Macdonald&author=H.+J.+M.+Gijsen&author=A.+Ahnaou&author=W.+H.+I.+M.+Drinkenburg&title=%5B1%2C2%2C4%5DTriazolo%5B1%2C5-a%5Dpyrimidine+Derivatives+as+PDE2+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50a&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2BRoosbroeck%26aufirst%3DY.%2BE.%2BM.%26atitle%3D%255B1%252C2%252C4%255DTriazolo%255B1%252C5-a%255Dpyrimidine%2520Derivatives%2520as%2520PDE2%2520Inhibitors%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit50b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group"><span class="NLM_string-name">Van Roosbroeck, Y. E. M.</span>; <span class="NLM_string-name">Buijnsters, P. J. J. A.</span>; <span class="NLM_string-name">Tresadern, G. J.</span>; <span class="NLM_string-name">Jacoby, E.</span>; <span class="NLM_string-name">Oehlrich, D.</span>; <span class="NLM_string-name">Gijsen, H. J. M.</span></span> <span> </span><span class="NLM_article-title">[1,2,4]Triazolo[1,5-a]pyrimidine Compounds as PDE2 Inhibitors</span>. <span class="NLM_patent">WO 2018083103 A1</span>, May 11, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=Y.+E.+M.+Van+Roosbroeck&author=P.+J.+J.+A.+Buijnsters&author=G.+J.+Tresadern&author=E.+Jacoby&author=D.+Oehlrich&author=H.+J.+M.+Gijsen&title=%5B1%2C2%2C4%5DTriazolo%5B1%2C5-a%5Dpyrimidine+Compounds+as+PDE2+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50b&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2BRoosbroeck%26aufirst%3DY.%2BE.%2BM.%26atitle%3D%255B1%252C2%252C4%255DTriazolo%255B1%252C5-a%255Dpyrimidine%2520Compounds%2520as%2520PDE2%2520Inhibitors%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="note"><p class="first last">The absolute configuration was determined using vibrational VCD. See <a href="/doi/suppl/10.1021/acs.jmedchem.0c01272/suppl_file/jm0c01272_si_001.pdf" class="ext-link">Supporting Information</a>.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghanakota, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bos, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konze, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staker, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marques, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marshall, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leswing, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhat, S.</span></span> <span> </span><span class="NLM_article-title">Combining cloud-based free energy calculations, synthetically aware enumerations and goal-directed generative machine learning for rapid large scale chemical exploration and optimization</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.0c00120</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.0c00120" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=P.+Ghanakotaauthor=P.+H.+Bosauthor=K.+Konzeauthor=J.+Stakerauthor=G.+Marquesauthor=K.+Marshallauthor=K.+Leswingauthor=R.+Abelauthor=S.+Bhat&title=Combining+cloud-based+free+energy+calculations%2C+synthetically+aware+enumerations+and+goal-directed+generative+machine+learning+for+rapid+large+scale+chemical+exploration+and+optimization&doi=10.1021%2Facs.jcim.0c00120"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.0c00120&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.0c00120%26sid%3Dliteratum%253Aachs%26aulast%3DGhanakota%26aufirst%3DP.%26aulast%3DBos%26aufirst%3DP.%2BH.%26aulast%3DKonze%26aufirst%3DK.%26aulast%3DStaker%26aufirst%3DJ.%26aulast%3DMarques%26aufirst%3DG.%26aulast%3DMarshall%26aufirst%3DK.%26aulast%3DLeswing%26aufirst%3DK.%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DBhat%26aufirst%3DS.%26atitle%3DCombining%2520cloud-based%2520free%2520energy%2520calculations%252C%2520synthetically%2520aware%2520enumerations%2520and%2520goal-directed%2520generative%2520machine%2520learning%2520for%2520rapid%2520large%2520scale%2520chemical%2520exploration%2520and%2520optimization%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2020%26doi%3D10.1021%2Facs.jcim.0c00120" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schindler, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baumann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blum, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Böse, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchstaller, H.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgdorf, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cappel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chekler, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czodrowski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dorsch, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eguida, M. K. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Follows, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuchß, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grädler, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunera, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jorand Lebrun, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karra, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knehans, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kötzner, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krier, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leiendecker, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leuthner, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mochalkin, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Musil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neagu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rippmann, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiemann, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulz, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinbrecher, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanzer, E.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unzue
Lopez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viacava Follis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wegener, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, D.</span></span> <span> </span><span class="NLM_article-title">Large-Scale assessment of binding free energy calculations in active drug discovery projects</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.0c00900</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.0c00900" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=C.+Schindlerauthor=H.+Baumannauthor=A.+Blumauthor=D.+B%C3%B6seauthor=H.-P.+Buchstallerauthor=L.+Burgdorfauthor=D.+Cappelauthor=E.+Cheklerauthor=P.+Czodrowskiauthor=D.+Dorschauthor=M.+K.+I.+Eguidaauthor=B.+Followsauthor=T.+Fuch%C3%9Fauthor=U.+Gr%C3%A4dlerauthor=J.+Guneraauthor=T.+Johnsonauthor=C.+Jorand+Lebrunauthor=S.+Karraauthor=M.+Kleinauthor=T.+Knehansauthor=L.+K%C3%B6tznerauthor=M.+Krierauthor=M.+Leiendeckerauthor=B.+Leuthnerauthor=L.+Liauthor=I.+Mochalkinauthor=D.+Musilauthor=C.+Neaguauthor=F.+Rippmannauthor=K.+Schiemannauthor=R.+Schulzauthor=T.+B.+Steinbrecherauthor=E.-M.+Tanzerauthor=A.+Unzue%0ALopezauthor=A.+Viacava+Follisauthor=A.+Wegenerauthor=D.+Kuhn&title=Large-Scale+assessment+of+binding+free+energy+calculations+in+active+drug+discovery+projects&doi=10.1021%2Facs.jcim.0c00900"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.0c00900&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.0c00900%26sid%3Dliteratum%253Aachs%26aulast%3DSchindler%26aufirst%3DC.%26aulast%3DBaumann%26aufirst%3DH.%26aulast%3DBlum%26aufirst%3DA.%26aulast%3DB%25C3%25B6se%26aufirst%3DD.%26aulast%3DBuchstaller%26aufirst%3DH.-P.%26aulast%3DBurgdorf%26aufirst%3DL.%26aulast%3DCappel%26aufirst%3DD.%26aulast%3DChekler%26aufirst%3DE.%26aulast%3DCzodrowski%26aufirst%3DP.%26aulast%3DDorsch%26aufirst%3DD.%26aulast%3DEguida%26aufirst%3DM.%2BK.%2BI.%26aulast%3DFollows%26aufirst%3DB.%26aulast%3DFuch%25C3%259F%26aufirst%3DT.%26aulast%3DGr%25C3%25A4dler%26aufirst%3DU.%26aulast%3DGunera%26aufirst%3DJ.%26aulast%3DJohnson%26aufirst%3DT.%26aulast%3DJorand%2BLebrun%26aufirst%3DC.%26aulast%3DKarra%26aufirst%3DS.%26aulast%3DKlein%26aufirst%3DM.%26aulast%3DKnehans%26aufirst%3DT.%26aulast%3DK%25C3%25B6tzner%26aufirst%3DL.%26aulast%3DKrier%26aufirst%3DM.%26aulast%3DLeiendecker%26aufirst%3DM.%26aulast%3DLeuthner%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DMochalkin%26aufirst%3DI.%26aulast%3DMusil%26aufirst%3DD.%26aulast%3DNeagu%26aufirst%3DC.%26aulast%3DRippmann%26aufirst%3DF.%26aulast%3DSchiemann%26aufirst%3DK.%26aulast%3DSchulz%26aufirst%3DR.%26aulast%3DSteinbrecher%26aufirst%3DT.%2BB.%26aulast%3DTanzer%26aufirst%3DE.-M.%26aulast%3DUnzue%2BLopez%26aufirst%3DA.%26aulast%3DViacava%2BFollis%26aufirst%3DA.%26aulast%3DWegener%26aufirst%3DA.%26aulast%3DKuhn%26aufirst%3DD.%26atitle%3DLarge-Scale%2520assessment%2520of%2520binding%2520free%2520energy%2520calculations%2520in%2520active%2520drug%2520discovery%2520projects%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2020%26doi%3D10.1021%2Facs.jcim.0c00900" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Madhavi
Sastry, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adzhigirey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Annabhimoju, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, W.</span></span> <span> </span><span class="NLM_article-title">Protein and ligand preparation: Parameters, protocols, and influence on virtual screening enrichments</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">221</span>– <span class="NLM_lpage">234</span>, <span class="refDoi"> DOI: 10.1007/s10822-013-9644-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=10.1007%2Fs10822-013-9644-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=23579614" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmslalu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2013&pages=221-234&author=G.+Madhavi%0ASastryauthor=M.+Adzhigireyauthor=T.+Dayauthor=R.+Annabhimojuauthor=W.+Sherman&title=Protein+and+ligand+preparation%3A+Parameters%2C+protocols%2C+and+influence+on+virtual+screening+enrichments&doi=10.1007%2Fs10822-013-9644-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Protein and ligand preparation: parameters, protocols, and influence on virtual screening enrichments</span></div><div class="casAuthors">Madhavi Sastry, G.; Adzhigirey, Matvey; Day, Tyler; Annabhimoju, Ramakrishna; Sherman, Woody</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">221-234</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Structure-based virtual screening plays an important role in drug discovery and complements other screening approaches.  In general, protein crystal structures are prepd. prior to docking in order to add hydrogen atoms, optimize hydrogen bonds, remove at. clashes, and perform other operations that are not part of the x-ray crystal structure refinement process.  In addn., ligands must be prepd. to create 3-dimensional geometries, assign proper bond orders, and generate accessible tautomer and ionization states prior to virtual screening.  While the prerequisite for proper system prepn. is generally accepted in the field, an extensive study of the prepn. steps and their effect on virtual screening enrichments has not been performed.  In this work, we systematically explore each of the steps involved in prepg. a system for virtual screening.  We first explore a large no. of parameters using the Glide validation set of 36 crystal structures and 1,000 decoys.  We then apply a subset of protocols to the DUD database.  We show that database enrichment is improved with proper prepn. and that neglecting certain steps of the prepn. process produces a systematic degrdn. in enrichments, which can be large for some targets.  We provide examples illustrating the structural changes introduced by the prepn. that impact database enrichment.  While the work presented here was performed with the Protein Prepn. Wizard and Glide, the insights and guidance are expected to be generalizable to structure-based virtual screening with other docking methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruqN5nIOJtR7Vg90H21EOLACvtfcHk0ljZAj9GdFZOdA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmslalu7c%253D&md5=259a6d547ef3e1310e091fb50fe8de16</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1007%2Fs10822-013-9644-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-013-9644-8%26sid%3Dliteratum%253Aachs%26aulast%3DMadhavi%2BSastry%26aufirst%3DG.%26aulast%3DAdzhigirey%26aufirst%3DM.%26aulast%3DDay%26aufirst%3DT.%26aulast%3DAnnabhimoju%26aufirst%3DR.%26aulast%3DSherman%26aufirst%3DW.%26atitle%3DProtein%2520and%2520ligand%2520preparation%253A%2520Parameters%252C%2520protocols%252C%2520and%2520influence%2520on%2520virtual%2520screening%2520enrichments%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2013%26volume%3D27%26spage%3D221%26epage%3D234%26doi%3D10.1007%2Fs10822-013-9644-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group">Schrodinger
LLC</span>. 120 West 45th Street 17th Floor, Tower
45 New York, NY 10036-4041, USA at <a href="http://www.schrodinger.com/Maestro/" class="extLink">http://www.schrodinger.com/Maestro/</a>, <span class="NLM_year">2014</span>. (accessed Sept 18, 2020).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Schrodinger%0ALLC.+120+West+45th+Street+17th+Floor%2C+Tower%0A45+New+York%2C+NY+10036-4041%2C+USA+at+http%3A%2F%2Fwww.schrodinger.com%2FMaestro%2F%2C+2014.+%28accessed+Sept+18%2C+2020%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3D%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZND" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZND','PDB','6ZND'); return false;">PDB: 6ZND</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZQZ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZQZ','PDB','6ZQZ'); return false;">PDB: 6ZQZ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1Z1L" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1Z1L','PDB','1Z1L'); return false;">PDB: 1Z1L</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i126"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01272">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_84256"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c01272?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01272</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Selectivity data for all compounds; data for retrospective validation of FEP approach; comparison of binding affinity predictions from docking with experiment; and additional chemistry details (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01272/suppl_file/jm0c01272_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings for <b>3</b>–<b>46</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01272/suppl_file/jm0c01272_si_002.csv">CSV</a>)</p></li><li><p class="inline">Input structures for FEP calculations (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01272/suppl_file/jm0c01272_si_003.zip">ZIP</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01272/suppl_file/jm0c01272_si_001.pdf">jm0c01272_si_001.pdf (527.64 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01272/suppl_file/jm0c01272_si_002.csv">jm0c01272_si_002.csv (3.31 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01272/suppl_file/jm0c01272_si_003.zip">jm0c01272_si_003.zip (796.92 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Atomic coordinates and structure factors for compounds <b>3</b> and <b>39</b> have been deposited in the RCSB Protein Data Bank under the accession code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZND">6ZND</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZQZ">6ZQZ</a> respectively. Authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c01272&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c01272%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-21" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c01272" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679917c398f123f8","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
