
<html lang="en"     class="pb-page"  data-request-id="b7b678ae-2c74-4cae-af5c-418227a459f0"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;issue:issue:10.1021/jmcmar.2017.60.issue-21;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;article:article:10.1021/acs.jmedchem.7b00845;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Direct NMR Probing of Hydration Shells of Protein Ligand Interfaces and Its Application to Drug Design" /></meta><meta name="dc.Creator" content="Leonhard  Geist" /></meta><meta name="dc.Creator" content="Moriz  Mayer" /></meta><meta name="dc.Creator" content="Xiao-Ling  Cockcroft" /></meta><meta name="dc.Creator" content="Bernhard  Wolkerstorfer" /></meta><meta name="dc.Creator" content="Dirk  Kessler" /></meta><meta name="dc.Creator" content="Harald  Engelhardt" /></meta><meta name="dc.Creator" content="Darryl B.  McConnell" /></meta><meta name="dc.Creator" content="Robert  Konrat" /></meta><meta name="dc.Description" content="Fragment-based drug design exploits initial screening of low molecular weight compounds and their concomitant affinity improvement. The multitude of possible chemical modifications highlights the n..." /></meta><meta name="Description" content="Fragment-based drug design exploits initial screening of low molecular weight compounds and their concomitant affinity improvement. The multitude of possible chemical modifications highlights the n..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="October 1, 2017" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.7b00845" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2017 American Chemical Society" /></meta><meta name="citation_fulltext_world_readable" content=""/></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b00845" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.7b00845" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b00845" /></link>
        
    
    

<title>Direct NMR Probing of Hydration Shells of Protein Ligand Interfaces and Its Application to Drug Design | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.7b00845" /></meta><meta property="og:title" content="Direct NMR Probing of Hydration Shells of Protein Ligand Interfaces and Its Application to Drug Design" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-21/acs.jmedchem.7b00845/20171103/images/large/jm-2017-00845s_0006.jpeg" /></meta><meta property="og:description" content="Fragment-based drug design exploits initial screening of low molecular weight compounds and their concomitant affinity improvement. The multitude of possible chemical modifications highlights the necessity to obtain structural information about the binding mode of a fragment. Herein we describe a novel NMR methodology (LOGSY titration) that allows the determination of binding modes of low affinity binders in the protein–ligand interface and reveals suitable ligand positions for the addition of functional groups that either address or substitute protein-bound water, information of utmost importance for drug design. The particular benefit of the methodology and in contrast to conventional ligand-based methods is the independence of the molecular weight of the protein under study. The validity of the novel approach is demonstrated on two ligands interacting with bromodomain 1 of bromodomain containing protein 4, a prominent cancer target in pharmaceutical industry." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.7b00845"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.7b00845">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.7b00845&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.7b00845&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.7b00845&amp;href=/doi/10.1021/acs.jmedchem.7b00845" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2017</span><span class="cit-fg-volume">, 60</span><span class="cit-fg-issue">, 21</span><span class="cit-fg-pageRange">, 8708-8715</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/60/21" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.7b01497" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Featured Article</span><a href="/doi/10.1021/acs.jmedchem.7b00353" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Direct NMR Probing of Hydration Shells of Protein Ligand Interfaces and Its Application to Drug Design</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Leonhard++Geist">Leonhard Geist</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0002-3241-676X" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Moriz++Mayer">Moriz Mayer</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Xiao-Ling++Cockcroft">Xiao-Ling Cockcroft</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Bernhard++Wolkerstorfer">Bernhard Wolkerstorfer</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Dirk++Kessler">Dirk Kessler</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Harald++Engelhardt">Harald Engelhardt</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Darryl+B.++McConnell">Darryl B. McConnell</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Robert++Konrat">Robert Konrat</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Christian Doppler Laboratory for High-Content Structural Biology and Biotechnology, Department of Structural and Computational Biology, University of Vienna, Campus Vienna Biocenter 5, A-1030 Vienna, Austria</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Boehringer Ingelheim RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, A-1121 Vienna, Austria</span></div><div class="corresp-info"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#a6d4c9c4c3d4d288cdc9c8d4c7d2e6d3c8cfd0cfc388c7c588c7d2"><span class="__cf_email__" data-cfemail="87f5e8e5e2f5f3a9ece8e9f5e6f3c7f2e9eef1eee2a9e6e4a9e6f3">[email protected]</span></a>. Phone: +43-(0)1-4277-52202.</div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b00845&amp;href=/doi/10.1021%2Facs.jmedchem.7b00845" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2017</span></span><span class="cit-volume">, 60</span><span class="cit-issue">, 21</span><span class="cit-pageRange">, 8708–8715</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">September 14, 2017</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>21 June 2017</li><li><span class="item_label"><b>Published</b> online</span>1 October 2017</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 9 November 2017</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.7b00845" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b00845</a></div><div class="article_header-article-copyright"><strong>Copyright © 2017 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="/page/policy/authorchoice_termsofuse.html"><img src="/pb-assets/images/programsLogos/OA_Icons_Ec-1612971054143.svg" alt="ACS Editors’ Choice" /></img><span>ACS Editors’ Choice</span></a></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.7b00845"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">4846</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">20</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.7b00845" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Direct NMR Probing of Hydration Shells of Protein Ligand Interfaces and Its Application to Drug Design&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Leonhard&quot;,&quot;last_name&quot;:&quot;Geist&quot;},{&quot;first_name&quot;:&quot;Moriz&quot;,&quot;last_name&quot;:&quot;Mayer&quot;},{&quot;first_name&quot;:&quot;Xiao-Ling&quot;,&quot;last_name&quot;:&quot;Cockcroft&quot;},{&quot;first_name&quot;:&quot;Bernhard&quot;,&quot;last_name&quot;:&quot;Wolkerstorfer&quot;},{&quot;first_name&quot;:&quot;Dirk&quot;,&quot;last_name&quot;:&quot;Kessler&quot;},{&quot;first_name&quot;:&quot;Harald&quot;,&quot;last_name&quot;:&quot;Engelhardt&quot;},{&quot;first_name&quot;:&quot;Darryl&quot;,&quot;last_name&quot;:&quot;B. McConnell&quot;},{&quot;first_name&quot;:&quot;Robert&quot;,&quot;last_name&quot;:&quot;Konrat&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2017&quot;,&quot;month&quot;:&quot;10&quot;,&quot;day&quot;:&quot;01&quot;,&quot;issue&quot;:&quot;21&quot;,&quot;volume&quot;:&quot;60&quot;,&quot;pages&quot;:&quot;8708-8715&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.7b00845&quot;},&quot;abstract&quot;:&quot;Fragment-based drug design exploits initial screening of low molecular weight compounds and their concomitant affinity improvement. The multitude of possible chemical modifications highlights the necessity to obtain structural information about the binding mode of a fragment. Herein we describe a novel NMR methodology (LOGSY titration) that allows the determination of binding modes of low affinity binders in the protein–ligand interface and reveals suitable ligand positions for the addition of functional groups that either address or substitute protein-bound water, information of utmost importance for drug design. The particular benefit of the methodology and in contrast to conventional ligand-based methods is the independence of the molecular weight of the protein under study. The validity of the novel approach is demonstrated on two ligands interacting with bromodomain 1 of bromodomain containing protein 4, a prominent cancer target in pharmaceutical industry.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b00845&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00845" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b00845&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00845" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b00845&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00845" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b00845&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00845&amp;href=/doi/10.1021/acs.jmedchem.7b00845" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.7b00845" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.7b00845" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (2 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b00845&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.7b00845%26sid%3Dliteratum%253Aachs%26pmid%3D28910100%26genre%3Darticle%26aulast%3DGeist%26date%3D2017%26atitle%3DDirect%2BNMR%2BProbing%2Bof%2BHydration%2BShells%2Bof%2BProtein%2BLigand%2BInterfaces%2Band%2BIts%2BApplication%2Bto%2BDrug%2BDesign%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D60%26issue%3D21%26spage%3D8708%26epage%3D8715%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=291754" title="Molecules">Molecules</a>,</li><li><a href="/action/doSearch?ConceptID=291316" title="Mixtures">Mixtures</a>,</li><li><a href="/action/doSearch?ConceptID=292017" title="Solvents">Solvents</a>,</li><li><a href="/action/doSearch?ConceptID=291774" title="Nuclear magnetic resonance spectroscopy">Nuclear magnetic resonance spectroscopy</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/60/21" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-21/jmcmar.2017.60.issue-21/20171109/jmcmar.2017.60.issue-21.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-21/acs.jmedchem.7b00845/20171103/images/medium/jm-2017-00845s_0006.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-21/acs.jmedchem.7b00845/20171103/images/large/jm-2017-00845s_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00845&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Fragment-based drug design exploits initial screening of low molecular weight compounds and their concomitant affinity improvement. The multitude of possible chemical modifications highlights the necessity to obtain structural information about the binding mode of a fragment. Herein we describe a novel NMR methodology (LOGSY titration) that allows the determination of binding modes of low affinity binders in the protein–ligand interface and reveals suitable ligand positions for the addition of functional groups that either address or substitute protein-bound water, information of utmost importance for drug design. The particular benefit of the methodology and in contrast to conventional ligand-based methods is the independence of the molecular weight of the protein under study. The validity of the novel approach is demonstrated on two ligands interacting with bromodomain 1 of bromodomain containing protein 4, a prominent cancer target in pharmaceutical industry.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_35445" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_35445" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Rational drug design is an iterative process, in which 3D structural information about a specific protein–ligand interaction is used to generate an inhibitor that specifically blocks a targeted interaction. NMR spectroscopy provides well-established methods for initial library screening, as saturation transfer difference (STD),<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1, 2)</a> water ligand observed via gradient spectroscopy (WaterLOGSY),<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3, 4)</a> fluorine NMR (<sup>19</sup>F NMR),<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> structure–activity relationship by NMR (SAR by NMR).<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> Additionally, NMR methods were developed that allow the characterization of the ligand orientation in the protein-bound state, e.g., adiabatic fast passage nuclear Overhauser effect spectroscopy (AFP-NOESY),<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> interligand NOE (ILOE) for pharmacophore mapping (INPHARMA),<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> solvent accessibility, ligand binding, and mapping of ligand orientation by NMR (SALMON),<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> group epitope mapping (GEM) by STD,<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> difference of inversion recovery rate with and without target irradiation (DIRECTION),<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> transferred nuclear Overhauser effect (NOE) spectroscopy.<a onclick="showRef(event, 'ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref11 ref12">(11, 12)</a> Since X-ray cocrystallographic structure determination of protein–ligand complexes of initial hits is sometimes not feasible, independent techniques are required that provide specific information about ligand epitopes, as well as the protein surroundings of the bound ligand, to aid the systematic optimization of compounds. Especially, knowledge of ligand positions that are close to protein-bound water molecules will be very valuable for the choice of chemical modification to be introduced at certain ligand positions.</div><div class="NLM_p">In our study we employ GEM by STD<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> and a novel application of the WaterLOGSY<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> method called LOGSY titration to distinguish between buried and solvent-accessible parts of ligands, as well as to map protein-bound water molecules that are in proximity to the ligand. As published, WaterLOGSY<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> experiments also allow the extraction of information about solvent-accessible epitopes of a ligand. The SALMON approach uses the sign of a WaterLOGSY cross peak in the spectrum of the ligand in the presence of the protein as a measure of solvent accessibility.<a onclick="showRef(event, 'ref2 ref9'); return false;" href="javascript:void(0);" class="ref ref2 ref9">(2, 9)</a> Negative cross peaks are observed only for highly solvent exposed ligand protons, whereas positive cross peaks stem from buried ligand protons. We noticed that for weakly binding ligands, especially in combination with small proteins, positive WaterLOGSY cross peaks are hardly observed, which hampers the extraction of differential solvent accessibility scores with the SALMON methodology. Therefore, we propose performing WaterLOGSY experiments at increasing protein concentrations and using the titration slopes as a measure of solvent accessibility. A major advantage of the LOGSY-titration method is its applicability to small proteins, as it probes the influence of the protein on the solvation of the ligand. The potential of this new methodology is illustrated with an application to the bromodomain 1 of bromodomain containing protein 4 (Brd4-BD1).<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Brd4 is a member of the bromodomain and extra-terminal (BET) family that contains two tandem bromodomains (BD1 and BD2) and binds to acetylated lysines in histones.<a onclick="showRef(event, 'ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref13 ref14">(13, 14)</a> Inhibition of Brd4-chromatin interaction is relevant for the treatment of various forms of cancer and has been shown to have potent antiproliferative effects in a variety of hematological cancers through the suppression of c-MYC and FOSL1 transcription.<a onclick="showRef(event, 'ref8 ref15 ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref8 ref15 ref16 ref17">(8, 15-17)</a></div><div class="NLM_p last">We successfully could map buried and solvent-accessible epitopes in two ligands interacting with Brd4-BD1, revealing an unexpected flip of a core aromatic scaffold shared by the two ligands. Furthermore, we predicted ligand positions, which are in proximity to protein-bound water. Concomitant introduction of a secondary amine at a suggested position led to significant improvement of affinity of ligands by replacing weakly interacting H<sub>2</sub>O and forming a hydrogen bond to a nearby asparagine residue.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_24440" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_24440" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Determination of Ligand Binding Modes with LOGSY Titrations</h3><div class="NLM_p"><a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a> illustrates the LOGSY-titration experiment. For a reversibly binding ligand the magnetization transferred by cross-relaxation from irradiated or selectively inverted water to ligand protons is a population weighted average of free (positive) and bound (negative) cross-relaxation contributions (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>a). Increasing the amount of protein will shift the equilibrium toward the protein-bound state and eventually lead to a sign change of the observed cross peaks. The amount of bound cross-relaxation received via the protein depends on the peculiarities of the binding pocket (nearby bound water molecules, proton density at the interaction site) as well as the kinetics of the protein–ligand interaction. As individual ligand protons will experience a different surrounding in the protein-bound state, cross peaks of different ligand protons will behave differently during the LOGSY-titration (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>b).</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-21/acs.jmedchem.7b00845/20171103/images/medium/jm-2017-00845s_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-21/acs.jmedchem.7b00845/20171103/images/large/jm-2017-00845s_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Concept of the LOGSY-titration experiment. (a) For reversibly binding ligands (mM to μM) magnetization received from water during the WaterLOGSY experiment is a combination of cross-relaxation obtained from bulk water and cross-relaxation transferred either directly via protein-bound water molecules or indirectly via spin-diffusion mediated by protein protons. (b) LOGSY-titration: <i>I</i><sub>logsy</sub>/<i>I</i><sub>ref</sub> is plotted as a function of the protein concentration. Intensities in the WaterLOGSY are normalized to the <sup>1</sup>H 1D reference spectrum measured at the same protein concentration. The slopes are fitted to a linear function and normalized to the graph with the strongest slope (LOGSY factor of 100%).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-21/acs.jmedchem.7b00845/20171103/images/large/jm-2017-00845s_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00845&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We used mixing times of τ<sub>mix</sub> = 1s for the WaterLOGSY experiments, where the cross peak intensities for our ligands have their maximum. Increased amounts of protein will lead to an increase in macromolecular cross-relaxation from water to the ligand, either directly via protein-bound water molecules or indirectly via spin-diffusion mediated by protein protons. Ligand protons that are close to the protein in the bound state will show stronger LOGSY-titration slopes than ligand protons that are solvent-accessible in the bound state and not in close contact with the protein. The fitted slopes are normalized to the signal with the strongest slope (100% LOGSY factor) (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>b). <a href="/doi/suppl/10.1021/acs.jmedchem.7b00845/suppl_file/jm7b00845_si_001.pdf" class="ext-link">Figure S2 in the Supporting Information</a> provides simulations of expected LOGSY-titration cross peak intensities, demonstrating that at low protein concentrations the intensity changes approximately linearly with protein concentration.</div><div class="NLM_p">Typically, values around 100% correspond to ligand protons that receive a maximum of magnetization via the protein or protein-bound water molecules, whereas values smaller than 20% delineate protons that are facing the solvent in the bound state. Intermediate values correspond to protons positioned between protein and solvent. Importantly, since only the slope of the titration curve is analyzed, relevant information can be obtained also in cases where no zero crossing occurs (for example, in the case of weak binders in combination with low molecular weight proteins, as well as ligand signals for highly solvent-accessible protons). This feature is particularly attractive given that in early stages of the drug design process ligands tend to have moderate affinities.</div><div class="NLM_p"><a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a> shows LOGSY-titration factors and STD amplification factors (STD-AF<sub>0</sub>) of two ligands <b>1</b> and <b>2</b>, which reversibly bind to Brd4-BD1 with IC<sub>50</sub>(<b>1</b>) = 24 μM, and IC<sub>50</sub>(<b>2</b>) = 2 μM, respectively. X-ray cocrystal structures of both ligands were available and could be compared to the NMR derived LOGSY and STD enhancements. In both ligands, protons of the triazolopyridazine moiety have the largest LOGSY and STD-AF<sub>0</sub> values, showing that this part of the ligand is deeply embedded in the protein-binding site. The different substituents, for example, the methoxy groups 2 and 3 and aromatic protons of the phenyl moiety in ligand <b>1</b>, as well as aromatic protons of the phenyl group in ligand <b>2</b>, display smaller LOGSY and STD-AF<sub>0</sub> values, respectively.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-21/acs.jmedchem.7b00845/20171103/images/medium/jm-2017-00845s_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-21/acs.jmedchem.7b00845/20171103/images/large/jm-2017-00845s_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. LOGSY factors calculated from the slopes of the LOGSY titrations and normalized STD-AF<sub>0</sub> values for (a, b) ligand <b>1</b> and (d, e) ligand <b>2</b>. (c, f) X-ray cocrystal structures of the complexes Brd4-BD1 with ligand <b>1</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5M39">5M39</a>) and ligand <b>2</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5M3A">5M3A</a>), respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-21/acs.jmedchem.7b00845/20171103/images/large/jm-2017-00845s_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00845&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Still, while the distribution of STD values is nearly the same for both ligands, LOGSY factors reveal significant differences for both ligands and thereby indicate different ligand binding modes. Most important, while the STD-AF<sub>0</sub> values of the methyl group 1 of the two ligands are similar (99% and 100%), they have a significantly different LOGSY factor. In ligand <b>2</b> it is clearly reduced (58% compared to 100%) and is indicative of a different hydration shell located around the position of the methyl group (compare <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>a and <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>d). The binding mode of the triazolopyridazine moiety is indeed flipped, as confirmed in the X-ray structure, leading to a higher solvent accessibility for this methyl group in ligand <b>2</b> as shown in <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>f. This flip would not have been predicted from the STD-NMR spectra, most likely due to the fact that the protein environment with regard to the nonexchangeable protons is similar in the two binding modes leading to similar STD enhancements. The NMR results are therefore in excellent general agreement with the position of the ligand in the binding pocket obtained from the X-ray crystal structure.</div><div class="NLM_p last">In sum, both STD-NMR and LOGSY titrations provide information about ligand binding epitopes; however only the LOGSY factors revealed the flipped binding mode of the triazolopyridazine ring system of ligand <b>2</b>. It can thus be concluded that in medicinal chemistry applications STD data are valuable for the identification of ligand positions in close proximity to the protein-binding interface, whereas additional information about their hydration shells can be obtained using the LOGSY-titration technique.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Determination of Protein-Bound Water in Proximity to Ligand Protons</h3><div class="NLM_p">It has to be noted that in the LOGSY experiment the NOESY transfer between ligand and protein also involves spin-diffusion via protein protons in the binding cleft. We therefore performed LOGSY titrations with perdeuterated protein to suppress spin-diffusion via the protein in order to detect direct NOEs from protein-bound water molecules to the ligand (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>a and <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>c). Accordingly, large LOGSY factors are found for protons exhibiting NOEs to interfacial water molecules (embedded in the interaction interface), while ligand protons accessible to solvent water molecules display smaller values. Maximal LOGSY factors were found for protons experiencing cross-relaxation to ordered water molecules located at the bottom of the binding cleft (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>b and <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>d). Specifically, strong (direct) water NOEs were observed for methyl-group 1 of ligand <b>1</b> and (due to the swapped orientation of the triazolopyridazine ring) to the aromatic proton 8 of ligand <b>2</b>. Most interestingly, however, a significant water NOE was found for aromatic proton 8 of ligand <b>1</b> (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>b).</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-21/acs.jmedchem.7b00845/20171103/images/medium/jm-2017-00845s_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-21/acs.jmedchem.7b00845/20171103/images/large/jm-2017-00845s_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. LOGSY factors calculated from the slopes of the LOGSY titrations with perdeuterated Brd4-BD1 for (a) ligand <b>1</b> and (c) ligand <b>2</b>. 100% corresponds to the signal with the strongest slope. (b, d) X-ray cocrystal structures of the complexes Brd4-BD1 with ligand <b>1</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5M39">5M39</a>) and ligand <b>2</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5M3A">5M3A</a>), respectively. H<sub>2</sub>O molecules resolved in the electron density map and within a distance of 6 Å from the ligand  are shown as blue spheres.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-21/acs.jmedchem.7b00845/20171103/images/large/jm-2017-00845s_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00845&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">It is instructive to compare our findings with available X-ray data from other ligand–Brd4 complexes available in-house. In total, 11 complex structures were examined in which the ligands do not contain functional groups corresponding to the position 8, which would interfere with potential protein-bound water at that site. In all 11 crystal structures highly ordered water molecules were found in conserved positions of the binding pocket of Brd4-BD1 and close to methyl group 1 of ligand <b>1</b> (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>, blue spheres). The interaction interface contains a second water cluster although with higher positional variability (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>, yellow spheres) as well as water molecules in an exposed cluster (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>, red spheres), which were detectable in only 6 out of 11 complex structures, suggesting that water molecules in this part of the hydration shell are less ordered and presumably part of a dynamic water cluster. Water molecules located in this cluster are presumably responsible for the observed water NOE to the aromatic proton at position 8 of ligand <b>1</b>. Here it should be noted that there is ample evidence that X-ray protein crystallography does not reliably detect loosely defined hydration water molecules.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> In contrast, it is well established that NMR also identifies transiently bound or “disordered” water molecules that are not always resolved in the electron density map.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> Our findings nicely demonstrate that the LOGSY-titration experiment reported here is a powerful and efficient technique to probe the existence and location of transiently bound water molecules in protein ligand complexes. These water molecules could be valid starting points for chemical replacement strategies to displace or substitute bound water molecules by suitable functional groups.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-21/acs.jmedchem.7b00845/20171103/images/medium/jm-2017-00845s_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-21/acs.jmedchem.7b00845/20171103/images/large/jm-2017-00845s_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Superposition of H<sub>2</sub>O molecules in Brd4-BD1 complex structures of six different ligand cocrystal structures. Ligand <b>1</b> is shown as representative ligand structure. Spheres represent H<sub>2</sub>O molecules found in the X-ray structures of the complexes (blue, highly ordered, and consistently found in all complexes; yellow, low positional variation; red, large positional variation and not conserved in all protein–ligand complexes).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-21/acs.jmedchem.7b00845/20171103/images/large/jm-2017-00845s_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00845&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">We thus suggest exploiting the experiment with perdeuterated protein for the identification of protons with large LOGSY factors as the primary candidates for substitution with groups that either displace or address protein-bound water molecules. Successful applications for this novel strategy are shown in the <a href="/doi/suppl/10.1021/acs.jmedchem.7b00845/suppl_file/jm7b00845_si_001.pdf" class="ext-link">Supporting Information (Table S1)</a>. Overall, the replacement of the aromatic proton 8 with an amino or aminomethyl group in a series of ligands containing the triazolopyridazine ring led up to a 100-fold increase in binding affinity (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00845/suppl_file/jm7b00845_si_001.pdf" class="ext-link">Table S1</a>). Consistently, in all X-ray cocrystal structures obtained with such a modification, the triazolopyridazine ring binding mode of ligand <b>1</b> was found (methyl group pointing inward), and the (substituted) amino group was found to be hydrogen-bonded to the side chain of Asn140 of Brd4-BD1.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i5">Discussion and Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_06824" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_06824" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Detailed information about the binding mode of a fragment to its target as well as its environment in the bound state is indispensable for medicinal chemists to optimize early phase lead candidates with respect to affinity and specificity. Although X-ray crystallography is the method of choice to provide high-resolution structural information, it often fails in early stages of the drug optimization process because of low binding affinity and concomitantly compromised protein crystallizability. Additionally, it is well established that water molecules are often found at the protein–ligand interaction interface and contribute to the binding affinity by forming distinct hydrogen bonding patterns. Despite their crucial role in ligand-binding events, they are sometimes undetectable by X-ray crystallography. To overcome this bottleneck, we present a novel ligand-based NMR spectroscopy approach for the probing of ligand binding modes and the characterization of water hydration shells in protein–ligand binding clefts.</div><div class="NLM_p">Although our methodology shares some similarities with the already mentioned SALMON approach using short mixing times (τ<sub>mix</sub> = 150 ms), it is important to stress the fundamental differences.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> The use of short mixing times was shown to be necessary for the extraction of solvent-accessible epitopes of very buried ligands.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Unfortunately, due to the small size of Brd4-BD1 (MW = 15 kDa), it was impossible to record positive WaterLOGSY cross peaks for any of the ligands. Switching back to longer mixing times (τ<sub>mix</sub> = 1s), we realized that the distribution of positive and negative cross peaks strongly depends on the affinity of the ligand and on the protein concentration.</div><div class="NLM_p"><a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>a shows the 1D spectrum and two WaterLOGSY spectra of ligand <b>2</b> recorded at two different protein concentrations (24 μM Brd4-BD1 in blue, 45 μM Brd4-BD1 in red). It is important to note that the distribution of positive and negative signals is fairly different depending on the protein concentration. As can be seen from <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>a, although the SALMON approach provides a rough assessment of the ligand binding epitope, further details cannot be concluded from a single point measurement. In contrast, <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>b clearly shows that analysis of the corresponding LOGSY-titration curves reveals far more details about the ligand binding mode and internally bound water molecules. Most importantly, we could demonstrate that the method also identifies transiently bound or “disordered” water molecules that are not always resolved in the electron density map and that these water molecules are valid starting points for chemical replacement strategies to substitute bound water molecules by suitable functional groups. Several studies have shown that hydration water plays a pivotal role not only for the folding of proteins but also for protein–ligand interaction.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> An example of the beneficial impact of replacing water molecules to binding affinities was provided by Grzesiek and co-workers, who could show that a carbonyl group of the high-affinity HIV protease inhibitor DMP323 replaces a conserved H<sub>2</sub>O molecule and thereby contributes to the total binding free energy.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> The possibility of identifying protein-bound water molecules close to the bound ligand will clearly be of great value for the decision of medicinal chemists to incorporate additional functional groups, which either address retainable water or replace substitutable water. It is important to note that this valuable information is provided without atomistic structural information commonly provided by X-ray crystallography.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-21/acs.jmedchem.7b00845/20171103/images/medium/jm-2017-00845s_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-21/acs.jmedchem.7b00845/20171103/images/large/jm-2017-00845s_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Comparison between SALMON and LOGSY titration of ligand <b>2</b> interacting with protonated Brd4-BD1: (a) 1D spectrum of ligand <b>2</b> (1 mM) and WaterLOGSY spectra of ligand <b>2</b> with 24 μM Brd4-BD1 (blue) and 45 μM Brd4-BD1 (red); (b) LOGSY-titration of ligand <b>2</b> interacting with Brd4-BD1 (5–45 μM).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-21/acs.jmedchem.7b00845/20171103/images/large/jm-2017-00845s_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00845&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">In summary, we present a novel NMR approach for probing solvent accessibility and ligand binding modes of protein–ligand complexes. Most importantly, the method allows the identification of ligand positions for medicinal chemistry modifications to improve binding affinities through substituting or addressing bound water molecules. Given the facile applicability of the approach and the advantage of not needing a complete structure of the protein or a protein–ligand complex, we anticipate wide usage of the method particularly at early stages of drug discovery and especially for targets where obtaining X-ray cocrystal structures is limited or not possible.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i6">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_32627" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_32627" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Synthetic Methods. General Remarks</h3><div class="NLM_p">Chemicals and reagents were obtained from commercial suppliers and were used without further purification. Proton and carbon NMR spectra were obtained on a Bruker Avance 400 FT-NMR or Bruker Avance 500 FT-NMR instrument with chemical shifts (δ) reported relative to tetramethylsilane as an internal standard. High resolution mass spectrometry data were obtained on a LTQ Orbitrap XL (Thermo Scientific) equipped with a NSI source (Advion Nanomate) in ESI positive mode. Analytical HPLC–MS analyses were conducted using an Agilent 1100/1200 series LC/MSD system or UPLC-micromass triple quad. The analytic methods A1, A2, and A3 are defined in <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>. Compound purities were calculated as the percentage peak area of the analyzed compound by UV detection at 230–400 nm. If purity data are not explicitly mentioned, the compound displays a purity of >95%. Flash column chromatography was carried out using hand packed silica gel 60 (100–200 mesh), and product was eluted under medium pressure liquid chromatography. Preparative high performance chromatography was carried out on a Gilson system (pump system, 333 and 334 prep-scale HPLC pump; fraction collector, 215 liquid handler; detector, Gilson UV/vis 155) using prepacked reversed phase silica gel columns from Waters. The methods for preparative high performance chromatography P1_a, P1_b, and P2 are defined in <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Chromatography Methods</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center">column<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">solvent A</th><th class="colsep0 rowsep0" align="center">solvent B</th><th class="colsep0 rowsep0" align="center">flow rate (mL/min)</th><th class="colsep0 rowsep0" align="center">gradient<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">A1</td><td class="colsep0 rowsep0" align="left">Aquity UPLC BEH, C18, 1.7 μm, 2,1 mm × 100 mm, 40 °C</td><td class="colsep0 rowsep0" align="left">water with 0.1% formic acid</td><td class="colsep0 rowsep0" align="left">acetonitrile with 0.1% formic acid</td><td class="colsep0 rowsep0" align="left">0.4</td><td class="colsep0 rowsep0" align="left">05% → 100%, 7 min</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">A2</td><td class="colsep0 rowsep0" align="left">Venusil XBP-C18, 5 μm, 2.1 mm × 50 mm, 50 °C</td><td class="colsep0 rowsep0" align="left">water with 0.04% TFA</td><td class="colsep0 rowsep0" align="left">acetonitrile with 0.02% TFA</td><td class="colsep0 rowsep0" align="left">1.0</td><td class="colsep0 rowsep0" align="left">10% → 80%, 3 min</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">A3</td><td class="colsep0 rowsep0" align="left">Waters XBridge, C18, 2.5 μm, 2.1 mm × 20 mm, 60 °C</td><td class="colsep0 rowsep0" align="left">water, pH 9.2 (20 mM NH<sub>4</sub>HCO<sub>3</sub>)</td><td class="colsep0 rowsep0" align="left">acetonitrile</td><td class="colsep0 rowsep0" align="left">1.0</td><td class="colsep0 rowsep0" align="left">10% → 95%, 1.5 min</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">P1_a</td><td class="colsep0 rowsep0" align="left">Waters XBridge, C18, 10 μm, 30 mm × 100 mm</td><td class="colsep0 rowsep0" align="left">water, pH 8 (buffer: NH<sub>3</sub>, NH<sub>4</sub>HCO<sub>3</sub>)</td><td class="colsep0 rowsep0" align="left">acetonitrile</td><td class="colsep0 rowsep0" align="left">100</td><td class="colsep0 rowsep0" align="left">05% → 60%, 6 min</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">P1_b</td><td class="colsep0 rowsep0" align="left">Waters XBridge, C18, 10 μm, 50 mm × 100 mm</td><td class="colsep0 rowsep0" align="left">water, pH 8 (buffer: NH<sub>3</sub>, NH<sub>4</sub>HCO<sub>3</sub>)</td><td class="colsep0 rowsep0" align="left">acetonitrile</td><td class="colsep0 rowsep0" align="left">100</td><td class="colsep0 rowsep0" align="left">05% → 95%, 12 min</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">P2</td><td class="colsep0 rowsep0" align="left">Waters Sunfire, C18, 10 μm, 63 mm × 100 mm</td><td class="colsep0 rowsep0" align="left">water with 0.2% formic acid</td><td class="colsep0 rowsep0" align="left">acetonitrile</td><td class="colsep0 rowsep0" align="left">100</td><td class="colsep0 rowsep0" align="left">02% → 30%, 6 min</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>a</sup><p class="last">Company, column name, kind of particle, particle size, column dimension, column temperature.</p></div><div class="footnote" id="t1fn2"><sup>b</sup><p class="last">% of solvent B at gradient start → % of solvent B at gradient end, gradient time.</p></div></div></div></div><div id="sec4_2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Selected Experimental Procedures. Synthesis of Ligand <b>1</b></h3><div class="NLM_p last"><a href="/doi/suppl/10.1021/acs.jmedchem.7b00845/suppl_file/jm7b00845_si_001.pdf" class="ext-link">Scheme S1 in the Supporting Information</a> shows the structures of the compounds involved in the synthesis of ligand <b>1</b>.</div></div><div id="sec4_2_1_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> 6-(3,4-Dimethoxyphenyl)-3-methyl[1,2,4]triazolo[4,3-<i>b</i>]pyridazine (<b>1</b>)</h3><div class="NLM_p last">6-Chloro-3-methyl[1,2,4]triazolo[4,3-<i>b</i>]pyridazine (30 mg, 0.178 mmol), 3,4-dimethoxyphenylboronic acid (35 mg, 0.19 mmol), cesium carbonate (139 mg, 0.43 mmol), and 1,1′-(bis(diphenylphosphino)ferrocenepalladium(II) dichloride, dichloromethane (15 mg, 0.02 mmol) were dissolved in 0.2 mL of THF. The mixture was stirred for 30 min at 100 °C. The crude product was purified using method P1_a, yielding 0.04 g (38%, 0.12 mmol) of the title compound. Purity by method A3: >95%, <i>t</i><sub>R</sub> = 0.82 min. MS (ESI<sup>+</sup>) <i>m</i>/<i>z</i> 271 (M + H)<sup>+</sup>. HRMS (<i>m</i>/<i>z</i>): [M]<sup>+</sup> calcd for C<sub>14</sub>H<sub>14</sub>N<sub>4</sub>O<sub>2</sub>, 270.1117; found, 270.1111. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 500 MHz): δ = 8.46 (d, <i>J</i> = 9.8 Hz, 1H), 8.16 (d, <i>J</i> = 9.8 Hz, 1H), 7.78 (dd, <i>J</i> = 8.5, 2.2 Hz, 1H), 7.72 (d, <i>J</i> = 2.2 Hz, 1H), 7.16 (d, <i>J</i> = 8.5 Hz, 1H), 3.90 (s, 3H), 3.86 (s, 3H), 2.82 ppm (s, 3H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 125 MHz): δ = 153.3, 151.5, 149.1, 146.7, 142.2, 125.8, 123.8, 121.4, 120.9, 111.8, 110.1, 55.7, 55.6, 9.2 ppm.</div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Synthesis of Ligand <b>2</b></h3><div class="NLM_p last"><a href="/doi/suppl/10.1021/acs.jmedchem.7b00845/suppl_file/jm7b00845_si_001.pdf" class="ext-link">Scheme S2 in the Supporting Information</a> shows the structures of the compounds involved in the synthesis of ligand <b>2</b>.</div></div><div id="sec4_2_2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> 1-Methyl-5-phenoxy-1<i>H</i>-pyrazole-4-carboxylic Acid Ethyl Ester (<b>2</b>)</h3><div class="NLM_p last">Phenol (18.17 g; 193.08 mmol) was dissolved in 500 mL of DMA. Potassium carbonate (53.37 g; 386.16 mmol) was added in small portions, and the mixture was stirred for 10 min at room temperature. Afterward 5-bromo-1-methyl-1<i>H</i>-pyrazole-4-carboxylic acid ethyl ester (30.00 g; 128.72 mmol) was added to the reaction mixture dropwise and stirred for 16 h at 140 °C. 10% citric acid solution and dichloromethane were added. The organic layer was separated, dried, and the solvent was removed under reduced pressure. The crude product was purified using silica gel flash column chromatography with a solvent mixture of ethyl acetate and hexane for elution. The solvent of the corresponding fractions was evaporated under reduced pressure, yielding 18 g (57%, 73.0 mmol) of the title compound. Purity by method A2: >95%; <i>t</i><sub>R</sub> = 1.07 min. MS (ESI<sup>+</sup>) <i>m</i>/<i>z</i> 247 (M + H)<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ (ppm) 7.92 (s, 1H), 7.28–7.22 (m, 2H), 7.14–7.05 (m, 1H), 6.92–6.86 (d, 2H), 4.15–4.02 (m, 2H), 3.71 (s, 3H), 1.02–1.10 (m, 3H).</div></div><div id="sec4_2_2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> 1-Methyl-5-phenoxy-1<i>H</i>-pyrazole-4-carboxylic Acid (<b>3</b>)</h3><div class="NLM_p last">1-Methyl-5-phenoxy-1<i>H</i>-pyrazole-4-carboxylic acid ethyl ester <b>3</b> (0.45 g; 1.83 mmol) was dissolved in 10 mL of a 1:1 mixture of methanol and THF. Lithium hydroxide (0.15 g; 3.66 mmol) in 5 mL of water was added, and the reaction mixture was stirred for 16 h at 20 °C. Ethyl acetate was added to the mixture, and the aqueous layer was acidified with 1 M aqueous hydrochloric acid. The organic layer was removed under reduced pressure. The crude product yields 0.25 g (63%, 1.15 mmol) of the title compound. Purity by method A1: >95%; <i>t</i><sub>R</sub> = 2.92 min. MS (ESI<sup>+</sup>) <i>m</i>/<i>z</i> 219 (M + H)<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ (ppm) 9.80–12.20 (br, 1H), 7.92 (s, 1H), 7.34–7.22 (m, 2H), 7.12 (m, 1H), 6.92–6.85 (m, 2H), 2.65 (s, 3H).</div></div><div id="sec4_2_2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> 4-Iodo-1-methyl-5-phenoxy-1<i>H</i>-pyrazole (<b>5</b>)</h3><div class="NLM_p last">Under an atmosphere of argon, 1-methyl-5-phenoxy-1<i>H</i>-pyrazole-4-carboxylic acid <b>4</b> (20.00 g; 91.66 mmol) was suspended in 6.40 mL of <i>tert</i>-butanol and 160 mL of 1,4-dioxane. DIPEA (45.56 mL; 274.97 mmol) and diphenyl phosphorazidate (52.97 g; 192.48 mmol) were added to the mixture and stirred for 10 min at 20 °C. Afterward the mixture was stirred for 3 h at 110 °C. The solvent was removed under reduced pressure. The residue was purified using silica gel flash column chromatography with a solvent mixture of ethyl acetate and hexane for elution. The solvent of the corresponding fractions was evaporated under reduced pressure. This intermediate was dissolved in 4 M hydrochloric acid in 1,4-dioxane (3.20 mL; 12.80 mmol) and stirred for 2 days at 20 °C. The solvent was removed under reduced pressure. Aqueous sodium hydrogen carbonate solution and ethyl acetate were added to the residue. The organic layer was separated, dried, and the solvent was removed under reduced pressure. The crude product 1-methyl-5-phenoxy-1<i>H</i>-pyrazol-4-ylamine <b>5</b> (4 g, 21.0 mmol, 23%) was dissolved in aqueous hydrochloric acid (162 mL; 648 mmol) and cooled down to 0 °C. Afterward sodium nitrite (1.9 g; 2.7 mmol) dissolved in water was added and the mixture was stirred for 1 h at 0 °C. Then potassium iodide (10.5 g; 63 mmol) was added with vigorous stirring, and within 30 min the mixture warmed to 20 °C. An amount of 2 mL of water was added to the reaction mixture and neutralized with aqueous sodium hydrogen carbonate solution. Dichloromethane was added, the organic layer was separated and dried, and the solvent was removed under reduced pressure. The crude product was purified using silica gel flash column chromatography with a solvent mixture of ethyl acetate and petroleum ether for elution. The solvent of the corresponding fractions was evaporated under reduced pressure, yielding 3.8 g (61%; 12.7 mmol) of the title compound. Purity by method A2: >95%; <i>t</i><sub>R</sub> = 1.93 min. MS (ESI<sup>+</sup>) <i>m</i>/<i>z</i> 301 (M + H)<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ (ppm) 7.46 (s, 1H), 7.34–7.30 (m, 2H), 7.10 (m, 1H), 6.89–6.87 (m, 2H), 3.69 (s, 3H).</div></div><div id="sec4_2_2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> (1-Methyl-5-phenoxy-1<i>H</i>-pyrazol-4-yl)boronic Acid (<b>6</b>)</h3><div class="NLM_p last">4-Iodo-1-methyl-5-phenoxypyrazole <b>6</b> (0.5 g; 1.67 mmol) was dissolved in THF and cooled down at −70 °C. Afterward triisopropyl borate (0.35 g; 1.83 mmol) and 2.5 M <i>n</i>-butyllithium (1.06 mL; 2.67 mmol) were added to the solution and warmed overnight to 20 °C. The crude product was purified using silica gel flash column chromatography with a solvent mixture of ethyl acetate and petroleum ether for elution. The solvent of the corresponding fractions was evaporated under reduced pressure, yielding 0.15 (41%, 0.69 mmol) of the title compound. Purity by method A2 > 95%; <i>t</i><sub>R</sub> = 2.84 min. MS (ESI<sup>+</sup>) <i>m</i>/<i>z</i> 219 (M + H)<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ (ppm) 7.67 (s, 1H), 7.38–7.34 (m, 2H), 7.12 (m, 1H), 6.93–6.91 (m, 2H), 3.67 (s, 3H).</div></div><div id="sec4_2_2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> 3-Methyl-6-(1-methyl-5-phenoxy-1<i>H</i>-pyrazol-4-yl)[1,2,4]triazolo[4,3-<i>b</i>]pyridazine (<b>7</b>)</h3><div class="NLM_p last">6-Chloro-3-methyl[1,2,4]triazolo[4,3-<i>b</i>]pyridazine (70.00 mg; 0.42 mmol), (1-methyl-5-phenoxy-1<i>H</i>-pyrazol-4-yl)boronic acid (99.58 mg; 0.46 mmol), and sodium carbonate (88.02 mg; 0.83 mmol) were suspended in water (0.35 mL; 19.12 mmol) and <i>N</i>,<i>N</i>-dimethylformamide (4.00 mL; 51.98 mmol). The reaction mixture was flushed with nitrogen. Afterward 1,1′-bis(diphenylphosphino)ferrocenepalladium(II) dichloride and dichloromethane (16.95 mg; 0.02 mmol) were added, and the mixture was stirred in the microwave for 90 min at 120 °C. The reaction mixture was diluted with water and ethyl acetate. The organic layer was separated, washed with brine, dried, and the solvent was removed under reduced pressure. The crude product was purified using method P1_a, yielding 15 mg (12%, 0.049 mmol) of the title compound. Purity by method A3: >95%, <i>t</i><sub>R</sub> = 0.91 min. MS (ESI<sup>+</sup>) <i>m</i>/<i>z</i> 307 (M + H)<sup>+</sup>. HRMS (<i>m</i>/<i>z</i>): [M]<sup>+</sup> calcd for C<sub>16</sub>H<sub>14</sub>N<sub>6</sub>O, 306.1229; found, 306.1223. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 500 MHz): δ = 8.33 (s, 1H), 8.24 (d, <i>J</i> = 9.8 Hz, 1H), 7.60 (d, <i>J</i> = 9.8 Hz, 1H), 7.37 (br t, <i>J</i> = 7.7 Hz, 2H), 7.12 (br t, <i>J</i> = 7.4 Hz, 1H), 7.00 (br d, <i>J</i> = 8.2 Hz, 2H), 3.73 (s, 3H), 2.28 ppm (s, 3H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 125 MHz): δ = 156.3, 147.3, 146.7, 146.4, 142.9, 138.8, 130.6, 125.2, 124.0, 119.3, 115.5, 104.9, 35.1, 9.3 ppm.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Protein Expression and Purification</h3><div class="NLM_p last">Recombinant human Brd4-BD1 (bromodomain 1 of bromodomain containing protein 4) was expressed in <i>E. coli</i> BL21(DE3) and contains an N-terminal TEV-cleavable His6-tag. (The expression plasmid was kindly provided by Boehringer Ingelheim.) <sup>15</sup>N labeled H6-TEV-Brd4-BD1 was expressed following the expression protocol for efficient isotopic labeling of recombinant proteins using a 4-fold cell concentration in isotopically labeled minimal medium.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> After resuspension in minimal medium, cells were put for 5 min on ice and 30 min at 18 °C before induction with 0.4 mM IPTG (isopropyl-β-<span class="smallcaps smallerCapital">d</span>-1-thiogalactopyranoside). Perdeuterated H6-TEV-Brd4-BD1 was expressed by transferring small volumes of <i>E. coli</i> BL21(DE3) cells carrying the expression plasmid from M9 medium (no labeled material, 100% H<sub>2</sub>O) to M9<sup>50%D2O</sup> medium (no labeled material, 50% D<sub>2</sub>O, 50% H<sub>2</sub>O), and finally to 800 mL of M9<sup>100%D2O</sup> (no labeled material, 100% D<sub>2</sub>O). After reaching an OD<sub>600</sub> of 0.7–0.9, cells were centrifuged at 4000<i>g</i> for 5 min at 20 °C, washed once with a small amount of M9<sup>100%D2O</sup> (no labeled material, 100% D<sub>2</sub>O) and again centrifuged at 4000<i>g</i> for 5 min at 20 °C. The pellet was resuspended in 200 mL of M9<sup>100%D2O</sup> (containing 1 g/L <sup>15</sup>NH<sub>4</sub>Cl and <span class="smallcaps smallerCapital">d</span>-glucose-1,2,3,4,5,6,6-d<sub>7</sub>). Cells were put for 5 min on ice and 1 h at 18 °C before induction with 0.4 mM IPTG (isopropyl-β-<span class="smallcaps smallerCapital">d</span>-1-thiogalactopyranoside). Cells were collected after 18 h of expression at 18 °C, and the pellet was resuspended in 40 mL of buffer containing 20 mM sodium phosphate, 500 mM NaCl, 20 mM imidazole (pH 7.5), 5% glycerol, and protease inhibitor (Roche cOmplete Mini, EDTA free). Bacteria were lysed by sonication. Proteins were purified by Ni<sup>2+</sup> affinity chromatography (HiTrap Chelating HP, 5 mL, GE Healthcare), incorporating a high salt wash (buffer containing 1.5 M NaCl), and cleaved with TEV protease overnight at 4 °C (20 mM sodium phosphate, 500 mM NaCl, 2.5% glycerol, 2 mM β-mercaptoethanol). The cleaved protein was again loaded onto a Ni<sup>2+</sup> column to bind the cleaved His6-tag and the His6-tagged TEV protease and the flow-through containing Brd4-BD1 concentrated and loaded onto a gel filtration column equilibrated in 10 mM sodium phosphate, 100 mM NaCl (pH 7.5) (HiLoad 16/60 Superdex 75pg, GE Healthcare). Brd4-BD1 containing fractions were concentrated in an Amicon Ultra-15 centrifugal filter device 3K NMWL and stored at −20 °C.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> NMR Spectroscopy</h3><div class="NLM_p">1D, STD, and WaterLOGSY experiments were carried out at 277 and 298 K on a Varian Inova 500 MHz spectrometer using a standard 5 mm <sup>1</sup>H–<sup>13</sup>C–<sup>15</sup>N triple-resonance probe head. NMR spectra for the characterization of small molecules were collected at 298 K on a Bruker Avance III HD 500 MHz spectrometer. The system was equipped with a 5 mm TCI cryoprobe.</div><div id="sec4_4_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> 1D</h4><div class="NLM_p last">1D spectra were recorded with a double WATERGATE<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> suppression element.</div></div><div id="sec4_4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> WaterLOGSY</h4><div class="NLM_p last">The WaterLOGSY pulse sequence has been described before.<a onclick="showRef(event, 'ref4 ref24'); return false;" href="javascript:void(0);" class="ref ref4 ref24">(4, 24)</a> Selective water inversion was achieved with a double excitation sculpting element employing water selective refocusing pulses (RE-BURP,<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> inversion bandwidth of 0.5 ppm). The mixing time was set to 1 s, incorporating a hard 180° pulse in the middle of the mixing time and a water selective flip-back pulse at the beginning of the double WATERGATE water suppression module. The acquisition time and interscan delay were set to 1 and 2 s, respectively. The number of scans was set to 2048. Sample concentrations were typically 1 mM ligand and 0–45 μM Brd4-BD1 in a buffer (10 mM sodium phosphate, 100 mM NaCl, pH 7.5) containing 10% D<sub>2</sub>O. Intensity ratios <i>I</i><sub>logsy</sub>/<i>I</i><sub>1D</sub> are derived from scaling of the 1D intensity to the corresponding LOGSY intensity with the Bruker TopSpin 3.5pl5 software.</div></div><div id="sec4_4_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> STD</h4><div class="NLM_p last">STD experiments were performed in H<sub>2</sub>O. STD-NMR spectra were acquired as a pseudo-2D version, interleaving on- and off-resonance saturation. Selective saturation was achieved by a cascade of Gaussian pulses (bandwidth of 0.5 ppm, pulse length of 7.5 ms, power of 15 dB). On-resonance saturation was applied at 0 ppm and off-resonance saturation at −42 ppm. A T1ρ-filter of 50 ms was applied to eliminate protein resonances from the spectrum. The acquisition time and interscan delay were set to 0.5 and 4 s, respectively. The number of scans for either on- or off-resonance saturation was set to 256. Sample concentrations were 1 mM ligand and 45 μM Brd4-BD1 in a buffer (10 mM sodium phosphate, 100 mM NaCl, pH 7.5) containing 10% D<sub>2</sub>O. STD amplification factors (STD-AF) were calculated as described before.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> (<i>I</i><sub>ref</sub> – <i>I</i><sub>sat</sub>)/<i>I</i><sub>ref</sub> needed for STD-AF calculation was derived from scaling of the <i>I</i><sub>ref</sub> intensity to the corresponding STD intensity (<i>I</i><sub>ref</sub> – <i>I</i><sub>sat</sub>) with the Bruker TopSpin 3.5pl5 software. Initial growth rate STD amplification factors (STD-AF<sub>0</sub>) were obtained by fitting STD-AFs measured at different saturation times (<i>t</i><sub>sat</sub> = 0.5, 1, 2, 3, 4 s) and calculated as described before.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a></div></div><div id="sec4_4_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> Small Molecule NMR Spectroscopy</h4><div class="NLM_p last">Samples were dissolved in 600 μL of DMSO-<i>d</i><sub>6</sub>, and TMS was added as an internal standard. 1D <sup>1</sup>H spectra were acquired with 30° excitation pulses and an interpulse delay of 4.2 s with 64K data points and 20 ppm sweep width. 1D <sup>13</sup>C spectra were acquired with broadband composite pulse decoupling (WALTZ16) and an interpulse delay of 3.3 s with 64K data points and a sweep width of 240 ppm. Spectra were processed with Topspin 3.1 software from Bruker and analyzed with ACDlabs NMR workbook 2016.</div></div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Cocrystallization, Processing, and Refinement of BRD4-BD1 Ligand Structures</h3><div class="NLM_p">Recombinant protein of Brd4-BD1 for crystallographic studies was produced as formerly described.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> The protein was concentrated to a final concentration of 11 mg/mL in a buffer containing 10 mM HEPES at pH 7.5 and 100 mM sodium chloride.</div><div class="NLM_p">Brd4-BD1 protein was incubated with 2 mM ligand, and cocrystals in complex with ligand <b>1</b> and ligand <b>2</b> were grown by mixing 1 μL of protein solution with 1 μL of reservoir solution (29% PEG 3350 w/v, 200 mM disodium malonate, and HEPES, pH 7.2) using the hanging drop vapor diffusion method at 293 K. Crystals appeared after 4 days. Before flash-freezing in liquid nitrogen, the crystals were cryoprotected by the addition of ethylene glycol in the crystallization drop to a final concentration of 25% (v/v).</div><div class="NLM_p">Images were processed with autoPROC.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> The resolution limits were set using default autoPROC settings. The structures were solved by molecular replacement using the Brd4-BD1 structure <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2OSS">2OSS</a> as a search model. Subsequent model building and refinement were done using standard protocols using CCP4,<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> COOT,<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> and autoBUSTER.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> Crystallographic data collection statistics and refinement statistics can be found in the <a href="/doi/suppl/10.1021/acs.jmedchem.7b00845/suppl_file/jm7b00845_si_001.pdf" class="ext-link">Supporting Information (Tables S2 and S3)</a>.</div><div class="NLM_p">Compound <b>1</b> (space group = <i>P</i>1, unit cell <i>a</i> = 30.38 Å, <i>b</i> = 39.51 Å, <i>c</i> = 57.97 Å, α = 99,71°, β = 105,05°, γ = 89,90°, resolution = 1.38 Å) was refined to <i>R</i>/<i>R</i><sub>free</sub> = 22.2/24.2% with 100% of the residues in Ramachandran allowed regions.</div><div class="NLM_p last">Compound <b>2</b> (space group = <i>P</i>2<sub>1</sub>2<sub>1</sub>2<sub>1</sub>, unit cell <i>a</i> = 37.26 Å, <i>b</i> = 44.18 Å, <i>c</i> = 78.49 Å, α = β = γ = 90°, resolution = 1.65 Å) was refined to <i>R</i>/<i>R</i><sub>free</sub> = 18.7/21.4% with 100% of the residues in Ramachandran allowed regions.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i24"><a href="/doi/suppl/10.1021/acs.jmedchem.7b00845">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78321" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78321" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.7b00845" class="ext-link">10.1021/acs.jmedchem.7b00845</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Description of the LOGSY-titration methodology and the extraction of LOGSY factors; simulation of LOGSY-titration intensities; comparison of LOGSY titrations with protonated and deuterated protein; table of compounds that show an affinity increase after introduction of a secondary amine; schemes for the synthesis of ligand <b>1</b> and ligand <b>2</b>; crystallographic data collection and refinement statistics (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00845/suppl_file/jm7b00845_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and IC<sub>50</sub> values (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00845/suppl_file/jm7b00845_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00845/suppl_file/jm7b00845_si_001.pdf">jm7b00845_si_001.pdf (4.94 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00845/suppl_file/jm7b00845_si_002.csv">jm7b00845_si_002.csv (0.37 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Atomic coordinates of Brd4-BD1 bound to ligand <b>1</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5M39">5M39</a>) and ligand <b>2</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5M3A">5M3A</a>) have been deposited in the Protein Data Bank. Authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.7b00845" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_31024" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_31024" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Robert Konrat</span> - <span class="hlFld-Affiliation affiliation">Christian Doppler Laboratory for High-Content Structural
Biology and Biotechnology, Department of Structural and Computational
Biology, University of Vienna, Campus Vienna Biocenter 5, A-1030 Vienna, Austria</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#d2a0bdb0b7a0a6fcb9bdbca0b3a692a7bcbba4bbb7fcb3b1fcb3a6"><span class="__cf_email__" data-cfemail="89fbe6ebecfbfda7e2e6e7fbe8fdc9fce7e0ffe0eca7e8eaa7e8fd">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Leonhard Geist</span> - <span class="hlFld-Affiliation affiliation">Christian Doppler Laboratory for High-Content Structural
Biology and Biotechnology, Department of Structural and Computational
Biology, University of Vienna, Campus Vienna Biocenter 5, A-1030 Vienna, Austria</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3241-676X" title="Orcid link">http://orcid.org/0000-0002-3241-676X</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Moriz Mayer</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, A-1121 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiao-Ling Cockcroft</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, A-1121 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bernhard Wolkerstorfer</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, A-1121 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dirk Kessler</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, A-1121 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Harald Engelhardt</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, A-1121 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Darryl B. McConnell</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, A-1121 Vienna, Austria</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>L.G. performed the NMR experiments. L.G., M.M., and R.K. wrote the manuscript. M.M. and H.E. supervised the medicinal chemistry team. H.E. designed the synthetic strategies. D.K. and B.W. determined and analyzed the cocrystal structures. X.-L.C. performed the analysis on cocrystal structures. R.K. and D.B.M. supervised the study. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d49e1533-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i27">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_13535" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_13535" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors thank Dr. Andreas Zoephel for providing plasmids and protocols for Brd4-BD1 expression and purification, Teresa Adacker for writing the experimental section on the synthesis of the compounds, as well as Georg Kontaxis for pulse sequence programming and general NMR expertise. Funding for the work was provided by Boehringer Ingelheim and the Christian Doppler Laboratory for High-Content Structural Biology and Biotechnology, Austria.</p></div><div class="postAsapNotesSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="postAsapNotesSection">Note Added After ASAP Publication</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_74634" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_74634" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="first last">Due to a production error, the version of this paper that was published ASAP October 1, 2017, contained an error in the Acknowledgments. The corrected version was reposted October 2, 2017.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i28" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i28"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i29" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i29"> Abbreviations Used</h2><tr><td class="NLM_term">AFP-NOESY</td><td class="NLM_def"><p class="first last">adiabatic fast passage nuclear Overhauser effect spectroscopy</p></td></tr><tr><td class="NLM_term">BD</td><td class="NLM_def"><p class="first last">bromodomain</p></td></tr><tr><td class="NLM_term">BET</td><td class="NLM_def"><p class="first last">bromodomain and extra-terminal</p></td></tr><tr><td class="NLM_term">Brd4-BD1</td><td class="NLM_def"><p class="first last">bromodomain 1 of bromodomain containing protein 4</p></td></tr><tr><td class="NLM_term">DIRECTION</td><td class="NLM_def"><p class="first last">difference of inversion recovery rate with and without target irradiation</p></td></tr><tr><td class="NLM_term">GEM</td><td class="NLM_def"><p class="first last">group epitope mapping</p></td></tr><tr><td class="NLM_term">ILOE</td><td class="NLM_def"><p class="first last">interligand nuclear Overhauser effect</p></td></tr><tr><td class="NLM_term">INPHARMA</td><td class="NLM_def"><p class="first last">interligand nuclear Overhauser effect for pharmacophore mapping</p></td></tr><tr><td class="NLM_term">NOE</td><td class="NLM_def"><p class="first last">nuclear Overhauser effect</p></td></tr><tr><td class="NLM_term">SALMON</td><td class="NLM_def"><p class="first last">solvent accessibility, ligand binding, and mapping of ligand orientation by NMR</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">STD</td><td class="NLM_def"><p class="first last">saturation transfer difference</p></td></tr><tr><td class="NLM_term">STD-AF<sub>0</sub></td><td class="NLM_def"><p class="first last">initial growth rate saturation transfer difference amplification factor</p></td></tr><tr><td class="NLM_term">WaterLOGSY</td><td class="NLM_def"><p class="first last">water ligand observed via gradient spectroscopy</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i30">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_48736" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_48736" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 31 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Mayer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meyer, B.</span><span> </span><span class="NLM_article-title">Group epitope mapping by saturation transfer difference NMR to identify segments of a ligand in direct contact with a protein receptor</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">123</span><span class="NLM_x">, </span> <span class="NLM_fpage">6108</span><span class="NLM_x">–</span> <span class="NLM_lpage">6117</span><span class="refDoi"> DOI: 10.1021/ja0100120</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja0100120" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=1%3ACAS%3A528%3ADC%252BD3MXktFamsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=123&publication_year=2001&pages=6108-6117&author=M.+Mayerauthor=B.+Meyer&title=Group+epitope+mapping+by+saturation+transfer+difference+NMR+to+identify+segments+of+a+ligand+in+direct+contact+with+a+protein+receptor&doi=10.1021%2Fja0100120"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Group Epitope Mapping by Saturation Transfer Difference NMR To Identify Segments of a Ligand in Direct Contact with a Protein Receptor</span></div><div class="casAuthors">Mayer, Moriz; Meyer, Bernd</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">123</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">6108-6117</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A protocol based on satn. transfer difference (STD) NMR spectra was developed to characterize the binding interactions at an atom level, termed group epitope mapping (GEM).  As an example we chose the well-studied system of galactose binding to the 120-kDa lectin Ricinus communis agglutinin I (RCA120).  As ligands we used Me β-D-galactoside and a biantennary decasaccharide.  Anal. of the satn. transfer effects of Me β-D-galactoside showed that the H2, H3, and H4 protons are satd. to the highest degree, giving evidence of their close proximity to protons of the RCA120 lectin.  The direct interaction of the lectin with this region of the galactose is in excellent agreement with results obtained from the anal. of the binding specificities of many chem. modified galactose derivs. (Bhattacharyya, L.; Brewer, C. F., 1988).  This new NMR technique can identify the binding epitope of even complex ligands very quickly, which is a great improvement over time-consuming chem. modifications.  Efficient GEM benefits from a relatively high off rate of the ligand and a large excess of the ligand over the receptor.  Even for a ligand like the biantennary decasaccharide with micromolar binding affinity, the binding epitopes could easily be mapped to the terminal β-D-Gal-(1-4)-β-D-GlcNAc (β-D-GlcNAc = N-acetyl-D-glucosamine) residues located at the nonreducing end of the two carbohydrate chains.  The binding contribution of the terminal galactose residue is stronger than those of the penultimate GlcNAc residues.  We could show that the GlcNAc residues bind "edge-on" with the region from H2 to H4, making contact with the protein.  Anal. of STD NMR expts. performed under competitive conditions proved that the two saccharides studied bind at the same receptor site, thereby ruling out unspecific binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9BVgPNWHCsLVg90H21EOLACvtfcHk0lhBjUBMimPB1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXktFamsbg%253D&md5=5fe9d7d46114ee1cc96a8a50f8da972c</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1021%2Fja0100120&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja0100120%26sid%3Dliteratum%253Aachs%26aulast%3DMayer%26aufirst%3DM.%26aulast%3DMeyer%26aufirst%3DB.%26atitle%3DGroup%2520epitope%2520mapping%2520by%2520saturation%2520transfer%2520difference%2520NMR%2520to%2520identify%2520segments%2520of%2520a%2520ligand%2520in%2520direct%2520contact%2520with%2520a%2520protein%2520receptor%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2001%26volume%3D123%26spage%3D6108%26epage%3D6117%26doi%3D10.1021%2Fja0100120" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Ludwig, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michiels, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lodi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ride, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunce, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gunther, U. L.</span><span> </span><span class="NLM_article-title">Evaluation of solvent accessibility epitopes for different dehydrogenase inhibitors</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">1371</span><span class="NLM_x">–</span> <span class="NLM_lpage">1376</span><span class="refDoi"> DOI: 10.1002/cmdc.200800110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=10.1002%2Fcmdc.200800110" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2008&pages=1371-1376&author=C.+Ludwigauthor=P.+J.+Michielsauthor=A.+Lodiauthor=J.+Rideauthor=C.+Bunceauthor=U.+L.+Gunther&title=Evaluation+of+solvent+accessibility+epitopes+for+different+dehydrogenase+inhibitors&doi=10.1002%2Fcmdc.200800110"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.200800110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.200800110%26sid%3Dliteratum%253Aachs%26aulast%3DLudwig%26aufirst%3DC.%26aulast%3DMichiels%26aufirst%3DP.%2BJ.%26aulast%3DLodi%26aufirst%3DA.%26aulast%3DRide%26aufirst%3DJ.%26aulast%3DBunce%26aufirst%3DC.%26aulast%3DGunther%26aufirst%3DU.%2BL.%26atitle%3DEvaluation%2520of%2520solvent%2520accessibility%2520epitopes%2520for%2520different%2520dehydrogenase%2520inhibitors%26jtitle%3DChemMedChem%26date%3D2008%26volume%3D3%26spage%3D1371%26epage%3D1376%26doi%3D10.1002%2Fcmdc.200800110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Dalvit, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fogliatto, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stewart, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Veronesi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stockman, B.</span><span> </span><span class="NLM_article-title">WaterLOGSY as a method for primary NMR screening: practical aspects and range of applicability</span> <span class="citation_source-journal">J. Biomol. NMR</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">349</span><span class="NLM_x">–</span> <span class="NLM_lpage">359</span><span class="refDoi"> DOI: 10.1023/A:1013302231549</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=10.1023%2FA%3A1013302231549" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=11824754" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=1%3ACAS%3A528%3ADC%252BD38XptVyksw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2001&pages=349-359&author=C.+Dalvitauthor=G.+Fogliattoauthor=A.+Stewartauthor=M.+Veronesiauthor=B.+Stockman&title=WaterLOGSY+as+a+method+for+primary+NMR+screening%3A+practical+aspects+and+range+of+applicability&doi=10.1023%2FA%3A1013302231549"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">WaterLOGSY as a method for primary NMR screening: practical aspects and range of applicability</span></div><div class="casAuthors">Dalvit, Claudio; Fogliatto, GianPaolo; Stewart, Albert; Veronesi, Marina; Stockman, Brian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular NMR</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">349-359</span>CODEN:
                <span class="NLM_cas:coden">JBNME9</span>;
        ISSN:<span class="NLM_cas:issn">0925-2738</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic Publishers</span>)
        </div><div class="casAbstract">WaterLOGSY represents a powerful method for primary NMR screening in the identification of compds. interacting with macromols., including proteins and DNA or RNA fragments.  Several relay pathways are used constructively in the expt. for transferring bulk water magnetization to the ligand.  The method is particularly useful for the identification of novel scaffolds of micromolar affinity that can be then optimized using directed screening, combinatorial chem., medicinal chem. and structure-based drug design.  The practical aspects and range of applicability of the WaterLOGSY expt. are analyzed in detail here.  Competition binding and titrn. WaterLOGSY permit, after proper correction, the evaluation of the dissocn. binding const.  The high sensitivity of the technique in combination with the easy deconvolution of the mixts. for the identification of the active components, significantly reduces the amt. of material and time needed for the NMR screening process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHqqsXXXVX77Vg90H21EOLACvtfcHk0liEbnICMYyvEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XptVyksw%253D%253D&md5=e7b431d72aa17c499c1e751956605dee</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1023%2FA%3A1013302231549&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1013302231549%26sid%3Dliteratum%253Aachs%26aulast%3DDalvit%26aufirst%3DC.%26aulast%3DFogliatto%26aufirst%3DG.%26aulast%3DStewart%26aufirst%3DA.%26aulast%3DVeronesi%26aufirst%3DM.%26aulast%3DStockman%26aufirst%3DB.%26atitle%3DWaterLOGSY%2520as%2520a%2520method%2520for%2520primary%2520NMR%2520screening%253A%2520practical%2520aspects%2520and%2520range%2520of%2520applicability%26jtitle%3DJ.%2520Biomol.%2520NMR%26date%3D2001%26volume%3D21%26spage%3D349%26epage%3D359%26doi%3D10.1023%2FA%3A1013302231549" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Dalvit, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pevarello, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tatò, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Veronesi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vulpetti, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sundström, M.</span><span> </span><span class="NLM_article-title">Identification of compounds with binding affinity to proteins via magnetization transfer from bulk water</span> <span class="citation_source-journal">J. Biomol. NMR</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">65</span><span class="NLM_x">–</span> <span class="NLM_lpage">68</span><span class="refDoi"> DOI: 10.1023/A:1008354229396</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=10.1023%2FA%3A1008354229396" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=11061229" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=1%3ACAS%3A528%3ADC%252BD3cXnvVGit7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2000&pages=65-68&author=C.+Dalvitauthor=P.+Pevarelloauthor=M.+Tat%C3%B2author=M.+Veronesiauthor=A.+Vulpettiauthor=M.+Sundstr%C3%B6m&title=Identification+of+compounds+with+binding+affinity+to+proteins+via+magnetization+transfer+from+bulk+water&doi=10.1023%2FA%3A1008354229396"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of compounds with binding affinity to proteins via magnetization transfer from bulk water</span></div><div class="casAuthors">Dalvit, Claudio; Pevarello, Paolo; Tato, Marco; Veronesi, Marina; Vulpetti, Anna; Sundstrom, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular NMR</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">65-68</span>CODEN:
                <span class="NLM_cas:coden">JBNME9</span>;
        ISSN:<span class="NLM_cas:issn">0925-2738</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic Publishers</span>)
        </div><div class="casAbstract">A powerful screening by NMR methodol. (WaterLOGSY), based on transfer of magnetization from bulk water, for the identification of compds. that interact with target biomols. (proteins, RNA and DNA fragments) is described.  The method exploits efficiently the large reservoir of H2O magnetization.  The high sensitivity of the technique reduces the amt. of biomol. and ligands needed for the screening, which constitutes an important requirement for high throughput screening by NMR of large libraries of compds.  Application of the method to a compd. mixt. against the cyclin-dependent kinase 2 (cdk2) protein is presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoR8Jo0PHYdALVg90H21EOLACvtfcHk0liEbnICMYyvEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXnvVGit7o%253D&md5=e0fd0e5815f9bd20ed09d8033882feb6</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1023%2FA%3A1008354229396&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1008354229396%26sid%3Dliteratum%253Aachs%26aulast%3DDalvit%26aufirst%3DC.%26aulast%3DPevarello%26aufirst%3DP.%26aulast%3DTat%25C3%25B2%26aufirst%3DM.%26aulast%3DVeronesi%26aufirst%3DM.%26aulast%3DVulpetti%26aufirst%3DA.%26aulast%3DSundstr%25C3%25B6m%26aufirst%3DM.%26atitle%3DIdentification%2520of%2520compounds%2520with%2520binding%2520affinity%2520to%2520proteins%2520via%2520magnetization%2520transfer%2520from%2520bulk%2520water%26jtitle%3DJ.%2520Biomol.%2520NMR%26date%3D2000%26volume%3D18%26spage%3D65%26epage%3D68%26doi%3D10.1023%2FA%3A1008354229396" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Dalvit, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fagerness, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hadden, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sarver, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stockman, B. J.</span><span> </span><span class="NLM_article-title">Fluorine-NMR experiments for high-throughput screening: theoretical aspects, practical considerations, and range of applicability</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">125</span><span class="NLM_x">, </span> <span class="NLM_fpage">7696</span><span class="NLM_x">–</span> <span class="NLM_lpage">7703</span><span class="refDoi"> DOI: 10.1021/ja034646d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja034646d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjvFOitbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2003&pages=7696-7703&author=C.+Dalvitauthor=P.+E.+Fagernessauthor=D.+T.+Haddenauthor=R.+W.+Sarverauthor=B.+J.+Stockman&title=Fluorine-NMR+experiments+for+high-throughput+screening%3A+theoretical+aspects%2C+practical+considerations%2C+and+range+of+applicability&doi=10.1021%2Fja034646d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorine-NMR experiments for high-throughput screening: Theoretical aspects, practical considerations, and range of applicability</span></div><div class="casAuthors">Dalvit, Claudio; Fagerness, Paul E.; Hadden, Daneen T. A.; Sarver, Ronald W.; Stockman, Brian J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">7696-7703</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Competition ligand-based NMR screening expts. have recently been introduced to overcome most of the problems assocd. with traditional ligand-based NMR screening.  Mols. with marginal soly. and high affinity for a given target can be easily identified in a high-throughput manner by screening chem. mixts. against the target in the presence of a weak- to medium-affinity ligand of known binding const.  While the original competition-based approaches utilized 1H detection, significant advantages are obtained using 19F detection.  The absence of spectral overlap permits the screening of large chem. mixts. and allows for automated anal. of the spectra, even in the presence of protonated buffers, solvents, and detergents.  The large chem. shift anisotropy of fluorine and the significant exchange contribution allow for the selection of a weak-affinity spy mol., thus resulting in a lower binding affinity threshold for the identified NMR hits.  The method, labeled FAXS (fluorine chem. shift anisotropy and exchange for screening) is rapid and requires only a limited amt. of protein and, therefore, compares favorably with the other established non-NMR techniques used in high-throughput screening.  Herein the theor. aspects of this powerful 19F-based approach are presented and discussed in detail.  The exptl. conditions together with the detection limits and binding const. measurements are investigated using human serum albumin as the target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrluTqHSItDBLVg90H21EOLACvtfcHk0liEbnICMYyvEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjvFOitbs%253D&md5=58b7a2db41871c28e06873302d4a1ecb</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Fja034646d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja034646d%26sid%3Dliteratum%253Aachs%26aulast%3DDalvit%26aufirst%3DC.%26aulast%3DFagerness%26aufirst%3DP.%2BE.%26aulast%3DHadden%26aufirst%3DD.%2BT.%26aulast%3DSarver%26aufirst%3DR.%2BW.%26aulast%3DStockman%26aufirst%3DB.%2BJ.%26atitle%3DFluorine-NMR%2520experiments%2520for%2520high-throughput%2520screening%253A%2520theoretical%2520aspects%252C%2520practical%2520considerations%252C%2520and%2520range%2520of%2520applicability%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2003%26volume%3D125%26spage%3D7696%26epage%3D7703%26doi%3D10.1021%2Fja034646d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Shuker, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajduk, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meadows, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fesik, S. W.</span><span> </span><span class="NLM_article-title">Discovering high-affinity ligands for proteins: SAR by NMR</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">274</span><span class="NLM_x">, </span> <span class="NLM_fpage">1531</span><span class="NLM_x">–</span> <span class="NLM_lpage">1534</span><span class="refDoi"> DOI: 10.1126/science.274.5292.1531</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=10.1126%2Fscience.274.5292.1531" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=8929414" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=1%3ACAS%3A528%3ADyaK28Xnt1ansLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=274&publication_year=1996&pages=1531-1534&author=S.+B.+Shukerauthor=P.+J.+Hajdukauthor=R.+P.+Meadowsauthor=S.+W.+Fesik&title=Discovering+high-affinity+ligands+for+proteins%3A+SAR+by+NMR&doi=10.1126%2Fscience.274.5292.1531"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Discovering high-affinity ligands for proteins: SAR by NMR</span></div><div class="casAuthors">Shuker, Suzanne B.; Hajduk, Philip J.; Meadows, Robert P.; Fesik, Stephen W.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">274</span>
        (<span class="NLM_cas:issue">5292</span>),
    <span class="NLM_cas:pages">1531-1534</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A NMR-based method is described in which small org. mols. that bind to proximal subsites of a protein are identified, optimized, and linked together to produce high-affinity ligands.  The approach is called "SAR by NMR" because structure-activity relationships (SAR) are obtained from NMR.  With this technique, compds. with nanomolar affinities for the FK506 binding protein were rapidly discovered by tethering two ligands with micromolar affinities.  The method reduces the amt. of chem. synthesis and time required for the discovery of high-affinity ligands and appears particularly useful in target-directed drug research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwkuE4EVU_yrVg90H21EOLACvtfcHk0liEbnICMYyvEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xnt1ansLs%253D&md5=1b9a5a5e4f2d6ee4b1c249e3e1faf2ec</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1126%2Fscience.274.5292.1531&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.274.5292.1531%26sid%3Dliteratum%253Aachs%26aulast%3DShuker%26aufirst%3DS.%2BB.%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26aulast%3DMeadows%26aufirst%3DR.%2BP.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26atitle%3DDiscovering%2520high-affinity%2520ligands%2520for%2520proteins%253A%2520SAR%2520by%2520NMR%26jtitle%3DScience%26date%3D1996%26volume%3D274%26spage%3D1531%26epage%3D1534%26doi%3D10.1126%2Fscience.274.5292.1531" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Auer, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kloiber, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vavrinska, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geist, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coudevylle, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Konrat, R.</span><span> </span><span class="NLM_article-title">Pharmacophore mapping via cross-relaxation during adiabatic fast passage</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">132</span><span class="NLM_x">, </span> <span class="NLM_fpage">1480</span><span class="NLM_x">–</span> <span class="NLM_lpage">1481</span><span class="refDoi"> DOI: 10.1021/ja910098s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja910098s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=1480-1481&author=R.+Auerauthor=K.+Kloiberauthor=A.+Vavrinskaauthor=L.+Geistauthor=N.+Coudevylleauthor=R.+Konrat&title=Pharmacophore+mapping+via+cross-relaxation+during+adiabatic+fast+passage&doi=10.1021%2Fja910098s"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fja910098s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja910098s%26sid%3Dliteratum%253Aachs%26aulast%3DAuer%26aufirst%3DR.%26aulast%3DKloiber%26aufirst%3DK.%26aulast%3DVavrinska%26aufirst%3DA.%26aulast%3DGeist%26aufirst%3DL.%26aulast%3DCoudevylle%26aufirst%3DN.%26aulast%3DKonrat%26aufirst%3DR.%26atitle%3DPharmacophore%2520mapping%2520via%2520cross-relaxation%2520during%2520adiabatic%2520fast%2520passage%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26spage%3D1480%26epage%3D1481%26doi%3D10.1021%2Fja910098s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Villicana, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cruz, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zurita, M.</span><span> </span><span class="NLM_article-title">The basal transcription machinery as a target for cancer therapy</span> <span class="citation_source-journal">Cancer Cell Int.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">18</span><span class="refDoi"> DOI: 10.1186/1475-2867-14-18</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=10.1186%2F1475-2867-14-18" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=24576043" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjsVartbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2014&pages=18&author=C.+Villicanaauthor=G.+Cruzauthor=M.+Zurita&title=The+basal+transcription+machinery+as+a+target+for+cancer+therapy&doi=10.1186%2F1475-2867-14-18"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">The basal transcription machinery as a target for cancer therapy</span></div><div class="casAuthors">Villicana, Claudia; Cruz, Grisel; Zurita, Mario</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell International</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">18/1-18/15, 15 pp.</span>CODEN:
                <span class="NLM_cas:coden">CCIACC</span>;
        ISSN:<span class="NLM_cas:issn">1475-2867</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">General transcription is required for the growth and survival of all living cells.  However, tumor cells require extraordinary levels of transcription, including the transcription of rRNA genes by RNA polymerase I (RNPI) and mRNA by RNA polymerase II (RNPII).  In fact, cancer cells have mutations that directly enhance transcription and are frequently required for cancer transformation.  For example, the recent discovery that MYC enhances the transcription of the majority genes in the genome correlates with the fact that several transcription interfering drugs preferentially kill cancer cells.  In recent years, advances in the mechanistic studies of the basal transcription machinery and the discovery of drugs that interfere with multiple components of transcription are being used to combat cancer.  For example, drugs such as triptolide that targets the general transcription factors TFIIH and JQ1 to inhibit BRD4 are administered to target the high proliferative rate of cancer cells.  Given the importance of finding new strategies to preferentially sensitize tumor cells, this review primarily focuses on several transcription inhibitory drugs to demonstrate that the basal transcription machinery constitutes a potential target for the design of novel cancer drugs.  We highlight the drugs' mechanisms for interfering with tumor cell survival, their importance in cancer treatment and the challenges of clin. application.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7aEZQbw_0xLVg90H21EOLACvtfcHk0lhKmMCGgPXhyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjsVartbo%253D&md5=806c757e799d3a318038191c9e1c3520</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1186%2F1475-2867-14-18&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1475-2867-14-18%26sid%3Dliteratum%253Aachs%26aulast%3DVillicana%26aufirst%3DC.%26aulast%3DCruz%26aufirst%3DG.%26aulast%3DZurita%26aufirst%3DM.%26atitle%3DThe%2520basal%2520transcription%2520machinery%2520as%2520a%2520target%2520for%2520cancer%2520therapy%26jtitle%3DCancer%2520Cell%2520Int.%26date%3D2014%26volume%3D14%26spage%3D18%26doi%3D10.1186%2F1475-2867-14-18" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Ludwig, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michiels, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kavanagh, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pilka, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jansson, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oppermann, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gunther, U. L.</span><span> </span><span class="NLM_article-title">SALMON: solvent accessibility, ligand binding, and mapping of ligand orientation by NMR spectroscopy</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">3</span><span class="refDoi"> DOI: 10.1021/jm701020f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm701020f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=1-3&author=C.+Ludwigauthor=P.+J.+Michielsauthor=X.+Wuauthor=K.+L.+Kavanaghauthor=E.+Pilkaauthor=A.+Janssonauthor=U.+Oppermannauthor=U.+L.+Gunther&title=SALMON%3A+solvent+accessibility%2C+ligand+binding%2C+and+mapping+of+ligand+orientation+by+NMR+spectroscopy&doi=10.1021%2Fjm701020f"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fjm701020f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm701020f%26sid%3Dliteratum%253Aachs%26aulast%3DLudwig%26aufirst%3DC.%26aulast%3DMichiels%26aufirst%3DP.%2BJ.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DKavanagh%26aufirst%3DK.%2BL.%26aulast%3DPilka%26aufirst%3DE.%26aulast%3DJansson%26aufirst%3DA.%26aulast%3DOppermann%26aufirst%3DU.%26aulast%3DGunther%26aufirst%3DU.%2BL.%26atitle%3DSALMON%253A%2520solvent%2520accessibility%252C%2520ligand%2520binding%252C%2520and%2520mapping%2520of%2520ligand%2520orientation%2520by%2520NMR%2520spectroscopy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D1%26epage%3D3%26doi%3D10.1021%2Fjm701020f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Mizukoshi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abe, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takizawa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanzawa, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fukunishi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shimada, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takahashi, H.</span><span> </span><span class="NLM_article-title">An accurate pharmacophore mapping method by NMR spectroscopy</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">1362</span><span class="NLM_x">–</span> <span class="NLM_lpage">1365</span><span class="refDoi"> DOI: 10.1002/anie.201104905</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=10.1002%2Fanie.201104905" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=1362-1365&author=Y.+Mizukoshiauthor=A.+Abeauthor=T.+Takizawaauthor=H.+Hanzawaauthor=Y.+Fukunishiauthor=I.+Shimadaauthor=H.+Takahashi&title=An+accurate+pharmacophore+mapping+method+by+NMR+spectroscopy&doi=10.1002%2Fanie.201104905"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1002%2Fanie.201104905&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201104905%26sid%3Dliteratum%253Aachs%26aulast%3DMizukoshi%26aufirst%3DY.%26aulast%3DAbe%26aufirst%3DA.%26aulast%3DTakizawa%26aufirst%3DT.%26aulast%3DHanzawa%26aufirst%3DH.%26aulast%3DFukunishi%26aufirst%3DY.%26aulast%3DShimada%26aufirst%3DI.%26aulast%3DTakahashi%26aufirst%3DH.%26atitle%3DAn%2520accurate%2520pharmacophore%2520mapping%2520method%2520by%2520NMR%2520spectroscopy%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2012%26volume%3D51%26spage%3D1362%26epage%3D1365%26doi%3D10.1002%2Fanie.201104905" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Ni, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, Y.</span><span> </span><span class="NLM_article-title">Accounting for ligand-protein interactions in the relaxation-matrix analysis of transferred nuclear Overhauser effects</span> <span class="citation_source-journal">J. Magn. Reson., Ser. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">103</span><span class="NLM_x">, </span> <span class="NLM_fpage">180</span><span class="NLM_x">–</span> <span class="NLM_lpage">184</span><span class="refDoi"> DOI: 10.1006/jmrb.1994.1027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=10.1006%2Fjmrb.1994.1027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=8137078" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=1%3ACAS%3A528%3ADyaK2cXjtFymtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=1994&pages=180-184&author=F.+Niauthor=Y.+Zhu&title=Accounting+for+ligand-protein+interactions+in+the+relaxation-matrix+analysis+of+transferred+nuclear+Overhauser+effects&doi=10.1006%2Fjmrb.1994.1027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Accounting for ligand-protein interactions in the relaxation-matrix analysis of transferred nuclear Overhauser effects</span></div><div class="casAuthors">Ni, Feng; Zhu, Yi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Magnetic Resonance, Series B</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">180-4</span>CODEN:
                <span class="NLM_cas:coden">JMRBE5</span>;
        ISSN:<span class="NLM_cas:issn">1064-1866</span>.
    </div><div class="casAbstract">Transferred NOE is used to study ligand-protein interactions.  A simplified procedure is presented to account for relaxation effects mediated by specific ligand-protein interactions.  The influence on distance detn. will be discussed using simulation results.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqyfOc2JSPN2LVg90H21EOLACvtfcHk0lhhfnw0cR8KjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXjtFymtrc%253D&md5=0587e0be016ac736bcab3cd814c9ef17</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1006%2Fjmrb.1994.1027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmrb.1994.1027%26sid%3Dliteratum%253Aachs%26aulast%3DNi%26aufirst%3DF.%26aulast%3DZhu%26aufirst%3DY.%26atitle%3DAccounting%2520for%2520ligand-protein%2520interactions%2520in%2520the%2520relaxation-matrix%2520analysis%2520of%2520transferred%2520nuclear%2520Overhauser%2520effects%26jtitle%3DJ.%2520Magn.%2520Reson.%252C%2520Ser.%2520B%26date%3D1994%26volume%3D103%26spage%3D180%26epage%3D184%26doi%3D10.1006%2Fjmrb.1994.1027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Balaram, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bothner-By, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dadok, J.</span><span> </span><span class="NLM_article-title">Negative nuclear Overhauser effects as probes of macromolecular structure</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1972</span><span class="NLM_x">, </span> <span class="NLM_volume">94</span><span class="NLM_x">, </span> <span class="NLM_fpage">4015</span><span class="NLM_x">–</span> <span class="NLM_lpage">4017</span><span class="refDoi"> DOI: 10.1021/ja00766a063</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00766a063" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=1972&pages=4015-4017&author=P.+Balaramauthor=A.+A.+Bothner-Byauthor=J.+Dadok&title=Negative+nuclear+Overhauser+effects+as+probes+of+macromolecular+structure&doi=10.1021%2Fja00766a063"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fja00766a063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00766a063%26sid%3Dliteratum%253Aachs%26aulast%3DBalaram%26aufirst%3DP.%26aulast%3DBothner-By%26aufirst%3DA.%2BA.%26aulast%3DDadok%26aufirst%3DJ.%26atitle%3DNegative%2520nuclear%2520Overhauser%2520effects%2520as%2520probes%2520of%2520macromolecular%2520structure%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1972%26volume%3D94%26spage%3D4015%26epage%3D4017%26doi%3D10.1021%2Fja00766a063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Zeng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, M. M.</span><span> </span><span class="NLM_article-title">Bromodomain: an acetyl-lysine binding domain</span> <span class="citation_source-journal">FEBS Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">513</span><span class="NLM_x">, </span> <span class="NLM_fpage">124</span><span class="NLM_x">–</span> <span class="NLM_lpage">128</span><span class="refDoi"> DOI: 10.1016/S0014-5793(01)03309-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=10.1016%2FS0014-5793%2801%2903309-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=11911891" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=1%3ACAS%3A528%3ADC%252BD38XitFejtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=513&publication_year=2002&pages=124-128&author=L.+Zengauthor=M.+M.+Zhou&title=Bromodomain%3A+an+acetyl-lysine+binding+domain&doi=10.1016%2FS0014-5793%2801%2903309-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomain: an acetyl-lysine binding domain</span></div><div class="casAuthors">Zeng, Lei; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">513</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">124-128</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">A review with 43 refs.  Bromodomains, an extensive family of evolutionarily conserved protein modules originally found in proteins assocd. with chromatin and in nearly all nuclear histone acetyltransferases, have been recently discovered to function as acetyllysine-binding domains.  More recent structural studies of bromodomain/peptide ligand complexes have enriched the understanding of differences in ligand selectivity of bromodomains.  These new findings demonstrate that bromodomain/acetyllysine recognition can serve as a pivotal mechanism for regulating protein-protein interactions in numerous cellular processes including chromatin remodeling and transcriptional activation, and reinforce the concept that functional diversity of a conserved protein modular structure is achieved by evolutionary changes of amino acid sequences in the ligand binding site.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMESL8iZ9agLVg90H21EOLACvtfcHk0lhhfnw0cR8KjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XitFejtbo%253D&md5=79070a63cd171f846ab577aa9f7092bb</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2FS0014-5793%2801%2903309-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0014-5793%252801%252903309-9%26sid%3Dliteratum%253Aachs%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26atitle%3DBromodomain%253A%2520an%2520acetyl-lysine%2520binding%2520domain%26jtitle%3DFEBS%2520Lett.%26date%3D2002%26volume%3D513%26spage%3D124%26epage%3D128%26doi%3D10.1016%2FS0014-5793%2801%2903309-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Wu, S. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiang, C. M.</span><span> </span><span class="NLM_article-title">The double bromodomain-containing chromatin adaptor Brd4 and transcriptional regulation</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">282</span><span class="NLM_x">, </span> <span class="NLM_fpage">13141</span><span class="NLM_x">–</span> <span class="NLM_lpage">13145</span><span class="refDoi"> DOI: 10.1074/jbc.R700001200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=10.1074%2Fjbc.R700001200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=17329240" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=1%3ACAS%3A528%3ADC%252BD2sXks1Ghtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=2007&pages=13141-13145&author=S.+Y.+Wuauthor=C.+M.+Chiang&title=The+double+bromodomain-containing+chromatin+adaptor+Brd4+and+transcriptional+regulation&doi=10.1074%2Fjbc.R700001200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">The Double Bromodomain-containing Chromatin Adaptor Brd4 and Transcriptional Regulation</span></div><div class="casAuthors">Wu, Shwu-Yuan; Chiang, Cheng-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">282</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">13141-13145</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">A review.  Brd4 is a double bromodomain-contg. protein that binds preferentially to acetylated chromatin.  It belongs to the BET (bromodomains and extraterminal) family that includes mammalian Brd2, Brd3, Brd4, Brdt, Drosophila Fsh, yeast Bdf1, Bdf2, and corresponding homologues in other species.  Brd4 is essential for cellular growth and has been implicated in cell cycle control, DNA replication, and gene rearrangement found in t(15;19)-assocd. carcinomas.  Recently, Brd4 has been found in several transcription complexes, including the general cofactor Mediator and the P-TEFb elongation factor, and is capable of stimulating HIV-1 transcription in a Tat-independent manner.  In addn., Brd4 is used as a cellular adaptor by some animal and human papillomaviruses (HPV) for anchoring viral genomes to mitotic chromosomes.  This tethering, mediated by Brd4 interaction with virus-encoded E2 protein, facilitates viral genome segregation during mitosis.  Interestingly, Brd4 is also identified in a transcriptional silencing complex assembled by HPV E2 and turns out to be the long sought cellular corepressor that inhibits the expression of HPV-encoded E6 and E7 oncoproteins that antagonize p53 and pRB tumor suppressor activity, resp.  The dual role of Brd4 in gene activation and repression illustrates how a dynamic chromatin-binding adaptor is able to recruit distinct transcriptional regulators to modulate promoter activity through cell cycle progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohccDdZC9iKLVg90H21EOLACvtfcHk0lhhfnw0cR8KjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXks1Ghtbo%253D&md5=b025a5222b510a7009eca4289370b1d4</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1074%2Fjbc.R700001200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.R700001200%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DS.%2BY.%26aulast%3DChiang%26aufirst%3DC.%2BM.%26atitle%3DThe%2520double%2520bromodomain-containing%2520chromatin%2520adaptor%2520Brd4%2520and%2520transcriptional%2520regulation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2007%26volume%3D282%26spage%3D13141%26epage%3D13145%26doi%3D10.1074%2Fjbc.R700001200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Delmore, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Issa, G. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lemieux, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rahl, P. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobs, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kastritis, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilpatrick, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paranal, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chesi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schinzel, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKeown, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heffernan, T. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vakoc, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bergsagel, P. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghobrial, I. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, P. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hahn, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anderson, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kung, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitsiades, C. S.</span><span> </span><span class="NLM_article-title">BET bromodomain inhibition as a therapeutic strategy to target c-Myc</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">146</span><span class="NLM_x">, </span> <span class="NLM_fpage">904</span><span class="NLM_x">–</span> <span class="NLM_lpage">917</span><span class="refDoi"> DOI: 10.1016/j.cell.2011.08.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=10.1016%2Fj.cell.2011.08.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=21889194" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtF2ht7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=146&publication_year=2011&pages=904-917&author=J.+E.+Delmoreauthor=G.+C.+Issaauthor=M.+E.+Lemieuxauthor=P.+B.+Rahlauthor=J.+Shiauthor=H.+M.+Jacobsauthor=E.+Kastritisauthor=T.+Gilpatrickauthor=R.+M.+Paranalauthor=J.+Qiauthor=M.+Chesiauthor=A.+C.+Schinzelauthor=M.+R.+McKeownauthor=T.+P.+Heffernanauthor=C.+R.+Vakocauthor=P.+L.+Bergsagelauthor=I.+M.+Ghobrialauthor=P.+G.+Richardsonauthor=R.+A.+Youngauthor=W.+C.+Hahnauthor=K.+C.+Andersonauthor=A.+L.+Kungauthor=J.+E.+Bradnerauthor=C.+S.+Mitsiades&title=BET+bromodomain+inhibition+as+a+therapeutic+strategy+to+target+c-Myc&doi=10.1016%2Fj.cell.2011.08.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">BET Bromodomain Inhibition as a Therapeutic Strategy to Target c-Myc</span></div><div class="casAuthors">Delmore, Jake E.; Issa, Ghayas C.; Lemieux, Madeleine E.; Rahl, Peter B.; Shi, Jun-Wei; Jacobs, Hannah M.; Kastritis, Efstathios; Gilpatrick, Timothy; Paranal, Ronald M.; Qi, Jun; Chesi, Marta; Schinzel, Anna C.; McKeown, Michael R.; Heffernan, Timothy P.; Vakoc, Christopher R.; Bergsagel, P. Leif; Ghobrial, Irene M.; Richardson, Paul G.; Young, Richard A.; Hahn, William C.; Anderson, Kenneth C.; Kung, Andrew L.; Bradner, James E.; Mitsiades, Constantine S.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">146</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">904-917</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: MYC contributes to the pathogenesis of a majority of human cancers, yet strategies to modulate the function of the c-Myc oncoprotein do not exist.  Toward this objective, we have targeted MYC transcription by interfering with chromatin-dependent signal transduction to RNA polymerase, specifically by inhibiting the acetyl-lysine recognition domains (bromodomains) of putative coactivator proteins implicated in transcriptional initiation and elongation.  Using a selective small-mol. bromodomain inhibitor, JQ1, we identify BET bromodomain proteins as regulatory factors for c-Myc.  BET inhibition by JQ1 downregulates MYC transcription, followed by genome-wide downregulation of Myc-dependent target genes.  In exptl. models of multiple myeloma, a Myc-dependent hematol. malignancy, JQ1 produces a potent antiproliferative effect assocd. with cell-cycle arrest and cellular senescence.  Efficacy of JQ1 in three murine models of multiple myeloma establishes the therapeutic rationale for BET bromodomain inhibition in this disease and other malignancies characterized by pathol. activation of c-Myc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpevD1QHKmOhLVg90H21EOLACvtfcHk0ljVfmggQ9gyFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtF2ht7fF&md5=013674b8224214a9a1c8f0b75583a5a6</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2011.08.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2011.08.017%26sid%3Dliteratum%253Aachs%26aulast%3DDelmore%26aufirst%3DJ.%2BE.%26aulast%3DIssa%26aufirst%3DG.%2BC.%26aulast%3DLemieux%26aufirst%3DM.%2BE.%26aulast%3DRahl%26aufirst%3DP.%2BB.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DJacobs%26aufirst%3DH.%2BM.%26aulast%3DKastritis%26aufirst%3DE.%26aulast%3DGilpatrick%26aufirst%3DT.%26aulast%3DParanal%26aufirst%3DR.%2BM.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DChesi%26aufirst%3DM.%26aulast%3DSchinzel%26aufirst%3DA.%2BC.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DHeffernan%26aufirst%3DT.%2BP.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26aulast%3DBergsagel%26aufirst%3DP.%2BL.%26aulast%3DGhobrial%26aufirst%3DI.%2BM.%26aulast%3DRichardson%26aufirst%3DP.%2BG.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26aulast%3DHahn%26aufirst%3DW.%2BC.%26aulast%3DAnderson%26aufirst%3DK.%2BC.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DMitsiades%26aufirst%3DC.%2BS.%26atitle%3DBET%2520bromodomain%2520inhibition%2520as%2520a%2520therapeutic%2520strategy%2520to%2520target%2520c-Myc%26jtitle%3DCell%26date%3D2011%26volume%3D146%26spage%3D904%26epage%3D917%26doi%3D10.1016%2Fj.cell.2011.08.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Zuber, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rappaport, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herrmann, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sison, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Magoon, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blatt, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wunderlich, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johns, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chicas, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mulloy, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kogan, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valent, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lowe, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vakoc, C. R.</span><span> </span><span class="NLM_article-title">RNAi screen identifies Brd4 as a therapeutic target in acute myeloid leukaemia</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">478</span><span class="NLM_x">, </span> <span class="NLM_fpage">524</span><span class="NLM_x">–</span> <span class="NLM_lpage">528</span><span class="refDoi"> DOI: 10.1038/nature10334</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=10.1038%2Fnature10334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=21814200" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFers7%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=478&publication_year=2011&pages=524-528&author=J.+Zuberauthor=J.+Shiauthor=E.+Wangauthor=A.+R.+Rappaportauthor=H.+Herrmannauthor=E.+A.+Sisonauthor=D.+Magoonauthor=J.+Qiauthor=K.+Blattauthor=M.+Wunderlichauthor=M.+J.+Taylorauthor=C.+Johnsauthor=A.+Chicasauthor=J.+C.+Mulloyauthor=S.+C.+Koganauthor=P.+Brownauthor=P.+Valentauthor=J.+E.+Bradnerauthor=S.+W.+Loweauthor=C.+R.+Vakoc&title=RNAi+screen+identifies+Brd4+as+a+therapeutic+target+in+acute+myeloid+leukaemia&doi=10.1038%2Fnature10334"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">RNAi screen identifies Brd4 as a therapeutic target in acute myeloid leukaemia</span></div><div class="casAuthors">Zuber, Johannes; Shi, Junwei; Wang, Eric; Rappaport, Amy R.; Herrmann, Harald; Sison, Edward A.; Magoon, Daniel; Qi, Jun; Blatt, Katharina; Wunderlich, Mark; Taylor, Meredith J.; Johns, Christopher; Chicas, Agustin; Mulloy, James C.; Kogan, Scott C.; Brown, Patrick; Valent, Peter; Bradner, James E.; Lowe, Scott W.; Vakoc, Christopher R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">478</span>
        (<span class="NLM_cas:issue">7370</span>),
    <span class="NLM_cas:pages">524-528</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic pathways can regulate gene expression by controlling and interpreting chromatin modifications.  Cancer cells are characterized by altered epigenetic landscapes, and commonly exploit the chromatin regulatory machinery to enforce oncogenic gene expression programs.  Although chromatin alterations are, in principle, reversible and often amenable to drug intervention, the promise of targeting such pathways therapeutically has been limited by an incomplete understanding of cancer-specific dependencies on epigenetic regulators.  Here the authors describe a non-biased approach to probe epigenetic vulnerabilities in acute myeloid leukemia (AML), an aggressive hematopoietic malignancy that is often assocd. with aberrant chromatin states.  By screening a custom library of small hairpin RNAs (shRNAs) targeting known chromatin regulators in a genetically defined AML mouse model, the authors identify the protein bromodomain-contg. 4 (Brd4) as being critically required for disease maintenance.  Suppression of Brd4 using shRNAs or the small-mol. inhibitor JQ1 led to robust antileukemic effects in vitro and in vivo, accompanied by terminal myeloid differentiation and elimination of leukemia stem cells.  Similar sensitivities were obsd. in a variety of human AML cell lines and primary patient samples, revealing that JQ1 has broad activity in diverse AML subtypes.  The effects of Brd4 suppression are, at least in part, due to its role in sustaining Myc expression to promote aberrant self-renewal, which implicates JQ1 as a pharmacol. means to suppress MYC in cancer.  The authors' results establish small-mol. inhibition of Brd4 as a promising therapeutic strategy in AML and, potentially, other cancers, and highlight the utility of RNA interference (RNAi) screening for revealing epigenetic vulnerabilities that can be exploited for direct pharmacol. intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVyWFIqFkUVrVg90H21EOLACvtfcHk0ljVfmggQ9gyFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFers7%252FP&md5=618793a766f037bff5909524743fedb7</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnature10334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10334%26sid%3Dliteratum%253Aachs%26aulast%3DZuber%26aufirst%3DJ.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DE.%26aulast%3DRappaport%26aufirst%3DA.%2BR.%26aulast%3DHerrmann%26aufirst%3DH.%26aulast%3DSison%26aufirst%3DE.%2BA.%26aulast%3DMagoon%26aufirst%3DD.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DBlatt%26aufirst%3DK.%26aulast%3DWunderlich%26aufirst%3DM.%26aulast%3DTaylor%26aufirst%3DM.%2BJ.%26aulast%3DJohns%26aufirst%3DC.%26aulast%3DChicas%26aufirst%3DA.%26aulast%3DMulloy%26aufirst%3DJ.%2BC.%26aulast%3DKogan%26aufirst%3DS.%2BC.%26aulast%3DBrown%26aufirst%3DP.%26aulast%3DValent%26aufirst%3DP.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DLowe%26aufirst%3DS.%2BW.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26atitle%3DRNAi%2520screen%2520identifies%2520Brd4%2520as%2520a%2520therapeutic%2520target%2520in%2520acute%2520myeloid%2520leukaemia%26jtitle%3DNature%26date%3D2011%26volume%3D478%26spage%3D524%26epage%3D528%26doi%3D10.1038%2Fnature10334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Lockwood, W. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zejnullahu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varmus, H.</span><span> </span><span class="NLM_article-title">Sensitivity of human lung adenocarcinoma cell lines to targeted inhibition of BET epigenetic signaling proteins</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">109</span><span class="NLM_x">, </span> <span class="NLM_fpage">19408</span><span class="NLM_x">–</span> <span class="NLM_lpage">19413</span><span class="refDoi"> DOI: 10.1073/pnas.1216363109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=10.1073%2Fpnas.1216363109" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=19408-19413&author=W.+W.+Lockwoodauthor=K.+Zejnullahuauthor=J.+E.+Bradnerauthor=H.+Varmus&title=Sensitivity+of+human+lung+adenocarcinoma+cell+lines+to+targeted+inhibition+of+BET+epigenetic+signaling+proteins&doi=10.1073%2Fpnas.1216363109"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1216363109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1216363109%26sid%3Dliteratum%253Aachs%26aulast%3DLockwood%26aufirst%3DW.%2BW.%26aulast%3DZejnullahu%26aufirst%3DK.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DVarmus%26aufirst%3DH.%26atitle%3DSensitivity%2520of%2520human%2520lung%2520adenocarcinoma%2520cell%2520lines%2520to%2520targeted%2520inhibition%2520of%2520BET%2520epigenetic%2520signaling%2520proteins%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26spage%3D19408%26epage%3D19413%26doi%3D10.1073%2Fpnas.1216363109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Halle, B.</span><span> </span><span class="NLM_article-title">Protein hydration dynamics in solution: a critical survey</span> <span class="citation_source-journal">Philos. Trans. R. Soc., B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">359</span><span class="NLM_x">, </span> <span class="NLM_fpage">1207</span><span class="NLM_x">–</span> <span class="NLM_lpage">1223</span><div class="note"><p class="first last">(discussion 1223–1224, 1323–1328)</p></div><span class="refDoi"> DOI: 10.1098/rstb.2004.1499</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=10.1098%2Frstb.2004.1499" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=15306377" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=1%3ACAS%3A528%3ADC%252BD2cXnvFWhsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=359&publication_year=2004&pages=1207-1223&author=B.+Halle&title=Protein+hydration+dynamics+in+solution%3A+a+critical+survey&doi=10.1098%2Frstb.2004.1499"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Protein hydration dynamics in solution: a critical survey</span></div><div class="casAuthors">Halle, Bertil</div><div class="citationInfo"><span class="NLM_cas:title">Philosophical Transactions of the Royal Society of London, Series B: Biological Sciences</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">359</span>
        (<span class="NLM_cas:issue">1448</span>),
    <span class="NLM_cas:pages">1207-1224</span>CODEN:
                <span class="NLM_cas:coden">PTRBAE</span>;
        ISSN:<span class="NLM_cas:issn">0962-8436</span>.
    
            (<span class="NLM_cas:orgname">Royal Society</span>)
        </div><div class="casAbstract">A review and discussion.  The properties of water in biol. systems have been studied for well over a century by a wide range of phys. techniques, but progress has been slow and erratic.  Protein hydration, the perturbation of water structure and dynamics by the protein surface, has been a particularly rich source of controversy and confusion.  Here, the authors' aim has been to critically examine central concepts in the description of protein hydration, and to assess the exptl. basis for the current view of protein hydration, with the focus on dynamic aspects.  Recent 17O magnetic relaxation dispersion (MRD) expts. have shown that the vast majority of water mols. in the protein hydration layer suffer a mere 2-fold dynamic retardation compared with bulk water.  The high mobility of hydration water ensures that all thermally activated processes at the protein-water interface, such as binding, recognition, and catalysis, can proceed at high rates.  The MRD-derived picture of a highly mobile hydration layer is consistent with recent mol. dynamics simulations, but is incompatible with results deduced from intermol. NOE spectroscopy, dielec. relaxation, and fluorescence spectroscopy.  It is also inconsistent with the common view of hydration effects on protein hydrodynamics.  Here, the authors show how these discrepancies can be resolved.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptKKWGehwlY7Vg90H21EOLACvtfcHk0lj_Tjf8w2XRgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXnvFWhsbc%253D&md5=34ed99bb9fe54b2fc6264ed232bd966c</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1098%2Frstb.2004.1499&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1098%252Frstb.2004.1499%26sid%3Dliteratum%253Aachs%26aulast%3DHalle%26aufirst%3DB.%26atitle%3DProtein%2520hydration%2520dynamics%2520in%2520solution%253A%2520a%2520critical%2520survey%26jtitle%3DPhilos.%2520Trans.%2520R.%2520Soc.%252C%2520B%26date%3D2004%26volume%3D359%26spage%3D1207%26epage%3D1223%26doi%3D10.1098%2Frstb.2004.1499" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Ernst, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clubb, R. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, H. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gronenborn, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clore, G. M.</span><span> </span><span class="NLM_article-title">Demonstration of positionally disordered water within a protein hydrophobic cavity by NMR</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">267</span><span class="NLM_x">, </span> <span class="NLM_fpage">1813</span><span class="NLM_x">–</span> <span class="NLM_lpage">1817</span><span class="refDoi"> DOI: 10.1126/science.7892604</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=10.1126%2Fscience.7892604" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=7892604" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=1%3ACAS%3A528%3ADyaK2MXkslWgtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=267&publication_year=1995&pages=1813-1817&author=J.+A.+Ernstauthor=R.+T.+Clubbauthor=H.+X.+Zhouauthor=A.+M.+Gronenbornauthor=G.+M.+Clore&title=Demonstration+of+positionally+disordered+water+within+a+protein+hydrophobic+cavity+by+NMR&doi=10.1126%2Fscience.7892604"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Demonstration of positionally disordered water within a protein hydrophobic cavity by NMR</span></div><div class="casAuthors">Ernst, James A.; Clubb, Robert T.; Zhou, Huan-Xiang; Gronenborn, Angela M.; Clore, G. Marius</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">267</span>
        (<span class="NLM_cas:issue">5205</span>),
    <span class="NLM_cas:pages">1813-17</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The presence and location of water of hydration (i.e., bound water) in the soln. structure of human interleukin-1β (hIL-1β) was investigated with water-selective 2-dimensional heteronuclear magnetic resonance spectroscopy.  It is shown here that in addn. to water at the surface of the protein and ordered internal water mols. involved in bridging hydrogen bonds, positionally disordered water is present within a large, naturally occurring hydrophobic cavity located at the center of the mol.  These water mols. of hydration have residency times in the range of 1-2 ns to 100-200 μs and can be readily detected by NMR.  Thus, large hydrophobic cavities in proteins may not be truly empty, as anal. of crystal structures appears to show, but may contain mobile water mols. that are crystallog. invisible but detectable by NMR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfOkMo6iFnWrVg90H21EOLACvtfcHk0lj_Tjf8w2XRgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXkslWgtLc%253D&md5=0877f8e2068584119b3365d12952ffec</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1126%2Fscience.7892604&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.7892604%26sid%3Dliteratum%253Aachs%26aulast%3DErnst%26aufirst%3DJ.%2BA.%26aulast%3DClubb%26aufirst%3DR.%2BT.%26aulast%3DZhou%26aufirst%3DH.%2BX.%26aulast%3DGronenborn%26aufirst%3DA.%2BM.%26aulast%3DClore%26aufirst%3DG.%2BM.%26atitle%3DDemonstration%2520of%2520positionally%2520disordered%2520water%2520within%2520a%2520protein%2520hydrophobic%2520cavity%2520by%2520NMR%26jtitle%3DScience%26date%3D1995%26volume%3D267%26spage%3D1813%26epage%3D1817%26doi%3D10.1126%2Fscience.7892604" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Otting, G.</span><span> </span><span class="NLM_article-title">NMR studies of water bound to biological molecules</span> <span class="citation_source-journal">Prog. Nucl. Magn. Reson. Spectrosc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">259</span><span class="NLM_x">–</span> <span class="NLM_lpage">285</span><span class="refDoi"> DOI: 10.1016/S0079-6565(97)00012-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=10.1016%2FS0079-6565%2897%2900012-5" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=1997&pages=259-285&author=G.+Otting&title=NMR+studies+of+water+bound+to+biological+molecules&doi=10.1016%2FS0079-6565%2897%2900012-5"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2FS0079-6565%2897%2900012-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0079-6565%252897%252900012-5%26sid%3Dliteratum%253Aachs%26aulast%3DOtting%26aufirst%3DG.%26atitle%3DNMR%2520studies%2520of%2520water%2520bound%2520to%2520biological%2520molecules%26jtitle%3DProg.%2520Nucl.%2520Magn.%2520Reson.%2520Spectrosc.%26date%3D1997%26volume%3D31%26spage%3D259%26epage%3D285%26doi%3D10.1016%2FS0079-6565%2897%2900012-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Grzesiek, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bax, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicholson, L. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamazaki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wingfield, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stahl, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eyermann, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Torchia, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hodge, C. N.</span><span> </span><span class="NLM_article-title">NMR evidence for the displacement of a conserved interior water molecule in HIV protease by a non-peptide cyclic urea-based inhibitor</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">116</span><span class="NLM_x">, </span> <span class="NLM_fpage">1581</span><span class="NLM_x">–</span> <span class="NLM_lpage">1582</span><span class="refDoi"> DOI: 10.1021/ja00083a058</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00083a058" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=1994&pages=1581-1582&author=S.+Grzesiekauthor=A.+Baxauthor=L.+K.+Nicholsonauthor=T.+Yamazakiauthor=P.+Wingfieldauthor=S.+J.+Stahlauthor=C.+J.+Eyermannauthor=D.+A.+Torchiaauthor=C.+N.+Hodge&title=NMR+evidence+for+the+displacement+of+a+conserved+interior+water+molecule+in+HIV+protease+by+a+non-peptide+cyclic+urea-based+inhibitor&doi=10.1021%2Fja00083a058"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fja00083a058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00083a058%26sid%3Dliteratum%253Aachs%26aulast%3DGrzesiek%26aufirst%3DS.%26aulast%3DBax%26aufirst%3DA.%26aulast%3DNicholson%26aufirst%3DL.%2BK.%26aulast%3DYamazaki%26aufirst%3DT.%26aulast%3DWingfield%26aufirst%3DP.%26aulast%3DStahl%26aufirst%3DS.%2BJ.%26aulast%3DEyermann%26aufirst%3DC.%2BJ.%26aulast%3DTorchia%26aufirst%3DD.%2BA.%26aulast%3DHodge%26aufirst%3DC.%2BN.%26atitle%3DNMR%2520evidence%2520for%2520the%2520displacement%2520of%2520a%2520conserved%2520interior%2520water%2520molecule%2520in%2520HIV%2520protease%2520by%2520a%2520non-peptide%2520cyclic%2520urea-based%2520inhibitor%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1994%26volume%3D116%26spage%3D1581%26epage%3D1582%26doi%3D10.1021%2Fja00083a058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Marley, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bracken, C.</span><span> </span><span class="NLM_article-title">A method for efficient isotopic labeling of recombinant proteins</span> <span class="citation_source-journal">J. Biomol. NMR</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">71</span><span class="NLM_x">–</span> <span class="NLM_lpage">75</span><span class="refDoi"> DOI: 10.1023/A:1011254402785</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=10.1023%2FA%3A1011254402785" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=11430757" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=1%3ACAS%3A528%3ADC%252BD3MXksFCitL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2001&pages=71-75&author=J.+Marleyauthor=M.+Luauthor=C.+Bracken&title=A+method+for+efficient+isotopic+labeling+of+recombinant+proteins&doi=10.1023%2FA%3A1011254402785"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">A method for efficient isotopic labeling of recombinant proteins</span></div><div class="casAuthors">Marley, Jonathan; Lu, Min; Bracken, Clay</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular NMR</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">71-75</span>CODEN:
                <span class="NLM_cas:coden">JBNME9</span>;
        ISSN:<span class="NLM_cas:issn">0925-2738</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic Publishers</span>)
        </div><div class="casAbstract">A rapid and efficient approach for prepg. isotopically labeled recombinant proteins is presented.  The method is demonstrated for 13C labeling of the C-terminal domain of angiopoietin-2, 15N labeling of ubiquitin and for 2H/13C/15N labeling of the Escherichia coli outer-membrane lipoprotein Lpp-56.  The prodn. method generates cell mass using unlabeled rich media followed by exchange into a small vol. of labeled media at high cell d.  Following a short period for growth recovery and unlabeled metabolite clearance, the cells are induced.  The expression yields obtained provide a fourfold to eightfold redn. in isotope costs using simple shake flask growths.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoF39kyXJyC6rVg90H21EOLACvtfcHk0lj53sMcgPNJ9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXksFCitL0%253D&md5=db4495b09a80894194d3fdf8c7137073</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1023%2FA%3A1011254402785&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1011254402785%26sid%3Dliteratum%253Aachs%26aulast%3DMarley%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DM.%26aulast%3DBracken%26aufirst%3DC.%26atitle%3DA%2520method%2520for%2520efficient%2520isotopic%2520labeling%2520of%2520recombinant%2520proteins%26jtitle%3DJ.%2520Biomol.%2520NMR%26date%3D2001%26volume%3D20%26spage%3D71%26epage%3D75%26doi%3D10.1023%2FA%3A1011254402785" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Piotto, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saudek, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sklenář, V.</span><span> </span><span class="NLM_article-title">Gradient-tailored excitation for single-quantum NMR spectroscopy of aqueous solutions</span> <span class="citation_source-journal">J. Biomol. NMR</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1992</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">661</span><span class="NLM_x">–</span> <span class="NLM_lpage">665</span><span class="refDoi"> DOI: 10.1007/BF02192855</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=10.1007%2FBF02192855" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=1490109" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=1%3ACAS%3A528%3ADyaK3sXitVyktrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=1992&pages=661-665&author=M.+Piottoauthor=V.+Saudekauthor=V.+Sklen%C3%A1%C5%99&title=Gradient-tailored+excitation+for+single-quantum+NMR+spectroscopy+of+aqueous+solutions&doi=10.1007%2FBF02192855"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Gradient-tailored excitation for single-quantum NMR spectroscopy of aqueous solutions</span></div><div class="casAuthors">Piotto, Martial; Saudek, Valdimir; Sklenar, Vladimir</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular NMR</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">661-5</span>CODEN:
                <span class="NLM_cas:coden">JBNME9</span>;
        ISSN:<span class="NLM_cas:issn">0925-2738</span>.
    </div><div class="casAbstract">A novel approach to tailored selective excitation for the measurement of NMR spectra in non-deuterated aq. solns. (WATERGATE, WATER suppression by GrAdient-Tailored Excitation) is described.  The gradient echo sequence, which effectively combines one selective 180° radiofrequency pulse and two field gradient pulses, achieves highly selective and effective water suppression.  This technique is ideally suited for the rapid collection of multi-dimensional data since a single-scan acquisition produces a pure phase NMR spectrum with a perfectly flat baseline, at the highest possible sensitivity.  Application to the fast measurement of 2D NOE data of a 2.2. mM soln. of a double-stranded DNA fragment in 90% H2O at 5° is presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXTdL0-J714LVg90H21EOLACvtfcHk0lj53sMcgPNJ9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXitVyktrY%253D&md5=895aeeaffe2a1d9019763b39fdd0e0ef</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1007%2FBF02192855&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF02192855%26sid%3Dliteratum%253Aachs%26aulast%3DPiotto%26aufirst%3DM.%26aulast%3DSaudek%26aufirst%3DV.%26aulast%3DSklen%25C3%25A1%25C5%2599%26aufirst%3DV.%26atitle%3DGradient-tailored%2520excitation%2520for%2520single-quantum%2520NMR%2520spectroscopy%2520of%2520aqueous%2520solutions%26jtitle%3DJ.%2520Biomol.%2520NMR%26date%3D1992%26volume%3D2%26spage%3D661%26epage%3D665%26doi%3D10.1007%2FBF02192855" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Dalvit, C.</span><span> </span><span class="NLM_article-title">Efficient multiple-solvent suppression for the study of the interactions of organic solvents with biomolecules</span> <span class="citation_source-journal">J. Biomol. NMR</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">437</span><span class="NLM_x">–</span> <span class="NLM_lpage">444</span><span class="refDoi"> DOI: 10.1023/A:1008272928075</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=10.1023%2FA%3A1008272928075" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=1%3ACAS%3A528%3ADyaK1cXkslSjsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=1998&pages=437-444&author=C.+Dalvit&title=Efficient+multiple-solvent+suppression+for+the+study+of+the+interactions+of+organic+solvents+with+biomolecules&doi=10.1023%2FA%3A1008272928075"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Efficient multiple-solvent suppression for the study of the interactions of organic solvents with biomolecules</span></div><div class="casAuthors">Dalvit, Claudio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular NMR</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">437-444</span>CODEN:
                <span class="NLM_cas:coden">JBNME9</span>;
        ISSN:<span class="NLM_cas:issn">0925-2738</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic Publishers</span>)
        </div><div class="casAbstract">A modification of the excitation sculpting sequence for achieving efficient multiple-solvents suppression in 1D and 2D 1H NMR expts. is presented.  Implementation of this scheme in the ePHOGSY expt. allows rapid and sensitive detection of weak interactions between org. solvents or small mols. and biomols.  Application of the techniques to the peptide Sandostatin dissolved in H2O and DMSO is demonstrated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoiPrbdEy7HtLVg90H21EOLACvtfcHk0lj53sMcgPNJ9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXkslSjsb4%253D&md5=25848c35395da80d52d29cd1a941f804</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1023%2FA%3A1008272928075&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1008272928075%26sid%3Dliteratum%253Aachs%26aulast%3DDalvit%26aufirst%3DC.%26atitle%3DEfficient%2520multiple-solvent%2520suppression%2520for%2520the%2520study%2520of%2520the%2520interactions%2520of%2520organic%2520solvents%2520with%2520biomolecules%26jtitle%3DJ.%2520Biomol.%2520NMR%26date%3D1998%26volume%3D11%26spage%3D437%26epage%3D444%26doi%3D10.1023%2FA%3A1008272928075" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Geen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freeman, R.</span><span> </span><span class="NLM_article-title">Band-selective radiofrequency pulses</span> <span class="citation_source-journal">J. Magn. Reson. (1969-1992)</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1991</span><span class="NLM_x">, </span> <span class="NLM_volume">93</span><span class="NLM_x">, </span> <span class="NLM_fpage">93</span><span class="NLM_x">–</span> <span class="NLM_lpage">141</span><span class="refDoi"> DOI: 10.1016/0022-2364(91)90034-Q</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=10.1016%2F0022-2364%2891%2990034-Q" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=1991&pages=93-141&author=H.+Geenauthor=R.+Freeman&title=Band-selective+radiofrequency+pulses&doi=10.1016%2F0022-2364%2891%2990034-Q"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2F0022-2364%2891%2990034-Q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0022-2364%252891%252990034-Q%26sid%3Dliteratum%253Aachs%26aulast%3DGeen%26aufirst%3DH.%26aulast%3DFreeman%26aufirst%3DR.%26atitle%3DBand-selective%2520radiofrequency%2520pulses%26jtitle%3DJ.%2520Magn.%2520Reson.%2520%25281969-1992%2529%26date%3D1991%26volume%3D93%26spage%3D93%26epage%3D141%26doi%3D10.1016%2F0022-2364%2891%2990034-Q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Angulo, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Enriquez-Navas, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nieto, P. M.</span><span> </span><span class="NLM_article-title">Ligand-receptor binding affinities from saturation transfer difference (STD) NMR spectroscopy: the binding isotherm of STD initial growth rates</span> <span class="citation_source-journal">Chem. - Eur. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">7803</span><span class="NLM_x">–</span> <span class="NLM_lpage">7812</span><span class="refDoi"> DOI: 10.1002/chem.200903528</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=10.1002%2Fchem.200903528" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=20496354" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=1%3ACAS%3A528%3ADC%252BC3cXoslyrsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2010&pages=7803-7812&author=J.+Anguloauthor=P.+M.+Enriquez-Navasauthor=P.+M.+Nieto&title=Ligand-receptor+binding+affinities+from+saturation+transfer+difference+%28STD%29+NMR+spectroscopy%3A+the+binding+isotherm+of+STD+initial+growth+rates&doi=10.1002%2Fchem.200903528"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand-Receptor Binding Affinities from Saturation Transfer Difference (STD) NMR Spectroscopy: The Binding Isotherm of STD Initial Growth Rates</span></div><div class="casAuthors">Angulo, Jesus; Enriquez-Navas, Pedro M.; Nieto, Pedro M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - A European Journal</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">7803-7812, S7803/1-S7803/11</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The direct evaluation of dissocn. consts. (KD) from the variation of satn. transfer difference (STD) NMR spectroscopy values with the receptor-ligand ratio is not feasible due to the complex dependence of STD intensities on the spectral properties of the obsd. signals.  Indirect evaluation, by competition expts., allows the detn. of KD, as long as a ligand of known affinity is available for the protein under study.  Herein, the authors present a novel protocol based on STD NMR spectroscopy for the direct measurements of receptor-ligand dissocn. consts. (KD) from single-ligand titrn. expts.  The influence of several exptl. factors on STD values has been studied in detail, confirming the marked impact on std. detns. of protein-ligand affinities by STD NMR spectroscopy.  These factors, namely, STD satn. time, ligand residence time in the complex, and the intensity of the signal, affect the accumulation of satn. in the free ligand by processes closely related to fast protein-ligand rebinding and longitudinal relaxation of the ligand signals.  The proposed method avoids the dependence of the magnitudes of ligand STD signals at a given satn. time on spurious factors by constructing the binding isotherms using the initial growth rates of the STD amplification factors, in a similar way to the use of NOE growing rates to est. cross relaxation rates for distance evaluations.  Herein, it is demonstrated that the effects of these factors are cancelled out by analyzing the protein-ligand assocn. curve using STD values at the limit of zero satn. time, when virtually no ligand rebinding or relaxation takes place.  The approach is validated for two well-studied protein-ligand systems: the binding of the saccharides GlcNAc and GlcNAcβ1,4GlcNAc (chitobiose) to the wheat germ agglutinin (WGA) lectin, and the interaction of the amino acid L-tryptophan to bovine serum albumin (BSA).  In all cases, the exptl. KD measured under different exptl. conditions converged to the thermodn. values.  The proposed protocol allows accurate detns. of protein-ligand dissocn. consts., extending the applicability of the STD NMR spectroscopy for affinity measurements, which is of particular relevance for those proteins for which a ligand of known affinity is not available.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXZTY96tfffLVg90H21EOLACvtfcHk0liZhQea6ovMjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXoslyrsrY%253D&md5=df2bd44c799e3e5d88a08c0dcc0cb9dc</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1002%2Fchem.200903528&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fchem.200903528%26sid%3Dliteratum%253Aachs%26aulast%3DAngulo%26aufirst%3DJ.%26aulast%3DEnriquez-Navas%26aufirst%3DP.%2BM.%26aulast%3DNieto%26aufirst%3DP.%2BM.%26atitle%3DLigand-receptor%2520binding%2520affinities%2520from%2520saturation%2520transfer%2520difference%2520%2528STD%2529%2520NMR%2520spectroscopy%253A%2520the%2520binding%2520isotherm%2520of%2520STD%2520initial%2520growth%2520rates%26jtitle%3DChem.%2520-%2520Eur.%2520J.%26date%3D2010%26volume%3D16%26spage%3D7803%26epage%3D7812%26doi%3D10.1002%2Fchem.200903528" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mangos, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keates, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lambert, J.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Felletar, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Volkmer, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Müller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pawson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gingras, A.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span> </span><span class="NLM_article-title">Histone recognition and large-scale structural analysis of the human bromodomain family</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">149</span><span class="NLM_x">, </span> <span class="NLM_fpage">214</span><span class="NLM_x">–</span> <span class="NLM_lpage">231</span><span class="refDoi"> DOI: 10.1016/j.cell.2012.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=10.1016%2Fj.cell.2012.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=22464331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=149&publication_year=2012&pages=214-231&author=P.+Filippakopoulosauthor=S.+Picaudauthor=M.+Mangosauthor=T.+Keatesauthor=J.-P.+Lambertauthor=D.+Barsyte-Lovejoyauthor=I.+Felletarauthor=R.+Volkmerauthor=S.+M%C3%BCllerauthor=T.+Pawsonauthor=A.-C.+Gingrasauthor=C.+H.+Arrowsmithauthor=S.+Knapp&title=Histone+recognition+and+large-scale+structural+analysis+of+the+human+bromodomain+family&doi=10.1016%2Fj.cell.2012.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Recognition and Large-Scale Structural Analysis of the Human Bromodomain Family</span></div><div class="casAuthors">Filippakopoulos, Panagis; Picaud, Sarah; Mangos, Maria; Keates, Tracy; Lambert, Jean-Philippe; Barsyte-Lovejoy, Dalia; Felletar, Ildiko; Volkmer, Rudolf; Muller, Susanne; Pawson, Tony; Gingras, Anne-Claude; Arrowsmith, Cheryl H.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">149</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">214-231</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) are protein interaction modules that specifically recognize ε-N-lysine acetylation motifs, a key event in the reading process of epigenetic marks.  The 61 BRDs in the human genome cluster into eight families based on structure/sequence similarity.  Here, we present 29 high-resoln. crystal structures, covering all BRD families.  These proteins are: ASH1L, ATAD2, BAZ2B, BPTF, BRD1, BRD3(1), BRD3(2), BRD4(1), BRD4(2), BRD9, BRDT(1), CECR2, EP300, CREBBP, GCN5L2, KIAA1240, PB1(1), PB1(2), PB1(3), PB1(4), PB1(5), PB1(6), PCAF, PHIP(2), TAF1(2), WDR9(2), BRD4(1).  Comprehensive crossfamily structural anal. identifies conserved and family-specific structural features that are necessary for specific acetylation-dependent substrate recognition.  Screening of more than 30 representative BRDs against systematic histone-peptide arrays identifies new BRD substrates and reveals a strong influence of flanking posttranslational modifications, such as acetylation and phosphorylation, suggesting that BRDs recognize combinations of marks rather than singly acetylated sequences.  We further uncovered a structural mechanism for the simultaneous binding and recognition of diverse diacetyl-contg. peptides by BRD4.  These data provide a foundation for structure-based drug design of specific inhibitors for this emerging target family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK2FSIcPoXZLVg90H21EOLACvtfcHk0liZhQea6ovMjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D&md5=05583a01a1cd4c3b59fc59c00e4c4777</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DMangos%26aufirst%3DM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DLambert%26aufirst%3DJ.-P.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DVolkmer%26aufirst%3DR.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DPawson%26aufirst%3DT.%26aulast%3DGingras%26aufirst%3DA.-C.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DHistone%2520recognition%2520and%2520large-scale%2520structural%2520analysis%2520of%2520the%2520human%2520bromodomain%2520family%26jtitle%3DCell%26date%3D2012%26volume%3D149%26spage%3D214%26epage%3D231%26doi%3D10.1016%2Fj.cell.2012.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Vonrhein, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flensburg, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keller, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharff, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smart, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paciorek, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Womack, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bricogne, G.</span><span> </span><span class="NLM_article-title">Data processing and analysis with the autoPROC toolbox</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x">, </span> <span class="NLM_fpage">293</span><span class="NLM_x">–</span> <span class="NLM_lpage">302</span><span class="refDoi"> DOI: 10.1107/S0907444911007773</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=10.1107%2FS0907444911007773" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=21460447" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=293-302&author=C.+Vonrheinauthor=C.+Flensburgauthor=P.+Kellerauthor=A.+Sharffauthor=O.+Smartauthor=W.+Paciorekauthor=T.+Womackauthor=G.+Bricogne&title=Data+processing+and+analysis+with+the+autoPROC+toolbox&doi=10.1107%2FS0907444911007773"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Data processing and analysis with the autoPROC toolbox</span></div><div class="casAuthors">Vonrhein, Clemens; Flensburg, Claus; Keller, Peter; Sharff, Andrew; Smart, Oliver; Paciorek, Wlodek; Womack, Thomas; Bricogne, Gerard</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">293-302</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A typical diffraction expt. will generate many images and data sets from different crystals in a very short time.  This creates a challenge for the high-throughput operation of modern synchrotron beamlines as well as for the subsequent data processing.  Novice users in particular may feel overwhelmed by the tables, plots and nos. that the different data-processing programs and software packages present to them.  Here, some of the more common problems that a user has to deal with when processing a set of images that will finally make up a processed data set are shown, concg. on difficulties that may often show up during the first steps along the path of turning the expt. (i.e. data collection) into a model (i.e. interpreted electron d.).  Difficulties such as unexpected crystal forms, issues in crystal handling and suboptimal choices of data-collection strategies can often be dealt with, or at least diagnosed, by analyzing specific data characteristics during processing.  In the end, one wants to distinguish problems over which one has no immediate control once the expt. is finished from problems that can be remedied a posteriori.  A new software package, autoPROC, is also presented that combines third-party processing programs with new tools and an automated workflow script that is intended to provide users with both guidance and insight into the offline processing of data affected by the difficulties mentioned above, with particular emphasis on the automated treatment of multi-sweep data sets collected on multi-axis goniostats.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmah7iUrMLXLVg90H21EOLACvtfcHk0liZhQea6ovMjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLs%253D&md5=a0a3579f6b371b6312819883864aa80d</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1107%2FS0907444911007773&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444911007773%26sid%3Dliteratum%253Aachs%26aulast%3DVonrhein%26aufirst%3DC.%26aulast%3DFlensburg%26aufirst%3DC.%26aulast%3DKeller%26aufirst%3DP.%26aulast%3DSharff%26aufirst%3DA.%26aulast%3DSmart%26aufirst%3DO.%26aulast%3DPaciorek%26aufirst%3DW.%26aulast%3DWomack%26aufirst%3DT.%26aulast%3DBricogne%26aufirst%3DG.%26atitle%3DData%2520processing%2520and%2520analysis%2520with%2520the%2520autoPROC%2520toolbox%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D293%26epage%3D302%26doi%3D10.1107%2FS0907444911007773" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Winn, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ballard, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodson, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evans, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keegan, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krissinel, E. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leslie, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCoy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McNicholas, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murshudov, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pannu, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Potterton, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Powell, H. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vagin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, K. S.</span><span> </span><span class="NLM_article-title">Overview of the CCP4 suite and current developments</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x"> (</span><span class="NLM_issue">4</span><span class="NLM_x">) </span> <span class="NLM_fpage">235</span><span class="NLM_x">–</span> <span class="NLM_lpage">242</span><span class="refDoi"> DOI: 10.1107/S0907444910045749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=235-242&issue=4&author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=R.+M.+Keeganauthor=E.+B.+Krissinelauthor=A.+G.+Leslieauthor=A.+McCoyauthor=S.+J.+McNicholasauthor=G.+N.+Murshudovauthor=N.+S.+Pannuauthor=E.+A.+Pottertonauthor=H.+R.+Powellauthor=R.+J.+Readauthor=A.+Vaginauthor=K.+S.+Wilson&title=Overview+of+the+CCP4+suite+and+current+developments&doi=10.1107%2FS0907444910045749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0lh46CfdE9WgSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DKeegan%26aufirst%3DR.%2BM.%26aulast%3DKrissinel%26aufirst%3DE.%2BB.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%26aulast%3DMcCoy%26aufirst%3DA.%26aulast%3DMcNicholas%26aufirst%3DS.%2BJ.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DPotterton%26aufirst%3DE.%2BA.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520the%2520CCP4%2520suite%2520and%2520current%2520developments%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26issue%3D4%26spage%3D235%26epage%3D242%26doi%3D10.1107%2FS0907444910045749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lohkamp, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Features and development of Coot</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x"> (</span><span class="NLM_issue">4</span><span class="NLM_x">) </span> <span class="NLM_fpage">486</span><span class="NLM_x">–</span> <span class="NLM_lpage">501</span><span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&issue=4&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lh46CfdE9WgSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26issue%3D4%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Bricogne, G.; Blanc, E.; Brandl, M.; Flensburg, C.; Keller, P.; Paciorek, P.; Roversi, P.; Sharff, A.; Smart, O.; Vonrhein, C.; Womack, T.</span>BUSTER, version <span class="NLM_edition">2.11.2</span>; <span class="NLM_publisher-name">Global Phasing Ltd.</span>: <span class="NLM_publisher-loc">Cambridge, U.K.</span>,<span class="NLM_x"> </span><span class="NLM_year">2011</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&author=G.+Bricogne&author=E.+Blanc&author=M.+Brandl&author=C.+Flensburg&author=P.+Keller&author=P.+Paciorek&author=P.+Roversi&author=A.+Sharff&author=O.+Smart&author=C.+Vonrhein&author=T.+Womack&title=BUSTER"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DBricogne%26aufirst%3DG.%26jtitle%3DBUSTER%26pub%3DGlobal%2520Phasing%2520Ltd%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 20 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Gaochan Wu, Tong Zhao, Dongwei Kang, Jian Zhang, Yuning Song, Vigneshwaran Namasivayam, Jacob Kongsted, Christophe Pannecouque, Erik De Clercq, Vasanthanathan Poongavanam, Xinyong Liu, <span class="NLM_string-name hlFld-ContribAuthor">Peng Zhan</span>. </span><span class="cited-content_cbyCitation_article-title">Overview of Recent Strategic Advances in Medicinal Chemistry. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (21)
                                     , 9375-9414. <a href="https://doi.org/10.1021/acs.jmedchem.9b00359" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00359</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00359&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b00359%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DOverview%252Bof%252BRecent%252BStrategic%252BAdvances%252Bin%252BMedicinal%252BChemistry%26aulast%3DWu%26aufirst%3DGaochan%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D26022019%26date%3D28052019%26date%3D03052019%26volume%3D62%26issue%3D21%26spage%3D9375%26epage%3D9414" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ann-Christin  Pöppler</span>  . </span><span class="cited-content_cbyCitation_article-title">Filling Blank Spots on the Map: Identification of Ligand Binding Modes and Interacting Water Molecules for Brd4-BD1 by WaterLOGSY Titrations. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>60 </em>
                                    (21)
                                     , 8706-8707. <a href="https://doi.org/10.1021/acs.jmedchem.7b01497" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b01497</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b01497&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b01497%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DFilling%252BBlank%252BSpots%252Bon%252Bthe%252BMap%25253A%252BIdentification%252Bof%252BLigand%252BBinding%252BModes%252Band%252BInteracting%252BWater%252BMolecules%252Bfor%252BBrd4-BD1%252Bby%252BWaterLOGSY%252BTitrations%26aulast%3DP%25C3%25B6ppler%26aufirst%3DAnn-Christin%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D10102017%26date%3D30102017%26date%3D09112017%26volume%3D60%26issue%3D21%26spage%3D8706%26epage%3D8707" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dirk  Kessler</span>, <span class="hlFld-ContribAuthor ">Moriz  Mayer</span>, <span class="hlFld-ContribAuthor ">Stephan K.  Zahn</span>, <span class="hlFld-ContribAuthor ">Markus  Zeeb</span>, <span class="hlFld-ContribAuthor ">Simon  Wöhrle</span>, <span class="hlFld-ContribAuthor ">Andreas  Bergner</span>, <span class="hlFld-ContribAuthor ">Jens  Bruchhaus</span>, <span class="hlFld-ContribAuthor ">Tuncay  Ciftci</span>, <span class="hlFld-ContribAuthor ">Georg  Dahmann</span>, <span class="hlFld-ContribAuthor ">Maike  Dettling</span>, <span class="hlFld-ContribAuthor ">Sandra  Döbel</span>, <span class="hlFld-ContribAuthor ">Julian E.  Fuchs</span>, <span class="hlFld-ContribAuthor ">Leonhard  Geist</span>, <span class="hlFld-ContribAuthor ">Wolfgang  Hela</span>, <span class="hlFld-ContribAuthor ">Christiane  Kofink</span>, <span class="hlFld-ContribAuthor ">Roland  Kousek</span>, <span class="hlFld-ContribAuthor ">Franziska  Moser</span>, <span class="hlFld-ContribAuthor ">Teresa  Puchner</span>, <span class="hlFld-ContribAuthor ">Klaus  Rumpel</span>, <span class="hlFld-ContribAuthor ">Maximilian  Scharnweber</span>, <span class="hlFld-ContribAuthor ">Patrick  Werni</span>, <span class="hlFld-ContribAuthor ">Bernhard  Wolkerstorfer</span>, <span class="hlFld-ContribAuthor ">Dennis  Breitsprecher</span>, <span class="hlFld-ContribAuthor ">Philipp  Baaske</span>, <span class="hlFld-ContribAuthor ">Mark  Pearson</span>, <span class="hlFld-ContribAuthor ">Darryl B.  McConnell</span>, <span class="hlFld-ContribAuthor ">Jark  Böttcher</span>. </span><span class="cited-content_cbyCitation_article-title">Getting a Grip on the Undrugged: Targeting β‐Catenin with Fragment‐Based Methods. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2021,</strong> <em>16 </em>
                                    (9)
                                     , 1420-1424. <a href="https://doi.org/10.1002/cmdc.202000839" title="DOI URL">https://doi.org/10.1002/cmdc.202000839</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.202000839&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.202000839%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DGetting%252Ba%252BGrip%252Bon%252Bthe%252BUndrugged%25253A%252BTargeting%252B%2525CE%2525B2%2525E2%252580%252590Catenin%252Bwith%252BFragment%2525E2%252580%252590Based%252BMethods%26aulast%3DKessler%26aufirst%3DDirk%26date%3D2021%26date%3D2021%26volume%3D16%26issue%3D9%26spage%3D1420%26epage%3D1424" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Benjamin  Diethelm‐Varela</span>. </span><span class="cited-content_cbyCitation_article-title">Using NMR Spectroscopy in the Fragment‐Based Drug Discovery of Small‐Molecule Anticancer Targeted Therapies. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2021,</strong> <em>16 </em>
                                    (5)
                                     , 725-742. <a href="https://doi.org/10.1002/cmdc.202000756" title="DOI URL">https://doi.org/10.1002/cmdc.202000756</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.202000756&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.202000756%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DUsing%252BNMR%252BSpectroscopy%252Bin%252Bthe%252BFragment%2525E2%252580%252590Based%252BDrug%252BDiscovery%252Bof%252BSmall%2525E2%252580%252590Molecule%252BAnticancer%252BTargeted%252BTherapies%26aulast%3DDiethelm%25E2%2580%2590Varela%26aufirst%3DBenjamin%26date%3D2021%26date%3D2020%26volume%3D16%26issue%3D5%26spage%3D725%26epage%3D742" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bin  Yu</span>, <span class="hlFld-ContribAuthor ">Bin  Zhao</span>, <span class="hlFld-ContribAuthor ">Zesheng  Hao</span>, <span class="hlFld-ContribAuthor ">Lei  Chen</span>, <span class="hlFld-ContribAuthor ">Lixin  Cao</span>, <span class="hlFld-ContribAuthor ">Xiaofeng  Guo</span>, <span class="hlFld-ContribAuthor ">Nailou  Zhang</span>, <span class="hlFld-ContribAuthor ">Dongyan  Yang</span>, <span class="hlFld-ContribAuthor ">Liangfu  Tang</span>, <span class="hlFld-ContribAuthor ">Zhijin  Fan</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis and biological evaluation of pyrazole-aromatic containing carboxamides as potent SDH inhibitors. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>214 </em>, 113230. <a href="https://doi.org/10.1016/j.ejmech.2021.113230" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113230</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113230&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113230%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%252Band%252Bbiological%252Bevaluation%252Bof%252Bpyrazole-aromatic%252Bcontaining%252Bcarboxamides%252Bas%252Bpotent%252BSDH%252Binhibitors%26aulast%3DYu%26aufirst%3DBin%26date%3D2021%26volume%3D214%26spage%3D113230" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nipanshu  Agarwal</span>, <span class="hlFld-ContribAuthor ">Vivek  Chetry</span>, <span class="hlFld-ContribAuthor ">Krishna Mohan  Poluri</span>. </span><span class="cited-content_cbyCitation_article-title">Elucidating Protein-Ligand Interactions Using High Throughput Biophysical Techniques. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 205-244. <a href="https://doi.org/10.1007/978-981-15-8936-2_9" title="DOI URL">https://doi.org/10.1007/978-981-15-8936-2_9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-981-15-8936-2_9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-981-15-8936-2_9%26sid%3Dliteratum%253Aachs%26atitle%3DElucidating%252BProtein-Ligand%252BInteractions%252BUsing%252BHigh%252BThroughput%252BBiophysical%252BTechniques%26aulast%3DAgarwal%26aufirst%3DNipanshu%26date%3D2021%26date%3D2021%26spage%3D205%26epage%3D244%26pub%3DSpringer%2520Singapore%26atitle%3DInnovations%252Band%252BImplementations%252Bof%252BComputer%252BAided%252BDrug%252BDiscovery%252BStrategies%252Bin%252BRational%252BDrug%252BDesign%26aulast%3DSingh%26aufirst%3DSanjeev%2BKumar%26date%3D2021%26volume%3D81" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Abhinav  Dubey</span>, <span class="hlFld-ContribAuthor ">Koh  Takeuchi</span>, <span class="hlFld-ContribAuthor ">Mikhail  Reibarkh</span>, <span class="hlFld-ContribAuthor ">Haribabu  Arthanari</span>. </span><span class="cited-content_cbyCitation_article-title">The role of NMR in leveraging dynamics and entropy in drug design. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biomolecular NMR</span><span> <strong>2020,</strong> <em>74 </em>
                                    (10-11)
                                     , 479-498. <a href="https://doi.org/10.1007/s10858-020-00335-9" title="DOI URL">https://doi.org/10.1007/s10858-020-00335-9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10858-020-00335-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10858-020-00335-9%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biomolecular%2520NMR%26atitle%3DThe%252Brole%252Bof%252BNMR%252Bin%252Bleveraging%252Bdynamics%252Band%252Bentropy%252Bin%252Bdrug%252Bdesign%26aulast%3DDubey%26aufirst%3DAbhinav%26date%3D2020%26date%3D2020%26volume%3D74%26issue%3D10-11%26spage%3D479%26epage%3D498" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Abdul-Hamid  Emwas</span>, <span class="hlFld-ContribAuthor ">Kacper  Szczepski</span>, <span class="hlFld-ContribAuthor ">Benjamin Gabriel  Poulson</span>, <span class="hlFld-ContribAuthor ">Kousik  Chandra</span>, <span class="hlFld-ContribAuthor ">Ryan T.  McKay</span>, <span class="hlFld-ContribAuthor ">Manel  Dhahri</span>, <span class="hlFld-ContribAuthor ">Fatimah  Alahmari</span>, <span class="hlFld-ContribAuthor ">Lukasz  Jaremko</span>, <span class="hlFld-ContribAuthor ">Joanna Izabela  Lachowicz</span>, <span class="hlFld-ContribAuthor ">Mariusz  Jaremko</span>. </span><span class="cited-content_cbyCitation_article-title">NMR as a “Gold Standard” Method in Drug Design and Discovery. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2020,</strong> <em>25 </em>
                                    (20)
                                     , 4597. <a href="https://doi.org/10.3390/molecules25204597" title="DOI URL">https://doi.org/10.3390/molecules25204597</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules25204597&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules25204597%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DNMR%252Bas%252Ba%252B%2525E2%252580%25259CGold%252BStandard%2525E2%252580%25259D%252BMethod%252Bin%252BDrug%252BDesign%252Band%252BDiscovery%26aulast%3DEmwas%26aufirst%3DAbdul-Hamid%26date%3D2020%26date%3D2020%26volume%3D25%26issue%3D20%26spage%3D4597" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Manuel  Martin‐Pastor</span>, <span class="hlFld-ContribAuthor ">Edmont  Stoyanov</span>. </span><span class="cited-content_cbyCitation_article-title">Mechanism of interaction between hydroxypropyl cellulose and water in aqueous solutions: Importance of polymer chain length. </span><span class="cited-content_cbyCitation_journal-name">Journal of Polymer Science</span><span> <strong>2020,</strong> <em>58 </em>
                                    (12)
                                     , 1632-1641. <a href="https://doi.org/10.1002/pol.20200185" title="DOI URL">https://doi.org/10.1002/pol.20200185</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/pol.20200185&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fpol.20200185%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Polymer%2520Science%26atitle%3DMechanism%252Bof%252Binteraction%252Bbetween%252Bhydroxypropyl%252Bcellulose%252Band%252Bwater%252Bin%252Baqueous%252Bsolutions%25253A%252BImportance%252Bof%252Bpolymer%252Bchain%252Blength%26aulast%3DMartin%25E2%2580%2590Pastor%26aufirst%3DManuel%26date%3D2020%26date%3D2020%26volume%3D58%26issue%3D12%26spage%3D1632%26epage%3D1641" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Aditya  Pandey</span>, <span class="hlFld-ContribAuthor ">Sacha Thierry  Larda</span>, <span class="hlFld-ContribAuthor ">Steven  La Plante</span>, <span class="hlFld-ContribAuthor ">Robert Scott  Prosser</span>. </span><span class="cited-content_cbyCitation_article-title">NMR-based approaches to the study of GPCRs and GPCR-ligand interactions. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 65-80. <a href="https://doi.org/10.1016/B978-0-12-816228-6.00004-0" title="DOI URL">https://doi.org/10.1016/B978-0-12-816228-6.00004-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-816228-6.00004-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-816228-6.00004-0%26sid%3Dliteratum%253Aachs%26atitle%3DNMR-based%252Bapproaches%252Bto%252Bthe%252Bstudy%252Bof%252BGPCRs%252Band%252BGPCR-ligand%252Binteractions%26aulast%3DPandey%26aufirst%3DAditya%26date%3D2020%26spage%3D65%26epage%3D80%26pub%3DElsevier%26atitle%3DGPCRs%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fei-peng  Zhai</span>, <span class="hlFld-ContribAuthor ">Hong-en  Wei</span>, <span class="hlFld-ContribAuthor ">Yi  Liu</span>, <span class="hlFld-ContribAuthor ">Feng-yun  Hu</span>. </span><span class="cited-content_cbyCitation_article-title">Theoretical explanation for the pharmaceutical incompatibility through the cooperativity effect of the drug–drug intermolecular interactions in the phenobarbital∙∙∙paracetamol∙∙∙H2O complex. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Modeling</span><span> <strong>2019,</strong> <em>25 </em>
                                    (7)
                                     <a href="https://doi.org/10.1007/s00894-019-4060-1" title="DOI URL">https://doi.org/10.1007/s00894-019-4060-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00894-019-4060-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00894-019-4060-1%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Modeling%26atitle%3DTheoretical%252Bexplanation%252Bfor%252Bthe%252Bpharmaceutical%252Bincompatibility%252Bthrough%252Bthe%252Bcooperativity%252Beffect%252Bof%252Bthe%252Bdrug%2525E2%252580%252593drug%252Bintermolecular%252Binteractions%252Bin%252Bthe%252Bphenobarbital%2525E2%252588%252599%2525E2%252588%252599%2525E2%252588%252599paracetamol%2525E2%252588%252599%2525E2%252588%252599%2525E2%252588%252599H2O%252Bcomplex%26aulast%3DZhai%26aufirst%3DFei-peng%26date%3D2019%26date%3D2019%26volume%3D25%26issue%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Claudio  Dalvit</span>, <span class="hlFld-ContribAuthor ">Annick  Parent</span>, <span class="hlFld-ContribAuthor ">Francois  Vallée</span>, <span class="hlFld-ContribAuthor ">Magali  Mathieu</span>, <span class="hlFld-ContribAuthor ">Alexey  Rak</span>. </span><span class="cited-content_cbyCitation_article-title">Fast NMR Methods for Measuring in the Direct and/or Competition Mode the Dissociation Constants of Chemical Fragments Interacting with a Receptor. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2019,</strong> <em>14 </em>
                                    (11)
                                     , 1115-1127. <a href="https://doi.org/10.1002/cmdc.201900152" title="DOI URL">https://doi.org/10.1002/cmdc.201900152</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201900152&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201900152%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DFast%252BNMR%252BMethods%252Bfor%252BMeasuring%252Bin%252Bthe%252BDirect%252Band%25252For%252BCompetition%252BMode%252Bthe%252BDissociation%252BConstants%252Bof%252BChemical%252BFragments%252BInteracting%252Bwith%252Ba%252BReceptor%26aulast%3DDalvit%26aufirst%3DClaudio%26date%3D2019%26date%3D2019%26volume%3D14%26issue%3D11%26spage%3D1115%26epage%3D1127" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sanhita  Maity</span>, <span class="hlFld-ContribAuthor ">Ravi Kumar  Gundampati</span>, <span class="hlFld-ContribAuthor ">Thallapuranam Krishnaswamy  Suresh Kumar</span>. </span><span class="cited-content_cbyCitation_article-title">NMR Methods to Characterize Protein-Ligand Interactions. </span><span class="cited-content_cbyCitation_journal-name">Natural Product Communications</span><span> <strong>2019,</strong> <em>14 </em>
                                    (5)
                                     , 1934578X1984929. <a href="https://doi.org/10.1177/1934578X19849296" title="DOI URL">https://doi.org/10.1177/1934578X19849296</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1177/1934578X19849296&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1177%2F1934578X19849296%26sid%3Dliteratum%253Aachs%26jtitle%3DNatural%2520Product%2520Communications%26atitle%3DNMR%252BMethods%252Bto%252BCharacterize%252BProtein-Ligand%252BInteractions%26aulast%3DMaity%26aufirst%3DSanhita%26date%3D2019%26date%3D2019%26volume%3D14%26issue%3D5%26spage%3D1934578X1984929" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Pedro M  Nieto</span>. </span><span class="cited-content_cbyCitation_article-title">Weak Carbohydrate–Protein Interactions by
              NMR. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 1-8. <a href="https://doi.org/10.1002/9780470015902.a0028398" title="DOI URL">https://doi.org/10.1002/9780470015902.a0028398</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9780470015902.a0028398&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9780470015902.a0028398%26sid%3Dliteratum%253Aachs%26atitle%3DWeak%252BCarbohydrate%2525E2%252580%252593Protein%252BInteractions%252Bby%252BNMR%26aulast%3DNieto%26aufirst%3DPedro%2BM%26date%3D2019%26date%3D2019%26spage%3D1%26epage%3D8%26pub%3DWiley%26atitle%3DeLS%26aulast%3DJohn%2BWiley%2B%2526%2BSons%252C%2BLtd%26date%3D2001%26date%3D2001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">William C. K.  Pomerantz</span>, <span class="hlFld-ContribAuthor ">Jorden A.  Johnson</span>, <span class="hlFld-ContribAuthor ">Peter D.  Ycas</span>. </span><span class="cited-content_cbyCitation_article-title">Applied Biophysics for Bromodomain Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 287-337. <a href="https://doi.org/10.1007/7355_2019_79" title="DOI URL">https://doi.org/10.1007/7355_2019_79</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/7355_2019_79&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F7355_2019_79%26sid%3Dliteratum%253Aachs%26atitle%3DApplied%252BBiophysics%252Bfor%252BBromodomain%252BDrug%252BDiscovery%26aulast%3DPomerantz%26aufirst%3DWilliam%2BC.%2BK.%26date%3D2019%26date%3D2019%26spage%3D287%26epage%3D337%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DChemical%252BEpigenetics%26aulast%3DMai%26aufirst%3DAntonello%26date%3D2020%26volume%3D33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Renjie  Huang</span>, <span class="hlFld-ContribAuthor ">Ivanhoe K.H.  Leung</span>. </span><span class="cited-content_cbyCitation_article-title">Protein–Small Molecule Interactions by WaterLOGSY. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 477-500. <a href="https://doi.org/10.1016/bs.mie.2018.08.020" title="DOI URL">https://doi.org/10.1016/bs.mie.2018.08.020</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/bs.mie.2018.08.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fbs.mie.2018.08.020%26sid%3Dliteratum%253Aachs%26atitle%3DProtein%2525E2%252580%252593Small%252BMolecule%252BInteractions%252Bby%252BWaterLOGSY%26aulast%3DHuang%26aufirst%3DRenjie%26date%3D2019%26spage%3D477%26epage%3D500%26pub%3DElsevier%26atitle%3DBiological%252BNMR%252BPart%252BB%26date%3D2019%26volume%3D615" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Claire  Raingeval</span>, <span class="hlFld-ContribAuthor ">Olivier  Cala</span>, <span class="hlFld-ContribAuthor ">Béatrice  Brion</span>, <span class="hlFld-ContribAuthor ">Marc  Le Borgne</span>, <span class="hlFld-ContribAuthor ">Roderick Eliot  Hubbard</span>, <span class="hlFld-ContribAuthor ">Isabelle  Krimm</span>. </span><span class="cited-content_cbyCitation_article-title">1D NMR WaterLOGSY as an efficient method for fragment-based lead discovery. </span><span class="cited-content_cbyCitation_journal-name">Journal of Enzyme Inhibition and Medicinal Chemistry</span><span> <strong>2019,</strong> <em>34 </em>
                                    (1)
                                     , 1218-1225. <a href="https://doi.org/10.1080/14756366.2019.1636235" title="DOI URL">https://doi.org/10.1080/14756366.2019.1636235</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/14756366.2019.1636235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F14756366.2019.1636235%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Enzyme%2520Inhibition%2520and%2520Medicinal%2520Chemistry%26atitle%3D1D%252BNMR%252BWaterLOGSY%252Bas%252Ban%252Befficient%252Bmethod%252Bfor%252Bfragment-based%252Blead%252Bdiscovery%26aulast%3DRaingeval%26aufirst%3DClaire%26date%3D2019%26date%3D2019%26volume%3D34%26issue%3D1%26spage%3D1218%26epage%3D1225" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fernando  Prieto-Martínez</span>, <span class="hlFld-ContribAuthor ">José  Medina-Franco</span>. </span><span class="cited-content_cbyCitation_article-title">Flavonoids as Putative Epi-Modulators: Insight into Their Binding Mode with BRD4 Bromodomains Using Molecular Docking and Dynamics. </span><span class="cited-content_cbyCitation_journal-name">Biomolecules</span><span> <strong>2018,</strong> <em>8 </em>
                                    (3)
                                     , 61. <a href="https://doi.org/10.3390/biom8030061" title="DOI URL">https://doi.org/10.3390/biom8030061</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/biom8030061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fbiom8030061%26sid%3Dliteratum%253Aachs%26jtitle%3DBiomolecules%26atitle%3DFlavonoids%252Bas%252BPutative%252BEpi-Modulators%25253A%252BInsight%252Binto%252BTheir%252BBinding%252BMode%252Bwith%252BBRD4%252BBromodomains%252BUsing%252BMolecular%252BDocking%252Band%252BDynamics%26aulast%3DPrieto-Mart%25C3%25ADnez%26aufirst%3DFernando%26date%3D2018%26date%3D2018%26volume%3D8%26issue%3D3%26spage%3D61" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Vitor  Pomin</span>, <span class="hlFld-ContribAuthor ">Xu  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Glycosaminoglycan-Protein Interactions by Nuclear Magnetic Resonance (NMR) Spectroscopy. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2018,</strong> <em>23 </em>
                                    (9)
                                     , 2314. <a href="https://doi.org/10.3390/molecules23092314" title="DOI URL">https://doi.org/10.3390/molecules23092314</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules23092314&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules23092314%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DGlycosaminoglycan-Protein%252BInteractions%252Bby%252BNuclear%252BMagnetic%252BResonance%252B%252528NMR%252529%252BSpectroscopy%26aulast%3DPomin%26aufirst%3DVitor%26date%3D2018%26date%3D2018%26volume%3D23%26issue%3D9%26spage%3D2314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Pedro M.  Nieto</span>. </span><span class="cited-content_cbyCitation_article-title">The Use of NMR to Study Transient Carbohydrate—Protein Interactions. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Molecular Biosciences</span><span> <strong>2018,</strong> <em>5 </em><a href="https://doi.org/10.3389/fmolb.2018.00033" title="DOI URL">https://doi.org/10.3389/fmolb.2018.00033</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fmolb.2018.00033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffmolb.2018.00033%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Molecular%2520Biosciences%26atitle%3DThe%252BUse%252Bof%252BNMR%252Bto%252BStudy%252BTransient%252BCarbohydrate%2525E2%252580%252594Protein%252BInteractions%26aulast%3DNieto%26aufirst%3DPedro%2BM.%26date%3D2018%26date%3D2018%26volume%3D5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-21/acs.jmedchem.7b00845/20171103/images/medium/jm-2017-00845s_0006.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-21/acs.jmedchem.7b00845/20171103/images/large/jm-2017-00845s_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00845&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-21/acs.jmedchem.7b00845/20171103/images/medium/jm-2017-00845s_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-21/acs.jmedchem.7b00845/20171103/images/large/jm-2017-00845s_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Concept of the LOGSY-titration experiment. (a) For reversibly binding ligands (mM to μM) magnetization received from water during the WaterLOGSY experiment is a combination of cross-relaxation obtained from bulk water and cross-relaxation transferred either directly via protein-bound water molecules or indirectly via spin-diffusion mediated by protein protons. (b) LOGSY-titration: <i>I</i><sub>logsy</sub>/<i>I</i><sub>ref</sub> is plotted as a function of the protein concentration. Intensities in the WaterLOGSY are normalized to the <sup>1</sup>H 1D reference spectrum measured at the same protein concentration. The slopes are fitted to a linear function and normalized to the graph with the strongest slope (LOGSY factor of 100%).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-21/acs.jmedchem.7b00845/20171103/images/large/jm-2017-00845s_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00845&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-21/acs.jmedchem.7b00845/20171103/images/medium/jm-2017-00845s_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-21/acs.jmedchem.7b00845/20171103/images/large/jm-2017-00845s_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. LOGSY factors calculated from the slopes of the LOGSY titrations and normalized STD-AF<sub>0</sub> values for (a, b) ligand <b>1</b> and (d, e) ligand <b>2</b>. (c, f) X-ray cocrystal structures of the complexes Brd4-BD1 with ligand <b>1</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5M39">5M39</a>) and ligand <b>2</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5M3A">5M3A</a>), respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-21/acs.jmedchem.7b00845/20171103/images/large/jm-2017-00845s_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00845&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-21/acs.jmedchem.7b00845/20171103/images/medium/jm-2017-00845s_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-21/acs.jmedchem.7b00845/20171103/images/large/jm-2017-00845s_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. LOGSY factors calculated from the slopes of the LOGSY titrations with perdeuterated Brd4-BD1 for (a) ligand <b>1</b> and (c) ligand <b>2</b>. 100% corresponds to the signal with the strongest slope. (b, d) X-ray cocrystal structures of the complexes Brd4-BD1 with ligand <b>1</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5M39">5M39</a>) and ligand <b>2</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5M3A">5M3A</a>), respectively. H<sub>2</sub>O molecules resolved in the electron density map and within a distance of 6 Å from the ligand  are shown as blue spheres.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-21/acs.jmedchem.7b00845/20171103/images/large/jm-2017-00845s_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00845&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-21/acs.jmedchem.7b00845/20171103/images/medium/jm-2017-00845s_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-21/acs.jmedchem.7b00845/20171103/images/large/jm-2017-00845s_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Superposition of H<sub>2</sub>O molecules in Brd4-BD1 complex structures of six different ligand cocrystal structures. Ligand <b>1</b> is shown as representative ligand structure. Spheres represent H<sub>2</sub>O molecules found in the X-ray structures of the complexes (blue, highly ordered, and consistently found in all complexes; yellow, low positional variation; red, large positional variation and not conserved in all protein–ligand complexes).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-21/acs.jmedchem.7b00845/20171103/images/large/jm-2017-00845s_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00845&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-21/acs.jmedchem.7b00845/20171103/images/medium/jm-2017-00845s_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-21/acs.jmedchem.7b00845/20171103/images/large/jm-2017-00845s_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Comparison between SALMON and LOGSY titration of ligand <b>2</b> interacting with protonated Brd4-BD1: (a) 1D spectrum of ligand <b>2</b> (1 mM) and WaterLOGSY spectra of ligand <b>2</b> with 24 μM Brd4-BD1 (blue) and 45 μM Brd4-BD1 (red); (b) LOGSY-titration of ligand <b>2</b> interacting with Brd4-BD1 (5–45 μM).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-21/acs.jmedchem.7b00845/20171103/images/large/jm-2017-00845s_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00845&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i30">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_02562" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_02562" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 31 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Mayer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meyer, B.</span><span> </span><span class="NLM_article-title">Group epitope mapping by saturation transfer difference NMR to identify segments of a ligand in direct contact with a protein receptor</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">123</span><span class="NLM_x">, </span> <span class="NLM_fpage">6108</span><span class="NLM_x">–</span> <span class="NLM_lpage">6117</span><span class="refDoi"> DOI: 10.1021/ja0100120</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja0100120" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=1%3ACAS%3A528%3ADC%252BD3MXktFamsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=123&publication_year=2001&pages=6108-6117&author=M.+Mayerauthor=B.+Meyer&title=Group+epitope+mapping+by+saturation+transfer+difference+NMR+to+identify+segments+of+a+ligand+in+direct+contact+with+a+protein+receptor&doi=10.1021%2Fja0100120"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Group Epitope Mapping by Saturation Transfer Difference NMR To Identify Segments of a Ligand in Direct Contact with a Protein Receptor</span></div><div class="casAuthors">Mayer, Moriz; Meyer, Bernd</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">123</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">6108-6117</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A protocol based on satn. transfer difference (STD) NMR spectra was developed to characterize the binding interactions at an atom level, termed group epitope mapping (GEM).  As an example we chose the well-studied system of galactose binding to the 120-kDa lectin Ricinus communis agglutinin I (RCA120).  As ligands we used Me β-D-galactoside and a biantennary decasaccharide.  Anal. of the satn. transfer effects of Me β-D-galactoside showed that the H2, H3, and H4 protons are satd. to the highest degree, giving evidence of their close proximity to protons of the RCA120 lectin.  The direct interaction of the lectin with this region of the galactose is in excellent agreement with results obtained from the anal. of the binding specificities of many chem. modified galactose derivs. (Bhattacharyya, L.; Brewer, C. F., 1988).  This new NMR technique can identify the binding epitope of even complex ligands very quickly, which is a great improvement over time-consuming chem. modifications.  Efficient GEM benefits from a relatively high off rate of the ligand and a large excess of the ligand over the receptor.  Even for a ligand like the biantennary decasaccharide with micromolar binding affinity, the binding epitopes could easily be mapped to the terminal β-D-Gal-(1-4)-β-D-GlcNAc (β-D-GlcNAc = N-acetyl-D-glucosamine) residues located at the nonreducing end of the two carbohydrate chains.  The binding contribution of the terminal galactose residue is stronger than those of the penultimate GlcNAc residues.  We could show that the GlcNAc residues bind "edge-on" with the region from H2 to H4, making contact with the protein.  Anal. of STD NMR expts. performed under competitive conditions proved that the two saccharides studied bind at the same receptor site, thereby ruling out unspecific binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9BVgPNWHCsLVg90H21EOLACvtfcHk0lhdjh_Ml9yrTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXktFamsbg%253D&md5=5fe9d7d46114ee1cc96a8a50f8da972c</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1021%2Fja0100120&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja0100120%26sid%3Dliteratum%253Aachs%26aulast%3DMayer%26aufirst%3DM.%26aulast%3DMeyer%26aufirst%3DB.%26atitle%3DGroup%2520epitope%2520mapping%2520by%2520saturation%2520transfer%2520difference%2520NMR%2520to%2520identify%2520segments%2520of%2520a%2520ligand%2520in%2520direct%2520contact%2520with%2520a%2520protein%2520receptor%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2001%26volume%3D123%26spage%3D6108%26epage%3D6117%26doi%3D10.1021%2Fja0100120" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Ludwig, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michiels, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lodi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ride, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunce, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gunther, U. L.</span><span> </span><span class="NLM_article-title">Evaluation of solvent accessibility epitopes for different dehydrogenase inhibitors</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">1371</span><span class="NLM_x">–</span> <span class="NLM_lpage">1376</span><span class="refDoi"> DOI: 10.1002/cmdc.200800110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=10.1002%2Fcmdc.200800110" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2008&pages=1371-1376&author=C.+Ludwigauthor=P.+J.+Michielsauthor=A.+Lodiauthor=J.+Rideauthor=C.+Bunceauthor=U.+L.+Gunther&title=Evaluation+of+solvent+accessibility+epitopes+for+different+dehydrogenase+inhibitors&doi=10.1002%2Fcmdc.200800110"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.200800110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.200800110%26sid%3Dliteratum%253Aachs%26aulast%3DLudwig%26aufirst%3DC.%26aulast%3DMichiels%26aufirst%3DP.%2BJ.%26aulast%3DLodi%26aufirst%3DA.%26aulast%3DRide%26aufirst%3DJ.%26aulast%3DBunce%26aufirst%3DC.%26aulast%3DGunther%26aufirst%3DU.%2BL.%26atitle%3DEvaluation%2520of%2520solvent%2520accessibility%2520epitopes%2520for%2520different%2520dehydrogenase%2520inhibitors%26jtitle%3DChemMedChem%26date%3D2008%26volume%3D3%26spage%3D1371%26epage%3D1376%26doi%3D10.1002%2Fcmdc.200800110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Dalvit, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fogliatto, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stewart, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Veronesi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stockman, B.</span><span> </span><span class="NLM_article-title">WaterLOGSY as a method for primary NMR screening: practical aspects and range of applicability</span> <span class="citation_source-journal">J. Biomol. NMR</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">349</span><span class="NLM_x">–</span> <span class="NLM_lpage">359</span><span class="refDoi"> DOI: 10.1023/A:1013302231549</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=10.1023%2FA%3A1013302231549" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=11824754" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=1%3ACAS%3A528%3ADC%252BD38XptVyksw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2001&pages=349-359&author=C.+Dalvitauthor=G.+Fogliattoauthor=A.+Stewartauthor=M.+Veronesiauthor=B.+Stockman&title=WaterLOGSY+as+a+method+for+primary+NMR+screening%3A+practical+aspects+and+range+of+applicability&doi=10.1023%2FA%3A1013302231549"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">WaterLOGSY as a method for primary NMR screening: practical aspects and range of applicability</span></div><div class="casAuthors">Dalvit, Claudio; Fogliatto, GianPaolo; Stewart, Albert; Veronesi, Marina; Stockman, Brian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular NMR</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">349-359</span>CODEN:
                <span class="NLM_cas:coden">JBNME9</span>;
        ISSN:<span class="NLM_cas:issn">0925-2738</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic Publishers</span>)
        </div><div class="casAbstract">WaterLOGSY represents a powerful method for primary NMR screening in the identification of compds. interacting with macromols., including proteins and DNA or RNA fragments.  Several relay pathways are used constructively in the expt. for transferring bulk water magnetization to the ligand.  The method is particularly useful for the identification of novel scaffolds of micromolar affinity that can be then optimized using directed screening, combinatorial chem., medicinal chem. and structure-based drug design.  The practical aspects and range of applicability of the WaterLOGSY expt. are analyzed in detail here.  Competition binding and titrn. WaterLOGSY permit, after proper correction, the evaluation of the dissocn. binding const.  The high sensitivity of the technique in combination with the easy deconvolution of the mixts. for the identification of the active components, significantly reduces the amt. of material and time needed for the NMR screening process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHqqsXXXVX77Vg90H21EOLACvtfcHk0libbQrhlpBS3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XptVyksw%253D%253D&md5=e7b431d72aa17c499c1e751956605dee</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1023%2FA%3A1013302231549&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1013302231549%26sid%3Dliteratum%253Aachs%26aulast%3DDalvit%26aufirst%3DC.%26aulast%3DFogliatto%26aufirst%3DG.%26aulast%3DStewart%26aufirst%3DA.%26aulast%3DVeronesi%26aufirst%3DM.%26aulast%3DStockman%26aufirst%3DB.%26atitle%3DWaterLOGSY%2520as%2520a%2520method%2520for%2520primary%2520NMR%2520screening%253A%2520practical%2520aspects%2520and%2520range%2520of%2520applicability%26jtitle%3DJ.%2520Biomol.%2520NMR%26date%3D2001%26volume%3D21%26spage%3D349%26epage%3D359%26doi%3D10.1023%2FA%3A1013302231549" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Dalvit, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pevarello, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tatò, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Veronesi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vulpetti, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sundström, M.</span><span> </span><span class="NLM_article-title">Identification of compounds with binding affinity to proteins via magnetization transfer from bulk water</span> <span class="citation_source-journal">J. Biomol. NMR</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">65</span><span class="NLM_x">–</span> <span class="NLM_lpage">68</span><span class="refDoi"> DOI: 10.1023/A:1008354229396</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=10.1023%2FA%3A1008354229396" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=11061229" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=1%3ACAS%3A528%3ADC%252BD3cXnvVGit7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2000&pages=65-68&author=C.+Dalvitauthor=P.+Pevarelloauthor=M.+Tat%C3%B2author=M.+Veronesiauthor=A.+Vulpettiauthor=M.+Sundstr%C3%B6m&title=Identification+of+compounds+with+binding+affinity+to+proteins+via+magnetization+transfer+from+bulk+water&doi=10.1023%2FA%3A1008354229396"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of compounds with binding affinity to proteins via magnetization transfer from bulk water</span></div><div class="casAuthors">Dalvit, Claudio; Pevarello, Paolo; Tato, Marco; Veronesi, Marina; Vulpetti, Anna; Sundstrom, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular NMR</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">65-68</span>CODEN:
                <span class="NLM_cas:coden">JBNME9</span>;
        ISSN:<span class="NLM_cas:issn">0925-2738</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic Publishers</span>)
        </div><div class="casAbstract">A powerful screening by NMR methodol. (WaterLOGSY), based on transfer of magnetization from bulk water, for the identification of compds. that interact with target biomols. (proteins, RNA and DNA fragments) is described.  The method exploits efficiently the large reservoir of H2O magnetization.  The high sensitivity of the technique reduces the amt. of biomol. and ligands needed for the screening, which constitutes an important requirement for high throughput screening by NMR of large libraries of compds.  Application of the method to a compd. mixt. against the cyclin-dependent kinase 2 (cdk2) protein is presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoR8Jo0PHYdALVg90H21EOLACvtfcHk0libbQrhlpBS3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXnvVGit7o%253D&md5=e0fd0e5815f9bd20ed09d8033882feb6</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1023%2FA%3A1008354229396&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1008354229396%26sid%3Dliteratum%253Aachs%26aulast%3DDalvit%26aufirst%3DC.%26aulast%3DPevarello%26aufirst%3DP.%26aulast%3DTat%25C3%25B2%26aufirst%3DM.%26aulast%3DVeronesi%26aufirst%3DM.%26aulast%3DVulpetti%26aufirst%3DA.%26aulast%3DSundstr%25C3%25B6m%26aufirst%3DM.%26atitle%3DIdentification%2520of%2520compounds%2520with%2520binding%2520affinity%2520to%2520proteins%2520via%2520magnetization%2520transfer%2520from%2520bulk%2520water%26jtitle%3DJ.%2520Biomol.%2520NMR%26date%3D2000%26volume%3D18%26spage%3D65%26epage%3D68%26doi%3D10.1023%2FA%3A1008354229396" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Dalvit, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fagerness, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hadden, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sarver, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stockman, B. J.</span><span> </span><span class="NLM_article-title">Fluorine-NMR experiments for high-throughput screening: theoretical aspects, practical considerations, and range of applicability</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">125</span><span class="NLM_x">, </span> <span class="NLM_fpage">7696</span><span class="NLM_x">–</span> <span class="NLM_lpage">7703</span><span class="refDoi"> DOI: 10.1021/ja034646d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja034646d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjvFOitbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2003&pages=7696-7703&author=C.+Dalvitauthor=P.+E.+Fagernessauthor=D.+T.+Haddenauthor=R.+W.+Sarverauthor=B.+J.+Stockman&title=Fluorine-NMR+experiments+for+high-throughput+screening%3A+theoretical+aspects%2C+practical+considerations%2C+and+range+of+applicability&doi=10.1021%2Fja034646d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorine-NMR experiments for high-throughput screening: Theoretical aspects, practical considerations, and range of applicability</span></div><div class="casAuthors">Dalvit, Claudio; Fagerness, Paul E.; Hadden, Daneen T. A.; Sarver, Ronald W.; Stockman, Brian J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">7696-7703</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Competition ligand-based NMR screening expts. have recently been introduced to overcome most of the problems assocd. with traditional ligand-based NMR screening.  Mols. with marginal soly. and high affinity for a given target can be easily identified in a high-throughput manner by screening chem. mixts. against the target in the presence of a weak- to medium-affinity ligand of known binding const.  While the original competition-based approaches utilized 1H detection, significant advantages are obtained using 19F detection.  The absence of spectral overlap permits the screening of large chem. mixts. and allows for automated anal. of the spectra, even in the presence of protonated buffers, solvents, and detergents.  The large chem. shift anisotropy of fluorine and the significant exchange contribution allow for the selection of a weak-affinity spy mol., thus resulting in a lower binding affinity threshold for the identified NMR hits.  The method, labeled FAXS (fluorine chem. shift anisotropy and exchange for screening) is rapid and requires only a limited amt. of protein and, therefore, compares favorably with the other established non-NMR techniques used in high-throughput screening.  Herein the theor. aspects of this powerful 19F-based approach are presented and discussed in detail.  The exptl. conditions together with the detection limits and binding const. measurements are investigated using human serum albumin as the target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrluTqHSItDBLVg90H21EOLACvtfcHk0libbQrhlpBS3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjvFOitbs%253D&md5=58b7a2db41871c28e06873302d4a1ecb</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Fja034646d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja034646d%26sid%3Dliteratum%253Aachs%26aulast%3DDalvit%26aufirst%3DC.%26aulast%3DFagerness%26aufirst%3DP.%2BE.%26aulast%3DHadden%26aufirst%3DD.%2BT.%26aulast%3DSarver%26aufirst%3DR.%2BW.%26aulast%3DStockman%26aufirst%3DB.%2BJ.%26atitle%3DFluorine-NMR%2520experiments%2520for%2520high-throughput%2520screening%253A%2520theoretical%2520aspects%252C%2520practical%2520considerations%252C%2520and%2520range%2520of%2520applicability%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2003%26volume%3D125%26spage%3D7696%26epage%3D7703%26doi%3D10.1021%2Fja034646d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Shuker, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajduk, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meadows, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fesik, S. W.</span><span> </span><span class="NLM_article-title">Discovering high-affinity ligands for proteins: SAR by NMR</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">274</span><span class="NLM_x">, </span> <span class="NLM_fpage">1531</span><span class="NLM_x">–</span> <span class="NLM_lpage">1534</span><span class="refDoi"> DOI: 10.1126/science.274.5292.1531</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=10.1126%2Fscience.274.5292.1531" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=8929414" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=1%3ACAS%3A528%3ADyaK28Xnt1ansLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=274&publication_year=1996&pages=1531-1534&author=S.+B.+Shukerauthor=P.+J.+Hajdukauthor=R.+P.+Meadowsauthor=S.+W.+Fesik&title=Discovering+high-affinity+ligands+for+proteins%3A+SAR+by+NMR&doi=10.1126%2Fscience.274.5292.1531"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Discovering high-affinity ligands for proteins: SAR by NMR</span></div><div class="casAuthors">Shuker, Suzanne B.; Hajduk, Philip J.; Meadows, Robert P.; Fesik, Stephen W.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">274</span>
        (<span class="NLM_cas:issue">5292</span>),
    <span class="NLM_cas:pages">1531-1534</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A NMR-based method is described in which small org. mols. that bind to proximal subsites of a protein are identified, optimized, and linked together to produce high-affinity ligands.  The approach is called "SAR by NMR" because structure-activity relationships (SAR) are obtained from NMR.  With this technique, compds. with nanomolar affinities for the FK506 binding protein were rapidly discovered by tethering two ligands with micromolar affinities.  The method reduces the amt. of chem. synthesis and time required for the discovery of high-affinity ligands and appears particularly useful in target-directed drug research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwkuE4EVU_yrVg90H21EOLACvtfcHk0ljzWxFuhGRt1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xnt1ansLs%253D&md5=1b9a5a5e4f2d6ee4b1c249e3e1faf2ec</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1126%2Fscience.274.5292.1531&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.274.5292.1531%26sid%3Dliteratum%253Aachs%26aulast%3DShuker%26aufirst%3DS.%2BB.%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26aulast%3DMeadows%26aufirst%3DR.%2BP.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26atitle%3DDiscovering%2520high-affinity%2520ligands%2520for%2520proteins%253A%2520SAR%2520by%2520NMR%26jtitle%3DScience%26date%3D1996%26volume%3D274%26spage%3D1531%26epage%3D1534%26doi%3D10.1126%2Fscience.274.5292.1531" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Auer, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kloiber, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vavrinska, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geist, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coudevylle, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Konrat, R.</span><span> </span><span class="NLM_article-title">Pharmacophore mapping via cross-relaxation during adiabatic fast passage</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">132</span><span class="NLM_x">, </span> <span class="NLM_fpage">1480</span><span class="NLM_x">–</span> <span class="NLM_lpage">1481</span><span class="refDoi"> DOI: 10.1021/ja910098s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja910098s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=1480-1481&author=R.+Auerauthor=K.+Kloiberauthor=A.+Vavrinskaauthor=L.+Geistauthor=N.+Coudevylleauthor=R.+Konrat&title=Pharmacophore+mapping+via+cross-relaxation+during+adiabatic+fast+passage&doi=10.1021%2Fja910098s"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fja910098s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja910098s%26sid%3Dliteratum%253Aachs%26aulast%3DAuer%26aufirst%3DR.%26aulast%3DKloiber%26aufirst%3DK.%26aulast%3DVavrinska%26aufirst%3DA.%26aulast%3DGeist%26aufirst%3DL.%26aulast%3DCoudevylle%26aufirst%3DN.%26aulast%3DKonrat%26aufirst%3DR.%26atitle%3DPharmacophore%2520mapping%2520via%2520cross-relaxation%2520during%2520adiabatic%2520fast%2520passage%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26spage%3D1480%26epage%3D1481%26doi%3D10.1021%2Fja910098s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Villicana, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cruz, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zurita, M.</span><span> </span><span class="NLM_article-title">The basal transcription machinery as a target for cancer therapy</span> <span class="citation_source-journal">Cancer Cell Int.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">18</span><span class="refDoi"> DOI: 10.1186/1475-2867-14-18</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=10.1186%2F1475-2867-14-18" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=24576043" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjsVartbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2014&pages=18&author=C.+Villicanaauthor=G.+Cruzauthor=M.+Zurita&title=The+basal+transcription+machinery+as+a+target+for+cancer+therapy&doi=10.1186%2F1475-2867-14-18"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">The basal transcription machinery as a target for cancer therapy</span></div><div class="casAuthors">Villicana, Claudia; Cruz, Grisel; Zurita, Mario</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell International</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">18/1-18/15, 15 pp.</span>CODEN:
                <span class="NLM_cas:coden">CCIACC</span>;
        ISSN:<span class="NLM_cas:issn">1475-2867</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">General transcription is required for the growth and survival of all living cells.  However, tumor cells require extraordinary levels of transcription, including the transcription of rRNA genes by RNA polymerase I (RNPI) and mRNA by RNA polymerase II (RNPII).  In fact, cancer cells have mutations that directly enhance transcription and are frequently required for cancer transformation.  For example, the recent discovery that MYC enhances the transcription of the majority genes in the genome correlates with the fact that several transcription interfering drugs preferentially kill cancer cells.  In recent years, advances in the mechanistic studies of the basal transcription machinery and the discovery of drugs that interfere with multiple components of transcription are being used to combat cancer.  For example, drugs such as triptolide that targets the general transcription factors TFIIH and JQ1 to inhibit BRD4 are administered to target the high proliferative rate of cancer cells.  Given the importance of finding new strategies to preferentially sensitize tumor cells, this review primarily focuses on several transcription inhibitory drugs to demonstrate that the basal transcription machinery constitutes a potential target for the design of novel cancer drugs.  We highlight the drugs' mechanisms for interfering with tumor cell survival, their importance in cancer treatment and the challenges of clin. application.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7aEZQbw_0xLVg90H21EOLACvtfcHk0ljzWxFuhGRt1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjsVartbo%253D&md5=806c757e799d3a318038191c9e1c3520</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1186%2F1475-2867-14-18&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1475-2867-14-18%26sid%3Dliteratum%253Aachs%26aulast%3DVillicana%26aufirst%3DC.%26aulast%3DCruz%26aufirst%3DG.%26aulast%3DZurita%26aufirst%3DM.%26atitle%3DThe%2520basal%2520transcription%2520machinery%2520as%2520a%2520target%2520for%2520cancer%2520therapy%26jtitle%3DCancer%2520Cell%2520Int.%26date%3D2014%26volume%3D14%26spage%3D18%26doi%3D10.1186%2F1475-2867-14-18" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Ludwig, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michiels, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kavanagh, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pilka, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jansson, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oppermann, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gunther, U. L.</span><span> </span><span class="NLM_article-title">SALMON: solvent accessibility, ligand binding, and mapping of ligand orientation by NMR spectroscopy</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">3</span><span class="refDoi"> DOI: 10.1021/jm701020f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm701020f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=1-3&author=C.+Ludwigauthor=P.+J.+Michielsauthor=X.+Wuauthor=K.+L.+Kavanaghauthor=E.+Pilkaauthor=A.+Janssonauthor=U.+Oppermannauthor=U.+L.+Gunther&title=SALMON%3A+solvent+accessibility%2C+ligand+binding%2C+and+mapping+of+ligand+orientation+by+NMR+spectroscopy&doi=10.1021%2Fjm701020f"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fjm701020f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm701020f%26sid%3Dliteratum%253Aachs%26aulast%3DLudwig%26aufirst%3DC.%26aulast%3DMichiels%26aufirst%3DP.%2BJ.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DKavanagh%26aufirst%3DK.%2BL.%26aulast%3DPilka%26aufirst%3DE.%26aulast%3DJansson%26aufirst%3DA.%26aulast%3DOppermann%26aufirst%3DU.%26aulast%3DGunther%26aufirst%3DU.%2BL.%26atitle%3DSALMON%253A%2520solvent%2520accessibility%252C%2520ligand%2520binding%252C%2520and%2520mapping%2520of%2520ligand%2520orientation%2520by%2520NMR%2520spectroscopy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D1%26epage%3D3%26doi%3D10.1021%2Fjm701020f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Mizukoshi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abe, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takizawa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanzawa, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fukunishi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shimada, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takahashi, H.</span><span> </span><span class="NLM_article-title">An accurate pharmacophore mapping method by NMR spectroscopy</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">1362</span><span class="NLM_x">–</span> <span class="NLM_lpage">1365</span><span class="refDoi"> DOI: 10.1002/anie.201104905</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=10.1002%2Fanie.201104905" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=1362-1365&author=Y.+Mizukoshiauthor=A.+Abeauthor=T.+Takizawaauthor=H.+Hanzawaauthor=Y.+Fukunishiauthor=I.+Shimadaauthor=H.+Takahashi&title=An+accurate+pharmacophore+mapping+method+by+NMR+spectroscopy&doi=10.1002%2Fanie.201104905"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1002%2Fanie.201104905&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201104905%26sid%3Dliteratum%253Aachs%26aulast%3DMizukoshi%26aufirst%3DY.%26aulast%3DAbe%26aufirst%3DA.%26aulast%3DTakizawa%26aufirst%3DT.%26aulast%3DHanzawa%26aufirst%3DH.%26aulast%3DFukunishi%26aufirst%3DY.%26aulast%3DShimada%26aufirst%3DI.%26aulast%3DTakahashi%26aufirst%3DH.%26atitle%3DAn%2520accurate%2520pharmacophore%2520mapping%2520method%2520by%2520NMR%2520spectroscopy%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2012%26volume%3D51%26spage%3D1362%26epage%3D1365%26doi%3D10.1002%2Fanie.201104905" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Ni, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, Y.</span><span> </span><span class="NLM_article-title">Accounting for ligand-protein interactions in the relaxation-matrix analysis of transferred nuclear Overhauser effects</span> <span class="citation_source-journal">J. Magn. Reson., Ser. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">103</span><span class="NLM_x">, </span> <span class="NLM_fpage">180</span><span class="NLM_x">–</span> <span class="NLM_lpage">184</span><span class="refDoi"> DOI: 10.1006/jmrb.1994.1027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=10.1006%2Fjmrb.1994.1027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=8137078" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=1%3ACAS%3A528%3ADyaK2cXjtFymtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=1994&pages=180-184&author=F.+Niauthor=Y.+Zhu&title=Accounting+for+ligand-protein+interactions+in+the+relaxation-matrix+analysis+of+transferred+nuclear+Overhauser+effects&doi=10.1006%2Fjmrb.1994.1027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Accounting for ligand-protein interactions in the relaxation-matrix analysis of transferred nuclear Overhauser effects</span></div><div class="casAuthors">Ni, Feng; Zhu, Yi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Magnetic Resonance, Series B</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">180-4</span>CODEN:
                <span class="NLM_cas:coden">JMRBE5</span>;
        ISSN:<span class="NLM_cas:issn">1064-1866</span>.
    </div><div class="casAbstract">Transferred NOE is used to study ligand-protein interactions.  A simplified procedure is presented to account for relaxation effects mediated by specific ligand-protein interactions.  The influence on distance detn. will be discussed using simulation results.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqyfOc2JSPN2LVg90H21EOLACvtfcHk0lg2RVDJfk-1Og"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXjtFymtrc%253D&md5=0587e0be016ac736bcab3cd814c9ef17</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1006%2Fjmrb.1994.1027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmrb.1994.1027%26sid%3Dliteratum%253Aachs%26aulast%3DNi%26aufirst%3DF.%26aulast%3DZhu%26aufirst%3DY.%26atitle%3DAccounting%2520for%2520ligand-protein%2520interactions%2520in%2520the%2520relaxation-matrix%2520analysis%2520of%2520transferred%2520nuclear%2520Overhauser%2520effects%26jtitle%3DJ.%2520Magn.%2520Reson.%252C%2520Ser.%2520B%26date%3D1994%26volume%3D103%26spage%3D180%26epage%3D184%26doi%3D10.1006%2Fjmrb.1994.1027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Balaram, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bothner-By, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dadok, J.</span><span> </span><span class="NLM_article-title">Negative nuclear Overhauser effects as probes of macromolecular structure</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1972</span><span class="NLM_x">, </span> <span class="NLM_volume">94</span><span class="NLM_x">, </span> <span class="NLM_fpage">4015</span><span class="NLM_x">–</span> <span class="NLM_lpage">4017</span><span class="refDoi"> DOI: 10.1021/ja00766a063</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00766a063" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=1972&pages=4015-4017&author=P.+Balaramauthor=A.+A.+Bothner-Byauthor=J.+Dadok&title=Negative+nuclear+Overhauser+effects+as+probes+of+macromolecular+structure&doi=10.1021%2Fja00766a063"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fja00766a063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00766a063%26sid%3Dliteratum%253Aachs%26aulast%3DBalaram%26aufirst%3DP.%26aulast%3DBothner-By%26aufirst%3DA.%2BA.%26aulast%3DDadok%26aufirst%3DJ.%26atitle%3DNegative%2520nuclear%2520Overhauser%2520effects%2520as%2520probes%2520of%2520macromolecular%2520structure%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1972%26volume%3D94%26spage%3D4015%26epage%3D4017%26doi%3D10.1021%2Fja00766a063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Zeng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, M. M.</span><span> </span><span class="NLM_article-title">Bromodomain: an acetyl-lysine binding domain</span> <span class="citation_source-journal">FEBS Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">513</span><span class="NLM_x">, </span> <span class="NLM_fpage">124</span><span class="NLM_x">–</span> <span class="NLM_lpage">128</span><span class="refDoi"> DOI: 10.1016/S0014-5793(01)03309-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=10.1016%2FS0014-5793%2801%2903309-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=11911891" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=1%3ACAS%3A528%3ADC%252BD38XitFejtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=513&publication_year=2002&pages=124-128&author=L.+Zengauthor=M.+M.+Zhou&title=Bromodomain%3A+an+acetyl-lysine+binding+domain&doi=10.1016%2FS0014-5793%2801%2903309-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomain: an acetyl-lysine binding domain</span></div><div class="casAuthors">Zeng, Lei; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">513</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">124-128</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">A review with 43 refs.  Bromodomains, an extensive family of evolutionarily conserved protein modules originally found in proteins assocd. with chromatin and in nearly all nuclear histone acetyltransferases, have been recently discovered to function as acetyllysine-binding domains.  More recent structural studies of bromodomain/peptide ligand complexes have enriched the understanding of differences in ligand selectivity of bromodomains.  These new findings demonstrate that bromodomain/acetyllysine recognition can serve as a pivotal mechanism for regulating protein-protein interactions in numerous cellular processes including chromatin remodeling and transcriptional activation, and reinforce the concept that functional diversity of a conserved protein modular structure is achieved by evolutionary changes of amino acid sequences in the ligand binding site.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMESL8iZ9agLVg90H21EOLACvtfcHk0lg2RVDJfk-1Og"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XitFejtbo%253D&md5=79070a63cd171f846ab577aa9f7092bb</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2FS0014-5793%2801%2903309-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0014-5793%252801%252903309-9%26sid%3Dliteratum%253Aachs%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26atitle%3DBromodomain%253A%2520an%2520acetyl-lysine%2520binding%2520domain%26jtitle%3DFEBS%2520Lett.%26date%3D2002%26volume%3D513%26spage%3D124%26epage%3D128%26doi%3D10.1016%2FS0014-5793%2801%2903309-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Wu, S. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiang, C. M.</span><span> </span><span class="NLM_article-title">The double bromodomain-containing chromatin adaptor Brd4 and transcriptional regulation</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">282</span><span class="NLM_x">, </span> <span class="NLM_fpage">13141</span><span class="NLM_x">–</span> <span class="NLM_lpage">13145</span><span class="refDoi"> DOI: 10.1074/jbc.R700001200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=10.1074%2Fjbc.R700001200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=17329240" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=1%3ACAS%3A528%3ADC%252BD2sXks1Ghtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=2007&pages=13141-13145&author=S.+Y.+Wuauthor=C.+M.+Chiang&title=The+double+bromodomain-containing+chromatin+adaptor+Brd4+and+transcriptional+regulation&doi=10.1074%2Fjbc.R700001200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">The Double Bromodomain-containing Chromatin Adaptor Brd4 and Transcriptional Regulation</span></div><div class="casAuthors">Wu, Shwu-Yuan; Chiang, Cheng-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">282</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">13141-13145</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">A review.  Brd4 is a double bromodomain-contg. protein that binds preferentially to acetylated chromatin.  It belongs to the BET (bromodomains and extraterminal) family that includes mammalian Brd2, Brd3, Brd4, Brdt, Drosophila Fsh, yeast Bdf1, Bdf2, and corresponding homologues in other species.  Brd4 is essential for cellular growth and has been implicated in cell cycle control, DNA replication, and gene rearrangement found in t(15;19)-assocd. carcinomas.  Recently, Brd4 has been found in several transcription complexes, including the general cofactor Mediator and the P-TEFb elongation factor, and is capable of stimulating HIV-1 transcription in a Tat-independent manner.  In addn., Brd4 is used as a cellular adaptor by some animal and human papillomaviruses (HPV) for anchoring viral genomes to mitotic chromosomes.  This tethering, mediated by Brd4 interaction with virus-encoded E2 protein, facilitates viral genome segregation during mitosis.  Interestingly, Brd4 is also identified in a transcriptional silencing complex assembled by HPV E2 and turns out to be the long sought cellular corepressor that inhibits the expression of HPV-encoded E6 and E7 oncoproteins that antagonize p53 and pRB tumor suppressor activity, resp.  The dual role of Brd4 in gene activation and repression illustrates how a dynamic chromatin-binding adaptor is able to recruit distinct transcriptional regulators to modulate promoter activity through cell cycle progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohccDdZC9iKLVg90H21EOLACvtfcHk0lg2RVDJfk-1Og"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXks1Ghtbo%253D&md5=b025a5222b510a7009eca4289370b1d4</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1074%2Fjbc.R700001200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.R700001200%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DS.%2BY.%26aulast%3DChiang%26aufirst%3DC.%2BM.%26atitle%3DThe%2520double%2520bromodomain-containing%2520chromatin%2520adaptor%2520Brd4%2520and%2520transcriptional%2520regulation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2007%26volume%3D282%26spage%3D13141%26epage%3D13145%26doi%3D10.1074%2Fjbc.R700001200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Delmore, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Issa, G. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lemieux, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rahl, P. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobs, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kastritis, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilpatrick, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paranal, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chesi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schinzel, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKeown, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heffernan, T. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vakoc, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bergsagel, P. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghobrial, I. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, P. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hahn, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anderson, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kung, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitsiades, C. S.</span><span> </span><span class="NLM_article-title">BET bromodomain inhibition as a therapeutic strategy to target c-Myc</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">146</span><span class="NLM_x">, </span> <span class="NLM_fpage">904</span><span class="NLM_x">–</span> <span class="NLM_lpage">917</span><span class="refDoi"> DOI: 10.1016/j.cell.2011.08.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=10.1016%2Fj.cell.2011.08.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=21889194" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtF2ht7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=146&publication_year=2011&pages=904-917&author=J.+E.+Delmoreauthor=G.+C.+Issaauthor=M.+E.+Lemieuxauthor=P.+B.+Rahlauthor=J.+Shiauthor=H.+M.+Jacobsauthor=E.+Kastritisauthor=T.+Gilpatrickauthor=R.+M.+Paranalauthor=J.+Qiauthor=M.+Chesiauthor=A.+C.+Schinzelauthor=M.+R.+McKeownauthor=T.+P.+Heffernanauthor=C.+R.+Vakocauthor=P.+L.+Bergsagelauthor=I.+M.+Ghobrialauthor=P.+G.+Richardsonauthor=R.+A.+Youngauthor=W.+C.+Hahnauthor=K.+C.+Andersonauthor=A.+L.+Kungauthor=J.+E.+Bradnerauthor=C.+S.+Mitsiades&title=BET+bromodomain+inhibition+as+a+therapeutic+strategy+to+target+c-Myc&doi=10.1016%2Fj.cell.2011.08.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">BET Bromodomain Inhibition as a Therapeutic Strategy to Target c-Myc</span></div><div class="casAuthors">Delmore, Jake E.; Issa, Ghayas C.; Lemieux, Madeleine E.; Rahl, Peter B.; Shi, Jun-Wei; Jacobs, Hannah M.; Kastritis, Efstathios; Gilpatrick, Timothy; Paranal, Ronald M.; Qi, Jun; Chesi, Marta; Schinzel, Anna C.; McKeown, Michael R.; Heffernan, Timothy P.; Vakoc, Christopher R.; Bergsagel, P. Leif; Ghobrial, Irene M.; Richardson, Paul G.; Young, Richard A.; Hahn, William C.; Anderson, Kenneth C.; Kung, Andrew L.; Bradner, James E.; Mitsiades, Constantine S.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">146</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">904-917</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: MYC contributes to the pathogenesis of a majority of human cancers, yet strategies to modulate the function of the c-Myc oncoprotein do not exist.  Toward this objective, we have targeted MYC transcription by interfering with chromatin-dependent signal transduction to RNA polymerase, specifically by inhibiting the acetyl-lysine recognition domains (bromodomains) of putative coactivator proteins implicated in transcriptional initiation and elongation.  Using a selective small-mol. bromodomain inhibitor, JQ1, we identify BET bromodomain proteins as regulatory factors for c-Myc.  BET inhibition by JQ1 downregulates MYC transcription, followed by genome-wide downregulation of Myc-dependent target genes.  In exptl. models of multiple myeloma, a Myc-dependent hematol. malignancy, JQ1 produces a potent antiproliferative effect assocd. with cell-cycle arrest and cellular senescence.  Efficacy of JQ1 in three murine models of multiple myeloma establishes the therapeutic rationale for BET bromodomain inhibition in this disease and other malignancies characterized by pathol. activation of c-Myc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpevD1QHKmOhLVg90H21EOLACvtfcHk0lhKCwQ6IDzOkg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtF2ht7fF&md5=013674b8224214a9a1c8f0b75583a5a6</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2011.08.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2011.08.017%26sid%3Dliteratum%253Aachs%26aulast%3DDelmore%26aufirst%3DJ.%2BE.%26aulast%3DIssa%26aufirst%3DG.%2BC.%26aulast%3DLemieux%26aufirst%3DM.%2BE.%26aulast%3DRahl%26aufirst%3DP.%2BB.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DJacobs%26aufirst%3DH.%2BM.%26aulast%3DKastritis%26aufirst%3DE.%26aulast%3DGilpatrick%26aufirst%3DT.%26aulast%3DParanal%26aufirst%3DR.%2BM.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DChesi%26aufirst%3DM.%26aulast%3DSchinzel%26aufirst%3DA.%2BC.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DHeffernan%26aufirst%3DT.%2BP.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26aulast%3DBergsagel%26aufirst%3DP.%2BL.%26aulast%3DGhobrial%26aufirst%3DI.%2BM.%26aulast%3DRichardson%26aufirst%3DP.%2BG.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26aulast%3DHahn%26aufirst%3DW.%2BC.%26aulast%3DAnderson%26aufirst%3DK.%2BC.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DMitsiades%26aufirst%3DC.%2BS.%26atitle%3DBET%2520bromodomain%2520inhibition%2520as%2520a%2520therapeutic%2520strategy%2520to%2520target%2520c-Myc%26jtitle%3DCell%26date%3D2011%26volume%3D146%26spage%3D904%26epage%3D917%26doi%3D10.1016%2Fj.cell.2011.08.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Zuber, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rappaport, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herrmann, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sison, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Magoon, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blatt, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wunderlich, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johns, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chicas, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mulloy, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kogan, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valent, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lowe, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vakoc, C. R.</span><span> </span><span class="NLM_article-title">RNAi screen identifies Brd4 as a therapeutic target in acute myeloid leukaemia</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">478</span><span class="NLM_x">, </span> <span class="NLM_fpage">524</span><span class="NLM_x">–</span> <span class="NLM_lpage">528</span><span class="refDoi"> DOI: 10.1038/nature10334</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=10.1038%2Fnature10334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=21814200" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFers7%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=478&publication_year=2011&pages=524-528&author=J.+Zuberauthor=J.+Shiauthor=E.+Wangauthor=A.+R.+Rappaportauthor=H.+Herrmannauthor=E.+A.+Sisonauthor=D.+Magoonauthor=J.+Qiauthor=K.+Blattauthor=M.+Wunderlichauthor=M.+J.+Taylorauthor=C.+Johnsauthor=A.+Chicasauthor=J.+C.+Mulloyauthor=S.+C.+Koganauthor=P.+Brownauthor=P.+Valentauthor=J.+E.+Bradnerauthor=S.+W.+Loweauthor=C.+R.+Vakoc&title=RNAi+screen+identifies+Brd4+as+a+therapeutic+target+in+acute+myeloid+leukaemia&doi=10.1038%2Fnature10334"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">RNAi screen identifies Brd4 as a therapeutic target in acute myeloid leukaemia</span></div><div class="casAuthors">Zuber, Johannes; Shi, Junwei; Wang, Eric; Rappaport, Amy R.; Herrmann, Harald; Sison, Edward A.; Magoon, Daniel; Qi, Jun; Blatt, Katharina; Wunderlich, Mark; Taylor, Meredith J.; Johns, Christopher; Chicas, Agustin; Mulloy, James C.; Kogan, Scott C.; Brown, Patrick; Valent, Peter; Bradner, James E.; Lowe, Scott W.; Vakoc, Christopher R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">478</span>
        (<span class="NLM_cas:issue">7370</span>),
    <span class="NLM_cas:pages">524-528</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic pathways can regulate gene expression by controlling and interpreting chromatin modifications.  Cancer cells are characterized by altered epigenetic landscapes, and commonly exploit the chromatin regulatory machinery to enforce oncogenic gene expression programs.  Although chromatin alterations are, in principle, reversible and often amenable to drug intervention, the promise of targeting such pathways therapeutically has been limited by an incomplete understanding of cancer-specific dependencies on epigenetic regulators.  Here the authors describe a non-biased approach to probe epigenetic vulnerabilities in acute myeloid leukemia (AML), an aggressive hematopoietic malignancy that is often assocd. with aberrant chromatin states.  By screening a custom library of small hairpin RNAs (shRNAs) targeting known chromatin regulators in a genetically defined AML mouse model, the authors identify the protein bromodomain-contg. 4 (Brd4) as being critically required for disease maintenance.  Suppression of Brd4 using shRNAs or the small-mol. inhibitor JQ1 led to robust antileukemic effects in vitro and in vivo, accompanied by terminal myeloid differentiation and elimination of leukemia stem cells.  Similar sensitivities were obsd. in a variety of human AML cell lines and primary patient samples, revealing that JQ1 has broad activity in diverse AML subtypes.  The effects of Brd4 suppression are, at least in part, due to its role in sustaining Myc expression to promote aberrant self-renewal, which implicates JQ1 as a pharmacol. means to suppress MYC in cancer.  The authors' results establish small-mol. inhibition of Brd4 as a promising therapeutic strategy in AML and, potentially, other cancers, and highlight the utility of RNA interference (RNAi) screening for revealing epigenetic vulnerabilities that can be exploited for direct pharmacol. intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVyWFIqFkUVrVg90H21EOLACvtfcHk0lhKCwQ6IDzOkg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFers7%252FP&md5=618793a766f037bff5909524743fedb7</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnature10334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10334%26sid%3Dliteratum%253Aachs%26aulast%3DZuber%26aufirst%3DJ.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DE.%26aulast%3DRappaport%26aufirst%3DA.%2BR.%26aulast%3DHerrmann%26aufirst%3DH.%26aulast%3DSison%26aufirst%3DE.%2BA.%26aulast%3DMagoon%26aufirst%3DD.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DBlatt%26aufirst%3DK.%26aulast%3DWunderlich%26aufirst%3DM.%26aulast%3DTaylor%26aufirst%3DM.%2BJ.%26aulast%3DJohns%26aufirst%3DC.%26aulast%3DChicas%26aufirst%3DA.%26aulast%3DMulloy%26aufirst%3DJ.%2BC.%26aulast%3DKogan%26aufirst%3DS.%2BC.%26aulast%3DBrown%26aufirst%3DP.%26aulast%3DValent%26aufirst%3DP.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DLowe%26aufirst%3DS.%2BW.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26atitle%3DRNAi%2520screen%2520identifies%2520Brd4%2520as%2520a%2520therapeutic%2520target%2520in%2520acute%2520myeloid%2520leukaemia%26jtitle%3DNature%26date%3D2011%26volume%3D478%26spage%3D524%26epage%3D528%26doi%3D10.1038%2Fnature10334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Lockwood, W. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zejnullahu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varmus, H.</span><span> </span><span class="NLM_article-title">Sensitivity of human lung adenocarcinoma cell lines to targeted inhibition of BET epigenetic signaling proteins</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">109</span><span class="NLM_x">, </span> <span class="NLM_fpage">19408</span><span class="NLM_x">–</span> <span class="NLM_lpage">19413</span><span class="refDoi"> DOI: 10.1073/pnas.1216363109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=10.1073%2Fpnas.1216363109" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=19408-19413&author=W.+W.+Lockwoodauthor=K.+Zejnullahuauthor=J.+E.+Bradnerauthor=H.+Varmus&title=Sensitivity+of+human+lung+adenocarcinoma+cell+lines+to+targeted+inhibition+of+BET+epigenetic+signaling+proteins&doi=10.1073%2Fpnas.1216363109"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1216363109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1216363109%26sid%3Dliteratum%253Aachs%26aulast%3DLockwood%26aufirst%3DW.%2BW.%26aulast%3DZejnullahu%26aufirst%3DK.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DVarmus%26aufirst%3DH.%26atitle%3DSensitivity%2520of%2520human%2520lung%2520adenocarcinoma%2520cell%2520lines%2520to%2520targeted%2520inhibition%2520of%2520BET%2520epigenetic%2520signaling%2520proteins%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26spage%3D19408%26epage%3D19413%26doi%3D10.1073%2Fpnas.1216363109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Halle, B.</span><span> </span><span class="NLM_article-title">Protein hydration dynamics in solution: a critical survey</span> <span class="citation_source-journal">Philos. Trans. R. Soc., B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">359</span><span class="NLM_x">, </span> <span class="NLM_fpage">1207</span><span class="NLM_x">–</span> <span class="NLM_lpage">1223</span><div class="note"><p class="first last">(discussion 1223–1224, 1323–1328)</p></div><span class="refDoi"> DOI: 10.1098/rstb.2004.1499</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=10.1098%2Frstb.2004.1499" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=15306377" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=1%3ACAS%3A528%3ADC%252BD2cXnvFWhsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=359&publication_year=2004&pages=1207-1223&author=B.+Halle&title=Protein+hydration+dynamics+in+solution%3A+a+critical+survey&doi=10.1098%2Frstb.2004.1499"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Protein hydration dynamics in solution: a critical survey</span></div><div class="casAuthors">Halle, Bertil</div><div class="citationInfo"><span class="NLM_cas:title">Philosophical Transactions of the Royal Society of London, Series B: Biological Sciences</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">359</span>
        (<span class="NLM_cas:issue">1448</span>),
    <span class="NLM_cas:pages">1207-1224</span>CODEN:
                <span class="NLM_cas:coden">PTRBAE</span>;
        ISSN:<span class="NLM_cas:issn">0962-8436</span>.
    
            (<span class="NLM_cas:orgname">Royal Society</span>)
        </div><div class="casAbstract">A review and discussion.  The properties of water in biol. systems have been studied for well over a century by a wide range of phys. techniques, but progress has been slow and erratic.  Protein hydration, the perturbation of water structure and dynamics by the protein surface, has been a particularly rich source of controversy and confusion.  Here, the authors' aim has been to critically examine central concepts in the description of protein hydration, and to assess the exptl. basis for the current view of protein hydration, with the focus on dynamic aspects.  Recent 17O magnetic relaxation dispersion (MRD) expts. have shown that the vast majority of water mols. in the protein hydration layer suffer a mere 2-fold dynamic retardation compared with bulk water.  The high mobility of hydration water ensures that all thermally activated processes at the protein-water interface, such as binding, recognition, and catalysis, can proceed at high rates.  The MRD-derived picture of a highly mobile hydration layer is consistent with recent mol. dynamics simulations, but is incompatible with results deduced from intermol. NOE spectroscopy, dielec. relaxation, and fluorescence spectroscopy.  It is also inconsistent with the common view of hydration effects on protein hydrodynamics.  Here, the authors show how these discrepancies can be resolved.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptKKWGehwlY7Vg90H21EOLACvtfcHk0lhOb3w16jcTUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXnvFWhsbc%253D&md5=34ed99bb9fe54b2fc6264ed232bd966c</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1098%2Frstb.2004.1499&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1098%252Frstb.2004.1499%26sid%3Dliteratum%253Aachs%26aulast%3DHalle%26aufirst%3DB.%26atitle%3DProtein%2520hydration%2520dynamics%2520in%2520solution%253A%2520a%2520critical%2520survey%26jtitle%3DPhilos.%2520Trans.%2520R.%2520Soc.%252C%2520B%26date%3D2004%26volume%3D359%26spage%3D1207%26epage%3D1223%26doi%3D10.1098%2Frstb.2004.1499" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Ernst, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clubb, R. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, H. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gronenborn, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clore, G. M.</span><span> </span><span class="NLM_article-title">Demonstration of positionally disordered water within a protein hydrophobic cavity by NMR</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">267</span><span class="NLM_x">, </span> <span class="NLM_fpage">1813</span><span class="NLM_x">–</span> <span class="NLM_lpage">1817</span><span class="refDoi"> DOI: 10.1126/science.7892604</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=10.1126%2Fscience.7892604" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=7892604" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=1%3ACAS%3A528%3ADyaK2MXkslWgtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=267&publication_year=1995&pages=1813-1817&author=J.+A.+Ernstauthor=R.+T.+Clubbauthor=H.+X.+Zhouauthor=A.+M.+Gronenbornauthor=G.+M.+Clore&title=Demonstration+of+positionally+disordered+water+within+a+protein+hydrophobic+cavity+by+NMR&doi=10.1126%2Fscience.7892604"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Demonstration of positionally disordered water within a protein hydrophobic cavity by NMR</span></div><div class="casAuthors">Ernst, James A.; Clubb, Robert T.; Zhou, Huan-Xiang; Gronenborn, Angela M.; Clore, G. Marius</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">267</span>
        (<span class="NLM_cas:issue">5205</span>),
    <span class="NLM_cas:pages">1813-17</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The presence and location of water of hydration (i.e., bound water) in the soln. structure of human interleukin-1β (hIL-1β) was investigated with water-selective 2-dimensional heteronuclear magnetic resonance spectroscopy.  It is shown here that in addn. to water at the surface of the protein and ordered internal water mols. involved in bridging hydrogen bonds, positionally disordered water is present within a large, naturally occurring hydrophobic cavity located at the center of the mol.  These water mols. of hydration have residency times in the range of 1-2 ns to 100-200 μs and can be readily detected by NMR.  Thus, large hydrophobic cavities in proteins may not be truly empty, as anal. of crystal structures appears to show, but may contain mobile water mols. that are crystallog. invisible but detectable by NMR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfOkMo6iFnWrVg90H21EOLACvtfcHk0lhOb3w16jcTUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXkslWgtLc%253D&md5=0877f8e2068584119b3365d12952ffec</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1126%2Fscience.7892604&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.7892604%26sid%3Dliteratum%253Aachs%26aulast%3DErnst%26aufirst%3DJ.%2BA.%26aulast%3DClubb%26aufirst%3DR.%2BT.%26aulast%3DZhou%26aufirst%3DH.%2BX.%26aulast%3DGronenborn%26aufirst%3DA.%2BM.%26aulast%3DClore%26aufirst%3DG.%2BM.%26atitle%3DDemonstration%2520of%2520positionally%2520disordered%2520water%2520within%2520a%2520protein%2520hydrophobic%2520cavity%2520by%2520NMR%26jtitle%3DScience%26date%3D1995%26volume%3D267%26spage%3D1813%26epage%3D1817%26doi%3D10.1126%2Fscience.7892604" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Otting, G.</span><span> </span><span class="NLM_article-title">NMR studies of water bound to biological molecules</span> <span class="citation_source-journal">Prog. Nucl. Magn. Reson. Spectrosc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">259</span><span class="NLM_x">–</span> <span class="NLM_lpage">285</span><span class="refDoi"> DOI: 10.1016/S0079-6565(97)00012-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=10.1016%2FS0079-6565%2897%2900012-5" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=1997&pages=259-285&author=G.+Otting&title=NMR+studies+of+water+bound+to+biological+molecules&doi=10.1016%2FS0079-6565%2897%2900012-5"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2FS0079-6565%2897%2900012-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0079-6565%252897%252900012-5%26sid%3Dliteratum%253Aachs%26aulast%3DOtting%26aufirst%3DG.%26atitle%3DNMR%2520studies%2520of%2520water%2520bound%2520to%2520biological%2520molecules%26jtitle%3DProg.%2520Nucl.%2520Magn.%2520Reson.%2520Spectrosc.%26date%3D1997%26volume%3D31%26spage%3D259%26epage%3D285%26doi%3D10.1016%2FS0079-6565%2897%2900012-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Grzesiek, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bax, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicholson, L. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamazaki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wingfield, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stahl, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eyermann, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Torchia, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hodge, C. N.</span><span> </span><span class="NLM_article-title">NMR evidence for the displacement of a conserved interior water molecule in HIV protease by a non-peptide cyclic urea-based inhibitor</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">116</span><span class="NLM_x">, </span> <span class="NLM_fpage">1581</span><span class="NLM_x">–</span> <span class="NLM_lpage">1582</span><span class="refDoi"> DOI: 10.1021/ja00083a058</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00083a058" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=1994&pages=1581-1582&author=S.+Grzesiekauthor=A.+Baxauthor=L.+K.+Nicholsonauthor=T.+Yamazakiauthor=P.+Wingfieldauthor=S.+J.+Stahlauthor=C.+J.+Eyermannauthor=D.+A.+Torchiaauthor=C.+N.+Hodge&title=NMR+evidence+for+the+displacement+of+a+conserved+interior+water+molecule+in+HIV+protease+by+a+non-peptide+cyclic+urea-based+inhibitor&doi=10.1021%2Fja00083a058"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fja00083a058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00083a058%26sid%3Dliteratum%253Aachs%26aulast%3DGrzesiek%26aufirst%3DS.%26aulast%3DBax%26aufirst%3DA.%26aulast%3DNicholson%26aufirst%3DL.%2BK.%26aulast%3DYamazaki%26aufirst%3DT.%26aulast%3DWingfield%26aufirst%3DP.%26aulast%3DStahl%26aufirst%3DS.%2BJ.%26aulast%3DEyermann%26aufirst%3DC.%2BJ.%26aulast%3DTorchia%26aufirst%3DD.%2BA.%26aulast%3DHodge%26aufirst%3DC.%2BN.%26atitle%3DNMR%2520evidence%2520for%2520the%2520displacement%2520of%2520a%2520conserved%2520interior%2520water%2520molecule%2520in%2520HIV%2520protease%2520by%2520a%2520non-peptide%2520cyclic%2520urea-based%2520inhibitor%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1994%26volume%3D116%26spage%3D1581%26epage%3D1582%26doi%3D10.1021%2Fja00083a058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Marley, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bracken, C.</span><span> </span><span class="NLM_article-title">A method for efficient isotopic labeling of recombinant proteins</span> <span class="citation_source-journal">J. Biomol. NMR</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">71</span><span class="NLM_x">–</span> <span class="NLM_lpage">75</span><span class="refDoi"> DOI: 10.1023/A:1011254402785</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=10.1023%2FA%3A1011254402785" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=11430757" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=1%3ACAS%3A528%3ADC%252BD3MXksFCitL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2001&pages=71-75&author=J.+Marleyauthor=M.+Luauthor=C.+Bracken&title=A+method+for+efficient+isotopic+labeling+of+recombinant+proteins&doi=10.1023%2FA%3A1011254402785"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">A method for efficient isotopic labeling of recombinant proteins</span></div><div class="casAuthors">Marley, Jonathan; Lu, Min; Bracken, Clay</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular NMR</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">71-75</span>CODEN:
                <span class="NLM_cas:coden">JBNME9</span>;
        ISSN:<span class="NLM_cas:issn">0925-2738</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic Publishers</span>)
        </div><div class="casAbstract">A rapid and efficient approach for prepg. isotopically labeled recombinant proteins is presented.  The method is demonstrated for 13C labeling of the C-terminal domain of angiopoietin-2, 15N labeling of ubiquitin and for 2H/13C/15N labeling of the Escherichia coli outer-membrane lipoprotein Lpp-56.  The prodn. method generates cell mass using unlabeled rich media followed by exchange into a small vol. of labeled media at high cell d.  Following a short period for growth recovery and unlabeled metabolite clearance, the cells are induced.  The expression yields obtained provide a fourfold to eightfold redn. in isotope costs using simple shake flask growths.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoF39kyXJyC6rVg90H21EOLACvtfcHk0lhermFb4pFeNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXksFCitL0%253D&md5=db4495b09a80894194d3fdf8c7137073</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1023%2FA%3A1011254402785&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1011254402785%26sid%3Dliteratum%253Aachs%26aulast%3DMarley%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DM.%26aulast%3DBracken%26aufirst%3DC.%26atitle%3DA%2520method%2520for%2520efficient%2520isotopic%2520labeling%2520of%2520recombinant%2520proteins%26jtitle%3DJ.%2520Biomol.%2520NMR%26date%3D2001%26volume%3D20%26spage%3D71%26epage%3D75%26doi%3D10.1023%2FA%3A1011254402785" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Piotto, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saudek, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sklenář, V.</span><span> </span><span class="NLM_article-title">Gradient-tailored excitation for single-quantum NMR spectroscopy of aqueous solutions</span> <span class="citation_source-journal">J. Biomol. NMR</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1992</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">661</span><span class="NLM_x">–</span> <span class="NLM_lpage">665</span><span class="refDoi"> DOI: 10.1007/BF02192855</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=10.1007%2FBF02192855" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=1490109" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=1%3ACAS%3A528%3ADyaK3sXitVyktrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=1992&pages=661-665&author=M.+Piottoauthor=V.+Saudekauthor=V.+Sklen%C3%A1%C5%99&title=Gradient-tailored+excitation+for+single-quantum+NMR+spectroscopy+of+aqueous+solutions&doi=10.1007%2FBF02192855"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Gradient-tailored excitation for single-quantum NMR spectroscopy of aqueous solutions</span></div><div class="casAuthors">Piotto, Martial; Saudek, Valdimir; Sklenar, Vladimir</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular NMR</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">661-5</span>CODEN:
                <span class="NLM_cas:coden">JBNME9</span>;
        ISSN:<span class="NLM_cas:issn">0925-2738</span>.
    </div><div class="casAbstract">A novel approach to tailored selective excitation for the measurement of NMR spectra in non-deuterated aq. solns. (WATERGATE, WATER suppression by GrAdient-Tailored Excitation) is described.  The gradient echo sequence, which effectively combines one selective 180° radiofrequency pulse and two field gradient pulses, achieves highly selective and effective water suppression.  This technique is ideally suited for the rapid collection of multi-dimensional data since a single-scan acquisition produces a pure phase NMR spectrum with a perfectly flat baseline, at the highest possible sensitivity.  Application to the fast measurement of 2D NOE data of a 2.2. mM soln. of a double-stranded DNA fragment in 90% H2O at 5° is presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXTdL0-J714LVg90H21EOLACvtfcHk0lhermFb4pFeNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXitVyktrY%253D&md5=895aeeaffe2a1d9019763b39fdd0e0ef</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1007%2FBF02192855&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF02192855%26sid%3Dliteratum%253Aachs%26aulast%3DPiotto%26aufirst%3DM.%26aulast%3DSaudek%26aufirst%3DV.%26aulast%3DSklen%25C3%25A1%25C5%2599%26aufirst%3DV.%26atitle%3DGradient-tailored%2520excitation%2520for%2520single-quantum%2520NMR%2520spectroscopy%2520of%2520aqueous%2520solutions%26jtitle%3DJ.%2520Biomol.%2520NMR%26date%3D1992%26volume%3D2%26spage%3D661%26epage%3D665%26doi%3D10.1007%2FBF02192855" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Dalvit, C.</span><span> </span><span class="NLM_article-title">Efficient multiple-solvent suppression for the study of the interactions of organic solvents with biomolecules</span> <span class="citation_source-journal">J. Biomol. NMR</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">437</span><span class="NLM_x">–</span> <span class="NLM_lpage">444</span><span class="refDoi"> DOI: 10.1023/A:1008272928075</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=10.1023%2FA%3A1008272928075" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=1%3ACAS%3A528%3ADyaK1cXkslSjsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=1998&pages=437-444&author=C.+Dalvit&title=Efficient+multiple-solvent+suppression+for+the+study+of+the+interactions+of+organic+solvents+with+biomolecules&doi=10.1023%2FA%3A1008272928075"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Efficient multiple-solvent suppression for the study of the interactions of organic solvents with biomolecules</span></div><div class="casAuthors">Dalvit, Claudio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular NMR</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">437-444</span>CODEN:
                <span class="NLM_cas:coden">JBNME9</span>;
        ISSN:<span class="NLM_cas:issn">0925-2738</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic Publishers</span>)
        </div><div class="casAbstract">A modification of the excitation sculpting sequence for achieving efficient multiple-solvents suppression in 1D and 2D 1H NMR expts. is presented.  Implementation of this scheme in the ePHOGSY expt. allows rapid and sensitive detection of weak interactions between org. solvents or small mols. and biomols.  Application of the techniques to the peptide Sandostatin dissolved in H2O and DMSO is demonstrated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoiPrbdEy7HtLVg90H21EOLACvtfcHk0lhermFb4pFeNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXkslSjsb4%253D&md5=25848c35395da80d52d29cd1a941f804</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1023%2FA%3A1008272928075&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1008272928075%26sid%3Dliteratum%253Aachs%26aulast%3DDalvit%26aufirst%3DC.%26atitle%3DEfficient%2520multiple-solvent%2520suppression%2520for%2520the%2520study%2520of%2520the%2520interactions%2520of%2520organic%2520solvents%2520with%2520biomolecules%26jtitle%3DJ.%2520Biomol.%2520NMR%26date%3D1998%26volume%3D11%26spage%3D437%26epage%3D444%26doi%3D10.1023%2FA%3A1008272928075" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Geen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freeman, R.</span><span> </span><span class="NLM_article-title">Band-selective radiofrequency pulses</span> <span class="citation_source-journal">J. Magn. Reson. (1969-1992)</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1991</span><span class="NLM_x">, </span> <span class="NLM_volume">93</span><span class="NLM_x">, </span> <span class="NLM_fpage">93</span><span class="NLM_x">–</span> <span class="NLM_lpage">141</span><span class="refDoi"> DOI: 10.1016/0022-2364(91)90034-Q</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=10.1016%2F0022-2364%2891%2990034-Q" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=1991&pages=93-141&author=H.+Geenauthor=R.+Freeman&title=Band-selective+radiofrequency+pulses&doi=10.1016%2F0022-2364%2891%2990034-Q"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2F0022-2364%2891%2990034-Q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0022-2364%252891%252990034-Q%26sid%3Dliteratum%253Aachs%26aulast%3DGeen%26aufirst%3DH.%26aulast%3DFreeman%26aufirst%3DR.%26atitle%3DBand-selective%2520radiofrequency%2520pulses%26jtitle%3DJ.%2520Magn.%2520Reson.%2520%25281969-1992%2529%26date%3D1991%26volume%3D93%26spage%3D93%26epage%3D141%26doi%3D10.1016%2F0022-2364%2891%2990034-Q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Angulo, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Enriquez-Navas, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nieto, P. M.</span><span> </span><span class="NLM_article-title">Ligand-receptor binding affinities from saturation transfer difference (STD) NMR spectroscopy: the binding isotherm of STD initial growth rates</span> <span class="citation_source-journal">Chem. - Eur. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">7803</span><span class="NLM_x">–</span> <span class="NLM_lpage">7812</span><span class="refDoi"> DOI: 10.1002/chem.200903528</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=10.1002%2Fchem.200903528" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=20496354" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=1%3ACAS%3A528%3ADC%252BC3cXoslyrsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2010&pages=7803-7812&author=J.+Anguloauthor=P.+M.+Enriquez-Navasauthor=P.+M.+Nieto&title=Ligand-receptor+binding+affinities+from+saturation+transfer+difference+%28STD%29+NMR+spectroscopy%3A+the+binding+isotherm+of+STD+initial+growth+rates&doi=10.1002%2Fchem.200903528"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand-Receptor Binding Affinities from Saturation Transfer Difference (STD) NMR Spectroscopy: The Binding Isotherm of STD Initial Growth Rates</span></div><div class="casAuthors">Angulo, Jesus; Enriquez-Navas, Pedro M.; Nieto, Pedro M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - A European Journal</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">7803-7812, S7803/1-S7803/11</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The direct evaluation of dissocn. consts. (KD) from the variation of satn. transfer difference (STD) NMR spectroscopy values with the receptor-ligand ratio is not feasible due to the complex dependence of STD intensities on the spectral properties of the obsd. signals.  Indirect evaluation, by competition expts., allows the detn. of KD, as long as a ligand of known affinity is available for the protein under study.  Herein, the authors present a novel protocol based on STD NMR spectroscopy for the direct measurements of receptor-ligand dissocn. consts. (KD) from single-ligand titrn. expts.  The influence of several exptl. factors on STD values has been studied in detail, confirming the marked impact on std. detns. of protein-ligand affinities by STD NMR spectroscopy.  These factors, namely, STD satn. time, ligand residence time in the complex, and the intensity of the signal, affect the accumulation of satn. in the free ligand by processes closely related to fast protein-ligand rebinding and longitudinal relaxation of the ligand signals.  The proposed method avoids the dependence of the magnitudes of ligand STD signals at a given satn. time on spurious factors by constructing the binding isotherms using the initial growth rates of the STD amplification factors, in a similar way to the use of NOE growing rates to est. cross relaxation rates for distance evaluations.  Herein, it is demonstrated that the effects of these factors are cancelled out by analyzing the protein-ligand assocn. curve using STD values at the limit of zero satn. time, when virtually no ligand rebinding or relaxation takes place.  The approach is validated for two well-studied protein-ligand systems: the binding of the saccharides GlcNAc and GlcNAcβ1,4GlcNAc (chitobiose) to the wheat germ agglutinin (WGA) lectin, and the interaction of the amino acid L-tryptophan to bovine serum albumin (BSA).  In all cases, the exptl. KD measured under different exptl. conditions converged to the thermodn. values.  The proposed protocol allows accurate detns. of protein-ligand dissocn. consts., extending the applicability of the STD NMR spectroscopy for affinity measurements, which is of particular relevance for those proteins for which a ligand of known affinity is not available.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXZTY96tfffLVg90H21EOLACvtfcHk0ljS38B5LXae2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXoslyrsrY%253D&md5=df2bd44c799e3e5d88a08c0dcc0cb9dc</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1002%2Fchem.200903528&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fchem.200903528%26sid%3Dliteratum%253Aachs%26aulast%3DAngulo%26aufirst%3DJ.%26aulast%3DEnriquez-Navas%26aufirst%3DP.%2BM.%26aulast%3DNieto%26aufirst%3DP.%2BM.%26atitle%3DLigand-receptor%2520binding%2520affinities%2520from%2520saturation%2520transfer%2520difference%2520%2528STD%2529%2520NMR%2520spectroscopy%253A%2520the%2520binding%2520isotherm%2520of%2520STD%2520initial%2520growth%2520rates%26jtitle%3DChem.%2520-%2520Eur.%2520J.%26date%3D2010%26volume%3D16%26spage%3D7803%26epage%3D7812%26doi%3D10.1002%2Fchem.200903528" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mangos, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keates, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lambert, J.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Felletar, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Volkmer, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Müller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pawson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gingras, A.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span> </span><span class="NLM_article-title">Histone recognition and large-scale structural analysis of the human bromodomain family</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">149</span><span class="NLM_x">, </span> <span class="NLM_fpage">214</span><span class="NLM_x">–</span> <span class="NLM_lpage">231</span><span class="refDoi"> DOI: 10.1016/j.cell.2012.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=10.1016%2Fj.cell.2012.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=22464331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=149&publication_year=2012&pages=214-231&author=P.+Filippakopoulosauthor=S.+Picaudauthor=M.+Mangosauthor=T.+Keatesauthor=J.-P.+Lambertauthor=D.+Barsyte-Lovejoyauthor=I.+Felletarauthor=R.+Volkmerauthor=S.+M%C3%BCllerauthor=T.+Pawsonauthor=A.-C.+Gingrasauthor=C.+H.+Arrowsmithauthor=S.+Knapp&title=Histone+recognition+and+large-scale+structural+analysis+of+the+human+bromodomain+family&doi=10.1016%2Fj.cell.2012.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Recognition and Large-Scale Structural Analysis of the Human Bromodomain Family</span></div><div class="casAuthors">Filippakopoulos, Panagis; Picaud, Sarah; Mangos, Maria; Keates, Tracy; Lambert, Jean-Philippe; Barsyte-Lovejoy, Dalia; Felletar, Ildiko; Volkmer, Rudolf; Muller, Susanne; Pawson, Tony; Gingras, Anne-Claude; Arrowsmith, Cheryl H.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">149</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">214-231</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) are protein interaction modules that specifically recognize ε-N-lysine acetylation motifs, a key event in the reading process of epigenetic marks.  The 61 BRDs in the human genome cluster into eight families based on structure/sequence similarity.  Here, we present 29 high-resoln. crystal structures, covering all BRD families.  These proteins are: ASH1L, ATAD2, BAZ2B, BPTF, BRD1, BRD3(1), BRD3(2), BRD4(1), BRD4(2), BRD9, BRDT(1), CECR2, EP300, CREBBP, GCN5L2, KIAA1240, PB1(1), PB1(2), PB1(3), PB1(4), PB1(5), PB1(6), PCAF, PHIP(2), TAF1(2), WDR9(2), BRD4(1).  Comprehensive crossfamily structural anal. identifies conserved and family-specific structural features that are necessary for specific acetylation-dependent substrate recognition.  Screening of more than 30 representative BRDs against systematic histone-peptide arrays identifies new BRD substrates and reveals a strong influence of flanking posttranslational modifications, such as acetylation and phosphorylation, suggesting that BRDs recognize combinations of marks rather than singly acetylated sequences.  We further uncovered a structural mechanism for the simultaneous binding and recognition of diverse diacetyl-contg. peptides by BRD4.  These data provide a foundation for structure-based drug design of specific inhibitors for this emerging target family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK2FSIcPoXZLVg90H21EOLACvtfcHk0ljS38B5LXae2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D&md5=05583a01a1cd4c3b59fc59c00e4c4777</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DMangos%26aufirst%3DM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DLambert%26aufirst%3DJ.-P.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DVolkmer%26aufirst%3DR.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DPawson%26aufirst%3DT.%26aulast%3DGingras%26aufirst%3DA.-C.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DHistone%2520recognition%2520and%2520large-scale%2520structural%2520analysis%2520of%2520the%2520human%2520bromodomain%2520family%26jtitle%3DCell%26date%3D2012%26volume%3D149%26spage%3D214%26epage%3D231%26doi%3D10.1016%2Fj.cell.2012.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Vonrhein, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flensburg, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keller, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharff, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smart, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paciorek, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Womack, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bricogne, G.</span><span> </span><span class="NLM_article-title">Data processing and analysis with the autoPROC toolbox</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x">, </span> <span class="NLM_fpage">293</span><span class="NLM_x">–</span> <span class="NLM_lpage">302</span><span class="refDoi"> DOI: 10.1107/S0907444911007773</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=10.1107%2FS0907444911007773" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=21460447" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=293-302&author=C.+Vonrheinauthor=C.+Flensburgauthor=P.+Kellerauthor=A.+Sharffauthor=O.+Smartauthor=W.+Paciorekauthor=T.+Womackauthor=G.+Bricogne&title=Data+processing+and+analysis+with+the+autoPROC+toolbox&doi=10.1107%2FS0907444911007773"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Data processing and analysis with the autoPROC toolbox</span></div><div class="casAuthors">Vonrhein, Clemens; Flensburg, Claus; Keller, Peter; Sharff, Andrew; Smart, Oliver; Paciorek, Wlodek; Womack, Thomas; Bricogne, Gerard</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">293-302</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A typical diffraction expt. will generate many images and data sets from different crystals in a very short time.  This creates a challenge for the high-throughput operation of modern synchrotron beamlines as well as for the subsequent data processing.  Novice users in particular may feel overwhelmed by the tables, plots and nos. that the different data-processing programs and software packages present to them.  Here, some of the more common problems that a user has to deal with when processing a set of images that will finally make up a processed data set are shown, concg. on difficulties that may often show up during the first steps along the path of turning the expt. (i.e. data collection) into a model (i.e. interpreted electron d.).  Difficulties such as unexpected crystal forms, issues in crystal handling and suboptimal choices of data-collection strategies can often be dealt with, or at least diagnosed, by analyzing specific data characteristics during processing.  In the end, one wants to distinguish problems over which one has no immediate control once the expt. is finished from problems that can be remedied a posteriori.  A new software package, autoPROC, is also presented that combines third-party processing programs with new tools and an automated workflow script that is intended to provide users with both guidance and insight into the offline processing of data affected by the difficulties mentioned above, with particular emphasis on the automated treatment of multi-sweep data sets collected on multi-axis goniostats.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmah7iUrMLXLVg90H21EOLACvtfcHk0ljS38B5LXae2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLs%253D&md5=a0a3579f6b371b6312819883864aa80d</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1107%2FS0907444911007773&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444911007773%26sid%3Dliteratum%253Aachs%26aulast%3DVonrhein%26aufirst%3DC.%26aulast%3DFlensburg%26aufirst%3DC.%26aulast%3DKeller%26aufirst%3DP.%26aulast%3DSharff%26aufirst%3DA.%26aulast%3DSmart%26aufirst%3DO.%26aulast%3DPaciorek%26aufirst%3DW.%26aulast%3DWomack%26aufirst%3DT.%26aulast%3DBricogne%26aufirst%3DG.%26atitle%3DData%2520processing%2520and%2520analysis%2520with%2520the%2520autoPROC%2520toolbox%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D293%26epage%3D302%26doi%3D10.1107%2FS0907444911007773" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Winn, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ballard, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodson, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evans, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keegan, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krissinel, E. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leslie, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCoy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McNicholas, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murshudov, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pannu, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Potterton, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Powell, H. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vagin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, K. S.</span><span> </span><span class="NLM_article-title">Overview of the CCP4 suite and current developments</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x"> (</span><span class="NLM_issue">4</span><span class="NLM_x">) </span> <span class="NLM_fpage">235</span><span class="NLM_x">–</span> <span class="NLM_lpage">242</span><span class="refDoi"> DOI: 10.1107/S0907444910045749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=235-242&issue=4&author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=R.+M.+Keeganauthor=E.+B.+Krissinelauthor=A.+G.+Leslieauthor=A.+McCoyauthor=S.+J.+McNicholasauthor=G.+N.+Murshudovauthor=N.+S.+Pannuauthor=E.+A.+Pottertonauthor=H.+R.+Powellauthor=R.+J.+Readauthor=A.+Vaginauthor=K.+S.+Wilson&title=Overview+of+the+CCP4+suite+and+current+developments&doi=10.1107%2FS0907444910045749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0lh6bk2PclCanA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DKeegan%26aufirst%3DR.%2BM.%26aulast%3DKrissinel%26aufirst%3DE.%2BB.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%26aulast%3DMcCoy%26aufirst%3DA.%26aulast%3DMcNicholas%26aufirst%3DS.%2BJ.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DPotterton%26aufirst%3DE.%2BA.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520the%2520CCP4%2520suite%2520and%2520current%2520developments%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26issue%3D4%26spage%3D235%26epage%3D242%26doi%3D10.1107%2FS0907444910045749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lohkamp, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Features and development of Coot</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x"> (</span><span class="NLM_issue">4</span><span class="NLM_x">) </span> <span class="NLM_fpage">486</span><span class="NLM_x">–</span> <span class="NLM_lpage">501</span><span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&issue=4&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lh6bk2PclCanA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26issue%3D4%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Bricogne, G.; Blanc, E.; Brandl, M.; Flensburg, C.; Keller, P.; Paciorek, P.; Roversi, P.; Sharff, A.; Smart, O.; Vonrhein, C.; Womack, T.</span>BUSTER, version <span class="NLM_edition">2.11.2</span>; <span class="NLM_publisher-name">Global Phasing Ltd.</span>: <span class="NLM_publisher-loc">Cambridge, U.K.</span>,<span class="NLM_x"> </span><span class="NLM_year">2011</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&author=G.+Bricogne&author=E.+Blanc&author=M.+Brandl&author=C.+Flensburg&author=P.+Keller&author=P.+Paciorek&author=P.+Roversi&author=A.+Sharff&author=O.+Smart&author=C.+Vonrhein&author=T.+Womack&title=BUSTER"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DBricogne%26aufirst%3DG.%26jtitle%3DBUSTER%26pub%3DGlobal%2520Phasing%2520Ltd%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5M39" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5M39','PDB','5M39'); return false;">PDB: 5M39</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5M3A" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5M3A','PDB','5M3A'); return false;">PDB: 5M3A</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2OSS" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2OSS','PDB','2OSS'); return false;">PDB: 2OSS</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i24"><a href="/doi/suppl/10.1021/acs.jmedchem.7b00845">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_85851"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.7b00845">10.1021/acs.jmedchem.7b00845</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Description of the LOGSY-titration methodology and the extraction of LOGSY factors; simulation of LOGSY-titration intensities; comparison of LOGSY titrations with protonated and deuterated protein; table of compounds that show an affinity increase after introduction of a secondary amine; schemes for the synthesis of ligand <b>1</b> and ligand <b>2</b>; crystallographic data collection and refinement statistics (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b00845/suppl_file/jm7b00845_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and IC<sub>50</sub> values (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b00845/suppl_file/jm7b00845_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00845/suppl_file/jm7b00845_si_001.pdf">jm7b00845_si_001.pdf (4.94 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00845/suppl_file/jm7b00845_si_002.csv">jm7b00845_si_002.csv (0.37 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Atomic coordinates of Brd4-BD1 bound to ligand <b>1</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5M39">5M39</a>) and ligand <b>2</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5M3A">5M3A</a>) have been deposited in the Protein Data Bank. Authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.7b00845&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2017.60.issue-21%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.7b00845%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.7b00845" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                2MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799687149393c2e","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
