
<html lang="en"     class="pb-page"  data-request-id="b4efd4e7-4675-4588-8059-62f06b4e9c94"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;article:article:10.1021/acs.jmedchem.9b01566;issue:issue:10.1021/jmcmar.2020.63.issue-3;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Discovery of Potent and Selective Epidermal Growth Factor Receptor (EGFR) Bifunctional Small-Molecule Degraders" /></meta><meta name="dc.Creator" content="Meng  Cheng" /></meta><meta name="dc.Creator" content="Xufen  Yu" /></meta><meta name="dc.Creator" content="Kaylene  Lu" /></meta><meta name="dc.Creator" content="Ling  Xie" /></meta><meta name="dc.Creator" content="Li  Wang" /></meta><meta name="dc.Creator" content="Fanye  Meng" /></meta><meta name="dc.Creator" content="Xiaoran  Han" /></meta><meta name="dc.Creator" content="Xian  Chen" /></meta><meta name="dc.Creator" content="Jing  Liu" /></meta><meta name="dc.Creator" content="Yue  Xiong" /></meta><meta name="dc.Creator" content="Jian  Jin" /></meta><meta name="dc.Description" content="Several epidermal growth factor receptor (EGFR) tyrosine kinase inhibitors have been developed and approved by Food and Drug Administration for the treatment of non-small-cell lung cancers, but the..." /></meta><meta name="Description" content="Several epidermal growth factor receptor (EGFR) tyrosine kinase inhibitors have been developed and approved by Food and Drug Administration for the treatment of non-small-cell lung cancers, but the..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="January 2, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b01566" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01566" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b01566" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01566" /></link>
        
    
    

<title>Discovery of Potent and Selective Epidermal Growth Factor Receptor (EGFR) Bifunctional Small-Molecule Degraders | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b01566" /></meta><meta property="og:title" content="Discovery of Potent and Selective Epidermal Growth Factor Receptor (EGFR) Bifunctional Small-Molecule Degraders" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0003.jpeg" /></meta><meta property="og:description" content="Several epidermal growth factor receptor (EGFR) tyrosine kinase inhibitors have been developed and approved by Food and Drug Administration for the treatment of non-small-cell lung cancers, but their efficacy can be compromised by acquired drug resistance conferred by EGFR-mutant variants. Here, we described the discovery of a novel E3 ligase von Hippel–Lindau-recruiting EGFR degrader, MS39 (compound 6), and a first-in-class E3 ligase cereblon-recruiting EGFR degrader, MS154 (compound 10), using the proteolysis targeting chimera technology. These compounds potently induced the degradation of mutant but not wild-type EGFR in an E3 ligase-dependent manner in cancer cell lines and effectively suppressed the growth of lung cancer cells compared with the corresponding negative controls. The global proteomic analyses revealed that the compounds were highly selective for EGFR. Furthermore, both compounds were bioavailable in mouse pharmacokinetic studies, and compound 6 is the first EGFR degrader suitable for in vivo efficacy studies. Overall, we provide a set of well-characterized chemical tools to the research community." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b01566"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01566">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b01566&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b01566&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b01566&amp;href=/doi/10.1021/acs.jmedchem.9b01566" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 3</span><span class="cit-fg-pageRange">, 1216-1232</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/3" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b01554" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b01414" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of Potent and Selective Epidermal Growth Factor Receptor (EGFR) Bifunctional Small-Molecule Degraders</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Meng Cheng</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Meng Cheng</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Curriculum in Genetics and Molecular Biology,  Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Meng++Cheng">Meng Cheng</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xufen Yu</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xufen Yu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Mount Sinai Center for Therapeutics Discovery, Departments of Pharmacological Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#c8b0bdaeada6e6b1bd88a5bbbba5e6adacbd"><span class="__cf_email__" data-cfemail="c8b0bdaeada6e6b1bd88a5bbbba5e6adacbd">[email protected]</span></a> (X.Y.).</span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xufen++Yu">Xufen Yu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kaylene Lu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kaylene Lu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kaylene++Lu">Kaylene Lu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ling Xie</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ling Xie</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Biophysics, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ling++Xie">Ling Xie</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Li Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Li Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Biophysics, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Li++Wang">Li Wang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Fanye Meng</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Fanye Meng</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Mount Sinai Center for Therapeutics Discovery, Departments of Pharmacological Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Fanye++Meng">Fanye Meng</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xiaoran Han</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xiaoran Han</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xiaoran++Han">Xiaoran Han</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xian Chen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xian Chen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Biophysics, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xian++Chen">Xian Chen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jing Liu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jing Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Mount Sinai Center for Therapeutics Discovery, Departments of Pharmacological Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jing++Liu">Jing Liu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yue Xiong</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yue Xiong</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Curriculum in Genetics and Molecular Biology,  Lineberger Comprehensive Cancer Center,  Department of Biochemistry and Biophysics, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#1960617076777e597c74787075376c777a377c7d6c"><span class="__cf_email__" data-cfemail="cdb4b5a4a2a3aa8da8a0aca4a1e3b8a3aee3a8a9b8">[email protected]</span></a> (Y.X.).</span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yue++Xiong">Yue Xiong</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Jian Jin</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jian Jin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Mount Sinai Center for Therapeutics Discovery, Departments of Pharmacological Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#0d67646c63236764634d607e7e6023686978"><span class="__cf_email__" data-cfemail="3a50535b54145053547a57494957145f5e4f">[email protected]</span></a> (J.J.).</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jian++Jin">Jian Jin</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2387-3862" title="Orcid link">http://orcid.org/0000-0002-2387-3862</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01566&amp;href=/doi/10.1021%2Facs.jmedchem.9b01566" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 3</span><span class="cit-pageRange">, 1216–1232</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">January 2, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>19 September 2019</li><li><span class="item_label"><b>Published</b> online</span>2 January 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 13 February 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b01566" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01566</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D1216%26pageCount%3D17%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DMeng%2BCheng%252C%2BXufen%2BYu%252C%2BKaylene%2BLu%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D3%26contentID%3Dacs.jmedchem.9b01566%26title%3DDiscovery%2Bof%2BPotent%2Band%2BSelective%2BEpidermal%2BGrowth%2BFactor%2BReceptor%2B%2528EGFR%2529%2BBifunctional%2BSmall-Molecule%2BDegraders%26numPages%3D17%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D1232%26publicationDate%3DFebruary%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b01566"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">4520</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">30</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b01566" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of Potent and Selective Epidermal Growth Factor Receptor (EGFR) Bifunctional Small-Molecule Degraders&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Meng&quot;,&quot;last_name&quot;:&quot;Cheng&quot;},{&quot;first_name&quot;:&quot;Xufen&quot;,&quot;last_name&quot;:&quot;Yu&quot;},{&quot;first_name&quot;:&quot;Kaylene&quot;,&quot;last_name&quot;:&quot;Lu&quot;},{&quot;first_name&quot;:&quot;Ling&quot;,&quot;last_name&quot;:&quot;Xie&quot;},{&quot;first_name&quot;:&quot;Li&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Fanye&quot;,&quot;last_name&quot;:&quot;Meng&quot;},{&quot;first_name&quot;:&quot;Xiaoran&quot;,&quot;last_name&quot;:&quot;Han&quot;},{&quot;first_name&quot;:&quot;Xian&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Jing&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Yue&quot;,&quot;last_name&quot;:&quot;Xiong&quot;},{&quot;first_name&quot;:&quot;Jian&quot;,&quot;last_name&quot;:&quot;Jin&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;01&quot;,&quot;day&quot;:&quot;02&quot;,&quot;issue&quot;:&quot;3&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;1216-1232&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b01566&quot;},&quot;abstract&quot;:&quot;Several epidermal growth factor receptor (EGFR) tyrosine kinase inhibitors have been developed and approved by Food and Drug Administration for the treatment of non-small-cell lung cancers, but their efficacy can be compromised by acquired drug resistance conferred by EGFR-mutant variants. Here, we described the discovery of a novel E3 ligase von Hippel–Lindau-recruiting EGFR degrader, MS39 (compound 6), and a first-in-class E3 ligase cereblon-recruiting EGFR degrader, MS154 (compound 10), using the proteolysis targeting chimera technology. These compounds potently induced the degradation of mutant but not wild-type EGFR in an E3 ligase-dependent manner in cancer cell lines and effectively suppressed the growth of lung cancer cells compared with the corresponding negative controls. The global proteomic analyses revealed that the compounds were highly selective for EGFR. Furthermore, both compounds were bioavailable in mouse pharmacokinetic studies, and compound 6 is the first EGFR degrader suitable for in viv&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01566&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01566" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01566&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01566" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01566&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01566" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01566&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01566&amp;href=/doi/10.1021/acs.jmedchem.9b01566" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b01566" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b01566" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (8 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01566&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b01566%26sid%3Dliteratum%253Aachs%26pmid%3D31895569%26genre%3Darticle%26aulast%3DCheng%26date%3D2020%26atitle%3DDiscovery%2Bof%2BPotent%2Band%2BSelective%2BEpidermal%2BGrowth%2BFactor%2BReceptor%2B%2528EGFR%2529%2BBifunctional%2BSmall-Molecule%2BDegraders%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D3%26spage%3D1216%26epage%3D1232%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291781" title="Degradation">Degradation</a>,</li><li><a href="/action/doSearch?ConceptID=291297" title="Cell signaling">Cell signaling</a>,</li><li><a href="/action/doSearch?ConceptID=292258" title="Cancer">Cancer</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/3" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/jmcmar.2020.63.issue-3/20200213/jmcmar.2020.63.issue-3.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/medium/jm9b01566_0003.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01566&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Several epidermal growth factor receptor (EGFR) tyrosine kinase inhibitors have been developed and approved by Food and Drug Administration for the treatment of non-small-cell lung cancers, but their efficacy can be compromised by acquired drug resistance conferred by <i>EGFR</i>-mutant variants. Here, we described the discovery of a novel E3 ligase von Hippel–Lindau-recruiting EGFR degrader, MS39 (compound <b>6</b>), and a first-in-class E3 ligase cereblon-recruiting EGFR degrader, MS154 (compound <b>10</b>), using the proteolysis targeting chimera technology. These compounds potently induced the degradation of mutant but not wild-type EGFR in an E3 ligase-dependent manner in cancer cell lines and effectively suppressed the growth of lung cancer cells compared with the corresponding negative controls. The global proteomic analyses revealed that the compounds were highly selective for EGFR. Furthermore, both compounds were bioavailable in mouse pharmacokinetic studies, and compound <b>6</b> is the first EGFR degrader suitable for in vivo efficacy studies. Overall, we provide a set of well-characterized chemical tools to the research community.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_20831" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_20831" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Epidermal growth factor receptor (EGFR) is a transmembrane protein tyrosine kinase that functions as a receptor for members of the EGF family to trigger EGFR signal pathway in human epithelial cells, thereby regulating cell proliferation, invasion, metastasis, apoptosis, and angiogenesis.<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1−3)</a> Increased EGFR activity resulting from overexpression, mutation, or amplification of EGFR gene contributes to many human malignancies, including esophagus cancers, glioblastoma, anal cancers, epithelial cancers of the head and neck, breast cancers, and lung cancers, especially in non-small-cell lung cancers (NSCLCs).<a onclick="showRef(event, 'ref1 ref4 ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref1 ref4 ref5 ref6">(1,4−6)</a> Activating mutations, such as in-frame deletions of exon 19 and L858R mutation, which occur in the EGFR kinase adenosine triphosphate (ATP)-binding domain, have been categorized as the oncogenic driver in NSCLC.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Lung cancer is the leading cause of cancer death worldwide, and NSCLCs belong to the major category of lung cancers.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> As a therapeutic target for NSCLC, EGFR has been extensively investigated by the biomedical community. In particular, studies on targeting the activity of mutant EGFR ATP-binding domain have led to the discovery of a number of Food and Drug Administration (FDA)-approved EGFR tyrosine kinase inhibitors (TKIs) (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). The first-generation TKIs gefitinib<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> and erlotinib<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> showed dramatic response initially and prolonged the survival in NSCLC patients. However, the secondary “gatekeeper” T790M mutation has increased the ATP-binding affinity and caused a relapse in most NSCLC patients after the 9–14 months of treatment.<a onclick="showRef(event, 'ref10 ref11 ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref10 ref11 ref12 ref13">(10−13)</a> To overcome the resistance, a number of the second-generation EGFR inhibitors targeting EGFR harboring T790M activating mutation have been developed, including afatinib and dacomitinib. The irreversible covalent bond with the Cys797 confers enhanced sensitivity and selectivity of these TKIs, but these drugs exhibited severe side effects due to their activity against wild-type (WT) EGFR. Subsequently, the third-generation EGFR covalent inhibitors with improved selectivity over WT EGFR have been developed. However, C797S point mutation and/or other mechanisms have been shown to be associated with the acquired resistance to irreversible EGFR TKIs, which make NSCLC refractory to these inhibitors.<a onclick="showRef(event, 'ref14 ref15 ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref14 ref15 ref16 ref17">(14−17)</a> The fourth-generation EGFR TKIs such as EAI045<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> and other noncovalent inhibitors,<a onclick="showRef(event, 'ref19 ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref19 ref20 ref21">(19−21)</a> which target allosteric binding site(s), appear to be a major breakthrough against these tertiary mutations. Despite that, there is still an unmet medical need to develop novel small-molecule inhibitors or therapeutic approaches to overcome multipoint mutations of EGFR.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/medium/jm9b01566_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0010.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Chemical structures of representative first-, second-, third-, and fourth-generation EGFR TKIs.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01566&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">As a potential therapeutic approach, proteolysis targeting chimeras (PROTACs) are capable of targeting a specific protein for degradation.<a onclick="showRef(event, 'ref23 ref24 ref25 ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref23 ref24 ref25 ref26 ref27">(23−27)</a> PROTACs are heterobifunctional molecules, typically consisting of a small molecule as the binder to the protein target of interest, a small-molecule ligand that recruits an E3 ligase, and a linker to conjugate the above two moieties.<a onclick="showRef(event, 'ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref28 ref29">(28,29)</a> The induced proximity by PROTACs leads to selective polyubiquitination of the target protein and its subsequent degradation at the proteasome. Unlike traditional enzyme inhibitors, which inhibit the catalytic activity of the target enzyme, PROTACs induce the degradation of its target protein, thus eliminating other functions of the protein in addition to its enzymatic activity. Moreover, while some kinase inhibitors are restricted to bind to the catalytic pocket,<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> PROTACs could potentially utilize all surface binding pockets on its target protein to achieve degradation. Thus, the emerging bifunctional small-molecule-mediated protein degradation paradigm could overcome the limitations of conventionally occupancy-driven inhibitors. Over the past several years, this technology has been successfully applied to the degradation of numerous proteins.<a onclick="showRef(event, 'ref28 ref29 ref31 ref32 ref33 ref34 ref35 ref36 ref37 ref38 ref39 ref40'); return false;" href="javascript:void(0);" class="ref ref28 ref29 ref31 ref32 ref33 ref34 ref35 ref36 ref37 ref38 ref39 ref40">(28,29,31−40)</a> However, targeting membrane-bound receptors or receptor tyrosine kinases (RTKs) using this technology has not been extensively studied. Of note, the first successful EGFR PROTAC using a binder to the E3 ligase von Hippel–Lindau (VHL), PROTAC3, has recently been reported.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a></div><div class="NLM_p last">Herein, we describe the design, synthesis, and characterization of a novel gefitinib-based VHL-recruiting EGFR degrader (<b>6</b>, MS39) that is more potent than the previously reported EGFR degrader PROTAC3 and a first-in-class gefitinib-based cereblon (CRBN)-recruiting EGFR degrader (<b>10</b>, MS154). We also developed two structurally similar compounds <b>27</b> (MS39N) and <b>28</b> (MS154N) that do not bind the corresponding E3 ligase as degrader negative controls for compounds <b>6</b> and <b>10</b>. We characterized these compounds in a battery of biochemical and cellular assays. In particular, we found that compounds <b>6</b> and <b>10</b> potently induced the degradation of mutant EGFR while sparing WT EGFR and inhibited cell proliferation more effectively than the degrader negative controls <b>27</b> and <b>28</b> in lung cancer cells. We also found that serum starvation enhanced the degradation effect, which might lead to improved effectiveness in tumors where serum deprivation is commonly observed. Our mechanism of action studies confirmed that the induced EGFR degradation was mediated by the corresponding E3 ubiquitin ligase. Importantly, our global proteomic analysis results indicate that compounds <b>6</b> and <b>10</b> are highly selective EGFR degraders. Moreover, compounds <b>6</b> and <b>10</b> were bioavailable in mouse pharmacokinetic (PK) studies. In particular, compound <b>6</b> is the first EGFR degrader with sufficient in vivo PK properties, thus suitable for in vivo efficacy studies. Overall, we provide the research community a set of well-characterized chemical tools. Our study has also provided further evidence that RTKs such as EGFR can be targeted via the PROTAC technology and paved the way for further developing and exploring EGFR degraders as a potential therapeutic strategy for NSCLC and other cancers.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_70527" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_70527" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Design, Synthesis, and Initial Evaluation of EGFR Bifunctional Small-Molecule Degraders</h3><div class="NLM_p">Gefitinib (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>), a highly potent and selective mutant EGFR TKI, has been approved since 2003 for the treatment of NSCLC.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> It has been classified as a first-generation EGFR inhibitor. Gefitinib selectively binds the EGFR kinase domain, preventing ATP from binding and resulting in the inhibition of EGFR downstream signaling. While gefitinib binds WT EGFR with high affinity, it has a higher binding affinity for EGFR mutants compared to that of the WT.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> We chose gefitinib as the binder to EGFR to develop EGFR bifunctional small-molecule degraders, mainly because of gefitinib’s high binding affinity to mutant EGFR. The cocrystal structure of the EGFR kinase domain in complex with gefitinib (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4I22">4I22</a>) indicates that the morpholine group of gefitinib projects the surface of the binding pocket (Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.9b01566/suppl_file/jm9b01566_si_001.pdf" class="ext-link">Figure S1</a>). We therefore replaced the solvent-exposed morpholine group with a piperazine group and used the outer nitrogen of the piperazine group as the tethering site. We designed and synthesized a set of putative EGFR degraders by linking this piperazine moiety to VHL-1, which binds the E3 ligase VHL,<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> and pomalidomide/thalidomide, which bind the E3 ligase CRBN,<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> through several linkers (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/medium/jm9b01566_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0017.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Chemical structures of designed gefitinib-based EGFR degraders <b>1</b>–<b>10</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01566&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Synthetic routes for preparing compounds <b>1</b>–<b>10</b> are outlined in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. A simple nucleophilic substitution reaction between commercially available FAAH-IN-2 (<b>11</b>) and <i>tert</i>-butyl 4-(3-bromopropyl)piperazine-1-carboxylate (<b>12</b>),<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> followed by a deprotection reaction, produced intermediate <b>13</b>, which was subsequently converted to intermediate <b>14</b> via a straightforward two-step sequence. Intermediates <b>15</b>–<b>20</b> were prepared by amide-coupling reactions of VHL-1<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> with commercially available dicarboxylic acids. The canonical amide coupling of intermediate <b>13</b> with intermediates <b>15</b>–<b>20</b> afforded the desired compounds <b>1</b>–<b>6</b>. Compounds <b>7</b>–<b>9</b> were synthesized following the same amide-coupling procedures between intermediate <b>14</b> and reported pomalidomide-based linkers <b>21</b>–<b>23</b>.<a onclick="showRef(event, 'ref44 ref45'); return false;" href="javascript:void(0);" class="ref ref44 ref45">(44,45)</a> Intermediate <b>26</b> was obtained via a nucleophilic substitution reaction and subsequent deprotection reaction from the commercially available thalidomide analog <b>24</b> and <i>tert</i>-butyl (8-iodooctyl)carbamate (<b>25</b>), which was prepared according to reported procedures.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> Finally, the amide-coupling reaction between intermediate <b>14</b> and amine intermediate <b>26</b> generated thalidomide-based compound <b>10</b>.</div><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/medium/jm9b01566_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Gefitinib-Based EGFR Degraders <b>1</b>–<b>10</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01566&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reaction conditions: (a) K<sub>2</sub>CO<sub>3</sub>, dimethylformamide (DMF), 80 °C; (b) trifluoroacetic acid (TFA), dichloromethane (DCM), room temperature (rt), 30 min; (c) ethyl 3-bromopropanoate, K<sub>2</sub>CO<sub>3</sub>, DMF, 80 °C; (d) LiOH, tetrahydrofuran (THF), H<sub>2</sub>O, rt, 12 h; (e) 1-ethyl-3-(3-dimethylaminopropyl)carbodiimide (EDCI), 1-hydroxy-7-azabenzo-triazole (HOAt), <i>N</i>-methylmorpholine (NMM), dimethyl sulfoxide (DMSO), rt, 4 h; (f) NaHCO<sub>3</sub>, NaI, DMF, 65 °C, 2 days; (g) TFA/DCM, rt, 30 min; (h) <b>14</b>, EDCI, HOAt, NMM, DMSO, rt, 12 h.</p></p></figure><div class="NLM_p">The effect of these bifunctional compounds on reducing EGFR protein levels in cells was assessed via Western blotting (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). We found that compounds <b>1</b>–<b>3</b>, which contain several poly(ethylene glycol) (PEG) linkers conjugated to VHL-1, induced EGFR protein degradation weakly in HCC-827 cells treated with the compounds at up to 1 μM concentrations for 16 h. Among the three all-carbon linkers (in compounds <b>4</b>–<b>6</b>) we explored, the longest linker with nine methylene groups (in compound <b>6</b>) led to effective EGFR protein degradation even at 10 nM. Among the four CRBN-recruiting compounds we explored, compound <b>9</b>, which contains a linker with six methylene groups, was more effective at reducing EGFR protein levels than compounds <b>7</b>–<b>8</b>, which contain PEG linkers. Interestingly, compound <b>10</b>, which contains a longer carbon linker (eight methylene groups) and the thalidomide moiety instead of the pomalidomide moiety, was slightly more effective at reducing EGFR protein levels than compound <b>9</b>. We therefore selected compounds <b>6</b> and <b>10</b> for further characterization.</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/medium/jm9b01566_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0012.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Western blotting analysis of EGFR protein in HCC-827 cells treated with compounds <b>1</b>–<b>10</b>. (A) VHL-1-based compounds <b>1</b>–<b>6</b> and (B) CRBN-based compounds <b>7</b>–<b>10</b> were tested in HCC-827 cell lines using three different concentrations (10, 100, and 1000 nM). EGFR e19d: EGFR mutant with exon 19 deletion.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01566&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We also designed compounds <b>27</b> and <b>28</b> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>) as structurally similar negative controls for degraders <b>6</b> and <b>10</b>, respectively. Compound <b>27</b> contains a diastereoisomer of VHL-1, which is known to be incapable of binding the VHL E3 ligase,<a onclick="showRef(event, 'ref47 ref48 ref49'); return false;" href="javascript:void(0);" class="ref ref47 ref48 ref49">(47−49)</a> while maintaining the same EGFR binding moiety and very similar physicochemical properties as degrader <b>6</b>. Similarly, compound <b>28</b> contains an extra methyl group on the glutarimide moiety of thalidomide, which is known to abrogate its ability to bind the E3 ligase CRBN,<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> while also maintaining the same EGFR binding motif and similar physicochemical properties as degrader <b>10</b>.</div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/medium/jm9b01566_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0001.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Chemical structures of compounds <b>27</b> and <b>28</b> and the previously reported EGFR degrader PROTAC3.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01566&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Synthetic routes for preparing compounds <b>27</b> and <b>28</b> are outlined in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. Intermediate <b>31</b> was prepared by an amide-coupling reaction of compound <b>29</b>, a diastereoisomer of VHL-1, with undecanedioic acid (<b>30</b>). The canonical amide coupling of intermediate <b>13</b> with intermediate <b>31</b> afforded the desired compound <b>27</b>. Intermediate <b>33</b> was prepared by methylation of intermediate <b>32</b>, which was prepared according to the synthetic route in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>D, followed by the removal of the protecting group. Finally, an amide-coupling reaction between intermediate <b>14</b> and amine intermediate <b>33</b> furnished the target compound <b>28</b>.</div><figure id="sch2" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/medium/jm9b01566_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0011.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Negative Control Compounds <b>27</b> and <b>28</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01566&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reaction conditions: (a) 1-ethyl-3-(3-dimethylaminopropyl)carbodiimide (EDCI), 1-hydroxy-7-azabenzo-triazole (HOAt), N-methylmorpholine (NMM), dimethyl sulfoxide (DMSO), rt, 4 h; (b) <b>13</b>, EDCI, HOAt, NMM, DMSO, rt, 12 h; (c) K<sub>2</sub>CO<sub>3</sub>, CH<sub>3</sub>I, DMF, rt, 4 h; (d) TFA/DCM, rt, 30 min; (e) <b>14</b>, EDCI, HOAt, NMM, DMSO, rt, 12 h.</p></p></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Binding Affinities of Compounds <b>6</b>, <b>10</b>, <b>27</b>, and <b>28</b> to WT and Mutant EGFR</h3><div class="NLM_p">We next assessed the binding affinities of the parental inhibitor gefitinib, degraders <b>6</b> and <b>10</b>, and their negative controls <b>27</b> and <b>28</b> to EGFR WT and L858R mutants in a competitive binding assay (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>).<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> Gefitinib displayed similar binding affinity to EGFR WT (<i>K</i><sub>d</sub> = 1.1 ± 2 nM) and L858R mutants (<i>K</i><sub>d</sub> = 0.8 ± 2 nM). Compared to gefitinib, the VHL-recruiting degrader <b>6</b> showed approximately 10-fold weaker binding affinities to EGFR WT (<i>K</i><sub>d</sub> = 11 ± 3 nM) and L858R mutants (<i>K</i><sub>d</sub> = 12 ± 7 nM). Interestingly, compared with gefitinib, the CRBN-recruiting degrader <b>10</b> preserved high binding affinity to EGFR WT (<i>K</i><sub>d</sub> = 1.8 ± 4 nM) but displayed a 5-fold weaker binding affinity to EGFR L858R mutant (<i>K</i><sub>d</sub> = 3.8 ± 5 nM). The negative control compounds <b>27</b> and <b>28</b> exhibited similar binding affinities to both EGFR WT and L858R mutants as degraders <b>6</b> and <b>10</b>. Overall, these compounds exhibited high binding affinities to both EGFR WT and L858R mutants, providing experimental evidence supporting our degrader and negative control design hypothesis that the linker and E3 ligand attachment at the solvent-exposed region do not significantly affect the binding of these compounds to EGFR.</div><figure id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/medium/jm9b01566_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0004.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Compounds <b>6</b>, <b>27</b>, <b>10</b>, and <b>28</b> bind EGFR WT and L858R mutants with high affinities. The <i>K</i><sub>d</sub> determinations were performed in a competitive binding assay in duplicate. Error bars represent ± standard error of the mean (SEM) in duplicated independent experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01566&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Compounds <b>6</b> and <b>10</b> Potently Decrease EGFR Protein Level and Inhibit Downstream Signaling in Lung Cancer Cells</h3><div class="NLM_p">To evaluate the effects of compounds <b>6</b> and <b>10</b> on degrading EGFR mutants and inhibiting downstream signaling, we chose two EGFR-mutant-bearing lung cancer cell lines that are highly sensitive to gefitinib, HCC-827 cells bearing an exon 19 deletion and H3255 cells bearing a L858R point mutation. The expression levels of both VHL and CRBN E3 ligases were confirmed in both cell lines (Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.9b01566/suppl_file/jm9b01566_si_001.pdf" class="ext-link">Figure S2</a>). The effects of the compounds were assessed using combined Western blotting for the total EGFR protein level, autophosphorylation of EGFR, and downstream target AKT (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>).</div><figure id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/medium/jm9b01566_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0013.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Compounds <b>6</b> and <b>10</b> potently reduced protein levels of EGFR mutants and inhibited downstream signaling in lung cancer cells. NSCLC cell lines HCC-827 (A) and H3255 (B) were cultured to 80% confluent and then switched to a serum-free medium for 8 h before treating with indicated concentrations of the compounds for another 16 h. EGFR e19d: EGFR mutant with exon 19 deletion; EGFR L858R: EGFR L858R single point mutant; Gef: gefitinib (parental EGFR inhibitor; 1 μM); 1k: 1000; 10k: 10 000.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01566&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We first examined the EGFR protein degradation induced by compounds <b>6</b> and <b>10</b> in a wide range of concentrations. As illustrated in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>, compounds <b>6</b> and <b>10</b> effectively reduced the mutant EGFR protein level in a concentration-dependent manner in both HCC-827 and H3255 cells. The maximum EGFR protein degradation was achieved over 95% when cells were treated with these two degraders at a concentration as low as 50 nM. Quantification of the Western blotting data determined that the DC<sub>50</sub> values for compound <b>6</b> in HCC-827 and H3255 cells were 5.0 nM and 3.3 nM, respectively, with a 16 h treatment. The DC<sub>50</sub> values of compound <b>10</b> were 11 nM and 25 nM, respectively. These results suggest that our degraders have a potent chemical knockdown effect. Consistent with previous reports,<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> we observed a slight “hook effect” in HCC-827 cells treated with both degraders at the highest concentration (10 μM), but this effect was undetectable in H3255 cells. Interestingly, the parental inhibitor gefitinib at 1 μM did not significantly reduce the EGFR protein level in HCC-827 cells but partially reduced the EGFR protein level in H3255 cells. It is unclear which factors are the main contributors to the observed difference. Nevertheless, gefitinib at 1 μM effectively inhibited the downstream signaling in both cell lines. Our degraders <b>6</b> and <b>10</b> also potently inhibited the EGFR autophosphorylation (p-EGFR) and AKT phosphorylation (p-AKT), which is a downstream target of EGFR signaling pathway, in both cell lines. It is worth to note that our EGFR degraders <b>6</b> and <b>10</b> did not affect the AKT protein levels in both cell lines. We observed that the VHL-recruiting degrader <b>6</b> displayed slightly better EGFR degradation and downstream signaling inhibition effects than the CRBN-recruiting degrader <b>10</b> in HCC-827 cells, and the difference was more significant in H3255 cells. We also performed a side-by-side comparison of our degraders <b>6</b> and <b>10</b> with the previously reported VHL-recruiting EGFR degrader PROTAC3.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> The Western blotting results indicate that compound <b>6</b> and PROTAC3 displayed similar EGFR degradation and downstream signaling inhibition effects in both HCC-827 and H3255 cells. However, at high concentrations (e.g., 1 and 10 μM), PROTAC3 exhibited a much more significant hook effect than compounds <b>6</b> and <b>10</b> in both cell lines (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>).</div><div class="NLM_p">We next evaluated the effect of compounds <b>6</b> and <b>10</b> on inducing the degradation of the WT EGFR. We chose to use the ovarian cancer OVCAR-8 cell line and NSCLC H1299 cell line, which bear the WT EGFR, for the study. We first confirmed that both VHL and CRBN E3 ligases were expressed in these two cell lines (Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.9b01566/suppl_file/jm9b01566_si_001.pdf" class="ext-link">Figure S2</a>). As illustrated in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>, compounds <b>6</b> and <b>10</b> did not significantly reduce the WT EGFR protein levels in both OVCAR-8 and H1299 cells. This result is consistent with the previous finding that PROTAC3 did not degrade the WT EGFR.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> It is unclear why compounds <b>6</b> and <b>10</b>, as well as PROTAC3, selectively degrade EGFR mutants over the WT EGFR. This warrants further investigation.</div><figure id="fig7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/medium/jm9b01566_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0014.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Compounds <b>6</b> and <b>10</b> did not significantly reduce WT EGFR protein levels in OVCAR-8 (<b>A</b>) and H1299 (<b>B</b>) cells. OVCAR-8 and H1299 cells were cultured to 80% confluent and then switched to a serum-free medium for 8 h before treating with indicated concentrations of the compounds for another 16 h. Gef: gefitinib (1 μM).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01566&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We also evaluated the effect of compounds <b>27</b> and <b>28</b> on reducing EGFR protein levels in HCC-827 and H3255 cells. As expected, these designed negative controls, which do not bind the E3 ligases VHL and CRBN, did not significantly induce the degradation of EGFR mutants in both cell lines (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>). The combination of high structural similarity (between compounds <b>27</b> and <b>28</b> and compounds <b>6</b> and <b>10</b>) and maintaining high binding affinity to WT and mutant EGFR (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>) makes compounds <b>27</b> and <b>28</b> excellent negative controls for studying the degradation activity of compounds <b>6</b> and <b>10</b>. In addition, compared with compounds <b>6</b> and <b>10</b>, these two negative controls (<b>27</b> and <b>28</b>) displayed lower potency at inhibiting EGFR autophosphorylation (p-EGFR) and its downstream signaling (e.g., AKT phosphorylation (p-AKT)), suggesting that the pharmacological degradation of EGFR can enhance the inhibition of the downstream signaling.</div><figure id="fig8" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/medium/jm9b01566_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0005.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Compounds <b>27</b> and <b>28</b> did not significantly reduce the protein levels of EGFR mutants in HCC-827 and H3255 cells. HCC-827 (A) and H3255 (B) cells were cultured to 80% confluent and then switched to a serum-free medium for 8 h before treating with indicated concentrations of the compounds for another 16 h.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01566&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We next performed time-course studies to assess the kinetics of the EGFR degradation and downstream signaling inhibition by compounds <b>6</b> and <b>10</b> in both HCC-827 and H3255 cells (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>). In HCC-827 cells, compound <b>6</b> reduced the EGFR protein level substantially (∼50%) after a 4 h treatment, while the robust inhibition of p-AKT was observed after a 1 h treatment. The near-complete degradation of the mutant EGFR by compound <b>6</b> was achieved around 12 h, and the complete degradation effect could sustain to 24 h. Compound <b>10</b> displayed a similar trend, reducing the half-maximum of the EGFR protein level after 3.5 h and achieving the maximum degradation after a 12 h treatment. In H3255 cells, compounds <b>6</b> and <b>10</b> reduced the EGFR protein level and inhibited downstream signaling also in a time-dependent manner and with similar kinetics. The rapid inhibition of p-AKT is most likely due to the inhibitory but not degradation activity of these compounds as they maintained high binding affinities to WT and mutant EGFR (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>).</div><figure id="fig9" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/medium/jm9b01566_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0006.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Compounds <b>6</b> and <b>10</b> rapidly and time-dependently elicit the degradation of EGFR mutants in lung cancer cells. A final concentration of 100 nM compound <b>6</b> or <b>10</b> was added to HCC-827 cells (A) and H3255 cells (B) after 8 h of serum starvation. At each time point, cells were collected for Western blot. The time needed for the degradation of 50% protein (<i>t</i><sub>1/2</sub>) was determined by the intensity of Western bands and normalized to GAPDH. Error bars represent the standard errors in three independent experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01566&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Solid tumor tissues often experience serum deprivation due to rapid proliferation and aberrant angiogenesis. We therefore opted to evaluate the effect of our degraders during serum starvation. As shown in <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>, degraders <b>6</b> and <b>10</b> showed a more pronounced effect on EGFR degradation after an 8 h treatment in a fetal bovine serum (FBS)-free medium in both HCC-827 and H3255 cells. It is unclear why serum starvation enhanced the degradation effect of these compounds. One plausible mechanism is that serum starvation may promote EGFR internalization,<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> which facilitates the degrader-mediated binding of the substrate EGFR to the cytoplasmic E3 ligases.</div><figure id="fig10" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/medium/jm9b01566_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0007.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Serum starvation enhances the EGFR degradation induced by compounds <b>6</b> and <b>10</b>. Cultured HCC-827 cells (A) and H3255 cells (B) were switched to a serum-free/serum-containing medium for 8 h and then treated with the indicated concentrations of the degraders for 16 h.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01566&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We next investigated the mechanism of action of compounds <b>6</b> and <b>10</b> in H3255 cells using a set of rescue assays (<a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>A). Compared to DMSO control, a 2 h pretreatment of MLN4924,<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> an NEDD8-activating enzyme inhibitor that blocks cullins neddylation and hence inactivating cullin RING E3 ligases, blocked the EGFR degradation induced by compounds <b>6</b> and <b>10</b>. Pretreatment of cells with the VHL inhibitor VH-298,<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> negative control compound <b>27</b>, or parental EGFR inhibitor gefitinib also blocked the EGFR degradation induced by the VHL-recruiting degrader <b>6</b>. Similarly, pretreatment with the CRBN E3 ligase binder thalidomide,<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> negative control compound <b>28</b>, or gefitinib also blocked compound <b>10</b>’s EGFR degradation effect in H3255 cells. Taken together, these results confirmed that the EGFR degradation effect induced by compounds <b>6</b> and <b>10</b> was mediated through the E3 ligase (VHL/CRBN)/ubiquitination pathway.</div><figure id="fig11" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/medium/jm9b01566_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0015.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. EGFR degradation effect of compounds <b>6</b> and <b>10</b> is mediated through the E3 ubiquitin ligase components VHL and CRBN, respectively, and is reversible upon washout. (A) H3255 cells were starved for 4 h and then pretreated with the indicated chemicals for 2 h before degrader (100 nM) treatment for another 8 h. The final concentrations of the compounds used for pretreatment are DMSO (0.1% v/v), MLN4924 (1 μM), VH-298 (10 μM), thalidomide (10 μM), negative controls (compound <b>27</b> for degrader <b>6</b> and compound <b>28</b> for degrader <b>10</b>, 10 μM), and gefitinib (10 μM). (B) HCC-827 and H3255 cells were starved for 8 h before a 12 h degrader treatment, then washed with PBS three times, and harvested at the indicated time points for Western blot analysis.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01566&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">To assess the reversibility of the EGFR degradation effect induced by compounds <b>6</b> and <b>10</b>, we performed washout experiments. In HCC-827 cells, pretreated with compounds <b>6</b> and <b>10</b> for 12 h and washed with fresh medium, the protein level was recovered after 24 h (<a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>B), indicating that the EGFR degradation effect mediated by compounds <b>6</b> and <b>10</b> is reversible. We also observed that p-EGFR was recovered after 24 h and p-AKT was recovered after 48 h. Similarly, the effect of compounds <b>6</b> and <b>10</b> on the EGFR degradation was reversible in H3255 cells. Interestingly, slightly prolonged EGFR degradation and EGFR autophosphorylation effects were observed in H3255 cells for both degraders, suggesting that HCC3255 cells might be more sensitive to compounds <b>6</b> and <b>10</b>.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Compounds <b>6</b> and <b>10</b> Inhibit Proliferation of Lung Cancer Cells</h3><div class="NLM_p">We next evaluated cell growth inhibition effects of EGFR degraders in H3255 cells. The VHL-recruiting degrader <b>6</b> effectively inhibited the growth of H3255 cells and was significantly more potent than its control compound <b>27</b> (<a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a></a>), indicating that the cell growth inhibition effect of compound <b>6</b> is at least partially due to its EGFR degradation activity. Compound <b>6</b> was also more potent than the previously reported PROTAC3. The CRBN-recruiting degrader <b>10</b> also inhibited the growth of H3255 cell, albeit it was not as potent as compound <b>6</b> and PROTAC3. A potential contributor to the lower potency of compound <b>10</b> may be that compound <b>10</b> was less effective in degrading mutant EGFR than compound <b>6</b> and PROTAC3 in H3255 cells (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>). Compared with the FDA-approved gefitinib, compounds <b>6</b> and <b>10</b> and PROTAC3 were less potent at cell growth inhibition, which is likely due to lower cell permeability of these degraders. Nevertheless, compounds <b>6</b> and <b>10</b> were more effective at inhibiting cell proliferation than their corresponding controls <b>27</b> and <b>28</b>. It is worth to note that these two pairs of compounds (compounds <b>6</b> and <b>27</b> and compounds <b>10</b> and <b>28</b>) have very similar physicochemical properties due to their very high structural similarities.</div><figure id="fig12" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/medium/jm9b01566_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0008.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 12. Compounds <b>6</b> and <b>10</b> inhibit the proliferation of lung cancer H3255 cells. H3255 cells were seeded to 96-well plates a night before at 4000 cells/well and then treated with serial dilutions of degraders or controls for 3 days. Cell number/growth was measured using Cell Counting Kit-8 and error bars indicating the standard errors in three independent experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01566&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Compounds <b>6</b> and <b>10</b> Are Highly Selective EGFR Degraders</h3><div class="NLM_p">To assess the selectivity of compounds <b>6</b> and <b>10</b> across the proteome, we performed the mass spectrometry (MS)-based label-free quantitative (LFQ) proteomics analysis in HCC-827 cells (<a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>). A total of 57 unique peptides of EGFR were detected with over 50% sequence coverage. For EGFR, the log 2 ratio of LFQ intensity value for compound <b>6</b> or compound <b>10</b> treatment vs DMSO control was about −2 with the highest confidence among all identified proteins (4794 proteins from the compound <b>6</b> experiment and 4446 proteins from the compound <b>10</b> experiment). Over 75% of EGFR protein was degraded upon compound <b>6</b> or compound <b>10</b> treatment. The study also identified several other proteins being potentially downregulated after degrader treatment, but with lower fold change or confidence level. Overall, our global proteomic study results indicate that degraders <b>6</b> and <b>10</b> are highly selective for EGFR.</div><figure id="fig13" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Figure 13</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/medium/jm9b01566_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0016.jpeg" id="gr15" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 13. Global proteomic analyses indicate that compounds <b>6</b> and <b>10</b> are highly selective EGFR degraders. HCC-827 cells were starved for 8 h and treated with compound <b>6</b> (A) or compound <b>10</b> (B) at 100 nM for 16 h and then collected for liquid chromatography–mass spectrometry (LC–MS) analyses. Label-free quantification (LFQ) was used to calculate peptide intensity; the top panels showed the relative abundance of EGFR (LFQ intensity value) between the degrader-treated and control groups. Error bars indicate the standard errors in three independent experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01566&amp;id=fig13"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Compounds <b>6</b> and <b>10</b> Are Bioavailable in Mice</h3><div class="NLM_p">We next assessed in vivo pharmacokinetic (PK) properties of degraders <b>6</b> and <b>10</b>. Each compound was administrated to six mice via a single intraperitoneal (IP) injection at a dose of 50 mg/kg. We were pleased to find that high plasma concentrations of compound <b>6</b> were achieved (<a data-tab="pane-pcw-Figures" href="#fig14" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig14" id="" class=" internalNav">14</a></a>A). In particular, the high plasma concentrations (approximately 5 μM) were maintained over 8 h, and approximately 1 μM plasma concentration was achieved at 24 h post dosing. The high plasma concentrations and relatively long half-life of compound <b>6</b> are expected to be sufficient to induce EGFR degradation in vivo. To the best of our knowledge, compound <b>6</b> is the first EGFR degrader that has good in vivo PK properties and is suitable for in vivo efficacy studies. Compound <b>10</b> was also bioavailable in mice via IP injection, albeit the plasma concentrations of compound <b>10</b> decreased by about 10- to 100-fold in comparison to those of compound <b>6</b> (<a data-tab="pane-pcw-Figures" href="#fig14" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig14" id="" class=" internalNav">14</a></a>B). The highest plasma concentration (approximately 500 nM) was achieved at 8 h post dosing for compound <b>10</b>. Importantly, both degraders were well tolerated, and no clinical signs were observed for the mice treated with either compound.</div><figure id="fig14" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Figure 14</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/medium/jm9b01566_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0009.jpeg" id="gr16" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 14. EGFR degraders <b>6</b> and <b>10</b> are bioavailable in mouse PK studies. Plasma concentrations of compound <b>6</b> (<b>A</b>) and compound <b>10</b> (<b>B</b>) following a single 50 mg/kg IP injection over 24 h in male Swiss Albino mice. Experiments were carried out in biological triplicates, with points representing mean concentrations ± SEM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01566&amp;id=fig14"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_77447" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_77447" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In summary, we have developed a novel gefitinib-based VHL-recruiting EGFR degrader (compound <b>6</b>) and a first-in-class gefitinib-based CRBN-recruiting EGFR degrader (compound <b>10</b>). Compounds <b>6</b> and <b>10</b> potently induced the degradation of EGFR mutants but not EGFR WT and inhibited the downstream signaling in cells. We conducted the mechanism of action studies, which confirmed that the effect of compounds <b>6</b> and <b>10</b> on EGFR degradation was mediated by the E3 ubiquitin ligases VHL and CRBN, respectively. Our kinetic and washout study results indicate that the EGFR degradation effect of compounds <b>6</b> and <b>10</b> has fast kinetics and is reversible. We also found that serum starvation enhanced the EGFR degradation effect of compounds <b>6</b> and <b>10</b>. In cell proliferation assays, compounds <b>6</b> and <b>10</b> effectively inhibited the growth of lung cancer cells. We also performed global proteomic analyses and found that compounds <b>6</b> and <b>10</b> were highly selective for EGFR. In addition, while both compounds <b>6</b> and <b>10</b> were bioavailable in mice, compound <b>6</b> is the first EGFR degrader that has sufficient in vivo PK properties and is suitable for in vivo efficacy studies. Furthermore, compound <b>6</b> was more potent than the previously reported VHL-recruiting EGFR degrader PROTAC3 in inhibiting the proliferation of lung cancer cells. Finally, we also developed compounds <b>27</b> and <b>28</b>, which do not bind the corresponding E3 ligase as degrader negative controls for compounds <b>6</b> and <b>10</b>. Compounds <b>27</b> and <b>28</b> maintained high binding affinities to WT and mutant EGFR, similar to compounds <b>6</b> and <b>10</b>, but did not induce EGFR degradation in cells. The very high structural similarities of compounds <b>27</b> and <b>28</b> with compounds <b>6</b> and <b>10</b> make compounds <b>27</b> and <b>28</b> excellent degrader negative controls for compounds <b>6</b> and <b>10</b>. Overall, degraders <b>6</b> and <b>10</b> and their negative controls <b>27</b> and <b>28</b> are well-characterized chemical tools, and we provide these tools to the research community for further investigating the roles of EGFR in pathophysiology. Further optimization of these EGFR degraders will likely result in improved lead compounds for exploring a potential novel therapeutic strategy for NSCLC and overcoming drug resistance. Finally, our study has also provided further evidence that the PROTAC technology can be applied to targeting RTKs.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_27443" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_27443" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Chemistry General Procedures</h3><div class="NLM_p">All chemical reagents were purchased from commercial vendors and used in syntheses without the need for further purification. A Teledyne ISCO Combi<i>Flash</i> Rf<sup>+</sup> instrument equipped with a variable wavelength UV detector and a fraction collector was used to conduct flash column chromatography. HP C18 Redi<i>Sep</i> Rf reverse-phase silica columns were also used for the purification of certain polar products. All compounds received final purification with preparative high-performance liquid chromatography (HPLC) on an Agilent Prep 1200 series with the UV detector set to 254 nm. Separation was performed at room temperature with a flow rate of 40 mL/min. Samples were injected into a Phenomenex Luna 750 × 30 mm, 5 μm C18 column, with the gradient program set to 10% of methanol (A) in H<sub>2</sub>O containing 0.1% TFA (B) progressing to 100% of methanol or acetonitrile (A). All compounds assessed for biological activity were assured to have >95% purity after HPLC purification. HPLC spectra were obtained for all compounds using an Agilent 1200 series system with a DAD detector and a 2.1 mm × 150 mm Zorbax 300SB-C18 5 μm column for chromatography. Samples (0.5 μL) were injected onto a C18 column at room temperature, and the flow rate was set to 0.4 mL/min. Chromatography was performed with the solvents as follows: water containing 0.1% formic acid was designated as solvent A, while acetonitrile containing 0.1% formic acid was designated as solvent B. The linear gradient was set such that 1% B was used from 0 to 1 min, 1–99% B from 1 to 4 min, and 99% B from 4 to 8 min. High-resolution mass spectra (HRMS) data was acquired in positive ion mode using an Agilent G1969A API-TOF with an electrospray ionization (ESI) source. All compounds were also characterized using either a Bruker (Billerica, MA) DRX-600 NMR spectrometer (600 MHz, <sup>1</sup>H NMR) or a Bruker DXI 800 MHz spectrometer (800 MHz <sup>1</sup>H NMR, 201 MHz, <sup>13</sup>C NMR). Chemical shifts for all compounds are reported in units of parts per million (ppm, δ) relative to residual solvent peaks. <sup>1</sup>H NMR data are reported in the following format: chemical shift, multiplicity (s = singlet, d = doublet, t = triplet, q = quartet, m = multiplet), coupling constant, and integration.</div><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i15" class="anchor-spacer"></div><h4 class="article-section__title" id="_i15"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(3-(3-(4-(3-((4-((3-Chloro-4-fluorophenyl)amino)-7-methoxyquinazolin-6-yl)oxy)propyl)piperazin-1-yl)-3-oxopropoxy)propanamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>1</b>)</h4><div class="NLM_p last">To the solution of <i>N</i>-(3-chloro-4-fluorophenyl)-7-methoxy-6-(3-(piperazin-1-yl)propoxy)quinazolin-4-amine (<b>13</b>, 11.0 mg, 0.02 mmol) in DMSO (1 mL) were added 3-(3-(((<i>S</i>)-1-((2<i>S</i>,4<i>R</i>)-4-hydroxy-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl) pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-3-oxopropoxy)propanoic acid (<b>15</b>, 11.5 mg, 0.02 mmol, 1.0 equiv), EDCI (6.0 mg, 0.03 mmol, 1.5 equiv), HOAt (4.2 mg, 0.03 mmol, 1.5 equiv), and NMM (6.1 mg, 0.06 mmol, 3.0 equiv). After being stirring overnight at room temperature, the resulting mixture was purified by preparative HPLC (10–100% methanol/0.1% TFA in H<sub>2</sub>O) to afford the title compound <b>1</b> as white solid in TFA salt form (10.1 mg, 82% yield). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 8.91 (d, <i>J</i> = 4.0 Hz, 1H), 8.74 (s, 1H), 7.97 (d, <i>J</i> = 4.0 Hz, 1H), 7.95–7.85 (m, 2H), 7.69–7.62 (m, 1H), 7.50–7.33 (m, 4H), 7.26 (d, <i>J</i> = 4.1 Hz, 1H), 4.66 (td, <i>J</i> = 8.5, 7.6, 4.8 Hz, 1H), 4.59–4.44 (m, 3H), 4.43–4.29 (m, 4H), 4.07 (d, <i>J</i> = 4.2 Hz, 3H), 3.90 (d, <i>J</i> = 11.1 Hz, 1H), 3.83–3.55 (m, 11H), 3.50 (q, <i>J</i> = 7.5, 6.5 Hz, 2H), 2.80–2.60 (m, 2H), 2.54 (q, <i>J</i> = 5.5 Hz, 3H), 2.50–2.37 (m, 5H), 2.23 (dd, <i>J</i> = 13.6, 7.7 Hz, 1H), 2.07 (ddt, <i>J</i> = 13.5, 9.4, 4.6 Hz, 1H), 1.03 (s, 9H). HPLC 98% pure, <i>t</i><sub>R</sub> = 3.77 min; HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calculated for C<sub>50</sub>H<sub>61</sub>ClFN<sub>9</sub>O<sub>8</sub>S<sup>+</sup> 1002.4109, found 1002.4141.</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(<i>tert</i>-Butyl)-16-(4-(3-((4-((3-chloro-4-fluorophenyl)amino)-7-methoxyquinazolin-6-yl)oxy)propyl)piperazin-1-yl)-4,16-dioxo-7,10,13-trioxa-3-azahexadecanoyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>2</b>)</h4><div class="NLM_p last">Compound <b>2</b> was prepared following the general procedure for preparing compound <b>1</b> from <i>N</i>-(3-chloro-4-fluorophenyl)-7-methoxy-6-(3-(piperazin-1-yl)propoxy)quinazolin-4-amine (<b>13</b>, 11.0 mg, 0.02 mmol) and (<i>S</i>)-15-((2<i>S</i>,4<i>R</i>)-4-hydroxy-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl)pyrrolidine-1-carbonyl)-16,16-dimethyl-13-oxo-4,7,10-trioxa-14-azaheptadecanoic acid (<b>16</b>, 13.3 mg, 0.02 mmol). White solid (17.6 mg, 81% yield). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 8.98 (s, 1H), 8.74 (s, 1H), 7.99 (s, 1H), 7.93 (dd, <i>J</i> = 6.6, 2.7 Hz, 1H), 7.66 (ddd, <i>J</i> = 9.0, 4.2, 2.7 Hz, 1H), 7.51–7.33 (m, 5H), 7.28 (s, 1H), 4.64 (s, 1H), 4.60–4.46 (m, 4H), 4.40–4.33 (m, 3H), 4.08 (s, 3H), 3.89 (dd, <i>J</i> = 11.1, 4.3 Hz, 1H), 3.83–3.67 (m, 6H), 3.61 (pd, <i>J</i> = 10.7, 9.6, 5.6 Hz, 14H), 3.48 (t, <i>J</i> = 7.4 Hz, 2H), 2.62–2.50 (m, 2H), 2.50–2.39 (m, 7H), 2.22 (ddt, <i>J</i> = 11.9, 7.7, 2.1 Hz, 1H), 2.07 (ddt, <i>J</i> = 13.3, 9.0, 4.2 Hz, 1H), 1.03 (s, 9H). HPLC 98% pure, <i>t</i><sub>R</sub> = 4.89 min; HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calculated for C<sub>54</sub>H<sub>70</sub>ClFN<sub>9</sub>O<sub>10</sub>S<sup>+</sup> 1090.4633, found 1090.4536.</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(<i>tert</i>-Butyl)-22-(4-(3-((4-((3-chloro-4-fluorophenyl)amino)-7-methoxyquinazolin-6-yl)oxy)propyl)piperazin-1-yl)-4,22-dioxo-7,10,13,16,19-pentaoxa-3-azadocosanoyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>3</b>)</h4><div class="NLM_p last">Compound <b>3</b> was prepared following the general procedure for preparing compound <b>1</b> from <i>N</i>-(3-chloro-4-fluorophenyl)-7-methoxy-6-(3-(piperazin-1-yl)propoxy)quinazolin-4-amine (<b>13</b>, 11.0 mg, 0.02 mmol) and (<i>S</i>)-19-((2<i>S</i>,4<i>R</i>)-4-hydroxy-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl) pyrrolidine-1-carbonyl)-20,20-dimethyl-17-oxo-3,6,9,12,15-pentaoxa-18-azahenicosanoic acid (<b>17</b>, 15 mg, 0.02 mmol). White solid (9.5 mg, 40% yield). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 8.92 (s, 1H), 8.74 (s, 1H), 7.99 (s, 1H), 7.93 (dd, <i>J</i> = 6.6, 2.7 Hz, 1H), 7.66 (ddd, <i>J</i> = 8.9, 4.2, 2.6 Hz, 1H), 7.49–7.33 (m, 5H), 7.28 (s, 1H), 4.63 (s, 1H), 4.59–4.44 (m, 3H), 4.41–4.31 (m, 4H), 4.09 (s, 3H), 3.88 (d, <i>J</i> = 10.9 Hz, 1H), 3.83–3.66 (m, 6H), 3.66–3.52 (m, 22H), 3.49 (t, <i>J</i> = 7.4 Hz, 3H), 2.57 (ddd, <i>J</i> = 15.0, 7.3, 5.2 Hz, 1H), 2.51–2.38 (m, 7H), 2.22 (ddt, <i>J</i> = 11.7, 7.6, 2.0 Hz, 1H), 2.07 (ddd, <i>J</i> = 13.3, 9.2, 4.4 Hz, 1H), 1.03 (s, 9H). HPLC 99% pure, <i>t</i><sub>R</sub> = 3.81 min; HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calculated for C<sub>58</sub>H<sub>78</sub>ClFN<sub>9</sub>O<sub>12</sub>S<sup>+</sup> 1178.5158, found 1178.5191.</div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(4-(4-(3-((4-((3-Chloro-4-fluorophenyl)amino)-7-methoxyquinazolin-6-yl)oxy)propyl)piperazin-1-yl)-4-oxobutanamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>4</b>)</h4><div class="NLM_p last">Compound <b>4</b> was prepared following the general procedure for preparing compound <b>1</b> from <i>N</i>-(3-chloro-4-fluorophenyl)-7-methoxy-6-(3-(piperazin-1-yl)propoxy)quinazolin-4-amine (<b>13</b>, 11.0 mg, 0.02 mmol) and 4-(((<i>S</i>)-1-((2<i>S</i>,4<i>R</i>)-4-hydroxy-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl) pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-4-oxobutanoic acid (<b>18</b>, 10.6 mg, 0.02 mmol). White solid (9.0 mg, 47% yield). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 8.95 (s, 1H), 8.74 (s, 1H), 8.03–7.91 (m, 2H), 7.70–7.62 (m, 1H), 7.53–7.33 (m, 5H), 7.28 (s, 1H), 4.60 (d, <i>J</i> = 6.5 Hz, 1H), 4.58–4.46 (m, 2H), 4.46–4.30 (m, 4H), 4.09 (s, 5H), 3.95–3.74 (m, 2H), 3.70–3.39 (m, 4H), 2.84–2.55 (m, 6H), 2.55–2.39 (m, 6H), 2.22 (dd, <i>J</i> = 13.2, 7.7 Hz, 1H), 2.08 (ddd, <i>J</i> = 13.4, 9.2, 4.6 Hz, 2H), 1.04 (s, 9H). HPLC 96% pure, <i>t</i><sub>R</sub> = 4.99 min; HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calculated for C<sub>48</sub>H<sub>58</sub>ClFN<sub>9</sub>O<sub>7</sub>S<sup>+</sup> 958.3847, found 958.3788.</div></div><div id="sec4_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(7-(4-(3-((4-((3-Chloro-4-fluorophenyl)amino)-7-methoxyquinazolin-6-yl)oxy)propyl)piperazin-1-yl)-7-oxoheptanamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>5</b>)</h4><div class="NLM_p last">Compound <b>5</b> was prepared following the general procedure for preparing compound <b>1</b> from <i>N</i>-(3-chloro-4-fluorophenyl)-7-methoxy-6-(3-(piperazin-1-yl)propoxy)quinazolin-4-amine (<b>13</b>, 11.0 mg, 0.02 mmol) and 7-(((<i>S</i>)-1-((2<i>S</i>,4<i>R</i>)-4-hydroxy-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl) pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-7-oxoheptanoic acid (<b>19</b>, 11.5 mg, 0.02 mmol). White solid (18.2 mg, 91% yield). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 8.94 (s, 1H), 8.74 (s, 1H), 8.00 (s, 2H), 7.94 (dd, <i>J</i> = 6.6, 2.7 Hz, 1H), 7.70–7.63 (m, 1H), 7.50–7.34 (m, 4H), 7.29 (d, <i>J</i> = 4.5 Hz, 1H), 4.64 (s, 1H), 4.61–4.46 (m, 2H), 4.46–4.32 (m, 3H), 4.08 (s, 5H), 3.90 (d, <i>J</i> = 11.0 Hz, 1H), 3.80 (dd, <i>J</i> = 10.9, 4.0 Hz, 1H), 3.48 (t, <i>J</i> = 7.3 Hz, 2H), 2.58–2.38 (m, 9H), 2.36–2.16 (m, 2H), 2.14–2.03 (m, 1H), 1.69–1.57 (m, 6H), 1.49–1.32 (m, 6H), 1.03 (s, 9H). HPLC 98% pure, <i>t</i><sub>R</sub> = 3.81 min; HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calculated for C<sub>51</sub>H<sub>64</sub>ClFN<sub>9</sub>O<sub>7</sub>S<sup>+</sup> 1000.4316, found 1000.4342.</div></div><div id="sec4_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(11-(4-(3-((4-((3-Chloro-4-fluorophenyl)amino)-7-methoxyquinazolin-6-yl)oxy)propyl)piperazin-1-yl)-11-oxoundecanamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>6</b>)</h4><div class="NLM_p last">Compound <b>6</b> was prepared following the general procedure for preparing compound <b>1</b> from <i>N</i>-(3-chloro-4-fluorophenyl)-7-methoxy-6-(3-(piperazin-1-yl)propoxy)quinazolin-4-amine (<b>13</b>, 11.0 mg, 0.02 mmol) and 11-(((<i>S</i>)-1-((2<i>S</i>,4<i>R</i>)-4-hydroxy-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl)pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-11-oxoundecanoic acid (<b>20</b>, 13.6 mg, 0.02 mmol). White solid (17.4 mg, 82% yield). <sup>1</sup>H NMR (800 MHz, CD<sub>3</sub>OD) δ 9.12 (s, 1H), 8.76 (s, 1H), 8.02 (s, 1H), 7.96 (dd, <i>J</i> = 6.7, 2.7 Hz, 1H), 7.69 (dt, <i>J</i> = 7.4, 3.3 Hz, 1H), 7.49 (d, <i>J</i> = 7.8 Hz, 2H), 7.44 (d, <i>J</i> = 7.8 Hz, 2H), 7.36 (t, <i>J</i> = 8.9 Hz, 1H), 7.33 (s, 1H), 4.66 (s, 1H), 4.63–4.58 (m, 1H), 4.58–4.49 (m, 2H), 4.43–4.35 (m, 3H), 4.11 (s, 3H), 3.93 (d, <i>J</i> = 10.9 Hz, 1H), 3.83 (dd, <i>J</i> = 10.9, 4.0 Hz, 1H), 3.78–3.55 (m, 4H), 3.51 (t, <i>J</i> = 7.4 Hz, 2H), 3.37 (s, 2H), 3.30–2.97 (m, 4H), 2.56–2.41 (m, 7H), 2.33 (dt, <i>J</i> = 14.8, 7.6 Hz, 1H), 2.29–2.20 (m, 2H), 2.11 (ddd, <i>J</i> = 13.2, 9.1, 4.5 Hz, 1H), 1.69–1.56 (m, 4H), 1.44–1.28 (m, 8H), 1.06 (s, 9H). <sup>13</sup>C NMR (201 MHz, CD<sub>3</sub>OD) δ 174.66, 173.08, 172.84, 170.99, 158.76, 157.61, 156.78, 155.55, 152.02, 150.04, 148.37, 139.10, 135.75, 133.68, 128.97, 127.62, 126.47, 124.48, 120.40 (d, <i>J</i> (C, F) = 18.1 Hz, C-F), 116.37 (d, <i>J</i> (C, F) = 24.1 Hz, C-H), 107.34, 103.63, 99.35, 69.68, 66.79, 59.46, 57.59, 56.63, 56.14, 54.75, 51.81, 51.52, 48.47, 47.41, 42.30, 38.31, 37.55, 35.28, 35.19, 32.22, 29.05, 28.96, 28.88, 25.66, 25.60, 24.78, 23.41, 14.27. HPLC 99% pure, <i>t</i><sub>R</sub> = 4.00 min; HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calculated for C<sub>55</sub>H<sub>72</sub>ClFN<sub>9</sub>O<sub>7</sub>S<sup>+</sup> 1056.4942, found 1056.4626.</div></div><div id="sec4_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> 3-(4-(3-((4-((3-Chloro-4-fluorophenyl)amino)-7-methoxyquinazolin-6-yl)oxy)propyl)piperazin-1-yl)-<i>N</i>-(2-(2-((2-(2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)ethoxy)ethyl)propenamide (<b>7</b>)</h4><div class="NLM_p last">Compound <b>7</b> was prepared following the general procedure for preparing compound <b>1</b> from 3-(4-(3-((4-((3-chloro-4-fluorophenyl)amino)-7-methoxyquinazolin-6-yl)oxy)propyl)piperazin-1-yl)propanoic acid (<b>14</b>, 12.0 mg, 0.02 mmol) and <b>21</b> (8.2 mg, 0.02 mmol). Yellow solid (6.4 mg, 37% yield). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 8.73 (s, 1H), 7.96–7.92 (m, 2H), 7.69–7.63 (m, 1H), 7.54 (dd, <i>J</i> = 8.6, 7.1 Hz, 1H), 7.37 (t, <i>J</i> = 8.9 Hz, 1H), 7.26 (s, 1H), 7.08 (d, <i>J</i> = 8.6 Hz, 1H), 7.02 (d, <i>J</i> = 7.0 Hz, 1H), 5.05 (dd, <i>J</i> = 12.7, 5.5 Hz, 1H), 4.32 (t, <i>J</i> = 5.6 Hz, 2H), 4.10–4.04 (m, 4H), 3.72 (t, <i>J</i> = 5.1 Hz, 2H), 3.60 (t, <i>J</i> = 5.2 Hz, 2H), 3.49 (t, <i>J</i> = 5.1 Hz, 2H), 3.46–3.39 (m, 2H), 3.39–3.32 (m, 2H), 3.28–3.07 (m, 8H), 2.85 (ddd, <i>J</i> = 17.3, 13.8, 5.3 Hz, 1H), 2.77–2.62 (m, 3H), 2.59 (t, <i>J</i> = 6.6 Hz, 2H), 2.35–2.27 (m, 2H), 2.15–2.05 (m, 1H). HPLC 98% pure, <i>t</i><sub>R</sub> = 3.68 min; HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calculated for C<sub>42</sub>H<sub>47</sub>ClFN<sub>9</sub>O<sub>8</sub><sup>+</sup> 860.3293, found 860.3317.</div></div><div id="sec4_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> 3-(4-(3-((4-((3-Chloro-4-fluorophenyl)amino)-7-methoxyquinazolin-6-yl)oxy)propyl)piperazin-1-yl)-<i>N</i>-(17-((2-(2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)-3,6,9,12,15-pentaoxaheptadecyl)propenamide (<b>8</b>)</h4><div class="NLM_p last">Compound <b>8</b> was prepared following the general procedure for preparing compound <b>1</b> from 3-(4-(3-((4-((3-chloro-4-fluorophenyl)amino)-7-methoxyquinazolin-6-yl)oxy)propyl)piperazin-1-yl)propanoic acid (<b>14</b>, 12.0 mg, 0.02 mmol) and <b>22</b> (11.6 mg, 0.02 mmol). Yellow solid (12.3 mg, 59% yield). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 8.73 (d, <i>J</i> = 1.7 Hz, 1H), 7.97–7.90 (m, 2H), 7.66 (ddd, <i>J</i> = 9.0, 4.3, 2.4 Hz, 1H), 7.51 (ddd, <i>J</i> = 8.7, 7.0, 1.6 Hz, 1H), 7.36 (td, <i>J</i> = 8.8, 1.6 Hz, 1H), 7.26 (d, <i>J</i> = 1.6 Hz, 1H), 7.04 (dd, <i>J</i> = 8.7, 1.6 Hz, 1H), 7.00 (dd, <i>J</i> = 7.1, 1.6 Hz, 1H), 5.04 (ddd, <i>J</i> = 12.6, 5.5, 1.7 Hz, 1H), 4.34 (t, <i>J</i> = 5.6 Hz, 2H), 4.08 (s, 3H), 3.71 (td, <i>J</i> = 5.2, 1.6 Hz, 2H), 3.68–3.56 (m, 18H), 3.54 (td, <i>J</i> = 5.4, 1.6 Hz, 2H), 3.46 (q, <i>J</i> = 8.5, 6.9 Hz, 5H), 3.40–3.32 (m, 7H), 3.25 (t, <i>J</i> = 6.8 Hz, 2H), 2.90–2.79 (m, 1H), 2.78–2.59 (m, 4H), 2.41–2.30 (m, 2H), 2.14–2.06 (m, 1H). HPLC 97% pure, <i>t</i><sub>R</sub> = 3.77 min; HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calculated for C<sub>50</sub>H<sub>63</sub>ClFN<sub>9</sub>O<sub>12</sub><sup>+</sup> 1036.4269, found 1036.4352.</div></div><div id="sec4_1_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> 3-(4-(3-((4-((3-Chloro-4-fluorophenyl)amino)-7-methoxyquinazolin-6-yl)oxy)propyl)piperazin-1-yl)-<i>N</i>-(6-((2-(2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)hexyl)propenamide (<b>9</b>)</h4><div class="NLM_p last">Compound <b>9</b> was prepared following the general procedure for preparing compound <b>1</b> from 3-(4-(3-((4-((3-chloro-4-fluorophenyl)amino)-7-methoxyquinazolin-6-yl)oxy)propyl)piperazin-1-yl)propanoic acid (<b>14</b>, 12.0 mg, 0.02 mmol) and <b>23</b> (9.2 mg, 0.02 mmol). Yellow solid (14.1 mg, 85% yield). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 8.73 (s, 1H), 7.97–7.90 (m, 2H), 7.66 (ddd, <i>J</i> = 8.9, 4.1, 2.6 Hz, 1H), 7.51 (dd, <i>J</i> = 8.6, 7.1 Hz, 1H), 7.36 (t, <i>J</i> = 8.9 Hz, 1H), 7.26 (s, 1H), 7.00 (dd, <i>J</i> = 11.4, 7.8 Hz, 2H), 5.04 (dd, <i>J</i> = 12.8, 5.5 Hz, 1H), 4.35 (t, <i>J</i> = 5.5 Hz, 2H), 4.08 (s, 3H), 3.68–3.43 (m, 10H), 3.41 (t, <i>J</i> = 7.1 Hz, 2H), 3.34 (t, <i>J</i> = 6.7 Hz, 2H), 3.21 (t, <i>J</i> = 7.0 Hz, 2H), 2.84 (ddd, <i>J</i> = 17.6, 14.0, 5.3 Hz, 1H), 2.76–2.63 (m, 4H), 2.42–2.35 (m, 2H), 2.13–2.05 (m, 1H), 1.71–1.62 (m, 2H), 1.59–1.50 (m, 2H), 1.50–1.38 (m, 4H). HPLC 95% pure, <i>t</i><sub>R</sub> = 3.84 min; HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calculated for C<sub>44</sub>H<sub>52</sub>ClFN<sub>9</sub>O<sub>7</sub><sup>+</sup> 872.3657, found 872.3655.</div></div><div id="sec4_1_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> 3-(4-(3-((4-((3-Chloro-4-fluorophenyl)amino)-7-methoxyquinazolin-6-yl)oxy)propyl)piperazin-1-yl)-<i>N</i>-(8-((2-(2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)octyl)propanamide (<b>10</b>)</h4><div class="NLM_p last">To the solution of 3-(4-(3-((4-((3-chloro-4-fluorophenyl)amino)-7-methoxyquinazolin-6-yl)oxy)propyl)piperazin-1-yl)propanoic acid (<b>4</b>, 10.4 mg, 0.02 mmol) in DMSO (1 mL) were added 4-((8-aminooctyl)oxy)-2-(2,6-dioxopiperidin-3-yl)isoindoline-1,3-dione (<b>26</b>, 8.0 mg, 0.02 mmol, 1.0 equiv), EDCI (6.0 mg, 0.03 mmol, 1.5 equiv), HOAt (4.2 mg, 0.03 mmol, 1.5 equiv), and NMM (6.1 mg, 0.06 mmol, 3.0 equiv). After being stirring overnight at room temperature, the resulting mixture was purified by preparative HPLC (10–100% methanol/0.1% TFA in H<sub>2</sub>O) to afford the title compound <b>10</b> as white solid in TFA salt form (16.8 mg, 93% yield). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 8.73 (s, 1H), 7.94 (s, 2H), 7.79–7.57 (m, 2H), 7.47–7.30 (m, 3H), 7.26 (s, 1H), 5.09 (dd, <i>J</i> = 12.8, 5.4 Hz, 1H), 4.35 (t, <i>J</i> = 5.5 Hz, 2H), 4.19 (t, <i>J</i> = 6.2 Hz, 2H), 4.08 (s, 3H), 3.61–3.37 (m, 11H), 3.20 (t, <i>J</i> = 7.0 Hz, 2H), 2.91–2.82 (m, 1H), 2.78–2.62 (m, 4H), 2.41–2.35 (m, 2H), 2.21–2.02 (m, 1H), 1.86–1.80 (m, 2H), 1.56–1.48 (m, 4H), 1.44–1.33 (m, 7H). <sup>13</sup>C NMR (201 MHz, <i>d</i>-DMSO) δ 173.29, 170.44, 168.83, 167.33, 165.80, 158.59, 156.85, 156.48, 154.84, 149.31(d, <i>J</i> (C, F) = 50.1 Hz, C-F), 137.52, 136.27, 134.73, 133.69, 126.99, 125.77, 120.26, 119.54, 119.44, 117.14 (d, <i>J</i> (C, F) = 23.1 Hz, C-H) 116.65, 115.60, 107.80, 106.00, 100.36, 69.28, 67.63, 57.03, 49.21, 49.05, 48.52, 39.14, 31.43, 29.94, 29.40, 29.11, 28.88, 26.83, 25.73, 23.69, 22.48. HPLC 98% pure, <i>t</i><sub>R</sub> = 4.22 min; HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calculated for C<sub>46</sub>H<sub>55</sub>ClFN<sub>8</sub>O<sub>8</sub><sup>+</sup> 901.3810, found 901.3916.</div></div><div id="sec4_1_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> <i>N</i>-(3-Chloro-4-fluorophenyl)-7-methoxy-6-(3-(piperazin-1-yl)propoxy)quinazolin-4-amine (<b>13</b>)</h4><div class="NLM_p last">To the suspension of commercially available 4-((3-chloro-4-fluorophenyl)amino)-7-methoxyquinazolin-6-ol (<b>11</b>, 942 mg, 2.94 mmol) and potassium carbonate (1.2 g, 8.82 mmol, 3.0 equiv) in DMF (20 mL) was added <i>tert</i>-butyl 4-(3-bromopropyl)piperazine-1-carboxylate (<b>12</b>, 903 mg, 2.94 mmol, 1.0 equiv). After stirring overnight at 80 °C, water was added, and the mixture was extracted with ethyl acetate (3 × 20 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. The crude product was purified by reverse-phase ISCO (10–100% methanol/0.1% TFA in H<sub>2</sub>O) to afford white solid in TFA salt form (1.56 g, 97% yield). This product was dissolved in DCM (8 mL) and TFA (8 mL). The resulting mixture was stirred for 30 min. Then, it was concentrated and purified by reverse-phase ISCO (10–100% methanol/0.1% TFA in H<sub>2</sub>O) to afford the title compound <b>3</b> as white solid in TFA salt form (1.25 g, 98% yield). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 8.74 (s, 1H), 7.98 (s, 1H), 7.66 (dt, <i>J</i> = 9.2, 3.4 Hz, 1H), 7.43–7.31 (m, 1H), 7.27 (d, <i>J</i> = 4.6 Hz, 2H), 4.37 (t, <i>J</i> = 5.6 Hz, 2H), 4.08 (s, 3H), 3.71–3.37 (m, 6H), 3.33–3.21 (m, 4H), 2.39 (t, <i>J</i> = 7.8 Hz, 2H). HPLC 98% pure, <i>t</i><sub>R</sub> = 3.40 min; HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calculated for C<sub>22</sub>H<sub>26</sub>ClFN<sub>5</sub>O<sub>2</sub><sup>+</sup> 446.1754, found 446.1743.</div></div><div id="sec4_1_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> 3-(4-(3-((4-((3-Chloro-4-fluorophenyl)amino)-7-methoxyquinazolin-6-yl)oxy)propyl)piperazin-1-yl)propanoic Acid (<b>14</b>)</h4><div class="NLM_p last">To the suspension of compound <b>13</b> (888 mg, 2 mmol) and potassium carbonate (828 mg, 6 mmol, 3.0 equiv) in DMF (10 mL) was added ethyl 3-bromopropanoate (724 mg, 4 mmol, 2.0 equiv). After being stirring overnight at 80 °C, water was added, and the mixture was extracted with ethyl acetate (3 × 15 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. The crude product was purified by reverse-phase ISCO (10–100% methanol/0.1% TFA in H<sub>2</sub>O) to afford white solid in TFA salt form (822 mg, 75% yield). This product was dissolved in THF (5 mL) and water (5 mL). After stirring overnight at ambient temperature, it was concentrated and purified by reverse-phase ISCO (10–100% methanol/0.1% TFA in H<sub>2</sub>O) to afford the title compound <b>14</b> as white solid in TFA salt form (834 mg, 98% yield). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 8.74 (s, 1H), 7.98 (s, 1H), 7.94 (dd, <i>J</i> = 6.6, 2.7 Hz, 1H), 7.72–7.59 (m, 1H), 7.37 (t, <i>J</i> = 8.9 Hz, 1H), 7.27 (s, 1H), 4.35 (t, <i>J</i> = 5.6 Hz, 2H), 4.09 (s, 3H), 3.38–3.32 (m, 4H), 3.27 (t, <i>J</i> = 7.3 Hz, 2H), 3.22–3.16 (m, 4H), 3.13 (t, <i>J</i> = 6.9 Hz, 2H), 2.70 (t, <i>J</i> = 6.9 Hz, 2H), 2.34 (t, <i>J</i> = 6.7 Hz, 2H). HPLC 97% pure, <i>t</i><sub>R</sub> = 3.56 min; HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calculated for C<sub>25</sub>H<sub>30</sub>ClFN<sub>5</sub>O<sub>4</sub><sup>+</sup> 518.1965, found 518.1976.</div></div><div id="sec4_1_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> 3-(3-(((<i>S</i>)-1-((2<i>S</i>,4<i>R</i>)-4-Hydroxy-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl) pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-3-oxopropoxy)propanoic Acid (<b>15</b>)</h4><div class="NLM_p last">To a solution of 3,3′-oxydipropionic acid (648 mg, 4 mmol) in DMSO (10 mL) were added <i>N</i>-methylmorpholine (NMM, 10 mmol), (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-amino-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (VHL-1, 860 mg, 2 mmol), 1-hydroxy-7-azabenzo-triazole (HOAt, 326 mg, 2.4 mmol), and 1-ethyl-3-(3-dimethylaminopropyl)carbodiimide (EDCI, 461 mg, 2.4 mmol) at 0 °C. The resulting reaction solution was stirred at 0 °C for 6 h and then at RT overnight. The progress of the reaction was monitored by LC/MS. After VHL-1 was totally consumed, the reaction was concentrated, and the resulting residue was purified by reverse-phase chromatography to yield the product (450 mg, 63%) as white solid. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 9.00 (s, 1H), 7.45 (d, <i>J</i> = 22.1 Hz, 4H), 4.64 (s, 1H), 4.61–4.44 (m, 3H), 4.36 (d, <i>J</i> = 15.4 Hz, 1H), 3.84 (dd, <i>J</i> = 57.3, 10.5 Hz, 2H), 3.75–3.56 (m, 4H), 2.60–2.39 (m, 7H), 2.24–2.17 (m, 1H), 2.11–2.03 (m, 1H), 1.03 (s, 9H). HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calculated for C<sub>28</sub>H<sub>39</sub>N<sub>4</sub>O<sub>7</sub>S, 575.2534; found: 575.2543.</div></div><div id="sec4_1_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> (<i>S</i>)-15-((2<i>S</i>,4<i>R</i>)-4-Hydroxy-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl) pyrrolidine-1-carbonyl)-16,16-dimethyl-13-oxo-4,7,10-trioxa-14-azaheptadecanoic Acid (<b>16</b>)</h4><div class="NLM_p last">(677 mg, 57%) as white solid. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 8.95 (s, 1H), 7.47 (d, <i>J</i> = 8.1 Hz, 2H), 7.42 (d, <i>J</i> = 8.1 Hz, 2H), 4.65 (s, 1H), 4.59–4.51 (m, 2H), 4.49 (s, 1H), 4.35 (d, <i>J</i> = 15.5 Hz, 1H), 3.89 (d, <i>J</i> = 11.1 Hz, 1H), 3.80 (dd, <i>J</i> = 10.9, 3.9 Hz, 1H), 3.76–3.67 (m, 4H), 3.66–3.54 (m, 8H), 2.60–2.50 (m, 3H), 2.50–2.43 (m, 4H), 2.21 (dd, <i>J</i> = 13.1, 7.6 Hz, 1H), 2.08 (ddd, <i>J</i> = 13.3, 9.1, 4.5 Hz, 1H), 1.04 (s, 9H). HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calculated for C<sub>32</sub>H<sub>47</sub>N<sub>4</sub>O<sub>9</sub>S, 663.3058; found: 663.3059.</div></div><div id="sec4_1_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> (<i>S</i>)-19-((2<i>S</i>,4<i>R</i>)-4-Hydroxy-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl) pyrrolidine-1-carbonyl)-20,20-dimethyl-17-oxo-3,6,9,12,15-pentaoxa-18-azahenicosanoic Acid (<b>17</b>)</h4><div class="NLM_p last">(496 mg, 54%) as white solid. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 8.89 (s, 1H), 7.47 (d, <i>J</i> = 8.1 Hz, 2H), 7.42 (d, <i>J</i> = 8.1 Hz, 2H), 4.69 (s, 1H), 4.59–4.46 (m, 3H), 4.36 (d, <i>J</i> = 15.5 Hz, 1H), 4.16–4.00 (m, 4H), 3.87 (d, <i>J</i> = 11.0 Hz, 1H), 3.80 (dd, <i>J</i> = 11.0, 3.7 Hz, 1H), 3.76–3.53 (m, 16H), 2.48 (s, 3H), 2.22 (dd, <i>J</i> = 13.1, 7.6 Hz, 1H), 2.08 (ddd, <i>J</i> = 13.3, 9.2, 4.4 Hz, 1H), 1.04 (s, 7H). HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calculated for C<sub>34</sub>H<sub>51</sub>N<sub>4</sub>O<sub>11</sub>S, 723.3270; found: 723.3269.</div></div><div id="sec4_1_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> 4-(((<i>S</i>)-1-((2<i>S</i>,4<i>R</i>)-4-Hydroxy-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl) pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-4-oxobutanoic Acid (<b>18</b>)</h4><div class="NLM_p last">(810 mg, 85%) as white solid. <sup>1</sup>H NMR (600 MHz CD<sub>3</sub>OD) δ 9.10 (s, 1H), 7.51 (d, <i>J</i> = 7.8 Hz, 2H), 7.44 (d, <i>J</i> = 8.4 Hz, 2H), 4.64 (s, 1H), 4.60–4.49 (m, 3H), 4.39 (d, <i>J</i> = 15.6 Hz, 1H), 3.91 (d, <i>J</i> = 10.8 Hz, 1H), 3.82 (dd, <i>J</i> = 9.6, 3.6 Hz, 1H), 2.67–2.55 (m, 4H), 2.52 (s, 3H), 2.25–2.22 (m, 1H), 2.12–2.07 (m, 1H), 1.06 (s, 9H). HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calculated for C<sub>26</sub>H<sub>35</sub>N<sub>4</sub>O<sub>6</sub>S, 531.2272, found 531.2280.</div></div><div id="sec4_1_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> 7-(((<i>S</i>)-1-((2<i>S</i>,4<i>R</i>)-4-Hydroxy-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl) pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-7-oxoheptanoic Acid (<b>19</b>)</h4><div class="NLM_p last">(810 mg, 79%) as white solid. <sup>1</sup>H NMR (600 MHz CD<sub>3</sub>OD) δ 8.98 (s, 1H), 7.50 (d, <i>J</i> = 8.4 Hz, 2H), 7.44 (d, <i>J</i> = 9.0 Hz, 2H), 4.65 (s, 1H), 4.60–4.49 (m, 3H), 4.38 (d, <i>J</i> = 15.6 Hz, 1H), 3.93 (d, <i>J</i> = 10.8 Hz, 1H), 3.82 (dd, <i>J</i> = 11.4, 3.6 Hz, 1H), 2.51 (s, 3H), 2.35–2.22 (m, 5H), 2.13–2.08 (m, 1H), 1.68–1.59 (m, 4H),1.42–1.34 (m, 2H), 1.06 (s, 9H). HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calculated for C<sub>29</sub>H<sub>41</sub>N<sub>4</sub>O<sub>6</sub>S, 573.2741, found 573.2754.</div></div><div id="sec4_1_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> 11-(((<i>S</i>)-1-((2<i>S</i>,4<i>R</i>)-4-Hydroxy-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl)pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-11-oxoundecanoic Acid (<b>20</b>)</h4><div class="NLM_p last">(574 mg, 76% yield) as white solid. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 8.99 (d, <i>J</i> = 5.3 Hz, 1H), 7.47 (dd, <i>J</i> = 8.2, 2.2 Hz, 2H), 7.42 (dd, <i>J</i> = 8.3, 2.9 Hz, 2H), 4.65–4.61 (m, 1H), 4.59–4.46 (m, 3H), 4.35 (d, <i>J</i> = 15.5 Hz, 1H), 3.93–3.86 (m, 1H), 3.80 (dt, <i>J</i> = 11.0, 3.5 Hz, 1H), 2.48 (s, 3H), 2.36–2.16 (m, 4H), 2.08 (ddd, <i>J</i> = 13.3, 9.1, 4.5 Hz, 1H), 1.65–1.52 (m, 4H), 1.39–1.22 (m, 11H), 1.03 (s, 9H). HPLC 95% pure, <i>t</i><sub>R</sub> = 4.25 min; HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calculated for C<sub>33</sub>H<sub>49</sub>N<sub>4</sub>O<sub>6</sub>S<sup>+</sup> 629.3367, found 629.3343.</div></div><div id="sec4_1_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> 4-((8-Aminooctyl)oxy)-2-(2,6-dioxopiperidin-3-yl)isoindoline-1,3-dione (<b>26</b>)</h4><div class="NLM_p last">To the suspension of 2-(2,6-dioxopiperidin-3-yl)-4-hydroxyisoindoline-1,3-dione (<b>24</b>, 55.0 mg, 0.2 mmol), sodium bicarbonate (1.2 g, 0.46 mmol, 2.3 equiv), and sodium iodide (6 mg, 0.04 mmol, 0.2 equiv) in DMF (2 mL) was added <i>tert</i>-butyl (8-iodooctyl)carbamate, which was prepared according to the previously reported procedures<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> (<b>25</b>, 85.2 mg, 0.24 mmol, 1.2 equiv). After stirring at 65 °C for 2 days, the reaction mixture was purified by preparative HPLC to yield white solid (46.8 mg, 47%). MS (ESI) <i>m</i>/<i>z</i> 502.5 [M + H]<sup>+</sup>. This white solid was dissolved in DCM (1 mL) and TFA (1 mL). The resulting mixture was stirred for 30 min. Then, it was concentrated and purified by preparative HPLC (10–100% acetonitrile/0.1% TFA in H<sub>2</sub>O) to yield title compound <b>26</b> as white solid (28.2 mg, 76%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.75 (t, <i>J</i> = 7.9 Hz, 1H), 7.41 (dd, <i>J</i> = 8.3, 3.8 Hz, 2H), 5.09 (dd, <i>J</i> = 12.9, 5.4 Hz, 1H), 4.21 (t, <i>J</i> = 6.4 Hz, 2H), 2.96–2.79 (m, 3H), 2.79–2.61 (m, 2H), 2.12 (dd, <i>J</i> = 12.6, 5.9 Hz, 1H), 1.85 (p, <i>J</i> = 6.7 Hz, 2H), 1.65 (p, <i>J</i> = 7.3 Hz, 2H), 1.54 (q, <i>J</i> = 7.4 Hz, 2H), 1.47–1.30 (m, 6H). HPLC 95% pure, <i>t</i><sub>R</sub> = 4.19 min; HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calculated for C<sub>21</sub>H<sub>28</sub>N<sub>3</sub>O<sub>5</sub><sup>+</sup> 402.2023, found 402.2046.</div></div><div id="sec4_1_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> (2<i>R</i>,4<i>S</i>)-1-((<i>S</i>)-2-(11-(4-(3-((4-((3-Chloro-4-fluorophenyl)amino)-7-methoxyquinazolin-6-yl)oxy)propyl)piperazin-1-yl)-11-oxoundecanamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>27</b>)</h4><div class="NLM_p last">Compound <b>27</b> was prepared following the general procedure for preparing compound <b>1</b> from <i>N</i>-(3-chloro-4-fluorophenyl)-7-methoxy-6-(3-(piperazin-1-yl)propoxy)quinazolin-4-amine (<b>13</b>, 11.0 mg, 0.02 mmol) and 11-(((<i>S</i>)-1-((2<i>R</i>,4<i>S</i>)-4-hydroxy-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl)pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-11-oxoundecanoic acid (<b>31</b>, 13.6 mg, 0.02 mmol). White solid (18.4 mg, 87% yield). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 9.00 (s, 1H), 8.74 (s, 1H), 8.06–7.86 (m, 2H), 7.74–7.59 (m, 1H), 7.54–7.08 (m, 6H), 4.58 (dd, <i>J</i> = 8.4, 6.5 Hz, 1H), 4.54–4.46 (m, 2H), 4.43 (s, 1H), 4.41–4.30 (m, 3H), 4.08 (s, 3H), 3.99 (dd, <i>J</i> = 10.8, 4.9 Hz, 1H), 3.93–3.35 (m, 6H), 3.19 (d, <i>J</i> = 1.5 Hz, 5H), 2.50 (s, 3H), 2.43 (dd, <i>J</i> = 9.1, 6.2 Hz, 4H), 2.31–2.09 (m, 3H), 2.02 (ddd, <i>J</i> = 14.6, 9.0, 6.0 Hz, 1H), 1.57 (p, <i>J</i> = 7.5 Hz, 2H), 1.50–1.12 (m, 12H), 1.07 (s, 9H). <sup>13</sup>C NMR (201 MHz, CD<sub>3</sub>OD) δ 175.56, 173.11, 172.78, 171.19, 158.78, 157.58, 156.79, 155.56, 151.48, 150.02, 148.47, 147.51, 138.87, 135.92, 133.70, 132.13, 129.99, 128.97, 127.29, 126.49, 124.50 (d, <i>J</i> (C, F) = 16.0 Hz), 116.49 (d, <i>J</i> (C, F) = 22.1 Hz, C-F), 107.36, 103.60, 99.43, 69.12, 66.79, 59.58, 58.69, 56.11, 55.42, 54.77, 51.82, 51.52, 42.26, 42.10, 38.31, 37.71, 34.90, 33.95, 32.21, 29.07, 28.96, 25.66, 25.38, 24.77, 23.43, 14.55. HPLC 98% pure, <i>t</i><sub>R</sub> = 4.14 min; HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calculated for C<sub>55</sub>H<sub>72</sub>ClFN<sub>9</sub>O<sub>7</sub>S<sup>+</sup> 1056.4942, found 1056.4910.</div></div><div id="sec4_1_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> 3-(4-(3-((4-((3-Chloro-4-fluorophenyl)amino)-7-methoxyquinazolin-6-yl)oxy)propyl)piperazin-1-yl)-<i>N</i>-(8-((2-(1-methyl-2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)octyl)propanamide (<b>28</b>)</h4><div class="NLM_p last">Compound <b>28</b> was prepared following the general procedure for preparing compound <b>10</b> from 3-(4-(3-((4-((3-chloro-4-fluorophenyl)amino)-7-methoxyquinazolin-6-yl)oxy)propyl)piperazin-1-yl)propanoic acid (<b>14</b>, 13.4 mg, 0.025 mmol) and 4-((8-aminooctyl)oxy)-2-(1-methyl-2,6-dioxopiperidin-3-yl)isoindoline-1,3-dione (<b>33</b>, 10.8 mg, 0.025 mmol). White solid (13.8 mg, yield 60%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 8.74 (d, <i>J</i> = 1.2 Hz, 1H), 7.98–7.89 (m, 2H), 7.76–7.69 (m, 1H), 7.66 (ddd, <i>J</i> = 8.9, 4.1, 2.6 Hz, 1H), 7.43–7.32 (m, 3H), 7.27 (s, 1H), 5.12 (dd, <i>J</i> = 12.9, 5.4 Hz, 1H), 4.37 (t, <i>J</i> = 5.5 Hz, 2H), 4.19 (t, <i>J</i> = 6.3 Hz, 2H), 4.09 (s, 3H), 3.77–3.62 (m, 7H), 3.48 (dt, <i>J</i> = 21.2, 6.9 Hz, 4H), 3.19 (t, <i>J</i> = 7.2 Hz, 2H), 3.14 (s, 3H), 2.76–2.62 (m, 5H), 2.42 (p, <i>J</i> = 6.7 Hz, 2H), 2.13–2.05 (m, 1H), 1.88–1.77 (m, 2H), 1.52 (q, <i>J</i> = 7.0 Hz, 4H), 1.44–1.31 (m, 7H). <sup>13</sup>C NMR (201 MHz, CD<sub>3</sub>OD) δ 172.24, 170.87, 170.04, 167.24, 165.95, 158.67, 157.59, 156.76, 156.61, 155.52, 150.12, 148.32, 136.55, 135.68, 133.61, 126.42, 124.47, 120.38 (d, <i>J</i> (C, F) = 20.1 Hz), 119.14, 116.62, 116.37 (d, <i>J</i> (C, F) = 22.1 Hz), 114.88, 107.31, 103.46, 99.32, 69.13, 66.90, 56.15, 54.48, 52.81, 50.09, 49.66, 49.55, 39.13, 31.07, 30.47, 28.88, 28.84, 28.57, 26.48, 26.00, 25.46, 23.92, 21.51. HPLC 98% pure, <i>t</i><sub>R</sub> = 4.32 min; HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calculated for C<sub>47</sub>H<sub>57</sub>ClFN<sub>8</sub>O<sub>8</sub><sup>+</sup> 915.3966, found 915.3967.</div></div><div id="sec4_1_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> 11-(((<i>S</i>)-1-((2<i>R</i>,4<i>S</i>)-4-Hydroxy-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl)pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-11-oxoundecanoic Acid (<b>31</b>)</h4><div class="NLM_p last">Compound <b>31</b> was prepared following the general procedure for preparing compound <b>15</b> from (2<i>R</i>,4<i>S</i>)-1-((<i>S</i>)-2-amino-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>29</b>, 108 mg, 0.25 mmol) and undecanedioic acid (<b>30</b>, 65 mg, 0.3 mmol, 1.2 equiv). White solid (67.2 mg, 43% yield). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 8.99 (s, 1H), 7.46–7.42 (m, 2H), 7.39 (d, <i>J</i> = 8.2 Hz, 2H), 4.59 (dd, <i>J</i> = 8.4, 6.5 Hz, 1H), 4.53 (d, <i>J</i> = 15.7 Hz, 1H), 4.51–4.46 (m, 1H), 4.43 (s, 1H), 4.34 (d, <i>J</i> = 15.7 Hz, 1H), 3.99 (dt, <i>J</i> = 12.1, 6.0 Hz, 1H), 3.72 (ddd, <i>J</i> = 11.0, 3.7, 1.1 Hz, 1H), 2.50 (s, 3H), 2.30–2.12 (m, 5H), 2.00 (ddd, <i>J</i> = 14.6, 8.9, 6.0 Hz, 1H), 1.54 (p, <i>J</i> = 7.4 Hz, 2H), 1.48–1.39 (m, 1H), 1.38–1.13 (m, 11H), 1.07 (s, 9H). HPLC 97% pure, <i>t</i><sub>R</sub> = 4.30 min; HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calculated for C<sub>33</sub>H<sub>49</sub>N<sub>4</sub>O<sub>6</sub>S<sup>+</sup> 629.3367, found 629.3378.</div></div><div id="sec4_1_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> 4-((8-Aminooctyl)oxy)-2-(1-methyl-2,6-dioxopiperidin-3-yl)isoindoline-1,3-dione (<b>33</b>)</h4><div class="NLM_p last">To the suspension of <i>tert</i>-butyl (8-((2-(2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)octyl)carbamate (265 mg, 0.53 mmol) and potassium carbonate (110 mg, 0.8 mmol, 1.5 equiv) in DMF (5 mL) was added iodomethane (114 mg, 0.8 mmol, 1.5 equiv). After stirring overnight at room temperature for 4 h, water was added, and the mixture was extracted with ethyl acetate (3 × 15 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. The crude product was dissolved in DCM (3 mL) and TFA (3 mL). The resulting mixture was stirred for 30 min. Then, it was concentrated and purified by reverse-phase ISCO (10–100% methanol/0.1% TFA in H<sub>2</sub>O) to afford the title compound <b>33</b> as white solid in TFA salt form (179 mg, 81% yield). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.76 (ddd, <i>J</i> = 8.5, 7.2, 1.0 Hz, 1H), 7.43 (dd, <i>J</i> = 7.9, 4.7 Hz, 2H), 5.16–5.05 (m, 1H), 4.22 (t, <i>J</i> = 6.3 Hz, 2H), 3.15 (s, 3H), 2.89 (ddd, <i>J</i> = 12.8, 7.8, 5.2 Hz, 4H), 2.68 (qd, <i>J</i> = 12.6, 6.1 Hz, 1H), 2.14–2.05 (m, 1H), 1.90–1.81 (m, 2H), 1.69–1.61 (m, 2H), 1.59–1.52 (m, 2H), 1.48–1.35 (m, 6H). HPLC 98% pure, <i>t</i><sub>R</sub> = 4.28 min; HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calculated for C<sub>22</sub>H<sub>30</sub>N<sub>3</sub>O<sub>5</sub><sup>+</sup> 416.2180, found 416.2167.</div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> EGFR Binding Assay</h3><div class="NLM_p last">Binding affinities of gefitinib, compounds <b>6</b>, <b>10</b>, <b>27</b>, and <b>28</b> to EGFR WT and EGFR L858R were determined by DiscoverX using the KINOMEscan assay, a competition binding assay that quantitatively measures the ability of a compound to compete with an immobilized, active-site-directed ligand. The assay was performed by combining three components: DNA-tagged EGFR, immobilized ligand, and a test compound. The ability of the test compound to compete with the immobilized ligand was measured by quantitative PCR of the DNA tag. The <i>K</i><sub>d</sub> values were determined using an 11-point 3-fold compound dilution (the top concentration of 1 μM for gefitinib and 30 μM for other compounds) with three DMSO control points in duplicates.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> Cell Culture</h3><div class="NLM_p last">HCC-827 cells and H3255 cells were purchased from ATCC. Both cell lines were cultured in RPMI/1640 medium supplemented with 20% FBS (HCC-827 cells)/20% FBS (H3255 cells) and 50 μg/mL penicillin/streptomycin. Cells were cultured in an incubator at 37 °C with 5% CO<sub>2</sub>.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> Western Blot Analysis</h3><div class="NLM_p last">Cultured cells were washed with PBS twice and then lysed in a 1X Laemmli sample buffer (BIO-RAD) directly. Lysates were heated at 100 °C for 10 min and centrifuged at 14 000 rpm for 10 min before loading to the 4–15% precast gels (BIO-RAD). Samples were running under 80 V for 15 min, then switched to 150 V for another 1 h, and then transferred to PVDF membranes at 100 V, 90 min. Membranes were then blocked in a 5% skim milk TBST buffer for 1 h at room temperature, followed by primary antibody incubation overnight at 4 °C. HRP-conjugated secondary antibody was then applied and incubated for 2 h at room temperature. Membranes were imaged by a ChemiDoc MP Imaging system (BIO-RAD) and organized with Image Lab software.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> Cell Proliferation Assay</h3><div class="NLM_p last">2000/well HCC-827 cells or 4000/well H3255 cells were planted to 96 wells a night before and then treated with indicated doses of EGFR degraders or controls for another 3 days. Cell proliferation/survival was measured using Cell Counting Kit-8 (Sigma-Aldrich) following the manufacturer’s instruction and plotted via Graphpad 6.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> Liquid Chromatography–Mass Spectrometry (LC–MS)</h3><div class="NLM_p">The clean peptides were dissolved in 0.1% formic acid and analyzed on a Q-Exactive HF-X coupled with an Easy nanoLC 1200 (Thermo Fisher Scientific, San Jose, CA). Peptides of 1 mg were loaded onto an Acclain PepMap RSLC C18 column (250 mm × 75 μm ID, C18, 2 μm, Thermo Fisher). Analytical separation of all peptides was achieved with a 100 min or 140 min gradient. A linear gradient of 5–30% buffer B over 75 min or 100 min was executed at a 300 nL/min flow rate following a ramp to 100% B in 1 or 5, and 9 min or 15 min wash with 100% B, where buffer A was aqueous 0.1% formic acid and buffer B was 80% acetonitrile and 0.1% formic acid.</div><div class="NLM_p">LC–MS experiments were also performed in a data-dependent mode with full MS (externally calibrated to a mass accuracy of <5 ppm and a resolution of 60 000 at <i>m</i>/<i>z</i> 200) followed by high-energy collision-activated dissociation-MS/MS of the top 20 most intense ions with a resolution of 15 000 at <i>m</i>/<i>z</i> 200. High-energy collision-activated dissociation-MS/MS was used to dissociate peptides at a normalized collision energy of 27 eV in the presence of nitrogen bath gas atoms. Dynamic exclusion was 30.0 s. There were three biological replicates for one treatment, and each sample was subjected to two technical LC–MS replicates.</div><div class="NLM_p last">Raw data processing and analysis. Mass spectra were processed, and peptide identification was performed using the Andromeda search engine found in MaxQuant software version 1.6.0.16 (Max Planck Institute, Germany). All protein database searches were performed against the UniProt human protein sequence database (UP000005640). Peptides were identified with a target-decoy approach using a combined database consisting of reverse protein sequences of the UniProt human. A false discovery rate for both peptide-spectrum match and protein assignment was set at 1%. Search parameters included up to two missed cleavages at Lys/Arg on the sequence, oxidation of methionine, and protein N-terminal acetylation as a dynamic modification. Carbamidomethylation of cysteine residues was considered as a static modification. Peptide identifications are reported by the filtering of reverse and contaminant entries and assigning to their leading razor protein. LFQ was performed based on peak area. The measured area under the curve of <i>m</i>/<i>z</i> and the retention time-aligned extracted ion chromatogram of a peptide were performed via the label-free quantitation module found in MaxQuant version 1.6.0.16. Data processing and statistical analysis were performed on Perseus (Version 1.6.0.7). Protein quantitation was performed on technical replicates, and a two-sample t-test statistics was used with a <i>p</i>-value of 5% to report statistically significant protein abundance fold changes.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> Statistical Analysis</h3><div class="NLM_p last">Data was analyzed using GraphPad 6.0. Half-life (<i>t</i><sub>1/2</sub>) (time needed for degrading a half of the protein), DC<sub>50</sub> (concentration that resulted in a 50% EGFR degradation), and IC<sub>50</sub> (concentration that led to 50% of cell growth inhibition) were calculated using the nonlinear regression-“log (inhibitor) vs response-variable slope” analytical protocol.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> Mouse Pharmacokinetic Study</h3><div class="NLM_p last">Compounds <b>6</b> and <b>10</b> were dissolved in 5% DMA, 25% Solutol HS-15, and 70% normal saline as formulation. Six male Swiss Albino mice were administered intraperitoneally with a solution formulation of each compound at a dose of 50 mg/kg. Plasma samples (approximately 60 μL) were collected from three mice at 0.5, 2, 8, 12, 16, and 24 h. Compound concentrations in plasma at each time point are the average values from three test mice. Error bars represent ± SEM. Experiments involving mice were performed according to the Institutional Animal Care and Use Committee-approved protocol.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i45"><a href="/doi/suppl/10.1021/acs.jmedchem.9b01566" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_31601" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_31601" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.9b01566?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01566</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline"><sup>1</sup>H NMR, <sup>13</sup>C NMR, and HPLC spectra of compounds <b>6</b>, <b>10</b>, <b>27</b>, and <b>28</b>; <sup>1</sup>H NMR and HPLC spectra of compounds <b>1</b>–<b>5</b> and <b>7</b>–<b>9</b>; illustration of binding of gefitinib to the EGFR kinase domain based on the cocrystal structure of the EGFR kinase domain in complex with gefitinib (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4I22">4I22</a>); and expression levels of VHL and CRBN in HCC-827 (EGFR e19d), H3255 (EGFR L858R), OVCAR-8 (EGFR WT), and H1299 (EGFR WT) cells (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01566/suppl_file/jm9b01566_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings for all compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01566/suppl_file/jm9b01566_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01566/suppl_file/jm9b01566_si_001.pdf">jm9b01566_si_001.pdf (3.89 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01566/suppl_file/jm9b01566_si_002.csv">jm9b01566_si_002.csv (2.56 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.9b01566" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_70052" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_70052" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xufen Yu</span> - <span class="hlFld-Affiliation affiliation">Mount
Sinai Center for Therapeutics Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#641c1102010a4a1d1124091717094a010011"><span class="__cf_email__" data-cfemail="176f62717279396e62577a64647a39727362">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yue Xiong</span> - <span class="hlFld-Affiliation affiliation">Curriculum
in Genetics and Molecular Biology, 
    Lineberger Comprehensive Cancer
Center, 
    Department of Biochemistry and Biophysics, 
    , , and , University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#017879686e6f6641646c60686d2f746f622f646574"><span class="__cf_email__" data-cfemail="4c35342523222b0c29212d25206239222f62292839">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jian Jin</span> - <span class="hlFld-Affiliation affiliation">Mount
Sinai Center for Therapeutics Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2387-3862" title="Orcid link">http://orcid.org/0000-0002-2387-3862</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#8be1e2eae5a5e1e2e5cbe6f8f8e6a5eeeffe"><span class="__cf_email__" data-cfemail="12787b737c3c787b7c527f61617f3c777667">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Meng Cheng</span> - <span class="hlFld-Affiliation affiliation">Curriculum
in Genetics and Molecular Biology, 
    Lineberger Comprehensive Cancer
Center, 
    , , and , University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kaylene Lu</span> - <span class="hlFld-Affiliation affiliation">Lineberger Comprehensive Cancer
Center, , and , University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ling Xie</span> - <span class="hlFld-Affiliation affiliation">Department of Biochemistry and Biophysics, , and , University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Li Wang</span> - <span class="hlFld-Affiliation affiliation">Department of Biochemistry and Biophysics, , and , University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fanye Meng</span> - <span class="hlFld-Affiliation affiliation">Mount
Sinai Center for Therapeutics Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiaoran Han</span> - <span class="hlFld-Affiliation affiliation">Lineberger Comprehensive Cancer
Center, , and , University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span>; 
    <span>Present Address:
                        Cullgen (Shanghai), Inc. 230 Chuan Hong Road, Shanghai 201202, People’s Republic of China (X.H.)</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xian Chen</span> - <span class="hlFld-Affiliation affiliation">Department of Biochemistry and Biophysics, , and , University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jing Liu</span> - <span class="hlFld-Affiliation affiliation">Mount
Sinai Center for Therapeutics Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>M.C. and X.F. contributed equally to this work.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e3970-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i48">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_07844" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_07844" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">J.J. acknowledges the support by an endowed professorship by the Icahn School of Medicine at Mount Sinai.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">EGFR</td><td class="NLM_def"><p class="first last">epidermal growth factor receptor</p></td></tr><tr><td class="NLM_term">NSCLC</td><td class="NLM_def"><p class="first last">non-small-cell lung cancers</p></td></tr><tr><td class="NLM_term">PROTACs</td><td class="NLM_def"><p class="first last">proteolysis targeting chimeras</p></td></tr><tr><td class="NLM_term">TKI</td><td class="NLM_def"><p class="first last">tyrosine kinase inhibitor</p></td></tr><tr><td class="NLM_term">PEG</td><td class="NLM_def"><p class="first last">poly(ethylene glycol)</p></td></tr><tr><td class="NLM_term">ATP</td><td class="NLM_def"><p class="first last">adenosine triphosphate</p></td></tr><tr><td class="NLM_term">EGFR e19d</td><td class="NLM_def"><p class="first last">EGFR mutant with exo 19 deletion</p></td></tr><tr><td class="NLM_term">EGFR L858R</td><td class="NLM_def"><p class="first last">EGFR L858R single point mutant</p></td></tr><tr><td class="NLM_term">RTK</td><td class="NLM_def"><p class="first last">receptor tyrosine kinase</p></td></tr><tr><td class="NLM_term">WT</td><td class="NLM_def"><p class="first last">wild type</p></td></tr><tr><td class="NLM_term">VHL</td><td class="NLM_def"><p class="first last">von Hippel–Lindau</p></td></tr><tr><td class="NLM_term">CRBN</td><td class="NLM_def"><p class="first last">cereblon</p></td></tr><tr><td class="NLM_term">PK</td><td class="NLM_def"><p class="first last">pharmacokinetic</p></td></tr><tr><td class="NLM_term">TFA</td><td class="NLM_def"><p class="first last">trifluoroacetic acid</p></td></tr><tr><td class="NLM_term">EDCI</td><td class="NLM_def"><p class="first last">1-ethyl-3-(3-dimethylaminopropyl)carbodiimide</p></td></tr><tr><td class="NLM_term">HOAt</td><td class="NLM_def"><p class="first last">1-hydroxy-7-azabenzo-triazole</p></td></tr><tr><td class="NLM_term">NMM</td><td class="NLM_def"><p class="first last"><i>N</i>-methylmorpholine</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i50">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_27422" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_27422" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 55 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Settleman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haber, D. A.</span></span> <span> </span><span class="NLM_article-title">Epidermal growth factor receptor mutations in lung cancer</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">169</span>– <span class="NLM_lpage">181</span>, <span class="refDoi"> DOI: 10.1038/nrc2088</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1038%2Fnrc2088" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=17318210" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BD2sXitVGrsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2007&pages=169-181&author=S.+V.+Sharmaauthor=D.+W.+Bellauthor=J.+Settlemanauthor=D.+A.+Haber&title=Epidermal+growth+factor+receptor+mutations+in+lung+cancer&doi=10.1038%2Fnrc2088"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Epidermal growth factor receptor mutations in lung cancer</span></div><div class="casAuthors">Sharma, Sreenath V.; Bell, Daphne W.; Settleman, Jeffrey; Haber, Daniel A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">169-181</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The development and clin. application of inhibitors that target the epidermal growth factor receptor (EGFR) provide important insights for new lung cancer therapies, as well as for the broader field of targeted cancer therapies.  We review the results of genetic, biochem. and clin. studies focused on somatic mutations of EGFR that are assocd. with the phenomenon of oncogene addiction, describing 'oncogenic shock' as a mechanistic explanation for the apoptosis that follows the acute treatment of susceptible cells with kinase inhibitors.  Understanding the genetic heterogeneity of epithelial tumors and devising strategies to circumvent their rapid acquisition of resistance to targeted kinase inhibitors are essential to the successful use of targeted therapies in common epithelial cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMKw7Y4FUFM7Vg90H21EOLACvtfcHk0lgWiF3Yfsex6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXitVGrsbo%253D&md5=46219ea3ae7b9f0977e4a4b7243fd875</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrc2088&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2088%26sid%3Dliteratum%253Aachs%26aulast%3DSharma%26aufirst%3DS.%2BV.%26aulast%3DBell%26aufirst%3DD.%2BW.%26aulast%3DSettleman%26aufirst%3DJ.%26aulast%3DHaber%26aufirst%3DD.%2BA.%26atitle%3DEpidermal%2520growth%2520factor%2520receptor%2520mutations%2520in%2520lung%2520cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2007%26volume%3D7%26spage%3D169%26epage%3D181%26doi%3D10.1038%2Fnrc2088" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wieduwilt, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moasser, M. M.</span></span> <span> </span><span class="NLM_article-title">The epidermal growth factor receptor family: biology driving targeted therapeutics</span>. <i>Cell Mol. Life Sci.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>65</i></span>,  <span class="NLM_fpage">1566</span>– <span class="NLM_lpage">1584</span>, <span class="refDoi"> DOI: 10.1007/s00018-008-7440-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1007%2Fs00018-008-7440-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=18259690" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BD1cXmt12qu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2008&pages=1566-1584&author=M.+J.+Wieduwiltauthor=M.+M.+Moasser&title=The+epidermal+growth+factor+receptor+family%3A+biology+driving+targeted+therapeutics&doi=10.1007%2Fs00018-008-7440-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">The epidermal growth factor receptor family: Biology driving targeted therapeutics</span></div><div class="casAuthors">Wieduwilt, M. J.; Moasser, M. M.</div><div class="citationInfo"><span class="NLM_cas:title">Cellular and Molecular Life Sciences</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1566-1584</span>CODEN:
                <span class="NLM_cas:coden">CMLSFI</span>;
        ISSN:<span class="NLM_cas:issn">1420-682X</span>.
    
            (<span class="NLM_cas:orgname">Birkhaeuser Verlag</span>)
        </div><div class="casAbstract">A review of the biol. of ErbB receptors, including their structure, signaling, regulation, and roles in development and disease, and their increasing roles as targets for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcSSPMwgxaFLVg90H21EOLACvtfcHk0li6QT3wFYOnag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXmt12qu7o%253D&md5=d0e8b0634b2b401b1c94838fe389ca91</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1007%2Fs00018-008-7440-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00018-008-7440-8%26sid%3Dliteratum%253Aachs%26aulast%3DWieduwilt%26aufirst%3DM.%2BJ.%26aulast%3DMoasser%26aufirst%3DM.%2BM.%26atitle%3DThe%2520epidermal%2520growth%2520factor%2520receptor%2520family%253A%2520biology%2520driving%2520targeted%2520therapeutics%26jtitle%3DCell%2520Mol.%2520Life%2520Sci.%26date%3D2008%26volume%3D65%26spage%3D1566%26epage%3D1584%26doi%3D10.1007%2Fs00018-008-7440-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seshacharyulu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ponnusamy, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haridas, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ganti, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Batra, S. K.</span></span> <span> </span><span class="NLM_article-title">Targeting the EGFR signaling pathway in cancer therapy</span>. <i>Expert Opin. Ther. Targets</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">15</span>– <span class="NLM_lpage">31</span>, <span class="refDoi"> DOI: 10.1517/14728222.2011.648617</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1517%2F14728222.2011.648617" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=22239438" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC38Xos1Wksw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2012&pages=15-31&author=P.+Seshacharyuluauthor=M.+P.+Ponnusamyauthor=D.+Haridasauthor=M.+Jainauthor=A.+K.+Gantiauthor=S.+K.+Batra&title=Targeting+the+EGFR+signaling+pathway+in+cancer+therapy&doi=10.1517%2F14728222.2011.648617"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the EGFR signaling pathway in cancer therapy</span></div><div class="casAuthors">Seshacharyulu, Parthasarathy; Ponnusamy, Moorthy P.; Haridas, Dhanya; Jain, Maneesh; Ganti, Apar K.; Batra, Surinder K.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Targets</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">15-31</span>CODEN:
                <span class="NLM_cas:coden">EOTTAO</span>;
        ISSN:<span class="NLM_cas:issn">1472-8222</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Introduction: Cancer is a devastating disease; however, several therapeutic advances have recently been made, wherein EGFR and its family members have emerged as useful biomarkers and therapeutic targets.  EGFR, a transmembrane glycoprotein is a member of the ERBB receptor tyrosine kinase superfamily.  EGFR binds to its cognate ligand EGF, which further induces tyrosine phosphorylation and receptor dimerization with other family members leading to enhanced uncontrolled proliferation.  Several anti-EGFR therapies such as monoclonal antibodies and tyrosine kinase inhibitors have been developed, which has enabled clinicians to identify and treat specific patient cohorts.Areas covered: This review covers the basic mechanism of EGFR activation and the role of EGFR signaling in cancer progression.  Furthermore, current developments made toward targeting the EGFR signaling pathway for the treatment of epithelial cancers and a summary of the various anti-EGFR therapeutic agents that are currently in use are also presented in this review.Expert opinion: EGFR signaling is a part of a complex network that has been the target of effective cancer therapies.  However, a further understanding of the system is required to develop an effective anticancer regimen.  A combination therapy that comprises an anti-EGFR and a chemotherapeutic/chemopreventive agent will exhibit a multi-pronged approach that can be developed into a highly attractive and specific mol. oriented remedy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDi_-cVSomx7Vg90H21EOLACvtfcHk0li6QT3wFYOnag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xos1Wksw%253D%253D&md5=91357f1fb76bd06c03d4269f90ad907a</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1517%2F14728222.2011.648617&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F14728222.2011.648617%26sid%3Dliteratum%253Aachs%26aulast%3DSeshacharyulu%26aufirst%3DP.%26aulast%3DPonnusamy%26aufirst%3DM.%2BP.%26aulast%3DHaridas%26aufirst%3DD.%26aulast%3DJain%26aufirst%3DM.%26aulast%3DGanti%26aufirst%3DA.%2BK.%26aulast%3DBatra%26aufirst%3DS.%2BK.%26atitle%3DTargeting%2520the%2520EGFR%2520signaling%2520pathway%2520in%2520cancer%2520therapy%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Targets%26date%3D2012%26volume%3D16%26spage%3D15%26epage%3D31%26doi%3D10.1517%2F14728222.2011.648617" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krause, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Etten, R. A.</span></span> <span> </span><span class="NLM_article-title">Tyrosine kinases as targets for cancer therapy</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>353</i></span>,  <span class="NLM_fpage">172</span>– <span class="NLM_lpage">187</span>, <span class="refDoi"> DOI: 10.1056/NEJMra044389</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1056%2FNEJMra044389" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=16014887" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmtFClsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=353&publication_year=2005&pages=172-187&author=D.+S.+Krauseauthor=R.+A.+Van+Etten&title=Tyrosine+kinases+as+targets+for+cancer+therapy&doi=10.1056%2FNEJMra044389"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Tyrosine kinases as targets for cancer therapy</span></div><div class="casAuthors">Krause, Daniela S.; Van Etten, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">353</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">172-187</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">A review discusses the mechanisms of aberrant tyrosine kinase signaling and strategies to inhibit TKs in cancer.  It also summarizes the status of TK-directed cancer therapies, and discusses challenges and prospects for the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq89QOl9E0UMLVg90H21EOLACvtfcHk0li6QT3wFYOnag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmtFClsbs%253D&md5=0a98c8840a983300f6b0743673cf480c</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1056%2FNEJMra044389&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMra044389%26sid%3Dliteratum%253Aachs%26aulast%3DKrause%26aufirst%3DD.%2BS.%26aulast%3DVan%2BEtten%26aufirst%3DR.%2BA.%26atitle%3DTyrosine%2520kinases%2520as%2520targets%2520for%2520cancer%2520therapy%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2005%26volume%3D353%26spage%3D172%26epage%3D187%26doi%3D10.1056%2FNEJMra044389" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nadeem
Abbas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kausar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, H.</span></span> <span> </span><span class="NLM_article-title">Advances in targeting the epidermal growth factor receptor pathway by synthetic products and its regulation by epigenetic modulators as a therapy for glioblastoma</span>. <i>Cells</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">350</span>– <span class="NLM_lpage">361</span>, <span class="refDoi"> DOI: 10.3390/cells8040350</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.3390%2Fcells8040350" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2019&pages=350-361&author=M.+Nadeem%0AAbbasauthor=S.+Kausarauthor=F.+Wangauthor=Y.+Zhaoauthor=H.+Cui&title=Advances+in+targeting+the+epidermal+growth+factor+receptor+pathway+by+synthetic+products+and+its+regulation+by+epigenetic+modulators+as+a+therapy+for+glioblastoma&doi=10.3390%2Fcells8040350"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.3390%2Fcells8040350&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fcells8040350%26sid%3Dliteratum%253Aachs%26aulast%3DNadeem%2BAbbas%26aufirst%3DM.%26aulast%3DKausar%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DCui%26aufirst%3DH.%26atitle%3DAdvances%2520in%2520targeting%2520the%2520epidermal%2520growth%2520factor%2520receptor%2520pathway%2520by%2520synthetic%2520products%2520and%2520its%2520regulation%2520by%2520epigenetic%2520modulators%2520as%2520a%2520therapy%2520for%2520glioblastoma%26jtitle%3DCells%26date%3D2019%26volume%3D8%26spage%3D350%26epage%3D361%26doi%3D10.3390%2Fcells8040350" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nicholson, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gee, J. M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, M. E.</span></span> <span> </span><span class="NLM_article-title">EGFR and cancer prognosis</span>. <i>Eur. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">9</span>– <span class="NLM_lpage">15</span>, <span class="refDoi"> DOI: 10.1016/S0959-8049(01)00231-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1016%2FS0959-8049%2801%2900231-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=11165124" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BD3MXntlOku7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2001&pages=9-15&author=R.+I.+Nicholsonauthor=J.+M.+W.+Geeauthor=M.+E.+Harper&title=EGFR+and+cancer+prognosis&doi=10.1016%2FS0959-8049%2801%2900231-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">EGFR and cancer prognosis</span></div><div class="casAuthors">Nicholson, R. I.; Gee, J. M. W.; Harper, M. E.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Cancer</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">Suppl. 4</span>),
    <span class="NLM_cas:pages">S9-S15</span>CODEN:
                <span class="NLM_cas:coden">EJCAEL</span>;
        ISSN:<span class="NLM_cas:issn">0959-8049</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">Elevated levels of the epidermal growth factor receptor (EGFR), a growth-factor-receptor tyrosine kinase, and/or its cognate ligands have been identified as a common component of multiple cancer types and appear to promote solid tumor growth.  This article examines the relationship between EGFR expression and cancer prognosis based on literature compiled on PubMed between 1985 and Sept. 2000.  More than 200 studies were identified that analyzed relapse-free-interval or survival data directly in relation to EGFR levels in over 20000 patients.  Anal. of the data showed that 10 cancer types both express elevated levels of EGFR relative to normal tissues and have been studied in sufficient depth to allow sound judgments to be made concerning the assocn. between EGFR and patient outlook.  The EGFR was found to act as a strong prognostic indicator in head and neck, ovarian, cervical, bladder and esophageal cancers.  In these cancers, increased EGFR expression was assocd. with reduced recurrence-free or overall survival rates in 70% (52/74) of studies.  In gastric, breast, endometrial and colorectal cancers, the EGFR provided more modest prognostic information, correlating to poor survival rates in 52% (13/25) of studies, while in non-small cell lung cancer (NSCLC), EGFR expression only rarely (3/10 studies) related to patient outlook.  However, it is likely that the true prognostic significance of the EGFR has been underestimated as the published studies only assessed total cellular EGFR levels, rather than the activated form of the receptor, and were not standardized with regard to patient populations or assay methods.  Finally, it is important to stress that failure to detect a prognostic significance for EGFR in any one cancer type does not necessarily preclude patients from benefiting from anti-EGFR therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_RCCyV1BxyLVg90H21EOLACvtfcHk0lhX48sPx3cnQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXntlOku7w%253D&md5=26c5b720b0cc173a586f652336b20c25</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2FS0959-8049%2801%2900231-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0959-8049%252801%252900231-3%26sid%3Dliteratum%253Aachs%26aulast%3DNicholson%26aufirst%3DR.%2BI.%26aulast%3DGee%26aufirst%3DJ.%2BM.%2BW.%26aulast%3DHarper%26aufirst%3DM.%2BE.%26atitle%3DEGFR%2520and%2520cancer%2520prognosis%26jtitle%3DEur.%2520J.%2520Cancer%26date%3D2001%26volume%3D37%26spage%3D9%26epage%3D15%26doi%3D10.1016%2FS0959-8049%2801%2900231-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hirsch, F. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scagliotti, G. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulshine, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwon, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curran, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paz-Ares, L.</span></span> <span> </span><span class="NLM_article-title">Lung cancer: current therapies and new targeted treatments</span>. <i>Lancet</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>389</i></span>,  <span class="NLM_fpage">299</span>– <span class="NLM_lpage">311</span>, <span class="refDoi"> DOI: 10.1016/S0140-6736(16)30958-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1016%2FS0140-6736%2816%2930958-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=27574741" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVamsbzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=389&publication_year=2017&pages=299-311&author=F.+R.+Hirschauthor=G.+V.+Scagliottiauthor=J.+L.+Mulshineauthor=R.+Kwonauthor=W.+J.+Curranauthor=Y.-L.+Wuauthor=L.+Paz-Ares&title=Lung+cancer%3A+current+therapies+and+new+targeted+treatments&doi=10.1016%2FS0140-6736%2816%2930958-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Lung cancer: current therapies and new targeted treatments</span></div><div class="casAuthors">Hirsch, Fred R.; Scagliotti, Giorgio V.; Mulshine, James L.; Kwon, Regina; Curran, Walter J., Jr.; Wu, Yi-Long; Paz-Ares, Luis</div><div class="citationInfo"><span class="NLM_cas:title">Lancet</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">389</span>
        (<span class="NLM_cas:issue">10066</span>),
    <span class="NLM_cas:pages">299-311</span>CODEN:
                <span class="NLM_cas:coden">LANCAO</span>;
        ISSN:<span class="NLM_cas:issn">0140-6736</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Lung cancer is the most frequent cause of cancer-related deaths worldwide.  Every year, 1·8 million people are diagnosed with lung cancer, and 1·6 million people die as a result of the disease. 5-yr survival rates vary from 4-17% depending on stage and regional differences.  In this Seminar, we discuss existing treatment for patients with lung cancer and the promise of precision medicine, with special emphasis on new targeted therapies.  Some subgroups, eg-patients with poor performance status and elderly patients-are not specifically addressed, because these groups require special treatment considerations and no frameworks have been established in terms of new targeted therapies.  We discuss prevention and early detection of lung cancer with an emphasis on lung cancer screening.  Although we acknowledge the importance of smoking prevention and cessation, this is a large topic beyond the scope of this Seminar.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqP_NrglINv67Vg90H21EOLACvtfcHk0lhX48sPx3cnQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVamsbzP&md5=36ace1a455a803dc6e7af83dbffb2e34</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2FS0140-6736%2816%2930958-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0140-6736%252816%252930958-8%26sid%3Dliteratum%253Aachs%26aulast%3DHirsch%26aufirst%3DF.%2BR.%26aulast%3DScagliotti%26aufirst%3DG.%2BV.%26aulast%3DMulshine%26aufirst%3DJ.%2BL.%26aulast%3DKwon%26aufirst%3DR.%26aulast%3DCurran%26aufirst%3DW.%2BJ.%26aulast%3DWu%26aufirst%3DY.-L.%26aulast%3DPaz-Ares%26aufirst%3DL.%26atitle%3DLung%2520cancer%253A%2520current%2520therapies%2520and%2520new%2520targeted%2520treatments%26jtitle%3DLancet%26date%3D2017%26volume%3D389%26spage%3D299%26epage%3D311%26doi%3D10.1016%2FS0140-6736%2816%2930958-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Herbst, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukuoka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baselga, J.</span></span> <span> </span><span class="NLM_article-title">Gefitinib--a novel targeted approach to treating cancer</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">956</span>– <span class="NLM_lpage">965</span>, <span class="refDoi"> DOI: 10.1038/nrc1506</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1038%2Fnrc1506" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=15573117" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVarsLjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2004&pages=956-965&author=R.+S.+Herbstauthor=M.+Fukuokaauthor=J.+Baselga&title=Gefitinib%2D%2Da+novel+targeted+approach+to+treating+cancer&doi=10.1038%2Fnrc1506"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Timeline: Gefitinib - a novel targeted approach to treating cancer</span></div><div class="casAuthors">Herbst, Roy S.; Fukuoka, Masahiro; Baselga, Jose</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">956-965</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Twenty years after the epidermal growth factor receptor (EGFR) was identified as a potential anticancer target, the EGFR inhibitor gefitinib (Iressa; AstraZeneca) has been approved for the treatment of patients with advanced non-small-cell lung cancer in many countries.  Studies have indicated its potential for treating patients with other types of solid tumors.  Investigation of gefitinib has not only increased our knowledge about the biol. of EGFR signaling, but is contributing to our evolving understanding of which tumors are EGFR dependent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8V2qzLg_-rrVg90H21EOLACvtfcHk0lhX48sPx3cnQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVarsLjJ&md5=03a6c08b947bc3e0ca59e1b92fa8d214</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnrc1506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc1506%26sid%3Dliteratum%253Aachs%26aulast%3DHerbst%26aufirst%3DR.%2BS.%26aulast%3DFukuoka%26aufirst%3DM.%26aulast%3DBaselga%26aufirst%3DJ.%26atitle%3DGefitinib--a%2520novel%2520targeted%2520approach%2520to%2520treating%2520cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2004%26volume%3D4%26spage%3D956%26epage%3D965%26doi%3D10.1038%2Fnrc1506" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cappuzzo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciuleanu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stelmakh, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cicenas, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szczésna, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Juhász, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esteban, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molinier, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brugger, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melezínek, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klingelschmitt, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klughammer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giaccone, G.</span></span> <span> </span><span class="NLM_article-title">Erlotinib as maintenance treatment in advanced non-small-cell lung cancer: a multicentre, randomised, placebo-controlled phase 3 study</span>. <i>Lancet Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">521</span>– <span class="NLM_lpage">529</span>, <span class="refDoi"> DOI: 10.1016/S1470-2045(10)70112-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1016%2FS1470-2045%2810%2970112-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=20493771" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC3cXntVCksb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=521-529&author=F.+Cappuzzoauthor=T.+Ciuleanuauthor=L.+Stelmakhauthor=S.+Cicenasauthor=A.+Szcz%C3%A9snaauthor=E.+Juh%C3%A1szauthor=E.+Estebanauthor=O.+Molinierauthor=W.+Bruggerauthor=I.+Melez%C3%ADnekauthor=G.+Klingelschmittauthor=B.+Klughammerauthor=G.+Giaccone&title=Erlotinib+as+maintenance+treatment+in+advanced+non-small-cell+lung+cancer%3A+a+multicentre%2C+randomised%2C+placebo-controlled+phase+3+study&doi=10.1016%2FS1470-2045%2810%2970112-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Erlotinib as maintenance treatment in advanced non-small-cell lung cancer: a multicentre, randomised, placebo-controlled phase 3 study</span></div><div class="casAuthors">Cappuzzo, Federico; Ciuleanu, Tudor; Stelmakh, Lilia; Cicenas, Saulius; Szczesna, Aleksandra; Juhasz, Erzsebet; Esteban, Emilio; Molinier, Olivier; Brugger, Wolfram; Melezinek, Ivan; Klingelschmitt, Gaelle; Klughammer, Barbara; Giaccone, Giuseppe</div><div class="citationInfo"><span class="NLM_cas:title">Lancet Oncology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">521-529</span>CODEN:
                <span class="NLM_cas:coden">LOANBN</span>;
        ISSN:<span class="NLM_cas:issn">1470-2045</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">First-line chemotherapy for advanced non-small-cell lung cancer (NSCLC) is usually limited to four to six cycles.  Maintenance therapy can delay progression and prolong survival.  The oral epidermal growth factor receptor (EGFR) tyrosine-kinase inhibitor erlotinib has proven efficacy and tolerability in second-line NSCLC.  We designed the phase 3, placebo-controlled Sequential Tarceva in Unresectable NSCLC (SATURN; BO18192) study to assess use of erlotinib as maintenance therapy in patients with non-progressive disease following first-line platinum-doublet chemotherapy.  Between Dec., 2005, and May, 2008, 1949 patients were included in the run-in phase (four cycles of platinum-based chemotherapy).  At the end of the run-in phase, 889 patients who did not have progressive disease were entered into the main study, and were randomly allocated using a 1:1 adaptive randomisation method through a third-party interactive voice response system to receive erlotinib (150 mg/day; n=438) or placebo (n=451) until progression or unacceptable toxicity.  Patients were stratified by EGFR immunohistochem. status, stage, Eastern Cooperative Oncol. Group performance status, chemotherapy regimen, smoking history, and region.  Co-primary endpoints were progression-free survival (PFS) in all analysable patients irresp. of EGFR status, and PFS in patients whose tumors had EGFR protein overexpression, as detd. by immunohistochem.  This study is registered with www.ClinicalTrials.gov, no. NCT00556712.  884 patients were analysable for PFS; 437 in the erlotinib group and 447 in the placebo group.  After a median follow-up of 11.4 mo for the erlotinib group and 11.5 mo for the placebo group, median PFS was significantly longer with erlotinib than with placebo: 12.3 wk for patients in the erlotinib group vs. 11.1 wk for those in the placebo group (HR 0.71, 95% CI 0.62-0.82; p<0.0001).  PFS was also significantly longer in patients with EGFR-pos. immunohistochem. who were treated with erlotinib (n=307) compared with EGFR-pos. patients given placebo (n=311; median PFS 12.3 wk in the erlotinib group vs 11.1 wk in the placebo group; HR 0.69, 0.58-0.82; p<0.0001).  The most common grade 3 or higher adverse events were rash (37 [9%] of 443 patients in the erlotinib group vs none of 445 in the placebo group) and diarrhoea (seven [2%] of 443 patients vs none of 445).  Serious adverse events were reported in 47 patients (11%) on erlotinib compared with 34 patients (8%) on placebo.  The most common serious adverse event was pneumonia (seven cases [2%] with erlotinib and four [<1%] with placebo).  Maintenance therapy with erlotinib for patients with NSCLC is well tolerated and significantly prolongs PFS compared with placebo.  First-line maintenance with erlotinib could be considered in patients who do not progress after four cycles of chemotherapy.  Funding: F Hoffmann-La Roche Ltd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQ1stlOmTgyrVg90H21EOLACvtfcHk0lgNiJCnP7ul5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXntVCksb0%253D&md5=384f65f7c8ad10ca34d2d0fed7183484</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2FS1470-2045%2810%2970112-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1470-2045%252810%252970112-1%26sid%3Dliteratum%253Aachs%26aulast%3DCappuzzo%26aufirst%3DF.%26aulast%3DCiuleanu%26aufirst%3DT.%26aulast%3DStelmakh%26aufirst%3DL.%26aulast%3DCicenas%26aufirst%3DS.%26aulast%3DSzcz%25C3%25A9sna%26aufirst%3DA.%26aulast%3DJuh%25C3%25A1sz%26aufirst%3DE.%26aulast%3DEsteban%26aufirst%3DE.%26aulast%3DMolinier%26aufirst%3DO.%26aulast%3DBrugger%26aufirst%3DW.%26aulast%3DMelez%25C3%25ADnek%26aufirst%3DI.%26aulast%3DKlingelschmitt%26aufirst%3DG.%26aulast%3DKlughammer%26aufirst%3DB.%26aulast%3DGiaccone%26aufirst%3DG.%26atitle%3DErlotinib%2520as%2520maintenance%2520treatment%2520in%2520advanced%2520non-small-cell%2520lung%2520cancer%253A%2520a%2520multicentre%252C%2520randomised%252C%2520placebo-controlled%2520phase%25203%2520study%26jtitle%3DLancet%2520Oncol.%26date%3D2010%26volume%3D11%26spage%3D521%26epage%3D529%26doi%3D10.1016%2FS1470-2045%2810%2970112-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yun, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mengwasser, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toms, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woo, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greulich, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyerson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eck, M. J.</span></span> <span> </span><span class="NLM_article-title">The T790M mutation in EGFR kinase causes drug resistance by increasing the affinity for ATP</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>105</i></span>,  <span class="NLM_fpage">2070</span>– <span class="NLM_lpage">2075</span>, <span class="refDoi"> DOI: 10.1073/pnas.0709662105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1073%2Fpnas.0709662105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=18227510" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BD1cXitl2mu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2008&pages=2070-2075&author=C.+H.+Yunauthor=K.+E.+Mengwasserauthor=A.+V.+Tomsauthor=M.+S.+Wooauthor=H.+Greulichauthor=K.+K.+Wongauthor=M.+Meyersonauthor=M.+J.+Eck&title=The+T790M+mutation+in+EGFR+kinase+causes+drug+resistance+by+increasing+the+affinity+for+ATP&doi=10.1073%2Fpnas.0709662105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">The T790M mutation in EGFR kinase causes drug resistance by increasing the affinity for ATP</span></div><div class="casAuthors">Yun, Cai-Hong; Mengwasser, Kristen E.; Toms, Angela V.; Woo, Michele S.; Greulich, Heidi; Wong, Kwok-Kin; Meyerson, Matthew; Eck, Michael J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2070-2075</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Lung cancers caused by activating mutations in the epidermal growth factor receptor (EGFR) are initially responsive to small mol. tyrosine kinase inhibitors (TKIs), but the efficacy of these agents is often limited because of the emergence of drug resistance conferred by a second mutation, T790M.  Threonine 790 is the "gatekeeper" residue, an important determinant of inhibitor specificity in the ATP binding pocket.  The T790M mutation has been thought to cause resistance by sterically blocking binding of TKIs such as gefitinib and erlotinib, but this explanation is difficult to reconcile with the fact that it remains sensitive to structurally similar irreversible inhibitors.  Here, we show by using a direct binding assay that T790M mutants retain low-nanomolar affinity for gefitinib.  Furthermore, we show that the T790M mutation activates WT EGFR and that introduction of the T790M mutation increases the ATP affinity of the oncogenic L858R mutant by more than an order of magnitude.  The increased ATP affinity is the primary mechanism by which the T790M mutation confers drug resistance.  Crystallog. anal. of the T790M mutant shows how it can adapt to accommodate tight binding of diverse inhibitors, including the irreversible inhibitor HKI-272, and also suggests a structural mechanism for catalytic activation.  We conclude that the T790M mutation is a "generic" resistance mutation that will reduce the potency of any ATP-competitive kinase inhibitor and that irreversible inhibitors overcome this resistance simply through covalent binding, not as a result of an alternative binding mode.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXXP7TBlyrdrVg90H21EOLACvtfcHk0lgNiJCnP7ul5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXitl2mu7k%253D&md5=f5f5d791a65f73ca628d95e9922b8ea8</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0709662105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0709662105%26sid%3Dliteratum%253Aachs%26aulast%3DYun%26aufirst%3DC.%2BH.%26aulast%3DMengwasser%26aufirst%3DK.%2BE.%26aulast%3DToms%26aufirst%3DA.%2BV.%26aulast%3DWoo%26aufirst%3DM.%2BS.%26aulast%3DGreulich%26aufirst%3DH.%26aulast%3DWong%26aufirst%3DK.%2BK.%26aulast%3DMeyerson%26aufirst%3DM.%26aulast%3DEck%26aufirst%3DM.%2BJ.%26atitle%3DThe%2520T790M%2520mutation%2520in%2520EGFR%2520kinase%2520causes%2520drug%2520resistance%2520by%2520increasing%2520the%2520affinity%2520for%2520ATP%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2008%26volume%3D105%26spage%3D2070%26epage%3D2075%26doi%3D10.1073%2Fpnas.0709662105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Godin-Heymann, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ulkus, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brannigan, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDermott, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamb, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maheswaran, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Settleman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haber, D. A.</span></span> <span> </span><span class="NLM_article-title">The T790M “gatekeeper” mutation in EGFR mediates resistance to low concentrations of an irreversible EGFR inhibitor</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">874</span>– <span class="NLM_lpage">879</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.MCT-07-2387</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1158%2F1535-7163.MCT-07-2387" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=18413800" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BD1cXks1GmsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=874-879&author=N.+Godin-Heymannauthor=L.+Ulkusauthor=B.+W.+Branniganauthor=U.+McDermottauthor=J.+Lambauthor=S.+Maheswaranauthor=J.+Settlemanauthor=D.+A.+Haber&title=The+T790M+%E2%80%9Cgatekeeper%E2%80%9D+mutation+in+EGFR+mediates+resistance+to+low+concentrations+of+an+irreversible+EGFR+inhibitor&doi=10.1158%2F1535-7163.MCT-07-2387"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">The T790M "gatekeeper" mutation in EGFR mediates resistance to low concentrations of an irreversible EGFR inhibitor</span></div><div class="casAuthors">Godin-Heymann, Nadia; Ulkus, Lindsey; Brannigan, Brian W.; McDermott, Ultan; Lamb, Jennifer; Maheswaran, Shyamala; Settleman, Jeffrey; Haber, Daniel A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">874-879</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Patients with non-small cell lung cancer (NSCLC) harboring activating mutations in the epidermal growth factor receptor (EGFR) kinase domain tend to respond well to the tyrosine kinase inhibitors, gefitinib and erlotinib.  However, following clin. response, these patients typically relapse within a year of treatment.  In many cases, resistance is caused by an acquired secondary EGFR kinase domain mutation, T790M.  In vitro studies have shown that a new class of EGFR-irreversible inhibitors could overcome the resistance conferred by T790M.  Clin. trials are under way to examine the efficacy of one of these inhibitors, HKI-272, in patients with NSCLC who initially responded to gefitinib/erlotinib and subsequently relapsed.  To anticipate the possibility that patients who respond to irreversible inhibitors will develop secondary resistance to such inhibitors, as has been seen in other similar settings, we modeled acquired resistance to the dual EGFR/HER2-irreversible tyrosine kinase inhibitor HKI-272 in a NSCLC cell culture model.  We found that HKI-272-resistant clones fall into two biochem. groups based on the retention of EGFR phosphorylation in the presence of the drug.  Cells that retain phosphorylated EGFR have acquired the secondary mutation T790M.  Moreover, HKI-272 can overcome T790M resistance only at suprapharmacol. concns.  We further model mutations at EGFR C797 as a mechanism of resistance to irreversible EGFR inhibitors and show that although these mutants are resistant to the irreversible inhibitor, they retain erlotinib sensitivity.  Our findings suggest that HKI-272 treatment at maximally tolerated dosing may lead to the emergence of T790M-mediated resistance, whereas treatment with a more potent irreversible inhibitor could yield a resistance mutation at EGFR C797.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGra5FtRNHnlNLVg90H21EOLACvtfcHk0lgNiJCnP7ul5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXks1GmsLo%253D&md5=f71cd83bfabef2d7b2c5d8b6e5cfac6a</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-07-2387&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-07-2387%26sid%3Dliteratum%253Aachs%26aulast%3DGodin-Heymann%26aufirst%3DN.%26aulast%3DUlkus%26aufirst%3DL.%26aulast%3DBrannigan%26aufirst%3DB.%2BW.%26aulast%3DMcDermott%26aufirst%3DU.%26aulast%3DLamb%26aufirst%3DJ.%26aulast%3DMaheswaran%26aufirst%3DS.%26aulast%3DSettleman%26aufirst%3DJ.%26aulast%3DHaber%26aufirst%3DD.%2BA.%26atitle%3DThe%2520T790M%2520%25E2%2580%259Cgatekeeper%25E2%2580%259D%2520mutation%2520in%2520EGFR%2520mediates%2520resistance%2520to%2520low%2520concentrations%2520of%2520an%2520irreversible%2520EGFR%2520inhibitor%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2008%26volume%3D7%26spage%3D874%26epage%3D879%26doi%3D10.1158%2F1535-7163.MCT-07-2387" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Politi, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riely, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Somwar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zakowski, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kris, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varmus, H.</span></span> <span> </span><span class="NLM_article-title">Acquired resistance of lung adenocarcinomas to gefitinib or erlotinib is associated with a second mutation in the EGFR kinase domain</span>. <i>PLoS Med.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>2</i></span>, <span class="NLM_elocation-id">e73</span> <span class="refDoi"> DOI: 10.1371/journal.pmed.0020073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1371%2Fjournal.pmed.0020073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=15737014" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A280%3ADC%252BD2MzjtVehsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2005&author=W.+Paoauthor=V.+A.+Millerauthor=K.+A.+Politiauthor=G.+J.+Rielyauthor=R.+Somwarauthor=M.+F.+Zakowskiauthor=M.+G.+Krisauthor=H.+Varmus&title=Acquired+resistance+of+lung+adenocarcinomas+to+gefitinib+or+erlotinib+is+associated+with+a+second+mutation+in+the+EGFR+kinase+domain&doi=10.1371%2Fjournal.pmed.0020073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Acquired resistance of lung adenocarcinomas to gefitinib or erlotinib is associated with a second mutation in the EGFR kinase domain</span></div><div class="casAuthors">Pao William; Miller Vincent A; Politi Katerina A; Riely Gregory J; Somwar Romel; Zakowski Maureen F; Kris Mark G; Varmus Harold</div><div class="citationInfo"><span class="NLM_cas:title">PLoS medicine</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e73</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Lung adenocarcinomas from patients who respond to the tyrosine kinase inhibitors gefitinib (Iressa) or erlotinib (Tarceva) usually harbor somatic gain-of-function mutations in exons encoding the kinase domain of the epidermal growth factor receptor (EGFR).  Despite initial responses, patients eventually progress by unknown mechanisms of "acquired" resistance.  METHODS AND FINDINGS:  We show that in two of five patients with acquired resistance to gefitinib or erlotinib, progressing tumors contain, in addition to a primary drug-sensitive mutation in EGFR, a secondary mutation in exon 20, which leads to substitution of methionine for threonine at position 790 (T790M) in the kinase domain.  Tumor cells from a sixth patient with a drug-sensitive EGFR mutation whose tumor progressed on adjuvant gefitinib after complete resection also contained the T790M mutation.  This mutation was not detected in untreated tumor samples.  Moreover, no tumors with acquired resistance had KRAS mutations, which have been associated with primary resistance to these drugs.  Biochemical analyses of transfected cells and growth inhibition studies with lung cancer cell lines demonstrate that the T790M mutation confers resistance to EGFR mutants usually sensitive to either gefitinib or erlotinib.  Interestingly, a mutation analogous to T790M has been observed in other kinases with acquired resistance to another kinase inhibitor, imatinib (Gleevec).  CONCLUSION:  In patients with tumors bearing gefitinib- or erlotinib-sensitive EGFR mutations, resistant subclones containing an additional EGFR mutation emerge in the presence of drug.  This observation should help guide the search for more effective therapy against a specific subset of lung cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcREkXMV3u1nwwMthz5_Rx5rfW6udTcc2eY--_hTxK8Q5rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2MzjtVehsw%253D%253D&md5=3e626105e0dea2a38997156ae513f56b</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pmed.0020073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pmed.0020073%26sid%3Dliteratum%253Aachs%26aulast%3DPao%26aufirst%3DW.%26aulast%3DMiller%26aufirst%3DV.%2BA.%26aulast%3DPoliti%26aufirst%3DK.%2BA.%26aulast%3DRiely%26aufirst%3DG.%2BJ.%26aulast%3DSomwar%26aufirst%3DR.%26aulast%3DZakowski%26aufirst%3DM.%2BF.%26aulast%3DKris%26aufirst%3DM.%2BG.%26aulast%3DVarmus%26aufirst%3DH.%26atitle%3DAcquired%2520resistance%2520of%2520lung%2520adenocarcinomas%2520to%2520gefitinib%2520or%2520erlotinib%2520is%2520associated%2520with%2520a%2520second%2520mutation%2520in%2520the%2520EGFR%2520kinase%2520domain%26jtitle%3DPLoS%2520Med.%26date%3D2005%26volume%3D2%26doi%3D10.1371%2Fjournal.pmed.0020073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boggon, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dayaram, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janne, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kocher, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyerson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eck, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tenen, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halmos, B.</span></span> <span> </span><span class="NLM_article-title">EGFR mutation and resistance of non-small-cell lung cancer to gefitinib</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>352</i></span>,  <span class="NLM_fpage">786</span>– <span class="NLM_lpage">792</span>, <span class="refDoi"> DOI: 10.1056/NEJMoa044238</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1056%2FNEJMoa044238" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=15728811" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhsFCiurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=352&publication_year=2005&pages=786-792&author=S.+Kobayashiauthor=T.+J.+Boggonauthor=T.+Dayaramauthor=P.+A.+Janneauthor=O.+Kocherauthor=M.+Meyersonauthor=B.+E.+Johnsonauthor=M.+J.+Eckauthor=D.+G.+Tenenauthor=B.+Halmos&title=EGFR+mutation+and+resistance+of+non-small-cell+lung+cancer+to+gefitinib&doi=10.1056%2FNEJMoa044238"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">EGFR mutation and resistance of non-small-cell lung cancer to gefitinib</span></div><div class="casAuthors">Kobayashi, Susumu; Boggon, Titus J.; Dayaram, Tajhal; Jaenne, Pasi A.; Kocher, Olivier; Meyerson, Matthew; Johnson, Bruce E.; Eck, Michael J.; Tenen, Daniel G.; Halmos, Balizs</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">352</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">786-792</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">Mutations of the epidermal growth factor receptor (EGFR) gene were identified in specimens from patients with non-small-cell lung cancer who have a response to anilinoquinazoline EGFR inhibitors.  Despite the dramatic responses to such inhibitors, most patients ultimately have a relapse.  The mechanism of the drug resistance is unknown.  Here the authors report the case of a patient with EGFR-mutant, gefitinib-responsive, advanced non-small-cell lung cancer who had a relapse after 2 years of complete remission during treatment with gefitinib.  The DNA sequence of the EGFR gene in his tumor biopsy specimen at relapse revealed the presence of a second point mutation, resulting in threonine-to-methionine amino acid change at position 790 of EGFR.  Structural modeling and biochem. studies showed that this second mutation led to gefitinib resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8XXhT1hrbe7Vg90H21EOLACvtfcHk0lieVHxXBhVQjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhsFCiurk%253D&md5=808223ad566e2a3748977727cc5a9846</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1056%2FNEJMoa044238&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMoa044238%26sid%3Dliteratum%253Aachs%26aulast%3DKobayashi%26aufirst%3DS.%26aulast%3DBoggon%26aufirst%3DT.%2BJ.%26aulast%3DDayaram%26aufirst%3DT.%26aulast%3DJanne%26aufirst%3DP.%2BA.%26aulast%3DKocher%26aufirst%3DO.%26aulast%3DMeyerson%26aufirst%3DM.%26aulast%3DJohnson%26aufirst%3DB.%2BE.%26aulast%3DEck%26aufirst%3DM.%2BJ.%26aulast%3DTenen%26aufirst%3DD.%2BG.%26aulast%3DHalmos%26aufirst%3DB.%26atitle%3DEGFR%2520mutation%2520and%2520resistance%2520of%2520non-small-cell%2520lung%2520cancer%2520to%2520gefitinib%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2005%26volume%3D352%26spage%3D786%26epage%3D792%26doi%3D10.1056%2FNEJMoa044238" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thress, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paweletz, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felip, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stetson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dougherty, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Markovets, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vivancos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ercan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthews, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantarini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janne, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oxnard, G. R.</span></span> <span> </span><span class="NLM_article-title">Acquired EGFR C797S mutation mediates resistance to AZD9291 in non-small cell lung cancer harboring EGFR T790M</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">560</span>– <span class="NLM_lpage">562</span>, <span class="refDoi"> DOI: 10.1038/nm.3854</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1038%2Fnm.3854" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=25939061" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFemsbfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=560-562&author=K.+S.+Thressauthor=C.+P.+Paweletzauthor=E.+Felipauthor=B.+C.+Choauthor=D.+Stetsonauthor=B.+Doughertyauthor=Z.+Laiauthor=A.+Markovetsauthor=A.+Vivancosauthor=Y.+Kuangauthor=D.+Ercanauthor=S.+E.+Matthewsauthor=M.+Cantariniauthor=J.+C.+Barrettauthor=P.+A.+Janneauthor=G.+R.+Oxnard&title=Acquired+EGFR+C797S+mutation+mediates+resistance+to+AZD9291+in+non-small+cell+lung+cancer+harboring+EGFR+T790M&doi=10.1038%2Fnm.3854"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Acquired EGFR C797S mutation mediates resistance to AZD9291 in non-small cell lung cancer harboring EGFR T790M</span></div><div class="casAuthors">Thress, Kenneth S.; Paweletz, Cloud P.; Felip, Enriqueta; Cho, Byoung Chul; Stetson, Daniel; Dougherty, Brian; Lai, Zhongwu; Markovets, Aleksandra; Vivancos, Ana; Kuang, Yanan; Ercan, Dalia; Matthews, Sarah E.; Cantarini, Mireille; Barrett, J. Carl; Janne, Pasi A.; Oxnard, Geoffrey R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">560-562</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Here we studied cell-free plasma DNA (cfDNA) collected from subjects with advanced lung cancer whose tumors had developed resistance to the epidermal growth factor receptor (EGFR) tyrosine kinase inhibitor (TKI) AZD9291.  We first performed next-generation sequencing of cfDNA from seven subjects and detected an acquired EGFR C797S mutation in one; expression of this mutant EGFR construct in a cell line rendered it resistant to AZD9291.  We then performed droplet digital PCR on serial cfDNA specimens collected from 15 AZD9291-treated subjects.  All were pos. for the T790M mutation before treatment, but upon developing AZD9291 resistance three mol. subtypes emerged: six cases acquired the C797S mutation, five cases maintained the T790M mutation but did not acquire the C797S mutation and four cases lost the T790M mutation despite the presence of the underlying EGFR activating mutation.  Our findings provide insight into the diversity of mechanisms through which tumors acquire resistance to AZD9291 and highlight the need for therapies that are able to overcome resistance mediated by the EGFR C797S mutation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTggAW5fpGrbVg90H21EOLACvtfcHk0lieVHxXBhVQjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFemsbfJ&md5=adac4cb6c7057855aff61fd62890d010</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fnm.3854&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.3854%26sid%3Dliteratum%253Aachs%26aulast%3DThress%26aufirst%3DK.%2BS.%26aulast%3DPaweletz%26aufirst%3DC.%2BP.%26aulast%3DFelip%26aufirst%3DE.%26aulast%3DCho%26aufirst%3DB.%2BC.%26aulast%3DStetson%26aufirst%3DD.%26aulast%3DDougherty%26aufirst%3DB.%26aulast%3DLai%26aufirst%3DZ.%26aulast%3DMarkovets%26aufirst%3DA.%26aulast%3DVivancos%26aufirst%3DA.%26aulast%3DKuang%26aufirst%3DY.%26aulast%3DErcan%26aufirst%3DD.%26aulast%3DMatthews%26aufirst%3DS.%2BE.%26aulast%3DCantarini%26aufirst%3DM.%26aulast%3DBarrett%26aufirst%3DJ.%2BC.%26aulast%3DJanne%26aufirst%3DP.%2BA.%26aulast%3DOxnard%26aufirst%3DG.%2BR.%26atitle%3DAcquired%2520EGFR%2520C797S%2520mutation%2520mediates%2520resistance%2520to%2520AZD9291%2520in%2520non-small%2520cell%2520lung%2520cancer%2520harboring%2520EGFR%2520T790M%26jtitle%3DNat.%2520Med.%26date%3D2015%26volume%3D21%26spage%3D560%26epage%3D562%26doi%3D10.1038%2Fnm.3854" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smaill, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, K.</span></span> <span> </span><span class="NLM_article-title">Targeting EGFR(L858R/T790M) and EGFR(L858R/T790M/C797S) resistance mutations in NSCLC: Current developments in medicinal chemistry</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">1550</span>– <span class="NLM_lpage">1581</span>, <span class="refDoi"> DOI: 10.1002/med.21488</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1002%2Fmed.21488" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=29377179" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVyqu73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2018&pages=1550-1581&author=X.+Luauthor=L.+Yuauthor=Z.+Zhangauthor=X.+Renauthor=J.+B.+Smaillauthor=K.+Ding&title=Targeting+EGFR%28L858R%2FT790M%29+and+EGFR%28L858R%2FT790M%2FC797S%29+resistance+mutations+in+NSCLC%3A+Current+developments+in+medicinal+chemistry&doi=10.1002%2Fmed.21488"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting EGFRL858R/T790M and EGFRL858R/T790M/C797S resistance mutations in NSCLC: Current developments in medicinal chemistry</span></div><div class="casAuthors">Lu, Xiaoyun; Yu, Lei; Zhang, Zhang; Ren, Xiaomei; Smaill, Jeff B.; Ding, Ke</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1550-1581</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Both the first-generation reversible epidermal growth factor receptor (EGFR) inhibitors gefitinib and erlotinib and the second-generation covalent epidermal growth factor receptor tyrosine kinase inhibitor (EGFR-TKI) afatinib have significantly improved the survival of non-small-cell lung cancer (NSCLC) patients with activating EGFR mutations.  However, a secondary EGFRT790M mutation leads to the clin. acquired resistance to the first- and second-generation EGFR-TKIs drugs.  A no. of the third-generation wild-type sparing EGFR inhibitors, for example, WZ4002, CO1686, AZD9291, HM61713, EGF816, ASP8173, and PF0674775, have been developed, among which AZD9291 has been approved by US FDA for the treatment of NSCLC patients with EGFRT790M.  More recently, a tertiary EGFRC797S mutation was reported as the dominant resistance mechanism to the third-generation irreversible inhibitors.  It is highly desirable to develop the fourth-generation EGFR inhibitors.  This review summarizes the mechanisms of acquired resistance and the latest medicinal chem. advances on the third- and fourth-generation EGFR inhibitors, with special attention being paid to the allosteric and reversible inhibitors combating the tertiary EGFRC797S mutation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNK_1c8SI0qbVg90H21EOLACvtfcHk0lgk1cvQVsFMRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVyqu73P&md5=71288118ca1d9eaab6d07b056bb5ab56</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1002%2Fmed.21488&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21488%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DX.%26aulast%3DYu%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DRen%26aufirst%3DX.%26aulast%3DSmaill%26aufirst%3DJ.%2BB.%26aulast%3DDing%26aufirst%3DK.%26atitle%3DTargeting%2520EGFR%2528L858R%252FT790M%2529%2520and%2520EGFR%2528L858R%252FT790M%252FC797S%2529%2520resistance%2520mutations%2520in%2520NSCLC%253A%2520Current%2520developments%2520in%2520medicinal%2520chemistry%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2018%26volume%3D38%26spage%3D1550%26epage%3D1581%26doi%3D10.1002%2Fmed.21488" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Patel, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pawara, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ansari, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Surana, S.</span></span> <span> </span><span class="NLM_article-title">Recent updates on third generation EGFR inhibitors and emergence of fourth generation EGFR inhibitors to combat C797S resistance</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>142</i></span>,  <span class="NLM_fpage">32</span>– <span class="NLM_lpage">47</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2017.05.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1016%2Fj.ejmech.2017.05.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=28526474" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnvFGmtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=142&publication_year=2017&pages=32-47&author=H.+Patelauthor=R.+Pawaraauthor=A.+Ansariauthor=S.+Surana&title=Recent+updates+on+third+generation+EGFR+inhibitors+and+emergence+of+fourth+generation+EGFR+inhibitors+to+combat+C797S+resistance&doi=10.1016%2Fj.ejmech.2017.05.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Recent updates on third generation EGFR inhibitors and emergence of fourth generation EGFR inhibitors to combat C797S resistance</span></div><div class="casAuthors">Patel, Harun; Pawara, Rahul; Ansari, Azim; Surana, Sanjay</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">142</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">32-47</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A review.  EGFR T790M mutation leads to resistance to most of clin. available EGFR TKIs.  Third-generation EGFR TKIs against the T790M mutation have been in active clin. development, which includes osimertinib, rociletinib, HM61713, ASP8273, EGF816, and PF-06747775.  On the other hand recently EGFR C797S mutation was reported to be a leading mechanism of resistance to the third-generation inhibitors.  The C797S mutation appears to be an ideal target for overcoming the acquired resistance to the third generation inhibitors.  This review summarizes the third generation inhibitors, synthesis, their mechanism of resistance and latest development on the discovery of a fourth-generation EGFR TKIs and U to Y allosteric strategies to combat the C797S EGFR resistance problem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprj-V5hUSHGrVg90H21EOLACvtfcHk0lgk1cvQVsFMRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnvFGmtbo%253D&md5=d0ed35bc7deae5ea37a95910df356c1d</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2017.05.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2017.05.027%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DH.%26aulast%3DPawara%26aufirst%3DR.%26aulast%3DAnsari%26aufirst%3DA.%26aulast%3DSurana%26aufirst%3DS.%26atitle%3DRecent%2520updates%2520on%2520third%2520generation%2520EGFR%2520inhibitors%2520and%2520emergence%2520of%2520fourth%2520generation%2520EGFR%2520inhibitors%2520to%2520combat%2520C797S%2520resistance%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D142%26spage%3D32%26epage%3D47%26doi%3D10.1016%2Fj.ejmech.2017.05.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tan, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soo, R. A.</span></span> <span> </span><span class="NLM_article-title">Next-generation epidermal growth factor receptor tyrosine kinase inhibitors in epidermal growth factor receptor-mutant non-small cell lung cancer</span>. <i>Lung Cancer</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">59</span>– <span class="NLM_lpage">68</span>, <span class="refDoi"> DOI: 10.1016/j.lungcan.2016.01.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1016%2Fj.lungcan.2016.01.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=26898616" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A280%3ADC%252BC28jivVymtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=2016&pages=59-68&author=C.+S.+Tanauthor=B.+C.+Choauthor=R.+A.+Soo&title=Next-generation+epidermal+growth+factor+receptor+tyrosine+kinase+inhibitors+in+epidermal+growth+factor+receptor-mutant+non-small+cell+lung+cancer&doi=10.1016%2Fj.lungcan.2016.01.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Next-generation epidermal growth factor receptor tyrosine kinase inhibitors in epidermal growth factor receptor -mutant non-small cell lung cancer</span></div><div class="casAuthors">Tan Chee-Seng; Cho Byoung-Chul; Soo Ross A</div><div class="citationInfo"><span class="NLM_cas:title">Lung cancer (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">59-68</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Since the discovery of sensitizing EGFR mutations as a predictive marker of sensitivity to EGFR tyrosine kinase inhibitors (TKIs), the field of targeted therapy in non-small cell lung cancer (NSCLC) has been revolutionized.  Patients harbouring these sensitizing mutations treated with EGFR TKI have derived significant clinical outcome when compared with standard platinum based chemotherapy doublets.  However disease progression invariably occurs at a median of about 9-13 months from initiation treatment, if acquired resistance commonly due to the development of EGFR T790M mutation.  A novel class of "third generation" EGFR TKIs have been developed that is sensitising and T790M mutant-specific whilst sparing WT EGFR, representing a significant breakthrough in the treatment in NSCLC patients with acquired resistance harboring these genotypes.  Early phase clinical data suggest the third generation EGFR TKIs such as osimertinib, rociletinib, and HM61713 are highly efficacious and well tolerated.  Another promising class of EGFR TKI such as AZD3759 has been designed to penetrate blood brain barrier to treat brain metastases and leptomeningeal disease and has showed promising responses in patients with brain metastases.  Acquired resistance to third generation EGFR TKIs has been reported including EGFR C797S.  Given its non-invasive nature, plasma ctDNA is being explored as a possible approach to detect T790M mutation and to also inform on novel molecular mechansims of tertiary resistance to third generation EGFR TKIs.  An understanding of the mechanisms of acquired resistance to the third-generation EGFR TKIs will greatly aid in the development of the next generation of EGFR TKIs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ6MbARC-V4Af3jf77P2BG3fW6udTcc2eYGepQOy2litbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28jivVymtA%253D%253D&md5=b1ab30df28665605ce44ce1b5de385ff</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.lungcan.2016.01.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.lungcan.2016.01.003%26sid%3Dliteratum%253Aachs%26aulast%3DTan%26aufirst%3DC.%2BS.%26aulast%3DCho%26aufirst%3DB.%2BC.%26aulast%3DSoo%26aufirst%3DR.%2BA.%26atitle%3DNext-generation%2520epidermal%2520growth%2520factor%2520receptor%2520tyrosine%2520kinase%2520inhibitors%2520in%2520epidermal%2520growth%2520factor%2520receptor-mutant%2520non-small%2520cell%2520lung%2520cancer%26jtitle%3DLung%2520Cancer%26date%3D2016%26volume%3D93%26spage%3D59%26epage%3D68%26doi%3D10.1016%2Fj.lungcan.2016.01.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span> <span> </span><span class="NLM_article-title">EAI045: The fourth-generation EGFR inhibitor overcoming T790M and C797S resistance</span>. <i>Cancer Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>385</i></span>,  <span class="NLM_fpage">51</span>– <span class="NLM_lpage">54</span>, <span class="refDoi"> DOI: 10.1016/j.canlet.2016.11.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1016%2Fj.canlet.2016.11.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=27840244" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVeqtL%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=385&publication_year=2017&pages=51-54&author=S.+Wangauthor=Y.+Songauthor=D.+Liu&title=EAI045%3A+The+fourth-generation+EGFR+inhibitor+overcoming+T790M+and+C797S+resistance&doi=10.1016%2Fj.canlet.2016.11.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">EAI045: The fourth-generation EGFR inhibitor overcoming T790M and C797S resistance</span></div><div class="casAuthors">Wang, Shuhang; Song, Yongping; Liu, Delong</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Letters (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">385</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">51-54</span>CODEN:
                <span class="NLM_cas:coden">CALEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The third-generation tyrosine kinase inhibitors (TKI), AZD9291 (osimertinib) and CO-1686 (rociletinib) of epidermal growth factor receptor (EGFR) are highly active against T790M pos. non-small cell lung cancer (NSCLC).  However, resistance develops rapidly.  EGFR C797S mutation was reported to be a leading mechanism of resistance to the third-generation inhibitors.  The C797S mutation appears to be an ideal target for overcoming the acquired resistance to the third-generation inhibitors.  This review summarizes the latest development on the discovery of a fourth-generation EGFR TKI, EAI045.3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2GMAIDfgfeLVg90H21EOLACvtfcHk0liuG4pmT1kj-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVeqtL%252FM&md5=e429055412bec2ec2da28b2b959753ae</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.canlet.2016.11.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.canlet.2016.11.008%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DS.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DD.%26atitle%3DEAI045%253A%2520The%2520fourth-generation%2520EGFR%2520inhibitor%2520overcoming%2520T790M%2520and%2520C797S%2520resistance%26jtitle%3DCancer%2520Lett.%26date%3D2017%26volume%3D385%26spage%3D51%26epage%3D54%26doi%3D10.1016%2Fj.canlet.2016.11.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span> <span> </span><span class="NLM_article-title">Discovery of potent and noncovalent reversible EGFR kinase inhibitors of EGFR(L858R/T790M/C797S)</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">869</span>– <span class="NLM_lpage">873</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.8b00564</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.8b00564" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC1MXpvFams7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=869-873&author=Q.+Liauthor=T.+Zhangauthor=S.+Liauthor=L.+Tongauthor=J.+Liauthor=Z.+Suauthor=F.+Fengauthor=D.+Sunauthor=Y.+Tongauthor=X.+Wangauthor=Z.+Zhaoauthor=L.+Zhuauthor=J.+Dingauthor=H.+Liauthor=H.+Xieauthor=Y.+Xu&title=Discovery+of+potent+and+noncovalent+reversible+EGFR+kinase+inhibitors+of+EGFR%28L858R%2FT790M%2FC797S%29&doi=10.1021%2Facsmedchemlett.8b00564"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Potent and Noncovalent Reversible EGFR Kinase Inhibitors of EGFRL858R/T790M/C797S</span></div><div class="casAuthors">Li, Qiannan; Zhang, Tao; Li, Shiliang; Tong, Linjiang; Li, Junyu; Su, Zhicheng; Feng, Fang; Sun, Deheng; Tong, Yi; Wang, Xia; Zhao, Zhenjiang; Zhu, Lili; Ding, Jian; Li, Honglin; Xie, Hua; Xu, Yufang</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">869-873</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In this paper, we describe the discovery and optimization of a series of noncovalent reversible epidermal growth factor receptor inhibitors of EGFRL858R/T790M/C797S.  One of the most promising compds., I, inhibited the enzymic activity of EGFRL858R/T790M/C797S with an IC50 value of 2.2 nM.  Cell proliferation assays showed that I effectively and selectively inhibited the growth of EGFRL858R/T790M/C797S-dependent cells.  This series of compds., which occupy both the ATP binding site and the allosteric site of the EGFR kinase, may serve as a basis for the development of fourth-generation EGFR inhibitors for L858R/T790M/C797S mutants.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqcQ04p4560bVg90H21EOLACvtfcHk0liuG4pmT1kj-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXpvFams7o%253D&md5=396eb9b29217fbb74189193a544819b3</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.8b00564&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.8b00564%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DTong%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DSu%26aufirst%3DZ.%26aulast%3DFeng%26aufirst%3DF.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DTong%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DZhu%26aufirst%3DL.%26aulast%3DDing%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DXie%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DY.%26atitle%3DDiscovery%2520of%2520potent%2520and%2520noncovalent%2520reversible%2520EGFR%2520kinase%2520inhibitors%2520of%2520EGFR%2528L858R%252FT790M%252FC797S%2529%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D10%26spage%3D869%26epage%3D873%26doi%3D10.1021%2Facsmedchemlett.8b00564" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Park, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, H.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mah, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of EGF receptor inhibitors that are selective for the d746-750/T790M/C797S mutant through structure-based de novo design</span>. <i>Angew. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>129</i></span>,  <span class="NLM_fpage">7742</span>– <span class="NLM_lpage">7746</span>, <span class="refDoi"> DOI: 10.1002/ange.201703389</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1002%2Fange.201703389" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2017&pages=7742-7746&author=H.+Parkauthor=H.-Y.+Jungauthor=S.+Mahauthor=S.+Hong&title=Discovery+of+EGF+receptor+inhibitors+that+are+selective+for+the+d746-750%2FT790M%2FC797S+mutant+through+structure-based+de+novo+design&doi=10.1002%2Fange.201703389"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1002%2Fange.201703389&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fange.201703389%26sid%3Dliteratum%253Aachs%26aulast%3DPark%26aufirst%3DH.%26aulast%3DJung%26aufirst%3DH.-Y.%26aulast%3DMah%26aufirst%3DS.%26aulast%3DHong%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520EGF%2520receptor%2520inhibitors%2520that%2520are%2520selective%2520for%2520the%2520d746-750%252FT790M%252FC797S%2520mutant%2520through%2520structure-based%2520de%2520novo%2520design%26jtitle%3DAngew.%2520Chem.%26date%3D2017%26volume%3D129%26spage%3D7742%26epage%3D7746%26doi%3D10.1002%2Fange.201703389" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yun, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, K.</span></span> <span> </span><span class="NLM_article-title">Structure-based design of 5-methylpyrimidopyridone derivatives as new wild-type sparing inhibitors of the epidermal growth factor receptor triple mutant (EGFR(L858R/T790M/C797S))</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">7302</span>– <span class="NLM_lpage">7308</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00576</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00576" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtlOhurnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=7302-7308&author=J.+Shenauthor=T.+Zhangauthor=S.+J.+Zhuauthor=M.+Sunauthor=L.+Tongauthor=M.+Laiauthor=R.+Zhangauthor=W.+Xuauthor=R.+Wuauthor=J.+Dingauthor=C.+H.+Yunauthor=H.+Xieauthor=X.+Luauthor=K.+Ding&title=Structure-based+design+of+5-methylpyrimidopyridone+derivatives+as+new+wild-type+sparing+inhibitors+of+the+epidermal+growth+factor+receptor+triple+mutant+%28EGFR%28L858R%2FT790M%2FC797S%29%29&doi=10.1021%2Facs.jmedchem.9b00576"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of 5-Methylpyrimidopyridone Derivatives as New Wild-Type Sparing Inhibitors of the Epidermal Growth Factor Receptor Triple Mutant (EGFRL858R/T790M/C797S)</span></div><div class="casAuthors">Shen, Jiayi; Zhang, Tao; Zhu, Su-Jie; Sun, Min; Tong, Linjiang; Lai, Mengzhen; Zhang, Rong; Xu, Wei; Wu, Ruibo; Ding, Jian; Yun, Cai-Hong; Xie, Hua; Lu, Xiaoyun; Ding, Ke</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">7302-7308</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Azaheterocyclyl-substituted pyridopyrimidinones such as I were prepd. as selective inhibitors of the mutant epidermal growth factor receptor (EGFR) EGFRL858R/T790M/C797S with selectivity over the wild-type EGFR for potential use in treating osimertinib-resistant non-small-cell lung cancer.  I exhibited an IC50 of 27.5 nM against the EGFRL858R/T790M/C797S mutant, while being significantly less potent against the wild-type EGFR with an IC50 value of > 1.0 μM.  Cocrystallog. structure detn. and computational investigation were conducted to elucidate its target selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQFRlnoJItB7Vg90H21EOLACvtfcHk0liTVqMmxCNMVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtlOhurnI&md5=70ca1144f3b73d0ccd412406b1072a1f</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00576&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00576%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DZhu%26aufirst%3DS.%2BJ.%26aulast%3DSun%26aufirst%3DM.%26aulast%3DTong%26aufirst%3DL.%26aulast%3DLai%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DR.%26aulast%3DXu%26aufirst%3DW.%26aulast%3DWu%26aufirst%3DR.%26aulast%3DDing%26aufirst%3DJ.%26aulast%3DYun%26aufirst%3DC.%2BH.%26aulast%3DXie%26aufirst%3DH.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DDing%26aufirst%3DK.%26atitle%3DStructure-based%2520design%2520of%25205-methylpyrimidopyridone%2520derivatives%2520as%2520new%2520wild-type%2520sparing%2520inhibitors%2520of%2520the%2520epidermal%2520growth%2520factor%2520receptor%2520triple%2520mutant%2520%2528EGFR%2528L858R%252FT790M%252FC797S%2529%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D7302%26epage%3D7308%26doi%3D10.1021%2Facs.jmedchem.9b00576" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grabe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lategahn, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rauh, D.</span></span> <span> </span><span class="NLM_article-title">C797S Resistance: The undruggable EGFR mutation in non-small cell lung cancer?</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">779</span>– <span class="NLM_lpage">782</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.8b00314</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.8b00314" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlGhu7vN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=779-782&author=T.+Grabeauthor=J.+Lategahnauthor=D.+Rauh&title=C797S+Resistance%3A+The+undruggable+EGFR+mutation+in+non-small+cell+lung+cancer%3F&doi=10.1021%2Facsmedchemlett.8b00314"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">C797S Resistance: The Undruggable EGFR Mutation in Non-Small Cell Lung Cancer?</span></div><div class="casAuthors">Grabe, Tobias; Lategahn, Jonas; Rauh, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">779-782</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The first evidence of osimertinib resistance mediated by the epidermal growth factor receptor (EGFR) mutation C797S was reported three years ago.  Since then, no major breakthroughs have been achieved to target the clin. relevant mutant variant that impedes covalent bond formation with irreversible EGFR inhibitors.  Although several biochem. active compds. have been described, only a few inhibitors that potently act on the cellular level or in vivo have been introduced so far.  Herein, the authors give an overview of current approaches in the field and highlight the challenges that need to be addressed in future research projects to overcome the C797S-mediated drug resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkYihQlaOa1bVg90H21EOLACvtfcHk0liTVqMmxCNMVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlGhu7vN&md5=8bafb477f8dbbc4a74e0fec125cd3455</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.8b00314&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.8b00314%26sid%3Dliteratum%253Aachs%26aulast%3DGrabe%26aufirst%3DT.%26aulast%3DLategahn%26aufirst%3DJ.%26aulast%3DRauh%26aufirst%3DD.%26atitle%3DC797S%2520Resistance%253A%2520The%2520undruggable%2520EGFR%2520mutation%2520in%2520non-small%2520cell%2520lung%2520cancer%253F%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2018%26volume%3D9%26spage%3D779%26epage%3D782%26doi%3D10.1021%2Facsmedchemlett.8b00314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Induced protein degradation: an emerging drug discovery paradigm</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">101</span>– <span class="NLM_lpage">114</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1038%2Fnrd.2016.211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=27885283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=101-114&author=A.+C.+Laiauthor=C.+M.+Crews&title=Induced+protein+degradation%3A+an+emerging+drug+discovery+paradigm&doi=10.1038%2Fnrd.2016.211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Induced protein degradation: an emerging drug discovery paradigm</span></div><div class="casAuthors">Lai, Ashton C.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">101-114</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small-mol. drug discovery has traditionally focused on occupancy of a binding site that directly affects protein function, and this approach typically precludes targeting proteins that lack such amenable sites.  Furthermore, high systemic drug exposures may be needed to maintain sufficient target inhibition in vivo, increasing the risk of undesirable off-target effects.  Induced protein degrdn. is an alternative approach that is event-driven: upon drug binding, the target protein is tagged for elimination.  Emerging technologies based on proteolysis-targeting chimaeras (PROTACs) that exploit cellular quality control machinery to selectively degrade target proteins are attracting considerable attention in the pharmaceutical industry owing to the advantages they could offer over traditional small-mol. strategies.  These advantages include the potential to reduce systemic drug exposure, the ability to counteract increased target protein expression that often accompanies inhibition of protein function and the potential ability to target proteins that are not currently therapeutically tractable, such as transcription factors, scaffolding and regulatory proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj-O7zXGOpXLVg90H21EOLACvtfcHk0liTVqMmxCNMVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP&md5=a01ff43e9b696fe4dc03f0815158a98f</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.211%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DInduced%2520protein%2520degradation%253A%2520an%2520emerging%2520drug%2520discovery%2520paradigm%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D101%26epage%3D114%26doi%3D10.1038%2Fnrd.2016.211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Toure, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Small-molecule PROTACS: new approaches to protein degradation</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">1966</span>– <span class="NLM_lpage">1973</span>, <span class="refDoi"> DOI: 10.1002/anie.201507978</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1002%2Fanie.201507978" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC28XnsF2ltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=1966-1973&author=M.+Toureauthor=C.+M.+Crews&title=Small-molecule+PROTACS%3A+new+approaches+to+protein+degradation&doi=10.1002%2Fanie.201507978"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule PROTACS: New Approaches to Protein Degradation</span></div><div class="casAuthors">Toure, Momar; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1966-1973</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The current inhibitor-based approach to therapeutics has inherent limitations owing to its occupancy-based model: there is a need to maintain high systemic exposure to ensure sufficient in vivo inhibition, high in vivo concns. bring potential for off-target side effects, and there is a need to bind to an active site, thus limiting the drug target space.  As an alternative, induced protein degrdn. lacks these limitations.  Based on an event-driven model, this approach offers a novel catalytic mechanism to irreversibly inhibit protein function by targeting protein destruction through recruitment to the cellular quality control machinery.  Prior protein degrading strategies have lacked therapeutic potential.  However, recent reports of small-mol.-based proteolysis-targeting chimeras (PROTACs) have demonstrated that this technol. can effectively decrease the cellular levels of several protein classes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdaqlIjCSsgrVg90H21EOLACvtfcHk0lhER2Uo2r4_wQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnsF2ltA%253D%253D&md5=47838cd15719757317cb8282c26cc59e</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1002%2Fanie.201507978&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201507978%26sid%3Dliteratum%253Aachs%26aulast%3DToure%26aufirst%3DM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DSmall-molecule%2520PROTACS%253A%2520new%2520approaches%2520to%2520protein%2520degradation%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26spage%3D1966%26epage%3D1973%26doi%3D10.1002%2Fanie.201507978" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Small-molecule modulation of protein homeostasis</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>117</i></span>,  <span class="NLM_fpage">11269</span>– <span class="NLM_lpage">11301</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.7b00077</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.7b00077" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1Ohur3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2017&pages=11269-11301&author=G.+M.+Burslemauthor=C.+M.+Crews&title=Small-molecule+modulation+of+protein+homeostasis&doi=10.1021%2Facs.chemrev.7b00077"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Modulation of Protein Homeostasis</span></div><div class="casAuthors">Burslem, George M.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">11269-11301</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Control of protein levels by nucleic-acid-based technologies has proven to be a useful research tool but lacks the advantages of small mols. with respect to cell permeability, temporal control, and the potential generation of therapeutics.  In this Review, we discuss the technologies available for the control of intracellular protein levels with small mols. and compare the various systems available.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcARydbgPK9bVg90H21EOLACvtfcHk0lhER2Uo2r4_wQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1Ohur3K&md5=f9f8dd98e2c71c7166d2d500b514d578</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.7b00077&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.7b00077%26sid%3Dliteratum%253Aachs%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DSmall-molecule%2520modulation%2520of%2520protein%2520homeostasis%26jtitle%3DChem.%2520Rev.%26date%3D2017%26volume%3D117%26spage%3D11269%26epage%3D11301%26doi%3D10.1021%2Facs.chemrev.7b00077" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumagai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercurio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshaies, R. J.</span></span> <span> </span><span class="NLM_article-title">Protacs: chimeric molecules that target proteins to the Skp1-Cullin-F box complex for ubiquitination and degradation</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">8554</span>– <span class="NLM_lpage">8559</span>, <span class="refDoi"> DOI: 10.1073/pnas.141230798</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1073%2Fpnas.141230798" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=11438690" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BD3MXls1Wisbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2001&pages=8554-8559&author=K.+M.+Sakamotoauthor=K.+B.+Kimauthor=A.+Kumagaiauthor=F.+Mercurioauthor=C.+M.+Crewsauthor=R.+J.+Deshaies&title=Protacs%3A+chimeric+molecules+that+target+proteins+to+the+Skp1-Cullin-F+box+complex+for+ubiquitination+and+degradation&doi=10.1073%2Fpnas.141230798"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Protacs: chimeric molecules that target proteins to the Skp1-Cullin-F box complex for ubiquitination and degradation</span></div><div class="casAuthors">Sakamoto, Kathleen M.; Kim, Kyung B.; Kumagai, Akiko; Mercurio, Frank; Crews, Craig M.; Deshaies, Raymond J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">8554-8559</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The intracellular levels of many proteins are regulated by ubiquitin-dependent proteolysis.  One of the best-characterized enzymes that catalyzes the attachment of ubiquitin to proteins is a ubiquitin ligase complex, Skp1-Cullin-F box complex contg. Hrt1 (SCF).  We sought to artificially target a protein to the SCF complex for ubiquitination and degrdn.  To this end, we tested methionine aminopeptidase-2 (MetAP-2), which covalently binds the angiogenesis inhibitor ovalicin.  A chimeric compd., protein-targeting chimeric mol. 1 (Protac-1), was synthesized to recruit MetAP-2 to SCF.  One domain of Protac-1 contains the IκBα phosphopeptide that is recognized by the F-box protein β-TRCP, whereas the other domain is composed of ovalicin.  We show that MetAP-2 can be tethered to SCFβ-TRCP, ubiquitinated, and degraded in a Protac-1-dependent manner.  In the future, this approach may be useful for conditional inactivation of proteins, and for targeting disease-causing proteins for destruction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp49fYtuEVlJrVg90H21EOLACvtfcHk0lhER2Uo2r4_wQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXls1Wisbk%253D&md5=540358035222f745f7b6367b38781a21</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1073%2Fpnas.141230798&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.141230798%26sid%3Dliteratum%253Aachs%26aulast%3DSakamoto%26aufirst%3DK.%2BM.%26aulast%3DKim%26aufirst%3DK.%2BB.%26aulast%3DKumagai%26aufirst%3DA.%26aulast%3DMercurio%26aufirst%3DF.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DDeshaies%26aufirst%3DR.%2BJ.%26atitle%3DProtacs%253A%2520chimeric%2520molecules%2520that%2520target%2520proteins%2520to%2520the%2520Skp1-Cullin-F%2520box%2520complex%2520for%2520ubiquitination%2520and%2520degradation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2001%26volume%3D98%26spage%3D8554%26epage%3D8559%26doi%3D10.1073%2Fpnas.141230798" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Salami, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Waste disposal-An attractive strategy for cancer therapy</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>355</i></span>,  <span class="NLM_fpage">1163</span>– <span class="NLM_lpage">1167</span>, <span class="refDoi"> DOI: 10.1126/science.aam7340</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1126%2Fscience.aam7340" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=28302825" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC2sXktFOmt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=355&publication_year=2017&pages=1163-1167&author=J.+Salamiauthor=C.+M.+Crews&title=Waste+disposal-An+attractive+strategy+for+cancer+therapy&doi=10.1126%2Fscience.aam7340"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Waste disposal-An attractive strategy for cancer therapy</span></div><div class="casAuthors">Salami, Jemilat; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">355</span>
        (<span class="NLM_cas:issue">6330</span>),
    <span class="NLM_cas:pages">1163-1167</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review.  Targeted therapies for cancer are typically small mols. or monoclonal antibodies that act by inhibiting the activity of specific proteins that drive tumor growth.  Although many of these drugs are effective in cancer patients, the response is often not durable because tumor cells develop resistance to the drugs.  Another limitation of this strategy is that not all oncogenic driver proteins are "druggable" enzymes or receptors with activities that can be inhibited.  Here the authors describe an alternative approach to targeted therapy that is based on co-opting the cellular quality-control machinery-the ubiquitin-proteasome system-to remove specific cancer-causing proteins from the cell.  The authors first discuss examples of existing cancer drugs that work by degrading specific proteins and then review recent progress in the rational design and preclin. testing of small mols. that induce selective degrdn. of specific target proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVoZn-YpfXmbVg90H21EOLACvtfcHk0lhKFySuNYp6bQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXktFOmt74%253D&md5=918621d50a82b7797aa0300537a4486e</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1126%2Fscience.aam7340&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aam7340%26sid%3Dliteratum%253Aachs%26aulast%3DSalami%26aufirst%3DJ.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DWaste%2520disposal-An%2520attractive%2520strategy%2520for%2520cancer%2520therapy%26jtitle%3DScience%26date%3D2017%26volume%3D355%26spage%3D1163%26epage%3D1167%26doi%3D10.1126%2Fscience.aam7340" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">DRUG DEVELOPMENT. Phthalimide conjugation as a strategy for in vivo target protein degradation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>348</i></span>,  <span class="NLM_fpage">1376</span>– <span class="NLM_lpage">1381</span>, <span class="refDoi"> DOI: 10.1126/science.aab1433</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1126%2Fscience.aab1433" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=25999370" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC2MXpvFansLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=348&publication_year=2015&pages=1376-1381&author=G.+E.+Winterauthor=D.+L.+Buckleyauthor=J.+Paulkauthor=J.+M.+Robertsauthor=A.+Souzaauthor=S.+Dhe-Paganonauthor=J.+E.+Bradner&title=DRUG+DEVELOPMENT.+Phthalimide+conjugation+as+a+strategy+for+in+vivo+target+protein+degradation&doi=10.1126%2Fscience.aab1433"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Phthalimide conjugation as a strategy for in vivo target protein degradation</span></div><div class="casAuthors">Winter, Georg E.; Buckley, Dennis L.; Paulk, Joshiawa; Roberts, Justin M.; Souza, Amanda; Dhe-Paganon, Sirano; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">348</span>
        (<span class="NLM_cas:issue">6241</span>),
    <span class="NLM_cas:pages">1376-1381</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The development of effective pharmacol. inhibitors of multidomain scaffold proteins, notably transcription factors, is a particularly challenging problem.  In part, this is because many small-mol. antagonists disrupt the activity of only one domain in the target protein.  The authors devised a chem. strategy that promotes ligand-dependent target protein degrdn. using as an example the transcriptional coactivator BRD4, a protein crit. for cancer cell growth and survival.  The authors appended a competitive antagonist of BET bromodomains to a phthalimide moiety to hijack the cereblon E3 ubiquitin ligase complex.  The resultant compd., dBET1, induced highly selective cereblon-dependent BET protein degrdn. in vitro and in vivo and delayed leukemia progression in mice.  A second series of probes resulted in selective degrdn. of the cytosolic protein FKBP12.  This chem. strategy for controlling target protein stability may have implications for therapeutically targeting previously intractable proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1ZW4zjRjXi7Vg90H21EOLACvtfcHk0lhKFySuNYp6bQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXpvFansLk%253D&md5=b0277cc95d318d926b96e7a119066e3b</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1126%2Fscience.aab1433&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aab1433%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DG.%2BE.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DSouza%26aufirst%3DA.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DDRUG%2520DEVELOPMENT.%2520Phthalimide%2520conjugation%2520as%2520a%2520strategy%2520for%2520in%2520vivo%2520target%2520protein%2520degradation%26jtitle%3DScience%26date%3D2015%26volume%3D348%26spage%3D1376%26epage%3D1381%26doi%3D10.1126%2Fscience.aab1433" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mares, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, I. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campos, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miah, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulholland, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Routly, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafson, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zinn, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimamura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergamini, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faelth-Savitski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willard, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flanagan, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casillas, L. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Votta, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">den Besten, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Famm, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruidenier, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harling, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Churcher, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Catalytic in vivo protein knockdown by small-molecule PROTACs</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">611</span>– <span class="NLM_lpage">617</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1858</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1038%2Fnchembio.1858" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=26075522" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=611-617&author=D.+P.+Bondesonauthor=A.+Maresauthor=I.+E.+Smithauthor=E.+Koauthor=S.+Camposauthor=A.+H.+Miahauthor=K.+E.+Mulhollandauthor=N.+Routlyauthor=D.+L.+Buckleyauthor=J.+L.+Gustafsonauthor=N.+Zinnauthor=P.+Grandiauthor=S.+Shimamuraauthor=G.+Bergaminiauthor=M.+Faelth-Savitskiauthor=M.+Bantscheffauthor=C.+Coxauthor=D.+A.+Gordonauthor=R.+R.+Willardauthor=J.+J.+Flanaganauthor=L.+N.+Casillasauthor=B.+J.+Vottaauthor=W.+den+Bestenauthor=K.+Fammauthor=L.+Kruidenierauthor=P.+S.+Carterauthor=J.+D.+Harlingauthor=I.+Churcherauthor=C.+M.+Crews&title=Catalytic+in+vivo+protein+knockdown+by+small-molecule+PROTACs&doi=10.1038%2Fnchembio.1858"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Catalytic in vivo protein knockdown by small-molecule PROTACs</span></div><div class="casAuthors">Bondeson, Daniel P.; Mares, Alina; Smith, Ian E. D.; Ko, Eunhwa; Campos, Sebastien; Miah, Afjal H.; Mulholland, Katie E.; Routly, Natasha; Buckley, Dennis L.; Gustafson, Jeffrey L.; Zinn, Nico; Grandi, Paola; Shimamura, Satoko; Bergamini, Giovanna; Faelth-Savitski, Maria; Bantscheff, Marcus; Cox, Carly; Gordon, Deborah A.; Willard, Ryan R.; Flanagan, John J.; Casillas, Linda N.; Votta, Bartholomew J.; den Besten, Willem; Famm, Kristoffer; Kruidenier, Laurens; Carter, Paul S.; Harling, John D.; Churcher, Ian; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">611-617</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The current predominant therapeutic paradigm is based on maximizing drug-receptor occupancy to achieve clin. benefit.  This strategy, however, generally requires excessive drug concns. to ensure sufficient occupancy, often leading to adverse side effects.  Here, we describe major improvements to the proteolysis targeting chimeras (PROTACs) method, a chem. knockdown strategy in which a heterobifunctional mol. recruits a specific protein target to an E3 ubiquitin ligase, resulting in the target's ubiquitination and degrdn.  These compds. behave catalytically in their ability to induce the ubiquitination of super-stoichiometric quantities of proteins, providing efficacy that is not limited by equil. occupancy.  We present two PROTACs that are capable of specifically reducing protein levels by >90% at nanomolar concns.  In addn., mouse studies indicate that they provide broad tissue distribution and knockdown of the targeted protein in tumor xenografts.  Together, these data demonstrate a protein knockdown system combining many of the favorable properties of small-mol. agents with the potent protein knockdown of RNAi and CRISPR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonhnajD3CxS7Vg90H21EOLACvtfcHk0lhKFySuNYp6bQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7rP&md5=31eb8756c5661f78ab4527a50dc1e251</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1858&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1858%26sid%3Dliteratum%253Aachs%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DMares%26aufirst%3DA.%26aulast%3DSmith%26aufirst%3DI.%2BE.%26aulast%3DKo%26aufirst%3DE.%26aulast%3DCampos%26aufirst%3DS.%26aulast%3DMiah%26aufirst%3DA.%2BH.%26aulast%3DMulholland%26aufirst%3DK.%2BE.%26aulast%3DRoutly%26aufirst%3DN.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DGustafson%26aufirst%3DJ.%2BL.%26aulast%3DZinn%26aufirst%3DN.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DShimamura%26aufirst%3DS.%26aulast%3DBergamini%26aufirst%3DG.%26aulast%3DFaelth-Savitski%26aufirst%3DM.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DCox%26aufirst%3DC.%26aulast%3DGordon%26aufirst%3DD.%2BA.%26aulast%3DWillard%26aufirst%3DR.%2BR.%26aulast%3DFlanagan%26aufirst%3DJ.%2BJ.%26aulast%3DCasillas%26aufirst%3DL.%2BN.%26aulast%3DVotta%26aufirst%3DB.%2BJ.%26aulast%3Dden%2BBesten%26aufirst%3DW.%26aulast%3DFamm%26aufirst%3DK.%26aulast%3DKruidenier%26aufirst%3DL.%26aulast%3DCarter%26aufirst%3DP.%2BS.%26aulast%3DHarling%26aufirst%3DJ.%2BD.%26aulast%3DChurcher%26aufirst%3DI.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DCatalytic%2520in%2520vivo%2520protein%2520knockdown%2520by%2520small-molecule%2520PROTACs%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D611%26epage%3D617%26doi%3D10.1038%2Fnchembio.1858" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Noble, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Endicott, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, L. N.</span></span> <span> </span><span class="NLM_article-title">Protein kinase inhibitors: insights into drug design from structure</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>303</i></span>,  <span class="NLM_fpage">1800</span>– <span class="NLM_lpage">1805</span>, <span class="refDoi"> DOI: 10.1126/science.1095920</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1126%2Fscience.1095920" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=15031492" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BD2cXitFehurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=303&publication_year=2004&pages=1800-1805&author=M.+E.+Nobleauthor=J.+A.+Endicottauthor=L.+N.+Johnson&title=Protein+kinase+inhibitors%3A+insights+into+drug+design+from+structure&doi=10.1126%2Fscience.1095920"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Kinase Inhibitors: Insights into Drug Design from Structure</span></div><div class="casAuthors">Noble, Martin E. M.; Endicott, Jane A.; Johnson, Louise N.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">303</span>
        (<span class="NLM_cas:issue">5665</span>),
    <span class="NLM_cas:pages">1800-1805</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review.  Protein kinases are targets for treatment of a no. of diseases.  This review focuses on kinase inhibitors that are in the clinic or in clin. trials and for which structural information is available.  Structures have informed drug design and have illuminated the mechanism of inhibition.  We review progress with the receptor tyrosine kinases (growth factor receptors EGFR, VEGFR, and FGFR) and nonreceptor tyrosine kinases (Bcr-Abl), where advances have been made with cancer therapeutic agents such as Herceptin and Gleevec.  Among the serine-threonine kinases, p38, Rho-kinase, cyclin-dependent kinases, and Chk1 have been targeted with productive results for inflammation and cancer.  Structures have provided insights into targeting the inactive or active form of the kinase, for targeting the global constellation of residues at the ATP site or less conserved addnl. pockets or single residues, and into targeting noncatalytic domains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWqtKz7cvN_rVg90H21EOLACvtfcHk0lh0WrCZJtGPCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXitFehurc%253D&md5=7fd3f60bee46e59307436eeed285d6ae</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1126%2Fscience.1095920&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1095920%26sid%3Dliteratum%253Aachs%26aulast%3DNoble%26aufirst%3DM.%2BE.%26aulast%3DEndicott%26aufirst%3DJ.%2BA.%26aulast%3DJohnson%26aufirst%3DL.%2BN.%26atitle%3DProtein%2520kinase%2520inhibitors%253A%2520insights%2520into%2520drug%2520design%2520from%2520structure%26jtitle%3DScience%26date%3D2004%26volume%3D303%26spage%3D1800%26epage%3D1805%26doi%3D10.1126%2Fscience.1095920" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buhimschi, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaime-Figueroa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamman, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishchenko, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Lessons in PROTAC design from selective degradation with a promiscuous warhead</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">78</span>– <span class="NLM_lpage">87</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.09.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1016%2Fj.chembiol.2017.09.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=29129718" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=78-87&author=D.+P.+Bondesonauthor=B.+E.+Smithauthor=G.+M.+Burslemauthor=A.+D.+Buhimschiauthor=J.+Hinesauthor=S.+Jaime-Figueroaauthor=J.+Wangauthor=B.+D.+Hammanauthor=A.+Ishchenkoauthor=C.+M.+Crews&title=Lessons+in+PROTAC+design+from+selective+degradation+with+a+promiscuous+warhead&doi=10.1016%2Fj.chembiol.2017.09.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Lessons in PROTAC Design from Selective Degradation with a Promiscuous Warhead</span></div><div class="casAuthors">Bondeson, Daniel P.; Smith, Blake E.; Burslem, George M.; Buhimschi, Alexandru D.; Hines, John; Jaime-Figueroa, Saul; Wang, Jing; Hamman, Brian D.; Ishchenko, Alexey; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">78-87.e5</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Inhibiting protein function selectively is a major goal of modern drug discovery.  Here, we report a previously understudied benefit of small mol. proteolysis-targeting chimeras (PROTACs) that recruit E3 ubiquitin ligases to target proteins for their ubiquitination and subsequent proteasome-mediated degrdn.  Using promiscuous CRBN- and VHL-recruiting PROTACs that bind >50 kinases, we show that only a subset of bound targets is degraded.  The basis of this selectivity relies on protein-protein interactions between the E3 ubiquitin ligase and the target protein, as illustrated by engaged proteins that are not degraded as a result of unstable ternary complexes with PROTAC-recruited E3 ligases.  In contrast, weak PROTAC:target protein affinity can be stabilized by high-affinity target:PROTAC:ligase trimer interactions, leading to efficient degrdn.  This study highlights design guidelines for generating potent PROTACs as well as possibilities for degrading undruggable proteins immune to traditional small-mol. inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoISOBudOLyFrVg90H21EOLACvtfcHk0lh0WrCZJtGPCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjN&md5=30fac502705a2344c4261cf531e6bf9d</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.09.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.09.010%26sid%3Dliteratum%253Aachs%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DSmith%26aufirst%3DB.%2BE.%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DBuhimschi%26aufirst%3DA.%2BD.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DHamman%26aufirst%3DB.%2BD.%26aulast%3DIshchenko%26aufirst%3DA.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DLessons%2520in%2520PROTAC%2520design%2520from%2520selective%2520degradation%2520with%2520a%2520promiscuous%2520warhead%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D78%26epage%3D87%26doi%3D10.1016%2Fj.chembiol.2017.09.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ottis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Proteolysis-targeting chimeras: induced protein degradation as a therapeutic strategy</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">892</span>– <span class="NLM_lpage">898</span>, <span class="refDoi"> DOI: 10.1021/acschembio.6b01068</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.6b01068" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjs12ns70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=892-898&author=P.+Ottisauthor=C.+M.+Crews&title=Proteolysis-targeting+chimeras%3A+induced+protein+degradation+as+a+therapeutic+strategy&doi=10.1021%2Facschembio.6b01068"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis-Targeting Chimeras: Induced Protein Degradation as a Therapeutic Strategy</span></div><div class="casAuthors">Ottis, Philipp; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">892-898</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Until recently, the only ways to reduce specific protein signaling were to either knock down the target by RNAi or to interfere with the signaling by inhibiting an enzyme or receptor within the signal transduction cascade.  Herein, we review an emerging class of small mol. pharmacol. agents, called PROTACs, that present a novel approach to specifically target proteins and their resp. signaling pathways.  These heterobifunctional mols. utilize endogenous cellular quality control machinery by recruiting it to target proteins in order to induce their degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohB6xwkQMTprVg90H21EOLACvtfcHk0lh0WrCZJtGPCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjs12ns70%253D&md5=eada521c10faa1e456e0885fd390c554</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Facschembio.6b01068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.6b01068%26sid%3Dliteratum%253Aachs%26aulast%3DOttis%26aufirst%3DP.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DProteolysis-targeting%2520chimeras%253A%2520induced%2520protein%2520degradation%2520as%2520a%2520therapeutic%2520strategy%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26spage%3D892%26epage%3D898%26doi%3D10.1021%2Facschembio.6b01068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cromm, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Targeted protein degradation: from chemical biology to drug discovery</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">1181</span>– <span class="NLM_lpage">1190</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.05.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1016%2Fj.chembiol.2017.05.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=28648379" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVOjtrrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2017&pages=1181-1190&author=P.+M.+Crommauthor=C.+M.+Crews&title=Targeted+protein+degradation%3A+from+chemical+biology+to+drug+discovery&doi=10.1016%2Fj.chembiol.2017.05.024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted Protein Degradation: from Chemical Biology to Drug Discovery</span></div><div class="casAuthors">Cromm, Philipp M.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1181-1190</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Traditional pharmaceutical drug discovery is almost exclusively focused on directly controlling protein activity to cure diseases.  Modulators of protein activity, esp. inhibitors, are developed and applied at high concn. to achieve maximal effects.  Thereby, reduced bioavailability and off-target effects can hamper compd. efficacy.  Nucleic acid-based strategies that control protein function by affecting expression have emerged as an alternative.  However, metabolic stability and broad bioavailability represent development hurdles that remain to be overcome for these approaches.  More recently, utilizing the cell's own protein destruction machinery for selective degrdn. of essential drivers of human disorders has opened up a new and exciting area of drug discovery.  Small-mol.-induced proteolysis of selected substrates offers the potential of reaching beyond the limitations of the current pharmaceutical paradigm to expand the druggable target space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrosuhmnPE0RrVg90H21EOLACvtfcHk0lgsRut0A60QYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVOjtrrO&md5=3c8c6d9488396d5eb08ca51cee20bbf4</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.05.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.05.024%26sid%3Dliteratum%253Aachs%26aulast%3DCromm%26aufirst%3DP.%2BM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DTargeted%2520protein%2520degradation%253A%2520from%2520chemical%2520biology%2520to%2520drug%2520discovery%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2017%26volume%3D24%26spage%3D1181%26epage%3D1190%26doi%3D10.1016%2Fj.chembiol.2017.05.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Kinase inhibitors: the road ahead</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">353</span>– <span class="NLM_lpage">377</span>, <span class="refDoi"> DOI: 10.1038/nrd.2018.21</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1038%2Fnrd.2018.21" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=29545548" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC1cXks12ktbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=353-377&author=F.+M.+Fergusonauthor=N.+S.+Gray&title=Kinase+inhibitors%3A+the+road+ahead&doi=10.1038%2Fnrd.2018.21"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Kinase inhibitors: the road ahead</span></div><div class="casAuthors">Ferguson, Fleur M.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">353-377</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  Receptor tyrosine kinase signaling pathways have been successfully targeted to inhibit proliferation and angiogenesis for cancer therapy.  However, kinase deregulation has been firmly demonstrated to play an essential role in virtually all major disease areas.  Kinase inhibitor drug discovery programs have recently broadened their focus to include an expanded range of kinase targets and therapeutic areas.  In this Review, we provide an overview of the novel targets, biol. processes and disease areas that kinase-targeting small mols. are being developed against, highlight the assocd. challenges and assess the strategies and technologies that are enabling efficient generation of highly optimized kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEZPospb1TP7Vg90H21EOLACvtfcHk0lgsRut0A60QYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXks12ktbg%253D&md5=34e42911d8a70301905460c3507e737a</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2018.21&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2018.21%26sid%3Dliteratum%253Aachs%26aulast%3DFerguson%26aufirst%3DF.%2BM.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DKinase%2520inhibitors%253A%2520the%2520road%2520ahead%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2018%26volume%3D17%26spage%3D353%26epage%3D377%26doi%3D10.1038%2Fnrd.2018.21" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pettersson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">PROteolysis TArgeting Chimeras (PROTACs)- past, present and future</span>. <i>Drug Discovery Today Technol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">15</span>– <span class="NLM_lpage">27</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2019.01.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1016%2Fj.ddtec.2019.01.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=31200855" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A280%3ADC%252BB3M3mvFWkuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2019&pages=15-27&author=M.+Petterssonauthor=C.+M.+Crews&title=PROteolysis+TArgeting+Chimeras+%28PROTACs%29-+past%2C+present+and+future&doi=10.1016%2Fj.ddtec.2019.01.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">PROteolysis TArgeting Chimeras (PROTACs) - Past, present and future</span></div><div class="casAuthors">Pettersson Mariell; Crews Craig M</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today. Technologies</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">15-27</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The majority of currently used therapeutics are small molecule-based and utilize occupancy-driven pharmacology as the mode of action (MOA), in which the protein function is modulated via temporary inhibition.  New modalities that operate using alternative MOAs are essential for tapping into the "undruggable" proteome.  The PROteolysis Targeting Chimera (PROTAC) technology provides an attractive new approach that utilizes an event-driven MOA.  Small molecule-based heterobifunctional PROTACs modulate protein target levels by hijacking the ubiquitin-proteasome system to induce degradation of the target.  Here, we address important milestones in the development of the PROTAC technology, as well as emphasize key findings from this previous year and highlight future directions of this promising drug discovery modality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQCRHissLLXoRfKb6ELK8t0fW6udTcc2eYbvFZC_DhIjbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3M3mvFWkuw%253D%253D&md5=c629f3128321e136f1f247e05ee6f182</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2019.01.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2019.01.002%26sid%3Dliteratum%253Aachs%26aulast%3DPettersson%26aufirst%3DM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DPROteolysis%2520TArgeting%2520Chimeras%2520%2528PROTACs%2529-%2520past%252C%2520present%2520and%2520future%26jtitle%3DDrug%2520Discovery%2520Today%2520Technol.%26date%3D2019%26volume%3D31%26spage%3D15%26epage%3D27%26doi%3D10.1016%2Fj.ddtec.2019.01.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Olson, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erb, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doctor, Z. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwiatkowski, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boukhali, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haas, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nomanbhoy, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Pharmacological perturbation of CDK9 using selective CDK9 inhibition or degradation</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">163</span>– <span class="NLM_lpage">170</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2538</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1038%2Fnchembio.2538" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=29251720" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFyrsr7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2018&pages=163-170&author=C.+M.+Olsonauthor=B.+Jiangauthor=M.+A.+Erbauthor=Y.+Liangauthor=Z.+M.+Doctorauthor=Z.+Zhangauthor=T.+Zhangauthor=N.+Kwiatkowskiauthor=M.+Boukhaliauthor=J.+L.+Greenauthor=W.+Haasauthor=T.+Nomanbhoyauthor=E.+S.+Fischerauthor=R.+A.+Youngauthor=J.+E.+Bradnerauthor=G.+E.+Winterauthor=N.+S.+Gray&title=Pharmacological+perturbation+of+CDK9+using+selective+CDK9+inhibition+or+degradation&doi=10.1038%2Fnchembio.2538"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological perturbation of CDK9 using selective CDK9 inhibition or degradation</span></div><div class="casAuthors">Olson, Calla M.; Jiang, Baishan; Erb, Michael A.; Liang, Yanke; Doctor, Zainab M.; Zhang, Zinan; Zhang, Tinghu; Kwiatkowski, Nicholas; Boukhali, Myriam; Green, Jennifer L.; Haas, Wilhelm; Nomanbhoy, Tyzoon; Fischer, Eric S.; Young, Richard A.; Bradner, James E.; Winter, Georg E.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">163-170</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Cyclin-dependent kinase 9 (CDK9), an important regulator of transcriptional elongation, is a promising target for cancer therapy, particularly for cancers driven by transcriptional dysregulation.  The authors characterized NVP-2, a selective ATP-competitive CDK9 inhibitor, and THAL-SNS-032, a selective CDK9 degrader consisting of a CDK-binding SNS-032 ligand linked to a thalidomide deriv. that binds the E3 ubiquitin ligase Cereblon (CRBN).  To the authors' surprise, THAL-SNS-032 induced rapid degrdn. of CDK9 without affecting the levels of other SNS-032 targets.  Moreover, the transcriptional changes elicited by THAL-SNS-032 were more like those caused by NVP-2 than those induced by SNS-032.  Notably, compd. washout did not significantly reduce levels of THAL-SNS-032-induced apoptosis, suggesting that CDK9 degrdn. had prolonged cytotoxic effects compared with CDK9 inhibition.  Thus, the authors' findings suggest that thalidomide conjugation represents a promising strategy for converting multi-targeted inhibitors into selective degraders and reveal that kinase degrdn. can induce distinct pharmacol. effects compared with inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWnT9r0GIlmbVg90H21EOLACvtfcHk0liMydNKoL2jUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFyrsr7O&md5=88aabb209b4cb7516a652d6c75a35667</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2538&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2538%26sid%3Dliteratum%253Aachs%26aulast%3DOlson%26aufirst%3DC.%2BM.%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DErb%26aufirst%3DM.%2BA.%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DDoctor%26aufirst%3DZ.%2BM.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DKwiatkowski%26aufirst%3DN.%26aulast%3DBoukhali%26aufirst%3DM.%26aulast%3DGreen%26aufirst%3DJ.%2BL.%26aulast%3DHaas%26aufirst%3DW.%26aulast%3DNomanbhoy%26aufirst%3DT.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DWinter%26aufirst%3DG.%2BE.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DPharmacological%2520perturbation%2520of%2520CDK9%2520using%2520selective%2520CDK9%2520inhibition%2520or%2520degradation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2018%26volume%3D14%26spage%3D163%26epage%3D170%26doi%3D10.1038%2Fnchembio.2538" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgess, K.</span></span> <span> </span><span class="NLM_article-title">PROTACs suppression of CDK4/6, crucial kinases for cell cycle regulation in cancer</span>. <i>Chem. Commun</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">2704</span>– <span class="NLM_lpage">2707</span>, <span class="refDoi"> DOI: 10.1039/C9CC00163H</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1039%2FC9CC00163H" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=30758029" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjtFGht7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2019&pages=2704-2707&author=B.+Zhaoauthor=K.+Burgess&title=PROTACs+suppression+of+CDK4%2F6%2C+crucial+kinases+for+cell+cycle+regulation+in+cancer&doi=10.1039%2FC9CC00163H"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">PROTACs suppression of CDK4/6, crucial kinases for cell cycle regulation in cancer</span></div><div class="casAuthors">Zhao, Bosheng; Burgess, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">2704-2707</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">PROTACs based on two selective, FDA approved, CDK4/6 inhibitors were formed.  One of them, based on palbociclib, potently initiates degrdn. of these CDK proteins, and suppresses phosphorylation of retinoblastoma protein (Rb) leading to cell cycle arrest.  These PROTACs are active at nanomolar concns., and appear to be the first for CDK4/6.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBWDRf65Fl6bVg90H21EOLACvtfcHk0liMydNKoL2jUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjtFGht7c%253D&md5=8798e9aa7ecc0e6504c2717e815b04fc</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1039%2FC9CC00163H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC9CC00163H%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DB.%26aulast%3DBurgess%26aufirst%3DK.%26atitle%3DPROTACs%2520suppression%2520of%2520CDK4%252F6%252C%2520crucial%2520kinases%2520for%2520cell%2520cycle%2520regulation%2520in%2520cancer%26jtitle%3DChem.%2520Commun%26date%3D2019%26volume%3D55%26spage%3D2704%26epage%3D2707%26doi%3D10.1039%2FC9CC00163H" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Development of dual and selective degraders of cyclin-dependent kinases 4 and 6</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">6321</span>– <span class="NLM_lpage">6326</span>, <span class="refDoi"> DOI: 10.1002/anie.201901336</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1002%2Fanie.201901336" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC1MXmtlKqsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2019&pages=6321-6326&author=B.+Jiangauthor=E.+S.+Wangauthor=K.+A.+Donovanauthor=Y.+Liangauthor=E.+S.+Fischerauthor=T.+Zhangauthor=N.+S.+Gray&title=Development+of+dual+and+selective+degraders+of+cyclin-dependent+kinases+4+and+6&doi=10.1002%2Fanie.201901336"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Development of Dual and Selective Degraders of Cyclin-Dependent Kinases 4 and 6</span></div><div class="casAuthors">Jiang, Baishan; Wang, Eric S.; Donovan, Katherine A.; Liang, Yanke; Fischer, Eric S.; Zhang, Tinghu; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6321-6326</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Cyclin-dependent kinases 4 and 6 (CDK4/6) are key regulators of the cell cycle, and there are FDA-approved CDK4/6 inhibitors for treating patients with metastatic breast cancer.  However, due to conservation of their ATP-binding sites, development of selective agents has remained elusive.  Here, we report imide-based degrader mols. capable of degrading both CDK4/6, or selectively degrading either CDK4 or CDK6.  We were also able to tune the activity of these mols. against Ikaros (IKZF1) and Aiolos (IKZF3), which are well-established targets of imide-based degraders.  We found that in mantle cell lymphoma cell lines, combined IKZF1/3 degrdn. with dual CDK4/6 degrdn. produced enhanced anti-proliferative effects compared to CDK4/6 inhibition, CDK4/6 degrdn., or IKZF1/3 degrdn.  In summary, we report here the first compds. capable of inducing selective degrdn. of CDK4 and CDK6 as tools to pharmacol. dissect their distinct biol. functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_Vsqs-SM6hrVg90H21EOLACvtfcHk0liMydNKoL2jUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXmtlKqsrw%253D&md5=39a6937b24abad0940d359020e2b2f74</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1002%2Fanie.201901336&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201901336%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DWang%26aufirst%3DE.%2BS.%26aulast%3DDonovan%26aufirst%3DK.%2BA.%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DDevelopment%2520of%2520dual%2520and%2520selective%2520degraders%2520of%2520cyclin-dependent%2520kinases%25204%2520and%25206%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2019%26volume%3D58%26spage%3D6321%26epage%3D6326%26doi%3D10.1002%2Fanie.201901336" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brand, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwiatkowski, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span> <span> </span><span class="NLM_article-title">Homolog-selective degradation as a strategy to probe the function of CDK6 in AML</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">300</span>– <span class="NLM_lpage">306 e9</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2018.11.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1016%2Fj.chembiol.2018.11.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=30595531" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC1MXnsVSj" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2019&pages=300-306+e9&author=M.+Brandauthor=B.+Jiangauthor=S.+Bauerauthor=K.+A.+Donovanauthor=Y.+Liangauthor=E.+S.+Wangauthor=R.+P.+Nowakauthor=J.+C.+Yuanauthor=T.+Zhangauthor=N.+Kwiatkowskiauthor=A.+C.+Mullerauthor=E.+S.+Fischerauthor=N.+S.+Grayauthor=G.+E.+Winter&title=Homolog-selective+degradation+as+a+strategy+to+probe+the+function+of+CDK6+in+AML&doi=10.1016%2Fj.chembiol.2018.11.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Homolog-Selective Degradation as a Strategy to Probe the Function of CDK6 in AML</span></div><div class="casAuthors">Brand, Matthias; Jiang, Baishan; Bauer, Sophie; Donovan, Katherine A.; Liang, Yanke; Wang, Eric S.; Nowak, Radoslaw P.; Yuan, Jingting C.; Zhang, Tinghu; Kwiatkowski, Nicholas; Mueller, Andre C.; Fischer, Eric S.; Gray, Nathanael S.; Winter, Georg E.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">300-306.e9</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The design of selective small mols. is often stymied by similar ligand binding pockets.  Here, we report BSJ-03-123, a phthalimide-based degrader that exploits protein-interface determinants to achieve proteome-wide selectivity for the degrdn. of cyclin-dependent kinase 6 (CDK6).  Pharmacol. CDK6 degrdn. targets a selective dependency of acute myeloid leukemia cells, and transcriptomics and phosphoproteomics profiling of acute degrdn. of CDK6 enabled dynamic mapping of its immediate role in coordinating signaling and transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGognoz8FFoR9LVg90H21EOLACvtfcHk0ljcalHLm5c2Hg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXnsVSj&md5=f8c1c3520c6c0218763d8769c59b4342</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2018.11.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2018.11.006%26sid%3Dliteratum%253Aachs%26aulast%3DBrand%26aufirst%3DM.%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DBauer%26aufirst%3DS.%26aulast%3DDonovan%26aufirst%3DK.%2BA.%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DE.%2BS.%26aulast%3DNowak%26aufirst%3DR.%2BP.%26aulast%3DYuan%26aufirst%3DJ.%2BC.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DKwiatkowski%26aufirst%3DN.%26aulast%3DMuller%26aufirst%3DA.%2BC.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DWinter%26aufirst%3DG.%2BE.%26atitle%3DHomolog-selective%2520degradation%2520as%2520a%2520strategy%2520to%2520probe%2520the%2520function%2520of%2520CDK6%2520in%2520AML%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2019%26volume%3D26%26spage%3D300%26epage%3D306%2520e9%26doi%3D10.1016%2Fj.chembiol.2018.11.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cromm, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samarasinghe, K. T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Addressing kinase-independent functions of Fak via PROTAC-mediated degradation</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>140</i></span>,  <span class="NLM_fpage">17019</span>– <span class="NLM_lpage">17026</span>, <span class="refDoi"> DOI: 10.1021/jacs.8b08008</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.8b08008" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC1cXit1CitLnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2018&pages=17019-17026&author=P.+M.+Crommauthor=K.+T.+G.+Samarasingheauthor=J.+Hinesauthor=C.+M.+Crews&title=Addressing+kinase-independent+functions+of+Fak+via+PROTAC-mediated+degradation&doi=10.1021%2Fjacs.8b08008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Addressing Kinase-Independent Functions of Fak via PROTAC-Mediated Degradation</span></div><div class="casAuthors">Cromm, Philipp M.; Samarasinghe, Kusal T. G.; Hines, John; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">17019-17026</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Enzymic inhibition proved to be a successful modality for the development of many small-mol. drugs.  In recent years, small-mol.-induced protein degrdn. has emerged as an orthogonal therapeutic strategy that has the potential to expand the druggable target space.  Focal adhesion kinase (Fak) is a key player in tumor invasion and metastasis, acting simultaneously as a kinase and a scaffold for several signaling proteins.  While previous efforts to modulate Fak activity were limited to kinase inhibitors with low success in clin. studies, protein degrdn. offers a possibility to simultaneously block Fak's kinase signaling and scaffolding capabilities.  Here, the authors report the development of a selective and potent Fak degrader, PROTAC-3, which outperforms a clin. candidate, defactinib, with respect to Fak activation as well as Fak-mediated cell migration and invasion.  These results underline the potential that PROTACs offer in expanding the druggable space and controlling protein functions that are not easily addressed by traditional small-mol. therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTn8r4NmDDObVg90H21EOLACvtfcHk0ljcalHLm5c2Hg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXit1CitLnF&md5=b97b1a646a8f9dbf7a2bed640d0f001d</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fjacs.8b08008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.8b08008%26sid%3Dliteratum%253Aachs%26aulast%3DCromm%26aufirst%3DP.%2BM.%26aulast%3DSamarasinghe%26aufirst%3DK.%2BT.%2BG.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DAddressing%2520kinase-independent%2520functions%2520of%2520Fak%2520via%2520PROTAC-mediated%2520degradation%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2018%26volume%3D140%26spage%3D17019%26epage%3D17026%26doi%3D10.1021%2Fjacs.8b08008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaime-Figueroa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McQuaid, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toure, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">The advantages of targeted protein degradation over inhibition: An RTK case study</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">67</span>– <span class="NLM_lpage">77</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.09.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1016%2Fj.chembiol.2017.09.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=29129716" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=67-77&author=G.+M.+Burslemauthor=B.+E.+Smithauthor=A.+C.+Laiauthor=S.+Jaime-Figueroaauthor=D.+C.+McQuaidauthor=D.+P.+Bondesonauthor=M.+Toureauthor=H.+Dongauthor=Y.+Qianauthor=J.+Wangauthor=A.+P.+Crewauthor=J.+Hinesauthor=C.+M.+Crews&title=The+advantages+of+targeted+protein+degradation+over+inhibition%3A+An+RTK+case+study&doi=10.1016%2Fj.chembiol.2017.09.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">The Advantages of Targeted Protein Degradation Over Inhibition: An RTK Case Study</span></div><div class="casAuthors">Burslem, George M.; Smith, Blake E.; Lai, Ashton C.; Jaime-Figueroa, Saul; McQuaid, Daniel C.; Bondeson, Daniel P.; Toure, Momar; Dong, Hanqing; Qian, Yimin; Wang, Jing; Crew, Andrew P.; Hines, John; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">67-77.e3</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Proteolysis targeting chimera (PROTAC) technol. has emerged over the last two decades as a powerful tool for targeted degrdn. of endogenous proteins.  Herein we describe the development of PROTACs for receptor tyrosine kinases, a protein family yet to be targeted for induced protein degrdn.  The use of VHL-recruiting PROTACs against this protein family reveals several advantages of degrdn. over inhibition alone: direct comparisons of fully functional, target-degrading PROTACs with target-inhibiting variants that contain an inactivated E3 ligase-recruiting ligand show that degrdn. leads to more potent inhibition of cell proliferation and a more durable and sustained downstream signaling response, and thus addresses the kinome rewiring challenge seen with many receptor tyrosine kinase inhibitors.  Combined, these findings demonstrate the ability to target receptor tyrosine kinases for degrdn. using the PROTAC technol. and outline the advantages of this degrdn.-based approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4yDhRDWqNv7Vg90H21EOLACvtfcHk0ljcalHLm5c2Hg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjM&md5=0ac39a6f5836ea3772a0396356efb630</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.09.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.09.009%26sid%3Dliteratum%253Aachs%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DSmith%26aufirst%3DB.%2BE.%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DMcQuaid%26aufirst%3DD.%2BC.%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DToure%26aufirst%3DM.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DThe%2520advantages%2520of%2520targeted%2520protein%2520degradation%2520over%2520inhibition%253A%2520An%2520RTK%2520case%2520study%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D67%26epage%3D77%26doi%3D10.1016%2Fj.chembiol.2017.09.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paez, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janne, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tracy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greulich, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabriel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herman, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaye, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindeman, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boggon, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naoki, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasaki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujii, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eck, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellers, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyerson, M.</span></span> <span> </span><span class="NLM_article-title">EGFR mutations in lung cancer: correlation with clinical response to gefitinib therapy</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>304</i></span>,  <span class="NLM_fpage">1497</span>– <span class="NLM_lpage">500</span>, <span class="refDoi"> DOI: 10.1126/science.1099314</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1126%2Fscience.1099314" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=15118125" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksVGmsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=304&publication_year=2004&pages=1497-500&author=J.+G.+Paezauthor=P.+A.+Janneauthor=J.+C.+Leeauthor=S.+Tracyauthor=H.+Greulichauthor=S.+Gabrielauthor=P.+Hermanauthor=F.+J.+Kayeauthor=N.+Lindemanauthor=T.+J.+Boggonauthor=K.+Naokiauthor=H.+Sasakiauthor=Y.+Fujiiauthor=M.+J.+Eckauthor=W.+R.+Sellersauthor=B.+E.+Johnsonauthor=M.+Meyerson&title=EGFR+mutations+in+lung+cancer%3A+correlation+with+clinical+response+to+gefitinib+therapy&doi=10.1126%2Fscience.1099314"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">EGFR Mutations in Lung Cancer: Correlation with Clinical Response to Gefitinib Therapy</span></div><div class="casAuthors">Paez, J. Guillermo; Jaenne, Pasi A.; Lee, Jeffrey C.; Tracy, Sean; Greulich, Heidi; Gabriel, Stacey; Herman, Paula; Kaye, Frederic J.; Lindeman, Neal; Boggon, Titus J.; Naoki, Katsuhiko; Sasaki, Hidefumi; Fujii, Yoshitaka; Eck, Michael J.; Sellers, William R.; Johnson, Bruce E.; Meyerson, Matthew</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">304</span>
        (<span class="NLM_cas:issue">5676</span>),
    <span class="NLM_cas:pages">1497-1500</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Receptor tyrosine kinase genes were sequenced in non-small cell lung cancer (NSCLC) and matched normal tissue.  Somatic mutations of the epidermal growth factor receptor gene EGFR were found in 15 of 58 unselected tumors from Japan and 1 of 61 from the United States.  Treatment with the EGFR kinase inhibitor gefitinib (Iressa) causes tumor regression in some patients with NSCLC, more frequently in Japan.  EGFR mutations were found in addnl. lung cancer samples from U.S. patients who responded to gefitinib therapy and in a lung adenocarcinoma cell line that was hypersensitive to growth inhibition by gefitinib, but not in gefitinib-insensitive tumors or cell lines.  These results suggest that EGFR mutations may predict sensitivity to gefitinib.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGxLJH5f3Kb7Vg90H21EOLACvtfcHk0lgyKEPH9TO60g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksVGmsbs%253D&md5=6df1b11bf9b3c284fbe000e34c61ebac</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1126%2Fscience.1099314&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1099314%26sid%3Dliteratum%253Aachs%26aulast%3DPaez%26aufirst%3DJ.%2BG.%26aulast%3DJanne%26aufirst%3DP.%2BA.%26aulast%3DLee%26aufirst%3DJ.%2BC.%26aulast%3DTracy%26aufirst%3DS.%26aulast%3DGreulich%26aufirst%3DH.%26aulast%3DGabriel%26aufirst%3DS.%26aulast%3DHerman%26aufirst%3DP.%26aulast%3DKaye%26aufirst%3DF.%2BJ.%26aulast%3DLindeman%26aufirst%3DN.%26aulast%3DBoggon%26aufirst%3DT.%2BJ.%26aulast%3DNaoki%26aufirst%3DK.%26aulast%3DSasaki%26aufirst%3DH.%26aulast%3DFujii%26aufirst%3DY.%26aulast%3DEck%26aufirst%3DM.%2BJ.%26aulast%3DSellers%26aufirst%3DW.%2BR.%26aulast%3DJohnson%26aufirst%3DB.%2BE.%26aulast%3DMeyerson%26aufirst%3DM.%26atitle%3DEGFR%2520mutations%2520in%2520lung%2520cancer%253A%2520correlation%2520with%2520clinical%2520response%2520to%2520gefitinib%2520therapy%26jtitle%3DScience%26date%3D2004%26volume%3D304%26spage%3D1497%26epage%3D500%26doi%3D10.1126%2Fscience.1099314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tahirovic, Y. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Truax, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jecs, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuo, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sum, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cvijic, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schroeder, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liotta, D. C.</span></span> <span> </span><span class="NLM_article-title">Discovery of N-alkyl piperazine side chain based CXCR4 antagonists with improved drug-like properties</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">446</span>– <span class="NLM_lpage">451</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.8b00030</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.8b00030" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnt1amt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=446-451&author=Y.+A.+Tahirovicauthor=V.+M.+Truaxauthor=R.+J.+Wilsonauthor=E.+Jecsauthor=H.+H.+Nguyenauthor=E.+J.+Millerauthor=M.+B.+Kimauthor=K.+M.+Kuoauthor=T.+Wangauthor=C.+S.+Sumauthor=M.+E.+Cvijicauthor=G.+M.+Schroederauthor=L.+J.+Wilsonauthor=D.+C.+Liotta&title=Discovery+of+N-alkyl+piperazine+side+chain+based+CXCR4+antagonists+with+improved+drug-like+properties&doi=10.1021%2Facsmedchemlett.8b00030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of N-Alkyl Piperazine Side Chain Based CXCR4 Antagonists with Improved Drug-like Properties</span></div><div class="casAuthors">Tahirovic, Yesim A.; Truax, Valarie M.; Wilson, Robert J.; Jecs, Edgars; Nguyen, Huy H.; Miller, Eric J.; Kim, Michelle B.; Kuo, Katie M.; Wang, Tao; Sum, Chi S.; Cvijic, Mary E.; Schroeder, Gretchen M.; Wilson, Lawrence J.; Liotta, Dennis C.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">446-451</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel series of CXCR4 antagonists with piperidinyl and piperazinyl alkylamine side chains designed as Bu amine replacements are described.  Several of these compds. showed similar activity to the parent compd. TIQ-15 (5) in a SDF-1 induced calcium flux assay.  Preliminary structure-activity relationship investigations led us to identify a series contg. N-Pr piperazine side chain analogs exemplified by 16 with improved off-target effects as measured in a muscarinic acetylcholine receptor (mAChR) calcium flux assay and in a limited drug safety panel screen.  Further efforts to explore SAR and optimize drug properties led to the identification of the N'-ethyl-N-propyl-piperazine tetrahydroisoquinoline deriv. 44 and the N-propyl-piperazine benzimidazole compd. 37, which gave the best overall profiles with no mAChR or CYP450 inhibition, good permeability in PAMPA assays, and metabolic stability in human liver microsomes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0J9JgLuoU4bVg90H21EOLACvtfcHk0lgyKEPH9TO60g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnt1amt7w%253D&md5=601d8a648b9a8b814ff2cb5c4382382c</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.8b00030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.8b00030%26sid%3Dliteratum%253Aachs%26aulast%3DTahirovic%26aufirst%3DY.%2BA.%26aulast%3DTruax%26aufirst%3DV.%2BM.%26aulast%3DWilson%26aufirst%3DR.%2BJ.%26aulast%3DJecs%26aufirst%3DE.%26aulast%3DNguyen%26aufirst%3DH.%2BH.%26aulast%3DMiller%26aufirst%3DE.%2BJ.%26aulast%3DKim%26aufirst%3DM.%2BB.%26aulast%3DKuo%26aufirst%3DK.%2BM.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DSum%26aufirst%3DC.%2BS.%26aulast%3DCvijic%26aufirst%3DM.%2BE.%26aulast%3DSchroeder%26aufirst%3DG.%2BM.%26aulast%3DWilson%26aufirst%3DL.%2BJ.%26aulast%3DLiotta%26aufirst%3DD.%2BC.%26atitle%3DDiscovery%2520of%2520N-alkyl%2520piperazine%2520side%2520chain%2520based%2520CXCR4%2520antagonists%2520with%2520improved%2520drug-like%2520properties%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2018%26volume%3D9%26spage%3D446%26epage%3D451%26doi%3D10.1021%2Facsmedchemlett.8b00030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, X. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Proteolysis targeting chimeras (PROTACs) of anaplastic lymphoma linase (ALK)</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>151</i></span>,  <span class="NLM_fpage">304</span>– <span class="NLM_lpage">314</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.03.071</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1016%2Fj.ejmech.2018.03.071" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=29627725" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnt1ejs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=151&publication_year=2018&pages=304-314&author=C.+Zhangauthor=X.+R.+Hanauthor=X.+Yangauthor=B.+Jiangauthor=J.+Liuauthor=Y.+Xiongauthor=J.+Jin&title=Proteolysis+targeting+chimeras+%28PROTACs%29+of+anaplastic+lymphoma+linase+%28ALK%29&doi=10.1016%2Fj.ejmech.2018.03.071"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis Targeting Chimeras (PROTACs) of Anaplastic Lymphoma Kinase (ALK)</span></div><div class="casAuthors">Zhang, Chengwei; Han, Xiao-Ran; Yang, Xiaobao; Jiang, Biao; Liu, Jing; Xiong, Yue; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">151</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">304-314</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Anaplastic lymphoma kinase (ALK) activation has been assocd. with many types of human cancer.  Significant efforts have been devoted to the development of ALK inhibitors to antagonize the kinase activity of ALK.  Four ALK inhibitors have been approved by the FDA to date for treating patients with ALK-pos. nonsmall cell lung cancers (NSCLC).  However, drug resistance has been obsd. in the majority of patients treated with these inhibitors.  New therapeutic strategies (e.g., compds. with novel mechanisms of action) are needed to overcome the drug resistance issue.  The emerging PROTAC (Proteolysis Targeting Chimera) technol. has been successfully applied to selective degrdn. of multiple protein targets, but not ALK.  Since ALK protein levels are not important for viability in mammals, ALK PROTACs could lead to novel therapeutics with minimal toxicity.  Here the authors report the design, synthesis and biol. evaluation of novel PROTACs (degraders) of ALK.  MS4077 (I) and MS4078 (II) potently decreased cellular levels of oncogenic active ALK fusion proteins in a concn.- and time-dependent manner in SU-DHL-1 lymphoma and NCI-H2228 lung cancer cells.  The ALK protein degrdn. induced by compds. I and II was cereblon and proteasome dependent.  In addn., compds. I and II potently inhibited proliferation of SU-DHL-1 cells.  Furthermore, compd. II displayed good plasma exposure in a mouse pharmacokinetic study, thus is suitable for in vivo efficacy studies.  The authors also developed MS4748 and MS4740 , very close analogs of I and II resp., which are incapable to degrade the ALK fusion proteins, as neg. controls.  All 4 compds. are valuable chem. tools for investigating effects of ALK pharmacol. degrdn.  The study paved the way for developing the next generation of ALK PROTACs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPGM3aAZZ6h7Vg90H21EOLACvtfcHk0lhJfJ_6gMqqfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnt1ejs7Y%253D&md5=804324b95034f38b46e45f89e18e55be</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.03.071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.03.071%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DHan%26aufirst%3DX.%2BR.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DProteolysis%2520targeting%2520chimeras%2520%2528PROTACs%2529%2520of%2520anaplastic%2520lymphoma%2520linase%2520%2528ALK%2529%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D151%26spage%3D304%26epage%3D314%26doi%3D10.1016%2Fj.ejmech.2018.03.071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguilar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of MD-224 as a first-in-class, highly potent, and efficacious proteolysis targeting chimera Murine Double Minute 2 degrader capable of achieving complete and durable tumor regression</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">448</span>– <span class="NLM_lpage">466</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00909</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00909" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVChtrfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=448-466&author=Y.+Liauthor=J.+Yangauthor=A.+Aguilarauthor=D.+McEachernauthor=S.+Przybranowskiauthor=L.+Liuauthor=C.+Y.+Yangauthor=M.+Wangauthor=X.+Hanauthor=S.+Wang&title=Discovery+of+MD-224+as+a+first-in-class%2C+highly+potent%2C+and+efficacious+proteolysis+targeting+chimera+Murine+Double+Minute+2+degrader+capable+of+achieving+complete+and+durable+tumor+regression&doi=10.1021%2Facs.jmedchem.8b00909"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of MD-224 as a First-in-Class, Highly Potent, and Efficacious Proteolysis Targeting Chimera Murine Double Minute 2 Degrader Capable of Achieving Complete and Durable Tumor Regression</span></div><div class="casAuthors">Li, Yangbing; Yang, Jiuling; Aguilar, Angelo; McEachern, Donna; Przybranowski, Sally; Liu, Liu; Yang, Chao-Yie; Wang, Mi; Han, Xin; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">448-466</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Human murine double minute 2 (MDM2) protein is a primary endogenous cellular inhibitor of the tumor suppressor p53 and has been pursued as an attractive cancer therapeutic target.  Several potent, nonpeptide, small-mol. inhibitors of MDM2 are currently in clin. development.  In this paper, we report our design, synthesis, and evaluation of small-mol. MDM2 degraders based on the proteolysis targeting chimera (PROTAC) concept.  The most promising compd. (MD-224, I) effectively induces rapid degrdn. of MDM2 at concns. <1 nM in human leukemia cells.  It achieves an IC50 value of 1.5 nM in inhibition of growth of RS4;11 cells and also low nanomolar IC50 values in a panel of leukemia cell lines.  MD-224 (I) achieves complete and durable tumor regression in vivo in the RS4;11 xenograft tumor model in mice at well-tolerated dose schedules.  MD-224 (I) is thus a highly potent and efficacious MDM2 degrader and warrants extensive evaluations as a new class of anticancer agent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfVzK-tVEdk7Vg90H21EOLACvtfcHk0li2g7IeoowIAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVChtrfK&md5=3eaa40a8b0ae15c0172ef44a2fd1b902</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00909&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00909%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DAguilar%26aufirst%3DA.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DHan%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520MD-224%2520as%2520a%2520first-in-class%252C%2520highly%2520potent%252C%2520and%2520efficacious%2520proteolysis%2520targeting%2520chimera%2520Murine%2520Double%2520Minute%25202%2520degrader%2520capable%2520of%2520achieving%2520complete%2520and%2520durable%2520tumor%2520regression%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D448%26epage%3D466%26doi%3D10.1021%2Facs.jmedchem.8b00909" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schramm, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agnetta, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bermudez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerwe, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irmen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holze, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Littmann, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolber, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trankle, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Decker, M.</span></span> <span> </span><span class="NLM_article-title">Novel BQCA- and TBPB-derived M1 receptor hybrid ligands: orthosteric carbachol differentially regulates partial agonism</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">1349</span>– <span class="NLM_lpage">1358</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201900283</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1002%2Fcmdc.201900283" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=31166078" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC1MXht1yrsLrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=1349-1358&author=S.+Schrammauthor=L.+Agnettaauthor=M.+Bermudezauthor=H.+Gerweauthor=M.+Irmenauthor=J.+Holzeauthor=T.+Littmannauthor=G.+Wolberauthor=C.+Trankleauthor=M.+Decker&title=Novel+BQCA-+and+TBPB-derived+M1+receptor+hybrid+ligands%3A+orthosteric+carbachol+differentially+regulates+partial+agonism&doi=10.1002%2Fcmdc.201900283"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Novel BQCA- and TBPB-derived M1 receptor hybrid ligands: orthosteric carbachol differentially regulates partial agonism</span></div><div class="casAuthors">Schramm, Simon; Agnetta, Luca; Bermudez, Marcel; Gerwe, Hubert; Irmen, Matthias; Holze, Janine; Littmann, Timo; Wolber, Gerhard; Traenkle, Christian; Decker, Michael</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1349-1358</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Recently, investigations of the complex mechanisms of allostery have led to a deeper understanding of G protein-coupled receptor (GPCR) activation and signaling processes.  In this context, muscarinic acetylcholine receptors (mAChRs) are highly relevant due to their exemplary role in the study of allosteric modulation.  In this work, we compare and discuss two sets of putatively dualsteric ligands, which were designed to connect carbachol to different types of allosteric ligands.  We chose derivs. of TBPB [1-(1'-(2-tolyl)-1,4'-bipiperidin-4-yl)-1H-benzo[d]imidazol-2(3H)-one] as M1-selective putative bitopic ligands, and derivs. of benzyl quinolone carboxylic acid (BQCA) as an M1 pos. allosteric modulator, varying the distance between the allosteric and orthosteric building blocks.  Luciferase protein complementation assays demonstrated that linker length must be carefully chosen to yield either agonist or antagonist behavior.  These findings may help to design biased signaling and/or different extents of efficacy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmzb2TzuROxbVg90H21EOLACvtfcHk0ljKdXjfhLFjUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXht1yrsLrJ&md5=717880b115bc2c32ff1ccdf73a22dec3</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201900283&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201900283%26sid%3Dliteratum%253Aachs%26aulast%3DSchramm%26aufirst%3DS.%26aulast%3DAgnetta%26aufirst%3DL.%26aulast%3DBermudez%26aufirst%3DM.%26aulast%3DGerwe%26aufirst%3DH.%26aulast%3DIrmen%26aufirst%3DM.%26aulast%3DHolze%26aufirst%3DJ.%26aulast%3DLittmann%26aufirst%3DT.%26aulast%3DWolber%26aufirst%3DG.%26aulast%3DTrankle%26aufirst%3DC.%26aulast%3DDecker%26aufirst%3DM.%26atitle%3DNovel%2520BQCA-%2520and%2520TBPB-derived%2520M1%2520receptor%2520hybrid%2520ligands%253A%2520orthosteric%2520carbachol%2520differentially%2520regulates%2520partial%2520agonism%26jtitle%3DChemMedChem%26date%3D2019%26volume%3D14%26spage%3D1349%26epage%3D1358%26doi%3D10.1002%2Fcmdc.201900283" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Molle, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gareiss, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tae, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noblin, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Targeting the von Hippel-Lindau E3 ubiquitin ligase using small molecules to disrupt the VHL/HIF-1alpha interaction</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>134</i></span>,  <span class="NLM_fpage">4465</span>– <span class="NLM_lpage">4468</span>, <span class="refDoi"> DOI: 10.1021/ja209924v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja209924v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC38XivVOgurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2012&pages=4465-4468&author=D.+L.+Buckleyauthor=I.+Van+Molleauthor=P.+C.+Gareissauthor=H.+S.+Taeauthor=J.+Michelauthor=D.+J.+Noblinauthor=W.+L.+Jorgensenauthor=A.+Ciulliauthor=C.+M.+Crews&title=Targeting+the+von+Hippel-Lindau+E3+ubiquitin+ligase+using+small+molecules+to+disrupt+the+VHL%2FHIF-1alpha+interaction&doi=10.1021%2Fja209924v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the von Hippel-Lindau E3 Ubiquitin Ligase Using Small Molecules To Disrupt the VHL/HIF-1α Interaction</span></div><div class="casAuthors">Buckley, Dennis L.; Van Molle, Inge; Gareiss, Peter C.; Tae, Hyun Seop; Michel, Julien; Noblin, Devin J.; Jorgensen, William L.; Ciulli, Alessio; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4465-4468</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">E3 ubiquitin ligases, which bind protein targets, leading to their ubiquitination and subsequent degrdn., are attractive drug targets due to their exquisite substrate specificity.  However, the development of small-mol. inhibitors has proven extraordinarily challenging as modulation of E3 ligase activities requires the targeting of protein-protein interactions.  Using rational design, we have generated the first small mol. targeting the von Hippel-Lindau protein (VHL), the substrate recognition subunit of an E3 ligase, and an important target in cancer, chronic anemia, and ischemia.  We have also obtained the crystal structure of VHL bound to our most potent inhibitor, confirming that the compd. mimics the binding mode of the transcription factor HIF-1α, a substrate of VHL.  These results have the potential to guide future development of improved lead compds. as therapeutics for the treatment of chronic anemia and ischemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolmBVM_mfWzrVg90H21EOLACvtfcHk0ljKdXjfhLFjUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XivVOgurw%253D&md5=60f7384b6bc932c9828dc12999283a1e</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fja209924v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja209924v%26sid%3Dliteratum%253Aachs%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DVan%2BMolle%26aufirst%3DI.%26aulast%3DGareiss%26aufirst%3DP.%2BC.%26aulast%3DTae%26aufirst%3DH.%2BS.%26aulast%3DMichel%26aufirst%3DJ.%26aulast%3DNoblin%26aufirst%3DD.%2BJ.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DCiulli%26aufirst%3DA.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DTargeting%2520the%2520von%2520Hippel-Lindau%2520E3%2520ubiquitin%2520ligase%2520using%2520small%2520molecules%2520to%2520disrupt%2520the%2520VHL%252FHIF-1alpha%2520interaction%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2012%26volume%3D134%26spage%3D4465%26epage%3D4468%26doi%3D10.1021%2Fja209924v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafson, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Molle, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tae, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gareiss, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Small-molecule inhibitors of the interaction between the E3 ligase VHL and HIF1alpha</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">11463</span>– <span class="NLM_lpage">11467</span>, <span class="refDoi"> DOI: 10.1002/anie.201206231</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1002%2Fanie.201206231" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsV2rs7nL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=11463-11467&author=D.+L.+Buckleyauthor=J.+L.+Gustafsonauthor=I.+Van+Molleauthor=A.+G.+Rothauthor=H.+S.+Taeauthor=P.+C.+Gareissauthor=W.+L.+Jorgensenauthor=A.+Ciulliauthor=C.+M.+Crews&title=Small-molecule+inhibitors+of+the+interaction+between+the+E3+ligase+VHL+and+HIF1alpha&doi=10.1002%2Fanie.201206231"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Inhibitors of the Interaction between the E3 Ligase VHL and HIF1α</span></div><div class="casAuthors">Buckley, Dennis L.; Gustafson, Jeffrey L.; Van Molle, Inge; Roth, Anke G.; Tae, Hyun Seop; Gareiss, Peter C.; Jorgensen, William L.; Ciulli, Alessio; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">46</span>),
    <span class="NLM_cas:pages">11463-11467</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">This study reports the structure-based design and synthesis of novel VHL ligands, illustrating structure-activity relationships and improving potency.  Two fragments of the ligands were individually optimized, arriving at a new class of N-terminal benzamide fragments.  The interactions made by this new class were then elucidated through X-ray crystallog. of the protein-ligand complex.  Structural studies of this ligand class revealed both a new binding mode as well as a novel water-mediated' hydrogen bond.  Finally, using these optimized fragments, we assembled new VHL ligands in a combinatorial fashion, arriving at (I), the first small-mol. submicromolar inhibitor of the interaction between VHL and a peptide derived from HIF1α.  While no cell-based activity has yet been obsd.,we hope to take advantage of the interactions highlighted by the crystal structure of I to design more drug-like inhibitors of VHL with improved cell permeability.  These ligands and the novel interactions and binding modes highlighted have the potential to guide the development of future chem. probes that can target the E3 ligase VHL or stabilize HIF1α proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqs0HINAsXwwLVg90H21EOLACvtfcHk0ljKdXjfhLFjUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsV2rs7nL&md5=eefeb63a3491737738e94c00be083cfd</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1002%2Fanie.201206231&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201206231%26sid%3Dliteratum%253Aachs%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DGustafson%26aufirst%3DJ.%2BL.%26aulast%3DVan%2BMolle%26aufirst%3DI.%26aulast%3DRoth%26aufirst%3DA.%2BG.%26aulast%3DTae%26aufirst%3DH.%2BS.%26aulast%3DGareiss%26aufirst%3DP.%2BC.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DCiulli%26aufirst%3DA.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DSmall-molecule%2520inhibitors%2520of%2520the%2520interaction%2520between%2520the%2520E3%2520ligase%2520VHL%2520and%2520HIF1alpha%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2012%26volume%3D51%26spage%3D11463%26epage%3D11467%26doi%3D10.1002%2Fanie.201206231" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darricarrere, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafson, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, I. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miah, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harling, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">HaloPROTACS: use of small molecule PROTACs to induce degradation of HaloTag fusion proteins</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1831</span>– <span class="NLM_lpage">1837</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00442</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00442" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVSht7fN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1831-1837&author=D.+L.+Buckleyauthor=K.+Rainaauthor=N.+Darricarrereauthor=J.+Hinesauthor=J.+L.+Gustafsonauthor=I.+E.+Smithauthor=A.+H.+Miahauthor=J.+D.+Harlingauthor=C.+M.+Crews&title=HaloPROTACS%3A+use+of+small+molecule+PROTACs+to+induce+degradation+of+HaloTag+fusion+proteins&doi=10.1021%2Facschembio.5b00442"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">HaloPROTACS: Use of Small Molecule PROTACs to Induce Degradation of HaloTag Fusion Proteins</span></div><div class="casAuthors">Buckley, Dennis L.; Raina, Kanak; Darricarrere, Nicole; Hines, John; Gustafson, Jeffrey L.; Smith, Ian E.; Miah, Afjal H.; Harling, John D.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1831-1837</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Small mol.-induced protein degrdn. is an attractive strategy for the development of chem. probes.  One method for inducing targeted protein degrdn. involves the use of PROTACs, heterobifunctional mols. that can recruit specific E3 ligases to a desired protein of interest.  PROTACs have been successfully used to degrade numerous proteins in cells, but the peptidic E3 ligase ligands used in previous PROTACs have hindered their development into more mature chem. probes or therapeutics.  The authors report the design of a novel class of PROTACs that incorporate small mol. VHL ligands to successfully degrade HaloTag7 fusion proteins.  These HaloPROTACs will inspire the development of future PROTACs with more drug-like properties.  Addnl., these HaloPROTACs are useful chem. genetic tools, due to their ability to chem. knock down widely used HaloTag7 fusion proteins in a general fashion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotj-Sgj4TyKrVg90H21EOLACvtfcHk0liVGpiJgbahjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVSht7fN&md5=b074c58e983f15c384818777b25e248d</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00442&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00442%26sid%3Dliteratum%253Aachs%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DDarricarrere%26aufirst%3DN.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DGustafson%26aufirst%3DJ.%2BL.%26aulast%3DSmith%26aufirst%3DI.%2BE.%26aulast%3DMiah%26aufirst%3DA.%2BH.%26aulast%3DHarling%26aufirst%3DJ.%2BD.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DHaloPROTACS%253A%2520use%2520of%2520small%2520molecule%2520PROTACs%2520to%2520induce%2520degradation%2520of%2520HaloTag%2520fusion%2520proteins%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D1831%26epage%3D1837%26doi%3D10.1021%2Facschembio.5b00442" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bohm, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lydeard, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stadler, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavadini, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serluca, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acker, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingaraju, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tichkule, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schebesta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forrester, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schirle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassiepen, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottl, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hild, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beckwith, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thoma, N. H.</span></span> <span> </span><span class="NLM_article-title">Structure of the DDB1-CRBN E3 ubiquitin ligase in complex with thalidomide</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>512</i></span>,  <span class="NLM_fpage">49</span>– <span class="NLM_lpage">53</span>, <span class="refDoi"> DOI: 10.1038/nature13527</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1038%2Fnature13527" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=25043012" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht12ms7%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=512&publication_year=2014&pages=49-53&author=E.+S.+Fischerauthor=K.+Bohmauthor=J.+R.+Lydeardauthor=H.+Yangauthor=M.+B.+Stadlerauthor=S.+Cavadiniauthor=J.+Nagelauthor=F.+Serlucaauthor=V.+Ackerauthor=G.+M.+Lingarajuauthor=R.+B.+Tichkuleauthor=M.+Schebestaauthor=W.+C.+Forresterauthor=M.+Schirleauthor=U.+Hassiepenauthor=J.+Ottlauthor=M.+Hildauthor=R.+E.+Beckwithauthor=J.+W.+Harperauthor=J.+L.+Jenkinsauthor=N.+H.+Thoma&title=Structure+of+the+DDB1-CRBN+E3+ubiquitin+ligase+in+complex+with+thalidomide&doi=10.1038%2Fnature13527"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the DDB1-CRBN E3 ubiquitin ligase in complex with thalidomide</span></div><div class="casAuthors">Fischer, Eric S.; Bohm, Kerstin; Lydeard, John R.; Yang, Haidi; Stadler, Michael B.; Cavadini, Simone; Nagel, Jane; Serluca, Fabrizio; Acker, Vincent; Lingaraju, Gondichatnahalli M.; Tichkule, Ritesh B.; Schebesta, Michael; Forrester, William C.; Schirle, Markus; Hassiepen, Ulrich; Ottl, Johannes; Hild, Marc; Beckwith, Rohan E. J.; Harper, J. Wade; Jenkins, Jeremy L.; Thoma, Nicolas H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">512</span>
        (<span class="NLM_cas:issue">7512</span>),
    <span class="NLM_cas:pages">49-53</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">In the 1950s, the drug thalidomide, administered as a sedative to pregnant women, led to the birth of thousands of children with multiple defects.  Despite the teratogenicity of thalidomide and its derivs. lenalidomide and pomalidomide, these immunomodulatory drugs (IMiDs) recently emerged as effective treatments for multiple myeloma and 5q-deletion-assocd. dysplasia.  IMiDs target the E3 ubiquitin ligase CUL4-RBX1-DDB1-CRBN (known as CRL4CRBN) and promote the ubiquitination of the IKAROS family transcription factors IKZF1 and IKZF3 by CRL4CRBN.  Here we present crystal structures of the DDB1-CRBN complex bound to thalidomide, lenalidomide and pomalidomide.  The structure establishes that CRBN is a substrate receptor within CRL4CRBN and enantioselectively binds IMiDs.  Using an unbiased screen, we identified the homeobox transcription factor MEIS2 as an endogenous substrate of CRL4CRBN.  Our studies suggest that IMiDs block endogenous substrates (MEIS2) from binding to CRL4CRBN while the ligase complex is recruiting IKZF1 or IKZF3 for degrdn.  This dual activity implies that small mols. can modulate an E3 ubiquitin ligase and thereby upregulate or downregulate the ubiquitination of proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGt2Y9GecKr7Vg90H21EOLACvtfcHk0liVGpiJgbahjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht12ms7%252FP&md5=1c1eaa48bef87463cdd3f2b1e1bd400d</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1038%2Fnature13527&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13527%26sid%3Dliteratum%253Aachs%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DBohm%26aufirst%3DK.%26aulast%3DLydeard%26aufirst%3DJ.%2BR.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DStadler%26aufirst%3DM.%2BB.%26aulast%3DCavadini%26aufirst%3DS.%26aulast%3DNagel%26aufirst%3DJ.%26aulast%3DSerluca%26aufirst%3DF.%26aulast%3DAcker%26aufirst%3DV.%26aulast%3DLingaraju%26aufirst%3DG.%2BM.%26aulast%3DTichkule%26aufirst%3DR.%2BB.%26aulast%3DSchebesta%26aufirst%3DM.%26aulast%3DForrester%26aufirst%3DW.%2BC.%26aulast%3DSchirle%26aufirst%3DM.%26aulast%3DHassiepen%26aufirst%3DU.%26aulast%3DOttl%26aufirst%3DJ.%26aulast%3DHild%26aufirst%3DM.%26aulast%3DBeckwith%26aufirst%3DR.%2BE.%26aulast%3DHarper%26aufirst%3DJ.%2BW.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26aulast%3DThoma%26aufirst%3DN.%2BH.%26atitle%3DStructure%2520of%2520the%2520DDB1-CRBN%2520E3%2520ubiquitin%2520ligase%2520in%2520complex%2520with%2520thalidomide%26jtitle%3DNature%26date%3D2014%26volume%3D512%26spage%3D49%26epage%3D53%26doi%3D10.1038%2Fnature13527" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fabian, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biggs, W. H.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atteridge, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azimioara, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benedetti, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edeen, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Floyd, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galvin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerlach, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grotzfeld, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrgard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Insko, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Insko, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lelias, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehta, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milanov, Z. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velasco, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wodicka, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarrinkar, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lockhart, D. J.</span></span> <span> </span><span class="NLM_article-title">A small molecule-kinase interaction map for clinical kinase inhibitors</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">329</span>– <span class="NLM_lpage">336</span>, <span class="refDoi"> DOI: 10.1038/nbt1068</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1038%2Fnbt1068" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=15711537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BD2MXitF2nt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2005&pages=329-336&author=M.+A.+Fabianauthor=W.+H.+Biggsauthor=D.+K.+Treiberauthor=C.+E.+Atteridgeauthor=M.+D.+Azimioaraauthor=M.+G.+Benedettiauthor=T.+A.+Carterauthor=P.+Ciceriauthor=P.+T.+Edeenauthor=M.+Floydauthor=J.+M.+Fordauthor=M.+Galvinauthor=J.+L.+Gerlachauthor=R.+M.+Grotzfeldauthor=S.+Herrgardauthor=D.+E.+Inskoauthor=M.+A.+Inskoauthor=A.+G.+Laiauthor=J.+M.+Leliasauthor=S.+A.+Mehtaauthor=Z.+V.+Milanovauthor=A.+M.+Velascoauthor=L.+M.+Wodickaauthor=H.+K.+Patelauthor=P.+P.+Zarrinkarauthor=D.+J.+Lockhart&title=A+small+molecule-kinase+interaction+map+for+clinical+kinase+inhibitors&doi=10.1038%2Fnbt1068"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">A small molecule-kinase interaction map for clinical kinase inhibitors</span></div><div class="casAuthors">Fabian, Miles A.; Biggs, William H.; Treiber, Daniel K.; Atteridge, Corey E.; Azimioara, Mihai D.; Benedetti, Michael G.; Carter, Todd A.; Ciceri, Pietro; Edeen, Philip T.; Floyd, Mark; Ford, Julia M.; Galvin, Margaret; Gerlach, Jay L.; Grotzfeld, Robert M.; Herrgard, Sanna; Insko, Darren E.; Insko, Michael A.; Lai, Andiliy G.; Lelias, Jean-Michel; Mehta, Shamal A.; Milanov, Zdravko V.; Velasco, Anne Marie; Wodicka, Lisa M.; Patel, Hitesh K.; Zarrinkar, Patrick P.; Lockhart, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">329-336</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Kinase inhibitors show great promise as a new class of therapeutics.  Here the authors describe an efficient way to det. kinase inhibitor specificity by measuring binding of small mols. to the ATP site of kinases.  The authors have profiled 20 kinase inhibitors, including 16 that are approved drugs or in clin. development, against a panel of 119 protein kinases.  The authors find that specificity varies widely and is not strongly correlated with chem. structure or the identity of the intended target.  Many novel interactions were identified, including tight binding of the p38 inhibitor BIRB-796 to an imatinib-resistant variant of the ABL kinase, and binding of imatinib to the SRC-family kinase LCK.  The authors also show that mutations in the epidermal growth factor receptor (EGFR) found in gefitinib-responsive patients do not affect the binding affinity of gefitinib or erlotinib.  Our results represent a systematic small mol.-protein interaction map for clin. compds. across a large no. of related proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqA0vNmMOXrnrVg90H21EOLACvtfcHk0liVGpiJgbahjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXitF2nt7w%253D&md5=6aedd5ceb8f77cd26ee50425dcec7bdf</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1038%2Fnbt1068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1068%26sid%3Dliteratum%253Aachs%26aulast%3DFabian%26aufirst%3DM.%2BA.%26aulast%3DBiggs%26aufirst%3DW.%2BH.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DAtteridge%26aufirst%3DC.%2BE.%26aulast%3DAzimioara%26aufirst%3DM.%2BD.%26aulast%3DBenedetti%26aufirst%3DM.%2BG.%26aulast%3DCarter%26aufirst%3DT.%2BA.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DEdeen%26aufirst%3DP.%2BT.%26aulast%3DFloyd%26aufirst%3DM.%26aulast%3DFord%26aufirst%3DJ.%2BM.%26aulast%3DGalvin%26aufirst%3DM.%26aulast%3DGerlach%26aufirst%3DJ.%2BL.%26aulast%3DGrotzfeld%26aufirst%3DR.%2BM.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DInsko%26aufirst%3DD.%2BE.%26aulast%3DInsko%26aufirst%3DM.%2BA.%26aulast%3DLai%26aufirst%3DA.%2BG.%26aulast%3DLelias%26aufirst%3DJ.%2BM.%26aulast%3DMehta%26aufirst%3DS.%2BA.%26aulast%3DMilanov%26aufirst%3DZ.%2BV.%26aulast%3DVelasco%26aufirst%3DA.%2BM.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DPatel%26aufirst%3DH.%2BK.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26aulast%3DLockhart%26aufirst%3DD.%2BJ.%26atitle%3DA%2520small%2520molecule-kinase%2520interaction%2520map%2520for%2520clinical%2520kinase%2520inhibitors%26jtitle%3DNat.%2520Biotechnol.%26date%3D2005%26volume%3D23%26spage%3D329%26epage%3D336%26doi%3D10.1038%2Fnbt1068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Douglass, E. F.,  Jr</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sparer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shapiro, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spiegel, D. A.</span></span> <span> </span><span class="NLM_article-title">A comprehensive mathematical model for three-body binding equilibria</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>135</i></span>,  <span class="NLM_fpage">6092</span>– <span class="NLM_lpage">6099</span>, <span class="refDoi"> DOI: 10.1021/ja311795d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja311795d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC3sXltVCktr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=6092-6099&author=E.+F.+Douglassauthor=C.+J.+Millerauthor=G.+Sparerauthor=H.+Shapiroauthor=D.+A.+Spiegel&title=A+comprehensive+mathematical+model+for+three-body+binding+equilibria&doi=10.1021%2Fja311795d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">A Comprehensive Mathematical Model for Three-Body Binding Equilibria</span></div><div class="casAuthors">Douglass, Eugene F.; Miller, Chad J.; Sparer, Gerson; Shapiro, Harold; Spiegel, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">6092-6099</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Three-component systems are often more complex than their two-component counterparts.  Although the reversible assocn. of three components in soln. is crit. for a vast array of chem. and biol. processes, no general phys. picture of such systems has emerged.  Here the authors have developed a general, comprehensive framework for understanding ternary complex equil., which relates directly to familiar concepts such as EC50 and IC50 from simpler (binary complex) equil.  Importantly, application of the authors' model to data from the published literature has enabled the authors to achieve new insights into complex systems ranging from coagulation to therapeutic dosing regimens for monoclonal antibodies.  The authors also provide an Excel spreadsheet to assist readers in both conceptualizing and applying the authors' models.  Overall, the anal. has the potential to render complex three-component systems-which have previously been characterized as "anal. intractable"-readily comprehensible to theoreticians and experimentalists alike.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4T36t0kgZ3LVg90H21EOLACvtfcHk0ljK0UrInbtkbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXltVCktr4%253D&md5=44a6da2858f9813d7f1329c96a7085ac</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Fja311795d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja311795d%26sid%3Dliteratum%253Aachs%26aulast%3DDouglass%26aufirst%3DE.%2BF.%26aulast%3DMiller%26aufirst%3DC.%2BJ.%26aulast%3DSparer%26aufirst%3DG.%26aulast%3DShapiro%26aufirst%3DH.%26aulast%3DSpiegel%26aufirst%3DD.%2BA.%26atitle%3DA%2520comprehensive%2520mathematical%2520model%2520for%2520three-body%2520binding%2520equilibria%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26spage%3D6092%26epage%3D6099%26doi%3D10.1021%2Fja311795d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thapa, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, R. A.</span></span> <span> </span><span class="NLM_article-title">A kinase-independent role for EGF receptor in autophagy initiation</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>160</i></span>,  <span class="NLM_fpage">145</span>– <span class="NLM_lpage">160</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2014.12.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1016%2Fj.cell.2014.12.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=25594178" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtF2lsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=160&publication_year=2015&pages=145-160&author=X.+Tanauthor=N.+Thapaauthor=Y.+Sunauthor=R.+A.+Anderson&title=A+kinase-independent+role+for+EGF+receptor+in+autophagy+initiation&doi=10.1016%2Fj.cell.2014.12.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">A Kinase-Independent Role for EGF Receptor in Autophagy Initiation</span></div><div class="casAuthors">Tan, Xiaojun; Thapa, Narendra; Sun, Yue; Anderson, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">160</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">145-160</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The epidermal growth factor receptor (EGFR) is upregulated in numerous human cancers.  Inhibition of EGFR signaling induces autophagy in tumor cells.  Here, we report an unanticipated role for the inactive EGFR in autophagy initiation.  Inactive EGFR interacts with the oncoprotein LAPTM4B that is required for the endosomal accumulation of EGFR upon serum starvation.  Inactive EGFR and LAPTM4B stabilize each other at endosomes and recruit the exocyst subcomplex contg. Sec5.  We show that inactive EGFR, LAPTM4B, and the Sec5 subcomplex are required for basal and starvation-induced autophagy.  LAPTM4B and Sec5 promote EGFR assocn. with the autophagy inhibitor Rubicon, which in turn disassocs. Beclin 1 from Rubicon to initiate autophagy.  Thus, the oncoprotein LAPTM4B facilitates the role of inactive EGFR in autophagy initiation.  This pathway is positioned to control tumor metab. and promote tumor cell survival upon serum deprivation or metabolic stress.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGosEWXApqULAbVg90H21EOLACvtfcHk0ljK0UrInbtkbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtF2lsbw%253D&md5=a19af31e428f3c08d1c460be0c021730</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2014.12.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2014.12.006%26sid%3Dliteratum%253Aachs%26aulast%3DTan%26aufirst%3DX.%26aulast%3DThapa%26aufirst%3DN.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DAnderson%26aufirst%3DR.%2BA.%26atitle%3DA%2520kinase-independent%2520role%2520for%2520EGF%2520receptor%2520in%2520autophagy%2520initiation%26jtitle%3DCell%26date%3D2015%26volume%3D160%26spage%3D145%26epage%3D160%26doi%3D10.1016%2Fj.cell.2014.12.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Soucy, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milhollen, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gavin, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adhikari, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brownell, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardin, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Critchley, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cullis, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doucette, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garnsey, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaulin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gershman, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lublinsky, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizutani, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narayanan, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olhava, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peluso, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rezaei, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sintchak, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talreja, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Traore, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vyskocil, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weatherhead, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dick, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Claiborne, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rolfe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolen, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langston, S. P.</span></span> <span> </span><span class="NLM_article-title">An inhibitor of NEDD8-activating enzyme as a new approach to treat cancer</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>458</i></span>,  <span class="NLM_fpage">732</span>– <span class="NLM_lpage">736</span>, <span class="refDoi"> DOI: 10.1038/nature07884</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1038%2Fnature07884" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=19360080" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BD1MXksVenu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=458&publication_year=2009&pages=732-736&author=T.+A.+Soucyauthor=P.+G.+Smithauthor=M.+A.+Milhollenauthor=A.+J.+Bergerauthor=J.+M.+Gavinauthor=S.+Adhikariauthor=J.+E.+Brownellauthor=K.+E.+Burkeauthor=D.+P.+Cardinauthor=S.+Critchleyauthor=C.+A.+Cullisauthor=A.+Doucetteauthor=J.+J.+Garnseyauthor=J.+L.+Gaulinauthor=R.+E.+Gershmanauthor=A.+R.+Lublinskyauthor=A.+McDonaldauthor=H.+Mizutaniauthor=U.+Narayananauthor=E.+J.+Olhavaauthor=S.+Pelusoauthor=M.+Rezaeiauthor=M.+D.+Sintchakauthor=T.+Talrejaauthor=M.+P.+Thomasauthor=T.+Traoreauthor=S.+Vyskocilauthor=G.+S.+Weatherheadauthor=J.+Yuauthor=J.+Zhangauthor=L.+R.+Dickauthor=C.+F.+Claiborneauthor=M.+Rolfeauthor=J.+B.+Bolenauthor=S.+P.+Langston&title=An+inhibitor+of+NEDD8-activating+enzyme+as+a+new+approach+to+treat+cancer&doi=10.1038%2Fnature07884"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">An inhibitor of NEDD8-activating enzyme as a new approach to treat cancer</span></div><div class="casAuthors">Soucy, Teresa A.; Smith, Peter G.; Milhollen, Michael A.; Berger, Allison J.; Gavin, James M.; Adhikari, Sharmila; Brownell, James E.; Burke, Kristine E.; Cardin, David P.; Critchley, Stephen; Cullis, Courtney A.; Doucette, Amanda; Garnsey, James J.; Gaulin, Jeffrey L.; Gershman, Rachel E.; Lublinsky, Anna R.; McDonald, Alice; Mizutani, Hirotake; Narayanan, Usha; Olhava, Edward J.; Peluso, Stephane; Rezaei, Mansoureh; Sintchak, Michael D.; Talreja, Tina; Thomas, Michael P.; Traore, Tary; Vyskocil, Stepan; Weatherhead, Gabriel S.; Yu, Jie; Zhang, Julie; Dick, Lawrence R.; Claiborne, Christopher F.; Rolfe, Mark; Bolen, Joseph B.; Langston, Steven P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">458</span>
        (<span class="NLM_cas:issue">7239</span>),
    <span class="NLM_cas:pages">732-736</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The clin. development of an inhibitor of cellular proteasome function suggests that compds. targeting other components of the ubiquitin-proteasome system might prove useful for the treatment of human malignancies.  NEDD8-activating enzyme (NAE) is an essential component of the NEDD8 conjugation pathway that controls the activity of the cullin-RING subtype of ubiquitin ligases, thereby regulating the turnover of a subset of proteins upstream of the proteasome.  Substrates of cullin-RING ligases have important roles in cellular processes assocd. with cancer cell growth and survival pathways.  Here we describe MLN4924, a potent and selective inhibitor of NAE.  MLN4924 disrupts cullin-RING ligase-mediated protein turnover leading to apoptotic death in human tumor cells by a new mechanism of action, the deregulation of S-phase DNA synthesis.  MLN4924 suppressed the growth of human tumor xenografts in mice at compd. exposures that were well tolerated.  Our data suggest that NAE inhibitors may hold promise for the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8fx7OnZtYjbVg90H21EOLACvtfcHk0lghaAQQYctL_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXksVenu7Y%253D&md5=365d4b4feac3994849050e00f1aae5e0</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1038%2Fnature07884&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature07884%26sid%3Dliteratum%253Aachs%26aulast%3DSoucy%26aufirst%3DT.%2BA.%26aulast%3DSmith%26aufirst%3DP.%2BG.%26aulast%3DMilhollen%26aufirst%3DM.%2BA.%26aulast%3DBerger%26aufirst%3DA.%2BJ.%26aulast%3DGavin%26aufirst%3DJ.%2BM.%26aulast%3DAdhikari%26aufirst%3DS.%26aulast%3DBrownell%26aufirst%3DJ.%2BE.%26aulast%3DBurke%26aufirst%3DK.%2BE.%26aulast%3DCardin%26aufirst%3DD.%2BP.%26aulast%3DCritchley%26aufirst%3DS.%26aulast%3DCullis%26aufirst%3DC.%2BA.%26aulast%3DDoucette%26aufirst%3DA.%26aulast%3DGarnsey%26aufirst%3DJ.%2BJ.%26aulast%3DGaulin%26aufirst%3DJ.%2BL.%26aulast%3DGershman%26aufirst%3DR.%2BE.%26aulast%3DLublinsky%26aufirst%3DA.%2BR.%26aulast%3DMcDonald%26aufirst%3DA.%26aulast%3DMizutani%26aufirst%3DH.%26aulast%3DNarayanan%26aufirst%3DU.%26aulast%3DOlhava%26aufirst%3DE.%2BJ.%26aulast%3DPeluso%26aufirst%3DS.%26aulast%3DRezaei%26aufirst%3DM.%26aulast%3DSintchak%26aufirst%3DM.%2BD.%26aulast%3DTalreja%26aufirst%3DT.%26aulast%3DThomas%26aufirst%3DM.%2BP.%26aulast%3DTraore%26aufirst%3DT.%26aulast%3DVyskocil%26aufirst%3DS.%26aulast%3DWeatherhead%26aufirst%3DG.%2BS.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DDick%26aufirst%3DL.%2BR.%26aulast%3DClaiborne%26aufirst%3DC.%2BF.%26aulast%3DRolfe%26aufirst%3DM.%26aulast%3DBolen%26aufirst%3DJ.%2BB.%26aulast%3DLangston%26aufirst%3DS.%2BP.%26atitle%3DAn%2520inhibitor%2520of%2520NEDD8-activating%2520enzyme%2520as%2520a%2520new%2520approach%2520to%2520treat%2520cancer%26jtitle%3DNature%26date%3D2009%26volume%3D458%26spage%3D732%26epage%3D736%26doi%3D10.1038%2Fnature07884" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Frost, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galdeano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soares, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadd, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grzes, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Epemolu, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimamura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantrell, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rocha, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Potent and selective chemical probe of hypoxic signalling downstream of HIF-alpha hydroxylation via VHL inhibition</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">13312</span> <span class="refDoi"> DOI: 10.1038/ncomms13312</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1038%2Fncomms13312" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=27811928" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVSitLfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&author=J.+Frostauthor=C.+Galdeanoauthor=P.+Soaresauthor=M.+S.+Gaddauthor=K.+M.+Grzesauthor=L.+Ellisauthor=O.+Epemoluauthor=S.+Shimamuraauthor=M.+Bantscheffauthor=P.+Grandiauthor=K.+D.+Readauthor=D.+A.+Cantrellauthor=S.+Rochaauthor=A.+Ciulli&title=Potent+and+selective+chemical+probe+of+hypoxic+signalling+downstream+of+HIF-alpha+hydroxylation+via+VHL+inhibition&doi=10.1038%2Fncomms13312"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Potent and selective chemical probe of hypoxic signalling downstream of HIF-α hydroxylation via VHL inhibition</span></div><div class="casAuthors">Frost, Julianty; Galdeano, Carles; Soares, Pedro; Gadd, Morgan S.; Grzes, Katarzyna M.; Ellis, Lucy; Epemolu, Ola; Shimamura, Satoko; Bantscheff, Marcus; Grandi, Paola; Read, Kevin D.; Cantrell, Doreen A.; Rocha, Sonia; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">13312</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Chem. strategies to using small mols. to stimulate hypoxia inducible factors (HIFs) activity and trigger a hypoxic response under normoxic conditions, such as iron chelators and inhibitors of prolyl hydroxylase domain (PHD) enzymes, have broad-spectrum activities and off-target effects.  Here we disclose VH298, a potent VHL inhibitor that stabilizes HIF-α and elicits a hypoxic response via a different mechanism, that is the blockade of the VHL:HIF-α protein-protein interaction downstream of HIF-α hydroxylation by PHD enzymes.  We show that VH298 engages with high affinity and specificity with VHL as its only major cellular target, leading to selective on-target accumulation of hydroxylated HIF-α in a concn.- and time-dependent fashion in different cell lines, with subsequent upregulation of HIF-target genes at both mRNA and protein levels.  VH298 represents a high-quality chem. probe of the HIF signalling cascade and an attractive starting point to the development of potential new therapeutics targeting hypoxia signalling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLB1WvVF4EFLVg90H21EOLACvtfcHk0lghaAQQYctL_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVSitLfJ&md5=7a884b6193b3ca5e119fecea2eb31d3e</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1038%2Fncomms13312&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms13312%26sid%3Dliteratum%253Aachs%26aulast%3DFrost%26aufirst%3DJ.%26aulast%3DGaldeano%26aufirst%3DC.%26aulast%3DSoares%26aufirst%3DP.%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DGrzes%26aufirst%3DK.%2BM.%26aulast%3DEllis%26aufirst%3DL.%26aulast%3DEpemolu%26aufirst%3DO.%26aulast%3DShimamura%26aufirst%3DS.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DRead%26aufirst%3DK.%2BD.%26aulast%3DCantrell%26aufirst%3DD.%2BA.%26aulast%3DRocha%26aufirst%3DS.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DPotent%2520and%2520selective%2520chemical%2520probe%2520of%2520hypoxic%2520signalling%2520downstream%2520of%2520HIF-alpha%2520hydroxylation%2520via%2520VHL%2520inhibition%26jtitle%3DNat.%2520Commun.%26date%3D2016%26volume%3D7%26doi%3D10.1038%2Fncomms13312" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 30 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Shaowen Xie, Yuan Sun, Yulin Liu, Xinnan Li, Xinuo Li, Wenyi Zhong, Feiyan Zhan, Jingjie Zhu, Hong Yao, Dong-Hua Yang, Zhe-Sheng Chen, Jinyi Xu, <span class="NLM_string-name hlFld-ContribAuthor">Shengtao Xu</span>. </span><span class="cited-content_cbyCitation_article-title">Development of Alectinib-Based PROTACs as Novel Potent Degraders of Anaplastic Lymphoma Kinase (ALK). </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (13)
                                     , 9120-9140. <a href="https://doi.org/10.1021/acs.jmedchem.1c00270" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00270</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00270&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00270%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDevelopment%252Bof%252BAlectinib-Based%252BPROTACs%252Bas%252BNovel%252BPotent%252BDegraders%252Bof%252BAnaplastic%252BLymphoma%252BKinase%252B%252528ALK%252529%26aulast%3DXie%26aufirst%3DShaowen%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D11022021%26date%3D28062021%26volume%3D64%26issue%3D13%26spage%3D9120%26epage%3D9140" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Julie K. L. Sinclair, Wesley E. Robertson, Deepto Mozumdar, Kim Quach, <span class="NLM_string-name hlFld-ContribAuthor">Alanna Schepartz</span>. </span><span class="cited-content_cbyCitation_article-title">Allosteric Inhibition of the Epidermal Growth Factor Receptor. </span><span class="cited-content_cbyCitation_journal-name">Biochemistry</span><span> <strong>2021,</strong> <em>60 </em>
                                    (7)
                                     , 500-512. <a href="https://doi.org/10.1021/acs.biochem.0c00978" title="DOI URL">https://doi.org/10.1021/acs.biochem.0c00978</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.biochem.0c00978&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.biochem.0c00978%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemistry%26atitle%3DAllosteric%252BInhibition%252Bof%252Bthe%252BEpidermal%252BGrowth%252BFactor%252BReceptor%26aulast%3DSinclair%26aufirst%3DJulie%2BK.%2BL.%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D2021%26date%3D21122020%26date%3D26012021%26date%3D08022021%26volume%3D60%26issue%3D7%26spage%3D500%26epage%3D512" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Yonggang Meng, Bin Yu, He Huang, Youmei Peng, Ertong Li, Yongfang Yao, Chuanjun Song, Wenquan Yu, Kaikai Zhu, Kai Wang, Dongxu Yi, Jinfa Du, <span class="NLM_string-name hlFld-ContribAuthor">Junbiao Chang</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of Dosimertinib, a Highly Potent, Selective, and Orally Efficacious Deuterated EGFR Targeting Clinical Candidate for the Treatment of Non-Small-Cell Lung Cancer. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (2)
                                     , 925-937. <a href="https://doi.org/10.1021/acs.jmedchem.0c02005" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02005</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02005%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BDosimertinib%25252C%252Ba%252BHighly%252BPotent%25252C%252BSelective%25252C%252Band%252BOrally%252BEfficacious%252BDeuterated%252BEGFR%252BTargeting%252BClinical%252BCandidate%252Bfor%252Bthe%252BTreatment%252Bof%252BNon-Small-Cell%252BLung%252BCancer%26aulast%3DMeng%26aufirst%3DYonggang%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D19112020%26date%3D18012021%26volume%3D64%26issue%3D2%26spage%3D925%26epage%3D937" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Liqun Chen, Yanke Chen, Chunyan Zhang, Bingyang Jiao, Sheng Liang, Qiong Tan, Hongyu Chai, Weihua Yu, Yongzheng Qian, Hui Yang, Wuyi Yao, Jianguo Yu, Ying Luo, Michael Plewe, Jialiang Wang, Xiao-Ran Han, <span class="NLM_string-name hlFld-ContribAuthor">Jing Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of First-In-Class Potent and Selective Tropomyosin Receptor Kinase Degraders. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (23)
                                     , 14562-14575. <a href="https://doi.org/10.1021/acs.jmedchem.0c01342" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01342</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01342&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01342%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BFirst-In-Class%252BPotent%252Band%252BSelective%252BTropomyosin%252BReceptor%252BKinase%252BDegraders%26aulast%3DChen%26aufirst%3DLiqun%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D07082020%26date%3D15102020%26volume%3D63%26issue%3D23%26spage%3D14562%26epage%3D14575" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Yudao Shen, Guozhen Gao, Xufen Yu, Huensuk Kim, Li Wang, Ling Xie, Megan Schwarz, Xian Chen, Ernesto Guccione, Jing Liu, Mark T. Bedford, <span class="NLM_string-name hlFld-ContribAuthor">Jian Jin</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of First-in-Class Protein Arginine Methyltransferase 5 (PRMT5) Degraders. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (17)
                                     , 9977-9989. <a href="https://doi.org/10.1021/acs.jmedchem.0c01111" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01111</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01111%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BFirst-in-Class%252BProtein%252BArginine%252BMethyltransferase%252B5%252B%252528PRMT5%252529%252BDegraders%26aulast%3DShen%26aufirst%3DYudao%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D29062020%26date%3D21082020%26date%3D11082020%26volume%3D63%26issue%3D17%26spage%3D9977%26epage%3D9989" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jing  Liu</span>, <span class="hlFld-ContribAuthor ">Yunhua  Peng</span>, <span class="hlFld-ContribAuthor ">Wenyi  Wei</span>. </span><span class="cited-content_cbyCitation_article-title">Light-Controllable PROTACs for Temporospatial Control of Protein Degradation. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Cell and Developmental Biology</span><span> <strong>2021,</strong> <em>9 </em><a href="https://doi.org/10.3389/fcell.2021.678077" title="DOI URL">https://doi.org/10.3389/fcell.2021.678077</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fcell.2021.678077&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffcell.2021.678077%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Cell%2520and%2520Developmental%2520Biology%26atitle%3DLight-Controllable%252BPROTACs%252Bfor%252BTemporospatial%252BControl%252Bof%252BProtein%252BDegradation%26aulast%3DLiu%26aufirst%3DJing%26date%3D2021%26date%3D2021%26volume%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kusal T.G.  Samarasinghe</span>, <span class="hlFld-ContribAuthor ">Craig M.  Crews</span>. </span><span class="cited-content_cbyCitation_article-title">Targeted protein degradation: A promise for undruggable proteins. </span><span class="cited-content_cbyCitation_journal-name">Cell Chemical Biology</span><span> <strong>2021,</strong> <em>28 </em>
                                    (7)
                                     , 934-951. <a href="https://doi.org/10.1016/j.chembiol.2021.04.011" title="DOI URL">https://doi.org/10.1016/j.chembiol.2021.04.011</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chembiol.2021.04.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chembiol.2021.04.011%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Chemical%2520Biology%26atitle%3DTargeted%252Bprotein%252Bdegradation%25253A%252BA%252Bpromise%252Bfor%252Bundruggable%252Bproteins%26aulast%3DSamarasinghe%26aufirst%3DKusal%2BT.G.%26date%3D2021%26volume%3D28%26issue%3D7%26spage%3D934%26epage%3D951" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fei  Yu</span>, <span class="hlFld-ContribAuthor ">Ming  Cai</span>, <span class="hlFld-ContribAuthor ">Liang  Shao</span>, <span class="hlFld-ContribAuthor ">Jihong  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting Protein Kinases Degradation by PROTACs. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Chemistry</span><span> <strong>2021,</strong> <em>9 </em><a href="https://doi.org/10.3389/fchem.2021.679120" title="DOI URL">https://doi.org/10.3389/fchem.2021.679120</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fchem.2021.679120&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffchem.2021.679120%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Chemistry%26atitle%3DTargeting%252BProtein%252BKinases%252BDegradation%252Bby%252BPROTACs%26aulast%3DYu%26aufirst%3DFei%26date%3D2021%26date%3D2021%26volume%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Brandon  Dale</span>, <span class="hlFld-ContribAuthor ">Meng  Cheng</span>, <span class="hlFld-ContribAuthor ">Kwang-Su  Park</span>, <span class="hlFld-ContribAuthor ">H. Ümit  Kaniskan</span>, <span class="hlFld-ContribAuthor ">Yue  Xiong</span>, <span class="hlFld-ContribAuthor ">Jian  Jin</span>. </span><span class="cited-content_cbyCitation_article-title">Advancing targeted protein degradation for cancer therapy. </span><span class="cited-content_cbyCitation_journal-name">Nature Reviews Cancer</span><span> <strong>2021,</strong> <em>170 </em><a href="https://doi.org/10.1038/s41568-021-00365-x" title="DOI URL">https://doi.org/10.1038/s41568-021-00365-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41568-021-00365-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41568-021-00365-x%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Reviews%2520Cancer%26atitle%3DAdvancing%252Btargeted%252Bprotein%252Bdegradation%252Bfor%252Bcancer%252Btherapy%26aulast%3DDale%26aufirst%3DBrandon%26date%3D2021%26date%3D2021%26volume%3D170" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Baishan  Jiang</span>, <span class="hlFld-ContribAuthor ">Yang  Gao</span>, <span class="hlFld-ContribAuthor ">Jianwei  Che</span>, <span class="hlFld-ContribAuthor ">Wenchao  Lu</span>, <span class="hlFld-ContribAuthor ">Ines H.  Kaltheuner</span>, <span class="hlFld-ContribAuthor ">Ruben  Dries</span>, <span class="hlFld-ContribAuthor ">Marian  Kalocsay</span>, <span class="hlFld-ContribAuthor ">Matthew J.  Berberich</span>, <span class="hlFld-ContribAuthor ">Jie  Jiang</span>, <span class="hlFld-ContribAuthor ">Inchul  You</span>, <span class="hlFld-ContribAuthor ">Nicholas  Kwiatkowski</span>, <span class="hlFld-ContribAuthor ">Kristin M.  Riching</span>, <span class="hlFld-ContribAuthor ">Danette L.  Daniels</span>, <span class="hlFld-ContribAuthor ">Peter K.  Sorger</span>, <span class="hlFld-ContribAuthor ">Matthias  Geyer</span>, <span class="hlFld-ContribAuthor ">Tinghu  Zhang</span>, <span class="hlFld-ContribAuthor ">Nathanael S.  Gray</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery and resistance mechanism of a selective CDK12 degrader. </span><span class="cited-content_cbyCitation_journal-name">Nature Chemical Biology</span><span> <strong>2021,</strong> <em>17 </em>
                                    (6)
                                     , 675-683. <a href="https://doi.org/10.1038/s41589-021-00765-y" title="DOI URL">https://doi.org/10.1038/s41589-021-00765-y</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41589-021-00765-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41589-021-00765-y%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Chemical%2520Biology%26atitle%3DDiscovery%252Band%252Bresistance%252Bmechanism%252Bof%252Ba%252Bselective%252BCDK12%252Bdegrader%26aulast%3DJiang%26aufirst%3DBaishan%26date%3D2021%26date%3D2021%26volume%3D17%26issue%3D6%26spage%3D675%26epage%3D683" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hui  Xu</span>, <span class="hlFld-ContribAuthor ">Xiaomei  Yang</span>, <span class="hlFld-ContribAuthor ">Xiaofeng  Xuan</span>, <span class="hlFld-ContribAuthor ">Di  Wu</span>, <span class="hlFld-ContribAuthor ">Jieru  Zhang</span>, <span class="hlFld-ContribAuthor ">Xinchun  Xu</span>, <span class="hlFld-ContribAuthor ">Yuanjie  Zhao</span>, <span class="hlFld-ContribAuthor ">Chunping  Ma</span>, <span class="hlFld-ContribAuthor ">Dawei  Li</span>. </span><span class="cited-content_cbyCitation_article-title">STAMBP promotes lung adenocarcinoma metastasis by regulating the EGFR/MAPK signaling pathway. </span><span class="cited-content_cbyCitation_journal-name">Neoplasia</span><span> <strong>2021,</strong> <em>23 </em>
                                    (6)
                                     , 607-623. <a href="https://doi.org/10.1016/j.neo.2021.05.011" title="DOI URL">https://doi.org/10.1016/j.neo.2021.05.011</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.neo.2021.05.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.neo.2021.05.011%26sid%3Dliteratum%253Aachs%26jtitle%3DNeoplasia%26atitle%3DSTAMBP%252Bpromotes%252Blung%252Badenocarcinoma%252Bmetastasis%252Bby%252Bregulating%252Bthe%252BEGFR%25252FMAPK%252Bsignaling%252Bpathway%26aulast%3DXu%26aufirst%3DHui%26date%3D2021%26volume%3D23%26issue%3D6%26spage%3D607%26epage%3D623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiaojuan  Qu</span>, <span class="hlFld-ContribAuthor ">Haixia  Liu</span>, <span class="hlFld-ContribAuthor ">Xiaoling  Song</span>, <span class="hlFld-ContribAuthor ">Ning  Sun</span>, <span class="hlFld-ContribAuthor ">Hui  Zhong</span>, <span class="hlFld-ContribAuthor ">Xing  Qiu</span>, <span class="hlFld-ContribAuthor ">Xiaobao  Yang</span>, <span class="hlFld-ContribAuthor ">Biao  Jiang</span>. </span><span class="cited-content_cbyCitation_article-title">Effective degradation of EGFRL858R+T790M mutant proteins by CRBN-based PROTACs through both proteosome and autophagy/lysosome degradation systems. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>218 </em>, 113328. <a href="https://doi.org/10.1016/j.ejmech.2021.113328" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113328</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113328&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113328%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DEffective%252Bdegradation%252Bof%252BEGFRL858R%25252BT790M%252Bmutant%252Bproteins%252Bby%252BCRBN-based%252BPROTACs%252Bthrough%252Bboth%252Bproteosome%252Band%252Bautophagy%25252Flysosome%252Bdegradation%252Bsystems%26aulast%3DQu%26aufirst%3DXiaojuan%26date%3D2021%26volume%3D218%26spage%3D113328" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rui-Fang  Dong</span>, <span class="hlFld-ContribAuthor ">Miao-Lin  Zhu</span>, <span class="hlFld-ContribAuthor ">Ming-Ming  Liu</span>, <span class="hlFld-ContribAuthor ">Yi-Ting  Xu</span>, <span class="hlFld-ContribAuthor ">Liu-Liu  Yuan</span>, <span class="hlFld-ContribAuthor ">Jing  Bian</span>, <span class="hlFld-ContribAuthor ">Yuan-Zheng  Xia</span>, <span class="hlFld-ContribAuthor ">Ling-Yi  Kong</span>. </span><span class="cited-content_cbyCitation_article-title">EGFR mutation mediates resistance to EGFR tyrosine kinase inhibitors in NSCLC: From molecular mechanisms to clinical research. </span><span class="cited-content_cbyCitation_journal-name">Pharmacological Research</span><span> <strong>2021,</strong> <em>167 </em>, 105583. <a href="https://doi.org/10.1016/j.phrs.2021.105583" title="DOI URL">https://doi.org/10.1016/j.phrs.2021.105583</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.phrs.2021.105583&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.phrs.2021.105583%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmacological%2520Research%26atitle%3DEGFR%252Bmutation%252Bmediates%252Bresistance%252Bto%252BEGFR%252Btyrosine%252Bkinase%252Binhibitors%252Bin%252BNSCLC%25253A%252BFrom%252Bmolecular%252Bmechanisms%252Bto%252Bclinical%252Bresearch%26aulast%3DDong%26aufirst%3DRui-Fang%26date%3D2021%26volume%3D167%26spage%3D105583" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kai  Wang</span>, <span class="hlFld-ContribAuthor ">Hongping  Zhou</span>. </span><span class="cited-content_cbyCitation_article-title">Proteolysis targeting chimera (
              PROTAC
              ) for epidermal growth factor receptor enhances anti‐tumor immunity in non‐small cell lung cancer. </span><span class="cited-content_cbyCitation_journal-name">Drug Development Research</span><span> <strong>2021,</strong> <em>82 </em>
                                    (3)
                                     , 422-429. <a href="https://doi.org/10.1002/ddr.21765" title="DOI URL">https://doi.org/10.1002/ddr.21765</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ddr.21765&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fddr.21765%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Development%2520Research%26atitle%3DProteolysis%252Btargeting%252Bchimera%252B%252528%252BPROTAC%252B%252529%252Bfor%252Bepidermal%252Bgrowth%252Bfactor%252Breceptor%252Benhances%252Banti%2525E2%252580%252590tumor%252Bimmunity%252Bin%252Bnon%2525E2%252580%252590small%252Bcell%252Blung%252Bcancer%26aulast%3DWang%26aufirst%3DKai%26date%3D2021%26date%3D2020%26volume%3D82%26issue%3D3%26spage%3D422%26epage%3D429" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bin  Zhang</span>, <span class="hlFld-ContribAuthor ">Zhikun  Liu</span>, <span class="hlFld-ContribAuthor ">Shengjin  Xia</span>, <span class="hlFld-ContribAuthor ">Qingqing  Liu</span>, <span class="hlFld-ContribAuthor ">Shaohua  Gou</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis and biological evaluation of sulfamoylphenyl-quinazoline derivatives as potential EGFR/CAIX dual inhibitors. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>216 </em>, 113300. <a href="https://doi.org/10.1016/j.ejmech.2021.113300" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113300</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113300&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113300%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%252Band%252Bbiological%252Bevaluation%252Bof%252Bsulfamoylphenyl-quinazoline%252Bderivatives%252Bas%252Bpotential%252BEGFR%25252FCAIX%252Bdual%252Binhibitors%26aulast%3DZhang%26aufirst%3DBin%26date%3D2021%26volume%3D216%26spage%3D113300" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yvonne A.  Nagel</span>, <span class="hlFld-ContribAuthor ">Adrian  Britschgi</span>, <span class="hlFld-ContribAuthor ">Antonio  Ricci</span>. </span><span class="cited-content_cbyCitation_article-title">From Degraders to Molecular Glues: New Ways of Breaking Down Disease‐Associated Proteins. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 47-85. <a href="https://doi.org/10.1002/9783527826872.ch2" title="DOI URL">https://doi.org/10.1002/9783527826872.ch2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527826872.ch2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527826872.ch2%26sid%3Dliteratum%253Aachs%26atitle%3DFrom%252BDegraders%252Bto%252BMolecular%252BGlues%25253A%252BNew%252BWays%252Bof%252BBreaking%252BDown%252BDisease%2525E2%252580%252590Associated%252BProteins%26aulast%3DNagel%26aufirst%3DYvonne%2BA.%26date%3D2021%26date%3D2021%26spage%3D47%26epage%3D85%26pub%3DWiley%26atitle%3DSuccessful%252BDrug%252BDiscovery%26aulast%3DFischer%26aufirst%3DJ%25C3%25A1nos%26date%3D2021%26date%3D2021%26volume%3D115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Prateek  Pathak</span>, <span class="hlFld-ContribAuthor ">Hrvoje  Rimac</span>, <span class="hlFld-ContribAuthor ">Maria  Grishina</span>, <span class="hlFld-ContribAuthor ">Amita  Verma</span>, <span class="hlFld-ContribAuthor ">Vladimir  Potemkin</span>. </span><span class="cited-content_cbyCitation_article-title">Hybrid Quinazoline 1,3,5‐Triazines as Epidermal Growth Factor Receptor (EGFR) Inhibitors with Anticancer Activity: Design, Synthesis, and Computational Study. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2021,</strong> <em>16 </em>
                                    (5)
                                     , 822-838. <a href="https://doi.org/10.1002/cmdc.202000646" title="DOI URL">https://doi.org/10.1002/cmdc.202000646</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.202000646&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.202000646%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DHybrid%252BQuinazoline%252B1%25252C3%25252C5%2525E2%252580%252590Triazines%252Bas%252BEpidermal%252BGrowth%252BFactor%252BReceptor%252B%252528EGFR%252529%252BInhibitors%252Bwith%252BAnticancer%252BActivity%25253A%252BDesign%25252C%252BSynthesis%25252C%252Band%252BComputational%252BStudy%26aulast%3DPathak%26aufirst%3DPrateek%26date%3D2021%26date%3D2020%26volume%3D16%26issue%3D5%26spage%3D822%26epage%3D838" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xinrui  Yang</span>, <span class="hlFld-ContribAuthor ">He  Yin</span>, <span class="hlFld-ContribAuthor ">Richard D.  Kim</span>, <span class="hlFld-ContribAuthor ">Jason B.  Fleming</span>, <span class="hlFld-ContribAuthor ">Hao  Xie</span>. </span><span class="cited-content_cbyCitation_article-title">Preclinical and Clinical Advances of Targeted Protein Degradation as a Novel Cancer Therapeutic Strategy: An Oncologist Perspective. </span><span class="cited-content_cbyCitation_journal-name">Targeted Oncology</span><span> <strong>2021,</strong> <em>16 </em>
                                    (1)
                                     , 1-12. <a href="https://doi.org/10.1007/s11523-020-00782-2" title="DOI URL">https://doi.org/10.1007/s11523-020-00782-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s11523-020-00782-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs11523-020-00782-2%26sid%3Dliteratum%253Aachs%26jtitle%3DTargeted%2520Oncology%26atitle%3DPreclinical%252Band%252BClinical%252BAdvances%252Bof%252BTargeted%252BProtein%252BDegradation%252Bas%252Ba%252BNovel%252BCancer%252BTherapeutic%252BStrategy%25253A%252BAn%252BOncologist%252BPerspective%26aulast%3DYang%26aufirst%3DXinrui%26date%3D2021%26date%3D2020%26volume%3D16%26issue%3D1%26spage%3D1%26epage%3D12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yichao  Wan</span>, <span class="hlFld-ContribAuthor ">Jiabing  Long</span>, <span class="hlFld-ContribAuthor ">Han  Gao</span>, <span class="hlFld-ContribAuthor ">Zilong  Tang</span>. </span><span class="cited-content_cbyCitation_article-title">2-Aminothiazole: A privileged scaffold for the discovery of anti-cancer agents. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>210 </em>, 112953. <a href="https://doi.org/10.1016/j.ejmech.2020.112953" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112953</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112953&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112953%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3D2-Aminothiazole%25253A%252BA%252Bprivileged%252Bscaffold%252Bfor%252Bthe%252Bdiscovery%252Bof%252Banti-cancer%252Bagents%26aulast%3DWan%26aufirst%3DYichao%26date%3D2021%26volume%3D210%26spage%3D112953" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shenxin  Zeng</span>, <span class="hlFld-ContribAuthor ">Wenhai  Huang</span>, <span class="hlFld-ContribAuthor ">Xiaoliang  Zheng</span>, <span class="hlFld-ContribAuthor ">Liyan cheng</span>, <span class="hlFld-ContribAuthor ">Zhimin  Zhang</span>, <span class="hlFld-ContribAuthor ">Jian  Wang</span>, <span class="hlFld-ContribAuthor ">Zhengrong  Shen</span>. </span><span class="cited-content_cbyCitation_article-title">Proteolysis targeting chimera (PROTAC) in drug discovery paradigm: Recent progress and future challenges. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>210 </em>, 112981. <a href="https://doi.org/10.1016/j.ejmech.2020.112981" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112981</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112981&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112981%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DProteolysis%252Btargeting%252Bchimera%252B%252528PROTAC%252529%252Bin%252Bdrug%252Bdiscovery%252Bparadigm%25253A%252BRecent%252Bprogress%252Band%252Bfuture%252Bchallenges%26aulast%3DZeng%26aufirst%3DShenxin%26date%3D2021%26volume%3D210%26spage%3D112981" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hong-Yi  Zhao</span>, <span class="hlFld-ContribAuthor ">Xue-Yan  Yang</span>, <span class="hlFld-ContribAuthor ">Hao  Lei</span>, <span class="hlFld-ContribAuthor ">Xiao-Xiao  Xi</span>, <span class="hlFld-ContribAuthor ">She-Min  Lu</span>, <span class="hlFld-ContribAuthor ">Jun-Jie  Zhang</span>, <span class="hlFld-ContribAuthor ">Minhang  Xin</span>, <span class="hlFld-ContribAuthor ">San-Qi  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of potent small molecule PROTACs targeting mutant EGFR. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>208 </em>, 112781. <a href="https://doi.org/10.1016/j.ejmech.2020.112781" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112781</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112781&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112781%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Bpotent%252Bsmall%252Bmolecule%252BPROTACs%252Btargeting%252Bmutant%252BEGFR%26aulast%3DZhao%26aufirst%3DHong-Yi%26date%3D2020%26volume%3D208%26spage%3D112781" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nan  Yao</span>, <span class="hlFld-ContribAuthor ">Chen-Ran  Wang</span>, <span class="hlFld-ContribAuthor ">Ming-Qun  Liu</span>, <span class="hlFld-ContribAuthor ">Ying-Jie  Li</span>, <span class="hlFld-ContribAuthor ">Wei-Min  Chen</span>, <span class="hlFld-ContribAuthor ">Zheng-Qiu  Li</span>, <span class="hlFld-ContribAuthor ">Qi  Qi</span>, <span class="hlFld-ContribAuthor ">Jin-Jian  Lu</span>, <span class="hlFld-ContribAuthor ">Chun-Lin  Fan</span>, <span class="hlFld-ContribAuthor ">Min-Feng  Chen</span>, <span class="hlFld-ContribAuthor ">Ming  Qi</span>, <span class="hlFld-ContribAuthor ">Xiao-Bo  Li</span>, <span class="hlFld-ContribAuthor ">Jian  Hong</span>, <span class="hlFld-ContribAuthor ">Dong-Mei  Zhang</span>, <span class="hlFld-ContribAuthor ">Wen-Cai  Ye</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of a novel EGFR ligand DPBA that degrades EGFR and suppresses EGFR-positive NSCLC growth. </span><span class="cited-content_cbyCitation_journal-name">Signal Transduction and Targeted Therapy</span><span> <strong>2020,</strong> <em>5 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41392-020-00251-2" title="DOI URL">https://doi.org/10.1038/s41392-020-00251-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41392-020-00251-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41392-020-00251-2%26sid%3Dliteratum%253Aachs%26jtitle%3DSignal%2520Transduction%2520and%2520Targeted%2520Therapy%26atitle%3DDiscovery%252Bof%252Ba%252Bnovel%252BEGFR%252Bligand%252BDPBA%252Bthat%252Bdegrades%252BEGFR%252Band%252Bsuppresses%252BEGFR-positive%252BNSCLC%252Bgrowth%26aulast%3DYao%26aufirst%3DNan%26date%3D2020%26date%3D2020%26volume%3D5%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Robert I.  Troup</span>, <span class="hlFld-ContribAuthor ">Charlene  Fallan</span>, <span class="hlFld-ContribAuthor ">Matthias G. J.  Baud</span>. </span><span class="cited-content_cbyCitation_article-title">Current strategies for the design of PROTAC linkers: a critical review. </span><span class="cited-content_cbyCitation_journal-name">Exploration of Targeted Anti-tumor Therapy</span><span> <strong>2020,</strong> <em>1 </em>
                                    (5)
                                     <a href="https://doi.org/10.37349/etat.2020.00018" title="DOI URL">https://doi.org/10.37349/etat.2020.00018</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.37349/etat.2020.00018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.37349%2Fetat.2020.00018%26sid%3Dliteratum%253Aachs%26jtitle%3DExploration%2520of%2520Targeted%2520Anti-tumor%2520Therapy%26atitle%3DCurrent%252Bstrategies%252Bfor%252Bthe%252Bdesign%252Bof%252BPROTAC%252Blinkers%25253A%252Ba%252Bcritical%252Breview%26aulast%3DTroup%26aufirst%3DRobert%2BI.%26date%3D2020%26date%3D2020%26volume%3D1%26issue%3D5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Darren  Stead</span>. </span><span class="cited-content_cbyCitation_article-title">The diastereoselective synthesis of 2,3-diaryl-3-cyano-substituted pyrrolidines via the MgI2 mediated ring expansion of aryl cyclopropyl nitriles. </span><span class="cited-content_cbyCitation_journal-name">Tetrahedron Letters</span><span> <strong>2020,</strong> <em>61 </em>
                                    (39)
                                     , 152325. <a href="https://doi.org/10.1016/j.tetlet.2020.152325" title="DOI URL">https://doi.org/10.1016/j.tetlet.2020.152325</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tetlet.2020.152325&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tetlet.2020.152325%26sid%3Dliteratum%253Aachs%26jtitle%3DTetrahedron%2520Letters%26atitle%3DThe%252Bdiastereoselective%252Bsynthesis%252Bof%252B2%25252C3-diaryl-3-cyano-substituted%252Bpyrrolidines%252Bvia%252Bthe%252BMgI2%252Bmediated%252Bring%252Bexpansion%252Bof%252Baryl%252Bcyclopropyl%252Bnitriles%26aulast%3DStead%26aufirst%3DDarren%26date%3D2020%26volume%3D61%26issue%3D39%26spage%3D152325" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shuhua  Zheng</span>, <span class="hlFld-ContribAuthor ">Wensi  Tao</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting Cullin-RING E3 Ligases for Radiosensitization: From NEDDylation Inhibition to PROTACs. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Oncology</span><span> <strong>2020,</strong> <em>10 </em><a href="https://doi.org/10.3389/fonc.2020.01517" title="DOI URL">https://doi.org/10.3389/fonc.2020.01517</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fonc.2020.01517&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffonc.2020.01517%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Oncology%26atitle%3DTargeting%252BCullin-RING%252BE3%252BLigases%252Bfor%252BRadiosensitization%25253A%252BFrom%252BNEDDylation%252BInhibition%252Bto%252BPROTACs%26aulast%3DZheng%26aufirst%3DShuhua%26date%3D2020%26date%3D2020%26volume%3D10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jaebong  Jang</span>, <span class="hlFld-ContribAuthor ">Ciric  To</span>, <span class="hlFld-ContribAuthor ">Dries J. H.  De Clercq</span>, <span class="hlFld-ContribAuthor ">Eunyoung  Park</span>, <span class="hlFld-ContribAuthor ">Charles M.  Ponthier</span>, <span class="hlFld-ContribAuthor ">Bo Hee  Shin</span>, <span class="hlFld-ContribAuthor ">Mierzhati  Mushajiang</span>, <span class="hlFld-ContribAuthor ">Radosław P.  Nowak</span>, <span class="hlFld-ContribAuthor ">Eric S.  Fischer</span>, <span class="hlFld-ContribAuthor ">Michael J.  Eck</span>, <span class="hlFld-ContribAuthor ">Pasi A.  Jänne</span>, <span class="hlFld-ContribAuthor ">Nathanael S.  Gray</span>. </span><span class="cited-content_cbyCitation_article-title">Mutant‐Selective Allosteric EGFR Degraders are Effective Against a Broad Range of Drug‐Resistant Mutations. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie</span><span> <strong>2020,</strong> <em>132 </em>
                                    (34)
                                     , 14589-14597. <a href="https://doi.org/10.1002/ange.202003500" title="DOI URL">https://doi.org/10.1002/ange.202003500</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ange.202003500&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fange.202003500%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%26atitle%3DMutant%2525E2%252580%252590Selective%252BAllosteric%252BEGFR%252BDegraders%252Bare%252BEffective%252BAgainst%252Ba%252BBroad%252BRange%252Bof%252BDrug%2525E2%252580%252590Resistant%252BMutations%26aulast%3DJang%26aufirst%3DJaebong%26date%3D2020%26date%3D2020%26volume%3D132%26issue%3D34%26spage%3D14589%26epage%3D14597" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jaebong  Jang</span>, <span class="hlFld-ContribAuthor ">Ciric  To</span>, <span class="hlFld-ContribAuthor ">Dries J. H.  De Clercq</span>, <span class="hlFld-ContribAuthor ">Eunyoung  Park</span>, <span class="hlFld-ContribAuthor ">Charles M.  Ponthier</span>, <span class="hlFld-ContribAuthor ">Bo Hee  Shin</span>, <span class="hlFld-ContribAuthor ">Mierzhati  Mushajiang</span>, <span class="hlFld-ContribAuthor ">Radosław P.  Nowak</span>, <span class="hlFld-ContribAuthor ">Eric S.  Fischer</span>, <span class="hlFld-ContribAuthor ">Michael J.  Eck</span>, <span class="hlFld-ContribAuthor ">Pasi A.  Jänne</span>, <span class="hlFld-ContribAuthor ">Nathanael S.  Gray</span>. </span><span class="cited-content_cbyCitation_article-title">Mutant‐Selective Allosteric EGFR Degraders are Effective Against a Broad Range of Drug‐Resistant Mutations. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie International Edition</span><span> <strong>2020,</strong> <em>59 </em>
                                    (34)
                                     , 14481-14489. <a href="https://doi.org/10.1002/anie.202003500" title="DOI URL">https://doi.org/10.1002/anie.202003500</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/anie.202003500&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fanie.202003500%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%2520International%2520Edition%26atitle%3DMutant%2525E2%252580%252590Selective%252BAllosteric%252BEGFR%252BDegraders%252Bare%252BEffective%252BAgainst%252Ba%252BBroad%252BRange%252Bof%252BDrug%2525E2%252580%252590Resistant%252BMutations%26aulast%3DJang%26aufirst%3DJaebong%26date%3D2020%26date%3D2020%26volume%3D59%26issue%3D34%26spage%3D14481%26epage%3D14489" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kailun  He</span>, <span class="hlFld-ContribAuthor ">Zhuo  Zhang</span>, <span class="hlFld-ContribAuthor ">Wenbing  Wang</span>, <span class="hlFld-ContribAuthor ">Xiaoliang  Zheng</span>, <span class="hlFld-ContribAuthor ">Xiaoju  Wang</span>, <span class="hlFld-ContribAuthor ">Xingxian  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery and biological evaluation of proteolysis targeting chimeras (PROTACs) as an EGFR degraders based on osimertinib and lenalidomide. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>30 </em>
                                    (12)
                                     , 127167. <a href="https://doi.org/10.1016/j.bmcl.2020.127167" title="DOI URL">https://doi.org/10.1016/j.bmcl.2020.127167</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2020.127167&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2020.127167%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDiscovery%252Band%252Bbiological%252Bevaluation%252Bof%252Bproteolysis%252Btargeting%252Bchimeras%252B%252528PROTACs%252529%252Bas%252Ban%252BEGFR%252Bdegraders%252Bbased%252Bon%252Bosimertinib%252Band%252Blenalidomide%26aulast%3DHe%26aufirst%3DKailun%26date%3D2020%26volume%3D30%26issue%3D12%26spage%3D127167" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Manvendra  Kumar</span>, <span class="hlFld-ContribAuthor ">Gaurav  Joshi</span>, <span class="hlFld-ContribAuthor ">Joydeep  Chatterjee</span>, <span class="hlFld-ContribAuthor ">Raj  Kumar</span>. </span><span class="cited-content_cbyCitation_article-title">Epidermal Growth Factor Receptor and its Trafficking Regulation by Acetylation: Implication in Resistance and Exploring the Newer Therapeutic Avenues in Cancer. </span><span class="cited-content_cbyCitation_journal-name">Current Topics in Medicinal Chemistry</span><span> <strong>2020,</strong> <em>20 </em>
                                    (12)
                                     , 1105-1123. <a href="https://doi.org/10.2174/1568026620666200207100227" title="DOI URL">https://doi.org/10.2174/1568026620666200207100227</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/1568026620666200207100227&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F1568026620666200207100227%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Topics%2520in%2520Medicinal%2520Chemistry%26atitle%3DEpidermal%252BGrowth%252BFactor%252BReceptor%252Band%252Bits%252BTrafficking%252BRegulation%252Bby%252BAcetylation%25253A%252BImplication%252Bin%252BResistance%252Band%252BExploring%252Bthe%252BNewer%252BTherapeutic%252BAvenues%252Bin%252BCancer%26aulast%3DKumar%26aufirst%3DManvendra%26date%3D2020%26volume%3D20%26issue%3D12%26spage%3D1105%26epage%3D1123" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xin  Zhang</span>, <span class="hlFld-ContribAuthor ">Fang  Xu</span>, <span class="hlFld-ContribAuthor ">Linjiang  Tong</span>, <span class="hlFld-ContribAuthor ">Tao  Zhang</span>, <span class="hlFld-ContribAuthor ">Hua  Xie</span>, <span class="hlFld-ContribAuthor ">Xiaoyun  Lu</span>, <span class="hlFld-ContribAuthor ">Xiaomei  Ren</span>, <span class="hlFld-ContribAuthor ">Ke  Ding</span>. </span><span class="cited-content_cbyCitation_article-title">Design and synthesis of selective degraders of EGFRL858R/T790M mutant. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>192 </em>, 112199. <a href="https://doi.org/10.1016/j.ejmech.2020.112199" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112199</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112199&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112199%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%252Band%252Bsynthesis%252Bof%252Bselective%252Bdegraders%252Bof%252BEGFRL858R%25252FT790M%252Bmutant%26aulast%3DZhang%26aufirst%3DXin%26date%3D2020%26volume%3D192%26spage%3D112199" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/medium/jm9b01566_0003.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01566&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/medium/jm9b01566_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0010.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Chemical structures of representative first-, second-, third-, and fourth-generation EGFR TKIs.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01566&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/medium/jm9b01566_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0017.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Chemical structures of designed gefitinib-based EGFR degraders <b>1</b>–<b>10</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01566&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/medium/jm9b01566_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Gefitinib-Based EGFR Degraders <b>1</b>–<b>10</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01566&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reaction conditions: (a) K<sub>2</sub>CO<sub>3</sub>, dimethylformamide (DMF), 80 °C; (b) trifluoroacetic acid (TFA), dichloromethane (DCM), room temperature (rt), 30 min; (c) ethyl 3-bromopropanoate, K<sub>2</sub>CO<sub>3</sub>, DMF, 80 °C; (d) LiOH, tetrahydrofuran (THF), H<sub>2</sub>O, rt, 12 h; (e) 1-ethyl-3-(3-dimethylaminopropyl)carbodiimide (EDCI), 1-hydroxy-7-azabenzo-triazole (HOAt), <i>N</i>-methylmorpholine (NMM), dimethyl sulfoxide (DMSO), rt, 4 h; (f) NaHCO<sub>3</sub>, NaI, DMF, 65 °C, 2 days; (g) TFA/DCM, rt, 30 min; (h) <b>14</b>, EDCI, HOAt, NMM, DMSO, rt, 12 h.</p></p></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/medium/jm9b01566_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0012.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Western blotting analysis of EGFR protein in HCC-827 cells treated with compounds <b>1</b>–<b>10</b>. (A) VHL-1-based compounds <b>1</b>–<b>6</b> and (B) CRBN-based compounds <b>7</b>–<b>10</b> were tested in HCC-827 cell lines using three different concentrations (10, 100, and 1000 nM). EGFR e19d: EGFR mutant with exon 19 deletion.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01566&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/medium/jm9b01566_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0001.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Chemical structures of compounds <b>27</b> and <b>28</b> and the previously reported EGFR degrader PROTAC3.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01566&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch2" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/medium/jm9b01566_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0011.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Negative Control Compounds <b>27</b> and <b>28</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01566&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reaction conditions: (a) 1-ethyl-3-(3-dimethylaminopropyl)carbodiimide (EDCI), 1-hydroxy-7-azabenzo-triazole (HOAt), N-methylmorpholine (NMM), dimethyl sulfoxide (DMSO), rt, 4 h; (b) <b>13</b>, EDCI, HOAt, NMM, DMSO, rt, 12 h; (c) K<sub>2</sub>CO<sub>3</sub>, CH<sub>3</sub>I, DMF, rt, 4 h; (d) TFA/DCM, rt, 30 min; (e) <b>14</b>, EDCI, HOAt, NMM, DMSO, rt, 12 h.</p></p></figure><figure data-id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/medium/jm9b01566_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0004.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Compounds <b>6</b>, <b>27</b>, <b>10</b>, and <b>28</b> bind EGFR WT and L858R mutants with high affinities. The <i>K</i><sub>d</sub> determinations were performed in a competitive binding assay in duplicate. Error bars represent ± standard error of the mean (SEM) in duplicated independent experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01566&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/medium/jm9b01566_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0013.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Compounds <b>6</b> and <b>10</b> potently reduced protein levels of EGFR mutants and inhibited downstream signaling in lung cancer cells. NSCLC cell lines HCC-827 (A) and H3255 (B) were cultured to 80% confluent and then switched to a serum-free medium for 8 h before treating with indicated concentrations of the compounds for another 16 h. EGFR e19d: EGFR mutant with exon 19 deletion; EGFR L858R: EGFR L858R single point mutant; Gef: gefitinib (parental EGFR inhibitor; 1 μM); 1k: 1000; 10k: 10 000.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01566&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/medium/jm9b01566_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0014.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Compounds <b>6</b> and <b>10</b> did not significantly reduce WT EGFR protein levels in OVCAR-8 (<b>A</b>) and H1299 (<b>B</b>) cells. OVCAR-8 and H1299 cells were cultured to 80% confluent and then switched to a serum-free medium for 8 h before treating with indicated concentrations of the compounds for another 16 h. Gef: gefitinib (1 μM).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01566&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/medium/jm9b01566_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0005.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Compounds <b>27</b> and <b>28</b> did not significantly reduce the protein levels of EGFR mutants in HCC-827 and H3255 cells. HCC-827 (A) and H3255 (B) cells were cultured to 80% confluent and then switched to a serum-free medium for 8 h before treating with indicated concentrations of the compounds for another 16 h.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01566&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/medium/jm9b01566_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0006.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Compounds <b>6</b> and <b>10</b> rapidly and time-dependently elicit the degradation of EGFR mutants in lung cancer cells. A final concentration of 100 nM compound <b>6</b> or <b>10</b> was added to HCC-827 cells (A) and H3255 cells (B) after 8 h of serum starvation. At each time point, cells were collected for Western blot. The time needed for the degradation of 50% protein (<i>t</i><sub>1/2</sub>) was determined by the intensity of Western bands and normalized to GAPDH. Error bars represent the standard errors in three independent experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01566&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/medium/jm9b01566_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0007.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Serum starvation enhances the EGFR degradation induced by compounds <b>6</b> and <b>10</b>. Cultured HCC-827 cells (A) and H3255 cells (B) were switched to a serum-free/serum-containing medium for 8 h and then treated with the indicated concentrations of the degraders for 16 h.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01566&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig11" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/medium/jm9b01566_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0015.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. EGFR degradation effect of compounds <b>6</b> and <b>10</b> is mediated through the E3 ubiquitin ligase components VHL and CRBN, respectively, and is reversible upon washout. (A) H3255 cells were starved for 4 h and then pretreated with the indicated chemicals for 2 h before degrader (100 nM) treatment for another 8 h. The final concentrations of the compounds used for pretreatment are DMSO (0.1% v/v), MLN4924 (1 μM), VH-298 (10 μM), thalidomide (10 μM), negative controls (compound <b>27</b> for degrader <b>6</b> and compound <b>28</b> for degrader <b>10</b>, 10 μM), and gefitinib (10 μM). (B) HCC-827 and H3255 cells were starved for 8 h before a 12 h degrader treatment, then washed with PBS three times, and harvested at the indicated time points for Western blot analysis.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01566&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig12" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/medium/jm9b01566_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0008.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 12. Compounds <b>6</b> and <b>10</b> inhibit the proliferation of lung cancer H3255 cells. H3255 cells were seeded to 96-well plates a night before at 4000 cells/well and then treated with serial dilutions of degraders or controls for 3 days. Cell number/growth was measured using Cell Counting Kit-8 and error bars indicating the standard errors in three independent experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01566&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig13" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Figure 13</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/medium/jm9b01566_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0016.jpeg" id="rightTab-gr15" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 13. Global proteomic analyses indicate that compounds <b>6</b> and <b>10</b> are highly selective EGFR degraders. HCC-827 cells were starved for 8 h and treated with compound <b>6</b> (A) or compound <b>10</b> (B) at 100 nM for 16 h and then collected for liquid chromatography–mass spectrometry (LC–MS) analyses. Label-free quantification (LFQ) was used to calculate peptide intensity; the top panels showed the relative abundance of EGFR (LFQ intensity value) between the degrader-treated and control groups. Error bars indicate the standard errors in three independent experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01566&amp;id=fig13"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig14" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Figure 14</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/medium/jm9b01566_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0009.jpeg" id="rightTab-gr16" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 14. EGFR degraders <b>6</b> and <b>10</b> are bioavailable in mouse PK studies. Plasma concentrations of compound <b>6</b> (<b>A</b>) and compound <b>10</b> (<b>B</b>) following a single 50 mg/kg IP injection over 24 h in male Swiss Albino mice. Experiments were carried out in biological triplicates, with points representing mean concentrations ± SEM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b01566/20200207/images/large/jm9b01566_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01566&amp;id=fig14"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i50">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_73731" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_73731" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 55 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Settleman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haber, D. A.</span></span> <span> </span><span class="NLM_article-title">Epidermal growth factor receptor mutations in lung cancer</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">169</span>– <span class="NLM_lpage">181</span>, <span class="refDoi"> DOI: 10.1038/nrc2088</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1038%2Fnrc2088" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=17318210" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BD2sXitVGrsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2007&pages=169-181&author=S.+V.+Sharmaauthor=D.+W.+Bellauthor=J.+Settlemanauthor=D.+A.+Haber&title=Epidermal+growth+factor+receptor+mutations+in+lung+cancer&doi=10.1038%2Fnrc2088"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Epidermal growth factor receptor mutations in lung cancer</span></div><div class="casAuthors">Sharma, Sreenath V.; Bell, Daphne W.; Settleman, Jeffrey; Haber, Daniel A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">169-181</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The development and clin. application of inhibitors that target the epidermal growth factor receptor (EGFR) provide important insights for new lung cancer therapies, as well as for the broader field of targeted cancer therapies.  We review the results of genetic, biochem. and clin. studies focused on somatic mutations of EGFR that are assocd. with the phenomenon of oncogene addiction, describing 'oncogenic shock' as a mechanistic explanation for the apoptosis that follows the acute treatment of susceptible cells with kinase inhibitors.  Understanding the genetic heterogeneity of epithelial tumors and devising strategies to circumvent their rapid acquisition of resistance to targeted kinase inhibitors are essential to the successful use of targeted therapies in common epithelial cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMKw7Y4FUFM7Vg90H21EOLACvtfcHk0ljHLqDTB8W_Rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXitVGrsbo%253D&md5=46219ea3ae7b9f0977e4a4b7243fd875</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrc2088&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2088%26sid%3Dliteratum%253Aachs%26aulast%3DSharma%26aufirst%3DS.%2BV.%26aulast%3DBell%26aufirst%3DD.%2BW.%26aulast%3DSettleman%26aufirst%3DJ.%26aulast%3DHaber%26aufirst%3DD.%2BA.%26atitle%3DEpidermal%2520growth%2520factor%2520receptor%2520mutations%2520in%2520lung%2520cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2007%26volume%3D7%26spage%3D169%26epage%3D181%26doi%3D10.1038%2Fnrc2088" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wieduwilt, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moasser, M. M.</span></span> <span> </span><span class="NLM_article-title">The epidermal growth factor receptor family: biology driving targeted therapeutics</span>. <i>Cell Mol. Life Sci.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>65</i></span>,  <span class="NLM_fpage">1566</span>– <span class="NLM_lpage">1584</span>, <span class="refDoi"> DOI: 10.1007/s00018-008-7440-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1007%2Fs00018-008-7440-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=18259690" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BD1cXmt12qu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2008&pages=1566-1584&author=M.+J.+Wieduwiltauthor=M.+M.+Moasser&title=The+epidermal+growth+factor+receptor+family%3A+biology+driving+targeted+therapeutics&doi=10.1007%2Fs00018-008-7440-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">The epidermal growth factor receptor family: Biology driving targeted therapeutics</span></div><div class="casAuthors">Wieduwilt, M. J.; Moasser, M. M.</div><div class="citationInfo"><span class="NLM_cas:title">Cellular and Molecular Life Sciences</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1566-1584</span>CODEN:
                <span class="NLM_cas:coden">CMLSFI</span>;
        ISSN:<span class="NLM_cas:issn">1420-682X</span>.
    
            (<span class="NLM_cas:orgname">Birkhaeuser Verlag</span>)
        </div><div class="casAbstract">A review of the biol. of ErbB receptors, including their structure, signaling, regulation, and roles in development and disease, and their increasing roles as targets for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcSSPMwgxaFLVg90H21EOLACvtfcHk0lh0fdcL_NKFnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXmt12qu7o%253D&md5=d0e8b0634b2b401b1c94838fe389ca91</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1007%2Fs00018-008-7440-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00018-008-7440-8%26sid%3Dliteratum%253Aachs%26aulast%3DWieduwilt%26aufirst%3DM.%2BJ.%26aulast%3DMoasser%26aufirst%3DM.%2BM.%26atitle%3DThe%2520epidermal%2520growth%2520factor%2520receptor%2520family%253A%2520biology%2520driving%2520targeted%2520therapeutics%26jtitle%3DCell%2520Mol.%2520Life%2520Sci.%26date%3D2008%26volume%3D65%26spage%3D1566%26epage%3D1584%26doi%3D10.1007%2Fs00018-008-7440-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seshacharyulu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ponnusamy, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haridas, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ganti, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Batra, S. K.</span></span> <span> </span><span class="NLM_article-title">Targeting the EGFR signaling pathway in cancer therapy</span>. <i>Expert Opin. Ther. Targets</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">15</span>– <span class="NLM_lpage">31</span>, <span class="refDoi"> DOI: 10.1517/14728222.2011.648617</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1517%2F14728222.2011.648617" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=22239438" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC38Xos1Wksw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2012&pages=15-31&author=P.+Seshacharyuluauthor=M.+P.+Ponnusamyauthor=D.+Haridasauthor=M.+Jainauthor=A.+K.+Gantiauthor=S.+K.+Batra&title=Targeting+the+EGFR+signaling+pathway+in+cancer+therapy&doi=10.1517%2F14728222.2011.648617"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the EGFR signaling pathway in cancer therapy</span></div><div class="casAuthors">Seshacharyulu, Parthasarathy; Ponnusamy, Moorthy P.; Haridas, Dhanya; Jain, Maneesh; Ganti, Apar K.; Batra, Surinder K.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Targets</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">15-31</span>CODEN:
                <span class="NLM_cas:coden">EOTTAO</span>;
        ISSN:<span class="NLM_cas:issn">1472-8222</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Introduction: Cancer is a devastating disease; however, several therapeutic advances have recently been made, wherein EGFR and its family members have emerged as useful biomarkers and therapeutic targets.  EGFR, a transmembrane glycoprotein is a member of the ERBB receptor tyrosine kinase superfamily.  EGFR binds to its cognate ligand EGF, which further induces tyrosine phosphorylation and receptor dimerization with other family members leading to enhanced uncontrolled proliferation.  Several anti-EGFR therapies such as monoclonal antibodies and tyrosine kinase inhibitors have been developed, which has enabled clinicians to identify and treat specific patient cohorts.Areas covered: This review covers the basic mechanism of EGFR activation and the role of EGFR signaling in cancer progression.  Furthermore, current developments made toward targeting the EGFR signaling pathway for the treatment of epithelial cancers and a summary of the various anti-EGFR therapeutic agents that are currently in use are also presented in this review.Expert opinion: EGFR signaling is a part of a complex network that has been the target of effective cancer therapies.  However, a further understanding of the system is required to develop an effective anticancer regimen.  A combination therapy that comprises an anti-EGFR and a chemotherapeutic/chemopreventive agent will exhibit a multi-pronged approach that can be developed into a highly attractive and specific mol. oriented remedy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDi_-cVSomx7Vg90H21EOLACvtfcHk0lh1QE8W3gUVlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xos1Wksw%253D%253D&md5=91357f1fb76bd06c03d4269f90ad907a</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1517%2F14728222.2011.648617&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F14728222.2011.648617%26sid%3Dliteratum%253Aachs%26aulast%3DSeshacharyulu%26aufirst%3DP.%26aulast%3DPonnusamy%26aufirst%3DM.%2BP.%26aulast%3DHaridas%26aufirst%3DD.%26aulast%3DJain%26aufirst%3DM.%26aulast%3DGanti%26aufirst%3DA.%2BK.%26aulast%3DBatra%26aufirst%3DS.%2BK.%26atitle%3DTargeting%2520the%2520EGFR%2520signaling%2520pathway%2520in%2520cancer%2520therapy%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Targets%26date%3D2012%26volume%3D16%26spage%3D15%26epage%3D31%26doi%3D10.1517%2F14728222.2011.648617" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krause, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Etten, R. A.</span></span> <span> </span><span class="NLM_article-title">Tyrosine kinases as targets for cancer therapy</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>353</i></span>,  <span class="NLM_fpage">172</span>– <span class="NLM_lpage">187</span>, <span class="refDoi"> DOI: 10.1056/NEJMra044389</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1056%2FNEJMra044389" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=16014887" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmtFClsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=353&publication_year=2005&pages=172-187&author=D.+S.+Krauseauthor=R.+A.+Van+Etten&title=Tyrosine+kinases+as+targets+for+cancer+therapy&doi=10.1056%2FNEJMra044389"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Tyrosine kinases as targets for cancer therapy</span></div><div class="casAuthors">Krause, Daniela S.; Van Etten, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">353</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">172-187</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">A review discusses the mechanisms of aberrant tyrosine kinase signaling and strategies to inhibit TKs in cancer.  It also summarizes the status of TK-directed cancer therapies, and discusses challenges and prospects for the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq89QOl9E0UMLVg90H21EOLACvtfcHk0lhGjhpF8-zpUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmtFClsbs%253D&md5=0a98c8840a983300f6b0743673cf480c</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1056%2FNEJMra044389&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMra044389%26sid%3Dliteratum%253Aachs%26aulast%3DKrause%26aufirst%3DD.%2BS.%26aulast%3DVan%2BEtten%26aufirst%3DR.%2BA.%26atitle%3DTyrosine%2520kinases%2520as%2520targets%2520for%2520cancer%2520therapy%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2005%26volume%3D353%26spage%3D172%26epage%3D187%26doi%3D10.1056%2FNEJMra044389" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nadeem
Abbas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kausar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, H.</span></span> <span> </span><span class="NLM_article-title">Advances in targeting the epidermal growth factor receptor pathway by synthetic products and its regulation by epigenetic modulators as a therapy for glioblastoma</span>. <i>Cells</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">350</span>– <span class="NLM_lpage">361</span>, <span class="refDoi"> DOI: 10.3390/cells8040350</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.3390%2Fcells8040350" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2019&pages=350-361&author=M.+Nadeem%0AAbbasauthor=S.+Kausarauthor=F.+Wangauthor=Y.+Zhaoauthor=H.+Cui&title=Advances+in+targeting+the+epidermal+growth+factor+receptor+pathway+by+synthetic+products+and+its+regulation+by+epigenetic+modulators+as+a+therapy+for+glioblastoma&doi=10.3390%2Fcells8040350"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.3390%2Fcells8040350&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fcells8040350%26sid%3Dliteratum%253Aachs%26aulast%3DNadeem%2BAbbas%26aufirst%3DM.%26aulast%3DKausar%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DCui%26aufirst%3DH.%26atitle%3DAdvances%2520in%2520targeting%2520the%2520epidermal%2520growth%2520factor%2520receptor%2520pathway%2520by%2520synthetic%2520products%2520and%2520its%2520regulation%2520by%2520epigenetic%2520modulators%2520as%2520a%2520therapy%2520for%2520glioblastoma%26jtitle%3DCells%26date%3D2019%26volume%3D8%26spage%3D350%26epage%3D361%26doi%3D10.3390%2Fcells8040350" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nicholson, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gee, J. M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, M. E.</span></span> <span> </span><span class="NLM_article-title">EGFR and cancer prognosis</span>. <i>Eur. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">9</span>– <span class="NLM_lpage">15</span>, <span class="refDoi"> DOI: 10.1016/S0959-8049(01)00231-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1016%2FS0959-8049%2801%2900231-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=11165124" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BD3MXntlOku7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2001&pages=9-15&author=R.+I.+Nicholsonauthor=J.+M.+W.+Geeauthor=M.+E.+Harper&title=EGFR+and+cancer+prognosis&doi=10.1016%2FS0959-8049%2801%2900231-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">EGFR and cancer prognosis</span></div><div class="casAuthors">Nicholson, R. I.; Gee, J. M. W.; Harper, M. E.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Cancer</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">Suppl. 4</span>),
    <span class="NLM_cas:pages">S9-S15</span>CODEN:
                <span class="NLM_cas:coden">EJCAEL</span>;
        ISSN:<span class="NLM_cas:issn">0959-8049</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">Elevated levels of the epidermal growth factor receptor (EGFR), a growth-factor-receptor tyrosine kinase, and/or its cognate ligands have been identified as a common component of multiple cancer types and appear to promote solid tumor growth.  This article examines the relationship between EGFR expression and cancer prognosis based on literature compiled on PubMed between 1985 and Sept. 2000.  More than 200 studies were identified that analyzed relapse-free-interval or survival data directly in relation to EGFR levels in over 20000 patients.  Anal. of the data showed that 10 cancer types both express elevated levels of EGFR relative to normal tissues and have been studied in sufficient depth to allow sound judgments to be made concerning the assocn. between EGFR and patient outlook.  The EGFR was found to act as a strong prognostic indicator in head and neck, ovarian, cervical, bladder and esophageal cancers.  In these cancers, increased EGFR expression was assocd. with reduced recurrence-free or overall survival rates in 70% (52/74) of studies.  In gastric, breast, endometrial and colorectal cancers, the EGFR provided more modest prognostic information, correlating to poor survival rates in 52% (13/25) of studies, while in non-small cell lung cancer (NSCLC), EGFR expression only rarely (3/10 studies) related to patient outlook.  However, it is likely that the true prognostic significance of the EGFR has been underestimated as the published studies only assessed total cellular EGFR levels, rather than the activated form of the receptor, and were not standardized with regard to patient populations or assay methods.  Finally, it is important to stress that failure to detect a prognostic significance for EGFR in any one cancer type does not necessarily preclude patients from benefiting from anti-EGFR therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_RCCyV1BxyLVg90H21EOLACvtfcHk0lhGjhpF8-zpUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXntlOku7w%253D&md5=26c5b720b0cc173a586f652336b20c25</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2FS0959-8049%2801%2900231-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0959-8049%252801%252900231-3%26sid%3Dliteratum%253Aachs%26aulast%3DNicholson%26aufirst%3DR.%2BI.%26aulast%3DGee%26aufirst%3DJ.%2BM.%2BW.%26aulast%3DHarper%26aufirst%3DM.%2BE.%26atitle%3DEGFR%2520and%2520cancer%2520prognosis%26jtitle%3DEur.%2520J.%2520Cancer%26date%3D2001%26volume%3D37%26spage%3D9%26epage%3D15%26doi%3D10.1016%2FS0959-8049%2801%2900231-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hirsch, F. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scagliotti, G. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulshine, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwon, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curran, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paz-Ares, L.</span></span> <span> </span><span class="NLM_article-title">Lung cancer: current therapies and new targeted treatments</span>. <i>Lancet</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>389</i></span>,  <span class="NLM_fpage">299</span>– <span class="NLM_lpage">311</span>, <span class="refDoi"> DOI: 10.1016/S0140-6736(16)30958-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1016%2FS0140-6736%2816%2930958-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=27574741" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVamsbzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=389&publication_year=2017&pages=299-311&author=F.+R.+Hirschauthor=G.+V.+Scagliottiauthor=J.+L.+Mulshineauthor=R.+Kwonauthor=W.+J.+Curranauthor=Y.-L.+Wuauthor=L.+Paz-Ares&title=Lung+cancer%3A+current+therapies+and+new+targeted+treatments&doi=10.1016%2FS0140-6736%2816%2930958-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Lung cancer: current therapies and new targeted treatments</span></div><div class="casAuthors">Hirsch, Fred R.; Scagliotti, Giorgio V.; Mulshine, James L.; Kwon, Regina; Curran, Walter J., Jr.; Wu, Yi-Long; Paz-Ares, Luis</div><div class="citationInfo"><span class="NLM_cas:title">Lancet</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">389</span>
        (<span class="NLM_cas:issue">10066</span>),
    <span class="NLM_cas:pages">299-311</span>CODEN:
                <span class="NLM_cas:coden">LANCAO</span>;
        ISSN:<span class="NLM_cas:issn">0140-6736</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Lung cancer is the most frequent cause of cancer-related deaths worldwide.  Every year, 1·8 million people are diagnosed with lung cancer, and 1·6 million people die as a result of the disease. 5-yr survival rates vary from 4-17% depending on stage and regional differences.  In this Seminar, we discuss existing treatment for patients with lung cancer and the promise of precision medicine, with special emphasis on new targeted therapies.  Some subgroups, eg-patients with poor performance status and elderly patients-are not specifically addressed, because these groups require special treatment considerations and no frameworks have been established in terms of new targeted therapies.  We discuss prevention and early detection of lung cancer with an emphasis on lung cancer screening.  Although we acknowledge the importance of smoking prevention and cessation, this is a large topic beyond the scope of this Seminar.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqP_NrglINv67Vg90H21EOLACvtfcHk0lgJpGWabzAK4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVamsbzP&md5=36ace1a455a803dc6e7af83dbffb2e34</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2FS0140-6736%2816%2930958-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0140-6736%252816%252930958-8%26sid%3Dliteratum%253Aachs%26aulast%3DHirsch%26aufirst%3DF.%2BR.%26aulast%3DScagliotti%26aufirst%3DG.%2BV.%26aulast%3DMulshine%26aufirst%3DJ.%2BL.%26aulast%3DKwon%26aufirst%3DR.%26aulast%3DCurran%26aufirst%3DW.%2BJ.%26aulast%3DWu%26aufirst%3DY.-L.%26aulast%3DPaz-Ares%26aufirst%3DL.%26atitle%3DLung%2520cancer%253A%2520current%2520therapies%2520and%2520new%2520targeted%2520treatments%26jtitle%3DLancet%26date%3D2017%26volume%3D389%26spage%3D299%26epage%3D311%26doi%3D10.1016%2FS0140-6736%2816%2930958-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Herbst, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukuoka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baselga, J.</span></span> <span> </span><span class="NLM_article-title">Gefitinib--a novel targeted approach to treating cancer</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">956</span>– <span class="NLM_lpage">965</span>, <span class="refDoi"> DOI: 10.1038/nrc1506</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1038%2Fnrc1506" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=15573117" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVarsLjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2004&pages=956-965&author=R.+S.+Herbstauthor=M.+Fukuokaauthor=J.+Baselga&title=Gefitinib%2D%2Da+novel+targeted+approach+to+treating+cancer&doi=10.1038%2Fnrc1506"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Timeline: Gefitinib - a novel targeted approach to treating cancer</span></div><div class="casAuthors">Herbst, Roy S.; Fukuoka, Masahiro; Baselga, Jose</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">956-965</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Twenty years after the epidermal growth factor receptor (EGFR) was identified as a potential anticancer target, the EGFR inhibitor gefitinib (Iressa; AstraZeneca) has been approved for the treatment of patients with advanced non-small-cell lung cancer in many countries.  Studies have indicated its potential for treating patients with other types of solid tumors.  Investigation of gefitinib has not only increased our knowledge about the biol. of EGFR signaling, but is contributing to our evolving understanding of which tumors are EGFR dependent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8V2qzLg_-rrVg90H21EOLACvtfcHk0lgF2x5aenBFkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVarsLjJ&md5=03a6c08b947bc3e0ca59e1b92fa8d214</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnrc1506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc1506%26sid%3Dliteratum%253Aachs%26aulast%3DHerbst%26aufirst%3DR.%2BS.%26aulast%3DFukuoka%26aufirst%3DM.%26aulast%3DBaselga%26aufirst%3DJ.%26atitle%3DGefitinib--a%2520novel%2520targeted%2520approach%2520to%2520treating%2520cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2004%26volume%3D4%26spage%3D956%26epage%3D965%26doi%3D10.1038%2Fnrc1506" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cappuzzo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciuleanu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stelmakh, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cicenas, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szczésna, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Juhász, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esteban, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molinier, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brugger, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melezínek, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klingelschmitt, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klughammer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giaccone, G.</span></span> <span> </span><span class="NLM_article-title">Erlotinib as maintenance treatment in advanced non-small-cell lung cancer: a multicentre, randomised, placebo-controlled phase 3 study</span>. <i>Lancet Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">521</span>– <span class="NLM_lpage">529</span>, <span class="refDoi"> DOI: 10.1016/S1470-2045(10)70112-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1016%2FS1470-2045%2810%2970112-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=20493771" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC3cXntVCksb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=521-529&author=F.+Cappuzzoauthor=T.+Ciuleanuauthor=L.+Stelmakhauthor=S.+Cicenasauthor=A.+Szcz%C3%A9snaauthor=E.+Juh%C3%A1szauthor=E.+Estebanauthor=O.+Molinierauthor=W.+Bruggerauthor=I.+Melez%C3%ADnekauthor=G.+Klingelschmittauthor=B.+Klughammerauthor=G.+Giaccone&title=Erlotinib+as+maintenance+treatment+in+advanced+non-small-cell+lung+cancer%3A+a+multicentre%2C+randomised%2C+placebo-controlled+phase+3+study&doi=10.1016%2FS1470-2045%2810%2970112-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Erlotinib as maintenance treatment in advanced non-small-cell lung cancer: a multicentre, randomised, placebo-controlled phase 3 study</span></div><div class="casAuthors">Cappuzzo, Federico; Ciuleanu, Tudor; Stelmakh, Lilia; Cicenas, Saulius; Szczesna, Aleksandra; Juhasz, Erzsebet; Esteban, Emilio; Molinier, Olivier; Brugger, Wolfram; Melezinek, Ivan; Klingelschmitt, Gaelle; Klughammer, Barbara; Giaccone, Giuseppe</div><div class="citationInfo"><span class="NLM_cas:title">Lancet Oncology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">521-529</span>CODEN:
                <span class="NLM_cas:coden">LOANBN</span>;
        ISSN:<span class="NLM_cas:issn">1470-2045</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">First-line chemotherapy for advanced non-small-cell lung cancer (NSCLC) is usually limited to four to six cycles.  Maintenance therapy can delay progression and prolong survival.  The oral epidermal growth factor receptor (EGFR) tyrosine-kinase inhibitor erlotinib has proven efficacy and tolerability in second-line NSCLC.  We designed the phase 3, placebo-controlled Sequential Tarceva in Unresectable NSCLC (SATURN; BO18192) study to assess use of erlotinib as maintenance therapy in patients with non-progressive disease following first-line platinum-doublet chemotherapy.  Between Dec., 2005, and May, 2008, 1949 patients were included in the run-in phase (four cycles of platinum-based chemotherapy).  At the end of the run-in phase, 889 patients who did not have progressive disease were entered into the main study, and were randomly allocated using a 1:1 adaptive randomisation method through a third-party interactive voice response system to receive erlotinib (150 mg/day; n=438) or placebo (n=451) until progression or unacceptable toxicity.  Patients were stratified by EGFR immunohistochem. status, stage, Eastern Cooperative Oncol. Group performance status, chemotherapy regimen, smoking history, and region.  Co-primary endpoints were progression-free survival (PFS) in all analysable patients irresp. of EGFR status, and PFS in patients whose tumors had EGFR protein overexpression, as detd. by immunohistochem.  This study is registered with www.ClinicalTrials.gov, no. NCT00556712.  884 patients were analysable for PFS; 437 in the erlotinib group and 447 in the placebo group.  After a median follow-up of 11.4 mo for the erlotinib group and 11.5 mo for the placebo group, median PFS was significantly longer with erlotinib than with placebo: 12.3 wk for patients in the erlotinib group vs. 11.1 wk for those in the placebo group (HR 0.71, 95% CI 0.62-0.82; p<0.0001).  PFS was also significantly longer in patients with EGFR-pos. immunohistochem. who were treated with erlotinib (n=307) compared with EGFR-pos. patients given placebo (n=311; median PFS 12.3 wk in the erlotinib group vs 11.1 wk in the placebo group; HR 0.69, 0.58-0.82; p<0.0001).  The most common grade 3 or higher adverse events were rash (37 [9%] of 443 patients in the erlotinib group vs none of 445 in the placebo group) and diarrhoea (seven [2%] of 443 patients vs none of 445).  Serious adverse events were reported in 47 patients (11%) on erlotinib compared with 34 patients (8%) on placebo.  The most common serious adverse event was pneumonia (seven cases [2%] with erlotinib and four [<1%] with placebo).  Maintenance therapy with erlotinib for patients with NSCLC is well tolerated and significantly prolongs PFS compared with placebo.  First-line maintenance with erlotinib could be considered in patients who do not progress after four cycles of chemotherapy.  Funding: F Hoffmann-La Roche Ltd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQ1stlOmTgyrVg90H21EOLACvtfcHk0liNWv_59G1-tg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXntVCksb0%253D&md5=384f65f7c8ad10ca34d2d0fed7183484</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2FS1470-2045%2810%2970112-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1470-2045%252810%252970112-1%26sid%3Dliteratum%253Aachs%26aulast%3DCappuzzo%26aufirst%3DF.%26aulast%3DCiuleanu%26aufirst%3DT.%26aulast%3DStelmakh%26aufirst%3DL.%26aulast%3DCicenas%26aufirst%3DS.%26aulast%3DSzcz%25C3%25A9sna%26aufirst%3DA.%26aulast%3DJuh%25C3%25A1sz%26aufirst%3DE.%26aulast%3DEsteban%26aufirst%3DE.%26aulast%3DMolinier%26aufirst%3DO.%26aulast%3DBrugger%26aufirst%3DW.%26aulast%3DMelez%25C3%25ADnek%26aufirst%3DI.%26aulast%3DKlingelschmitt%26aufirst%3DG.%26aulast%3DKlughammer%26aufirst%3DB.%26aulast%3DGiaccone%26aufirst%3DG.%26atitle%3DErlotinib%2520as%2520maintenance%2520treatment%2520in%2520advanced%2520non-small-cell%2520lung%2520cancer%253A%2520a%2520multicentre%252C%2520randomised%252C%2520placebo-controlled%2520phase%25203%2520study%26jtitle%3DLancet%2520Oncol.%26date%3D2010%26volume%3D11%26spage%3D521%26epage%3D529%26doi%3D10.1016%2FS1470-2045%2810%2970112-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yun, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mengwasser, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toms, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woo, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greulich, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyerson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eck, M. J.</span></span> <span> </span><span class="NLM_article-title">The T790M mutation in EGFR kinase causes drug resistance by increasing the affinity for ATP</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>105</i></span>,  <span class="NLM_fpage">2070</span>– <span class="NLM_lpage">2075</span>, <span class="refDoi"> DOI: 10.1073/pnas.0709662105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1073%2Fpnas.0709662105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=18227510" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BD1cXitl2mu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2008&pages=2070-2075&author=C.+H.+Yunauthor=K.+E.+Mengwasserauthor=A.+V.+Tomsauthor=M.+S.+Wooauthor=H.+Greulichauthor=K.+K.+Wongauthor=M.+Meyersonauthor=M.+J.+Eck&title=The+T790M+mutation+in+EGFR+kinase+causes+drug+resistance+by+increasing+the+affinity+for+ATP&doi=10.1073%2Fpnas.0709662105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">The T790M mutation in EGFR kinase causes drug resistance by increasing the affinity for ATP</span></div><div class="casAuthors">Yun, Cai-Hong; Mengwasser, Kristen E.; Toms, Angela V.; Woo, Michele S.; Greulich, Heidi; Wong, Kwok-Kin; Meyerson, Matthew; Eck, Michael J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2070-2075</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Lung cancers caused by activating mutations in the epidermal growth factor receptor (EGFR) are initially responsive to small mol. tyrosine kinase inhibitors (TKIs), but the efficacy of these agents is often limited because of the emergence of drug resistance conferred by a second mutation, T790M.  Threonine 790 is the "gatekeeper" residue, an important determinant of inhibitor specificity in the ATP binding pocket.  The T790M mutation has been thought to cause resistance by sterically blocking binding of TKIs such as gefitinib and erlotinib, but this explanation is difficult to reconcile with the fact that it remains sensitive to structurally similar irreversible inhibitors.  Here, we show by using a direct binding assay that T790M mutants retain low-nanomolar affinity for gefitinib.  Furthermore, we show that the T790M mutation activates WT EGFR and that introduction of the T790M mutation increases the ATP affinity of the oncogenic L858R mutant by more than an order of magnitude.  The increased ATP affinity is the primary mechanism by which the T790M mutation confers drug resistance.  Crystallog. anal. of the T790M mutant shows how it can adapt to accommodate tight binding of diverse inhibitors, including the irreversible inhibitor HKI-272, and also suggests a structural mechanism for catalytic activation.  We conclude that the T790M mutation is a "generic" resistance mutation that will reduce the potency of any ATP-competitive kinase inhibitor and that irreversible inhibitors overcome this resistance simply through covalent binding, not as a result of an alternative binding mode.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXXP7TBlyrdrVg90H21EOLACvtfcHk0liNWv_59G1-tg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXitl2mu7k%253D&md5=f5f5d791a65f73ca628d95e9922b8ea8</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0709662105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0709662105%26sid%3Dliteratum%253Aachs%26aulast%3DYun%26aufirst%3DC.%2BH.%26aulast%3DMengwasser%26aufirst%3DK.%2BE.%26aulast%3DToms%26aufirst%3DA.%2BV.%26aulast%3DWoo%26aufirst%3DM.%2BS.%26aulast%3DGreulich%26aufirst%3DH.%26aulast%3DWong%26aufirst%3DK.%2BK.%26aulast%3DMeyerson%26aufirst%3DM.%26aulast%3DEck%26aufirst%3DM.%2BJ.%26atitle%3DThe%2520T790M%2520mutation%2520in%2520EGFR%2520kinase%2520causes%2520drug%2520resistance%2520by%2520increasing%2520the%2520affinity%2520for%2520ATP%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2008%26volume%3D105%26spage%3D2070%26epage%3D2075%26doi%3D10.1073%2Fpnas.0709662105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Godin-Heymann, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ulkus, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brannigan, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDermott, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamb, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maheswaran, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Settleman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haber, D. A.</span></span> <span> </span><span class="NLM_article-title">The T790M “gatekeeper” mutation in EGFR mediates resistance to low concentrations of an irreversible EGFR inhibitor</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">874</span>– <span class="NLM_lpage">879</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.MCT-07-2387</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1158%2F1535-7163.MCT-07-2387" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=18413800" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BD1cXks1GmsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=874-879&author=N.+Godin-Heymannauthor=L.+Ulkusauthor=B.+W.+Branniganauthor=U.+McDermottauthor=J.+Lambauthor=S.+Maheswaranauthor=J.+Settlemanauthor=D.+A.+Haber&title=The+T790M+%E2%80%9Cgatekeeper%E2%80%9D+mutation+in+EGFR+mediates+resistance+to+low+concentrations+of+an+irreversible+EGFR+inhibitor&doi=10.1158%2F1535-7163.MCT-07-2387"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">The T790M "gatekeeper" mutation in EGFR mediates resistance to low concentrations of an irreversible EGFR inhibitor</span></div><div class="casAuthors">Godin-Heymann, Nadia; Ulkus, Lindsey; Brannigan, Brian W.; McDermott, Ultan; Lamb, Jennifer; Maheswaran, Shyamala; Settleman, Jeffrey; Haber, Daniel A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">874-879</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Patients with non-small cell lung cancer (NSCLC) harboring activating mutations in the epidermal growth factor receptor (EGFR) kinase domain tend to respond well to the tyrosine kinase inhibitors, gefitinib and erlotinib.  However, following clin. response, these patients typically relapse within a year of treatment.  In many cases, resistance is caused by an acquired secondary EGFR kinase domain mutation, T790M.  In vitro studies have shown that a new class of EGFR-irreversible inhibitors could overcome the resistance conferred by T790M.  Clin. trials are under way to examine the efficacy of one of these inhibitors, HKI-272, in patients with NSCLC who initially responded to gefitinib/erlotinib and subsequently relapsed.  To anticipate the possibility that patients who respond to irreversible inhibitors will develop secondary resistance to such inhibitors, as has been seen in other similar settings, we modeled acquired resistance to the dual EGFR/HER2-irreversible tyrosine kinase inhibitor HKI-272 in a NSCLC cell culture model.  We found that HKI-272-resistant clones fall into two biochem. groups based on the retention of EGFR phosphorylation in the presence of the drug.  Cells that retain phosphorylated EGFR have acquired the secondary mutation T790M.  Moreover, HKI-272 can overcome T790M resistance only at suprapharmacol. concns.  We further model mutations at EGFR C797 as a mechanism of resistance to irreversible EGFR inhibitors and show that although these mutants are resistant to the irreversible inhibitor, they retain erlotinib sensitivity.  Our findings suggest that HKI-272 treatment at maximally tolerated dosing may lead to the emergence of T790M-mediated resistance, whereas treatment with a more potent irreversible inhibitor could yield a resistance mutation at EGFR C797.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGra5FtRNHnlNLVg90H21EOLACvtfcHk0ljYLhZtRmE8Aw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXks1GmsLo%253D&md5=f71cd83bfabef2d7b2c5d8b6e5cfac6a</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-07-2387&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-07-2387%26sid%3Dliteratum%253Aachs%26aulast%3DGodin-Heymann%26aufirst%3DN.%26aulast%3DUlkus%26aufirst%3DL.%26aulast%3DBrannigan%26aufirst%3DB.%2BW.%26aulast%3DMcDermott%26aufirst%3DU.%26aulast%3DLamb%26aufirst%3DJ.%26aulast%3DMaheswaran%26aufirst%3DS.%26aulast%3DSettleman%26aufirst%3DJ.%26aulast%3DHaber%26aufirst%3DD.%2BA.%26atitle%3DThe%2520T790M%2520%25E2%2580%259Cgatekeeper%25E2%2580%259D%2520mutation%2520in%2520EGFR%2520mediates%2520resistance%2520to%2520low%2520concentrations%2520of%2520an%2520irreversible%2520EGFR%2520inhibitor%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2008%26volume%3D7%26spage%3D874%26epage%3D879%26doi%3D10.1158%2F1535-7163.MCT-07-2387" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Politi, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riely, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Somwar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zakowski, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kris, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varmus, H.</span></span> <span> </span><span class="NLM_article-title">Acquired resistance of lung adenocarcinomas to gefitinib or erlotinib is associated with a second mutation in the EGFR kinase domain</span>. <i>PLoS Med.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>2</i></span>, <span class="NLM_elocation-id">e73</span> <span class="refDoi"> DOI: 10.1371/journal.pmed.0020073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1371%2Fjournal.pmed.0020073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=15737014" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A280%3ADC%252BD2MzjtVehsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2005&author=W.+Paoauthor=V.+A.+Millerauthor=K.+A.+Politiauthor=G.+J.+Rielyauthor=R.+Somwarauthor=M.+F.+Zakowskiauthor=M.+G.+Krisauthor=H.+Varmus&title=Acquired+resistance+of+lung+adenocarcinomas+to+gefitinib+or+erlotinib+is+associated+with+a+second+mutation+in+the+EGFR+kinase+domain&doi=10.1371%2Fjournal.pmed.0020073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Acquired resistance of lung adenocarcinomas to gefitinib or erlotinib is associated with a second mutation in the EGFR kinase domain</span></div><div class="casAuthors">Pao William; Miller Vincent A; Politi Katerina A; Riely Gregory J; Somwar Romel; Zakowski Maureen F; Kris Mark G; Varmus Harold</div><div class="citationInfo"><span class="NLM_cas:title">PLoS medicine</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e73</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Lung adenocarcinomas from patients who respond to the tyrosine kinase inhibitors gefitinib (Iressa) or erlotinib (Tarceva) usually harbor somatic gain-of-function mutations in exons encoding the kinase domain of the epidermal growth factor receptor (EGFR).  Despite initial responses, patients eventually progress by unknown mechanisms of "acquired" resistance.  METHODS AND FINDINGS:  We show that in two of five patients with acquired resistance to gefitinib or erlotinib, progressing tumors contain, in addition to a primary drug-sensitive mutation in EGFR, a secondary mutation in exon 20, which leads to substitution of methionine for threonine at position 790 (T790M) in the kinase domain.  Tumor cells from a sixth patient with a drug-sensitive EGFR mutation whose tumor progressed on adjuvant gefitinib after complete resection also contained the T790M mutation.  This mutation was not detected in untreated tumor samples.  Moreover, no tumors with acquired resistance had KRAS mutations, which have been associated with primary resistance to these drugs.  Biochemical analyses of transfected cells and growth inhibition studies with lung cancer cell lines demonstrate that the T790M mutation confers resistance to EGFR mutants usually sensitive to either gefitinib or erlotinib.  Interestingly, a mutation analogous to T790M has been observed in other kinases with acquired resistance to another kinase inhibitor, imatinib (Gleevec).  CONCLUSION:  In patients with tumors bearing gefitinib- or erlotinib-sensitive EGFR mutations, resistant subclones containing an additional EGFR mutation emerge in the presence of drug.  This observation should help guide the search for more effective therapy against a specific subset of lung cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcREkXMV3u1nwwMthz5_Rx5rfW6udTcc2eatz6oFsd6NLLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2MzjtVehsw%253D%253D&md5=3e626105e0dea2a38997156ae513f56b</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pmed.0020073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pmed.0020073%26sid%3Dliteratum%253Aachs%26aulast%3DPao%26aufirst%3DW.%26aulast%3DMiller%26aufirst%3DV.%2BA.%26aulast%3DPoliti%26aufirst%3DK.%2BA.%26aulast%3DRiely%26aufirst%3DG.%2BJ.%26aulast%3DSomwar%26aufirst%3DR.%26aulast%3DZakowski%26aufirst%3DM.%2BF.%26aulast%3DKris%26aufirst%3DM.%2BG.%26aulast%3DVarmus%26aufirst%3DH.%26atitle%3DAcquired%2520resistance%2520of%2520lung%2520adenocarcinomas%2520to%2520gefitinib%2520or%2520erlotinib%2520is%2520associated%2520with%2520a%2520second%2520mutation%2520in%2520the%2520EGFR%2520kinase%2520domain%26jtitle%3DPLoS%2520Med.%26date%3D2005%26volume%3D2%26doi%3D10.1371%2Fjournal.pmed.0020073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boggon, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dayaram, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janne, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kocher, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyerson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eck, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tenen, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halmos, B.</span></span> <span> </span><span class="NLM_article-title">EGFR mutation and resistance of non-small-cell lung cancer to gefitinib</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>352</i></span>,  <span class="NLM_fpage">786</span>– <span class="NLM_lpage">792</span>, <span class="refDoi"> DOI: 10.1056/NEJMoa044238</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1056%2FNEJMoa044238" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=15728811" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhsFCiurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=352&publication_year=2005&pages=786-792&author=S.+Kobayashiauthor=T.+J.+Boggonauthor=T.+Dayaramauthor=P.+A.+Janneauthor=O.+Kocherauthor=M.+Meyersonauthor=B.+E.+Johnsonauthor=M.+J.+Eckauthor=D.+G.+Tenenauthor=B.+Halmos&title=EGFR+mutation+and+resistance+of+non-small-cell+lung+cancer+to+gefitinib&doi=10.1056%2FNEJMoa044238"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">EGFR mutation and resistance of non-small-cell lung cancer to gefitinib</span></div><div class="casAuthors">Kobayashi, Susumu; Boggon, Titus J.; Dayaram, Tajhal; Jaenne, Pasi A.; Kocher, Olivier; Meyerson, Matthew; Johnson, Bruce E.; Eck, Michael J.; Tenen, Daniel G.; Halmos, Balizs</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">352</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">786-792</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">Mutations of the epidermal growth factor receptor (EGFR) gene were identified in specimens from patients with non-small-cell lung cancer who have a response to anilinoquinazoline EGFR inhibitors.  Despite the dramatic responses to such inhibitors, most patients ultimately have a relapse.  The mechanism of the drug resistance is unknown.  Here the authors report the case of a patient with EGFR-mutant, gefitinib-responsive, advanced non-small-cell lung cancer who had a relapse after 2 years of complete remission during treatment with gefitinib.  The DNA sequence of the EGFR gene in his tumor biopsy specimen at relapse revealed the presence of a second point mutation, resulting in threonine-to-methionine amino acid change at position 790 of EGFR.  Structural modeling and biochem. studies showed that this second mutation led to gefitinib resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8XXhT1hrbe7Vg90H21EOLACvtfcHk0liknxx63L5M5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhsFCiurk%253D&md5=808223ad566e2a3748977727cc5a9846</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1056%2FNEJMoa044238&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMoa044238%26sid%3Dliteratum%253Aachs%26aulast%3DKobayashi%26aufirst%3DS.%26aulast%3DBoggon%26aufirst%3DT.%2BJ.%26aulast%3DDayaram%26aufirst%3DT.%26aulast%3DJanne%26aufirst%3DP.%2BA.%26aulast%3DKocher%26aufirst%3DO.%26aulast%3DMeyerson%26aufirst%3DM.%26aulast%3DJohnson%26aufirst%3DB.%2BE.%26aulast%3DEck%26aufirst%3DM.%2BJ.%26aulast%3DTenen%26aufirst%3DD.%2BG.%26aulast%3DHalmos%26aufirst%3DB.%26atitle%3DEGFR%2520mutation%2520and%2520resistance%2520of%2520non-small-cell%2520lung%2520cancer%2520to%2520gefitinib%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2005%26volume%3D352%26spage%3D786%26epage%3D792%26doi%3D10.1056%2FNEJMoa044238" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thress, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paweletz, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felip, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stetson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dougherty, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Markovets, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vivancos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ercan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthews, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantarini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janne, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oxnard, G. R.</span></span> <span> </span><span class="NLM_article-title">Acquired EGFR C797S mutation mediates resistance to AZD9291 in non-small cell lung cancer harboring EGFR T790M</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">560</span>– <span class="NLM_lpage">562</span>, <span class="refDoi"> DOI: 10.1038/nm.3854</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1038%2Fnm.3854" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=25939061" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFemsbfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=560-562&author=K.+S.+Thressauthor=C.+P.+Paweletzauthor=E.+Felipauthor=B.+C.+Choauthor=D.+Stetsonauthor=B.+Doughertyauthor=Z.+Laiauthor=A.+Markovetsauthor=A.+Vivancosauthor=Y.+Kuangauthor=D.+Ercanauthor=S.+E.+Matthewsauthor=M.+Cantariniauthor=J.+C.+Barrettauthor=P.+A.+Janneauthor=G.+R.+Oxnard&title=Acquired+EGFR+C797S+mutation+mediates+resistance+to+AZD9291+in+non-small+cell+lung+cancer+harboring+EGFR+T790M&doi=10.1038%2Fnm.3854"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Acquired EGFR C797S mutation mediates resistance to AZD9291 in non-small cell lung cancer harboring EGFR T790M</span></div><div class="casAuthors">Thress, Kenneth S.; Paweletz, Cloud P.; Felip, Enriqueta; Cho, Byoung Chul; Stetson, Daniel; Dougherty, Brian; Lai, Zhongwu; Markovets, Aleksandra; Vivancos, Ana; Kuang, Yanan; Ercan, Dalia; Matthews, Sarah E.; Cantarini, Mireille; Barrett, J. Carl; Janne, Pasi A.; Oxnard, Geoffrey R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">560-562</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Here we studied cell-free plasma DNA (cfDNA) collected from subjects with advanced lung cancer whose tumors had developed resistance to the epidermal growth factor receptor (EGFR) tyrosine kinase inhibitor (TKI) AZD9291.  We first performed next-generation sequencing of cfDNA from seven subjects and detected an acquired EGFR C797S mutation in one; expression of this mutant EGFR construct in a cell line rendered it resistant to AZD9291.  We then performed droplet digital PCR on serial cfDNA specimens collected from 15 AZD9291-treated subjects.  All were pos. for the T790M mutation before treatment, but upon developing AZD9291 resistance three mol. subtypes emerged: six cases acquired the C797S mutation, five cases maintained the T790M mutation but did not acquire the C797S mutation and four cases lost the T790M mutation despite the presence of the underlying EGFR activating mutation.  Our findings provide insight into the diversity of mechanisms through which tumors acquire resistance to AZD9291 and highlight the need for therapies that are able to overcome resistance mediated by the EGFR C797S mutation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTggAW5fpGrbVg90H21EOLACvtfcHk0ljGvCXRSc_txw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFemsbfJ&md5=adac4cb6c7057855aff61fd62890d010</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fnm.3854&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.3854%26sid%3Dliteratum%253Aachs%26aulast%3DThress%26aufirst%3DK.%2BS.%26aulast%3DPaweletz%26aufirst%3DC.%2BP.%26aulast%3DFelip%26aufirst%3DE.%26aulast%3DCho%26aufirst%3DB.%2BC.%26aulast%3DStetson%26aufirst%3DD.%26aulast%3DDougherty%26aufirst%3DB.%26aulast%3DLai%26aufirst%3DZ.%26aulast%3DMarkovets%26aufirst%3DA.%26aulast%3DVivancos%26aufirst%3DA.%26aulast%3DKuang%26aufirst%3DY.%26aulast%3DErcan%26aufirst%3DD.%26aulast%3DMatthews%26aufirst%3DS.%2BE.%26aulast%3DCantarini%26aufirst%3DM.%26aulast%3DBarrett%26aufirst%3DJ.%2BC.%26aulast%3DJanne%26aufirst%3DP.%2BA.%26aulast%3DOxnard%26aufirst%3DG.%2BR.%26atitle%3DAcquired%2520EGFR%2520C797S%2520mutation%2520mediates%2520resistance%2520to%2520AZD9291%2520in%2520non-small%2520cell%2520lung%2520cancer%2520harboring%2520EGFR%2520T790M%26jtitle%3DNat.%2520Med.%26date%3D2015%26volume%3D21%26spage%3D560%26epage%3D562%26doi%3D10.1038%2Fnm.3854" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smaill, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, K.</span></span> <span> </span><span class="NLM_article-title">Targeting EGFR(L858R/T790M) and EGFR(L858R/T790M/C797S) resistance mutations in NSCLC: Current developments in medicinal chemistry</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">1550</span>– <span class="NLM_lpage">1581</span>, <span class="refDoi"> DOI: 10.1002/med.21488</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1002%2Fmed.21488" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=29377179" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVyqu73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2018&pages=1550-1581&author=X.+Luauthor=L.+Yuauthor=Z.+Zhangauthor=X.+Renauthor=J.+B.+Smaillauthor=K.+Ding&title=Targeting+EGFR%28L858R%2FT790M%29+and+EGFR%28L858R%2FT790M%2FC797S%29+resistance+mutations+in+NSCLC%3A+Current+developments+in+medicinal+chemistry&doi=10.1002%2Fmed.21488"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting EGFRL858R/T790M and EGFRL858R/T790M/C797S resistance mutations in NSCLC: Current developments in medicinal chemistry</span></div><div class="casAuthors">Lu, Xiaoyun; Yu, Lei; Zhang, Zhang; Ren, Xiaomei; Smaill, Jeff B.; Ding, Ke</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1550-1581</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Both the first-generation reversible epidermal growth factor receptor (EGFR) inhibitors gefitinib and erlotinib and the second-generation covalent epidermal growth factor receptor tyrosine kinase inhibitor (EGFR-TKI) afatinib have significantly improved the survival of non-small-cell lung cancer (NSCLC) patients with activating EGFR mutations.  However, a secondary EGFRT790M mutation leads to the clin. acquired resistance to the first- and second-generation EGFR-TKIs drugs.  A no. of the third-generation wild-type sparing EGFR inhibitors, for example, WZ4002, CO1686, AZD9291, HM61713, EGF816, ASP8173, and PF0674775, have been developed, among which AZD9291 has been approved by US FDA for the treatment of NSCLC patients with EGFRT790M.  More recently, a tertiary EGFRC797S mutation was reported as the dominant resistance mechanism to the third-generation irreversible inhibitors.  It is highly desirable to develop the fourth-generation EGFR inhibitors.  This review summarizes the mechanisms of acquired resistance and the latest medicinal chem. advances on the third- and fourth-generation EGFR inhibitors, with special attention being paid to the allosteric and reversible inhibitors combating the tertiary EGFRC797S mutation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNK_1c8SI0qbVg90H21EOLACvtfcHk0lhdXUFEY98gsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVyqu73P&md5=71288118ca1d9eaab6d07b056bb5ab56</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1002%2Fmed.21488&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21488%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DX.%26aulast%3DYu%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DRen%26aufirst%3DX.%26aulast%3DSmaill%26aufirst%3DJ.%2BB.%26aulast%3DDing%26aufirst%3DK.%26atitle%3DTargeting%2520EGFR%2528L858R%252FT790M%2529%2520and%2520EGFR%2528L858R%252FT790M%252FC797S%2529%2520resistance%2520mutations%2520in%2520NSCLC%253A%2520Current%2520developments%2520in%2520medicinal%2520chemistry%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2018%26volume%3D38%26spage%3D1550%26epage%3D1581%26doi%3D10.1002%2Fmed.21488" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Patel, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pawara, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ansari, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Surana, S.</span></span> <span> </span><span class="NLM_article-title">Recent updates on third generation EGFR inhibitors and emergence of fourth generation EGFR inhibitors to combat C797S resistance</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>142</i></span>,  <span class="NLM_fpage">32</span>– <span class="NLM_lpage">47</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2017.05.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1016%2Fj.ejmech.2017.05.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=28526474" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnvFGmtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=142&publication_year=2017&pages=32-47&author=H.+Patelauthor=R.+Pawaraauthor=A.+Ansariauthor=S.+Surana&title=Recent+updates+on+third+generation+EGFR+inhibitors+and+emergence+of+fourth+generation+EGFR+inhibitors+to+combat+C797S+resistance&doi=10.1016%2Fj.ejmech.2017.05.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Recent updates on third generation EGFR inhibitors and emergence of fourth generation EGFR inhibitors to combat C797S resistance</span></div><div class="casAuthors">Patel, Harun; Pawara, Rahul; Ansari, Azim; Surana, Sanjay</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">142</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">32-47</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A review.  EGFR T790M mutation leads to resistance to most of clin. available EGFR TKIs.  Third-generation EGFR TKIs against the T790M mutation have been in active clin. development, which includes osimertinib, rociletinib, HM61713, ASP8273, EGF816, and PF-06747775.  On the other hand recently EGFR C797S mutation was reported to be a leading mechanism of resistance to the third-generation inhibitors.  The C797S mutation appears to be an ideal target for overcoming the acquired resistance to the third generation inhibitors.  This review summarizes the third generation inhibitors, synthesis, their mechanism of resistance and latest development on the discovery of a fourth-generation EGFR TKIs and U to Y allosteric strategies to combat the C797S EGFR resistance problem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprj-V5hUSHGrVg90H21EOLACvtfcHk0lhNifqEGmuZfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnvFGmtbo%253D&md5=d0ed35bc7deae5ea37a95910df356c1d</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2017.05.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2017.05.027%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DH.%26aulast%3DPawara%26aufirst%3DR.%26aulast%3DAnsari%26aufirst%3DA.%26aulast%3DSurana%26aufirst%3DS.%26atitle%3DRecent%2520updates%2520on%2520third%2520generation%2520EGFR%2520inhibitors%2520and%2520emergence%2520of%2520fourth%2520generation%2520EGFR%2520inhibitors%2520to%2520combat%2520C797S%2520resistance%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D142%26spage%3D32%26epage%3D47%26doi%3D10.1016%2Fj.ejmech.2017.05.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tan, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soo, R. A.</span></span> <span> </span><span class="NLM_article-title">Next-generation epidermal growth factor receptor tyrosine kinase inhibitors in epidermal growth factor receptor-mutant non-small cell lung cancer</span>. <i>Lung Cancer</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">59</span>– <span class="NLM_lpage">68</span>, <span class="refDoi"> DOI: 10.1016/j.lungcan.2016.01.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1016%2Fj.lungcan.2016.01.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=26898616" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A280%3ADC%252BC28jivVymtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=2016&pages=59-68&author=C.+S.+Tanauthor=B.+C.+Choauthor=R.+A.+Soo&title=Next-generation+epidermal+growth+factor+receptor+tyrosine+kinase+inhibitors+in+epidermal+growth+factor+receptor-mutant+non-small+cell+lung+cancer&doi=10.1016%2Fj.lungcan.2016.01.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Next-generation epidermal growth factor receptor tyrosine kinase inhibitors in epidermal growth factor receptor -mutant non-small cell lung cancer</span></div><div class="casAuthors">Tan Chee-Seng; Cho Byoung-Chul; Soo Ross A</div><div class="citationInfo"><span class="NLM_cas:title">Lung cancer (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">59-68</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Since the discovery of sensitizing EGFR mutations as a predictive marker of sensitivity to EGFR tyrosine kinase inhibitors (TKIs), the field of targeted therapy in non-small cell lung cancer (NSCLC) has been revolutionized.  Patients harbouring these sensitizing mutations treated with EGFR TKI have derived significant clinical outcome when compared with standard platinum based chemotherapy doublets.  However disease progression invariably occurs at a median of about 9-13 months from initiation treatment, if acquired resistance commonly due to the development of EGFR T790M mutation.  A novel class of "third generation" EGFR TKIs have been developed that is sensitising and T790M mutant-specific whilst sparing WT EGFR, representing a significant breakthrough in the treatment in NSCLC patients with acquired resistance harboring these genotypes.  Early phase clinical data suggest the third generation EGFR TKIs such as osimertinib, rociletinib, and HM61713 are highly efficacious and well tolerated.  Another promising class of EGFR TKI such as AZD3759 has been designed to penetrate blood brain barrier to treat brain metastases and leptomeningeal disease and has showed promising responses in patients with brain metastases.  Acquired resistance to third generation EGFR TKIs has been reported including EGFR C797S.  Given its non-invasive nature, plasma ctDNA is being explored as a possible approach to detect T790M mutation and to also inform on novel molecular mechansims of tertiary resistance to third generation EGFR TKIs.  An understanding of the mechanisms of acquired resistance to the third-generation EGFR TKIs will greatly aid in the development of the next generation of EGFR TKIs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ6MbARC-V4Af3jf77P2BG3fW6udTcc2ebajZAU83XeArntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28jivVymtA%253D%253D&md5=b1ab30df28665605ce44ce1b5de385ff</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.lungcan.2016.01.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.lungcan.2016.01.003%26sid%3Dliteratum%253Aachs%26aulast%3DTan%26aufirst%3DC.%2BS.%26aulast%3DCho%26aufirst%3DB.%2BC.%26aulast%3DSoo%26aufirst%3DR.%2BA.%26atitle%3DNext-generation%2520epidermal%2520growth%2520factor%2520receptor%2520tyrosine%2520kinase%2520inhibitors%2520in%2520epidermal%2520growth%2520factor%2520receptor-mutant%2520non-small%2520cell%2520lung%2520cancer%26jtitle%3DLung%2520Cancer%26date%3D2016%26volume%3D93%26spage%3D59%26epage%3D68%26doi%3D10.1016%2Fj.lungcan.2016.01.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span> <span> </span><span class="NLM_article-title">EAI045: The fourth-generation EGFR inhibitor overcoming T790M and C797S resistance</span>. <i>Cancer Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>385</i></span>,  <span class="NLM_fpage">51</span>– <span class="NLM_lpage">54</span>, <span class="refDoi"> DOI: 10.1016/j.canlet.2016.11.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1016%2Fj.canlet.2016.11.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=27840244" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVeqtL%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=385&publication_year=2017&pages=51-54&author=S.+Wangauthor=Y.+Songauthor=D.+Liu&title=EAI045%3A+The+fourth-generation+EGFR+inhibitor+overcoming+T790M+and+C797S+resistance&doi=10.1016%2Fj.canlet.2016.11.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">EAI045: The fourth-generation EGFR inhibitor overcoming T790M and C797S resistance</span></div><div class="casAuthors">Wang, Shuhang; Song, Yongping; Liu, Delong</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Letters (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">385</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">51-54</span>CODEN:
                <span class="NLM_cas:coden">CALEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The third-generation tyrosine kinase inhibitors (TKI), AZD9291 (osimertinib) and CO-1686 (rociletinib) of epidermal growth factor receptor (EGFR) are highly active against T790M pos. non-small cell lung cancer (NSCLC).  However, resistance develops rapidly.  EGFR C797S mutation was reported to be a leading mechanism of resistance to the third-generation inhibitors.  The C797S mutation appears to be an ideal target for overcoming the acquired resistance to the third-generation inhibitors.  This review summarizes the latest development on the discovery of a fourth-generation EGFR TKI, EAI045.3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2GMAIDfgfeLVg90H21EOLACvtfcHk0lhPOXhboycAEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVeqtL%252FM&md5=e429055412bec2ec2da28b2b959753ae</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.canlet.2016.11.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.canlet.2016.11.008%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DS.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DD.%26atitle%3DEAI045%253A%2520The%2520fourth-generation%2520EGFR%2520inhibitor%2520overcoming%2520T790M%2520and%2520C797S%2520resistance%26jtitle%3DCancer%2520Lett.%26date%3D2017%26volume%3D385%26spage%3D51%26epage%3D54%26doi%3D10.1016%2Fj.canlet.2016.11.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span> <span> </span><span class="NLM_article-title">Discovery of potent and noncovalent reversible EGFR kinase inhibitors of EGFR(L858R/T790M/C797S)</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">869</span>– <span class="NLM_lpage">873</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.8b00564</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.8b00564" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC1MXpvFams7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=869-873&author=Q.+Liauthor=T.+Zhangauthor=S.+Liauthor=L.+Tongauthor=J.+Liauthor=Z.+Suauthor=F.+Fengauthor=D.+Sunauthor=Y.+Tongauthor=X.+Wangauthor=Z.+Zhaoauthor=L.+Zhuauthor=J.+Dingauthor=H.+Liauthor=H.+Xieauthor=Y.+Xu&title=Discovery+of+potent+and+noncovalent+reversible+EGFR+kinase+inhibitors+of+EGFR%28L858R%2FT790M%2FC797S%29&doi=10.1021%2Facsmedchemlett.8b00564"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Potent and Noncovalent Reversible EGFR Kinase Inhibitors of EGFRL858R/T790M/C797S</span></div><div class="casAuthors">Li, Qiannan; Zhang, Tao; Li, Shiliang; Tong, Linjiang; Li, Junyu; Su, Zhicheng; Feng, Fang; Sun, Deheng; Tong, Yi; Wang, Xia; Zhao, Zhenjiang; Zhu, Lili; Ding, Jian; Li, Honglin; Xie, Hua; Xu, Yufang</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">869-873</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In this paper, we describe the discovery and optimization of a series of noncovalent reversible epidermal growth factor receptor inhibitors of EGFRL858R/T790M/C797S.  One of the most promising compds., I, inhibited the enzymic activity of EGFRL858R/T790M/C797S with an IC50 value of 2.2 nM.  Cell proliferation assays showed that I effectively and selectively inhibited the growth of EGFRL858R/T790M/C797S-dependent cells.  This series of compds., which occupy both the ATP binding site and the allosteric site of the EGFR kinase, may serve as a basis for the development of fourth-generation EGFR inhibitors for L858R/T790M/C797S mutants.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqcQ04p4560bVg90H21EOLACvtfcHk0lhjAVuG0y4Z8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXpvFams7o%253D&md5=396eb9b29217fbb74189193a544819b3</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.8b00564&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.8b00564%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DTong%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DSu%26aufirst%3DZ.%26aulast%3DFeng%26aufirst%3DF.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DTong%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DZhu%26aufirst%3DL.%26aulast%3DDing%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DXie%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DY.%26atitle%3DDiscovery%2520of%2520potent%2520and%2520noncovalent%2520reversible%2520EGFR%2520kinase%2520inhibitors%2520of%2520EGFR%2528L858R%252FT790M%252FC797S%2529%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D10%26spage%3D869%26epage%3D873%26doi%3D10.1021%2Facsmedchemlett.8b00564" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Park, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, H.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mah, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of EGF receptor inhibitors that are selective for the d746-750/T790M/C797S mutant through structure-based de novo design</span>. <i>Angew. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>129</i></span>,  <span class="NLM_fpage">7742</span>– <span class="NLM_lpage">7746</span>, <span class="refDoi"> DOI: 10.1002/ange.201703389</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1002%2Fange.201703389" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2017&pages=7742-7746&author=H.+Parkauthor=H.-Y.+Jungauthor=S.+Mahauthor=S.+Hong&title=Discovery+of+EGF+receptor+inhibitors+that+are+selective+for+the+d746-750%2FT790M%2FC797S+mutant+through+structure-based+de+novo+design&doi=10.1002%2Fange.201703389"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1002%2Fange.201703389&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fange.201703389%26sid%3Dliteratum%253Aachs%26aulast%3DPark%26aufirst%3DH.%26aulast%3DJung%26aufirst%3DH.-Y.%26aulast%3DMah%26aufirst%3DS.%26aulast%3DHong%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520EGF%2520receptor%2520inhibitors%2520that%2520are%2520selective%2520for%2520the%2520d746-750%252FT790M%252FC797S%2520mutant%2520through%2520structure-based%2520de%2520novo%2520design%26jtitle%3DAngew.%2520Chem.%26date%3D2017%26volume%3D129%26spage%3D7742%26epage%3D7746%26doi%3D10.1002%2Fange.201703389" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yun, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, K.</span></span> <span> </span><span class="NLM_article-title">Structure-based design of 5-methylpyrimidopyridone derivatives as new wild-type sparing inhibitors of the epidermal growth factor receptor triple mutant (EGFR(L858R/T790M/C797S))</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">7302</span>– <span class="NLM_lpage">7308</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00576</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00576" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtlOhurnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=7302-7308&author=J.+Shenauthor=T.+Zhangauthor=S.+J.+Zhuauthor=M.+Sunauthor=L.+Tongauthor=M.+Laiauthor=R.+Zhangauthor=W.+Xuauthor=R.+Wuauthor=J.+Dingauthor=C.+H.+Yunauthor=H.+Xieauthor=X.+Luauthor=K.+Ding&title=Structure-based+design+of+5-methylpyrimidopyridone+derivatives+as+new+wild-type+sparing+inhibitors+of+the+epidermal+growth+factor+receptor+triple+mutant+%28EGFR%28L858R%2FT790M%2FC797S%29%29&doi=10.1021%2Facs.jmedchem.9b00576"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of 5-Methylpyrimidopyridone Derivatives as New Wild-Type Sparing Inhibitors of the Epidermal Growth Factor Receptor Triple Mutant (EGFRL858R/T790M/C797S)</span></div><div class="casAuthors">Shen, Jiayi; Zhang, Tao; Zhu, Su-Jie; Sun, Min; Tong, Linjiang; Lai, Mengzhen; Zhang, Rong; Xu, Wei; Wu, Ruibo; Ding, Jian; Yun, Cai-Hong; Xie, Hua; Lu, Xiaoyun; Ding, Ke</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">7302-7308</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Azaheterocyclyl-substituted pyridopyrimidinones such as I were prepd. as selective inhibitors of the mutant epidermal growth factor receptor (EGFR) EGFRL858R/T790M/C797S with selectivity over the wild-type EGFR for potential use in treating osimertinib-resistant non-small-cell lung cancer.  I exhibited an IC50 of 27.5 nM against the EGFRL858R/T790M/C797S mutant, while being significantly less potent against the wild-type EGFR with an IC50 value of > 1.0 μM.  Cocrystallog. structure detn. and computational investigation were conducted to elucidate its target selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQFRlnoJItB7Vg90H21EOLACvtfcHk0lhnmukJetP6_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtlOhurnI&md5=70ca1144f3b73d0ccd412406b1072a1f</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00576&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00576%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DZhu%26aufirst%3DS.%2BJ.%26aulast%3DSun%26aufirst%3DM.%26aulast%3DTong%26aufirst%3DL.%26aulast%3DLai%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DR.%26aulast%3DXu%26aufirst%3DW.%26aulast%3DWu%26aufirst%3DR.%26aulast%3DDing%26aufirst%3DJ.%26aulast%3DYun%26aufirst%3DC.%2BH.%26aulast%3DXie%26aufirst%3DH.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DDing%26aufirst%3DK.%26atitle%3DStructure-based%2520design%2520of%25205-methylpyrimidopyridone%2520derivatives%2520as%2520new%2520wild-type%2520sparing%2520inhibitors%2520of%2520the%2520epidermal%2520growth%2520factor%2520receptor%2520triple%2520mutant%2520%2528EGFR%2528L858R%252FT790M%252FC797S%2529%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D7302%26epage%3D7308%26doi%3D10.1021%2Facs.jmedchem.9b00576" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grabe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lategahn, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rauh, D.</span></span> <span> </span><span class="NLM_article-title">C797S Resistance: The undruggable EGFR mutation in non-small cell lung cancer?</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">779</span>– <span class="NLM_lpage">782</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.8b00314</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.8b00314" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlGhu7vN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=779-782&author=T.+Grabeauthor=J.+Lategahnauthor=D.+Rauh&title=C797S+Resistance%3A+The+undruggable+EGFR+mutation+in+non-small+cell+lung+cancer%3F&doi=10.1021%2Facsmedchemlett.8b00314"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">C797S Resistance: The Undruggable EGFR Mutation in Non-Small Cell Lung Cancer?</span></div><div class="casAuthors">Grabe, Tobias; Lategahn, Jonas; Rauh, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">779-782</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The first evidence of osimertinib resistance mediated by the epidermal growth factor receptor (EGFR) mutation C797S was reported three years ago.  Since then, no major breakthroughs have been achieved to target the clin. relevant mutant variant that impedes covalent bond formation with irreversible EGFR inhibitors.  Although several biochem. active compds. have been described, only a few inhibitors that potently act on the cellular level or in vivo have been introduced so far.  Herein, the authors give an overview of current approaches in the field and highlight the challenges that need to be addressed in future research projects to overcome the C797S-mediated drug resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkYihQlaOa1bVg90H21EOLACvtfcHk0lhbheSKyWmpVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlGhu7vN&md5=8bafb477f8dbbc4a74e0fec125cd3455</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.8b00314&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.8b00314%26sid%3Dliteratum%253Aachs%26aulast%3DGrabe%26aufirst%3DT.%26aulast%3DLategahn%26aufirst%3DJ.%26aulast%3DRauh%26aufirst%3DD.%26atitle%3DC797S%2520Resistance%253A%2520The%2520undruggable%2520EGFR%2520mutation%2520in%2520non-small%2520cell%2520lung%2520cancer%253F%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2018%26volume%3D9%26spage%3D779%26epage%3D782%26doi%3D10.1021%2Facsmedchemlett.8b00314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Induced protein degradation: an emerging drug discovery paradigm</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">101</span>– <span class="NLM_lpage">114</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1038%2Fnrd.2016.211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=27885283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=101-114&author=A.+C.+Laiauthor=C.+M.+Crews&title=Induced+protein+degradation%3A+an+emerging+drug+discovery+paradigm&doi=10.1038%2Fnrd.2016.211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Induced protein degradation: an emerging drug discovery paradigm</span></div><div class="casAuthors">Lai, Ashton C.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">101-114</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small-mol. drug discovery has traditionally focused on occupancy of a binding site that directly affects protein function, and this approach typically precludes targeting proteins that lack such amenable sites.  Furthermore, high systemic drug exposures may be needed to maintain sufficient target inhibition in vivo, increasing the risk of undesirable off-target effects.  Induced protein degrdn. is an alternative approach that is event-driven: upon drug binding, the target protein is tagged for elimination.  Emerging technologies based on proteolysis-targeting chimaeras (PROTACs) that exploit cellular quality control machinery to selectively degrade target proteins are attracting considerable attention in the pharmaceutical industry owing to the advantages they could offer over traditional small-mol. strategies.  These advantages include the potential to reduce systemic drug exposure, the ability to counteract increased target protein expression that often accompanies inhibition of protein function and the potential ability to target proteins that are not currently therapeutically tractable, such as transcription factors, scaffolding and regulatory proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj-O7zXGOpXLVg90H21EOLACvtfcHk0lhbheSKyWmpVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP&md5=a01ff43e9b696fe4dc03f0815158a98f</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.211%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DInduced%2520protein%2520degradation%253A%2520an%2520emerging%2520drug%2520discovery%2520paradigm%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D101%26epage%3D114%26doi%3D10.1038%2Fnrd.2016.211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Toure, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Small-molecule PROTACS: new approaches to protein degradation</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">1966</span>– <span class="NLM_lpage">1973</span>, <span class="refDoi"> DOI: 10.1002/anie.201507978</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1002%2Fanie.201507978" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC28XnsF2ltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=1966-1973&author=M.+Toureauthor=C.+M.+Crews&title=Small-molecule+PROTACS%3A+new+approaches+to+protein+degradation&doi=10.1002%2Fanie.201507978"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule PROTACS: New Approaches to Protein Degradation</span></div><div class="casAuthors">Toure, Momar; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1966-1973</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The current inhibitor-based approach to therapeutics has inherent limitations owing to its occupancy-based model: there is a need to maintain high systemic exposure to ensure sufficient in vivo inhibition, high in vivo concns. bring potential for off-target side effects, and there is a need to bind to an active site, thus limiting the drug target space.  As an alternative, induced protein degrdn. lacks these limitations.  Based on an event-driven model, this approach offers a novel catalytic mechanism to irreversibly inhibit protein function by targeting protein destruction through recruitment to the cellular quality control machinery.  Prior protein degrading strategies have lacked therapeutic potential.  However, recent reports of small-mol.-based proteolysis-targeting chimeras (PROTACs) have demonstrated that this technol. can effectively decrease the cellular levels of several protein classes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdaqlIjCSsgrVg90H21EOLACvtfcHk0lhbheSKyWmpVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnsF2ltA%253D%253D&md5=47838cd15719757317cb8282c26cc59e</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1002%2Fanie.201507978&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201507978%26sid%3Dliteratum%253Aachs%26aulast%3DToure%26aufirst%3DM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DSmall-molecule%2520PROTACS%253A%2520new%2520approaches%2520to%2520protein%2520degradation%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26spage%3D1966%26epage%3D1973%26doi%3D10.1002%2Fanie.201507978" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Small-molecule modulation of protein homeostasis</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>117</i></span>,  <span class="NLM_fpage">11269</span>– <span class="NLM_lpage">11301</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.7b00077</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.7b00077" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1Ohur3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2017&pages=11269-11301&author=G.+M.+Burslemauthor=C.+M.+Crews&title=Small-molecule+modulation+of+protein+homeostasis&doi=10.1021%2Facs.chemrev.7b00077"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Modulation of Protein Homeostasis</span></div><div class="casAuthors">Burslem, George M.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">11269-11301</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Control of protein levels by nucleic-acid-based technologies has proven to be a useful research tool but lacks the advantages of small mols. with respect to cell permeability, temporal control, and the potential generation of therapeutics.  In this Review, we discuss the technologies available for the control of intracellular protein levels with small mols. and compare the various systems available.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcARydbgPK9bVg90H21EOLACvtfcHk0lgmz0a5-Pe8rA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1Ohur3K&md5=f9f8dd98e2c71c7166d2d500b514d578</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.7b00077&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.7b00077%26sid%3Dliteratum%253Aachs%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DSmall-molecule%2520modulation%2520of%2520protein%2520homeostasis%26jtitle%3DChem.%2520Rev.%26date%3D2017%26volume%3D117%26spage%3D11269%26epage%3D11301%26doi%3D10.1021%2Facs.chemrev.7b00077" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumagai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercurio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshaies, R. J.</span></span> <span> </span><span class="NLM_article-title">Protacs: chimeric molecules that target proteins to the Skp1-Cullin-F box complex for ubiquitination and degradation</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">8554</span>– <span class="NLM_lpage">8559</span>, <span class="refDoi"> DOI: 10.1073/pnas.141230798</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1073%2Fpnas.141230798" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=11438690" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BD3MXls1Wisbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2001&pages=8554-8559&author=K.+M.+Sakamotoauthor=K.+B.+Kimauthor=A.+Kumagaiauthor=F.+Mercurioauthor=C.+M.+Crewsauthor=R.+J.+Deshaies&title=Protacs%3A+chimeric+molecules+that+target+proteins+to+the+Skp1-Cullin-F+box+complex+for+ubiquitination+and+degradation&doi=10.1073%2Fpnas.141230798"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Protacs: chimeric molecules that target proteins to the Skp1-Cullin-F box complex for ubiquitination and degradation</span></div><div class="casAuthors">Sakamoto, Kathleen M.; Kim, Kyung B.; Kumagai, Akiko; Mercurio, Frank; Crews, Craig M.; Deshaies, Raymond J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">8554-8559</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The intracellular levels of many proteins are regulated by ubiquitin-dependent proteolysis.  One of the best-characterized enzymes that catalyzes the attachment of ubiquitin to proteins is a ubiquitin ligase complex, Skp1-Cullin-F box complex contg. Hrt1 (SCF).  We sought to artificially target a protein to the SCF complex for ubiquitination and degrdn.  To this end, we tested methionine aminopeptidase-2 (MetAP-2), which covalently binds the angiogenesis inhibitor ovalicin.  A chimeric compd., protein-targeting chimeric mol. 1 (Protac-1), was synthesized to recruit MetAP-2 to SCF.  One domain of Protac-1 contains the IκBα phosphopeptide that is recognized by the F-box protein β-TRCP, whereas the other domain is composed of ovalicin.  We show that MetAP-2 can be tethered to SCFβ-TRCP, ubiquitinated, and degraded in a Protac-1-dependent manner.  In the future, this approach may be useful for conditional inactivation of proteins, and for targeting disease-causing proteins for destruction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp49fYtuEVlJrVg90H21EOLACvtfcHk0lgmz0a5-Pe8rA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXls1Wisbk%253D&md5=540358035222f745f7b6367b38781a21</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1073%2Fpnas.141230798&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.141230798%26sid%3Dliteratum%253Aachs%26aulast%3DSakamoto%26aufirst%3DK.%2BM.%26aulast%3DKim%26aufirst%3DK.%2BB.%26aulast%3DKumagai%26aufirst%3DA.%26aulast%3DMercurio%26aufirst%3DF.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DDeshaies%26aufirst%3DR.%2BJ.%26atitle%3DProtacs%253A%2520chimeric%2520molecules%2520that%2520target%2520proteins%2520to%2520the%2520Skp1-Cullin-F%2520box%2520complex%2520for%2520ubiquitination%2520and%2520degradation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2001%26volume%3D98%26spage%3D8554%26epage%3D8559%26doi%3D10.1073%2Fpnas.141230798" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Salami, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Waste disposal-An attractive strategy for cancer therapy</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>355</i></span>,  <span class="NLM_fpage">1163</span>– <span class="NLM_lpage">1167</span>, <span class="refDoi"> DOI: 10.1126/science.aam7340</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1126%2Fscience.aam7340" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=28302825" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC2sXktFOmt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=355&publication_year=2017&pages=1163-1167&author=J.+Salamiauthor=C.+M.+Crews&title=Waste+disposal-An+attractive+strategy+for+cancer+therapy&doi=10.1126%2Fscience.aam7340"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Waste disposal-An attractive strategy for cancer therapy</span></div><div class="casAuthors">Salami, Jemilat; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">355</span>
        (<span class="NLM_cas:issue">6330</span>),
    <span class="NLM_cas:pages">1163-1167</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review.  Targeted therapies for cancer are typically small mols. or monoclonal antibodies that act by inhibiting the activity of specific proteins that drive tumor growth.  Although many of these drugs are effective in cancer patients, the response is often not durable because tumor cells develop resistance to the drugs.  Another limitation of this strategy is that not all oncogenic driver proteins are "druggable" enzymes or receptors with activities that can be inhibited.  Here the authors describe an alternative approach to targeted therapy that is based on co-opting the cellular quality-control machinery-the ubiquitin-proteasome system-to remove specific cancer-causing proteins from the cell.  The authors first discuss examples of existing cancer drugs that work by degrading specific proteins and then review recent progress in the rational design and preclin. testing of small mols. that induce selective degrdn. of specific target proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVoZn-YpfXmbVg90H21EOLACvtfcHk0lg9qQdiptI9Pw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXktFOmt74%253D&md5=918621d50a82b7797aa0300537a4486e</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1126%2Fscience.aam7340&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aam7340%26sid%3Dliteratum%253Aachs%26aulast%3DSalami%26aufirst%3DJ.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DWaste%2520disposal-An%2520attractive%2520strategy%2520for%2520cancer%2520therapy%26jtitle%3DScience%26date%3D2017%26volume%3D355%26spage%3D1163%26epage%3D1167%26doi%3D10.1126%2Fscience.aam7340" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">DRUG DEVELOPMENT. Phthalimide conjugation as a strategy for in vivo target protein degradation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>348</i></span>,  <span class="NLM_fpage">1376</span>– <span class="NLM_lpage">1381</span>, <span class="refDoi"> DOI: 10.1126/science.aab1433</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1126%2Fscience.aab1433" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=25999370" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC2MXpvFansLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=348&publication_year=2015&pages=1376-1381&author=G.+E.+Winterauthor=D.+L.+Buckleyauthor=J.+Paulkauthor=J.+M.+Robertsauthor=A.+Souzaauthor=S.+Dhe-Paganonauthor=J.+E.+Bradner&title=DRUG+DEVELOPMENT.+Phthalimide+conjugation+as+a+strategy+for+in+vivo+target+protein+degradation&doi=10.1126%2Fscience.aab1433"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Phthalimide conjugation as a strategy for in vivo target protein degradation</span></div><div class="casAuthors">Winter, Georg E.; Buckley, Dennis L.; Paulk, Joshiawa; Roberts, Justin M.; Souza, Amanda; Dhe-Paganon, Sirano; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">348</span>
        (<span class="NLM_cas:issue">6241</span>),
    <span class="NLM_cas:pages">1376-1381</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The development of effective pharmacol. inhibitors of multidomain scaffold proteins, notably transcription factors, is a particularly challenging problem.  In part, this is because many small-mol. antagonists disrupt the activity of only one domain in the target protein.  The authors devised a chem. strategy that promotes ligand-dependent target protein degrdn. using as an example the transcriptional coactivator BRD4, a protein crit. for cancer cell growth and survival.  The authors appended a competitive antagonist of BET bromodomains to a phthalimide moiety to hijack the cereblon E3 ubiquitin ligase complex.  The resultant compd., dBET1, induced highly selective cereblon-dependent BET protein degrdn. in vitro and in vivo and delayed leukemia progression in mice.  A second series of probes resulted in selective degrdn. of the cytosolic protein FKBP12.  This chem. strategy for controlling target protein stability may have implications for therapeutically targeting previously intractable proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1ZW4zjRjXi7Vg90H21EOLACvtfcHk0lg9qQdiptI9Pw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXpvFansLk%253D&md5=b0277cc95d318d926b96e7a119066e3b</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1126%2Fscience.aab1433&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aab1433%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DG.%2BE.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DSouza%26aufirst%3DA.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DDRUG%2520DEVELOPMENT.%2520Phthalimide%2520conjugation%2520as%2520a%2520strategy%2520for%2520in%2520vivo%2520target%2520protein%2520degradation%26jtitle%3DScience%26date%3D2015%26volume%3D348%26spage%3D1376%26epage%3D1381%26doi%3D10.1126%2Fscience.aab1433" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mares, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, I. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campos, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miah, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulholland, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Routly, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafson, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zinn, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimamura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergamini, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faelth-Savitski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willard, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flanagan, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casillas, L. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Votta, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">den Besten, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Famm, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruidenier, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harling, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Churcher, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Catalytic in vivo protein knockdown by small-molecule PROTACs</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">611</span>– <span class="NLM_lpage">617</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1858</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1038%2Fnchembio.1858" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=26075522" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=611-617&author=D.+P.+Bondesonauthor=A.+Maresauthor=I.+E.+Smithauthor=E.+Koauthor=S.+Camposauthor=A.+H.+Miahauthor=K.+E.+Mulhollandauthor=N.+Routlyauthor=D.+L.+Buckleyauthor=J.+L.+Gustafsonauthor=N.+Zinnauthor=P.+Grandiauthor=S.+Shimamuraauthor=G.+Bergaminiauthor=M.+Faelth-Savitskiauthor=M.+Bantscheffauthor=C.+Coxauthor=D.+A.+Gordonauthor=R.+R.+Willardauthor=J.+J.+Flanaganauthor=L.+N.+Casillasauthor=B.+J.+Vottaauthor=W.+den+Bestenauthor=K.+Fammauthor=L.+Kruidenierauthor=P.+S.+Carterauthor=J.+D.+Harlingauthor=I.+Churcherauthor=C.+M.+Crews&title=Catalytic+in+vivo+protein+knockdown+by+small-molecule+PROTACs&doi=10.1038%2Fnchembio.1858"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Catalytic in vivo protein knockdown by small-molecule PROTACs</span></div><div class="casAuthors">Bondeson, Daniel P.; Mares, Alina; Smith, Ian E. D.; Ko, Eunhwa; Campos, Sebastien; Miah, Afjal H.; Mulholland, Katie E.; Routly, Natasha; Buckley, Dennis L.; Gustafson, Jeffrey L.; Zinn, Nico; Grandi, Paola; Shimamura, Satoko; Bergamini, Giovanna; Faelth-Savitski, Maria; Bantscheff, Marcus; Cox, Carly; Gordon, Deborah A.; Willard, Ryan R.; Flanagan, John J.; Casillas, Linda N.; Votta, Bartholomew J.; den Besten, Willem; Famm, Kristoffer; Kruidenier, Laurens; Carter, Paul S.; Harling, John D.; Churcher, Ian; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">611-617</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The current predominant therapeutic paradigm is based on maximizing drug-receptor occupancy to achieve clin. benefit.  This strategy, however, generally requires excessive drug concns. to ensure sufficient occupancy, often leading to adverse side effects.  Here, we describe major improvements to the proteolysis targeting chimeras (PROTACs) method, a chem. knockdown strategy in which a heterobifunctional mol. recruits a specific protein target to an E3 ubiquitin ligase, resulting in the target's ubiquitination and degrdn.  These compds. behave catalytically in their ability to induce the ubiquitination of super-stoichiometric quantities of proteins, providing efficacy that is not limited by equil. occupancy.  We present two PROTACs that are capable of specifically reducing protein levels by >90% at nanomolar concns.  In addn., mouse studies indicate that they provide broad tissue distribution and knockdown of the targeted protein in tumor xenografts.  Together, these data demonstrate a protein knockdown system combining many of the favorable properties of small-mol. agents with the potent protein knockdown of RNAi and CRISPR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonhnajD3CxS7Vg90H21EOLACvtfcHk0ljGEsS6llvD4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7rP&md5=31eb8756c5661f78ab4527a50dc1e251</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1858&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1858%26sid%3Dliteratum%253Aachs%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DMares%26aufirst%3DA.%26aulast%3DSmith%26aufirst%3DI.%2BE.%26aulast%3DKo%26aufirst%3DE.%26aulast%3DCampos%26aufirst%3DS.%26aulast%3DMiah%26aufirst%3DA.%2BH.%26aulast%3DMulholland%26aufirst%3DK.%2BE.%26aulast%3DRoutly%26aufirst%3DN.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DGustafson%26aufirst%3DJ.%2BL.%26aulast%3DZinn%26aufirst%3DN.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DShimamura%26aufirst%3DS.%26aulast%3DBergamini%26aufirst%3DG.%26aulast%3DFaelth-Savitski%26aufirst%3DM.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DCox%26aufirst%3DC.%26aulast%3DGordon%26aufirst%3DD.%2BA.%26aulast%3DWillard%26aufirst%3DR.%2BR.%26aulast%3DFlanagan%26aufirst%3DJ.%2BJ.%26aulast%3DCasillas%26aufirst%3DL.%2BN.%26aulast%3DVotta%26aufirst%3DB.%2BJ.%26aulast%3Dden%2BBesten%26aufirst%3DW.%26aulast%3DFamm%26aufirst%3DK.%26aulast%3DKruidenier%26aufirst%3DL.%26aulast%3DCarter%26aufirst%3DP.%2BS.%26aulast%3DHarling%26aufirst%3DJ.%2BD.%26aulast%3DChurcher%26aufirst%3DI.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DCatalytic%2520in%2520vivo%2520protein%2520knockdown%2520by%2520small-molecule%2520PROTACs%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D611%26epage%3D617%26doi%3D10.1038%2Fnchembio.1858" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Noble, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Endicott, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, L. N.</span></span> <span> </span><span class="NLM_article-title">Protein kinase inhibitors: insights into drug design from structure</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>303</i></span>,  <span class="NLM_fpage">1800</span>– <span class="NLM_lpage">1805</span>, <span class="refDoi"> DOI: 10.1126/science.1095920</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1126%2Fscience.1095920" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=15031492" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BD2cXitFehurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=303&publication_year=2004&pages=1800-1805&author=M.+E.+Nobleauthor=J.+A.+Endicottauthor=L.+N.+Johnson&title=Protein+kinase+inhibitors%3A+insights+into+drug+design+from+structure&doi=10.1126%2Fscience.1095920"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Kinase Inhibitors: Insights into Drug Design from Structure</span></div><div class="casAuthors">Noble, Martin E. M.; Endicott, Jane A.; Johnson, Louise N.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">303</span>
        (<span class="NLM_cas:issue">5665</span>),
    <span class="NLM_cas:pages">1800-1805</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review.  Protein kinases are targets for treatment of a no. of diseases.  This review focuses on kinase inhibitors that are in the clinic or in clin. trials and for which structural information is available.  Structures have informed drug design and have illuminated the mechanism of inhibition.  We review progress with the receptor tyrosine kinases (growth factor receptors EGFR, VEGFR, and FGFR) and nonreceptor tyrosine kinases (Bcr-Abl), where advances have been made with cancer therapeutic agents such as Herceptin and Gleevec.  Among the serine-threonine kinases, p38, Rho-kinase, cyclin-dependent kinases, and Chk1 have been targeted with productive results for inflammation and cancer.  Structures have provided insights into targeting the inactive or active form of the kinase, for targeting the global constellation of residues at the ATP site or less conserved addnl. pockets or single residues, and into targeting noncatalytic domains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWqtKz7cvN_rVg90H21EOLACvtfcHk0lh-LrLttTbjwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXitFehurc%253D&md5=7fd3f60bee46e59307436eeed285d6ae</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1126%2Fscience.1095920&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1095920%26sid%3Dliteratum%253Aachs%26aulast%3DNoble%26aufirst%3DM.%2BE.%26aulast%3DEndicott%26aufirst%3DJ.%2BA.%26aulast%3DJohnson%26aufirst%3DL.%2BN.%26atitle%3DProtein%2520kinase%2520inhibitors%253A%2520insights%2520into%2520drug%2520design%2520from%2520structure%26jtitle%3DScience%26date%3D2004%26volume%3D303%26spage%3D1800%26epage%3D1805%26doi%3D10.1126%2Fscience.1095920" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buhimschi, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaime-Figueroa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamman, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishchenko, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Lessons in PROTAC design from selective degradation with a promiscuous warhead</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">78</span>– <span class="NLM_lpage">87</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.09.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1016%2Fj.chembiol.2017.09.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=29129718" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=78-87&author=D.+P.+Bondesonauthor=B.+E.+Smithauthor=G.+M.+Burslemauthor=A.+D.+Buhimschiauthor=J.+Hinesauthor=S.+Jaime-Figueroaauthor=J.+Wangauthor=B.+D.+Hammanauthor=A.+Ishchenkoauthor=C.+M.+Crews&title=Lessons+in+PROTAC+design+from+selective+degradation+with+a+promiscuous+warhead&doi=10.1016%2Fj.chembiol.2017.09.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Lessons in PROTAC Design from Selective Degradation with a Promiscuous Warhead</span></div><div class="casAuthors">Bondeson, Daniel P.; Smith, Blake E.; Burslem, George M.; Buhimschi, Alexandru D.; Hines, John; Jaime-Figueroa, Saul; Wang, Jing; Hamman, Brian D.; Ishchenko, Alexey; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">78-87.e5</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Inhibiting protein function selectively is a major goal of modern drug discovery.  Here, we report a previously understudied benefit of small mol. proteolysis-targeting chimeras (PROTACs) that recruit E3 ubiquitin ligases to target proteins for their ubiquitination and subsequent proteasome-mediated degrdn.  Using promiscuous CRBN- and VHL-recruiting PROTACs that bind >50 kinases, we show that only a subset of bound targets is degraded.  The basis of this selectivity relies on protein-protein interactions between the E3 ubiquitin ligase and the target protein, as illustrated by engaged proteins that are not degraded as a result of unstable ternary complexes with PROTAC-recruited E3 ligases.  In contrast, weak PROTAC:target protein affinity can be stabilized by high-affinity target:PROTAC:ligase trimer interactions, leading to efficient degrdn.  This study highlights design guidelines for generating potent PROTACs as well as possibilities for degrading undruggable proteins immune to traditional small-mol. inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoISOBudOLyFrVg90H21EOLACvtfcHk0lh-LrLttTbjwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjN&md5=30fac502705a2344c4261cf531e6bf9d</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.09.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.09.010%26sid%3Dliteratum%253Aachs%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DSmith%26aufirst%3DB.%2BE.%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DBuhimschi%26aufirst%3DA.%2BD.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DHamman%26aufirst%3DB.%2BD.%26aulast%3DIshchenko%26aufirst%3DA.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DLessons%2520in%2520PROTAC%2520design%2520from%2520selective%2520degradation%2520with%2520a%2520promiscuous%2520warhead%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D78%26epage%3D87%26doi%3D10.1016%2Fj.chembiol.2017.09.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ottis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Proteolysis-targeting chimeras: induced protein degradation as a therapeutic strategy</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">892</span>– <span class="NLM_lpage">898</span>, <span class="refDoi"> DOI: 10.1021/acschembio.6b01068</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.6b01068" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjs12ns70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=892-898&author=P.+Ottisauthor=C.+M.+Crews&title=Proteolysis-targeting+chimeras%3A+induced+protein+degradation+as+a+therapeutic+strategy&doi=10.1021%2Facschembio.6b01068"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis-Targeting Chimeras: Induced Protein Degradation as a Therapeutic Strategy</span></div><div class="casAuthors">Ottis, Philipp; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">892-898</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Until recently, the only ways to reduce specific protein signaling were to either knock down the target by RNAi or to interfere with the signaling by inhibiting an enzyme or receptor within the signal transduction cascade.  Herein, we review an emerging class of small mol. pharmacol. agents, called PROTACs, that present a novel approach to specifically target proteins and their resp. signaling pathways.  These heterobifunctional mols. utilize endogenous cellular quality control machinery by recruiting it to target proteins in order to induce their degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohB6xwkQMTprVg90H21EOLACvtfcHk0lh5WvUlZCHtcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjs12ns70%253D&md5=eada521c10faa1e456e0885fd390c554</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Facschembio.6b01068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.6b01068%26sid%3Dliteratum%253Aachs%26aulast%3DOttis%26aufirst%3DP.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DProteolysis-targeting%2520chimeras%253A%2520induced%2520protein%2520degradation%2520as%2520a%2520therapeutic%2520strategy%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26spage%3D892%26epage%3D898%26doi%3D10.1021%2Facschembio.6b01068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cromm, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Targeted protein degradation: from chemical biology to drug discovery</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">1181</span>– <span class="NLM_lpage">1190</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.05.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1016%2Fj.chembiol.2017.05.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=28648379" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVOjtrrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2017&pages=1181-1190&author=P.+M.+Crommauthor=C.+M.+Crews&title=Targeted+protein+degradation%3A+from+chemical+biology+to+drug+discovery&doi=10.1016%2Fj.chembiol.2017.05.024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted Protein Degradation: from Chemical Biology to Drug Discovery</span></div><div class="casAuthors">Cromm, Philipp M.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1181-1190</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Traditional pharmaceutical drug discovery is almost exclusively focused on directly controlling protein activity to cure diseases.  Modulators of protein activity, esp. inhibitors, are developed and applied at high concn. to achieve maximal effects.  Thereby, reduced bioavailability and off-target effects can hamper compd. efficacy.  Nucleic acid-based strategies that control protein function by affecting expression have emerged as an alternative.  However, metabolic stability and broad bioavailability represent development hurdles that remain to be overcome for these approaches.  More recently, utilizing the cell's own protein destruction machinery for selective degrdn. of essential drivers of human disorders has opened up a new and exciting area of drug discovery.  Small-mol.-induced proteolysis of selected substrates offers the potential of reaching beyond the limitations of the current pharmaceutical paradigm to expand the druggable target space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrosuhmnPE0RrVg90H21EOLACvtfcHk0li7O7nJbcEDSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVOjtrrO&md5=3c8c6d9488396d5eb08ca51cee20bbf4</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.05.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.05.024%26sid%3Dliteratum%253Aachs%26aulast%3DCromm%26aufirst%3DP.%2BM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DTargeted%2520protein%2520degradation%253A%2520from%2520chemical%2520biology%2520to%2520drug%2520discovery%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2017%26volume%3D24%26spage%3D1181%26epage%3D1190%26doi%3D10.1016%2Fj.chembiol.2017.05.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Kinase inhibitors: the road ahead</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">353</span>– <span class="NLM_lpage">377</span>, <span class="refDoi"> DOI: 10.1038/nrd.2018.21</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1038%2Fnrd.2018.21" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=29545548" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC1cXks12ktbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=353-377&author=F.+M.+Fergusonauthor=N.+S.+Gray&title=Kinase+inhibitors%3A+the+road+ahead&doi=10.1038%2Fnrd.2018.21"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Kinase inhibitors: the road ahead</span></div><div class="casAuthors">Ferguson, Fleur M.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">353-377</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  Receptor tyrosine kinase signaling pathways have been successfully targeted to inhibit proliferation and angiogenesis for cancer therapy.  However, kinase deregulation has been firmly demonstrated to play an essential role in virtually all major disease areas.  Kinase inhibitor drug discovery programs have recently broadened their focus to include an expanded range of kinase targets and therapeutic areas.  In this Review, we provide an overview of the novel targets, biol. processes and disease areas that kinase-targeting small mols. are being developed against, highlight the assocd. challenges and assess the strategies and technologies that are enabling efficient generation of highly optimized kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEZPospb1TP7Vg90H21EOLACvtfcHk0lhUx9uT1dQ68w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXks12ktbg%253D&md5=34e42911d8a70301905460c3507e737a</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2018.21&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2018.21%26sid%3Dliteratum%253Aachs%26aulast%3DFerguson%26aufirst%3DF.%2BM.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DKinase%2520inhibitors%253A%2520the%2520road%2520ahead%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2018%26volume%3D17%26spage%3D353%26epage%3D377%26doi%3D10.1038%2Fnrd.2018.21" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pettersson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">PROteolysis TArgeting Chimeras (PROTACs)- past, present and future</span>. <i>Drug Discovery Today Technol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">15</span>– <span class="NLM_lpage">27</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2019.01.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1016%2Fj.ddtec.2019.01.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=31200855" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A280%3ADC%252BB3M3mvFWkuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2019&pages=15-27&author=M.+Petterssonauthor=C.+M.+Crews&title=PROteolysis+TArgeting+Chimeras+%28PROTACs%29-+past%2C+present+and+future&doi=10.1016%2Fj.ddtec.2019.01.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">PROteolysis TArgeting Chimeras (PROTACs) - Past, present and future</span></div><div class="casAuthors">Pettersson Mariell; Crews Craig M</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today. Technologies</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">15-27</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The majority of currently used therapeutics are small molecule-based and utilize occupancy-driven pharmacology as the mode of action (MOA), in which the protein function is modulated via temporary inhibition.  New modalities that operate using alternative MOAs are essential for tapping into the "undruggable" proteome.  The PROteolysis Targeting Chimera (PROTAC) technology provides an attractive new approach that utilizes an event-driven MOA.  Small molecule-based heterobifunctional PROTACs modulate protein target levels by hijacking the ubiquitin-proteasome system to induce degradation of the target.  Here, we address important milestones in the development of the PROTAC technology, as well as emphasize key findings from this previous year and highlight future directions of this promising drug discovery modality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQCRHissLLXoRfKb6ELK8t0fW6udTcc2eZ5QAYbrIQyUbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3M3mvFWkuw%253D%253D&md5=c629f3128321e136f1f247e05ee6f182</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2019.01.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2019.01.002%26sid%3Dliteratum%253Aachs%26aulast%3DPettersson%26aufirst%3DM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DPROteolysis%2520TArgeting%2520Chimeras%2520%2528PROTACs%2529-%2520past%252C%2520present%2520and%2520future%26jtitle%3DDrug%2520Discovery%2520Today%2520Technol.%26date%3D2019%26volume%3D31%26spage%3D15%26epage%3D27%26doi%3D10.1016%2Fj.ddtec.2019.01.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Olson, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erb, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doctor, Z. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwiatkowski, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boukhali, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haas, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nomanbhoy, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Pharmacological perturbation of CDK9 using selective CDK9 inhibition or degradation</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">163</span>– <span class="NLM_lpage">170</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2538</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1038%2Fnchembio.2538" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=29251720" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFyrsr7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2018&pages=163-170&author=C.+M.+Olsonauthor=B.+Jiangauthor=M.+A.+Erbauthor=Y.+Liangauthor=Z.+M.+Doctorauthor=Z.+Zhangauthor=T.+Zhangauthor=N.+Kwiatkowskiauthor=M.+Boukhaliauthor=J.+L.+Greenauthor=W.+Haasauthor=T.+Nomanbhoyauthor=E.+S.+Fischerauthor=R.+A.+Youngauthor=J.+E.+Bradnerauthor=G.+E.+Winterauthor=N.+S.+Gray&title=Pharmacological+perturbation+of+CDK9+using+selective+CDK9+inhibition+or+degradation&doi=10.1038%2Fnchembio.2538"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological perturbation of CDK9 using selective CDK9 inhibition or degradation</span></div><div class="casAuthors">Olson, Calla M.; Jiang, Baishan; Erb, Michael A.; Liang, Yanke; Doctor, Zainab M.; Zhang, Zinan; Zhang, Tinghu; Kwiatkowski, Nicholas; Boukhali, Myriam; Green, Jennifer L.; Haas, Wilhelm; Nomanbhoy, Tyzoon; Fischer, Eric S.; Young, Richard A.; Bradner, James E.; Winter, Georg E.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">163-170</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Cyclin-dependent kinase 9 (CDK9), an important regulator of transcriptional elongation, is a promising target for cancer therapy, particularly for cancers driven by transcriptional dysregulation.  The authors characterized NVP-2, a selective ATP-competitive CDK9 inhibitor, and THAL-SNS-032, a selective CDK9 degrader consisting of a CDK-binding SNS-032 ligand linked to a thalidomide deriv. that binds the E3 ubiquitin ligase Cereblon (CRBN).  To the authors' surprise, THAL-SNS-032 induced rapid degrdn. of CDK9 without affecting the levels of other SNS-032 targets.  Moreover, the transcriptional changes elicited by THAL-SNS-032 were more like those caused by NVP-2 than those induced by SNS-032.  Notably, compd. washout did not significantly reduce levels of THAL-SNS-032-induced apoptosis, suggesting that CDK9 degrdn. had prolonged cytotoxic effects compared with CDK9 inhibition.  Thus, the authors' findings suggest that thalidomide conjugation represents a promising strategy for converting multi-targeted inhibitors into selective degraders and reveal that kinase degrdn. can induce distinct pharmacol. effects compared with inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWnT9r0GIlmbVg90H21EOLACvtfcHk0ljFqRTiyOSv-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFyrsr7O&md5=88aabb209b4cb7516a652d6c75a35667</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2538&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2538%26sid%3Dliteratum%253Aachs%26aulast%3DOlson%26aufirst%3DC.%2BM.%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DErb%26aufirst%3DM.%2BA.%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DDoctor%26aufirst%3DZ.%2BM.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DKwiatkowski%26aufirst%3DN.%26aulast%3DBoukhali%26aufirst%3DM.%26aulast%3DGreen%26aufirst%3DJ.%2BL.%26aulast%3DHaas%26aufirst%3DW.%26aulast%3DNomanbhoy%26aufirst%3DT.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DWinter%26aufirst%3DG.%2BE.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DPharmacological%2520perturbation%2520of%2520CDK9%2520using%2520selective%2520CDK9%2520inhibition%2520or%2520degradation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2018%26volume%3D14%26spage%3D163%26epage%3D170%26doi%3D10.1038%2Fnchembio.2538" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgess, K.</span></span> <span> </span><span class="NLM_article-title">PROTACs suppression of CDK4/6, crucial kinases for cell cycle regulation in cancer</span>. <i>Chem. Commun</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">2704</span>– <span class="NLM_lpage">2707</span>, <span class="refDoi"> DOI: 10.1039/C9CC00163H</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1039%2FC9CC00163H" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=30758029" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjtFGht7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2019&pages=2704-2707&author=B.+Zhaoauthor=K.+Burgess&title=PROTACs+suppression+of+CDK4%2F6%2C+crucial+kinases+for+cell+cycle+regulation+in+cancer&doi=10.1039%2FC9CC00163H"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">PROTACs suppression of CDK4/6, crucial kinases for cell cycle regulation in cancer</span></div><div class="casAuthors">Zhao, Bosheng; Burgess, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">2704-2707</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">PROTACs based on two selective, FDA approved, CDK4/6 inhibitors were formed.  One of them, based on palbociclib, potently initiates degrdn. of these CDK proteins, and suppresses phosphorylation of retinoblastoma protein (Rb) leading to cell cycle arrest.  These PROTACs are active at nanomolar concns., and appear to be the first for CDK4/6.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBWDRf65Fl6bVg90H21EOLACvtfcHk0lhEZ4Bc_qeLtg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjtFGht7c%253D&md5=8798e9aa7ecc0e6504c2717e815b04fc</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1039%2FC9CC00163H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC9CC00163H%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DB.%26aulast%3DBurgess%26aufirst%3DK.%26atitle%3DPROTACs%2520suppression%2520of%2520CDK4%252F6%252C%2520crucial%2520kinases%2520for%2520cell%2520cycle%2520regulation%2520in%2520cancer%26jtitle%3DChem.%2520Commun%26date%3D2019%26volume%3D55%26spage%3D2704%26epage%3D2707%26doi%3D10.1039%2FC9CC00163H" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Development of dual and selective degraders of cyclin-dependent kinases 4 and 6</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">6321</span>– <span class="NLM_lpage">6326</span>, <span class="refDoi"> DOI: 10.1002/anie.201901336</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1002%2Fanie.201901336" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC1MXmtlKqsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2019&pages=6321-6326&author=B.+Jiangauthor=E.+S.+Wangauthor=K.+A.+Donovanauthor=Y.+Liangauthor=E.+S.+Fischerauthor=T.+Zhangauthor=N.+S.+Gray&title=Development+of+dual+and+selective+degraders+of+cyclin-dependent+kinases+4+and+6&doi=10.1002%2Fanie.201901336"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Development of Dual and Selective Degraders of Cyclin-Dependent Kinases 4 and 6</span></div><div class="casAuthors">Jiang, Baishan; Wang, Eric S.; Donovan, Katherine A.; Liang, Yanke; Fischer, Eric S.; Zhang, Tinghu; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6321-6326</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Cyclin-dependent kinases 4 and 6 (CDK4/6) are key regulators of the cell cycle, and there are FDA-approved CDK4/6 inhibitors for treating patients with metastatic breast cancer.  However, due to conservation of their ATP-binding sites, development of selective agents has remained elusive.  Here, we report imide-based degrader mols. capable of degrading both CDK4/6, or selectively degrading either CDK4 or CDK6.  We were also able to tune the activity of these mols. against Ikaros (IKZF1) and Aiolos (IKZF3), which are well-established targets of imide-based degraders.  We found that in mantle cell lymphoma cell lines, combined IKZF1/3 degrdn. with dual CDK4/6 degrdn. produced enhanced anti-proliferative effects compared to CDK4/6 inhibition, CDK4/6 degrdn., or IKZF1/3 degrdn.  In summary, we report here the first compds. capable of inducing selective degrdn. of CDK4 and CDK6 as tools to pharmacol. dissect their distinct biol. functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_Vsqs-SM6hrVg90H21EOLACvtfcHk0lhEZ4Bc_qeLtg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXmtlKqsrw%253D&md5=39a6937b24abad0940d359020e2b2f74</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1002%2Fanie.201901336&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201901336%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DWang%26aufirst%3DE.%2BS.%26aulast%3DDonovan%26aufirst%3DK.%2BA.%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DDevelopment%2520of%2520dual%2520and%2520selective%2520degraders%2520of%2520cyclin-dependent%2520kinases%25204%2520and%25206%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2019%26volume%3D58%26spage%3D6321%26epage%3D6326%26doi%3D10.1002%2Fanie.201901336" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brand, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwiatkowski, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span> <span> </span><span class="NLM_article-title">Homolog-selective degradation as a strategy to probe the function of CDK6 in AML</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">300</span>– <span class="NLM_lpage">306 e9</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2018.11.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1016%2Fj.chembiol.2018.11.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=30595531" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC1MXnsVSj" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2019&pages=300-306+e9&author=M.+Brandauthor=B.+Jiangauthor=S.+Bauerauthor=K.+A.+Donovanauthor=Y.+Liangauthor=E.+S.+Wangauthor=R.+P.+Nowakauthor=J.+C.+Yuanauthor=T.+Zhangauthor=N.+Kwiatkowskiauthor=A.+C.+Mullerauthor=E.+S.+Fischerauthor=N.+S.+Grayauthor=G.+E.+Winter&title=Homolog-selective+degradation+as+a+strategy+to+probe+the+function+of+CDK6+in+AML&doi=10.1016%2Fj.chembiol.2018.11.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Homolog-Selective Degradation as a Strategy to Probe the Function of CDK6 in AML</span></div><div class="casAuthors">Brand, Matthias; Jiang, Baishan; Bauer, Sophie; Donovan, Katherine A.; Liang, Yanke; Wang, Eric S.; Nowak, Radoslaw P.; Yuan, Jingting C.; Zhang, Tinghu; Kwiatkowski, Nicholas; Mueller, Andre C.; Fischer, Eric S.; Gray, Nathanael S.; Winter, Georg E.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">300-306.e9</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The design of selective small mols. is often stymied by similar ligand binding pockets.  Here, we report BSJ-03-123, a phthalimide-based degrader that exploits protein-interface determinants to achieve proteome-wide selectivity for the degrdn. of cyclin-dependent kinase 6 (CDK6).  Pharmacol. CDK6 degrdn. targets a selective dependency of acute myeloid leukemia cells, and transcriptomics and phosphoproteomics profiling of acute degrdn. of CDK6 enabled dynamic mapping of its immediate role in coordinating signaling and transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGognoz8FFoR9LVg90H21EOLACvtfcHk0lhEZ4Bc_qeLtg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXnsVSj&md5=f8c1c3520c6c0218763d8769c59b4342</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2018.11.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2018.11.006%26sid%3Dliteratum%253Aachs%26aulast%3DBrand%26aufirst%3DM.%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DBauer%26aufirst%3DS.%26aulast%3DDonovan%26aufirst%3DK.%2BA.%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DE.%2BS.%26aulast%3DNowak%26aufirst%3DR.%2BP.%26aulast%3DYuan%26aufirst%3DJ.%2BC.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DKwiatkowski%26aufirst%3DN.%26aulast%3DMuller%26aufirst%3DA.%2BC.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DWinter%26aufirst%3DG.%2BE.%26atitle%3DHomolog-selective%2520degradation%2520as%2520a%2520strategy%2520to%2520probe%2520the%2520function%2520of%2520CDK6%2520in%2520AML%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2019%26volume%3D26%26spage%3D300%26epage%3D306%2520e9%26doi%3D10.1016%2Fj.chembiol.2018.11.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cromm, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samarasinghe, K. T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Addressing kinase-independent functions of Fak via PROTAC-mediated degradation</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>140</i></span>,  <span class="NLM_fpage">17019</span>– <span class="NLM_lpage">17026</span>, <span class="refDoi"> DOI: 10.1021/jacs.8b08008</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.8b08008" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC1cXit1CitLnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2018&pages=17019-17026&author=P.+M.+Crommauthor=K.+T.+G.+Samarasingheauthor=J.+Hinesauthor=C.+M.+Crews&title=Addressing+kinase-independent+functions+of+Fak+via+PROTAC-mediated+degradation&doi=10.1021%2Fjacs.8b08008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Addressing Kinase-Independent Functions of Fak via PROTAC-Mediated Degradation</span></div><div class="casAuthors">Cromm, Philipp M.; Samarasinghe, Kusal T. G.; Hines, John; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">17019-17026</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Enzymic inhibition proved to be a successful modality for the development of many small-mol. drugs.  In recent years, small-mol.-induced protein degrdn. has emerged as an orthogonal therapeutic strategy that has the potential to expand the druggable target space.  Focal adhesion kinase (Fak) is a key player in tumor invasion and metastasis, acting simultaneously as a kinase and a scaffold for several signaling proteins.  While previous efforts to modulate Fak activity were limited to kinase inhibitors with low success in clin. studies, protein degrdn. offers a possibility to simultaneously block Fak's kinase signaling and scaffolding capabilities.  Here, the authors report the development of a selective and potent Fak degrader, PROTAC-3, which outperforms a clin. candidate, defactinib, with respect to Fak activation as well as Fak-mediated cell migration and invasion.  These results underline the potential that PROTACs offer in expanding the druggable space and controlling protein functions that are not easily addressed by traditional small-mol. therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTn8r4NmDDObVg90H21EOLACvtfcHk0lgyRz3YIvCiLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXit1CitLnF&md5=b97b1a646a8f9dbf7a2bed640d0f001d</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fjacs.8b08008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.8b08008%26sid%3Dliteratum%253Aachs%26aulast%3DCromm%26aufirst%3DP.%2BM.%26aulast%3DSamarasinghe%26aufirst%3DK.%2BT.%2BG.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DAddressing%2520kinase-independent%2520functions%2520of%2520Fak%2520via%2520PROTAC-mediated%2520degradation%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2018%26volume%3D140%26spage%3D17019%26epage%3D17026%26doi%3D10.1021%2Fjacs.8b08008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaime-Figueroa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McQuaid, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toure, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">The advantages of targeted protein degradation over inhibition: An RTK case study</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">67</span>– <span class="NLM_lpage">77</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.09.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1016%2Fj.chembiol.2017.09.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=29129716" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=67-77&author=G.+M.+Burslemauthor=B.+E.+Smithauthor=A.+C.+Laiauthor=S.+Jaime-Figueroaauthor=D.+C.+McQuaidauthor=D.+P.+Bondesonauthor=M.+Toureauthor=H.+Dongauthor=Y.+Qianauthor=J.+Wangauthor=A.+P.+Crewauthor=J.+Hinesauthor=C.+M.+Crews&title=The+advantages+of+targeted+protein+degradation+over+inhibition%3A+An+RTK+case+study&doi=10.1016%2Fj.chembiol.2017.09.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">The Advantages of Targeted Protein Degradation Over Inhibition: An RTK Case Study</span></div><div class="casAuthors">Burslem, George M.; Smith, Blake E.; Lai, Ashton C.; Jaime-Figueroa, Saul; McQuaid, Daniel C.; Bondeson, Daniel P.; Toure, Momar; Dong, Hanqing; Qian, Yimin; Wang, Jing; Crew, Andrew P.; Hines, John; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">67-77.e3</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Proteolysis targeting chimera (PROTAC) technol. has emerged over the last two decades as a powerful tool for targeted degrdn. of endogenous proteins.  Herein we describe the development of PROTACs for receptor tyrosine kinases, a protein family yet to be targeted for induced protein degrdn.  The use of VHL-recruiting PROTACs against this protein family reveals several advantages of degrdn. over inhibition alone: direct comparisons of fully functional, target-degrading PROTACs with target-inhibiting variants that contain an inactivated E3 ligase-recruiting ligand show that degrdn. leads to more potent inhibition of cell proliferation and a more durable and sustained downstream signaling response, and thus addresses the kinome rewiring challenge seen with many receptor tyrosine kinase inhibitors.  Combined, these findings demonstrate the ability to target receptor tyrosine kinases for degrdn. using the PROTAC technol. and outline the advantages of this degrdn.-based approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4yDhRDWqNv7Vg90H21EOLACvtfcHk0lje7rxpUitywA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjM&md5=0ac39a6f5836ea3772a0396356efb630</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.09.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.09.009%26sid%3Dliteratum%253Aachs%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DSmith%26aufirst%3DB.%2BE.%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DMcQuaid%26aufirst%3DD.%2BC.%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DToure%26aufirst%3DM.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DThe%2520advantages%2520of%2520targeted%2520protein%2520degradation%2520over%2520inhibition%253A%2520An%2520RTK%2520case%2520study%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D67%26epage%3D77%26doi%3D10.1016%2Fj.chembiol.2017.09.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paez, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janne, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tracy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greulich, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabriel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herman, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaye, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindeman, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boggon, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naoki, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasaki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujii, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eck, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellers, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyerson, M.</span></span> <span> </span><span class="NLM_article-title">EGFR mutations in lung cancer: correlation with clinical response to gefitinib therapy</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>304</i></span>,  <span class="NLM_fpage">1497</span>– <span class="NLM_lpage">500</span>, <span class="refDoi"> DOI: 10.1126/science.1099314</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1126%2Fscience.1099314" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=15118125" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksVGmsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=304&publication_year=2004&pages=1497-500&author=J.+G.+Paezauthor=P.+A.+Janneauthor=J.+C.+Leeauthor=S.+Tracyauthor=H.+Greulichauthor=S.+Gabrielauthor=P.+Hermanauthor=F.+J.+Kayeauthor=N.+Lindemanauthor=T.+J.+Boggonauthor=K.+Naokiauthor=H.+Sasakiauthor=Y.+Fujiiauthor=M.+J.+Eckauthor=W.+R.+Sellersauthor=B.+E.+Johnsonauthor=M.+Meyerson&title=EGFR+mutations+in+lung+cancer%3A+correlation+with+clinical+response+to+gefitinib+therapy&doi=10.1126%2Fscience.1099314"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">EGFR Mutations in Lung Cancer: Correlation with Clinical Response to Gefitinib Therapy</span></div><div class="casAuthors">Paez, J. Guillermo; Jaenne, Pasi A.; Lee, Jeffrey C.; Tracy, Sean; Greulich, Heidi; Gabriel, Stacey; Herman, Paula; Kaye, Frederic J.; Lindeman, Neal; Boggon, Titus J.; Naoki, Katsuhiko; Sasaki, Hidefumi; Fujii, Yoshitaka; Eck, Michael J.; Sellers, William R.; Johnson, Bruce E.; Meyerson, Matthew</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">304</span>
        (<span class="NLM_cas:issue">5676</span>),
    <span class="NLM_cas:pages">1497-1500</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Receptor tyrosine kinase genes were sequenced in non-small cell lung cancer (NSCLC) and matched normal tissue.  Somatic mutations of the epidermal growth factor receptor gene EGFR were found in 15 of 58 unselected tumors from Japan and 1 of 61 from the United States.  Treatment with the EGFR kinase inhibitor gefitinib (Iressa) causes tumor regression in some patients with NSCLC, more frequently in Japan.  EGFR mutations were found in addnl. lung cancer samples from U.S. patients who responded to gefitinib therapy and in a lung adenocarcinoma cell line that was hypersensitive to growth inhibition by gefitinib, but not in gefitinib-insensitive tumors or cell lines.  These results suggest that EGFR mutations may predict sensitivity to gefitinib.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGxLJH5f3Kb7Vg90H21EOLACvtfcHk0lgrtjNijGik2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksVGmsbs%253D&md5=6df1b11bf9b3c284fbe000e34c61ebac</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1126%2Fscience.1099314&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1099314%26sid%3Dliteratum%253Aachs%26aulast%3DPaez%26aufirst%3DJ.%2BG.%26aulast%3DJanne%26aufirst%3DP.%2BA.%26aulast%3DLee%26aufirst%3DJ.%2BC.%26aulast%3DTracy%26aufirst%3DS.%26aulast%3DGreulich%26aufirst%3DH.%26aulast%3DGabriel%26aufirst%3DS.%26aulast%3DHerman%26aufirst%3DP.%26aulast%3DKaye%26aufirst%3DF.%2BJ.%26aulast%3DLindeman%26aufirst%3DN.%26aulast%3DBoggon%26aufirst%3DT.%2BJ.%26aulast%3DNaoki%26aufirst%3DK.%26aulast%3DSasaki%26aufirst%3DH.%26aulast%3DFujii%26aufirst%3DY.%26aulast%3DEck%26aufirst%3DM.%2BJ.%26aulast%3DSellers%26aufirst%3DW.%2BR.%26aulast%3DJohnson%26aufirst%3DB.%2BE.%26aulast%3DMeyerson%26aufirst%3DM.%26atitle%3DEGFR%2520mutations%2520in%2520lung%2520cancer%253A%2520correlation%2520with%2520clinical%2520response%2520to%2520gefitinib%2520therapy%26jtitle%3DScience%26date%3D2004%26volume%3D304%26spage%3D1497%26epage%3D500%26doi%3D10.1126%2Fscience.1099314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tahirovic, Y. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Truax, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jecs, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuo, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sum, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cvijic, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schroeder, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liotta, D. C.</span></span> <span> </span><span class="NLM_article-title">Discovery of N-alkyl piperazine side chain based CXCR4 antagonists with improved drug-like properties</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">446</span>– <span class="NLM_lpage">451</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.8b00030</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.8b00030" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnt1amt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=446-451&author=Y.+A.+Tahirovicauthor=V.+M.+Truaxauthor=R.+J.+Wilsonauthor=E.+Jecsauthor=H.+H.+Nguyenauthor=E.+J.+Millerauthor=M.+B.+Kimauthor=K.+M.+Kuoauthor=T.+Wangauthor=C.+S.+Sumauthor=M.+E.+Cvijicauthor=G.+M.+Schroederauthor=L.+J.+Wilsonauthor=D.+C.+Liotta&title=Discovery+of+N-alkyl+piperazine+side+chain+based+CXCR4+antagonists+with+improved+drug-like+properties&doi=10.1021%2Facsmedchemlett.8b00030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of N-Alkyl Piperazine Side Chain Based CXCR4 Antagonists with Improved Drug-like Properties</span></div><div class="casAuthors">Tahirovic, Yesim A.; Truax, Valarie M.; Wilson, Robert J.; Jecs, Edgars; Nguyen, Huy H.; Miller, Eric J.; Kim, Michelle B.; Kuo, Katie M.; Wang, Tao; Sum, Chi S.; Cvijic, Mary E.; Schroeder, Gretchen M.; Wilson, Lawrence J.; Liotta, Dennis C.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">446-451</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel series of CXCR4 antagonists with piperidinyl and piperazinyl alkylamine side chains designed as Bu amine replacements are described.  Several of these compds. showed similar activity to the parent compd. TIQ-15 (5) in a SDF-1 induced calcium flux assay.  Preliminary structure-activity relationship investigations led us to identify a series contg. N-Pr piperazine side chain analogs exemplified by 16 with improved off-target effects as measured in a muscarinic acetylcholine receptor (mAChR) calcium flux assay and in a limited drug safety panel screen.  Further efforts to explore SAR and optimize drug properties led to the identification of the N'-ethyl-N-propyl-piperazine tetrahydroisoquinoline deriv. 44 and the N-propyl-piperazine benzimidazole compd. 37, which gave the best overall profiles with no mAChR or CYP450 inhibition, good permeability in PAMPA assays, and metabolic stability in human liver microsomes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0J9JgLuoU4bVg90H21EOLACvtfcHk0ljk9lDi3H3FTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnt1amt7w%253D&md5=601d8a648b9a8b814ff2cb5c4382382c</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.8b00030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.8b00030%26sid%3Dliteratum%253Aachs%26aulast%3DTahirovic%26aufirst%3DY.%2BA.%26aulast%3DTruax%26aufirst%3DV.%2BM.%26aulast%3DWilson%26aufirst%3DR.%2BJ.%26aulast%3DJecs%26aufirst%3DE.%26aulast%3DNguyen%26aufirst%3DH.%2BH.%26aulast%3DMiller%26aufirst%3DE.%2BJ.%26aulast%3DKim%26aufirst%3DM.%2BB.%26aulast%3DKuo%26aufirst%3DK.%2BM.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DSum%26aufirst%3DC.%2BS.%26aulast%3DCvijic%26aufirst%3DM.%2BE.%26aulast%3DSchroeder%26aufirst%3DG.%2BM.%26aulast%3DWilson%26aufirst%3DL.%2BJ.%26aulast%3DLiotta%26aufirst%3DD.%2BC.%26atitle%3DDiscovery%2520of%2520N-alkyl%2520piperazine%2520side%2520chain%2520based%2520CXCR4%2520antagonists%2520with%2520improved%2520drug-like%2520properties%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2018%26volume%3D9%26spage%3D446%26epage%3D451%26doi%3D10.1021%2Facsmedchemlett.8b00030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, X. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Proteolysis targeting chimeras (PROTACs) of anaplastic lymphoma linase (ALK)</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>151</i></span>,  <span class="NLM_fpage">304</span>– <span class="NLM_lpage">314</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.03.071</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1016%2Fj.ejmech.2018.03.071" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=29627725" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnt1ejs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=151&publication_year=2018&pages=304-314&author=C.+Zhangauthor=X.+R.+Hanauthor=X.+Yangauthor=B.+Jiangauthor=J.+Liuauthor=Y.+Xiongauthor=J.+Jin&title=Proteolysis+targeting+chimeras+%28PROTACs%29+of+anaplastic+lymphoma+linase+%28ALK%29&doi=10.1016%2Fj.ejmech.2018.03.071"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis Targeting Chimeras (PROTACs) of Anaplastic Lymphoma Kinase (ALK)</span></div><div class="casAuthors">Zhang, Chengwei; Han, Xiao-Ran; Yang, Xiaobao; Jiang, Biao; Liu, Jing; Xiong, Yue; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">151</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">304-314</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Anaplastic lymphoma kinase (ALK) activation has been assocd. with many types of human cancer.  Significant efforts have been devoted to the development of ALK inhibitors to antagonize the kinase activity of ALK.  Four ALK inhibitors have been approved by the FDA to date for treating patients with ALK-pos. nonsmall cell lung cancers (NSCLC).  However, drug resistance has been obsd. in the majority of patients treated with these inhibitors.  New therapeutic strategies (e.g., compds. with novel mechanisms of action) are needed to overcome the drug resistance issue.  The emerging PROTAC (Proteolysis Targeting Chimera) technol. has been successfully applied to selective degrdn. of multiple protein targets, but not ALK.  Since ALK protein levels are not important for viability in mammals, ALK PROTACs could lead to novel therapeutics with minimal toxicity.  Here the authors report the design, synthesis and biol. evaluation of novel PROTACs (degraders) of ALK.  MS4077 (I) and MS4078 (II) potently decreased cellular levels of oncogenic active ALK fusion proteins in a concn.- and time-dependent manner in SU-DHL-1 lymphoma and NCI-H2228 lung cancer cells.  The ALK protein degrdn. induced by compds. I and II was cereblon and proteasome dependent.  In addn., compds. I and II potently inhibited proliferation of SU-DHL-1 cells.  Furthermore, compd. II displayed good plasma exposure in a mouse pharmacokinetic study, thus is suitable for in vivo efficacy studies.  The authors also developed MS4748 and MS4740 , very close analogs of I and II resp., which are incapable to degrade the ALK fusion proteins, as neg. controls.  All 4 compds. are valuable chem. tools for investigating effects of ALK pharmacol. degrdn.  The study paved the way for developing the next generation of ALK PROTACs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPGM3aAZZ6h7Vg90H21EOLACvtfcHk0ljk9lDi3H3FTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnt1ejs7Y%253D&md5=804324b95034f38b46e45f89e18e55be</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.03.071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.03.071%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DHan%26aufirst%3DX.%2BR.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DProteolysis%2520targeting%2520chimeras%2520%2528PROTACs%2529%2520of%2520anaplastic%2520lymphoma%2520linase%2520%2528ALK%2529%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D151%26spage%3D304%26epage%3D314%26doi%3D10.1016%2Fj.ejmech.2018.03.071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguilar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of MD-224 as a first-in-class, highly potent, and efficacious proteolysis targeting chimera Murine Double Minute 2 degrader capable of achieving complete and durable tumor regression</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">448</span>– <span class="NLM_lpage">466</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00909</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00909" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVChtrfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=448-466&author=Y.+Liauthor=J.+Yangauthor=A.+Aguilarauthor=D.+McEachernauthor=S.+Przybranowskiauthor=L.+Liuauthor=C.+Y.+Yangauthor=M.+Wangauthor=X.+Hanauthor=S.+Wang&title=Discovery+of+MD-224+as+a+first-in-class%2C+highly+potent%2C+and+efficacious+proteolysis+targeting+chimera+Murine+Double+Minute+2+degrader+capable+of+achieving+complete+and+durable+tumor+regression&doi=10.1021%2Facs.jmedchem.8b00909"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of MD-224 as a First-in-Class, Highly Potent, and Efficacious Proteolysis Targeting Chimera Murine Double Minute 2 Degrader Capable of Achieving Complete and Durable Tumor Regression</span></div><div class="casAuthors">Li, Yangbing; Yang, Jiuling; Aguilar, Angelo; McEachern, Donna; Przybranowski, Sally; Liu, Liu; Yang, Chao-Yie; Wang, Mi; Han, Xin; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">448-466</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Human murine double minute 2 (MDM2) protein is a primary endogenous cellular inhibitor of the tumor suppressor p53 and has been pursued as an attractive cancer therapeutic target.  Several potent, nonpeptide, small-mol. inhibitors of MDM2 are currently in clin. development.  In this paper, we report our design, synthesis, and evaluation of small-mol. MDM2 degraders based on the proteolysis targeting chimera (PROTAC) concept.  The most promising compd. (MD-224, I) effectively induces rapid degrdn. of MDM2 at concns. <1 nM in human leukemia cells.  It achieves an IC50 value of 1.5 nM in inhibition of growth of RS4;11 cells and also low nanomolar IC50 values in a panel of leukemia cell lines.  MD-224 (I) achieves complete and durable tumor regression in vivo in the RS4;11 xenograft tumor model in mice at well-tolerated dose schedules.  MD-224 (I) is thus a highly potent and efficacious MDM2 degrader and warrants extensive evaluations as a new class of anticancer agent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfVzK-tVEdk7Vg90H21EOLACvtfcHk0liSy37kmAGM8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVChtrfK&md5=3eaa40a8b0ae15c0172ef44a2fd1b902</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00909&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00909%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DAguilar%26aufirst%3DA.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DHan%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520MD-224%2520as%2520a%2520first-in-class%252C%2520highly%2520potent%252C%2520and%2520efficacious%2520proteolysis%2520targeting%2520chimera%2520Murine%2520Double%2520Minute%25202%2520degrader%2520capable%2520of%2520achieving%2520complete%2520and%2520durable%2520tumor%2520regression%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D448%26epage%3D466%26doi%3D10.1021%2Facs.jmedchem.8b00909" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schramm, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agnetta, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bermudez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerwe, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irmen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holze, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Littmann, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolber, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trankle, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Decker, M.</span></span> <span> </span><span class="NLM_article-title">Novel BQCA- and TBPB-derived M1 receptor hybrid ligands: orthosteric carbachol differentially regulates partial agonism</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">1349</span>– <span class="NLM_lpage">1358</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201900283</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1002%2Fcmdc.201900283" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=31166078" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC1MXht1yrsLrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=1349-1358&author=S.+Schrammauthor=L.+Agnettaauthor=M.+Bermudezauthor=H.+Gerweauthor=M.+Irmenauthor=J.+Holzeauthor=T.+Littmannauthor=G.+Wolberauthor=C.+Trankleauthor=M.+Decker&title=Novel+BQCA-+and+TBPB-derived+M1+receptor+hybrid+ligands%3A+orthosteric+carbachol+differentially+regulates+partial+agonism&doi=10.1002%2Fcmdc.201900283"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Novel BQCA- and TBPB-derived M1 receptor hybrid ligands: orthosteric carbachol differentially regulates partial agonism</span></div><div class="casAuthors">Schramm, Simon; Agnetta, Luca; Bermudez, Marcel; Gerwe, Hubert; Irmen, Matthias; Holze, Janine; Littmann, Timo; Wolber, Gerhard; Traenkle, Christian; Decker, Michael</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1349-1358</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Recently, investigations of the complex mechanisms of allostery have led to a deeper understanding of G protein-coupled receptor (GPCR) activation and signaling processes.  In this context, muscarinic acetylcholine receptors (mAChRs) are highly relevant due to their exemplary role in the study of allosteric modulation.  In this work, we compare and discuss two sets of putatively dualsteric ligands, which were designed to connect carbachol to different types of allosteric ligands.  We chose derivs. of TBPB [1-(1'-(2-tolyl)-1,4'-bipiperidin-4-yl)-1H-benzo[d]imidazol-2(3H)-one] as M1-selective putative bitopic ligands, and derivs. of benzyl quinolone carboxylic acid (BQCA) as an M1 pos. allosteric modulator, varying the distance between the allosteric and orthosteric building blocks.  Luciferase protein complementation assays demonstrated that linker length must be carefully chosen to yield either agonist or antagonist behavior.  These findings may help to design biased signaling and/or different extents of efficacy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmzb2TzuROxbVg90H21EOLACvtfcHk0lhIBi6zOo856A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXht1yrsLrJ&md5=717880b115bc2c32ff1ccdf73a22dec3</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201900283&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201900283%26sid%3Dliteratum%253Aachs%26aulast%3DSchramm%26aufirst%3DS.%26aulast%3DAgnetta%26aufirst%3DL.%26aulast%3DBermudez%26aufirst%3DM.%26aulast%3DGerwe%26aufirst%3DH.%26aulast%3DIrmen%26aufirst%3DM.%26aulast%3DHolze%26aufirst%3DJ.%26aulast%3DLittmann%26aufirst%3DT.%26aulast%3DWolber%26aufirst%3DG.%26aulast%3DTrankle%26aufirst%3DC.%26aulast%3DDecker%26aufirst%3DM.%26atitle%3DNovel%2520BQCA-%2520and%2520TBPB-derived%2520M1%2520receptor%2520hybrid%2520ligands%253A%2520orthosteric%2520carbachol%2520differentially%2520regulates%2520partial%2520agonism%26jtitle%3DChemMedChem%26date%3D2019%26volume%3D14%26spage%3D1349%26epage%3D1358%26doi%3D10.1002%2Fcmdc.201900283" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Molle, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gareiss, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tae, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noblin, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Targeting the von Hippel-Lindau E3 ubiquitin ligase using small molecules to disrupt the VHL/HIF-1alpha interaction</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>134</i></span>,  <span class="NLM_fpage">4465</span>– <span class="NLM_lpage">4468</span>, <span class="refDoi"> DOI: 10.1021/ja209924v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja209924v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC38XivVOgurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2012&pages=4465-4468&author=D.+L.+Buckleyauthor=I.+Van+Molleauthor=P.+C.+Gareissauthor=H.+S.+Taeauthor=J.+Michelauthor=D.+J.+Noblinauthor=W.+L.+Jorgensenauthor=A.+Ciulliauthor=C.+M.+Crews&title=Targeting+the+von+Hippel-Lindau+E3+ubiquitin+ligase+using+small+molecules+to+disrupt+the+VHL%2FHIF-1alpha+interaction&doi=10.1021%2Fja209924v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the von Hippel-Lindau E3 Ubiquitin Ligase Using Small Molecules To Disrupt the VHL/HIF-1α Interaction</span></div><div class="casAuthors">Buckley, Dennis L.; Van Molle, Inge; Gareiss, Peter C.; Tae, Hyun Seop; Michel, Julien; Noblin, Devin J.; Jorgensen, William L.; Ciulli, Alessio; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4465-4468</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">E3 ubiquitin ligases, which bind protein targets, leading to their ubiquitination and subsequent degrdn., are attractive drug targets due to their exquisite substrate specificity.  However, the development of small-mol. inhibitors has proven extraordinarily challenging as modulation of E3 ligase activities requires the targeting of protein-protein interactions.  Using rational design, we have generated the first small mol. targeting the von Hippel-Lindau protein (VHL), the substrate recognition subunit of an E3 ligase, and an important target in cancer, chronic anemia, and ischemia.  We have also obtained the crystal structure of VHL bound to our most potent inhibitor, confirming that the compd. mimics the binding mode of the transcription factor HIF-1α, a substrate of VHL.  These results have the potential to guide future development of improved lead compds. as therapeutics for the treatment of chronic anemia and ischemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolmBVM_mfWzrVg90H21EOLACvtfcHk0lhIBi6zOo856A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XivVOgurw%253D&md5=60f7384b6bc932c9828dc12999283a1e</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fja209924v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja209924v%26sid%3Dliteratum%253Aachs%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DVan%2BMolle%26aufirst%3DI.%26aulast%3DGareiss%26aufirst%3DP.%2BC.%26aulast%3DTae%26aufirst%3DH.%2BS.%26aulast%3DMichel%26aufirst%3DJ.%26aulast%3DNoblin%26aufirst%3DD.%2BJ.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DCiulli%26aufirst%3DA.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DTargeting%2520the%2520von%2520Hippel-Lindau%2520E3%2520ubiquitin%2520ligase%2520using%2520small%2520molecules%2520to%2520disrupt%2520the%2520VHL%252FHIF-1alpha%2520interaction%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2012%26volume%3D134%26spage%3D4465%26epage%3D4468%26doi%3D10.1021%2Fja209924v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafson, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Molle, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tae, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gareiss, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Small-molecule inhibitors of the interaction between the E3 ligase VHL and HIF1alpha</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">11463</span>– <span class="NLM_lpage">11467</span>, <span class="refDoi"> DOI: 10.1002/anie.201206231</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1002%2Fanie.201206231" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsV2rs7nL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=11463-11467&author=D.+L.+Buckleyauthor=J.+L.+Gustafsonauthor=I.+Van+Molleauthor=A.+G.+Rothauthor=H.+S.+Taeauthor=P.+C.+Gareissauthor=W.+L.+Jorgensenauthor=A.+Ciulliauthor=C.+M.+Crews&title=Small-molecule+inhibitors+of+the+interaction+between+the+E3+ligase+VHL+and+HIF1alpha&doi=10.1002%2Fanie.201206231"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Inhibitors of the Interaction between the E3 Ligase VHL and HIF1α</span></div><div class="casAuthors">Buckley, Dennis L.; Gustafson, Jeffrey L.; Van Molle, Inge; Roth, Anke G.; Tae, Hyun Seop; Gareiss, Peter C.; Jorgensen, William L.; Ciulli, Alessio; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">46</span>),
    <span class="NLM_cas:pages">11463-11467</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">This study reports the structure-based design and synthesis of novel VHL ligands, illustrating structure-activity relationships and improving potency.  Two fragments of the ligands were individually optimized, arriving at a new class of N-terminal benzamide fragments.  The interactions made by this new class were then elucidated through X-ray crystallog. of the protein-ligand complex.  Structural studies of this ligand class revealed both a new binding mode as well as a novel water-mediated' hydrogen bond.  Finally, using these optimized fragments, we assembled new VHL ligands in a combinatorial fashion, arriving at (I), the first small-mol. submicromolar inhibitor of the interaction between VHL and a peptide derived from HIF1α.  While no cell-based activity has yet been obsd.,we hope to take advantage of the interactions highlighted by the crystal structure of I to design more drug-like inhibitors of VHL with improved cell permeability.  These ligands and the novel interactions and binding modes highlighted have the potential to guide the development of future chem. probes that can target the E3 ligase VHL or stabilize HIF1α proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqs0HINAsXwwLVg90H21EOLACvtfcHk0lj7HJmhF8zKXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsV2rs7nL&md5=eefeb63a3491737738e94c00be083cfd</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1002%2Fanie.201206231&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201206231%26sid%3Dliteratum%253Aachs%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DGustafson%26aufirst%3DJ.%2BL.%26aulast%3DVan%2BMolle%26aufirst%3DI.%26aulast%3DRoth%26aufirst%3DA.%2BG.%26aulast%3DTae%26aufirst%3DH.%2BS.%26aulast%3DGareiss%26aufirst%3DP.%2BC.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DCiulli%26aufirst%3DA.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DSmall-molecule%2520inhibitors%2520of%2520the%2520interaction%2520between%2520the%2520E3%2520ligase%2520VHL%2520and%2520HIF1alpha%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2012%26volume%3D51%26spage%3D11463%26epage%3D11467%26doi%3D10.1002%2Fanie.201206231" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darricarrere, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafson, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, I. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miah, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harling, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">HaloPROTACS: use of small molecule PROTACs to induce degradation of HaloTag fusion proteins</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1831</span>– <span class="NLM_lpage">1837</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00442</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00442" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVSht7fN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1831-1837&author=D.+L.+Buckleyauthor=K.+Rainaauthor=N.+Darricarrereauthor=J.+Hinesauthor=J.+L.+Gustafsonauthor=I.+E.+Smithauthor=A.+H.+Miahauthor=J.+D.+Harlingauthor=C.+M.+Crews&title=HaloPROTACS%3A+use+of+small+molecule+PROTACs+to+induce+degradation+of+HaloTag+fusion+proteins&doi=10.1021%2Facschembio.5b00442"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">HaloPROTACS: Use of Small Molecule PROTACs to Induce Degradation of HaloTag Fusion Proteins</span></div><div class="casAuthors">Buckley, Dennis L.; Raina, Kanak; Darricarrere, Nicole; Hines, John; Gustafson, Jeffrey L.; Smith, Ian E.; Miah, Afjal H.; Harling, John D.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1831-1837</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Small mol.-induced protein degrdn. is an attractive strategy for the development of chem. probes.  One method for inducing targeted protein degrdn. involves the use of PROTACs, heterobifunctional mols. that can recruit specific E3 ligases to a desired protein of interest.  PROTACs have been successfully used to degrade numerous proteins in cells, but the peptidic E3 ligase ligands used in previous PROTACs have hindered their development into more mature chem. probes or therapeutics.  The authors report the design of a novel class of PROTACs that incorporate small mol. VHL ligands to successfully degrade HaloTag7 fusion proteins.  These HaloPROTACs will inspire the development of future PROTACs with more drug-like properties.  Addnl., these HaloPROTACs are useful chem. genetic tools, due to their ability to chem. knock down widely used HaloTag7 fusion proteins in a general fashion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotj-Sgj4TyKrVg90H21EOLACvtfcHk0lj7HJmhF8zKXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVSht7fN&md5=b074c58e983f15c384818777b25e248d</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00442&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00442%26sid%3Dliteratum%253Aachs%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DDarricarrere%26aufirst%3DN.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DGustafson%26aufirst%3DJ.%2BL.%26aulast%3DSmith%26aufirst%3DI.%2BE.%26aulast%3DMiah%26aufirst%3DA.%2BH.%26aulast%3DHarling%26aufirst%3DJ.%2BD.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DHaloPROTACS%253A%2520use%2520of%2520small%2520molecule%2520PROTACs%2520to%2520induce%2520degradation%2520of%2520HaloTag%2520fusion%2520proteins%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D1831%26epage%3D1837%26doi%3D10.1021%2Facschembio.5b00442" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bohm, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lydeard, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stadler, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavadini, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serluca, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acker, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingaraju, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tichkule, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schebesta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forrester, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schirle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassiepen, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottl, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hild, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beckwith, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thoma, N. H.</span></span> <span> </span><span class="NLM_article-title">Structure of the DDB1-CRBN E3 ubiquitin ligase in complex with thalidomide</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>512</i></span>,  <span class="NLM_fpage">49</span>– <span class="NLM_lpage">53</span>, <span class="refDoi"> DOI: 10.1038/nature13527</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1038%2Fnature13527" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=25043012" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht12ms7%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=512&publication_year=2014&pages=49-53&author=E.+S.+Fischerauthor=K.+Bohmauthor=J.+R.+Lydeardauthor=H.+Yangauthor=M.+B.+Stadlerauthor=S.+Cavadiniauthor=J.+Nagelauthor=F.+Serlucaauthor=V.+Ackerauthor=G.+M.+Lingarajuauthor=R.+B.+Tichkuleauthor=M.+Schebestaauthor=W.+C.+Forresterauthor=M.+Schirleauthor=U.+Hassiepenauthor=J.+Ottlauthor=M.+Hildauthor=R.+E.+Beckwithauthor=J.+W.+Harperauthor=J.+L.+Jenkinsauthor=N.+H.+Thoma&title=Structure+of+the+DDB1-CRBN+E3+ubiquitin+ligase+in+complex+with+thalidomide&doi=10.1038%2Fnature13527"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the DDB1-CRBN E3 ubiquitin ligase in complex with thalidomide</span></div><div class="casAuthors">Fischer, Eric S.; Bohm, Kerstin; Lydeard, John R.; Yang, Haidi; Stadler, Michael B.; Cavadini, Simone; Nagel, Jane; Serluca, Fabrizio; Acker, Vincent; Lingaraju, Gondichatnahalli M.; Tichkule, Ritesh B.; Schebesta, Michael; Forrester, William C.; Schirle, Markus; Hassiepen, Ulrich; Ottl, Johannes; Hild, Marc; Beckwith, Rohan E. J.; Harper, J. Wade; Jenkins, Jeremy L.; Thoma, Nicolas H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">512</span>
        (<span class="NLM_cas:issue">7512</span>),
    <span class="NLM_cas:pages">49-53</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">In the 1950s, the drug thalidomide, administered as a sedative to pregnant women, led to the birth of thousands of children with multiple defects.  Despite the teratogenicity of thalidomide and its derivs. lenalidomide and pomalidomide, these immunomodulatory drugs (IMiDs) recently emerged as effective treatments for multiple myeloma and 5q-deletion-assocd. dysplasia.  IMiDs target the E3 ubiquitin ligase CUL4-RBX1-DDB1-CRBN (known as CRL4CRBN) and promote the ubiquitination of the IKAROS family transcription factors IKZF1 and IKZF3 by CRL4CRBN.  Here we present crystal structures of the DDB1-CRBN complex bound to thalidomide, lenalidomide and pomalidomide.  The structure establishes that CRBN is a substrate receptor within CRL4CRBN and enantioselectively binds IMiDs.  Using an unbiased screen, we identified the homeobox transcription factor MEIS2 as an endogenous substrate of CRL4CRBN.  Our studies suggest that IMiDs block endogenous substrates (MEIS2) from binding to CRL4CRBN while the ligase complex is recruiting IKZF1 or IKZF3 for degrdn.  This dual activity implies that small mols. can modulate an E3 ubiquitin ligase and thereby upregulate or downregulate the ubiquitination of proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGt2Y9GecKr7Vg90H21EOLACvtfcHk0lj7HJmhF8zKXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht12ms7%252FP&md5=1c1eaa48bef87463cdd3f2b1e1bd400d</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1038%2Fnature13527&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13527%26sid%3Dliteratum%253Aachs%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DBohm%26aufirst%3DK.%26aulast%3DLydeard%26aufirst%3DJ.%2BR.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DStadler%26aufirst%3DM.%2BB.%26aulast%3DCavadini%26aufirst%3DS.%26aulast%3DNagel%26aufirst%3DJ.%26aulast%3DSerluca%26aufirst%3DF.%26aulast%3DAcker%26aufirst%3DV.%26aulast%3DLingaraju%26aufirst%3DG.%2BM.%26aulast%3DTichkule%26aufirst%3DR.%2BB.%26aulast%3DSchebesta%26aufirst%3DM.%26aulast%3DForrester%26aufirst%3DW.%2BC.%26aulast%3DSchirle%26aufirst%3DM.%26aulast%3DHassiepen%26aufirst%3DU.%26aulast%3DOttl%26aufirst%3DJ.%26aulast%3DHild%26aufirst%3DM.%26aulast%3DBeckwith%26aufirst%3DR.%2BE.%26aulast%3DHarper%26aufirst%3DJ.%2BW.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26aulast%3DThoma%26aufirst%3DN.%2BH.%26atitle%3DStructure%2520of%2520the%2520DDB1-CRBN%2520E3%2520ubiquitin%2520ligase%2520in%2520complex%2520with%2520thalidomide%26jtitle%3DNature%26date%3D2014%26volume%3D512%26spage%3D49%26epage%3D53%26doi%3D10.1038%2Fnature13527" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fabian, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biggs, W. H.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atteridge, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azimioara, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benedetti, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edeen, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Floyd, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galvin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerlach, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grotzfeld, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrgard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Insko, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Insko, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lelias, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehta, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milanov, Z. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velasco, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wodicka, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarrinkar, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lockhart, D. J.</span></span> <span> </span><span class="NLM_article-title">A small molecule-kinase interaction map for clinical kinase inhibitors</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">329</span>– <span class="NLM_lpage">336</span>, <span class="refDoi"> DOI: 10.1038/nbt1068</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1038%2Fnbt1068" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=15711537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BD2MXitF2nt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2005&pages=329-336&author=M.+A.+Fabianauthor=W.+H.+Biggsauthor=D.+K.+Treiberauthor=C.+E.+Atteridgeauthor=M.+D.+Azimioaraauthor=M.+G.+Benedettiauthor=T.+A.+Carterauthor=P.+Ciceriauthor=P.+T.+Edeenauthor=M.+Floydauthor=J.+M.+Fordauthor=M.+Galvinauthor=J.+L.+Gerlachauthor=R.+M.+Grotzfeldauthor=S.+Herrgardauthor=D.+E.+Inskoauthor=M.+A.+Inskoauthor=A.+G.+Laiauthor=J.+M.+Leliasauthor=S.+A.+Mehtaauthor=Z.+V.+Milanovauthor=A.+M.+Velascoauthor=L.+M.+Wodickaauthor=H.+K.+Patelauthor=P.+P.+Zarrinkarauthor=D.+J.+Lockhart&title=A+small+molecule-kinase+interaction+map+for+clinical+kinase+inhibitors&doi=10.1038%2Fnbt1068"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">A small molecule-kinase interaction map for clinical kinase inhibitors</span></div><div class="casAuthors">Fabian, Miles A.; Biggs, William H.; Treiber, Daniel K.; Atteridge, Corey E.; Azimioara, Mihai D.; Benedetti, Michael G.; Carter, Todd A.; Ciceri, Pietro; Edeen, Philip T.; Floyd, Mark; Ford, Julia M.; Galvin, Margaret; Gerlach, Jay L.; Grotzfeld, Robert M.; Herrgard, Sanna; Insko, Darren E.; Insko, Michael A.; Lai, Andiliy G.; Lelias, Jean-Michel; Mehta, Shamal A.; Milanov, Zdravko V.; Velasco, Anne Marie; Wodicka, Lisa M.; Patel, Hitesh K.; Zarrinkar, Patrick P.; Lockhart, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">329-336</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Kinase inhibitors show great promise as a new class of therapeutics.  Here the authors describe an efficient way to det. kinase inhibitor specificity by measuring binding of small mols. to the ATP site of kinases.  The authors have profiled 20 kinase inhibitors, including 16 that are approved drugs or in clin. development, against a panel of 119 protein kinases.  The authors find that specificity varies widely and is not strongly correlated with chem. structure or the identity of the intended target.  Many novel interactions were identified, including tight binding of the p38 inhibitor BIRB-796 to an imatinib-resistant variant of the ABL kinase, and binding of imatinib to the SRC-family kinase LCK.  The authors also show that mutations in the epidermal growth factor receptor (EGFR) found in gefitinib-responsive patients do not affect the binding affinity of gefitinib or erlotinib.  Our results represent a systematic small mol.-protein interaction map for clin. compds. across a large no. of related proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqA0vNmMOXrnrVg90H21EOLACvtfcHk0lgvHcnwUF47Xw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXitF2nt7w%253D&md5=6aedd5ceb8f77cd26ee50425dcec7bdf</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1038%2Fnbt1068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1068%26sid%3Dliteratum%253Aachs%26aulast%3DFabian%26aufirst%3DM.%2BA.%26aulast%3DBiggs%26aufirst%3DW.%2BH.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DAtteridge%26aufirst%3DC.%2BE.%26aulast%3DAzimioara%26aufirst%3DM.%2BD.%26aulast%3DBenedetti%26aufirst%3DM.%2BG.%26aulast%3DCarter%26aufirst%3DT.%2BA.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DEdeen%26aufirst%3DP.%2BT.%26aulast%3DFloyd%26aufirst%3DM.%26aulast%3DFord%26aufirst%3DJ.%2BM.%26aulast%3DGalvin%26aufirst%3DM.%26aulast%3DGerlach%26aufirst%3DJ.%2BL.%26aulast%3DGrotzfeld%26aufirst%3DR.%2BM.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DInsko%26aufirst%3DD.%2BE.%26aulast%3DInsko%26aufirst%3DM.%2BA.%26aulast%3DLai%26aufirst%3DA.%2BG.%26aulast%3DLelias%26aufirst%3DJ.%2BM.%26aulast%3DMehta%26aufirst%3DS.%2BA.%26aulast%3DMilanov%26aufirst%3DZ.%2BV.%26aulast%3DVelasco%26aufirst%3DA.%2BM.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DPatel%26aufirst%3DH.%2BK.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26aulast%3DLockhart%26aufirst%3DD.%2BJ.%26atitle%3DA%2520small%2520molecule-kinase%2520interaction%2520map%2520for%2520clinical%2520kinase%2520inhibitors%26jtitle%3DNat.%2520Biotechnol.%26date%3D2005%26volume%3D23%26spage%3D329%26epage%3D336%26doi%3D10.1038%2Fnbt1068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Douglass, E. F.,  Jr</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sparer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shapiro, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spiegel, D. A.</span></span> <span> </span><span class="NLM_article-title">A comprehensive mathematical model for three-body binding equilibria</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>135</i></span>,  <span class="NLM_fpage">6092</span>– <span class="NLM_lpage">6099</span>, <span class="refDoi"> DOI: 10.1021/ja311795d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja311795d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC3sXltVCktr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=6092-6099&author=E.+F.+Douglassauthor=C.+J.+Millerauthor=G.+Sparerauthor=H.+Shapiroauthor=D.+A.+Spiegel&title=A+comprehensive+mathematical+model+for+three-body+binding+equilibria&doi=10.1021%2Fja311795d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">A Comprehensive Mathematical Model for Three-Body Binding Equilibria</span></div><div class="casAuthors">Douglass, Eugene F.; Miller, Chad J.; Sparer, Gerson; Shapiro, Harold; Spiegel, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">6092-6099</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Three-component systems are often more complex than their two-component counterparts.  Although the reversible assocn. of three components in soln. is crit. for a vast array of chem. and biol. processes, no general phys. picture of such systems has emerged.  Here the authors have developed a general, comprehensive framework for understanding ternary complex equil., which relates directly to familiar concepts such as EC50 and IC50 from simpler (binary complex) equil.  Importantly, application of the authors' model to data from the published literature has enabled the authors to achieve new insights into complex systems ranging from coagulation to therapeutic dosing regimens for monoclonal antibodies.  The authors also provide an Excel spreadsheet to assist readers in both conceptualizing and applying the authors' models.  Overall, the anal. has the potential to render complex three-component systems-which have previously been characterized as "anal. intractable"-readily comprehensible to theoreticians and experimentalists alike.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4T36t0kgZ3LVg90H21EOLACvtfcHk0ljm-1K_ClLKwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXltVCktr4%253D&md5=44a6da2858f9813d7f1329c96a7085ac</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Fja311795d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja311795d%26sid%3Dliteratum%253Aachs%26aulast%3DDouglass%26aufirst%3DE.%2BF.%26aulast%3DMiller%26aufirst%3DC.%2BJ.%26aulast%3DSparer%26aufirst%3DG.%26aulast%3DShapiro%26aufirst%3DH.%26aulast%3DSpiegel%26aufirst%3DD.%2BA.%26atitle%3DA%2520comprehensive%2520mathematical%2520model%2520for%2520three-body%2520binding%2520equilibria%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26spage%3D6092%26epage%3D6099%26doi%3D10.1021%2Fja311795d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thapa, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, R. A.</span></span> <span> </span><span class="NLM_article-title">A kinase-independent role for EGF receptor in autophagy initiation</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>160</i></span>,  <span class="NLM_fpage">145</span>– <span class="NLM_lpage">160</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2014.12.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1016%2Fj.cell.2014.12.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=25594178" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtF2lsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=160&publication_year=2015&pages=145-160&author=X.+Tanauthor=N.+Thapaauthor=Y.+Sunauthor=R.+A.+Anderson&title=A+kinase-independent+role+for+EGF+receptor+in+autophagy+initiation&doi=10.1016%2Fj.cell.2014.12.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">A Kinase-Independent Role for EGF Receptor in Autophagy Initiation</span></div><div class="casAuthors">Tan, Xiaojun; Thapa, Narendra; Sun, Yue; Anderson, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">160</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">145-160</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The epidermal growth factor receptor (EGFR) is upregulated in numerous human cancers.  Inhibition of EGFR signaling induces autophagy in tumor cells.  Here, we report an unanticipated role for the inactive EGFR in autophagy initiation.  Inactive EGFR interacts with the oncoprotein LAPTM4B that is required for the endosomal accumulation of EGFR upon serum starvation.  Inactive EGFR and LAPTM4B stabilize each other at endosomes and recruit the exocyst subcomplex contg. Sec5.  We show that inactive EGFR, LAPTM4B, and the Sec5 subcomplex are required for basal and starvation-induced autophagy.  LAPTM4B and Sec5 promote EGFR assocn. with the autophagy inhibitor Rubicon, which in turn disassocs. Beclin 1 from Rubicon to initiate autophagy.  Thus, the oncoprotein LAPTM4B facilitates the role of inactive EGFR in autophagy initiation.  This pathway is positioned to control tumor metab. and promote tumor cell survival upon serum deprivation or metabolic stress.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGosEWXApqULAbVg90H21EOLACvtfcHk0ljm-1K_ClLKwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtF2lsbw%253D&md5=a19af31e428f3c08d1c460be0c021730</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2014.12.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2014.12.006%26sid%3Dliteratum%253Aachs%26aulast%3DTan%26aufirst%3DX.%26aulast%3DThapa%26aufirst%3DN.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DAnderson%26aufirst%3DR.%2BA.%26atitle%3DA%2520kinase-independent%2520role%2520for%2520EGF%2520receptor%2520in%2520autophagy%2520initiation%26jtitle%3DCell%26date%3D2015%26volume%3D160%26spage%3D145%26epage%3D160%26doi%3D10.1016%2Fj.cell.2014.12.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Soucy, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milhollen, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gavin, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adhikari, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brownell, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardin, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Critchley, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cullis, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doucette, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garnsey, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaulin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gershman, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lublinsky, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizutani, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narayanan, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olhava, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peluso, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rezaei, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sintchak, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talreja, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Traore, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vyskocil, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weatherhead, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dick, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Claiborne, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rolfe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolen, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langston, S. P.</span></span> <span> </span><span class="NLM_article-title">An inhibitor of NEDD8-activating enzyme as a new approach to treat cancer</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>458</i></span>,  <span class="NLM_fpage">732</span>– <span class="NLM_lpage">736</span>, <span class="refDoi"> DOI: 10.1038/nature07884</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1038%2Fnature07884" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=19360080" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BD1MXksVenu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=458&publication_year=2009&pages=732-736&author=T.+A.+Soucyauthor=P.+G.+Smithauthor=M.+A.+Milhollenauthor=A.+J.+Bergerauthor=J.+M.+Gavinauthor=S.+Adhikariauthor=J.+E.+Brownellauthor=K.+E.+Burkeauthor=D.+P.+Cardinauthor=S.+Critchleyauthor=C.+A.+Cullisauthor=A.+Doucetteauthor=J.+J.+Garnseyauthor=J.+L.+Gaulinauthor=R.+E.+Gershmanauthor=A.+R.+Lublinskyauthor=A.+McDonaldauthor=H.+Mizutaniauthor=U.+Narayananauthor=E.+J.+Olhavaauthor=S.+Pelusoauthor=M.+Rezaeiauthor=M.+D.+Sintchakauthor=T.+Talrejaauthor=M.+P.+Thomasauthor=T.+Traoreauthor=S.+Vyskocilauthor=G.+S.+Weatherheadauthor=J.+Yuauthor=J.+Zhangauthor=L.+R.+Dickauthor=C.+F.+Claiborneauthor=M.+Rolfeauthor=J.+B.+Bolenauthor=S.+P.+Langston&title=An+inhibitor+of+NEDD8-activating+enzyme+as+a+new+approach+to+treat+cancer&doi=10.1038%2Fnature07884"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">An inhibitor of NEDD8-activating enzyme as a new approach to treat cancer</span></div><div class="casAuthors">Soucy, Teresa A.; Smith, Peter G.; Milhollen, Michael A.; Berger, Allison J.; Gavin, James M.; Adhikari, Sharmila; Brownell, James E.; Burke, Kristine E.; Cardin, David P.; Critchley, Stephen; Cullis, Courtney A.; Doucette, Amanda; Garnsey, James J.; Gaulin, Jeffrey L.; Gershman, Rachel E.; Lublinsky, Anna R.; McDonald, Alice; Mizutani, Hirotake; Narayanan, Usha; Olhava, Edward J.; Peluso, Stephane; Rezaei, Mansoureh; Sintchak, Michael D.; Talreja, Tina; Thomas, Michael P.; Traore, Tary; Vyskocil, Stepan; Weatherhead, Gabriel S.; Yu, Jie; Zhang, Julie; Dick, Lawrence R.; Claiborne, Christopher F.; Rolfe, Mark; Bolen, Joseph B.; Langston, Steven P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">458</span>
        (<span class="NLM_cas:issue">7239</span>),
    <span class="NLM_cas:pages">732-736</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The clin. development of an inhibitor of cellular proteasome function suggests that compds. targeting other components of the ubiquitin-proteasome system might prove useful for the treatment of human malignancies.  NEDD8-activating enzyme (NAE) is an essential component of the NEDD8 conjugation pathway that controls the activity of the cullin-RING subtype of ubiquitin ligases, thereby regulating the turnover of a subset of proteins upstream of the proteasome.  Substrates of cullin-RING ligases have important roles in cellular processes assocd. with cancer cell growth and survival pathways.  Here we describe MLN4924, a potent and selective inhibitor of NAE.  MLN4924 disrupts cullin-RING ligase-mediated protein turnover leading to apoptotic death in human tumor cells by a new mechanism of action, the deregulation of S-phase DNA synthesis.  MLN4924 suppressed the growth of human tumor xenografts in mice at compd. exposures that were well tolerated.  Our data suggest that NAE inhibitors may hold promise for the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8fx7OnZtYjbVg90H21EOLACvtfcHk0lgRxUT9B3k4fQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXksVenu7Y%253D&md5=365d4b4feac3994849050e00f1aae5e0</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1038%2Fnature07884&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature07884%26sid%3Dliteratum%253Aachs%26aulast%3DSoucy%26aufirst%3DT.%2BA.%26aulast%3DSmith%26aufirst%3DP.%2BG.%26aulast%3DMilhollen%26aufirst%3DM.%2BA.%26aulast%3DBerger%26aufirst%3DA.%2BJ.%26aulast%3DGavin%26aufirst%3DJ.%2BM.%26aulast%3DAdhikari%26aufirst%3DS.%26aulast%3DBrownell%26aufirst%3DJ.%2BE.%26aulast%3DBurke%26aufirst%3DK.%2BE.%26aulast%3DCardin%26aufirst%3DD.%2BP.%26aulast%3DCritchley%26aufirst%3DS.%26aulast%3DCullis%26aufirst%3DC.%2BA.%26aulast%3DDoucette%26aufirst%3DA.%26aulast%3DGarnsey%26aufirst%3DJ.%2BJ.%26aulast%3DGaulin%26aufirst%3DJ.%2BL.%26aulast%3DGershman%26aufirst%3DR.%2BE.%26aulast%3DLublinsky%26aufirst%3DA.%2BR.%26aulast%3DMcDonald%26aufirst%3DA.%26aulast%3DMizutani%26aufirst%3DH.%26aulast%3DNarayanan%26aufirst%3DU.%26aulast%3DOlhava%26aufirst%3DE.%2BJ.%26aulast%3DPeluso%26aufirst%3DS.%26aulast%3DRezaei%26aufirst%3DM.%26aulast%3DSintchak%26aufirst%3DM.%2BD.%26aulast%3DTalreja%26aufirst%3DT.%26aulast%3DThomas%26aufirst%3DM.%2BP.%26aulast%3DTraore%26aufirst%3DT.%26aulast%3DVyskocil%26aufirst%3DS.%26aulast%3DWeatherhead%26aufirst%3DG.%2BS.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DDick%26aufirst%3DL.%2BR.%26aulast%3DClaiborne%26aufirst%3DC.%2BF.%26aulast%3DRolfe%26aufirst%3DM.%26aulast%3DBolen%26aufirst%3DJ.%2BB.%26aulast%3DLangston%26aufirst%3DS.%2BP.%26atitle%3DAn%2520inhibitor%2520of%2520NEDD8-activating%2520enzyme%2520as%2520a%2520new%2520approach%2520to%2520treat%2520cancer%26jtitle%3DNature%26date%3D2009%26volume%3D458%26spage%3D732%26epage%3D736%26doi%3D10.1038%2Fnature07884" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Frost, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galdeano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soares, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadd, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grzes, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Epemolu, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimamura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantrell, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rocha, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Potent and selective chemical probe of hypoxic signalling downstream of HIF-alpha hydroxylation via VHL inhibition</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">13312</span> <span class="refDoi"> DOI: 10.1038/ncomms13312</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=10.1038%2Fncomms13312" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=27811928" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVSitLfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&author=J.+Frostauthor=C.+Galdeanoauthor=P.+Soaresauthor=M.+S.+Gaddauthor=K.+M.+Grzesauthor=L.+Ellisauthor=O.+Epemoluauthor=S.+Shimamuraauthor=M.+Bantscheffauthor=P.+Grandiauthor=K.+D.+Readauthor=D.+A.+Cantrellauthor=S.+Rochaauthor=A.+Ciulli&title=Potent+and+selective+chemical+probe+of+hypoxic+signalling+downstream+of+HIF-alpha+hydroxylation+via+VHL+inhibition&doi=10.1038%2Fncomms13312"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Potent and selective chemical probe of hypoxic signalling downstream of HIF-α hydroxylation via VHL inhibition</span></div><div class="casAuthors">Frost, Julianty; Galdeano, Carles; Soares, Pedro; Gadd, Morgan S.; Grzes, Katarzyna M.; Ellis, Lucy; Epemolu, Ola; Shimamura, Satoko; Bantscheff, Marcus; Grandi, Paola; Read, Kevin D.; Cantrell, Doreen A.; Rocha, Sonia; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">13312</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Chem. strategies to using small mols. to stimulate hypoxia inducible factors (HIFs) activity and trigger a hypoxic response under normoxic conditions, such as iron chelators and inhibitors of prolyl hydroxylase domain (PHD) enzymes, have broad-spectrum activities and off-target effects.  Here we disclose VH298, a potent VHL inhibitor that stabilizes HIF-α and elicits a hypoxic response via a different mechanism, that is the blockade of the VHL:HIF-α protein-protein interaction downstream of HIF-α hydroxylation by PHD enzymes.  We show that VH298 engages with high affinity and specificity with VHL as its only major cellular target, leading to selective on-target accumulation of hydroxylated HIF-α in a concn.- and time-dependent fashion in different cell lines, with subsequent upregulation of HIF-target genes at both mRNA and protein levels.  VH298 represents a high-quality chem. probe of the HIF signalling cascade and an attractive starting point to the development of potential new therapeutics targeting hypoxia signalling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLB1WvVF4EFLVg90H21EOLACvtfcHk0lgRxUT9B3k4fQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVSitLfJ&md5=7a884b6193b3ca5e119fecea2eb31d3e</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1038%2Fncomms13312&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms13312%26sid%3Dliteratum%253Aachs%26aulast%3DFrost%26aufirst%3DJ.%26aulast%3DGaldeano%26aufirst%3DC.%26aulast%3DSoares%26aufirst%3DP.%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DGrzes%26aufirst%3DK.%2BM.%26aulast%3DEllis%26aufirst%3DL.%26aulast%3DEpemolu%26aufirst%3DO.%26aulast%3DShimamura%26aufirst%3DS.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DRead%26aufirst%3DK.%2BD.%26aulast%3DCantrell%26aufirst%3DD.%2BA.%26aulast%3DRocha%26aufirst%3DS.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DPotent%2520and%2520selective%2520chemical%2520probe%2520of%2520hypoxic%2520signalling%2520downstream%2520of%2520HIF-alpha%2520hydroxylation%2520via%2520VHL%2520inhibition%26jtitle%3DNat.%2520Commun.%26date%3D2016%26volume%3D7%26doi%3D10.1038%2Fncomms13312" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4I22" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4I22','PDB','4I22'); return false;">PDB: 4I22</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i45"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01566">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_48167"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.9b01566?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01566</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline"><sup>1</sup>H NMR, <sup>13</sup>C NMR, and HPLC spectra of compounds <b>6</b>, <b>10</b>, <b>27</b>, and <b>28</b>; <sup>1</sup>H NMR and HPLC spectra of compounds <b>1</b>–<b>5</b> and <b>7</b>–<b>9</b>; illustration of binding of gefitinib to the EGFR kinase domain based on the cocrystal structure of the EGFR kinase domain in complex with gefitinib (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4I22">4I22</a>); and expression levels of VHL and CRBN in HCC-827 (EGFR e19d), H3255 (EGFR L858R), OVCAR-8 (EGFR WT), and H1299 (EGFR WT) cells (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01566/suppl_file/jm9b01566_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings for all compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01566/suppl_file/jm9b01566_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01566/suppl_file/jm9b01566_si_001.pdf">jm9b01566_si_001.pdf (3.89 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01566/suppl_file/jm9b01566_si_002.csv">jm9b01566_si_002.csv (2.56 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b01566%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-3%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b01566" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                8MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67999d677eeb124c","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
