
<html lang="en"     class="pb-page"  data-request-id="4bf255ea-39e7-4007-b131-a015eceb69ab"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";issue:issue:10.1021/jmcmar.2018.61.issue-4;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;article:article:10.1021/acs.jmedchem.7b01422;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Discovery of Bisubstrate Inhibitors of Nicotinamide N-Methyltransferase (NNMT)" /></meta><meta name="dc.Creator" content="Nicolas  Babault" /></meta><meta name="dc.Creator" content="Abdellah  Allali-Hassani" /></meta><meta name="dc.Creator" content="Fengling  Li" /></meta><meta name="dc.Creator" content="Jie  Fan" /></meta><meta name="dc.Creator" content="Alex  Yue" /></meta><meta name="dc.Creator" content="Kevin  Ju" /></meta><meta name="dc.Creator" content="Feng  Liu" /></meta><meta name="dc.Creator" content="Masoud  Vedadi" /></meta><meta name="dc.Creator" content="Jing  Liu" /></meta><meta name="dc.Creator" content="Jian  Jin" /></meta><meta name="dc.Description" content="Nicotinamide N-methyltransferase (NNMT) catalyzes the N-methylation of pyridine-containing compounds using the cofactor S-5′-adenosyl-l-methionine (SAM) as the methyl group donor. Through the regul..." /></meta><meta name="Description" content="Nicotinamide N-methyltransferase (NNMT) catalyzes the N-methylation of pyridine-containing compounds using the cofactor S-5′-adenosyl-l-methionine (SAM) as the methyl group donor. Through the regul..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="January 31, 2018" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.7b01422" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2018 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b01422" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.7b01422" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b01422" /></link>
        
    
    

<title>Discovery of Bisubstrate Inhibitors of Nicotinamide N-Methyltransferase (NNMT) | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.7b01422" /></meta><meta property="og:title" content="Discovery of Bisubstrate Inhibitors of Nicotinamide N-Methyltransferase (NNMT)" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-4/acs.jmedchem.7b01422/20180215/images/large/jm-2017-01422z_0007.jpeg" /></meta><meta property="og:description" content="Nicotinamide N-methyltransferase (NNMT) catalyzes the N-methylation of pyridine-containing compounds using the cofactor S-5′-adenosyl-l-methionine (SAM) as the methyl group donor. Through the regulation of the levels of its substrates, cofactor, and products, NNMT plays an important role in physiology and pathophysiology. Overexpression of NNMT has been implicated in various human diseases. Potent and selective small-molecule NNMT inhibitors are valuable chemical tools for testing biological and therapeutic hypotheses. However, very few NNMT inhibitors have been reported. Here, we describe the discovery of a bisubstrate NNMT inhibitor MS2734 (6) and characterization of this inhibitor in biochemical, biophysical, kinetic, and structural studies. Importantly, we obtained the first crystal structure of human NNMT in complex with a small-molecule inhibitor. The structure of the NNMT–6 complex has unambiguously demonstrated that 6 occupied both substrate and cofactor binding sites. The findings paved the way for developing more potent and selective NNMT inhibitors in the future." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.7b01422"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.7b01422">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.7b01422&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.7b01422&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.7b01422&amp;href=/doi/10.1021/acs.jmedchem.7b01422" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2018</span><span class="cit-fg-volume">, 61</span><span class="cit-fg-issue">, 4</span><span class="cit-fg-pageRange">, 1541-1551</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/61/4" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.7b01407" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.7b01465" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="article_relation"><span class="article_relation-correction"><i class="icon-asterisk"></i><span class="relation-text"><a href="/doi/10.1021/acs.jmedchem.8b00849"><span class="relation-type">ADDITION / CORRECTION</span><span class="relation-desc">This article has been corrected. View the notice.</span></a></span></span></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of Bisubstrate Inhibitors of Nicotinamide <i>N</i>-Methyltransferase (NNMT)</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Nicolas++Babault">Nicolas Babault</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Abdellah++Allali-Hassani">Abdellah Allali-Hassani</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Fengling++Li">Fengling Li</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jie++Fan">Jie Fan</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Alex++Yue">Alex Yue</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Kevin++Ju">Kevin Ju</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Feng++Liu">Feng Liu</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0003-2669-5448" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Masoud++Vedadi">Masoud Vedadi</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jing++Liu">Jing Liu</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jian++Jin">Jian Jin</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0002-2387-3862" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Center for Chemical Biology and Drug Discovery, Departments of Pharmacological Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">§</span> <span class="aff-text">Accutar Biotechnology, Brooklyn, New York 11226, United States</span></div><div class="aff-info" id="aff4"><span class="aff-symbol">∥</span> <span class="aff-text">Jiangsu Key Laboratory of Translational Research and Therapy for Neuro-Psycho-Diseases and Department of Medicinal Chemistry, College of Pharmaceutical Sciences, Soochow University, Suzhou, Jiangsu 215123, People’s Republic of China</span></div><div class="aff-info" id="aff5"><span class="aff-symbol">⊥</span> <span class="aff-text">Department of Pharmacology and Toxicology, University of Toronto, Toronto, Ontario M5S 1A8, Canada</span></div><div class="corresp-info"><strong>*</strong>Phone: (212) 659-8699. E-mail: <a href="/cdn-cgi/l/email-protection#d1bbb8b0bfffbbb8bf91bca2a2bcffb4b5a4"><span class="__cf_email__" data-cfemail="771d1e1619591d1e19371a04041a59121302">[email protected]</span></a></div><div class="corresp-info"><strong>*</strong>Phone: (212) 659-5499. E-mail: <a href="/cdn-cgi/l/email-protection#4e042720296022273b0e233d3d23602b2a3b"><span class="__cf_email__" data-cfemail="53193a3d347d3f3a26133e20203e7d363726">[email protected]</span></a></div><div class="corresp-info"><strong>*</strong>Phone: (416) 976-0897. E-mail: <a href="/cdn-cgi/l/email-protection#81ecaff7e4e5e0e5e8c1f4f5eef3eeeff5eeafe2e0"><span class="__cf_email__" data-cfemail="aec380d8cbcacfcac7eedbdac1dcc1c0dac180cdcf">[email protected]</span></a></div><div class="corresp-info"><strong>*</strong>Phone: 86-(0512)-65882569. E-mail: <a href="/cdn-cgi/l/email-protection#77111b1e0245370402131659121302591419"><span class="__cf_email__" data-cfemail="6402080d115624171100054a0100114a070a">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b01422&amp;href=/doi/10.1021%2Facs.jmedchem.7b01422" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2018</span></span><span class="cit-volume">, 61</span><span class="cit-issue">, 4</span><span class="cit-pageRange">, 1541–1551</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">January 10, 2018</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>25 September 2017</li><li><span class="item_label"><b>Published</b> online</span>31 January 2018</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 22 February 2018</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.7b01422" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b01422</a></div><div class="article_header-article-copyright"><strong>Copyright © 2018 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D1541%26pageCount%3D11%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DNicolas%2BBabault%252C%2BAbdellah%2BAllali-Hassani%252C%2BFengling%2BLi%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D61%26issueNum%3D4%26contentID%3Dacs.jmedchem.7b01422%26title%3DDiscovery%2Bof%2BBisubstrate%2BInhibitors%2Bof%2BNicotinamide%2BN-Methyltransferase%2B%2528NNMT%2529%26numPages%3D11%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D1551%26publicationDate%3DFebruary%2B2018">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.7b01422"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">3457</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">21</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.7b01422" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of Bisubstrate Inhibitors of Nicotinamide N-Methyltransferase (NNMT)&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Nicolas&quot;,&quot;last_name&quot;:&quot;Babault&quot;},{&quot;first_name&quot;:&quot;Abdellah&quot;,&quot;last_name&quot;:&quot;Allali-Hassani&quot;},{&quot;first_name&quot;:&quot;Fengling&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Jie&quot;,&quot;last_name&quot;:&quot;Fan&quot;},{&quot;first_name&quot;:&quot;Alex&quot;,&quot;last_name&quot;:&quot;Yue&quot;},{&quot;first_name&quot;:&quot;Kevin&quot;,&quot;last_name&quot;:&quot;Ju&quot;},{&quot;first_name&quot;:&quot;Feng&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Masoud&quot;,&quot;last_name&quot;:&quot;Vedadi&quot;},{&quot;first_name&quot;:&quot;Jing&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Jian&quot;,&quot;last_name&quot;:&quot;Jin&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2018&quot;,&quot;month&quot;:&quot;01&quot;,&quot;day&quot;:&quot;31&quot;,&quot;issue&quot;:&quot;4&quot;,&quot;volume&quot;:&quot;61&quot;,&quot;pages&quot;:&quot;1541-1551&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.7b01422&quot;},&quot;abstract&quot;:&quot;Nicotinamide N-methyltransferase (NNMT) catalyzes the N-methylation of pyridine-containing compounds using the cofactor S-5′-adenosyl-l-methionine (SAM) as the methyl group donor. Through the regulation of the levels of its substrates, cofactor, and products, NNMT plays an important role in physiology and pathophysiology. Overexpression of NNMT has been implicated in various human diseases. Potent and selective small-molecule NNMT inhibitors are valuable chemical tools for testing biological and therapeutic hypotheses. However, very few NNMT inhibitors have been reported. Here, we describe the discovery of a bisubstrate NNMT inhibitor MS2734 (6) and characterization of this inhibitor in biochemical, biophysical, kinetic, and structural studies. Importantly, we obtained the first crystal structure of human NNMT in complex with a small-molecule inhibitor. The structure of the NNMT–6 complex has unambiguously demonstrated that 6 occupied both substrate and cofactor binding sites. The findings paved the way for d&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b01422&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01422" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b01422&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01422" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b01422&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01422" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b01422&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01422&amp;href=/doi/10.1021/acs.jmedchem.7b01422" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.7b01422" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.7b01422" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b01422&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.7b01422%26sid%3Dliteratum%253Aachs%26pmid%3D29320176%26genre%3Darticle%26aulast%3DBabault%26date%3D2018%26atitle%3DDiscovery%2Bof%2BBisubstrate%2BInhibitors%2Bof%2BNicotinamide%2BN-Methyltransferase%2B%2528NNMT%2529%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D61%26issue%3D4%26spage%3D1541%26epage%3D1551%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291880" title="Crystal structure">Crystal structure</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291071" title="Nicotinamide">Nicotinamide</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/61/4" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-4/jmcmar.2018.61.issue-4/20180222/jmcmar.2018.61.issue-4.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-4/acs.jmedchem.7b01422/20180215/images/medium/jm-2017-01422z_0007.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-4/acs.jmedchem.7b01422/20180215/images/large/jm-2017-01422z_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01422&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Nicotinamide <i>N</i>-methyltransferase (NNMT) catalyzes the N-methylation of pyridine-containing compounds using the cofactor <i>S</i>-5′-adenosyl-<span class="smallcaps smallerCapital">l</span>-methionine (SAM) as the methyl group donor. Through the regulation of the levels of its substrates, cofactor, and products, NNMT plays an important role in physiology and pathophysiology. Overexpression of NNMT has been implicated in various human diseases. Potent and selective small-molecule NNMT inhibitors are valuable chemical tools for testing biological and therapeutic hypotheses. However, very few NNMT inhibitors have been reported. Here, we describe the discovery of a bisubstrate NNMT inhibitor MS2734 (<b>6</b>) and characterization of this inhibitor in biochemical, biophysical, kinetic, and structural studies. Importantly, we obtained the first crystal structure of human NNMT in complex with a small-molecule inhibitor. The structure of the NNMT–<b>6</b> complex has unambiguously demonstrated that <b>6</b> occupied both substrate and cofactor binding sites. The findings paved the way for developing more potent and selective NNMT inhibitors in the future.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_84700" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_84700" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Nicotinamide <i>N</i>-methyltransferase (NNMT) catalyzes the transfer of the methyl group from the cofactor <i>S</i>-5′-adenosyl-<span class="smallcaps smallerCapital">l</span>-methionine (SAM) to the N1-positions of nicotinamide and a variety of structurally related pyridine-containing compounds, such as thionicotinamide, quinoline, isoquinoline, 3-acetylpyridine, and others.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1, 2)</a> The NNMT gene is predominantly expressed in liver, and significant expression levels of NNMT are detected in adipose tissue, kidney, lung, skeletal muscle, placenta, and heart.<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3, 4)</a> The function of NNMT has traditionally been assigned to the nicotinamide clearance and xenobiotic detoxification.<a onclick="showRef(event, 'ref1 ref5'); return false;" href="javascript:void(0);" class="ref ref1 ref5">(1, 5)</a> Recently, more NNMT functions have been revealed in both health and diseases through modulation of the levels of its products, substrates, and cofactor.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Several beneficial effects of NNMT activity have been reported through the generation of the endogenous product N1-methyl nicotinamide (<b>1</b>). For example, <b>1</b> up-regulates the reactive oxygen species (ROS) and leads to lifespan extension in <i>Caenorhabditis elegans</i>.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a><b>1</b> also increases the secretion of prostacyclin from endothelial cells to regulate thrombotic and inflammatory processes.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> In addition, <b>1</b> enhances the release of nitric oxide from endothelial cells and leads to vasodilation of blood vessels.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Moreover, <b>1</b> stabilizes Sirt1 protein in liver, decreases hepatocyte fatty acid and cholesterol synthesis, and suppresses liver triglyceride, cholesterol content, and liver inflammation.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> However, other NNMT metabolized N-methylated pyridinium products, such as <i>N</i>-methyl-4-phenylpyridinium ion, are well-established dopaminergic toxins.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> These pyridinium metabolites have been deemed as the causative agents for Parkinson’s disease.<a onclick="showRef(event, 'ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref10 ref11">(10, 11)</a> Indeed, NNMT expression and activity are elevated in the brain tissues of Parkinson’s disease patients.<a onclick="showRef(event, 'ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref10 ref11">(10, 11)</a> To date, the best-studied NNMT substrate is nicotinamide, which is a biosynthetic precursor of nicotinamide adenine dinucleotide (NAD<sup>+</sup>), an important molecule involved in redox reactions and posttranslational modifications. The cofactor of NNMT, SAM, provides propylamine groups for polyamine biosynthesis. SAM is also the common methyl group donor for a variety of methyltransferases. By direct alteration of NAD<sup>+</sup> and SAM levels, NNMT regulates the energy metabolism and functions as a regulator of adiposity.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> NNMT expression has been observed in white adipose tissue (WAT) of obese and diabetic mice. Knockdown NNMT with antisense oligonucleotides (ASOs) increases energy expenditure and prevents weight gain in high-fat diet mice.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Down-regulations of the SAM level and consequently the methylation states of histones and nonhistone proteins have been demonstrated as the pathological mechanism of NNMT overexpression in cancers.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> NNMT activity is elevated in a diverse set of cancers, including brain,<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> thyroid,<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> lung,<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> liver,<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> kidney,<a onclick="showRef(event, 'ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref18 ref19">(18, 19)</a> stomach,<a onclick="showRef(event, 'ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref20 ref21">(20, 21)</a> bladder,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> pancreas,<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> oral,<a onclick="showRef(event, 'ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref24 ref25">(24, 25)</a> and colon.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> NNMT knockdown decreases cell proliferation, migration, and/or metastasis of bladder,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> kidney,<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> pancreas,<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> and oral<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> cancer cells. Although the endogenous roles of NNMT in physiology and disease seem to be contradicting and need to be further investigated, NNMT could be a potential therapeutic target in diseases with abnormally high expression of NNMT.</div><div class="NLM_p">Potent and selective small-molecule inhibitors of NNMT are valuable chemical tools for deciphering the complex regulatory mechanisms mediated by NNMT and for testing biological and therapeutic hypotheses associated with NNMT. However, only very few NNMT inhibitors have been reported to date (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>).<a onclick="showRef(event, 'ref3 ref27 ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref3 ref27 ref28 ref29">(3, 27-29)</a><b>1</b>,<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> the product of the NNMT enzymatic reaction, has been widely used as an NNMT inhibitor to study the biological function of NNMT.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Since last year, three types of new NNMT inhibitors have been published. An NNMT covalent inhibitor, RS004 (<b>2</b>),<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> was reported targeting the Cys165 residue within the SAM-binding pocket, and N-methylated quinolinium compounds <b>3</b> and <b>4</b><a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> were disclosed as substrate mimetic inhibitors. During the preparation of this manuscript, a bisubstrate NNMT inhibitor, <b>5</b>,<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> was reported targeting both substrate and cofactor binding sites. However, <b>5</b> displayed modest potency (IC<sub>50</sub> = 29 μM) in a biochemical assay and was not characterized in biophysical assays and selectivity assays. In fact, except for the covalent inhibitor <b>2</b>, no selectivity profiling data were provided for the reported NNMT inhibitors. In addition, no co-crystal structures of these NNMT inhibitors in complex with human NNMT (hNNMT) have been reported.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-4/acs.jmedchem.7b01422/20180215/images/medium/jm-2017-01422z_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-4/acs.jmedchem.7b01422/20180215/images/large/jm-2017-01422z_0008.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Chemical structures of NNMT inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-4/acs.jmedchem.7b01422/20180215/images/large/jm-2017-01422z_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01422&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">A bisubstrate inhibitor consists of two covalently linked fragments, each targeting either the substrate binding site or the cofactor binding site, thus potentially mimicking the ternary transition state of a bireactant enzymatic reaction. This inhibitor design strategy has the potential to achieve high potency and selectivity. A number of bisubstrate inhibitors have been successfully developed for a variety of enzymes, including kinases<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> and acetyltransferases.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> Inspired by the recently published ternary crystal structure of hNNMT in complex with the substrate nicotinamide and the cofactor product <i>S</i>-5′-adenosyl-<span class="smallcaps smallerCapital">l</span>-homocysteine (SAH),<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> we designed and synthesized NNMT bisubstrate inhibitors, MS2734 (<b>6</b>) and MS2756 (<b>7</b>). We characterized these compounds in a battery of biochemical and biophysical assays. In addition, using the bisubstrate inhibitor <b>6</b>, we studied the kinetic mechanism of NNMT inhibition. Furthermore, we obtained a co-crystal structure of <b>6</b> in complex with hNNMT. Here, we report the design, synthesis, and biochemical and biophysical characterization of these bisubstrate inhibitors.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_43433" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_43433" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Design</h3><div class="NLM_p">The crystal structure of the hNNMT–nicotinamide–SAH ternary complex (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ROD">3ROD</a>) revealed that nicotinamide and SAH occupy two adjacent binding sites.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> The distance between the nicotinamide nitrogen atom and the SAH sulfur atom ranges from 3.5 to 4.2 Å.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> We thus hypothesized that by covalently linking the nicotinamide mimic group <b>8</b> and the SAM mimic moieties <b>9</b> using a 2-carbon-atom linker (total linear distance ∼4.5 Å), we could generate compounds <b>6</b> and <b>7</b> as bisubstrate inhibitors of NNMT (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>). We docked compound <b>6</b> into the SAH binding site of the crystal structure of the hNNMT–SAH binary complex (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2IIP">2IIP</a>) by utilizing a side chain flexible docking strategy<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> to probe whether the 3-amido-phenyl moiety of <b>6</b> could bind to the substrate-binding pocket of hNNMT (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>A and the docking process movie is available in the <a href="/doi/suppl/10.1021/acs.jmedchem.7b01422/suppl_file/jm7b01422_si_003.mpg" class="ext-link">SI</a>). Comparing our docking model with the hNNMT–nicotinamide–SAH ternary structure, we found that the 3-amido-phenyl moiety of <b>6</b> does occupy the nicotinamide binding site (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>B). Moreover, the orientation of docked 3-amido-phenyl group is almost identical to that of nicotinamide in one of the four ternary structures (chain B) in the published asymmetric unit (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ROD">3ROD</a>).<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> The oxygen atom of the 3-amido group interacts with Ser201 and Ser213 residues through hydrogen bonds (H-bonds), and the nitrogen atom of the 3-amido group forms a H-bond with Ser213 (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>A). We also conducted similar docking studies for compound 7 (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01422/suppl_file/jm7b01422_si_001.pdf" class="ext-link">Supporting Information</a>, Figure S1), which suggested that compound <b>7</b> can also fit into the two adjacent binding sites, but is less favorable compared with compound <b>6</b>.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-4/acs.jmedchem.7b01422/20180215/images/medium/jm-2017-01422z_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-4/acs.jmedchem.7b01422/20180215/images/large/jm-2017-01422z_0001.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Design of the bisubstrate NNMT inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-4/acs.jmedchem.7b01422/20180215/images/large/jm-2017-01422z_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01422&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-4/acs.jmedchem.7b01422/20180215/images/medium/jm-2017-01422z_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-4/acs.jmedchem.7b01422/20180215/images/large/jm-2017-01422z_0002.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Docking analysis of compound <b>6</b> in the binding sites of hNNMT. (A) Docking of compound <b>6</b> (green) in the hNNMT (cyan) structure (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2IIP">2IIP</a>). H-bond interactions are shown in yellow dotted lines. Water molecules are illustrated as red balls. (B) Overlay of the docking model with the hNNMT (gray)–nicotinamide (magenta)–SAH (yellow) complex (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ROD">3ROD</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-4/acs.jmedchem.7b01422/20180215/images/large/jm-2017-01422z_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01422&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Synthesis</h3><div class="NLM_p">Synthesis of compounds <b>6</b> and <b>7</b> started with the preparation of intermediate <b>12</b> from the reductive amination of commercially available starting materials <b>10</b> and 3-(2-oxoethyl)benzonitrile <b>11</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>).<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> Coupling of intermediate <b>12</b> with aldehydes <b>13</b><a onclick="showRef(event, 'ref35 ref36'); return false;" href="javascript:void(0);" class="ref ref35 ref36">(35, 36)</a> via another reductive amination reaction provided the nitrile intermediates <b>14</b>. After hydrolysis of the cyano group to the amido group under basic condition in the presence of hydrogen peroxide, the protecting groups on the resulting intermediates <b>15</b> were removed to provide the final products <b>6</b> and <b>7</b>.</div><figure id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-4/acs.jmedchem.7b01422/20180215/images/medium/jm-2017-01422z_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-4/acs.jmedchem.7b01422/20180215/images/large/jm-2017-01422z_0006.jpeg" id="GRAPHIC-d345e721-autogenerated" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthetic route for <b>6</b> and <b>7</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-4/acs.jmedchem.7b01422/20180215/images/large/jm-2017-01422z_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01422&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) NaBH(OAc)<sub>3</sub>, AcOH, DCM, rt, 16 h, 61–71%; (b) K<sub>2</sub>CO<sub>3</sub>, DMSO, H<sub>2</sub>O<sub>2</sub>, 0 °C to rt, 3 h; (c) HCl in dioxane, 0 °C to rt, 4 h; then 2 drops of water, rt, 4 h, 46–57% over 2 steps.</p></p></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Biochemical and Biophysical Characterization</h3><div class="NLM_p">To evaluate the biological activities of compounds <b>6</b> and <b>7</b>, we used a SAH hydrolase (SAHH)-coupled fluorescent assay.<a onclick="showRef(event, 'ref37 ref38'); return false;" href="javascript:void(0);" class="ref ref37 ref38">(37, 38)</a> SAHH hydrolyzes SAH, the NNMT cofactor product, to adenosine and homocysteine. The free thiol group of homocysteine conjugates to a thiol-sensitive fluorophore (ThioGlo) and generates fluorescence signal. Thus, the methylation reaction can be followed by measuring the increase of the fluorescence intensity. We performed full kinetic characterization of NNMT using SAHH-coupled assay and determined <i>K</i><sub>m</sub> values for nicotinamide (20 ± 3 μM) and SAM (24 ± 6.8 μM) with a <i>k</i><sub>cat</sub> of 4.1 ± 0.2 min<sup>–1</sup> (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01422/suppl_file/jm7b01422_si_001.pdf" class="ext-link">Supporting Information</a>, Figure S3). The <i>K</i><sub>m</sub> value we obtained for nicotinamide was lower than what has been reported in the literature using end-point assays.<a onclick="showRef(event, 'ref2 ref3 ref28 ref29 ref32'); return false;" href="javascript:void(0);" class="ref ref2 ref3 ref28 ref29 ref32">(2, 3, 28, 29, 32)</a> The differences are likely due to using different assay conditions such as temperature, pH, and additives.</div><div class="NLM_p">Under the SAHH-coupled assay conditions, both compounds <b>6</b> and <b>7</b> inhibited the methyltransferase activity of NNMT with IC<sub>50</sub> values of 14 ± 1.5 μM (Hill Slope: 1.1) and 160 ± 1 μM, respectively (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>A). Next, we confirmed the binding of compounds <b>6</b> and <b>7</b> to hNNMT using isothermal titration calorimetry (ITC). Consistent with the SAR trend from the biochemical assay, compound <b>6</b> showed better binding affinity (<i>K</i><sub><i>d</i></sub> = 2.7 ± 0.2 μM) than compound <b>7</b> (<i>K</i><sub><i>d</i></sub> = 42.8 ± 6.3 μM) (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>B). These assay results indicate that the length of the amino acid moiety of the bisubstrate inhibitors is important for NNMT potency and binding affinity. These assay results support the prediction from our docking studies that compound <b>7</b> is less favorable to bind the NNMT substrate and cofactor binding site compared with compound <b>6</b>.</div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-4/acs.jmedchem.7b01422/20180215/images/medium/jm-2017-01422z_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-4/acs.jmedchem.7b01422/20180215/images/large/jm-2017-01422z_0003.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Biochemical and biophysical characterization of <b>6</b> and <b>7</b> and selectivity assessment of <b>6</b>. (A) IC<sub>50</sub> determination for <b>6</b> (14 ± 1.5 μM; HS: 1.1) and <b>7</b> (160 ± 1 μM) against hNNMT (<i>n</i> = 3). (B) ITC was used to assess binding of hNNMT to <b>6</b> (left) and <b>7</b> (right). (C) Selectivity of <b>6</b> was determined against a panel of 34 MTs and 1 HAT at two compound concentrations (10 μM (blue) and 50 μM (red)). (D) IC<sub>50</sub> determination for <b>6</b> against DOT1L (1.3 ± 0.2 μM; HS: 1.1), PRMT7 (20 ± 2 μM; HS: 1.1), BCDIN3D (40 ± 2 μM; HS: 1.3), and SMYD2 (62 ± 7 μM; HS: 0.6) were performed in triplicate. HS: Hill slope.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-4/acs.jmedchem.7b01422/20180215/images/large/jm-2017-01422z_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01422&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Selectivity Assessment</h3><div class="NLM_p">To assess selectivity of compound <b>6</b>, we tested it against 34 histone, DNA and RNA methyltransferases, and a histone acetyltransferase (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01422/suppl_file/jm7b01422_si_001.pdf" class="ext-link">Supporting Information</a>, Table S1). We were pleased to find that compound <b>6</b> did not inhibit the majority of these enzymes at up to 50 μM (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>C). Out of these 35 examined enzymes, only four enzymes (DOT1L, PRMT7, BCDIN3D, and SMYD2) were significantly inhibited with IC<sub>50</sub> and Hill slope (HS) values of 1.3 ± 0.2 μM (HS: 1.1), 20 ± 2 μM (HS: 1.1), 40 ± 2 μM (HS: 1.3), and 62 ± 7 μM (HS: 0.6), respectively (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>D). The selectivity profiling of <b>6</b> over other methyltransferases in general indicates that the bisubstrate inhibitor design strategy could provide potent and selective NNMT inhibitors. Interestingly, <b>6</b> inhibited DOT1L with low μM potency. This result is consistent with previous reports that SAM analogs are effective DOT1L inhibitors.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> In addition, <b>6</b> could serve as a potential starting point for developing selective inhibitors of PRMT7 and BCDIN3D as no selective inhibitors of PRMT7 and BCDIN3D have been reported to date.</div><div class="NLM_p last">To gain insights into the structural basis of <b>6</b> binding to these off-targets, we docked compound <b>6</b> into two previously published crystal structures: DOT1L (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3QOW">3QOW</a>)<a onclick="showRef(event, 'ref100'); return false;" href="javascript:void(0);" class="ref ref100">(39)</a> and SMYD2 (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3TG4">3TG4</a>)<a onclick="showRef(event, 'ref101'); return false;" href="javascript:void(0);" class="ref ref101">(40)</a> (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01422/suppl_file/jm7b01422_si_001.pdf" class="ext-link">Supporting Information</a>, Figure S2). Our docking models suggest that the amidophenyl group of <b>6</b> occupies the substrate binding sites of both DOT1L and SMYD2, which is consistent with our selectivity assay results. It appears that, although the amino acid residues and the size and shape of the substrate binding sites of NNMT, DOT1L, and SMYD2 are different, these substrate binding sites can accommodate a small moiety such as the amidophenyl group of inhibitor <b>6</b>. We believe it is possible to achieve selectivity among these targets by modifying the amidophenyl group.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Mechanism of Action (MOA) Studies</h3><div class="NLM_p">Bisubstrate inhibitors are useful tools for studying enzymatic mechanisms.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> Specifically, kinetic studies of bisubstrate inhibitors could provide valuable information to determine whether the enzymes catalyze the biological reaction through a random or an ordered mechanism.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> To assess the MOA of <b>6</b>, we evaluated the effect of SAM and nicotinamide concentrations on the rate of the enzymatic reaction at various concentrations of the bisubstrate inhibitor <b>6</b>. As illustrated in <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>A, the bisubstrate inhibitor <b>6</b> is SAM competitive as evidenced by changes in <i>K</i><sub>m</sub> values for SAM but no change in <i>V</i><sub>max</sub> values at various concentrations of the inhibitor (the 1/rate versus 1/[SAM] intersecting on the <i>y</i>-axis of the Lineweaver–Burk plot). As illustrated in <a class="ref internalNav" href="#fig5" aria-label="Figures 5">Figures 5</a>B, <b>6</b> is noncompetitive with the nicotinamide substrate, with no change in <i>K</i><sub>m</sub> values for nicotinamide but significant changes in <i>V</i><sub>max</sub> values as the concentration of <b>6</b> was increased (1/rate versus 1/[nicotinamide] intersecting on the <i>x</i>-axis of the Lineweaver–Burk plot). Although both SAM (<i>K</i><sub><i>d</i></sub> = 56.4 ± 9.5 μM) and compound <b>6 (</b><i>K</i><sub><i>d</i></sub> = 2.7 ± 0.2 μM) bound to hNNMT in ITC experiments (<a class="ref internalNav" href="#fig4" aria-label="Figures 4">Figures 4</a>B and <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>C), reliably measurable nicotinamide binding to hNNMT was not observed in the absence of the cofactor (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>D). In the presence of SAH, however, nicotinamide was able to bind to hNNMT (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01422/suppl_file/jm7b01422_si_001.pdf" class="ext-link">Supporting Information</a>, Figure S4). These binding phenomena suggest a possible ordered mechanism for hNNMT enzymatic activity, where SAM binds first, followed by nicotinamide binding. This is consistent with previously reported competitive patterns that indicate an ordered enzymatic kinetic mechanism.<a onclick="showRef(event, 'ref31 ref39'); return false;" href="javascript:void(0);" class="ref ref31 ref39">(31, 41)</a> However, recently a random mechanism has been reported for NNMT activity,<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(42)</a> where SAM and quinoline would independently bind to NNMT. The authors also observed that both SAM and quinoline bound to the complementary binary complexes with a much higher affinity (20-fold) compared to binding to the apoenzyme.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(42)</a> This may be consistent with the nicotinamide binding we observed in the absence and presence of SAH (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>D and <a href="/doi/suppl/10.1021/acs.jmedchem.7b01422/suppl_file/jm7b01422_si_001.pdf" class="ext-link">Supporting Information</a>, Figure S4).</div><figure id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-4/acs.jmedchem.7b01422/20180215/images/medium/jm-2017-01422z_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-4/acs.jmedchem.7b01422/20180215/images/large/jm-2017-01422z_0004.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. MOA study of NNMT inhibition with <b>6</b>. (A) Lineweaver–Burk plot of 1/rate versus 1/[SAM] with varying concentrations of inhibitor <b>6</b> showing competitive inhibition and (B) Lineweaver–Burk plot of 1/rate versus 1/[nicotinamide] with varying concentrations of inhibitor <b>6</b> showing noncompetitive inhibition. Eight concentrations of inhibitor <b>6</b> (from 2 to 250 μM) were used for both experiments: 2 μM (●), 4 μM (○), 8 μM (▲), 16 μM (△), 31 μM (■), 63 μM (□), 125 μM (◆), and 250 μM (◇). ITC was used to assess binding of (C) SAM and (D) nicotinamide to hNNMT.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-4/acs.jmedchem.7b01422/20180215/images/large/jm-2017-01422z_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01422&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Co-crystal Structure of Compound <b>6</b> in Complex with hNNMT</h3><div class="NLM_p">To elucidate the interactions between hNNMT and compound <b>6</b> at an atomic level, we obtained an X-ray co-crystal structure of hNNMT in complex with <b>6</b> at 2.3 Å (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6B1A">6B1A</a>, <a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>A,B, and <a href="/doi/suppl/10.1021/acs.jmedchem.7b01422/suppl_file/jm7b01422_si_001.pdf" class="ext-link">Supporting Information</a>, Table S2). The crystal asymmetric unit contains four independent, conformationally similar NNMT molecules. Overlay of these molecules gives rmsd values ranging from 0.105 to 0.205 Å. Electron density of compound <b>6</b> is unambiguously defined in all of the NNMT molecules. We superimposed our hNNMT–<b>6</b> structure with the published hNNMT–nicotinamide–SAH complex (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>C).<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> Consistent with our bisubstrate inhibitor design hypothesis and docking model study, compound <b>6</b> does occupy both cofactor and substrate binding sites. The cofactor mimicking moiety of compound <b>6</b> overlays nicely with SAH, and the majority of the ligand-protein interactions in this binding pocket are conserved. For example, the carboxylate oxygen atoms form H-bonds with four hydroxyl groups from the side chains of Tyr20, Tyr25, Tyr69, and Thr163. In addition, the ribose hydroxyl groups interact with the side chains of Asp85 and Asn90 through H-bonds. Moreover, the adenine moiety is sandwiched between Tyr86 and Ala169, and the adenine N1 nitrogen forms a H-bond with the main chain amide nitrogen of Val143. Compound <b>6</b> also possesses a few different interactions compared with SAH in the cofactor binding site. While the homocysteine amino group of SAH forms H-bonds with Gly63, Thr163, and a water molecule, the corresponding amino group of <b>6</b> orients to a different direction and locates within the hydrogen bond distance of two additional residues, Ser64 and Thr67. Other different interactions occur at the adenine N6-amino group. Besides the conserved H-bond with the Asp142, compound <b>6</b> indirectly interacts with Ala168 through water mediated H-bonds. The 3-amido phenyl group of compound <b>6</b> nests nicely between Leu164 and Tyr204 at the nicotinamide-binding site. Three important H-bond interactions with Ser201 and Ser203 are conserved. Compared to nicotinamide, the 3-amido phenyl moiety projects a little bit deeper in the substrate-binding site. Therefore, the direct hydrogen bond between the amide nitrogen of nicotinamide and Asp197 is lost. Instead, water-molecule-mediated H-bonds are observed between the amide nitrogen of compound <b>6</b> and three residues (Ala168, Leu164, and Asp197). Since the N1-nitrogen of nicotinamide was removed on <b>6</b>, the interaction between the nicotinamide N1-nitrogen and Tyr20 is lost.</div><figure id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-4/acs.jmedchem.7b01422/20180215/images/medium/jm-2017-01422z_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-4/acs.jmedchem.7b01422/20180215/images/large/jm-2017-01422z_0005.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. X-ray crystal structures. (A) The X-ray crystal structure of a binary complex of <b>6</b> (green) and hNNMT (cyan) (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6B1A">6B1A</a>). The surface of the protein is shown in gray. (B) Key interactions between <b>6</b> and hNNMT. H-bond interactions are shown in yellow dotted lines. Water molecules are illustrated as red balls. (C) Overlay of the crystal structure of the hNNMT (cyan)–<b>6</b> (green) binary complex (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6B1A">6B1A</a>) with the crystal structure of hNNMT (not shown)–nicotinamide (magenta)–SAH (yellow) complex (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ROD">3ROD</a>). (D) Overlay of the crystal structure of the hNNMT (cyan)–<b>6</b> (green) binary complex (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6B1A">6B1A</a>) with the crystal structure of monkey NNMT (not shown)–<b>1</b> (blue)–SAH (yellow) complex (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5XVQ">5XVQ</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-4/acs.jmedchem.7b01422/20180215/images/large/jm-2017-01422z_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01422&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">During the preparation of this manuscript, crystal structures of monkey and mouse NNMT in complex with <b>1</b> and SAH were published.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(43)</a> Our hNNMT–<b>6</b> structure overlays very well with these two new structures with almost identical ligand-protein interactions (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>D and <a href="/doi/suppl/10.1021/acs.jmedchem.7b01422/suppl_file/jm7b01422_si_001.pdf" class="ext-link">Supporting Information</a> Figure S5). Interestingly, while the 3-amido phenyl group of compound <b>6</b> is located deeper in the substrate binding site compared to nicotinamide, this group sits at almost the same location as <b>1</b> in the monkey NNMT–<b>1</b> complex (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>C,D).</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i10">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_27800" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_27800" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">Using a structure-based bisubstrate inhibitor design strategy, we designed and synthesized compounds <b>6</b> and <b>7</b>. In a biochemical assay, compound <b>6</b> inhibited the hNNMT methyltransferase activity with an IC<sub>50</sub> of 14 ± 1.5 μM. We confirmed that compound <b>6</b> was a tight binding ligand for hNNMT using ITC (<i>K</i><sub><i>d</i></sub> = 2.7 ± 0.2 μM). Compound <b>7</b> was less potent than compound <b>6</b> in biochemical and biophysical assays, consistent with our modeling results. Compound <b>6</b> was selective for hNNMT over a broad range of methyltransferases with the exception of DOT1L and PRMT7. In MOA studies, compound <b>6</b> was competitive with the cofactor SAM and noncompetitive with the substrate nicotinamide. We obtained the crystal structure of human NNMT in complex with compound <b>6</b>, which has clearly demonstrated that <b>6</b> occupied both substrate and cofactor binding sites, thus validating our bisubstrate inhibitor design strategy. These results are valuable for the research community to develop the next generation of NNMT inhibitors with improved potency and selectivity.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78601" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78601" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Chemistry General Procedures</h3><div class="NLM_p">HPLC spectra for all compounds were acquired using an Agilent 1200 Series system with DAD detector. Chromatography was performed on a 2.1 × 150 mm Zorbax 300SB-C<sub>18</sub> 5 μm column with water containing 0.1% formic acid as solvent A and acetonitrile containing 0.1% formic acid as solvent B at a flow rate of 0.4 mL/min. The gradient program was as follows: 1% B (0–1 min), 1–99% B (1–4 min), and 99% B (4–8 min). High-resolution mass spectra (HRMS) data were acquired in positive-ion mode using an Agilent G1969A API-TOF with an electrospray ionization (ESI) source. Nuclear Magnetic Resonance (NMR) spectra were acquired on a Bruker DRX-600 spectrometer (600 MHz <sup>1</sup>H, 150 MHz <sup>13</sup>C). Chemical shifts are reported in ppm (δ). Preparative HPLC was performed on Agilent Prep 1200 series with UV detector set to 254 nm. Samples were injected into a Phenomenex Luna 250 × 30 mm, 5 μm, C<sub>18</sub> column at room temperature. The flow rate was 40 mL/min. A linear gradient was used with 10% (or 50%) of MeOH (A) in H<sub>2</sub>O (with 0.1% TFA) (B) to 100% of MeOH (A). HPLC was used to establish the purity of target compounds. All final compounds had >95% purity using the HPLC methods described above.</div><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i13" class="anchor-spacer"></div><h4 class="article-section__title" id="_i13"> 3-(2-((((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-(6-Amino-9<i>H</i>-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl)amino)ethyl)benzonitrile (<b>12</b>)</h4><div class="NLM_p last">To a solution of 3-(2-oxoethyl)benzonitrile <b>11</b> (0.15 g, 1 mmol) in 25 mL DCM was added <b>12</b> (0.46 g, 1.5 mmol) followed by 5 drops of AcOH. The resulting mixture was stirred for 30 min before NaBH(OAc)<sub>3</sub> (0.32 g, 1.5 mmol) was added. After being stirred for 16 h at rt, the reaction was quenched with saturated NaHCO<sub>3</sub> and extracted with EtOAc (5×). The combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. The residue was purified with silica gel column to provide desired product <b>12</b> (0.31 g, 71%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 8.49 (s, 1H), 8.44 (s, 1H), 7.61–7.58 (m, 2H), 7.54 (d, <i>J</i> = 8.0 Hz, 1H), 7.48 (t, <i>J</i> = 7.9 Hz, 1H), 6.39 (d, <i>J</i> = 2.1 Hz, 1H), 5.41 (dd, <i>J</i> = 6.3, 2.2 Hz, 1H), 5.16 (dd, <i>J</i> = 6.3, 4.0 Hz, 1H), 4.59 (dt, <i>J</i> = 10.0, 3.6 Hz, 1H), 3.61–3.50 (m, 2H), 3.27 (t, <i>J</i> = 8.2 Hz, 2H), 3.02–2.98 (m, 2H), 1.61 (s, 3H), 1.38 (s, 3H). HRMS calcd for C<sub>22</sub>H<sub>25</sub>N<sub>7</sub>O<sub>3</sub> + H: 436.2097; found: 436.2094 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i14" class="anchor-spacer"></div><h4 class="article-section__title" id="_i14"> <i>tert</i>-Butyl (<i>S</i>)-5-((((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-(6-Amino-9<i>H</i>-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl)(3-cyanophenethyl)amino)-2-((bis(<i>tert</i>-butoxycarbonyl))amino)-pentanoate (<b>14a</b>, Compound <b>14</b> when <i>n</i> = 2)</h4><div class="NLM_p last">To a solution of <b>12</b> (0.065 g, 0.15 mmol) in 5 mL DCM was added aldehyde <b>13a</b> (compound <b>13</b> when <i>n</i> = 2) (0.058g, 0.15 mmol) followed by 2 drops of AcOH. The resulting mixture was stirred 30 for min before NaBH(OAc)<sub>3</sub> (0.048 g, 0.23 mmol) was added. After being stirred for 16 h at rt, the reaction was quenched with saturated NaHCO<sub>3</sub> and extracted with EtOAc (5×). The combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. The residue was purified with silica gel column to provide desired product <b>14a</b> (0.074 g, 61%). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>) δ 8.28 (s, 1H), 7.86 (s, 1H), 7.39 (dt, <i>J</i> = 6.8, 1.9 Hz, 1H), 7.33–7.31 (bs, 1H), 7.26–7.24 (m, 2H), 5.98 (d, <i>J</i> = 2.2 Hz, 1H), 5.91 (s, 2H), 5.45 (dd, <i>J</i> = 6.5, 2.2 Hz, 1H), 4.88 (dd, <i>J</i> = 6.4, 3.3 Hz, 1H), 4.63 (dd, <i>J</i> = 9.6, 5.3 Hz, 1H), 4.25 (td, <i>J</i> = 6.7, 3.3 Hz, 1H), 2.75–2.68 (m, 1H), 2.65–2.54 (m, 5H), 2.51–2.42 (m, 2H), 1.98–1.90 (m, 1H), 1.80–1.69 (m, 1H), 1.54 (s, 3H), 1.43 (s, 18H), 1.38 (s, 11H), 1.33 (s, 3H). HRMS calcd for C<sub>41</sub>H<sub>58</sub>N<sub>8</sub>O<sub>9</sub> + H: 807.4405; found: 807.4402 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i15" class="anchor-spacer"></div><h4 class="article-section__title" id="_i15"> <i>tert</i>-Butyl (<i>S</i>)-4-((((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-(6-Amino-9<i>H</i>-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl)(3-cyanophenethyl)amino)-2-((bis(<i>tert</i>-butoxycarbonyl))amino)-butanoate (<b>14b</b>, Compound <b>14</b> when <i>n</i> = 1)</h4><div class="NLM_p last">Intermediate <b>14b</b> (66%) was prepared according to the procedure for <b>14a</b> and further purified by reverse-phase HPLC. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 8.46 (s, 1H), 8.44 (s, 1H), 7.62 (d, <i>J</i> = 6.8 Hz, 1H), 7.58 (s, 1H), 7.54–7.44 (m, 2H), 6.41 (d, <i>J</i> = 2.1 Hz, 1H), 5.37 (dd, <i>J</i> = 6.4, 2.1 Hz, 1H), 5.17 (dd, <i>J</i> = 6.3, 4.3 Hz, 1H), 4.77–4.67 (m, 2H), 3.91 (dd, <i>J</i> = 14.1, 10.4 Hz, 1H), 3.80 (dd, <i>J</i> = 14.0, 2.3 Hz, 1H), 3.53–3.40 (m, 3H), 3.30–3.23 (m, 1H), 3.14–2.91 (m, 2H), 2.53–2.41 (m, 1H), 2.06–1.93 (m, 1H), 1.65 (s, 3H), 1.49 (s, 18H), 1.44 (s, 9H), 1.39 (s, 3H). HRMS calcd for C<sub>40</sub>H<sub>56</sub>N<sub>8</sub>O<sub>9</sub> + H: 793.4249; found: 793.4251 [M + H]<sup>+</sup>.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> <i>tert</i>-Butyl (<i>S</i>)-5-((((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-(6-Amino-9<i>H</i>-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl)(3-carbamoylphenethyl)amino)-2-((bis(<i>tert</i>-butoxycarbonyl))amino)-pentanoate (<b>15a</b>, Compound <b>15</b> when <i>n</i> = 2)</h4><div class="NLM_p last">To a solution of <b>14a</b> (0.040 g, 0.05 mmol) in DMSO (1 mL) was added K<sub>2</sub>CO<sub>3</sub> (0.028 g, 0.2 mmol). The mixture was cooled to 0 °C and treated with H<sub>2</sub>O<sub>2</sub> (0.1 mL). The reaction mixture was warmed to rt and stirred for 3 h at rt. The reaction was diluted with water and extracted with EtOAc (3×). The combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. The residue was purified by silica gel column to provided <b>15a</b> (0.034 g, 82%). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>) δ 8.28 (s, 1H), 7.99 (s, 1H), 7.64 (d, <i>J</i> = 7.7 Hz, 1H), 7.48 (s, 1H), 7.23 (t, <i>J</i> = 7.6 Hz, 1H), 7.13 (d, <i>J</i> = 7.6 Hz, 1H), 7.02 (s, 1H), 6.44 (s, 1H), 6.42 (s, 2H), 6.06 (d, <i>J</i> = 2.0 Hz, 1H), 5.45 (d, <i>J</i> = 5.8 Hz, 1H), 4.97 (s, 1H), 4.65 (dd, <i>J</i> = 9.6, 5.2 Hz, 1H), 4.43 (s, 1H), 2.98–2.85 (m, 2H), 2.84–2.73 (m, 2H), 2.72–2.61 (m, 4H), 2.05–1.96 (m, 1H), 1.88–1.77 (m, 1H), 1.57 (s, 3H), 1.45 (s, 18H), 1.41 (s, 11H), 1.34 (s, 3H). MS (ESI) <i>m</i>/<i>z</i> 825.6 [M + H]<sup>+</sup></div></div><div id="sec4_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17"> (<i>S</i>)-2-Amino-5-((((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl)(3-carbamoylphenethyl)amino)pentanoic acid (<b>6</b>)</h4><div class="NLM_p last">The intermediate <b>15a</b> (0.034 g, 0.041 mmol) was dissolved in dioxane (0.25 mL). The resulting solution was cooled to 0 °C and treated with HCl (4 M in dioxane, 0.25 mL). The reaction mixture was warmed to rt and stirred for 4 h before 2 drops of water was added. The resulting mixture was stirred for another 4 h at rt. After concentration, the residue was dissolved in MeOH (0.1 mL). To the solution Et<sub>2</sub>O was added to triturate the desired product <b>6</b> as a white solid. After filtration and wash with Et<sub>2</sub>O, the desired product <b>6</b> (0.015 g, 69%) was obtained. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 8.57 (s, 1H), 8.46 (s, 1H), 7.82–7.65 (m, 2H), 7.44–7.29 (m, 2H), 6.18 (s, 1H), 4.74 (d, <i>J</i> = 28.4 Hz, 1H), 4.55 (t, <i>J</i> = 8.0 Hz, 1H), 4.49 (t, <i>J</i> = 5.5 Hz, 1H), 4.07 (s, 1H), 3.94–3.82 (m, 1H), 3.81–3.72 (m, 1H), 3.60–3.49 (m, 2H), 3.49–3.38 (m, 2H), 3.20–3.04 (m, 2H), 2.14–2.00 (m, 3H), 2.00–1.89 (m, 1H). <sup>13</sup>C NMR (151 MHz, CD<sub>3</sub>OD) δ 171.8, 171.2, 152.0, 149.7, 145.8 (broad peak), 144.9 (broad peak), 137.9, 135.6, 133.4, 130.1, 129.2, 127.4, 121.1, 92.0, 80.4 (0.5C), 79.7 (0.5C), 74.6, 73.5, 56.8 (0.5C), 56.0 (0.5C), 55.7 (0.5C), 55.3 (0.5C), 54.2, 53.2, 30.8 (0.5C), 30.2 (0.5C), 28.5, 21.2. Note: A couple of peaks are broad and overlap: 145.8 and 144.9. Four sets of peaks are doubled peaks: 80.4 and 79.7; 56.8 and 56.0; 55.7 and 55.3; 30.8 and 30.2. Similar doubled peaks have been observed in other SAM analogs.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(44)</a> HRMS calcd for C<sub>24</sub>H<sub>32</sub>N<sub>8</sub>O<sub>6</sub> + H: 529.2523; found: 529.2524 [M + H]<sup>+</sup>.</div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> (<i>S</i>)-2-Amino-4-((((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl)(3-carbamoylphenethyl)amino)butanoic acid (<b>7</b>)</h4><div class="NLM_p last">Compound <b>7</b> (46% over 2 steps) was prepared according to the procedure for <b>6</b> and purified by reverse-phase HPLC. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 8.40 (s, 1H), 8.32 (s, 1H), 7.76–7.65 (m, 2H), 7.38–7.25 (m, 2H), 6.11 (d, <i>J</i> = 4.1 Hz, 1H), 4.75 (t, <i>J</i> = 4.7 Hz, 1H), 4.54–4.48 (m, 1H), 4.46 (t, <i>J</i> = 5.4 Hz, 1H), 4.00–3.93 (m, 1H), 3.82–3.73 (m, 1H), 3.72–3.65 (m, 1H), 3.63–3.57 (m, 1H), 3.55–3.42 (m, 3H), 3.15–3.02 (m, 2H), 2.46–2.34 (m, 1H), 2.22–2.13 (m, 1H). HRMS calcd for C<sub>23</sub>H<sub>30</sub>N<sub>8</sub>O<sub>6</sub> + H: 515.2367; found: 515.2365 [M + H]<sup>+</sup>.</div></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Side Chain Flexible Docking Study of NNMT with Inhibitors <b>6</b> and <b>7</b></h3><div class="NLM_p last">Multiple 3D structures of compounds <b>6</b> and <b>7</b> were created from smile files. The hNNMT structure (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2IIP">2IIP</a>) was used as the template for initial docking of the ligands by deep neural network-based docking program (Orbital).<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> As illustrated in the movie in the <a href="/doi/suppl/10.1021/acs.jmedchem.7b01422/suppl_file/jm7b01422_si_003.mpg" class="ext-link">Supporting Information</a>, upon obtaining initial docking poses of the ligands, protein side chains of 2IIP were allowed to cofold in the presence of the top-scoring dock ligand by side chain prediction platform.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> After multiple rounds of the docking–refolding process, the converged protein–ligand complex structure was created as the final docking result.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> hNNMT Protein Expression and Purification for Biochemical Assay and MOA Studies</h3><div class="NLM_p last">DNA fragment encoding hNNMT (residues 1–264) was amplified by PCR and subcloned into the pET28a-LIC vector, downstream of the poly histidine coding region. The overexpression of the recombinant proteins in <i>E. coli</i> BL21 (DE3) pRARE2-V2R was induced by addition of 0.5 mM isopropyl-1-thio-<span class="smallcaps smallerCapital">d</span>-galactopyranoside (IPTG) and overnight incubation at 16 °C. Harvested cells were resuspended in 20 mM Tris-HCl buffer, pH 7.5, containing 500 mM NaCl, 5 mM imidazole and 5% glycerol, 1 mM PMSF, 1 mM DTT, and Roche complete EDTA-free protease inhibitor cocktail tablet. The cells were lysed chemically by rotating 30 min with CHAPS (final concentration of 0.5%) and 2.5 μg benzonase nuclease, followed by sonication for 5 min (Sonicator 3000, Misoni). The crude extract was clarified by high-speed centrifugation (60 min at 36,000 × <i>g</i> at 4 °C). After centrifugation, the lysate was then loaded onto a Ni-NTA resin (Qiagen) column. The column was washed and eluted by running 20 mM Tris-HCl, pH 7.5, 500 mM NaCl, 5% glycerol, containing 30 mM and 250 mM imidazole, respectively. The purity of the protein was judged by SDS-PAGE gel, and the pure protein solution was dialyzed overnight against 20 mM Tris, pH7.5, 250 mM NaCl, 1 mM DTT, and 5% glycerol and then concentrated, aliquoted, flash frozen in liquid nitrogen and stored at −80 °C.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> hNNMT Biochemical Assays and Enzyme Kinetics Study</h3><div class="NLM_p last">SAHH-coupled assay was used as previously described<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> to determine the NNMT kinetic parameters, the IC<sub>50</sub> values of <b>6</b> and <b>7</b>, and the mechanism of action (MOA) of compound <b>6</b>. 200 nM hNNMT was used in all reactions with nicotinamide (cat. no. N-3376, Sigma) and SAM (cat. no. S018, AK scientific) as substrates in the presence of 5 μM SAHH (purified in house) and 15 μM ThioGlo (cat. no. 595501; Calbiochem, Gibbstown, NJ). The methylation assays were performed in 50 mM Tris, pH 7.5, 0.01% Triton X-100 with a constant DMSO concentration of 2% (v/v). IC<sub>50</sub> values for compounds <b>6</b> and <b>7</b> were determined in triplicate by varying the concentration of inhibitor from 250 nM to 250 μM with SAM and nicotinamide concentrations kept constant at 25 μM (∼<i>K</i><sub>m</sub> value of each substrate). Data were then fit to Four Parameter Logistic equation using GraphPad Prism software 7.02. To study the MOA of compound <b>6</b>, 200 μM fixed concentration of nicotinamide and varying concentrations of SAM (from 12.5 to 200 μM) or 250 μM fixed concentration of SAM and varying concentrations of nicotinamide (from 12.5 to 200 μM) were included in reactions at 0, 1, 2, 4, 8, 16, 31, 63, 125, and 250 μM of compound <b>6</b>. Reactions (in quadruplicates) were initiated by adding SAM and followed by measuring the increase in fluorescence using Synergy 4 microplate reader with 360/40 nm excitation filter and 528/20 nm emission filter in 384-well plate format. The data were analyzed by enzyme kinetic module of SigmaPlot software 11.0. The relationships between the measured activity and the concentration of SAM or nicotinamide for various fixed compound <b>6</b> concentrations were fitted globally with equations describing competitive and noncompetitive inhibition with respect to SAM and nicotinamide, respectively.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Selectivity Assays</h3><div class="NLM_p">Effect of compound <b>6</b> on methyltransferase activity of G9a, GLP, SUV39H1, SUV39H2, SETDB1, SETD8, SUV420H1, SUV420H2, SETD7, MLL1 pentameric complex, MLL3 pentameric complex, EZH2 trimeric complex, PRMT1, PRMT3, PRMT4, PRMT5/MEP50 complex, PRMT6, PRMT7, PRMT8, PRMT9, PRDM9, SETD2, SMYD2, SMYD3, DNMT1, BCDIN3D, METTL3_METTL14 complex, and acetyltransferase activity of EP300 were assessed by monitoring the incorporation of tritium-labeled methyl/acetyl group to substrates using Scintillation Proximity Assay (SPA) as previously described.<a onclick="showRef(event, 'ref43 ref44'); return false;" href="javascript:void(0);" class="ref ref43 ref44">(45, 46)</a> Assays were performed in a 10 μL reaction mixture containing <sup>3</sup>H-SAM or <sup>3</sup>H-acetyl coenzyme A for EP300 (PerkinElmer; <a href="http://www.perkinelmer.com" class="extLink">www.perkinelmer.com</a>) at substrate concentrations close to <i>K</i><sub>m</sub> values for each enzyme. Two concentrations (10 μM and 50 μM) for all selectivity assays and titrations from 200 nM to 200 μM of compound <b>6</b> for hit IC<sub>50</sub> determinations were used. To stop the enzymatic reactions, 10 μL of 7.5 M guanidine hydrochloride was added, followed by 180 μL of buffer (20 mM Tris, pH 8.0), mixed and then transferred to a 96-well FlashPlate (cat. no. SMP103; PerkinElmer). After mixing, the reaction mixtures in Flash plates were incubated for 1 h, and the CPM were measured using Topcount plate reader (PerkinElmer). The CPM counts in the absence of compound for each data set were defined as 100% activity. In the absence of the enzyme, the CPM counts in each data set were defined as background (0%). The IC<sub>50</sub> values were determined using GraphPad Prism 7 software.</div><div class="NLM_p last">For DOT1L, NSD1, NSD2, NSD3, ASH1L, DNMT3<i>A</i>/3L, and DNMT3<i>B</i>/3L, a filter-based assay was used. In this assay, 10 μL of reaction mixtures were incubated at rt for 1 h, 50 μL of 10% TCA was added, mixed, and transferred to filter-plates (Millipore; cat. no. MSFBN6B10; <a href="http://www.millipore.com" class="extLink">www.millipore.com</a>). Plates were centrifuged at 2000 rpm (Allegra X-15R - Beckman Coulter, Inc.) for 2 min, followed by 2 additional 10% TCA wash and one ethanol wash (180 μL), followed by centrifugation. Plates were dried and 100 μL MicroO (MicroScint-O; cat. no. 6013611, PerkinElmer) was added to each well, centrifuged, and removed. 70 μL of MicroO was added again, and CPM was measured using Topcount plate reader.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> hNNMT Protein Expression and Purification for ITC and Crystallography</h3><div class="NLM_p last">A construct encoding the full-length human NNMT was cloned into the pET28MHL expression vector using NdeI and <i>Hin</i>dIII restriction sites. Full-length human NNMT triple mutant (K100A:E101A:E103A) used for crystallization was ordered from the nonprofit plasmid repository addgene (Plasmid #40734). The recombinant hNNMT and hNNMTtm (triple mutant) were overexpressed in <i>E. coli</i> BL21 (DE3) codon plus RIL strain (Stratagene) by addition of 0.3 mM isopropyl-1-thio-<span class="smallcaps smallerCapital">d</span>-galactopyranoside with a supplement of 1 mM zinc sulfate and incubated overnight at 15 °C. Harvested cells were resuspended in 50 mM sodium phosphate buffer, pH 7.5, supplemented with 0.5 mM sodium chloride and 5% glycerol, and lysed using a microfluidizer (Microfluidics) at 10,000 psi. After clarification of the crude extract by high-speed centrifugation, the lysate was loaded onto a 5 mL HiTrap chelating column (GE Healthcare) charged with Ni<sup>2+</sup>. The column was washed, and the protein was eluted with 50 mM Tris buffer, pH 8.0, 250 mM sodium chloride, 250 mM imidazole and supplemented with 0.5 mM of TCEP. The protein was next purified on a Superdex 200 column (26/600; GE Healthcare) equilibrated with 50 mM Tris-HCl buffer, pH 8.0, and 100 mM sodium chloride, and the elution fractions were pooled and supplemented with 0.5 mM of TCEP. All purification steps were performed at 4 °C and in the presence of a protease inhibitor AEBSF (Goldbio). The proteins were concentrated to ∼20 mg/mL, and hNNMTtm was supplemented with 5% glycerol. The size and the purity of the recombinant hNNMT and hNNMTtm proteins were checked by SDS-PAGE and mass spectroscopy.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Isothermal Titration Calorimetry</h3><div class="NLM_p last">Isothermal titration calorimetry (ITC) measurements were performed at 25 °C using a MicroCal ITC200 Instrument (Malvern Instruments). hNNMT was diluted at 100–200 μM in ITC buffer [50 mM Tris pH 8.0, 100 mM NaCl] supplemented with 2% DMSO for the binding with the compounds. Compounds <b>6</b> and <b>7</b> were prepared in DMSO at 50 mM and diluted to 1 mM in ITC buffer with a final DMSO concentration of 2%. The cofactors SAM, SAH, and the substrate nicotinamide were diluted to 2 mM. For the binding of the substrate nicotinamide in the presence of SAH, the protein was mixed at a ratio of 1:4 for 1 h at 4 °C prior the measurement. Binding constants were calculated by fitting the data using the ITC data analysis module in Origin 7.0 (OriginLab Corp).</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Crystallization, Data Collection, and Structure Determination</h3><div class="NLM_p">Purified hNNMTtm at 20 mg/mL was mixed with compound <b>6</b> at a molar ratio of 1:4 and crystallized using the hanging drop vapor diffusion method at 17 °C by mixing equal volumes of the protein solution with the reservoir solution. The hNNMTtm–<b>6</b> binary complex was crystallized in 1.5 ammonium sulfate, 0.1 M HEPES pH 7.2. The crystal was soaked in the corresponding mother liquor supplemented with 20% ethylene glycol as cryoprotectant before flash freezing in liquid nitrogen.</div><div class="NLM_p last">X-ray diffraction data were collected at 100 K at NE-CAT beamline 24-ID-E of Advanced Photon Source (APS) at Argonne National Laboratory. The data were processed with MOSFLM, SCALA, and other programs from the CCP4 suite.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(47)</a> The structure of hNNMTtm complex was solved by molecular replacement using PHASER<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(48)</a> using the atomic model of the hNNMTtm (Protein Data Bank ID code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2IIP">2IIP</a>). The location of the bound molecule was determined from a <i>F</i><sub>o</sub>–<i>F</i><sub>c</sub> difference electron density map. REFMAC<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(49)</a> and phenix.refine<a onclick="showRef(event, 'ref48 ref49'); return false;" href="javascript:void(0);" class="ref ref48 ref49">(50, 51)</a> were used for structure refinement. Graphic program COOT<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(52)</a> was used for model building and visualization. The overall assessment of model quality was performed using MolProbity.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(53)</a> Crystal diffraction data and refinement statistics for the structure are displayed in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01422/suppl_file/jm7b01422_si_001.pdf" class="ext-link">Supporting Information</a> Table 1.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i26"><a href="/doi/suppl/10.1021/acs.jmedchem.7b01422">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_43715" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_43715" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.7b01422" class="ext-link">10.1021/acs.jmedchem.7b01422</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline"><sup>1</sup>H and <sup>13</sup>C NMR spectra of compound <b>6</b>; Docking analysis of <b>7</b> in the binding sites of hNNMT; Docking analysis of <b>6</b> in the binding sites of DOT1L and SMYD2; Kinetic parameter determination for NNMT; Binding affinity of SAH to hNNMT and binding affinity of nicotinamide to hNNMT in the presence of SAH; Overlay of the crystal structure of the hNNMT (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6B1A">6B1A</a>) with the crystal structure of mouse NNMT (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5XVK">5XVK</a>); Proteins in the methyltransferase selectivity panel; Crystallography data and refinement statistics (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01422/suppl_file/jm7b01422_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01422/suppl_file/jm7b01422_si_002.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Docking process movie for <b>6</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01422/suppl_file/jm7b01422_si_003.mpg" class="ext-link">MPG</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01422/suppl_file/jm7b01422_si_001.pdf">jm7b01422_si_001.pdf (1.27 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01422/suppl_file/jm7b01422_si_002.csv">jm7b01422_si_002.csv (0.25 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01422/suppl_file/jm7b01422_si_003.mpg">jm7b01422_si_003.mpg (46.27 MB)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The structure of human NNMT in complex with <b>6</b> has been deposited under PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6B1A">6B1A</a>. Authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.7b01422" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_25022" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_25022" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Feng Liu</span> - <span class="hlFld-Affiliation affiliation">Jiangsu
Key Laboratory of Translational Research and Therapy for Neuro-Psycho-Diseases
and Department of Medicinal Chemistry, College of Pharmaceutical Sciences, Soochow University, Suzhou, Jiangsu 215123, People’s Republic of China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-2669-5448" title="Orcid link">http://orcid.org/0000-0003-2669-5448</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#bddbd1d4c88ffdcec8d9dc93d8d9c893ded3"><span class="__cf_email__" data-cfemail="8ceae0e5f9beccfff9e8eda2e9e8f9a2efe2">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Masoud Vedadi</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Pharmacology and Toxicology, University
of Toronto, Toronto, Ontario M5S 1A8, Canada</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#2944075f4c4d484d40695c5d465b46475d46074a48"><span class="__cf_email__" data-cfemail="7d10530b18191c19143d0809120f12130912531e1c">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jing Liu</span> - <span class="hlFld-Affiliation affiliation">Center
for Chemical Biology and Drug Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#42082b2c256c2e2b37022f31312f6c272637"><span class="__cf_email__" data-cfemail="87cdeee9e0a9ebeef2c7eaf4f4eaa9e2e3f2">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jian Jin</span> - <span class="hlFld-Affiliation affiliation">Center
for Chemical Biology and Drug Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2387-3862" title="Orcid link">http://orcid.org/0000-0002-2387-3862</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#90faf9f1febefaf9fed0fde3e3fdbef5f4e5"><span class="__cf_email__" data-cfemail="9ef4f7fff0b0f4f7f0def3ededf3b0fbfaeb">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nicolas Babault</span> - <span class="hlFld-Affiliation affiliation">Center
for Chemical Biology and Drug Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Abdellah Allali-Hassani</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fengling Li</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jie Fan</span> - <span class="hlFld-Affiliation affiliation">Accutar
Biotechnology, Brooklyn, New York 11226, United
States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alex Yue</span> - <span class="hlFld-Affiliation affiliation">Center
for Chemical Biology and Drug Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kevin Ju</span> - <span class="hlFld-Affiliation affiliation">Center
for Chemical Biology and Drug Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): J.F. is an employee and shareholder of Accutar Biotechnology. J.J. is a consultant of Accutar Biotechnology.<br /></br></div></li></ul></div><div class="ack" id="ACK-d345e2088-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i28">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_12708" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_12708" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Albina Bolotokova for compound management and Elisa Gibson for protein purification. The research described here was supported by the grants R01GM122749, R01CA218600, and R01HD088626 (to J.J.) from the U.S. National Institutes of Health and 21302134 (to F.L.) from the National Natural Science Foundation of China. The SGC is a registered charity (number 1097737) that receives funds from AbbVie, Bayer Pharma AG, Boehringer Ingelheim, Canada Foundation for Innovation, Eshelman Institute for Innovation, Genome Canada, Innovative Medicines Initiative (EU/EFPIA) [ULTRA-DD grant no. 115766], Janssen, Merck & Co., Novartis Pharma AG, Ontario Ministry of Economic Development and Innovation, Pfizer, São Paulo Research Foundation-FAPESP, Takeda, and the Wellcome Trust. This work is based upon research conducted at the Northeastern Collaborative Access Team beamlines, which are funded by the National Institute of General Medical Sciences from the National Institutes of Health (P41 GM103403). The Eiger 16M detector on 24-ID-E beamline is funded by a NIH-ORIP HEI grant (S10OD021527). This research used resources of the Advanced Photon Source, a U.S. Department of Energy (DOE) Office of Science User Facility operated for the DOE Office of Science by Argonne National Laboratory under contract no. DE-AC02-06CH11357.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i29" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i29"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i30" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i30"> Abbreviations Used</h2><tr><td class="NLM_term">NNMT</td><td class="NLM_def"><p class="first last">nicotinamide <i>N</i>-methyltransferase</p></td></tr><tr><td class="NLM_term">SAM</td><td class="NLM_def"><p class="first last"><i>S</i>-5′-adenosyl-<span class="smallcaps smallerCapital">l</span>-methionine</p></td></tr><tr><td class="NLM_term">ROS</td><td class="NLM_def"><p class="first last">reactive oxygen species</p></td></tr><tr><td class="NLM_term">NAD<sup>+</sup></td><td class="NLM_def"><p class="first last">nicotinamide adenine dinucleotide</p></td></tr><tr><td class="NLM_term">WAT</td><td class="NLM_def"><p class="first last">white adipose tissue</p></td></tr><tr><td class="NLM_term">ASO</td><td class="NLM_def"><p class="first last">antisense oligonucleotide</p></td></tr><tr><td class="NLM_term">hNNMT</td><td class="NLM_def"><p class="first last">human NNMT</p></td></tr><tr><td class="NLM_term">SAH</td><td class="NLM_def"><p class="first last"><i>S</i>-5′-adenosyl-<span class="smallcaps smallerCapital">l</span>-homocysteine</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">SAHH</td><td class="NLM_def"><p class="first last">SAH hydrolase</p></td></tr><tr><td class="NLM_term">ITC</td><td class="NLM_def"><p class="first last">isothermal titration calorimetry</p></td></tr><tr><td class="NLM_term">DOT1L</td><td class="NLM_def"><p class="first last">disruptor of telomeric silencing 1-like</p></td></tr><tr><td class="NLM_term">PRMT7</td><td class="NLM_def"><p class="first last">protein arginine methyltransferase 7</p></td></tr><tr><td class="NLM_term">SMYD2</td><td class="NLM_def"><p class="first last">SET and MYND domain containing protein 2</p></td></tr><tr><td class="NLM_term">BCDIN3D</td><td class="NLM_def"><p class="first last">BCDIN3 domain containing RNA methyltransferase</p></td></tr><tr><td class="NLM_term">MOA</td><td class="NLM_def"><p class="first last">mechanism of action</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i31">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_30552" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_30552" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 53 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Alston, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abeles, R. H.</span><span> </span><span class="NLM_article-title">Substrate specificity of nicotinamide methyltransferase isolated from porcine liver</span> <span class="citation_source-journal">Arch. Biochem. Biophys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">260</span><span class="NLM_x">, </span> <span class="NLM_fpage">601</span><span class="NLM_x">–</span> <span class="NLM_lpage">608</span><span class="refDoi"> DOI: 10.1016/0003-9861(88)90487-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1016%2F0003-9861%2888%2990487-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=2963591" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADyaL1cXhs1elu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=260&publication_year=1988&pages=601-608&author=T.+A.+Alstonauthor=R.+H.+Abeles&title=Substrate+specificity+of+nicotinamide+methyltransferase+isolated+from+porcine+liver&doi=10.1016%2F0003-9861%2888%2990487-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Substrate specificity of nicotinamide methyltransferase isolated from porcine liver</span></div><div class="casAuthors">Alston, Theodore A.; Abeles, Robert H.</div><div class="citationInfo"><span class="NLM_cas:title">Archives of Biochemistry and Biophysics</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">260</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">601-8</span>CODEN:
                <span class="NLM_cas:coden">ABBIA4</span>;
        ISSN:<span class="NLM_cas:issn">0003-9861</span>.
    </div><div class="casAbstract">Nicotinamide methyltransferase (EC 2.1.1.1) was purified >1300-fold from porcine liver.  The enzyme is electrophoretically homogeneous, exhibiting a relative mol. wt. of 27,000.  In addn. to acting on nicotinamide and close structural analogs such as thionicotinamide and 3-acetylpyridine, the enzyme actively accommodates poor analogs such as quinoline, isoquinoline, and 1,2,3,4-tetrahydroisoquinoline as Me group acceptors.  Thus, the enzyme may detoxify numerous alkaloids in vivo.  In some cases, this enzyme might paradoxically increase toxicities of substrates, but the hepatotoxic antibiotic pyrazinamide, which is considered as a potential enzyme-activated electrophile, did not function detectably as a substrate for the isolated enzyme.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrApJV903Z9o7Vg90H21EOLACvtfcHk0lh1F09BT6y4HA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXhs1elu7c%253D&md5=ff4b08c064606a2373a4fa5164c837ba</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2F0003-9861%2888%2990487-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0003-9861%252888%252990487-0%26sid%3Dliteratum%253Aachs%26aulast%3DAlston%26aufirst%3DT.%2BA.%26aulast%3DAbeles%26aufirst%3DR.%2BH.%26atitle%3DSubstrate%2520specificity%2520of%2520nicotinamide%2520methyltransferase%2520isolated%2520from%2520porcine%2520liver%26jtitle%3DArch.%2520Biochem.%2520Biophys.%26date%3D1988%26volume%3D260%26spage%3D601%26epage%3D608%26doi%3D10.1016%2F0003-9861%2888%2990487-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">van Haren, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sastre Torano, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sartini, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emanuelli, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parsons, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, N. I.</span><span> </span><span class="NLM_article-title">A rapid and efficient assay for the characterization of substrates and inhibitors of nicotinamide N-methyltransferase</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">5307</span><span class="NLM_x">–</span> <span class="NLM_lpage">5315</span><span class="refDoi"> DOI: 10.1021/acs.biochem.6b00733</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biochem.6b00733" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVWntL7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=5307-5315&author=M.+J.+van+Harenauthor=J.+Sastre+Toranoauthor=D.+Sartiniauthor=M.+Emanuelliauthor=R.+B.+Parsonsauthor=N.+I.+Martin&title=A+rapid+and+efficient+assay+for+the+characterization+of+substrates+and+inhibitors+of+nicotinamide+N-methyltransferase&doi=10.1021%2Facs.biochem.6b00733"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">A rapid and efficient assay for the characterization of substrates and inhibitors of nicotinamide N-methyltransferase</span></div><div class="casAuthors">van Haren, Matthijs J.; Sastre Torano, Javier; Sartini, Davide; Emanuelli, Monica; Parsons, Richard B.; Martin, Nathaniel I.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">5307-5315</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) is one of the most abundant small mol. methyltransferases in the human body and is primarily responsible for the N-methylation of the nicotinamide (vitamin B3).  Employing the cofactor, S-adenosyl-L-methionine, NNMT transfers a Me group to the pyridine N atom of nicotinamide to generate N-methylnicotinamide.  Interestingly, NNMT is also able to N-methylate a variety of other pyridine-contg. small mols., suggesting a secondary role for the enzyme in the detoxification of xenobiotics.  A no. of recent studies have also revealed links between NNMT overexpression and a variety of diseases, including multiple cancers, Parkinson's disease, diabetes, and obesity.  Here, to facilitate further study of both the substrate scope and potential for inhibitor development, the authors describe the development of a new activity assay for human NNMT.  The assay makes use of ultra-high-performance hydrophilic interaction chromatog., allowing for the rapid sepn. of the reaction products, coupled with quadrupole time-of-flight mass spectrometric detection, providing for enhanced sensitivity and enabling high-throughput sample anal.  The authors successfully demonstrate the general applicability of the method by performing kinetic analyses of NNMT-mediated methylation for a range of pyridine-based substrates.  These findings also provide new insight into the diversity of substrate recognition by NNMT in a quant. manner.  In addn., the authors further established the suitability of the assay for the identification and characterization of small mol. inhibitors of NNMT.  To do so, the authors investigated the inhibition of NNMT by the nonspecific methyltransferase inhibitors, sinefungin and S-adenosyl-L-homocysteine, revealing IC50 values in the low micromolar range.  The results of these inhibition studies are particularly noteworthy as they will permit future efforts toward the development of new NNMT-specific inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBC8U-yKb1brVg90H21EOLACvtfcHk0lhRuytHdchQhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVWntL7O&md5=0ebb58b665b91dd5983b2bef91886c10</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Facs.biochem.6b00733&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biochem.6b00733%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BHaren%26aufirst%3DM.%2BJ.%26aulast%3DSastre%2BTorano%26aufirst%3DJ.%26aulast%3DSartini%26aufirst%3DD.%26aulast%3DEmanuelli%26aufirst%3DM.%26aulast%3DParsons%26aufirst%3DR.%2BB.%26aulast%3DMartin%26aufirst%3DN.%2BI.%26atitle%3DA%2520rapid%2520and%2520efficient%2520assay%2520for%2520the%2520characterization%2520of%2520substrates%2520and%2520inhibitors%2520of%2520nicotinamide%2520N-methyltransferase%26jtitle%3DBiochemistry%26date%3D2016%26volume%3D55%26spage%3D5307%26epage%3D5315%26doi%3D10.1021%2Facs.biochem.6b00733" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Aksoy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szumlanski, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weinshilboum, R. M.</span><span> </span><span class="NLM_article-title">Human liver nicotinamide N-methyltransferase. cDNA cloning, expression, and biochemical characterization</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">269</span><span class="NLM_x">, </span> <span class="NLM_fpage">14835</span><span class="NLM_x">–</span> <span class="NLM_lpage">14840</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=8182091" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADyaK2cXkslCqtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=269&publication_year=1994&pages=14835-14840&author=S.+Aksoyauthor=C.+L.+Szumlanskiauthor=R.+M.+Weinshilboum&title=Human+liver+nicotinamide+N-methyltransferase.+cDNA+cloning%2C+expression%2C+and+biochemical+characterization"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Human liver nicotinamide N-methyltransferase. cDNA cloning, expression, and biochemical characterization</span></div><div class="casAuthors">Aksoy, Saime; Szumlanski, Carol L.; Weinshilboum, Richard M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">269</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">14835-40</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) catalyzes the N-methylation of nicotinamide and other pyridines.  Human liver NNMT activity has a bimodal frequency distribution, an observation which raises the possibility that this enzyme might be regulated by a genetic polymorphism, a polymorphism that could have functional implications for individual differences in drug and xenobiotic toxicity.  As a first step toward testing this hypothesis, the authors set out to clone and express a cDNA for human liver NNMT.  Human liver NNMT was partially purified, photoaffinity-labeled, subjected to limited proteolysis, and partial amino acid sequence information was obtained.  The polymerase chain reaction was then used to amplify a 550-nucleotide sequence with human liver cDNA as template and primers designed on the basis of the NNMT amino acid sequence.  The 5'- and 3'-ends of a human liver NNMT cDNA were obtained by use of the rapid amplification of cDNA ends.  The combined use of these approaches resulted in the isolation of a human liver NNMT cDNA that was 969 nucleotides in length, with a 792-nucleotide open reading frame that encoded a 264-amino acid protein with a calcd. mol. mass of 29.6 kDa.  The human liver NNMT cDNA was transcribed in vitro and translated with a reticulocyte lysate system to yield a protein with a mol. mass of approx. 29 kDa that comigrated during SDS-polyacrylamide gel electrophoresis with photoaffinity-labeled human liver NNMT.  The NNMT cDNA was also subcloned into the eukaryotic expression vector p91023(B).  COS-1 cells transfected with this construct expressed a high level of NNMT enzymic activity, and the biochem. properties of this activity were similar to those of human liver NNMT.  Human liver NNMT and transfected COS-1 cell NNMT had apparent Km values for the two cosubstrates for the reaction, nicotinamide and S-adenosyl-L-methionine, of 0.43 and 0.38 mM and of 1.8 and 2.2 μM, resp.  IC50 values for the inhibition of NNMT by N1-methylnicotinamide were 60 and 30 μM for human liver and COS-1 cell-expressed NNMT, resp.  Cloning of a cDNA for human liver NNMT should make it possible to test the hypothesis that inheritance may play a role in the regulation of individual differences in human liver NNMT activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZRn3_QD5atbVg90H21EOLACvtfcHk0lhRuytHdchQhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXkslCqtro%253D&md5=d5d9edf09cb329ecc01b49fd819fad47</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAksoy%26aufirst%3DS.%26aulast%3DSzumlanski%26aufirst%3DC.%2BL.%26aulast%3DWeinshilboum%26aufirst%3DR.%2BM.%26atitle%3DHuman%2520liver%2520nicotinamide%2520N-methyltransferase.%2520cDNA%2520cloning%252C%2520expression%252C%2520and%2520biochemical%2520characterization%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1994%26volume%3D269%26spage%3D14835%26epage%3D14840" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Riederer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Erwa, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zimmermann, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frank, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zechner, R.</span><span> </span><span class="NLM_article-title">Adipose tissue as a source of nicotinamide N-methyltransferase and homocysteine</span> <span class="citation_source-journal">Atherosclerosis</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">204</span><span class="NLM_x">, </span> <span class="NLM_fpage">412</span><span class="NLM_x">–</span> <span class="NLM_lpage">417</span><span class="refDoi"> DOI: 10.1016/j.atherosclerosis.2008.09.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1016%2Fj.atherosclerosis.2008.09.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=18996527" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmtl2ltr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=204&publication_year=2009&pages=412-417&author=M.+Riedererauthor=W.+Erwaauthor=R.+Zimmermannauthor=S.+Frankauthor=R.+Zechner&title=Adipose+tissue+as+a+source+of+nicotinamide+N-methyltransferase+and+homocysteine&doi=10.1016%2Fj.atherosclerosis.2008.09.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Adipose tissue as a source of nicotinamide N-methyltransferase and homocysteine</span></div><div class="casAuthors">Riederer, Monika; Erwa, Wolfgang; Zimmermann, Robert; Frank, Sasa; Zechner, Rudolf</div><div class="citationInfo"><span class="NLM_cas:title">Atherosclerosis (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">204</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">412-417</span>CODEN:
                <span class="NLM_cas:coden">ATHSBL</span>;
        ISSN:<span class="NLM_cas:issn">0021-9150</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) catalyzes the conversion of nicotinamide to 1-methylnicotinamide and plays an important role in hepatic detoxification reactions.  Here we show that, in addn. to the liver, 3T3-L1 adipocytes as well as human and murine adipose tissue explants express high amts. of enzymically active NNMT.  NNMT mRNA levels and enzyme activity increased in 3T3-L1 cells in a differentiation-dependent manner.  Homocysteine, the atherogenic product of the NNMT-catalyzed reaction, was secreted from 3T3-L1 cells or adipose tissue cultures.  Homocysteine release increased during 3T3-L1 differentiation and was reduced when adipose tissue was treated with the NNMT inhibitor 1-methylnicotinamide.  Nicotinic acid (NA), a widely used drug to lower elevated plasma lipid levels, induced NNMT enzyme activity in white adipose tissue of mice.  In tissue culture nicotinamide treatment led to an increase in adipose tissue homocysteine secretion.  These data support the concept that adipose tissue NNMT contributes to the increased plasma homocysteine levels in patients treated with NA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAPqArSvc84rVg90H21EOLACvtfcHk0lh_7OAJ_Z-f7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmtl2ltr0%253D&md5=4dd8fbd5fbb95aab930ce2cf85d8c825</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.atherosclerosis.2008.09.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.atherosclerosis.2008.09.015%26sid%3Dliteratum%253Aachs%26aulast%3DRiederer%26aufirst%3DM.%26aulast%3DErwa%26aufirst%3DW.%26aulast%3DZimmermann%26aufirst%3DR.%26aulast%3DFrank%26aufirst%3DS.%26aulast%3DZechner%26aufirst%3DR.%26atitle%3DAdipose%2520tissue%2520as%2520a%2520source%2520of%2520nicotinamide%2520N-methyltransferase%2520and%2520homocysteine%26jtitle%3DAtherosclerosis%26date%3D2009%26volume%3D204%26spage%3D412%26epage%3D417%26doi%3D10.1016%2Fj.atherosclerosis.2008.09.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Pissios, P.</span><span> </span><span class="NLM_article-title">Nicotinamide N-methyltransferase: More than a vitamin B3 clearance enzyme</span> <span class="citation_source-journal">Trends Endocrinol. Metab.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">340</span><span class="NLM_x">–</span> <span class="NLM_lpage">353</span><span class="refDoi"> DOI: 10.1016/j.tem.2017.02.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1016%2Fj.tem.2017.02.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=28291578" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BC2sXivFehsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2017&pages=340-353&author=P.+Pissios&title=Nicotinamide+N-methyltransferase%3A+More+than+a+vitamin+B3+clearance+enzyme&doi=10.1016%2Fj.tem.2017.02.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Nicotinamide N-Methyltransferase: More Than a Vitamin B3 Clearance Enzyme</span></div><div class="casAuthors">Pissios, Pavlos</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Endocrinology and Metabolism</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">340-353</span>CODEN:
                <span class="NLM_cas:coden">TENME4</span>;
        ISSN:<span class="NLM_cas:issn">1043-2760</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Nicotinamide (NAM) N-methyltransferase (NNMT) was originally identified as the enzyme responsible for the methylation of NAM, one of the forms of vitamin B3.  Methylated NAM is eventually excreted from the body.  Recent evidence has expanded the role of NNMT beyond clearance of excess vitamin B3.  NNMT has been implicated in the regulation of multiple metabolic pathways in tissues such as adipose and liver as well as cancer cells through the consumption of Me donors and generation of active metabolites.  This review examines recent findings regarding the function of NNMT in physiol. and disease and highlights potential new avenues for therapeutic intervention.  Finally, key gaps in our knowledge about this enzymic system and future areas of investigation are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHRx9vhXz5LbVg90H21EOLACvtfcHk0lh_7OAJ_Z-f7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXivFehsrk%253D&md5=cf088bcc9434e006062d72de0c47c62a</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.tem.2017.02.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tem.2017.02.004%26sid%3Dliteratum%253Aachs%26aulast%3DPissios%26aufirst%3DP.%26atitle%3DNicotinamide%2520N-methyltransferase%253A%2520More%2520than%2520a%2520vitamin%2520B3%2520clearance%2520enzyme%26jtitle%3DTrends%2520Endocrinol.%2520Metab.%26date%3D2017%26volume%3D28%26spage%3D340%26epage%3D353%26doi%3D10.1016%2Fj.tem.2017.02.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Schmeisser, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mansfeld, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhlow, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weimer, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Priebe, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heiland, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Birringer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Groth, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Segref, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kanfi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Price, N. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmeisser, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schuster, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pfeiffer, A. F. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guthke, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Platzer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoppe, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cohen, H. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zarse, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sinclair, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ristow, M.</span><span> </span><span class="NLM_article-title">Role of sirtuins in lifespan regulation is linked to methylation of nicotinamide</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">693</span><span class="NLM_x">–</span> <span class="NLM_lpage">700</span><span class="refDoi"> DOI: 10.1038/nchembio.1352</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1038%2Fnchembio.1352" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=24077178" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFalu7bP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=693-700&author=K.+Schmeisserauthor=J.+Mansfeldauthor=D.+Kuhlowauthor=S.+Weimerauthor=S.+Priebeauthor=I.+Heilandauthor=M.+Birringerauthor=M.+Grothauthor=A.+Segrefauthor=Y.+Kanfiauthor=N.+L.+Priceauthor=S.+Schmeisserauthor=S.+Schusterauthor=A.+F.+H.+Pfeifferauthor=R.+Guthkeauthor=M.+Platzerauthor=T.+Hoppeauthor=H.+Y.+Cohenauthor=K.+Zarseauthor=D.+A.+Sinclairauthor=M.+Ristow&title=Role+of+sirtuins+in+lifespan+regulation+is+linked+to+methylation+of+nicotinamide&doi=10.1038%2Fnchembio.1352"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Role of sirtuins in lifespan regulation is linked to methylation of nicotinamide</span></div><div class="casAuthors">Schmeisser, Kathrin; Mansfeld, Johannes; Kuhlow, Doreen; Weimer, Sandra; Priebe, Steffen; Heiland, Ines; Birringer, Marc; Groth, Marco; Segref, Alexandra; Kanfi, Yariv; Price, Nathan L.; Schmeisser, Sebastian; Schuster, Stefan; Pfeiffer, Andreas F. H.; Guthke, Reinhard; Platzer, Matthias; Hoppe, Thorsten; Cohen, Haim Y.; Zarse, Kim; Sinclair, David A.; Ristow, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">693-700</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Sirtuins, a family of histone deacetylases, have a fiercely debated role in regulating lifespan.  In contrast with recent observations, here we find that overexpression of sir-2.1, the ortholog of mammalian SirT1, does extend Caenorhabditis elegans lifespan.  Sirtuins mandatorily convert NAD+ into nicotinamide (NAM).  We here find that NAM and its metabolite, 1-methylnicotinamide (MNA), extend C. elegans lifespan, even in the absence of sir-2.1.  We identify a previously unknown C. elegans nicotinamide-N-methyltransferase, encoded by a gene now named anmt-1, to generate MNA from NAM.  Disruption and overexpression of anmt-1 have opposing effects on lifespan independent of sirtuins, with loss of anmt-1 fully inhibiting sir-2.1-mediated lifespan extension.  MNA serves as a substrate for a newly identified aldehyde oxidase, GAD-3, to generate hydrogen peroxide, which acts as a mitohormetic reactive oxygen species signal to promote C. elegans longevity.  Taken together, sirtuin-mediated lifespan extension depends on methylation of NAM, providing an unexpected mechanistic role for sirtuins beyond histone deacetylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVNUIcRbjzP7Vg90H21EOLACvtfcHk0lgzhePB3EOCAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFalu7bP&md5=6a109a82f2141577cc460ff2e278e1c5</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1352&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1352%26sid%3Dliteratum%253Aachs%26aulast%3DSchmeisser%26aufirst%3DK.%26aulast%3DMansfeld%26aufirst%3DJ.%26aulast%3DKuhlow%26aufirst%3DD.%26aulast%3DWeimer%26aufirst%3DS.%26aulast%3DPriebe%26aufirst%3DS.%26aulast%3DHeiland%26aufirst%3DI.%26aulast%3DBirringer%26aufirst%3DM.%26aulast%3DGroth%26aufirst%3DM.%26aulast%3DSegref%26aufirst%3DA.%26aulast%3DKanfi%26aufirst%3DY.%26aulast%3DPrice%26aufirst%3DN.%2BL.%26aulast%3DSchmeisser%26aufirst%3DS.%26aulast%3DSchuster%26aufirst%3DS.%26aulast%3DPfeiffer%26aufirst%3DA.%2BF.%2BH.%26aulast%3DGuthke%26aufirst%3DR.%26aulast%3DPlatzer%26aufirst%3DM.%26aulast%3DHoppe%26aufirst%3DT.%26aulast%3DCohen%26aufirst%3DH.%2BY.%26aulast%3DZarse%26aufirst%3DK.%26aulast%3DSinclair%26aufirst%3DD.%2BA.%26aulast%3DRistow%26aufirst%3DM.%26atitle%3DRole%2520of%2520sirtuins%2520in%2520lifespan%2520regulation%2520is%2520linked%2520to%2520methylation%2520of%2520nicotinamide%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2013%26volume%3D9%26spage%3D693%26epage%3D700%26doi%3D10.1038%2Fnchembio.1352" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Chlopicki, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swies, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mogielnicki, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buczko, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bartus, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lomnicka, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adamus, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gebicki, J.</span><span> </span><span class="NLM_article-title">1-Methylnicotinamide (MNA), a primary metabolite of nicotinamide, exerts anti-thrombotic activity mediated by a cyclooxygenase-2/prostacyclin pathway</span> <span class="citation_source-journal">Br. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">152</span><span class="NLM_x">, </span> <span class="NLM_fpage">230</span><span class="NLM_x">–</span> <span class="NLM_lpage">239</span><span class="refDoi"> DOI: 10.1038/sj.bjp.0707383</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1038%2Fsj.bjp.0707383" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=17641676" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVWhsLzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=152&publication_year=2007&pages=230-239&author=S.+Chlopickiauthor=J.+Swiesauthor=A.+Mogielnickiauthor=W.+Buczkoauthor=M.+Bartusauthor=M.+Lomnickaauthor=J.+Adamusauthor=J.+Gebicki&title=1-Methylnicotinamide+%28MNA%29%2C+a+primary+metabolite+of+nicotinamide%2C+exerts+anti-thrombotic+activity+mediated+by+a+cyclooxygenase-2%2Fprostacyclin+pathway&doi=10.1038%2Fsj.bjp.0707383"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">1-Methylnicotinamide (MNA), a primary metabolite of nicotinamide, exerts anti-thrombotic activity mediated by a cyclooxygenase-2/prostacyclin pathway</span></div><div class="casAuthors">Chlopicki, S.; Swies, J.; Mogielnicki, A.; Buczko, W.; Bartus, M.; Lomnicka, M.; Adamus, J.; Gebicki, J.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">152</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">230-239</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">0007-1188</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">1-Methylnicotinamide (MNA) has been considered to be an inactive metabolite of nicotinamide.  Here we assessed the anti-thrombotic activity of MNA in vivo.  Antithrombotic action of MNA was studied in normotensive rats with extracorporeal thrombus formation (thrombolysis), in renovascular hypertensive rats with intraarterial thrombus formation (arterial thrombosis) and in a venous thrombosis model in rats (venous thrombosis).  MNA (3-100 mg kg-1) induced a dose-dependent and sustained thrombolytic response, assocd. with a rise in 6-keto-PGF1α in blood.  Various compds. structurally related to MNA were either inactive or weaker thrombolytics.  Rofecoxib (0.01-1 mg kg-1), dose-dependently inhibited the thrombolytic response of MNA, indomethacin (5 mg kg-1) abolished it, while L-NAME (5 mg kg-1) were without effect.  MNA (3-30 mg kg-1) also reduced arterial thrombosis and this effect was abrogated by indomethacin (2.5 mg kg-1) as well as by rofecoxib (1 mg kg-1).  MNA, however, did not affect venous thrombosis.  In vitro MNA did not modify platelet aggregation nor induce vasodilation.  MNA displayed a profile of anti-thrombotic activity in vivo that surpasses that of closely related compds.  MNA inhibited platelet-dependent thrombosis by a mechanism involving cyclooxygenase-2 and prostacyclin.  Our findings suggest that endogenous MNA, produced in the liver by nicotinamide N-methyltransferase, could be an endogenous activator of prostacyclin prodn. and thus may regulate thrombotic as well as inflammatory processes in the cardiovascular system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptun94rn-oo7Vg90H21EOLACvtfcHk0lgzhePB3EOCAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVWhsLzI&md5=ec9a668f372d4ddefaba82b6da551158</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjp.0707383&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjp.0707383%26sid%3Dliteratum%253Aachs%26aulast%3DChlopicki%26aufirst%3DS.%26aulast%3DSwies%26aufirst%3DJ.%26aulast%3DMogielnicki%26aufirst%3DA.%26aulast%3DBuczko%26aufirst%3DW.%26aulast%3DBartus%26aufirst%3DM.%26aulast%3DLomnicka%26aufirst%3DM.%26aulast%3DAdamus%26aufirst%3DJ.%26aulast%3DGebicki%26aufirst%3DJ.%26atitle%3D1-Methylnicotinamide%2520%2528MNA%2529%252C%2520a%2520primary%2520metabolite%2520of%2520nicotinamide%252C%2520exerts%2520anti-thrombotic%2520activity%2520mediated%2520by%2520a%2520cyclooxygenase-2%252Fprostacyclin%2520pathway%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2007%26volume%3D152%26spage%3D230%26epage%3D239%26doi%3D10.1038%2Fsj.bjp.0707383" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Domagala, T. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szeffler, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dobrucki, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dropinski, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Polanski, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leszczynska-Wiloch, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kotula-Horowitz, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wojciechowski, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wojnowski, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szczeklik, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kalinowski, L.</span><span> </span><span class="NLM_article-title">Nitric oxide production and endothelium-dependent vasorelaxation ameliorated by N1-methylnicotinamide in human blood vessels</span> <span class="citation_source-journal">Hypertension</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">825</span><span class="NLM_x">–</span> <span class="NLM_lpage">832</span><span class="refDoi"> DOI: 10.1161/HYPERTENSIONAHA.111.183210</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1161%2FHYPERTENSIONAHA.111.183210" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2012&pages=825-832&author=T.+B.+Domagalaauthor=A.+Szefflerauthor=L.+W.+Dobruckiauthor=J.+Dropinskiauthor=S.+Polanskiauthor=M.+Leszczynska-Wilochauthor=K.+Kotula-Horowitzauthor=J.+Wojciechowskiauthor=L.+Wojnowskiauthor=A.+Szczeklikauthor=L.+Kalinowski&title=Nitric+oxide+production+and+endothelium-dependent+vasorelaxation+ameliorated+by+N1-methylnicotinamide+in+human+blood+vessels&doi=10.1161%2FHYPERTENSIONAHA.111.183210"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1161%2FHYPERTENSIONAHA.111.183210&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1161%252FHYPERTENSIONAHA.111.183210%26sid%3Dliteratum%253Aachs%26aulast%3DDomagala%26aufirst%3DT.%2BB.%26aulast%3DSzeffler%26aufirst%3DA.%26aulast%3DDobrucki%26aufirst%3DL.%2BW.%26aulast%3DDropinski%26aufirst%3DJ.%26aulast%3DPolanski%26aufirst%3DS.%26aulast%3DLeszczynska-Wiloch%26aufirst%3DM.%26aulast%3DKotula-Horowitz%26aufirst%3DK.%26aulast%3DWojciechowski%26aufirst%3DJ.%26aulast%3DWojnowski%26aufirst%3DL.%26aulast%3DSzczeklik%26aufirst%3DA.%26aulast%3DKalinowski%26aufirst%3DL.%26atitle%3DNitric%2520oxide%2520production%2520and%2520endothelium-dependent%2520vasorelaxation%2520ameliorated%2520by%2520N1-methylnicotinamide%2520in%2520human%2520blood%2520vessels%26jtitle%3DHypertension%26date%3D2012%26volume%3D59%26spage%3D825%26epage%3D832%26doi%3D10.1161%2FHYPERTENSIONAHA.111.183210" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Hong, S. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moreno-Navarrete, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, X. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kikukawa, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tzameli, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prasad, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Asara, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fernandez-Real, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maratos-Flier, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pissios, P.</span><span> </span><span class="NLM_article-title">Nicotinamide N-methyltransferase regulates hepatic nutrient metabolism through Sirt1 protein stabilization</span> <span class="citation_source-journal">Nat. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">887</span><span class="NLM_x">–</span> <span class="NLM_lpage">894</span><span class="refDoi"> DOI: 10.1038/nm.3882</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1038%2Fnm.3882" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=26168293" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1WltLjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=887-894&author=S.+Y.+Hongauthor=J.+M.+Moreno-Navarreteauthor=X.+J.+Weiauthor=Y.+Kikukawaauthor=I.+Tzameliauthor=D.+Prasadauthor=Y.+Leeauthor=J.+M.+Asaraauthor=J.+M.+Fernandez-Realauthor=E.+Maratos-Flierauthor=P.+Pissios&title=Nicotinamide+N-methyltransferase+regulates+hepatic+nutrient+metabolism+through+Sirt1+protein+stabilization&doi=10.1038%2Fnm.3882"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Nicotinamide N-methyltransferase regulates hepatic nutrient metabolism through Sirt1 protein stabilization</span></div><div class="casAuthors">Hong, Shangyu; Moreno-Navarrete, Jose M.; Wei, Xiaojing; Kikukawa, Yusuke; Tzameli, Iphigenia; Prasad, Deepthi; Lee, Yoonjin; Asara, John M.; Fernandez-Real, Jose Manuel; Maratos-Flier, Eleftheria; Pissios, Pavlos</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">887-894</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (Nnmt) methylates nicotinamide, a form of vitamin B3, to produce N1-methylnicotinamide (MNAM).  Nnmt has emerged as a metabolic regulator in adipocytes, but its role in the liver, the tissue with the strongest Nnmt expression, is not known.  In spite of its overall high expression, here we find that hepatic expression of Nnmt is highly variable and correlates with multiple metabolic parameters in mice and humans.  Further, we find that suppression of hepatic Nnmt expression in vivo alters glucose and cholesterol metab. and that the metabolic effects of Nnmt in the liver are mediated by its product MNAM.  Supplementation of high-fat diet with MNAM decreases serum and liver cholesterol and liver triglycerides levels in mice.  Mechanistically, increasing Nnmt expression or MNAM levels stabilizes sirtuin 1 protein, an effect that is required for their metabolic benefits.  In summary, we describe here a novel regulatory pathway for vitamin B3 that could provide a new opportunity for metabolic disease therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaiKWlS3XZlrVg90H21EOLACvtfcHk0liDdzXmtY7MRQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1WltLjO&md5=d3f54ad774f4ea0bf07ce9bb389c0139</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnm.3882&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.3882%26sid%3Dliteratum%253Aachs%26aulast%3DHong%26aufirst%3DS.%2BY.%26aulast%3DMoreno-Navarrete%26aufirst%3DJ.%2BM.%26aulast%3DWei%26aufirst%3DX.%2BJ.%26aulast%3DKikukawa%26aufirst%3DY.%26aulast%3DTzameli%26aufirst%3DI.%26aulast%3DPrasad%26aufirst%3DD.%26aulast%3DLee%26aufirst%3DY.%26aulast%3DAsara%26aufirst%3DJ.%2BM.%26aulast%3DFernandez-Real%26aufirst%3DJ.%2BM.%26aulast%3DMaratos-Flier%26aufirst%3DE.%26aulast%3DPissios%26aufirst%3DP.%26atitle%3DNicotinamide%2520N-methyltransferase%2520regulates%2520hepatic%2520nutrient%2520metabolism%2520through%2520Sirt1%2520protein%2520stabilization%26jtitle%3DNat.%2520Med.%26date%3D2015%26volume%3D21%26spage%3D887%26epage%3D894%26doi%3D10.1038%2Fnm.3882" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Williams, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramsden, D. B.</span><span> </span><span class="NLM_article-title">Autotoxicity, methylation and a road to the prevention of Parkinson’s disease</span> <span class="citation_source-journal">J. Clin. Neurosci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">6</span><span class="NLM_x">–</span> <span class="NLM_lpage">11</span><span class="refDoi"> DOI: 10.1016/j.jocn.2004.10.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1016%2Fj.jocn.2004.10.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=15639403" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BD2MXitFansQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2005&pages=6-11&author=A.+C.+Williamsauthor=D.+B.+Ramsden&title=Autotoxicity%2C+methylation+and+a+road+to+the+prevention+of+Parkinson%E2%80%99s+disease&doi=10.1016%2Fj.jocn.2004.10.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Autotoxicity, methylation and a road to the prevention of Parkinson's disease</span></div><div class="casAuthors">Williams, Adrian C.; Ramsden, David B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Neuroscience</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">6-11</span>CODEN:
                <span class="NLM_cas:coden">JCNUE6</span>;
        ISSN:<span class="NLM_cas:issn">0967-5868</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Xenobiotic enzymes normally protect against toxins but on occasion can convert protoxins into toxins.  N-methylated pyridines (such as the N-methyl-4 phenylpyridinium ion (MPP+)) are well-established dopaminergic toxins.  The enzyme nicotinamide N-methyltransferase (NNMT) can covert otherwise harmless pyridines such as 4-phenylpyridine into MPP+ like compds.  This enzyme has recently been shown to be present in the human brain, which is a necessity for neurotoxicity, as charged compds. such as MPP+ cannot cross the blood brain barrier.  Moreover, it is present in increased concn. in the brain of patients with Parkinson's disease (PD).  This would increase MPP+ like compds. at the same time as decreasing intraneuronal nicotinamide, a neuroprotectant at several levels, thus creating a "multiple hit", as addnl. complex 1 of the mitochondrial complex would also be poisoned and starved of its major substrate, NAD.  Thus, PD may be a disease of autointoxication.  Xenobiotic enzyme inhibitors of NNMT, with or without dietary modification, would be a novel way to attempt primary prevention of PD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkCefdGhG03bVg90H21EOLACvtfcHk0lhkPd-OFL9QdA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXitFansQ%253D%253D&md5=0fe92fa41debe0b2f43d15b3c55ea6c7</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.jocn.2004.10.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jocn.2004.10.002%26sid%3Dliteratum%253Aachs%26aulast%3DWilliams%26aufirst%3DA.%2BC.%26aulast%3DRamsden%26aufirst%3DD.%2BB.%26atitle%3DAutotoxicity%252C%2520methylation%2520and%2520a%2520road%2520to%2520the%2520prevention%2520of%2520Parkinson%25E2%2580%2599s%2520disease%26jtitle%3DJ.%2520Clin.%2520Neurosci.%26date%3D2005%26volume%3D12%26spage%3D6%26epage%3D11%26doi%3D10.1016%2Fj.jocn.2004.10.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Parsons, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waring, R. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramsden, D. B.</span><span> </span><span class="NLM_article-title">Expression of nicotinamide N-methyltransferase (EC 2.1.1.1) in the parkinsonian brain</span> <span class="citation_source-journal">J. Neuropathol. Exp. Neurol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">61</span><span class="NLM_x">, </span> <span class="NLM_fpage">111</span><span class="NLM_x">–</span> <span class="NLM_lpage">124</span><span class="refDoi"> DOI: 10.1093/jnen/61.2.111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1093%2Fjnen%2F61.2.111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=11853016" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BD38XitFOku7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2002&pages=111-124&author=R.+B.+Parsonsauthor=M.+L.+Smithauthor=A.+C.+Williamsauthor=R.+H.+Waringauthor=D.+B.+Ramsden&title=Expression+of+nicotinamide+N-methyltransferase+%28EC+2.1.1.1%29+in+the+parkinsonian+brain&doi=10.1093%2Fjnen%2F61.2.111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Expression of nicotinamide N-methyltransferase (E.C. 2.1.1.1) in the Parkinsonian brain</span></div><div class="casAuthors">Parsons, Richard B.; Smith, Marie-Louise; Williams, Adrian C.; Waring, Rosemary H.; Ramsden, David B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Neuropathology and Experimental Neurology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">111-124</span>CODEN:
                <span class="NLM_cas:coden">JNENAD</span>;
        ISSN:<span class="NLM_cas:issn">0022-3069</span>.
    
            (<span class="NLM_cas:orgname">American Association of Neuropathologists, Inc.</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) has been proposed as a link between the environmental and genetic factors of Parkinson disease (PD).  Therefore, we explored the hypothesis that high levels of NNMT expression may predispose to the development of PD.  Regions of high mRNA expression were shown in the spinal cord, medulla, and temporal lobe, with lowest expression in the cerebellum, subthalamic nucleus, and caudate nucleus.  Using 2 NNMT antibodies, the protein was shown to be expressed in multipolar neurons in the temporal lobe, caudate nucleus, and spinal cord, granular neurons of the cerebellum, dopaminergic neurons in the substantia nigra, and in the axons of the third nerve.  Expression of NNMT was compared in PD and non-PD control cerebella and caudate nucleus.  PD tissue exhibited significantly increased levels of NNMT protein and activity.  PD disease duration was inversely correlated with the level of expression in cerebellum.  This is the first demonstration that patients with PD have higher levels of NNMT activity and protein in brain tissue than those without PD and that NNMT expression is assocd. with neurons that degenerate in PD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTQojjaTnox7Vg90H21EOLACvtfcHk0lhkPd-OFL9QdA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XitFOku7w%253D&md5=ac3a0e25e3ae442fc9d3e81df1757949</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1093%2Fjnen%2F61.2.111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjnen%252F61.2.111%26sid%3Dliteratum%253Aachs%26aulast%3DParsons%26aufirst%3DR.%2BB.%26aulast%3DSmith%26aufirst%3DM.%2BL.%26aulast%3DWilliams%26aufirst%3DA.%2BC.%26aulast%3DWaring%26aufirst%3DR.%2BH.%26aulast%3DRamsden%26aufirst%3DD.%2BB.%26atitle%3DExpression%2520of%2520nicotinamide%2520N-methyltransferase%2520%2528EC%25202.1.1.1%2529%2520in%2520the%2520parkinsonian%2520brain%26jtitle%3DJ.%2520Neuropathol.%2520Exp.%2520Neurol.%26date%3D2002%26volume%3D61%26spage%3D111%26epage%3D124%26doi%3D10.1093%2Fjnen%2F61.2.111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Kraus, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kong, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banks, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodgers, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pirinen, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pulinilkunnil, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gong, F. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cen, Y. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sauve, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Asara, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peroni, O. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Monia, B. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhanot, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alhonen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Puigserver, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kahn, B. B.</span><span> </span><span class="NLM_article-title">Nicotinamide N-methyltransferase knockdown protects against diet-induced obesity</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">508</span><span class="NLM_x">, </span> <span class="NLM_fpage">258</span><span class="NLM_x">–</span> <span class="NLM_lpage">262</span><span class="refDoi"> DOI: 10.1038/nature13198</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1038%2Fnature13198" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=24717514" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtlWhsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=508&publication_year=2014&pages=258-262&author=D.+Krausauthor=Q.+Yangauthor=D.+Kongauthor=A.+S.+Banksauthor=L.+Zhangauthor=J.+T.+Rodgersauthor=E.+Pirinenauthor=T.+C.+Pulinilkunnilauthor=F.+Y.+Gongauthor=Y.+C.+Wangauthor=Y.+N.+Cenauthor=A.+A.+Sauveauthor=J.+M.+Asaraauthor=O.+D.+Peroniauthor=B.+P.+Moniaauthor=S.+Bhanotauthor=L.+Alhonenauthor=P.+Puigserverauthor=B.+B.+Kahn&title=Nicotinamide+N-methyltransferase+knockdown+protects+against+diet-induced+obesity&doi=10.1038%2Fnature13198"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Nicotinamide N-methyltransferase knockdown protects against diet-induced obesity</span></div><div class="casAuthors">Kraus, Daniel; Yang, Qin; Kong, Dong; Banks, Alexander S.; Zhang, Lin; Rodgers, Joseph T.; Pirinen, Eija; Pulinilkunnil, Thomas C.; Gong, Fengying; Wang, Ya-chin; Cen, Yana; Sauve, Anthony A.; Asara, John M.; Peroni, Odile D.; Monia, Brett P.; Bhanot, Sanjay; Alhonen, Leena; Puigserver, Pere; Kahn, Barbara B.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">508</span>
        (<span class="NLM_cas:issue">7495</span>),
    <span class="NLM_cas:pages">258-262</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">In obesity and type 2 diabetes, Glut4 glucose transporter expression is decreased selectively in adipocytes.  Adipose-specific knockout or overexpression of Glut4 alters systemic insulin sensitivity.  Here we show, using DNA array analyses, that nicotinamide N-methyltransferase (Nnmt) is the most strongly reciprocally regulated gene when comparing gene expression in white adipose tissue (WAT) from adipose-specific Glut4-knockout or adipose-specific Glut4-overexpressing mice with their resp. controls.  NNMT methylates nicotinamide (vitamin B3) using S-adenosylmethionine (SAM) as a Me donor.  Nicotinamide is a precursor of NAD+, an important cofactor linking cellular redox states with energy metab.  SAM provides propylamine for polyamine biosynthesis and donates a Me group for histone methylation.  Polyamine flux including synthesis, catabolism and excretion, is controlled by the rate-limiting enzymes ornithine decarboxylase (ODC) and spermidine-spermine N1-acetyltransferase (SSAT; encoded by Sat1) and by polyamine oxidase (PAO), and has a major role in energy metab.  We report that NNMT expression is increased in WAT and liver of obese and diabetic mice.  Nnmt knockdown in WAT and liver protects against diet-induced obesity by augmenting cellular energy expenditure.  NNMT inhibition increases adipose SAM and NAD+ levels and upregulates ODC and SSAT activity as well as expression, owing to the effects of NNMT on histone H3 lysine 4 methylation in adipose tissue.  Direct evidence for increased polyamine flux resulting from NNMT inhibition includes elevated urinary excretion and adipocyte secretion of diacetylspermine, a product of polyamine metab.  NNMT inhibition in adipocytes increases oxygen consumption in an ODC-, SSAT- and PAO-dependent manner.  Thus, NNMT is a novel regulator of histone methylation, polyamine flux and NAD+-dependent SIRT1 signaling, and is a unique and attractive target for treating obesity and type 2 diabetes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKuWPUvPnIqLVg90H21EOLACvtfcHk0liKbK7gYhMtKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtlWhsL8%253D&md5=c37b6137cd4537a0f6f1701a5a212dd6</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnature13198&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13198%26sid%3Dliteratum%253Aachs%26aulast%3DKraus%26aufirst%3DD.%26aulast%3DYang%26aufirst%3DQ.%26aulast%3DKong%26aufirst%3DD.%26aulast%3DBanks%26aufirst%3DA.%2BS.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DRodgers%26aufirst%3DJ.%2BT.%26aulast%3DPirinen%26aufirst%3DE.%26aulast%3DPulinilkunnil%26aufirst%3DT.%2BC.%26aulast%3DGong%26aufirst%3DF.%2BY.%26aulast%3DWang%26aufirst%3DY.%2BC.%26aulast%3DCen%26aufirst%3DY.%2BN.%26aulast%3DSauve%26aufirst%3DA.%2BA.%26aulast%3DAsara%26aufirst%3DJ.%2BM.%26aulast%3DPeroni%26aufirst%3DO.%2BD.%26aulast%3DMonia%26aufirst%3DB.%2BP.%26aulast%3DBhanot%26aufirst%3DS.%26aulast%3DAlhonen%26aufirst%3DL.%26aulast%3DPuigserver%26aufirst%3DP.%26aulast%3DKahn%26aufirst%3DB.%2BB.%26atitle%3DNicotinamide%2520N-methyltransferase%2520knockdown%2520protects%2520against%2520diet-induced%2520obesity%26jtitle%3DNature%26date%3D2014%26volume%3D508%26spage%3D258%26epage%3D262%26doi%3D10.1038%2Fnature13198" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Ulanovskaya, O. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zuhl, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cravatt, B. F.</span><span> </span><span class="NLM_article-title">NNMT promotes epigenetic remodeling in cancer by creating a metabolic methylation sink</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">300</span><span class="NLM_x">–</span> <span class="NLM_lpage">306</span><span class="refDoi"> DOI: 10.1038/nchembio.1204</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1038%2Fnchembio.1204" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=23455543" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtlKnsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=300-306&author=O.+A.+Ulanovskayaauthor=A.+M.+Zuhlauthor=B.+F.+Cravatt&title=NNMT+promotes+epigenetic+remodeling+in+cancer+by+creating+a+metabolic+methylation+sink&doi=10.1038%2Fnchembio.1204"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">NNMT promotes epigenetic remodeling in cancer by creating a metabolic methylation sink</span></div><div class="casAuthors">Ulanovskaya, Olesya A.; Zuhl, Andrea M.; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">300-306</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) is overexpressed in a variety of human cancers, where it contributes to tumorigenesis by a mechanism that is still poorly understood.  Here we show using metabolomics that NNMT impairs the methylation potential of cancer cells by consuming Me units from S-adenosyl methionine to create the stable metabolic product 1-methylnicotinamide.  As a result, NNMT-expressing cancer cells have an altered epigenetic state that includes hypomethylated histones and other cancer-related proteins combined with heightened expression of protumorigenic gene products.  Our findings thus point to a direct mechanistic link between the deregulation of a metabolic enzyme and widespread changes in the methylation landscape of cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsOQPNFxw5kbVg90H21EOLACvtfcHk0liKbK7gYhMtKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtlKnsbs%253D&md5=8775d8b27fba0d039b0f6da7244d28a4</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1204&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1204%26sid%3Dliteratum%253Aachs%26aulast%3DUlanovskaya%26aufirst%3DO.%2BA.%26aulast%3DZuhl%26aufirst%3DA.%2BM.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DNNMT%2520promotes%2520epigenetic%2520remodeling%2520in%2520cancer%2520by%2520creating%2520a%2520metabolic%2520methylation%2520sink%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2013%26volume%3D9%26spage%3D300%26epage%3D306%26doi%3D10.1038%2Fnchembio.1204" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Markert, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fuller, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gillespie, G. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bubien, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McLean, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hong, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gullans, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mapstone, T. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benos, D. J.</span><span> </span><span class="NLM_article-title">Differential gene expression profiling in human brain tumors</span> <span class="citation_source-journal">Physiol. Genomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">21</span><span class="NLM_x">–</span> <span class="NLM_lpage">33</span><span class="refDoi"> DOI: 10.1152/physiolgenomics.2001.5.1.21</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1152%2Fphysiolgenomics.2001.5.1.21" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=11161003" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BD3MXks1yksrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2001&pages=21-33&author=J.+M.+Markertauthor=C.+M.+Fullerauthor=G.+Y.+Gillespieauthor=J.+K.+Bubienauthor=L.+A.+McLeanauthor=R.+L.+Hongauthor=K.+Leeauthor=S.+R.+Gullansauthor=T.+B.+Mapstoneauthor=D.+J.+Benos&title=Differential+gene+expression+profiling+in+human+brain+tumors&doi=10.1152%2Fphysiolgenomics.2001.5.1.21"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Differential gene expression profiling in human brain tumors</span></div><div class="casAuthors">Markert, James M.; Fuller, Catherine M.; Gillespie, G. Yancey; Bubien, James K.; McLean, Lee Anne; Hong, Robert L.; Lee, Kailin; Gullans, Steven R.; Mapstone, Timothy B.; Benos, Dale J.</div><div class="citationInfo"><span class="NLM_cas:title">Physiological Genomics [online computer file]</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">21-33</span>CODEN:
                <span class="NLM_cas:coden">PHGEFP</span>;
        ISSN:<span class="NLM_cas:issn">1094-8341</span>.
    
            (<span class="NLM_cas:orgname">American Physiological Society</span>)
        </div><div class="casAbstract">Gene expression profiling of three human temporal lobe brain tissue samples (normal) and four primary glioblastoma multiforme (GBM) tumors using oligonucleotide microarrays was done.  Moreover, confirmation of altered expression was performed by whole cell patch clamp, immunohistochem. staining, and RT-PCR.  Our results identified several ion and solute transport-related genes, such as N-methyl-D-aspartate (NMDA) receptors, α-amino-3-hydroxy-5-methyl-4-isoxazole propionate (AMPA)-2 receptors, GABAA receptor subunits α3, β1, β2, and β3, the glutamate transporter, the glutamate/aspartate transporter II, the potassium channel KV2.1, hKVb3, and the sodium/proton exchanger 1 (NHE-1), that are all downregulated in the tumors compared with the normal tissues.  In contrast, aquaporin-1, possibly aquaporins-3 and -5, and GLUT-3 message appeared upregulated in the tumors.  Our results also confirmed previous work showing that osteopontin, nicotinamide N-methyltransferase, murine double minute 2 (MDM2), and epithelin (granulin) are upregulated in GBMs.  We also demonstrate for the first time that the cytokine and p53 binding protein, macrophage migration inhibitory factor (MIF), appears upregulated in GBMs.  These results indicate that the modulation of ion and solute transport genes and heretofore unsuspected cytokines (i.e., MIF) may have profound implications for brain tumor cell biol. and thus may identify potential useful therapeutic targets in GBMs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoa8TEeUcqEvrVg90H21EOLACvtfcHk0liKbK7gYhMtKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXks1yksrw%253D&md5=4846aec92fd8673b96d9cb6a97219912</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1152%2Fphysiolgenomics.2001.5.1.21&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1152%252Fphysiolgenomics.2001.5.1.21%26sid%3Dliteratum%253Aachs%26aulast%3DMarkert%26aufirst%3DJ.%2BM.%26aulast%3DFuller%26aufirst%3DC.%2BM.%26aulast%3DGillespie%26aufirst%3DG.%2BY.%26aulast%3DBubien%26aufirst%3DJ.%2BK.%26aulast%3DMcLean%26aufirst%3DL.%2BA.%26aulast%3DHong%26aufirst%3DR.%2BL.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DGullans%26aufirst%3DS.%2BR.%26aulast%3DMapstone%26aufirst%3DT.%2BB.%26aulast%3DBenos%26aufirst%3DD.%2BJ.%26atitle%3DDifferential%2520gene%2520expression%2520profiling%2520in%2520human%2520brain%2520tumors%26jtitle%3DPhysiol.%2520Genomics%26date%3D2001%26volume%3D5%26spage%3D21%26epage%3D33%26doi%3D10.1152%2Fphysiolgenomics.2001.5.1.21" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Xu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moatamed, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caldwell, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walker, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kraiem, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taki, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brent, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hershman, J. M.</span><span> </span><span class="NLM_article-title">Enhanced expression of nicotinamide N-methyltransferase in human papillary thyroid carcinoma cells</span> <span class="citation_source-journal">J. Clin. Endocrinol. Metab.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">88</span><span class="NLM_x">, </span> <span class="NLM_fpage">4990</span><span class="NLM_x">–</span> <span class="NLM_lpage">4996</span><span class="refDoi"> DOI: 10.1210/jc.2002-021843</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1210%2Fjc.2002-021843" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=14557485" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BD3sXoslemu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=88&publication_year=2003&pages=4990-4996&author=J.+Xuauthor=F.+Moatamedauthor=J.+S.+Caldwellauthor=J.+R.+Walkerauthor=Z.+Kraiemauthor=K.+Takiauthor=G.+A.+Brentauthor=J.+M.+Hershman&title=Enhanced+expression+of+nicotinamide+N-methyltransferase+in+human+papillary+thyroid+carcinoma+cells&doi=10.1210%2Fjc.2002-021843"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Enhanced expression of nicotinamide N-methyltransferase in human papillary thyroid carcinoma cells</span></div><div class="casAuthors">Xu, Jimin; Moatamed, Farhad; Caldwell, Jeremy S.; Walker, John R.; Kraiem, Zaki; Taki, Katsumi; Brent, Gregory A.; Hershman, Jerome M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Endocrinology and Metabolism</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">88</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4990-4996</span>CODEN:
                <span class="NLM_cas:coden">JCEMAZ</span>;
        ISSN:<span class="NLM_cas:issn">0021-972X</span>.
    
            (<span class="NLM_cas:orgname">Endocrine Society</span>)
        </div><div class="casAbstract">To gain an understanding of the mol. pathogenesis of thyroid cancer, we used DNA microarray to study the expression profiles of 10 different human thyroid carcinoma cell lines.  These included papillary lines BHP 2-7, BHP 7-13, BHP 10-3, BHP 18-21, NPA 87, and TPC1; anaplastic lines ARO 81-1 and DRO 90-1; follicular line WRO 82-1; and medullary line HRO 85-1.  Among the genes with increased expression in the cancer cell lines, a gene coding for nicotinamide N-methyltransferase (NNMT) was identified for being highly expressed only in the papillary cell lines.  NNMT catalyzes N-methylation of nicotinamide and other structurally related compds. and is highly expressed in the human liver.  The results were further confirmed by semiquant. RT-PCR and Northern blot anal.  NNMT catalytic activities were detd. in all of the cells described above and in addnl. cell lines.  Significantly higher NNMT enzyme activities were detected in eight of 10 of the papillary lines and three of six of the follicular cell lines tested.  Normal thyroid tissue, thyroid primary cultures, anaplastic cancer cells, and medullary cancer cells showed no or low enzyme activity.  Immunohistochem. staining for NNMT of human thyroid specimens showed strong and abundant cytoplasmic reactions in the sections of papillary carcinomas, and weak or scanty reaction in the normal thyroid tissues.  These results indicate that NNMT is a potential biomarker for papillary thyroid carcinoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6AB9Lmnxdd7Vg90H21EOLACvtfcHk0li5oI9vha3wEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXoslemu7Y%253D&md5=40b3d7c6b5abc7073ffe40791269231f</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1210%2Fjc.2002-021843&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1210%252Fjc.2002-021843%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DMoatamed%26aufirst%3DF.%26aulast%3DCaldwell%26aufirst%3DJ.%2BS.%26aulast%3DWalker%26aufirst%3DJ.%2BR.%26aulast%3DKraiem%26aufirst%3DZ.%26aulast%3DTaki%26aufirst%3DK.%26aulast%3DBrent%26aufirst%3DG.%2BA.%26aulast%3DHershman%26aufirst%3DJ.%2BM.%26atitle%3DEnhanced%2520expression%2520of%2520nicotinamide%2520N-methyltransferase%2520in%2520human%2520papillary%2520thyroid%2520carcinoma%2520cells%26jtitle%3DJ.%2520Clin.%2520Endocrinol.%2520Metab.%26date%3D2003%26volume%3D88%26spage%3D4990%26epage%3D4996%26doi%3D10.1210%2Fjc.2002-021843" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Tomida, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mikami, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takeuchi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nishimura, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Akiyama, H.</span><span> </span><span class="NLM_article-title">Serum levels of nicotinamide N-methyltransferase in patients with lung cancer</span> <span class="citation_source-journal">J. Cancer Res. Clin. Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">135</span><span class="NLM_x">, </span> <span class="NLM_fpage">1223</span><span class="NLM_x">–</span> <span class="NLM_lpage">1229</span><span class="refDoi"> DOI: 10.1007/s00432-009-0563-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1007%2Fs00432-009-0563-y" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2009&pages=1223-1229&author=M.+Tomidaauthor=I.+Mikamiauthor=S.+Takeuchiauthor=H.+Nishimuraauthor=H.+Akiyama&title=Serum+levels+of+nicotinamide+N-methyltransferase+in+patients+with+lung+cancer&doi=10.1007%2Fs00432-009-0563-y"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1007%2Fs00432-009-0563-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00432-009-0563-y%26sid%3Dliteratum%253Aachs%26aulast%3DTomida%26aufirst%3DM.%26aulast%3DMikami%26aufirst%3DI.%26aulast%3DTakeuchi%26aufirst%3DS.%26aulast%3DNishimura%26aufirst%3DH.%26aulast%3DAkiyama%26aufirst%3DH.%26atitle%3DSerum%2520levels%2520of%2520nicotinamide%2520N-methyltransferase%2520in%2520patients%2520with%2520lung%2520cancer%26jtitle%3DJ.%2520Cancer%2520Res.%2520Clin.%2520Oncol.%26date%3D2009%26volume%3D135%26spage%3D1223%26epage%3D1229%26doi%3D10.1007%2Fs00432-009-0563-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Kim, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hong, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, E. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, Y. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roh, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Do, I. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joh, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, D. S.</span><span> </span><span class="NLM_article-title">Expression of nicotinamide N-methyltransferase in hepatocellular carcinoma is associated with poor prognosis</span> <span class="citation_source-journal">J. Exp. Clin. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">20</span><span class="refDoi"> DOI: 10.1186/1756-9966-28-20</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1186%2F1756-9966-28-20" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=19216803" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A280%3ADC%252BD1M3hslejtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2009&pages=20&author=J.+Kimauthor=S.+J.+Hongauthor=E.+K.+Limauthor=Y.+S.+Yuauthor=S.+W.+Kimauthor=J.+H.+Rohauthor=I.+G.+Doauthor=J.+W.+Johauthor=D.+S.+Kim&title=Expression+of+nicotinamide+N-methyltransferase+in+hepatocellular+carcinoma+is+associated+with+poor+prognosis&doi=10.1186%2F1756-9966-28-20"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Expression of nicotinamide N-methyltransferase in hepatocellular carcinoma is associated with poor prognosis</span></div><div class="casAuthors">Kim Jongmin; Hong Seok Joo; Lim Eun Kyung; Yu Yun-Suk; Kim Seung Whan; Roh Ji Hyeon; Do In-Gu; Joh Jae-Won; Kim Dae Shick</div><div class="citationInfo"><span class="NLM_cas:title">Journal of experimental & clinical cancer research : CR</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">20</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Hepatocellular carcinoma (HCC) is the most common tumor in the adult liver, with high relapse and mortality rates despite diverse treatment modalities.  In this study, nicotinamide N-methyltransferase (NNMT), a key enzyme in drug metabolism, was investigated as a potential prognostic factor.  METHODS:  Frozen tumors and non-cancerous surrounding tissues from 120 patients with primary HCC were studied.  Expressions of NNMT and internal control genes were measured by real-time reverse-transcription PCR (RT-PCR).  The relationship of NNMT mRNA level with clinicopathologic parameters and clinical outcome was evaluated.  RESULTS:  NNMT mRNA level is markedly reduced in HCCs compared to non-cancerous surrounding tissues (P < 0.0001), and NNMT expression in tumors was significantly correlated with tumor stage (P = 0.010).  Moreover, stratification of patients based on tumor NNMT mRNA levels revealed that the patients who expressed higher NNMT mRNA levels tended to have a shorter overall survival (OS) time (P = 0.053) and a significantly shorter disease-free survival (DFS) time (P = 0.016).  Both NNMT expression (P = 0.0096) and tumor stage (P = 0.0017) were found to be significant prognostic factors for DFS in a multivariate analysis.  CONCLUSION:  The results of this study indicated that NNMT gene expression is associated with tumor stage and DFS time in HCC cases.  Because of the broad substrate specificity of NNMT, which could alter the efficacy and adverse effects of chemotherapy, NNMT merits further investigation regarding its role as a prognostic factor with a larger cohort of HCC patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSTRP98CJkGv1QKX3ooHxWjfW6udTcc2eZzGY_3jGGzorntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1M3hslejtw%253D%253D&md5=95063b5fa31780728476e4142f89603c</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1186%2F1756-9966-28-20&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1756-9966-28-20%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DHong%26aufirst%3DS.%2BJ.%26aulast%3DLim%26aufirst%3DE.%2BK.%26aulast%3DYu%26aufirst%3DY.%2BS.%26aulast%3DKim%26aufirst%3DS.%2BW.%26aulast%3DRoh%26aufirst%3DJ.%2BH.%26aulast%3DDo%26aufirst%3DI.%2BG.%26aulast%3DJoh%26aufirst%3DJ.%2BW.%26aulast%3DKim%26aufirst%3DD.%2BS.%26atitle%3DExpression%2520of%2520nicotinamide%2520N-methyltransferase%2520in%2520hepatocellular%2520carcinoma%2520is%2520associated%2520with%2520poor%2520prognosis%26jtitle%3DJ.%2520Exp.%2520Clin.%2520Cancer%2520Res.%26date%3D2009%26volume%3D28%26spage%3D20%26doi%3D10.1186%2F1756-9966-28-20" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Yao, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tabuchi, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nagashima, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baba, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakaigawa, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ishiguro, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamada, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inayama, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kishida, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hattori, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamada-Okabe, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kubota, Y.</span><span> </span><span class="NLM_article-title">Gene expression analysis of renal carcinoma: adipose differentiation-related protein as a potential diagnostic and prognostic biomarker for clear-cell renal carcinoma</span> <span class="citation_source-journal">J. Pathol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">205</span><span class="NLM_x">, </span> <span class="NLM_fpage">377</span><span class="NLM_x">–</span> <span class="NLM_lpage">387</span><span class="refDoi"> DOI: 10.1002/path.1693</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1002%2Fpath.1693" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=15682440" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BD2MXit1Oruro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=205&publication_year=2005&pages=377-387&author=M.+Yaoauthor=H.+Tabuchiauthor=Y.+Nagashimaauthor=M.+Babaauthor=N.+Nakaigawaauthor=H.+Ishiguroauthor=K.+Hamadaauthor=Y.+Inayamaauthor=T.+Kishidaauthor=K.+Hattoriauthor=H.+Yamada-Okabeauthor=Y.+Kubota&title=Gene+expression+analysis+of+renal+carcinoma%3A+adipose+differentiation-related+protein+as+a+potential+diagnostic+and+prognostic+biomarker+for+clear-cell+renal+carcinoma&doi=10.1002%2Fpath.1693"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Gene expression analysis of renal carcinoma: adipose differentiation-related protein as a potential diagnostic and prognostic biomarker for clear-cell renal carcinoma</span></div><div class="casAuthors">Yao, Masahiro; Tabuchi, Hisahiro; Nagashima, Yoji; Baba, Masaya; Nakaigawa, Noburo; Ishiguro, Hitoshi; Hamada, Kenji; Inayama, Yoshiaki; Kishida, Takeshi; Hattori, Keiko; Yamada-Okabe, Hisafumi; Kubota, Yoshinobu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pathology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">205</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">377-387</span>CODEN:
                <span class="NLM_cas:coden">JPTLAS</span>;
        ISSN:<span class="NLM_cas:issn">0022-3417</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Ltd.</span>)
        </div><div class="casAbstract">The gene expression profiles of 33 renal cell carcinomas (RCCs) and nine normal kidney samples were examd. using high-d. oligonucleotide microarrays in an attempt to identify biomol. markers for the diagnosis of tumor subtypes and also for prediction of prognosis.  Hierarchical clustering demonstrated that clear-cell RCC, chromophobe RCC, and normal kidney tissue showed distinctive gene expression profiles.  The mean expression levels of 149 of 12,500 genes were more than three times higher in clear-cell RCC than in chromophobe RCC and normal kidney tissue.  Among the genes whose expression was upregulated in clear-cell RCC, adipose differentiation-related protein (ADFP) and nicotinamide N-methyltransferase (NNMT) were selected for further anal.  Consistent with the results of the microarray, increased levels of ADFP and NNMT mRNA were found more frequently in clear-cell RCCs than in other non-clear-cell tumor subtypes using real-time quant. PCR.  Immunohistochem. for ADFP showed strong and unique tumor cell staining patterns in the majority of clear-cell RCCs.  More importantly, patients bearing tumors with higher AFDP mRNA levels showed significantly better survival in both univariate and multivariate analyses.  ADFP is a lipid storage droplet-assocd. protein and its transcription is considered to be regulated by the von Hippel-Lindau/hypoxia-inducible factor pathway.  It is known that clear-cell RCC contains abundant lipids and cholesterols.  Thus it is likely that sustained upregulation of ADFP following VHL inactivation is involved in the morphol. appearance of clear-cell RCC.  Moreover ADFP expression status may provide useful prognostic information as a biomol. marker in patients with clear-cell RCC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZJ-PjqY6hHLVg90H21EOLACvtfcHk0limXL9_-g1hDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXit1Oruro%253D&md5=30d95e39fda0b3d35b1dcb1aece0a12e</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1002%2Fpath.1693&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpath.1693%26sid%3Dliteratum%253Aachs%26aulast%3DYao%26aufirst%3DM.%26aulast%3DTabuchi%26aufirst%3DH.%26aulast%3DNagashima%26aufirst%3DY.%26aulast%3DBaba%26aufirst%3DM.%26aulast%3DNakaigawa%26aufirst%3DN.%26aulast%3DIshiguro%26aufirst%3DH.%26aulast%3DHamada%26aufirst%3DK.%26aulast%3DInayama%26aufirst%3DY.%26aulast%3DKishida%26aufirst%3DT.%26aulast%3DHattori%26aufirst%3DK.%26aulast%3DYamada-Okabe%26aufirst%3DH.%26aulast%3DKubota%26aufirst%3DY.%26atitle%3DGene%2520expression%2520analysis%2520of%2520renal%2520carcinoma%253A%2520adipose%2520differentiation-related%2520protein%2520as%2520a%2520potential%2520diagnostic%2520and%2520prognostic%2520biomarker%2520for%2520clear-cell%2520renal%2520carcinoma%26jtitle%3DJ.%2520Pathol.%26date%3D2005%26volume%3D205%26spage%3D377%26epage%3D387%26doi%3D10.1002%2Fpath.1693" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Tang, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, W. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lai, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, J. Y.</span><span> </span><span class="NLM_article-title">Nicotinamide N-methyltransferase induces cellular invasion through activating matrix metalloproteinase-2 expression in clear cell renal cell carcinoma cells</span> <span class="citation_source-journal">Carcinogenesis</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">138</span><span class="NLM_x">–</span> <span class="NLM_lpage">145</span><span class="refDoi"> DOI: 10.1093/carcin/bgq225</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1093%2Fcarcin%2Fbgq225" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=21045016" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVSktrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2011&pages=138-145&author=S.+W.+Tangauthor=T.+C.+Yangauthor=W.+C.+Linauthor=W.+H.+Changauthor=C.+C.+Wangauthor=M.+K.+Laiauthor=J.+Y.+Lin&title=Nicotinamide+N-methyltransferase+induces+cellular+invasion+through+activating+matrix+metalloproteinase-2+expression+in+clear+cell+renal+cell+carcinoma+cells&doi=10.1093%2Fcarcin%2Fbgq225"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Nicotinamide N-methyltransferase induces cellular invasion through activating matrix metalloproteinase-2 expression in clear cell renal cell carcinoma cells</span></div><div class="casAuthors">Tang, Sai-Wen; Yang, Tsung-Cheng; Lin, Wei-Chou; Chang, Wen-Hsin; Wang, Chung-Chieh; Lai, Ming-Kuen; Lin, Jung-Yaw</div><div class="citationInfo"><span class="NLM_cas:title">Carcinogenesis</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">138-145</span>CODEN:
                <span class="NLM_cas:coden">CRNGDP</span>;
        ISSN:<span class="NLM_cas:issn">0143-3334</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) was recently identified as one clear cell renal cell carcinoma (ccRCC)-assocd. gene by analyzing full-length complementary DNA-enriched libraries of ccRCC tissues.  The aim of this study is to investigate the potential role of NNMT in cellular invasion.  A strong NNMT expression is accompanied with a high invasive activity in ccRCC cell lines, and small interfering RNA-mediated NNMT knockdown effectively suppressed the invasive capacity of ccRCC cells, whereas NNMT overexpression markedly enhanced that of human embryonic kidney 293 (HEK293) cells.  A pos. correlation between the expression of NNMT and matrix metallopeptidase (MMP)-2 was found in ccRCC cell lines and clin. tissues.  The treatment of blocking antibody or inhibitor specific to MMP-2 significantly suppressed NNMT-dependent cellular invasion in HEK293 cells.  Furthermore, SP-1-binding region of MMP-2 promoter was found to be essential in NNMT-induced MMP-2 expression.  The specific inhibitors of PI3K/Akt signaling markedly decreased the binding of SP1 to MMP-2 promoter as shown by chromatin immunopptn. assay.  We also demonstrated that PI3K/Akt pathway plays a role in NNMT-dependent cellular invasion and MMP-2 activation.  Moreover, short hairpin RNA-mediated knockdown of NNMT expression efficiently inhibited the growth and metastasis of ccRCC cells in non-obese diabetic severe combined immunodeficiency mice.  Taken together, the present study suggests that NNMT has a crucial role in cellular invasion via activating PI3K/Akt/SP1/MMP-2 pathway in ccRCC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoN72MlY14gnbVg90H21EOLACvtfcHk0lhwNJrZoiZEvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVSktrc%253D&md5=c0a4aedaa1f85ffb7ee0eed126940799</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1093%2Fcarcin%2Fbgq225&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fcarcin%252Fbgq225%26sid%3Dliteratum%253Aachs%26aulast%3DTang%26aufirst%3DS.%2BW.%26aulast%3DYang%26aufirst%3DT.%2BC.%26aulast%3DLin%26aufirst%3DW.%2BC.%26aulast%3DChang%26aufirst%3DW.%2BH.%26aulast%3DWang%26aufirst%3DC.%2BC.%26aulast%3DLai%26aufirst%3DM.%2BK.%26aulast%3DLin%26aufirst%3DJ.%2BY.%26atitle%3DNicotinamide%2520N-methyltransferase%2520induces%2520cellular%2520invasion%2520through%2520activating%2520matrix%2520metalloproteinase-2%2520expression%2520in%2520clear%2520cell%2520renal%2520cell%2520carcinoma%2520cells%26jtitle%3DCarcinogenesis%26date%3D2011%26volume%3D32%26spage%3D138%26epage%3D145%26doi%3D10.1093%2Fcarcin%2Fbgq225" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Jang, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cho, H. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, Y. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ha, W. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, H. W.</span><span> </span><span class="NLM_article-title">The differential proteome profile of stomach cancer: Identification of the biomarker candidates</span> <span class="citation_source-journal">Oncol. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">491</span><span class="NLM_x">–</span> <span class="NLM_lpage">499</span><span class="refDoi"> DOI: 10.3727/0965040042380441</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.3727%2F0965040042380441" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=15559763" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVyktr3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2004&pages=491-499&author=J.+S.+Jangauthor=H.+Y.+Choauthor=Y.+J.+Leeauthor=W.+S.+Haauthor=H.+W.+Kim&title=The+differential+proteome+profile+of+stomach+cancer%3A+Identification+of+the+biomarker+candidates&doi=10.3727%2F0965040042380441"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">The differential proteome profile of stomach cancer: identification of the biomarker candidates</span></div><div class="casAuthors">Jang, Joung S.; Cho, Hee Y.; Lee, Young J.; Ha, Woo S.; Kim, Hwal W.</div><div class="citationInfo"><span class="NLM_cas:title">Oncology Research</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">491-499</span>CODEN:
                <span class="NLM_cas:coden">ONREE8</span>;
        ISSN:<span class="NLM_cas:issn">0965-0407</span>.
    
            (<span class="NLM_cas:orgname">Cognizant Communication Corp.</span>)
        </div><div class="casAbstract">By comparative proteome anal. we searched for characteristic alterations of human stomach adenocarcinoma tissue and paired surrounding normal tissue.  Selected differential protein spots were identified with peptide mass fingerprinting based on matrix-assisted laser desorption/ionization time-of-flight mass spectrometry (MALDI-TOF) and database searching.  We identified protein alterations in 18 stomach cancer tissues compared with normal controls, comprising elevated levels of eight proteins, including 14-3-3 zeta, calcyclin, keratin, apolipoprotein A-1 precursor, proteasome activator complex subunit, nucleoside diphosphate kinase, nicotinamide N-methyltransferase, and pyridoxal kinase.  Five proteins (CA11, prohibitin, peroxiredoxin 4, serum amyloid P component, and NADH-ubiquinone oxidoreductase 23 kDa subunit) were decreased.  These data are valuable for identification of differentially expressed proteins involved in stomach cancer carcinogenesis, providing biomarker candidates to develop diagnostic and therapeutic tools.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLMiz7Zm0YeLVg90H21EOLACvtfcHk0lhwNJrZoiZEvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVyktr3K&md5=5e691decbf5b8563f91f710d0fd840b4</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.3727%2F0965040042380441&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3727%252F0965040042380441%26sid%3Dliteratum%253Aachs%26aulast%3DJang%26aufirst%3DJ.%2BS.%26aulast%3DCho%26aufirst%3DH.%2BY.%26aulast%3DLee%26aufirst%3DY.%2BJ.%26aulast%3DHa%26aufirst%3DW.%2BS.%26aulast%3DKim%26aufirst%3DH.%2BW.%26atitle%3DThe%2520differential%2520proteome%2520profile%2520of%2520stomach%2520cancer%253A%2520Identification%2520of%2520the%2520biomarker%2520candidates%26jtitle%3DOncol.%2520Res.%26date%3D2004%26volume%3D14%26spage%3D491%26epage%3D499%26doi%3D10.3727%2F0965040042380441" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Lim, B. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cho, B. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, Y. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, S. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, C. W.</span><span> </span><span class="NLM_article-title">Overexpression of nicotinamide N-methyltransferase in gastric cancer tissues and its potential post-translational modification</span> <span class="citation_source-journal">Exp. Mol. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">455</span><span class="NLM_x">–</span> <span class="NLM_lpage">465</span><span class="refDoi"> DOI: 10.1038/emm.2006.54</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1038%2Femm.2006.54" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=17079861" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtF2qs7fK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2006&pages=455-465&author=B.+H.+Limauthor=B.+I.+Choauthor=Y.+N.+Kimauthor=J.+W.+Kimauthor=S.+T.+Parkauthor=C.+W.+Lee&title=Overexpression+of+nicotinamide+N-methyltransferase+in+gastric+cancer+tissues+and+its+potential+post-translational+modification&doi=10.1038%2Femm.2006.54"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Overexpression of nicotinamide N-methyltransferase in gastric cancer tissues and its potential post-translational modification</span></div><div class="casAuthors">Lim, Bo-Hyun; Cho, Bok-Im; Kim, Yu Na; Kim, Jae Won; Park, Soon-Tae; Lee, Chang-Won</div><div class="citationInfo"><span class="NLM_cas:title">Experimental and Molecular Medicine</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">455-465</span>CODEN:
                <span class="NLM_cas:coden">EMMEF3</span>;
        ISSN:<span class="NLM_cas:issn">1226-3613</span>.
    
            (<span class="NLM_cas:orgname">Korean Society of Medical Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Gastric cancer is one of the most common cancers worldwide.  The purpose of this study was to find out potential markers for gastric cancer.  Tumor and normal tissues from 152 gastric cancer cases were analyzed by two-dimensional gel electrophoresis (2-DE).  The images of silver stained gels were analyzed and statistical anal. of spot intensities revealed that spot 4262 showed higher expression (5.7-fold increase) in cancer tissues than in normal tissues (P < 0.001).  It was identified by peptide mass fingerprinting as nicotinamide N-methyltransferase (NNMT).  A monoclonal antibody with a detection limit down to 10 ng was produced against NNMT in mouse.  Using the prepd. monoclonal antibody, western blot anal. of NNMT was performed for gastric tissues from 15 gastric cancer patients and two gastric ulcer patients.  The results corroborated those of 2-DE expts.  A single spot was detected in gastric ulcer tissues while four to five spots were detected in gastric cancer tissues.  In cancer tissues, two addnl. spots of acidic and basic form were mainly detected on 2-DE gels.  This suggests that NNMT receives a post-translational modification in cancer-specific manner.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTyS3vucBqULVg90H21EOLACvtfcHk0lguAkfzbihNAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtF2qs7fK&md5=e88bc3183a7968b6b23f89c715b387ba</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Femm.2006.54&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Femm.2006.54%26sid%3Dliteratum%253Aachs%26aulast%3DLim%26aufirst%3DB.%2BH.%26aulast%3DCho%26aufirst%3DB.%2BI.%26aulast%3DKim%26aufirst%3DY.%2BN.%26aulast%3DKim%26aufirst%3DJ.%2BW.%26aulast%3DPark%26aufirst%3DS.%2BT.%26aulast%3DLee%26aufirst%3DC.%2BW.%26atitle%3DOverexpression%2520of%2520nicotinamide%2520N-methyltransferase%2520in%2520gastric%2520cancer%2520tissues%2520and%2520its%2520potential%2520post-translational%2520modification%26jtitle%3DExp.%2520Mol.%2520Med.%26date%3D2006%26volume%3D38%26spage%3D455%26epage%3D465%26doi%3D10.1038%2Femm.2006.54" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Wu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siadaty, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berens, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hampton, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Theodorescu, D.</span><span> </span><span class="NLM_article-title">Overlapping gene expression profiles of cell migration and tumor invasion in human bladder cancer identify metallothionein 1E and nicotinamide N-methyltransferase as novel regulators of cell migration</span> <span class="citation_source-journal">Oncogene</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">6679</span><span class="NLM_x">–</span> <span class="NLM_lpage">6689</span><span class="refDoi"> DOI: 10.1038/onc.2008.264</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1038%2Fonc.2008.264" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2008&pages=6679-6689&author=Y.+Wuauthor=M.+S.+Siadatyauthor=M.+E.+Berensauthor=G.+M.+Hamptonauthor=D.+Theodorescu&title=Overlapping+gene+expression+profiles+of+cell+migration+and+tumor+invasion+in+human+bladder+cancer+identify+metallothionein+1E+and+nicotinamide+N-methyltransferase+as+novel+regulators+of+cell+migration&doi=10.1038%2Fonc.2008.264"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fonc.2008.264&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2008.264%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DY.%26aulast%3DSiadaty%26aufirst%3DM.%2BS.%26aulast%3DBerens%26aufirst%3DM.%2BE.%26aulast%3DHampton%26aufirst%3DG.%2BM.%26aulast%3DTheodorescu%26aufirst%3DD.%26atitle%3DOverlapping%2520gene%2520expression%2520profiles%2520of%2520cell%2520migration%2520and%2520tumor%2520invasion%2520in%2520human%2520bladder%2520cancer%2520identify%2520metallothionein%25201E%2520and%2520nicotinamide%2520N-methyltransferase%2520as%2520novel%2520regulators%2520of%2520cell%2520migration%26jtitle%3DOncogene%26date%3D2008%26volume%3D27%26spage%3D6679%26epage%3D6689%26doi%3D10.1038%2Fonc.2008.264" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Yu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bi, H. C.</span><span> </span><span class="NLM_article-title">Effects of nicotinamide N-methyltransferase on PANC-1 cells proliferation, metastatic potential and survival under metabolic stress</span> <span class="citation_source-journal">Cell. Physiol. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">710</span><span class="NLM_x">–</span> <span class="NLM_lpage">721</span><span class="refDoi"> DOI: 10.1159/000369731</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1159%2F000369731" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=25592232" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvV2msbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2015&pages=710-721&author=T.+Yuauthor=Y.+T.+Wangauthor=P.+Chenauthor=Y.+H.+Liauthor=Y.+X.+Chenauthor=H.+Zengauthor=A.+M.+Yuauthor=M.+Huangauthor=H.+C.+Bi&title=Effects+of+nicotinamide+N-methyltransferase+on+PANC-1+cells+proliferation%2C+metastatic+potential+and+survival+under+metabolic+stress&doi=10.1159%2F000369731"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Effects of Nicotinamide N-Methyltransferase on PANC-1 Cells Proliferation, Metastatic Potential and Survival Under Metabolic Stress</span></div><div class="casAuthors">Yu, Tao; Wang, Yong-Tao; Chen, Pan; Li, Yu-Hua; Chen, Yi-Xin; Zeng, Hang; Yu, Ai-Ming; Huang, Min; Bi, Hui-Chang</div><div class="citationInfo"><span class="NLM_cas:title">Cellular Physiology and Biochemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">710-721</span>CODEN:
                <span class="NLM_cas:coden">CEPBEW</span>;
        ISSN:<span class="NLM_cas:issn">1015-8987</span>.
    
            (<span class="NLM_cas:orgname">S. Karger AG</span>)
        </div><div class="casAbstract">Background: Aberrant expression of Nicotinamide N-methyltransferase (NNMT) has been reported in pancreatic cancer.  However, the role of NNMT in pancreatic cancer development remains elusive.  Therefore, the present study was to investigate the impact of NNMT on pancreatic cancer cell proliferation, metastatic potential and survival under metabolic stress.  Methods: Pancreatic cancer cell line PANC-1 was transfected with NNMT expression plasmid or small interfering RNA of NNMT to overexpress or knockdown intracellular NNMT expression, resp.  Rate of cell proliferation was monitored.  Transwell migration and matrigel invasion assays were conducted to assess cell migration and invasion capacity.  Resistance to glucose deprivation, sensitivity to glycolytic inhibition, mitochondrial inhibtion and resistance to rapamycin were examd. to evaluate cell survival under metabolic stress.  Results: NNMT silencing markedly reduced cell proliferation, whereas NNMT overexpression promoted cell growth moderately.  Knocking down NNMT also significantly suppressed the migration and invasion capacities of PANC-1 cells.  Conversely, NNMT upregulation enhanced cell migration and invasion capacities.  In addn., NNMT knockdown cells were much less resistant to glucose deprivation and rapamycin as well as glycolytic inhibitor 2-deoxyglucose whereas NNMT-expressing cells showed opposite effects although the effects were not so striking.  Conclusions: These data sugguest that NNMT plays an important role in PANC-1 cell proliferation, metastatic potential and survival under metabolic stress.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqcIoH5JmFE37Vg90H21EOLACvtfcHk0lguAkfzbihNAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvV2msbs%253D&md5=96916089c2d252b0409157e8a7efe61b</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1159%2F000369731&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1159%252F000369731%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DY.%2BT.%26aulast%3DChen%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DY.%2BH.%26aulast%3DChen%26aufirst%3DY.%2BX.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DYu%26aufirst%3DA.%2BM.%26aulast%3DHuang%26aufirst%3DM.%26aulast%3DBi%26aufirst%3DH.%2BC.%26atitle%3DEffects%2520of%2520nicotinamide%2520N-methyltransferase%2520on%2520PANC-1%2520cells%2520proliferation%252C%2520metastatic%2520potential%2520and%2520survival%2520under%2520metabolic%2520stress%26jtitle%3DCell.%2520Physiol.%2520Biochem.%26date%3D2015%26volume%3D35%26spage%3D710%26epage%3D721%26doi%3D10.1159%2F000369731" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Sartini, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santarelli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rossi, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goteri, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rubini, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciavarella, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lo Muzio, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emanuelli, M.</span><span> </span><span class="NLM_article-title">Nicotinamide N-methyltransferase upregulation inversely correlates with lymph node metastasis in oral squamous cell carcinoma</span> <span class="citation_source-journal">Mol. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">415</span><span class="NLM_x">–</span> <span class="NLM_lpage">421</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.2119%2F2007-00035.Sartini" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=17622326" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVynsrjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2007&pages=415-421&author=D.+Sartiniauthor=A.+Santarelliauthor=V.+Rossiauthor=G.+Goteriauthor=C.+Rubiniauthor=D.+Ciavarellaauthor=L.+Lo+Muzioauthor=M.+Emanuelli&title=Nicotinamide+N-methyltransferase+upregulation+inversely+correlates+with+lymph+node+metastasis+in+oral+squamous+cell+carcinoma"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Nicotinamide N-methyltransferase upregulation inversely correlates with lymph node metastasis in oral squamous cell carcinoma</span></div><div class="casAuthors">Sartini, Davide; Santarelli, Andrea; Rossi, Valentina; Goteri, Gaia; Rubini, Corrado; Ciavarella, Domenico; Lo Muzio, Lorenzo; Emanuelli, Monica</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Medicine (Manhasset, NY, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">7-8</span>),
    <span class="NLM_cas:pages">415-421</span>CODEN:
                <span class="NLM_cas:coden">MOMEF3</span>;
        ISSN:<span class="NLM_cas:issn">1076-1551</span>.
    
            (<span class="NLM_cas:orgname">Feinstein Institute for Medical Research</span>)
        </div><div class="casAbstract">We investigated expression levels of Nicotinamide N-Methyltransferase (NNMT), an enzyme involved in the biotransformation of many drugs and xenobiotic compds., in oral squamous cell carcinoma (OSCC).  Measurements were performed by semi-quant. RT-PCR and quant. real-time PCR in tumor and matched adjacent healthy tissue.  Interestingly, NNMT was up-regulated in most of the favorable OSCCs, while no marked NNMT expression alterations between tumor and normal mucosa were detected in most of the unfavorable OSCCs.  Western blot and immunohistochem. analyses also were performed and the relationship between tumor characteristics and NNMT levels in OSCC were studied to evaluate the effectiveness of NNMT as a prognostic marker in the squamous cell carcinoma of the oral cavity.  In summary, the present study suggests that NNMT may have potential as a biomarker and a therapeutic target for OSCC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrg_Vx2wEahmbVg90H21EOLACvtfcHk0li3IkPD2wZi-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVynsrjL&md5=9b78d16ea787bc3d544faffed81822aa</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.2119%2F2007-00035.Sartini&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2119%252F2007-00035.Sartini%26sid%3Dliteratum%253Aachs%26aulast%3DSartini%26aufirst%3DD.%26aulast%3DSantarelli%26aufirst%3DA.%26aulast%3DRossi%26aufirst%3DV.%26aulast%3DGoteri%26aufirst%3DG.%26aulast%3DRubini%26aufirst%3DC.%26aulast%3DCiavarella%26aufirst%3DD.%26aulast%3DLo%2BMuzio%26aufirst%3DL.%26aulast%3DEmanuelli%26aufirst%3DM.%26atitle%3DNicotinamide%2520N-methyltransferase%2520upregulation%2520inversely%2520correlates%2520with%2520lymph%2520node%2520metastasis%2520in%2520oral%2520squamous%2520cell%2520carcinoma%26jtitle%3DMol.%2520Med.%26date%3D2007%26volume%3D13%26spage%3D415%26epage%3D421" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Pozzi, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sartini, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morganti, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giuliante, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Di Ruscio, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santarelli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rocchetti, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rubini, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tomasetti, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giannatempo, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orlando, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Provinciali, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lo Muzio, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emanuelli, M.</span><span> </span><span class="NLM_article-title">RNA-mediated gene silencing of nicotinamide N-methyltransferase is associated with decreased tumorigenicity in human oral carcinoma cells</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">e71272</span><span class="refDoi"> DOI: 10.1371/journal.pone.0071272</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1371%2Fjournal.pone.0071272" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=e71272&author=V.+Pozziauthor=D.+Sartiniauthor=S.+Morgantiauthor=R.+Giulianteauthor=G.+Di+Ruscioauthor=A.+Santarelliauthor=R.+Rocchettiauthor=C.+Rubiniauthor=M.+Tomasettiauthor=G.+Giannatempoauthor=F.+Orlandoauthor=M.+Provincialiauthor=L.+Lo+Muzioauthor=M.+Emanuelli&title=RNA-mediated+gene+silencing+of+nicotinamide+N-methyltransferase+is+associated+with+decreased+tumorigenicity+in+human+oral+carcinoma+cells&doi=10.1371%2Fjournal.pone.0071272"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0071272&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0071272%26sid%3Dliteratum%253Aachs%26aulast%3DPozzi%26aufirst%3DV.%26aulast%3DSartini%26aufirst%3DD.%26aulast%3DMorganti%26aufirst%3DS.%26aulast%3DGiuliante%26aufirst%3DR.%26aulast%3DDi%2BRuscio%26aufirst%3DG.%26aulast%3DSantarelli%26aufirst%3DA.%26aulast%3DRocchetti%26aufirst%3DR.%26aulast%3DRubini%26aufirst%3DC.%26aulast%3DTomasetti%26aufirst%3DM.%26aulast%3DGiannatempo%26aufirst%3DG.%26aulast%3DOrlando%26aufirst%3DF.%26aulast%3DProvinciali%26aufirst%3DM.%26aulast%3DLo%2BMuzio%26aufirst%3DL.%26aulast%3DEmanuelli%26aufirst%3DM.%26atitle%3DRNA-mediated%2520gene%2520silencing%2520of%2520nicotinamide%2520N-methyltransferase%2520is%2520associated%2520with%2520decreased%2520tumorigenicity%2520in%2520human%2520oral%2520carcinoma%2520cells%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26spage%3De71272%26doi%3D10.1371%2Fjournal.pone.0071272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Roessler, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rollinger, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palme, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hagmann, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berndt, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engel, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneidinger, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pfeffer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andres, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karl, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bodenmuller, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruschoff, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henkel, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rohr, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rossol, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosch, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Langen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zolg, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tacke, M.</span><span> </span><span class="NLM_article-title">Identification of nicotinamide N-methyltransferase as a novel serum tumor marker for colorectal cancer</span> <span class="citation_source-journal">Clin. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">6550</span><span class="NLM_x">–</span> <span class="NLM_lpage">6557</span><span class="refDoi"> DOI: 10.1158/1078-0432.CCR-05-0983</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1158%2F1078-0432.CCR-05-0983" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=16166432" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVShsb%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2005&pages=6550-6557&author=M.+Roesslerauthor=W.+Rollingerauthor=S.+Palmeauthor=M.+L.+Hagmannauthor=P.+Berndtauthor=A.+M.+Engelauthor=B.+Schneidingerauthor=M.+Pfefferauthor=H.+Andresauthor=J.+Karlauthor=H.+Bodenmullerauthor=J.+Ruschoffauthor=T.+Henkelauthor=G.+Rohrauthor=S.+Rossolauthor=W.+Roschauthor=H.+Langenauthor=W.+Zolgauthor=M.+Tacke&title=Identification+of+nicotinamide+N-methyltransferase+as+a+novel+serum+tumor+marker+for+colorectal+cancer&doi=10.1158%2F1078-0432.CCR-05-0983"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Nicotinamide N-Methyltransferase as a Novel Serum Tumor Marker for Colorectal Cancer</span></div><div class="casAuthors">Roessler, Markus; Rollinger, Wolfgang; Palme, Stefan; Hagmann, Marie-Luise; Berndt, Peter; Engel, Alfred M.; Schneidinger, Bernd; Pfeffer, Michael; Andres, Herbert; Karl, Johann; Bodenmueller, Heinz; Rueschoff, Josef; Henkel, Thomas; Rohr, Gerhard; Rossol, Siegbert; Roesch, Wolfgang; Langen, Hanno; Zolg, Werner; Tacke, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">6550-6557</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: The goal of this study was to identify and validate novel serum markers of human colorectal cancer as potential candidates for noninvasive detection of early colorectal neoplasm.  Exptl. Design: Employing two-dimensional gel electrophoresis and mass spectrometry, we analyzed 16 matched colorectal cancer and adjacent normal tissue samples.  Proteins found to be elevated in cancer tissue were further validated by generating antibodies which were used for immunoblotting of tissue samples and for the development of highly sensitive immunoassays for assessment of serum samples.  Results: In total, 735 different proteins were identified in colon tissue.  Strong elevation in colorectal cancer for five proteins was confirmed by immunoblot anal.: transforming growth factor-β induced protein ig-h3 (βIG-H3), nicotinamide N-methyltransferase (NNMT), nucleoside diphosphate kinase A (nm23-H1), purine nucleoside phosphorylase (PNPH), and mannose-6-phosphate receptor binding protein 1 (M6P1).  Elevated levels of NNMT, which is not predicted to be secreted but is known as a cytoplasmic protein, were found in serum from patients with colorectal cancer.  Employing a receiver-operating characteristic curve based on the measurement of 109 patients with colorectal cancer and 317 healthy controls, we obtained an area under the curve of 0.84 for NNMT, which was superior to the established tumor marker carcinoembryogenic antigen with an area under the curve of 0.78.  Conclusions: It is proposed that NNMT serum levels may have significance in the early detection and in the management of patients with colorectal cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwGM5NZ6JMdLVg90H21EOLACvtfcHk0ljARvjTrfa17w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVShsb%252FP&md5=e2e24797b2c90ad48b11202539e1edaa</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-05-0983&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-05-0983%26sid%3Dliteratum%253Aachs%26aulast%3DRoessler%26aufirst%3DM.%26aulast%3DRollinger%26aufirst%3DW.%26aulast%3DPalme%26aufirst%3DS.%26aulast%3DHagmann%26aufirst%3DM.%2BL.%26aulast%3DBerndt%26aufirst%3DP.%26aulast%3DEngel%26aufirst%3DA.%2BM.%26aulast%3DSchneidinger%26aufirst%3DB.%26aulast%3DPfeffer%26aufirst%3DM.%26aulast%3DAndres%26aufirst%3DH.%26aulast%3DKarl%26aufirst%3DJ.%26aulast%3DBodenmuller%26aufirst%3DH.%26aulast%3DRuschoff%26aufirst%3DJ.%26aulast%3DHenkel%26aufirst%3DT.%26aulast%3DRohr%26aufirst%3DG.%26aulast%3DRossol%26aufirst%3DS.%26aulast%3DRosch%26aufirst%3DW.%26aulast%3DLangen%26aufirst%3DH.%26aulast%3DZolg%26aufirst%3DW.%26aulast%3DTacke%26aufirst%3DM.%26atitle%3DIdentification%2520of%2520nicotinamide%2520N-methyltransferase%2520as%2520a%2520novel%2520serum%2520tumor%2520marker%2520for%2520colorectal%2520cancer%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2005%26volume%3D11%26spage%3D6550%26epage%3D6557%26doi%3D10.1158%2F1078-0432.CCR-05-0983" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Horning, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suciu, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghadiri, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ulanovskaya, O. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matthews, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lum, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Backus, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cravatt, B. F.</span><span> </span><span class="NLM_article-title">Chemical proteomic profiling of human methyltransferases</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">138</span><span class="NLM_x">, </span> <span class="NLM_fpage">13335</span><span class="NLM_x">–</span> <span class="NLM_lpage">13343</span><span class="refDoi"> DOI: 10.1021/jacs.6b07830</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.6b07830" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFymu7bP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2016&pages=13335-13343&author=B.+D.+Horningauthor=R.+M.+Suciuauthor=D.+A.+Ghadiriauthor=O.+A.+Ulanovskayaauthor=M.+L.+Matthewsauthor=K.+M.+Lumauthor=K.+M.+Backusauthor=S.+J.+Brownauthor=H.+Rosenauthor=B.+F.+Cravatt&title=Chemical+proteomic+profiling+of+human+methyltransferases&doi=10.1021%2Fjacs.6b07830"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical Proteomic Profiling of Human Methyltransferases</span></div><div class="casAuthors">Horning, Benjamin D.; Suciu, Radu M.; Ghadiri, Darian A.; Ulanovskaya, Olesya A.; Matthews, Megan L.; Lum, Kenneth M.; Backus, Keriann M.; Brown, Steven J.; Rosen, Hugh; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">13335-13343</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Methylation is a fundamental mechanism used in Nature to modify the structure and function of biomols., including proteins, DNA, RNA, and metabolites.  Me groups are predominantly installed into biomols. by a large and diverse class of S-adenosyl methionine (SAM)-dependent methyltransferases (MTs), of which there are ∼200 known or putative members in the human proteome.  Deregulated MT activity contributes to numerous diseases, including cancer, and several MT inhibitors are in clin. development.  Nonetheless, a large fraction of the human MT family remains poorly characterized, underscoring the need for new technologies to characterize MTs and their inhibitors in native biol. systems.  Here, the authors describe a suite of S-adenosyl homocysteine (SAH) photoreactive probes and their application in chem. proteomic expts. to profile and enrich a large no. of MTs (>50) from human cancer cell lysates with remarkable specificity over other classes of proteins.  The authors further demonstrate that the SAH probes can enrich MT-assocd. proteins and be used to screen for and assess the selectivity of MT inhibitors, leading to the discovery of a covalent inhibitor of nicotinamide N-methyltransferase (NNMT), an enzyme implicated in cancer and metabolic disorders.  The chem. proteomics probes and methods for their utilization reported herein should prove of value for the functional characterization of MTs, MT complexes, and MT inhibitors in mammalian biol. and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplJsKzvFF-e7Vg90H21EOLACvtfcHk0ljARvjTrfa17w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFymu7bP&md5=0e07845e59e33381f3149beae7500e18</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fjacs.6b07830&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.6b07830%26sid%3Dliteratum%253Aachs%26aulast%3DHorning%26aufirst%3DB.%2BD.%26aulast%3DSuciu%26aufirst%3DR.%2BM.%26aulast%3DGhadiri%26aufirst%3DD.%2BA.%26aulast%3DUlanovskaya%26aufirst%3DO.%2BA.%26aulast%3DMatthews%26aufirst%3DM.%2BL.%26aulast%3DLum%26aufirst%3DK.%2BM.%26aulast%3DBackus%26aufirst%3DK.%2BM.%26aulast%3DBrown%26aufirst%3DS.%2BJ.%26aulast%3DRosen%26aufirst%3DH.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DChemical%2520proteomic%2520profiling%2520of%2520human%2520methyltransferases%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2016%26volume%3D138%26spage%3D13335%26epage%3D13343%26doi%3D10.1021%2Fjacs.6b07830" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Neelakantan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, H. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vance, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hommel, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McHardy, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watowich, S. J.</span><span> </span><span class="NLM_article-title">Structure-activity relationship for small molecule inhibitors of nicotinamide N-methyltransferase</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">5015</span><span class="NLM_x">–</span> <span class="NLM_lpage">5028</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00389</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00389" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BC2sXos1Sgtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=5015-5028&author=H.+Neelakantanauthor=H.+Y.+Wangauthor=V.+Vanceauthor=J.+D.+Hommelauthor=S.+F.+McHardyauthor=S.+J.+Watowich&title=Structure-activity+relationship+for+small+molecule+inhibitors+of+nicotinamide+N-methyltransferase&doi=10.1021%2Facs.jmedchem.7b00389"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Activity Relationship for Small Molecule Inhibitors of Nicotinamide N-Methyltransferase</span></div><div class="casAuthors">Neelakantan, Harshini; Wang, Hua-Yu; Vance, Virginia; Hommel, Jonathan D.; McHardy, Stanton F.; Watowich, Stanley J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5015-5028</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) is a fundamental cytosolic biotransforming enzyme that catalyzes the N-methylation of endogenous and exogenous xenobiotics.  We have identified small mol. inhibitors of NNMT with >1000-fold range of activity and developed comprehensive structure-activity relationships (SARs) for NNMT inhibitors.  Screening of N-methylated quinolinium, isoquinolinium, pyrididium, and benzimidazolium/benzothiazolium analogs resulted in the identification of quinoliniums as a promising scaffold with very low micromolar (IC50 ∼ 1 μM) NNMT inhibition.  Computer-based docking of inhibitors to the NNMT substrate (nicotinamide)-binding site produced a robust correlation between ligand-enzyme interaction docking scores and exptl. calcd. IC50 values.  Predicted binding orientation of the quinolinium analogs revealed selective binding to the NNMT substrate-binding site residues and essential chem. features driving protein-ligand intermol. interactions and NNMT inhibition.  The development of this new series of small mol. NNMT inhibitors direct the future design of lead drug-like inhibitors to treat several metabolic and chronic disease conditions characterized by abnormal NNMT activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMxYy3poogIbVg90H21EOLACvtfcHk0ljnZ3FJuUyx8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXos1Sgtrc%253D&md5=2cd7159b9e15569221849168e49e39cd</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00389&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00389%26sid%3Dliteratum%253Aachs%26aulast%3DNeelakantan%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DH.%2BY.%26aulast%3DVance%26aufirst%3DV.%26aulast%3DHommel%26aufirst%3DJ.%2BD.%26aulast%3DMcHardy%26aufirst%3DS.%2BF.%26aulast%3DWatowich%26aufirst%3DS.%2BJ.%26atitle%3DStructure-activity%2520relationship%2520for%2520small%2520molecule%2520inhibitors%2520of%2520nicotinamide%2520N-methyltransferase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D5015%26epage%3D5028%26doi%3D10.1021%2Facs.jmedchem.7b00389" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">van Haren, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taig, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuppens, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sastre Torano, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moret, E. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parsons, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sartini, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emanuelli, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, N. I.</span><span> </span><span class="NLM_article-title">Inhibitors of nicotinamide N-methyltransferase designed to mimic the methylation reaction transition state</span> <span class="citation_source-journal">Org. Biomol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">6656</span><span class="NLM_x">–</span> <span class="NLM_lpage">6667</span><span class="refDoi"> DOI: 10.1039/C7OB01357D</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1039%2FC7OB01357D" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=28758655" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1aktLfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2017&pages=6656-6667&author=M.+J.+van+Harenauthor=R.+Taigauthor=J.+Kuppensauthor=J.+Sastre+Toranoauthor=E.+E.+Moretauthor=R.+B.+Parsonsauthor=D.+Sartiniauthor=M.+Emanuelliauthor=N.+I.+Martin&title=Inhibitors+of+nicotinamide+N-methyltransferase+designed+to+mimic+the+methylation+reaction+transition+state&doi=10.1039%2FC7OB01357D"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of nicotinamide N-methyltransferase designed to mimic the methylation reaction transition state</span></div><div class="casAuthors">van Haren, Matthijs J.; Taig, Rebecca; Kuppens, Jilles; Sastre Torano, Javier; Moret, Ed E.; Parsons, Richard B.; Sartini, Davide; Emanuelli, Monica; Martin, Nathaniel I.</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">6656-6667</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) is an enzyme that catalyzes the methylation of nicotinamide to form N'-methylnicotinamide.  Both NNMT and its methylated product have recently been linked to a variety of diseases, suggesting a role for the enzyme as a therapeutic target beyond its previously ascribed metabolic function in detoxification.  We here describe the systematic development of NNMT inhibitors derived from the structures of the substrates involved in the methylation reaction.  By covalently linking fragments of the NNMT substrates a diverse library of bisubstrate-like compds. was prepd.  The ability of these compds. to inhibit NNMT was evaluated providing valuable insights into the structural tolerances of the enzyme active site.  These studies led to the identification of new NNMT inhibitors that mimic the transition state of the methylation reaction and inhibit the enzyme with activity on par with established methyltransferase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxwrd9fP8Bf7Vg90H21EOLACvtfcHk0ljnZ3FJuUyx8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1aktLfI&md5=74280c50135611d361ab81b8fc55fa89</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1039%2FC7OB01357D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7OB01357D%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BHaren%26aufirst%3DM.%2BJ.%26aulast%3DTaig%26aufirst%3DR.%26aulast%3DKuppens%26aufirst%3DJ.%26aulast%3DSastre%2BTorano%26aufirst%3DJ.%26aulast%3DMoret%26aufirst%3DE.%2BE.%26aulast%3DParsons%26aufirst%3DR.%2BB.%26aulast%3DSartini%26aufirst%3DD.%26aulast%3DEmanuelli%26aufirst%3DM.%26aulast%3DMartin%26aufirst%3DN.%2BI.%26atitle%3DInhibitors%2520of%2520nicotinamide%2520N-methyltransferase%2520designed%2520to%2520mimic%2520the%2520methylation%2520reaction%2520transition%2520state%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2017%26volume%3D15%26spage%3D6656%26epage%3D6667%26doi%3D10.1039%2FC7OB01357D" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Lavogina, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Enkvist, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uri, A.</span><span> </span><span class="NLM_article-title">Bisubstrate inhibitors of protein kinases: from principle to practical applications</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">23</span><span class="NLM_x">–</span> <span class="NLM_lpage">34</span><span class="refDoi"> DOI: 10.1002/cmdc.200900252</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1002%2Fcmdc.200900252" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=19774589" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhs1SqsbzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=23-34&author=D.+Lavoginaauthor=E.+Enkvistauthor=A.+Uri&title=Bisubstrate+inhibitors+of+protein+kinases%3A+from+principle+to+practical+applications&doi=10.1002%2Fcmdc.200900252"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Bisubstrate Inhibitors of Protein Kinases: from Principle to Practical Applications</span></div><div class="casAuthors">Lavogina, Darja; Enkvist, Erki; Uri, Asko</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">23-34</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Bisubstrate inhibitors consist of two conjugated fragments, each targeted to a different binding site of a bisubstrate enzyme.  The design of bisubstrate inhibitors presupposes the formation of the ternary complex in the course of the catalyzed reaction.  The principle advantage of bisubstrate inhibitors is their ability to generate more interactions with the target enzyme that could result in improved affinity and selectivity of the conjugates, when compared with single-site inhibitors.  Among phosphotransferases, the approach was first successfully used for adenylate kinase in 1973.  Since then, several types of bisubstrate inhibitors have been developed for protein kinases, including conjugates of peptides with nucleotides, adenosine derivs. and potent ATP-competitive inhibitors.  Earlier bisubstrate inhibitors had pharmacokinetic qualities that were unsuitable for cellular expts. and hence were mostly used for in vitro studies.  The recently constructed conjugates of adenosine derivs. and D-arginine-rich peptides (ARCs) possess high kinase affinity, high biol. and chem. stability and good cell plasma membrane penetrative properties that enable their application in the regulation of cellular protein phosphorylation balances in cell and tissue expts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSsmmKerEiPrVg90H21EOLACvtfcHk0ljnZ3FJuUyx8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhs1SqsbzP&md5=7efb7b5aa44347f5fa8fc40f98065337</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.200900252&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.200900252%26sid%3Dliteratum%253Aachs%26aulast%3DLavogina%26aufirst%3DD.%26aulast%3DEnkvist%26aufirst%3DE.%26aulast%3DUri%26aufirst%3DA.%26atitle%3DBisubstrate%2520inhibitors%2520of%2520protein%2520kinases%253A%2520from%2520principle%2520to%2520practical%2520applications%26jtitle%3DChemMedChem%26date%3D2010%26volume%3D5%26spage%3D23%26epage%3D34%26doi%3D10.1002%2Fcmdc.200900252" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Yu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Magalhaes, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cook, P. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blanchard, J. S.</span><span> </span><span class="NLM_article-title">Bisubstrate inhibition: Theory and application to N-acetyltransferases</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">14788</span><span class="NLM_x">–</span> <span class="NLM_lpage">14794</span><span class="refDoi"> DOI: 10.1021/bi061621t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi061621t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1SjtbfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2006&pages=14788-14794&author=M.+Yuauthor=M.+L.+Magalhaesauthor=P.+F.+Cookauthor=J.+S.+Blanchard&title=Bisubstrate+inhibition%3A+Theory+and+application+to+N-acetyltransferases&doi=10.1021%2Fbi061621t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Bisubstrate Inhibition: Theory and Application to N-Acetyltransferases</span></div><div class="casAuthors">Yu, Michael; Magalhaes, Maria L. B.; Cook, Paul F.; Blanchard, John S.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">14788-14794</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bisubstrate inhibitors represent a potentially powerful group of compds. that have found significant therapeutic utility.  Although these compds. have been synthesized and tested against a no. of enzymes that catalyze sequential bireactant reactions, the detailed theory for predicting the expected patterns of inhibition against the two substrates for various bireactant kinetic mechanisms has, heretofore, not been presented.  We have derived the rate equations for all likely sequential bireactant mechanisms and provide two examples in which bisubstrate inhibitors allow the kinetic mechanism to be detd.  Bisubstrate inhibitor kinetics is a powerful diagnostic for the detn. of kinetic mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrk_sv2txdK0LVg90H21EOLACvtfcHk0lhjdypNc3xioA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1SjtbfN&md5=6ddb7dcb30e7d97a3577a6088db49433</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fbi061621t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi061621t%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DM.%26aulast%3DMagalhaes%26aufirst%3DM.%2BL.%26aulast%3DCook%26aufirst%3DP.%2BF.%26aulast%3DBlanchard%26aufirst%3DJ.%2BS.%26atitle%3DBisubstrate%2520inhibition%253A%2520Theory%2520and%2520application%2520to%2520N-acetyltransferases%26jtitle%3DBiochemistry%26date%3D2006%26volume%3D45%26spage%3D14788%26epage%3D14794%26doi%3D10.1021%2Fbi061621t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Peng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sartini, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pozzi, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilk, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emanuelli, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yee, V. C.</span><span> </span><span class="NLM_article-title">Structural basis of substrate recognition in human nicotinamide N-methyltransferase</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">7800</span><span class="NLM_x">–</span> <span class="NLM_lpage">7808</span><span class="refDoi"> DOI: 10.1021/bi2007614</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi2007614" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtValsbnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2011&pages=7800-7808&author=Y.+Pengauthor=D.+Sartiniauthor=V.+Pozziauthor=D.+Wilkauthor=M.+Emanuelliauthor=V.+C.+Yee&title=Structural+basis+of+substrate+recognition+in+human+nicotinamide+N-methyltransferase&doi=10.1021%2Fbi2007614"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of substrate recognition in human nicotinamide N-methyltransferase</span></div><div class="casAuthors">Peng, Yi; Sartini, Davide; Pozzi, Valentina; Wilk, Dennis; Emanuelli, Monica; Yee, Vivien C.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">7800-7808</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) catalyzes the N-methylation of nicotinamide, pyridines, and other analogs using S-adenosyl-L-methionine as donor.  NNMT plays a significant role in the regulation of metabolic pathways and is expressed at markedly high levels in several kinds of cancers, presenting it as a potential mol. target for cancer therapy.  Here, the authors detd. the crystal structure of human NNMT as a ternary complex bound to both the demethylated donor, S-adenosyl-L-homocysteine, and the acceptor substrate, nicotinamide, to 2.7 Å resoln.  These studies reveal the structural basis for nicotinamide binding and highlight several residues in the active site which may play roles in nicotinamide recognition and NNMT catalysis.  The functional importance of these residues was probed by mutagenesis.  Of three residues near the nicotinamide's amide group, substitution of S201 and S213 had no effect on enzyme activity while replacement of D197 dramatically decreased activity.  Substitutions of Y20, whose side chain hydroxyl interacts with both the nicotinamide arom. ring and AdoHcy carboxylate, also compromised activity.  Enzyme kinetics anal. revealed kcat/Km decreases of 2-3 orders of magnitude for the D197A and Y20A mutants, confirming the functional importance of these active site residues.  The mutants exhibited substantially increased Km for both NCA and AdoMet and modestly decreased kcat.  MD simulations revealed long-range conformational effects which provide an explanation for the large increase in Km(AdoMet) for the D197A mutant, which interacts directly only with nicotinamide in the ternary complex crystal structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqh1yxD6WIJFLVg90H21EOLACvtfcHk0ljS60kvh_ouVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtValsbnF&md5=926af50d2464e21094ffb17205793d19</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fbi2007614&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi2007614%26sid%3Dliteratum%253Aachs%26aulast%3DPeng%26aufirst%3DY.%26aulast%3DSartini%26aufirst%3DD.%26aulast%3DPozzi%26aufirst%3DV.%26aulast%3DWilk%26aufirst%3DD.%26aulast%3DEmanuelli%26aufirst%3DM.%26aulast%3DYee%26aufirst%3DV.%2BC.%26atitle%3DStructural%2520basis%2520of%2520substrate%2520recognition%2520in%2520human%2520nicotinamide%2520N-methyltransferase%26jtitle%3DBiochemistry%26date%3D2011%26volume%3D50%26spage%3D7800%26epage%3D7808%26doi%3D10.1021%2Fbi2007614" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Liu, K.; Sun, X.; Ma, J.; Zhou, Z.; Dong, Q.; Peng, S.; Wu, J.; Tan, S.; Blobel, G.; Fan, J.</span><span> </span><span class="NLM_article-title">Prediction of amino acid side chain conformation using a deep neural network</span>.  <span class="citation_source-journal">ArXiv e-prints</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span>arXiv:1707.08381.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=K.+Liu&author=X.+Sun&author=J.+Ma&author=Z.+Zhou&author=Q.+Dong&author=S.+Peng&author=J.+Wu&author=S.+Tan&author=G.+Blobel&author=J.+Fan&title=Prediction+of+amino+acid+side+chain+conformation+using+a+deep+neural+network"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DK.%26atitle%3DPrediction%2520of%2520amino%2520acid%2520side%2520chain%2520conformation%2520using%2520a%2520deep%2520neural%2520network%26jtitle%3DArXiv%2520e-prints%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Yao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anglin, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prasad, B. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, Y.</span><span> </span><span class="NLM_article-title">Selective inhibitors of histone methyltransferase DOT1L: design, synthesis, and crystallographic studies</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">133</span><span class="NLM_x">, </span> <span class="NLM_fpage">16746</span><span class="NLM_x">–</span> <span class="NLM_lpage">16749</span><span class="refDoi"> DOI: 10.1021/ja206312b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja206312b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2011&pages=16746-16749&author=Y.+Yaoauthor=P.+Chenauthor=J.+Diaoauthor=G.+Chengauthor=L.+Dengauthor=J.+L.+Anglinauthor=B.+V.+Prasadauthor=Y.+Song&title=Selective+inhibitors+of+histone+methyltransferase+DOT1L%3A+design%2C+synthesis%2C+and+crystallographic+studies&doi=10.1021%2Fja206312b"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fja206312b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja206312b%26sid%3Dliteratum%253Aachs%26aulast%3DYao%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DP.%26aulast%3DDiao%26aufirst%3DJ.%26aulast%3DCheng%26aufirst%3DG.%26aulast%3DDeng%26aufirst%3DL.%26aulast%3DAnglin%26aufirst%3DJ.%2BL.%26aulast%3DPrasad%26aufirst%3DB.%2BV.%26aulast%3DSong%26aufirst%3DY.%26atitle%3DSelective%2520inhibitors%2520of%2520histone%2520methyltransferase%2520DOT1L%253A%2520design%252C%2520synthesis%252C%2520and%2520crystallographic%2520studies%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2011%26volume%3D133%26spage%3D16746%26epage%3D16749%26doi%3D10.1021%2Fja206312b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Cox, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gibson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mayo Martin, M. B.</span><span> </span><span class="NLM_article-title">Aspartyl phosphonates and phosphoramidates: the first synthetic inhibitors of bacterial aspartate-semialdehyde dehydrogenase</span> <span class="citation_source-journal">ChemBioChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">874</span><span class="NLM_x">–</span> <span class="NLM_lpage">886</span><span class="refDoi"> DOI: 10.1002/1439-7633(20020902)3:9<874::AID-CBIC874>3.0.CO;2-V</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1002%2F1439-7633%2820020902%293%3A9%3C874%3A%3AAID-CBIC874%3E3.0.CO%3B2-V" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2002&pages=874-886&author=R.+J.+Coxauthor=J.+S.+Gibsonauthor=M.+B.+Mayo+Martin&title=Aspartyl+phosphonates+and+phosphoramidates%3A+the+first+synthetic+inhibitors+of+bacterial+aspartate-semialdehyde+dehydrogenase&doi=10.1002%2F1439-7633%2820020902%293%3A9%3C874%3A%3AAID-CBIC874%3E3.0.CO%3B2-V"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1002%2F1439-7633%2820020902%293%3A9%3C874%3A%3AAID-CBIC874%3E3.0.CO%3B2-V&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F1439-7633%252820020902%25293%253A9%253C874%253A%253AAID-CBIC874%253E3.0.CO%253B2-V%26sid%3Dliteratum%253Aachs%26aulast%3DCox%26aufirst%3DR.%2BJ.%26aulast%3DGibson%26aufirst%3DJ.%2BS.%26aulast%3DMayo%2BMartin%26aufirst%3DM.%2BB.%26atitle%3DAspartyl%2520phosphonates%2520and%2520phosphoramidates%253A%2520the%2520first%2520synthetic%2520inhibitors%2520of%2520bacterial%2520aspartate-semialdehyde%2520dehydrogenase%26jtitle%3DChemBioChem%26date%3D2002%26volume%3D3%26spage%3D874%26epage%3D886%26doi%3D10.1002%2F1439-7633%2820020902%293%3A9%3C874%3A%3AAID-CBIC874%3E3.0.CO%3B2-V" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Usuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sugimura, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Komatsu, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koseki, Y.</span><span> </span><span class="NLM_article-title">Biomimetic Chichibabin pyridine synthesis of the COPD biomarkers and elastin cross-linkers isodesmosine and desmosine</span> <span class="citation_source-journal">Org. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">1672</span><span class="NLM_x">–</span> <span class="NLM_lpage">1675</span><span class="refDoi"> DOI: 10.1021/ol500333t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol500333t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2014&pages=1672-1675&author=T.+Usukiauthor=T.+Sugimuraauthor=A.+Komatsuauthor=Y.+Koseki&title=Biomimetic+Chichibabin+pyridine+synthesis+of+the+COPD+biomarkers+and+elastin+cross-linkers+isodesmosine+and+desmosine&doi=10.1021%2Fol500333t"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fol500333t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol500333t%26sid%3Dliteratum%253Aachs%26aulast%3DUsuki%26aufirst%3DT.%26aulast%3DSugimura%26aufirst%3DT.%26aulast%3DKomatsu%26aufirst%3DA.%26aulast%3DKoseki%26aufirst%3DY.%26atitle%3DBiomimetic%2520Chichibabin%2520pyridine%2520synthesis%2520of%2520the%2520COPD%2520biomarkers%2520and%2520elastin%2520cross-linkers%2520isodesmosine%2520and%2520desmosine%26jtitle%3DOrg.%2520Lett.%26date%3D2014%26volume%3D16%26spage%3D1672%26epage%3D1675%26doi%3D10.1021%2Fol500333t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Collazo, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Couture, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bulfer, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trievel, R. C.</span><span> </span><span class="NLM_article-title">A coupled fluorescent assay for histone methyltransferases</span> <span class="citation_source-journal">Anal. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">342</span><span class="NLM_x">, </span> <span class="NLM_fpage">86</span><span class="NLM_x">–</span> <span class="NLM_lpage">92</span><span class="refDoi"> DOI: 10.1016/j.ab.2005.04.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1016%2Fj.ab.2005.04.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=15958184" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BD2MXlt1GktL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=342&publication_year=2005&pages=86-92&author=E.+Collazoauthor=J.+F.+Coutureauthor=S.+Bulferauthor=R.+C.+Trievel&title=A+coupled+fluorescent+assay+for+histone+methyltransferases&doi=10.1016%2Fj.ab.2005.04.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">A coupled fluorescent assay for histone methyltransferases</span></div><div class="casAuthors">Collazo, Evys; Couture, Jean-Francois; Bulfer, Stacie; Trievel, Raymond C.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">342</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">86-92</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Histone methyltransferases (HMTs) catalyze the S-adenosylmethionine (AdoMet)-dependent methylation of lysines and arginines in the nucleosomal core histones H3 and H4 and the linker histone H1b.  Methylation of these residues regulates either transcriptional activation or silencing, depending on the residue modified and its degree of methylation.  Despite an intense interest in elucidating the functions of HMTs in transcriptional regulation, these enzymes have remained challenging to quant. assay.  To characterize the substrate specificity of HMTs, we have developed a coupled-fluorescence-based assay for AdoMet-dependent methyltransferases.  This assay utilizes S-adenosylhomocysteine hydrolase (SAHH) to hydrolyze the methyltransfer product S-adenosylhomocysteine (AdoHcy) to homocysteine (Hcy) and adenosine (Ado).  The Hcy concn. is then detd. through conjugation of its free sulfhydryl moiety to a thiol-sensitive fluorophore.  Using this assay, we have detd. the kinetic parameters for the methylation of a synthetic histone H3 peptide (corresponding to residues 1-15 of the native protein) by Schizosaccharomyces pombe CLR4, an H3 Lys-9-specific methyltransferase.  The fluorescent SAHH-coupled assay allows rapid and facile detn. of HMT kinetics and can be adapted to measure the enzymic activity of a wide variety of AdoMet-dependent methyltransferases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfHslBzi9obbVg90H21EOLACvtfcHk0ljRHDOYKqaeBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXlt1GktL4%253D&md5=2b949dcde263d15559202131cc74da1e</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2005.04.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2005.04.007%26sid%3Dliteratum%253Aachs%26aulast%3DCollazo%26aufirst%3DE.%26aulast%3DCouture%26aufirst%3DJ.%2BF.%26aulast%3DBulfer%26aufirst%3DS.%26aulast%3DTrievel%26aufirst%3DR.%2BC.%26atitle%3DA%2520coupled%2520fluorescent%2520assay%2520for%2520histone%2520methyltransferases%26jtitle%3DAnal.%2520Biochem.%26date%3D2005%26volume%3D342%26spage%3D86%26epage%3D92%26doi%3D10.1016%2Fj.ab.2005.04.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siarheyeva, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajian, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span> </span><span class="NLM_article-title">Fluorescence-based methods for screening writers and readers of histone methyl marks</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">71</span><span class="NLM_x">–</span> <span class="NLM_lpage">84</span><span class="refDoi"> DOI: 10.1177/1087057111422256</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1177%2F1087057111422256" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=21972038" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BC38Xit1Olt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2012&pages=71-84&author=A.+Allali-Hassaniauthor=G.+A.+Wasneyauthor=A.+Siarheyevaauthor=T.+Hajianauthor=C.+H.+Arrowsmithauthor=M.+Vedadi&title=Fluorescence-based+methods+for+screening+writers+and+readers+of+histone+methyl+marks&doi=10.1177%2F1087057111422256"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorescence-based methods for screening writers and readers of histone methyl marks</span></div><div class="casAuthors">Allali-Hassani, Abdellah; Wasney, Gregory A.; Siarheyeva, Alena; Hajian, Taraneh; Arrowsmith, Cheryl H.; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">71-84</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">The histone methyltransferase (HMT) family of proteins consists of enzymes that methylate lysine or arginine residues on histone tails as well as other proteins.  Such modifications affect chromatin structure and play a significant regulatory role in gene expression.  Many HMTs have been implicated in tumorigenesis and progression of multiple malignancies and play essential roles in embryonic development and stem cell renewal.  Overexpression of some HMTs has been obsd. and is correlated pos. with various types of cancer.  Here the authors report development of a continuous fluorescence-based methyltransferase assay in a 384-well format and its application in detg. kinetic parameters for EHMT1, G9a, PRMT3, SETD7, and SUV39H2 as well as for screening against libraries of small mols. to identify enzyme inhibitors.  They also report the development of a peptide displacement assay using fluorescence polarization in a 384-well format to assay and screen protein peptide interactions such as those of WDR5 and EED, components of MLL and EZH2 methyltransferase complexes.  Using these high-throughput screening methods, the authors have identified potent inhibitors and ligands for some of these proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0s8Ex9rZOXrVg90H21EOLACvtfcHk0ljRHDOYKqaeBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xit1Olt7g%253D&md5=105a20c2473ab603ade42a38764def09</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1177%2F1087057111422256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057111422256%26sid%3Dliteratum%253Aachs%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DHajian%26aufirst%3DT.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DFluorescence-based%2520methods%2520for%2520screening%2520writers%2520and%2520readers%2520of%2520histone%2520methyl%2520marks%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2012%26volume%3D17%26spage%3D71%26epage%3D84%26doi%3D10.1177%2F1087057111422256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref100"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref100'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit100"><span><span class="NLM_contrib-group">Richon, V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnston, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sneeringer, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majer, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elliston, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jerva, L. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span> </span><span class="NLM_article-title">Chemogenetic analysis of human protein methyltransferases</span> <span class="citation_source-journal">Chem. Biol. Drug Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">78</span><span class="NLM_x">, </span> <span class="NLM_fpage">199</span><span class="NLM_x">–</span> <span class="NLM_lpage">210</span><span class="refDoi"> DOI: 10.1111/j.1747-0285.2011.01135.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1111%2Fj.1747-0285.2011.01135.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=21564555" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpsF2qsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2011&pages=199-210&author=V.+M.+Richonauthor=D.+Johnstonauthor=C.+J.+Sneeringerauthor=L.+Jinauthor=C.+R.+Majerauthor=K.+Ellistonauthor=L.+F.+Jervaauthor=M.+P.+Scottauthor=R.+A.+Copeland&title=Chemogenetic+analysis+of+human+protein+methyltransferases&doi=10.1111%2Fj.1747-0285.2011.01135.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit100R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Chemogenetic analysis of human protein methyltransferases</span></div><div class="casAuthors">Richon, Victoria M.; Johnston, Danielle; Sneeringer, Christopher J.; Jin, Lei; Majer, Christina R.; Elliston, Keith; Jerva, L. Fred; Scott, Margaret Porter; Copeland, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">199-210</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A survey of the human genome was performed to understand the constituency of protein methyltransferases (both protein arginine and lysine methyltransferases) and the relatedness of their catalytic domains.  We identified 51 protein lysine methyltransferase proteins based on similarity to the canonical Drosophila Su(var)3-9, enhancer of zeste (E(z)), and trithorax (trx) domain.  Disruptor of telomeric silencing-1-like, a known protein lysine methyltransferase, did not fit within the protein lysine methyltransferase family, but did group with the protein arginine methyltransferases, along with 44 other proteins, including the METTL and NOP2/Sun domain family proteins.  We show that a representative METTL, METTL11A, demonstrates catalytic activity as a histone methyltransferase.  We also solved the co-crystal structures of disruptor of telomeric silencing-1-like with S-adenosylmethionine and S-adenosylhomocysteine bound in its active site.  The conformation of both ligands is virtually identical to that found in known protein arginine methyltransferases, METTL and NOP2/Sun domain family proteins and is distinct from that seen in the Drosophila Su(var)3-9, enhancer of zeste (E(z)), and trithorax (trx) domain protein lysine methyltransferases.  We have developed biochem. assays for 11 members of the protein methyltransferase target class and have profiled the affinity of three ligands for these enzymes: the common methyl-donating substrate S-adenosylmethionine; the common reaction product S-adenosylhomocysteine; and the natural product sinefungin.  The affinity of each of these ligands is mapped onto the family trees of the protein lysine methyltransferases and protein arginine methyltransferases to reveal patterns of ligand recognition by these enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJylTD5PqDfLVg90H21EOLACvtfcHk0liGFFkfmeCuog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpsF2qsrk%253D&md5=ab02523741396a36afc7d38f2a90b4f0</span></div><a href="/servlet/linkout?suffix=cit100&amp;dbid=16384&amp;doi=10.1111%2Fj.1747-0285.2011.01135.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1747-0285.2011.01135.x%26sid%3Dliteratum%253Aachs%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DJohnston%26aufirst%3DD.%26aulast%3DSneeringer%26aufirst%3DC.%2BJ.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DElliston%26aufirst%3DK.%26aulast%3DJerva%26aufirst%3DL.%2BF.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26atitle%3DChemogenetic%2520analysis%2520of%2520human%2520protein%2520methyltransferases%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2011%26volume%3D78%26spage%3D199%26epage%3D210%26doi%3D10.1111%2Fj.1747-0285.2011.01135.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref101"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref101'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit101"><span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dai, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atadja, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, K.</span><span> </span><span class="NLM_article-title">Structure of human SMYD2 protein reveals the basis of p53 tumor suppressor methylation</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">286</span><span class="NLM_x">, </span> <span class="NLM_fpage">38725</span><span class="NLM_x">–</span> <span class="NLM_lpage">38737</span><span class="refDoi"> DOI: 10.1074/jbc.M111.262410</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1074%2Fjbc.M111.262410" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=21880715" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlykurvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2011&pages=38725-38737&author=L.+Wangauthor=L.+Liauthor=H.+Zhangauthor=X.+Luoauthor=J.+Daiauthor=S.+Zhouauthor=J.+Guauthor=J.+Zhuauthor=P.+Atadjaauthor=C.+Luauthor=E.+Liauthor=K.+Zhao&title=Structure+of+human+SMYD2+protein+reveals+the+basis+of+p53+tumor+suppressor+methylation&doi=10.1074%2Fjbc.M111.262410"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit101R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of Human SMYD2 Protein Reveals the Basis of p53 Tumor Suppressor Methylation</span></div><div class="casAuthors">Wang, Li; Li, Ling; Zhang, Hailong; Luo, Xiao; Dai, Jingquan; Zhou, Shaolian; Gu, Justin; Zhu, Jidong; Atadja, Peter; Lu, Chris; Li, En; Zhao, Kehao</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">286</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">38725-38737, S38725/1-S38725/5</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">SMYD2 belongs to a subfamily of histone lysine methyltransferase and was recently identified to methylate tumor suppressor p53 and Rb.  Here we report that SMYD2 prefers to methylate p53 Lys-370 over histone substrates in vitro.  Consistently, the level of endogenous p53 Lys-370 monomethylation is significantly elevated when SMYD2 is overexpressed in vivo.  We have solved the high resoln. crystal structures of the full-length SMYD2 protein in binary complex with its cofactor S-adenosylmethionine and in ternary complex with cofactor product S-adenosylhomocysteine and p53 substrate peptide (residues 368-375), resp. p53 peptide binds to a deep pocket of the interface between catalytic SET(1-282) and C-terminal domain (CTD) with an unprecedented U-shaped conformation.  Subtle conformational change exists around the p53 binding site between the binary and ternary structures, in particular the tetratricopeptide repeat motif of the CTD.  In addn., a unique EDEE motif between the loop of anti-parallel β7 and β8 sheets of the SET core not only interacts with p53 substrate but also forms a hydrogen bond network with residues from CTD.  These observations suggest that the tetratricopeptide repeat and EDEE motif may play an important role in detg. p53 substrate binding specificity.  This is further verified by the findings that deletion of the CTD domain drastically reduces the methylation activity of SMYD2 to p53 protein.  Meanwhile, mutation of EDEE residues impairs both the binding and the enzymic activity of SMYD2 to p53 Lys-370.  These data together reveal the mol. basis of SMYD2 in specifically recognizing and regulating functions of p53 tumor suppressor through Lys-370 monomethylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpX6G4Yhf_H47Vg90H21EOLACvtfcHk0liGFFkfmeCuog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlykurvI&md5=671857a22846de5d607e83a03dd52d37</span></div><a href="/servlet/linkout?suffix=cit101&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M111.262410&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M111.262410%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DDai%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DS.%26aulast%3DGu%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DAtadja%26aufirst%3DP.%26aulast%3DLu%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DE.%26aulast%3DZhao%26aufirst%3DK.%26atitle%3DStructure%2520of%2520human%2520SMYD2%2520protein%2520reveals%2520the%2520basis%2520of%2520p53%2520tumor%2520suppressor%2520methylation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2011%26volume%3D286%26spage%3D38725%26epage%3D38737%26doi%3D10.1074%2Fjbc.M111.262410" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Galijasevic, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abdulhamid, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abu-Soud, H. M.</span><span> </span><span class="NLM_article-title">Melatonin is a potent inhibitor for myeloperoxidase</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">2668</span><span class="NLM_x">–</span> <span class="NLM_lpage">2677</span><span class="refDoi"> DOI: 10.1021/bi702016q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi702016q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1Omsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2008&pages=2668-2677&author=S.+Galijasevicauthor=I.+Abdulhamidauthor=H.+M.+Abu-Soud&title=Melatonin+is+a+potent+inhibitor+for+myeloperoxidase&doi=10.1021%2Fbi702016q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Melatonin Is a Potent Inhibitor for Myeloperoxidase</span></div><div class="casAuthors">Galijasevic, Semira; Abdulhamid, Ibrahim; Abu-Soud, Husam M.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2668-2677</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Myeloperoxidase (MPO) catalyzes the formation of potent oxidants that have been implicated in the pathogenesis of various diseases including atherosclerosis, asthma, arthritis, and cancer.  Melatonin plays an important part in the regulation of various body functions including circadian sleep rhythms, blood pressure, oncogenesis, retinal function, seasonal reprodn., and immunity.  Here, we demonstrate that melatonin serves as a potent inhibitor of MPO under physiol.-like conditions.  In the presence of chloride (Cl-), melatonin inactivated MPO at two points in the classic peroxidase cycle through binding to MPO to form an inactive complex, melatonin-MPO-Cl, and accelerating MPO compd. II formation, an inactive form of MPO.  Inactivation of MPO was mirrored by the direct conversion of MPO-Fe(III) to MPO compd. II without any sign of compd. I accumulation.  This behavior indicates that melatonin binding modulates the formation of MPO intermediates and their decay rates.  The Cl- presence enhanced the affinity of MPO toward melatonin, which switches the enzyme activity from peroxidn. to catalase-like activity.  In the absence of Cl-, melatonin served as a 1e- substrate for MPO compd. I, but at higher concn. it limited the reaction by its dissocn. from the corresponding complex.  Importantly, melatonin-dependent inhibition of MPO occurred with a wide range of concns. that span various physiol. and supplemental ranges.  Thus, the interplay between MPO and melatonin may have a broader implication in the function of several biol. systems.  This dual regulation by melatonin is unique and represents a new means through which melatonin can control MPO and its downstream inflammatory pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRFdB_4Le6XrVg90H21EOLACvtfcHk0licxHfZqbPNyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1Omsbw%253D&md5=3b707a0557ed92becb329a004ea806a8</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fbi702016q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi702016q%26sid%3Dliteratum%253Aachs%26aulast%3DGalijasevic%26aufirst%3DS.%26aulast%3DAbdulhamid%26aufirst%3DI.%26aulast%3DAbu-Soud%26aufirst%3DH.%2BM.%26atitle%3DMelatonin%2520is%2520a%2520potent%2520inhibitor%2520for%2520myeloperoxidase%26jtitle%3DBiochemistry%26date%3D2008%26volume%3D47%26spage%3D2668%26epage%3D2677%26doi%3D10.1021%2Fbi702016q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Neelakantan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vance, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, H. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McHardy, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watowich, S. J.</span><span> </span><span class="NLM_article-title">Noncoupled Fluorescent Assay for Direct Real-Time Monitoring of Nicotinamide N-Methyltransferase Activity</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">824</span><span class="NLM_x">–</span> <span class="NLM_lpage">832</span><span class="refDoi"> DOI: 10.1021/acs.biochem.6b01215</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biochem.6b01215" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFajs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=824-832&author=H.+Neelakantanauthor=V.+Vanceauthor=H.+L.+Wangauthor=S.+F.+McHardyauthor=S.+J.+Watowich&title=Noncoupled+Fluorescent+Assay+for+Direct+Real-Time+Monitoring+of+Nicotinamide+N-Methyltransferase+Activity&doi=10.1021%2Facs.biochem.6b01215"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Noncoupled Fluorescent Assay for Direct Real-Time Monitoring of Nicotinamide N-Methyltransferase Activity</span></div><div class="casAuthors">Neelakantan, Harshini; Vance, Virginia; Wang, Hua-Yu Leo; McHardy, Stanton F.; Watowich, Stanley J.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">824-832</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) is an important biotransforming enzyme that catalyzes the transfer of a labile Me group from the ubiquitous cofactor, S-adenosyl-L-methionine (SAM), to endogenous and exogenous small mols. such as quinoline to form methylated end-products.  NNMT has been implicated in a no. of chronic disease conditions, including metabolic disorders, cardiovascular disease, cancer, osteoarthritis, kidney disease, and Parkinson's disease.  Here, the authors developed a novel noncoupled fluorescence-based methyltransferase assay that enabled direct ultrasensitive real-time detection of the NNMT reaction product, 1-methylquinolinium.  This is the 1st assay reported to date to utilize fluorescence spectroscopy to directly monitor NNMT product formation and activity in real-time.  This assay provided accurate kinetic data that enabled detailed comparative anal. of the NNMT reaction mechanism and kinetic parameters.  A reaction model based on a random bi-reactant mechanism produced global curve fits that were most consistent with steady-state initial velocity data collected across an array of substrate concns.  Based on the reaction mechanism, each substrate could independently bind to the NNMT apoenzyme; however, both substrates bound to the complementary binary complexes with ∼20-fold higher affinity relative to independent binding.  This reaction mechanism implies either substrate-induced conformational changes or bi-reactant intermol. interactions may stabilize substrate binding to the binary complex and formation of the ternary complex.  Importantly, this assay could rapidly generate concn. response curves for known NNMT inhibitors, suggesting its applicability for high-throughput screening (HTS) of chem. libraries to identify novel NNMT inhibitors.  Furthermore, this novel assay potentially offers a robust detection technol. for use in SAM substrate competition assays for the discovery and development of SAM-dependent methyltransferase inhibitors such as S-adenosyl-L-homocysteine and 1-methylnicotinamide.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6X5AP5dGb77Vg90H21EOLACvtfcHk0licxHfZqbPNyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFajs74%253D&md5=bf243bd7362322b094c31b1e247b7240</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Facs.biochem.6b01215&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biochem.6b01215%26sid%3Dliteratum%253Aachs%26aulast%3DNeelakantan%26aufirst%3DH.%26aulast%3DVance%26aufirst%3DV.%26aulast%3DWang%26aufirst%3DH.%2BL.%26aulast%3DMcHardy%26aufirst%3DS.%2BF.%26aulast%3DWatowich%26aufirst%3DS.%2BJ.%26atitle%3DNoncoupled%2520Fluorescent%2520Assay%2520for%2520Direct%2520Real-Time%2520Monitoring%2520of%2520Nicotinamide%2520N-Methyltransferase%2520Activity%26jtitle%3DBiochemistry%26date%3D2017%26volume%3D56%26spage%3D824%26epage%3D832%26doi%3D10.1021%2Facs.biochem.6b01215" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Swaminathan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Birudukota, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thakur, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parveen, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kandan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Juluri, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaik, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anand, N. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burri, R. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kristam, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hallur, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rajagopal, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schreuder, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Langer, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rudolph, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruf, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhakshinamoorthy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gosu, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kannt, A.</span><span> </span><span class="NLM_article-title">Crystal structures of monkey and mouse nicotinamide N-methyltransferase (NNMT) bound with end product, 1-methyl nicotinamide</span> <span class="citation_source-journal">Biochem. Biophys. Res. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">491</span><span class="NLM_x">, </span> <span class="NLM_fpage">416</span><span class="NLM_x">–</span> <span class="NLM_lpage">422</span><span class="refDoi"> DOI: 10.1016/j.bbrc.2017.07.087</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1016%2Fj.bbrc.2017.07.087" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=491&publication_year=2017&pages=416-422&author=S.+Swaminathanauthor=S.+Birudukotaauthor=M.+K.+Thakurauthor=R.+Parveenauthor=S.+Kandanauthor=S.+Juluriauthor=S.+Shaikauthor=N.+N.+Anandauthor=R.+R.+Burriauthor=R.+Kristamauthor=M.+S.+Hallurauthor=S.+Rajagopalauthor=H.+Schreuderauthor=T.+Langerauthor=C.+Rudolphauthor=S.+Rufauthor=S.+Dhakshinamoorthyauthor=R.+Gosuauthor=A.+Kannt&title=Crystal+structures+of+monkey+and+mouse+nicotinamide+N-methyltransferase+%28NNMT%29+bound+with+end+product%2C+1-methyl+nicotinamide&doi=10.1016%2Fj.bbrc.2017.07.087"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2017.07.087&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2017.07.087%26sid%3Dliteratum%253Aachs%26aulast%3DSwaminathan%26aufirst%3DS.%26aulast%3DBirudukota%26aufirst%3DS.%26aulast%3DThakur%26aufirst%3DM.%2BK.%26aulast%3DParveen%26aufirst%3DR.%26aulast%3DKandan%26aufirst%3DS.%26aulast%3DJuluri%26aufirst%3DS.%26aulast%3DShaik%26aufirst%3DS.%26aulast%3DAnand%26aufirst%3DN.%2BN.%26aulast%3DBurri%26aufirst%3DR.%2BR.%26aulast%3DKristam%26aufirst%3DR.%26aulast%3DHallur%26aufirst%3DM.%2BS.%26aulast%3DRajagopal%26aufirst%3DS.%26aulast%3DSchreuder%26aufirst%3DH.%26aulast%3DLanger%26aufirst%3DT.%26aulast%3DRudolph%26aufirst%3DC.%26aulast%3DRuf%26aufirst%3DS.%26aulast%3DDhakshinamoorthy%26aufirst%3DS.%26aulast%3DGosu%26aufirst%3DR.%26aulast%3DKannt%26aufirst%3DA.%26atitle%3DCrystal%2520structures%2520of%2520monkey%2520and%2520mouse%2520nicotinamide%2520N-methyltransferase%2520%2528NNMT%2529%2520bound%2520with%2520end%2520product%252C%25201-methyl%2520nicotinamide%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2017%26volume%3D491%26spage%3D416%26epage%3D422%26doi%3D10.1016%2Fj.bbrc.2017.07.087" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Ramadan, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bremner-Hay, N. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlson, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Comstock, L. R.</span><span> </span><span class="NLM_article-title">Synthesis and evaluation of N6-substituted azide- and alkyne-bearing N-mustard analogs of S-adenosyl-l-methionine</span> <span class="citation_source-journal">Tetrahedron</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">70</span><span class="NLM_x">, </span> <span class="NLM_fpage">5291</span><span class="NLM_x">–</span> <span class="NLM_lpage">5297</span><span class="refDoi"> DOI: 10.1016/j.tet.2014.05.055</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1016%2Fj.tet.2014.05.055" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2014&pages=5291-5297&author=M.+Ramadanauthor=N.+K.+Bremner-Hayauthor=S.+A.+Carlsonauthor=L.+R.+Comstock&title=Synthesis+and+evaluation+of+N6-substituted+azide-+and+alkyne-bearing+N-mustard+analogs+of+S-adenosyl-l-methionine&doi=10.1016%2Fj.tet.2014.05.055"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.tet.2014.05.055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tet.2014.05.055%26sid%3Dliteratum%253Aachs%26aulast%3DRamadan%26aufirst%3DM.%26aulast%3DBremner-Hay%26aufirst%3DN.%2BK.%26aulast%3DCarlson%26aufirst%3DS.%2BA.%26aulast%3DComstock%26aufirst%3DL.%2BR.%26atitle%3DSynthesis%2520and%2520evaluation%2520of%2520N6-substituted%2520azide-%2520and%2520alkyne-bearing%2520N-mustard%2520analogs%2520of%2520S-adenosyl-l-methionine%26jtitle%3DTetrahedron%26date%3D2014%26volume%3D70%26spage%3D5291%26epage%3D5297%26doi%3D10.1016%2Fj.tet.2014.05.055" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oudhoff, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tatlock, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freeman, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuznetsova, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marcellus, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kennedy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lambert, J.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Couzens, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aman, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gingras, A.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Awar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fish, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerstenberger, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benn, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grimley, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Braam, M. J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rossi, F. M. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sudol, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunnage, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Owen, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zaph, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span> </span><span class="NLM_article-title">(R)-PFI-2 is a potent and selective inhibitor of SETD7 methyltransferase activity in cells</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">111</span><span class="NLM_x">, </span> <span class="NLM_fpage">12853</span><span class="NLM_x">–</span> <span class="NLM_lpage">12858</span><span class="refDoi"> DOI: 10.1073/pnas.1407358111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1073%2Fpnas.1407358111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=25136132" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVartr7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=12853-12858&author=D.+Barsyte-Lovejoyauthor=F.+Liauthor=M.+J.+Oudhoffauthor=J.+H.+Tatlockauthor=A.+Dongauthor=H.+Zengauthor=H.+Wuauthor=S.+A.+Freemanauthor=M.+Schapiraauthor=G.+A.+Senisterraauthor=E.+Kuznetsovaauthor=R.+Marcellusauthor=A.+Allali-Hassaniauthor=S.+Kennedyauthor=J.-P.+Lambertauthor=A.+L.+Couzensauthor=A.+Amanauthor=A.-C.+Gingrasauthor=R.+Al-Awarauthor=P.+V.+Fishauthor=B.+S.+Gerstenbergerauthor=L.+Robertsauthor=C.+L.+Bennauthor=R.+L.+Grimleyauthor=M.+J.+S.+Braamauthor=F.+M.+V.+Rossiauthor=M.+Sudolauthor=P.+J.+Brownauthor=M.+E.+Bunnageauthor=D.+R.+Owenauthor=C.+Zaphauthor=M.+Vedadiauthor=C.+H.+Arrowsmith&title=%28R%29-PFI-2+is+a+potent+and+selective+inhibitor+of+SETD7+methyltransferase+activity+in+cells&doi=10.1073%2Fpnas.1407358111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">(R)-PFI-2 is a potent and selective inhibitor of SETD7 methyltransferase activity in cells</span></div><div class="casAuthors">Barsyte-Lovejoy, Dalia; Li, Fengling; Oudhoff, Menno J.; Tatlock, John H.; Dong, Aiping; Zeng, Hong; Wu, Hong; Freeman, Spencer A.; Schapira, Matthieu; Senisterra, Guillermo A.; Kuznetsova, Ekaterina; Marcellus, Richard; Allali-Hassani, Abdellah; Kennedy, Steven; Lambert, Jean-Philippe; Couzens, Amber L.; Aman, Ahmed; Gingras, Anne-Claude; Al-Awar, Rima; Fish, Paul V.; Gerstenberger, Brian S.; Roberts, Lee; Benn, Caroline L.; Grimley, Rachel L.; Braam, Mitchell J. S.; Rossi, Fabio M. V.; Sudol, Marius; Brown, Peter J.; Bunnage, Mark E.; Owen, Dafydd R.; Zaph, Colby; Vedadi, Masoud; Arrowsmith, Cheryl H.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">12853-12858</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">SET domain contg. (lysine methyltransferase) 7 (SETD7) is implicated in multiple signaling and disease related pathways with a broad diversity of reported substrates.  Here, we report the discovery of (R)-PFI-2-a first-in-class, potent (Kiapp = 0.33 nM), selective, and cell-active inhibitor of the methyltransferase activity of human SETD7-and its 500-fold less active enantiomer, (S)-PFI-2.  (R)-PFI-2 exhibits an unusual cofactor-dependent and substrate-competitive inhibitory mechanism by occupying the substrate peptide binding groove of SETD7, including the catalytic lysine-binding channel, and by making direct contact with the donor Me group of the cofactor, S-adenosylmethionine.  Chemoproteomics expts. using a biotinylated deriv. of (R)-PFI-2 demonstrated dose-dependent competition for binding to endogenous SETD7 in MCF7 cells pretreated with (R)-PFI-2.  In murine embryonic fibroblasts, (R)-PFI-2 treatment phenocopied the effects of Setd7 deficiency on Hippo pathway signaling, via modulation of the transcriptional coactivator Yes-assocd. protein (YAP) and regulation of YAP target genes.  In confluent MCF7 cells, (R)-PFI-2 rapidly altered YAP localization, suggesting continuous and dynamic regulation of YAP by the methyltransferase activity of SETD7.  These data establish (R)-PFI-2 and related compds. as a valuable tool-kit for the study of the diverse roles of SETD7 in cells and further validate protein methyltransferases as a druggable target class.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqv-HTBIcjMnbVg90H21EOLACvtfcHk0lh26KyysOXwrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVartr7E&md5=0b8ad07ae23c4eba5cf11060d231b3f4</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1407358111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1407358111%26sid%3Dliteratum%253Aachs%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DOudhoff%26aufirst%3DM.%2BJ.%26aulast%3DTatlock%26aufirst%3DJ.%2BH.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DFreeman%26aufirst%3DS.%2BA.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DSenisterra%26aufirst%3DG.%2BA.%26aulast%3DKuznetsova%26aufirst%3DE.%26aulast%3DMarcellus%26aufirst%3DR.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DLambert%26aufirst%3DJ.-P.%26aulast%3DCouzens%26aufirst%3DA.%2BL.%26aulast%3DAman%26aufirst%3DA.%26aulast%3DGingras%26aufirst%3DA.-C.%26aulast%3DAl-Awar%26aufirst%3DR.%26aulast%3DFish%26aufirst%3DP.%2BV.%26aulast%3DGerstenberger%26aufirst%3DB.%2BS.%26aulast%3DRoberts%26aufirst%3DL.%26aulast%3DBenn%26aufirst%3DC.%2BL.%26aulast%3DGrimley%26aufirst%3DR.%2BL.%26aulast%3DBraam%26aufirst%3DM.%2BJ.%2BS.%26aulast%3DRossi%26aufirst%3DF.%2BM.%2BV.%26aulast%3DSudol%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DZaph%26aufirst%3DC.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26atitle%3D%2528R%2529-PFI-2%2520is%2520a%2520potent%2520and%2520selective%2520inhibitor%2520of%2520SETD7%2520methyltransferase%2520activity%2520in%2520cells%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2014%26volume%3D111%26spage%3D12853%26epage%3D12858%26doi%3D10.1073%2Fpnas.1407358111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Bromberg, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitchell, T. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Upadhyay, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jakob, C. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jhala, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Comess, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lasko, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tuzon, C. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eram, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nuber, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soni, N. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manaves, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Algire, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sweis, R. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Torrent, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schotta, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michaelides, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoemaker, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santhakumar, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rice, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiang, G. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pappano, W. N.</span><span> </span><span class="NLM_article-title">The SUV4–20 inhibitor A-196 verifies a role for epigenetics in genomic integrity</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">317</span><span class="NLM_x">–</span> <span class="NLM_lpage">324</span><span class="refDoi"> DOI: 10.1038/nchembio.2282</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1038%2Fnchembio.2282" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=28114273" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVKisbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=317-324&author=K.+D.+Brombergauthor=T.+R.+Mitchellauthor=A.+K.+Upadhyayauthor=C.+G.+Jakobauthor=M.+A.+Jhalaauthor=K.+M.+Comessauthor=L.+M.+Laskoauthor=C.+Liauthor=C.+T.+Tuzonauthor=Y.+Daiauthor=F.+Liauthor=M.+S.+Eramauthor=A.+Nuberauthor=N.+B.+Soniauthor=V.+Manavesauthor=M.+A.+Algireauthor=R.+F.+Sweisauthor=M.+Torrentauthor=G.+Schottaauthor=C.+Sunauthor=M.+R.+Michaelidesauthor=A.+R.+Shoemakerauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=V.+Santhakumarauthor=A.+Martinauthor=J.+C.+Riceauthor=G.+G.+Chiangauthor=M.+Vedadiauthor=D.+Barsyte-Lovejoyauthor=W.+N.+Pappano&title=The+SUV4%E2%80%9320+inhibitor+A-196+verifies+a+role+for+epigenetics+in+genomic+integrity&doi=10.1038%2Fnchembio.2282"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">The SUV4-20 inhibitor A-196 verifies a role for epigenetics in genomic integrity</span></div><div class="casAuthors">Bromberg, Kenneth D.; Mitchell, Taylor R. H.; Upadhyay, Anup K.; Jakob, Clarissa G.; Jhala, Manisha A.; Comess, Kenneth M.; Lasko, Loren M.; Li, Conglei; Tuzon, Creighton T.; Dai, Yujia; Li, Fengling; Eram, Mohammad S.; Nuber, Alexander; Soni, Niru B.; Manaves, Vlasios; Algire, Mikkel A.; Sweis, Ramzi F.; Torrent, Maricel; Schotta, Gunnar; Sun, Chaohong; Michaelides, Michael R.; Shoemaker, Alex R.; Arrowsmith, Cheryl H.; Brown, Peter J.; Santhakumar, Vijayaratnam; Martin, Alberto; Rice, Judd C.; Chiang, Gary G.; Vedadi, Masoud; Barsyte-Lovejoy, Dalia; Pappano, William N.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">317-324</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein lysine methyltransferases (PKMTs) regulate diverse physiol. processes including transcription and the maintenance of genomic integrity.  Genetic studies suggest that the PKMTs SUV420H1 and SUV420H2 facilitate proficient nonhomologous end-joining (NHEJ)-directed DNA repair by catalyzing the di- and trimethylation (me2 and me3, resp.) of lysine 20 on histone 4 (H4K20).  Here we report the identification of A-196, a potent and selective inhibitor of SUV420H1 and SUV420H2.  Biochem. and co-crystn. analyses demonstrate that A-196 is a substrate-competitive inhibitor of both SUV4-20 enzymes.  In cells, A-196 induced a global decrease in H4K20me2 and H4K20me3 and a concomitant increase in H4K20me1.  A-196 inhibited 53BP1 foci formation upon ionizing radiation and reduced NHEJ-mediated DNA-break repair but did not affect homol.-directed repair.  These results demonstrate the role of SUV4-20 enzymic activity in H4K20 methylation and DNA repair.  A-196 represents a first-in-class chem. probe of SUV4-20 to investigate the role of histone methyltransferases in genomic integrity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdOSfp0N-TMrVg90H21EOLACvtfcHk0lj5tdISAOJ6Qg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVKisbg%253D&md5=9e25d8894455f5db43a6bb6dc3492e71</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2282&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2282%26sid%3Dliteratum%253Aachs%26aulast%3DBromberg%26aufirst%3DK.%2BD.%26aulast%3DMitchell%26aufirst%3DT.%2BR.%26aulast%3DUpadhyay%26aufirst%3DA.%2BK.%26aulast%3DJakob%26aufirst%3DC.%2BG.%26aulast%3DJhala%26aufirst%3DM.%2BA.%26aulast%3DComess%26aufirst%3DK.%2BM.%26aulast%3DLasko%26aufirst%3DL.%2BM.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DTuzon%26aufirst%3DC.%2BT.%26aulast%3DDai%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DNuber%26aufirst%3DA.%26aulast%3DSoni%26aufirst%3DN.%2BB.%26aulast%3DManaves%26aufirst%3DV.%26aulast%3DAlgire%26aufirst%3DM.%2BA.%26aulast%3DSweis%26aufirst%3DR.%2BF.%26aulast%3DTorrent%26aufirst%3DM.%26aulast%3DSchotta%26aufirst%3DG.%26aulast%3DSun%26aufirst%3DC.%26aulast%3DMichaelides%26aufirst%3DM.%2BR.%26aulast%3DShoemaker%26aufirst%3DA.%2BR.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DSanthakumar%26aufirst%3DV.%26aulast%3DMartin%26aufirst%3DA.%26aulast%3DRice%26aufirst%3DJ.%2BC.%26aulast%3DChiang%26aufirst%3DG.%2BG.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DPappano%26aufirst%3DW.%2BN.%26atitle%3DThe%2520SUV4%25E2%2580%259320%2520inhibitor%2520A-196%2520verifies%2520a%2520role%2520for%2520epigenetics%2520in%2520genomic%2520integrity%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26spage%3D317%26epage%3D324%26doi%3D10.1038%2Fnchembio.2282" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Bailey, S.</span><span> </span><span class="NLM_article-title">The CCP4 suite - programs for protein crystallography</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">760</span><span class="NLM_x">–</span> <span class="NLM_lpage">763</span><span class="refDoi"> DOI: 10.1107/S0907444994003112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1107%2FS0907444994003112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=15299374" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=1994&pages=760-763&author=S.+Bailey&title=The+CCP4+suite+-+programs+for+protein+crystallography&doi=10.1107%2FS0907444994003112"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1107%2FS0907444994003112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444994003112%26sid%3Dliteratum%253Aachs%26aulast%3DBailey%26aufirst%3DS.%26atitle%3DThe%2520CCP4%2520suite%2520-%2520programs%2520for%2520protein%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1994%26volume%3D50%26spage%3D760%26epage%3D763%26doi%3D10.1107%2FS0907444994003112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">McCoy, A. J.</span><span> </span><span class="NLM_article-title">Solving structures of protein complexes by molecular replacement with Phaser</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">63</span><span class="NLM_x">, </span> <span class="NLM_fpage">32</span><span class="NLM_x">–</span> <span class="NLM_lpage">41</span><span class="refDoi"> DOI: 10.1107/S0907444906045975</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1107%2FS0907444906045975" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=17164524" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtlSqu77F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2007&pages=32-41&author=A.+J.+McCoy&title=Solving+structures+of+protein+complexes+by+molecular+replacement+with+Phaser&doi=10.1107%2FS0907444906045975"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Solving structures of protein complexes by molecular replacement with Phaser</span></div><div class="casAuthors">McCoy, Airlie J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">32-41</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Mol. replacement (MR) generally becomes more difficult as the no. of components in the asym. unit requiring sep. MR models (i.e. the dimensionality of the search) increases.  When the proportion of the total scattering contributed by each search component is small, the signal in the search for each component in isolation is weak or non-existent.  Maximum-likelihood MR functions enable complex asym. units to be built up from individual components with a 'tree search with pruning' approach.  This method, as implemented in the automated search procedure of the program Phaser, has been very successful in solving many previously intractable MR problems.  However, there are a no. of cases in which the automated search procedure of Phaser is suboptimal or encounters difficulties.  These include cases where there are a large no. of copies of the same component in the asym. unit or where the components of the asym. unit have greatly varying B factors.  Two case studies are presented to illustrate how Phaser can be used to best advantage in the std. 'automated MR' mode and two case studies are used to show how to modify the automated search strategy for problematic cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAvs9W8hQ3l7Vg90H21EOLACvtfcHk0lgV05kzoPMDkg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtlSqu77F&md5=323a5297bdf622ebaf398a87c39dc794</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1107%2FS0907444906045975&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444906045975%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26atitle%3DSolving%2520structures%2520of%2520protein%2520complexes%2520by%2520molecular%2520replacement%2520with%2520Phaser%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2007%26volume%3D63%26spage%3D32%26epage%3D41%26doi%3D10.1107%2FS0907444906045975" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Murshudov, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vagin, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodson, E. J.</span><span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">240</span><span class="NLM_x">–</span> <span class="NLM_lpage">255</span><span class="refDoi"> DOI: 10.1107/S0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method&doi=10.1107%2FS0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0lgV05kzoPMDkg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255%26doi%3D10.1107%2FS0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Afonine, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunkoczi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, I. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Echols, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hung, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapral, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moriarty, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oeffner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terwilliger, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zwart, P. H.</span><span> </span><span class="NLM_article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">213</span><span class="NLM_x">–</span> <span class="NLM_lpage">221</span><span class="refDoi"> DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunkocziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.+W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+a+comprehensive+Python-based+system+for+macromolecular+structure+solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0lgV05kzoPMDkg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunkoczi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.%2BW.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520a%2520comprehensive%2520Python-based%2520system%2520for%2520macromolecular%2520structure%2520solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Afonine, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Echols, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moriarty, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mustyakimov, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terwilliger, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Urzhumtsev, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zwart, P. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, P. D.</span><span> </span><span class="NLM_article-title">Towards automated crystallographic structure refinement with phenix.refine</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">68</span><span class="NLM_x">, </span> <span class="NLM_fpage">352</span><span class="NLM_x">–</span> <span class="NLM_lpage">367</span><span class="refDoi"> DOI: 10.1107/S0907444912001308</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1107%2FS0907444912001308" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=22505256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BC38Xlt1Ggu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2012&pages=352-367&author=P.+V.+Afonineauthor=R.+W.+Grosse-Kunstleveauthor=N.+Echolsauthor=J.+J.+Headdauthor=N.+W.+Moriartyauthor=M.+Mustyakimovauthor=T.+C.+Terwilligerauthor=A.+Urzhumtsevauthor=P.+H.+Zwartauthor=P.+D.+Adams&title=Towards+automated+crystallographic+structure+refinement+with+phenix.refine&doi=10.1107%2FS0907444912001308"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Towards automated crystallographic structure refinement with phenix.refine</span></div><div class="casAuthors">Afonine, Pavel V.; Grosse-Kunstleve, Ralf W.; Echols, Nathaniel; Headd, Jeffrey J.; Moriarty, Nigel W.; Mustyakimov, Marat; Terwilliger, Thomas C.; Urzhumtsev, Alexandre; Zwart, Peter H.; Adams, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">352-367</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phenix.refine is a program within the PHENIX package that supports crystallog. structure refinement against exptl. data with a wide range of upper resoln. limits using a large repertoire of model parameterizations.  It has several automation features and is also highly flexible.  Several hundred parameters enable extensive customizations for complex use cases.  Multiple user-defined refinement strategies can be applied to specific parts of the model in a single refinement run.  An intuitive graphical user interface is available to guide novice users and to assist advanced users in managing refinement projects.  X-ray or neutron diffraction data can be used sep. or jointly in refinement. phenix.refine is tightly integrated into the PHENIX suite, where it serves as a crit. component in automated model building, final structure refinement, structure validation and deposition to the wwPDB.  This paper presents an overview of the major phenix.refine features, with extensive literature refs. for readers interested in more detailed discussions of the methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnhn9QwIEkTbVg90H21EOLACvtfcHk0lj1Y4BSbCczbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xlt1Ggu7c%253D&md5=1cc7da2cea396f1a75646ea709140cf1</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1107%2FS0907444912001308&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444912001308%26sid%3Dliteratum%253Aachs%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DMustyakimov%26aufirst%3DM.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DUrzhumtsev%26aufirst%3DA.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26atitle%3DTowards%2520automated%2520crystallographic%2520structure%2520refinement%2520with%2520phenix.refine%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2012%26volume%3D68%26spage%3D352%26epage%3D367%26doi%3D10.1107%2FS0907444912001308" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">2126</span><span class="NLM_x">–</span> <span class="NLM_lpage">2132</span><span class="refDoi"> DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lj1Y4BSbCczbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Chen, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arendall, W. B.,  3rd</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keedy, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Immormino, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapral, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, D. C.</span><span> </span><span class="NLM_article-title">MolProbity: all-atom structure validation for macromolecular crystallography</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">12</span><span class="NLM_x">–</span> <span class="NLM_lpage">21</span><span class="refDoi"> DOI: 10.1107/S0907444909042073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1107%2FS0907444909042073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=20057044" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=12-21&author=V.+B.+Chenauthor=W.+B.+Arendallauthor=J.+J.+Headdauthor=D.+A.+Keedyauthor=R.+M.+Immorminoauthor=G.+J.+Kapralauthor=L.+W.+Murrayauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=MolProbity%3A+all-atom+structure+validation+for+macromolecular+crystallography&doi=10.1107%2FS0907444909042073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">MolProbity: all-atom structure validation for macromolecular crystallography</span></div><div class="casAuthors">Chen, Vincent B.; Arendall, W. Bryan, III; Headd, Jeffrey J.; Keedy, Daniel A.; Immormino, Robert M.; Kapral, Gary J.; Murray, Laura W.; Richardson, Jane S.; Richardson, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12-21</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">MolProbity is a structure-validation web service that provides broad-spectrum solidly based evaluation of model quality at both the global and local levels for both proteins and nucleic acids.  It relies heavily on the power and sensitivity provided by optimized hydrogen placement and all-atom contact anal., complemented by updated versions of covalent-geometry and torsion-angle criteria.  Some of the local corrections can be performed automatically in MolProbity and all of the diagnostics are presented in chart and graphical forms that help guide manual rebuilding.  X-ray crystallog. provides a wealth of biol. important mol. data in the form of at. three-dimensional structures of proteins, nucleic acids and increasingly large complexes in multiple forms and states.  Advances in automation, in everything from crystn. to data collection to phasing to model building to refinement, have made solving a structure using crystallog. easier than ever.  However, despite these improvements, local errors that can affect biol. interpretation are widespread at low resoln. and even high-resoln. structures nearly all contain at least a few local errors such as Ramachandran outliers, flipped branched protein side chains and incorrect sugar puckers.  It is crit. both for the crystallographer and for the end user that there are easy and reliable methods to diagnose and correct these sorts of errors in structures.  MolProbity is the authors' contribution to helping solve this problem and this article reviews its general capabilities, reports on recent enhancements and usage, and presents evidence that the resulting improvements are now beneficially affecting the global database.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIjgsh59dO-7Vg90H21EOLACvtfcHk0ljp3F5i94H_Gg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D&md5=b5fc7574f43f01dd6e43c3663ca4f779</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1107%2FS0907444909042073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909042073%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DArendall%26aufirst%3DW.%2BB.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DKeedy%26aufirst%3DD.%2BA.%26aulast%3DImmormino%26aufirst%3DR.%2BM.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DMurray%26aufirst%3DL.%2BW.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMolProbity%253A%2520all-atom%2520structure%2520validation%2520for%2520macromolecular%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D12%26epage%3D21%26doi%3D10.1107%2FS0907444909042073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref100':['cit100'],'ref101':['cit101'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 21 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Kyohei Hayashi, Shota Uehara, Shiho Yamamoto, Douglas R. Cary, Junichi Nishikawa, Taichi Ueda, Hiroki Ozasa, Kousuke Mihara, Norito Yoshimura, Taeko Kawai, Takashi Ono, Saki Yamamoto, Masataka Fumoto, <span class="NLM_string-name hlFld-ContribAuthor">Hidenori Mikamiyama</span>. </span><span class="cited-content_cbyCitation_article-title">Macrocyclic Peptides as a Novel Class of NNMT Inhibitors: A SAR Study Aimed at Inhibitory Activity in the Cell. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2021,</strong> <em>12 </em>
                                    (7)
                                     , 1093-1101. <a href="https://doi.org/10.1021/acsmedchemlett.1c00134" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.1c00134</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.1c00134&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.1c00134%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DMacrocyclic%252BPeptides%252Bas%252Ba%252BNovel%252BClass%252Bof%252BNNMT%252BInhibitors%25253A%252BA%252BSAR%252BStudy%252BAimed%252Bat%252BInhibitory%252BActivity%252Bin%252Bthe%252BCell%26aulast%3DHayashi%26aufirst%3DKyohei%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D05032021%26date%3D24052021%26date%3D02062021%26volume%3D12%26issue%3D7%26spage%3D1093%26epage%3D1101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jian Lu, Aaron G. Bart, Qian Wu, Kevin R. Criscione, Michael J. McLeish, Emily E. Scott, <span class="NLM_string-name hlFld-ContribAuthor">Gary L. Grunewald</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-Based Drug Design of Bisubstrate Inhibitors of Phenylethanolamine N-Methyltransferase Possessing Low Nanomolar Affinity at Both Substrate Binding Domains1. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (22)
                                     , 13878-13898. <a href="https://doi.org/10.1021/acs.jmedchem.0c01475" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01475</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01475&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01475%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure-Based%252BDrug%252BDesign%252Bof%252BBisubstrate%252BInhibitors%252Bof%252BPhenylethanolamine%252BN-Methyltransferase%252BPossessing%252BLow%252BNanomolar%252BAffinity%252Bat%252BBoth%252BSubstrate%252BBinding%252BDomains1%26aulast%3DLu%26aufirst%3DJian%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D24082020%26date%3D04112020%26volume%3D63%26issue%3D22%26spage%3D13878%26epage%3D13898" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Dongxing Chen, Cheng Dong, Guangping Dong, Karthik Srinivasan, Jinrong Min, Nicholas Noinaj, <span class="NLM_string-name hlFld-ContribAuthor">Rong Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Probing the Plasticity in the Active Site of Protein N-terminal Methyltransferase 1 Using Bisubstrate Analogues. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (15)
                                     , 8419-8431. <a href="https://doi.org/10.1021/acs.jmedchem.0c00770" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00770</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00770&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00770%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DProbing%252Bthe%252BPlasticity%252Bin%252Bthe%252BActive%252BSite%252Bof%252BProtein%252BN-terminal%252BMethyltransferase%252B1%252BUsing%252BBisubstrate%252BAnalogues%26aulast%3DChen%26aufirst%3DDongxing%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D06052020%26date%3D16072020%26date%3D01072020%26volume%3D63%26issue%3D15%26spage%3D8419%26epage%3D8431" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Ludovic Decultot, Rocco L. Policarpo, Brandon A. Wright, Danny Huang, <span class="NLM_string-name hlFld-ContribAuthor">Matthew D. Shair</span>. </span><span class="cited-content_cbyCitation_article-title">Asymmetric Total Synthesis of C9′-epi-Sinefungin. </span><span class="cited-content_cbyCitation_journal-name">Organic Letters</span><span> <strong>2020,</strong> <em>22 </em>
                                    (14)
                                     , 5594-5599. <a href="https://doi.org/10.1021/acs.orglett.0c01956" title="DOI URL">https://doi.org/10.1021/acs.orglett.0c01956</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.orglett.0c01956&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.orglett.0c01956%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520Letters%26atitle%3DAsymmetric%252BTotal%252BSynthesis%252Bof%252BC9%2525E2%252580%2525B2-epi-Sinefungin%26aulast%3DDecultot%26aufirst%3DLudovic%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D10062020%26date%3D06072020%26volume%3D22%26issue%3D14%26spage%3D5594%26epage%3D5599" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Dongxing Chen, Linjie Li, Krystal Diaz, Iredia D. Iyamu, Ravi Yadav, Nicholas Noinaj, <span class="NLM_string-name hlFld-ContribAuthor">Rong Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Novel Propargyl-Linked Bisubstrate Analogues as Tight-Binding Inhibitors for Nicotinamide N-Methyltransferase. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (23)
                                     , 10783-10797. <a href="https://doi.org/10.1021/acs.jmedchem.9b01255" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01255</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01255%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DNovel%252BPropargyl-Linked%252BBisubstrate%252BAnalogues%252Bas%252BTight-Binding%252BInhibitors%252Bfor%252BNicotinamide%252BN-Methyltransferase%26aulast%3DChen%26aufirst%3DDongxing%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D31072019%26date%3D03122019%26date%3D14112019%26volume%3D62%26issue%3D23%26spage%3D10783%26epage%3D10797" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Gaochan Wu, Tong Zhao, Dongwei Kang, Jian Zhang, Yuning Song, Vigneshwaran Namasivayam, Jacob Kongsted, Christophe Pannecouque, Erik De Clercq, Vasanthanathan Poongavanam, Xinyong Liu, <span class="NLM_string-name hlFld-ContribAuthor">Peng Zhan</span>. </span><span class="cited-content_cbyCitation_article-title">Overview of Recent Strategic Advances in Medicinal Chemistry. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (21)
                                     , 9375-9414. <a href="https://doi.org/10.1021/acs.jmedchem.9b00359" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00359</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00359&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b00359%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DOverview%252Bof%252BRecent%252BStrategic%252BAdvances%252Bin%252BMedicinal%252BChemistry%26aulast%3DWu%26aufirst%3DGaochan%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D26022019%26date%3D28052019%26date%3D03052019%26volume%3D62%26issue%3D21%26spage%3D9375%26epage%3D9414" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Rocco
L. Policarpo, Ludovic Decultot, Elizabeth May, Petr Kuzmič, Samuel Carlson, Danny Huang, Vincent Chu, Brandon A. Wright, Saravanakumar Dhakshinamoorthy, Aimo Kannt, Shilpa Rani, Sreekanth Dittakavi, Joseph D. Panarese, Rachelle Gaudet, <span class="NLM_string-name hlFld-ContribAuthor">Matthew D. Shair</span>. </span><span class="cited-content_cbyCitation_article-title">High-Affinity Alkynyl Bisubstrate Inhibitors of Nicotinamide N-Methyltransferase (NNMT). </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (21)
                                     , 9837-9873. <a href="https://doi.org/10.1021/acs.jmedchem.9b01238" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01238</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01238&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01238%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DHigh-Affinity%252BAlkynyl%252BBisubstrate%252BInhibitors%252Bof%252BNicotinamide%252BN-Methyltransferase%252B%252528NNMT%252529%26aulast%3DPolicarpo%26aufirst%3DRocco%2BL.%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D05082019%26date%3D25102019%26date%3D07102019%26volume%3D62%26issue%3D21%26spage%3D9837%26epage%3D9873" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Yongzhi Gao, Matthijs J. van Haren, Ed E. Moret, Johannes J. M. Rood, Davide Sartini, Alessia Salvucci, Monica Emanuelli, Pierrick Craveur, Nicolas Babault, Jian Jin, <span class="NLM_string-name hlFld-ContribAuthor">Nathaniel I. Martin</span>. </span><span class="cited-content_cbyCitation_article-title">Bisubstrate Inhibitors of Nicotinamide N-Methyltransferase (NNMT) with Enhanced Activity. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (14)
                                     , 6597-6614. <a href="https://doi.org/10.1021/acs.jmedchem.9b00413" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00413</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00413&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b00413%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DBisubstrate%252BInhibitors%252Bof%252BNicotinamide%252BN-Methyltransferase%252B%252528NNMT%252529%252Bwith%252BEnhanced%252BActivity%26aulast%3DGao%26aufirst%3DYongzhi%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D09032019%26date%3D12072019%26date%3D02072019%26volume%3D62%26issue%3D14%26spage%3D6597%26epage%3D6614" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Sudeshna Sen, Santanu Mondal, Li Zheng, Ari J. Salinger, Walter Fast, Eranthie Weerapana, <span class="NLM_string-name hlFld-ContribAuthor">Paul R. Thompson</span>. </span><span class="cited-content_cbyCitation_article-title">Development of a Suicide Inhibition-Based Protein Labeling Strategy for Nicotinamide N-Methyltransferase. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2019,</strong> <em>14 </em>
                                    (4)
                                     , 613-618. <a href="https://doi.org/10.1021/acschembio.9b00211" title="DOI URL">https://doi.org/10.1021/acschembio.9b00211</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.9b00211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.9b00211%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DDevelopment%252Bof%252Ba%252BSuicide%252BInhibition-Based%252BProtein%252BLabeling%252BStrategy%252Bfor%252BNicotinamide%252BN-Methyltransferase%26aulast%3DSen%26aufirst%3DSudeshna%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D16032019%26date%3D01042019%26date%3D05042019%26date%3D01042019%26volume%3D14%26issue%3D4%26spage%3D613%26epage%3D618" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Dongxing Chen, Guangping Dong, Nicholas Noinaj, <span class="NLM_string-name hlFld-ContribAuthor">Rong Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of Bisubstrate Inhibitors for Protein N-Terminal Methyltransferase 1. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (7)
                                     , 3773-3779. <a href="https://doi.org/10.1021/acs.jmedchem.9b00206" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00206</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b00206%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BBisubstrate%252BInhibitors%252Bfor%252BProtein%252BN-Terminal%252BMethyltransferase%252B1%26aulast%3DChen%26aufirst%3DDongxing%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D31012019%26date%3D27032019%26date%3D18032019%26volume%3D62%26issue%3D7%26spage%3D3773%26epage%3D3779" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jie-Ru  Liu</span>, <span class="hlFld-ContribAuthor ">Zhao-Hui  Deng</span>, <span class="hlFld-ContribAuthor ">Xiao-Juan  Zhu</span>, <span class="hlFld-ContribAuthor ">Yu-Rong  Zeng</span>, <span class="hlFld-ContribAuthor ">Xiang-Xiang  Guan</span>, <span class="hlFld-ContribAuthor ">Jiang-Hua  Li</span>, . </span><span class="cited-content_cbyCitation_article-title">Roles of Nicotinamide N-Methyltransferase in Obesity and Type 2 Diabetes. </span><span class="cited-content_cbyCitation_journal-name">BioMed Research International</span><span> <strong>2021,</strong> <em>2021 </em>, 1-8. <a href="https://doi.org/10.1155/2021/9924314" title="DOI URL">https://doi.org/10.1155/2021/9924314</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1155/2021/9924314&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1155%2F2021%2F9924314%26sid%3Dliteratum%253Aachs%26jtitle%3DBioMed%2520Research%2520International%26atitle%3DRoles%252Bof%252BNicotinamide%252BN-Methyltransferase%252Bin%252BObesity%252Band%252BType%252B2%252BDiabetes%26aulast%3DLiu%26aufirst%3DJie-Ru%26date%3D2021%26volume%3D2021%26spage%3D1%26epage%3D8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yongzhi  Gao</span>, <span class="hlFld-ContribAuthor ">Nathaniel I.  Martin</span>, <span class="hlFld-ContribAuthor ">Matthijs J.  van Haren</span>. </span><span class="cited-content_cbyCitation_article-title">Nicotinamide N-methyl transferase (NNMT): An emerging therapeutic target. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2021,</strong> <em>12 </em><a href="https://doi.org/10.1016/j.drudis.2021.05.011" title="DOI URL">https://doi.org/10.1016/j.drudis.2021.05.011</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2021.05.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2021.05.011%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DNicotinamide%252BN-methyl%252Btransferase%252B%252528NNMT%252529%25253A%252BAn%252Bemerging%252Btherapeutic%252Btarget%26aulast%3DGao%26aufirst%3DYongzhi%26date%3D2021%26volume%3D12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Aimo  Kannt</span>, <span class="hlFld-ContribAuthor ">Sridharan  Rajagopal</span>, <span class="hlFld-ContribAuthor ">Mahanandeesha S.  Hallur</span>, <span class="hlFld-ContribAuthor ">Indu  Swamy</span>, <span class="hlFld-ContribAuthor ">Rajendra  Kristam</span>, <span class="hlFld-ContribAuthor ">Saravanakumar  Dhakshinamoorthy</span>, <span class="hlFld-ContribAuthor ">Joerg  Czech</span>, <span class="hlFld-ContribAuthor ">Gernot  Zech</span>, <span class="hlFld-ContribAuthor ">Herman  Schreuder</span>, <span class="hlFld-ContribAuthor ">Sven  Ruf</span>. </span><span class="cited-content_cbyCitation_article-title">Novel Inhibitors of Nicotinamide-N-Methyltransferase for the Treatment of Metabolic Disorders. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2021,</strong> <em>26 </em>
                                    (4)
                                     , 991. <a href="https://doi.org/10.3390/molecules26040991" title="DOI URL">https://doi.org/10.3390/molecules26040991</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules26040991&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules26040991%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DNovel%252BInhibitors%252Bof%252BNicotinamide-N-Methyltransferase%252Bfor%252Bthe%252BTreatment%252Bof%252BMetabolic%252BDisorders%26aulast%3DKannt%26aufirst%3DAimo%26date%3D2021%26date%3D2021%26volume%3D26%26issue%3D4%26spage%3D991" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Iredia D.  Iyamu</span>, <span class="hlFld-ContribAuthor ">Rong  Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Mechanisms and inhibitors of nicotinamide
              N
              -methyltransferase. </span><span class="cited-content_cbyCitation_journal-name">RSC Medicinal Chemistry</span><span> <strong>2021,</strong> <em>28 </em><a href="https://doi.org/10.1039/D1MD00016K" title="DOI URL">https://doi.org/10.1039/D1MD00016K</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D1MD00016K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD1MD00016K%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Medicinal%2520Chemistry%26atitle%3DMechanisms%252Band%252Binhibitors%252Bof%252Bnicotinamide%252BN%252B-methyltransferase%26aulast%3DIyamu%26aufirst%3DIredia%2BD.%26date%3D2021%26date%3D2021%26volume%3D28" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anna  Reustle</span>, <span class="hlFld-ContribAuthor ">Moreno  Di Marco</span>, <span class="hlFld-ContribAuthor ">Carolin  Meyerhoff</span>, <span class="hlFld-ContribAuthor ">Annika  Nelde</span>, <span class="hlFld-ContribAuthor ">Juliane S.  Walz</span>, <span class="hlFld-ContribAuthor ">Stefan  Winter</span>, <span class="hlFld-ContribAuthor ">Siahei  Kandabarau</span>, <span class="hlFld-ContribAuthor ">Florian  Büttner</span>, <span class="hlFld-ContribAuthor ">Mathias  Haag</span>, <span class="hlFld-ContribAuthor ">Linus  Backert</span>, <span class="hlFld-ContribAuthor ">Daniel J.  Kowalewski</span>, <span class="hlFld-ContribAuthor ">Steffen  Rausch</span>, <span class="hlFld-ContribAuthor ">Jörg  Hennenlotter</span>, <span class="hlFld-ContribAuthor ">Viktoria  Stühler</span>, <span class="hlFld-ContribAuthor ">Marcus  Scharpf</span>, <span class="hlFld-ContribAuthor ">Falko  Fend</span>, <span class="hlFld-ContribAuthor ">Arnulf  Stenzl</span>, <span class="hlFld-ContribAuthor ">Hans-Georg  Rammensee</span>, <span class="hlFld-ContribAuthor ">Jens  Bedke</span>, <span class="hlFld-ContribAuthor ">Stefan  Stevanović</span>, <span class="hlFld-ContribAuthor ">Matthias  Schwab</span>, <span class="hlFld-ContribAuthor ">Elke  Schaeffeler</span>. </span><span class="cited-content_cbyCitation_article-title">Integrative -omics and HLA-ligandomics analysis to identify novel drug targets for ccRCC immunotherapy. </span><span class="cited-content_cbyCitation_journal-name">Genome Medicine</span><span> <strong>2020,</strong> <em>12 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13073-020-00731-8" title="DOI URL">https://doi.org/10.1186/s13073-020-00731-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13073-020-00731-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13073-020-00731-8%26sid%3Dliteratum%253Aachs%26jtitle%3DGenome%2520Medicine%26atitle%3DIntegrative%252B-omics%252Band%252BHLA-ligandomics%252Banalysis%252Bto%252Bidentify%252Bnovel%252Bdrug%252Btargets%252Bfor%252BccRCC%252Bimmunotherapy%26aulast%3DReustle%26aufirst%3DAnna%26date%3D2020%26date%3D2020%26volume%3D12%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Iredia D.  Iyamu</span>, <span class="hlFld-ContribAuthor ">Rong  Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Development of fluorescence polarization-based competition assay for nicotinamide N-methyltransferase. </span><span class="cited-content_cbyCitation_journal-name">Analytical Biochemistry</span><span> <strong>2020,</strong> <em>604 </em>, 113833. <a href="https://doi.org/10.1016/j.ab.2020.113833" title="DOI URL">https://doi.org/10.1016/j.ab.2020.113833</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ab.2020.113833&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ab.2020.113833%26sid%3Dliteratum%253Aachs%26jtitle%3DAnalytical%2520Biochemistry%26atitle%3DDevelopment%252Bof%252Bfluorescence%252Bpolarization-based%252Bcompetition%252Bassay%252Bfor%252Bnicotinamide%252BN-methyltransferase%26aulast%3DIyamu%26aufirst%3DIredia%2BD.%26date%3D2020%26volume%3D604%26spage%3D113833" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rostom  Ahmed-Belkacem</span>, <span class="hlFld-ContribAuthor ">Priscila  Sutto-Ortiz</span>, <span class="hlFld-ContribAuthor ">Etienne  Decroly</span>, <span class="hlFld-ContribAuthor ">Jean-Jacques  Vasseur</span>, <span class="hlFld-ContribAuthor ">Françoise  Debart</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis of Adenine Dinucleosides 2′,5′-Bridged by Sulfur-Containing Linkers as Bisubstrate SAM Analogues for Viral RNA 2′-
              O
              -Methyltransferases. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Organic Chemistry</span><span> <strong>2019,</strong> <em>2019 </em>
                                    (38)
                                     , 6486-6495. <a href="https://doi.org/10.1002/ejoc.201901120" title="DOI URL">https://doi.org/10.1002/ejoc.201901120</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ejoc.201901120&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fejoc.201901120%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DSynthesis%252Bof%252BAdenine%252BDinucleosides%252B2%2525E2%252580%2525B2%25252C5%2525E2%252580%2525B2-Bridged%252Bby%252BSulfur-Containing%252BLinkers%252Bas%252BBisubstrate%252BSAM%252BAnalogues%252Bfor%252BViral%252BRNA%252B2%2525E2%252580%2525B2-%252BO%252B-Methyltransferases%26aulast%3DAhmed-Belkacem%26aufirst%3DRostom%26date%3D2019%26date%3D2019%26volume%3D2019%26issue%3D38%26spage%3D6486%26epage%3D6495" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Marlene L.  Rothe</span>, <span class="hlFld-ContribAuthor ">Jie  Li</span>, <span class="hlFld-ContribAuthor ">Ernesto  Garibay</span>, <span class="hlFld-ContribAuthor ">Bradley S.  Moore</span>, <span class="hlFld-ContribAuthor ">Shaun M. K.  McKinnie</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis, bioactivity, and enzymatic modification of antibacterial thiotetromycin derivatives. </span><span class="cited-content_cbyCitation_journal-name">Organic & Biomolecular Chemistry</span><span> <strong>2019,</strong> <em>17 </em>
                                    (13)
                                     , 3416-3423. <a href="https://doi.org/10.1039/C8OB03109F" title="DOI URL">https://doi.org/10.1039/C8OB03109F</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C8OB03109F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC8OB03109F%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520%2526%2520Biomolecular%2520Chemistry%26atitle%3DSynthesis%25252C%252Bbioactivity%25252C%252Band%252Benzymatic%252Bmodification%252Bof%252Bantibacterial%252Bthiotetromycin%252Bderivatives%26aulast%3DRothe%26aufirst%3DMarlene%2BL.%26date%3D2019%26date%3D2019%26volume%3D17%26issue%3D13%26spage%3D3416%26epage%3D3423" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hsin-Yu  Lee</span>, <span class="hlFld-ContribAuthor ">Radu M.  Suciu</span>, <span class="hlFld-ContribAuthor ">Benjamin D.  Horning</span>, <span class="hlFld-ContribAuthor ">Ekaterina V.  Vinogradova</span>, <span class="hlFld-ContribAuthor ">Olesya A.  Ulanovskaya</span>, <span class="hlFld-ContribAuthor ">Benjamin F.  Cravatt</span>. </span><span class="cited-content_cbyCitation_article-title">Covalent inhibitors of nicotinamide N-methyltransferase (NNMT) provide evidence for target engagement challenges in situ. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2018,</strong> <em>28 </em>
                                    (16)
                                     , 2682-2687. <a href="https://doi.org/10.1016/j.bmcl.2018.04.017" title="DOI URL">https://doi.org/10.1016/j.bmcl.2018.04.017</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2018.04.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2018.04.017%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DCovalent%252Binhibitors%252Bof%252Bnicotinamide%252BN-methyltransferase%252B%252528NNMT%252529%252Bprovide%252Bevidence%252Bfor%252Btarget%252Bengagement%252Bchallenges%252Bin%252Bsitu%26aulast%3DLee%26aufirst%3DHsin-Yu%26date%3D2018%26volume%3D28%26issue%3D16%26spage%3D2682%26epage%3D2687" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jane  Panteleev</span>, <span class="hlFld-ContribAuthor ">Hua  Gao</span>, <span class="hlFld-ContribAuthor ">Lei  Jia</span>. </span><span class="cited-content_cbyCitation_article-title">Recent applications of machine learning in medicinal chemistry. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2018,</strong> <em>28 </em>
                                    (17)
                                     , 2807-2815. <a href="https://doi.org/10.1016/j.bmcl.2018.06.046" title="DOI URL">https://doi.org/10.1016/j.bmcl.2018.06.046</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2018.06.046&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2018.06.046%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DRecent%252Bapplications%252Bof%252Bmachine%252Blearning%252Bin%252Bmedicinal%252Bchemistry%26aulast%3DPanteleev%26aufirst%3DJane%26date%3D2018%26volume%3D28%26issue%3D17%26spage%3D2807%26epage%3D2815" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Colette  Atdjian</span>, <span class="hlFld-ContribAuthor ">Laura  Iannazzo</span>, <span class="hlFld-ContribAuthor ">Emmanuelle  Braud</span>, <span class="hlFld-ContribAuthor ">Mélanie  Ethève-Quelquejeu</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis of SAM-Adenosine Conjugates for the Study of m
              6
              A-RNA Methyltransferases. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Organic Chemistry</span><span> <strong>2018,</strong> <em>2018 </em>
                                    (32)
                                     , 4411-4425. <a href="https://doi.org/10.1002/ejoc.201800798" title="DOI URL">https://doi.org/10.1002/ejoc.201800798</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ejoc.201800798&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fejoc.201800798%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DSynthesis%252Bof%252BSAM-Adenosine%252BConjugates%252Bfor%252Bthe%252BStudy%252Bof%252Bm%252B6%252BA-RNA%252BMethyltransferases%26aulast%3DAtdjian%26aufirst%3DColette%26date%3D2018%26date%3D2018%26volume%3D2018%26issue%3D32%26spage%3D4411%26epage%3D4425" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-4/acs.jmedchem.7b01422/20180215/images/medium/jm-2017-01422z_0007.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-4/acs.jmedchem.7b01422/20180215/images/large/jm-2017-01422z_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01422&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-4/acs.jmedchem.7b01422/20180215/images/medium/jm-2017-01422z_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-4/acs.jmedchem.7b01422/20180215/images/large/jm-2017-01422z_0008.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Chemical structures of NNMT inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-4/acs.jmedchem.7b01422/20180215/images/large/jm-2017-01422z_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01422&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-4/acs.jmedchem.7b01422/20180215/images/medium/jm-2017-01422z_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-4/acs.jmedchem.7b01422/20180215/images/large/jm-2017-01422z_0001.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Design of the bisubstrate NNMT inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-4/acs.jmedchem.7b01422/20180215/images/large/jm-2017-01422z_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01422&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-4/acs.jmedchem.7b01422/20180215/images/medium/jm-2017-01422z_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-4/acs.jmedchem.7b01422/20180215/images/large/jm-2017-01422z_0002.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Docking analysis of compound <b>6</b> in the binding sites of hNNMT. (A) Docking of compound <b>6</b> (green) in the hNNMT (cyan) structure (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2IIP">2IIP</a>). H-bond interactions are shown in yellow dotted lines. Water molecules are illustrated as red balls. (B) Overlay of the docking model with the hNNMT (gray)–nicotinamide (magenta)–SAH (yellow) complex (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ROD">3ROD</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-4/acs.jmedchem.7b01422/20180215/images/large/jm-2017-01422z_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01422&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-4/acs.jmedchem.7b01422/20180215/images/medium/jm-2017-01422z_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-4/acs.jmedchem.7b01422/20180215/images/large/jm-2017-01422z_0006.jpeg" id="rightTab-GRAPHIC-d345e721-autogenerated" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthetic route for <b>6</b> and <b>7</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-4/acs.jmedchem.7b01422/20180215/images/large/jm-2017-01422z_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01422&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) NaBH(OAc)<sub>3</sub>, AcOH, DCM, rt, 16 h, 61–71%; (b) K<sub>2</sub>CO<sub>3</sub>, DMSO, H<sub>2</sub>O<sub>2</sub>, 0 °C to rt, 3 h; (c) HCl in dioxane, 0 °C to rt, 4 h; then 2 drops of water, rt, 4 h, 46–57% over 2 steps.</p></p></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-4/acs.jmedchem.7b01422/20180215/images/medium/jm-2017-01422z_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-4/acs.jmedchem.7b01422/20180215/images/large/jm-2017-01422z_0003.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Biochemical and biophysical characterization of <b>6</b> and <b>7</b> and selectivity assessment of <b>6</b>. (A) IC<sub>50</sub> determination for <b>6</b> (14 ± 1.5 μM; HS: 1.1) and <b>7</b> (160 ± 1 μM) against hNNMT (<i>n</i> = 3). (B) ITC was used to assess binding of hNNMT to <b>6</b> (left) and <b>7</b> (right). (C) Selectivity of <b>6</b> was determined against a panel of 34 MTs and 1 HAT at two compound concentrations (10 μM (blue) and 50 μM (red)). (D) IC<sub>50</sub> determination for <b>6</b> against DOT1L (1.3 ± 0.2 μM; HS: 1.1), PRMT7 (20 ± 2 μM; HS: 1.1), BCDIN3D (40 ± 2 μM; HS: 1.3), and SMYD2 (62 ± 7 μM; HS: 0.6) were performed in triplicate. HS: Hill slope.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-4/acs.jmedchem.7b01422/20180215/images/large/jm-2017-01422z_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01422&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-4/acs.jmedchem.7b01422/20180215/images/medium/jm-2017-01422z_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-4/acs.jmedchem.7b01422/20180215/images/large/jm-2017-01422z_0004.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. MOA study of NNMT inhibition with <b>6</b>. (A) Lineweaver–Burk plot of 1/rate versus 1/[SAM] with varying concentrations of inhibitor <b>6</b> showing competitive inhibition and (B) Lineweaver–Burk plot of 1/rate versus 1/[nicotinamide] with varying concentrations of inhibitor <b>6</b> showing noncompetitive inhibition. Eight concentrations of inhibitor <b>6</b> (from 2 to 250 μM) were used for both experiments: 2 μM (●), 4 μM (○), 8 μM (▲), 16 μM (△), 31 μM (■), 63 μM (□), 125 μM (◆), and 250 μM (◇). ITC was used to assess binding of (C) SAM and (D) nicotinamide to hNNMT.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-4/acs.jmedchem.7b01422/20180215/images/large/jm-2017-01422z_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01422&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-4/acs.jmedchem.7b01422/20180215/images/medium/jm-2017-01422z_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-4/acs.jmedchem.7b01422/20180215/images/large/jm-2017-01422z_0005.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. X-ray crystal structures. (A) The X-ray crystal structure of a binary complex of <b>6</b> (green) and hNNMT (cyan) (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6B1A">6B1A</a>). The surface of the protein is shown in gray. (B) Key interactions between <b>6</b> and hNNMT. H-bond interactions are shown in yellow dotted lines. Water molecules are illustrated as red balls. (C) Overlay of the crystal structure of the hNNMT (cyan)–<b>6</b> (green) binary complex (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6B1A">6B1A</a>) with the crystal structure of hNNMT (not shown)–nicotinamide (magenta)–SAH (yellow) complex (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ROD">3ROD</a>). (D) Overlay of the crystal structure of the hNNMT (cyan)–<b>6</b> (green) binary complex (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6B1A">6B1A</a>) with the crystal structure of monkey NNMT (not shown)–<b>1</b> (blue)–SAH (yellow) complex (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5XVQ">5XVQ</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-4/acs.jmedchem.7b01422/20180215/images/large/jm-2017-01422z_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01422&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i31">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_31283" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_31283" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 53 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Alston, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abeles, R. H.</span><span> </span><span class="NLM_article-title">Substrate specificity of nicotinamide methyltransferase isolated from porcine liver</span> <span class="citation_source-journal">Arch. Biochem. Biophys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">260</span><span class="NLM_x">, </span> <span class="NLM_fpage">601</span><span class="NLM_x">–</span> <span class="NLM_lpage">608</span><span class="refDoi"> DOI: 10.1016/0003-9861(88)90487-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1016%2F0003-9861%2888%2990487-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=2963591" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADyaL1cXhs1elu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=260&publication_year=1988&pages=601-608&author=T.+A.+Alstonauthor=R.+H.+Abeles&title=Substrate+specificity+of+nicotinamide+methyltransferase+isolated+from+porcine+liver&doi=10.1016%2F0003-9861%2888%2990487-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Substrate specificity of nicotinamide methyltransferase isolated from porcine liver</span></div><div class="casAuthors">Alston, Theodore A.; Abeles, Robert H.</div><div class="citationInfo"><span class="NLM_cas:title">Archives of Biochemistry and Biophysics</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">260</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">601-8</span>CODEN:
                <span class="NLM_cas:coden">ABBIA4</span>;
        ISSN:<span class="NLM_cas:issn">0003-9861</span>.
    </div><div class="casAbstract">Nicotinamide methyltransferase (EC 2.1.1.1) was purified >1300-fold from porcine liver.  The enzyme is electrophoretically homogeneous, exhibiting a relative mol. wt. of 27,000.  In addn. to acting on nicotinamide and close structural analogs such as thionicotinamide and 3-acetylpyridine, the enzyme actively accommodates poor analogs such as quinoline, isoquinoline, and 1,2,3,4-tetrahydroisoquinoline as Me group acceptors.  Thus, the enzyme may detoxify numerous alkaloids in vivo.  In some cases, this enzyme might paradoxically increase toxicities of substrates, but the hepatotoxic antibiotic pyrazinamide, which is considered as a potential enzyme-activated electrophile, did not function detectably as a substrate for the isolated enzyme.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrApJV903Z9o7Vg90H21EOLACvtfcHk0lg_GLMEqfThCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXhs1elu7c%253D&md5=ff4b08c064606a2373a4fa5164c837ba</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2F0003-9861%2888%2990487-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0003-9861%252888%252990487-0%26sid%3Dliteratum%253Aachs%26aulast%3DAlston%26aufirst%3DT.%2BA.%26aulast%3DAbeles%26aufirst%3DR.%2BH.%26atitle%3DSubstrate%2520specificity%2520of%2520nicotinamide%2520methyltransferase%2520isolated%2520from%2520porcine%2520liver%26jtitle%3DArch.%2520Biochem.%2520Biophys.%26date%3D1988%26volume%3D260%26spage%3D601%26epage%3D608%26doi%3D10.1016%2F0003-9861%2888%2990487-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">van Haren, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sastre Torano, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sartini, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emanuelli, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parsons, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, N. I.</span><span> </span><span class="NLM_article-title">A rapid and efficient assay for the characterization of substrates and inhibitors of nicotinamide N-methyltransferase</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">5307</span><span class="NLM_x">–</span> <span class="NLM_lpage">5315</span><span class="refDoi"> DOI: 10.1021/acs.biochem.6b00733</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biochem.6b00733" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVWntL7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=5307-5315&author=M.+J.+van+Harenauthor=J.+Sastre+Toranoauthor=D.+Sartiniauthor=M.+Emanuelliauthor=R.+B.+Parsonsauthor=N.+I.+Martin&title=A+rapid+and+efficient+assay+for+the+characterization+of+substrates+and+inhibitors+of+nicotinamide+N-methyltransferase&doi=10.1021%2Facs.biochem.6b00733"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">A rapid and efficient assay for the characterization of substrates and inhibitors of nicotinamide N-methyltransferase</span></div><div class="casAuthors">van Haren, Matthijs J.; Sastre Torano, Javier; Sartini, Davide; Emanuelli, Monica; Parsons, Richard B.; Martin, Nathaniel I.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">5307-5315</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) is one of the most abundant small mol. methyltransferases in the human body and is primarily responsible for the N-methylation of the nicotinamide (vitamin B3).  Employing the cofactor, S-adenosyl-L-methionine, NNMT transfers a Me group to the pyridine N atom of nicotinamide to generate N-methylnicotinamide.  Interestingly, NNMT is also able to N-methylate a variety of other pyridine-contg. small mols., suggesting a secondary role for the enzyme in the detoxification of xenobiotics.  A no. of recent studies have also revealed links between NNMT overexpression and a variety of diseases, including multiple cancers, Parkinson's disease, diabetes, and obesity.  Here, to facilitate further study of both the substrate scope and potential for inhibitor development, the authors describe the development of a new activity assay for human NNMT.  The assay makes use of ultra-high-performance hydrophilic interaction chromatog., allowing for the rapid sepn. of the reaction products, coupled with quadrupole time-of-flight mass spectrometric detection, providing for enhanced sensitivity and enabling high-throughput sample anal.  The authors successfully demonstrate the general applicability of the method by performing kinetic analyses of NNMT-mediated methylation for a range of pyridine-based substrates.  These findings also provide new insight into the diversity of substrate recognition by NNMT in a quant. manner.  In addn., the authors further established the suitability of the assay for the identification and characterization of small mol. inhibitors of NNMT.  To do so, the authors investigated the inhibition of NNMT by the nonspecific methyltransferase inhibitors, sinefungin and S-adenosyl-L-homocysteine, revealing IC50 values in the low micromolar range.  The results of these inhibition studies are particularly noteworthy as they will permit future efforts toward the development of new NNMT-specific inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBC8U-yKb1brVg90H21EOLACvtfcHk0ljezHzchSOCEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVWntL7O&md5=0ebb58b665b91dd5983b2bef91886c10</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Facs.biochem.6b00733&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biochem.6b00733%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BHaren%26aufirst%3DM.%2BJ.%26aulast%3DSastre%2BTorano%26aufirst%3DJ.%26aulast%3DSartini%26aufirst%3DD.%26aulast%3DEmanuelli%26aufirst%3DM.%26aulast%3DParsons%26aufirst%3DR.%2BB.%26aulast%3DMartin%26aufirst%3DN.%2BI.%26atitle%3DA%2520rapid%2520and%2520efficient%2520assay%2520for%2520the%2520characterization%2520of%2520substrates%2520and%2520inhibitors%2520of%2520nicotinamide%2520N-methyltransferase%26jtitle%3DBiochemistry%26date%3D2016%26volume%3D55%26spage%3D5307%26epage%3D5315%26doi%3D10.1021%2Facs.biochem.6b00733" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Aksoy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szumlanski, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weinshilboum, R. M.</span><span> </span><span class="NLM_article-title">Human liver nicotinamide N-methyltransferase. cDNA cloning, expression, and biochemical characterization</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">269</span><span class="NLM_x">, </span> <span class="NLM_fpage">14835</span><span class="NLM_x">–</span> <span class="NLM_lpage">14840</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=8182091" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADyaK2cXkslCqtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=269&publication_year=1994&pages=14835-14840&author=S.+Aksoyauthor=C.+L.+Szumlanskiauthor=R.+M.+Weinshilboum&title=Human+liver+nicotinamide+N-methyltransferase.+cDNA+cloning%2C+expression%2C+and+biochemical+characterization"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Human liver nicotinamide N-methyltransferase. cDNA cloning, expression, and biochemical characterization</span></div><div class="casAuthors">Aksoy, Saime; Szumlanski, Carol L.; Weinshilboum, Richard M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">269</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">14835-40</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) catalyzes the N-methylation of nicotinamide and other pyridines.  Human liver NNMT activity has a bimodal frequency distribution, an observation which raises the possibility that this enzyme might be regulated by a genetic polymorphism, a polymorphism that could have functional implications for individual differences in drug and xenobiotic toxicity.  As a first step toward testing this hypothesis, the authors set out to clone and express a cDNA for human liver NNMT.  Human liver NNMT was partially purified, photoaffinity-labeled, subjected to limited proteolysis, and partial amino acid sequence information was obtained.  The polymerase chain reaction was then used to amplify a 550-nucleotide sequence with human liver cDNA as template and primers designed on the basis of the NNMT amino acid sequence.  The 5'- and 3'-ends of a human liver NNMT cDNA were obtained by use of the rapid amplification of cDNA ends.  The combined use of these approaches resulted in the isolation of a human liver NNMT cDNA that was 969 nucleotides in length, with a 792-nucleotide open reading frame that encoded a 264-amino acid protein with a calcd. mol. mass of 29.6 kDa.  The human liver NNMT cDNA was transcribed in vitro and translated with a reticulocyte lysate system to yield a protein with a mol. mass of approx. 29 kDa that comigrated during SDS-polyacrylamide gel electrophoresis with photoaffinity-labeled human liver NNMT.  The NNMT cDNA was also subcloned into the eukaryotic expression vector p91023(B).  COS-1 cells transfected with this construct expressed a high level of NNMT enzymic activity, and the biochem. properties of this activity were similar to those of human liver NNMT.  Human liver NNMT and transfected COS-1 cell NNMT had apparent Km values for the two cosubstrates for the reaction, nicotinamide and S-adenosyl-L-methionine, of 0.43 and 0.38 mM and of 1.8 and 2.2 μM, resp.  IC50 values for the inhibition of NNMT by N1-methylnicotinamide were 60 and 30 μM for human liver and COS-1 cell-expressed NNMT, resp.  Cloning of a cDNA for human liver NNMT should make it possible to test the hypothesis that inheritance may play a role in the regulation of individual differences in human liver NNMT activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZRn3_QD5atbVg90H21EOLACvtfcHk0ljezHzchSOCEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXkslCqtro%253D&md5=d5d9edf09cb329ecc01b49fd819fad47</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAksoy%26aufirst%3DS.%26aulast%3DSzumlanski%26aufirst%3DC.%2BL.%26aulast%3DWeinshilboum%26aufirst%3DR.%2BM.%26atitle%3DHuman%2520liver%2520nicotinamide%2520N-methyltransferase.%2520cDNA%2520cloning%252C%2520expression%252C%2520and%2520biochemical%2520characterization%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1994%26volume%3D269%26spage%3D14835%26epage%3D14840" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Riederer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Erwa, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zimmermann, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frank, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zechner, R.</span><span> </span><span class="NLM_article-title">Adipose tissue as a source of nicotinamide N-methyltransferase and homocysteine</span> <span class="citation_source-journal">Atherosclerosis</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">204</span><span class="NLM_x">, </span> <span class="NLM_fpage">412</span><span class="NLM_x">–</span> <span class="NLM_lpage">417</span><span class="refDoi"> DOI: 10.1016/j.atherosclerosis.2008.09.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1016%2Fj.atherosclerosis.2008.09.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=18996527" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmtl2ltr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=204&publication_year=2009&pages=412-417&author=M.+Riedererauthor=W.+Erwaauthor=R.+Zimmermannauthor=S.+Frankauthor=R.+Zechner&title=Adipose+tissue+as+a+source+of+nicotinamide+N-methyltransferase+and+homocysteine&doi=10.1016%2Fj.atherosclerosis.2008.09.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Adipose tissue as a source of nicotinamide N-methyltransferase and homocysteine</span></div><div class="casAuthors">Riederer, Monika; Erwa, Wolfgang; Zimmermann, Robert; Frank, Sasa; Zechner, Rudolf</div><div class="citationInfo"><span class="NLM_cas:title">Atherosclerosis (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">204</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">412-417</span>CODEN:
                <span class="NLM_cas:coden">ATHSBL</span>;
        ISSN:<span class="NLM_cas:issn">0021-9150</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) catalyzes the conversion of nicotinamide to 1-methylnicotinamide and plays an important role in hepatic detoxification reactions.  Here we show that, in addn. to the liver, 3T3-L1 adipocytes as well as human and murine adipose tissue explants express high amts. of enzymically active NNMT.  NNMT mRNA levels and enzyme activity increased in 3T3-L1 cells in a differentiation-dependent manner.  Homocysteine, the atherogenic product of the NNMT-catalyzed reaction, was secreted from 3T3-L1 cells or adipose tissue cultures.  Homocysteine release increased during 3T3-L1 differentiation and was reduced when adipose tissue was treated with the NNMT inhibitor 1-methylnicotinamide.  Nicotinic acid (NA), a widely used drug to lower elevated plasma lipid levels, induced NNMT enzyme activity in white adipose tissue of mice.  In tissue culture nicotinamide treatment led to an increase in adipose tissue homocysteine secretion.  These data support the concept that adipose tissue NNMT contributes to the increased plasma homocysteine levels in patients treated with NA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAPqArSvc84rVg90H21EOLACvtfcHk0ljezHzchSOCEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmtl2ltr0%253D&md5=4dd8fbd5fbb95aab930ce2cf85d8c825</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.atherosclerosis.2008.09.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.atherosclerosis.2008.09.015%26sid%3Dliteratum%253Aachs%26aulast%3DRiederer%26aufirst%3DM.%26aulast%3DErwa%26aufirst%3DW.%26aulast%3DZimmermann%26aufirst%3DR.%26aulast%3DFrank%26aufirst%3DS.%26aulast%3DZechner%26aufirst%3DR.%26atitle%3DAdipose%2520tissue%2520as%2520a%2520source%2520of%2520nicotinamide%2520N-methyltransferase%2520and%2520homocysteine%26jtitle%3DAtherosclerosis%26date%3D2009%26volume%3D204%26spage%3D412%26epage%3D417%26doi%3D10.1016%2Fj.atherosclerosis.2008.09.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Pissios, P.</span><span> </span><span class="NLM_article-title">Nicotinamide N-methyltransferase: More than a vitamin B3 clearance enzyme</span> <span class="citation_source-journal">Trends Endocrinol. Metab.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">340</span><span class="NLM_x">–</span> <span class="NLM_lpage">353</span><span class="refDoi"> DOI: 10.1016/j.tem.2017.02.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1016%2Fj.tem.2017.02.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=28291578" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BC2sXivFehsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2017&pages=340-353&author=P.+Pissios&title=Nicotinamide+N-methyltransferase%3A+More+than+a+vitamin+B3+clearance+enzyme&doi=10.1016%2Fj.tem.2017.02.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Nicotinamide N-Methyltransferase: More Than a Vitamin B3 Clearance Enzyme</span></div><div class="casAuthors">Pissios, Pavlos</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Endocrinology and Metabolism</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">340-353</span>CODEN:
                <span class="NLM_cas:coden">TENME4</span>;
        ISSN:<span class="NLM_cas:issn">1043-2760</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Nicotinamide (NAM) N-methyltransferase (NNMT) was originally identified as the enzyme responsible for the methylation of NAM, one of the forms of vitamin B3.  Methylated NAM is eventually excreted from the body.  Recent evidence has expanded the role of NNMT beyond clearance of excess vitamin B3.  NNMT has been implicated in the regulation of multiple metabolic pathways in tissues such as adipose and liver as well as cancer cells through the consumption of Me donors and generation of active metabolites.  This review examines recent findings regarding the function of NNMT in physiol. and disease and highlights potential new avenues for therapeutic intervention.  Finally, key gaps in our knowledge about this enzymic system and future areas of investigation are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHRx9vhXz5LbVg90H21EOLACvtfcHk0ljezHzchSOCEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXivFehsrk%253D&md5=cf088bcc9434e006062d72de0c47c62a</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.tem.2017.02.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tem.2017.02.004%26sid%3Dliteratum%253Aachs%26aulast%3DPissios%26aufirst%3DP.%26atitle%3DNicotinamide%2520N-methyltransferase%253A%2520More%2520than%2520a%2520vitamin%2520B3%2520clearance%2520enzyme%26jtitle%3DTrends%2520Endocrinol.%2520Metab.%26date%3D2017%26volume%3D28%26spage%3D340%26epage%3D353%26doi%3D10.1016%2Fj.tem.2017.02.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Schmeisser, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mansfeld, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhlow, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weimer, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Priebe, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heiland, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Birringer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Groth, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Segref, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kanfi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Price, N. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmeisser, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schuster, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pfeiffer, A. F. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guthke, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Platzer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoppe, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cohen, H. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zarse, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sinclair, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ristow, M.</span><span> </span><span class="NLM_article-title">Role of sirtuins in lifespan regulation is linked to methylation of nicotinamide</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">693</span><span class="NLM_x">–</span> <span class="NLM_lpage">700</span><span class="refDoi"> DOI: 10.1038/nchembio.1352</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1038%2Fnchembio.1352" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=24077178" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFalu7bP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=693-700&author=K.+Schmeisserauthor=J.+Mansfeldauthor=D.+Kuhlowauthor=S.+Weimerauthor=S.+Priebeauthor=I.+Heilandauthor=M.+Birringerauthor=M.+Grothauthor=A.+Segrefauthor=Y.+Kanfiauthor=N.+L.+Priceauthor=S.+Schmeisserauthor=S.+Schusterauthor=A.+F.+H.+Pfeifferauthor=R.+Guthkeauthor=M.+Platzerauthor=T.+Hoppeauthor=H.+Y.+Cohenauthor=K.+Zarseauthor=D.+A.+Sinclairauthor=M.+Ristow&title=Role+of+sirtuins+in+lifespan+regulation+is+linked+to+methylation+of+nicotinamide&doi=10.1038%2Fnchembio.1352"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Role of sirtuins in lifespan regulation is linked to methylation of nicotinamide</span></div><div class="casAuthors">Schmeisser, Kathrin; Mansfeld, Johannes; Kuhlow, Doreen; Weimer, Sandra; Priebe, Steffen; Heiland, Ines; Birringer, Marc; Groth, Marco; Segref, Alexandra; Kanfi, Yariv; Price, Nathan L.; Schmeisser, Sebastian; Schuster, Stefan; Pfeiffer, Andreas F. H.; Guthke, Reinhard; Platzer, Matthias; Hoppe, Thorsten; Cohen, Haim Y.; Zarse, Kim; Sinclair, David A.; Ristow, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">693-700</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Sirtuins, a family of histone deacetylases, have a fiercely debated role in regulating lifespan.  In contrast with recent observations, here we find that overexpression of sir-2.1, the ortholog of mammalian SirT1, does extend Caenorhabditis elegans lifespan.  Sirtuins mandatorily convert NAD+ into nicotinamide (NAM).  We here find that NAM and its metabolite, 1-methylnicotinamide (MNA), extend C. elegans lifespan, even in the absence of sir-2.1.  We identify a previously unknown C. elegans nicotinamide-N-methyltransferase, encoded by a gene now named anmt-1, to generate MNA from NAM.  Disruption and overexpression of anmt-1 have opposing effects on lifespan independent of sirtuins, with loss of anmt-1 fully inhibiting sir-2.1-mediated lifespan extension.  MNA serves as a substrate for a newly identified aldehyde oxidase, GAD-3, to generate hydrogen peroxide, which acts as a mitohormetic reactive oxygen species signal to promote C. elegans longevity.  Taken together, sirtuin-mediated lifespan extension depends on methylation of NAM, providing an unexpected mechanistic role for sirtuins beyond histone deacetylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVNUIcRbjzP7Vg90H21EOLACvtfcHk0lir2qHcPYGuCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFalu7bP&md5=6a109a82f2141577cc460ff2e278e1c5</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1352&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1352%26sid%3Dliteratum%253Aachs%26aulast%3DSchmeisser%26aufirst%3DK.%26aulast%3DMansfeld%26aufirst%3DJ.%26aulast%3DKuhlow%26aufirst%3DD.%26aulast%3DWeimer%26aufirst%3DS.%26aulast%3DPriebe%26aufirst%3DS.%26aulast%3DHeiland%26aufirst%3DI.%26aulast%3DBirringer%26aufirst%3DM.%26aulast%3DGroth%26aufirst%3DM.%26aulast%3DSegref%26aufirst%3DA.%26aulast%3DKanfi%26aufirst%3DY.%26aulast%3DPrice%26aufirst%3DN.%2BL.%26aulast%3DSchmeisser%26aufirst%3DS.%26aulast%3DSchuster%26aufirst%3DS.%26aulast%3DPfeiffer%26aufirst%3DA.%2BF.%2BH.%26aulast%3DGuthke%26aufirst%3DR.%26aulast%3DPlatzer%26aufirst%3DM.%26aulast%3DHoppe%26aufirst%3DT.%26aulast%3DCohen%26aufirst%3DH.%2BY.%26aulast%3DZarse%26aufirst%3DK.%26aulast%3DSinclair%26aufirst%3DD.%2BA.%26aulast%3DRistow%26aufirst%3DM.%26atitle%3DRole%2520of%2520sirtuins%2520in%2520lifespan%2520regulation%2520is%2520linked%2520to%2520methylation%2520of%2520nicotinamide%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2013%26volume%3D9%26spage%3D693%26epage%3D700%26doi%3D10.1038%2Fnchembio.1352" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Chlopicki, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swies, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mogielnicki, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buczko, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bartus, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lomnicka, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adamus, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gebicki, J.</span><span> </span><span class="NLM_article-title">1-Methylnicotinamide (MNA), a primary metabolite of nicotinamide, exerts anti-thrombotic activity mediated by a cyclooxygenase-2/prostacyclin pathway</span> <span class="citation_source-journal">Br. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">152</span><span class="NLM_x">, </span> <span class="NLM_fpage">230</span><span class="NLM_x">–</span> <span class="NLM_lpage">239</span><span class="refDoi"> DOI: 10.1038/sj.bjp.0707383</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1038%2Fsj.bjp.0707383" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=17641676" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVWhsLzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=152&publication_year=2007&pages=230-239&author=S.+Chlopickiauthor=J.+Swiesauthor=A.+Mogielnickiauthor=W.+Buczkoauthor=M.+Bartusauthor=M.+Lomnickaauthor=J.+Adamusauthor=J.+Gebicki&title=1-Methylnicotinamide+%28MNA%29%2C+a+primary+metabolite+of+nicotinamide%2C+exerts+anti-thrombotic+activity+mediated+by+a+cyclooxygenase-2%2Fprostacyclin+pathway&doi=10.1038%2Fsj.bjp.0707383"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">1-Methylnicotinamide (MNA), a primary metabolite of nicotinamide, exerts anti-thrombotic activity mediated by a cyclooxygenase-2/prostacyclin pathway</span></div><div class="casAuthors">Chlopicki, S.; Swies, J.; Mogielnicki, A.; Buczko, W.; Bartus, M.; Lomnicka, M.; Adamus, J.; Gebicki, J.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">152</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">230-239</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">0007-1188</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">1-Methylnicotinamide (MNA) has been considered to be an inactive metabolite of nicotinamide.  Here we assessed the anti-thrombotic activity of MNA in vivo.  Antithrombotic action of MNA was studied in normotensive rats with extracorporeal thrombus formation (thrombolysis), in renovascular hypertensive rats with intraarterial thrombus formation (arterial thrombosis) and in a venous thrombosis model in rats (venous thrombosis).  MNA (3-100 mg kg-1) induced a dose-dependent and sustained thrombolytic response, assocd. with a rise in 6-keto-PGF1α in blood.  Various compds. structurally related to MNA were either inactive or weaker thrombolytics.  Rofecoxib (0.01-1 mg kg-1), dose-dependently inhibited the thrombolytic response of MNA, indomethacin (5 mg kg-1) abolished it, while L-NAME (5 mg kg-1) were without effect.  MNA (3-30 mg kg-1) also reduced arterial thrombosis and this effect was abrogated by indomethacin (2.5 mg kg-1) as well as by rofecoxib (1 mg kg-1).  MNA, however, did not affect venous thrombosis.  In vitro MNA did not modify platelet aggregation nor induce vasodilation.  MNA displayed a profile of anti-thrombotic activity in vivo that surpasses that of closely related compds.  MNA inhibited platelet-dependent thrombosis by a mechanism involving cyclooxygenase-2 and prostacyclin.  Our findings suggest that endogenous MNA, produced in the liver by nicotinamide N-methyltransferase, could be an endogenous activator of prostacyclin prodn. and thus may regulate thrombotic as well as inflammatory processes in the cardiovascular system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptun94rn-oo7Vg90H21EOLACvtfcHk0lir2qHcPYGuCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVWhsLzI&md5=ec9a668f372d4ddefaba82b6da551158</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjp.0707383&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjp.0707383%26sid%3Dliteratum%253Aachs%26aulast%3DChlopicki%26aufirst%3DS.%26aulast%3DSwies%26aufirst%3DJ.%26aulast%3DMogielnicki%26aufirst%3DA.%26aulast%3DBuczko%26aufirst%3DW.%26aulast%3DBartus%26aufirst%3DM.%26aulast%3DLomnicka%26aufirst%3DM.%26aulast%3DAdamus%26aufirst%3DJ.%26aulast%3DGebicki%26aufirst%3DJ.%26atitle%3D1-Methylnicotinamide%2520%2528MNA%2529%252C%2520a%2520primary%2520metabolite%2520of%2520nicotinamide%252C%2520exerts%2520anti-thrombotic%2520activity%2520mediated%2520by%2520a%2520cyclooxygenase-2%252Fprostacyclin%2520pathway%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2007%26volume%3D152%26spage%3D230%26epage%3D239%26doi%3D10.1038%2Fsj.bjp.0707383" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Domagala, T. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szeffler, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dobrucki, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dropinski, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Polanski, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leszczynska-Wiloch, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kotula-Horowitz, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wojciechowski, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wojnowski, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szczeklik, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kalinowski, L.</span><span> </span><span class="NLM_article-title">Nitric oxide production and endothelium-dependent vasorelaxation ameliorated by N1-methylnicotinamide in human blood vessels</span> <span class="citation_source-journal">Hypertension</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">825</span><span class="NLM_x">–</span> <span class="NLM_lpage">832</span><span class="refDoi"> DOI: 10.1161/HYPERTENSIONAHA.111.183210</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1161%2FHYPERTENSIONAHA.111.183210" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2012&pages=825-832&author=T.+B.+Domagalaauthor=A.+Szefflerauthor=L.+W.+Dobruckiauthor=J.+Dropinskiauthor=S.+Polanskiauthor=M.+Leszczynska-Wilochauthor=K.+Kotula-Horowitzauthor=J.+Wojciechowskiauthor=L.+Wojnowskiauthor=A.+Szczeklikauthor=L.+Kalinowski&title=Nitric+oxide+production+and+endothelium-dependent+vasorelaxation+ameliorated+by+N1-methylnicotinamide+in+human+blood+vessels&doi=10.1161%2FHYPERTENSIONAHA.111.183210"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1161%2FHYPERTENSIONAHA.111.183210&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1161%252FHYPERTENSIONAHA.111.183210%26sid%3Dliteratum%253Aachs%26aulast%3DDomagala%26aufirst%3DT.%2BB.%26aulast%3DSzeffler%26aufirst%3DA.%26aulast%3DDobrucki%26aufirst%3DL.%2BW.%26aulast%3DDropinski%26aufirst%3DJ.%26aulast%3DPolanski%26aufirst%3DS.%26aulast%3DLeszczynska-Wiloch%26aufirst%3DM.%26aulast%3DKotula-Horowitz%26aufirst%3DK.%26aulast%3DWojciechowski%26aufirst%3DJ.%26aulast%3DWojnowski%26aufirst%3DL.%26aulast%3DSzczeklik%26aufirst%3DA.%26aulast%3DKalinowski%26aufirst%3DL.%26atitle%3DNitric%2520oxide%2520production%2520and%2520endothelium-dependent%2520vasorelaxation%2520ameliorated%2520by%2520N1-methylnicotinamide%2520in%2520human%2520blood%2520vessels%26jtitle%3DHypertension%26date%3D2012%26volume%3D59%26spage%3D825%26epage%3D832%26doi%3D10.1161%2FHYPERTENSIONAHA.111.183210" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Hong, S. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moreno-Navarrete, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, X. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kikukawa, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tzameli, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prasad, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Asara, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fernandez-Real, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maratos-Flier, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pissios, P.</span><span> </span><span class="NLM_article-title">Nicotinamide N-methyltransferase regulates hepatic nutrient metabolism through Sirt1 protein stabilization</span> <span class="citation_source-journal">Nat. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">887</span><span class="NLM_x">–</span> <span class="NLM_lpage">894</span><span class="refDoi"> DOI: 10.1038/nm.3882</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1038%2Fnm.3882" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=26168293" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1WltLjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=887-894&author=S.+Y.+Hongauthor=J.+M.+Moreno-Navarreteauthor=X.+J.+Weiauthor=Y.+Kikukawaauthor=I.+Tzameliauthor=D.+Prasadauthor=Y.+Leeauthor=J.+M.+Asaraauthor=J.+M.+Fernandez-Realauthor=E.+Maratos-Flierauthor=P.+Pissios&title=Nicotinamide+N-methyltransferase+regulates+hepatic+nutrient+metabolism+through+Sirt1+protein+stabilization&doi=10.1038%2Fnm.3882"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Nicotinamide N-methyltransferase regulates hepatic nutrient metabolism through Sirt1 protein stabilization</span></div><div class="casAuthors">Hong, Shangyu; Moreno-Navarrete, Jose M.; Wei, Xiaojing; Kikukawa, Yusuke; Tzameli, Iphigenia; Prasad, Deepthi; Lee, Yoonjin; Asara, John M.; Fernandez-Real, Jose Manuel; Maratos-Flier, Eleftheria; Pissios, Pavlos</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">887-894</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (Nnmt) methylates nicotinamide, a form of vitamin B3, to produce N1-methylnicotinamide (MNAM).  Nnmt has emerged as a metabolic regulator in adipocytes, but its role in the liver, the tissue with the strongest Nnmt expression, is not known.  In spite of its overall high expression, here we find that hepatic expression of Nnmt is highly variable and correlates with multiple metabolic parameters in mice and humans.  Further, we find that suppression of hepatic Nnmt expression in vivo alters glucose and cholesterol metab. and that the metabolic effects of Nnmt in the liver are mediated by its product MNAM.  Supplementation of high-fat diet with MNAM decreases serum and liver cholesterol and liver triglycerides levels in mice.  Mechanistically, increasing Nnmt expression or MNAM levels stabilizes sirtuin 1 protein, an effect that is required for their metabolic benefits.  In summary, we describe here a novel regulatory pathway for vitamin B3 that could provide a new opportunity for metabolic disease therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaiKWlS3XZlrVg90H21EOLACvtfcHk0lh5hQb1k4YbMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1WltLjO&md5=d3f54ad774f4ea0bf07ce9bb389c0139</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnm.3882&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.3882%26sid%3Dliteratum%253Aachs%26aulast%3DHong%26aufirst%3DS.%2BY.%26aulast%3DMoreno-Navarrete%26aufirst%3DJ.%2BM.%26aulast%3DWei%26aufirst%3DX.%2BJ.%26aulast%3DKikukawa%26aufirst%3DY.%26aulast%3DTzameli%26aufirst%3DI.%26aulast%3DPrasad%26aufirst%3DD.%26aulast%3DLee%26aufirst%3DY.%26aulast%3DAsara%26aufirst%3DJ.%2BM.%26aulast%3DFernandez-Real%26aufirst%3DJ.%2BM.%26aulast%3DMaratos-Flier%26aufirst%3DE.%26aulast%3DPissios%26aufirst%3DP.%26atitle%3DNicotinamide%2520N-methyltransferase%2520regulates%2520hepatic%2520nutrient%2520metabolism%2520through%2520Sirt1%2520protein%2520stabilization%26jtitle%3DNat.%2520Med.%26date%3D2015%26volume%3D21%26spage%3D887%26epage%3D894%26doi%3D10.1038%2Fnm.3882" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Williams, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramsden, D. B.</span><span> </span><span class="NLM_article-title">Autotoxicity, methylation and a road to the prevention of Parkinson’s disease</span> <span class="citation_source-journal">J. Clin. Neurosci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">6</span><span class="NLM_x">–</span> <span class="NLM_lpage">11</span><span class="refDoi"> DOI: 10.1016/j.jocn.2004.10.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1016%2Fj.jocn.2004.10.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=15639403" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BD2MXitFansQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2005&pages=6-11&author=A.+C.+Williamsauthor=D.+B.+Ramsden&title=Autotoxicity%2C+methylation+and+a+road+to+the+prevention+of+Parkinson%E2%80%99s+disease&doi=10.1016%2Fj.jocn.2004.10.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Autotoxicity, methylation and a road to the prevention of Parkinson's disease</span></div><div class="casAuthors">Williams, Adrian C.; Ramsden, David B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Neuroscience</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">6-11</span>CODEN:
                <span class="NLM_cas:coden">JCNUE6</span>;
        ISSN:<span class="NLM_cas:issn">0967-5868</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Xenobiotic enzymes normally protect against toxins but on occasion can convert protoxins into toxins.  N-methylated pyridines (such as the N-methyl-4 phenylpyridinium ion (MPP+)) are well-established dopaminergic toxins.  The enzyme nicotinamide N-methyltransferase (NNMT) can covert otherwise harmless pyridines such as 4-phenylpyridine into MPP+ like compds.  This enzyme has recently been shown to be present in the human brain, which is a necessity for neurotoxicity, as charged compds. such as MPP+ cannot cross the blood brain barrier.  Moreover, it is present in increased concn. in the brain of patients with Parkinson's disease (PD).  This would increase MPP+ like compds. at the same time as decreasing intraneuronal nicotinamide, a neuroprotectant at several levels, thus creating a "multiple hit", as addnl. complex 1 of the mitochondrial complex would also be poisoned and starved of its major substrate, NAD.  Thus, PD may be a disease of autointoxication.  Xenobiotic enzyme inhibitors of NNMT, with or without dietary modification, would be a novel way to attempt primary prevention of PD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkCefdGhG03bVg90H21EOLACvtfcHk0lh5hQb1k4YbMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXitFansQ%253D%253D&md5=0fe92fa41debe0b2f43d15b3c55ea6c7</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.jocn.2004.10.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jocn.2004.10.002%26sid%3Dliteratum%253Aachs%26aulast%3DWilliams%26aufirst%3DA.%2BC.%26aulast%3DRamsden%26aufirst%3DD.%2BB.%26atitle%3DAutotoxicity%252C%2520methylation%2520and%2520a%2520road%2520to%2520the%2520prevention%2520of%2520Parkinson%25E2%2580%2599s%2520disease%26jtitle%3DJ.%2520Clin.%2520Neurosci.%26date%3D2005%26volume%3D12%26spage%3D6%26epage%3D11%26doi%3D10.1016%2Fj.jocn.2004.10.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Parsons, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waring, R. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramsden, D. B.</span><span> </span><span class="NLM_article-title">Expression of nicotinamide N-methyltransferase (EC 2.1.1.1) in the parkinsonian brain</span> <span class="citation_source-journal">J. Neuropathol. Exp. Neurol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">61</span><span class="NLM_x">, </span> <span class="NLM_fpage">111</span><span class="NLM_x">–</span> <span class="NLM_lpage">124</span><span class="refDoi"> DOI: 10.1093/jnen/61.2.111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1093%2Fjnen%2F61.2.111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=11853016" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BD38XitFOku7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2002&pages=111-124&author=R.+B.+Parsonsauthor=M.+L.+Smithauthor=A.+C.+Williamsauthor=R.+H.+Waringauthor=D.+B.+Ramsden&title=Expression+of+nicotinamide+N-methyltransferase+%28EC+2.1.1.1%29+in+the+parkinsonian+brain&doi=10.1093%2Fjnen%2F61.2.111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Expression of nicotinamide N-methyltransferase (E.C. 2.1.1.1) in the Parkinsonian brain</span></div><div class="casAuthors">Parsons, Richard B.; Smith, Marie-Louise; Williams, Adrian C.; Waring, Rosemary H.; Ramsden, David B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Neuropathology and Experimental Neurology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">111-124</span>CODEN:
                <span class="NLM_cas:coden">JNENAD</span>;
        ISSN:<span class="NLM_cas:issn">0022-3069</span>.
    
            (<span class="NLM_cas:orgname">American Association of Neuropathologists, Inc.</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) has been proposed as a link between the environmental and genetic factors of Parkinson disease (PD).  Therefore, we explored the hypothesis that high levels of NNMT expression may predispose to the development of PD.  Regions of high mRNA expression were shown in the spinal cord, medulla, and temporal lobe, with lowest expression in the cerebellum, subthalamic nucleus, and caudate nucleus.  Using 2 NNMT antibodies, the protein was shown to be expressed in multipolar neurons in the temporal lobe, caudate nucleus, and spinal cord, granular neurons of the cerebellum, dopaminergic neurons in the substantia nigra, and in the axons of the third nerve.  Expression of NNMT was compared in PD and non-PD control cerebella and caudate nucleus.  PD tissue exhibited significantly increased levels of NNMT protein and activity.  PD disease duration was inversely correlated with the level of expression in cerebellum.  This is the first demonstration that patients with PD have higher levels of NNMT activity and protein in brain tissue than those without PD and that NNMT expression is assocd. with neurons that degenerate in PD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTQojjaTnox7Vg90H21EOLACvtfcHk0lh5hQb1k4YbMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XitFOku7w%253D&md5=ac3a0e25e3ae442fc9d3e81df1757949</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1093%2Fjnen%2F61.2.111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjnen%252F61.2.111%26sid%3Dliteratum%253Aachs%26aulast%3DParsons%26aufirst%3DR.%2BB.%26aulast%3DSmith%26aufirst%3DM.%2BL.%26aulast%3DWilliams%26aufirst%3DA.%2BC.%26aulast%3DWaring%26aufirst%3DR.%2BH.%26aulast%3DRamsden%26aufirst%3DD.%2BB.%26atitle%3DExpression%2520of%2520nicotinamide%2520N-methyltransferase%2520%2528EC%25202.1.1.1%2529%2520in%2520the%2520parkinsonian%2520brain%26jtitle%3DJ.%2520Neuropathol.%2520Exp.%2520Neurol.%26date%3D2002%26volume%3D61%26spage%3D111%26epage%3D124%26doi%3D10.1093%2Fjnen%2F61.2.111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Kraus, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kong, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banks, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodgers, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pirinen, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pulinilkunnil, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gong, F. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cen, Y. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sauve, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Asara, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peroni, O. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Monia, B. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhanot, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alhonen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Puigserver, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kahn, B. B.</span><span> </span><span class="NLM_article-title">Nicotinamide N-methyltransferase knockdown protects against diet-induced obesity</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">508</span><span class="NLM_x">, </span> <span class="NLM_fpage">258</span><span class="NLM_x">–</span> <span class="NLM_lpage">262</span><span class="refDoi"> DOI: 10.1038/nature13198</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1038%2Fnature13198" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=24717514" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtlWhsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=508&publication_year=2014&pages=258-262&author=D.+Krausauthor=Q.+Yangauthor=D.+Kongauthor=A.+S.+Banksauthor=L.+Zhangauthor=J.+T.+Rodgersauthor=E.+Pirinenauthor=T.+C.+Pulinilkunnilauthor=F.+Y.+Gongauthor=Y.+C.+Wangauthor=Y.+N.+Cenauthor=A.+A.+Sauveauthor=J.+M.+Asaraauthor=O.+D.+Peroniauthor=B.+P.+Moniaauthor=S.+Bhanotauthor=L.+Alhonenauthor=P.+Puigserverauthor=B.+B.+Kahn&title=Nicotinamide+N-methyltransferase+knockdown+protects+against+diet-induced+obesity&doi=10.1038%2Fnature13198"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Nicotinamide N-methyltransferase knockdown protects against diet-induced obesity</span></div><div class="casAuthors">Kraus, Daniel; Yang, Qin; Kong, Dong; Banks, Alexander S.; Zhang, Lin; Rodgers, Joseph T.; Pirinen, Eija; Pulinilkunnil, Thomas C.; Gong, Fengying; Wang, Ya-chin; Cen, Yana; Sauve, Anthony A.; Asara, John M.; Peroni, Odile D.; Monia, Brett P.; Bhanot, Sanjay; Alhonen, Leena; Puigserver, Pere; Kahn, Barbara B.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">508</span>
        (<span class="NLM_cas:issue">7495</span>),
    <span class="NLM_cas:pages">258-262</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">In obesity and type 2 diabetes, Glut4 glucose transporter expression is decreased selectively in adipocytes.  Adipose-specific knockout or overexpression of Glut4 alters systemic insulin sensitivity.  Here we show, using DNA array analyses, that nicotinamide N-methyltransferase (Nnmt) is the most strongly reciprocally regulated gene when comparing gene expression in white adipose tissue (WAT) from adipose-specific Glut4-knockout or adipose-specific Glut4-overexpressing mice with their resp. controls.  NNMT methylates nicotinamide (vitamin B3) using S-adenosylmethionine (SAM) as a Me donor.  Nicotinamide is a precursor of NAD+, an important cofactor linking cellular redox states with energy metab.  SAM provides propylamine for polyamine biosynthesis and donates a Me group for histone methylation.  Polyamine flux including synthesis, catabolism and excretion, is controlled by the rate-limiting enzymes ornithine decarboxylase (ODC) and spermidine-spermine N1-acetyltransferase (SSAT; encoded by Sat1) and by polyamine oxidase (PAO), and has a major role in energy metab.  We report that NNMT expression is increased in WAT and liver of obese and diabetic mice.  Nnmt knockdown in WAT and liver protects against diet-induced obesity by augmenting cellular energy expenditure.  NNMT inhibition increases adipose SAM and NAD+ levels and upregulates ODC and SSAT activity as well as expression, owing to the effects of NNMT on histone H3 lysine 4 methylation in adipose tissue.  Direct evidence for increased polyamine flux resulting from NNMT inhibition includes elevated urinary excretion and adipocyte secretion of diacetylspermine, a product of polyamine metab.  NNMT inhibition in adipocytes increases oxygen consumption in an ODC-, SSAT- and PAO-dependent manner.  Thus, NNMT is a novel regulator of histone methylation, polyamine flux and NAD+-dependent SIRT1 signaling, and is a unique and attractive target for treating obesity and type 2 diabetes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKuWPUvPnIqLVg90H21EOLACvtfcHk0lhtPjjDT7Pnuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtlWhsL8%253D&md5=c37b6137cd4537a0f6f1701a5a212dd6</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnature13198&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13198%26sid%3Dliteratum%253Aachs%26aulast%3DKraus%26aufirst%3DD.%26aulast%3DYang%26aufirst%3DQ.%26aulast%3DKong%26aufirst%3DD.%26aulast%3DBanks%26aufirst%3DA.%2BS.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DRodgers%26aufirst%3DJ.%2BT.%26aulast%3DPirinen%26aufirst%3DE.%26aulast%3DPulinilkunnil%26aufirst%3DT.%2BC.%26aulast%3DGong%26aufirst%3DF.%2BY.%26aulast%3DWang%26aufirst%3DY.%2BC.%26aulast%3DCen%26aufirst%3DY.%2BN.%26aulast%3DSauve%26aufirst%3DA.%2BA.%26aulast%3DAsara%26aufirst%3DJ.%2BM.%26aulast%3DPeroni%26aufirst%3DO.%2BD.%26aulast%3DMonia%26aufirst%3DB.%2BP.%26aulast%3DBhanot%26aufirst%3DS.%26aulast%3DAlhonen%26aufirst%3DL.%26aulast%3DPuigserver%26aufirst%3DP.%26aulast%3DKahn%26aufirst%3DB.%2BB.%26atitle%3DNicotinamide%2520N-methyltransferase%2520knockdown%2520protects%2520against%2520diet-induced%2520obesity%26jtitle%3DNature%26date%3D2014%26volume%3D508%26spage%3D258%26epage%3D262%26doi%3D10.1038%2Fnature13198" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Ulanovskaya, O. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zuhl, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cravatt, B. F.</span><span> </span><span class="NLM_article-title">NNMT promotes epigenetic remodeling in cancer by creating a metabolic methylation sink</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">300</span><span class="NLM_x">–</span> <span class="NLM_lpage">306</span><span class="refDoi"> DOI: 10.1038/nchembio.1204</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1038%2Fnchembio.1204" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=23455543" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtlKnsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=300-306&author=O.+A.+Ulanovskayaauthor=A.+M.+Zuhlauthor=B.+F.+Cravatt&title=NNMT+promotes+epigenetic+remodeling+in+cancer+by+creating+a+metabolic+methylation+sink&doi=10.1038%2Fnchembio.1204"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">NNMT promotes epigenetic remodeling in cancer by creating a metabolic methylation sink</span></div><div class="casAuthors">Ulanovskaya, Olesya A.; Zuhl, Andrea M.; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">300-306</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) is overexpressed in a variety of human cancers, where it contributes to tumorigenesis by a mechanism that is still poorly understood.  Here we show using metabolomics that NNMT impairs the methylation potential of cancer cells by consuming Me units from S-adenosyl methionine to create the stable metabolic product 1-methylnicotinamide.  As a result, NNMT-expressing cancer cells have an altered epigenetic state that includes hypomethylated histones and other cancer-related proteins combined with heightened expression of protumorigenic gene products.  Our findings thus point to a direct mechanistic link between the deregulation of a metabolic enzyme and widespread changes in the methylation landscape of cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsOQPNFxw5kbVg90H21EOLACvtfcHk0lhtPjjDT7Pnuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtlKnsbs%253D&md5=8775d8b27fba0d039b0f6da7244d28a4</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1204&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1204%26sid%3Dliteratum%253Aachs%26aulast%3DUlanovskaya%26aufirst%3DO.%2BA.%26aulast%3DZuhl%26aufirst%3DA.%2BM.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DNNMT%2520promotes%2520epigenetic%2520remodeling%2520in%2520cancer%2520by%2520creating%2520a%2520metabolic%2520methylation%2520sink%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2013%26volume%3D9%26spage%3D300%26epage%3D306%26doi%3D10.1038%2Fnchembio.1204" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Markert, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fuller, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gillespie, G. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bubien, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McLean, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hong, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gullans, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mapstone, T. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benos, D. J.</span><span> </span><span class="NLM_article-title">Differential gene expression profiling in human brain tumors</span> <span class="citation_source-journal">Physiol. Genomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">21</span><span class="NLM_x">–</span> <span class="NLM_lpage">33</span><span class="refDoi"> DOI: 10.1152/physiolgenomics.2001.5.1.21</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1152%2Fphysiolgenomics.2001.5.1.21" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=11161003" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BD3MXks1yksrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2001&pages=21-33&author=J.+M.+Markertauthor=C.+M.+Fullerauthor=G.+Y.+Gillespieauthor=J.+K.+Bubienauthor=L.+A.+McLeanauthor=R.+L.+Hongauthor=K.+Leeauthor=S.+R.+Gullansauthor=T.+B.+Mapstoneauthor=D.+J.+Benos&title=Differential+gene+expression+profiling+in+human+brain+tumors&doi=10.1152%2Fphysiolgenomics.2001.5.1.21"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Differential gene expression profiling in human brain tumors</span></div><div class="casAuthors">Markert, James M.; Fuller, Catherine M.; Gillespie, G. Yancey; Bubien, James K.; McLean, Lee Anne; Hong, Robert L.; Lee, Kailin; Gullans, Steven R.; Mapstone, Timothy B.; Benos, Dale J.</div><div class="citationInfo"><span class="NLM_cas:title">Physiological Genomics [online computer file]</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">21-33</span>CODEN:
                <span class="NLM_cas:coden">PHGEFP</span>;
        ISSN:<span class="NLM_cas:issn">1094-8341</span>.
    
            (<span class="NLM_cas:orgname">American Physiological Society</span>)
        </div><div class="casAbstract">Gene expression profiling of three human temporal lobe brain tissue samples (normal) and four primary glioblastoma multiforme (GBM) tumors using oligonucleotide microarrays was done.  Moreover, confirmation of altered expression was performed by whole cell patch clamp, immunohistochem. staining, and RT-PCR.  Our results identified several ion and solute transport-related genes, such as N-methyl-D-aspartate (NMDA) receptors, α-amino-3-hydroxy-5-methyl-4-isoxazole propionate (AMPA)-2 receptors, GABAA receptor subunits α3, β1, β2, and β3, the glutamate transporter, the glutamate/aspartate transporter II, the potassium channel KV2.1, hKVb3, and the sodium/proton exchanger 1 (NHE-1), that are all downregulated in the tumors compared with the normal tissues.  In contrast, aquaporin-1, possibly aquaporins-3 and -5, and GLUT-3 message appeared upregulated in the tumors.  Our results also confirmed previous work showing that osteopontin, nicotinamide N-methyltransferase, murine double minute 2 (MDM2), and epithelin (granulin) are upregulated in GBMs.  We also demonstrate for the first time that the cytokine and p53 binding protein, macrophage migration inhibitory factor (MIF), appears upregulated in GBMs.  These results indicate that the modulation of ion and solute transport genes and heretofore unsuspected cytokines (i.e., MIF) may have profound implications for brain tumor cell biol. and thus may identify potential useful therapeutic targets in GBMs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoa8TEeUcqEvrVg90H21EOLACvtfcHk0lhtPjjDT7Pnuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXks1yksrw%253D&md5=4846aec92fd8673b96d9cb6a97219912</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1152%2Fphysiolgenomics.2001.5.1.21&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1152%252Fphysiolgenomics.2001.5.1.21%26sid%3Dliteratum%253Aachs%26aulast%3DMarkert%26aufirst%3DJ.%2BM.%26aulast%3DFuller%26aufirst%3DC.%2BM.%26aulast%3DGillespie%26aufirst%3DG.%2BY.%26aulast%3DBubien%26aufirst%3DJ.%2BK.%26aulast%3DMcLean%26aufirst%3DL.%2BA.%26aulast%3DHong%26aufirst%3DR.%2BL.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DGullans%26aufirst%3DS.%2BR.%26aulast%3DMapstone%26aufirst%3DT.%2BB.%26aulast%3DBenos%26aufirst%3DD.%2BJ.%26atitle%3DDifferential%2520gene%2520expression%2520profiling%2520in%2520human%2520brain%2520tumors%26jtitle%3DPhysiol.%2520Genomics%26date%3D2001%26volume%3D5%26spage%3D21%26epage%3D33%26doi%3D10.1152%2Fphysiolgenomics.2001.5.1.21" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Xu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moatamed, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caldwell, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walker, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kraiem, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taki, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brent, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hershman, J. M.</span><span> </span><span class="NLM_article-title">Enhanced expression of nicotinamide N-methyltransferase in human papillary thyroid carcinoma cells</span> <span class="citation_source-journal">J. Clin. Endocrinol. Metab.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">88</span><span class="NLM_x">, </span> <span class="NLM_fpage">4990</span><span class="NLM_x">–</span> <span class="NLM_lpage">4996</span><span class="refDoi"> DOI: 10.1210/jc.2002-021843</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1210%2Fjc.2002-021843" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=14557485" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BD3sXoslemu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=88&publication_year=2003&pages=4990-4996&author=J.+Xuauthor=F.+Moatamedauthor=J.+S.+Caldwellauthor=J.+R.+Walkerauthor=Z.+Kraiemauthor=K.+Takiauthor=G.+A.+Brentauthor=J.+M.+Hershman&title=Enhanced+expression+of+nicotinamide+N-methyltransferase+in+human+papillary+thyroid+carcinoma+cells&doi=10.1210%2Fjc.2002-021843"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Enhanced expression of nicotinamide N-methyltransferase in human papillary thyroid carcinoma cells</span></div><div class="casAuthors">Xu, Jimin; Moatamed, Farhad; Caldwell, Jeremy S.; Walker, John R.; Kraiem, Zaki; Taki, Katsumi; Brent, Gregory A.; Hershman, Jerome M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Endocrinology and Metabolism</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">88</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4990-4996</span>CODEN:
                <span class="NLM_cas:coden">JCEMAZ</span>;
        ISSN:<span class="NLM_cas:issn">0021-972X</span>.
    
            (<span class="NLM_cas:orgname">Endocrine Society</span>)
        </div><div class="casAbstract">To gain an understanding of the mol. pathogenesis of thyroid cancer, we used DNA microarray to study the expression profiles of 10 different human thyroid carcinoma cell lines.  These included papillary lines BHP 2-7, BHP 7-13, BHP 10-3, BHP 18-21, NPA 87, and TPC1; anaplastic lines ARO 81-1 and DRO 90-1; follicular line WRO 82-1; and medullary line HRO 85-1.  Among the genes with increased expression in the cancer cell lines, a gene coding for nicotinamide N-methyltransferase (NNMT) was identified for being highly expressed only in the papillary cell lines.  NNMT catalyzes N-methylation of nicotinamide and other structurally related compds. and is highly expressed in the human liver.  The results were further confirmed by semiquant. RT-PCR and Northern blot anal.  NNMT catalytic activities were detd. in all of the cells described above and in addnl. cell lines.  Significantly higher NNMT enzyme activities were detected in eight of 10 of the papillary lines and three of six of the follicular cell lines tested.  Normal thyroid tissue, thyroid primary cultures, anaplastic cancer cells, and medullary cancer cells showed no or low enzyme activity.  Immunohistochem. staining for NNMT of human thyroid specimens showed strong and abundant cytoplasmic reactions in the sections of papillary carcinomas, and weak or scanty reaction in the normal thyroid tissues.  These results indicate that NNMT is a potential biomarker for papillary thyroid carcinoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6AB9Lmnxdd7Vg90H21EOLACvtfcHk0lhtPjjDT7Pnuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXoslemu7Y%253D&md5=40b3d7c6b5abc7073ffe40791269231f</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1210%2Fjc.2002-021843&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1210%252Fjc.2002-021843%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DMoatamed%26aufirst%3DF.%26aulast%3DCaldwell%26aufirst%3DJ.%2BS.%26aulast%3DWalker%26aufirst%3DJ.%2BR.%26aulast%3DKraiem%26aufirst%3DZ.%26aulast%3DTaki%26aufirst%3DK.%26aulast%3DBrent%26aufirst%3DG.%2BA.%26aulast%3DHershman%26aufirst%3DJ.%2BM.%26atitle%3DEnhanced%2520expression%2520of%2520nicotinamide%2520N-methyltransferase%2520in%2520human%2520papillary%2520thyroid%2520carcinoma%2520cells%26jtitle%3DJ.%2520Clin.%2520Endocrinol.%2520Metab.%26date%3D2003%26volume%3D88%26spage%3D4990%26epage%3D4996%26doi%3D10.1210%2Fjc.2002-021843" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Tomida, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mikami, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takeuchi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nishimura, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Akiyama, H.</span><span> </span><span class="NLM_article-title">Serum levels of nicotinamide N-methyltransferase in patients with lung cancer</span> <span class="citation_source-journal">J. Cancer Res. Clin. Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">135</span><span class="NLM_x">, </span> <span class="NLM_fpage">1223</span><span class="NLM_x">–</span> <span class="NLM_lpage">1229</span><span class="refDoi"> DOI: 10.1007/s00432-009-0563-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1007%2Fs00432-009-0563-y" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2009&pages=1223-1229&author=M.+Tomidaauthor=I.+Mikamiauthor=S.+Takeuchiauthor=H.+Nishimuraauthor=H.+Akiyama&title=Serum+levels+of+nicotinamide+N-methyltransferase+in+patients+with+lung+cancer&doi=10.1007%2Fs00432-009-0563-y"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1007%2Fs00432-009-0563-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00432-009-0563-y%26sid%3Dliteratum%253Aachs%26aulast%3DTomida%26aufirst%3DM.%26aulast%3DMikami%26aufirst%3DI.%26aulast%3DTakeuchi%26aufirst%3DS.%26aulast%3DNishimura%26aufirst%3DH.%26aulast%3DAkiyama%26aufirst%3DH.%26atitle%3DSerum%2520levels%2520of%2520nicotinamide%2520N-methyltransferase%2520in%2520patients%2520with%2520lung%2520cancer%26jtitle%3DJ.%2520Cancer%2520Res.%2520Clin.%2520Oncol.%26date%3D2009%26volume%3D135%26spage%3D1223%26epage%3D1229%26doi%3D10.1007%2Fs00432-009-0563-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Kim, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hong, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, E. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, Y. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roh, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Do, I. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joh, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, D. S.</span><span> </span><span class="NLM_article-title">Expression of nicotinamide N-methyltransferase in hepatocellular carcinoma is associated with poor prognosis</span> <span class="citation_source-journal">J. Exp. Clin. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">20</span><span class="refDoi"> DOI: 10.1186/1756-9966-28-20</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1186%2F1756-9966-28-20" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=19216803" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A280%3ADC%252BD1M3hslejtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2009&pages=20&author=J.+Kimauthor=S.+J.+Hongauthor=E.+K.+Limauthor=Y.+S.+Yuauthor=S.+W.+Kimauthor=J.+H.+Rohauthor=I.+G.+Doauthor=J.+W.+Johauthor=D.+S.+Kim&title=Expression+of+nicotinamide+N-methyltransferase+in+hepatocellular+carcinoma+is+associated+with+poor+prognosis&doi=10.1186%2F1756-9966-28-20"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Expression of nicotinamide N-methyltransferase in hepatocellular carcinoma is associated with poor prognosis</span></div><div class="casAuthors">Kim Jongmin; Hong Seok Joo; Lim Eun Kyung; Yu Yun-Suk; Kim Seung Whan; Roh Ji Hyeon; Do In-Gu; Joh Jae-Won; Kim Dae Shick</div><div class="citationInfo"><span class="NLM_cas:title">Journal of experimental & clinical cancer research : CR</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">20</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Hepatocellular carcinoma (HCC) is the most common tumor in the adult liver, with high relapse and mortality rates despite diverse treatment modalities.  In this study, nicotinamide N-methyltransferase (NNMT), a key enzyme in drug metabolism, was investigated as a potential prognostic factor.  METHODS:  Frozen tumors and non-cancerous surrounding tissues from 120 patients with primary HCC were studied.  Expressions of NNMT and internal control genes were measured by real-time reverse-transcription PCR (RT-PCR).  The relationship of NNMT mRNA level with clinicopathologic parameters and clinical outcome was evaluated.  RESULTS:  NNMT mRNA level is markedly reduced in HCCs compared to non-cancerous surrounding tissues (P < 0.0001), and NNMT expression in tumors was significantly correlated with tumor stage (P = 0.010).  Moreover, stratification of patients based on tumor NNMT mRNA levels revealed that the patients who expressed higher NNMT mRNA levels tended to have a shorter overall survival (OS) time (P = 0.053) and a significantly shorter disease-free survival (DFS) time (P = 0.016).  Both NNMT expression (P = 0.0096) and tumor stage (P = 0.0017) were found to be significant prognostic factors for DFS in a multivariate analysis.  CONCLUSION:  The results of this study indicated that NNMT gene expression is associated with tumor stage and DFS time in HCC cases.  Because of the broad substrate specificity of NNMT, which could alter the efficacy and adverse effects of chemotherapy, NNMT merits further investigation regarding its role as a prognostic factor with a larger cohort of HCC patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSTRP98CJkGv1QKX3ooHxWjfW6udTcc2eakVdgJWA7b-7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1M3hslejtw%253D%253D&md5=95063b5fa31780728476e4142f89603c</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1186%2F1756-9966-28-20&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1756-9966-28-20%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DHong%26aufirst%3DS.%2BJ.%26aulast%3DLim%26aufirst%3DE.%2BK.%26aulast%3DYu%26aufirst%3DY.%2BS.%26aulast%3DKim%26aufirst%3DS.%2BW.%26aulast%3DRoh%26aufirst%3DJ.%2BH.%26aulast%3DDo%26aufirst%3DI.%2BG.%26aulast%3DJoh%26aufirst%3DJ.%2BW.%26aulast%3DKim%26aufirst%3DD.%2BS.%26atitle%3DExpression%2520of%2520nicotinamide%2520N-methyltransferase%2520in%2520hepatocellular%2520carcinoma%2520is%2520associated%2520with%2520poor%2520prognosis%26jtitle%3DJ.%2520Exp.%2520Clin.%2520Cancer%2520Res.%26date%3D2009%26volume%3D28%26spage%3D20%26doi%3D10.1186%2F1756-9966-28-20" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Yao, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tabuchi, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nagashima, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baba, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakaigawa, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ishiguro, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamada, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inayama, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kishida, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hattori, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamada-Okabe, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kubota, Y.</span><span> </span><span class="NLM_article-title">Gene expression analysis of renal carcinoma: adipose differentiation-related protein as a potential diagnostic and prognostic biomarker for clear-cell renal carcinoma</span> <span class="citation_source-journal">J. Pathol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">205</span><span class="NLM_x">, </span> <span class="NLM_fpage">377</span><span class="NLM_x">–</span> <span class="NLM_lpage">387</span><span class="refDoi"> DOI: 10.1002/path.1693</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1002%2Fpath.1693" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=15682440" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BD2MXit1Oruro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=205&publication_year=2005&pages=377-387&author=M.+Yaoauthor=H.+Tabuchiauthor=Y.+Nagashimaauthor=M.+Babaauthor=N.+Nakaigawaauthor=H.+Ishiguroauthor=K.+Hamadaauthor=Y.+Inayamaauthor=T.+Kishidaauthor=K.+Hattoriauthor=H.+Yamada-Okabeauthor=Y.+Kubota&title=Gene+expression+analysis+of+renal+carcinoma%3A+adipose+differentiation-related+protein+as+a+potential+diagnostic+and+prognostic+biomarker+for+clear-cell+renal+carcinoma&doi=10.1002%2Fpath.1693"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Gene expression analysis of renal carcinoma: adipose differentiation-related protein as a potential diagnostic and prognostic biomarker for clear-cell renal carcinoma</span></div><div class="casAuthors">Yao, Masahiro; Tabuchi, Hisahiro; Nagashima, Yoji; Baba, Masaya; Nakaigawa, Noburo; Ishiguro, Hitoshi; Hamada, Kenji; Inayama, Yoshiaki; Kishida, Takeshi; Hattori, Keiko; Yamada-Okabe, Hisafumi; Kubota, Yoshinobu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pathology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">205</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">377-387</span>CODEN:
                <span class="NLM_cas:coden">JPTLAS</span>;
        ISSN:<span class="NLM_cas:issn">0022-3417</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Ltd.</span>)
        </div><div class="casAbstract">The gene expression profiles of 33 renal cell carcinomas (RCCs) and nine normal kidney samples were examd. using high-d. oligonucleotide microarrays in an attempt to identify biomol. markers for the diagnosis of tumor subtypes and also for prediction of prognosis.  Hierarchical clustering demonstrated that clear-cell RCC, chromophobe RCC, and normal kidney tissue showed distinctive gene expression profiles.  The mean expression levels of 149 of 12,500 genes were more than three times higher in clear-cell RCC than in chromophobe RCC and normal kidney tissue.  Among the genes whose expression was upregulated in clear-cell RCC, adipose differentiation-related protein (ADFP) and nicotinamide N-methyltransferase (NNMT) were selected for further anal.  Consistent with the results of the microarray, increased levels of ADFP and NNMT mRNA were found more frequently in clear-cell RCCs than in other non-clear-cell tumor subtypes using real-time quant. PCR.  Immunohistochem. for ADFP showed strong and unique tumor cell staining patterns in the majority of clear-cell RCCs.  More importantly, patients bearing tumors with higher AFDP mRNA levels showed significantly better survival in both univariate and multivariate analyses.  ADFP is a lipid storage droplet-assocd. protein and its transcription is considered to be regulated by the von Hippel-Lindau/hypoxia-inducible factor pathway.  It is known that clear-cell RCC contains abundant lipids and cholesterols.  Thus it is likely that sustained upregulation of ADFP following VHL inactivation is involved in the morphol. appearance of clear-cell RCC.  Moreover ADFP expression status may provide useful prognostic information as a biomol. marker in patients with clear-cell RCC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZJ-PjqY6hHLVg90H21EOLACvtfcHk0lhVL7id0tGETQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXit1Oruro%253D&md5=30d95e39fda0b3d35b1dcb1aece0a12e</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1002%2Fpath.1693&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpath.1693%26sid%3Dliteratum%253Aachs%26aulast%3DYao%26aufirst%3DM.%26aulast%3DTabuchi%26aufirst%3DH.%26aulast%3DNagashima%26aufirst%3DY.%26aulast%3DBaba%26aufirst%3DM.%26aulast%3DNakaigawa%26aufirst%3DN.%26aulast%3DIshiguro%26aufirst%3DH.%26aulast%3DHamada%26aufirst%3DK.%26aulast%3DInayama%26aufirst%3DY.%26aulast%3DKishida%26aufirst%3DT.%26aulast%3DHattori%26aufirst%3DK.%26aulast%3DYamada-Okabe%26aufirst%3DH.%26aulast%3DKubota%26aufirst%3DY.%26atitle%3DGene%2520expression%2520analysis%2520of%2520renal%2520carcinoma%253A%2520adipose%2520differentiation-related%2520protein%2520as%2520a%2520potential%2520diagnostic%2520and%2520prognostic%2520biomarker%2520for%2520clear-cell%2520renal%2520carcinoma%26jtitle%3DJ.%2520Pathol.%26date%3D2005%26volume%3D205%26spage%3D377%26epage%3D387%26doi%3D10.1002%2Fpath.1693" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Tang, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, W. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lai, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, J. Y.</span><span> </span><span class="NLM_article-title">Nicotinamide N-methyltransferase induces cellular invasion through activating matrix metalloproteinase-2 expression in clear cell renal cell carcinoma cells</span> <span class="citation_source-journal">Carcinogenesis</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">138</span><span class="NLM_x">–</span> <span class="NLM_lpage">145</span><span class="refDoi"> DOI: 10.1093/carcin/bgq225</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1093%2Fcarcin%2Fbgq225" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=21045016" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVSktrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2011&pages=138-145&author=S.+W.+Tangauthor=T.+C.+Yangauthor=W.+C.+Linauthor=W.+H.+Changauthor=C.+C.+Wangauthor=M.+K.+Laiauthor=J.+Y.+Lin&title=Nicotinamide+N-methyltransferase+induces+cellular+invasion+through+activating+matrix+metalloproteinase-2+expression+in+clear+cell+renal+cell+carcinoma+cells&doi=10.1093%2Fcarcin%2Fbgq225"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Nicotinamide N-methyltransferase induces cellular invasion through activating matrix metalloproteinase-2 expression in clear cell renal cell carcinoma cells</span></div><div class="casAuthors">Tang, Sai-Wen; Yang, Tsung-Cheng; Lin, Wei-Chou; Chang, Wen-Hsin; Wang, Chung-Chieh; Lai, Ming-Kuen; Lin, Jung-Yaw</div><div class="citationInfo"><span class="NLM_cas:title">Carcinogenesis</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">138-145</span>CODEN:
                <span class="NLM_cas:coden">CRNGDP</span>;
        ISSN:<span class="NLM_cas:issn">0143-3334</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) was recently identified as one clear cell renal cell carcinoma (ccRCC)-assocd. gene by analyzing full-length complementary DNA-enriched libraries of ccRCC tissues.  The aim of this study is to investigate the potential role of NNMT in cellular invasion.  A strong NNMT expression is accompanied with a high invasive activity in ccRCC cell lines, and small interfering RNA-mediated NNMT knockdown effectively suppressed the invasive capacity of ccRCC cells, whereas NNMT overexpression markedly enhanced that of human embryonic kidney 293 (HEK293) cells.  A pos. correlation between the expression of NNMT and matrix metallopeptidase (MMP)-2 was found in ccRCC cell lines and clin. tissues.  The treatment of blocking antibody or inhibitor specific to MMP-2 significantly suppressed NNMT-dependent cellular invasion in HEK293 cells.  Furthermore, SP-1-binding region of MMP-2 promoter was found to be essential in NNMT-induced MMP-2 expression.  The specific inhibitors of PI3K/Akt signaling markedly decreased the binding of SP1 to MMP-2 promoter as shown by chromatin immunopptn. assay.  We also demonstrated that PI3K/Akt pathway plays a role in NNMT-dependent cellular invasion and MMP-2 activation.  Moreover, short hairpin RNA-mediated knockdown of NNMT expression efficiently inhibited the growth and metastasis of ccRCC cells in non-obese diabetic severe combined immunodeficiency mice.  Taken together, the present study suggests that NNMT has a crucial role in cellular invasion via activating PI3K/Akt/SP1/MMP-2 pathway in ccRCC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoN72MlY14gnbVg90H21EOLACvtfcHk0lhVL7id0tGETQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVSktrc%253D&md5=c0a4aedaa1f85ffb7ee0eed126940799</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1093%2Fcarcin%2Fbgq225&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fcarcin%252Fbgq225%26sid%3Dliteratum%253Aachs%26aulast%3DTang%26aufirst%3DS.%2BW.%26aulast%3DYang%26aufirst%3DT.%2BC.%26aulast%3DLin%26aufirst%3DW.%2BC.%26aulast%3DChang%26aufirst%3DW.%2BH.%26aulast%3DWang%26aufirst%3DC.%2BC.%26aulast%3DLai%26aufirst%3DM.%2BK.%26aulast%3DLin%26aufirst%3DJ.%2BY.%26atitle%3DNicotinamide%2520N-methyltransferase%2520induces%2520cellular%2520invasion%2520through%2520activating%2520matrix%2520metalloproteinase-2%2520expression%2520in%2520clear%2520cell%2520renal%2520cell%2520carcinoma%2520cells%26jtitle%3DCarcinogenesis%26date%3D2011%26volume%3D32%26spage%3D138%26epage%3D145%26doi%3D10.1093%2Fcarcin%2Fbgq225" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Jang, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cho, H. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, Y. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ha, W. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, H. W.</span><span> </span><span class="NLM_article-title">The differential proteome profile of stomach cancer: Identification of the biomarker candidates</span> <span class="citation_source-journal">Oncol. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">491</span><span class="NLM_x">–</span> <span class="NLM_lpage">499</span><span class="refDoi"> DOI: 10.3727/0965040042380441</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.3727%2F0965040042380441" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=15559763" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVyktr3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2004&pages=491-499&author=J.+S.+Jangauthor=H.+Y.+Choauthor=Y.+J.+Leeauthor=W.+S.+Haauthor=H.+W.+Kim&title=The+differential+proteome+profile+of+stomach+cancer%3A+Identification+of+the+biomarker+candidates&doi=10.3727%2F0965040042380441"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">The differential proteome profile of stomach cancer: identification of the biomarker candidates</span></div><div class="casAuthors">Jang, Joung S.; Cho, Hee Y.; Lee, Young J.; Ha, Woo S.; Kim, Hwal W.</div><div class="citationInfo"><span class="NLM_cas:title">Oncology Research</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">491-499</span>CODEN:
                <span class="NLM_cas:coden">ONREE8</span>;
        ISSN:<span class="NLM_cas:issn">0965-0407</span>.
    
            (<span class="NLM_cas:orgname">Cognizant Communication Corp.</span>)
        </div><div class="casAbstract">By comparative proteome anal. we searched for characteristic alterations of human stomach adenocarcinoma tissue and paired surrounding normal tissue.  Selected differential protein spots were identified with peptide mass fingerprinting based on matrix-assisted laser desorption/ionization time-of-flight mass spectrometry (MALDI-TOF) and database searching.  We identified protein alterations in 18 stomach cancer tissues compared with normal controls, comprising elevated levels of eight proteins, including 14-3-3 zeta, calcyclin, keratin, apolipoprotein A-1 precursor, proteasome activator complex subunit, nucleoside diphosphate kinase, nicotinamide N-methyltransferase, and pyridoxal kinase.  Five proteins (CA11, prohibitin, peroxiredoxin 4, serum amyloid P component, and NADH-ubiquinone oxidoreductase 23 kDa subunit) were decreased.  These data are valuable for identification of differentially expressed proteins involved in stomach cancer carcinogenesis, providing biomarker candidates to develop diagnostic and therapeutic tools.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLMiz7Zm0YeLVg90H21EOLACvtfcHk0lhKa93JBoGz2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVyktr3K&md5=5e691decbf5b8563f91f710d0fd840b4</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.3727%2F0965040042380441&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3727%252F0965040042380441%26sid%3Dliteratum%253Aachs%26aulast%3DJang%26aufirst%3DJ.%2BS.%26aulast%3DCho%26aufirst%3DH.%2BY.%26aulast%3DLee%26aufirst%3DY.%2BJ.%26aulast%3DHa%26aufirst%3DW.%2BS.%26aulast%3DKim%26aufirst%3DH.%2BW.%26atitle%3DThe%2520differential%2520proteome%2520profile%2520of%2520stomach%2520cancer%253A%2520Identification%2520of%2520the%2520biomarker%2520candidates%26jtitle%3DOncol.%2520Res.%26date%3D2004%26volume%3D14%26spage%3D491%26epage%3D499%26doi%3D10.3727%2F0965040042380441" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Lim, B. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cho, B. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, Y. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, S. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, C. W.</span><span> </span><span class="NLM_article-title">Overexpression of nicotinamide N-methyltransferase in gastric cancer tissues and its potential post-translational modification</span> <span class="citation_source-journal">Exp. Mol. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">455</span><span class="NLM_x">–</span> <span class="NLM_lpage">465</span><span class="refDoi"> DOI: 10.1038/emm.2006.54</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1038%2Femm.2006.54" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=17079861" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtF2qs7fK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2006&pages=455-465&author=B.+H.+Limauthor=B.+I.+Choauthor=Y.+N.+Kimauthor=J.+W.+Kimauthor=S.+T.+Parkauthor=C.+W.+Lee&title=Overexpression+of+nicotinamide+N-methyltransferase+in+gastric+cancer+tissues+and+its+potential+post-translational+modification&doi=10.1038%2Femm.2006.54"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Overexpression of nicotinamide N-methyltransferase in gastric cancer tissues and its potential post-translational modification</span></div><div class="casAuthors">Lim, Bo-Hyun; Cho, Bok-Im; Kim, Yu Na; Kim, Jae Won; Park, Soon-Tae; Lee, Chang-Won</div><div class="citationInfo"><span class="NLM_cas:title">Experimental and Molecular Medicine</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">455-465</span>CODEN:
                <span class="NLM_cas:coden">EMMEF3</span>;
        ISSN:<span class="NLM_cas:issn">1226-3613</span>.
    
            (<span class="NLM_cas:orgname">Korean Society of Medical Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Gastric cancer is one of the most common cancers worldwide.  The purpose of this study was to find out potential markers for gastric cancer.  Tumor and normal tissues from 152 gastric cancer cases were analyzed by two-dimensional gel electrophoresis (2-DE).  The images of silver stained gels were analyzed and statistical anal. of spot intensities revealed that spot 4262 showed higher expression (5.7-fold increase) in cancer tissues than in normal tissues (P < 0.001).  It was identified by peptide mass fingerprinting as nicotinamide N-methyltransferase (NNMT).  A monoclonal antibody with a detection limit down to 10 ng was produced against NNMT in mouse.  Using the prepd. monoclonal antibody, western blot anal. of NNMT was performed for gastric tissues from 15 gastric cancer patients and two gastric ulcer patients.  The results corroborated those of 2-DE expts.  A single spot was detected in gastric ulcer tissues while four to five spots were detected in gastric cancer tissues.  In cancer tissues, two addnl. spots of acidic and basic form were mainly detected on 2-DE gels.  This suggests that NNMT receives a post-translational modification in cancer-specific manner.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTyS3vucBqULVg90H21EOLACvtfcHk0lhKa93JBoGz2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtF2qs7fK&md5=e88bc3183a7968b6b23f89c715b387ba</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Femm.2006.54&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Femm.2006.54%26sid%3Dliteratum%253Aachs%26aulast%3DLim%26aufirst%3DB.%2BH.%26aulast%3DCho%26aufirst%3DB.%2BI.%26aulast%3DKim%26aufirst%3DY.%2BN.%26aulast%3DKim%26aufirst%3DJ.%2BW.%26aulast%3DPark%26aufirst%3DS.%2BT.%26aulast%3DLee%26aufirst%3DC.%2BW.%26atitle%3DOverexpression%2520of%2520nicotinamide%2520N-methyltransferase%2520in%2520gastric%2520cancer%2520tissues%2520and%2520its%2520potential%2520post-translational%2520modification%26jtitle%3DExp.%2520Mol.%2520Med.%26date%3D2006%26volume%3D38%26spage%3D455%26epage%3D465%26doi%3D10.1038%2Femm.2006.54" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Wu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siadaty, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berens, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hampton, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Theodorescu, D.</span><span> </span><span class="NLM_article-title">Overlapping gene expression profiles of cell migration and tumor invasion in human bladder cancer identify metallothionein 1E and nicotinamide N-methyltransferase as novel regulators of cell migration</span> <span class="citation_source-journal">Oncogene</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">6679</span><span class="NLM_x">–</span> <span class="NLM_lpage">6689</span><span class="refDoi"> DOI: 10.1038/onc.2008.264</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1038%2Fonc.2008.264" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2008&pages=6679-6689&author=Y.+Wuauthor=M.+S.+Siadatyauthor=M.+E.+Berensauthor=G.+M.+Hamptonauthor=D.+Theodorescu&title=Overlapping+gene+expression+profiles+of+cell+migration+and+tumor+invasion+in+human+bladder+cancer+identify+metallothionein+1E+and+nicotinamide+N-methyltransferase+as+novel+regulators+of+cell+migration&doi=10.1038%2Fonc.2008.264"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fonc.2008.264&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2008.264%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DY.%26aulast%3DSiadaty%26aufirst%3DM.%2BS.%26aulast%3DBerens%26aufirst%3DM.%2BE.%26aulast%3DHampton%26aufirst%3DG.%2BM.%26aulast%3DTheodorescu%26aufirst%3DD.%26atitle%3DOverlapping%2520gene%2520expression%2520profiles%2520of%2520cell%2520migration%2520and%2520tumor%2520invasion%2520in%2520human%2520bladder%2520cancer%2520identify%2520metallothionein%25201E%2520and%2520nicotinamide%2520N-methyltransferase%2520as%2520novel%2520regulators%2520of%2520cell%2520migration%26jtitle%3DOncogene%26date%3D2008%26volume%3D27%26spage%3D6679%26epage%3D6689%26doi%3D10.1038%2Fonc.2008.264" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Yu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bi, H. C.</span><span> </span><span class="NLM_article-title">Effects of nicotinamide N-methyltransferase on PANC-1 cells proliferation, metastatic potential and survival under metabolic stress</span> <span class="citation_source-journal">Cell. Physiol. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">710</span><span class="NLM_x">–</span> <span class="NLM_lpage">721</span><span class="refDoi"> DOI: 10.1159/000369731</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1159%2F000369731" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=25592232" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvV2msbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2015&pages=710-721&author=T.+Yuauthor=Y.+T.+Wangauthor=P.+Chenauthor=Y.+H.+Liauthor=Y.+X.+Chenauthor=H.+Zengauthor=A.+M.+Yuauthor=M.+Huangauthor=H.+C.+Bi&title=Effects+of+nicotinamide+N-methyltransferase+on+PANC-1+cells+proliferation%2C+metastatic+potential+and+survival+under+metabolic+stress&doi=10.1159%2F000369731"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Effects of Nicotinamide N-Methyltransferase on PANC-1 Cells Proliferation, Metastatic Potential and Survival Under Metabolic Stress</span></div><div class="casAuthors">Yu, Tao; Wang, Yong-Tao; Chen, Pan; Li, Yu-Hua; Chen, Yi-Xin; Zeng, Hang; Yu, Ai-Ming; Huang, Min; Bi, Hui-Chang</div><div class="citationInfo"><span class="NLM_cas:title">Cellular Physiology and Biochemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">710-721</span>CODEN:
                <span class="NLM_cas:coden">CEPBEW</span>;
        ISSN:<span class="NLM_cas:issn">1015-8987</span>.
    
            (<span class="NLM_cas:orgname">S. Karger AG</span>)
        </div><div class="casAbstract">Background: Aberrant expression of Nicotinamide N-methyltransferase (NNMT) has been reported in pancreatic cancer.  However, the role of NNMT in pancreatic cancer development remains elusive.  Therefore, the present study was to investigate the impact of NNMT on pancreatic cancer cell proliferation, metastatic potential and survival under metabolic stress.  Methods: Pancreatic cancer cell line PANC-1 was transfected with NNMT expression plasmid or small interfering RNA of NNMT to overexpress or knockdown intracellular NNMT expression, resp.  Rate of cell proliferation was monitored.  Transwell migration and matrigel invasion assays were conducted to assess cell migration and invasion capacity.  Resistance to glucose deprivation, sensitivity to glycolytic inhibition, mitochondrial inhibtion and resistance to rapamycin were examd. to evaluate cell survival under metabolic stress.  Results: NNMT silencing markedly reduced cell proliferation, whereas NNMT overexpression promoted cell growth moderately.  Knocking down NNMT also significantly suppressed the migration and invasion capacities of PANC-1 cells.  Conversely, NNMT upregulation enhanced cell migration and invasion capacities.  In addn., NNMT knockdown cells were much less resistant to glucose deprivation and rapamycin as well as glycolytic inhibitor 2-deoxyglucose whereas NNMT-expressing cells showed opposite effects although the effects were not so striking.  Conclusions: These data sugguest that NNMT plays an important role in PANC-1 cell proliferation, metastatic potential and survival under metabolic stress.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqcIoH5JmFE37Vg90H21EOLACvtfcHk0ljOMdOD-zT97A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvV2msbs%253D&md5=96916089c2d252b0409157e8a7efe61b</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1159%2F000369731&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1159%252F000369731%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DY.%2BT.%26aulast%3DChen%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DY.%2BH.%26aulast%3DChen%26aufirst%3DY.%2BX.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DYu%26aufirst%3DA.%2BM.%26aulast%3DHuang%26aufirst%3DM.%26aulast%3DBi%26aufirst%3DH.%2BC.%26atitle%3DEffects%2520of%2520nicotinamide%2520N-methyltransferase%2520on%2520PANC-1%2520cells%2520proliferation%252C%2520metastatic%2520potential%2520and%2520survival%2520under%2520metabolic%2520stress%26jtitle%3DCell.%2520Physiol.%2520Biochem.%26date%3D2015%26volume%3D35%26spage%3D710%26epage%3D721%26doi%3D10.1159%2F000369731" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Sartini, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santarelli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rossi, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goteri, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rubini, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciavarella, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lo Muzio, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emanuelli, M.</span><span> </span><span class="NLM_article-title">Nicotinamide N-methyltransferase upregulation inversely correlates with lymph node metastasis in oral squamous cell carcinoma</span> <span class="citation_source-journal">Mol. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">415</span><span class="NLM_x">–</span> <span class="NLM_lpage">421</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.2119%2F2007-00035.Sartini" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=17622326" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVynsrjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2007&pages=415-421&author=D.+Sartiniauthor=A.+Santarelliauthor=V.+Rossiauthor=G.+Goteriauthor=C.+Rubiniauthor=D.+Ciavarellaauthor=L.+Lo+Muzioauthor=M.+Emanuelli&title=Nicotinamide+N-methyltransferase+upregulation+inversely+correlates+with+lymph+node+metastasis+in+oral+squamous+cell+carcinoma"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Nicotinamide N-methyltransferase upregulation inversely correlates with lymph node metastasis in oral squamous cell carcinoma</span></div><div class="casAuthors">Sartini, Davide; Santarelli, Andrea; Rossi, Valentina; Goteri, Gaia; Rubini, Corrado; Ciavarella, Domenico; Lo Muzio, Lorenzo; Emanuelli, Monica</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Medicine (Manhasset, NY, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">7-8</span>),
    <span class="NLM_cas:pages">415-421</span>CODEN:
                <span class="NLM_cas:coden">MOMEF3</span>;
        ISSN:<span class="NLM_cas:issn">1076-1551</span>.
    
            (<span class="NLM_cas:orgname">Feinstein Institute for Medical Research</span>)
        </div><div class="casAbstract">We investigated expression levels of Nicotinamide N-Methyltransferase (NNMT), an enzyme involved in the biotransformation of many drugs and xenobiotic compds., in oral squamous cell carcinoma (OSCC).  Measurements were performed by semi-quant. RT-PCR and quant. real-time PCR in tumor and matched adjacent healthy tissue.  Interestingly, NNMT was up-regulated in most of the favorable OSCCs, while no marked NNMT expression alterations between tumor and normal mucosa were detected in most of the unfavorable OSCCs.  Western blot and immunohistochem. analyses also were performed and the relationship between tumor characteristics and NNMT levels in OSCC were studied to evaluate the effectiveness of NNMT as a prognostic marker in the squamous cell carcinoma of the oral cavity.  In summary, the present study suggests that NNMT may have potential as a biomarker and a therapeutic target for OSCC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrg_Vx2wEahmbVg90H21EOLACvtfcHk0ljOMdOD-zT97A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVynsrjL&md5=9b78d16ea787bc3d544faffed81822aa</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.2119%2F2007-00035.Sartini&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2119%252F2007-00035.Sartini%26sid%3Dliteratum%253Aachs%26aulast%3DSartini%26aufirst%3DD.%26aulast%3DSantarelli%26aufirst%3DA.%26aulast%3DRossi%26aufirst%3DV.%26aulast%3DGoteri%26aufirst%3DG.%26aulast%3DRubini%26aufirst%3DC.%26aulast%3DCiavarella%26aufirst%3DD.%26aulast%3DLo%2BMuzio%26aufirst%3DL.%26aulast%3DEmanuelli%26aufirst%3DM.%26atitle%3DNicotinamide%2520N-methyltransferase%2520upregulation%2520inversely%2520correlates%2520with%2520lymph%2520node%2520metastasis%2520in%2520oral%2520squamous%2520cell%2520carcinoma%26jtitle%3DMol.%2520Med.%26date%3D2007%26volume%3D13%26spage%3D415%26epage%3D421" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Pozzi, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sartini, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morganti, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giuliante, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Di Ruscio, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santarelli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rocchetti, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rubini, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tomasetti, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giannatempo, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orlando, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Provinciali, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lo Muzio, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emanuelli, M.</span><span> </span><span class="NLM_article-title">RNA-mediated gene silencing of nicotinamide N-methyltransferase is associated with decreased tumorigenicity in human oral carcinoma cells</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">e71272</span><span class="refDoi"> DOI: 10.1371/journal.pone.0071272</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1371%2Fjournal.pone.0071272" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=e71272&author=V.+Pozziauthor=D.+Sartiniauthor=S.+Morgantiauthor=R.+Giulianteauthor=G.+Di+Ruscioauthor=A.+Santarelliauthor=R.+Rocchettiauthor=C.+Rubiniauthor=M.+Tomasettiauthor=G.+Giannatempoauthor=F.+Orlandoauthor=M.+Provincialiauthor=L.+Lo+Muzioauthor=M.+Emanuelli&title=RNA-mediated+gene+silencing+of+nicotinamide+N-methyltransferase+is+associated+with+decreased+tumorigenicity+in+human+oral+carcinoma+cells&doi=10.1371%2Fjournal.pone.0071272"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0071272&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0071272%26sid%3Dliteratum%253Aachs%26aulast%3DPozzi%26aufirst%3DV.%26aulast%3DSartini%26aufirst%3DD.%26aulast%3DMorganti%26aufirst%3DS.%26aulast%3DGiuliante%26aufirst%3DR.%26aulast%3DDi%2BRuscio%26aufirst%3DG.%26aulast%3DSantarelli%26aufirst%3DA.%26aulast%3DRocchetti%26aufirst%3DR.%26aulast%3DRubini%26aufirst%3DC.%26aulast%3DTomasetti%26aufirst%3DM.%26aulast%3DGiannatempo%26aufirst%3DG.%26aulast%3DOrlando%26aufirst%3DF.%26aulast%3DProvinciali%26aufirst%3DM.%26aulast%3DLo%2BMuzio%26aufirst%3DL.%26aulast%3DEmanuelli%26aufirst%3DM.%26atitle%3DRNA-mediated%2520gene%2520silencing%2520of%2520nicotinamide%2520N-methyltransferase%2520is%2520associated%2520with%2520decreased%2520tumorigenicity%2520in%2520human%2520oral%2520carcinoma%2520cells%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26spage%3De71272%26doi%3D10.1371%2Fjournal.pone.0071272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Roessler, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rollinger, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palme, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hagmann, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berndt, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engel, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneidinger, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pfeffer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andres, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karl, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bodenmuller, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruschoff, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henkel, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rohr, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rossol, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosch, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Langen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zolg, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tacke, M.</span><span> </span><span class="NLM_article-title">Identification of nicotinamide N-methyltransferase as a novel serum tumor marker for colorectal cancer</span> <span class="citation_source-journal">Clin. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">6550</span><span class="NLM_x">–</span> <span class="NLM_lpage">6557</span><span class="refDoi"> DOI: 10.1158/1078-0432.CCR-05-0983</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1158%2F1078-0432.CCR-05-0983" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=16166432" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVShsb%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2005&pages=6550-6557&author=M.+Roesslerauthor=W.+Rollingerauthor=S.+Palmeauthor=M.+L.+Hagmannauthor=P.+Berndtauthor=A.+M.+Engelauthor=B.+Schneidingerauthor=M.+Pfefferauthor=H.+Andresauthor=J.+Karlauthor=H.+Bodenmullerauthor=J.+Ruschoffauthor=T.+Henkelauthor=G.+Rohrauthor=S.+Rossolauthor=W.+Roschauthor=H.+Langenauthor=W.+Zolgauthor=M.+Tacke&title=Identification+of+nicotinamide+N-methyltransferase+as+a+novel+serum+tumor+marker+for+colorectal+cancer&doi=10.1158%2F1078-0432.CCR-05-0983"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Nicotinamide N-Methyltransferase as a Novel Serum Tumor Marker for Colorectal Cancer</span></div><div class="casAuthors">Roessler, Markus; Rollinger, Wolfgang; Palme, Stefan; Hagmann, Marie-Luise; Berndt, Peter; Engel, Alfred M.; Schneidinger, Bernd; Pfeffer, Michael; Andres, Herbert; Karl, Johann; Bodenmueller, Heinz; Rueschoff, Josef; Henkel, Thomas; Rohr, Gerhard; Rossol, Siegbert; Roesch, Wolfgang; Langen, Hanno; Zolg, Werner; Tacke, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">6550-6557</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: The goal of this study was to identify and validate novel serum markers of human colorectal cancer as potential candidates for noninvasive detection of early colorectal neoplasm.  Exptl. Design: Employing two-dimensional gel electrophoresis and mass spectrometry, we analyzed 16 matched colorectal cancer and adjacent normal tissue samples.  Proteins found to be elevated in cancer tissue were further validated by generating antibodies which were used for immunoblotting of tissue samples and for the development of highly sensitive immunoassays for assessment of serum samples.  Results: In total, 735 different proteins were identified in colon tissue.  Strong elevation in colorectal cancer for five proteins was confirmed by immunoblot anal.: transforming growth factor-β induced protein ig-h3 (βIG-H3), nicotinamide N-methyltransferase (NNMT), nucleoside diphosphate kinase A (nm23-H1), purine nucleoside phosphorylase (PNPH), and mannose-6-phosphate receptor binding protein 1 (M6P1).  Elevated levels of NNMT, which is not predicted to be secreted but is known as a cytoplasmic protein, were found in serum from patients with colorectal cancer.  Employing a receiver-operating characteristic curve based on the measurement of 109 patients with colorectal cancer and 317 healthy controls, we obtained an area under the curve of 0.84 for NNMT, which was superior to the established tumor marker carcinoembryogenic antigen with an area under the curve of 0.78.  Conclusions: It is proposed that NNMT serum levels may have significance in the early detection and in the management of patients with colorectal cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwGM5NZ6JMdLVg90H21EOLACvtfcHk0ljsKAPg14-K-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVShsb%252FP&md5=e2e24797b2c90ad48b11202539e1edaa</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-05-0983&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-05-0983%26sid%3Dliteratum%253Aachs%26aulast%3DRoessler%26aufirst%3DM.%26aulast%3DRollinger%26aufirst%3DW.%26aulast%3DPalme%26aufirst%3DS.%26aulast%3DHagmann%26aufirst%3DM.%2BL.%26aulast%3DBerndt%26aufirst%3DP.%26aulast%3DEngel%26aufirst%3DA.%2BM.%26aulast%3DSchneidinger%26aufirst%3DB.%26aulast%3DPfeffer%26aufirst%3DM.%26aulast%3DAndres%26aufirst%3DH.%26aulast%3DKarl%26aufirst%3DJ.%26aulast%3DBodenmuller%26aufirst%3DH.%26aulast%3DRuschoff%26aufirst%3DJ.%26aulast%3DHenkel%26aufirst%3DT.%26aulast%3DRohr%26aufirst%3DG.%26aulast%3DRossol%26aufirst%3DS.%26aulast%3DRosch%26aufirst%3DW.%26aulast%3DLangen%26aufirst%3DH.%26aulast%3DZolg%26aufirst%3DW.%26aulast%3DTacke%26aufirst%3DM.%26atitle%3DIdentification%2520of%2520nicotinamide%2520N-methyltransferase%2520as%2520a%2520novel%2520serum%2520tumor%2520marker%2520for%2520colorectal%2520cancer%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2005%26volume%3D11%26spage%3D6550%26epage%3D6557%26doi%3D10.1158%2F1078-0432.CCR-05-0983" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Horning, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suciu, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghadiri, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ulanovskaya, O. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matthews, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lum, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Backus, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cravatt, B. F.</span><span> </span><span class="NLM_article-title">Chemical proteomic profiling of human methyltransferases</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">138</span><span class="NLM_x">, </span> <span class="NLM_fpage">13335</span><span class="NLM_x">–</span> <span class="NLM_lpage">13343</span><span class="refDoi"> DOI: 10.1021/jacs.6b07830</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.6b07830" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFymu7bP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2016&pages=13335-13343&author=B.+D.+Horningauthor=R.+M.+Suciuauthor=D.+A.+Ghadiriauthor=O.+A.+Ulanovskayaauthor=M.+L.+Matthewsauthor=K.+M.+Lumauthor=K.+M.+Backusauthor=S.+J.+Brownauthor=H.+Rosenauthor=B.+F.+Cravatt&title=Chemical+proteomic+profiling+of+human+methyltransferases&doi=10.1021%2Fjacs.6b07830"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical Proteomic Profiling of Human Methyltransferases</span></div><div class="casAuthors">Horning, Benjamin D.; Suciu, Radu M.; Ghadiri, Darian A.; Ulanovskaya, Olesya A.; Matthews, Megan L.; Lum, Kenneth M.; Backus, Keriann M.; Brown, Steven J.; Rosen, Hugh; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">13335-13343</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Methylation is a fundamental mechanism used in Nature to modify the structure and function of biomols., including proteins, DNA, RNA, and metabolites.  Me groups are predominantly installed into biomols. by a large and diverse class of S-adenosyl methionine (SAM)-dependent methyltransferases (MTs), of which there are ∼200 known or putative members in the human proteome.  Deregulated MT activity contributes to numerous diseases, including cancer, and several MT inhibitors are in clin. development.  Nonetheless, a large fraction of the human MT family remains poorly characterized, underscoring the need for new technologies to characterize MTs and their inhibitors in native biol. systems.  Here, the authors describe a suite of S-adenosyl homocysteine (SAH) photoreactive probes and their application in chem. proteomic expts. to profile and enrich a large no. of MTs (>50) from human cancer cell lysates with remarkable specificity over other classes of proteins.  The authors further demonstrate that the SAH probes can enrich MT-assocd. proteins and be used to screen for and assess the selectivity of MT inhibitors, leading to the discovery of a covalent inhibitor of nicotinamide N-methyltransferase (NNMT), an enzyme implicated in cancer and metabolic disorders.  The chem. proteomics probes and methods for their utilization reported herein should prove of value for the functional characterization of MTs, MT complexes, and MT inhibitors in mammalian biol. and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplJsKzvFF-e7Vg90H21EOLACvtfcHk0ljsKAPg14-K-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFymu7bP&md5=0e07845e59e33381f3149beae7500e18</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fjacs.6b07830&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.6b07830%26sid%3Dliteratum%253Aachs%26aulast%3DHorning%26aufirst%3DB.%2BD.%26aulast%3DSuciu%26aufirst%3DR.%2BM.%26aulast%3DGhadiri%26aufirst%3DD.%2BA.%26aulast%3DUlanovskaya%26aufirst%3DO.%2BA.%26aulast%3DMatthews%26aufirst%3DM.%2BL.%26aulast%3DLum%26aufirst%3DK.%2BM.%26aulast%3DBackus%26aufirst%3DK.%2BM.%26aulast%3DBrown%26aufirst%3DS.%2BJ.%26aulast%3DRosen%26aufirst%3DH.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DChemical%2520proteomic%2520profiling%2520of%2520human%2520methyltransferases%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2016%26volume%3D138%26spage%3D13335%26epage%3D13343%26doi%3D10.1021%2Fjacs.6b07830" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Neelakantan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, H. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vance, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hommel, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McHardy, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watowich, S. J.</span><span> </span><span class="NLM_article-title">Structure-activity relationship for small molecule inhibitors of nicotinamide N-methyltransferase</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">5015</span><span class="NLM_x">–</span> <span class="NLM_lpage">5028</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00389</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00389" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BC2sXos1Sgtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=5015-5028&author=H.+Neelakantanauthor=H.+Y.+Wangauthor=V.+Vanceauthor=J.+D.+Hommelauthor=S.+F.+McHardyauthor=S.+J.+Watowich&title=Structure-activity+relationship+for+small+molecule+inhibitors+of+nicotinamide+N-methyltransferase&doi=10.1021%2Facs.jmedchem.7b00389"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Activity Relationship for Small Molecule Inhibitors of Nicotinamide N-Methyltransferase</span></div><div class="casAuthors">Neelakantan, Harshini; Wang, Hua-Yu; Vance, Virginia; Hommel, Jonathan D.; McHardy, Stanton F.; Watowich, Stanley J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5015-5028</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) is a fundamental cytosolic biotransforming enzyme that catalyzes the N-methylation of endogenous and exogenous xenobiotics.  We have identified small mol. inhibitors of NNMT with >1000-fold range of activity and developed comprehensive structure-activity relationships (SARs) for NNMT inhibitors.  Screening of N-methylated quinolinium, isoquinolinium, pyrididium, and benzimidazolium/benzothiazolium analogs resulted in the identification of quinoliniums as a promising scaffold with very low micromolar (IC50 ∼ 1 μM) NNMT inhibition.  Computer-based docking of inhibitors to the NNMT substrate (nicotinamide)-binding site produced a robust correlation between ligand-enzyme interaction docking scores and exptl. calcd. IC50 values.  Predicted binding orientation of the quinolinium analogs revealed selective binding to the NNMT substrate-binding site residues and essential chem. features driving protein-ligand intermol. interactions and NNMT inhibition.  The development of this new series of small mol. NNMT inhibitors direct the future design of lead drug-like inhibitors to treat several metabolic and chronic disease conditions characterized by abnormal NNMT activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMxYy3poogIbVg90H21EOLACvtfcHk0ljsKAPg14-K-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXos1Sgtrc%253D&md5=2cd7159b9e15569221849168e49e39cd</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00389&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00389%26sid%3Dliteratum%253Aachs%26aulast%3DNeelakantan%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DH.%2BY.%26aulast%3DVance%26aufirst%3DV.%26aulast%3DHommel%26aufirst%3DJ.%2BD.%26aulast%3DMcHardy%26aufirst%3DS.%2BF.%26aulast%3DWatowich%26aufirst%3DS.%2BJ.%26atitle%3DStructure-activity%2520relationship%2520for%2520small%2520molecule%2520inhibitors%2520of%2520nicotinamide%2520N-methyltransferase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D5015%26epage%3D5028%26doi%3D10.1021%2Facs.jmedchem.7b00389" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">van Haren, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taig, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuppens, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sastre Torano, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moret, E. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parsons, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sartini, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emanuelli, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, N. I.</span><span> </span><span class="NLM_article-title">Inhibitors of nicotinamide N-methyltransferase designed to mimic the methylation reaction transition state</span> <span class="citation_source-journal">Org. Biomol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">6656</span><span class="NLM_x">–</span> <span class="NLM_lpage">6667</span><span class="refDoi"> DOI: 10.1039/C7OB01357D</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1039%2FC7OB01357D" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=28758655" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1aktLfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2017&pages=6656-6667&author=M.+J.+van+Harenauthor=R.+Taigauthor=J.+Kuppensauthor=J.+Sastre+Toranoauthor=E.+E.+Moretauthor=R.+B.+Parsonsauthor=D.+Sartiniauthor=M.+Emanuelliauthor=N.+I.+Martin&title=Inhibitors+of+nicotinamide+N-methyltransferase+designed+to+mimic+the+methylation+reaction+transition+state&doi=10.1039%2FC7OB01357D"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of nicotinamide N-methyltransferase designed to mimic the methylation reaction transition state</span></div><div class="casAuthors">van Haren, Matthijs J.; Taig, Rebecca; Kuppens, Jilles; Sastre Torano, Javier; Moret, Ed E.; Parsons, Richard B.; Sartini, Davide; Emanuelli, Monica; Martin, Nathaniel I.</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">6656-6667</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) is an enzyme that catalyzes the methylation of nicotinamide to form N'-methylnicotinamide.  Both NNMT and its methylated product have recently been linked to a variety of diseases, suggesting a role for the enzyme as a therapeutic target beyond its previously ascribed metabolic function in detoxification.  We here describe the systematic development of NNMT inhibitors derived from the structures of the substrates involved in the methylation reaction.  By covalently linking fragments of the NNMT substrates a diverse library of bisubstrate-like compds. was prepd.  The ability of these compds. to inhibit NNMT was evaluated providing valuable insights into the structural tolerances of the enzyme active site.  These studies led to the identification of new NNMT inhibitors that mimic the transition state of the methylation reaction and inhibit the enzyme with activity on par with established methyltransferase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxwrd9fP8Bf7Vg90H21EOLACvtfcHk0ljZr3Z10CnMbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1aktLfI&md5=74280c50135611d361ab81b8fc55fa89</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1039%2FC7OB01357D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7OB01357D%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BHaren%26aufirst%3DM.%2BJ.%26aulast%3DTaig%26aufirst%3DR.%26aulast%3DKuppens%26aufirst%3DJ.%26aulast%3DSastre%2BTorano%26aufirst%3DJ.%26aulast%3DMoret%26aufirst%3DE.%2BE.%26aulast%3DParsons%26aufirst%3DR.%2BB.%26aulast%3DSartini%26aufirst%3DD.%26aulast%3DEmanuelli%26aufirst%3DM.%26aulast%3DMartin%26aufirst%3DN.%2BI.%26atitle%3DInhibitors%2520of%2520nicotinamide%2520N-methyltransferase%2520designed%2520to%2520mimic%2520the%2520methylation%2520reaction%2520transition%2520state%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2017%26volume%3D15%26spage%3D6656%26epage%3D6667%26doi%3D10.1039%2FC7OB01357D" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Lavogina, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Enkvist, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uri, A.</span><span> </span><span class="NLM_article-title">Bisubstrate inhibitors of protein kinases: from principle to practical applications</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">23</span><span class="NLM_x">–</span> <span class="NLM_lpage">34</span><span class="refDoi"> DOI: 10.1002/cmdc.200900252</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1002%2Fcmdc.200900252" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=19774589" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhs1SqsbzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=23-34&author=D.+Lavoginaauthor=E.+Enkvistauthor=A.+Uri&title=Bisubstrate+inhibitors+of+protein+kinases%3A+from+principle+to+practical+applications&doi=10.1002%2Fcmdc.200900252"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Bisubstrate Inhibitors of Protein Kinases: from Principle to Practical Applications</span></div><div class="casAuthors">Lavogina, Darja; Enkvist, Erki; Uri, Asko</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">23-34</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Bisubstrate inhibitors consist of two conjugated fragments, each targeted to a different binding site of a bisubstrate enzyme.  The design of bisubstrate inhibitors presupposes the formation of the ternary complex in the course of the catalyzed reaction.  The principle advantage of bisubstrate inhibitors is their ability to generate more interactions with the target enzyme that could result in improved affinity and selectivity of the conjugates, when compared with single-site inhibitors.  Among phosphotransferases, the approach was first successfully used for adenylate kinase in 1973.  Since then, several types of bisubstrate inhibitors have been developed for protein kinases, including conjugates of peptides with nucleotides, adenosine derivs. and potent ATP-competitive inhibitors.  Earlier bisubstrate inhibitors had pharmacokinetic qualities that were unsuitable for cellular expts. and hence were mostly used for in vitro studies.  The recently constructed conjugates of adenosine derivs. and D-arginine-rich peptides (ARCs) possess high kinase affinity, high biol. and chem. stability and good cell plasma membrane penetrative properties that enable their application in the regulation of cellular protein phosphorylation balances in cell and tissue expts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSsmmKerEiPrVg90H21EOLACvtfcHk0ljZr3Z10CnMbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhs1SqsbzP&md5=7efb7b5aa44347f5fa8fc40f98065337</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.200900252&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.200900252%26sid%3Dliteratum%253Aachs%26aulast%3DLavogina%26aufirst%3DD.%26aulast%3DEnkvist%26aufirst%3DE.%26aulast%3DUri%26aufirst%3DA.%26atitle%3DBisubstrate%2520inhibitors%2520of%2520protein%2520kinases%253A%2520from%2520principle%2520to%2520practical%2520applications%26jtitle%3DChemMedChem%26date%3D2010%26volume%3D5%26spage%3D23%26epage%3D34%26doi%3D10.1002%2Fcmdc.200900252" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Yu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Magalhaes, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cook, P. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blanchard, J. S.</span><span> </span><span class="NLM_article-title">Bisubstrate inhibition: Theory and application to N-acetyltransferases</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">14788</span><span class="NLM_x">–</span> <span class="NLM_lpage">14794</span><span class="refDoi"> DOI: 10.1021/bi061621t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi061621t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1SjtbfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2006&pages=14788-14794&author=M.+Yuauthor=M.+L.+Magalhaesauthor=P.+F.+Cookauthor=J.+S.+Blanchard&title=Bisubstrate+inhibition%3A+Theory+and+application+to+N-acetyltransferases&doi=10.1021%2Fbi061621t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Bisubstrate Inhibition: Theory and Application to N-Acetyltransferases</span></div><div class="casAuthors">Yu, Michael; Magalhaes, Maria L. B.; Cook, Paul F.; Blanchard, John S.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">14788-14794</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bisubstrate inhibitors represent a potentially powerful group of compds. that have found significant therapeutic utility.  Although these compds. have been synthesized and tested against a no. of enzymes that catalyze sequential bireactant reactions, the detailed theory for predicting the expected patterns of inhibition against the two substrates for various bireactant kinetic mechanisms has, heretofore, not been presented.  We have derived the rate equations for all likely sequential bireactant mechanisms and provide two examples in which bisubstrate inhibitors allow the kinetic mechanism to be detd.  Bisubstrate inhibitor kinetics is a powerful diagnostic for the detn. of kinetic mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrk_sv2txdK0LVg90H21EOLACvtfcHk0ljZr3Z10CnMbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1SjtbfN&md5=6ddb7dcb30e7d97a3577a6088db49433</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fbi061621t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi061621t%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DM.%26aulast%3DMagalhaes%26aufirst%3DM.%2BL.%26aulast%3DCook%26aufirst%3DP.%2BF.%26aulast%3DBlanchard%26aufirst%3DJ.%2BS.%26atitle%3DBisubstrate%2520inhibition%253A%2520Theory%2520and%2520application%2520to%2520N-acetyltransferases%26jtitle%3DBiochemistry%26date%3D2006%26volume%3D45%26spage%3D14788%26epage%3D14794%26doi%3D10.1021%2Fbi061621t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Peng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sartini, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pozzi, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilk, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emanuelli, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yee, V. C.</span><span> </span><span class="NLM_article-title">Structural basis of substrate recognition in human nicotinamide N-methyltransferase</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">7800</span><span class="NLM_x">–</span> <span class="NLM_lpage">7808</span><span class="refDoi"> DOI: 10.1021/bi2007614</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi2007614" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtValsbnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2011&pages=7800-7808&author=Y.+Pengauthor=D.+Sartiniauthor=V.+Pozziauthor=D.+Wilkauthor=M.+Emanuelliauthor=V.+C.+Yee&title=Structural+basis+of+substrate+recognition+in+human+nicotinamide+N-methyltransferase&doi=10.1021%2Fbi2007614"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of substrate recognition in human nicotinamide N-methyltransferase</span></div><div class="casAuthors">Peng, Yi; Sartini, Davide; Pozzi, Valentina; Wilk, Dennis; Emanuelli, Monica; Yee, Vivien C.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">7800-7808</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) catalyzes the N-methylation of nicotinamide, pyridines, and other analogs using S-adenosyl-L-methionine as donor.  NNMT plays a significant role in the regulation of metabolic pathways and is expressed at markedly high levels in several kinds of cancers, presenting it as a potential mol. target for cancer therapy.  Here, the authors detd. the crystal structure of human NNMT as a ternary complex bound to both the demethylated donor, S-adenosyl-L-homocysteine, and the acceptor substrate, nicotinamide, to 2.7 Å resoln.  These studies reveal the structural basis for nicotinamide binding and highlight several residues in the active site which may play roles in nicotinamide recognition and NNMT catalysis.  The functional importance of these residues was probed by mutagenesis.  Of three residues near the nicotinamide's amide group, substitution of S201 and S213 had no effect on enzyme activity while replacement of D197 dramatically decreased activity.  Substitutions of Y20, whose side chain hydroxyl interacts with both the nicotinamide arom. ring and AdoHcy carboxylate, also compromised activity.  Enzyme kinetics anal. revealed kcat/Km decreases of 2-3 orders of magnitude for the D197A and Y20A mutants, confirming the functional importance of these active site residues.  The mutants exhibited substantially increased Km for both NCA and AdoMet and modestly decreased kcat.  MD simulations revealed long-range conformational effects which provide an explanation for the large increase in Km(AdoMet) for the D197A mutant, which interacts directly only with nicotinamide in the ternary complex crystal structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqh1yxD6WIJFLVg90H21EOLACvtfcHk0lgmnL1_qPSlwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtValsbnF&md5=926af50d2464e21094ffb17205793d19</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fbi2007614&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi2007614%26sid%3Dliteratum%253Aachs%26aulast%3DPeng%26aufirst%3DY.%26aulast%3DSartini%26aufirst%3DD.%26aulast%3DPozzi%26aufirst%3DV.%26aulast%3DWilk%26aufirst%3DD.%26aulast%3DEmanuelli%26aufirst%3DM.%26aulast%3DYee%26aufirst%3DV.%2BC.%26atitle%3DStructural%2520basis%2520of%2520substrate%2520recognition%2520in%2520human%2520nicotinamide%2520N-methyltransferase%26jtitle%3DBiochemistry%26date%3D2011%26volume%3D50%26spage%3D7800%26epage%3D7808%26doi%3D10.1021%2Fbi2007614" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Liu, K.; Sun, X.; Ma, J.; Zhou, Z.; Dong, Q.; Peng, S.; Wu, J.; Tan, S.; Blobel, G.; Fan, J.</span><span> </span><span class="NLM_article-title">Prediction of amino acid side chain conformation using a deep neural network</span>.  <span class="citation_source-journal">ArXiv e-prints</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span>arXiv:1707.08381.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=K.+Liu&author=X.+Sun&author=J.+Ma&author=Z.+Zhou&author=Q.+Dong&author=S.+Peng&author=J.+Wu&author=S.+Tan&author=G.+Blobel&author=J.+Fan&title=Prediction+of+amino+acid+side+chain+conformation+using+a+deep+neural+network"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DK.%26atitle%3DPrediction%2520of%2520amino%2520acid%2520side%2520chain%2520conformation%2520using%2520a%2520deep%2520neural%2520network%26jtitle%3DArXiv%2520e-prints%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Yao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anglin, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prasad, B. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, Y.</span><span> </span><span class="NLM_article-title">Selective inhibitors of histone methyltransferase DOT1L: design, synthesis, and crystallographic studies</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">133</span><span class="NLM_x">, </span> <span class="NLM_fpage">16746</span><span class="NLM_x">–</span> <span class="NLM_lpage">16749</span><span class="refDoi"> DOI: 10.1021/ja206312b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja206312b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2011&pages=16746-16749&author=Y.+Yaoauthor=P.+Chenauthor=J.+Diaoauthor=G.+Chengauthor=L.+Dengauthor=J.+L.+Anglinauthor=B.+V.+Prasadauthor=Y.+Song&title=Selective+inhibitors+of+histone+methyltransferase+DOT1L%3A+design%2C+synthesis%2C+and+crystallographic+studies&doi=10.1021%2Fja206312b"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fja206312b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja206312b%26sid%3Dliteratum%253Aachs%26aulast%3DYao%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DP.%26aulast%3DDiao%26aufirst%3DJ.%26aulast%3DCheng%26aufirst%3DG.%26aulast%3DDeng%26aufirst%3DL.%26aulast%3DAnglin%26aufirst%3DJ.%2BL.%26aulast%3DPrasad%26aufirst%3DB.%2BV.%26aulast%3DSong%26aufirst%3DY.%26atitle%3DSelective%2520inhibitors%2520of%2520histone%2520methyltransferase%2520DOT1L%253A%2520design%252C%2520synthesis%252C%2520and%2520crystallographic%2520studies%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2011%26volume%3D133%26spage%3D16746%26epage%3D16749%26doi%3D10.1021%2Fja206312b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Cox, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gibson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mayo Martin, M. B.</span><span> </span><span class="NLM_article-title">Aspartyl phosphonates and phosphoramidates: the first synthetic inhibitors of bacterial aspartate-semialdehyde dehydrogenase</span> <span class="citation_source-journal">ChemBioChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">874</span><span class="NLM_x">–</span> <span class="NLM_lpage">886</span><span class="refDoi"> DOI: 10.1002/1439-7633(20020902)3:9<874::AID-CBIC874>3.0.CO;2-V</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1002%2F1439-7633%2820020902%293%3A9%3C874%3A%3AAID-CBIC874%3E3.0.CO%3B2-V" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2002&pages=874-886&author=R.+J.+Coxauthor=J.+S.+Gibsonauthor=M.+B.+Mayo+Martin&title=Aspartyl+phosphonates+and+phosphoramidates%3A+the+first+synthetic+inhibitors+of+bacterial+aspartate-semialdehyde+dehydrogenase&doi=10.1002%2F1439-7633%2820020902%293%3A9%3C874%3A%3AAID-CBIC874%3E3.0.CO%3B2-V"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1002%2F1439-7633%2820020902%293%3A9%3C874%3A%3AAID-CBIC874%3E3.0.CO%3B2-V&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F1439-7633%252820020902%25293%253A9%253C874%253A%253AAID-CBIC874%253E3.0.CO%253B2-V%26sid%3Dliteratum%253Aachs%26aulast%3DCox%26aufirst%3DR.%2BJ.%26aulast%3DGibson%26aufirst%3DJ.%2BS.%26aulast%3DMayo%2BMartin%26aufirst%3DM.%2BB.%26atitle%3DAspartyl%2520phosphonates%2520and%2520phosphoramidates%253A%2520the%2520first%2520synthetic%2520inhibitors%2520of%2520bacterial%2520aspartate-semialdehyde%2520dehydrogenase%26jtitle%3DChemBioChem%26date%3D2002%26volume%3D3%26spage%3D874%26epage%3D886%26doi%3D10.1002%2F1439-7633%2820020902%293%3A9%3C874%3A%3AAID-CBIC874%3E3.0.CO%3B2-V" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Usuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sugimura, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Komatsu, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koseki, Y.</span><span> </span><span class="NLM_article-title">Biomimetic Chichibabin pyridine synthesis of the COPD biomarkers and elastin cross-linkers isodesmosine and desmosine</span> <span class="citation_source-journal">Org. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">1672</span><span class="NLM_x">–</span> <span class="NLM_lpage">1675</span><span class="refDoi"> DOI: 10.1021/ol500333t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol500333t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2014&pages=1672-1675&author=T.+Usukiauthor=T.+Sugimuraauthor=A.+Komatsuauthor=Y.+Koseki&title=Biomimetic+Chichibabin+pyridine+synthesis+of+the+COPD+biomarkers+and+elastin+cross-linkers+isodesmosine+and+desmosine&doi=10.1021%2Fol500333t"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fol500333t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol500333t%26sid%3Dliteratum%253Aachs%26aulast%3DUsuki%26aufirst%3DT.%26aulast%3DSugimura%26aufirst%3DT.%26aulast%3DKomatsu%26aufirst%3DA.%26aulast%3DKoseki%26aufirst%3DY.%26atitle%3DBiomimetic%2520Chichibabin%2520pyridine%2520synthesis%2520of%2520the%2520COPD%2520biomarkers%2520and%2520elastin%2520cross-linkers%2520isodesmosine%2520and%2520desmosine%26jtitle%3DOrg.%2520Lett.%26date%3D2014%26volume%3D16%26spage%3D1672%26epage%3D1675%26doi%3D10.1021%2Fol500333t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Collazo, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Couture, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bulfer, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trievel, R. C.</span><span> </span><span class="NLM_article-title">A coupled fluorescent assay for histone methyltransferases</span> <span class="citation_source-journal">Anal. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">342</span><span class="NLM_x">, </span> <span class="NLM_fpage">86</span><span class="NLM_x">–</span> <span class="NLM_lpage">92</span><span class="refDoi"> DOI: 10.1016/j.ab.2005.04.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1016%2Fj.ab.2005.04.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=15958184" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BD2MXlt1GktL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=342&publication_year=2005&pages=86-92&author=E.+Collazoauthor=J.+F.+Coutureauthor=S.+Bulferauthor=R.+C.+Trievel&title=A+coupled+fluorescent+assay+for+histone+methyltransferases&doi=10.1016%2Fj.ab.2005.04.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">A coupled fluorescent assay for histone methyltransferases</span></div><div class="casAuthors">Collazo, Evys; Couture, Jean-Francois; Bulfer, Stacie; Trievel, Raymond C.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">342</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">86-92</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Histone methyltransferases (HMTs) catalyze the S-adenosylmethionine (AdoMet)-dependent methylation of lysines and arginines in the nucleosomal core histones H3 and H4 and the linker histone H1b.  Methylation of these residues regulates either transcriptional activation or silencing, depending on the residue modified and its degree of methylation.  Despite an intense interest in elucidating the functions of HMTs in transcriptional regulation, these enzymes have remained challenging to quant. assay.  To characterize the substrate specificity of HMTs, we have developed a coupled-fluorescence-based assay for AdoMet-dependent methyltransferases.  This assay utilizes S-adenosylhomocysteine hydrolase (SAHH) to hydrolyze the methyltransfer product S-adenosylhomocysteine (AdoHcy) to homocysteine (Hcy) and adenosine (Ado).  The Hcy concn. is then detd. through conjugation of its free sulfhydryl moiety to a thiol-sensitive fluorophore.  Using this assay, we have detd. the kinetic parameters for the methylation of a synthetic histone H3 peptide (corresponding to residues 1-15 of the native protein) by Schizosaccharomyces pombe CLR4, an H3 Lys-9-specific methyltransferase.  The fluorescent SAHH-coupled assay allows rapid and facile detn. of HMT kinetics and can be adapted to measure the enzymic activity of a wide variety of AdoMet-dependent methyltransferases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfHslBzi9obbVg90H21EOLACvtfcHk0lgmnL1_qPSlwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXlt1GktL4%253D&md5=2b949dcde263d15559202131cc74da1e</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2005.04.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2005.04.007%26sid%3Dliteratum%253Aachs%26aulast%3DCollazo%26aufirst%3DE.%26aulast%3DCouture%26aufirst%3DJ.%2BF.%26aulast%3DBulfer%26aufirst%3DS.%26aulast%3DTrievel%26aufirst%3DR.%2BC.%26atitle%3DA%2520coupled%2520fluorescent%2520assay%2520for%2520histone%2520methyltransferases%26jtitle%3DAnal.%2520Biochem.%26date%3D2005%26volume%3D342%26spage%3D86%26epage%3D92%26doi%3D10.1016%2Fj.ab.2005.04.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siarheyeva, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajian, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span> </span><span class="NLM_article-title">Fluorescence-based methods for screening writers and readers of histone methyl marks</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">71</span><span class="NLM_x">–</span> <span class="NLM_lpage">84</span><span class="refDoi"> DOI: 10.1177/1087057111422256</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1177%2F1087057111422256" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=21972038" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BC38Xit1Olt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2012&pages=71-84&author=A.+Allali-Hassaniauthor=G.+A.+Wasneyauthor=A.+Siarheyevaauthor=T.+Hajianauthor=C.+H.+Arrowsmithauthor=M.+Vedadi&title=Fluorescence-based+methods+for+screening+writers+and+readers+of+histone+methyl+marks&doi=10.1177%2F1087057111422256"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorescence-based methods for screening writers and readers of histone methyl marks</span></div><div class="casAuthors">Allali-Hassani, Abdellah; Wasney, Gregory A.; Siarheyeva, Alena; Hajian, Taraneh; Arrowsmith, Cheryl H.; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">71-84</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">The histone methyltransferase (HMT) family of proteins consists of enzymes that methylate lysine or arginine residues on histone tails as well as other proteins.  Such modifications affect chromatin structure and play a significant regulatory role in gene expression.  Many HMTs have been implicated in tumorigenesis and progression of multiple malignancies and play essential roles in embryonic development and stem cell renewal.  Overexpression of some HMTs has been obsd. and is correlated pos. with various types of cancer.  Here the authors report development of a continuous fluorescence-based methyltransferase assay in a 384-well format and its application in detg. kinetic parameters for EHMT1, G9a, PRMT3, SETD7, and SUV39H2 as well as for screening against libraries of small mols. to identify enzyme inhibitors.  They also report the development of a peptide displacement assay using fluorescence polarization in a 384-well format to assay and screen protein peptide interactions such as those of WDR5 and EED, components of MLL and EZH2 methyltransferase complexes.  Using these high-throughput screening methods, the authors have identified potent inhibitors and ligands for some of these proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0s8Ex9rZOXrVg90H21EOLACvtfcHk0li9IVAZTkSz-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xit1Olt7g%253D&md5=105a20c2473ab603ade42a38764def09</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1177%2F1087057111422256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057111422256%26sid%3Dliteratum%253Aachs%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DHajian%26aufirst%3DT.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DFluorescence-based%2520methods%2520for%2520screening%2520writers%2520and%2520readers%2520of%2520histone%2520methyl%2520marks%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2012%26volume%3D17%26spage%3D71%26epage%3D84%26doi%3D10.1177%2F1087057111422256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref100"><div class="reference"><strong class="refLabel"><a href="#ref100" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit100"><span><span class="NLM_contrib-group">Richon, V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnston, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sneeringer, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majer, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elliston, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jerva, L. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span> </span><span class="NLM_article-title">Chemogenetic analysis of human protein methyltransferases</span> <span class="citation_source-journal">Chem. Biol. Drug Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">78</span><span class="NLM_x">, </span> <span class="NLM_fpage">199</span><span class="NLM_x">–</span> <span class="NLM_lpage">210</span><span class="refDoi"> DOI: 10.1111/j.1747-0285.2011.01135.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1111%2Fj.1747-0285.2011.01135.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=21564555" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpsF2qsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2011&pages=199-210&author=V.+M.+Richonauthor=D.+Johnstonauthor=C.+J.+Sneeringerauthor=L.+Jinauthor=C.+R.+Majerauthor=K.+Ellistonauthor=L.+F.+Jervaauthor=M.+P.+Scottauthor=R.+A.+Copeland&title=Chemogenetic+analysis+of+human+protein+methyltransferases&doi=10.1111%2Fj.1747-0285.2011.01135.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit100R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Chemogenetic analysis of human protein methyltransferases</span></div><div class="casAuthors">Richon, Victoria M.; Johnston, Danielle; Sneeringer, Christopher J.; Jin, Lei; Majer, Christina R.; Elliston, Keith; Jerva, L. Fred; Scott, Margaret Porter; Copeland, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">199-210</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A survey of the human genome was performed to understand the constituency of protein methyltransferases (both protein arginine and lysine methyltransferases) and the relatedness of their catalytic domains.  We identified 51 protein lysine methyltransferase proteins based on similarity to the canonical Drosophila Su(var)3-9, enhancer of zeste (E(z)), and trithorax (trx) domain.  Disruptor of telomeric silencing-1-like, a known protein lysine methyltransferase, did not fit within the protein lysine methyltransferase family, but did group with the protein arginine methyltransferases, along with 44 other proteins, including the METTL and NOP2/Sun domain family proteins.  We show that a representative METTL, METTL11A, demonstrates catalytic activity as a histone methyltransferase.  We also solved the co-crystal structures of disruptor of telomeric silencing-1-like with S-adenosylmethionine and S-adenosylhomocysteine bound in its active site.  The conformation of both ligands is virtually identical to that found in known protein arginine methyltransferases, METTL and NOP2/Sun domain family proteins and is distinct from that seen in the Drosophila Su(var)3-9, enhancer of zeste (E(z)), and trithorax (trx) domain protein lysine methyltransferases.  We have developed biochem. assays for 11 members of the protein methyltransferase target class and have profiled the affinity of three ligands for these enzymes: the common methyl-donating substrate S-adenosylmethionine; the common reaction product S-adenosylhomocysteine; and the natural product sinefungin.  The affinity of each of these ligands is mapped onto the family trees of the protein lysine methyltransferases and protein arginine methyltransferases to reveal patterns of ligand recognition by these enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJylTD5PqDfLVg90H21EOLACvtfcHk0li9IVAZTkSz-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpsF2qsrk%253D&md5=ab02523741396a36afc7d38f2a90b4f0</span></div><a href="/servlet/linkout?suffix=cit100&amp;dbid=16384&amp;doi=10.1111%2Fj.1747-0285.2011.01135.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1747-0285.2011.01135.x%26sid%3Dliteratum%253Aachs%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DJohnston%26aufirst%3DD.%26aulast%3DSneeringer%26aufirst%3DC.%2BJ.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DElliston%26aufirst%3DK.%26aulast%3DJerva%26aufirst%3DL.%2BF.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26atitle%3DChemogenetic%2520analysis%2520of%2520human%2520protein%2520methyltransferases%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2011%26volume%3D78%26spage%3D199%26epage%3D210%26doi%3D10.1111%2Fj.1747-0285.2011.01135.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref101"><div class="reference"><strong class="refLabel"><a href="#ref101" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit101"><span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dai, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atadja, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, K.</span><span> </span><span class="NLM_article-title">Structure of human SMYD2 protein reveals the basis of p53 tumor suppressor methylation</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">286</span><span class="NLM_x">, </span> <span class="NLM_fpage">38725</span><span class="NLM_x">–</span> <span class="NLM_lpage">38737</span><span class="refDoi"> DOI: 10.1074/jbc.M111.262410</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1074%2Fjbc.M111.262410" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=21880715" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlykurvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2011&pages=38725-38737&author=L.+Wangauthor=L.+Liauthor=H.+Zhangauthor=X.+Luoauthor=J.+Daiauthor=S.+Zhouauthor=J.+Guauthor=J.+Zhuauthor=P.+Atadjaauthor=C.+Luauthor=E.+Liauthor=K.+Zhao&title=Structure+of+human+SMYD2+protein+reveals+the+basis+of+p53+tumor+suppressor+methylation&doi=10.1074%2Fjbc.M111.262410"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit101R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of Human SMYD2 Protein Reveals the Basis of p53 Tumor Suppressor Methylation</span></div><div class="casAuthors">Wang, Li; Li, Ling; Zhang, Hailong; Luo, Xiao; Dai, Jingquan; Zhou, Shaolian; Gu, Justin; Zhu, Jidong; Atadja, Peter; Lu, Chris; Li, En; Zhao, Kehao</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">286</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">38725-38737, S38725/1-S38725/5</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">SMYD2 belongs to a subfamily of histone lysine methyltransferase and was recently identified to methylate tumor suppressor p53 and Rb.  Here we report that SMYD2 prefers to methylate p53 Lys-370 over histone substrates in vitro.  Consistently, the level of endogenous p53 Lys-370 monomethylation is significantly elevated when SMYD2 is overexpressed in vivo.  We have solved the high resoln. crystal structures of the full-length SMYD2 protein in binary complex with its cofactor S-adenosylmethionine and in ternary complex with cofactor product S-adenosylhomocysteine and p53 substrate peptide (residues 368-375), resp. p53 peptide binds to a deep pocket of the interface between catalytic SET(1-282) and C-terminal domain (CTD) with an unprecedented U-shaped conformation.  Subtle conformational change exists around the p53 binding site between the binary and ternary structures, in particular the tetratricopeptide repeat motif of the CTD.  In addn., a unique EDEE motif between the loop of anti-parallel β7 and β8 sheets of the SET core not only interacts with p53 substrate but also forms a hydrogen bond network with residues from CTD.  These observations suggest that the tetratricopeptide repeat and EDEE motif may play an important role in detg. p53 substrate binding specificity.  This is further verified by the findings that deletion of the CTD domain drastically reduces the methylation activity of SMYD2 to p53 protein.  Meanwhile, mutation of EDEE residues impairs both the binding and the enzymic activity of SMYD2 to p53 Lys-370.  These data together reveal the mol. basis of SMYD2 in specifically recognizing and regulating functions of p53 tumor suppressor through Lys-370 monomethylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpX6G4Yhf_H47Vg90H21EOLACvtfcHk0li9IVAZTkSz-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlykurvI&md5=671857a22846de5d607e83a03dd52d37</span></div><a href="/servlet/linkout?suffix=cit101&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M111.262410&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M111.262410%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DDai%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DS.%26aulast%3DGu%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DAtadja%26aufirst%3DP.%26aulast%3DLu%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DE.%26aulast%3DZhao%26aufirst%3DK.%26atitle%3DStructure%2520of%2520human%2520SMYD2%2520protein%2520reveals%2520the%2520basis%2520of%2520p53%2520tumor%2520suppressor%2520methylation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2011%26volume%3D286%26spage%3D38725%26epage%3D38737%26doi%3D10.1074%2Fjbc.M111.262410" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Galijasevic, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abdulhamid, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abu-Soud, H. M.</span><span> </span><span class="NLM_article-title">Melatonin is a potent inhibitor for myeloperoxidase</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">2668</span><span class="NLM_x">–</span> <span class="NLM_lpage">2677</span><span class="refDoi"> DOI: 10.1021/bi702016q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi702016q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1Omsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2008&pages=2668-2677&author=S.+Galijasevicauthor=I.+Abdulhamidauthor=H.+M.+Abu-Soud&title=Melatonin+is+a+potent+inhibitor+for+myeloperoxidase&doi=10.1021%2Fbi702016q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Melatonin Is a Potent Inhibitor for Myeloperoxidase</span></div><div class="casAuthors">Galijasevic, Semira; Abdulhamid, Ibrahim; Abu-Soud, Husam M.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2668-2677</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Myeloperoxidase (MPO) catalyzes the formation of potent oxidants that have been implicated in the pathogenesis of various diseases including atherosclerosis, asthma, arthritis, and cancer.  Melatonin plays an important part in the regulation of various body functions including circadian sleep rhythms, blood pressure, oncogenesis, retinal function, seasonal reprodn., and immunity.  Here, we demonstrate that melatonin serves as a potent inhibitor of MPO under physiol.-like conditions.  In the presence of chloride (Cl-), melatonin inactivated MPO at two points in the classic peroxidase cycle through binding to MPO to form an inactive complex, melatonin-MPO-Cl, and accelerating MPO compd. II formation, an inactive form of MPO.  Inactivation of MPO was mirrored by the direct conversion of MPO-Fe(III) to MPO compd. II without any sign of compd. I accumulation.  This behavior indicates that melatonin binding modulates the formation of MPO intermediates and their decay rates.  The Cl- presence enhanced the affinity of MPO toward melatonin, which switches the enzyme activity from peroxidn. to catalase-like activity.  In the absence of Cl-, melatonin served as a 1e- substrate for MPO compd. I, but at higher concn. it limited the reaction by its dissocn. from the corresponding complex.  Importantly, melatonin-dependent inhibition of MPO occurred with a wide range of concns. that span various physiol. and supplemental ranges.  Thus, the interplay between MPO and melatonin may have a broader implication in the function of several biol. systems.  This dual regulation by melatonin is unique and represents a new means through which melatonin can control MPO and its downstream inflammatory pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRFdB_4Le6XrVg90H21EOLACvtfcHk0liHPPJoQ6CR6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1Omsbw%253D&md5=3b707a0557ed92becb329a004ea806a8</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fbi702016q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi702016q%26sid%3Dliteratum%253Aachs%26aulast%3DGalijasevic%26aufirst%3DS.%26aulast%3DAbdulhamid%26aufirst%3DI.%26aulast%3DAbu-Soud%26aufirst%3DH.%2BM.%26atitle%3DMelatonin%2520is%2520a%2520potent%2520inhibitor%2520for%2520myeloperoxidase%26jtitle%3DBiochemistry%26date%3D2008%26volume%3D47%26spage%3D2668%26epage%3D2677%26doi%3D10.1021%2Fbi702016q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Neelakantan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vance, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, H. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McHardy, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watowich, S. J.</span><span> </span><span class="NLM_article-title">Noncoupled Fluorescent Assay for Direct Real-Time Monitoring of Nicotinamide N-Methyltransferase Activity</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">824</span><span class="NLM_x">–</span> <span class="NLM_lpage">832</span><span class="refDoi"> DOI: 10.1021/acs.biochem.6b01215</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biochem.6b01215" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFajs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=824-832&author=H.+Neelakantanauthor=V.+Vanceauthor=H.+L.+Wangauthor=S.+F.+McHardyauthor=S.+J.+Watowich&title=Noncoupled+Fluorescent+Assay+for+Direct+Real-Time+Monitoring+of+Nicotinamide+N-Methyltransferase+Activity&doi=10.1021%2Facs.biochem.6b01215"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Noncoupled Fluorescent Assay for Direct Real-Time Monitoring of Nicotinamide N-Methyltransferase Activity</span></div><div class="casAuthors">Neelakantan, Harshini; Vance, Virginia; Wang, Hua-Yu Leo; McHardy, Stanton F.; Watowich, Stanley J.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">824-832</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) is an important biotransforming enzyme that catalyzes the transfer of a labile Me group from the ubiquitous cofactor, S-adenosyl-L-methionine (SAM), to endogenous and exogenous small mols. such as quinoline to form methylated end-products.  NNMT has been implicated in a no. of chronic disease conditions, including metabolic disorders, cardiovascular disease, cancer, osteoarthritis, kidney disease, and Parkinson's disease.  Here, the authors developed a novel noncoupled fluorescence-based methyltransferase assay that enabled direct ultrasensitive real-time detection of the NNMT reaction product, 1-methylquinolinium.  This is the 1st assay reported to date to utilize fluorescence spectroscopy to directly monitor NNMT product formation and activity in real-time.  This assay provided accurate kinetic data that enabled detailed comparative anal. of the NNMT reaction mechanism and kinetic parameters.  A reaction model based on a random bi-reactant mechanism produced global curve fits that were most consistent with steady-state initial velocity data collected across an array of substrate concns.  Based on the reaction mechanism, each substrate could independently bind to the NNMT apoenzyme; however, both substrates bound to the complementary binary complexes with ∼20-fold higher affinity relative to independent binding.  This reaction mechanism implies either substrate-induced conformational changes or bi-reactant intermol. interactions may stabilize substrate binding to the binary complex and formation of the ternary complex.  Importantly, this assay could rapidly generate concn. response curves for known NNMT inhibitors, suggesting its applicability for high-throughput screening (HTS) of chem. libraries to identify novel NNMT inhibitors.  Furthermore, this novel assay potentially offers a robust detection technol. for use in SAM substrate competition assays for the discovery and development of SAM-dependent methyltransferase inhibitors such as S-adenosyl-L-homocysteine and 1-methylnicotinamide.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6X5AP5dGb77Vg90H21EOLACvtfcHk0liHPPJoQ6CR6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFajs74%253D&md5=bf243bd7362322b094c31b1e247b7240</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Facs.biochem.6b01215&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biochem.6b01215%26sid%3Dliteratum%253Aachs%26aulast%3DNeelakantan%26aufirst%3DH.%26aulast%3DVance%26aufirst%3DV.%26aulast%3DWang%26aufirst%3DH.%2BL.%26aulast%3DMcHardy%26aufirst%3DS.%2BF.%26aulast%3DWatowich%26aufirst%3DS.%2BJ.%26atitle%3DNoncoupled%2520Fluorescent%2520Assay%2520for%2520Direct%2520Real-Time%2520Monitoring%2520of%2520Nicotinamide%2520N-Methyltransferase%2520Activity%26jtitle%3DBiochemistry%26date%3D2017%26volume%3D56%26spage%3D824%26epage%3D832%26doi%3D10.1021%2Facs.biochem.6b01215" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Swaminathan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Birudukota, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thakur, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parveen, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kandan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Juluri, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaik, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anand, N. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burri, R. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kristam, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hallur, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rajagopal, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schreuder, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Langer, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rudolph, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruf, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhakshinamoorthy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gosu, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kannt, A.</span><span> </span><span class="NLM_article-title">Crystal structures of monkey and mouse nicotinamide N-methyltransferase (NNMT) bound with end product, 1-methyl nicotinamide</span> <span class="citation_source-journal">Biochem. Biophys. Res. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">491</span><span class="NLM_x">, </span> <span class="NLM_fpage">416</span><span class="NLM_x">–</span> <span class="NLM_lpage">422</span><span class="refDoi"> DOI: 10.1016/j.bbrc.2017.07.087</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1016%2Fj.bbrc.2017.07.087" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=491&publication_year=2017&pages=416-422&author=S.+Swaminathanauthor=S.+Birudukotaauthor=M.+K.+Thakurauthor=R.+Parveenauthor=S.+Kandanauthor=S.+Juluriauthor=S.+Shaikauthor=N.+N.+Anandauthor=R.+R.+Burriauthor=R.+Kristamauthor=M.+S.+Hallurauthor=S.+Rajagopalauthor=H.+Schreuderauthor=T.+Langerauthor=C.+Rudolphauthor=S.+Rufauthor=S.+Dhakshinamoorthyauthor=R.+Gosuauthor=A.+Kannt&title=Crystal+structures+of+monkey+and+mouse+nicotinamide+N-methyltransferase+%28NNMT%29+bound+with+end+product%2C+1-methyl+nicotinamide&doi=10.1016%2Fj.bbrc.2017.07.087"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2017.07.087&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2017.07.087%26sid%3Dliteratum%253Aachs%26aulast%3DSwaminathan%26aufirst%3DS.%26aulast%3DBirudukota%26aufirst%3DS.%26aulast%3DThakur%26aufirst%3DM.%2BK.%26aulast%3DParveen%26aufirst%3DR.%26aulast%3DKandan%26aufirst%3DS.%26aulast%3DJuluri%26aufirst%3DS.%26aulast%3DShaik%26aufirst%3DS.%26aulast%3DAnand%26aufirst%3DN.%2BN.%26aulast%3DBurri%26aufirst%3DR.%2BR.%26aulast%3DKristam%26aufirst%3DR.%26aulast%3DHallur%26aufirst%3DM.%2BS.%26aulast%3DRajagopal%26aufirst%3DS.%26aulast%3DSchreuder%26aufirst%3DH.%26aulast%3DLanger%26aufirst%3DT.%26aulast%3DRudolph%26aufirst%3DC.%26aulast%3DRuf%26aufirst%3DS.%26aulast%3DDhakshinamoorthy%26aufirst%3DS.%26aulast%3DGosu%26aufirst%3DR.%26aulast%3DKannt%26aufirst%3DA.%26atitle%3DCrystal%2520structures%2520of%2520monkey%2520and%2520mouse%2520nicotinamide%2520N-methyltransferase%2520%2528NNMT%2529%2520bound%2520with%2520end%2520product%252C%25201-methyl%2520nicotinamide%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2017%26volume%3D491%26spage%3D416%26epage%3D422%26doi%3D10.1016%2Fj.bbrc.2017.07.087" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Ramadan, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bremner-Hay, N. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlson, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Comstock, L. R.</span><span> </span><span class="NLM_article-title">Synthesis and evaluation of N6-substituted azide- and alkyne-bearing N-mustard analogs of S-adenosyl-l-methionine</span> <span class="citation_source-journal">Tetrahedron</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">70</span><span class="NLM_x">, </span> <span class="NLM_fpage">5291</span><span class="NLM_x">–</span> <span class="NLM_lpage">5297</span><span class="refDoi"> DOI: 10.1016/j.tet.2014.05.055</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1016%2Fj.tet.2014.05.055" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2014&pages=5291-5297&author=M.+Ramadanauthor=N.+K.+Bremner-Hayauthor=S.+A.+Carlsonauthor=L.+R.+Comstock&title=Synthesis+and+evaluation+of+N6-substituted+azide-+and+alkyne-bearing+N-mustard+analogs+of+S-adenosyl-l-methionine&doi=10.1016%2Fj.tet.2014.05.055"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.tet.2014.05.055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tet.2014.05.055%26sid%3Dliteratum%253Aachs%26aulast%3DRamadan%26aufirst%3DM.%26aulast%3DBremner-Hay%26aufirst%3DN.%2BK.%26aulast%3DCarlson%26aufirst%3DS.%2BA.%26aulast%3DComstock%26aufirst%3DL.%2BR.%26atitle%3DSynthesis%2520and%2520evaluation%2520of%2520N6-substituted%2520azide-%2520and%2520alkyne-bearing%2520N-mustard%2520analogs%2520of%2520S-adenosyl-l-methionine%26jtitle%3DTetrahedron%26date%3D2014%26volume%3D70%26spage%3D5291%26epage%3D5297%26doi%3D10.1016%2Fj.tet.2014.05.055" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oudhoff, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tatlock, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freeman, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuznetsova, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marcellus, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kennedy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lambert, J.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Couzens, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aman, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gingras, A.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Awar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fish, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerstenberger, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benn, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grimley, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Braam, M. J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rossi, F. M. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sudol, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunnage, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Owen, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zaph, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span> </span><span class="NLM_article-title">(R)-PFI-2 is a potent and selective inhibitor of SETD7 methyltransferase activity in cells</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">111</span><span class="NLM_x">, </span> <span class="NLM_fpage">12853</span><span class="NLM_x">–</span> <span class="NLM_lpage">12858</span><span class="refDoi"> DOI: 10.1073/pnas.1407358111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1073%2Fpnas.1407358111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=25136132" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVartr7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=12853-12858&author=D.+Barsyte-Lovejoyauthor=F.+Liauthor=M.+J.+Oudhoffauthor=J.+H.+Tatlockauthor=A.+Dongauthor=H.+Zengauthor=H.+Wuauthor=S.+A.+Freemanauthor=M.+Schapiraauthor=G.+A.+Senisterraauthor=E.+Kuznetsovaauthor=R.+Marcellusauthor=A.+Allali-Hassaniauthor=S.+Kennedyauthor=J.-P.+Lambertauthor=A.+L.+Couzensauthor=A.+Amanauthor=A.-C.+Gingrasauthor=R.+Al-Awarauthor=P.+V.+Fishauthor=B.+S.+Gerstenbergerauthor=L.+Robertsauthor=C.+L.+Bennauthor=R.+L.+Grimleyauthor=M.+J.+S.+Braamauthor=F.+M.+V.+Rossiauthor=M.+Sudolauthor=P.+J.+Brownauthor=M.+E.+Bunnageauthor=D.+R.+Owenauthor=C.+Zaphauthor=M.+Vedadiauthor=C.+H.+Arrowsmith&title=%28R%29-PFI-2+is+a+potent+and+selective+inhibitor+of+SETD7+methyltransferase+activity+in+cells&doi=10.1073%2Fpnas.1407358111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">(R)-PFI-2 is a potent and selective inhibitor of SETD7 methyltransferase activity in cells</span></div><div class="casAuthors">Barsyte-Lovejoy, Dalia; Li, Fengling; Oudhoff, Menno J.; Tatlock, John H.; Dong, Aiping; Zeng, Hong; Wu, Hong; Freeman, Spencer A.; Schapira, Matthieu; Senisterra, Guillermo A.; Kuznetsova, Ekaterina; Marcellus, Richard; Allali-Hassani, Abdellah; Kennedy, Steven; Lambert, Jean-Philippe; Couzens, Amber L.; Aman, Ahmed; Gingras, Anne-Claude; Al-Awar, Rima; Fish, Paul V.; Gerstenberger, Brian S.; Roberts, Lee; Benn, Caroline L.; Grimley, Rachel L.; Braam, Mitchell J. S.; Rossi, Fabio M. V.; Sudol, Marius; Brown, Peter J.; Bunnage, Mark E.; Owen, Dafydd R.; Zaph, Colby; Vedadi, Masoud; Arrowsmith, Cheryl H.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">12853-12858</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">SET domain contg. (lysine methyltransferase) 7 (SETD7) is implicated in multiple signaling and disease related pathways with a broad diversity of reported substrates.  Here, we report the discovery of (R)-PFI-2-a first-in-class, potent (Kiapp = 0.33 nM), selective, and cell-active inhibitor of the methyltransferase activity of human SETD7-and its 500-fold less active enantiomer, (S)-PFI-2.  (R)-PFI-2 exhibits an unusual cofactor-dependent and substrate-competitive inhibitory mechanism by occupying the substrate peptide binding groove of SETD7, including the catalytic lysine-binding channel, and by making direct contact with the donor Me group of the cofactor, S-adenosylmethionine.  Chemoproteomics expts. using a biotinylated deriv. of (R)-PFI-2 demonstrated dose-dependent competition for binding to endogenous SETD7 in MCF7 cells pretreated with (R)-PFI-2.  In murine embryonic fibroblasts, (R)-PFI-2 treatment phenocopied the effects of Setd7 deficiency on Hippo pathway signaling, via modulation of the transcriptional coactivator Yes-assocd. protein (YAP) and regulation of YAP target genes.  In confluent MCF7 cells, (R)-PFI-2 rapidly altered YAP localization, suggesting continuous and dynamic regulation of YAP by the methyltransferase activity of SETD7.  These data establish (R)-PFI-2 and related compds. as a valuable tool-kit for the study of the diverse roles of SETD7 in cells and further validate protein methyltransferases as a druggable target class.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqv-HTBIcjMnbVg90H21EOLACvtfcHk0lixGWuZi-iLrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVartr7E&md5=0b8ad07ae23c4eba5cf11060d231b3f4</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1407358111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1407358111%26sid%3Dliteratum%253Aachs%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DOudhoff%26aufirst%3DM.%2BJ.%26aulast%3DTatlock%26aufirst%3DJ.%2BH.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DFreeman%26aufirst%3DS.%2BA.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DSenisterra%26aufirst%3DG.%2BA.%26aulast%3DKuznetsova%26aufirst%3DE.%26aulast%3DMarcellus%26aufirst%3DR.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DLambert%26aufirst%3DJ.-P.%26aulast%3DCouzens%26aufirst%3DA.%2BL.%26aulast%3DAman%26aufirst%3DA.%26aulast%3DGingras%26aufirst%3DA.-C.%26aulast%3DAl-Awar%26aufirst%3DR.%26aulast%3DFish%26aufirst%3DP.%2BV.%26aulast%3DGerstenberger%26aufirst%3DB.%2BS.%26aulast%3DRoberts%26aufirst%3DL.%26aulast%3DBenn%26aufirst%3DC.%2BL.%26aulast%3DGrimley%26aufirst%3DR.%2BL.%26aulast%3DBraam%26aufirst%3DM.%2BJ.%2BS.%26aulast%3DRossi%26aufirst%3DF.%2BM.%2BV.%26aulast%3DSudol%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DZaph%26aufirst%3DC.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26atitle%3D%2528R%2529-PFI-2%2520is%2520a%2520potent%2520and%2520selective%2520inhibitor%2520of%2520SETD7%2520methyltransferase%2520activity%2520in%2520cells%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2014%26volume%3D111%26spage%3D12853%26epage%3D12858%26doi%3D10.1073%2Fpnas.1407358111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Bromberg, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitchell, T. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Upadhyay, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jakob, C. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jhala, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Comess, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lasko, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tuzon, C. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eram, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nuber, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soni, N. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manaves, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Algire, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sweis, R. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Torrent, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schotta, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michaelides, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoemaker, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santhakumar, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rice, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiang, G. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pappano, W. N.</span><span> </span><span class="NLM_article-title">The SUV4–20 inhibitor A-196 verifies a role for epigenetics in genomic integrity</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">317</span><span class="NLM_x">–</span> <span class="NLM_lpage">324</span><span class="refDoi"> DOI: 10.1038/nchembio.2282</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1038%2Fnchembio.2282" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=28114273" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVKisbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=317-324&author=K.+D.+Brombergauthor=T.+R.+Mitchellauthor=A.+K.+Upadhyayauthor=C.+G.+Jakobauthor=M.+A.+Jhalaauthor=K.+M.+Comessauthor=L.+M.+Laskoauthor=C.+Liauthor=C.+T.+Tuzonauthor=Y.+Daiauthor=F.+Liauthor=M.+S.+Eramauthor=A.+Nuberauthor=N.+B.+Soniauthor=V.+Manavesauthor=M.+A.+Algireauthor=R.+F.+Sweisauthor=M.+Torrentauthor=G.+Schottaauthor=C.+Sunauthor=M.+R.+Michaelidesauthor=A.+R.+Shoemakerauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=V.+Santhakumarauthor=A.+Martinauthor=J.+C.+Riceauthor=G.+G.+Chiangauthor=M.+Vedadiauthor=D.+Barsyte-Lovejoyauthor=W.+N.+Pappano&title=The+SUV4%E2%80%9320+inhibitor+A-196+verifies+a+role+for+epigenetics+in+genomic+integrity&doi=10.1038%2Fnchembio.2282"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">The SUV4-20 inhibitor A-196 verifies a role for epigenetics in genomic integrity</span></div><div class="casAuthors">Bromberg, Kenneth D.; Mitchell, Taylor R. H.; Upadhyay, Anup K.; Jakob, Clarissa G.; Jhala, Manisha A.; Comess, Kenneth M.; Lasko, Loren M.; Li, Conglei; Tuzon, Creighton T.; Dai, Yujia; Li, Fengling; Eram, Mohammad S.; Nuber, Alexander; Soni, Niru B.; Manaves, Vlasios; Algire, Mikkel A.; Sweis, Ramzi F.; Torrent, Maricel; Schotta, Gunnar; Sun, Chaohong; Michaelides, Michael R.; Shoemaker, Alex R.; Arrowsmith, Cheryl H.; Brown, Peter J.; Santhakumar, Vijayaratnam; Martin, Alberto; Rice, Judd C.; Chiang, Gary G.; Vedadi, Masoud; Barsyte-Lovejoy, Dalia; Pappano, William N.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">317-324</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein lysine methyltransferases (PKMTs) regulate diverse physiol. processes including transcription and the maintenance of genomic integrity.  Genetic studies suggest that the PKMTs SUV420H1 and SUV420H2 facilitate proficient nonhomologous end-joining (NHEJ)-directed DNA repair by catalyzing the di- and trimethylation (me2 and me3, resp.) of lysine 20 on histone 4 (H4K20).  Here we report the identification of A-196, a potent and selective inhibitor of SUV420H1 and SUV420H2.  Biochem. and co-crystn. analyses demonstrate that A-196 is a substrate-competitive inhibitor of both SUV4-20 enzymes.  In cells, A-196 induced a global decrease in H4K20me2 and H4K20me3 and a concomitant increase in H4K20me1.  A-196 inhibited 53BP1 foci formation upon ionizing radiation and reduced NHEJ-mediated DNA-break repair but did not affect homol.-directed repair.  These results demonstrate the role of SUV4-20 enzymic activity in H4K20 methylation and DNA repair.  A-196 represents a first-in-class chem. probe of SUV4-20 to investigate the role of histone methyltransferases in genomic integrity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdOSfp0N-TMrVg90H21EOLACvtfcHk0liykgsjf2dInQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVKisbg%253D&md5=9e25d8894455f5db43a6bb6dc3492e71</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2282&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2282%26sid%3Dliteratum%253Aachs%26aulast%3DBromberg%26aufirst%3DK.%2BD.%26aulast%3DMitchell%26aufirst%3DT.%2BR.%26aulast%3DUpadhyay%26aufirst%3DA.%2BK.%26aulast%3DJakob%26aufirst%3DC.%2BG.%26aulast%3DJhala%26aufirst%3DM.%2BA.%26aulast%3DComess%26aufirst%3DK.%2BM.%26aulast%3DLasko%26aufirst%3DL.%2BM.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DTuzon%26aufirst%3DC.%2BT.%26aulast%3DDai%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DNuber%26aufirst%3DA.%26aulast%3DSoni%26aufirst%3DN.%2BB.%26aulast%3DManaves%26aufirst%3DV.%26aulast%3DAlgire%26aufirst%3DM.%2BA.%26aulast%3DSweis%26aufirst%3DR.%2BF.%26aulast%3DTorrent%26aufirst%3DM.%26aulast%3DSchotta%26aufirst%3DG.%26aulast%3DSun%26aufirst%3DC.%26aulast%3DMichaelides%26aufirst%3DM.%2BR.%26aulast%3DShoemaker%26aufirst%3DA.%2BR.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DSanthakumar%26aufirst%3DV.%26aulast%3DMartin%26aufirst%3DA.%26aulast%3DRice%26aufirst%3DJ.%2BC.%26aulast%3DChiang%26aufirst%3DG.%2BG.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DPappano%26aufirst%3DW.%2BN.%26atitle%3DThe%2520SUV4%25E2%2580%259320%2520inhibitor%2520A-196%2520verifies%2520a%2520role%2520for%2520epigenetics%2520in%2520genomic%2520integrity%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26spage%3D317%26epage%3D324%26doi%3D10.1038%2Fnchembio.2282" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Bailey, S.</span><span> </span><span class="NLM_article-title">The CCP4 suite - programs for protein crystallography</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">760</span><span class="NLM_x">–</span> <span class="NLM_lpage">763</span><span class="refDoi"> DOI: 10.1107/S0907444994003112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1107%2FS0907444994003112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=15299374" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=1994&pages=760-763&author=S.+Bailey&title=The+CCP4+suite+-+programs+for+protein+crystallography&doi=10.1107%2FS0907444994003112"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1107%2FS0907444994003112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444994003112%26sid%3Dliteratum%253Aachs%26aulast%3DBailey%26aufirst%3DS.%26atitle%3DThe%2520CCP4%2520suite%2520-%2520programs%2520for%2520protein%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1994%26volume%3D50%26spage%3D760%26epage%3D763%26doi%3D10.1107%2FS0907444994003112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">McCoy, A. J.</span><span> </span><span class="NLM_article-title">Solving structures of protein complexes by molecular replacement with Phaser</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">63</span><span class="NLM_x">, </span> <span class="NLM_fpage">32</span><span class="NLM_x">–</span> <span class="NLM_lpage">41</span><span class="refDoi"> DOI: 10.1107/S0907444906045975</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1107%2FS0907444906045975" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=17164524" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtlSqu77F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2007&pages=32-41&author=A.+J.+McCoy&title=Solving+structures+of+protein+complexes+by+molecular+replacement+with+Phaser&doi=10.1107%2FS0907444906045975"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Solving structures of protein complexes by molecular replacement with Phaser</span></div><div class="casAuthors">McCoy, Airlie J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">32-41</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Mol. replacement (MR) generally becomes more difficult as the no. of components in the asym. unit requiring sep. MR models (i.e. the dimensionality of the search) increases.  When the proportion of the total scattering contributed by each search component is small, the signal in the search for each component in isolation is weak or non-existent.  Maximum-likelihood MR functions enable complex asym. units to be built up from individual components with a 'tree search with pruning' approach.  This method, as implemented in the automated search procedure of the program Phaser, has been very successful in solving many previously intractable MR problems.  However, there are a no. of cases in which the automated search procedure of Phaser is suboptimal or encounters difficulties.  These include cases where there are a large no. of copies of the same component in the asym. unit or where the components of the asym. unit have greatly varying B factors.  Two case studies are presented to illustrate how Phaser can be used to best advantage in the std. 'automated MR' mode and two case studies are used to show how to modify the automated search strategy for problematic cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAvs9W8hQ3l7Vg90H21EOLACvtfcHk0liykgsjf2dInQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtlSqu77F&md5=323a5297bdf622ebaf398a87c39dc794</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1107%2FS0907444906045975&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444906045975%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26atitle%3DSolving%2520structures%2520of%2520protein%2520complexes%2520by%2520molecular%2520replacement%2520with%2520Phaser%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2007%26volume%3D63%26spage%3D32%26epage%3D41%26doi%3D10.1107%2FS0907444906045975" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Murshudov, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vagin, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodson, E. J.</span><span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">240</span><span class="NLM_x">–</span> <span class="NLM_lpage">255</span><span class="refDoi"> DOI: 10.1107/S0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method&doi=10.1107%2FS0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0liykgsjf2dInQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255%26doi%3D10.1107%2FS0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Afonine, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunkoczi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, I. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Echols, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hung, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapral, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moriarty, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oeffner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terwilliger, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zwart, P. H.</span><span> </span><span class="NLM_article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">213</span><span class="NLM_x">–</span> <span class="NLM_lpage">221</span><span class="refDoi"> DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunkocziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.+W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+a+comprehensive+Python-based+system+for+macromolecular+structure+solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0ljCXzOL-JAR9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunkoczi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.%2BW.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520a%2520comprehensive%2520Python-based%2520system%2520for%2520macromolecular%2520structure%2520solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Afonine, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Echols, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moriarty, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mustyakimov, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terwilliger, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Urzhumtsev, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zwart, P. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, P. D.</span><span> </span><span class="NLM_article-title">Towards automated crystallographic structure refinement with phenix.refine</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">68</span><span class="NLM_x">, </span> <span class="NLM_fpage">352</span><span class="NLM_x">–</span> <span class="NLM_lpage">367</span><span class="refDoi"> DOI: 10.1107/S0907444912001308</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1107%2FS0907444912001308" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=22505256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BC38Xlt1Ggu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2012&pages=352-367&author=P.+V.+Afonineauthor=R.+W.+Grosse-Kunstleveauthor=N.+Echolsauthor=J.+J.+Headdauthor=N.+W.+Moriartyauthor=M.+Mustyakimovauthor=T.+C.+Terwilligerauthor=A.+Urzhumtsevauthor=P.+H.+Zwartauthor=P.+D.+Adams&title=Towards+automated+crystallographic+structure+refinement+with+phenix.refine&doi=10.1107%2FS0907444912001308"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Towards automated crystallographic structure refinement with phenix.refine</span></div><div class="casAuthors">Afonine, Pavel V.; Grosse-Kunstleve, Ralf W.; Echols, Nathaniel; Headd, Jeffrey J.; Moriarty, Nigel W.; Mustyakimov, Marat; Terwilliger, Thomas C.; Urzhumtsev, Alexandre; Zwart, Peter H.; Adams, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">352-367</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phenix.refine is a program within the PHENIX package that supports crystallog. structure refinement against exptl. data with a wide range of upper resoln. limits using a large repertoire of model parameterizations.  It has several automation features and is also highly flexible.  Several hundred parameters enable extensive customizations for complex use cases.  Multiple user-defined refinement strategies can be applied to specific parts of the model in a single refinement run.  An intuitive graphical user interface is available to guide novice users and to assist advanced users in managing refinement projects.  X-ray or neutron diffraction data can be used sep. or jointly in refinement. phenix.refine is tightly integrated into the PHENIX suite, where it serves as a crit. component in automated model building, final structure refinement, structure validation and deposition to the wwPDB.  This paper presents an overview of the major phenix.refine features, with extensive literature refs. for readers interested in more detailed discussions of the methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnhn9QwIEkTbVg90H21EOLACvtfcHk0ljCXzOL-JAR9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xlt1Ggu7c%253D&md5=1cc7da2cea396f1a75646ea709140cf1</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1107%2FS0907444912001308&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444912001308%26sid%3Dliteratum%253Aachs%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DMustyakimov%26aufirst%3DM.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DUrzhumtsev%26aufirst%3DA.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26atitle%3DTowards%2520automated%2520crystallographic%2520structure%2520refinement%2520with%2520phenix.refine%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2012%26volume%3D68%26spage%3D352%26epage%3D367%26doi%3D10.1107%2FS0907444912001308" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">2126</span><span class="NLM_x">–</span> <span class="NLM_lpage">2132</span><span class="refDoi"> DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0ljCXzOL-JAR9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Chen, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arendall, W. B.,  3rd</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keedy, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Immormino, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapral, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, D. C.</span><span> </span><span class="NLM_article-title">MolProbity: all-atom structure validation for macromolecular crystallography</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">12</span><span class="NLM_x">–</span> <span class="NLM_lpage">21</span><span class="refDoi"> DOI: 10.1107/S0907444909042073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=10.1107%2FS0907444909042073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=20057044" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;key=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=12-21&author=V.+B.+Chenauthor=W.+B.+Arendallauthor=J.+J.+Headdauthor=D.+A.+Keedyauthor=R.+M.+Immorminoauthor=G.+J.+Kapralauthor=L.+W.+Murrayauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=MolProbity%3A+all-atom+structure+validation+for+macromolecular+crystallography&doi=10.1107%2FS0907444909042073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">MolProbity: all-atom structure validation for macromolecular crystallography</span></div><div class="casAuthors">Chen, Vincent B.; Arendall, W. Bryan, III; Headd, Jeffrey J.; Keedy, Daniel A.; Immormino, Robert M.; Kapral, Gary J.; Murray, Laura W.; Richardson, Jane S.; Richardson, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12-21</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">MolProbity is a structure-validation web service that provides broad-spectrum solidly based evaluation of model quality at both the global and local levels for both proteins and nucleic acids.  It relies heavily on the power and sensitivity provided by optimized hydrogen placement and all-atom contact anal., complemented by updated versions of covalent-geometry and torsion-angle criteria.  Some of the local corrections can be performed automatically in MolProbity and all of the diagnostics are presented in chart and graphical forms that help guide manual rebuilding.  X-ray crystallog. provides a wealth of biol. important mol. data in the form of at. three-dimensional structures of proteins, nucleic acids and increasingly large complexes in multiple forms and states.  Advances in automation, in everything from crystn. to data collection to phasing to model building to refinement, have made solving a structure using crystallog. easier than ever.  However, despite these improvements, local errors that can affect biol. interpretation are widespread at low resoln. and even high-resoln. structures nearly all contain at least a few local errors such as Ramachandran outliers, flipped branched protein side chains and incorrect sugar puckers.  It is crit. both for the crystallographer and for the end user that there are easy and reliable methods to diagnose and correct these sorts of errors in structures.  MolProbity is the authors' contribution to helping solve this problem and this article reviews its general capabilities, reports on recent enhancements and usage, and presents evidence that the resulting improvements are now beneficially affecting the global database.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIjgsh59dO-7Vg90H21EOLACvtfcHk0lhr3TvSmC7zhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D&md5=b5fc7574f43f01dd6e43c3663ca4f779</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1107%2FS0907444909042073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909042073%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DArendall%26aufirst%3DW.%2BB.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DKeedy%26aufirst%3DD.%2BA.%26aulast%3DImmormino%26aufirst%3DR.%2BM.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DMurray%26aufirst%3DL.%2BW.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMolProbity%253A%2520all-atom%2520structure%2520validation%2520for%2520macromolecular%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D12%26epage%3D21%26doi%3D10.1107%2FS0907444909042073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ROD" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ROD','PDB','3ROD'); return false;">PDB: 3ROD</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2IIP" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2IIP','PDB','2IIP'); return false;">PDB: 2IIP</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3QOW" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3QOW','PDB','3QOW'); return false;">PDB: 3QOW</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3TG4" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3TG4','PDB','3TG4'); return false;">PDB: 3TG4</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6B1A" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6B1A','PDB','6B1A'); return false;">PDB: 6B1A</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5XVQ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5XVQ','PDB','5XVQ'); return false;">PDB: 5XVQ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5XVK" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5XVK','PDB','5XVK'); return false;">PDB: 5XVK</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i26"><a href="/doi/suppl/10.1021/acs.jmedchem.7b01422">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_20778"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.7b01422">10.1021/acs.jmedchem.7b01422</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline"><sup>1</sup>H and <sup>13</sup>C NMR spectra of compound <b>6</b>; Docking analysis of <b>7</b> in the binding sites of hNNMT; Docking analysis of <b>6</b> in the binding sites of DOT1L and SMYD2; Kinetic parameter determination for NNMT; Binding affinity of SAH to hNNMT and binding affinity of nicotinamide to hNNMT in the presence of SAH; Overlay of the crystal structure of the hNNMT (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6B1A">6B1A</a>) with the crystal structure of mouse NNMT (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5XVK">5XVK</a>); Proteins in the methyltransferase selectivity panel; Crystallography data and refinement statistics (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b01422/suppl_file/jm7b01422_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b01422/suppl_file/jm7b01422_si_002.csv">CSV</a>)</p></li><li><p class="inline">Docking process movie for <b>6</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b01422/suppl_file/jm7b01422_si_003.mpg">MPG</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01422/suppl_file/jm7b01422_si_001.pdf">jm7b01422_si_001.pdf (1.27 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01422/suppl_file/jm7b01422_si_002.csv">jm7b01422_si_002.csv (0.25 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01422/suppl_file/jm7b01422_si_003.mpg">jm7b01422_si_003.mpg (46.27 MB)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The structure of human NNMT in complex with <b>6</b> has been deposited under PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6B1A">6B1A</a>. Authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;pbContext=%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2018.61.issue-4%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.7b01422%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.7b01422" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67996d4beaf43c52","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
